
<html lang="en"     class="pb-page"  data-request-id="c24f55ad-61df-4419-ab8f-a889f3288334"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00327;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Macrolide Inspired Macrocycles as Modulators of the IL-17A/IL-17RA Interaction" /></meta><meta name="dc.Creator" content="Sanja  Koštrun" /></meta><meta name="dc.Creator" content="Andrea  Fajdetić" /></meta><meta name="dc.Creator" content="Dijana  Pešić" /></meta><meta name="dc.Creator" content="Karmen  Brajša" /></meta><meta name="dc.Creator" content="Vlatka  Bencetić Mihaljević" /></meta><meta name="dc.Creator" content="Dubravko  Jelić" /></meta><meta name="dc.Creator" content="Adriana  Petrinić Grba" /></meta><meta name="dc.Creator" content="Ivaylo  Elenkov" /></meta><meta name="dc.Creator" content="Renata  Rupčić" /></meta><meta name="dc.Creator" content="Samra  Kapić" /></meta><meta name="dc.Creator" content="Ivana  Ozimec Landek" /></meta><meta name="dc.Creator" content="Kristina  Butković" /></meta><meta name="dc.Creator" content="Ana  Grgičević" /></meta><meta name="dc.Creator" content="Dinko  Žiher" /></meta><meta name="dc.Creator" content="Ana  Čikoš" /></meta><meta name="dc.Creator" content="Jasna  Padovan" /></meta><meta name="dc.Creator" content="Gordon  Saxty" /></meta><meta name="dc.Creator" content="Kevin  Dack" /></meta><meta name="dc.Creator" content="Haakan  Bladh" /></meta><meta name="dc.Creator" content="Tine  Skak-Nielsen" /></meta><meta name="dc.Creator" content="Simon  Feldbaek Nielsen" /></meta><meta name="dc.Creator" content="Maja  Lambert" /></meta><meta name="dc.Creator" content="Martin  Stahlhut" /></meta><meta name="dc.Description" content="Interleukin 17 (IL-17) cytokines promote inflammatory pathophysiology in many autoimmune diseases, including psoriasis, multiple sclerosis, rheumatoid arthritis, and inflammatory bowel disease. Suc..." /></meta><meta name="Description" content="Interleukin 17 (IL-17) cytokines promote inflammatory pathophysiology in many autoimmune diseases, including psoriasis, multiple sclerosis, rheumatoid arthritis, and inflammatory bowel disease. Suc..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 8, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00327" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00327" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00327" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00327" /></link>
        
    
    

<title>Macrolide Inspired Macrocycles as Modulators of the IL-17A/IL-17RA Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00327" /></meta><meta property="og:title" content="Macrolide Inspired Macrocycles as Modulators of the IL-17A/IL-17RA Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0022.jpeg" /></meta><meta property="og:description" content="Interleukin 17 (IL-17) cytokines promote inflammatory pathophysiology in many autoimmune diseases, including psoriasis, multiple sclerosis, rheumatoid arthritis, and inflammatory bowel disease. Such broad involvement of IL-17 in various autoimmune diseases makes it an ideal target for drug discovery. Psoriasis is a chronic inflammatory disease characterized by numerous defective components of the immune system. Significantly higher levels of IL-17A have been noticed in lesions of psoriatic patients, if compared to non-lesion parts. Therefore, this paper is focused on the macrolide inspired macrocycles as potential IL-17A/IL-17RA modulators and covers the molecular design, synthesis, and in vitro profiling. Macrocycles are designed to diversify and enrich chemical space through different ring sizes and a variety of three-dimensional shapes. Inhibitors in the nM range were identified in both target-based and phenotypic assays. In vitro ADME as well as in vivo PK properties are reported." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00327"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00327">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00327&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00327&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00327&amp;href=/doi/10.1021/acs.jmedchem.1c00327" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8354-8383</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00305" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00380" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Macrolide Inspired Macrocycles as Modulators of the IL-17A/IL-17RA Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sanja Koštrun</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sanja Koštrun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#becddfd0d4df90d5d1cdcacccbd0fed8d7dadbd2cadf90dbcb"><span class="__cf_email__" data-cfemail="6211030c08034c090d111610170c22040b06070e16034c0717">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sanja++Ko%C5%A1trun">Sanja Koštrun</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2827-1964" title="Orcid link">https://orcid.org/0000-0003-2827-1964</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrea Fajdetić</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Fajdetić</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Fajdeti%C4%87">Andrea Fajdetić</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dijana Pešić</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dijana Pešić</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dijana++Pe%C5%A1i%C4%87">Dijana Pešić</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Karmen Brajša</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Karmen Brajša</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Karmen++Braj%C5%A1a">Karmen Brajša</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vlatka Bencetić Mihaljević</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vlatka Bencetić Mihaljević</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vlatka++Benceti%C4%87+Mihaljevi%C4%87">Vlatka Bencetić Mihaljević</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dubravko Jelić</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dubravko Jelić</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dubravko++Jeli%C4%87">Dubravko Jelić</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adriana Petrinić Grba</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adriana Petrinić Grba</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adriana++Petrini%C4%87+Grba">Adriana Petrinić Grba</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ivaylo Elenkov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ivaylo Elenkov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ivaylo++Elenkov">Ivaylo Elenkov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renata Rupčić</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renata Rupčić</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renata++Rup%C4%8Di%C4%87">Renata Rupčić</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samra Kapić</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samra Kapić</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samra++Kapi%C4%87">Samra Kapić</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ivana Ozimec Landek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ivana Ozimec Landek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ivana++Ozimec+Landek">Ivana Ozimec Landek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kristina Butković</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kristina Butković</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kristina++Butkovi%C4%87">Kristina Butković</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ana Grgičević</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ana Grgičević</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ana++Grgi%C4%8Devi%C4%87">Ana Grgičević</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dinko Žiher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dinko Žiher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dinko++%C5%BDiher">Dinko Žiher</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ana Čikoš</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ana Čikoš</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ana++%C4%8Ciko%C5%A1">Ana Čikoš</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jasna Padovan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jasna Padovan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jasna++Padovan">Jasna Padovan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gordon Saxty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gordon Saxty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fidelta Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gordon++Saxty">Gordon Saxty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kevin Dack</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kevin Dack</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">LEO Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d5beb1b1b1be95b9b0baf8a5bdb4a7b8b4fbb6bab8"><span class="__cf_email__" data-cfemail="69020d0d0d0229050c06441901081b0408470a0604">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kevin++Dack">Kevin Dack</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haakan Bladh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haakan Bladh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">LEO Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haakan++Bladh">Haakan Bladh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tine Skak-Nielsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tine Skak-Nielsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">LEO Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tine++Skak-Nielsen">Tine Skak-Nielsen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simon Feldbaek Nielsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon Feldbaek Nielsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">LEO Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Feldbaek+Nielsen">Simon Feldbaek Nielsen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maja Lambert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maja Lambert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">LEO Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maja++Lambert">Maja Lambert</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Martin Stahlhut</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Stahlhut</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">LEO Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Stahlhut">Martin Stahlhut</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00327&amp;href=/doi/10.1021%2Facs.jmedchem.1c00327" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8354–8383</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 8, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 February 2021</li><li><span class="item_label"><b>Published</b> online</span>8 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00327" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00327</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8354%26pageCount%3D30%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSanja%2BKo%25C5%25A1trun%252C%2BAndrea%2BFajdeti%25C4%2587%252C%2BDijana%2BPe%25C5%25A1i%25C4%2587%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.1c00327%26title%3DMacrolide%2BInspired%2BMacrocycles%2Bas%2BModulators%2Bof%2Bthe%2BIL-17A%252FIL-17RA%2BInteraction%26numPages%3D30%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8383%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00327"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">718</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00327" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Macrolide Inspired Macrocycles as Modulators of the IL-17A/IL-17RA Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sanja&quot;,&quot;last_name&quot;:&quot;Koštrun&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Fajdetić&quot;},{&quot;first_name&quot;:&quot;Dijana&quot;,&quot;last_name&quot;:&quot;Pešić&quot;},{&quot;first_name&quot;:&quot;Karmen&quot;,&quot;last_name&quot;:&quot;Brajša&quot;},{&quot;first_name&quot;:&quot;Vlatka&quot;,&quot;last_name&quot;:&quot;Bencetić Mihaljević&quot;},{&quot;first_name&quot;:&quot;Dubravko&quot;,&quot;last_name&quot;:&quot;Jelić&quot;},{&quot;first_name&quot;:&quot;Adriana&quot;,&quot;last_name&quot;:&quot;Petrinić Grba&quot;},{&quot;first_name&quot;:&quot;Ivaylo&quot;,&quot;last_name&quot;:&quot;Elenkov&quot;},{&quot;first_name&quot;:&quot;Renata&quot;,&quot;last_name&quot;:&quot;Rupčić&quot;},{&quot;first_name&quot;:&quot;Samra&quot;,&quot;last_name&quot;:&quot;Kapić&quot;},{&quot;first_name&quot;:&quot;Ivana&quot;,&quot;last_name&quot;:&quot;Ozimec Landek&quot;},{&quot;first_name&quot;:&quot;Kristina&quot;,&quot;last_name&quot;:&quot;Butković&quot;},{&quot;first_name&quot;:&quot;Ana&quot;,&quot;last_name&quot;:&quot;Grgičević&quot;},{&quot;first_name&quot;:&quot;Dinko&quot;,&quot;last_name&quot;:&quot;Žiher&quot;},{&quot;first_name&quot;:&quot;Ana&quot;,&quot;last_name&quot;:&quot;Čikoš&quot;},{&quot;first_name&quot;:&quot;Jasna&quot;,&quot;last_name&quot;:&quot;Padovan&quot;},{&quot;first_name&quot;:&quot;Gordon&quot;,&quot;last_name&quot;:&quot;Saxty&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;Dack&quot;},{&quot;first_name&quot;:&quot;Haakan&quot;,&quot;last_name&quot;:&quot;Bladh&quot;},{&quot;first_name&quot;:&quot;Tine&quot;,&quot;last_name&quot;:&quot;Skak-Nielsen&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Feldbaek Nielsen&quot;},{&quot;first_name&quot;:&quot;Maja&quot;,&quot;last_name&quot;:&quot;Lambert&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Stahlhut&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8354-8383&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00327&quot;},&quot;abstract&quot;:&quot;Interleukin 17 (IL-17) cytokines promote inflammatory pathophysiology in many autoimmune diseases, including psoriasis, multiple sclerosis, rheumatoid arthritis, and inflammatory bowel disease. Such broad involvement of IL-17 in various autoimmune diseases makes it an ideal target for drug discovery. Psoriasis is a chronic inflammatory disease characterized by numerous defective components of the immune system. Significantly higher levels of IL-17A have been noticed in lesions of psoriatic patients, if compared to non-lesion parts. Therefore, this paper is focused on the macrolide inspired macrocycles as potential IL-17A/IL-17RA modulators and covers the molecular design, synthesis, and in vitro profiling. Macrocycles are designed to diversify and enrich chemical space through different ring sizes and a variety of three-dimensional shapes. Inhibitors in the nM range were identified in both target-based and phenotypic assays. In vitro ADME as well as in vivo PK properties are reported.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00327&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00327" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00327&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00327" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00327&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00327" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00327&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00327&amp;href=/doi/10.1021/acs.jmedchem.1c00327" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00327" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00327" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00327%26sid%3Dliteratum%253Aachs%26pmid%3D34100601%26genre%3Darticle%26aulast%3DKo%25C5%25A1trun%26date%3D2021%26atitle%3DMacrolide%2BInspired%2BMacrocycles%2Bas%2BModulators%2Bof%2Bthe%2BIL-17A%252FIL-17RA%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8354%26epage%3D8383%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290925" title="Carbohydrates">Carbohydrates</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Interleukin 17 (IL-17) cytokines promote inflammatory pathophysiology in many autoimmune diseases, including psoriasis, multiple sclerosis, rheumatoid arthritis, and inflammatory bowel disease. Such broad involvement of IL-17 in various autoimmune diseases makes it an ideal target for drug discovery. Psoriasis is a chronic inflammatory disease characterized by numerous defective components of the immune system. Significantly higher levels of IL-17A have been noticed in lesions of psoriatic patients, if compared to non-lesion parts. Therefore, this paper is focused on the macrolide inspired macrocycles as potential IL-17A/IL-17RA modulators and covers the molecular design, synthesis, and <i>in vitro</i> profiling. Macrocycles are designed to diversify and enrich chemical space through different ring sizes and a variety of three-dimensional shapes. Inhibitors in the nM range were identified in both target-based and phenotypic assays. <i>In vitro</i> ADME as well as <i>in vivo</i> PK properties are reported.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Psoriasis is a chronic immune-mediated inflammatory disease with global prevalence of up to 4%.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> In some countries, it exceeds 10%, making it a serious global problem, with at least 100 million individuals affected worldwide. Plaque psoriasis, which is considered as a chronic autoimmune condition, is the most common form of the disease, accounting for 80–90% of all cases. The major driver of plaque psoriasis is the IL-23/IL-17A immunological pathway with interleukin-17A being a critical cytokine involved in pathological changes in affected tissue.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Significantly, elevated levels of IL-17A have not only been detected in psoriatic lesional plaques, but also in non-lesional skin of patients, accompanied with keratinocytes proliferation, recruitment,, and activation of immune cells such as mast cells, neutrophils, and macrophages.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Interleukin-17A is the founding member of a cytokine family composed of six members (IL-17A-F). In comparison to other family members, biological activity of IL-17A has been studied the most. The important cellular source of IL-17A are hematopoietic cells, Th17 (CD4<sup>+</sup> T cells), Tc17 (cytotoxic CD8<sup>+</sup> T cells), T cells, NK cells, ILC3 (group 3 innate lymphoid cells), and “natural” Th17 cells (CD4<sup>+</sup> RORγT<sup>+</sup> thymocytes).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> IL-17 proteins play protective roles in host defense against pathogens at epithelial and mucosal barriers and mediate immunological function by inducing pro-inflammatory cytokines (such as IL-1, IL-6, and IL-21) and chemokines (such as CXCL1), secretion by responder cells that triggers the recruitment of innate immune cells to the site of infection and elimination of the pathogen.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Although crucial in host defense, deregulation of IL-17A production can result in excessive pro-inflammatory cytokine production, which can lead to tissue damage and chronic inflammation. In the skin of psoriatic patients, Th17 cells produce excess amounts of IL-17A, resulting in significantly higher levels of cytokines in lesions and worsened disease activity.</div><div class="NLM_p">IL-17 family cytokines mediate their biological functions via surface receptors on target cells. IL-17RA was the first identified IL-17 receptor, and four other IL-17R family members (IL-17R B-E) followed. Functional receptors for IL-17 family cytokines often exist in the form of heterodimers, with IL-17RA as a common subunit.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> IL-17RA is expressed in all cell types, particularly on fibroblasts, epithelial cells, and innate immune cells, such as macrophages and neutrophils, and it primarily binds IL-17A (human IL-17RA).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> IL-17A, IL-17A/F, and IL-17F bind to a receptor complex consisting of one IL-17RA and one IL-17RC subunit, which interact via SEFIR domains and mediate signaling through an Act1-dependent pathway, culminating in the activation of pro-inflammatory factors such as NF-κB.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">Currently, several monoclonal antibodies targeting IL-17A or IL-17 receptor are approved by the FDA for the treatment of moderate-to-severe plaque psoriasis:<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> secukinumab (Cosentyx, Novartis), ixekizumab (Taltz, Eli Lilly), and brodalumab (Kyntheum, LEO Pharma/AstraZeneca; Siliq, Amgen/AstraZeneca) while bimekizumab (UCB) is a bispecific anti-IL-17A/IL-17F humanized monoclonal antibody that is still in clinical development.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">However, since monoclonal antibodies have a number of disadvantages, such as high cost-of-goods, non-oral applications, poor tissue penetration, and often long half-lives,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> the search for small molecules that would exhibit the same biological outcome is still ongoing.</div><div class="NLM_p">Recent drug discovery activities have been directed toward designing new artificial macrocycles as potent inhibitors of the IL-17A/IL-17RA interactions.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Ensemble Therapeutics made a breakthrough in targeting the IL-17A/IL-17RA epitope by preparing DNA-encoded libraries of synthetic macrocycles.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> These synthetic macrocycles showed activity in cellular <i>in vitro</i> assays as inhibitors of IL-17A-driven IL-6 production in human rheumatoid arthritis synovial fibroblasts (RASF) and human adenocarcinoma cells (HT29). More importantly, binding to human IL-17A was demonstrated.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Linear peptides were also identified as potent IL-17A inhibitors.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> To our knowledge, the most advanced Ensemble compounds are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. Furthermore, Pfizer also successfully designed potent macrocycle derivatives and obtained their high-resolution co-crystal structure with IL-17A (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Eli Lilly, on the other hand, used a peptide phage display approach to generate a peptide library of potential IL-17A modulators.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Two series of peptides resulted from affinity maturation of peptides 585-1 and 18-1, proposed to bind to the α-helix and β-sheet region of IL-17A dimer, respectively. The two most active peptides obtained from this approach are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, and the most noteworthy 585 analogue is the one reported in the Protein Data Bank (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known linear and macrocyclic IL-17A inhibitors.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17 ref18">(12−18)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As a compound class, macrocycles belong to “The Middle” chemical space with structural and physico-chemical properties positioned between small molecules and biologicals. Although most of the time they do not follow Lipinski’s rule-of-five,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> they do fit into the druggable physicochemical property space.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The structural complexity of macrocycles results in conformational richness that determines the physicochemical and biological properties in a way that differs from the small-molecule chemical space. The 3D character of macrocycles results in a population of bioactive geometries, conformational mimicry, and conformational adaptation to different environments resulting in non-obvious properties (like permeability and oral bioavailability) that could not be deduced from the 2D structure alone.</div><div class="NLM_p">The majority of approved macrocyclic drugs belong either to the class of macrolides, polyketide class of natural products, or cyclic peptides,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> with each class carrying its own benefits and liabilities. Alongside the proven clinical relevance, macrolide stereochemical and functional richness is a long-term challenge and inspiration for synthetic organic chemists.<a onclick="showRef(event, 'ref20 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref22">(20,22)</a> In contrast, cyclic peptides are often readily synthesized but usually suffer from metabolic instability and poor oral bioavailability.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a></div><div class="NLM_p">In order to combine and utilize the advantages of both the macrolide and cyclic peptide chemical classes a hybrid technology, named FideltaMacro, was applied<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and a diverse library of macrolide inspired macrocycles was generated as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Macrolide fragment can be coupled with a variety of natural or synthetic fragments ranging from amino acids known to interact with PPI hot-spots, different heterocycles, alkyl substituents, or sugars. This technology is exemplified here through preparation of IL-17A/IL-17RA macrocyclic disruptors. Macrocycles are designed to diversify and enrich chemical space with different ring sizes, a variety of 3D shapes, and potential pharmacophoric features to modulate challenging targets. Introduction of the macrolide fragment bearing sugar moieties and their possible modifications was expected to be beneficial for optimizing PK properties of designed compounds.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Modular Synthesis of Novel Macrocyclic Scaffolds Starting from Natural or Semi-Synthetic Macrolides<a onclick="showRef(event, 'ref20 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref25">(20,25)</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">This paper covers the design, synthesis, and <i>in vitro</i> profiling of the macrolide inspired macrocycles designed as potential disruptors of the IL-17A/IL-17RA protein–protein interaction (PPI). We report discovery of compounds active in the nM range, in both target binding assays, and an IL-17 stimulated human primary keratinocyte cell-based assay. The identified best compounds were further profiled in <i>in vitro</i> ADME assays and <i>in vivo</i> PK studies forming a good basis for further investigation and development of this class of compounds as potential orally bioavailable agents.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05817" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05817" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Design of the macrolide inspired macrocyclic inhibitors of IL-17A was based on in-house modeling, ligand conformational experiments, and the available crystal structures, as well as the literature data available at the time this work was carried out<a onclick="showRef(event, 'ref13 ref16 ref17 ref18 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref13 ref16 ref17 ref18 ref26 ref27 ref28">(13,16−18,26−28)</a> with an aim to optimize both potency as well as ADME properties.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> State-of-the-Art and Initial Design</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Structure-Based Insights</h4><div class="NLM_p">In the last decade, a number of structural information about the apo structure of IL-17A protein, its interaction with IL-17RA, as well as linear and macrocyclic inhibitors were published (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref17 ref18 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref26 ref27 ref28">(17,18,26−28)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Crystal Structures of IL-17A Dimer Currently Available in the Protein Data Bank</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">structure</th><th class="colsep0 rowsep0" align="center">PDB code</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer apo<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="left">4HR9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A/IL-17R complex<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="left">4HSA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer in complex with antibody</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer/CAT-2000<sup>27</sup></td><td class="colsep0 rowsep0" align="left">2VXS</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer/FAB 6785</td><td class="colsep0 rowsep0" align="left">4QHU</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer in complex with antibody and inhibitor</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer/CAT-2200/linear peptide<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td><td class="colsep0 rowsep0" align="left">5HHX</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer/CAT-2000/linear inhibitor<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></td><td class="colsep0 rowsep0" align="left">5HI3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17A dimer/CAT-2200/macrocyclic inhibitor<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></td><td class="colsep0 rowsep0" align="left">5HI4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IL-17 dimer in complex with macrocyclic peptide<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></td><td class="colsep0 rowsep0" align="left">5VB9</td></tr></tbody></table></div></div><div class="NLM_p">IL-17A binds as a homodimer to IL-17R,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> burying a total of 2220 Å<sup>2</sup> of surface area (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a>). There are three major regions on the IL-17A dimer that interact with the IL-17 receptor, noted as site 1, site 2, and site 3 in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Site 2, formed by the central β-strands of IL-17A, that interact with the D1 domain of IL-17RA was proposed as the most probable region for inhibitor binding, based on the mass spectroscopy deuterium exchange studies<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> of the Ensemble macrocyclic compound 159 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and the co-crystal structure of a disulfide linked macrocyclic inhibitor published by Eli Lilly.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The same central binding pocket (site 2) of the IL-17A dimer was also identified by the Pfizer group<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> as a potentially druggable pocket, which could eventually open up, due to protein dynamics, upon the binding of the inhibitor. Pfizer authors subsequently revealed that their macrocyclic inhibitor is actually binding to a new binding pocket formed by disrupting the Il-17A dimer structure. <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> shows the overlap of the linear peptide inhibitor (yellow)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and macrocyclic compound 2 (green) published by Pfizer (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a>)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> as well as macrocyclic peptide inhibitor (magenta) published by Eli Lilly (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a>)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> complexed with the IL-17A homodimer. While the macrocyclic peptide inhibitor from the 5VB9 structure binds in the central β-sheet region (site 2), MC compound 2 from the 5HI4 structure binds between two IL-17A monomers. Pfizer linear peptide (HAP) binds close to the site 1, opposite to the antibody binding region in vicinity of site 3. Aligned binding areas of PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a> and PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a> with key amino acids are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Figure S7</a>.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Available X-ray structures of the IL-17A homodimer. Possible binding sites according to the PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> structure (blue and gray surfaces); IL-17A in complex with macrocyclic inhibitors, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a> (green)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> aligned with PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a> (magenta)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HHX">5HHX</a> (yellow)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> structures; MC inhibitors are shown by colored van der Waals’ surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Initial compound design and docking experiments were carried out using the IL-17A dimer structure from its complex with IL-17RA (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>). Publication of the newer crystal structures (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>) revealed that Eli Lilly’s cyclic peptide binds in site 2, in agreement with our docking hypothesis. In order to explore if FideltaMacro scaffolds could bind analogously to the Pfizer macrocyclic inhibitor, retrospective docking analysis has been performed as discussed later.</div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Ligand-Based Insights</h4><div class="NLM_p">SAR around the tool compound (MC compound 159, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> is indicated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, emphasizing the importance of the cyclopentyl moiety as well as the linear peptide chain for the observed activity. Aliphatic chain of the macrocycle tolerates polar, charged, and bulky substituents, indicating that it is probably directed toward the solvent. Therefore, as the initial step, small hydrophobic moieties resembling the cyclopentyl group and linear peptide chain were incorporated in the FideltaMacro scaffold using <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and are presented in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>a.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. NMR constrained 3D structure of the tool compound and corresponding SAR features, aligned with predicted conformation of compound <b>2</b> (a). Predicted binding pose of the tool compound (magenta) docked into PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and aligned with Eli Lilly’s cyclic peptide (orange) from PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (3D presentation (b) and 2D projection of protein–ligand interactions (c)).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Strategy for the Preparation of Compounds <b>1</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Symbols Q, X, Y, Z, R1, R2, R3, R4, and LSC are described in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Reagents and conditions: (i) Pb(OAc)<sub>4</sub>, AcOH. (ii) <i>N</i>-Fmoc-amino acid, HATU, HOAt, TMP, DCM, r.t. (iii) piperidine, DCM, r.t. (iv) <i>N</i>-Fmoc-amino acid, HATU, HOAt, TMP, DCM, r.t. (v) LiOH, THF/water, r.t. (vi) HATU, DIPEA, DMF, r.t. (vii) Pd/C; hydrogen, MeOH, r.t. (viii) <b>64</b>, HATU, DIPEA, DMF, r.t.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Compounds Prepared According to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> (a) and <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> (b, c)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0017.gif" alt="" id="gr20" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0018.gif" alt="" id="gr17" /></img><div></div></div><div class="NLM_p last">In order to get a reliable solution-state 3D structure of the tool compound, to be used as the starting point for docking, an NMR constrained conformational analysis was performed. Conformational sampling was restricted by distance and angle constraints generated from nOe interactions (NOESY) and coupling constants (<sup>3</sup><i>J</i><sub>H,H</sub>) given in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S1</a>. The spiro-cyclopentyl motif of the macrocycle is well characterized through the network of strong nOe contacts, providing evidence of the relative rigidity of this part of the molecule. The conformation of the macrocyclic ring is further stabilized by the presence of four amide bonds, all in the <i>trans</i> configuration. The aliphatic chain region, on the other hand, could accommodate different conformations because it contains freely rotating string of methylene groups. These two opposite sides of the macrocycle exhibit some weak interactions, while the linear peptide side chain has no relevant interactions with the macrocycle free rotation in an extended conformation. Obtained results indicate transferability of the linear side chain to a different macrocyclic ring. The obtained 3D conformation aligned with the predicted conformation for compound <b>2</b> and main SAR findings is shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, indicating the rationale for the design of the FideltaMacro scaffolds.</div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Docking Hypothesis</h4><div class="NLM_p">The predicted conformation of the tool compound was used to perform initial docking experiments into the 4HSA structure. Partial structure minimization and stepwise docking protocol was used as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Methods">Experimental Methods</a> section. Docking poses that agreed with the known SAR were selected for further minimization using the MM-GBSA method and ranked according to the predicted Δ<i>G</i>. Computational experiments converged to the potential binding mode for the tool compound as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The tool compound is positioned in the IL-17A binding pocket with the cyclopentyl moiety filling the small hydrophobic subpocket, the macrocycle being placed in the central part of the binding area forming a hydrogen bond with GluA95. Nitrogen from the terminal amino acid forms a hydrogen bond with AspB42 at the rim of the binding pocket, and the Cl atom from the central phenyl ring forms a halogen hydrogen bond with TyrB43. The chlorophenyl moiety from the linear side chain is placed in the hydrophobic subpocket formed by TyrB43, TyrB44, TrpB51, LeuB53, and IleA92, the first phenyl ring forms hydrophobic interactions with LeuA116 while the terminal phenyl ring is placed in the vicinity of the ArgB39 on the rim of the binding area.</div><div class="NLM_p last">It is also possible that the phenylalanine and the terminal phenyl rings interchange positions depending on the nature and position of the substituents. The flexible aliphatic part of the macrocyclic ring is oriented toward the solvent in agreement with the observed SAR since polar and charged substituents are tolerated in that region.</div></div><div id="sec2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Synthesis of Compounds</h4><div class="NLM_p">Macrocyclic compounds presented in this paper were synthesized using the FideltaMacro approach (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The first step is macrocyclic ring opening of <b>A</b> by oxidative cleavage<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> of the 11,12-diol moiety with lead(IV) acetate to afford the seco compound <b>B</b>, which is suitable for further modification of the secondary amino group and insertion of desired motifs, fragments, potential pharmacophores, and hot-spots from protein–protein interactions. Intermediate <b>B</b> still contains the 11C-13C fragment of the macrolide, which serves as a protecting group for the carboxylic acid group on 1C atom. The amidation of <b>B</b> was performed under standard reaction conditions with corresponding protected amino acids. After deprotection of the amino group, an amide was formed that is suitable either for further modification of inserted fragment (insertion of second amino acid) or for macrocyclization. After subsequent deprotection a seco amino acid of type <b>C</b> was formed and subjected to macrocyclization under pseudo-high-dilution amidation conditions.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> It is worth mentioning that, in this case, at least one of the inserted amino acids was <i>p</i>-nitro α/β-phenylalanine, while the other contains an aliphatic side chain. Once the macrocycles <b>D</b> were formed, reduction of the nitro group in the side chain gave the desired intermediates <b>E</b>. The remaining aromatic moieties of the target molecules <b>1–27</b> were introduced by amidation with dipeptide <b>64</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Intermediate <b>E-1a</b> was also used for the preparation of compounds <b>28–64</b><named-content content-type="anchor" rid="sch2" type="simple"></named-content>(<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). In some cases (compounds <b>34</b> and <b>40</b>) the side chains were introduced in a similar way from prepared dipeptides (<b>65</b> and <b>66</b>), while in most cases synthesis included two subsequent sets of amidation/deprotection reactions (<b>28–45</b> and <b>56–64</b>). Ureas <b>48–55</b> and sulfonamide <b>47</b> were similarly prepared from intermediate <b>E-1b</b> using an appropriate isocyanate or sulfonyl chloride, respectively. Compounds prepared by the described synthetic routes are shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Compounds in SMILES notation are given in the Supporting information (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S2</a>).</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Strategy for the Preparation of Compounds <b>28</b>–<b>64</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Fmoc-2-chloro-<span class="smallcaps smallerCapital">l</span>-phenylalanine, HATU, DIPEA, DMF, r.t. (ii) piperidine, DCM, r.t. (iii) amidation: corresponding acid, HATU, DIPEA, DMF, r.t.; urea formation: corresponding isocyanate, DCM, r.t.; sulfonamide formation: corresponding sulfonyl chloride, TEA., DCM, r.t. (iv) corresponding Fmoc-protected amino acid HATU, DIPEA, DMF, r.t.</p></p></figure></div><div id="sec2_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Initial SAR Exploration</h4><div class="NLM_p">The initial SAR exploration started by hybridizing the seco-macrolide scaffold using hot-spots revealed earlier by a modeling work. Three main areas were optimized through (1) linear side chain modifications, (2) insertions of different α- and β-amino acids to change the ring sizes of the macrocycle, as well as to diversify stereochemistry, and (3) changes in the sugar moieties as shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Medicinal chemistry exploration of FideltaMacro IL-17 macrocyclic disruptors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To identify antagonists of IL-17/IL-17RA protein binding, two protein binding ELISA biochemical assays were developed. In the first ELISA type (ELISA-I), recombinant human IL-17A was coated on plates and incubated with compounds. Thereafter, the receptor was bound and a peroxidase-labeled antibody against human IL-17R/Fc was added. Finally, detection included binding of a highly sensitive colorimetric substrate for peroxidase detection (details are described in the Supporting information and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Figure S1</a>). In the second ELISA assay (ELISA-II), the IL-17A receptor was added to plates pre-coated with anti-human IgG followed by addition of compounds and recombinant human IL-17A homodimer, biotinylated rhIL-17 antibody and, finally, streptavidin-labeled peroxidase. Two ELISA assays were set up to cover potential binding to both, IL-17A protein as well as to IL-17A receptor and to explore if there is the difference of compounds binding to immobilized and solution state proteins.</div><div class="NLM_p last">Results of the ELISA-I primary assay are discussed below and shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S3</a>. Most active compounds were further profiled in ELISA-II, and their efficacy was tested with the psoriasis-like human keratinocyte assay. With keratinocytes being the most common cell type in epidermis and their pathologically altered in psoriasis, human keratinocytes were used to confirm the anti-inflammatory effect by measuring IL-8 secretion.</div></div><div id="sec2_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Optimization of the Macrocyclic Ring</h4><div class="NLM_p">Aiming to explore different macrocycle sizes, stereochemistry, and order of inserted amino acids, the linear tripeptide side chain was combined with 1,2-cyclopentyl, spiro-cyclopentyl, and proline moieties as shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>a.</div><div class="NLM_p">The most encouraging result in the 1,2-cyclopentyl series was observed for the β<i>-</i>(<i>R</i>)-Phe, as a first aromatic motif of the side chain connected to the (<i>R</i>,<i>R</i>)-1,2-cyclopentyl within the macrocycle (<b>1</b>) having an IC<sub>50</sub> of 5.9 μM in ELISA-I and 8.2 μM in the ELISA-II assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). A comparable activity for the spiro-cyclopentyl series was achieved for compound <b>2</b>, where the CH<sub>2</sub>-spiro-cyclopentyl derivative replaces the cyclopentyl motif and the side chain requires the opposite stereochemistry (<i>S</i>)-β-Phe. Other derivatives containing (<i>R</i>)-β-Phe were not active. The proline analogs did not exhibit any significant activity, and only compound <b>3</b> showed μM activity in the ELISA-II assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Other explored cycloalkyl modifications are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S3a</a>. Switching the position of amino acids and changing stereochemistry resulted in inactive compounds (<b>S1–S10</b>), demonstrating again tight SAR sensitive to the orientation of the side chains. Removal of the cladinose (<b>62</b>) and desozamine (<b>63</b>) sugars, as well as changing the 1,2-cyclopentyl (<b>1</b>) to 1,2-cyclohexyl moiety (<b>S8</b>), resulted in a diminished activity.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Most active macrocycles with inserted dipeptide containing cyclopentyl/proline moieties. Compounds are synthesized following the general synthetic pathway described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Since further modifications of the cyclopentyl and proline moieties to six-membered rings (cyclohexane and piperidine) were not tolerated (<b>S7–S10</b>), efforts were switched to explore smaller hydrophobic groups with macrocyclic ring size and stereochemistry modifications (<b>4–27</b>). Compounds containing different combinations of (<i>R</i>)/(<i>S</i>)-α/β-Phe, (<i>R</i>)/(<i>S</i>)-Val, and (<i>R</i>)/(<i>S</i>)-β/β2-homo-Val resulted in several active compounds. 16-membered macrocycle compound <b>21</b> and 18-membered compound <b>25</b> showed an activity of 1.4 and 2.9 μM in ELISA-I, respectively. Interestingly, the (<i>R</i>)-β-Phe-(<i>S</i>)-Val dipeptide combination in 17-membered compound <b>4</b> showed an activity of 170 nM. Also, compound <b>14</b> with an inverse order of amino acids and stereochemistry ((<i>R</i>)-Val-(<i>S</i>)-β-Phe) exhibited an activity of 256 nM. A strong stereochemical influence was observed since the (<i>R</i>)-β-Phe-(<i>R</i>)-Val dipeptide combination also showed no activity. Likewise, already mentioned compound <b>21</b>, with the (<i>R</i>)-α-Phe-(<i>S</i>)-Val combination within 16-membered macrocycle, was 10-fold less active than compound <b>4</b> as shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. However, attempts to replace Val with Ala, Leu and Gly as well as cyclopropyl, <i>t</i>-butyl, ethyl, and propyl as side chains resulted in no or weak μM activity (compounds <b>8</b>–<b>13</b>).</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Most active compounds resulted from the optimization of the ring size, stereochemistry, and order of inserted amino acids. Compounds are synthesized following the general synthetic pathway described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a><b>.</b></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With the first nanomolar compound in hand, the project focused on the structural investigations of compound <b>4</b>. The study involved a detailed conformational analysis and investigation of possible binding interactions with IL-17A. Due to the low solubility in aqueous medium, the conformational analysis was performed in CDCl<sub>3</sub> at 45 °C, where it exhibited two rotational isomers (around 9N—10C═O amide bond) with a molar ratio 90:10. The presence of these two species in conformational exchange resulted in large signal overlap in the proton spectrum and few extractable coupling constants.</div><div class="NLM_p">Fortunately, the NOESY spectrum showed correlation signals that mainly belonged to the major conformer and allowed a more comprehensive nOe analysis. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> shows the key nOe contacts used in the constrained conformational search and the resulting three-dimensional structure of compound <b>4</b> that is well defined by three <i>trans</i>-amide bonds, two sugar moieties rigidifying right-hand side of the macrocycle, and (<i>R</i>)-β-Phe-(<i>S</i>)-Val defining the left-hand side. Valine residue seems to be pointing toward the cladinose sugar showing weak nOe contacts between its methyl groups and 1″H. The sugar moieties adopt their usual Everett–Tyler chair conformation,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> but the presence of only one weak inter-sugar contact (2′H/5″Me) indicates significantly different orientations of the sugars to the macrocyclic ring in comparison to the parent compound azithromycin.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In fact, its sugar moieties are placed wider apart with the desosamine sugar positioned perpendicular to the macrocyclic ring. In agreement with data obtained for the tool compound, a linear side chain and its substituents have no significant contacts with the macrocyclic ring, suggesting independent free rotation. Conformational sampling was restricted by distance and angle constraints generated from nOe interactions (NOESY) shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a, and coupling constants (<sup>1</sup>H) given in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S4</a>. Full assignment of compound <b>4</b> is given in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S5</a>, and it was used in epitope mapping studies described below.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Conformational analysis of compound <b>4</b>: (a) 2D structure with key nOe contacts used as constraints in the modeling process and (b) predicted 3D conformation (sugars, green; linear side chain, magenta; macrocycle, gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A 3D solution-state structure of compound <b>4</b>, determined by NMR constrained conformational analysis, was used in the docking experiments into the IL-17A dimer from the 4HSA X-ray structure (binding hypothesis 1). <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a shows docking posed of compound <b>4</b> aligned with the previously discussed docking pose of tool compound, and <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b shows 2D projection of compound <b>4</b> interacting with the IL-17A dimer. The docking protocol developed for the tool compound was used. Macrocycles are well aligned, as well as the linear side chain, with the valine moiety occupying same small hydrophobic pocket as the spiro-cyclopentyl from the tool compound, as well as the Leu side chain from Eli Lilly’s macrocyclic peptide. Furthermore, Arg15 from Eli Lilly’s compound overlaps with the desosamine basic dimethylamino group of compound <b>4</b>, providing an additional interaction spot with the polar rim of the binding pocket. Three hydrogen bonds are formed between the N–H from macrocyclic amide bonds and Glu85(A), Lys11(A), and Tyr62(B). Additional hydrogen bonds are formed by oxygen from the first amide bond in the linear side chain and Ser40(B) and the Cl atom from the central phenyl ring with Gln93(A). The first phenyl ring is buried within the hydrophobic pocket, while the terminal phenyl ring points to the solvent. Sugar moieties are oriented toward the rim of the binding area.</div><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Predicted docking poses of compound <b>4</b> (blue) generated by hypothesis 1: docked into PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a> and aligned with tool compound (magenta) (3D presentation (a) and 2D projection of protein–ligand interactions (b)) and hypothesis 2: docked into PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a> and aligned with a macrocyclic compound (green) from Pfizer (3D presentation (c) and 2D projection of protein–ligand interactions (d)).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to explore possibility of presented macrocycles binding in a similar way to Pfizer MC compounds by disrupting the IL-17A dimer structure, compound <b>4</b> was also retrospectively docked in the 5HI4 X-ray structure. Characteristic H-bonds observed in the X-ray structure are used as constrains in the docking experiments. The predicted binding pose is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>c, and the 2D projection with major protein–ligand interactions in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>d. Aromatic rings from the linear side chain are well aligned with Pfizer’s compound, amides forming four hydrogen bonds with Leu97 and the valine side chain being placed in the vicinity of the cyclopentyl moiety. The macrocyclic ring and sugar moieties are placed in the adjacent shallow subpocket. The macrocyclic amide forms an additional hydrogen bond with Trp67 and Lys114. Sugar moieties are also involved in the network of hydrogen bonds; desosamine ring with Glu95 and cladinose with Tyr62.</div><div class="NLM_p">Although binding pockets from 4HSA and 5HI4 are quite different, major interactions of compound <b>4</b> with the IL-17A dimer are similar in terms of the side chain interactions. Aromatic rings are placed in the hydrophobic subpockets, and amide groups are involved in the network of hydrogen bonds. The central and terminal aromatic ring of the side chain could be interchanged in both cases. However, in binding to the 4HSA structure, the macrocyclic ring has a dominant contribution by forming three hydrogen bonds and having the valine moiety placed in the buried hydrophobic subpocket in line with the observed strong stereochemistry influence. Binding to the 5HI4 structure is more biased toward the interactions of the linear side chain, in agreement with smaller and more constrained Pfizer’s macrocycle, as well as previously reported activity of analogous linear compounds. Sugars also have a more important role, spanning the binding over two adjacent subpockets. With the current information in hand, both binding hypotheses could be valid and further structural investigation is needed in order to confirm binding mode of FideltaMacro IL-17A inhibitors. However, binding hypothesis to the 4HSA structure provided useful guidance in the design of potent IL-17A macrocyclic inhibitors.</div><div class="NLM_p">In order to further explore interactions of compound <b>4</b> with the IL-17A homodimer and validate the proposed binding hypotheses, STD NMR<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> was performed. It presents a solution-state alternative to X-ray analysis when probing interactions between ligands and targets. The biggest challenge in designing STD NMR experiments for the macrocycle/human IL-17A dimer was finding experimental conditions, which would be optimal for both the ligand and protein with respect to their poor solubility and incompatible stability in acidic conditions. The results confirmed that compound <b>4</b> binds to the recombinant human IL-17A dimer (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a). Although it was not possible to determine which part of the protein the molecule binds to, valuable information was obtained about ligand binding through epitope mapping. The percentage of STD enhancement indicated which protons in the molecule have received the biggest saturation transfer and, hence, have been closest to the surface of the protein during the interaction.</div><figure id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Epitope mapping by STD NMR: (a) proton spectrum (black), STD NMR spectrum without human IL-17A homodimer (red), and STD NMR spectrum with hIL-17A protein (green); (b) STD NMR enhancements in compound <b>4</b> interacting with hIL-17A; (c) 3D conformation with STD enhancements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">From the results shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b, it was evident that the motif closest to the IL-17A is the linear side chain with the strongest STD enhancements observed in the first and terminal phenyl moieties, as well as medium STD enhancements in the central chlorophenyl ring. When looking at the macrocyclic ring, the valine residue immediately stands out, confirming the placement of this group very close to the protein surface, most probably in a small hydrophobic pocket. The sugar moieties showed low STD enhancements on the dimethylamino group of desosamine and the 3″OMe group of cladinose, suggesting that they may be oriented toward the protein surface. STD enhancements should be interpreted in terms of areas of the ligand rather than looking into each group or proton individually since effects of overlap and water suppression may influence the value of STD enhancement. Strong STD enhancements of the terminal phenyl moiety may indicate that the adjacent phenyl rings are interchanged as already pointed out during docking exercise of tool compound and compound <b>4</b>.</div></div><div id="sec2_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Optimization of Linear Peptidic Side Chain</h4><div class="NLM_p">Further exploration of the peptidic linear side chain was performed not only to optimize binding activity but also activity in the functional assay as well as ADME properties. Modifications encompassed the replacement of the aromatic rings with aliphatic groups, introduction of smaller and more polar heterocycles, diversifying the substitution patterns, and modification or replacement of the amide bonds. Design of the proposed modifications was based on the insights from the STD NMR experiment for compound <b>4</b> as well as usual medicinal chemistry isosteric and bioisosteric replacements. Target molecules were prioritized by docking experiments and <i>in silico</i> predictions of physico-chemical properties in order to increase interaction with the protein, optimize lipophilicity, and facilitate potential conformational changes that would influence permeability through the biological membranes and metabolic stability. Modifications of the side chain are depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.</div><div class="NLM_p">Replacement of the terminal phenyl ring with linear and cyclic aliphatic substituents as well as with pyridine and thiazole significantly reduced activity. Likewise, different solubilizing groups were not tolerated, indicating that the terminal phenyl ring binds within the lipophilic pocket (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S3</a>). Therefore, we focused to explore differently substituted phenyl moieties (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>b). <i>Meta</i>-substitution of the terminal amino acid led to compounds <b>30</b> and <b>33</b> with improvement in activity over compound <b>4</b>. Small substituents such as methyl and halogens increase the activity, while larger and more polar substituents were not tolerated. Insertion of methyl and di-F into the methylene bridge was tolerated, while OH and the cyclopropyl moiety lowered the activity to the μM region. Elongation of the methylene bridge also reduced the activity.</div><div class="NLM_p">Regarding the central aromatic ring, (<i>S</i>)-epimer is required for the activity. Explored phenyl substitutions all resulted in a weaker activity than for the <i>o</i>-Cl substitution of compound <b>4</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>c). Methylation of amide nitrogen atoms lead to completely inactive compounds, indicating the importance of hydrogen bonding in agreement with both proposed binding modes. Due to the synthetic challenges, limited modifications of the first phenyl ring were explored. However, exclusion of the first phenyl ring, replacement with triazole and piperidine moieties, as well as extension with benzoxazole led to inactive compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S3d</a>).</div><div class="NLM_p">The terminal amide bond was replaced with a sulfonamide, urea, carbamate or amine, and in combination with the terminal aromatic moiety ureas and carbamates were well tolerated. Synthesized compounds are listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>b, and the most active compounds are shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>. Compounds showed good activity in both ELISA assays, as well as in the keratinocyte-based bioassay for IL-17A inhibition. Compounds <b>46</b> and <b>48</b> are most active with an IC<sub>50</sub>(ELISA-I) of 59 and 40 nM and IC<sub>50</sub>(IL-8) of 147 and 420 nM, respectively. Replacement of the terminal phenyl ring with a methyl (compound <b>59</b>) results in the 3-fold decrease in activity in comparison with compound <b>4</b>. However, <i>in vitro</i> ADME properties for compound <b>59</b> improved over compound <b>4</b> as shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><figure id="fig10" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Advanced compounds resulted from the modification of the terminal amino acid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>In Vitro</i> ADME Properties for the Most Advanced Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">assay</th><th class="colsep0 rowsep0" align="center"><b>4</b></th><th class="colsep0 rowsep0" align="center"><b>48</b></th><th class="colsep0 rowsep0" align="center"><b>49</b></th><th class="colsep0 rowsep0" align="center"><b>56</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinetic solubility (μM)</td><td class="colsep0 rowsep0" align="left">10–30</td><td class="colsep0 rowsep0" align="left">10–30</td><td class="colsep0 rowsep0" align="left">30–100</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChromLogD</td><td class="colsep0 rowsep0" align="left">4.4</td><td class="colsep0 rowsep0" align="left">4.4</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hPPB (% bound)</td><td class="colsep0 rowsep0" align="left">94.2</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">95.6</td><td class="colsep0 rowsep0" align="left">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rCYP3A4 inhibition DBF/7BQ (<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="32627b7c720302">[email protected]</a> μM)</td><td class="colsep0 rowsep0" align="left">29/26</td><td class="colsep0 rowsep0" align="left">28/53</td><td class="colsep0 rowsep0" align="left">3/24</td><td class="colsep0 rowsep0" align="left">6/11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDCKII-MDR1 P<sub>app</sub>(A2B) (×10<sup>–6</sup> cm/s)</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">0.3</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">blood stability mouse/human (%<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e89a8d85a8d9">[email protected]</a> h)</td><td class="colsep0 rowsep0" align="left">1/100</td><td class="colsep0 rowsep0" align="left">100/100</td><td class="colsep0 rowsep0" align="left">69/54</td><td class="colsep0 rowsep0" align="left">83/100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">liver microsome stability mouse/human (% liver blood flow)</td><td class="colsep0 rowsep0" align="left">78/44</td><td class="colsep0 rowsep0" align="left">78/54</td><td class="colsep0 rowsep0" align="left">45/6</td><td class="colsep0 rowsep0" align="left">51/25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cytotoxicity in THP-1 cells (IC<sub>50</sub>/μM)</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hERG (IC<sub>50</sub>/μM)</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr></tbody></table></div></div></div><div id="sec2_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Sugar Modifications</h4><div class="NLM_p">In order to investigate the influence of sugar moieties on activity of compound <b>4</b>, compounds without the cladinose or both sugars were prepared and tested. The initial hypothesis was that sugars might be beneficial for binding since the rim of the PPI interaction site is usually polar and charged. Sugars are also known to influence the pharmacokinetics of original macrolide compounds.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Therefore, having a desosamine with an ionizable dimethylamino group and cladinose with additional hydrogen bonding potential is expected to be favorable for the activity. Removal of the cladinose sugar reduced the activity 5.4-fold, while removal of both sugars reduces the activity 3.2-fold in ELISA-I with the same ranking found in the cellular IL-8 assay (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). These results confirm the importance of sugar moieties for the binding and functional activity of prepared compounds.</div><figure id="fig11" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Influence of the sugar moieties on the activity of compound <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>In Vitro</i> ADME Profiling of Advanced Compounds</h4><div class="NLM_p">In order to further profile the most promising compounds, <i>in vitro</i> ADME properties were characterized and are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_p">Compounds were characterized by low to moderate kinetic solubility and very low passive permeability (P<sub>app</sub>(A2B) < 2 × 10<sup>–6</sup> cm/s). Most compounds exhibit moderate PPB binding, as expected from their complex 3D structure;<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> however, binding to plasma proteins was increased by the lipophilic nature of the side chain for compounds <b>4</b>, <b>48</b>, and <b>49</b>. For compound <b>56</b>, which has a terminal phenyl group replaced by methyl PPB, this dropped to 62.6%. A moderate to high clearance was predicted from mouse and human liver microsomes, except for compounds <b>49</b> and <b>56</b>, which have low predicted clearance from human liver microsomes. Compounds <b>49</b> and <b>56</b> were also stable in mouse and human blood. A higher instability in mouse vs human blood was observed for compounds <b>4</b> and <b>48</b>. Cleavage of the amide bond was confirmed, with intermediate <b>E1-a</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>) identified as the major metabolite, suggesting that replacement of the first amide group in the linear side chain could increase blood stability. However, structural modifications aimed to replace this metabolic weak spot resulted in the loss of activity as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S3</a> d and e.</div><div class="NLM_p">Screening for CYP3A4 inhibition in recombinant enzymes revealed a low potential (<50% at 10 μM) for DBF and 7BQ as probe substrates; however, inhibition using the BFC substrate was higher than 50%, suggesting that further confirmation in human liver microsomes using clinical probe substrates is required. Compounds were non-toxic on THP-1 cells, and advanced compounds were also clean on hERG.</div><div class="NLM_p">The disposition kinetics and oral bioavailability were investigated for compounds <b>4</b> and <b>49</b>, following intravenous bolus (1 mg/kg) and oral dosing (5 and/or 10 mg/kg) to male CD-1 mice. Compound <b>4</b> was characterized by a high clearance (66 mL/min/kg, 73% liver blood flow), in line with the predicted <i>in vivo</i> hepatic clearance from liver microsomes, a moderate volume of distribution (Vss 2.3 L/kg) and low oral bioavailability following dosing at both 5 and 10 mg/kg (ca. 1%). Compound <b>49</b> was characterized by a high clearance (66 mL/min/kg, 77% liver blood flow), higher than the predicted <i>in vivo</i> hepatic clearance from liver microsomes, a moderate volume of distribution (Vss 3.1 L/kg), half-life of 0.9 h, and low oral bioavailability (ca. 1%).</div><div class="NLM_p">Based on the good oral bioavailability of macrolides, especially azithromycin, used as a starting point for FideltaMacro scaffolds, a low permeability of presented compounds is most likely cumulative effect of chemical instability of the cladinose sugar in acidic conditions, increased molecular weight, and large number of hydrogen bond donors in agreement with the oral bioavailability thresholds determined by Khilberg<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and Villar<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S6</a>). Different strategies for optimization of oral bioavailability of macrocycles have been published including methylation of amide bonds, introduction of heterocyclic constrains, side chain optimizations, intra-molecular/trans-annular hydrogen bonds, and stereochemistry variations.<a onclick="showRef(event, 'ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39 ref40">(36−40)</a> All these strategies, as a net result, influenced the 3D structure of macrocycles, enabling shielding of polar and/or metabolically unstable groups and solvent-induced conformational changes. It has been also demonstrated that the 3D structure and consequently physico-chemical properties and membrane permeability of macrolides are highly influenced by the ring decorations.<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a> Importance of the 3D structure as well as induced conformational changes on the oral absorption of macrocycles were quantified by Whitty et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> They concluded that, for high MW compounds, there is no single polar surface area (PSA) cutoff value that can simultaneously satisfy the conditions of 0.2*MW ≤ PSA ≤ 140 Å<sup>2</sup>, required for orally bioavailable small molecules. Therefore, in order to achieve oral bioavailability, high MW compounds must exert some kind of chameleonic properties by which part of PSA will be hidden. A minimum PSA that a chameleonic compound must conceal to have both good aqueous solubility and good membrane permeability was estimated from Eqn (1), as shown in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>. The amount of concealed PSA is 107 Å<sup>2</sup>, which is higher than a ΔPSA threshold of 92 Å<sup>2</sup>, indicating that analogues of compound <b>4</b> might exhibit a chameleonic behavior and thereby improved permeability through biological membranes.</div><figure id="fig12" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Van der Waals’ surface of compound <b>4</b> color-coded by electrostatic potential used for the calculation of 3D polar surface area (PSA) followed by estimation of the potential chameleonic behavior based on the difference between minimal and buried ΔPSA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although modifications explored in this paper, like amide bond methylation and replacement, introduction of heterocyclic constrains, reduction in MW, stereochemistry, and ring-size variations did not result in orally bioavailable compounds, collected data indicate that further optimization of <i>in vivo</i> pharmacokinetics of the compounds described in this paper might be possible. Medicinal chemistry focuses on larger modifications of the linear side chain using scaffold hopping to reduce the number of amide bonds, aromatic rings, rotational bonds, and HBD are expected to yield to improved permeability and metabolic stability while still maintaining good inhibition of IL-17A. Furthermore, optimization of different coupling procedures of the azithromycin fragment with the linear peptide chain would result in diverse macrocycles with a decreased number of amide bonds as well as HBD and HBA. Influence of sugar modifications and methylation of hydroxy groups on the macrocycle on the permeability as well as activity still remains to be explored as well.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36048" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36048" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Discovery of novel macrolide inspired macrocycles as effective inhibitors of IL-17/IL-17RA protein binding with anti-inflammatory activity, as confirmed by the keratinocyte phenotypic assay, has been demonstrated. Broad SAR was explored including a variety of macrocyclic ring sizes, stereochemistry combinations, and modifications of the linear side chain, yielding to low nM inhibitors. Structural insights into possible binding interactions have been described and two binding hypotheses have been proposed. The most advanced compounds were characterized by low plasma-protein binding, moderate microsomal clearance, and blood stability, as well as low permeability. In order to achieve oral bioavailability, further medicinal chemistry explorations are required. This paper contributes to the body of literature demonstrating that preparation of diverse synthetic and semi-synthetic macrocyclic compounds as modulators of challenging protein–protein interactions is an effective strategy to bridge the space between biologicals and small molecules.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54042" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54042" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Methods and Compound Characterization</h4><div class="NLM_p">All solvents and chemicals purchased from commercial suppliers where used as received. All reactions involving air- or moisture-sensitive reagents were performed under a nitrogen atmosphere using dry solvents. Proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR spectra were recorded at 25 °C, unless otherwise specified. Solutions were typically prepared in deuterated dimethyl sulfoxide (DMSO-<i>d</i><sub>6</sub>), deuterated chloroform (CDCl<sub>3</sub>), or deuterated methanol (CD<sub>3</sub>OD) with chemical shifts referenced to a solvent as an internal standard. <sup>1</sup>H NMR data are reported, indicating the chemical shift (δ), the multiplicity (s, singlet; d, doublet; t, triplet; q, quartet; p, quintet; m, multiplet; br, broad; dd, doublet of doublets; etc.), the coupling constant (<i>J</i>) in Hz, and the integration (e.g., 1H). Thin layer chromatography (TLC) was carried out on silica gel 60 F<sub>254</sub> glass plates and visualized with 5% ethanolic solution of H<sub>2</sub>SO<sub>4</sub>. Chromatographic purifications were affected by medium pressure (“flash”) chromatography on silica gel unless noted otherwise.</div><div class="NLM_p">Purification of lead compounds was performed using preparative high-performance liquid chromatography (HPLC) using a Waters Auto-purification system (Waters Corporation, Milford, MA, USA) equipped with a Waters XBridge MS C18 OBD 150 × 30 nm (5 μm d.p.) stationary phase (Waters Corporation, Milford, MA, USA) using 10 mM ammonium bicarbonate in water at pH 10 and acetonitrile as eluents. For mass directed mass spectrometry, Waters ZQ equipped with an ESI source was used.</div><div class="NLM_p">Preparative HPLC was performed with a Waters Fraction Lynx system with integrated MS detection or Gilson GX-281 with integrated UV detection using Waters Sunfire C18 OBD 5 μm, XBridge BEH C18 OBD 5 μm, Xselect CSH C18 OBD 5 μm, or Kromasil C8 10 μm columns.</div><div class="NLM_p">Due to the synthetic complexity of reported compounds, purity threshold was set to 90%. Compounds <b>5</b>, <b>8</b>, and <b>23</b> have lower purity and, since they were inactive, re-synthesis was not done. Compounds are included in the paper for the completeness of SAR discussions.</div><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> [(1<i>R</i>)-1-Ethyl-2-oxo-propyl](2<i>R</i>,3<i>S</i>,4<i>S</i>,5<i>R</i>,6<i>R</i>,8<i>R</i>)-5-[(2<i>S</i>,3<i>R</i>,4<i>S</i>,6<i>R</i>)-4-(dimethylamino)-3-hydroxy-6-methyltetrahydropyran-2-yl]oxy-6-hydroxy-3-[(2<i>R</i>,4<i>R</i>,5<i>S</i>,6<i>S</i>)-5-hydroxy-4-methoxy-4,6-dimethyltetrahydropyran-2-yl]oxy-2,4,6,8-tetramethyl-9-(methylamino)nonanoate (<b>B</b>)</h5><div class="NLM_p last">To a solution of <b>A</b> (azitromycin, CAS 83905-01-5, 20 g, 26.7 mmol) in acetic acid (glacial, 150 mL), lead(IV) acetate 23.7 g, 53.45 mmol) was added and the reaction mixture was stirred at room temperature for 1.5 h. The solvent was evaporated, and the residue was dissolved in DCM (100 mL) and water (40 mL). The pH of the water layer was adjusted to 6.8 (40% NaOH), layers separated, and the organic layer washed with water. Combined water layers were basified to pH 8 (40% NaOH) and extracted with diethyl ether (3 × 100 mL) and then with DCM/isopropanol (3 × 150 mL). Combined DCM layers were dried over anh. Na<sub>2</sub>SO<sub>4</sub> and solvent evaporated. The crude product was purified by column chromatography (silica gel, eluent DCM:MeOH:NH<sub>4</sub>OH = 90:15:1.5) to yield the title product (14.1 g, yield 76%, purity 79%). ES+ <i>m</i>/<i>z</i> 691.6 [MH]<sup>+</sup>.</div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure 1 for the Introduction of the First Amino Acid (<b>C-1a</b> to <b>C-6a</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S7</a>)</h4><div class="NLM_p last">To a solution of an appropriate Fmoc-protected amino acid (1.1 eq) in dry DCM, TMP (2.5 eq), HOAt (1.1 eq), and HATU (1.1 eq) were added. After stirring for 1 h at room temperature, amine <b>B</b> (1 eq) was added and stirring was continued overnight. The reaction mixture was washed with sat. NaHCO<sub>3</sub> and brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure and the crude product. The crude product was dissolved in dry DCM, piperidine (5 eq) was added, and the resulting mixture was stirred at room temperature overnight. The reaction mixture was concentrated under reduced pressure and purified by flash column chromatography DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent to give <b>C-1a</b> to <b>C-6a</b>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Procedure 2 for the Introduction of the Second Amino Acid Moiety (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S8</a>)</h4><div class="NLM_p last">To a solution of appropriate Fmoc-protected amino acid (1.1 eq) in dry DCM, TMP (2.5 eq), HOAt, (1.1 eq) and HATU (1.1 eq) were added. After stirring for 1 h at room temperature, intermediate <b>C-1a</b> (1 eq) was added and stirring was continued at room temperature for 20 h. The reaction mixture was washed with sat. NaHCO<sub>3</sub> and brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure. The crude product was dissolved in THF, water solution of LiOH (5 eq) was added dropwise, and the resulting mixture was stirred at room temperature overnight. The solvent was evaporated, and the residue was dissolved in water and extracted with DCM. The pH of the water layer was adjusted at pH 4 and extracted with DCM. After extraction in acidic media, the pH of the water layer is adjusted at pH 10 and extracted with DCM/<i>i</i>-PrOH (2:1). Combined organic layers (obtained by extraction at pH 10) were dried over Na<sub>2</sub>SO<sub>4</sub> and solvent evaporated under reduced pressure to yield the desired intermediate (<b>C-1b</b> to <b>C-1o</b>).</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure 3 for Macrocyclization (<b>D-1</b> to <b>D-27</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S9</a>)</h4><div class="NLM_p last">A DMF solution of an appropriate intermediate from <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S8</a> (1 eq, 60–150 mM) and DIPEA (2 eq) was added (dropwise over 30–60 min) to a DMF (anhydrous) solution of HATU (1.5 eq, 100–120 mM), and the reaction mixture was stirred at room temperature for additional 1 h. EtOAc was added, and the organic layer was washed with sat. NaHCO<sub>3</sub> and brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure yielding a crude material, which was purified by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:0.5 as an eluent to give intermediates <b>D-1</b> to <b>D-27</b>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> General Procedure 4 for the Synthesis of Intermediates E (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S10</a>)</h4><div class="NLM_p">A mixture of corresponding analogue <b>D</b> (1 eq) and palladium on carbon (10% wt.) in MeOH was stirred under a hydrogen atmosphere (2.5 bar) at room temperature overnight. The reaction mixture was filtered through Celite and washed with MeOH and filtrate concentrated, and the residue was suspended in EtOAc. The organic layer was washed with sat. NaHCO<sub>3</sub>, brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure to give corresponding intermediate <b>E</b>.</div><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> (<i>S</i>)-3-(2-Chlorophenyl)-2-[(2-phenylacetyl)amino]propanoic Acid (<b>64</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Scheme S7</a>)</h5><div class="NLM_p last">A round-bottom flask was charged under argon with phenylacetic acid (816 mg, 6 mmol), (2<i>S</i>)-methyl-2-amino-3-(2-chlorophenyl)propionate hydrochloride (1 g, 4 mmol), and DMF (5 mL). DIPEA (2 mL, 12 mmol) and HATU (850 mg, 2.23 mmol) were added, and a yellow turbid solution was stirred at room temperature for 18 h. HATU (700 mg, 1.84 mmol) was added, and the reaction mixture was stirred under the same conditions for an additional 3 h. The reaction was quenched with sat. NaHCO<sub>3</sub> (15 mL) and extracted with DCM (3 × 10 mL). Combined organics were washed with water (3 × 10 mL), and volatiles were removed under reduced pressure. The residue was taken up in THF (5 mL). Water (5 mL) and LiOH (515 mg) were added, and the reaction mixture was stirred at room temperature for 3 h. THF was evaporated under vacuum. The remaining suspension turned to a thick gel. It was diluted with water till it turned to a solution, and the pH was lowered from 12 to 3.5. The white precipitate thus formed was filtered and washed with water. The product thus obtained was used in the next step without further purification. Yield 53% (380 mg). ES<sup>+</sup><i>m</i>/<i>z</i> 318.2/320.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm: 12.75 (bs, 1H); 8.48 (d, <i>J</i> = 8.4 Hz, 1H); 7.38 (dd, <i>J</i> = 7.7 Hz, <i>J</i> = 1.8 Hz, 1H); 7.16–7.31 (m, 6H); 7.07–7.1 (m, 2H)); 4.53 (ddd, <i>J</i> = 10.3 Hz, <i>J</i> = 8.4 Hz, <i>J</i> = 4.7 Hz, 1H); 3.37 (s, 2H); 3.26 (dd, <i>J</i> = 13.7 Hz, <i>J</i> = 4.7 Hz, 1H); 2.91 (dd, <i>J</i> = 13.7 Hz, <i>J</i> = 10.3 Hz, 1H).</div></div><div id="sec4_1_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> (<i>S</i>)-3-(2-Chlorophenyl)-2-[(2,2-difluoro-2-phenylacetyl)amino]propanoic Acid (<b>65</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Scheme S7</a>)</h5><div class="NLM_p last">A round-bottom flask was charged under argon with 4-fluoro-phenylacetic acid (423 mg, 2.75 mmol), (<i>S</i>)-2′-chlorophenylalanine methyl ester hydrochloride (625 mg, 2.5 mmol), and DMF (5 mL). DIPEA (1.3 mL, 7.5 mmol) and HATU (450 mg, 3.25 mmol) were added, and the reaction mixture was stirred at room temperature for 18 h. The reaction was quenched with sat. aq. NaHCO<sub>3</sub> (15 mL) and extracted with DCM (3 × 10 mL). Combined organics were washed with water (3 × 10 mL), and volatiles were removed under reduced pressure. The residue was taken up in THF (5 mL). Water (5 mL) and LiOH (515 mg) were added, and the reaction mixture was stirred at room temperature for 3 h. THF was evaporated under vacuum, and residues were diluted with water (80 mL) under sonication. The pH was lowered from 12 to 3.5. The white precipitate thus formed was filtered washed with water and dried by suction. Yield 37% (330 mg). ES+ <i>m</i>/<i>z</i> 354.2/356.2 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm: 13.1 (bs, 1H); 9.3 (d, <i>J</i> = 8.9 Hz, 1H); 7.06–7.53 (m, 9H); 4.61 (ddd, <i>J</i> = 11.1 Hz, <i>J</i> = 8.9 Hz, <i>J</i> = 4 Hz, 1H); 3.37 (dd, <i>J</i> = 14 Hz, <i>J</i> = 4 Hz, 1H); 3.06 (dd, <i>J</i> = 14 Hz, <i>J</i> = 11.1 Hz, 1H).</div></div><div id="sec4_1_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> (2<i>S</i>)-3-(2-Chlorophenyl)-2-[[2-(4-fluorophenyl)acetyl]amino]propanoic Acid (<b>66</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Scheme S7</a>)</h5><div class="NLM_p last">A round-bottom flask was charged under argon with 4-fluoro-phenylacetic acid (423 mg, 2.75 mmol), (S)-2′-chlorophenylalanine methyl ester hydrochloride (625 mg, 2.5 mmol), and DMF (5 mL). DIPEA (1.3 mL, 7.5 mmol) and HATU (450 mg, 3.25 mmol) were added, and the reaction mixture was stirred at r.t. for 18 h. The reaction was quenched with sat. NaHCO<sub>3</sub> (15 mL) and extracted with DCM (3 × 10 mL). Combined organics were washed with water (3 × 10 mL), and volatiles were removed under reduced pressure. The residue was taken up in THF (5 mL). Water (5 mL) and LiOH (515 mg) were added, and the reaction mixture was stirred at room temperature for 3 h. THF was evaporated under vacuum. The sample was dissolved in water (80 mL) by sonication, and the pH was lowered from 12 to 3.5. The white precipitate thus formed was filtered and washed with water. Yield 40% (335 mg). ES<sup>+</sup><i>m</i>/<i>z</i> 336.1/338.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm: 12.75 (bs, 1H); 8.48 (d, <i>J</i> = 8.4 Hz, 1H); 7.38 (dd, <i>J</i> = 7.7 Hz, <i>J</i> = 1.8 Hz, 1H); 7.16–7.31 (m, 3H); 7–7.14 (m, 3H); 4.51 (ddd, <i>J</i> = 10.3 Hz, <i>J</i> = 8.4 Hz, <i>J</i> = 4.7 Hz, 1H); 3.35 (s, 2H); 3.25 (dd, <i>J</i> = 13.7 Hz, <i>J</i> = 4.7 Hz, 1H); 2.9 (dd, <i>J</i> = 13.7 Hz, <i>J</i> = 10.3 Hz, 1H).</div></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> General Procedure 5 for the Synthesis of Compounds <b>1</b>–<b>21</b>, <b>23</b>, <b>25</b>–<b>27</b>, <b>34</b>, <b>40</b>, <b>45</b>, and <b>46</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h4><div class="NLM_p">To a solution of corresponding intermediate <b>E</b> (1 eq) in dry DMF, HATU (1.5 eq), DIPEA (1.5 eq), and dipeptide (<b>64–66)</b> (1 eq) were added and stirred at room temperature for 1 h. After the reaction was completed, DMF was evaporated and then extracted with water and DCM. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The crude product was purified by prep HPLC purification (extended A, high pH) or by column chromatography using DCM:MeOH:NH<sub>4</sub>OH = 10:1:0.1 as an eluent to get the title compound.</div><div id="sec4_1_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(1<i>R</i>,4<i>R</i>,5<i>S</i>,6<i>S</i>,7<i>R</i>,8<i>R</i>,10<i>R</i>,15<i>R</i>,18<i>R</i>)-7-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-8-hydroxy-5-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-4,6,8,10,12-pentamethyl-3,13,17-trioxo-2,12,16-triazabicyclo[16.3.0]henicosan-15-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>1</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (36 mg, 0.11 mmol) and <b>E-2</b> (100 mg, 0.11 mmol), the title product was obtained by prep HPLC purification (15 mg, yield 12%, purity 98.41%). ES<sup>+</sup><i>m</i>/<i>z</i> 1176.15 [MH]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.97 (br s, 1H), 8.49–8.46 (m, 2H), 8.18–8.13 (m, 1H), 7.82–7.77 (m, 1H), 7.45–7.41 (m, 2H), 7.4–7.37 (m, 1H), 7.29 (br d, <i>J</i> = 7.3 Hz, 2H), 7.26 (br s, 1H), 7.21–7.17 (m, <i>J</i> = 0.9 Hz, 2H), 7.14 (br d, <i>J</i> = 8.6 Hz, 2H), 7.11–7.07 (m, 2H), 4.79 (br s, 2H), 4.57–4.52 (m, 2H), 4.4–4.33 (m, 2H), 4.31 (br s, 1H), 4.2 (br s, 1H), 4.09 (br s, 1H), 3.97 (br s, 1H), 3.71–3.65 (m, 2H), 3.62–3.59 (m, 1H), 3.22 (s, 6H), 3.18 (s, 3H), 3.16 (d, <i>J</i> = 5.7 Hz, 2H), 3.11–3.07 (m, 3H), 3.01 (br s, 1H), 2.92–2.9 (m, 2H), 2.8–2.76 (m, 1H), 2.76–2.7 (m, 1H), 2.63–2.6 (m, 3H), 2.56 (br d, <i>J</i> = 2.2 Hz, 1H), 2.4–2.39 (m, 2H), 2.32–2.27 (m, 1H), 2.23 (s, 6H), 2.09–2.04 (m, 1H), 1.8–1.76 (m, 1H), 1.76–1.7 (m, 1H), 1.6 (br s, 1H), 1.53–1.48 (m, 1H), 1.33–1.22 (m, 2H), 1.13–1.08 (m, 10H), 1 (t, <i>J</i> = 6.9 Hz, 3H), 0.88 (d, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_1_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(9<i>R</i>,10<i>S</i>,11<i>S</i>,12<i>R</i>,13<i>R</i>,15<i>R</i>,20<i>R</i>)-12-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-13-hydroxy-10-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-9,11,13,15,17-pentamethyl-8,18,22-trioxo-7,17,21-triazaspiro[4.18]tricosan-20-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>2</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (93 mg, 0.292 mmol) and <b>E-3</b> (240 mg, 0.26 mmol), the title product was obtained (114.4 mg, yield 36%, purity 96.6%) as a white solid foam. Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1203.71 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01–9.99 (m, 1H), 8.5 (d, <i>J</i> = 8.2 Hz, 1H), 8.04 (br t, <i>J</i> = 5.5 Hz, 1H), 7.85 (br d, <i>J</i> = 7.9 Hz, 1H), 7.49–7.45 (m, 2H), 7.4 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.31 (dd, <i>J</i> = 7.3, 1.8 Hz, 1H), 7.27–7.22 (m, 3H), 7.22–7.17 (m, 2H), 7.17–7.12 (m, 4H), 4.85–4.73 (m, 2H), 4.36 (d, <i>J</i> = 7 Hz, 1H), 4.28 (d, <i>J</i> = 8.9 Hz, 1H), 4.24 (s, 1H), 4.23–4.2 (m, 1H), 4.14 (br s, 2H), 4.08–4.04 (m, 2H), 3.60–3.53 (m, 1H), 3.48–3.4 (m, 4H), 3.28–3.16 (m, 5H), 3.08–2.97 (m, 3H), 2.94–2.87 (m, 3H), 2.78–2.74 (m, 4H), 2.68–2.57 (m, 2H), 2.28–2.21 (m, 8H), 2.19–2.12 (m, 1H), 2.12–2.05 (m, 1H), 2.01 (br d, <i>J</i> = 12.8 Hz, 1H), 1.93–1.86 (m, 2H), 1.65–1.59 (m, 1H), 1.59–1.54 (m, 1H), 1.53–1.42 (m, 6H), 1.28–1.23 (m, 2H), 1.21–1.04 (m, 17H), 1 (br d, <i>J</i> = 7 Hz, 2H), 0.84 (br d, <i>J</i> = 6.7 Hz, 1H), 0.81 (d, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_1_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>R</i>,4<i>S</i>,5<i>S</i>,6<i>R</i>,7<i>R</i>,9<i>R</i>,14<i>R</i>)-6-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-7-hydroxy-4-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-3,5,7,9,11-pentamethyl-2,12,16-trioxo-1,11,15-triazabicyclo[15.3.0]icosan-14-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>3</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (33 mg, 0.104 mmol) and <b>E-4</b> (90 mg, 0.104 mmol), the title product was obtained (44 mg, yield 36%, purity 98.8%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1161.64 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.04 (s, 1H), 9.96 (s, 1H), 8.54–8.45 (m, 3H), 7.53 (br d, <i>J</i> = 8.2 Hz, 3H), 7.46 (br d, <i>J</i> = 8.2 Hz, 1H), 7.39 (d, <i>J</i> = 7.9 Hz, 3H), 7.3 (t, <i>J</i> = 6.9 Hz, 3H), 7.27–7.23 (m, 7H), 7.22–7.17 (m, 6H), 7.15 (br d, <i>J</i> = 7.9 Hz, 6H), 7.11 (br d, <i>J</i> = 8.5 Hz, 4H), 4.80–4.76 (m, 2H), 4.41–4.35 (m, 3H), 4.17 (br d, <i>J</i> = 8.5 Hz, 3H), 4.12–4.06 (m, 5H), 4.04 (br d, <i>J</i> = 10.4 Hz, 1H), 3.81–3.76 (m, 2H), 3.46 (br d, <i>J</i> = 9.8 Hz, 5H), 3.24–3.15 (m, 12H), 3.08–3.03 (m, 3H), 2.96–2.87 (m, 7H), 2.68 (br s, 5H), 2.55–2.55 (m, 1H), 2.44–2.39 (m, 3H), 2.37–2.29 (m, 12H), 2.27–2.23 (m, 4H), 2.10–2.02 (m, 4H), 1.71 (br s, 2H), 1.64 (br dd, <i>J</i> = 18.8, 8.1 Hz, 3H), 1.49–1.41 (m, 4H), 1.24–1.08 (m, 36H), 1.06–0.97 (m, 17H), 0.93 (br d, <i>J</i> = 6.7 Hz, 4H).</div></div><div id="sec4_1_6_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>4</b>)</h5><div class="NLM_p last">To a solution of <b>64</b> (314.3 mg, 0.989 mmol), <b>E-1a</b> (814.1 mg, 0.942 mmol), and DIPEA (0.361 mL, 2.073 mmol) in dry DMF (9.4 mL), HATU (394 mg, 1.036 mmol) was added. The reaction mixture was stirred at room temperature overnight. EtOAc (50 mL) was added, and the organic layer was washed with sat. NaHCO<sub>3</sub> (3 × 30 mL) and brine (30 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure, yielding the crude product. The crude product was purified by flash column chromatography (SiO<sub>2</sub>, 12 g) with CH<sub>2</sub>Cl<sub>2</sub>/MeOH/NH<sub>4</sub>OH 90:9:1.5 as an eluent to give the title product (737.1 mg, yield 67.2%, purity 99%) as a yellowish solid foam. ES<sup>+</sup>, <i>m</i>/<i>z</i> 1164 [M + H]<sup>+</sup>. The NMR analysis revealed two sets of resonance lines corresponding to two rotamers around the amide (27NH—28C═O) bond. Ratio of rotamers being approximately 43:57. Analysis was performed in CDCl<sub>3</sub> at 45 °C. Atom numbering is given in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S4</a>. <sup>1</sup>H NMR (600 MHz, Chloroform-<i>d</i>) δ/ppm 8.24 (br s, 1H, H27), 8.19 (s, 1H, H27), 7.77 (d, <i>J</i> = 7.7 Hz, 2H, H13), 7.25–7.31 (m, 12H, H42, H44, H34, H38, H25, and H23), 7.02–7.17 (m, 14H, H36, H45, H43, H35, H37, H26, and H22), 6.41 (br s, 2H, H16), 6.11 (d, <i>J</i> = 7.8 Hz, 1H, H30), 6.09 (d, <i>J</i> = 7.7 Hz, 1H, H30), 4.83 (br quin, <i>J</i> = 7.5 Hz, 2H, H29), 4.56 (d, <i>J</i> = 4.5 Hz, 2H, H1″), 4.31 (ddd, <i>J</i> = 9.2, 7.2, 2 Hz, 2H, H15), 4.26 (d, <i>J</i> = 7.3 Hz, 2H, H1′), 4.24 (br s, 2H, H12), 4.17–4.2 (m, 2H, H3), 4.17 (dd, <i>J</i> = 13.6, 11 Hz, 2H, H9′), 4.09 (s, 2H, H6 OH), 3.98 (dq, <i>J</i> = 10, 6.5 Hz, 2H, H5″), 3.52–3.58 (m, 2H, H5′), 3.48–3.51 (m, 4H, H32), 3.32 (dd, <i>J</i> = 10.2, 7.4 Hz, 2H, H2′), 3.24 (s, 6H, H3″OMe), 3.21–3.26 (m, 2H, 39′), 3.19 (br s, 2H, H5), 3.14 (dd, <i>J</i> = 9, 3.5 Hz, 1H, H39″), 3.11 (dd, <i>J</i> = 9.4, 3.9 Hz, 1H, H39″), 3.02 (dd, <i>J</i> = 13.4, 6.1 Hz, 2H, H20′), 2.95 (d, <i>J</i> = 10.8 Hz, 2H, H4″), 2.71–2.77 (m, 10H, H20″, H2, and H9NMe), 2.57 (ddd, <i>J</i> = 12.5, 10.6, 4.2 Hz, 2H, H3′), 2.36 (dd, <i>J</i> = 13.6, 2.9 Hz, 2H, H9″), 2.31–2.33 (m, 2H, H17), 2.29 (s, 16H, H11, H3′Nme and H3′NMe), 2.23 (d, <i>J</i> = 15.4 Hz, 2H, H2″ and H2′), 2.16 (br s, 2H, H8), 1.94 (q, <i>J</i> = 7.2 Hz, 2H, H4), 1.7 (t, <i>J</i> = 13.8 Hz, 2H, H7′), 1.67 (ddd, <i>J</i> = 12.8, 3.8, 1.9 Hz, 2H, 4′ and H4′), 1.46 (dd, <i>J</i> = 15.2, 4.8 Hz, 2H, H2″ and H2″), 1.25 (td, <i>J</i> = 12, 11 Hz, 2H, H4′ and H4″), 1.21 (d, <i>J</i> = 6.4 Hz, 6H, H5′Me), 1.2 (d, <i>J</i> = 6.5 Hz, 6H, H5″Me), 1.17–1.19 (m, 12H, 6Me, H3″Me), 1.16 (d, <i>J</i> = 7 Hz, 6H, H2Me), 1.1 (dd, <i>J</i> = 13.8, 5.3 Hz, 2H, H7″), 1.07 (d, <i>J</i> = 7.1 Hz, 6H, H4Me), 1 (d, <i>J</i> = 6.8 Hz, 6H, H8Me), 0.85–0.93 (m, 12H, H18, and H19). 13C NMR (151 MHz, Chloroform-<i>d</i>) δ/ppm 175.5 (s, C1), 171.9 (s, C31), 171.9 (s, C10), 171 (s, C14), 168.9 (s, C28), 136.1 (s, C24), 135.4 (s, C21), 134.6 (br s, C40), 134.3 (s, C33), 131.7 (s, C45), 129.8 (s, C26, C22, and C42), 129.5 (s, C35 and C37), 129.3 (s, C38 and C34), 128.7 (s, C43), 127.7 (s, C44), 127.3 (s, C36), 120.5 (s, C25 and C23), 106.8 (s, C1′), 94.6 (s, C1″), 91.5 (s, 5), 78.2 (s, C4″), 77.1 (s, C3), 74.2 (s, C6), 73 (s, C3″), 70.9 (s, C2′), 70.4 (s, C5′), 65.8 (s, C5″), 65.2 (s, C3′), 58.8 (s, C15), 55.1 (s, C9), 54.6 (m, C29), 54.5 (s, C29), 49.5 (s, C3″OMe), 48.2 (s, C12), 44.1 (s, C2), 43.8 (s, C32), 43.1 (s, C7), 40.6 (s, C3′Nme and 3′NMe), 39.9 (s, C4), 39.4 (s, C20), 35.5 (s, C9NMe), 35.2 (s, C2″), 35 (s, C39), 34.4 (s, C11), 29.5 (s, C4′), 28.6 (s, C17), 26.8 (s, C8), 23.2 (s, C6Me), 22.1 (s, C8Me), 21.8 (s, C3″Me), 21.4 (s, C5′Me), 20 (s, C18 and C19), 18.1 (s, C5″Me), 18 (s, C18 and C19), 11.9 (s, C4Me), 10.5 (m, C2Me).</div></div><div id="sec4_1_6_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>R</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>5</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (35 mg, 0.11 mmol) and <b>E-5</b> (90 mg, 0.1 mmol), the title product was obtained (27 mg, yield 21%; purity 82.85%) by prep HPLC purification. ES+ <i>m</i>/<i>z</i> 1163.69 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 0.68 (m, 6H), 0.77 (d, 3H, <i>J</i> = 7.1 Hz), 0.99–1.01 (m, 7H), 1.05–1.07 (m, 4H), 1.1–1.17 (m, 12H), 1.25 (s, 2H), 1.45 (dd, 1H, <i>J</i> = 14.74, 4.15 Hz), 1.59 (d, 1H, <i>J</i> = 6.2 Hz), 1.69–1.7 (m, 1H), 1.82–1.83 (m, 1H), 2.22 (s, 6H), 1.95–1.97 (m, 2H), 2.27 (br s, 1H), 2.82 (br s, 1H), 2.71 (s, 3H), 2.91–2.2.95 (m, 1H), 2.96 (br. s. 1H), 3.03–3.08 (m, 1H), 3.08–3.14 (m, 2H), 3.17–3.18 (m, 1H), 3.18–3.19 (m, 1H), 3.21 ( s, 3H), 3.63 (br. s, 3H), 3.77–3.79 (m, 2H), 4–4.05 (m, 2H), 4.14 (s, 1H), 4.31 (br. s. 1H), 4.46–4.47(m, 1H), 4.74–4.76 (m, 1H), 4.99 (br. s. 1H), 7.08 (d, 2H, <i>J</i> = 8.1), 7.13 (d, 2H, <i>J</i> = 7.3 Hz), 7.18–7.17 (m, 2H), 7.21–7.23 (m, 3H), 7.27 (d, 1H, <i>J</i> = 7.25), 7.37 (d, 1H, <i>J</i> = 7.8 Hz), 7.42 (d, 2H, <i>J</i> = 8.25 Hz), 7.95 (d, 1H, <i>J</i> = 8.7 Hz), 8.13 (br. s., 1H), 8.47 (d, 1H, <i>J</i> = 5.8 Hz), 9.65 (s, 1H).</div></div><div id="sec4_1_6_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>S</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>6</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (37 mg, 0.12 mmol) and <b>E-6</b> (100 mg, 0.12 mmol), the title product was obtained (15 mg, yield 11%, purity 95.18%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.5 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.99 (s, 1H), 8.54–8.44 (m, 1H), 7.5 (br d, <i>J</i> = 7 Hz, 1H), 7.46 (br d, <i>J</i> = 8.2 Hz, 2H), 7.38 (br d, <i>J</i> = 7.9 Hz, 1H), 7.29 (br d, <i>J</i> = 6.7 Hz, 1H), 7.26–7.21 (m, 3H), 7.19 (br d, <i>J</i> = 7 Hz, 2H), 7.14 (br d, <i>J</i> = 7.6 Hz, 4H), 4.82 (br d, <i>J</i> = 4.3 Hz, 1H), 4.79–4.73 (m, 1H), 4.46 (br d, <i>J</i> = 7 Hz, 1H), 4.29 (br d, <i>J</i> = 7.6 Hz, 1H), 4.25–4.13 (m, 1H), 4.09–3.98 (m, 4H), 3.97–3.9 (m, 1H), 3.71–3.65 (m, 1H), 3.64–3.56 (m, 1H), 3.5 (br s, 1H), 3.45 (br d, <i>J</i> = 8.5 Hz, 2H), 3.22 (s, 3H), 3.19–3.14 (m, 3H), 3.09–3.02 (m, 2H), 2.96–2.85 (m, 4H), 2.8 (s, 2H), 2.77–2.67 (m, 3H), 2.59–2.54 (m, 1H), 2.23 (s, 6H), 1.6 (br d, <i>J</i> = 11.3 Hz, 1H), 1.5 (br dd, <i>J</i> = 15.1, 4.4 Hz, 1H), 1.19 (br s, 3H), 1.15–1.07 (m, 16H), 1.02–0.89 (m, 9H), 0.8 (br d, <i>J</i> = 6.4 Hz, 1H), 0.76 (br d, <i>J</i> = 6.4 Hz, 3H), 0.67 (br d, <i>J</i> = 6.7 Hz, 2H).</div></div><div id="sec4_1_6_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>R</i>,5<i>S</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>, 15<i>S</i>,16<i>R</i>)-13-[(2<i>S</i>,3<i>R</i>,4<i>S</i>,6<i>R</i>)-4-(dimethylamino)-3-hydroxy-6-methyltetrahydropyran-2-yl]oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>7</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (54 mg, 0.17 mmol) and <b>E-7</b> (145 mg, 0.17 mmol), the title product was obtained (11 mg, yield 6%, purity 93.5%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.72 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 0.55 (d, 3H, <i>J</i> = 6.65), 0.61 (d, 3H, <i>J</i> = 6.7), 0.84 (d, 3H, <i>J</i> = 6.5), 0.99–1.02 (m, 6H), 1.07–1.13 (m, 16H), 1.22 (s, 2H), 1.45 (d, 1H, <i>J</i> = 10), 1.59 (d, 1H, <i>J</i> = 8.5), 1.64–1.68 (m, 1H), 1.79–1.8 (m, 1H), 1.98 (br s, 1H), 2.24 (s, 9H), 2.48–2.5 (br. s. 1H), 2.66–2.69 (m, 2H), 2.78 (s, 5H), 2.87–2.9 (m, 4H), 3.04–3.07 (m, 2H), 3.14–3.18 (m, 2H), 3.21 (s, 3H), 3.45 (dq, 3H, <i>J</i> = 8.1, 14.15), 3.6 (br s, 1H), 3.64 (s, 1H), 3.84 (br s, 1H), 4–4.03 (m, 3H), 4.15 (s, 1H), 4.26 (d, 1H, <i>J</i> = 6.1), 4.41 (d, 1H, <i>J</i> = 7.2), 4.77 (dq, 1H, <i>J</i> = 6.5, 8), 4.9 (d, 1H, <i>J</i> = 3.95), 7.13–7.18 (m, 6H), 7.2 (d, 3H, <i>J</i> = 3.9), 7.37–7.42 (m, 3H), 8.05 (t, 2H, <i>J</i> = 8.4, 8.9), 8.48 (d, 1H, <i>J</i> = 8.35), 9.94 (s, 1H).</div></div><div id="sec4_1_6_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2,8,10,12,14,16-hexamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>8</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (8 mg, 0.024 mmol) and <b>E-8</b> (20 mg, 0.024 mmol), the title product was obtained (17 mg, 55.1%, purity 88.4%). Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:0.9 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1135.74 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.40–10.2 (m, 1H), 9.18–8.95 (m, 1H), 8.06–7.97 (m, 1H), 7.59 (br d, <i>J</i> = 6.1 Hz, 1H), 7.56–7.45 (m, 3H), 7.39 (br d, <i>J</i> = 7.8 Hz, 1H), 7.34 (br d, <i>J</i> = 5.5 Hz, 1H), 7.25–7.14 (m, 8H), 7.11–7.07 (m, 2H), 4.76–4.64 (m, 2H), 4.56 (br d, <i>J</i> = 3.7 Hz, 1H), 4.43–4.4 (m, 1H), 4.23–4.18 (m, 1H), 4.14–4.06 (m, 3H), 4.05–3.98 (m, 2H), 3.94 (br d, <i>J</i> = 7 Hz, 1H), 3.63–3.55 (m, 1H), 3.53–3.45 (m, 4H), 3.2 (d, <i>J</i> = 7.6 Hz, 4H), 3.14–3.04 (m, 3H), 2.97 (br d, <i>J</i> = 11.9 Hz, 1H), 2.92–2.86 (m, 3H), 2.81 (s, 2H), 2.69–2.62 (m, 2H), 2.59–2.54 (m, 1H), 2.49–2.42 (m, 1H), 2.41–2.35 (m, 1H), 2.23 (s, 7H), 2.16–2.06 (m, 1H), 2.02–1.89 (m, 1H), 1.66 (s, 2H), 1.64–1.59 (m, 1H), 1.52–1.34 (m, 2H), 1.30–1.22 (m, 2H), 1.18 (br d, <i>J</i> = 5.8 Hz, 3H), 1.15–1.07 (m, 16H), 1.07–1.03 (m, 4H), 1.03–0.98 (m, 6H), 0.91 (br dd, <i>J</i> = 14.8, 6.6 Hz, 4H).</div></div><div id="sec4_1_6_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>R</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2,8,10,12,14,16-hexamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (9)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (20.1 mg, 0.063 mmol) and <b>E-9</b> (52.8 mg, 0.063 mmol), the title product was obtained (35.4 mg, yield 49.5%, purity 97.2%). Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1135.88 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.97 (s, 1H), 8.48 (br d, <i>J</i> = 7.6 Hz, 1H), 8.21 (br d, <i>J</i> = 7.6 Hz, 1H), 7.97 (br s, 1H), 7.45 (d, <i>J</i> = 8.5 Hz, 2H), 7.39 (d, <i>J</i> = 7.6 Hz, 1H), 7.32–7.27 (m, 1H), 7.24 (d, <i>J</i> = 7.3 Hz, 3H), 7.21–7.18 (m, 2H), 7.15 (d, <i>J</i> = 7.3 Hz, 2H), 7.06 (d, <i>J</i> = 8.2 Hz, 2H), 4.82–4.77 (m, 2H), 4.4 (br d, <i>J</i> = 7.3 Hz, 1H), 4.28 (d, <i>J</i> = 8.5 Hz, 1H), 4.22–4.14 (m, 1H), 4.10–4.03 (m, 2H), 3.99 (br t, <i>J</i> = 7.3 Hz, 2H), 3.87 (s, 1H), 3.59 (br s, 1H), 3.49 (br d, <i>J</i> = 7 Hz, 2H), 3.46 (d, <i>J</i> = 9.5 Hz, 2H), 3.23 (s, 4H), 3.16 (br d, <i>J</i> = 6.1 Hz, 2H), 3.10–3.01 (m, 2H), 2.89 (t, <i>J</i> = 8.9 Hz, 1H), 2.84–2.8 (m, 1H), 2.74 (s, 3H), 2.71–2.64 (m, 1H), 2.24 (s, 6H), 2.05–2.02 (m, 1H), 1.90–1.8 (m, 1H), 1.61 (br s, 1H), 1.48 (br dd, <i>J</i> = 15.0, 4.6 Hz, 1H), 1.36–1.21 (m, 3H), 1.19–1.15 (m, 7H), 1.13 (s, 4H), 1.1 (br d, <i>J</i> = 5.8 Hz, 4H), 1.07 (br d, <i>J</i> = 7.3 Hz, 4H), 1.03 (br d, <i>J</i> = 7 Hz, 3H), 0.87 (t, <i>J</i> = 6.7 Hz, 3H), 0.8 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_6_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>, 15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isobutyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propnnamide (<b>10</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (61 mg, 0.19 mmol) and <b>E-10</b> (168 mg, 0.29 mmol), the title product was obtained (40 mg, yield 12%, purity 95.04%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> = 1177.79 [M + H]<sup>+</sup><sub>.</sub><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (br s, 1H), 10 (br d, <i>J</i> = 2.4 Hz, 1H), 8.52–8.47 (m, 2H), 8.09 (br d, <i>J</i> = 7.9 Hz, 1H), 7.52–7.48 (m, 2H), 7.46 (br d, <i>J</i> = 9.2 Hz, 3H), 7.39 (br d, <i>J</i> = 7.6 Hz, 2H), 7.33–7.28 (m, 3H), 7.27–7.21 (m, 7H), 7.21–7.17 (m, 4H), 7.15 (br d, <i>J</i> = 7.3 Hz, 5H), 7.12–7.07 (m, 4H), 4.82–4.76 (m, 3H), 4.58 (br d, <i>J</i> = 4 Hz, 1H), 4.45 (d, <i>J</i> = 7.3 Hz, 1H), 4.41 (br d, <i>J</i> = 7 Hz, 1H), 4.28 (d, <i>J</i> = 7.6 Hz, 1H), 4.21 (br d, <i>J</i> = 8.2 Hz, 2H), 4.16–3.99 (m, 7H), 3.89 (s, 1H), 3.86 (br d, <i>J</i> = 7.9 Hz, 1H), 3.73–3.65 (m, 1H), 3.64–3.57 (m, 1H), 3.52–3.48 (m, 1H), 3.45 (d, <i>J</i> = 9.8 Hz, 3H), 3.42–3.4 (m, 1H), 3.21 (s, 3H), 3.19 (s, 3H), 3.17–3.07 (m, 6H), 3.03 (br dd, <i>J</i> = 14.2, 9 Hz, 3H), 2.93–2.86 (m, 4H), 2.81 (s, 3H), 2.75–2.69 (m, 2H), 2.68–2.62 (m, 2H), 2.59–2.54 (m, 1H), 2.41–2.3 (m, 3H), 2.26 (br s, 11H), 2.21–2.15 (m, 2H), 2.12–2.06 (m, 2H), 2.01–1.99 (m, 1H), 1.98–1.92 (m, 2H), 1.64–1.6 (m, 2H), 1.52–1.39 (m, 2H), 1.37–1.3 (m, 2H), 1.28–1.19 (m, 5H), 1.18 (br s, 5H), 1.16–1.02 (m, 32H), 1 (br d, <i>J</i> = 7 Hz, 6H), 0.95–0.88 (m, 7H), 0.80–0.7 (m, 14H).</div></div><div id="sec4_1_6_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>, 15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-8,10,12,14,16-pentamethyl-2-[(1<i>S</i>)-1-methylpropyl]-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2[(2-phenylacetyl)amino]propanamide (<b>11</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (160 mg, 0.5 mmol) and <b>E-11</b> (440 mg, 0.5 mmol), the title product was obtained (30 mg, yield 5%, purity 99.3%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.85 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.99 (s, 1H), 9.96 (s, 1H), 8.52–8.44 (m, 3H), 8.1 (br d, <i>J</i> = 8.9 Hz, 2H), 7.90–7.83 (m, 2H), 7.47–7.42 (m, 3H), 7.39 (br d, <i>J</i> = 7.6 Hz, 2H), 7.31–7.28 (m, 2H), 7.27–7.22 (m, 6H), 7.21–7.18 (m, 4H), 7.15 (br d, <i>J</i> = 7.3 Hz, 4H), 7.14–7.07 (m, 4H), 4.81–4.76 (m, 3H), 4.63–4.59 (m, 1H), 4.45 (br d, <i>J</i> = 7 Hz, 2H), 4.37 (br d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 2H), 4.14 (br s, 2H), 4.10–4.03 (m, 4H), 4.01–3.97 (m, 2H), 3.85 (br d, <i>J</i> = 7.9 Hz, 2H), 3.74 (s, 1H), 3.64 (br dd, <i>J</i> = 10.4, 5.8 Hz, 2H), 3.62–3.55 (m, 2H), 3.54–3.49 (m, 2H), 3.46 (br d, <i>J</i> = 7.9 Hz, 4H), 3.43–3.41 (m, 2H), 3.21 (s, 4H), 3.19 (s, 2H), 3.17–3.1 (m, 3H), 3.10–2.99 (m, 6H), 2.93–2.86 (m, 3H), 2.8 (s, 4H), 2.70–2.55 (m, 7H), 2.49–2.43 (m, 3H), 2.41–2.33 (m, 2H), 2.29 (br d, <i>J</i> = 15.3 Hz, 2H), 2.27–2.22 (m, 12H), 2.16–2.05 (m, 2H), 1.96 (br t, <i>J</i> = 6.9 Hz, 2H), 1.83–1.75 (m, 1H), 1.63–1.54 (m, 2H), 1.51–1.45 (m, 3H), 1.24 (br s, 4H), 1.21–1.04 (m, 29H), 0.99 (br d, <i>J</i> = 7 Hz, 5H), 0.96–0.84 (m, 6H), 0.72 (br t, <i>J</i> = 7.2 Hz, 3H), 0.64–0.62 (m, 6H), 0.47 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_6_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> (2<i>S</i>)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-2-<i>tert</i>-Butyl-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-3-(2-chlorophenyl)-2-[(2-phenylacetyl)amino]propanamide (<b>12</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (31.3 mg, 0.099 mmol) and <b>E-12</b> (86.5 mg, 0.099 mmol), the title product was obtained (60.9 mg, yield 52.2%, purity 92.6%). Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.83 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.97 (s, 1H), 9.94 (d, <i>J</i> = 2.7 Hz, 1H), 8.50–8.46 (m, 3H), 8.1 (br d, <i>J</i> = 8.9 Hz, 2H), 7.78 (br d, <i>J</i> = 9.2 Hz, 2H), 7.47–7.41 (m, 4H), 7.39 (d, <i>J</i> = 7.6 Hz, 2H), 7.32–7.11 (m, 18H), 6.95 (br d, <i>J</i> = 8.5 Hz, 2H), 4.86 (br d, <i>J</i> = 4.6 Hz, 2H), 4.82–4.75 (m, 3H), 4.7 (br d, <i>J</i> = 4.6 Hz, 2H), 4.47 (d, <i>J</i> = 7 Hz, 2H), 4.35 (br d, <i>J</i> = 7.3 Hz, 2H), 4.32 (d, <i>J</i> = 7.9 Hz, 2H), 4.23 (br d, <i>J</i> = 8.5 Hz, 2H), 4.15 (br d, <i>J</i> = 8.5 Hz, 2H), 4.11 (br s, 2H), 4.09–4.02 (m, 2H), 3.9 (br d, <i>J</i> = 7.6 Hz, 2H), 3.85 (s, 1H), 3.70–3.61 (m, 5H), 3.59–3.38 (m, 6H), 3.23–3.15 (m, 6H), 3.13–3.01 (m, 6H), 2.99–2.86 (m, 5H), 2.84 (s, 3H), 2.76 (s, 6H), 2.71–2.53 (m, 6H), 2.40–2.35 (m, 1H), 2.31–2.2 (m, 13H), 2.18–2.03 (m, 4H), 1.62 (br d, <i>J</i> = 11 Hz, 2H), 1.54–1.38 (m, 2H), 1.25–1.07 (m, 32H), 1.01 (br d, <i>J</i> = 7.3 Hz, 6H), 0.98–0.88 (m, 6H), 0.82 (s, 4H), 0.62 (s, 12H).</div></div><div id="sec4_1_6_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>, 15<i>S</i>,16<i>R</i>)-2-cyclopropyl-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>13</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (28.6 mg, 0.09 mmol) and <b>E-13</b> (77.7 mg, 0.09 mmol), the title product was obtained as a beige solid (29.8 mg, yield 28.5%, purity 94.8%). Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1161.89 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.02 (s, 1H), 9.98 (s, 1H), 8.48 (s, 2H), 7.96 (br d, <i>J</i> = 8.5 Hz, 1H), 7.9 (br d, <i>J</i> = 9.2 Hz, 1H), 7.64 (br d, <i>J</i> = 6.4 Hz, 1H), 7.62–7.55 (m, 1H), 7.48–7.43 (m, 3H), 7.39 (d, <i>J</i> = 7.7 Hz, 2H), 7.32–7.29 (m, 2H), 7.27–7.06 (m, 19H), 4.83–4.74 (m, 3H), 4.69–4.64 (m, 1H), 4.45 (br d, <i>J</i> = 7 Hz, 1H), 4.42 (br d, <i>J</i> = 7.3 Hz, 1H), 4.29 (d, <i>J</i> = 7.9 Hz, 1H), 4.23 (br d, <i>J</i> = 8.2 Hz, 1H), 4.09–3.96 (m, 4H), 3.88 (br d, <i>J</i> = 8.2 Hz, 1H), 3.86 (s, 1H), 3.72–3.59 (m, 2H), 3.57–3.4 (m, 10H), 3.25–3.11 (m, 11H), 3.10–2.97 (m, 5H), 2.96–2.85 (m, 4H), 2.81 (s, 4H), 2.80–2.63 (m, 6H), 2.61–2.53 (m, 2H), 2.39 (br s, 12H), 2.30–2.2 (m, 4H), 2.16–2.03 (m, 3H), 2.01–1.9 (m, 3H), 1.80–1.77 (m, 1H), 1.77–1.64 (m, 2H), 1.5 (br dd, <i>J</i> = 14.6, 4.6 Hz, 2H), 1.46–1.36 (m, 1H), 1.33–1.22 (m, 4H), 1.21–1.07 (m, 33H), 1.06–0.96 (m, 9H), 0.95–0.82 (m, 10H), 0.34–0.22 (m, 5H), 0.22–0.14 (m, 3H), 0.13–0.06 (m, 3H).</div></div><div id="sec4_1_6_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>R</i>,7<i>S</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-3-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-7-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>14</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (28 mg, 0.089 mmol) and <b>E-14</b> (70 mg, 0.081 mmol), the title product was obtained (38 mg, yield 36%, purity 91.9%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.59 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.93 (br d, <i>J</i> = 4.3 Hz, 1H), 8.48 (t, <i>J</i> = 8.7 Hz, 1H), 7.93 (br d, <i>J</i> = 5.8 Hz, 1H), 7.45 (dd, <i>J</i> = 8.5, 2.1 Hz, 2H), 7.39 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.34–7.28 (m, 1H), 7.26–7.21 (m, 3H), 7.20–7.16 (m, 4H), 7.15–7.11 (m, 3H), 4.82–4.74 (m, 1H), 4.60–4.55 (m, 1H), 4.52 (br t, <i>J</i> = 5.5 Hz, 1H), 4.36 (d, <i>J</i> = 7.3 Hz, 1H), 4.17–4.11 (m, 1H), 4.10–4.05 (m, 2H), 4.04 (br dd, <i>J</i> = 7.9, 2.7 Hz, 1H), 3.98 (br s, 1H), 3.63 (br s, 1H), 3.49–3.4 (m, 3H), 3.21 (s, 3H), 3.20–3.17 (m, 2H), 3.16 (s, 3H), 3.10–2.99 (m, 3H), 2.93–2.86 (m, 1H), 2.69 (br s, 1H), 2.48–2.43 (m, 3H), 2.44–2.38 (m, 1H), 2.25–2.22 (m, 2H), 2.2 (s, 6H), 2.15 (br s, 2H), 2.07–2.02 (m, 1H), 1.97 (br d, <i>J</i> = 11.3 Hz, 1H), 1.61–1.55 (m, 1H), 1.41 (br dd, <i>J</i> = 14.8, 4.7 Hz, 1H), 1.18–1.14 (m, 6H), 1.12 (s, 2H), 1.08 (br d, <i>J</i> = 6.1 Hz, 4H), 1.04–1 (m, 4H), 0.98 (br d, <i>J</i> = 7 Hz, 8H), 0.96 (br s, 3H), 0.92 (br d, <i>J</i> = 7 Hz, 3H), 0.84 (br t, <i>J</i> = 7.5 Hz, 1H).</div></div><div id="sec4_1_6_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>R</i>,7<i>R</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-3-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-7-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>15</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (28 mg, 0.089 mmol) and <b>E-15</b> (70 mg, 0.081 mmol), the title product was obtained (10 mg, yield 10%, purity 97.9%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.54 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.02 (s, 1H), 9.96 (s, 1H), 8.52–8.43 (m, 3H), 8.09 (br d, <i>J</i> = 8.3 Hz, 2H), 7.52–7.47 (m, 4H), 7.44 (br d, <i>J</i> = 8.3 Hz, 2H), 7.40–7.35 (m, 2H), 7.3 (dd, <i>J</i> = 7.5, 1.8 Hz, 2H), 7.28 (dd, <i>J</i> = 7.4, 1.7 Hz, 2H), 7.24–7.2 (m, 5H), 7.20–7.15 (m, 3H), 7.14–7.1 (m, 3H), 7.04 (d, <i>J</i> = 8.4 Hz, 4H), 4.88 (br d, <i>J</i> = 3.9 Hz, 2H), 4.80–4.73 (m, 2H), 4.58 (br s, 2H), 4.46 (br d, <i>J</i> = 7 Hz, 2H), 4.35 (br d, <i>J</i> = 7.3 Hz, 2H), 4.32–4.27 (m, 2H), 4.22 (br d, <i>J</i> = 7.3 Hz, 2H), 4.17 (br s, 2H), 4.1 (br d, <i>J</i> = 5 Hz, 2H), 4.05 (d, <i>J</i> = 8.4 Hz, 2H), 4.03–3.96 (m, 2H), 3.76–3.59 (m, 6H), 3.49–3.4 (m, 4H), 3.38–3.36 (m, 2H), 3.23 (br s, 2H), 3.18–3.12 (m, 8H), 3.02 (dd, <i>J</i> = 14.1, 9 Hz, 3H), 2.96 (s, 3H), 2.92–2.87 (m, 1H), 2.84 (br t, <i>J</i> = 9 Hz, 2H), 2.74 (s, 3H), 2.70–2.58 (m, 4H), 2.58–2.52 (m, 4H), 2.41–2.28 (m, 12H), 2.22 (br d, <i>J</i> = 15.2 Hz, 2H), 2.12–2.04 (m, 6H), 2–1.9 (m, 3H), 1.83 (br s, 1H), 1.76 (br s, 2H), 1.68 (br s, 2H), 1.43 (br dd, <i>J</i> = 14.8, 4.9 Hz, 2H), 1.37 (br dd, <i>J</i> = 14.8, 4.3 Hz, 2H), 1.25–1.15 (m, 3H), 1.14–1.03 (m, 25H), 1 (br d, <i>J</i> = 7.2 Hz, 3H), 0.93 (br d, <i>J</i> = 6.8 Hz, 4H), 0.89 (br d, <i>J</i> = 6.6 Hz, 3H), 0.84 (d, <i>J</i> = 6.6 Hz, 3H), 0.8 (br d, <i>J</i> = 6.8 Hz, 3H), 0.76 (dd, <i>J</i> = 6.3, 3.9 Hz, 6H).</div></div><div id="sec4_1_6_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>S</i>,7<i>S</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-3-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-7-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>16</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (40 mg, 0.127 mmol) and <b>E-16</b> (100 mg, 0.116 mmol), the title product was obtained (55 mg, yield 41%, purity 98.5%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.46 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.02–9.98 (m, 1H), 8.53–8.49 (m, 1H), 8.40–8.36 (m, 1H), 7.97–7.93 (m, 1H), 7.83–7.79 (m, 1H), 7.49–7.45 (m, 2H), 7.38 (d, <i>J</i> = 7.9 Hz, 1H), 7.32–7.29 (m, 1H), 7.24–7.11 (m, 8H), 7 (br d, <i>J</i> = 6.8 Hz, 1H), 4.80–4.74 (m, 1H), 4.48–4.43 (m, 2H), 4.27 (br t, <i>J</i> = 8.4 Hz, 1H), 4.17–4.12 (m, 1H), 4.06–4 (m, 2H), 3.9 (s, 1H), 3.81 (br s, 1H), 3.70–3.64 (m, 1H), 3.62–3.57 (m, 1H), 3.47–3.42 (m, 2H), 3.37 (br d, <i>J</i> = 6.6 Hz, 1H), 3.22 (s, 2H), 3.21–3.13 (m, 3H), 3.08–3.01 (m, 3H), 2.96–2.91 (m, 1H), 2.88 (dd, <i>J</i> = 9.1, 7.8 Hz, 1H), 2.79 (d, <i>J</i> = 1.7 Hz, 2H), 2.59–2.54 (m, 1H), 2.43 (br d, <i>J</i> = 8.3 Hz, 1H), 2.39–2.36 (m, 1H), 2.26 (br d, <i>J</i> = 15 Hz, 1H), 2.22 (s, 6H), 2.14 (br d, <i>J</i> = 14.7 Hz, 2H), 2.06 (s, 2H), 1.99–1.93 (m, 1H), 1.63–1.55 (m, 1H), 1.45 (br dd, <i>J</i> = 14.7, 4.8 Hz, 1H), 1.33–1.26 (m, 1H), 1.24–1.21 (m, 1H), 1.18–1.15 (m, 3H), 1.14–1.11 (m, 5H), 1.09 (s, 2H), 1.06 (d, <i>J</i> = 5.9 Hz, 3H), 1.03 (br d, <i>J</i> = 7.3 Hz, 1H), 0.98–0.92 (m, 5H), 0.90–0.85 (m, 3H), 0.81–0.74 (m, 6H).</div></div><div id="sec4_1_6_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>S</i>,7<i>R</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-3-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-7-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>17</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (40 mg, 0.127 mmol) and <b>E-17</b> (100 mg, 0.116 mmol), the title product was obtained (60 mg, yield 44%, purity 97%) by prep HPLC purification. ES+ <i>m</i>/<i>z</i> 1163.72 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.99 (s, 1H), 8.52–8.44 (m, 1H), 8.25–8.17 (m, 1H), 7.89 (br d, <i>J</i> = 9.2 Hz, 1H), 7.45 (d, <i>J</i> = 8.5 Hz, 2H), 7.39 (d, <i>J</i> = 7.9 Hz, 1H), 7.29 (br d, <i>J</i> = 7.3 Hz, 1H), 7.27–7.22 (m, 3H), 7.22–7.17 (m, 2H), 7.15 (br d, <i>J</i> = 7.3 Hz, 2H), 7.06 (d, <i>J</i> = 8.2 Hz, 2H), 4.84–4.71 (m, 2H), 4.44–4.34 (m, 2H), 4.32–4.21 (m, 2H), 4.17–4.09 (m, 1H), 4.08–3.98 (m, 1H), 3.89–3.8 (m, 2H), 3.66–3.55 (m, 1H), 3.52–3.39 (m, 4H), 3.21–3.14 (m, 5H), 3.13–2.99 (m, 3H), 2.91–2.83 (m, 1H), 2.81–2.71 (m, 4H), 2.23 (s, 6H), 2.21–2.16 (m, 1H), 2.11–1.91 (m, 3H), 1.80–1.71 (m, 1H), 1.61 (br d, <i>J</i> = 10.7 Hz, 1H), 1.43 (br dd, <i>J</i> = 15.0, 4.6 Hz, 1H), 1.39–1.32 (m, 1H), 1.32–1.19 (m, 2H), 1.16–1.08 (m, 13H), 1 (br d, <i>J</i> = 7.3 Hz, 3H), 0.88–0.79 (m, 9H), 0.64 (br s, 2H).</div></div><div id="sec4_1_6_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>R</i>,7<i>S</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-7-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-3-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>18</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (38.6 mg, 0.122 mmol) and <b>E-18</b> (100 mg, 0.116 mmol), the title product was obtained as a yellowish solid foam (73.8 mg, yield 54.7%, purity 91.6%). Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.52/1165.44 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.98–9.94 (m, 1H), 8.49 (dd, <i>J</i> = 17.5, 8.4 Hz, 2H), 8.21 (br d, <i>J</i> = 7.9 Hz, 1H), 7.62 (br d, <i>J</i> = 7.9 Hz, 1H), 7.47–7.38 (m, 8H), 7.31 (t, <i>J</i> = 7.9 Hz, 3H), 7.27–7.22 (m, 7H), 7.21–7.18 (m, 4H), 7.18–7.14 (m, 7H), 7.03 (d, <i>J</i> = 8.2 Hz, 2H), 5.06 (br d, <i>J</i> = 3.7 Hz, 1H), 4.95–4.89 (m, 2H), 4.82–4.75 (m, 2H), 4.65 (br s, 1H), 4.49 (br d, <i>J</i> = 7.3 Hz, 1H), 4.35 (d, <i>J</i> = 7.3 Hz, 1H), 4.25–4.16 (m, 2H), 4.11 (d, <i>J</i> = 8.5 Hz, 2H), 4.03 (br d, <i>J</i> = 6.1 Hz, 2H), 3.91–3.85 (m, 2H), 3.82 (br d, <i>J</i> = 6.1 Hz, 1H), 3.74 (br s, 2H), 3.69–3.61 (m, 1H), 3.48–3.44 (m, 4H), 3.24 (s, 3H), 3.18 (s, 5H), 3.11–3 (m, 4H), 2.97–2.89 (m, 3H), 2.87 (s, 3H), 2.85–2.81 (m, 2H), 2.74–2.71 (m, 4H), 2.49–2.41 (m, 4H), 2.24 (d, <i>J</i> = 4.3 Hz, 13H), 2.12 (br d, <i>J</i> = 15.3 Hz, 3H), 2.02 (br d, <i>J</i> = 11 Hz, 2H), 1.93–1.81 (m, 2H), 1.76–1.7 (m, 1H), 1.68–1.58 (m, 4H), 1.55 (br s, 1H), 1.46 (br dd, <i>J</i> = 14.5, 4.7 Hz, 1H), 1.35 (br dd, <i>J</i> = 15.0, 4.6 Hz, 1H), 1.16–1.13 (m, 8H), 1.12 (s, 9H), 1.10–1.07 (m, 10H), 1.03 (br d, <i>J</i> = 7 Hz, 6H), 0.99 (br d, <i>J</i> = 7 Hz, 4H), 0.95 (br dd, <i>J</i> = 6.4, 3.7 Hz, 10H), 0.78 (dd, <i>J</i> = 6.9, 3.2 Hz, 8H), 0.76–0.71 (m, 7H), 0.69 (br d, <i>J</i> = 5.5 Hz, 3H).</div></div><div id="sec4_1_6_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>R</i>,7<i>R</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-7-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-3-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>19</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (22 mg, 0.07 mmol) and <b>E-19</b> (60 mg, 0.07 mmol), the title product (14 mg, yield 17%, purity 90.94%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.45 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.04 (s, 1H), 9.99 (d, <i>J</i> = 9.2 Hz, 1H), 8.63 (br d, <i>J</i> = 9.5 Hz, 2H), 8.51 (br dd, <i>J</i> = 8.1, 3.2 Hz, 2H), 8.08 (br s, 2H), 7.49 (br d, <i>J</i> = 7.6 Hz, 2H), 7.44 (br dd, <i>J</i> = 8.4, 4.4 Hz, 2H), 7.39 (d, <i>J</i> = 7.9 Hz, 2H), 7.33–7.28 (m, 3H), 7.26–7.06 (m, 16H), 4.92–4.83 (m, 3H), 4.82–4.75 (m, 2H), 4.7 (q, <i>J</i> = 7 Hz, 2H), 4.61 (br d, <i>J</i> = 4.6 Hz, 2H), 4.43 (br d, <i>J</i> = 7.3 Hz, 2H), 4.39–4.32 (m, 2H), 4.16–4.1 (m, 2H), 4.04 (br d, <i>J</i> = 8.2 Hz, 4H), 3.88–3.81 (m, 1H), 3.72–3.52 (m, 3H), 3.51–3.37 (m, 5H), 3.29–3.25 (m, 1H), 3.2 (d, <i>J</i> = 2.1 Hz, 4H), 3.18–3 (m, 8H), 2.97–2.8 (m, 10H), 2.80–2.72 (m, 3H), 2.73–2.61 (m, 2H), 2.49–2.35 (m, 4H), 2.29–2.17 (m, 18H), 2.15 (br s, 1H), 2.1 (br s, 1H), 2.05–1.93 (m, 1H), 1.81 (br s, 1H), 1.62 (br d, <i>J</i> = 10.7 Hz, 2H), 1.52 (br dd, <i>J</i> = 14.8, 4.7 Hz, 2H), 1.4 (br d, <i>J</i> = 10.7 Hz, 2H), 1.19–1.05 (m, 27H), 1 (br dd, <i>J</i> = 11.7, 6.9 Hz, 9H), 0.92 (br d, <i>J</i> = 6.4 Hz, 3H), 0.87 (d, <i>J</i> = 6.4 Hz, 4H), 0.8 (br d, <i>J</i> = 6.7 Hz, 3H), 0.71 (br d, <i>J</i> = 6.7 Hz, 3H), 0.64 (br d, <i>J</i> = 7 Hz, 6H).</div></div><div id="sec4_1_6_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>R</i>,6<i>S</i>,10<i>R</i>,11<i>S</i>,12<i>S</i>,13<i>R</i>,14<i>R</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-14-hydroxy-11-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-6-isopropyl-1,10,12,14,16-pentamethyl-2,5,9-trioxo-1,4,8-triazacycloheptadec-3-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>20</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (22 mg, 0.07 mmol) and <b>E-20</b> (60 mg, 0.07 mmol), the title product (15 mg, yield 18%, purity 98.15%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1163.46 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (s, 1H), 8.6–8.54 (m, 1H), 7.74 (br d, <i>J</i> = 7.3 Hz, 1H), 7.47 (br d, <i>J</i> = 8.4 Hz, 1H), 7.45–7.43 (m, 1H), 7.39 (d, <i>J</i> = 7.7 Hz, 1H), 7.29 (br d, <i>J</i> = 7.7 Hz, 1H), 7.26–7.22 (m, 6H), 7.22–7.17 (m, 3H), 7.15 (br d, <i>J</i> = 7.2 Hz, 3H), 5.19 (br d, <i>J</i> = 4.6 Hz, 1H), 4.77 (br d, <i>J</i> = 8.4 Hz, 1H), 4.72–4.68 (m, 1H), 4.38–4.34 (m, 1H), 4.22 (br d, <i>J</i> = 6.6 Hz, 1H), 4.09–4.01 (m, 3H), 3.90–3.83 (m, 1H), 3.76 (br d, <i>J</i> = 9.7 Hz, 1H), 3.60–3.54 (m, 1H), 3.5–3.49 (m, 1H), 3.48–3.42 (m, 3H), 3.27–3.22 (m, 2H), 3.22–3.13 (m, 6H), 3.08–3.03 (m, 2H), 3.01 (d, <i>J</i> = 3.5 Hz, 3H), 2.91 (br d, <i>J</i> = 14.3 Hz, 2H), 2.87–2.8 (m, 1H), 2.48–2.39 (m, 2H), 2.3–2.26 (m, 1H), 2.24–2.19 (m, 6H), 2.19–2.14 (m, 2H), 1.92 (td, <i>J</i> = 10.7, 6 Hz, 1H), 1.77–1.7 (m, 2H), 1.57 (br d, <i>J</i> = 14.9 Hz, 1H), 1.42–1.36 (m, 1H), 1.36–1.3 (m, 1H), 1.25–1.21 (m, 1H), 1.2–1.14 (m, 4H), 1.12 (br d, <i>J</i> = 6.1 Hz, 4H), 1.1 (s, 4H), 1.08 (br d, <i>J</i> = 6.2 Hz, 4H), 1.06–1.03 (m, 3H), 1.02–0.95 (m, 9H), 0.89–0.83 (m, 3H), 0.58 (br d, <i>J</i> = 5 Hz, 3H).</div></div><div id="sec4_1_6_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>S</i>,6<i>R</i>,9<i>R</i>,10<i>S</i>,11<i>S</i>,12<i>R</i>,13<i>R</i>,15<i>R</i>)-12-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-13-hydroxy-10-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-6-isopropyl-1,9,11,13,15-pentamethyl-2,5,8-trioxo-1,4,7-triazacyclohexadec-3-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>21</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (172 mg, 0.49 mmol, 1.1 eq) and <b>E-21</b> (382 mg, 0.45 mmol) title product (40 mg, yield 8%, purity 98%) was obtained by prep HPLC purification. ES+ <i>m</i>/<i>z</i> 1149.48 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.94 (s, 1H), 8.66–8.56 (m, 1H), 8.48 (br d, <i>J</i> = 8.2 Hz, 1H), 8.08 (br t, <i>J</i> = 5.3 Hz, 1H), 7.43–7.35 (m, 3H), 7.28 (d, <i>J</i> = 7.3 Hz, 1H), 7.25–7.12 (m, 8H), 7.09 (br d, <i>J</i> = 7.9 Hz, 2H), 4.91 (br d, <i>J</i> = 6.1 Hz, 1H), 4.80–4.73 (m, 1H), 4.66 (br d, <i>J</i> = 4.3 Hz, 1H), 4.31–4.24 (m, 2H), 4.17 (d, <i>J</i> = 2.4 Hz, 1H), 4.09–3.97 (m, 2H), 3.83 (br s, 1H), 3.74 (br t, <i>J</i> = 6.3 Hz, 1H), 3.55–3.48 (m, 1H), 3.47–3.37 (m, 4H), 3.2 (s, 3H), 3.16 (br dd, <i>J</i> = 14.2, 5.6 Hz, 1H), 3.08–2.98 (m, 3H), 2.93–2.79 (m, 3H), 2.71 (s, 3H), 2.64–2.53 (m, 1H), 2.48–2.42 (m, 1H), 2.25 (br d, <i>J</i> = 14.6 Hz, 1H), 2.22 (s, 6H), 1.95 (br s, 1H), 1.83–1.72 (m, 2H), 1.59 (br d, <i>J</i> = 8.5 Hz, 1H), 1.46 (br dd, <i>J</i> = 14.8, 4.4 Hz, 1H), 1.40–1.31 (m, 1H), 1.30–1.21 (m, 2H), 1.19–1.11 (m, 9H), 1.09–1.03 (m, 9H), 0.78 (br d, <i>J</i> = 6.4 Hz, 3H), 0.69–0.64 (m, 5H).</div></div><div id="sec4_1_6_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (2<i>R</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(4<i>R</i>,8<i>R</i>,11<i>R</i>,12<i>S</i>,13<i>S</i>,14<i>R</i>,15<i>R</i>,17<i>R</i>)-14-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-hydroxy-12-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-8-isopropyl-1,11,13,15,17-pentamethyl-2,6,10-trioxo-1,5,9-triazacycloctadec-4-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>23</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (46 mg, 0.13 mmol) and <b>E-23</b> (110 mg, 0.13 mmol), the title product (33 mg, yield 21%, purity 85.4%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.83 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz) δ/ppm 10 (s, 1H), 9.98 (s, 1H), 8.53–8.48 (m, 2H), 7.83 (br d, <i>J</i> = 8.9 Hz, 1H), 7.52–7.46 (m, 4H), 7.44 (br d, <i>J</i> = 7.6 Hz, 2H), 7.4 (br d, <i>J</i> = 7.9 Hz, 2H), 7.33–7.29 (m, 2H), 7.26–7.13 (m, 20H), 7.06 (br d, <i>J</i> = 8.2 Hz, 2H), 4.82–4.74 (m, 2H), 4.71 (br d, <i>J</i> = 4.6 Hz, 1H), 4.47 (br d, <i>J</i> = 3.4 Hz, 1H), 4.41 (br d, <i>J</i> = 7.3 Hz, 1H), 4.34 (br d, <i>J</i> = 7.3 Hz, 1H), 4.25–4.15 (m, 4H), 4.14–3.92 (m, 8H), 3.88 (br d, <i>J</i> = 7.9 Hz, 1H), 3.70–3.63 (m, 3H), 3.50–3.41 (m, 7H), 3.30–3.13 (m, 13H), 3.09–3 (m, 4H), 2.90–2.85 (m, 6H), 2.82–2.78 (m, 5H), 2.76–2.68 (m, 2H), 2.55 (br d, <i>J</i> = 5.8 Hz, 2H), 2.48–2.36 (m, 6H), 2.23 (br d, <i>J</i> = 8.9 Hz, 14H), 2.11–2.01 (m, 4H), 1.65–1.59 (m, 3H), 1.44 (br d, <i>J</i> = 10.4 Hz, 3H), 1.29–1.25 (m, 1H), 1.16–1.11 (m, 13H), 1.10–1.06 (m, 16H), 0.99–0.94 (m, 13H), 0.89 (br d, <i>J</i> = 6.7 Hz, 4H), 0.86 (br d, <i>J</i> = 6.4 Hz, 3H), 0.8 (br t, <i>J</i> = 6.6 Hz, 7H).</div></div><div id="sec4_1_6_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(4<i>R</i>,7<i>R</i>,11<i>R</i>,12<i>S</i>,13<i>S</i>,14<i>R</i>,15<i>R</i>,17<i>R</i>)-14-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-hydroxy-12-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-7-isopropyl-1,11,13,15,17-pentamethyl-2,6,10-trioxo-1,5,9-triazacyclooctadec-4-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>25</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (39.8 mg, 0.125 mmol) and <b>E-25</b> (100 mg, 0.114 mmol), the title product was obtained as a white solid foam (93.7 mg, yield 69.8%, purity 93.1%). Purification was performed by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.82 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.96 (s, 1H), 9.95 (s, 1H), 8.49 (br d, <i>J</i> = 7.3 Hz, 2H), 7.88 (br d, <i>J</i> = 8.9 Hz, 1H), 7.82–7.76 (m, 1H), 7.72 (br d, <i>J</i> = 9.5 Hz, 1H), 7.48–7.38 (m, 7H), 7.30–7.14 (m, 20H), 7.1 (d, <i>J</i> = 8.5 Hz, 2H), 4.89–4.85 (m, 1H), 4.80–4.75 (m, 2H), 4.70–4.68 (m, 2H), 4.42–4.36 (m, 3H), 4.35–4.23 (m, 4H), 4.12 (d, <i>J</i> = 7.9 Hz, 1H), 4.07–3.95 (m, 6H), 3.83–3.77 (m, 1H), 3.58 (br d, <i>J</i> = 13.1 Hz, 3H), 3.50–3.41 (m, 6H), 3.26–3.14 (m, 10H), 3.10–3.01 (m, 6H), 3 (s, 3H), 2.94–2.84 (m, 3H), 2.79 (br d, <i>J</i> = 4 Hz, 1H), 2.76 (s, 4H), 2.69–2.63 (m, 2H), 2.61–2.53 (m, 3H), 2.50–2.46 (m, 1H), 2.28 (br s, 1H), 2.23 (br d, <i>J</i> = 2.7 Hz, 12H), 2.13–2.05 (m, 3H), 1.90–1.84 (m, 1H), 1.81 (br s, 1H), 1.69–1.58 (m, 4H), 1.56–1.51 (m, 1H), 1.42 (td, <i>J</i> = 15.0, 4.4 Hz, 2H), 1.32 (br d, <i>J</i> = 14 Hz, 2H), 1.26–1.19 (m, 2H), 1.17–1.09 (m, 25H), 1.08–1.02 (m, 11H), 1 (br d, <i>J</i> = 6.7 Hz, 3H), 0.97–0.92 (m, 9H), 0.81 (br d, <i>J</i> = 6.1 Hz, 3H), 0.69 (br d, <i>J</i> = 6.7 Hz, 3H), 0.57 (br d, <i>J</i> = 6.4 Hz, 3H), 0.51 (br d, <i>J</i> = 5.8 Hz, 2H).</div></div><div id="sec4_1_6_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(4<i>R</i>,8<i>R</i>,11<i>R</i>,12<i>S</i>,13<i>S</i>,14<i>R</i>,15<i>R</i>,17<i>R</i>)-14-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-hydroxy-12-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-8-isobutyl-1,11,13,15,17-pentamethyl-2,6,10-trioxo-1,5,9-triazacyclooctadec-4-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>26</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (25 mg, 0.08 mmol) and <b>E-26</b> (70 mg, 0.08 mmol), the title product (10 mg, yield 11%, purity 95.86%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1191.83 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.03 (s, 1H), 8.51 (d, <i>J</i> = 8.6 Hz, 1H), 8.14 (br d, <i>J</i> = 8.3 Hz, 1H), 8.01–7.97 (m, 1H), 7.53–7.46 (m, 2H), 7.4 (dd, <i>J</i> = 8.0, 0.8 Hz, 1H), 7.32 (br d, <i>J</i> = 7.5 Hz, 1H), 7.26–7.22 (m, 3H), 7.22–7.17 (m, 2H), 7.14 (br d, <i>J</i> = 7.2 Hz, 2H), 7.06 (br d, <i>J</i> = 8.3 Hz, 2H), 4.85–4.78 (m, 2H), 4.4 (br d, <i>J</i> = 7 Hz, 1H), 4.28 (br d, <i>J</i> = 8.3 Hz, 1H), 4.21–4.15 (m, 1H), 4.11–4.01 (m, 1H), 3.98–3.93 (m, 1H), 3.91–3.86 (m, 1H), 3.82 (s, 1H), 3.64–3.57 (m, 1H), 3.48 (br s, 2H), 3.44–3.41 (m, 1H), 3.29–3.25 (m, 1H), 3.22 (s, 3H), 3.21–3.19 (m, 1H), 3.18–3.12 (m, 3H), 3.09–3.01 (m, 2H), 2.91–2.87 (m, 1H), 2.85–2.8 (m, 1H), 2.73 (s, 3H), 2.25–2.22 (m, 7H), 2.06–2.01 (m, 1H), 2–1.95 (m, 1H), 1.85–1.8 (m, 1H), 1.64–1.59 (m, 1H), 1.50–1.44 (m, 1H), 1.32–1.24 (m, 3H), 1.19–1.05 (m, 19H), 1.01 (br d, <i>J</i> = 6.8 Hz, 5H), 0.94–0.87 (m, 1H), 0.83–0.79 (m, 4H), 0.71 (ddd, <i>J</i> = 12.7, 6.6, 2.9 Hz, 6H).</div></div><div id="sec4_1_6_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(4<i>R</i>,11<i>R</i>,12<i>S</i>,13<i>S</i>,14<i>R</i>,15<i>R</i>,17<i>R</i>)-14-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-hydroxy-12-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-1,11,13,15,17-pentamethyl-2,6,10-trioxo-1,5,9-triazacyclooctadec-4-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>27</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>64</b> (71 mg, 0.222 mmol) and <b>E-27</b> (186 mg, 0.222 mmol), the title product (55 mg, yield 20%, purity 92.3%) was obtained by prep HPLC purification. ES<sup>+</sup>, <i>m</i>/<i>z</i> 1135.9 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (br s, 1H), 8.51 (br d, <i>J</i> = 8.2 Hz, 1H), 7.81 (br d, <i>J</i> = 8.9 Hz, 1H), 7.50–7.45 (m, 3H), 7.4 (br d, <i>J</i> = 7.6 Hz, 1H), 7.32 (br d, <i>J</i> = 6.4 Hz, 1H), 7.24 (br d, <i>J</i> = 7.3 Hz, 3H), 7.21 (br s, 2H), 7.15 (br d, <i>J</i> = 7 Hz, 3H), 7.1 (br d, <i>J</i> = 8.2 Hz, 2H), 4.79 (br d, <i>J</i> = 5.2 Hz, 1H), 4.71–4.68 (m, 1H), 4.37 (br d, <i>J</i> = 7 Hz, 1H), 4.28–4.25 (m, 2H), 4.13 (br d, <i>J</i> = 7.9 Hz, 1H), 4.08–4.02 (m, 3H), 3.53–3.49 (m, 2H), 3.46 (br d, <i>J</i> = 8.5 Hz, 2H), 3.40–3.37 (m, 1H), 3.23–3.19 (m, 4H), 3.05 (br dd, <i>J</i> = 13.3, 9 Hz, 3H), 2.91–2.88 (m, 2H), 2.73 (s, 4H), 2.66 (br d, <i>J</i> = 8.2 Hz, 1H), 2.30–2.26 (m, 2H), 2.23 (br s, 8H), 2.18 (br d, <i>J</i> = 11 Hz, 1H), 2.09–2 (m, 3H), 1.66–1.58 (m, 1H), 1.50–1.43 (m, 1H), 1.29 (br s, 2H), 1.18 (s, 3H), 1.16–1.1 (m, 10H), 1.07–1.03 (m, 6H), 0.96 (br dd, <i>J</i> = 16.8, 6.7 Hz, 2H), 0.88 (br d, <i>J</i> = 6.7 Hz, 2H).</div></div><div id="sec4_1_6_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>, 15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(4-fluorophenyl)acetyl]amino]propenamide (<b>34</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>66</b> (40 mg, 0.12 mmol) and <b>E1a</b> (100 mg, 0.12 mmol), the title product (40 mg, yield 28%, purity 89.61%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> = 1181.71 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers).<sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.99 (s, 1H), 9.95 (s, 1H), 8.53–8.47 (m, 3H), 8.11 (br s, 2H), 7.45 (d, <i>J</i> = 8.4 Hz, 2H), 7.42 (d, <i>J</i> = 7.9 Hz, 2H), 7.38 (br d, <i>J</i> = 7.9 Hz, 2H), 7.27 (br d, <i>J</i> = 7.7 Hz, 2H), 7.24–7.21 (m, 3H), 7.19–7.16 (m, 6H), 7.13–7.1 (m, 6H), 7.06 (t, <i>J</i> = 8.6 Hz, 4H), 4.79–4.76 (m, 2H), 4.65–4.61 (m, 2H), 4.45 (d, <i>J</i> = 7.2 Hz, 2H), 4.40–4.35 (m, 2H), 4.29 (d, <i>J</i> = 7.7 Hz, 2H), 4.23 (d, <i>J</i> = 8.3 Hz, 2H), 4.17–4.11 (m, 2H), 4.06–4.01 (m, 4H), 3.99–3.94 (m, 1H), 3.91–3.87 (m, 1H), 3.86 (s, 2H), 3.71 (s, 1H), 3.68–3.64 (m, 2H), 3.51 (br d, <i>J</i> = 7 Hz, 2H), 3.48 (s, 1H), 3.44 (d, <i>J</i> = 10.1 Hz, 5H), 3.22–3.14 (m, 9H), 3.09–3.01 (m, 6H), 3–2.94 (m, 1H), 2.91–2.87 (m, 2H), 2.79 (s, 3H), 2.72–2.67 (m, 2H), 2.62–2.56 (m, 5H), 2.47 (br s, 3H), 2.40–2.36 (m, 2H), 2.26 (br d, <i>J</i> = 15.2 Hz, 2H), 2.23 (s, 13H), 2–1.94 (m, 2H), 1.79–1.73 (m, 1H), 1.63–1.58 (m, 2H), 1.51–1.39 (m, 2H), 1.24 (br s, 2H), 1.20–1.07 (m, 30H), 0.99 (br d, <i>J</i> = 7.2 Hz, 5H), 0.94–0.89 (m, 5H), 0.68 (br d, <i>J</i> = 6.8 Hz, 3H), 0.63 (br d, <i>J</i> = 6.2 Hz, 3H), 0.57 (br d, <i>J</i> = 6.8 Hz, 3H), 0.5 (br d, <i>J</i> = 6.2 Hz, 3H).</div></div><div id="sec4_1_6_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> (2<i>R</i>)-3-(2-Chlorophenyl)-2-[(2,2-difluoro-2-phenylacetyl)amino]-<i>N</i>-[4-[[(2<i>S</i>,5<i>S</i>,10<i>S</i>,12<i>S</i>,13<i>S</i>,14<i>R</i>,15<i>R</i>,16<i>S</i>)-13-[(2<i>R</i>,3<i>S</i>,4<i>R</i>,6<i>S</i>)-4-(dimethylamino)-3-hydroxy-6-methyltetrahydropyran-2-yl]oxy-12-hydroxy-15-[(2<i>S</i>,4<i>S</i>,5<i>R</i>,6<i>R</i>)-5-hydroxy-4-methoxy-4,6-dimethyltetrahydropyran-2-yl]oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propanamide (<b>40</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from <b>65</b> (42 mg, 0.12 mmol) and <b>E-1a</b> (100 mg, 0.12 mmol), the title product (13 mg, yield 9%, purity 99%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1199.74 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 0.49 (d, 3H, <i>J</i> = 6.72), 0.56 (d, 3H, <i>J</i> = 6.66), 0.61 (d, 2H, <i>J</i> = 6.42), 0.66 (d, 2H, <i>J</i> = 6.54), 0.88 (d, 4H, <i>J</i> = 6.48), 0.91 (d, 2H, <i>J</i> = 6.60), 0.97 (d, 6H, <i>J</i> = 7.26), 1.06–1.08 (m, 6H), 1.09–1.1 (m, 5H), 1.11–1.12 (m, 10H), 1.14–1.15 (m, 7H), 1.18 (s, 2H), 1.22 (s, 3H), 1.32–1.48 (m, 3H), 1.59–1.63 (m, 2H), 1.72–1.77 (m, 1H), 1.93–1.99 (m, 3H), 2.06–2.1 (m, 2H), 2.18–2.2 (m, 1H), 2.23 (s, 12H), 2.26 (s, 1H), 2.34–2.38 (m, 2H), 2.54–2.61 (m, 4H), 2.66–2.71 (m, 1H), 2.75 (d, 1H, <i>J</i> = 11.1), 2.78 (s, 4H), 2.84–2.9 (m, 4H), 2.95–3.02 (m, 2H), 3.03–3.12 (m, 3H), 3.15–3.24 (m, 12H), 3.43 (d, 1H, <i>J</i> = 4.98), 3.5 (d, 1H, <i>J</i> = 6.78), 3.63–3.67 (m, 1H), 3.68 (s, 1H), 3.82 (d, 1H, <i>J</i> = 9.24), 3.87 (s, 1H), 3.93–3.98 (m, 1H), 3.98–4.05 (m, 3H), 4.07–4.14 (m, 2H), 4.2 (d, 1H, <i>J</i> = 10.68), 4.26 (d, 2H, <i>J</i> = 6.36), 4.43 (d, 1H, <i>J</i> = 7.32), 4.77–4.81 (m, 3H), 5.74 (s, 1H), 7.09–7.12 (m, 6H), 7.18–7.21 (m, 3H), 7.23–7.27 (m, 3H), 7.36 (d, 2H, <i>J</i> = 7.56), 7.4 (d, 2H, <i>J</i> = 7.92), 7.43–7.46 (m, 9H), 7.49–7.53 (m, 2H), 8.1 (d, 1H, 1H, <i>J</i> = 7.14), 9.21–9.24 (m, 1H), 10 (s, 1H).</div></div><div id="sec4_1_6_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>, 15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenoxyacetyl)amino]propanamide (<b>45</b>)</h5><div class="NLM_p last">Following general procedure 5: starting from (2<i>S</i>)-3-(2-chlorophenyl)-2-[(2-phenoxyacetyl)amino]propanoic acid (43 mg, 0.128 mmol) and <b>E-1a</b> (100 mg, 0.116 mmol), the title product (13 mg, yield 9.5%, purity 92.98%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1179.61 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.03 (s, 1H), 9.99 (s, 1H), 8.40–8.27 (m, 3H), 8.21 (s, 2H), 8.12 (br d, <i>J</i> = 8.9 Hz, 2H), 7.92 (br d, <i>J</i> = 9.5 Hz, 2H), 7.49–7.37 (m, 5H), 7.34–7.19 (m, 9H), 7.17–7.09 (m, 3H), 7.01–6.91 (m, 2H), 6.88 (d, <i>J</i> = 7.9 Hz, 3H), 4.93–4.73 (m, 3H), 4.72–4.57 (m, 1H), 4.5 (s, 3H), 4.49–4.43 (m, 2H), 4.38 (br d, <i>J</i> = 6.7 Hz, 2H), 4.28 (br s, 2H), 4.17–3.96 (m, 6H), 3.93–3.82 (m, 2H), 3.69–3.59 (m, 4H), 3.52 (br d, <i>J</i> = 6.7 Hz, 3H), 3.46 (br d, <i>J</i> = 5.5 Hz, 3H), 3.40–3.31 (m, 8H), 3.25 (br d, <i>J</i> = 5.5 Hz, 2H), 3.23–3.21 (m, 5H), 3.19 (s, 2H), 3.13 (br dd, <i>J</i> = 14.3, 8.5 Hz, 4H), 3.08 (br d, <i>J</i> = 7.9 Hz, 2H), 3.06–2.96 (m, 3H), 2.95–2.85 (m, 4H), 2.82–2.76 (m, 4H), 2.74–2.53 (m, 6H), 2.43–2.33 (m, 2H), 2.31–2.25 (m, 10H), 2.12–2.05 (m, 2H), 2.03–1.9 (m, 2H), 1.84–1.71 (m, 1H), 1.64 (br d, <i>J</i> = 6.7 Hz, 2H), 1.57–1.37 (m, 3H), 1.24 (s, 6H), 1.21–1.07 (m, 25H), 0.99 (br d, <i>J</i> = 7.3 Hz, 6H), 0.96–0.85 (m, 6H), 0.69 (br d, <i>J</i> = 6.7 Hz, 3H), 0.64–0.55 (m, 5H), 0.52 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_6_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> Benzyl-<i>N</i>-[(1<i>S</i>)-1-[(2-chlorophenyl)methyl]-2-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]anilino]-2-oxo-ethyl]carbamate (<b>46</b>)</h5><div class="NLM_p last">Starting from (2<i>S</i>)-2-(benzyloxycarbonylamino)-3-(2-chlorophenyl)propanoic acid (43 mg, 0.127 mmol) and <b>E-1a</b> (100 mg, 0.116 mmol), the title product (74 mg, yield 54%, purity 98.23%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1179.58 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.92 (s, 1H), 9.88 (s, 1H), 8.1 (br d, <i>J</i> = 8.9 Hz, 1H), 7.9 (br d, <i>J</i> = 8.9 Hz, 1H), 7.73–7.65 (m, 2H), 7.46 (br d, <i>J</i> = 8.2 Hz, 1H), 7.41 (br t, <i>J</i> = 8.1 Hz, 5H), 7.37–7.3 (m, 6H), 7.30–7.16 (m, 12H), 7.14–7.07 (m, 4H), 5.04–4.91 (m, 4H), 4.78 (br d, <i>J</i> = 4.6 Hz, 1H), 4.62 (br d, <i>J</i> = 4 Hz, 1H), 4.53–4.46 (m, 2H), 4.44 (br d, <i>J</i> = 7.3 Hz, 1H), 4.36 (br d, <i>J</i> = 7.3 Hz, 1H), 4.28 (d, <i>J</i> = 7.6 Hz, 1H), 4.28–4.24 (m, 1H), 4.22 (d, <i>J</i> = 8.2 Hz, 1H), 4.16–4.08 (m, 3H), 4.07–3.94 (m, 5H), 3.88 (s, 1H), 3.83 (br d, <i>J</i> = 7.6 Hz, 1H), 3.69 (s, 1H), 3.66 (br dq, <i>J</i> = 10.7, 6.1 Hz, 1H), 3.62–3.55 (m, 1H), 3.51 (br d, <i>J</i> = 7 Hz, 1H), 3.45 (br d, <i>J</i> = 5.2 Hz, 1H), 3.24 (br d, <i>J</i> = 8.5 Hz, 2H), 3.2 (s, 4H), 3.18 (s, 2H), 3.17–3.11 (m, 2H), 3.10–3.04 (m, 2H), 3.01 (br dd, <i>J</i> = 14.2, 9 Hz, 3H), 2.96 (br s, 1H), 2.91 (br t, <i>J</i> = 9.2 Hz, 2H), 2.88 (s, 2H), 2.79 (s, 4H), 2.76 (br d, <i>J</i> = 8.5 Hz, 1H), 2.69 (br dd, <i>J</i> = 13.4, 6.7 Hz, 1H), 2.63–2.53 (m, 5H), 2.48–2.41 (m, 3H), 2.37 (dt, <i>J</i> = 14.2, 7.2 Hz, 2H), 2.26 (d, <i>J</i> = 15 Hz, 2H), 2.22 (s, 11H), 2.15–2.07 (m, 1H), 2.04–1.91 (m, 3H), 1.76 (dq, <i>J</i> = 13.7, 6.6 Hz, 1H), 1.6 (br d, <i>J</i> = 7.3 Hz, 2H), 1.48 (br dd, <i>J</i> = 14.6, 4.9 Hz, 1H), 1.46–1.4 (m, 1H), 1.38 (br s, 1H), 1.28–1.21 (m, 3H), 1.20–1.05 (m, 33H), 0.98 (br d, <i>J</i> = 7 Hz, 6H), 0.92 (br d, <i>J</i> = 6.7 Hz, 2H), 0.9 (br d, <i>J</i> = 6.4 Hz, 4H), 0.68 (br d, <i>J</i> = 6.4 Hz, 2H), 0.62 (br d, <i>J</i> = 6.7 Hz, 2H), 0.57 (br d, <i>J</i> = 6.4 Hz, 4H), 0.51 (br d, <i>J</i> = 6.4 Hz, 4H).</div></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Preparation of Compound <b>22</b></h4><div id="sec4_1_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> (2<i>S</i>)-2-Amino-3-(2-chlorophenyl)-<i>N</i>-[4-[[(3<i>S</i>,6<i>S</i>,9<i>R</i>,10<i>S</i>, 11<i>S</i>,12<i>R</i>,13<i>R</i>,15<i>R</i>)-12-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-13-hydroxy-10-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-6-isopropyl-1,9,11,13,15-pentamethyl-2,5,8-trioxo-1,4,7-triazacyclohexadec-3-yl]methyl]phenyl]propanamide (<b>22a</b>)</h5><div class="NLM_p last">Step a: To a solution of (2<i>S</i>)-3-(2-chlorophenyl)-2-(9<i>H</i>-fluoren-9-ylmethoxycarbonylamino)propanoic acid (86.6 mg, 0.2 mmol), amine <b>E-22</b> (226 mg, 0.266 mmol), and DIPEA (101.9 μL, 0.585 mmol) in dry DMF (2.66 mL), HATU (111.2 mg, 0.292 mmol) was added. The reaction mixture was left to stir at room temperature overnight. EtOAc (30 mL) was added, and the organic layer was washed with sat. NaHCO<sub>3</sub> (3 × 15 mL) and brine (25 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the residue (330.8 mg; ES<sup>+</sup><i>m</i>/<i>z</i> 1253.41 [M + H]<sup>+</sup>) was dissolved in dry DCM (4.33 mL). Piperidine (131.3 μL, 1.329 mmol) was added, and the resulting mixture was stirred at room temperature for 2 h. The reaction mixture was concentrated under reduced pressure to afford the yellowish crude product, which was purified by flash column chromatography (SiO<sub>2</sub>, 12 g) with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent to give the title product (162.5 mg, yield 59.2%) as a white solid foam. ES<sup>+</sup><i>m</i>/<i>z</i> 1031.46 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(3<i>S</i>,6<i>S</i>,9<i>R</i>,10<i>S</i>,11<i>S</i>,12<i>R</i>,13<i>R</i>,15<i>R</i>)-12-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-13-hydroxy-10-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-6-isopropyl-1,9,11,13,15-pentamethyl-2,5,8-trioxo-1,4,7-triazacyclohexadec-3-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>22</b>)</h5><div class="NLM_p last">Step b: PS-Carbodiimide resin (81 mg, 0.089 mmol) was added to a dry reaction vessel. 2-Phenylacetic acid (9.3 mg, 0.069 mmol) and HOBt (7.4 mg, 0.048 mmol), dissolved in a mixture of DCM (1.25 mL) and DMF (75 μL), were added to the dry resin. The mixture was stirred at room temperature for 5 min upon which a solution of <b>22a</b> (78.6 mg, 0.069 mmol) in a mixture of DCM (1.25 mL) and DMF (75 μL) was added. The reaction mixture was left to stir at room temperature overnight. To the reaction, a mixture of PS-carbodiimide resin (16.2 mg, 0.018 mmol), 2-phenylacetic acid (1.9 mg, 0.014 mmol), and HOBt (1.5 mg, 0.01 mmol) suspended in DCM (0.5 mL) and DMF (30 μL) were added. Stirring at room temperature was continued overnight. The product was separated from the resin by filtration, and the resin was washed with DCM (3 × 3 mL), MeOH (3 × 3 mL), and DCM/MeOH 1:1 (3 × 3 mL). The combined organic solvents were evaporated under reduced pressure to afford the raw product as a yellowish oil, which was then suspended in EtOAc (30 mL). The organic layer was washed with sat. NaHCO<sub>3</sub> (3 × 25 mL) and brine (25 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure yielding the crude product. The crude product was purified by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent to give the title product (50 mg, yield 63%, purity 93.9%) as a white solid foam. ES<sup>+</sup><i>m</i>/<i>z</i> 1149.7 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.97 (s, 1H), 9.95 (s, 1H), 8.48 (dd, <i>J</i> = 8.3, 4.6 Hz, 2H), 8.45 (d, <i>J</i> = 6.8 Hz, 1H), 8.16 (d, <i>J</i> = 8.7 Hz, 1H), 7.82 (s, 1H), 7.5 (s, 1H), 7.43 (dd, <i>J</i> = 12.1, 8.3 Hz, 4H), 7.40–7.35 (m, 2H), 7.33–7.26 (m, 2H), 7.26–7.09 (m, 16H), 4.98–4.89 (m, 1H), 4.89–4.81 (m, 2H), 4.81–4.73 (m, 2H), 4.5 (d, <i>J</i> = 7.2 Hz, 1H), 4.4 (d, <i>J</i> = 7.3 Hz, 2H), 4.33 (d, <i>J</i> = 7.8 Hz, 1H), 4.23 (d, <i>J</i> = 7.5 Hz, 1H), 4.12 (s, 2H), 4.10–3.96 (m, 2H), 3.96–3.83 (m, 2H), 3.80–3.56 (m, 4H), 3.54–3.4 (m, 4H), 3.28–3.07 (m, 2H), 3.21 (d, <i>J</i> = 6.7 Hz, 6H), 3.07–2.96 (m, 4H), 2.96–2.85 (m, 4H), 2.81–2.63 (m, 4H), 2.63–2.52 (m, 1H), 2.48–2.35 (m, 1H), 2.34–2.09 (m, 4H), 2.22 (d, <i>J</i> = 9.2 Hz, 12H), 2.04–1.91 (m, 2H), 1.85 (t, <i>J</i> = 7.4 Hz, 2H), 1.65–1.45 (m, 4H), 1.38–1.01 (m, 34H), 0.96 (dd, <i>J</i> = 10.2, 7.3 Hz, 6H), 0.87 (d, <i>J</i> = 6.6 Hz, 2H), 0.85–0.78 (m, 6H), 0.72 (dd, <i>J</i> = 11.0, 6.6 Hz, 6H).</div></div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Preparation of Compound <b>24</b></h4><div id="sec4_1_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> (2<i>S</i>)-2-Amino-3-(2-chlorophenyl)-<i>N</i>-[4-[[(4<i>R</i>,7<i>S</i>,11<i>R</i>,12<i>S</i>, 13<i>S</i>,14<i>R</i>,15<i>R</i>,17<i>R</i>)-14-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-hydroxy-12-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-7-isopropyl-1,11,13,15,17-pentamethyl-2,6,10-trioxo-1,5,9-triazacyclooctadec-4-yl]methyl]phenyl]propanamide (<b>24a</b>)</h5><div class="NLM_p last">Step a: To a solution of (2<i>S</i>)-3-(2-chlorophenyl)-2-(9<i>H</i>-fluoren-9-ylmethoxycarbonylamino) propanoic acid (86.6 mg, 0.2 mmol), <b>E-24</b> (166.5 mg, 0.19 mmol), and DIPEA (72.7 μL, 0.417 mmol) in dry DMF (1.9 mL), HATU (79.3 mg, 0.209 mmol) was added. The reaction mixture was left to stir at room temperature overnight. EtOAc (30 mL) was added, and the organic layer was washed with sat. NaHCO<sub>3</sub> (3 × 15 mL) and brine (25 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the residue [241.6 mg), ES<sup>+</sup>, <i>m</i>/<i>z</i> 1281.49 [M + H]<sup>+</sup>] was dissolved in dry DCM (3.09 mL). Piperidine (93.6 μL, 0.948 mmol) was added, and the resulting mixture was stirred at room temperature for 2 h. The reaction mixture was concentrated under reduced pressure to afford the yellowish crude product, which was purified by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent to give the title product (137 mg, yield 68%) as a white solid foam. ES<sup>+</sup><i>m</i>/<i>z</i> 1059.46 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(4<i>R</i>,7<i>S</i>,11<i>R</i>,12<i>S</i>,13<i>S</i>,14<i>R</i>,15<i>R</i>,17<i>R</i>)-14-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-hydroxy-12-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-7-isopropyl-1,11,13,15,17-pentamethyl-2,6,10-trioxo-1,5,9-triazacyclooctadec-4-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>24</b>)</h5><div class="NLM_p last">Step b: PS-Carbodiimide resin (76.2 mg, 0.084 mmol) was added to a dry reaction vessel. 2-Phenylacetic acid (8.8 mg, 0.064 mmol) and HOBt (6.9 mg, 0.045 mmol), dissolved in a mixture of DCM (1.15 mL) and DMF (70 μL), were added to the dry resin. The mixture was stirred at room temperature for 5 min upon which a solution of <b>24a</b> (68.3 mg, 0.064 mmol) in a mixture of DCM (1.15 mL) and DMF (70 μL) was added. The reaction mixture was left to stir at room temperature overnight. The product was separated from the resin by filtration, and the resin was washed with DCM (3 × 2 mL), MeOH (3 × 2 mL), and DCM/MeOH 1:1 (3 × 2 mL). The combined organic solvents were evaporated under reduced pressure to afford the raw product as a yellowish oil, which was then suspended in EtOAc (25 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure, yielding the crude product. The crude product was purified by flash column chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent to give the title product (60.2 mg, yield 82.4%, purity 95.7%) as a white solid foam. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.53 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.02 (s, 1H), 9.96 (s, 1H), 8.55–8.47 (m, 3H), 7.71 (br s, 1H), 7.61–7.53 (m, 2H), 7.5 (d, <i>J</i> = 8.2 Hz, 2H), 7.48–7.42 (m, 2H), 7.39 (d, <i>J</i> = 7 Hz, 2H), 7.3 (dd, <i>J</i> = 7.6, 1.5 Hz, 2H), 7.26–7.14 (m, 13H), 7.13–7.08 (m, 3H), 4.93 (br d, <i>J</i> = 4.6 Hz, 1H), 4.81–4.74 (m, 2H), 4.62 (br d, <i>J</i> = 4.6 Hz, 2H), 4.42 (br d, <i>J</i> = 7.3 Hz, 2H), 4.37 (br d, <i>J</i> = 7 Hz, 2H), 4.23–4.17 (m, 2H), 4.15–4.07 (m, 4H), 4.07–3.9 (m, 4H), 3.66 (br s, 2H), 3.55–3.37 (m, 8H), 3.24–3.15 (m, 8H), 3.14–2.99 (m, 8H), 2.97–2.8 (m, 8H), 2.78 (s, 2H), 2.65 (br dd, <i>J</i> = 13.6, 8.1 Hz, 2H), 2.60–2.56 (m, 1H), 2.49–2.42 (m, 4H), 2.38–2.3 (m, 1H), 2.26–2.21 (m, 14H), 2.14–2.02 (m, 2H), 1.97–1.83 (m, 4H), 1.81–1.73 (m, 1H), 1.72–1.54 (m, 3H), 1.48–1.39 (m, 3H), 1.27–1.2 (m, 3H), 1.17–1.07 (m, 26H), 1.05 (br d, <i>J</i> = 7 Hz, 4H), 1.02–0.94 (m, 13H), 0.9 (br d, <i>J</i> = 6.7 Hz, 2H), 0.87 (d, <i>J</i> = 6.7 Hz, 3H), 0.78 (br d, <i>J</i> = 6.4 Hz, 3H), 0.69 (br d, <i>J</i> = 6.4 Hz, 3H), 0.64 (d, <i>J</i> = 6.4 Hz, 3H).</div></div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> General Procedure 6 for the Preparation of Intermediates <b>E-1b</b> to <b>E-1g</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Table S11</a>)</h4><div class="NLM_p last">To a solution of <b>E-1a</b> (1 eq) in dry DMF, HATU (1.1 eq), DIPEA (2.2 eq), and corresponding <i>N</i>-Fmoc-amino acid (1.05 eq) were added. The reaction mixture was stirred at room temperature for 1 h. Then, piperidine (1.5 eq) was added and the reaction mixture was stirred at room temperature overnight. DMF was evaporated, and the residue was dissolved in DCM. The mixture was washed with sat. NaHCO<sub>3</sub> (3×). The organic layer was washed with water (3×), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness to get the crude product. The crude product was purified by column chromatography using DCM:MeOH:NH<sub>4</sub>OH = 10:1:0.1 as an eluent to get the corresponding product (<b>E-1b</b> to <b>E-1g</b>).</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> General Procedure 7 for Compounds <b>28</b>–<b>33</b>, <b>35</b>–<b>38</b>, and <b>41</b>–<b>44</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h4><div class="NLM_p">To a solution of <b>E-1b</b> (1 eq) in dry DMF, HATU (1.4 eq), DIPEA (1.7 eq), and corresponding carboxylic acid (1 eq) were added and stirred at room temperature for 1 h. After the reaction was completed, DMF was evaporated and then extracted with water and DCM. The organic layer was dried over sodium sulfate, filtered, and evaporated to dryness. The crude product was purified by prep HPLC purification (extended A, high pH) or by column chromatography using DCM:MeOH:NH<sub>4</sub>OH = 10:1:0.1 as an eluent to get the title compound.</div><div id="sec4_1_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> <i>N</i>-[(1<i>S</i>)-1-[(2-Chlorophenyl)methyl]-2-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]anilino]-2-oxo-ethyl]-4-methoxybenzamide (<b>28</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 4-methoxybenzoic acid (10.2 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (48 mg, yield 61%, purity 99%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1179.6 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.99 (s, 1H), 9.96 (s, 1H), 8.60–8.56 (m, 3H), 8.11 (br d, <i>J</i> = 8.5 Hz, 2H), 7.9 (br d, <i>J</i> = 9.2 Hz, 2H), 7.84 (d, <i>J</i> = 7.9 Hz, 3H), 7.50–7.4 (m, 8H), 7.28 (br d, <i>J</i> = 8.5 Hz, 2H), 7.25–7.22 (m, 2H), 7.21 (d, <i>J</i> = 4 Hz, 2H), 7.14–7.1 (m, 3H), 6.99 (d, <i>J</i> = 7.9 Hz, 3H), 4.93 (td, <i>J</i> = 8.4, 6.1 Hz, 2H), 4.79 (br d, <i>J</i> = 4.9 Hz, 2H), 4.63 (br d, <i>J</i> = 4.3 Hz, 2H), 4.45 (d, <i>J</i> = 7.3 Hz, 2H), 4.37 (d, <i>J</i> = 7 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 2H), 4.17–4.08 (m, 2H), 4.07–4 (m, 4H), 4–3.94 (m, 2H), 3.89 (s, 1H), 3.85–3.78 (m, 6H), 3.73–3.65 (m, 2H), 3.59 (br dd, <i>J</i> = 11.4, 6.3 Hz, 2H), 3.51 (br d, <i>J</i> = 7 Hz, 2H), 3.48–3.43 (m, 2H), 3.31–3.23 (m, 4H), 3.22–3.17 (m, 5H), 3.13–2.96 (m, 5H), 2.93–2.85 (m, 4H), 2.79 (s, 5H), 2.72–2.53 (m, 5H), 2.50–2.34 (m, 5H), 2.28–2.2 (m, 12H), 2.13–2.04 (m, 2H), 2.03–1.94 (m, 2H), 1.77 (dq, <i>J</i> = 13.8, 6.9 Hz, 2H), 1.6 (br d, <i>J</i> = 7 Hz, 2H), 1.52–1.37 (m, 2H), 1.30–1.22 (m, 2H), 1.21–1.06 (m, 30H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.95–0.88 (m, 5H), 0.7 (d, <i>J</i> = 6.7 Hz, 3H), 0.64 (br d, <i>J</i> = 6.4 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.52 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(<i>o</i>-tolyl)acetyl]amino]propenamide (<b>29</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 2-(<i>o</i>-tolyl)acetic acid (10.1 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (25 mg, yield 31%, purity 96.27%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.41 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.98 (s, 1H), 9.94 (s, 1H), 8.41–8.34 (m, 3H), 8.16–8.09 (m, 2H), 7.94–7.89 (m, 1H), 7.46 (d, <i>J</i> = 8.5 Hz, 2H), 7.44–7.38 (m, 5H), 7.34–7.28 (m, 4H), 7.26–7.19 (m, 4H), 7.15–7.08 (m, 9H), 7.07–7.02 (m, 4H), 5.04–4.92 (m, 1H) 4.82–4.77 (m, 2H), 4.68–4.61 (m, 1H), 4.47 (br d, <i>J</i> = 7 Hz, 2H), 4.39 (br d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 2H), 4.16–4.09 (m, 3H), 4.05 (br d, <i>J</i> = 7 Hz, 3H), 4.01–3.95 (m, 1H), 3.94–3.89 (m, 1H), 3.84 (br d, <i>J</i> = 7.9 Hz, 2H), 3.69 (s, 2H), 3.67–3.63 (m, 2H), 3.52 (br d, <i>J</i> = 7.3 Hz, 3H), 3.48 (d, <i>J</i> = 3.4 Hz, 4H), 3.46–3.42 (m, 1H), 3.21 (s, 3H), 3.20–3.18 (m, 2H), 3.18–3.12 (m, 2H), 3.08–3.03 (m, 2H), 3.02–2.94 (m, 1H), 2.93–2.88 (m, 3H), 2.8 (s, 3H), 2.74–2.66 (m, 2H), 2.65–2.58 (m, 5H), 2.40–2.35 (m, 2H), 2.27 (br d, <i>J</i> = 15 Hz, 4H), 2.23–2.19 (m, 1H), 2.13 (s, 5H), 2.03–1.94 (m, 2H), 1.81–1.74 (m, 1H), 1.54–1.45 (m, 2H), 1.28–1.23 (m, 5H), 1.22–1.19 (m, 4H), 1.19–1.16 (m, 10H), 1.15–1.07 (m, 21H), 0.99 (br d, <i>J</i> = 7 Hz, 6H), 0.94–0.89 (m, 6H), 0.71–0.66 (m, 3H), 0.64–0.59 (m, 3H), 0.57 (br d, <i>J</i> = 6.4 Hz, 3H), 0.51 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(<i>m</i>-tolyl)acetyl]amino]propanamide (<b>30</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 2-(<i>m</i>-tolyl)acetic acid (10 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (46 mg, yield 58%, purity 98.6%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.56 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.96 (s, 1H), 9.92 (s, 1H), 8.53–8.35 (m, 3H), 8.09 (br d, <i>J</i> = 8.9 Hz, 2H), 7.89 (br d, <i>J</i> = 9.2 Hz, 2H), 7.48–7.35 (m, 6H), 7.31–7.25 (m, 4H), 7.24–7.2 (m, 2H), 7.19–7.14 (m, 2H), 7.14–7.07 (m, 5H), 7.01–6.91 (m, 6H), 4.81–4.7 (m, 3H), 4.62 (br d, <i>J</i> = 4 Hz, 2H), 4.44 (br d, <i>J</i> = 7.3 Hz, 2H), 4.36 (br d, <i>J</i> = 7.3 Hz, 2H), 4.28 (d, <i>J</i> = 7.6 Hz, 2H), 4.27–4.24 (m, 1H), 4.22 (d, <i>J</i> = 8.2 Hz, 2H), 4.12 (br s, 3H), 4.07–3.98 (m, 4H), 3.96 (br t, <i>J</i> = 7.5 Hz, 1H), 3.88 (s, 1H), 3.82 (br d, <i>J</i> = 7.9 Hz, 2H), 3.69 (s, 1H), 3.66 (br dq, <i>J</i> = 10.4, 6.4 Hz, 1H), 3.58 (br dq, <i>J</i> = 11.9, 7.3 Hz, 1H), 3.5 (br d, <i>J</i> = 6.7 Hz, 1H), 3.46–3.43 (m, 1H), 3.45–3.36 (m, 3H), 3.24 (br s, 3H), 3.2 (s, 3H), 3.18 (s, 3H), 3.17–3.1 (m, 2H), 3.10–2.99 (m, 4H), 2.96 (br d, <i>J</i> = 11 Hz, 1H), 2.92–2.88 (m, 1H), 2.89–2.87 (m, 1H), 2.78 (s, 3H), 2.64–2.54 (m, 4H), 2.48–2.41 (m, 2H), 2.41–2.31 (m, 2H), 2.26 (br d, <i>J</i> = 15 Hz, 2H), 2.23 (s, 6H), 2.22 (s, 8H), 2.14–2.07 (m, 2H), 1.97 (dt, <i>J</i> = 13.5, 6.8 Hz, 2H), 1.76 (dq, <i>J</i> = 13.6, 6.8 Hz, 2H), 1.6 (br d, <i>J</i> = 7.3 Hz, 2H), 1.47 (br dd, <i>J</i> = 14.8, 5 Hz, 2H), 1.23 (br s, 2H), 1.20–1.05 (m, 29H), 0.98 (br d, <i>J</i> = 7.3 Hz, 6H), 0.92 (br d, <i>J</i> = 6.7 Hz, 2H), 0.9 (br d, <i>J</i> = 6.4 Hz, 4H), 0.68 (br d, <i>J</i> = 6.7 Hz, 3H), 0.62 (br d, <i>J</i> = 6.7 Hz, 3H), 0.57 (br d, <i>J</i> = 6.7 Hz, 3H), 0.49 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(<i>p</i>-tolyl)acetyl]amino]propanamide (<b>31</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 2-(<i>p</i>-tolyl)acetic acid (10 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (50 mg, yield 63%, purity 97.96%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.55 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.96 (s, 1H), 9.92 (s, 1H), 8.42 (t, <i>J</i> = 7.4 Hz, 3H), 8.1 (br d, <i>J</i> = 8.9 Hz, 2H), 7.9 (br d, <i>J</i> = 9.2 Hz, 2H), 7.47–7.38 (m, 6H), 7.31–7.14 (m, 8H), 7.13–7.09 (m, 3H), 7.06–7.02 (m, 7H), 4.82–4.74 (m, 3H), 4.63 (br d, <i>J</i> = 4.3 Hz, 2H), 4.45 (d, <i>J</i> = 7.3 Hz, 2H), 4.37 (br d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 1H), 4.15–3.95 (m, 10H), 3.92–3.87 (m, 1H), 3.84 (br d, <i>J</i> = 7 Hz, 2H), 3.7 (s, 1H), 3.67 (br dd, <i>J</i> = 10.5, 6.6 Hz, 2H), 3.6 (br dd, <i>J</i> = 11.6, 7 Hz, 2H), 3.52 (br d, <i>J</i> = 6.7 Hz, 2H), 3.48–3.43 (m, 2H), 3.4 (d, <i>J</i> = 5.8 Hz, 4H), 3.21 (s, 3H), 3.20–3.18 (m, 3H), 3.14 (d, <i>J</i> = 6.1 Hz, 2H), 3.11–2.95 (m, 5H), 2.93–2.86 (m, 3H), 2.8 (s, 4H), 2.77–2.65 (m, 2H), 2.64–2.55 (m, 5H), 2.48–2.33 (m, 3H), 2.24 (d, <i>J</i> = 12.2 Hz, 16H), 2.28–2.21 (m, 1H), 2.16–2.09 (m, 1H), 2.08 (s, 1H), 1.98 (dt, <i>J</i> = 13.4, 6.7 Hz, 2H), 1.80–1.74 (m, 1H), 1.61 (br d, <i>J</i> = 7.3 Hz, 2H), 1.52–1.34 (m, 3H), 1.32–1.22 (m, 3H), 1.21–1.15 (m, 12H), 1.15–1.06 (m, 18H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.94–0.89 (m, 4H), 0.69 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.60–0.55 (m, 3H), 0.5 (d, <i>J</i> = 7 Hz, 3H).</div></div><div id="sec4_1_10_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(2-fluorophenyl)acetyl]amino]propenamide (<b>32</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 3-fluorophenylacetic acid (11 mg, 0.73 mmol) and <b>E-1b</b> (60 mg, 0.057 mmol), the title product (25 mg, yield 37%, purity 99.91%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1181.61 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.98 (s, 1H), 9.94 (s, 1H), 8.55–8.49 (m, 3H), 8.13–8.1 (m, 1H), 7.88 (br d, <i>J</i> = 1.7 Hz, 1H), 7.45 (d, <i>J</i> = 8.1 Hz, 3H), 7.43–7.39 (m, 8H), 7.31 (br d, <i>J</i> = 7.5 Hz, 2H), 7.30–7.28 (m, 2H), 7.26–7.2 (m, 9H), 7.19 (s, 1H), 7.14–7.1 (m, 11H), 7.09 (s, 1H), 4.78 (br d, <i>J</i> = 6.8 Hz, 2H), 4.45 (d, <i>J</i> = 7.2 Hz, 1H), 4.31 (d, <i>J</i> = 7.9 Hz, 2H), 4.17–4.12 (m, 3H), 4.08–4.06 (m, 3H), 4.05–3.99 (m, 4H), 3.92–3.89 (m, 1H), 3.86–3.79 (m, 2H), 3.73–3.69 (m, 2H), 3.57–3.49 (m, 12H), 3.22–3.18 (m, 12H), 3.16 (br d, <i>J</i> = 5.9 Hz, 2H), 3.07 (br s, 1H), 3.06–3.05 (m, 3H), 3–2.94 (m, 3H), 2.89 (br s, 2H), 2.79 (s, 5H), 2.64–2.56 (m, 8H), 2.29–2.25 (m, 4H), 2.23 (s, 11H), 2.13–2.08 (m, 2H), 2.01–1.95 (m, 4H), 1.81–1.74 (m, 2H), 1.63–1.59 (m, 2H), 1.52–1.45 (m, 3H), 1.44–1.38 (m, 1H), 1.25 (br d, <i>J</i> = 6.4 Hz, 5H), 1.2 (br s, 2H), 1.18–1.08 (m, 55H), 0.99 (br d, <i>J</i> = 6.8 Hz, 8H), 0.94–0.9 (m, 10H), 0.69 (br d, <i>J</i> = 6.8 Hz, 3H), 0.65–0.62 (m, 2H), 0.58 (br d, <i>J</i> = 6.4 Hz, 5H), 0.5 (br d, <i>J</i> = 6.6 Hz, 5H).</div></div><div id="sec4_1_10_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(3-fluorophenyl)acetyl]amino]propanamide (<b>33</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 3-fluorophenylacetic acid (10.17 mg, 0.07 mmol and <b>E-1b</b> (70 mg, 0.07 mmol), the title product (24.87 mg, yield 30%, purity 94.68%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1181.52 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (s, 1H), 9.97 (s, 1H), 8.56–8.52 (m, 3H), 8.1 (br d, <i>J</i> = 8.4 Hz, 2H), 7.9 (br d, <i>J</i> = 9.7 Hz, 2H), 7.46 (d, <i>J</i> = 8.6 Hz, 2H), 7.42 (d, <i>J</i> = 8.6 Hz, 2H), 7.39 (d, <i>J</i> = 8.1 Hz, 2H), 7.31–7.26 (m, 5H), 7.24–7.19 (m, 2H), 7.19–7.14 (m, 3H), 7.14–7.1 (m, 3H), 7.05–6.97 (m, 5H), 4.82–4.76 (m, 3H), 4.63 (br d, <i>J</i> = 4.2 Hz, 2H), 4.45 (d, <i>J</i> = 7.3 Hz, 2H), 4.37 (d, <i>J</i> = 7.2 Hz, 2H), 4.29 (d, <i>J</i> = 7.7 Hz, 2H), 4.23 (d, <i>J</i> = 8.3 Hz, 2H), 4.13 (br s, 2H), 4.10–3.94 (m, 5H), 4–3.94 (m, 1H), 3.89 (s, 1H), 3.84 (br d, <i>J</i> = 8.1 Hz, 2H), 3.7 (s, 1H), 3.67 (br dd, <i>J</i> = 11.4, 6.2 Hz, 1H), 3.55–3.44 (m, 6H), 3.34–3.34 (m, 25H), 3.25–3.15 (m, 7H), 3.12–2.96 (m, 5H), 2.94–2.85 (m, 3H), 2.8 (s, 3H), 2.7 (br dd, <i>J</i> = 13.5, 6.5 Hz, 2H), 2.64–2.55 (m, 4H), 2.50–2.35 (m, 4H), 2.28–2.18 (m, 11H), 2.1 (s, 2H), 2.03–1.91 (m, 2H), 1.81–1.72 (m, 1H), 1.61 (br d, <i>J</i> = 6.8 Hz, 2H), 1.54–1.39 (m, 2H), 1.28–1.22 (m, 2H), 1.21–1.07 (m, 22H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.96–0.87 (m, 5H), 0.69 (d, <i>J</i> = 6.8 Hz, 3H), 0.63 (br d, <i>J</i> = 6.8 Hz, 3H), 0.58 (d, <i>J</i> = 6.8 Hz, 3H), 0.51 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_1_10_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> (2<i>S</i>)-3-(2-Chlorophenyl)-2-[[2-(3-cyanophenyl)acetyl]amino]-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propenamide (<b>35</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 3-cyanophenylacetic acid (11 mg, 0.68 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (50 mg, yield 63%, purity 98.83%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1188.35 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.97 (s, 1H), 8.58 (t, <i>J</i> = 7.4 Hz, 3H), 8.1 (br d, <i>J</i> = 8.9 Hz, 2H), 7.89 (br d, <i>J</i> = 8.9 Hz, 2H), 7.71–7.64 (m, 2H), 7.57 (s, 2H), 7.50–7.45 (m, 5H), 7.45–7.4 (m, 3H), 7.37 (d, <i>J</i> = 7.7 Hz, 2H), 7.26 (t, <i>J</i> = 6.7 Hz, 3H), 7.23–7.18 (m, 2H), 7.17–7.14 (m, 2H), 7.13–7.08 (m, 4H), 4.81–4.75 (m, 3H), 4.62 (br d, <i>J</i> = 4 Hz, 1H), 4.44 (br d, <i>J</i> = 7 Hz, 2H), 4.36 (br d, <i>J</i> = 7.3 Hz, 2H), 4.28 (d, <i>J</i> = 7.6 Hz, 2H), 4.22 (d, <i>J</i> = 8.5 Hz, 2H), 4.16–4.09 (m, 2H), 4.08–3.99 (m, 5H), 3.98–3.93 (m, 1H), 3.88 (s, 1H), 3.83 (br d, <i>J</i> = 7.3 Hz, 2H), 3.69 (s, 1H), 3.68–3.63 (m, 2H), 3.59–3.56 (m, 1H), 3.53 (br d, <i>J</i> = 13.4 Hz, 4H), 3.50–3.43 (m, 3H), 3.2 (s, 4H), 3.19–3.16 (m, 2H), 3.15 (d, <i>J</i> = 5.5 Hz, 1H), 3.12–3.07 (m, 1H), 3.06–3 (m, 3H), 2.99–2.94 (m, 2H), 2.92–2.84 (m, 4H), 2.79 (s, 4H), 2.77–2.71 (m, 1H), 2.70–2.62 (m, 2H), 2.60–2.55 (m, 4H), 2.41–2.33 (m, 2H), 2.28–2.19 (m, 12H), 2.12–2.05 (m, 4H), 2.01–1.93 (m, 2H), 1.79–1.71 (m, 1H), 1.63–1.57 (m, 2H), 1.56–1.38 (m, 4H), 1.35–1.21 (m, 8H), 1.20–1.12 (m, 19H), 1.11–1.06 (m, 11H), 0.98 (br d, <i>J</i> = 7 Hz, 5H), 0.95–0.88 (m, 6H), 0.87–0.79 (m, 1H), 0.68 (d, <i>J</i> = 6.7 Hz, 3H), 0.62 (br d, <i>J</i> = 6.4 Hz, 3H), 0.57 (d, <i>J</i> = 6.7 Hz, 3H), 0.5 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> (2<i>S</i>)-3-(2-Chlorophenyl)-2-[[2-(4-cyanophenyl)acetyl]amino]-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propenamide (<b>36</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 4-cyanophenylac etic acid (10.8 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (60 mg, yield 75%, purity 99.26%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> = 1188.35 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.02 (s, 1H), 9.98 (s, 1H), 8.65–8.56 (m, 2H), 8.11 (br d, <i>J</i> = 8.9 Hz, 2H), 7.9 (br d, <i>J</i> = 8.9 Hz, 2H), 7.72 (d, <i>J</i> = 8.5 Hz, 4H), 7.45 (d, <i>J</i> = 8.5 Hz, 3H), 7.42 (d, <i>J</i> = 8.5 Hz, 3H), 7.38 (dd, <i>J</i> = 7.9, 1.2 Hz, 3H), 7.34 (d, <i>J</i> = 8.2 Hz, 3H), 7.30–7.26 (m, 3H), 7.25–7.2 (m, 2H), 7.20–7.15 (m, 2H), 7.14–7.1 (m, 3H), 4.82–4.77 (m, 3H), 4.63 (br d, <i>J</i> = 4 Hz, 1H), 4.45 (d, <i>J</i> = 7 Hz, 1H), 4.37 (d, <i>J</i> = 7 Hz, 1H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 1H), 4.15–4.11 (m, 2H), 4.10–4 (m, 5H), 4–3.94 (m, 1H), 3.89 (s, 1H), 3.84 (br d, <i>J</i> = 8.5 Hz, 2H), 3.7 (s, 1H), 3.69–3.64 (m, 1H), 3.61 (s, 1H), 3.58 (d, <i>J</i> = 7.3 Hz, 3H), 3.55–3.5 (m, 2H), 3.46 (br d, <i>J</i> = 5.2 Hz, 1H), 3.21 (s, 3H), 3.19 (s, 4H), 3.18–3.13 (m, 1H), 3.12–3.07 (m, 1H), 3.06–3.02 (m, 2H), 3.02–2.94 (m, 2H), 2.93–2.85 (m, 3H), 2.8 (s, 3H), 2.78–2.67 (m, 2H), 2.65–2.54 (m, 5H), 2.48–2.42 (m, 2H), 2.40–2.33 (m, 1H), 2.29–2.19 (m, 12H), 2.12–2.05 (m, 1H), 2–1.94 (m, 2H), 1.81–1.72 (m, 2H), 1.66–1.57 (m, 2H), 1.53–1.41 (m, 2H), 1.40–1.33 (m, 1H), 1.31–1.26 (m, 1H), 1.26–1.22 (m, 6H), 1.19 (br d, <i>J</i> = 12.2 Hz, 6H), 1.17–1.12 (m, 14H), 1.12–1.07 (m, 12H), 0.99 (br d, <i>J</i> = 7 Hz, 5H), 0.94–0.84 (m, 6H), 0.68 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.4 Hz, 3H), 0.58 (d, <i>J</i> = 6.4 Hz, 3H), 0.51 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(4-methylsulfonylphenyl)acetyl]amino]propenamide (<b>37</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 3-(methylsulfonyl)phenylacetic acid (15 mg, 0.7 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (50 mg, yield 60%, purity 98.8%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1241.34 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.97 (s, 1H), 8.64–8.59 (m, 2H), 8.11 (br d, <i>J</i> = 8.5 Hz, 2H), 7.9 (br d, <i>J</i> = 8.5 Hz, 2H), 7.81 (s, 2H), 7.78 (d, <i>J</i> = 7.9 Hz, 2H), 7.56–7.52 (m, 2H), 7.5 (s, 1H), 7.48 (s, 1H), 7.45 (d, <i>J</i> = 8.2 Hz, 2H), 7.42 (d, <i>J</i> = 8.5 Hz, 2H), 7.37 (dd, <i>J</i> = 7.9, 1.2 Hz, 2H), 7.29–7.25 (m, 3H), 7.21 (td, <i>J</i> = 7.6, 1.5 Hz, 2H), 7.18–7.14 (m, 2H), 7.14–7.09 (m, 4H), 4.80–4.76 (m, 3H), 4.63 (br d, <i>J</i> = 4.3 Hz, 1H), 4.45 (d, <i>J</i> = 7.3 Hz, 1H), 4.37 (br d, <i>J</i> = 7 Hz, 1H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.5 Hz, 1H), 4.16–4.09 (m, 2H), 4.08–4 (m, 5H), 3.99–3.94 (m, 1H), 3.91–3.88 (m, 1H), 3.84 (br d, <i>J</i> = 8.5 Hz, 2H), 3.7 (s, 1H), 3.69–3.63 (m, 1H), 3.61 (s, 3H), 3.58 (br s, 1H), 3.52 (br d, <i>J</i> = 6.7 Hz, 3H), 3.46 (br d, <i>J</i> = 5.2 Hz, 3H), 3.42–3.37 (m, 2H), 3.23–3.2 (m, 3H), 3.19–3.17 (m, 7H), 3.17–3.14 (m, 1H), 3.12–3.08 (m, 1H), 3.07–3.01 (m, 3H), 3–2.94 (m, 2H), 2.94–2.85 (m, 3H), 2.8 (s, 3H), 2.71–2.66 (m, 2H), 2.63–2.55 (m, 5H), 2.43–2.33 (m, 3H), 2.29–2.2 (m, 12H), 2.13–2.07 (m, 1H), 2–1.95 (m, 2H), 1.81–1.72 (m, 2H), 1.68–1.58 (m, 2H), 1.57–1.53 (m, 2H), 1.51–1.45 (m, 2H), 1.44–1.38 (m, 1H), 1.35–1.28 (m, 3H), 1.24 (br s, 3H), 1.21–1.15 (m, 12H), 1.15–1.07 (m, 18H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.94–0.9 (m, 5H), 0.69 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.51 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[[2-(3-methylsulfonylphenyl)acetyl]amino]propenamide (<b>38</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 4-(methylsulfonyl)phenylacetic acid (14.3 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (50 mg, yield 60%, purity 98.88%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1241.33 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.99 (s, 1H), 8.66–8.6 (m, 2H), 8.11 (br d, <i>J</i> = 8.5 Hz, 2H), 7.9 (br d, <i>J</i> = 8.9 Hz, 2H), 7.81 (d, <i>J</i> = 8.2 Hz, 4H), 7.48–7.36 (m, 12H), 7.30–7.26 (m, 3H), 7.24–7.1 (m, 7H), 4.83–4.76 (m, 3H), 4.63 (br d, <i>J</i> = 4.3 Hz, 1H), 4.45 (d, <i>J</i> = 7 Hz, 1H), 4.37 (br d, <i>J</i> = 7.3 Hz, 1H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 1H), 4.17–4.11 (m, 2H), 4.08–3.99 (m, 5H), 3.97 (br t, <i>J</i> = 7.5 Hz, 1H), 3.89 (s, 1H), 3.84 (br d, <i>J</i> = 7.6 Hz, 2H), 3.7 (s, 1H), 3.69–3.64 (m, 1H), 3.63 (s, 1H), 3.6 (d, <i>J</i> = 7 Hz, 3H), 3.56 (s, 1H), 3.52 (br d, <i>J</i> = 6.7 Hz, 2H), 3.46 (br d, <i>J</i> = 5.2 Hz, 1H), 3.21 (s, 4H), 3.19 (s, 7H), 3.16 (d, <i>J</i> = 5.8 Hz, 1H), 3.11–2.96 (m, 7H), 2.93–2.85 (m, 3H), 2.8 (s, 3H), 2.65–2.61 (m, 1H), 2.61–2.55 (m, 3H), 2.40–2.34 (m, 1H), 2.29–2.26 (m, 1H), 2.23 (s, 12H), 2.01–1.94 (m, 2H), 1.81–1.73 (m, 2H), 1.64–1.58 (m, 2H), 1.52–1.45 (m, 1H), 1.45–1.38 (m, 1H), 1.36–1.3 (m, 1H), 1.28–1.22 (m, 6H), 1.19 (br d, <i>J</i> = 13.1 Hz, 6H), 1.17–1.12 (m, 14H), 1.12–1.07 (m, 12H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.93 (br d, <i>J</i> = 6.7 Hz, 2H), 0.91 (br d, <i>J</i> = 6.4 Hz, 3H), 0.88–0.83 (m, 2H), 0.69 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.51 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> <i>N</i>-[(1<i>S</i>)-1-[(2-Chlorophenyl)methyl]-2-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-[(2<i>R</i>,4<i>R</i>,5<i>S</i>,6<i>S</i>)-5-hydroxy-4-methoxy-4,6-dimethyltetrahydropyran-2-yl]oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]anilino]-2-oxo-ethyl]benzamide (<b>41</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from benzoic acid (8.2 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (67.1 mg, 87.1%, purity 95.8%) was obtained. Purification was performed by flash column chromatography. ES<sup>+</sup><i>m</i>/<i>z</i> 1149.62 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.97 (s, 1H), 8.76–8.72 (m, 3H), 8.11 (br d, <i>J</i> = 8.9 Hz, 1H), 7.9 (br d, <i>J</i> = 8.9 Hz, 1H), 7.87–7.83 (m, 4H), 7.56–7.51 (m, 2H), 7.51–7.44 (m, 8H), 7.42 (td, <i>J</i> = 4.7, 2 Hz, 3H), 7.29 (br d, <i>J</i> = 8.5 Hz, 1H), 7.25–7.2 (m, 4H), 7.15–7.1 (m, 4H), 4.96 (td, <i>J</i> = 8.5, 5.8 Hz, 2H), 4.79 (br d, <i>J</i> = 4.6 Hz, 2H), 4.63 (br d, <i>J</i> = 4 Hz, 1H), 4.45 (d, <i>J</i> = 7.3 Hz, 2H), 4.37 (d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.9 Hz, 1H), 4.23 (d, <i>J</i> = 8.5 Hz, 1H), 4.16–4.08 (m, 2H), 4.08–4 (m, 4H), 4–3.94 (m, 1H), 3.89 (s, 1H), 3.84 (br d, <i>J</i> = 7.6 Hz, 1H), 3.7 (s, 1H), 3.67 (br dd, <i>J</i> = 11.1, 6.3 Hz, 1H), 3.62–3.56 (m, 1H), 3.52 (br d, <i>J</i> = 7 Hz, 2H), 3.46 (br d, <i>J</i> = 5.5 Hz, 1H), 3.33–3.22 (m, 5H), 3.21 (s, 4H), 3.19 (s, 2H), 3.14–3.05 (m, 3H), 3.05–2.94 (m, 2H), 2.93–2.85 (m, 4H), 2.8 (s, 4H), 2.73–2.66 (m, 1H), 2.64–2.55 (m, 4H), 2.49–2.35 (m, 4H), 2.28–2.17 (m, 14H), 2.09 (s, 2H), 2.03–1.93 (m, 3H), 1.81–1.74 (m, 1H), 1.6 (br d, <i>J</i> = 7.3 Hz, 2H), 1.51–1.38 (m, 3H), 1.24 (br s, 3H), 1.22–1.05 (m, 33H), 0.99 (br d, <i>J</i> = 7.3 Hz, 6H), 0.95–0.89 (m, 6H), 0.7 (d, <i>J</i> = 6.7 Hz, 2H), 0.64 (br d, <i>J</i> = 6.7 Hz, 2H), 0.59 (d, <i>J</i> = 6.7 Hz, 4H), 0.52 (d, <i>J</i> = 6.7 Hz, 4H).</div></div><div id="sec4_1_10_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> <i>N</i>-[(1<i>S</i>)-1-[(2-Chlorophenyl)methyl]-2-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]anilino]-2-oxo-ethyl]-1-(4-methoxyphenyl)cyclopropanecarboxamide (<b>42</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from 1-(4-methoxyphenyl)cyclopropanecarboxylic acid (13 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (54 mg, yield 66%, purity 96.7%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1219.59 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.04 (s, 1H), 10.01 (s, 1H), 8.11 (br d, <i>J</i> = 8.9 Hz, 2H), 7.91 (br d, <i>J</i> = 8.5 Hz, 2H), 7.41–7.35 (m, 6H), 7.30–7.19 (m, 9H), 7.16–7.09 (m, 5H), 6.91 (d, <i>J</i> = 8.9 Hz, 3H), 6.3 (d, <i>J</i> = 7.9 Hz, 3H), 4.81–4.73 (m, 3H), 4.63 (br d, <i>J</i> = 4.9 Hz, 2H), 4.45 (d, <i>J</i> = 7.3 Hz, 1H), 4.37 (br d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.5 Hz, 2H), 4.13 (br s, 2H), 4.08–4.03 (m, 4H), 4.03–4 (m, 1H), 3.99–3.94 (m, 1H), 3.92–3.87 (m, 1H), 3.83 (br d, <i>J</i> = 7.3 Hz, 2H), 3.78 (s, 5H), 3.7 (s, 1H), 3.69–3.63 (m, 2H), 3.62–3.55 (m, 1H), 3.52 (br d, <i>J</i> = 7 Hz, 2H), 3.48–3.43 (m, 1H), 3.23–3.2 (m, 4H), 3.19 (s, 2H), 3.13–3.07 (m, 3H), 3.06–3.01 (m, 3H), 3.01–2.94 (m, 2H), 2.94–2.85 (m, 4H), 2.8 (s, 4H), 2.77–2.73 (m, 1H), 2.72–2.64 (m, 2H), 2.63–2.55 (m, 5H), 2.49–2.42 (m, 2H), 2.42–2.33 (m, 2H), 2.29–2.19 (m, 13H), 2.11–2.06 (m, 4H), 2–1.94 (m, 2H), 1.76 (dq, <i>J</i> = 13.8, 6.9 Hz, 1H), 1.61 (br d, <i>J</i> = 7.3 Hz, 2H), 1.51–1.38 (m, 3H), 1.32–1.26 (m, 3H), 1.26–1.18 (m, 7H), 1.18–1.15 (m, 9H), 1.15–1.11 (m, 13H), 1.10–1.07 (m, 6H), 0.99 (br d, <i>J</i> = 7.3 Hz, 6H), 0.94–0.89 (m, 8H), 0.68 (d, <i>J</i> = 6.4 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.5 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_10_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-hydroxy-2-phenylacetyl)amino]propenamide (<b>43</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from (<i>rac</i>)-mandelic acid (10.2 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (50 mg, yield 63%) was obtained by prep HPLC purification as two epimers. ES<sup>+</sup><i>m</i>/<i>z</i> 1179.58 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.99 (s, 1H), 9.97–9.95 (m, 1H), 9.93 (s, 1H), 8.18–8.15 (m, 1H), 8.13–8.07 (m, 2H), 7.93–7.88 (m, 1H), 7.46 (d, <i>J</i> = 8.6 Hz, 1H), 7.44–7.41 (m, 2H), 7.41–7.4 (m, 1H), 7.40–7.34 (m, 4H), 7.31–7.22 (m, 8H), 7.22–7.17 (m, 4H), 7.17–7.1 (m, 6H), 6.36–6.33 (m, 1H), 6.25–6.21 (m, 1H), 4.95 (d, <i>J</i> = 5 Hz, 1H), 4.92 (d, <i>J</i> = 4.4 Hz, 1H), 4.82–4.78 (m, 2H), 4.76–4.71 (m, 1H), 4.63 (br d, <i>J</i> = 4.4 Hz, 1H), 4.45 (d, <i>J</i> = 7.3 Hz, 1H), 4.38 (d, <i>J</i> = 7.3 Hz, 1H), 4.29 (d, <i>J</i> = 7.7 Hz, 2H), 4.23 (d, <i>J</i> = 8.4 Hz, 1H), 4.18–4.11 (m, 2H), 4.11–4.08 (m, 1H), 4.08–4 (m, 5H), 4–3.95 (m, 1H), 3.89 (br s, 1H), 3.86–3.83 (m, 1H), 3.71 (d, <i>J</i> = 2 Hz, 1H), 3.70–3.64 (m, 1H), 3.63–3.57 (m, 1H), 3.52 (br d, <i>J</i> = 6.8 Hz, 1H), 3.48–3.44 (m, 1H), 3.28–3.23 (m, 1H), 3.21 (s, 4H), 3.20–3.16 (m, 6H), 3.15–3.08 (m, 2H), 3.07–2.96 (m, 4H), 2.94–2.85 (m, 4H), 2.8 (d, <i>J</i> = 1.3 Hz, 4H), 2.78–2.74 (m, 1H), 2.73–2.67 (m, 1H), 2.64–2.57 (m, 4H), 2.50–2.45 (m, 2H), 2.43–2.38 (m, 2H), 2.30–2.21 (m, 12H), 2.30–2.2 (m, 1H), 2.14–2.1 (m, 1H), 2.10–2.07 (m, 1H), 1.98 (td, <i>J</i> = 13.1, 6.7 Hz, 3H), 1.80–1.75 (m, 1H), 1.64–1.59 (m, 2H), 1.58–1.52 (m, 1H), 1.51–1.45 (m, 2H), 1.44–1.38 (m, 1H), 1.28–1.23 (m, 3H), 1.22–1.19 (m, 3H), 1.18–1.16 (m, 8H), 1.15–1.13 (m, 10H), 1.12–1.1 (m, 6H), 1.09 (br d, <i>J</i> = 6.1 Hz, 8H), 1 (br d, <i>J</i> = 7.2 Hz, 4H), 0.94 (br d, <i>J</i> = 6.2 Hz, 2H), 0.91 (dd, <i>J</i> = 6.6, 1.7 Hz, 4H), 0.69 (t, <i>J</i> = 7 Hz, 2H), 0.66–0.62 (m, 2H), 0.59 (dd, <i>J</i> = 9.4, 6.8 Hz, 4H), 0.51 (dd, <i>J</i> = 11.5, 6.7 Hz, 4H).</div></div><div id="sec4_1_10_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> (2<i>S</i>)-3-(2-Chlorophenyl)-2-(2-phenylpropanoylamino)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-12-hydroxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propenamide (<b>44</b>)</h5><div class="NLM_p last">Following general procedure 7: starting from <i>rac</i>-2-phenylpropionic acid (11 mg, 0.73 mmol) and <b>E-1b</b> (60 mg, 0.057 mmol), the title product (30 mg, yield 45%, purity 96.79%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1177.5 [M + H]<sup>+</sup>. Two sets of signals in the NMR (mixture of diasteroisomers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.98 (s, 1H), 9.93 (s, 1H), 9.89 (m, 1H), 9.85 (s, 1H), 8.38–8.29 (m, 3H), 8.14–8.06 (m, 2H), 7.93–7.86 (m, 1H), 7.46 (d, <i>J</i> = 8.2 Hz, 1H), 7.44–7.41 (m, 3H), 7.41–7.37 (m, 2H), 7.35 (br d, <i>J</i> = 8.5 Hz, 2H), 7.32 (br d, <i>J</i> = 7 Hz, 4H), 7.30–7.2 (m, 24H), 7.20–7.14 (m, 4H), 7.14–7.05 (m, 10H), 7.03–6.97 (m, 2H), 4.81–4.76 (m, 4H), 4.76–4.69 (m, 2H), 4.65–4.6 (m, 2H), 4.45 (br d, <i>J</i> = 7 Hz, 2H), 4.37 (br d, <i>J</i> = 7 Hz, 2H), 4.29 (d, <i>J</i> = 7.9 Hz, 3H), 4.23 (d, <i>J</i> = 8.2 Hz, 2H), 4.14–4.01 (m, 13H), 4–3.93 (m, 2H), 3.89 (br d, <i>J</i> = 2.1 Hz, 2H), 3.86–3.8 (m, 4H), 3.77–3.68 (m, 8H), 3.68–3.57 (m, 4H), 3.51 (br d, <i>J</i> = 6.1 Hz, 4H), 3.45 (br d, <i>J</i> = 2.1 Hz, 2H), 3.21 (s, 8H), 3.19 (s, 4H), 3.17–3.11 (m, 2H), 3.10–3.03 (m, 8H), 3.02–2.93 (m, 5H), 2.93–2.87 (m, 7H), 2.79 (d, <i>J</i> = 3.7 Hz, 7H), 2.71–2.66 (m, 2H), 2.64 (br d, <i>J</i> = 1.8 Hz, 2H), 2.63–2.55 (m, 9H), 2.49–2.42 (m, 4H), 2.42–2.33 (m, 4H), 2.25–2.21 (m, 25H), 2.14–2.07 (m, 3H), 2.03–1.93 (m, 6H), 1.82–1.7 (m, 2H), 1.6 (br d, <i>J</i> = 7.9 Hz, 4H), 1.52–1.44 (m, 4H), 1.44–1.37 (m, 4H), 1.34 (s, 2H), 1.29 (d, <i>J</i> = 7 Hz, 8H), 1.27–1.23 (m, 12H), 1.23–1.19 (m, 12H), 1.19–1.15 (m, 18H), 1.15–1.11 (m, 30H), 1.09 (br d, <i>J</i> = 5.8 Hz, 14H), 0.99 (br d, <i>J</i> = 7 Hz, 12H), 0.94–0.89 (m, 10H), 0.71–0.66 (m, 4H), 0.65–0.6 (m, 4H), 0.57 (br dd, <i>J</i> = 9.8, 6.7 Hz, 7H), 0.5 (br dd, <i>J</i> = 15.0, 6.7 Hz, 7H).</div></div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> Preparation of Compound <b>39</b></h4><div id="sec4_1_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[[4-[[(2<i>S</i>)-2-[[2-(4-Methoxycarbonylphenyl)acetyl]amino]-3-(2-chlorophenyl)propanoyl]amino]phenyl]methyl]-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-13-(β-<span class="smallcaps smallerCapital">d</span>-desasaminyl)oxy-12,13,15-trihydroxy-2-isopropyl-8,10,12,14,16-pentamethyl-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>39a</b>)</h5><div class="NLM_p last">To a solution of (<i>p</i>-methoxycarbonyl)phenylacetic acid (21 mg, 0.11 mmol), HOAT (7 mg, 0.05 mmol), and HATU (42 mg, 0.11 mmol) in DCM (1 mL), TMP (28 μL, 0.175 mmol) was added and the reaction mixture was stirred at room temperature for 30 min. A solution of <b>E-1b</b> (105 mg, 0.1 mmol) in DCM (1 mL) was added, and the reaction mixture was stirred at room temperature for 1 h. The reaction mixture was diluted with DCM (5 mL), and the reaction was quenched with sat. NaHCO<sub>3</sub> (5 mL). Layers were separated, and the aqueous one was extracted with DCM (5 mL). Combined organics were washed with water (5 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated to dryness. The sample was purified by flash chromatography with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5 as an eluent to yield the title compound as a white solid (88 mg). ES<sup>+</sup><i>m</i>/<i>z</i> 1221.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[[4-[[(2<i>S</i>)-2-[[2-(4-Carboxyphenyl)Acetyl]Amino]-3-(2-Chlorophenyl)Propanoyl] Amino]Phenyl] Methyl]-15-(α-L-Cladinosyl)Oxy-13- (β-D-Desasaminyl)Oxy-12,13,15-Trihydroxy-2-Isopropyl-8,10,12,14, 16-Pentamethyl-1,4,8-Triazacyclo Heptadecane-3,7,17-Trione (39)</h5><div class="NLM_p last">To a solution of <b>39a</b> (70 mg, 0.05 mmol) in THF (3 mL) solution of LiOH·H<sub>2</sub>O (11 mg, 0.27 mmol) in water (0.3 mL) was added. Reaction mixture was stirred at room temperature for 24 h. Reaction mixture was diluted with water (5 mL) and neutralised with sat. KH<sub>2</sub>PO<sub>4</sub>. A white precipitate was formed that was filtered, washed with water, and dried to yield the title compound (35 mg, yield 53%, purity 96.6%). ES<sup>+</sup><i>m</i>/<i>z</i> 1207.49 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.02 (s, 1H), 9.99 (s, 1H), 8.64–8.54 (m, 2H), 8.11 (br d, <i>J</i> = 8.9 Hz, 1H), 7.93–7.87 (m, 1H), 7.81 (d, <i>J</i> = 7.9 Hz, 3H), 7.47–7.37 (m, 5H), 7.31–7.26 (m, 3H), 7.26–7.15 (m, 8H), 7.14–7.09 (m, 3H), 4.84–4.74 (m, 3H), 4.64 (br s, 1H), 4.45 (br d, <i>J</i> = 7.3 Hz, 2H), 4.38 (br d, <i>J</i> = 6.7 Hz, 1H), 4.3 (br d, <i>J</i> = 7.3 Hz, 2H), 4.23 (br d, <i>J</i> = 7.6 Hz, 2H), 4.13 (br s, 2H), 4.10–3.95 (m, 5H), 3.91 (br s, 1H), 3.83 (br d, <i>J</i> = 7.9 Hz, 2H), 3.76–3.64 (m, 5H), 3.62–3.49 (m, 13H), 3.24–2.84 (m, 24H), 2.8 (s, 4H), 2.74–2.65 (m, 2H), 2.64–2.54 (m, 5H), 2.42–2.33 (m, 2H), 2.26 (s, 11H), 2.15–2.02 (m, 2H), 2.02–1.94 (m, 2H), 1.82–1.73 (m, 1H), 1.68–1.59 (m, 2H), 1.52–1.44 (m, 2H), 1.45–1.35 (m, 1H), 1.24 (br s, 4H), 1.22–1.06 (m, 30H), 0.99 (br d, <i>J</i> = 7 Hz, 5H), 0.96–0.87 (m, 5H), 0.69 (br d, <i>J</i> = 6.4 Hz, 2H), 0.63 (br d, <i>J</i> = 6.4 Hz, 2H), 0.58 (br d, <i>J</i> = 6.7 Hz, 3H), 0.51 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Preparation of Compound <b>47</b></h4><div id="sec4_1_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> (2<i>S</i>)-2-(Benzylsulfonylamino)-3-(2-chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propenamide (<b>47</b>)</h5><div class="NLM_p last">To a solution of <b>E-1b</b> (85 mg, 0.08 mmol) in dry DCM (2 mL), TEA (0.017 mL, 0.14 mmol) and phenylmethanesulfonyl chloride (16 mg, 0.08 mmol) were added and stirred at RT for 1 h. The reaction mixture was evaporated to obtain the crude product. The crude product was purified by prep HPLC purification to get the title compound (10 mg, yield 21%, 93.86%). ES<sup>+</sup><i>m</i>/<i>z</i> 1200.31 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.99 (s, 1H), 8.1 (br d, <i>J</i> = 8.9 Hz, 1H), 7.96 (br d, <i>J</i> = 8.2 Hz, 1H), 7.84 (br s, 2H), 7.46–7.42 (m, 6H), 7.42–7.38 (m, 8H), 7.37–7.32 (m, 3H), 7.32–7.28 (m, 6H), 7.28–7.21 (m, 7H), 7.12 (d, <i>J</i> = 8.2 Hz, 3H), 4.83 (br d, <i>J</i> = 4.3 Hz, 2H), 4.72 (s, 1H), 4.70–4.67 (m, 2H), 4.65 (s, 1H), 4.63–4.61 (m, 1H), 4.45 (br d, <i>J</i> = 7 Hz, 1H), 4.37 (br dd, <i>J</i> = 10.1, 6.4 Hz, 2H), 4.32–4.28 (m, 3H), 4.20–4.15 (m, 2H), 4.15–4.09 (m, 4H), 4.06–3.99 (m, 2H), 3.92 (s, 1H), 3.86 (br d, <i>J</i> = 7.9 Hz, 1H), 3.66 (s, 1H), 3.59–3.5 (m, 2H), 3.44 (br d, <i>J</i> = 6.4 Hz, 2H), 3.42–3.39 (m, 1H), 3.31–3.26 (m, 1H), 3.23–3.21 (m, 4H), 3.21–3.19 (m, 1H), 3.15–3.03 (m, 6H), 3 (br d, <i>J</i> = 11.6 Hz, 1H), 2.9 (s, 1H), 2.8 (s, 4H), 2.72 (br d, <i>J</i> = 7.3 Hz, 1H), 2.61–2.54 (m, 4H), 2.45–2.37 (m, 2H), 2.33–2.28 (m, 2H), 2.28–2.25 (m, 9H), 2.15–2.08 (m, 1H), 2.01–1.89 (m, 2H), 1.82–1.74 (m, 1H), 1.72–1.65 (m, 1H), 1.65–1.57 (m, 2H), 1.36–1.28 (m, 2H), 1.27–1.23 (m, 3H), 1.21 (br d, <i>J</i> = 5.8 Hz, 6H), 1.19 (br d, <i>J</i> = 3.1 Hz, 3H), 1.18–1.14 (m, 7H), 1.11–1.08 (m, 2H), 1.07–1.02 (m, 7H), 1.01–0.97 (m, 8H), 0.94–0.88 (m, 4H), 0.65 (d, <i>J</i> = 6.4 Hz, 3H), 0.52 (br d, <i>J</i> = 6.4 Hz, 5H), 0.49 (br d, <i>J</i> = 6.7 Hz, 4H).</div></div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> General Procedure 8 for Compounds <b>48</b>–<b>55</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h4><div class="NLM_p">To a solution of <b>E-1b</b> (1 eq) in dry DCM (2 mL), corresponding isocyanate (1 eq) was added and stirred at room temperature for 1 h. The reaction mixture was evaporated to obtain the crude product. The crude product was purified by prep HPLC purification (extended A, high pH) or by column chromatography using DCM:MeOH:NH<sub>4</sub>OH = 10:1:0.1 as an eluent to get the title compound.</div><div id="sec4_1_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-(phenylcarbamoylamino)propanamide (<b>48</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from phenyl isocyanate (0.0073 mL, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (71 mg, yield 91%, purity 98.5%) was obtained by column chromatography purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1164.58 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.11 (m, 1H), 10.07 (s, 1H), 8.7 (s, 3H), 8.11 (br d, <i>J</i> = 8.2 Hz, 2H), 7.9 (br d, <i>J</i> = 8.9 Hz, 2H), 7.47–7.37 (m, 5H), 7.36–7.19 (m, 13H), 7.16–7.08 (m, 3H), 6.9 (t, <i>J</i> = 7.3 Hz, 2H), 6.51 (d, <i>J</i> = 8.5 Hz, 2H), 4.84–4.69 (m, 3H), 4.63 (br d, <i>J</i> = 4.6 Hz, 2H), 4.45 (d, <i>J</i> = 7.3 Hz, 2H), 4.37 (d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.5 Hz, 2H), 4.14 (br s, 2H), 4.11–3.95 (m, 6H), 3.89 (s, 2H), 3.83 (br d, <i>J</i> = 7.6 Hz, 2H), 3.7 (s, 3H), 3.69–3.65 (m, 1H), 3.63–3.56 (m, 1H), 3.51 (br d, <i>J</i> = 6.7 Hz, 2H), 3.45 (br d, <i>J</i> = 5.5 Hz, 2H), 3.26–3.03 (m, 12H), 3.02–2.95 (m, 2H), 2.94–2.85 (m, 5H), 2.8 (s, 6H), 2.72–2.53 (m, 5H), 2.49–2.46 (m, 1H), 2.44–2.33 (m, 3H), 2.24 (s, 10H), 2.1 (br s, 2H), 2.03–1.9 (m, 3H), 1.76 (dq, <i>J</i> = 13.7, 6.7 Hz, 2H), 1.61 (br d, <i>J</i> = 7 Hz, 2H), 1.55–1.37 (m, 3H), 1.31–1.22 (m, 2H), 1.21–1.06 (m, 26H), 0.99 (br d, <i>J</i> = 7.3 Hz, 7H), 0.96–0.87 (m, 7H), 0.68 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.5 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> (2<i>S</i>)-2-(Benzylcarbamoylamino)-3-(2-chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propanamide (<b>49</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from benzyl isocyanate (9 mg, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (62 mg, yield 78%, purity 99.14%) was obtained by column chromatography purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1178.58 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (s, 1H), 9.97 (s, 1H), 8.1 (br d, <i>J</i> = 8.5 Hz, 2H), 7.9 (br d, <i>J</i> = 8.9 Hz, 2H), 7.45 (d, <i>J</i> = 8.2 Hz, 2H), 7.41 (d, <i>J</i> = 8.5 Hz, 2H), 7.39 (dd, <i>J</i> = 7.0, 2.1 Hz, 2H), 7.34–7.29 (m, 3H), 7.27 (d, <i>J</i> = 7.3 Hz, 4H), 7.25–7.19 (m, 6H), 7.17 (br d, <i>J</i> = 7.3 Hz, 3H), 7.13–7.06 (m, 4H), 6.58 (dt, <i>J</i> = 5.9, 3 Hz, 2H), 6.35 (br d, <i>J</i> = 8.5 Hz, 2H), 6.34 (br d, <i>J</i> = 8.5 Hz, 2H), 4.79 (br d, <i>J</i> = 4.6 Hz, 2H), 4.72–4.65 (m, 2H), 4.62 (br d, <i>J</i> = 4.3 Hz, 2H), 4.44 (br d, <i>J</i> = 7.3 Hz, 2H), 4.36 (br d, <i>J</i> = 7.3 Hz, 2H), 4.28 (d, <i>J</i> = 7.9 Hz, 2H), 4.28–4.24 (m, 2H), 4.22 (d, <i>J</i> = 7.9 Hz, 2H), 4.18 (dd, <i>J</i> = 9.0, 6.3 Hz, 4H), 4.14 (br d, <i>J</i> = 5.8 Hz, 2H), 4.08–3.99 (m, 4H), 3.88 (s, 1H), 3.83 (br d, <i>J</i> = 7.9 Hz, 1H), 3.69 (s, 1H), 3.66 (br dq, <i>J</i> = 10.5, 6.3 Hz, 1H), 3.62–3.55 (m, 1H), 3.51 (br d, <i>J</i> = 7 Hz, 2H), 3.45 (br d, <i>J</i> = 5.2 Hz, 2H), 3.24 (br d, <i>J</i> = 8.9 Hz, 2H), 3.2 (s, 4H), 3.18 (s, 2H), 3.14–2.99 (m, 8H), 2.99–2.94 (m, 2H), 2.9 (br t, <i>J</i> = 8.9 Hz, 2H), 2.88 (s, 2H), 2.79 (s, 3H), 2.63–2.54 (m, 4H), 2.46 (br t, <i>J</i> = 10.5 Hz, 2H), 2.42–2.32 (m, 2H), 2.26 (br d, <i>J</i> = 15 Hz, 2H), 2.22 (s, 10H), 2.15–2.03 (m, 2H), 2.02–1.91 (m, 2H), 1.76 (dq, <i>J</i> = 13.7, 6.7 Hz, 1H), 1.6 (br d, <i>J</i> = 7.3 Hz, 2H), 1.48 (br dd, <i>J</i> = 14.8, 4.7 Hz, 1H), 1.45–1.39 (m, 1H), 1.38 (br s, 1H), 1.29–1.21 (m, 2H), 1.19 (s, 2H), 1.17–1.08 (m, 20H), 1.08 (br d, <i>J</i> = 6.1 Hz, 4H), 0.98 (br d, <i>J</i> = 7 Hz, 8H), 0.92 (br d, <i>J</i> = 6.4 Hz, 3H), 0.9 (br d, <i>J</i> = 6.4 Hz, 3H), 0.68 (br d, <i>J</i> = 6.7 Hz, 3H), 0.62 (br d, <i>J</i> = 6.7 Hz, 3H), 0.57 (br d, <i>J</i> = 6.7 Hz, 3H), 0.5 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-fluorophenyl)carbamoylamino]propanamide (<b>50</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from 2-fluorophenyl isocyanate (0.0128 mL, 0.11 mmol) and <b>E-1b</b> (120 mg, 0.11 mmol), the title product (29 mg, yield 23%, purity 98.1%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1183.68 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.11 (s, 1H), 10.08 (s, 1H), 8.56 (s, 3H), 8.11 (br d, <i>J</i> = 8.5 Hz, 2H), 8.06 (td, <i>J</i> = 8.3, 1.4 Hz, 2H), 7.9 (br d, <i>J</i> = 10.1 Hz, 2H), 7.48–7.37 (m, 5H), 7.36–7.3 (m, 2H), 7.29–7.21 (m, 5H), 7.20–7.1 (m, 7H), 7.06 (t, <i>J</i> = 7.6 Hz, 2H), 6.95–6.9 (m, 2H), 4.83–4.7 (m, 3H), 4.63 (br d, <i>J</i> = 4.9 Hz, 2H), 4.45 (d, <i>J</i> = 7.3 Hz, 2H), 4.37 (br d, <i>J</i> = 7 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 2H), 4.13 (br s, 1H), 4.10–3.95 (m, 5H), 3.89 (s, 1H), 3.83 (br d, <i>J</i> = 8.2 Hz, 2H), 3.7 (s, 1H), 3.67 (br dd, <i>J</i> = 10.4, 6.1 Hz, 2H), 3.62–3.56 (m, 1H), 3.51 (br d, <i>J</i> = 7 Hz, 2H), 3.48–3.43 (m, 2H), 3.28–3.23 (m, 1H), 3.21 (s, 3H), 3.18 (s, 3H), 3.17–3.05 (m, 5H), 3.04–2.95 (m, 2H), 2.94–2.85 (m, 4H), 2.82–2.74 (m, 4H), 2.73–2.66 (m, 1H), 2.63–2.56 (m, 3H), 2.53–2.52 (m, 1H), 2.48–2.39 (m, 4H), 2.3–2.16 (m, 13H), 2.09 (s, 2H), 2.03–1.89 (m, 3H), 1.76 (dq, <i>J</i> = 13.7, 6.7 Hz, 2H), 1.6 (br d, <i>J</i> = 7.3 Hz, 2H), 1.55–1.37 (m, 3H), 1.32–1.22 (m, 3H), 1.21–1.06 (m, 28H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.96–0.86 (m, 6H), 0.68 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.5 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(3-fluorophenyl)carbamoylamino]propanamide (<b>51</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from 3-fluoro isocyanate (14 mg, 0.12 mmol) and <b>E1-b</b> (127 mg, 0.12 mmol), the title product (50 mg, yield 35%, purity 99.1%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1182.52 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.11 (s, 1H), 10.08 (s, 1H), 8.97–8.93 (m, 2H), 8.11 (br d, <i>J</i> = 8.9 Hz, 1H), 7.9 (br d, <i>J</i> = 9.5 Hz, 1H), 7.45 (br d, <i>J</i> = 8.2 Hz, 1H), 7.44–7.38 (m, 6H), 7.35–7.3 (m, 2H), 7.30–7.21 (m, 8H), 7.15–7.07 (m, 4H), 7.01–6.96 (m, 2H), 6.71 (td, <i>J</i> = 8.5, 2.4 Hz, 2H), 6.58 (br d, <i>J</i> = 8.5 Hz, 2H), 4.79 (br d, <i>J</i> = 4.6 Hz, 1H), 4.74 (q, <i>J</i> = 7.2 Hz, 2H), 4.63 (br d, <i>J</i> = 4 Hz, 1H), 4.45 (d, <i>J</i> = 7 Hz, 1H), 4.37 (br d, <i>J</i> = 7 Hz, 1H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.25–4.22 (m, 1H), 4.18–4.09 (m, 3H), 4.08–4 (m, 5H), 3.89 (s, 1H), 3.83 (br d, <i>J</i> = 7.9 Hz, 1H), 3.73–3.63 (m, 3H), 3.51 (br d, <i>J</i> = 7 Hz, 2H), 3.45 (br d, <i>J</i> = 4.9 Hz, 1H), 3.21 (s, 4H), 3.18 (s, 3H), 3.17–3.1 (m, 4H), 3.10–2.96 (m, 5H), 2.93–2.85 (m, 5H), 2.82–2.76 (m, 5H), 2.75–2.67 (m, 1H), 2.65–2.55 (m, 5H), 2.40–2.35 (m, 3H), 2.23 (s, 14H), 2.02–1.94 (m, 3H), 1.80–1.72 (m, 1H), 1.61 (br d, <i>J</i> = 7.9 Hz, 2H), 1.52–1.36 (m, 3H), 1.27–1.22 (m, 2H), 1.2 (s, 2H), 1.18–1.15 (m, 7H), 1.15–1.11 (m, 16H), 1.09 (br d, <i>J</i> = 5.5 Hz, 7H), 0.99 (br d, <i>J</i> = 7 Hz, 7H), 0.95–0.87 (m, 7H), 0.68 (br d, <i>J</i> = 6.7 Hz, 2H), 0.62 (br d, <i>J</i> = 6.7 Hz, 2H), 0.57 (br d, <i>J</i> = 6.4 Hz, 4H), 0.5 (br d, <i>J</i> = 6.4 Hz, 4H).</div></div><div id="sec4_1_13_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(4-fluorophenyl)carbamoylamino]propanamide (<b>52</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from 4-fluorophenyl isocyanate (0.0067 mL, 0.059 mmol) and <b>E-1b</b> (62.5 mg, 0.059 mmol), the title product (6.36 mg, yield 9%, purity 94%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1182.6 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.09 (s, 1H), 10.05 (s, 1H), 8.78–8.73 (m, 2H), 8.73–8.72 (m, 1H), 8.12 (d, <i>J</i> = 8.9 Hz, 2H), 7.9 (d, <i>J</i> = 8.5 Hz, 2H), 7.46–7.37 (m, 9H), 7.36–7.29 (m, 9H), 7.28–7.2 (m, 8H), 7.15–7.09 (m, 7H), 7.04 (s, 2H), 7.01 (br d, <i>J</i> = 1.8 Hz, 2H), 6.47 (d, <i>J</i> = 8.2 Hz, 2H), 4.76 (d, <i>J</i> = 4.6 Hz, 2H), 4.73–4.68 (m, 2H), 4.58 (s, 1H), 4.44 (d, <i>J</i> = 7.3 Hz, 2H), 4.40–4.33 (m, 2H), 4.3 (d, <i>J</i> = 8.5 Hz, 2H), 4.06–3.93 (m, 9H), 3.75 (br s, 2H), 3.19 (s, 4H), 3.17 (s, 2H), 3.15–3.05 (m, 8H), 2.93–2.87 (m, 4H), 2.8 (s, 3H), 2.62–2.56 (m, 4H), 2.25–2.21 (m, 12H), 2.01–1.95 (m, 3H), 1.80–1.75 (m, 2H), 1.64–1.58 (m, 2H), 1.52–1.46 (m, 1H), 1.26–1.23 (m, 4H), 1.21–1.15 (m, 12H), 1.15–1.12 (m, 12H), 1.12–1.07 (m, 12H), 0.99 (br d, <i>J</i> = 7.3 Hz, 5H), 0.94–0.9 (m, 5H), 0.69 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.51 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_13_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-fluorophenyl)methylcarbamoylamino]propanamide (<b>53</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from 2-fluorobenzyl isocyanate (0.0087 mL, 0.078 mmol) and <b>E-1b</b> (85 mg, 0.078 mmol), the title product (15.89 mg, yield 17%, purity 99.5%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1197.76 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (s, 1H), 9.97 (s, 1H), 8.11 (br d, <i>J</i> = 8.4 Hz, 2H), 7.91 (br d, <i>J</i> = 1.7 Hz, 2H), 7.45 (d, <i>J</i> = 8.4 Hz, 2H), 7.43–7.38 (m, 4H), 7.34–7.26 (m, 5H), 7.26–7.19 (m, 6H), 7.16–7.1 (m, 7H), 6.62–6.59 (m, 2H), 6.41 (d, <i>J</i> = 8.9 Hz, 2H), 4.8 (br d, <i>J</i> = 4.4 Hz, 1H), 4.70–4.65 (m, 2H), 4.64 (br d, <i>J</i> = 2 Hz, 1H), 4.45 (d, <i>J</i> = 7.2 Hz, 1H), 4.37 (d, <i>J</i> = 7.2 Hz, 1H), 4.29 (d, <i>J</i> = 7.7 Hz, 2H), 4.27–4.25 (m, 1H), 4.25–4.21 (m, 4H), 4.2 (br d, <i>J</i> = 6.2 Hz, 1H), 4.13 (br s, 2H), 4.10–4.01 (m, 5H), 3.98 (br d, <i>J</i> = 10.5 Hz, 1H), 3.89 (br d, <i>J</i> = 0.7 Hz, 1H), 3.84 (br d, <i>J</i> = 8.1 Hz, 1H), 3.7 (s, 1H), 3.69–3.66 (m, 1H), 3.59 (br d, <i>J</i> = 12.5 Hz, 1H), 3.52 (br d, <i>J</i> = 7 Hz, 2H), 3.46 (br d, <i>J</i> = 5.1 Hz, 1H), 3.21 (s, 4H), 3.19 (s, 2H), 3.13–3.01 (m, 7H), 2.98 (br d, <i>J</i> = 10.3 Hz, 1H), 2.93–2.86 (m, 4H), 2.82–2.76 (m, 4H), 2.73–2.68 (m, 1H), 2.64–2.56 (m, 5H), 2.50–2.35 (m, 6H), 2.27 (br d, <i>J</i> = 14.7 Hz, 1H), 2.25–2.22 (m, 12H), 2.11 (br s, 1H), 2.01–1.94 (m, 2H), 1.80–1.74 (m, 1H), 1.61 (br d, <i>J</i> = 7.2 Hz, 2H), 1.51–1.39 (m, 4H), 1.25 (br d, <i>J</i> = 8.1 Hz, 2H), 1.2 (s, 3H), 1.18–1.15 (m, 8H), 1.15–1.12 (m, 10H), 1.12–1.1 (m, 5H), 1.09 (d, <i>J</i> = 6.1 Hz, 6H), 0.99 (br d, <i>J</i> = 7.3 Hz, 6H), 0.96–0.88 (m, 6H), 0.69 (d, <i>J</i> = 6.8 Hz, 3H), 0.63 (br d, <i>J</i> = 6.8 Hz, 3H), 0.58 (d, <i>J</i> = 6.8 Hz, 3H), 0.51 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_1_13_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(3-fluorophenyl)methylcarbamoylamino]propanamide (<b>54</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from 3-fluorobenzyl isocyanate (0.0076 mL, 0.059 mmol) and <b>E-1b</b> (62.5 mg, 0.059 mmol), the title product (13.13 mg, yield 18%, purity 95%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1196.86 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (br s, 1H), 9.98 (s, 1H), 8.27–8.26 (m, 2H), 8.13 (br d, <i>J</i> = 10.7 Hz, 2H), 7.93 (br s, 4H), 7.45–7.37 (m, 5H), 7.34 (br d, <i>J</i> = 5.5 Hz, 2H), 7.27–7.19 (m, 5H), 7.14–7.09 (m, 7H), 6.97 (s, 1H), 6.71–6.66 (m, 3H), 6.37 (br s, 1H), 4.76 (d, <i>J</i> = 4.6 Hz, 2H), 4.68 (br s, 2H), 4.47–4.42 (m, 3H), 4.36 (s, 3H), 4.3 (br s, 3H), 4.25–4.22 (m, 4H), 4.19 (dd, <i>J</i> = 9.3, 6.3 Hz, 2H), 4.16–4.14 (m, 5H), 4.11–4.07 (m, 5H), 4.06 (br s, 3H), 3.93 (br s, 3H), 3.75 (br s, 3H), 3.21–3.16 (m, 3H), 3.13–3 (m, 3H), 2.99–2.84 (m, 4H), 2.77 (s, 4H), 2.62–2.55 (m, 5H), 2.48–2.41 (m, 4H), 2.30–2.23 (m, 12H), 2.21 (s, 3H), 2.11–2.05 (m, 2H), 2.02–1.92 (m, 3H), 1.77–1.69 (m, 2H), 1.65–1.57 (m, 3H), 1.51–1.35 (m, 3H), 1.24 (br s, 3H), 1.19–1.05 (m, 29H), 0.97 (br d, <i>J</i> = 7.3 Hz, 5H), 0.93–0.88 (m, 6H), 0.69–0.65 (m, 3H), 0.61 (d, <i>J</i> = 6.4 Hz, 3H), 0.57 (d, <i>J</i> = 6.7 Hz, 3H), 0.47 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_13_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(4-fluorophenyl)methylcarbamoylamino]propenamide (<b>55</b>)</h5><div class="NLM_p last">Following general procedure 8: starting from 3-fluorobenzyl isocyanate (0.0087 mL, 0.078 mmol) and <b>E-1b</b> (85 mg, 0.078 mmol), the title product (25.58 mg, yield 26%, purity 94.4%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1196.8 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.01 (s, 1H), 9.98 (s, 1H), 8.11 (br d, <i>J</i> = 8.9 Hz, 2H), 7.9 (br d, <i>J</i> = 9.5 Hz, 2H), 7.46 (d, <i>J</i> = 8.5 Hz, 2H), 7.44–7.4 (m, 3H), 7.39 (d, <i>J</i> = 2.4 Hz, 2H), 7.34–7.3 (m, 2H), 7.28 (br d, <i>J</i> = 8.5 Hz, 2H), 7.27–7.18 (m, 8H), 7.16–7.08 (m, 7H), 6.62 (br t, <i>J</i> = 6 Hz, 2H), 6.37 (d, <i>J</i> = 8.2 Hz, 2H), 4.8 (br d, <i>J</i> = 4.6 Hz, 2H), 4.72–4.65 (m, 2H), 4.63 (br d, <i>J</i> = 4.9 Hz, 2H), 4.45 (d, <i>J</i> = 7 Hz, 2H), 4.37 (d, <i>J</i> = 7.3 Hz, 2H), 4.29 (d, <i>J</i> = 7.6 Hz, 2H), 4.23 (d, <i>J</i> = 8.2 Hz, 2H), 4.2 (br d, <i>J</i> = 6.1 Hz, 2H), 4.19–4.11 (m, 5H), 4.08–4 (m, 4H), 4–3.94 (m, 1H), 3.89 (s, 1H), 3.84 (br d, <i>J</i> = 7.6 Hz, 1H), 3.7 (s, 1H), 3.67 (br dd, <i>J</i> = 9.8, 6.7 Hz, 1H), 3.62–3.57 (m, 1H), 3.52 (br d, <i>J</i> = 6.7 Hz, 2H), 3.48–3.43 (m, 1H), 3.21 (s, 4H), 3.19 (s, 2H), 3.12 (br d, <i>J</i> = 6.4 Hz, 1H), 3.10–3.07 (m, 2H), 3.06–3.01 (m, 3H), 3–2.95 (m, 1H), 2.94–2.85 (m, 4H), 2.82–2.74 (m, 4H), 2.73–2.67 (m, 1H), 2.66–2.63 (m, 1H), 2.62–2.56 (m, 3H), 2.48–2.44 (m, 2H), 2.44–2.39 (m, 1H), 2.38–2.34 (m, 2H), 2.29–2.17 (m, 13H), 2.09 (br s, 2H), 1.98 (dt, <i>J</i> = 13.4, 6.7 Hz, 3H), 1.77 (dq, <i>J</i> = 13.8, 6.8 Hz, 1H), 1.6 (br d, <i>J</i> = 7 Hz, 2H), 1.55–1.37 (m, 3H), 1.32–1.22 (m, 3H), 1.21–1.06 (m, 30H), 0.99 (br d, <i>J</i> = 7.3 Hz, 6H), 0.96–0.88 (m, 6H), 0.69 (d, <i>J</i> = 6.7 Hz, 3H), 0.63 (br d, <i>J</i> = 6.7 Hz, 3H), 0.58 (d, <i>J</i> = 6.7 Hz, 3H), 0.51 (d, <i>J</i> = 6.7 Hz, 3H).</div></div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> General Procedure 9 for Preparation of Compounds <b>56</b>–<b>61</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h4><div class="NLM_p">Corresponding acid (1.1 eq) in dry DCM was activated with HATU (1.2 eq), HOAt (1.2 eq), and TMP (2.5 eq) for 15 min. Then, a solution of <b>E-1b</b> to <b>E-1g</b> (1 eq) in DCM was added and the reaction mixture was stirred for 1 h at room temperature. Sat. NaHCO<sub>3</sub> was added, layers were separated, and the organic layer was dried over anhydrous sodium sulfate, filtered, and evaporated to dryness to get the crude material. The crude material was purified by column chromatography using DCM:MeOH:NH<sub>4</sub>OH = 10:1:0.1 as an eluent or by prep HPLC purification (extended A, high pH) or by column chromatography using DCM:MeOH:NH<sub>4</sub>OH = 10:1:0.1 as an eluent to get the title compound.</div><div id="sec4_1_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> (2<i>S</i>)-2-Acetamido-3-(2-chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]propanamide (<b>56</b>)</h5><div class="NLM_p last">Following general procedure 9: starting from acetic acid (0.0038 mL, 0.067 mmol) and <b>E-1b</b> (70 mg, 0.067 mmol), the title product (31 mg, yield 43%, purity 98.2%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1087.63 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.93 (s, 1H), 9.89 (s, 1H), 8.28 (br d, <i>J</i> = 8.2 Hz, 1H), 8.27 (br d, <i>J</i> = 8.2 Hz, 1H), 8.09 (br d, <i>J</i> = 8.5 Hz, 1H), 7.89 (br d, <i>J</i> = 8.9 Hz, 1H), 7.44 (br d, <i>J</i> = 8.5 Hz, 2H), 7.42–7.4 (m, 2H), 7.40–7.38 (m, 2H), 7.33–7.29 (m, 2H), 7.27 (br d, <i>J</i> = 8.5 Hz, 2H), 7.24–7.22 (m, 2H), 7.21 (s, 2H), 7.1 (br d, <i>J</i> = 8.5 Hz, 2H), 7.10–7.07 (m, 2H), 4.78 (br d, <i>J</i> = 4.9 Hz, 1H), 4.76–4.7 (m, 2H), 4.62 (br d, <i>J</i> = 4.3 Hz, 1H), 4.44 (d, <i>J</i> = 7.3 Hz, 1H), 4.36 (br d, <i>J</i> = 7.3 Hz, 1H), 4.28 (d, <i>J</i> = 7.6 Hz, 1H), 4.26–4.24 (m, 1H), 4.21 (d, <i>J</i> = 8.2 Hz, 1H), 4.16–4.11 (m, 1H), 4.10–4.08 (m, 1H), 4.08–4.03 (m, 2H), 4.03–3.99 (m, 1H), 3.96 (br t, <i>J</i> = 7.5 Hz, 1H), 3.82 (br d, <i>J</i> = 7.6 Hz, 1H), 3.66 (br dd, <i>J</i> = 10.2, 6.6 Hz, 1H), 3.62–3.57 (m, 1H), 3.5 (br d, <i>J</i> = 6.7 Hz, 1H), 3.45 (br d, <i>J</i> = 5.5 Hz, 1H), 3.26–3.22 (m, 2H), 3.2 (s, 3H), 3.17 (s, 3H), 3.13 (br dd, <i>J</i> = 14.2, 6.3 Hz, 2H), 3.10–3.04 (m, 2H), 3.04–3.02 (m, 1H), 2.98 (br dd, <i>J</i> = 14.2, 8.4 Hz, 2H), 2.91 (br d, <i>J</i> = 8.9 Hz, 2H), 2.88 (s, 3H), 2.78 (s, 3H), 2.72–2.54 (m, 6H), 2.47–2.42 (m, 2H), 2.41–2.32 (m, 4H), 2.29–2.23 (m, 2H), 2.23 (s, 12H), 2.13–2.1 (m, 1H), 2.09–2.06 (m, 1H), 2.02–1.98 (m, 2H), 1.97–1.91 (m, 1H), 1.81 (s, 6H), 1.78–1.72 (m, 1H), 1.64–1.56 (m, 2H), 1.51–1.43 (m, 1H), 1.42–1.35 (m, 1H), 1.26–1.21 (m, 4H), 1.19 (s, 3H), 1.15 (s, 3H), 1.17–1.14 (m, 3H), 1.13–1.1 (m, 18H), 1.10–1.09 (m, 2H), 1.08 (br d, <i>J</i> = 5.8 Hz, 3H), 0.98 (br d, <i>J</i> = 7.3 Hz, 6H), 0.92 (br d, <i>J</i> = 6.4 Hz, 3H), 0.9 (br d, <i>J</i> = 6.7 Hz, 3H), 0.68 (br d, <i>J</i> = 6.7 Hz, 3H), 0.62 (br d, <i>J</i> = 6.7 Hz, 3H), 0.57 (br d, <i>J</i> = 6.7 Hz, 3H), 0.5 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> (2<i>S</i>)-3-(2,4-Dichlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>57</b>)</h5><div class="NLM_p last">Following general procedure 9: starting from phenylacetic acid (10.71 mg, 0.078 mmol) and <b>E-1c</b> (85 mg, 0.078 mmol), the title product (21 mg, yield 22%, purity 98%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1197.8 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.99 (s, 1H), 9.96 (s, 1H), 8.50–8.45 (m, 3H), 8.11 (br d, <i>J</i> = 8.9 Hz, 2H), 7.93–7.87 (m, 1H), 7.5 (d, <i>J</i> = 2.1 Hz, 2H), 7.53–7.49 (m, 1H), 7.45–7.35 (m, 4H), 7.29–7.17 (m, 11H), 7.15–7.08 (m, 8H), 4.78–4.72 (m, 3H), 4.61 (d, <i>J</i> = 4.3 Hz, 2H), 4.44 (d, <i>J</i> = 7.3 Hz, 2H), 4.39–4.35 (m, 2H), 4.3 (d, <i>J</i> = 8.2 Hz, 2H), 4.28–4.22 (m, 2H), 4.18–4.1 (m, 4H), 4.08–3.92 (m, 6H), 3.75 (s, 2H), 3.77 (br s, 2H), 3.45–3.39 (m, 12H), 3.2 (s, 4H), 3.17 (s, 1H), 3.16–3.1 (m, 4H), 3.10–3.04 (m, 2H), 3 (br dd, <i>J</i> = 14.0, 8.5 Hz, 3H), 2.94–2.84 (m, 5H), 2.77 (s, 3H), 2.62–2.56 (m, 4H), 2.24 (s, 11H), 2–1.93 (m, 2H), 1.76–1.68 (m, 1H), 1.65–1.58 (m, 2H), 1.52–1.36 (m, 2H), 1.24–1.21 (m, 3H), 1.18 (s, 2H), 1.17–1.06 (m, 34H), 0.98 (br d, <i>J</i> = 7.3 Hz, 5H), 0.89 (d, <i>J</i> = 6.7 Hz, 3H), 0.66 (d, <i>J</i> = 6.7 Hz, 3H), 0.61 (br d, <i>J</i> = 6.7 Hz, 3H), 0.56 (d, <i>J</i> = 6.7 Hz, 3H), 0.46 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_14_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> (2<i>S</i>)-3-(2,6-Dichlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>58</b>)</h5><div class="NLM_p last">Following general procedure 9: starting from phenylacetic acid (10.71 mg, 0.078 mmol) and <b>E-1d</b> (85 mg, 0.078 mmol), the title product (28 mg, yield 30%, purity 97.55%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1197.7 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10 (s, 1H), 9.95 (s, 1H), 8.50–8.44 (m, 3H), 8.13–8.08 (m, 1H), 7.93 (br s, 1H), 7.53–7.49 (m, 3H), 7.44–7.37 (m, 5H), 7.28–7.19 (m, 6H), 7.14–7.1 (m, 12H), 4.71 (br d, <i>J</i> = 0.9 Hz, 3H), 4.62–4.59 (m, 2H), 4.44 (d, <i>J</i> = 7 Hz, 2H), 4.37 (t, <i>J</i> = 6.4 Hz, 2H), 4.09 (s, 2H), 4.07–4.01 (m, 4H), 4.01–3.92 (m, 1H), 3.75 (br s, 3H), 3.48–3.38 (m, 51H), 3.20–3.15 (m, 15H), 3.10–3.05 (m, 3H), 3.03–2.97 (m, 2H), 2.87 (s, 4H), 2.77 (s, 7H), 2.60–2.56 (m, 4H), 2.28–2.24 (m, 2H), 2.21 (s, 24H), 2–1.94 (m, 3H), 1.75–1.7 (m, 1H), 1.64–1.57 (m, 2H), 1.51–1.36 (m, 1H), 1.22 (br d, <i>J</i> = 4.6 Hz, 4H), 1.18 (s, 4H), 1.16–1.06 (m, 62H), 0.97 (br d, <i>J</i> = 7.3 Hz, 9H), 0.93–0.88 (m, 10H), 0.66 (d, <i>J</i> = 6.7 Hz, 3H), 0.61 (d, <i>J</i> = 7 Hz, 3H), 0.56 (d, <i>J</i> = 6.7 Hz, 3H), 0.46 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_14_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> (2<i>S</i>)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-3-(2-fluorophenyl)-2-[(2-phenylacetyl)amino]propanamide (<b>59</b>)</h5><div class="NLM_p last">Following general procedure 9: starting from phenylacetic acid (9.3 mg, 0.068 mmol) and <b>E-1e</b> (66.7 mg, 0.065 mmol), the title product (40 mg, yield 54%, purity 94.49%) was obtained using column chromatography for purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1147.5 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10 (s, 1H), 9.97 (s, 1H), 8.45–8.4 (m, 3H), 8.13 (br d, <i>J</i> = 8.2 Hz, 2H), 7.91 (br d, <i>J</i> = 8.9 Hz, 2H), 7.43–7.37 (m, 3H), 7.31–7.27 (m, 1H), 7.26–7.2 (m, 7H), 7.20–7.16 (m, 2H), 7.14–7.01 (m, 10H), 4.81 (br d, <i>J</i> = 4.3 Hz, 2H), 4.77–4.69 (m, 2H), 4.66 (br d, <i>J</i> = 4 Hz, 2H), 4.47 (br d, <i>J</i> = 7 Hz, 2H), 4.4 (br d, <i>J</i> = 7 Hz, 2H), 4.31–4.19 (m, 2H), 4.1 (br s, 1H), 4.08–3.97 (m, 4H), 3.95–3.9 (m, 1H), 3.84 (br d, <i>J</i> = 8.2 Hz, 2H), 3.71 (br s, 1H), 3.67 (s, 1H), 3.56–3.37 (m, 8H), 3.24–3.15 (m, 7H), 3.06 (br dd, <i>J</i> = 13.9, 6 Hz, 3H), 3.01–2.83 (m, 8H), 2.8 (s, 4H), 2.77–2.66 (m, 2H), 2.63–2.53 (m, 6H), 2.48–2.45 (m, 4H), 2.40–2.31 (m, 4H), 2.31–2.19 (m, 2H), 2.1 (br s, 1H), 1.99 (s, 2H), 1.96 (br d, <i>J</i> = 6.7 Hz, 2H), 1.77 (br dd, <i>J</i> = 13.3, 6.6 Hz, 3H), 1.70–1.63 (m, 1H), 1.56–1.38 (m, 4H), 1.38–1.25 (m, 9H), 1.21–1.08 (m, 30H), 0.98 (br d, <i>J</i> = 7.3 Hz, 6H), 0.95–0.85 (m, 6H), 0.67 (br d, <i>J</i> = 6.4 Hz, 3H), 0.58 (br d, <i>J</i> = 6.4 Hz, 3H), 0.54 (br d, <i>J</i> = 6.7 Hz, 3H), 0.5 (br d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_14_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> (2<i>S</i>)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12-hydroxy-15-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)oxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-3-(<i>o</i>-tolyl)-2-[(2-phenylacetyl)amino]propanamide (<b>60</b>)</h5><div class="NLM_p last">Following general procedure 9: starting from phenylacetic acid (15 mg, 0.112 mmol) and <b>E-1f</b> (110 mg, 0.107 mmol), the title product (32 mg, yield 26%, purity 99.4%) was obtained by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1143.64 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 9.95 (s, 1H), 9.91 (s, 1H), 8.50–8.45 (m, 3H), 8.09 (br d, <i>J</i> = 8.5 Hz, 1H), 7.89 (br d, <i>J</i> = 8.5 Hz, 1H), 7.44–7.36 (m, 3H), 7.29–7.2 (m, 5H), 7.20–6.96 (m, 15H), 4.79 (br d, <i>J</i> = 4.6 Hz, 2H), 4.72–4.66 (m, 2H), 4.62 (br d, <i>J</i> = 4.6 Hz, 2H), 4.44 (d, <i>J</i> = 7 Hz, 2H), 4.36 (d, <i>J</i> = 7.3 Hz, 2H), 4.28 (d, <i>J</i> = 7.6 Hz, 2H), 4.25–4.14 (m, 1H), 4.11 (br s, 2H), 4.08–3.94 (m, 4H), 3.88 (s, 1H), 3.83 (br d, <i>J</i> = 7.6 Hz, 2H), 3.69 (s, 1H), 3.68–3.64 (m, 1H), 3.61–3.55 (m, 1H), 3.53–3.4 (m, 6H), 3.24 (br dd, <i>J</i> = 9.0, 5.6 Hz, 2H), 3.22–3.17 (m, 5H), 3.13–2.94 (m, 6H), 2.92–2.85 (m, 5H), 2.80–2.73 (m, 4H), 2.71–2.53 (m, 5H), 2.42–2.33 (m, 3H), 2.3 (s, 5H), 2.24 (s, 9H), 2.09 (br d, <i>J</i> = 8.5 Hz, 2H), 2.03–1.88 (m, 3H), 1.76 (dq, <i>J</i> = 13.7, 6.8 Hz, 2H), 1.61 (br d, <i>J</i> = 7 Hz, 2H), 1.55–1.36 (m, 3H), 1.28–1.21 (m, 5H), 1.20–1.06 (m, 27H), 0.98 (br d, <i>J</i> = 7.3 Hz, 5H), 0.94–0.83 (m, 8H), 0.67 (d, <i>J</i> = 6.7 Hz, 3H), 0.61 (br d, <i>J</i> = 6.7 Hz, 3H), 0.57 (d, <i>J</i> = 6.7 Hz, 3H), 0.49 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_1_14_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[[4-[[(2<i>S</i>)-3-Phenyl-2-[(2-phenylacetyl)amino]phenyl]methyl]-15-<i>O</i>-(α-<span class="smallcaps smallerCapital">l</span>-cladinosyl)-13-<i>O</i>-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)-12,13,15-trihydroxy-2-isopropyl-8,10,12,14,16-pentamethyl-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>61</b>)</h5><div class="NLM_p last">Following general procedure 9: starting from 2-phenylacetic acid (8.8 mg, 0.065 mmol) and <b>E-1g</b> (65.3 mg, 0.065 mmol), the title product was obtained (60.1 mg, 81.9%, purity 92.47%) as a yellowish solid foam. ES<sup>+</sup><i>m</i>/<i>z</i> 1129.65 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 10.05 (s, 1H), 10.02 (s, 1H), 8.48–8.45 (m, 3H), 8.11 (br d, <i>J</i> = 8.5 Hz, 1H), 7.93–7.88 (m, 1H), 7.48–7.42 (m, 3H), 7.35–7.15 (m, 15H), 7.14–7.1 (m, 6H), 4.8 (br d, <i>J</i> = 4.6 Hz, 1H), 4.68 (td, <i>J</i> = 8.7, 5.5 Hz, 2H), 4.63 (br d, <i>J</i> = 4.3 Hz, 1H), 4.45 (d, <i>J</i> = 7.3 Hz, 1H), 4.37 (d, <i>J</i> = 7 Hz, 1H), 4.29 (d, <i>J</i> = 7.6 Hz, 1H), 4.23 (d, <i>J</i> = 8.5 Hz, 1H), 4.17–4.08 (m, 2H), 4.08–4 (m, 4H), 3.98 (br t, <i>J</i> = 7.2 Hz, 1H), 3.91–3.88 (m, 1H), 3.84 (br d, <i>J</i> = 7.9 Hz, 1H), 3.71 (s, 1H), 3.70–3.64 (m, 1H), 3.62–3.57 (m, 1H), 3.54–3.5 (m, 1H), 3.49–3.44 (m, 2H), 3.43–3.39 (m, 2H), 3.25 (br d, <i>J</i> = 10.1 Hz, 1H), 3.21 (s, 3H), 3.19 (s, 2H), 3.13–2.96 (m, 5H), 2.93–2.84 (m, 5H), 2.8 (s, 3H), 2.78–2.65 (m, 2H), 2.64–2.55 (m, 4H), 2.49–2.35 (m, 4H), 2.28–2.2 (m, 11H), 2.16–2.05 (m, 2H), 2.03–1.94 (m, 2H), 1.82–1.72 (m, 1H), 1.61 (br dd, <i>J</i> = 10.8, 3.2 Hz, 2H), 1.52–1.37 (m, 2H), 1.24 (br s, 2H), 1.20–1.06 (m, 27H), 1 (br d, <i>J</i> = 7.3 Hz, 5H), 0.95–0.86 (m, 6H), 0.71–0.68 (m, 3H), 0.64 (br d, <i>J</i> = 6.7 Hz, 3H), 0.59 (d, <i>J</i> = 6.7 Hz, 3H), 0.51 (d, <i>J</i> = 6.7 Hz, 3H).</div></div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Preparation of Compound <b>62</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Scheme S9</a>)</h4><div id="sec4_1_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[(4-Aminophenyl)methyl]-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12,15-dihydroxy-2-isopropyl-8,10,12,14,16-pentamethyl-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>E-1h</b>)</h5><div class="NLM_p last">Step a: To the solution of <b>E-1a</b> (260 mg, 0.19 mmol) in acetonitrile (6 mL) was added 6 M HCl (11.4 mL, 22.8 mmol). The reaction mixture was stirred at room temperature overnight. The pH was adjusted to 10 and extracted with DCM (3 × 20 mL). Organic layers were combined and dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under vacuum to get compound <b>E-1h</b> (220 mg). ES+ <i>m</i>/<i>z</i> 706.56 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> (2<i>S</i>)-3-(2-Chlorophenyl)-<i>N</i>-[4-[[(2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-13-(β-<span class="smallcaps smallerCapital">d</span>-desosaminyl)oxy-12,15-dihydroxy-2-isopropyl-8,10,12,14,16-pentamethyl-3,7,17-trioxo-1,4,8-triazacycloheptadec-5-yl]methyl]phenyl]-2-[(2-phenylacetyl)amino]propanamide (<b>62</b>)</h5><div class="NLM_p last">Step b: Following general procedure 5: starting from <b>64</b> (54 mg, 0.17 mmol) and <b>E-1h</b> (120 mg, 0.17 mmol), the title product was obtained (28 mg, yield 16%, purity 93.6%) by prep HPLC purification. ES<sup>+</sup><i>m</i>/<i>z</i> 1005.65 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 0.46 (d, 3H, <i>J</i> = 6.72), 0.55 (t, 2H, <i>J</i> = 5.64, 6.42), 0.57 (d, 3H, <i>J</i> = 6.66), 0.69–0.71 (m, 5H), 0.78 (d, 3H, <i>J</i> = 6.9), 0.91 (t, 6H, <i>J</i> = 5.76, 6.36), 1.1 (d, 6H, <i>J</i> = 4.92), 1.14 (t, 9H, <i>J</i> = 5.88, 6.72), 1.2 (d, 3H, <i>J</i> = 6.12), 1.2–1.21 (m, 2H). 1.49–1.52 (m, 1H), 1.69–1.72 (m, 4H), 1.94–1.99 (m, 1H), 2.06 (d, 2H, <i>J</i> = 7.44), 2.16–2.2 (m, 1H), 2.22 (d, 12H, <i>J</i> = 7.92), 2.24–2.25 (m, 1H), 2.42–2.43 (m, 2H), 2.45–2.46 (br. s. 1H), 5.51 (br. s. 1 h), 2.54–2.57 (m, 1H), 2.58–2.61 (m, 1H),2.62–2.64 (m, 1H), 2.65–2.69 (m, 1H), 2.74 (d, 3H, <i>J</i> = 26.46), 2.79 (s, 3H), 2.88–2.9 (m, 2H), 3–3.04 (m, 2H), 3.13–3.19 (m, 5H), 3.4–3.47 (m, 6H), 3.66–3.69 (m, 6H), 3.95 (t, 1H, <i>J</i> = 8.16, 8.34), 4.18–4.3 (m, 3H), 4.31 (d, 1H, <i>J</i> = 7.38), 4.37 (dd, 1H, <i>J</i> = 7.38, 9.06), 4.61 (s, 1H). 4.75–4.78 (m, 3H), 4.87 (d, 1H, <i>J</i> = 4.98), 5.29 (s, 1H), 5.41 (s, 1H), 6.89 (d, 1H, <i>J</i> = 8.7), 7.11–7.14 (m, 8H), 7.16–7.18 (m, 5H), 7.2–7.24 (m, 6H), 7.26–7.29 (t, 2H, <i>J</i> = 7.32, 7.5), 7.36–7.4 (m, 4H), 7.44 (d, 2H, <i>J</i> = 8.46), 7.92 (m, 1H), 8.07 (d, 1H, <i>J</i> = 8.82), 8.44–8.49 (m, 2H), 9.91 (s, 1H), 9.97 (s, 1H).</div></div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Preparation of Compound <b>63</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">Scheme S10</a>)</h4><div id="sec4_1_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> 5,6-<i>O</i>-Isopropylidene-9a-methyl-9-deoxo-9a-aza-9a-homoerythronolide (<b>68</b>)</h5><div class="NLM_p last">Compound <b>67</b><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (2 g, 4.6 mmol) was dissolved in dry DCM (40 mL). <i>rac</i>-Camphorsulfonic acid (1.6 g, 6.9 mmol) and 2,2-dimethoxypropane (1.69 mL, 13.8 mmol) were added. The reaction mixture was stirred at room temperature for 3 h. 2.2-Dimethoxyethane (0.56 mL, 4.6 mmol) was added, and stirring proceeded for 24 h. The reaction was quenched with sat. aq. NaHCO<sub>3</sub> (40 mL). Layers were separated, and the aqueous phase was extracted with DCM (40 mL). Organic extracts were evaporated under reduced pressure to yield the title product as a white foam (2.18 g). The analytical sample was purified by flash chromatography and eluted with mixtures of DCM and methanolic ammonia. The title compound was obtained as a white foam. ES<sup>+</sup><i>m</i>/<i>z</i> 474.42 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 4.88 (dd, <i>J</i> = 10.4 Hz, <i>J</i> = 2.4 Hz, 1H); 4.42 (d, <i>J</i> = 6.7 Hz, 1H); 4.22 (bs, 1H); 4.19 (d, <i>J</i> = 7.2 Hz, 1H); 3.9 (t, <i>J</i> = 5.2 Hz, 1H); 3.71 (d, <i>J</i> = 5.6 Hz, 1H); 3.45 (dd, <i>J</i> = 7.5 Hz, <i>J</i> = 3 Hz, 1H); 2.79 (m, 1H); 2.46 (m, 1H); 2.28 (dd, <i>J</i> = 12.5 Hz, <i>J</i> = 4 Hz, 1H); 2.18 (s, 3H); 2 (dd, <i>J</i> = 12.5 Hz, <i>J</i> = 8.5 Hz, 1H); 1.82 (d, <i>J</i> = 14.6 Hz, 1H); 1.66–1.78 (m, 3H); 1.43 (m, 1H); 1.3 (s, 3H); 1.22 (s, 3H); 1.21 (m, 1H); 1.24 (d, <i>J</i> = 6.9 Hz, 3H); 1.02 (s, 3H); 1.01 (s, 3H); 0.95 (d, <i>J</i> = 6.8 Hz, 6H); 0.79 (t, <i>J</i> = 7.2 Hz, 3H); 0.78 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 174.0; 105.0; 82.4; 80.7; 78.2; 75.3; 74.6; 71.3; 62.7; 62.4, 45.0; 42.3, 37.5; 37.1; 28.7; 28.1; 26.8; 24.5; 22.3; 21.4; 12.7; 11.5; 10.0; 7.35.</div></div><div id="sec4_1_16_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> (1-Ethyl-2-oxo-propyl)-(2<i>R</i>,3<i>S</i>,4<i>S</i>,5<i>R</i>,6<i>R</i>,8<i>R</i>)-3,5,6-trihydroxy-2,4,6,8-tetramethyl-9-(methylamino)nonanoate (<b>69</b>)</h5><div class="NLM_p last">Compound <b>68</b> (2.2 g, 4.18 mmol, 90% purity) was dissolved in chloroform (115 mL). The solution was cooled to 0 °C, and lead(IV) acetate (2.4 g, 5 mmol, 1.2 eq) was added. The reaction mixture was stirred at room temperature for 3 h. The reaction mixture was re-cooled to 0 °C, lead(IV) acetate (200 mg, 0.4 mmol, 0.1 eq) was added, and stirring was continued for another 2.5 h at room temperature. The white precipitate formed in the course of the reaction was filtered off and washed with chloroform. The filtrate was evaporated, and the residue was purified by flash chromatography on silica and eluted with DCM:MeOH:NH<sub>4</sub>OH = 90:9:1.5. The title product was obtained as a brownish viscous syrup (850 mg, yield 49%). ES<sup>+</sup><i>m</i>/<i>z</i> = 416.39 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 4.96 (dd, <i>J</i> = 7.3 Hz, <i>J</i> = 4.7 Hz, 1H); 3.79 (d, <i>J</i> = 9.2 Hz, 1H); 3.61 (d, <i>J</i> = 9.7 Hz, 1H); 2.59 (dq, <i>J</i> = 9.7 Hz, <i>J</i> = 6.7 Hz, 1H); 2.36 (dd, <i>J</i> = 11.8 Hz, <i>J</i> = 5.1 Hz, 1H); 2.31 (dd, <i>J</i> = 11.8 Hz, <i>J</i> = 8.4 Hz, 1H); 2.3 (s, 3H); 2.15 (s, 3H); 1.72–1.87 (m, 4H); 1.69 (dd, <i>J</i> = 14.8 Hz, <i>J</i> = 2.4 Hz, 1H); 1.43 (dd, <i>J</i> = 14.8 Hz, <i>J</i> = 7.5 Hz, 1H); 1.34 (s, 3H); 1.27 (s, 3H); 1.23 (d, <i>J</i> = 6.7 Hz, 1H); 1.06 (s, 3H); 1.01 (d, <i>J</i> = 6.5 Hz, 1H); 0.97 (d, <i>J</i> = 6.6 Hz, 1H); 0.91 (t, <i>J</i> = 7.5 Hz, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ/ppm 204.1; 174.2; 104.5; 87.7; 80.9; 78.9; 69.4; 58.3; 43.4; 43.2; 36.2; 36.1; 28.9; 28.6; 26.7; 26.0; 22.8; 22.7; 20.9; 15.4; 10.2; 9.1.</div></div><div id="sec4_1_16_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> (1-Ethyl-2-oxo-propyl)-(2<i>R</i>,3<i>S</i>,4<i>S</i>,5<i>R</i>,6<i>R</i>,8<i>R</i>,13<i>R</i>)-13-amino-3,5,6-trihydroxy-2,4,6,8,10-5,6-<i>O</i>-isopropylidene-pentamethyl-14-(4-nitrophenyl)-11-oxo-tetradecanoate (<b>70</b>)</h5><div class="NLM_p last">(3<i>R</i>)-3-(Fmoc-amino)-4-(4-nitrophenyl)butanoic acid (785 mg, 1.76 mmol) was suspended in DCM (10 mL). TMP (635 μL, 4 mmol), HOAt (109 mg, 0.87 mmol), and HATU (640 mg, 1.76 mmol) were added, and a clear, yellow reaction mixture was stirred at room temperature for 30 min. A solution of <b>69</b> (640 mg, 1.6 mmol) in DCM was added dropwise, and the solution was stirred for 3 h at room temperature. Piperidine (925 μL, 9.6 mmol) was added, and the reaction mixture was stirred at room temperature overnight. The reaction mixture was extracted with sat. NaHCO<sub>3</sub> (10 mL) and washed with water (3 × 10 mL). Organics were evaporated under reduced pressure, and the residue was purified on silica and eluted with mixtures of DCM and methanolic ammonia. The title product was obtained as a yellowish foam (830 mg) and used in the next synthetic step without further purification. ES<sup>+</sup><i>m</i>/<i>z</i> 622.45 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> (2<i>R</i>,3<i>S</i>,4<i>S</i>,5<i>R</i>,6<i>R</i>,8<i>R</i>,13<i>R</i>,16<i>S</i>)-16-Amino-13-[(4-nitrophenyl)methyl]-3,5,6-trihydroxy-5,6-<i>O</i>-isopropylidene-2,4,6,8,10,17-hexamethyl-11,15-dioxo-10,14-diaza-octadecanoic Acid (<b>71</b>)</h5><div class="NLM_p last">Fmoc-(<i>S</i>)-Val (500 mg, 1.47 mmol) was suspended in DCM (10 mL). TMP (532 μL, 3.35 mmol), HOAt (91 mg, 0.64 mmol), and HATU (560 mg, 1.47 mmol) were added, and a clear, yellow reaction mixture was stirred at room temperature for 30 min. A solution of <b>70</b> (830 mg, 1.34 mmol) in DCM (10 mL) was added dropwise, and the solution was stirred for 1.5 h at room temperature. Piperidine (775 μL, 8 mmol) was added, and the reaction mixture was stirred at room temperature overnight. After complete Fmoc deprotection, the reaction mixture was washed with sat. NaHCO<sub>3</sub> (10 mL) and water (3 × 10 mL). The solvent was removed under reduced pressure, and the residue was dissolved in THF. A solution of LiOH (270 mg, 6.7 mmol) in water (2 mL) was added, and the reaction mixture was stirred at room temperature for 1 h. The reaction mixture was neutralized with sat. KH<sub>2</sub>PO<sub>4</sub>, and the solvent was removed under reduced pressure. The residue was extracted with acetone (3 × 20 mL). Solids were filtered off, while the filtrate was evaporated, and the residue was purified by SPE and eluted with a mixture of DCM and methanolic ammonia. The title product was obtained as a yellow foam (500 mg), which was used in the next synthetic step without further purification. ES<sup>+</sup><i>m</i>/<i>z</i> 637.44 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-12,13,15-Trihydroxy-2-isopropyl-12,13-<i>O</i>-isopropylidene-8,10,12,14,16-pentamethyl-5-[(4-nitroophenyl)methyl]-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>72</b>)</h5><div class="NLM_p last">A flask was charged under argon with HATU (355 mg, 0.94 mmol), DMF (1 mL), and DIPEA (150 μL, 0.86 mmol). The reaction mixture turned orange-brown. <b>71</b> (496 mg, 0.78 mmol) was dissolved in DMF (2 mL), and this solution was added via a syringe pump to the solution of the coupling reagents<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (35 μL/min). The reaction mixture was stirred at room temperature for 4 h. The reaction was quenched with sat. NaHCO<sub>3</sub> (10 mL), and the mixture was extracted with DCM (3 × 10 mL). Combined organics were washed with water (3 × 10 mL), and the solvent was removed under reduced pressure. The residue was purified on silica and eluted with mixtures of cyclohexane:EtOAc (0–60%). The title product was obtained as a yellow foam (165 mg, 34%). ES<sup>+</sup><i>m</i>/<i>z</i> 619.43 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[(4-Aminophenyl)methyl]-12,13,15-trihydroxy-2-isopropyl-12,13-<i>O</i>-isopropylidene-8,10,12,14,16-pentamethyl-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>73</b>)</h5><div class="NLM_p last">Compound <b>72</b> (160 mg, 0.26 mmol) was dissolved in EtOAc (3 mL). Ammonium formate (82 mg, 1.3 mmol) and palladium on carbon (15 mg) were added, and the reaction mixture was stirred at 100 °C in a MW reactor for 35 min. The catalyst was filtered off, and new portions of ammonium formate (82 mg, 1.3 mmol) and palladium on carbon (15 mg) were added. The reaction was performed under the abovementioned conditions for an additional 20 min. Solids were filtered off, and the filtrate was evaporated to dryness. The crude product thus obtained (130 mg) was used in the next synthetic step without further purification. ES<sup>+</sup><i>m</i>/<i>z</i> 589.43 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[[4-[[(2<i>S</i>)-3-(2-Chlorophenyl)-2-[(2-phenylacetyl)amino]propanoyl]amino]phenyl]methyl]-12,13,15-trihydroxy-2-isopropyl-12,13-<i>O</i>-isopropylidene-8,10,12,14,16-pentamethyl-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>74</b>)</h5><div class="NLM_p last">Compound 73 (126 mg, 0.21 mmol) was dissolved in DMF (1 mL). <b>64</b> (73 mg, 0.23 mmol), DIPEA (71 μL, 0.42 mmol), and HATU (104 mg, 0.27 mmol) were added, and the yellow reaction mixture was stirred at room temperature for 1.5 h. The reaction was quenched with sat. NaHCO<sub>3</sub> (10 mL) and extracted with DCM (10 + 2 × 5 mL), combined organics were washed with water (3 × 5 mL), and the solvent was evaporated under reduced pressure. The residue was preabsorbed on silica and purified by column chromatography on silica and eluted with mixtures of DCM and methanolic ammonia (0–2.4%, 0.6% step, 25 mL each, 3%, 100 mL). The title product was obtained as an off-white foam (150 mg). The sample (25 mg,) was purified by prep HPLC-MS (purity 96.2%). ES+ <i>m</i>/<i>z</i> 888.21/890.21 [M + H]+.</div></div><div id="sec4_1_16_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> (2<i>S</i>,5<i>R</i>,10<i>R</i>,12<i>R</i>,13<i>R</i>,14<i>S</i>,15<i>S</i>,16<i>R</i>)-5-[[4-[[(2<i>S</i>)-3-(2-Chlorophenyl)-2-[(2-phenylacetyl)amino]propanoyl]amino]phenyl]methyl]-12,13,15-trihydroxy-2-isopropyl-8,10,12,14,16-pentamethyl-1,4,8-triazacycloheptadecane-3,7,17-trione (<b>63</b>)</h5><div class="NLM_p last">Compound <b>74</b> (124 mg, 0.14 mmol) was dissolved in THF (2 mL). A 1 N HCl (2 mL) solution was added, and the reaction mixture was stirred at room temperature for 4 h. After neutralization with 1 N NaOH, it was extracted with DCM (3 × 5 mL). Combined organics were washed with water (3 × 5 mL) and brine (5 mL). The solvent was removed under reduced pressure, and the residue was purified by prep HPLC-MS. Title product was obtained as a white powder (35 mg, purity 91.9%). ES<sup>+</sup><i>m</i>/<i>z</i> 848.39/850.4 [M + H]<sup>+</sup>. Two sets of signals in the NMR (most likely conformers). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ:4.96 (dd, <i>J</i> = 7.3 Hz, <i>J</i> = 4.7 Hz, 1H); 3.79 (d, <i>J</i> = 9.2 Hz, 1H); 3.61 (d, <i>J</i> = 9.7 Hz, 1H); 2.59 (dq, <i>J</i> = 9.7 Hz, <i>J</i> = 6.7 Hz, 1H); 2.36 (dd, <i>J</i> = 11.8 Hz, <i>J</i> = 5.1 Hz, 1H); 2.31 (dd, <i>J</i> = 11.8 Hz, <i>J</i> = 8.4 Hz, 1H); 2.3 (s, 3H); 2.15 (s, 3H); 1.72–1.87 (m, 4H); 1.69 (dd, <i>J</i> = 14.8 Hz, <i>J</i> = 2.4 Hz, 1H); 1.43 (dd, <i>J</i> = 14.8 Hz, <i>J</i> = 7.5 Hz, 1H); 1.34 (s, 3H); 1.27 (s, 3H); 1.23 (d, <i>J</i> = 6.7 Hz, 1H); 1.06 (s, 3H); 1.01 (d, <i>J</i> = 6.5 Hz, 1H); 0.97 (d, <i>J</i> = 6.6 Hz, 1H); 0.91 (t, <i>J</i> = 7.5 Hz, 1H).</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i117"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00327" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00327?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00327</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Methods description (chemical synthesis, molecular modeling, and <i>in vitro</i> and <i>in vivo</i> profiling); detailed experimental data from NMR studies; supplementary chemical structures and accompanied bioactivities; and synthetic procedures and characterization of compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">PDB files of cpd4 docked into 4HSA and 5HI4 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_002.pdb" class="ext-link">PDB</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf">jm1c00327_si_001.pdf (2.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_002.pdb">jm1c00327_si_002.pdb (383.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_003.pdb">jm1c00327_si_003.pdb (347.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_004.csv">jm1c00327_si_004.csv (15.36 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00327" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67517" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67517" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sanja Koštrun</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2827-1964" title="Orcid link">https://orcid.org/0000-0003-2827-1964</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0477656a6e652a6f6b777076716a44626d60616870652a6171"><span class="__cf_email__" data-cfemail="8efdefe0e4efa0e5e1fdfafcfbe0cee8e7eaebe2faefa0ebfb">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin Dack</span> - <span class="hlFld-Affiliation affiliation">LEO
Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dbb0bfbfbfb09bb7beb4f6abb3baa9b6baf5b8b4b6"><span class="__cf_email__" data-cfemail="e18a8585858aa18d848ecc918980938c80cf828e8c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Fajdetić</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span>; 
    <span>Present Address:
                        Present address: Xellia Ltd., Slavonska avenija 24/6, 10000 Zagreb, Croatia (A.F.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dijana Pešić</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karmen Brajša</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vlatka Bencetić Mihaljević</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dubravko Jelić</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adriana Petrinić Grba</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ivaylo Elenkov</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renata Rupčić</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span>; 
    <span>Present Address:
                        Present address: HALMED Agency for Medicinal Productss and Medical Devices of Croatia, Ksaverska cesta 4, 10000 Zagreb, Croatia (R.R.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samra Kapić</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span>; 
    <span>Present Address:
                        Present address: PLIVA Hrvatska d.o.o., Baruna Filipovića 25, 10000 Zagreb, Croatia (S.K.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ivana Ozimec Landek</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kristina Butković</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ana Grgičević</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dinko Žiher</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ana Čikoš</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span>; 
    <span>Present Address:
                        Present address: Rudjer Bošković Institute, Bijenička c. 54, 10000 Zagreb, Croatia (A.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jasna Padovan</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gordon Saxty</span> - <span class="hlFld-Affiliation affiliation">Fidelta
Ltd, Prilaz baruna Filipovića 29, 10000 Zagreb, Croatia</span>; 
    <span>Present Address:
                        Present address: 37, Woodpecker Way, Great Cambourne, Cambridge CB236GZ, U.K (G.S.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haakan Bladh</span> - <span class="hlFld-Affiliation affiliation">LEO
Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tine Skak-Nielsen</span> - <span class="hlFld-Affiliation affiliation">LEO
Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Feldbaek Nielsen</span> - <span class="hlFld-Affiliation affiliation">LEO
Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maja Lambert</span> - <span class="hlFld-Affiliation affiliation">LEO
Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Stahlhut</span> - <span class="hlFld-Affiliation affiliation">LEO
Pharma A/S, Industriparken 55, 2750 Ballerup, Denmark</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i124">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04812" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04812" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The success and final outcome of this project would not be the same without contributions from many people. We thank Đurđica Lugarić, Malajka Zlogleđa, and Kristina Kraljić for their contribution to compound library synthesis and scientists at the DMPK department who helped with collection of <i>in vitro</i> ADME data. We would also like to thank the colleagues at Fidelta Ltd. and LEO Pharma who did not contribute directly, but their support was vital for completion of this paper.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS Used</td></tr><tr><td class="NLM_term">2D</td><td class="NLM_def"><p class="first last">two-dimensional</p></td></tr><tr><td class="NLM_term">3D</td><td class="NLM_def"><p class="first last">three-dimensional<sup>3</sup></p></td></tr><tr><td class="NLM_term"><i>J</i><sub>H,H</sub></td><td class="NLM_def"><p class="first last">vicinal proton-proton coupling constant</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism and excretion</p></td></tr><tr><td class="NLM_term">CAT</td><td class="NLM_def"><p class="first last">Cambridge antibody technology</p></td></tr><tr><td class="NLM_term">CD4</td><td class="NLM_def"><p class="first last">cluster of differentiation 4</p></td></tr><tr><td class="NLM_term">CD8</td><td class="NLM_def"><p class="first last">cluster of differentiation 8</p></td></tr><tr><td class="NLM_term">CXCL1</td><td class="NLM_def"><p class="first last">chemokine (C-X-C motif) ligand 1</p></td></tr><tr><td class="NLM_term">CYPs</td><td class="NLM_def"><p class="first last">cytochromes P450</p></td></tr><tr><td class="NLM_term">ELISA</td><td class="NLM_def"><p class="first last">enzyme linked immunosorbent assays</p></td></tr><tr><td class="NLM_term">Fab</td><td class="NLM_def"><p class="first last">antigen binding fragment</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go-related gene potassium channel</p></td></tr><tr><td class="NLM_term">HT29</td><td class="NLM_def"><p class="first last">human adenocarcinoma cells</p></td></tr><tr><td class="NLM_term">IL</td><td class="NLM_def"><p class="first last">interleukin</p></td></tr><tr><td class="NLM_term">ILC3</td><td class="NLM_def"><p class="first last">group 3 innate lymphoid cells</p></td></tr><tr><td class="NLM_term">MC</td><td class="NLM_def"><p class="first last">macrocyclic</p></td></tr><tr><td class="NLM_term">MDCKII-MDR1</td><td class="NLM_def"><p class="first last">Madin-Darby canine kidney cell line transfected by human MDR1 gene</p></td></tr><tr><td class="NLM_term">MM-GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics combined with the generalized Born and surface area continuum solvation method</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear factor κ-light-chain-enhancer of activated B cells</p></td></tr><tr><td class="NLM_term">NK cells</td><td class="NLM_def"><p class="first last">natural killer cells</p></td></tr><tr><td class="NLM_term">nOe</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect</p></td></tr><tr><td class="NLM_term">NOESY</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect spectroscopy</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PPB</td><td class="NLM_def"><p class="first last">plasma protein binding</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interactions</p></td></tr><tr><td class="NLM_term">RASF</td><td class="NLM_def"><p class="first last">rheumatoid arthritis synovial fibroblasts</p></td></tr><tr><td class="NLM_term">RORgT</td><td class="NLM_def"><p class="first last">retinoic-acid-receptor-related orphan nuclear receptor gamma</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SMILES</td><td class="NLM_def"><p class="first last">simplified molecular-input line-entry system</p></td></tr><tr><td class="NLM_term">STD NMR</td><td class="NLM_def"><p class="first last">saturation transfer difference nuclear magnetic resonance method</p></td></tr><tr><td class="NLM_term">Tc17 cells</td><td class="NLM_def"><p class="first last">cytotoxic CD8<sup>+</sup> T cells</p></td></tr><tr><td class="NLM_term">Th17 cells</td><td class="NLM_def"><p class="first last">T helper 17 cells</p></td></tr><tr><td class="NLM_term">THP1</td><td class="NLM_def"><p class="first last">human monocytic cell line derived from an acute monocytic leukemia patient</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25116" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25116" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parisi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Symmons, D. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, C. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashcroft, D. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Management of Psoriasis and Associated ComorbidiTy (IMPACT) project team. Global epidemiology of psoriasis: a systematic review of incidence and prevalence</span>. <i>J. Inv. Derm.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1038/jid.2012.339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fjid.2012.339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=23014338" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2013&pages=377-385&author=R.+Parisiauthor=D.+P.+M.+Symmonsauthor=C.+E.+M.+Griffithsauthor=D.+M.+Ashcroft&title=Identification+and+Management+of+Psoriasis+and+Associated+ComorbidiTy+%28IMPACT%29+project+team.+Global+epidemiology+of+psoriasis%3A+a+systematic+review+of+incidence+and+prevalence&doi=10.1038%2Fjid.2012.339"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fjid.2012.339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2012.339%26sid%3Dliteratum%253Aachs%26aulast%3DParisi%26aufirst%3DR.%26aulast%3DSymmons%26aufirst%3DD.%2BP.%2BM.%26aulast%3DGriffiths%26aufirst%3DC.%2BE.%2BM.%26aulast%3DAshcroft%26aufirst%3DD.%2BM.%26atitle%3DIdentification%2520and%2520Management%2520of%2520Psoriasis%2520and%2520Associated%2520ComorbidiTy%2520%2528IMPACT%2529%2520project%2520team.%2520Global%2520epidemiology%2520of%2520psoriasis%253A%2520a%2520systematic%2520review%2520of%2520incidence%2520and%2520prevalence%26jtitle%3DJ.%2520Inv.%2520Derm.%26date%3D2013%26volume%3D133%26spage%3D377%26epage%3D385%26doi%3D10.1038%2Fjid.2012.339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>WHO Global Report on Psoriasis</i>;
Geneva, Switzerland, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=World+Health+Organization&title=WHO+Global+Report+on+Psoriasis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DWHO%2520Global%2520Report%2520on%2520Psoriasis%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brembilla, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senra, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehncke, W.-H.</span></span> <span> </span><span class="NLM_article-title">The IL-17 family of cytokines in psoriasis: IL-17A and beyond</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1682</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.01682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.3389%2Ffimmu.2018.01682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=30127781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWmurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1682&author=N.+C.+Brembillaauthor=L.+Senraauthor=W.-H.+Boehncke&title=The+IL-17+family+of+cytokines+in+psoriasis%3A+IL-17A+and+beyond&doi=10.3389%2Ffimmu.2018.01682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The IL-17 family of cytokines in Psoriasis: IL-17A and beyond</span></div><div class="casAuthors">Brembilla, Nicolo Costantino; Senra, Luisa; Boehncke, Wolf-Henning</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1682/1-1682/13</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Psoriasis is a frequent chronic inflammatory skin disease, nowadays considered a major global health problem.  Several new drugs, targeting the IL-23/IL-17A pathway, have been recently licensed or are in clin. development.  These therapies represent a major improvement of the way in which psoriasis is managed, since they show an unprecedented efficacy on skin symptoms of psoriasis.  This has been made possible, thanks to an increasingly more accurate pathogenic view of psoriasis.  Today, the belief that Th17 cells mediate psoriasis is moving to the concept of psoriasis as an IL-17A-driven disease.  New questions arise at the horizon, given that IL-17A is part of a newly described family of cytokines, which has five distinct homologous: IL-17B, IL-17C, IL-17D, IL-17E, also known as IL-25 and IL-17F.  IL-17 family cytokines elicit similar effects in target cells, but simultaneously trigger different and sometimes opposite functions in a tissue-specific manner.  This is complicated by the fact that IL-17 cytokines show a high capacity of synergisms with other inflammatory stimuli.  In this review, we will summarize the current knowledge around the cytokines belonging to the IL-17 family in relation to skin inflammation in general and psoriasis in particular, and discuss possible clin. implications.  A comprehensive understanding of the different roles played by the IL-17 cytokines is crucial to appreciate current and developing therapies and to allow an effective pathogenesis- and mechanisms-driven drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Q2Yx4qUoprVg90H21EOLACvtfcHk0liJa6D0W2hlFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWmurrJ&md5=d3975343c94cc7fd523d55a8c3013ecf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.01682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.01682%26sid%3Dliteratum%253Aachs%26aulast%3DBrembilla%26aufirst%3DN.%2BC.%26aulast%3DSenra%26aufirst%3DL.%26aulast%3DBoehncke%26aufirst%3DW.-H.%26atitle%3DThe%2520IL-17%2520family%2520of%2520cytokines%2520in%2520psoriasis%253A%2520IL-17A%2520and%2520beyond%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D1682%26doi%3D10.3389%2Ffimmu.2018.01682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blauvelt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiricozzi, A.</span></span> <span> </span><span class="NLM_article-title">The immunologic role of IL-17 in psoriasis and psoriatic arthritis pathogenesis</span>. <i>Clin. Rev. Allerg. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1007/s12016-018-8702-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1007%2Fs12016-018-8702-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=30109481" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2018&pages=379-390&author=A.+Blauveltauthor=A.+Chiricozzi&title=The+immunologic+role+of+IL-17+in+psoriasis+and+psoriatic+arthritis+pathogenesis&doi=10.1007%2Fs12016-018-8702-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs12016-018-8702-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12016-018-8702-3%26sid%3Dliteratum%253Aachs%26aulast%3DBlauvelt%26aufirst%3DA.%26aulast%3DChiricozzi%26aufirst%3DA.%26atitle%3DThe%2520immunologic%2520role%2520of%2520IL-17%2520in%2520psoriasis%2520and%2520psoriatic%2520arthritis%2520pathogenesis%26jtitle%3DClin.%2520Rev.%2520Allerg.%2520Immunol.%26date%3D2018%26volume%3D55%26spage%3D379%26epage%3D390%26doi%3D10.1007%2Fs12016-018-8702-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffatellu, M.</span></span> <span> </span><span class="NLM_article-title">Cytokines IL-17 and IL-22 in the host response to infection</span>. <i>Pathog. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">ftw111</span>, <span class="refDoi"> DOI: 10.1093/femspd/ftw111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1093%2Ffemspd%2Fftw111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27915228" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2016&pages=ftw111&author=M.+Valeriauthor=M.+Raffatellu&title=Cytokines+IL-17+and+IL-22+in+the+host+response+to+infection&doi=10.1093%2Ffemspd%2Fftw111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Ffemspd%2Fftw111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ffemspd%252Fftw111%26sid%3Dliteratum%253Aachs%26aulast%3DValeri%26aufirst%3DM.%26aulast%3DRaffatellu%26aufirst%3DM.%26atitle%3DCytokines%2520IL-17%2520and%2520IL-22%2520in%2520the%2520host%2520response%2520to%2520infection%26jtitle%3DPathog.%2520Dis.%26date%3D2016%26volume%3D74%26spage%3Dftw111%26doi%3D10.1093%2Ffemspd%2Fftw111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kirkham, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavanaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, K.</span></span> <span> </span><span class="NLM_article-title">Interleukin-17A: a unique pathway in immune-mediated diseases: psoriasis, psoriatic arthritis and rheumatoid arthritis</span>. <i>Immunology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1111/imm.12142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1111%2Fimm.12142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=23819583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktFahtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2014&pages=133-142&author=B.+W.+Kirkhamauthor=A.+Kavanaughauthor=K.+Reich&title=Interleukin-17A%3A+a+unique+pathway+in+immune-mediated+diseases%3A+psoriasis%2C+psoriatic+arthritis+and+rheumatoid+arthritis&doi=10.1111%2Fimm.12142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Interleukin-17A: a unique pathway in immune-mediated diseases: psoriasis, psoriatic arthritis and rheumatoid arthritis</span></div><div class="casAuthors">Kirkham, Bruce W.; Kavanaugh, Arthur; Reich, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-142</span>CODEN:
                <span class="NLM_cas:coden">IMMUAM</span>;
        ISSN:<span class="NLM_cas:issn">0019-2805</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Exptl. evidence points to the importance of the cytokine interleukin-17A (IL-17A) in the pathogenesis of several immunoinflammatory diseases including psoriasis, psoriatic arthritis and rheumatoid arthritis.  Although a principal effector of T helper type 17 cells, IL-17A is produced by many other cell types including CD8+ T cells and γδ T cells, and is found at high levels assocd. with mast cells and neutrophils at sites of skin and joint disease in humans.  IL-17A up-regulates expression of numerous inflammation-related genes in target cells such as keratinocytes and fibroblasts, leading to increased prodn. of chemokines, cytokines, antimicrobial peptides and other mediators that contribute to clin. disease features.  Importantly, IL-17A must be considered within the context of the local microenvironment, because it acts synergistically or additively with other pro-inflammatory cytokines, including tumor necrosis factor.  Several direct IL-17A inhibitors have shown promising activity in proof of concept and phase 2 clin. studies, thereby providing confirmation of exptl. data supporting IL-17A in disease pathogenesis, although levels of response are not predicted by pre-clin. findings.  IL-17A inhibitors produced rapid down-regulation of the psoriasis gene signature and high clin. response rates in patients with moderate-to-severe plaque psoriasis, consistent with an important role for IL-17A in psoriasis pathogenesis.  Clin. response rates with IL-17A inhibitors in psoriatic arthritis and rheumatoid arthritis, however, were improved to a lesser degree compared with placebo, suggesting that IL-17A is either important in a subset of patients or plays a relatively minor role in inflammatory joint disease.  Ongoing phase 3 clin. trials should provide further information on the role of IL-17A in these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0u3C1Z5BR6LVg90H21EOLACvtfcHk0lgVcXJEQ4MtHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktFahtA%253D%253D&md5=36150b3dceffde2c4fcbe862894cfbe6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fimm.12142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimm.12142%26sid%3Dliteratum%253Aachs%26aulast%3DKirkham%26aufirst%3DB.%2BW.%26aulast%3DKavanaugh%26aufirst%3DA.%26aulast%3DReich%26aufirst%3DK.%26atitle%3DInterleukin-17A%253A%2520a%2520unique%2520pathway%2520in%2520immune-mediated%2520diseases%253A%2520psoriasis%252C%2520psoriatic%2520arthritis%2520and%2520rheumatoid%2520arthritis%26jtitle%3DImmunology%26date%3D2014%26volume%3D141%26spage%3D133%26epage%3D142%26doi%3D10.1111%2Fimm.12142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffen, S. L.</span></span> <span> </span><span class="NLM_article-title">IL-17RC: a partner in IL-17 signaling and beyond</span>. <i>Semin. Immunopathol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1007/s00281-009-0185-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1007%2Fs00281-009-0185-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=20012905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=33-42&author=A.+W.+Hoauthor=S.+L.+Gaffen&title=IL-17RC%3A+a+partner+in+IL-17+signaling+and+beyond&doi=10.1007%2Fs00281-009-0185-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">IL-17RC: a partner in IL-17 signaling and beyond</span></div><div class="casAuthors">Ho, Allen W.; Gaffen, Sarah L.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Immunopathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-42</span>CODEN:
                <span class="NLM_cas:coden">SIEMB7</span>;
        ISSN:<span class="NLM_cas:issn">1863-2297</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The interleukin (IL)-17 cytokine family members IL-17A and IL-17F mediate inflammatory activities via the IL-17 receptor (IL-17R) complex, comprised of the IL-17RA and IL-17RC subunits.  Proper regulation of the IL-17 signaling axis results in effective host defense against extracellular pathogens, while aberrant signaling can drive autoimmune pathol.  Elucidating the mol. mechanisms underlying IL-17 signal transduction can yield an enhanced understanding of inflammatory immune processes and also create an avenue for therapeutic intervention in the treatment of IL-17-dependent diseases.  To date, the fundamental signaling mechanisms used by the IL-17R complex are still incompletely defined.  While current structure-function studies have primarily focused on the IL-17RA subunit, recent research indicates that the IL-17RC subunit plays a key role in modulating IL-17 responses.  This review will examine what is known regarding IL-17RC function and provide a framework for future work on this subunit and its impact on human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2VeCEngWLlLVg90H21EOLACvtfcHk0lgi4mcbCOZDbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyisrg%253D&md5=150f5d096961d947c037a61838e14405</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00281-009-0185-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00281-009-0185-0%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DA.%2BW.%26aulast%3DGaffen%26aufirst%3DS.%2BL.%26atitle%3DIL-17RC%253A%2520a%2520partner%2520in%2520IL-17%2520signaling%2520and%2520beyond%26jtitle%3DSemin.%2520Immunopathol.%26date%3D2010%26volume%3D32%26spage%3D33%26epage%3D42%26doi%3D10.1007%2Fs00281-009-0185-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velichko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">A Novel nuclear function for the interleukin-17 signaling adaptor protein Act1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">e0163323/1</span>– <span class="NLM_lpage">e0163323/17</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1371%2Fjournal.pone.0163323" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0163323%2F1-e0163323%2F17&author=S.+Velichkoauthor=X.+Zhouauthor=L.+Zhuauthor=J.+D.+Andersonauthor=R.+Wuauthor=Y.+Chen&title=A+Novel+nuclear+function+for+the+interleukin-17+signaling+adaptor+protein+Act1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0163323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0163323%26sid%3Dliteratum%253Aachs%26aulast%3DVelichko%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DAnderson%26aufirst%3DJ.%2BD.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DA%2520Novel%2520nuclear%2520function%2520for%2520the%2520interleukin-17%2520signaling%2520adaptor%2520protein%2520Act1%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26spage%3De0163323%252F1%26epage%3De0163323%252F17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silfvast-Kaiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paek, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menter, A.</span></span> <span> </span><span class="NLM_article-title">Anti-IL17 therapies for psoriasis</span>. <i>Expert Opin. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1080/14712598.2019.1555235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1080%2F14712598.2019.1555235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=30500317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKitLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=45-54&author=A.+Silfvast-Kaiserauthor=S.+Y.+Paekauthor=A.+Menter&title=Anti-IL17+therapies+for+psoriasis&doi=10.1080%2F14712598.2019.1555235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-IL17 therapies for psoriasis</span></div><div class="casAuthors">Silfvast-Kaiser, Annika; Paek, So Yeon; Menter, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Biological Therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-54</span>CODEN:
                <span class="NLM_cas:coden">EOBTA2</span>;
        ISSN:<span class="NLM_cas:issn">1471-2598</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Interleukin-17 (IL-17) is a proinflammatory cytokine considered to play a significant role in the immunopathogenesis of plaque psoriasis.  As a result, focus in clin. trials has undergone a shift towards disease specific targets, with the goals of more effective treatment and redn. in the incidence of serious adverse events.: Two monoclonal antibodies targeting IL-17A (secukinumab, ixekizumab) and one against the IL-17 receptor (brodalumab) are approved for the treatment of moderate-to-severe plaque psoriasis.  Herein, the clin. efficacy, safety and tolerability of each is reviewed by summarizing the existing literature (found via PubMed database).: The development and approval of the IL-17 inhibitor agents secukinumab, ixekizumab, and brodalumab has expanded psoriatic treatment with effective options, validating the importance of the pro-inflammatory role of IL-17 psoriatic pathophysiol.  Biol. treatment options for psoriasis will continue to grow, esp. IL-17 and IL-23 related agents, with an increasing specificity of agents to be available in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-h0nKu45xbVg90H21EOLACvtfcHk0lh9m2ISOrUclw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKitLzK&md5=287376738f5c578789fab39754c43a3d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F14712598.2019.1555235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14712598.2019.1555235%26sid%3Dliteratum%253Aachs%26aulast%3DSilfvast-Kaiser%26aufirst%3DA.%26aulast%3DPaek%26aufirst%3DS.%2BY.%26aulast%3DMenter%26aufirst%3DA.%26atitle%3DAnti-IL17%2520therapies%2520for%2520psoriasis%26jtitle%3DExpert%2520Opin.%2520Biol.%2520Ther.%26date%3D2019%26volume%3D19%26spage%3D45%26epage%3D54%26doi%3D10.1080%2F14712598.2019.1555235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakkas, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zafiriou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogdanos, D. P.</span></span> <span> </span><span class="NLM_article-title">Mini review: new treatments in psoriatic arthritis. Focus on the IL-23/17 axis</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">872</span>, <span class="refDoi"> DOI: 10.3389/fphar.2019.00872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.3389%2Ffphar.2019.00872" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=31447673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFSht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=872&author=L.+I.+Sakkasauthor=E.+Zafiriouauthor=D.+P.+Bogdanos&title=Mini+review%3A+new+treatments+in+psoriatic+arthritis.+Focus+on+the+IL-23%2F17+axis&doi=10.3389%2Ffphar.2019.00872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mini review: new treatments in psoriatic arthritis. Focus on the IL-23/17 axis</span></div><div class="casAuthors">Sakkas, Lazaros I.; Zafiriou, Efterpi; Bogdanos, Dimitrios P.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">872</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Psoriasis, an inflammatory skin disease, and psoriatic arthritis (PsA), an inflammatory arthritis, share clin., genetic, and pathogenic factors and may be summed as one disease, the psoriatic disease.  Interleukin (IL)-17 plays a major role in the development of both psoriasis and PsA.  IL-23 is important in the proliferation and maintenance of IL-17, and therefore, cytokines of the IL-23/IL-17 axis attracted much interest as therapeutic targets in psoriasis and PsA.  Therapeutic agents targeting the IL-23/IL-17 axis have been proven to be very effective in psoriasis and PsA, some are already in the therapeutic armamentarium and others are in the development.  Some agents, target IL-23 and others IL-17 and include anti-IL-12/IL-23 p40 (ustekinumab, briankizumab), anti-IL-23p19 (guselkumab, tildrakizumab, risankizumab, brazikumab, mirikizumab), anti-IL-17A (secukinumab, ixekizumab), dual anti-IL-17A and anti-IL-17F (bimekizumab), or anti-IL-17 receptor (brodalumab) monoclonal antibodies.  Janus tyrosine kinase(JAK) inhibitors also directly affect IL-23 and, thus, IL-17.  After the first-generation pan-JAK inhibitors have been shown efficacy (tofacitinib, baricitinib), new-generation selective JAK inhibitors (filgotinib, upadacitinib) are under investigation in psoriasis and PsA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mVvPfw0KkLVg90H21EOLACvtfcHk0lh9m2ISOrUclw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFSht70%253D&md5=c10f25a0258886777a060da0abe4796e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2019.00872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2019.00872%26sid%3Dliteratum%253Aachs%26aulast%3DSakkas%26aufirst%3DL.%2BI.%26aulast%3DZafiriou%26aufirst%3DE.%26aulast%3DBogdanos%26aufirst%3DD.%2BP.%26atitle%3DMini%2520review%253A%2520new%2520treatments%2520in%2520psoriatic%2520arthritis.%2520Focus%2520on%2520the%2520IL-23%252F17%2520axis%26jtitle%3DFront.%2520Pharmacol.%26date%3D2019%26volume%3D10%26spage%3D872%26doi%3D10.3389%2Ffphar.2019.00872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: progressing toward the reality</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+progressing+toward+the+reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0lgpMwB-OmhD6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520progressing%2520toward%2520the%2520reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">Artificial macrocycles as IL-17A - IL-17RA antagonists</span>. <i>MedChemComm.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1039/C7MD00464H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1039%2FC7MD00464H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=29750108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=22-26&author=W.+Wangauthor=M.+R.+Grovesauthor=A.+D%C3%B6mling&title=Artificial+macrocycles+as+IL-17A+-+IL-17RA+antagonists&doi=10.1039%2FC7MD00464H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial macrocycles as IL-17A/IL-17RA antagonists</span></div><div class="casAuthors">Wang, Wenjia; Groves, Matthew R.; Doemling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-26</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Interleukin 17(A) is a pro-inflammatory cytokine involved in several auto-immune and inflammatory diseases.  Current antagonists against IL-17(A) or its receptor (IL-17RA) that show efficacy in clin. trials are monoclonal antibodies (mAbs).  However, recently designed artificial macrocycles are potent IL-17A/IL-17RA antagonists.  Based on co-crystal structures, a better understanding of the biol. activity and SAR of the macrocycles has been gained, demonstrating that they can compete with mAbs for difficult targets such as PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMncYtypRDh7Vg90H21EOLACvtfcHk0lgpMwB-OmhD6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L&md5=62b3ef7b447d2367e337dad6e5a3a204</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2FC7MD00464H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00464H%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3DArtificial%2520macrocycles%2520as%2520IL-17A%2520-%2520IL-17RA%2520antagonists%26jtitle%3DMedChemComm.%26date%3D2018%26volume%3D9%26spage%3D22%26epage%3D26%26doi%3D10.1039%2FC7MD00464H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M.</span>; <span class="NLM_string-name">Terrett, N. K.</span>; <span class="NLM_string-name">Connors, W. H.</span>; <span class="NLM_string-name">Shortsleeves, K. C.</span>; <span class="NLM_string-name">Seigal, B. A.</span>; <span class="NLM_string-name">Snedeker, C.</span>; <span class="NLM_string-name">Hale, S. P.</span>; <span class="NLM_string-name">Briggs, T. F.</span>; <span class="NLM_string-name">Favaloro, F. G.</span>; <span class="NLM_string-name">Cipriani, T. J.</span>; <span class="NLM_string-name">Yan, D</span>; <span class="NLM_string-name">Alexander, S. L.</span>; <span class="NLM_string-name">Thorarensen, A.</span>; <span class="NLM_string-name">Xing, L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Macrocyclic Compounds as IL-17 Modulators</span>.  <i>Int. Pat. Appl.</i> <span class="NLM_patent">WO2013116682A1</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=M.+Taylor&author=N.+K.+Terrett&author=W.+H.+Connors&author=K.+C.+Shortsleeves&author=B.+A.+Seigal&author=C.+Snedeker&author=S.+P.+Hale&author=T.+F.+Briggs&author=F.+G.+Favaloro&author=T.+J.+Cipriani&author=D+Yan&author=S.+L.+Alexander&author=A.+Thorarensen&author=L.+Xing&title=Preparation+of+Macrocyclic+Compounds+as+IL-17+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DM.%26atitle%3DPreparation%2520of%2520Macrocyclic%2520Compounds%2520as%2520IL-17%2520Modulators%26jtitle%3DInt.%2520Pat.%2520Appl.%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Espada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broughton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodge, J. A.</span></span> <span> </span><span class="NLM_article-title">A binding site on IL-17A for inhibitory macrocycles revealed by hydrogen-deuterium exchange mass spectrometry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2255</span>– <span class="NLM_lpage">2260</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Ojsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2255-2260&author=A.+Espadaauthor=H.+Broughtonauthor=S.+Jonesauthor=M.+J.+Chalmersauthor=J.+A.+Dodge&title=A+binding+site+on+IL-17A+for+inhibitory+macrocycles+revealed+by+hydrogen-deuterium+exchange+mass+spectrometry&doi=10.1021%2Facs.jmedchem.5b01693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Binding Site on IL-17A for Inhibitory Macrocycles Revealed by Hydrogen/Deuterium Exchange Mass Spectrometry</span></div><div class="casAuthors">Espada, Alfonso; Broughton, Howard; Jones, Spencer; Chalmers, Michael J.; Dodge, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2255-2260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational assessment of the IL-17A structure identified two distinct binding pockets, the β-hairpin pocket and the α-helix pocket.  The β-hairpin pocket was hypothesized to be the site of binding for peptide macrocycles.  Support for this hypothesis was obtained using HDX-MS which revealed protection to exchange only within the β-hairpin pocket.  This data represents the first direct structural evidence of a small mol. binding site on IL-17A that functions to disrupt the interaction with its receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsr5NPNUfSbVg90H21EOLACvtfcHk0liVbtfYIc-sbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Ojsrk%253D&md5=8ce4090423e451c6365b9dc74a6c57c3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01693%26sid%3Dliteratum%253Aachs%26aulast%3DEspada%26aufirst%3DA.%26aulast%3DBroughton%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DChalmers%26aufirst%3DM.%2BJ.%26aulast%3DDodge%26aufirst%3DJ.%2BA.%26atitle%3DA%2520binding%2520site%2520on%2520IL-17A%2520for%2520inhibitory%2520macrocycles%2520revealed%2520by%2520hydrogen-deuterium%2520exchange%2520mass%2520spectrometry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2255%26epage%3D2260%26doi%3D10.1021%2Facs.jmedchem.5b01693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, D.</span>; <span class="NLM_string-name">Alexander, S.</span>; <span class="NLM_string-name">Bond, J.</span>; <span class="NLM_string-name">Briggs, T.</span>; <span class="NLM_string-name">Fraley, A.</span>; <span class="NLM_string-name">Hale, S.</span>; <span class="NLM_string-name">Landsman, T.</span>; <span class="NLM_string-name">Martinelli, R.</span>; <span class="NLM_string-name">Shortsleeves, K.</span>; <span class="NLM_string-name">Terrett, N.</span>; <span class="NLM_string-name">Walsh, N</span></span>. <span> </span><span class="NLM_article-title">Identication and Characterization of Synthetic Small Molecule Macrocycle Antagonists of Human IL17A</span>.  <i>Meeting Abstracts</i>; <i>No 1810</i>, <span class="NLM_publisher-name">2012 ACR/ARHP Annual Meeting</span>: <span class="NLM_publisher-loc">Washington, DC</span>, Nov  <span class="NLM_volume">9-14</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+Livingston&author=S.+Alexander&author=J.+Bond&author=T.+Briggs&author=A.+Fraley&author=S.+Hale&author=T.+Landsman&author=R.+Martinelli&author=K.+Shortsleeves&author=N.+Terrett&author=N+Walsh&title=Identication+and+Characterization+of+Synthetic+Small+Molecule+Macrocycle+Antagonists+of+Human+IL17A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DLivingston%26aufirst%3DD.%26atitle%3DIdentication%2520and%2520Characterization%2520of%2520Synthetic%2520Small%2520Molecule%2520Macrocycle%2520Antagonists%2520of%2520Human%2520IL17A%26jtitle%3DMeeting%2520Abstracts%26pub%3D2012%2520ACR%252FARHP%2520Annual%2520Meeting%26volume%3D9-14%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, S. L.</span>; <span class="NLM_string-name">Briggs, T. F.</span>; <span class="NLM_string-name">Favaloro, F. G.,  Jr.</span>; <span class="NLM_string-name">Hale, S. P.</span>; <span class="NLM_string-name">Seigal, B. A.</span>; <span class="NLM_string-name">Shortsleeves, K. C.</span>; <span class="NLM_string-name">Snedeker, C.</span>; <span class="NLM_string-name">Taylor, M.</span>; <span class="NLM_string-name">Terrett, N. K.</span>; <span class="NLM_string-name">Yan, D.</span>; <span class="NLM_string-name">Nowak, P. W.</span>; <span class="NLM_string-name">Denny, R. A.</span>; <span class="NLM_string-name">Chrunyk, B. A.</span>; <span class="NLM_string-name">Sahasrabudhe, P. V.</span>; <span class="NLM_string-name">Withka, J. M.</span></span> <span> </span><span class="NLM_article-title">Compounds for Modulating IL-17</span>.  <i>Int. Pat. Appl.</i> <span class="NLM_patent">WO2014066726A2</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+L.+Alexander&author=T.+F.+Briggs&author=F.+G.+Favaloro&author=S.+P.+Hale&author=B.+A.+Seigal&author=K.+C.+Shortsleeves&author=C.+Snedeker&author=M.+Taylor&author=N.+K.+Terrett&author=D.+Yan&author=P.+W.+Nowak&author=R.+A.+Denny&author=B.+A.+Chrunyk&author=P.+V.+Sahasrabudhe&author=J.+M.+Withka&title=Compounds+for+Modulating+IL-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DS.%2BL.%26atitle%3DCompounds%2520for%2520Modulating%2520IL-17%26jtitle%3DInt.%2520Pat.%2520Appl.%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balbo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrunyk, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussenius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalsamy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span> <span> </span><span class="NLM_article-title">Binding site elucidation and structure guided design of macrocyclic IL-17A antagonists</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30859</span>, <span class="refDoi"> DOI: 10.1038/srep30859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fsrep30859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27527709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlylt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=30859&author=S.+Liuauthor=L.+A.+Dakinauthor=L.+Xingauthor=J.+M.+Withkaauthor=P.+V.+Sahasrabudheauthor=W.+Liauthor=M.+E.+Bankerauthor=P.+Balboauthor=S.+Shankerauthor=B.+A.+Chrunykauthor=Z.+Guoauthor=J.+M.+Chenauthor=J.+A.+Youngauthor=G.+Baiauthor=J.+T.+Starrauthor=S.+W.+Wrightauthor=J.+Busseniusauthor=S.+Tanauthor=A.+Gopalsamyauthor=B.+A.+Lefkerauthor=F.+Vincentauthor=L.+H.+Jonesauthor=H.+Xuauthor=L.+R.+Hothauthor=K.+F.+Geogheganauthor=X.+Qiuauthor=M.+E.+Bunnageauthor=A.+Thorarensen&title=Binding+site+elucidation+and+structure+guided+design+of+macrocyclic+IL-17A+antagonists&doi=10.1038%2Fsrep30859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Binding site elucidation and structure guided design of macrocyclic IL-17A antagonists</span></div><div class="casAuthors">Liu, Shenping; Dakin, Leslie A.; Xing, Li; Withka, Jane M.; Sahasrabudhe, Parag V.; Li, Wei; Banker, Mary Ellen; Balbo, Paul; Shanker, Suman; Chrunyk, Boris A.; Guo, Zuojun; Chen, Jinshan M.; Young, Jennifer A.; Bai, Guoyun; Starr, Jeremy T.; Wright, Stephen W.; Bussenius, Joerg; Tan, Sheng; Gopalsamy, Ariamala; Lefker, Bruce A.; Vincent, Fabien; Jones, Lyn H.; Xu, Hua; Hoth, Lise R.; Geoghegan, Kieran F.; Qiu, Xiayang; Bunnage, Mark E.; Thorarensen, Atli</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30859</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interleukin-17A (IL-17A) is a principal driver of multiple inflammatory and immune disorders.  Antibodies that neutralize IL-17A or its receptor (IL-17RA) deliver efficacy in autoimmune diseases, but no small-mol. IL-17A antagonists have yet progressed into clin. trials.  Investigation of a series of linear peptide ligands to IL-17A and characterization of their binding site has enabled the design of novel macrocyclic ligands that are themselves potent IL-17A antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpATWLU4utbrVg90H21EOLACvtfcHk0liriuchjOlzAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlylt7bM&md5=333917e3ad408ed5e48a98f99faf7aca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fsrep30859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep30859%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DXing%26aufirst%3DL.%26aulast%3DWithka%26aufirst%3DJ.%2BM.%26aulast%3DSahasrabudhe%26aufirst%3DP.%2BV.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DBalbo%26aufirst%3DP.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DChrunyk%26aufirst%3DB.%2BA.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DStarr%26aufirst%3DJ.%2BT.%26aulast%3DWright%26aufirst%3DS.%2BW.%26aulast%3DBussenius%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DS.%26aulast%3DGopalsamy%26aufirst%3DA.%26aulast%3DLefker%26aufirst%3DB.%2BA.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DThorarensen%26aufirst%3DA.%26atitle%3DBinding%2520site%2520elucidation%2520and%2520structure%2520guided%2520design%2520of%2520macrocyclic%2520IL-17A%2520antagonists%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D30859%26doi%3D10.1038%2Fsrep30859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ting, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broughton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bina, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodge, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groshong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglicmot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afshar, S.</span></span> <span> </span><span class="NLM_article-title">Utilization of peptide phage display to investigate hotspots on IL-17A and what it means for drug discovery</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e0190850/1</span>– <span class="NLM_lpage">e0190850/18</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1371%2Fjournal.pone.0190850" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0190850%2F1-e0190850%2F18&author=J.+P.+Tingauthor=F.+Tungauthor=S.+Antonysamyauthor=S.+Wassermanauthor=S.+B.+Jonesauthor=F.+F.+Zhangauthor=A.+Espadaauthor=H.+Broughtonauthor=M.+J.+Chalmersauthor=M.+E.+Woodmanauthor=H.+A.+Binaauthor=J.+A.+Dodgeauthor=J.+Benachauthor=A.+Zhangauthor=C.+Groshongauthor=D.+Manglicmotauthor=M.+Russellauthor=S.+Afshar&title=Utilization+of+peptide+phage+display+to+investigate+hotspots+on+IL-17A+and+what+it+means+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0190850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0190850%26sid%3Dliteratum%253Aachs%26aulast%3DTing%26aufirst%3DJ.%2BP.%26aulast%3DTung%26aufirst%3DF.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWasserman%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DS.%2BB.%26aulast%3DZhang%26aufirst%3DF.%2BF.%26aulast%3DEspada%26aufirst%3DA.%26aulast%3DBroughton%26aufirst%3DH.%26aulast%3DChalmers%26aufirst%3DM.%2BJ.%26aulast%3DWoodman%26aufirst%3DM.%2BE.%26aulast%3DBina%26aufirst%3DH.%2BA.%26aulast%3DDodge%26aufirst%3DJ.%2BA.%26aulast%3DBenach%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DGroshong%26aufirst%3DC.%26aulast%3DManglicmot%26aufirst%3DD.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DAfshar%26aufirst%3DS.%26atitle%3DUtilization%2520of%2520peptide%2520phage%2520display%2520to%2520investigate%2520hotspots%2520on%2520IL-17A%2520and%2520what%2520it%2520means%2520for%2520drug%2520discovery%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26spage%3De0190850%252F1%26epage%3De0190850%252F18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span>. <i>Drug Discov. Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lgehhYlbS44IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discov.%2520Today%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alihodžić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukvić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenkov, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutinec, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pešić, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašković, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žiher, D.</span></span> <span> </span><span class="NLM_article-title">Current trends in macrocyclic drug discovery and beyond-Ro5</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1016/bs.pmch.2018.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fbs.pmch.2018.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=29680148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=113-233&author=S.+Alihod%C5%BEi%C4%87author=M.+Bukvi%C4%87author=I.+J.+Elenkovauthor=A.+Hutinecauthor=S.+Ko%C5%A1trunauthor=D.+Pe%C5%A1i%C4%87author=G.+Saxtyauthor=L.+Toma%C5%A1kovi%C4%87author=D.+%C5%BDiher&title=Current+trends+in+macrocyclic+drug+discovery+and+beyond-Ro5&doi=10.1016%2Fbs.pmch.2018.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Macrocyclic Drug Discovery and beyond-Ro5</span></div><div class="casAuthors">Alihodzic Sulejman; Bukvic Mirjana; Elenkov Ivaylo J; Hutinec Antun; Kostrun Sanja; Pesic Dijana; Saxty Gordon; Tomaskovic Linda; Ziher Dinko</div><div class="citationInfo"><span class="NLM_cas:title">Progress in medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-233</span>
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    </div><div class="casAbstract">This chapter will discuss the recent literature of macrocycles and drug-like property space moving beyond the rule of five (bRo5).  Trends in chemical classes that fall within this definition are discussed and the impact of the latest technologies in the field assessed.  The physicochemical properties, which have provided both successes and challenges, especially in scale-up, are discussed.  A recent patent literature is reviewed and the chapter concludes with a perspective on the future of macrocyclic drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZ9pfcWWVZ5hWfpMXQLRstfW6udTcc2ebuUbBoOqTmt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D&md5=ac41d1a69cb0056d106d554b0b323e61</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fbs.pmch.2018.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.pmch.2018.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DS.%26aulast%3DBukvi%25C4%2587%26aufirst%3DM.%26aulast%3DElenkov%26aufirst%3DI.%2BJ.%26aulast%3DHutinec%26aufirst%3DA.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DPe%25C5%25A1i%25C4%2587%26aufirst%3DD.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DToma%25C5%25A1kovi%25C4%2587%26aufirst%3DL.%26aulast%3D%25C5%25BDiher%26aufirst%3DD.%26atitle%3DCurrent%2520trends%2520in%2520macrocyclic%2520drug%2520discovery%2520and%2520beyond-Ro5%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D57%26spage%3D113%26epage%3D233%26doi%3D10.1016%2Fbs.pmch.2018.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stotani, S</span></span>, <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span> <span> </span><span class="NLM_article-title">Overview of Macrocycles in Clinical Development and Clinically Used</span>. In:  <i>Practical Medicinal Chemistry with Macrocycles: Design, Synthesis, and Case Studies</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span>; <span class="NLM_string-name">Peterson, M. L.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">Hoboken</span>, <span class="NLM_year">2017</span>; pp.  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">500</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2F9781119092599.ch16" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=413-500&author=S+Stotaniauthor=F.+Giordanettoauthor=E.+Marsault&author=M.+L.+Peterson&title=Practical+Medicinal+Chemistry+with+Macrocycles%3A+Design%2C+Synthesis%2C+and+Case+Studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F9781119092599.ch16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781119092599.ch16%26sid%3Dliteratum%253Aachs%26aulast%3DStotani%26aufirst%3DS%26aulast%3DGiordanetto%26aufirst%3DF.%26atitle%3DOverview%2520of%2520Macrocycles%2520in%2520Clinical%2520Development%2520and%2520Clinically%2520Used%26btitle%3DPractical%2520Medicinal%2520Chemistry%2520with%2520Macrocycles%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Case%2520Studies%26aulast%3DMarsault%26aufirst%3DE.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2017%26spage%3D413%26epage%3D500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seiple, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubec, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois-Mercier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hog, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuzaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsen, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Condakes, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczypiński, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, A. G.</span></span> <span> </span><span class="NLM_article-title">A platform for the discovery of new macrolide antibiotics</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">338</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1038/nature17967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fnature17967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27193679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xot1SmtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=338-345&author=I.+B.+Seipleauthor=Z.+Zhangauthor=P.+Jakubecauthor=A.+Langlois-Mercierauthor=P.+M.+Wrightauthor=D.+T.+Hogauthor=K.+Yabuauthor=S.+R.+Alluauthor=T.+Fukuzakiauthor=P.+N.+Carlsenauthor=Y.+Kitamuraauthor=X.+Zhouauthor=M.+L.+Condakesauthor=F.+T.+Szczypi%C5%84skiauthor=W.+D.+Greenauthor=A.+G.+Myers&title=A+platform+for+the+discovery+of+new+macrolide+antibiotics&doi=10.1038%2Fnature17967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A platform for the discovery of new macrolide antibiotics</span></div><div class="casAuthors">Seiple, Ian B.; Zhang, Ziyang; Jakubec, Pavol; Langlois-Mercier, Audrey; Wright, Peter M.; Hog, Daniel T.; Yabu, Kazuo; Allu, Senkara Rao; Fukuzaki, Takehiro; Carlsen, Peter N.; Kitamura, Yoshiaki; Zhou, Xiang; Condakes, Matthew L.; Szczypinski, Filip T.; Green, William D.; Myers, Andrew G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7603</span>),
    <span class="NLM_cas:pages">338-345</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The chem. modification of structurally complex fermn. products, a process known as semisynthesis, has been an important tool in the discovery and manuf. of antibiotics for the treatment of various infectious diseases.  However, many of the therapeutics obtained in this way are no longer effective, because bacterial resistance to these compds. has developed.  Here, we present a practical, fully synthetic route to macrolide antibiotics by the convergent assembly of simple chem. building blocks, enabling the synthesis of diverse structures not accessible by traditional semisynthetic approaches.  More than 300 new macrolide antibiotic candidates, as well as the clin. candidate solithromycin, have been synthesized using our convergent approach.  Evaluation of these compds. against a panel of pathogenic bacteria revealed that the majority of these structures had antibiotic activity, some efficacious against strains resistant to macrolides in current use.  The chem. we describe here provides a platform for the discovery of new macrolide antibiotics and may also serve as the basis for their manuf.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGXBS8IF57cbVg90H21EOLACvtfcHk0lgmt7Mcz0szMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xot1SmtLw%253D&md5=be2af0c211fdc10ef5f1203833bd6c9b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature17967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17967%26sid%3Dliteratum%253Aachs%26aulast%3DSeiple%26aufirst%3DI.%2BB.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DJakubec%26aufirst%3DP.%26aulast%3DLanglois-Mercier%26aufirst%3DA.%26aulast%3DWright%26aufirst%3DP.%2BM.%26aulast%3DHog%26aufirst%3DD.%2BT.%26aulast%3DYabu%26aufirst%3DK.%26aulast%3DAllu%26aufirst%3DS.%2BR.%26aulast%3DFukuzaki%26aufirst%3DT.%26aulast%3DCarlsen%26aufirst%3DP.%2BN.%26aulast%3DKitamura%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DCondakes%26aufirst%3DM.%2BL.%26aulast%3DSzczypi%25C5%2584ski%26aufirst%3DF.%2BT.%26aulast%3DGreen%26aufirst%3DW.%2BD.%26aulast%3DMyers%26aufirst%3DA.%2BG.%26atitle%3DA%2520platform%2520for%2520the%2520discovery%2520of%2520new%2520macrolide%2520antibiotics%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D338%26epage%3D345%26doi%3D10.1038%2Fnature17967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclic peptide therapeutics: past, present and future</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.cbpa.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=28249193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=24-29&author=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclic+peptide+therapeutics%3A+past%2C+present+and+future&doi=10.1016%2Fj.cbpa.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide therapeutics: past, present and future</span></div><div class="casAuthors">Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-29</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic peptides combine several favorable properties such as good binding affinity, target selectivity and low toxicity that make them an attractive modality for the development of therapeutics.  Over 40 cyclic peptide drugs are currently in clin. use and around one new cyclic peptide drug enters the market every year on av.  The vast majority of clin. approved cyclic peptides are derived from natural products, such as antimicrobials or human peptide hormones.  New powerful techniques based on rational design and in vitro evolution have enabled the de novo development of cyclic peptide ligands to targets for which nature does not offer solns.  A look at the cyclic peptides currently under clin. evaluation shows that several have been developed using such techniques.  This new source for cyclic peptide ligands introduces a freshness to the field, and it is likely that de novo developed cyclic peptides will be in clin. use in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwchcMOj1bSbVg90H21EOLACvtfcHk0lgjMdA_wgNBlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D&md5=4b9581365b92b649dad8c7b22c5bd9c3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclic%2520peptide%2520therapeutics%253A%2520past%252C%2520present%2520and%2520future%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D24%26epage%3D29%26doi%3D10.1016%2Fj.cbpa.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emery, F. S.</span></span> <span> </span><span class="NLM_article-title">Oral administration of peptide-based drugs: beyond Lipinski’s rule</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2245</span>– <span class="NLM_lpage">2251</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2Fcmdc.201600288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27596610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2245-2251&author=G.+B.+Santosauthor=A.+Ganesanauthor=F.+S.+Emery&title=Oral+administration+of+peptide-based+drugs%3A+beyond+Lipinski%E2%80%99s+rule&doi=10.1002%2Fcmdc.201600288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Administration of Peptide-Based Drugs: Beyond Lipinski's Rule</span></div><div class="casAuthors">Santos, Gabriela B.; Ganesan, A.; Emery, Flavio S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2245-2251</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The use of peptides in therapy presents several limitations, from physicochem. characteristics to inadequate pharmacokinetic profiles for oral absorption.  As peptides are gaining importance in the therapeutic arsenal, there is an increasing need to rationalize the main characteristics of this compd. class in the market.  Therefore, we performed an extensive anal. of all known peptide drugs and clin. candidates based on their peptide features, physicochem. and structural properties, and correlated these with their administration route and therapeutic classes.  Peptide drugs are widely distributed across drug and pharmacol. space, covering several therapeutic areas with structural diversity and complexity, distributed between groups of cyclic and linear compds.  Although structural and physicochem. properties are clear within these groups, we counter the consensus that cyclic peptides have better oral availability than linear peptides, as most of the orally administrated peptides have linear structures.  This study and review furnishes information that could support peptide drug design, with a new cutoff of known descriptors that go beyond the Rule of Five.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr88nw1qqAJuLVg90H21EOLACvtfcHk0ljD5_DTm1XGKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI&md5=b2e5b6a1dff54c2c1f85671ab7314031</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600288%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DG.%2BB.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DEmery%26aufirst%3DF.%2BS.%26atitle%3DOral%2520administration%2520of%2520peptide-based%2520drugs%253A%2520beyond%2520Lipinski%25E2%2580%2599s%2520rule%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2245%26epage%3D2251%26doi%3D10.1002%2Fcmdc.201600288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fajdetić, A.</span>; <span class="NLM_string-name">Matanović
Škugor, M.</span>; <span class="NLM_string-name">Elenkov, I.</span>; <span class="NLM_string-name">Kragol, G.</span>; <span class="NLM_string-name">Bukvić, M.</span>; <span class="NLM_string-name">Marušić Ištuk, Z.</span>; <span class="NLM_string-name">Koštrun, S.</span>; <span class="NLM_string-name">Žiher, D.</span>; <span class="NLM_string-name">Rupčić, R.</span>; <span class="NLM_string-name">Butković, K.</span>, <span class="NLM_string-name">Dukši, M.</span>; <span class="NLM_string-name">Ozimec Landek, I.</span>; <span class="NLM_string-name">Pešić, D.</span>; <span class="NLM_string-name">Hutinec, A.</span>; <span class="NLM_string-name">Mesić, M.</span>; <span class="NLM_string-name">Saxty, G.</span>; <span class="NLM_string-name">Poljak, V.</span></span> <span> </span><span class="NLM_article-title">Preparation of Seco Macrolide Aminoglycosides for Use in the Treatment of Prophylaxis</span>.  <i>Int. Pat. Appl.</i> <span class="NLM_patent">WO2017194452A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=A.+Fajdeti%C4%87&author=M.+Matanovi%C4%87%0A%C5%A0kugor&author=I.+Elenkov&author=G.+Kragol&author=M.+Bukvi%C4%87&author=Z.+Maru%C5%A1i%C4%87+I%C5%A1tuk&author=S.+Ko%C5%A1trun&author=D.+%C5%BDiher&author=R.+Rup%C4%8Di%C4%87&author=K.+Butkovi%C4%87&author=M.+Duk%C5%A1i&author=I.+Ozimec+Landek&author=D.+Pe%C5%A1i%C4%87&author=A.+Hutinec&author=M.+Mesi%C4%87&author=G.+Saxty&author=V.+Poljak&title=Preparation+of+Seco+Macrolide+Aminoglycosides+for+Use+in+the+Treatment+of+Prophylaxis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFajdeti%25C4%2587%26aufirst%3DA.%26atitle%3DPreparation%2520of%2520Seco%2520Macrolide%2520Aminoglycosides%2520for%2520Use%2520in%2520the%2520Treatment%2520of%2520Prophylaxis%26jtitle%3DInt.%2520Pat.%2520Appl.%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrunyk, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marr, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffor, M. C.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of interleukin 17A and its complex with IL-17 receptor A</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1888</span>, <span class="refDoi"> DOI: 10.1038/ncomms2880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fncomms2880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=23695682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A280%3ADC%252BC3snlvFCrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1888&author=S.+Liuauthor=X.+Songauthor=B.+A.+Chrunykauthor=S.+Shankerauthor=L.+R.+Hothauthor=E.+S.+Marrauthor=M.+C.+Griffor&title=Crystal+structures+of+interleukin+17A+and+its+complex+with+IL-17+receptor+A&doi=10.1038%2Fncomms2880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of interleukin 17A and its complex with IL-17 receptor A</span></div><div class="casAuthors">Liu Shenping; Song Xi; Chrunyk Boris A; Shanker Suman; Hoth Lise R; Marr Eric S; Griffor Matthew C</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1888</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The constituent polypeptides of the interleukin-17 family form six different homodimeric cytokines (IL-17A-F) and the heterodimeric IL-17A/F.  Their interactions with IL-17 receptors A-E (IL-17RA-E) mediate host defenses while also contributing to inflammatory and autoimmune responses.  IL-17A and IL-17F both preferentially engage a receptor complex containing one molecule of IL-17RA and one molecule of IL-17RC.  More generally, IL-17RA appears to be a shared receptor that pairs with other members of its family to allow signaling of different IL-17 cytokines.  Here we report crystal structures of homodimeric IL-17A and its complex with IL-17RA.  Binding to IL-17RA at one side of the IL-17A molecule induces a conformational change in the second, symmetry-related receptor site of IL-17A.  This change favors, and is sufficient to account for, the selection of a different receptor polypeptide to complete the cytokine-receptor complex.  The structural results are supported by biophysical studies with IL-17A variants produced by site-directed mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQPgY8GEu_hzSSm0e6ZOaemfW6udTcc2eZoPXbBAU2qvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snlvFCrsw%253D%253D&md5=18a5bb02c1f0389b7e02277d0860c2e2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fncomms2880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2880%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DChrunyk%26aufirst%3DB.%2BA.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DMarr%26aufirst%3DE.%2BS.%26aulast%3DGriffor%26aufirst%3DM.%2BC.%26atitle%3DCrystal%2520structures%2520of%2520interleukin%252017A%2520and%2520its%2520complex%2520with%2520IL-17%2520receptor%2520A%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D1888%26doi%3D10.1038%2Fncomms2880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerhardt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauptit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Needham, M. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langham, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aziz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snow, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welsh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaugan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beste, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sleeman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coales, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamuro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, C.</span></span> <span> </span><span class="NLM_article-title">Structure of IL-17A in complex with a potent, fully human neutralizing antibody</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.jmb.2009.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=19835883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKnurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2009&pages=905-921&author=S.+Gerhardtauthor=W.+M.+Abbottauthor=D.+Hargreavesauthor=R.+A.+Pauptitauthor=R.+A.+Daviesauthor=M.+R.+C.+Needhamauthor=C.+Langhamauthor=W.+Barkerauthor=A.+Azizauthor=M.+J.+Snowauthor=S.+Dawsonauthor=F.+Welshauthor=T.+Wilkinsonauthor=T.+Vauganauthor=G.+Besteauthor=S.+Bishopauthor=B.+Popovicauthor=G.+Reesauthor=M.+Sleemanauthor=S.+J.+Tuskeauthor=S.+J.+Coalesauthor=Y.+Hamuroauthor=C.+Russell&title=Structure+of+IL-17A+in+complex+with+a+potent%2C+fully+human+neutralizing+antibody&doi=10.1016%2Fj.jmb.2009.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of IL-17A in Complex with a Potent, Fully Human Neutralizing Antibody</span></div><div class="casAuthors">Gerhardt, Stefan; Abbott, W. Mark; Hargreaves, David; Pauptit, Richard A.; Davies, Rick A.; Needham, Maurice R. C.; Langham, Caroline; Barker, Wendy; Aziz, Azad; Snow, Melanie J.; Dawson, Sarah; Welsh, Fraser; Wilkinson, Trevor; Vaugan, Tris; Beste, Gerald; Bishop, Sarah; Popovic, Bojana; Rees, Gareth; Sleeman, Matthew; Tuske, Steven J.; Coales, Stephen J.; Hamuro, Yoshitomo; Russell, Caroline</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">905-921</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">IL-17A is a pro-inflammatory cytokine produced by the newly identified Th17 subset of T-cells.  We have isolated a human monoclonal antibody to IL-17A (CAT-2200) that can potently neutralize the effects of recombinant and native human IL-17A.  We detd. the crystal structure of IL-17A in complex with the CAT-2200 Fab at 2.6 Å resoln. in order to provide a definitive characterization of the epitope and paratope regions.  Approx. a third of the IL-17A dimer is disordered in this crystal structure.  The disorder occurs in both independent copies of the complex in the asym. unit and does not appear to be influenced by crystal packing.  The complex contains one IL-17A dimer sandwiched between two CAT-2200 Fab fragments.  The IL-17A is a disulfide-linked homodimer that is similar in structure to IL-17F, adopting a cystine-knot fold.  The structure is not inconsistent with the previous prediction of a receptor binding cavity on IL-17 family members.  The epitope recognized by CAT-2200 is shown to involve 12 amino acid residues from the quaternary structure of IL-17A, with each Fab contacting both monomers in the dimer.  All complementarity-detg. regions (CDRs) in the Fab contribute to a total of 16 amino acid residues in the antibody paratope.  In vitro affinity optimization was used to generate CAT-2200 from a parental lead antibody using random mutagenesis of CDR3 loops.  This resulted in seven amino acid changes (three in VL-CDR3 and four in VH-CDR3) and gave an approx. 30-fold increase in potency in a cell-based neutralization assay.  Two of the seven amino acids form part of the CAT-2200 paratope.  The obsd. interaction site between CAT-2200 and IL-17A is consistent with data from hydrogen/deuterium exchange mass spectrometry and mutagenesis approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bLLa1mvN07Vg90H21EOLACvtfcHk0lgYP75W4FF2_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKnurvI&md5=c79a71a621832d30cf705ba70563096f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DAbbott%26aufirst%3DW.%2BM.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DPauptit%26aufirst%3DR.%2BA.%26aulast%3DDavies%26aufirst%3DR.%2BA.%26aulast%3DNeedham%26aufirst%3DM.%2BR.%2BC.%26aulast%3DLangham%26aufirst%3DC.%26aulast%3DBarker%26aufirst%3DW.%26aulast%3DAziz%26aufirst%3DA.%26aulast%3DSnow%26aufirst%3DM.%2BJ.%26aulast%3DDawson%26aufirst%3DS.%26aulast%3DWelsh%26aufirst%3DF.%26aulast%3DWilkinson%26aufirst%3DT.%26aulast%3DVaugan%26aufirst%3DT.%26aulast%3DBeste%26aufirst%3DG.%26aulast%3DBishop%26aufirst%3DS.%26aulast%3DPopovic%26aufirst%3DB.%26aulast%3DRees%26aufirst%3DG.%26aulast%3DSleeman%26aufirst%3DM.%26aulast%3DTuske%26aufirst%3DS.%2BJ.%26aulast%3DCoales%26aufirst%3DS.%2BJ.%26aulast%3DHamuro%26aufirst%3DY.%26aulast%3DRussell%26aufirst%3DC.%26atitle%3DStructure%2520of%2520IL-17A%2520in%2520complex%2520with%2520a%2520potent%252C%2520fully%2520human%2520neutralizing%2520antibody%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D394%26spage%3D905%26epage%3D921%26doi%3D10.1016%2Fj.jmb.2009.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desharnais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oates, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrunyk, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumelty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodnutt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lappe, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, L. D.</span></span> <span> </span><span class="NLM_article-title">Inhibiting complex IL-17A and IL-17RA interactions with a linear peptide</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">26071</span>, <span class="refDoi"> DOI: 10.1038/srep26071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fsrep26071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27184415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xot1Gnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=26071&author=S.+Liuauthor=J.+Desharnaisauthor=P.+V.+Sahasrabudheauthor=P.+Jinauthor=W.+Liauthor=B.+D.+Oatesauthor=S.+Shankerauthor=M.+E.+Bankerauthor=B.+A.+Chrunykauthor=X.+Songauthor=X.+Fengauthor=M.+Grifforauthor=J.+Jimenezauthor=G.+Chenauthor=D.+Tumeltyauthor=A.+Bhatauthor=C.+W.+Bradshawauthor=G.+Woodnuttauthor=R.+W.+Lappeauthor=A.+Thorarensenauthor=X.+Qiuauthor=J.+M.+Withkaauthor=L.+D.+Wood&title=Inhibiting+complex+IL-17A+and+IL-17RA+interactions+with+a+linear+peptide&doi=10.1038%2Fsrep26071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting complex IL-17A and IL-17RA interactions with a linear peptide</span></div><div class="casAuthors">Liu, Shenping; Desharnais, Joel; Sahasrabudhe, Parag V.; Jin, Ping; Li, Wei; Oates, Bryan D.; Shanker, Suman; Banker, Mary Ellen; Chrunyk, Boris A.; Song, Xi; Feng, Xidong; Griffor, Matt; Jimenez, Judith; Chen, Gang; Tumelty, David; Bhat, Abhijit; Bradshaw, Curt W.; Woodnutt, Gary; Lappe, Rodney W.; Thorarensen, Atli; Qiu, Xiayang; Withka, Jane M.; Wood, Lauren D.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26071</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">IL-17A is a pro-inflammatory cytokine that has been implicated in autoimmune and inflammatory diseases.  Monoclonal antibodies inhibiting IL-17A signaling have demonstrated remarkable efficacy, but an oral therapy is still lacking.  A high affinity IL-17A peptide antagonist (HAP) of 15 residues was identified through phage-display screening followed by satn. mutagenesis optimization and amino acid substitutions.  HAP binds specifically to IL-17A and inhibits the interaction of the cytokine with its receptor, IL-17RA.  Tested in primary human cells, HAP blocked the prodn. of multiple inflammatory cytokines.  Crystal structure studies revealed that two HAP mols. bind to one IL-17A dimer sym.  The N-terminal portions of HAP form a β-strand that inserts between two IL-17A monomers while the C-terminal section forms an α helix that directly blocks IL-17RA from binding to the same region of IL-17A.  This mode of inhibition suggests opportunities for developing peptide antagonists against this challenging target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLkbdJqT9hyrVg90H21EOLACvtfcHk0ljmeaTVJ6fdPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xot1Gnurg%253D&md5=73da55aa47ada18bcb1820b99969f568</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsrep26071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep26071%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDesharnais%26aufirst%3DJ.%26aulast%3DSahasrabudhe%26aufirst%3DP.%2BV.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DOates%26aufirst%3DB.%2BD.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DChrunyk%26aufirst%3DB.%2BA.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DGriffor%26aufirst%3DM.%26aulast%3DJimenez%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DTumelty%26aufirst%3DD.%26aulast%3DBhat%26aufirst%3DA.%26aulast%3DBradshaw%26aufirst%3DC.%2BW.%26aulast%3DWoodnutt%26aufirst%3DG.%26aulast%3DLappe%26aufirst%3DR.%2BW.%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWithka%26aufirst%3DJ.%2BM.%26aulast%3DWood%26aufirst%3DL.%2BD.%26atitle%3DInhibiting%2520complex%2520IL-17A%2520and%2520IL-17RA%2520interactions%2520with%2520a%2520linear%2520peptide%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D26071%26doi%3D10.1038%2Fsrep26071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanikawa, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis and antibacterial activity of a novel class of 15-membered macrolide antibiotics, “11a-azalides.”</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1021/ml100252s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100252s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVan" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=234-237&author=T.+Sugimotoauthor=T.+Tanikawa&title=Synthesis+and+antibacterial+activity+of+a+novel+class+of+15-membered+macrolide+antibiotics%2C+%E2%80%9C11a-azalides.%E2%80%9D&doi=10.1021%2Fml100252s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and antibacterial activity of a novel class of 15-membered macrolide antibiotics, "11a-azalides"</span></div><div class="casAuthors">Sugimoto, Tomohiro; Tanikawa, Tetsuya</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-237</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient method for the reconstruction of the 9-dihydroerythromycin A macrolactone skeleton has been established.  The key steps are oxidative cleavage at the 11,12-position and reconstruction after insertion of an appropriate functionalized amino alc.  Novel 15-membered macrolides, named as "11a-azalides", were synthesized based on the above methodol. and evaluated for their antibacterial activity.  Among them, (13R)-benzyloxymethyl-11a-azalide showed the most potent Streptococcus pneumoniae activity, with improved activity against a representative erythromycin-resistant strain compared to clarithromycin (CAM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7VwCBoxE6BLVg90H21EOLACvtfcHk0ljJE23qgASxBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVan&md5=45025a54eb891d2a3d861cbf26a823b3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fml100252s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100252s%26sid%3Dliteratum%253Aachs%26aulast%3DSugimoto%26aufirst%3DT.%26aulast%3DTanikawa%26aufirst%3DT.%26atitle%3DSynthesis%2520and%2520antibacterial%2520activity%2520of%2520a%2520novel%2520class%2520of%252015-membered%2520macrolide%2520antibiotics%252C%2520%25E2%2580%259C11a-azalides.%25E2%2580%259D%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D234%26epage%3D237%26doi%3D10.1021%2Fml100252s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Everett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, J. W.</span></span> <span> </span><span class="NLM_article-title">The conformational analysis of erythromycin A</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1659</span>– <span class="NLM_lpage">1667</span>, <span class="refDoi"> DOI: 10.1039/P29870001659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1039%2Fp29870001659" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1987&pages=1659-1667&author=J.+R.+Everettauthor=J.+W.+Tyler&title=The+conformational+analysis+of+erythromycin+A&doi=10.1039%2FP29870001659"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2Fp29870001659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29870001659%26sid%3Dliteratum%253Aachs%26aulast%3DEverett%26aufirst%3DJ.%2BR.%26aulast%3DTyler%26aufirst%3DJ.%2BW.%26atitle%3DThe%2520conformational%2520analysis%2520of%2520erythromycin%2520A%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1987%26volume%3D11%26spage%3D1659%26epage%3D1667%26doi%3D10.1039%2FP29870001659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazarevski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinković, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobrehel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Đokić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metelko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikić-Topić, D.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of azithromycin by NMR spectroscopy and molecular modeling</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/S0040-4020(01)86274-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2FS0040-4020%2801%2986274-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADyaK3sXht1Cru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1993&pages=721-730&author=G.+Lazarevskiauthor=M.+Vinkovi%C4%87author=G.+Kobrehelauthor=S.+%C4%90oki%C4%87author=B.+Metelkoauthor=D.+Viki%C4%87-Topi%C4%87&title=Conformational+analysis+of+azithromycin+by+NMR+spectroscopy+and+molecular+modeling&doi=10.1016%2FS0040-4020%2801%2986274-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of azithromycin by NMR spectroscopy and molecular modeling</span></div><div class="casAuthors">Lazarevski, Gorjana; Vinkovic, Mladen; Kobrehel, Gabrijela; Dokic, Slobodan; Metelko, Biserka; Vikic-Topic, Drazen</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">721-30</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">The conformation of azithromycin (I) in the soln. was detd. by NMR spectroscopy and mol. mechanics calcns. and compared with its crystal structure and with some erythromycin derivs.  In soln. I exists predominantly in a "folded-in" conformation in the C(3) to C(5) region, whereas its crystal state conformation is "folded-out".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruw1FZO-XoOLVg90H21EOLACvtfcHk0lhDnEchv7bfCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXht1Cru7s%253D&md5=8b346c265609627263d82b19523953e2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2801%2986274-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252801%252986274-8%26sid%3Dliteratum%253Aachs%26aulast%3DLazarevski%26aufirst%3DG.%26aulast%3DVinkovi%25C4%2587%26aufirst%3DM.%26aulast%3DKobrehel%26aufirst%3DG.%26aulast%3D%25C4%2590oki%25C4%2587%26aufirst%3DS.%26aulast%3DMetelko%26aufirst%3DB.%26aulast%3DViki%25C4%2587-Topi%25C4%2587%26aufirst%3DD.%26atitle%3DConformational%2520analysis%2520of%2520azithromycin%2520by%2520NMR%2520spectroscopy%2520and%2520molecular%2520modeling%26jtitle%3DTetrahedron%26date%3D1993%26volume%3D49%26spage%3D721%26epage%3D730%26doi%3D10.1016%2FS0040-4020%2801%2986274-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span> <span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1784</span>– <span class="NLM_lpage">1788</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lhDnEchv7bfCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stepanić, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žiher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica-Marković, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jelić, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span> <span> </span><span class="NLM_article-title">Physicochemical profile of macrolides and their comparison with small molecules</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2011.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.ejmech.2011.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=22152985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVyqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=462-472&author=V.+Stepani%C4%87author=D.+%C5%BDiherauthor=V.+Gabelica-Markovi%C4%87author=D.+Jeli%C4%87author=S.+Nunhuckauthor=K.+Valkoauthor=S.+Ko%C5%A1trun&title=Physicochemical+profile+of+macrolides+and+their+comparison+with+small+molecules&doi=10.1016%2Fj.ejmech.2011.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical profile of macrolides and their comparison with small molecules</span></div><div class="casAuthors">Stepanic, Visnja; Ziher, Dinko; Gabelica-Markovic, Vesna; Jelic, Dubravko; Nunhuck, Shenaz; Valko, Klara; Kostrun, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">462-472</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Macrolides are stereospecific macrolactones of high mol. wts.  Herein, 600 mostly semisynthetic macrolides are compared with 50,000 small non-macrolide synthetic mols. in terms of measured physicochem. properties in order to assess the drug-likeness and developability chances of macrolides.  The pre-selected set of diverse macrolides is comprised mostly of derivs. of clarithromycin and azithromycin cores.  Lipophilicity (CHI logD), affinity for immobilized artificial membranes (CHI IAM), human serum albumin (HSA) and α1-acid glycoprotein (AGP) plasma protein bindings (PPB), DMSO precipitative soly. as well as artificial membrane permeability (AMP) have been detd. by high-throughput screening methods.  It has been found that macrolides and small mols. have similar lipophilicity profiles, though macrolides show weaker PPB and have better soly. than small discovery mols.  However, macrolides are poorly permeable and have high affinity for immobilized artificial membranes signifying their strong interaction with biol. phospholipids.  In order to retain the drug-like profile, the design of novel macrolide mols. should be focused on optimization of macrolide cores, that is macrolactone moiety with sugars and other small substituents avoiding large substituents and flexible linkers such as in conjugate derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpyTdevkgZsrVg90H21EOLACvtfcHk0ljB2thoquWG5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVyqsQ%253D%253D&md5=203aec4ec40cecb71feaf36345d2fe4d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2011.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2011.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DStepani%25C4%2587%26aufirst%3DV.%26aulast%3D%25C5%25BDiher%26aufirst%3DD.%26aulast%3DGabelica-Markovi%25C4%2587%26aufirst%3DV.%26aulast%3DJeli%25C4%2587%26aufirst%3DD.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26atitle%3DPhysicochemical%2520profile%2520of%2520macrolides%2520and%2520their%2520comparison%2520with%2520small%2520molecules%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D47%26spage%3D462%26epage%3D472%26doi%3D10.1016%2Fj.ejmech.2011.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lh2e9q4tm58RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lh2e9q4tm58RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahlbach, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Beyond cyclosporine A: conformation-dependent passive membrane permeabilities of cyclic peptide natural products</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2121</span>– <span class="NLM_lpage">2130</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.78</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.4155%2Ffmc.15.78" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=26067057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVehs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=2121-2130&author=C.+L.+Ahlbachauthor=K.+W.+Lexaauthor=A.+T.+Bockusauthor=V.+Chenauthor=P.+Crewsauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Beyond+cyclosporine+A%3A+conformation-dependent+passive+membrane+permeabilities+of+cyclic+peptide+natural+products&doi=10.4155%2Ffmc.15.78"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond cyclosporine A: conformation-dependent passive membrane permeabilities of cyclic peptide natural products</span></div><div class="casAuthors">Ahlbach, Christopher L.; Lexa, Katrina W.; Bockus, Andrew T.; Chen, Valerie; Crews, Phillip; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2121-2130</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Many cyclic peptide natural products are larger and structurally more complex than conventional small mol. drugs.  Although some mols. in this class are known to possess favorable pharmacokinetic properties, there have been few reports on the membrane permeabilities of cyclic peptide natural products.  Here, we present the passive membrane permeabilities of 39 cyclic peptide natural products, and interpret the results using a computational permeability prediction algorithm based on their known or calcd. 3D conformations.  We found that the permeabilities of these compds., measured in a parallel artificial membrane permeability assay, spanned a wide range and demonstrated the important influence of conformation on membrane permeability.  These results will aid in the development of these compds. as a viable drug paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4_Ar_CGW03bVg90H21EOLACvtfcHk0lgxEjuAnU5NwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVehs7%252FM&md5=69d8628920ebc6ad72b13162f3214f2d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.78%26sid%3Dliteratum%253Aachs%26aulast%3DAhlbach%26aufirst%3DC.%2BL.%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DV.%26aulast%3DCrews%26aufirst%3DP.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DBeyond%2520cyclosporine%2520A%253A%2520conformation-dependent%2520passive%2520membrane%2520permeabilities%2520of%2520cyclic%2520peptide%2520natural%2520products%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D2121%26epage%3D2130%26doi%3D10.4155%2Ffmc.15.78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cell-permeable cyclic peptides from synthetic libraries inspired by natural products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1021/ja508766b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja508766b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=715-721&author=W.+M.+Hewittauthor=S.+S.+F.+Leungauthor=C.+R.+Pyeauthor=A.+R.+Ponkeyauthor=M.+Bednarekauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Cell-permeable+cyclic+peptides+from+synthetic+libraries+inspired+by+natural+products&doi=10.1021%2Fja508766b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span></div><div class="casAuthors">Hewitt, William M.; Leung, Siegfried S. F.; Pye, Cameron R.; Ponkey, Alexandra R.; Bednarek, Maria; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">715-721</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug design efforts are turning to a new generation of therapeutic targets, such as protein-protein interactions (PPIs), that had previously been considered "undruggable" by typical small mols.  There is an emerging view that accessing these targets will require mols. that are larger and more complex than typical small mol. drugs.  Here, the authors present a methodol. for the discovery of geometrically diverse, membrane permeable cyclic peptide scaffolds based on the synthesis and permeability screening of a combinatorial library, followed by deconvolution of membrane-permeable scaffolds to identify cyclic peptides with good to excellent passive cell permeabilities.  The authors use a combination of exptl. and computational approaches to investigate structure-permeability relationships in one of these scaffolds, and uncover structural and conformational factors that govern passive membrane diffusion in a related set of cyclic peptide diastereomers.  Further, the authors investigate the dependency of permeability on side-chain identity of one of these scaffolds through single-point diversifications to show the adaptability of these scaffolds toward development of permeability-biased libraries suitable for bioactivity screens.  Overall, the results demonstrate that many novel, cell permeable scaffolds exist beyond those found in extant natural products, and that such scaffolds can be rapidly identified using a combination of synthesis and deconvolution which can, in principle, be applied to any type of macrocyclic template.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wbqbKaUG9rVg90H21EOLACvtfcHk0liCU9Ipr6xR4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF&md5=0d112bf0da119b6a50b55997f64ac214</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja508766b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja508766b%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DBednarek%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCell-permeable%2520cyclic%2520peptides%2520from%2520synthetic%2520libraries%2520inspired%2520by%2520natural%2520products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D715%26epage%3D721%26doi%3D10.1021%2Fja508766b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poullennec, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakemore, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiva, J.-Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Highton, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoisin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9473</span>– <span class="NLM_lpage">9499</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00802</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00802" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9473-9499&author=R.+L.+Mackmanauthor=V.+A.+Steadmanauthor=D.+K.+Deanauthor=P.+Jansaauthor=K.+G.+Poullennecauthor=T.+Applebyauthor=C.+Austinauthor=C.+A.+Blakemoreauthor=R.+Caiauthor=C.+Cannizzaroauthor=G.+Chinauthor=J.-Y.+C.+Chivaauthor=N.+A.+Dunbarauthor=H.+Fliriauthor=A.+J.+Hightonauthor=H.+Huiauthor=M.+Jiauthor=H.+Jinauthor=K.+Karkiauthor=A.+J.+Keatsauthor=L.+Lazaridesauthor=Y.-J.+Leeauthor=A.+Liclicanauthor=M.+Mishauthor=B.+Murrayauthor=S.+B.+Pettitauthor=P.+Pyunauthor=M.+Sangiauthor=R.+Santosauthor=J.+Sanvoisinauthor=U.+Schmitzauthor=A.+Schrierauthor=D.+Siegelauthor=D.+Sperandioauthor=G.+Stepanauthor=Y.+Tianauthor=G.+M.+Wattauthor=H.+Yangauthor=B.+E.+Schultz&title=Discovery+of+a+potent+and+orally+bioavailable+cyclophilin+inhibitor+derived+from+the+sanglifehrin+macrocycle&doi=10.1021%2Facs.jmedchem.8b00802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span></div><div class="casAuthors">Mackman, Richard L.; Steadman, Victoria A.; Dean, David K.; Jansa, Petr; Poullennec, Karine G.; Appleby, Todd; Austin, Carol; Blakemore, Caroline A.; Cai, Ruby; Cannizzaro, Carina; Chin, Gregory; Chiva, Jean-Yves C.; Dunbar, Neil A.; Fliri, Hans; Highton, Adrian J.; Hui, Hon; Ji, Mingzhe; Jin, Haolun; Karki, Kapil; Keats, Andrew J.; Lazarides, Linos; Lee, Yu-Jen; Liclican, Albert; Mish, Michael; Murray, Bernard; Pettit, Simon B.; Pyun, Peter; Sangi, Michael; Santos, Rex; Sanvoisin, Jonathan; Schmitz, Uli; Schrier, Adam; Siegel, Dustin; Sperandio, David; Stepan, George; Tian, Yang; Watt, Gregory M.; Yang, Hai; Schultz, Brian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9473-9499</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclophilins are a family of peptidyl-prolyl isomerases that are implicated in a wide range of diseases including hepatitis C.  Our aim was to discover through total synthesis an orally bioavailable, non-immunosuppressive cyclophilin (Cyp) inhibitor with potent anti-hepatitis C virus (HCV) activity that could serve as part of an all oral antiviral combination therapy.  An initial lead (I) derived from the sanglifehrin A macrocycle was optimized using structure based design to produce a potent and orally bioavailable inhibitor (II).  The macrocycle ring size was reduced by one atom, and an internal hydrogen bond drove improved permeability and drug-like properties. II demonstrates potent Cyp inhibition (Kd = 5 nM), potent anti-HCV 2a activity (EC50 = 98 nM), and high oral bioavailability in rat (100%) and dog (55%).  The synthetic accessibility and properties of II support its potential as an anti-HCV agent and for interrogating the role of Cyp inhibition in a variety of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEpEZcwXK8z7Vg90H21EOLACvtfcHk0lj6WcqTVyaZcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI&md5=616cf72e0d2766e159769166c98a9e86</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00802%26sid%3Dliteratum%253Aachs%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DDean%26aufirst%3DD.%2BK.%26aulast%3DJansa%26aufirst%3DP.%26aulast%3DPoullennec%26aufirst%3DK.%2BG.%26aulast%3DAppleby%26aufirst%3DT.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBlakemore%26aufirst%3DC.%2BA.%26aulast%3DCai%26aufirst%3DR.%26aulast%3DCannizzaro%26aufirst%3DC.%26aulast%3DChin%26aufirst%3DG.%26aulast%3DChiva%26aufirst%3DJ.-Y.%2BC.%26aulast%3DDunbar%26aufirst%3DN.%2BA.%26aulast%3DFliri%26aufirst%3DH.%26aulast%3DHighton%26aufirst%3DA.%2BJ.%26aulast%3DHui%26aufirst%3DH.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DKarki%26aufirst%3DK.%26aulast%3DKeats%26aufirst%3DA.%2BJ.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DY.-J.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DMish%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DPettit%26aufirst%3DS.%2BB.%26aulast%3DPyun%26aufirst%3DP.%26aulast%3DSangi%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DSanvoisin%26aufirst%3DJ.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DSchrier%26aufirst%3DA.%26aulast%3DSiegel%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DD.%26aulast%3DStepan%26aufirst%3DG.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DWatt%26aufirst%3DG.%2BM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520orally%2520bioavailable%2520cyclophilin%2520inhibitor%2520derived%2520from%2520the%2520sanglifehrin%2520macrocycle%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9473%26epage%3D9499%26doi%3D10.1021%2Facs.jmedchem.8b00802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmonds, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Improving on nature: making a cyclic heptapeptide orally bioavailable</span>. <i>Angew. Chem. Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12059</span>– <span class="NLM_lpage">12063</span>, <span class="refDoi"> DOI: 10.1002/anie.201405364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2Fanie.201405364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=25219505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A280%3ADC%252BC2M7htlSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=12059-12063&author=D.+S.+Nielsenauthor=H.+N.+Hoangauthor=R.+J.+Lohmanauthor=T.+A.+Hillauthor=A.+J.+Luckeauthor=D.+J.+Craikauthor=D.+J.+Edmondsauthor=D.+A.+Griffithauthor=C.+J.+Rotterauthor=R.+B.+Ruggeriauthor=D.+A.+Priceauthor=S.+Lirasauthor=D.+P.+Fairlie&title=Improving+on+nature%3A+making+a+cyclic+heptapeptide+orally+bioavailable&doi=10.1002%2Fanie.201405364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Improving on nature: making a cyclic heptapeptide orally bioavailable</span></div><div class="casAuthors">Nielsen Daniel S; Hoang Huy N; Lohman Rink-Jan; Hill Timothy A; Lucke Andrew J; Craik David J; Edmonds David J; Griffith David A; Rotter Charles J; Ruggeri Roger B; Price David A; Liras Spiros; Fairlie David P</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">12059-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The use of peptides in medicine is limited by low membrane permeability, metabolic instability, high clearance, and negligible oral bioavailability.  The prediction of oral bioavailability of drugs relies on physicochemical properties that favor passive permeability and oxidative metabolic stability, but these may not be useful for peptides.  Here we investigate effects of heterocyclic constraints, intramolecular hydrogen bonds, and side chains on the oral bioavailability of cyclic heptapeptides.  NMR-derived structures, amide H-D exchange rates, and temperature-dependent chemical shifts showed that the combination of rigidification, stronger hydrogen bonds, and solvent shielding by branched side chains enhances the oral bioavailability of cyclic heptapeptides in rats without the need for N-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT0Jn0fIRQaxLt7lRqEFUJVfW6udTcc2eamQua-a5Tz3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7htlSquw%253D%253D&md5=10b02994390bbb4d93e2089e18a13a98</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.201405364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201405364%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DLohman%26aufirst%3DR.%2BJ.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DEdmonds%26aufirst%3DD.%2BJ.%26aulast%3DGriffith%26aufirst%3DD.%2BA.%26aulast%3DRotter%26aufirst%3DC.%2BJ.%26aulast%3DRuggeri%26aufirst%3DR.%2BB.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DImproving%2520on%2520nature%253A%2520making%2520a%2520cyclic%2520heptapeptide%2520orally%2520bioavailable%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%2520Engl.%26date%3D2014%26volume%3D53%26spage%3D12059%26epage%3D12063%26doi%3D10.1002%2Fanie.201405364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marjanović, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Čikoš, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span> <span> </span><span class="NLM_article-title">A new screening tool to determine chameleonic properties of macrocycles</span>. <i>J. Mol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1198</i></span>,  <span class="NLM_fpage">126929</span>, <span class="refDoi"> DOI: 10.1016/j.molstruc.2019.126929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.molstruc.2019.126929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1enu7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1198&publication_year=2019&pages=126929&author=N.+Marjanovi%C4%87author=A.+%C4%8Ciko%C5%A1author=S.+Ko%C5%A1trun&title=A+new+screening+tool+to+determine+chameleonic+properties+of+macrocycles&doi=10.1016%2Fj.molstruc.2019.126929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A new screening tool to determine chameleonic properties of macrocycles</span></div><div class="casAuthors">Marjanovic, Nera; Cikos, Ana; Kostrun, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Structure</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1198</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126929</span>CODEN:
                <span class="NLM_cas:coden">JMOSB4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2860</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Membrane permeability is one of the major caveats for the broader use of macrocyclic compds. in the drug discovery.  Some macrocyclic drugs can increase their membrane permeability through conformational changes triggered by different biol. environments, a property referred to as chameleonic ability.  In this study, the combined NMR spectroscopy and mol. modeling workflow was developed as a screening tool to identify macrocycles prone to conformational changes induced by complexation with metal ions.  Three structurally diverse macrocyclic compds. have been investigated to develop and validate the approach; Cyclosporin A as a ref. compd. known to possess chameleonic properties, Rifampicin and Fidaxomicin.  The combination of NMR spectroscopy and mol. modeling proved to be a promising tool for quick assessment of the potential chameleonic behavior of macrocycles through their complexation with metal ions.  Addnl., the results confirmed that ability to form intra-mol., transannular hydrogen bonds is important prerequisite for induced conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMmhirGLLWDLVg90H21EOLACvtfcHk0liFFOPgowxQlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1enu7vL&md5=20a4a290f3371bc3ed9ebe3cab972639</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.molstruc.2019.126929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molstruc.2019.126929%26sid%3Dliteratum%253Aachs%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DN.%26aulast%3D%25C4%258Ciko%25C5%25A1%26aufirst%3DA.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26atitle%3DA%2520new%2520screening%2520tool%2520to%2520determine%2520chameleonic%2520properties%2520of%2520macrocycles%26jtitle%3DJ.%2520Mol.%2520Struct.%26date%3D2019%26volume%3D1198%26spage%3D126929%26doi%3D10.1016%2Fj.molstruc.2019.126929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paljetak, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaskovic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matijasic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fajdetic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbanac, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peric, M.</span></span> <span> </span><span class="NLM_article-title">Macrolide hybrid compounds: drug discovery opportunities in anti- infective and anti- inflammatory Area</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.2174/1568026616666160927160036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.2174%2F1568026616666160927160036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27697049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFems78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=919-940&author=H.+C.+Paljetakauthor=L.+Tomaskovicauthor=M.+Matijasicauthor=M.+Bukvicauthor=A.+Fajdeticauthor=D.+Verbanacauthor=M.+Peric&title=Macrolide+hybrid+compounds%3A+drug+discovery+opportunities+in+anti-+infective+and+anti-+inflammatory+Area&doi=10.2174%2F1568026616666160927160036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Macrolide Hybrid Compounds: Drug Discovery Opportunities in Anti- Infective and Anti-inflammatory Area</span></div><div class="casAuthors">Paljetak, Hana Cipcic; Tomaskovic, Linda; Matijasic, Mario; Bukvic, Mirjana; Fajdetic, Andrea; Verbanac, Donatella; Peric, Mihaela</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">919-940</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Macrolides, polyketide natural products, and their 15-membered semi-synthetic derivs. are composed of substituted macrocyclic lactone ring and used primarily as potent antibiotics.  Recently their usefulness was extended to antimalarial and anti-inflammatory area.  Hybrid macrolides presented in this article are the next generation semi-synthetic compds. that combine pharmacophores from antibacterial, antimalarial and anti-inflammatory area with 14- and 15-membered azalide scaffolds.  Antibacterial azalide hybrids with sulfonamides showed improved activity against resistant streptococci while quinolone conjugates demonstrated full coverage of respiratory pathogens including macrolide resistant strains and their efficacy was confirmed in mouse pneumonia model.  Antimalarial macrolide hybrids, mainly involving (chloro)quinoline pharmacophores, showed outstanding activity against chloroquine resistant strains, favorable pharmacokinetics, promising in vivo efficacy as well as encouraging developmental potential.  Anti-inflammatory hybrids were obtained by combining macrolides with corticosteroid and non-steroidal anti-inflammatory drugs.  They were found active in in vivo animal models of locally induced inflammation, asthma, inflammatory bowel disease and rheumatoid arthritis and demonstrated improved safety over parent steroid drugs.  Overall, macrolide hybrids possess significant potential to be developed as potent novel medicines in therapeutic areas of utmost pharmaceutical interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwFnd8uMkUd7Vg90H21EOLACvtfcHk0liFFOPgowxQlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFems78%253D&md5=cc1251fb2a71647da18bb3b37bcdffd8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2174%2F1568026616666160927160036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026616666160927160036%26sid%3Dliteratum%253Aachs%26aulast%3DPaljetak%26aufirst%3DH.%2BC.%26aulast%3DTomaskovic%26aufirst%3DL.%26aulast%3DMatijasic%26aufirst%3DM.%26aulast%3DBukvic%26aufirst%3DM.%26aulast%3DFajdetic%26aufirst%3DA.%26aulast%3DVerbanac%26aufirst%3DD.%26aulast%3DPeric%26aufirst%3DM.%26atitle%3DMacrolide%2520hybrid%2520compounds%253A%2520drug%2520discovery%2520opportunities%2520in%2520anti-%2520infective%2520and%2520anti-%2520inflammatory%2520Area%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D919%26epage%3D940%26doi%3D10.2174%2F1568026616666160927160036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kos, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Škugor, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakopović, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malnar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragojević, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralić, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alihodžić, S.</span></span> <span> </span><span class="NLM_article-title">Around the macrolide - impact of 3D structure of macrocycles on lipophilicity and cellular accumulation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.ejmech.2017.03.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=28410508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVeksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=351-364&author=S.+Ko%C5%A1trunauthor=V.+M.+Kosauthor=M.+M.+%C5%A0kugorauthor=I.+P.+Jakopovi%C4%87author=I.+Malnarauthor=S.+Dragojevi%C4%87author=J.+Rali%C4%87author=S.+Alihod%C5%BEi%C4%87&title=Around+the+macrolide+-+impact+of+3D+structure+of+macrocycles+on+lipophilicity+and+cellular+accumulation&doi=10.1016%2Fj.ejmech.2017.03.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Around the macrolide - Impact of 3D structure of macrocycles on lipophilicity and cellular accumulation</span></div><div class="casAuthors">Kostrun, Sanja; Munic Kos, Vesna; Matanovic Skugor, Maja; Palej Jakopovic, Ivana; Malnar, Ivica; Dragojevic, Snjezana; Ralic, Jovica; Alihodzic, Sulejman</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">351-364</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The aim of this study was to investigate lipophilicity and cellular accumulation of rationally designed azithromycin and clarithromycin derivs. at the mol. level.  The effect of substitution site and substituent properties on a global physico-chem. profile and cellular accumulation of investigated compds. was studied using calcd. structural parameters as well as exptl. detd. lipophilicity.  In silico models based on the 3D structure of mols. were generated to investigate conformational effect on studied properties and to enable prediction of lipophilicity and cellular accumulation for this class of mols. based on non-empirical parameters.  The applicability of developed models was explored on a validation and test sets and compared with previously developed empirical models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXNqSLGZkuVLVg90H21EOLACvtfcHk0ljFaE6VluvpkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVeksbw%253D&md5=7cf75a8da5ea5c0e7fb9de5e5e685cb5</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.056%26sid%3Dliteratum%253Aachs%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DKos%26aufirst%3DV.%2BM.%26aulast%3D%25C5%25A0kugor%26aufirst%3DM.%2BM.%26aulast%3DJakopovi%25C4%2587%26aufirst%3DI.%2BP.%26aulast%3DMalnar%26aufirst%3DI.%26aulast%3DDragojevi%25C4%2587%26aufirst%3DS.%26aulast%3DRali%25C4%2587%26aufirst%3DJ.%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DS.%26atitle%3DAround%2520the%2520macrolide%2520-%2520impact%2520of%25203D%2520structure%2520of%2520macrocycles%2520on%2520lipophilicity%2520and%2520cellular%2520accumulation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D133%26spage%3D351%26epage%3D364%26doi%3D10.1016%2Fj.ejmech.2017.03.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stepanić, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malnar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hlevnjak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butković, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ćaleta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukši, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kragol, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makaruha-Stegić, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikac, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralić, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatić, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavčar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zulfikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munić, V.</span></span> <span> </span><span class="NLM_article-title">Modeling cellular pharmacokinetics of 14- and 15-membered macrolides with physicochemical properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1021/jm101317f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101317f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=719-733&author=V.+Stepani%C4%87author=S.+Ko%C5%A1trunauthor=I.+Malnarauthor=M.+Hlevnjakauthor=K.+Butkovi%C4%87author=I.+%C4%86aletaauthor=M.+Duk%C5%A1iauthor=G.+Kragolauthor=O.+Makaruha-Stegi%C4%87author=L.+Mikacauthor=J.+Rali%C4%87author=I.+Tati%C4%87author=B.+Tav%C4%8Darauthor=K.+Valkoauthor=S.+Zulfikariauthor=V.+Muni%C4%87&title=Modeling+cellular+pharmacokinetics+of+14-+and+15-membered+macrolides+with+physicochemical+properties&doi=10.1021%2Fjm101317f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Cellular Pharmacokinetics of 14- and 15-Membered Macrolides with Physicochemical Properties</span></div><div class="casAuthors">Stepanic, Visnja; Kostrun, Sanja; Malnar, Ivica; Hlevnjak, Mario; Butkovic, Kristina; Caleta, Irena; Duksi, Marko; Kragol, Goran; Makaruha-Stegic, Oresta; Mikac, Lara; Ralic, Jovica; Tatic, Iva; Tavcar, Branka; Valko, Klara; Zulfikari, Selvira; Munic, Vesna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">719-733</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrolides with 14- and 15-membered ring are characterized by high and extensive tissue distribution, as well as good cellular accumulation and retention.  Since macrolide structures do not fit the Lipinski rule of five, macrolide pharmacokinetic properties cannot be successfully predicted by common models based on data for small mols.  Here we describe the development of the first models for macrolide cellular pharmacokinetics.  By comparison of cellular accumulation and retention in six human primary cell cultures of leukocytic and lung origin, as well as in lung carcinoma cell line NCI-H292, this cell line was found to be an adequate representative cell type for modeling macrolide cellular pharmacokinetics.  Accumulation and retention in the NCI-H292 cells, as well as various physicochem. properties, were detd. for a set of 48 rationally designed basic macrolide compds.  Classification models for predicting macrolide cellular accumulation and retention were developed using relatively easily detd. and conceptually simple descriptors: exptl. detd. physicochem. parameters ChromlogD and CHI IAM, as well as a calcd. no. of pos. charged atoms (POS).  The models were further tested and improved by addn. of 37 structurally diverse macrolide mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLVpV1pFm_H7Vg90H21EOLACvtfcHk0lioNXf9heYgLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gmsg%253D%253D&md5=9ff804c79b33d707b6e327b7ea21aef9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm101317f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101317f%26sid%3Dliteratum%253Aachs%26aulast%3DStepani%25C4%2587%26aufirst%3DV.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DMalnar%26aufirst%3DI.%26aulast%3DHlevnjak%26aufirst%3DM.%26aulast%3DButkovi%25C4%2587%26aufirst%3DK.%26aulast%3D%25C4%2586aleta%26aufirst%3DI.%26aulast%3DDuk%25C5%25A1i%26aufirst%3DM.%26aulast%3DKragol%26aufirst%3DG.%26aulast%3DMakaruha-Stegi%25C4%2587%26aufirst%3DO.%26aulast%3DMikac%26aufirst%3DL.%26aulast%3DRali%25C4%2587%26aufirst%3DJ.%26aulast%3DTati%25C4%2587%26aufirst%3DI.%26aulast%3DTav%25C4%258Dar%26aufirst%3DB.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DZulfikari%26aufirst%3DS.%26aulast%3DMuni%25C4%2587%26aufirst%3DV.%26atitle%3DModeling%2520cellular%2520pharmacokinetics%2520of%252014-%2520and%252015-membered%2520macrolides%2520with%2520physicochemical%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D719%26epage%3D733%26doi%3D10.1021%2Fjm101317f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">712</span>– <span class="NLM_lpage">717</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.drudis.2016.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=26891978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=712-717&author=A.+Whittyauthor=M.+Zhongauthor=L.+Viarengoauthor=D.+Beglovauthor=D.+R.+Hallauthor=S.+Vajda&title=Quantifying+the+chameleonic+properties+of+macrocycles+and+other+high-molecular-weight+drugs&doi=10.1016%2Fj.drudis.2016.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span></div><div class="casAuthors">Whitty, Adrian; Zhong, Mengqi; Viarengo, Lauren; Beglov, Dmitri; Hall, David R.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">712-717</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Key to the pharmaceutical utility of certain macrocyclic drugs is a 'chameleonic' ability to change their conformation to expose polar groups in aq. soln., but bury them when traversing lipid membranes.  Based on anal. of the structures of 20 macrocyclic compds. that are approved oral drugs, we propose that good soly. requires a topol. polar surface area (TPSA, in Å2) of ≥0.2 × mol. wt. (MW).  Meanwhile, good passive membrane permeability requires a mol. (i.e., 3D) PSA in nonpolar environments of ≤140 Å2.  We show that one or other of these limits is almost invariably violated for compds. with MW > 600 Da, suggesting that some degree of chameleonic behavior is required for most high MW oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlN7rCqts5wLVg90H21EOLACvtfcHk0lhM9oiHyLRoSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D&md5=024476298f7b72efa3f0a9303bed5900</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DViarengo%26aufirst%3DL.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DQuantifying%2520the%2520chameleonic%2520properties%2520of%2520macrocycles%2520and%2520other%2520high-molecular-weight%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D712%26epage%3D717%26doi%3D10.1016%2Fj.drudis.2016.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Faham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albericio, F.</span></span> <span> </span><span class="NLM_article-title">Peptide coupling reagents, More than a letter soup</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">6557</span>– <span class="NLM_lpage">6602</span>, <span class="refDoi"> DOI: 10.1021/cr100048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr100048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVyhu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2011&pages=6557-6602&author=A.+El-Fahamauthor=F.+Albericio&title=Peptide+coupling+reagents%2C+More+than+a+letter+soup&doi=10.1021%2Fcr100048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Coupling Reagents, More than a Letter Soup</span></div><div class="casAuthors">El-Faham, Ayman; Albericio, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6557-6602</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The prepn. and uses of carbodiimides, anhydrides (sym. and mixed), active esters, acylazoles, acyl azides, acid halides, acid fluorides, phosphonium salts, aminium/uronium salts, organophosphorus compds., organosulfur compds., triazine reagents, pyridinium reagents, polymer-supported reagents as coupling reagents in peptide synthesis are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DDv3PT7yrLVg90H21EOLACvtfcHk0lhM9oiHyLRoSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVyhu7zO&md5=461d8c78dffcbf16a390fc78abd33cac</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fcr100048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr100048w%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Faham%26aufirst%3DA.%26aulast%3DAlbericio%26aufirst%3DF.%26atitle%3DPeptide%2520coupling%2520reagents%252C%2520More%2520than%2520a%2520letter%2520soup%26jtitle%3DChem.%2520Rev.%26date%3D2011%26volume%3D111%26spage%3D6557%26epage%3D6602%26doi%3D10.1021%2Fcr100048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djokić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobrehel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopotar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamenar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrvos, D.</span></span> <span> </span><span class="NLM_article-title">Erythromycin series. Part 13. Synthesis and structure elucidation of 10-dihydro-10-deoxo-11-methyl-11-azaerythromycin A</span>. <i>J. Chem. Res. Synopses.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">153</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1988&pages=152-153&author=S.+Djoki%C4%87author=G.+Kobrehelauthor=N.+Lopotarauthor=B.+Kamenarauthor=A.+Naglauthor=D.+Mrvos&title=Erythromycin+series.+Part+13.+Synthesis+and+structure+elucidation+of+10-dihydro-10-deoxo-11-methyl-11-azaerythromycin+A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDjoki%25C4%2587%26aufirst%3DS.%26aulast%3DKobrehel%26aufirst%3DG.%26aulast%3DLopotar%26aufirst%3DN.%26aulast%3DKamenar%26aufirst%3DB.%26aulast%3DNagl%26aufirst%3DA.%26aulast%3DMrvos%26aufirst%3DD.%26atitle%3DErythromycin%2520series.%2520Part%252013.%2520Synthesis%2520and%2520structure%2520elucidation%2520of%252010-dihydro-10-deoxo-11-methyl-11-azaerythromycin%2520A%26jtitle%3DJ.%2520Chem.%2520Res.%2520Synopses.%26date%3D1988%26volume%3D5%26spage%3D152%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known linear and macrocyclic IL-17A inhibitors.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16 ref17 ref18">(12−18)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Modular Synthesis of Novel Macrocyclic Scaffolds Starting from Natural or Semi-Synthetic Macrolides<a onclick="showRef(event, 'ref20 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref25">(20,25)</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Available X-ray structures of the IL-17A homodimer. Possible binding sites according to the PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> structure (blue and gray surfaces); IL-17A in complex with macrocyclic inhibitors, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a> (green)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> aligned with PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a> (magenta)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HHX">5HHX</a> (yellow)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> structures; MC inhibitors are shown by colored van der Waals’ surface.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. NMR constrained 3D structure of the tool compound and corresponding SAR features, aligned with predicted conformation of compound <b>2</b> (a). Predicted binding pose of the tool compound (magenta) docked into PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and aligned with Eli Lilly’s cyclic peptide (orange) from PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9">5VB9</a><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (3D presentation (b) and 2D projection of protein–ligand interactions (c)).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Strategy for the Preparation of Compounds <b>1</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Symbols Q, X, Y, Z, R1, R2, R3, R4, and LSC are described in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Reagents and conditions: (i) Pb(OAc)<sub>4</sub>, AcOH. (ii) <i>N</i>-Fmoc-amino acid, HATU, HOAt, TMP, DCM, r.t. (iii) piperidine, DCM, r.t. (iv) <i>N</i>-Fmoc-amino acid, HATU, HOAt, TMP, DCM, r.t. (v) LiOH, THF/water, r.t. (vi) HATU, DIPEA, DMF, r.t. (vii) Pd/C; hydrogen, MeOH, r.t. (viii) <b>64</b>, HATU, DIPEA, DMF, r.t.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Strategy for the Preparation of Compounds <b>28</b>–<b>64</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Fmoc-2-chloro-<span class="smallcaps smallerCapital">l</span>-phenylalanine, HATU, DIPEA, DMF, r.t. (ii) piperidine, DCM, r.t. (iii) amidation: corresponding acid, HATU, DIPEA, DMF, r.t.; urea formation: corresponding isocyanate, DCM, r.t.; sulfonamide formation: corresponding sulfonyl chloride, TEA., DCM, r.t. (iv) corresponding Fmoc-protected amino acid HATU, DIPEA, DMF, r.t.</p></p></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Medicinal chemistry exploration of FideltaMacro IL-17 macrocyclic disruptors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Most active macrocycles with inserted dipeptide containing cyclopentyl/proline moieties. Compounds are synthesized following the general synthetic pathway described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Most active compounds resulted from the optimization of the ring size, stereochemistry, and order of inserted amino acids. Compounds are synthesized following the general synthetic pathway described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a><b>.</b></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Conformational analysis of compound <b>4</b>: (a) 2D structure with key nOe contacts used as constraints in the modeling process and (b) predicted 3D conformation (sugars, green; linear side chain, magenta; macrocycle, gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Predicted docking poses of compound <b>4</b> (blue) generated by hypothesis 1: docked into PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA">4HSA</a> and aligned with tool compound (magenta) (3D presentation (a) and 2D projection of protein–ligand interactions (b)) and hypothesis 2: docked into PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4">5HI4</a> and aligned with a macrocyclic compound (green) from Pfizer (3D presentation (c) and 2D projection of protein–ligand interactions (d)).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Epitope mapping by STD NMR: (a) proton spectrum (black), STD NMR spectrum without human IL-17A homodimer (red), and STD NMR spectrum with hIL-17A protein (green); (b) STD NMR enhancements in compound <b>4</b> interacting with hIL-17A; (c) 3D conformation with STD enhancements.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Advanced compounds resulted from the modification of the terminal amino acid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Influence of the sugar moieties on the activity of compound <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/medium/jm1c00327_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Van der Waals’ surface of compound <b>4</b> color-coded by electrostatic potential used for the calculation of 3D polar surface area (PSA) followed by estimation of the potential chameleonic behavior based on the difference between minimal and buried ΔPSA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00327/20210616/images/large/jm1c00327_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00327&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i126">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27187" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27187" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parisi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Symmons, D. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, C. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashcroft, D. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Management of Psoriasis and Associated ComorbidiTy (IMPACT) project team. Global epidemiology of psoriasis: a systematic review of incidence and prevalence</span>. <i>J. Inv. Derm.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1038/jid.2012.339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fjid.2012.339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=23014338" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2013&pages=377-385&author=R.+Parisiauthor=D.+P.+M.+Symmonsauthor=C.+E.+M.+Griffithsauthor=D.+M.+Ashcroft&title=Identification+and+Management+of+Psoriasis+and+Associated+ComorbidiTy+%28IMPACT%29+project+team.+Global+epidemiology+of+psoriasis%3A+a+systematic+review+of+incidence+and+prevalence&doi=10.1038%2Fjid.2012.339"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fjid.2012.339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2012.339%26sid%3Dliteratum%253Aachs%26aulast%3DParisi%26aufirst%3DR.%26aulast%3DSymmons%26aufirst%3DD.%2BP.%2BM.%26aulast%3DGriffiths%26aufirst%3DC.%2BE.%2BM.%26aulast%3DAshcroft%26aufirst%3DD.%2BM.%26atitle%3DIdentification%2520and%2520Management%2520of%2520Psoriasis%2520and%2520Associated%2520ComorbidiTy%2520%2528IMPACT%2529%2520project%2520team.%2520Global%2520epidemiology%2520of%2520psoriasis%253A%2520a%2520systematic%2520review%2520of%2520incidence%2520and%2520prevalence%26jtitle%3DJ.%2520Inv.%2520Derm.%26date%3D2013%26volume%3D133%26spage%3D377%26epage%3D385%26doi%3D10.1038%2Fjid.2012.339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>WHO Global Report on Psoriasis</i>;
Geneva, Switzerland, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=World+Health+Organization&title=WHO+Global+Report+on+Psoriasis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DWHO%2520Global%2520Report%2520on%2520Psoriasis%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brembilla, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senra, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehncke, W.-H.</span></span> <span> </span><span class="NLM_article-title">The IL-17 family of cytokines in psoriasis: IL-17A and beyond</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1682</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.01682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.3389%2Ffimmu.2018.01682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=30127781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWmurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1682&author=N.+C.+Brembillaauthor=L.+Senraauthor=W.-H.+Boehncke&title=The+IL-17+family+of+cytokines+in+psoriasis%3A+IL-17A+and+beyond&doi=10.3389%2Ffimmu.2018.01682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The IL-17 family of cytokines in Psoriasis: IL-17A and beyond</span></div><div class="casAuthors">Brembilla, Nicolo Costantino; Senra, Luisa; Boehncke, Wolf-Henning</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1682/1-1682/13</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Psoriasis is a frequent chronic inflammatory skin disease, nowadays considered a major global health problem.  Several new drugs, targeting the IL-23/IL-17A pathway, have been recently licensed or are in clin. development.  These therapies represent a major improvement of the way in which psoriasis is managed, since they show an unprecedented efficacy on skin symptoms of psoriasis.  This has been made possible, thanks to an increasingly more accurate pathogenic view of psoriasis.  Today, the belief that Th17 cells mediate psoriasis is moving to the concept of psoriasis as an IL-17A-driven disease.  New questions arise at the horizon, given that IL-17A is part of a newly described family of cytokines, which has five distinct homologous: IL-17B, IL-17C, IL-17D, IL-17E, also known as IL-25 and IL-17F.  IL-17 family cytokines elicit similar effects in target cells, but simultaneously trigger different and sometimes opposite functions in a tissue-specific manner.  This is complicated by the fact that IL-17 cytokines show a high capacity of synergisms with other inflammatory stimuli.  In this review, we will summarize the current knowledge around the cytokines belonging to the IL-17 family in relation to skin inflammation in general and psoriasis in particular, and discuss possible clin. implications.  A comprehensive understanding of the different roles played by the IL-17 cytokines is crucial to appreciate current and developing therapies and to allow an effective pathogenesis- and mechanisms-driven drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Q2Yx4qUoprVg90H21EOLACvtfcHk0ljEODG7-xtLlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWmurrJ&md5=d3975343c94cc7fd523d55a8c3013ecf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.01682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.01682%26sid%3Dliteratum%253Aachs%26aulast%3DBrembilla%26aufirst%3DN.%2BC.%26aulast%3DSenra%26aufirst%3DL.%26aulast%3DBoehncke%26aufirst%3DW.-H.%26atitle%3DThe%2520IL-17%2520family%2520of%2520cytokines%2520in%2520psoriasis%253A%2520IL-17A%2520and%2520beyond%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D1682%26doi%3D10.3389%2Ffimmu.2018.01682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blauvelt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiricozzi, A.</span></span> <span> </span><span class="NLM_article-title">The immunologic role of IL-17 in psoriasis and psoriatic arthritis pathogenesis</span>. <i>Clin. Rev. Allerg. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1007/s12016-018-8702-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1007%2Fs12016-018-8702-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=30109481" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2018&pages=379-390&author=A.+Blauveltauthor=A.+Chiricozzi&title=The+immunologic+role+of+IL-17+in+psoriasis+and+psoriatic+arthritis+pathogenesis&doi=10.1007%2Fs12016-018-8702-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs12016-018-8702-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12016-018-8702-3%26sid%3Dliteratum%253Aachs%26aulast%3DBlauvelt%26aufirst%3DA.%26aulast%3DChiricozzi%26aufirst%3DA.%26atitle%3DThe%2520immunologic%2520role%2520of%2520IL-17%2520in%2520psoriasis%2520and%2520psoriatic%2520arthritis%2520pathogenesis%26jtitle%3DClin.%2520Rev.%2520Allerg.%2520Immunol.%26date%3D2018%26volume%3D55%26spage%3D379%26epage%3D390%26doi%3D10.1007%2Fs12016-018-8702-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffatellu, M.</span></span> <span> </span><span class="NLM_article-title">Cytokines IL-17 and IL-22 in the host response to infection</span>. <i>Pathog. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">ftw111</span>, <span class="refDoi"> DOI: 10.1093/femspd/ftw111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1093%2Ffemspd%2Fftw111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27915228" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2016&pages=ftw111&author=M.+Valeriauthor=M.+Raffatellu&title=Cytokines+IL-17+and+IL-22+in+the+host+response+to+infection&doi=10.1093%2Ffemspd%2Fftw111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Ffemspd%2Fftw111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ffemspd%252Fftw111%26sid%3Dliteratum%253Aachs%26aulast%3DValeri%26aufirst%3DM.%26aulast%3DRaffatellu%26aufirst%3DM.%26atitle%3DCytokines%2520IL-17%2520and%2520IL-22%2520in%2520the%2520host%2520response%2520to%2520infection%26jtitle%3DPathog.%2520Dis.%26date%3D2016%26volume%3D74%26spage%3Dftw111%26doi%3D10.1093%2Ffemspd%2Fftw111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kirkham, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavanaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, K.</span></span> <span> </span><span class="NLM_article-title">Interleukin-17A: a unique pathway in immune-mediated diseases: psoriasis, psoriatic arthritis and rheumatoid arthritis</span>. <i>Immunology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1111/imm.12142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1111%2Fimm.12142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=23819583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktFahtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2014&pages=133-142&author=B.+W.+Kirkhamauthor=A.+Kavanaughauthor=K.+Reich&title=Interleukin-17A%3A+a+unique+pathway+in+immune-mediated+diseases%3A+psoriasis%2C+psoriatic+arthritis+and+rheumatoid+arthritis&doi=10.1111%2Fimm.12142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Interleukin-17A: a unique pathway in immune-mediated diseases: psoriasis, psoriatic arthritis and rheumatoid arthritis</span></div><div class="casAuthors">Kirkham, Bruce W.; Kavanaugh, Arthur; Reich, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-142</span>CODEN:
                <span class="NLM_cas:coden">IMMUAM</span>;
        ISSN:<span class="NLM_cas:issn">0019-2805</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Exptl. evidence points to the importance of the cytokine interleukin-17A (IL-17A) in the pathogenesis of several immunoinflammatory diseases including psoriasis, psoriatic arthritis and rheumatoid arthritis.  Although a principal effector of T helper type 17 cells, IL-17A is produced by many other cell types including CD8+ T cells and γδ T cells, and is found at high levels assocd. with mast cells and neutrophils at sites of skin and joint disease in humans.  IL-17A up-regulates expression of numerous inflammation-related genes in target cells such as keratinocytes and fibroblasts, leading to increased prodn. of chemokines, cytokines, antimicrobial peptides and other mediators that contribute to clin. disease features.  Importantly, IL-17A must be considered within the context of the local microenvironment, because it acts synergistically or additively with other pro-inflammatory cytokines, including tumor necrosis factor.  Several direct IL-17A inhibitors have shown promising activity in proof of concept and phase 2 clin. studies, thereby providing confirmation of exptl. data supporting IL-17A in disease pathogenesis, although levels of response are not predicted by pre-clin. findings.  IL-17A inhibitors produced rapid down-regulation of the psoriasis gene signature and high clin. response rates in patients with moderate-to-severe plaque psoriasis, consistent with an important role for IL-17A in psoriasis pathogenesis.  Clin. response rates with IL-17A inhibitors in psoriatic arthritis and rheumatoid arthritis, however, were improved to a lesser degree compared with placebo, suggesting that IL-17A is either important in a subset of patients or plays a relatively minor role in inflammatory joint disease.  Ongoing phase 3 clin. trials should provide further information on the role of IL-17A in these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0u3C1Z5BR6LVg90H21EOLACvtfcHk0ljoWmDv94lNVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktFahtA%253D%253D&md5=36150b3dceffde2c4fcbe862894cfbe6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fimm.12142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimm.12142%26sid%3Dliteratum%253Aachs%26aulast%3DKirkham%26aufirst%3DB.%2BW.%26aulast%3DKavanaugh%26aufirst%3DA.%26aulast%3DReich%26aufirst%3DK.%26atitle%3DInterleukin-17A%253A%2520a%2520unique%2520pathway%2520in%2520immune-mediated%2520diseases%253A%2520psoriasis%252C%2520psoriatic%2520arthritis%2520and%2520rheumatoid%2520arthritis%26jtitle%3DImmunology%26date%3D2014%26volume%3D141%26spage%3D133%26epage%3D142%26doi%3D10.1111%2Fimm.12142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffen, S. L.</span></span> <span> </span><span class="NLM_article-title">IL-17RC: a partner in IL-17 signaling and beyond</span>. <i>Semin. Immunopathol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1007/s00281-009-0185-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1007%2Fs00281-009-0185-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=20012905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=33-42&author=A.+W.+Hoauthor=S.+L.+Gaffen&title=IL-17RC%3A+a+partner+in+IL-17+signaling+and+beyond&doi=10.1007%2Fs00281-009-0185-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">IL-17RC: a partner in IL-17 signaling and beyond</span></div><div class="casAuthors">Ho, Allen W.; Gaffen, Sarah L.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Immunopathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-42</span>CODEN:
                <span class="NLM_cas:coden">SIEMB7</span>;
        ISSN:<span class="NLM_cas:issn">1863-2297</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The interleukin (IL)-17 cytokine family members IL-17A and IL-17F mediate inflammatory activities via the IL-17 receptor (IL-17R) complex, comprised of the IL-17RA and IL-17RC subunits.  Proper regulation of the IL-17 signaling axis results in effective host defense against extracellular pathogens, while aberrant signaling can drive autoimmune pathol.  Elucidating the mol. mechanisms underlying IL-17 signal transduction can yield an enhanced understanding of inflammatory immune processes and also create an avenue for therapeutic intervention in the treatment of IL-17-dependent diseases.  To date, the fundamental signaling mechanisms used by the IL-17R complex are still incompletely defined.  While current structure-function studies have primarily focused on the IL-17RA subunit, recent research indicates that the IL-17RC subunit plays a key role in modulating IL-17 responses.  This review will examine what is known regarding IL-17RC function and provide a framework for future work on this subunit and its impact on human health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2VeCEngWLlLVg90H21EOLACvtfcHk0ljoWmDv94lNVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyisrg%253D&md5=150f5d096961d947c037a61838e14405</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00281-009-0185-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00281-009-0185-0%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DA.%2BW.%26aulast%3DGaffen%26aufirst%3DS.%2BL.%26atitle%3DIL-17RC%253A%2520a%2520partner%2520in%2520IL-17%2520signaling%2520and%2520beyond%26jtitle%3DSemin.%2520Immunopathol.%26date%3D2010%26volume%3D32%26spage%3D33%26epage%3D42%26doi%3D10.1007%2Fs00281-009-0185-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velichko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">A Novel nuclear function for the interleukin-17 signaling adaptor protein Act1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">e0163323/1</span>– <span class="NLM_lpage">e0163323/17</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1371%2Fjournal.pone.0163323" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=e0163323%2F1-e0163323%2F17&author=S.+Velichkoauthor=X.+Zhouauthor=L.+Zhuauthor=J.+D.+Andersonauthor=R.+Wuauthor=Y.+Chen&title=A+Novel+nuclear+function+for+the+interleukin-17+signaling+adaptor+protein+Act1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0163323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0163323%26sid%3Dliteratum%253Aachs%26aulast%3DVelichko%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DAnderson%26aufirst%3DJ.%2BD.%26aulast%3DWu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DA%2520Novel%2520nuclear%2520function%2520for%2520the%2520interleukin-17%2520signaling%2520adaptor%2520protein%2520Act1%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26spage%3De0163323%252F1%26epage%3De0163323%252F17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silfvast-Kaiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paek, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menter, A.</span></span> <span> </span><span class="NLM_article-title">Anti-IL17 therapies for psoriasis</span>. <i>Expert Opin. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1080/14712598.2019.1555235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1080%2F14712598.2019.1555235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=30500317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKitLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=45-54&author=A.+Silfvast-Kaiserauthor=S.+Y.+Paekauthor=A.+Menter&title=Anti-IL17+therapies+for+psoriasis&doi=10.1080%2F14712598.2019.1555235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-IL17 therapies for psoriasis</span></div><div class="casAuthors">Silfvast-Kaiser, Annika; Paek, So Yeon; Menter, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Biological Therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-54</span>CODEN:
                <span class="NLM_cas:coden">EOBTA2</span>;
        ISSN:<span class="NLM_cas:issn">1471-2598</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Interleukin-17 (IL-17) is a proinflammatory cytokine considered to play a significant role in the immunopathogenesis of plaque psoriasis.  As a result, focus in clin. trials has undergone a shift towards disease specific targets, with the goals of more effective treatment and redn. in the incidence of serious adverse events.: Two monoclonal antibodies targeting IL-17A (secukinumab, ixekizumab) and one against the IL-17 receptor (brodalumab) are approved for the treatment of moderate-to-severe plaque psoriasis.  Herein, the clin. efficacy, safety and tolerability of each is reviewed by summarizing the existing literature (found via PubMed database).: The development and approval of the IL-17 inhibitor agents secukinumab, ixekizumab, and brodalumab has expanded psoriatic treatment with effective options, validating the importance of the pro-inflammatory role of IL-17 psoriatic pathophysiol.  Biol. treatment options for psoriasis will continue to grow, esp. IL-17 and IL-23 related agents, with an increasing specificity of agents to be available in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-h0nKu45xbVg90H21EOLACvtfcHk0lgcyRG0NISxwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKitLzK&md5=287376738f5c578789fab39754c43a3d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F14712598.2019.1555235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14712598.2019.1555235%26sid%3Dliteratum%253Aachs%26aulast%3DSilfvast-Kaiser%26aufirst%3DA.%26aulast%3DPaek%26aufirst%3DS.%2BY.%26aulast%3DMenter%26aufirst%3DA.%26atitle%3DAnti-IL17%2520therapies%2520for%2520psoriasis%26jtitle%3DExpert%2520Opin.%2520Biol.%2520Ther.%26date%3D2019%26volume%3D19%26spage%3D45%26epage%3D54%26doi%3D10.1080%2F14712598.2019.1555235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakkas, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zafiriou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogdanos, D. P.</span></span> <span> </span><span class="NLM_article-title">Mini review: new treatments in psoriatic arthritis. Focus on the IL-23/17 axis</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">872</span>, <span class="refDoi"> DOI: 10.3389/fphar.2019.00872</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.3389%2Ffphar.2019.00872" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=31447673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFSht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=872&author=L.+I.+Sakkasauthor=E.+Zafiriouauthor=D.+P.+Bogdanos&title=Mini+review%3A+new+treatments+in+psoriatic+arthritis.+Focus+on+the+IL-23%2F17+axis&doi=10.3389%2Ffphar.2019.00872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mini review: new treatments in psoriatic arthritis. Focus on the IL-23/17 axis</span></div><div class="casAuthors">Sakkas, Lazaros I.; Zafiriou, Efterpi; Bogdanos, Dimitrios P.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">872</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Psoriasis, an inflammatory skin disease, and psoriatic arthritis (PsA), an inflammatory arthritis, share clin., genetic, and pathogenic factors and may be summed as one disease, the psoriatic disease.  Interleukin (IL)-17 plays a major role in the development of both psoriasis and PsA.  IL-23 is important in the proliferation and maintenance of IL-17, and therefore, cytokines of the IL-23/IL-17 axis attracted much interest as therapeutic targets in psoriasis and PsA.  Therapeutic agents targeting the IL-23/IL-17 axis have been proven to be very effective in psoriasis and PsA, some are already in the therapeutic armamentarium and others are in the development.  Some agents, target IL-23 and others IL-17 and include anti-IL-12/IL-23 p40 (ustekinumab, briankizumab), anti-IL-23p19 (guselkumab, tildrakizumab, risankizumab, brazikumab, mirikizumab), anti-IL-17A (secukinumab, ixekizumab), dual anti-IL-17A and anti-IL-17F (bimekizumab), or anti-IL-17 receptor (brodalumab) monoclonal antibodies.  Janus tyrosine kinase(JAK) inhibitors also directly affect IL-23 and, thus, IL-17.  After the first-generation pan-JAK inhibitors have been shown efficacy (tofacitinib, baricitinib), new-generation selective JAK inhibitors (filgotinib, upadacitinib) are under investigation in psoriasis and PsA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mVvPfw0KkLVg90H21EOLACvtfcHk0lhwmXp4CucaNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFSht70%253D&md5=c10f25a0258886777a060da0abe4796e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2019.00872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2019.00872%26sid%3Dliteratum%253Aachs%26aulast%3DSakkas%26aufirst%3DL.%2BI.%26aulast%3DZafiriou%26aufirst%3DE.%26aulast%3DBogdanos%26aufirst%3DD.%2BP.%26atitle%3DMini%2520review%253A%2520new%2520treatments%2520in%2520psoriatic%2520arthritis.%2520Focus%2520on%2520the%2520IL-23%252F17%2520axis%26jtitle%3DFront.%2520Pharmacol.%26date%3D2019%26volume%3D10%26spage%3D872%26doi%3D10.3389%2Ffphar.2019.00872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of protein-protein interactions: progressing toward the reality</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.chembiol.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=25237857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1102-1114&author=M.+R.+Arkinauthor=Y.+Tangauthor=J.+A.+Wells&title=Small-molecule+inhibitors+of+protein-protein+interactions%3A+progressing+toward+the+reality&doi=10.1016%2Fj.chembiol.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing toward the Reality</span></div><div class="casAuthors">Arkin, Michelle R.; Tang, Yinyan; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1114</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The past 20 years have seen many advances in our understanding of protein-protein interactions (PPIs) and how to target them with small-mol. therapeutics.  In 2004, we reviewed some early successes; since then, potent inhibitors have been developed for diverse protein complexes, and compds. are now in clin. trials for six targets.  Surprisingly, many of these PPI clin. candidates have efficiency metrics typical of "lead-like" or "drug-like" mols. and are orally available.  Successful discovery efforts have integrated multiple disciplines and make use of all the modern tools of target-based discovery-structure, computation, screening, and biomarkers.  PPIs become progressively more challenging as the interfaces become more complex, i.e., as binding epitopes are displayed on primary, secondary, or tertiary structures.  Here, we review the last 10 years of progress, focusing on the properties of PPI inhibitors that have advanced to clin. trials and prospects for the future of PPI drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP01wt-4lfE7Vg90H21EOLACvtfcHk0lhwmXp4CucaNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyrur7J&md5=aec3897dc9bc66656723bc56c8d69155</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520progressing%2520toward%2520the%2520reality%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D1102%26epage%3D1114%26doi%3D10.1016%2Fj.chembiol.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">Artificial macrocycles as IL-17A - IL-17RA antagonists</span>. <i>MedChemComm.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1039/C7MD00464H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1039%2FC7MD00464H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=29750108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=22-26&author=W.+Wangauthor=M.+R.+Grovesauthor=A.+D%C3%B6mling&title=Artificial+macrocycles+as+IL-17A+-+IL-17RA+antagonists&doi=10.1039%2FC7MD00464H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial macrocycles as IL-17A/IL-17RA antagonists</span></div><div class="casAuthors">Wang, Wenjia; Groves, Matthew R.; Doemling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-26</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Interleukin 17(A) is a pro-inflammatory cytokine involved in several auto-immune and inflammatory diseases.  Current antagonists against IL-17(A) or its receptor (IL-17RA) that show efficacy in clin. trials are monoclonal antibodies (mAbs).  However, recently designed artificial macrocycles are potent IL-17A/IL-17RA antagonists.  Based on co-crystal structures, a better understanding of the biol. activity and SAR of the macrocycles has been gained, demonstrating that they can compete with mAbs for difficult targets such as PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMncYtypRDh7Vg90H21EOLACvtfcHk0lhwmXp4CucaNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L&md5=62b3ef7b447d2367e337dad6e5a3a204</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1039%2FC7MD00464H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00464H%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3DArtificial%2520macrocycles%2520as%2520IL-17A%2520-%2520IL-17RA%2520antagonists%26jtitle%3DMedChemComm.%26date%3D2018%26volume%3D9%26spage%3D22%26epage%3D26%26doi%3D10.1039%2FC7MD00464H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M.</span>; <span class="NLM_string-name">Terrett, N. K.</span>; <span class="NLM_string-name">Connors, W. H.</span>; <span class="NLM_string-name">Shortsleeves, K. C.</span>; <span class="NLM_string-name">Seigal, B. A.</span>; <span class="NLM_string-name">Snedeker, C.</span>; <span class="NLM_string-name">Hale, S. P.</span>; <span class="NLM_string-name">Briggs, T. F.</span>; <span class="NLM_string-name">Favaloro, F. G.</span>; <span class="NLM_string-name">Cipriani, T. J.</span>; <span class="NLM_string-name">Yan, D</span>; <span class="NLM_string-name">Alexander, S. L.</span>; <span class="NLM_string-name">Thorarensen, A.</span>; <span class="NLM_string-name">Xing, L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Macrocyclic Compounds as IL-17 Modulators</span>.  <i>Int. Pat. Appl.</i> <span class="NLM_patent">WO2013116682A1</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=M.+Taylor&author=N.+K.+Terrett&author=W.+H.+Connors&author=K.+C.+Shortsleeves&author=B.+A.+Seigal&author=C.+Snedeker&author=S.+P.+Hale&author=T.+F.+Briggs&author=F.+G.+Favaloro&author=T.+J.+Cipriani&author=D+Yan&author=S.+L.+Alexander&author=A.+Thorarensen&author=L.+Xing&title=Preparation+of+Macrocyclic+Compounds+as+IL-17+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DM.%26atitle%3DPreparation%2520of%2520Macrocyclic%2520Compounds%2520as%2520IL-17%2520Modulators%26jtitle%3DInt.%2520Pat.%2520Appl.%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Espada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broughton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodge, J. A.</span></span> <span> </span><span class="NLM_article-title">A binding site on IL-17A for inhibitory macrocycles revealed by hydrogen-deuterium exchange mass spectrometry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2255</span>– <span class="NLM_lpage">2260</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Ojsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2255-2260&author=A.+Espadaauthor=H.+Broughtonauthor=S.+Jonesauthor=M.+J.+Chalmersauthor=J.+A.+Dodge&title=A+binding+site+on+IL-17A+for+inhibitory+macrocycles+revealed+by+hydrogen-deuterium+exchange+mass+spectrometry&doi=10.1021%2Facs.jmedchem.5b01693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Binding Site on IL-17A for Inhibitory Macrocycles Revealed by Hydrogen/Deuterium Exchange Mass Spectrometry</span></div><div class="casAuthors">Espada, Alfonso; Broughton, Howard; Jones, Spencer; Chalmers, Michael J.; Dodge, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2255-2260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Computational assessment of the IL-17A structure identified two distinct binding pockets, the β-hairpin pocket and the α-helix pocket.  The β-hairpin pocket was hypothesized to be the site of binding for peptide macrocycles.  Support for this hypothesis was obtained using HDX-MS which revealed protection to exchange only within the β-hairpin pocket.  This data represents the first direct structural evidence of a small mol. binding site on IL-17A that functions to disrupt the interaction with its receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsr5NPNUfSbVg90H21EOLACvtfcHk0lhL7SDSRDEAyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Ojsrk%253D&md5=8ce4090423e451c6365b9dc74a6c57c3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01693%26sid%3Dliteratum%253Aachs%26aulast%3DEspada%26aufirst%3DA.%26aulast%3DBroughton%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DChalmers%26aufirst%3DM.%2BJ.%26aulast%3DDodge%26aufirst%3DJ.%2BA.%26atitle%3DA%2520binding%2520site%2520on%2520IL-17A%2520for%2520inhibitory%2520macrocycles%2520revealed%2520by%2520hydrogen-deuterium%2520exchange%2520mass%2520spectrometry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2255%26epage%3D2260%26doi%3D10.1021%2Facs.jmedchem.5b01693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, D.</span>; <span class="NLM_string-name">Alexander, S.</span>; <span class="NLM_string-name">Bond, J.</span>; <span class="NLM_string-name">Briggs, T.</span>; <span class="NLM_string-name">Fraley, A.</span>; <span class="NLM_string-name">Hale, S.</span>; <span class="NLM_string-name">Landsman, T.</span>; <span class="NLM_string-name">Martinelli, R.</span>; <span class="NLM_string-name">Shortsleeves, K.</span>; <span class="NLM_string-name">Terrett, N.</span>; <span class="NLM_string-name">Walsh, N</span></span>. <span> </span><span class="NLM_article-title">Identication and Characterization of Synthetic Small Molecule Macrocycle Antagonists of Human IL17A</span>.  <i>Meeting Abstracts</i>; <i>No 1810</i>, <span class="NLM_publisher-name">2012 ACR/ARHP Annual Meeting</span>: <span class="NLM_publisher-loc">Washington, DC</span>, Nov  <span class="NLM_volume">9-14</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=D.+Livingston&author=S.+Alexander&author=J.+Bond&author=T.+Briggs&author=A.+Fraley&author=S.+Hale&author=T.+Landsman&author=R.+Martinelli&author=K.+Shortsleeves&author=N.+Terrett&author=N+Walsh&title=Identication+and+Characterization+of+Synthetic+Small+Molecule+Macrocycle+Antagonists+of+Human+IL17A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DLivingston%26aufirst%3DD.%26atitle%3DIdentication%2520and%2520Characterization%2520of%2520Synthetic%2520Small%2520Molecule%2520Macrocycle%2520Antagonists%2520of%2520Human%2520IL17A%26jtitle%3DMeeting%2520Abstracts%26pub%3D2012%2520ACR%252FARHP%2520Annual%2520Meeting%26volume%3D9-14%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, S. L.</span>; <span class="NLM_string-name">Briggs, T. F.</span>; <span class="NLM_string-name">Favaloro, F. G.,  Jr.</span>; <span class="NLM_string-name">Hale, S. P.</span>; <span class="NLM_string-name">Seigal, B. A.</span>; <span class="NLM_string-name">Shortsleeves, K. C.</span>; <span class="NLM_string-name">Snedeker, C.</span>; <span class="NLM_string-name">Taylor, M.</span>; <span class="NLM_string-name">Terrett, N. K.</span>; <span class="NLM_string-name">Yan, D.</span>; <span class="NLM_string-name">Nowak, P. W.</span>; <span class="NLM_string-name">Denny, R. A.</span>; <span class="NLM_string-name">Chrunyk, B. A.</span>; <span class="NLM_string-name">Sahasrabudhe, P. V.</span>; <span class="NLM_string-name">Withka, J. M.</span></span> <span> </span><span class="NLM_article-title">Compounds for Modulating IL-17</span>.  <i>Int. Pat. Appl.</i> <span class="NLM_patent">WO2014066726A2</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+L.+Alexander&author=T.+F.+Briggs&author=F.+G.+Favaloro&author=S.+P.+Hale&author=B.+A.+Seigal&author=K.+C.+Shortsleeves&author=C.+Snedeker&author=M.+Taylor&author=N.+K.+Terrett&author=D.+Yan&author=P.+W.+Nowak&author=R.+A.+Denny&author=B.+A.+Chrunyk&author=P.+V.+Sahasrabudhe&author=J.+M.+Withka&title=Compounds+for+Modulating+IL-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DS.%2BL.%26atitle%3DCompounds%2520for%2520Modulating%2520IL-17%26jtitle%3DInt.%2520Pat.%2520Appl.%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balbo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrunyk, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussenius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalsamy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span> <span> </span><span class="NLM_article-title">Binding site elucidation and structure guided design of macrocyclic IL-17A antagonists</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30859</span>, <span class="refDoi"> DOI: 10.1038/srep30859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fsrep30859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27527709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlylt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=30859&author=S.+Liuauthor=L.+A.+Dakinauthor=L.+Xingauthor=J.+M.+Withkaauthor=P.+V.+Sahasrabudheauthor=W.+Liauthor=M.+E.+Bankerauthor=P.+Balboauthor=S.+Shankerauthor=B.+A.+Chrunykauthor=Z.+Guoauthor=J.+M.+Chenauthor=J.+A.+Youngauthor=G.+Baiauthor=J.+T.+Starrauthor=S.+W.+Wrightauthor=J.+Busseniusauthor=S.+Tanauthor=A.+Gopalsamyauthor=B.+A.+Lefkerauthor=F.+Vincentauthor=L.+H.+Jonesauthor=H.+Xuauthor=L.+R.+Hothauthor=K.+F.+Geogheganauthor=X.+Qiuauthor=M.+E.+Bunnageauthor=A.+Thorarensen&title=Binding+site+elucidation+and+structure+guided+design+of+macrocyclic+IL-17A+antagonists&doi=10.1038%2Fsrep30859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Binding site elucidation and structure guided design of macrocyclic IL-17A antagonists</span></div><div class="casAuthors">Liu, Shenping; Dakin, Leslie A.; Xing, Li; Withka, Jane M.; Sahasrabudhe, Parag V.; Li, Wei; Banker, Mary Ellen; Balbo, Paul; Shanker, Suman; Chrunyk, Boris A.; Guo, Zuojun; Chen, Jinshan M.; Young, Jennifer A.; Bai, Guoyun; Starr, Jeremy T.; Wright, Stephen W.; Bussenius, Joerg; Tan, Sheng; Gopalsamy, Ariamala; Lefker, Bruce A.; Vincent, Fabien; Jones, Lyn H.; Xu, Hua; Hoth, Lise R.; Geoghegan, Kieran F.; Qiu, Xiayang; Bunnage, Mark E.; Thorarensen, Atli</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30859</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interleukin-17A (IL-17A) is a principal driver of multiple inflammatory and immune disorders.  Antibodies that neutralize IL-17A or its receptor (IL-17RA) deliver efficacy in autoimmune diseases, but no small-mol. IL-17A antagonists have yet progressed into clin. trials.  Investigation of a series of linear peptide ligands to IL-17A and characterization of their binding site has enabled the design of novel macrocyclic ligands that are themselves potent IL-17A antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpATWLU4utbrVg90H21EOLACvtfcHk0lh1fHYz2uMD1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlylt7bM&md5=333917e3ad408ed5e48a98f99faf7aca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fsrep30859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep30859%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DXing%26aufirst%3DL.%26aulast%3DWithka%26aufirst%3DJ.%2BM.%26aulast%3DSahasrabudhe%26aufirst%3DP.%2BV.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DBalbo%26aufirst%3DP.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DChrunyk%26aufirst%3DB.%2BA.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DStarr%26aufirst%3DJ.%2BT.%26aulast%3DWright%26aufirst%3DS.%2BW.%26aulast%3DBussenius%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DS.%26aulast%3DGopalsamy%26aufirst%3DA.%26aulast%3DLefker%26aufirst%3DB.%2BA.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DThorarensen%26aufirst%3DA.%26atitle%3DBinding%2520site%2520elucidation%2520and%2520structure%2520guided%2520design%2520of%2520macrocyclic%2520IL-17A%2520antagonists%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D30859%26doi%3D10.1038%2Fsrep30859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ting, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broughton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bina, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodge, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groshong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglicmot, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afshar, S.</span></span> <span> </span><span class="NLM_article-title">Utilization of peptide phage display to investigate hotspots on IL-17A and what it means for drug discovery</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e0190850/1</span>– <span class="NLM_lpage">e0190850/18</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1371%2Fjournal.pone.0190850" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0190850%2F1-e0190850%2F18&author=J.+P.+Tingauthor=F.+Tungauthor=S.+Antonysamyauthor=S.+Wassermanauthor=S.+B.+Jonesauthor=F.+F.+Zhangauthor=A.+Espadaauthor=H.+Broughtonauthor=M.+J.+Chalmersauthor=M.+E.+Woodmanauthor=H.+A.+Binaauthor=J.+A.+Dodgeauthor=J.+Benachauthor=A.+Zhangauthor=C.+Groshongauthor=D.+Manglicmotauthor=M.+Russellauthor=S.+Afshar&title=Utilization+of+peptide+phage+display+to+investigate+hotspots+on+IL-17A+and+what+it+means+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0190850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0190850%26sid%3Dliteratum%253Aachs%26aulast%3DTing%26aufirst%3DJ.%2BP.%26aulast%3DTung%26aufirst%3DF.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWasserman%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DS.%2BB.%26aulast%3DZhang%26aufirst%3DF.%2BF.%26aulast%3DEspada%26aufirst%3DA.%26aulast%3DBroughton%26aufirst%3DH.%26aulast%3DChalmers%26aufirst%3DM.%2BJ.%26aulast%3DWoodman%26aufirst%3DM.%2BE.%26aulast%3DBina%26aufirst%3DH.%2BA.%26aulast%3DDodge%26aufirst%3DJ.%2BA.%26aulast%3DBenach%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DGroshong%26aufirst%3DC.%26aulast%3DManglicmot%26aufirst%3DD.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DAfshar%26aufirst%3DS.%26atitle%3DUtilization%2520of%2520peptide%2520phage%2520display%2520to%2520investigate%2520hotspots%2520on%2520IL-17A%2520and%2520what%2520it%2520means%2520for%2520drug%2520discovery%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26spage%3De0190850%252F1%26epage%3De0190850%252F18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span>. <i>Drug Discov. Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lhSgN_vlu0Q0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discov.%2520Today%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alihodžić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukvić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenkov, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutinec, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pešić, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašković, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žiher, D.</span></span> <span> </span><span class="NLM_article-title">Current trends in macrocyclic drug discovery and beyond-Ro5</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1016/bs.pmch.2018.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fbs.pmch.2018.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=29680148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=113-233&author=S.+Alihod%C5%BEi%C4%87author=M.+Bukvi%C4%87author=I.+J.+Elenkovauthor=A.+Hutinecauthor=S.+Ko%C5%A1trunauthor=D.+Pe%C5%A1i%C4%87author=G.+Saxtyauthor=L.+Toma%C5%A1kovi%C4%87author=D.+%C5%BDiher&title=Current+trends+in+macrocyclic+drug+discovery+and+beyond-Ro5&doi=10.1016%2Fbs.pmch.2018.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Macrocyclic Drug Discovery and beyond-Ro5</span></div><div class="casAuthors">Alihodzic Sulejman; Bukvic Mirjana; Elenkov Ivaylo J; Hutinec Antun; Kostrun Sanja; Pesic Dijana; Saxty Gordon; Tomaskovic Linda; Ziher Dinko</div><div class="citationInfo"><span class="NLM_cas:title">Progress in medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-233</span>
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    </div><div class="casAbstract">This chapter will discuss the recent literature of macrocycles and drug-like property space moving beyond the rule of five (bRo5).  Trends in chemical classes that fall within this definition are discussed and the impact of the latest technologies in the field assessed.  The physicochemical properties, which have provided both successes and challenges, especially in scale-up, are discussed.  A recent patent literature is reviewed and the chapter concludes with a perspective on the future of macrocyclic drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZ9pfcWWVZ5hWfpMXQLRstfW6udTcc2eZloO985bv6Irntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D&md5=ac41d1a69cb0056d106d554b0b323e61</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fbs.pmch.2018.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.pmch.2018.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DS.%26aulast%3DBukvi%25C4%2587%26aufirst%3DM.%26aulast%3DElenkov%26aufirst%3DI.%2BJ.%26aulast%3DHutinec%26aufirst%3DA.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DPe%25C5%25A1i%25C4%2587%26aufirst%3DD.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DToma%25C5%25A1kovi%25C4%2587%26aufirst%3DL.%26aulast%3D%25C5%25BDiher%26aufirst%3DD.%26atitle%3DCurrent%2520trends%2520in%2520macrocyclic%2520drug%2520discovery%2520and%2520beyond-Ro5%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D57%26spage%3D113%26epage%3D233%26doi%3D10.1016%2Fbs.pmch.2018.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stotani, S</span></span>, <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span> <span> </span><span class="NLM_article-title">Overview of Macrocycles in Clinical Development and Clinically Used</span>. In:  <i>Practical Medicinal Chemistry with Macrocycles: Design, Synthesis, and Case Studies</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span>; <span class="NLM_string-name">Peterson, M. L.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">Hoboken</span>, <span class="NLM_year">2017</span>; pp.  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">500</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2F9781119092599.ch16" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=413-500&author=S+Stotaniauthor=F.+Giordanettoauthor=E.+Marsault&author=M.+L.+Peterson&title=Practical+Medicinal+Chemistry+with+Macrocycles%3A+Design%2C+Synthesis%2C+and+Case+Studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F9781119092599.ch16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781119092599.ch16%26sid%3Dliteratum%253Aachs%26aulast%3DStotani%26aufirst%3DS%26aulast%3DGiordanetto%26aufirst%3DF.%26atitle%3DOverview%2520of%2520Macrocycles%2520in%2520Clinical%2520Development%2520and%2520Clinically%2520Used%26btitle%3DPractical%2520Medicinal%2520Chemistry%2520with%2520Macrocycles%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Case%2520Studies%26aulast%3DMarsault%26aufirst%3DE.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2017%26spage%3D413%26epage%3D500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seiple, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubec, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois-Mercier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hog, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuzaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsen, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Condakes, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczypiński, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, A. G.</span></span> <span> </span><span class="NLM_article-title">A platform for the discovery of new macrolide antibiotics</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">338</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1038/nature17967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fnature17967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27193679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xot1SmtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=338-345&author=I.+B.+Seipleauthor=Z.+Zhangauthor=P.+Jakubecauthor=A.+Langlois-Mercierauthor=P.+M.+Wrightauthor=D.+T.+Hogauthor=K.+Yabuauthor=S.+R.+Alluauthor=T.+Fukuzakiauthor=P.+N.+Carlsenauthor=Y.+Kitamuraauthor=X.+Zhouauthor=M.+L.+Condakesauthor=F.+T.+Szczypi%C5%84skiauthor=W.+D.+Greenauthor=A.+G.+Myers&title=A+platform+for+the+discovery+of+new+macrolide+antibiotics&doi=10.1038%2Fnature17967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A platform for the discovery of new macrolide antibiotics</span></div><div class="casAuthors">Seiple, Ian B.; Zhang, Ziyang; Jakubec, Pavol; Langlois-Mercier, Audrey; Wright, Peter M.; Hog, Daniel T.; Yabu, Kazuo; Allu, Senkara Rao; Fukuzaki, Takehiro; Carlsen, Peter N.; Kitamura, Yoshiaki; Zhou, Xiang; Condakes, Matthew L.; Szczypinski, Filip T.; Green, William D.; Myers, Andrew G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7603</span>),
    <span class="NLM_cas:pages">338-345</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The chem. modification of structurally complex fermn. products, a process known as semisynthesis, has been an important tool in the discovery and manuf. of antibiotics for the treatment of various infectious diseases.  However, many of the therapeutics obtained in this way are no longer effective, because bacterial resistance to these compds. has developed.  Here, we present a practical, fully synthetic route to macrolide antibiotics by the convergent assembly of simple chem. building blocks, enabling the synthesis of diverse structures not accessible by traditional semisynthetic approaches.  More than 300 new macrolide antibiotic candidates, as well as the clin. candidate solithromycin, have been synthesized using our convergent approach.  Evaluation of these compds. against a panel of pathogenic bacteria revealed that the majority of these structures had antibiotic activity, some efficacious against strains resistant to macrolides in current use.  The chem. we describe here provides a platform for the discovery of new macrolide antibiotics and may also serve as the basis for their manuf.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGXBS8IF57cbVg90H21EOLACvtfcHk0lgy4stpmlpvsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xot1SmtLw%253D&md5=be2af0c211fdc10ef5f1203833bd6c9b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature17967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17967%26sid%3Dliteratum%253Aachs%26aulast%3DSeiple%26aufirst%3DI.%2BB.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DJakubec%26aufirst%3DP.%26aulast%3DLanglois-Mercier%26aufirst%3DA.%26aulast%3DWright%26aufirst%3DP.%2BM.%26aulast%3DHog%26aufirst%3DD.%2BT.%26aulast%3DYabu%26aufirst%3DK.%26aulast%3DAllu%26aufirst%3DS.%2BR.%26aulast%3DFukuzaki%26aufirst%3DT.%26aulast%3DCarlsen%26aufirst%3DP.%2BN.%26aulast%3DKitamura%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DCondakes%26aufirst%3DM.%2BL.%26aulast%3DSzczypi%25C5%2584ski%26aufirst%3DF.%2BT.%26aulast%3DGreen%26aufirst%3DW.%2BD.%26aulast%3DMyers%26aufirst%3DA.%2BG.%26atitle%3DA%2520platform%2520for%2520the%2520discovery%2520of%2520new%2520macrolide%2520antibiotics%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D338%26epage%3D345%26doi%3D10.1038%2Fnature17967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclic peptide therapeutics: past, present and future</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.cbpa.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=28249193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=24-29&author=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclic+peptide+therapeutics%3A+past%2C+present+and+future&doi=10.1016%2Fj.cbpa.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide therapeutics: past, present and future</span></div><div class="casAuthors">Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-29</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic peptides combine several favorable properties such as good binding affinity, target selectivity and low toxicity that make them an attractive modality for the development of therapeutics.  Over 40 cyclic peptide drugs are currently in clin. use and around one new cyclic peptide drug enters the market every year on av.  The vast majority of clin. approved cyclic peptides are derived from natural products, such as antimicrobials or human peptide hormones.  New powerful techniques based on rational design and in vitro evolution have enabled the de novo development of cyclic peptide ligands to targets for which nature does not offer solns.  A look at the cyclic peptides currently under clin. evaluation shows that several have been developed using such techniques.  This new source for cyclic peptide ligands introduces a freshness to the field, and it is likely that de novo developed cyclic peptides will be in clin. use in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwchcMOj1bSbVg90H21EOLACvtfcHk0lgy4stpmlpvsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D&md5=4b9581365b92b649dad8c7b22c5bd9c3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclic%2520peptide%2520therapeutics%253A%2520past%252C%2520present%2520and%2520future%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D24%26epage%3D29%26doi%3D10.1016%2Fj.cbpa.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emery, F. S.</span></span> <span> </span><span class="NLM_article-title">Oral administration of peptide-based drugs: beyond Lipinski’s rule</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2245</span>– <span class="NLM_lpage">2251</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2Fcmdc.201600288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27596610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2245-2251&author=G.+B.+Santosauthor=A.+Ganesanauthor=F.+S.+Emery&title=Oral+administration+of+peptide-based+drugs%3A+beyond+Lipinski%E2%80%99s+rule&doi=10.1002%2Fcmdc.201600288"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Administration of Peptide-Based Drugs: Beyond Lipinski's Rule</span></div><div class="casAuthors">Santos, Gabriela B.; Ganesan, A.; Emery, Flavio S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2245-2251</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The use of peptides in therapy presents several limitations, from physicochem. characteristics to inadequate pharmacokinetic profiles for oral absorption.  As peptides are gaining importance in the therapeutic arsenal, there is an increasing need to rationalize the main characteristics of this compd. class in the market.  Therefore, we performed an extensive anal. of all known peptide drugs and clin. candidates based on their peptide features, physicochem. and structural properties, and correlated these with their administration route and therapeutic classes.  Peptide drugs are widely distributed across drug and pharmacol. space, covering several therapeutic areas with structural diversity and complexity, distributed between groups of cyclic and linear compds.  Although structural and physicochem. properties are clear within these groups, we counter the consensus that cyclic peptides have better oral availability than linear peptides, as most of the orally administrated peptides have linear structures.  This study and review furnishes information that could support peptide drug design, with a new cutoff of known descriptors that go beyond the Rule of Five.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr88nw1qqAJuLVg90H21EOLACvtfcHk0lipWc9E2k7U8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVGgu7rI&md5=b2e5b6a1dff54c2c1f85671ab7314031</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600288%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DG.%2BB.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DEmery%26aufirst%3DF.%2BS.%26atitle%3DOral%2520administration%2520of%2520peptide-based%2520drugs%253A%2520beyond%2520Lipinski%25E2%2580%2599s%2520rule%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2245%26epage%3D2251%26doi%3D10.1002%2Fcmdc.201600288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fajdetić, A.</span>; <span class="NLM_string-name">Matanović
Škugor, M.</span>; <span class="NLM_string-name">Elenkov, I.</span>; <span class="NLM_string-name">Kragol, G.</span>; <span class="NLM_string-name">Bukvić, M.</span>; <span class="NLM_string-name">Marušić Ištuk, Z.</span>; <span class="NLM_string-name">Koštrun, S.</span>; <span class="NLM_string-name">Žiher, D.</span>; <span class="NLM_string-name">Rupčić, R.</span>; <span class="NLM_string-name">Butković, K.</span>, <span class="NLM_string-name">Dukši, M.</span>; <span class="NLM_string-name">Ozimec Landek, I.</span>; <span class="NLM_string-name">Pešić, D.</span>; <span class="NLM_string-name">Hutinec, A.</span>; <span class="NLM_string-name">Mesić, M.</span>; <span class="NLM_string-name">Saxty, G.</span>; <span class="NLM_string-name">Poljak, V.</span></span> <span> </span><span class="NLM_article-title">Preparation of Seco Macrolide Aminoglycosides for Use in the Treatment of Prophylaxis</span>.  <i>Int. Pat. Appl.</i> <span class="NLM_patent">WO2017194452A1</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=A.+Fajdeti%C4%87&author=M.+Matanovi%C4%87%0A%C5%A0kugor&author=I.+Elenkov&author=G.+Kragol&author=M.+Bukvi%C4%87&author=Z.+Maru%C5%A1i%C4%87+I%C5%A1tuk&author=S.+Ko%C5%A1trun&author=D.+%C5%BDiher&author=R.+Rup%C4%8Di%C4%87&author=K.+Butkovi%C4%87&author=M.+Duk%C5%A1i&author=I.+Ozimec+Landek&author=D.+Pe%C5%A1i%C4%87&author=A.+Hutinec&author=M.+Mesi%C4%87&author=G.+Saxty&author=V.+Poljak&title=Preparation+of+Seco+Macrolide+Aminoglycosides+for+Use+in+the+Treatment+of+Prophylaxis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFajdeti%25C4%2587%26aufirst%3DA.%26atitle%3DPreparation%2520of%2520Seco%2520Macrolide%2520Aminoglycosides%2520for%2520Use%2520in%2520the%2520Treatment%2520of%2520Prophylaxis%26jtitle%3DInt.%2520Pat.%2520Appl.%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrunyk, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marr, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffor, M. C.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of interleukin 17A and its complex with IL-17 receptor A</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1888</span>, <span class="refDoi"> DOI: 10.1038/ncomms2880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fncomms2880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=23695682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A280%3ADC%252BC3snlvFCrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1888&author=S.+Liuauthor=X.+Songauthor=B.+A.+Chrunykauthor=S.+Shankerauthor=L.+R.+Hothauthor=E.+S.+Marrauthor=M.+C.+Griffor&title=Crystal+structures+of+interleukin+17A+and+its+complex+with+IL-17+receptor+A&doi=10.1038%2Fncomms2880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of interleukin 17A and its complex with IL-17 receptor A</span></div><div class="casAuthors">Liu Shenping; Song Xi; Chrunyk Boris A; Shanker Suman; Hoth Lise R; Marr Eric S; Griffor Matthew C</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1888</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The constituent polypeptides of the interleukin-17 family form six different homodimeric cytokines (IL-17A-F) and the heterodimeric IL-17A/F.  Their interactions with IL-17 receptors A-E (IL-17RA-E) mediate host defenses while also contributing to inflammatory and autoimmune responses.  IL-17A and IL-17F both preferentially engage a receptor complex containing one molecule of IL-17RA and one molecule of IL-17RC.  More generally, IL-17RA appears to be a shared receptor that pairs with other members of its family to allow signaling of different IL-17 cytokines.  Here we report crystal structures of homodimeric IL-17A and its complex with IL-17RA.  Binding to IL-17RA at one side of the IL-17A molecule induces a conformational change in the second, symmetry-related receptor site of IL-17A.  This change favors, and is sufficient to account for, the selection of a different receptor polypeptide to complete the cytokine-receptor complex.  The structural results are supported by biophysical studies with IL-17A variants produced by site-directed mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQPgY8GEu_hzSSm0e6ZOaemfW6udTcc2ebSDaECOG48-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snlvFCrsw%253D%253D&md5=18a5bb02c1f0389b7e02277d0860c2e2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fncomms2880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2880%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DChrunyk%26aufirst%3DB.%2BA.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DHoth%26aufirst%3DL.%2BR.%26aulast%3DMarr%26aufirst%3DE.%2BS.%26aulast%3DGriffor%26aufirst%3DM.%2BC.%26atitle%3DCrystal%2520structures%2520of%2520interleukin%252017A%2520and%2520its%2520complex%2520with%2520IL-17%2520receptor%2520A%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D1888%26doi%3D10.1038%2Fncomms2880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerhardt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauptit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Needham, M. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langham, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aziz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snow, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welsh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaugan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beste, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sleeman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coales, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamuro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, C.</span></span> <span> </span><span class="NLM_article-title">Structure of IL-17A in complex with a potent, fully human neutralizing antibody</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.jmb.2009.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=19835883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKnurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2009&pages=905-921&author=S.+Gerhardtauthor=W.+M.+Abbottauthor=D.+Hargreavesauthor=R.+A.+Pauptitauthor=R.+A.+Daviesauthor=M.+R.+C.+Needhamauthor=C.+Langhamauthor=W.+Barkerauthor=A.+Azizauthor=M.+J.+Snowauthor=S.+Dawsonauthor=F.+Welshauthor=T.+Wilkinsonauthor=T.+Vauganauthor=G.+Besteauthor=S.+Bishopauthor=B.+Popovicauthor=G.+Reesauthor=M.+Sleemanauthor=S.+J.+Tuskeauthor=S.+J.+Coalesauthor=Y.+Hamuroauthor=C.+Russell&title=Structure+of+IL-17A+in+complex+with+a+potent%2C+fully+human+neutralizing+antibody&doi=10.1016%2Fj.jmb.2009.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of IL-17A in Complex with a Potent, Fully Human Neutralizing Antibody</span></div><div class="casAuthors">Gerhardt, Stefan; Abbott, W. Mark; Hargreaves, David; Pauptit, Richard A.; Davies, Rick A.; Needham, Maurice R. C.; Langham, Caroline; Barker, Wendy; Aziz, Azad; Snow, Melanie J.; Dawson, Sarah; Welsh, Fraser; Wilkinson, Trevor; Vaugan, Tris; Beste, Gerald; Bishop, Sarah; Popovic, Bojana; Rees, Gareth; Sleeman, Matthew; Tuske, Steven J.; Coales, Stephen J.; Hamuro, Yoshitomo; Russell, Caroline</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">394</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">905-921</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">IL-17A is a pro-inflammatory cytokine produced by the newly identified Th17 subset of T-cells.  We have isolated a human monoclonal antibody to IL-17A (CAT-2200) that can potently neutralize the effects of recombinant and native human IL-17A.  We detd. the crystal structure of IL-17A in complex with the CAT-2200 Fab at 2.6 Å resoln. in order to provide a definitive characterization of the epitope and paratope regions.  Approx. a third of the IL-17A dimer is disordered in this crystal structure.  The disorder occurs in both independent copies of the complex in the asym. unit and does not appear to be influenced by crystal packing.  The complex contains one IL-17A dimer sandwiched between two CAT-2200 Fab fragments.  The IL-17A is a disulfide-linked homodimer that is similar in structure to IL-17F, adopting a cystine-knot fold.  The structure is not inconsistent with the previous prediction of a receptor binding cavity on IL-17 family members.  The epitope recognized by CAT-2200 is shown to involve 12 amino acid residues from the quaternary structure of IL-17A, with each Fab contacting both monomers in the dimer.  All complementarity-detg. regions (CDRs) in the Fab contribute to a total of 16 amino acid residues in the antibody paratope.  In vitro affinity optimization was used to generate CAT-2200 from a parental lead antibody using random mutagenesis of CDR3 loops.  This resulted in seven amino acid changes (three in VL-CDR3 and four in VH-CDR3) and gave an approx. 30-fold increase in potency in a cell-based neutralization assay.  Two of the seven amino acids form part of the CAT-2200 paratope.  The obsd. interaction site between CAT-2200 and IL-17A is consistent with data from hydrogen/deuterium exchange mass spectrometry and mutagenesis approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bLLa1mvN07Vg90H21EOLACvtfcHk0licdKKqi8giLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKnurvI&md5=c79a71a621832d30cf705ba70563096f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DGerhardt%26aufirst%3DS.%26aulast%3DAbbott%26aufirst%3DW.%2BM.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DPauptit%26aufirst%3DR.%2BA.%26aulast%3DDavies%26aufirst%3DR.%2BA.%26aulast%3DNeedham%26aufirst%3DM.%2BR.%2BC.%26aulast%3DLangham%26aufirst%3DC.%26aulast%3DBarker%26aufirst%3DW.%26aulast%3DAziz%26aufirst%3DA.%26aulast%3DSnow%26aufirst%3DM.%2BJ.%26aulast%3DDawson%26aufirst%3DS.%26aulast%3DWelsh%26aufirst%3DF.%26aulast%3DWilkinson%26aufirst%3DT.%26aulast%3DVaugan%26aufirst%3DT.%26aulast%3DBeste%26aufirst%3DG.%26aulast%3DBishop%26aufirst%3DS.%26aulast%3DPopovic%26aufirst%3DB.%26aulast%3DRees%26aufirst%3DG.%26aulast%3DSleeman%26aufirst%3DM.%26aulast%3DTuske%26aufirst%3DS.%2BJ.%26aulast%3DCoales%26aufirst%3DS.%2BJ.%26aulast%3DHamuro%26aufirst%3DY.%26aulast%3DRussell%26aufirst%3DC.%26atitle%3DStructure%2520of%2520IL-17A%2520in%2520complex%2520with%2520a%2520potent%252C%2520fully%2520human%2520neutralizing%2520antibody%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D394%26spage%3D905%26epage%3D921%26doi%3D10.1016%2Fj.jmb.2009.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desharnais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oates, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banker, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrunyk, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumelty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodnutt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lappe, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorarensen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, L. D.</span></span> <span> </span><span class="NLM_article-title">Inhibiting complex IL-17A and IL-17RA interactions with a linear peptide</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">26071</span>, <span class="refDoi"> DOI: 10.1038/srep26071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fsrep26071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27184415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28Xot1Gnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=26071&author=S.+Liuauthor=J.+Desharnaisauthor=P.+V.+Sahasrabudheauthor=P.+Jinauthor=W.+Liauthor=B.+D.+Oatesauthor=S.+Shankerauthor=M.+E.+Bankerauthor=B.+A.+Chrunykauthor=X.+Songauthor=X.+Fengauthor=M.+Grifforauthor=J.+Jimenezauthor=G.+Chenauthor=D.+Tumeltyauthor=A.+Bhatauthor=C.+W.+Bradshawauthor=G.+Woodnuttauthor=R.+W.+Lappeauthor=A.+Thorarensenauthor=X.+Qiuauthor=J.+M.+Withkaauthor=L.+D.+Wood&title=Inhibiting+complex+IL-17A+and+IL-17RA+interactions+with+a+linear+peptide&doi=10.1038%2Fsrep26071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting complex IL-17A and IL-17RA interactions with a linear peptide</span></div><div class="casAuthors">Liu, Shenping; Desharnais, Joel; Sahasrabudhe, Parag V.; Jin, Ping; Li, Wei; Oates, Bryan D.; Shanker, Suman; Banker, Mary Ellen; Chrunyk, Boris A.; Song, Xi; Feng, Xidong; Griffor, Matt; Jimenez, Judith; Chen, Gang; Tumelty, David; Bhat, Abhijit; Bradshaw, Curt W.; Woodnutt, Gary; Lappe, Rodney W.; Thorarensen, Atli; Qiu, Xiayang; Withka, Jane M.; Wood, Lauren D.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26071</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">IL-17A is a pro-inflammatory cytokine that has been implicated in autoimmune and inflammatory diseases.  Monoclonal antibodies inhibiting IL-17A signaling have demonstrated remarkable efficacy, but an oral therapy is still lacking.  A high affinity IL-17A peptide antagonist (HAP) of 15 residues was identified through phage-display screening followed by satn. mutagenesis optimization and amino acid substitutions.  HAP binds specifically to IL-17A and inhibits the interaction of the cytokine with its receptor, IL-17RA.  Tested in primary human cells, HAP blocked the prodn. of multiple inflammatory cytokines.  Crystal structure studies revealed that two HAP mols. bind to one IL-17A dimer sym.  The N-terminal portions of HAP form a β-strand that inserts between two IL-17A monomers while the C-terminal section forms an α helix that directly blocks IL-17RA from binding to the same region of IL-17A.  This mode of inhibition suggests opportunities for developing peptide antagonists against this challenging target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLkbdJqT9hyrVg90H21EOLACvtfcHk0lgUaDDKlc2bOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xot1Gnurg%253D&md5=73da55aa47ada18bcb1820b99969f568</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fsrep26071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep26071%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDesharnais%26aufirst%3DJ.%26aulast%3DSahasrabudhe%26aufirst%3DP.%2BV.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DOates%26aufirst%3DB.%2BD.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DBanker%26aufirst%3DM.%2BE.%26aulast%3DChrunyk%26aufirst%3DB.%2BA.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DGriffor%26aufirst%3DM.%26aulast%3DJimenez%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DTumelty%26aufirst%3DD.%26aulast%3DBhat%26aufirst%3DA.%26aulast%3DBradshaw%26aufirst%3DC.%2BW.%26aulast%3DWoodnutt%26aufirst%3DG.%26aulast%3DLappe%26aufirst%3DR.%2BW.%26aulast%3DThorarensen%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWithka%26aufirst%3DJ.%2BM.%26aulast%3DWood%26aufirst%3DL.%2BD.%26atitle%3DInhibiting%2520complex%2520IL-17A%2520and%2520IL-17RA%2520interactions%2520with%2520a%2520linear%2520peptide%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D26071%26doi%3D10.1038%2Fsrep26071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanikawa, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis and antibacterial activity of a novel class of 15-membered macrolide antibiotics, “11a-azalides.”</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1021/ml100252s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100252s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVan" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=234-237&author=T.+Sugimotoauthor=T.+Tanikawa&title=Synthesis+and+antibacterial+activity+of+a+novel+class+of+15-membered+macrolide+antibiotics%2C+%E2%80%9C11a-azalides.%E2%80%9D&doi=10.1021%2Fml100252s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and antibacterial activity of a novel class of 15-membered macrolide antibiotics, "11a-azalides"</span></div><div class="casAuthors">Sugimoto, Tomohiro; Tanikawa, Tetsuya</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-237</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient method for the reconstruction of the 9-dihydroerythromycin A macrolactone skeleton has been established.  The key steps are oxidative cleavage at the 11,12-position and reconstruction after insertion of an appropriate functionalized amino alc.  Novel 15-membered macrolides, named as "11a-azalides", were synthesized based on the above methodol. and evaluated for their antibacterial activity.  Among them, (13R)-benzyloxymethyl-11a-azalide showed the most potent Streptococcus pneumoniae activity, with improved activity against a representative erythromycin-resistant strain compared to clarithromycin (CAM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7VwCBoxE6BLVg90H21EOLACvtfcHk0lgUaDDKlc2bOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVan&md5=45025a54eb891d2a3d861cbf26a823b3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fml100252s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100252s%26sid%3Dliteratum%253Aachs%26aulast%3DSugimoto%26aufirst%3DT.%26aulast%3DTanikawa%26aufirst%3DT.%26atitle%3DSynthesis%2520and%2520antibacterial%2520activity%2520of%2520a%2520novel%2520class%2520of%252015-membered%2520macrolide%2520antibiotics%252C%2520%25E2%2580%259C11a-azalides.%25E2%2580%259D%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D234%26epage%3D237%26doi%3D10.1021%2Fml100252s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Everett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, J. W.</span></span> <span> </span><span class="NLM_article-title">The conformational analysis of erythromycin A</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1659</span>– <span class="NLM_lpage">1667</span>, <span class="refDoi"> DOI: 10.1039/P29870001659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1039%2Fp29870001659" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1987&pages=1659-1667&author=J.+R.+Everettauthor=J.+W.+Tyler&title=The+conformational+analysis+of+erythromycin+A&doi=10.1039%2FP29870001659"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2Fp29870001659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29870001659%26sid%3Dliteratum%253Aachs%26aulast%3DEverett%26aufirst%3DJ.%2BR.%26aulast%3DTyler%26aufirst%3DJ.%2BW.%26atitle%3DThe%2520conformational%2520analysis%2520of%2520erythromycin%2520A%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1987%26volume%3D11%26spage%3D1659%26epage%3D1667%26doi%3D10.1039%2FP29870001659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazarevski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinković, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobrehel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Đokić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metelko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vikić-Topić, D.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of azithromycin by NMR spectroscopy and molecular modeling</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/S0040-4020(01)86274-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2FS0040-4020%2801%2986274-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADyaK3sXht1Cru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1993&pages=721-730&author=G.+Lazarevskiauthor=M.+Vinkovi%C4%87author=G.+Kobrehelauthor=S.+%C4%90oki%C4%87author=B.+Metelkoauthor=D.+Viki%C4%87-Topi%C4%87&title=Conformational+analysis+of+azithromycin+by+NMR+spectroscopy+and+molecular+modeling&doi=10.1016%2FS0040-4020%2801%2986274-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of azithromycin by NMR spectroscopy and molecular modeling</span></div><div class="casAuthors">Lazarevski, Gorjana; Vinkovic, Mladen; Kobrehel, Gabrijela; Dokic, Slobodan; Metelko, Biserka; Vikic-Topic, Drazen</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">721-30</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">The conformation of azithromycin (I) in the soln. was detd. by NMR spectroscopy and mol. mechanics calcns. and compared with its crystal structure and with some erythromycin derivs.  In soln. I exists predominantly in a "folded-in" conformation in the C(3) to C(5) region, whereas its crystal state conformation is "folded-out".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruw1FZO-XoOLVg90H21EOLACvtfcHk0lhdRPYjAWgV6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXht1Cru7s%253D&md5=8b346c265609627263d82b19523953e2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2801%2986274-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252801%252986274-8%26sid%3Dliteratum%253Aachs%26aulast%3DLazarevski%26aufirst%3DG.%26aulast%3DVinkovi%25C4%2587%26aufirst%3DM.%26aulast%3DKobrehel%26aufirst%3DG.%26aulast%3D%25C4%2590oki%25C4%2587%26aufirst%3DS.%26aulast%3DMetelko%26aufirst%3DB.%26aulast%3DViki%25C4%2587-Topi%25C4%2587%26aufirst%3DD.%26atitle%3DConformational%2520analysis%2520of%2520azithromycin%2520by%2520NMR%2520spectroscopy%2520and%2520molecular%2520modeling%26jtitle%3DTetrahedron%26date%3D1993%26volume%3D49%26spage%3D721%26epage%3D730%26doi%3D10.1016%2FS0040-4020%2801%2986274-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span> <span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1784</span>– <span class="NLM_lpage">1788</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lhdRPYjAWgV6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stepanić, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žiher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica-Marković, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jelić, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span> <span> </span><span class="NLM_article-title">Physicochemical profile of macrolides and their comparison with small molecules</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2011.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.ejmech.2011.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=22152985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVyqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=462-472&author=V.+Stepani%C4%87author=D.+%C5%BDiherauthor=V.+Gabelica-Markovi%C4%87author=D.+Jeli%C4%87author=S.+Nunhuckauthor=K.+Valkoauthor=S.+Ko%C5%A1trun&title=Physicochemical+profile+of+macrolides+and+their+comparison+with+small+molecules&doi=10.1016%2Fj.ejmech.2011.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical profile of macrolides and their comparison with small molecules</span></div><div class="casAuthors">Stepanic, Visnja; Ziher, Dinko; Gabelica-Markovic, Vesna; Jelic, Dubravko; Nunhuck, Shenaz; Valko, Klara; Kostrun, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">462-472</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Macrolides are stereospecific macrolactones of high mol. wts.  Herein, 600 mostly semisynthetic macrolides are compared with 50,000 small non-macrolide synthetic mols. in terms of measured physicochem. properties in order to assess the drug-likeness and developability chances of macrolides.  The pre-selected set of diverse macrolides is comprised mostly of derivs. of clarithromycin and azithromycin cores.  Lipophilicity (CHI logD), affinity for immobilized artificial membranes (CHI IAM), human serum albumin (HSA) and α1-acid glycoprotein (AGP) plasma protein bindings (PPB), DMSO precipitative soly. as well as artificial membrane permeability (AMP) have been detd. by high-throughput screening methods.  It has been found that macrolides and small mols. have similar lipophilicity profiles, though macrolides show weaker PPB and have better soly. than small discovery mols.  However, macrolides are poorly permeable and have high affinity for immobilized artificial membranes signifying their strong interaction with biol. phospholipids.  In order to retain the drug-like profile, the design of novel macrolide mols. should be focused on optimization of macrolide cores, that is macrolactone moiety with sugars and other small substituents avoiding large substituents and flexible linkers such as in conjugate derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpyTdevkgZsrVg90H21EOLACvtfcHk0lhdRPYjAWgV6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVyqsQ%253D%253D&md5=203aec4ec40cecb71feaf36345d2fe4d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2011.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2011.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DStepani%25C4%2587%26aufirst%3DV.%26aulast%3D%25C5%25BDiher%26aufirst%3DD.%26aulast%3DGabelica-Markovi%25C4%2587%26aufirst%3DV.%26aulast%3DJeli%25C4%2587%26aufirst%3DD.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26atitle%3DPhysicochemical%2520profile%2520of%2520macrolides%2520and%2520their%2520comparison%2520with%2520small%2520molecules%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D47%26spage%3D462%26epage%3D472%26doi%3D10.1016%2Fj.ejmech.2011.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0ljeiFnE58xlWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljeiFnE58xlWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahlbach, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Beyond cyclosporine A: conformation-dependent passive membrane permeabilities of cyclic peptide natural products</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2121</span>– <span class="NLM_lpage">2130</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.78</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.4155%2Ffmc.15.78" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=26067057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVehs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=2121-2130&author=C.+L.+Ahlbachauthor=K.+W.+Lexaauthor=A.+T.+Bockusauthor=V.+Chenauthor=P.+Crewsauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Beyond+cyclosporine+A%3A+conformation-dependent+passive+membrane+permeabilities+of+cyclic+peptide+natural+products&doi=10.4155%2Ffmc.15.78"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond cyclosporine A: conformation-dependent passive membrane permeabilities of cyclic peptide natural products</span></div><div class="casAuthors">Ahlbach, Christopher L.; Lexa, Katrina W.; Bockus, Andrew T.; Chen, Valerie; Crews, Phillip; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2121-2130</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Many cyclic peptide natural products are larger and structurally more complex than conventional small mol. drugs.  Although some mols. in this class are known to possess favorable pharmacokinetic properties, there have been few reports on the membrane permeabilities of cyclic peptide natural products.  Here, we present the passive membrane permeabilities of 39 cyclic peptide natural products, and interpret the results using a computational permeability prediction algorithm based on their known or calcd. 3D conformations.  We found that the permeabilities of these compds., measured in a parallel artificial membrane permeability assay, spanned a wide range and demonstrated the important influence of conformation on membrane permeability.  These results will aid in the development of these compds. as a viable drug paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4_Ar_CGW03bVg90H21EOLACvtfcHk0lj215E7LeVDtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVehs7%252FM&md5=69d8628920ebc6ad72b13162f3214f2d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.78%26sid%3Dliteratum%253Aachs%26aulast%3DAhlbach%26aufirst%3DC.%2BL.%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DV.%26aulast%3DCrews%26aufirst%3DP.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DBeyond%2520cyclosporine%2520A%253A%2520conformation-dependent%2520passive%2520membrane%2520permeabilities%2520of%2520cyclic%2520peptide%2520natural%2520products%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D2121%26epage%3D2130%26doi%3D10.4155%2Ffmc.15.78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cell-permeable cyclic peptides from synthetic libraries inspired by natural products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1021/ja508766b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja508766b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=715-721&author=W.+M.+Hewittauthor=S.+S.+F.+Leungauthor=C.+R.+Pyeauthor=A.+R.+Ponkeyauthor=M.+Bednarekauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Cell-permeable+cyclic+peptides+from+synthetic+libraries+inspired+by+natural+products&doi=10.1021%2Fja508766b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span></div><div class="casAuthors">Hewitt, William M.; Leung, Siegfried S. F.; Pye, Cameron R.; Ponkey, Alexandra R.; Bednarek, Maria; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">715-721</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug design efforts are turning to a new generation of therapeutic targets, such as protein-protein interactions (PPIs), that had previously been considered "undruggable" by typical small mols.  There is an emerging view that accessing these targets will require mols. that are larger and more complex than typical small mol. drugs.  Here, the authors present a methodol. for the discovery of geometrically diverse, membrane permeable cyclic peptide scaffolds based on the synthesis and permeability screening of a combinatorial library, followed by deconvolution of membrane-permeable scaffolds to identify cyclic peptides with good to excellent passive cell permeabilities.  The authors use a combination of exptl. and computational approaches to investigate structure-permeability relationships in one of these scaffolds, and uncover structural and conformational factors that govern passive membrane diffusion in a related set of cyclic peptide diastereomers.  Further, the authors investigate the dependency of permeability on side-chain identity of one of these scaffolds through single-point diversifications to show the adaptability of these scaffolds toward development of permeability-biased libraries suitable for bioactivity screens.  Overall, the results demonstrate that many novel, cell permeable scaffolds exist beyond those found in extant natural products, and that such scaffolds can be rapidly identified using a combination of synthesis and deconvolution which can, in principle, be applied to any type of macrocyclic template.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wbqbKaUG9rVg90H21EOLACvtfcHk0lj215E7LeVDtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF&md5=0d112bf0da119b6a50b55997f64ac214</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja508766b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja508766b%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DBednarek%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCell-permeable%2520cyclic%2520peptides%2520from%2520synthetic%2520libraries%2520inspired%2520by%2520natural%2520products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D715%26epage%3D721%26doi%3D10.1021%2Fja508766b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poullennec, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakemore, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiva, J.-Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Highton, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoisin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9473</span>– <span class="NLM_lpage">9499</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00802</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00802" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9473-9499&author=R.+L.+Mackmanauthor=V.+A.+Steadmanauthor=D.+K.+Deanauthor=P.+Jansaauthor=K.+G.+Poullennecauthor=T.+Applebyauthor=C.+Austinauthor=C.+A.+Blakemoreauthor=R.+Caiauthor=C.+Cannizzaroauthor=G.+Chinauthor=J.-Y.+C.+Chivaauthor=N.+A.+Dunbarauthor=H.+Fliriauthor=A.+J.+Hightonauthor=H.+Huiauthor=M.+Jiauthor=H.+Jinauthor=K.+Karkiauthor=A.+J.+Keatsauthor=L.+Lazaridesauthor=Y.-J.+Leeauthor=A.+Liclicanauthor=M.+Mishauthor=B.+Murrayauthor=S.+B.+Pettitauthor=P.+Pyunauthor=M.+Sangiauthor=R.+Santosauthor=J.+Sanvoisinauthor=U.+Schmitzauthor=A.+Schrierauthor=D.+Siegelauthor=D.+Sperandioauthor=G.+Stepanauthor=Y.+Tianauthor=G.+M.+Wattauthor=H.+Yangauthor=B.+E.+Schultz&title=Discovery+of+a+potent+and+orally+bioavailable+cyclophilin+inhibitor+derived+from+the+sanglifehrin+macrocycle&doi=10.1021%2Facs.jmedchem.8b00802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span></div><div class="casAuthors">Mackman, Richard L.; Steadman, Victoria A.; Dean, David K.; Jansa, Petr; Poullennec, Karine G.; Appleby, Todd; Austin, Carol; Blakemore, Caroline A.; Cai, Ruby; Cannizzaro, Carina; Chin, Gregory; Chiva, Jean-Yves C.; Dunbar, Neil A.; Fliri, Hans; Highton, Adrian J.; Hui, Hon; Ji, Mingzhe; Jin, Haolun; Karki, Kapil; Keats, Andrew J.; Lazarides, Linos; Lee, Yu-Jen; Liclican, Albert; Mish, Michael; Murray, Bernard; Pettit, Simon B.; Pyun, Peter; Sangi, Michael; Santos, Rex; Sanvoisin, Jonathan; Schmitz, Uli; Schrier, Adam; Siegel, Dustin; Sperandio, David; Stepan, George; Tian, Yang; Watt, Gregory M.; Yang, Hai; Schultz, Brian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9473-9499</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclophilins are a family of peptidyl-prolyl isomerases that are implicated in a wide range of diseases including hepatitis C.  Our aim was to discover through total synthesis an orally bioavailable, non-immunosuppressive cyclophilin (Cyp) inhibitor with potent anti-hepatitis C virus (HCV) activity that could serve as part of an all oral antiviral combination therapy.  An initial lead (I) derived from the sanglifehrin A macrocycle was optimized using structure based design to produce a potent and orally bioavailable inhibitor (II).  The macrocycle ring size was reduced by one atom, and an internal hydrogen bond drove improved permeability and drug-like properties. II demonstrates potent Cyp inhibition (Kd = 5 nM), potent anti-HCV 2a activity (EC50 = 98 nM), and high oral bioavailability in rat (100%) and dog (55%).  The synthetic accessibility and properties of II support its potential as an anti-HCV agent and for interrogating the role of Cyp inhibition in a variety of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEpEZcwXK8z7Vg90H21EOLACvtfcHk0lgW5RH7V2-ilw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI&md5=616cf72e0d2766e159769166c98a9e86</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00802%26sid%3Dliteratum%253Aachs%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DDean%26aufirst%3DD.%2BK.%26aulast%3DJansa%26aufirst%3DP.%26aulast%3DPoullennec%26aufirst%3DK.%2BG.%26aulast%3DAppleby%26aufirst%3DT.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBlakemore%26aufirst%3DC.%2BA.%26aulast%3DCai%26aufirst%3DR.%26aulast%3DCannizzaro%26aufirst%3DC.%26aulast%3DChin%26aufirst%3DG.%26aulast%3DChiva%26aufirst%3DJ.-Y.%2BC.%26aulast%3DDunbar%26aufirst%3DN.%2BA.%26aulast%3DFliri%26aufirst%3DH.%26aulast%3DHighton%26aufirst%3DA.%2BJ.%26aulast%3DHui%26aufirst%3DH.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DKarki%26aufirst%3DK.%26aulast%3DKeats%26aufirst%3DA.%2BJ.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DY.-J.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DMish%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DPettit%26aufirst%3DS.%2BB.%26aulast%3DPyun%26aufirst%3DP.%26aulast%3DSangi%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DSanvoisin%26aufirst%3DJ.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DSchrier%26aufirst%3DA.%26aulast%3DSiegel%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DD.%26aulast%3DStepan%26aufirst%3DG.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DWatt%26aufirst%3DG.%2BM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520orally%2520bioavailable%2520cyclophilin%2520inhibitor%2520derived%2520from%2520the%2520sanglifehrin%2520macrocycle%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9473%26epage%3D9499%26doi%3D10.1021%2Facs.jmedchem.8b00802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmonds, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Improving on nature: making a cyclic heptapeptide orally bioavailable</span>. <i>Angew. Chem. Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12059</span>– <span class="NLM_lpage">12063</span>, <span class="refDoi"> DOI: 10.1002/anie.201405364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1002%2Fanie.201405364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=25219505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A280%3ADC%252BC2M7htlSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=12059-12063&author=D.+S.+Nielsenauthor=H.+N.+Hoangauthor=R.+J.+Lohmanauthor=T.+A.+Hillauthor=A.+J.+Luckeauthor=D.+J.+Craikauthor=D.+J.+Edmondsauthor=D.+A.+Griffithauthor=C.+J.+Rotterauthor=R.+B.+Ruggeriauthor=D.+A.+Priceauthor=S.+Lirasauthor=D.+P.+Fairlie&title=Improving+on+nature%3A+making+a+cyclic+heptapeptide+orally+bioavailable&doi=10.1002%2Fanie.201405364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Improving on nature: making a cyclic heptapeptide orally bioavailable</span></div><div class="casAuthors">Nielsen Daniel S; Hoang Huy N; Lohman Rink-Jan; Hill Timothy A; Lucke Andrew J; Craik David J; Edmonds David J; Griffith David A; Rotter Charles J; Ruggeri Roger B; Price David A; Liras Spiros; Fairlie David P</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">12059-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The use of peptides in medicine is limited by low membrane permeability, metabolic instability, high clearance, and negligible oral bioavailability.  The prediction of oral bioavailability of drugs relies on physicochemical properties that favor passive permeability and oxidative metabolic stability, but these may not be useful for peptides.  Here we investigate effects of heterocyclic constraints, intramolecular hydrogen bonds, and side chains on the oral bioavailability of cyclic heptapeptides.  NMR-derived structures, amide H-D exchange rates, and temperature-dependent chemical shifts showed that the combination of rigidification, stronger hydrogen bonds, and solvent shielding by branched side chains enhances the oral bioavailability of cyclic heptapeptides in rats without the need for N-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT0Jn0fIRQaxLt7lRqEFUJVfW6udTcc2eYir-xB6URBaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7htlSquw%253D%253D&md5=10b02994390bbb4d93e2089e18a13a98</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.201405364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201405364%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DLohman%26aufirst%3DR.%2BJ.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DEdmonds%26aufirst%3DD.%2BJ.%26aulast%3DGriffith%26aufirst%3DD.%2BA.%26aulast%3DRotter%26aufirst%3DC.%2BJ.%26aulast%3DRuggeri%26aufirst%3DR.%2BB.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DImproving%2520on%2520nature%253A%2520making%2520a%2520cyclic%2520heptapeptide%2520orally%2520bioavailable%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%2520Engl.%26date%3D2014%26volume%3D53%26spage%3D12059%26epage%3D12063%26doi%3D10.1002%2Fanie.201405364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marjanović, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Čikoš, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span> <span> </span><span class="NLM_article-title">A new screening tool to determine chameleonic properties of macrocycles</span>. <i>J. Mol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1198</i></span>,  <span class="NLM_fpage">126929</span>, <span class="refDoi"> DOI: 10.1016/j.molstruc.2019.126929</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.molstruc.2019.126929" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1enu7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1198&publication_year=2019&pages=126929&author=N.+Marjanovi%C4%87author=A.+%C4%8Ciko%C5%A1author=S.+Ko%C5%A1trun&title=A+new+screening+tool+to+determine+chameleonic+properties+of+macrocycles&doi=10.1016%2Fj.molstruc.2019.126929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A new screening tool to determine chameleonic properties of macrocycles</span></div><div class="casAuthors">Marjanovic, Nera; Cikos, Ana; Kostrun, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Structure</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1198</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126929</span>CODEN:
                <span class="NLM_cas:coden">JMOSB4</span>;
        ISSN:<span class="NLM_cas:issn">0022-2860</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Membrane permeability is one of the major caveats for the broader use of macrocyclic compds. in the drug discovery.  Some macrocyclic drugs can increase their membrane permeability through conformational changes triggered by different biol. environments, a property referred to as chameleonic ability.  In this study, the combined NMR spectroscopy and mol. modeling workflow was developed as a screening tool to identify macrocycles prone to conformational changes induced by complexation with metal ions.  Three structurally diverse macrocyclic compds. have been investigated to develop and validate the approach; Cyclosporin A as a ref. compd. known to possess chameleonic properties, Rifampicin and Fidaxomicin.  The combination of NMR spectroscopy and mol. modeling proved to be a promising tool for quick assessment of the potential chameleonic behavior of macrocycles through their complexation with metal ions.  Addnl., the results confirmed that ability to form intra-mol., transannular hydrogen bonds is important prerequisite for induced conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMmhirGLLWDLVg90H21EOLACvtfcHk0lg_EA3cEU0j5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1enu7vL&md5=20a4a290f3371bc3ed9ebe3cab972639</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.molstruc.2019.126929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molstruc.2019.126929%26sid%3Dliteratum%253Aachs%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DN.%26aulast%3D%25C4%258Ciko%25C5%25A1%26aufirst%3DA.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26atitle%3DA%2520new%2520screening%2520tool%2520to%2520determine%2520chameleonic%2520properties%2520of%2520macrocycles%26jtitle%3DJ.%2520Mol.%2520Struct.%26date%3D2019%26volume%3D1198%26spage%3D126929%26doi%3D10.1016%2Fj.molstruc.2019.126929" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paljetak, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaskovic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matijasic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fajdetic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbanac, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peric, M.</span></span> <span> </span><span class="NLM_article-title">Macrolide hybrid compounds: drug discovery opportunities in anti- infective and anti- inflammatory Area</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.2174/1568026616666160927160036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.2174%2F1568026616666160927160036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=27697049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFems78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=919-940&author=H.+C.+Paljetakauthor=L.+Tomaskovicauthor=M.+Matijasicauthor=M.+Bukvicauthor=A.+Fajdeticauthor=D.+Verbanacauthor=M.+Peric&title=Macrolide+hybrid+compounds%3A+drug+discovery+opportunities+in+anti-+infective+and+anti-+inflammatory+Area&doi=10.2174%2F1568026616666160927160036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Macrolide Hybrid Compounds: Drug Discovery Opportunities in Anti- Infective and Anti-inflammatory Area</span></div><div class="casAuthors">Paljetak, Hana Cipcic; Tomaskovic, Linda; Matijasic, Mario; Bukvic, Mirjana; Fajdetic, Andrea; Verbanac, Donatella; Peric, Mihaela</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">919-940</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Macrolides, polyketide natural products, and their 15-membered semi-synthetic derivs. are composed of substituted macrocyclic lactone ring and used primarily as potent antibiotics.  Recently their usefulness was extended to antimalarial and anti-inflammatory area.  Hybrid macrolides presented in this article are the next generation semi-synthetic compds. that combine pharmacophores from antibacterial, antimalarial and anti-inflammatory area with 14- and 15-membered azalide scaffolds.  Antibacterial azalide hybrids with sulfonamides showed improved activity against resistant streptococci while quinolone conjugates demonstrated full coverage of respiratory pathogens including macrolide resistant strains and their efficacy was confirmed in mouse pneumonia model.  Antimalarial macrolide hybrids, mainly involving (chloro)quinoline pharmacophores, showed outstanding activity against chloroquine resistant strains, favorable pharmacokinetics, promising in vivo efficacy as well as encouraging developmental potential.  Anti-inflammatory hybrids were obtained by combining macrolides with corticosteroid and non-steroidal anti-inflammatory drugs.  They were found active in in vivo animal models of locally induced inflammation, asthma, inflammatory bowel disease and rheumatoid arthritis and demonstrated improved safety over parent steroid drugs.  Overall, macrolide hybrids possess significant potential to be developed as potent novel medicines in therapeutic areas of utmost pharmaceutical interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwFnd8uMkUd7Vg90H21EOLACvtfcHk0lgWS0A0scrcvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFems78%253D&md5=cc1251fb2a71647da18bb3b37bcdffd8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2174%2F1568026616666160927160036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026616666160927160036%26sid%3Dliteratum%253Aachs%26aulast%3DPaljetak%26aufirst%3DH.%2BC.%26aulast%3DTomaskovic%26aufirst%3DL.%26aulast%3DMatijasic%26aufirst%3DM.%26aulast%3DBukvic%26aufirst%3DM.%26aulast%3DFajdetic%26aufirst%3DA.%26aulast%3DVerbanac%26aufirst%3DD.%26aulast%3DPeric%26aufirst%3DM.%26atitle%3DMacrolide%2520hybrid%2520compounds%253A%2520drug%2520discovery%2520opportunities%2520in%2520anti-%2520infective%2520and%2520anti-%2520inflammatory%2520Area%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D919%26epage%3D940%26doi%3D10.2174%2F1568026616666160927160036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kos, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Škugor, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakopović, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malnar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragojević, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralić, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alihodžić, S.</span></span> <span> </span><span class="NLM_article-title">Around the macrolide - impact of 3D structure of macrocycles on lipophilicity and cellular accumulation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.03.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.ejmech.2017.03.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=28410508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVeksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=351-364&author=S.+Ko%C5%A1trunauthor=V.+M.+Kosauthor=M.+M.+%C5%A0kugorauthor=I.+P.+Jakopovi%C4%87author=I.+Malnarauthor=S.+Dragojevi%C4%87author=J.+Rali%C4%87author=S.+Alihod%C5%BEi%C4%87&title=Around+the+macrolide+-+impact+of+3D+structure+of+macrocycles+on+lipophilicity+and+cellular+accumulation&doi=10.1016%2Fj.ejmech.2017.03.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Around the macrolide - Impact of 3D structure of macrocycles on lipophilicity and cellular accumulation</span></div><div class="casAuthors">Kostrun, Sanja; Munic Kos, Vesna; Matanovic Skugor, Maja; Palej Jakopovic, Ivana; Malnar, Ivica; Dragojevic, Snjezana; Ralic, Jovica; Alihodzic, Sulejman</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">351-364</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The aim of this study was to investigate lipophilicity and cellular accumulation of rationally designed azithromycin and clarithromycin derivs. at the mol. level.  The effect of substitution site and substituent properties on a global physico-chem. profile and cellular accumulation of investigated compds. was studied using calcd. structural parameters as well as exptl. detd. lipophilicity.  In silico models based on the 3D structure of mols. were generated to investigate conformational effect on studied properties and to enable prediction of lipophilicity and cellular accumulation for this class of mols. based on non-empirical parameters.  The applicability of developed models was explored on a validation and test sets and compared with previously developed empirical models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXNqSLGZkuVLVg90H21EOLACvtfcHk0lgWS0A0scrcvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVeksbw%253D&md5=7cf75a8da5ea5c0e7fb9de5e5e685cb5</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.03.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.03.056%26sid%3Dliteratum%253Aachs%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DKos%26aufirst%3DV.%2BM.%26aulast%3D%25C5%25A0kugor%26aufirst%3DM.%2BM.%26aulast%3DJakopovi%25C4%2587%26aufirst%3DI.%2BP.%26aulast%3DMalnar%26aufirst%3DI.%26aulast%3DDragojevi%25C4%2587%26aufirst%3DS.%26aulast%3DRali%25C4%2587%26aufirst%3DJ.%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DS.%26atitle%3DAround%2520the%2520macrolide%2520-%2520impact%2520of%25203D%2520structure%2520of%2520macrocycles%2520on%2520lipophilicity%2520and%2520cellular%2520accumulation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D133%26spage%3D351%26epage%3D364%26doi%3D10.1016%2Fj.ejmech.2017.03.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stepanić, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malnar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hlevnjak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butković, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ćaleta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dukši, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kragol, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makaruha-Stegić, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikac, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralić, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatić, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavčar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zulfikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munić, V.</span></span> <span> </span><span class="NLM_article-title">Modeling cellular pharmacokinetics of 14- and 15-membered macrolides with physicochemical properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1021/jm101317f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101317f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=719-733&author=V.+Stepani%C4%87author=S.+Ko%C5%A1trunauthor=I.+Malnarauthor=M.+Hlevnjakauthor=K.+Butkovi%C4%87author=I.+%C4%86aletaauthor=M.+Duk%C5%A1iauthor=G.+Kragolauthor=O.+Makaruha-Stegi%C4%87author=L.+Mikacauthor=J.+Rali%C4%87author=I.+Tati%C4%87author=B.+Tav%C4%8Darauthor=K.+Valkoauthor=S.+Zulfikariauthor=V.+Muni%C4%87&title=Modeling+cellular+pharmacokinetics+of+14-+and+15-membered+macrolides+with+physicochemical+properties&doi=10.1021%2Fjm101317f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Cellular Pharmacokinetics of 14- and 15-Membered Macrolides with Physicochemical Properties</span></div><div class="casAuthors">Stepanic, Visnja; Kostrun, Sanja; Malnar, Ivica; Hlevnjak, Mario; Butkovic, Kristina; Caleta, Irena; Duksi, Marko; Kragol, Goran; Makaruha-Stegic, Oresta; Mikac, Lara; Ralic, Jovica; Tatic, Iva; Tavcar, Branka; Valko, Klara; Zulfikari, Selvira; Munic, Vesna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">719-733</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrolides with 14- and 15-membered ring are characterized by high and extensive tissue distribution, as well as good cellular accumulation and retention.  Since macrolide structures do not fit the Lipinski rule of five, macrolide pharmacokinetic properties cannot be successfully predicted by common models based on data for small mols.  Here we describe the development of the first models for macrolide cellular pharmacokinetics.  By comparison of cellular accumulation and retention in six human primary cell cultures of leukocytic and lung origin, as well as in lung carcinoma cell line NCI-H292, this cell line was found to be an adequate representative cell type for modeling macrolide cellular pharmacokinetics.  Accumulation and retention in the NCI-H292 cells, as well as various physicochem. properties, were detd. for a set of 48 rationally designed basic macrolide compds.  Classification models for predicting macrolide cellular accumulation and retention were developed using relatively easily detd. and conceptually simple descriptors: exptl. detd. physicochem. parameters ChromlogD and CHI IAM, as well as a calcd. no. of pos. charged atoms (POS).  The models were further tested and improved by addn. of 37 structurally diverse macrolide mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLVpV1pFm_H7Vg90H21EOLACvtfcHk0lh2iWmpI-ZNvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gmsg%253D%253D&md5=9ff804c79b33d707b6e327b7ea21aef9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm101317f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101317f%26sid%3Dliteratum%253Aachs%26aulast%3DStepani%25C4%2587%26aufirst%3DV.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DMalnar%26aufirst%3DI.%26aulast%3DHlevnjak%26aufirst%3DM.%26aulast%3DButkovi%25C4%2587%26aufirst%3DK.%26aulast%3D%25C4%2586aleta%26aufirst%3DI.%26aulast%3DDuk%25C5%25A1i%26aufirst%3DM.%26aulast%3DKragol%26aufirst%3DG.%26aulast%3DMakaruha-Stegi%25C4%2587%26aufirst%3DO.%26aulast%3DMikac%26aufirst%3DL.%26aulast%3DRali%25C4%2587%26aufirst%3DJ.%26aulast%3DTati%25C4%2587%26aufirst%3DI.%26aulast%3DTav%25C4%258Dar%26aufirst%3DB.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DZulfikari%26aufirst%3DS.%26aulast%3DMuni%25C4%2587%26aufirst%3DV.%26atitle%3DModeling%2520cellular%2520pharmacokinetics%2520of%252014-%2520and%252015-membered%2520macrolides%2520with%2520physicochemical%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D719%26epage%3D733%26doi%3D10.1021%2Fjm101317f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">712</span>– <span class="NLM_lpage">717</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=10.1016%2Fj.drudis.2016.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=26891978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=712-717&author=A.+Whittyauthor=M.+Zhongauthor=L.+Viarengoauthor=D.+Beglovauthor=D.+R.+Hallauthor=S.+Vajda&title=Quantifying+the+chameleonic+properties+of+macrocycles+and+other+high-molecular-weight+drugs&doi=10.1016%2Fj.drudis.2016.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span></div><div class="casAuthors">Whitty, Adrian; Zhong, Mengqi; Viarengo, Lauren; Beglov, Dmitri; Hall, David R.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">712-717</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Key to the pharmaceutical utility of certain macrocyclic drugs is a 'chameleonic' ability to change their conformation to expose polar groups in aq. soln., but bury them when traversing lipid membranes.  Based on anal. of the structures of 20 macrocyclic compds. that are approved oral drugs, we propose that good soly. requires a topol. polar surface area (TPSA, in Å2) of ≥0.2 × mol. wt. (MW).  Meanwhile, good passive membrane permeability requires a mol. (i.e., 3D) PSA in nonpolar environments of ≤140 Å2.  We show that one or other of these limits is almost invariably violated for compds. with MW > 600 Da, suggesting that some degree of chameleonic behavior is required for most high MW oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlN7rCqts5wLVg90H21EOLACvtfcHk0lh2iWmpI-ZNvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D&md5=024476298f7b72efa3f0a9303bed5900</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DViarengo%26aufirst%3DL.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DQuantifying%2520the%2520chameleonic%2520properties%2520of%2520macrocycles%2520and%2520other%2520high-molecular-weight%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D712%26epage%3D717%26doi%3D10.1016%2Fj.drudis.2016.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Faham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albericio, F.</span></span> <span> </span><span class="NLM_article-title">Peptide coupling reagents, More than a letter soup</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">6557</span>– <span class="NLM_lpage">6602</span>, <span class="refDoi"> DOI: 10.1021/cr100048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr100048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVyhu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2011&pages=6557-6602&author=A.+El-Fahamauthor=F.+Albericio&title=Peptide+coupling+reagents%2C+More+than+a+letter+soup&doi=10.1021%2Fcr100048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Coupling Reagents, More than a Letter Soup</span></div><div class="casAuthors">El-Faham, Ayman; Albericio, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6557-6602</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The prepn. and uses of carbodiimides, anhydrides (sym. and mixed), active esters, acylazoles, acyl azides, acid halides, acid fluorides, phosphonium salts, aminium/uronium salts, organophosphorus compds., organosulfur compds., triazine reagents, pyridinium reagents, polymer-supported reagents as coupling reagents in peptide synthesis are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9DDv3PT7yrLVg90H21EOLACvtfcHk0lgSQgL8w47Ypg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVyhu7zO&md5=461d8c78dffcbf16a390fc78abd33cac</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fcr100048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr100048w%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Faham%26aufirst%3DA.%26aulast%3DAlbericio%26aufirst%3DF.%26atitle%3DPeptide%2520coupling%2520reagents%252C%2520More%2520than%2520a%2520letter%2520soup%26jtitle%3DChem.%2520Rev.%26date%3D2011%26volume%3D111%26spage%3D6557%26epage%3D6602%26doi%3D10.1021%2Fcr100048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djokić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobrehel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopotar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamenar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrvos, D.</span></span> <span> </span><span class="NLM_article-title">Erythromycin series. Part 13. Synthesis and structure elucidation of 10-dihydro-10-deoxo-11-methyl-11-azaerythromycin A</span>. <i>J. Chem. Res. Synopses.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">153</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1988&pages=152-153&author=S.+Djoki%C4%87author=G.+Kobrehelauthor=N.+Lopotarauthor=B.+Kamenarauthor=A.+Naglauthor=D.+Mrvos&title=Erythromycin+series.+Part+13.+Synthesis+and+structure+elucidation+of+10-dihydro-10-deoxo-11-methyl-11-azaerythromycin+A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDjoki%25C4%2587%26aufirst%3DS.%26aulast%3DKobrehel%26aufirst%3DG.%26aulast%3DLopotar%26aufirst%3DN.%26aulast%3DKamenar%26aufirst%3DB.%26aulast%3DNagl%26aufirst%3DA.%26aulast%3DMrvos%26aufirst%3DD.%26atitle%3DErythromycin%2520series.%2520Part%252013.%2520Synthesis%2520and%2520structure%2520elucidation%2520of%252010-dihydro-10-deoxo-11-methyl-11-azaerythromycin%2520A%26jtitle%3DJ.%2520Chem.%2520Res.%2520Synopses.%26date%3D1988%26volume%3D5%26spage%3D152%26epage%3D153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VB9','PDB','5VB9'); return false;">PDB: 5VB9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HSA','PDB','4HSA'); return false;">PDB: 4HSA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HI4','PDB','5HI4'); return false;">PDB: 5HI4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HHX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HHX','PDB','5HHX'); return false;">PDB: 5HHX</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i117"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00327">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_44860"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00327?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00327</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Methods description (chemical synthesis, molecular modeling, and <i>in vitro</i> and <i>in vivo</i> profiling); detailed experimental data from NMR studies; supplementary chemical structures and accompanied bioactivities; and synthetic procedures and characterization of compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf">PDF</a>)</p></li><li><p class="inline">PDB files of cpd4 docked into 4HSA and 5HI4 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_002.pdb">PDB</a>, <a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_001.pdf">jm1c00327_si_001.pdf (2.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_002.pdb">jm1c00327_si_002.pdb (383.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_003.pdb">jm1c00327_si_003.pdb (347.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00327/suppl_file/jm1c00327_si_004.csv">jm1c00327_si_004.csv (15.36 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00327&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00327%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00327" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991338e9ee3cc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
