
<html lang="en"     class="pb-page"  data-request-id="e130e405-fcf9-49a7-88c9-d695848f197b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01043;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-20;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rapid Solid-Phase Construction of Serine Hydrolase Probes Results in Selective Activity-Based Probes for Acyl Protein Thioesterases-1/2" /></meta><meta name="dc.Creator" content="Roeland  Vanhoutte" /></meta><meta name="dc.Creator" content="Merel A. T.  van de Plassche" /></meta><meta name="dc.Creator" content="Steven H. L.  Verhelst" /></meta><meta name="dc.Description" content="Serine hydrolases (SHs) are a large, diverse family of enzymes that play various biomedically important roles. Their study has been substantially advanced by activity-based protein profiling, which..." /></meta><meta name="Description" content="Serine hydrolases (SHs) are a large, diverse family of enzymes that play various biomedically important roles. Their study has been substantially advanced by activity-based protein profiling, which..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 29, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01043" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01043" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01043" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01043" /></link>
        
    
    

<title>Rapid Solid-Phase Construction of Serine Hydrolase Probes Results in Selective Activity-Based Probes for Acyl Protein Thioesterases-1/2 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01043" /></meta><meta property="og:title" content="Rapid Solid-Phase Construction of Serine Hydrolase Probes Results in Selective Activity-Based Probes for Acyl Protein Thioesterases-1/2" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0006.jpeg" /></meta><meta property="og:description" content="Serine hydrolases (SHs) are a large, diverse family of enzymes that play various biomedically important roles. Their study has been substantially advanced by activity-based protein profiling, which makes use of covalent chemical probes for labeling the active site and detection by various methodologies. However, highly selective probes for individual SHs are scarce because probe synthesis usually takes place by time-consuming solution phase chemistry. We here report a general solid-phase synthesis toward SH chemical probes, which will speed up probe library synthesis. It involves the construction of a recognition element ending in a secondary amine followed by capping with different electrophiles. We illustrate the power of this approach by the discovery of selective chemical probes for the depalmitoylating enzymes APT-1/2. Overall, this study reports new methodologies to synthesize SH probes, while providing new reagents to study protein depalmitoylation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01043"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01043">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01043&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01043&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01043&amp;href=/doi/10.1021/acs.jmedchem.0c01043" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 11845-11853</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01033" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01076" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rapid Solid-Phase Construction of Serine Hydrolase Probes Results in Selective Activity-Based Probes for Acyl Protein Thioesterases-1/2</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Roeland Vanhoutte</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roeland Vanhoutte</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cellular and Molecular Medicine, Laboratory of Chemical Biology, KU Leuven, Herestr. 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roeland++Vanhoutte">Roeland Vanhoutte</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Merel A. T. van de Plassche</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Merel A. T. van de Plassche</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cellular and Molecular Medicine, Laboratory of Chemical Biology, KU Leuven, Herestr. 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Merel+A.+T.++van+de+Plassche">Merel A. T. van de Plassche</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Steven H. L. Verhelst</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven H. L. Verhelst</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cellular and Molecular Medicine, Laboratory of Chemical Biology, KU Leuven, Herestr. 49, 3000 Leuven, Belgium</div><div class="loa-info-affiliations-info">Leibniz Institute for Analytical Sciences ISAS, e.V., Otto-Hahn-Str. 6b, 44227 Dortmund, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ee9d9a8b988b80c0988b9c868b829d9aae879d8f9dc08a8b"><span class="__cf_email__" data-cfemail="8efdfaebf8ebe0a0f8ebfce6ebe2fdfacee7fdeffda0eaeb">[email protected]</span></a>, <a href="/cdn-cgi/l/email-protection#6516110013000b4b1300170d00091611250e1009001013000b4b0700"><span class="__cf_email__" data-cfemail="7102051407141f5f07140319141d0205311a041d140407141f5f1314">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven+H.+L.++Verhelst">Steven H. L. Verhelst</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7400-1319" title="Orcid link">http://orcid.org/0000-0002-7400-1319</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01043&amp;href=/doi/10.1021%2Facs.jmedchem.0c01043" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 11845–11853</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 29, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 June 2020</li><li><span class="item_label"><b>Published</b> online</span>29 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01043" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01043</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11845%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRoeland%2BVanhoutte%252C%2BMerel%2BA.%2BT.%2Bvan%2Bde%2BPlassche%252C%2BSteven%2BH.%2BL.%2BVerhelst%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D20%26contentID%3Dacs.jmedchem.0c01043%26title%3DRapid%2BSolid-Phase%2BConstruction%2Bof%2BSerine%2BHydrolase%2BProbes%2BResults%2Bin%2BSelective%2BActivity-Based%2BProbes%2Bfor%2BAcyl%2BProtein%2BThioesterases-1%252F2%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11853%26publicationDate%3DOctober%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01043"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">619</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01043" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rapid Solid-Phase Construction of Serine Hydrolase Probes Results in Selective Activity-Based Probes for Acyl Protein Thioesterases-1/2&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Roeland&quot;,&quot;last_name&quot;:&quot;Vanhoutte&quot;},{&quot;first_name&quot;:&quot;Merel&quot;,&quot;last_name&quot;:&quot;A. T. van de Plassche&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;H. L. Verhelst&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;29&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;11845-11853&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01043&quot;},&quot;abstract&quot;:&quot;Serine hydrolases (SHs) are a large, diverse family of enzymes that play various biomedically important roles. Their study has been substantially advanced by activity-based protein profiling, which makes use of covalent chemical probes for labeling the active site and detection by various methodologies. However, highly selective probes for individual SHs are scarce because probe synthesis usually takes place by time-consuming solution phase chemistry. We here report a general solid-phase synthesis toward SH chemical probes, which will speed up probe library synthesis. It involves the construction of a recognition element ending in a secondary amine followed by capping with different electrophiles. We illustrate the power of this approach by the discovery of selective chemical probes for the depalmitoylating enzymes APT-1/2. Overall, this study reports new methodologies to synthesize SH probes, while providing new reagents to study protein depalmitoylation.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01043&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01043" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01043&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01043" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01043&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01043" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01043&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01043&amp;href=/doi/10.1021/acs.jmedchem.0c01043" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01043" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01043" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01043%26sid%3Dliteratum%253Aachs%26pmid%3D32990443%26genre%3Darticle%26aulast%3DVanhoutte%26date%3D2020%26atitle%3DRapid%2BSolid-Phase%2BConstruction%2Bof%2BSerine%2BHydrolase%2BProbes%2BResults%2Bin%2BSelective%2BActivity-Based%2BProbes%2Bfor%2BAcyl%2BProtein%2BThioesterases-1%252F2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D20%26spage%3D11845%26epage%3D11853%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290692" title="Elements">Elements</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291503" title="Organic polymers">Organic polymers</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/jmcmar.2020.63.issue-20/20201022/jmcmar.2020.63.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Serine hydrolases (SHs) are a large, diverse family of enzymes that play various biomedically important roles. Their study has been substantially advanced by activity-based protein profiling, which makes use of covalent chemical probes for labeling the active site and detection by various methodologies. However, highly selective probes for individual SHs are scarce because probe synthesis usually takes place by time-consuming solution phase chemistry. We here report a general solid-phase synthesis toward SH chemical probes, which will speed up probe library synthesis. It involves the construction of a recognition element ending in a secondary amine followed by capping with different electrophiles. We illustrate the power of this approach by the discovery of selective chemical probes for the depalmitoylating enzymes APT-1/2. Overall, this study reports new methodologies to synthesize SH probes, while providing new reagents to study protein depalmitoylation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40633" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40633" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Activity-based protein profiling (ABPP) emerged two decades ago to help discover the function of enzymes.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> ABPP uses small molecules called activity-based probes (ABPs), which bind covalently to the active site of the enzymes of interest, allowing a multitude of read-out possibilities and application areas,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> including profiling in different disease states, imaging, target identification, as well as inhibitor screening by competitive ABPP (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). ABPs generally consist of three parts: a warhead, which binds covalently to a catalytic residue of the target enzyme; a recognition element, which influences the selectivity of a probe; and a reporter tag to visualize probe labeled proteins.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. ABPP and ABPs. (A) Left (boxed): ABPs generally consist of three elements: a reactive electrophile as warhead that engages in a mechanism-based reaction with the active site nucleophile of the target, a tag that allows detection and a spacer and/or recognition element to prevent a steric clash of the tag in the active site and to influence the selectivity of the probe. Right: ABPP allows the tagging of active enzyme targets in a complex proteome, such as a cell lysate, whole cell or in vivo setting, followed by various types of read-out. Inhibitor evaluation in complex proteomes may be done by “competitive ABPP”, in which the proteome is first treated with a potential inhibitor before incubation with the ABP. The disappearance of gel bands in sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) will then reveal inhibitor selectivity. (B) Structure of FP-based probes and the mechanism of reaction with SHs: the active site serine side chain, activated by hydrogen bonding to a histidine residue, engages in a mechanism-based reaction and results in a stable, covalent phosphonate ester probe–protein complex. (C) Modular design of SH ABPs for construction on a solid support.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Fluorophosphonate (FP) probes (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) were developed as the first serine hydrolase (SH) reactive ABPs—either equipped with a biotin tag (FP-biotin)<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> or a fluorophore (FP-rhodamine; FP-Rh).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> SHs are an enzyme superfamily comprising 1–2% of the human genome. Among them are several pharmaceutically interesting proteins,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> such as monoacyl glycerol lipase (MAGL) and α,β-hydrolase domain containing protein 6 (ABDH6), which play a role in the generation of endocannabinoids in the nervous system;<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> acyl protein thioesterases-1/2 (APT-1/2; previously known as lysophospholipases-1/2; LYPLA-1/2), which are involved in depalmitoylation of Ras GTPase;<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and protein phosphatase methyl esterase 1, which plays a role in neuroblastoma.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> For most SH members, however, the exact function remains elusive. Conveniently, FP-Rh reacts with more than 80% of all metabolic SHs in mice.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Hence, FP-Rh has not only been useful for the study of well-known SHs but also for uncharacterized ones. However, there are also some limitations of this reagent: (1) the synthesis is not trivial and takes considerable effort. (2) It may not react with all SHs. (3) Because of its general nature, it is problematic to implement this probe in studies for the visualization of a single SH of interest. Therefore, serine-reactive ABPs with other warheads and recognition elements have been developed over the past two decades. These have the potential to be used for in vivo imaging studies, as recently illustrated for MAGL.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Nevertheless, it is extremely challenging and time-consuming to obtain highly selective ABPs because the synthesis and optimization of these ABPs usually takes place by solution phase chemistry. Hence, selective SH ABPs have only been reported for a few SHs, including neutral cholesterol ester hydrolase 1 (KIAA1363),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> diacylglycerol lipase (DAGL),<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> and MAGL,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> although probes for the last two showed some cross-reactivity toward ABDH6. In order to produce and optimize ABPs in a fast and efficient manner, we set out to speed up the process of SH probe development by applying solid-phase synthesis. Here, we present a mix-and-match strategy of recognition elements and serine reactive warheads for the design and full on resin synthesis of a SH ABP library. Triazole urea probes in this library were potent and selective against APT-1/2, and they represent new tools for the study of protein depalmitoylation.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design & Synthesis of Probes</h3><div class="NLM_p">We aimed at making ABPs for SHs in a time-efficient manner by a full on resin synthesis. Therefore, our probes comprised the following modular design (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C): (1) a C-terminal amide which forms the anchor to the common Rink amide resin, (2) an azide tag amenable to click chemistry-mediated detection of probe targets and incorporated in (3) an azidolysine-aminohexanoic acid spacer that connects to (4) a recognition element consisting of a natural amino acid, an unnatural amino acid or a peptoid (or a combination thereof), and (5) a serine reactive electrophile. In this design, the reactive group is introduced on the resin as an N-terminal cap in the very last step prior to resin cleavage. As warheads, we selected five types of reactive electrophiles (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A) which have previously been shown to react with SHs and that can potentially be installed at an amine in a last step on solid support: a diphenylphosphoramidate,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> an <i>N</i>-hydroxysuccinimidyl carbamate (NHS carbamate),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> a (nitro-)phenylcarbamate, a 2-aminopyridineurea,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and a 1,2,3-triazoleurea.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The introduction at this point of the synthesis would minimize the incompatibility with any used reagents and conditions except for the coupling conditions of the reactive group itself and the cleavage conditions from the resin. Whereas diphenylphosphoramidates have already been synthesized on resin,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> this possibility has not yet been explored for the other reactive electrophiles. In a pilot experiment (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A), we explored coupling these reactive groups to recognition elements ending in a primary amine (a phenylalanine), a secondary amine (a phenylalanine peptoid), or a secondary amine in a six-membered ring [an isonipecotic acid (Inp)]. We found that primary amines capped as <i>N</i>-hydroxysuccinimidylcarbamates converted into hydantoins (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S1</a>) and could not form triazole ureas onto primary or noncyclic secondary amines. However, all reactive groups could be placed onto the piperidine ring of Inp as detected by LC–MS of crude compound mixtures after resin cleavage (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S2</a>). This inspired us to make a series of recognition elements, consisting of a piperidine or piperazine moiety connected to an additional aromatic group, a combination which has previously shown to have affinity for various SHs.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Mix-and-match SH ABP Libraries<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(A) On-resin coupling of warheads to different types of amines. Reagents and conditions: (i) diphenylphosphoryl chloride, DIEA, and DCM. (ii) Disuccinimidylcarbonate, DIEA, and DCM. (iii) 2-Aminopyridinyl phenyl carbamate, DIEA, and DCM. (iv) Phenyl chloroformate, DIEA, and DCM. (v) (1). Triphosgene, DIEA, and DCM; (2) 1<i>H</i>-1,2,3-triazole, DIEA, and DCM. (B) Library synthesis toward compounds <b>4a–6f</b>. (vi) SPPS elongation by (1). DMF/piperidine 4/1; (2) Fmoc-aa-OH, HBTU, DIEA, and DMF. (vii) (1) DMF/piperidine 4/1; (2) bromoacetic acid, DIC, HOBt, and DMF. (viii) Benzylamine, DIEA, and DMF. (ix) Fmoc-Inp-OH, DIC, HOBt, and DMF. (x) <i>p</i>-Bromomethylbenzoic acid, PyBrOP, DIEA, and DMF. (xi) Piperazine and DMF. (xii) 4-nitrophenyl chloroformate, DIEA, and DCM.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Library Synthesis</h3><div class="NLM_p last">The three recognition elements in the probe library (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B) comprised Inp coupled to phenylalanine (<b>1</b>), Inp coupled to a phenylalanine peptoid (<b>2</b>), or <i>para</i>-piperazinylmethyl benzoic acid (<b>3</b>). These recognition elements were synthesized by modified SPPS and installed onto a Lys(N<sub>3</sub>)–Ahx click-tag/linker moiety. Specifically, elongation with Phe took place under standard conditions, whereas the peptoid Phe was synthesized by coupling a bromoacetic acid followed by on-resin nucleophilic substitution of the bromide by benzylamine. These two recognition elements were completed by coupling of Inp under the influence of DIC/HOBt. The third recognition element was formed by the installation of a 4-bromomethylbenzoyl substituent followed by S<sub>N</sub>2 substitution with piperazine. All three recognition elements were then capped with six warheads. Gratifyingly, these compounds were all compatible with Rink amide resin cleavage conditions, giving a 3 × 6 library of ABPs <b>4a–6f</b>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Activity Against SHs in Competitive ABPP</h3><div class="NLM_p">Next, we set out to test whether these compounds display reactivity against SHs. A mouse brain lysate was chosen as a suitable proteome for screening because it contains a wide variety of pharmacologically interesting SHs. Previous studies have identified between approximately 25 and 40 different SHs in this lysate and, conveniently, the retention of several of these on SDS-PAGE is known.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> Hence, the cytosolic and membrane fractions of mouse brain lysates were pretreated with compounds <b>4a–6f</b> and residually active SHs were labeled with FP-Rh (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Interestingly, this competitive ABPP experiment revealed that five library members (the three triazole ureas <b>4f</b>, <b>5f</b> and <b>6f</b>, and two NHS-carbamates <b>5b</b> and <b>6b</b>) showed selective inhibition of two close-running gel bands that have previously been identified as APT-1/2 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> A titration of these compounds showed that the potency of the NHS carbamates <b>5b</b> and <b>6b</b> for these targets was moderate (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S3</a>), whereas the triazole ureas were much more potent (low to mid nanomolar range) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, see also <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S4</a> for representative gels).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Screening of SH ABP library for inhibition by competitive ABPP. (A) Fluorescent gels of competitive ABPP in the membrane fraction (upper) or soluble fraction (lower) of mouse brain lysates. Pretreatment of the lysate with the indicated probe was followed by measurement of residual SH activity with FP-Rh. (B) Apparent IC<sub>50</sub> curves were determined by competitive ABPP of serial dilutions, followed by quantification using gel band densitometry. (C) Competitive ABPP in the mouse brain lysate (soluble fraction) with known APT1/2 inhibitor JCP-174-alk (left panel) or APT1-selective inhibitor ML348 and APT2-selective inhibitor ML349 (right panel) in comparison with compound <b>5f</b>. Residual SH activity in treated tissue lysates was measured by incubation with FP-Rh. (D) Inhibition of APT1/2 by compounds <b>5f</b> and <b>6f</b> in live A431 cells. After treatment, compounds were washed out, and residual SH activity in cell lysates was detected with FP-Rh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Probe <b>4f</b>, with a Phe–Inp recognition element, had an apparent IC<sub>50</sub> of 460 ± 50 nM for APT-1/2. For the other targets of FP-Rh, this probe had apparent IC<sub>50</sub> values above 30 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S4</a>), resulting in a larger than 65-fold selectivity. Remarkably, the potency for APT-1/2 increased 37-fold by replacing the Phe in the recognition element by its peptoid analogue in <b>5f</b> (IC<sub class="stack">50</sub><sup class="stack">App</sup> = 11.8 ± 1.8 nM) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), while having excellent selectivity (over 2500-fold, as measured by apparent IC<sub>50</sub> values against the other FP-Rh targets, which were above 30 μM, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S4</a>). Probe <b>6f</b>, with a linear benzyl-piperazinyl recognition element also showed high activity against APT-1/2 (IC<sub class="stack">50</sub><sup class="stack">App</sup> = 27.9 ± 2.8 nM), although at micromolar concentrations this probe also inhibited ABHD6 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S4</a>) with an apparent IC<sub>50</sub> of 7.4 ± 1.7 μM (approximately 250-fold selectivity).</div><div class="NLM_p last">We next compared probe <b>5f</b> with the chloro-isocoumarin-based probe JCP-174-alk<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and the isoform selective APT-1 and APT-2 inhibitors ML-348 and ML349.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> To this end, competitive ABPP was performed in the soluble fraction of the mouse brain lysate. We found that the gel bands that competed with probe <b>5f</b> also disappear upon treatment with JCP-174-alk (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C, left panel). Moreover, inhibitors ML-348 and ML-349 only compete one of the two close-running gel bands inhibited by probe <b>5f</b>. Overall, this reveals that <b>5f</b> has similar inhibitory capacity and confirms APT-1/2 as a target. We assessed the cell permeability of probes <b>5f</b> and <b>6f</b> using the human squamous carcinoma cell line A431. In a competitive ABPP assay with FP-Rh, both <b>5f</b> and <b>6f</b> fully inhibited all active APT-1/2 at 1 μM concentration (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D), confirming their target engagement in live cells.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Direct Profiling of Probe Targets</h3><div class="NLM_p">To show the utility of the synthesized molecules as novel, highly selective ABPs against SHs, probes were applied in a labeling experiment using copper-catalyzed azide alkyne cycloaddition (CuAAC) with an alkyne-functionalized fluorophore. At 1 μM concentration, all three probes showed strong labeling of the APT-1/2 band, and this labeling was even retained at 100 nM concentration for probes <b>5f</b> and <b>6f</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). Interestingly, a few other targets besides the APT-1/2 gel bands were revealed in this direct labeling experiment (indicated with a red asterisk, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B), which were not detected in the competitive ABPP experiment that made use of FP-Rh. This illustrates that not all probe targets overlap with general SH probe FP-Rh. Specifically, probe <b>6f</b> showed strong labeling of a soluble protein at 30 kDa (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and membrane proteins at 35 and 55 kDa (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Probes <b>4f</b> and <b>5f</b> both showed reactivity toward a different membrane protein at 33 kDa at 1 μM. However, SDS-PAGE analysis of <b>5f</b> at 100 nM concentration did not show any other labeling than APT-1/2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, right panel).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Covalent interaction with APT-1/2. (A) Labeling of a soluble fraction of the mouse brain lysate with probes <b>4f–6f</b>, followed by click chemistry with a TAMRA-alkyne derivative shows selective detection of the ATP-1/2 gel bands. DMSO control (−) shows a click background. Other protein targets than APT-1/2 are indicated by a *. (B) Same as (A) but in the membrane fraction of the mouse brain lysate. (C) After click chemistry with a trifunctional biotin-TAMRA-alkyne tag and enrichment on streptavidin beads, fluorescent detection (upper panels) confirms target engagement of the probes, while detection by anti-APT-1 or anti-APT-2 antibodies (lower panels) confirms target identity. Note that the targets in the membrane fraction had very low abundance and led to very weak bands. Full fluorescent gels shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S5</a>. (D) Covalent docking of <b>5f</b> in a crystal structure of APT-1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QGS">6QGS</a>). Protein in a cartoon format with α-helices in cyan, β-sheets in magenta, and random coil in pink. A co-crystallized palmitate is shown in orange sticks, whereas <b>5f</b> is shown in magenta. Active site residues are depicted in green, whereas interacting residues are shown in white. Picture rendered with PyMol.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (E) Covalent docking of <b>6f</b> (in yellow) overlayed with co-crystallized palmitate (in orange) in a crystal structure of APT-1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Previously reported proteomics-based assessments of the targets of FP-Rh in brain lysates show that it labels APT-1/2 as targets at 25 kD.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Given our competitive-based ABPP results with FP-Rh, we suspected these as being the targets of our NHS-carbamate and triazole ureaprobes. To confirm that these close-running gel bands correspond to APT-1/2, we incubated mouse brain lysates with <b>5f</b> or <b>6f</b>, followed by click labeling of the probe-enzyme complexes with a trifunctional TAMRA-biotin-alkyne detection tag. This tag allowed affinity-based enrichment of the probe hits on streptavidin-coated beads, followed by both fluorescent visualization (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, upper panels, full gels in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S5</a>) as well as Western blot analysis (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C; lower panels). The detected bands with anti-APT-1 and anti-APT-2 antibodies in the samples treated with probes, but not in DMSO controls, unequivocally reveal the identity of the probe targets.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Model of Probe Binding</h3><div class="NLM_p last">To gain insights into the binding mode of the two most active compounds <b>5f</b> and <b>6f</b> toward their targets, we performed covalent docking using the flexible sidechain methods of AutoDock 4.2.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> In a crystal structure of APT-1 with a bound palmitate molecule (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QGS">6QGS</a>), both probes overlay well with this substrate (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D,E) and occupy a long, hydrophobic channel, which is also the binding site of the recently reported, reversible APT-1 inhibitor ML348 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S6</a>).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The Inp residue of the most potent probe <b>5f</b> seems locked between hydrophobic residues F73, I75, W145, and L176, whereas its peptoid Phe moiety binds into a pocket created by the residues F73, I75, I76, and the alkyl chain of E84 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Although the slightly less active probe <b>6f</b> does not have a peptoid Phe, it may compensate for this by a stacking interaction of the benzoyl group in its backbone with W145, whereas its piperidine ring has similar hydrophobic interactions as the Inp in probe <b>5f</b> with L30, I75, W145, and L176 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37724" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37724" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In conclusion, we report synthetic procedures for a full on resin synthesis of SH chemical probes compatible with six different types of serine-reactive electrophiles. Using this procedure, we constructed a library of ABPs by combining the electrophiles with three different recognition elements consisting of an aromatic unit and a piperidine or piperazine moiety. Evaluation of these compounds in a serine-hydrolase rich mouse brain lysate revealed activity of five of these compounds against protein depalmitoylases APT-1 and APT-2, with triazole urea <b>5f</b> showing nanomolar potency and excellent selectivity. The structure–activity relationship from this work may provide new information on pharmacophores for the design of drug-like APT-1/2 inhibitors and the in silico docking presented here provides a plausible model of the binding mode of these compounds. Compared with other recently reported inhibitors and probes for APT1/2,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> the compounds presented here can be readily synthesized on a solid support and allow for easy future optimization by medicinal chemistry studies.</div><div class="NLM_p last">We expect that the novel selective ABPs <b>5f</b> and <b>6f</b> resulting from this work will find application in the study of APT1/2-mediated protein depalmitoylation. Furthermore, the synthetic procedures reported in this work may be automated in future work and facilitate more rapid development of SH inhibitors and ABPs by library synthesis on a solid support. Because many SHs are still poorly characterized, we expect that this will open the door to functional studies of these.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76176" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76176" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General Materials and Methods</h3><div class="NLM_p last">All starting materials for chemical reactions were purchased from commercial suppliers and used without purification. All solvents were of synthesis grade or higher. Progress of solution phase reactions was monitored by TLC on pre-coated 0.20 mm thick ALUGRAM TLC sheets (silica gel-60 with UV indicator) using UV light and/or staining with cerium ammonium molybdate or ninhydrin followed by heating. Nuclear magnetic resonance spectra were recorded on a Bruker Ultrashield 400 or 600 MHz NMR spectrometer. Chemical shifts are reported in ppm relative to the residual solvent peak. LC–MS spectra were recorded on a Prominence ultra-fast liquid chromatography system (Shimadzu) using a Waters X-bridge 2.1 mm C18 column with a gradient of 5–80% acetonitrile in water (with 0.1% formic acid) over 22 min. HPLC purification was performed on a Prominence ultra-fast liquid chromatography system (Shimadzu) using a Waters X-bridge 150 mm C18 prep column with a gradient of acetonitrile in water (with 0.1% trifluoroacetic acid) over 32 min. Purity >95% of all final compounds was assessed by the integration of LC chromatograms.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Solid-Phase Chemistry Procedures</h3><div class="NLM_p last">The Fmoc-protected Rink amide resin (1.35 g, 0.74 mmol/g) was treated with 20% piperidine in DMF for 5 min to swell the resin and deprotect the Fmoc group. Next, the resin was washed three times with DMF. All following Fmoc deprotections were performed according to this procedure.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Backbone Synthesis</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Method 1</h4><div class="NLM_p last">Fmoc-<span class="smallcaps smallerCapital">l</span>-Lys(N<sub>3</sub>)–OH was coupled to 1 mmol of the deprotected rink amide resin by adding amino acid (1.5 equiv), HBTU (1.5 equiv) and DIPEA (3 equiv) in 4 mL of DMF to the resin and incubating for 4 h at room temperature. After this and each following coupling, the resin was washed three times with DMF and three times with DCM. After deprotection, Fmoc-6-aminohexanoic acid was coupled by adding amino acid (2 equiv), HBTU (2 equiv), and DIEA (4 equiv) in 4 mL of DMF to the resin and incubating overnight at room temperature.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Recognition Element Synthesis</h3><div class="NLM_p">After synthesis of the peptide backbone, the resin was divided into portions of 100 μmol on which the different recognition elements were formed. Peptide bonds in recognition elements were formed with the corresponding (amino)acid (3 equiv), DIC (3 equiv), and HOBt (3 equiv) in 1.5 mL of DMF overnight at room temperature unless stated otherwise.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Method 2.1</h4><div class="NLM_p last">Phe-peptoid was formed by coupling bromoacetic acid to the peptide backbone under standard SPPS conditions and subsequent substitution with benzylamine (20 equiv) and DIEA (10 equiv) in 1.5 mL of DMF per 100 μmol overnight at room temperature. Next, Fmoc-Inp-OH was coupled using standard SPPS conditions.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Method 2.2</h4><div class="NLM_p last">4-Bromomethyl phenylacetic acid (3 equiv) was coupled to the peptide backbone with PyBrOP (3 equiv) and DIEA (6 equiv) in 1.5 mL of DMF per 100 μmol overnight at room temperature. Subsequently, the bromide was substituted by piperazine (20 equiv) using DIEA (10 equiv) in 1.5 mL of DMF overnight at room temperature.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Method 2.3</h4><div class="NLM_p last">Fmoc-<span class="smallcaps smallerCapital">l</span>-Phe-OH was coupled to the peptide backbone using standard SPPS conditions, followed by coupling of Fmoc-Inp-OH under standard SPPS conditions.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Warhead Coupling</h3><div class="NLM_p">After synthesis of the recognition elements, the resins were divided into portions of 10 μmol for coupling of the warheads.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Method 3.1</h4><div id="sec4_5_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Synthesis of Diphenyl Phosphoramidates</h5><div class="NLM_p last">The resin (1 equiv) was swollen in 0.5 mL of DMF per 10 μmol of the resin, to which DIEA (5 equiv) and diphenylphosphoryl chloride (5 equiv) were added. Reactions were shaken overnight at room temperature and checked for completion using LC–MS. After completion, the resin was washed three times with DMF and twice with DCM. Probes were then cleaved from the resin by using a 95:2.5:2.5 mixture of TFA/TIS/water.</div></div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Method 3.2</h4><div id="sec4_5_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Synthesis of Hydroxysuccinimidylcarbamates</h5><div class="NLM_p last">The resin (1 equiv) was swollen in 0.5 mL of DMF per 10 μmol of the resin, to which DIEA (5 equiv) and disuccinimidylcarbonate (3 equiv) were added. Reactions were shaken overnight at room temperature and checked for completion using LC–MS. After completion, the resin was washed three times with DMF and twice with DCM. Probes were then cleaved from the resin by using a 95:2.5:2.5 mixture of TFA/TIS/water.</div></div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Method 3.3</h4><div id="sec4_5_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Synthesis of Aminopyridine Ureas</h5><div class="NLM_p last">The resin (1 equiv) was swollen in 0.5 mL of DMF per 10 μmol of the resin, to which were added DIEA (5 equiv) and phenyl 2-pyridinylcarbamate (3 equiv; synthesized from 2-aminopyridine and phenyl chloroformate according to the method described by Swanson et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and isolated in 29% yield). Reactions were shaken overnight at room temperature and checked for completion using LC–MS. After completion, the resin was washed three times with DMF and twice with DCM. Probes were then cleaved from the resin by using a 95:2.5:2.5 mixture of TFA/TIS/water.</div></div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Method 3.4</h4><div id="sec4_5_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Synthesis of Phenylcarbamates</h5><div class="NLM_p last">The resin (1 equiv) was swollen in 0.5 mL of DMF per 10 μmol of the resin, to which DIEA (10 equiv) and phenyl chloroformate (10 equiv) were added. Reactions were shaken overnight at room temperature and checked for completion using LC–MS. After completion, the resin was washed three times with DMF and twice with DCM. Probes were then cleaved from the resin by using a 95:2.5:2.5 mixture of TFA/TIS/water.</div></div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Method 3.5</h4><div id="sec4_5_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Synthesis of <i>p</i>-Nitrophenylcarbamates</h5><div class="NLM_p last">The resin (1 equiv) was swollen in 0.5 mL of DMF per 10 μmol of resin, to which DIEA (10 equiv) and <i>p</i>-nitrophenyl chloroformate (10 equiv) were added. Reactions were shaken overnight at room temperature and checked for completion using LC–MS. After completion, the resin was washed three times with DMF and twice with DCM. Probes were then cleaved from the resin by using a 95:2.5:2.5 mixture of TFA/TIS/water.</div></div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Method 3.6</h4><div id="sec4_5_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Synthesis of Triazole Ureas</h5><div class="NLM_p last">The resin (1 equiv) was swollen in 1 mL of dry DCM per 10 μmol of the resin, to which DIEA (15 equiv) and triphosgene (3 equiv) were added. Reactions were shaken at room temperature for 30 min, after which the cartridge was drained into a solution of 10% NaOH in water. The resin was washed three times with dry DCM, which was drained into a 10% NaOH solution. Next, the resin was resuspended in 1 mL of dry DCM, to which 10 equiv of 1<i>H</i>-1,2,3-triazole was added. Reactions were shaken overnight at room temperature and checked for completion using LC–MS. After completion, the resin was washed three times with DCM. Probes were then cleaved from the resin by using a 95:2.5:2.5 mixture of TFA/TIS/water. Note that with this on resin synthesis, the N1-isomer of the triazole ureas was formed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S7</a>).</div></div><div id="sec4_5_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> <i>N</i>-(Diphenylphosphoryl)-isonipecotyl-<span class="smallcaps smallerCapital">l</span>-phenylalanyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>4a</b>)</h5><div class="NLM_p last">Probe <b>4a</b> was synthesized using subsequently methods 1, 2.3, and 3.1 and purified using HPLC. The product was obtained as a white powder. (1.84 mg, yield = 23.7%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>51</sub>N<sub>8</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 775.3691; found, 775.3412.</div></div><div id="sec4_5_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> <i>N</i>-(Succinimidyloxycarbonyl)-isonipecotyl-<span class="smallcaps smallerCapital">l</span>-phenylalanyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>4b</b>)</h5><div class="NLM_p last">Probe <b>4b</b> was synthesized using subsequently methods 1, 2.3, and 3.2 and purified using HPLC. The product was obtained as a white powder. (3.31 mg, yield = 48.4%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>45</sub>N<sub>9</sub>O<sub>8</sub> [M + H]<sup>+</sup>, 684.3464; found, 684.3167.</div></div><div id="sec4_5_6_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> <i>N</i>-(2-Aminopyridin-2-yl-carbonyl)-isonipecotyl-<span class="smallcaps smallerCapital">l</span>-phenylalanyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>4c</b>)</h5><div class="NLM_p last">Probe <b>4c</b> was synthesized using subsequently methods 1, 2.3, and 3.3 and purified using HPLC. The product was obtained as a white powder. (2.99 mg, yield = 45.1%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>46</sub>N<sub>10</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 663.3725; found, 663.3597.</div></div><div id="sec4_5_6_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> <i>N</i>-(Phenyloxycarbonyl)-isonipecotyl-<span class="smallcaps smallerCapital">l</span>-phenylalanyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>4d</b>)</h5><div class="NLM_p last">Probe <b>4d</b> was synthesized using subsequently methods 1, 2.3, and 3.4 and purified using HPLC. The product was obtained as a white powder. (3.37 mg, yield = 50.8%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>46</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 663.3613; found, 663.3211.</div></div><div id="sec4_5_6_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>N</i>-(4-Nitrophenyloxycarbonyl)-isonipecotyl-<span class="smallcaps smallerCapital">l</span>-phenylalanyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>4e</b>)</h5><div class="NLM_p last">Probe <b>4e</b> was synthesized using subsequently methods 1, 2.3, and 3.5 and purified using HPLC. The product was obtained as a white powder. (1.93 mg, yield = 27.3%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>45</sub>N<sub>9</sub>O<sub>8</sub> [M + H]<sup>+</sup>, 708.3464; found, 708.3064.</div></div><div id="sec4_5_6_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> <i>N</i>-(1<i>H</i>-1,2,3-Triazol-1-yl-carbonyl)-isonipecotyl-<span class="smallcaps smallerCapital">l</span>-phenylalanyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>4f</b>)</h5><div class="NLM_p last">Probe <b>4f</b> was synthesized using subsequently methods 1, 2.3, and 3.6 and purified using HPLC. The product was obtained as a white powder. (0.36 mg, yield = 5.6%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>43</sub>N<sub>11</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 638.3521; found, 638.3790.</div></div><div id="sec4_5_6_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> <i>N</i>-(Diphenylphosphoryl)-isonipecotyl-(<i>N</i>-benzyl)glycyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>5a</b>)</h5><div class="NLM_p last">Probe <b>5a</b> was synthesized using subsequently methods 1, 2.1, and 3.1 and purified using HPLC. The product was obtained as a white powder. (0.33 mg, yield = 4.3%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>51</sub>N<sub>8</sub>O<sub>7</sub>P [M + H]<sup>+</sup>, 775.3691; found, 775.3742.</div></div><div id="sec4_5_6_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> <i>N</i>-(Succinimidyloxycarbonyl)-isonipecotyl-(<i>N</i>-benzyl)glycyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>5b</b>)</h5><div class="NLM_p last">Probe <b>5b</b> was synthesized using subsequently methods 1, 2.1, and 3.2 and purified using HPLC. The product was obtained as a white powder. (1.29 mg, yield = 18.9%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>45</sub>N<sub>9</sub>O<sub>8</sub> [M + H]<sup>+</sup>, 684.3391; found, 684.3342.</div></div><div id="sec4_5_6_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>N</i>-(2-Aminopyridin-2-yl-carbonyl)-isonipecotyl-(<i>N</i>-benzyl)glycyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>5c</b>)</h5><div class="NLM_p last">Probe <b>5c</b> was synthesized using subsequently methods 1, 2.1, and 3.3 and purified using HPLC. The product was obtained as a white powder. (2.82 mg, yield = 42.5%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>46</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 663.37301; found, 663.3648.</div></div><div id="sec4_5_6_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> <i>N</i>-(Phenyloxycarbonyl)-isonipecotyl-(<i>N</i>-benzyl)glycyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>5d</b>)</h5><div class="NLM_p last">Probe <b>5d</b> was synthesized using subsequently methods 1, 2.1, and 3.4 and purified using HPLC. The product was obtained as a white powder. (2.80 mg, yield = 42.2%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>46</sub>N<sub>10</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 663.3619; found, 663.3772.</div></div><div id="sec4_5_6_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> <i>N</i>-(4-Nitrophenyloxycarbonyl)-isonipecotyl-(<i>N</i>-benzyl)glycyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>5e</b>)</h5><div class="NLM_p last">Probe <b>5e</b> was synthesized using subsequently methods 1, 2.1, and 3.5 and purified using HPLC. The product was obtained as a white powder. (2.97 mg, yield = 41.9%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>45</sub>N<sub>9</sub>O<sub>8</sub> [M + H]<sup>+</sup>, 708.3469; found, 708.3151.</div></div><div id="sec4_5_6_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> <i>N</i>-(1<i>H</i>-1,2,3-Triazol-1-yl-carbonyl)-isonipecotyl-(<i>N</i>-benzyl)glycyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>5f</b>)</h5><div class="NLM_p last">Probe <b>5f</b> was synthesized using subsequently methods 1, 2.1, and 3.6 and purified using HPLC. The product was obtained as a white powder. (1.06 mg, yield = 16.6%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>43</sub>N<sub>11</sub>O<sub>5</sub> [M + Na]<sup>+</sup>, 660.3346; found, 660.2908.</div></div><div id="sec4_5_6_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> <i>N</i>-(Diphenylphosphoryl)-4-(piperazinyl-methyl)-benzoyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>6a</b>)</h5><div class="NLM_p last">Probe <b>6a</b> was synthesized using subsequently methods 1, 2.2, and 3.1 and purified using HPLC. The product was obtained as a white powder. (0.93 mg, yield = 12.9%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>47</sub>N<sub>8</sub>O<sub>6</sub>P [M + H]<sup>+</sup>, 719.3435; found, 719.3851.</div></div><div id="sec4_5_6_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> <i>N</i>-(Succinimidyloxycarbonyl)-4-(piperazinyl-methyl)-benzoyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>6b</b>)</h5><div class="NLM_p last">Probe <b>6b</b> was synthesized using subsequently methods 1, 2.2, and 3.2 and purified using HPLC. The product was obtained as a white powder. (0.93 mg, yield = 12.9%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>41</sub>N<sub>9</sub>O<sub>7</sub> [M + H]<sup>+</sup>, 628.3201; found, 628.3192.</div></div><div id="sec4_5_6_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> <i>N</i>-(2-Aminopyridin-2-yl-carbonyl)-4-(piperazinyl-methyl)-benzoyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>6c</b>)</h5><div class="NLM_p last">Probe <b>6c</b> was synthesized using subsequently methods 1, 2.2, and 3.3 and purified using HPLC. The product was obtained as a white powder. (0.69 mg, yield = 11.4%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>31</sub>H<sub>42</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 607.3351; found, 607.3222.</div></div><div id="sec4_5_6_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> <i>N</i>-(Phenyloxycarbonyl)-4-(piperazinyl-methyl)-benzoyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>6d</b>)</h5><div class="NLM_p last">Probe <b>6d</b> was synthesized using subsequently methods 1, 2.2, and 3.4 and purified using HPLC. The product was obtained as a white powder. (0.49 mg, yield = 8.1%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>42</sub>N<sub>10</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 607.3463; found, 607.3647.</div></div><div id="sec4_5_6_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> <i>N</i>-(4-Nitrophenyloxycarbonyl)-4-(piperazinyl-methyl)-benzoyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>6e</b>)</h5><div class="NLM_p last">Probe <b>6e</b> was synthesized using subsequently methods 1, 2.2, and 3.5 and purified using HPLC. The product was obtained as a white powder. (0.63 mg, yield = 9.7%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>31</sub>H<sub>41</sub>N<sub>9</sub>O<sub>7</sub> [M + H]<sup>+</sup>, 652.3202; found, 652.2670.</div></div><div id="sec4_5_6_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> <i>N</i>-(1<i>H</i>-1,2,3-Triazol-1-yl-carbonyl)-4-(piperazinyl-methyl)-benzoyl-6-aminohexanoyl-<span class="smallcaps smallerCapital">l</span>-ε-azidolysine Amide (<b>6f</b>)</h5><div class="NLM_p last">Probe <b>6f</b> was synthesized using subsequently methods 1, 2.2, and 3.6 and purified using HPLC. The product was obtained as a white powder. (0.34 mg, yield = 5.8%) HRMS (MALDI-TOF) <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>39</sub>N<sub>11</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 582.3259; found, 582.3079.</div></div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Preparation of Tissue Lysates</h3><div class="NLM_p last">Mouse brains were thawed on ice, after which the hemispheres were separated using a razor blade and cut into small pieces. The pieces were collected in a Dounce homogenizer and homogenized on ice using 1.5 mL of cold (4 °C) lysis buffer (20 mM HEPES, 2 mM DTT, 1 mM MgCl<sub>2</sub>, 25 U/mL DNAse I, pH 7.2) until thorough lysis of the organ was visually assessed. After homogenization, the suspension was collected into Eppendorf tubes and the Dounce homogenizer was rinsed twice with small aliquots of lysis buffer, which were added to these tubes. The suspension was centrifuged (2500 g, 3 min, 4 °C) to remove cell debris. The supernatant was carefully pipetted and transferred to an ultracentrifuge tube. The debris was resuspended in 0.25 mL of lysis buffer and centrifuged again using the same procedure. The supernatant was added to the ultracentrifuge tubes, which were then spun in an ultracentrifuge (100,000<i>g</i>, 45 min, 4 °C, Beckman Optima TLX, TLA 120.2 rotor). The supernatant (=cytosolic protein fraction) was collected and the pellet (=membrane protein fraction) was suspended in 1.5 mL storage buffer (20 mM HEPES, 2 mM DTT, pH 7.2). Protein concentrations were determined using a BCA assay on a SpectraMax iD3 platereader. Aliquots were snap frozen in liquid nitrogen and stored at −80 °C until use.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Competition of Probes with FP-Rh</h3><div class="NLM_p last">Cytosolic or membrane fraction (30 μL) of the mouse brain lysate (1 mg/mL) was incubated for 1 h at room temperature with 0.3 μL of a stock solution of the probe (10 mM, DMSO) or 0.3 μL DMSO. Then, 0.3 μL of FP-Rh (100 μM, DMSO) was added to each sample and the samples were incubated for another hour. Next, 10 μL of 4× sample buffer was added to the samples, which were subsequently boiled at 95 °C for 2 min. Samples were resolved by SDS-PAGE on 12% acrylamide gels and visualized by measuring in-gel fluorescence on a Typhoon FLA 9500 scanner with excitation at 532 nm and detection at 568 nm with the photomultiplier set at 900 V. Patterns of probe-treated samples were compared with an FP-Rh treated control sample. Compounds that led to the full disappearance of gel bands were considered hits.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Comparison of 5f with JCP-174 alk</h3><div class="NLM_p last">Cytosolic or membrane fraction (30 μL) of the mouse brain lysate (1 mg/mL) was incubated for 1 h at room temperature with either 0.3 μL of a stock solution of probe <b>5f</b> (10 μM, DMSO), 0.3 μL of a stock solution of JCP-174 alk (1 mM, DMSO), or 0.3 μL DMSO. Then, 0.3 μL of FP-Rh (100 μM, DMSO) was added to each sample and the samples were incubated for 1 h at room temperature. Next, 10 μL of 4× sample buffer was added to the samples, which were subsequently boiled at 95 °C for 2 min. Samples were resolved by SDS-PAGE on 12% acrylamide gels and visualized by measuring in-gel fluorescence on a Typhoon FLA 9500 scanner with settings as described above.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Comparison of <b>5f</b> with ML-348 and ML-349</h3><div class="NLM_p last">Cytosolic fraction (30 μL) of the mouse brain lysate (1 mg/mL) was incubated for 1 h at room temperature with either 0.3 μL of a stock solution of probe <b>5f</b> (10 μM, DMSO), 0.3 μL of a stock solution of ML-348 or ML-349 (50 mM, DMSO), or 0.3 μL of DMSO. Then, 0.3 μL of FP-Rh (100 μM, DMSO) was added to each sample and the samples were incubated for 5 min at room temperature. Next, 10 μL of 4× sample buffer was added to the samples, which were subsequently boiled at 95 °C for 2 min. Samples were resolved by SDS-PAGE on 12% acrylamide gels and visualized by measuring in-gel fluorescence on a Typhoon FLA 9500 scanner with settings as described above.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Cell Permeability Assay in A431 Cells</h3><div class="NLM_p last">A431 cells were seeded at 300,000 cells per well into a 6-well plate and allowed to adhere overnight. The media was refreshed with 1 mL of Dulbecco’s modified Eagle medium (with 10% FBS and pen/strep), spiked with either 1 μL of <b>5f</b> or <b>6f</b> (1 mM, DMSO) or 1 μL of DMSO for a 2 h treatment at 37 °C. Next, the medium was aspirated and the cells were washed three times with PBS (5 min). Lysates were prepared by incubating cells in 150 μL of lysis buffer (20 mM HEPES, 150 mM NaCl, 2 mM DTT, 25 U/mL Benzonase, 0.5% NP-40, pH 7.2) before scraping cells. Lysates were centrifuged (13,000<i>g</i>, 5 min, room temperature) to pellet debris, after which the supernatant was pipetted off. Protein concentrations of all samples were measured using a BCA assay, and normalized to 2.39 mg/mL with storage buffer (20 mM HEPES, 2 mM DTT, pH 7.2). Each sample (90 μL) was incubated with either 0.9 μL of FP-Rh (DMSO, 100 μM) or 0.9 μL of DMSO at room temperature for 1 h. Next, 30 μL of 4× sample buffer was added to the samples, which were subsequently boiled at 95 °C for 2 min. Samples were resolved by SDS-PAGE on 12% acrylamide gels and visualized by measuring in-gel fluorescence on a Typhoon FLA 9500 scanner with settings as described above.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> IC<sub>50</sub> Determination</h3><div class="NLM_p last">Apparent IC<sub>50</sub> values were measured by using six different compound concentrations in a 1:3 serial dilution, starting at a highest concentration of 30 μM or 370 nM, depending on the potency of the compound. To this end 30 μL of cytosolic or membrane fraction of the mouse brain lysate (1 mg/mL) was incubated for 1 h at room temperature with 0.3 μL of a 100× stock solution of the probe in DMSO. Then, 0.3 μL of FP-Rh (100 μM, DMSO) was added to each sample and the samples were incubated for another hour. Next, 10 μL of 4× sample buffer was added to the samples, which were subsequently boiled at 95 °C for 2 min. Samples were resolved by SDS-PAGE on 12% acrylamide gels and visualized by in-gel fluorescence measuring on a Typhoon scanner as for the initial screening. Remaining enzyme activity was determined by measuring the integrated optical gel band intensity using ImageJ software. Values were normalized to the FP-Rh labeled control sample without inhibitor, which was set at 100% activity. IC<sub>50</sub> values were determined from a dose–response curve generated in GraphPad Prism.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Direct Labeling of Probe Targets</h3><div class="NLM_p last">Cytosolic or membrane fraction (30 μL) of the mouse brain lysate (1 mg/mL) was incubated for 1 h with 0.3 μL of a 100× stock solution of the appropriate concentration. Next, 0.5 μL of TAMRA-alkyne (1.5 mM in DMSO), 0.3 μL of THPTA (5 mM in 4:1 <i>t</i>-BuOH: DMSO), 0.6 μL of sodium ascorbate (50 mM in water), and 0.6 μL of CuSO<sub>4</sub> (50 mM in water) were added to each sample and the samples were incubated 1 h at room temperature. Afterward, 10 μL of 4× sample buffer was added to the samples, which were subsequently boiled at 95 °C for 2 min. Samples were resolved on an SDS-PAGE 12% acrylamide gel and visualized by in-gel fluorescence measuring on a Typhoon scanner.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Western Blot against APT-1/2</h3><div class="NLM_p">Mouse brain lysates (90 μL, 1 mg/mL of either cytosolic or membrane fraction) were treated for 1 h with 1 μM probe (from a 100× DMSO stock) or a corresponding amount of DMSO (control sample). All samples were subsequently treated with 1,5 μL of tritag (TAMRA-Lys(biotin)–NH–propyne; 1.5 mM in DMSO), 0.9 μL of THPTA (5 mM in water), 1.8 μL of CuSO<sub>4</sub> (50 mM in water), and 1.8 μL of sodium ascorbate (50 mM in water), after which they were incubated for another hour at room temperature. Next, 360 μL of cold acetone (−80 °C) was added to each sample to precipitate proteins and eliminate residual click reagents. The samples were incubated at −20 °C for 30 min to ensure efficient protein precipitation, after which they were spun down using a microcentrifuge at max speed. The supernatant was discarded, another 360 μL of cold acetone was added to the pellet of the samples and the procedure was repeated once more.</div><div class="NLM_p">Protein pellets were then redissolved in 90 μL of PBS with 2% SDS. Next, SDS was diluted with steps of 90 μL to a final concentration of 0.4% SDS. Then, 30 μL of a UltraLink Streptavidin resin slurry was added to each sample to pull down all biotin-labeled proteins. To this end, all samples were incubated on a thermomixer (1 h, 1200 rpm, 25 °C). Afterward, the samples were spun down using a microcentrifuge at full speed.</div><div class="NLM_p">The supernatant was carefully pipetted from the beads, after which the beads were washed with 120 μL of 0.1% SDS in PBS, subsequently with 120 μL of 1 M NaCl, then with 120 μL 1 M urea, then with 120 μL PBS, and finally with 120 μL Milli-Q water. After each washing step, the samples were spun down using a microcentrifuge at full speed and the supernatant was discarded. The beads were resuspended in 120 μL of sample buffer and boiled for 10 min to release the proteins from the beads. Samples were then resolved on a 12% acrylamide gel using SDS-PAGE.</div><div class="NLM_p last">A fluorescent image of the gels was made (Typhoon scanner), after which the proteins were transferred onto nitrocellulose membranes. Both membranes were incubated for 1 h with 3% milk/PBST. The milk was discarded and one of the gels was incubated overnight with 15 mL of anti-APT-1 antibody Ab 91606 (0.5 μg/mL in PBST), whereas the other membrane was incubated overnight with 15 mL of anti-APT-2 antibody Ab 151578 (0.5 μg/mL in PBST). The antibody-solutions were then decanted and the membranes washed four times with PBST. Next, both membranes were incubated with the anti-rabbit HRP antibody (0.2 μg/mL in PBST) for 1 h. Afterward, the membranes were washed four times with PBST and incubated with a SuperSignal West Pico PLUS chemiluminescent substrate (Life Technologies) just prior to development using a BioRad ChemiDoc imaging system.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Covalent Docking</h3><div class="NLM_p last">The different probes were drawn in ChemDraw professional 15.1, converted to a pdb file using Chem3D 15.1 and converted into a PDBQT by AutoDockTools 1.5.6. Probes were attached with their carbonyl function (with elimination of the leaving group) to the side chain of residue S119 of APT-1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QGS">6QGS</a>, from which water molecules and the palmitate ligand were removed), using the provided script at the autodock website (see for download: <a href="http://autodock.scripps.edu/resources/covalentdocking" class="extLink">http://autodock.scripps.edu/resources/covalentdocking</a>) with a slight modification. Molecules were then docked according to the flexible side chain method of AutoDock 4.2, as described by Bianco et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The connected molecules were treated as a fully flexible side chain. The binding site was covered by preparing a 60 × 76 × 70 size grid box with a grid spacing of 0.375 Å and the center at 7.837, 127.491, and 18.586.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i60"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01043" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18871" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18871" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01043?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01043</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional synthetic procedures, mechanism of the formation of hydantoins, coupling conditions and crude LC/MS traces, inhibition curves, representative gels for IC<sub>50</sub><sup>App</sup> determination, full gels after labeling mouse brain lysate, covalent docking, triazole urea structure elucidation, copies of NMR spectra and HPLC purity data, and coomassie stains of gels (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Coordinates of docked probes (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_002.pdb" class="ext-link">PDB</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf">jm0c01043_si_001.pdf (6.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_002.pdb">jm0c01043_si_002.pdb (2.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_003.pdb">jm0c01043_si_003.pdb (2.65 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_004.csv">jm0c01043_si_004.csv (2.01 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01043" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven H. L. Verhelst</span> - <span class="hlFld-Affiliation affiliation">Department
of Cellular and Molecular Medicine, Laboratory of Chemical Biology, KU Leuven, Herestr. 49, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Leibniz
Institute for Analytical Sciences ISAS, e.V., Otto-Hahn-Str. 6b, 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7400-1319" title="Orcid link">http://orcid.org/0000-0002-7400-1319</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#47343322312229693122352f222b3433072e34263469232267343322312229693122352f222b3433072c322b2232312229692522"><span class="__cf_email__" data-cfemail="7506011003101b5b0310071d10190601351c0614065b1110">[email protected]</span> <span class="__cf_email__" data-cfemail="691a1d0c1f0c07471f0c1b010c051a1d29021c050c1c1f0c07470b0c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roeland Vanhoutte</span> - <span class="hlFld-Affiliation affiliation">Department
of Cellular and Molecular Medicine, Laboratory of Chemical Biology, KU Leuven, Herestr. 49, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Merel A. T. van de Plassche</span> - <span class="hlFld-Affiliation affiliation">Department
of Cellular and Molecular Medicine, Laboratory of Chemical Biology, KU Leuven, Herestr. 49, 3000 Leuven, Belgium</span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>We acknowledge financial support from Research Foundation Flanders FWO (project number G0E3617N), the Ministerium für Kultur und Wissenschaft des Landes Nordrhein-Westfalen, the Regierende Bürgermeister von Berlin-inkl. Wissenschaft und Forschung, and the Bundesministerium für Bildung und Forschung.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30820" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30820" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Luc Baudemprez for recording NMR spectra and Rita Derua for HRMS measurements.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ABDH6</td><td class="NLM_def"><p class="first last">α,β-hydrolase domain containing protein 6</p></td></tr><tr><td class="NLM_term">ABP</td><td class="NLM_def"><p class="first last">activity-based probe</p></td></tr><tr><td class="NLM_term">ABPP</td><td class="NLM_def"><p class="first last">activity-based protein profiling</p></td></tr><tr><td class="NLM_term">APT-1/2</td><td class="NLM_def"><p class="first last">acyl protein thioesterase-1/2</p></td></tr><tr><td class="NLM_term">CuAAC</td><td class="NLM_def"><p class="first last">copper(I)-catalyzed azide alkyne cycloaddition</p></td></tr><tr><td class="NLM_term">DAGL</td><td class="NLM_def"><p class="first last">diacylglycerol lipase</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorophosphonate</p></td></tr><tr><td class="NLM_term">MAGL</td><td class="NLM_def"><p class="first last">monoacylglycerol lipase</p></td></tr><tr><td class="NLM_term">NHS</td><td class="NLM_def"><p class="first last"><i>N</i>-hydroxysuccinimide</p></td></tr><tr><td class="NLM_term">SH</td><td class="NLM_def"><p class="first last">serine hydrolase</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 32 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozarich, J. W.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: from enzyme chemistry to proteomic chemistry</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.75.101304.124125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1146%2Fannurev.biochem.75.101304.124125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=18366325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2008&pages=383-414&author=B.+F.+Cravattauthor=A.+T.+Wrightauthor=J.+W.+Kozarich&title=Activity-based+protein+profiling%3A+from+enzyme+chemistry+to+proteomic+chemistry&doi=10.1146%2Fannurev.biochem.75.101304.124125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: From enzyme chemistry to proteomic chemistry</span></div><div class="casAuthors">Cravatt, Benjamin F.; Wright, Aaron T.; Kozarich, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">383-414</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Genome sequencing projects have provided researchers with a complete inventory of the predicted proteins produced by eukaryotic and prokaryotic organisms.  Assignment of functions to these proteins represents one of the principal challenges for the field of proteomics.  Activity-based protein profiling (ABPP) has emerged as a powerful chem. proteomic strategy to characterize enzyme function directly in native biol. systems on a global scale.  Here, the authors review the basic technol. of ABPP, the enzyme classes addressable by this method, and the biol. discoveries attributable to its application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQFnUWDsqzOLVg90H21EOLACvtfcHk0li2g7ixzOsoew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D&md5=2f2c471169d2473f7024eeb41d4eaecd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.75.101304.124125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.75.101304.124125%26sid%3Dliteratum%253Aachs%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DWright%26aufirst%3DA.%2BT.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520from%2520enzyme%2520chemistry%2520to%2520proteomic%2520chemistry%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2008%26volume%3D77%26spage%3D383%26epage%3D414%26doi%3D10.1146%2Fannurev.biochem.75.101304.124125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadaghiani, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelst, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Tagging and detection strategies for activity-based proteomics</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2006.11.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.cbpa.2006.11.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=17174138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=20-28&author=A.+M.+Sadaghianiauthor=S.+H.+Verhelstauthor=M.+Bogyo&title=Tagging+and+detection+strategies+for+activity-based+proteomics&doi=10.1016%2Fj.cbpa.2006.11.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Tagging and detection strategies for activity-based proteomics</span></div><div class="casAuthors">Sadaghiani, Amir M.; Verhelst, Steven HL; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-28</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The field of activity-based proteomics is a relatively new discipline that makes use of small mols., termed activity-based probes (ABPs), to tag and monitor distinct sets of proteins within a complex proteome.  These activity-dependant labels facilitate anal. of systems-wide changes at the level of enzyme activity rather than simple protein abundance.  While the use of small mol. inhibitors to label enzyme targets is not a new concept, the past ten years have seen a rapid expansion in the diversity of probe families that have been developed.  In addn. to increasing the no. and types of enzymes that can be targeted by this method, there has also been an increase in the no. of methods used to visualize probes once they are bound to target enzymes.  In particular, the use of small org. fluorophores has created a wealth of applications for ABPs that range from biochem. profiling of diverse proteomes to direct imaging of active enzymes in live cells and even whole animals.  In addn., the advent of new bioorthogonal coupling chemistries now enables a diverse array of tags to be added after targets are labeled with an ABP.  This strategy has opened the door to new in vivo applications for activity-based proteomic methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj1rUOkmEji7Vg90H21EOLACvtfcHk0li70vYR9mqkjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFOhs7o%253D&md5=4ff5bfd6aa704847c90b51c015d37d74</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2006.11.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2006.11.030%26sid%3Dliteratum%253Aachs%26aulast%3DSadaghiani%26aufirst%3DA.%2BM.%26aulast%3DVerhelst%26aufirst%3DS.%2BH.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DTagging%2520and%2520detection%2520strategies%2520for%2520activity-based%2520proteomics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D20%26epage%3D28%26doi%3D10.1016%2Fj.cbpa.2006.11.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: The serine hydrolases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14699</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.26.14694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1073%2Fpnas.96.26.14694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10611275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=14694-14699&author=Y.+Liuauthor=M.+P.+Patricelliauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling%3A+The+serine+hydrolases&doi=10.1073%2Fpnas.96.26.14694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: the serine hydrolases</span></div><div class="casAuthors">Liu, Yongsheng; Patricelli, Matthew P.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">14694-14699</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">With the postgenome era rapidly approaching, new strategies for the functional anal. of proteins are needed.  To date, proteomics efforts have primarily been confined to recording variations in protein level rather than activity.  The ability to profile classes of proteins on the basis of changes in their activity would greatly accelerate both the assignment of protein function and the identification of potential pharmaceutical targets.  Here, we describe the chem. synthesis and utility of an active-site directed probe for visualizing dynamics in the expression and function of an entire enzyme family, the serine hydrolases.  By reacting this probe, a biotinylated fluorophosphonate referred to as FP-biotin, with crude tissue exts., we quickly and with high sensitivity detect numerous serine hydrolases, many of which display tissue-restricted patterns of expression.  Addnl., we show that FP-biotin labels these proteins in an activity-dependent manner that can be followed kinetically, offering a powerful means to monitor dynamics simultaneously in both protein function and expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8i5m0pTTunrVg90H21EOLACvtfcHk0li70vYR9mqkjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D&md5=d5d1a09328fc7715e9d30ca392733426</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.26.14694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.26.14694%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520The%2520serine%2520hydrolases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D14694%26epage%3D14699%26doi%3D10.1073%2Fpnas.96.26.14694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giang, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamp, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbaum, J. J.</span></span> <span> </span><span class="NLM_article-title">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/1615-9861(200109)1:9<1067::aid-prot1067>3.0.co;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=11990500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1067-1071&author=M.+P.+Patricelliauthor=D.+K.+Giangauthor=L.+M.+Stampauthor=J.+J.+Burbaum&title=Direct+visualization+of+serine+hydrolase+activities+in+complex+proteomes+using+fluorescent+active+site-directed+probes&doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3Aaid-prot1067%3E3.0.co%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span></div><div class="casAuthors">Patricelli, Matthew P.; Giang, Dan K.; Stamp, Lisa M.; Burbaum, Jonathan J.</div><div class="citationInfo"><span class="NLM_cas:title">Proteomics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1067-1071</span>CODEN:
                <span class="NLM_cas:coden">PROTC7</span>;
        ISSN:<span class="NLM_cas:issn">1615-9853</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The field of biochem. is currently faced with the enormous challenge of assigning functional significance to more than thirty thousand predicted protein products encoded by the human genome.  In order to accomplish this daunting task, methods will be required that facilitate the global anal. of proteins in complex biol. systems.  Recently, methods have been described for simultaneously monitoring the activity of multiple enzymes in crude proteomes based on their reactivity with tagged chem. probes.  These activity based probes (ABPs) have used either radiochem. or biotin/avidin-based detection methods to allow consolidated visualization of numerous enzyme activities.  Here we report the synthesis and evaluation of fluorescent activity based probes for the serine hydrolase super-family of enzymes.  The fluorescent methods detailed herein provide superior throughput, sensitivity, and quant. accuracy when compared to previously described ABPs, and provide a straight-forward platform for high-throughput proteome anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrunfp6WitoIbVg90H21EOLACvtfcHk0li70vYR9mqkjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D&md5=5ed5063f140af14e24f1ab2444785eb4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1615-9861%2528200109%25291%253A9%253C1067%253A%253AAID-PROT1067%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DGiang%26aufirst%3DD.%2BK.%26aulast%3DStamp%26aufirst%3DL.%2BM.%26aulast%3DBurbaum%26aufirst%3DJ.%2BJ.%26atitle%3DDirect%2520visualization%2520of%2520serine%2520hydrolase%2520activities%2520in%2520complex%2520proteomes%2520using%2520fluorescent%2520active%2520site-directed%2520probes%26jtitle%3DProteomics%26date%3D2001%26volume%3D1%26spage%3D1067%26epage%3D1071%26doi%3D10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3Aaid-prot1067%3E3.0.co%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jessani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Enzyme activity profiles of the secreted and membrane proteome that depict cancer cell invasiveness</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">10335</span>– <span class="NLM_lpage">10340</span>, <span class="refDoi"> DOI: 10.1073/pnas.162187599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1073%2Fpnas.162187599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=12149457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmt1Ciurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=10335-10340&author=N.+Jessaniauthor=Y.+Liuauthor=M.+Humphreyauthor=B.+F.+Cravatt&title=Enzyme+activity+profiles+of+the+secreted+and+membrane+proteome+that+depict+cancer+cell+invasiveness&doi=10.1073%2Fpnas.162187599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme activity profiles of the secreted and membrane proteome that depict cancer cell invasiveness</span></div><div class="casAuthors">Jessani, Nadim; Liu, Yongsheng; Humphrey, Mark; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">10335-10340</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">By primarily measuring changes in transcript and protein abundance, conventional genomics and proteomics methods may fail to detect significant posttranslational events that regulate protein activity and, ultimately, cell behavior.  To address these limitations, activity-based proteomic technologies that measure dynamics in protein function on a global scale would be of particular value.  Here, we describe the application of a chem. proteomics strategy to quant. compare enzyme activities across a panel of human breast and melanoma cancer cell lines.  A global anal. of the activity, subcellular distribution, and glycosylation state for the serine hydrolase superfamily resulted in the identification of a cluster of proteases, lipases, and esterases that distinguished cancer lines based on tissue of origin.  Strikingly, nearly all of these enzyme activities were down-regulated in the most invasive cancer lines examd., which instead up-regulated a distinct set of secreted and membrane-assocd. enzyme activities.  These invasiveness-assocd. enzymes included urokinase, a secreted serine protease with a recognized role in tumor progression, and a membrane-assocd. hydrolase KIAA1363, for which no previous link to cancer had been made.  Collectively, these results suggest that invasive cancer cells share discrete proteomic signatures that are more reflective of their biol. phenotype than cellular heritage, highlighting that a common set of enzymes may support the progression of tumors from a variety of origins and thus represent attractive targets for the diagnosis and treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Ym_lV8DsLrVg90H21EOLACvtfcHk0lhf3q_4YkJ-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmt1Ciurw%253D&md5=2c8cc24fc9ef2c84e8b0839eba33b3b8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.162187599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.162187599%26sid%3Dliteratum%253Aachs%26aulast%3DJessani%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHumphrey%26aufirst%3DM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DEnzyme%2520activity%2520profiles%2520of%2520the%2520secreted%2520and%2520membrane%2520proteome%2520that%2520depict%2520cancer%2520cell%2520invasiveness%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D10335%26epage%3D10340%26doi%3D10.1073%2Fpnas.162187599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">The pharmacological landscape and therapeutic potential of serine hydrolases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1038/nrd3620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1038%2Fnrd3620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=22212679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=52-68&author=D.+A.+Bachovchinauthor=B.+F.+Cravatt&title=The+pharmacological+landscape+and+therapeutic+potential+of+serine+hydrolases&doi=10.1038%2Fnrd3620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The pharmacological landscape and therapeutic potential of serine hydrolases</span></div><div class="casAuthors">Bachovchin, Daniel A.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-68</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Serine hydrolases perform crucial roles in many biol. processes, and several of these enzymes are targets of approved drugs for indications such as type 2 diabetes, Alzheimer's disease and infectious diseases.  Despite this, most of the human serine hydrolases (of which there are more than 200) remain poorly characterized with respect to their physiol. substrates and functions, and the vast majority lack selective, in vivo-active inhibitors.  Here, we review the current state of pharmacol. for mammalian serine hydrolases, including marketed drugs, compds. that are under clin. investigation and selective inhibitors emerging from academic probe development efforts.  We also highlight recent methodol. advances that have accelerated the rate of inhibitor discovery and optimization for serine hydrolases, which we anticipate will aid in their biol. characterization and, in some cases, therapeutic validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsWC4iV3Xc57Vg90H21EOLACvtfcHk0lhf3q_4YkJ-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOktA%253D%253D&md5=4fde1e23117ad11848a9c2eef85376b7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd3620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3620%26sid%3Dliteratum%253Aachs%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DThe%2520pharmacological%2520landscape%2520and%2520therapeutic%2520potential%2520of%2520serine%2520hydrolases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D52%26epage%3D68%26doi%3D10.1038%2Fnrd3620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savinainen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saario, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, J. T.</span></span> <span> </span><span class="NLM_article-title">The serine hydrolases MAGL, ABHD6 and ABHD12 as guardians of 2-arachidonoylglycerol signalling through cannabinoid receptors</span>. <i>Acta Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1111/j.1748-1716.2011.02280.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1111%2Fj.1748-1716.2011.02280.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21418147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1eiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2012&pages=267-276&author=J.+R.+Savinainenauthor=S.+M.+Saarioauthor=J.+T.+Laitinen&title=The+serine+hydrolases+MAGL%2C+ABHD6+and+ABHD12+as+guardians+of+2-arachidonoylglycerol+signalling+through+cannabinoid+receptors&doi=10.1111%2Fj.1748-1716.2011.02280.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The serine hydrolases MAGL, ABHD6 and ABHD12 as guardians of 2-arachidonoylglycerol signalling through cannabinoid receptors</span></div><div class="casAuthors">Savinainen, J. R.; Saario, S. M.; Laitinen, J. T.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Physiologica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-276</span>CODEN:
                <span class="NLM_cas:coden">APCHC4</span>;
        ISSN:<span class="NLM_cas:issn">1748-1708</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The endocannabinoid 2-arachidonoylglycerol (2-AG) is a lipid mediator involved in various physiol. processes.  In response to neural activity, 2-AG is synthesized post-synaptically, then activates pre-synaptic cannabinoid CB1 receptors (CB1Rs) in a retrograde manner, resulting in transient and long-lasting redn. of neurotransmitter release.  The signalling competence of 2-AG is tightly regulated by the balanced action between "on demand" biosynthesis and degrdn.  We review recent research on monoacylglycerol lipase (MAGL), ABHD6 and ABHD12, three serine hydrolases that together account for approx. 99% of brain 2-AG hydrolase activity.  MAGL is responsible for approx. 85% of 2-AG hydrolysis and colocalizes with CB1R in axon terminals.  It is therefore ideally positioned to terminate 2-AG-CB1R signalling regardless of the source of this endocannabinoid.  Its acute pharmacol. inhibition leads to 2-AG accumulation and CB1R-mediated behavioral responses.  Chronic MAGL inactivation results in 2-AG overload, desensitization of CB1R signalling and behavioral tolerance.  ABHD6 accounts for approx. 4% of brain 2-AG hydrolase activity but in neurons it rivals MAGL in efficacy.  Neuronal ABHD6 resides post-synaptically, often juxtaposed with CB1Rs, and its acute inhibition leads to activity-dependent accumulation of 2-AG.  In cortical slices, selective ABHD6 blockade facilitates CB1R-dependent long-term synaptic depression.  ABHD6 is therefore positioned to guard intracellular pools of 2-AG at the site of generation.  ABHD12 is highly expressed in microglia and accounts for approx. 9% of total brain 2-AG hydrolysis.  Mutations in ABHD12 gene are causally linked to a neurodegenerative disease called PHARC.  Whether ABHD12 qualifies as a bona fide member to the endocannabinoid system remains to be established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIBgLJavyxUbVg90H21EOLACvtfcHk0lhf3q_4YkJ-Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1eiuro%253D&md5=399f8c1e38ae5fd12886573245bec0ba</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2Fj.1748-1716.2011.02280.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1748-1716.2011.02280.x%26sid%3Dliteratum%253Aachs%26aulast%3DSavinainen%26aufirst%3DJ.%2BR.%26aulast%3DSaario%26aufirst%3DS.%2BM.%26aulast%3DLaitinen%26aufirst%3DJ.%2BT.%26atitle%3DThe%2520serine%2520hydrolases%2520MAGL%252C%2520ABHD6%2520and%2520ABHD12%2520as%2520guardians%2520of%25202-arachidonoylglycerol%2520signalling%2520through%2520cannabinoid%2520receptors%26jtitle%3DActa%2520Physiol.%26date%3D2012%26volume%3D204%26spage%3D267%26epage%3D276%26doi%3D10.1111%2Fj.1748-1716.2011.02280.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span> <span> </span><span class="NLM_article-title">Acyl protein thioesterase inhibitors as probes of dynamic S-palmitoylation</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1039/c3md00333g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1039%2FC3MD00333G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=25558349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=268-276&author=D.+Davdaauthor=B.+R.+Martin&title=Acyl+protein+thioesterase+inhibitors+as+probes+of+dynamic+S-palmitoylation&doi=10.1039%2Fc3md00333g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Acyl protein thioesterase inhibitors as probes of dynamic S-palmitoylation</span></div><div class="casAuthors">Davda, Dahvid; Martin, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">268-276</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Protein palmitoylation describes the hydrophobic post-translational modification of cysteine residues in certain proteins, and is required for the spatial organization and compn. of cellular membrane environments.  Certain palmitoylated proteins are processed by acyl protein thioesterase (APT) enzymes, which catalyze thioester hydrolysis of palmitoylated cysteine residues.  Inhibiting APT enzymes disrupts Ras trafficking and attenuates oncogenic growth signaling, highlighting these enzymes as potential therapeutic targets.  As members of the serine hydrolase enzyme family, APT enzymes can be assayed by fluorophosphonate activity-based protein profiling (ABPP) methods, allowing rapid profiling of inhibitor selectivity and potency.  In this review, the authors discuss recent progress in the development of potent and selective inhibitors to APT enzymes, including both reversible and irreversible inhibitor chemotypes.  These examples highlight how ABPP methods integrate with medicinal chem. for the discovery and optimization of inhibitors in complex proteomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFb0Q1vE52H7Vg90H21EOLACvtfcHk0lghb-KsglmAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymurY%253D&md5=782483184bf879838c70267583862374</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2FC3MD00333G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00333G%26sid%3Dliteratum%253Aachs%26aulast%3DDavda%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26atitle%3DAcyl%2520protein%2520thioesterase%2520inhibitors%2520as%2520probes%2520of%2520dynamic%2520S-palmitoylation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D268%26epage%3D276%26doi%3D10.1039%2Fc3md00333g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sontag, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunbhakdi-Craig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitterhuber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogris, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontag, E.</span></span> <span> </span><span class="NLM_article-title">Regulation of protein phosphatase 2A methylation by LCMT1 and PME-1 plays a critical role in differentiation of neuroblastoma cells</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">1455</span>– <span class="NLM_lpage">1465</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2010.07049.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1111%2Fj.1471-4159.2010.07049.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21044074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1aiurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2010&pages=1455-1465&author=J.-M.+Sontagauthor=V.+Nunbhakdi-Craigauthor=M.+Mitterhuberauthor=E.+Ogrisauthor=E.+Sontag&title=Regulation+of+protein+phosphatase+2A+methylation+by+LCMT1+and+PME-1+plays+a+critical+role+in+differentiation+of+neuroblastoma+cells&doi=10.1111%2Fj.1471-4159.2010.07049.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of protein phosphatase 2A methylation by LCMT1 and PME-1 plays a critical role in differentiation of neuroblastoma cells</span></div><div class="casAuthors">Sontag, Jean-Marie; Nunbhakdi-Craig, Viyada; Mitterhuber, Martina; Ogris, Egon; Sontag, Estelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1455-1465</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Neuritic alterations are a major feature of many neurodegenerative disorders.  Methylation of protein phosphatase 2A (PP2A) catalytic C subunit by the leucine carboxyl methyltransferase (LCMT1), and demethylation by the protein phosphatase methylesterase 1, is a crit. PP2A regulatory mechanism.  It modulates the formation of PP2A holoenzymes contg. the Bα subunit, which dephosphorylate key neuronal cytoskeletal proteins, including tau.  Significantly, we have reported that LCMT1, methylated C and Bα expression levels are down-regulated in Alzheimer disease-affected brain regions.  In this study, we show that enhanced expression of LCMT1 in cultured N2a neuroblastoma cells, which increases endogenous methylated C and Bα levels, induces changes in E-actin organization.  It promotes serum-independent neuritogenesis and development of extended tau-pos. processes upon N2a cell differentiation.  These stimulatory effects can be abrogated by LCMT1 knockdown and S-adenosylhomocysteine, an inhibitor of methylation reactions.  Expression of protein phosphatase methylesterase 1 and the methylation-site L309Δ C subunit mutant, which decrease intracellular methylated C and Bα levels, block N2a cell differentiation and LCMT1-mediated neurite formation.  Lastly, inducible and non-inducible knockdown of Bα in N2a cells inhibit process out-growth.  Altogether,these results establish a novel mechanistic link between PP2A methylation and development of neurite-like processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPi_pU7tjl97Vg90H21EOLACvtfcHk0lghb-KsglmAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1aiurrI&md5=d3fcf045dc77ab4e01f85d908a6a51b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2010.07049.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2010.07049.x%26sid%3Dliteratum%253Aachs%26aulast%3DSontag%26aufirst%3DJ.-M.%26aulast%3DNunbhakdi-Craig%26aufirst%3DV.%26aulast%3DMitterhuber%26aufirst%3DM.%26aulast%3DOgris%26aufirst%3DE.%26aulast%3DSontag%26aufirst%3DE.%26atitle%3DRegulation%2520of%2520protein%2520phosphatase%25202A%2520methylation%2520by%2520LCMT1%2520and%2520PME-1%2520plays%2520a%2520critical%2520role%2520in%2520differentiation%2520of%2520neuroblastoma%2520cells%26jtitle%3DJ.%2520Neurochem.%26date%3D2010%26volume%3D115%26spage%3D1455%26epage%3D1465%26doi%3D10.1111%2Fj.1471-4159.2010.07049.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blankman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Superfamily-wide portrait of serine hydrolase inhibition achieved by library-versus-library screening</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">20941</span>– <span class="NLM_lpage">20946</span>, <span class="refDoi"> DOI: 10.1073/pnas.1011663107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1073%2Fpnas.1011663107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21084632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFyrsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=20941-20946&author=D.+A.+Bachovchinauthor=T.+Jiauthor=W.+Liauthor=G.+M.+Simonauthor=J.+L.+Blankmanauthor=A.+Adibekianauthor=H.+Hooverauthor=S.+Niessenauthor=B.+F.+Cravatt&title=Superfamily-wide+portrait+of+serine+hydrolase+inhibition+achieved+by+library-versus-library+screening&doi=10.1073%2Fpnas.1011663107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Superfamily-wide portrait of serine hydrolase inhibition achieved by library-versus-library screening</span></div><div class="casAuthors">Bachovchin, Daniel A.; Ji, Tianyang; Li, Weiwei; Simon, Gabriel M.; Blankman, Jacqueline L.; Adibekian, Alexander; Hoover, Heather; Niessen, Sherry; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">20941-20946, S20941/1-S20941/172</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Serine hydrolases (SHs) are-one of the largest and most diverse enzyme classes in mammals.  They play fundamental roles in virtually all physiol. processes and are targeted by drugs to treat diseases such as diabetes, obesity, and neurodegenerative disorders.  Despite this, we lack biol. understanding for most of the 110+ predicted mammalian metabolic SHs, in large part because of a dearth of assays to assess their biochem. activities and a lack of selective inhibitors to probe their function in living systems.  We show here that the vast majority (>80%) of mammalian metabolic SHs can be labeled in proteomes by a single, active site-directed fluorophosphonate probe.  We exploit this universal activity-based assay in a library-vs.-library format to screen 70+ SHs against 140+ structurally diverse carbamates.  Lead inhibitors were discovered for ∼40% of the screened enzymes, including many poorly characterized SHs: Global profiles identified carbamate inhibitors that discriminate among highly sequence-related SHs and, conversely, enzymes that share inhibitor sensitivity profiles despite lacking sequence homol.  These findings indicate that sequence relatedness is not a strong predictor of shared pharmaol. within the SH superfamily.  Finally, we show that lead carbamate inhibitors can be optimized into pharmacol. probes that inactivate individual SHs with high specificity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonb1W0UOt-7rVg90H21EOLACvtfcHk0lghb-KsglmAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFyrsbnO&md5=18e26c50a5f005089fcf5763ff9edab8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1011663107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1011663107%26sid%3Dliteratum%253Aachs%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DJi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DBlankman%26aufirst%3DJ.%2BL.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DHoover%26aufirst%3DH.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DSuperfamily-wide%2520portrait%2520of%2520serine%2520hydrolase%2520inhibition%2520achieved%2520by%2520library-versus-library%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D20941%26epage%3D20946%26doi%3D10.1073%2Fpnas.1011663107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhouayek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minamimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrechen, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasdev, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S. H.</span></span> <span> </span><span class="NLM_article-title">In Vitro and in Vivo Evaluation of (11)C-Labeled Azetidinecarboxylates for Imaging Monoacylglycerol Lipase by PET Imaging Studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2278</span>– <span class="NLM_lpage">2291</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01400</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01400" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtlOmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2278-2291&author=R.+Chengauthor=W.+Moriauthor=L.+Maauthor=M.+Alhouayekauthor=A.+Hatoriauthor=Y.+Zhangauthor=D.+Ogasawaraauthor=G.+Yuanauthor=Z.+Chenauthor=X.+Zhangauthor=H.+Shiauthor=T.+Yamasakiauthor=L.+Xieauthor=K.+Kumataauthor=M.+Fujinagaauthor=Y.+Nagaiauthor=T.+Minamimotoauthor=M.+Svenssonauthor=L.+Wangauthor=Y.+Duauthor=M.+J.+Ondrechenauthor=N.+Vasdevauthor=B.+F.+Cravattauthor=C.+Fowlerauthor=M.-R.+Zhangauthor=S.+H.+Liang&title=In+Vitro+and+in+Vivo+Evaluation+of+%2811%29C-Labeled+Azetidinecarboxylates+for+Imaging+Monoacylglycerol+Lipase+by+PET+Imaging+Studies&doi=10.1021%2Facs.jmedchem.7b01400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro and in Vivo Evaluation of 11C-Labeled Azetidinecarboxylates for Imaging Monoacylglycerol Lipase by PET Imaging Studies</span></div><div class="casAuthors">Cheng, Ran; Mori, Wakana; Ma, Longle; Alhouayek, Mireille; Hatori, Akiko; Zhang, Yiding; Ogasawara, Daisuke; Yuan, Gengyang; Chen, Zhen; Zhang, Xiaofei; Shi, Hang; Yamasaki, Tomoteru; Xie, Lin; Kumata, Katsushi; Fujinaga, Masayuki; Nagai, Yuji; Minamimoto, Takafumi; Svensson, Mona; Wang, Lu; Du, Yunfei; Ondrechen, Mary Jo; Vasdev, Neil; Cravatt, Benjamin F.; Fowler, Christopher; Zhang, Ming-Rong; Liang, Steven H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2278-2291</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) is the principle enzyme for metabolizing endogenous cannabinoid ligand 2-arachidonoyglycerol (2-AG).  Blockade of MAGL increases 2-AG levels, resulting in subsequent activation of the endocannabinoid system, and has emerged as a novel therapeutic strategy to treat drug addiction, inflammation, and neurodegenerative diseases.  Herein we report a new series of MAGL inhibitors, which were radiolabeled by site-specific labeling technologies, including 11C-carbonylation and spirocyclic iodonium ylide (SCIDY) radiofluorination.  The lead compd. [11C]10 (MAGL-0519) demonstrated high specific binding and selectivity in vitro and in vivo.  We also obsd. unexpected washout kinetics with these irreversible radiotracers, in which in vivo evidence for turnover of the covalent residue was unveiled between MAGL and azetidine carboxylates.  This work may lead to new directions for drug discovery and PET tracer development based on azetidine carboxylate inhibitor scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm2LL5zxwjx7Vg90H21EOLACvtfcHk0lgQJbX92-PdRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtlOmtb8%253D&md5=b2b4ddcfcccadac13be8d731127df7db</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01400%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%26aulast%3DMori%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAlhouayek%26aufirst%3DM.%26aulast%3DHatori%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DYamasaki%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DKumata%26aufirst%3DK.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DMinamimoto%26aufirst%3DT.%26aulast%3DSvensson%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DOndrechen%26aufirst%3DM.%2BJ.%26aulast%3DVasdev%26aufirst%3DN.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DFowler%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DM.-R.%26aulast%3DLiang%26aufirst%3DS.%2BH.%26atitle%3DIn%2520Vitro%2520and%2520in%2520Vivo%2520Evaluation%2520of%2520%252811%2529C-Labeled%2520Azetidinecarboxylates%2520for%2520Imaging%2520Monoacylglycerol%2520Lipase%2520by%2520PET%2520Imaging%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2278%26epage%3D2291%26doi%3D10.1021%2Facs.jmedchem.7b01400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafroth, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giffenig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papisov, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collier, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josephson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S. H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Evaluation of (18)F-Labeled Monoacylglycerol Lipase Inhibitors as Novel Positron Emission Tomography Probes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8866</span>– <span class="NLM_lpage">8872</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00936</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00936" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslKis7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8866-8872&author=Z.+Chenauthor=W.+Moriauthor=H.+Fuauthor=M.+A.+Schafrothauthor=A.+Hatoriauthor=T.+Shaoauthor=G.+Zhangauthor=R.+S.+Vanauthor=Y.+Zhangauthor=K.+Huauthor=M.+Fujinagaauthor=L.+Wangauthor=V.+Belovauthor=D.+Ogasawaraauthor=P.+Giffenigauthor=X.+Dengauthor=J.+Rongauthor=Q.+Yuauthor=X.+Zhangauthor=M.+I.+Papisovauthor=Y.+Shaoauthor=T.+L.+Collierauthor=J.-A.+Maauthor=B.+F.+Cravattauthor=L.+Josephsonauthor=M.-R.+Zhangauthor=S.+H.+Liang&title=Design%2C+Synthesis%2C+and+Evaluation+of+%2818%29F-Labeled+Monoacylglycerol+Lipase+Inhibitors+as+Novel+Positron+Emission+Tomography+Probes&doi=10.1021%2Facs.jmedchem.9b00936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of 18F-Labeled Monoacylglycerol Lipase Inhibitors as Novel Positron Emission Tomography Probes</span></div><div class="casAuthors">Chen, Zhen; Mori, Wakana; Fu, Hualong; Schafroth, Michael A.; Hatori, Akiko; Shao, Tuo; Zhang, Genwei; Van, Richard S.; Zhang, Yiding; Hu, Kuan; Fujinaga, Masayuki; Wang, Lu; Belov, Vasily; Ogasawara, Daisuke; Giffenig, Pilar; Deng, Xiaoyun; Rong, Jian; Yu, Qingzhen; Zhang, Xiaofei; Papisov, Mikhail I.; Shao, Yihan; Collier, Thomas L.; Ma, Jun-An; Cravatt, Benjamin F.; Josephson, Lee; Zhang, Ming-Rong; Liang, Steven H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8866-8872</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dysfunction of monoacylglycerol lipase (MAGL) is assocd. with several psychopathol. disorders, including drug addiction and neurodegenerative diseases.  Herein we design, synthesize, and evaluate several irreversible fluorine-contg. MAGL inhibitors for positron emission tomog. (PET) ligand development.  Compd. 6 (identified from a therapeutic agent) was advanced for 18F-labeling via a novel spirocyclic iodonium ylide (SCIDY) strategy, which demonstrated high brain permeability and excellent specific binding.  This work supports further development of novel 18F-labeled MAGL PET probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ9noubfh-w7Vg90H21EOLACvtfcHk0lgQJbX92-PdRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslKis7bF&md5=be7433643e89a96579a7be18198b5546</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00936%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMori%26aufirst%3DW.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DSchafroth%26aufirst%3DM.%2BA.%26aulast%3DHatori%26aufirst%3DA.%26aulast%3DShao%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DVan%26aufirst%3DR.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DK.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBelov%26aufirst%3DV.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DGiffenig%26aufirst%3DP.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DRong%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DPapisov%26aufirst%3DM.%2BI.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DCollier%26aufirst%3DT.%2BL.%26aulast%3DMa%26aufirst%3DJ.-A.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DJosephson%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DM.-R.%26aulast%3DLiang%26aufirst%3DS.%2BH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Evaluation%2520of%2520%252818%2529F-Labeled%2520Monoacylglycerol%2520Lipase%2520Inhibitors%2520as%2520Novel%2520Positron%2520Emission%2520Tomography%2520Probes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8866%26epage%3D8872%26doi%3D10.1021%2Facs.jmedchem.9b00936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">An activity-based imaging probe for the integral membrane hydrolase KIAA1363</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">966</span>– <span class="NLM_lpage">970</span>, <span class="refDoi"> DOI: 10.1002/anie.201107236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fanie.201107236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsF2lur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=966-970&author=J.+W.+Changauthor=R.+E.+Moelleringauthor=B.+F.+Cravatt&title=An+activity-based+imaging+probe+for+the+integral+membrane+hydrolase+KIAA1363&doi=10.1002%2Fanie.201107236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">An Activity-Based Imaging Probe for the Integral Membrane Hydrolase KIAA1363</span></div><div class="casAuthors">Chang, Jae Won; Moellering, Raymond E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">966-970, S966/1-S966/4</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">, The authors report herein a versatile fluorescent probe for visualizing KIAA1363 activity in cancer cells and in vivo.  This agent JW576 is potent (sub-μM IC50 values in cells) and selective for KIAA1363 and was used (1) in imaging expts. to det. that KIAA1363 is largely, but not exclusively localized to the ER in cancer cells, and (2) in pulse-chase expts. to est. a half-life of 14-23 h for KIAA1363 in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro5_GtuVmlf7Vg90H21EOLACvtfcHk0lj8NNjxrp2pVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsF2lur7F&md5=1f023c2ba887da6397b6613e02ae16ea</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107236%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DAn%2520activity-based%2520imaging%2520probe%2520for%2520the%2520integral%2520membrane%2520hydrolase%2520KIAA1363%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D966%26epage%3D970%26doi%3D10.1002%2Fanie.201107236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohsiek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Nieuwendijk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van
den Berg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Quenched Activity-based Probes for Diacylglycerol Lipase and alpha,beta-Hydrolase Domain Containing Protein 6</span>. <i>Chem.—Asian J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3491</span>– <span class="NLM_lpage">3500</span>, <span class="refDoi"> DOI: 10.1002/asia.201800452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fasia.201800452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=29901868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3491-3500&author=E.+J.+van%0ARoodenauthor=M.+Kohsiekauthor=R.+Kreekelauthor=A.+C.+M.+van+Esbroeckauthor=A.+van+den+Nieuwendijkauthor=A.+P.+A.+Janssenauthor=R.+van%0Aden+Bergauthor=H.+S.+Overkleeftauthor=M.+van+der+Stelt&title=Design+and+Synthesis+of+Quenched+Activity-based+Probes+for+Diacylglycerol+Lipase+and+alpha%2Cbeta-Hydrolase+Domain+Containing+Protein+6&doi=10.1002%2Fasia.201800452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Quenched Activity-based Probes for Diacylglycerol Lipase and α,β-Hydrolase Domain Containing Protein 6</span></div><div class="casAuthors">van Rooden, E. J.; Kohsiek, M.; Kreekel, R.; van Esbroeck, A. C. M.; van den Nieuwendijk, A. M. C. H.; Janssen, A. P. A.; van den Berg, R. J. B. H. N.; Overkleeft, H. S.; van der Stelt, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - An Asian Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3491-3500</span>CODEN:
                <span class="NLM_cas:coden">CAAJBI</span>;
        ISSN:<span class="NLM_cas:issn">1861-4728</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Diacylglycerol lipases (DAGL) are responsible for the biosynthesis of the endocannabinoid 2-arachidonoylglycerol.  The fluorescent activity-based probes (ABPs) DH379 and HT-01 have been previously shown to label DAGLs and to cross-react with the serine hydrolase ABHD6.  Here, we report the synthesis and characterization of two new quenched activity-based probes (qABPs) 1 and 2, the design of which was based on the structures of DH379 and HT-01, resp.  Probe 1 contains a BODIPY-FL and a 2,4-dinitroaniline moiety as a fluorophore-quencher pair, whereas probe 2 employs a Cy5-fluorophore and a cAB40-quencher.  The fluorescence of both probes was quenched with relative quantum yields of 0.34 and 0.0081, resp.  The probes showed target inhibition as characterized in activity-based protein profiling assays using human cell- and mouse brain lysates, but were unfortunately not active in living cells, presumably due to limited cell permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7O8u5-q5bXrVg90H21EOLACvtfcHk0lj8NNjxrp2pVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemurzN&md5=cc33901cc7c37c68a51f71223dc08b26</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fasia.201800452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fasia.201800452%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3DKohsiek%26aufirst%3DM.%26aulast%3DKreekel%26aufirst%3DR.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dvan%2Bden%2BNieuwendijk%26aufirst%3DA.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3Dvan%2Bden%2BBerg%26aufirst%3DR.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Quenched%2520Activity-based%2520Probes%2520for%2520Diacylglycerol%2520Lipase%2520and%2520alpha%252Cbeta-Hydrolase%2520Domain%2520Containing%2520Protein%25206%26jtitle%3DChem.%25E2%2580%2594Asian%2520J.%26date%3D2018%26volume%3D13%26spage%3D3491%26epage%3D3500%26doi%3D10.1002%2Fasia.201800452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Dulk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Berg, R. J. B. H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Codée, J. D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Two-step activity-based protein profiling of diacylglycerol lipase</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">5250</span>– <span class="NLM_lpage">5253</span>, <span class="refDoi"> DOI: 10.1039/c8ob01499j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1039%2FC8OB01499J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=30004552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlamsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=5250-5253&author=E.+J.+van%0ARoodenauthor=R.+Kreekelauthor=T.+Hansenauthor=A.+P.+A.+Janssenauthor=A.+C.+M.+van+Esbroeckauthor=H.+den+Dulkauthor=R.+J.+B.+H.+N.+van+den%0ABergauthor=J.+D.+C.+Cod%C3%A9eauthor=M.+van+der+Stelt&title=Two-step+activity-based+protein+profiling+of+diacylglycerol+lipase&doi=10.1039%2Fc8ob01499j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Two-step activity-based protein profiling of diacylglycerol lipase</span></div><div class="casAuthors">van Rooden, Eva J.; Kreekel, Roy; Hansen, Thomas; Janssen, Antonius P. A.; van Esbroeck, Annelot C. M.; den Dulk, Hans; van den Berg, Richard J. B. H. N.; Codee, Jeroen D. C.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">5250-5253</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Diacylglycerol lipases (DAGL) produce the endocannabinoid 2-arachidonoylglycerol, a key modulator of neurotransmitter release.  Chem. tools that visualize endogenous DAGL activity are desired.  Here, we report the design, synthesis and application of a triazole urea probe for DAGL equipped with a norbornene as a biorthogonal handle.  The activity and selectivity of the probe was assessed with activity-based protein profiling.  This probe was potent against endogenous DAGLα (IC50 = 5 nM) and it was successfully applied as a two-step activity-based probe for labeling of DAGLα using an inverse electron-demand Diels-Alder ligation in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0ykcj8l_rsbVg90H21EOLACvtfcHk0lj8NNjxrp2pVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlamsrzF&md5=8768f003f256f2731f80c645e51b72ae</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1039%2FC8OB01499J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB01499J%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3DKreekel%26aufirst%3DR.%26aulast%3DHansen%26aufirst%3DT.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dden%2BDulk%26aufirst%3DH.%26aulast%3Dvan%2Bden%2BBerg%26aufirst%3DR.%2BJ.%2BB.%2BH.%2BN.%26aulast%3DCod%25C3%25A9e%26aufirst%3DJ.%2BD.%2BC.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DTwo-step%2520activity-based%2520protein%2520profiling%2520of%2520diacylglycerol%2520lipase%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D5250%26epage%3D5253%26doi%3D10.1039%2Fc8ob01499j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niphakis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide reactivity profiling identifies diverse carbamate chemotypes tuned for serine hydrolase inhibition</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1590</span>– <span class="NLM_lpage">1599</span>, <span class="refDoi"> DOI: 10.1021/cb400261h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400261h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3sXot1Sru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1590-1599&author=J.+W.+Changauthor=A.+B.+Cognettaauthor=M.+J.+Niphakisauthor=B.+F.+Cravatt&title=Proteome-wide+reactivity+profiling+identifies+diverse+carbamate+chemotypes+tuned+for+serine+hydrolase+inhibition&doi=10.1021%2Fcb400261h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-Wide Reactivity Profiling Identifies Diverse Carbamate Chemotypes Tuned for Serine Hydrolase Inhibition</span></div><div class="casAuthors">Chang, Jae Won; Cognetta, Armand B., III; Niphakis, Micah J.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1590-1599</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Serine hydrolases are one of the largest and most diverse enzyme classes in Nature.  Inhibitors of serine hydrolases are used to treat many diseases, including obesity, diabetes, cognitive dementia, and bacterial and viral infections.  Nonetheless, the majority of the 200+ serine hydrolases in mammals still lack selective inhibitors for their functional characterization.  We and others have shown that activated carbamates, through covalent reaction with the conserved serine nucleophile of serine hydrolases, can serve as useful inhibitors for members of this enzyme family.  The extent to which carbamates, however, cross-react with other protein classes remains mostly unexplored.  Here, we address this problem by investigating the proteome-wide reactivity of a diverse set of activated carbamates in vitro and in vivo, using a combination of competitive and click chem. (CC)-activity-based protein profiling (ABPP).  We identify multiple classes of carbamates, including O-aryl, O-hexafluoroisopropyl (HFIP), and O-N-hydroxysuccinimidyl (NHS) carbamates that react selectively with serine hydrolases across entire mouse tissue proteomes in vivo.  We exploit the proteome-wide specificity of HFIP carbamates to create in situ imaging probes for the endocannabinoid hydrolases monoacylglycerol lipase (MAGL) and α-β hydrolase-6 (ABHD6).  These findings, taken together, designate the carbamate as a privileged reactive group for serine hydrolases that can accommodate diverse structural modifications to produce inhibitors that display exceptional potency and selectivity across the mammalian proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE-176v-DnJrVg90H21EOLACvtfcHk0lgNavCSu9ohSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXot1Sru7k%253D&md5=7286011459791bcdea13f918b9ef97c3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fcb400261h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400261h%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DNiphakis%26aufirst%3DM.%2BJ.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520reactivity%2520profiling%2520identifies%2520diverse%2520carbamate%2520chemotypes%2520tuned%2520for%2520serine%2520hydrolase%2520inhibition%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1590%26epage%3D1599%26doi%3D10.1021%2Fcb400261h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haedke, U. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frommel, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelst, S. H. L.</span></span> <span> </span><span class="NLM_article-title">Phosphoramidates as novel activity-based probes for serine proteases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1106</span>– <span class="NLM_lpage">1110</span>, <span class="refDoi"> DOI: 10.1002/cbic.201400013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fcbic.201400013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=24817682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslWku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1106-1110&author=U.+R.+Haedkeauthor=S.+C.+Frommelauthor=F.+Hansenauthor=H.+Hahneauthor=B.+Kusterauthor=M.+Bogyoauthor=S.+H.+L.+Verhelst&title=Phosphoramidates+as+novel+activity-based+probes+for+serine+proteases&doi=10.1002%2Fcbic.201400013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoramidates as Novel Activity-Based Probes for Serine Proteases</span></div><div class="casAuthors">Haedke, Ute R.; Frommel, Sandra C.; Hansen, Fabian; Hahne, Hannes; Kuster, Bernhard; Bogyo, Matthew; Verhelst, Steven H. L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1106-1110</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Activity-based probes (ABPs) are small mols. that exclusively form covalent bonds with catalytically active enzymes.  In the last decade, they have esp. been used in functional proteomics studies of proteases.  Here, we present phosphoramidate peptides as a novel type of ABP for serine proteases.  These mols. can be made in a straightforward manner by std. Fmoc-based solid-phase peptide synthesis, allowing rapid diversification.  The resulting ABPs covalently bind different serine proteases, depending on the amino acid recognition element adjacent to the reactive group.  A reporter tag enables downstream gel-based anal. or LC-MS/MS-mediated identification of the targeted proteases.  Overall, we believe that these readily accessible probes will provide new avenues for the functional study of serine proteases in complex proteomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxMEsjjMpPVLVg90H21EOLACvtfcHk0lgNavCSu9ohSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslWku7o%253D&md5=91c72566a3d9aeabd8949f56943e5282</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201400013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201400013%26sid%3Dliteratum%253Aachs%26aulast%3DHaedke%26aufirst%3DU.%2BR.%26aulast%3DFrommel%26aufirst%3DS.%2BC.%26aulast%3DHansen%26aufirst%3DF.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DBogyo%26aufirst%3DM.%26aulast%3DVerhelst%26aufirst%3DS.%2BH.%2BL.%26atitle%3DPhosphoramidates%2520as%2520novel%2520activity-based%2520probes%2520for%2520serine%2520proteases%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26spage%3D1106%26epage%3D1110%26doi%3D10.1002%2Fcbic.201400013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otrubova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Discovery libraries targeting the major enzyme classes: the serine hydrolases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3807</span>– <span class="NLM_lpage">3813</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.bmcl.2014.06.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=25037918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOjsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3807-3813&author=K.+Otrubovaauthor=V.+Srinivasanauthor=D.+L.+Boger&title=Discovery+libraries+targeting+the+major+enzyme+classes%3A+the+serine+hydrolases&doi=10.1016%2Fj.bmcl.2014.06.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery libraries targeting the major enzyme classes: The serine hydrolases</span></div><div class="casAuthors">Otrubova, Katerina; Srinivasan, Venkat; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3807-3813</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two libraries of modestly reactive ureas contg. either electron-deficient acyl anilines or acyl pyrazoles were prepd. and are reported as screening libraries for candidate serine hydrolase inhibitors.  Within each library is a small but powerful subset of compds. that serve as a chemotype fragment screening library capable of subsequent structural diversification.  Elaboration of the pyrazole-based ureas provided remarkably potent irreversible inhibitors of fatty acid amide hydrolase (FAAH, apparent Ki = 100-200 pM) complementary to those previously disclosed enlisting electron-deficient aniline-based ureas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0mdvToLulcrVg90H21EOLACvtfcHk0lgNavCSu9ohSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOjsL%252FK&md5=2dbb15e7145fadbeda928b1b0eba4560</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.063%26sid%3Dliteratum%253Aachs%26aulast%3DOtrubova%26aufirst%3DK.%26aulast%3DSrinivasan%26aufirst%3DV.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DDiscovery%2520libraries%2520targeting%2520the%2520major%2520enzyme%2520classes%253A%2520the%2520serine%2520hydrolases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3807%26epage%3D3813%26doi%3D10.1016%2Fj.bmcl.2014.06.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Click-generated triazole ureas as ultrapotent in vivo-active serine hydrolase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">478</span>, <span class="refDoi"> DOI: 10.1038/nchembio.579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1038%2Fnchembio.579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21572424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtV2ht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=469-478&author=A.+Adibekianauthor=B.+R.+Martinauthor=C.+Wangauthor=K.-L.+Hsuauthor=D.+A.+Bachovchinauthor=S.+Niessenauthor=H.+Hooverauthor=B.+F.+Cravatt&title=Click-generated+triazole+ureas+as+ultrapotent+in+vivo-active+serine+hydrolase+inhibitors&doi=10.1038%2Fnchembio.579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Click-generated triazole ureas as ultrapotent in vivo-active serine hydrolase inhibitors</span></div><div class="casAuthors">Adibekian, Alexander; Martin, Brent R.; Wang, Chu; Hsu, Ku-Lung; Bachovchin, Daniel A.; Niessen, Sherry; Hoover, Heather; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">469-478</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Serine hydrolases are a diverse enzyme class representing ∼1% of all human proteins.  The biol. functions of most serine hydrolases remain poorly characterized owing to a lack of selective inhibitors to probe their activity in living systems.  Here we show that a substantial no. of serine hydrolases can be irreversibly inactivated by 1,2,3-triazole ureas, which show negligible cross-reactivity with other protein classes.  Rapid lead optimization by click chem.-enabled synthesis and competitive activity-based profiling identified 1,2,3-triazole ureas that selectively inhibit enzymes from diverse branches of the serine hydrolase class, including peptidases (acyl-peptide hydrolase, or APEH), lipases (platelet-activating factor acetylhydrolase-2, or PAFAH2) and uncharacterized hydrolases (α,β-hydrolase-11, or ABHD11), with exceptional potency in cells (sub-nanomolar) and mice (<1 mg kg-1).  We show that APEH inhibition leads to accumulation of N-acetylated proteins and promotes proliferation in T cells.  These data indicate 1,2,3-triazole ureas are a pharmacol. privileged chemotype for serine hydrolase inhibition, combining broad activity across the serine hydrolase class with tunable selectivity for individual enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_5ZfJ_B1ZrVg90H21EOLACvtfcHk0lg4OHgfeNHOkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtV2ht7s%253D&md5=236ff8d9ef723ed09c00cf1c47b6727d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.579%26sid%3Dliteratum%253Aachs%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHsu%26aufirst%3DK.-L.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DHoover%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DClick-generated%2520triazole%2520ureas%2520as%2520ultrapotent%2520in%2520vivo-active%2520serine%2520hydrolase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D469%26epage%3D478%26doi%3D10.1038%2Fnchembio.579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Characterization of tunable piperidine and piperazine carbamates as inhibitors of endocannabinoid hydrolases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1830</span>– <span class="NLM_lpage">1842</span>, <span class="refDoi"> DOI: 10.1021/jm9016976</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9016976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1830-1842&author=J.+Z.+Longauthor=X.+Jinauthor=A.+Adibekianauthor=W.+Liauthor=B.+F.+Cravatt&title=Characterization+of+tunable+piperidine+and+piperazine+carbamates+as+inhibitors+of+endocannabinoid+hydrolases&doi=10.1021%2Fjm9016976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Tunable Piperidine and Piperazine Carbamates as Inhibitors of Endocannabinoid Hydrolases</span></div><div class="casAuthors">Long, Jonathan Z.; Jin, Xin; Adibekian, Alexander; Li, Weiwei; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1830-1842</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) and fatty acid amide hydrolase (FAAH) are two enzymes from the serine hydrolase superfamily that degrade the endocannabinoids 2-arachidonoylglycerol and anandamide, resp.  We have recently discovered that MAGL and FAAH are both inhibited by carbamates bearing an N-piperidine/piperazine group.  Piperidine/piperazine carbamates show excellent in vivo activity, raising brain endocannabinoid levels and producing CB1-dependent behavioral effects in mice, suggesting that they represent a promising class of inhibitors for studying the endogenous functions of MAGL and FAAH.  Herein, we disclose a full account of the syntheses, structure-activity relationships, and inhibitory activities of piperidine/piperazine carbamates against members of the serine hydrolase family.  These scaffolds can be tuned for MAGL-selective or dual MAGL-FAAH inhibition by the attachment of an appropriately substituted bisarylcarbinol or aryloxybenzyl moiety, resp., on the piperidine/piperazine ring.  Modifications to the piperidine/piperazine ring ablated inhibitory activity, suggesting a strict requirement for a six-membered ring to maintain potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphBYZrLsvC87Vg90H21EOLACvtfcHk0lg4OHgfeNHOkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSitLY%253D&md5=0a93fb0dcfb589ce52e0834d63fc0dda</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm9016976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9016976%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DJ.%2BZ.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DCharacterization%2520of%2520tunable%2520piperidine%2520and%2520piperazine%2520carbamates%2520as%2520inhibitors%2520of%2520endocannabinoid%2520hydrolases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1830%26epage%3D1842%26doi%3D10.1021%2Fjm9016976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aaltonen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savinainen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönkkö, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusisto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korhonen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navia-Paldanius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häyrinen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takabe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Käsnänen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantsar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasonen-Seppänen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevalainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, J. T.</span></span> <span> </span><span class="NLM_article-title">Piperazine and piperidine triazole ureas as ultrapotent and highly selective inhibitors of monoacylglycerol lipase</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.chembiol.2013.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=23521796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFOlur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=379-390&author=N.+Aaltonenauthor=J.+R.+Savinainenauthor=C.+R.+Ribasauthor=J.+R%C3%B6nkk%C3%B6author=A.+Kuusistoauthor=J.+Korhonenauthor=D.+Navia-Paldaniusauthor=J.+H%C3%A4yrinenauthor=P.+Takabeauthor=H.+K%C3%A4sn%C3%A4nenauthor=T.+Pantsarauthor=T.+Laitinenauthor=M.+Lehtonenauthor=S.+Pasonen-Sepp%C3%A4nenauthor=A.+Posoauthor=T.+Nevalainenauthor=J.+T.+Laitinen&title=Piperazine+and+piperidine+triazole+ureas+as+ultrapotent+and+highly+selective+inhibitors+of+monoacylglycerol+lipase&doi=10.1016%2Fj.chembiol.2013.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Piperazine and Piperidine Triazole Ureas as Ultrapotent and Highly Selective Inhibitors of Monoacylglycerol Lipase</span></div><div class="casAuthors">Aaltonen, Niina; Savinainen, Juha R.; Ribas, Casandra Riera; Ronkko, Jani; Kuusisto, Anne; Korhonen, Jani; Navia-Paldanius, Dina; Hayrinen, Jukka; Takabe, Piia; Kasnanen, Heikki; Pantsar, Tatu; Laitinen, Tuomo; Lehtonen, Marko; Pasonen-Seppanen, Sanna; Poso, Antti; Nevalainen, Tapio; Laitinen, Jarmo T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">379-390</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) terminates the signaling function of the endocannabinoid, 2-arachidonoylglycerol (2-AG).  During 2-AG hydrolysis, MAGL liberates arachidonic acid, feeding the principal substrate for the neuroinflammatory prostaglandins.  In cancer cells, MAGL redirects lipid stores toward protumorigenic signaling lipids.  Thus MAGL inhibitors may have great therapeutic potential.  Although potent and increasingly selective MAGL inhibitors have been described, their no. is still limited.  Here, we have characterized piperazine and piperidine triazole ureas that combine the high potency attributable to the triazole leaving group together with the bulky arom. benzodioxolyl moiety required for selectivity, culminating in compd. JJKK-048 that potently (IC50 < 0.4 nM) inhibited human and rodent MAGL.  JJKK-048 displayed low cross-reactivity with other endocannabinoid targets.  Activity-based protein profiling of mouse brain and human melanoma cell proteomes suggested high specificity also among the metabolic serine hydrolases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIDOxKwD3xf7Vg90H21EOLACvtfcHk0lg4OHgfeNHOkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFOlur0%253D&md5=3061a03d341bfb1de6b25c8d925b5b9f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DAaltonen%26aufirst%3DN.%26aulast%3DSavinainen%26aufirst%3DJ.%2BR.%26aulast%3DRibas%26aufirst%3DC.%2BR.%26aulast%3DR%25C3%25B6nkk%25C3%25B6%26aufirst%3DJ.%26aulast%3DKuusisto%26aufirst%3DA.%26aulast%3DKorhonen%26aufirst%3DJ.%26aulast%3DNavia-Paldanius%26aufirst%3DD.%26aulast%3DH%25C3%25A4yrinen%26aufirst%3DJ.%26aulast%3DTakabe%26aufirst%3DP.%26aulast%3DK%25C3%25A4sn%25C3%25A4nen%26aufirst%3DH.%26aulast%3DPantsar%26aufirst%3DT.%26aulast%3DLaitinen%26aufirst%3DT.%26aulast%3DLehtonen%26aufirst%3DM.%26aulast%3DPasonen-Sepp%25C3%25A4nen%26aufirst%3DS.%26aulast%3DPoso%26aufirst%3DA.%26aulast%3DNevalainen%26aufirst%3DT.%26aulast%3DLaitinen%26aufirst%3DJ.%2BT.%26atitle%3DPiperazine%2520and%2520piperidine%2520triazole%2520ureas%2520as%2520ultrapotent%2520and%2520highly%2520selective%2520inhibitors%2520of%2520monoacylglycerol%2520lipase%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D379%26epage%3D390%26doi%3D10.1016%2Fj.chembiol.2013.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malamas, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farah, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerts, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farizatto, K. L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahr, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makriyannis, A.</span></span> <span> </span><span class="NLM_article-title">Piperidine and piperazine inhibitors of fatty acid amide hydrolase targeting excitotoxic pathology</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">115096</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.115096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.bmc.2019.115096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=31629610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVSmu7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=115096&author=M.+Lamaniauthor=M.+S.+Malamasauthor=S.+I.+Farahauthor=V.+G.+Shuklaauthor=M.+F.+Almeidaauthor=C.+M.+Weertsauthor=J.+Andersonauthor=J.+T.+Woodauthor=K.+L.+G.+Farizattoauthor=B.+A.+Bahrauthor=A.+Makriyannis&title=Piperidine+and+piperazine+inhibitors+of+fatty+acid+amide+hydrolase+targeting+excitotoxic+pathology&doi=10.1016%2Fj.bmc.2019.115096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Piperidine and piperazine inhibitors of fatty acid amide hydrolase targeting excitotoxic pathology</span></div><div class="casAuthors">Lamani, Manjunath; Malamas, Michael S.; Farah, Shrouq I.; Shukla, Vidyanand G.; Almeida, Michael F.; Weerts, Catherine M.; Anderson, Joseph; Wood, JodiAnne T.; Farizatto, Karen L. G.; Bahr, Ben A.; Makriyannis, Alexandros</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">115096</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">FAAH inhibitors offer safety advantages by augmenting the anandamide levels "on demand" to promote neuroprotective mechanisms without the adverse psychotropic effects usually seen with direct and chronic activation of the CB1 receptor.  FAAH is an enzyme implicated in the hydrolysis of the endocannabinoid N-arachidonoylethanolamine (AEA), which is a partial agonist of the CB1 receptor.  Herein, we report the discovery of a new series of highly potent and selective carbamate FAAH inhibitors and their evaluation for neuroprotection.  The new inhibitors showed potent nanomolar inhibitory activity against human recombinant and purified rat FAAH, were selective (>1000-fold) against serine hydrolases MGL and ABHD6 and lacked any affinity for the cannabinoid receptors CB1 and CB2.  Evaluation of FAAH inhibitors 9(I) and 31(II) using the in vitro competitive activity-based protein profiling (ABPP) assay confirmed that both inhibitors were highly selective for FAAH in the brain, since none of the other FP-reactive serine hydrolases in this tissue were inhibited by these agents.  Our design strategy followed a traditional SAR approach and was supported by mol. modeling studies based on known FAAH cocrystal structures.  To rationally design new mols. that are irreversibly bound to FAAH, we have constructed "precovalent" FAAH-ligand complexes to identify good binding geometries of the ligands within the binding pocket of FAAH and then calcd. covalent docking poses to select compds. for synthesis.  FAAH inhibitors I and II were evaluated for neuroprotection in rat hippocampal slice cultures.  In the brain tissue, both inhibitors displayed protection against synaptic deterioration produced by kainic acid-induced excitotoxicity.  Thus, the resultant compds. produced through rational design are providing early leads for developing therapeutics against seizure-related damage assocd. with a variety of disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6K8DNdlhP_bVg90H21EOLACvtfcHk0lg83PFKL36u_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVSmu7fJ&md5=e61cc308e9f193ee715419635c61be55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.115096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.115096%26sid%3Dliteratum%253Aachs%26aulast%3DLamani%26aufirst%3DM.%26aulast%3DMalamas%26aufirst%3DM.%2BS.%26aulast%3DFarah%26aufirst%3DS.%2BI.%26aulast%3DShukla%26aufirst%3DV.%2BG.%26aulast%3DAlmeida%26aufirst%3DM.%2BF.%26aulast%3DWeerts%26aufirst%3DC.%2BM.%26aulast%3DAnderson%26aufirst%3DJ.%26aulast%3DWood%26aufirst%3DJ.%2BT.%26aulast%3DFarizatto%26aufirst%3DK.%2BL.%2BG.%26aulast%3DBahr%26aufirst%3DB.%2BA.%26aulast%3DMakriyannis%26aufirst%3DA.%26atitle%3DPiperidine%2520and%2520piperazine%2520inhibitors%2520of%2520fatty%2520acid%2520amide%2520hydrolase%2520targeting%2520excitotoxic%2520pathology%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D115096%26doi%3D10.1016%2Fj.bmc.2019.115096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baggelaar, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsicano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaouloff, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Chemical Proteomics Maps Brain Region Specific Activity of Endocannabinoid Hydrolases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">861</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1KlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=852-861&author=M.+P.+Baggelaarauthor=A.+C.+M.+van+Esbroeckauthor=E.+J.+van+Roodenauthor=B.+I.+Floreaauthor=H.+S.+Overkleeftauthor=G.+Marsicanoauthor=F.+Chaouloffauthor=M.+van+der+Stelt&title=Chemical+Proteomics+Maps+Brain+Region+Specific+Activity+of+Endocannabinoid+Hydrolases&doi=10.1021%2Facschembio.6b01052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomics Maps Brain Region Specific Activity of Endocannabinoid Hydrolases</span></div><div class="casAuthors">Baggelaar, Marc P.; van Esbroeck, Annelot C. M.; van Rooden, Eva J.; Florea, Bogdan I.; Overkleeft, Herman S.; Marsicano, Giovanni; Chaouloff, Francis; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">852-861</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biosynthetic and catabolic enzymes of the endocannabinoids tightly regulate endocannabinoid-mediated activation of the cannabinoid CB1 receptor.  Monitoring the activities of these endocannabinoid hydrolases in different brain regions is, therefore, key to gain insight in spatiotemporal control of CB1 receptor-mediated physiol.  We have employed a comparative chem. proteomics approach to quant. map the activity profile of endocannabinoid hydrolases in various mouse brain regions at the same time.  To this end, we used two different activity-based probes: fluorophosphonate-biotin (FP-biotin), which quantifies FAAH, ABHD6 and MAG-lipase activity, and MB108 that detects DAGL-α, ABHD4, ABHD6 and ABHD12.  In total 32 serine hydrolases were evaluated in the frontal cortex, hippocampus, striatum and cerebellum.  Comparison of endocannabinoid hydrolases activity in the four brain regions revealed that FAAH activity was highest in hippocampus, MAGL activity was most pronounced in the frontal cortex, whereas DAGL-α was most active in cerebellum.  Comparison of the activity profiles with a global proteomics dataset revealed pronounced differences.  This could indicate that post-translational modification of the endocannabinoid hydrolases is important to regulate their activity.  Next, the effect of genetic deletion of the CB1 receptor was studied.  No difference in the enzymic activity was found in the cerebellum, striatum, frontal cortex and hippocampus of CB1 receptor knockout animals compared to wild type mice.  Our results are in line with previous reports and indicate that the CB1 receptor exerts no regulatory control over the basal prodn. and degrdn. of endocannabinoids and that genetic deletion of the CB1 receptor does not induce compensatory mechanisms in endocannabinoid hydrolase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMiXmX3tZPyrVg90H21EOLACvtfcHk0lg83PFKL36u_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1KlsLc%253D&md5=33a113b4a16adb69e9cca18674daa5b3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01052%26sid%3Dliteratum%253Aachs%26aulast%3DBaggelaar%26aufirst%3DM.%2BP.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3DMarsicano%26aufirst%3DG.%26aulast%3DChaouloff%26aufirst%3DF.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DChemical%2520Proteomics%2520Maps%2520Brain%2520Region%2520Specific%2520Activity%2520of%2520Endocannabinoid%2520Hydrolases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D852%26epage%3D861%26doi%3D10.1021%2Facschembio.6b01052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blankman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">A comprehensive profile of brain enzymes that hydrolyze the endocannabinoid 2-arachidonoylglycerol</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1356</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2007.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.chembiol.2007.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=18096503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOktrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1347-1356&author=J.+L.+Blankmanauthor=G.+M.+Simonauthor=B.+F.+Cravatt&title=A+comprehensive+profile+of+brain+enzymes+that+hydrolyze+the+endocannabinoid+2-arachidonoylglycerol&doi=10.1016%2Fj.chembiol.2007.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Profile of Brain Enzymes that Hydrolyze the Endocannabinoid 2-Arachidonoylglycerol</span></div><div class="casAuthors">Blankman, Jacqueline L.; Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1347-1356</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Endogenous ligands for cannabinoid receptors ("endocannabinoids") include the lipid transmitters anandamide and 2-arachidonoylglycerol (2-AG).  Endocannabinoids modulate a diverse set of physiol. processes and are tightly regulated by enzymic biosynthesis and degrdn.  Termination of anandamide signaling by fatty acid amide hydrolase (FAAH) is well characterized, but less is known about the inactivation of 2-AG, which can be hydrolyzed by multiple enzymes in vitro, including FAAH and monoacylglycerol lipase (MAGL).  Here, we have taken a functional proteomic approach to comprehensively map 2-AG hydrolases in the mouse brain.  Our data reveal that ∼85% of brain 2-AG hydrolase activity can be ascribed to MAGL, and that the remaining 15% is mostly catalyzed by two uncharacterized enzymes, ABHD6 and ABHD12.  Interestingly, MAGL, ABHD6, and ABHD12 display distinct subcellular distributions, suggesting that they may control different pools of 2-AG in the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF_Kmd_7v5urVg90H21EOLACvtfcHk0lg83PFKL36u_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOktrfE&md5=78c6355a3909bff3cbae25989ab6a956</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2007.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2007.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlankman%26aufirst%3DJ.%2BL.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DA%2520comprehensive%2520profile%2520of%2520brain%2520enzymes%2520that%2520hydrolyze%2520the%2520endocannabinoid%25202-arachidonoylglycerol%26jtitle%3DChem.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1347%26epage%3D1356%26doi%3D10.1016%2Fj.chembiol.2007.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggelaar, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, A. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottenhoff, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Chemical Proteomic Analysis of Serine Hydrolase Activity in Niemann-Pick Type C Mouse Brain</span>. <i>Front. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">440</span>, <span class="refDoi"> DOI: 10.3389/fnins.2018.00440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.3389%2Ffnins.2018.00440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=30018533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252Fpt1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=440&author=E.+J.+van%0ARoodenauthor=A.+C.+M.+van+Esbroeckauthor=M.+P.+Baggelaarauthor=H.+Dengauthor=B.+I.+Floreaauthor=A.+R.+A.+Marquesauthor=R.+Ottenhoffauthor=R.+G.+Bootauthor=H.+S.+Overkleeftauthor=J.+Aertsauthor=M.+van+der%0AStelt&title=Chemical+Proteomic+Analysis+of+Serine+Hydrolase+Activity+in+Niemann-Pick+Type+C+Mouse+Brain&doi=10.3389%2Ffnins.2018.00440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomic Analysis of Serine Hydrolase Activity in Niemann-Pick Type C Mouse Brain</span></div><div class="casAuthors">van Rooden Eva J; van Esbroeck Annelot C M; Baggelaar Marc P; Deng Hui; van der Stelt Mario; Florea Bogdan I; Overkleeft Herman S; Marques Andre R A; Ottenhoff Roelof; Boot Rolf G; Aerts Johannes M F G</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">440</span>
        ISSN:<span class="NLM_cas:issn">1662-4548</span>.
    </div><div class="casAbstract">The endocannabinoid system (ECS) is considered to be an endogenous protective system in various neurodegenerative diseases.  Niemann-Pick type C (NPC) is a neurodegenerative disease in which the role of the ECS has not been studied yet.  Most of the endocannabinoid enzymes are serine hydrolases, which can be studied using activity-based protein profiling (ABPP).  Here, we report the serine hydrolase activity in brain proteomes of a NPC mouse model as measured by ABPP.  Two ABPP methods are used: a gel-based method and a chemical proteomics method.  The activities of the following endocannabinoid enzymes were quantified: diacylglycerol lipase (DAGL) α, α/β-hydrolase domain-containing protein 4, α/β-hydrolase domain-containing protein 6, α/β-hydrolase domain-containing protein 12, fatty acid amide hydrolase, and monoacylglycerol lipase.  Using the gel-based method, two bands were observed for DAGL α.  Only the upper band corresponding to this enzyme was significantly decreased in the NPC mouse model.  Chemical proteomics showed that three lysosomal serine hydrolase activities (retinoid-inducible serine carboxypeptidase, cathepsin A, and palmitoyl-protein thioesterase 1) were increased in Niemann-Pick C1 protein knockout mouse brain compared to wild-type brain, whereas no difference in endocannabinoid hydrolase activity was observed.  We conclude that these targets might be interesting therapeutic targets for future validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCz2J82E1UuxBrI9DTGjC0fW6udTcc2eaNhE-lf0XOL7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252Fpt1emsA%253D%253D&md5=00bc6207ceee7de4f4c996206dec98e8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3389%2Ffnins.2018.00440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnins.2018.00440%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3DBaggelaar%26aufirst%3DM.%2BP.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DMarques%26aufirst%3DA.%2BR.%2BA.%26aulast%3DOttenhoff%26aufirst%3DR.%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3DAerts%26aufirst%3DJ.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DChemical%2520Proteomic%2520Analysis%2520of%2520Serine%2520Hydrolase%2520Activity%2520in%2520Niemann-Pick%2520Type%2520C%2520Mouse%2520Brain%26jtitle%3DFront.%2520Neurosci.%26date%3D2018%26volume%3D12%26spage%3D440%26doi%3D10.3389%2Ffnins.2018.00440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Vega, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Confirming target engagement for reversible inhibitors in vivo by kinetically tuned activity-based probes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">10345</span>– <span class="NLM_lpage">10348</span>, <span class="refDoi"> DOI: 10.1021/ja303400u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303400u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC38Xotlelsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=10345-10348&author=A.+Adibekianauthor=B.+R.+Martinauthor=J.+W.+Changauthor=K.-L.+Hsuauthor=K.+Tsuboiauthor=D.+A.+Bachovchinauthor=A.+E.+Speersauthor=S.+J.+Brownauthor=T.+Spicerauthor=V.+Fernandez-Vegaauthor=J.+Fergusonauthor=P.+S.+Hodderauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Confirming+target+engagement+for+reversible+inhibitors+in+vivo+by+kinetically+tuned+activity-based+probes&doi=10.1021%2Fja303400u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Confirming Target Engagement for Reversible Inhibitors in Vivo by Kinetically Tuned Activity-Based Probes</span></div><div class="casAuthors">Adibekian, Alexander; Martin, Brent R.; Chang, Jae Won; Hsu, Ku-Lung; Tsuboi, Katsunori; Bachovchin, Daniel A.; Speers, Anna E.; Brown, Steven J.; Spicer, Timothy; Fernandez-Vega, Virneliz; Ferguson, Jill; Hodder, Peter S.; Rosen, Hugh; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10345-10348</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of small-mol. inhibitors for perturbing enzyme function requires assays to confirm that the inhibitors interact with their enzymic targets in vivo.  Detg. target engagement in vivo can be particularly challenging for poorly characterized enzymes that lack known biomarkers (e.g., endogenous substrates and products) to report on their inhibition.  Here, we describe a competitive activity-based protein profiling (ABPP) method for measuring the binding of reversible inhibitors to enzymes in animal models.  Key to the success of this approach is the use of activity-based probes that show tempered rates of reactivity with enzymes, such that competition for target engagement with reversible inhibitors can be measured in vivo.  We apply the competitive ABPP strategy to evaluate a newly described class of piperazine amide reversible inhibitors for the serine hydrolases LYPLA1 and LYPLA2, two enzymes for which selective, in vivo active inhibitors are lacking.  Competitive ABPP identified individual piperazine amides that selectively inhibit LYPLA1 or LYPLA2 in mice.  In summary, competitive ABPP adapted to operate with moderately reactive probes can assess the target engagement of reversible inhibitors in animal models to facilitate the discovery of small-mol. probes for characterizing enzyme function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbIIWu6eV4H7Vg90H21EOLACvtfcHk0lhCsStpYYyWRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xotlelsr0%253D&md5=dde167e236af3e9e27739ea1b4926bff</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja303400u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303400u%26sid%3Dliteratum%253Aachs%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DHsu%26aufirst%3DK.-L.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DSpicer%26aufirst%3DT.%26aulast%3DFernandez-Vega%26aufirst%3DV.%26aulast%3DFerguson%26aufirst%3DJ.%26aulast%3DHodder%26aufirst%3DP.%2BS.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DConfirming%2520target%2520engagement%2520for%2520reversible%2520inhibitors%2520in%2520vivo%2520by%2520kinetically%2520tuned%2520activity-based%2520probes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D10345%26epage%3D10348%26doi%3D10.1021%2Fja303400u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foe, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Linden, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Child, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Development of an activity-based probe for acyl-protein thioesterases</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e0190255</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0190255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1371%2Fjournal.pone.0190255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=29364904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12lsLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&author=M.+Garlandauthor=C.+J.+Schulzeauthor=I.+T.+Foeauthor=W.+A.+van+der+Lindenauthor=M.+A.+Childauthor=M.+Bogyo&title=Development+of+an+activity-based+probe+for+acyl-protein+thioesterases&doi=10.1371%2Fjournal.pone.0190255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an activity-based probe for acyl-protein thioesterases</span></div><div class="casAuthors">Garland, Megan; Schulze, Christopher J.; Foe, Ian T.; van der Linden, Wouter A.; Child, Matthew A.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0190255/1-e0190255/17</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein palmitoylation is a dynamic post-translational modification (PTM) important for cellular functions such as protein stability, trafficking, localization, and protein-protein interactions.  S-palmitoylation occurs via the addn. of palmitate to cysteine residues via a thioester linkage, catalyzed by palmitoyl acyl transferases (PATs), with removal of the palmitate catalyzed by acyl protein thioesterases (APTs) and palmitoyl-protein thioesterases (PPTs).  Tools that target the regulators of palmitoylation-PATs, APTs and PPTs-will improve understanding of this essential PTM.  Here, we describe the synthesis and application of a cell-permeable activity-based probe (ABP) that targets APTs in intact mammalian cells and the parasite Toxoplasma gondii.  Using a focused library of substituted chloroisocoumarins, we identified a probe scaffold with nanomolar affinity for human APTs (HsAPT1 and HsAPT2) and synthesized a fluorescent ABP, JCP174-BODIPY TMR (JCP174-BT).  We use JCP174-BT to profile HsAPT activity in situ in mammalian cells, to detect an APT in T. gondii (TgPPT1).  We show discordance between HsAPT activity levels and total protein concn. in some cell lines, indicating that total protein levels may not be representative of APT activity in complex systems, highlighting the utility of this probe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLW3SGhjwxIbVg90H21EOLACvtfcHk0lhCsStpYYyWRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12lsLnI&md5=46e3ee718dfc37bf7770bb2ee96a61e4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0190255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0190255%26sid%3Dliteratum%253Aachs%26aulast%3DGarland%26aufirst%3DM.%26aulast%3DSchulze%26aufirst%3DC.%2BJ.%26aulast%3DFoe%26aufirst%3DI.%2BT.%26aulast%3Dvan%2Bder%2BLinden%26aufirst%3DW.%2BA.%26aulast%3DChild%26aufirst%3DM.%2BA.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520an%2520activity-based%2520probe%2520for%2520acyl-protein%2520thioesterases%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26doi%3D10.1371%2Fjournal.pone.0190255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhEUZri056J5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bianco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1002/pro.2733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fpro.2733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=26103917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=295-301&author=G.+Biancoauthor=S.+Forliauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=Covalent+docking+using+autodock%3A+Two-point+attractor+and+flexible+side+chain+methods&doi=10.1002%2Fpro.2733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span></div><div class="casAuthors">Bianco, Giulia; Forli, Stefano; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">295-301</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We describe two methods of automated covalent docking using Autodock4: the two-point attractor method and the flexible side chain method.  Both methods were applied to a training set of 20 diverse protein-ligand covalent complexes, evaluating their reliability in predicting the crystallog. pose of the ligands.  The flexible side chain method performed best, recovering the pose in 75% of cases, with failures for the largest inhibitors tested.  Both methods are freely available at the AutoDock website ().</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbgwvwS_NTKrVg90H21EOLACvtfcHk0lhEUZri056J5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI&md5=4e380f340983e78e149f48934ccc59fd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fpro.2733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2733%26sid%3Dliteratum%253Aachs%26aulast%3DBianco%26aufirst%3DG.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DCovalent%2520docking%2520using%2520autodock%253A%2520Two-point%2520attractor%2520and%2520flexible%2520side%2520chain%2520methods%26jtitle%3DProtein%2520Sci.%26date%3D2016%26volume%3D25%26spage%3D295%26epage%3D301%26doi%3D10.1002%2Fpro.2733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Won, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labby, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pricer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majmudar, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armacost, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torossian, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palakurthi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, C. L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism for Isoform-Selective Inhibition of Acyl Protein Thioesterases 1 and 2 (APT1 and APT2)</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3374</span>– <span class="NLM_lpage">3382</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00720</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00720" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yjtb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=3374-3382&author=S.+J.+Wonauthor=D.+Davdaauthor=K.+J.+Labbyauthor=S.+Y.+Hwangauthor=R.+Pricerauthor=J.+D.+Majmudarauthor=K.+A.+Armacostauthor=L.+A.+Rodriguezauthor=C.+L.+Rodriguezauthor=F.+S.+Chongauthor=K.+A.+Torossianauthor=J.+Palakurthiauthor=E.+S.+Hurauthor=J.+L.+Meagherauthor=C.+L.+Brooksauthor=J.+A.+Stuckeyauthor=B.+R.+Martin&title=Molecular+Mechanism+for+Isoform-Selective+Inhibition+of+Acyl+Protein+Thioesterases+1+and+2+%28APT1+and+APT2%29&doi=10.1021%2Facschembio.6b00720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism for Isoform-Selective Inhibition of Acyl Protein Thioesterases 1 and 2 (APT1 and APT2)</span></div><div class="casAuthors">Won, Sang Joon; Davda, Dahvid; Labby, Kristin J.; Hwang, Sin Ye; Pricer, Rachel; Majmudar, Jaimeen D.; Armacost, Kira A.; Rodriguez, Laura A.; Rodriguez, Christina L.; Chong, Fei San; Torossian, Kristopher A.; Palakurthi, Jasmine; Hur, Edward S.; Meagher, Jennifer L.; Brooks, Charles L.; Stuckey, Jeanne A.; Martin, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3374-3382</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational S-palmitoylation directs the trafficking and membrane localization of hundreds of cellular proteins, often involving a coordinated palmitoylation cycle that requires both protein acyltransferases (PATs) and acylprotein thioesterases (APTs) to actively redistribute S-palmitoylated proteins toward different cellular membrane compartments.  This process is necessary for the trafficking and oncogenic signaling of S-palmitoylated Ras isoforms, and potentially other peripheral membrane proteins.  Depalmitoylating enzymes APT1 and APT2 are sep. conserved in all vertebrates, suggesting unique functional roles for each enzyme.  The recent discovery of APT isoform-selective inhibitors ML348 and ML349 has opened new possibilities to probe the function of each enzyme, yet it remains unclear how each inhibitor achieves orthogonal inhibition.  Here, the authors report the high-resoln. structure of human APT2 in complex with ML349 (1.64 Å), as well as the complementary structure of human APT1 bound to ML348 (1.55 Å).  Although the overall peptide backbone structures are nearly identical, each inhibitor adopted a distinct conformation within each active site.  In APT1, ML348 was positioned above the catalytic triad, but in APT2, the sulfonyl group of ML349 formed H-bonds with active site resident water mols. to indirectly engage the catalytic triad and oxyanion hole.  Reciprocal mutagenesis and activity profiling revealed several differing residues surrounding the active site that served as crit. gatekeepers for isoform accessibility and dynamics.  Structural and biochem. anal. suggested that the inhibitors occupy a putative acyl-binding region, establishing the mechanism for isoform-specific inhibition, hydrolysis of acyl substrates, and structural orthogonality important for future probe development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rqk6_xozJ7Vg90H21EOLACvtfcHk0lgRKVmyx6P87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yjtb7O&md5=ea0a24c33162441e1d53d1223d05f79a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00720%26sid%3Dliteratum%253Aachs%26aulast%3DWon%26aufirst%3DS.%2BJ.%26aulast%3DDavda%26aufirst%3DD.%26aulast%3DLabby%26aufirst%3DK.%2BJ.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DPricer%26aufirst%3DR.%26aulast%3DMajmudar%26aufirst%3DJ.%2BD.%26aulast%3DArmacost%26aufirst%3DK.%2BA.%26aulast%3DRodriguez%26aufirst%3DL.%2BA.%26aulast%3DRodriguez%26aufirst%3DC.%2BL.%26aulast%3DChong%26aufirst%3DF.%2BS.%26aulast%3DTorossian%26aufirst%3DK.%2BA.%26aulast%3DPalakurthi%26aufirst%3DJ.%26aulast%3DHur%26aufirst%3DE.%2BS.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26atitle%3DMolecular%2520Mechanism%2520for%2520Isoform-Selective%2520Inhibition%2520of%2520Acyl%2520Protein%2520Thioesterases%25201%2520and%25202%2520%2528APT1%2520and%2520APT2%2529%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D3374%26epage%3D3382%26doi%3D10.1021%2Facschembio.6b00720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delano, W. L.</span></span> <i>The PyMOL Molecular Graphics Systems</i>, <span class="NLM_year">2002</span>; <a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+Delano&title=The+PyMOL+Molecular+Graphics+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDelano%26aufirst%3DW.%2BL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520Systems%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubin, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dax, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzolio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogenstaetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Codd, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaplan, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carruthers, N. I.</span></span> <span> </span><span class="NLM_article-title">Identification and biological evaluation of 4-(3-trifluoromethylpyridin-2-yl)piperazine-1-carboxylic acid (5-trifluoromethylpyridin-2-yl)amide, a high affinity TRPV1 (VR1) vanilloid receptor antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1857</span>– <span class="NLM_lpage">1872</span>, <span class="refDoi"> DOI: 10.1021/jm0495071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0495071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD2cXps1Ogu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1857-1872&author=D.+M.+Swansonauthor=A.+E.+Dubinauthor=C.+Shahauthor=N.+Nasserauthor=L.+Changauthor=S.+L.+Daxauthor=M.+Jetterauthor=J.+G.+Breitenbucherauthor=C.+Liuauthor=C.+Mazurauthor=B.+Lordauthor=L.+Gonzalesauthor=K.+Hoeyauthor=M.+Rizzolioauthor=M.+Bogenstaetterauthor=E.+E.+Coddauthor=D.+H.+Leeauthor=S.-P.+Zhangauthor=S.+R.+Chaplanauthor=N.+I.+Carruthers&title=Identification+and+biological+evaluation+of+4-%283-trifluoromethylpyridin-2-yl%29piperazine-1-carboxylic+acid+%285-trifluoromethylpyridin-2-yl%29amide%2C+a+high+affinity+TRPV1+%28VR1%29+vanilloid+receptor+antagonist&doi=10.1021%2Fjm0495071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and biological evaluation of 4-(3-trifluoromethylpyridin-2-yl)piperazine-1-carboxylic acid (5-trifluoromethylpyridin-2-yl)amide, a high affinity TRPV1 (VR1) vanilloid receptor antagonist</span></div><div class="casAuthors">Swanson, Devin M.; Dubin, Adrienne E.; Shah, Chandra; Nasser, Nadia; Chang, Leon; Dax, Scott L.; Jetter, Michele; Breitenbucher, J. Guy; Liu, Changlu; Mazur, Curt; Lord, Brian; Gonzales, Lisa; Hoey, Kenway; Rizzolio, Michele; Bogenstaetter, Michael; Codd, Ellen E.; Lee, Doo H.; Zhang, Sui-Po; Chaplan, Sandra R.; Carruthers, Nicholas I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1857-1872</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High throughput screening using the recombinant human TRPV1 receptor was used to identify a series of pyridinylpiperazine ureas as TRPV1 vanilloid receptor ligands.  Exploration of the structure-activity relationships by parallel synthesis identified the essential pharmacophoric elements for antagonism that permitted further optimization via targeted synthesis to provide a potent orally bioavailable and selective TRPV1 modulator 41 active in several in vivo models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_mnRdFjQEq7Vg90H21EOLACvtfcHk0lgRKVmyx6P87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXps1Ogu7g%253D&md5=068c51ef0f08d6885921032a66e90c64</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm0495071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0495071%26sid%3Dliteratum%253Aachs%26aulast%3DSwanson%26aufirst%3DD.%2BM.%26aulast%3DDubin%26aufirst%3DA.%2BE.%26aulast%3DShah%26aufirst%3DC.%26aulast%3DNasser%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DDax%26aufirst%3DS.%2BL.%26aulast%3DJetter%26aufirst%3DM.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DMazur%26aufirst%3DC.%26aulast%3DLord%26aufirst%3DB.%26aulast%3DGonzales%26aufirst%3DL.%26aulast%3DHoey%26aufirst%3DK.%26aulast%3DRizzolio%26aufirst%3DM.%26aulast%3DBogenstaetter%26aufirst%3DM.%26aulast%3DCodd%26aufirst%3DE.%2BE.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DZhang%26aufirst%3DS.-P.%26aulast%3DChaplan%26aufirst%3DS.%2BR.%26aulast%3DCarruthers%26aufirst%3DN.%2BI.%26atitle%3DIdentification%2520and%2520biological%2520evaluation%2520of%25204-%25283-trifluoromethylpyridin-2-yl%2529piperazine-1-carboxylic%2520acid%2520%25285-trifluoromethylpyridin-2-yl%2529amide%252C%2520a%2520high%2520affinity%2520TRPV1%2520%2528VR1%2529%2520vanilloid%2520receptor%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1857%26epage%3D1872%26doi%3D10.1021%2Fjm0495071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonie J.  van der Zouwen</span>, <span class="hlFld-ContribAuthor ">Martin D.  Witte</span>. </span><span class="cited-content_cbyCitation_article-title">Modular Approaches to Synthesize Activity- and Affinity-Based Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.644811" title="DOI URL">https://doi.org/10.3389/fchem.2021.644811</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.644811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.644811%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DModular%252BApproaches%252Bto%252BSynthesize%252BActivity-%252Band%252BAffinity-Based%252BChemical%252BProbes%26aulast%3Dvan%2Bder%2BZouwen%26aufirst%3DAntonie%2BJ.%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. ABPP and ABPs. (A) Left (boxed): ABPs generally consist of three elements: a reactive electrophile as warhead that engages in a mechanism-based reaction with the active site nucleophile of the target, a tag that allows detection and a spacer and/or recognition element to prevent a steric clash of the tag in the active site and to influence the selectivity of the probe. Right: ABPP allows the tagging of active enzyme targets in a complex proteome, such as a cell lysate, whole cell or in vivo setting, followed by various types of read-out. Inhibitor evaluation in complex proteomes may be done by “competitive ABPP”, in which the proteome is first treated with a potential inhibitor before incubation with the ABP. The disappearance of gel bands in sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) will then reveal inhibitor selectivity. (B) Structure of FP-based probes and the mechanism of reaction with SHs: the active site serine side chain, activated by hydrogen bonding to a histidine residue, engages in a mechanism-based reaction and results in a stable, covalent phosphonate ester probe–protein complex. (C) Modular design of SH ABPs for construction on a solid support.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Mix-and-match SH ABP Libraries<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(A) On-resin coupling of warheads to different types of amines. Reagents and conditions: (i) diphenylphosphoryl chloride, DIEA, and DCM. (ii) Disuccinimidylcarbonate, DIEA, and DCM. (iii) 2-Aminopyridinyl phenyl carbamate, DIEA, and DCM. (iv) Phenyl chloroformate, DIEA, and DCM. (v) (1). Triphosgene, DIEA, and DCM; (2) 1<i>H</i>-1,2,3-triazole, DIEA, and DCM. (B) Library synthesis toward compounds <b>4a–6f</b>. (vi) SPPS elongation by (1). DMF/piperidine 4/1; (2) Fmoc-aa-OH, HBTU, DIEA, and DMF. (vii) (1) DMF/piperidine 4/1; (2) bromoacetic acid, DIC, HOBt, and DMF. (viii) Benzylamine, DIEA, and DMF. (ix) Fmoc-Inp-OH, DIC, HOBt, and DMF. (x) <i>p</i>-Bromomethylbenzoic acid, PyBrOP, DIEA, and DMF. (xi) Piperazine and DMF. (xii) 4-nitrophenyl chloroformate, DIEA, and DCM.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Screening of SH ABP library for inhibition by competitive ABPP. (A) Fluorescent gels of competitive ABPP in the membrane fraction (upper) or soluble fraction (lower) of mouse brain lysates. Pretreatment of the lysate with the indicated probe was followed by measurement of residual SH activity with FP-Rh. (B) Apparent IC<sub>50</sub> curves were determined by competitive ABPP of serial dilutions, followed by quantification using gel band densitometry. (C) Competitive ABPP in the mouse brain lysate (soluble fraction) with known APT1/2 inhibitor JCP-174-alk (left panel) or APT1-selective inhibitor ML348 and APT2-selective inhibitor ML349 (right panel) in comparison with compound <b>5f</b>. Residual SH activity in treated tissue lysates was measured by incubation with FP-Rh. (D) Inhibition of APT1/2 by compounds <b>5f</b> and <b>6f</b> in live A431 cells. After treatment, compounds were washed out, and residual SH activity in cell lysates was detected with FP-Rh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/medium/jm0c01043_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Covalent interaction with APT-1/2. (A) Labeling of a soluble fraction of the mouse brain lysate with probes <b>4f–6f</b>, followed by click chemistry with a TAMRA-alkyne derivative shows selective detection of the ATP-1/2 gel bands. DMSO control (−) shows a click background. Other protein targets than APT-1/2 are indicated by a *. (B) Same as (A) but in the membrane fraction of the mouse brain lysate. (C) After click chemistry with a trifunctional biotin-TAMRA-alkyne tag and enrichment on streptavidin beads, fluorescent detection (upper panels) confirms target engagement of the probes, while detection by anti-APT-1 or anti-APT-2 antibodies (lower panels) confirms target identity. Note that the targets in the membrane fraction had very low abundance and led to very weak bands. Full fluorescent gels shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf" class="ext-link">Figure S5</a>. (D) Covalent docking of <b>5f</b> in a crystal structure of APT-1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QGS">6QGS</a>). Protein in a cartoon format with α-helices in cyan, β-sheets in magenta, and random coil in pink. A co-crystallized palmitate is shown in orange sticks, whereas <b>5f</b> is shown in magenta. Active site residues are depicted in green, whereas interacting residues are shown in white. Picture rendered with PyMol.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (E) Covalent docking of <b>6f</b> (in yellow) overlayed with co-crystallized palmitate (in orange) in a crystal structure of APT-1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01043/20201019/images/large/jm0c01043_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01043&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i63">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45718" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45718" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 32 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozarich, J. W.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: from enzyme chemistry to proteomic chemistry</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.75.101304.124125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1146%2Fannurev.biochem.75.101304.124125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=18366325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2008&pages=383-414&author=B.+F.+Cravattauthor=A.+T.+Wrightauthor=J.+W.+Kozarich&title=Activity-based+protein+profiling%3A+from+enzyme+chemistry+to+proteomic+chemistry&doi=10.1146%2Fannurev.biochem.75.101304.124125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: From enzyme chemistry to proteomic chemistry</span></div><div class="casAuthors">Cravatt, Benjamin F.; Wright, Aaron T.; Kozarich, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">383-414</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Genome sequencing projects have provided researchers with a complete inventory of the predicted proteins produced by eukaryotic and prokaryotic organisms.  Assignment of functions to these proteins represents one of the principal challenges for the field of proteomics.  Activity-based protein profiling (ABPP) has emerged as a powerful chem. proteomic strategy to characterize enzyme function directly in native biol. systems on a global scale.  Here, the authors review the basic technol. of ABPP, the enzyme classes addressable by this method, and the biol. discoveries attributable to its application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQFnUWDsqzOLVg90H21EOLACvtfcHk0liIb4-jP0eIqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekur0%253D&md5=2f2c471169d2473f7024eeb41d4eaecd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.75.101304.124125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.75.101304.124125%26sid%3Dliteratum%253Aachs%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DWright%26aufirst%3DA.%2BT.%26aulast%3DKozarich%26aufirst%3DJ.%2BW.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520from%2520enzyme%2520chemistry%2520to%2520proteomic%2520chemistry%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2008%26volume%3D77%26spage%3D383%26epage%3D414%26doi%3D10.1146%2Fannurev.biochem.75.101304.124125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadaghiani, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelst, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Tagging and detection strategies for activity-based proteomics</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">20</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2006.11.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.cbpa.2006.11.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=17174138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=20-28&author=A.+M.+Sadaghianiauthor=S.+H.+Verhelstauthor=M.+Bogyo&title=Tagging+and+detection+strategies+for+activity-based+proteomics&doi=10.1016%2Fj.cbpa.2006.11.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Tagging and detection strategies for activity-based proteomics</span></div><div class="casAuthors">Sadaghiani, Amir M.; Verhelst, Steven HL; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">20-28</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The field of activity-based proteomics is a relatively new discipline that makes use of small mols., termed activity-based probes (ABPs), to tag and monitor distinct sets of proteins within a complex proteome.  These activity-dependant labels facilitate anal. of systems-wide changes at the level of enzyme activity rather than simple protein abundance.  While the use of small mol. inhibitors to label enzyme targets is not a new concept, the past ten years have seen a rapid expansion in the diversity of probe families that have been developed.  In addn. to increasing the no. and types of enzymes that can be targeted by this method, there has also been an increase in the no. of methods used to visualize probes once they are bound to target enzymes.  In particular, the use of small org. fluorophores has created a wealth of applications for ABPs that range from biochem. profiling of diverse proteomes to direct imaging of active enzymes in live cells and even whole animals.  In addn., the advent of new bioorthogonal coupling chemistries now enables a diverse array of tags to be added after targets are labeled with an ABP.  This strategy has opened the door to new in vivo applications for activity-based proteomic methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj1rUOkmEji7Vg90H21EOLACvtfcHk0liIb4-jP0eIqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFOhs7o%253D&md5=4ff5bfd6aa704847c90b51c015d37d74</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2006.11.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2006.11.030%26sid%3Dliteratum%253Aachs%26aulast%3DSadaghiani%26aufirst%3DA.%2BM.%26aulast%3DVerhelst%26aufirst%3DS.%2BH.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DTagging%2520and%2520detection%2520strategies%2520for%2520activity-based%2520proteomics%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D20%26epage%3D28%26doi%3D10.1016%2Fj.cbpa.2006.11.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Activity-based protein profiling: The serine hydrolases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">14694</span>– <span class="NLM_lpage">14699</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.26.14694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1073%2Fpnas.96.26.14694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10611275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=14694-14699&author=Y.+Liuauthor=M.+P.+Patricelliauthor=B.+F.+Cravatt&title=Activity-based+protein+profiling%3A+The+serine+hydrolases&doi=10.1073%2Fpnas.96.26.14694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based protein profiling: the serine hydrolases</span></div><div class="casAuthors">Liu, Yongsheng; Patricelli, Matthew P.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">14694-14699</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">With the postgenome era rapidly approaching, new strategies for the functional anal. of proteins are needed.  To date, proteomics efforts have primarily been confined to recording variations in protein level rather than activity.  The ability to profile classes of proteins on the basis of changes in their activity would greatly accelerate both the assignment of protein function and the identification of potential pharmaceutical targets.  Here, we describe the chem. synthesis and utility of an active-site directed probe for visualizing dynamics in the expression and function of an entire enzyme family, the serine hydrolases.  By reacting this probe, a biotinylated fluorophosphonate referred to as FP-biotin, with crude tissue exts., we quickly and with high sensitivity detect numerous serine hydrolases, many of which display tissue-restricted patterns of expression.  Addnl., we show that FP-biotin labels these proteins in an activity-dependent manner that can be followed kinetically, offering a powerful means to monitor dynamics simultaneously in both protein function and expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8i5m0pTTunrVg90H21EOLACvtfcHk0liTSgjanUJY_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtFaitw%253D%253D&md5=d5d1a09328fc7715e9d30ca392733426</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.26.14694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.26.14694%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DActivity-based%2520protein%2520profiling%253A%2520The%2520serine%2520hydrolases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D14694%26epage%3D14699%26doi%3D10.1073%2Fpnas.96.26.14694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giang, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamp, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbaum, J. J.</span></span> <span> </span><span class="NLM_article-title">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/1615-9861(200109)1:9<1067::aid-prot1067>3.0.co;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=11990500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1067-1071&author=M.+P.+Patricelliauthor=D.+K.+Giangauthor=L.+M.+Stampauthor=J.+J.+Burbaum&title=Direct+visualization+of+serine+hydrolase+activities+in+complex+proteomes+using+fluorescent+active+site-directed+probes&doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3Aaid-prot1067%3E3.0.co%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Direct visualization of serine hydrolase activities in complex proteomes using fluorescent active site-directed probes</span></div><div class="casAuthors">Patricelli, Matthew P.; Giang, Dan K.; Stamp, Lisa M.; Burbaum, Jonathan J.</div><div class="citationInfo"><span class="NLM_cas:title">Proteomics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1067-1071</span>CODEN:
                <span class="NLM_cas:coden">PROTC7</span>;
        ISSN:<span class="NLM_cas:issn">1615-9853</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The field of biochem. is currently faced with the enormous challenge of assigning functional significance to more than thirty thousand predicted protein products encoded by the human genome.  In order to accomplish this daunting task, methods will be required that facilitate the global anal. of proteins in complex biol. systems.  Recently, methods have been described for simultaneously monitoring the activity of multiple enzymes in crude proteomes based on their reactivity with tagged chem. probes.  These activity based probes (ABPs) have used either radiochem. or biotin/avidin-based detection methods to allow consolidated visualization of numerous enzyme activities.  Here we report the synthesis and evaluation of fluorescent activity based probes for the serine hydrolase super-family of enzymes.  The fluorescent methods detailed herein provide superior throughput, sensitivity, and quant. accuracy when compared to previously described ABPs, and provide a straight-forward platform for high-throughput proteome anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrunfp6WitoIbVg90H21EOLACvtfcHk0liTSgjanUJY_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsFOmtrs%253D&md5=5ed5063f140af14e24f1ab2444785eb4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3AAID-PROT1067%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1615-9861%2528200109%25291%253A9%253C1067%253A%253AAID-PROT1067%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DGiang%26aufirst%3DD.%2BK.%26aulast%3DStamp%26aufirst%3DL.%2BM.%26aulast%3DBurbaum%26aufirst%3DJ.%2BJ.%26atitle%3DDirect%2520visualization%2520of%2520serine%2520hydrolase%2520activities%2520in%2520complex%2520proteomes%2520using%2520fluorescent%2520active%2520site-directed%2520probes%26jtitle%3DProteomics%26date%3D2001%26volume%3D1%26spage%3D1067%26epage%3D1071%26doi%3D10.1002%2F1615-9861%28200109%291%3A9%3C1067%3A%3Aaid-prot1067%3E3.0.co%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jessani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Enzyme activity profiles of the secreted and membrane proteome that depict cancer cell invasiveness</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">10335</span>– <span class="NLM_lpage">10340</span>, <span class="refDoi"> DOI: 10.1073/pnas.162187599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1073%2Fpnas.162187599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=12149457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmt1Ciurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=10335-10340&author=N.+Jessaniauthor=Y.+Liuauthor=M.+Humphreyauthor=B.+F.+Cravatt&title=Enzyme+activity+profiles+of+the+secreted+and+membrane+proteome+that+depict+cancer+cell+invasiveness&doi=10.1073%2Fpnas.162187599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme activity profiles of the secreted and membrane proteome that depict cancer cell invasiveness</span></div><div class="casAuthors">Jessani, Nadim; Liu, Yongsheng; Humphrey, Mark; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">10335-10340</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">By primarily measuring changes in transcript and protein abundance, conventional genomics and proteomics methods may fail to detect significant posttranslational events that regulate protein activity and, ultimately, cell behavior.  To address these limitations, activity-based proteomic technologies that measure dynamics in protein function on a global scale would be of particular value.  Here, we describe the application of a chem. proteomics strategy to quant. compare enzyme activities across a panel of human breast and melanoma cancer cell lines.  A global anal. of the activity, subcellular distribution, and glycosylation state for the serine hydrolase superfamily resulted in the identification of a cluster of proteases, lipases, and esterases that distinguished cancer lines based on tissue of origin.  Strikingly, nearly all of these enzyme activities were down-regulated in the most invasive cancer lines examd., which instead up-regulated a distinct set of secreted and membrane-assocd. enzyme activities.  These invasiveness-assocd. enzymes included urokinase, a secreted serine protease with a recognized role in tumor progression, and a membrane-assocd. hydrolase KIAA1363, for which no previous link to cancer had been made.  Collectively, these results suggest that invasive cancer cells share discrete proteomic signatures that are more reflective of their biol. phenotype than cellular heritage, highlighting that a common set of enzymes may support the progression of tumors from a variety of origins and thus represent attractive targets for the diagnosis and treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Ym_lV8DsLrVg90H21EOLACvtfcHk0liTSgjanUJY_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmt1Ciurw%253D&md5=2c8cc24fc9ef2c84e8b0839eba33b3b8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.162187599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.162187599%26sid%3Dliteratum%253Aachs%26aulast%3DJessani%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHumphrey%26aufirst%3DM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DEnzyme%2520activity%2520profiles%2520of%2520the%2520secreted%2520and%2520membrane%2520proteome%2520that%2520depict%2520cancer%2520cell%2520invasiveness%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D10335%26epage%3D10340%26doi%3D10.1073%2Fpnas.162187599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">The pharmacological landscape and therapeutic potential of serine hydrolases</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1038/nrd3620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1038%2Fnrd3620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=22212679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=52-68&author=D.+A.+Bachovchinauthor=B.+F.+Cravatt&title=The+pharmacological+landscape+and+therapeutic+potential+of+serine+hydrolases&doi=10.1038%2Fnrd3620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The pharmacological landscape and therapeutic potential of serine hydrolases</span></div><div class="casAuthors">Bachovchin, Daniel A.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-68</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Serine hydrolases perform crucial roles in many biol. processes, and several of these enzymes are targets of approved drugs for indications such as type 2 diabetes, Alzheimer's disease and infectious diseases.  Despite this, most of the human serine hydrolases (of which there are more than 200) remain poorly characterized with respect to their physiol. substrates and functions, and the vast majority lack selective, in vivo-active inhibitors.  Here, we review the current state of pharmacol. for mammalian serine hydrolases, including marketed drugs, compds. that are under clin. investigation and selective inhibitors emerging from academic probe development efforts.  We also highlight recent methodol. advances that have accelerated the rate of inhibitor discovery and optimization for serine hydrolases, which we anticipate will aid in their biol. characterization and, in some cases, therapeutic validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsWC4iV3Xc57Vg90H21EOLACvtfcHk0ljJUAJDt6K2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOktA%253D%253D&md5=4fde1e23117ad11848a9c2eef85376b7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd3620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3620%26sid%3Dliteratum%253Aachs%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DThe%2520pharmacological%2520landscape%2520and%2520therapeutic%2520potential%2520of%2520serine%2520hydrolases%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D52%26epage%3D68%26doi%3D10.1038%2Fnrd3620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savinainen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saario, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, J. T.</span></span> <span> </span><span class="NLM_article-title">The serine hydrolases MAGL, ABHD6 and ABHD12 as guardians of 2-arachidonoylglycerol signalling through cannabinoid receptors</span>. <i>Acta Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>204</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1111/j.1748-1716.2011.02280.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1111%2Fj.1748-1716.2011.02280.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21418147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1eiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=204&publication_year=2012&pages=267-276&author=J.+R.+Savinainenauthor=S.+M.+Saarioauthor=J.+T.+Laitinen&title=The+serine+hydrolases+MAGL%2C+ABHD6+and+ABHD12+as+guardians+of+2-arachidonoylglycerol+signalling+through+cannabinoid+receptors&doi=10.1111%2Fj.1748-1716.2011.02280.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The serine hydrolases MAGL, ABHD6 and ABHD12 as guardians of 2-arachidonoylglycerol signalling through cannabinoid receptors</span></div><div class="casAuthors">Savinainen, J. R.; Saario, S. M.; Laitinen, J. T.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Physiologica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">204</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-276</span>CODEN:
                <span class="NLM_cas:coden">APCHC4</span>;
        ISSN:<span class="NLM_cas:issn">1748-1708</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The endocannabinoid 2-arachidonoylglycerol (2-AG) is a lipid mediator involved in various physiol. processes.  In response to neural activity, 2-AG is synthesized post-synaptically, then activates pre-synaptic cannabinoid CB1 receptors (CB1Rs) in a retrograde manner, resulting in transient and long-lasting redn. of neurotransmitter release.  The signalling competence of 2-AG is tightly regulated by the balanced action between "on demand" biosynthesis and degrdn.  We review recent research on monoacylglycerol lipase (MAGL), ABHD6 and ABHD12, three serine hydrolases that together account for approx. 99% of brain 2-AG hydrolase activity.  MAGL is responsible for approx. 85% of 2-AG hydrolysis and colocalizes with CB1R in axon terminals.  It is therefore ideally positioned to terminate 2-AG-CB1R signalling regardless of the source of this endocannabinoid.  Its acute pharmacol. inhibition leads to 2-AG accumulation and CB1R-mediated behavioral responses.  Chronic MAGL inactivation results in 2-AG overload, desensitization of CB1R signalling and behavioral tolerance.  ABHD6 accounts for approx. 4% of brain 2-AG hydrolase activity but in neurons it rivals MAGL in efficacy.  Neuronal ABHD6 resides post-synaptically, often juxtaposed with CB1Rs, and its acute inhibition leads to activity-dependent accumulation of 2-AG.  In cortical slices, selective ABHD6 blockade facilitates CB1R-dependent long-term synaptic depression.  ABHD6 is therefore positioned to guard intracellular pools of 2-AG at the site of generation.  ABHD12 is highly expressed in microglia and accounts for approx. 9% of total brain 2-AG hydrolysis.  Mutations in ABHD12 gene are causally linked to a neurodegenerative disease called PHARC.  Whether ABHD12 qualifies as a bona fide member to the endocannabinoid system remains to be established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIBgLJavyxUbVg90H21EOLACvtfcHk0ljJUAJDt6K2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1eiuro%253D&md5=399f8c1e38ae5fd12886573245bec0ba</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2Fj.1748-1716.2011.02280.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1748-1716.2011.02280.x%26sid%3Dliteratum%253Aachs%26aulast%3DSavinainen%26aufirst%3DJ.%2BR.%26aulast%3DSaario%26aufirst%3DS.%2BM.%26aulast%3DLaitinen%26aufirst%3DJ.%2BT.%26atitle%3DThe%2520serine%2520hydrolases%2520MAGL%252C%2520ABHD6%2520and%2520ABHD12%2520as%2520guardians%2520of%25202-arachidonoylglycerol%2520signalling%2520through%2520cannabinoid%2520receptors%26jtitle%3DActa%2520Physiol.%26date%3D2012%26volume%3D204%26spage%3D267%26epage%3D276%26doi%3D10.1111%2Fj.1748-1716.2011.02280.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span> <span> </span><span class="NLM_article-title">Acyl protein thioesterase inhibitors as probes of dynamic S-palmitoylation</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1039/c3md00333g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1039%2FC3MD00333G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=25558349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=268-276&author=D.+Davdaauthor=B.+R.+Martin&title=Acyl+protein+thioesterase+inhibitors+as+probes+of+dynamic+S-palmitoylation&doi=10.1039%2Fc3md00333g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Acyl protein thioesterase inhibitors as probes of dynamic S-palmitoylation</span></div><div class="casAuthors">Davda, Dahvid; Martin, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">268-276</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Protein palmitoylation describes the hydrophobic post-translational modification of cysteine residues in certain proteins, and is required for the spatial organization and compn. of cellular membrane environments.  Certain palmitoylated proteins are processed by acyl protein thioesterase (APT) enzymes, which catalyze thioester hydrolysis of palmitoylated cysteine residues.  Inhibiting APT enzymes disrupts Ras trafficking and attenuates oncogenic growth signaling, highlighting these enzymes as potential therapeutic targets.  As members of the serine hydrolase enzyme family, APT enzymes can be assayed by fluorophosphonate activity-based protein profiling (ABPP) methods, allowing rapid profiling of inhibitor selectivity and potency.  In this review, the authors discuss recent progress in the development of potent and selective inhibitors to APT enzymes, including both reversible and irreversible inhibitor chemotypes.  These examples highlight how ABPP methods integrate with medicinal chem. for the discovery and optimization of inhibitors in complex proteomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFb0Q1vE52H7Vg90H21EOLACvtfcHk0ljJUAJDt6K2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymurY%253D&md5=782483184bf879838c70267583862374</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2FC3MD00333G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00333G%26sid%3Dliteratum%253Aachs%26aulast%3DDavda%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26atitle%3DAcyl%2520protein%2520thioesterase%2520inhibitors%2520as%2520probes%2520of%2520dynamic%2520S-palmitoylation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D268%26epage%3D276%26doi%3D10.1039%2Fc3md00333g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sontag, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunbhakdi-Craig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitterhuber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogris, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontag, E.</span></span> <span> </span><span class="NLM_article-title">Regulation of protein phosphatase 2A methylation by LCMT1 and PME-1 plays a critical role in differentiation of neuroblastoma cells</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">1455</span>– <span class="NLM_lpage">1465</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2010.07049.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1111%2Fj.1471-4159.2010.07049.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21044074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1aiurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2010&pages=1455-1465&author=J.-M.+Sontagauthor=V.+Nunbhakdi-Craigauthor=M.+Mitterhuberauthor=E.+Ogrisauthor=E.+Sontag&title=Regulation+of+protein+phosphatase+2A+methylation+by+LCMT1+and+PME-1+plays+a+critical+role+in+differentiation+of+neuroblastoma+cells&doi=10.1111%2Fj.1471-4159.2010.07049.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of protein phosphatase 2A methylation by LCMT1 and PME-1 plays a critical role in differentiation of neuroblastoma cells</span></div><div class="casAuthors">Sontag, Jean-Marie; Nunbhakdi-Craig, Viyada; Mitterhuber, Martina; Ogris, Egon; Sontag, Estelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1455-1465</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Neuritic alterations are a major feature of many neurodegenerative disorders.  Methylation of protein phosphatase 2A (PP2A) catalytic C subunit by the leucine carboxyl methyltransferase (LCMT1), and demethylation by the protein phosphatase methylesterase 1, is a crit. PP2A regulatory mechanism.  It modulates the formation of PP2A holoenzymes contg. the Bα subunit, which dephosphorylate key neuronal cytoskeletal proteins, including tau.  Significantly, we have reported that LCMT1, methylated C and Bα expression levels are down-regulated in Alzheimer disease-affected brain regions.  In this study, we show that enhanced expression of LCMT1 in cultured N2a neuroblastoma cells, which increases endogenous methylated C and Bα levels, induces changes in E-actin organization.  It promotes serum-independent neuritogenesis and development of extended tau-pos. processes upon N2a cell differentiation.  These stimulatory effects can be abrogated by LCMT1 knockdown and S-adenosylhomocysteine, an inhibitor of methylation reactions.  Expression of protein phosphatase methylesterase 1 and the methylation-site L309Δ C subunit mutant, which decrease intracellular methylated C and Bα levels, block N2a cell differentiation and LCMT1-mediated neurite formation.  Lastly, inducible and non-inducible knockdown of Bα in N2a cells inhibit process out-growth.  Altogether,these results establish a novel mechanistic link between PP2A methylation and development of neurite-like processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPi_pU7tjl97Vg90H21EOLACvtfcHk0ljJUAJDt6K2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1aiurrI&md5=d3fcf045dc77ab4e01f85d908a6a51b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2010.07049.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2010.07049.x%26sid%3Dliteratum%253Aachs%26aulast%3DSontag%26aufirst%3DJ.-M.%26aulast%3DNunbhakdi-Craig%26aufirst%3DV.%26aulast%3DMitterhuber%26aufirst%3DM.%26aulast%3DOgris%26aufirst%3DE.%26aulast%3DSontag%26aufirst%3DE.%26atitle%3DRegulation%2520of%2520protein%2520phosphatase%25202A%2520methylation%2520by%2520LCMT1%2520and%2520PME-1%2520plays%2520a%2520critical%2520role%2520in%2520differentiation%2520of%2520neuroblastoma%2520cells%26jtitle%3DJ.%2520Neurochem.%26date%3D2010%26volume%3D115%26spage%3D1455%26epage%3D1465%26doi%3D10.1111%2Fj.1471-4159.2010.07049.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blankman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Superfamily-wide portrait of serine hydrolase inhibition achieved by library-versus-library screening</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">20941</span>– <span class="NLM_lpage">20946</span>, <span class="refDoi"> DOI: 10.1073/pnas.1011663107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1073%2Fpnas.1011663107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21084632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFyrsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=20941-20946&author=D.+A.+Bachovchinauthor=T.+Jiauthor=W.+Liauthor=G.+M.+Simonauthor=J.+L.+Blankmanauthor=A.+Adibekianauthor=H.+Hooverauthor=S.+Niessenauthor=B.+F.+Cravatt&title=Superfamily-wide+portrait+of+serine+hydrolase+inhibition+achieved+by+library-versus-library+screening&doi=10.1073%2Fpnas.1011663107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Superfamily-wide portrait of serine hydrolase inhibition achieved by library-versus-library screening</span></div><div class="casAuthors">Bachovchin, Daniel A.; Ji, Tianyang; Li, Weiwei; Simon, Gabriel M.; Blankman, Jacqueline L.; Adibekian, Alexander; Hoover, Heather; Niessen, Sherry; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">20941-20946, S20941/1-S20941/172</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Serine hydrolases (SHs) are-one of the largest and most diverse enzyme classes in mammals.  They play fundamental roles in virtually all physiol. processes and are targeted by drugs to treat diseases such as diabetes, obesity, and neurodegenerative disorders.  Despite this, we lack biol. understanding for most of the 110+ predicted mammalian metabolic SHs, in large part because of a dearth of assays to assess their biochem. activities and a lack of selective inhibitors to probe their function in living systems.  We show here that the vast majority (>80%) of mammalian metabolic SHs can be labeled in proteomes by a single, active site-directed fluorophosphonate probe.  We exploit this universal activity-based assay in a library-vs.-library format to screen 70+ SHs against 140+ structurally diverse carbamates.  Lead inhibitors were discovered for ∼40% of the screened enzymes, including many poorly characterized SHs: Global profiles identified carbamate inhibitors that discriminate among highly sequence-related SHs and, conversely, enzymes that share inhibitor sensitivity profiles despite lacking sequence homol.  These findings indicate that sequence relatedness is not a strong predictor of shared pharmaol. within the SH superfamily.  Finally, we show that lead carbamate inhibitors can be optimized into pharmacol. probes that inactivate individual SHs with high specificity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonb1W0UOt-7rVg90H21EOLACvtfcHk0ljAPiqli64ClQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFyrsbnO&md5=18e26c50a5f005089fcf5763ff9edab8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1011663107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1011663107%26sid%3Dliteratum%253Aachs%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DJi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DBlankman%26aufirst%3DJ.%2BL.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DHoover%26aufirst%3DH.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DSuperfamily-wide%2520portrait%2520of%2520serine%2520hydrolase%2520inhibition%2520achieved%2520by%2520library-versus-library%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D20941%26epage%3D20946%26doi%3D10.1073%2Fpnas.1011663107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhouayek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minamimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrechen, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasdev, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S. H.</span></span> <span> </span><span class="NLM_article-title">In Vitro and in Vivo Evaluation of (11)C-Labeled Azetidinecarboxylates for Imaging Monoacylglycerol Lipase by PET Imaging Studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2278</span>– <span class="NLM_lpage">2291</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01400</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01400" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtlOmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2278-2291&author=R.+Chengauthor=W.+Moriauthor=L.+Maauthor=M.+Alhouayekauthor=A.+Hatoriauthor=Y.+Zhangauthor=D.+Ogasawaraauthor=G.+Yuanauthor=Z.+Chenauthor=X.+Zhangauthor=H.+Shiauthor=T.+Yamasakiauthor=L.+Xieauthor=K.+Kumataauthor=M.+Fujinagaauthor=Y.+Nagaiauthor=T.+Minamimotoauthor=M.+Svenssonauthor=L.+Wangauthor=Y.+Duauthor=M.+J.+Ondrechenauthor=N.+Vasdevauthor=B.+F.+Cravattauthor=C.+Fowlerauthor=M.-R.+Zhangauthor=S.+H.+Liang&title=In+Vitro+and+in+Vivo+Evaluation+of+%2811%29C-Labeled+Azetidinecarboxylates+for+Imaging+Monoacylglycerol+Lipase+by+PET+Imaging+Studies&doi=10.1021%2Facs.jmedchem.7b01400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro and in Vivo Evaluation of 11C-Labeled Azetidinecarboxylates for Imaging Monoacylglycerol Lipase by PET Imaging Studies</span></div><div class="casAuthors">Cheng, Ran; Mori, Wakana; Ma, Longle; Alhouayek, Mireille; Hatori, Akiko; Zhang, Yiding; Ogasawara, Daisuke; Yuan, Gengyang; Chen, Zhen; Zhang, Xiaofei; Shi, Hang; Yamasaki, Tomoteru; Xie, Lin; Kumata, Katsushi; Fujinaga, Masayuki; Nagai, Yuji; Minamimoto, Takafumi; Svensson, Mona; Wang, Lu; Du, Yunfei; Ondrechen, Mary Jo; Vasdev, Neil; Cravatt, Benjamin F.; Fowler, Christopher; Zhang, Ming-Rong; Liang, Steven H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2278-2291</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) is the principle enzyme for metabolizing endogenous cannabinoid ligand 2-arachidonoyglycerol (2-AG).  Blockade of MAGL increases 2-AG levels, resulting in subsequent activation of the endocannabinoid system, and has emerged as a novel therapeutic strategy to treat drug addiction, inflammation, and neurodegenerative diseases.  Herein we report a new series of MAGL inhibitors, which were radiolabeled by site-specific labeling technologies, including 11C-carbonylation and spirocyclic iodonium ylide (SCIDY) radiofluorination.  The lead compd. [11C]10 (MAGL-0519) demonstrated high specific binding and selectivity in vitro and in vivo.  We also obsd. unexpected washout kinetics with these irreversible radiotracers, in which in vivo evidence for turnover of the covalent residue was unveiled between MAGL and azetidine carboxylates.  This work may lead to new directions for drug discovery and PET tracer development based on azetidine carboxylate inhibitor scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm2LL5zxwjx7Vg90H21EOLACvtfcHk0ljAPiqli64ClQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtlOmtb8%253D&md5=b2b4ddcfcccadac13be8d731127df7db</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01400%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%26aulast%3DMori%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAlhouayek%26aufirst%3DM.%26aulast%3DHatori%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DYamasaki%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DKumata%26aufirst%3DK.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DMinamimoto%26aufirst%3DT.%26aulast%3DSvensson%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DOndrechen%26aufirst%3DM.%2BJ.%26aulast%3DVasdev%26aufirst%3DN.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DFowler%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DM.-R.%26aulast%3DLiang%26aufirst%3DS.%2BH.%26atitle%3DIn%2520Vitro%2520and%2520in%2520Vivo%2520Evaluation%2520of%2520%252811%2529C-Labeled%2520Azetidinecarboxylates%2520for%2520Imaging%2520Monoacylglycerol%2520Lipase%2520by%2520PET%2520Imaging%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2278%26epage%3D2291%26doi%3D10.1021%2Facs.jmedchem.7b01400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafroth, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giffenig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papisov, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collier, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josephson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S. H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Evaluation of (18)F-Labeled Monoacylglycerol Lipase Inhibitors as Novel Positron Emission Tomography Probes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8866</span>– <span class="NLM_lpage">8872</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00936</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00936" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslKis7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8866-8872&author=Z.+Chenauthor=W.+Moriauthor=H.+Fuauthor=M.+A.+Schafrothauthor=A.+Hatoriauthor=T.+Shaoauthor=G.+Zhangauthor=R.+S.+Vanauthor=Y.+Zhangauthor=K.+Huauthor=M.+Fujinagaauthor=L.+Wangauthor=V.+Belovauthor=D.+Ogasawaraauthor=P.+Giffenigauthor=X.+Dengauthor=J.+Rongauthor=Q.+Yuauthor=X.+Zhangauthor=M.+I.+Papisovauthor=Y.+Shaoauthor=T.+L.+Collierauthor=J.-A.+Maauthor=B.+F.+Cravattauthor=L.+Josephsonauthor=M.-R.+Zhangauthor=S.+H.+Liang&title=Design%2C+Synthesis%2C+and+Evaluation+of+%2818%29F-Labeled+Monoacylglycerol+Lipase+Inhibitors+as+Novel+Positron+Emission+Tomography+Probes&doi=10.1021%2Facs.jmedchem.9b00936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of 18F-Labeled Monoacylglycerol Lipase Inhibitors as Novel Positron Emission Tomography Probes</span></div><div class="casAuthors">Chen, Zhen; Mori, Wakana; Fu, Hualong; Schafroth, Michael A.; Hatori, Akiko; Shao, Tuo; Zhang, Genwei; Van, Richard S.; Zhang, Yiding; Hu, Kuan; Fujinaga, Masayuki; Wang, Lu; Belov, Vasily; Ogasawara, Daisuke; Giffenig, Pilar; Deng, Xiaoyun; Rong, Jian; Yu, Qingzhen; Zhang, Xiaofei; Papisov, Mikhail I.; Shao, Yihan; Collier, Thomas L.; Ma, Jun-An; Cravatt, Benjamin F.; Josephson, Lee; Zhang, Ming-Rong; Liang, Steven H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8866-8872</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dysfunction of monoacylglycerol lipase (MAGL) is assocd. with several psychopathol. disorders, including drug addiction and neurodegenerative diseases.  Herein we design, synthesize, and evaluate several irreversible fluorine-contg. MAGL inhibitors for positron emission tomog. (PET) ligand development.  Compd. 6 (identified from a therapeutic agent) was advanced for 18F-labeling via a novel spirocyclic iodonium ylide (SCIDY) strategy, which demonstrated high brain permeability and excellent specific binding.  This work supports further development of novel 18F-labeled MAGL PET probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ9noubfh-w7Vg90H21EOLACvtfcHk0lhbrz7QNp5HNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslKis7bF&md5=be7433643e89a96579a7be18198b5546</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00936%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMori%26aufirst%3DW.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DSchafroth%26aufirst%3DM.%2BA.%26aulast%3DHatori%26aufirst%3DA.%26aulast%3DShao%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DVan%26aufirst%3DR.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DK.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBelov%26aufirst%3DV.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DGiffenig%26aufirst%3DP.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DRong%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DPapisov%26aufirst%3DM.%2BI.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DCollier%26aufirst%3DT.%2BL.%26aulast%3DMa%26aufirst%3DJ.-A.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DJosephson%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DM.-R.%26aulast%3DLiang%26aufirst%3DS.%2BH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Evaluation%2520of%2520%252818%2529F-Labeled%2520Monoacylglycerol%2520Lipase%2520Inhibitors%2520as%2520Novel%2520Positron%2520Emission%2520Tomography%2520Probes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8866%26epage%3D8872%26doi%3D10.1021%2Facs.jmedchem.9b00936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">An activity-based imaging probe for the integral membrane hydrolase KIAA1363</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">966</span>– <span class="NLM_lpage">970</span>, <span class="refDoi"> DOI: 10.1002/anie.201107236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fanie.201107236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsF2lur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=966-970&author=J.+W.+Changauthor=R.+E.+Moelleringauthor=B.+F.+Cravatt&title=An+activity-based+imaging+probe+for+the+integral+membrane+hydrolase+KIAA1363&doi=10.1002%2Fanie.201107236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">An Activity-Based Imaging Probe for the Integral Membrane Hydrolase KIAA1363</span></div><div class="casAuthors">Chang, Jae Won; Moellering, Raymond E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">966-970, S966/1-S966/4</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">, The authors report herein a versatile fluorescent probe for visualizing KIAA1363 activity in cancer cells and in vivo.  This agent JW576 is potent (sub-μM IC50 values in cells) and selective for KIAA1363 and was used (1) in imaging expts. to det. that KIAA1363 is largely, but not exclusively localized to the ER in cancer cells, and (2) in pulse-chase expts. to est. a half-life of 14-23 h for KIAA1363 in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro5_GtuVmlf7Vg90H21EOLACvtfcHk0lhbrz7QNp5HNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsF2lur7F&md5=1f023c2ba887da6397b6613e02ae16ea</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107236%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DAn%2520activity-based%2520imaging%2520probe%2520for%2520the%2520integral%2520membrane%2520hydrolase%2520KIAA1363%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D966%26epage%3D970%26doi%3D10.1002%2Fanie.201107236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohsiek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Nieuwendijk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van
den Berg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Quenched Activity-based Probes for Diacylglycerol Lipase and alpha,beta-Hydrolase Domain Containing Protein 6</span>. <i>Chem.—Asian J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3491</span>– <span class="NLM_lpage">3500</span>, <span class="refDoi"> DOI: 10.1002/asia.201800452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fasia.201800452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=29901868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3491-3500&author=E.+J.+van%0ARoodenauthor=M.+Kohsiekauthor=R.+Kreekelauthor=A.+C.+M.+van+Esbroeckauthor=A.+van+den+Nieuwendijkauthor=A.+P.+A.+Janssenauthor=R.+van%0Aden+Bergauthor=H.+S.+Overkleeftauthor=M.+van+der+Stelt&title=Design+and+Synthesis+of+Quenched+Activity-based+Probes+for+Diacylglycerol+Lipase+and+alpha%2Cbeta-Hydrolase+Domain+Containing+Protein+6&doi=10.1002%2Fasia.201800452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Quenched Activity-based Probes for Diacylglycerol Lipase and α,β-Hydrolase Domain Containing Protein 6</span></div><div class="casAuthors">van Rooden, E. J.; Kohsiek, M.; Kreekel, R.; van Esbroeck, A. C. M.; van den Nieuwendijk, A. M. C. H.; Janssen, A. P. A.; van den Berg, R. J. B. H. N.; Overkleeft, H. S.; van der Stelt, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - An Asian Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3491-3500</span>CODEN:
                <span class="NLM_cas:coden">CAAJBI</span>;
        ISSN:<span class="NLM_cas:issn">1861-4728</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Diacylglycerol lipases (DAGL) are responsible for the biosynthesis of the endocannabinoid 2-arachidonoylglycerol.  The fluorescent activity-based probes (ABPs) DH379 and HT-01 have been previously shown to label DAGLs and to cross-react with the serine hydrolase ABHD6.  Here, we report the synthesis and characterization of two new quenched activity-based probes (qABPs) 1 and 2, the design of which was based on the structures of DH379 and HT-01, resp.  Probe 1 contains a BODIPY-FL and a 2,4-dinitroaniline moiety as a fluorophore-quencher pair, whereas probe 2 employs a Cy5-fluorophore and a cAB40-quencher.  The fluorescence of both probes was quenched with relative quantum yields of 0.34 and 0.0081, resp.  The probes showed target inhibition as characterized in activity-based protein profiling assays using human cell- and mouse brain lysates, but were unfortunately not active in living cells, presumably due to limited cell permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7O8u5-q5bXrVg90H21EOLACvtfcHk0lhbrz7QNp5HNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemurzN&md5=cc33901cc7c37c68a51f71223dc08b26</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fasia.201800452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fasia.201800452%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3DKohsiek%26aufirst%3DM.%26aulast%3DKreekel%26aufirst%3DR.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dvan%2Bden%2BNieuwendijk%26aufirst%3DA.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3Dvan%2Bden%2BBerg%26aufirst%3DR.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Quenched%2520Activity-based%2520Probes%2520for%2520Diacylglycerol%2520Lipase%2520and%2520alpha%252Cbeta-Hydrolase%2520Domain%2520Containing%2520Protein%25206%26jtitle%3DChem.%25E2%2580%2594Asian%2520J.%26date%3D2018%26volume%3D13%26spage%3D3491%26epage%3D3500%26doi%3D10.1002%2Fasia.201800452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Dulk, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Berg, R. J. B. H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Codée, J. D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Two-step activity-based protein profiling of diacylglycerol lipase</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">5250</span>– <span class="NLM_lpage">5253</span>, <span class="refDoi"> DOI: 10.1039/c8ob01499j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1039%2FC8OB01499J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=30004552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlamsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=5250-5253&author=E.+J.+van%0ARoodenauthor=R.+Kreekelauthor=T.+Hansenauthor=A.+P.+A.+Janssenauthor=A.+C.+M.+van+Esbroeckauthor=H.+den+Dulkauthor=R.+J.+B.+H.+N.+van+den%0ABergauthor=J.+D.+C.+Cod%C3%A9eauthor=M.+van+der+Stelt&title=Two-step+activity-based+protein+profiling+of+diacylglycerol+lipase&doi=10.1039%2Fc8ob01499j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Two-step activity-based protein profiling of diacylglycerol lipase</span></div><div class="casAuthors">van Rooden, Eva J.; Kreekel, Roy; Hansen, Thomas; Janssen, Antonius P. A.; van Esbroeck, Annelot C. M.; den Dulk, Hans; van den Berg, Richard J. B. H. N.; Codee, Jeroen D. C.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">5250-5253</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Diacylglycerol lipases (DAGL) produce the endocannabinoid 2-arachidonoylglycerol, a key modulator of neurotransmitter release.  Chem. tools that visualize endogenous DAGL activity are desired.  Here, we report the design, synthesis and application of a triazole urea probe for DAGL equipped with a norbornene as a biorthogonal handle.  The activity and selectivity of the probe was assessed with activity-based protein profiling.  This probe was potent against endogenous DAGLα (IC50 = 5 nM) and it was successfully applied as a two-step activity-based probe for labeling of DAGLα using an inverse electron-demand Diels-Alder ligation in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0ykcj8l_rsbVg90H21EOLACvtfcHk0ljM5VpFazbeDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlamsrzF&md5=8768f003f256f2731f80c645e51b72ae</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1039%2FC8OB01499J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB01499J%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3DKreekel%26aufirst%3DR.%26aulast%3DHansen%26aufirst%3DT.%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dden%2BDulk%26aufirst%3DH.%26aulast%3Dvan%2Bden%2BBerg%26aufirst%3DR.%2BJ.%2BB.%2BH.%2BN.%26aulast%3DCod%25C3%25A9e%26aufirst%3DJ.%2BD.%2BC.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DTwo-step%2520activity-based%2520protein%2520profiling%2520of%2520diacylglycerol%2520lipase%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D5250%26epage%3D5253%26doi%3D10.1039%2Fc8ob01499j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niphakis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide reactivity profiling identifies diverse carbamate chemotypes tuned for serine hydrolase inhibition</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1590</span>– <span class="NLM_lpage">1599</span>, <span class="refDoi"> DOI: 10.1021/cb400261h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400261h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3sXot1Sru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1590-1599&author=J.+W.+Changauthor=A.+B.+Cognettaauthor=M.+J.+Niphakisauthor=B.+F.+Cravatt&title=Proteome-wide+reactivity+profiling+identifies+diverse+carbamate+chemotypes+tuned+for+serine+hydrolase+inhibition&doi=10.1021%2Fcb400261h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-Wide Reactivity Profiling Identifies Diverse Carbamate Chemotypes Tuned for Serine Hydrolase Inhibition</span></div><div class="casAuthors">Chang, Jae Won; Cognetta, Armand B., III; Niphakis, Micah J.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1590-1599</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Serine hydrolases are one of the largest and most diverse enzyme classes in Nature.  Inhibitors of serine hydrolases are used to treat many diseases, including obesity, diabetes, cognitive dementia, and bacterial and viral infections.  Nonetheless, the majority of the 200+ serine hydrolases in mammals still lack selective inhibitors for their functional characterization.  We and others have shown that activated carbamates, through covalent reaction with the conserved serine nucleophile of serine hydrolases, can serve as useful inhibitors for members of this enzyme family.  The extent to which carbamates, however, cross-react with other protein classes remains mostly unexplored.  Here, we address this problem by investigating the proteome-wide reactivity of a diverse set of activated carbamates in vitro and in vivo, using a combination of competitive and click chem. (CC)-activity-based protein profiling (ABPP).  We identify multiple classes of carbamates, including O-aryl, O-hexafluoroisopropyl (HFIP), and O-N-hydroxysuccinimidyl (NHS) carbamates that react selectively with serine hydrolases across entire mouse tissue proteomes in vivo.  We exploit the proteome-wide specificity of HFIP carbamates to create in situ imaging probes for the endocannabinoid hydrolases monoacylglycerol lipase (MAGL) and α-β hydrolase-6 (ABHD6).  These findings, taken together, designate the carbamate as a privileged reactive group for serine hydrolases that can accommodate diverse structural modifications to produce inhibitors that display exceptional potency and selectivity across the mammalian proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE-176v-DnJrVg90H21EOLACvtfcHk0ljM5VpFazbeDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXot1Sru7k%253D&md5=7286011459791bcdea13f918b9ef97c3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fcb400261h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400261h%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DNiphakis%26aufirst%3DM.%2BJ.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520reactivity%2520profiling%2520identifies%2520diverse%2520carbamate%2520chemotypes%2520tuned%2520for%2520serine%2520hydrolase%2520inhibition%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1590%26epage%3D1599%26doi%3D10.1021%2Fcb400261h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haedke, U. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frommel, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhelst, S. H. L.</span></span> <span> </span><span class="NLM_article-title">Phosphoramidates as novel activity-based probes for serine proteases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1106</span>– <span class="NLM_lpage">1110</span>, <span class="refDoi"> DOI: 10.1002/cbic.201400013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fcbic.201400013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=24817682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslWku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1106-1110&author=U.+R.+Haedkeauthor=S.+C.+Frommelauthor=F.+Hansenauthor=H.+Hahneauthor=B.+Kusterauthor=M.+Bogyoauthor=S.+H.+L.+Verhelst&title=Phosphoramidates+as+novel+activity-based+probes+for+serine+proteases&doi=10.1002%2Fcbic.201400013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoramidates as Novel Activity-Based Probes for Serine Proteases</span></div><div class="casAuthors">Haedke, Ute R.; Frommel, Sandra C.; Hansen, Fabian; Hahne, Hannes; Kuster, Bernhard; Bogyo, Matthew; Verhelst, Steven H. L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1106-1110</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Activity-based probes (ABPs) are small mols. that exclusively form covalent bonds with catalytically active enzymes.  In the last decade, they have esp. been used in functional proteomics studies of proteases.  Here, we present phosphoramidate peptides as a novel type of ABP for serine proteases.  These mols. can be made in a straightforward manner by std. Fmoc-based solid-phase peptide synthesis, allowing rapid diversification.  The resulting ABPs covalently bind different serine proteases, depending on the amino acid recognition element adjacent to the reactive group.  A reporter tag enables downstream gel-based anal. or LC-MS/MS-mediated identification of the targeted proteases.  Overall, we believe that these readily accessible probes will provide new avenues for the functional study of serine proteases in complex proteomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxMEsjjMpPVLVg90H21EOLACvtfcHk0ljM5VpFazbeDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslWku7o%253D&md5=91c72566a3d9aeabd8949f56943e5282</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201400013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201400013%26sid%3Dliteratum%253Aachs%26aulast%3DHaedke%26aufirst%3DU.%2BR.%26aulast%3DFrommel%26aufirst%3DS.%2BC.%26aulast%3DHansen%26aufirst%3DF.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DBogyo%26aufirst%3DM.%26aulast%3DVerhelst%26aufirst%3DS.%2BH.%2BL.%26atitle%3DPhosphoramidates%2520as%2520novel%2520activity-based%2520probes%2520for%2520serine%2520proteases%26jtitle%3DChemBioChem%26date%3D2014%26volume%3D15%26spage%3D1106%26epage%3D1110%26doi%3D10.1002%2Fcbic.201400013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otrubova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Discovery libraries targeting the major enzyme classes: the serine hydrolases</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3807</span>– <span class="NLM_lpage">3813</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.bmcl.2014.06.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=25037918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOjsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3807-3813&author=K.+Otrubovaauthor=V.+Srinivasanauthor=D.+L.+Boger&title=Discovery+libraries+targeting+the+major+enzyme+classes%3A+the+serine+hydrolases&doi=10.1016%2Fj.bmcl.2014.06.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery libraries targeting the major enzyme classes: The serine hydrolases</span></div><div class="casAuthors">Otrubova, Katerina; Srinivasan, Venkat; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3807-3813</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two libraries of modestly reactive ureas contg. either electron-deficient acyl anilines or acyl pyrazoles were prepd. and are reported as screening libraries for candidate serine hydrolase inhibitors.  Within each library is a small but powerful subset of compds. that serve as a chemotype fragment screening library capable of subsequent structural diversification.  Elaboration of the pyrazole-based ureas provided remarkably potent irreversible inhibitors of fatty acid amide hydrolase (FAAH, apparent Ki = 100-200 pM) complementary to those previously disclosed enlisting electron-deficient aniline-based ureas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0mdvToLulcrVg90H21EOLACvtfcHk0ljM5VpFazbeDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOjsL%252FK&md5=2dbb15e7145fadbeda928b1b0eba4560</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.063%26sid%3Dliteratum%253Aachs%26aulast%3DOtrubova%26aufirst%3DK.%26aulast%3DSrinivasan%26aufirst%3DV.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DDiscovery%2520libraries%2520targeting%2520the%2520major%2520enzyme%2520classes%253A%2520the%2520serine%2520hydrolases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3807%26epage%3D3813%26doi%3D10.1016%2Fj.bmcl.2014.06.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Click-generated triazole ureas as ultrapotent in vivo-active serine hydrolase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">478</span>, <span class="refDoi"> DOI: 10.1038/nchembio.579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1038%2Fnchembio.579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=21572424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtV2ht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=469-478&author=A.+Adibekianauthor=B.+R.+Martinauthor=C.+Wangauthor=K.-L.+Hsuauthor=D.+A.+Bachovchinauthor=S.+Niessenauthor=H.+Hooverauthor=B.+F.+Cravatt&title=Click-generated+triazole+ureas+as+ultrapotent+in+vivo-active+serine+hydrolase+inhibitors&doi=10.1038%2Fnchembio.579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Click-generated triazole ureas as ultrapotent in vivo-active serine hydrolase inhibitors</span></div><div class="casAuthors">Adibekian, Alexander; Martin, Brent R.; Wang, Chu; Hsu, Ku-Lung; Bachovchin, Daniel A.; Niessen, Sherry; Hoover, Heather; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">469-478</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Serine hydrolases are a diverse enzyme class representing ∼1% of all human proteins.  The biol. functions of most serine hydrolases remain poorly characterized owing to a lack of selective inhibitors to probe their activity in living systems.  Here we show that a substantial no. of serine hydrolases can be irreversibly inactivated by 1,2,3-triazole ureas, which show negligible cross-reactivity with other protein classes.  Rapid lead optimization by click chem.-enabled synthesis and competitive activity-based profiling identified 1,2,3-triazole ureas that selectively inhibit enzymes from diverse branches of the serine hydrolase class, including peptidases (acyl-peptide hydrolase, or APEH), lipases (platelet-activating factor acetylhydrolase-2, or PAFAH2) and uncharacterized hydrolases (α,β-hydrolase-11, or ABHD11), with exceptional potency in cells (sub-nanomolar) and mice (<1 mg kg-1).  We show that APEH inhibition leads to accumulation of N-acetylated proteins and promotes proliferation in T cells.  These data indicate 1,2,3-triazole ureas are a pharmacol. privileged chemotype for serine hydrolase inhibition, combining broad activity across the serine hydrolase class with tunable selectivity for individual enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_5ZfJ_B1ZrVg90H21EOLACvtfcHk0ljY2a_beMIbDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtV2ht7s%253D&md5=236ff8d9ef723ed09c00cf1c47b6727d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.579%26sid%3Dliteratum%253Aachs%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHsu%26aufirst%3DK.-L.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DHoover%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DClick-generated%2520triazole%2520ureas%2520as%2520ultrapotent%2520in%2520vivo-active%2520serine%2520hydrolase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D469%26epage%3D478%26doi%3D10.1038%2Fnchembio.579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Characterization of tunable piperidine and piperazine carbamates as inhibitors of endocannabinoid hydrolases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1830</span>– <span class="NLM_lpage">1842</span>, <span class="refDoi"> DOI: 10.1021/jm9016976</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9016976" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1830-1842&author=J.+Z.+Longauthor=X.+Jinauthor=A.+Adibekianauthor=W.+Liauthor=B.+F.+Cravatt&title=Characterization+of+tunable+piperidine+and+piperazine+carbamates+as+inhibitors+of+endocannabinoid+hydrolases&doi=10.1021%2Fjm9016976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Tunable Piperidine and Piperazine Carbamates as Inhibitors of Endocannabinoid Hydrolases</span></div><div class="casAuthors">Long, Jonathan Z.; Jin, Xin; Adibekian, Alexander; Li, Weiwei; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1830-1842</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) and fatty acid amide hydrolase (FAAH) are two enzymes from the serine hydrolase superfamily that degrade the endocannabinoids 2-arachidonoylglycerol and anandamide, resp.  We have recently discovered that MAGL and FAAH are both inhibited by carbamates bearing an N-piperidine/piperazine group.  Piperidine/piperazine carbamates show excellent in vivo activity, raising brain endocannabinoid levels and producing CB1-dependent behavioral effects in mice, suggesting that they represent a promising class of inhibitors for studying the endogenous functions of MAGL and FAAH.  Herein, we disclose a full account of the syntheses, structure-activity relationships, and inhibitory activities of piperidine/piperazine carbamates against members of the serine hydrolase family.  These scaffolds can be tuned for MAGL-selective or dual MAGL-FAAH inhibition by the attachment of an appropriately substituted bisarylcarbinol or aryloxybenzyl moiety, resp., on the piperidine/piperazine ring.  Modifications to the piperidine/piperazine ring ablated inhibitory activity, suggesting a strict requirement for a six-membered ring to maintain potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphBYZrLsvC87Vg90H21EOLACvtfcHk0ljY2a_beMIbDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSitLY%253D&md5=0a93fb0dcfb589ce52e0834d63fc0dda</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm9016976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9016976%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DJ.%2BZ.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DCharacterization%2520of%2520tunable%2520piperidine%2520and%2520piperazine%2520carbamates%2520as%2520inhibitors%2520of%2520endocannabinoid%2520hydrolases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1830%26epage%3D1842%26doi%3D10.1021%2Fjm9016976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aaltonen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savinainen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rönkkö, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusisto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korhonen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navia-Paldanius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häyrinen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takabe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Käsnänen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantsar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasonen-Seppänen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevalainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laitinen, J. T.</span></span> <span> </span><span class="NLM_article-title">Piperazine and piperidine triazole ureas as ultrapotent and highly selective inhibitors of monoacylglycerol lipase</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.chembiol.2013.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=23521796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFOlur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=379-390&author=N.+Aaltonenauthor=J.+R.+Savinainenauthor=C.+R.+Ribasauthor=J.+R%C3%B6nkk%C3%B6author=A.+Kuusistoauthor=J.+Korhonenauthor=D.+Navia-Paldaniusauthor=J.+H%C3%A4yrinenauthor=P.+Takabeauthor=H.+K%C3%A4sn%C3%A4nenauthor=T.+Pantsarauthor=T.+Laitinenauthor=M.+Lehtonenauthor=S.+Pasonen-Sepp%C3%A4nenauthor=A.+Posoauthor=T.+Nevalainenauthor=J.+T.+Laitinen&title=Piperazine+and+piperidine+triazole+ureas+as+ultrapotent+and+highly+selective+inhibitors+of+monoacylglycerol+lipase&doi=10.1016%2Fj.chembiol.2013.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Piperazine and Piperidine Triazole Ureas as Ultrapotent and Highly Selective Inhibitors of Monoacylglycerol Lipase</span></div><div class="casAuthors">Aaltonen, Niina; Savinainen, Juha R.; Ribas, Casandra Riera; Ronkko, Jani; Kuusisto, Anne; Korhonen, Jani; Navia-Paldanius, Dina; Hayrinen, Jukka; Takabe, Piia; Kasnanen, Heikki; Pantsar, Tatu; Laitinen, Tuomo; Lehtonen, Marko; Pasonen-Seppanen, Sanna; Poso, Antti; Nevalainen, Tapio; Laitinen, Jarmo T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">379-390</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) terminates the signaling function of the endocannabinoid, 2-arachidonoylglycerol (2-AG).  During 2-AG hydrolysis, MAGL liberates arachidonic acid, feeding the principal substrate for the neuroinflammatory prostaglandins.  In cancer cells, MAGL redirects lipid stores toward protumorigenic signaling lipids.  Thus MAGL inhibitors may have great therapeutic potential.  Although potent and increasingly selective MAGL inhibitors have been described, their no. is still limited.  Here, we have characterized piperazine and piperidine triazole ureas that combine the high potency attributable to the triazole leaving group together with the bulky arom. benzodioxolyl moiety required for selectivity, culminating in compd. JJKK-048 that potently (IC50 < 0.4 nM) inhibited human and rodent MAGL.  JJKK-048 displayed low cross-reactivity with other endocannabinoid targets.  Activity-based protein profiling of mouse brain and human melanoma cell proteomes suggested high specificity also among the metabolic serine hydrolases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIDOxKwD3xf7Vg90H21EOLACvtfcHk0ljY2a_beMIbDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFOlur0%253D&md5=3061a03d341bfb1de6b25c8d925b5b9f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DAaltonen%26aufirst%3DN.%26aulast%3DSavinainen%26aufirst%3DJ.%2BR.%26aulast%3DRibas%26aufirst%3DC.%2BR.%26aulast%3DR%25C3%25B6nkk%25C3%25B6%26aufirst%3DJ.%26aulast%3DKuusisto%26aufirst%3DA.%26aulast%3DKorhonen%26aufirst%3DJ.%26aulast%3DNavia-Paldanius%26aufirst%3DD.%26aulast%3DH%25C3%25A4yrinen%26aufirst%3DJ.%26aulast%3DTakabe%26aufirst%3DP.%26aulast%3DK%25C3%25A4sn%25C3%25A4nen%26aufirst%3DH.%26aulast%3DPantsar%26aufirst%3DT.%26aulast%3DLaitinen%26aufirst%3DT.%26aulast%3DLehtonen%26aufirst%3DM.%26aulast%3DPasonen-Sepp%25C3%25A4nen%26aufirst%3DS.%26aulast%3DPoso%26aufirst%3DA.%26aulast%3DNevalainen%26aufirst%3DT.%26aulast%3DLaitinen%26aufirst%3DJ.%2BT.%26atitle%3DPiperazine%2520and%2520piperidine%2520triazole%2520ureas%2520as%2520ultrapotent%2520and%2520highly%2520selective%2520inhibitors%2520of%2520monoacylglycerol%2520lipase%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D379%26epage%3D390%26doi%3D10.1016%2Fj.chembiol.2013.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malamas, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farah, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerts, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farizatto, K. L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahr, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makriyannis, A.</span></span> <span> </span><span class="NLM_article-title">Piperidine and piperazine inhibitors of fatty acid amide hydrolase targeting excitotoxic pathology</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">115096</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.115096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.bmc.2019.115096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=31629610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVSmu7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=115096&author=M.+Lamaniauthor=M.+S.+Malamasauthor=S.+I.+Farahauthor=V.+G.+Shuklaauthor=M.+F.+Almeidaauthor=C.+M.+Weertsauthor=J.+Andersonauthor=J.+T.+Woodauthor=K.+L.+G.+Farizattoauthor=B.+A.+Bahrauthor=A.+Makriyannis&title=Piperidine+and+piperazine+inhibitors+of+fatty+acid+amide+hydrolase+targeting+excitotoxic+pathology&doi=10.1016%2Fj.bmc.2019.115096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Piperidine and piperazine inhibitors of fatty acid amide hydrolase targeting excitotoxic pathology</span></div><div class="casAuthors">Lamani, Manjunath; Malamas, Michael S.; Farah, Shrouq I.; Shukla, Vidyanand G.; Almeida, Michael F.; Weerts, Catherine M.; Anderson, Joseph; Wood, JodiAnne T.; Farizatto, Karen L. G.; Bahr, Ben A.; Makriyannis, Alexandros</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">115096</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">FAAH inhibitors offer safety advantages by augmenting the anandamide levels "on demand" to promote neuroprotective mechanisms without the adverse psychotropic effects usually seen with direct and chronic activation of the CB1 receptor.  FAAH is an enzyme implicated in the hydrolysis of the endocannabinoid N-arachidonoylethanolamine (AEA), which is a partial agonist of the CB1 receptor.  Herein, we report the discovery of a new series of highly potent and selective carbamate FAAH inhibitors and their evaluation for neuroprotection.  The new inhibitors showed potent nanomolar inhibitory activity against human recombinant and purified rat FAAH, were selective (>1000-fold) against serine hydrolases MGL and ABHD6 and lacked any affinity for the cannabinoid receptors CB1 and CB2.  Evaluation of FAAH inhibitors 9(I) and 31(II) using the in vitro competitive activity-based protein profiling (ABPP) assay confirmed that both inhibitors were highly selective for FAAH in the brain, since none of the other FP-reactive serine hydrolases in this tissue were inhibited by these agents.  Our design strategy followed a traditional SAR approach and was supported by mol. modeling studies based on known FAAH cocrystal structures.  To rationally design new mols. that are irreversibly bound to FAAH, we have constructed "precovalent" FAAH-ligand complexes to identify good binding geometries of the ligands within the binding pocket of FAAH and then calcd. covalent docking poses to select compds. for synthesis.  FAAH inhibitors I and II were evaluated for neuroprotection in rat hippocampal slice cultures.  In the brain tissue, both inhibitors displayed protection against synaptic deterioration produced by kainic acid-induced excitotoxicity.  Thus, the resultant compds. produced through rational design are providing early leads for developing therapeutics against seizure-related damage assocd. with a variety of disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6K8DNdlhP_bVg90H21EOLACvtfcHk0lgTNc6dHrpVXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVSmu7fJ&md5=e61cc308e9f193ee715419635c61be55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.115096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.115096%26sid%3Dliteratum%253Aachs%26aulast%3DLamani%26aufirst%3DM.%26aulast%3DMalamas%26aufirst%3DM.%2BS.%26aulast%3DFarah%26aufirst%3DS.%2BI.%26aulast%3DShukla%26aufirst%3DV.%2BG.%26aulast%3DAlmeida%26aufirst%3DM.%2BF.%26aulast%3DWeerts%26aufirst%3DC.%2BM.%26aulast%3DAnderson%26aufirst%3DJ.%26aulast%3DWood%26aufirst%3DJ.%2BT.%26aulast%3DFarizatto%26aufirst%3DK.%2BL.%2BG.%26aulast%3DBahr%26aufirst%3DB.%2BA.%26aulast%3DMakriyannis%26aufirst%3DA.%26atitle%3DPiperidine%2520and%2520piperazine%2520inhibitors%2520of%2520fatty%2520acid%2520amide%2520hydrolase%2520targeting%2520excitotoxic%2520pathology%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D115096%26doi%3D10.1016%2Fj.bmc.2019.115096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baggelaar, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsicano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaouloff, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Chemical Proteomics Maps Brain Region Specific Activity of Endocannabinoid Hydrolases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">861</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1KlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=852-861&author=M.+P.+Baggelaarauthor=A.+C.+M.+van+Esbroeckauthor=E.+J.+van+Roodenauthor=B.+I.+Floreaauthor=H.+S.+Overkleeftauthor=G.+Marsicanoauthor=F.+Chaouloffauthor=M.+van+der+Stelt&title=Chemical+Proteomics+Maps+Brain+Region+Specific+Activity+of+Endocannabinoid+Hydrolases&doi=10.1021%2Facschembio.6b01052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomics Maps Brain Region Specific Activity of Endocannabinoid Hydrolases</span></div><div class="casAuthors">Baggelaar, Marc P.; van Esbroeck, Annelot C. M.; van Rooden, Eva J.; Florea, Bogdan I.; Overkleeft, Herman S.; Marsicano, Giovanni; Chaouloff, Francis; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">852-861</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biosynthetic and catabolic enzymes of the endocannabinoids tightly regulate endocannabinoid-mediated activation of the cannabinoid CB1 receptor.  Monitoring the activities of these endocannabinoid hydrolases in different brain regions is, therefore, key to gain insight in spatiotemporal control of CB1 receptor-mediated physiol.  We have employed a comparative chem. proteomics approach to quant. map the activity profile of endocannabinoid hydrolases in various mouse brain regions at the same time.  To this end, we used two different activity-based probes: fluorophosphonate-biotin (FP-biotin), which quantifies FAAH, ABHD6 and MAG-lipase activity, and MB108 that detects DAGL-α, ABHD4, ABHD6 and ABHD12.  In total 32 serine hydrolases were evaluated in the frontal cortex, hippocampus, striatum and cerebellum.  Comparison of endocannabinoid hydrolases activity in the four brain regions revealed that FAAH activity was highest in hippocampus, MAGL activity was most pronounced in the frontal cortex, whereas DAGL-α was most active in cerebellum.  Comparison of the activity profiles with a global proteomics dataset revealed pronounced differences.  This could indicate that post-translational modification of the endocannabinoid hydrolases is important to regulate their activity.  Next, the effect of genetic deletion of the CB1 receptor was studied.  No difference in the enzymic activity was found in the cerebellum, striatum, frontal cortex and hippocampus of CB1 receptor knockout animals compared to wild type mice.  Our results are in line with previous reports and indicate that the CB1 receptor exerts no regulatory control over the basal prodn. and degrdn. of endocannabinoids and that genetic deletion of the CB1 receptor does not induce compensatory mechanisms in endocannabinoid hydrolase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMiXmX3tZPyrVg90H21EOLACvtfcHk0lgTNc6dHrpVXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1KlsLc%253D&md5=33a113b4a16adb69e9cca18674daa5b3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01052%26sid%3Dliteratum%253Aachs%26aulast%3DBaggelaar%26aufirst%3DM.%2BP.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3DMarsicano%26aufirst%3DG.%26aulast%3DChaouloff%26aufirst%3DF.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DChemical%2520Proteomics%2520Maps%2520Brain%2520Region%2520Specific%2520Activity%2520of%2520Endocannabinoid%2520Hydrolases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D852%26epage%3D861%26doi%3D10.1021%2Facschembio.6b01052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blankman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">A comprehensive profile of brain enzymes that hydrolyze the endocannabinoid 2-arachidonoylglycerol</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1356</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2007.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1016%2Fj.chembiol.2007.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=18096503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOktrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1347-1356&author=J.+L.+Blankmanauthor=G.+M.+Simonauthor=B.+F.+Cravatt&title=A+comprehensive+profile+of+brain+enzymes+that+hydrolyze+the+endocannabinoid+2-arachidonoylglycerol&doi=10.1016%2Fj.chembiol.2007.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Profile of Brain Enzymes that Hydrolyze the Endocannabinoid 2-Arachidonoylglycerol</span></div><div class="casAuthors">Blankman, Jacqueline L.; Simon, Gabriel M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1347-1356</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Endogenous ligands for cannabinoid receptors ("endocannabinoids") include the lipid transmitters anandamide and 2-arachidonoylglycerol (2-AG).  Endocannabinoids modulate a diverse set of physiol. processes and are tightly regulated by enzymic biosynthesis and degrdn.  Termination of anandamide signaling by fatty acid amide hydrolase (FAAH) is well characterized, but less is known about the inactivation of 2-AG, which can be hydrolyzed by multiple enzymes in vitro, including FAAH and monoacylglycerol lipase (MAGL).  Here, we have taken a functional proteomic approach to comprehensively map 2-AG hydrolases in the mouse brain.  Our data reveal that ∼85% of brain 2-AG hydrolase activity can be ascribed to MAGL, and that the remaining 15% is mostly catalyzed by two uncharacterized enzymes, ABHD6 and ABHD12.  Interestingly, MAGL, ABHD6, and ABHD12 display distinct subcellular distributions, suggesting that they may control different pools of 2-AG in the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF_Kmd_7v5urVg90H21EOLACvtfcHk0lgTNc6dHrpVXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOktrfE&md5=78c6355a3909bff3cbae25989ab6a956</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2007.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2007.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlankman%26aufirst%3DJ.%2BL.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DA%2520comprehensive%2520profile%2520of%2520brain%2520enzymes%2520that%2520hydrolyze%2520the%2520endocannabinoid%25202-arachidonoylglycerol%26jtitle%3DChem.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1347%26epage%3D1356%26doi%3D10.1016%2Fj.chembiol.2007.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Rooden, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Esbroeck, A. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggelaar, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florea, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, A. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottenhoff, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boot, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overkleeft, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Chemical Proteomic Analysis of Serine Hydrolase Activity in Niemann-Pick Type C Mouse Brain</span>. <i>Front. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">440</span>, <span class="refDoi"> DOI: 10.3389/fnins.2018.00440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.3389%2Ffnins.2018.00440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=30018533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252Fpt1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=440&author=E.+J.+van%0ARoodenauthor=A.+C.+M.+van+Esbroeckauthor=M.+P.+Baggelaarauthor=H.+Dengauthor=B.+I.+Floreaauthor=A.+R.+A.+Marquesauthor=R.+Ottenhoffauthor=R.+G.+Bootauthor=H.+S.+Overkleeftauthor=J.+Aertsauthor=M.+van+der%0AStelt&title=Chemical+Proteomic+Analysis+of+Serine+Hydrolase+Activity+in+Niemann-Pick+Type+C+Mouse+Brain&doi=10.3389%2Ffnins.2018.00440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomic Analysis of Serine Hydrolase Activity in Niemann-Pick Type C Mouse Brain</span></div><div class="casAuthors">van Rooden Eva J; van Esbroeck Annelot C M; Baggelaar Marc P; Deng Hui; van der Stelt Mario; Florea Bogdan I; Overkleeft Herman S; Marques Andre R A; Ottenhoff Roelof; Boot Rolf G; Aerts Johannes M F G</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">440</span>
        ISSN:<span class="NLM_cas:issn">1662-4548</span>.
    </div><div class="casAbstract">The endocannabinoid system (ECS) is considered to be an endogenous protective system in various neurodegenerative diseases.  Niemann-Pick type C (NPC) is a neurodegenerative disease in which the role of the ECS has not been studied yet.  Most of the endocannabinoid enzymes are serine hydrolases, which can be studied using activity-based protein profiling (ABPP).  Here, we report the serine hydrolase activity in brain proteomes of a NPC mouse model as measured by ABPP.  Two ABPP methods are used: a gel-based method and a chemical proteomics method.  The activities of the following endocannabinoid enzymes were quantified: diacylglycerol lipase (DAGL) α, α/β-hydrolase domain-containing protein 4, α/β-hydrolase domain-containing protein 6, α/β-hydrolase domain-containing protein 12, fatty acid amide hydrolase, and monoacylglycerol lipase.  Using the gel-based method, two bands were observed for DAGL α.  Only the upper band corresponding to this enzyme was significantly decreased in the NPC mouse model.  Chemical proteomics showed that three lysosomal serine hydrolase activities (retinoid-inducible serine carboxypeptidase, cathepsin A, and palmitoyl-protein thioesterase 1) were increased in Niemann-Pick C1 protein knockout mouse brain compared to wild-type brain, whereas no difference in endocannabinoid hydrolase activity was observed.  We conclude that these targets might be interesting therapeutic targets for future validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCz2J82E1UuxBrI9DTGjC0fW6udTcc2eZIkVJUcKIGrLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252Fpt1emsA%253D%253D&md5=00bc6207ceee7de4f4c996206dec98e8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3389%2Ffnins.2018.00440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnins.2018.00440%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BRooden%26aufirst%3DE.%2BJ.%26aulast%3Dvan%2BEsbroeck%26aufirst%3DA.%2BC.%2BM.%26aulast%3DBaggelaar%26aufirst%3DM.%2BP.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DFlorea%26aufirst%3DB.%2BI.%26aulast%3DMarques%26aufirst%3DA.%2BR.%2BA.%26aulast%3DOttenhoff%26aufirst%3DR.%26aulast%3DBoot%26aufirst%3DR.%2BG.%26aulast%3DOverkleeft%26aufirst%3DH.%2BS.%26aulast%3DAerts%26aufirst%3DJ.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DChemical%2520Proteomic%2520Analysis%2520of%2520Serine%2520Hydrolase%2520Activity%2520in%2520Niemann-Pick%2520Type%2520C%2520Mouse%2520Brain%26jtitle%3DFront.%2520Neurosci.%26date%3D2018%26volume%3D12%26spage%3D440%26doi%3D10.3389%2Ffnins.2018.00440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuboi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachovchin, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Vega, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Confirming target engagement for reversible inhibitors in vivo by kinetically tuned activity-based probes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">10345</span>– <span class="NLM_lpage">10348</span>, <span class="refDoi"> DOI: 10.1021/ja303400u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303400u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC38Xotlelsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=10345-10348&author=A.+Adibekianauthor=B.+R.+Martinauthor=J.+W.+Changauthor=K.-L.+Hsuauthor=K.+Tsuboiauthor=D.+A.+Bachovchinauthor=A.+E.+Speersauthor=S.+J.+Brownauthor=T.+Spicerauthor=V.+Fernandez-Vegaauthor=J.+Fergusonauthor=P.+S.+Hodderauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Confirming+target+engagement+for+reversible+inhibitors+in+vivo+by+kinetically+tuned+activity-based+probes&doi=10.1021%2Fja303400u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Confirming Target Engagement for Reversible Inhibitors in Vivo by Kinetically Tuned Activity-Based Probes</span></div><div class="casAuthors">Adibekian, Alexander; Martin, Brent R.; Chang, Jae Won; Hsu, Ku-Lung; Tsuboi, Katsunori; Bachovchin, Daniel A.; Speers, Anna E.; Brown, Steven J.; Spicer, Timothy; Fernandez-Vega, Virneliz; Ferguson, Jill; Hodder, Peter S.; Rosen, Hugh; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10345-10348</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of small-mol. inhibitors for perturbing enzyme function requires assays to confirm that the inhibitors interact with their enzymic targets in vivo.  Detg. target engagement in vivo can be particularly challenging for poorly characterized enzymes that lack known biomarkers (e.g., endogenous substrates and products) to report on their inhibition.  Here, we describe a competitive activity-based protein profiling (ABPP) method for measuring the binding of reversible inhibitors to enzymes in animal models.  Key to the success of this approach is the use of activity-based probes that show tempered rates of reactivity with enzymes, such that competition for target engagement with reversible inhibitors can be measured in vivo.  We apply the competitive ABPP strategy to evaluate a newly described class of piperazine amide reversible inhibitors for the serine hydrolases LYPLA1 and LYPLA2, two enzymes for which selective, in vivo active inhibitors are lacking.  Competitive ABPP identified individual piperazine amides that selectively inhibit LYPLA1 or LYPLA2 in mice.  In summary, competitive ABPP adapted to operate with moderately reactive probes can assess the target engagement of reversible inhibitors in animal models to facilitate the discovery of small-mol. probes for characterizing enzyme function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbIIWu6eV4H7Vg90H21EOLACvtfcHk0lgiyBxlr3Jq0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xotlelsr0%253D&md5=dde167e236af3e9e27739ea1b4926bff</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja303400u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303400u%26sid%3Dliteratum%253Aachs%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DHsu%26aufirst%3DK.-L.%26aulast%3DTsuboi%26aufirst%3DK.%26aulast%3DBachovchin%26aufirst%3DD.%2BA.%26aulast%3DSpeers%26aufirst%3DA.%2BE.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DSpicer%26aufirst%3DT.%26aulast%3DFernandez-Vega%26aufirst%3DV.%26aulast%3DFerguson%26aufirst%3DJ.%26aulast%3DHodder%26aufirst%3DP.%2BS.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DConfirming%2520target%2520engagement%2520for%2520reversible%2520inhibitors%2520in%2520vivo%2520by%2520kinetically%2520tuned%2520activity-based%2520probes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D10345%26epage%3D10348%26doi%3D10.1021%2Fja303400u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foe, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Linden, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Child, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">Development of an activity-based probe for acyl-protein thioesterases</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e0190255</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0190255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1371%2Fjournal.pone.0190255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=29364904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12lsLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&author=M.+Garlandauthor=C.+J.+Schulzeauthor=I.+T.+Foeauthor=W.+A.+van+der+Lindenauthor=M.+A.+Childauthor=M.+Bogyo&title=Development+of+an+activity-based+probe+for+acyl-protein+thioesterases&doi=10.1371%2Fjournal.pone.0190255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an activity-based probe for acyl-protein thioesterases</span></div><div class="casAuthors">Garland, Megan; Schulze, Christopher J.; Foe, Ian T.; van der Linden, Wouter A.; Child, Matthew A.; Bogyo, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0190255/1-e0190255/17</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein palmitoylation is a dynamic post-translational modification (PTM) important for cellular functions such as protein stability, trafficking, localization, and protein-protein interactions.  S-palmitoylation occurs via the addn. of palmitate to cysteine residues via a thioester linkage, catalyzed by palmitoyl acyl transferases (PATs), with removal of the palmitate catalyzed by acyl protein thioesterases (APTs) and palmitoyl-protein thioesterases (PPTs).  Tools that target the regulators of palmitoylation-PATs, APTs and PPTs-will improve understanding of this essential PTM.  Here, we describe the synthesis and application of a cell-permeable activity-based probe (ABP) that targets APTs in intact mammalian cells and the parasite Toxoplasma gondii.  Using a focused library of substituted chloroisocoumarins, we identified a probe scaffold with nanomolar affinity for human APTs (HsAPT1 and HsAPT2) and synthesized a fluorescent ABP, JCP174-BODIPY TMR (JCP174-BT).  We use JCP174-BT to profile HsAPT activity in situ in mammalian cells, to detect an APT in T. gondii (TgPPT1).  We show discordance between HsAPT activity levels and total protein concn. in some cell lines, indicating that total protein levels may not be representative of APT activity in complex systems, highlighting the utility of this probe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLW3SGhjwxIbVg90H21EOLACvtfcHk0lgiyBxlr3Jq0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12lsLnI&md5=46e3ee718dfc37bf7770bb2ee96a61e4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0190255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0190255%26sid%3Dliteratum%253Aachs%26aulast%3DGarland%26aufirst%3DM.%26aulast%3DSchulze%26aufirst%3DC.%2BJ.%26aulast%3DFoe%26aufirst%3DI.%2BT.%26aulast%3Dvan%2Bder%2BLinden%26aufirst%3DW.%2BA.%26aulast%3DChild%26aufirst%3DM.%2BA.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520an%2520activity-based%2520probe%2520for%2520acyl-protein%2520thioesterases%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26doi%3D10.1371%2Fjournal.pone.0190255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhGty6f2KvClw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bianco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1002/pro.2733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=10.1002%2Fpro.2733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=26103917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=295-301&author=G.+Biancoauthor=S.+Forliauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=Covalent+docking+using+autodock%3A+Two-point+attractor+and+flexible+side+chain+methods&doi=10.1002%2Fpro.2733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking using autodock: Two-point attractor and flexible side chain methods</span></div><div class="casAuthors">Bianco, Giulia; Forli, Stefano; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">295-301</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">We describe two methods of automated covalent docking using Autodock4: the two-point attractor method and the flexible side chain method.  Both methods were applied to a training set of 20 diverse protein-ligand covalent complexes, evaluating their reliability in predicting the crystallog. pose of the ligands.  The flexible side chain method performed best, recovering the pose in 75% of cases, with failures for the largest inhibitors tested.  Both methods are freely available at the AutoDock website ().</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbgwvwS_NTKrVg90H21EOLACvtfcHk0lhGty6f2KvClw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFehs7rI&md5=4e380f340983e78e149f48934ccc59fd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fpro.2733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2733%26sid%3Dliteratum%253Aachs%26aulast%3DBianco%26aufirst%3DG.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DCovalent%2520docking%2520using%2520autodock%253A%2520Two-point%2520attractor%2520and%2520flexible%2520side%2520chain%2520methods%26jtitle%3DProtein%2520Sci.%26date%3D2016%26volume%3D25%26spage%3D295%26epage%3D301%26doi%3D10.1002%2Fpro.2733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Won, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labby, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pricer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majmudar, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armacost, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torossian, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palakurthi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, C. L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, B. R.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanism for Isoform-Selective Inhibition of Acyl Protein Thioesterases 1 and 2 (APT1 and APT2)</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3374</span>– <span class="NLM_lpage">3382</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00720</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00720" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yjtb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=3374-3382&author=S.+J.+Wonauthor=D.+Davdaauthor=K.+J.+Labbyauthor=S.+Y.+Hwangauthor=R.+Pricerauthor=J.+D.+Majmudarauthor=K.+A.+Armacostauthor=L.+A.+Rodriguezauthor=C.+L.+Rodriguezauthor=F.+S.+Chongauthor=K.+A.+Torossianauthor=J.+Palakurthiauthor=E.+S.+Hurauthor=J.+L.+Meagherauthor=C.+L.+Brooksauthor=J.+A.+Stuckeyauthor=B.+R.+Martin&title=Molecular+Mechanism+for+Isoform-Selective+Inhibition+of+Acyl+Protein+Thioesterases+1+and+2+%28APT1+and+APT2%29&doi=10.1021%2Facschembio.6b00720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanism for Isoform-Selective Inhibition of Acyl Protein Thioesterases 1 and 2 (APT1 and APT2)</span></div><div class="casAuthors">Won, Sang Joon; Davda, Dahvid; Labby, Kristin J.; Hwang, Sin Ye; Pricer, Rachel; Majmudar, Jaimeen D.; Armacost, Kira A.; Rodriguez, Laura A.; Rodriguez, Christina L.; Chong, Fei San; Torossian, Kristopher A.; Palakurthi, Jasmine; Hur, Edward S.; Meagher, Jennifer L.; Brooks, Charles L.; Stuckey, Jeanne A.; Martin, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3374-3382</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational S-palmitoylation directs the trafficking and membrane localization of hundreds of cellular proteins, often involving a coordinated palmitoylation cycle that requires both protein acyltransferases (PATs) and acylprotein thioesterases (APTs) to actively redistribute S-palmitoylated proteins toward different cellular membrane compartments.  This process is necessary for the trafficking and oncogenic signaling of S-palmitoylated Ras isoforms, and potentially other peripheral membrane proteins.  Depalmitoylating enzymes APT1 and APT2 are sep. conserved in all vertebrates, suggesting unique functional roles for each enzyme.  The recent discovery of APT isoform-selective inhibitors ML348 and ML349 has opened new possibilities to probe the function of each enzyme, yet it remains unclear how each inhibitor achieves orthogonal inhibition.  Here, the authors report the high-resoln. structure of human APT2 in complex with ML349 (1.64 Å), as well as the complementary structure of human APT1 bound to ML348 (1.55 Å).  Although the overall peptide backbone structures are nearly identical, each inhibitor adopted a distinct conformation within each active site.  In APT1, ML348 was positioned above the catalytic triad, but in APT2, the sulfonyl group of ML349 formed H-bonds with active site resident water mols. to indirectly engage the catalytic triad and oxyanion hole.  Reciprocal mutagenesis and activity profiling revealed several differing residues surrounding the active site that served as crit. gatekeepers for isoform accessibility and dynamics.  Structural and biochem. anal. suggested that the inhibitors occupy a putative acyl-binding region, establishing the mechanism for isoform-specific inhibition, hydrolysis of acyl substrates, and structural orthogonality important for future probe development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rqk6_xozJ7Vg90H21EOLACvtfcHk0lhGty6f2KvClw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yjtb7O&md5=ea0a24c33162441e1d53d1223d05f79a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00720%26sid%3Dliteratum%253Aachs%26aulast%3DWon%26aufirst%3DS.%2BJ.%26aulast%3DDavda%26aufirst%3DD.%26aulast%3DLabby%26aufirst%3DK.%2BJ.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DPricer%26aufirst%3DR.%26aulast%3DMajmudar%26aufirst%3DJ.%2BD.%26aulast%3DArmacost%26aufirst%3DK.%2BA.%26aulast%3DRodriguez%26aufirst%3DL.%2BA.%26aulast%3DRodriguez%26aufirst%3DC.%2BL.%26aulast%3DChong%26aufirst%3DF.%2BS.%26aulast%3DTorossian%26aufirst%3DK.%2BA.%26aulast%3DPalakurthi%26aufirst%3DJ.%26aulast%3DHur%26aufirst%3DE.%2BS.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMartin%26aufirst%3DB.%2BR.%26atitle%3DMolecular%2520Mechanism%2520for%2520Isoform-Selective%2520Inhibition%2520of%2520Acyl%2520Protein%2520Thioesterases%25201%2520and%25202%2520%2528APT1%2520and%2520APT2%2529%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D3374%26epage%3D3382%26doi%3D10.1021%2Facschembio.6b00720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delano, W. L.</span></span> <i>The PyMOL Molecular Graphics Systems</i>, <span class="NLM_year">2002</span>; <a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+Delano&title=The+PyMOL+Molecular+Graphics+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DDelano%26aufirst%3DW.%2BL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520Systems%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubin, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dax, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzolio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogenstaetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Codd, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaplan, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carruthers, N. I.</span></span> <span> </span><span class="NLM_article-title">Identification and biological evaluation of 4-(3-trifluoromethylpyridin-2-yl)piperazine-1-carboxylic acid (5-trifluoromethylpyridin-2-yl)amide, a high affinity TRPV1 (VR1) vanilloid receptor antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1857</span>– <span class="NLM_lpage">1872</span>, <span class="refDoi"> DOI: 10.1021/jm0495071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0495071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;key=1%3ACAS%3A528%3ADC%252BD2cXps1Ogu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1857-1872&author=D.+M.+Swansonauthor=A.+E.+Dubinauthor=C.+Shahauthor=N.+Nasserauthor=L.+Changauthor=S.+L.+Daxauthor=M.+Jetterauthor=J.+G.+Breitenbucherauthor=C.+Liuauthor=C.+Mazurauthor=B.+Lordauthor=L.+Gonzalesauthor=K.+Hoeyauthor=M.+Rizzolioauthor=M.+Bogenstaetterauthor=E.+E.+Coddauthor=D.+H.+Leeauthor=S.-P.+Zhangauthor=S.+R.+Chaplanauthor=N.+I.+Carruthers&title=Identification+and+biological+evaluation+of+4-%283-trifluoromethylpyridin-2-yl%29piperazine-1-carboxylic+acid+%285-trifluoromethylpyridin-2-yl%29amide%2C+a+high+affinity+TRPV1+%28VR1%29+vanilloid+receptor+antagonist&doi=10.1021%2Fjm0495071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and biological evaluation of 4-(3-trifluoromethylpyridin-2-yl)piperazine-1-carboxylic acid (5-trifluoromethylpyridin-2-yl)amide, a high affinity TRPV1 (VR1) vanilloid receptor antagonist</span></div><div class="casAuthors">Swanson, Devin M.; Dubin, Adrienne E.; Shah, Chandra; Nasser, Nadia; Chang, Leon; Dax, Scott L.; Jetter, Michele; Breitenbucher, J. Guy; Liu, Changlu; Mazur, Curt; Lord, Brian; Gonzales, Lisa; Hoey, Kenway; Rizzolio, Michele; Bogenstaetter, Michael; Codd, Ellen E.; Lee, Doo H.; Zhang, Sui-Po; Chaplan, Sandra R.; Carruthers, Nicholas I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1857-1872</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High throughput screening using the recombinant human TRPV1 receptor was used to identify a series of pyridinylpiperazine ureas as TRPV1 vanilloid receptor ligands.  Exploration of the structure-activity relationships by parallel synthesis identified the essential pharmacophoric elements for antagonism that permitted further optimization via targeted synthesis to provide a potent orally bioavailable and selective TRPV1 modulator 41 active in several in vivo models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_mnRdFjQEq7Vg90H21EOLACvtfcHk0lhNlOUQBqta9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXps1Ogu7g%253D&md5=068c51ef0f08d6885921032a66e90c64</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm0495071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0495071%26sid%3Dliteratum%253Aachs%26aulast%3DSwanson%26aufirst%3DD.%2BM.%26aulast%3DDubin%26aufirst%3DA.%2BE.%26aulast%3DShah%26aufirst%3DC.%26aulast%3DNasser%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DDax%26aufirst%3DS.%2BL.%26aulast%3DJetter%26aufirst%3DM.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DMazur%26aufirst%3DC.%26aulast%3DLord%26aufirst%3DB.%26aulast%3DGonzales%26aufirst%3DL.%26aulast%3DHoey%26aufirst%3DK.%26aulast%3DRizzolio%26aufirst%3DM.%26aulast%3DBogenstaetter%26aufirst%3DM.%26aulast%3DCodd%26aufirst%3DE.%2BE.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DZhang%26aufirst%3DS.-P.%26aulast%3DChaplan%26aufirst%3DS.%2BR.%26aulast%3DCarruthers%26aufirst%3DN.%2BI.%26atitle%3DIdentification%2520and%2520biological%2520evaluation%2520of%25204-%25283-trifluoromethylpyridin-2-yl%2529piperazine-1-carboxylic%2520acid%2520%25285-trifluoromethylpyridin-2-yl%2529amide%252C%2520a%2520high%2520affinity%2520TRPV1%2520%2528VR1%2529%2520vanilloid%2520receptor%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1857%26epage%3D1872%26doi%3D10.1021%2Fjm0495071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QGS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QGS','PDB','6QGS'); return false;">PDB: 6QGS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i60"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01043">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_08584"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01043?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01043</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional synthetic procedures, mechanism of the formation of hydantoins, coupling conditions and crude LC/MS traces, inhibition curves, representative gels for IC<sub>50</sub><sup>App</sup> determination, full gels after labeling mouse brain lysate, covalent docking, triazole urea structure elucidation, copies of NMR spectra and HPLC purity data, and coomassie stains of gels (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Coordinates of docked probes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_002.pdb">PDB</a>, <a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_001.pdf">jm0c01043_si_001.pdf (6.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_002.pdb">jm0c01043_si_002.pdb (2.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_003.pdb">jm0c01043_si_003.pdb (2.65 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01043/suppl_file/jm0c01043_si_004.csv">jm0c01043_si_004.csv (2.01 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01043&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01043%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01043" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996faf5c90223c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
