
<html lang="en"     class="pb-page"  data-request-id="1b6d91b5-14f7-4b63-8f11-ccdae1d972b1"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-21;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.9b00846"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders" /></meta><meta name="dc.Creator" content="Jiuling  Yang" /></meta><meta name="dc.Creator" content="Yangbing  Li" /></meta><meta name="dc.Creator" content="Angelo  Aguilar" /></meta><meta name="dc.Creator" content="Zhaomin  Liu" /></meta><meta name="dc.Creator" content="Chao-Yie  Yang" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="Inducing protein degradation by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies. Based upon our previously reported PRO..." /></meta><meta name="Description" content="Inducing protein degradation by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies. Based upon our previously reported PRO..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 27, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00846" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00846" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00846" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00846" /></link>
        
    
    

<title>Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00846" /></meta><meta property="og:title" content="Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0017.jpeg" /></meta><meta property="og:description" content="Inducing protein degradation by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies. Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized additional analogues. Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a “molecular glue”, as exemplified by MG-277. MG-277 induces only moderate MDM2 degradation and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner. Our mechanistic investigation established that MG-277 is not a PROTAC MDM2 degrader but instead works as a molecular glue, inducing degradation of a translation termination factor, GSPT1 to achieve its potent anticancer activity. Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a molecular glue compound, which has a completely different mechanism of action." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00846"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00846">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00846&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00846&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00846&amp;href=/doi/10.1021/acs.jmedchem.9b00846" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9471-9487</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00812" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00864" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jiuling Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiuling Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology,  Department of Internal Medicine,  Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiuling++Yang">Jiuling Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yangbing Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yangbing Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Internal Medicine,  Department of Medicinal Chemistry,  Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yangbing++Li">Yangbing Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Angelo Aguilar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Angelo Aguilar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Internal Medicine,  Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Angelo++Aguilar">Angelo Aguilar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhaomin Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaomin Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Internal Medicine,  Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaomin++Liu">Zhaomin Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao-Yie Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao-Yie Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Internal Medicine,  Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao-Yie++Yang">Chao-Yie Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology,  Department of Internal Medicine,  Department of Medicinal Chemistry,  Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#81f2e9e0eeece4efe6c1f4ece8e2e9afe4e5f4"><span class="__cf_email__" data-cfemail="186b707977757d767f586d75717b70367d7c6d">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00846&amp;href=/doi/10.1021%2Facs.jmedchem.9b00846" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9471–9487</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 27, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 May 2019</li><li><span class="item_label"><b>Published</b> online</span>27 September 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00846" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00846</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9471%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJiuling%2BYang%252C%2BYangbing%2BLi%252C%2BAngelo%2BAguilar%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D21%26contentID%3Dacs.jmedchem.9b00846%26title%3DSimple%2BStructural%2BModifications%2BConverting%2Ba%2BBona%2Bfide%2BMDM2%2BPROTAC%2BDegrader%2Binto%2Ba%2BMolecular%2BGlue%2BMolecule%253A%2BA%2BCautionary%2BTale%2Bin%2Bthe%2BDesign%2Bof%2BPROTAC%2BDegraders%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9487%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00846"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7235</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">32</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00846" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jiuling&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Yangbing&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Angelo&quot;,&quot;last_name&quot;:&quot;Aguilar&quot;},{&quot;first_name&quot;:&quot;Zhaomin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Chao-Yie&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9471-9487&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00846&quot;},&quot;abstract&quot;:&quot;Inducing protein degradation by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies. Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized additional analogues. Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a “molecular glue”, as exemplified by MG-277. MG-277 induces only moderate MDM2 degradation and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner. Our mechanistic investigation established that MG-277 is not a PROTAC MDM2 degrader but instead works as a molecular glue, inducing degradation of a translation termination factor, GSPT1 to achieve its potent anticancer activity. Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a molecular glue compound, which has a completely different mechanis&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00846&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00846" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00846&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00846" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00846&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00846" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00846&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00846&amp;href=/doi/10.1021/acs.jmedchem.9b00846" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00846" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00846" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00846%26sid%3Dliteratum%253Aachs%26pmid%3D31560543%26genre%3Darticle%26aulast%3DYang%26date%3D2019%26atitle%3DSimple%2BStructural%2BModifications%2BConverting%2Ba%2BBona%2Bfide%2BMDM2%2BPROTAC%2BDegrader%2Binto%2Ba%2BMolecular%2BGlue%2BMolecule%253A%2BA%2BCautionary%2BTale%2Bin%2Bthe%2BDesign%2Bof%2BPROTAC%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D21%26spage%3D9471%26epage%3D9487%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/jmcmar.2019.62.issue-21/20191114/jmcmar.2019.62.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inducing protein degradation by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies. Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized additional analogues. Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a “molecular glue”, as exemplified by <b>MG-277</b>. <b>MG-277</b> induces only moderate MDM2 degradation and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner. Our mechanistic investigation established that <b>MG-277</b> is not a PROTAC MDM2 degrader but instead works as a molecular glue, inducing degradation of a translation termination factor, GSPT1 to achieve its potent anticancer activity. Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a molecular glue compound, which has a completely different mechanism of action.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47883" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47883" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The proteolysis targeting chimeric (PROTAC) concept was formally proposed in 2001 with the objective of inducing targeted protein degradation by hijacking the powerful cellular degradation systems.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The very first PROTAC molecule employed the Skp1-Cullin-F box (SCF<sup>β-TrCP</sup>) E3 ligase complex and successfully targeted methionine aminopeptidase-2 (MetAP-2) for degradation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In recent years, the PROTAC approach has gained momentum, showing promise for the discovery and development of new therapeutic agents.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7 ref8">(2−8)</a></div><div class="NLM_p">A typical PROTAC small-molecule degrader consists of three essential components: a ligand that binds the protein of interest, a second ligand that binds to and recruits an E3 ligase degradation complex, and a linker tethering the two ligands together.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7 ref8">(2−8)</a> Since 2001, a number of E3 ligases have been tested for the design of PROTAC molecules and have enjoyed different levels of success.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18">(1−18)</a> The Cullin 4A E3 ligase complex and the Cullin 2 E3 ligase complex have emerged as two powerful E3 ligase complexes for the design of highly potent and effective PROTAC molecules, in part because of the availability of potent and druglike small-molecule ligands to recruit these degradation complexes.<a onclick="showRef(event, 'ref2 ref15'); return false;" href="javascript:void(0);" class="ref ref2 ref15">(2,15)</a> In the case of Cullin 4A, a class of well-known small-molecule drugs, represented by thalidomide and lenalidomide and collectively known as immunomodulatory imide drugs (IMiDs), have been discovered as potent small-molecule ligands for cereblon, a receptor protein for the Cullin 4A complex.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a></div><div class="NLM_p">The landmark discovery that cereblon is the molecular target for IMiDs has also revealed a completely new mechanism of drug action.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> By binding to cereblon, an E3 ubiquitin ligase receptor, IMiDs alter the specificity of the Cullin 4A E3 ubiquitin ligase and cause it to recruit a set of new substrates, or neo-substrates for ubiquitination by the Cullin 4A complex. This is followed by proteasomal degradation.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a> The term “molecular glue” has been used to describe the unprecedented mechanism of action of IMiDs.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> In addition to IMiDs, indisulam has been recently discovered as another class of molecular glue.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Indisulam brings together the CUL4-DDB1-DDA1-DCAF15 E3 ubiquitin ligase complex with neo-substrates such as RBM39 for ubiquitination and proteasomal degradation.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> The “molecular glue” mechanism of action has also opened an avenue for the discovery of completely new types of drugs.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">Theoretically, a putative PROTAC molecule can function as either a bona fide PROTAC degrader to bind to and degrade the intended target protein(s) or it can act as a molecular glue by recruiting and degrading neo-substrates. Recently, Ishoey et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> demonstrated that a class of phthalimide conjugate putative PROTAC degraders fails to induce degradation of their consensus kinase targets in cells but instead functions as a molecular glue.</div><div class="NLM_p last">We recently reported the discovery of the first-in-class highly potent PROTAC degraders of MDM2, exemplified by MD-222 and MD-224, which induce rapid degradation of the MDM2 protein and activation of wild-type p53 in cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> These PROTAC MDM2 degraders potently inhibit cell growth in human cancer cell lines carrying wild-type p53 and demonstrate significant specificity over cancer cell lines with mutated or deleted p53. In our further modifications of MD-222 and its analogues, we identified another type of compounds, exemplified by <b>MG-277</b>. In contrast to MD-222 and MD-224, <b>MG-277</b> is much less effective than MD-222 in inducing degradation of MDM2 and fails to activate wild-type p53. Furthermore, <b>MG-277</b> potently inhibits cell growth in cancer cell lines in a p53-independent manner, displaying a completely different mode of action from that of MD-222 or MD-224. Our mechanistic investigation revealed that <b>MG-277</b> is not a PROTAC MDM2 degrader and instead behaves as a molecular glue. Herein, we report the discovery of <b>MG-277</b> and our detailed investigation of its mechanism of action.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26340" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26340" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of <b>MG-277</b> as a New Analogue of MD-222, a Bona fide PROTAC MDM2 Degrader</h3><div class="NLM_p">Previously, we reported the discovery and extensive investigation of MD-222 and its analogues as the first-in-class PROTAC MDM2 degraders.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> MD-222 was designed and synthesized using a potent and selective MDM2 inhibitor (MI-1061, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), which was conjugated, through a linker, with a lenalidomide moiety.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of <b>MG-277</b> as a new, putative PROTAC MDM2 degrader. (A) Structures of MDM2 inhibitors MI-1061 and MI-2103, the bona fide MDM2 degrader MD-222 and a putative MDM2 degrader <b>MG-277</b>. (B–D) Binding affinities of MI-1061 (B), MI-2103 (C) and <b>MG-277</b> (D) to MDM2 by an optimized FP competitive binding assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We further investigated how the binding affinity of the MDM2 inhibitor component to MDM2 protein in our MDM2 degraders affects their ability to degrade the MDM2 protein. To this end, we replaced the benzoic acid fragment in MI-1061 with a methyl group, obtaining MI-2103 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). In our optimized fluorescence-polarization (FP) based binding assay,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> the MDM2 inhibitors MI-1061 and MI-2103, bind to MDM2 with IC<sub>50</sub> values of 9.5 and 48.1 nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B–D). While MI-2103 is ∼5-times less potent than MI-1061, it is still a high-affinity inhibitor of MDM2. Employing MI-2103 for the MDM2 inhibitor moiety and using the same linker and lenalidomide moiety as those in MD-222, we synthesized <b>MG-277</b> as a putative PROTAC MDM2 degrader.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> <b>MG-277</b> Displays High Potencies in Inhibition of Cell Growth in Cancer Cell Lines with Different p53 Status, in Contrast to the Bona fide PROTAC MDM2 Degrader MD-222</h3><div class="NLM_p">In our previous study,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> we showed that acute leukemia cell lines RS4;11, MOLM-13, and MV4;11 carrying wild-type p53 are very responsive to MDM2 inhibitors and degraders. We therefore employed these three leukemia cell lines to evaluate the bona fide MDM2 degrader MD-222, the new putative MDM2 degrader <b>MG-277</b>, and the MDM2 inhibitors MI-1061 and MI-2103.</div><div class="NLM_p">MD-222 and <b>MG-277</b> potently and effectively inhibit cell growth in each of these three leukemia cell lines and are much more potent than their corresponding MDM2 inhibitors (Figure S1A–C in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Supporting Information</a>). For example, <b>MG-277</b> displays IC<sub>50</sub> values of 1.3, 24.6, and 7.9 nM, respectively, in these three cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S1A–C</a>). In comparison, the corresponding MDM2 inhibitor MI-2103 has IC<sub>50</sub> values of 669, 1238, and 898 nM, respectively, in the same three cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S1A–C</a>). These data show that the new, putative MDM2 degrader <b>MG-277</b> is >50-times more potent than its corresponding MDM2 inhibitor, MI-2103, in inhibition of cell growth in each of these three cell lines.</div><div class="NLM_p">Our previous study<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> showed that similar to MDM2 inhibitors, the PROTAC MDM2 degraders MD-222 and MD-224 also display a high cellular specificity over cancer cell lines carrying mutated or deleted p53. We tested <b>MG-277</b>, MD-222, and MI-2103 for their specificity in the RS4;11/IRMI-2 leukemia cell line,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> which contains p53 hot-spot mutations (Y236H and R249G), and in the HL-60 leukemia cell line which has a loss of p53. As expected, the bona fide MDM2 degrader MD-222 and the MDM2 inhibitor MI-2103 are completely ineffective in the RS4;11/IRMI-2 and HL-60 leukemia lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S1D,E</a>). Surprisingly, <b>MG-277</b> is very potent and effective in inhibition of cell growth in both the RS4;11/IRMI-2 and HL-60 leukemia lines, achieving IC<sub>50</sub> values of 3.9 and 8.3 nM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S1D,E</a>).</div><div class="NLM_p">To further investigate their cellular specificity, we evaluated <b>MG-277</b>, MD-222, and MI-2103 for their cell growth inhibitory activity in the MDA-MB-231 and MDA-MB-468 breast cancer cell lines, which harbor a mutated p53 protein.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> While MD-222 and MI-2103 fail to inhibit cell growth in these two breast cancer cell lines, <b>MG-277</b> potently inhibits cell growth in MDA-MB-231 and MDA-MB-468 breast cancer cell lines, with IC<sub>50</sub> values of 39.4 and 26.4 nM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S1F,G</a>).</div><div class="NLM_p last">These cell growth inhibition data showed that while the bona fide PROTAC MDM2 degrader, MD-222, displays high cellular specificity for p53 wild-type cancer cell lines over p53 mutated or deleted cell lines; <b>MG-277</b> is potent and effective against cancer cell lines carrying wild-type, mutated, or deleted p53. Therefore, it is very clear that <b>MG-277</b> and MD-222 have different cellular mechanisms of action.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis and Evaluation of <b>MG-277</b> Analogues with Different Linkers</h3><div class="NLM_p">The linker between the MDM2 inhibitor moiety and the lenalidomide moiety in <b>MG-277</b> is much shorter than that in MD-222, and this may contribute to their different cellular mechanisms of action. To investigate this, we synthesized a series of analogues of <b>MG-277</b> with different linker lengths and/or compositions (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We evaluated these compounds for their cell growth inhibitory activity in the RS4;11 cells with wild-type p53 and RS4;11/IRMI-2 cells carrying a mutated p53, obtaining the data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Linker Exploration between MI-2103 and Lenalidomide Moiety and the Corresponding IC<sub>50</sub> in the RS4;11 Cell Line and the p53 Mutant Cell Line RS4;11/IRMI-2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0015.gif" alt="" id="GRAPHIC-d7e503-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0016.gif" alt="" id="gr15" /></img><div></div></div><div class="NLM_p last">MG-274, in which MI-2103 and lenalidomide are connected directly without a linker, display similar potencies in both cell lines, and MG-274 is slightly more potent than MI-2103 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>). MG-275 has a (CH<sub>2</sub>)<sub>3</sub> linker, two methylenes less than the linker in <b>MG-277</b>, and is approximately 10 times more potent than MG-274 in both cell lines and displays essentially no selectivity between the RS4;11 cell line and the RS4;11/IRMI-2 cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>). MG-276, which has a linker one methylene shorter than that in <b>MG-277</b>, is equipotent with <b>MG-277</b> in both cell lines and shows no selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>). Insertion of one oxygen atom into the linker in <b>MG-277</b> yielded MG-278, which is also equipotent with <b>MG-277</b> in both cell lines and displays no cellular selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>). Replacement of the (CH<sub>2</sub>)<sub>5</sub> linker in <b>MG-277</b> with the longer and more hydrophilic linker −(CH<sub>2</sub>)<sub>2</sub>O–CH<sub>2</sub>O–(CH<sub>2</sub>)<sub>3</sub>–, afforded MG-279, which is similarly potent and nonselective as <b>MG-277</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>). Changing the (CH<sub>2</sub>)<sub>5</sub> linker in <b>MG-277</b> to a (CH<sub>2</sub>O)<sub>3</sub>(CH<sub>2</sub>)<sub>3</sub> linker yielded MG-280, which displays IC<sub>50</sub> values of 14 and 14 nM in the RS4;11 cell line carrying wild-type p53 and the RS4;11/IRMI-2 cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>), respectively. MG-280 is therefore 5-times less potent than <b>MG-277</b>, and displays no selectivity between these two cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2A,B</a>). Taken together, these data show that while the linker length and composition have clear influence on the cellular potencies, the lack of cellular selectivity for <b>MG-277</b> between cell lines carrying wild-type p53 and cell lines containing a mutated or deleted p53 is not simply because of its shorter linker than that in MD-222.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> <b>MG-277</b> Is Much Less Effective and Less Potent Than MD-222 in Inducing MDM2 Degradation and Fails To Activate Wild-Type p53</h3><div class="NLM_p">A bona fide PROTAC MDM2 degrader such as MD-222 is potent and effective in inducing MDM2 degradation and in activating wild-type p53 in cancer cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> We evaluated <b>MG-277</b> for its ability to induce MDM2 degradation and activation of p53 in the RS4;11 cell line, with MD-222 and MI-2103 used as controls.</div><div class="NLM_p">Western blotting analysis showed that MD-222 is very effective in reducing the levels of MDM2 protein and increasing the levels of p53 after treatment for just 1 h in RS4;11 cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). This is consistent with our previous data<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and its mechanism as an MDM2 degrader. In comparison, with a treatment time of 1 h MI-2103 induces upregulation of both MDM2 and p53 proteins (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), consistent with its mechanism as an MDM2 inhibitor. In contrast, <b>MG-277</b> fails to reduce the levels of MDM2 protein and increase the levels of p53 with 1 h treatment time in RS4;11 cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Treatment of the RS4;11 cells with MD-222 for 2 h shows that it reduces the level of MDM2 and increases the level of p53 more than was observed with a treatment time of 1 h (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Interestingly, with a 2 h treatment time, <b>MG-277</b> reduces the MDM2 level moderately but fails to increase the level of p53 in the RS4;11 cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <b>MG-277</b> is much less effective and less potent than MD-222 in inducing MDM2 degradation and fails to activate wild-type p53. (A,B) Western blotting analysis of MDM2 and p53 protein levels in RS4;11 cells after treatment with MI-2103, <b>MG-277</b>, and MD-222 for 1 (A) or 2 h (B), and in p53 mutant RS4;11/IRMI-2 cell line after 2 h treatment (C). The cells were treated for 1 or 2 h with the compounds at the indicated concentrations, and proteins were probed by specific antibodies. (D–G) Changes of mRNA levels of p53 downstream target genes, including <i>MDM2</i>, <i>CDKN1A</i>, <i>PUMA</i>, and <i>BAX</i> were analyzed after treatment with 3 nM <b>MG-277</b> or 10 nM MD-222 in the RS4;11 cell line by qRT-PCR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the RS4;11/IRMI-2 cell line carrying a mutated p53, both MD-222 and <b>MG-277</b> can reduce the levels of MDM2 with 2 h treatment time but again MD-222 is much more potent and effective than <b>MG-277</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).</div><div class="NLM_p">Quantitative reverse transcription polymerase chain reaction (qRT-PCR) experiments show that MD-222 is effective in increasing the expression of several p53-targeted genes, such as <i>MDM2</i>, <i>CDKN1A</i>, <i>PUMA</i>, and <i>BAX</i> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D–G) in RS4;11 cells, indicating strong activation of p53. In contrast, <b>MG-277</b> fails entirely to increase the expression of any of these p53-targeted genes (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D–G), showing lack of p53 activation.</div><div class="NLM_p last">Hence, while MD-222 is very effective in reducing the levels of MDM2 and activating wild-type p53 in RS4;11 cells, <b>MG-277</b> is much less potent and effective than MD-222 in reducing the levels of MDM2 and fails to activate wild-type p53.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> The Cell Growth Inhibition Activity of <b>MG-277</b> Is MDM2-Independent</h3><div class="NLM_p"><b>MG-277</b> is much less potent and effective than MD-222 in reducing the levels of MDM2 in the RS4;11 cells but retains a moderate effect (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). We examined if the cell growth inhibition activity of <b>MG-277</b> is MDM2-dependent. Because it is difficult to knock-down MDM2 efficiently in leukemia cells, we knocked down MDM2 by siRNA in the MDA-MB-231 and MDA-MB-468 breast cancer cell lines and assessed the cell growth inhibitory activity of <b>MG-277</b> in these MDM2 knocked-down cell lines.</div><div class="NLM_p">Our data show that efficient knock-down of MDM2 by siRNA in these two cell lines has no effect on the cell growth inhibitory activity of <b>MG-277</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D), when compared to the siRNA controls (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). These data also show that <b>MG-277</b> achieves its cell growth inhibitory activity in both the MDA-MB-231 and MDA-MB-468 cell lines through an MDM2-independent mechanism.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cell growth inhibition activity of <b>MG-277</b> is MDM2-independent. (A–D) Effect of knock-down of MDM2 on the cell growth inhibitory activity of MI-2103 and <b>MG-277</b> in MDA-MB-231 and MDA-MB-468 cells after transfection with MDM2 siRNAs (C,D) or nontargeting siRNA control vector (si CRL) (A,B) for 2 days. (E) MDM2 knockdown efficiency was determined by Western blotting analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> The Cell Growth Inhibition Activity of <b>MG-277</b> Does Not Require Binding to MDM2</h3><div class="NLM_p"><b>MG-277</b> consists of an MDM2 binding moiety, and a cereblon binding moiety, tethered together through a linker. We investigated if the binding of <b>MG-277</b> to MDM2 is required for its cellular growth inhibition activity by synthesizing and evaluating a number of analogues using an MDM2 inhibitor which is much weaker than <b>MG-277</b>.</div><div class="NLM_p">In our previous studies,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> we showed that the halogen atoms in our MDM2 inhibitors play an important role in enhancing the binding affinity to MDM2. Therefore, we made three MDM2 inhibitors by removal of one or more of the Cl or F atoms. Using these three much weaker MDM2 inhibitors, we have synthesized three analogues of <b>MG-277</b> using the same linker and the lenalidomide moiety as those in <b>MG-277</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A–C). Our binding data showed that the conjugated compounds based on these weaker MDM2 inhibitors show minimal binding to MDM2 (IC<sub>50</sub> > 20 μM) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>F).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cell growth inhibition activity of <b>MG-277</b> does not require binding to MDM2. (A–C) Structures of MC-215, MC-216, and MC-217, three control analogues of <b>MG-277</b> formed by removal of fluorine and/or one or two chlorine atoms from the MI-2103 portion of <b>MG-277</b>. (D,E) Cell growth inhibitory activity of <b>MG-277</b>, MC-215, MC-216, and MC-217 in RS4;11 (D) and RS4;11/IRMI-2 (E) cell lines. (F) Binding affinity of the three analogues of <b>MG-277</b> with MDM2 protein was determined by an FP assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">These three new analogues, MC-215, MC-216, and MC-217, however, all display potent cell growth inhibitory activity in both the RS4;11 and RS4;11/IRMI-2 cell lines and show no selectivity over the RS4;11/IRMI-2 cell line (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D,E). These data indicate that the cell growth inhibition activity by MC-215, MC-216, MC-217, and <b>MG-277</b> in different cell lines is independent of their binding to MDM2.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> The Cell Growth Inhibitory Activity of <b>MG-277</b> Is Dependent on Cereblon Binding</h3><div class="NLM_p">Because <b>MG-277</b> contains a cereblon-binding moiety, we investigated if its cell growth inhibitory activity is dependent upon its binding to cereblon using several complementary methods.</div><div class="NLM_p">The co-crystal structure of lenalidomide in a complex with cereblon shows that the amino group of the glutarimide in lenalidomide forms a hydrogen bond with cereblon and methylation of this amino group should completely block the binding to cereblon.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> We therefore synthesized MC-024 in which the amino group of the glutarimide in <b>MG-277</b> is methylated (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>I). MC-024 is >1000-times less potent than <b>MG-277</b> in inhibition of cell growth in RS4;11, RS4;11/IRMI-2, MDA-MB-231, and MDA-MB-468 cell lines (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E–5H), indicating that binding of <b>MG-277</b> to cereblon is required for its cell growth inhibitory activity in cell lines with wild-type and mutated p53.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular activity of <b>MG-277</b> is dependent on cereblon binding. (A–D) Cell growth inhibitory activity of MI-2103, MD-222, and <b>MG-277</b> in RS4;11, RS4;11/IRMI-2, MDA-MB-231, and MDA-MB-468 cells in the presence or absence of excess lenalidomide to occupy the binding site on cereblon. (E–H) Cell growth inhibitory activity of MI-2103, <b>MG-277</b>, and MC-024 with no cereblon binding in RS4;11, RS4;11/IRMI-2, MDA-MB-231, and MDA-MB-468 cell lines by a 4-day cell viability assay. (I) Structure of MC-024 with the amino group of the glutarimide in <b>MG-277</b> methylated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further examine the effect of cereblon binding on cell growth inhibition by <b>MG-277</b>, we used an excess amount of lenalidomide to compete with <b>MG-277</b> for binding with cereblon. Our data show that lenalidomide is very effective in reducing the cell growth inhibitory activity of <b>MG-277</b> in a dose-dependent manner in p53 wild-type RS4;11 cells and p53 mutant RS4;11/IRMI-2, MDA-MB-231, and MDA-MB-468 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–D).</div><div class="NLM_p">We knocked down <i>CRBN</i> in the MDA-MB-231 and MDA-MB-468 cell lines to examine the effect on cell growth inhibition by <b>MG-277</b>, and found that efficient knock-down of <i>CRBN</i> dramatically reduces the cell growth inhibition by <b>MG-277</b> in both cell lines (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cell growth inhibition activity of <b>MG-277</b> requires binding to cereblon. The effect of siRNA <i>CRBN</i> knock-down on the cell growth inhibitory activity of <b>MG-277</b>. (A–F) MDA-MB-231 (A–C) and MDA-MB-468 (D–F) cells were transfected with two different <i>CRBN</i> siRNAs targeting two different sequences (B–C, E–F) or nontargeting siRNA control vector (A,D) before treatment with MI-2103 or <b>MG-277</b> for 3 days. Cell growth inhibition was determined by cell viability assay. (G) Western blotting for the cereblon protein level was used to determine the knockdown efficiency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">These three sets of complementary experiments clearly demonstrate that binding of <b>MG-277</b> to cereblon is required for the cell growth inhibitory activity of <b>MG-277</b> in cancer cell lines with wild-type or mutated p53.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <b>MG-277</b> Effectively and Specifically Induces Degradation of GSPT1</h3><div class="NLM_p">Because the cell growth inhibitory activity of <b>MG-277</b> is MDM2-independent and cereblon-dependent, we hypothesized that <b>MG-277</b> may in fact function as a molecular glue and promote recruitment of certain cell growth-related protein(s) to the CUL4-DDB1-CRBN E3 ligase complex (CRL4<sup>CRBN</sup>) for ubiquitination and subsequent proteasomal degradation.</div><div class="NLM_p">To identify the actual cellular target(s) of <b>MG-277</b>, we performed an unbiased proteomic analysis using the RS4;11 cells with wild-type p53 and the RS4;11/IRMI-2 cells carrying mutated p53. This analysis showed that upon treatment with 0.1 μM of <b>MG-277</b> for 3 h, the G1 to S phase transition 1 (GSPT1) protein, a translation termination factor, is the only protein whose level is greatly reduced in both RS4;11 (5.3-fold, <i>P</i> < 0.001) and RS4;11/IRMI-2 (3.7-fold, <i>P</i> < 0.001) cell lines (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <b>MG-277</b> effectively and specifically induces the degradation of GSPT1. (A,B) Proteomic analysis/protein profiling after a 3 h treatment with 0.1 μM of <b>MG-277</b> in p53 wild-type RS4;11 (A) and p53 mutated RS4;11/IRMI-2 (B) cells. The <i>Y</i>-axis represents <i>P</i>-value and <i>X</i>-axis represents the log 2 ratio of the level of a certain protein after the treatment to its protein level before the treatment. (C) GSPT1 mRNA level was measured by qRT-PCR after treatment with <b>MG-277</b> and MD-222 at indicated time points. (D–G) Western blotting analysis of the GSPT1 protein level after treatment with MI-2103, <b>MG-277</b>, and MD-222 for 2 h (D,E) or treatment with <b>MG-277</b> or MD-222 for different time lengths as indicated (F,G) in RS4;11 and RS4;11/IRMI-2 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">GSPT1, also known as eRF3a, mediates stop codon recognition and nascent protein release from the ribosome through the interaction with a release factor, eRF1.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> A recent study reported the similar finding that two series of phthalimide conjugate degraders originally designed to target multiple kinases for degradation actually induce significant GSPT1 degradation and fail to degrade their consensus targets.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> GSPT1 Protein Level Is Greatly Reduced by Treatment with <b>MG-277</b> and Its Analogues but Not by Treatment with MD-222, a Bona fide PROTAC MDM2 Degrader</h3><div class="NLM_p">The data in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B suggest that GSPT1 may be the true molecular target for <b>MG-277</b>. Accordingly, we investigated if degradation of GSPT1 protein is required for the cell growth inhibitory activity of <b>MG-277</b>.</div><div class="NLM_p">Western blot analysis showed that <b>MG-277</b>, but not MD-222, effectively reduces the level of the GSPT1 protein in a dose- and time-dependent manner in both p53 wild-type and mutant cell lines (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D–G). The kinetics data showed that the GSPT1 protein level is markedly decreased starting from 1 to 3 h with <b>MG-277</b> at a concentration as low as 10 nM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>F,G), indicating rapid degradation of GSPT1 induced by <b>MG-277</b>. No reduction of GSPT1 mRNA level is observed after treatment with <b>MG-277</b>, indicating that <b>MG-277</b> has no effect on GSPT1 transcription (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C).</div><div class="NLM_p">We further evaluated additional analogues of <b>MG-277</b> containing linkers of different lengths and compositions (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All of these analogues, with the exception of MG-274 and MG-275, effectively reduce the GSPT1 protein level after 2 h of treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2C,D</a>). MG-274 with no linker and MG-275 with a (CH<sub>2</sub>)<sub>3</sub> linker are also both capable of reducing the level of GSPT1 but require higher concentrations (1–3 μM) and 24 h treatment time (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2E,F</a>), consistent with their weaker potencies in cell growth inhibition. MG-276, <b>MG-277</b>, MG-278, and MG-279 achieve DC<sub>50</sub> (concentration required to degrade 50% of protein) values of 1.1, 1.3, 1.8, and 1.2 nM, respectively, in the RS4;11 cell line with 24 h treatment. MG-276, <b>MG-277</b>, MG-278, and MG-279 are also highly potent in inducing degradation of GSPT1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S3B–D,F</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Table S1</a>). MG-274 (DC<sub>50</sub> = 2.3 μM) with no linker and MG-275 (DC<sub>50</sub> = 0.12 μM) with the shortest linker are much less potent than <b>MG-277</b> in inducing GSPT1 degradation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S3A,F</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Table S1</a>). The MDM2 inhibitor MI-2103 has no effect on the level of GSPT1 protein (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S2C,D</a>).</div><div class="NLM_p last">These data establish an excellent correlation between the induced degradation of GSPT1 and the cell growth inhibition by <b>MG-277</b> and its analogues in both p53 wild-type and p53-mutated cell lines.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Decrease of the GSPT1 Protein Level Induced by <b>MG-277</b> Is Dependent on Its Binding with Cereblon, Proteasome, and Cullin-RING E3 Ligase Function but Independent of MDM2</h3><div class="NLM_p">Our data show that cell growth inhibition by <b>MG-277</b> depends upon cereblon but not on MDM2. We next investigated the dependence of GSPT1 degradation induced by <b>MG-277</b> on cereblon, proteasome, CUL E3 ligase, and MDM2.</div><div class="NLM_p">Co-treatment with an excess amount of lenalidomide (30 μM) effectively blocks the degradation of GSPT1 protein induced by <b>MG-277</b> in both MDA-MB-231 and MDA-MB-468 cell lines (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A,B). Efficient siRNA knock-down of <i>CRBN</i> in these two breast cancer cell lines also blocks the degradation of GSPT1 induced by <b>MG-277</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C,D). These data establish that degradation of GSPT1 protein induced by <b>MG-277</b> depends upon cereblon.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Decrease of the GSPT1 protein level induced by <b>MG-277</b> is dependent on its binding with cereblon, proteasome, and CRLs but independent of MDM2. (A,B) Western blotting for the GSPT1 protein level after 2 h treatment with MI-2103, <b>MG-277</b>, or MD-222 in MDA-MB-231 (A) and MDA-MB-468 (B) cells with or without lenalidomide (30 μM) to block binding with cereblon. (C,D) MDA-MB-231 (C) and MDA-MB-468 (D) cells were transfected with <i>CRBN</i> siRNAs targeting two different sequences or nontargeting siRNA control. Protein levels of GSPT1 and cereblon were determined by western blot analysis after treatment with 0.1 μM <b>MG-277</b> for 2 h or 24 h in the transfected cells. (E) Western blot analysis for the GSPT1 protein level after 2 h treatment with <b>MG-277</b> in the presence or absence of 10 μM MI-2103 [or dimethylsulfoxide (DMSO) as control] to block binding with MDM2 in MDA-MB-231 cells. (F) Western blotting for MDM2 and GSPT1 protein levels after 2 h treatment with <b>MG-277</b> and three analogues (MC-215, MC-216 and MC-217) with weaker MDM2 binding affinity in RS4;11 cells. (G) RS4;11 cells were pre-incubated with a proteasome inhibitor, PR-171 or a Nedd8 inhibitor, MLN4924, or DMSO as control for 4 h to block proteasome or CRL function. The GSPT1 protein level was measured by western blotting after 2 h treatment with MI-2103, <b>MG-277</b>, or MD-222.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We investigated if degradation of GSPT1 protein induced by <b>MG-277</b> requires binding to MDM2. An excess of MI-2103 was found to fail to negate degradation of GSPT1 protein by <b>MG-277</b> in the MDA-MB-231 cell line (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>E). We evaluated the ability of several analogues of <b>MG-277</b> (MC-215, MC-216, and MC-217) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A–C), each of which contains a very weak MDM2 inhibitor, to reduce the level of GSPT1 protein in the RS4;11 cells. These compounds are all very potent and effective in reducing the GSPT1 protein level but have no obvious effect on the level of MDM2 protein (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figures S3D–F</a> and <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>F). Hence, the binding to MDM2 of <b>MG-277</b> and its analogues is not required for their effective degradation of GSPT1 protein.</div><div class="NLM_p last">We investigated whether proteasomal activity and Cullin-RING E3 ligases (CRLs) activity are essential for <b>MG-277-</b>induced GSPT1 degradation. Both PR-171, a proteasome inhibitor, and MLN4924, a Nedd8 E1 conjugating enzyme (NAE) inhibitor, completely block the degradation of GSPT1 induced by <b>MG-277</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>G). Hence, the GSPT1 degradation induced by <b>MG-277</b> requires the activity of both the CUL4 E3 ubiquitin ligase and the proteasome.</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Structural Requirements within the PROTAC Molecules for the GSPT1 Recruitment to Cereblon</h3><div class="NLM_p">To shed light on the structural requirements for <b>MG-277</b> to induce GSPT1 degradation, we modeled the binding of <b>MG-277</b> with the cereblon:GSPT1 complex based upon the co-crystal structure of CC-885 in a complex with cereblon and GSPT1 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(40)</a> Our modeling showed that <b>MG-277</b> captures some of the key interactions with residues uniquely critical for recruiting GSPT1 to cereblon that had been shown in the previous study.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(40)</a> Specifically, Phe150 in cereblon is hydrophobically packed with GPST1 and the cyclohexyl group of <b>MG-277</b>. Glu377 of cereblon enjoys hydrogen bonding and electrostatic interactions with residues in GSPT1 and the positively charged nitrogen atom in the pyrrolidine group of <b>MG-277</b>. Additional interactions among the ligand and the two proteins are also observed (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S4</a> for more details).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Molecular modeling of <b>MG-277</b> (yellow) binding with GSPT1 (blue) and cereblon (orange) with highlighted residues, Phe150 and Glu377, which are uniquely critical for recruitment of GSPT1. The available structure of GSPT1-cereblon complex co-crystalized with small ligand CC-885<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(40)</a> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HXB">5HXB</a>) was used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further evaluated the structural requirements for the potent cell activity and GSPT-1 degradation by <b>MG-277</b>. The reported DDB1-GSPT1-CC-885 complex shows hydrogen bonding interaction between the two urea nitrogens of CC-885 and cereblon.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(40)</a> We postulated that it may be possible that the adjacent carboxamide and pyrrolidine nitrogens in <b>MG-277</b> mimic this interaction. If this hypothesis is correct, blocking one of the nitrogens would abolish the activity. Blocking the nitrogen of the pyrrolidine produced MC-295 which retains the cellular activity, suggesting that this position plays a minor role in the GSPT-1 degradation (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Next, the substituents around the pyrrolidine core were systematically removed. Removal of the cyclohexyl ring on carbon-2, resulted in MC-296, which has the same activity as <b>MG-277</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). However, modification of the 4-aryl substituent produced compounds MC-297, MC-293, and MC-294, all having weak cell potency suggesting both aryl groups are necessary for GSPT-1 degradation and consequently potent cell growth-inhibitory activity (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Further studies will be performed to precisely define the minimal structural requirements for induced GSPT1 protein degradation.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Chemical structural requirements for the GSPT1 recruitment to cereblon. (A) Chemical structures of the new compounds after modifications of the inhibitor portion to investigate the structural requirement of GSPT1 recruitment. (B,C) Potencies of the compounds in cell growth inhibition were evaluated in both p53 wild-type RS4;11 cell line (B) and the p53 mutant RS4;11/IRMI-2 cell line (C) by a 4-day cell viability assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56866" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56866" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We previously reported the discovery of a class of bona fide PROTAC MDM2 degraders, exemplified by MD-222 and MD-224, that are highly potent and effective in inducing degradation of MDM2 and in activating wild-type p53 in cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> In the present study, we report the discovery of another type of compounds, exemplified by <b>MG-277</b> which was obtained by the removal of the benzoic acid moiety of the MDM2 inhibitor segment of MD-222. In contrast to MD-222 and MD-224, <b>MG-277</b> has a significantly decreased potency in reducing the level of MDM2 protein in cells and fails to activate wild-type p53 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A–C). However, <b>MG-277</b> is highly potent and effective in inhibition of cell growth in cancer cell lines with wild-type p53, mutated p53, or deleted p53, indicating a p53-independent mechanism. We found that <b>MG-277</b> and a number of its analogues achieve cell growth inhibition in a cereblon-dependent and MDM2-independent manner. Our unbiased proteomic analysis revealed that GSPT1 is the only protein whose level is dramatically and significantly reduced by <b>MG-277</b>. Furthermore, <b>MG-277</b> induces rapid GSPT1 degradation in cancer cells in a p53- and MDM2-independent manner but in a manner dependent upon cereblon, CUL4 E3 ubiquitin ligase, and proteasomes. Moreover, the potencies of different analogues of <b>MG-277</b> in induction of GSPT1 degradation correlate well with their cell growth inhibitory activity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Figure S3</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">Table S1</a>). Our data therefore strongly support our proposal that induction of GSPT1 degradation by <b>MG-277</b> and its analogues is responsible for their cell growth inhibitory activity.</div><div class="NLM_p">A recent study by Ishoey et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> has demonstrated that while phthalimide conjugate degraders fail to induce degradation of their consensus kinase targets in cells, these compounds effectively induce degradation of GSPT1 protein. Their study also provided convincing evidence that degradation of GSPT1 is responsible for the cell growth inhibitory activity of these phthalimide conjugate degraders in leukemia cell lines.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Our present study provides an example that while some of our designed putative MDM2 degraders such as MD-222 and MD-224 indeed function as bona fide PROTAC MDM2 degraders,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> structural modifications of the MDM2 inhibitor portion of MD-222 yielded compounds such <b>MG-277</b> which has a cellular mode of action completely different from that of MD-222. Despite the fact that <b>MG-277</b> contains an MDM2 inhibitor portion, which binds to MDM2 with a high affinity (IC<sub>50</sub> = 48.1 nM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), it is much less potent and effective in inducing degradation of MDM2 than MD-222 and fails to activate p53. Instead, <b>MG-277</b> achieves its remarkable cell growth inhibitory activity by inducing degradation of the GSPT1 protein.</div><div class="NLM_p last">Our data suggest that for any designed phthalimide conjugate degrader, there are two potential mechanisms of action. On the one hand, the phthalimide conjugate can function as a bona fide degrader to degrade the intended protein(s) of interest. On the other hand, the conjugate can function as a molecular glue to recruit neo-substrate protein(s) to cereblon and Cullin 4A for ubiquitination and subsequent degradation. Although further studies are needed to determine the precise structural requirements for a conjugate to operate either as a bona fide PROTAC degrader or as a molecular glue, both cellular mechanisms of action must be considered in the design of phthalimide conjugate degraders.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13084" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13084" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">By modification of our previously discovered bona fide PROTAC MDM2 degraders, we discovered a new type of compounds with a completely different mechanism of action. This new type of compounds, exemplified by <b>MG-277</b>, exerts potent anticancer activity by functioning as molecular glues. In contrast to our previously reported bona fide PROTAC MDM2 degraders, <b>MG-277</b> shows very weak activity in inducing degradation of MDM2 and fails to activate p53 but is potent and effective in inducing degradation of GSPT1. Our study shows that for a designed phthalimide conjugate degrader, it can function either as a bona fide degrader for the intended protein(s) of interest, or as a molecular glue which recruits neo-substrate protein(s) to CRL4<sup>CRBN</sup> E3 ligase for ubiquitination and subsequent degradation. Both mechanisms must be considered in the design and evaluation of a PROTAC degrader. Further investigation is ongoing to define the precise structural requirements for a phthalimide conjugate to operate either as a bona fide PROTAC degrader or as a molecular glue.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64164" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64164" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Spiroxindoles 1–7,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> 8<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and all amino-lenalidomide linkers<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> were obtained as previously described. Amide coupling of the respective cores and amino-lenalidomide linkers produced the target molecules as illustrated in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<named-content content-type="anchor" rid="sch2" type="simple"></named-content><named-content content-type="anchor" rid="sch3" type="simple"></named-content><a class="ref internalNav" href="#sch4" aria-label="4">4</a>.</div><figure id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Linker Modifications</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Removal of Halogens and Methylation of Pyrrolidine and Glutarimide Nitrogens</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Systematic Removal of Pyrrolidine Substituents</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Removal of Pyrrolidine Substituents</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06680" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06680" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Compound Synthesis</h3><div class="NLM_p">Unless otherwise stated, all commercial reagents were used as supplied without further purification. NMR spectra were obtained on a Bruker 400 Ascend spectrometer at a <sup>1</sup>H frequency of 400 MHz. Chemical shifts (δ) are reported in parts per million relative to an internal standard. The final products were purified on a preparative HPLC column (Waters 2545, Quaternary Gradient Module) with a SunFire Prep C18 OBD 5 μm 50 × 100 mm reverse-phase column. The mobile phase was a gradient of solvent A [H<sub>2</sub>O with 0.1% trifluoroacetic acid (TFA)] and solvent B (MeCN with 0.1% TFA) at a flow rate of 60 mL/min and 1%/min increase of solvent B. All final compounds have purity ≥ 95% as determined by Waters ACQUITY UPLC using a reverse-phase column (SunFire, C18-5 μm, 4.6 × 150 mm) and a solvent gradients of A (H<sub>2</sub>O with 0.1% of TFA) and B (MeCN with 0.1% of TFA). Mass spectral analyses were carried out with a Waters ACQUITY UPLC/QDa mass detector.</div><div class="NLM_p">All the compounds were synthesized according to the procedure described for <b>MG-277</b>.</div><div id="sec6_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(5-(2-(1-methyl-2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MC-024)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>42</sub>H<sub>45</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 788.28; found 788.31; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.69–7.61 (m, 2H), 7.50–7.35 (m, 4H), 7.17 (t, <i>J</i> = 8.0 Hz, 1H), 7.12 (dt, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.77 (d, <i>J</i> = 1.9 Hz, 1H), 5.21 (dt, <i>J</i> = 13.5, 4.8 Hz, 1H), 5.08 (d, <i>J</i> = 10.9 Hz, 1H), 4.75 (dd, <i>J</i> = 11.1, 2.5 Hz, 1H), 4.53–4.36 (m, 2H), 3.47–3.35 (m, 1H), 3.15 (s, 3H), 3.05–2.86 (m, 3H), 2.84–2.68 (m, 1H), 2.60 (t, <i>J</i> = 7.7 Hz, 2H), 2.50 (qd, <i>J</i> = 12.9, 5.5 Hz, 1H), 2.24–2.10 (m, 2H), 2.01–1.82 (m, 3H), 1.76 (d, <i>J</i> = 13.3 Hz, 2H), 1.61–1.14 (m, 8H), 1.14–1.00 (m, 2H).</div></div><div id="sec6_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (3′<i>R</i>,4′<i>R</i>,5′<i>R</i>)-6″-Chloro-<i>N</i>-(5-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-2″-oxo-4′-phenyldispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MC-215)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>41</sub>H<sub>45</sub>ClN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 722.30; found 722.41; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.66 (t, <i>J</i> = 7.2 Hz, 1H), 7.55 (dd, <i>J</i> = 8.2, 5.6 Hz, 1H), 7.52–7.42 (m, 1H), 7.40 (ddd, <i>J</i> = 7.6, 2.7, 1.1 Hz, 1H), 7.26–7.08 (m, 6H), 6.71 (dd, <i>J</i> = 2.0, 0.9 Hz, 1H), 5.27–5.15 (m, 2H), 4.54–4.36 (m, 2H), 4.25 (dd, <i>J</i> = 23.8, 11.2 Hz, 1H), 3.04–2.86 (m, 2H), 2.86–2.71 (m, 2H), 2.61–2.45 (m, 3H), 2.27–2.10 (m, 2H), 2.02–1.80 (m, 3H), 1.82–1.70 (m, 2H), 1.58–1.14 (m, 8H), 1.11–0.97 (m, 2H).</div></div><div id="sec6_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (3′<i>R</i>,4′<i>R</i>,5′<i>R</i>)-<i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-2″-oxo-4′-phenyldispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MC-216)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>41</sub>H<sub>46</sub>N<sub>5</sub>O<sub>5</sub><sup>+</sup> = 688.35; found 688.45; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.66 (t, <i>J</i> = 7.6 Hz, 1H), 7.60–7.52 (m, 1H), 7.52–7.42 (m, 1H), 7.40 (ddd, <i>J</i> = 7.6, 2.4, 1.1 Hz, 1H), 7.28–7.05 (m, 7H), 6.69 (d, <i>J</i> = 7.8 Hz, 1H), 5.28–5.14 (m, 2H), 4.53–4.37 (m, 2H), 4.25 (dd, <i>J</i> = 24.0, 11.1 Hz, 1H), 3.05–2.86 (m, 3H), 2.85–2.73 (m, 2H), 2.60–2.46 (m, 2H), 2.26–2.13 (m, 2H), 2.00–1.85 (m, 3H), 1.81–1.69 (m, 2H), 1.58–1.13 (m, 8H), 1.11–0.98 (m, 2H).</div></div><div id="sec6_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-<i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-4′-(2-fluorophenyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MC-217)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>41</sub>H<sub>45</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 706.34; found 706.50; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.72 (td, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.66 (dd, <i>J</i> = 7.4, 1.3 Hz, 1H), 7.51–7.39 (m, 3H), 7.26–7.17 (m, 2H), 7.14 (t, <i>J</i> = 7.7 Hz, 1H), 7.07 (t, <i>J</i> = 7.7 Hz, 1H), 6.90–6.79 (m, 1H), 6.71 (d, <i>J</i> = 7.8 Hz, 1H), 5.24–5.14 (m, 1H), 4.77 (d, <i>J</i> = 10.8 Hz, 1H), 4.55–4.37 (m, 2H), 3.08–2.86 (m, 2H), 2.86–2.66 (m, 2H), 2.60 (t, <i>J</i> = 7.8 Hz, 2H), 2.58–2.43 (m, 1H), 2.25–2.08 (m, 2H), 1.99–1.80 (m, 3H), 1.80–1.65 (m, 2H), 1.64–1.24 (m, 7H), 1.24–1.00 (m, 4H).</div></div><div id="sec6_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MG-274)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>36</sub>H<sub>33</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 704.19; found 704.34; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.78–7.63 (m, 3H), 7.60–7.51 (m, 2H), 7.38 (t, <i>J</i> = 7.5 Hz, 1H), 7.18 (t, <i>J</i> = 8.1 Hz, 1H), 7.10 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.79 (t, <i>J</i> = 1.6 Hz, 1H), 5.34–5.21 (m, 1H), 5.21–5.10 (m, 1H), 4.29–3.95 (m, 3H), 3.02–2.86 (m, 2H), 2.86–2.74 (m, 1H), 2.76–2.63 (m, 1H), 2.43–2.28 (m, 1H), 2.23–2.08 (m, 2H), 1.97–1.69 (m, 5H), 1.65–1.50 (m, 2H), 1.25–1.12 (m, 2H).</div></div><div id="sec6_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(3-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)propyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MG-275)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>39</sub>H<sub>39</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 746.23; found 746.45; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.73–7.58 (m, 2H), 7.50 (dd, <i>J</i> = 8.3, 2.5 Hz, 1H), 7.43 (td, <i>J</i> = 7.6, 2.1 Hz, 1H), 7.34 (dddd, <i>J</i> = 8.6, 7.3, 5.7, 1.6 Hz, 1H), 7.24 (ddd, <i>J</i> = 7.6, 5.4, 1.0 Hz, 1H), 7.18 (tt, <i>J</i> = 8.1, 1.2 Hz, 1H), 7.13–7.06 (m, 1H), 6.78 (d, <i>J</i> = 1.9 Hz, 1H), 5.26–5.11 (m, 2H), 4.82 (d, <i>J</i> = 11.2 Hz, 1H), 4.51–4.29 (m, 2H), 3.50–3.34 (m, 1H), 3.12–2.99 (m, 1H), 2.99–2.74 (m, 3H), 2.57–2.36 (m, 3H), 2.27–2.09 (m, 2H), 2.01–1.83 (m, 3H), 1.82–1.64 (m, 4H), 1.50 (dd, <i>J</i> = 15.3, 11.9 Hz, 1H), 1.21 (td, <i>J</i> = 13.8, 4.0 Hz, 2H).</div></div><div id="sec6_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(4-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)butyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MG-276)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>40</sub>H<sub>41</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 760.25; found 760.51; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.69–7.58 (m, 2H), 7.54–7.41 (m, 2H), 7.41–7.31 (m, 1H), 7.31–7.06 (m, 3H), 6.77 (d, <i>J</i> = 2.0 Hz, 1H), 5.22–5.06 (m, 2H), 4.77 (d, <i>J</i> = 11.1 Hz, 1H), 4.52–4.34 (m, 2H), 3.47–3.33 (m, 1H), 3.27–3.11 (m, 1H), 3.11–2.99 (m, 1H), 3.00–2.84 (m, 1H), 2.85–2.73 (m, 2H), 2.63–2.45 (m, 3H), 2.25–2.11 (m, 2H), 2.01–1.82 (m, 3H), 1.76 (d, <i>J</i> = 13.4 Hz, 2H), 1.58–1.30 (m, 5H), 1.29–1.11 (m, 2H).</div></div><div id="sec6_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(5-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (<b>MG-277</b>)</h4><div class="NLM_p last">HATU (9 mg, 0.0237 mmol) was added to a solution of <b>1</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (10 mg, 0.0216 mmol), <b>2</b><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (10 mg, 0.0226 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (16 μL, 0.094 mmol) in dimethylformamide (0.5 mL). After 10 min at rt, the reaction was complete as determined by a UPLC/QDa mass detector. The reaction was diluted with MeOH (0.5 mL), acidified with TFA, then H<sub>2</sub>O (1 mL) was added, and this solution was injected into a preparative HPLC for purification. The HPLC fractions containing the product were partially concentrated and the remaining solution was frozen and lyophilized to produce 6.4 mg of <b>MG-277</b> as a white powder. LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>41</sub>H<sub>43</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 774.26; Found 774.42; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.65 (d, <i>J</i> = 7.3 Hz, 2H), 7.53–7.38 (m, 3H), 7.40–7.31 (m, 1H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 7.10 (dt, <i>J</i> = 8.2, 2.4 Hz, 1H), 6.77 (d, <i>J</i> = 2.0 Hz, 1H), 5.19 (dt, <i>J</i> = 13.3, 5.5 Hz, 1H), 5.08 (d, <i>J</i> = 11.0 Hz, 1H), 4.75 (d, <i>J</i> = 11.0 Hz, 1H), 4.55–4.38 (m, 2H), 3.45–3.33 (m, 1H), 3.08–2.86 (m, 2H), 2.85–2.65 (m, 2H), 2.65–2.45 (m, 3H), 2.40–2.30 (m, 1H), 2.25–2.07 (m, 2H), 1.99–1.80 (m, 3H), 1.79–1.67 (m, 2H), 1.60–1.26 (m, 5H), 1.23–1.04 (m, 3H).</div></div><div id="sec6_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(2-(3-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)propoxy)ethyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MG-278)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>41</sub>H<sub>43</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>6</sub><sup>+</sup> = 790.26; found 790.23; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.69–7.60 (m, 2H), 7.53–7.44 (m, 2H), 7.41 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.27 (dddd, <i>J</i> = 23.7, 8.5, 7.1, 1.5 Hz, 1H), 7.17–7.06 (m, 2H), 6.76 (dd, <i>J</i> = 1.9, 0.8 Hz, 1H), 5.25–5.12 (m, 1H), 4.79 (d, <i>J</i> = 11.0 Hz, 1H), 4.55–4.35 (m, 2H), 3.58 (dtd, <i>J</i> = 13.9, 6.2, 4.1 Hz, 1H), 3.38–3.09 (m, 5H), 3.02–2.85 (m, 2H), 2.84–2.76 (m, 2H), 2.63 (td, <i>J</i> = 7.5, 2.3 Hz, 2H), 2.53 (qd, <i>J</i> = 13.2, 4.6 Hz, 1H), 2.24–2.12 (m, 2H), 2.00–1.87 (m, 3H), 1.81–1.68 (m, 4H), 1.51 (q, <i>J</i> = 13.8 Hz, 1H), 1.25–1.15 (m, 2H).</div></div><div id="sec6_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(2-(2-(3-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)propoxy)ethoxy)ethyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MG-279)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>43</sub>H<sub>47</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>7</sub><sup>+</sup> = 834.28; found 834.35; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.67 (dd, <i>J</i> = 6.4, 2.3 Hz, 1H), 7.65–7.58 (m, 1H), 7.57–7.41 (m, 3H), 7.29 (dd, <i>J</i> = 8.0 Hz, 1H), 7.19–7.01 (m, 2H), 6.75 (d, <i>J</i> = 2.0 Hz, 1H), 5.19 (dd, <i>J</i> = 13.3, 5.2 Hz, 1H), 4.75 (d, <i>J</i> = 10.4 Hz, 1H), 4.56–4.37 (m, 2H), 3.66–3.31 (m, 12H), 3.00–2.86 (m, 1H), 2.86–2.72 (m, 3H), 2.56–2.45 (m, 1H), 2.22–2.13 (m, 1H), 2.08–1.61 (m, 8H), 1.58–1.45 (m, 1H), 1.22–1.01 (m, 2H).</div></div><div id="sec6_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(2-(2-(2-(3-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)propoxy)ethoxy)ethoxy)ethyl)-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MG-280)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>45</sub>H<sub>51</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>8</sub><sup>+</sup> = 878.31; found 878.65; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.73–7.61 (m, 2H), 7.55–7.45 (m, 3H), 7.35 (t, <i>J</i> = 7.5 Hz, 1H), 7.20–7.04 (m, 2H), 6.78 (d, <i>J</i> = 1.9 Hz, 1H), 5.21 (dt, <i>J</i> = 13.3, 4.8 Hz, 1H), 4.79 (d, <i>J</i> = 10.7 Hz, 1H), 4.60–4.40 (m, 2H), 3.74–3.38 (m, 13H), 3.33–3.18 (m, 2H), 2.95 (ddd, <i>J</i> = 17.6, 13.5, 5.4 Hz, 1H), 2.88–2.77 (m, 3H), 2.75–2.61 (m, 1H), 2.61–2.45 (m, 1H), 2.25–2.16 (m, 1H), 2.09 (d, <i>J</i> = 13.8 Hz, 1H), 2.02–1.67 (m, 7H), 1.55 (q, <i>J</i> = 13.9 Hz, 1H), 1.27–1.05 (m, 2H).</div></div><div id="sec6_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>R</i>,4<i>R</i>)-<i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-4-phenylpyrrolidine-2-carboxamide (MC-293)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O<sub>4</sub><sup>+</sup> = 503.26; found 503.17; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.60 (ddd, <i>J</i> = 7.4, 3.1, 1.3 Hz, 1H), 7.48–7.34 (m, 4H), 7.34–7.25 (m, 3H), 5.17 (td, <i>J</i> = 13.0, 5.2 Hz, 1H), 4.57–4.39 (m, 3H), 3.87–3.77 (m, 1H), 3.53–3.40 (m, 1H), 3.29–3.16 (m, 2H), 2.98–2.84 (m, 1H), 2.84–2.68 (m, 3H), 2.60–2.38 (m, 2H), 2.38–2.25 (m, 1H), 2.25–2.14 (m, 1H), 1.83–1.51 (m, 4H), 1.47–1.32 (m, 2H).</div></div><div id="sec6_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (2<i>R</i>,4<i>S</i>)–<i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-4-phenylpyrrolidine-2-carboxamide (MC-294)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O<sub>4</sub><sup>+</sup> = 503.26; found 503.17; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.57 (ddd, <i>J</i> = 7.5, 3.0, 1.2 Hz, 1H), 7.44 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.42–7.34 (m, 3H), 7.36–7.24 (m, 3H), 5.16 (ddd, <i>J</i> = 13.5, 8.6, 5.2 Hz, 1H), 4.57–4.28 (m, 3H), 3.82–3.71 (m, 1H), 3.69–3.53 (m, 1H), 3.40–3.33 (m, 1H), 3.28–3.17 (m, 1H), 2.97–2.69 (m, 5H), 2.53 (qd, <i>J</i> = 13.2, 4.7 Hz, 1H), 2.24–2.13 (m, 1H), 2.01–1.87 (m, 1H), 1.80–1.52 (m, 5H), 1.49–1.32 (m, 2H).</div></div><div id="sec6_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (3′<i>R</i>,4′<i>S</i>,5′<i>R</i>)-6″-Chloro-4′-(3-chloro-2-fluorophenyl)-<i>N</i>-(5-(2-(2,6-dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-1′-methyl-2″-oxodispiro[cyclohexane-1,2′-pyrrolidine-3′,3″-indoline]-5′-carboxamide (MC-295)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>42</sub>H<sub>45</sub>Cl<sub>2</sub>FN<sub>5</sub>O<sub>5</sub><sup>+</sup> = 788.27; found 788.19; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.66 (dd, <i>J</i> = 7.4, 1.2 Hz, 1H), 7.60–7.50 (m, 2H), 7.50–7.40 (m, 2H), 7.36 (q, <i>J</i> = 6.7 Hz, 1H), 7.16–7.09 (m, 2H), 6.77 (dd, <i>J</i> = 2.0, 0.8 Hz, 1H), 5.19 (dt, <i>J</i> = 13.3, 5.0 Hz, 1H), 5.02–4.93 (m, 3H), 4.66 (dd, <i>J</i> = 11.9, 4.4 Hz, 1H), 4.54–4.37 (m, 2H), 3.44–3.33 (m, 2H), 3.25 (d, <i>J</i> = 4.6 Hz, 3H), 3.12–2.97 (m, 1H), 2.99–2.87 (m, 1H), 2.85–2.75 (m, 1H), 2.65–2.43 (m, 4H), 2.31–2.10 (m, 3H), 1.81–1.45 (m, 7H), 1.45–1.27 (m, 2H), 1.27–1.15 (m, 1H), 1.15–0.95 (m, 2H).</div></div><div id="sec6_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (3<i>R</i>,4′<i>R</i>,5′<i>R</i>)-<i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-2′,2′-dimethyl-2-oxo-4′-phenylspiro[indoline-3,3′-pyrrolidine]-5′-carboxamide (MC-296)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>38</sub>H<sub>42</sub>N<sub>5</sub>O<sub>5</sub><sup>+</sup> = 648.31; found 648.24; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.68–7.63 (m, 1H), 7.50–7.43 (m, 1H), 7.44–7.35 (m, 2H), 7.26–7.02 (m, 7H), 6.73 (dd, <i>J</i> = 18.1, 7.8 Hz, 1H), 5.25–5.14 (m, 1H), 5.02–4.94 (m, 1H), 4.53–4.25 (m, 3H), 3.41–3.34 (m, 1H), 3.08–2.96 (m, 1H), 2.96–2.87 (m, 1H), 2.85–2.75 (m, 1H), 2.64–2.45 (m, 3H), 2.26–2.13 (m, 1H), 1.94–1.80 (m, 3H), 1.59–1.44 (m, 2H), 1.44–1.19 (m, 4H), 1.14–0.97 (m, 2H).</div></div><div id="sec6_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (3<i>R</i>,5′<i>R</i>)-<i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-2-oxospiro[indoline-3,3′-pyrrolidine]-5′-carboxamide (MC-297)</h4><div class="NLM_p last">LC–MS (ESI) <i>m</i>/<i>z</i> (MH)<sup>+</sup> calcd for C<sub>30</sub>H<sub>34</sub>N<sub>5</sub>O<sub>5</sub><sup>+</sup> = 544.25; found 544.15; >95% purity. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.63–7.54 (m, 1H), 7.45–7.35 (m, 3H), 7.36–7.26 (m, 1H), 7.11 (qd, <i>J</i> = 7.5, 1.1 Hz, 1H), 7.00–6.91 (m, 1H), 5.18 (td, <i>J</i> = 13.7, 5.2 Hz, 1H), 4.56–4.39 (m, 2H), 3.76–3.63 (m, 2H), 3.40–3.33 (m, 1H), 3.28–3.17 (m, 1H), 3.00–2.85 (m, 1H), 2.86–2.74 (m, 2H), 2.76–2.44 (m, 4H), 2.34–2.22 (m, 1H), 2.25–2.13 (m, 1H), 1.78–1.49 (m, 4H), 1.43–1.32 (m, 2H).</div></div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> FP-Based Protein Binding Assay</h3><div class="NLM_p last">The binding affinity of the compounds to MDM2 protein was determined by an optimized, sensitive, and quantitative FP-based binding assay using a recombinant human His-tagged MDM2 protein (residues 1–118) and a FAM-tagged p53-based peptide as the fluorescent probe, as described previously.<a onclick="showRef(event, 'ref32 ref43'); return false;" href="javascript:void(0);" class="ref ref32 ref43">(32,48)</a></div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Cell Lines</h3><div class="NLM_p last">The human leukemia RS4;11 cell lines (ATCC CRL-1873), MV-4-11 (ATCC CRL-9591), HL-60 (ATCC CCL-240) and breast cancer cell lines MDA-MB-468 (ATCC HTB-132) and MDA-MB-231 (ATCC HTB-26) were purchased from the American Type Culture Collection (ATCC). MOLM-13 (ACC 554, DSMZ) cell line was purchased from DSMZ. RS4;11/IRMI-2 cell line is an induced MDM2 inhibitor-resistant cell line generated from p53 wild-type RS4;11 cells by repeated treatment with a MDM2 inhibitor, SAR405838 (MI-73301) as described in our earlier study.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> The p53 mutations (Y236 Y/H, R249 R/G) in RS4;11/IRMI-2 cells were confirmed by DNA sequencing.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> In all experiments, RS4;11, MOLM-13, MDA-MB-468, and MDA-MB-231 cells were cultured in RPMI 1640 media (Cat#11875119, ThermoFisher), MV-4-11 and HL-60 cells were cultured in IMDM media (Cat# 12440061, ThermoFisher), supplemented with 10% HyClone fetal bovine serum (Cat#SH30910.03, GE Healthcare Life Sciences) and 1% penicillin–streptomycin (Cat#15140122, Life Technologies) at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub> and were used within three months of thawing fresh vials.</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Cell Growth Inhibition Assay</h3><div class="NLM_p last">In cell growth inhibition experiments, cells were seeded in 96-well cell culture plates at densities of 10 000–20 000 cells/well for leukemia cell lines and 5000–10 000 cells/well for breast cancer cell lines in 100 μL of culture medium. Each compound that was tested was serially diluted in the culture medium, and 100 μL of the diluted solution containing the tested compound was added to the appropriate wells of the cell plate. After addition of the tested compound, the cells were incubated for 4 days at 37 °C in an atmosphere of 5% CO<sub>2</sub>. Cell growth inhibition was then evaluated by a lactate dehydrogenase-based colorimetric assay (Dojindo Molecular Technologies) using a water soluble tetrazolium salt, WST-8.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(48)</a> After the WST-8 reagent was added (20 μL/well), the plates were incubated for at least 1 h, and read at 450 nm using Tecan Infinite M1000 multimode microplate reader (Tecan, Morrisville, NC).</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Western Blotting Assay and Quantitative Proteomics Profiling</h3><div class="NLM_p">For western blot analysis, 2 × 10<sup>6</sup> cells/well were added into 6-well plates and treated with compounds at the indicated concentrations for various lengths of time. Cells were then collected and lysed in ice-cold Cell Lysis Buffer (Cell Signaling Technology, #9803) containing a protease inhibitor. Lysate protein (25 μg) was run in each lane of a PAGE-SDS and blotted into polyvinylidene fluoride membranes. Expression levels of the indicated proteins were probed by the following primary antibodies: MDM2 (SMP14) (sc-965, Santa Cruz), p53 (DO-1) (sc-126, Santa Cruz), HRP-conjugated GAPDH (sc-25778 HRP, sc-47724 HRP, Santa Cruz) from Santa Cruz, and eRF3/GSPT1 (ab126090, ab49878) from Abcam.</div><div class="NLM_p last">Total protein samples after treatment with indicated compounds in RS4;11 and RS4;11/IRMI-2 cell lines were prepared with ice-cold RIPA buffer containing protease inhibitor. The abundance of 5264 cellular proteins in each sample was analyzed by high-resolution Orbitrap liquid chromatography mass spectrometry (LC–MS/MS) using TMT10plex Isobaric Label Reagent Set (Cat#90111, ThermoFisher) for protein labeling.</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Quantitative Real-Time Polymerase Chain Reaction</h3><div class="NLM_p last">Total RNA was prepared from RS4;11 cells after the indicated treatments using the RNeasy Mini Kit (74106; QIAGEN) according to the manufacturer’s protocol. RNA concentration in each sample was determined with a Tecan Infinite M1000 multimode microplate reader (Tecan, Morrisville, NC). cDNA was generated using High-Capacity RNA-to-cDNA Kit (4387406; ThermoFisher Scientific). qRT-PCR was performed in triplicate with <i>MDM2, p21, Puma, Bax,</i> and <i>GAPDH</i>-specific primer pairs using the Applied Biosystems QuantStudio 7 Flex Real-Time PCR System with TaqMan Fast Advanced Master Mix (Cat#4444557; Applied Biosystems). Target gene expression levels were normalized to GAPDH levels, made relative to untreated control samples and presented as 2<sup>–ΔΔ<i>C</i><sub>t</sub></sup>. All primers used for qRT-PCR were purchased from ThermoFisher and as follows: MDM2 (Hs01066930_m1; ThermoFisher); p21 (Hs00355782_m1; ThermoFisher); Puma (Hs00248075_m1; ThermoFisher); Bax (Hs00180269_m1; ThermoFisher), GSPT1 (Hs01093019_m1; ThermoFisher) and GAPDH (Hs02758991_g1; ThermoFisher).</div></div><div id="sec6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Small Interfering RNA (siRNA) Knockdown</h3><div class="NLM_p last">To downregulate MDM2 and cereblon, ON-TARGETplus siRNAs (SMARTpool) for human MDM2 (LU-003279-00-0005; Dharmacon), ON-TARGETplus siRNAs (individual) for cereblon (LQ-021086-10-0002, LQ-021086-11-0002; Dharmacon), and nontargeting negative control siRNAs (D-001810-10-20; Dharmacon) were used. Transfection was performed using Lipofectamine RNAiMAX (Cat#13778150, ThermoFisher). MDA-MB-231 and MDA-MB-468 cells were transfected with MDM2 or cereblon siRNAs or nontargeting negative control siRNAs for 2 days, followed by indicated drug treatment.</div></div><div id="sec6_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Quantification and Statistical Analysis</h3><div class="NLM_p last">All the cell data of protein binding assay, cell growth inhibition assay and qRT-PCR were presented as mean ± standard error from at least three independent experiments. Statistical analyses presented in all figures were performed using GraphPad Prism software (version 7.00).</div></div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Molecular Modeling</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81550" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81550" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cocrystal structures of cereblon-GSPT1 in complex with ligand CC-885 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HXB">5HXB</a>) were used to model the binding poses of MG-277 cereblon-GSPT1 complex.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(40)</a> The protein structure was prepared using Molecular Operating Environment (MOE) structure preparation (adding missing residues, assigning physiological relevant protonation states with protonate_3D).<a onclick="showRef(event, 'ref144'); return false;" href="javascript:void(0);" class="ref ref144">(43)</a> In the induced-fit docking, Ligand was firstly placed with the guidance of a 3-sphere pharmacophore, which reserves the hydrogen bond between the lenalidomide moiety and cereblon. These binding poses were then refined in the induced-fit mode. GBVI/WSA dG scoring was selected to rank and select the docking pose.</div><div class="NLM_p last">The binding mode was then submitted to run an MD simulation (10 ns) in AMBER 18<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(44)</a> for further refinement. The system was firstly solvated using TIP3P solvation model<a onclick="showRef(event, 'ref146'); return false;" href="javascript:void(0);" class="ref ref146">(45)</a> and neutralized by adding counter ions. A 12 000 steps minimization was given, with the first 3000 steps in steepest descent followed by conjugate gradient. The temperature of the system was gradually raised to 310 K in a 0.2 ns <i>NVT</i> simulation with a 5 (kcal/mol)/Å<sup>2</sup> force constraint given to the protein−ligand complex but not water molecules. Another 0.2 ns <i>NPT</i> simulation with the pressure set to 1 atm was performed with the same constraint, followed by a 2 ns relaxation to gradually reduce the force constant to 0. A 10 ns unconstrained simulation was then performed. The SHAKE algorithm<a onclick="showRef(event, 'ref147'); return false;" href="javascript:void(0);" class="ref ref147">(46)</a> was applied to fix the bond vibration involved in hydrogens, and the PME method<a onclick="showRef(event, 'ref148'); return false;" href="javascript:void(0);" class="ref ref148">(47)</a> was used to consider the long-range nonbonding interactions with cutoff of 10 Å.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00846" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50782" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50782" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00846" class="ext-link">10.1021/acs.jmedchem.9b00846</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Cell growth inhibitory activity of the MDM2 inhibitors, bona fide MDM2 degraders, MD-222, and the new putative MDM2 degrader <b>MG-277</b> in p53 wild-type and p53 mutated/deleted cell lines; potency of different lenalidomide conjugated analogues of <b>MG-277</b> in cell growth inhibition and GSPT1 degradation; determination of the half maximal degradation concentration (DC<sub>50</sub>) of GSPT1 degradation by different analogues of <b>MG-277</b> in RS4;11 cell line after a 24 h treatment; correlation between GSPT1 degradation DC<sub>50</sub> of the compounds in RS4;11 cells and IC<sub>50</sub> in growth inhibition in RS4;11 and IRMI-2 cell lines; detailed illustration of the predicted binding mode of <b>MG-277</b> to the cereblon-GSPT1 complex; NMR spectra of <b>MG-277</b>; mass spectrum and purity spectrum of <b>MG-277</b>; CSV file containing SMILES, binding and cellular data for final target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">JMC, SMILES, molecular glue paper 091719 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Atomic coordinates of models MG277, GSPT1, and CRBN (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf">jm9b00846_si_001.pdf (2.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_002.csv">jm9b00846_si_002.csv (2.4 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_003.pdb">jm9b00846_si_003.pdb (729.72 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00846" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, 
    Department of Internal Medicine, 
    Department of Medicinal
Chemistry, 
    Rogel Cancer Center, 
    , , , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#77041f16181a12191037021a1e141f59121302"><span class="__cf_email__" data-cfemail="017269606e6c646f6641746c6862692f646574">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiuling Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, 
    Department of Internal Medicine, 
    Rogel Cancer Center, 
    , , , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yangbing Li</span> - <span class="hlFld-Affiliation affiliation">Department of Internal Medicine, 
    Department of Medicinal
Chemistry, 
    Rogel Cancer Center, 
    , , , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angelo Aguilar</span> - <span class="hlFld-Affiliation affiliation">Department of Internal Medicine, 
    Rogel Cancer Center, 
    , , , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaomin Liu</span> - <span class="hlFld-Affiliation affiliation">Department of Internal Medicine, 
    Rogel Cancer Center, 
    , , , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao-Yie Yang</span> - <span class="hlFld-Affiliation affiliation">Department of Internal Medicine, 
    Rogel Cancer Center, 
    , , , and , University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.Y., Y.L., and A.A. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38531" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38531" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful for the financial support from the National Cancer Institute, NIH (1R01CA208267 to S.W.), the University of Michigan Rogel Cancer Center support grant from the National Cancer Institute, NIH (P30 CA046592) and the Endowment for the Basic Sciences (EBS) EDGE Student Fellowship from the University of Michigan Medical School (to J.Y.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">MDM2</td><td class="NLM_def"><p class="first last">human murine double minute 2</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">GSPT1</td><td class="NLM_def"><p class="first last">G1 to S phase transition 1</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">PUMA</td><td class="NLM_def"><p class="first last">p53 upregulated modulator of apoptosis</p></td></tr><tr><td class="NLM_term">MetAP-2</td><td class="NLM_def"><p class="first last">methionine aminopeptidase-2</p></td></tr><tr><td class="NLM_term">IMiDs</td><td class="NLM_def"><p class="first last">immunomodulatory imide drugs</p></td></tr><tr><td class="NLM_term">CRL4<sup>CRBN</sup></td><td class="NLM_def"><p class="first last">CUL4-DDB1-RBX1-CRBN E3 ligase complex</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence-polarization</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">quantitative reverse transcription polymerase chain reaction</p></td></tr><tr><td class="NLM_term">CRLs</td><td class="NLM_def"><p class="first last">Cullin-RING E3 ligases</p></td></tr><tr><td class="NLM_term">NAE</td><td class="NLM_def"><p class="first last">Nedd8 E1 conjugating enzyme</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51778" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51778" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lh6laZvI5tblA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: An Emerging Targeting Technique for Protein Degradation in Drug Discovery</span>. <i>Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1700247</span>, <span class="refDoi"> DOI: 10.1002/bies.201700247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fbies.201700247" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2018&pages=1700247&author=S.+Guauthor=D.+Cuiauthor=X.+Chenauthor=X.+Xiongauthor=Y.+Zhao&title=PROTACs%3A+An+Emerging+Targeting+Technique+for+Protein+Degradation+in+Drug+Discovery&doi=10.1002%2Fbies.201700247"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fbies.201700247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbies.201700247%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520An%2520Emerging%2520Targeting%2520Technique%2520for%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%26date%3D2018%26volume%3D40%26spage%3D1700247%26doi%3D10.1002%2Fbies.201700247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Waste Disposal-An Attractive Strategy for Cancer Therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1126/science.aam7340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.aam7340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28302825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1163-1167&author=J.+Salamiauthor=C.+M.+Crews&title=Waste+Disposal-An+Attractive+Strategy+for+Cancer+Therapy&doi=10.1126%2Fscience.aam7340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Waste disposal-An attractive strategy for cancer therapy</span></div><div class="casAuthors">Salami, Jemilat; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Targeted therapies for cancer are typically small mols. or monoclonal antibodies that act by inhibiting the activity of specific proteins that drive tumor growth.  Although many of these drugs are effective in cancer patients, the response is often not durable because tumor cells develop resistance to the drugs.  Another limitation of this strategy is that not all oncogenic driver proteins are "druggable" enzymes or receptors with activities that can be inhibited.  Here the authors describe an alternative approach to targeted therapy that is based on co-opting the cellular quality-control machinery-the ubiquitin-proteasome system-to remove specific cancer-causing proteins from the cell.  The authors first discuss examples of existing cancer drugs that work by degrading specific proteins and then review recent progress in the rational design and preclin. testing of small mols. that induce selective degrdn. of specific target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVoZn-YpfXmbVg90H21EOLACvtfcHk0lhi8LYDc58ugg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D&md5=918621d50a82b7797aa0300537a4486e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7340%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DWaste%2520Disposal-An%2520Attractive%2520Strategy%2520for%2520Cancer%2520Therapy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.aam7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras%3A+Induced+Protein+Degradation+as+a+Therapeutic+Strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lhi8LYDc58ugg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%253A%2520Induced%2520Protein%2520Degradation%2520as%2520a%2520Therapeutic%2520Strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by PROTACs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.pharmthera.2017.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28223226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=138-144&author=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=C.+M.+Crews&title=Targeted+Protein+Degradation+by+PROTACs&doi=10.1016%2Fj.pharmthera.2017.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by PROTACs</span></div><div class="casAuthors">Neklesa, Taavi K.; Winkler, James D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-144</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using the PROTAC technol. is emerging as a novel therapeutic method to address diseases driven by the aberrant expression of a disease-causing protein.  PROTAC mols. are bifunctional small mols. that simultaneously bind a target protein and an E3-ubiquitin ligase, thus causing ubiquitination and degrdn. of the target protein by the proteasome.  Like small mols., PROTAC mols. possess good tissue distribution and the ability to target intracellular proteins.  Herein, we highlight the advantages of protein degrdn. using PROTACs, and provide specific examples where degrdn. offers therapeutic benefit over classical enzyme inhibition.  Foremost, PROTACs can degrade proteins regardless of their function.  This includes the currently "undruggable" proteome, which comprises approx. 85% of all human proteins.  Other beneficial aspects of protein degrdn. include the ability to target overexpressed and mutated proteins, as well as the potential to demonstrate prolonged pharmacodynamics effect beyond drug exposure.  Lastly, due to their catalytic nature and the pre-requisite ubiquitination step, an exquisitely potent mols. with a high degree of degrdn. selectivity can be designed.  Impressive preclin. in vitro and in vivo PROTAC data have been published, and these data have propelled the development of clin. viable PROTACs.  With the mol. wt. falling in the 700-1000 Da range, the delivery and bioavailability of PROTACs remain the largest hurdles on the way to the clinic.  Solving these issues and demonstrating proof of concept clin. data will be the focus of many labs over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4RSLWYqqBMrVg90H21EOLACvtfcHk0lhi8LYDc58ugg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D&md5=08a74402a274a13c8c484cd68167a5a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520PROTACs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D174%26spage%3D138%26epage%3D144%26doi%3D10.1016%2Fj.pharmthera.2017.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced Protein Degradation: An Emerging Drug Discovery Paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+Protein+Degradation%3A+An+Emerging+Drug+Discovery+Paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lgG-_HuuImN8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520Protein%2520Degradation%253A%2520An%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+Protein+Degradation%3A+from+Chemical+Biology+to+Drug+Discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lgG-_HuuImN8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%253A%2520from%2520Chemical%2520Biology%2520to%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: New Approaches to Protein Degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+New+Approaches+to+Protein+Degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lgG-_HuuImN8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520New%2520Approaches%2520to%2520Protein%2520Degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span> <span> </span><span class="NLM_article-title">ARV-330: Androgen Receptor PROTAC Degrader for Prostate Cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">267</span>, <span class="refDoi"> DOI: 10.1200/jco.2016.34.2_suppl.267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1200%2Fjco.2016.34.2_suppl.267" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=267&author=T.+K.+Neklesaauthor=M.+Jinauthor=A.+P.+Crewauthor=A.+K.+Rossiauthor=R.+R.+Willardauthor=H.+Dongauthor=K.+Siuauthor=J.+Wangauthor=D.+A.+Gordonauthor=X.+Chenauthor=C.+Ferraroauthor=C.+M.+Crewsauthor=K.+Colemanauthor=J.+D.+Winkler&title=ARV-330%3A+Androgen+Receptor+PROTAC+Degrader+for+Prostate+Cancer&doi=10.1200%2Fjco.2016.34.2_suppl.267"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1200%2Fjco.2016.34.2_suppl.267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2016.34.2_suppl.267%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRossi%26aufirst%3DA.%2BK.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26atitle%3DARV-330%253A%2520Androgen%2520Receptor%2520PROTAC%2520Degrader%2520for%2520Prostate%2520Cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D34%26spage%3D267%26doi%3D10.1200%2Fjco.2016.34.2_suppl.267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhCY73upQ3iUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+Small+Molecule+Induced+Degradation+of+the+BET+Bromodomain+Protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhCY73upQ3iUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520Small%2520Molecule%2520Induced%2520Degradation%2520of%2520the%2520BET%2520Bromodomain%2520Protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+Von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0lhpEoEm-LN18Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span>. <i>BCR-ABL</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+Design+for+the+Degradation+of+Oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520Design%2520for%2520the%2520Degradation%2520of%2520Oncogenic%2520BCR-ABL%26jtitle%3DBCR-ABL%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.-Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+First-in-Class%2C+Highly+Potent%2C+and+Efficacious+Proteolysis+Targeting+Chimera+Murine+Double+Minute+2+Degrader+Capable+of+Achieving+Complete+and+Durable+Tumor+Regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0lhpEoEm-LN18Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520First-in-Class%252C%2520Highly%2520Potent%252C%2520and%2520Efficacious%2520Proteolysis%2520Targeting%2520Chimera%2520Murine%2520Double%2520Minute%25202%2520Degrader%2520Capable%2520of%2520Achieving%2520Complete%2520and%2520Durable%2520Tumor%2520Regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo Protein Knockdown by Small-Molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+Protein+Knockdown+by+Small-Molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lgaOPR2fU8xzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520Protein%2520Knockdown%2520by%2520Small-Molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2476</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1158%2F0008-5472.can-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476&author=L.+Baiauthor=B.+Zhouauthor=C.-Y.+Yangauthor=J.+Jiauthor=D.+McEachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+Huauthor=F.+Xuauthor=Y.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+Douauthor=B.+Wenauthor=D.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Liauthor=M.+J.+Ellisauthor=S.+Wang&title=Targeted+Degradation+of+BET+Proteins+in+Triple-Negative+Breast+Cancer&doi=10.1158%2F0008-5472.can-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0lgaOPR2fU8xzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeted%2520Degradation%2520of%2520BET%2520Proteins%2520in%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2476%26doi%3D10.1158%2F0008-5472.can-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ransick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Development of Protacs to Target Cancer-promoting Proteins for Ubiquitination and Degradation</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.1074/mcp.t300009-mcp200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1074%2Fmcp.t300009-mcp200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=14525958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVShu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=1350-1358&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=R.+Vermaauthor=A.+Ransickauthor=B.+Steinauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Development+of+Protacs+to+Target+Cancer-promoting+Proteins+for+Ubiquitination+and+Degradation&doi=10.1074%2Fmcp.t300009-mcp200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Protacs to target cancer-promoting proteins for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Verma, Rati; Ransick, Andy; Stein, Bernd; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1350-1358</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The proteome contains hundreds of proteins that in theory could be excellent therapeutic targets for the treatment of human diseases.  However, many of these proteins are from functional classes that have never been validated as viable candidates for the development of small mol. inhibitors.  Thus, to exploit fully the potential of the Human Genome Project to advance human medicine, there is a need to develop generic methods of inhibiting protein activity that do not rely on the target protein's function.  The authors previously demonstrated that a normally stable protein, methionine aminopeptidase-2 or MetAP-2, could be artificially targeted to an Skp1-Cullin-F-box (SCF) ubiquitin ligase complex for ubiquitination and degrdn. through a chimeric bridging mol. or Protac (proteolysis targeting chimeric mol.).  This Protac consisted of an SCFβ-TRCP-binding phosphopeptide derived from IκBα linked to ovalicin, which covalently binds MetAP-2.  In this study, the authors employed this approach to target two different proteins, the estrogen (ER) and androgen (AR) receptors, which have been implicated in the progression of breast and prostate cancer, resp.  The authors show here that an estradiol-based Protac can enforce the ubiquitination and degrdn. of the α isoform of ER in vitro, and a dihydroxytestosterone-based Protac introduced into cells promotes the rapid disappearance of AR in a proteasome-dependent manner.  Future improvements to this technol. may yield a general approach to treat a no. of human diseases, including cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3SZSMptsCorVg90H21EOLACvtfcHk0liX8tpGwfxPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVShu7jE&md5=5a9f4cf5d5cc1d990bf59ac824902111</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fmcp.t300009-mcp200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.t300009-mcp200%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DVerma%26aufirst%3DR.%26aulast%3DRansick%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DB.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DDevelopment%2520of%2520Protacs%2520to%2520Target%2520Cancer-promoting%2520Proteins%2520for%2520Ubiquitination%2520and%2520Degradation%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2003%26volume%3D2%26spage%3D1350%26epage%3D1358%26doi%3D10.1074%2Fmcp.t300009-mcp200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide Conjugation as A Strategy for in vivo Target Protein Degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+Conjugation+as+A+Strategy+for+in+vivo+Target+Protein+Degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0liX8tpGwfxPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520Conjugation%2520as%2520A%2520Strategy%2520for%2520in%2520vivo%2520Target%2520Protein%2520Degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span> <span> </span><span class="NLM_article-title">Identification of A Primary Target of Thalidomide Teratogenicity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1345</span>– <span class="NLM_lpage">1350</span>, <span class="refDoi"> DOI: 10.1126/science.1177319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.1177319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=20223979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFensrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1345-1350&author=T.+Itoauthor=H.+Andoauthor=T.+Suzukiauthor=T.+Oguraauthor=K.+Hottaauthor=Y.+Imamuraauthor=Y.+Yamaguchiauthor=H.+Handa&title=Identification+of+A+Primary+Target+of+Thalidomide+Teratogenicity&doi=10.1126%2Fscience.1177319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Primary Target of Thalidomide Teratogenicity</span></div><div class="casAuthors">Ito, Takumi; Ando, Hideki; Suzuki, Takayuki; Ogura, Toshihiko; Hotta, Kentaro; Imamura, Yoshimasa; Yamaguchi, Yuki; Handa, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5971</span>),
    <span class="NLM_cas:pages">1345-1350</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Half a century ago, thalidomide was widely prescribed to pregnant women as a sedative but was found to be teratogenic, causing multiple birth defects.  Today, thalidomide is still used in the treatment of leprosy and multiple myeloma, although how it causes limb malformation and other developmental defects is unknown.  Here, we identified cereblon (CRBN) as a thalidomide-binding protein.  CRBN forms an E3 ubiquitin ligase complex with damaged DNA binding protein 1 (DDB1) and Cul4A that is important for limb outgrowth and expression of the fibroblast growth factor Fgf8 in zebrafish and chicks.  Thalidomide initiates its teratogenic effects by binding to CRBN and inhibiting the assocd. ubiquitin ligase activity.  This study reveals a basis for thalidomide teratogenicity and may contribute to the development of new thalidomide derivs. without teratogenic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPC_7RXLkfabVg90H21EOLACvtfcHk0liX8tpGwfxPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFensrs%253D&md5=4565a67837e0ec34ede2c93c1036f312</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.1177319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1177319%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DT.%26aulast%3DAndo%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DOgura%26aufirst%3DT.%26aulast%3DHotta%26aufirst%3DK.%26aulast%3DImamura%26aufirst%3DY.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DHanda%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520A%2520Primary%2520Target%2520of%2520Thalidomide%2520Teratogenicity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1345%26epage%3D1350%26doi%3D10.1126%2Fscience.1177319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karasawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbasian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoudi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rychak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Cereblon is A Direct Protein Target for Immunomodulatory and Antiproliferative Activities of Lenalidomide and Pomalidomide</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2326</span>– <span class="NLM_lpage">2335</span>, <span class="refDoi"> DOI: 10.1038/leu.2012.119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fleu.2012.119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=22552008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Giur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=2326-2335&author=A.+Lopez-Gironaauthor=D.+Mendyauthor=T.+Itoauthor=K.+Millerauthor=A.+K.+Gandhiauthor=J.+Kangauthor=S.+Karasawaauthor=G.+Carmelauthor=P.+Jacksonauthor=M.+Abbasianauthor=A.+Mahmoudiauthor=B.+Cathersauthor=E.+Rychakauthor=S.+Gaidarovaauthor=R.+Chenauthor=P.+H.+Schaferauthor=H.+Handaauthor=T.+O.+Danielauthor=J.+F.+Evansauthor=R.+Chopra&title=Cereblon+is+A+Direct+Protein+Target+for+Immunomodulatory+and+Antiproliferative+Activities+of+Lenalidomide+and+Pomalidomide&doi=10.1038%2Fleu.2012.119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon is a direct protein target for immunomodulatory and antiproliferative activities of lenalidomide and pomalidomide</span></div><div class="casAuthors">Lopez-Girona, A.; Mendy, D.; Ito, T.; Miller, K.; Gandhi, A. K.; Kang, J.; Karasawa, S.; Carmel, G.; Jackson, P.; Abbasian, M.; Mahmoudi, A.; Cathers, B.; Rychak, E.; Gaidarova, S.; Chen, R.; Schafer, P. H.; Handa, H.; Daniel, T. O.; Evans, J. F.; Chopra, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2326-2335</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and the immunomodulatory drug, lenalidomide, are therapeutically active in hematol. malignancies.  The ubiquitously expressed E3 ligase protein cereblon (CRBN) has been identified as the primary teratogenic target of thalidomide.  Our studies demonstrate that thalidomide, lenalidomide and another immunomodulatory drug, pomalidomide, bound endogenous CRBN and recombinant CRBN-DNA damage binding protein-1 (DDB1) complexes.  CRBN mediated antiproliferative activities of lenalidomide and pomalidomide in myeloma cells, as well as lenalidomide- and pomalidomide-induced cytokine prodn. in T cells.  Lenalidomide and pomalidomide inhibited autoubiquitination of CRBN in HEK293T cells expressing thalidomide-binding competent wild-type CRBN, but not thalidomide-binding defective CRBNYW/AA.  Overexpression of CRBN wild-type protein, but not CRBNYW/AA mutant protein, in KMS12 myeloma cells, amplified pomalidomide-mediated redns. in c-myc and IRF4 expression and increases in p21WAF-1 expression.  Long-term selection for lenalidomide resistance in H929 myeloma cell lines was accompanied by a redn. in CRBN, while in DF15R myeloma cells resistant to both pomalidomide and lenalidomide, CRBN protein was undetectable.  Our biophys., biochem. and gene silencing studies show that CRBN is a proximate, therapeutically important mol. target of lenalidomide and pomalidomide.  Leukemia (2012) 26, 2326-2335; doi:10.1038/leu.2012.119; published online 15 June 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp85RvB33HGjbVg90H21EOLACvtfcHk0lhiNsSOg_KgIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Giur%252FI&md5=81f83b5f6c4c043a35a049df3b4d63ae</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fleu.2012.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2012.119%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DMendy%26aufirst%3DD.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKarasawa%26aufirst%3DS.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DJackson%26aufirst%3DP.%26aulast%3DAbbasian%26aufirst%3DM.%26aulast%3DMahmoudi%26aufirst%3DA.%26aulast%3DCathers%26aufirst%3DB.%26aulast%3DRychak%26aufirst%3DE.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DEvans%26aufirst%3DJ.%2BF.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DCereblon%2520is%2520A%2520Direct%2520Protein%2520Target%2520for%2520Immunomodulatory%2520and%2520Antiproliferative%2520Activities%2520of%2520Lenalidomide%2520and%2520Pomalidomide%26jtitle%3DLeukemia%26date%3D2012%26volume%3D26%26spage%3D2326%26epage%3D2335%26doi%3D10.1038%2Fleu.2012.119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Lenalidomide-Induced CK1α Degradation by the CRL4<sup>CRBN</sup> Ubiquitin Ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>532</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature16979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26909574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+Basis+of+Lenalidomide-Induced+CK1%CE%B1+Degradation+by+the+CRL4CRBN+Ubiquitin+Ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Petzold, Georg; Fischer, Eric S.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">532</span>
        (<span class="NLM_cas:issue">7597</span>),
    <span class="NLM_cas:pages">127-130</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs., lenalidomide and pomalidomide, are immune modulatory drugs (IMiDs) used in the treatment of hematol. malignancies.  IMiDs bind CRBN, the substrate receptor of the CUL4-RBX1-DDB1-CRBN (also known as CRL4CRBN) E3 ubiquitin ligase, and inhibit ubiquitination of endogenous CRL4CRBN substrates.  Unexpectedly, IMiDs also repurpose the ligase to target new proteins for degrdn.  Lenalidomide induces degrdn. of the lymphoid transcription factors Ikaros and Aiolos (also known as IKZF1 and IKZF3), and casein kinase 1α (CK1α), which contributes to its clin. efficacy in the treatment of multiple myeloma and 5q-deletion assocd. myelodysplastic syndrome (del(5q) MDS), resp.  How lenalidomide alters the specificity of the ligase to degrade these proteins remains elusive.  Here we present the 2.45 Å crystal structure of DDB1-CRBN bound to lenalidomide and CK1α.  CRBN and lenalidomide jointly provide the binding interface for a CK1α β-hairpin-loop located in the kinase N-lobe.  We show that CK1α binding to CRL4CRBN is strictly dependent on the presence of an IMiD.  Binding of IKZF1 to CRBN similarly requires the compd. and both, IKZF1 and CK1α, use a related binding mode.  Our study provides a mechanistic explanation for the selective efficacy of lenalidomide in del(5q) MDS therapy.  We anticipate that high-affinity protein-protein interactions induced by small mols. will provide opportunities for drug development, particularly for targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4FxWriI1ObVg90H21EOLACvtfcHk0lhiNsSOg_KgIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D&md5=320fc6ff811da335fa8e02feab2d702d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520Basis%2520of%2520Lenalidomide-Induced%2520CK1%25CE%25B1%2520Degradation%2520by%2520the%2520CRL4CRBN%2520Ubiquitin%2520Ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naniong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The Myeloma Drug Lenalidomide Promotes the Cereblon-Dependent Destruction of Ikaros Proteins</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1126/science.1244917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.1244917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=24292623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2ktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=305-309&author=G.+Luauthor=R.+E.+Middletonauthor=H.+Sunauthor=M.+Naniongauthor=C.+J.+Ottauthor=C.+S.+Mitsiadesauthor=K.-K.+Wongauthor=J.+E.+Bradnerauthor=W.+G.+Kaelin&title=The+Myeloma+Drug+Lenalidomide+Promotes+the+Cereblon-Dependent+Destruction+of+Ikaros+Proteins&doi=10.1126%2Fscience.1244917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Myeloma Drug Lenalidomide Promotes the Cereblon-Dependent Destruction of Ikaros Proteins</span></div><div class="casAuthors">Lu, Gang; Middleton, Richard E.; Sun, Huahang; Naniong, MarkVic; Ott, Christopher J.; Mitsiades, Constantine S.; Wong, Kwok-Kin; Bradner, James E.; Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6168</span>),
    <span class="NLM_cas:pages">305-309</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Thalidomide-like drugs such as lenalidomide are clin. important treatments for multiple myeloma and show promise for other B cell malignancies.  The biochem. mechanisms underlying their antitumor activity are unknown.  Thalidomide was recently shown to bind to, and inhibit, the cereblon ubiquitin ligase.  Cereblon loss in zebrafish causes fin defects reminiscent of the limb defects seen in children exposed to thalidomide in utero.  Here we show that lenalidomide-bound cereblon acquires the ability to target for proteasomal degrdn. two specific B cell transcription factors, Ikaros family zinc finger proteins 1 and 3 (IKZF1 and IKZF3).  Anal. of myeloma cell lines revealed that loss of IKZF1 and IKZF3 is both necessary and sufficient for lenalidomide's therapeutic effect, suggesting that the antitumor and teratogenic activities of thalidomide-like drugs are dissociable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0froz4Kipu7Vg90H21EOLACvtfcHk0lhiNsSOg_KgIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2ktg%253D%253D&md5=a9df5bbde0f676b92b1bec48fdbbb502</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244917%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DG.%26aulast%3DMiddleton%26aufirst%3DR.%2BE.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNaniong%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DThe%2520Myeloma%2520Drug%2520Lenalidomide%2520Promotes%2520the%2520Cereblon-Dependent%2520Destruction%2520of%2520Ikaros%2520Proteins%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D305%26epage%3D309%26doi%3D10.1126%2Fscience.1244917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kronke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grauman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConkey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarlo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">Lenalidomide Causes Selective Degradation of IKZF1 and IKZF3 in Multiple Myeloma Cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1126/science.1244851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.1244851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=24292625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=301-305&author=J.+Kronkeauthor=N.+D.+Udeshiauthor=A.+Narlaauthor=P.+Graumanauthor=S.+N.+Hurstauthor=M.+McConkeyauthor=T.+Svinkinaauthor=D.+Hecklauthor=E.+Comerauthor=X.+Liauthor=C.+Ciarloauthor=E.+Hartmanauthor=N.+Munshiauthor=M.+Schenoneauthor=S.+L.+Schreiberauthor=S.+A.+Carrauthor=B.+L.+Ebert&title=Lenalidomide+Causes+Selective+Degradation+of+IKZF1+and+IKZF3+in+Multiple+Myeloma+Cells&doi=10.1126%2Fscience.1244851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells</span></div><div class="casAuthors">Kronke Jan; Udeshi Namrata D; Narla Anupama; Grauman Peter; Hurst Slater N; McConkey Marie; Svinkina Tanya; Heckl Dirk; Comer Eamon; Li Xiaoyu; Ciarlo Christie; Hartman Emily; Munshi Nikhil; Schenone Monica; Schreiber Stuart L; Carr Steven A; Ebert Benjamin L</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6168</span>),
    <span class="NLM_cas:pages">301-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lenalidomide is a drug with clinical efficacy in multiple myeloma and other B cell neoplasms, but its mechanism of action is unknown.  Using quantitative proteomics, we found that lenalidomide causes selective ubiquitination and degradation of two lymphoid transcription factors, IKZF1 and IKZF3, by the CRBN-CRL4 ubiquitin ligase.  IKZF1 and IKZF3 are essential transcription factors in multiple myeloma.  A single amino acid substitution of IKZF3 conferred resistance to lenalidomide-induced degradation and rescued lenalidomide-induced inhibition of cell growth.  Similarly, we found that lenalidomide-induced interleukin-2 production in T cells is due to depletion of IKZF1 and IKZF3.  These findings reveal a previously unknown mechanism of action for a therapeutic agent: alteration of the activity of an E3 ubiquitin ligase, leading to selective degradation of specific targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQiiGR0Yx7nuYFEOhA_Im3cfW6udTcc2ebS2UbvrIUS9bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D&md5=8cae3e554399308493aa38f0d05ed610</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244851%26sid%3Dliteratum%253Aachs%26aulast%3DKronke%26aufirst%3DJ.%26aulast%3DUdeshi%26aufirst%3DN.%2BD.%26aulast%3DNarla%26aufirst%3DA.%26aulast%3DGrauman%26aufirst%3DP.%26aulast%3DHurst%26aufirst%3DS.%2BN.%26aulast%3DMcConkey%26aufirst%3DM.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DHeckl%26aufirst%3DD.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCiarlo%26aufirst%3DC.%26aulast%3DHartman%26aufirst%3DE.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DLenalidomide%2520Causes%2520Selective%2520Degradation%2520of%2520IKZF1%2520and%2520IKZF3%2520in%2520Multiple%2520Myeloma%2520Cells%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D301%26epage%3D305%26doi%3D10.1126%2Fscience.1244851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krönke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenbach, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacBeth, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConkey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Järås, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>523</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1038/nature14610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature14610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26131937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1entr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=523&publication_year=2015&pages=183-188&author=J.+Kr%C3%B6nkeauthor=E.+C.+Finkauthor=P.+W.+Hollenbachauthor=K.+J.+MacBethauthor=S.+N.+Hurstauthor=N.+D.+Udeshiauthor=P.+P.+Chamberlainauthor=D.+R.+Maniauthor=H.+W.+Manauthor=A.+K.+Gandhiauthor=T.+Svinkinaauthor=R.+K.+Schneiderauthor=M.+McConkeyauthor=M.+J%C3%A4r%C3%A5sauthor=E.+Griffithsauthor=M.+Wetzlerauthor=L.+Bullingerauthor=B.+E.+Cathersauthor=S.+A.+Carrauthor=R.+Chopraauthor=B.+L.+Ebert&title=Lenalidomide+induces+ubiquitination+and+degradation+of+CK1%CE%B1+in+del%285q%29+MDS&doi=10.1038%2Fnature14610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS</span></div><div class="casAuthors">Kronke, Jan; Fink, Emma C.; Hollenbach, Paul W.; MacBeth, Kyle J.; Hurst, Slater N.; Udeshi, Namrata D.; Chamberlain, Philip P.; Mani, D. R.; Man, Hon Wah; Gandhi, Anita K.; Svinkina, Tanya; Schneider, Rebekka K.; McConkey, Marie; Jaras, Marcus; Griffiths, Elizabeth; Wetzler, Meir; Bullinger, Lars; Cathers, Brian E.; Carr, Steven A.; Chopra, Rajesh; Ebert, Benjamin L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">523</span>
        (<span class="NLM_cas:issue">7559</span>),
    <span class="NLM_cas:pages">183-188</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lenalidomide is a highly effective treatment for myelodysplastic syndrome (MDS) with deletion of chromosome 5q (del(5q)).  Lenalidomide induces the ubiquitination of casein kinase 1A1 (CK1α) by the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN), resulting in CK1α degrdn.  CK1α is encoded by a gene within the common deleted region for del(5q) MDS and haplo-insufficient expression sensitizes cells to lenalidomide therapy, providing a mechanistic basis for the therapeutic window of lenalidomide in del(5q) MDS.  The authors found that mouse cells are resistant to lenalidomide but that changing a single amino acid in mouse Crbn to the corresponding human residue enables lenalidomide-dependent degrdn. of CK1α.  The authors further demonstrate that minor side chain modifications in thalidomide and a novel analog, CC-122, can modulate the spectrum of substrates targeted by CRL4CRBN.  These findings have implications for the clin. activity of lenalidomide and related compds., and demonstrate the therapeutic potential of novel modulators of E3 ubiquitin ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiLiHx3FKesbVg90H21EOLACvtfcHk0ljufsARqB8ESg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1entr%252FN&md5=9b20a91c13e888e6569223ffa25af2cd</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature14610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14610%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25B6nke%26aufirst%3DJ.%26aulast%3DFink%26aufirst%3DE.%2BC.%26aulast%3DHollenbach%26aufirst%3DP.%2BW.%26aulast%3DMacBeth%26aufirst%3DK.%2BJ.%26aulast%3DHurst%26aufirst%3DS.%2BN.%26aulast%3DUdeshi%26aufirst%3DN.%2BD.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DMani%26aufirst%3DD.%2BR.%26aulast%3DMan%26aufirst%3DH.%2BW.%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DSchneider%26aufirst%3DR.%2BK.%26aulast%3DMcConkey%26aufirst%3DM.%26aulast%3DJ%25C3%25A4r%25C3%25A5s%26aufirst%3DM.%26aulast%3DGriffiths%26aufirst%3DE.%26aulast%3DWetzler%26aufirst%3DM.%26aulast%3DBullinger%26aufirst%3DL.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DChopra%26aufirst%3DR.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DLenalidomide%2520induces%2520ubiquitination%2520and%2520degradation%2520of%2520CK1%25CE%25B1%2520in%2520del%25285q%2529%2520MDS%26jtitle%3DNature%26date%3D2015%26volume%3D523%26spage%3D183%26epage%3D188%26doi%3D10.1038%2Fnature14610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">SPLINTS: Small-molecule Protein Ligand Interface Stabilizers</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2016.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.sbi.2016.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26829757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=115-122&author=E.+S.+Fischerauthor=E.+Parkauthor=M.+J.+Eckauthor=N.+H.+Thom%C3%A4&title=SPLINTS%3A+Small-molecule+Protein+Ligand+Interface+Stabilizers&doi=10.1016%2Fj.sbi.2016.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">SPLINTS: small-molecule protein ligand interface stabilizers</span></div><div class="casAuthors">Fischer, Eric S.; Park, Eunyoung; Eck, Michael J.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">115-122</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Regulatory protein-protein interactions are ubiquitous in biol., and small mol. protein-protein interaction inhibitors are an important focus in drug discovery.  Remarkably little attention has been given to the opposite strategy - stabilization of protein-protein interactions, despite the fact that several well-known therapeutics act through this mechanism.  From a structural perspective, we consider representative examples of small mols. that induce or stabilize the assocn. of protein domains to inhibit, or alter, signaling for nuclear hormone, GTPase, kinase, phosphatase, and ubiquitin ligase pathways.  These SPLINTS (small-mol. protein ligand interface stabilizers) drive interactions that are in some cases physiol. relevant, and in others entirely adventitious.  The diverse structural mechanisms employed suggest approaches for a broader and systematic search for such compds. in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBxkNU6Walm7Vg90H21EOLACvtfcHk0liJZTe3FmVyfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKqsr0%253D&md5=014c6488fbbc458b395d38e3220a443f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2016.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2016.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DSPLINTS%253A%2520Small-molecule%2520Protein%2520Ligand%2520Interface%2520Stabilizers%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2016%26volume%3D37%26spage%3D115%26epage%3D122%26doi%3D10.1016%2Fj.sbi.2016.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Ternary Complexes: A New Dimension in the Structure-guided Design of Chemical Degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/ebc20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1042%2Febc20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+Recognition+of+Ternary+Complexes%3A+A+New+Dimension+in+the+Structure-guided+Design+of+Chemical+Degraders&doi=10.1042%2Febc20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2eZPIRkUWKJRbbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1042%2Febc20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Febc20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520Recognition%2520of%2520Ternary%2520Complexes%253A%2520A%2520New%2520Dimension%2520in%2520the%2520Structure-guided%2520Design%2520of%2520Chemical%2520Degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D505%26epage%3D516%26doi%3D10.1042%2Febc20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goralski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaskill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ting, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nijhawan, D.</span></span> <span> </span><span class="NLM_article-title">Anticancer Sulfonamides Target Splicing by Inducing RBM39 Degradation via Recruitment to DCAF15</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>, <span class="NLM_elocation-id">eaal3755</span> <span class="refDoi"> DOI: 10.1126/science.aal3755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.aal3755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28302793" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&author=T.+Hanauthor=M.+Goralskiauthor=N.+Gaskillauthor=E.+Capotaauthor=J.+Kimauthor=T.+C.+Tingauthor=Y.+Xieauthor=N.+S.+Williamsauthor=D.+Nijhawan&title=Anticancer+Sulfonamides+Target+Splicing+by+Inducing+RBM39+Degradation+via+Recruitment+to+DCAF15&doi=10.1126%2Fscience.aal3755"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal3755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal3755%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DT.%26aulast%3DGoralski%26aufirst%3DM.%26aulast%3DGaskill%26aufirst%3DN.%26aulast%3DCapota%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DTing%26aufirst%3DT.%2BC.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DNijhawan%26aufirst%3DD.%26atitle%3DAnticancer%2520Sulfonamides%2520Target%2520Splicing%2520by%2520Inducing%2520RBM39%2520Degradation%2520via%2520Recruitment%2520to%2520DCAF15%26jtitle%3DScience%26date%3D2017%26volume%3D356%26doi%3D10.1126%2Fscience.aal3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugi, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuhashi, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotake, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uesugi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mabuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owa, T.</span></span> <span> </span><span class="NLM_article-title">Selective degradation of splicing factor CAPERα by anticancer sulfonamides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">675</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnchembio.2363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28437394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVCltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=675-680&author=T.+Ueharaauthor=Y.+Minoshimaauthor=K.+Saganeauthor=N.+H.+Sugiauthor=K.+O.+Mitsuhashiauthor=N.+Yamamotoauthor=H.+Kamiyamaauthor=K.+Takahashiauthor=Y.+Kotakeauthor=M.+Uesugiauthor=A.+Yokoiauthor=A.+Inoueauthor=T.+Yoshidaauthor=M.+Mabuchiauthor=A.+Tanakaauthor=T.+Owa&title=Selective+degradation+of+splicing+factor+CAPER%CE%B1+by+anticancer+sulfonamides&doi=10.1038%2Fnchembio.2363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selective degradation of splicing factor CAPERα by anticancer sulfonamides</span></div><div class="casAuthors">Uehara, Taisuke; Minoshima, Yukinori; Sagane, Koji; Sugi, Naoko Hata; Mitsuhashi, Kaoru Ogawa; Yamamoto, Noboru; Kamiyama, Hiroshi; Takahashi, Kentaro; Kotake, Yoshihiko; Uesugi, Mai; Yokoi, Akira; Inoue, Atsushi; Yoshida, Taku; Mabuchi, Miyuki; Tanaka, Akito; Owa, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">675-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Target-protein degrdn. is an emerging field in drug discovery and development.  In particular, the substrate-receptor proteins of the cullin-ubiquitin ligase system play a key role in selective protein degrdn., which is an essential component of the anti-myeloma activity of immunomodulatory drugs (IMiDs), such as lenalidomide.  Here, we demonstrate that a series of anticancer sulfonamides NSC 719239 (E7820), indisulam, and NSC 339004 (chloroquinoxaline sulfonamide, CQS) induce proteasomal degrdn. of the U2AF-related splicing factor coactivator of activating protein-1 and estrogen receptors (CAPERα) via CRL4DCAF15 mediated ubiquitination in human cancer cell lines.  Both CRISPR-Cas9-based knockout of DCAF15 and a single amino acid substitution of CAPERα conferred resistance against sulfonamide-induced CAPERα degrdn. and cell-growth inhibition.  Thus, these sulfonamides represent selective chem. probes for disrupting CAPERα function and designate DCAFs as promising drug targets for promoting selective protein degrdn. in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4hxIDPBvmVLVg90H21EOLACvtfcHk0ljd0UoiVu45qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVCltbY%253D&md5=59078969183a22084ce31973fd026095</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2363%26sid%3Dliteratum%253Aachs%26aulast%3DUehara%26aufirst%3DT.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DSagane%26aufirst%3DK.%26aulast%3DSugi%26aufirst%3DN.%2BH.%26aulast%3DMitsuhashi%26aufirst%3DK.%2BO.%26aulast%3DYamamoto%26aufirst%3DN.%26aulast%3DKamiyama%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DUesugi%26aufirst%3DM.%26aulast%3DYokoi%26aufirst%3DA.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DYoshida%26aufirst%3DT.%26aulast%3DMabuchi%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DOwa%26aufirst%3DT.%26atitle%3DSelective%2520degradation%2520of%2520splicing%2520factor%2520CAPER%25CE%25B1%2520by%2520anticancer%2520sulfonamides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D675%26epage%3D680%26doi%3D10.1038%2Fnchembio.2363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+Termination+Factor+GSPT1+Is+a+Phenotypically+Relevant+Off-Target+of+Heterobifunctional+Phthalimide+Degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0ljd0UoiVu45qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520Termination%2520Factor%2520GSPT1%2520Is%2520a%2520Phenotypically%2520Relevant%2520Off-Target%2520of%2520Heterobifunctional%2520Phthalimide%2520Degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochsenbein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carry, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Diastereomeric Spirooxindoles as Highly Potent and Efficacious MDM2 Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">7223</span>– <span class="NLM_lpage">7234</span>, <span class="refDoi"> DOI: 10.1021/ja3125417</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3125417" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVWrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=7223-7234&author=Y.+Zhaoauthor=L.+Liuauthor=W.+Sunauthor=J.+Luauthor=D.+McEachernauthor=X.+Liauthor=S.+Yuauthor=D.+Bernardauthor=P.+Ochsenbeinauthor=V.+Fereyauthor=J.-C.+Carryauthor=J.+R.+Deschampsauthor=D.+Sunauthor=S.+Wang&title=Diastereomeric+Spirooxindoles+as+Highly+Potent+and+Efficacious+MDM2+Inhibitors&doi=10.1021%2Fja3125417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Diastereomeric Spirooxindoles as Highly Potent and Efficacious MDM2 Inhibitors</span></div><div class="casAuthors">Zhao, Yujun; Liu, Liu; Sun, Wei; Lu, Jianfeng; McEachern, Donna; Li, Xiaoqin; Yu, Shanghai; Bernard, Denzil; Ochsenbein, Philippe; Ferey, Vincent; Carry, Jean-Christophe; Deschamps, Jeffrey R.; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7223-7234</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that block the MDM2-p53 protein-protein interaction (MDM2 inhibitors) are being intensely pursued as a new therapeutic strategy for cancer treatment.  We previously published a series of spirooxindole-contg. compds. as a new class of MDM2 small-mol. inhibitors.  We report herein a reversible ring-opening-cyclization reaction for some of these spirooxindoles, which affords four diastereomers from a single compd.  Our biochem. binding data showed that the stereochem. in this class of compds. has a major effect on their binding affinities to MDM2, with >100-fold difference between the most potent and the least potent stereoisomers.  Our study has led to the identification of a set of highly potent MDM2 inhibitors with a stereochem. that is different from that of our previously reported compds.  The most potent compd. (MI-888) binds to MDM2 with a Ki value of 0.44 nM and achieves complete and long-lasting tumor regression in an animal model of human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOEWm0CbFl2rVg90H21EOLACvtfcHk0liIXrDn7Kj-_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVWrtr0%253D&md5=5c05b0adc0d14506602d0884b64411b0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja3125417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3125417%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DOchsenbein%26aufirst%3DP.%26aulast%3DFerey%26aufirst%3DV.%26aulast%3DCarry%26aufirst%3DJ.-C.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiastereomeric%2520Spirooxindoles%2520as%2520Highly%2520Potent%2520and%2520Efficacious%2520MDM2%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D7223%26epage%3D7234%26doi%3D10.1021%2Fja3125417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman-Luca, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debussche, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Elucidation of Acquired Resistance to Bcl-2 and MDM2 Inhibitors in Acute Leukemia in Vitro and in Vivo</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2558</span>– <span class="NLM_lpage">2568</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-14-2506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1158%2F1078-0432.ccr-14-2506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25754349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1aku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2558-2568&author=C.+G.+Hoffman-Lucaauthor=D.+Ziazadehauthor=D.+McEachernauthor=Y.+Zhaoauthor=W.+Sunauthor=L.+Debusscheauthor=S.+Wang&title=Elucidation+of+Acquired+Resistance+to+Bcl-2+and+MDM2+Inhibitors+in+Acute+Leukemia+in+Vitro+and+in+Vivo&doi=10.1158%2F1078-0432.ccr-14-2506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation of Acquired Resistance to Bcl-2 and MDM2 Inhibitors in Acute Leukemia In Vitro and In Vivo</span></div><div class="casAuthors">Hoffman-Luca, C. Gianna; Ziazadeh, Daniel; McEachern, Donna; Zhao, Yujun; Sun, Wei; Debussche, Laurent; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2558-2568</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Two clin.-stage anticancer drugs, the Bcl-2 inhibitor ABT-263, and the MDM2 inhibitor SAR405838 achieve complete tumor regression in animal models of leukemia but also induce acquired resistance.  Elucidation of acquired resistance mechanisms and development of strategies to overcome the resistance are crit. for their successful clin. development.  Exptl. Design: We employed RS4;11 and MV4;11 cell lines, two acute leukemia models, to investigate acquired resistance mechanisms for both drugs in vitro and in vivo and evaluated several treatment regimens in xenograft mouse models to improve long-term, complete tumor regression.  Results: Resistance to either SAR405838 or ABT-263 (or its analog ABT-737) develops in acute leukemia models in vitro and in vivo.  RS4;11 and MV4;11 tumors treated with SAR405838 acquire resistance to the drug by mutation of the TP53 gene or compromise of p53 protein function.  RS4;11 tumors treated with either ABT-263 or ABT-737 acquire resistance primarily through downregulation of BAX but not BAK.  When acute leukemia cells become highly resistant to the MDM2 inhibitor, they retain their sensitivity to the Bcl-2 inhibitors, or vice versa.  Certain sequential or combination treatment of SAR405838 and ABT-263 can achieve longer-term tumor regression than treatment with either agent alone.  Conclusions: Our study provides new insights into the mechanisms of acquired resistance of Bcl-2 and MDM2 inhibitors in acute leukemia models and suggests that certain sequential or combination treatment of these two distinct classes of apoptosis-inducing agents should be tested as new treatment strategies for acute leukemia in the clinic.  Clin Cancer Res; 21(11); 2558-68. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8iM179w0AXrVg90H21EOLACvtfcHk0liIXrDn7Kj-_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1aku7o%253D&md5=faa6180e88c8eaf3c6af59c0b3b09d3c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-14-2506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-14-2506%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman-Luca%26aufirst%3DC.%2BG.%26aulast%3DZiazadeh%26aufirst%3DD.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DDebussche%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DElucidation%2520of%2520Acquired%2520Resistance%2520to%2520Bcl-2%2520and%2520MDM2%2520Inhibitors%2520in%2520Acute%2520Leukemia%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2558%26epage%3D2568%26doi%3D10.1158%2F1078-0432.ccr-14-2506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meaux, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barriere, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman-Luca, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shangary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos-Santos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besret, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorge-Bernat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debussche, L.</span></span> <span> </span><span class="NLM_article-title">SAR405838: An Optimized Inhibitor of MDM2-p53 Interaction That Induces Complete and Durable Tumor Regression</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">5855</span>– <span class="NLM_lpage">5865</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-14-0799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1158%2F0008-5472.CAN-14-0799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25145672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGjsr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=5855-5865&author=S.+Wangauthor=W.+Sunauthor=Y.+Zhaoauthor=D.+McEachernauthor=I.+Meauxauthor=C.+Barriereauthor=J.+A.+Stuckeyauthor=J.+L.+Meagherauthor=L.+Baiauthor=L.+Liuauthor=C.+G.+Hoffman-Lucaauthor=J.+Luauthor=S.+Shangaryauthor=S.+Yuauthor=D.+Bernardauthor=A.+Aguilarauthor=O.+Dos-Santosauthor=L.+Besretauthor=S.+Guerifauthor=P.+Pannierauthor=D.+Gorge-Bernatauthor=L.+Debussche&title=SAR405838%3A+An+Optimized+Inhibitor+of+MDM2-p53+Interaction+That+Induces+Complete+and+Durable+Tumor+Regression&doi=10.1158%2F0008-5472.can-14-0799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">SAR405838: An Optimized Inhibitor of MDM2-p53 Interaction That Induces Complete and Durable Tumor Regression</span></div><div class="casAuthors">Wang, Shaomeng; Sun, Wei; Zhao, Yujun; McEachern, Donna; Meaux, Isabelle; Barriere, Cedric; Stuckey, Jeanne A.; Meagher, Jennifer L.; Bai, Longchuan; Liu, Liu; Hoffman-Luca, Cassandra Gianna; Lu, Jianfeng; Shangary, Sanjeev; Yu, Shanghai; Bernard, Denzil; Aguilar, Angelo; Dos-Santos, Odette; Besret, Laurent; Guerif, Stephane; Pannier, Pascal; Gorge-Bernat, Dimitri; Debussche, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5855-5865</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Despite the risk of applying a selection for p53 mutations that escape MDM2 control, blocking MDM2-p53 protein-protein interaction has long been considered by many to offer an attractive cancer therapeutic strategy, a position strongly supported by the findings of this preclin. study.  Blocking the oncoprotein murine double minute 2 (MDM2)-p53 protein-protein interaction has long been considered to offer a broad cancer therapeutic strategy, despite the potential risks of selecting tumors harboring p53 mutations that escape MDM2 control.  In this study, we report a novel small-mol. inhibitor of the MDM2-p53 interaction, SAR405838 (MI-77301), that has been advanced into phase I clin. trials.  SAR405838 binds to MDM2 with Ki = 0.88 nmol/L and has high specificity over other proteins.  A cocrystal structure of the SAR405838:MDM2 complex shows that, in addn. to mimicking three key p53 amino acid residues, the inhibitor captures addnl. interactions not obsd. in the p53-MDM2 complex and induces refolding of the short, unstructured MDM2 N-terminal region to achieve its high affinity.  SAR405838 effectively activates wild-type p53 in vitro and in xenograft tumor tissue of leukemia and solid tumors, leading to p53-dependent cell-cycle arrest and/or apoptosis.  At well-tolerated dose schedules, SAR405838 achieves either durable tumor regression or complete tumor growth inhibition in mouse xenograft models of SJSA-1 osteosarcoma, RS4;11 acute leukemia, LNCaP prostate cancer, and HCT-116 colon cancer.  Remarkably, a single oral dose of SAR405838 is sufficient to achieve complete tumor regression in the SJSA-1 model.  Mechanistically, robust transcriptional upregulation of PUMA induced by SAR405838 results in strong apoptosis in tumor tissue, leading to complete tumor regression.  Our findings provide a preclin. basis upon which to evaluate SAR405838 as a therapeutic agent in patients whose tumors retain wild-type p53.  Cancer Res; 74(20); 5855-65. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptKSwfNhNzNLVg90H21EOLACvtfcHk0liIXrDn7Kj-_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGjsr7M&md5=5b35759e14fe4c1ae4e734775c4334a0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-0799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-0799%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DMeaux%26aufirst%3DI.%26aulast%3DBarriere%26aufirst%3DC.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHoffman-Luca%26aufirst%3DC.%2BG.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DShangary%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DDos-Santos%26aufirst%3DO.%26aulast%3DBesret%26aufirst%3DL.%26aulast%3DGuerif%26aufirst%3DS.%26aulast%3DPannier%26aufirst%3DP.%26aulast%3DGorge-Bernat%26aufirst%3DD.%26aulast%3DDebussche%26aufirst%3DL.%26atitle%3DSAR405838%253A%2520An%2520Optimized%2520Inhibitor%2520of%2520MDM2-p53%2520Interaction%2520That%2520Induces%2520Complete%2520and%2520Durable%2520Tumor%2520Regression%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D5855%26epage%3D5865%26doi%3D10.1158%2F0008-5472.can-14-0799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eeles, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hainaut, P.</span></span> <span> </span><span class="NLM_article-title">The IARC TP53 Database: New Online Mutation Analysis and Recommendations to Users</span>. <i>Human Mutation</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1002/humu.10081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fhumu.10081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=12007217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD38XkvVGktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2002&pages=607-614&author=M.+Olivierauthor=R.+Eelesauthor=M.+Hollsteinauthor=M.+A.+Khanauthor=C.+C.+Harrisauthor=P.+Hainaut&title=The+IARC+TP53+Database%3A+New+Online+Mutation+Analysis+and+Recommendations+to+Users&doi=10.1002%2Fhumu.10081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The IARC TP53 database: new online mutation analysis and recommendations to users</span></div><div class="casAuthors">Olivier, Magali; Eeles, Ros; Hollstein, Monica; Khan, Mohammed A.; Harris, Curtis C.; Hainaut, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Human Mutation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">607-614</span>CODEN:
                <span class="NLM_cas:coden">HUMUE3</span>;
        ISSN:<span class="NLM_cas:issn">1059-7794</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Mutations in the tumor suppressor gene TP53 are frequent in most human cancers.  Comparison of the mutation patterns in different cancers may reveal clues on the natural history of the disease.  Over the past 10 yr, several databases of TP53 mutations have been developed.  The most extensive of these databases is maintained and developed at the International Agency for Research on Cancer.  The database compiles all mutations (somatic and inherited), as well as polymorphisms, that have been reported in the published literature since 1989.  The IARC TP53 mutation dataset is the largest dataset available on the variations of any human gene.  The database is available at www.iarc.fr/P53/.  In this paper, we describe recent developments of the database.  These developments include restructuring of the database, which is now patient-centered, with more detailed annotations on the patient (carcinogen exposure, virus infection, genetic background).  In addn., a new online application to retrieve somatic mutation data and analyze mutation patterns is now available.  We also discuss limitations on the use of the database and provide recommendations to users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHsjPDhuXnq7Vg90H21EOLACvtfcHk0ljEswJgGJQtfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XkvVGktro%253D&md5=314c1e17d468d982f47f354b48e742e4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fhumu.10081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhumu.10081%26sid%3Dliteratum%253Aachs%26aulast%3DOlivier%26aufirst%3DM.%26aulast%3DEeles%26aufirst%3DR.%26aulast%3DHollstein%26aufirst%3DM.%26aulast%3DKhan%26aufirst%3DM.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DHainaut%26aufirst%3DP.%26atitle%3DThe%2520IARC%2520TP53%2520Database%253A%2520New%2520Online%2520Mutation%2520Analysis%2520and%2520Recommendations%2520to%2520Users%26jtitle%3DHuman%2520Mutation%26date%3D2002%26volume%3D19%26spage%3D607%26epage%3D614%26doi%3D10.1002%2Fhumu.10081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrish, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Potent Non-Peptide MDM2 Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">10130</span>– <span class="NLM_lpage">10131</span>, <span class="refDoi"> DOI: 10.1021/ja051147z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja051147z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1ahtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=10130-10131&author=K.+Dingauthor=Y.+Luauthor=Z.+Nikolovska-Coleskaauthor=S.+Qiuauthor=Y.+Dingauthor=W.+Gaoauthor=J.+Stuckeyauthor=K.+Krajewskiauthor=P.+P.+Rollerauthor=Y.+Tomitaauthor=D.+A.+Parrishauthor=J.+R.+Deschampsauthor=S.+Wang&title=Structure-Based+Design+of+Potent+Non-Peptide+MDM2+Inhibitors&doi=10.1021%2Fja051147z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent Non-Peptide MDM2 Inhibitors</span></div><div class="casAuthors">Ding, Ke; Lu, Yipin; Nikolovska-Coleska, Zaneta; Qiu, Su; Ding, Yousong; Gao, Wei; Stuckey, Jeanne; Krajewski, Krzysztof; Roller, Peter P.; Tomita, York; Parrish, Damon A.; Deschamps, Jeffrey R.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">10130-10131</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A successful structure-based design of a class of nonpeptide small-mol. MDM2 inhibitors targeting the p53-MDM2 protein-protein interaction is reported.  The most potent compd. (I) binds to MDM2 protein with a Ki value of 86 nM and is 18 times more potent than a natural p53 peptide (residues 16-27).  Compd. I is potent in inhibition of cell growth in LNCaP prostate cancer cells with wild-type p53 and shows only a weak activity in PC-3 prostate cancer cells with a deleted p53.  Importantly, I has a minimal toxicity to normal prostate epithelial cells.  Our studies provide a convincing example that structure-based strategy can be employed to design highly potent, nonpeptide, cell-permeable, small-mol. inhibitors to target protein-protein interaction, which remains a very challenging area in chem. biol. and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXaOqJGFoCeLVg90H21EOLACvtfcHk0ljEswJgGJQtfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1ahtL0%253D&md5=44cc1b7e2ddc86e1bc27c79c150f84c7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja051147z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja051147z%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DQiu%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DParrish%26aufirst%3DD.%2BA.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520Potent%2520Non-Peptide%2520MDM2%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D10130%26epage%3D10131%26doi%3D10.1021%2Fja051147z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">García-Echeverría, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chène, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent Antagonists of the Interaction between Human Double Minute 2 and Tumor Suppressor p53</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">3205</span>– <span class="NLM_lpage">3208</span>, <span class="refDoi"> DOI: 10.1021/jm990966p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990966p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltlOruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3205-3208&author=C.+Garc%C3%ADa-Echeverr%C3%ADaauthor=P.+Ch%C3%A8neauthor=M.+J.+J.+Blommersauthor=P.+Furet&title=Discovery+of+Potent+Antagonists+of+the+Interaction+between+Human+Double+Minute+2+and+Tumor+Suppressor+p53&doi=10.1021%2Fjm990966p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Antagonists of the Interaction between Human Double Minute 2 and Tumor Suppressor p53</span></div><div class="casAuthors">Garcia-Echeverria, Carlos; Chene, Patrick; Blommers, Marcel J. J.; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3205-3208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of a drug discovery program to identify antagonists of the p53/hdm2 (human double minute 2) protein-protein interaction, the authors have attempted to det. the amino acid specificities of hdm2's binding pockets to establish a pharmacophore model for this protein-protein interaction.  This work has resulted in the identification of highly potent peptide antagonists.  Structural information has been exploited to increase the hdm2-binding affinity of short peptide motifs derived from the N-terminal domain of the human wild-type p53 protein.  Combining conformational constraints as selected by mol. modeling with functional groups that are able to establish addnl. electrostatic and van der Waals interactions with the hdm2 protein, the authors have been able to increase the hdm2-binding affinity of the authors initial peptide 1700-fold.  Particularly interesting is the increase in binding affinity obtained by replacing tryptophan with 6-chlorotryptophan (IC50 = 314 nM vs. IC50 = 5 nM, 63-fold).  The new interactions identified and exptl. confirmed in this work could be directly applied to the optimization of nonpeptidic leads or incorporated into the "de novo" design of antagonists of the p53/hdm2 protein-protein interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpOq3ikfGqz7Vg90H21EOLACvtfcHk0ljEswJgGJQtfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltlOruro%253D&md5=b921adfdd6a5c85510276a6446e600a8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm990966p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990966p%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Echeverr%25C3%25ADa%26aufirst%3DC.%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%2BJ.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Potent%2520Antagonists%2520of%2520the%2520Interaction%2520between%2520Human%2520Double%2520Minute%25202%2520and%2520Tumor%2520Suppressor%2520p53%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3205%26epage%3D3208%26doi%3D10.1021%2Fjm990966p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lgGF46dWlzVoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lgGF46dWlzVoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisarev, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisareva, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestova, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Round, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svergun, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span> <span> </span><span class="NLM_article-title">Structural Insights Into eRF3 and Stop Codon Recognition by eRF1</span>. <i>Genes Develop.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1106</span>– <span class="NLM_lpage">1118</span>, <span class="refDoi"> DOI: 10.1101/gad.1770109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1101%2Fgad.1770109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=19417105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Crs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=1106-1118&author=Z.+Chengauthor=K.+Saitoauthor=A.+V.+Pisarevauthor=M.+Wadaauthor=V.+P.+Pisarevaauthor=T.+V.+Pestovaauthor=M.+Gajdaauthor=A.+Roundauthor=C.+Kongauthor=M.+Limauthor=Y.+Nakamuraauthor=D.+I.+Svergunauthor=K.+Itoauthor=H.+Song&title=Structural+Insights+Into+eRF3+and+Stop+Codon+Recognition+by+eRF1&doi=10.1101%2Fgad.1770109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into eRF3 and stop codon recognition by eRF1</span></div><div class="casAuthors">Cheng, Zhihong; Saito, Kazuki; Pisarev, Andrey V.; Wada, Miki; Pisareva, Vera P.; Pestova, Tatyana V.; Gajda, Michal; Round, Adam; Kong, Chunguang; Lim, Mengkiat; Nakamura, Yoshikazu; Svergun, Dmitri I.; Ito, Koichi; Song, Haiwei</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1106-1118</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Eukaryotic translation termination is mediated by two interacting release factors, eRF1 and eRF3, which act cooperatively to ensure efficient stop codon recognition and fast polypeptide release.  The crystal structures of human and Schizosaccharomyces pombe full-length eRF1 in complex with eRF3 lacking the GTPase domain revealed details of the interaction between these two factors and marked conformational changes in eRF1 that occur upon binding to eRF3, leading eRF1 to resemble a tRNA mol.  Small-angle X-ray scattering anal. of the eRF1/eRF3/GTP complex suggested that eRF1's M domain contacts eRF3's GTPase domain.  Consistently, mutation of Arg192, which is predicted to come in close contact with the switch regions of eRF3, revealed its important role for eRF1's stimulatory effect on eRF3's GTPase activity.  An ATP mol. used as a crystn. additive was bound in eRF1's putative decoding area.  Mutational anal. of the ATP-binding site shed light on the mechanism of stop codon recognition by eRF1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdD0dkm3bQLVg90H21EOLACvtfcHk0lgGF46dWlzVoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Crs7c%253D&md5=57df8961d0eb44ab277dc2f45c25a63c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1101%2Fgad.1770109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1770109%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DPisarev%26aufirst%3DA.%2BV.%26aulast%3DWada%26aufirst%3DM.%26aulast%3DPisareva%26aufirst%3DV.%2BP.%26aulast%3DPestova%26aufirst%3DT.%2BV.%26aulast%3DGajda%26aufirst%3DM.%26aulast%3DRound%26aufirst%3DA.%26aulast%3DKong%26aufirst%3DC.%26aulast%3DLim%26aufirst%3DM.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DSvergun%26aufirst%3DD.%2BI.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DH.%26atitle%3DStructural%2520Insights%2520Into%2520eRF3%2520and%2520Stop%2520Codon%2520Recognition%2520by%2520eRF1%26jtitle%3DGenes%2520Develop.%26date%3D2009%26volume%3D23%26spage%3D1106%26epage%3D1118%26doi%3D10.1101%2Fgad.1770109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrio-Garcia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyler, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berninghausen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, R.</span></span> <span> </span><span class="NLM_article-title">Cryoelectron Microscopic Structures of Eukaryotic Translation Termination Complexes Containing eRF1-eRF3 or eRF1-ABCE1</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.04.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.celrep.2014.04.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25001285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFans7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=59-65&author=A.+Preisauthor=A.+Heuerauthor=C.+Barrio-Garciaauthor=A.+Hauserauthor=D.+E.+Eylerauthor=O.+Berninghausenauthor=R.+Greenauthor=T.+Beckerauthor=R.+Beckmann&title=Cryoelectron+Microscopic+Structures+of+Eukaryotic+Translation+Termination+Complexes+Containing+eRF1-eRF3+or+eRF1-ABCE1&doi=10.1016%2Fj.celrep.2014.04.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Cryoelectron Microscopic Structures of Eukaryotic Translation Termination Complexes Containing eRF1-eRF3 or eRF1-ABCE1</span></div><div class="casAuthors">Preis, Anne; Heuer, Andre; Barrio-Garcia, Clara; Hauser, Andreas; Eyler, Daniel E.; Berninghausen, Otto; Green, Rachel; Becker, Thomas; Beckmann, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-65</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Termination and ribosome recycling are essential processes in translation.  In eukaryotes, a stop codon in the ribosomal A site is decoded by a ternary complex consisting of release factors eRF1 and guanosine triphosphate (GTP)-bound eRF3.  After GTP hydrolysis, eRF3 dissocs., and ABCE1 can bind to eRF1-loaded ribosomes to stimulate peptide release and ribosomal subunit dissocn.  Here, we present cryoelectron microscopic (cryo-EM) structures of a pretermination complex contg. eRF1-eRF3 and a termination/prerecycling complex contg. eRF1-ABCE1. eRF1 undergoes drastic conformational changes: its central domain harboring the catalytically important GGQ loop is either packed against eRF3 or swung toward the peptidyl transferase center when bound to ABCE1.  Addnl., in complex with eRF3, the N-terminal domain of eRF1 positions the conserved NIKS motif proximal to the stop codon, supporting its suggested role in decoding, yet it appears to be delocalized in the presence of ABCE1.  These results suggest that stop codon decoding and peptide release can be uncoupled during termination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovl2-tKYfb5bVg90H21EOLACvtfcHk0lgcO6vuTaEFOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFans7jM&md5=28bff59bf85863587f9fb6234d266731</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.04.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.04.058%26sid%3Dliteratum%253Aachs%26aulast%3DPreis%26aufirst%3DA.%26aulast%3DHeuer%26aufirst%3DA.%26aulast%3DBarrio-Garcia%26aufirst%3DC.%26aulast%3DHauser%26aufirst%3DA.%26aulast%3DEyler%26aufirst%3DD.%2BE.%26aulast%3DBerninghausen%26aufirst%3DO.%26aulast%3DGreen%26aufirst%3DR.%26aulast%3DBecker%26aufirst%3DT.%26aulast%3DBeckmann%26aufirst%3DR.%26atitle%3DCryoelectron%2520Microscopic%2520Structures%2520of%2520Eukaryotic%2520Translation%2520Termination%2520Complexes%2520Containing%2520eRF1-eRF3%2520or%2520eRF1-ABCE1%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D59%26epage%3D65%26doi%3D10.1016%2Fj.celrep.2014.04.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L.'A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.%27A.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4CRBN+ubiquitin+ligase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lgcO6vuTaEFOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%2527A.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of Chemically Stable, Potent, and Efficacious MDM2 Inhibitors That Exploit the Retro-Mannich Ring-Opening-Cyclization Reaction Mechanism in Spiro-oxindoles</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10486</span>– <span class="NLM_lpage">10498</span>, <span class="refDoi"> DOI: 10.1021/jm501541j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501541j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10486-10498&author=A.+Aguilarauthor=W.+Sunauthor=L.+Liuauthor=J.+Luauthor=D.+McEachernauthor=D.+Bernardauthor=J.+R.+Deschampsauthor=S.+Wang&title=Design+of+Chemically+Stable%2C+Potent%2C+and+Efficacious+MDM2+Inhibitors+That+Exploit+the+Retro-Mannich+Ring-Opening-Cyclization+Reaction+Mechanism+in+Spiro-oxindoles&doi=10.1021%2Fjm501541j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Chemically Stable, Potent, and Efficacious MDM2 Inhibitors That Exploit the Retro-Mannich Ring-Opening-Cyclization Reaction Mechanism in Spiro-oxindoles</span></div><div class="casAuthors">Aguilar, Angelo; Sun, Wei; Liu, Liu; Lu, Jianfeng; McEachern, Donna; Bernard, Denzil; Deschamps, Jeffrey R.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10486-10498</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spiropyrrolidineoxindoles such as I (R = trans-4-hydroxycyclohexyl, 4-HO2CC6H4) were prepd. as inhibitors of the interaction between the proteins MDM2 and p53 for use as antitumor agents; the second-generation inhibitors incorporate sym. disubstitution in the pyrrolidine ring next to the spiro junction in order to facilitate isomerization via retro-Mannich/Mannich reactions to the more active stereoisomers (in which the aryl rings attached to the pyrrolidine are trans) as well as to improve stability in vitro and in vivo.  In particular, I (R = 4-HO2CC6H4) (MI-1061), which bound to MDM2 with a Ki value of 0.16 nM, showed excellent chem. stability and caused regression of human SJSA-1 xenograft tumors in mice.  The structures of diphenylhydroxyethyl spiropyrrolidineoxindole and spiropyrrolidineoxindolecarboxylate intermediates in the prepn. of I (R = trans-4-hydroxycyclohexyl, 4-HO2CC6H4) were detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_es94k2y2-rVg90H21EOLACvtfcHk0lhCxj5wUbNpvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nt73O&md5=6399fb5e19fc8d69e40e6fd19721e270</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm501541j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501541j%26sid%3Dliteratum%253Aachs%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520Chemically%2520Stable%252C%2520Potent%252C%2520and%2520Efficacious%2520MDM2%2520Inhibitors%2520That%2520Exploit%2520the%2520Retro-Mannich%2520Ring-Opening-Cyclization%2520Reaction%2520Mechanism%2520in%2520Spiro-oxindoles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10486%26epage%3D10498%26doi%3D10.1021%2Fjm501541j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Efremov, I.
V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajdos, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capetta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorff, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutra, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansour, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McColl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oborski, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, T.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J. M.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of a Novel Spiropyrrolidine Inhibitor of β-Secretase (BACE1) through Fragment-Based Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9069</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/jm201715d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201715d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFOnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9069-9088&author=I.%0AV.+Efremovauthor=F.+F.+Vajdosauthor=K.+A.+Borzilleriauthor=S.+Capettaauthor=H.+Chenauthor=P.+H.+Dorffauthor=J.+K.+Dutraauthor=S.+W.+Goldsteinauthor=M.+Mansourauthor=A.+McCollauthor=S.+Noellauthor=C.+E.+Oborskiauthor=T.+N.+O%E2%80%99Connellauthor=T.%0AJ.+O%E2%80%99Sullivanauthor=J.+Panditauthor=H.+Wangauthor=B.+Weiauthor=J.+M.+Withka&title=Discovery+and+Optimization+of+a+Novel+Spiropyrrolidine+Inhibitor+of+%CE%B2-Secretase+%28BACE1%29+through+Fragment-Based+Drug+Design&doi=10.1021%2Fjm201715d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of a novel spiropyrrolidine inhibitor of β-secretase (BACE1) through fragment-based drug design</span></div><div class="casAuthors">Efremov, Ivan V.; Vajdos, Felix F.; Borzilleri, Kris A.; Capetta, Steven; Chen, Hou; Dorff, Peter H.; Dutra, Jason K.; Goldstein, Steven W.; Mansour, Mahmoud; McColl, Alexander; Noell, Stephen; Oborski, Christine E.; O?Connell, Thomas N.; O?Sullivan, Theresa J.; Pandit, Jayvardhan; Wang, Hong; Wei, BinQing; Withka, Jane M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9069-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The aspartyl protease β-secretase, or BACE, has been demonstrated to be a key factor in the proteolytic formation of A,β-peptide, a major component of plaques in the brains of Alzheimer's disease (AD) patients, and inhibition of this enzyme has emerged as a major strategy for pharmacol. intervention in AD.  An X-ray-based fragment screen of Pfizer's proprietary fragment collection has resulted in the identification of a novel BACE binder featuring spiropyrrolidine framework.  Although exhibiting only weak inhibitory activity against the BACE enzyme, the small compd. was verified by biophys. and NMR-based methods as a bona fide BACE inhibitor.  Subsequent optimization of the lead compd., relying heavily on structure-based drug design and computational prediction of physiochem. properties, resulted in a nearly 1000-fold improvement in potency while maintaining ligand efficiency and properties predictive of good permeability and low P-gp liability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB8e9NnCgYXrVg90H21EOLACvtfcHk0lhCxj5wUbNpvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFOnsLs%253D&md5=c87263dd8136630e2e1f17a7fa9b5b9f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm201715d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201715d%26sid%3Dliteratum%253Aachs%26aulast%3DEfremov%26aufirst%3DI.%2BV.%26aulast%3DVajdos%26aufirst%3DF.%2BF.%26aulast%3DBorzilleri%26aufirst%3DK.%2BA.%26aulast%3DCapetta%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DDorff%26aufirst%3DP.%2BH.%26aulast%3DDutra%26aufirst%3DJ.%2BK.%26aulast%3DGoldstein%26aufirst%3DS.%2BW.%26aulast%3DMansour%26aufirst%3DM.%26aulast%3DMcColl%26aufirst%3DA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DOborski%26aufirst%3DC.%2BE.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DT.%2BN.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DT.%2BJ.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWithka%26aufirst%3DJ.%2BM.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520a%2520Novel%2520Spiropyrrolidine%2520Inhibitor%2520of%2520%25CE%25B2-Secretase%2520%2528BACE1%2529%2520through%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9069%26epage%3D9088%26doi%3D10.1021%2Fjm201715d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit141"><span><span> </span><span class="NLM_article-title">Molecular
Operating Environment (MOE), 2013.08</span>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span>,
1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular%0AOperating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+ULC%2C%0A1010+Sherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%250AOperating%2520Environment%2520%2528MOE%2529%252C%25202013.08%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Ben-Shalom, I. Y.</span>; <span class="NLM_string-name">Brozell, S. R.</span>; <span class="NLM_string-name">Cerutti, T. E.</span>; <span class="NLM_string-name">Cheatham, V. W. D.,  III</span>; <span class="NLM_string-name">Cruzeiro, T. A.</span>; <span class="NLM_string-name">Darden, R. E.</span>; <span class="NLM_string-name">Duke, D.</span>; <span class="NLM_string-name">Ghoreishi, M. K.</span>; <span class="NLM_string-name">Gilson, H.</span>; <span class="NLM_string-name">Gohlke, A. W.</span>; <span class="NLM_string-name">Goetz, D.</span>; <span class="NLM_string-name">Greene, R. H.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Izadi, S.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Kurtzman, T.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Li, P.</span>; <span class="NLM_string-name">Lin, C.</span>; <span class="NLM_string-name">Liu, J.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Mermelstein, D. J.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Miao, Y.</span>; <span class="NLM_string-name">Monard, G.</span>; <span class="NLM_string-name">Nguyen, C.</span>; <span class="NLM_string-name">Nguyen, H.</span>; <span class="NLM_string-name">Omelyan, I.</span>; <span class="NLM_string-name">Onufriev, A.</span>; <span class="NLM_string-name">Pan, F.</span>; <span class="NLM_string-name">Qi, R.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Schott-Verdugo, S.</span>; <span class="NLM_string-name">Shen, J.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, J.</span>; <span class="NLM_string-name">Salomon-Ferrer, R.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wei, H.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Xiao, L.</span>; <span class="NLM_string-name">York, D. M.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 2018</i>; University of California, San Francisco, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=D.+A.+Case&author=I.+Y.+Ben-Shalom&author=S.+R.+Brozell&author=T.+E.+Cerutti&author=V.+W.+D.+Cheatham&author=T.+A.+Cruzeiro&author=R.+E.+Darden&author=D.+Duke&author=M.+K.+Ghoreishi&author=H.+Gilson&author=A.+W.+Gohlke&author=D.+Goetz&author=R.+H.+Greene&author=N.+Homeyer&author=S.+Izadi&author=A.+Kovalenko&author=T.+Kurtzman&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=C.+Lin&author=J.+Liu&author=T.+Luchko&author=R.+Luo&author=D.+J.+Mermelstein&author=K.+M.+Merz&author=Y.+Miao&author=G.+Monard&author=C.+Nguyen&author=H.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=F.+Pan&author=R.+Qi&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=S.+Schott-Verdugo&author=J.+Shen&author=C.+L.+Simmerling&author=J.+Smith&author=R.+Salomon-Ferrer&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=H.+Wei&author=R.+M.+Wolf&author=X.+Wu&author=L.+Xiao&author=D.+M.+York&author=P.+A.+Kollman&title=AMBER+2018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%25202018%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lhDZ2Xo6FUeXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lhDZ2Xo6FUeXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lhDZ2Xo6FUeXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shargary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A Potent Small-Molecule Inhibitor of the MDM2-p53 Interaction (MI-888) Achieved Complete and Durable Tumor Regression in Mice</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5553</span>– <span class="NLM_lpage">5561</span>, <span class="refDoi"> DOI: 10.1021/jm4005708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4005708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslCisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5553-5561&author=Y.+Zhaoauthor=S.+Yuauthor=W.+Sunauthor=L.+Liuauthor=J.+Luauthor=D.+McEachernauthor=S.+Shargaryauthor=D.+Bernardauthor=X.+Liauthor=T.+Zhaoauthor=P.+Zouauthor=D.+Sunauthor=S.+Wang&title=A+Potent+Small-Molecule+Inhibitor+of+the+MDM2-p53+Interaction+%28MI-888%29+Achieved+Complete+and+Durable+Tumor+Regression+in+Mice&doi=10.1021%2Fjm4005708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent Small-Molecule Inhibitor of the MDM2-p53 Interaction (MI-888) Achieved Complete and Durable Tumor Regression in Mice</span></div><div class="casAuthors">Zhao, Yujun; Yu, Shanghai; Sun, Wei; Liu, Liu; Lu, Jianfeng; McEachern, Donna; Shargary, Sanjeev; Bernard, Denzil; Li, Xiaoqin; Zhao, Ting; Zou, Peng; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5553-5561</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors previously reported the discovery of a class of spirooxindoles as potent and selective small-mol. inhibitors of the MDM2-p53 interaction (MDM2 inhibitors).  The authors report herein the authors' efforts to improve their pharmacokinetic properties and in vivo antitumor activity.  The authors' efforts led to the identification of MI-888 as a potent MDM2 inhibitor (Ki = 0.44 nM) with a superior pharmacokinetic profile and enhanced in vivo efficacy.  MI-888 is capable of achieving rapid, complete, and durable tumor regression in two types of xenograft models of human cancer with oral administration and represents the most potent and efficacious MDM2 inhibitor reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr06DpGqpt0TLVg90H21EOLACvtfcHk0liKINYrZmJnmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslCisb4%253D&md5=a53931e3f91a1c1a302ac88bbb4d5c0d</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm4005708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4005708%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DShargary%26aufirst%3DS.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DZou%26aufirst%3DP.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520Potent%2520Small-Molecule%2520Inhibitor%2520of%2520the%2520MDM2-p53%2520Interaction%2520%2528MI-888%2529%2520Achieved%2520Complete%2520and%2520Durable%2520Tumor%2520Regression%2520in%2520Mice%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5553%26epage%3D5561%26doi%3D10.1021%2Fjm4005708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref140':['cit140'],'ref41':['cit41'],'ref42':['cit42'],'ref144':['cit141'],'ref145':['cit142'],'ref146':['cit143'],'ref147':['cit147'],'ref148':['cit148'],'ref43':['cit43']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 32 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Gisele Nishiguchi, Fatemeh Keramatnia, Jaeki Min, Yunchao Chang, Barbara Jonchere, Sourav Das, Marisa Actis, Jeanine Price, Divyabharathi Chepyala, Brandon Young, Kevin McGowan, P. Jake Slavish, Anand Mayasundari, Jamie A. Jarusiewicz, Lei Yang, Yong Li, Xiang Fu, Shalandus H. Garrett, James B. Papizan, Kiran Kodali, Junmin Peng, Shondra M. Pruett Miller, Martine F. Roussel, Charles Mullighan, Marcus Fischer, <span class="NLM_string-name hlFld-ContribAuthor">Zoran Rankovic</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Potent, Selective, and Orally Bioavailable Small-Molecule GSPT1/2 Degraders from a Focused Library of Cereblon Modulators. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7296-7311. <a href="https://doi.org/10.1021/acs.jmedchem.0c01313" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01313</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01313%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BPotent%25252C%252BSelective%25252C%252Band%252BOrally%252BBioavailable%252BSmall-Molecule%252BGSPT1%25252F2%252BDegraders%252Bfrom%252Ba%252BFocused%252BLibrary%252Bof%252BCereblon%252BModulators%26aulast%3DNishiguchi%26aufirst%3DGisele%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28072020%26date%3D27052021%26volume%3D64%26issue%3D11%26spage%3D7296%26epage%3D7311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maha Hanafi, Xinde Chen, <span class="NLM_string-name hlFld-ContribAuthor">Nouri Neamati</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1626-1648. <a href="https://doi.org/10.1021/acs.jmedchem.0c01897" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01897</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01897%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNapabucasin%252BPROTAC%252Bas%252Ban%252BEffective%252BDegrader%252Bof%252Bthe%252BE3%252BLigase%252BZFP91%26aulast%3DHanafi%26aufirst%3DMaha%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D03112020%26date%3D28012021%26volume%3D64%26issue%3D3%26spage%3D1626%26epage%3D1648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Liqun Chen, Yanke Chen, Chunyan Zhang, Bingyang Jiao, Sheng Liang, Qiong Tan, Hongyu Chai, Weihua Yu, Yongzheng Qian, Hui Yang, Wuyi Yao, Jianguo Yu, Ying Luo, Michael Plewe, Jialiang Wang, Xiao-Ran Han, <span class="NLM_string-name hlFld-ContribAuthor">Jing Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14562-14575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01342" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01342</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01342%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-In-Class%252BPotent%252Band%252BSelective%252BTropomyosin%252BReceptor%252BKinase%252BDegraders%26aulast%3DChen%26aufirst%3DLiqun%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07082020%26date%3D15102020%26volume%3D63%26issue%3D23%26spage%3D14562%26epage%3D14575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Duncan E. Scott, Timothy P. C. Rooney, Elliott D. Bayle, Tashfina Mirza, Henriette M. G. Willems, Jonathan H. Clarke, Stephen P. Andrews, <span class="NLM_string-name hlFld-ContribAuthor">John Skidmore</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Investigation of the Permeability of Androgen Receptor PROTACs. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1539-1547. <a href="https://doi.org/10.1021/acsmedchemlett.0c00194" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00194%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSystematic%252BInvestigation%252Bof%252Bthe%252BPermeability%252Bof%252BAndrogen%252BReceptor%252BPROTACs%26aulast%3DScott%26aufirst%3DDuncan%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15042020%26date%3D01062020%26date%3D08062020%26volume%3D11%26issue%3D8%26spage%3D1539%26epage%3D1547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hongying Gao, Xiuyun Sun, <span class="NLM_string-name hlFld-ContribAuthor">Yu Rao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC Technology: Opportunities and Challenges. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (3)
                                     , 237-240. <a href="https://doi.org/10.1021/acsmedchemlett.9b00597" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPROTAC%252BTechnology%25253A%252BOpportunities%252Band%252BChallenges%26aulast%3DGao%26aufirst%3DHongying%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D3%26spage%3D237%26epage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aleša  Bricelj</span>, <span class="hlFld-ContribAuthor ">Christian  Steinebach</span>, <span class="hlFld-ContribAuthor ">Robert  Kuchta</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Izidor  Sosič</span>. </span><span class="cited-content_cbyCitation_article-title">E3 Ligase Ligands in Successful PROTACs: An Overview of Syntheses and Linker Attachment Points. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.707317" title="DOI URL">https://doi.org/10.3389/fchem.2021.707317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.707317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.707317%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DE3%252BLigase%252BLigands%252Bin%252BSuccessful%252BPROTACs%25253A%252BAn%252BOverview%252Bof%252BSyntheses%252Band%252BLinker%252BAttachment%252BPoints%26aulast%3DBricelj%26aufirst%3DAle%25C5%25A1a%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Wang</span>, <span class="hlFld-ContribAuthor ">Jin  Liu</span>, <span class="hlFld-ContribAuthor ">Ira  Tandon</span>, <span class="hlFld-ContribAuthor ">Suzhen  Wu</span>, <span class="hlFld-ContribAuthor ">Peng  Teng</span>, <span class="hlFld-ContribAuthor ">Jianhua  Liao</span>, <span class="hlFld-ContribAuthor ">Weiping  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of MDM2 degraders based on ligands derived from Ugi reactions: Lessons and discoveries. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>219 </em>, 113425. <a href="https://doi.org/10.1016/j.ejmech.2021.113425" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113425</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113425%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252BMDM2%252Bdegraders%252Bbased%252Bon%252Bligands%252Bderived%252Bfrom%252BUgi%252Breactions%25253A%252BLessons%252Band%252Bdiscoveries%26aulast%3DWang%26aufirst%3DBo%26date%3D2021%26volume%3D219%26spage%3D113425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue‐Yan  Ai</span>, <span class="hlFld-ContribAuthor ">Dong‐Ai  Li</span>, <span class="hlFld-ContribAuthor ">Guo  Li</span>, <span class="hlFld-ContribAuthor ">He‐Ping  Li</span>, <span class="hlFld-ContribAuthor ">Xiang‐Hong  He</span>, <span class="hlFld-ContribAuthor ">Xue‐Ju  Fu</span>, <span class="hlFld-ContribAuthor ">Yu‐Ting  Wang</span>, <span class="hlFld-ContribAuthor ">Gu  Zhan</span>, <span class="hlFld-ContribAuthor ">Bo  Han</span>. </span><span class="cited-content_cbyCitation_article-title">Asymmetric Synthesis of Spirocyclopentane Oxindoles
              via
              [2+3] Annulation with 2‐(2‐Oxoindolin‐3‐yl)malononitriles as 1,2‐Carbon Bisnucleophiles. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2021,</strong> <em>363 </em>
                                    (13)
                                     , 3283-3289. <a href="https://doi.org/10.1002/adsc.202100235" title="DOI URL">https://doi.org/10.1002/adsc.202100235</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.202100235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.202100235%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DAsymmetric%252BSynthesis%252Bof%252BSpirocyclopentane%252BOxindoles%252Bvia%252B%25255B2%25252B3%25255D%252BAnnulation%252Bwith%252B2%2525E2%252580%252590%2525282%2525E2%252580%252590Oxoindolin%2525E2%252580%2525903%2525E2%252580%252590yl%252529malononitriles%252Bas%252B1%25252C2%2525E2%252580%252590Carbon%252BBisnucleophiles%26aulast%3DAi%26aufirst%3DYue%25E2%2580%2590Yan%26date%3D2021%26date%3D2021%26volume%3D363%26issue%3D13%26spage%3D3283%26epage%3D3289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Reynders</span>, <span class="hlFld-ContribAuthor ">Dirk  Trauner</span>. </span><span class="cited-content_cbyCitation_article-title">Optical control of targeted protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 969-986. <a href="https://doi.org/10.1016/j.chembiol.2021.05.010" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.05.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.05.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DOptical%252Bcontrol%252Bof%252Btargeted%252Bprotein%252Bdegradation%26aulast%3DReynders%26aufirst%3DMartin%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D969%26epage%3D986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhiwen  Qi</span>, <span class="hlFld-ContribAuthor ">Guliang  Yang</span>, <span class="hlFld-ContribAuthor ">Tao  Deng</span>, <span class="hlFld-ContribAuthor ">Jianmin  Wang</span>, <span class="hlFld-ContribAuthor ">Hao  Zhou</span>, <span class="hlFld-ContribAuthor ">Sergey A.  Popov</span>, <span class="hlFld-ContribAuthor ">Elvira E.  Shults</span>, <span class="hlFld-ContribAuthor ">Chengzhang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Design and linkage optimization of ursane-thalidomide-based PROTACs and identification of their targeted-degradation properties to MDM2 protein. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>111 </em>, 104901. <a href="https://doi.org/10.1016/j.bioorg.2021.104901" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104901</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104901%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%252Band%252Blinkage%252Boptimization%252Bof%252Bursane-thalidomide-based%252BPROTACs%252Band%252Bidentification%252Bof%252Btheir%252Btargeted-degradation%252Bproperties%252Bto%252BMDM2%252Bprotein%26aulast%3DQi%26aufirst%3DZhiwen%26date%3D2021%26volume%3D111%26spage%3D104901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Yujing  Zhang</span>, <span class="hlFld-ContribAuthor ">Dongming  Xing</span>, <span class="hlFld-ContribAuthor ">Renshuai  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs technology for targeting non-oncoproteins: advances and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>98 </em>, 105109. <a href="https://doi.org/10.1016/j.bioorg.2021.105109" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105109%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DPROTACs%252Btechnology%252Bfor%252Btargeting%252Bnon-oncoproteins%25253A%252Badvances%252Band%252Bperspectives%26aulast%3DWang%26aufirst%3DChao%26date%3D2021%26volume%3D98%26spage%3D105109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shipeng  He</span>, <span class="hlFld-ContribAuthor ">Junhui  Ma</span>, <span class="hlFld-ContribAuthor ">Yuxin  Fang</span>, <span class="hlFld-ContribAuthor ">Ying  Liu</span>, <span class="hlFld-ContribAuthor ">Shanchao  Wu</span>, <span class="hlFld-ContribAuthor ">Guoqiang  Dong</span>, <span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Chunquan  Sheng</span>. </span><span class="cited-content_cbyCitation_article-title">Homo-PROTAC mediated suicide of MDM2 to treat non-small cell lung cancer. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (6)
                                     , 1617-1628. <a href="https://doi.org/10.1016/j.apsb.2020.11.022" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.11.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.11.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.11.022%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DHomo-PROTAC%252Bmediated%252Bsuicide%252Bof%252BMDM2%252Bto%252Btreat%252Bnon-small%252Bcell%252Blung%252Bcancer%26aulast%3DHe%26aufirst%3DShipeng%26date%3D2021%26volume%3D11%26issue%3D6%26spage%3D1617%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqiang  Zheng</span>, <span class="hlFld-ContribAuthor ">Jin  Yan</span>, <span class="hlFld-ContribAuthor ">Weiming  You</span>, <span class="hlFld-ContribAuthor ">Fanni  Li</span>, <span class="hlFld-ContribAuthor ">Jiajie  Diao</span>, <span class="hlFld-ContribAuthor ">Wangxiao  He</span>, <span class="hlFld-ContribAuthor ">Yu  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">De Novo Nano‐Erythrocyte Structurally Braced by Biomimetic Au(I)‐peptide Skeleton for MDM2/MDMX Predation toward Augmented Pulmonary Adenocarcinoma Immunotherapy. </span><span class="cited-content_cbyCitation_journal-name">Small</span><span> <strong>2021,</strong> <em>17 </em>
                                    (20)
                                     , 2100394. <a href="https://doi.org/10.1002/smll.202100394" title="DOI URL">https://doi.org/10.1002/smll.202100394</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/smll.202100394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fsmll.202100394%26sid%3Dliteratum%253Aachs%26jtitle%3DSmall%26atitle%3DDe%252BNovo%252BNano%2525E2%252580%252590Erythrocyte%252BStructurally%252BBraced%252Bby%252BBiomimetic%252BAu%252528I%252529%2525E2%252580%252590peptide%252BSkeleton%252Bfor%252BMDM2%25252FMDMX%252BPredation%252Btoward%252BAugmented%252BPulmonary%252BAdenocarcinoma%252BImmunotherapy%26aulast%3DZheng%26aufirst%3DXiaoqiang%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D20%26spage%3D2100394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zunyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Zhen  Ma</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Selective degradation of the estrogen receptor in the treatment of cancers. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Steroid Biochemistry and Molecular Biology</span><span> <strong>2021,</strong> <em>209 </em>, 105848. <a href="https://doi.org/10.1016/j.jsbmb.2021.105848" title="DOI URL">https://doi.org/10.1016/j.jsbmb.2021.105848</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jsbmb.2021.105848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jsbmb.2021.105848%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Steroid%2520Biochemistry%2520and%2520Molecular%2520Biology%26atitle%3DSelective%252Bdegradation%252Bof%252Bthe%252Bestrogen%252Breceptor%252Bin%252Bthe%252Btreatment%252Bof%252Bcancers%26aulast%3DWang%26aufirst%3DZunyuan%26date%3D2021%26volume%3D209%26spage%3D105848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander J.  Boddy</span>, <span class="hlFld-ContribAuthor ">James A.  Bull</span>. </span><span class="cited-content_cbyCitation_article-title">Stereoselective synthesis and applications of spirocyclic oxindoles. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2021,</strong> <em>8 </em>
                                    (5)
                                     , 1026-1084. <a href="https://doi.org/10.1039/D0QO01085E" title="DOI URL">https://doi.org/10.1039/D0QO01085E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0QO01085E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0QO01085E%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DStereoselective%252Bsynthesis%252Band%252Bapplications%252Bof%252Bspirocyclic%252Boxindoles%26aulast%3DBoddy%26aufirst%3DAlexander%2BJ.%26date%3D2021%26date%3D2021%26volume%3D8%26issue%3D5%26spage%3D1026%26epage%3D1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chen-xi  Zhao</span>, <span class="hlFld-ContribAuthor ">Chen-ming  Zeng</span>, <span class="hlFld-ContribAuthor ">Ke  Wang</span>, <span class="hlFld-ContribAuthor ">Qiao-jun  He</span>, <span class="hlFld-ContribAuthor ">Bo  Yang</span>, <span class="hlFld-ContribAuthor ">Fan-fan  Zhou</span>, <span class="hlFld-ContribAuthor ">Hong  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Ubiquitin–proteasome system-targeted therapy for uveal melanoma: what is the evidence?. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2021,</strong> <em>42 </em>
                                    (2)
                                     , 179-188. <a href="https://doi.org/10.1038/s41401-020-0441-3" title="DOI URL">https://doi.org/10.1038/s41401-020-0441-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41401-020-0441-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41401-020-0441-3%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DUbiquitin%2525E2%252580%252593proteasome%252Bsystem-targeted%252Btherapy%252Bfor%252Buveal%252Bmelanoma%25253A%252Bwhat%252Bis%252Bthe%252Bevidence%25253F%26aulast%3DZhao%26aufirst%3DChen-xi%26date%3D2021%26date%3D2020%26volume%3D42%26issue%3D2%26spage%3D179%26epage%3D188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Shiliang  Wu</span>, <span class="hlFld-ContribAuthor ">Lifei  Wang</span>, <span class="hlFld-ContribAuthor ">Zhiyong  Yang</span>, <span class="hlFld-ContribAuthor ">Juan  Zhao</span>, <span class="hlFld-ContribAuthor ">Lulu  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Binding selectivity of inhibitors toward the first over the second bromodomain of BRD4: theoretical insights from free energy calculations and multiple short molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 745-759. <a href="https://doi.org/10.1039/D0RA09469B" title="DOI URL">https://doi.org/10.1039/D0RA09469B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA09469B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA09469B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DBinding%252Bselectivity%252Bof%252Binhibitors%252Btoward%252Bthe%252Bfirst%252Bover%252Bthe%252Bsecond%252Bbromodomain%252Bof%252BBRD4%25253A%252Btheoretical%252Binsights%252Bfrom%252Bfree%252Benergy%252Bcalculations%252Band%252Bmultiple%252Bshort%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DWang%26aufirst%3DYan%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D745%26epage%3D759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Ma</span>, <span class="hlFld-ContribAuthor ">Xiao-Xiao  Han</span>, <span class="hlFld-ContribAuthor ">Xiao-Ming  Yang</span>, <span class="hlFld-ContribAuthor ">Song-Lin  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera technology: a novel strategy for treating diseases of the central nervous system. </span><span class="cited-content_cbyCitation_journal-name">Neural Regeneration Research</span><span> <strong>2021,</strong> <em>16 </em>
                                    (10)
                                     , 1944. <a href="https://doi.org/10.4103/1673-5374.308075" title="DOI URL">https://doi.org/10.4103/1673-5374.308075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4103/1673-5374.308075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4103%2F1673-5374.308075%26sid%3Dliteratum%253Aachs%26jtitle%3DNeural%2520Regeneration%2520Research%26atitle%3DProteolysis%252Btargeting%252Bchimera%252Btechnology%25253A%252Ba%252Bnovel%252Bstrategy%252Bfor%252Btreating%252Bdiseases%252Bof%252Bthe%252Bcentral%252Bnervous%252Bsystem%26aulast%3DMa%26aufirst%3DKe%26date%3D2021%26volume%3D16%26issue%3D10%26spage%3D1944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Nicola  Mautone</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112750. <a href="https://doi.org/10.1016/j.ejmech.2020.112750" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bepigenetic%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528Epi-PROTACs%252529%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26volume%3D207%26spage%3D112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Jieqing  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC: A Novel Technology for Drug Development**. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (42)
                                     , 13232-13247. <a href="https://doi.org/10.1002/slct.202003162" title="DOI URL">https://doi.org/10.1002/slct.202003162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202003162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202003162%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DPROTAC%25253A%252BA%252BNovel%252BTechnology%252Bfor%252BDrug%252BDevelopment%252A%252A%26aulast%3DLi%26aufirst%3DJie%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D42%26spage%3D13232%26epage%3D13247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingxing  Hu</span>, <span class="hlFld-ContribAuthor ">Weilin  Zhou</span>, <span class="hlFld-ContribAuthor ">Yijie  Wang</span>, <span class="hlFld-ContribAuthor ">Dongping  Yao</span>, <span class="hlFld-ContribAuthor ">Tinghong  Ye</span>, <span class="hlFld-ContribAuthor ">Yuqin  Yao</span>, <span class="hlFld-ContribAuthor ">Bin  Chen</span>, <span class="hlFld-ContribAuthor ">Gongping  Liu</span>, <span class="hlFld-ContribAuthor ">Xifei  Yang</span>, <span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Yongmei  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of the first potent proteolysis targeting chimera (PROTAC) degrader of indoleamine 2,3-dioxygenase 1. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>10 </em>
                                    (10)
                                     , 1943-1953. <a href="https://doi.org/10.1016/j.apsb.2020.02.010" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.02.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.02.010%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bthe%252Bfirst%252Bpotent%252Bproteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bdegrader%252Bof%252Bindoleamine%252B2%25252C3-dioxygenase%252B1%26aulast%3DHu%26aufirst%3DMingxing%26date%3D2020%26volume%3D10%26issue%3D10%26spage%3D1943%26epage%3D1953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura M.  Luh</span>, <span class="hlFld-ContribAuthor ">Ulrike  Scheib</span>, <span class="hlFld-ContribAuthor ">Katrin  Juenemann</span>, <span class="hlFld-ContribAuthor ">Lars  Wortmann</span>, <span class="hlFld-ContribAuthor ">Michael  Brands</span>, <span class="hlFld-ContribAuthor ">Philipp M.  Cromm</span>. </span><span class="cited-content_cbyCitation_article-title">Beute für das Proteasom: Gezielter Proteinabbau aus medizinalchemischer Perspektive. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (36)
                                     , 15576-15595. <a href="https://doi.org/10.1002/ange.202004310" title="DOI URL">https://doi.org/10.1002/ange.202004310</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202004310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202004310%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DBeute%252Bf%2525C3%2525BCr%252Bdas%252BProteasom%25253A%252BGezielter%252BProteinabbau%252Baus%252Bmedizinalchemischer%252BPerspektive%26aulast%3DLuh%26aufirst%3DLaura%2BM.%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D36%26spage%3D15576%26epage%3D15595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura M.  Luh</span>, <span class="hlFld-ContribAuthor ">Ulrike  Scheib</span>, <span class="hlFld-ContribAuthor ">Katrin  Juenemann</span>, <span class="hlFld-ContribAuthor ">Lars  Wortmann</span>, <span class="hlFld-ContribAuthor ">Michael  Brands</span>, <span class="hlFld-ContribAuthor ">Philipp M.  Cromm</span>. </span><span class="cited-content_cbyCitation_article-title">Prey for the Proteasome: Targeted Protein Degradation—A Medicinal Chemist's Perspective. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (36)
                                     , 15448-15466. <a href="https://doi.org/10.1002/anie.202004310" title="DOI URL">https://doi.org/10.1002/anie.202004310</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202004310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202004310%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DPrey%252Bfor%252Bthe%252BProteasome%25253A%252BTargeted%252BProtein%252BDegradation%2525E2%252580%252594A%252BMedicinal%252BChemist%252527s%252BPerspective%26aulast%3DLuh%26aufirst%3DLaura%2BM.%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D36%26spage%3D15448%26epage%3D15466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yilin  Chen</span>, <span class="hlFld-ContribAuthor ">Jianping  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">The application of ubiquitin ligases in the PROTAC drug design. </span><span class="cited-content_cbyCitation_journal-name">Acta Biochimica et Biophysica Sinica</span><span> <strong>2020,</strong> <em>52 </em>
                                    (7)
                                     , 776-790. <a href="https://doi.org/10.1093/abbs/gmaa053" title="DOI URL">https://doi.org/10.1093/abbs/gmaa053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abbs/gmaa053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabbs%2Fgmaa053%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Biochimica%2520et%2520Biophysica%2520Sinica%26atitle%3DThe%252Bapplication%252Bof%252Bubiquitin%252Bligases%252Bin%252Bthe%252BPROTAC%252Bdrug%252Bdesign%26aulast%3DChen%26aufirst%3DYilin%26date%3D2020%26date%3D2020%26volume%3D52%26issue%3D7%26spage%3D776%26epage%3D790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Fang</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule MDM2/X inhibitors and PROTAC degraders for cancer therapy: advances and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>10 </em>
                                    (7)
                                     , 1253-1278. <a href="https://doi.org/10.1016/j.apsb.2020.01.003" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.01.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.01.003%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DSmall-molecule%252BMDM2%25252FX%252Binhibitors%252Band%252BPROTAC%252Bdegraders%252Bfor%252Bcancer%252Btherapy%25253A%252Badvances%252Band%252Bperspectives%26aulast%3DFang%26aufirst%3DYuan%26date%3D2020%26volume%3D10%26issue%3D7%26spage%3D1253%26epage%3D1278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abiodun  Anifowose</span>, <span class="hlFld-ContribAuthor ">Ayodeji A.  Agbowuro</span>, <span class="hlFld-ContribAuthor ">Xiaoxiao  Yang</span>, <span class="hlFld-ContribAuthor ">Binghe  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer strategies by upregulating p53 through inhibition of its ubiquitination by MDM2. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2020,</strong> <em>29 </em>
                                    (7)
                                     , 1105-1121. <a href="https://doi.org/10.1007/s00044-020-02574-9" title="DOI URL">https://doi.org/10.1007/s00044-020-02574-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-020-02574-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-020-02574-9%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DAnticancer%252Bstrategies%252Bby%252Bupregulating%252Bp53%252Bthrough%252Binhibition%252Bof%252Bits%252Bubiquitination%252Bby%252BMDM2%26aulast%3DAnifowose%26aufirst%3DAbiodun%26date%3D2020%26date%3D2020%26volume%3D29%26issue%3D7%26spage%3D1105%26epage%3D1121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ka  Yang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhao</span>, <span class="hlFld-ContribAuthor ">Xueqing  Nie</span>, <span class="hlFld-ContribAuthor ">Hao  Wu</span>, <span class="hlFld-ContribAuthor ">Bo  Wang</span>, <span class="hlFld-ContribAuthor ">Chelsi M.  Almodovar-Rivera</span>, <span class="hlFld-ContribAuthor ">Haibo  Xie</span>, <span class="hlFld-ContribAuthor ">Weiping  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">A Cell-Based Target Engagement Assay for the Identification of Cereblon E3 Ubiquitin Ligase Ligands and Their Application in HDAC6 Degraders. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (7)
                                     , 866-876.e8. <a href="https://doi.org/10.1016/j.chembiol.2020.04.008" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.04.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.04.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DA%252BCell-Based%252BTarget%252BEngagement%252BAssay%252Bfor%252Bthe%252BIdentification%252Bof%252BCereblon%252BE3%2525C2%2525A0Ubiquitin%252BLigase%252BLigands%252Band%252BTheir%252BApplication%252Bin%252BHDAC6%252BDegraders%26aulast%3DYang%26aufirst%3DKa%26date%3D2020%26volume%3D27%26issue%3D7%26spage%3D866%26epage%3D876.e8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Pan  Chen</span>, <span class="hlFld-ContribAuthor ">Peiyu  Zhu</span>, <span class="hlFld-ContribAuthor ">Peiyuan  Zheng</span>, <span class="hlFld-ContribAuthor ">Tao  Wang</span>, <span class="hlFld-ContribAuthor ">Lixun  Wang</span>, <span class="hlFld-ContribAuthor ">Changliang  Xu</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>, <span class="hlFld-ContribAuthor ">Huibin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of small-molecule BRD4 degraders based on pyrrolopyridone derivative. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103817. <a href="https://doi.org/10.1016/j.bioorg.2020.103817" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103817</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103817%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDevelopment%252Bof%252Bsmall-molecule%252BBRD4%252Bdegraders%252Bbased%252Bon%252Bpyrrolopyridone%252Bderivative%26aulast%3DZhang%26aufirst%3DJian%26date%3D2020%26volume%3D99%26spage%3D103817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Fang  Xu</span>, <span class="hlFld-ContribAuthor ">Linjiang  Tong</span>, <span class="hlFld-ContribAuthor ">Tao  Zhang</span>, <span class="hlFld-ContribAuthor ">Hua  Xie</span>, <span class="hlFld-ContribAuthor ">Xiaoyun  Lu</span>, <span class="hlFld-ContribAuthor ">Xiaomei  Ren</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of selective degraders of EGFRL858R/T790M mutant. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112199. <a href="https://doi.org/10.1016/j.ejmech.2020.112199" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112199</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112199%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bselective%252Bdegraders%252Bof%252BEGFRL858R%25252FT790M%252Bmutant%26aulast%3DZhang%26aufirst%3DXin%26date%3D2020%26volume%3D192%26spage%3D112199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kevin  Moreau</span>, <span class="hlFld-ContribAuthor ">Muireann  Coen</span>, <span class="hlFld-ContribAuthor ">Andrew X.  Zhang</span>, <span class="hlFld-ContribAuthor ">Fiona  Pachl</span>, <span class="hlFld-ContribAuthor ">M. Paola  Castaldi</span>, <span class="hlFld-ContribAuthor ">Goran  Dahl</span>, <span class="hlFld-ContribAuthor ">Helen  Boyd</span>, <span class="hlFld-ContribAuthor ">Clay  Scott</span>, <span class="hlFld-ContribAuthor ">Pete  Newham</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis‐targeting chimeras in drug development: A safety perspective. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2020,</strong> <em>177 </em>
                                    (8)
                                     , 1709-1718. <a href="https://doi.org/10.1111/bph.15014" title="DOI URL">https://doi.org/10.1111/bph.15014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.15014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.15014%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DProteolysis%2525E2%252580%252590targeting%252Bchimeras%252Bin%252Bdrug%252Bdevelopment%25253A%252BA%252Bsafety%252Bperspective%26aulast%3DMoreau%26aufirst%3DKevin%26date%3D2020%26date%3D2020%26volume%3D177%26issue%3D8%26spage%3D1709%26epage%3D1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuyun  Sun</span>, <span class="hlFld-ContribAuthor ">Hongying  Gao</span>, <span class="hlFld-ContribAuthor ">Yiqing  Yang</span>, <span class="hlFld-ContribAuthor ">Ming  He</span>, <span class="hlFld-ContribAuthor ">Yue  Wu</span>, <span class="hlFld-ContribAuthor ">Yugang  Song</span>, <span class="hlFld-ContribAuthor ">Yan  Tong</span>, <span class="hlFld-ContribAuthor ">Yu  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: great opportunities for academia and industry. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-019-0101-6" title="DOI URL">https://doi.org/10.1038/s41392-019-0101-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DPROTACs%25253A%252Bgreat%252Bopportunities%252Bfor%252Bacademia%252Band%252Bindustry%26aulast%3DSun%26aufirst%3DXiuyun%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of <b>MG-277</b> as a new, putative PROTAC MDM2 degrader. (A) Structures of MDM2 inhibitors MI-1061 and MI-2103, the bona fide MDM2 degrader MD-222 and a putative MDM2 degrader <b>MG-277</b>. (B–D) Binding affinities of MI-1061 (B), MI-2103 (C) and <b>MG-277</b> (D) to MDM2 by an optimized FP competitive binding assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <b>MG-277</b> is much less effective and less potent than MD-222 in inducing MDM2 degradation and fails to activate wild-type p53. (A,B) Western blotting analysis of MDM2 and p53 protein levels in RS4;11 cells after treatment with MI-2103, <b>MG-277</b>, and MD-222 for 1 (A) or 2 h (B), and in p53 mutant RS4;11/IRMI-2 cell line after 2 h treatment (C). The cells were treated for 1 or 2 h with the compounds at the indicated concentrations, and proteins were probed by specific antibodies. (D–G) Changes of mRNA levels of p53 downstream target genes, including <i>MDM2</i>, <i>CDKN1A</i>, <i>PUMA</i>, and <i>BAX</i> were analyzed after treatment with 3 nM <b>MG-277</b> or 10 nM MD-222 in the RS4;11 cell line by qRT-PCR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cell growth inhibition activity of <b>MG-277</b> is MDM2-independent. (A–D) Effect of knock-down of MDM2 on the cell growth inhibitory activity of MI-2103 and <b>MG-277</b> in MDA-MB-231 and MDA-MB-468 cells after transfection with MDM2 siRNAs (C,D) or nontargeting siRNA control vector (si CRL) (A,B) for 2 days. (E) MDM2 knockdown efficiency was determined by Western blotting analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cell growth inhibition activity of <b>MG-277</b> does not require binding to MDM2. (A–C) Structures of MC-215, MC-216, and MC-217, three control analogues of <b>MG-277</b> formed by removal of fluorine and/or one or two chlorine atoms from the MI-2103 portion of <b>MG-277</b>. (D,E) Cell growth inhibitory activity of <b>MG-277</b>, MC-215, MC-216, and MC-217 in RS4;11 (D) and RS4;11/IRMI-2 (E) cell lines. (F) Binding affinity of the three analogues of <b>MG-277</b> with MDM2 protein was determined by an FP assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cellular activity of <b>MG-277</b> is dependent on cereblon binding. (A–D) Cell growth inhibitory activity of MI-2103, MD-222, and <b>MG-277</b> in RS4;11, RS4;11/IRMI-2, MDA-MB-231, and MDA-MB-468 cells in the presence or absence of excess lenalidomide to occupy the binding site on cereblon. (E–H) Cell growth inhibitory activity of MI-2103, <b>MG-277</b>, and MC-024 with no cereblon binding in RS4;11, RS4;11/IRMI-2, MDA-MB-231, and MDA-MB-468 cell lines by a 4-day cell viability assay. (I) Structure of MC-024 with the amino group of the glutarimide in <b>MG-277</b> methylated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cell growth inhibition activity of <b>MG-277</b> requires binding to cereblon. The effect of siRNA <i>CRBN</i> knock-down on the cell growth inhibitory activity of <b>MG-277</b>. (A–F) MDA-MB-231 (A–C) and MDA-MB-468 (D–F) cells were transfected with two different <i>CRBN</i> siRNAs targeting two different sequences (B–C, E–F) or nontargeting siRNA control vector (A,D) before treatment with MI-2103 or <b>MG-277</b> for 3 days. Cell growth inhibition was determined by cell viability assay. (G) Western blotting for the cereblon protein level was used to determine the knockdown efficiency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <b>MG-277</b> effectively and specifically induces the degradation of GSPT1. (A,B) Proteomic analysis/protein profiling after a 3 h treatment with 0.1 μM of <b>MG-277</b> in p53 wild-type RS4;11 (A) and p53 mutated RS4;11/IRMI-2 (B) cells. The <i>Y</i>-axis represents <i>P</i>-value and <i>X</i>-axis represents the log 2 ratio of the level of a certain protein after the treatment to its protein level before the treatment. (C) GSPT1 mRNA level was measured by qRT-PCR after treatment with <b>MG-277</b> and MD-222 at indicated time points. (D–G) Western blotting analysis of the GSPT1 protein level after treatment with MI-2103, <b>MG-277</b>, and MD-222 for 2 h (D,E) or treatment with <b>MG-277</b> or MD-222 for different time lengths as indicated (F,G) in RS4;11 and RS4;11/IRMI-2 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Decrease of the GSPT1 protein level induced by <b>MG-277</b> is dependent on its binding with cereblon, proteasome, and CRLs but independent of MDM2. (A,B) Western blotting for the GSPT1 protein level after 2 h treatment with MI-2103, <b>MG-277</b>, or MD-222 in MDA-MB-231 (A) and MDA-MB-468 (B) cells with or without lenalidomide (30 μM) to block binding with cereblon. (C,D) MDA-MB-231 (C) and MDA-MB-468 (D) cells were transfected with <i>CRBN</i> siRNAs targeting two different sequences or nontargeting siRNA control. Protein levels of GSPT1 and cereblon were determined by western blot analysis after treatment with 0.1 μM <b>MG-277</b> for 2 h or 24 h in the transfected cells. (E) Western blot analysis for the GSPT1 protein level after 2 h treatment with <b>MG-277</b> in the presence or absence of 10 μM MI-2103 [or dimethylsulfoxide (DMSO) as control] to block binding with MDM2 in MDA-MB-231 cells. (F) Western blotting for MDM2 and GSPT1 protein levels after 2 h treatment with <b>MG-277</b> and three analogues (MC-215, MC-216 and MC-217) with weaker MDM2 binding affinity in RS4;11 cells. (G) RS4;11 cells were pre-incubated with a proteasome inhibitor, PR-171 or a Nedd8 inhibitor, MLN4924, or DMSO as control for 4 h to block proteasome or CRL function. The GSPT1 protein level was measured by western blotting after 2 h treatment with MI-2103, <b>MG-277</b>, or MD-222.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Molecular modeling of <b>MG-277</b> (yellow) binding with GSPT1 (blue) and cereblon (orange) with highlighted residues, Phe150 and Glu377, which are uniquely critical for recruitment of GSPT1. The available structure of GSPT1-cereblon complex co-crystalized with small ligand CC-885<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(40)</a> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HXB">5HXB</a>) was used.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Chemical structural requirements for the GSPT1 recruitment to cereblon. (A) Chemical structures of the new compounds after modifications of the inhibitor portion to investigate the structural requirement of GSPT1 recruitment. (B,C) Potencies of the compounds in cell growth inhibition were evaluated in both p53 wild-type RS4;11 cell line (B) and the p53 mutant RS4;11/IRMI-2 cell line (C) by a 4-day cell viability assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Linker Modifications</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Removal of Halogens and Methylation of Pyrrolidine and Glutarimide Nitrogens</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Systematic Removal of Pyrrolidine Substituents</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/medium/jm9b00846_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Removal of Pyrrolidine Substituents</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00846/20191108/images/large/jm9b00846_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00846&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82745" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82745" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: Chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+Chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lgoFFZMaLl77A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520Chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: An Emerging Targeting Technique for Protein Degradation in Drug Discovery</span>. <i>Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1700247</span>, <span class="refDoi"> DOI: 10.1002/bies.201700247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fbies.201700247" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2018&pages=1700247&author=S.+Guauthor=D.+Cuiauthor=X.+Chenauthor=X.+Xiongauthor=Y.+Zhao&title=PROTACs%3A+An+Emerging+Targeting+Technique+for+Protein+Degradation+in+Drug+Discovery&doi=10.1002%2Fbies.201700247"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fbies.201700247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbies.201700247%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520An%2520Emerging%2520Targeting%2520Technique%2520for%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%26date%3D2018%26volume%3D40%26spage%3D1700247%26doi%3D10.1002%2Fbies.201700247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Waste Disposal-An Attractive Strategy for Cancer Therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1126/science.aam7340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.aam7340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28302825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1163-1167&author=J.+Salamiauthor=C.+M.+Crews&title=Waste+Disposal-An+Attractive+Strategy+for+Cancer+Therapy&doi=10.1126%2Fscience.aam7340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Waste disposal-An attractive strategy for cancer therapy</span></div><div class="casAuthors">Salami, Jemilat; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Targeted therapies for cancer are typically small mols. or monoclonal antibodies that act by inhibiting the activity of specific proteins that drive tumor growth.  Although many of these drugs are effective in cancer patients, the response is often not durable because tumor cells develop resistance to the drugs.  Another limitation of this strategy is that not all oncogenic driver proteins are "druggable" enzymes or receptors with activities that can be inhibited.  Here the authors describe an alternative approach to targeted therapy that is based on co-opting the cellular quality-control machinery-the ubiquitin-proteasome system-to remove specific cancer-causing proteins from the cell.  The authors first discuss examples of existing cancer drugs that work by degrading specific proteins and then review recent progress in the rational design and preclin. testing of small mols. that induce selective degrdn. of specific target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVoZn-YpfXmbVg90H21EOLACvtfcHk0ljs0xgiC7-kDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D&md5=918621d50a82b7797aa0300537a4486e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7340%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DWaste%2520Disposal-An%2520Attractive%2520Strategy%2520for%2520Cancer%2520Therapy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.aam7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras%3A+Induced+Protein+Degradation+as+a+Therapeutic+Strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0ljs0xgiC7-kDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%253A%2520Induced%2520Protein%2520Degradation%2520as%2520a%2520Therapeutic%2520Strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by PROTACs</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.pharmthera.2017.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28223226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=138-144&author=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=C.+M.+Crews&title=Targeted+Protein+Degradation+by+PROTACs&doi=10.1016%2Fj.pharmthera.2017.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by PROTACs</span></div><div class="casAuthors">Neklesa, Taavi K.; Winkler, James D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-144</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using the PROTAC technol. is emerging as a novel therapeutic method to address diseases driven by the aberrant expression of a disease-causing protein.  PROTAC mols. are bifunctional small mols. that simultaneously bind a target protein and an E3-ubiquitin ligase, thus causing ubiquitination and degrdn. of the target protein by the proteasome.  Like small mols., PROTAC mols. possess good tissue distribution and the ability to target intracellular proteins.  Herein, we highlight the advantages of protein degrdn. using PROTACs, and provide specific examples where degrdn. offers therapeutic benefit over classical enzyme inhibition.  Foremost, PROTACs can degrade proteins regardless of their function.  This includes the currently "undruggable" proteome, which comprises approx. 85% of all human proteins.  Other beneficial aspects of protein degrdn. include the ability to target overexpressed and mutated proteins, as well as the potential to demonstrate prolonged pharmacodynamics effect beyond drug exposure.  Lastly, due to their catalytic nature and the pre-requisite ubiquitination step, an exquisitely potent mols. with a high degree of degrdn. selectivity can be designed.  Impressive preclin. in vitro and in vivo PROTAC data have been published, and these data have propelled the development of clin. viable PROTACs.  With the mol. wt. falling in the 700-1000 Da range, the delivery and bioavailability of PROTACs remain the largest hurdles on the way to the clinic.  Solving these issues and demonstrating proof of concept clin. data will be the focus of many labs over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4RSLWYqqBMrVg90H21EOLACvtfcHk0ljs0xgiC7-kDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVemsbg%253D&md5=08a74402a274a13c8c484cd68167a5a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520PROTACs%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D174%26spage%3D138%26epage%3D144%26doi%3D10.1016%2Fj.pharmthera.2017.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced Protein Degradation: An Emerging Drug Discovery Paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+Protein+Degradation%3A+An+Emerging+Drug+Discovery+Paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljs0xgiC7-kDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520Protein%2520Degradation%253A%2520An%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+Protein+Degradation%3A+from+Chemical+Biology+to+Drug+Discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0ljYp9sd30umPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%253A%2520from%2520Chemical%2520Biology%2520to%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: New Approaches to Protein Degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+New+Approaches+to+Protein+Degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljYp9sd30umPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520New%2520Approaches%2520to%2520Protein%2520Degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span> <span> </span><span class="NLM_article-title">ARV-330: Androgen Receptor PROTAC Degrader for Prostate Cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">267</span>, <span class="refDoi"> DOI: 10.1200/jco.2016.34.2_suppl.267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1200%2Fjco.2016.34.2_suppl.267" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=267&author=T.+K.+Neklesaauthor=M.+Jinauthor=A.+P.+Crewauthor=A.+K.+Rossiauthor=R.+R.+Willardauthor=H.+Dongauthor=K.+Siuauthor=J.+Wangauthor=D.+A.+Gordonauthor=X.+Chenauthor=C.+Ferraroauthor=C.+M.+Crewsauthor=K.+Colemanauthor=J.+D.+Winkler&title=ARV-330%3A+Androgen+Receptor+PROTAC+Degrader+for+Prostate+Cancer&doi=10.1200%2Fjco.2016.34.2_suppl.267"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1200%2Fjco.2016.34.2_suppl.267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252Fjco.2016.34.2_suppl.267%26sid%3Dliteratum%253Aachs%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRossi%26aufirst%3DA.%2BK.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26atitle%3DARV-330%253A%2520Androgen%2520Receptor%2520PROTAC%2520Degrader%2520for%2520Prostate%2520Cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D34%26spage%3D267%26doi%3D10.1200%2Fjco.2016.34.2_suppl.267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0ljYp9sd30umPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+Small+Molecule+Induced+Degradation+of+the+BET+Bromodomain+Protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhPiz4uMA-A6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520Small%2520Molecule%2520Induced%2520Degradation%2520of%2520the%2520BET%2520Bromodomain%2520Protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+Von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0lhPiz4uMA-A6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span>. <i>BCR-ABL</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+Design+for+the+Degradation+of+Oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520Design%2520for%2520the%2520Degradation%2520of%2520Oncogenic%2520BCR-ABL%26jtitle%3DBCR-ABL%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.-Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+First-in-Class%2C+Highly+Potent%2C+and+Efficacious+Proteolysis+Targeting+Chimera+Murine+Double+Minute+2+Degrader+Capable+of+Achieving+Complete+and+Durable+Tumor+Regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0ljx78pKlwDqoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520First-in-Class%252C%2520Highly%2520Potent%252C%2520and%2520Efficacious%2520Proteolysis%2520Targeting%2520Chimera%2520Murine%2520Double%2520Minute%25202%2520Degrader%2520Capable%2520of%2520Achieving%2520Complete%2520and%2520Durable%2520Tumor%2520Regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo Protein Knockdown by Small-Molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+Protein+Knockdown+by+Small-Molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lixnKRAQHrRlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520Protein%2520Knockdown%2520by%2520Small-Molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2476</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1158%2F0008-5472.can-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476&author=L.+Baiauthor=B.+Zhouauthor=C.-Y.+Yangauthor=J.+Jiauthor=D.+McEachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+Huauthor=F.+Xuauthor=Y.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+Douauthor=B.+Wenauthor=D.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Liauthor=M.+J.+Ellisauthor=S.+Wang&title=Targeted+Degradation+of+BET+Proteins+in+Triple-Negative+Breast+Cancer&doi=10.1158%2F0008-5472.can-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0lixnKRAQHrRlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeted%2520Degradation%2520of%2520BET%2520Proteins%2520in%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2476%26doi%3D10.1158%2F0008-5472.can-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ransick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Development of Protacs to Target Cancer-promoting Proteins for Ubiquitination and Degradation</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.1074/mcp.t300009-mcp200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1074%2Fmcp.t300009-mcp200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=14525958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVShu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=1350-1358&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=R.+Vermaauthor=A.+Ransickauthor=B.+Steinauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Development+of+Protacs+to+Target+Cancer-promoting+Proteins+for+Ubiquitination+and+Degradation&doi=10.1074%2Fmcp.t300009-mcp200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Protacs to target cancer-promoting proteins for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Verma, Rati; Ransick, Andy; Stein, Bernd; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1350-1358</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The proteome contains hundreds of proteins that in theory could be excellent therapeutic targets for the treatment of human diseases.  However, many of these proteins are from functional classes that have never been validated as viable candidates for the development of small mol. inhibitors.  Thus, to exploit fully the potential of the Human Genome Project to advance human medicine, there is a need to develop generic methods of inhibiting protein activity that do not rely on the target protein's function.  The authors previously demonstrated that a normally stable protein, methionine aminopeptidase-2 or MetAP-2, could be artificially targeted to an Skp1-Cullin-F-box (SCF) ubiquitin ligase complex for ubiquitination and degrdn. through a chimeric bridging mol. or Protac (proteolysis targeting chimeric mol.).  This Protac consisted of an SCFβ-TRCP-binding phosphopeptide derived from IκBα linked to ovalicin, which covalently binds MetAP-2.  In this study, the authors employed this approach to target two different proteins, the estrogen (ER) and androgen (AR) receptors, which have been implicated in the progression of breast and prostate cancer, resp.  The authors show here that an estradiol-based Protac can enforce the ubiquitination and degrdn. of the α isoform of ER in vitro, and a dihydroxytestosterone-based Protac introduced into cells promotes the rapid disappearance of AR in a proteasome-dependent manner.  Future improvements to this technol. may yield a general approach to treat a no. of human diseases, including cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3SZSMptsCorVg90H21EOLACvtfcHk0lixnKRAQHrRlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVShu7jE&md5=5a9f4cf5d5cc1d990bf59ac824902111</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fmcp.t300009-mcp200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.t300009-mcp200%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DVerma%26aufirst%3DR.%26aulast%3DRansick%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DB.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DDevelopment%2520of%2520Protacs%2520to%2520Target%2520Cancer-promoting%2520Proteins%2520for%2520Ubiquitination%2520and%2520Degradation%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2003%26volume%3D2%26spage%3D1350%26epage%3D1358%26doi%3D10.1074%2Fmcp.t300009-mcp200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide Conjugation as A Strategy for in vivo Target Protein Degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+Conjugation+as+A+Strategy+for+in+vivo+Target+Protein+Degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lgtueR1FcMrPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520Conjugation%2520as%2520A%2520Strategy%2520for%2520in%2520vivo%2520Target%2520Protein%2520Degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span> <span> </span><span class="NLM_article-title">Identification of A Primary Target of Thalidomide Teratogenicity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1345</span>– <span class="NLM_lpage">1350</span>, <span class="refDoi"> DOI: 10.1126/science.1177319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.1177319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=20223979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFensrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1345-1350&author=T.+Itoauthor=H.+Andoauthor=T.+Suzukiauthor=T.+Oguraauthor=K.+Hottaauthor=Y.+Imamuraauthor=Y.+Yamaguchiauthor=H.+Handa&title=Identification+of+A+Primary+Target+of+Thalidomide+Teratogenicity&doi=10.1126%2Fscience.1177319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Primary Target of Thalidomide Teratogenicity</span></div><div class="casAuthors">Ito, Takumi; Ando, Hideki; Suzuki, Takayuki; Ogura, Toshihiko; Hotta, Kentaro; Imamura, Yoshimasa; Yamaguchi, Yuki; Handa, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5971</span>),
    <span class="NLM_cas:pages">1345-1350</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Half a century ago, thalidomide was widely prescribed to pregnant women as a sedative but was found to be teratogenic, causing multiple birth defects.  Today, thalidomide is still used in the treatment of leprosy and multiple myeloma, although how it causes limb malformation and other developmental defects is unknown.  Here, we identified cereblon (CRBN) as a thalidomide-binding protein.  CRBN forms an E3 ubiquitin ligase complex with damaged DNA binding protein 1 (DDB1) and Cul4A that is important for limb outgrowth and expression of the fibroblast growth factor Fgf8 in zebrafish and chicks.  Thalidomide initiates its teratogenic effects by binding to CRBN and inhibiting the assocd. ubiquitin ligase activity.  This study reveals a basis for thalidomide teratogenicity and may contribute to the development of new thalidomide derivs. without teratogenic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPC_7RXLkfabVg90H21EOLACvtfcHk0lgtueR1FcMrPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFensrs%253D&md5=4565a67837e0ec34ede2c93c1036f312</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.1177319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1177319%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DT.%26aulast%3DAndo%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DOgura%26aufirst%3DT.%26aulast%3DHotta%26aufirst%3DK.%26aulast%3DImamura%26aufirst%3DY.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DHanda%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520A%2520Primary%2520Target%2520of%2520Thalidomide%2520Teratogenicity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1345%26epage%3D1350%26doi%3D10.1126%2Fscience.1177319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karasawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbasian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoudi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rychak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Cereblon is A Direct Protein Target for Immunomodulatory and Antiproliferative Activities of Lenalidomide and Pomalidomide</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2326</span>– <span class="NLM_lpage">2335</span>, <span class="refDoi"> DOI: 10.1038/leu.2012.119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fleu.2012.119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=22552008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Giur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=2326-2335&author=A.+Lopez-Gironaauthor=D.+Mendyauthor=T.+Itoauthor=K.+Millerauthor=A.+K.+Gandhiauthor=J.+Kangauthor=S.+Karasawaauthor=G.+Carmelauthor=P.+Jacksonauthor=M.+Abbasianauthor=A.+Mahmoudiauthor=B.+Cathersauthor=E.+Rychakauthor=S.+Gaidarovaauthor=R.+Chenauthor=P.+H.+Schaferauthor=H.+Handaauthor=T.+O.+Danielauthor=J.+F.+Evansauthor=R.+Chopra&title=Cereblon+is+A+Direct+Protein+Target+for+Immunomodulatory+and+Antiproliferative+Activities+of+Lenalidomide+and+Pomalidomide&doi=10.1038%2Fleu.2012.119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon is a direct protein target for immunomodulatory and antiproliferative activities of lenalidomide and pomalidomide</span></div><div class="casAuthors">Lopez-Girona, A.; Mendy, D.; Ito, T.; Miller, K.; Gandhi, A. K.; Kang, J.; Karasawa, S.; Carmel, G.; Jackson, P.; Abbasian, M.; Mahmoudi, A.; Cathers, B.; Rychak, E.; Gaidarova, S.; Chen, R.; Schafer, P. H.; Handa, H.; Daniel, T. O.; Evans, J. F.; Chopra, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2326-2335</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and the immunomodulatory drug, lenalidomide, are therapeutically active in hematol. malignancies.  The ubiquitously expressed E3 ligase protein cereblon (CRBN) has been identified as the primary teratogenic target of thalidomide.  Our studies demonstrate that thalidomide, lenalidomide and another immunomodulatory drug, pomalidomide, bound endogenous CRBN and recombinant CRBN-DNA damage binding protein-1 (DDB1) complexes.  CRBN mediated antiproliferative activities of lenalidomide and pomalidomide in myeloma cells, as well as lenalidomide- and pomalidomide-induced cytokine prodn. in T cells.  Lenalidomide and pomalidomide inhibited autoubiquitination of CRBN in HEK293T cells expressing thalidomide-binding competent wild-type CRBN, but not thalidomide-binding defective CRBNYW/AA.  Overexpression of CRBN wild-type protein, but not CRBNYW/AA mutant protein, in KMS12 myeloma cells, amplified pomalidomide-mediated redns. in c-myc and IRF4 expression and increases in p21WAF-1 expression.  Long-term selection for lenalidomide resistance in H929 myeloma cell lines was accompanied by a redn. in CRBN, while in DF15R myeloma cells resistant to both pomalidomide and lenalidomide, CRBN protein was undetectable.  Our biophys., biochem. and gene silencing studies show that CRBN is a proximate, therapeutically important mol. target of lenalidomide and pomalidomide.  Leukemia (2012) 26, 2326-2335; doi:10.1038/leu.2012.119; published online 15 June 2012.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp85RvB33HGjbVg90H21EOLACvtfcHk0lgtueR1FcMrPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Giur%252FI&md5=81f83b5f6c4c043a35a049df3b4d63ae</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fleu.2012.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2012.119%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DMendy%26aufirst%3DD.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKarasawa%26aufirst%3DS.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DJackson%26aufirst%3DP.%26aulast%3DAbbasian%26aufirst%3DM.%26aulast%3DMahmoudi%26aufirst%3DA.%26aulast%3DCathers%26aufirst%3DB.%26aulast%3DRychak%26aufirst%3DE.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DEvans%26aufirst%3DJ.%2BF.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DCereblon%2520is%2520A%2520Direct%2520Protein%2520Target%2520for%2520Immunomodulatory%2520and%2520Antiproliferative%2520Activities%2520of%2520Lenalidomide%2520and%2520Pomalidomide%26jtitle%3DLeukemia%26date%3D2012%26volume%3D26%26spage%3D2326%26epage%3D2335%26doi%3D10.1038%2Fleu.2012.119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Lenalidomide-Induced CK1α Degradation by the CRL4<sup>CRBN</sup> Ubiquitin Ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>532</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature16979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26909574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+Basis+of+Lenalidomide-Induced+CK1%CE%B1+Degradation+by+the+CRL4CRBN+Ubiquitin+Ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Petzold, Georg; Fischer, Eric S.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">532</span>
        (<span class="NLM_cas:issue">7597</span>),
    <span class="NLM_cas:pages">127-130</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs., lenalidomide and pomalidomide, are immune modulatory drugs (IMiDs) used in the treatment of hematol. malignancies.  IMiDs bind CRBN, the substrate receptor of the CUL4-RBX1-DDB1-CRBN (also known as CRL4CRBN) E3 ubiquitin ligase, and inhibit ubiquitination of endogenous CRL4CRBN substrates.  Unexpectedly, IMiDs also repurpose the ligase to target new proteins for degrdn.  Lenalidomide induces degrdn. of the lymphoid transcription factors Ikaros and Aiolos (also known as IKZF1 and IKZF3), and casein kinase 1α (CK1α), which contributes to its clin. efficacy in the treatment of multiple myeloma and 5q-deletion assocd. myelodysplastic syndrome (del(5q) MDS), resp.  How lenalidomide alters the specificity of the ligase to degrade these proteins remains elusive.  Here we present the 2.45 Å crystal structure of DDB1-CRBN bound to lenalidomide and CK1α.  CRBN and lenalidomide jointly provide the binding interface for a CK1α β-hairpin-loop located in the kinase N-lobe.  We show that CK1α binding to CRL4CRBN is strictly dependent on the presence of an IMiD.  Binding of IKZF1 to CRBN similarly requires the compd. and both, IKZF1 and CK1α, use a related binding mode.  Our study provides a mechanistic explanation for the selective efficacy of lenalidomide in del(5q) MDS therapy.  We anticipate that high-affinity protein-protein interactions induced by small mols. will provide opportunities for drug development, particularly for targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4FxWriI1ObVg90H21EOLACvtfcHk0lgP_d01MESU2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D&md5=320fc6ff811da335fa8e02feab2d702d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520Basis%2520of%2520Lenalidomide-Induced%2520CK1%25CE%25B1%2520Degradation%2520by%2520the%2520CRL4CRBN%2520Ubiquitin%2520Ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naniong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The Myeloma Drug Lenalidomide Promotes the Cereblon-Dependent Destruction of Ikaros Proteins</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1126/science.1244917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.1244917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=24292623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtV2ktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=305-309&author=G.+Luauthor=R.+E.+Middletonauthor=H.+Sunauthor=M.+Naniongauthor=C.+J.+Ottauthor=C.+S.+Mitsiadesauthor=K.-K.+Wongauthor=J.+E.+Bradnerauthor=W.+G.+Kaelin&title=The+Myeloma+Drug+Lenalidomide+Promotes+the+Cereblon-Dependent+Destruction+of+Ikaros+Proteins&doi=10.1126%2Fscience.1244917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Myeloma Drug Lenalidomide Promotes the Cereblon-Dependent Destruction of Ikaros Proteins</span></div><div class="casAuthors">Lu, Gang; Middleton, Richard E.; Sun, Huahang; Naniong, MarkVic; Ott, Christopher J.; Mitsiades, Constantine S.; Wong, Kwok-Kin; Bradner, James E.; Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6168</span>),
    <span class="NLM_cas:pages">305-309</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Thalidomide-like drugs such as lenalidomide are clin. important treatments for multiple myeloma and show promise for other B cell malignancies.  The biochem. mechanisms underlying their antitumor activity are unknown.  Thalidomide was recently shown to bind to, and inhibit, the cereblon ubiquitin ligase.  Cereblon loss in zebrafish causes fin defects reminiscent of the limb defects seen in children exposed to thalidomide in utero.  Here we show that lenalidomide-bound cereblon acquires the ability to target for proteasomal degrdn. two specific B cell transcription factors, Ikaros family zinc finger proteins 1 and 3 (IKZF1 and IKZF3).  Anal. of myeloma cell lines revealed that loss of IKZF1 and IKZF3 is both necessary and sufficient for lenalidomide's therapeutic effect, suggesting that the antitumor and teratogenic activities of thalidomide-like drugs are dissociable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0froz4Kipu7Vg90H21EOLACvtfcHk0lgP_d01MESU2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtV2ktg%253D%253D&md5=a9df5bbde0f676b92b1bec48fdbbb502</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244917%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DG.%26aulast%3DMiddleton%26aufirst%3DR.%2BE.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNaniong%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DThe%2520Myeloma%2520Drug%2520Lenalidomide%2520Promotes%2520the%2520Cereblon-Dependent%2520Destruction%2520of%2520Ikaros%2520Proteins%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D305%26epage%3D309%26doi%3D10.1126%2Fscience.1244917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kronke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grauman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConkey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarlo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">Lenalidomide Causes Selective Degradation of IKZF1 and IKZF3 in Multiple Myeloma Cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1126/science.1244851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.1244851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=24292625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=301-305&author=J.+Kronkeauthor=N.+D.+Udeshiauthor=A.+Narlaauthor=P.+Graumanauthor=S.+N.+Hurstauthor=M.+McConkeyauthor=T.+Svinkinaauthor=D.+Hecklauthor=E.+Comerauthor=X.+Liauthor=C.+Ciarloauthor=E.+Hartmanauthor=N.+Munshiauthor=M.+Schenoneauthor=S.+L.+Schreiberauthor=S.+A.+Carrauthor=B.+L.+Ebert&title=Lenalidomide+Causes+Selective+Degradation+of+IKZF1+and+IKZF3+in+Multiple+Myeloma+Cells&doi=10.1126%2Fscience.1244851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells</span></div><div class="casAuthors">Kronke Jan; Udeshi Namrata D; Narla Anupama; Grauman Peter; Hurst Slater N; McConkey Marie; Svinkina Tanya; Heckl Dirk; Comer Eamon; Li Xiaoyu; Ciarlo Christie; Hartman Emily; Munshi Nikhil; Schenone Monica; Schreiber Stuart L; Carr Steven A; Ebert Benjamin L</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6168</span>),
    <span class="NLM_cas:pages">301-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lenalidomide is a drug with clinical efficacy in multiple myeloma and other B cell neoplasms, but its mechanism of action is unknown.  Using quantitative proteomics, we found that lenalidomide causes selective ubiquitination and degradation of two lymphoid transcription factors, IKZF1 and IKZF3, by the CRBN-CRL4 ubiquitin ligase.  IKZF1 and IKZF3 are essential transcription factors in multiple myeloma.  A single amino acid substitution of IKZF3 conferred resistance to lenalidomide-induced degradation and rescued lenalidomide-induced inhibition of cell growth.  Similarly, we found that lenalidomide-induced interleukin-2 production in T cells is due to depletion of IKZF1 and IKZF3.  These findings reveal a previously unknown mechanism of action for a therapeutic agent: alteration of the activity of an E3 ubiquitin ligase, leading to selective degradation of specific targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQiiGR0Yx7nuYFEOhA_Im3cfW6udTcc2eZQO8ZdOf0lO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D&md5=8cae3e554399308493aa38f0d05ed610</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244851%26sid%3Dliteratum%253Aachs%26aulast%3DKronke%26aufirst%3DJ.%26aulast%3DUdeshi%26aufirst%3DN.%2BD.%26aulast%3DNarla%26aufirst%3DA.%26aulast%3DGrauman%26aufirst%3DP.%26aulast%3DHurst%26aufirst%3DS.%2BN.%26aulast%3DMcConkey%26aufirst%3DM.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DHeckl%26aufirst%3DD.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCiarlo%26aufirst%3DC.%26aulast%3DHartman%26aufirst%3DE.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DLenalidomide%2520Causes%2520Selective%2520Degradation%2520of%2520IKZF1%2520and%2520IKZF3%2520in%2520Multiple%2520Myeloma%2520Cells%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D301%26epage%3D305%26doi%3D10.1126%2Fscience.1244851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krönke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenbach, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacBeth, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConkey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Järås, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>523</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1038/nature14610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature14610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26131937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1entr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=523&publication_year=2015&pages=183-188&author=J.+Kr%C3%B6nkeauthor=E.+C.+Finkauthor=P.+W.+Hollenbachauthor=K.+J.+MacBethauthor=S.+N.+Hurstauthor=N.+D.+Udeshiauthor=P.+P.+Chamberlainauthor=D.+R.+Maniauthor=H.+W.+Manauthor=A.+K.+Gandhiauthor=T.+Svinkinaauthor=R.+K.+Schneiderauthor=M.+McConkeyauthor=M.+J%C3%A4r%C3%A5sauthor=E.+Griffithsauthor=M.+Wetzlerauthor=L.+Bullingerauthor=B.+E.+Cathersauthor=S.+A.+Carrauthor=R.+Chopraauthor=B.+L.+Ebert&title=Lenalidomide+induces+ubiquitination+and+degradation+of+CK1%CE%B1+in+del%285q%29+MDS&doi=10.1038%2Fnature14610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS</span></div><div class="casAuthors">Kronke, Jan; Fink, Emma C.; Hollenbach, Paul W.; MacBeth, Kyle J.; Hurst, Slater N.; Udeshi, Namrata D.; Chamberlain, Philip P.; Mani, D. R.; Man, Hon Wah; Gandhi, Anita K.; Svinkina, Tanya; Schneider, Rebekka K.; McConkey, Marie; Jaras, Marcus; Griffiths, Elizabeth; Wetzler, Meir; Bullinger, Lars; Cathers, Brian E.; Carr, Steven A.; Chopra, Rajesh; Ebert, Benjamin L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">523</span>
        (<span class="NLM_cas:issue">7559</span>),
    <span class="NLM_cas:pages">183-188</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lenalidomide is a highly effective treatment for myelodysplastic syndrome (MDS) with deletion of chromosome 5q (del(5q)).  Lenalidomide induces the ubiquitination of casein kinase 1A1 (CK1α) by the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN), resulting in CK1α degrdn.  CK1α is encoded by a gene within the common deleted region for del(5q) MDS and haplo-insufficient expression sensitizes cells to lenalidomide therapy, providing a mechanistic basis for the therapeutic window of lenalidomide in del(5q) MDS.  The authors found that mouse cells are resistant to lenalidomide but that changing a single amino acid in mouse Crbn to the corresponding human residue enables lenalidomide-dependent degrdn. of CK1α.  The authors further demonstrate that minor side chain modifications in thalidomide and a novel analog, CC-122, can modulate the spectrum of substrates targeted by CRL4CRBN.  These findings have implications for the clin. activity of lenalidomide and related compds., and demonstrate the therapeutic potential of novel modulators of E3 ubiquitin ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiLiHx3FKesbVg90H21EOLACvtfcHk0lgFj6zDsudYvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1entr%252FN&md5=9b20a91c13e888e6569223ffa25af2cd</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature14610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14610%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25B6nke%26aufirst%3DJ.%26aulast%3DFink%26aufirst%3DE.%2BC.%26aulast%3DHollenbach%26aufirst%3DP.%2BW.%26aulast%3DMacBeth%26aufirst%3DK.%2BJ.%26aulast%3DHurst%26aufirst%3DS.%2BN.%26aulast%3DUdeshi%26aufirst%3DN.%2BD.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DMani%26aufirst%3DD.%2BR.%26aulast%3DMan%26aufirst%3DH.%2BW.%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DSchneider%26aufirst%3DR.%2BK.%26aulast%3DMcConkey%26aufirst%3DM.%26aulast%3DJ%25C3%25A4r%25C3%25A5s%26aufirst%3DM.%26aulast%3DGriffiths%26aufirst%3DE.%26aulast%3DWetzler%26aufirst%3DM.%26aulast%3DBullinger%26aufirst%3DL.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DChopra%26aufirst%3DR.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DLenalidomide%2520induces%2520ubiquitination%2520and%2520degradation%2520of%2520CK1%25CE%25B1%2520in%2520del%25285q%2529%2520MDS%26jtitle%3DNature%26date%3D2015%26volume%3D523%26spage%3D183%26epage%3D188%26doi%3D10.1038%2Fnature14610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">SPLINTS: Small-molecule Protein Ligand Interface Stabilizers</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2016.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.sbi.2016.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26829757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=115-122&author=E.+S.+Fischerauthor=E.+Parkauthor=M.+J.+Eckauthor=N.+H.+Thom%C3%A4&title=SPLINTS%3A+Small-molecule+Protein+Ligand+Interface+Stabilizers&doi=10.1016%2Fj.sbi.2016.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">SPLINTS: small-molecule protein ligand interface stabilizers</span></div><div class="casAuthors">Fischer, Eric S.; Park, Eunyoung; Eck, Michael J.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">115-122</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Regulatory protein-protein interactions are ubiquitous in biol., and small mol. protein-protein interaction inhibitors are an important focus in drug discovery.  Remarkably little attention has been given to the opposite strategy - stabilization of protein-protein interactions, despite the fact that several well-known therapeutics act through this mechanism.  From a structural perspective, we consider representative examples of small mols. that induce or stabilize the assocn. of protein domains to inhibit, or alter, signaling for nuclear hormone, GTPase, kinase, phosphatase, and ubiquitin ligase pathways.  These SPLINTS (small-mol. protein ligand interface stabilizers) drive interactions that are in some cases physiol. relevant, and in others entirely adventitious.  The diverse structural mechanisms employed suggest approaches for a broader and systematic search for such compds. in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBxkNU6Walm7Vg90H21EOLACvtfcHk0lgFj6zDsudYvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKqsr0%253D&md5=014c6488fbbc458b395d38e3220a443f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2016.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2016.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DSPLINTS%253A%2520Small-molecule%2520Protein%2520Ligand%2520Interface%2520Stabilizers%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2016%26volume%3D37%26spage%3D115%26epage%3D122%26doi%3D10.1016%2Fj.sbi.2016.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Ternary Complexes: A New Dimension in the Structure-guided Design of Chemical Degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/ebc20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1042%2Febc20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+Recognition+of+Ternary+Complexes%3A+A+New+Dimension+in+the+Structure-guided+Design+of+Chemical+Degraders&doi=10.1042%2Febc20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2ea8nmtOf_Nomrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1042%2Febc20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Febc20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520Recognition%2520of%2520Ternary%2520Complexes%253A%2520A%2520New%2520Dimension%2520in%2520the%2520Structure-guided%2520Design%2520of%2520Chemical%2520Degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D505%26epage%3D516%26doi%3D10.1042%2Febc20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goralski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaskill, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ting, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nijhawan, D.</span></span> <span> </span><span class="NLM_article-title">Anticancer Sulfonamides Target Splicing by Inducing RBM39 Degradation via Recruitment to DCAF15</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>356</i></span>, <span class="NLM_elocation-id">eaal3755</span> <span class="refDoi"> DOI: 10.1126/science.aal3755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1126%2Fscience.aal3755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28302793" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2017&author=T.+Hanauthor=M.+Goralskiauthor=N.+Gaskillauthor=E.+Capotaauthor=J.+Kimauthor=T.+C.+Tingauthor=Y.+Xieauthor=N.+S.+Williamsauthor=D.+Nijhawan&title=Anticancer+Sulfonamides+Target+Splicing+by+Inducing+RBM39+Degradation+via+Recruitment+to+DCAF15&doi=10.1126%2Fscience.aal3755"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal3755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal3755%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DT.%26aulast%3DGoralski%26aufirst%3DM.%26aulast%3DGaskill%26aufirst%3DN.%26aulast%3DCapota%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DTing%26aufirst%3DT.%2BC.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DNijhawan%26aufirst%3DD.%26atitle%3DAnticancer%2520Sulfonamides%2520Target%2520Splicing%2520by%2520Inducing%2520RBM39%2520Degradation%2520via%2520Recruitment%2520to%2520DCAF15%26jtitle%3DScience%26date%3D2017%26volume%3D356%26doi%3D10.1126%2Fscience.aal3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugi, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuhashi, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotake, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uesugi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mabuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owa, T.</span></span> <span> </span><span class="NLM_article-title">Selective degradation of splicing factor CAPERα by anticancer sulfonamides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">675</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnchembio.2363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=28437394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVCltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=675-680&author=T.+Ueharaauthor=Y.+Minoshimaauthor=K.+Saganeauthor=N.+H.+Sugiauthor=K.+O.+Mitsuhashiauthor=N.+Yamamotoauthor=H.+Kamiyamaauthor=K.+Takahashiauthor=Y.+Kotakeauthor=M.+Uesugiauthor=A.+Yokoiauthor=A.+Inoueauthor=T.+Yoshidaauthor=M.+Mabuchiauthor=A.+Tanakaauthor=T.+Owa&title=Selective+degradation+of+splicing+factor+CAPER%CE%B1+by+anticancer+sulfonamides&doi=10.1038%2Fnchembio.2363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selective degradation of splicing factor CAPERα by anticancer sulfonamides</span></div><div class="casAuthors">Uehara, Taisuke; Minoshima, Yukinori; Sagane, Koji; Sugi, Naoko Hata; Mitsuhashi, Kaoru Ogawa; Yamamoto, Noboru; Kamiyama, Hiroshi; Takahashi, Kentaro; Kotake, Yoshihiko; Uesugi, Mai; Yokoi, Akira; Inoue, Atsushi; Yoshida, Taku; Mabuchi, Miyuki; Tanaka, Akito; Owa, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">675-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Target-protein degrdn. is an emerging field in drug discovery and development.  In particular, the substrate-receptor proteins of the cullin-ubiquitin ligase system play a key role in selective protein degrdn., which is an essential component of the anti-myeloma activity of immunomodulatory drugs (IMiDs), such as lenalidomide.  Here, we demonstrate that a series of anticancer sulfonamides NSC 719239 (E7820), indisulam, and NSC 339004 (chloroquinoxaline sulfonamide, CQS) induce proteasomal degrdn. of the U2AF-related splicing factor coactivator of activating protein-1 and estrogen receptors (CAPERα) via CRL4DCAF15 mediated ubiquitination in human cancer cell lines.  Both CRISPR-Cas9-based knockout of DCAF15 and a single amino acid substitution of CAPERα conferred resistance against sulfonamide-induced CAPERα degrdn. and cell-growth inhibition.  Thus, these sulfonamides represent selective chem. probes for disrupting CAPERα function and designate DCAFs as promising drug targets for promoting selective protein degrdn. in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4hxIDPBvmVLVg90H21EOLACvtfcHk0lgJCrvuPfb3FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVCltbY%253D&md5=59078969183a22084ce31973fd026095</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2363%26sid%3Dliteratum%253Aachs%26aulast%3DUehara%26aufirst%3DT.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DSagane%26aufirst%3DK.%26aulast%3DSugi%26aufirst%3DN.%2BH.%26aulast%3DMitsuhashi%26aufirst%3DK.%2BO.%26aulast%3DYamamoto%26aufirst%3DN.%26aulast%3DKamiyama%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DUesugi%26aufirst%3DM.%26aulast%3DYokoi%26aufirst%3DA.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DYoshida%26aufirst%3DT.%26aulast%3DMabuchi%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DOwa%26aufirst%3DT.%26atitle%3DSelective%2520degradation%2520of%2520splicing%2520factor%2520CAPER%25CE%25B1%2520by%2520anticancer%2520sulfonamides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D675%26epage%3D680%26doi%3D10.1038%2Fnchembio.2363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+Termination+Factor+GSPT1+Is+a+Phenotypically+Relevant+Off-Target+of+Heterobifunctional+Phthalimide+Degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0lgJCrvuPfb3FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520Termination%2520Factor%2520GSPT1%2520Is%2520a%2520Phenotypically%2520Relevant%2520Off-Target%2520of%2520Heterobifunctional%2520Phthalimide%2520Degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochsenbein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carry, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Diastereomeric Spirooxindoles as Highly Potent and Efficacious MDM2 Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">7223</span>– <span class="NLM_lpage">7234</span>, <span class="refDoi"> DOI: 10.1021/ja3125417</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3125417" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVWrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=7223-7234&author=Y.+Zhaoauthor=L.+Liuauthor=W.+Sunauthor=J.+Luauthor=D.+McEachernauthor=X.+Liauthor=S.+Yuauthor=D.+Bernardauthor=P.+Ochsenbeinauthor=V.+Fereyauthor=J.-C.+Carryauthor=J.+R.+Deschampsauthor=D.+Sunauthor=S.+Wang&title=Diastereomeric+Spirooxindoles+as+Highly+Potent+and+Efficacious+MDM2+Inhibitors&doi=10.1021%2Fja3125417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Diastereomeric Spirooxindoles as Highly Potent and Efficacious MDM2 Inhibitors</span></div><div class="casAuthors">Zhao, Yujun; Liu, Liu; Sun, Wei; Lu, Jianfeng; McEachern, Donna; Li, Xiaoqin; Yu, Shanghai; Bernard, Denzil; Ochsenbein, Philippe; Ferey, Vincent; Carry, Jean-Christophe; Deschamps, Jeffrey R.; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7223-7234</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that block the MDM2-p53 protein-protein interaction (MDM2 inhibitors) are being intensely pursued as a new therapeutic strategy for cancer treatment.  We previously published a series of spirooxindole-contg. compds. as a new class of MDM2 small-mol. inhibitors.  We report herein a reversible ring-opening-cyclization reaction for some of these spirooxindoles, which affords four diastereomers from a single compd.  Our biochem. binding data showed that the stereochem. in this class of compds. has a major effect on their binding affinities to MDM2, with >100-fold difference between the most potent and the least potent stereoisomers.  Our study has led to the identification of a set of highly potent MDM2 inhibitors with a stereochem. that is different from that of our previously reported compds.  The most potent compd. (MI-888) binds to MDM2 with a Ki value of 0.44 nM and achieves complete and long-lasting tumor regression in an animal model of human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOEWm0CbFl2rVg90H21EOLACvtfcHk0lhorb6-mkA8eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVWrtr0%253D&md5=5c05b0adc0d14506602d0884b64411b0</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja3125417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3125417%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DOchsenbein%26aufirst%3DP.%26aulast%3DFerey%26aufirst%3DV.%26aulast%3DCarry%26aufirst%3DJ.-C.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiastereomeric%2520Spirooxindoles%2520as%2520Highly%2520Potent%2520and%2520Efficacious%2520MDM2%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D7223%26epage%3D7234%26doi%3D10.1021%2Fja3125417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman-Luca, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debussche, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Elucidation of Acquired Resistance to Bcl-2 and MDM2 Inhibitors in Acute Leukemia in Vitro and in Vivo</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2558</span>– <span class="NLM_lpage">2568</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-14-2506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1158%2F1078-0432.ccr-14-2506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25754349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1aku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2558-2568&author=C.+G.+Hoffman-Lucaauthor=D.+Ziazadehauthor=D.+McEachernauthor=Y.+Zhaoauthor=W.+Sunauthor=L.+Debusscheauthor=S.+Wang&title=Elucidation+of+Acquired+Resistance+to+Bcl-2+and+MDM2+Inhibitors+in+Acute+Leukemia+in+Vitro+and+in+Vivo&doi=10.1158%2F1078-0432.ccr-14-2506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation of Acquired Resistance to Bcl-2 and MDM2 Inhibitors in Acute Leukemia In Vitro and In Vivo</span></div><div class="casAuthors">Hoffman-Luca, C. Gianna; Ziazadeh, Daniel; McEachern, Donna; Zhao, Yujun; Sun, Wei; Debussche, Laurent; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2558-2568</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Two clin.-stage anticancer drugs, the Bcl-2 inhibitor ABT-263, and the MDM2 inhibitor SAR405838 achieve complete tumor regression in animal models of leukemia but also induce acquired resistance.  Elucidation of acquired resistance mechanisms and development of strategies to overcome the resistance are crit. for their successful clin. development.  Exptl. Design: We employed RS4;11 and MV4;11 cell lines, two acute leukemia models, to investigate acquired resistance mechanisms for both drugs in vitro and in vivo and evaluated several treatment regimens in xenograft mouse models to improve long-term, complete tumor regression.  Results: Resistance to either SAR405838 or ABT-263 (or its analog ABT-737) develops in acute leukemia models in vitro and in vivo.  RS4;11 and MV4;11 tumors treated with SAR405838 acquire resistance to the drug by mutation of the TP53 gene or compromise of p53 protein function.  RS4;11 tumors treated with either ABT-263 or ABT-737 acquire resistance primarily through downregulation of BAX but not BAK.  When acute leukemia cells become highly resistant to the MDM2 inhibitor, they retain their sensitivity to the Bcl-2 inhibitors, or vice versa.  Certain sequential or combination treatment of SAR405838 and ABT-263 can achieve longer-term tumor regression than treatment with either agent alone.  Conclusions: Our study provides new insights into the mechanisms of acquired resistance of Bcl-2 and MDM2 inhibitors in acute leukemia models and suggests that certain sequential or combination treatment of these two distinct classes of apoptosis-inducing agents should be tested as new treatment strategies for acute leukemia in the clinic.  Clin Cancer Res; 21(11); 2558-68. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8iM179w0AXrVg90H21EOLACvtfcHk0lhorb6-mkA8eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1aku7o%253D&md5=faa6180e88c8eaf3c6af59c0b3b09d3c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-14-2506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-14-2506%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman-Luca%26aufirst%3DC.%2BG.%26aulast%3DZiazadeh%26aufirst%3DD.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DDebussche%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DElucidation%2520of%2520Acquired%2520Resistance%2520to%2520Bcl-2%2520and%2520MDM2%2520Inhibitors%2520in%2520Acute%2520Leukemia%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2558%26epage%3D2568%26doi%3D10.1158%2F1078-0432.ccr-14-2506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meaux, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barriere, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman-Luca, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shangary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos-Santos, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besret, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorge-Bernat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debussche, L.</span></span> <span> </span><span class="NLM_article-title">SAR405838: An Optimized Inhibitor of MDM2-p53 Interaction That Induces Complete and Durable Tumor Regression</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">5855</span>– <span class="NLM_lpage">5865</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-14-0799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1158%2F0008-5472.CAN-14-0799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25145672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGjsr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=5855-5865&author=S.+Wangauthor=W.+Sunauthor=Y.+Zhaoauthor=D.+McEachernauthor=I.+Meauxauthor=C.+Barriereauthor=J.+A.+Stuckeyauthor=J.+L.+Meagherauthor=L.+Baiauthor=L.+Liuauthor=C.+G.+Hoffman-Lucaauthor=J.+Luauthor=S.+Shangaryauthor=S.+Yuauthor=D.+Bernardauthor=A.+Aguilarauthor=O.+Dos-Santosauthor=L.+Besretauthor=S.+Guerifauthor=P.+Pannierauthor=D.+Gorge-Bernatauthor=L.+Debussche&title=SAR405838%3A+An+Optimized+Inhibitor+of+MDM2-p53+Interaction+That+Induces+Complete+and+Durable+Tumor+Regression&doi=10.1158%2F0008-5472.can-14-0799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">SAR405838: An Optimized Inhibitor of MDM2-p53 Interaction That Induces Complete and Durable Tumor Regression</span></div><div class="casAuthors">Wang, Shaomeng; Sun, Wei; Zhao, Yujun; McEachern, Donna; Meaux, Isabelle; Barriere, Cedric; Stuckey, Jeanne A.; Meagher, Jennifer L.; Bai, Longchuan; Liu, Liu; Hoffman-Luca, Cassandra Gianna; Lu, Jianfeng; Shangary, Sanjeev; Yu, Shanghai; Bernard, Denzil; Aguilar, Angelo; Dos-Santos, Odette; Besret, Laurent; Guerif, Stephane; Pannier, Pascal; Gorge-Bernat, Dimitri; Debussche, Laurent</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5855-5865</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Despite the risk of applying a selection for p53 mutations that escape MDM2 control, blocking MDM2-p53 protein-protein interaction has long been considered by many to offer an attractive cancer therapeutic strategy, a position strongly supported by the findings of this preclin. study.  Blocking the oncoprotein murine double minute 2 (MDM2)-p53 protein-protein interaction has long been considered to offer a broad cancer therapeutic strategy, despite the potential risks of selecting tumors harboring p53 mutations that escape MDM2 control.  In this study, we report a novel small-mol. inhibitor of the MDM2-p53 interaction, SAR405838 (MI-77301), that has been advanced into phase I clin. trials.  SAR405838 binds to MDM2 with Ki = 0.88 nmol/L and has high specificity over other proteins.  A cocrystal structure of the SAR405838:MDM2 complex shows that, in addn. to mimicking three key p53 amino acid residues, the inhibitor captures addnl. interactions not obsd. in the p53-MDM2 complex and induces refolding of the short, unstructured MDM2 N-terminal region to achieve its high affinity.  SAR405838 effectively activates wild-type p53 in vitro and in xenograft tumor tissue of leukemia and solid tumors, leading to p53-dependent cell-cycle arrest and/or apoptosis.  At well-tolerated dose schedules, SAR405838 achieves either durable tumor regression or complete tumor growth inhibition in mouse xenograft models of SJSA-1 osteosarcoma, RS4;11 acute leukemia, LNCaP prostate cancer, and HCT-116 colon cancer.  Remarkably, a single oral dose of SAR405838 is sufficient to achieve complete tumor regression in the SJSA-1 model.  Mechanistically, robust transcriptional upregulation of PUMA induced by SAR405838 results in strong apoptosis in tumor tissue, leading to complete tumor regression.  Our findings provide a preclin. basis upon which to evaluate SAR405838 as a therapeutic agent in patients whose tumors retain wild-type p53.  Cancer Res; 74(20); 5855-65. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptKSwfNhNzNLVg90H21EOLACvtfcHk0lhorb6-mkA8eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGjsr7M&md5=5b35759e14fe4c1ae4e734775c4334a0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-14-0799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-14-0799%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DMeaux%26aufirst%3DI.%26aulast%3DBarriere%26aufirst%3DC.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHoffman-Luca%26aufirst%3DC.%2BG.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DShangary%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DDos-Santos%26aufirst%3DO.%26aulast%3DBesret%26aufirst%3DL.%26aulast%3DGuerif%26aufirst%3DS.%26aulast%3DPannier%26aufirst%3DP.%26aulast%3DGorge-Bernat%26aufirst%3DD.%26aulast%3DDebussche%26aufirst%3DL.%26atitle%3DSAR405838%253A%2520An%2520Optimized%2520Inhibitor%2520of%2520MDM2-p53%2520Interaction%2520That%2520Induces%2520Complete%2520and%2520Durable%2520Tumor%2520Regression%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D5855%26epage%3D5865%26doi%3D10.1158%2F0008-5472.can-14-0799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eeles, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hainaut, P.</span></span> <span> </span><span class="NLM_article-title">The IARC TP53 Database: New Online Mutation Analysis and Recommendations to Users</span>. <i>Human Mutation</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1002/humu.10081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1002%2Fhumu.10081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=12007217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD38XkvVGktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2002&pages=607-614&author=M.+Olivierauthor=R.+Eelesauthor=M.+Hollsteinauthor=M.+A.+Khanauthor=C.+C.+Harrisauthor=P.+Hainaut&title=The+IARC+TP53+Database%3A+New+Online+Mutation+Analysis+and+Recommendations+to+Users&doi=10.1002%2Fhumu.10081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The IARC TP53 database: new online mutation analysis and recommendations to users</span></div><div class="casAuthors">Olivier, Magali; Eeles, Ros; Hollstein, Monica; Khan, Mohammed A.; Harris, Curtis C.; Hainaut, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Human Mutation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">607-614</span>CODEN:
                <span class="NLM_cas:coden">HUMUE3</span>;
        ISSN:<span class="NLM_cas:issn">1059-7794</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Mutations in the tumor suppressor gene TP53 are frequent in most human cancers.  Comparison of the mutation patterns in different cancers may reveal clues on the natural history of the disease.  Over the past 10 yr, several databases of TP53 mutations have been developed.  The most extensive of these databases is maintained and developed at the International Agency for Research on Cancer.  The database compiles all mutations (somatic and inherited), as well as polymorphisms, that have been reported in the published literature since 1989.  The IARC TP53 mutation dataset is the largest dataset available on the variations of any human gene.  The database is available at www.iarc.fr/P53/.  In this paper, we describe recent developments of the database.  These developments include restructuring of the database, which is now patient-centered, with more detailed annotations on the patient (carcinogen exposure, virus infection, genetic background).  In addn., a new online application to retrieve somatic mutation data and analyze mutation patterns is now available.  We also discuss limitations on the use of the database and provide recommendations to users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHsjPDhuXnq7Vg90H21EOLACvtfcHk0lisHdbYbntq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XkvVGktro%253D&md5=314c1e17d468d982f47f354b48e742e4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fhumu.10081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhumu.10081%26sid%3Dliteratum%253Aachs%26aulast%3DOlivier%26aufirst%3DM.%26aulast%3DEeles%26aufirst%3DR.%26aulast%3DHollstein%26aufirst%3DM.%26aulast%3DKhan%26aufirst%3DM.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26aulast%3DHainaut%26aufirst%3DP.%26atitle%3DThe%2520IARC%2520TP53%2520Database%253A%2520New%2520Online%2520Mutation%2520Analysis%2520and%2520Recommendations%2520to%2520Users%26jtitle%3DHuman%2520Mutation%26date%3D2002%26volume%3D19%26spage%3D607%26epage%3D614%26doi%3D10.1002%2Fhumu.10081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roller, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrish, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Potent Non-Peptide MDM2 Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">10130</span>– <span class="NLM_lpage">10131</span>, <span class="refDoi"> DOI: 10.1021/ja051147z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja051147z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1ahtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=10130-10131&author=K.+Dingauthor=Y.+Luauthor=Z.+Nikolovska-Coleskaauthor=S.+Qiuauthor=Y.+Dingauthor=W.+Gaoauthor=J.+Stuckeyauthor=K.+Krajewskiauthor=P.+P.+Rollerauthor=Y.+Tomitaauthor=D.+A.+Parrishauthor=J.+R.+Deschampsauthor=S.+Wang&title=Structure-Based+Design+of+Potent+Non-Peptide+MDM2+Inhibitors&doi=10.1021%2Fja051147z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent Non-Peptide MDM2 Inhibitors</span></div><div class="casAuthors">Ding, Ke; Lu, Yipin; Nikolovska-Coleska, Zaneta; Qiu, Su; Ding, Yousong; Gao, Wei; Stuckey, Jeanne; Krajewski, Krzysztof; Roller, Peter P.; Tomita, York; Parrish, Damon A.; Deschamps, Jeffrey R.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">10130-10131</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A successful structure-based design of a class of nonpeptide small-mol. MDM2 inhibitors targeting the p53-MDM2 protein-protein interaction is reported.  The most potent compd. (I) binds to MDM2 protein with a Ki value of 86 nM and is 18 times more potent than a natural p53 peptide (residues 16-27).  Compd. I is potent in inhibition of cell growth in LNCaP prostate cancer cells with wild-type p53 and shows only a weak activity in PC-3 prostate cancer cells with a deleted p53.  Importantly, I has a minimal toxicity to normal prostate epithelial cells.  Our studies provide a convincing example that structure-based strategy can be employed to design highly potent, nonpeptide, cell-permeable, small-mol. inhibitors to target protein-protein interaction, which remains a very challenging area in chem. biol. and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXaOqJGFoCeLVg90H21EOLACvtfcHk0lisHdbYbntq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1ahtL0%253D&md5=44cc1b7e2ddc86e1bc27c79c150f84c7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja051147z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja051147z%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DQiu%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DParrish%26aufirst%3DD.%2BA.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520Potent%2520Non-Peptide%2520MDM2%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D10130%26epage%3D10131%26doi%3D10.1021%2Fja051147z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">García-Echeverría, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chène, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent Antagonists of the Interaction between Human Double Minute 2 and Tumor Suppressor p53</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">3205</span>– <span class="NLM_lpage">3208</span>, <span class="refDoi"> DOI: 10.1021/jm990966p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990966p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltlOruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3205-3208&author=C.+Garc%C3%ADa-Echeverr%C3%ADaauthor=P.+Ch%C3%A8neauthor=M.+J.+J.+Blommersauthor=P.+Furet&title=Discovery+of+Potent+Antagonists+of+the+Interaction+between+Human+Double+Minute+2+and+Tumor+Suppressor+p53&doi=10.1021%2Fjm990966p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Antagonists of the Interaction between Human Double Minute 2 and Tumor Suppressor p53</span></div><div class="casAuthors">Garcia-Echeverria, Carlos; Chene, Patrick; Blommers, Marcel J. J.; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3205-3208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of a drug discovery program to identify antagonists of the p53/hdm2 (human double minute 2) protein-protein interaction, the authors have attempted to det. the amino acid specificities of hdm2's binding pockets to establish a pharmacophore model for this protein-protein interaction.  This work has resulted in the identification of highly potent peptide antagonists.  Structural information has been exploited to increase the hdm2-binding affinity of short peptide motifs derived from the N-terminal domain of the human wild-type p53 protein.  Combining conformational constraints as selected by mol. modeling with functional groups that are able to establish addnl. electrostatic and van der Waals interactions with the hdm2 protein, the authors have been able to increase the hdm2-binding affinity of the authors initial peptide 1700-fold.  Particularly interesting is the increase in binding affinity obtained by replacing tryptophan with 6-chlorotryptophan (IC50 = 314 nM vs. IC50 = 5 nM, 63-fold).  The new interactions identified and exptl. confirmed in this work could be directly applied to the optimization of nonpeptidic leads or incorporated into the "de novo" design of antagonists of the p53/hdm2 protein-protein interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpOq3ikfGqz7Vg90H21EOLACvtfcHk0lisHdbYbntq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltlOruro%253D&md5=b921adfdd6a5c85510276a6446e600a8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm990966p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990966p%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Echeverr%25C3%25ADa%26aufirst%3DC.%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%2BJ.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Potent%2520Antagonists%2520of%2520the%2520Interaction%2520between%2520Human%2520Double%2520Minute%25202%2520and%2520Tumor%2520Suppressor%2520p53%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3205%26epage%3D3208%26doi%3D10.1021%2Fjm990966p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhFc34PhV8B3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lhFc34PhV8B3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisarev, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisareva, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestova, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Round, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svergun, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span> <span> </span><span class="NLM_article-title">Structural Insights Into eRF3 and Stop Codon Recognition by eRF1</span>. <i>Genes Develop.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1106</span>– <span class="NLM_lpage">1118</span>, <span class="refDoi"> DOI: 10.1101/gad.1770109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1101%2Fgad.1770109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=19417105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Crs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=1106-1118&author=Z.+Chengauthor=K.+Saitoauthor=A.+V.+Pisarevauthor=M.+Wadaauthor=V.+P.+Pisarevaauthor=T.+V.+Pestovaauthor=M.+Gajdaauthor=A.+Roundauthor=C.+Kongauthor=M.+Limauthor=Y.+Nakamuraauthor=D.+I.+Svergunauthor=K.+Itoauthor=H.+Song&title=Structural+Insights+Into+eRF3+and+Stop+Codon+Recognition+by+eRF1&doi=10.1101%2Fgad.1770109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into eRF3 and stop codon recognition by eRF1</span></div><div class="casAuthors">Cheng, Zhihong; Saito, Kazuki; Pisarev, Andrey V.; Wada, Miki; Pisareva, Vera P.; Pestova, Tatyana V.; Gajda, Michal; Round, Adam; Kong, Chunguang; Lim, Mengkiat; Nakamura, Yoshikazu; Svergun, Dmitri I.; Ito, Koichi; Song, Haiwei</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1106-1118</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Eukaryotic translation termination is mediated by two interacting release factors, eRF1 and eRF3, which act cooperatively to ensure efficient stop codon recognition and fast polypeptide release.  The crystal structures of human and Schizosaccharomyces pombe full-length eRF1 in complex with eRF3 lacking the GTPase domain revealed details of the interaction between these two factors and marked conformational changes in eRF1 that occur upon binding to eRF3, leading eRF1 to resemble a tRNA mol.  Small-angle X-ray scattering anal. of the eRF1/eRF3/GTP complex suggested that eRF1's M domain contacts eRF3's GTPase domain.  Consistently, mutation of Arg192, which is predicted to come in close contact with the switch regions of eRF3, revealed its important role for eRF1's stimulatory effect on eRF3's GTPase activity.  An ATP mol. used as a crystn. additive was bound in eRF1's putative decoding area.  Mutational anal. of the ATP-binding site shed light on the mechanism of stop codon recognition by eRF1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdD0dkm3bQLVg90H21EOLACvtfcHk0lhM2u7qaifnYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Crs7c%253D&md5=57df8961d0eb44ab277dc2f45c25a63c</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1101%2Fgad.1770109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1770109%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DPisarev%26aufirst%3DA.%2BV.%26aulast%3DWada%26aufirst%3DM.%26aulast%3DPisareva%26aufirst%3DV.%2BP.%26aulast%3DPestova%26aufirst%3DT.%2BV.%26aulast%3DGajda%26aufirst%3DM.%26aulast%3DRound%26aufirst%3DA.%26aulast%3DKong%26aufirst%3DC.%26aulast%3DLim%26aufirst%3DM.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DSvergun%26aufirst%3DD.%2BI.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DH.%26atitle%3DStructural%2520Insights%2520Into%2520eRF3%2520and%2520Stop%2520Codon%2520Recognition%2520by%2520eRF1%26jtitle%3DGenes%2520Develop.%26date%3D2009%26volume%3D23%26spage%3D1106%26epage%3D1118%26doi%3D10.1101%2Fgad.1770109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrio-Garcia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyler, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berninghausen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, R.</span></span> <span> </span><span class="NLM_article-title">Cryoelectron Microscopic Structures of Eukaryotic Translation Termination Complexes Containing eRF1-eRF3 or eRF1-ABCE1</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.04.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2Fj.celrep.2014.04.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=25001285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFans7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=59-65&author=A.+Preisauthor=A.+Heuerauthor=C.+Barrio-Garciaauthor=A.+Hauserauthor=D.+E.+Eylerauthor=O.+Berninghausenauthor=R.+Greenauthor=T.+Beckerauthor=R.+Beckmann&title=Cryoelectron+Microscopic+Structures+of+Eukaryotic+Translation+Termination+Complexes+Containing+eRF1-eRF3+or+eRF1-ABCE1&doi=10.1016%2Fj.celrep.2014.04.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Cryoelectron Microscopic Structures of Eukaryotic Translation Termination Complexes Containing eRF1-eRF3 or eRF1-ABCE1</span></div><div class="casAuthors">Preis, Anne; Heuer, Andre; Barrio-Garcia, Clara; Hauser, Andreas; Eyler, Daniel E.; Berninghausen, Otto; Green, Rachel; Becker, Thomas; Beckmann, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-65</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Termination and ribosome recycling are essential processes in translation.  In eukaryotes, a stop codon in the ribosomal A site is decoded by a ternary complex consisting of release factors eRF1 and guanosine triphosphate (GTP)-bound eRF3.  After GTP hydrolysis, eRF3 dissocs., and ABCE1 can bind to eRF1-loaded ribosomes to stimulate peptide release and ribosomal subunit dissocn.  Here, we present cryoelectron microscopic (cryo-EM) structures of a pretermination complex contg. eRF1-eRF3 and a termination/prerecycling complex contg. eRF1-ABCE1. eRF1 undergoes drastic conformational changes: its central domain harboring the catalytically important GGQ loop is either packed against eRF3 or swung toward the peptidyl transferase center when bound to ABCE1.  Addnl., in complex with eRF3, the N-terminal domain of eRF1 positions the conserved NIKS motif proximal to the stop codon, supporting its suggested role in decoding, yet it appears to be delocalized in the presence of ABCE1.  These results suggest that stop codon decoding and peptide release can be uncoupled during termination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovl2-tKYfb5bVg90H21EOLACvtfcHk0lhM2u7qaifnYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFans7jM&md5=28bff59bf85863587f9fb6234d266731</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.04.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.04.058%26sid%3Dliteratum%253Aachs%26aulast%3DPreis%26aufirst%3DA.%26aulast%3DHeuer%26aufirst%3DA.%26aulast%3DBarrio-Garcia%26aufirst%3DC.%26aulast%3DHauser%26aufirst%3DA.%26aulast%3DEyler%26aufirst%3DD.%2BE.%26aulast%3DBerninghausen%26aufirst%3DO.%26aulast%3DGreen%26aufirst%3DR.%26aulast%3DBecker%26aufirst%3DT.%26aulast%3DBeckmann%26aufirst%3DR.%26atitle%3DCryoelectron%2520Microscopic%2520Structures%2520of%2520Eukaryotic%2520Translation%2520Termination%2520Complexes%2520Containing%2520eRF1-eRF3%2520or%2520eRF1-ABCE1%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D59%26epage%3D65%26doi%3D10.1016%2Fj.celrep.2014.04.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L.'A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.%27A.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4CRBN+ubiquitin+ligase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lilB4nAPzcmsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%2527A.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschamps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of Chemically Stable, Potent, and Efficacious MDM2 Inhibitors That Exploit the Retro-Mannich Ring-Opening-Cyclization Reaction Mechanism in Spiro-oxindoles</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10486</span>– <span class="NLM_lpage">10498</span>, <span class="refDoi"> DOI: 10.1021/jm501541j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501541j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10486-10498&author=A.+Aguilarauthor=W.+Sunauthor=L.+Liuauthor=J.+Luauthor=D.+McEachernauthor=D.+Bernardauthor=J.+R.+Deschampsauthor=S.+Wang&title=Design+of+Chemically+Stable%2C+Potent%2C+and+Efficacious+MDM2+Inhibitors+That+Exploit+the+Retro-Mannich+Ring-Opening-Cyclization+Reaction+Mechanism+in+Spiro-oxindoles&doi=10.1021%2Fjm501541j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Chemically Stable, Potent, and Efficacious MDM2 Inhibitors That Exploit the Retro-Mannich Ring-Opening-Cyclization Reaction Mechanism in Spiro-oxindoles</span></div><div class="casAuthors">Aguilar, Angelo; Sun, Wei; Liu, Liu; Lu, Jianfeng; McEachern, Donna; Bernard, Denzil; Deschamps, Jeffrey R.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10486-10498</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spiropyrrolidineoxindoles such as I (R = trans-4-hydroxycyclohexyl, 4-HO2CC6H4) were prepd. as inhibitors of the interaction between the proteins MDM2 and p53 for use as antitumor agents; the second-generation inhibitors incorporate sym. disubstitution in the pyrrolidine ring next to the spiro junction in order to facilitate isomerization via retro-Mannich/Mannich reactions to the more active stereoisomers (in which the aryl rings attached to the pyrrolidine are trans) as well as to improve stability in vitro and in vivo.  In particular, I (R = 4-HO2CC6H4) (MI-1061), which bound to MDM2 with a Ki value of 0.16 nM, showed excellent chem. stability and caused regression of human SJSA-1 xenograft tumors in mice.  The structures of diphenylhydroxyethyl spiropyrrolidineoxindole and spiropyrrolidineoxindolecarboxylate intermediates in the prepn. of I (R = trans-4-hydroxycyclohexyl, 4-HO2CC6H4) were detd. by X-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_es94k2y2-rVg90H21EOLACvtfcHk0lilB4nAPzcmsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nt73O&md5=6399fb5e19fc8d69e40e6fd19721e270</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm501541j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501541j%26sid%3Dliteratum%253Aachs%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DDeschamps%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520Chemically%2520Stable%252C%2520Potent%252C%2520and%2520Efficacious%2520MDM2%2520Inhibitors%2520That%2520Exploit%2520the%2520Retro-Mannich%2520Ring-Opening-Cyclization%2520Reaction%2520Mechanism%2520in%2520Spiro-oxindoles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10486%26epage%3D10498%26doi%3D10.1021%2Fjm501541j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Efremov, I.
V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajdos, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capetta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorff, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutra, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansour, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McColl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oborski, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, T.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withka, J. M.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of a Novel Spiropyrrolidine Inhibitor of β-Secretase (BACE1) through Fragment-Based Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9069</span>– <span class="NLM_lpage">9088</span>, <span class="refDoi"> DOI: 10.1021/jm201715d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201715d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFOnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9069-9088&author=I.%0AV.+Efremovauthor=F.+F.+Vajdosauthor=K.+A.+Borzilleriauthor=S.+Capettaauthor=H.+Chenauthor=P.+H.+Dorffauthor=J.+K.+Dutraauthor=S.+W.+Goldsteinauthor=M.+Mansourauthor=A.+McCollauthor=S.+Noellauthor=C.+E.+Oborskiauthor=T.+N.+O%E2%80%99Connellauthor=T.%0AJ.+O%E2%80%99Sullivanauthor=J.+Panditauthor=H.+Wangauthor=B.+Weiauthor=J.+M.+Withka&title=Discovery+and+Optimization+of+a+Novel+Spiropyrrolidine+Inhibitor+of+%CE%B2-Secretase+%28BACE1%29+through+Fragment-Based+Drug+Design&doi=10.1021%2Fjm201715d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of a novel spiropyrrolidine inhibitor of β-secretase (BACE1) through fragment-based drug design</span></div><div class="casAuthors">Efremov, Ivan V.; Vajdos, Felix F.; Borzilleri, Kris A.; Capetta, Steven; Chen, Hou; Dorff, Peter H.; Dutra, Jason K.; Goldstein, Steven W.; Mansour, Mahmoud; McColl, Alexander; Noell, Stephen; Oborski, Christine E.; O?Connell, Thomas N.; O?Sullivan, Theresa J.; Pandit, Jayvardhan; Wang, Hong; Wei, BinQing; Withka, Jane M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9069-9088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The aspartyl protease β-secretase, or BACE, has been demonstrated to be a key factor in the proteolytic formation of A,β-peptide, a major component of plaques in the brains of Alzheimer's disease (AD) patients, and inhibition of this enzyme has emerged as a major strategy for pharmacol. intervention in AD.  An X-ray-based fragment screen of Pfizer's proprietary fragment collection has resulted in the identification of a novel BACE binder featuring spiropyrrolidine framework.  Although exhibiting only weak inhibitory activity against the BACE enzyme, the small compd. was verified by biophys. and NMR-based methods as a bona fide BACE inhibitor.  Subsequent optimization of the lead compd., relying heavily on structure-based drug design and computational prediction of physiochem. properties, resulted in a nearly 1000-fold improvement in potency while maintaining ligand efficiency and properties predictive of good permeability and low P-gp liability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB8e9NnCgYXrVg90H21EOLACvtfcHk0lilB4nAPzcmsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFOnsLs%253D&md5=c87263dd8136630e2e1f17a7fa9b5b9f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm201715d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201715d%26sid%3Dliteratum%253Aachs%26aulast%3DEfremov%26aufirst%3DI.%2BV.%26aulast%3DVajdos%26aufirst%3DF.%2BF.%26aulast%3DBorzilleri%26aufirst%3DK.%2BA.%26aulast%3DCapetta%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DDorff%26aufirst%3DP.%2BH.%26aulast%3DDutra%26aufirst%3DJ.%2BK.%26aulast%3DGoldstein%26aufirst%3DS.%2BW.%26aulast%3DMansour%26aufirst%3DM.%26aulast%3DMcColl%26aufirst%3DA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DOborski%26aufirst%3DC.%2BE.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DT.%2BN.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DT.%2BJ.%26aulast%3DPandit%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWithka%26aufirst%3DJ.%2BM.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520a%2520Novel%2520Spiropyrrolidine%2520Inhibitor%2520of%2520%25CE%25B2-Secretase%2520%2528BACE1%2529%2520through%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9069%26epage%3D9088%26doi%3D10.1021%2Fjm201715d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span> </span><span class="NLM_article-title">Molecular
Operating Environment (MOE), 2013.08</span>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span>,
1010 Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular%0AOperating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+ULC%2C%0A1010+Sherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%250AOperating%2520Environment%2520%2528MOE%2529%252C%25202013.08%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Ben-Shalom, I. Y.</span>; <span class="NLM_string-name">Brozell, S. R.</span>; <span class="NLM_string-name">Cerutti, T. E.</span>; <span class="NLM_string-name">Cheatham, V. W. D.,  III</span>; <span class="NLM_string-name">Cruzeiro, T. A.</span>; <span class="NLM_string-name">Darden, R. E.</span>; <span class="NLM_string-name">Duke, D.</span>; <span class="NLM_string-name">Ghoreishi, M. K.</span>; <span class="NLM_string-name">Gilson, H.</span>; <span class="NLM_string-name">Gohlke, A. W.</span>; <span class="NLM_string-name">Goetz, D.</span>; <span class="NLM_string-name">Greene, R. H.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Izadi, S.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Kurtzman, T.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Li, P.</span>; <span class="NLM_string-name">Lin, C.</span>; <span class="NLM_string-name">Liu, J.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Mermelstein, D. J.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Miao, Y.</span>; <span class="NLM_string-name">Monard, G.</span>; <span class="NLM_string-name">Nguyen, C.</span>; <span class="NLM_string-name">Nguyen, H.</span>; <span class="NLM_string-name">Omelyan, I.</span>; <span class="NLM_string-name">Onufriev, A.</span>; <span class="NLM_string-name">Pan, F.</span>; <span class="NLM_string-name">Qi, R.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Schott-Verdugo, S.</span>; <span class="NLM_string-name">Shen, J.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, J.</span>; <span class="NLM_string-name">Salomon-Ferrer, R.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wei, H.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Xiao, L.</span>; <span class="NLM_string-name">York, D. M.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 2018</i>; University of California, San Francisco, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=D.+A.+Case&author=I.+Y.+Ben-Shalom&author=S.+R.+Brozell&author=T.+E.+Cerutti&author=V.+W.+D.+Cheatham&author=T.+A.+Cruzeiro&author=R.+E.+Darden&author=D.+Duke&author=M.+K.+Ghoreishi&author=H.+Gilson&author=A.+W.+Gohlke&author=D.+Goetz&author=R.+H.+Greene&author=N.+Homeyer&author=S.+Izadi&author=A.+Kovalenko&author=T.+Kurtzman&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=C.+Lin&author=J.+Liu&author=T.+Luchko&author=R.+Luo&author=D.+J.+Mermelstein&author=K.+M.+Merz&author=Y.+Miao&author=G.+Monard&author=C.+Nguyen&author=H.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=F.+Pan&author=R.+Qi&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=S.+Schott-Verdugo&author=J.+Shen&author=C.+L.+Simmerling&author=J.+Smith&author=R.+Salomon-Ferrer&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=H.+Wei&author=R.+M.+Wolf&author=X.+Wu&author=L.+Xiao&author=D.+M.+York&author=P.+A.+Kollman&title=AMBER+2018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%25202018%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lg24KQieDBdyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lg24KQieDBdyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lgBq-Crjvk3rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shargary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A Potent Small-Molecule Inhibitor of the MDM2-p53 Interaction (MI-888) Achieved Complete and Durable Tumor Regression in Mice</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5553</span>– <span class="NLM_lpage">5561</span>, <span class="refDoi"> DOI: 10.1021/jm4005708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4005708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslCisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5553-5561&author=Y.+Zhaoauthor=S.+Yuauthor=W.+Sunauthor=L.+Liuauthor=J.+Luauthor=D.+McEachernauthor=S.+Shargaryauthor=D.+Bernardauthor=X.+Liauthor=T.+Zhaoauthor=P.+Zouauthor=D.+Sunauthor=S.+Wang&title=A+Potent+Small-Molecule+Inhibitor+of+the+MDM2-p53+Interaction+%28MI-888%29+Achieved+Complete+and+Durable+Tumor+Regression+in+Mice&doi=10.1021%2Fjm4005708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent Small-Molecule Inhibitor of the MDM2-p53 Interaction (MI-888) Achieved Complete and Durable Tumor Regression in Mice</span></div><div class="casAuthors">Zhao, Yujun; Yu, Shanghai; Sun, Wei; Liu, Liu; Lu, Jianfeng; McEachern, Donna; Shargary, Sanjeev; Bernard, Denzil; Li, Xiaoqin; Zhao, Ting; Zou, Peng; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5553-5561</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors previously reported the discovery of a class of spirooxindoles as potent and selective small-mol. inhibitors of the MDM2-p53 interaction (MDM2 inhibitors).  The authors report herein the authors' efforts to improve their pharmacokinetic properties and in vivo antitumor activity.  The authors' efforts led to the identification of MI-888 as a potent MDM2 inhibitor (Ki = 0.44 nM) with a superior pharmacokinetic profile and enhanced in vivo efficacy.  MI-888 is capable of achieving rapid, complete, and durable tumor regression in two types of xenograft models of human cancer with oral administration and represents the most potent and efficacious MDM2 inhibitor reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr06DpGqpt0TLVg90H21EOLACvtfcHk0lgBq-Crjvk3rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslCisb4%253D&md5=a53931e3f91a1c1a302ac88bbb4d5c0d</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm4005708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4005708%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DShargary%26aufirst%3DS.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DZou%26aufirst%3DP.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520Potent%2520Small-Molecule%2520Inhibitor%2520of%2520the%2520MDM2-p53%2520Interaction%2520%2528MI-888%2529%2520Achieved%2520Complete%2520and%2520Durable%2520Tumor%2520Regression%2520in%2520Mice%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5553%26epage%3D5561%26doi%3D10.1021%2Fjm4005708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HXB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HXB','PDB','5HXB'); return false;">PDB: 5HXB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i49"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00846">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_75557"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00846">10.1021/acs.jmedchem.9b00846</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Cell growth inhibitory activity of the MDM2 inhibitors, bona fide MDM2 degraders, MD-222, and the new putative MDM2 degrader <b>MG-277</b> in p53 wild-type and p53 mutated/deleted cell lines; potency of different lenalidomide conjugated analogues of <b>MG-277</b> in cell growth inhibition and GSPT1 degradation; determination of the half maximal degradation concentration (DC<sub>50</sub>) of GSPT1 degradation by different analogues of <b>MG-277</b> in RS4;11 cell line after a 24 h treatment; correlation between GSPT1 degradation DC<sub>50</sub> of the compounds in RS4;11 cells and IC<sub>50</sub> in growth inhibition in RS4;11 and IRMI-2 cell lines; detailed illustration of the predicted binding mode of <b>MG-277</b> to the cereblon-GSPT1 complex; NMR spectra of <b>MG-277</b>; mass spectrum and purity spectrum of <b>MG-277</b>; CSV file containing SMILES, binding and cellular data for final target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf">PDF</a>)</p></li><li><p class="inline">JMC, SMILES, molecular glue paper 091719 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_002.csv">CSV</a>)</p></li><li><p class="inline">Atomic coordinates of models MG277, GSPT1, and CRBN (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_001.pdf">jm9b00846_si_001.pdf (2.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_002.csv">jm9b00846_si_002.csv (2.4 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00846/suppl_file/jm9b00846_si_003.pdb">jm9b00846_si_003.pdb (729.72 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-21%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00846" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00846" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679987f9da783cd6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
