
<html lang="en"     class="pb-page"  data-request-id="b3761b42-344d-4d19-a636-f98df5b03c74"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm100478y;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2010.53.issue-15;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines." /></meta><meta name="dc.Creator" content="Feng  Liu" /></meta><meta name="dc.Creator" content="Xin  Chen" /></meta><meta name="dc.Creator" content="Abdellah  Allali-Hassani" /></meta><meta name="dc.Creator" content="Amy M.  Quinn" /></meta><meta name="dc.Creator" content="Tim J.  Wigle" /></meta><meta name="dc.Creator" content="Gregory A.  Wasney" /></meta><meta name="dc.Creator" content="Aiping  Dong" /></meta><meta name="dc.Creator" content="Guillermo  Senisterra" /></meta><meta name="dc.Creator" content="Irene  Chau" /></meta><meta name="dc.Creator" content="Alena  Siarheyeva" /></meta><meta name="dc.Creator" content="Jacqueline L.  Norris" /></meta><meta name="dc.Creator" content="Dmitri B.  Kireev" /></meta><meta name="dc.Creator" content="Ajit  Jadhav" /></meta><meta name="dc.Creator" content="J. Martin  Herold" /></meta><meta name="dc.Creator" content="William P.  Janzen" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Stephen V.  Frye" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Anton  Simeonov" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers. Genetic knockdown of G9a inhibits..." /></meta><meta name="Description" content="Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers. Genetic knockdown of G9a inhibits..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 9, 2010" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm100478y" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2010 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm100478y" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm100478y" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm100478y" /></link>
        
    
    

<title>Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines. | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm100478y" /></meta><meta property="og:title" content="Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines." /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0017.jpeg" /></meta><meta property="og:description" content="Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers. Genetic knockdown of G9a inhibits cancer cell growth, and the dimethylation of p53 K373 results in the inactivation of p53. Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by 3a (BIX01294), a selective small molecule inhibitor of G9a and GLP, led to the discovery of 10 (UNC0224) as a potent G9a inhibitor with excellent selectivity. A high resolution X-ray crystal structure of the G9a−10 complex, the first cocrystal structure of G9a with a small molecule inhibitor, was obtained. On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-dimethylaminopropoxy side chain of 10 resulted in the discovery of 29 (UNC0321) (Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm100478y"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm100478y">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm100478y&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm100478y&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm100478y&amp;href=/doi/10.1021/jm100478y" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2010</span><span class="cit-fg-volume">, 53</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 5844-5857</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/53/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm100482n" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm100574m" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines.<a class="ref internalNav" href="#ac1" aria-label=""></a></span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Liu">Feng Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Chen">Xin Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Abdellah++Allali-Hassani">Abdellah Allali-Hassani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Amy+M.++Quinn">Amy M. Quinn</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tim+J.++Wigle">Tim J. Wigle</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gregory+A.++Wasney">Gregory A. Wasney</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aiping++Dong">Aiping Dong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guillermo++Senisterra">Guillermo Senisterra</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Irene++Chau">Irene Chau</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alena++Siarheyeva">Alena Siarheyeva</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+L.++Norris">Jacqueline L. Norris</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dmitri+B.++Kireev">Dmitri B. Kireev</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ajit++Jadhav">Ajit Jadhav</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Martin++Herold">J. Martin Herold</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+P.++Janzen">William P. Janzen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+V.++Frye">Stephen V. Frye</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Simeonov">Anton Simeonov</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">‡</span> <span class="aff-text">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">§</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">∥</span> <span class="aff-text">NIH Chemical Genomics Center, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892</span></div><div class="corresp-info"><strong>*</strong>To whom correspondence should be addressed. Phone: 919-843-8459. Fax: 919-843-8465. E-mail: <a href="/cdn-cgi/l/email-protection#a1cbc8c0cfcbc8cfe1d4cfc28fc4c5d4"><span class="__cf_email__" data-cfemail="dfb5b6beb1b5b6b19faab1bcf1babbaa">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm100478y&amp;href=/doi/10.1021%2Fjm100478y" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2010</span></span><span class="cit-volume">, 53</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 5844–5857</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 9, 2010</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 April 2010</li><li><span class="item_label"><b>Published</b> online</span>9 July 2010</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 August 2010</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm100478y" title="DOI URL">https://doi.org/10.1021/jm100478y</a></div><div class="article_header-article-copyright"><strong>Copyright © 2010 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5844%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFeng%2BLiu%252C%2BXin%2BChen%252C%2BAbdellah%2BAllali-Hassani%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D53%26issueNum%3D15%26contentID%3Djm100478y%26title%3DProtein%2BLysine%2BMethyltransferase%2BG9a%2BInhibitors%253A%2BDesign%252C%2BSynthesis%252C%2Band%2BStructure%2BActivity%2BRelationships%2Bof%2B2%252C4-Diamino-7-aminoalkoxy-quinazolines.%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5857%26publicationDate%3DAugust%2B2010">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm100478y"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5380</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">143</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm100478y" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines.&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Abdellah&quot;,&quot;last_name&quot;:&quot;Allali-Hassani&quot;},{&quot;first_name&quot;:&quot;Amy&quot;,&quot;last_name&quot;:&quot;M. Quinn&quot;},{&quot;first_name&quot;:&quot;Tim&quot;,&quot;last_name&quot;:&quot;J. Wigle&quot;},{&quot;first_name&quot;:&quot;Gregory&quot;,&quot;last_name&quot;:&quot;A. Wasney&quot;},{&quot;first_name&quot;:&quot;Aiping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Guillermo&quot;,&quot;last_name&quot;:&quot;Senisterra&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;Chau&quot;},{&quot;first_name&quot;:&quot;Alena&quot;,&quot;last_name&quot;:&quot;Siarheyeva&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;L. Norris&quot;},{&quot;first_name&quot;:&quot;Dmitri&quot;,&quot;last_name&quot;:&quot;B. Kireev&quot;},{&quot;first_name&quot;:&quot;Ajit&quot;,&quot;last_name&quot;:&quot;Jadhav&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Martin Herold&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;P. Janzen&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;V. Frye&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Simeonov&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2010&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;53&quot;,&quot;pages&quot;:&quot;5844-5857&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm100478y&quot;},&quot;abstract&quot;:&quot;Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers. Genetic knockdown of G9a inhibits cancer cell growth, and the dimethylation of p53 K373 results in the inactivation of p53. Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by 3a (BIX01294), a selective small molecule inhibitor of G9a and GLP, led to the discovery of 10 (UNC0224) as a potent G9a inhibitor with excellent selectivity. A high resolution X-ray crystal structure of the G9a−10 complex, the first cocrystal structure of G9a with a small molecule inhibitor, was obtained. On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-dimethylaminopropoxy side chain of 10 resulted in the discovery of 29 (UNC0321) (Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm100478y&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100478y" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm100478y&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100478y" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm100478y&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100478y" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm100478y&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100478y&amp;href=/doi/10.1021/jm100478y" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm100478y" /></input><a href="/doi/pdf/10.1021/jm100478y" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26pmid%3D20614940%26genre%3Darticle%26aulast%3DLiu%26date%3D2010%26atitle%3DProtein%2BLysine%2BMethyltransferase%2BG9a%2BInhibitors%253A%2BDesign%252C%2BSynthesis%252C%2Band%2BStructure%2BActivity%2BRelationships%2Bof%2B2%252C4-Diamino-7-aminoalkoxy-quinazolines.%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D53%26issue%3D15%26spage%3D5844%26epage%3D5857%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/53/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jmcmar.2010.53.issue-15/production/jmcmar.2010.53.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers. Genetic knockdown of G9a inhibits cancer cell growth, and the dimethylation of p53 K373 results in the inactivation of p53. Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by <b>3a</b> (BIX01294), a selective small molecule inhibitor of G9a and GLP, led to the discovery of <b>10</b> (UNC0224) as a potent G9a inhibitor with excellent selectivity. A high resolution X-ray crystal structure of the G9a−<b>10</b> complex, the first cocrystal structure of G9a with a small molecule inhibitor, was obtained. On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-dimethylaminopropoxy side chain of <b>10</b> resulted in the discovery of <b>29</b> (UNC0321) (Morrison <i>K</i><sub>i</sub> = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote withdrawn"><h4><span>†</span> </h4><p class="last">The coordinates and structure factors of the cocrystal structure of the G9a−<b>10</b> complex have been deposited in the Protein Data Bank (<a href="http://www.pdb.org" class="extLink">www.pdb.org</a>, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a>).</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetics is defined as changes in gene function that are mitotically and/or meiotically heritable and that do not entail a change in DNA sequence. Over the past decade, the cellular machinery that creates these heritable changes has been intensely investigated by the biomedical research community as there is no area of biology, or for that matter no area of human health, where epigenetics may not play a fundamental role.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The template upon which the epigenome is written is chromatin that is primarily made up of DNA wrapped around octamers of histone proteins. The state of chromatin, and therefore access to the genetic code, is mainly regulated by covalent and reversible PTMs to histones and DNA, and the recognition of these marks by other proteins and protein complexes. The PTMs<a class="ref internalNav" href="#fn2" aria-label="a">a</a><div class="NLM_fn" id="fn2">a<p class="last">Abbreviations: PTMs, post-translational modifications; PKMT, protein lysine methyltransferase; EHMT2, euchromatic histone lysine methyltransferase 2; H3K9, histone H3 lysine 9; p53 K373, lysine 373 of p53; SAM, <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine; SAR, structure−activity relationships; iPS cells, induced pluripotent stem cells; GLP, G9a like protein; EHMT1, euchromatic histone lysine methyltransferase 1; SET, suppressor of variegation 3−9, enhancer of zeste, and trithorax; SAH, <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine; CLOT, chemiluminescence-based oxygen tunneling; ECSD, enzyme-coupled SAH detection; ITC, isothermal titration calorimetry; FP, fluorescence polarization; DSF, differential scanning fluorimetry; MCE, microfluidic capillary electrophoresis; PRMT, protein arginine methyltransferase.</p></div> of histones and DNA include: histone lysine methylation, arginine methylation, lysine acetylation, sumoylation, ADP-ribosylation, ubiquitination, glycosylation and phosphorylation, and DNA methylation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Given the widespread importance of chromatin regulation to cell biology, the enzymes that produce these modifications (the “writers”), the proteins that recognize them (the “readers”), and the enzymes that remove them (the “erasers”) are critical targets for manipulation to further understand the histone code<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> and its role in human disease. Indeed, small molecule histone deacetylase inhibitors<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and DNA methyltransferase inhibitors<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> have already proven useful in the treatment of cancer.</div><div class="NLM_p">Among the “writers” of the histone code, protein lysine methyltransferases (PKMTs) that catalyze mono-, di-, and/or trimethylation of lysine residues of various proteins including histones have received great attention because of the essential function of histone lysine methylation in many biological processes including gene expression and transcriptional regulation, heterochromatin formation, and X-chromosome inactivation.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Since the first PKMT was characterized a decade ago,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> more than 50 human PKMTs have been identified.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> Growing evidence suggests that PKMTs play important roles in the development of various human diseases including cancer,<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10-13)</a> inflammation,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> drug addiction,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> mental retardation,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and HIV-1 latency maintenance.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In particular, G9a (also known as EHMT2), which was initially identified as a H3K9 methyltransferase,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> is overexpressed in human cancers. It has been shown that knockdown of G9a inhibits cancer cell growth.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> In addition to catalyzing mono- and dimethylation of H3K9, it has been shown that G9a methylates lysine 373 (K373) of p53, a tumor suppressor.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The dimethylation of p53 K373 results in the inactivation of p53, which is implicated in over 50% of cancers.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> These observations suggest that inhibition of G9a is a potential approach for cancer treatment. Additionally, <b>3a</b> (BIX01294), a small molecule G9a inhibitor,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> is efficacious as a replacement for <i>oct</i>3/4, one of the four original genetic factors used for reprogramming of mammalian somatic cells into induced pluripotent stem (iPS) cells.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> Thus, small molecule PKMT inhibitors could play an important role in stem cell biology and regenerative medicine.</div><div class="NLM_p">Despite the tremendous progress made in identifying new PKMTs and elucidating their biological function, generating small molecule PKMT inhibitors is lagging significantly behind. In fact, only two selective small molecule PKMT inhibitors (SU(VAR)3−9 inhibitor Chaetocin and G9a inhibitor <b>3a</b>)<a onclick="showRef(event, 'ref22 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref22 ref25 ref26 ref27">(22, 25-27)</a> have been reported since the first PKMT was characterized in 2000.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> This gap in creating small molecule PKMT inhibitors presents a major challenge and opportunity for the biomedical research community. The creation of a “tool-kit” of potent, selective, and well-characterized chemical probes<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> of PKMTs will permit biological and therapeutic hypotheses to be tested with high confidence in cell-based and animal models of human biology and disease.</div><div class="NLM_p">To provide potent and selective G9a inhibitors as research tools and make them available to the biomedical research community without restrictions on their use, we have explored the 2,4-diamino-6,7-dimethoxyquinazoline template represented by <b>3a</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). When we began our work, this compound was the only known selective small molecule inhibitor of G9a and GLP (also known as EHMT1), which is another H3K9 and p53 K373 methyltransferase that shares 80% sequence identity with G9a in their respective SET domains.<a onclick="showRef(event, 'ref22 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref29">(22, 29)</a> Although <b>3a</b> was initially reported to be selective for G9a (IC<sub>50</sub> = 1.7 μM) over GLP (IC<sub>50</sub> = 38 μM),<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> a second study using linear assay conditions for both enzymes revealed that <b>3a</b> inhibited GLP (IC<sub>50</sub> = 0.7 μM) as well as or better than G9a (IC<sub>50</sub> = 1.9 μM).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> This observed selectivity difference is likely due to that the original study did not use identical assay conditions for both enzymes: the GLP assay was conducted under conditions where the reaction was oversaturated while the G9a assay was carried out under linear assay conditions. More recently, <b>3a</b> has been shown to be about 10-fold selective for GLP (IC<sub>50</sub> = 27 nM) over G9a (IC<sub>50</sub> = 250 nM) in chemiluminescence-based oxygen tunneling (CLOT) assays.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Because no SAR was reported for this quinazoline scaffold, we decided to investigate multiple regions of this template to elucidate SAR and improve potency and selectivity. In our previous letter,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> we reported preliminary SAR findings, the discovery of <b>10</b> (UNC0224) as a potent and selective G9a inhibitor, and a 1.7 Å X-ray crystal structure of the G9a−<b>10</b> complex, the first cocrystal structure of G9a with a small molecule inhibitor. In this article, we describe full accounts of the design, synthesis, biological evaluation, and SAR of compounds that explore three regions (i.e., 2-amino, 4-amino, and 7-methoxy) of this quinazoline scaffold. In particular, using the structural insights revealed by the G9a−<b>10</b> cocrystal structure, we optimized the 7-dimethylaminopropoxy side chain, which was not extensively explored in our previous letter.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> This optimization resulted in the discovery of <b>29</b> (UNC0321) (Morrison <i>K</i><sub>i</sub> = 63 pM), the most potent G9a inhibitor to date.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of <b>3a</b> (BIX01294).<a onclick="showRef(event, 'ref22 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref29">(22, 29)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our initial inhibitor design was based on the structural insights revealed by the X-ray crystal structure of the GLP−<b>3a</b> complex.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Because the benzyl group of <b>3a</b> is outside the peptide binding groove and does not make any interactions (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>),<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> it was proposed that the 1-benzyl piperidin-4-yl-amino group (marked in red in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) of <b>3a</b> could be replaced with a smaller group such as 1-methyl piperidin-4-yl-amino and other smaller amino groups as shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>. This modification would lower the molecular weight, which is an important consideration for cellular activity. Additionally, exploration of the 2-amino region (marked in green in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) with various cyclic and acyclic amines shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> was carried out.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The lipophilic benzyl group (top) is exposed to solvent and does not makes any interactions with the protein in the X-ray crystal structure of the GLP−<b>3a</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FPD">3FPD</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of 4- and 2-Amino Regions of the Quinazoline Template<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div><p class="last"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0004.gif" alt="" id="fx1" /></img></p></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0005.gif" alt="" id="fx2" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">IC<sub>50</sub> values are the average of at least duplicate assay runs with standard deviation (SD) values that are 3-fold less than the average.</p></div></div></div><div class="NLM_p">An efficient two-step synthetic sequence was developed to prepare the compounds in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). Displacing the 4-chloro of commercially available 2,4-dichloro-6,7-dimethoxyquinazoline (<b>1</b>) with the first set of amines at room temperature afforded the intermediates <b>2</b>. The displacement of the 2-chloro of the intermediates <b>2</b> with the second set of amines under microwave heating conditions yielded the desired 2,4-diamino-6,7-dimethoxyquinazolines <b>3</b>. Using this efficient synthesis, the compounds listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> were rapidly prepared in high yields.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 2,4-Diamino-6,7-dimethoxy Quinazolines <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) R′ amines, DMF, DIEA, RT; (b) R′′ amines, <i>i</i>-PrOH, 4 M HCl/dioxane, microwave, 160 °C.</p></p></figure><div class="NLM_p">Newly synthesized compounds were evaluated in two orthogonal and complementary G9a biochemical assays: enzyme-coupled SAH detection (ECSD) and chemiluminescence-based oxygen tunneling (CLOT). The ECSD assay monitors the conversion of cofactor SAM to SAH. This assay technology was previously employed to study <i>Schizosaccharomyces pombe</i> CLR4, a H3K9 specific PKMT.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The CLOT assay detects monomethylation of histone H3 peptide 1−11.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The use of these two orthogonal and complementary biochemical assays in parallel enables us to rule out false positives and other artifacts of one particular assay format and increases our confidence in SAR findings.</div><div class="NLM_p">For the 4-amino moiety of the quinazoline scaffold, replacing the <i>N</i>-(1-benzyl piperidin-4-yl)-amino group (<b>3a</b>) with the <i>N</i>-(1-methyl piperidin-4-yl)-amino group (<b>3b</b>) resulted in little or no potency loss (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). This result confirmed our prediction that the benzyl group of <b>3a</b> could be replaced by a small alkyl group such as methyl in <b>3b</b> based on structural insights revealed by the cocrystal structure of the GLP−<b>3a</b> complex.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> This SAR finding allowed us to reduce the molecular weight while maintaining potency for this chemical series. On the other hand, replacing the <i>N</i>-(1-methyl piperidin-4-yl)-amino group (<b>3b</b>) with the <i>N</i>-(piperidin-4-yl)-amino (<b>3c</b>), <i>N</i>-(tetrahydropyran-4-yl)-amino (<b>3d</b>), or <i>N</i>-cyclohexyl-amino (<b>3e</b>) group led to significant potency loss, indicating that an alkylated nitrogen is important for inhibitory activity. We next explored the preferred substitution pattern (<i>N</i>,<i>N</i>-disubstitution versus <i>N</i>-monosubstitution) of the 4-amino group and whether the nitrogen could be replaced by an oxygen atom. Compound <b>3f</b> that contains a <i>N</i>-(1-methyl piperidin-4-yl),<i>N</i>-methyl-4-amino group was drastically less potent compared with compound <b>3b</b> that has a <i>N</i>-monosubstituted-4-amino group. Similarly, compound <b>3g</b>, an oxygen analogue of compound <b>3b</b>, had no G9a inhibitory activity in both assays. These results together indicate that the secondary nitrogen is important for G9a inhibition, likely via a hydrogen bond interaction. This finding was subsequently confirmed by the cocrystal structure of G9a-<b>10</b> (vide infra). Replacing the piperidine ring in <b>3b</b> with a smaller pyrrolidine ring (<b>3h</b>) led to some potency loss, but compound <b>3h</b><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> still had appreciable G9a inhibitory activity. In addition, analogues containing a smaller amino group such as <i>N</i>-cyclopropyl-4-amino (<b>3i</b>), <i>N</i>-isopropyl-4-amino (<b>3j</b>), and <i>N</i>-methyl-4-amino (<b>3k</b>) were significantly less potent compared with compounds <b>3a</b> and <b>3b</b>.</div><div class="NLM_p">Modifications to the 2-amino region of the quinazoline template were well tolerated in general (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). For example, the methyl homopiperazine group (<b>3b</b>) could be replaced with the methyl piperazine (<b>3l</b>) or piperidine (<b>3m</b>) group without significant potency loss. Analogues containing a morpholine (<b>3n</b>), diethylamine (<b>3o</b>), or dimethylamine (<b>3p</b>) group had moderate potency. The 2-choloro analogue <b>2a</b> had poor potency in both assays. In general, SAR trends from the ECSD and CLOT assays are in good agreement.</div><div class="NLM_p">Having established tractable SAR for the 2- and 4-amino regions, we next explored the 7-methoxy moiety of the quinazoline scaffold, again using structure-based design and synthesis. The X-ray cocrystal structure of the GLP−<b>3a</b> complex revealed that <b>3a</b> occupied the GLP histone peptide binding site but did not interact with the narrow lysine binding channel (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> To take advantage of this nearby lysine binding channel, it was hypothesized that adding a 7-aminoalkoxy side chain to the quinazoline template would make new interactions with the lysine binding channel while the rest of molecule maintained interactions with the peptide binding groove. Thus, compound <b>10</b> (structure shown in Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>), which possesses a 7-dimethylaminopropoxy chain and also combines the best 2- and 4-amino moieties identified previously, was designed and modeled. Replacing GLP and <b>3a</b> in the X-ray cocrystal structure of the GLP−<b>3a</b> complex<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> with G9a and <b>10</b> yielded a model that suggests that <b>10</b> is capable of interacting with the narrow lysine binding channel. This docking model (not shown here) is virtually identical to the G9a-<b>10</b> X-ray cocrystal structure subsequently determined (vide infra).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>3a</b> (gray, red, and blue) does not occupy the lysine binding channel in the X-ray cocrystal structure of the GLP−<b>3a</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FPD">3FPD</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> A fragment of the histone peptide (magenta) was transposed into this crystal structure to illustrate the lysine binding channel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, RT; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to RT; (c) Fe dust, NH<sub>4</sub>Cl, <i>i</i>-PrOH-H<sub>2</sub>O, reflux, 67% over 3 steps; (d) methyl chloroformate, DIEA, DMF-DCM, 0 °C to RT; (e) NaOH, H<sub>2</sub>O<sub>2</sub>, EtOH, reflux, 70% over 2 steps; (f) <i>N</i>,<i>N</i>-diethylaniline, POCl<sub>3</sub>, reflux, 59%; (g) 4-amino-1-methylpiperidine, DIEA, THF, RT; (h) 1-methylhomopiperazine, HCl, <i>i</i>-PrOH, 160 °C, microwave, 82% over 2 steps; (i) Pd/C, 1,4-cyclohexadiene, EtOH, reflux; (j) 3-(dimethylamino)propan-1-ol, PPh<sub>3</sub>, DIAD, THF, 0 °C to RT, 46% over 2 steps.</p></p></figure><div class="NLM_p">To test this hypothesis, compound <b>10</b> was synthesized via the 10-step synthetic sequence outlined in Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. Benzyl protection of commercially available 2-methoxy-4-cyanophenol (<b>4</b>), followed by nitration and subsequent reduction of the nitro group afforded the aniline <b>5</b>, which was then converted to the quinazolinedione <b>6</b> via formation of methyl carbamate and subsequent saponification of the cyano group and ring closure. POCl<sub>3</sub> treatment of the quinazolinedione <b>6</b> afforded the 2,4-dichloro quinazoline <b>7</b>, which underwent two consecutive chloro displacement reactions to yield the 2,4-diamino quinazoline <b>8</b>. Debenzylation of the intermediate <b>8</b>, followed by a Mitsunobu reaction with 3-(dimethylamino)propan-1-ol, produced the desired compound <b>10</b>.</div><div class="NLM_p">In G9a ECSD and CLOT assays, <b>10</b> was about 5-fold more potent compared to <b>3a</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The high potency of <b>10</b> was confirmed in several secondary assays. In ITC experiments that measure the binding affinity of a small molecule to the G9a protein,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a><b>10</b> had about 5-fold higher binding affinity compared to <b>3a</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). In addition, <b>10</b> displaced a fluorescein labeled 15-mer H3 peptide (1−15) with higher efficiency than <b>3a</b> in a G9a fluorescence polarization (FP) assay and stabilized G9a better compared to <b>3a</b> in differential scanning fluorimetry (DSF) experiments.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> These results together suggest that <b>10</b> is a more potent G9a inhibitor compared to <b>3a</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Potency Comparison of <b>10</b> with <b>3a</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="3" align="center" char=".">G9a</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">ECSD IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char=".">CLOT IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char=".">ITC <i>K</i><sub>d</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="char" char=".">180</td><td class="colsep0 rowsep0" align="char" char=".">250</td><td class="colsep0 rowsep0" align="char" char=".">130</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">IC<sub>50</sub> and <i>K</i><sub>d</sub> values are the average of at least duplicate assay runs with SD values that are 3-fold less than the average.</p></div></div></div><div class="NLM_p">A high resolution (1.7 Å) X-ray cocrystal structure of the G9a−<b>10</b> complex was obtained. This cocrystal structure, the first X-ray crystal structure of G9a in complex with a small molecule inhibitor, revealed that the 7-dimethylamino propoxy side chain of <b>10</b> indeed occupied the lysine binding channel of G9a and the basic nitrogen interacted with Tyr1154 and Leu1086 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>), thus validating our binding hypothesis. The higher potency of <b>10</b> compared to <b>3a</b> can be explained by these additional interactions between the 7-dimethylamino propoxy side chain and the lysine binding channel. These interactions were absent in the GLP−<b>3a</b> complex as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. Other key structural insights revealed by the G9a−<b>10</b> cocrystal structure include: (1) the bulk of <b>10</b> occupied the histone peptide binding site, similar to the GLP−<b>3a</b> complex, (2) the hydrogen of the secondary amino group at the 4-position formed a hydrogen bond with Asp1083, providing a potential explanation for the drastic potency loss of compound <b>3f</b> compared to compound <b>3b</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), and (3) the lysine binding channel was not fully occupied by the 7-dimethylamino propoxy group and there is space at the end of the channel to accommodate a longer chain or larger amino group (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystal structure of the G9a−<b>10</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a>). The inhibitor−protein interactions are annotated as follows: “Es”, electrostatic; “Hb”, hydrogen bonding; “⊕π”, cation−π interactions; “(s)”, interaction with a side chain; “(b)”, interaction with the backbone.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. The 7-dimethylamino propoxy side chain (gray, blue, and red) of <b>10</b> does not fully occupy the lysine binding channel in the X-ray crystal structure of the G9a−<b>10</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a>). A fragment of the H3K9 peptide (magenta) was transposed into this crystal structure to illustrate the lysine binding channel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of this observation, the compounds in Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>−<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> were designed and synthesized to extensively explore the SAR of this 7-aminoalkoxy region and optimize potency of this series. Compounds <b>11</b>, <b>14</b>−<b>16</b>, <b>19</b>−<b>23</b>, <b>25</b>−<b>29</b> were synthesized from the phenol <b>9</b> via Mitsunobu reactions with corresponding alcohols (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>), similar to the synthesis of <b>10</b>. Compound <b>24</b> was made via Mitsunobu reaction of the phenol <b>9</b> with the Boc-protected 5-amino-1-pentanol, followed by the deprotection of the Boc group. Compounds <b>12</b>, <b>13</b>, <b>17</b>, and <b>18</b> were synthesized from the chlorides <b>30</b>, which were prepared from the phenol <b>9</b> via Mitsunobu reactions with corresponding chloroalkyl alcohols.</div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>11</b>−<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) various alcohols, PPh<sub>3</sub>, DIAD, THF, 0 °C to RT; (b) TFA, DCM, RT; (c) various amines, <i>i</i>-PrOH, 160 °C, microwave.</p></p></figure><div class="NLM_p">These newly synthesized compounds bearing various 7-alkoxy side chains were evaluated in G9a ECSD and CLOT assays and SAR results are summarized in Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>−<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>. Compounds <b>11</b>, <b>10</b>, <b>12</b>, and <b>13</b> that contain, respectively, a 2−5-carbon chain, had similar potency while the 6-carbon chain containing analogue <b>14</b> was significantly less potent (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), indicating the lysine binding channel of G9a can accommodate a longer aminoalkoxy side chain up to 5 carbons in length. Compounds possessing various amino capping groups were evaluated next (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Compounds <b>15</b>, <b>16</b>, and <b>17</b> containing, respectively, <i>N</i>-methylamino, <i>N</i>,<i>N</i>-diethylamino, and <i>N</i>-methyl,<i>N</i>-propylamino groups, were as potent as <b>10</b>, which has a <i>N</i>,<i>N</i>-dimethylamino group. Similarly, replacing the tertiary amino group of compound <b>13</b> with the primary amino group (compound <b>24</b>) did not result in significant potency loss. Cyclic amino groups were also explored. Although the morpholine <b>20</b> was >10-fold less potent, the pyrrolidine <b>18</b> and piperidine <b>19</b> were more potent compared to their acyclic amine analogues such as <b>16</b>. In particular, the pyrrolidine <b>18</b> with a respective IC<sub>50</sub> of 8 and 11 nM in ECSD and CLOT assays was about 5-fold more potent than <b>10</b>. These results together suggest that modifications to the substituents on the basic nitrogen are well tolerated and a larger amino capping group can be accommodated. These SAR findings are consistent with the structural insights revealed by the cocrystal structure of the G9a−<b>10</b> complex (vide supra).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the Chain Length<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div><p class="last"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0011.gif" alt="" id="fx3" /></img></p></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">G9a IC<sub>50</sub> (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center" char=".">ECSD</th><th class="colsep0 rowsep0" align="center" char=".">CLOT</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">110</td><td class="colsep0 rowsep0" align="char" char=".">120</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">140</td><td class="colsep0 rowsep0" align="char" char=".">110</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">95</td><td class="colsep0 rowsep0" align="char" char=".">49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">1500</td><td class="colsep0 rowsep0" align="char" char=".">3200</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">IC<sub>50</sub> values are the average of at least duplicate assay runs with SD values that are 3-fold less than the average.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the Capping Group<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div><p class="last"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0012.gif" alt="" id="fx4" /></img></p></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0013.gif" alt="" id="fx5" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">IC<sub>50</sub> values are the average of at least duplicate assay runs with SD values that are 3-fold less than the average.</p></div></div></div><div class="NLM_p">The necessity for a basic nitrogen in the aminoalkoxy side chain was investigated next (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Replacing the nitrogen in the 7-dimethylamino propoxy group of <b>10</b> with a methine (<b>21</b>) led to about a 100-fold potency loss. Switching the <i>N</i>,<i>N</i>-dimethylamino group in compound <b>12</b> to the methoxy group in compound <b>22</b> or the <i>N</i>-Boc protected amino group in compound <b>23</b> completely abolished G9a inhibitory activity. Similarly, compound <b>25</b> that contains an amide group was about 100-fold less potent compared to its amine analogue <b>24</b>. Taken together, these results suggest that the basic nitrogen in the aminoalkoxy side chain is critical for retaining high potency for G9a.</div><div class="NLM_p">In addition, it was found that replacing the 5-carbon chain in compound <b>13</b> with an ethoxyethyl chain resulted in compound <b>29</b> (IC<sub>50</sub> = 6 nM (CLOT) and 9 nM (ECSD)), the most potent G9a inhibitor to date (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). Increasing the size of the amino capping group from dimethylamino (<b>29</b>) to pyrrolidine (<b>26</b>) led to >5-fold potency decrease. Compound <b>27</b> that possesses a 2-(<i>N</i>-ethyl,<i>N</i>-methylamino) ethyl chain was also less potent compared to its carbon analogue <b>13</b> and oxygen analogue <b>29</b>. Additionally, conformationally constrained analogues were explored. For example, compound <b>28</b><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> containing a (piperidin-3-yl)methoxy group was significantly less potent compared to its acyclic analogues <b>15</b> and <b>17</b>. In general, the extensive exploration of the 7-aminoalkoxy region of this quinazoline scaffold yielded valuable SAR findings, which are useful for future inhibitor design.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Additional SAR of the 7-Aminoalkoxy Side Chain<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div><p class="last"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0014.gif" alt="" id="fx6" /></img></p></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0015.gif" alt="" id="fx7" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">IC<sub>50</sub> values are the average of at least duplicate assay runs with SD values that are 3-fold less than the average.</p></div></div></div><div class="NLM_p">Because compound <b>29</b> likely reached the IC<sub>50</sub> limits of the G9a ECSD and CLOT assays, a newly developed endoproteinase-coupled microfluidic capillary electrophoresis (MCE) assay<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> that is suitable for mechanism of action and kinetics studies of G9a inhibitors was utilized to determine the Morrison <i>K</i><sub>i</sub>, which is an effective method developed by Morrison and co-workers that uses a quadratic fit for steep dose response data over a narrow compound concentration range for assessing tight binding inhibitors.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a> As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, <b>29</b> had a Morrison <i>K</i><sub>i</sub> of 63 pM and was about 40-fold more potent than <b>10</b> (Morrison <i>K</i><sub>i</sub> = 2.6 nM) and 250-fold more potent than <b>3a</b> (Morrison <i>K</i><sub>i</sub> = 16 nM) in this G9a MCE assay. In addition, the Morrison <i>K</i><sub>i</sub> (2.6 nM) of <b>10</b> matched well with its <i>K</i><sub>i</sub> (1.6 nM),<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> which was previously determined using this G9a MCE assay.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0016.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Morrison <i>K</i><sub>i</sub>s of compounds <b>29</b>, <b>10</b>, and <b>3a</b> in the G9a MCE assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The selectivity of compounds <b>29</b>, <b>10</b>, and <b>3a</b> were evaluated next. As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>, <b>3a</b> was more selective for GLP over G9a in ECSD assays. This result is consistent with the previous finding: <b>3a</b> was about 10-fold selective for GLP over G9a in CLOT assays.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In contrast to <b>3a</b>, <b>29</b> inhibited G9a as well as or better than GLP while <b>10</b> was equipotent versus G9a and GLP in both ECSD and CLOT assays. This different selectivity profile makes <b>29</b> and <b>10</b> potentially useful tool compounds that are complementary to <b>3a</b>. In addition to G9a and GLP, compounds <b>29</b> and <b>10</b> were evaluated in SET7/9 (a H3K4 PKMT), SET8/PreSET7 (a H4K20 PKMT), and PRMT3 (a protein arginine methyltransferase (PRMT)) ECSD assays and found to be inactive (IC<sub>50</sub> > 40 μM) versus these protein lysine and arginine methyltransferases, providing a >1000-fold selectivity window for G9a over these PKMTs and PRMT. <b>29</b> and <b>10</b> were also inactive (IC<sub>50</sub> > 57 μM) in the formaldehyde dehydrogenase-coupled biochemical assay of JMJD2E,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> a Jumonji protein demethylase. Additionally, <b>10</b> was clean (less than 20% inhibition at 1 μM) against a broad range of G-protein coupled receptors, ion channels, and transporters in a 30-target selectivity panel (tested by MDS Pharma Services) with the exception of 82% inhibition of the muscarinic M<sub>2</sub> receptor at 1 μM and 31% inhibition of the histamine H<sub>1</sub> receptor at 1 μM.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Selectivity of Compounds <b>29</b>, <b>10</b>, and <b>3a</b><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">G9a IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">GLP IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">ECSD</th><th class="colsep0 rowsep0" align="center" char=".">CLOT</th><th class="colsep0 rowsep0" align="center" char=".">ECSD</th><th class="colsep0 rowsep0" align="center" char=".">CLOT</th><th class="colsep0 rowsep0" align="center">SET7/9 <br /></br>IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SET8/PreSET7 <br /></br>IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">PRMT3 <br /></br>IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">JMJD2E <br /></br>IC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">9.0</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="left">>40000</td><td class="colsep0 rowsep0" align="left">>40000</td><td class="colsep0 rowsep0" align="left">>40000</td><td class="colsep0 rowsep0" align="left">>57000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="left">>40000</td><td class="colsep0 rowsep0" align="left">>40000</td><td class="colsep0 rowsep0" align="left">>40000</td><td class="colsep0 rowsep0" align="left">>57000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>3a</b></td><td class="colsep0 rowsep0" align="char" char=".">180</td><td class="colsep0 rowsep0" align="char" char=".">250</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">IC<sub>50</sub> values are the average of at least duplicate assay runs with SD values that are 3-fold less than the average. ND: not determined.</p></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51372" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Inhibitor design and synthesis based on the GLP−<b>3a</b> X-ray cocrystal structure in combination with early SAR exploration led to the discovery of <b>10</b>, a 2,4-diamino-7-aminoalkoxy-quinazoline, as a potent and selective inhibitor of PKMT G9a. A high resolution X-ray crystal structure of the G9a−<b>10</b> complex, the first cocrystal structure of G9a with a small molecule inhibitor, was obtained. This high resolution cocrystal structure of the G9a−<b>10</b> complex validated our binding and inhibitor design hypothesis. Using the structural insights revealed by this cocrystal structure, the 7-dimethylaminopropoxy side chain of <b>10</b> was optimized, resulting in the discovery of <b>29</b> (Morrison <i>K</i><sub>i</sub> = 63 pM), the most potent G9a inhibitor to date. Exploration of the 2-amino, 4-amino, and 7-aminoalkoxy regions of this quinazoline scaffold yielded valuable SAR findings, which can be exploited for design of novel inhibitors in the future. Compounds <b>29</b> and <b>10</b> are potentially useful small molecule tools for the biomedical research community to investigate the biology of G9a and its role in chromatin remodeling as well as PTMs of other proteins.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01465" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01465" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry General Procedures</h3><div class="NLM_p last">HPLC spectra of all compounds except <b>3k</b>, <b>10−14</b>, <b>17</b>−<b>20</b>, <b>22</b>, <b>23</b>, <b>25</b>, and <b>27</b>−<b>29</b> were acquired from an Agilent 6110 series system with UV detector set to 254 nm. Samples were injected (5 μL) onto an Agilent Eclipse Plus 4.6 mm × 50 mm, 1.8 μM, C18 column at room temperature. A linear gradient from 10% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min was followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid. The flow rate was 1.0 mL/min. The HPLC spectra of <b>3k</b>, <b>10</b>−<b>14</b>, <b>17</b>−<b>20</b>, <b>22</b>, <b>23</b>, <b>25</b>, and <b>27</b>−<b>29</b> were recorded from an Agilent 6110 series system with UV detector set to 254 nm. Samples were injected (10 μL) onto a Phenomenex Kinetex 2.6 μM, 100 mm × 2.1 mm, C18 column at room temperature. Then they were eluted out with 72% B (MeCN) and A being H<sub>2</sub>O + 0.1% formic acid. The flow rate was 0.3 mL/min. Mass spectra (MS) data were acquired in positive ion mode using an Agilent 6110 single quadrupole mass spectrometer with an electrospray ionization (ESI) source. High-resolution (negative ion) mass spectra (HRMS) were acquired using a Shimadzu LCMS-IT-Tof time-of-flight mass spectrometer. Nuclear magnetic resonance (NMR) spectra were recorded at Varian Mercury spectrometer with 400 MHz for proton (<sup>1</sup>H NMR) and 100 MHz for carbon (<sup>13</sup>C NMR) or Varian 300 MHz for proton (<sup>1</sup>H NMR); chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 220 nm. Samples were injected onto a Phenomenex Luna 75 mm × 30 mm, 5 μΜ, C18 column at room temperature. The flow rate was 30 mL/min. Various linear gradients were used with A being H<sub>2</sub>O + 0.5% TFA and B being MeOH. HPLC was used to establish the purity of targeted compounds. All compounds that were evaluated in biological assays had >95% purity using the HPLC methods described above.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> <i>N</i>-(1-Benzylpiperidin-4-yl)-6,7-dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)quinazolin-4-amine (<b>3a</b>)</h3><div class="NLM_p last">4-Amino-1-benzylpiperidine (6.8 mL, 36.02 mmol) was added to a solution of 2,4-dichloro-6,7-dimethoxyquinazoline (<b>1</b>, 4.22 g, 16.28 mmol) in DMF (40 mL), followed by the addition of <i>N,N</i>-diisopropylethylamine (3 mL, 17.24 mmol), and the resulting mixture was stirred at room temperature for 2 h until TLC showed that the starting material had disappeared. Water was added to the reaction mixture, and the resulting solution was extracted with ethyl acetate. The organic layer was washed with 0.5% acetic acid aqueous solution and brine, dried, and concentrated to give the crude product, which was purified on an ISCO chromatography system using 40 g silica gel column eluting with hexane−ethyl acetate to give 6.2 g of the desired compound as a yellow solid, yield 92%. <sup>1</sup>H NMR (300 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.70 (s, 1H), 7.59−7.42 (m, 5H), 7.11 (s, 1H), 4.47−4.36 (m, 1H), 4.09 (s, 3H), 4.08 (s, 3H), 3.87 (s, 2H), 3.26 (d, <i>J</i> = 12.0 Hz, 2H), 2.61−2.52 (m, 2H), 2.24 (d, <i>J</i> = 12.0 Hz, 2H), 2.02−1.82 (m, 2H). HPLC: 99%, RT 4.112 min. MS (ESI) 413 [M + H]<sup>+</sup>. <i>N</i>-(1-Benzylpiperidin-4-yl)-2-chloro-6,7-dimethoxyquinazolin-4-amine (232 mg, 0.56 mmol) was dissolved in 3 mL of isopropyl alcohol. To this solution was added 1-methylhomopiperazine (0.14 mL, 1.12 mmol) and HCl in dioxane (4.0 M, 0.28 mL, 1.12 mmol). The resulting solution was heated using microwave irradiation to 160 °C for 10 min. After cooling, TLC indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was redissolved in CH<sub>2</sub>Cl<sub>2</sub> and washed with saturated NaHCO<sub>3</sub> solution. The organic layer was dried, concentrated, and purified by ISCO to give 221.2 mg of the desired compound <b>3a</b> as a yellow solid, yield 80%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.36 (s, 1H), 7.32−7.24 (m, 5H), 6.87 (s, 1H), 4.09 (m, 1H), 3.89 (m, 2H), 3.86 (s, 3H), 3.85 (s, 3H), 3.81 (t, <i>J</i> = 4 Hz, 2H), 3.53 (s, 2H), 2.96 (d, <i>J</i> = 12 Hz, 2H), 2.71 (m, 2H), 2.56 (m, 2H), 2.32 (s, 3H), 2.14 (t, <i>J</i> = 12 Hz, 2H), 2.06 (d, <i>J</i> = 12 Hz, 2H), 1.98 (m, 2H), 1.69 (m, 2H). HPLC: 100%, RT 2.044 min. MS (ESI) 491 [M + H]<sup>+</sup>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 6,7-Dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>3b</b>)</h3><div class="NLM_p last">4-Amino-1-methylpiperidine (1.12 mL, 8.92 mmol) was added to a solution of 2,4-dichloro-6,7-dimethoxyquinazoline (<b>1</b>, 1.05 g, 4.05 mmol) in THF (10 mL), followed by the addition of <i>N</i>,<i>N</i>-diisopropylethylamine (1.41 mL, 8.11 mmol), and the resulting mixture was stirred at room temperature for 2 h until the TLC indicated that the reaction completed. The solvent was removed on the rotary evaporation, and the residue was purified by an ISCO chromatography system using 40 g silica gel column eluting with hexane−ethyl acetate to give 1.26 g of the targeted compound <b>2a</b> as a yellow solid, yield 92%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.13 (d, <i>J</i> = 7.4 Hz, 1H), 7.78 (s, 1H), 7.18 (s, 1H), 4.28−4.10 (m, 1H), 4.00 (s, 3H), 4.02 (s, 3H), 2.96 (d, <i>J</i> = 11.5 Hz, 2H), 2.32 (s, 3H), 2.19−1.95 (m, 4H), 1.84−1.71 (m, 2H). HPLC: 100%, RT 4.148 min. MS (ESI) 333 [M + H]<sup>+</sup>. <b>3b</b> (239.5 mg, yield 78%) was prepared as a yellow solid from 2-chloro-6,7-dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>2a</b>, 247.6 mg, 0.74 mmol), 1-methylhomopiperazine (0.18 mL, 1.47 mmol), HCl in dioxane (4.0 M, 0.37 mL, 1.47 mmol), and isopropyl alcohol (3 mL) using the same procedure as <b>3a</b>. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.38 (s, 1H), 6.89 (s, 1H), 4.15−4.07 (m, 1H), 3.93−3.89 (m, 2H), 3.88 (s, 3H), 3.87 (s, 3H), 3.82 (t, <i>J</i> = 6.4 Hz, 2H), 2.94 (d, <i>J</i> = 12.1 Hz, 2H), 2.76−2.70 (m, 2H), 2.60−2.57 (m, 2H), 2.34 (s, 3H), 2.30 (s, 3H), 2.17 (td, <i>J</i> = 12.1, 2.2 Hz, 2H), 2.12−2.04 (m, 2H), 2.04−1.96 (m, 2H), 1.76−1.66 (m, 2H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 160.24, 159.36, 156.07, 147.11, 105.26, 104.84, 104.28, 59.62, 58.19, 56.98, 56.39, 56.12, 46.97, 46.71, 46.63, 46.31, 32.38, 28.43. HPLC: 100%, RT 0.571 min. MS (ESI) 415 [M + H]<sup>+</sup>. HRMS calcd for C<sub>22</sub>H<sub>34</sub>N<sub>6</sub>O<sub>2</sub>: 414.2743. Found: 413.2685 [M − H].</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 6,7-Dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(piperidin-4-yl)quinazolin-4-amine (<b>3c</b>)</h3><div class="NLM_p last"><b>3c</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 81%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.05 (s, 1H), 6.97 (s, 1H), 4.15 (d, <i>J</i> = 13.3 Hz, 2H), 4.00−3.95 (m, 2H), 3.94 (s, 3H), 3.91−3.85 (m, 5H), 3.12−3.06 (m, 2H), 3.01−2.92 (m, 1H), 2.80−2.73 (m, 2H), 2.64−2.57 (m, 2H), 2.38 (s, 3H), 2.10−1.94 (m, 4H), 1.67−1.57 (m, 2H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 166.40, 159.50, 156.35, 152.68, 146.43, 106.55, 106.24, 106.22, 59.70, 58.28, 56.71, 56.41, 50.25, 50.03, 47.02, 46.81, 46.68, 35.56, 28.49. HPLC: 100%, RT 0.670 min. MS (ESI) 401 [M + H]<sup>+</sup>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 6,7-Dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(tetrahydro-<i>2H</i>-pyran-4-yl)quinazolin-4-amine (<b>3d</b>)</h3><div class="NLM_p last"><b>3d</b> was prepared as an off-white solid by the same procedure as <b>3b</b>, yield 84%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.44 (s, 1H), 6.95 (s, 1H), 4.44−4.36 (m, 1H), 4.08−4.05 (m, 2H), 4.00−3.95 (m, 2H), 3.94 (s, 3H), 3.93 (s, 3H), 3.89 (t, <i>J</i> = 6.3 Hz, 2H), 3.60 (t, <i>J</i> = 11.9 Hz, 2H), 2.83−2.77 (m, 2H), 2.69−2.63 (m, 2H), 2.41 (s, 3H), 2.10−2.04 (m, 4H), 1.83−1.72 (m, 2H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 160.15, 160.12, 155.92, 150.15, 146.84, 105.89, 104.89, 104.17, 68.53, 59.83, 58.27, 56.96, 56.32, 46.91, 46.78, 46.70, 34.04, 28.60. HPLC: 100%, RT 1.316 min. MS (ESI) 402 [M + H]<sup>+</sup>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> <i>N</i>-Cyclohexyl-6,7-dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)quinazolin-4-amine (<b>3e</b>)</h3><div class="NLM_p last"><b>3e</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 82%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.38 (s, 1H), 6.89 (s, 1H), 4.12−4.04 (m, 1H), 3.93−3.89 (m, 2H), 3.88 (s, 3H), 3.87 (s, 3H), 3.82 (t, <i>J</i> = 6.4 Hz, 2H), 2.76−2.70 (m, 2H), 2.60−2.54 (m, 2H), 2.33 (s, 3H), 2.10−2.04 (m, 2H), 2.03−1.95 (m, 2H), 1.85−1.81 (m, 2H), 1.75−1.66 (m, 1H), 1.44−1.34 (m, 4H), 1.28−1.21 (m, 1H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 159.97, 159.92, 155.76, 149.54, 146.76, 105.64, 104.94, 104.27, 59.75, 58.16, 56.96, 56.29, 51.77, 46.90, 46.74, 46.71, 33.93, 28.53, 27.17, 26.92. HPLC: 100%, RT 2.841 min. MS (ESI) 400 [M + H]<sup>+</sup>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 6,7-Dimethoxy-<i>N</i>-methyl-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>3f</b>)</h3><div class="NLM_p last"><b>3f</b> was prepared as an off-white solid by the same procedure as <b>3b</b>, yield 80%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.14 (s, 1H), 6.96 (s, 1H), 4.29−4.19 (m, 1H), 3.94 (m, 5H), 3.90−3.83 (m, 5H), 3.15 (s, 3H), 3.02 (d, <i>J</i> = 11.5 Hz, 2H), 2.79−2.71 (m, 2H), 2.64−2.56 (m, 2H), 2.37 (s, 3H), 2.32 (s, 3H), 2.15 (t, <i>J</i> = 11.5 Hz, 2H), 2.10−1.97 (m, 4H), 1.92 (d, <i>J</i> = 10.5 Hz, 2H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 165.57, 159.30, 155.99, 153.00, 145.83, 107.08, 106.21, 106.19, 59.79, 58.70, 58.26, 56.81, 56.55, 56.35, 46.94, 46.80, 46.74, 46.31, 34.55, 29.61, 28.59. HPLC: 100%, RT 0.868 min. MS (ESI) 429 [M + H]<sup>+</sup>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 6,7-Dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)-4-(1-methylpiperidin-4-yloxy)quinazoline (<b>3g</b>)</h3><div class="NLM_p last"><b>3g</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 77%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.25 (s, 1H), 6.99 (s, 1H), 5.38−5.28 (m, 1H), 4.00−3.93 (m, 5H), 3.93−3.86 (m, 5H), 2.88−2.75 (m, 4H), 2.67−2.60 (m, 2H), 2.52−2.42 (m, 2H), 2.40 (s, 3H), 2.37 (s, 3H), 2.23−2.13 (m, 2H), 2.10−1.93 (m, 4H). HPLC: 99%, RT 1.724 min. MS (ESI) 416 [M + H]<sup>+</sup>.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 6,7-Dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpyrrolidin-3-yl)quinazolin-4-amine (<b>3h</b>)</h3><div class="NLM_p last"><b>3h</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 79%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.39 (s, 1H), 6.94 (s, 1H), 4.84−4.74 (m, 1H), 4.00−3.95 (m, 2H), 3.93 (s, 3H), 3.92 (s, 3H), 3.88 (t, <i>J</i> = 6.3 Hz, 2H), 3.00 (dd, <i>J</i> = 9.8, 7.1 Hz, 1H), 2.87−2.75 (m, 3H), 2.72−2.60 (m, 4H), 2.50−2.41 (m, 4H), 2.39 (s, 3H), 2.09−2.01 (m, 2H), 2.01−1.92 (m, 1H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.43, 159.98, 156.01, 149.97, 146.91, 105.84, 104.83, 104.08, 63.66, 59.83, 58.26, 56.95, 56.34, 56.24, 52.25, 46.93, 46.78, 46.65, 42.48, 33.18, 28.57. HPLC: 100%, RT 0.528 min. MS (ESI) 401 [M + H]<sup>+</sup>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>-Cyclopropyl-6,7-dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)quinazolin-4-amine (<b>3i</b>)</h3><div class="NLM_p last"><b>3i</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 79%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.17 (s, 1H), 6.80 (s, 1H), 5.38 (s, 0.34 H, NH), 3.91−3.84 (m, 2H), 3.83−3.71 (m, 8H), 2.85−2.79 (m, 1H), 2.68−2.64 (m, 2H), 2.52−2.48 (m, 2H), 2.25 (s, 3H), 1.97−1.89 (m, 2H), 0.73−0.67 (m, 2H), 0.55−0.50 (m, 2H). HPLC: 100%, RT 2.631 min. MS (ESI) 358 [M + H]<sup>+</sup>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>N</i>-Isopropyl-6,7-dimethoxy-2-(4-methyl-1,4-diazepan-1-yl)quinazolin-4-amine (<b>3j</b>)</h3><div class="NLM_p last"><b>3j</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 76%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.38 (s, 1H), 6.90 (s, 1H), 5.48 (s, 1H, NH), 4.47 (dt, <i>J</i> = 13.1, 6.6 Hz, 1H), 3.97−3.91 (m, 2H), 3.89 (s, 3H), 3.88 (s, 3H), 3.84 (t, <i>J</i> = 6.4 Hz, 2H), 2.76−2.71 (m, 2H), 2.61−2.55 (m, 2H), 2.34 (s, 3H), 2.05−1.97 (m, 2H), 1.31 (d, <i>J</i> = 6.6 Hz, 6H). HPLC: 99%, RT 2.532 min. MS (ESI) 360 [M + H]<sup>+</sup>.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6,7-Dimethoxy-<i>N</i>-methyl-2-(4-methyl-1,4-diazepan-1-yl)quinazolin-4-amine (<b>3k</b>)</h3><div class="NLM_p last"><b>3k</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 78%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.89 (s, 1H), 6.76 (s, 1H), 5.34 (s, 1H), 4.06−3.97 (m, 2H), 3.95−3.83 (m, 8H), 3.10 (d, <i>J</i> = 4.7 Hz, 3H), 2.73−2.67 (m, 2H), 2.60−2.52 (m, 2H), 2.36 (s, 3H), 2.08−1.95 (m, 2H). HPLC: 100%, RT 0.673 min. MS (ESI) 332 [M + H]<sup>+</sup>.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 6,7-Dimethoxy-2-(4-methylpiperazin-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>3l</b>)</h3><div class="NLM_p last"><b>3l</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 78%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.39 (s, 1H), 6.85 (s, 1H), 4.16−4.08 (m, 1H), 3.88 (s, 3H), 3.87 (s, 3H), 3.80 (m, 4H), 2.94 (d, <i>J</i> = 12.1 Hz, 2H), 2.55−2.45 (m, 4H), 2.31 (s, 3H), 2.30 (s, 3H), 2.22−2.16 (m, 2H), 2.11−2.02 (m, 2H), 1.76−1.66 (m, 2H). HPLC: 100%, RT 1.068 min. MS (ESI) 401 [M + H]<sup>+</sup>.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 6,7-Dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-(piperidin-1-yl)quinazolin-4-amine (<b>3m</b>)</h3><div class="NLM_p last"><b>3m</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 80%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.64 (s, 1H), 7.11 (s, 1H), 4.58−4.46 (m, 1H), 3.94 (s, 3H), 3.90 (s, 3H), 3.87−3.81 (m, 4H), 3.63 (d, <i>J</i> = 12.8 Hz, 2H), 3.26−3.19 (m, 1H), 2.89 (s, 3H), 2.34 (d, <i>J</i> = 12.9 Hz, 2H), 2.10−1.96 (m, 2H), 1.82−1.66 (m, 7H). HPLC: 100%, RT 3.896 min. MS (ESI) 386 [M + H]<sup>+</sup>.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 6,7-Dimethoxy-<i>N</i>-(1-methylpiperidin-4-yl)-2-morpholinoquinazolin-4-amine (<b>3n</b>)</h3><div class="NLM_p last"><b>3n</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 81%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.40 (s, 1H), 6.86 (s, 1H), 4.17−4.09 (m, 1H), 3.88 (s, 3H), 3.87 (s, 3H), 3.73 (s, 8H), 2.94 (d, <i>J</i> = 11.8 Hz, 2H), 2.31 (s, 3H), 2.19 (t, <i>J</i> = 12.2 Hz, 2H), 2.10−2.04 (m, 2H), 1.77−1.67 (m, 2H). HPLC: 100%, RT 0.554 min. MS (ESI) 388 [M + H]<sup>+</sup>.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i><sup><i>2</i></sup><i>,N</i><sup><i>2</i></sup>-diethyl-6,7-dimethoxy-<i>N</i><sup><i>4</i></sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>3o</b>)</h3><div class="NLM_p last"><b>3o</b> was prepared as a yellow solid by the same procedure as <b>3b</b>, yield 77%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.39 (s, 1H), 6.92 (s, 1H), 4.20−4.13 (m, 1H), 3.91 (s, 3H), 3.90 (s, 3H), 3.66 (q, <i>J</i> = 7.0 Hz, 4H), 2.97 (d, <i>J</i> = 12.0 Hz, 2H), 2.33 (s, 3H), 2.20 (dd, <i>J</i> = 12.1, 2.3 Hz, 2H), 2.16−2.07 (m, 2H), 1.73 (qd, <i>J</i> = 12.3, 3.7 Hz, 2H), 1.22 (t, <i>J</i> = 7.0 Hz, 6H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 160.28, 159.73, 155.77, 150.28, 146.53, 105.74, 104.73, 104.23, 56.95, 56.27, 46.41, 42.93, 32.63, 14.24. HPLC: 100%, RT 2.711 min. MS (ESI) 374 [M + H]<sup>+</sup>.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 6,7-Dimethoxy-<i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>-dimethyl-<i>N</i><sup>4</sup>-(1-methylpiperidin-4-yl)quinazoline-2,4-diamine (<b>3p</b>)</h3><div class="NLM_p last"><b>3p</b> was prepared as a white solid by the same procedure as <b>3b</b>, yield 76%. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.44 (s, 1H), 6.94 (s, 1H), 4.25−4.17 (m, 1H), 3.94 (s, 3H), 3.92 (s, 3H), 3.21 (s, 6H), 3.00 (d, <i>J</i> = 12.5 Hz, 2H), 2.35 (s, 3H), 2.27−2.20 (m, 2H), 2.19−2.11 (m, 2H), 1.81−1.71 (m, 2H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 161.04, 160.21, 155.91, 149.84, 146.82, 105.65, 104.67, 104.28, 56.99, 56.32, 56.24, 46.41, 37.73, 32.46. HPLC: 99%, RT 2.623 min. MS (ESI) 346 [M + H]<sup>+</sup>.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-Amino-4-benzyloxy-5-methoxybenzonitrile (<b>5</b>)</h3><div class="NLM_p last">To an ice-cooled solution of 4-hydroxy-3-methoxybenzonitrile (3.40 g, 22.8 mmol) in DMF (45 mL) was slowly added potassium carbonate (4.73 g, 34.2 mmol) and then benzyl bromide (3.0 mL, 25.1 mmol). The reaction mixture was stirred overnight at room temperature, and saturated sodium chloride aqueous solution (80 mL) was added. The resulting precipitate was collected, washed with water and dried to provide 4-benzyloxy-3-methoxybenzonitrile as a yellow solid (5.40 g, 93% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.36−7.27 (m, 5H), 7.14 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.02 (d, <i>J</i> = 2.0 Hz, 1H), 6.83 (d, <i>J</i> = 8.4 Hz, 1H), 5.13 (s, 2H), 3.83 (s, 3H). A solution of 4-benzyloxy-3-methoxybenzonitrile (3.50 g, 14.62 mmol) in acetic anhydride (40 mL) was cooled to 0 °C, and 69 wt % nitric acid (3.6 mL, 55 mmol) was added slowly. The reaction mixture was stirred overnight at room temperature and then poured into ice−water. The resulting precipitate was collected, washed with water, and dried to provide 4-benzyloxy-5-methoxy-2-nitrobenzonitrile. A mixture of crude 4-benzyloxy-5-methoxy-2-nitrobenzonitrile, iron dust (3.0 g, 54.8 mmol), and ammonium chloride (4.4 g, 82.2 mmol) in isopropyl alcohol−water (50:30 mL) was heated to reflux for 4 h. Then, the resulting mixture was filtered and washed with 5% methanol in dichloromethane (150 mL). The organic phase of the filtrate was dried, concentrated, and purified by silica gel chromatography (DCM to 10% MeOH in DCM) to afford 2-amino-4-benzyloxy-5-methoxybenzonitrile as a yellow solid (<b>5</b>, 2.68 g, 72% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.43−7.28 (m, 5H), 7.12 (s, 1H), 6.41 (s, 1H), 5.07 (s, 2H), 3.76 (s, 3H).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 7-(Benzyloxy)-6-methoxyquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>6</b>)</h3><div class="NLM_p last">Methyl chloroformate (0.94 mL, 12.1 mmol) was added into the solution of 2-amino-4-benzyloxy-5-methoxybenzonitrile (2.05 g, 8.06 mmol) and DIEA (2.6 mL, 16.1 mmol) in cosolvent DMF:DCM (20:10 mL) at 0 °C. The reaction solution was stirred overnight at room temperature, and then 60 mL of water was added. The resulting precipitate was collected, washed with water, dried, and used in the next step without further purification. A mixture of the product, 30 wt % H<sub>2</sub>O<sub>2</sub> (20 mL), and NaOH (322 mg, 8.06 mmol) in ethanol (100 mL) was stirred for 2 h under reflux. After adding 100 mL of water, the reaction mixture was cooled to RT, and the resulting precipitate was collected, washed with water, and dried to afford compound <b>6</b> as a white solid (1.68 g, 70% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 11.03 (br, s, 2H), 7.48−7.36 (m, 5H), 7.28 (s, 1H), 6.78 (s, 1H), 5.14 (s, 2H), 3.79 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 162.8, 154.2, 150.8, 145.6, 136.8, 136.3, 128.9, 128.6, 128.5, 107.8, 106.8, 99.5, 70.5, 56.2. MS (ESI): 299 [M + H]<sup>+</sup>.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 7-(Benzyloxy)-2,4-dichloro-6-methoxyquinazoline (<b>7</b>)</h3><div class="NLM_p last">A mixture of compound <b>5</b> (1.95 g, 6.54 mmol) and <i>N</i>,<i>N</i>-diethylaniline (1.15 mL, 7.19 mmol) in POCl<sub>3</sub> (10 mL) was stirred for 7 h under reflux. The reaction mixture was concentrated in vacuo, and saturated sodium bicarbonate aqueous solution (20 mL) was added. The resulting mixture was extracted with chloroform (30 mL × 2). The combined organic phases were dried, filtered, concentrated, and purified by silica gel chromatography (hexanes to 20% ethyl acetate in hexanes) to afford compound <b>7</b> as a yellow solid (1.30 g, 59% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.39−7.27 (m, 5H), 7.25 (s, 1H), 7.19 (s, 1H), 5.21 (s, 2H), 3.97 (s, 3H).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 7-(Benzyloxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>8</b>)</h3><div class="NLM_p last">A mixture of compound <b>7</b> (1.30 g, 3.88 mmol), 4-amino-1-methylpiperidine (1.33 g, 11.64 mmol), and DIEA (0.96 mL, 5.82 mmol) in THF (10 mL) was stirred overnight at room temperature. After concentration in vacuo, the crude product was purified by silica gel chromatography (DCM to 10% MeOH in DCM) to afford 7-(benzyloxy)-2-chloro-6-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine as a yellow solid (1.60 g, 100% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.46−7.26 (m, 5H), 7.13 (s, 1H), 6.86 (s, 1H), 5.51 (d, <i>J</i> = 7.9 Hz, 1H), 5.22 (s, 2H), 4.42−4.22 (m, 1H), 3.99 (s, 3H), 2.97 (d, <i>J</i> = 11.9 Hz, 2H), 2.38 (s, 3H), 2.32 (td, <i>J</i> = 11.9, 2.2 Hz, 2H), 2.15 (d, <i>J</i> = 9.6 Hz, 2H), 1.81−1.71 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 159.16, 156.07, 153.95, 149.30, 147.75, 135.48, 128.65, 128.18, 127.18, 108.66, 106.90, 100.25, 70.75, 56.36, 54.57, 47.98, 46.17, 32.11. MS (ESI): 413 [M + H]<sup>+</sup>. Then, a mixture of 7-(benzyloxy)-2-chloro-6-methoxy-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (1.31 g, 3.17 mmol), 1-methylhomopiperazine (0.80 mL, 6.35 mmol), and 4.0 M HCl in dioxane (1.6 mL, 6.4 mmol) in isopropyl alcohol was heated by microwave irradiation for 15 min to 160 °C in a sealed tube. The reaction mixture was diluted with 30 mL of DCM and washed with saturated sodium bicarbonate aqueous solution (10 mL). The organic phase was dried, filtered, concentrated, and purified by silica gel chromatography (DCM to 20% MeOH in DCM) to afford compound <b>8</b> as a yellow solid (1.30 g, 83% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 7.42 (s, 1H), 7.40−7.20 (m, 5H), 6.96 (s, 1H), 5.08 (s, 2H), 4.11−4.03 (m, 1H), 3.90−3.84 (m, 5H), 3.78 (t, <i>J</i> = 6.3 Hz, 2H), 2.93 (d, <i>J</i> = 12.1 Hz, 2H), 2.80−2.68 (m, 2H), 2.58 (dd, <i>J</i> = 6.4, 4.4 Hz, 2H), 2.34 (s, 3H), 2.29 (s, 3H), 2.17 (dd, <i>J</i> = 12.0, 10.1 Hz, 2H), 2.05 (d, <i>J</i> = 10.5 Hz, 2H), 2.00−1.94 (m, 2H), 1.75−1.65 (m, 2H). <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.08, 158.83, 155.09, 147.93, 147.45, 137.98, 129.66, 129.15, 128.68, 106.64, 105.00, 104.60, 71.56, 59.50, 58.15, 57.06, 56.00, 49.47, 46.98, 46.67, 46.53, 46.23, 32.24, 28.30. MS (ESI): 491 [M + H]<sup>+</sup>.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-4-(1-methylpiperidin-4-ylamino)quinazolin-7-ol (<b>9</b>)</h3><div class="NLM_p last">A mixture of compound <b>8</b> (1.29 g, 2.63 mmol), 1,4-cyclohexadiene (2.6 mL, 26 mmol), and 10 wt % Pd/C (400 mg) in ethanol (30 mL) was heated to reflux 2 h. The reaction mixture was filtered and concentrated to provide the debenzylated product <b>9</b> as a yellow solid, which was used without purification. MS (ESI): 401 [M + H]<sup>+</sup>.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 7-(3-(Dimethylamino)propoxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>10</b>)</h3><div class="NLM_p last">Under nitrogen gas, DIAD (250 μL, 1.25 mmol) was added into the ice-cooled mixture of compound <b>9</b> (100 mg, 0.250 mmol), 3-(dimethylamino)propan-1-ol (120 μL, 1.00 mmol), and PPh<sub>3</sub> (362 mg, 1.38 mmol) in THF (3.0 mL). The reaction mixture was stirred overnight at RT. After concentration in vacuo, the crude product was purified by preparative HPLC with a gradient from 10% of MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% of MeOH (A). The resulting product was basified with a saturated solution of NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub> to afford 56 mg of the desired product <b>10</b> as a white solid (46% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 6.87 (s, 1H), 6.71 (s, 1H), 4.96 (d, <i>J</i> = 7.0 Hz, 1H), 4.11 (t, <i>J</i> = 6.8 Hz, 2H), 4.08−3.98 (m, 1H), 3.94 (dd, <i>J</i> = 5.6, 4.0 Hz, 2H), 3.87 (s, 3H), 3.84 (t, <i>J</i> = 6.4 Hz, 2H), 2.83 (d, <i>J</i> = 11.8 Hz, 2H), 2.72−2.63 (m, 2H), 2.59−2.51 (m, 2H), 2.47−2.37 (m, 2H), 2.34 (s, 3H), 2.29 (s, 3H), 2.20 (s, 6H), 2.10−2.17 (m, 4H), 2.08−1.94 (m, 4H), 1.68−1.49 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.52, 157.99, 153.91, 149.58, 145.15, 106.95, 102.67, 101.44, 67.14, 58.90, 57.30, 56.65, 56.25, 54.75, 47.87, 46.66, 46.25, 45.83, 45.43, 32.25, 27.81, 27.20. MS (ESI): 486 [M + H]<sup>+</sup>. HPLC: 100%; RT: 0.765 min. HRMS calcd for C<sub>26</sub>H<sub>43</sub>N<sub>7</sub>O<sub>2</sub>: 485.3478. Found: 484.3410 [M − H].</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 7-(2-(Dimethylamino)ethoxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>11</b>)</h3><div class="NLM_p last">Compound <b>11</b> was prepared as a white solid from phenol <b>9</b> and 2-(dimethylamino)ethanol via Mitsunobu reaction (43% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 6.81 (s, 1H), 6.72 (s, 1H), 5.05 (d, <i>J</i> = 6.8 Hz, 1H), 4.09 (t, <i>J</i> = 6.0 Hz, 2H), 3.95−4.05 (m, 1H), 3.94−3.86 (m, 2H), 3.82−3.80 (m, 5H), 2.78 (d, <i>J</i> = 11.2 Hz, 2H), 2.72 (t, <i>J</i> = 6.0 Hz, 2H), 2.62 (t, <i>J</i> = 4.3 Hz, 2H), 2.53−2.45 (m, 2H), 2.29 (s, 3H), 2.24 (s, 9H), 2.08 (t, <i>J</i> = 9.3 Hz, 4H), 1.94 (dd, <i>J</i> = 10.5, 5.5 Hz, 2H), 1.53 (dd, <i>J</i> = 21.4, 10.6 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.58, 157.99, 153.65, 149.54, 145.07, 106.79, 102.82, 101.34, 66.57, 58.94, 57.83, 57.34, 56.38, 54.77, 47.96, 46.71, 46.26, 45.86, 45.84, 32.23, 27.89. HPLC: 99%, RT: 0.713 min. MS (ESI): 472 [M + H]<sup>+</sup>.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 7-(4-(Dimethylamino)butoxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>12</b>)</h3><div class="NLM_p last">Intermediate <b>30</b> was prepared from phenol <b>9</b> (117 mg, 0.292 mmol) and 4-chlorobutan-1-ol (127 mg, 1.17 mmol) and purified by basic alumina chromatography (DCM to 5% MeOH in DCM) to afford the crude product (95 mg). Then a mixture of <b>30</b>, dimethylamine (0.8 mL) and <i>i</i>-PrOH (2.5 mL) was stirred for 15 min at 160 °C in a microwave reactor. After purification (same as <b>10</b>), 17 mg of compound <b>12</b> was obtained as a yellow solid (12% yield over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 6.80 (s, 1H), 6.65 (s, 1H), 4.87 (d, <i>J</i> = 6.9 Hz, 1H), 3.97−4.06 (m, 3H), 3.93−3.76 (m, 7H), 2.80 (d, <i>J</i> = 11.0 Hz, 2H), 2.68−2.58 (m, 2H), 2.55−2.45 (m, 2H), 2.31 (s, 3H), 2.28−2.22 (m, 5H), 2.16 (s, 6H), 2.10 (t, <i>J</i> = 11.1 Hz, 4H), 1.95 (dd, <i>J</i> = 14.9, 9.8 Hz, 2H), 1.89−1.78 (m, 2H), 1.65−1.48 (m, 4H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.59, 158.00, 154.01, 149.65, 145.23, 106.86, 102.64, 101.47, 68.47, 59.37, 58.97, 57.35, 56.70, 54.78, 47.88, 46.71, 46.30, 45.87, 45.83, 45.47, 32.32, 27.88, 26.83, 24.23. HPLC: 100%; RT: 0.762 min. MS (ESI): 500 [M + H]<sup>+</sup>.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 7-(5-(Dimethylamino)pentyloxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>13</b>)</h3><div class="NLM_p last">Compound <b>13</b> was prepared as an off-white solid by the procedure as the preparation of compound <b>12</b> (28% yield over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.82 (s, 1H), 6.70 (s, 1H), 4.96 (d, <i>J</i> = 6.9 Hz, 1H), 4.04 (m, 3H), 3.97−3.90 (m, 2H), 3.89−3.78 (m, 5H), 2.82 (d, <i>J</i> = 11.3 Hz, 2H), 2.70−2.61 (m, 2H), 2.57−2.48 (m, 2H), 2.33 (s, 3H), 2.28 (s, 3H), 2.25−2.20 (m, 2H), 2.17 (s, 6H), 2.12 (t, <i>J</i> = 10.2 Hz, 4H), 1.97 (dd, <i>J</i> = 10.9, 5.5 Hz, 2H), 1.91−1.81 (m, 2H), 1.65−1.37 (m, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.59, 157.99, 153.95, 149.64, 145.16, 106.79, 102.63, 101.36, 68.60, 59.66, 58.96, 57.35, 56.60, 54.78, 47.90, 46.72, 46.29, 45.87, 45.84, 45.49, 32.29, 28.85, 27.89, 27.44, 23.88. HPLC: 100%; RT: 0.762 min. MS (ESI): 514 [M + H]<sup>+</sup>.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 7-(6-(Dimethylamino)ethoxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>14</b>)</h3><div class="NLM_p last">Compound <b>14</b> was prepared as a white solid from phenol <b>9</b> and 6-(dimethylamino)hexan-1-ol via Mitsunobu reaction (39% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.81 (s, 1H), 6.71 (s, 1H), 4.99 (d, <i>J</i> = 6.9 Hz, 1H), 4.02 (m, 3H), 3.96−3.88 (m, 2H), 3.90−3.76 (m, 5H), 2.81 (d, <i>J</i> = 11.3 Hz, 2H), 2.69−2.60 (m, 2H), 2.57−2.45 (m, 2H), 2.32 (s, 3H), 2.26 (s, 3H), 2.23−2.03 (m, 12H), 1.96 (dd, <i>J</i> = 10.7, 5.5 Hz, 2H), 1.83 (dt, <i>J</i> = 14.2, 6.9 Hz, 2H), 1.56 (dd, <i>J</i> = 21.3, 10.7 Hz, 2H), 1.46−1.39 (m, 4H), 1.36−1.25 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.57, 157.98, 153.94, 149.63, 145.13, 106.79, 102.61, 101.40, 68.63, 59.75, 58.94, 57.33, 56.58, 54.76, 47.90, 46.70, 46.27, 45.84, 45.47, 32.26, 28.83, 27.87, 27.61, 27.17, 25.91. HPLC: 100%, RT: 0.758 min. MS (ESI): 528 [M + H]<sup>+</sup>.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-7-(3-(methylamino)propoxy)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>15</b>)</h3><div class="NLM_p last">Compound <b>15</b> was prepared as an off-white solid from phenol <b>9</b> and 3-(methylamino)propan-1-ol via Mitsunobu reaction (33% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.43 (s, 1H), 6.92 (s, 1H), 4.22−4.10 (m, 3H), 3.98−3.94 (m, 2H), 3.93 (s, 3H), 3.87 (t, <i>J</i> = 6.3 Hz, 2H), 2.99 (d, <i>J</i> = 11.8 Hz, 2H), 2.86 (t, <i>J</i> = 6.8 Hz, 2H), 2.80−2.74 (m, 2H), 2.65−2.60 (m, 2H), 2.47 (d, <i>J</i> = 0.9 Hz, 3H), 2.39 (s, 3H), 2.35 (s, 3H), 2.22 (t, <i>J</i> = 11.8 Hz, 2H), 2.12−2.02 (m, 6H), 1.76 (dd, <i>J</i> = 21.1, 12.0 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 160.27, 160.16, 155.10, 150.02, 147.01, 106.80, 105.04, 104.28, 68.62, 59.84, 58.25, 57.02, 56.22, 50.20, 46.90, 46.79, 46.72, 46.41, 35.99, 32.52, 29.43, 28.61. HPLC: 99%, RT 0.513 min. MS (ESI): 472 [M + H]<sup>+</sup>.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 7-(3-(Diethylamino)propoxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>16</b>)</h3><div class="NLM_p last">Compound <b>16</b> was prepared as a yellow solid from phenol <b>9</b> and 3-(diethylamino)propan-1-ol via Mitsunobu reaction (47% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.42 (s, 1H), 6.92 (s, 1H), 4.19−4.11 (m, 3H), 3.99−3.93 (m, 2H), 3.91 (s, 3H), 3.87 (t, <i>J</i> = 6.3 Hz, 2H), 2.99 (d, <i>J</i> = 12.0 Hz, 2H), 2.80−2.70 (m, 4H), 2.67−2.59 (m, 6H), 2.39 (s, 3H), 2.35 (s, 3H), 2.21 (t, <i>J</i> = 12.0 Hz, 2H), 2.13 (d, <i>J</i> = 11.9 Hz, 2H), 2.08−1.98 (m, 4H), 1.74 (dt, <i>J</i> = 12.6, 8.2 Hz, 2H), 1.11 (t, <i>J</i> = 7.0 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 160.26, 160.16, 155.30, 150.07, 147.09, 106.84, 104.96, 104.46, 68.11, 59.85, 58.25, 57.06, 56.22, 50.41, 48.02, 46.91, 46.79, 46.72, 46.41, 32.53, 28.61, 26.93, 11.56. HPLC: 99%, RT 0.525 min. MS (ESI): 514 [M + H]<sup>+</sup>.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 6-Methoxy-7-(3-(methyl(propyl)amino)propoxy)-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>17</b>)</h3><div class="NLM_p last">Compound <b>17</b> was prepared as a yellow solid from phenol <b>9</b> by the same general procedure as described for the preparation of compound <b>12</b> (50% yield over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.86 (s, 1H), 6.70 (s, 1H), 4.93 (d, <i>J</i> = 6.9 Hz, 1H), 4.11 (t, <i>J</i> = 6.7 Hz, 2H), 4.08−3.99 (m, 1H), 3.99−3.90 (m, 2H), 3.90−3.78 (m, 5H), 2.84 (d, <i>J</i> = 11.5 Hz, 2H), 2.70−2.63 (m, 2H), 2.58−2.51 (m, 2H), 2.48 (t, <i>J</i> = 7.1 Hz, 2H), 2.34 (s, 3H), 2.31−2.23 (m, 5H), 2.19 (s, 3H), 2.13 (t, <i>J</i> = 10.1 Hz, 4H), 2.06−1.93 (m, 4H), 1.59 (dd, <i>J</i> = 21.1, 11.0 Hz, 2H), 1.45 (dq, <i>J</i> = 14.8, 7.4 Hz, 2H), 0.85 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.57, 158.00, 154.01, 149.66, 145.19, 106.97, 102.66, 101.47, 67.24, 59.72, 58.96, 57.35, 56.69, 54.78, 54.15, 47.90, 46.71, 46.30, 45.86, 42.22, 32.31, 27.87, 26.84, 20.46, 11.90. HPLC: 100%; RT: 0.759 min. MS (ESI): 514 [M + H]<sup>+</sup>.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)-7-(3-(pyrrolidin-1-yl)propoxy)quinazolin-4-amine (<b>18</b>)</h3><div class="NLM_p last">Compound <b>18</b> was prepared as a white solid from phenol <b>9</b> by the same general procedure as described for the preparation of compound <b>12</b> (48% yield over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.71 (s, 1H), 4.95 (d, <i>J</i> = 6.6 Hz, 1H), 4.14 (t, <i>J</i> = 6.7 Hz, 2H), 4.10−3.99 (m, 1H), 3.98−3.90 (m, 2H), 3.90−3.79 (m, 5H), 2.84 (d, <i>J</i> = 10.5 Hz, 2H), 2.66 (d, <i>J</i> = 3.9 Hz, 2H), 2.64−2.42 (m, 8H), 2.34 (s, 3H), 2.29 (s, 3H), 2.21−2.02 (m, 6H), 2.03−1.91 (m, 2H), 1.83−1.69 (m, 4H), 1.59 (dd, <i>J</i> = 21.6, 11.1 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.53, 157.99, 153.96, 149.61, 145.15, 106.95, 102.65, 101.47, 67.28, 58.93, 57.32, 56.66, 54.76, 54.14, 52.91, 47.88, 46.69, 46.27, 45.84, 32.28, 28.50, 27.84, 23.44. HPLC: 100%; RT: 0.761 min. MS (ESI): 512 [M + H]<sup>+</sup>.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)-7-(3-(piperidin-1-yl)propoxy)quinazolin-4-amine (<b>19</b>)</h3><div class="NLM_p last">Compound <b>19</b> was prepared as a white solid from phenol <b>9</b> and 1-piperidinepropanol via Mitsunobu reaction (36% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.87 (s, 1H), 6.69 (s, 1H), 4.92 (d, <i>J</i> = 6.9 Hz, 1H), 4.12 (t, <i>J</i> = 6.6 Hz, 2H), 4.05 (dd, <i>J</i> = 14.2, 8.5 Hz, 1H), 3.99−3.91 (m, 2H), 3.91−3.79 (m, 5H), 2.84 (d, <i>J</i> = 11.4 Hz, 2H), 2.72−2.63 (m, 2H), 2.60−2.50 (m, 2H), 2.44 (t, <i>J</i> = 7.1 Hz, 2H), 2.35 (m, 7H), 2.30 (s, 3H), 2.14 (t, <i>J</i> = 10.0 Hz, 4H), 2.08−1.92 (m, 4H), 1.66−1.50 (m, 6H), 1.47−1.35 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.57, 157.98, 154.00, 149.65, 145.18, 106.96, 102.63, 101.46, 67.41, 58.95, 57.34, 56.69, 55.76, 54.76, 54.56, 47.86, 46.70, 46.28, 45.85, 45.83, 32.30, 27.86, 26.48, 25.98, 24.44. HPLC: 99%; RT: 0.713 min. MS (ESI): 526 [M + H]<sup>+</sup>.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)-7-(3-morpholinopropoxy)quinazolin-4-amine (<b>20</b>)</h3><div class="NLM_p last">Compound <b>20</b> was prepared as a white solid from phenol <b>9</b> and 3-morpholinopropan-1-ol via Mitsunobu reaction (38% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.88 (s, 1H), 6.69 (s, 1H), 4.91 (d, <i>J</i> = 6.7 Hz, 1H), 4.15 (t, <i>J</i> = 6.6 Hz, 2H), 4.09−4.02 (m, 1H), 3.94 (d, <i>J</i> = 4.1 Hz, 2H), 3.89 (s, 3H), 3.85 (t, <i>J</i> = 6.3 Hz, 2H), 3.75−3.64 (m, 4H), 2.84 (d, <i>J</i> = 10.8 Hz, 2H), 2.67 (d, <i>J</i> = 3.9 Hz, 2H), 2.58−2.37 (m, 8H), 2.33 (d, <i>J</i> = 13.6 Hz, 3H), 2.30 (s, 3H), 2.13 (d, <i>J</i> = 10.4 Hz, 4H), 2.03 (td, <i>J</i> = 13.9, 6.1 Hz, 4H), 1.59 (dd, <i>J</i> = 21.4, 10.4 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 157.63, 157.00, 152.91, 148.69, 144.15, 106.00, 101.70, 100.38, 66.01, 57.97, 56.38, 55.66, 54.35, 53.79, 52.70, 46.92, 45.74, 45.31, 44.89, 31.34, 28.68, 26.91, 25.10. HPLC: 99%; RT: 0.762 min. MS (ESI): 528 [M + H]<sup>+</sup>.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-7-(4-methylpentyloxy)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>21</b>)</h3><div class="NLM_p last">Compound <b>21</b> was prepared as a white solid from phenol <b>9</b> and 4-methylpentan-1-ol via Mitsunobu reaction (45% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.41 (s, 1H), 6.89 (s, 1H), 4.12 (ddd, <i>J</i> = 11.4, 9.8, 3.8 Hz, 1H), 4.04 (t, <i>J</i> = 6.5 Hz, 2H), 3.96−3.88 (m, 5H), 3.84 (t, <i>J</i> = 6.3 Hz, 2H), 2.95 (d, <i>J</i> = 11.6 Hz, 2H), 2.78−2.70 (m, 2H), 2.63−2.55 (m, 2H), 2.36 (s, 3H), 2.32 (s, 3H), 2.18 (t, <i>J</i> = 11.9 Hz, 2H), 2.10 (d, <i>J</i> = 12.0 Hz, 2H), 2.05−1.97 (m, 2H), 1.88−1.79 (m, 2H), 1.78−1.66 (m, 2H), 1.61 (td, <i>J</i> = 13.2, 6.6 Hz, 1H), 1.41−1.32 (m, 2H), 0.93 (d, <i>J</i> = 6.6 Hz, 6H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 160.21, 160.03, 155.44, 149.97, 147.07, 106.63, 104.82, 104.43, 70.14, 59.85, 58.24, 57.05, 56.20, 46.92, 46.78, 46.69, 46.41, 36.46, 32.51, 29.21, 28.59, 28.19, 23.12. HPLC: 99%; RT 2.570 min. MS (ESI): 485 [M + H]<sup>+</sup>.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 6-Methoxy-7-(4-methoxybutoxy)-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>22</b>)</h3><div class="NLM_p last">Compound <b>22</b> was prepared as an off-white solid from phenol <b>9</b> and 4-methoxybutan-1-ol via Mitsunobu reaction (38% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.84 (s, 1H), 6.71 (s, 1H), 4.96 (d, <i>J</i> = 7.0 Hz, 1H), 4.12−3.99 (m, 3H), 3.93 (dd, <i>J</i> = 5.7, 4.0 Hz, 2H), 3.86 (s, 3H), 3.84 (t, <i>J</i> = 6.4 Hz, 2H), 3.40 (t, <i>J</i> = 6.4 Hz, 2H), 3.30 (s, 3H), 2.83 (d, <i>J</i> = 11.8 Hz, 2H), 2.69−2.63 (m, 2H), 2.53 (dd, <i>J</i> = 6.5, 4.4 Hz, 2H), 2.33 (s, 3H), 2.28 (s, 3H), 2.20−2.07 (m, 4H), 2.04−1.86 (m, 4H), 1.79−1.67 (m, 2H), 1.66−1.50 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.58, 157.99, 153.92, 149.62, 145.18, 106.82, 102.65, 101.46, 72.24, 68.34, 58.94, 58.52, 57.34, 56.63, 54.77, 47.88, 46.70, 46.27, 45.85, 45.82, 32.27, 27.87, 26.15, 25.67. HPLC: 99%; RT 0.700 min. MS (ESI): 487 [M + H]<sup>+</sup>.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>tert</i>-Butyl 4-(6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-4-(1-methylpiperidin-4-ylamino)quinazolin-7-yloxy)butylcarbamate (<b>23</b>)</h3><div class="NLM_p last">Compound <b>23</b> was prepared as a white solid from phenol <b>9</b> and <i>tert</i>-butyl 4-hydroxybutylcarbamate via Mitsunobu reaction (45% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.82 (s, 1H), 6.69 (s, 1H), 4.93−4.82 (m, 2H), 4.15−4.00 (m, 3H), 3.96−3.93 (m, 2H), 3.91 (s, 3H), 3.85 (t, <i>J</i> = 6.1 Hz, 2H), 3.23−3.10 (m, 2H), 2.85 (d, <i>J</i> = 11.4 Hz, 2H), 2.72−2.62 (m, 2H), 2.61−2.51 (m, 2H), 2.35 (s, 3H), 2.31 (s, 3H), 2.15 (t, <i>J</i> = 9.5 Hz, 4H), 2.04−1.84 (m, 4H), 1.72−1.53 (m, 4H), 1.43 (s, 9H). HPLC: 99%, RT 0.703 min. MS (ESI): 572 [M + H]<sup>+</sup>.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 7-(5-Aminopentyloxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>24</b>)</h3><div class="NLM_p last">Compound <b>24</b> was prepared as a white solid from phenol <b>9</b> and <i>tert</i>-butyl 5-hydroxypentylcarbamate via Mitsunobu reaction, followed by the deprotection of the Boc group by TFA (34% yield over three steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.82 (s, 1H), 6.71 (s, 1H), 4.09−3.97 (m, 3H), 3.92 (d, <i>J</i> = 4.1 Hz, 2H), 3.89−3.77 (m, 5H), 2.82 (d, <i>J</i> = 10.9 Hz, 2H), 2.64 (m, 4H), 2.51 (d, <i>J</i> = 5.0 Hz, 2H), 2.32 (s, 3H), 2.27 (s, 3H), 2.16−2.09 (m, 4H), 2.03−1.90 (m, 2H), 1.90−1.79 (m, 2H), 1.57 (dd, <i>J</i> = 21.3, 10.3 Hz, 2H), 1.45 (m, 4H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.60, 157.94, 153.88, 149.63, 145.12, 106.79, 102.61, 101.39, 68.54, 58.94, 57.35, 56.57, 54.77, 47.79, 46.71, 46.27, 45.85, 41.82, 33.43, 32.23, 28.72, 27.88, 23.28. HPLC: 99%; RT 0.539 min. MS (ESI): 486 [M + H]<sup>+</sup>.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 5-(6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-4-(1-methylpiperidin-4-ylamino)quinazolin-7-yloxy)pentanamide (<b>25</b>)</h3><div class="NLM_p last">Compound <b>25</b> was prepared as a yellow solid from phenol <b>9</b> and 5-hydroxypentanamide via Mitsunobu reaction (17% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.40 (s, 1H), 6.89 (s, 1H), 4.13 (m, 3H), 3.95−3.90 (m, 2H), 3.89 (s, 3H), 3.84 (t, <i>J</i> = 6.4 Hz, 2H), 2.96 (d, <i>J</i> = 12.0 Hz, 2H), 2.79−2.70 (m, 2H), 2.60 (dd, <i>J</i> = 6.5, 4.4 Hz, 2H), 2.36 (s, 3H), 2.34−2.29 (m, 5H), 2.25−2.14 (m, 2H), 2.10 (d, <i>J</i> = 12.2 Hz, 2H), 2.06−1.97 (m, 2H), 1.95−1.66 (m, 6H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 177.43, 158.64, 158.50, 153.69, 148.40, 145.39, 105.05, 103.29, 102.75, 68.03, 58.25, 56.65, 55.40, 54.61, 45.32, 45.19, 45.11, 44.81, 34.68, 30.92, 27.91, 27.00, 22.41. HPLC: 99%; RT 0.692 min. MS (ESI): 500 [M + H]<sup>+</sup>.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)-7-(2-(2-(pyrrolidin-1-yl)ethoxy)ethoxy)quinazolin-4-amine (<b>26</b>)</h3><div class="NLM_p last">Compound <b>26</b> was prepared as a yellow solid from phenol <b>9</b> and 2-(2-(pyrrolidin-1-yl)ethoxy)ethanol via Mitsunobu reaction (49% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.83 (s, 1H), 6.73 (s, 1H), 5.06 (d, <i>J</i> = 6.9 Hz, 1H), 4.19 (t, <i>J</i> = 5.0 Hz, 2H), 4.09−3.96 (m, 1H), 3.95−3.88 (m, 2H), 3.88−3.76 (m, 7H), 3.62 (t, <i>J</i> = 6.0 Hz, 2H), 2.81 (d, <i>J</i> = 11.5 Hz, 2H), 2.63 (t, <i>J</i> = 5.9 Hz, 4H), 2.53−2.47 (m, 6H), 2.31 (s, 3H), 2.26 (s, 3H), 2.10 (t, <i>J</i> = 10.1 Hz, 4H), 2.01−1.92 (m, 2H), 1.78−1.65 (m, 4H), 1.64−1.50 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.51, 158.01, 153.64, 149.40, 145.12, 106.87, 102.93, 101.58, 70.41, 69.00, 67.85, 58.90, 57.31, 56.47, 55.50, 54.77, 54.53, 47.94, 46.67, 46.23, 45.84, 45.78, 32.19, 27.82, 23.38. HPLC: 99%; RT 0.549 min. MS (ESI): 542 [M + H]<sup>+</sup>.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <i>N</i><sup><i>1</i></sup>-(2-(6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-4-(1-methylpiperidin-4-ylamino)quinazolin-7-yloxy)ethyl)-<i>N</i><sup><i>1</i></sup><i>,N</i><sup><i>2</i></sup><i>,N</i><sup><i>2</i></sup>-trimethylethane-1,2-diamine (<b>27</b>)</h3><div class="NLM_p last">Compound <b>27</b> was prepared as a yellow solid from phenol <b>9</b> and 2-((2-(dimethylamino)ethyl)(methyl)amino)ethanol via Mitsunobu reaction (29% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.86 (s, 1H), 6.68 (s, 1H), 4.91 (d, <i>J</i> = 6.7 Hz, 1H), 4.18 (t, <i>J</i> = 6.1 Hz, 2H), 4.12−3.99 (m, 1H), 3.95 (d, <i>J</i> = 4.3 Hz, 2H), 3.88−3.84 (m, 5H), 2.92 (t, <i>J</i> = 6.1 Hz, 2H), 2.85 (d, <i>J</i> = 11.1 Hz, 2H), 2.72−2.63 (m, 2H), 2.62−2.53 (m, 4H), 2.42 (t, <i>J</i> = 6.9 Hz, 2H), 2.35 (s, 6H), 2.30 (s, 3H), 2.23 (s, 6H), 2.15 (t, <i>J</i> = 10.2 Hz, 4H), 2.04−1.90 (m, 2H), 1.60 (dd, <i>J</i> = 21.1, 10.6 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.50, 157.97, 153.78, 149.48, 145.20, 106.83, 102.74, 101.23, 66.59, 58.91, 57.35, 57.31, 56.54, 56.20, 55.93, 54.75, 47.88, 46.67, 46.27, 45.82, 45.79, 45.76, 43.09, 32.29, 27.79. HPLC: 99%; RT 0.727 min. MS (ESI): 529 [M + H]<sup>+</sup>.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 6-Methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)-7-(piperidin-3-ylmethoxy)quinazolin-4-amine (<b>28</b>)</h3><div class="NLM_p last">Compound <b>28</b> was prepared as a white solid from phenol <b>9</b> and 3-piperidinemethanol via Mitsunobu reaction (40% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.81 (s, 1H), 6.71 (s, 1H), 4.94 (d, <i>J</i> = 7.0 Hz, 1H), 4.08−3.99 (m, 1H), 3.97−3.90 (m, 2H), 3.90−3.78 (m, 7H), 3.23 (dd, <i>J</i> = 12.0, 2.7 Hz, 1H), 2.98 (d, <i>J</i> = 12.0 Hz, 1H), 2.82 (d, <i>J</i> = 11.6 Hz, 2H), 2.72−2.62 (m, 2H), 2.61−2.48 (m, 3H), 2.48−2.36 (m, 1H), 2.33 (s, 3H), 2.28 (s, 3H), 2.21−2.01 (m, 5H), 2.01−1.93 (m, 2H), 1.89 (d, <i>J</i> = 9.5 Hz, 1H), 1.81 (br, 1H), 1.70−1.38 (m, 4H), 1.20 (ddd, <i>J</i> = 24.4, 12.1, 3.8 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.59, 157.99, 154.14, 149.70, 145.24, 106.98, 102.70, 101.96, 71.77, 58.96, 57.35, 56.89, 54.77, 50.12, 47.90, 46.98, 46.73, 46.31, 45.87, 36.77, 32.31, 27.99, 27.90, 26.03. HPLC: 99%; RT 0.763 min. MS (ESI): 498 [M + H]<sup>+</sup>.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 7-(2-(2-(Dimethylamino)ethoxy)ethoxy)-6-methoxy-2-(4-methyl-1,4-diazepan-1-yl)-<i>N</i>-(1-methylpiperidin-4-yl)quinazolin-4-amine (<b>29</b>)</h3><div class="NLM_p last">Compound <b>29</b> was prepared as an off-white solid from phenol <b>9</b> and 2-(2-(dimethylamino)ethoxy)ethanol via Mitsunobu reaction (44% yield over 2 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.80 (s, 1H), 6.72 (s, 1H), 5.02 (d, <i>J</i> = 6.9 Hz, 1H), 4.17 (t, <i>J</i> = 5.0 Hz, 2H), 4.01 (qd, <i>J</i> = 13.7, 8.3 Hz, 1H), 3.93−3.86 (m, 2H), 3.85−3.74 (m, 7H), 3.56 (t, <i>J</i> = 5.8 Hz, 2H), 2.79 (d, <i>J</i> = 11.2 Hz, 2H), 2.67−2.57 (m, 2H), 2.53−2.45 (m, 2H), 2.43 (t, <i>J</i> = 5.8 Hz, 2H), 2.30 (s, 3H), 2.24 (s, 3H), 2.18 (s, 6H), 2.08 (t, <i>J</i> = 9.0 Hz, 4H), 1.96−1.88 (m, 2H), 1.53 (dd, <i>J</i> = 21.4, 10.4 Hz, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 158.57, 157.98, 153.62, 149.54, 145.06, 106.95, 102.92, 101.59, 69.50, 68.99, 67.87, 58.93, 58.74, 57.33, 56.47, 54.77, 47.94, 46.71, 46.27, 45.84, 45.81, 32.23, 27.88. HPLC: 100%; RT 0.759 min. MS (ESI): 516 [M + H]<sup>+</sup>.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Biochemical Assays General Procedures</h3><div class="NLM_p last">The G9a enzyme used in the following assays was cloned and purified in Structural Genomics Consortium (SGC) [Mammalian Gene Collection; MGC AU80-H7 (BC018718)]. The construct amino acid sequence is from 913 to 1193.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> ECSD Assay Experimental Procedures</h3><div class="NLM_p last">The methyltransferase activity of PKMTs were measured using a coupled assay.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> In this assay SAHH (<i>S</i>-adenosylhomocysteine hydrolase) and adenosine deaminase convert the methyltransferase reaction product SAH to homocysteine and inosine. The abundance of homocysteine can be quantified using ThioGlo (Calbiochem), which reacts with thiols and fluoresces strongly. The substrate peptide used in this assay was the first 25 residues of histone 3 [H3 (1−25)]. The SAHH clone was provided by Dr. Raymond Trievel (University of Michigan). For IC<sub>50</sub> determination, assay mixtures were prepared with 5 μM SAHH, about 0.3 U/mL of adenosine deaminase from Sigma, 16 μM SAM, 25 nM G9a, and 15 μM Thioglo. The inhibitors were added at concentrations ranging from 6 nM to 25 μM. After 5 min incubation, reactions were initiated by the addition of 5 μM H3 (1−25) peptide. The methylation reaction was followed by monitoring the increase in fluorescence using BioTek Synergy2 plate reader with 360/40 nm excitation filter and 528/20 nm emission filter for 20 min in 384 well-plate format. Homocysteine generated in the assay was quantitated using standard curves. Activity values were corrected by subtracting background caused by the peptide and the protein. IC<sub>50</sub> values were calculated using four parameter logistic equation by Sigmaplot software. Standard deviations were calculated from two independent experiments.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> CLOT Assay Experimental Procedures</h3><div class="NLM_p last">Substrate peptide b-H3(1−11) (biotin-ARTKQTARKST) was synthesized, HPLC-purified, and mass-analyzed by the Tufts Medical School Department of Physiology Core Facility (Boston, MA). To each well of a 1536-well white solid-bottom plate (Greiner Bio-One, Monroe, NC), 3 μL of G9a enzyme (final 7.5 nM) in PBS buffer, pH 7.4, containing 0.01% Tween-20 were added using a BioRAPTR (Beckman Coulter, Fullerton, CA) flying reagent dispenser (FRD). Compounds in DMSO (23 nL) were transferred to the assay plate with a Kalypsys pintool and assayed in duplicate in a 12-point dilution series, yielding a final concentration range of 32 pM to 57 μM and 0.5% DMSO. Following a 15 min incubation of compound with enzyme, b-H3(1−11) peptide and SAM substrates (final 500 nM and 20 μM, respectively) were added in a 1 μL FRD dispense. Reactions were incubated for 2 h at room temperature. To detect methylated b-H3(1−11) peptide, 0.5 μg/mL rabbit <i>anti</i>-methyllysine histone H3K9 antibody (Abcam Inc., Cambridge, MA) was added with 15 μg/mL streptavidin-coated donor and <i>anti</i>-rabbit IgG acceptor AlphaScreen (PerkinElmer, Waltham, MA) beads in a 1 μL FRD dispense. Following a 20 min incubation in the dark, plates were read on an EnVision multilabel plate reader (PerkinElmer) using the 1536 Plate HTS AlphaScreen aperture (80 ms excitation time, 240 ms measurement time). Data were normalized to no-enzyme and no-compound controls and dose−response curves were constructed. IC<sub>50</sub> values were determined with in-house curve fitting software using a four-parameter Hill equation as previously described (<a href="http://www.ncgc.nih.gov/pub/openhts/curvefit/" class="extLink">http://www.ncgc.nih.gov/pub/openhts/curvefit/</a>).</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> DSF Experimental Procedures</h3><div class="NLM_p last">A real-time PCR device (RTPCR 480 II) from Roche was used to monitor protein unfolding by monitoring the increase in the fluorescence of the fluorophor SYPRO Orange (Invitrogen, Carlsbad, CA) as described before.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> Protein samples at 0.1 mg/mL in 100 mM Hepes buffer (pH 7.5) containing 150 mM NaCl, and 0, 6.25, 12.5, 25, 50, 100, and 200 μM of compounds were screened in the presence and absence of 500 μM SAH. Compound dilutions were made from stock solutions of 100% DMSO. The final concentration of DMSO was kept at 0.2% throughout the dilutions. All these solutions contained 5× Sypro Orange. Twenty μL aliquots were transferred to a 384-well PCR plate and scanned at a heating rate of 1 °C/min from 20 to 95 °C. Fluorescence intensities were plotted as a function of temperature by using an internally developed software package.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Peptide Displacement Experimental Procedures</h3><div class="NLM_p last">Fluorescence polarization measurements were performed in 384-well plates, using Synergy 2 microplate reader from BioTek.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The H3 (1−15) peptide (ARTKQTARKSTGGKA) was synthesized, N-terminally labeled with fluorescein [F−H3 (1−15)] and purified by Tufts University Core Services (Boston, MA). Displacement of the F−H3 (1−15) peptide was carried out using the fluorescence polarization signal obtained upon peptide binding to G9a protein. Five μM of G9a was incubated with 36 nM peptide, and different concentrations of <b>3a</b>, <b>10</b>, or unlabeled H3 (1−25) from 0.5 to 500 μM were added. Displacement of the peptide was monitored by following the decrease in FP signal. Data were plotted and fit to a hyperbolic function using Sigma Plot software.</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> ITC Experimental Procedures</h3><div class="NLM_p last">ITC measurements were performed in duplicate at 25 °C, using a VP-ITC microcalorimeter (MicroCal Inc.). Experiments were performed by injecting 10 μL of 300 μM solution of the compound <b>3a</b> or <b>10</b> into a sample cell containing 25 μM G9a that was extensively dialyzed in ITC buffer (25 mM Tris-HCl, pH 7.5, 200 mM NaCl, and 2 mM β-mercaptoethanol) and degassed. A total of 25 injections were performed with a spacing of 180 s and a reference power of 13 μcal/s. The ITC experiments were carried out in the presence of 500 μM SAH by adding it to both G9a and final solution of compounds. Compounds were first dissolved in 100% DMSO at 100 mM and were diluted in ITC buffer (final concentration of 0.3%). The concentration of DMSO was adjusted to 0.3% for the G9a solution. Heat of dilution generated by compounds was subtracted, and binding isotherms were plotted and analyzed using Origin Software (MicroCal Inc.). The ITC measurements were fit to a one-site binding model.</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Morrison <i>K</i><sub>i</sub> Determination Using the Endoproteinase-coupled MCE Assay</h3><div class="NLM_p last">The test compounds were titrated in 10% DMSO in 1X assay buffer (20 mM Tris-HCl pH = 8, 25 mM NaCl, 2 mM DTT and 0.025% Tween 20) using a 1.5-fold dilution scheme over 16 points. The compounds (5 μL) were then spotted into the assay wells of a 384-well microplate in triplicate. In the final assay volume of 50 μL, the compounds spanned a concentration range of 100 nM to 228 pM, and DMSO was 1%. A cocktail made in 1X assay buffer containing G9a (12.5 nM) and an H3K9 (5 μM) substrate peptide, ARTKme1QTARKSTGGK-5/6-FAM, where Kme1 indicates the presence of a monomethyl lysine and 5/6-FAM is a carboxyfluorescein tracer attached to the terminal lysine, was added and allowed to preincubate with the compounds for 10 min at 25 °C. The methyltransferase reaction was initiated by the addition of 10 μM SAM and allowed to proceed for 20 min at 25 °C, and then 10 μL of Endo-LysC (40 pg/μL) was added to proteolyze the remaining unmethylated peptide. Maximum and minimum signal controls were performed in the presence of 1% DMSO, and the minimum signal control reactions were initiated with 1X assay buffer rather than SAM. After 1 h, the plate was read on a Caliper Life Sciences EZ Reader II using upstream voltage −500 V, downstream voltage −1200 V, and pressure of −1.5 psi. The fractional velocities of the reactions were calculated and fit to a quadratic equation described by Morrison and co-workers<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a> using a fixed enzyme concentration of 12.5 nM, a fixed substrate concentration of 5 μM, and a <i>K</i><sub>m</sub> of 31 μM for the peptide.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/jm100478y">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60021" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60021" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>10</b> and <b>29</b>. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm100478y/suppl_file/jm100478y_si_001.pdf">jm100478y_si_001.pdf (507.57 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm100478y" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#315b58505f5b585f71445f521f545544"><span class="__cf_email__" data-cfemail="90faf9f1fefaf9fed0e5fef3bef5f4e5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Liu</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Chen</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdellah Allali-Hassani</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amy M. Quinn</span> - <span class="hlFld-Affiliation affiliation">NIH Chemical Genomics Center, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tim J. Wigle</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gregory A. Wasney</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aiping Dong</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guillermo Senisterra</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene Chau</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alena Siarheyeva</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline L. Norris</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dmitri B. Kireev</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajit Jadhav</span> - <span class="hlFld-Affiliation affiliation">NIH Chemical Genomics Center, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Martin Herold</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William P. Janzen</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen V. Frye</span> - <span class="hlFld-Affiliation affiliation">Center for Integrative Chemical Biology and Drug Discovery, Division of Medicinal Chemistry and Natural Products, Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Simeonov</span> - <span class="hlFld-Affiliation affiliation">NIH Chemical Genomics Center, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78253" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78253" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Yizhou Dong and Professor K. H. Lee for HRMS and HPLC support and Dr. Stanley Ng for supplying the JMJD2E enzyme. The research described here was supported by the grant RC1GM090732 from the National Institute Of General Medical Sciences of NIH, the Carolina Partnership, the Molecular Libraries Initiative of the NIH Roadmap for Medical Research, the Intramural Research Program of NHGRI of NIH, the Ontario Research Fund and the Structural Genomics Consortium, a registered charity (number 1097737) that receives funds from the Canadian Institutes for Health Research, the Canada Foundation for Innovation, Genome Canada through the Ontario Genomics Institute, GlaxoSmithKline, Karolinska Institutet, the Knut and Alice Wallenberg Foundation, the Ontario Innovation Trust, the Ontario Ministry for Research and Innovation, Merck & Co., Inc., the Novartis Research Foundation, the Swedish Agency for Innovation Systems, the Swedish Foundation for Strategic Research, and the Wellcome Trust.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48573" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48573" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 41 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Bernstein, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">The mammalian epigenome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0ljY1ebBJj_QJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Gelato, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischle, W.</span><span> </span><span class="NLM_article-title">Role of histone modifications in defining chromatin structure and function</span> <span class="citation_source-journal">Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1515%2FBC.2008.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18225984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFait7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2008&pages=353-363&author=K.+A.+Gelatoauthor=W.+Fischle&title=Role+of+histone+modifications+in+defining+chromatin+structure+and+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Role of histone modifications in defining chromatin structure and function</span></div><div class="casAuthors">Gelato, Kathy A.; Fischle, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">353-363</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Chromosomes in eukaryotic cell nuclei are not uniformly organized, but rather contain distinct chromatin elements, with each state having a defined biochem. structure and biol. function.  These are recognizable by their distinct architectures and mol. components, which can change in response to cellular stimuli or metabolic requirements.  Chromatin elements are characterized by the fundamental histone and DNA components, as well as other assocd. non-histone proteins and factors.  Post-translational modifications of histones in particular often correlate with a specific chromatin structure and function.  Patterns of histone modifications are implicated as having a role in directing the level of chromatin compaction, as well as playing roles in multiple functional pathways directing the readout of distinct regions of the genome.  Here, the authors review the properties of various chromatin elements and the apparent links of histone modifications with chromatin organization and functional output.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm8YRKkBO4w7Vg90H21EOLACvtfcHk0ljY1ebBJj_QJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksFait7s%253D&md5=14b1db5766047ab8201007c5cde0405d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1515%2FBC.2008.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2008.048%26sid%3Dliteratum%253Aachs%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DFischle%26aufirst%3DW.%26atitle%3DRole%2520of%2520histone%2520modifications%2520in%2520defining%2520chromatin%2520structure%2520and%2520function%26jtitle%3DBiol.%2520Chem.%26date%3D2008%26volume%3D389%26spage%3D353%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The language of covalent histone modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eb-aZFeLpTuO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lh5c4I7C0vgNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Esteller, M.</span><span> </span><span class="NLM_article-title">Epigenetics in cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">358</span><span class="NLM_x">, </span> <span class="NLM_fpage">1148</span><span class="NLM_x">–</span> <span class="NLM_lpage">1159</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1056%2FNEJMra072067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18337604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1ahs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2008&pages=1148-1159&author=M.+Esteller&title=Epigenetics+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics in cancer</span></div><div class="casAuthors">Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1148-1159</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The epigenetic therapy in cancer is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzzFSlZotpmLVg90H21EOLACvtfcHk0lh5c4I7C0vgNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1ahs7Y%253D&md5=c354604e711e60b49e37f151f14a5dba</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra072067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra072067%26sid%3Dliteratum%253Aachs%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DEpigenetics%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2008%26volume%3D358%26spage%3D1148%26epage%3D1159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Lyko, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R.</span><span> </span><span class="NLM_article-title">DNA methyltransferase inhibitors and the development of epigenetic cancer therapies</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1498</span><span class="NLM_x">–</span> <span class="NLM_lpage">1506</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1093%2Fjnci%2Fdji311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16234563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCjtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2005&pages=1498-1506&author=F.+Lykoauthor=R.+Brown&title=DNA+methyltransferase+inhibitors+and+the+development+of+epigenetic+cancer+therapies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Methyltransferase Inhibitors and the Development of Epigenetic Cancer Therapies</span></div><div class="casAuthors">Lyko, Frank; Brown, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1498-1506</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Epimutations, such as the hypermethylation and epigenetic silencing of tumor suppressor genes, play a role in the etiol. of human cancers.  In contrast to DNA mutations, which are passively inherited through DNA replication, epimutations must be actively maintained because they are reversible.  In fact, the reversibility of epimutations by small-mol. inhibitors provides the foundation for the use of such inhibitors in novel cancer therapy strategies.  Among the compds. that inhibit epigenetic processes, the most extensively studied are DNA methyltransferase inhibitors.  In this review, we examine the literature on DNA methyltransferase inhibitors and discuss the efficacy of such compds. as antitumor agents, as evaluated in phase I-III clin. trials.  We also discuss future areas of research, including the development of nonnucleoside inhibitors, the application of novel bioanal. tools for DNA methylation anal. (which will be important for the clin. application of these compds. by allowing rational approaches to trial design), the need to optimize treatment schedules for maximal biol. effectiveness, and the need to define mol. endpoints so that changes induced by demethylating drugs in patients can be monitored during treatment.  Assays for genome-wide and tumor-specific DNA methylation also need to be further developed to establish the pharmacodynamic parameters of DNA methyltransferase inhibitors in patients and to provide rational approaches to maximizing the therapeutic efficacy of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2i9sqxc3_wrVg90H21EOLACvtfcHk0lgvLLKg_PKx7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCjtLfF&md5=6edee05ea71b9e68adf1b811fe592ddd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdji311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdji311%26sid%3Dliteratum%253Aachs%26aulast%3DLyko%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DR.%26atitle%3DDNA%2520methyltransferase%2520inhibitors%2520and%2520the%2520development%2520of%2520epigenetic%2520cancer%2520therapies%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2005%26volume%3D97%26spage%3D1498%26epage%3D1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nature Rev. Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lgvLLKg_PKx7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNature%2520Rev.%2520Mol.%2520Cell.%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenhaber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Carroll, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponting, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">406</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.+W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+chromatin+structure+by+site-specific+histone+H3+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lgvLLKg_PKx7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.%2BW.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520structure%2520by%2520site-specific%2520histone%2520H3%2520methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lg6-rfW6a8CiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nature Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNature%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Cancer treatment of the future: inhibitors of histone methyltransferases</span> <span class="citation_source-journal">Int. J. Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.biocel.2008.07.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18773966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWnu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=4-11&author=A.+Spannhoffauthor=W.+Sipplauthor=M.+Jung&title=Cancer+treatment+of+the+future%3A+inhibitors+of+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer treatment of the future: Inhibitors of histone methyltransferases</span></div><div class="casAuthors">Spannhoff, Astrid; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-11</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer in humans is the result of a multi-step process.  This process often involves the activation of oncogenes and/or the inactivation of tumor suppressor genes.  These 2 steps arise not only due to mutations, but can also be the result of a translocation or an altered transcription rate.  One important mechanism is the occurrence of epigenetic alterations like promotor methylation (which may lead to tumor suppressor silencing) or decreased histone acetylation (which can result in the downregulation of proteins involved in apoptosis).  Today, histone acetylation and DNA methylation are epigenetic modifications which were linked closely to the pathol. of human cancers and inhibitors of both enzyme classes for clin. use are at hand.  In contrast, other fields of epigenetics still lack of similarly thorough knowledge.  This is esp. true for the group of histone methyltransferases and their inhibitors.  Since connections between histone methylation patterns and cancer progression were recognized, histone methyltransferases represent promising targets for future cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSu9HVhLFV07Vg90H21EOLACvtfcHk0lg6-rfW6a8CiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWnu73O&md5=09e514f87c700b4a5b2393d9608b1af1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2008.07.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2008.07.024%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DCancer%2520treatment%2520of%2520the%2520future%253A%2520inhibitors%2520of%2520histone%2520methyltransferases%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2009%26volume%3D41%26spage%3D4%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Fog, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lund, A. H.</span><span> </span><span class="NLM_article-title">Chromatin-modifying proteins in cancer</span> <span class="citation_source-journal">Acta Pathol., Microbiol. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">1060</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1111%2Fj.1600-0463.2007.apm_776.xml.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18042144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2007&pages=1060-1089&author=C.+K.+Fogauthor=K.+T.+Jensenauthor=A.+H.+Lund&title=Chromatin-modifying+proteins+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-modifying proteins in cancer</span></div><div class="casAuthors">Fog, Cathrine K.; Jensen, Klaus T.; Lund, Anders H.</div><div class="citationInfo"><span class="NLM_cas:title">APMIS</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1060-1089</span>CODEN:
                <span class="NLM_cas:coden">APMSEL</span>;
        ISSN:<span class="NLM_cas:issn">0903-4641</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Chromatin-modifying proteins mold the genome into areas that are accessible for transcriptional activity and areas that are transcriptionally silent.  This epigenetic gene regulation allows for different transcriptional programs to be conducted in different cell types at different timepoints-despite the fact that all cells in the organism contain the same genetic information.  A large amt. of data gathered over the last decades has demonstrated that deregulation of chromatin-modifying proteins is etiol. involved in the development and progression of cancer.  Here we discuss how epigenetic alterations influence cancer development and review known cancer-assocd. alterations in chromatin-modifying proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxhso-Ej5KY7Vg90H21EOLACvtfcHk0lgEC1zQkK4t0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D&md5=060c4f35c69924a72a16b316b773a215</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0463.2007.apm_776.xml.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0463.2007.apm_776.xml.x%26sid%3Dliteratum%253Aachs%26aulast%3DFog%26aufirst%3DC.%2BK.%26aulast%3DJensen%26aufirst%3DK.%2BT.%26aulast%3DLund%26aufirst%3DA.%2BH.%26atitle%3DChromatin-modifying%2520proteins%2520in%2520cancer%26jtitle%3DActa%2520Pathol.%252C%2520Microbiol.%2520Immunol.%26date%3D2007%26volume%3D115%26spage%3D1060%26epage%3D1089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">The emerging therapeutic potential of histone methyltransferase and demethylase inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1568</span><span class="NLM_x">–</span> <span class="NLM_lpage">1582</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1568-1582&author=A.+Spannhoffauthor=A.+T.+Hauserauthor=R.+Heinkeauthor=W.+Sipplauthor=M.+Jung&title=The+emerging+therapeutic+potential+of+histone+methyltransferase+and+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DThe%2520emerging%2520therapeutic%2520potential%2520of%2520histone%2520methyltransferase%2520and%2520demethylase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D1568%26epage%3D1582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanmugam, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, J.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natarajan, R.</span><span> </span><span class="NLM_article-title">Role of the Histone H3 Lysine 4 Methyltransferase, SET7/9, in the Regulation of NF-{kappa}B-dependent Inflammatory Genes: Relevance to Diabetes and Inflammation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">26771</span><span class="NLM_x">–</span> <span class="NLM_lpage">26781</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=26771-26781&author=Y.+Liauthor=M.+A.+Reddyauthor=F.+Miaoauthor=N.+Shanmugamauthor=J.-K.+Yeeauthor=D.+Hawkinsauthor=B.+Renauthor=R.+Natarajan&title=Role+of+the+Histone+H3+Lysine+4+Methyltransferase%2C+SET7%2F9%2C+in+the+Regulation+of+NF-%7Bkappa%7DB-dependent+Inflammatory+Genes%3A+Relevance+to+Diabetes+and+Inflammation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DReddy%26aufirst%3DM.%2BA.%26aulast%3DMiao%26aufirst%3DF.%26aulast%3DShanmugam%26aufirst%3DN.%26aulast%3DYee%26aufirst%3DJ.-K.%26aulast%3DHawkins%26aufirst%3DD.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DNatarajan%26aufirst%3DR.%26atitle%3DRole%2520of%2520the%2520Histone%2520H3%2520Lysine%25204%2520Methyltransferase%252C%2520SET7%252F9%252C%2520in%2520the%2520Regulation%2520of%2520NF-%257Bkappa%257DB-dependent%2520Inflammatory%2520Genes%253A%2520Relevance%2520to%2520Diabetes%2520and%2520Inflammation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D26771%26epage%3D26781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Maze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, H. E.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPlant, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechanic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neve, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurd, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Essential role of the histone methyltransferase G9a in cocaine-induced plasticity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1126%2Fscience.1179438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20056891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=213-216&author=I.+Mazeauthor=H.+E.+Covingtonauthor=D.+M.+Dietzauthor=Q.+LaPlantauthor=W.+Renthalauthor=S.+J.+Russoauthor=M.+Mechanicauthor=E.+Mouzonauthor=R.+L.+Neveauthor=S.+J.+Haggartyauthor=Y.+Renauthor=S.+C.+Sampathauthor=Y.+L.+Hurdauthor=P.+Greengardauthor=A.+Tarakhovskyauthor=A.+Schaeferauthor=E.+J.+Nestler&title=Essential+role+of+the+histone+methyltransferase+G9a+in+cocaine-induced+plasticity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Essential Role of the Histone Methyltransferase G9a in Cocaine-Induced Plasticity</span></div><div class="casAuthors">Maze, Ian; Covington, Herbert E., III; Dietz, David M.; LaPlant, Quincey; Renthal, William; Russo, Scott J.; Mechanic, Max; Mouzon, Ezekiell; Neve, Rachael L.; Haggarty, Stephen J.; Ren, Yanhua; Sampath, Srihari C.; Hurd, Yasmin L.; Greengard, Paul; Tarakhovsky, Alexander; Schaefer, Anne; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5962</span>),
    <span class="NLM_cas:pages">213-216</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cocaine-induced alterations in gene expression cause changes in neuronal morphol. and behavior that may underlie cocaine addiction.  In mice, we identified an essential role for histone 3 lysine 9 (H3K9) dimethylation and the lysine dimethyltransferase G9a in cocaine-induced structural and behavioral plasticity.  Repeated cocaine administration reduced global levels of H3K9 dimethylation in the nucleus accumbens.  This redn. in histone methylation was mediated through the repression of G9a in this brain region, which was regulated by the cocaine-induced transcription factor ΔFosB.  Using conditional mutagenesis and viral-mediated gene transfer, we found that G9a down-regulation increased the dendritic spine plasticity of nucleus accumbens neurons and enhanced the preference for cocaine, thereby establishing a crucial role for histone methylation in the long-term actions of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SPD826wEGrVg90H21EOLACvtfcHk0lhldWkUspPCzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D&md5=a8347500f81862078787e50fff53b6b9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179438%26sid%3Dliteratum%253Aachs%26aulast%3DMaze%26aufirst%3DI.%26aulast%3DCovington%26aufirst%3DH.%2BE.%26aulast%3DDietz%26aufirst%3DD.%2BM.%26aulast%3DLaPlant%26aufirst%3DQ.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DRusso%26aufirst%3DS.%2BJ.%26aulast%3DMechanic%26aufirst%3DM.%26aulast%3DMouzon%26aufirst%3DE.%26aulast%3DNeve%26aufirst%3DR.%2BL.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DHurd%26aufirst%3DY.%2BL.%26aulast%3DGreengard%26aufirst%3DP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEssential%2520role%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520in%2520cocaine-induced%2520plasticity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D213%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Intrator, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gertler, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surmeier, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span> </span><span class="NLM_article-title">Control of Cognition and Adaptive Behavior by the GLP/G9a Epigenetic Suppressor Complex</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.neuron.2009.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20005824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2009&pages=678-691&author=A.+Schaeferauthor=S.+C.+Sampathauthor=A.+Intratorauthor=A.+Minauthor=T.+S.+Gertlerauthor=D.+J.+Surmeierauthor=A.+Tarakhovskyauthor=P.+Greengard&title=Control+of+Cognition+and+Adaptive+Behavior+by+the+GLP%2FG9a+Epigenetic+Suppressor+Complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span></div><div class="casAuthors">Schaefer, Anne; Sampath, Srihari C.; Intrator, Adam; Min, Alice; Gertler, Tracy S.; Surmeier, D. James; Tarakhovsky, Alexander; Greengard, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">678-691</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The genetic basis of cognition and behavioral adaptation to the environment remains poorly understood.  Here we demonstrate that the histone methyltransferase complex GLP/G9a controls cognition and adaptive responses in a region-specific fashion in the adult brain.  Using conditional mutagenesis in mice, we show that postnatal, neuron-specific deficiency of GLP/G9a leads to derepression of numerous nonneuronal and neuron progenitor genes in adult neurons.  This transcriptional alteration is assocd. with complex behavioral abnormalities, including defects in learning, motivation, and environmental adaptation.  The behavioral changes triggered by GLP/G9a deficiency are similar to key symptoms of the human 9q34 mental retardation syndrome that is assocd. with structural alterations of the GLP/EHMT1 gene.  The likely causal role of GLP/G9a in mental retardation in mice and humans suggests a key role for the GLP/G9a-controlled histone H3K9 dimethylation in regulation of brain function through maintenance of the transcriptional homeostasis in adult neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22UcjjqkKdLVg90H21EOLACvtfcHk0lhldWkUspPCzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D&md5=ece4e94ad097a8d0e67a5e8d27970e64</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2009.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2009.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DIntrator%26aufirst%3DA.%26aulast%3DMin%26aufirst%3DA.%26aulast%3DGertler%26aufirst%3DT.%2BS.%26aulast%3DSurmeier%26aufirst%3DD.%2BJ.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DGreengard%26aufirst%3DP.%26atitle%3DControl%2520of%2520Cognition%2520and%2520Adaptive%2520Behavior%2520by%2520the%2520GLP%252FG9a%2520Epigenetic%2520Suppressor%2520Complex%26jtitle%3DNeuron%26date%3D2009%26volume%3D64%26spage%3D678%26epage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Imai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togami, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, T.</span><span> </span><span class="NLM_article-title">Involvement of histone H3 Lysine 9 (H3K9) methyl transferase G9a in the maintenance of HIV-1 latency and its reactivation by BIX01294</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">16538</span><span class="NLM_x">–</span> <span class="NLM_lpage">16545</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=16538-16545&author=K.+Imaiauthor=H.+Togamiauthor=T.+Okamoto&title=Involvement+of+histone+H3+Lysine+9+%28H3K9%29+methyl+transferase+G9a+in+the+maintenance+of+HIV-1+latency+and+its+reactivation+by+BIX01294"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DK.%26aulast%3DTogami%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DT.%26atitle%3DInvolvement%2520of%2520histone%2520H3%2520Lysine%25209%2520%2528H3K9%2529%2520methyl%2520transferase%2520G9a%2520in%2520the%2520maintenance%2520of%2520HIV-1%2520latency%2520and%2520its%2520reactivation%2520by%2520BIX01294%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D16538%26epage%3D16545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1791</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1779-1791&author=M.+Tachibanaauthor=K.+Sugimotoauthor=M.+Nozakiauthor=J.+Uedaauthor=T.+Ohtaauthor=M.+Ohkiauthor=M.+Fukudaauthor=N.+Takedaauthor=H.+Niidaauthor=H.+Katoauthor=Y.+Shinkai&title=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DOhki%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DNiida%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%2520histone%2520methyltransferase%2520plays%2520a%2520dominant%2520role%2520in%2520euchromatic%2520histone%2520H3%2520lysine%25209%2520methylation%2520and%2520is%2520essential%2520for%2520early%2520embryogenesis%26jtitle%3DGenes%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D1779%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">McGarvey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahrner, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Silenced tumor suppressor genes reactivated by DNA demethylation do not return to a fully euchromatic chromatin state</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">3541</span><span class="NLM_x">–</span> <span class="NLM_lpage">3549</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1158%2F0008-5472.CAN-05-2481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16585178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=3541-3549&author=K.+M.+McGarveyauthor=J.+A.+Fahrnerauthor=E.+Greeneauthor=J.+Martensauthor=T.+Jenuweinauthor=S.+B.+Baylin&title=Silenced+tumor+suppressor+genes+reactivated+by+DNA+demethylation+do+not+return+to+a+fully+euchromatic+chromatin+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Silenced Tumor Suppressor Genes Reactivated by DNA Demethylation Do Not Return to a Fully Euchromatic Chromatin State</span></div><div class="casAuthors">McGarvey, Kelly M.; Fahrner, Jill A.; Greene, Eriko; Martens, Joost; Jenuwein, Thomas; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3541-3549</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone H3 lysine 9 (H3K9) and lysine 27 (H3K27) trimethylation are properties of stably silenced heterochromatin whereas H3K9 dimethylation (H3K9me2) is important for euchromatic gene repression.  In colorectal cancer cells, all of these marks, as well as the key enzymes which establish them, surround the hMLH1 promoter when it is DNA hypermethylated and aberrantly silenced, but are absent when the gene is unmethylated and fully expressed in a euchromatic state.  When the aberrantly silenced gene is DNA demethylated and reexpressed following 5-aza-2'-deoxycytidine treatment, H3K9me1 and H3K9me2 are the only silencing marks that are lost.  A series of other silenced and DNA hypermethylated gene promoters behave identically even when the genes are chronically DNA demethylated and reexpressed after genetic knockout of DNA methyltransferases.  Our data indicate that when transcription of DNA hypermethylated genes is activated in cancer cells, their promoters remain in an environment with certain heterochromatic characteristics.  This finding has important implications for the translational goal of reactivating aberrantly silenced cancer genes as a therapeutic maneuver.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEjoUm5wwMFbVg90H21EOLACvtfcHk0ljv8q5x_FIKmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D&md5=73b1830a5f7f839d53d5539c6e707462</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-2481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-2481%26sid%3Dliteratum%253Aachs%26aulast%3DMcGarvey%26aufirst%3DK.%2BM.%26aulast%3DFahrner%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DJ.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSilenced%2520tumor%2520suppressor%2520genes%2520reactivated%2520by%2520DNA%2520demethylation%2520do%2520not%2520return%2520to%2520a%2520fully%2520euchromatic%2520chromatin%2520state%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D3541%26epage%3D3549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boumber, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P.</span><span> </span><span class="NLM_article-title">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span> <span class="citation_source-journal">PLoS ONE</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2037</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1371%2Fjournal.pone.0002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18446223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2037&author=Y.+Kondoauthor=L.+Shenauthor=S.+Ahmedauthor=Y.+Boumberauthor=Y.+Sekidoauthor=B.+R.+Haddadauthor=J.+P.+Issa&title=Downregulation+of+histone+H3+lysine+9+methyltransferase+G9a+induces+centrosome+disruption+and+chromosome+instability+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span></div><div class="casAuthors">Kondo Yutaka; Shen Lanlan; Ahmed Saira; Boumber Yanis; Sekido Yoshitaka; Haddad Bassem R; Issa Jean-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e2037</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Modifications of the histone amino-terminal tails affect access of regulatory factors and complexes to chromatin and thereby influence biological processes.  Cancer cells are characterized by prominent epigenetic dysregulation, including histone modifications.  However, the functional roles of the histone methyltransferases (HMT) in cancer remain unclear.  METHODOLOGY/PRINCIPAL FINDINGS:  We studied RNAi-based inhibition (knockdown, KD) of 2 different H3K9 HMTs, SUV39H1 and G9a.  Knockdown of the 2 HMTs in PC3 cancer cell line markedly inhibited cell growth and caused profound morphological changes with loss of telomerase activity and shortened telomeres.  SUV39H1 KD cells showed substantial increase in G2/M fraction.  G9a KD cells showed increased DNA content (1.7-fold in 2 independent clones) compared with FACS analyses to control.  Karyotype analyses showed that this was due to an increased number of chromosomes (from 61 to 102) in G9a KD cells compared to parental PC3.  Intriguingly, we found abnormal centrosome morphology and number in about 25% of the G9a KD cells, while centrosomes were morphologically normal in control cells.  Microarray analyses after KD of SUV39H1 or G9a showed very few genes up-regulated among the 39,000 genes.  The silenced tumor-suppressor genes p16 and RASSF1A were not activated in KD cells.  CONCLUSIONS/SIGNIFICANCE:  These data suggest that the 2 HMTs, SUV39H1 and G9a are required to perpetuate the malignant phenotype.  Furthermore, G9a plays a critical role in regulating centrosome duplication presumably through chromatin structure rather than through affecting gene expression in cancer cells.  Targeting these histone methyltransferases may be of therapeutic benefit in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhnx1F8nyNwDEwO064Z95rfW6udTcc2ebhPq-AROrhdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D&md5=22936eeeffa4dc29e7254dfc546ebcfd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002037%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DAhmed%26aufirst%3DS.%26aulast%3DBoumber%26aufirst%3DY.%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DHaddad%26aufirst%3DB.%2BR.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26atitle%3DDownregulation%2520of%2520histone%2520H3%2520lysine%25209%2520methyltransferase%2520G9a%2520induces%2520centrosome%2520disruption%2520and%2520chromosome%2520instability%2520in%2520cancer%2520cells%26jtitle%3DPLoS%2520ONE%26date%3D2008%26volume%3D3%26spage%3De2037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">G9a and GLP methylate lysine 373 in the tumor suppressor p53</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">9636</span><span class="NLM_x">–</span> <span class="NLM_lpage">9641</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1074%2Fjbc.M109.062588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20118233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=9636-9641&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=X.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9a+and+GLP+methylate+lysine+373+in+the+tumor+suppressor+p53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">G9a and Glp Methylate Lysine 373 in the Tumor Suppressor p53</span></div><div class="casAuthors">Huang, Jing; Dorsey, Jean; Chuikov, Sergei; Zhang, Xinyue; Jenuwein, Thomas; Reinberg, Danny; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9636-9641</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is regulated by numerous post-translational modifications.  Lysine methylation has recently emerged as a key post-translational modification that alters the activity of p53.  Here, we describe a novel lysine methylation site in p53 that is carried out by two homologous histone methyltransferases, G9a and Glp.  G9a and Glp specifically methylate p53 at Lys373, resulting mainly in dimethylation.  During DNA damage, the overall level of p53 modified at Lys373me2 does not increase, despite the dramatic increase in total p53, indicating that Lys373me2 correlates with inactive p53.  Further, redn. of G9a and/or Glp levels leads to a larger population of apoptotic cells.  Examn. of the Oncomine data base shows that G9a and Glp are overexpressed in various cancers compared with corresponding normal tissues, suggesting that they are putative oncogenes.  These data reveal a new methylation site within p53 mediated by the methylases G9a and Glp and indicate that G9a is a potential inhibitory target for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozJXSsZlEE9LVg90H21EOLACvtfcHk0lj2KVbV5B2wbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D&md5=81adc2281f6344caf9a46ebcdf1c61e6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9a%2520and%2520GLP%2520methylate%2520lysine%2520373%2520in%2520the%2520tumor%2520suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D9636%26epage%3D9641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0lj2KVbV5B2wbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desponts, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahm, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholer, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, S.</span><span> </span><span class="NLM_article-title">A combined chemical and genetic approach for the generation of induced pluripotent stem cells</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">528</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2008&pages=525-528&author=Y.+Shiauthor=J.+T.+Doauthor=C.+Despontsauthor=H.+S.+Hahmauthor=H.+R.+Scholerauthor=S.+Ding&title=A+combined+chemical+and+genetic+approach+for+the+generation+of+induced+pluripotent+stem+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDo%26aufirst%3DJ.%2BT.%26aulast%3DDesponts%26aufirst%3DC.%26aulast%3DHahm%26aufirst%3DH.%2BS.%26aulast%3DScholer%26aufirst%3DH.%2BR.%26aulast%3DDing%26aufirst%3DS.%26atitle%3DA%2520combined%2520chemical%2520and%2520genetic%2520approach%2520for%2520the%2520generation%2520of%2520induced%2520pluripotent%2520stem%2520cells%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2008%26volume%3D2%26spage%3D525%26epage%3D528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desponts, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahm, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholer, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, S.</span><span> </span><span class="NLM_article-title">Induction of pluripotent stem cells from mouse embryonic fibroblasts by Oct4 and Klf4 with small-molecule compounds</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">568</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=568-574&author=Y.+Shiauthor=C.+Despontsauthor=J.+T.+Doauthor=H.+S.+Hahmauthor=H.+R.+Scholerauthor=S.+Ding&title=Induction+of+pluripotent+stem+cells+from+mouse+embryonic+fibroblasts+by+Oct4+and+Klf4+with+small-molecule+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDesponts%26aufirst%3DC.%26aulast%3DDo%26aufirst%3DJ.%2BT.%26aulast%3DHahm%26aufirst%3DH.%2BS.%26aulast%3DScholer%26aufirst%3DH.%2BR.%26aulast%3DDing%26aufirst%3DS.%26atitle%3DInduction%2520of%2520pluripotent%2520stem%2520cells%2520from%2520mouse%2520embryonic%2520fibroblasts%2520by%2520Oct4%2520and%2520Klf4%2520with%2520small-molecule%2520compounds%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2008%26volume%3D3%26spage%3D568%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Greiner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonaldi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eskeland, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roemer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span> </span><span class="NLM_article-title">Identification of a specific inhibitor of the histone methyltransferase SU(VAR)3−9</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">145</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnchembio721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16408017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1ygsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=143-145&author=D.+Greinerauthor=T.+Bonaldiauthor=R.+Eskelandauthor=E.+Roemerauthor=A.+Imhof&title=Identification+of+a+specific+inhibitor+of+the+histone+methyltransferase+SU%28VAR%293%E2%88%929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a specific inhibitor of the histone methyltransferase SU(VAR)3-9</span></div><div class="casAuthors">Greiner, Dorothea; Bonaldi, Tiziana; Eskeland, Ragnhild; Roemer, Ernst; Imhof, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">143-145</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation plays a key role in establishing and maintaining stable gene expression patterns during cellular differentiation and embryonic development.  Here, we report the characterization of the fungal metabolite chaetocin as the first inhibitor of a lysine-specific histone methyltransferase.  Chaetocin is specific for the methyltransferase SU(VAR)3-9 both in vitro and in vivo and may therefore be used to study heterochromatin-mediated gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_LIt2UeAM9rVg90H21EOLACvtfcHk0lg7JyH28pPrPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1ygsr4%253D&md5=27aa101782ac63ef70b7e8c3d57ab2fe</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio721%26sid%3Dliteratum%253Aachs%26aulast%3DGreiner%26aufirst%3DD.%26aulast%3DBonaldi%26aufirst%3DT.%26aulast%3DEskeland%26aufirst%3DR.%26aulast%3DRoemer%26aufirst%3DE.%26aulast%3DImhof%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520a%2520specific%2520inhibitor%2520of%2520the%2520histone%2520methyltransferase%2520SU%2528VAR%25293%25E2%2588%25929%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2005%26volume%3D1%26spage%3D143%26epage%3D145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Chemical probes for histone-modifying enzymes</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnchembio.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18800048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=590-597&author=P.+A.+Cole&title=Chemical+probes+for+histone-modifying+enzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for histone-modifying enzymes</span></div><div class="casAuthors">Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">590-597</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The histone-modifying enzymes that catalyze reversible lysine acetylation and methylation are central to the epigenetic regulation of chromatin remodeling.  From the early discovery of histone deacetylase inhibitors to the more recent identification of histone demethylase blockers, chem. approaches offer increasingly sophisticated tools for the investigation of the structure and function of these lysine-modifying enzymes.  This review summarizes progress to date on compds. identified from screens or by design that can modulate the activity of classical histone deacetylases, sirtuins, histone acetyltransferases, histone methyltransferases and histone demethylases.  We highlight applications of compds. to mechanistic and functional studies involving these enzymes and discuss future challenges regarding target specificity and general utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ5RKIrQsO7Vg90H21EOLACvtfcHk0ljETx30csPHNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL&md5=71fe323c232af42b67a7dfc134c4d49b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.111%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DChemical%2520probes%2520for%2520histone-modifying%2520enzymes%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D590%26epage%3D597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Frye, S. V.; Heightman, T.; Jin, J.</span><span> </span><span class="NLM_article-title">Targeting Methyl Lysine</span>.  <span class="citation_source-journal">Annu. Rep. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span>in press, DOI: <span class="refDoi"> DOI: 10.1016/S0065-7743(10)45020-4</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2FS0065-7743%2810%2945020-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=S.+V.+Frye&author=T.+Heightman&author=J.+Jin&title=Targeting+Methyl+Lysine&doi=10.1016%2FS0065-7743%2810%2945020-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0065-7743%2810%2945020-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-7743%252810%252945020-4%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DTargeting%2520Methyl%2520Lysine%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2010%26doi%3D10.1016%2FS0065-7743%2810%2945020-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0ljETx30csPHNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnsmb.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=19219047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=312-317&author=Y.+Changauthor=X.+Zhangauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=A.+Spannhoffauthor=J.+Liuauthor=J.+P.+Snyderauthor=M.+T.+Bedfordauthor=X.+Cheng&title=Structural+basis+for+G9a-like+protein+lysine+methyltransferase+inhibition+by+BIX-01294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span></div><div class="casAuthors">Chang, Yanqi; Zhang, Xing; Horton, John R.; Upadhyay, Anup K.; Spannhoff, Astrid; Liu, Jin; Snyder, James P.; Bedford, Mark T.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">312-317</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone lysine methylation is an important epigenetic mark that regulates gene expression and chromatin organization.  G9a and G9a-like protein (GLP) are euchromatin-assocd. methyltransferases that repress transcription by methylating histone H3 Lys9.  BIX-01294 was originally identified as a G9a inhibitor during a chem. library screen of small mols. and has previously been used in the generation of induced pluripotent stem cells.  Here we present the crystal structure of the catalytic SET domain of GLP in complex with BIX-01294 and S-adenosyl-L-homocysteine.  The inhibitor is bound in the substrate peptide groove at the location where the histone H3 residues N-terminal to the target lysine lie in the previously solved structure of the complex with histone peptide.  The inhibitor resembles the bound conformation of histone H3 Lys4 to Arg8, and is positioned in place by residues specific for G9a and GLP through specific interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhQepqN3zr_bVg90H21EOLACvtfcHk0lj414bEcGIxIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D&md5=6a16baaf33420fc1caf1df70d103d270</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1560%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520G9a-like%2520protein%2520lysine%2520methyltransferase%2520inhibition%2520by%2520BIX-01294%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D312%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">A chemiluminescence-based method for identification of histone lysine methyltransferase inhibitors</span> <span class="citation_source-journal">Molecular BioSystems</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">782</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=782-788&author=A.+M.+Quinnauthor=A.+Allali-Hassaniauthor=M.+Vedadiauthor=A.+Simeonov&title=A+chemiluminescence-based+method+for+identification+of+histone+lysine+methyltransferase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DA%2520chemiluminescence-based%2520method%2520for%2520identification%2520of%2520histone%2520lysine%2520methyltransferase%2520inhibitors%26jtitle%3DMolecular%2520BioSystems%26date%3D2010%26volume%3D6%26spage%3D782%26epage%3D788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lj414bEcGIxIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">A coupled fluorescent assay for histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+coupled+fluorescent+assay+for+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0lhQTJvw6pUUZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520coupled%2520fluorescent%2520assay%2520for%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="note"><p class="first last">Prepared as a racemic mixture.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nady, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">L3MBTL1 recognition of mono- and dimethylated histones</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1229</span><span class="NLM_x">–</span> <span class="NLM_lpage">1230</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnsmb1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18026117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1229-1230&author=J.+Minauthor=A.+Allali-Hassaniauthor=N.+Nadyauthor=C.+Qiauthor=H.+Ouyangauthor=Y.+Liuauthor=F.+MacKenzieauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=L3MBTL1+recognition+of+mono-+and+dimethylated+histones"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">L3MBTL1 recognition of mono- and dimethylated histones</span></div><div class="casAuthors">Min, Jinrong; Allali-Hassani, Abdellah; Nady, Nataliya; Qi, Chao; Ouyang, Hui; Liu, Yongsong; MacKenzie, Farrell; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1229-1230</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Crystal structures of the L3MBTL1 MBT repeats in complex with histone H4 peptides dimethylated on Lys20 (H4K20me2) show that only the second of the three MBT repeats can bind mono- and dimethylated histone peptides.  Its binding pocket has similarities to that of 53BP1 and is able to recognize the degree of histone lysine methylation.  An unexpected mode of peptide-mediated dimerization suggests a possible mechanism for chromatin compaction by L3MBTL1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyCuRY6dV9-bVg90H21EOLACvtfcHk0lhQTJvw6pUUZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP&md5=1f1072bf117cb7459a644907ca66b74b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1340%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DNady%26aufirst%3DN.%26aulast%3DQi%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMacKenzie%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DL3MBTL1%2520recognition%2520of%2520mono-%2520and%2520dimethylated%2520histones%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1229%26epage%3D1230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Wigle, T. J.; Provencher, L. M.; Norris, J. L.; Jin, J.; Brown, P. J.; Frye, S. V.; Janzen, W. P.</span><span> </span><span class="NLM_article-title">Accessing Protein Methyltransferase and Demethylase Enzymology Using Microfluidic Capillary Electrophoresis</span>.  <span class="citation_source-journal">Chemistry & Biology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span>in press.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=T.+J.+Wigle&author=L.+M.+Provencher&author=J.+L.+Norris&author=J.+Jin&author=P.+J.+Brown&author=S.+V.+Frye&author=W.+P.+Janzen&title=Accessing+Protein+Methyltransferase+and+Demethylase+Enzymology+Using+Microfluidic+Capillary+Electrophoresis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26atitle%3DAccessing%2520Protein%2520Methyltransferase%2520and%2520Demethylase%2520Enzymology%2520Using%2520Microfluidic%2520Capillary%2520Electrophoresis%26jtitle%3DChemistry%2520%2526%2520Biology%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">185</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">286</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0lhQTJvw6pUUZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Williams, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">The kinetics of reversible tight-binding inhibition</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">437</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+kinetics+of+reversible+tight-binding+inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0lhQTJvw6pUUZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520kinetics%2520of%2520reversible%2520tight-binding%2520inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Sakurai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">A miniaturized screen for inhibitors of Jumonji histone demethylases</span> <span class="citation_source-journal">Mol. Biosyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1039%2Fb912993f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20094655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosF2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=357-364&author=M.+Sakuraiauthor=N.+R.+Roseauthor=L.+Schultzauthor=A.+M.+Quinnauthor=A.+Jadhavauthor=S.+S.+Ngauthor=U.+Oppermannauthor=C.+J.+Schofieldauthor=A.+Simeonov&title=A+miniaturized+screen+for+inhibitors+of+Jumonji+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A miniaturized screen for inhibitors of Jumonji histone demethylases</span></div><div class="casAuthors">Sakurai, Masaaki; Rose, Nathan R.; Schultz, Lena; Quinn, Amy M.; Jadhav, Ajit; Ng, Stanley S.; Oppermann, Udo; Schofield, Christopher J.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">357-364</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">2-Oxoglutarate- and Fe(ii)-dependent oxygenases are a major class of N.vepsiln.-Me lysine demethylases that are involved in epigenetic regulation.  Assays suitable for implementation in a high-throughput manner have been lacking for these enzymes.  Here, we describe the design and implementation of a robust and miniaturized high-throughput kinetic assay for inhibitors of JMJD2E using a formaldehyde dehydrogenase-coupled reaction with real-time fluorescence detection.  Reactant compatibility studies resulted in simplification of the assay scheme to the mixing of two reagent solns., both of which were stable overnight.  The assay was miniaturized to a 4 μL vol. in 1536-well format and was used to screen the library of pharmacol. active compds. (LOPAC1280).  Inhibitors identified by the screen were further characterized in secondary assays including FDH counterscreen and demethylation assays that monitored demethylation by MALDI-TOF MS.  The assay developed here will enable the screening of large compd. libraries against the Jumonji demethylases in a robust and automated fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMwSVBOYmD4rVg90H21EOLACvtfcHk0ljs_wSXdl4TKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosF2ktA%253D%253D&md5=b71ca06c07ec8e438bf87d69f390a993</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2Fb912993f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb912993f%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DA%2520miniaturized%2520screen%2520for%2520inhibitors%2520of%2520Jumonji%2520histone%2520demethylases%26jtitle%3DMol.%2520Biosyst.%26date%3D2010%26volume%3D6%26spage%3D357%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span> <span class="citation_source-journal">Nature Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2212</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0ljs_wSXdl4TKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNature%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finerty, P. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ball, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hui, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigelt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span> </span><span class="NLM_article-title">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">15835</span><span class="NLM_x">–</span> <span class="NLM_lpage">15840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1073%2Fpnas.0605224103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17035505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=15835-15840&author=M.+Vedadiauthor=F.+H.+Niesenauthor=A.+Allali-Hassaniauthor=O.+Y.+Fedorovauthor=P.+J.+Finertyauthor=G.+A.+Wasneyauthor=R.+Yeungauthor=C.+Arrowsmithauthor=L.+J.+Ballauthor=H.+Berglundauthor=R.+Huiauthor=B.+D.+Marsdenauthor=P.+Nordlundauthor=M.+Sundstromauthor=J.+Weigeltauthor=A.+M.+Edwards&title=Chemical+screening+methods+to+identify+ligands+that+promote+protein+stability%2C+protein+crystallization%2C+and+structure+determination"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span></div><div class="casAuthors">Vedadi, Masoud; Niesen, Frank H.; Allali-Hassani, Abdellah; Fedorov, Oleg Y.; Finerty, Patrick J., Jr.; Wasney, Gregory A.; Yeung, Ron; Arrowsmith, Cheryl; Ball, Linda J.; Berglund, Helena; Hui, Raymond; Marsden, Brian D.; Nordlund, Paer; Sundstrom, Michael; Weigelt, Johan; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15835-15840</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 3D structures of human therapeutic targets are enabling for drug discovery.  However, their purifn. and crystn. remain rate detg.  In individual cases, ligands have been used to increase the success rate of protein purifn. and crystn., but the broad applicability of this approach is unknown.  The authors implemented two screening platforms, based on either fluorometry or static light scattering, to measure the increase in protein thermal stability upon binding of a ligand without the need to monitor enzyme activity.  In total, 221 different proteins from humans and human parasites were screened against one or both of two sorts of small-mol. libraries.  The first library comprised different salts, pH conditions, and commonly found small mols. and was applicable to all proteins.  The second comprised compds. specific for protein families of particular interest (e.g., protein kinases).  In 20 cases, including nine unique human protein kinases, a small mol. was identified that stabilized the proteins and promoted structure detn.  The methods are cost-effective, can be implemented in any lab., promise to increase the success rates of purifying and crystg. human proteins significantly, and identify new ligands for these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4duhcFpFn-bVg90H21EOLACvtfcHk0ljs_wSXdl4TKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN&md5=6515225c1613291deb7bb774d900f605</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0605224103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0605224103%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%2BY.%26aulast%3DFinerty%26aufirst%3DP.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DYeung%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBall%26aufirst%3DL.%2BJ.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DWeigelt%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DChemical%2520screening%2520methods%2520to%2520identify%2520ligands%2520that%2520promote%2520protein%2520stability%252C%2520protein%2520crystallization%252C%2520and%2520structure%2520determination%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2006%26volume%3D103%26spage%3D15835%26epage%3D15840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moult, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A survey of proteins encoded by nonsynonymous single nucleotide polymorphisms reveals a significant fraction with altered stability and activity</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">424</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2009&pages=15-26&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=I.+Chauauthor=B.+S.+Hongauthor=G.+Senisterraauthor=P.+Loppnauauthor=Z.+Shiauthor=J.+Moultauthor=A.+M.+Edwardsauthor=C.+H.+Arrowsmithauthor=H.+W.+Parkauthor=M.+Schapiraauthor=M.+Vedadi&title=A+survey+of+proteins+encoded+by+nonsynonymous+single+nucleotide+polymorphisms+reveals+a+significant+fraction+with+altered+stability+and+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DHong%26aufirst%3DB.%2BS.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DMoult%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520survey%2520of%2520proteins%2520encoded%2520by%2520nonsynonymous%2520single%2520nucleotide%2520polymorphisms%2520reveals%2520a%2520significant%2520fraction%2520with%2520altered%2520stability%2520and%2520activity%26jtitle%3DBiochem.%2520J.%26date%3D2009%26volume%3D424%26spage%3D15%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':[],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 143 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">M. Ramya Chandar Charles, Mu-Chun Li, Hsing-Pang Hsieh, <span class="NLM_string-name hlFld-ContribAuthor">Mohane Selvaraj Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Mimicking H3 Substrate Arginine in the Design of G9a Lysine Methyltransferase Inhibitors for Cancer Therapy: A Computational Study for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (9)
                                     , 6100-6111. <a href="https://doi.org/10.1021/acsomega.0c04710" title="DOI URL">https://doi.org/10.1021/acsomega.0c04710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c04710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c04710%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMimicking%252BH3%252BSubstrate%252BArginine%252Bin%252Bthe%252BDesign%252Bof%252BG9a%252BLysine%252BMethyltransferase%252BInhibitors%252Bfor%252BCancer%252BTherapy%25253A%252BA%252BComputational%252BStudy%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DChandar%2BCharles%26aufirst%3DM.%2BRamya%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24092020%26date%3D09022021%26date%3D22022021%26volume%3D6%26issue%3D9%26spage%3D6100%26epage%3D6111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Guangping Dong, Adam Yasgar, Darrell L. Peterson, Alexey Zakharov, Daniel Talley, Ken Chih-Chien Cheng, Ajit Jadhav, Anton Simeonov, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of High-Throughput Methyltransferase Assays for the Discovery of Small Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2020,</strong> <em>22 </em>
                                    (8)
                                     , 422-432. <a href="https://doi.org/10.1021/acscombsci.0c00077" title="DOI URL">https://doi.org/10.1021/acscombsci.0c00077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.0c00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.0c00077%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DOptimization%252Bof%252BHigh-Throughput%252BMethyltransferase%252BAssays%252Bfor%252Bthe%252BDiscovery%252Bof%252BSmall%252BMolecule%252BInhibitors%26aulast%3DDong%26aufirst%3DGuangping%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D10062020%26date%3D27062020%26date%3D11062020%26volume%3D22%26issue%3D8%26spage%3D422%26epage%3D432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ciro Milite, Alessandra Feoli, John R. Horton, Donatella Rescigno, Alessandra Cipriano, Vincenzo Pisapia, Monica Viviano, Giacomo Pepe, Giorgio Amendola, Ettore Novellino, Sandro Cosconati, Xiaodong Cheng, Sabrina Castellano, <span class="NLM_string-name hlFld-ContribAuthor">Gianluca Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Novel Chemotype of Histone Lysine Methyltransferase EHMT1/2 (GLP/G9a) Inhibitors: Rational Design, Synthesis, Biological Evaluation, and Co-crystal Structure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (5)
                                     , 2666-2689. <a href="https://doi.org/10.1021/acs.jmedchem.8b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNovel%252BChemotype%252Bof%252BHistone%252BLysine%252BMethyltransferase%252BEHMT1%25252F2%252B%252528GLP%25252FG9a%252529%252BInhibitors%25253A%252BRational%252BDesign%25252C%252BSynthesis%25252C%252BBiological%252BEvaluation%25252C%252Band%252BCo-crystal%252BStructure%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D21122018%26date%3D26022019%26date%3D12022019%26volume%3D62%26issue%3D5%26spage%3D2666%26epage%3D2689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Obdulia Rabal, Juan Antonio Sánchez-Arias, Edurne San José-Enériz, Xabier Agirre, Irene de Miguel, Leire Garate, Estibaliz Miranda, Elena Sáez, Sergio Roa, José Angel Martínez-Climent, Yingying Liu, Wei Wu, Musheng Xu, Felipe Prosper, <span class="NLM_string-name hlFld-ContribAuthor">Julen Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6546-6573. <a href="https://doi.org/10.1021/acs.jmedchem.7b01925" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01925</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01925%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDetailed%252BExploration%252Baround%252B4-Aminoquinolines%252BChemical%252BSpace%252Bto%252BNavigate%252Bthe%252BLysine%252BMethyltransferase%252BG9a%252Band%252BDNA%252BMethyltransferase%252BBiological%252BSpaces%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D30122017%26date%3D19072018%26date%3D11062018%26volume%3D61%26issue%3D15%26spage%3D6546%26epage%3D6573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Obdulia Rabal, Edurne San José-Enériz, Xabier Agirre, Juan Antonio Sánchez-Arias, Amaia Vilas-Zornoza, Ana Ugarte, Irene de Miguel, Estíbaliz Miranda, Leire Garate, Mario Fraga, Pablo Santamarina, Raul Fernandez Perez, Raquel Ordoñez, Elena Sáez, Sergio Roa, María José García-Barchino, José Angel Martínez-Climent, Yingying Liu, Wei Wu, Musheng Xu, Felipe Prosper, <span class="NLM_string-name hlFld-ContribAuthor">Julen Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6518-6545. <a href="https://doi.org/10.1021/acs.jmedchem.7b01926" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01926</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BReversible%252BDNA%252BMethyltransferase%252Band%252BLysine%252BMethyltransferase%252BG9a%252BInhibitors%252Bwith%252BAntitumoral%252Bin%252BVivo%252BEfficacy%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D30122017%26date%3D19072018%26date%3D28062018%26volume%3D61%26issue%3D15%26spage%3D6518%26epage%3D6545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent and Selective Inhibitors for G9a-Like Protein (GLP) Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1876-1891. <a href="https://doi.org/10.1021/acs.jmedchem.6b01645" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01645</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01645%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252Band%252BSelective%252BInhibitors%252Bfor%252BG9a-Like%252BProtein%252B%252528GLP%252529%252BLysine%252BMethyltransferase%26aulast%3DXiong%26aufirst%3DYan%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D15112016%26date%3D14022017%26date%3D09032017%26date%3D30012017%26volume%3D60%26issue%3D5%26spage%3D1876%26epage%3D1891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Wolfram  Tempel</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Fabio  Pittella-Silva</span>, <span class="hlFld-ContribAuthor ">Jason  Shao</span>, <span class="hlFld-ContribAuthor ">Junyi  Wang</span>, <span class="hlFld-ContribAuthor ">Minkui  Luo</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (21)
                                     , 9881-9889. <a href="https://doi.org/10.1021/acs.jmedchem.6b01244" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BCovalent%252BInhibitor%252Bof%252Bthe%252BSET%252BDomain-Containing%252BProtein%252B8%252B%252528SETD8%252529%252BLysine%252BMethyltransferase%26aulast%3DButler%26aufirst%3DKyle%2BV.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D16082016%26date%3D02112016%26date%3D10112016%26volume%3D59%26issue%3D21%26spage%3D9881%26epage%3D9889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span> and <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>  . </span><span class="cited-content_cbyCitation_article-title">Protein Methyltransferases: A Distinct, Diverse, and Dynamic Family of Enzymes. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2016,</strong> <em>55 </em>
                                    (11)
                                     , 1557-1569. <a href="https://doi.org/10.1021/acs.biochem.5b01129" title="DOI URL">https://doi.org/10.1021/acs.biochem.5b01129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.5b01129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.5b01129%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DProtein%252BMethyltransferases%25253A%252BA%252BDistinct%25252C%252BDiverse%25252C%252Band%252BDynamic%252BFamily%252Bof%252BEnzymes%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BAnn%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D16102015%26date%3D09122015%26date%3D22122015%26date%3D22032016%26date%3D10122015%26volume%3D55%26issue%3D11%26spage%3D1557%26epage%3D1569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Selective Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (4)
                                     , 1596-1629. <a href="https://doi.org/10.1021/jm501234a" title="DOI URL">https://doi.org/10.1021/jm501234a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm501234a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D12082014%26date%3D02122014%26date%3D26022015%26date%3D19112014%26volume%3D58%26issue%3D4%26spage%3D1596%26epage%3D1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span> and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Chemical Probes of Histone Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2015,</strong> <em>10 </em>
                                    (1)
                                     , 40-50. <a href="https://doi.org/10.1021/cb500785t" title="DOI URL">https://doi.org/10.1021/cb500785t</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb500785t%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemical%252BProbes%252Bof%252BHistone%252BLysine%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D30092014%26date%3D17112014%26date%3D25112014%26date%3D16012015%26volume%3D10%26issue%3D1%26spage%3D40%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Rachel M.  Bleich</span>, <span class="hlFld-ContribAuthor ">J. Martin  Herold</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Victoria  Korboukh</span>, <span class="hlFld-ContribAuthor ">Ashutosh  Tripathy</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (15)
                                     , 6822-6833. <a href="https://doi.org/10.1021/jm500871s" title="DOI URL">https://doi.org/10.1021/jm500871s</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500871s%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BSelective%25252C%252BSubstrate-Competitive%252BInhibitor%252Bof%252Bthe%252BLysine%252BMethyltransferase%252BSETD8%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D09062014%26date%3D25072014%26date%3D14082014%26date%3D17072014%26volume%3D57%26issue%3D15%26spage%3D6822%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kanchan  Devkota</span>, <span class="hlFld-ContribAuthor ">Brian  Lohse</span>, <span class="hlFld-ContribAuthor ">Qing  Liu</span>, <span class="hlFld-ContribAuthor ">Ming-Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Dan  Stærk</span>, <span class="hlFld-ContribAuthor ">Jens  Berthelsen</span>, and <span class="hlFld-ContribAuthor ">Rasmus Prætorius  Clausen</span>  . </span><span class="cited-content_cbyCitation_article-title">Analogues of the Natural Product Sinefungin as Inhibitors of EHMT1 and EHMT2. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>5 </em>
                                    (4)
                                     , 293-297. <a href="https://doi.org/10.1021/ml4002503" title="DOI URL">https://doi.org/10.1021/ml4002503</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml4002503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml4002503%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DAnalogues%252Bof%252Bthe%252BNatural%252BProduct%252BSinefungin%252Bas%252BInhibitors%252Bof%252BEHMT1%252Band%252BEHMT2%26aulast%3DDevkota%26aufirst%3DKanchan%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D03072013%26date%3D31012014%26date%3D04022014%26date%3D10042014%26date%3D31012014%26volume%3D5%26issue%3D4%26spage%3D293%26epage%3D297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ramzi F. Sweis, Marina Pliushchev, Peter J. Brown, Jun Guo, Fengling Li, David Maag, Andrew M. Petros, Nirupama B. Soni, Chris Tse, Masoud Vedadi, Michael R. Michaelides, Gary G. Chiang, <span class="NLM_string-name hlFld-ContribAuthor">William N. Pappano</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>5 </em>
                                    (2)
                                     , 205-209. <a href="https://doi.org/10.1021/ml400496h" title="DOI URL">https://doi.org/10.1021/ml400496h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml400496h%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252BDevelopment%252Bof%252BPotent%252Band%252BSelective%252BInhibitors%252Bof%252BHistone%252BMethyltransferase%252BG9a%26aulast%3DSweis%26aufirst%3DRamzi%2BF.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D03122013%26date%3D02012014%26date%3D09012014%26date%3D13022014%26date%3D02012014%26volume%3D5%26issue%3D2%26spage%3D205%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Victoria  Korboukh</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Abdellah  Allali-Hassani</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of an in Vivo Chemical Probe of the Lysine Methyltransferases G9a and GLP. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (21)
                                     , 8931-8942. <a href="https://doi.org/10.1021/jm401480r" title="DOI URL">https://doi.org/10.1021/jm401480r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401480r%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ban%252Bin%252BVivo%252BChemical%252BProbe%252Bof%252Bthe%252BLysine%252BMethyltransferases%252BG9a%252Band%252BGLP%26aulast%3DLiu%26aufirst%3DFeng%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D24092013%26date%3D31102013%26date%3D14112013%26date%3D08102013%26volume%3D56%26issue%3D21%26spage%3D8931%26epage%3D8942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Trevor  Parton</span>, <span class="hlFld-ContribAuthor ">Christopher J.  MacNevin</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Kuznetsova</span>, <span class="hlFld-ContribAuthor ">Marie  Rougie</span>, <span class="hlFld-ContribAuthor ">Alice  Jiang</span>, <span class="hlFld-ContribAuthor ">Samantha G.  Pattenden</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Klaus M.  Hahn</span>, <span class="hlFld-ContribAuthor ">Gang Greg  Wang</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2013,</strong> <em>8 </em>
                                    (6)
                                     , 1324-1334. <a href="https://doi.org/10.1021/cb400133j" title="DOI URL">https://doi.org/10.1021/cb400133j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb400133j%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAn%252BOrally%252BBioavailable%252BChemical%252BProbe%252Bof%252Bthe%252BLysine%252BMethyltransferases%252BEZH2%252Band%252BEZH1%26aulast%3DKonze%26aufirst%3DKyle%2BD.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D22022013%26date%3D08042013%26date%3D24042013%26date%3D21062013%26date%3D08042013%26volume%3D8%26issue%3D6%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Elena  Dobrovetsky</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Ilia  Korboukh</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (5)
                                     , 2110-2124. <a href="https://doi.org/10.1021/jm3018332" title="DOI URL">https://doi.org/10.1021/jm3018332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm3018332%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExploiting%252Ban%252BAllosteric%252BBinding%252BSite%252Bof%252BPRMT3%252BYields%252BPotent%252Band%252BSelective%252BInhibitors%26aulast%3DLiu%26aufirst%3DFeng%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D12122012%26date%3D27022013%26date%3D14032013%26volume%3D56%26issue%3D5%26spage%3D2110%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sharad K.  Verma</span>, <span class="hlFld-ContribAuthor ">Xinrong  Tian</span>, <span class="hlFld-ContribAuthor ">Louis V.  LaFrance</span>, <span class="hlFld-ContribAuthor ">Céline  Duquenne</span>, <span class="hlFld-ContribAuthor ">Dominic P.  Suarez</span>, <span class="hlFld-ContribAuthor ">Kenneth A.  Newlander</span>, <span class="hlFld-ContribAuthor ">Stuart P.  Romeril</span>, <span class="hlFld-ContribAuthor ">Joelle L.  Burgess</span>, <span class="hlFld-ContribAuthor ">Seth W.  Grant</span>, <span class="hlFld-ContribAuthor ">James A.  Brackley</span>, <span class="hlFld-ContribAuthor ">Alan P.  Graves</span>, <span class="hlFld-ContribAuthor ">Daryl A.  Scherzer</span>, <span class="hlFld-ContribAuthor ">Art  Shu</span>, <span class="hlFld-ContribAuthor ">Christine  Thompson</span>, <span class="hlFld-ContribAuthor ">Heidi M.  Ott</span>, <span class="hlFld-ContribAuthor ">Glenn S. Van  Aller</span>, <span class="hlFld-ContribAuthor ">Carl A.  Machutta</span>, <span class="hlFld-ContribAuthor ">Elsie  Diaz</span>, <span class="hlFld-ContribAuthor ">Yong  Jiang</span>, <span class="hlFld-ContribAuthor ">Neil W.  Johnson</span>, <span class="hlFld-ContribAuthor ">Steven D.  Knight</span>, <span class="hlFld-ContribAuthor ">Ryan G.  Kruger</span>, <span class="hlFld-ContribAuthor ">Michael T.  McCabe</span>, <span class="hlFld-ContribAuthor ">Dashyant  Dhanak</span>, <span class="hlFld-ContribAuthor ">Peter J.  Tummino</span>, <span class="hlFld-ContribAuthor ">Caretha L.  Creasy</span>, and <span class="hlFld-ContribAuthor ">William H.  Miller</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2012,</strong> <em>3 </em>
                                    (12)
                                     , 1091-1096. <a href="https://doi.org/10.1021/ml3003346" title="DOI URL">https://doi.org/10.1021/ml3003346</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml3003346%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252BPotent%25252C%252BSelective%25252C%252BCell-Active%252BInhibitors%252Bof%252Bthe%252BHistone%252BLysine%252BMethyltransferase%252BEZH2%26aulast%3DVerma%26aufirst%3DSharad%2BK.%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D1162012%26date%3D11102012%26date%3D19102012%26date%3D13122012%26volume%3D3%26issue%3D12%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Justin L.  Anglin</span>, <span class="hlFld-ContribAuthor ">Lisheng  Deng</span>, <span class="hlFld-ContribAuthor ">Yuan  Yao</span>, <span class="hlFld-ContribAuthor ">Guobin  Cai</span>, <span class="hlFld-ContribAuthor ">Zhen  Liu</span>, <span class="hlFld-ContribAuthor ">Hong  Jiang</span>, <span class="hlFld-ContribAuthor ">Gang  Cheng</span>, <span class="hlFld-ContribAuthor ">Pinhong  Chen</span>, <span class="hlFld-ContribAuthor ">Shuo  Dong</span>, and <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>  . </span><span class="cited-content_cbyCitation_article-title">Synthesis and Structure–Activity Relationship Investigation of Adenosine-Containing Inhibitors of Histone Methyltransferase DOT1L. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (18)
                                     , 8066-8074. <a href="https://doi.org/10.1021/jm300917h" title="DOI URL">https://doi.org/10.1021/jm300917h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm300917h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm300917h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BInvestigation%252Bof%252BAdenosine-Containing%252BInhibitors%252Bof%252BHistone%252BMethyltransferase%252BDOT1L%26aulast%3DAnglin%26aufirst%3DJustin%2BL.%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D25062012%26date%3D06092012%26date%3D27092012%26date%3D27082012%26volume%3D55%26issue%3D18%26spage%3D8066%26epage%3D8074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Chao</span>, <span class="hlFld-ContribAuthor ">Xian-Kun  Tong</span>, <span class="hlFld-ContribAuthor ">Wei  Tang</span>, <span class="hlFld-ContribAuthor ">De-Wen  Li</span>, <span class="hlFld-ContribAuthor ">Pei-Lan  He</span>, <span class="hlFld-ContribAuthor ">Jean-Michel  Garcia</span>, <span class="hlFld-ContribAuthor ">Li-Min  Zeng</span>, <span class="hlFld-ContribAuthor ">An-Hui  Gao</span>, <span class="hlFld-ContribAuthor ">Li  Yang</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Fa-Jun  Nan</span>, <span class="hlFld-ContribAuthor ">Michael  Jacobs</span>, <span class="hlFld-ContribAuthor ">Ralf  Altmeyer</span>, <span class="hlFld-ContribAuthor ">Jian-Ping  Zuo</span>, and <span class="hlFld-ContribAuthor ">You-Hong  Hu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of 2,4-Diaminoquinazoline Derivatives as a New Class of Potent Dengue Virus Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (7)
                                     , 3135-3143. <a href="https://doi.org/10.1021/jm2015952" title="DOI URL">https://doi.org/10.1021/jm2015952</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm2015952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm2015952%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252B2%25252C4-Diaminoquinazoline%252BDerivatives%252Bas%252Ba%252BNew%252BClass%252Bof%252BPotent%252BDengue%252BVirus%252BInhibitors%26aulast%3DChao%26aufirst%3DBo%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D25112011%26date%3D26032012%26date%3D12042012%26volume%3D55%26issue%3D7%26spage%3D3135%26epage%3D3143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minkui  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Current Chemical Biology Approaches to Interrogate Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2012,</strong> <em>7 </em>
                                    (3)
                                     , 443-463. <a href="https://doi.org/10.1021/cb200519y" title="DOI URL">https://doi.org/10.1021/cb200519y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb200519y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb200519y%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DCurrent%252BChemical%252BBiology%252BApproaches%252Bto%252BInterrogate%252BProtein%252BMethyltransferases%26aulast%3DLuo%26aufirst%3DMinkui%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D07122011%26date%3D05012012%26date%3D01022012%26date%3D16032012%26date%3D05012012%26volume%3D7%26issue%3D3%26spage%3D443%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaushik  Chanda</span>, <span class="hlFld-ContribAuthor ">Barnali  Maiti</span>, <span class="hlFld-ContribAuthor ">Chih-Chung  Tseng</span>, and <span class="hlFld-ContribAuthor ">Chung-Ming  Sun</span>  . </span><span class="cited-content_cbyCitation_article-title">Microwave-Assisted Linear Approach Toward Highly Substituted Benzo[d]oxazol-5-yl-1H-benzo[d]imidazole on Ionic Liquid Support. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2012,</strong> <em>14 </em>
                                    (2)
                                     , 115-123. <a href="https://doi.org/10.1021/co200188g" title="DOI URL">https://doi.org/10.1021/co200188g</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/co200188g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fco200188g%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DMicrowave-Assisted%252BLinear%252BApproach%252BToward%252BHighly%252BSubstituted%252BBenzo%25255Bd%25255Doxazol-5-yl-1H-benzo%25255Bd%25255Dimidazole%252Bon%252BIonic%252BLiquid%252BSupport%26aulast%3DChanda%26aufirst%3DKaushik%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D2012%26date%3D09112011%26date%3D11012012%26date%3D20012012%26date%3D13022012%26volume%3D14%26issue%3D2%26spage%3D115%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Abdellah  Allali-Hassani</span>, <span class="hlFld-ContribAuthor ">Yunlong  He</span>, <span class="hlFld-ContribAuthor ">J. Martin  Herold</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Yates</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Optimization of Cellular Activity of G9a Inhibitors 7-Aminoalkoxy-quinazolines. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (17)
                                     , 6139-6150. <a href="https://doi.org/10.1021/jm200903z" title="DOI URL">https://doi.org/10.1021/jm200903z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm200903z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252BCellular%252BActivity%252Bof%252BG9a%252BInhibitors%252B7-Aminoalkoxy-quinazolines%26aulast%3DLiu%26aufirst%3DFeng%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D10072011%26date%3D05082011%26date%3D08092011%26date%3D22072011%26volume%3D54%26issue%3D17%26spage%3D6139%26epage%3D6150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Martin  Herold</span>, <span class="hlFld-ContribAuthor ">Tim J.  Wigle</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Robert  Lam</span>, <span class="hlFld-ContribAuthor ">Victoria K.  Korboukh</span>, <span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Lindsey A.  Ingerman</span>, <span class="hlFld-ContribAuthor ">Dmitri B.  Kireev</span>, <span class="hlFld-ContribAuthor ">Guillermo  Senisterra</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Ashutosh  Tripathy</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, and <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>  . </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Ligands of Methyl-Lysine Binding Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (7)
                                     , 2504-2511. <a href="https://doi.org/10.1021/jm200045v" title="DOI URL">https://doi.org/10.1021/jm200045v</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm200045v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm200045v%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BLigands%252Bof%252BMethyl-Lysine%252BBinding%252BProteins%26aulast%3DHerold%26aufirst%3DJ.%2BMartin%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D16012011%26date%3D18032011%26date%3D14042011%26volume%3D54%26issue%3D7%26spage%3D2504%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valérie  Campagna-Slater</span>, <span class="hlFld-ContribAuthor ">Man Wai  Mok</span>, <span class="hlFld-ContribAuthor ">Kong T.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Miklos  Feher</span>, <span class="hlFld-ContribAuthor ">Rafael  Najmanovich</span>, and <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>  . </span><span class="cited-content_cbyCitation_article-title">Structural Chemistry of the Histone Methyltransferases Cofactor Binding Site. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (3)
                                     , 612-623. <a href="https://doi.org/10.1021/ci100479z" title="DOI URL">https://doi.org/10.1021/ci100479z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100479z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100479z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructural%252BChemistry%252Bof%252Bthe%252BHistone%252BMethyltransferases%252BCofactor%252BBinding%252BSite%26aulast%3DCampagna-Slater%26aufirst%3DVal%25C3%25A9rie%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D06122010%26date%3D03032011%26date%3D28032011%26volume%3D51%26issue%3D3%26spage%3D612%26epage%3D623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan H  Trager</span>, <span class="hlFld-ContribAuthor ">Bindeshwar  Sah</span>, <span class="hlFld-ContribAuthor ">Zhongming  Chen</span>, <span class="hlFld-ContribAuthor ">Liang  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Control of Breast Cancer Pathogenesis by Histone Methylation and the Hairless Histone Demethylase. </span><span class="cited-content_cbyCitation_journal-name">Endocrinology</span><span> <strong>2021,</strong> <em>162 </em>
                                    (8)
                                     <a href="https://doi.org/10.1210/endocr/bqab088" title="DOI URL">https://doi.org/10.1210/endocr/bqab088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1210/endocr/bqab088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1210%2Fendocr%2Fbqab088%26sid%3Dliteratum%253Aachs%26jtitle%3DEndocrinology%26atitle%3DControl%252Bof%252BBreast%252BCancer%252BPathogenesis%252Bby%252BHistone%252BMethylation%252Band%252Bthe%252BHairless%252BHistone%252BDemethylase%26aulast%3DTrager%26aufirst%3DMegan%2BH%26date%3D2021%26date%3D2021%26volume%3D162%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert B. A.  Quinlan</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomics for drug discovery: clinical molecules from open access chemical probes. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>2 </em>
                                    (3)
                                     , 759-795. <a href="https://doi.org/10.1039/D1CB00016K" title="DOI URL">https://doi.org/10.1039/D1CB00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CB00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CB00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DChemogenomics%252Bfor%252Bdrug%252Bdiscovery%25253A%252Bclinical%252Bmolecules%252Bfrom%252Bopen%252Baccess%252Bchemical%252Bprobes%26aulast%3DQuinlan%26aufirst%3DRobert%2BB.%2BA.%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D3%26spage%3D759%26epage%3D795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bratati  Saha</span>, <span class="hlFld-ContribAuthor ">Robin J.  Parks</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of human adenovirus replication inhibitors from a library of small molecules targeting cellular epigenetic regulators. </span><span class="cited-content_cbyCitation_journal-name">Virology</span><span> <strong>2021,</strong> <em>555 </em>, 102-110. <a href="https://doi.org/10.1016/j.virol.2020.07.007" title="DOI URL">https://doi.org/10.1016/j.virol.2020.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.virol.2020.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.virol.2020.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DVirology%26atitle%3DIdentification%252Bof%252Bhuman%252Badenovirus%252Breplication%252Binhibitors%252Bfrom%252Ba%252Blibrary%252Bof%252Bsmall%252Bmolecules%252Btargeting%252Bcellular%252Bepigenetic%252Bregulators%26aulast%3DSaha%26aufirst%3DBratati%26date%3D2021%26volume%3D555%26spage%3D102%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Suraya  Jan</span>, <span class="hlFld-ContribAuthor ">Mohd Ishaq  Dar</span>, <span class="hlFld-ContribAuthor ">Rubiada  Wani</span>, <span class="hlFld-ContribAuthor ">Jagjeet  Sandey</span>, <span class="hlFld-ContribAuthor ">Iqra  Mushtaq</span>, <span class="hlFld-ContribAuthor ">Sammar  Lateef</span>, <span class="hlFld-ContribAuthor ">Sajad Hussain  Syed</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting EHMT2/ G9a for cancer therapy: Progress and perspective. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>893 </em>, 173827. <a href="https://doi.org/10.1016/j.ejphar.2020.173827" title="DOI URL">https://doi.org/10.1016/j.ejphar.2020.173827</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2020.173827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2020.173827%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DTargeting%252BEHMT2%25252F%252BG9a%252Bfor%252Bcancer%252Btherapy%25253A%252BProgress%252Band%252Bperspective%26aulast%3DJan%26aufirst%3DSuraya%26date%3D2021%26volume%3D893%26spage%3D173827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Pisapia</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Tanja  Bartoschik</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Microscale Thermophoresis-Based Method for Screening and Characterizing Inhibitors of the Methyl-Lysine Reader Protein MRG15. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (1)
                                     , 77-87. <a href="https://doi.org/10.1177/2472555220949166" title="DOI URL">https://doi.org/10.1177/2472555220949166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220949166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220949166%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DDevelopment%252Bof%252Ba%252BMicroscale%252BThermophoresis-Based%252BMethod%252Bfor%252BScreening%252Band%252BCharacterizing%252BInhibitors%252Bof%252Bthe%252BMethyl-Lysine%252BReader%252BProtein%252BMRG15%26aulast%3DFeoli%26aufirst%3DAlessandra%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D1%26spage%3D77%26epage%3D87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Reiner</span>, <span class="hlFld-ContribAuthor ">Ludwig  Seifert</span>, <span class="hlFld-ContribAuthor ">Caroline  Deck</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Holger  Stark</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics meets GPCR: inhibition of histone H3 methyltransferase (G9a) and histamine H3 receptor for Prader–Willi Syndrome. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-70523-y" title="DOI URL">https://doi.org/10.1038/s41598-020-70523-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-70523-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-70523-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DEpigenetics%252Bmeets%252BGPCR%25253A%252Binhibition%252Bof%252Bhistone%252BH3%252Bmethyltransferase%252B%252528G9a%252529%252Band%252Bhistamine%252BH3%252Breceptor%252Bfor%252BPrader%2525E2%252580%252593Willi%252BSyndrome%26aulast%3DReiner%26aufirst%3DDavid%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zi-Jun  Wang</span>, <span class="hlFld-ContribAuthor ">Ping  Zhong</span>, <span class="hlFld-ContribAuthor ">Kaijie  Ma</span>, <span class="hlFld-ContribAuthor ">Ji-Seon  Seo</span>, <span class="hlFld-ContribAuthor ">Fengwei  Yang</span>, <span class="hlFld-ContribAuthor ">Zihua  Hu</span>, <span class="hlFld-ContribAuthor ">Freddy  Zhang</span>, <span class="hlFld-ContribAuthor ">Lin  Lin</span>, <span class="hlFld-ContribAuthor ">Jie  Wang</span>, <span class="hlFld-ContribAuthor ">Tao  Liu</span>, <span class="hlFld-ContribAuthor ">Emmanuel  Matas</span>, <span class="hlFld-ContribAuthor ">Paul  Greengard</span>, <span class="hlFld-ContribAuthor ">Zhen  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Amelioration of autism-like social deficits by targeting histone methyltransferases EHMT1/2 in Shank3-deficient mice. </span><span class="cited-content_cbyCitation_journal-name">Molecular Psychiatry</span><span> <strong>2020,</strong> <em>25 </em>
                                    (10)
                                     , 2517-2533. <a href="https://doi.org/10.1038/s41380-019-0351-2" title="DOI URL">https://doi.org/10.1038/s41380-019-0351-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41380-019-0351-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41380-019-0351-2%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Psychiatry%26atitle%3DAmelioration%252Bof%252Bautism-like%252Bsocial%252Bdeficits%252Bby%252Btargeting%252Bhistone%252Bmethyltransferases%252BEHMT1%25252F2%252Bin%252BShank3-deficient%252Bmice%26aulast%3DWang%26aufirst%3DZi-Jun%26date%3D2020%26date%3D2019%26volume%3D25%26issue%3D10%26spage%3D2517%26epage%3D2533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging histone methyltransferases in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>56 </em>, 51-62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.11.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.11.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bhistone%252Bmethyltransferases%252Bin%252Bcancer%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2020%26volume%3D56%26spage%3D51%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shabir Ahmad  Ganai</span>. </span><span class="cited-content_cbyCitation_article-title">Modulating Epigenetic Modification Enzymes Through Relevant Epidrugs as a Timely Strategy in Anticancer Therapy. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 137-157. <a href="https://doi.org/10.1007/978-981-15-8179-3_7" title="DOI URL">https://doi.org/10.1007/978-981-15-8179-3_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-15-8179-3_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-15-8179-3_7%26sid%3Dliteratum%253Aachs%26atitle%3DModulating%252BEpigenetic%252BModification%252BEnzymes%252BThrough%252BRelevant%252BEpidrugs%252Bas%252Ba%252BTimely%252BStrategy%252Bin%252BAnticancer%252BTherapy%26aulast%3DGanai%26aufirst%3DShabir%2BAhmad%26date%3D2020%26date%3D2020%26spage%3D137%26epage%3D157%26pub%3DSpringer%2520Singapore%26atitle%3DHistone%252BDeacetylase%252BInhibitors%252Bin%252BCombinatorial%252BAnticancer%252BTherapy%26aulast%3DGanai%26aufirst%3DShabir%2BAhmad%26date%3D2020%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. Ramya Chandar  Charles</span>, <span class="hlFld-ContribAuthor ">Arun  Mahesh</span>, <span class="hlFld-ContribAuthor ">Shu-Yu  Lin</span>, <span class="hlFld-ContribAuthor ">Hsing-Pang  Hsieh</span>, <span class="hlFld-ContribAuthor ">Arunkumar  Dhayalan</span>, <span class="hlFld-ContribAuthor ">Mohane Selvaraj  Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel quinoline inhibitor for EHMT2/G9a through virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Biochimie</span><span> <strong>2020,</strong> <em>168 </em>, 220-230. <a href="https://doi.org/10.1016/j.biochi.2019.11.006" title="DOI URL">https://doi.org/10.1016/j.biochi.2019.11.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biochi.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biochi.2019.11.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimie%26atitle%3DIdentification%252Bof%252Bnovel%252Bquinoline%252Binhibitor%252Bfor%252BEHMT2%25252FG9a%252Bthrough%252Bvirtual%252Bscreening%26aulast%3DCharles%26aufirst%3DM.%2BRamya%2BChandar%26date%3D2020%26volume%3D168%26spage%3D220%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adrian J.  Carter</span>, <span class="hlFld-ContribAuthor ">Raina  Seupel</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Michael  Sundström</span>, <span class="hlFld-ContribAuthor ">Andrea  Introini</span>, <span class="hlFld-ContribAuthor ">Anke  Mueller‐Fahrnow</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Strategies for Evaluating New Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 1-24. <a href="https://doi.org/10.1002/9783527818242.ch1" title="DOI URL">https://doi.org/10.1002/9783527818242.ch1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527818242.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527818242.ch1%26sid%3Dliteratum%253Aachs%26atitle%3DChemical%252BStrategies%252Bfor%252BEvaluating%252BNew%252BDrug%252BTargets%26aulast%3DCarter%26aufirst%3DAdrian%2BJ.%26date%3D2019%26date%3D2019%26spage%3D1%26epage%3D24%26pub%3DWiley%26atitle%3DTarget%252BDiscovery%252Band%252BValidation%26aulast%3DPlowright%26aufirst%3DAlleyn%2BT%26date%3D2019%26date%3D2019%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Cao</span>, <span class="hlFld-ContribAuthor ">Ling  Li</span>, <span class="hlFld-ContribAuthor ">Deying  Yang</span>, <span class="hlFld-ContribAuthor ">Liming  Zeng</span>, <span class="hlFld-ContribAuthor ">Xie  Yewei</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Jianjun  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in histone methyltransferase (G9a) inhibitors as anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>179 </em>, 537-546. <a href="https://doi.org/10.1016/j.ejmech.2019.06.072" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.06.072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.06.072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.06.072%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bin%252Bhistone%252Bmethyltransferase%252B%252528G9a%252529%252Binhibitors%252Bas%252Banticancer%252Bagents%26aulast%3DCao%26aufirst%3DHao%26date%3D2019%26volume%3D179%26spage%3D537%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruben  Leenders</span>, <span class="hlFld-ContribAuthor ">Remco  Zijlmans</span>, <span class="hlFld-ContribAuthor ">Bart  van Bree</span>, <span class="hlFld-ContribAuthor ">Marc  van de Sande</span>, <span class="hlFld-ContribAuthor ">Federica  Trivarelli</span>, <span class="hlFld-ContribAuthor ">Eddy  Damen</span>, <span class="hlFld-ContribAuthor ">Anita  Wegert</span>, <span class="hlFld-ContribAuthor ">Daniel  Müller</span>, <span class="hlFld-ContribAuthor ">Jan Erik  Ehlert</span>, <span class="hlFld-ContribAuthor ">Daniel  Feger</span>, <span class="hlFld-ContribAuthor ">Carolin  Heidemann-Dinger</span>, <span class="hlFld-ContribAuthor ">Michael  Kubbutat</span>, <span class="hlFld-ContribAuthor ">Christoph  Schächtele</span>, <span class="hlFld-ContribAuthor ">Danny C.  Lenstra</span>, <span class="hlFld-ContribAuthor ">Jasmin  Mecinović</span>, <span class="hlFld-ContribAuthor ">Gerhard  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Novel SAR for quinazoline inhibitors of EHMT1 and EHMT2. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (17)
                                     , 2516-2524. <a href="https://doi.org/10.1016/j.bmcl.2019.06.012" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.06.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.06.012%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNovel%252BSAR%252Bfor%252Bquinazoline%252Binhibitors%252Bof%252BEHMT1%252Band%252BEHMT2%26aulast%3DLeenders%26aufirst%3DRuben%26date%3D2019%26volume%3D29%26issue%3D17%26spage%3D2516%26epage%3D2524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elif Kubat  Öktem</span>, <span class="hlFld-ContribAuthor ">Metin  Yazar</span>, <span class="hlFld-ContribAuthor ">Gizem  Gulfidan</span>, <span class="hlFld-ContribAuthor ">Kazim Yalcin  Arga</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer Drug Repositioning by Comparison of Gene Expression in Humans and Axolotl (
              Ambystoma mexicanum
              ) During Wound Healing. </span><span class="cited-content_cbyCitation_journal-name">OMICS: A Journal of Integrative Biology</span><span> <strong>2019,</strong> <em>23 </em>
                                    (8)
                                     , 389-405. <a href="https://doi.org/10.1089/omi.2019.0093" title="DOI URL">https://doi.org/10.1089/omi.2019.0093</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/omi.2019.0093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fomi.2019.0093%26sid%3Dliteratum%253Aachs%26jtitle%3DOMICS%253A%2520A%2520Journal%2520of%2520Integrative%2520Biology%26atitle%3DCancer%252BDrug%252BRepositioning%252Bby%252BComparison%252Bof%252BGene%252BExpression%252Bin%252BHumans%252Band%252BAxolotl%252B%252528%252BAmbystoma%252Bmexicanum%252B%252529%252BDuring%252BWound%252BHealing%26aulast%3D%25C3%2596ktem%26aufirst%3DElif%2BKubat%26date%3D2019%26volume%3D23%26issue%3D8%26spage%3D389%26epage%3D405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahima  Sharma</span>, <span class="hlFld-ContribAuthor ">Sreedharan  Sajikumar</span>. </span><span class="cited-content_cbyCitation_article-title">G9a/GLP Complex Acts as a Bidirectional Switch to Regulate Metabotropic Glutamate Receptor-Dependent Plasticity in Hippocampal CA1 Pyramidal Neurons. </span><span class="cited-content_cbyCitation_journal-name">Cerebral Cortex</span><span> <strong>2019,</strong> <em>29 </em>
                                    (7)
                                     , 2932-2946. <a href="https://doi.org/10.1093/cercor/bhy161" title="DOI URL">https://doi.org/10.1093/cercor/bhy161</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/cercor/bhy161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fcercor%2Fbhy161%26sid%3Dliteratum%253Aachs%26jtitle%3DCerebral%2520Cortex%26atitle%3DG9a%25252FGLP%252BComplex%252BActs%252Bas%252Ba%252BBidirectional%252BSwitch%252Bto%252BRegulate%252BMetabotropic%252BGlutamate%252BReceptor-Dependent%252BPlasticity%252Bin%252BHippocampal%252BCA1%252BPyramidal%252BNeurons%26aulast%3DSharma%26aufirst%3DMahima%26date%3D2019%26date%3D2018%26volume%3D29%26issue%3D7%26spage%3D2932%26epage%3D2946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariasoosai Ramya Chandar  Charles</span>, <span class="hlFld-ContribAuthor ">Arunkumar  Dhayalan</span>, <span class="hlFld-ContribAuthor ">Hsing-Pang  Hsieh</span>, <span class="hlFld-ContribAuthor ">Mohane Selvaraj  Coumar</span>. </span><span class="cited-content_cbyCitation_article-title">Insights for the design of protein lysine methyltransferase G9a inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (9)
                                     , 993-1014. <a href="https://doi.org/10.4155/fmc-2018-0396" title="DOI URL">https://doi.org/10.4155/fmc-2018-0396</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0396%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DInsights%252Bfor%252Bthe%252Bdesign%252Bof%252Bprotein%252Blysine%252Bmethyltransferase%252BG9a%252Binhibitors%26aulast%3DCharles%26aufirst%3DMariasoosai%2BRamya%2BChandar%26date%3D2019%26volume%3D11%26issue%3D9%26spage%3D993%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frida S.  Herrera-Vázquez</span>, <span class="hlFld-ContribAuthor ">Francisco  Hernández-Luis</span>, <span class="hlFld-ContribAuthor ">José L.  Medina Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Quinazolines as inhibitors of chromatin-associated proteins in histones. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2019,</strong> <em>28 </em>
                                    (4)
                                     , 395-416. <a href="https://doi.org/10.1007/s00044-019-02300-0" title="DOI URL">https://doi.org/10.1007/s00044-019-02300-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-019-02300-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-019-02300-0%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DQuinazolines%252Bas%252Binhibitors%252Bof%252Bchromatin-associated%252Bproteins%252Bin%252Bhistones%26aulast%3DHerrera-V%25C3%25A1zquez%26aufirst%3DFrida%2BS.%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D4%26spage%3D395%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pankaj Kumar  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Histone methyl transferases: A class of epigenetic opportunities to counter uncontrolled cell proliferation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>166 </em>, 351-368. <a href="https://doi.org/10.1016/j.ejmech.2019.01.069" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.069%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Bmethyl%252Btransferases%25253A%252BA%252Bclass%252Bof%252Bepigenetic%252Bopportunities%252Bto%252Bcounter%252Buncontrolled%252Bcell%252Bproliferation%26aulast%3DSingh%26aufirst%3DPankaj%2BKumar%26date%3D2019%26volume%3D166%26spage%3D351%26epage%3D368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Small‐Molecule Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 201-220. <a href="https://doi.org/10.1002/9783527809257.ch9" title="DOI URL">https://doi.org/10.1002/9783527809257.ch9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch9%26sid%3Dliteratum%253Aachs%26atitle%3DSelective%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2019%26date%3D2019%26spage%3D201%26epage%3D220%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carol A.  Eisenberg</span>, <span class="hlFld-ContribAuthor ">Leonard M.  Eisenberg</span>. </span><span class="cited-content_cbyCitation_article-title">G9a and G9a-Like Histone Methyltransferases and Their Effect on Cell Phenotype, Embryonic Development, and Human Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 399-433. <a href="https://doi.org/10.1007/978-3-030-14792-1_16" title="DOI URL">https://doi.org/10.1007/978-3-030-14792-1_16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-14792-1_16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-14792-1_16%26sid%3Dliteratum%253Aachs%26atitle%3DG9a%252Band%252BG9a-Like%252BHistone%252BMethyltransferases%252Band%252BTheir%252BEffect%252Bon%252BCell%252BPhenotype%25252C%252BEmbryonic%252BDevelopment%25252C%252Band%252BHuman%252BDisease%26aulast%3DEisenberg%26aufirst%3DCarol%2BA.%26date%3D2019%26date%3D2019%26spage%3D399%26epage%3D433%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DThe%252BDNA%25252C%252BRNA%25252C%252Band%252BHistone%252BMethylomes%26aulast%3DJurga%26aufirst%3DStefan%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thangavel  Vaijayanthi</span>, <span class="hlFld-ContribAuthor ">Ganesh N.  Pandian</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Sugiyama</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Control System of Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1833-1853. <a href="https://doi.org/10.1002/tcr.201800067" title="DOI URL">https://doi.org/10.1002/tcr.201800067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800067%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DChemical%252BControl%252BSystem%252Bof%252BEpigenetics%26aulast%3DVaijayanthi%26aufirst%3DThangavel%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1833%26epage%3D1853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sharwatie  Ramsaywack</span>, <span class="hlFld-ContribAuthor ">Allyson  Bos</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Vogels</span>, <span class="hlFld-ContribAuthor ">Christopher A.  Gray</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Westcott</span>. </span><span class="cited-content_cbyCitation_article-title">Preliminary investigations into the synthesis and antimicrobial activities of boron-containing capsaicinoids. </span><span class="cited-content_cbyCitation_journal-name">Canadian Journal of Chemistry</span><span> <strong>2018,</strong> <em>96 </em>
                                    (12)
                                     , 1065-1070. <a href="https://doi.org/10.1139/cjc-2018-0193" title="DOI URL">https://doi.org/10.1139/cjc-2018-0193</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1139/cjc-2018-0193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1139%2Fcjc-2018-0193%26sid%3Dliteratum%253Aachs%26jtitle%3DCanadian%2520Journal%2520of%2520Chemistry%26atitle%3DPreliminary%252Binvestigations%252Binto%252Bthe%252Bsynthesis%252Band%252Bantimicrobial%252Bactivities%252Bof%252Bboron-containing%252Bcapsaicinoids%26aulast%3DRamsaywack%26aufirst%3DSharwatie%26date%3D2018%26volume%3D96%26issue%3D12%26spage%3D1065%26epage%3D1070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elias  Orouji</span>, <span class="hlFld-ContribAuthor ">Jochen  Utikal</span>. </span><span class="cited-content_cbyCitation_article-title">Tackling malignant melanoma epigenetically: histone lysine methylation. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-018-0583-z" title="DOI URL">https://doi.org/10.1186/s13148-018-0583-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-018-0583-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-018-0583-z%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DTackling%252Bmalignant%252Bmelanoma%252Bepigenetically%25253A%252Bhistone%252Blysine%252Bmethylation%26aulast%3DOrouji%26aufirst%3DElias%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edgar  López-López</span>, <span class="hlFld-ContribAuthor ">Fernando  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Activity Landscape and Molecular Modeling to Explore the SAR of Dual Epigenetic Inhibitors: A Focus on G9a and DNMT1. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (12)
                                     , 3282. <a href="https://doi.org/10.3390/molecules23123282" title="DOI URL">https://doi.org/10.3390/molecules23123282</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23123282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23123282%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DActivity%252BLandscape%252Band%252BMolecular%252BModeling%252Bto%252BExplore%252Bthe%252BSAR%252Bof%252BDual%252BEpigenetic%252BInhibitors%25253A%252BA%252BFocus%252Bon%252BG9a%252Band%252BDNMT1%26aulast%3DL%25C3%25B3pez-L%25C3%25B3pez%26aufirst%3DEdgar%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D12%26spage%3D3282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhuo  Deng</span>, <span class="hlFld-ContribAuthor ">Jing  Wang</span>, <span class="hlFld-ContribAuthor ">Wentao  Lyu</span>, <span class="hlFld-ContribAuthor ">Xuwen  Wieneke</span>, <span class="hlFld-ContribAuthor ">Robert  Matts</span>, <span class="hlFld-ContribAuthor ">Xi  Ma</span>, <span class="hlFld-ContribAuthor ">Guolong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Cell-Based High-Throughput Screening Assay to Identify Porcine Host Defense Peptide-Inducing Compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Immunology Research</span><span> <strong>2018,</strong> <em>2018 </em>, 1-13. <a href="https://doi.org/10.1155/2018/5492941" title="DOI URL">https://doi.org/10.1155/2018/5492941</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2018/5492941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2018%2F5492941%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Immunology%2520Research%26atitle%3DDevelopment%252Bof%252Ba%252BCell-Based%252BHigh-Throughput%252BScreening%252BAssay%252Bto%252BIdentify%252BPorcine%252BHost%252BDefense%252BPeptide-Inducing%252BCompounds%26aulast%3DDeng%26aufirst%3DZhuo%26date%3D2018%26volume%3D2018%26spage%3D1%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lital Estrella  Weil</span>, <span class="hlFld-ContribAuthor ">Yulia  Shmidov</span>, <span class="hlFld-ContribAuthor ">Margarita  Kublanovsky</span>, <span class="hlFld-ContribAuthor ">David  Morgenstern</span>, <span class="hlFld-ContribAuthor ">Michal  Feldman</span>, <span class="hlFld-ContribAuthor ">Ronit  Bitton</span>, <span class="hlFld-ContribAuthor ">Dan  Levy</span>. </span><span class="cited-content_cbyCitation_article-title">Oligomerization and Auto-methylation of the Human Lysine Methyltransferase SETD6. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2018,</strong> <em>430 </em>
                                    (21)
                                     , 4359-4368. <a href="https://doi.org/10.1016/j.jmb.2018.08.028" title="DOI URL">https://doi.org/10.1016/j.jmb.2018.08.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2018.08.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2018.08.028%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DOligomerization%252Band%252BAuto-methylation%252Bof%252Bthe%252BHuman%252BLysine%252BMethyltransferase%252BSETD6%26aulast%3DWeil%26aufirst%3DLital%2BEstrella%26date%3D2018%26volume%3D430%26issue%3D21%26spage%3D4359%26epage%3D4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tilman  Schneider-Poetsch</span>, <span class="hlFld-ContribAuthor ">Minoru  Yoshida</span>. </span><span class="cited-content_cbyCitation_article-title">Along the Central Dogma—Controlling Gene Expression with Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Biochemistry</span><span> <strong>2018,</strong> <em>87 </em>
                                    (1)
                                     , 391-420. <a href="https://doi.org/10.1146/annurev-biochem-060614-033923" title="DOI URL">https://doi.org/10.1146/annurev-biochem-060614-033923</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-biochem-060614-033923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-biochem-060614-033923%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Biochemistry%26atitle%3DAlong%252Bthe%252BCentral%252BDogma%2525E2%252580%252594Controlling%252BGene%252BExpression%252Bwith%252BSmall%252BMolecules%26aulast%3DSchneider-Poetsch%26aufirst%3DTilman%26date%3D2018%26volume%3D87%26issue%3D1%26spage%3D391%26epage%3D420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>. </span><span class="cited-content_cbyCitation_article-title">Protein methyltransferase inhibitors as precision cancer therapeutics: a decade of discovery. </span><span class="cited-content_cbyCitation_journal-name">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span> <strong>2018,</strong> <em>373 </em>
                                    (1748)
                                     , 20170080. <a href="https://doi.org/10.1098/rstb.2017.0080" title="DOI URL">https://doi.org/10.1098/rstb.2017.0080</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1098/rstb.2017.0080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1098%2Frstb.2017.0080%26sid%3Dliteratum%253Aachs%26jtitle%3DPhilosophical%2520Transactions%2520of%2520the%2520Royal%2520Society%2520B%253A%2520Biological%2520Sciences%26atitle%3DProtein%252Bmethyltransferase%252Binhibitors%252Bas%252Bprecision%252Bcancer%252Btherapeutics%25253A%252Ba%252Bdecade%252Bof%252Bdiscovery%26aulast%3DCopeland%26aufirst%3DRobert%2BA.%26date%3D2018%26date%3D2018%26volume%3D373%26issue%3D1748%26spage%3D20170080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J.  Morrison</span>, <span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>, <span class="hlFld-ContribAuthor ">Tim J.  Wigle</span>, <span class="hlFld-ContribAuthor ">Dipti  Sadalge</span>, <span class="hlFld-ContribAuthor ">Kevin W.  Kuntz</span>, <span class="hlFld-ContribAuthor ">Margaret Porter  Scott</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">Darren M.  Harvey</span>, <span class="hlFld-ContribAuthor ">John W.  Lampe</span>, <span class="hlFld-ContribAuthor ">Lorna H.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>, . </span><span class="cited-content_cbyCitation_article-title">Identification of a peptide inhibitor for the histone methyltransferase WHSC1. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , e0197082. <a href="https://doi.org/10.1371/journal.pone.0197082" title="DOI URL">https://doi.org/10.1371/journal.pone.0197082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0197082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0197082%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIdentification%252Bof%252Ba%252Bpeptide%252Binhibitor%252Bfor%252Bthe%252Bhistone%252Bmethyltransferase%252BWHSC1%26aulast%3DMorrison%26aufirst%3DMichael%2BJ.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0197082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">C. Iluhí  Oviedo-Osornio</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Methods for Epigenetic Drug Discovery: A Focus on Activity Landscape Modeling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 65-83. <a href="https://doi.org/10.1016/bs.apcsb.2018.01.001" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2018.01.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2018.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2018.01.001%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BMethods%252Bfor%252BEpigenetic%252BDrug%252BDiscovery%25253A%252BA%252BFocus%252Bon%252BActivity%252BLandscape%252BModeling%26aulast%3DNaveja%26aufirst%3DJ.%2BJes%25C3%25BAs%26date%3D2018%26spage%3D65%26epage%3D83%26pub%3DElsevier%26atitle%3DComputational%252BMolecular%252BModelling%252Bin%252BStructural%252BBiology%26date%3D2018%26volume%3D113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moses  Moustakim</span>, <span class="hlFld-ContribAuthor ">Suet Ling  Felce</span>, <span class="hlFld-ContribAuthor ">Nancy  Zaarour</span>, <span class="hlFld-ContribAuthor ">Gillian  Farnie</span>, <span class="hlFld-ContribAuthor ">Fiona E.  McCann</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Target Identification Using Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 27-58. <a href="https://doi.org/10.1016/bs.mie.2018.09.013" title="DOI URL">https://doi.org/10.1016/bs.mie.2018.09.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.mie.2018.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.mie.2018.09.013%26sid%3Dliteratum%253Aachs%26atitle%3DTarget%252BIdentification%252BUsing%252BChemical%252BProbes%26aulast%3DMoustakim%26aufirst%3DMoses%26date%3D2018%26spage%3D27%26epage%3D58%26pub%3DElsevier%26atitle%3DModern%252BApproaches%252Bin%252BDrug%252BDiscovery%26date%3D2018%26volume%3D610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Insun  Park</span>, <span class="hlFld-ContribAuthor ">Yu Jin  Hwang</span>, <span class="hlFld-ContribAuthor ">TaeHun  Kim</span>, <span class="hlFld-ContribAuthor ">Ambily Nath Indu  Viswanath</span>, <span class="hlFld-ContribAuthor ">Ashwini M.  Londhe</span>, <span class="hlFld-ContribAuthor ">Seo Yun  Jung</span>, <span class="hlFld-ContribAuthor ">Kyoung Mi  Sim</span>, <span class="hlFld-ContribAuthor ">Sun-Joon  Min</span>, <span class="hlFld-ContribAuthor ">Ji Eun  Lee</span>, <span class="hlFld-ContribAuthor ">Jihye  Seong</span>, <span class="hlFld-ContribAuthor ">Yun Kyung  Kim</span>, <span class="hlFld-ContribAuthor ">Kyoung Tai  No</span>, <span class="hlFld-ContribAuthor ">Hoon  Ryu</span>, <span class="hlFld-ContribAuthor ">Ae Nim  Pae</span>. </span><span class="cited-content_cbyCitation_article-title">In silico probing and biological evaluation of SETDB1/ESET-targeted novel compounds that reduce tri-methylated histone H3K9 (H3K9me3) level. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2017,</strong> <em>31 </em>
                                    (10)
                                     , 877-889. <a href="https://doi.org/10.1007/s10822-017-0052-3" title="DOI URL">https://doi.org/10.1007/s10822-017-0052-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-017-0052-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-017-0052-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIn%252Bsilico%252Bprobing%252Band%252Bbiological%252Bevaluation%252Bof%252BSETDB1%25252FESET-targeted%252Bnovel%252Bcompounds%252Bthat%252Breduce%252Btri-methylated%252Bhistone%252BH3K9%252B%252528H3K9me3%252529%252Blevel%26aulast%3DPark%26aufirst%3DInsun%26date%3D2017%26date%3D2017%26volume%3D31%26issue%3D10%26spage%3D877%26epage%3D889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahima  Sharma</span>, <span class="hlFld-ContribAuthor ">Tobias  Dierkes</span>, <span class="hlFld-ContribAuthor ">Sreedharan  Sajikumar</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic regulation by G9a/GLP complex ameliorates amyloid-beta 1-42 induced deficits in long-term plasticity and synaptic tagging/capture in hippocampal pyramidal neurons. </span><span class="cited-content_cbyCitation_journal-name">Aging Cell</span><span> <strong>2017,</strong> <em>16 </em>
                                    (5)
                                     , 1062-1072. <a href="https://doi.org/10.1111/acel.12634" title="DOI URL">https://doi.org/10.1111/acel.12634</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/acel.12634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Facel.12634%26sid%3Dliteratum%253Aachs%26jtitle%3DAging%2520Cell%26atitle%3DEpigenetic%252Bregulation%252Bby%252BG9a%25252FGLP%252Bcomplex%252Bameliorates%252Bamyloid-beta%252B1-42%252Binduced%252Bdeficits%252Bin%252Blong-term%252Bplasticity%252Band%252Bsynaptic%252Btagging%25252Fcapture%252Bin%252Bhippocampal%252Bpyramidal%252Bneurons%26aulast%3DSharma%26aufirst%3DMahima%26date%3D2017%26date%3D2017%26volume%3D16%26issue%3D5%26spage%3D1062%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kelly A.  Teske</span>, <span class="hlFld-ContribAuthor ">M. Kyle  Hadden</span>. </span><span class="cited-content_cbyCitation_article-title">Methyllysine binding domains: Structural insight and small molecule probe development. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>136 </em>, 14-35. <a href="https://doi.org/10.1016/j.ejmech.2017.04.047" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.04.047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.04.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.04.047%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMethyllysine%252Bbinding%252Bdomains%25253A%252BStructural%252Binsight%252Band%252Bsmall%252Bmolecule%252Bprobe%252Bdevelopment%26aulast%3DTeske%26aufirst%3DKelly%2BA.%26date%3D2017%26volume%3D136%26spage%3D14%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edurne  San José-Enériz</span>, <span class="hlFld-ContribAuthor ">Xabier  Agirre</span>, <span class="hlFld-ContribAuthor ">Obdulia  Rabal</span>, <span class="hlFld-ContribAuthor ">Amaia  Vilas-Zornoza</span>, <span class="hlFld-ContribAuthor ">Juan A.  Sanchez-Arias</span>, <span class="hlFld-ContribAuthor ">Estibaliz  Miranda</span>, <span class="hlFld-ContribAuthor ">Ana  Ugarte</span>, <span class="hlFld-ContribAuthor ">Sergio  Roa</span>, <span class="hlFld-ContribAuthor ">Bruno  Paiva</span>, <span class="hlFld-ContribAuthor ">Ander  Estella-Hermoso de Mendoza</span>, <span class="hlFld-ContribAuthor ">Rosa María  Alvarez</span>, <span class="hlFld-ContribAuthor ">Noelia  Casares</span>, <span class="hlFld-ContribAuthor ">Victor  Segura</span>, <span class="hlFld-ContribAuthor ">José I.  Martín-Subero</span>, <span class="hlFld-ContribAuthor ">François-Xavier  Ogi</span>, <span class="hlFld-ContribAuthor ">Pierre  Soule</span>, <span class="hlFld-ContribAuthor ">Clara M.  Santiveri</span>, <span class="hlFld-ContribAuthor ">Ramón  Campos-Olivas</span>, <span class="hlFld-ContribAuthor ">Giancarlo  Castellano</span>, <span class="hlFld-ContribAuthor ">Maite Garcia Fernandez  de Barrena</span>, <span class="hlFld-ContribAuthor ">Juan Roberto  Rodriguez-Madoz</span>, <span class="hlFld-ContribAuthor ">Maria José  García-Barchino</span>, <span class="hlFld-ContribAuthor ">Juan Jose  Lasarte</span>, <span class="hlFld-ContribAuthor ">Matias A  Avila</span>, <span class="hlFld-ContribAuthor ">Jose Angel  Martinez-Climent</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>, <span class="hlFld-ContribAuthor ">Felipe  Prosper</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of first-in-class reversible dual small molecule inhibitors against G9a and DNMTs in hematological malignancies. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2017,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/ncomms15424" title="DOI URL">https://doi.org/10.1038/ncomms15424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/ncomms15424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fncomms15424%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DDiscovery%252Bof%252Bfirst-in-class%252Breversible%252Bdual%252Bsmall%252Bmolecule%252Binhibitors%252Bagainst%252BG9a%252Band%252BDNMTs%252Bin%252Bhematological%252Bmalignancies%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DEdurne%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationship studies of G9a-like protein (GLP) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (16)
                                     , 4414-4423. <a href="https://doi.org/10.1016/j.bmc.2017.06.021" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.06.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.06.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Brelationship%252Bstudies%252Bof%252BG9a-like%252Bprotein%252B%252528GLP%252529%252Binhibitors%26aulast%3DXiong%26aufirst%3DYan%26date%3D2017%26volume%3D25%26issue%3D16%26spage%3D4414%26epage%3D4423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Wang</span>, <span class="hlFld-ContribAuthor ">Mei-Mei  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiang-Shan  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Structurally diversified synthesis of 2,3-dihydroquinazolin-4-(1H)-ones from 2-aminobenzamides and 1,2-dicarbonyl compounds in ionic liquids catalyzed by iodine. </span><span class="cited-content_cbyCitation_journal-name">Research on Chemical Intermediates</span><span> <strong>2017,</strong> <em>43 </em>
                                    (5)
                                     , 2985-3005. <a href="https://doi.org/10.1007/s11164-016-2807-1" title="DOI URL">https://doi.org/10.1007/s11164-016-2807-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11164-016-2807-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11164-016-2807-1%26sid%3Dliteratum%253Aachs%26jtitle%3DResearch%2520on%2520Chemical%2520Intermediates%26atitle%3DStructurally%252Bdiversified%252Bsynthesis%252Bof%252B2%25252C3-dihydroquinazolin-4-%2525281H%252529-ones%252Bfrom%252B2-aminobenzamides%252Band%252B1%25252C2-dicarbonyl%252Bcompounds%252Bin%252Bionic%252Bliquids%252Bcatalyzed%252Bby%252Biodine%26aulast%3DWang%26aufirst%3DJie%26date%3D2017%26date%3D2016%26volume%3D43%26issue%3D5%26spage%3D2985%26epage%3D3005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jake  Shortt</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Ott</span>, <span class="hlFld-ContribAuthor ">Ricky W.  Johnstone</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>. </span><span class="cited-content_cbyCitation_article-title">A chemical probe toolbox for dissecting the cancer epigenome. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2017,</strong> <em>17 </em>
                                    (3)
                                     , 160-183. <a href="https://doi.org/10.1038/nrc.2016.148" title="DOI URL">https://doi.org/10.1038/nrc.2016.148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrc.2016.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrc.2016.148%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DA%252Bchemical%252Bprobe%252Btoolbox%252Bfor%252Bdissecting%252Bthe%252Bcancer%252Bepigenome%26aulast%3DShortt%26aufirst%3DJake%26date%3D2017%26date%3D2017%26volume%3D17%26issue%3D3%26spage%3D160%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chun-Yu  Liu</span>, <span class="hlFld-ContribAuthor ">Tzu-Ting  Huang</span>, <span class="hlFld-ContribAuthor ">Chun-Teng  Huang</span>, <span class="hlFld-ContribAuthor ">Ming-Hung  Hu</span>, <span class="hlFld-ContribAuthor ">Duen-Shian  Wang</span>, <span class="hlFld-ContribAuthor ">Wan-Lun  Wang</span>, <span class="hlFld-ContribAuthor ">Wen-Chun  Tsai</span>, <span class="hlFld-ContribAuthor ">Chia-Han  Lee</span>, <span class="hlFld-ContribAuthor ">Ka-Yi  Lau</span>, <span class="hlFld-ContribAuthor ">Hsiu-Ping  Yang</span>, <span class="hlFld-ContribAuthor ">Ming-Huang  Chen</span>, <span class="hlFld-ContribAuthor ">Chung-Wai  Shiau</span>, <span class="hlFld-ContribAuthor ">Ling-Ming  Tseng</span>, <span class="hlFld-ContribAuthor ">Kuen-Feng  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">EGFR-independent Elk1/CIP2A signalling mediates apoptotic effect of an erlotinib derivative TD52 in triple-negative breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Cancer</span><span> <strong>2017,</strong> <em>72 </em>, 112-123. <a href="https://doi.org/10.1016/j.ejca.2016.11.012" title="DOI URL">https://doi.org/10.1016/j.ejca.2016.11.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejca.2016.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejca.2016.11.012%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Cancer%26atitle%3DEGFR-independent%252BElk1%25252FCIP2A%252Bsignalling%252Bmediates%252Bapoptotic%252Beffect%252Bof%252Ban%252Berlotinib%252Bderivative%252BTD52%252Bin%252Btriple-negative%252Bbreast%252Bcancer%252Bcells%26aulast%3DLiu%26aufirst%3DChun-Yu%26date%3D2017%26volume%3D72%26spage%3D112%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kexin  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">DNA and Histone Methylation in Prostate Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 489-529. <a href="https://doi.org/10.1007/978-3-319-59786-7_18" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_18%26sid%3Dliteratum%253Aachs%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bin%252BProstate%252BCancer%26aulast%3DXu%26aufirst%3DKexin%26date%3D2017%26date%3D2017%26spage%3D489%26epage%3D529%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akihiro  Ito</span>, <span class="hlFld-ContribAuthor ">Minoru  Yoshida</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 37-74. <a href="https://doi.org/10.1007/978-4-431-56529-1_3" title="DOI URL">https://doi.org/10.1007/978-4-431-56529-1_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-4-431-56529-1_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-4-431-56529-1_3%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetics%26aulast%3DIto%26aufirst%3DAkihiro%26date%3D2017%26date%3D2017%26spage%3D37%26epage%3D74%26pub%3DSpringer%2520Japan%26atitle%3DBioprobes%26aulast%3DOsada%26aufirst%3DHiroyuki%26date%3D2017%26volume%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.  Harvey</span>, <span class="hlFld-ContribAuthor ">M.  Foley</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Epigenetic Regulation in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 281-307. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12393-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12393-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12393-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12393-5%26sid%3Dliteratum%253Aachs%26atitle%3DInhibitors%252Bof%252BEpigenetic%252BRegulation%252Bin%252BCancer%26aulast%3DHarvey%26aufirst%3DD.%26date%3D2017%26spage%3D281%26epage%3D307%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Patty B.  Chen</span>, <span class="hlFld-ContribAuthor ">Alexandra S.  Lubin</span>, <span class="hlFld-ContribAuthor ">Gregor A.  Lueg</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Andrew J. P.  White</span>, <span class="hlFld-ContribAuthor ">Nicholas A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Artur  Scherf</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine methyltransferase structure activity relationships that allow for segregation of G9a inhibition and anti-Plasmodium activity. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (5)
                                     , 1069-1092. <a href="https://doi.org/10.1039/C7MD00052A" title="DOI URL">https://doi.org/10.1039/C7MD00052A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00052A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00052A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DHistone%252Blysine%252Bmethyltransferase%252Bstructure%252Bactivity%252Brelationships%252Bthat%252Ballow%252Bfor%252Bsegregation%252Bof%252BG9a%252Binhibition%252Band%252Banti-Plasmodium%252Bactivity%26aulast%3DSundriyal%26aufirst%3DSandeep%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D5%26spage%3D1069%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shagufta  Shagufta</span>, <span class="hlFld-ContribAuthor ">Irshad  Ahmad</span>. </span><span class="cited-content_cbyCitation_article-title">An insight into the therapeutic potential of quinazoline derivatives as anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (5)
                                     , 871-885. <a href="https://doi.org/10.1039/C7MD00097A" title="DOI URL">https://doi.org/10.1039/C7MD00097A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00097A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00097A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DAn%252Binsight%252Binto%252Bthe%252Btherapeutic%252Bpotential%252Bof%252Bquinazoline%252Bderivatives%252Bas%252Banticancer%252Bagents%26aulast%3DShagufta%26aufirst%3DShagufta%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D5%26spage%3D871%26epage%3D885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ciro  Milite</span>, <span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Donatella  Rescigno</span>, <span class="hlFld-ContribAuthor ">Agostino  Cianciulli</span>, <span class="hlFld-ContribAuthor ">Amodio Luca  Balzano</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging role of lysine methyltransferase SETD8 in human diseases. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-016-0268-4" title="DOI URL">https://doi.org/10.1186/s13148-016-0268-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-016-0268-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-016-0268-4%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DThe%252Bemerging%252Brole%252Bof%252Blysine%252Bmethyltransferase%252BSETD8%252Bin%252Bhuman%252Bdiseases%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Uttara  Soumyanarayanan</span>, <span class="hlFld-ContribAuthor ">Brian W  Dymock</span>. </span><span class="cited-content_cbyCitation_article-title">Recently discovered EZH2 and EHMT2 (G9a) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1635-1654. <a href="https://doi.org/10.4155/fmc-2016-0096" title="DOI URL">https://doi.org/10.4155/fmc-2016-0096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0096%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecently%252Bdiscovered%252BEZH2%252Band%252BEHMT2%252B%252528G9a%252529%252Binhibitors%26aulast%3DSoumyanarayanan%26aufirst%3DUttara%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1635%26epage%3D1654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ciro  Milite</span>, <span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Donatella  Rescigno</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in the Development of Lysine Methyltransferase SETD8 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (16)
                                     , 1680-1685. <a href="https://doi.org/10.1002/cmdc.201600272" title="DOI URL">https://doi.org/10.1002/cmdc.201600272</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600272%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BLysine%252BMethyltransferase%252BSETD8%252BInhibitors%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D16%26spage%3D1680%26epage%3D1685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin E.  Paluch</span>, <span class="hlFld-ContribAuthor ">Abdul R.  Naqash</span>, <span class="hlFld-ContribAuthor ">Zachary  Brumberger</span>, <span class="hlFld-ContribAuthor ">Michael J.  Nemeth</span>, <span class="hlFld-ContribAuthor ">Elizabeth A.  Griffiths</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics: A primer for clinicians. </span><span class="cited-content_cbyCitation_journal-name">Blood Reviews</span><span> <strong>2016,</strong> <em>30 </em>
                                    (4)
                                     , 285-295. <a href="https://doi.org/10.1016/j.blre.2016.02.002" title="DOI URL">https://doi.org/10.1016/j.blre.2016.02.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.blre.2016.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.blre.2016.02.002%26sid%3Dliteratum%253Aachs%26jtitle%3DBlood%2520Reviews%26atitle%3DEpigenetics%25253A%252BA%252Bprimer%252Bfor%252Bclinicians%26aulast%3DPaluch%26aufirst%3DBenjamin%2BE.%26date%3D2016%26volume%3D30%26issue%3D4%26spage%3D285%26epage%3D295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahima  Sharma</span>, <span class="hlFld-ContribAuthor ">Nuralyah Bte  Razali</span>, <span class="hlFld-ContribAuthor ">Sreedharan  Sajikumar</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of G9a/GLP Complex Promotes Long-Term Potentiation and Synaptic Tagging/Capture in Hippocampal CA1 Pyramidal Neurons. </span><span class="cited-content_cbyCitation_journal-name">Cerebral Cortex</span><span> <strong>2016,</strong> <em>26 </em>, bhw170. <a href="https://doi.org/10.1093/cercor/bhw170" title="DOI URL">https://doi.org/10.1093/cercor/bhw170</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/cercor/bhw170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fcercor%2Fbhw170%26sid%3Dliteratum%253Aachs%26jtitle%3DCerebral%2520Cortex%26atitle%3DInhibition%252Bof%252BG9a%25252FGLP%252BComplex%252BPromotes%252BLong-Term%252BPotentiation%252Band%252BSynaptic%252BTagging%25252FCapture%252Bin%252BHippocampal%252BCA1%252BPyramidal%252BNeurons%26aulast%3DSharma%26aufirst%3DMahima%26date%3D2016%26date%3D2016%26volume%3D26%26spage%3Dbhw170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah E.  Graham</span>, <span class="hlFld-ContribAuthor ">Sara E.  Tweedy</span>, <span class="hlFld-ContribAuthor ">Heather A.  Carlson</span>. </span><span class="cited-content_cbyCitation_article-title">Dynamic behavior of the post-SET loop region of NSD1: Implications for histone binding and drug development. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2016,</strong> <em>25 </em>
                                    (5)
                                     , 1021-1029. <a href="https://doi.org/10.1002/pro.2912" title="DOI URL">https://doi.org/10.1002/pro.2912</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.2912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.2912%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DDynamic%252Bbehavior%252Bof%252Bthe%252Bpost-SET%252Bloop%252Bregion%252Bof%252BNSD1%25253A%252BImplications%252Bfor%252Bhistone%252Bbinding%252Band%252Bdrug%252Bdevelopment%26aulast%3DGraham%26aufirst%3DSarah%2BE.%26date%3D2016%26date%3D2016%26volume%3D25%26issue%3D5%26spage%3D1021%26epage%3D1029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xi  Zhang</span>, <span class="hlFld-ContribAuthor ">Danni  Peng</span>, <span class="hlFld-ContribAuthor ">Yuanxin  Xi</span>, <span class="hlFld-ContribAuthor ">Chao  Yuan</span>, <span class="hlFld-ContribAuthor ">Cari A.  Sagum</span>, <span class="hlFld-ContribAuthor ">Brianna J.  Klein</span>, <span class="hlFld-ContribAuthor ">Kaori  Tanaka</span>, <span class="hlFld-ContribAuthor ">Hong  Wen</span>, <span class="hlFld-ContribAuthor ">Tatiana G.  Kutateladze</span>, <span class="hlFld-ContribAuthor ">Wei  Li</span>, <span class="hlFld-ContribAuthor ">Mark T.  Bedford</span>, <span class="hlFld-ContribAuthor ">Xiaobing  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">G9a-mediated methylation of ERα links the PHF20/MOF histone acetyltransferase complex to hormonal gene expression. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2016,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/ncomms10810" title="DOI URL">https://doi.org/10.1038/ncomms10810</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/ncomms10810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fncomms10810%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DG9a-mediated%252Bmethylation%252Bof%252BER%2525CE%2525B1%252Blinks%252Bthe%252BPHF20%25252FMOF%252Bhistone%252Bacetyltransferase%252Bcomplex%252Bto%252Bhormonal%252Bgene%252Bexpression%26aulast%3DZhang%26aufirst%3DXi%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Yasgar</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>, <span class="hlFld-ContribAuthor ">Nathan P.  Coussens</span>. </span><span class="cited-content_cbyCitation_article-title">AlphaScreen-Based Assays: Ultra-High-Throughput Screening for Small-Molecule Inhibitors of Challenging Enzymes and Protein-Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 77-98. <a href="https://doi.org/10.1007/978-1-4939-3673-1_5" title="DOI URL">https://doi.org/10.1007/978-1-4939-3673-1_5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-3673-1_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-3673-1_5%26sid%3Dliteratum%253Aachs%26atitle%3DAlphaScreen-Based%252BAssays%25253A%252BUltra-High-Throughput%252BScreening%252Bfor%252BSmall-Molecule%252BInhibitors%252Bof%252BChallenging%252BEnzymes%252Band%252BProtein-Protein%252BInteractions%26aulast%3DYasgar%26aufirst%3DAdam%26date%3D2016%26date%3D2016%26spage%3D77%26epage%3D98%26pub%3DSpringer%2520New%2520York%26atitle%3DHigh%252BThroughput%252BScreening%26aulast%3DJanzen%26aufirst%3DWilliam%2BP.%26date%3D2016%26volume%3D1439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie  Lopez</span>, <span class="hlFld-ContribAuthor ">Maxime  Leroy</span>, <span class="hlFld-ContribAuthor ">Chantal  Etievant</span>, <span class="hlFld-ContribAuthor ">Fréderic  Ausseil</span>, <span class="hlFld-ContribAuthor ">Alexandre  Gagnon</span>, <span class="hlFld-ContribAuthor ">Paola  Arimondo</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 63-95. <a href="https://doi.org/10.1016/B978-0-12-802208-5.00003-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-802208-5.00003-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802208-5.00003-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802208-5.00003-5%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BDiscovery%252BMethods%26aulast%3DLopez%26aufirst%3DMarie%26date%3D2016%26spage%3D63%26epage%3D95%26pub%3DElsevier%26atitle%3DDrug%252BDiscovery%252Bin%252BCancer%252BEpigenetics%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Kelsey  Bohn</span>, <span class="hlFld-ContribAuthor ">Christine A.  Hrycyna</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Non-substrate based, small molecule inhibitors of the human isoprenylcysteine carboxyl methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2016,</strong> <em>7 </em>
                                    (5)
                                     , 1016-1021. <a href="https://doi.org/10.1039/C6MD00130K" title="DOI URL">https://doi.org/10.1039/C6MD00130K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00130K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00130K%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DNon-substrate%252Bbased%25252C%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Bthe%252Bhuman%252Bisoprenylcysteine%252Bcarboxyl%252Bmethyltransferase%26aulast%3DButler%26aufirst%3DKyle%2BV.%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D5%26spage%3D1016%26epage%3D1021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward  Curry</span>, <span class="hlFld-ContribAuthor ">Ian  Green</span>, <span class="hlFld-ContribAuthor ">Nadine  Chapman-Rothe</span>, <span class="hlFld-ContribAuthor ">Elham  Shamsaei</span>, <span class="hlFld-ContribAuthor ">Sarah  Kandil</span>, <span class="hlFld-ContribAuthor ">Fanny L  Cherblanc</span>, <span class="hlFld-ContribAuthor ">Luke  Payne</span>, <span class="hlFld-ContribAuthor ">Emma  Bell</span>, <span class="hlFld-ContribAuthor ">Thota  Ganesh</span>, <span class="hlFld-ContribAuthor ">Nitipol  Srimongkolpithak</span>, <span class="hlFld-ContribAuthor ">Joachim  Caron</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Uren</span>, <span class="hlFld-ContribAuthor ">James P.  Snyder</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>, <span class="hlFld-ContribAuthor ">Robert  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Dual EZH2 and EHMT2 histone methyltransferase inhibition increases biological efficacy in breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-015-0118-9" title="DOI URL">https://doi.org/10.1186/s13148-015-0118-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-015-0118-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-015-0118-9%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DDual%252BEZH2%252Band%252BEHMT2%252Bhistone%252Bmethyltransferase%252Binhibition%252Bincreases%252Bbiological%252Befficacy%252Bin%252Bbreast%252Bcancer%252Bcells%26aulast%3DCurry%26aufirst%3DEdward%26date%3D2015%26date%3D2015%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minkui  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of protein methyltransferases as chemical tools. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (8)
                                     , 1327-1338. <a href="https://doi.org/10.2217/epi.15.87" title="DOI URL">https://doi.org/10.2217/epi.15.87</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.15.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.15.87%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DInhibitors%252Bof%252Bprotein%252Bmethyltransferases%252Bas%252Bchemical%252Btools%26aulast%3DLuo%26aufirst%3DMinkui%26date%3D2015%26volume%3D7%26issue%3D8%26spage%3D1327%26epage%3D1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Bayo</span>, <span class="hlFld-ContribAuthor ">Maithili P  Dalvi</span>, <span class="hlFld-ContribAuthor ">Elisabeth D  Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">Successful strategies in the discovery of small-molecule epigenetic modulators with anticancer potential. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (16)
                                     , 2243-2261. <a href="https://doi.org/10.4155/fmc.15.140" title="DOI URL">https://doi.org/10.4155/fmc.15.140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.140%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSuccessful%252Bstrategies%252Bin%252Bthe%252Bdiscovery%252Bof%252Bsmall-molecule%252Bepigenetic%252Bmodulators%252Bwith%252Banticancer%252Bpotential%26aulast%3DBayo%26aufirst%3DJuan%26date%3D2015%26volume%3D7%26issue%3D16%26spage%3D2243%26epage%3D2261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunngai  Hui</span>, <span class="hlFld-ContribAuthor ">Tao  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of lysine methyltransferase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2015,</strong> <em>3 </em><a href="https://doi.org/10.3389/fchem.2015.00044" title="DOI URL">https://doi.org/10.3389/fchem.2015.00044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2015.00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2015.00044%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DSynthesis%252Bof%252Blysine%252Bmethyltransferase%252Binhibitors%26aulast%3DHui%26aufirst%3DChunngai%26date%3D2015%26date%3D2015%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  McGrath</span>, <span class="hlFld-ContribAuthor ">Patrick  Trojer</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone lysine methylation in cancer. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2015,</strong> <em>150 </em>, 1-22. <a href="https://doi.org/10.1016/j.pharmthera.2015.01.002" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2015.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bhistone%252Blysine%252Bmethylation%252Bin%252Bcancer%26aulast%3DMcGrath%26aufirst%3DJohn%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Simó-Riudalbas</span>, <span class="hlFld-ContribAuthor ">Manel  Esteller</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the histone orthography of cancer: drugs for writers, erasers and readers. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2015,</strong> <em>172 </em>
                                    (11)
                                     , 2716-2732. <a href="https://doi.org/10.1111/bph.12844" title="DOI URL">https://doi.org/10.1111/bph.12844</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.12844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.12844%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DTargeting%252Bthe%252Bhistone%252Borthography%252Bof%252Bcancer%25253A%252Bdrugs%252Bfor%252Bwriters%25252C%252Berasers%252Band%252Breaders%26aulast%3DSim%25C3%25B3-Riudalbas%26aufirst%3DLaia%26date%3D2015%26date%3D2014%26volume%3D172%26issue%3D11%26spage%3D2716%26epage%3D2732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xia  Jiang</span>, <span class="hlFld-ContribAuthor ">Cheryl Zi Hui  Lim</span>, <span class="hlFld-ContribAuthor ">Zhimei  Li</span>, <span class="hlFld-ContribAuthor ">Puay Leng  Lee</span>, <span class="hlFld-ContribAuthor ">Siti Maryam J. M.  Yatim</span>, <span class="hlFld-ContribAuthor ">Peiyong  Guan</span>, <span class="hlFld-ContribAuthor ">Juntao  Li</span>, <span class="hlFld-ContribAuthor ">Jianbiao  Zhou</span>, <span class="hlFld-ContribAuthor ">Jingxuan  Pan</span>, <span class="hlFld-ContribAuthor ">Wee-Joo  Chng</span>, <span class="hlFld-ContribAuthor ">Christina L. L.  Chai</span>, <span class="hlFld-ContribAuthor ">Qiang  Yu</span>, . </span><span class="cited-content_cbyCitation_article-title">Functional Characterization of D9, a Novel Deazaneplanocin A (DZNep) Analog, in Targeting Acute Myeloid Leukemia (AML). </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2015,</strong> <em>10 </em>
                                    (4)
                                     , e0122983. <a href="https://doi.org/10.1371/journal.pone.0122983" title="DOI URL">https://doi.org/10.1371/journal.pone.0122983</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0122983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0122983%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DFunctional%252BCharacterization%252Bof%252BD9%25252C%252Ba%252BNovel%252BDeazaneplanocin%252BA%252B%252528DZNep%252529%252BAnalog%25252C%252Bin%252BTargeting%252BAcute%252BMyeloid%252BLeukemia%252B%252528AML%252529%26aulast%3DJiang%26aufirst%3DXia%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D4%26spage%3De0122983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicholas A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Joachim  Caron</span>, <span class="hlFld-ContribAuthor ">Patty  Chen</span>, <span class="hlFld-ContribAuthor ">Benoit  Witkowski</span>, <span class="hlFld-ContribAuthor ">Didier  Menard</span>, <span class="hlFld-ContribAuthor ">Rossarin  Suwanarusk</span>, <span class="hlFld-ContribAuthor ">Laurent  Renia</span>, <span class="hlFld-ContribAuthor ">Francois  Nosten</span>, <span class="hlFld-ContribAuthor ">María Belén  Jiménez-Díaz</span>, <span class="hlFld-ContribAuthor ">Iñigo  Angulo-Barturen</span>, <span class="hlFld-ContribAuthor ">María Santos  Martínez</span>, <span class="hlFld-ContribAuthor ">Santiago  Ferrer</span>, <span class="hlFld-ContribAuthor ">Laura M.  Sanz</span>, <span class="hlFld-ContribAuthor ">Francisco-Javier  Gamo</span>, <span class="hlFld-ContribAuthor ">Sergio  Wittlin</span>, <span class="hlFld-ContribAuthor ">Sandra  Duffy</span>, <span class="hlFld-ContribAuthor ">Vicky M.  Avery</span>, <span class="hlFld-ContribAuthor ">Andrea  Ruecker</span>, <span class="hlFld-ContribAuthor ">Michael J.  Delves</span>, <span class="hlFld-ContribAuthor ">Robert E.  Sinden</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>, <span class="hlFld-ContribAuthor ">Artur  Scherf</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Methyltransferase Inhibitors Are Orally Bioavailable, Fast-Acting Molecules with Activity against Different Species Causing Malaria in Humans. </span><span class="cited-content_cbyCitation_journal-name">Antimicrobial Agents and Chemotherapy</span><span> <strong>2015,</strong> <em>59 </em>
                                    (2)
                                     , 950-959. <a href="https://doi.org/10.1128/AAC.04419-14" title="DOI URL">https://doi.org/10.1128/AAC.04419-14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AAC.04419-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAAC.04419-14%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%2520Agents%2520and%2520Chemotherapy%26atitle%3DHistone%252BMethyltransferase%252BInhibitors%252BAre%252BOrally%252BBioavailable%25252C%252BFast-Acting%252BMolecules%252Bwith%252BActivity%252Bagainst%252BDifferent%252BSpecies%252BCausing%252BMalaria%252Bin%252BHumans%26aulast%3DMalmquist%26aufirst%3DNicholas%2BA.%26date%3D2015%26date%3D2014%26volume%3D59%26issue%3D2%26spage%3D950%26epage%3D959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sharad K.  Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Progress in the Discovery of Epigenetic Inhibitors for the Treatment of Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 677-688. <a href="https://doi.org/10.1007/978-1-4939-1804-1_35" title="DOI URL">https://doi.org/10.1007/978-1-4939-1804-1_35</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-1804-1_35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-1804-1_35%26sid%3Dliteratum%253Aachs%26atitle%3DRecent%252BProgress%252Bin%252Bthe%252BDiscovery%252Bof%252BEpigenetic%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BCancer%26aulast%3DVerma%26aufirst%3DSharad%2BK.%26date%3D2015%26date%3D2014%26spage%3D677%26epage%3D688%26pub%3DSpringer%2520New%2520York%26atitle%3DCancer%252BEpigenetics%26aulast%3DVerma%26aufirst%3DMukesh%26date%3D2015%26volume%3D1238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Zhao</span>, <span class="hlFld-ContribAuthor ">Haitao  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Crystallography-Based Mechanistic Insights into Epigenetic Regulation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 125-147. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00007-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00007-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00007-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00007-7%26sid%3Dliteratum%253Aachs%26atitle%3DCrystallography-Based%252BMechanistic%252BInsights%252Binto%252BEpigenetic%252BRegulation%26aulast%3DZhao%26aufirst%3DShuai%26date%3D2015%26spage%3D125%26epage%3D147%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Zeng</span>, <span class="hlFld-ContribAuthor ">Wei  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Enzymatic Assays of Histone Methyltransferase Enzymes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 333-361. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00016-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00016-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00016-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00016-8%26sid%3Dliteratum%253Aachs%26atitle%3DEnzymatic%252BAssays%252Bof%252BHistone%252BMethyltransferase%252BEnzymes%26aulast%3DZeng%26aufirst%3DHao%26date%3D2015%26spage%3D333%26epage%3D361%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aili  Chen</span>, <span class="hlFld-ContribAuthor ">Gang  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Animal Model Study of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 447-477. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00020-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00020-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00020-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00020-X%26sid%3Dliteratum%253Aachs%26atitle%3DAnimal%252BModel%252BStudy%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DChen%26aufirst%3DAili%26date%3D2015%26spage%3D447%26epage%3D477%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Zhang</span>, <span class="hlFld-ContribAuthor ">Stacie Lynn  Richardson</span>, <span class="hlFld-ContribAuthor ">Yunfei  Mao</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2015,</strong> <em>13 </em>
                                    (14)
                                     , 4149-4154. <a href="https://doi.org/10.1039/C5OB00120J" title="DOI URL">https://doi.org/10.1039/C5OB00120J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5OB00120J%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bkinetic%252Banalysis%252Bof%252Bpotent%252Bprotein%252BN-terminal%252Bmethyltransferase%252B1%252Binhibitors%26aulast%3DZhang%26aufirst%3DGang%26date%3D2015%26date%3D2015%26volume%3D13%26issue%3D14%26spage%3D4149%26epage%3D4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nitipol  Srimongkolpithak</span>, <span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 2,4-diamino-6,7-dimethoxyquinoline derivatives as G9a inhibitors. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1821-1828. <a href="https://doi.org/10.1039/C4MD00274A" title="DOI URL">https://doi.org/10.1039/C4MD00274A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00274A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00274A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DIdentification%252Bof%252B2%25252C4-diamino-6%25252C7-dimethoxyquinoline%252Bderivatives%252Bas%252BG9a%252Binhibitors%26aulast%3DSrimongkolpithak%26aufirst%3DNitipol%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1821%26epage%3D1828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–activity relationship studies of SETD8 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1892-1898. <a href="https://doi.org/10.1039/C4MD00317A" title="DOI URL">https://doi.org/10.1039/C4MD00317A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00317A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00317A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DStructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bof%252BSETD8%252Binhibitors%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1892%26epage%3D1898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Nicholas A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">Joachim  Caron</span>, <span class="hlFld-ContribAuthor ">Scott  Blundell</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Nitipol  Srimongkolpithak</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Susan A.  Charman</span>, <span class="hlFld-ContribAuthor ">Artur  Scherf</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Diaminoquinazoline Histone Lysine Methyltransferase Inhibitors as Potent Blood-Stage Antimalarial Compounds. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2014,</strong> <em>9 </em>
                                    (10)
                                     , 2360-2373. <a href="https://doi.org/10.1002/cmdc.201402098" title="DOI URL">https://doi.org/10.1002/cmdc.201402098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201402098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201402098%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDevelopment%252Bof%252BDiaminoquinazoline%252BHistone%252BLysine%252BMethyltransferase%252BInhibitors%252Bas%252BPotent%252BBlood-Stage%252BAntimalarial%252BCompounds%26aulast%3DSundriyal%26aufirst%3DSandeep%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D10%26spage%3D2360%26epage%3D2373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Xue  Ke</span>, <span class="hlFld-ContribAuthor ">Dunke  Zhang</span>, <span class="hlFld-ContribAuthor ">Shunqin  Zhu</span>, <span class="hlFld-ContribAuthor ">Qingyou  Xia</span>, <span class="hlFld-ContribAuthor ">Zhonghuai  Xiang</span>, <span class="hlFld-ContribAuthor ">Hongjuan  Cui</span>, . </span><span class="cited-content_cbyCitation_article-title">Inhibition of H3K9 Methyltransferase G9a Repressed Cell Proliferation and Induced Autophagy in Neuroblastoma Cells. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (9)
                                     , e106962. <a href="https://doi.org/10.1371/journal.pone.0106962" title="DOI URL">https://doi.org/10.1371/journal.pone.0106962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0106962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0106962%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DInhibition%252Bof%252BH3K9%252BMethyltransferase%252BG9a%252BRepressed%252BCell%252BProliferation%252Band%252BInduced%252BAutophagy%252Bin%252BNeuroblastoma%252BCells%26aulast%3DKe%26aufirst%3DXiao-Xue%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3De106962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonella  Di Costanzo</span>, <span class="hlFld-ContribAuthor ">Nunzio  Del Gaudio</span>, <span class="hlFld-ContribAuthor ">Antimo  Migliaccio</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic drugs against cancer: an evolving landscape. </span><span class="cited-content_cbyCitation_journal-name">Archives of Toxicology</span><span> <strong>2014,</strong> <em>88 </em>
                                    (9)
                                     , 1651-1668. <a href="https://doi.org/10.1007/s00204-014-1315-6" title="DOI URL">https://doi.org/10.1007/s00204-014-1315-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00204-014-1315-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00204-014-1315-6%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Toxicology%26atitle%3DEpigenetic%252Bdrugs%252Bagainst%252Bcancer%25253A%252Ban%252Bevolving%252Blandscape%26aulast%3DDi%2BCostanzo%26aufirst%3DAntonella%26date%3D2014%26date%3D2014%26volume%3D88%26issue%3D9%26spage%3D1651%26epage%3D1668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Fjm100478y&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm100478y%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2010.53.issue-15%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=143&amp;pagesCount=2&amp;pageNumber=-1" title="Load all citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load all citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of <b>3a</b> (BIX01294).<a onclick="showRef(event, 'ref22 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref29">(22, 29)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The lipophilic benzyl group (top) is exposed to solvent and does not makes any interactions with the protein in the X-ray crystal structure of the GLP−<b>3a</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FPD">3FPD</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 2,4-Diamino-6,7-dimethoxy Quinazolines <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) R′ amines, DMF, DIEA, RT; (b) R′′ amines, <i>i</i>-PrOH, 4 M HCl/dioxane, microwave, 160 °C.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>3a</b> (gray, red, and blue) does not occupy the lysine binding channel in the X-ray cocrystal structure of the GLP−<b>3a</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FPD">3FPD</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> A fragment of the histone peptide (magenta) was transposed into this crystal structure to illustrate the lysine binding channel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) BnBr, K<sub>2</sub>CO<sub>3</sub>, DMF, RT; (b) HNO<sub>3</sub>, Ac<sub>2</sub>O, 0 °C to RT; (c) Fe dust, NH<sub>4</sub>Cl, <i>i</i>-PrOH-H<sub>2</sub>O, reflux, 67% over 3 steps; (d) methyl chloroformate, DIEA, DMF-DCM, 0 °C to RT; (e) NaOH, H<sub>2</sub>O<sub>2</sub>, EtOH, reflux, 70% over 2 steps; (f) <i>N</i>,<i>N</i>-diethylaniline, POCl<sub>3</sub>, reflux, 59%; (g) 4-amino-1-methylpiperidine, DIEA, THF, RT; (h) 1-methylhomopiperazine, HCl, <i>i</i>-PrOH, 160 °C, microwave, 82% over 2 steps; (i) Pd/C, 1,4-cyclohexadiene, EtOH, reflux; (j) 3-(dimethylamino)propan-1-ol, PPh<sub>3</sub>, DIAD, THF, 0 °C to RT, 46% over 2 steps.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystal structure of the G9a−<b>10</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a>). The inhibitor−protein interactions are annotated as follows: “Es”, electrostatic; “Hb”, hydrogen bonding; “⊕π”, cation−π interactions; “(s)”, interaction with a side chain; “(b)”, interaction with the backbone.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. The 7-dimethylamino propoxy side chain (gray, blue, and red) of <b>10</b> does not fully occupy the lysine binding channel in the X-ray crystal structure of the G9a−<b>10</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K">3K5K</a>). A fragment of the H3K9 peptide (magenta) was transposed into this crystal structure to illustrate the lysine binding channel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>11</b>−<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) various alcohols, PPh<sub>3</sub>, DIAD, THF, 0 °C to RT; (b) TFA, DCM, RT; (c) various amines, <i>i</i>-PrOH, 160 °C, microwave.</p></p></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/medium/jm-2010-00478y_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0016.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Morrison <i>K</i><sub>i</sub>s of compounds <b>29</b>, <b>10</b>, and <b>3a</b> in the G9a MCE assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-15/jm100478y/production/images/large/jm-2010-00478y_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100478y&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i65">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24775" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24775" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 41 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Bernstein, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">The mammalian epigenome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0li_KrStvOLe6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Gelato, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischle, W.</span><span> </span><span class="NLM_article-title">Role of histone modifications in defining chromatin structure and function</span> <span class="citation_source-journal">Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1515%2FBC.2008.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18225984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFait7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2008&pages=353-363&author=K.+A.+Gelatoauthor=W.+Fischle&title=Role+of+histone+modifications+in+defining+chromatin+structure+and+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Role of histone modifications in defining chromatin structure and function</span></div><div class="casAuthors">Gelato, Kathy A.; Fischle, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">353-363</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Chromosomes in eukaryotic cell nuclei are not uniformly organized, but rather contain distinct chromatin elements, with each state having a defined biochem. structure and biol. function.  These are recognizable by their distinct architectures and mol. components, which can change in response to cellular stimuli or metabolic requirements.  Chromatin elements are characterized by the fundamental histone and DNA components, as well as other assocd. non-histone proteins and factors.  Post-translational modifications of histones in particular often correlate with a specific chromatin structure and function.  Patterns of histone modifications are implicated as having a role in directing the level of chromatin compaction, as well as playing roles in multiple functional pathways directing the readout of distinct regions of the genome.  Here, the authors review the properties of various chromatin elements and the apparent links of histone modifications with chromatin organization and functional output.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm8YRKkBO4w7Vg90H21EOLACvtfcHk0li_KrStvOLe6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksFait7s%253D&md5=14b1db5766047ab8201007c5cde0405d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1515%2FBC.2008.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2008.048%26sid%3Dliteratum%253Aachs%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DFischle%26aufirst%3DW.%26atitle%3DRole%2520of%2520histone%2520modifications%2520in%2520defining%2520chromatin%2520structure%2520and%2520function%26jtitle%3DBiol.%2520Chem.%26date%3D2008%26volume%3D389%26spage%3D353%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">The language of covalent histone modifications</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">403</span><span class="NLM_x">, </span> <span class="NLM_fpage">41</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2F47412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=10638745" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=403&publication_year=2000&pages=41-45&author=B.+D.+Strahlauthor=C.+D.+Allis&title=The+language+of+covalent+histone+modifications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The language of covalent histone modifications</span></div><div class="casAuthors">Strahl B D; Allis C D</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">403</span>
        (<span class="NLM_cas:issue">6765</span>),
    <span class="NLM_cas:pages">41-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Histone proteins and the nucleosomes they form with DNA are the fundamental building blocks of eukaryotic chromatin.  A diverse array of post-translational modifications that often occur on tail domains of these proteins has been well documented.  Although the function of these highly conserved modifications has remained elusive, converging biochemical and genetic evidence suggests functions in several chromatin-based processes.  We propose that distinct histone modifications, on one or more tails, act sequentially or in combination to form a 'histone code' that is, read by other proteins to bring about distinct downstream events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSyzhdlYh9sX7t7lRqEFUJVfW6udTcc2eY_KFB3FgUBqbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c7gt1arsQ%253D%253D&md5=2eb7d093151c6122ccd70fa2adcfd94b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F47412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F47412%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DThe%2520language%2520of%2520covalent%2520histone%2520modifications%26jtitle%3DNature%26date%3D2000%26volume%3D403%26spage%3D41%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lgfoFDu22AtXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Esteller, M.</span><span> </span><span class="NLM_article-title">Epigenetics in cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">358</span><span class="NLM_x">, </span> <span class="NLM_fpage">1148</span><span class="NLM_x">–</span> <span class="NLM_lpage">1159</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1056%2FNEJMra072067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18337604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1ahs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2008&pages=1148-1159&author=M.+Esteller&title=Epigenetics+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics in cancer</span></div><div class="casAuthors">Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1148-1159</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The epigenetic therapy in cancer is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzzFSlZotpmLVg90H21EOLACvtfcHk0lgfoFDu22AtXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1ahs7Y%253D&md5=c354604e711e60b49e37f151f14a5dba</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra072067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra072067%26sid%3Dliteratum%253Aachs%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DEpigenetics%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2008%26volume%3D358%26spage%3D1148%26epage%3D1159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Lyko, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R.</span><span> </span><span class="NLM_article-title">DNA methyltransferase inhibitors and the development of epigenetic cancer therapies</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1498</span><span class="NLM_x">–</span> <span class="NLM_lpage">1506</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1093%2Fjnci%2Fdji311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16234563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCjtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2005&pages=1498-1506&author=F.+Lykoauthor=R.+Brown&title=DNA+methyltransferase+inhibitors+and+the+development+of+epigenetic+cancer+therapies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Methyltransferase Inhibitors and the Development of Epigenetic Cancer Therapies</span></div><div class="casAuthors">Lyko, Frank; Brown, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1498-1506</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Epimutations, such as the hypermethylation and epigenetic silencing of tumor suppressor genes, play a role in the etiol. of human cancers.  In contrast to DNA mutations, which are passively inherited through DNA replication, epimutations must be actively maintained because they are reversible.  In fact, the reversibility of epimutations by small-mol. inhibitors provides the foundation for the use of such inhibitors in novel cancer therapy strategies.  Among the compds. that inhibit epigenetic processes, the most extensively studied are DNA methyltransferase inhibitors.  In this review, we examine the literature on DNA methyltransferase inhibitors and discuss the efficacy of such compds. as antitumor agents, as evaluated in phase I-III clin. trials.  We also discuss future areas of research, including the development of nonnucleoside inhibitors, the application of novel bioanal. tools for DNA methylation anal. (which will be important for the clin. application of these compds. by allowing rational approaches to trial design), the need to optimize treatment schedules for maximal biol. effectiveness, and the need to define mol. endpoints so that changes induced by demethylating drugs in patients can be monitored during treatment.  Assays for genome-wide and tumor-specific DNA methylation also need to be further developed to establish the pharmacodynamic parameters of DNA methyltransferase inhibitors in patients and to provide rational approaches to maximizing the therapeutic efficacy of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2i9sqxc3_wrVg90H21EOLACvtfcHk0liMPsQ8XiWyPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCjtLfF&md5=6edee05ea71b9e68adf1b811fe592ddd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdji311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdji311%26sid%3Dliteratum%253Aachs%26aulast%3DLyko%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DR.%26atitle%3DDNA%2520methyltransferase%2520inhibitors%2520and%2520the%2520development%2520of%2520epigenetic%2520cancer%2520therapies%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2005%26volume%3D97%26spage%3D1498%26epage%3D1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nature Rev. Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lhPRan4IplXiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNature%2520Rev.%2520Mol.%2520Cell.%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenhaber, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Carroll, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponting, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">406</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2F35020506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=10949293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=406&publication_year=2000&pages=593-599&author=S.+Reaauthor=F.+Eisenhaberauthor=D.+O%E2%80%99Carrollauthor=B.+D.+Strahlauthor=Z.+W.+Sunauthor=M.+Schmidauthor=S.+Opravilauthor=K.+Mechtlerauthor=C.+P.+Pontingauthor=C.+D.+Allisauthor=T.+Jenuwein&title=Regulation+of+chromatin+structure+by+site-specific+histone+H3+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin structure by site-specific histone H3 methyltransferases</span></div><div class="casAuthors">Rea, Stephen; Elsenhaber, Frank; O'Carroll, Donal; Strahl, Brian D.; Sun, Zu-Wen; Schmid, Manfred; Opravil, Susanne; Mechtler, Karl; Ponting, Chris P.; Allis, C. David; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">6796</span>),
    <span class="NLM_cas:pages">593-599</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The organization of chromatin into higher-order structures influences chromosome function and epigenetic gene regulation.  Higher-order chromatin has been proposed to be nucleated by the covalent modification of histone tails and the subsequent establishment of chromosomal subdomains by non-histone modifier factors.  Here we show that human SUV39H1 and murine Suv39h1 - mammalian homologs of Drosophila Su(var)3-9 and of Schizosaccharomyces pombe clr4 - encode histone H3-specific methyltransferases that selectively methylate lysine 9 of the amino terminus of histone H3 in vitro.  We mapped the catalytic motif to the evolutionarily conserved SET domain, which requires adjacent cysteine-rich regions to confer histone methyltransferase activity.  Methylation of lysine 9 interferes with phosphorylation of serine 10, but is also influenced by pre-existing modifications in the amino terminus of H3.  In vivo, deregulated SUV39H1 or disrupted Suv39h activity modulate H3 serine 10 phosphorylation in native chromatin and induce aberrant mitotic divisions.  Our data reveal a functional interdependence of site-specific H3 tail modifications and suggest a dynamic mechanism for the regulation of higher-order chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2yioyTahy77Vg90H21EOLACvtfcHk0lhPRan4IplXiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmtFegs7s%253D&md5=65e14733d25f97fadaaeed602600bd23</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2F35020506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35020506%26sid%3Dliteratum%253Aachs%26aulast%3DRea%26aufirst%3DS.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DO%25E2%2580%2599Carroll%26aufirst%3DD.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DSun%26aufirst%3DZ.%2BW.%26aulast%3DSchmid%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520structure%2520by%2520site-specific%2520histone%2520H3%2520methyltransferases%26jtitle%3DNature%26date%3D2000%26volume%3D406%26spage%3D593%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lgEeY4J66915Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nature Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNature%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Cancer treatment of the future: inhibitors of histone methyltransferases</span> <span class="citation_source-journal">Int. J. Biochem. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.biocel.2008.07.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18773966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWnu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=4-11&author=A.+Spannhoffauthor=W.+Sipplauthor=M.+Jung&title=Cancer+treatment+of+the+future%3A+inhibitors+of+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer treatment of the future: Inhibitors of histone methyltransferases</span></div><div class="casAuthors">Spannhoff, Astrid; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-11</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer in humans is the result of a multi-step process.  This process often involves the activation of oncogenes and/or the inactivation of tumor suppressor genes.  These 2 steps arise not only due to mutations, but can also be the result of a translocation or an altered transcription rate.  One important mechanism is the occurrence of epigenetic alterations like promotor methylation (which may lead to tumor suppressor silencing) or decreased histone acetylation (which can result in the downregulation of proteins involved in apoptosis).  Today, histone acetylation and DNA methylation are epigenetic modifications which were linked closely to the pathol. of human cancers and inhibitors of both enzyme classes for clin. use are at hand.  In contrast, other fields of epigenetics still lack of similarly thorough knowledge.  This is esp. true for the group of histone methyltransferases and their inhibitors.  Since connections between histone methylation patterns and cancer progression were recognized, histone methyltransferases represent promising targets for future cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSu9HVhLFV07Vg90H21EOLACvtfcHk0lgEeY4J66915Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWnu73O&md5=09e514f87c700b4a5b2393d9608b1af1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2008.07.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2008.07.024%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DCancer%2520treatment%2520of%2520the%2520future%253A%2520inhibitors%2520of%2520histone%2520methyltransferases%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2009%26volume%3D41%26spage%3D4%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Fog, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lund, A. H.</span><span> </span><span class="NLM_article-title">Chromatin-modifying proteins in cancer</span> <span class="citation_source-journal">Acta Pathol., Microbiol. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">1060</span><span class="NLM_x">–</span> <span class="NLM_lpage">1089</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1111%2Fj.1600-0463.2007.apm_776.xml.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18042144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2007&pages=1060-1089&author=C.+K.+Fogauthor=K.+T.+Jensenauthor=A.+H.+Lund&title=Chromatin-modifying+proteins+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-modifying proteins in cancer</span></div><div class="casAuthors">Fog, Cathrine K.; Jensen, Klaus T.; Lund, Anders H.</div><div class="citationInfo"><span class="NLM_cas:title">APMIS</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1060-1089</span>CODEN:
                <span class="NLM_cas:coden">APMSEL</span>;
        ISSN:<span class="NLM_cas:issn">0903-4641</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Chromatin-modifying proteins mold the genome into areas that are accessible for transcriptional activity and areas that are transcriptionally silent.  This epigenetic gene regulation allows for different transcriptional programs to be conducted in different cell types at different timepoints-despite the fact that all cells in the organism contain the same genetic information.  A large amt. of data gathered over the last decades has demonstrated that deregulation of chromatin-modifying proteins is etiol. involved in the development and progression of cancer.  Here we discuss how epigenetic alterations influence cancer development and review known cancer-assocd. alterations in chromatin-modifying proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxhso-Ej5KY7Vg90H21EOLACvtfcHk0lgEeY4J66915Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslOnsQ%253D%253D&md5=060c4f35c69924a72a16b316b773a215</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0463.2007.apm_776.xml.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0463.2007.apm_776.xml.x%26sid%3Dliteratum%253Aachs%26aulast%3DFog%26aufirst%3DC.%2BK.%26aulast%3DJensen%26aufirst%3DK.%2BT.%26aulast%3DLund%26aufirst%3DA.%2BH.%26atitle%3DChromatin-modifying%2520proteins%2520in%2520cancer%26jtitle%3DActa%2520Pathol.%252C%2520Microbiol.%2520Immunol.%26date%3D2007%26volume%3D115%26spage%3D1060%26epage%3D1089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">The emerging therapeutic potential of histone methyltransferase and demethylase inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1568</span><span class="NLM_x">–</span> <span class="NLM_lpage">1582</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1568-1582&author=A.+Spannhoffauthor=A.+T.+Hauserauthor=R.+Heinkeauthor=W.+Sipplauthor=M.+Jung&title=The+emerging+therapeutic+potential+of+histone+methyltransferase+and+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DThe%2520emerging%2520therapeutic%2520potential%2520of%2520histone%2520methyltransferase%2520and%2520demethylase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D1568%26epage%3D1582" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanmugam, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, J.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natarajan, R.</span><span> </span><span class="NLM_article-title">Role of the Histone H3 Lysine 4 Methyltransferase, SET7/9, in the Regulation of NF-{kappa}B-dependent Inflammatory Genes: Relevance to Diabetes and Inflammation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">26771</span><span class="NLM_x">–</span> <span class="NLM_lpage">26781</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=26771-26781&author=Y.+Liauthor=M.+A.+Reddyauthor=F.+Miaoauthor=N.+Shanmugamauthor=J.-K.+Yeeauthor=D.+Hawkinsauthor=B.+Renauthor=R.+Natarajan&title=Role+of+the+Histone+H3+Lysine+4+Methyltransferase%2C+SET7%2F9%2C+in+the+Regulation+of+NF-%7Bkappa%7DB-dependent+Inflammatory+Genes%3A+Relevance+to+Diabetes+and+Inflammation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DReddy%26aufirst%3DM.%2BA.%26aulast%3DMiao%26aufirst%3DF.%26aulast%3DShanmugam%26aufirst%3DN.%26aulast%3DYee%26aufirst%3DJ.-K.%26aulast%3DHawkins%26aufirst%3DD.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DNatarajan%26aufirst%3DR.%26atitle%3DRole%2520of%2520the%2520Histone%2520H3%2520Lysine%25204%2520Methyltransferase%252C%2520SET7%252F9%252C%2520in%2520the%2520Regulation%2520of%2520NF-%257Bkappa%257DB-dependent%2520Inflammatory%2520Genes%253A%2520Relevance%2520to%2520Diabetes%2520and%2520Inflammation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D26771%26epage%3D26781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Maze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, H. E.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPlant, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechanic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neve, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurd, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Essential role of the histone methyltransferase G9a in cocaine-induced plasticity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">327</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1126%2Fscience.1179438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20056891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=213-216&author=I.+Mazeauthor=H.+E.+Covingtonauthor=D.+M.+Dietzauthor=Q.+LaPlantauthor=W.+Renthalauthor=S.+J.+Russoauthor=M.+Mechanicauthor=E.+Mouzonauthor=R.+L.+Neveauthor=S.+J.+Haggartyauthor=Y.+Renauthor=S.+C.+Sampathauthor=Y.+L.+Hurdauthor=P.+Greengardauthor=A.+Tarakhovskyauthor=A.+Schaeferauthor=E.+J.+Nestler&title=Essential+role+of+the+histone+methyltransferase+G9a+in+cocaine-induced+plasticity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Essential Role of the Histone Methyltransferase G9a in Cocaine-Induced Plasticity</span></div><div class="casAuthors">Maze, Ian; Covington, Herbert E., III; Dietz, David M.; LaPlant, Quincey; Renthal, William; Russo, Scott J.; Mechanic, Max; Mouzon, Ezekiell; Neve, Rachael L.; Haggarty, Stephen J.; Ren, Yanhua; Sampath, Srihari C.; Hurd, Yasmin L.; Greengard, Paul; Tarakhovsky, Alexander; Schaefer, Anne; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5962</span>),
    <span class="NLM_cas:pages">213-216</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cocaine-induced alterations in gene expression cause changes in neuronal morphol. and behavior that may underlie cocaine addiction.  In mice, we identified an essential role for histone 3 lysine 9 (H3K9) dimethylation and the lysine dimethyltransferase G9a in cocaine-induced structural and behavioral plasticity.  Repeated cocaine administration reduced global levels of H3K9 dimethylation in the nucleus accumbens.  This redn. in histone methylation was mediated through the repression of G9a in this brain region, which was regulated by the cocaine-induced transcription factor ΔFosB.  Using conditional mutagenesis and viral-mediated gene transfer, we found that G9a down-regulation increased the dendritic spine plasticity of nucleus accumbens neurons and enhanced the preference for cocaine, thereby establishing a crucial role for histone methylation in the long-term actions of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SPD826wEGrVg90H21EOLACvtfcHk0lhTRYZzrErIPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1egtA%253D%253D&md5=a8347500f81862078787e50fff53b6b9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.1179438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1179438%26sid%3Dliteratum%253Aachs%26aulast%3DMaze%26aufirst%3DI.%26aulast%3DCovington%26aufirst%3DH.%2BE.%26aulast%3DDietz%26aufirst%3DD.%2BM.%26aulast%3DLaPlant%26aufirst%3DQ.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DRusso%26aufirst%3DS.%2BJ.%26aulast%3DMechanic%26aufirst%3DM.%26aulast%3DMouzon%26aufirst%3DE.%26aulast%3DNeve%26aufirst%3DR.%2BL.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DHurd%26aufirst%3DY.%2BL.%26aulast%3DGreengard%26aufirst%3DP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEssential%2520role%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520in%2520cocaine-induced%2520plasticity%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D213%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Schaefer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Intrator, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gertler, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surmeier, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greengard, P.</span><span> </span><span class="NLM_article-title">Control of Cognition and Adaptive Behavior by the GLP/G9a Epigenetic Suppressor Complex</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.neuron.2009.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20005824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2009&pages=678-691&author=A.+Schaeferauthor=S.+C.+Sampathauthor=A.+Intratorauthor=A.+Minauthor=T.+S.+Gertlerauthor=D.+J.+Surmeierauthor=A.+Tarakhovskyauthor=P.+Greengard&title=Control+of+Cognition+and+Adaptive+Behavior+by+the+GLP%2FG9a+Epigenetic+Suppressor+Complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Control of cognition and adaptive behavior by the GLP/G9a epigenetic suppressor complex</span></div><div class="casAuthors">Schaefer, Anne; Sampath, Srihari C.; Intrator, Adam; Min, Alice; Gertler, Tracy S.; Surmeier, D. James; Tarakhovsky, Alexander; Greengard, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">678-691</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The genetic basis of cognition and behavioral adaptation to the environment remains poorly understood.  Here we demonstrate that the histone methyltransferase complex GLP/G9a controls cognition and adaptive responses in a region-specific fashion in the adult brain.  Using conditional mutagenesis in mice, we show that postnatal, neuron-specific deficiency of GLP/G9a leads to derepression of numerous nonneuronal and neuron progenitor genes in adult neurons.  This transcriptional alteration is assocd. with complex behavioral abnormalities, including defects in learning, motivation, and environmental adaptation.  The behavioral changes triggered by GLP/G9a deficiency are similar to key symptoms of the human 9q34 mental retardation syndrome that is assocd. with structural alterations of the GLP/EHMT1 gene.  The likely causal role of GLP/G9a in mental retardation in mice and humans suggests a key role for the GLP/G9a-controlled histone H3K9 dimethylation in regulation of brain function through maintenance of the transcriptional homeostasis in adult neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr22UcjjqkKdLVg90H21EOLACvtfcHk0lhTRYZzrErIPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Cls7c%253D&md5=ece4e94ad097a8d0e67a5e8d27970e64</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2009.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2009.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DSchaefer%26aufirst%3DA.%26aulast%3DSampath%26aufirst%3DS.%2BC.%26aulast%3DIntrator%26aufirst%3DA.%26aulast%3DMin%26aufirst%3DA.%26aulast%3DGertler%26aufirst%3DT.%2BS.%26aulast%3DSurmeier%26aufirst%3DD.%2BJ.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DGreengard%26aufirst%3DP.%26atitle%3DControl%2520of%2520Cognition%2520and%2520Adaptive%2520Behavior%2520by%2520the%2520GLP%252FG9a%2520Epigenetic%2520Suppressor%2520Complex%26jtitle%3DNeuron%26date%3D2009%26volume%3D64%26spage%3D678%26epage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Imai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togami, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, T.</span><span> </span><span class="NLM_article-title">Involvement of histone H3 Lysine 9 (H3K9) methyl transferase G9a in the maintenance of HIV-1 latency and its reactivation by BIX01294</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">16538</span><span class="NLM_x">–</span> <span class="NLM_lpage">16545</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=16538-16545&author=K.+Imaiauthor=H.+Togamiauthor=T.+Okamoto&title=Involvement+of+histone+H3+Lysine+9+%28H3K9%29+methyl+transferase+G9a+in+the+maintenance+of+HIV-1+latency+and+its+reactivation+by+BIX01294"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DImai%26aufirst%3DK.%26aulast%3DTogami%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DT.%26atitle%3DInvolvement%2520of%2520histone%2520H3%2520Lysine%25209%2520%2528H3K9%2529%2520methyl%2520transferase%2520G9a%2520in%2520the%2520maintenance%2520of%2520HIV-1%2520latency%2520and%2520its%2520reactivation%2520by%2520BIX01294%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D16538%26epage%3D16545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Tachibana, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nozaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span> </span><span class="NLM_article-title">G9a histone methyltransferase plays a dominant role in euchromatic histone H3 lysine 9 methylation and is essential for early embryogenesis</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1779</span><span class="NLM_x">–</span> <span class="NLM_lpage">1791</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1779-1791&author=M.+Tachibanaauthor=K.+Sugimotoauthor=M.+Nozakiauthor=J.+Uedaauthor=T.+Ohtaauthor=M.+Ohkiauthor=M.+Fukudaauthor=N.+Takedaauthor=H.+Niidaauthor=H.+Katoauthor=Y.+Shinkai&title=G9a+histone+methyltransferase+plays+a+dominant+role+in+euchromatic+histone+H3+lysine+9+methylation+and+is+essential+for+early+embryogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTachibana%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DK.%26aulast%3DNozaki%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DJ.%26aulast%3DOhta%26aufirst%3DT.%26aulast%3DOhki%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DM.%26aulast%3DTakeda%26aufirst%3DN.%26aulast%3DNiida%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DShinkai%26aufirst%3DY.%26atitle%3DG9a%2520histone%2520methyltransferase%2520plays%2520a%2520dominant%2520role%2520in%2520euchromatic%2520histone%2520H3%2520lysine%25209%2520methylation%2520and%2520is%2520essential%2520for%2520early%2520embryogenesis%26jtitle%3DGenes%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D1779%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">McGarvey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahrner, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Silenced tumor suppressor genes reactivated by DNA demethylation do not return to a fully euchromatic chromatin state</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">3541</span><span class="NLM_x">–</span> <span class="NLM_lpage">3549</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1158%2F0008-5472.CAN-05-2481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16585178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=3541-3549&author=K.+M.+McGarveyauthor=J.+A.+Fahrnerauthor=E.+Greeneauthor=J.+Martensauthor=T.+Jenuweinauthor=S.+B.+Baylin&title=Silenced+tumor+suppressor+genes+reactivated+by+DNA+demethylation+do+not+return+to+a+fully+euchromatic+chromatin+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Silenced Tumor Suppressor Genes Reactivated by DNA Demethylation Do Not Return to a Fully Euchromatic Chromatin State</span></div><div class="casAuthors">McGarvey, Kelly M.; Fahrner, Jill A.; Greene, Eriko; Martens, Joost; Jenuwein, Thomas; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3541-3549</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone H3 lysine 9 (H3K9) and lysine 27 (H3K27) trimethylation are properties of stably silenced heterochromatin whereas H3K9 dimethylation (H3K9me2) is important for euchromatic gene repression.  In colorectal cancer cells, all of these marks, as well as the key enzymes which establish them, surround the hMLH1 promoter when it is DNA hypermethylated and aberrantly silenced, but are absent when the gene is unmethylated and fully expressed in a euchromatic state.  When the aberrantly silenced gene is DNA demethylated and reexpressed following 5-aza-2'-deoxycytidine treatment, H3K9me1 and H3K9me2 are the only silencing marks that are lost.  A series of other silenced and DNA hypermethylated gene promoters behave identically even when the genes are chronically DNA demethylated and reexpressed after genetic knockout of DNA methyltransferases.  Our data indicate that when transcription of DNA hypermethylated genes is activated in cancer cells, their promoters remain in an environment with certain heterochromatic characteristics.  This finding has important implications for the translational goal of reactivating aberrantly silenced cancer genes as a therapeutic maneuver.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEjoUm5wwMFbVg90H21EOLACvtfcHk0ljo8alzXzB7tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtVOrsro%253D&md5=73b1830a5f7f839d53d5539c6e707462</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-2481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-2481%26sid%3Dliteratum%253Aachs%26aulast%3DMcGarvey%26aufirst%3DK.%2BM.%26aulast%3DFahrner%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DJ.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSilenced%2520tumor%2520suppressor%2520genes%2520reactivated%2520by%2520DNA%2520demethylation%2520do%2520not%2520return%2520to%2520a%2520fully%2520euchromatic%2520chromatin%2520state%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D3541%26epage%3D3549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Kondo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boumber, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekido, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haddad, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, J. P.</span><span> </span><span class="NLM_article-title">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span> <span class="citation_source-journal">PLoS ONE</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e2037</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1371%2Fjournal.pone.0002037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18446223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e2037&author=Y.+Kondoauthor=L.+Shenauthor=S.+Ahmedauthor=Y.+Boumberauthor=Y.+Sekidoauthor=B.+R.+Haddadauthor=J.+P.+Issa&title=Downregulation+of+histone+H3+lysine+9+methyltransferase+G9a+induces+centrosome+disruption+and+chromosome+instability+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Downregulation of histone H3 lysine 9 methyltransferase G9a induces centrosome disruption and chromosome instability in cancer cells</span></div><div class="casAuthors">Kondo Yutaka; Shen Lanlan; Ahmed Saira; Boumber Yanis; Sekido Yoshitaka; Haddad Bassem R; Issa Jean-Pierre J</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e2037</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Modifications of the histone amino-terminal tails affect access of regulatory factors and complexes to chromatin and thereby influence biological processes.  Cancer cells are characterized by prominent epigenetic dysregulation, including histone modifications.  However, the functional roles of the histone methyltransferases (HMT) in cancer remain unclear.  METHODOLOGY/PRINCIPAL FINDINGS:  We studied RNAi-based inhibition (knockdown, KD) of 2 different H3K9 HMTs, SUV39H1 and G9a.  Knockdown of the 2 HMTs in PC3 cancer cell line markedly inhibited cell growth and caused profound morphological changes with loss of telomerase activity and shortened telomeres.  SUV39H1 KD cells showed substantial increase in G2/M fraction.  G9a KD cells showed increased DNA content (1.7-fold in 2 independent clones) compared with FACS analyses to control.  Karyotype analyses showed that this was due to an increased number of chromosomes (from 61 to 102) in G9a KD cells compared to parental PC3.  Intriguingly, we found abnormal centrosome morphology and number in about 25% of the G9a KD cells, while centrosomes were morphologically normal in control cells.  Microarray analyses after KD of SUV39H1 or G9a showed very few genes up-regulated among the 39,000 genes.  The silenced tumor-suppressor genes p16 and RASSF1A were not activated in KD cells.  CONCLUSIONS/SIGNIFICANCE:  These data suggest that the 2 HMTs, SUV39H1 and G9a are required to perpetuate the malignant phenotype.  Furthermore, G9a plays a critical role in regulating centrosome duplication presumably through chromatin structure rather than through affecting gene expression in cancer cells.  Targeting these histone methyltransferases may be of therapeutic benefit in cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhnx1F8nyNwDEwO064Z95rfW6udTcc2eaaEpKZS__pcLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3otlyhtw%253D%253D&md5=22936eeeffa4dc29e7254dfc546ebcfd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0002037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0002037%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DAhmed%26aufirst%3DS.%26aulast%3DBoumber%26aufirst%3DY.%26aulast%3DSekido%26aufirst%3DY.%26aulast%3DHaddad%26aufirst%3DB.%2BR.%26aulast%3DIssa%26aufirst%3DJ.%2BP.%26atitle%3DDownregulation%2520of%2520histone%2520H3%2520lysine%25209%2520methyltransferase%2520G9a%2520induces%2520centrosome%2520disruption%2520and%2520chromosome%2520instability%2520in%2520cancer%2520cells%26jtitle%3DPLoS%2520ONE%26date%3D2008%26volume%3D3%26spage%3De2037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">G9a and GLP methylate lysine 373 in the tumor suppressor p53</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">9636</span><span class="NLM_x">–</span> <span class="NLM_lpage">9641</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1074%2Fjbc.M109.062588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20118233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=9636-9641&author=J.+Huangauthor=J.+Dorseyauthor=S.+Chuikovauthor=X.+Zhangauthor=T.+Jenuweinauthor=D.+Reinbergauthor=S.+L.+Berger&title=G9a+and+GLP+methylate+lysine+373+in+the+tumor+suppressor+p53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">G9a and Glp Methylate Lysine 373 in the Tumor Suppressor p53</span></div><div class="casAuthors">Huang, Jing; Dorsey, Jean; Chuikov, Sergei; Zhang, Xinyue; Jenuwein, Thomas; Reinberg, Danny; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9636-9641</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is regulated by numerous post-translational modifications.  Lysine methylation has recently emerged as a key post-translational modification that alters the activity of p53.  Here, we describe a novel lysine methylation site in p53 that is carried out by two homologous histone methyltransferases, G9a and Glp.  G9a and Glp specifically methylate p53 at Lys373, resulting mainly in dimethylation.  During DNA damage, the overall level of p53 modified at Lys373me2 does not increase, despite the dramatic increase in total p53, indicating that Lys373me2 correlates with inactive p53.  Further, redn. of G9a and/or Glp levels leads to a larger population of apoptotic cells.  Examn. of the Oncomine data base shows that G9a and Glp are overexpressed in various cancers compared with corresponding normal tissues, suggesting that they are putative oncogenes.  These data reveal a new methylation site within p53 mediated by the methylases G9a and Glp and indicate that G9a is a potential inhibitory target for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozJXSsZlEE9LVg90H21EOLACvtfcHk0ljo8alzXzB7tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsFeht7w%253D&md5=81adc2281f6344caf9a46ebcdf1c61e6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062588%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDorsey%26aufirst%3DJ.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DG9a%2520and%2520GLP%2520methylate%2520lysine%2520373%2520in%2520the%2520tumor%2520suppressor%2520p53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D9636%26epage%3D9641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0lgv457C7pjkKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desponts, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahm, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholer, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, S.</span><span> </span><span class="NLM_article-title">A combined chemical and genetic approach for the generation of induced pluripotent stem cells</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">528</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2008&pages=525-528&author=Y.+Shiauthor=J.+T.+Doauthor=C.+Despontsauthor=H.+S.+Hahmauthor=H.+R.+Scholerauthor=S.+Ding&title=A+combined+chemical+and+genetic+approach+for+the+generation+of+induced+pluripotent+stem+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDo%26aufirst%3DJ.%2BT.%26aulast%3DDesponts%26aufirst%3DC.%26aulast%3DHahm%26aufirst%3DH.%2BS.%26aulast%3DScholer%26aufirst%3DH.%2BR.%26aulast%3DDing%26aufirst%3DS.%26atitle%3DA%2520combined%2520chemical%2520and%2520genetic%2520approach%2520for%2520the%2520generation%2520of%2520induced%2520pluripotent%2520stem%2520cells%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2008%26volume%3D2%26spage%3D525%26epage%3D528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desponts, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahm, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholer, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, S.</span><span> </span><span class="NLM_article-title">Induction of pluripotent stem cells from mouse embryonic fibroblasts by Oct4 and Klf4 with small-molecule compounds</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">568</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=568-574&author=Y.+Shiauthor=C.+Despontsauthor=J.+T.+Doauthor=H.+S.+Hahmauthor=H.+R.+Scholerauthor=S.+Ding&title=Induction+of+pluripotent+stem+cells+from+mouse+embryonic+fibroblasts+by+Oct4+and+Klf4+with+small-molecule+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDesponts%26aufirst%3DC.%26aulast%3DDo%26aufirst%3DJ.%2BT.%26aulast%3DHahm%26aufirst%3DH.%2BS.%26aulast%3DScholer%26aufirst%3DH.%2BR.%26aulast%3DDing%26aufirst%3DS.%26atitle%3DInduction%2520of%2520pluripotent%2520stem%2520cells%2520from%2520mouse%2520embryonic%2520fibroblasts%2520by%2520Oct4%2520and%2520Klf4%2520with%2520small-molecule%2520compounds%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2008%26volume%3D3%26spage%3D568%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Greiner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonaldi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eskeland, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roemer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span> </span><span class="NLM_article-title">Identification of a specific inhibitor of the histone methyltransferase SU(VAR)3−9</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">145</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnchembio721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=16408017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1ygsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2005&pages=143-145&author=D.+Greinerauthor=T.+Bonaldiauthor=R.+Eskelandauthor=E.+Roemerauthor=A.+Imhof&title=Identification+of+a+specific+inhibitor+of+the+histone+methyltransferase+SU%28VAR%293%E2%88%929"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a specific inhibitor of the histone methyltransferase SU(VAR)3-9</span></div><div class="casAuthors">Greiner, Dorothea; Bonaldi, Tiziana; Eskeland, Ragnhild; Roemer, Ernst; Imhof, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">143-145</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation plays a key role in establishing and maintaining stable gene expression patterns during cellular differentiation and embryonic development.  Here, we report the characterization of the fungal metabolite chaetocin as the first inhibitor of a lysine-specific histone methyltransferase.  Chaetocin is specific for the methyltransferase SU(VAR)3-9 both in vitro and in vivo and may therefore be used to study heterochromatin-mediated gene repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_LIt2UeAM9rVg90H21EOLACvtfcHk0lgv457C7pjkKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1ygsr4%253D&md5=27aa101782ac63ef70b7e8c3d57ab2fe</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio721%26sid%3Dliteratum%253Aachs%26aulast%3DGreiner%26aufirst%3DD.%26aulast%3DBonaldi%26aufirst%3DT.%26aulast%3DEskeland%26aufirst%3DR.%26aulast%3DRoemer%26aufirst%3DE.%26aulast%3DImhof%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520a%2520specific%2520inhibitor%2520of%2520the%2520histone%2520methyltransferase%2520SU%2528VAR%25293%25E2%2588%25929%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2005%26volume%3D1%26spage%3D143%26epage%3D145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Chemical probes for histone-modifying enzymes</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnchembio.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18800048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=590-597&author=P.+A.+Cole&title=Chemical+probes+for+histone-modifying+enzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for histone-modifying enzymes</span></div><div class="casAuthors">Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">590-597</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The histone-modifying enzymes that catalyze reversible lysine acetylation and methylation are central to the epigenetic regulation of chromatin remodeling.  From the early discovery of histone deacetylase inhibitors to the more recent identification of histone demethylase blockers, chem. approaches offer increasingly sophisticated tools for the investigation of the structure and function of these lysine-modifying enzymes.  This review summarizes progress to date on compds. identified from screens or by design that can modulate the activity of classical histone deacetylases, sirtuins, histone acetyltransferases, histone methyltransferases and histone demethylases.  We highlight applications of compds. to mechanistic and functional studies involving these enzymes and discuss future challenges regarding target specificity and general utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ5RKIrQsO7Vg90H21EOLACvtfcHk0ljUR3EGDrhWJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL&md5=71fe323c232af42b67a7dfc134c4d49b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.111%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DChemical%2520probes%2520for%2520histone-modifying%2520enzymes%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D590%26epage%3D597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Frye, S. V.; Heightman, T.; Jin, J.</span><span> </span><span class="NLM_article-title">Targeting Methyl Lysine</span>.  <span class="citation_source-journal">Annu. Rep. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span>in press, DOI: <span class="refDoi"> DOI: 10.1016/S0065-7743(10)45020-4</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2FS0065-7743%2810%2945020-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=S.+V.+Frye&author=T.+Heightman&author=J.+Jin&title=Targeting+Methyl+Lysine&doi=10.1016%2FS0065-7743%2810%2945020-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0065-7743%2810%2945020-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-7743%252810%252945020-4%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DTargeting%2520Methyl%2520Lysine%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2010%26doi%3D10.1016%2FS0065-7743%2810%2945020-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0ljUR3EGDrhWJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upadhyay, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">317</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnsmb.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=19219047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=312-317&author=Y.+Changauthor=X.+Zhangauthor=J.+R.+Hortonauthor=A.+K.+Upadhyayauthor=A.+Spannhoffauthor=J.+Liuauthor=J.+P.+Snyderauthor=M.+T.+Bedfordauthor=X.+Cheng&title=Structural+basis+for+G9a-like+protein+lysine+methyltransferase+inhibition+by+BIX-01294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for G9a-like protein lysine methyltransferase inhibition by BIX-01294</span></div><div class="casAuthors">Chang, Yanqi; Zhang, Xing; Horton, John R.; Upadhyay, Anup K.; Spannhoff, Astrid; Liu, Jin; Snyder, James P.; Bedford, Mark T.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">312-317</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone lysine methylation is an important epigenetic mark that regulates gene expression and chromatin organization.  G9a and G9a-like protein (GLP) are euchromatin-assocd. methyltransferases that repress transcription by methylating histone H3 Lys9.  BIX-01294 was originally identified as a G9a inhibitor during a chem. library screen of small mols. and has previously been used in the generation of induced pluripotent stem cells.  Here we present the crystal structure of the catalytic SET domain of GLP in complex with BIX-01294 and S-adenosyl-L-homocysteine.  The inhibitor is bound in the substrate peptide groove at the location where the histone H3 residues N-terminal to the target lysine lie in the previously solved structure of the complex with histone peptide.  The inhibitor resembles the bound conformation of histone H3 Lys4 to Arg8, and is positioned in place by residues specific for G9a and GLP through specific interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhQepqN3zr_bVg90H21EOLACvtfcHk0lgHO5A0ibLIKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFKjt7g%253D&md5=6a16baaf33420fc1caf1df70d103d270</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1560%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DUpadhyay%26aufirst%3DA.%2BK.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520G9a-like%2520protein%2520lysine%2520methyltransferase%2520inhibition%2520by%2520BIX-01294%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D312%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">A chemiluminescence-based method for identification of histone lysine methyltransferase inhibitors</span> <span class="citation_source-journal">Molecular BioSystems</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">782</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=782-788&author=A.+M.+Quinnauthor=A.+Allali-Hassaniauthor=M.+Vedadiauthor=A.+Simeonov&title=A+chemiluminescence-based+method+for+identification+of+histone+lysine+methyltransferase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DA%2520chemiluminescence-based%2520method%2520for%2520identification%2520of%2520histone%2520lysine%2520methyltransferase%2520inhibitors%26jtitle%3DMolecular%2520BioSystems%26date%3D2010%26volume%3D6%26spage%3D782%26epage%3D788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lgHO5A0ibLIKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">A coupled fluorescent assay for histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+coupled+fluorescent+assay+for+histone+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0liJmqRsSyeV4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520coupled%2520fluorescent%2520assay%2520for%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="note"><p class="first last">Prepared as a racemic mixture.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nady, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">L3MBTL1 recognition of mono- and dimethylated histones</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1229</span><span class="NLM_x">–</span> <span class="NLM_lpage">1230</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnsmb1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=18026117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1229-1230&author=J.+Minauthor=A.+Allali-Hassaniauthor=N.+Nadyauthor=C.+Qiauthor=H.+Ouyangauthor=Y.+Liuauthor=F.+MacKenzieauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=L3MBTL1+recognition+of+mono-+and+dimethylated+histones"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">L3MBTL1 recognition of mono- and dimethylated histones</span></div><div class="casAuthors">Min, Jinrong; Allali-Hassani, Abdellah; Nady, Nataliya; Qi, Chao; Ouyang, Hui; Liu, Yongsong; MacKenzie, Farrell; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1229-1230</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Crystal structures of the L3MBTL1 MBT repeats in complex with histone H4 peptides dimethylated on Lys20 (H4K20me2) show that only the second of the three MBT repeats can bind mono- and dimethylated histone peptides.  Its binding pocket has similarities to that of 53BP1 and is able to recognize the degree of histone lysine methylation.  An unexpected mode of peptide-mediated dimerization suggests a possible mechanism for chromatin compaction by L3MBTL1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyCuRY6dV9-bVg90H21EOLACvtfcHk0liJmqRsSyeV4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP&md5=1f1072bf117cb7459a644907ca66b74b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1340%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DNady%26aufirst%3DN.%26aulast%3DQi%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMacKenzie%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DL3MBTL1%2520recognition%2520of%2520mono-%2520and%2520dimethylated%2520histones%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1229%26epage%3D1230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Wigle, T. J.; Provencher, L. M.; Norris, J. L.; Jin, J.; Brown, P. J.; Frye, S. V.; Janzen, W. P.</span><span> </span><span class="NLM_article-title">Accessing Protein Methyltransferase and Demethylase Enzymology Using Microfluidic Capillary Electrophoresis</span>.  <span class="citation_source-journal">Chemistry & Biology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span>in press.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=T.+J.+Wigle&author=L.+M.+Provencher&author=J.+L.+Norris&author=J.+Jin&author=P.+J.+Brown&author=S.+V.+Frye&author=W.+P.+Janzen&title=Accessing+Protein+Methyltransferase+and+Demethylase+Enzymology+Using+Microfluidic+Capillary+Electrophoresis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26atitle%3DAccessing%2520Protein%2520Methyltransferase%2520and%2520Demethylase%2520Enzymology%2520Using%2520Microfluidic%2520Capillary%2520Electrophoresis%26jtitle%3DChemistry%2520%2526%2520Biology%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">185</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">286</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0liddPqTF3uq7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Williams, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">The kinetics of reversible tight-binding inhibition</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">437</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+kinetics+of+reversible+tight-binding+inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0liddPqTF3uq7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520kinetics%2520of%2520reversible%2520tight-binding%2520inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Sakurai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">A miniaturized screen for inhibitors of Jumonji histone demethylases</span> <span class="citation_source-journal">Mol. Biosyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1039%2Fb912993f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=20094655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosF2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=357-364&author=M.+Sakuraiauthor=N.+R.+Roseauthor=L.+Schultzauthor=A.+M.+Quinnauthor=A.+Jadhavauthor=S.+S.+Ngauthor=U.+Oppermannauthor=C.+J.+Schofieldauthor=A.+Simeonov&title=A+miniaturized+screen+for+inhibitors+of+Jumonji+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A miniaturized screen for inhibitors of Jumonji histone demethylases</span></div><div class="casAuthors">Sakurai, Masaaki; Rose, Nathan R.; Schultz, Lena; Quinn, Amy M.; Jadhav, Ajit; Ng, Stanley S.; Oppermann, Udo; Schofield, Christopher J.; Simeonov, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">357-364</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">2-Oxoglutarate- and Fe(ii)-dependent oxygenases are a major class of N.vepsiln.-Me lysine demethylases that are involved in epigenetic regulation.  Assays suitable for implementation in a high-throughput manner have been lacking for these enzymes.  Here, we describe the design and implementation of a robust and miniaturized high-throughput kinetic assay for inhibitors of JMJD2E using a formaldehyde dehydrogenase-coupled reaction with real-time fluorescence detection.  Reactant compatibility studies resulted in simplification of the assay scheme to the mixing of two reagent solns., both of which were stable overnight.  The assay was miniaturized to a 4 μL vol. in 1536-well format and was used to screen the library of pharmacol. active compds. (LOPAC1280).  Inhibitors identified by the screen were further characterized in secondary assays including FDH counterscreen and demethylation assays that monitored demethylation by MALDI-TOF MS.  The assay developed here will enable the screening of large compd. libraries against the Jumonji demethylases in a robust and automated fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMwSVBOYmD4rVg90H21EOLACvtfcHk0lix79baRcpJ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosF2ktA%253D%253D&md5=b71ca06c07ec8e438bf87d69f390a993</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2Fb912993f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb912993f%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DA%2520miniaturized%2520screen%2520for%2520inhibitors%2520of%2520Jumonji%2520histone%2520demethylases%26jtitle%3DMol.%2520Biosyst.%26date%3D2010%26volume%3D6%26spage%3D357%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span> <span class="citation_source-journal">Nature Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2212</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lix79baRcpJ0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNature%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finerty, P. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ball, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hui, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigelt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span> </span><span class="NLM_article-title">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">15835</span><span class="NLM_x">–</span> <span class="NLM_lpage">15840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm100478y&amp;key=10.1073%2Fpnas.0605224103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm100478y&amp;key=17035505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm100478y&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=15835-15840&author=M.+Vedadiauthor=F.+H.+Niesenauthor=A.+Allali-Hassaniauthor=O.+Y.+Fedorovauthor=P.+J.+Finertyauthor=G.+A.+Wasneyauthor=R.+Yeungauthor=C.+Arrowsmithauthor=L.+J.+Ballauthor=H.+Berglundauthor=R.+Huiauthor=B.+D.+Marsdenauthor=P.+Nordlundauthor=M.+Sundstromauthor=J.+Weigeltauthor=A.+M.+Edwards&title=Chemical+screening+methods+to+identify+ligands+that+promote+protein+stability%2C+protein+crystallization%2C+and+structure+determination"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span></div><div class="casAuthors">Vedadi, Masoud; Niesen, Frank H.; Allali-Hassani, Abdellah; Fedorov, Oleg Y.; Finerty, Patrick J., Jr.; Wasney, Gregory A.; Yeung, Ron; Arrowsmith, Cheryl; Ball, Linda J.; Berglund, Helena; Hui, Raymond; Marsden, Brian D.; Nordlund, Paer; Sundstrom, Michael; Weigelt, Johan; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15835-15840</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 3D structures of human therapeutic targets are enabling for drug discovery.  However, their purifn. and crystn. remain rate detg.  In individual cases, ligands have been used to increase the success rate of protein purifn. and crystn., but the broad applicability of this approach is unknown.  The authors implemented two screening platforms, based on either fluorometry or static light scattering, to measure the increase in protein thermal stability upon binding of a ligand without the need to monitor enzyme activity.  In total, 221 different proteins from humans and human parasites were screened against one or both of two sorts of small-mol. libraries.  The first library comprised different salts, pH conditions, and commonly found small mols. and was applicable to all proteins.  The second comprised compds. specific for protein families of particular interest (e.g., protein kinases).  In 20 cases, including nine unique human protein kinases, a small mol. was identified that stabilized the proteins and promoted structure detn.  The methods are cost-effective, can be implemented in any lab., promise to increase the success rates of purifying and crystg. human proteins significantly, and identify new ligands for these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4duhcFpFn-bVg90H21EOLACvtfcHk0liEWQrK7a3Qrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN&md5=6515225c1613291deb7bb774d900f605</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0605224103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0605224103%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%2BY.%26aulast%3DFinerty%26aufirst%3DP.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DYeung%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBall%26aufirst%3DL.%2BJ.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DWeigelt%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DChemical%2520screening%2520methods%2520to%2520identify%2520ligands%2520that%2520promote%2520protein%2520stability%252C%2520protein%2520crystallization%252C%2520and%2520structure%2520determination%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2006%26volume%3D103%26spage%3D15835%26epage%3D15840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moult, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A survey of proteins encoded by nonsynonymous single nucleotide polymorphisms reveals a significant fraction with altered stability and activity</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">424</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2009&pages=15-26&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=I.+Chauauthor=B.+S.+Hongauthor=G.+Senisterraauthor=P.+Loppnauauthor=Z.+Shiauthor=J.+Moultauthor=A.+M.+Edwardsauthor=C.+H.+Arrowsmithauthor=H.+W.+Parkauthor=M.+Schapiraauthor=M.+Vedadi&title=A+survey+of+proteins+encoded+by+nonsynonymous+single+nucleotide+polymorphisms+reveals+a+significant+fraction+with+altered+stability+and+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DHong%26aufirst%3DB.%2BS.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DMoult%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520survey%2520of%2520proteins%2520encoded%2520by%2520nonsynonymous%2520single%2520nucleotide%2520polymorphisms%2520reveals%2520a%2520significant%2520fraction%2520with%2520altered%2520stability%2520and%2520activity%26jtitle%3DBiochem.%2520J.%26date%3D2009%26volume%3D424%26spage%3D15%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K5K','PDB','3K5K'); return false;">PDB: 3K5K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FPD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FPD','PDB','3FPD'); return false;">PDB: 3FPD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a href="/doi/suppl/10.1021/jm100478y">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81848"></div></div></div></div></div><hr /></hr><p class="last"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>10</b> and <b>29</b>. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm100478y/suppl_file/jm100478y_si_001.pdf">jm100478y_si_001.pdf (507.57 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm100478y&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm100478y%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2010.53.issue-15%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm100478y" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a39dcbf02495","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
