
<html lang="en"     class="pb-page"  data-request-id="77850450-b492-4271-9dd8-29d8a71a2e0e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00768;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Pyrido[2,3-b]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV" /></meta><meta name="dc.Creator" content="Yimin  Hu" /></meta><meta name="dc.Creator" content="Houguang  Shi" /></meta><meta name="dc.Creator" content="Mingwei  Zhou" /></meta><meta name="dc.Creator" content="Qingcheng  Ren" /></meta><meta name="dc.Creator" content="Wei  Zhu" /></meta><meta name="dc.Creator" content="Weixing  Zhang" /></meta><meta name="dc.Creator" content="Zhiwei  Zhang" /></meta><meta name="dc.Creator" content="Chengang  Zhou" /></meta><meta name="dc.Creator" content="Yongqiang  Liu" /></meta><meta name="dc.Creator" content="Xiao  Ding" /></meta><meta name="dc.Creator" content="Hong C.  Shen" /></meta><meta name="dc.Creator" content="S. Frank  Yan" /></meta><meta name="dc.Creator" content="Fabian  Dey" /></meta><meta name="dc.Creator" content="Waikwong  Wu" /></meta><meta name="dc.Creator" content="Guanglei  Zhai" /></meta><meta name="dc.Creator" content="Zheng  Zhou" /></meta><meta name="dc.Creator" content="Zhiheng  Xu" /></meta><meta name="dc.Creator" content="Ying  Ji" /></meta><meta name="dc.Creator" content="Hua  Lv" /></meta><meta name="dc.Creator" content="Tianyi  Jiang" /></meta><meta name="dc.Creator" content="Wen  Wang" /></meta><meta name="dc.Creator" content="Yunhua  Xu" /></meta><meta name="dc.Creator" content="Maarten  Vercruysse" /></meta><meta name="dc.Creator" content="Xiangyu  Yao" /></meta><meta name="dc.Creator" content="Yi  Mao" /></meta><meta name="dc.Creator" content="Xiaomin  Yu" /></meta><meta name="dc.Creator" content="Kenneth  Bradley" /></meta><meta name="dc.Creator" content="Xuefei  Tan" /></meta><meta name="dc.Description" content="The rise of multidrug resistant (MDR) Gram-negative (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine...." /></meta><meta name="Description" content="The rise of multidrug resistant (MDR) Gram-negative (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine...." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 4, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00768" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00768" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00768" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00768" /></link>
        
    
    

<title>Discovery of Pyrido[2,3-b]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00768" /></meta><meta property="og:title" content="Discovery of Pyrido[2,3-b]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0020.jpeg" /></meta><meta property="og:description" content="The rise of multidrug resistant (MDR) Gram-negative (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine. Developing novel antibiotics against MDR GN pathogens is particularly difficult as compounds have to permeate the GN double membrane, which has very different physicochemical properties, and have to circumvent a plethora of resistance mechanisms such as multiple efflux pumps and target modifications. The bacterial type II topoisomerases DNA gyrase (GyrA2B2) and Topoisomerase IV (ParC2E2) are highly conserved targets across all bacterial species and validated in the clinic by the fluoroquinolones. Dual inhibitors targeting the ATPase domains (GyrB/ParE) of type II topoisomerases can overcome target-based fluoroquinolone resistance. However, few ATPase inhibitors are active against GN pathogens. In this study, we demonstrated a successful strategy to convert a 2-carboxamide substituted azaindole chemical scaffold with only Gram-positive (GP) activity into a novel series with also potent activity against a range of MDR GN pathogens. By systematically fine-tuning the many physicochemical properties, we identified lead compounds such as 17r with a balanced profile showing potent GN activity, high aqueous solubility, and desirable PK features. Moreover, we showed the bactericidal efficacy of 17r using a neutropenic mouse thigh infection model." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00768"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00768">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00768&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00768&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00768&amp;href=/doi/10.1021/acs.jmedchem.0c00768" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9623-9649</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00691" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00781" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Pyrido[2,3-<i>b</i>]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yimin Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yimin Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yimin++Hu">Yimin Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Houguang Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Houguang Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Houguang++Shi">Houguang Shi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mingwei Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingwei Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingwei++Zhou">Mingwei Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qingcheng Ren</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qingcheng Ren</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec (Wuhan) Co., Ltd., No. 666 Gaoxin Road, Wuhan East Lake High-tech Development Zone, Hubei 430075, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qingcheng++Ren">Qingcheng Ren</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Zhu">Wei Zhu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4701-2474" title="Orcid link">http://orcid.org/0000-0003-4701-2474</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weixing Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weixing Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weixing++Zhang">Weixing Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhiwei Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhiwei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhiwei++Zhang">Zhiwei Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chengang Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chengang Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chengang++Zhou">Chengang Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yongqiang Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yongqiang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yongqiang++Liu">Yongqiang Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiao Ding</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiao Ding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiao++Ding">Xiao Ding</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong C. Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong C. Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong+C.++Shen">Hong C. Shen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3477-8932" title="Orcid link">http://orcid.org/0000-0002-3477-8932</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">S. Frank Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">S. Frank Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CADD, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=S.+Frank++Yan">S. Frank Yan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9121-9694" title="Orcid link">http://orcid.org/0000-0002-9121-9694</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fabian Dey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabian Dey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of CADD, Roche Pharma Research and Early Development, Grenzacherstrasse 124, CH-4070 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabian++Dey">Fabian Dey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Waikwong Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Waikwong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Waikwong++Wu">Waikwong Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guanglei Zhai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guanglei Zhai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guanglei++Zhai">Guanglei Zhai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zheng Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zheng Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zheng++Zhou">Zheng Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhiheng Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhiheng Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhiheng++Xu">Zhiheng Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Ji</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Ji</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">pCMC, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Ji">Ying Ji</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hua Lv</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hua Lv</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Sciences, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hua++Lv">Hua Lv</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tianyi Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tianyi Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Sciences, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tianyi++Jiang">Tianyi Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wen Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wen Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">MicuRx Pharmaceuticals, Inc. (Shanghai), Floor 3, Building B, 1976 Middle Gaoke Road, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wen++Wang">Wen Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yunhua Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yunhua Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">MicuRx Pharmaceuticals, Inc. (Shanghai), Floor 3, Building B, 1976 Middle Gaoke Road, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yunhua++Xu">Yunhua Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maarten Vercruysse</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maarten Vercruysse</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Infectious Diseases, Roche Innovation Center Basel, Roche Pharma Research and Early Development, Grenzacherstrasse 124, CH-4070 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maarten++Vercruysse">Maarten Vercruysse</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiangyu Yao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiangyu Yao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Infectious Diseases, Roche Innovation Center Shanghai, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiangyu++Yao">Xiangyu Yao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi Mao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi Mao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Infectious Diseases, Roche Innovation Center Shanghai, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi++Mao">Yi Mao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaomin Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaomin Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Infectious Diseases, Roche Innovation Center Shanghai, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaomin++Yu">Xiaomin Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kenneth Bradley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenneth Bradley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Infectious Diseases, Roche Innovation Center Basel, Roche Pharma Research and Early Development, Grenzacherstrasse 124, CH-4070 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth++Bradley">Kenneth Bradley</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xuefei Tan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuefei Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +86 21 28946746. Email: <a href="/cdn-cgi/l/email-protection#fe868b9b989b97d08a9f90be8c919d969bd09d9193"><span class="__cf_email__" data-cfemail="463e332320232f68322728063429252e236825292b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuefei++Tan">Xuefei Tan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7629-2255" title="Orcid link">http://orcid.org/0000-0001-7629-2255</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00768&amp;href=/doi/10.1021%2Facs.jmedchem.0c00768" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9623–9649</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 4, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 May 2020</li><li><span class="item_label"><b>Published</b> online</span>4 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00768" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00768</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9623%26pageCount%3D27%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYimin%2BHu%252C%2BHouguang%2BShi%252C%2BMingwei%2BZhou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c00768%26title%3DDiscovery%2Bof%2BPyrido%255B2%252C3-b%255Dindole%2BDerivatives%2Bwith%2BGram-Negative%2BActivity%2BTargeting%2BBoth%2BDNA%2BGyrase%2Band%2BTopoisomerase%2BIV%26numPages%3D27%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9649%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00768"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1173</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00768" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Pyrido[2,3-b]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yimin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Houguang&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Mingwei&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Qingcheng&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Weixing&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Zhiwei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Chengang&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Yongqiang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xiao&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;C. Shen&quot;},{&quot;first_name&quot;:&quot;S.&quot;,&quot;last_name&quot;:&quot;Frank Yan&quot;},{&quot;first_name&quot;:&quot;Fabian&quot;,&quot;last_name&quot;:&quot;Dey&quot;},{&quot;first_name&quot;:&quot;Waikwong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Guanglei&quot;,&quot;last_name&quot;:&quot;Zhai&quot;},{&quot;first_name&quot;:&quot;Zheng&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Zhiheng&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Ji&quot;},{&quot;first_name&quot;:&quot;Hua&quot;,&quot;last_name&quot;:&quot;Lv&quot;},{&quot;first_name&quot;:&quot;Tianyi&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Wen&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yunhua&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Maarten&quot;,&quot;last_name&quot;:&quot;Vercruysse&quot;},{&quot;first_name&quot;:&quot;Xiangyu&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Yi&quot;,&quot;last_name&quot;:&quot;Mao&quot;},{&quot;first_name&quot;:&quot;Xiaomin&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;Bradley&quot;},{&quot;first_name&quot;:&quot;Xuefei&quot;,&quot;last_name&quot;:&quot;Tan&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9623-9649&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00768&quot;},&quot;abstract&quot;:&quot;The rise of multidrug resistant (MDR) Gram-negative (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine. Developing novel antibiotics against MDR GN pathogens is particularly difficult as compounds have to permeate the GN double membrane, which has very different physicochemical properties, and have to circumvent a plethora of resistance mechanisms such as multiple efflux pumps and target modifications. The bacterial type II topoisomerases DNA gyrase (GyrA2B2) and Topoisomerase IV (ParC2E2) are highly conserved targets across all bacterial species and validated in the clinic by the fluoroquinolones. Dual inhibitors targeting the ATPase domains (GyrB/ParE) of type II topoisomerases can overcome target-based fluoroquinolone resistance. However, few ATPase inhibitors are active against GN pathogens. In this study, we demonstrated a successful strategy to convert a 2-carboxamide substituted azaindole chemical scaffold wit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00768&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00768" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00768&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00768" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00768&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00768" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00768&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00768&amp;href=/doi/10.1021/acs.jmedchem.0c00768" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00768" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00768" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00768%26sid%3Dliteratum%253Aachs%26pmid%3D32787097%26genre%3Darticle%26aulast%3DHu%26date%3D2020%26atitle%3DDiscovery%2Bof%2BPyrido%255B2%252C3-b%255Dindole%2BDerivatives%2Bwith%2BGram-Negative%2BActivity%2BTargeting%2BBoth%2BDNA%2BGyrase%2Band%2BTopoisomerase%2BIV%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9623%26epage%3D9649%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291214" title="Bacteria">Bacteria</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The rise of multidrug resistant (MDR) Gram-negative (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine. Developing novel antibiotics against MDR GN pathogens is particularly difficult as compounds have to permeate the GN double membrane, which has very different physicochemical properties, and have to circumvent a plethora of resistance mechanisms such as multiple efflux pumps and target modifications. The bacterial type II topoisomerases DNA gyrase (GyrA<sub>2</sub>B<sub>2</sub>) and Topoisomerase IV (ParC<sub>2</sub>E<sub>2</sub>) are highly conserved targets across all bacterial species and validated in the clinic by the fluoroquinolones. Dual inhibitors targeting the ATPase domains (GyrB/ParE) of type II topoisomerases can overcome target-based fluoroquinolone resistance. However, few ATPase inhibitors are active against GN pathogens. In this study, we demonstrated a successful strategy to convert a 2-carboxamide substituted azaindole chemical scaffold with only Gram-positive (GP) activity into a novel series with also potent activity against a range of MDR GN pathogens. By systematically fine-tuning the many physicochemical properties, we identified lead compounds such as <b>17r</b> with a balanced profile showing potent GN activity, high aqueous solubility, and desirable PK features. Moreover, we showed the bactericidal efficacy of <b>17r</b> using a neutropenic mouse thigh infection model.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07375" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07375" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The rapid increase in bacterial resistance against all antibiotics in the clinic is an urgent global health problem. The most severe bacterial infections are increasingly escaping first-line and even last-resort antibiotics treatments. These life-threatening multidrug resistant (MDR) infections are mainly caused by the so-called ESKAPE pathogens, i.e., <i>Enterococcus faecium</i>, <i>Staphylococcus aureus</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa</i>, and <i>Enterobacter</i> species.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The last four fall into the category of Gram-negative (GN) bacteria, which are particularly challenging to treat, as compounds with intracellular targets have to pass through both the negatively charged outer membrane and the hydrophobic inner membrane.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> The distinct physicochemical properties required of a compound to cross both barriers make targeting GN bacteria more challenging than Gram-positive (GP) bacteria, which lack an outer membrane. Also, many adaptive resistance mechanisms developed by the bacteria such as efflux pumps, inactivation of enzymes, mutation, or biofilm further complicate such a challenge. As a result, no new class of antibiotic with GN activity has reached the market in the last 40 years.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a></div><div class="NLM_p">The bacterial type IIA topoisomerases, i.e., DNA gyrase and topoisomerase IV, are well-known antibacterial targets. These essential enzymes solve critical DNA-topology changes during replication, transcription, and recombination. Specifically, DNA gyrase introduces negative supercoils into DNA and relaxes positive supercoils that accumulate during replication and transcription. Topoisomerase IV also removes positive supercoils as well as decatenates the replicated sister chromosomes. DNA gyrase is a heterotetramer composed of two A (GyrA) and two B subunits (GyrB). The two GyrB subunits hydrolyze two ATP molecules, which drive the catalytic function, i.e., binding, cleavage, and relegation of the DNA segments. Similarly, Topoisomerase IV consists of two C subunits (ParC) and two E subunits (ParE). The ParE subunits hydrolyze ATP driving the enzymatic activity.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Dual targeting of both GyrB and ParE subunits is possible as both ATP binding sites are very similar and results in very low spontaneous resistance frequencies. Also, because both enzymes are highly conserved across GP and GN bacteria, a dual GyrB/ParE ATPase inhibitor can have broad-spectrum activity.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Fluoroquinolones (FQ) are one of the most commonly prescribed broad-spectrum antibiotics that target both DNA gyrase and Topoisomerase IV. However, due to its overuse and misuse for decades FQ resistance in hospitals has increased significantly.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The resistance mechanisms include both target-based mutations and nontarget based such as plasmid-mediated resistance and efflux, which are difficult to overcome even by the newer generation FQ in the clinic.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> FQ antibiotics target the catalytic sites in the GyrA/ParC subunits and stabilize the cleaved DNA–enzyme complex, which results in lethal double-stranded DNA breaks. Targeting alternative binding sites such as GyrB/ParE ATPase domains, which is distinct from the FQ mode-of-action (MoA), enables new antibacterial agents to overcome target-based FQ resistance and to achieve broad-spectrum activity. Novobiocin previously established the clinical proof of concept of this new MoA in the 1950s. It was, however, the first and only marketed drug of this new class. The use of Novobiocin declined rapidly due to the later development of more effective antibiotics. It was eventually withdrawn from the market due to the rise of resistance development and potential safety concerns.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">Over the last 50 years, industry and academia continued to work on finding novel scaffolds targeting the GyrB/ParE ATP binding site. Along with the continuously improved technical understanding of the molecular details of the MoA and factors driving resistance development against antibiotics, numerous chemical scaffolds were identified showing gradual improvements in physicochemical properties, safety, and antibacterial coverage compared to Novabiocin.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> However, the spectrum of these GyrB/ParE dual inhibitors has been mainly limited to a GP spectrum, with only a few common respiratory GN pathogens such as <i>Haemophilus influenzae</i>.</div><div class="NLM_p">A breakthrough scaffold targeting the GyrB/ParE ATPase site having GN activity was discovered eventually by Trius Therapeutics. Their novel tricyclic pyrimido[4,5-<i>b</i>]indole series, derived from a pyrrolopyrimidine scaffold, showed for the first time antibacterial activity effectively covering also GN pathogens such as <i>P. aeruginosa</i>, <i>A. baumannii</i>, and <i>K. pneumoniae</i> (represented by <b>GP6</b> in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> The enhanced GN activity of this series is attributed to its rigidity, orientation of a critical amino group, and ability to adopt both neutral and charged states, all of which helps to penetrate the GN double membranes.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> This series showed dual activity against both GyrB and ParE ATP binding site, resulting in very low resistance frequencies. Additionally, Soo Yei Ho et al. recently showed that a GyrB/ParE scaffold with only GP activity could be modified to yield broad-spectrum activity by carefully modulating the physicochemical property of zwitterionic compounds (represented by <b>ETC44</b> in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. A few GyrB/ParE dual-inhibitor scaffolds. (a) AZ GN-positive only azaindole ureas, <b>AZ10</b> and <b>AZ17</b>; (b) Trius’ GN-active tricyclic pyrimido[4,5-<i>b</i>]indole, <b>GP6</b>; and (c) ETC’s GN-active zwitterionic pyridine urea, <b>ETC44</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In this study, we report our efforts in discovering pyrido[2,3-<i>b</i>]indole derivatives as a novel class of dual GyrB/ParE inhibitors with promising GN bacteria activities and <i>in vivo</i> bactericidal efficacy in a neutropenic mouse thigh infection model.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87025" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87025" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It has become increasingly apparent that in-house compound libraries of pharmaceutical companies are not well suited for the identification of antibacterial starting points due to their different physicochemical properties compared to most marketed antibiotics.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> Therefore, we decided not to rely on a high-throughput screening campaign but to utilize the rich published collection of various potent DNA Gyrase/Topoisomerase IV GP antibiotics as medicinal chemistry starting points for the development of antibiotics against GN bacteria. A literature survey identified an azaindole scaffold (represented by <b>AZ10</b> and <b>AZ17</b> in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) previously discovered by AstraZeneca (AZ).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> This class of compounds possesses a balanced dual enzymatic potency against both GyrB and ParE enzymes in GP bacteria. This feature could also be retained when improving the antibacterial activity against GN bacterial pathogens. Meanwhile, the overlay of the ATP binding sites of ParE crystallized with azaindole <b>AZ10</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EMV">4EMV</a>) active against GP bacteria (<i>S. pneumoniae</i>) and GyrB crystallized with tricyclic pyrimido[4,5-<i>b</i>]indole <b>GP1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>) active against GN bacteria (<i>E. coli</i>) showed a good structural alignment of the key ligand pharmacophores (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Notably, first, in both cores, the bidentate hydrogen bond interaction in donor–donor–acceptor pattern with strongly conserved aspartic acid (Asp78 in <i>S. pneumoniae</i> ParE, Asp73 in <i>E. coli</i> GyrB) and a tightly bound water molecule were present. Comparing the scaffolds in the alignment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) reveals a small shift. This shift is likely due to steric reasons of the substitution at the 2-position in <b>GP1</b>, pushing the compound further away from the Glu-Arg salt bridge. Nonetheless, the <b>AZ10</b> azaindole 5-position pyridine ring overlaps well with the pyrimidine ring of <b>GP1</b>, both forming cation−π stacking interactions with the neighboring arginine side chain (Arg 76 in <i>E. coli</i> GyrB and Arg 81 in <i>S. pneumoniae</i> ParE, respectively). While the meta carboxylate substituted R5′-pyridine of <b>AZ10</b> forms a direct bond, <b>GP1</b> engages in a water-mediated H-bond to Arg 136. A more pronounced difference can be observed in the more hydrophobic part of the pocket where the 2-carboxamide of the azaindole occupies the latter only partially compared to the <b>GP1</b> tricyclic core. Lastly, the substitution along the R4-vector, the position held by an amine-containing unsaturated ring disclosed by Trius, is filled by a substituted pyrazole. While the latter mostly engages with the floor of the pocket, the former also forms an H-bond with Asn 46.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overlay of X-ray crystal structures of ATP binding sites of GyrB with bound <b>GP1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>, protein shown with green ribbon and carbons, ligand with cyan carbons) and ParE with bound <b>AZ10</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EMV">4EMV</a>, protein shown with gray ribbon and carbons, ligand with magenta carbons). Hydrogen bonding interactions as red lines and residue labels shown for structure 4KFG. Further residues and water molecules omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We decided to first introduce further rigidity into the azaindole scaffold, a strategy that had been shown in several cases to be successful in obtaining anti-GN activity.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> After a few attempts, we were successful in replacing the latter by a fused ethyl aniline, which not only lowered the strain in the high energy bioactive conformation (H-bond donors facing each other) of the azaindole analog <b>(AZ10)</b><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> but also generated a rigid scaffold that could still maintain the same hydrogen-bonding network with the protein and the structural water (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). As such, a new type of tricyclic pyrido[2,3-<i>b</i>]indole analogs exemplified by <b>10a</b>–<b>d</b> was synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). A 6-fluorine substitution was incorporated in the tricyclic core based on modeling, to optimally fill the hydrophobic pocket and form additional van der Waals interactions.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of novel tricyclic pyrido[2,3-<i>b</i>]indole scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Enzymatic (Ki) and Antibacterial Potency (MIC) and Physicochemical Property of Pyrido[2,3-<i>b</i>]indole Derivatives, <b>10a</b>–<b>u</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0010.gif" alt="" id="GRAPHIC-d7e760-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0011.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>i</sub>: inhibition constant.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">MIC: minimum inhibitory concentration.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">p<i>K</i><sub>a</sub>: acid dissociation constant, measured in a photometric titration assay.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">mlogD<sub>7.4</sub>: machine learning prediction of intrinsic distribution coefficient between octanol and aqueous buffer (pH 7.4) based on in-house training set data, which measured in a CAMDIS assay.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Predicted from MoKa (version 2).</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Ionization constant of the conjugate acid.</p></div></div><div></div></div><div class="NLM_p">To our delight, these analogs showed superior enzymatic potency in both GyrB and ParE biochemical assays (<b>10b</b>–<b>d</b>, GyrB <i>K</i><sub><i>i</i></sub> < 1 nM, ParE <i>K</i><sub><i>i</i></sub> < 5 nM). The carboxylic acid analog <b>10a</b> is the least active compound with GyrB <i>K</i><sub><i>i</i></sub> = 45 nM and ParE <i>K</i><sub><i>i</i></sub> = 118 nM, respectively. Furthermore, despite the high lipophilicity (predicted LogD > 3), these analogs were not only potent against GP bacteria (represented by <i>S. aureus</i> ATCC 29213, MIC < 0.25 μg/mL) but also considerably active against GN bacteria such as <i>E. coli</i> ATCC 25922 (<b>10b</b>–<b>d</b>, MICs ∼ 0.5 μg/mL). However, we noticed that a hybrid molecule composed of an R4 nonaryl ring substitution, such as the 3-amino proline analog, was completely inactive (data not shown). Both pyridine 5<sup>’</sup>-carboxamide <b>10b</b> and 5′-nitrile analog <b>10d</b> also demonstrated a good potential for broad-spectrum antibacterial coverage as indicated in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and, to our delight, displayed activity against GN MDR strains. In a small panel of selected GN bacteria intended for an initial assessment of GN bacteria coverage, compounds <b>10b</b> and <b>10d</b> showed MICs < 6 μg/mL, with consistently better potencies against <i>E. coli</i> and <i>A. baumannii</i> pathogens but less active against <i>K. pneumoniae</i> and <i>P. aeruginosa</i> species. Except for <i>A. baumannii</i> ATCC 51432 (<b>10d</b> only), there was no significant potency shift between the GN bacteria wild-type and MDR strains indicating structural features and physicochemical properties favorable for GN double membrane permeation and devoid of strong efflux. Encouraged by these findings, we then explored the structure–activity relationship (SAR) of substitution at the 3-position of the pyrido[2,3-<i>b</i>]indole scaffold (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antibacterial GN Broad-Spectrum Coverage of Selected Compounds <b>10, 15</b>, and <b>17</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">MIC<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><i>E. coli</i> strains (μg/mL)</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">MIC <i>K. pneumoniae</i> strains (μg/mL)</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">MIC <i>A. baumannii</i> strains (μg/mL)</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">MIC <i>P. aeruginosa</i> strains (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cmpd</th><th class="colsep0 rowsep0" align="center" char=".">ATCC 25922<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC 35218<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC BAA-2340<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC 10031<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC 700603<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC BAA-2146<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC 19606<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC 51432<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC BAA-747<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">PAO-1<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">ATCC 27853<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">ATCC BAA-2113<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10b</b></td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">0.76</td><td class="colsep0 rowsep0" align="char" char=".">0.76</td><td class="colsep0 rowsep0" align="char" char=".">0.09</td><td class="colsep0 rowsep0" align="char" char=".">0.76</td><td class="colsep0 rowsep0" align="char" char=".">3.02</td><td class="colsep0 rowsep0" align="char" char=".">0.76</td><td class="colsep0 rowsep0" align="char" char=".">3.02</td><td class="colsep0 rowsep0" align="char" char=".">3.02</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">0.76</td><td class="colsep0 rowsep0" align="char" char=".">0.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10d</b></td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1.45</td><td class="colsep0 rowsep0" align="char" char=".">2.91</td><td class="colsep0 rowsep0" align="char" char=".">0.73</td><td class="colsep0 rowsep0" align="char" char=".">1.45</td><td class="colsep0 rowsep0" align="char" char=".">5.82</td><td class="colsep0 rowsep0" align="char" char=".">0.73</td><td class="colsep0 rowsep0" align="char" char=".">>23.3</td><td class="colsep0 rowsep0" align="char" char=".">1.45</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">5.82</td><td class="colsep0 rowsep0" align="char" char=".">1.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10h</b></td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">1.38</td><td class="colsep0 rowsep0" align="char" char=".">0.09</td><td class="colsep0 rowsep0" align="char" char=".">0.69</td><td class="colsep0 rowsep0" align="char" char=".">1.38</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td><td class="colsep0 rowsep0" align="char" char=".">1.38</td><td class="colsep0 rowsep0" align="char" char=".">0.69</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">1.38</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10i</b></td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">2.85</td><td class="colsep0 rowsep0" align="char" char=".">2.85</td><td class="colsep0 rowsep0" align="char" char=".">0.178</td><td class="colsep0 rowsep0" align="char" char=".">1.42</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="char" char=".">1.42</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="char" char=".">2.85</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">2.85</td><td class="colsep0 rowsep0" align="char" char=".">1.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10m</b></td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td><td class="colsep0 rowsep0" align="char" char=".">0.183</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td><td class="colsep0 rowsep0" align="char" char=".">>23.5</td><td class="colsep0 rowsep0" align="char" char=".">073</td><td class="colsep0 rowsep0" align="char" char=".">>23.5</td><td class="colsep0 rowsep0" align="char" char=".">>23.5</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">>23.5</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10t</b></td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15r</b></td><td class="colsep0 rowsep0" align="char" char="."><0.25</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="."><0.03</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17r</b></td><td class="colsep0 rowsep0" align="char" char=".">0.31</td><td class="colsep0 rowsep0" align="char" char=".">3.13</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="char" char="."><0.02</td><td class="colsep0 rowsep0" align="char" char=".">6.26</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td><td class="colsep0 rowsep0" align="char" char=".">0.39</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="left">1.56</td><td class="colsep0 rowsep0" align="char" char=".">1.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17x</b></td><td class="colsep0 rowsep0" align="char" char=".">0.81</td><td class="colsep0 rowsep0" align="char" char=".">5.56</td><td class="colsep0 rowsep0" align="char" char=".">2.78</td><td class="colsep0 rowsep0" align="char" char=".">0.09</td><td class="colsep0 rowsep0" align="char" char=".">22.2</td><td class="colsep0 rowsep0" align="char" char=".">22.2</td><td class="colsep0 rowsep0" align="char" char=".">5.56</td><td class="colsep0 rowsep0" align="char" char=".">5.56</td><td class="colsep0 rowsep0" align="char" char=".">2.78</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="left">11.1</td><td class="colsep0 rowsep0" align="char" char=".">11.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">MIC: minimum inhibitory concentration.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Representative GN wide-type bacteria strains.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Representative GN MDR bacterial strain.</p></div></div></div><div class="NLM_p">Based on the analysis of our new analogs and the X-ray cocrystal structure of <b>AZ10</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EMV">4EMV</a>), it was reasonable to assume that the R3-pyridine ring formed cation−π stacking interactions with Arg 81 (Arg 76 in <i>E. coli</i> GyrB), and a suitable substitution on the 5′-position of the pyridine ring could potentially interact with the adjacent amino acid Arg 140 (Arg 136 in <i>E. coli</i> GyrB). Both amino acids had been reported before as being critical for good enzymatic activity. Thus, additional substituents capable of forming optimal hydrogen-bonding interaction with Arg 81 and/or Arg 140 were evaluated. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> shows binding affinities of these analogs against both GyrB and ParE enzymes as well as their antibacterial activities in both GP bacteria (<i>S. aureus</i> ATCC 29213) and GN bacteria (e.g., <i>E. coli</i> ATCC 25922). The majority of substitutions were well tolerated, as indicated by their potent <i>K</i><sub><i>i</i></sub> values of either GyrB or ParE. The least potent compound <b>10k</b> (GyrB <i>K</i><sub><i>i</i></sub> = 53 nM and ParE <i>K</i><sub><i>i</i></sub> = 239.5 nM) still showed good GP activity (MIC = 0.5 μg/mL) in <i>S. aureus</i> ATCC 29213 and weak GN activity (MIC = 8 μg/mL) in <i>E. coli</i> ATCC 25922. Similar to the carboxylic acid analog <b>10a</b>, strongly acidic and basic compounds such as <b>10g</b>, <b>10k</b>, <b>10n</b>, and <b>10p</b> were much less active in <i>E. coli</i> ATCC 25922 (MICs ≥ 8 μg/mL). Interestingly, the basic analog <b>10l</b> (p<i>K</i><sub>a</sub> = 8.62) was the exception with <i>E. coli</i> ATCC 25922 MIC = 1 μg/mL. A potential reason for the latter could be related to the strong compound accumulation propensity that free amino groups can induce.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compared with <b>10b</b> and <b>10d</b>, more recent analogs <b>10h</b>, <b>10i</b>, and <b>10t</b> demonstrated 2-fold improvement of potency against <i>E. coli</i> ATCC 25922 (MICs = 0.25 μg/mL). Additionally, the R3 group optimization led to not only new compounds with improved anti-GN bacterial activity in <i>E. coli</i> ATCC 25922 but also an improved GN bacteria coverage (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, pyrimidine <b>10h</b> and pyrazolo[1,5-<i>a</i>]pyrimidine <b>10t</b>, MICs ≤ 2 μg/mL).</div><div class="NLM_p">Among these newly identified R3 modifications, further modification on the pyrimidine ring of <b>10h</b> seemed attractive. The 2′-position of the pyrimidine was envisioned as a prominent site for derivatization because this vector is pointing toward the solvent-exposed area, therefore presumably, various groups could be accommodated without a loss of good inhibitory activity. Also, variation in this region was highly feasible due to many building blocks commercially available. <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> lists the SAR of a selection of 2′-position modified R3-pyrimidine analogs. As expected, the majority of compounds maintained strong binding affinities to both enzymes, with ParE binding activity more strongly affected by these modifications. A protein overlay between GyrB and ParE, however, did not provide any reason for the observed potency difference. Except for compound <b>12g</b>, all analogs had potent GP activity in <i>S. aureus</i> ATCC 29213 (MIC ≤ 0.25 μg/mL), likely due to their favorable inner membrane permeability. In terms of GN activity against <i>E. coli</i> ATCC 25922, for the neutral analogs, except <b>12k</b>, analogs <b>12a</b>–<b>d</b>, <b>12h</b>–<b>j</b>, and <b>12v</b> all showed comparable antibiotic potency to compound <b>10h</b>. Additionally, introduction of small polar groups, such as alcohols, amides, or aniline amino groups, likely improved the analogs’ ability to better penetrate outer membranes of <i>E. coli</i> ATCC 25922 as indicated by the smaller potency shift from GP to GN strains (1–4-fold compared to 16-fold potency shift for analog <b>10h</b> between <i>S. aureus</i> ATCC 29213 and <i>E. coli</i> ATCC 25922). The comparable antibacterial potency was also maintained for the two weakly basic compounds <b>12p</b> and <b>12q</b> in <i>E. coli</i> ATCC 25922. Concerning charged molecules, the basic secondary amino compounds (<b>12l</b>–<b>n</b> and <b>12r</b>–<b>u</b>) had weaker anti-GN activity, which could be due to poor penetration through the inner membrane, as indicated by the decreased <i>S. aureus</i> ATCC 29213 potency when compared to compound <b>10h</b>. The GN activity varied for the few acid analogs (<b>12e</b>–<b>g</b>, <b>12o</b>, and <b>12w</b>). As these acids had similar p<i>K</i><sub>a</sub> values and identical GP activity against <i>S. aureus</i> ATCC 29213, it is reasonable to assume the particular charged form of these compounds could have different GN outer membrane permeability profiles.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Pyrido[2,3-<i>b</i>]indole GyrB/ParE Dual Inhibitors, <b>10h</b> and <b>12a</b>–<b>x</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0012.gif" alt="" id="GRAPHIC-d7e1657-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0013.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>i</sub>: inhibition constant.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">MIC: minimum inhibitory concentration.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">LM: scaled intrinsic clearance of compound in mouse liver microsomes.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">LYSA: solubility determined by lyophilization solubility assay (LYSA) at pH 6.5.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Predicted from MoKa (version 2).</p></div><div class="footnote" id="t3fn6"><sup><sup>f</sup></sup><p class="last">Ionization constant of conjugate acid.</p></div><div class="footnote" id="t3fn7"><sup><sup>g</sup></sup><p class="last">p<i>K</i><sub>a</sub>: acid dissociation constant, measured in a photometric titration assay.</p></div><div class="footnote" id="t3fn8"><sup><sup>h</sup></sup><p class="last"><b>12x</b> is a methyl aniline analog of compound <b>12f</b> (also see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</p></div></div><div></div></div><div class="NLM_p">Compound <b>10h</b> suffered from high microsomal instability. Although the 2′-positon of pyrimidine is not the predicted site of metabolism from Metasite (version 6.0.1), to our surprise, the majority of new analogs listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> yielded improved microsomal stability. Among those, three analogs <b>12j</b>, <b>12p</b>, and <b>12v</b> achieved relatively high metabolic stability in mouse microsomal assay.</div><div class="NLM_p">Due to the high lipophilicity (predicted LogD 3.48), compound <b>10h</b> also had very low aqueous solubility (LYSA < 0.2 μg/mL). For the newly synthesized analogs (<b>12e, 12f, 12s, 12t</b>, and <b>12w</b>), all of which contained a charged group, marginal to good improvement of solubility profiles (LYSA 8–133 μg/mL) was observed. The acidic analog <b>12f</b> displayed better solubility with an LYSA of 67 μg/mL, and it also exhibited potent anti-GN activity with <i>E. coli</i> ATCC 25922 MIC = 0.375 μg/mL.</div><div class="NLM_p">The X-ray cocrystal structure of compound <b>12o</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S">6M1S</a>) in complex with <i>P. aeruginosa</i> GyrB protein is shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. As expected, there is a strong hydrogen-bonding network formed between the ligand and amino acid Asp 75 and the structural water molecule. The tricyclic pyrido[2,3-<i>b</i>]indole scaffold adopts a hydrogen-bond donor–donor–acceptor pattern that is the same as that seen for the Trius tricyclic scaffold. Compared to the original azaindole scaffold, there is a minor dihedral angle change between the tricyclic ring and the R3-pyridine. The cation−π stacking interaction between Arg 78 and the pyridine ring was maintained. Meanwhile, the 2′-oxypropanoic acid-containing substituent formed a salt bridge interaction with Arg 78. Albeit the inherent flexibility of the latter substituent, it is well-defined in the electron density, which is partially stabilized by the adjoining molecule of the next asymmetric unit (not shown). As mentioned before, both Arg 78 and Arg 138 are important for good enzymatic activity. Whereas no direct interaction with Arg 138 was observed, there is room for a water molecule to form a bridged H-bond interaction between the latter and one of the pyrimidine nitrogens similar to the <b>GP1</b> structure. In addition, an H-bond of the polarized C4’ pyrimidine hydrogen to the backbone carbonyl oxygen of Gly 79 is observed. Due to the steric demand of the phenyl ring of the pyrido[2,3-<i>b</i>]indole and the R3 pyrimidine substituent, the R4 3′-CF<sub>3</sub> pyrazole substitution adopted a marginally bigger out of plane twist (58 °C in <b>12o</b> vs 53.1 °C in <b>AZ10</b>). The 3′-CF<sub>3</sub> group sits on top of the hydrophobic groove formed by amino acids Pro 81 and Ile 80 and 96. The 4’ and 5′-positions of the pyrazole ring point toward a solvent-exposed area opposite which two conserved amino acids (Asn 48 and Asp 51, omitted for clarity, see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) are located. These residues can be utilized to improve binding affinity further. Overlay with the Trius X-ray cocrystal structure (<b>GP1</b>, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>) revealed a shift of the scaffold partially due to the ethyl vs methyl difference. The limited room available in the back of the pocket results in the former being pushed slightly toward the opening of the pocket, i.e., along the Asp 75 acceptor and water donor while maintaining these key interactions. However, once a methyl aniline analog such as <b>12x</b> was used, the two tricyclic cores overlaid more closely (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Nonetheless, a small shift of the cores is still apparent, which is—as discussed for the AZ analog before—due to the change from the R2- to the R3-position substitution, with the latter yielding shorter interaction distances for the H-bonds due to tighter packing of the core with the protein. Conversely, the Trius compound has a closer π–π stacking to Arg 78 (76 in <i>E. coli</i> GyrB) than the R3-pyrimidine ring, and trying to replace R4-pyrazole with nonaryl groups such as Trius R4 proprietary amines failed to give better antibacterial potency indicating different SAR in this region. Nonetheless, it was apparent that one could take advantage of two additional conserved amino acids Asn 48 and Asp 51 for R4 group optimization.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray structure of <i>P. aeruginosa</i> GyrB in complex with compound <b>12o</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S">6M1S</a>). Key protein residues shown as sticks with gray and ligand with magenta carbons. All water molecules except for structural water are hidden for clarity. Interactions are shown as color-coded dashed lines (H-bonds in red, vdW in yellow, cation−π stacking in blue, and salt bridge in light pink).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray cocrystal structures overlay between <b>GP1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>, protein shown with green ribbon and carbons, ligand with cyan carbons) and compound <b>12x</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1J">6M1J</a>, protein shown with gray ribbon and carbons, ligand with magenta carbons). Further residues and water molecules omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Therefore, having explored the R3 area and its impact on physicochemical properties, we then focused on the R4 group refinement. <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> lists several R4 variations keeping the R3 pyridine-3-carbonitrile group fixed. Due to a balanced physicochemical property, the pyrazole 3′-CF<sub>3</sub> substitution not only had an excellent fit in the hydrophobic pocket (<i>K</i><sub><i>i</i></sub> < 1 nM) but also showed a MIC of 0.5 μg/mL against <i>E. coli</i> ATCC 25922. A change to 3′-CF<sub>2</sub> (<b>15i</b>) resulted in a 2-fold activity decrease. Furthermore, an effort to explore alternative hydrophobic substitutions such as Me (<b>15e</b>), Et (<b>15f</b>), isopropyl (<b>15g</b>), and <i>tert</i>-butyl (<b>15h</b>) all decreased anti-GN activities (4–16-fold). The pyrazole 3′-CN substitution (<b>15a</b>), however, showed a surprisingly 2-fold improvement in anti-GN potency (<i>E. coli</i> ATCC 25922 MIC = 0.125 μg/mL) and an 8-fold GP to GN potency shift. The few polar substituents such as carboxylic acid (<b>15b</b>) and amine (<b>15d</b>) were completely inactive (MICs > 16 μg/mL), which might be due to both the incompatibility with the hydrophobic floor of the pocket and the less favored physicochemical properties at R4. A few heterocycle alternatives to the pyrazole were explored as well. Although an acceptable anti-GP potency was observed (<i>S. aureus</i> ATCC 29213 MIC = 0.063 μg/mL), the phenyl 3′-CF<sub>3</sub> (<b>15j</b>) showed much decreased anti-GN activity (MIC > 16 μg/m) in <i>E. coli</i> ATCC 25922, and the imidazole 4′-CF<sub>3</sub> (<b>15k</b>) was even inactive. For the two other replacements, pyrazole 4′-CF<sub>3</sub> (<b>15l</b>) and thiazole 3′-CF<sub>3</sub>(<b>15m</b>), their MICs were maintained or slightly decreased. It is evident that specific heterocyclic rings at the R4 position are ideal for GN activity, although the precise reason remained elusive.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pyrido[2,3-<i>b</i>]indole GyrB/ParE Dual Inhibitors, <b>10d</b> and <b>15a</b>–<b>m</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0014.gif" alt="" id="GRAPHIC-d7e1949-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0015.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>i</sub>: inhibition constant.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">MIC: minimum inhibitory concentration.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">LYSA: solubility determined by lyophilization solubility assay (LYSA) at pH 6.5.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">p<i>K</i><sub>a</sub>: ionization constant predicted from MoKa (version 2).</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Conjugate acid.</p></div></div><div></div></div><div class="NLM_p">With pyrazole as the preferred heterocycle, we then evaluated the possibility to substitute the latter further. As indicated in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, among the compounds prepared, the 3′,4′-bis-substituted free amino compound <b>15r</b> maintained excellent anti-GN activity (<i>E. coli</i> 25922 MIC = 0.25 μg/mL). However, an attempt to further rigidify this molecule by ring cyclization gave decreased antibacterial potency as shown by analog <b>15u</b> (<i>E. coli</i> ATCC 25922 MIC = 1 μg/mL). Meanwhile, it was also noticed that the impact on potency for the CF<sub>3</sub> substitution on the R4 bicyclic core was not as significant as on the 3′-monosubstituted pyrazole ring (e.g., <b>15u</b> vs <b>15x</b>). The aqueous solubility of two amino analogs <b>15r</b> and <b>15x</b> achieved LYSA = 90 μg/mL and 34 μg/mL, respectively. We attributed such good solubility improvement to the molecules’ effective disruption of the intermolecular π–π stacking interaction that is partially responsible for the low solubility of this series of analogs.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Pyrido[2,3-<i>b</i>]indole GyrB/ParE Dual Inhibitors, <b>10d</b> and <b>15n</b>–<b>z</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0016.gif" alt="" id="GRAPHIC-d7e2037-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0017.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">MIC: minimum inhibitory concentration.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">p<i>K</i><sub>a</sub>: ionization constant predicted from MoKa (version 2); measured in a photometric titration assay.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">LM: scaled intrinsic clearance of compound in mouse liver microsomes.</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">LYSA: solubility determined by lyophilization solubility assay (LYSA) at pH 6.5.</p></div><div class="footnote" id="t5fn5"><sup><sup>e</sup></sup><p class="last">Conjugate acid dissociation constant, measured in a photometric titration assay.</p></div></div><div></div></div><div class="NLM_p">The subsequent combination of several newly identified R4 groups and promising R3 modifications, such as the 2-methoxypyrimidine led to the syntheses of analogs <b>17r, 17u</b>, and <b>17x</b>. To our delight, not only did the mono amino compound <b>17r</b> maintain the same good potency (<i>E. coli</i> ATCC 25922 MIC = 0.312 μg/mL), but the bicyclic compound <b>17x</b> improved its activity (<i>E. coli</i> ATCC 25922 MIC = 0.812 μg/mL) and had a good solubility of LYSA = 350 μg/mL, the highest solubility among the whole series.</div><div class="NLM_p">The good anti-GN activity, acceptable aqueous solubility, and metabolic stability of compounds <b>15r</b>, <b>17r</b>, and <b>17x</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) then supported their further evaluation <i>in vivo</i>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Comparison between Pyrido[2,3-<i>b</i>]indole GyrB/ParE Dual Inhibitors, <b>15</b> and <b>17</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0018.gif" alt="" id="GRAPHIC-d7e2124-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0019.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">MIC: minimum inhibitory concentration.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">LYSA: solubility determined by lyophilization solubility assay (LYSA) at pH 6.5.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">fu: percentage of compound free fraction in a plasma protein binding assay.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">Cl: scaled mouse intrinsic clearance of compound.</p></div></div><div></div></div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a> shows mouse SDPK IV study results of compounds <b>15r</b>, <b>17r</b>, and <b>17x</b>. All three basic compounds showed medium <i>in vivo</i> clearances (Cl<sub>iv</sub>), which correlate well with the <i>in vitro</i> hepatic clearances. The half-lives are short, around 1–2 h, and the area under the curve (AUC) falls into the range of 320–340 μg·h/mL. These compounds demonstrated a medium to high volume of distribution with the plasma protein binding in the range of 89–99%. Although not displaying optimal PK profiles, these compounds nonetheless proceeded into an animal efficacy study for the initial PK/PD relationship exploration.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Single-Dose Intravenous Pharmacokinetics of Representative Compounds in the Mouse</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cmpd</th><th class="colsep0 rowsep0" align="center" char=".">Cmax_D<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a> (kg·ng/mL/mg)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>iv</sub>_D<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a> (h·kg·ng/mL/mg)</th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>iv</sub><a class="ref internalNav" href="#t7fn3" aria-label="c">c</a> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub><a class="ref internalNav" href="#t7fn4" aria-label="d">d</a> (h)</th><th class="colsep0 rowsep0" align="center" char=".">V<sub>ss</sub><a class="ref internalNav" href="#t7fn5" aria-label="e">e</a> (L/kg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15r</b></td><td class="colsep0 rowsep0" align="char" char=".">305</td><td class="colsep0 rowsep0" align="char" char=".">324</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17r</b></td><td class="colsep0 rowsep0" align="char" char=".">494</td><td class="colsep0 rowsep0" align="char" char=".">341</td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17x</b></td><td class="colsep0 rowsep0" align="char" char=".">792</td><td class="colsep0 rowsep0" align="char" char=".">327</td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last"><i>C</i><sub>max</sub>_D: dose-normalized maximal concentration.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">AUC<sub>iv</sub>_D: dose-normalized area under the curve.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">Cl<sub>iv</sub>: plasma clearance.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last"><i>t</i><sub>1/2</sub>: half-live of compound in plasma.</p></div><div class="footnote" id="t7fn5"><sup><sup>e</sup></sup><p class="last">V<sub>ss</sub>: volume of distribution at the steady-state.</p></div></div></div><div class="NLM_p">A neutropenic mouse thigh infection model with <i>E. coli</i> ATCC 25922 was used to evaluate the <i>in vivo</i> efficacy of compounds. Exemplified by compound <b>17r</b>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> shows the corresponding log unit of CFU reductions under different dose regimens. A good dose–response was observed for compound <b>17r</b>, and at 45 mpk <i>b.i.d.</i>, the bactericidal efficacy that corresponds to one log unit of CFU reduction (vs vehicle) was achieved. Free AUC/MIC mostly predicts the PK/PD relationship of other topoisomerase antibiotics such as FQ. Even though no detailed PKPD study was performed, fAUC/MIC was still used as PK index in this project at the early screening stage. So, in the mouse thigh infection model, <b>17r</b> showed bactericidal efficacy at fAUC/MIC ≥ 6, which also accounts for a total of fAUC ∼ 30 μg·h/mL (See <a class="ref internalNav" href="#notes1" aria-label="SI">SI</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <i>In vivo</i> efficacy in neutropenic mouse thigh model. The plot shows the log unit CFU levels in <i>E. coli</i> ATCC 25922 after 12 h of infection with the treatment of either placebo (black dots), ciprofloxacin control (green dots), or compound <b>17r</b> (blue squares). The line for each animal group represents the mean, mpk is mg/kg, 2x is two doses with 3 h interval. Statistical significance (Dunnett’s multiple comparison test) vs placebo, one asterisk represents adjusted P values 0.01–0.05 and two asterisks <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although the <i>in vivo</i> efficacy in mice infected with <i>E. coli</i> ATCC 25922 was successfully demonstrated, the lower activity of compound <b>17r</b> against other GN pathogens was insufficient for additional <i>in vivo</i> efficacy profiling (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Additional chemistry efforts to further improve this chemical series’ GN broad-spectrum activity and physicochemical properties targeting other infection models such as pneumonia and urinary tract infection (UTI) will be reported in due course.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All the final compounds were synthesized via the key pyrido[2,3-<i>b</i>]indole tricyclic intermediate <b>7</b>, of which the preparation is illustrated in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. 2,4-Difluoro-1-nitro-benzene was reacted with ethyl amine in EtOH via nucleophilic aromatic substitution to afford nitroaniline <b>2</b>, which was then deprotonated by NaH and reacted with di-<i>tert</i>-butyl carbonate to give the Boc-protected nitroaniline <b>3</b>. The nitro group of the aniline <b>3</b> was reduced by H<sub>2</sub> with palladium on carbon, to afford aniline <b>4</b>. Subsequent Buchwald–Hartwig cross-coupling reaction of the aniline <b>4</b> with 2,3,5-trichloropyridine was carried out in the presence of palladium acetate, cesium carbonate, and BINAP to yield compound <b>5</b>. The tricyclic pyrido[2,3-<i>b</i>]indole <b>6</b> was obtained by cyclization of <b>5</b> via an intramolecular direct arylation using Pd<sub>2</sub>(dba)<sub>3</sub> catalyst and Xantphos ligand. Finally, the bichloro substituted tricyclic intermediate <b>7</b> was obtained from MCPBA oxidation of the tricyclic pyridine followed by POCl<sub>3</sub> chlorination.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Key Intermediate <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtNH<sub>2</sub>, EtOH, 0 °C, 3 h, 63% yield; (b) NaH, Boc<sub>2</sub>O, THF, 0 °C then rt, 16 h, 86% yield; (c) H<sub>2</sub> 50 psi, Pd/C, MeOH, rt, 18 h, 92% yield; (d) 2,3,5-trichloropyridine, Pd(OAc)<sub>2</sub> BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120 °C, 16 h, 74% yield; (e) Pd<sub>2</sub>(dba)<sub>3</sub>, X-Phos, DBU, DMA/o-xylene, 130 °C, 4 h, 51% yield; (f) m-CPBA, DCM, 30 °C; 12 h, then POCl<sub>3</sub>, DMF, −5 to 0 °C, 1 h, 42% yield.</p></p></figure><div class="NLM_p">The general preparation of final compounds <b>10a</b>–<b>u</b> and <b>12a</b>–<b>w</b> is depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The key intermediate <b>7</b> was first reacted with 3-(trifluoromethyl)-1H-pyrazole through base promoted nucleophilic aromatic substitution to give compound <b>8</b>. For the syntheses of compounds <b>10a</b>–<b>u</b>, compound <b>8</b> was reacted with various boronic acids or esters under palladium catalyzed Suzuki coupling reaction conditions to give compounds <b>9</b>. In some cases, additional chemical transformations were carried out before the Boc-protecting group was removed by trifluoroacetic acid treatment to give the final compounds <b>10a</b>–<b>u</b>. For the syntheses of pyrimidine compounds <b>12a</b>–<b>w</b>, intermediate <b>8</b> was reacted with a variety of pyrimidine R3′ substituted boronic acids or esters under Suzuki coupling conditions to afford compounds <b>11</b>; similarly, when necessary, appropriate chemical transformations were applied before the Boc-protecting group was removed by the treatment of trifluoroacetic acid to afford the final compounds <b>12a</b>–<b>w</b>.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Examplar Synthesis of Compounds <b>10a</b>–<b>u</b> and <b>12a</b>–<b>w</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and Conditions: (a) 3-(trifluoromethyl)pyrazole, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, 12 h, 86% yield; (b) <b>9d</b>, 3-cyanopyridine-5-boronic acid pinacol ester, Pd<sub>2</sub> db<sub>3</sub>, X-Phos, K<sub>3</sub>PO<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 12 h, 66% yield; (c) <b>10d</b>, TFA, DCM, rt, 2 h, 81% yield; (d) <b>11j</b>, (2-methoxypyrimidin-5-yl)boronic acid, Pd<sub>2</sub> db<sub>3</sub>, XPhos, K<sub>3</sub>PO<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 12 h, 64% yield; (e) <b>12j</b>, TFA, DCM, rt, 1 h, 40% yield.</p></p></figure><div class="NLM_p">Finally, the general synthesis for the preparation of final compounds <b>15a</b>–<b>z</b>, <b>17r, 17u</b>, and <b>17x</b> is depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Similar to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, intermediate <b>7</b> was first reacted with a variety of derivatized pyrazole reagents under either base promoted aromatic nucleophilic substitution or Buchwald–Hartwig cross-coupling reactions to afford compounds <b>13a</b>–<b>z</b>. Subsequent Suzuki coupling reaction with 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine-3-carbonitrile then yielded compounds <b>14a</b>–<b>z</b>. Whenever needed, additional chemical transformations were carried out before the Boc-protecting group was removed to give final compounds <b>15a</b>–<b>z</b>. Meanwhile, intermediates <b>13r</b>, <b>13u</b>, and <b>13x</b> were reacted with 2-methoxy-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrimidine individually via Suzuki coupling reactions to give compounds <b>16r</b>, <b>16u</b>, and <b>16x</b>. After the removal of the Boc-protecting group, final compounds <b>17r</b>, <b>17u</b>, and <b>17x</b> were then obtained.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Examplar Synthesis of <b>15a</b>–<b>z</b> and <b>17r</b>, <b>17u</b>, and <b>17x</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>13r</b>, ethyl 3-(trifluoromethyl)-1H-pyrazole-4-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, 33% yield; (b) <b>14r</b>, 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)nicotinonitrile, CsF, Pd<sub>2</sub>(dba)<sub>3</sub>, X-phos; 1,4-dioxane/H<sub>2</sub>O, 110 °C, 12 h, 89% yield; NH<sub>4</sub>Cl, HOBt, PyBOP, DIPEA, DMF, 30 °C, 1 h, 96% yield; BH<sub>3</sub>-THF, 80 °C, 0.5 h, 27% yield; (c) <b>15r</b>, TFA, DCM, rt, 1 h, 75% yield; (d) <b>16x</b>, <i>tert</i>-butyl 4,6-dihydro-1H-pyrrolo[3,4-<i>c</i>]pyrazole-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, 12 h, 93% yield; 2-methoxy-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrimidine, K<sub>3</sub>PO<sub>4</sub>, cataCXium A-Pd-G2, THF-H<sub>2</sub>O, 70 °C, 12 h, 93% yield; (e) <b>17x</b>, TFA, DCM, 30 °C, 1 h, 57% yield.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Due to the rapid increase of bacterial resistance to marketed antibiotics, novel antibacterial agents are needed urgently, in particular, medicines that can treat MDR GN infections because these infections are associated with the highest mortality rates and have limited treatment options. Dual ATPase inhibitors targeting DNA gyrase GyrB subunit and Topoisomerase IV ParE subunit have the potential to be broad-spectrum antibiotics due to the high level of protein conservation across pathogenic bacterial species and overcome target-based FQ resistance. Using an information-driven approach, we converted the 2-carboxamide substituted azaindole scaffold with only GP activity into a novel chemical series that is also very potent against the ESKAPE GN pathogens. Expanding the activity spectrum was mainly achieved by increasing the original AZ scaffold’s rigidity via a unique ring cyclization strategy. A systematic exploration of the physicochemical properties associated with GN antibacterial activity enabled us to identify the lead molecule <b>17r</b>, which has a balanced profile of GN activity, aqueous solubility, and SDPK properties. Moreover, we showed the dose-dependent cidal efficacy of <b>17r</b> in a neutropenic mouse thigh infection model. To further develop this series, the compounds need to show improved broad-spectrum <i>in vitro</i> coverage of GP and GN clinical isolates and in vivo efficacy in multiple infection models (pneumonia and UTI). A novel class of GyrB/ParE dual inhibitors that can cure the most difficult-to-treat resistant infections in the clinic<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> would address an urgent medical need considering the ever-increasing resistance toward existing antibiotics.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77746" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77746" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Methods for Chemistry</h3><div class="NLM_p last">Reactions involving air-sensitive reagents were carried out under an atmosphere of nitrogen or argon. The microwave assisted reactions were carried out in a Biotage Initiator Sixty. Solvents and reagents were obtained from commercial sources and were used without further purification unless otherwise noted. All reactions were monitored by thin-layer chromatography (TLC) on Merck silica gel 60 F254 TLC glass plates (visualized by UV fluorescence at λ = 254 nm) or analytical LC-MS. All intermediates and final compounds were purified by either silica gel flash chromatography or preparative HPLC (prep-HPLC) using one of the following instruments: (i) Biotage SP1 system and the Quad 12/25 cartridge module; (ii) ISCO Combi-flash chromatography instrument. Silica gel brand and pore size: (a) KP-SIL 60 Å, particle size 40–60 μm; (b) CAS registry no., silica gel, 63231-67-4, particle size 47–60 μm silica gel; (c) ZCX from Qingdao Haiyang Chemical Co., Ltd., pore 200–300 or 300–400; (iii) prep-HPLC on a reverse-phase column using a Waters XBridge OBD Phenyl (30 mm × 100 mm, 5 μm) or OBD RP18 (30 mm × 100 mm, 5 μm) column under acidic conditions (A, 0.1% formic acid in H<sub>2</sub>O; B, 0.1% formic acid in acetonitrile) or basic conditions (A, 0.01% ammonia in H<sub>2</sub>O; B, acetonitrile). For SFC chiral separation, either intermediates or final compounds were separated using a chiral column (Daicel Chiralpak IC, 30 mm × 250 mm, 5 μm) on a Mettler Teledo SFC-Multigram system (solvent system of 95% CO<sub>2</sub> and 5% IPA (0.5% Et<sub>3</sub>N in IPA), backpressure of 100 bar, UV detection at 254 nm). Optical rotation was measured using a Rudolph Autopol V automatic polarimeter at a wavelength of 589 nm. The purity of final compounds as measured by LC-MS was ≥95%. LC-MS spectra were obtained using UPLC coupled with single quadrupole mass detector (Waters Acquity UPLC-3100 Mass Detector, Waters Acquity UPLC-SQ Detector, Waters Acquity UPLC-SQ Detector). Standard LC-MS conditions were as follows. Columns: Waters Acquity BEH C18, 2.1 mm × 50 mm × 1.7 μm and Waters Acquity CSH C18 column, 2.1 mm × 50 mm × 1.7 μm. Flow rate: 0.8 mL/min. Gradient: 5–95% eluent B over 3 min under mobile phase conditions: (1) acidic condition: A, 0.1% formic acid in H<sub>2</sub>O; B, 0.1% formic acid in acetonitrile; or (2) basic condition: A, 0.05% NH<sub>3</sub>·H<sub>2</sub>O in H<sub>2</sub>O; B, acetonitrile. Mass spectra (MS): Generally only ions which indicate the parent mass are reported, and unless otherwise stated the mass ion quoted is the positive mass ion (M+H)<sup>+</sup>. NMR spectra were obtained using Bruker Avance 400 MHz spectrometer, operating at 400.13 MHz (<sup>1</sup>H) and 100.62 MHz (<sup>13</sup>C). High-resolution mass spectra (HRMS) were obtained on Xevo G2-XS-QTOF mass spectrometer equipped with an electrospray ionization source. All of the reported yields are for isolated products and not optimized.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthetic Procedure for the Synthesis of <i>tert</i>-Butyl N-(3,4-Dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (<b>7</b>)</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> N-Ethyl-5-fluoro-2-nitro-aniline (<b>2</b>)</h4><div class="NLM_p last">Ethylamine solution (234.26 g, 1.56 mmol, 30% in EtOH) was added dropwise to 2,4-difluoronitrobenzene (80 g, 0.503 mol) at 0 °C over 15 min. After completion of the addition, the reaction mixture was stirred at 0 °C for 3 h. The solution was diluted with EtOH (500 mL) and poured into 2.5 L of ice–water. The resulting precipitates were collected by filtration and dried <i>in vacuo</i> to afford the title compound (143 g, 63% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 185.0.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>tert</i>-Butyl N-Ethyl-N-(5-fluoro-2-nitro-phenyl)carbamate (<b>3</b>)</h4><div class="NLM_p last">To a suspension of sodium hydride (32.58 g, 814.48 mmol, 60% dispersion in mineral oil) in dry THF (800 mL) was added N-ethyl-5-fluoro-2-nitro-aniline (30 g, 162.9 mmol) portionwise at 0 °C. The solution was stirred at 0 °C for 1 h. Then the solution of di-<i>tert</i>-butyl dicarbonate (30 g, 162.9 mmol) in THF (200 mL) was added dropwise and the reaction continued for another 15 h at 15 °C. The reaction mixture was poured into 350 mL of ice–water and extracted with EtOAc (500 mL) two times. Combined organics were dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude product was purified by flash chromatography (silica gel, 1–5% ethyl acetate in petroleum ether) to afford the title compound (40 g, 86.4% yield) as a yellow solid. MS obsd (ESI+) [([M–100]+H)<sup>+</sup>] 185.1.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>tert</i>-Butyl N-(2-Amino-5-fluoro-phenyl)-N-ethyl-carbamate (<b>4</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl N-ethyl-N-(5-fluoro-2-nitro-phenyl)carbamate (120 g, 422.1 mmol) in MeOH (1 L) was added palladium on carbon (8 g, 50 wt % loading) under N<sub>2</sub>; then the suspension was degassed under vacuum and purged with H<sub>2</sub> several times. The reaction mixture was stirred at 15 °C for 18 h under H<sub>2</sub> atmosphere (50 psi). The remaining palladium catalyst was removed by filtration, and the filtrate was concentrated <i>in vacuo</i> to afford the title compound (98.2 g, 91.5% yield) as a white solid. MS obsd (ESI+) [([M–100]+H)<sup>+</sup>] 155.1.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>tert</i>-Butyl N-[2-[(3,5-Dichloro-2-pyridyl)amino]-5-fluoro-phenyl]-N-ethyl-carbamate (<b>5</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl N-(2-amino-5-fluoro-phenyl)-N-ethyl-carbamate (60 g, 235.94 mmol) and 2,3,5- trichloropyridine (45.2 g, 247.74 mmol) in 1,4-dioxane (800 mL) was added cesium carbonate (153.74 g, 471.88 mmol), palladium(II) acetate (2.65 g, 11.8 mmol), and BINAP (14.7 g, 22.36 mmol). The reaction was stirred at 120 °C for 16 h under nitrogen atmosphere. Then the reaction mixture was cooled down to room temperature and diluted with EtOAc (600 mL) and filtered. The precipitate was removed by filtration and the filtrate was concentrated <i>in vacuo</i> and the residue was purified by flash chromatography (silica gel, 0 to 5% EtOAc in petroleum ether) to afford the title compound (70 g, 74.1% yield) as a solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 400.0, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 402.0.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>tert</i>-Butyl N-(3-Chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (<b>6</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl N-[2-[(3,5-dichloro-2-pyridyl)amino]-5-fluoro-phenyl]-N-ethyl-carbamate (8 g, 20 mmol) and DBU (6.09 g, 40 mmol) in the mixture of <i>o</i>-xylene (13 mL) and N,N-dimethylacetamide (13 mL) was added Pd<sub>2</sub>(dba)<sub>3</sub> (3.66 g, 4 mmol), and Xantphos (2.86 g, 6 mmol). The mixture was degassed with nitrogen for 5 min and then stirred at 130 °C for 3.5 h. The reaction mixture was cooled back to room temperature, poured into water (100 mL), and then extracted with EtOAc (300 mL) three times. The organic layer was washed with aqueous CaCl<sub>2</sub> solution (1N, 120 mL) three times and brine (160 mL) twice. The organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude product was purified by flash chromatography (silica gel, EtOAc/pertrolieum ether = 30:1 to 3:1) and recrystallization in MeOH (40 mL) to afford the title compound (3.7 g, 50.9% yield) as a pale yellow solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 364.1, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 366.1.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>tert</i>-Butyl N-(3,4-Dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (<b>7</b>)</h4><div class="NLM_p">To a solution of <i>tert</i>-butyl N-(3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (6 g, 16.5 mmol) in DCM (200 mL) was added 3-chloroperbenzoic acid (11.4 g, 66 mmol) at 0 °C. After the addition, the reaction was continued at 30 °C for 12 h. The reaction mixture was then poured into aqueous sodium sulfite solution (10%, 150 mL) and stirred for 1 h followed by the extraction with EtOAc (750 mL) three times. Combined organics were washed with aqueous sodium bicarbonate solution (5 N, 200 mL) and brine (250 mL), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give <i>tert</i>-butyl N-(3-chloro-6-fluoro-1-oxido-9H-pyrido[2,3-<i>b</i>]indol-1-ium-8-yl)-N-ethyl-carbamate (6 g, 96% yield) as a brown solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 380.1, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 382.0.</div><div class="NLM_p last">To a solution of <i>tert</i>-butyl N-(3-chloro-6-fluoro-1-oxido-9H-pyrido[2,3-<i>b</i>]indol-1-ium-8-yl)-N-ethyl-carbamate (6 g, 15.8 mmol) in DMF (100 mL) was added phosphorus(V) oxychloride (27.1 g, 176.2 mmol) dropwise at −5 °C. The mixture was stirred at −5 to 0 °C for 1 h before being poured into saturated aqueous sodium bicarbonate solution (300 mL) at 0 °C. The mixture was extracted by EtOAc (750 mL) twice, and combined organics were washed with saturated aqueous NaHCO<sub>3</sub> solution (150 mL × 2), aqueous CaCl<sub>2</sub> solution (1 N, 150 mL × 2), and brine (150 mL × 2). The organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude product was washed with MeOH (120 mL) to afford the title compound (2.76 g, 43.8% yield) as a pale yellow solid. MS obsd (ESI+) 398 [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 398.0, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 400.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.57 (br. s., 1 H) 8.65 (s, 1 H) 8.08 (d, 1 H) 7.42 (dd, 2.13 Hz, 1 H) 3.67 (br. s., 2 H) 0.98–1.60 (m, 12 H).</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Synthetic Procedure for the Synthesis of 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinic Acid (<b>10a</b>)</h4><div id="sec5_2_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Step (a)</h5><div class="NLM_p last">Preparation of methyl 5-[8-(ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carboxylate. To a stirred solution of <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (<b>Intermediate 8</b>, 70 mg, 141 μmol) and methyl 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)nicotinate (74 mg, 281 μmol) in 1,4-dioxane (4 mL) and H<sub>2</sub>O (1.33 mL) was added dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine (13.4 mg, 28.1 μmol), potassium phosphate (89.5 mg, 422 μmol), and Pd<sub>2</sub> dba<sub>3</sub> (12.9 mg, 14.1 μmol). The resulting mixture solution was heated at 110 °C for 4 h under N<sub>2</sub>. The mixture was then cooled down to room temperature and evaporated to dryness under reduced pressure. The residue was partitioned between EtOAc and water, and the separated organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The residue was redissolved in DCM (5 mL) and added TFA (0.5 mL) and stirred at room temperature for 2 h. After the mixture was concentrated under reduced pressure, the residue was subject to prep-HPLC purification to afford the title compound (40 mg, 57% yield) as a white solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 499.4.</div></div><div id="sec5_2_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Step (b)</h5><div class="NLM_p last">Preparation of 5-(8-(ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinic acid. To a solution of methyl 5-(8-(ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinate (40 mg, 80.3 μmol) in CH<sub>3</sub>CN (4 mL) was added aqueous NaOH solution (1N, 1 mL). The reaction mixture was stirred at room temperature for 3 h before it was concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (7 mg, 18% yield) as a white solid. MS obsd (ESI+) 485.4. HRMS calcd [(M+H)<sup>+</sup>] 485.1349, measured [(M+H)<sup>+</sup>] 485.1349. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.94 (brs, 1H), 8.55 (s, 1H), 8.48 (brd, 1H), 8.04 (s, 1H), 7.85 (d, 1H), 6.78 (d, 1H), 6.37 (dd, 1H), 5.82 (dd, 1H), 2.66–2.99 (m, 2H), 0.91–1.37 (m, 3H).</div></div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Synthetic Procedure for the Synthesis of 5-[8-(Ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>10d</b>)</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>tert</i>-Butyl N-[3-Chloro-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (<b>8</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl N-(3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (Intermediate <b>7</b>, 80 mg, 0.2 mmol) in DMSO (1 mL) was added 3-(trifluoromethyl)pyrazole (54.4 mg, 0.4 mmol) and K<sub>2</sub>CO<sub>3</sub> (82.9 mg, 0.6 mmol). The resulting mixture was stirred at 120 °C for 12 h. The reaction mixture was then cooled back to room temperature, filtered through a thin Celite pad, and concentrated <i>in vacuo</i>. The residue was purified by prep-HPLC to afford the title compound (85 mg, 86% yield) as a white solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 498.2.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>tert</i>-Butyl N-[3-(5-Cyano-3-pyridyl)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (<b>9d</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (Intermediate <b>8</b>, 60 mg, 0.12 mmol), 3-cyanopyridine-5-boronic acid pinacol ester (55.7 mg, 0.24 mmol), Pd<sub>2</sub>dba<sub>3</sub> (21.9 mg, 0.024 mmol), Xantphos (11.4 mg, 0.024 mmol), and potassium phosphate tribasic (144.7 mg, 0.69 mmol) in 1,4-dioxane (4 mL) and water (0.4 mL) was stirred at 100 °C for 12 h under N<sub>2</sub> atmosphere. The mixture was cooled down to room temperature and poured into water (50 mL) and extracted with EtOAc (50 mL) three times. Combined organics were dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The residue was purified by prep-TLC to afford the title compound (45.3 mg, 66% yield) as a yellow oil. MS obsd (ESI+) [(M+H)<sup>+</sup>] 566.3.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 5-[8-(Ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>10d</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl N-[3-(5-cyano-3-pyridyl)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (45 mg, 0.08 mmol) in DCM (3 mL) was added TFA (2 mL) at 10 °C. Then the solution was stirred at room temperature for 2 h before it was concentrated <i>in vacuo</i>. The residue was purified by prep-HPLC to afford the title compound (30 mg, 81% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 466.1. HRMS calcd [(M+H)<sup>+</sup>] 466.1403, measured [(M+H)<sup>+</sup>] 466.1407. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.192 (s, 1H), 8.977–8.982 (d, 1H), 8.799 (s, 2H), 8.763 (s, 1H), 8.572–8.577 (d, 1H), 8.329 (s, 1H), 8.115 (s, 1H), 7.129–7.135 (d, 1H), 6.480–6.510 (m, 1H), 5.915 (m, 1H), 5.794–5.817 (m, 1H), 3.237–3.283 (m, 2H), 1.297–1.332 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 160.1, 158.2, 153.1, 153.0, 151.5, 148.3, 143.6, 143.3, 140.2, 138.3, 136.9, 136.8, 136.2, 131.8, 125.1, 122.7, 120.8, 120.5, 117.0, 116.9, 116.8, 111.8, 109.3, 106.7, 95.6, 93.5, 38.1, 14.5.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div class="NLM_p first">The following compounds were prepared by a similar procedure to that described for compound <b>10d</b>.</div><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinamide (<b>10b</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 484.2. HRMS calcd [(M+H)<sup>+</sup>] 484.1509, measured [(M+H)<sup>+</sup>] 484.1510. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.13 (s, 1H), 8.95 (d, 1H), 8.70–8.79 (m, 1H), 8.42 (d, 1H), 8.27 (d, 1H), 8.15 (s, 1H), 8.08 (m, 1H), 7.63 (br. s., 1H), 7.08 (d, 1H), 6.50 (m, 1H), 5.89 (br. s., 1H), 5.76 (m, 1H), 3.27 (m, 2H), 1.31–1.36 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm δ ppm 166.40, 157.94, 153.96, 152.92, 151.50, 148.39, 147.95, 143.28, 138.25, 136.82, 136.70, 135.92, 131.05, 129.89, 125.10, 122.07, 116.93, 112.06, 106.62, 95.54, 93.39, 38.09, 14.53.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-<i>N</i>-methylnicotinamide (<b>10c</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 498.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.78 (d, 1H), 8.56 (s, 1H), 8.31 (d, 1H), 8.04 (m, 1H), 7.87 (d, 1H), 6.79 (d, 1H), 6.39 (m, 1H), 5.79–5.86 (m, 1H), 3.15–3.20 (m, 2H), 2.84 (s, 3H), 1.26–1.35 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 165.16, 157.95, 152.92, 151.30, 148.37, 147.52, 143.46, 138.23, 136.77, 135.93, 135.59, 131.07, 130.13, 125.10, 122.06, 116.93, 112.07, 106.64, 95.56, 93.92, 93.18, 38.09, 26.66, 14.52.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 5-[8-(Ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-<i>N</i>-methyl-pyridine-3-carboxamidine (<b>10g</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 497.1. HRMS calcd [(M+H)<sup>+</sup>] 497.1825, measured [(M+H)<sup>+</sup>] 497.1826. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.74 (br s, 1H), 8.65 (s, 1H), 8.48 (s, 1H), 8.20 (m, 2H), 6.95 (d, 1H), 6.52 (m, 1H), 5.78 (dd, 1H), 3.21 (m, 2H), 3.05 (s, 3H), 1.42 (m, 3H).</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> N-Ethyl-6-fluoro-3-(pyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10h</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 442.2. HRMS calcd [(M+H)<sup>+</sup>] 442.1403, measured [(M+H)<sup>+</sup>] 442.1404. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.18 (br. s., 1H), 9.14 (s, 1H), 8.78 (s, 1H), 8.60 (s, 2H), 8.37 (d, 1H), 7.14 (d, 1H), 6.51 (m, 1H), 5.90 (br. s., 1H), 5.78 (m, 1H), 3.22–3.29 (m, 2H), 1.33 (m, 3H).</div></div><div id="sec5_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 3-(5-Aminopyridin-3-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10i</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 456.1. HRMS calcd [(M+H)<sup>+</sup>] 456.1560, measured [(M+H)<sup>+</sup>] 456.1661. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.62 (s, 1H), 8.08 (d, 1H), 7.95 (br s, 1H), 7.63 (s, 1H), 7.32–7.45 (m, 1H), 6.99 (d, 1H), 6.50 (dd, 1H), 5.98 (dd, 1H), 3.34–3.47 (m, 2H), 1.42 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 159.1, 153.0, 148.0, 146.7, 143.3, 138.2, 136.8, 136.0, 133.5, 131.8, 130.0, 125.1, 124.6, 121.6, 121.5, 116.9, 111.9, 106.6, 95.6, 93.5, 38.1, 14.5.</div></div><div id="sec5_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> N-Ethyl-6-fluoro-3-(5-morpholinopyridin-3-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10j</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 526.3. HRMS calcd [(M+H)<sup>+</sup>] 526.1978, measured [(M+H)<sup>+</sup>] 526.1980. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.59–8.70 (m, 1H), 8.21–8.36 (m, 1H), 7.89–8.06 (m, 2H), 7.20–7.37 (m, 1H), 6.96 (d, 1H), 6.50 (dd, 1H), 5.90 (dd, 1H), 3.75–3.87 (m, 4H), 3.03–3.24 (m, 4H), 2.84–3.02 (m, 2H), 1.27–1.49 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 159.1, 152.7, 148.2, 146.8, 143.1, 138.9, 138.2, 136.7, 136.2, 135.9, 131.4, 125.0, 122.7, 122.1, 121.2, 116.9, 112.3, 106.6, 95.5, 93.4, 66.2, 47.8, 38.1, 14.5.</div></div><div id="sec5_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> N-Ethyl-6-fluoro-3-(2-(piperazin-1-yl)pyridin-4-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10k</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 525.5. HRMS calcd [(M+H)<sup>+</sup>] 525.2138, measured [(M+H)<sup>+</sup>] 525.2138. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.55 (s, 1H), 7.98–8.10 (m, 1H), 7.85 (dd, 1H), 6.86 (d, 1H), 6.58–6.69 (m, 2H), 6.40 (dd, 1H), 5.79 (dd, 1H), 3.56–3.74 (m, 4H), 3.12–3.23 (m, 6H), 1.32 (t, 3H).</div></div><div id="sec5_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 3-(5-(Aminomethyl)pyridin-3-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10l</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 470.4. HRMS calcd [(M+H)<sup>+</sup>] 470.1716, measured [(M+H)<sup>+</sup>] 470.1715. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.57–8.68 (m, 2H), 8.34 (d, 1H), 7.93–8.09 (m, 2H), 6.95 (d, 1H), 6.43–6.57 (m, 1H), 5.93 (dd, 1H), 4.23 (s, 2H), 3.21–3.24 (m, 2H), 1.33–1.50 (m, 3H).</div></div><div id="sec5_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)methanol (<b>10m</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 471.1. HRMS calcd [(M+H)<sup>+</sup>] 471.1556, measured [(M+H)<sup>+</sup>] 471.1556. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.64 (s, 1H), 8.55 (s, 1H), 8.31 (s, 1H), 7.98 (d, 1H), 7.82 (s, 1H), 6.92 (d, 1H), 6.49 (dd, 1H), 5.93 (dd, 1H), 4.68 (s, 2H), 3.34–3.45 (m, 2H), 1.42 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 159.1, 152.8, 148.2, 147.5, 147.0, 143.2, 138.2, 138.0, 136.7, 135.9, 135.3, 131.0, 125.1, 122.6, 121.7, 116.9, 112.2, 106.5, 95.5, 93.5, 60.8, 38.1, 14.5.</div></div><div id="sec5_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> N-Ethyl-6-fluoro-3-[5-(oxazol-2-ylmethyl)-3-pyridyl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10q</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 522.0. HRMS calcd [(M+H)<sup>+</sup>] 522.1665, measured [(M+H)<sup>+</sup>] 522.1666. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.2 (s, 1H), 8.68 (s, 1H), 8.46 (s, 1H), 8.35 (s, 1H), 8.20 (s, 1H), 7.99 (s, 1H), 7.42 (s, 1H),7.12 (s, 1H), 7.01 (s, 1H), 6.46 (d, 1H), 5.95 (s, 1H), 5.68 (d, 1H), 4.12 (s, 2H), 3.20 (m, 2H), 1.31 (t, 3H).</div></div><div id="sec5_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> N-Ethyl-6-fluoro-3-(pyrazolo[1,5-<i>a</i>]pyrimidin-6-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10t</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 481.3. HRMS calcd [(M+H)<sup>+</sup>] 481.1512, measured [(M+H)<sup>+</sup>] 481.1512. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.16 (s, 1H), 9.15 (d, 1H), 8.84 (s, 1H), 8.40 (d, 1H), 8.28 (d, 1H), 8.10 (d, 1H), 7.13 (d, 1H), 6.76 (d, 1H), 6.51 (m, 1H), 5.90 (br. s., 1H), 5.81 (m, 1H), 3.22–3.31 (m, 2H), 1.30–1.37 (t, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 160.0, 158.2, 153.0, 150.0, 148.8, 147.1, 146.0, 138.5, 136.8, 136.7, 136.2, 135.1, 125.1, 119.0, 116.9, 116.8, 112.0, 106.8, 96.8, 95.5, 93.5, 38.1, 14.5.</div></div><div id="sec5_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> N-Ethyl-6-fluoro-3-(2-methylpyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12a</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 456.2. HRMS calcd [(M+H)<sup>+</sup>] 456.1560, measured [(M+H)<sup>+</sup>] 456.1661. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.15 (s, 1H), 8.75 (s, 1H), 8.48 (s, 2H), 8.33 (s, 1H), 7.14 (d, 1H), 6.44–6.54 (m, 1H), 5.88 (br s, 1H), 5.73 (dd, 1H), 3.22–3.30 (m, 2H), 2.62 (s, 3H), 1.32 (t, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 166.7, 160.0, 158.2, 156.6, 153.0, 148.2, 143.5, 143.2, 138.2, 136.8, 136.7, 136.0, 126.5, 125.1, 122.7, 120.6, 119.6, 116.9, 116.8, 112.1, 106.8, 95.6, 93.4, 38.1, 25.8, 14.5.</div></div><div id="sec5_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>S</i>)-1-(5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)ethanol (<b>12b</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 486.2. HRMS calcd [(M+H)<sup>+</sup>] 486.1665, measured [(M+H)<sup>+</sup>] 486.1665. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4,) δ ppm 8.6–8.7 (m, 1H), 8.6–8.6 (m, 2H), 8.0–8.1 (m, 1H), 6.9–7.0 (m, 1H), 6.4–6.5 (m, 1H), 5.9–5.9 (m, 1H), 4.94 (q, 1H), 3.2–3.3 (m, 2H), 1.5–1.5 (m, 3H), 1.4–1.4 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 171.4, 159.1, 156.6, 153.1, 148.3, 143.4, 138.3, 136.8, 136.1, 127.6, 125.1, 123.0, 120.3, 119.5, 116.8, 112.2, 106.8, 95.6, 93.4, 70.1, 38.1, 23.2, 14.5.</div></div><div id="sec5_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-1-(5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)ethanol (<b>12c</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 486.2. HRMS calcd [(M+H)<sup>+</sup>] 486.1665, measured [(M+H)<sup>+</sup>] 486.1665. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.6–8.7 (m, 1H), 8.61 (s, 2H), 8.1–8.1 (m, 1H), 6.9–7.0 (m, 1H), 6.5–6.5 (m, 1H), 5.9–6.0 (m, 1H), 4.95 (q, 1H), 3.3–3.3 (m, 2H), 1.5–1.5 (m, 3H), 1.4–1.5 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 171.4, 160.3, 158.0, 156.6, 153.1, 148.3, 143.4, 138.3, 136.8, 136.1, 127.6, 125.1, 119.5, 121.9, 116.8, 112.2, 106.8, 95.6, 93.4, 70.1, 38.1, 23.2, 14.5.</div></div><div id="sec5_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 2-(5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)propan-2-ol (<b>12d</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 500.3. HRMS calcd [(M+H)<sup>+</sup>] 500.1822, measured [(M+H)<sup>+</sup>] 500.1824. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.66 (s, 1H), 8.60 (s, 2H), 8.10 (d, 1H), 6.97 (d, 1H), 6.51 (m, 1H), 5.96 (m, 1H), 1.60 (s, 6H), 1.43 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 173.5, 160.3, 158.0, 156.3, 153.1, 148.3, 143.6, 138.3, 136.7, 136.1, 127.2, 125.1, 123.0, 120.3, 119.4, 116.9, 116.8, 112.0, 106.8, 95.6, 93.4, 73.1, 38.1, 30.1, 14.5.</div></div><div id="sec5_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidine-2-carboxamide (<b>12h</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 485.2.HRMS calcd [(M+H)<sup>+</sup>] 485.1461, measured [(M+H)<sup>+</sup>] 485.1463. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.7–8.8 (m, 3H), 8.0–8.1 (m, 1H), 6.9–7.0 (m, 1H), 6.5–6.6 (m, 1H), 5.9–6.0 (m, 1H), 3.3–3.3 (m, 2H), 1.3–1.5 (m, 3H). <sup>13</sup>C NMR (101 MHz, MeOH-<i>d4</i>) δ ppm −1.46, 13.12, 29.52, 37.84, 93.91, 94.11, 95.86, 96.11, 105.99, 112.66, 112.69, 117.06, 118.43, 125.05, 131.34, 134.78, 136.22, 136.31, 138.37, 146.90, 153.32, 156.63, 156.76, 158.59, 160.45, 165.29.</div></div><div id="sec5_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> N-Ethyl-6-fluoro-3-(2-methoxypyrimidin-5-yl)-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12j</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 472.2. HRMS calcd [(M+H)<sup>+</sup>] 472.1509, measured [(M+H)<sup>+</sup>] 472.1509. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 8.70 (s, 1H), 8.38 (s, 2H), 8.29 (s, 1H), 7.10 (s, 1H), 6.46 (m, 1H), 5.72 (m, 1H), 3.92 (s, 3H), 3.24 (m, 2H), 1.32 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 164.7, 159.1, 159.1, 152.9, 148.1, 143.3, 138.2, 136.7, 136.0, 125.1, 123.3, 119.4, 121.8, 116.8, 112.1, 106.7, 106.7, 95.5, 93.4, 55.2, 38.1, 14.5.</div></div><div id="sec5_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 3-(2-Ethoxypyrimidin-5-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12k</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 486.5. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.50 (s, 1H), 8.26 (s, 2H), 7.94 (d, 1H), 6.85 (d, 1H), 6.38 (m, 1H), 5.79 (m, 1H), 4.34 (m, 2H), 3.21 (d, 2H), 1.26–1.35 (m, 6H).</div></div><div id="sec5_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 3-(2-(Azetidin-3-yloxy)pyrimidin-5-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12l</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 513.2. HRMS calcd [(M+H)<sup>+</sup>] 513.1774, measured [(M+H)<sup>+</sup>] 513.1774. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.61 (s, 1H), 8.38 (s, 2H), 8.07 (s, 1H), 6.97 (s, 1H), 6.51 (m, 1H), 5.90 (m, 1H), 5.50 (m, 1H), 4.00 (m, 2H), 3.75 (m, 2H), 3.51 (m, 1H), 3.31 (m, 2H), 1.42 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 163.4, 160.0, 159.2, 158.2, 152.9, 148.1, 143.5, 143.2, 138.2, 136.8, 136.7, 136.1, 125.1, 123.7, 122.7, 120.6, 119.4, 116.8, 116.7, 112.0, 112.0, 106.7, 95.5, 93.4, 70.7, 53.8, 38.1, 14.5.</div></div><div id="sec5_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> N-Ethyl-6-fluoro-3-(2-(pyrrolidin-3-yloxy)pyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12m</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 527.3. HRMS calcd [(M+H)<sup>+</sup>] 527.1931, measured [(M+H)<sup>+</sup>] 527.1931. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.51 (s, 1H), 8.33 (s, 2H), 7.98 (s, 1H), 6.86 (s, 1H), 6.40 (m, 1H), 5.80 (m, 1H), 5.60 (s, 1H), 3.40 (m, 2H), 3.31 (m, 2H), 3.20 (m, 2H), 2.20 (m, 3H), 1.32 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 163.3, 160.0, 159.3, 158.2, 152.9, 148.1, 143.5, 143.2, 138.2, 136.8, 136.7, 136.1, 125.1, 124.0, 122.7, 119.2, 116.8, 116.7, 112.1, 106.7, 95.5, 93.3, 76.2, 50.8, 44.3, 38.1, 31.1, 14.5.</div></div><div id="sec5_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> N-Ethyl-6-fluoro-3-(2-(piperidin-4-yloxy)pyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12n</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 541.3. HRMS calcd [(M+H)<sup>+</sup>] 541.2087, measured [(M+H)<sup>+</sup>] 541.2090. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppmppm 8.5–8.5 (m, 1H), 8.3–8.3 (m, 2H), 8.0–8.0 (m, 1H), 7.2–7.3 (m, 1H), 6.8–6.9 (m, 1H), 6.4–6.4 (m, 1H), 5.8–5.8 (m, 1H), 5.2–5.3 (m, 1H), 3.3–3.3 (m, 1H), 3.1–3.1 (m, 1H), 3.0–3.0 (m, 1H), 2.1–2.2 (m, 2H), 1.9–2.0 (m, 3H), 1.7–1.9 (m, 1H), 1.3–1.3 (m, 3H).</div></div><div id="sec5_4_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 3-(2-Aminopyrimidin-5-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12p</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 457.2. HRMS calcd [(M+H)<sup>+</sup>] 457.1512, measured [(M+H)<sup>+</sup>] 457.1513. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.01 (s, 1H), 8.66 (s, 1H), 8.30 (d, 1H), 8.01 (s, 2H), 7.13 (d, 1H), 6.80 (s, 2H), 6.47 (m, 1H), 5.86 (m, 1H), 5.67 (m, 1H), 3.22–3.28 (m, 2H), 1.32 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 163.1, 159.9, 158.1, 157.6, 152.4, 147.9, 142.9, 137.8, 136.7, 136.6, 135.8, 125.0, 122.8, 121.0, 120.7, 117.5, 116.9, 116.8, 112.3, 106.6, 95.3, 93.3, 38.1, 14.5,</div></div><div id="sec5_4_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> N-Ethyl-6-fluoro-3-(2-(methylamino)pyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12q</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 471.3. HRMS calcd [(M+H)<sup>+</sup>] 471.1669, measured [(M+H)<sup>+</sup>] 471.1669. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.03 (s, 1H), 8.66 (s, 1H), 8.31 (s, 1H), 7.22 (br. s., 1H), 7.14 (d, 1H), 6.68 (br. s., 1H), 6.47 (m, 1H), 5.86 (br. s., 1H), 5.66 (m, 1H), 5.33 (m, 1H), 3.19–3.29 (m, 2H), 2.74 (d., 3H), 1.29–1.35 (t, 3H).</div></div><div id="sec5_4_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 3-(2-(Azetidin-3-ylamino)pyrimidin-5-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12r</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 512.1. HRMS calcd [(M+H)<sup>+</sup>] 512.1934, measured [(M+H)<sup>+</sup>] 512.1937. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.548 (s, 1 H), 8.119 (s, 2 H), 8.017(s, 1 H), 6.961–6.966 (d, 1 H), 6.462–6.489 (d, 1 H), 5.847–5.874 (d, 1 H), 3.879–3.974 (m, 2 H), 3.699–3.770 (m, 2 H), 3.369 (m, 1 H), 3.281–3.368 (m, 2 H), 1.399–1.435 (m, 3 H).</div></div><div id="sec5_4_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> N-Ethyl-6-fluoro-3-[2-(pyrrolidin-3-ylamino)pyrimidin-5-yl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12s</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 526.2. HRMS calcd [(M+H)<sup>+</sup>] 526.2091, measured [(M+H)<sup>+</sup>] 526.2093. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.54 (s, 1H), 8.17 (s, 2H), 8.02 (s, 1H), 6.96 (s, 1H),6.47 (d, 1H), 5.86 (d, 1H), 4.59 (m, 1H), 3.57 (m, 2H), 3.01 (m, 1H), 2.88 (m, 1H), 2.41 (m, 1H), 2.25 (m, 1H), 1.40–1.43 (m, 5H).</div></div><div id="sec5_4_25_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> N-Ethyl-6-fluoro-3-(2-(piperidin-4-ylamino)pyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12t</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 540.3. HRMS calcd [(M+H)<sup>+</sup>] 540.2247, measured [(M+H)<sup>+</sup>] 540.2247. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.05 (br. s., 1H), 8.66 (s, 1H), 8.34 (d, 1H), 8.06 (br. s., 1H), 7.42 (d, 1H), 7.15 (d, 1H), 6.47 (m, 1H), 5.87 (br. s., 1H), 5.66 (m, 1H), 5.27–5.36 (m, 1H), 3.25 (br. s., 2H), 3.10 (br. s., 1H), 2.65–2.76 (m, 2H), 1.97–2.06 (m, 2H), 1.87 (d, 2H), 1.46 (d, 2H), 1.27–1.34 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 161.0, 160.2, 157.8, 157.5, 152.4, 147.9, 143.3, 142.9, 137.8, 136.7, 135.8, 125.0, 123.1, 120.9, 120.5, 117.1, 116.8, 112.3, 106.6, 95.3, 93.3, 48.8, 45.6, 38.1, 33.4, 14.5.</div></div><div id="sec5_4_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> N-Ethyl-6-fluoro-3-(2-(piperazin-1-yl)pyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12u</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 526.4. HRMS calcd [(M+H)<sup>+</sup>] 526.2091, measured [(M+H)<sup>+</sup>] 526.2092. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 8.6–8.7 (m, 1H), 8.3–8.4 (m, 1H), 8.13 (s, 1H), 7.6–7.7 (m, 1H), 7.16 (d, 1H), 6.4–6.5 (m, 1H), 5.8–5.9 (m, 1H), 5.6–5.7 (m, 1H), 5.3–5.4 (m, 1H), 3.6–3.7 (m, 2H), 3.2–3.3 (m, 2H), 2.7–2.8 (m, 2H), 1.9–2.1 (m, 2H), 1.6–1.7 (m, 1H), 1.4–1.5 (m, 1H), 1.3–1.3 (m, 3H).</div></div><div id="sec5_4_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> N-Ethyl-6-fluoro-3-(2-morpholinopyrimidin-5-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>12v</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 527.4. HRMS calcd [(M+H)<sup>+</sup>] 527.1931, measured [(M+H)<sup>+</sup>] 527.1932. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.06 (br. s., 1H), 8.67 (s, 1H), 8.33 (d, 1H), 8.18 (s, 2H), 7.17 (d, 1H), 6.47 (m, 1H), 5.86 (br. s., 1H), 5.65 (m, 1H), 3.53–3.74 (m, 6H), 3.14–3.30 (m, 2H), 1.96–2.05 (m, 2H), 1.32 (m, 3H).</div></div><div id="sec5_4_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 5-[4-(3-Cyanopyrazol-1-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15a</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 423.2. HRMS calcd [(M+H)<sup>+</sup>] 423.1482, measured [(M+H)<sup>+</sup>] 423.1482. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.22 (s, 1H), 9.01 (d, 1H), 8.77 (s, 1H), 8.56 (d, 1H), 8.43 (d, 1H), 8.21 (s, 1H), 7.37 (d, 1H), 6.52 (m, 1H), 5.76 (m, 1H), 3.27 (q, 2H), 1.33 (t, 3H).</div></div><div id="sec5_4_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 5-[4-(3-Aminopyrazol-1-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15c</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 413.2. HRMS calcd [(M+H)<sup>+</sup>] 413.1638, measured [(M+H)<sup>+</sup>] 413.1636. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 11.97 (s, 1 H), 8.96 (d, 1 H), 8.66 (s, 1 H), 8.53 (d, 1 H), 8.10 (t, 1 H), 7.58 (d, 1 H), 6.35–6.55 (m, 2 H), 5.91 (d, 1 H), 3.25 (q, 2 H), 1.32 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 158.6, 158.3, 153.4, 153.0, 151.0, 148.5, 140.1, 139.7, 136.3, 134.1, 133.0, 124.6, 120.3, 117.9, 117.4, 109.3, 96.9, 95.3, 38.1, 14.6.</div></div><div id="sec5_4_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 5-[8-(Ethylamino)-6-fluoro-4-(3-methylpyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15e</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 412.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.936 (s, 1 H), 8.691 (s, 1 H), 8.505 (s, 1 H), 8.066 (s, 1 H), 7.823 (s, 1 H), 6.454–6.482 (d, 1 H), 6.397 (s, 1 H), 6.040–6.063 (d, 1 H), 5.860 (s, 3 H), 3.230–3.276 (m, 2 H), 2.281 (s, 1 H), 1.295–1.331 (t, 3 H).</div></div><div id="sec5_4_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 5-[8-(Ethylamino)-4-(3-ethylpyrazol-1-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15f</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 426.1. HRMS calcd [(M+H)<sup>+</sup>] 426.1842, measured [(M+H)<sup>+</sup>] 426.1843. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.01 (s, 1H), 8.94 (d, 1H), 8.71 (s, 1H), 8.56 (d, 1H), 7.97 (t, 1H), 7.83 (d, 1H), 6.43–6.49 (m, 2H), 6.10 (m, 1H), 5.84 (br. s, 1H), 3.26 (q, 2H), 2.64 (m, 2H), 1.32 (t, 3H), 1.19 (t, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 159.9, 158.2, 156.5, 153.3, 153.0, 151.0, 148.3, 139.8, 139.8, 136.5, 133.9, 132.6, 124.8, 120.6, 117.6, 117.2, 112.1, 109.2, 106.6, 95.4, 94.4, 38.1, 21.4, 14.6, 14.6.</div></div><div id="sec5_4_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 5-[8-(Ethylamino)-6-fluoro-4-(3-isopropylpyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15g</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 538.2. HRMS calcd [(M+H)<sup>+</sup>] 440.1999, measured [(M+H)<sup>+</sup>] 440.2001. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.01 (s, 1 H) 8.94 (d, 1 H) 8.71 (s, 1 H) 8.59 (d, 1 H) 7.77–7.92 (m, 2 H) 6.40–6.50 (m, 2 H) 6.11 (dd, 1 H) 5.84 (br. s., 1 H) 3.20–3.28 (m, 2 H) 2.94–3.05 (m, 1 H) 1.32 (t, 3 H) 1.23 (d, 6 H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 160.9, 159.9, 158.1, 153.3, 153.0, 151.0, 148.2, 139.8, 139.7, 136.6, 136.5, 133.7, 132.6, 124.8, 120.6, 117.6, 117.5, 117.2, 112.1, 112.1, 109.2, 105.3, 95.3, 94.4, 38.1, 27.8, 23.1, 14.5.</div></div><div id="sec5_4_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 5-[4-(3-<i>tert</i>-Butylpyrazol-1-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15h</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 454.3. HRMS calcd [(M+H)<sup>+</sup>] 454.2155, measured [(M+H)<sup>+</sup>] 454.2155. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.01 (s, 1 H), 8.93 (s, 1 H), 8.70 (s, 1 H), 8.60 (d, 1 H), 7.77–7.82 (m, 2 H), 6.42–6.50 (m, 2 H), 6.14 (d, 1 H), 5.84 (br. s., 1 H), 3.20–3.28 (m, 2 H), 1.25–1.34 (m, 12 H).</div></div><div id="sec5_4_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 5-[8-(Ethylamino)-6-fluoro-4-(3-fluoropyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15i</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 448.3. HRMS calcd [(M+H)<sup>+</sup>] 448.1498, measured [(M+H)<sup>+</sup>] 448.1498. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.84 (d, 1H), 8.64 (s, 1H), 8.58 (d, 1H), 8.02 (m, 1H), 7.96 (d, 1H), 6.83 (d, 1H), 6.49 (m, 1H), 5.99 (m, 1H), 5.17 (s, 1H), 4.58 (m, 2H), 1.41 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 159.9, 157.6, 154.5, 151.8, 150.9, 147.2, 143.2, 141.3, 136.6, 136.4, 134.5, 133.4, 129.8, 124.7, 124.4, 119.4, 119.3, 118.3, 117.3, 114.8, 114.7, 113.2, 110.8, 109.0, 94.8, 93.8, 38.1, 14.6.</div></div><div id="sec5_4_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)phenyl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinonitrile (<b>15j</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 476.0. HRMS calcd [(M+H)<sup>+</sup>] 476.1498, measured [(M+H)<sup>+</sup>] 476.1501. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 8.84 (d, 1H), 8.63–8.57 (m, 2H), 8.20–8.17 (m, 1H), 7.85 (br d, 1H), 7.78–7.67 (m, 3H), 6.42 (br d, 1H), 5.75 (dd, 1H), 3.25 (q, 2H), 1.38–1.29 (t, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 159.6, 157.8, 154.6, 151.8, 150.8, 147.4, 141.6, 141.4, 137.3, 134.3, 134.3, 129.9, 127.0, 125.7, 124.8, 123.3, 119.0, 119.1, 117.2, 113.8, 108.8, 94.8, 93.6, 38.1, 14.6.</div></div><div id="sec5_4_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 5-[8-(Ethylamino)-6-fluoro-4-[4-(trifluoromethyl)imidazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15k</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 466.2. HRMS calcd [(M+H)<sup>+</sup>] 466.1403, measured [(M+H)<sup>+</sup>] 466.1403. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.19 (s, 1 H) 9.02 (d, 1 H) 8.68–8.78 (m, 2 H) 8.32 (s, 1 H) 8.18–8.27 (m, 2 H) 6.52 (dd, 1 H) 5.69 (dd, 1 H) 3.26 (q, 2 H) 1.32 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 160.4, 158.1, 151.8, 148.2, 140.5, 140.3, 137.0, 136.9, 135.5, 131.4, 125.0, 123.5, 122.7, 121.3, 120.8, 117.0, 116.9, 116.8, 112.1, 109.2, 109.3, 95.6, 92.7, 38.1, 14.5.</div></div><div id="sec5_4_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 5-[8-(Ethylamino)-6-fluoro-4-[4-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15l</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 466.1. HRMS calcd [(M+H)<sup>+</sup>] 466.1403, measured [(M+H)<sup>+</sup>] 466.1404. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.19 (s, 1H), 9.00 (d, 1H), 8.82 (s, 1H), 8.77 (s, 1H), 8.58 (s, 1H), 8.44 (s, 1H), 8.12 (d, 1H), 6.51 (m, 1H), 5.84 (m, 1H), 3.26 (q, 2H), 1.32 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 160.3, 158.0, 153.1, 151.5, 148.3, 140.1, 139.3, 136.9, 136.7, 134.0, 131.8, 125.1, 124.2, 120.8, 117.0, 116.8, 114.4, 112.0, 109.3, 95.6, 93.6, 38.1, 14.5.</div></div><div id="sec5_4_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 5-(8-(Ethylamino)-6-fluoro-4-(4-(trifluoromethyl)thiazol-2-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinonitrile (<b>15m</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 483.2. HRMS calcd [(M+H)<sup>+</sup>] 483.1015, measured [(M+H)<sup>+</sup>] 483.1017. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.86 (d, 1H), 8.73 (d, 1H), 8.60 (s, 1H), 8.50 (s, 1H), 8.23 (t, 1H), 6.50 (dd, 1H), 6.08 (dd, 1H), 3.36–3.34 (m, 2H), 1.43 (t, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 164.8, 159.8, 157.9, 154.1, 151.8, 151.5, 147.6, 143.5, 143.2, 141.2, 136.7, 136.6, 133.7, 133.0, 127.9, 125.3, 124.1, 123.5, 122.0, 119.9, 118.3, 118.2, 117.1, 113.8, 113.8, 109.2, 95.5, 95.3, 94.0, 93.8, 38.1, 14.5.</div></div><div id="sec5_4_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 1-(3-(5-Cyanopyridin-3-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxylic acid (<b>15n</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 510.2. HRMS calcd [(M+H)<sup>+</sup>] 510.1302, measured [(M+H)<sup>+</sup>] 510.1302. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 13.39 (br. s., 1 H), 12.25 (s, 1 H), 9.02 (d, 1 H), 8.93 (s, 1 H), 8.78 (s, 1 H), 8.62 (d, 1 H), 8.22 (s, 1 H), 6.53 (dd, 1 H), 5.93 (br. s., 1 H), 5.78 (dd, 1 H), 3.27 (dd, 2 H), 1.33 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 161.6, 160.4, 158.1, 153.1, 151.8, 148.3, 142.5, 142.1, 140.5, 137.4, 137.0, 136.9, 131.5, 125.3, 122.1, 120.8, 119.4, 117.0, 116.7, 116.5, 115.4, 111.7, 111.7, 109.3, 95.8, 93.8, 38.1, 14.5.</div></div><div id="sec5_4_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Ethyl 1-[3-(5-cyano-3-pyridyl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]-3-(trifluoromethyl)pyrazole-4-carboxylate (<b>15o</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 538.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.25 (s, 1 H), 8.98–9.08 (m, 2 H), 8.78 (s, 1 H), 8.62 (d, 1 H), 8.16–8.26 (m, 1 H), 6.53 (dd, 1 H), 5.77 (dd, 1 H), 4.29 (q, 2 H), 3.27 (q, 2 H), 1.20–1.39 (m, 6 H).</div></div><div id="sec5_4_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 5-[8-(Ethylamino)-6-fluoro-4-[5-methyl-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15s</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 480.2. HRMS calcd [(M+H)<sup>+</sup>] 480.1560, measured [(M+H)<sup>+</sup>] 480.1561. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.219 (s, 1H), 8.992–8.996 (d, 1H), 8.832 (s, 1H), 8.625 (s, 1H), 8.147–8.152 (d, 1H), 6.911 (s, 1H), 6.490–6.526 (m, 1H), 5.549–5.577 (m, 1H), 3.236–3.254 (m, 2H), 1.882 (s, 3H), 1.301–1.336 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 160.5, 158.2, 153.1, 153.0, 151.7, 148.5, 143.7, 143.2, 140.2, 137.0, 136.9, 136.5, 131.5, 125.2, 117.0, 117.0, 116.9, 112.7, 112.7, 109.5, 105.6, 95.7, 92.6, 38.1, 14.5, 10.9.</div></div><div id="sec5_4_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 5-[8-(Ethylamino)-6-fluoro-4-[5-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15t</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 496.1. HRMS calcd [(M+H)<sup>+</sup>] 496.1509, measured [(M+H)<sup>+</sup>] 496.1509. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 13.534 (s, 1 H), 11.818 (s, 1 H), 8.890–8.925 (d, 1 H), 8.454 (s, 1 H), 8.239 (s, 1 H), 7.075–7.093 (d, 1 H), 6.488–6.519 (d, 1 H), 6.208 (s, 1 H), 5.826 (s, 1 H), 5.142 (s, 2 H), 3.264–3.403 (m, 2 H), 1.305–1.341(t, 3 H).</div></div><div id="sec5_4_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 5-[8-(Ethylamino)-6-fluoro-4-[3-(trifluoromethyl)-5,6-dihydro-4H-pyrrolo[3,4-<i>c</i>]pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15u</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 507.1. HRMS calcd [(M+H)<sup>+</sup>] 507.1669, measured [(M+H)<sup>+</sup>] 507.1668. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.30 (s, 1 H), 10.22 (br. s., 2 H), 9.05 (d, 1 H), 8.78 (s, 1 H), 8.56 (d, 1 H), 8.19 (s, 1 H), 6.54 (dd, 1 H), 6.11 (dd, 1 H), 4.55–4.64 (m, 1 H), 4.43–4.51 (m, 1 H), 4.22 (d, 1 H), 3.84 (d, 1 H), 3.27 (q, 2 H), 1.33 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 160.5, 158.9, 158.2, 153.1, 151.9, 148.6, 147.9, 140.2, 136.9, 136.0, 131.3, 125.3, 122.1, 121.8, 120.3, 117.1, 116.7, 116.6, 111.2, 109.7, 95.8, 93.8, 44.2, 38.1, 14.5.</div></div><div id="sec5_4_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 5-[4-[3-(Difluoromethyl)-5,6-dihydro-4H-pyrrolo[3,4-<i>c</i>]pyrazol-1-yl]-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15v</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 507.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.30 (s, 1H), 10.22 (br. s, 2H), 9.05 (d, 1H), 8.78 (s, 1H), 8.56 (d, 1H), 8.19 (s, 1H), 6.56 (m, 1H), 6.10 (m, 1H), 4.45–4.61 (m, 2H), 4.22 (d, 1H), 3.84 (d, 1H), 3.28 (q, 2H), 1.33 (t, 3H).</div></div><div id="sec5_4_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 5-[4-(5,6-Dihydro-4H-pyrrolo[3,4-<i>c</i>]pyrazol-2-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15w</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 439.3. HRMS calcd [(M+H)<sup>+</sup>] 439.1795, measured [(M+H)<sup>+</sup>] 439.1795. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.19 (s, 1 H), 10.16 (br. s., 2 H), 8.96 (d, 1 H), 8.65–8.71 (m, 1 H), 8.45–8.54 (m, 1 H), 8.16–8.29 (m, 1 H), 7.93 (s, 1 H), 6.46 (dd, 1 H), 6.07 (dd, 1 H), 4.45 (d, 4 H), 3.24 (q, 2 H), 1.31 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 158.6, 157.9, 155.7, 153.3, 153.0, 151.4, 148.6, 140.2, 139.1, 132.5, 128.0, 125.0, 120.9, 120.4, 117.2, 109.5, 95.7, 95.4, 44.4, 43.8, 38.1, 14.5.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="sec5_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 5-[4-(5,6-Dihydro-4H-pyrrolo[3,4-<i>c</i>]pyrazol-1-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15x</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 439.3. HRMS calcd [(M+H)<sup>+</sup>] 439.1795, measured [(M+H)<sup>+</sup>] 439.1792. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.26 (s, 1 H), 10.18 (br. s., 2 H), 9.03 (d, 1 H), 8.78 (s, 1 H), 8.55 (d, 1 H), 8.18 (s, 1 H), 7.89 (s, 1 H), 6.54 (dd, 1 H), 6.14 (dd, 1 H), 4.28–4.55 (m, 2 H), 4.15 (d, 1 H), 3.69 (d, 1 H), 3.30 (q, 2 H), 1.36 (t, 3 H).</div></div><div id="sec5_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> N-Ethyl-6-fluoro-3-(2-methoxypyrimidin-5-yl)-4-[3-(trifluoromethyl)-5,6-dihydro-4H-pyrrolo[3,4-<i>c</i>]pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>17u</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 513.1. HRMS calcd [(M+H)<sup>+</sup>] 513.1774, measured [(M+H)<sup>+</sup>] 513.1776. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.25 (s, 1H), 10.26 (br. s, 2H), 8.73 (s, 1H), 8.43 (s, 2H), 6.53 (m, 1H), 6.04 (m, 1H), 4.62 (d, 1H), 4.52 (d, 1H), 4.24 (d, 1H), 3.94 (d, 1H), 3.27 (q, 2H), 1.32 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 164.8, 159.2, 158.0, 154.9, 153.1, 152.9, 148.3, 136.9, 136.7, 135.8, 133.4, 125.4, 125.1, 123.2, 120.8, 119.0, 117.6, 108.5, 95.7, 93.4, 55.3, 44.7, 44.4, 38.1, 14.5.</div></div><div id="sec5_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 4-(5,6-Dihydro-4H-pyrrolo[3,4-<i>c</i>]pyrazol-1-yl)-N-ethyl-6-fluoro-3-(2-methoxypyrimidin-5-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>17x</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 445.3. HRMS calcd [(M+H)<sup>+</sup>] 445.1901, measured [(M+H)<sup>+</sup>] 445.1902. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.13 (s, 1H), 10.00 (br. s., 2H), 8.70 (s, 1H), 8.38 (s, 2H), 7.84 (s, 1H), 6.51 (m, 1H), 6.02 (m, 1H), 4.35–4.42 (m, 2H), 4.10 (s, 1H), 3.95 (s, 3H) 3.80 (m, 1H), 3.26 (m, 2H), 1.32 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 164.6, 160.2, 159.0, 157.9, 153.0, 151.6, 148.0, 138.4, 136.6, 134.5, 129.1, 124.9, 123.8, 119.1, 117.5, 117.3, 112.4, 95.3, 94.2, 55.2, 44.6, 38.1, 14.6.</div></div><div id="sec5_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Synthetic Procedure for the Synthesis of 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-<i>N</i>-hydroxynicotinamide (<b>10e</b>)</h4><div id="sec5_5_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> Step (a)</h5><div class="NLM_p last">Preparation of methyl 5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinate. A mixture solution of <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (<b>Intermediate 8</b>, 50 mg, 100 μmol), methyl 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)nicotinate (52.8 mg, 201 μmol), and CsF (61 mg, 402 μmol) in 1,4-dioxane (6 mL) was degassed with N<sub>2</sub> several times. This was followed by the addition of dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine (23.9 mg, 50.2 μmol) and Pd<sub>2</sub>dba<sub>3</sub> (23 mg, 25.1 μmol) at 0 °C, and the resulting mixture was heated at 110 °C for 4 h. The mixture was cooled down to room temperature and concentrated <i>in vacuo</i>. The residue was diluted with EtOAc (50 mL) and washed with water (20 mL). The organic layer was then dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product of the title compound (60 mg) as a solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 599.3.</div></div><div id="sec5_5_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl ethyl(6-fluoro-3-(5-(hydroxycarbamoyl)pyridin-3-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)carbamate. A solution of methyl 5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinate (60 mg, 100 μmol) and hydroxylamine (662 mg, 591 μL, 10 mmol) in MeOH (6 mL) was stirred at room temperature overnight. The reaction mixture was concentrated <i>in vacuo</i> to give a crude product of the title compound (60 mg) as a solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 600.3.</div></div><div id="sec5_5_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> Step (c)</h5><div class="NLM_p last">Preparation of 5-(8-(ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-<i>N</i>-hydroxynicotinamide. To a solution of <i>tert</i>-butyl ethyl(6-fluoro-3-(5-(hydroxycarbamoyl)pyridin-3-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)carbamate (60 mg, 100 μmol) in DCM (3 mL) was added TFA (2 mL, 26.3 mmol), and the resulting mixture was stirred at room temperature for 2 h. The mixture solution was concentrated under reduced pressure, and the residue was subject to prep-HPLC purification to afford the title compound (15 mg, 29% yield) as a white solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 500.1. HRMS calcd [(M+H)<sup>+</sup>] 500.1458, measured, 500.1456. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.1–12.2 (m, 1H), 11.3–11.5 (m, 1H), 9.2–9.3 (m, 1H), 8.8–8.9 (m, 1H), 8.7–8.8 (m, 1H), 8.4–8.4 (m, 1H), 8.2–8.3 (m, 1H), 7.9–8.0 (m, 1H), 7.0–7.1 (m,1H), 6.5–6.5 (m, 1H), 5.9–5.9 (m, 1H), 5.7–5.8 (m, 1H), 3.2–3.3 (m, 2H), 1.3–1.3 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i>6) δ ppm 162.5, 160.2, 157.9, 152.9, 151.5, 148.4, 147.1, 138.2, 135.9, 135.4, 131.2, 128.7, 125.1, 123.0, 121.9, 120.3, 117.0, 112.1, 106.3, 95.7, 93.6, 38.1, 14.5.</div></div></div><div id="sec5_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-<i>N</i>-methylpyrimidine-2-carboxamide (<b>12i</b>)</h4><div class="NLM_p last">The title compound was prepared by a similar procedure to that described for compound <b>10e</b>. MS obsd (ESI+) [(M+H)<sup>+</sup>] 499.0. HRMS calcd [(M+H)<sup>+</sup>] 500.1458, measured [(M+H)<sup>+</sup>] 500.1457. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.97 (d, 1H), 8.6–8.6 (m, 1H), 8.0–8.1 (m, 1H), 7.9–8.0 (m, 1H), 6.8–6.9 (m, 1H), 6.4–6.4 (m, 1H), 5.85 (dd, 1H), 3.2–3.2 (m, 2H), 3.1–3.1 (m, 3H), 1.3–1.3 (m, 3H).</div></div><div id="sec5_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Synthetic Procedure for the Synthesis of 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-<i>N</i>-methoxynicotinamide (<b>10f</b>)</h4><div id="sec5_5_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> Step (a)</h5><div class="NLM_p last">Preparation of 5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinic acid. A solution of methyl 5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinate (61 mg, 102 μmol), sodium hydroxide (40.8 mg, 1.02 mmol) in THF (3 mL), and H<sub>2</sub>O (3 mL) was stirred at room temperature for 30 min. Afterward, the mixture solution was concentrated <i>in vacuo</i> and the residue was diluted with EtOAc (50 mL) and washed with water (20 mL). The organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness to afford a crude product of the title compound (61 mg) as a solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 585.4.</div></div><div id="sec5_5_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl ethyl(6-fluoro-3-(5-(methoxycarbamoyl)pyridin-3-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)carbamate. A mixture solution of 5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinic acid (61 mg, 104 μmol), 2-(3H-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-3-yl)-1,1,3,3-tetramethylisouronium hexafluorophosphate(V) (71.4 mg, 188 μmol), O-methylhydroxylamine hydrochloride (26.1 mg, 313 μmol), and N-ethyl-<i>N</i>-isopropylpropan-2-amine (80.9 mg, 626 μmol) in DCM (3 mL) was stirred at room temperature overnight. Afterward, the mixture was diluted with DCM (50 mL) and washed with water (20 mL). The separated organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product of the title compound (61 mg) as a solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 614.2.</div></div><div id="sec5_5_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> Step (c)</h5><div class="NLM_p last">Preparation of 5-(8-(ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-<i>N</i>-methoxynicotinamid. A solution of <i>tert</i>-butyl ethyl(6-fluoro-3-(5-(methoxycarbamoyl)pyridin-3-yl)-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)carbamate (61 mg, 99.4 μmol) and TFA (11.3 mg, 99.4 μmol) in DCM (3 mL) was stirred at room temperature for 2 h. Afterward, the mixture solution was concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (16 mg, 31% yield) as a white solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 514.1. HRMS calcd [(M+H)<sup>+</sup>] 514.1615, measured [(M+H)<sup>+</sup>] 514.1617. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.7–8.8 (m, 1H), 8.5–8.6 (m, 1H), 8.4–8.4 (m, 1H), 7.9–8.0 (m, 1H), 7.8–7.9 (m, 1H), 6.7–6.8 (m, 1H), 6.3–6.4 (m, 1H), 5.8–5.9 (m, 1H), 3.7–3.8 (m, 3H), 1.3–1.3 (m, 3H).</div></div></div><div id="sec5_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> Synthetic Procedure for the Synthesis of 2-(5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetic acid (<b>10n</b>)</h4><div id="sec5_5_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> Step (a)</h5><div class="NLM_p last">Preparation of methyl 2-(5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetate. To a 25 mL microwave vial was added methyl 2-(5-bromopyridin-3-yl)acetate (100 mg, 435 μmol), bis(pinacolato)diboron (121 mg, 478 μmol), 1,1′-bis(diphenylphosphino)ferrocene-palladium(II)dichloride dichloromethane complex (35.5 mg, 43.5 μmol), and potassium acetate (128 mg, 1.3 mmol) in 1,4-dioxane (10 mL). The vial was capped and heated in the microwave reactor at 100 °C for 1 h under N<sub>2</sub>. Afterward, the reaction was cooled to room temperature, and <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (Intermediate <b>8</b>, 216 mg, 435 μmol), 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene palladium dichloride (14.2 mg, 21.7 μmol), cesium fluoride (198 mg, 1.3 mmol), and H<sub>2</sub>O (2 mL) were added. The resulting reaction mixture was reheated to 120 °C and stirred for 15 h under N<sub>2</sub>. The reaction mixture was cooled down to room temperature and filtered through glass fiber paper. The crude material was then purified by flash chromatography (silica gel, 0% to 10% MeOH in DCM) to afford the title compound (150 mg, 56.3% yield) as a solid.</div></div><div id="sec5_5_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> Step (b)</h5><div class="NLM_p last">Preparation of 2-(5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetic acid. To a solution of methyl 2-(5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetate (150 mg, 245 μmol) in MeOH (8 mL) was added LiOH (47.9 mg, 2 mmol) in H<sub>2</sub>O (2 mL), and the resulting mixture was stirred at room temperature for 2 h. After the solution was adjusted to pH ∼ 7, it was concentrated <i>in vacuo</i>. The residue was partitioned between H<sub>2</sub>O (25 mL) and EtOAc (25 mL). The organic layer was then dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to afford a crude product of the title compound (140 mg, 95.5% yield) as a solid. It was used directly in the next step without further purification.</div></div><div id="sec5_5_7_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> Step (c)</h5><div class="NLM_p last">Preparation of 2-(5-(8-(ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetic acid. To a solution of 2-(5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetic acid (140 mg, 234 μmol) in DCM (8 mL) was added TFA (1.33 g, 11.7 mmol). The resulting mixture was stirred at room temperature for 2h before it was concentrated <i>in vacuo</i>. The crude product was then subjected to prep-HPLC purification to afford the title compound (17.2 mg, 14% yield) as a solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 499.3. HRMS calcd [(M+H)<sup>+</sup>] 499.1506, measured [(M+H)<sup>+</sup>] 499.1509. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.0–12.2 (m, 1H), 8.7–8.8 (m, 1H), 8.5–8.6 (m, 1H), 8.3–8.4 (m, 1H), 8.2–8.3 (m, 1H), 7.6–7.7 (m, 1H), 7.0–7.1 (m, 1H), 6.4–6.5 (m, 1H), 5.6–5.8 (m, 1H), 3.6–3.7 (m, 2H), 3.2–3.3 (m, 2H), 1.3–1.4 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 172.0, 159.0, 152.9, 148.2, 146.0, 143.1, 139.6, 138.3, 136.8, 136.0, 131.8, 125.1, 123.0, 121.8, 120.4, 116.9, 112.2, 106.7, 95.6, 93.4, 38.1, 37.7, 26.8, 14.5.</div></div></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div class="NLM_p first">The following compounds were prepared by a similar procedure to that described for compound <b>10n</b>.</div><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 2-(5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)acetic Acid (<b>12e</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 500.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 8.7–8.8 (m, 1H), 8.5–8.6 (m, 2H), 8.3–8.4 (m, 1H), 7.1–7.2 (m, 1H), 6.6–6.7 (m, 1H), 6.5–6.5 (m, 1H), 5.9–5.9 (m, 1H), 5.7–5.8 (m, 1H), 5.3–5.4 (m, 1H), 3.8–3.9 (m, 2H), 3.3–3.3 (m, 2H), 1.3–1.3 (m, 3H).</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidine-2-carboxylic Acid (<b>12g</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 486.1. HRMS calcd [(M+H)<sup>+</sup>] 486.1302, measured [(M+H)<sup>+</sup>] 486.1302. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i>4) δ ppm 8.6–8.6 (m, 1H), 8.54 (s, 2H), 7.9–7.9 (m, 1H), 6.8–6.8 (m, 1H), 6.4–6.4 (m, 1H), 5.8–5.8 (m, 1H), 3.2–3.2 (m, 2H), 1.3–1.3 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d6</i>) δ ppm 216.9, 160.1, 158.2, 156.5, 153.1, 148.2, 143.6, 143.3, 138.2, 136.9, 136.8, 136.1, 125.2, 122.7, 120.6, 119.3, 116.9, 116.8, 112.2, 112.2, 106.8, 95.5, 93.4, 40.6, 40.4, 40.2, 38.1, 14.5.</div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 1-(5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)cyclopropanecarboxylic Acid (<b>12f</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 526.1. HRMS calcd [(M+H)<sup>+</sup>] 526.1615, measured [(M+H)<sup>+</sup>] 526.1617. <sup>1</sup>H NMR (MeOH-<i>d4</i>, 400 MHz) δ ppm 8.68 (s, 1H), 8.58 (s, 2H), 8.1–8.1 (m, 1H), 6.99 (d, 1H), 6.52 (dd, 1H), 5.98 (dd, 1H), 3.3–3.4 (m, 2H), 1.8–1.9 (m, 2H), 1.7–1.8 (m, 2H), 1.43 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 173.3, 166.5, 160.3, 158.0, 156.4, 153.1, 148.2, 143.7, 143.3, 138.3, 136.8, 136.7, 136.1, 127.1, 125.1, 123.0, 120.3, 119.2, 116.9, 116.8, 112.1, 112.0, 106.8, 95.6, 93.4, 38.1, 31.2, 18.1, 14.5.</div></div><div id="sec5_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 3-((5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)oxy)-2,2-dimethylpropanoic Acid (<b>12o</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 558.1. HRMS calcd [(M+H)<sup>+</sup>] 558.1877, measured [(M+H)<sup>+</sup>] 558.1878. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.63 (s, 1H), 8.39 (s, 2H), 8.05 (d, 1H), 6.98 (d, 1H), 6.50 (dd, 1H), 5.91 (dd, 1H), 4.43 (s, 2H), 3.3–3.4 (m, 1H), 3.29 (m, 2H), 1.43 (t, 3H), 1.34 (s, 6H).</div></div><div id="sec5_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 1-(((5-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyrimidin-2-yl)amino)methyl)cyclopropanecarboxylic Acid (<b>12w</b>)</h4><div class="NLM_p last">MS obsd (ESI+) [(M+H)<sup>+</sup>] 555.1. HRMS calcd [(M+H)<sup>+</sup>] 557.2037, measured [(M+H)<sup>+</sup>] 557.2037. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 8.57 (s, 1H), 8.09 (s, 2H), 8.02 (d, 1H), 6.98 (d, 1H), 6.49 (dd, 1H), 5.87 (dd, 1H), 3.66 (s, 2H), 3.3–3.3 (m, 2H), 1.42 (t, 3H), 1.2–1.2 (m, 2H), 1.0–1.0 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 178.5, 162.2, 157.4, 152.4, 147.9, 143.3, 137.8, 136.7, 135.9, 125.0, 123.1, 120.8, 117.6, 116.9, 112.3, 106.6, 95.3, 93.4, 48.6, 43.4, 38.1, 23.4, 14.5.</div></div><div id="sec5_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 1-[5-[6-Fluoro-8-(methylamino)-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyrimidin-2-yl]cyclopropanecarboxylic Acid (<b>12x</b>)</h4><div class="NLM_p last">HRMS calcd [(M+H)<sup>+</sup>] 512.1458, measured [(M+H)<sup>+</sup>] 512.1460. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 8.76 (s, 1H), 8.53 (s, 2H), 8.38 (br d, <i>J</i> = 1.0 Hz, 1H), 7.16 (d, <i>J</i> = 2.2 Hz, 1H), 6.47 (dd, <i>J</i> = 2.1, 12.1 Hz, 1H), 6.08 (br d, <i>J</i> = 3.7 Hz, 1H), 5.92–5.61 (m, 1H), 2.92 (br d, <i>J</i> = 3.9 Hz, 3H), 1.50–1.62 (m, 2H), 1.34–1.47 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 173.4, 166.6, 160.4, 158.1, 156.4, 153.1, 148.2, 143.5, 138.2, 137.8, 136.1, 127.1, 125.2, 119.2, 121.6, 116.7, 112.0, 106.8, 95.3, 93.4, 31.3, 30.2, 25.4, 18.0.</div></div><div id="sec5_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Synthetic Procedure for the Synthesis of 2-[5-[8-(Ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetamide (<b>10o</b>)</h4><div id="sec5_6_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> Step (a)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl (3-(5-(2-amino-2-oxoethyl)pyridin-3-yl)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate. To a 25 mL microwave vial was added 2-(5-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)pyridin-3-yl)acetic acid (128 mg, 214 μmol) and CDI (46.8 mg, 289 μmol) in THF (8 mL). The vial was capped and heated in the microwave reactor at 60 °C for 2 h. Afterward, the reaction mixture was cooled down to room temperature, and ammonia in isopropyl alcohol (10.7 mL, 10.7 mmol) was added. The resulting mixture was then stirred at room temperature overnight before it was concentrated <i>in vacuo</i>. The residue was suspended in H<sub>2</sub>O (25 mL) and extracted with EtOAc (25 mL × 3). Combined organics were washed with brine (25 mL), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to afford a crude product of the title compound (120 mg, 93.9% yield) as a solid.</div></div><div id="sec5_6_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> Step (b)</h5><div class="NLM_p last">Preparation of 2-[5-[8-(ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetamide. To a solution of <i>tert</i>-butyl (3-(5-(2-amino-2-oxoethyl)pyridin-3-yl)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate (120 mg, 201 μmol) in DCM (10 mL) was added TFA (1.14 g, 10 mmol), and the resulting reaction mixture was stirred at room temperature for 2 h. Afterward, the solution was concentrated <i>in vacuo</i> to give a crude product, which was purified by pHPLC to afford the title compound (22 mg, 19.5% yield) as a white solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 498.4. HRMS calcd [(M+H)<sup>+</sup>] 498.1665, measured [(M+H)<sup>+</sup>] 498.1665. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.0–12.2 (m, 1H), 8.6–8.7 (m, 1H), 8.4–8.5 (m, 1H), 8.2–8.3 (m, 2H), 7.4–7.6 (m, 2H), 7.05 (d, 1H), 6.9–7.0 (m, 1H), 6.48 (dd, 1H), 5.88 (brs, 1H), 5.70 (dd, 1H), 3.38 (s, 2H), 3.2–3.3 (m, 2H), 1.3–1.4 (m, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 171.6, 160.2, 157.9, 152.8, 149.6, 148.3, 147.4, 143.3, 138.2, 137.2, 136.7, 135.9, 125.1, 123.1, 122.6, 120.4, 116.9, 112.2, 106.5, 95.5, 93.5, 38.1, 27.3, 14.5.</div></div></div><div id="sec5_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> Synthetic Procedure for the Synthesis of N-Ethyl-6-fluoro-3-[5-(1H-tetrazol-5-ylmethyl)-3-pyridyl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10p</b>)</h4><div id="sec5_6_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> Step (a)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[3-[5-(cyanomethyl)-3-pyridyl]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. To a solution of <i>tert</i>-butyl N-(3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (250 mg, 0.502 mmol) and [5-(cyanomethyl)-3-pyridyl]boronic acid (110 mg, 0.679 mmol) in THF/H<sub>2</sub>O (10 mL/1 mL) was added Pd-Ad<sub>2</sub>nBuP Biphenyl Precat (40 mg, 0.06 mmol) and K<sub>3</sub>PO<sub>4</sub> (280 mg, 2.63 mmol); then the solution was stirred at 80 °C under Ar for 18 h. Then the mixture was cooled down to room temperature and concentrated <i>in vacuo</i>. The crude product was purified by prep-TLC (petroleum ether:EtOAc = 1:2) to afford the title compound (260 mg, 89.3% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 580.1.</div></div><div id="sec5_6_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-ethyl-N-[6-fluoro-3-[5-(1H-tetrazol-5-ylmethyl)-3-pyridyl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]carbamate. To a solution of <i>tert</i>-butyl N-[3-[5-(cyanomethyl)-3-pyridyl]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (50 mg, 0.086 mmol) in DMF (3 mL) was added NaN<sub>3</sub> (10 mg, 0.154 mmol) and NH<sub>4</sub>Cl (20 mg, 0.374 mmol); then the resulting mixture was stirred at 100 °C for 72 h under N<sub>2</sub>. The reaction mixture was cooled down to room temperature, poured into water (50 mL), and extracted with EtOAc (40 mL × 3). Combined organics were washed with brine (40 mL × 3), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude of the title compound (70 mg) as yellow gum. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 623.2.</div></div><div id="sec5_6_8_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> Step (c)</h5><div class="NLM_p last">Preparation of 3-(5-((1H-tetrazol-5-yl)methyl)pyridin-3-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine formate. To a solution of <i>tert</i>-butyl N-ethyl-N-[6-fluoro-3-[5-(1H-tetrazol-5-ylmethyl)-3-pyridyl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]carbamate (70 mg, 0.086 mmol) in DCM (2 mL) was added TFA (0.5 mL); then the solution was stirred at 20 °C for 3 h. The reaction mixture was concentrated <i>in vacuo</i> to give a crude material, which was purified by prep-HPLC to afford the title compound (30.5 mg, 61.6% yield) as a light-yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 523.2. HRMS calcd [(M+H)<sup>+</sup>] 523.1730, measured [(M+H)<sup>+</sup>] 523.1730. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.10 (brs, 1H), 8.66 (s, 1H), 8.48 (s, 1H), 8.23–8.20 (m, 2H), 8.15 (s, 0.25H), 7.52 (s, 1H), 7.00 (s, 1H), 6.49–6.45 (dd, 1H), 5.87 (s, 1H), 5.70 (m, 1H), 4.21 (s, 2H), 3.26–3.23 (q, 2H), 1.32–1.29 (t, 2H).</div></div></div><div id="sec5_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> Synthetic Procedure for the Synthesis of 5-[[5-[8-(Ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]methyl]-3H-1,3,4-oxadiazol-2-one (<b>10r</b>)</h4><div id="sec5_6_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> Step (a)</h5><div class="NLM_p last">Preparation of ethyl 2-[5-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetate. To a mixture of <i>tert</i>-butyl N-(3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (100 mg, 0.2 mmol), ethyl 2-[5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-3-pyridyl]acetate (83.56 mg, 0.4 mmol), K<sub>3</sub>PO<sub>4</sub> (127.91 mg, 0.6 mmol), and Catacxium A-PD-G2 (13.43 mg, 0.02 mmol) was added THF (2 mL) and H<sub>2</sub>O (0.2 mL); then the solution was stirred at 80 °C under Ar for 16 h. After the reaction was completed by LC-MS detection, the crude product was purified by prep-TLC (petroleum ether:EtOAc = 2:1) to afford the title compound (100 mg, 79.5% yield) as a brown solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 627.2.</div></div><div id="sec5_6_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> Step (b)</h5><div class="NLM_p last">Preparation of 2-[5-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetic acid. To a solution of ethyl 2-[5-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetate (100 mg, 0.16 mmol) in EtOH (5 mL) was added aqueous NaOH solution (1 N, 1 mL, 1 mmol); then the mixture was stirred at 20 °C for 1 h. After LC-MS indicated the reaction was completed, the mixture was adjusted to pH = 4–5 with aqueous HCl solution (1 N) and extracted with EtOAc (20 mL × 3). Combined organics were then dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product of the title compound (90 mg, 94.2% yield) as a yellow solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 599.0.</div></div><div id="sec5_6_9_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> Step (c)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[3-[5-[2-(2-tert-butoxycarbonylhydrazino)-2-oxo-ethyl]-3-pyridyl]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. A solution of 2-[5-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetic acid (90 mg, 0.15 mmol), HATU (68.61 mg, 0.18 mmol), and DIPEA (0.08 mL, 0.45 mmol) in DMF (3 mL) was stirred at 20 °C for 0.5 h; then <i>tert</i>-butyl carbazate (23.85 mg, 0.18 mmol) was added, and the resulting mixture was stirred at 20 °C for an additional 16 h. EtOAc (30 mL) was added, and the mixture solution was washed with brine (20 mL) and saturated aqueous NaHCO<sub>3</sub> solution (20 mL). The separated organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product of the title compound (100 mg, 93.3% yield) as a yellow solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 713.2.</div></div><div id="sec5_6_9_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> Step (d)</h5><div class="NLM_p last">Preparation of 2-[5-[8-(ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetohydrazide. To a solution of <i>tert</i>-butyl N-[3-[5-[2-(2-<i>tert</i>-butoxycarbonylhydrazino)-2-oxo-ethyl]-3-pyridyl]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (100 mg, 0.14 mmol) in EtOAc (5 mL) was added HCl/EtOAc (4 N, 1 mL, 4 mmol); then the mixture was stirred at 20 °C for 1 h until LC-MS indicated the completion of the reaction. The reaction mixture was then concentrated <i>in vacuo</i> to give a crude product of the title compound (100 mg, 65.2% yield) as a yellow solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 513.1.</div></div><div id="sec5_6_9_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> Step (e)</h5><div class="NLM_p last">Preparation of 5-[[5-[8-(ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]methyl]-3H-1,3,4-oxadiazol-2-one. To a solution of 2-[5-[8-(ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-3-pyridyl]acetohydrazide (100 mg, 0.2 mmol) in THF (5 mL) was added DIPEA (0.1 mL, 0.590 mmol), and the mixture was stirred at 20 °C for 0.5 h. Then CDI (63 mg, 0.390 mmol) was added, and the resulting mixture was stirred at 20 °C for an additional 16 h. EtOAc (20 mL) was added, and the mixture was washed with brine (20 mL) and saturated aqueous NH<sub>4</sub>Cl solution (20 mL × 2). The organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (19.2 mg, 18.3% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 539.1. HRMS calcd [(M+H)<sup>+</sup>] 539.1567, measured [(M+H)<sup>+</sup>] 539.1566. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.22 (brs, 1H) 8.71 (s, 1H) 8.49 (d, 1H) 8.44 (brs, 1H) 8.36 (d, 1H) 8.25 (d, 1H) 7.48 (s, 1H) 7.02 (d, 1H) 6.48 (dd, 1H) 5.96 (brs, 1H) 5.71 (dd, 1H) 3.93 (s, 2H) 3.18–3.28 (m, 2 H) 1.32 (t, 3 H).</div></div></div><div id="sec5_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> Synthetic Procedure for the Synthesis of N-Ethyl-6-fluoro-3-[5-(1H-imidazol-2-ylmethyl)-3-pyridyl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>10s</b>)</h4><div id="sec5_6_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> Step (a)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl (3-(5-(cyanomethyl)pyridin-3-yl)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate. To a solution of <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (250 mg, 0.5 mmol) and (5-(cyanomethyl)pyridin-3-yl)boronic acid (110 mg, 0.68 mmol) in THF/H<sub>2</sub>O (10 mL/1 mL) was added cataCXium A Pd G2 (40 mg, 0.06 mmol) and K<sub>3</sub>PO<sub>4</sub> (280 mg, 2.63 mmol). The resulting mixture was stirred at 80 °C for 18 h under Ar. Afterward, the reaction was cooled down to room temperature and the solution was concentrated <i>in vacuo</i>. The crude product was purified by prep-TLC (petroleum ether:EtOAc = 1:2) to afford the title compound (260 mg, 89.3% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 580.1.</div></div><div id="sec5_6_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl (3-(5-(2-amino-2-iminoethyl)pyridin-3-yl)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate. To a solution of <i>tert</i>-butyl (3-(5-(cyanomethyl)pyridin-3-yl)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate (80 mg, 0.138 mmol) in MeOH (5 mL) was added NaOMe (10 mg, 0.185 mmol). After the reaction mixture was stirred at 20 °C for 120 h under N<sub>2</sub>, NH<sub>4</sub>Cl (30 mg, 0.56 mmol) was added and then the resulting solution was stirred at 20 °C for an additional 20 h. The reaction mixture was concentrated <i>in vacuo</i> to give a crude product, which was redissolved in EtOAc (50 mL) and filtered. The filtrate was concentrated <i>in vacuo</i> to give a crude product of the title compound (76 mg) as a white solid. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 597.2.</div></div><div id="sec5_6_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> Step (c)</h5><div class="NLM_p last">Preparation of 3-(5-((1H-imidazol-2-yl)methyl)pyridin-3-yl)-N-ethyl-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine. To a solution of <i>tert</i>-butyl (3-(5-(2-amino-2-iminoethyl)pyridin-3-yl)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate (76 mg, 0.051 mmol) in MeOH (5 mL) was added 2,2-dimethoxyethanamine (15 mg, 0.143 mmol). After being stirred at 70 °C for 18 h, the reaction mixture was concentrated <i>in vacuo</i> to give a crude solid, which was dissolved in aqueous oxalic acid solution (1 N, 5 mL), and the resulting mixture was stirred at 80 °C for another 3 h. The reaction mixture was cooled back to room temperature, diluted with H<sub>2</sub>O (30 mL), and basified with saturated aqueous Na<sub>2</sub>CO<sub>3</sub> solution to pH ∼ 9. The aqueous solution was extracted with EtOAc (30 mL) three times, and the combined organics were dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude product was then purified by prep-HPLC to afford the title compound (13.2 mg, 49.8% yield) as a light-yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 521.1. HRMS calcd [(M+H)<sup>+</sup>] 521.1825, measured [(M+H)<sup>+</sup>] 521.1828. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.20 (brs, 1H), 8.65 (s, 1H), 8.41 (s, 1H), 8.30 (s, 1H), 8.26 (s, 0.23H), 8.19 (m, 2H), 7.50 (s, 1H), 7.02 (s, 1H), 6.89 (s, 2H), 6.48–6.45 (d, 1H), 5.95 (s, 1H), 5.69–5.67 (m, 1H), 3.95 (s, 2H), 3.27–3.22 (q, 2H), 1.32–1.29 (t, 2H).</div></div></div><div id="sec5_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Synthetic Procedure for the Synthesis of 6-(8-(Ethylamino)-6-fluoro-4-(3-(trifluoromethyl)-1H-pyrazol-1-yl)-9H-pyrido[2,3-<i>b</i>]indol-3-yl)-1-methyl-4-oxo-1,4-dihydro-1,8-naphthyridine-3-carboxylic Acid (<b>10u</b>)</h4><div id="sec5_6_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> Step (a)</h5><div class="NLM_p last">Preparation of ethyl 6-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-1-methyl-4-oxo-1,8-naphthyridine-3-carboxylate. To a solution of <i>tert</i>-butyl N-(3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (80 mg, 0.16 mmol), ethyl 1-methyl-4-oxo-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1,8-naphthyridine-3-carboxylate (132.5 mg, 0.48 mmol), and K<sub>3</sub>PO<sub>4</sub> (135.8 mg, 0.64 mmol) in THF/H<sub>2</sub>O (2.5 mL, v/v = 10:1) was added Ad<sub>2</sub>nBuP Biphenyl (21.4 mg, 0.032 mmol) under N<sub>2</sub> at 0 °C. The resulting reaction solution was degassed with N<sub>2</sub> several times before it was stirred at 70 °C for 12 h. After LC-MS showed the reaction was completed, the mixture was poured into water (50 mL) and extracted with EtOAc (50 mL × 3). Combined organics were dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (DCM:EtOH = 30:1) to afford the title compound (72.5 mg, 67.8% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 694.3.</div></div><div id="sec5_6_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> Step (b)</h5><div class="NLM_p last">Preparation of 6-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-1-methyl-4-oxo-1,8-naphthyridine-3-carboxylic acid. To a solution of ethyl 6-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-1-methyl-4-oxo-1,8-naphthyridine-3-carboxylate (62.5 mg, 0.091 mmol) in THF (3 mL) was added NaOH (36.2 mg, 0.91 mmol), and the resulting mixture was stirred at 30 °C for 1 h. Afterward, the reaction mixture was poured into water (5 mL), pH was adjusted to 7, and the reaction mixture was extracted with EtOAc (10 mL). The organic layer was washed with aqueous CaCl<sub>2</sub> solution (1 N, 100 mL) twice and brine (100 mL) twice. The organic layer was dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product of the title compound (59.3 mg) as a yellow oil. It was used directly in the next step without further purification. MS obsd (ESI+) [(M+H)<sup>+</sup>] 666.2 ([M + H]<sup>+</sup>).</div></div><div id="sec5_6_11_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> Step (c)</h5><div class="NLM_p last">Preparation of 6-[8-(ethylamino)-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-1-methyl-4-oxo-1,8-naphthyridine-3-carboxylic acid. To a solution of 6-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-6-fluoro-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]-1-methyl-4-oxo-1,8-naphthyridine-3-carboxylic acid (49.3 mg, 0.074 mmol) in DCM (2 mL) was added TFA (0.4 mL), and the resulting mixture was stirred at 20 °C for 1 h. Afterward, the mixture was concentrated to give a crude product, which was purified by prep-HPLC to afford the title compound (4.5 mg, 10.7% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 566.1. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d4</i>) δ ppm 9.198 (s, 1 H), 8.900 (s, 1 H), 8.840 (s, 1 H), 8.367 (s, 1 H), 8.316 (s, 1 H), 7.086 (s, 1 H), 6.469–6.499 (d, 1 H), 5.731–5.752 (d, 1 H), 4.0859 (s, 3 H), 3.252 (m, 2 H), 1.297–1.331 (m, 3 H).</div></div></div><div id="sec5_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> Synthetic Procedure for the Synthesis of N-Ethyl-6-fluoro-3-[5-(1H-imidazol-2-ylmethyl)-3-pyridyl]-4-[3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>15b</b>)</h4><div id="sec5_6_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> Step (a)</h5><div class="NLM_p last">Preparation of 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]pyrazole-3-carboxylic acid. A mixture solution of methyl 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]pyrazole-3-carboxylate (70 mg, 0.114 mmol) and NaOH (29 mg, 0.719 mmol) in MeOH (5 mL) and H<sub>2</sub>O (2 mL) was stirred at 10 °C for 12 h. Afterward, the mixture was first adjusted to pH = 3 with 15% aqueous HCl solution and then concentrated <i>in vauco</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (66.5 mg, 97.1% yield) as a yellow solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 474.0.</div></div><div id="sec5_6_12_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i123" class="anchor-spacer"></div><h5 class="article-section__title" id="_i123"> Step (b)</h5><div class="NLM_p last">Preparation of 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]pyrazole-3-carboxylic acid. A solution of 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]pyrazole-3-carboxylic acid (30 mg, 0.063 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine-3-carbonitrile (29.2 mg, 0.127 mmol) and CsF (28.7 mg, 0.189 mmol) in 1,4-dioxane/H<sub>2</sub>O (4 mL/0.4 mL) was first degassed with N<sub>2</sub> several times at 20 °C, and then Pd<sub>2</sub>(dba)<sub>3</sub>(29.3 mg, 0.032 mmol) and Xantphos (15.2 mg, 0.032 mmol) were added to the mixture. After the addition, the resulting reaction solution was again degassed with N<sub>2</sub> for five times at 20 °C before it was stirred at 110 °C for 12 h. The reaction mixture was cooled down to room temperature and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (petroleum ether:EtOAc = 2:1) to afford the title compound (26.3 mg, 76.6% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 542.1.</div></div><div id="sec5_6_12_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i124" class="anchor-spacer"></div><h5 class="article-section__title" id="_i124"> Step (c)</h5><div class="NLM_p last">Preparation of 1-(3-(5-cyanopyridin-3-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-1H-pyrazole-3-carboxylic acid. To a solution of 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]pyrazole-3-carboxylic acid (45 mg, 0.083 mmol) in DCM (2 mL) was added TFA (1 mL), and the resulting reaction mixture was stirred at 20 °C for 30 min. Afterward, the mixture was concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (14 mg, 38.1% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 442.1. HRMS calcd [(M+H)<sup>+</sup>] 442.1428, measured [(M+H)<sup>+</sup>] 442.1428. <sup>1</sup>H NMR (DMSO 400 MHz) δ ppm 12.142 (br, 1H), 8.971 (s, 1H), 8.755 (s, 1H), 8.545–8.550 (d, 1H), 8.150–8.188 (m, 2H), 7.016–7.022 (m, 1H), 6.476–6.511 (m, 1H), 5.887 (m, 1H), 5.808–5.831 (m, 1H), 3.243–3.261 (m, 2H), 1.304–1.340 (m, 3H).</div></div></div><div id="sec5_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> Synthetic Procedure for the Synthesis of 5-[4-[3-(Aminomethyl)pyrazol-1-yl]-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15d</b>)</h4><div id="sec5_6_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> Step (a)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[3-chloro-4-(3-cyanopyrazol-1-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. To a solution of <i>tert</i>-butyl N-(3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (140 mg, 0.352 mmol) and K<sub>2</sub>CO<sub>3</sub> (145.0 mg, 1.05 mmol) in DMSO (2 mL) was added 1H-pyrazole-3-carbonitrile (98.2 mg, 1.05 mmol), and the resulting reaction mixture was stirred at 120 °C for 12 h. The mixture was cooled down to room temperature and extracted between water (50 mL) and EtOAc (100 mL) twice. Combined organics were washed with brine (50 mL × 2), dried with anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (petroleum ether: EtOAc = 2:1) to afford the title compound (140 mg, 87.6% yield) as a yellow solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 455.1, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 457.1.</div></div><div id="sec5_6_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i127" class="anchor-spacer"></div><h5 class="article-section__title" id="_i127"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[4-[3-(aminomethyl)pyrazol-1-yl]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. To a solution of <i>tert</i>-butyl N-[3-chloro-4-(3-cyanopyrazol-1-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (140 mg, 0.308 mmol) in EtOH (6 mL) was added NaBH<sub>4</sub> (58.3 mg, 1.54 mmol) and CoCl<sub>2</sub> (80 mg, 0.616 mmol) at 0 °C, and then the reaction mixture was first stirred at 0 °C for 1 h and then warmed up to room temperature and stirred for additional 1 h. After LC-MS indicated the completion of the reaction, the mixture was poured into water (50 mL) and extracted with EtOA (60 mL) three times. Combined organics were washed with brine (50 mL), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (DCM:MeOH = 10:1) to afford the title compound (80 mg, 56.4% yield) as a yellow solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 459.2, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 461.3.</div></div><div id="sec5_6_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i128" class="anchor-spacer"></div><h5 class="article-section__title" id="_i128"> Step (c)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[4-[3-(aminomethyl)pyrazol-1-yl]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. A solution of <i>tert</i>-butyl N-[4-[3-(aminomethyl)pyrazol-1-yl]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (80 mg, 0.174 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine-3-carbonitrile (80.2 mg, 0.349 mmol) and CsF (106 mg, 0.698 mmol) in 1,4-dioxane/H<sub>2</sub>O (3 mL, v/v = 10:1) was first degassed with N<sub>2</sub> several times, and then Pd<sub>2</sub>(dba)<sub>3</sub> (16 mg, 0.0174 mmol) and Xantphos (16.6 mg, 0.0349 mmol) were added to the mixture. After the addition, the resulting reaction solution was again degassed with N<sub>2</sub> five times at 20 °C before it was stirred at 110 °C for 12 h. The reaction mixture was cooled down to room temperature and extracted between water (40 mL) and EtOAc (80 mL) four times. Combined organics were washed with brine (50 mL), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was further purified by prep-TLC (DCM:MeOH = 10:1) to afford <i>the</i> title compound (68 mg, 74.1% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 527.3.</div></div><div id="sec5_6_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> Step (d)</h5><div class="NLM_p last">Preparation of 5-[4-[3-(aminomethyl)pyrazol-1-yl]-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile. To a solution of <i>tert</i>-butyl N-[4-[3-(aminomethyl)pyrazol-1-yl]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (68 mg, 0.129 mmol) in DCM (3 mL) was added TFA (1.5 mL), and the resulting reaction mixture was stirred at room temperature for 1 h. Afterward, the mixture was concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (37 mg, 67.2% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 427.2. HRMS calcd [(M+H)<sup>+</sup>] 427.1795, measured [(M+H)<sup>+</sup>] 427.1794. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.13 (s, 1 H), 8.97 (d, 1 H), 8.72 (s, 1 H), 8.53 (d, 1 H), 8.32 (brs, 3 H), 8.18 (t, 1 H), 8.09 (d, 1 H), 6.69 (d, 1 H), 6.49 (dd, 1 H), 6.00 (dd, 1 H), 4.14 (q, 2 H), 3.25 (q, 2 H), 1.32 (t, 3 H).</div></div></div><div id="sec5_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> Synthetic Procedure for the Synthesis of 1-[3-(5-Cyano-3-pyridyl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]-3-(trifluoromethyl)pyrazole-4-carboxamide (<b>15p</b>)</h4><div id="sec5_6_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i131" class="anchor-spacer"></div><h5 class="article-section__title" id="_i131"> Step (a)</h5><div class="NLM_p last">Preparation of 1-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxylic acid. To a solution of <i>tert</i>-butyl (3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate (440 mg, 1.1 mmol) and K<sub>2</sub>CO<sub>3</sub> (610 mg, 4.42 mmol) in DMSO (6 mL) was added ethyl 3-(trifluoromethyl)-1H-pyrazole-4-carboxylate (920 mg, 4.42 mmol), and the resulting reaction mixture was stirred at 120 °C for 12 h. After the mixture was cooled down to room temperature, it was poured into water and extracted by EtOAc. The organic layer was washed with brine, dried with anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified first by prep-TLC and then by silica gel flash chromatography to afford the title compound (200 mg, 33.4% yield) as a solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 542.2, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 544.2.</div></div><div id="sec5_6_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i132" class="anchor-spacer"></div><h5 class="article-section__title" id="_i132"> Step (b)</h5><div class="NLM_p last">Preparation of 1-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-3-(5-cyanopyridin-3-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxylic acid. A solution of 1-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxylic acid (200 mg, 0.369 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)nicotinonitrile (170 mg, 0.738 mmol), and CsF (224 mg, 1.476 mmol) in 1,4-dioxane/H<sub>2</sub>O (6 mL, v/v = 10:1) was degassed with N<sub>2</sub> several times, and then Pd<sub>2</sub>(dba)<sub>3</sub> (34 mg, 0.037 mmol) and Xantphos (36 mg, 0.074 mmol) were added into the mixture at 0 °C under N<sub>2</sub>. The resulting reaction solution was degassed again with N<sub>2</sub> for five times at 20 °C before it was stirred at 110 °C for 12 h. The reaction mixture was cooled down to room temperature and extracted between water and EtOAc. The organic layer was washed with brine, dried with anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give a crude product, which was purified by silica gel flash chromatography to afford the title compound (200 mg, 88.9% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 610.2.</div></div><div id="sec5_6_14_2b" class="NLM_sec NLM_sec_level_4"><div id="ac_i133" class="anchor-spacer"></div><h5 class="article-section__title" id="_i133"> Step (c)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl (4-(4-carbamoyl-3-(trifluoromethyl)-1H-pyrazol-1-yl)-3-(5-cyanopyridin-3-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate. To a solution of 1-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-3-(5-cyanopyridin-3-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxylic acid (100 mg, 0.16 mmol), NH<sub>4</sub>Cl (43 mg, 0.8 mmol), HOBt (65 mg, 0.48 mmol), and PyBOP (250 mg, 0.48 mmol) in DMF (5 mL) was added DIPEA (124 mg, 0.96 mmol) under N<sub>2</sub>. The reaction mixture was stirred at 25 °C for 1 h before being concentrated <i>in vacuo</i> to give a crude product, which was then purified by silica gel flash chromatography to afford the title compound (90 mg, 90.2% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 609.3.</div></div><div id="sec5_6_14_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i134" class="anchor-spacer"></div><h5 class="article-section__title" id="_i134"> Step (d)</h5><div class="NLM_p last">Preparation of 1-(3-(5-cyanopyridin-3-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxamide. To a solution of <i>tert</i>-butyl (4-(4-carbamoyl-3-(trifluoromethyl)-1H-pyrazol-1-yl)-3-(5-cyanopyridin-3-yl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)(ethyl)carbamate (80 mg, 0.131 mmol) in DCM (3 mL) was added TFA (1.5 mL), and the resulting mixture was stirred at 25 °C for 1 h. The reaction mixture was concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (50 mg, 74.8% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 509.2. HRMS calcd [(M+H)<sup>+</sup>] 509.1461, measured [(M+H)<sup>+</sup>] 509.1459. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm12.23 (s, 1 H) 9.01 (d, 1 H), 8.77 (s, 1 H), 8.56–8.69 (m, 2 H), 8.24 (t, 1 H), 7.77 (brs, 1 H), 7.42 (brs, 1 H), 6.53 (dd, 1 H), 5.79–6.00 (m, 2 H), 3.27 (q, 2 H), 1.33 (t, 3 H).</div></div></div><div id="sec5_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> Synthetic Procedure for the Synthesis of 5-[8-(Ethylamino)-6-fluoro-4-[4-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15q</b>)</h4><div id="sec5_6_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i136" class="anchor-spacer"></div><h5 class="article-section__title" id="_i136"> Step (a)</h5><div class="NLM_p last">Preparation of ethyl 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]-3-(trifluoromethyl)pyrazole-4-carboxylate. To a solution of <i>tert</i>-butyl N-(3,4-dichloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl)-N-ethyl-carbamate (440 mg, 1.1 mmol) and K<sub>2</sub>CO<sub>3</sub> (610 mg, 4.42 mmol) in DMSO (6 mL) was added ethyl 3-(trifluoromethyl)-1H-pyrazole-4-carboxylate (920 mg, 4.42 mmol), and the reaction solution was stirred at 120 °C for 12 h. The mixture was poured into water (50 mL) and extracted by EtOAc (100 mL) twice. Combined organics were washed with brine (50 mL × 2), dried with anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (petroleum ether: EtOAc = 3:1) and then silica gel flash chromatography to afford the title compound (330 mg, 52.4% yield) as a solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 570.1, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 572.2.</div></div><div id="sec5_6_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i137" class="anchor-spacer"></div><h5 class="article-section__title" id="_i137"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[4-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. To a solution of ethyl 1-[8-[<i>tert</i>-butoxycarbonyl(ethyl)amino]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl]-3-(trifluoromethyl)pyrazole-4-carboxylate (180 mg, 0.316 mmol) in THF (0.316 mL) was added LAH (30 mg, 0.790 mmol) at 0 °C under N<sub>2</sub>, and the reaction mixture was stirred at 0 °C for 1 h. The mixture was poured into water (50 mL) and extracted with EtOAc (100 mL × 2). Combined organics were washed with brine (50 mL × 2), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude title compound (140 mg, 84% yield) as a yellow solid. It was used directly in the next step without further purification. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 528.1, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 530.1.</div></div><div id="sec5_6_15_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i138" class="anchor-spacer"></div><h5 class="article-section__title" id="_i138"> Step (c)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[3-(5-cyano-3-pyridyl)-6-fluoro-4-[4-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. A solution of <i>tert</i>-butyl N-[3-chloro-6-fluoro-4-[4-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (130 mg, 0.246 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine-3-carbonitrile (113 mg, 0.493 mmol), and CsF (150 mg, 0.986 mmol) in 1,4-dioxane/H<sub>2</sub>O (4 mL, v/v = 10:1) was degassed with N<sub>2</sub> five times, and then Pd<sub>2</sub>(dba)<sub>3</sub> (22.5 mg, 0.0246 mmol) and Xantphos (23.5 mg, 0.0493 mmol) were added to the mixture at 0 °C under N<sub>2</sub>. After the addition, the resulting reaction solution was degassed again with N<sub>2</sub> five times at 20 °C before it was stirred at 110 °C for 12 h. The reaction mixture was cooled down to room temperature and extracted between water (20 mL) and EtOAc (150 mL × 2). Combined organics were washed with brine (30 mL), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (DCM:MeOH = 30:1) to afford the title compound (120 mg, 81.8% yield) as a pale yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 596.3.</div></div><div id="sec5_6_15_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i139" class="anchor-spacer"></div><h5 class="article-section__title" id="_i139"> Step (d)</h5><div class="NLM_p last">Preparation of 5-[8-(ethylamino)-6-fluoro-4-[4-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile. To a solution of <i>tert</i>-butyl N-[3-(5-cyano-3-pyridyl)-6-fluoro-4-[4-(hydroxymethyl)-3-(trifluoromethyl)pyrazol-1-yl]-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (110 mg, 0.185 mmol) in DCM (3 mL) was added TFA (1.5 mL), and the resulting mixture solution was stirred at 25 °C for 1 h. Afterward, the mixture was concentrated in vacuo to give a crude product, which was purified by prep-HPLC (0.5% TFA in water) to afford the title compound (77 mg, 84.2% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 496.2. HRMS calcd [(M+H)<sup>+</sup>] 496.1509, measured [(M+H)<sup>+</sup>] 496.1509. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6)</i> δ ppm 12.16 (s, 1 H), 8.99 (d, 1 H), 8.74 (s, 1 H), 8.59 (d, 1 H), 8.08–8.24 (m, 2 H), 6.51 (d, 1 H), 5.92 (d, 1 H), 4.51 (s, 4 H), 3.26 (q, 2 H), 1.32 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6)</i> δ ppm 160.3, 158.0, 153.2, 153.0, 151.5, 148.4, 140.3, 140.2, 140.0, 138.4, 136.8, 136.7, 134.7, 131.9, 123.9, 123.3, 120.7, 120.6, 117.1, 117.0, 116.8, 111.9, 109.3, 95.6, 93.8, 53.3, 38.1, 14.5.</div></div></div><div id="sec5_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> Synthetic Procedure for the Synthesis of 5-[4-[4-(Aminomethyl)-3-(trifluoromethyl)pyrazol-1-yl]-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl]pyridine-3-carbonitrile (<b>15r</b>)</h4><div id="sec5_6_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i141" class="anchor-spacer"></div><h5 class="article-section__title" id="_i141"> Step (a)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[4-[4-carbamoyl-3-(trifluoromethyl)pyrazol-1-yl]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. To a solution of 1-(8-((<i>tert</i>-butoxycarbonyl)(ethyl)amino)-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-4-yl)-3-(trifluoromethyl)-1H-pyrazole-4-carboxylic acid (520 mg, 0.960 mmol), NH<sub>4</sub>Cl (257 mg, 4.8 mmol), HOBt (389 mg, 2.88 mmol), and PyBOP (1.5 g, 2.88 mmol) in DMF (12 mL) was added DIPEA (745 mg, 5.76 mmol) under N<sub>2</sub>. The reaction mixture was stirred at 30 °C for 1 h before being concentrated <i>in vacuo</i> to give a crude product, which was purified by silica gel flash chromatography to afford the title compound (500 mg, 96.3% yield) as a yellow solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 541.1, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 543.0.</div></div><div id="sec5_6_16_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i142" class="anchor-spacer"></div><h5 class="article-section__title" id="_i142"> Step (b)</h5><div class="NLM_p last">Preparation of <i>tert-</i>butyl N-[4-[4-(aminomethyl)-3-(trifluoromethyl)pyrazol-1-yl]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. To a solution of BH<sub>3</sub>/THF (1N, 10 mL, 10 mmol) was added <i>tert</i>-butyl N-[4-[4-carbamoyl-3-(trifluoromethyl)pyrazol-1-yl]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (100 mg, 0.185 mmol) at 0 °C under N<sub>2</sub>, and then the resulting reaction mixture was stirred at 80 °C for 0.5 h. Afterward, the mixture was cooled down to room temperature and poured into MeOH (150 mL) at 0 °C. The mixture was then concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (DCM:MeOH = 10:1) to afford the title compound (80 mg, 27.4% yield) as a yellow solid. MS obsd (ESI+) [({<sup>35</sup>Cl}M+H)<sup>+</sup>] 527.2, [({<sup>37</sup>Cl}M+H)<sup>+</sup>] 529.3.</div></div><div id="sec5_6_16_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i143" class="anchor-spacer"></div><h5 class="article-section__title" id="_i143"> Step (c)</h5><div class="NLM_p last">Preparation of <i>tert</i>-butyl N-[4-[4-(aminomethyl)-3-(trifluoromethyl)pyrazol-1-yl]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate. A solution of <i>tert-</i>butyl N-[4-[4-(aminomethyl)-3-(trifluoromethyl)pyrazol-1-yl]-3-chloro-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (35 mg, 0.066 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)nicotinonitrile (31 mg, 0.133 mmol), and CsF (40 mg, 0.264 mmol) in 1,4-dioxane/H<sub>2</sub>O (2 mL, v/v = 10:1) was degassed with N<sub>2</sub> several times, and then Pd<sub>2</sub>(dba)<sub>3</sub> (18.2 mg, 0.0198 mmol) and Xantphos (12.6 mg, 0.0264 mmol) were added to the mixture at 0 °C under N<sub>2</sub>. After the addition, the resulting reaction solution was degassed again with N<sub>2</sub> five times at 20 °C before it was stirred at 100 °C for 12 h. The reaction mixture was cooled down to room temperature and extracted between water (40 mL) and EtOAc (100 mL) twice. Combined organics were washed with brine (50 mL), dried over anhy. Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-TLC (DCM:MeOH = 10:1) to afford the title compound (30 mg, 76% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 595.2.</div></div><div id="sec5_6_16_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i144" class="anchor-spacer"></div><h5 class="article-section__title" id="_i144"> Step (d)</h5><div class="NLM_p last">Preparation of 5-(4-(4-(aminomethyl)-3-(trifluoromethyl)-1H-pyrazol-1-yl)-8-(ethylamino)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-3-yl)nicotinonitrile 2,2,2-trifluoroacetate. To a solution of <i>tert</i>-butyl N-[4-[4-(aminomethyl)-3-(trifluoromethyl)pyrazol-1-yl]-3-(5-cyano-3-pyridyl)-6-fluoro-9H-pyrido[2,3-<i>b</i>]indol-8-yl]-N-ethyl-carbamate (25 mg, 0.042 mmol) in DCM (2 mL) was added TFA (1 mL), and the resulting reaction mixture was stirred at 30 °C for 1 h. Afterward, the reaction mixture was concentrated <i>in vacuo</i> to give a crude product, which was purified by prep-HPLC to afford the title compound (13.5 mg, 67.5% yield) as a yellow solid. MS obsd (ESI+) [(M+H)<sup>+</sup>] 495.3. HRMS calcd [(M+H)<sup>+</sup>] 495.1669, measured [(M+H)<sup>+</sup>] 495.1669. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i>6) δ ppm 12.28 (s, 1 H), 9.00 (d, 1 H), 8.74 (s, 1 H), 8.55 (d, 1 H), 8.36 (br. s., 3 H), 8.30 (s, 1 H), 8.19 (s, 1 H), 6.51 (dd, 1.76 Hz, 1 H), 6.02 (dd, 1.76 Hz, 1 H), 4.09 (br. s., 2 H), 3.26 (q, 2 H), 1.32 (t, 3 H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ ppm 160.2, 157.9, 153.1, 151.5, 148.3, 140.1, 138.4, 136.7, 134.5, 131.9, 125.8, 125.1, 123.4, 120.6, 117.1, 117.0, 116.9, 111.7, 109.3, 95.6, 93.9, 38.1, 35.3, 14.5.</div></div></div><div id="sec5_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> 4-[4-(Aminomethyl)-3-(trifluoromethyl)pyrazol-1-yl]-N-ethyl-6-fluoro-3-(2-methoxypyrimidin-5-yl)-9H-pyrido[2,3-<i>b</i>]indol-8-amine (<b>17r</b>)</h4><div class="NLM_p last">The title compound was prepared by a similar procedure to that described for compound <b>15r</b>. MS obsd (ESI+) [(M+H)<sup>+</sup>] 501.1. HRMS calcd [(M+H)<sup>+</sup>] 501.1774, measured [(M+H)<sup>+</sup>] 501.1775. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d6</i>) δ ppm 12.23 (s, 1H), 8.71 (s, 1H), 8.31–8.40 (m, 6H), 6.48 (m, 1H), 5.94 (m, 1H), 4.12 (s, 2H), 3.94 (s, 3H), 3.25 (q, 2H), 1.32 (t, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d6</i>) δ 164.7, 160.2, 159.1, 157.9, 153.0, 148.1, 139.8, 138.3, 136.7, 134.2, 125.6, 125.1, 123.5, 123.5, 119.2, 116.8, 112.1, 95.4, 93.7, 55.2, 38.1, 35.3, 14.5.</div></div><div id="sec5_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> <i>In Vitro</i> Assay for the Determination of GyrB and ParE Inhibition (<i>K</i><sub><i>i</i></sub>)</h4><div class="NLM_p">The <i>E. coli</i> GyrB, ParE, and ParC proteins were purified from <i>E. coli</i> cells overexpressing the respective genes individually. <i>K</i><sub><i>i</i></sub> for the respective enzymes was determined by ATPase coupled reaction assay modified from a previously published method.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> All chemicals were purchased from Sigma Chemicals. The reaction volume was 60 μL. For the GyrB reaction, the reaction mixture contained final concentrations of 40 mM Tris·HCl (pH 8.0), 25 mM KCl, 2.5 mM Spermidine, 4 mM MgCl<sub>2</sub>, 0.4 mM phosphoenolpyruvate monopotassium, 0.5 mM NADH (B-NAD reduced form dipotassium), 0.4 uM pyruvate kinase (Sigma P1506), 0.4 uM L-lactic dehydrogenase (Sigma L3916), 20–60 nM GyrB (depending on the desired assay window), and 600 μM ATP. The reaction was started with the addition of ATP substrate and the initial OD340 of the reaction in 384 microtiter well was measured and recorded in a plate reader (Envision 2103, PerkinElmer). After the desired time (120–180 min) of incubation at room temperature, the OD340 of the reaction mixture was measured again. The change in the OD340 was used as the extent of ATP to ADP conversion during the reaction. To test compound inhibition, an 18 point 2-fold titration was done for each compound, including a no compound (100% reaction) control. After the reaction, the fractional reaction velocity at each compound concentration was obtained and data points were plotted against the linear concentration. The <i>K</i><sub><i>i</i></sub> value was obtained through a Morrison curve-fit with either GraphPad or XcelFit software, setting ATP Km at 600 μM.</div><div class="NLM_p last">For the ParE reaction, the condition was the same as for the GyrB except 50 nM ParE and 50 nM ParC proteins were mixed with DNA (Sigma D1626) and added instead of the GyrB protein. The ATP Km for the ParC/ParE was also previously determined to be 800 μM. Unlike GyrB, ParE protein alone contained very low level ATPase activity; therefore, ParC protein was needed for efficient reaction to occur.</div></div><div id="sec5_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> Minimal Inhibitory Concentration (MIC) Measurement</h4><div class="NLM_p last">The MIC of antibacterial compounds against multiple species and strains was determined using the broth microdilution method according to M07-A10 Clinical and Laboratory Standards Institute (CLSI) guidelines (CLSI (2015) Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically; Approved Standard, Tenth Edition. CLSI Document M7-A10 (ISBN 1-56238-987-4). Wayne, PA: Clinical and Laboratory Standards Institute, 19087, USA). In short, bacterial strain preparation was performed on Day 1 and Day 2. The bacterial strains were revived from storage frozen 2 days before the MIC screening and streaked onto the surface of agar plates, incubated in an ambient-air incubator with humidity for 20–24 h at 35 ± 2 °C. 5–10 well-isolated colonies of similar morphology were selected and restreaked onto fresh agar plates using sterile loops and incubated for 20–24 h at 35 ± 2 °C. Bacterial strains were subcultured two times to ensure fresh inoculums for MIC screening. On the day of MIC assay (Day 3), colonies were transferred from fresh culture plates into 5 mL of saline with sterile loops and mixed well, which was measured and turbidity was adjusted to 0.5 McFarland barium sulfate standard (1–2 × 10<sup>8</sup> Colony Forming Units (CFU)/mL) using a turbidity meter. Alternatively, 1–2 colonies were transferred into 500 μL of saline, and OD600 was adjusted to ∼0.1 using an OD meter. Bacterial inoculum was diluted by 1:280 for Gram-positive strains and 1:400 for Gram-negative strains into corresponding medium broth (CAMHB) (e.g., 35.6 μL of inoculum into 10 mL of CAMHB or 25 μL of inoculum into 10 mL of CAMHB). For the compound plate, the compound stock solutions were prepared in 100% DMSO on the day of MIC screening and used immediately. Compound stock concentration = [(highest testing concentration) × 100 μL/2 μL] (e.g., if the required highest testing concentration is 16 μg/mL in assay plates, stock concentration = 16 μg/mL × 100 μL/2 μL = 3.2 mg/mL). The compound plates were prepared by dispensing 100 μL of each stock solution in the first well of a 96 well microtiter plate (MTP) at a concentration 100-fold higher than the final concentration desired in broth. Eleven serial 2-fold dilutions of the highest concentration were made in DMSO for new compounds and for reference compounds (GP-6 and ciprofloxacin). One μL of each well was transferred in a new MTP, which served as the test plate by subsequent inoculation. The bacterial suspension prepared above is dispensed at 98 μL/well within 15 min after preparation. Negative controls (lack of bacterial cells) and growth control wells (lack of compound) were included in all plates. MTPs were incubated for 18–24 h at 35 ± 2 °C in ambient air. The MIC was recorded and CFUs were calculated on Day 4. The assay plate was placed on top of the MIC reader, and the magnification mirror was adjusted to read each well, recording growth status as raw data. Photo images of each assay plates were recorded using Q Count 530 (Spiral Biotech). The MIC value of each compound, expressed as μg/mL, was determined as the lowest concentration required for complete growth inhibition (no visible growth).</div></div><div id="sec5_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> p<i>K</i><sub>a</sub> Determination</h4><div class="NLM_p last">The acid dissociation constant (ionization constant), p<i>K</i><sub>a</sub>, is determined in a high-throughput assay based on photometric titration. This technique allows relating the UV-spectral changes of a molecule upon addition of acid or base to its degree of ionization. In the ProfilerSGA (Sirius Analytical Instruments Ltd.), an aqueous solution of sample compound (solubility up to 10<sup>–6</sup> M) is injected at constant flow rate into a flowing pH gradient. Changes in UV absorbance are monitored as a function of the pH gradient. The p<i>K</i><sub>a</sub> values are found and determined where the rate of change of absorbance is at a maximum. The pH gradient is established by proportionally mixing two flowing buffer solutions containing 10%/volume methanol. The buffer solutions contain a mixture of weak acids and bases that do not absorb significantly in the UV above 240 nm.</div></div><div id="sec5_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i149" class="anchor-spacer"></div><h4 class="article-section__title" id="_i149"> logD Determination</h4><div class="NLM_p last">The distribution coefficient of compounds was determined in a carrier-mediated distribution system (CAMDIS, EP2005102211A) assay, as previously reported.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div></div><div id="sec5_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> Solubility</h4><div class="NLM_p last">The solubility of compounds was determined in a lyophilization solubility assay (LYSA). The test samples were prepared in duplicate from 10 mM DMSO stock solution. After the evaporation of DMSO with a centrifugal vacuum evaporator, the compounds were redissolved in a phosphate buffer (0.05 M, pH 6.5). The mixture was stirred for 1 h, shaken for 2 h, and then allowed to stand overnight and filtered through a microtiter filter plate. The filtrate and its 1/10 dilution were analyzed by HPLC-UV. Solubility determination was determined from a four-point calibration curve; the percentage of the sample measured in solution after evaporation divided by the calculated maximum of sample amount was bigger than 80%; and the solubility data were reported as bigger than this value, with a unit of μg/mL.</div></div><div id="sec5_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> Liver Microsomal Stability (LM)</h4><div class="NLM_p last">Each test compound was preincubated in an incubation mixture consisted of 0.5 mg of microsomal protein/mL mouse liver microsomes, 1 mM NADP, 3 mM glucose 6-phosphate, 3 mM MgCl<sub>2</sub>, and 0.05 mg/mL glucose 6-phosphate dehydrogenase in a total volume of 400 μL of potassium phosphate buffer (100 mM, pH 7.4) for 10 min at 37 °C. The reactions were initiated by the addition of NADPH regenerating system. At different time points (0, 3, 6, 9, 15, and 30 min), an aliquot (50 μL) sample was taken and quenched with 150 μL of acetonitrile containing an internal standard. Following precipitation and centrifugation, the supernatants were analyzed by LC/MS-MS.</div></div><div id="sec5_6_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i152" class="anchor-spacer"></div><h4 class="article-section__title" id="_i152"> Hepatocyte Stability (Hepa Cl)</h4><div class="NLM_p last">Metabolic stability of compounds in mouse liver hepatocyte was determined in screening mode using hepatocyte suspension culture. Incubations of a test compound at different concentrations are performed in 96 well plates containing liver cells (5% CO<sub>2</sub> atmosphere and 37 °C). At defined time points, either cell medium is taken or the whole well is quenched with acetonitrile containing an internal standard. Samples are then cooled and centrifuged before analysis by LC-MS/MS. Log peak area ratios (test compound peak area/internal standard peak area) or concentrations are plotted against incubation time, and a linear fit is made to the data with emphasis upon the initial rate of compound disappearance. The slope of the fit is then used to calculate the intrinsic clearance: Cl<sub>int</sub> (μL/min/1 × 10<sup>6</sup> cells) = −slope (min<sup>–1</sup>) * 1000/[1 × 10<sup>6</sup> cells].</div></div><div id="sec5_6_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i153" class="anchor-spacer"></div><h4 class="article-section__title" id="_i153"> Plasma Protein Binding (fu)</h4><div class="NLM_p last">The percentage of unbound compound was determined using a 96-well Micro-Equilibrium dialysis device (HTDialysis, Gales Ferry, CT, USA) with a molecular weight cutoff membrane of 12–14 kDa (HTDialysis, Gales Ferry, CT, USA) and using Diazepam as a positive control. Pooled rat and cynomolgus monkey plasma was purchased from Biopredic (Rennes, France). Compounds were measured in a cassette of 2–5 with an initial total concentration of 1 μM, and one of the cassette compounds is the positive control. The integrity of membranes was validated by confirming the unbound fraction values of the positive control. Equal volumes of blank dialysis buffer (Soerensen buffer at pH 7.4) and matrix samples containing substances were loaded into the acceptor and donor compartment, respectively. The HTD dialysis block was then sealed and kept in an incubator at 37 °C for 5 h under 5% CO<sub>2</sub> environment. At the end of dialysis, the plasma and buffer samples were retrieved and the drug concentrations were quantified by LC/MS-MS.</div></div><div id="sec5_6_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i154" class="anchor-spacer"></div><h4 class="article-section__title" id="_i154"> Single Dose Intravenous PK Study in Female CD-1 Mouse</h4><div class="NLM_p last">The single-dose PK in female CD-1 mouse was performed to assess the compound’s pharmacokinetic properties. One group of animals were dosed via bolus intravenous (IV) of the respective compound. Blood samples (approximately 30 μL) were collected via saphenous vein at 5 min, 15 min, 30 min, 1, 2, 4, and 7 h. For the last time point (24 h), samples were collected via cardiac puncture while the mouse was under anesthesia. Blood samples were placed into tubes containing EDTA-K2 anticoagulant (2 μL of 0.5 M per tube) and centrifuged at 3000 rpm for 10 min at 4 °C within half an hour to separate plasma from the samples. Following centrifugation, plasma samples were stored in polypropylene tubes, quick-frozen over dry ice, and kept at −70 ± 10 °C until LC-MS/MS analysis. The pharmacokinetic parameters were calculated using a noncompartmental module of WinNonlin Professional 6.4.</div></div><div id="sec5_6_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i155" class="anchor-spacer"></div><h4 class="article-section__title" id="_i155"> Experimental Details for Protein Generation and Crystallization of <i>P. aeruginosa</i> GyrB with Compounds <b>12o</b> and <b>12x</b></h4><div class="NLM_p last">The ATP-binding domain of <i>P. aeruginosa</i> GyrB was cloned into vector pET30 (Novagen) with a C terminal 6xHis tag and overexpressed in <i>E. coli</i> strain BL21 (DE3) at 16 °C overnight. Cell pellets were lysed and purified with nickel affinity, ion exchange, and size exclusion chromatography. The purified protein was concentrated and stored in a buffer solution containing 20 mM Tris (pH 8.0) and 200 mM NaCl. The protein–ligand complex was cocrystallized by sitting drop vapor diffusion method with reservoir solution: 38% w/v PEG3350, 200 mM Lithium sulfate, and 100 mM sodium citrate (pH 5.6). Cocrystals appeared within 2 weeks and were flash-frozen with the addition of 20% glycerol. X-ray diffraction images were collected at 100 K at the beamline BL17U of the Shanghai Synchrotron Radiation Facility (SSRF).</div></div><div id="sec5_6_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i156" class="anchor-spacer"></div><h4 class="article-section__title" id="_i156"> Experimental Details for <i>in Vivo</i> Neutropenic Mouse Thigh Infection Experiments</h4><div class="NLM_p last">Female ICR mice weighing 23–25 g were purchased from Beijing Vital River Laboratory Animal Technology Co. Ltd. The animals were housed at 20–25 °C and 40–70% relative humidity, with food and water available ad libitum throughout the study. The study protocol was approved by the Institutional Animal Care and Use Committee (IACUC) of Roche Pharma Research and Early Development China. Mice were rendered neutropenic (neutrophils, less than 100/mm<sup>3</sup>) by injection of cyclophosphamide (Shanxi Pude Pharmaceuticals, Co. Ltd.) intraperitoneally 4 days (150 mg/kg body weight) and 1 day (100 mg/kg) before thigh infection. Thigh infection in mice was carried out by intramuscular injection of 0.1 mL inoculum into the right posterior thigh muscle, with about an inoculum size of 10<sup>6</sup> CFU/thigh (inoculum was prepared by inoculating single colony into fresh MHB media the day prior to thigh infection and cultured at 150 rpm and 35 °C for 15 h. The overnight culture (the bacterial suspension) was further diluted to a concentration of 10<sup>7</sup> CFU/mL in fresh MHB media prior to use). Two hours post thigh infection, mice were dosed with compound via intravenous tail injection. Blood samples were collected −5 min, 30 min, 55 min, 1.5, 2, 3, 5, 7, and 24 h after compound or vehicle application. The mice were sacrificed by euthanasia after 12 or 24 h of therapy, and the thighs were removed. Thigh muscles were transferred into 5 mL sterile EP tubes and homogenized in 2 mL saline for subsequent CFU determination.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i157"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00768" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00768?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00768</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR, <sup>13</sup>C NMR, and LC-MS spectra of compounds <b>15r</b>, <b>17r</b>, and <b>17x</b>; PK/PD correlation of compound <b>17r</b> in neutropenic mouse thigh infection model (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of compounds <b>10</b>, <b>12</b>, <b>15</b>, and <b>17</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_001.pdf">jm0c00768_si_001.pdf (877.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_002.csv">jm0c00768_si_002.csv (5.9 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB codes for <b>12o</b> and <b>12x</b> are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S">6M1S</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1J">6M1J</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00768" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuefei Tan</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7629-2255" title="Orcid link">http://orcid.org/0000-0001-7629-2255</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#611914040704084f15000f21130e0209044f020e0c"><span class="__cf_email__" data-cfemail="1f676a7a797a76316b7e715f6d707c777a317c7072">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yimin Hu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Houguang Shi</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingwei Zhou</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qingcheng Ren</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec (Wuhan) Co., Ltd., No. 666 Gaoxin
Road, Wuhan East Lake High-tech Development Zone, Hubei 430075, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Zhu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4701-2474" title="Orcid link">http://orcid.org/0000-0003-4701-2474</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weixing Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhiwei Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chengang Zhou</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongqiang Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao Ding</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong C. Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3477-8932" title="Orcid link">http://orcid.org/0000-0002-3477-8932</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">S. Frank Yan</span> - <span class="hlFld-Affiliation affiliation">CADD, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9121-9694" title="Orcid link">http://orcid.org/0000-0002-9121-9694</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabian Dey</span> - <span class="hlFld-Affiliation affiliation">Department of CADD and , Roche Pharma Research and Early Development, Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Waikwong Wu</span> - <span class="hlFld-Affiliation affiliation">Lead Discovery, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guanglei Zhai</span> - <span class="hlFld-Affiliation affiliation">Lead Discovery, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zheng Zhou</span> - <span class="hlFld-Affiliation affiliation">Lead Discovery, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhiheng Xu</span> - <span class="hlFld-Affiliation affiliation">Lead Discovery, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Ji</span> - <span class="hlFld-Affiliation affiliation">pCMC, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hua Lv</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical Sciences, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tianyi Jiang</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical Sciences, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen Wang</span> - <span class="hlFld-Affiliation affiliation">MicuRx Pharmaceuticals,
Inc. (Shanghai), Floor 3, Building B, 1976 Middle Gaoke Road, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yunhua Xu</span> - <span class="hlFld-Affiliation affiliation">MicuRx Pharmaceuticals,
Inc. (Shanghai), Floor 3, Building B, 1976 Middle Gaoke Road, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maarten Vercruysse</span> - <span class="hlFld-Affiliation affiliation">Discovery Infectious Diseases,
Roche Innovation Center
Basel and , Roche Pharma Research and Early Development, Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiangyu Yao</span> - <span class="hlFld-Affiliation affiliation">Discovery Infectious
Diseases, Roche Innovation Center Shanghai, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi Mao</span> - <span class="hlFld-Affiliation affiliation">Discovery Infectious
Diseases, Roche Innovation Center Shanghai, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaomin Yu</span> - <span class="hlFld-Affiliation affiliation">Discovery Infectious
Diseases, Roche Innovation Center Shanghai, , , ,  and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth Bradley</span> - <span class="hlFld-Affiliation affiliation">Discovery Infectious Diseases,
Roche Innovation Center
Basel and , Roche Pharma Research and Early Development, Grenzacherstrasse 124, CH-4070 Basel, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i160">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07141" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07141" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge Liqin Chen, Leimin Wang, Xiaojie Xia, Qingshan Gao, Yujuan Gao, and Jin Ge for technical assistance in the purification of final compounds and analytical analysis. We thank Yi Zhang, Hongxia Qiu, Yuxia Zhang, and Sheng Zhong for the support of DMPK profiling. This work was supported exclusively by F. Hoffmann-La Roche Ltd.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein data bank</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">SDPK</td><td class="NLM_def"><p class="first last">single dose pharmacokinetics</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr><tr><td class="NLM_term">UV</td><td class="NLM_def"><p class="first last">ultraviolet spectrometry</p></td></tr><tr><td class="NLM_term">CFU</td><td class="NLM_def"><p class="first last">colony forming units</p></td></tr><tr><td class="NLM_term">BINAP</td><td class="NLM_def"><p class="first last">2,2′-bis(diphenylphosphino)-1,1′-binaphthyl</p></td></tr><tr><td class="NLM_term">MCPBA</td><td class="NLM_def"><p class="first last">meta-chloroperoxybenzoic acid</p></td></tr><tr><td class="NLM_term">X-phos</td><td class="NLM_def"><p class="first last">2-dicyclohexylphosphino-2′,4′,6′-triisopropylbiphenyl</p></td></tr><tr><td class="NLM_term">DBU</td><td class="NLM_def"><p class="first last">1,8-diazabicyclo[5.4. 0]undec-7-ene</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last">dimethylacetamide</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">benzotriazol-1-yl-oxytripyrrolidinophosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">Xantphos</td><td class="NLM_def"><p class="first last">4,5-bis(diphenylphosphino)-9,9-dimethylxanthene</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i162">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78028" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78028" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, B. F.</span></span> <span> </span><span class="NLM_article-title">Clinical relevance of the ESKAPE pathogens</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1586/eri.13.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1586%2Feri.13.12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23458769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=297-308&author=J.+N.+Pendletonauthor=S.+P.+Gormanauthor=B.+F.+Gilmore&title=Clinical+relevance+of+the+ESKAPE+pathogens&doi=10.1586%2Feri.13.12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical relevance of the ESKAPE pathogens</span></div><div class="casAuthors">Pendleton, Jack N.; Gorman, Sean P.; Gilmore, Brendan F.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-308</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, the Infectious Diseases Society of America has highlighted a faction of antibiotic-resistant bacteria (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) - acronymically dubbed the ESKAPE pathogens' - capable of escaping' the biocidal action of antibiotics and mutually representing new paradigms in pathogenesis, transmission and resistance.  This review aims to consolidate clin. relevant background information on the ESKAPE pathogens and provide a contemporary summary of bacterial resistance, alongside pertinent microbiol. considerations necessary to face the mounting threat of antimicrobial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql_dEmz1UF37Vg90H21EOLACvtfcHk0likRJDlGxDgKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D&md5=7168500f85d84236a255e20dbf18930a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1586%2Feri.13.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.13.12%26sid%3Dliteratum%253Aachs%26aulast%3DPendleton%26aufirst%3DJ.%2BN.%26aulast%3DGorman%26aufirst%3DS.%2BP.%26aulast%3DGilmore%26aufirst%3DB.%2BF.%26atitle%3DClinical%2520relevance%2520of%2520the%2520ESKAPE%2520pathogens%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2013%26volume%3D11%26spage%3D297%26epage%3D308%26doi%3D10.1586%2Feri.13.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santajit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indrawattana, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of antimicrobial resistance in ESKAPE pathogens</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">2475067</span>, <span class="refDoi"> DOI: 10.1155/2016/2475067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1155%2F2016%2F2475067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=27274985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=2475067&author=S.+Santajitauthor=N.+Indrawattana&title=Mechanisms+of+antimicrobial+resistance+in+ESKAPE+pathogens&doi=10.1155%2F2016%2F2475067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Antimicrobial Resistance in ESKAPE Pathogens</span></div><div class="casAuthors">Santajit Sirijan; Indrawattana Nitaya</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2016</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2475067</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) are the leading cause of nosocomial infections throughout the world.  Most of them are multidrug resistant isolates, which is one of the greatest challenges in clinical practice.  Multidrug resistance is amongst the top three threats to global public health and is usually caused by excessive drug usage or prescription, inappropriate use of antimicrobials, and substandard pharmaceuticals.  Understanding the resistance mechanisms of these bacteria is crucial for the development of novel antimicrobial agents or other alternative tools to combat these public health challenges.  Greater mechanistic understanding would also aid in the prediction of underlying or even unknown mechanisms of resistance, which could be applied to other emerging multidrug resistant pathogens.  In this review, we summarize the known antimicrobial resistance mechanisms of ESKAPE pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSM_xN5GU5d0ntlzo9xastyfW6udTcc2ebuYpx7mPas_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D&md5=e42353b6d72db948734b5354e31b501e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1155%2F2016%2F2475067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2016%252F2475067%26sid%3Dliteratum%253Aachs%26aulast%3DSantajit%26aufirst%3DS.%26aulast%3DIndrawattana%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520antimicrobial%2520resistance%2520in%2520ESKAPE%2520pathogens%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2016%26volume%3D2016%26spage%3D2475067%26doi%3D10.1155%2F2016%2F2475067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosen, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepore, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coukell, A.</span></span> <span> </span><span class="NLM_article-title">On the road to discovering urgently needed antibiotics: so close yet so far away</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1292</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00100</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00100" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVOksbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1292-1294&author=W.+Kimauthor=K.+R.+Prosenauthor=C.+J.+Leporeauthor=A.+Coukell&title=On+the+road+to+discovering+urgently+needed+antibiotics%3A+so+close+yet+so+far+away&doi=10.1021%2Facsinfecdis.0c00100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">On the Road to Discovering Urgently Needed Antibiotics: So Close Yet So Far Away</span></div><div class="casAuthors">Kim, Wes; Prosen, Katherine R.; Lepore, Cara J.; Coukell, Allan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1292-1294</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Pew Charitable Trusts' 2016 publication "A Scientific Roadmap for Antibiotic Discovery" provided a consensus approach to accelerating the discovery of novel antibiotics targeting Gram-neg. pathogens.  Since then, encouraging initiatives have launched to catalyze antibiotics discovery, particularly by improving knowledge sharing and making discovery efforts more efficient and effective.  However, because the global pipeline remains insufficient to address current and future unmet needs, existing initiatives are not enough.  Sustained public funding is crit., particularly as private funding continues to dwindle.  And with public funding comes the responsibility of sharing what has been learned.  Finally, a "precompetitive" R&D model in which the financial return on investment is not a primary driver warrants further consideration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw1Ad8k_kdTrVg90H21EOLACvtfcHk0lhIdLGVF-VNNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVOksbjL&md5=9a6e4c581251ba32ae0a8cd2742a9e61</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00100%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.%26aulast%3DProsen%26aufirst%3DK.%2BR.%26aulast%3DLepore%26aufirst%3DC.%2BJ.%26aulast%3DCoukell%26aufirst%3DA.%26atitle%3DOn%2520the%2520road%2520to%2520discovering%2520urgently%2520needed%2520antibiotics%253A%2520so%2520close%2520yet%2520so%2520far%2520away%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2020%26volume%3D6%26spage%3D1292%26epage%3D1294%26doi%3D10.1021%2Facsinfecdis.0c00100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">A gestalt approach to gram-negative entry</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">6379</span>– <span class="NLM_lpage">6389</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.06.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmc.2016.06.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=27381365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVaqur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6379-6389&author=L.+L.+Silver&title=A+gestalt+approach+to+gram-negative+entry&doi=10.1016%2Fj.bmc.2016.06.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A Gestalt approach to Gram-negative entry</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6379-6389</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A major obstacle confronting the discovery and development of new antibacterial agents to combat resistant Gram-neg. (GN) organisms is the lack of a rational process for endowing compds. with properties that allow (or promote) entry into the bacterial cytoplasm.  The major permeability difference between GN and Gram-pos. (GP) bacteria is the GN outer membrane (OM) which is a permeability barrier itself and potentiates efflux pumps that expel compds.  Based on the fact that OM-permeable and efflux-deleted GNs are sensitive to many anti-GP drugs, recent efforts to approach the GN entry problem have focused on ways of avoiding efflux and transiting or compromising the OM, with the tacit assumption that this could allow entry of compds. into the GN cytoplasm.  But bypassing the OM and efflux obstacles does not take into account the addnl. requirement of penetrating the cytoplasmic membrane (CM) whose sieving properties appear to be orthogonal to that of the OM.  That is, tailoring compds. to transit the OM may well compromise their ability to enter the cytoplasm.  Thus, a Gestalt approach to understanding the chem. requirements for GN entry seems a useful adjunct.  This might consist of characterizing compds. which reach the cytoplasm, grouping (or binning) by routes of entry and formulating chem. rules for those bins.  This will require acquisition of data on large nos. of compds., using non-activity-dependent methods of measuring accumulation in the cytoplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqs-T6vYmkrrVg90H21EOLACvtfcHk0lhIdLGVF-VNNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVaqur%252FP&md5=08ca97526e7f3c8c8781307e52e96e9a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.06.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.06.044%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DA%2520gestalt%2520approach%2520to%2520gram-negative%2520entry%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6379%26epage%3D6389%26doi%3D10.1016%2Fj.bmc.2016.06.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span>Nabriva
Therapeutics Receives U.S. <span> </span><span class="NLM_article-title">FDA Approval of Xenleta
(lefamulin) to Treat Community-Acquired Bacteria Pneumonia (CABP)</span> [news release]. <a href="http://investors.nabriva.com" class="extLink">http://investors.nabriva.com</a> (accessed <span class="NLM_year">2019-08-19</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Nabriva%0ATherapeutics+Receives+U.S.+FDA+Approval+of+Xenleta%0A%28lefamulin%29+to+Treat+Community-Acquired+Bacteria+Pneumonia+%28CABP%29+%5Bnews+release%5D.+http%3A%2F%2Finvestors.nabriva.com+%28accessed+2019-08-19%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DFDA%2520Approval%2520of%2520Xenleta%250A%2528lefamulin%2529%2520to%2520Treat%2520Community-Acquired%2520Bacteria%2520Pneumonia%2520%2528CABP%2529%26date%3D2019-08-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">Challenges of antibacterial discovery</span>. <i>Clin. Microbio. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1128/CMR.00030-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1128%2FCMR.00030-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=21233508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=71-109&author=L.+L.+Silver&title=Challenges+of+antibacterial+discovery&doi=10.1128%2FCMR.00030-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges of antibacterial discovery</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-109</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The discovery of novel small-mol. antibacterial drugs has been stalled for many years.  The purpose of this review is to underscore and illustrate those scientific problems unique to the discovery and optimization of novel antibacterial agents that have adversely affected the output of the effort.  The major challenges fall into two areas: (i) proper target selection, particularly the necessity of pursuing mol. targets that are not prone to rapid resistance development, and (ii) improvement of chem. libraries to overcome limitations of diversity, esp. that which is necessary to overcome barriers to bacterial entry and proclivity to be effluxed, esp. in Gram-neg. organisms.  Failure to address these problems has led to a great deal of misdirected effort.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTDYwtK6vHbVg90H21EOLACvtfcHk0li1USBKPFTNXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D&md5=a77b7a1f2763b73d4bd63daf67f78cc7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FCMR.00030-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00030-10%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DChallenges%2520of%2520antibacterial%2520discovery%26jtitle%3DClin.%2520Microbio.%2520Rev.%26date%3D2011%26volume%3D24%26spage%3D71%26epage%3D109%26doi%3D10.1128%2FCMR.00030-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schillaci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanò, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montalbano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barraja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diana, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirrincione, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascioferro, S.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical approached to target antiobiotic resistance mechanisms</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8268</span>– <span class="NLM_lpage">8297</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8268-8297&author=D.+Schillaciauthor=V.+Span%C3%B2author=B.+Parrinoauthor=A.+Carboneauthor=A.+Montalbanoauthor=P.+Barrajaauthor=P.+Dianaauthor=G.+Cirrincioneauthor=S.+Cascioferro&title=Pharmaceutical+approached+to+target+antiobiotic+resistance+mechanisms&doi=10.1021%2Facs.jmedchem.7b00215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical Approaches to Target Antibiotic Resistance Mechanisms</span></div><div class="casAuthors">Schillaci, Domenico; Spano, Virginia; Parrino, Barbara; Carbone, Anna; Montalbano, Alessandra; Barraja, Paola; Diana, Patrizia; Cirrincione, Girolamo; Cascioferro, Stella</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8268-8297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is urgent need for new therapeutic strategies to fight the global threat of antibiotic resistance.  The focus of this Perspective is on chem. agents that target the most common mechanisms of antibiotic resistance such as enzymic inactivation of antibiotics, changes in cell permeability, and induction/activation of efflux pumps.  Here the authors assess the current landscape and challenges in the treatment of antibiotic resistance mechanisms at both bacterial cell and community levels.  The authors also discuss the potential clin. application of chem. inhibitors of antibiotic resistance mechanisms as add-on treatments for serious drug-resistant infections.  Enzymic inhibitors, such as the derivs. of the β-lactamase inhibitor avibactam, are closer to the clinic than other mols.  For example, MK-7655, in combination with imipenem, is in clin. development for the treatment of infections caused by carbapenem-resistant Enterobacteriaceae and Pseudomonas aeruginosa, which are difficult to treat.  In addn., other mols. targeting multidrug-resistance mechanisms, such as efflux pumps, are under development and hold promise for the treatment of multidrug resistant infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo5bQLECGVxLVg90H21EOLACvtfcHk0li1USBKPFTNXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D&md5=dfbd996313789479a5a10becb59257df</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00215%26sid%3Dliteratum%253Aachs%26aulast%3DSchillaci%26aufirst%3DD.%26aulast%3DSpan%25C3%25B2%26aufirst%3DV.%26aulast%3DParrino%26aufirst%3DB.%26aulast%3DCarbone%26aufirst%3DA.%26aulast%3DMontalbano%26aufirst%3DA.%26aulast%3DBarraja%26aufirst%3DP.%26aulast%3DDiana%26aufirst%3DP.%26aulast%3DCirrincione%26aufirst%3DG.%26aulast%3DCascioferro%26aufirst%3DS.%26atitle%3DPharmaceutical%2520approached%2520to%2520target%2520antiobiotic%2520resistance%2520mechanisms%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8268%26epage%3D8297%26doi%3D10.1021%2Facs.jmedchem.7b00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Champoux, J. J.</span></span> <span> </span><span class="NLM_article-title">DNA topoisomerases: structure, function, and mechanism</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.70.1.369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1146%2Fannurev.biochem.70.1.369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=11395412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2001&pages=369-413&author=J.+J.+Champoux&title=DNA+topoisomerases%3A+structure%2C+function%2C+and+mechanism&doi=10.1146%2Fannurev.biochem.70.1.369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">DNA topoisomerases: structure, function, and mechanism</span></div><div class="casAuthors">Champoux, James J.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">369-413</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 196 refs.  DNA topoisomerases solve the topol. problems assocd. with DNA replication, transcription, recombination, and chromatin remodeling by introducing temporary single- or double-strand breaks in the DNA.  In addn., these enzymes fine-tune the steady-state level of DNA supercoiling both to facilitate protein interactions with the DNA and to prevent excessive supercoiling that is deleterious.  In recent years, the crystal structures of a no. of topoisomerase fragments, representing nearly all of the known classes of enzymes, have been solved.  These structures provide remarkable insights into the mechanisms of these enzymes and complement previous conclusions based on biochem. analyses.  Surprisingly, despite little or no sequence homol., both type IA and type IIA topoisomerases from prokaryotes and the type IIA enzymes from eukaryotes share structural folds that appear to reflect functional motifs within crit. regions of the enzymes.  The type IB enzymes are structurally distinct from all other known topoisomerases but are similar to a class of enzymes referred to as tyrosine recombinases.  The structural themes common to all topoisomerases include hinged clamps that open and close to bind DNA, the presence of DNA binding cavities for temporary storage of DNA segments, and the coupling of protein conformational changes to DNA rotation or DNA movement.  For the type II topoisomerases, the binding and hydrolysis of ATP further modulate conformational changes in the enzymes to effect changes in DNA topol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Y4rDyIYy17Vg90H21EOLACvtfcHk0lgtHcqYuk3yhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7Y%253D&md5=67c3e38eb4fcb85527b33d7077309482</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.70.1.369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.70.1.369%26sid%3Dliteratum%253Aachs%26aulast%3DChampoux%26aufirst%3DJ.%2BJ.%26atitle%3DDNA%2520topoisomerases%253A%2520structure%252C%2520function%252C%2520and%2520mechanism%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2001%26volume%3D70%26spage%3D369%26epage%3D413%26doi%3D10.1146%2Fannurev.biochem.70.1.369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thathan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubie, S.</span></span> <span> </span><span class="NLM_article-title">Dual targeting DNA gyrase B (GyrB) and topoisomerse IV (ParE) inhibitors: a review</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2015.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bioorg.2015.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=26232660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KlsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=41-63&author=M.+A.+Azamauthor=J.+Thathanauthor=S.+Jubie&title=Dual+targeting+DNA+gyrase+B+%28GyrB%29+and+topoisomerse+IV+%28ParE%29+inhibitors%3A+a+review&doi=10.1016%2Fj.bioorg.2015.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting DNA gyrase B (GyrB) and topoisomerase IV (ParE) inhibitors: A review</span></div><div class="casAuthors">Azam, Mohammed Afzal; Thathan, Janarthanan; Jubie, Selvaraj</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-63</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">GyrB and ParE are type IIA topoisomerases and found in most bacteria.  Its function is vital for DNA replication, repair and decatenation.  The highly conserved ATP-binding subunits of DNA GyrB and ParE are structurally related and have been recognized as prime candidates for the development of dual-targeting antibacterial agents with broad-spectrum potential.  However, no natural product or small mol. inhibitors targeting ATPase catalytic domain of both GyrB and ParE enzymes have succeeded in the clinic.  Moreover, no inhibitors of these enzymes with broad-spectrum antibacterial activity against Gram-neg. pathogens have been reported.  Availability of high resoln. crystal structures of GyrB and ParE made it possible for the design of many different classes of inhibitors with dual mechanism of action.  Among them benzimidazoles, benzothiazoles, thiazolopyridines, imidiazopyridazoles, pyridines, indazoles, pyrazoles, imidazopyridines, triazolopyridines, pyrrolopyrimidines, pyrimidoindoles as well as related structures are disclosed in literatures.  Unfortunately most of these inhibitors are found to be active against Gram-pos. pathogens.  In the present review we discuss about studies on novel dual targeting ATPase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-_PqzrhxWbVg90H21EOLACvtfcHk0lgtHcqYuk3yhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KlsL3J&md5=37ff522c4d029c2e358be124bf6e9264</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2015.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2015.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DAzam%26aufirst%3DM.%2BA.%26aulast%3DThathan%26aufirst%3DJ.%26aulast%3DJubie%26aufirst%3DS.%26atitle%3DDual%2520targeting%2520DNA%2520gyrase%2520B%2520%2528GyrB%2529%2520and%2520topoisomerse%2520IV%2520%2528ParE%2529%2520inhibitors%253A%2520a%2520review%26jtitle%3DBioorg.%2520Chem.%26date%3D2015%26volume%3D62%26spage%3D41%26epage%3D63%26doi%3D10.1016%2Fj.bioorg.2015.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalhoff, A.</span></span> <span> </span><span class="NLM_article-title">Global fluoroquinolone resistance epidemiology and implictions for clinical use</span>. <i>Interdiscip. Perspect. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>2012</i></span>,  <span class="NLM_fpage">976273</span>, <span class="refDoi"> DOI: 10.1155/2012/976273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1155%2F2012%2F976273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23097666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A280%3ADC%252BC3s%252FotVOitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=976273&author=A.+Dalhoff&title=Global+fluoroquinolone+resistance+epidemiology+and+implictions+for+clinical+use&doi=10.1155%2F2012%2F976273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Global fluoroquinolone resistance epidemiology and implictions for clinical use</span></div><div class="casAuthors">Dalhoff Axel</div><div class="citationInfo"><span class="NLM_cas:title">Interdisciplinary perspectives on infectious diseases</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2012</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">976273</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This paper on the fluoroquinolone resistance epidemiology stratifies the data according to the different prescription patterns by either primary or tertiary caregivers and by indication.  Global surveillance studies demonstrate that fluoroquinolone resistance rates increased in the past years in almost all bacterial species except S. pneumoniae and H. influenzae, causing community-acquired respiratory tract infections.  However, 10 to 30% of these isolates harbored first-step mutations conferring low level fluoroquinolone resistance.  Fluoroquinolone resistance increased in Enterobacteriaceae causing community acquired or healthcare associated urinary tract infections and intraabdominal infections, exceeding 50% in some parts of the world, particularly in Asia.  One to two-thirds of Enterobacteriaceae producing extended spectrum β-lactamases were fluoroquinolone resistant too.  Furthermore, fluoroquinolones select for methicillin resistance in Staphylococci.  Neisseria gonorrhoeae acquired fluoroquinolone resistance rapidly; actual resistance rates are highly variable and can be as high as almost 100%, particularly in Asia, whereas resistance rates in Europe and North America range from <10% in rural areas to >30% in established sexual networks.  In general, the continued increase in fluoroquinolone resistance affects patient management and necessitates changes in some guidelines, for example, treatment of urinary tract, intra-abdominal, skin and skin structure infections, and traveller's diarrhea, or even precludes the use in indications like sexually transmitted diseases and enteric fever.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDedqxKx8QHLt7lRqEFUJVfW6udTcc2eY1cZiM_PtjTbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s%252FotVOitw%253D%253D&md5=d6ab6c51cb874358473dfe49bc7d0df4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1155%2F2012%2F976273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2012%252F976273%26sid%3Dliteratum%253Aachs%26aulast%3DDalhoff%26aufirst%3DA.%26atitle%3DGlobal%2520fluoroquinolone%2520resistance%2520epidemiology%2520and%2520implictions%2520for%2520clinical%2520use%26jtitle%3DInterdiscip.%2520Perspect.%2520Infect.%2520Dis.%26date%3D2012%26volume%3D2012%26spage%3D976273%26doi%3D10.1155%2F2012%2F976273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Redgrave, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piddock, L. J.V.</span></span> <span> </span><span class="NLM_article-title">Fluoroquinolone resistance: mechanisms, impact on bacteria, and role in evolutionary success</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2014.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.tim.2014.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=24842194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=438-445&author=L.+S.+Redgraveauthor=S.+B.+Suttonauthor=M.+A.+Webberauthor=L.+J.V.+Piddock&title=Fluoroquinolone+resistance%3A+mechanisms%2C+impact+on+bacteria%2C+and+role+in+evolutionary+success&doi=10.1016%2Fj.tim.2014.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Fluoroquinolone resistance: mechanisms, impact on bacteria, and role in evolutionary success</span></div><div class="casAuthors">Redgrave, Liam S.; Sutton, Sam B.; Webber, Mark A.; Piddock, Laura J. V.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">438-445</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Quinolone and fluoroquinolone antibiotics are potent, broad-spectrum agents commonly used to treat a range of infections.  Resistance to these agents is multifactorial and can be via one or a combination of target-site gene mutations, increased prodn. of multidrug-resistance (MDR) efflux pumps, modifying enzymes, and/or target-protection proteins.  Fluoroquinolone-resistant clin. isolates of bacteria have emerged readily and recent data have shown that resistance to this class of antibiotics can have diverse, species-dependent impacts on host-strain fitness.  Here we outline the impacts of quinolone-resistance mutations in relation to the fitness and evolutionary success of mutant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJrla4tEwdbrVg90H21EOLACvtfcHk0lgtHcqYuk3yhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D&md5=060fb6501e12ed1243e4f30190cb4917</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2014.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2014.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DRedgrave%26aufirst%3DL.%2BS.%26aulast%3DSutton%26aufirst%3DS.%2BB.%26aulast%3DWebber%26aufirst%3DM.%2BA.%26aulast%3DPiddock%26aufirst%3DL.%2BJ.V.%26atitle%3DFluoroquinolone%2520resistance%253A%2520mechanisms%252C%2520impact%2520on%2520bacteria%252C%2520and%2520role%2520in%2520evolutionary%2520success%26jtitle%3DTrends%2520Microbiol.%26date%3D2014%26volume%3D22%26spage%3D438%26epage%3D445%26doi%3D10.1016%2Fj.tim.2014.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirude, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, S.</span></span> <span> </span><span class="NLM_article-title">Nonfluoroquinolone-based inhibitors of mycobacterial type II topoisomerase as potential therapeutic agents for TB</span>. <i>Annu. Rep. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-396492-2.00021-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2FB978-0-12-396492-2.00021-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1eju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=319-330&author=P.+S.+Shirudeauthor=S.+Hameed&title=Nonfluoroquinolone-based+inhibitors+of+mycobacterial+type+II+topoisomerase+as+potential+therapeutic+agents+for+TB&doi=10.1016%2FB978-0-12-396492-2.00021-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Nonfluoroquinolone-based inhibitors of mycobacterial type II topoisomerase as potential therapeutic agents for TB</span></div><div class="casAuthors">Shirude, Pravin S.; Hameed, Shahul</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">319-330</span>CODEN:
                <span class="NLM_cas:coden">ARMCBI</span>;
        ISSN:<span class="NLM_cas:issn">0065-7743</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review focuses on small-mol. inhibitors of mycobacterial type II topoisomerase that bind at B subunit of DNA gyrase (GyrB) having ATP sites and non-fluoroquinolone inhibitors that bind at A subunit of DNA gyrase (GyrA) having other than ATP sites within the enzyme tetramers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7HVfdftzSBbVg90H21EOLACvtfcHk0lgsqsPCh0bg2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1eju7jM&md5=0ae5b4dfe642de20da0932708c24a48b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-396492-2.00021-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-396492-2.00021-7%26sid%3Dliteratum%253Aachs%26aulast%3DShirude%26aufirst%3DP.%2BS.%26aulast%3DHameed%26aufirst%3DS.%26atitle%3DNonfluoroquinolone-based%2520inhibitors%2520of%2520mycobacterial%2520type%2520II%2520topoisomerase%2520as%2520potential%2520therapeutic%2520agents%2520for%2520TB%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2012%26volume%3D47%26spage%3D319%26epage%3D330%26doi%3D10.1016%2FB978-0-12-396492-2.00021-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisacchi, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span> <span> </span><span class="NLM_article-title">A new-class antibacterial-almost. Lessons in drug discovery and development: a critical analysis of more than 50 years of effort toward ATPase inhibitors of DNA gyrase and topoisomerase IV</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1021/id500013t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/id500013t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=4-41&author=G.+S.+Bisacchiauthor=J.+I.+Manchester&title=A+new-class+antibacterial-almost.+Lessons+in+drug+discovery+and+development%3A+a+critical+analysis+of+more+than+50+years+of+effort+toward+ATPase+inhibitors+of+DNA+gyrase+and+topoisomerase+IV&doi=10.1021%2Fid500013t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A New-Class Antibacterial-Almost. Lessons in Drug Discovery and Development: A Critical Analysis of More than 50 Years of Effort toward ATPase Inhibitors of DNA Gyrase and Topoisomerase IV</span></div><div class="casAuthors">Bisacchi, Gregory S.; Manchester, John I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-41</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The introduction into clin. practice of an ATPase inhibitor of bacterial DNA gyrase and topoisomerase IV (topo IV) would represent a new-class agent for the treatment of resistant bacterial infections.  Novobiocin, the only historical member of this class, established the clin. proof of concept for this novel mechanism during the late 1950s, but its use declined rapidly and it was eventually withdrawn from the market.  Despite significant and prolonged effort across the biopharmaceutical industry to develop other agents of this class, novobiocin remains the only ATPase inhibitor of gyrase and topo IV ever to progress beyond Phase I.  In this review, we analyze the historical attempts to discover and develop agents within this class and highlight factors that might have hindered those efforts.  Within the last 15 years, however, our tech. understanding of the mol. details of the inhibition of the gyrase and topo IV ATPases, the factors governing resistance development to such inhibitors, and our knowledge of the phys. properties required for robust clin. drug candidates have all matured to the point wherein the industry may now address this mechanism of action with greater confidence.  The antibacterial spectrum within this class has recently been extended to begin to include serious Gram neg. pathogens such as Pseudomonas aeruginosa, Acinetobacter baumannii, and Klebsiella pneumoniae.  In spite of this recent tech. progress, adverse economics assocd. with antibacterial R&D over the last 20 years has diminished industry's ability to commit the resources and perseverance needed to bring new-class agents to launch.  Consequently, a no. of recent efforts in the ATPase class have been derailed by organizational rather than scientific factors.  Nevertheless, within this context we discuss the unique opportunity for the development of ATPase inhibitors of gyrase and topo IV as new-class antibacterial agents with broad spectrum potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXubyDLdrSrrVg90H21EOLACvtfcHk0lgsqsPCh0bg2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL&md5=95ce2833dab37f68a3a4a018e381e198</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fid500013t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fid500013t%26sid%3Dliteratum%253Aachs%26aulast%3DBisacchi%26aufirst%3DG.%2BS.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26atitle%3DA%2520new-class%2520antibacterial-almost.%2520Lessons%2520in%2520drug%2520discovery%2520and%2520development%253A%2520a%2520critical%2520analysis%2520of%2520more%2520than%252050%2520years%2520of%2520effort%2520toward%2520ATPase%2520inhibitors%2520of%2520DNA%2520gyrase%2520and%2520topoisomerase%2520IV%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D4%26epage%3D41%26doi%3D10.1021%2Fid500013t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span> <span> </span><span class="NLM_article-title">Overcoming problems of poor drug penetration into bacteria: challenges and strategies for medicinal chemists</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1080/17460441.2018.1455660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1080%2F17460441.2018.1455660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=29566560" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=497-507&author=D.+B.+Tizauthor=D.+Kikeljauthor=N.+Zidar&title=Overcoming+problems+of+poor+drug+penetration+into+bacteria%3A+challenges+and+strategies+for+medicinal+chemists&doi=10.1080%2F17460441.2018.1455660"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F17460441.2018.1455660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2018.1455660%26sid%3Dliteratum%253Aachs%26aulast%3DTiz%26aufirst%3DD.%2BB.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DZidar%26aufirst%3DN.%26atitle%3DOvercoming%2520problems%2520of%2520poor%2520drug%2520penetration%2520into%2520bacteria%253A%2520challenges%2520and%2520strategies%2520for%2520medicinal%2520chemists%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2018%26volume%3D13%26spage%3D497%26epage%3D507%26doi%3D10.1080%2F17460441.2018.1455660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durcik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span> <span> </span><span class="NLM_article-title">ATP competitive DNA gyrase and topoisomerase IV inhibitors as antibacterial agents</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1080/13543776.2019.1575362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1080%2F13543776.2019.1575362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=30686070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=171-180&author=M.+Durcikauthor=T.+Toma%C5%A1i%C4%8Dauthor=N.+Zidarauthor=A.+Zegaauthor=D.+Kikeljauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1&title=ATP+competitive+DNA+gyrase+and+topoisomerase+IV+inhibitors+as+antibacterial+agents&doi=10.1080%2F13543776.2019.1575362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-competitive DNA gyrase and topoisomerase IV inhibitors as antibacterial agents</span></div><div class="casAuthors">Durcik, Martina; Tomasic, Tihomir; Zidar, Nace; Zega, Anamarija; Kikelj, Danijel; Masic, Lucija Peterlin; Ilas, Janez</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">171-180</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The bacterial topoisomerases DNA gyrase and topoisomerase IV are validated targets for development of novel antibacterial agents.  Fluoroquinolones inhibit the catalytic GyrA and/or ParC(GrlA) subunit and have been commonly used, although these have toxicity liabilities that restrict their use.  The ATPase GyrB and ParE(GrlB) subunits have been much less explored and after withdrawal of novobiocin, there are no further marketed inhibitors.  ATP-competitive inhibitors of GyrB and/or ParE(GrlB) are of special interest, as this target has been validated, and it is expected that many of the problems assocd. with fluoroquinolones can be avoided.  This review summarises the development of ATP-competitive inhibitors of GyrB and/or ParE(GrlB) as novel antibacterial agents over the last 10 years.  Structural features of the new inhibitors and their optimization strategies are highlighted.  The development of novel ATP-competitive inhibitors of GyrB and/or ParE(GrlB) is ongoing in industrial and academical research.  Development of resistance is one of the most problematic issues, but GyrB/ParE(GrlB) inhibitors do not show cross-resistance with fluoroquinolones.  Other common issues, such as low soly., high protein binding, development of off-target resistance, are related to the structures of the inhibitors themselves, which is thus a main focus of design strategies.  With some now in early clin. development, there is reasonable expectation that novel ATP-competitive inhibitors of GyrB/ParE(GrlB) will reach the market in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPIQu6ZReEKbVg90H21EOLACvtfcHk0ljBkQPf6SjrrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D&md5=2b5f0b8ab2036bee44202ae45b95a9a5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1575362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1575362%26sid%3Dliteratum%253Aachs%26aulast%3DDurcik%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26atitle%3DATP%2520competitive%2520DNA%2520gyrase%2520and%2520topoisomerase%2520IV%2520inhibitors%2520as%2520antibacterial%2520agents%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2019%26volume%3D29%26spage%3D171%26epage%3D180%26doi%3D10.1080%2F13543776.2019.1575362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hough, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akers-Rodriguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown-Driver, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillar, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinabarger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Tricyclic GyrB/ParE (TriBE) inhibitors: a new class of broad-spectrum dual-targeting antibacterial agents</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e84409</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0084409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1371%2Fjournal.pone.0084409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=24386374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=L.+W.+Tariauthor=X.+Liauthor=M.+Trzossauthor=D.+C.+Bensenauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=S.+J.+Leeauthor=G.+Houghauthor=D.+Phillipsonauthor=S.+Akers-Rodriguezauthor=M.+L.+Cunninghamauthor=B.+P.+Kwanauthor=K.+L.+Nelsonauthor=A.+Castellanoauthor=J.+B.+Lockeauthor=V.+Brown-Driverauthor=T.+M.+Murphyauthor=V.+S.+Ongauthor=C.+M.+Pillarauthor=D.+L.+Shinabargerauthor=J.+Nixauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=K.+J.+Shawauthor=J.+Finn&title=Tricyclic+GyrB%2FParE+%28TriBE%29+inhibitors%3A+a+new+class+of+broad-spectrum+dual-targeting+antibacterial+agents&doi=10.1371%2Fjournal.pone.0084409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic GyrB/ParE (TriBE) inhibitors: a new class of broad-spectrum dual-targeting antibacterial agents</span></div><div class="casAuthors">Tari, Leslie W.; Li, Xiaoming; Trzoss, Michael; Bensen, Daniel C.; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Lee, Suk Joong; Hough, Grayson; Phillipson, Doug; Akers, Rodriguez Suzanne; Cunningham, Mark L.; Kwan, Bryan P.; Nelson, Kirk J.; Castellano, Amanda; Locke, Jeff B.; Brown-Driver, Vickie; Murphy, Timothy M.; Ong, Voon S.; Pillar, Chris M.; Shinabarger, Dean L.; Nix, Jay; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Shaw, Karen J.; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e84409/1-e84409/13, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increasing resistance to every major class of antibiotics and a dearth of novel classes of antibacterial agents in development pipelines has created a dwindling reservoir of treatment options for serious bacterial infections.  The bacterial type IIA topoisomerases, DNA gyrase and topoisomerase IV, are validated antibacterial drug targets with multiple prospective drug binding sites, including the catalytic site targeted by the fluoroquinolone antibiotics.  However, growing resistance to fluoroquinolones, frequently mediated by mutations in the drug-binding site, is increasingly limiting the utility of this antibiotic class, prompting the search for other inhibitor classes that target different sites on the topoisomerase complexes.  The highly conserved ATP-binding subunits of DNA gyrase (GyrB) and topoisomerase IV (ParE) have long been recognized as excellent candidates for the development of dual-targeting antibacterial agents with broad-spectrum potential.  However, to date, no natural product or small mol. inhibitors targeting these sites have succeeded in the clinic, and no inhibitors of these enzymes have yet been reported with broad-spectrum antibacterial activity encompassing the majority of Gram-neg. pathogens.  Using structure-based drug design (SBDD), we have created a novel dual-targeting pyrimidoindole inhibitor series with exquisite potency against GyrB and ParE enzymes from a broad range of clin. important pathogens.  Inhibitors from this series demonstrate potent, broad-spectrum antibacterial activity against Gram-pos. and Gram-neg. pathogens of clin. importance, including fluoroquinolone resistant and multidrug resistant strains.  Lead compds. have been discovered with clin. potential; they are well tolerated in animals, and efficacious in Gram-neg. infection models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ITJvl7tDrrVg90H21EOLACvtfcHk0ljBkQPf6SjrrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D&md5=2467ba25da5cb55226d83714d59fd2b8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084409%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DHough%26aufirst%3DG.%26aulast%3DPhillipson%26aufirst%3DD.%26aulast%3DAkers-Rodriguez%26aufirst%3DS.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DNelson%26aufirst%3DK.%2BL.%26aulast%3DCastellano%26aufirst%3DA.%26aulast%3DLocke%26aufirst%3DJ.%2BB.%26aulast%3DBrown-Driver%26aufirst%3DV.%26aulast%3DMurphy%26aufirst%3DT.%2BM.%26aulast%3DOng%26aufirst%3DV.%2BS.%26aulast%3DPillar%26aufirst%3DC.%2BM.%26aulast%3DShinabarger%26aufirst%3DD.%2BL.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DTricyclic%2520GyrB%252FParE%2520%2528TriBE%2529%2520inhibitors%253A%2520a%2520new%2520class%2520of%2520broad-spectrum%2520dual-targeting%2520antibacterial%2520agents%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0084409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part I: Structural guided discovery and optimization of dual targeting agents with potent, broad-spectrum enzymatic acticity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1536</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmcl.2012.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23352267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1529-1536&author=L.+W.+Tariauthor=X.+Liauthor=M.+Trzossauthor=D.+C.+Bensenauthor=X.+Liauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=C.+J.+Creightonauthor=M.+L.+Cunninghamauthor=B.+Kwanauthor=M.+Stidhamauthor=K.+J.+Shawauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=J.+Nixauthor=J.+Finn&title=Pyrrolopyrimidine+inhibitors+of+DNA+gyrase+B+%28GyrB%29+and+topoisomerase+IV+%28ParE%29.+Part+I%3A+Structural+guided+discovery+and+optimization+of+dual+targeting+agents+with+potent%2C+broad-spectrum+enzymatic+acticity&doi=10.1016%2Fj.bmcl.2012.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part I: Structure guided discovery and optimization of dual targeting agents with potent, broad-spectrum enzymatic activity</span></div><div class="casAuthors">Tari, Leslie W.; Trzoss, Michael; Bensen, Daniel C.; Li, Xiaoming; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Creighton, Christopher J.; Cunningham, Mark L.; Kwan, Bryan; Stidham, Mark; Shaw, Karen J.; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Nix, Jay; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1529-1536</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bacterial topoisomerases DNA gyrase (GyrB) and topoisomerase IV (ParE) are essential enzymes that control the topol. state of DNA during replication.  The high degree of conservation in the ATP-binding pockets of these enzymes make them appealing targets for broad-spectrum inhibitor development.  A pyrrolopyrimidine scaffold was identified from a pharmacophore-based fragment screen with optimization potential.  Structural characterization of inhibitor complexes conducted using selected GyrB/ParE orthologs aided in the identification of important steric, dynamic and compositional differences in the ATP-binding pockets of the targets, enabling the design of highly potent pyrrolopyrimidine inhibitors with broad enzymic spectrum and dual targeting activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn5kRWxZVjhbVg90H21EOLACvtfcHk0ljBkQPf6SjrrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D&md5=b8d572d3dc5320c2ec385668b3f60387</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%26aulast%3DStidham%26aufirst%3DM.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DPyrrolopyrimidine%2520inhibitors%2520of%2520DNA%2520gyrase%2520B%2520%2528GyrB%2529%2520and%2520topoisomerase%2520IV%2520%2528ParE%2529.%2520Part%2520I%253A%2520Structural%2520guided%2520discovery%2520and%2520optimization%2520of%2520dual%2520targeting%2520agents%2520with%2520potent%252C%2520broad-spectrum%2520enzymatic%2520acticity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1529%26epage%3D1536%26doi%3D10.1016%2Fj.bmcl.2012.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown-Driver, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part II: Development of inhibitors with broad-spectrum, Gram-negative antibacterial activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1543</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.11.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmcl.2012.11.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23294697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1537-1543&author=M.+Trzossauthor=D.+C.+Bensenauthor=X.+Liauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=C.+J.+Creightonauthor=M.+L.+Cunninghamauthor=B.+Kwanauthor=M.+Stidhamauthor=K.+Nelsonauthor=V.+Brown-Driverauthor=A.+Castellanoauthor=K.+J.+Shawauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=J.+Nixauthor=J.+Finnauthor=L.+W.+Tari&title=Pyrrolopyrimidine+inhibitors+of+DNA+gyrase+B+%28GyrB%29+and+topoisomerase+IV+%28ParE%29.+Part+II%3A+Development+of+inhibitors+with+broad-spectrum%2C+Gram-negative+antibacterial+activity&doi=10.1016%2Fj.bmcl.2012.11.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE), Part II: Development of inhibitors with broad spectrum, Gram-negative antibacterial activity</span></div><div class="casAuthors">Trzoss, Micheal; Bensen, Daniel C.; Li, Xiaoming; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Creighton, Christopher J.; Cunningham, Mark L.; Kwan, Bryan; Stidham, Mark; Nelson, Kirk; Brown-Driver, Vickie; Castellano, Amanda; Shaw, Karen J.; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Finn, John; Tari, Leslie W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1537-1543</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The structurally related bacterial topoisomerases DNA gyrase (GyrB) and topoisomerase IV (ParE) have long been recognized as prime candidates for the development of broad spectrum antibacterial agents.  However, GyrB/ParE targeting antibacterials with spectrum that encompasses robust Gram-neg. pathogens have not yet been reported.  Using structure-based inhibitor design, we optimized a novel pyrrolopyrimidine inhibitor series with potent, dual targeting activity against GyrB and ParE.  Compds. were discovered with broad antibacterial spectrum, including activity against Pseudomonas aeruginosa, Acinetobacter baumannii and Escherichia coli.  Here, the authors describe the SAR of the pyrrolopyrimidine series as it relates to key structural and electronic features necessary for Gram-neg. antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaEjbnVIUQM7Vg90H21EOLACvtfcHk0lhF_xhNueCuAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCgtA%253D%253D&md5=f54bbe3030bfcb30e1585a1029583b99</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.11.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.11.073%26sid%3Dliteratum%253Aachs%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%26aulast%3DStidham%26aufirst%3DM.%26aulast%3DNelson%26aufirst%3DK.%26aulast%3DBrown-Driver%26aufirst%3DV.%26aulast%3DCastellano%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DFinn%26aufirst%3DJ.%26aulast%3DTari%26aufirst%3DL.%2BW.%26atitle%3DPyrrolopyrimidine%2520inhibitors%2520of%2520DNA%2520gyrase%2520B%2520%2528GyrB%2529%2520and%2520topoisomerase%2520IV%2520%2528ParE%2529.%2520Part%2520II%253A%2520Development%2520of%2520inhibitors%2520with%2520broad-spectrum%252C%2520Gram-negative%2520antibacterial%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1537%26epage%3D1543%26doi%3D10.1016%2Fj.bmcl.2012.11.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poh, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liew, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendharkar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangthongpitag, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basarab, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherian, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of dual GyrB/ParE inhibitors active against gram-negative bacteria</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.ejmech.2018.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=30125722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2018&pages=610-621&author=S.+Y.+Hoauthor=W.+Wangauthor=F.+M.+Ngauthor=Y.+X.+Wongauthor=Z.+Y.+Pohauthor=S.+W.+W.+Tanauthor=S.+H.+Angauthor=S.+S.+Liewauthor=Y.+S.+J.+Wongauthor=Y.+Tanauthor=A.+Poulsenauthor=V.+Pendharkarauthor=K.+Sangthongpitagauthor=J.+Manchesterauthor=G.+Basarabauthor=J.+Hillauthor=T.+H.+Kellerauthor=J.+Cherian&title=Discovery+of+dual+GyrB%2FParE+inhibitors+active+against+gram-negative+bacteria&doi=10.1016%2Fj.ejmech.2018.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of dual GyrB/ParE inhibitors active against Gram-negative bacteria</span></div><div class="casAuthors">Ho, Soo Yei; Wang, Weiling; Ng, Fui Mee; Wong, Yun Xuan; Poh, Zhi Ying; Tan, Sum Wai Eldwin; Ang, Shi Hua; Liew, Si Si; Joyner Wong, Yin Sze; Tan, Yvonne; Poulsen, Anders; Pendharkar, Vishal; Sangthongpitag, Kanda; Manchester, John; Basarab, Gregory; Hill, Jeffrey; Keller, Thomas H.; Cherian, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">610-621</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Even though many GyrB and ParE inhibitors have been reported in the literature, few possess activity against Gram-neg. bacteria.  This is primarily due to limited permeability across Gram-neg. bacterial membrane as well as bacterial efflux mechanisms.  Permeability of compds. across Gram-neg. bacterial membranes depends on many factors including physicochem. properties of the inhibitors.  Herein, we show the optimization of pyridylureas leading to compds. with potent activity against Gram-neg. bacterial species such as P.aeruginosa, E.coli and A.baumannii.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-A5-xjaiNHbVg90H21EOLACvtfcHk0lhF_xhNueCuAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O&md5=89908d086984c0df2588ca862abeada3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DS.%2BY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DNg%26aufirst%3DF.%2BM.%26aulast%3DWong%26aufirst%3DY.%2BX.%26aulast%3DPoh%26aufirst%3DZ.%2BY.%26aulast%3DTan%26aufirst%3DS.%2BW.%2BW.%26aulast%3DAng%26aufirst%3DS.%2BH.%26aulast%3DLiew%26aufirst%3DS.%2BS.%26aulast%3DWong%26aufirst%3DY.%2BS.%2BJ.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DPendharkar%26aufirst%3DV.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DManchester%26aufirst%3DJ.%26aulast%3DBasarab%26aufirst%3DG.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DCherian%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520dual%2520GyrB%252FParE%2520inhibitors%2520active%2520against%2520gram-negative%2520bacteria%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D157%26spage%3D610%26epage%3D621%26doi%3D10.1016%2Fj.ejmech.2018.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span> <span> </span><span class="NLM_article-title">Physicochemical properties of antibacterial compounds: implications for drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2871</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1021/jm700967e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700967e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2871-2878&author=R.+O%E2%80%99Sheaauthor=H.+E.+Moser&title=Physicochemical+properties+of+antibacterial+compounds%3A+implications+for+drug+discovery&doi=10.1021%2Fjm700967e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical properties of antibacterial compounds: implications for drug discovery</span></div><div class="casAuthors">O'Shea, Rosemarie; Moser, Heinz E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2871-2878</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of multidrug-resistant pathogens and the need for novel antibiotics, it is crit. to understand as much as possible from prior efforts and to apply learned lessons to the discovery of future antibiotics.  One important parameter in particular has previously been mentioned but, in the view, not sufficiently analyzed: the physicochem. property space of antibacterial drugs.  The authors selected 147 antibacterially active compds. that encompass both currently used drugs and compds. that are still under clin. investigation (see Methods for details).  Where available, other property values were extd. from the literature, including protein binding and oral bioavailability in humans.  This anal. suggests that natural products should be increasingly investigated again to identify novel antibacterial hits.  Besides their high level of structural diversity, they are likely to better cover the required physicochem. property space for antibacterial compds. compared to synthetic mols. because of an increased d. of polar functionalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25EGWMkWdk7Vg90H21EOLACvtfcHk0lhF_xhNueCuAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D&md5=8aae7d962df53b70113e0fe6169abd68</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm700967e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700967e%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DR.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26atitle%3DPhysicochemical%2520properties%2520of%2520antibacterial%2520compounds%253A%2520implications%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2871%26epage%3D2878%26doi%3D10.1021%2Fjm700967e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May-Dracka, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span> <span> </span><span class="NLM_article-title">Trends and exceptions of physical properties on antibacterial activity for gram-positive and gram-negative pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10144</span>– <span class="NLM_lpage">10161</span>, <span class="refDoi"> DOI: 10.1021/jm501552x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501552x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10144-10161&author=D.+G.+Brownauthor=T.+L.+May-Drackaauthor=M.+M.+Gagnonauthor=R.+Tommasi&title=Trends+and+exceptions+of+physical+properties+on+antibacterial+activity+for+gram-positive+and+gram-negative+pathogens&doi=10.1021%2Fjm501552x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Trends and Exceptions of Physical Properties on Antibacterial Activity for Gram-Positive and Gram-Negative Pathogens</span></div><div class="casAuthors">Brown, Dean G.; May-Dracka, Tricia L.; Gagnon, Moriah M.; Tommasi, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10144-10161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the difficulties surrounding the identification of novel antibacterial compds. from corporate screening collections, phys. properties of ∼3200 antibacterial project compds. with whole cell activity against Gram-neg. or Gram-pos. pathogens were profiled and compared to actives found from high throughput (HTS) screens conducted on both biochem. and phenotypic bacterial targets.  The output from 23 antibacterial HTS screens illustrated that when compared to the properties of the antibacterial project compds., the HTS actives were significantly more hydrophobic than antibacterial project compds. (typically 2-4 log units higher), and furthermore, for 14/23 HTS screens, the av. clogD was higher than the screening collection av. (screening collection clogD = 2.45).  It was found that the consequences of this were the following: (a) lead identification programs often further gained hydrophobic character with increased biochem. potency, making the sepn. even larger between the physicochem. properties of known antibacterial agents and the HTS active starting point, (b) the probability of plasma protein binding and cytotoxicity are often increased, and (c) cell-based activity in Gram-neg. bacteria was severely limited or, if present, demonstrated significant efflux.  Our anal. illustrated that compds. least susceptible to efflux were those which were highly polar and small in MW or very large and typically zwitterionic.  Hydrophobicity was often the dominant driver for HTS actives but, more often than not, precluded whole cell antibacterial activity.  However, simply designing polar compds. was not sufficient for antibacterial activity and pointed to a lack of understanding of complex and specific bacterial penetration mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-f01NZg_bsbVg90H21EOLACvtfcHk0ljYJlO4VxDkSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN&md5=e107961414dc8cbf696e2287ddf9362d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm501552x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501552x%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DMay-Dracka%26aufirst%3DT.%2BL.%26aulast%3DGagnon%26aufirst%3DM.%2BM.%26aulast%3DTommasi%26aufirst%3DR.%26atitle%3DTrends%2520and%2520exceptions%2520of%2520physical%2520properties%2520on%2520antibacterial%2520activity%2520for%2520gram-positive%2520and%2520gram-negative%2520pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10144%26epage%3D10161%26doi%3D10.1021%2Fjm501552x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dussault, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uria-Nickelsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioannidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hull, K. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel azaindole class of antibacterial agents targeting the ATPase domains of DNA gyrase and Topoisomerase IV</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5150</span>– <span class="NLM_lpage">5156</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmcl.2012.05.128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=22814212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=5150-5156&author=J.+I.+Manchesterauthor=D.+D.+Dussaultauthor=J.+A.+Roseauthor=P.+A.+Boriack-Sjodinauthor=M.+Uria-Nickelsenauthor=G.+Ioannidisauthor=S.+Bistauthor=P.+Flemingauthor=K.+G.+Hull&title=Discovery+of+a+novel+azaindole+class+of+antibacterial+agents+targeting+the+ATPase+domains+of+DNA+gyrase+and+Topoisomerase+IV&doi=10.1016%2Fj.bmcl.2012.05.128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel azaindole class of antibacterial agents targeting the ATPase domains of DNA gyrase and Topoisomerase IV</span></div><div class="casAuthors">Manchester, John I.; Dussault, Daemian D.; Rose, Jonathan A.; Boriack-Sjodin, P. Ann; Uria-Nickelsen, Maria; Ioannidis, Georgine; Bist, Shanta; Fleming, Paul; Hull, Kenneth G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5150-5156</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery and optimization of a series of bacterial topoisomerase inhibitors, e.g., I, is disclosed.  Starting from a virtual screening hit, activity was optimized through a combination of structure-based design and phys. property optimization.  Synthesis of fewer than a dozen compds. was required to achieve inhibition of the growth of methicillin-resistant Staphyloccus aureus at compd. concns. of 1.56 μM.  These compds. simultaneously inhibit DNA gyrase and Topoisomerase IV at similar nanomolar concns., reducing the likelihood of the spontaneous occurrence of target-based mutations resulting in antibiotic resistance, an increasing threat in the treatment of serious infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm-t-_e2xcSbVg90H21EOLACvtfcHk0ljYJlO4VxDkSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7bL&md5=7c2e42ef7a21c7805736065e84d86a49</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.128%26sid%3Dliteratum%253Aachs%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DDussault%26aufirst%3DD.%2BD.%26aulast%3DRose%26aufirst%3DJ.%2BA.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DIoannidis%26aufirst%3DG.%26aulast%3DBist%26aufirst%3DS.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DHull%26aufirst%3DK.%2BG.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520azaindole%2520class%2520of%2520antibacterial%2520agents%2520targeting%2520the%2520ATPase%2520domains%2520of%2520DNA%2520gyrase%2520and%2520Topoisomerase%2520IV%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D5150%26epage%3D5156%26doi%3D10.1016%2Fj.bmcl.2012.05.128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1435</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1111/nyas.13598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1111%2Fnyas.13598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=29446459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1435&publication_year=2019&pages=18-38&author=M.+F.+Richterauthor=P.+J.+Hergenrother&title=The+challenge+of+converting+Gram-positive-only+compounds+into+broad-spectrum+antibiotics&doi=10.1111%2Fnyas.13598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span></div><div class="casAuthors">Richter, Michelle F.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1435</span>
        (<span class="NLM_cas:issue">1, Antimicrobial Therapeutics Reviews</span>),
    <span class="NLM_cas:pages">18-38</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistant Gram-neg. bacterial infections are on the rise, and there is a lack of new classes of drugs to treat these pathogens.  This drug shortage is largely due to the challenge of finding antibiotics that can permeate and persist inside Gram-neg. species.  Efforts to understand the mol. properties that enable certain compds. to accumulate in Gram-neg. bacteria based on retrospective studies of known antibiotics have not been generally actionable in the development of new antibiotics.  A recent assessment of the ability of >180 diverse small mols. to accumulate in Escherichia coli led to predictive guidelines for compd. accumulation in E. coli.  These "eNTRy rules" state that compds. are most likely to accumulate if they contain a nonsterically encumbered ionizable Nitrogen (primary amines are the best), have low Three-dimensionality (globularity ≤ 0.25), and are relatively Rigid (rotatable bonds ≤ 5).  In this review, we look back through 50+ years of antibacterial research and 1000s of derivs. and assess this historical data set through the lens of these predictive guidelines.  The results are consistent with the eNTRy rules, suggesting that the eNTRy rules may provide an actionable and general roadmap for the conversion of Gram-pos.-only compds. into broad-spectrum antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX3zbgiAcHdbVg90H21EOLACvtfcHk0ljYJlO4VxDkSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D&md5=12fffe0d10e7d3e5c134ac28ce929ab3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fnyas.13598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fnyas.13598%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520challenge%2520of%2520converting%2520Gram-positive-only%2520compounds%2520into%2520broad-spectrum%2520antibiotics%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2019%26volume%3D1435%26spage%3D18%26epage%3D38%26doi%3D10.1111%2Fnyas.13598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+compound+accumulation+rules+yield+a+broad-spectrum+antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lj0q7TVUePC_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520compound%2520accumulation%2520rules%2520yield%2520a%2520broad-spectrum%2520antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seelig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assmus, F.</span></span> <span> </span><span class="NLM_article-title">Carrier mediated distribution system (CAMDIS): a new approach for the measurement of octanol/water distribution coefficients</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2014.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.ejps.2014.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=25513709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVChtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2015&pages=68-77&author=B.+Wagnerauthor=H.+Fischerauthor=M.+Kansyauthor=A.+Seeligauthor=F.+Assmus&title=Carrier+mediated+distribution+system+%28CAMDIS%29%3A+a+new+approach+for+the+measurement+of+octanol%2Fwater+distribution+coefficients&doi=10.1016%2Fj.ejps.2014.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Carrier Mediated Distribution System (CAMDIS): A new approach for the measurement of octanol/water distribution coefficients</span></div><div class="casAuthors">Wagner, Bjoern; Fischer, Holger; Kansy, Manfred; Seelig, Anna; Assmus, Frauke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">68-77</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here we present a miniaturized assay, referred to as Carrier-Mediated Distribution System (CAMDIS) for fast and reliable measurement of octanol/water distribution coeffs., log Doct.  By introducing a filter support for octanol, phase sepn. from water is facilitated and the tendency of emulsion formation (emulsification) at the interface is reduced.  A guideline for the best practice of CAMDIS is given, describing a strategy to manage drug adsorption at the filter-supported octanol/buffer interface.  We validated the assay on a set of 52 structurally diverse drugs with known shake flask logDoct values.  Excellent agreement with literature data (r2 = 0.996, std. error of est., SEE = 0.111), high reproducibility (std. deviation, SD < 0.1 log Doct units), minimal sample consumption (10 μL of 100 μM DMSO stock soln.) and a broad anal. range (log Doct range = -0.5 to 4.2) make CAMDIS a valuable tool for the high-throughput assessment of log Doct.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsNSPSNv4bP7Vg90H21EOLACvtfcHk0lj0q7TVUePC_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVChtQ%253D%253D&md5=e0631985c73afe8838fce2eb24ccf686</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2014.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2014.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DFischer%26aufirst%3DH.%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSeelig%26aufirst%3DA.%26aulast%3DAssmus%26aufirst%3DF.%26atitle%3DCarrier%2520mediated%2520distribution%2520system%2520%2528CAMDIS%2529%253A%2520a%2520new%2520approach%2520for%2520the%2520measurement%2520of%2520octanol%252Fwater%2520distribution%2520coefficients%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2015%26volume%3D68%26spage%3D68%26epage%3D77%26doi%3D10.1016%2Fj.ejps.2014.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadri, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjemian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaulding, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricotta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prevots, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmore, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klompas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers III, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suffredini, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridkin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danner, R. L.</span></span> <span> </span><span class="NLM_article-title">Difficult-to-treat resistance in gram-negative bacteremia at 173 US hospitials: retrospective cohort analysis of prevalence, predictors, and outcome of resistnace to all first-line agents</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">1803</span>– <span class="NLM_lpage">1814</span>, <span class="refDoi"> DOI: 10.1093/cid/ciy378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1093%2Fcid%2Fciy378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=30052813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A280%3ADC%252BB3c7is1ensw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=1803-1814&author=S.+S.+Kadriauthor=J.+Adjemianauthor=Y.+L.+Laiauthor=A.+B.+Spauldingauthor=E.+Ricottaauthor=D.+R.+Prevotsauthor=T.+N.+Palmoreauthor=C.+Rheeauthor=M.+Klompasauthor=J.+P.+Dekkerauthor=J.+H.+Powers+IIIauthor=A.+F.+Suffrediniauthor=D.+C.+Hooperauthor=S.+Fridkinauthor=R.+L.+Danner&title=Difficult-to-treat+resistance+in+gram-negative+bacteremia+at+173+US+hospitials%3A+retrospective+cohort+analysis+of+prevalence%2C+predictors%2C+and+outcome+of+resistnace+to+all+first-line+agents&doi=10.1093%2Fcid%2Fciy378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Difficult-to-Treat Resistance in Gram-negative Bacteremia at 173 US Hospitals: Retrospective Cohort Analysis of Prevalence, Predictors, and Outcome of Resistance to All First-line Agents</span></div><div class="casAuthors">Kadri Sameer S; Suffredini Anthony F; Danner Robert L; Kadri Sameer S; Hooper David C; Adjemian Jennifer; Lai Yi Ling; Spaulding Alicen B; Ricotta Emily; Prevots D Rebecca; Adjemian Jennifer; Palmore Tara N; Rhee Chanu; Klompas Michael; Rhee Chanu; Klompas Michael; Dekker John P; Powers John H 3rd; Fridkin Scott</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1803-1814</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  Resistance to all first-line antibiotics necessitates the use of less effective or more toxic "reserve" agents.  Gram-negative bloodstream infections (GNBSIs) harboring such difficult-to-treat resistance (DTR) may have higher mortality than phenotypes that allow for ≥1 active first-line antibiotic.  Methods:  The Premier Database was analyzed for inpatients with select GNBSIs.  DTR was defined as intermediate/resistant in vitro to all ss-lactam categories, including carbapenems and fluoroquinolones.  Prevalence and aminoglycoside resistance of DTR episodes were compared with carbapenem-resistant, extended-spectrum cephalosporin-resistant, and fluoroquinolone-resistant episodes using CDC definitions.  Predictors of DTR were identified.  The adjusted relative risk (aRR) of mortality was examined for DTR, CDC-defined phenotypes susceptible to ≥1 first-line agent, and graded loss of active categories.  Results:  Between 2009-2013, 471 (1%) of 45011 GNBSI episodes at 92 (53.2%) of 173 hospitals exhibited DTR, ranging from 0.04% for Escherichia coli to 18.4% for Acinetobacter baumannii.  Among patients with DTR, 79% received parenteral aminoglycosides, tigecycline, or colistin/polymyxin-B; resistance to all aminoglycosides occurred in 33%.  Predictors of DTR included urban healthcare and higher baseline illness.  Crude mortality for GNBSIs with DTR was 43%; aRR was higher for DTR than for carbapenem-resistant (1.2; 95% confidence interval, 1.0-1.4; P = .02), extended-spectrum cephalosporin-resistant (1.2; 1.1-1.4; P = .001), or fluoroquinolone-resistant (1.2; 1.0-1.4; P = .008) infections.  The mortality aRR increased 20% per graded loss of active first-line categories, from 3-5 to 1-2 to 0.  Conclusion:  Nonsusceptibility to first-line antibiotics is associated with decreased survival in GNBSIs.  DTR is a simple bedside prognostic measure of treatment-limiting coresistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2slWy-KRGedHxxy173TfNfW6udTcc2ebZpgN6F2Y-m7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7is1ensw%253D%253D&md5=16bfa27705be420ac185817d3d9a5d8e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fcid%2Fciy378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcid%252Fciy378%26sid%3Dliteratum%253Aachs%26aulast%3DKadri%26aufirst%3DS.%2BS.%26aulast%3DAdjemian%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DY.%2BL.%26aulast%3DSpaulding%26aufirst%3DA.%2BB.%26aulast%3DRicotta%26aufirst%3DE.%26aulast%3DPrevots%26aufirst%3DD.%2BR.%26aulast%3DPalmore%26aufirst%3DT.%2BN.%26aulast%3DRhee%26aufirst%3DC.%26aulast%3DKlompas%26aufirst%3DM.%26aulast%3DDekker%26aufirst%3DJ.%2BP.%26aulast%3DPowers%2BIII%26aufirst%3DJ.%2BH.%26aulast%3DSuffredini%26aufirst%3DA.%2BF.%26aulast%3DHooper%26aufirst%3DD.%2BC.%26aulast%3DFridkin%26aufirst%3DS.%26aulast%3DDanner%26aufirst%3DR.%2BL.%26atitle%3DDifficult-to-treat%2520resistance%2520in%2520gram-negative%2520bacteremia%2520at%2520173%2520US%2520hospitials%253A%2520retrospective%2520cohort%2520analysis%2520of%2520prevalence%252C%2520predictors%252C%2520and%2520outcome%2520of%2520resistnace%2520to%2520all%2520first-line%2520agents%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2018%26volume%3D67%26spage%3D1803%26epage%3D1814%26doi%3D10.1093%2Fcid%2Fciy378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Qidi Kong, Wei Pan, Heng Xu, Yaru Xue, Bin Guo, Xin Meng, Cheng Luo, Ting Wang, Shuhua Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Yushe Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of Novel Pyrimido[4,5-b]indole Derivatives Against Gram-Negative Multidrug-Resistant Pathogens. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8644-8665. <a href="https://doi.org/10.1021/acs.jmedchem.1c00621" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00621</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00621%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252BNovel%252BPyrimido%25255B4%25252C5-b%25255Dindole%252BDerivatives%252BAgainst%252BGram-Negative%252BMultidrug-Resistant%252BPathogens%26aulast%3DKong%26aufirst%3DQidi%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D05042021%26date%3D03062021%26volume%3D64%26issue%3D12%26spage%3D8644%26epage%3D8665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rui Ma, Xiaoxue Wang, Qiaoyu Zhang, Lei Chen, Jian Gao, Jie Feng, Donghui Wei, <span class="NLM_string-name hlFld-ContribAuthor">Ding Du</span>. </span><span class="cited-content_cbyCitation_article-title">Atroposelective Synthesis of Axially Chiral 4-Aryl α-Carbolines via N-Heterocyclic Carbene Catalysis. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2021,</strong> <em>23 </em>
                                    (11)
                                     , 4267-4272. <a href="https://doi.org/10.1021/acs.orglett.1c01221" title="DOI URL">https://doi.org/10.1021/acs.orglett.1c01221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.1c01221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.1c01221%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DAtroposelective%252BSynthesis%252Bof%252BAxially%252BChiral%252B4-Aryl%252B%2525CE%2525B1-Carbolines%252Bvia%252BN-Heterocyclic%252BCarbene%252BCatalysis%26aulast%3DMa%26aufirst%3DRui%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D10042021%26date%3D11052021%26volume%3D23%26issue%3D11%26spage%3D4267%26epage%3D4272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kristen A. Muñoz, <span class="NLM_string-name hlFld-ContribAuthor">Paul J. Hergenrother</span>. </span><span class="cited-content_cbyCitation_article-title">Facilitating Compound Entry as a Means to Discover Antibiotics for Gram-Negative Bacteria. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2021,</strong> <em>54 </em>
                                    (6)
                                     , 1322-1333. <a href="https://doi.org/10.1021/acs.accounts.0c00895" title="DOI URL">https://doi.org/10.1021/acs.accounts.0c00895</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.0c00895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.0c00895%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DFacilitating%252BCompound%252BEntry%252Bas%252Ba%252BMeans%252Bto%252BDiscover%252BAntibiotics%252Bfor%252BGram-Negative%252BBacteria%26aulast%3DMu%25C3%25B1oz%26aufirst%3DKristen%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D04012021%26date%3D26022021%26volume%3D54%26issue%3D6%26spage%3D1322%26epage%3D1333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sarah J. Perlmutter, Emily J. Geddes, Bryon S. Drown, Stephen E. Motika, Myung Ryul Lee, <span class="NLM_string-name hlFld-ContribAuthor">Paul J. Hergenrother</span>. </span><span class="cited-content_cbyCitation_article-title">Compound Uptake into E. coli Can Be Facilitated by N-Alkyl Guanidiniums and Pyridiniums. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2021,</strong> <em>7 </em>
                                    (1)
                                     , 162-173. <a href="https://doi.org/10.1021/acsinfecdis.0c00715" title="DOI URL">https://doi.org/10.1021/acsinfecdis.0c00715</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00715%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DCompound%252BUptake%252Binto%252BE.%2525C2%2525A0coli%252BCan%252BBe%252BFacilitated%252Bby%252BN-Alkyl%252BGuanidiniums%252Band%252BPyridiniums%26aulast%3DPerlmutter%26aufirst%3DSarah%2BJ.%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D13102020%26date%3D23112020%26volume%3D7%26issue%3D1%26spage%3D162%26epage%3D173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Yu</span>, <span class="hlFld-ContribAuthor ">Junsong  Guo</span>, <span class="hlFld-ContribAuthor ">Zhengjun  Cai</span>, <span class="hlFld-ContribAuthor ">Yingchen  Ju</span>, <span class="hlFld-ContribAuthor ">Jun  Xu</span>, <span class="hlFld-ContribAuthor ">Qiong  Gu</span>, <span class="hlFld-ContribAuthor ">Huihao  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of new building blocks by fragment screening for discovering GyrB inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>114 </em>, 105040. <a href="https://doi.org/10.1016/j.bioorg.2021.105040" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105040%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnew%252Bbuilding%252Bblocks%252Bby%252Bfragment%252Bscreening%252Bfor%252Bdiscovering%252BGyrB%252Binhibitors%26aulast%3DYu%26aufirst%3DYing%26date%3D2021%26volume%3D114%26spage%3D105040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Henni-Karoliina  Ropponen</span>, <span class="hlFld-ContribAuthor ">Eleonora  Diamanti</span>, <span class="hlFld-ContribAuthor ">Alexandra  Siemens</span>, <span class="hlFld-ContribAuthor ">Boris  Illarionov</span>, <span class="hlFld-ContribAuthor ">Jörg  Haupenthal</span>, <span class="hlFld-ContribAuthor ">Markus  Fischer</span>, <span class="hlFld-ContribAuthor ">Matthias  Rottmann</span>, <span class="hlFld-ContribAuthor ">Matthias  Witschel</span>, <span class="hlFld-ContribAuthor ">Anna K. H.  Hirsch</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of the rules related to gaining activity against Gram-negative bacteria. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (4)
                                     , 593-601. <a href="https://doi.org/10.1039/D0MD00409J" title="DOI URL">https://doi.org/10.1039/D0MD00409J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00409J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00409J%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DAssessment%252Bof%252Bthe%252Brules%252Brelated%252Bto%252Bgaining%252Bactivity%252Bagainst%252BGram-negative%252Bbacteria%26aulast%3DRopponen%26aufirst%3DHenni-Karoliina%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D4%26spage%3D593%26epage%3D601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qidi  Kong</span>, <span class="hlFld-ContribAuthor ">Yushe  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in antibacterial agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>35 </em>, 127799. <a href="https://doi.org/10.1016/j.bmcl.2021.127799" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127799</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127799%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRecent%252Badvances%252Bin%252Bantibacterial%252Bagents%26aulast%3DKong%26aufirst%3DQidi%26date%3D2021%26volume%3D35%26spage%3D127799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Pyrido[2,3-b]indoles Targeting DNA Gyrase and Topisomerase IV. </span><span class="cited-content_cbyCitation_journal-name">Synfacts</span><span> <strong>2021,</strong>,, 0238. <a href="https://doi.org/10.1055/s-0040-1719366" title="DOI URL">https://doi.org/10.1055/s-0040-1719366</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/s-0040-1719366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fs-0040-1719366%26sid%3Dliteratum%253Aachs%26jtitle%3DSynfacts%26atitle%3DSynthesis%252Bof%252BPyrido%25255B2%25252C3-b%25255Dindoles%252BTargeting%252BDNA%252BGyrase%252Band%252BTopisomerase%252BIV%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D03%26spage%3D0238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oussama  Moussaoui</span>, <span class="hlFld-ContribAuthor ">Rajendra  Bhadane</span>, <span class="hlFld-ContribAuthor ">Riham  Sghyar</span>, <span class="hlFld-ContribAuthor ">El Mestafa  El Hadrami</span>, <span class="hlFld-ContribAuthor ">Soukaina  El Amrani</span>, <span class="hlFld-ContribAuthor ">Abdeslem  Ben Tama</span>, <span class="hlFld-ContribAuthor ">Youssef  Kandri Rodi</span>, <span class="hlFld-ContribAuthor ">Said  Chakroune</span>, <span class="hlFld-ContribAuthor ">Outi M. H.  Salo-Ahen</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Amino Acid Derivatives of Quinolines as Potential Antibacterial and Fluorophore Agents. </span><span class="cited-content_cbyCitation_journal-name">Scientia Pharmaceutica</span><span> <strong>2020,</strong> <em>88 </em>
                                    (4)
                                     , 57. <a href="https://doi.org/10.3390/scipharm88040057" title="DOI URL">https://doi.org/10.3390/scipharm88040057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/scipharm88040057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fscipharm88040057%26sid%3Dliteratum%253Aachs%26jtitle%3DScientia%2520Pharmaceutica%26atitle%3DNovel%252BAmino%252BAcid%252BDerivatives%252Bof%252BQuinolines%252Bas%252BPotential%252BAntibacterial%252Band%252BFluorophore%252BAgents%26aulast%3DMoussaoui%26aufirst%3DOussama%26date%3D2020%26date%3D2020%26volume%3D88%26issue%3D4%26spage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. A few GyrB/ParE dual-inhibitor scaffolds. (a) AZ GN-positive only azaindole ureas, <b>AZ10</b> and <b>AZ17</b>; (b) Trius’ GN-active tricyclic pyrimido[4,5-<i>b</i>]indole, <b>GP6</b>; and (c) ETC’s GN-active zwitterionic pyridine urea, <b>ETC44</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overlay of X-ray crystal structures of ATP binding sites of GyrB with bound <b>GP1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>, protein shown with green ribbon and carbons, ligand with cyan carbons) and ParE with bound <b>AZ10</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EMV">4EMV</a>, protein shown with gray ribbon and carbons, ligand with magenta carbons). Hydrogen bonding interactions as red lines and residue labels shown for structure 4KFG. Further residues and water molecules omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of novel tricyclic pyrido[2,3-<i>b</i>]indole scaffold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray structure of <i>P. aeruginosa</i> GyrB in complex with compound <b>12o</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S">6M1S</a>). Key protein residues shown as sticks with gray and ligand with magenta carbons. All water molecules except for structural water are hidden for clarity. Interactions are shown as color-coded dashed lines (H-bonds in red, vdW in yellow, cation−π stacking in blue, and salt bridge in light pink).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray cocrystal structures overlay between <b>GP1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>, protein shown with green ribbon and carbons, ligand with cyan carbons) and compound <b>12x</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1J">6M1J</a>, protein shown with gray ribbon and carbons, ligand with magenta carbons). Further residues and water molecules omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <i>In vivo</i> efficacy in neutropenic mouse thigh model. The plot shows the log unit CFU levels in <i>E. coli</i> ATCC 25922 after 12 h of infection with the treatment of either placebo (black dots), ciprofloxacin control (green dots), or compound <b>17r</b> (blue squares). The line for each animal group represents the mean, mpk is mg/kg, 2x is two doses with 3 h interval. Statistical significance (Dunnett’s multiple comparison test) vs placebo, one asterisk represents adjusted P values 0.01–0.05 and two asterisks <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Key Intermediate <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtNH<sub>2</sub>, EtOH, 0 °C, 3 h, 63% yield; (b) NaH, Boc<sub>2</sub>O, THF, 0 °C then rt, 16 h, 86% yield; (c) H<sub>2</sub> 50 psi, Pd/C, MeOH, rt, 18 h, 92% yield; (d) 2,3,5-trichloropyridine, Pd(OAc)<sub>2</sub> BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120 °C, 16 h, 74% yield; (e) Pd<sub>2</sub>(dba)<sub>3</sub>, X-Phos, DBU, DMA/o-xylene, 130 °C, 4 h, 51% yield; (f) m-CPBA, DCM, 30 °C; 12 h, then POCl<sub>3</sub>, DMF, −5 to 0 °C, 1 h, 42% yield.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Examplar Synthesis of Compounds <b>10a</b>–<b>u</b> and <b>12a</b>–<b>w</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and Conditions: (a) 3-(trifluoromethyl)pyrazole, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, 12 h, 86% yield; (b) <b>9d</b>, 3-cyanopyridine-5-boronic acid pinacol ester, Pd<sub>2</sub> db<sub>3</sub>, X-Phos, K<sub>3</sub>PO<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 12 h, 66% yield; (c) <b>10d</b>, TFA, DCM, rt, 2 h, 81% yield; (d) <b>11j</b>, (2-methoxypyrimidin-5-yl)boronic acid, Pd<sub>2</sub> db<sub>3</sub>, XPhos, K<sub>3</sub>PO<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 12 h, 64% yield; (e) <b>12j</b>, TFA, DCM, rt, 1 h, 40% yield.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/medium/jm0c00768_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Examplar Synthesis of <b>15a</b>–<b>z</b> and <b>17r</b>, <b>17u</b>, and <b>17x</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00768/20200903/images/large/jm0c00768_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00768&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>13r</b>, ethyl 3-(trifluoromethyl)-1H-pyrazole-4-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, 33% yield; (b) <b>14r</b>, 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)nicotinonitrile, CsF, Pd<sub>2</sub>(dba)<sub>3</sub>, X-phos; 1,4-dioxane/H<sub>2</sub>O, 110 °C, 12 h, 89% yield; NH<sub>4</sub>Cl, HOBt, PyBOP, DIPEA, DMF, 30 °C, 1 h, 96% yield; BH<sub>3</sub>-THF, 80 °C, 0.5 h, 27% yield; (c) <b>15r</b>, TFA, DCM, rt, 1 h, 75% yield; (d) <b>16x</b>, <i>tert</i>-butyl 4,6-dihydro-1H-pyrrolo[3,4-<i>c</i>]pyrazole-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMSO, 120 °C, 12 h, 93% yield; 2-methoxy-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrimidine, K<sub>3</sub>PO<sub>4</sub>, cataCXium A-Pd-G2, THF-H<sub>2</sub>O, 70 °C, 12 h, 93% yield; (e) <b>17x</b>, TFA, DCM, 30 °C, 1 h, 57% yield.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i162">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendleton, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, B. F.</span></span> <span> </span><span class="NLM_article-title">Clinical relevance of the ESKAPE pathogens</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1586/eri.13.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1586%2Feri.13.12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23458769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=297-308&author=J.+N.+Pendletonauthor=S.+P.+Gormanauthor=B.+F.+Gilmore&title=Clinical+relevance+of+the+ESKAPE+pathogens&doi=10.1586%2Feri.13.12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical relevance of the ESKAPE pathogens</span></div><div class="casAuthors">Pendleton, Jack N.; Gorman, Sean P.; Gilmore, Brendan F.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-308</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, the Infectious Diseases Society of America has highlighted a faction of antibiotic-resistant bacteria (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) - acronymically dubbed the ESKAPE pathogens' - capable of escaping' the biocidal action of antibiotics and mutually representing new paradigms in pathogenesis, transmission and resistance.  This review aims to consolidate clin. relevant background information on the ESKAPE pathogens and provide a contemporary summary of bacterial resistance, alongside pertinent microbiol. considerations necessary to face the mounting threat of antimicrobial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql_dEmz1UF37Vg90H21EOLACvtfcHk0lgYj2npSmBaKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKisbc%253D&md5=7168500f85d84236a255e20dbf18930a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1586%2Feri.13.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.13.12%26sid%3Dliteratum%253Aachs%26aulast%3DPendleton%26aufirst%3DJ.%2BN.%26aulast%3DGorman%26aufirst%3DS.%2BP.%26aulast%3DGilmore%26aufirst%3DB.%2BF.%26atitle%3DClinical%2520relevance%2520of%2520the%2520ESKAPE%2520pathogens%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2013%26volume%3D11%26spage%3D297%26epage%3D308%26doi%3D10.1586%2Feri.13.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santajit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indrawattana, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of antimicrobial resistance in ESKAPE pathogens</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">2475067</span>, <span class="refDoi"> DOI: 10.1155/2016/2475067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1155%2F2016%2F2475067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=27274985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=2475067&author=S.+Santajitauthor=N.+Indrawattana&title=Mechanisms+of+antimicrobial+resistance+in+ESKAPE+pathogens&doi=10.1155%2F2016%2F2475067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Antimicrobial Resistance in ESKAPE Pathogens</span></div><div class="casAuthors">Santajit Sirijan; Indrawattana Nitaya</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2016</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2475067</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) are the leading cause of nosocomial infections throughout the world.  Most of them are multidrug resistant isolates, which is one of the greatest challenges in clinical practice.  Multidrug resistance is amongst the top three threats to global public health and is usually caused by excessive drug usage or prescription, inappropriate use of antimicrobials, and substandard pharmaceuticals.  Understanding the resistance mechanisms of these bacteria is crucial for the development of novel antimicrobial agents or other alternative tools to combat these public health challenges.  Greater mechanistic understanding would also aid in the prediction of underlying or even unknown mechanisms of resistance, which could be applied to other emerging multidrug resistant pathogens.  In this review, we summarize the known antimicrobial resistance mechanisms of ESKAPE pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSM_xN5GU5d0ntlzo9xastyfW6udTcc2eaFIlo2qAyCvrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D&md5=e42353b6d72db948734b5354e31b501e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1155%2F2016%2F2475067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2016%252F2475067%26sid%3Dliteratum%253Aachs%26aulast%3DSantajit%26aufirst%3DS.%26aulast%3DIndrawattana%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520antimicrobial%2520resistance%2520in%2520ESKAPE%2520pathogens%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2016%26volume%3D2016%26spage%3D2475067%26doi%3D10.1155%2F2016%2F2475067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosen, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepore, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coukell, A.</span></span> <span> </span><span class="NLM_article-title">On the road to discovering urgently needed antibiotics: so close yet so far away</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1292</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00100</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00100" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVOksbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1292-1294&author=W.+Kimauthor=K.+R.+Prosenauthor=C.+J.+Leporeauthor=A.+Coukell&title=On+the+road+to+discovering+urgently+needed+antibiotics%3A+so+close+yet+so+far+away&doi=10.1021%2Facsinfecdis.0c00100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">On the Road to Discovering Urgently Needed Antibiotics: So Close Yet So Far Away</span></div><div class="casAuthors">Kim, Wes; Prosen, Katherine R.; Lepore, Cara J.; Coukell, Allan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1292-1294</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Pew Charitable Trusts' 2016 publication "A Scientific Roadmap for Antibiotic Discovery" provided a consensus approach to accelerating the discovery of novel antibiotics targeting Gram-neg. pathogens.  Since then, encouraging initiatives have launched to catalyze antibiotics discovery, particularly by improving knowledge sharing and making discovery efforts more efficient and effective.  However, because the global pipeline remains insufficient to address current and future unmet needs, existing initiatives are not enough.  Sustained public funding is crit., particularly as private funding continues to dwindle.  And with public funding comes the responsibility of sharing what has been learned.  Finally, a "precompetitive" R&D model in which the financial return on investment is not a primary driver warrants further consideration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw1Ad8k_kdTrVg90H21EOLACvtfcHk0lhLG_AP8VHLjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVOksbjL&md5=9a6e4c581251ba32ae0a8cd2742a9e61</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00100%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.%26aulast%3DProsen%26aufirst%3DK.%2BR.%26aulast%3DLepore%26aufirst%3DC.%2BJ.%26aulast%3DCoukell%26aufirst%3DA.%26atitle%3DOn%2520the%2520road%2520to%2520discovering%2520urgently%2520needed%2520antibiotics%253A%2520so%2520close%2520yet%2520so%2520far%2520away%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2020%26volume%3D6%26spage%3D1292%26epage%3D1294%26doi%3D10.1021%2Facsinfecdis.0c00100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">A gestalt approach to gram-negative entry</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">6379</span>– <span class="NLM_lpage">6389</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.06.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmc.2016.06.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=27381365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVaqur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=6379-6389&author=L.+L.+Silver&title=A+gestalt+approach+to+gram-negative+entry&doi=10.1016%2Fj.bmc.2016.06.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A Gestalt approach to Gram-negative entry</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6379-6389</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A major obstacle confronting the discovery and development of new antibacterial agents to combat resistant Gram-neg. (GN) organisms is the lack of a rational process for endowing compds. with properties that allow (or promote) entry into the bacterial cytoplasm.  The major permeability difference between GN and Gram-pos. (GP) bacteria is the GN outer membrane (OM) which is a permeability barrier itself and potentiates efflux pumps that expel compds.  Based on the fact that OM-permeable and efflux-deleted GNs are sensitive to many anti-GP drugs, recent efforts to approach the GN entry problem have focused on ways of avoiding efflux and transiting or compromising the OM, with the tacit assumption that this could allow entry of compds. into the GN cytoplasm.  But bypassing the OM and efflux obstacles does not take into account the addnl. requirement of penetrating the cytoplasmic membrane (CM) whose sieving properties appear to be orthogonal to that of the OM.  That is, tailoring compds. to transit the OM may well compromise their ability to enter the cytoplasm.  Thus, a Gestalt approach to understanding the chem. requirements for GN entry seems a useful adjunct.  This might consist of characterizing compds. which reach the cytoplasm, grouping (or binning) by routes of entry and formulating chem. rules for those bins.  This will require acquisition of data on large nos. of compds., using non-activity-dependent methods of measuring accumulation in the cytoplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqs-T6vYmkrrVg90H21EOLACvtfcHk0lhLG_AP8VHLjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVaqur%252FP&md5=08ca97526e7f3c8c8781307e52e96e9a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.06.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.06.044%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DA%2520gestalt%2520approach%2520to%2520gram-negative%2520entry%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D6379%26epage%3D6389%26doi%3D10.1016%2Fj.bmc.2016.06.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span>Nabriva
Therapeutics Receives U.S. <span> </span><span class="NLM_article-title">FDA Approval of Xenleta
(lefamulin) to Treat Community-Acquired Bacteria Pneumonia (CABP)</span> [news release]. <a href="http://investors.nabriva.com" class="extLink">http://investors.nabriva.com</a> (accessed <span class="NLM_year">2019-08-19</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Nabriva%0ATherapeutics+Receives+U.S.+FDA+Approval+of+Xenleta%0A%28lefamulin%29+to+Treat+Community-Acquired+Bacteria+Pneumonia+%28CABP%29+%5Bnews+release%5D.+http%3A%2F%2Finvestors.nabriva.com+%28accessed+2019-08-19%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DFDA%2520Approval%2520of%2520Xenleta%250A%2528lefamulin%2529%2520to%2520Treat%2520Community-Acquired%2520Bacteria%2520Pneumonia%2520%2528CABP%2529%26date%3D2019-08-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span> <span> </span><span class="NLM_article-title">Challenges of antibacterial discovery</span>. <i>Clin. Microbio. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1128/CMR.00030-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1128%2FCMR.00030-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=21233508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=71-109&author=L.+L.+Silver&title=Challenges+of+antibacterial+discovery&doi=10.1128%2FCMR.00030-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges of antibacterial discovery</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-109</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The discovery of novel small-mol. antibacterial drugs has been stalled for many years.  The purpose of this review is to underscore and illustrate those scientific problems unique to the discovery and optimization of novel antibacterial agents that have adversely affected the output of the effort.  The major challenges fall into two areas: (i) proper target selection, particularly the necessity of pursuing mol. targets that are not prone to rapid resistance development, and (ii) improvement of chem. libraries to overcome limitations of diversity, esp. that which is necessary to overcome barriers to bacterial entry and proclivity to be effluxed, esp. in Gram-neg. organisms.  Failure to address these problems has led to a great deal of misdirected effort.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTDYwtK6vHbVg90H21EOLACvtfcHk0li1rVMGwyzL8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D&md5=a77b7a1f2763b73d4bd63daf67f78cc7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FCMR.00030-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00030-10%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DChallenges%2520of%2520antibacterial%2520discovery%26jtitle%3DClin.%2520Microbio.%2520Rev.%26date%3D2011%26volume%3D24%26spage%3D71%26epage%3D109%26doi%3D10.1128%2FCMR.00030-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schillaci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanò, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montalbano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barraja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diana, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirrincione, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascioferro, S.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical approached to target antiobiotic resistance mechanisms</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8268</span>– <span class="NLM_lpage">8297</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8268-8297&author=D.+Schillaciauthor=V.+Span%C3%B2author=B.+Parrinoauthor=A.+Carboneauthor=A.+Montalbanoauthor=P.+Barrajaauthor=P.+Dianaauthor=G.+Cirrincioneauthor=S.+Cascioferro&title=Pharmaceutical+approached+to+target+antiobiotic+resistance+mechanisms&doi=10.1021%2Facs.jmedchem.7b00215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical Approaches to Target Antibiotic Resistance Mechanisms</span></div><div class="casAuthors">Schillaci, Domenico; Spano, Virginia; Parrino, Barbara; Carbone, Anna; Montalbano, Alessandra; Barraja, Paola; Diana, Patrizia; Cirrincione, Girolamo; Cascioferro, Stella</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8268-8297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is urgent need for new therapeutic strategies to fight the global threat of antibiotic resistance.  The focus of this Perspective is on chem. agents that target the most common mechanisms of antibiotic resistance such as enzymic inactivation of antibiotics, changes in cell permeability, and induction/activation of efflux pumps.  Here the authors assess the current landscape and challenges in the treatment of antibiotic resistance mechanisms at both bacterial cell and community levels.  The authors also discuss the potential clin. application of chem. inhibitors of antibiotic resistance mechanisms as add-on treatments for serious drug-resistant infections.  Enzymic inhibitors, such as the derivs. of the β-lactamase inhibitor avibactam, are closer to the clinic than other mols.  For example, MK-7655, in combination with imipenem, is in clin. development for the treatment of infections caused by carbapenem-resistant Enterobacteriaceae and Pseudomonas aeruginosa, which are difficult to treat.  In addn., other mols. targeting multidrug-resistance mechanisms, such as efflux pumps, are under development and hold promise for the treatment of multidrug resistant infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo5bQLECGVxLVg90H21EOLACvtfcHk0li1rVMGwyzL8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D&md5=dfbd996313789479a5a10becb59257df</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00215%26sid%3Dliteratum%253Aachs%26aulast%3DSchillaci%26aufirst%3DD.%26aulast%3DSpan%25C3%25B2%26aufirst%3DV.%26aulast%3DParrino%26aufirst%3DB.%26aulast%3DCarbone%26aufirst%3DA.%26aulast%3DMontalbano%26aufirst%3DA.%26aulast%3DBarraja%26aufirst%3DP.%26aulast%3DDiana%26aufirst%3DP.%26aulast%3DCirrincione%26aufirst%3DG.%26aulast%3DCascioferro%26aufirst%3DS.%26atitle%3DPharmaceutical%2520approached%2520to%2520target%2520antiobiotic%2520resistance%2520mechanisms%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8268%26epage%3D8297%26doi%3D10.1021%2Facs.jmedchem.7b00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Champoux, J. J.</span></span> <span> </span><span class="NLM_article-title">DNA topoisomerases: structure, function, and mechanism</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.70.1.369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1146%2Fannurev.biochem.70.1.369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=11395412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2001&pages=369-413&author=J.+J.+Champoux&title=DNA+topoisomerases%3A+structure%2C+function%2C+and+mechanism&doi=10.1146%2Fannurev.biochem.70.1.369"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">DNA topoisomerases: structure, function, and mechanism</span></div><div class="casAuthors">Champoux, James J.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">369-413</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 196 refs.  DNA topoisomerases solve the topol. problems assocd. with DNA replication, transcription, recombination, and chromatin remodeling by introducing temporary single- or double-strand breaks in the DNA.  In addn., these enzymes fine-tune the steady-state level of DNA supercoiling both to facilitate protein interactions with the DNA and to prevent excessive supercoiling that is deleterious.  In recent years, the crystal structures of a no. of topoisomerase fragments, representing nearly all of the known classes of enzymes, have been solved.  These structures provide remarkable insights into the mechanisms of these enzymes and complement previous conclusions based on biochem. analyses.  Surprisingly, despite little or no sequence homol., both type IA and type IIA topoisomerases from prokaryotes and the type IIA enzymes from eukaryotes share structural folds that appear to reflect functional motifs within crit. regions of the enzymes.  The type IB enzymes are structurally distinct from all other known topoisomerases but are similar to a class of enzymes referred to as tyrosine recombinases.  The structural themes common to all topoisomerases include hinged clamps that open and close to bind DNA, the presence of DNA binding cavities for temporary storage of DNA segments, and the coupling of protein conformational changes to DNA rotation or DNA movement.  For the type II topoisomerases, the binding and hydrolysis of ATP further modulate conformational changes in the enzymes to effect changes in DNA topol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Y4rDyIYy17Vg90H21EOLACvtfcHk0li1rVMGwyzL8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7Y%253D&md5=67c3e38eb4fcb85527b33d7077309482</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.70.1.369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.70.1.369%26sid%3Dliteratum%253Aachs%26aulast%3DChampoux%26aufirst%3DJ.%2BJ.%26atitle%3DDNA%2520topoisomerases%253A%2520structure%252C%2520function%252C%2520and%2520mechanism%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2001%26volume%3D70%26spage%3D369%26epage%3D413%26doi%3D10.1146%2Fannurev.biochem.70.1.369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thathan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubie, S.</span></span> <span> </span><span class="NLM_article-title">Dual targeting DNA gyrase B (GyrB) and topoisomerse IV (ParE) inhibitors: a review</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2015.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bioorg.2015.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=26232660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KlsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2015&pages=41-63&author=M.+A.+Azamauthor=J.+Thathanauthor=S.+Jubie&title=Dual+targeting+DNA+gyrase+B+%28GyrB%29+and+topoisomerse+IV+%28ParE%29+inhibitors%3A+a+review&doi=10.1016%2Fj.bioorg.2015.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting DNA gyrase B (GyrB) and topoisomerase IV (ParE) inhibitors: A review</span></div><div class="casAuthors">Azam, Mohammed Afzal; Thathan, Janarthanan; Jubie, Selvaraj</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-63</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">GyrB and ParE are type IIA topoisomerases and found in most bacteria.  Its function is vital for DNA replication, repair and decatenation.  The highly conserved ATP-binding subunits of DNA GyrB and ParE are structurally related and have been recognized as prime candidates for the development of dual-targeting antibacterial agents with broad-spectrum potential.  However, no natural product or small mol. inhibitors targeting ATPase catalytic domain of both GyrB and ParE enzymes have succeeded in the clinic.  Moreover, no inhibitors of these enzymes with broad-spectrum antibacterial activity against Gram-neg. pathogens have been reported.  Availability of high resoln. crystal structures of GyrB and ParE made it possible for the design of many different classes of inhibitors with dual mechanism of action.  Among them benzimidazoles, benzothiazoles, thiazolopyridines, imidiazopyridazoles, pyridines, indazoles, pyrazoles, imidazopyridines, triazolopyridines, pyrrolopyrimidines, pyrimidoindoles as well as related structures are disclosed in literatures.  Unfortunately most of these inhibitors are found to be active against Gram-pos. pathogens.  In the present review we discuss about studies on novel dual targeting ATPase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-_PqzrhxWbVg90H21EOLACvtfcHk0ljqtnwQ2UGOmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KlsL3J&md5=37ff522c4d029c2e358be124bf6e9264</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2015.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2015.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DAzam%26aufirst%3DM.%2BA.%26aulast%3DThathan%26aufirst%3DJ.%26aulast%3DJubie%26aufirst%3DS.%26atitle%3DDual%2520targeting%2520DNA%2520gyrase%2520B%2520%2528GyrB%2529%2520and%2520topoisomerse%2520IV%2520%2528ParE%2529%2520inhibitors%253A%2520a%2520review%26jtitle%3DBioorg.%2520Chem.%26date%3D2015%26volume%3D62%26spage%3D41%26epage%3D63%26doi%3D10.1016%2Fj.bioorg.2015.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalhoff, A.</span></span> <span> </span><span class="NLM_article-title">Global fluoroquinolone resistance epidemiology and implictions for clinical use</span>. <i>Interdiscip. Perspect. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>2012</i></span>,  <span class="NLM_fpage">976273</span>, <span class="refDoi"> DOI: 10.1155/2012/976273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1155%2F2012%2F976273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23097666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A280%3ADC%252BC3s%252FotVOitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=976273&author=A.+Dalhoff&title=Global+fluoroquinolone+resistance+epidemiology+and+implictions+for+clinical+use&doi=10.1155%2F2012%2F976273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Global fluoroquinolone resistance epidemiology and implictions for clinical use</span></div><div class="casAuthors">Dalhoff Axel</div><div class="citationInfo"><span class="NLM_cas:title">Interdisciplinary perspectives on infectious diseases</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2012</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">976273</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This paper on the fluoroquinolone resistance epidemiology stratifies the data according to the different prescription patterns by either primary or tertiary caregivers and by indication.  Global surveillance studies demonstrate that fluoroquinolone resistance rates increased in the past years in almost all bacterial species except S. pneumoniae and H. influenzae, causing community-acquired respiratory tract infections.  However, 10 to 30% of these isolates harbored first-step mutations conferring low level fluoroquinolone resistance.  Fluoroquinolone resistance increased in Enterobacteriaceae causing community acquired or healthcare associated urinary tract infections and intraabdominal infections, exceeding 50% in some parts of the world, particularly in Asia.  One to two-thirds of Enterobacteriaceae producing extended spectrum β-lactamases were fluoroquinolone resistant too.  Furthermore, fluoroquinolones select for methicillin resistance in Staphylococci.  Neisseria gonorrhoeae acquired fluoroquinolone resistance rapidly; actual resistance rates are highly variable and can be as high as almost 100%, particularly in Asia, whereas resistance rates in Europe and North America range from <10% in rural areas to >30% in established sexual networks.  In general, the continued increase in fluoroquinolone resistance affects patient management and necessitates changes in some guidelines, for example, treatment of urinary tract, intra-abdominal, skin and skin structure infections, and traveller's diarrhea, or even precludes the use in indications like sexually transmitted diseases and enteric fever.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDedqxKx8QHLt7lRqEFUJVfW6udTcc2eaqOS1zTFsJrrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s%252FotVOitw%253D%253D&md5=d6ab6c51cb874358473dfe49bc7d0df4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1155%2F2012%2F976273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2012%252F976273%26sid%3Dliteratum%253Aachs%26aulast%3DDalhoff%26aufirst%3DA.%26atitle%3DGlobal%2520fluoroquinolone%2520resistance%2520epidemiology%2520and%2520implictions%2520for%2520clinical%2520use%26jtitle%3DInterdiscip.%2520Perspect.%2520Infect.%2520Dis.%26date%3D2012%26volume%3D2012%26spage%3D976273%26doi%3D10.1155%2F2012%2F976273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Redgrave, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piddock, L. J.V.</span></span> <span> </span><span class="NLM_article-title">Fluoroquinolone resistance: mechanisms, impact on bacteria, and role in evolutionary success</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2014.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.tim.2014.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=24842194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=438-445&author=L.+S.+Redgraveauthor=S.+B.+Suttonauthor=M.+A.+Webberauthor=L.+J.V.+Piddock&title=Fluoroquinolone+resistance%3A+mechanisms%2C+impact+on+bacteria%2C+and+role+in+evolutionary+success&doi=10.1016%2Fj.tim.2014.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Fluoroquinolone resistance: mechanisms, impact on bacteria, and role in evolutionary success</span></div><div class="casAuthors">Redgrave, Liam S.; Sutton, Sam B.; Webber, Mark A.; Piddock, Laura J. V.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">438-445</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Quinolone and fluoroquinolone antibiotics are potent, broad-spectrum agents commonly used to treat a range of infections.  Resistance to these agents is multifactorial and can be via one or a combination of target-site gene mutations, increased prodn. of multidrug-resistance (MDR) efflux pumps, modifying enzymes, and/or target-protection proteins.  Fluoroquinolone-resistant clin. isolates of bacteria have emerged readily and recent data have shown that resistance to this class of antibiotics can have diverse, species-dependent impacts on host-strain fitness.  Here we outline the impacts of quinolone-resistance mutations in relation to the fitness and evolutionary success of mutant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJrla4tEwdbrVg90H21EOLACvtfcHk0ljqtnwQ2UGOmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotlOqsLc%253D&md5=060fb6501e12ed1243e4f30190cb4917</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2014.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2014.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DRedgrave%26aufirst%3DL.%2BS.%26aulast%3DSutton%26aufirst%3DS.%2BB.%26aulast%3DWebber%26aufirst%3DM.%2BA.%26aulast%3DPiddock%26aufirst%3DL.%2BJ.V.%26atitle%3DFluoroquinolone%2520resistance%253A%2520mechanisms%252C%2520impact%2520on%2520bacteria%252C%2520and%2520role%2520in%2520evolutionary%2520success%26jtitle%3DTrends%2520Microbiol.%26date%3D2014%26volume%3D22%26spage%3D438%26epage%3D445%26doi%3D10.1016%2Fj.tim.2014.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirude, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, S.</span></span> <span> </span><span class="NLM_article-title">Nonfluoroquinolone-based inhibitors of mycobacterial type II topoisomerase as potential therapeutic agents for TB</span>. <i>Annu. Rep. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-396492-2.00021-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2FB978-0-12-396492-2.00021-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1eju7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=319-330&author=P.+S.+Shirudeauthor=S.+Hameed&title=Nonfluoroquinolone-based+inhibitors+of+mycobacterial+type+II+topoisomerase+as+potential+therapeutic+agents+for+TB&doi=10.1016%2FB978-0-12-396492-2.00021-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Nonfluoroquinolone-based inhibitors of mycobacterial type II topoisomerase as potential therapeutic agents for TB</span></div><div class="casAuthors">Shirude, Pravin S.; Hameed, Shahul</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">319-330</span>CODEN:
                <span class="NLM_cas:coden">ARMCBI</span>;
        ISSN:<span class="NLM_cas:issn">0065-7743</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review focuses on small-mol. inhibitors of mycobacterial type II topoisomerase that bind at B subunit of DNA gyrase (GyrB) having ATP sites and non-fluoroquinolone inhibitors that bind at A subunit of DNA gyrase (GyrA) having other than ATP sites within the enzyme tetramers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7HVfdftzSBbVg90H21EOLACvtfcHk0ljqtnwQ2UGOmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1eju7jM&md5=0ae5b4dfe642de20da0932708c24a48b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-396492-2.00021-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-396492-2.00021-7%26sid%3Dliteratum%253Aachs%26aulast%3DShirude%26aufirst%3DP.%2BS.%26aulast%3DHameed%26aufirst%3DS.%26atitle%3DNonfluoroquinolone-based%2520inhibitors%2520of%2520mycobacterial%2520type%2520II%2520topoisomerase%2520as%2520potential%2520therapeutic%2520agents%2520for%2520TB%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2012%26volume%3D47%26spage%3D319%26epage%3D330%26doi%3D10.1016%2FB978-0-12-396492-2.00021-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisacchi, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span> <span> </span><span class="NLM_article-title">A new-class antibacterial-almost. Lessons in drug discovery and development: a critical analysis of more than 50 years of effort toward ATPase inhibitors of DNA gyrase and topoisomerase IV</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1021/id500013t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/id500013t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=4-41&author=G.+S.+Bisacchiauthor=J.+I.+Manchester&title=A+new-class+antibacterial-almost.+Lessons+in+drug+discovery+and+development%3A+a+critical+analysis+of+more+than+50+years+of+effort+toward+ATPase+inhibitors+of+DNA+gyrase+and+topoisomerase+IV&doi=10.1021%2Fid500013t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A New-Class Antibacterial-Almost. Lessons in Drug Discovery and Development: A Critical Analysis of More than 50 Years of Effort toward ATPase Inhibitors of DNA Gyrase and Topoisomerase IV</span></div><div class="casAuthors">Bisacchi, Gregory S.; Manchester, John I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-41</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The introduction into clin. practice of an ATPase inhibitor of bacterial DNA gyrase and topoisomerase IV (topo IV) would represent a new-class agent for the treatment of resistant bacterial infections.  Novobiocin, the only historical member of this class, established the clin. proof of concept for this novel mechanism during the late 1950s, but its use declined rapidly and it was eventually withdrawn from the market.  Despite significant and prolonged effort across the biopharmaceutical industry to develop other agents of this class, novobiocin remains the only ATPase inhibitor of gyrase and topo IV ever to progress beyond Phase I.  In this review, we analyze the historical attempts to discover and develop agents within this class and highlight factors that might have hindered those efforts.  Within the last 15 years, however, our tech. understanding of the mol. details of the inhibition of the gyrase and topo IV ATPases, the factors governing resistance development to such inhibitors, and our knowledge of the phys. properties required for robust clin. drug candidates have all matured to the point wherein the industry may now address this mechanism of action with greater confidence.  The antibacterial spectrum within this class has recently been extended to begin to include serious Gram neg. pathogens such as Pseudomonas aeruginosa, Acinetobacter baumannii, and Klebsiella pneumoniae.  In spite of this recent tech. progress, adverse economics assocd. with antibacterial R&D over the last 20 years has diminished industry's ability to commit the resources and perseverance needed to bring new-class agents to launch.  Consequently, a no. of recent efforts in the ATPase class have been derailed by organizational rather than scientific factors.  Nevertheless, within this context we discuss the unique opportunity for the development of ATPase inhibitors of gyrase and topo IV as new-class antibacterial agents with broad spectrum potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXubyDLdrSrrVg90H21EOLACvtfcHk0lhp-iB9kHpWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL&md5=95ce2833dab37f68a3a4a018e381e198</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fid500013t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fid500013t%26sid%3Dliteratum%253Aachs%26aulast%3DBisacchi%26aufirst%3DG.%2BS.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26atitle%3DA%2520new-class%2520antibacterial-almost.%2520Lessons%2520in%2520drug%2520discovery%2520and%2520development%253A%2520a%2520critical%2520analysis%2520of%2520more%2520than%252050%2520years%2520of%2520effort%2520toward%2520ATPase%2520inhibitors%2520of%2520DNA%2520gyrase%2520and%2520topoisomerase%2520IV%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D4%26epage%3D41%26doi%3D10.1021%2Fid500013t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span> <span> </span><span class="NLM_article-title">Overcoming problems of poor drug penetration into bacteria: challenges and strategies for medicinal chemists</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1080/17460441.2018.1455660</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1080%2F17460441.2018.1455660" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=29566560" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=497-507&author=D.+B.+Tizauthor=D.+Kikeljauthor=N.+Zidar&title=Overcoming+problems+of+poor+drug+penetration+into+bacteria%3A+challenges+and+strategies+for+medicinal+chemists&doi=10.1080%2F17460441.2018.1455660"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F17460441.2018.1455660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2018.1455660%26sid%3Dliteratum%253Aachs%26aulast%3DTiz%26aufirst%3DD.%2BB.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DZidar%26aufirst%3DN.%26atitle%3DOvercoming%2520problems%2520of%2520poor%2520drug%2520penetration%2520into%2520bacteria%253A%2520challenges%2520and%2520strategies%2520for%2520medicinal%2520chemists%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2018%26volume%3D13%26spage%3D497%26epage%3D507%26doi%3D10.1080%2F17460441.2018.1455660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durcik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span> <span> </span><span class="NLM_article-title">ATP competitive DNA gyrase and topoisomerase IV inhibitors as antibacterial agents</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1080/13543776.2019.1575362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1080%2F13543776.2019.1575362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=30686070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=171-180&author=M.+Durcikauthor=T.+Toma%C5%A1i%C4%8Dauthor=N.+Zidarauthor=A.+Zegaauthor=D.+Kikeljauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1&title=ATP+competitive+DNA+gyrase+and+topoisomerase+IV+inhibitors+as+antibacterial+agents&doi=10.1080%2F13543776.2019.1575362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-competitive DNA gyrase and topoisomerase IV inhibitors as antibacterial agents</span></div><div class="casAuthors">Durcik, Martina; Tomasic, Tihomir; Zidar, Nace; Zega, Anamarija; Kikelj, Danijel; Masic, Lucija Peterlin; Ilas, Janez</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">171-180</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The bacterial topoisomerases DNA gyrase and topoisomerase IV are validated targets for development of novel antibacterial agents.  Fluoroquinolones inhibit the catalytic GyrA and/or ParC(GrlA) subunit and have been commonly used, although these have toxicity liabilities that restrict their use.  The ATPase GyrB and ParE(GrlB) subunits have been much less explored and after withdrawal of novobiocin, there are no further marketed inhibitors.  ATP-competitive inhibitors of GyrB and/or ParE(GrlB) are of special interest, as this target has been validated, and it is expected that many of the problems assocd. with fluoroquinolones can be avoided.  This review summarises the development of ATP-competitive inhibitors of GyrB and/or ParE(GrlB) as novel antibacterial agents over the last 10 years.  Structural features of the new inhibitors and their optimization strategies are highlighted.  The development of novel ATP-competitive inhibitors of GyrB and/or ParE(GrlB) is ongoing in industrial and academical research.  Development of resistance is one of the most problematic issues, but GyrB/ParE(GrlB) inhibitors do not show cross-resistance with fluoroquinolones.  Other common issues, such as low soly., high protein binding, development of off-target resistance, are related to the structures of the inhibitors themselves, which is thus a main focus of design strategies.  With some now in early clin. development, there is reasonable expectation that novel ATP-competitive inhibitors of GyrB/ParE(GrlB) will reach the market in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPIQu6ZReEKbVg90H21EOLACvtfcHk0lhp-iB9kHpWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D&md5=2b5f0b8ab2036bee44202ae45b95a9a5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1575362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1575362%26sid%3Dliteratum%253Aachs%26aulast%3DDurcik%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26atitle%3DATP%2520competitive%2520DNA%2520gyrase%2520and%2520topoisomerase%2520IV%2520inhibitors%2520as%2520antibacterial%2520agents%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2019%26volume%3D29%26spage%3D171%26epage%3D180%26doi%3D10.1080%2F13543776.2019.1575362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hough, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akers-Rodriguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown-Driver, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillar, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinabarger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Tricyclic GyrB/ParE (TriBE) inhibitors: a new class of broad-spectrum dual-targeting antibacterial agents</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e84409</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0084409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1371%2Fjournal.pone.0084409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=24386374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=L.+W.+Tariauthor=X.+Liauthor=M.+Trzossauthor=D.+C.+Bensenauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=S.+J.+Leeauthor=G.+Houghauthor=D.+Phillipsonauthor=S.+Akers-Rodriguezauthor=M.+L.+Cunninghamauthor=B.+P.+Kwanauthor=K.+L.+Nelsonauthor=A.+Castellanoauthor=J.+B.+Lockeauthor=V.+Brown-Driverauthor=T.+M.+Murphyauthor=V.+S.+Ongauthor=C.+M.+Pillarauthor=D.+L.+Shinabargerauthor=J.+Nixauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=K.+J.+Shawauthor=J.+Finn&title=Tricyclic+GyrB%2FParE+%28TriBE%29+inhibitors%3A+a+new+class+of+broad-spectrum+dual-targeting+antibacterial+agents&doi=10.1371%2Fjournal.pone.0084409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic GyrB/ParE (TriBE) inhibitors: a new class of broad-spectrum dual-targeting antibacterial agents</span></div><div class="casAuthors">Tari, Leslie W.; Li, Xiaoming; Trzoss, Michael; Bensen, Daniel C.; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Lee, Suk Joong; Hough, Grayson; Phillipson, Doug; Akers, Rodriguez Suzanne; Cunningham, Mark L.; Kwan, Bryan P.; Nelson, Kirk J.; Castellano, Amanda; Locke, Jeff B.; Brown-Driver, Vickie; Murphy, Timothy M.; Ong, Voon S.; Pillar, Chris M.; Shinabarger, Dean L.; Nix, Jay; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Shaw, Karen J.; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e84409/1-e84409/13, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increasing resistance to every major class of antibiotics and a dearth of novel classes of antibacterial agents in development pipelines has created a dwindling reservoir of treatment options for serious bacterial infections.  The bacterial type IIA topoisomerases, DNA gyrase and topoisomerase IV, are validated antibacterial drug targets with multiple prospective drug binding sites, including the catalytic site targeted by the fluoroquinolone antibiotics.  However, growing resistance to fluoroquinolones, frequently mediated by mutations in the drug-binding site, is increasingly limiting the utility of this antibiotic class, prompting the search for other inhibitor classes that target different sites on the topoisomerase complexes.  The highly conserved ATP-binding subunits of DNA gyrase (GyrB) and topoisomerase IV (ParE) have long been recognized as excellent candidates for the development of dual-targeting antibacterial agents with broad-spectrum potential.  However, to date, no natural product or small mol. inhibitors targeting these sites have succeeded in the clinic, and no inhibitors of these enzymes have yet been reported with broad-spectrum antibacterial activity encompassing the majority of Gram-neg. pathogens.  Using structure-based drug design (SBDD), we have created a novel dual-targeting pyrimidoindole inhibitor series with exquisite potency against GyrB and ParE enzymes from a broad range of clin. important pathogens.  Inhibitors from this series demonstrate potent, broad-spectrum antibacterial activity against Gram-pos. and Gram-neg. pathogens of clin. importance, including fluoroquinolone resistant and multidrug resistant strains.  Lead compds. have been discovered with clin. potential; they are well tolerated in animals, and efficacious in Gram-neg. infection models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ITJvl7tDrrVg90H21EOLACvtfcHk0lj8W6kr_orsEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D&md5=2467ba25da5cb55226d83714d59fd2b8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084409%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DHough%26aufirst%3DG.%26aulast%3DPhillipson%26aufirst%3DD.%26aulast%3DAkers-Rodriguez%26aufirst%3DS.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DNelson%26aufirst%3DK.%2BL.%26aulast%3DCastellano%26aufirst%3DA.%26aulast%3DLocke%26aufirst%3DJ.%2BB.%26aulast%3DBrown-Driver%26aufirst%3DV.%26aulast%3DMurphy%26aufirst%3DT.%2BM.%26aulast%3DOng%26aufirst%3DV.%2BS.%26aulast%3DPillar%26aufirst%3DC.%2BM.%26aulast%3DShinabarger%26aufirst%3DD.%2BL.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DTricyclic%2520GyrB%252FParE%2520%2528TriBE%2529%2520inhibitors%253A%2520a%2520new%2520class%2520of%2520broad-spectrum%2520dual-targeting%2520antibacterial%2520agents%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0084409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part I: Structural guided discovery and optimization of dual targeting agents with potent, broad-spectrum enzymatic acticity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1536</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmcl.2012.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23352267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1529-1536&author=L.+W.+Tariauthor=X.+Liauthor=M.+Trzossauthor=D.+C.+Bensenauthor=X.+Liauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=C.+J.+Creightonauthor=M.+L.+Cunninghamauthor=B.+Kwanauthor=M.+Stidhamauthor=K.+J.+Shawauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=J.+Nixauthor=J.+Finn&title=Pyrrolopyrimidine+inhibitors+of+DNA+gyrase+B+%28GyrB%29+and+topoisomerase+IV+%28ParE%29.+Part+I%3A+Structural+guided+discovery+and+optimization+of+dual+targeting+agents+with+potent%2C+broad-spectrum+enzymatic+acticity&doi=10.1016%2Fj.bmcl.2012.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part I: Structure guided discovery and optimization of dual targeting agents with potent, broad-spectrum enzymatic activity</span></div><div class="casAuthors">Tari, Leslie W.; Trzoss, Michael; Bensen, Daniel C.; Li, Xiaoming; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Creighton, Christopher J.; Cunningham, Mark L.; Kwan, Bryan; Stidham, Mark; Shaw, Karen J.; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Nix, Jay; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1529-1536</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bacterial topoisomerases DNA gyrase (GyrB) and topoisomerase IV (ParE) are essential enzymes that control the topol. state of DNA during replication.  The high degree of conservation in the ATP-binding pockets of these enzymes make them appealing targets for broad-spectrum inhibitor development.  A pyrrolopyrimidine scaffold was identified from a pharmacophore-based fragment screen with optimization potential.  Structural characterization of inhibitor complexes conducted using selected GyrB/ParE orthologs aided in the identification of important steric, dynamic and compositional differences in the ATP-binding pockets of the targets, enabling the design of highly potent pyrrolopyrimidine inhibitors with broad enzymic spectrum and dual targeting activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn5kRWxZVjhbVg90H21EOLACvtfcHk0lj8W6kr_orsEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D&md5=b8d572d3dc5320c2ec385668b3f60387</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%26aulast%3DStidham%26aufirst%3DM.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DPyrrolopyrimidine%2520inhibitors%2520of%2520DNA%2520gyrase%2520B%2520%2528GyrB%2529%2520and%2520topoisomerase%2520IV%2520%2528ParE%2529.%2520Part%2520I%253A%2520Structural%2520guided%2520discovery%2520and%2520optimization%2520of%2520dual%2520targeting%2520agents%2520with%2520potent%252C%2520broad-spectrum%2520enzymatic%2520acticity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1529%26epage%3D1536%26doi%3D10.1016%2Fj.bmcl.2012.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown-Driver, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part II: Development of inhibitors with broad-spectrum, Gram-negative antibacterial activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1543</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.11.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmcl.2012.11.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=23294697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1537-1543&author=M.+Trzossauthor=D.+C.+Bensenauthor=X.+Liauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=C.+J.+Creightonauthor=M.+L.+Cunninghamauthor=B.+Kwanauthor=M.+Stidhamauthor=K.+Nelsonauthor=V.+Brown-Driverauthor=A.+Castellanoauthor=K.+J.+Shawauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=J.+Nixauthor=J.+Finnauthor=L.+W.+Tari&title=Pyrrolopyrimidine+inhibitors+of+DNA+gyrase+B+%28GyrB%29+and+topoisomerase+IV+%28ParE%29.+Part+II%3A+Development+of+inhibitors+with+broad-spectrum%2C+Gram-negative+antibacterial+activity&doi=10.1016%2Fj.bmcl.2012.11.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE), Part II: Development of inhibitors with broad spectrum, Gram-negative antibacterial activity</span></div><div class="casAuthors">Trzoss, Micheal; Bensen, Daniel C.; Li, Xiaoming; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Creighton, Christopher J.; Cunningham, Mark L.; Kwan, Bryan; Stidham, Mark; Nelson, Kirk; Brown-Driver, Vickie; Castellano, Amanda; Shaw, Karen J.; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Finn, John; Tari, Leslie W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1537-1543</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The structurally related bacterial topoisomerases DNA gyrase (GyrB) and topoisomerase IV (ParE) have long been recognized as prime candidates for the development of broad spectrum antibacterial agents.  However, GyrB/ParE targeting antibacterials with spectrum that encompasses robust Gram-neg. pathogens have not yet been reported.  Using structure-based inhibitor design, we optimized a novel pyrrolopyrimidine inhibitor series with potent, dual targeting activity against GyrB and ParE.  Compds. were discovered with broad antibacterial spectrum, including activity against Pseudomonas aeruginosa, Acinetobacter baumannii and Escherichia coli.  Here, the authors describe the SAR of the pyrrolopyrimidine series as it relates to key structural and electronic features necessary for Gram-neg. antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaEjbnVIUQM7Vg90H21EOLACvtfcHk0lj8W6kr_orsEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVCgtA%253D%253D&md5=f54bbe3030bfcb30e1585a1029583b99</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.11.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.11.073%26sid%3Dliteratum%253Aachs%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%26aulast%3DStidham%26aufirst%3DM.%26aulast%3DNelson%26aufirst%3DK.%26aulast%3DBrown-Driver%26aufirst%3DV.%26aulast%3DCastellano%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DFinn%26aufirst%3DJ.%26aulast%3DTari%26aufirst%3DL.%2BW.%26atitle%3DPyrrolopyrimidine%2520inhibitors%2520of%2520DNA%2520gyrase%2520B%2520%2528GyrB%2529%2520and%2520topoisomerase%2520IV%2520%2528ParE%2529.%2520Part%2520II%253A%2520Development%2520of%2520inhibitors%2520with%2520broad-spectrum%252C%2520Gram-negative%2520antibacterial%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1537%26epage%3D1543%26doi%3D10.1016%2Fj.bmcl.2012.11.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poh, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liew, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendharkar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangthongpitag, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basarab, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherian, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of dual GyrB/ParE inhibitors active against gram-negative bacteria</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.ejmech.2018.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=30125722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2018&pages=610-621&author=S.+Y.+Hoauthor=W.+Wangauthor=F.+M.+Ngauthor=Y.+X.+Wongauthor=Z.+Y.+Pohauthor=S.+W.+W.+Tanauthor=S.+H.+Angauthor=S.+S.+Liewauthor=Y.+S.+J.+Wongauthor=Y.+Tanauthor=A.+Poulsenauthor=V.+Pendharkarauthor=K.+Sangthongpitagauthor=J.+Manchesterauthor=G.+Basarabauthor=J.+Hillauthor=T.+H.+Kellerauthor=J.+Cherian&title=Discovery+of+dual+GyrB%2FParE+inhibitors+active+against+gram-negative+bacteria&doi=10.1016%2Fj.ejmech.2018.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of dual GyrB/ParE inhibitors active against Gram-negative bacteria</span></div><div class="casAuthors">Ho, Soo Yei; Wang, Weiling; Ng, Fui Mee; Wong, Yun Xuan; Poh, Zhi Ying; Tan, Sum Wai Eldwin; Ang, Shi Hua; Liew, Si Si; Joyner Wong, Yin Sze; Tan, Yvonne; Poulsen, Anders; Pendharkar, Vishal; Sangthongpitag, Kanda; Manchester, John; Basarab, Gregory; Hill, Jeffrey; Keller, Thomas H.; Cherian, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">610-621</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Even though many GyrB and ParE inhibitors have been reported in the literature, few possess activity against Gram-neg. bacteria.  This is primarily due to limited permeability across Gram-neg. bacterial membrane as well as bacterial efflux mechanisms.  Permeability of compds. across Gram-neg. bacterial membranes depends on many factors including physicochem. properties of the inhibitors.  Herein, we show the optimization of pyridylureas leading to compds. with potent activity against Gram-neg. bacterial species such as P.aeruginosa, E.coli and A.baumannii.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-A5-xjaiNHbVg90H21EOLACvtfcHk0lg2-8oYYPMWlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O&md5=89908d086984c0df2588ca862abeada3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DS.%2BY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DNg%26aufirst%3DF.%2BM.%26aulast%3DWong%26aufirst%3DY.%2BX.%26aulast%3DPoh%26aufirst%3DZ.%2BY.%26aulast%3DTan%26aufirst%3DS.%2BW.%2BW.%26aulast%3DAng%26aufirst%3DS.%2BH.%26aulast%3DLiew%26aufirst%3DS.%2BS.%26aulast%3DWong%26aufirst%3DY.%2BS.%2BJ.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DPendharkar%26aufirst%3DV.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DManchester%26aufirst%3DJ.%26aulast%3DBasarab%26aufirst%3DG.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DCherian%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520dual%2520GyrB%252FParE%2520inhibitors%2520active%2520against%2520gram-negative%2520bacteria%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D157%26spage%3D610%26epage%3D621%26doi%3D10.1016%2Fj.ejmech.2018.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span> <span> </span><span class="NLM_article-title">Physicochemical properties of antibacterial compounds: implications for drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2871</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1021/jm700967e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700967e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2871-2878&author=R.+O%E2%80%99Sheaauthor=H.+E.+Moser&title=Physicochemical+properties+of+antibacterial+compounds%3A+implications+for+drug+discovery&doi=10.1021%2Fjm700967e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical properties of antibacterial compounds: implications for drug discovery</span></div><div class="casAuthors">O'Shea, Rosemarie; Moser, Heinz E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2871-2878</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of multidrug-resistant pathogens and the need for novel antibiotics, it is crit. to understand as much as possible from prior efforts and to apply learned lessons to the discovery of future antibiotics.  One important parameter in particular has previously been mentioned but, in the view, not sufficiently analyzed: the physicochem. property space of antibacterial drugs.  The authors selected 147 antibacterially active compds. that encompass both currently used drugs and compds. that are still under clin. investigation (see Methods for details).  Where available, other property values were extd. from the literature, including protein binding and oral bioavailability in humans.  This anal. suggests that natural products should be increasingly investigated again to identify novel antibacterial hits.  Besides their high level of structural diversity, they are likely to better cover the required physicochem. property space for antibacterial compds. compared to synthetic mols. because of an increased d. of polar functionalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25EGWMkWdk7Vg90H21EOLACvtfcHk0lg2-8oYYPMWlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D&md5=8aae7d962df53b70113e0fe6169abd68</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm700967e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700967e%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DR.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26atitle%3DPhysicochemical%2520properties%2520of%2520antibacterial%2520compounds%253A%2520implications%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2871%26epage%3D2878%26doi%3D10.1021%2Fjm700967e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May-Dracka, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span> <span> </span><span class="NLM_article-title">Trends and exceptions of physical properties on antibacterial activity for gram-positive and gram-negative pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10144</span>– <span class="NLM_lpage">10161</span>, <span class="refDoi"> DOI: 10.1021/jm501552x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501552x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10144-10161&author=D.+G.+Brownauthor=T.+L.+May-Drackaauthor=M.+M.+Gagnonauthor=R.+Tommasi&title=Trends+and+exceptions+of+physical+properties+on+antibacterial+activity+for+gram-positive+and+gram-negative+pathogens&doi=10.1021%2Fjm501552x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Trends and Exceptions of Physical Properties on Antibacterial Activity for Gram-Positive and Gram-Negative Pathogens</span></div><div class="casAuthors">Brown, Dean G.; May-Dracka, Tricia L.; Gagnon, Moriah M.; Tommasi, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10144-10161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the difficulties surrounding the identification of novel antibacterial compds. from corporate screening collections, phys. properties of ∼3200 antibacterial project compds. with whole cell activity against Gram-neg. or Gram-pos. pathogens were profiled and compared to actives found from high throughput (HTS) screens conducted on both biochem. and phenotypic bacterial targets.  The output from 23 antibacterial HTS screens illustrated that when compared to the properties of the antibacterial project compds., the HTS actives were significantly more hydrophobic than antibacterial project compds. (typically 2-4 log units higher), and furthermore, for 14/23 HTS screens, the av. clogD was higher than the screening collection av. (screening collection clogD = 2.45).  It was found that the consequences of this were the following: (a) lead identification programs often further gained hydrophobic character with increased biochem. potency, making the sepn. even larger between the physicochem. properties of known antibacterial agents and the HTS active starting point, (b) the probability of plasma protein binding and cytotoxicity are often increased, and (c) cell-based activity in Gram-neg. bacteria was severely limited or, if present, demonstrated significant efflux.  Our anal. illustrated that compds. least susceptible to efflux were those which were highly polar and small in MW or very large and typically zwitterionic.  Hydrophobicity was often the dominant driver for HTS actives but, more often than not, precluded whole cell antibacterial activity.  However, simply designing polar compds. was not sufficient for antibacterial activity and pointed to a lack of understanding of complex and specific bacterial penetration mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-f01NZg_bsbVg90H21EOLACvtfcHk0liFYyjyQSPaQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN&md5=e107961414dc8cbf696e2287ddf9362d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm501552x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501552x%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DMay-Dracka%26aufirst%3DT.%2BL.%26aulast%3DGagnon%26aufirst%3DM.%2BM.%26aulast%3DTommasi%26aufirst%3DR.%26atitle%3DTrends%2520and%2520exceptions%2520of%2520physical%2520properties%2520on%2520antibacterial%2520activity%2520for%2520gram-positive%2520and%2520gram-negative%2520pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10144%26epage%3D10161%26doi%3D10.1021%2Fjm501552x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dussault, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uria-Nickelsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioannidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hull, K. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel azaindole class of antibacterial agents targeting the ATPase domains of DNA gyrase and Topoisomerase IV</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5150</span>– <span class="NLM_lpage">5156</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.05.128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.bmcl.2012.05.128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=22814212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=5150-5156&author=J.+I.+Manchesterauthor=D.+D.+Dussaultauthor=J.+A.+Roseauthor=P.+A.+Boriack-Sjodinauthor=M.+Uria-Nickelsenauthor=G.+Ioannidisauthor=S.+Bistauthor=P.+Flemingauthor=K.+G.+Hull&title=Discovery+of+a+novel+azaindole+class+of+antibacterial+agents+targeting+the+ATPase+domains+of+DNA+gyrase+and+Topoisomerase+IV&doi=10.1016%2Fj.bmcl.2012.05.128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel azaindole class of antibacterial agents targeting the ATPase domains of DNA gyrase and Topoisomerase IV</span></div><div class="casAuthors">Manchester, John I.; Dussault, Daemian D.; Rose, Jonathan A.; Boriack-Sjodin, P. Ann; Uria-Nickelsen, Maria; Ioannidis, Georgine; Bist, Shanta; Fleming, Paul; Hull, Kenneth G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5150-5156</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery and optimization of a series of bacterial topoisomerase inhibitors, e.g., I, is disclosed.  Starting from a virtual screening hit, activity was optimized through a combination of structure-based design and phys. property optimization.  Synthesis of fewer than a dozen compds. was required to achieve inhibition of the growth of methicillin-resistant Staphyloccus aureus at compd. concns. of 1.56 μM.  These compds. simultaneously inhibit DNA gyrase and Topoisomerase IV at similar nanomolar concns., reducing the likelihood of the spontaneous occurrence of target-based mutations resulting in antibiotic resistance, an increasing threat in the treatment of serious infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm-t-_e2xcSbVg90H21EOLACvtfcHk0liFYyjyQSPaQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOqt7bL&md5=7c2e42ef7a21c7805736065e84d86a49</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.05.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.05.128%26sid%3Dliteratum%253Aachs%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DDussault%26aufirst%3DD.%2BD.%26aulast%3DRose%26aufirst%3DJ.%2BA.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DIoannidis%26aufirst%3DG.%26aulast%3DBist%26aufirst%3DS.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DHull%26aufirst%3DK.%2BG.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520azaindole%2520class%2520of%2520antibacterial%2520agents%2520targeting%2520the%2520ATPase%2520domains%2520of%2520DNA%2520gyrase%2520and%2520Topoisomerase%2520IV%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D5150%26epage%3D5156%26doi%3D10.1016%2Fj.bmcl.2012.05.128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1435</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1111/nyas.13598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1111%2Fnyas.13598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=29446459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1435&publication_year=2019&pages=18-38&author=M.+F.+Richterauthor=P.+J.+Hergenrother&title=The+challenge+of+converting+Gram-positive-only+compounds+into+broad-spectrum+antibiotics&doi=10.1111%2Fnyas.13598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span></div><div class="casAuthors">Richter, Michelle F.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1435</span>
        (<span class="NLM_cas:issue">1, Antimicrobial Therapeutics Reviews</span>),
    <span class="NLM_cas:pages">18-38</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistant Gram-neg. bacterial infections are on the rise, and there is a lack of new classes of drugs to treat these pathogens.  This drug shortage is largely due to the challenge of finding antibiotics that can permeate and persist inside Gram-neg. species.  Efforts to understand the mol. properties that enable certain compds. to accumulate in Gram-neg. bacteria based on retrospective studies of known antibiotics have not been generally actionable in the development of new antibiotics.  A recent assessment of the ability of >180 diverse small mols. to accumulate in Escherichia coli led to predictive guidelines for compd. accumulation in E. coli.  These "eNTRy rules" state that compds. are most likely to accumulate if they contain a nonsterically encumbered ionizable Nitrogen (primary amines are the best), have low Three-dimensionality (globularity ≤ 0.25), and are relatively Rigid (rotatable bonds ≤ 5).  In this review, we look back through 50+ years of antibacterial research and 1000s of derivs. and assess this historical data set through the lens of these predictive guidelines.  The results are consistent with the eNTRy rules, suggesting that the eNTRy rules may provide an actionable and general roadmap for the conversion of Gram-pos.-only compds. into broad-spectrum antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX3zbgiAcHdbVg90H21EOLACvtfcHk0ljfH-_1uzXNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D&md5=12fffe0d10e7d3e5c134ac28ce929ab3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fnyas.13598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fnyas.13598%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520challenge%2520of%2520converting%2520Gram-positive-only%2520compounds%2520into%2520broad-spectrum%2520antibiotics%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2019%26volume%3D1435%26spage%3D18%26epage%3D38%26doi%3D10.1111%2Fnyas.13598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+compound+accumulation+rules+yield+a+broad-spectrum+antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0ljfH-_1uzXNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520compound%2520accumulation%2520rules%2520yield%2520a%2520broad-spectrum%2520antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seelig, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assmus, F.</span></span> <span> </span><span class="NLM_article-title">Carrier mediated distribution system (CAMDIS): a new approach for the measurement of octanol/water distribution coefficients</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2014.12.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1016%2Fj.ejps.2014.12.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=25513709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVChtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2015&pages=68-77&author=B.+Wagnerauthor=H.+Fischerauthor=M.+Kansyauthor=A.+Seeligauthor=F.+Assmus&title=Carrier+mediated+distribution+system+%28CAMDIS%29%3A+a+new+approach+for+the+measurement+of+octanol%2Fwater+distribution+coefficients&doi=10.1016%2Fj.ejps.2014.12.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Carrier Mediated Distribution System (CAMDIS): A new approach for the measurement of octanol/water distribution coefficients</span></div><div class="casAuthors">Wagner, Bjoern; Fischer, Holger; Kansy, Manfred; Seelig, Anna; Assmus, Frauke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">68-77</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here we present a miniaturized assay, referred to as Carrier-Mediated Distribution System (CAMDIS) for fast and reliable measurement of octanol/water distribution coeffs., log Doct.  By introducing a filter support for octanol, phase sepn. from water is facilitated and the tendency of emulsion formation (emulsification) at the interface is reduced.  A guideline for the best practice of CAMDIS is given, describing a strategy to manage drug adsorption at the filter-supported octanol/buffer interface.  We validated the assay on a set of 52 structurally diverse drugs with known shake flask logDoct values.  Excellent agreement with literature data (r2 = 0.996, std. error of est., SEE = 0.111), high reproducibility (std. deviation, SD < 0.1 log Doct units), minimal sample consumption (10 μL of 100 μM DMSO stock soln.) and a broad anal. range (log Doct range = -0.5 to 4.2) make CAMDIS a valuable tool for the high-throughput assessment of log Doct.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsNSPSNv4bP7Vg90H21EOLACvtfcHk0ljfH-_1uzXNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVChtQ%253D%253D&md5=e0631985c73afe8838fce2eb24ccf686</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2014.12.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2014.12.009%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DFischer%26aufirst%3DH.%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSeelig%26aufirst%3DA.%26aulast%3DAssmus%26aufirst%3DF.%26atitle%3DCarrier%2520mediated%2520distribution%2520system%2520%2528CAMDIS%2529%253A%2520a%2520new%2520approach%2520for%2520the%2520measurement%2520of%2520octanol%252Fwater%2520distribution%2520coefficients%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2015%26volume%3D68%26spage%3D68%26epage%3D77%26doi%3D10.1016%2Fj.ejps.2014.12.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadri, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjemian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaulding, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricotta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prevots, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmore, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klompas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers III, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suffredini, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridkin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danner, R. L.</span></span> <span> </span><span class="NLM_article-title">Difficult-to-treat resistance in gram-negative bacteremia at 173 US hospitials: retrospective cohort analysis of prevalence, predictors, and outcome of resistnace to all first-line agents</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">1803</span>– <span class="NLM_lpage">1814</span>, <span class="refDoi"> DOI: 10.1093/cid/ciy378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=10.1093%2Fcid%2Fciy378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=30052813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;key=1%3ACAS%3A280%3ADC%252BB3c7is1ensw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=1803-1814&author=S.+S.+Kadriauthor=J.+Adjemianauthor=Y.+L.+Laiauthor=A.+B.+Spauldingauthor=E.+Ricottaauthor=D.+R.+Prevotsauthor=T.+N.+Palmoreauthor=C.+Rheeauthor=M.+Klompasauthor=J.+P.+Dekkerauthor=J.+H.+Powers+IIIauthor=A.+F.+Suffrediniauthor=D.+C.+Hooperauthor=S.+Fridkinauthor=R.+L.+Danner&title=Difficult-to-treat+resistance+in+gram-negative+bacteremia+at+173+US+hospitials%3A+retrospective+cohort+analysis+of+prevalence%2C+predictors%2C+and+outcome+of+resistnace+to+all+first-line+agents&doi=10.1093%2Fcid%2Fciy378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Difficult-to-Treat Resistance in Gram-negative Bacteremia at 173 US Hospitals: Retrospective Cohort Analysis of Prevalence, Predictors, and Outcome of Resistance to All First-line Agents</span></div><div class="casAuthors">Kadri Sameer S; Suffredini Anthony F; Danner Robert L; Kadri Sameer S; Hooper David C; Adjemian Jennifer; Lai Yi Ling; Spaulding Alicen B; Ricotta Emily; Prevots D Rebecca; Adjemian Jennifer; Palmore Tara N; Rhee Chanu; Klompas Michael; Rhee Chanu; Klompas Michael; Dekker John P; Powers John H 3rd; Fridkin Scott</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1803-1814</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  Resistance to all first-line antibiotics necessitates the use of less effective or more toxic "reserve" agents.  Gram-negative bloodstream infections (GNBSIs) harboring such difficult-to-treat resistance (DTR) may have higher mortality than phenotypes that allow for ≥1 active first-line antibiotic.  Methods:  The Premier Database was analyzed for inpatients with select GNBSIs.  DTR was defined as intermediate/resistant in vitro to all ss-lactam categories, including carbapenems and fluoroquinolones.  Prevalence and aminoglycoside resistance of DTR episodes were compared with carbapenem-resistant, extended-spectrum cephalosporin-resistant, and fluoroquinolone-resistant episodes using CDC definitions.  Predictors of DTR were identified.  The adjusted relative risk (aRR) of mortality was examined for DTR, CDC-defined phenotypes susceptible to ≥1 first-line agent, and graded loss of active categories.  Results:  Between 2009-2013, 471 (1%) of 45011 GNBSI episodes at 92 (53.2%) of 173 hospitals exhibited DTR, ranging from 0.04% for Escherichia coli to 18.4% for Acinetobacter baumannii.  Among patients with DTR, 79% received parenteral aminoglycosides, tigecycline, or colistin/polymyxin-B; resistance to all aminoglycosides occurred in 33%.  Predictors of DTR included urban healthcare and higher baseline illness.  Crude mortality for GNBSIs with DTR was 43%; aRR was higher for DTR than for carbapenem-resistant (1.2; 95% confidence interval, 1.0-1.4; P = .02), extended-spectrum cephalosporin-resistant (1.2; 1.1-1.4; P = .001), or fluoroquinolone-resistant (1.2; 1.0-1.4; P = .008) infections.  The mortality aRR increased 20% per graded loss of active first-line categories, from 3-5 to 1-2 to 0.  Conclusion:  Nonsusceptibility to first-line antibiotics is associated with decreased survival in GNBSIs.  DTR is a simple bedside prognostic measure of treatment-limiting coresistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2slWy-KRGedHxxy173TfNfW6udTcc2eZas8A0N2Q2_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7is1ensw%253D%253D&md5=16bfa27705be420ac185817d3d9a5d8e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fcid%2Fciy378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcid%252Fciy378%26sid%3Dliteratum%253Aachs%26aulast%3DKadri%26aufirst%3DS.%2BS.%26aulast%3DAdjemian%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DY.%2BL.%26aulast%3DSpaulding%26aufirst%3DA.%2BB.%26aulast%3DRicotta%26aufirst%3DE.%26aulast%3DPrevots%26aufirst%3DD.%2BR.%26aulast%3DPalmore%26aufirst%3DT.%2BN.%26aulast%3DRhee%26aufirst%3DC.%26aulast%3DKlompas%26aufirst%3DM.%26aulast%3DDekker%26aufirst%3DJ.%2BP.%26aulast%3DPowers%2BIII%26aufirst%3DJ.%2BH.%26aulast%3DSuffredini%26aufirst%3DA.%2BF.%26aulast%3DHooper%26aufirst%3DD.%2BC.%26aulast%3DFridkin%26aufirst%3DS.%26aulast%3DDanner%26aufirst%3DR.%2BL.%26atitle%3DDifficult-to-treat%2520resistance%2520in%2520gram-negative%2520bacteremia%2520at%2520173%2520US%2520hospitials%253A%2520retrospective%2520cohort%2520analysis%2520of%2520prevalence%252C%2520predictors%252C%2520and%2520outcome%2520of%2520resistnace%2520to%2520all%2520first-line%2520agents%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2018%26volume%3D67%26spage%3D1803%26epage%3D1814%26doi%3D10.1093%2Fcid%2Fciy378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EMV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EMV','PDB','4EMV'); return false;">PDB: 4EMV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG','PDB','4KFG'); return false;">PDB: 4KFG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S','PDB','6M1S'); return false;">PDB: 6M1S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1J','PDB','6M1J'); return false;">PDB: 6M1J</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i157"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00768">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_27838"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00768?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00768</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR, <sup>13</sup>C NMR, and LC-MS spectra of compounds <b>15r</b>, <b>17r</b>, and <b>17x</b>; PK/PD correlation of compound <b>17r</b> in neutropenic mouse thigh infection model (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of compounds <b>10</b>, <b>12</b>, <b>15</b>, and <b>17</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_001.pdf">jm0c00768_si_001.pdf (877.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00768/suppl_file/jm0c00768_si_002.csv">jm0c00768_si_002.csv (5.9 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB codes for <b>12o</b> and <b>12x</b> are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1S">6M1S</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M1J">6M1J</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00768%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00768" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679913756af43d24","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
