
<html lang="en"     class="pb-page"  data-request-id="89814e33-5763-4f32-862c-8036454872bb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b01291;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-23"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based and Property-Driven Optimization of N-Aryl Imidazoles toward Potent and Selective Oral RORγt Inhibitors" /></meta><meta name="dc.Creator" content="Klemens  Hoegenauer" /></meta><meta name="dc.Creator" content="Joerg  Kallen" /></meta><meta name="dc.Creator" content="Eloísa  Jiménez-Núñez" /></meta><meta name="dc.Creator" content="Ross  Strang" /></meta><meta name="dc.Creator" content="Peter  Ertl" /></meta><meta name="dc.Creator" content="Nigel G.  Cooke" /></meta><meta name="dc.Creator" content="Samuel  Hintermann" /></meta><meta name="dc.Creator" content="Markus  Voegtle" /></meta><meta name="dc.Creator" content="Claudia  Betschart" /></meta><meta name="dc.Creator" content="Daniel J. J.  McKay" /></meta><meta name="dc.Creator" content="Juergen  Wagner" /></meta><meta name="dc.Creator" content="Johannes  Ottl" /></meta><meta name="dc.Creator" content="Christian  Beerli" /></meta><meta name="dc.Creator" content="Andreas  Billich" /></meta><meta name="dc.Creator" content="Janet  Dawson" /></meta><meta name="dc.Creator" content="Klemens  Kaupmann" /></meta><meta name="dc.Creator" content="Markus  Streiff" /></meta><meta name="dc.Creator" content="Nathalie  Gobeau" /></meta><meta name="dc.Creator" content="Stephanie  Harlfinger" /></meta><meta name="dc.Creator" content="Rowan  Stringer" /></meta><meta name="dc.Creator" content="Christine  Guntermann" /></meta><meta name="dc.Description" content="Retinoic acid receptor-related orphan receptor gamma-t (RORγt) is considered to be the master transcription factor for the development of Th17 cells that produce proinflammatory cytokines such as I..." /></meta><meta name="Description" content="Retinoic acid receptor-related orphan receptor gamma-t (RORγt) is considered to be the master transcription factor for the development of Th17 cells that produce proinflammatory cytokines such as I..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 15, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01291" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01291" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01291" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01291" /></link>
        
    
    

<title>Structure-Based and Property-Driven Optimization of N-Aryl Imidazoles toward Potent and Selective Oral RORγt Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01291" /></meta><meta property="og:title" content="Structure-Based and Property-Driven Optimization of N-Aryl Imidazoles toward Potent and Selective Oral RORγt Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0020.jpeg" /></meta><meta property="og:description" content="Retinoic acid receptor-related orphan receptor gamma-t (RORγt) is considered to be the master transcription factor for the development of Th17 cells that produce proinflammatory cytokines such as IL-17A. Overproportionate Th17 cell abundance is associated with the pathogenesis of many inflammatory conditions including psoriasis. In a high-throughput fluorescence resonance energy transfer (FRET) screen, we identified compound 1 as a hit with promising lipophilic efficiency (LipE). Using structure-based drug design based on a number of X-ray cocrystal structures, we morphed this hit class into potent imidazoles, exemplified by compound 3. To improve the poor absorption, distribution, metabolism, and excretion (ADME) properties of neutral imidazoles, we extended our ligands with carboxylic acid substituents toward a polar, water-rich area of the protein. This highly lipophilicity-efficient modification ultimately led to the discovery of compound 14, a potent and selective inhibitor of RORγt with good ADME properties and excellent in vivo pharmacokinetics. This compound showed good efficacy in an in vivo delayed-type hypersensitivity pharmacology model in rats." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01291"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01291">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01291&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01291&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01291&amp;href=/doi/10.1021/acs.jmedchem.9b01291" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10816-10832</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01290" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01416" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based and Property-Driven Optimization of <i>N</i>-Aryl Imidazoles toward Potent and Selective Oral RORγt Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Klemens Hoegenauer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Klemens Hoegenauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a5cec9c0c8c0cbd68bcdcac0c2c0cbc4d0c0d7e5cbcad3c4d7d1ccd68bc6cac8"><span class="__cf_email__" data-cfemail="ec8780898189829fc28483898b89828d99899eac82839a8d9e98859fc28f8381">[email protected]</span></a>. Phone: (+41) 79 6181814.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Klemens++Hoegenauer">Klemens Hoegenauer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7266-0137" title="Orcid link">http://orcid.org/0000-0001-7266-0137</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joerg Kallen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joerg Kallen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Biology & Therapeutics, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joerg++Kallen">Joerg Kallen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eloísa Jiménez-Núñez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eloísa Jiménez-Núñez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elo%C3%ADsa++Jim%C3%A9nez-N%C3%BA%C3%B1ez">Eloísa Jiménez-Núñez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ross Strang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ross Strang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ross++Strang">Ross Strang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Ertl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Ertl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Ertl">Peter Ertl</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6496-4448" title="Orcid link">http://orcid.org/0000-0001-6496-4448</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nigel G. Cooke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nigel G. Cooke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nigel+G.++Cooke">Nigel G. Cooke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samuel Hintermann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samuel Hintermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samuel++Hintermann">Samuel Hintermann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6853-6803" title="Orcid link">http://orcid.org/0000-0001-6853-6803</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Markus Voegtle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus Voegtle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Voegtle">Markus Voegtle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Claudia Betschart</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claudia Betschart</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claudia++Betschart">Claudia Betschart</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel J. J. McKay</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel J. J. McKay</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, 181 Massachusetts Avenue, 02139 Cambridge, Massachusetts, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+J.+J.++McKay">Daniel J. J. McKay</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Juergen Wagner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Juergen Wagner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Juergen++Wagner">Juergen Wagner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johannes Ottl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johannes Ottl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Biology & Therapeutics, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johannes++Ottl">Johannes Ottl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Beerli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Beerli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Autoimmunity, Transplantation and Inflammation, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Beerli">Christian Beerli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Billich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Billich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Autoimmunity, Transplantation and Inflammation, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Billich">Andreas Billich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Janet Dawson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janet Dawson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Autoimmunity, Transplantation and Inflammation, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janet++Dawson">Janet Dawson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Klemens Kaupmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Klemens Kaupmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Autoimmunity, Transplantation and Inflammation, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Klemens++Kaupmann">Klemens Kaupmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Markus Streiff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus Streiff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Autoimmunity, Transplantation and Inflammation, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Streiff">Markus Streiff</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nathalie Gobeau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nathalie Gobeau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PK Sciences, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nathalie++Gobeau">Nathalie Gobeau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephanie Harlfinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephanie Harlfinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PK Sciences, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephanie++Harlfinger">Stephanie Harlfinger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rowan Stringer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rowan Stringer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PK Sciences, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rowan++Stringer">Rowan Stringer</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Christine Guntermann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine Guntermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Autoimmunity, Transplantation and Inflammation, Novartis Institutes for BioMedical Research, 4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Guntermann">Christine Guntermann</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01291&amp;href=/doi/10.1021%2Facs.jmedchem.9b01291" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10816–10832</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 15, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 August 2019</li><li><span class="item_label"><b>Published</b> online</span>15 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01291" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01291</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10816%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKlemens%2BHoegenauer%252C%2BJoerg%2BKallen%252C%2BElo%25C3%25ADsa%2BJim%25C3%25A9nez-N%25C3%25BA%25C3%25B1ez%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D23%26contentID%3Dacs.jmedchem.9b01291%26title%3DStructure-Based%2Band%2BProperty-Driven%2BOptimization%2Bof%2BN-Aryl%2BImidazoles%2Btoward%2BPotent%2Band%2BSelective%2BOral%2BROR%25CE%25B3t%2BInhibitors%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10832%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01291"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1681</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01291" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based and Property-Driven Optimization of N-Aryl Imidazoles toward Potent and Selective Oral RORγt Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Klemens&quot;,&quot;last_name&quot;:&quot;Hoegenauer&quot;},{&quot;first_name&quot;:&quot;Joerg&quot;,&quot;last_name&quot;:&quot;Kallen&quot;},{&quot;first_name&quot;:&quot;Eloísa&quot;,&quot;last_name&quot;:&quot;Jiménez-Núñez&quot;},{&quot;first_name&quot;:&quot;Ross&quot;,&quot;last_name&quot;:&quot;Strang&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Ertl&quot;},{&quot;first_name&quot;:&quot;Nigel&quot;,&quot;last_name&quot;:&quot;G. Cooke&quot;},{&quot;first_name&quot;:&quot;Samuel&quot;,&quot;last_name&quot;:&quot;Hintermann&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Voegtle&quot;},{&quot;first_name&quot;:&quot;Claudia&quot;,&quot;last_name&quot;:&quot;Betschart&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;J. J. McKay&quot;},{&quot;first_name&quot;:&quot;Juergen&quot;,&quot;last_name&quot;:&quot;Wagner&quot;},{&quot;first_name&quot;:&quot;Johannes&quot;,&quot;last_name&quot;:&quot;Ottl&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Beerli&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Billich&quot;},{&quot;first_name&quot;:&quot;Janet&quot;,&quot;last_name&quot;:&quot;Dawson&quot;},{&quot;first_name&quot;:&quot;Klemens&quot;,&quot;last_name&quot;:&quot;Kaupmann&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Streiff&quot;},{&quot;first_name&quot;:&quot;Nathalie&quot;,&quot;last_name&quot;:&quot;Gobeau&quot;},{&quot;first_name&quot;:&quot;Stephanie&quot;,&quot;last_name&quot;:&quot;Harlfinger&quot;},{&quot;first_name&quot;:&quot;Rowan&quot;,&quot;last_name&quot;:&quot;Stringer&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Guntermann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10816-10832&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01291&quot;},&quot;abstract&quot;:&quot;Retinoic acid receptor-related orphan receptor gamma-t (RORγt) is considered to be the master transcription factor for the development of Th17 cells that produce proinflammatory cytokines such as IL-17A. Overproportionate Th17 cell abundance is associated with the pathogenesis of many inflammatory conditions including psoriasis. In a high-throughput fluorescence resonance energy transfer (FRET) screen, we identified compound 1 as a hit with promising lipophilic efficiency (LipE). Using structure-based drug design based on a number of X-ray cocrystal structures, we morphed this hit class into potent imidazoles, exemplified by compound 3. To improve the poor absorption, distribution, metabolism, and excretion (ADME) properties of neutral imidazoles, we extended our ligands with carboxylic acid substituents toward a polar, water-rich area of the protein. This highly lipophilicity-efficient modification ultimately led to the discovery of compound 14, a potent and selective inhibitor of RORγt with good ADME proper&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01291&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01291" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01291&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01291" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01291&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01291" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01291&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01291&amp;href=/doi/10.1021/acs.jmedchem.9b01291" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01291" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01291" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01291%26sid%3Dliteratum%253Aachs%26pmid%3D31729873%26genre%3Darticle%26aulast%3DHoegenauer%26date%3D2019%26atitle%3DStructure-Based%2Band%2BProperty-Driven%2BOptimization%2Bof%2BN-Aryl%2BImidazoles%2Btoward%2BPotent%2Band%2BSelective%2BOral%2BROR%25CE%25B3t%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D23%26spage%3D10816%26epage%3D10832%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292253" title="Imidazoles">Imidazoles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/jmcmar.2019.62.issue-23/20191212/jmcmar.2019.62.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Retinoic acid receptor-related orphan receptor gamma-t (RORγt) is considered to be the master transcription factor for the development of Th17 cells that produce proinflammatory cytokines such as IL-17A. Overproportionate Th17 cell abundance is associated with the pathogenesis of many inflammatory conditions including psoriasis. In a high-throughput fluorescence resonance energy transfer (FRET) screen, we identified compound 1 as a hit with promising lipophilic efficiency (LipE). Using structure-based drug design based on a number of X-ray cocrystal structures, we morphed this hit class into potent imidazoles, exemplified by compound <b>3</b>. To improve the poor absorption, distribution, metabolism, and excretion (ADME) properties of neutral imidazoles, we extended our ligands with carboxylic acid substituents toward a polar, water-rich area of the protein. This highly lipophilicity-efficient modification ultimately led to the discovery of compound <b>14</b>, a potent and selective inhibitor of RORγt with good ADME properties and excellent in vivo pharmacokinetics. This compound showed good efficacy in an in vivo delayed-type hypersensitivity pharmacology model in rats.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42556" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42556" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Retinoic acid receptor-related orphan receptor gamma-t (RORγt) is the key transcription factor that is required for the differentiation, effector function, and cytokine production of IL-17-producing T-helper 17 (Th17) cells and innate immune cells.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Although these cells are important for host immunity against various bacteria and against <i>Candida</i>, they also produce pro-inflammatory cytokines that are drivers of various autoimmune and inflammatory diseases including psoriasis, psoriatic arthritis, ankylosing spondylitis, and rheumatoid arthritis.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Based on the importance of this transcription factor to direct pro-inflammatory Th17 cytokine production and Th17 effector function, RORγt is considered to be a highly attractive drug target for modulating the pathological Th17/IL-17 immune axis. Two isoforms of this nuclear hormone receptor are known, the widely expressed isoform RORγ and the immune cell-specific isoform RORγt.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> As the ligand-binding domain (LBD) is identical, low molecular weight binders will most likely modulate the transcriptional activity of the RORγ and the RORγt isoform. Various low molecular weight ligands targeting the LBD of RORγt were recently identified and some of these were shown to attenuate pro-inflammatory diseases in various in vivo rodent models.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11">(6−11)</a> Our goal was to identify a novel, structurally different compound class and to advance a compound toward a clinical candidate. Here, we report how we discovered <i>N</i>-aryl imidazoles as potent inhibitors of RORγt after a scaffold-morph of a screening hit. We disclose how we improved the series properties and ultimately identified a compound that showed in vivo efficacy after a low oral dose.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47367" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47367" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Medicinal Chemistry</h3><div class="NLM_p">Our first hit-finding effort toward inhibitors of the RORγt function involved a high-throughput screen using a TR-FRET format using the RORγt LBD and fluorescently labeled RIP140 as a co-activator peptide. The TR-FRET screen was run in order to identify compounds that disrupt the interaction of the RORγt LBD and RIP140 (“displacers of co-activator binding”)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> in the anticipation that this mechanism of action (MOA) will result in an inhibition of transcription on a cellular level (“RORγt inhibitors”). This screen identified a number of hits, and many of them were highly lipophilic. Amongst these hits, ester <b>1</b> caught our attention as an attractive starting point because of its more favorable lipophilic efficiency (LipE), see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Originating from a previous Ciba-Geigy fungicide program, this hit is a mixture of four components: racemic with respect to the stereocenter next to the ester and in addition atropisomeric because of hindered rotation around the <i>N</i>-aryl bond. When we determined the X-ray structure of ester <b>1</b> bound to RORγt (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">Figures S1 and S2</a>), we found that only one isomer cocrystallized as a density fit, for any of the other isomers was not possible. In order to reduce the complexity caused by atropisomers, we aimed for a symmetrical substitution on the aromatic ring and found the ortho and <i>ortho</i>-dichloro substitution pattern to be optimal. In addition, we found that the thiophene ring can be replaced by a furan ring that showed a slightly better potency in most cases. Intriguingly, the ester substitution pattern showed a steep structure–activity relationship (SAR). Although FRET activity for methyl esters was barely detectable, the comparatively small change to an ethyl ester resulted in a large activity increase. Adding additional ester bulk such as in isopropyl ester <b>2</b> increased the activity even further. A cocrystal structure of isopropyl ester <b>2</b> with the RORγt LBD helped us rationalize these SAR observations and generate ideas for further optimization (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Only the (<i>S</i>)-enantiomer of ester <b>2</b> was observed in the electron density. The compound binds close to the helix 11–helix 12 interface, which is critical for the transcriptionally active conformation, and was therefore termed the “business end”.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The H-bond between H479 (on helix 11) and Y502 (on helix 12), shielded from the bulk solvent in a highly lipophilic environment, is critical for this assembly. When this interaction is stabilized, such as with cholesterol sulfate,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> helix 12 is part of a groove (together with helix 3 and helix 4/5), and the helical LXXLL co-activator motif can bind to the LBD (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">Figure S4A</a>). However, when the helix 11–helix 12 interaction is destabilized and helix 12 is displaced, co-activators can no longer bind, and transcription is unlikely to occur. Compound <b>2</b> destabilizes this interface by the following distinct changes within the binding site: (1) the amide bond carbonyl forms a strong H-bond acceptor with H479 (bond length 2.7 Å), depriving Y502 of its binding partner, (2) the isopropyl ester sterically clashes with W317 (on helix 11), and as a consequence, (3) W317 is pushed away and binds into the area that used to be occupied by Y502. As a result, helix 12 is destabilized, does not form the co-activator-binding groove, and co-activator binding is prevented overall (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">Figure S4B</a>). We termed this “push–pull” mechanism (“push” on W317 and “pull” on H479), and both elements in the ligand molecule are necessary in order to destabilize co-activator binding. In addition, compound <b>2</b> complements the largely hydrophobic surface of this part of the binding site via undirected van der Waals contacts. This binding mode model explains the steep ester SAR mentioned earlier: a methyl ester is likely too small to cause W317 movement; thus, co-activator displacement is not observed in the FRET assay. Compared to ligands that extend all the way to the “sulfate pocket”,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> we felt that this binding mode close to the “business end” was advantageous as we should be able to keep the overall size of the molecule small because of the proximity to the crucial helix 11–helix 12 interface.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Superposition of RORγt LBD X-ray cocrystal structures in the complex with cholesterol sulfate (stabilizer of coactivator binding; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NTI">5NTI</a>, carbons in white) and ester <b>2</b> at 2.30 Å resolution (displacer of co-activator binding; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O">6Q6O</a>, carbons in cyan/yellow) explaining the interplay of ligands, namely, W317, H479, and Y502 for helix 12 stabilization; water molecules are omitted for clarity; (A) overall view of the binding pocket illustrating the “business end” and the sterical clash of ligand <b>2</b> with W317 in the helix 12 on position (red circle); (B) different orientation highlighting the differences of the H479 H-bond for both ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Scaffold Morphing of High-Throughput Screening Hit <b>1</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0015.gif" alt="" id="GRAPHIC-d7e478-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0016.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Mean of a minimum of two independent experiments; standard deviation for pIC<sub>50</sub> values < 0.3.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Displacement of the RORγt LBD RIP140 cofactor peptide, TR-FRET format.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Inhibition of IL-17 secretion in RORγt-transduced HUT78 T-cells.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Inhibition of IL-17 secretion in rat CD4+ T-cells polarized with a Th17 cytokine mix.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Inhibition of ConA-induced IL-17 secretion in rat/human whole blood.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">High-throughput log <i>P</i> measurement using octanol-coated immobilized artificial membranes.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">LipE<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> using FRET assay pIC<sub>50</sub> and HT-log <i>P</i>/<i>D</i>.</p></div></div><div></div></div><div class="NLM_p">In line with sufficient potency in the FRET assay, ester <b>2</b> also displayed cellular assay activity and reduced PMA + CD3 mAb-induced IL-17A production (IC<sub>50</sub> = 452 nM) in a T-cell line assay stably expressing RORγt. In order to further improve potency, we probed limiting the conformational flexibility and designed imidazoles <b>3</b> and <b>4</b>. In addition to an overall shape overlap, we expected that a large position-4 substituent would lead to W317 displacement and that the position-3 nitrogen would act as a strong H-bond acceptor for H479.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> This scaffold morph was successful and led to the first very potent compounds. We successfully solved the cocrystal structure for imidazole <b>4</b>, and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> shows this structure superimposed with the structure obtained for ester <b>2</b>. As intended, the binding mode is similar, and both ligands overlay well: the 4-benzyl substituent of ligand <b>4</b> dislodges W317, and the imidazole position-3 nitrogen accepts the H-bond of H479 as anticipated. We concluded that the affinity increase for imidazoles compared to esters such as <b>2</b> can largely be attributed to the conformational constraint. Cellularly highly active imidazoles <b>3</b> and <b>4</b> were crucial tool compounds to help establish important project assays, in particular, our Th17 assay (inhibition of IL-17A production in either CD4 positive T-cells polarized with a Th17 cytokine mix) and whole blood assays [concanavalin A (ConA)-stimulated rat/human whole blood], see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Superposition of RORγt LBD X-ray cocrystal structures in the complex with ester <b>2</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O">6Q6O</a>, carbons in white) and imidazole <b>4</b> at 2.20 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7A">6Q7A</a>, carbons in cyan/yellow) illustrates a similar binding mode; water molecules are omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although these first imidazoles were highly potent, we realized that this activity gain was mainly lipophilicity driven. A log <i>P</i> above the limit of quantification (>6.9) implied a LipE drop to <1.5, and in vitro ADME parameters were poor as expected (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>): low solubility and very high microsomal clearance (human clearance correlated well with rat clearance for this series, data not shown). In order to progress candidate molecules to in vivo experiments, we needed to identify more lipophilic efficient analogues, and the permissive SAR space is now described in more detail. R<sup>1</sup> substituents needed to be lipophilic in order to maintain potency. Although larger alkyl or substituted alkyl R<sup>1</sup> substituents were tolerated, we found that additional molecular weight was not needed and that a trifluoromethyl group was sufficient to maintain activity for most combinations. At position-5, only small substituents were accepted: a methyl group generally led to more potent derivatives than leaving this position unsubstituted. Being aware of the risk of furan rings to form reactive metabolites,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> we discovered that an additional 2-chloro or 2-methyl substituent on the furan was able to further increase the activity. Substituting thiophenes in this way has been shown to reduce the amount of glutathione (GSH) adducts,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and we hypothesized that in analogy, 2-chloro or 2-methyl furans may also bear a reduced potential for reactive metabolite formation. We tested representative 2-chlorofuran derivatives in metabolite identification studies and did not find any GSH adducts. <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> contains key derivatives that illustrate our LipE-driven optimization efforts toward a more favorable ADME space. Compounds <b>3</b> and <b>5</b> are representatives of a highly potent, yet highly lipophilic chemical space. In order to significantly lower the lipophilicity, we replaced the two aromatic chlorines with nitriles, switched from the furan to an oxazole and omitted the 5-methyl group on the imidazole. Compound <b>6</b> is indeed much less lipophilic (log <i>P</i> = 2.0), and while solubility markedly increased, microsomal clearance remained high (RLM Cl<sub>int</sub> = 153 μL·min<sup>–1</sup>·mg<sup>–1</sup>). In contrast, compound <b>7</b> had a significantly reduced metabolic turnover (RLM Cl<sub>int</sub> = 33 μL·min<sup>–1</sup>·mg<sup>–1</sup>), presumably because of a favorable combination of lipophilicity space (log <i>P</i> = 1.1) and a complete lack of metabolically vulnerable terminal methyl groups. However, compound <b>7</b> only showed residual FRET activity. We had to accept that although we were making lipophilicity efficient improvements (LipE = 5.0 and 4.6 for compounds <b>6</b> and <b>7</b>, respectively), this progress was futile as FRET activity was poor in absolute terms and insufficient for cellular activity. We concluded that our initial strategy to stay close to the “business end” was successful in terms of identifying comparatively small ligands with the desired functional activity to disturb the helix 11–helix 12 interface. At the same time, we learned that this part of the binding pocket is largely coated by lipophilic amino acid residues which severely restricted our efforts to identify ligands with good ADME properties. As a consequence, we directed our efforts to find polar areas of the protein that would be within reach with an extension on our ligands. To this end, we took advantage of the extensive protein cavity knowledge based on a number of X-ray cocrystal structures. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> shows an overlay of the ligand <b>4</b> cocrystal structure with the crystallographic water molecules identified in 18 other RORγt LBD crystal structures (most of them internally generated). This overlay suggested the presence of a water-rich area that is part of a channel leading to solvent. Examination of the ligand <b>4</b> cocrystal structure using Water Analysis Tools for Molecular Dynamics (WATMD)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> revealed a high oxygen and hydrogen voxel occupancy in this channel (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">Figure S3</a>). A SiteMap<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> analysis of this structure also indicated the presence of H-bond donor and acceptor sites in this area of the protein. Taken together, we were confident that reaching out into this part of the protein with polar extensions on our ligands would be tolerated and lead to either direct or water-mediated interactions with the protein. In order to substantially lower lipophilicity, we designed molecules containing a terminal carboxylic acid via the exit vector next to the aromatic chlorosubstituent, see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> for an example supported by docking studies.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As the phenyl ring is unsymmetrically substituted in this design, atropisomers around the aromatic C–N bond were generated as a consequence. For analogues with an <i>o</i>,<i>o</i>-dichlorophenyl substituent, atropisomers can be separated by chiral chromatography, and they do not interconvert to any detectable degree even at elevated temperatures. Analogues with a 5-methyl group on the imidazole are particularly stable. For one intermediate, we experimentally determined the rotation barrier which was also in reasonable agreement with quantum-chemical calculations (see <a class="ref internalNav" href="#sec2_3" aria-label="Synthetic Chemistry">Synthetic Chemistry</a> section and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">Supporting Information</a>). Compounds <b>8–14</b> are single atropisomers, and only the data for the active enantiomer are reported in the tables. When both atropisomers were tested in the FRET assay, there was always one clear active isomer and the other enantiomer was typically >100-fold less active. These observations are in good agreement with the binding mode where the additional substituent on the distomer severely clashes with the protein structure. From our first set of compounds designed this way, azetidine-linked carboxylic acids <b>8</b> and <b>9</b> were amongst the most potent. The X-ray cocrystal structure of compound <b>9</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) shows that the carboxylic acid group is embedded in a water-mediated H-bonding network between the backbone carbonyl groups of F377 and H323 explaining the good activity. Compared to neutral imidazoles <b>3</b> and <b>5</b>, lipophilicity could be reduced by more than three log units, and a favorable lipophilicity space combined with no significant loss in the FRET activity (LipE = 5.4 for compound <b>8</b> and 5.3 for compound <b>9</b>) was a breakthrough in the series as we could place compounds in a favorable ADME space for the first time. Similar to their neutral counterparts, these carboxylic acid derivatives were permeable and their FRET activity translated into cells. As for the neutral matched pair <b>6</b> and <b>7</b>, there was also a noticeable difference in the microsomal turnover between carboxylic acids <b>8</b> and <b>9</b>. In both cases, metabolic stability was achieved by omitting terminal methyl groups on the imidazole position-4 substituent. Amide <b>10</b> and tetrazole <b>11</b> confirm that it is the charge at neutral pH that is important and not the substitution pattern by itself. Amide <b>10</b> is neutral because of its increased lipophilicity prone to metabolism (log <i>D</i>@pH7.4 = 4.9, RLM Cl<sub>int</sub> = 169 μL·min<sup>–1</sup>·mg<sup>–1</sup>), whereas acidic tetrazole <b>11</b> (log <i>D</i>@pH7.4 = 2.5, RLM Cl<sub>int</sub> = 49 μL·min<sup>–1</sup>·mg<sup>–1</sup>) is metabolically turned over to a similar degree as carboxylic acid <b>9</b>. In summary, to keep excessive metabolism under control and to avoid metabolic switching, compounds need to satisfy two conditions: (1) placement in a favorable lipophilicity space (log <i>P</i>/<i>D</i> < 3.0) and (2) primary metabolic weak spots need to be blocked (furan substitution, no terminal methyl groups). Our next goal was to confirm that our optimization of in vitro ADME parameters translated into favorable in vivo pharmacokinetics. The rat PK results for compounds <b>8</b> and <b>9</b> are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. As anticipated by its in vitro profile, compound <b>8</b> exhibited high in vivo clearance (CL = 228 mL·min<sup>–1</sup>·kg<sup>–1</sup>), and as a result, the overall exposure was poor (AUC d.n. = 156 nM·h). In contrast, compound <b>9</b> showed a markedly reduced clearance (CL = 30 mL·min<sup>–1</sup>·kg<sup>–1</sup>) and exposure increase (AUC d.n. = 1179 nM·h). In addition, bioavailability (BAV = 80%) also improved significantly.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Protein cavity indicating positions of crystallographic waters from 19 X-ray structures revealing possible areas for polar ligand extension. Water molecules that form direct interactions with the protein are mostly conserved but positions do not perfectly overlap between individual structures because of protein flexibility. Networks of water molecules without direct protein contact are more variable and adapt to the ligand. Compound <b>4</b> is shown in orange, and compound <b>9</b> (in cyan) was designed assuming that variable waters can be displaced.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. RORγt LBD X-ray cocrystal structure in the complex with carboxylic acid <b>9</b> at 2.30 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H">6Q7H</a>); selected water molecules are shown as white spheres; in addition to the interactions described for compound <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), the carboxylic acid group of compound <b>9</b> extends into the more hydrophilic part of the LBD and is embedded in a water-mediated hydrogen bond network between CO-F377 and CO-H323.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Improving LipE and ADME Properties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0017.gif" alt="" id="GRAPHIC-d7e870-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0018.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Mean of a minimum of two independent experiments; standard deviation for pIC<sub>50</sub> values < 0.3.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Displacement of the RORγt-LBD RIP140 cofactor peptide, TR-FRET format.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Inhibition of IL-17 secretion in RORγt-transduced HUT78 T-cells.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Intrinsic clearance in incubations of rat liver microsomes [μL·min<sup>–1</sup>·mg<sup>–1</sup> protein], <i>n</i> = 1.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">PAMPA log <i>P</i><sub>e</sub> (pH = 6.8), <i>n</i> = 1.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">High-throughput equilibrium solubility determination [mM] (pH = 6.8).</p></div><div class="footnote" id="t2fn7"><sup><sup>g</sup></sup><p class="last">High-throughput log <i>P</i> measurement using octanol-coated immobilized artificial membranes; for charged compounds, high-throughput log <i>D</i> measurement octanol-buffer (pH = 7.4) based on 96-well shake flask equilibrium and LC/MSMS.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div><div class="footnote" id="t2fn8"><sup><sup>h</sup></sup><p class="last">LipE using FRET assay pIC<sub>50</sub> and HT-log <i>P</i>/<i>D</i>.</p></div><div class="footnote" id="t2fn9"><sup><sup>j</sup></sup><p class="last">Single atropisomer.</p></div><div class="footnote" id="t2fn10"><sup><sup>k</sup></sup><p class="last">Saturation not reached at 10 μM.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Rat PK Parameters for Compounds <b>8</b>, <b>9</b>, and <b>14</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center"><b>8</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><b>8</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><b>9</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><b>9</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><b>14</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><b>14</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dose [mg·kg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">1 (iv)</td><td class="colsep0 rowsep0" align="left">3 (po)</td><td class="colsep0 rowsep0" align="left">1 (iv)</td><td class="colsep0 rowsep0" align="left">3 (po)</td><td class="colsep0 rowsep0" align="left">1 (iv)</td><td class="colsep0 rowsep0" align="left">5 (po)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL [mL·min<sup>–1</sup>·kg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">228 ± 47</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">30 ± 8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2.7 ± 1.0</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">V<sub>SS</sub> [L·kg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">3.6 ± 1.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">4.0 ± 0.8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.0 ± 0.7</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> term. [h]</td><td class="colsep0 rowsep0" align="left">0.2 ± 0.4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">4.6 ± 0.7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">5.3 ± 2.6</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC d.n.<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a> [nM·h]</td><td class="colsep0 rowsep0" align="left">156 ± 31</td><td class="colsep0 rowsep0" align="left">70 ± 12</td><td class="colsep0 rowsep0" align="left">1179 ± 316</td><td class="colsep0 rowsep0" align="left">945 ± 158</td><td class="colsep0 rowsep0" align="left">12 680 ± 4035</td><td class="colsep0 rowsep0" align="left">11 391 ± 323</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BAV [%]</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">45 ± 8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">80 ± 13</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> d.n.<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a> [nM]</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">166 ± 163</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">181 ± 96</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2448 ± 77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> [h]</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.6 ± 0.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.3 ± 1.8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Mean values with standard deviation in brackets.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Male SD rats, <i>n</i> = 4.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">High variability of PK data in SD rats; values taken from a 3-day PK study in female DA rats, <i>n</i> = 4.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">d.n. = dose normalized to 1 mg·kg<sup>–1</sup>.</p></div></div></div><div class="NLM_p">With the confirmation that our in vitro ADME-based optimization led to the desired in vivo profile, we turned our attention toward generating compounds with improved cellular potency to assess the effect of RORγt inhibition in an in vivo model. We discovered that compound <b>12</b>, a homologue of compound <b>9</b>, was 2–7 times more potent, and its single-dose pharmacokinetic parameters were similar to those of compound <b>9</b>. However, when we dosed 50 mg/kg per day over 3 days, we were surprised to find that the AUC on day 3 dropped ca. 10-fold (compared to day 1), leading to insufficient target coverage over time. In addition, we identified a severe time-dependent CYP3A4 inhibition (TDI) flag for compound <b>12</b> that we needed to address. We decided to focus our optimization on reducing the TDI signal in the hope that major structural modifications would also lead to a mitigation of the exposure-drop after multiple administrations. To identify structural motifs that are associated with a high k<sub>obs</sub> value, we profiled a number of orthogonally substituted imidazoles in the TDI assay. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A illustrates that although the presence of a 2-furyl substituent undoubtedly leads to compounds with a higher TDI <i>k</i><sub>obs</sub> value, still approximately half of the derivatives were in the low risk bin (<i>k</i><sub>obs</sub> < 0.010) or in the medium risk bin (<i>k</i><sub>obs</sub> < 0.025).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> However, the simultaneous presence of both a 2-furyl and a 4-trifluoromethyl group on the imidazole markedly increases the chances for a high risk <i>k</i><sub>obs</sub> value (<i>k</i><sub>obs</sub> > 0.025, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) and not a single compound containing these two structural motifs displayed a <i>k</i><sub>obs</sub> in the low/medium risk bins. As the 4-CF<sub>3</sub> substituent on the imidazole had been a crucial structural feature to maintain low clearance, we needed to identify a suitable replacement. In search for a bulky, yet metabolically stable motif, we came across the trifluoromethylcyclopropyl group that has been suggested as a <i>tert</i>-butyl group replacement.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Although we knew that any replacement of the furan would result in a loss of affinity, we tried again to avoid this functionality at this point. Based on earlier SAR observations, we knew that a cyclopropyl replacement for the furan was associated with the least activity loss. We anticipated that we could counterbalance the expected loss in affinity by additional interactions of a well-designed substituent within the polar channel. To this end, we designed a large virtual library of carboxylic acids with different linkers to the aromatic ring and docked them into the protein using flexible water positioning.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> We prioritized docking poses that extended into the polar channel rather over those that stretched toward the sulfate pocket. Next, we synthesized a selection of library members with the best docking score, among them is the 2-azaspiro[3.3]heptane-containing carboxylic acid <b>14</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> shows that these changes indeed reduced the risk for reactive metabolite formation; both trifluoromethylcyclopropyl-containing analogues <b>13</b> and <b>14</b> displayed a very low <i>k</i><sub>obs</sub>. We tested both compounds in multiple-dose PK studies (3 days) and found that while furan <b>13</b> still showed a considerable exposure drop after 3 days, cyclopropyl analogue <b>14</b> maintained exposure after multiple administrations. A clear rationale for these differences is lacking.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Box plots illustrate that TDI <i>k</i><sub>obs</sub> values are higher for (A) 2-furyl-substituted imidazoles and (B) 2-furyl-5-trifluoromethyl-substituted imidazoles; values inside the box are between the upper (Q3) and the lower (Q1) quartile; the white line within the box indicates the median value; values in between whiskers are within 1.5 times the interquartile range; outliers are defined as compounds outside 1.5 times the interquartile range and are depicted as separate dots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of the X-ray cocrystal structure of compound <b>9</b> (in cyan) and the docking pose for compound <b>14</b> (brown). Docking was performed using flexible water positioning.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Improving TDI by Modification of the Imidazole Substitution in Positions 2 and 4</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0019.gif" alt="" id="GRAPHIC-d7e1355-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">FRET IC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> [nM]</th><th class="colsep0 rowsep0" align="center">Gal4 Jurkat IC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a> [nM]</th><th class="colsep0 rowsep0" align="center" char=".">IL-17 HUT78 IC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn4" aria-label="d">d</a> [nM]</th><th class="colsep0 rowsep0" align="center" char=".">IL-17 rat Th17 IC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn5" aria-label="e">e</a> [nM]</th><th class="colsep0 rowsep0" align="center" char=".">IL-17 rat blood IC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn6" aria-label="f">f</a> [nM]</th><th class="colsep0 rowsep0" align="center">IL-17 hu blood IC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn6" aria-label="f">f</a> [nM]</th><th class="colsep0 rowsep0" align="center">CYP3A4 TDI <i>k</i><sub>obs</sub><a class="ref internalNav" href="#t4fn7" aria-label="g">g</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><a class="ref internalNav" href="#t4fn8" aria-label="h">h</a></th><th class="colsep0 rowsep0" align="center" char=".">LipE<a class="ref internalNav" href="#t4fn9" aria-label="j">j</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b><a class="ref internalNav" href="#t4fn10" aria-label="k">k</a></td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="char" char=".">393</td><td class="colsep0 rowsep0" align="char" char=".">1689</td><td class="colsep0 rowsep0" align="char" char=".">4508</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">0.047</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b><a class="ref internalNav" href="#t4fn10" aria-label="k">k</a></td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="left">120</td><td class="colsep0 rowsep0" align="char" char=".">111</td><td class="colsep0 rowsep0" align="char" char=".">869</td><td class="colsep0 rowsep0" align="char" char=".">696</td><td class="colsep0 rowsep0" align="left">1671</td><td class="colsep0 rowsep0" align="left">>0.1</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b><a class="ref internalNav" href="#t4fn10" aria-label="k">k</a></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">120</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">306</td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="left">331</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b><a class="ref internalNav" href="#t4fn10" aria-label="k">k</a></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="left">151</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">387</td><td class="colsep0 rowsep0" align="char" char=".">702</td><td class="colsep0 rowsep0" align="left">1342</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Mean of a minimum of two independent experiments; standard deviation for pIC<sub>50</sub> values < 0.3.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Displacement of the RORγt-LBD RIP140 cofactor peptide, TR-FRET format.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Inhibition of luciferase reporter gene activity in an RORγt LBD Gal4 transactivation assay (Jurkat cells).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Inhibition of IL-17 secretion in RORγt-transduced HUT78 T-cells.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Inhibition of IL-17 secretion in rat CD4+ T-cells polarized with a Th17 cytokine mix.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">Inhibition of ConA-induced IL-17 secretion in rat/human whole blood.</p></div><div class="footnote" id="t4fn7"><sup><sup>g</sup></sup><p class="last">TDI <i>k</i><sub>obs</sub> [min<sup>–1</sup>].</p></div><div class="footnote" id="t4fn8"><sup><sup>h</sup></sup><p class="last">High-throughput log <i>D</i> measurement octanol buffer (pH = 7.4) based on 96-well shake flask equilibrium and LC/MSMS.</p></div><div class="footnote" id="t4fn9"><sup><sup>j</sup></sup><p class="last">LipE using FRET assay pIC<sub>50</sub> and HT-log <i>D</i>.</p></div><div class="footnote" id="t4fn10"><sup><sup>k</sup></sup><p class="last">Single atropisomer.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> In Vitro and In Vivo Characterization of Compound <b>14</b></h3><div class="NLM_p">Carboxylic acid <b>14</b> displayed all the desired on-target characteristics, and it is also selective over other isoforms, other nuclear receptors, and safety panel targets (LBD-Gal4 luciferase reporter gene assays: RORγt IC<sub>50</sub> = 151 nM; RORα, RORβ, FXR, RXRα, ERα, PPARγ, PR, PXR, GR, and vitD3 IC<sub>50</sub> > 10 μM, safety target panel assays see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">Supporting Information</a>). The low lipophilicity (log <i>D</i> @pH7.4 = 2.0) and simultaneous blocking of all metabolic weak spots resulted in excellent in vitro and in vivo clearance (RLM = 14 μL·min<sup>–1</sup>·mg<sup>–1</sup>, in vivo CL < 3 mL·min<sup>–1</sup>·kg<sup>–1</sup>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), good exposure, and supportive half-life in rats. Aqueous solubility was sufficient (HT-sol = 0.146 mM), and a high BAV (90%) was observed after p.o. dosing as a suspension. Consequently, compound <b>14</b> was qualified to be tested in our rat delayed-type hypersensitivity (DTH) model. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A shows that while a dose of 1 mg/kg of compound <b>14</b> did not lead to a reduction of the ear swelling ratio after an immunization/challenge protocol using methylated BSA as the antigen, doses of 5 and 15 mg/kg inhibited ear swelling to a similar degree as an anti-IL-17A mAb which served as the positive control. In addition, compound <b>14</b> reduced the frequency of IL-17-producing cells isolated from draining lymph nodes in a dose-dependent manner with a maximum reduction of ca. 50% (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Compound <b>14</b> dose-dependently reduces mBSA-induced DTH responses in female Lewis rats; the mean (+SEM) of the swelling ratios between antigen-challenged and vehicle-injected ears is shown (<i>n</i> = 5); (B) compound <b>14</b> dose-dependently reduces mBSA-induced frequencies of IL-17-producing draining lymph node cells; individual data and mean ± SEM are shown (<i>n</i> = 5); data are a representative from two independent studies; means were compared using ANOVA followed by Dunnett’s test (*<i>p</i> < 0.05, **<i>p</i> < 0.01).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Synthetic Chemistry</h3><div class="NLM_p">The synthesis of isopropyl ester <b>2</b> is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. When 2,6-dichloroaniline was acylated with furan-2-carbonyl chloride, we exclusively observed bis N-acylation. Mild basic saponification of this bisacylated intermediate led to the formation of <i>N</i>-(2,6-dichlorophenyl)furan-2-carboxamide. N-alkylation with ethyl 2-bromopropanoate, followed by saponification and re-esterification, led to isopropyl ester <b>2</b>. The synthetic approach toward <i>N</i>-aryl imidazoles that were used early in the project is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, exemplified by the synthesis of imidazole <b>4</b>. 2,6-Dichloroaniline was N-alkylated with 3-bromo-1-phenylbutan-2-one, and then acylated with furan-2-carbonyl chloride. The resulting ketoamide was cyclized to imidazole <b>4</b> using ammonium acetate in acetic acid. This condensation was typically low-yielding, and we found that using amidines as intermediates led to a much improved imidazole formation. <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a> illustrate this synthetic approach which was used for all remaining <i>N</i>-aryl imidazoles in this article. Amidine <b>A1</b> was formed via Lewis acid-promoted addition of the aniline to an appropriately functionalized nitrile. Amidine N-alkylation followed by cyclocondensation led to <i>N</i>-aryl imidazole intermediates <b>A2</b> and <b>A3</b>. Neutral analogues <b>3</b> and <b>5</b> were also prepared accordingly. 2,6-Dicyanophenyl imidazoles <b>6</b> and <b>7</b> were prepared via palladium-catalyzed cyanation of dichloro intermediate <b>A2</b>. For electron deficient R<sub>1</sub> substituents, the direct incorporation of a methyl group into imidazole position-5 via the bromoketone building block (<b>A1</b> → <b>A3</b>) was often sluggish. In these cases, intermediate <b>A2</b> was prepared, and the methyl group was introduced after Buchwald–Hartwig amination of the aminoester side chain (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) via bromination/iodination of the imidazole followed by palladium-catalyzed cross-coupling using trimethylboroxine (<b>A4</b> → <b>A5</b>). Most atropisomer separations were performed at the stage of <b>A5</b> intermediates using chiral chromatography. Final ester saponification liberated acids <b>8</b>, <b>9</b>, <b>12</b>, and <b>13</b>. Further functionalization of carboxylic acid <b>9</b> via ammonolysis of the acyl-imidazole gave carboxamide <b>10</b>. Tetrazole <b>11</b> was obtained from amide <b>10</b> via dehydration to the nitrile and subsequent [3 + 2] cycloaddition with trimethyltin azide (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Isopropylester <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) furan-2-carbonyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 40 °C, 18 h; (ii) NaOH (1 M, aq), MeOH, 40 °C, 2 h; (b) (i) ethyl 2-bromopropanoate, NaH, THF, 70 °C, 6 h; (ii) LiOH, H<sub>2</sub>O, MeOH, 70 °C, 18 h; (iii) SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 1.5 h, then <i>i</i>-PrOH, rt, 18 h.</p></p></figure><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Imidazole <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) 3-bromo-1-phenylbutan-2-one, NaHCO<sub>3</sub>, <i>i</i>-PrOH, 83 °C, 18 h; (ii) furan-2-carbonyl chloride, 1,2-dichloroethane, 90 °C, 3 d; (b) NH<sub>4</sub>OAc, AcOH, 130 °C, 2 h.</p></p></figure><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General <i>N</i>-Aryl Imidazole Synthesis via Amidine <b>A1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) AlMe<sub>3</sub> or AlCl<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub> or toluene; (b) NaHCO<sub>3</sub>, Na<sub>2</sub>SO<sub>4,</sub><i>i</i>-PrOH; (c) Zn(CN)<sub>2</sub>, PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, DMF, microwave.</p></p></figure><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Imidazoles <b>8</b>, <b>9</b>, <b>12</b>, and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amine, various conditions for Buchwald–Hartwig cross-coupling, see experimental; (b) (i) NBS or <i>N</i>-iodosuccinimide (NIS), acetonitrile; (ii) trimethylboroxine, Cs<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane; (c) LiOH, THF, H<sub>2</sub>O.</p></p></figure><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Amide <b>10</b> and Tetrazole <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,1′-carbonyldiimidazole (CDI), NH<sub>3</sub>, (7 M in MeOH), THF, rt, 2 h; (b) (i) TFAA, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 1 h; (ii) Me<sub>3</sub>SnN<sub>3</sub>, toluene, 100 °C, 2 d.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a> shows the synthesis of compound <b>14</b>, which follows the strategy outlined in <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a>. Lewis acid-catalyzed reaction of aniline <b>15</b> with cyclopropanenitrile gave amidine <b>16</b>. Condensation with 2-bromo-1-(1-(trifluoromethyl)cyclopropyl)ethan-1-one under basic conditions then led to the formation of racemic imidazole <b>17</b>. The enantiomers were separated by chromatography using a chiral stationary phase. In order to assess the racemization risk for subsequent reactions run at higher temperatures, we experimentally determined the rotation barrier by solving the Eyring equation using four different epimerization rate constants (deduced from monitoring the racemization of <b>17b</b> in DMA at 170, 180, 190, and 200 °C). Using this protocol, the rotation barrier of <b>17b</b> was calculated to be Δ<i>H</i> (rot.) = 31 kcal/mol, inferring a racemization half-life of 2.7 × 10<sup>6</sup> years at 25 °C, 98 days at 110 °C or 2.5 min at 250 °C. Using this knowledge, the undesired atropisomer <b>17b</b> was recycled by re-equilibration to the racemate at 250 °C within minutes. Next, the desired atropisomer <b>17a</b> was converted to ester <b>18</b> via a Buchwald–Hartwig amination at 110 °C, a temperature where negligible racemization occurs within the timeframe of the experiment. The 5-methyl group on the imidazole was then introduced in two steps via bromination, followed by Suzuki coupling with trimethylboroxine. Final saponification of the ester led to carboxylic acid <b>14</b>. The absolute (S) configuration of the atropisomer was confirmed by X-ray crystallography (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).</div><figure id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Small molecule X-ray structure of compound <b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Preparation of Compound <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) AlMe<sub>3</sub>, toluene, 0 °C to rt, 18 h; (b) NaHCO<sub>3</sub>, <i>i</i>-PrOH, 90 °C, 48 h; (c) optical resolution by preparative chiral HPLC; (d) DMA, microwave, 250 °C, 30 min; (e) Cs<sub>2</sub>CO<sub>3</sub>, RuPhos, RuPhos precatalyst, toluene, 100 °C, 3 h; (f) NBS, CH<sub>3</sub>CN, rt, 1 h; (g) (i) trimethylboroxine, K<sub>2</sub>CO<sub>3</sub>, Pd(Ph<sub>3</sub>P)<sub>4</sub>, DMF, 100 °C, 24 h; (ii) NaOH, MeOH, rt, 18 h.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">A detailed understanding of the binding mode of high-throughput screening hit <b>1</b> as revealed by the X-ray cocrystal structure enabled us to morph this scaffold into <i>o</i>,<i>o</i>-disubstituted <i>N</i>-aryl imidazoles as a novel class of RORγt inhibitors. The prototypical imidazoles <b>3</b> and <b>4</b> displayed ADME liabilities that are commonly observed for such highly lipophilic compounds, among them low solubility and low metabolic stability. Once polar substituents extending ligand binding into a water-rich area of the protein were introduced, our LipE-guided optimization approach led to compounds in a good ADME space. Again, knowledge gained by a number of cocrystal structures was instrumental for the success of this approach. We effectively replaced two substituents on the imidazole that had caused a strong TDI potential and ultimately identified compound <b>14</b> as a candidate for in vivo profiling. Compound <b>14</b> showed excellent pharmacokinetic properties in rats and dose-dependently reduced ear swelling in an mBSA-induced DTH model in rats. The discovery of the promising profile of compound <b>14</b> coincided with the finding that prolonged RORγt inhibitor treatment in rats causes progressive thymic alterations similar to those in RORγ-deficient mice (constitutive knockout and induced knockout in adult mice) prior to T cell lymphoma development.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> There is a risk that RORγt inhibitors ultimately phenocopy the observations with RORγ-deficient mice, and long-term studies are needed in order to assess the safety of oral RORγt inhibitors for chronic indications.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11633" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11633" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General</h3><div class="NLM_p">Unless otherwise stated, all reagents and solvents were purchased from commercial suppliers and used without further purification. <sup>1</sup>H NMR spectra were recorded on a Bruker 400 MHz or a Bruker 600 MHz NMR spectrometer. Chemical shifts are reported in parts per million (ppm) relative to an internal solvent reference in which they were run. Recorded peaks are listed in the order multiplicity (s, singlet; d, doublet; t, triplet; q, quartet; qt, quintet; m, multiplet; and br, broad), coupling constants, and number of protons. Final compounds were purified to ≥95% purity as assessed by <sup>1</sup>H NMR spectra and analytical liquid chromatography using the methods mentioned below. Ultra-performance liquid chromatography (UPLC) and liquid chromatography–mass spectrometry analyses were conducted using a Waters Acquity UPLC Binary Solvent Manager/Sample Organizer/Sample Manager/Column Manager and a Waters Acquity Photo Diode Array UV–vis detector at 210–450 nm (full scan) with the MS detection performed on a MS Waters Acquity SQD (single quadrupole detector). Preparative reverse-phase high-performance liquid chromatography (HPLC) purifications were performed using either method A or method B, as described below. All flash column chromatography was performed with prepacked Biotage or Teledyne RediSep silica cartridges and performed on either a Biotage or Isco purification system. UPLC/MS analyses were performed using the conditions described in methods 1–4 below. High-resolution mass analyses (HRMS): LC/ESI–MS data were recorded using a Thermo Scientific Orbitrap mass spectrometer (Q Exactive Plus) with an electrospray ionization source and a Thermo Scientific UltiMate 3000 liquid chromatograph. The instrument was externally calibrated with Thermo Scientific Calibration Solution. The accurate mass was obtained by manual averaging scans at a mass resolution of ca. 70 000 (full width at half-maximum definition). The absolute configuration of compound <b>14</b> (single atropisomer) was confirmed by X-ray crystallography. The absolute configuration of compound <b>9</b> (single atropisomer) was confirmed by the RORγt LBD X-ray cocrystal structure in the complex with <b>9</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H">6Q7H</a>). The absolute configuration of single atropisomers <b>8</b>, <b>10</b>, <b>11</b>, <b>12</b>, and <b>13</b> was deduced by analogy based on their biological activity.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> UPLC/MS Method 1</h4><div class="NLM_p last">Column Acquity HSS T3 1.8 μm 2.1 × 50 mm at 60 °C; eluant A: water + 0.05% formic acid + 0.05% ammonium acetate; eluant B: acetonitrile + 0.04% formic acid; gradient: from 5 to 98% B in 1.4 min; and then 98% B over 0.4 min with solvent flow 1.0 mL/min.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> UPLC/MS Method 2</h4><div class="NLM_p last">Column Acquity UPLC BEH C18 1.7 μm 2.1 × 50 mm at 60 °C; eluant A: water + 0.1% formic acid; eluant B: acetonitrile; gradient: from 5 to 10% B in 1.00 min; then from 10 to 90% B over 3.00 min; and then 100% B for 0.40 min with solvent flow 0.7 mL/min.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> UPLC/MS Method 3</h4><div class="NLM_p last">Column AscentisExpress column C18 2.7 μm 30 × 2.1 mm (Supelco) at 60 °C; eluant A: water + 0.05% formic acid + 3.75 mM ammonium acetate; eluant B: acetonitrile + 0.04% formic acid; gradient: from 2 to 98% B in 1.40 min; and then 98% B over 0.75 min with solvent flow 1.0 mL/min.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> UPLC/MS Method 4</h4><div class="NLM_p last">Column Acquity HSS T3 1.8 μm 2.1 × 50 mm at 60 °C at 60 °C; eluant A: water + 0.05% formic acid; eluant B: acetonitrile + 0.04% formic acid; gradient: from 5 to 98% B in 1.4 min; and then 98% B over 0.4 min with solvent flow 1.0 mL/min.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Preparative Reverse-Phase Gilson HPLC Method A</h4><div class="NLM_p last">Column Sun Fire prep C18 OBD 5 μm, 30 × 100 mm from WATERS, with H<sub>2</sub>0 + 0.1% TFA and acetonitrile + 0.1% TFA as the mobile phase. Detection method: UV 220–400 nm.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Preparative Reverse-Phase Gilson HPLC Method B</h4><div class="NLM_p last">Column X-Select C18 OBD 5 μm, 19 × 50 mm from WATERS, with water + 1% formic acid and acetonitrile + 1% formic acid as the mobile phase. Detection method: UV 220–400 nm.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Flow Injection Mass Spectrometry Method 1</h4><div class="NLM_p last">Using a Waters Acquity SQD (single quadrupole detector) flow: 0.4 mL/min; eluant A: water + 0.05% formic acid + 0.05% ammonium acetate; eluant B: acetonitrile + 0.04% formic acid; gradient: isocratic A/B (40/60).</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Isopropyl <i>N</i>-(2,6-Dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)alaninate (<b>2</b>)</h4><div class="NLM_p">2,6-Dichloroaniline (5.0 g, 30.9 mmol) and Et<sub>3</sub>N (9.4 g, 93 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (100 mL). Then, furan-2-carbonyl chloride (10.1 g, 77 mmol) was slowly added, and the mixture was heated to 40 °C for 18 h. The reaction mixture was cooled and partitioned between EtOAc and a saturated aqueous NaHCO<sub>3</sub> solution. The organic phase was washed with a saturated aqueous NaHCO<sub>3</sub> solution and brine. The combined aqueous phases were extracted with EtOAc, and the combined organic phases were dried (MgSO<sub>4</sub>) and concentrated under reduced pressure. The residue was triturated with MeOH to give <i>N</i>-(2,6-dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)furan-2-carboxamide as a reddish solid (7.2 g, 66%) which was used in the next step without further purification. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 2.94 min, 350.0 [M + H]<sup>+</sup>.</div><div class="NLM_p"><i>N</i>-(2,6-Dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)furan-2-carboxamide (7.1 g, 20.4 mmol) was suspended in a mixture of MeOH (200 mL) and NaOH (1 M aqueous solution, 44.9 mL, 44.9 mmol). The mixture was heated to 40 °C for 2 h. The yellow solution was cooled to room temperature, and MeOH was removed under reduced pressure. The aqueous phase was extracted with EtOAc, and the organic phase was washed with NaOH (1 M aqueous solution), H<sub>2</sub>O (2×), and brine (2×). The combined aqueous phases were extracted again with EtOAc, and the combined organic phases were dried (MgSO<sub>4</sub>) and concentrated under reduced pressure to give <i>N</i>-(2,6-dichlorophenyl)furan-2-carboxamide as an off-white solid (4.8 g, 90%) which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 10.24 (s, 1H), 7.96 (d, <i>J</i> = 1.7 Hz, 1H), 7.59 (d, <i>J</i> = 8.1 Hz, 2H), 7.41 (dd, 1H), 7.33 (d, <i>J</i> = 3.5 Hz, 1H), 6.72 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 2.38 min, 256.0 [M + H]<sup>+</sup>.</div><div class="NLM_p">NaH (60% dispersion in mineral oil, 1.1 g, 27.5 mmol) was slowly added to a solution of <i>N</i>-(2,6-dichlorophenyl)furan-2-carboxamide (4.8 g, 18.3 mmol) in tetrahydrofuran (THF) (120 mL) under argon. The white suspension was stirred at room temperature for 30 min. Then, ethyl 2-bromopropanoate (3.34 mL, 25.7 mmol) was added, and the mixture was heated to 70 °C for 6 h. The reaction mixture was cooled to room temperature, diluted with EtOAc, and washed with brine (3×). The combined aqueous phases were extracted with EtOAc, and the organic phases were combined, dried (MgSO4), and concentrated under reduced pressure. The crude product was purified by repeated column chromatography (Isco Combiflash, 40 g SiO<sub>2</sub> RediSep) with cyclohexane/CH<sub>2</sub>Cl<sub>2</sub>, 80/20 to 0/100. The combined material was triturated with pentane to give ethyl <i>N</i>-(2,6-dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)alaninate (5.2 g, 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.70–7.61 (m, 3H), 7.51 (t, <i>J</i> = 8.1 Hz, 1H), 6.45 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 6.21 (d, <i>J</i> = 3.6 Hz, 1H), 4.51 (q, <i>J</i> = 7.1 Hz, 1H), 4.13 (qd, <i>J</i> = 7.1, 2.6 Hz, 2H), 1.24 (d, <i>J</i> = 7.2 Hz, 3H), 1.19 (t, <i>J</i> = 7.1 Hz, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 3.03 min, 356.1 [M + H]<sup>+</sup>.</div><div class="NLM_p">Ethyl <i>N</i>-(2,6-dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)alaninate (0.5 g, 1.4 mol) was suspended in a mixture of MeOH (10 mL) and H<sub>2</sub>O (2 mL) in a microwave vial. Then, LiOH (236 mg, 5.6 mmol) was added, the vial was capped, and the mixture was heated at 70 °C for 18 h until all solids were completely dissolved. After cooling to room temperature, the solution was neutralized with HCl (1 M aqueous, 1.4 mL) resulting in a white suspension. The solvents were removed under reduced pressure, and the residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (200 mL) and dried (MgSO<sub>4</sub>). Concentration under reduced pressure gave <i>N</i>-(2,6-dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)alanine (456 mg, 95%) which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 12.57 (s, 1H), 7.68–7.60 (m, 3H), 7.50 (t, <i>J</i> = 8.1 Hz, 1H), 6.44 (dd, 3.6, 1.8 Hz, 1H), 6.20 (d, 3.6 Hz, 1H), 4.43 (q, 7.2 Hz, 1H), 1.23 (d, 7.2 Hz, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 2.52 min, 328.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">SOCl<sub>2</sub> (0.07 mL, 0.9 mmol) was slowly added to a suspension of <i>N</i>-(2,6-dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)alanine (100 mg, 0.31 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). The mixture turned into a reddish solution and was stirred at room temperature for 1.5 h. <i>i</i>-PrOH (0.47 mL, 6.1 mmol) was added, and the resulting solution was stirred at room temperature for 18 h. The mixture was concentrated under reduced pressure, and the crude product was purified by column chromatography using CH<sub>2</sub>Cl<sub>2</sub>/MeOH 100/0 to 97.5/2.5 to give isopropyl <i>N</i>-(2,6-dichlorophenyl)-<i>N</i>-(furan-2-carbonyl)alaninate (<b>2</b>) as a yellow solid (81 mg, 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.69–7.61 (m, 3H), 7.52 (t, <i>J</i> = 8.2 Hz, 1H), 6.45 (dd, 3.5, 1.7 Hz, 1H), 6.20 (d, 3.6 Hz, 1H), 4.98–4.88 (m, 1H), 4.46 (q, 7.2 Hz, 1H), 1.25–1.16 (m, 9H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.16 min, 370.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>17</sub>H<sub>18</sub>Cl<sub>2</sub>NO<sub>4</sub><sup>+</sup> [M + H]<sup>+</sup>: 370.0607. Found: 369.0535.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 4-Benzyl-1-(2,6-dichlorophenyl)-2-(furan-2-yl)-5-methyl-1<i>H</i>-imidazole (<b>4</b>)</h4><div class="NLM_p">2,6-Dichloroaniline (2.33 g, 14.36 mmol), 3-bromo-1-phenylbutan-2-one (3.26 g, 14.36 mmol), and NaHCO<sub>3</sub> (1.33 g, 15.79 mmol) were dissolved in <i>i</i>-PrOH (10 mL) and heated at reflux for 18 h. The reaction mixture was concentrated under reduced pressure and partitioned between CH<sub>2</sub>Cl<sub>2</sub> (30 mL) and water (20 mL). The organic phase was separated by filtration through a phase separation tube and concentrated under reduced pressure. The crude product was purified by column chromatography on silica gel using heptane/CH<sub>2</sub>Cl<sub>2</sub>, 100/0 to 86/14 then 0/100 to give 3-((2,6-dichlorophenyl)amino)-1-phenylbutan-2-one as orange oil (740 mg, 16%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.38–7.15 (m, 7H), 6.89 (t, 1H), 5.15 (d, N–H), 4.70 (m, 1H), 4.00 (s, 2H), 1.29 (d, 3H) UPLC/MS (ESI) <i>m</i>/<i>z</i> method 3: 1.31 min, 307.9 [M + H]<sup>+</sup>.</div><div class="NLM_p last">In a glass vial, 3-((2,6-dichlorophenyl)amino)-1-phenylbutan-2-one (395 mg, 1.28 mmol) and furan-2-carbonyl chloride (184 mg, 1.41 mmol) were dissolved in 1,2-dichloroethane (4 mL). The vial was capped and heated with stirring for 72 h at 90 °C. The black solution was concentrated under reduced pressure, and glacial AcOH (2 mL) and ammonium acetate (988 mg, 12.82 mmol) were added to the residue. The reaction mixture was transferred to a glass vial, and the vial was capped and heated with stirring for 2 h at 130 °C. The mixture was cooled to room temperature and partitioned between EtOAc (100 mL) and saturated aqueous NaHCO<sub>3</sub> solution (100 mL). The organic layer was separated, washed with brine (2 × 20 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The mixture was purified using Biotage amino silica gel eluting with CH<sub>2</sub>Cl<sub>2</sub>/MeOH, 100/0 to 80/20. The crude product was triturated with diethylether/<i>n</i>-pentane to give 4-benzyl-1-(2,6-dichlorophenyl)-2-(furan-2-yl)-5-methyl-1<i>H</i>-imidazole (<b>4</b>) as an off-white solid (103 mg, 21%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ ppm 7.71–7.66 (m, 2H), 7.63–7.58 (m, 1H), 7.40 (dd, <i>J</i> = 1.9, 0.7 Hz, 1H), 7.33–7.24 (m, 4H), 7.22–7.16 (m, 1H), 6.40 (dd, <i>J</i> = 3.5, 1.9 Hz, 1H), 6.19 (dd, <i>J</i> = 3.6, 0.8 Hz, 1H), 4.04 (s, 2H), 1.94 (s, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.23 min, 383.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>21</sub>H<sub>17</sub>Cl<sub>2</sub>N<sub>2</sub>O<sup>+</sup> [M + H]<sup>+</sup>: 383.0712. Found: 383.0709.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-(5-Chlorofuran-2-yl)-1-(2,6-dichlorophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole (<b>5</b>)</h4><div class="NLM_p">5-Chlorofuran-2-carbonitrile (2.0 g, 15.7 mmol) was added to a solution of 2,6-dichloroaniline (2.54 g, 15.7 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (8 mL). Aluminum chloride (2.1 g, 15.7 mmol) was added in small portions. The resulting mixture was then stirred at room temperature for 18 h and poured onto ice (50 mL). NaOH (2.0 M, aqueous) was added until pH 14 was reached. The mixture was extracted with EtOAc (110 mL), and the organic layer was dried (MgSO4), filtered, and concentrated under reduced pressure to give 5-chloro-<i>N</i>-(2,6-dichlorophenyl)furan-2-carboximidamide as brown oil that solidified on standing (3.93 g, 82%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.39 (d, <i>J</i> = 8.0 Hz, 2H), 7.30–7.13 (m, 1H), 6.99 (t, <i>J</i> = 8.0 Hz, 1H), 6.75–6.55 (m, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 0.74 min, 288.9 [M + H]<sup>+</sup>.</div><div class="NLM_p last">In a glass vial, NaHCO<sub>3</sub> (174 mg, 2.07 mmol) and Na<sub>2</sub>SO<sub>4</sub> (147 mg, 1.04 mmol) were added to a solution of 5-chloro-<i>N</i>-(2,6-dichlorophenyl)furan-2-carboximidamide (300 mg, 1.04 mmol) in isopropanol (3 mL). The mixture was heated to 60 °C, and then 3-bromo-1,1,1-trifluorobutan-2-one (1.23 g, 6.0 mmol) was added. The vial was capped and heated with stirring for 24 h at 110 °C. Again, 3-bromo-1,1,1-trifluorobutan-2-one (1.23 g, 6.0 mmol) was added and heating was continued for 18 h at 110 °C. The mixture was cooled to room temperature and partitioned between EtOAc (100 mL) and saturated aqueous NaHCO<sub>3</sub> solution (100 mL). The organic layer was separated, washed with brine (2 × 20 mL), dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The mixture was purified by flash chromatography over silica gel eluting with cyclohexane/CH<sub>2</sub>Cl<sub>2</sub>, 100/0 to 50/50 to give 2-(5-chlorofuran-2-yl)-1-(2,6-dichlorophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole (<b>5</b>) as a pale yellow solid (243 mg, 58%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.87 (m, 2H), 7.76 (dd, <i>J</i> = 9.0, 7.4 Hz, 1H), 6.56 (d, <i>J</i> = 3.5 Hz, 1H), 6.30 (d, <i>J</i> = 3.6 Hz, 1H), 2.11 (s, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.30 min, 395.2 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>15</sub>H<sub>9</sub>Cl<sub>3</sub>F<sub>3</sub>N<sub>2</sub>O<sup>+</sup> [M + H]<sup>+</sup>: 394.9727. Found: 394.9726.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-(4-Isobutyl-2-(oxazol-5-yl)-1<i>H</i>-imidazole-1-yl)isophthalonitrile (<b>6</b>)</h4><div class="NLM_p">2,6-Dichloroaniline (500 mg, 3.09 mmol) and 5-oxazolecarbonitrile (290 mg, 3.09 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (4 mL) under argon. AlCl<sub>3</sub> (412 mg, 3.09 mmol) was added, and the suspension was stirred at room temperature under argon overnight. The reaction mixture was quenched with 2 M NaOH (5 mL) and extracted with EtOAc. The organic layer was washed twice with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash chromatography over silica gel eluting with cyclohexane/EtOAc, 100/0 to 0/20 to give <i>N</i>-(2,6-dichlorophenyl)oxazole-5-carboximidamide as colorless foam (360 mg, 45%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 8.53 (s, 1H), 7.81 (s, 1H), 7.40 (d, <i>J</i> = 8.1 Hz, 2H), 7.01 (t, <i>J</i> = 8.1 Hz, 1H), 6.82 (s, 2H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 0.93 min, 256.1 [M + H]<sup>+</sup>.</div><div class="NLM_p">In a microwave vial, <i>N</i>-(2,6-dichlorophenyl)oxazole-5-carboximidamide (150 mg, 0.586 mmol) and 1-bromo-4-methylpentan-2-one (210 mg, 1.05 mmol) were dissolved in <i>i</i>-PrOH (4 mL). NaHCO<sub>3</sub> (98 mg, 1.17 mmol) was added, the vial was sealed, and the mixture was stirred at 95 °C overnight. The reaction mixture was filtered, concentrated under reduced pressure, and purified by preparative reverse-phase HPLC (method A). The product fractions were treated with saturated NaHCO<sub>3</sub> solution and extracted twice with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give 5-(1-(2,6-dichlorophenyl)-4-isobutyl-1<i>H</i>-imidazole-2-yl)oxazole as an orange resin (137 mg, 66%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ ppm 8.16 (s, 1H), 7.72–7.65 (m, 2H), 7.61 (dd, <i>J</i> = 9.2, 6.9 Hz, 1H), 7.08 (s, 1H), 6.84 (s, 1H), 2.57 (d, <i>J</i> = 7.2 Hz, 2H), 2.06 (hept, <i>J</i> = 6.8 Hz, 1H), 1.01 (d, <i>J</i> = 6.6 Hz, 6H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 2.98 min, 336.1 [M + H]<sup>+</sup>.</div><div class="NLM_p last">In a microwave-vial, 5-(1-(2,6-dichlorophenyl)-4-isobutyl-1<i>H</i>-imidazole-2-yl)oxazole (122 mg, 0.363 mmol) and zinc cyanide (170 mg, 1.451 mmol) were suspended in dimethylformamide (DMF) (2 mL). The mixture was flushed with argon, and PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> (148 mg, 0.181 mmol) was added. The vial was sealed and heated in a microwave at 200 °C for 0.5 h. The reaction mixture was filtered through a Minisart RC 25, 0.45 μm filter and washed with CH<sub>2</sub>Cl<sub>2</sub> (15 mL). The resulting solution was concentrated under reduced pressure and purified by preparative reverse-phase HPLC (method A). The product was passed through an ion-exchange-cartridge (PL-HCO<sub>3</sub> MP-resin) eluting with CH<sub>3</sub>CN/H<sub>2</sub>O (9/1) and then lyophilized to give 2-(4-isobutyl-2-(oxazol-5-yl)-1<i>H</i>-imidazole-1-yl)isophthalonitrile (<b>6</b>) as a brown solid (42 mg, 35%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ ppm 8.29 (d, <i>J</i> = 8.0 Hz, 2H), 8.14 (s, 1H), 8.01–7.92 (m, 1H), 7.39 (s, 1H), 7.25 (s, 1H), 2.60 (dd, <i>J</i> = 7.1, 0.8 Hz, 2H), 2.09 (dp, <i>J</i> = 13.6, 6.8 Hz, 1H), 1.03 (d, <i>J</i> = 6.6 Hz, 6H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 0.89 min, 318.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>18</sub>H<sub>16</sub>N<sub>5</sub>O<sup>+</sup> [M + H]<sup>+</sup>: 318.1349. Found: 318.1346.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 2-(4-Cyclopropyl-2-(oxazol-5-yl)-1<i>H</i>-imidazole-1-yl)isophthalonitrile (<b>7</b>)</h4><div class="NLM_p">In a microwave vial, <i>N</i>-(2,6-dichlorophenyl)oxazole-5-carboximidamide (400 mg, 1.586 mmol) and 2-bromomethyl cyclopropylketone (804 mg, 4.69 mmol) were dissolved in <i>i</i>-PrOH (9 mL). NaHCO<sub>3</sub> (394 mg, 4.69 mmol) was added, and the vial was sealed and stirred at 95 °C overnight. The reaction mixture was filtered, concentrated under reduced pressure, and purified by flash chromatography eluting with cyclohexane/EtOAc; gradient 1–25% EtOAc, followed by a second purification via preparative reverse-phase HPLC (method A). The product fractions were treated with saturated NaHCO<sub>3</sub> solution and extracted twice with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give 5-(4-cyclopropyl-1-(2,6-dichlorophenyl)-1<i>H</i>-imidazole-2-yl)oxazole as a yellow solid (154 mg, 30%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ ppm 8.15 (s, 1H), 7.70–7.63 (m, 2H), 7.59 (dd, <i>J</i> = 9.2, 6.9 Hz, 1H), 7.05 (s, 1H), 6.84 (s, 1H), 1.98 (tt, <i>J</i> = 8.4, 5.1 Hz, 1H), 1.05–0.79 (m, 4H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 2: 2.91 min, 320.1 [M + H]<sup>+</sup>.</div><div class="NLM_p last">In a microwave-vial, 5-(4-cyclopropyl-1-(2,6-dichlorophenyl)-1<i>H</i>-imidazole-2-yl)oxazole (21 mg, 0.066 mmol) and zinc cyanide (30.8 mg, 0.262 mmol) were suspended in DMF (0.6 mL). The mixture was flushed with argon, and PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> (26.8 mg, 0.033 mmol) was added. The vial was sealed and heated in a microwave at 200 °C for 0.5 h. The reaction mixture was filtered through a Minisart RC 25, 0.45 μm filter and washed with CH<sub>2</sub>Cl<sub>2</sub> (15 mL). The resulting solution was concentrated under reduced pressure and purified by preparative reverse-phase HPLC (method A). The product was passed through an ion-exchange-cartridge (PL-HCO<sub>3</sub> MP-resin) eluting with CH<sub>3</sub>CN/H<sub>2</sub>O (9:1) and then lyophilized to give 2-(4-cyclopropyl-2-(oxazol-5-yl)-1<i>H</i>-imidazole-1-yl)isophthalonitrile (<b>7</b>) as an off-white solid (8.6 mg, 41%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ ppm 8.28 (d, <i>J</i> = 8.0 Hz, 2H), 8.12 (s, 1H), 7.95 (t, <i>J</i> = 8.0 Hz, 1H), 7.35 (s, 1H), 7.26 (s, 1H), 2.01 (tt, <i>J</i> = 8.4, 5.1 Hz, 1H), 1.05–0.93 (m, 2H), 0.96–0.83 (m, 2H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 0.76 min, 302.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>17</sub>H<sub>12</sub>N<sub>5</sub>O<sup>+</sup> [M + H]<sup>+</sup>: 302.1036. Found: 302.1037.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>S</i>)-1-(2,4-Dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-5-methyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylic Acid (<b>8</b>)</h4><div class="NLM_p">At 0 °C, NIS (12.25 g, 54.4 mmol) was added in portions to a solution of 1,3-dichloro-2-nitrobenzene (9.5 g, 49.5 mmol) in trifluoromethanesulfonic acid (25.05 mL, 282 mmol). The ice bath was removed, and the reaction mixture was then stirred at room temperature for 45 min. After that, the reaction mixture was quenched with ice water and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organic layers were washed with a saturated aqueous Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> solution, washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduce pressure to provide a light brown powder. MeOH (50 mL) was added, and this mixture was stirred for 1 h at room temperature and a white solid precipitated. The solid was filtered and washed with cold MeOH to give the title product as a white powder (9.5 g, 54%). The filtrate was concentrated under reduced pressure to give a light brown solid. The residue was again triturated in MeOH and filtered to give another batch of 1,3-dichloro-4-iodo-2-nitrobenzene as a beige solid (2.3 g, 12%). The material was used without further purification in the following step.</div><div class="NLM_p">AcOH (130 mL) and iron (17.13 g, 307 mmol) were added to a solution of 1,3-dichloro-4-iodo-2-nitrobenzene (19.5 g, 61.3 mmol) in ethanol (160 mL). The reaction mixture was heated to reflux for 2 h. The reaction mixture was allowed to cool down to room temperature and then filtered. The filtrate was concentrated under reduced pressure, and the residue was treated with water. The pH of the solution was adjusted to 7–8 by addition of a saturated aqueous NaHCO<sub>3</sub> solution. The aqueous solution was then extracted with EtOAc (3 × 500 mL), washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give 2,6-dichloro-3-iodoaniline (<b>15</b>) as a white solid (16.25 g, 89%) The material was used without further purification in the next step. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.17 min, 288.2 [M + H]<sup>+</sup>.</div><div class="NLM_p">5-Chlorofuran-2-carbonitrile (1.2 g, 9.63 mmol) and aluminum trichloride (1.4 g, 10.50 mmol) were added to a solution of 2,6-dichloro-3-iodoaniline (<b>15</b>) (3.0 g, 8.75 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL) under argon. The reaction mixture was stirred at room temperature for 2 h and then heated to 50 °C for 24 h. Once more, 5-chlorofuran-2-carbonitrile (0.446 g, 3.50 mmol) and aluminum trichloride (0.467 g, 3.50 mmol) were added, and the reaction mixture was stirred at 50 °C for 24 h. After that, the reaction mixture was allowed to cool to room temperature and concentrated under reduced pressure. The residue was suspended in water, and the mixture was adjusted to pH = 12 by addition of a NaOH solution (30%, aqueous). The mixture was then extracted with CH<sub>2</sub>Cl<sub>2</sub> (2×), and the combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give yellow oil. The crude product was purified by flash chromatography eluting with heptane/EtOAc (9/1) to give 5-chloro-<i>N</i>′-(2,6-dichloro-3-iodophenyl)furan-2-carboximidamide (1.8 g, 49%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.21(br s, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 6.79 (br s, 2H), 6.67 (d, <i>J</i> = 3.5 Hz, 1H) <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 148.2, 146.3, 145.6, 137.1, 133.5, 131.6, 129.8, 127.3, 114.9, 109.6, 98.4 UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.14 min, 414.8 [M + H]<sup>+</sup>.</div><div class="NLM_p">NaHCO<sub>3</sub> (1.02 g, 12.2 mmol) and 1-bromo-4-methylpentan-2-one (1.21 g, 90% purity, 6.08 mmol) were added to a solution of 5-chloro-<i>N</i>′-(2,6-dichloro-3-iodophenyl)furan-2-carboximidamide (1.7 g, 4.05 mmol) in isopropanol (10 mL) under argon. The reaction mixture was stirred at 95 °C for 16 h. The reaction mixture was allowed to cool to room temperature and concentrated under reduced pressure. The residue was dissolved in EtOAc and washed with water and brine. The organic phase was dried (MgSO<sub>4</sub>) and concentrated under reduced pressure. The crude product was purified by flash chromatography eluting with heptane/EtOAc (3/1) to give 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-4-isobutyl-1<i>H</i>-imidazole as yellow oil (1.74 g, 75% UPLC purity). The compound was used in the next step without further purification. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.43 min, 494.9 [M + H]<sup>+</sup>.</div><div class="NLM_p">In a microwave vial, 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-4-isobutyl-1<i>H</i>-imidazole (1.74 g, 75% UPLC purity, ca. 2.63 mmol), methyl azetidine-3-carboxylate (0.599 g, 3.95 mmol), Cs<sub>2</sub>CO<sub>3</sub> (2.57 g, 7.90 mmol), Xantphos (0.305 g, 0.527 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.241 g, 0.263 mmol) were mixed. The vial was sealed and purged with argon (3×). Next, dry dioxane (25 mL) was added, and the resulting suspension was heated at 95 °C for 62 h. After cooling to room temperature, the reaction mixture was filtered, and the filtrate was diluted with EtOAc. The solution was washed with water, a saturated aqueous NaHCO<sub>3</sub> solution, and brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give brown oil. This crude product was purified by flash chromatography eluting with heptan/EtOAc (3/1) to give methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate as light brown foam (0.7 g, 55% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.52 (d, <i>J</i> = 8.9 Hz, 1H), 7.05 (s, 1H), 6.86 (d, <i>J</i> = 9.1 Hz, 1H), 6.48 (d, <i>J</i> = 3.6 Hz, 1H), 5.95 (d, <i>J</i> = 3.6 Hz, 1H), 4.31–4.23 (m, 2H), 4.15–4.08 (m, 2H), 3.67 (s, 3H), 3.63–3.55 (m, 1H), 2.40 (d, <i>J</i> = 7 Hz, 2H), 2.00–1.90 (m, 1H), 0.91 (d, <i>J</i> = 6.6 Hz, 6H) ppm. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 147.4, 141.9, 136.1, 135.1, 133.7, 128.3, 121.3, 118.8, 116.2, 109.5, 108.8, 55.9, 52.0, 36.9, 32.6, 27.9, 22.2. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.34 min, 482.0 [M + H]<sup>+</sup>.</div><div class="NLM_p">NIS (358 mg, 1.591 mmol) was added to a solution of methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (640 mg, 1.326 mmol) in acetonitrile (13 mL) under argon. The reaction mixture was stirred at room temperature for 15 h. Next, EtOAc was added, and the organic solution was washed with a saturated aqueous NaHCO<sub>3</sub> solution, water, and brine. The organic layer was dried (MgSO<sub>4</sub>) and concentrated under reduced pressure to give yellow oil. The crude product was purified by flash chromatography eluting with heptane/EtOAc (3/1) to give methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-iodo-4-isobutyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate as yellow oil (0.6 g, 72% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.58 (d, <i>J</i> = 9.1 Hz, 1H), 6.91 (d, <i>J</i> = 9.2 Hz, 1H), 6.50 (d, <i>J</i> = 3.5 Hz, 1H), 5.97 (d, <i>J</i> = 3.7 Hz, 1H), 4.34–4.24 (m, 2H), 4.18–4.10 (m, 2H), 3.68 (s, 3H), 3.64–3.57 (m, 1H), 2.41 (d, <i>J</i> = 7.1 Hz, 2H), 2.10–1.95 (m, 1H), 0.92 (d, <i>J</i> = 6.6 Hz, 6H) ppm. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 172.8, 147.3, 146.2, 143.7, 139.1, 135.6, 133.4, 128.4, 122.0, 119.1, 116.7, 110.2, 109.0, 77.2, 55.9, 52.1, 36.9, 32.6, 28.6, 22.2 UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.43 min, 607.9 [M + H]<sup>+</sup>.</div><div class="NLM_p">In a flame-dried microwave vial, methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-iodo-4-isobutyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (460 mg, 0.756 mmol), Cs<sub>2</sub>CO<sub>3</sub> (616 mg, 1.889 mmol), and tetrakis(triphenylphosphine)palladium (87 mg, 0.076 mmol) were mixed under argon. The vial was purged with argon (3×). Then, dry dioxane (7.5 mL) and 2,4,6-trimethyl-1,3,5,2,4,6-trioxatriborinane (0.137 mL, 0.982 mmol) were added, and the reaction mixture was stirred at 80 °C for 16 h. After cooling to room temperature, the reaction mixture was filtered through a pad of Celite rinsing the filtering cake with EtOAc. The filtrate was partitioned between EtOAc and water, and the aqueous layer was extracted with EtOAc. The combined organic layers were washed with saturated aqueous NaHCO<sub>3</sub> solution and brine, dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure to give yellow oil. This crude product was purified by preparative reverse-phase HPLC (method B). The combined fractions were concentrated under reduced pressure. The remaining aqueous solution (pH = 2) was adjusted to pH = 9 with a saturated aqueous NaHCO<sub>3</sub> solution and then extracted with EtOAc (2×). The combined organic layers were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-5-methyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate as colorless foam (0.26 g, 69% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.56 (d, <i>J</i> = 8.9 Hz, 1H), 6.88 (d, <i>J</i> = 9 Hz, 1H), 6.45 (d, <i>J</i> = 3.6 Hz, 1H), 5.85 (d, <i>J</i> = 3.6 Hz, 1H), 4.33–4.24 (m, 2H), 4.19–4.10 (m, 2H), 3.67 (s, 3H), 3.63–3.55 (m, 1H), 2.38 (d, <i>J</i> = 7.1 Hz, 2H), 1.99–1.93 (m, 1H), 1.86 (s, 3H), 0.89 (d, <i>J</i> = 6.7 Hz, 6H) UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.35 min, 498.1 [M + H]<sup>+</sup>.</div><div class="NLM_p">A sample of methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-5-methyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (mixture of 2 atropisomers) (260 mg, 0.523 mmol) was separated by preparative chiral supercritical-fluid chromatography (SFC) to give two single, stable atropisomers: SFC-PicLab-Prep 100; column, CHIRALPAK AD-H 30 × 250 mm; BPR pressure, 150 bar; flow rate, 80 mL/min; mobile phase, scCO2/isopropanol/2-propylamine 85:15:0.15 isocratic; detector wavelength, 278 nm; injection, 0.3–0.8 mL (variable); cycle time, 12 min. The second peak eluting from the column provided (<i>S</i>)-methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-5-methyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (55 mg) as a single, stable atropisomer as a white solid. Chiral purity was determined using a Berger SFC; column, CHIRALPAK AD-H 4.6 × 250 mm; BPR pressure, 100 bar; flow rate, 3 mL/min; mobile phase, scCO2/isopropanol/2-propylamine 80:20:0.2 isocratic; detector wavelength, DAD 215 nm; injection volume, 10 μL; retention time: 3.95 min, chiral purity: >99.0%. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.35 min, 498.1 [M + H]<sup>+</sup>.</div><div class="NLM_p last">A solution of LiOH·H<sub>2</sub>O (8.45 mg, 0.201 mmol) in water (0.4 mL) was added to a solution of (<i>S</i>)-methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-5-methyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (50 mg, 0.101 mmol) in THF (0.8 mL). The mixture was stirred at room temperature for 2 h. After that time, the reaction mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> and quenched with HCl (1 M aqueous solution, 8 drops adjusting to pH = 4). The mixture was extracted with EtOAc, and the combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give white foam. Lyophilization from dioxane gave (<i>S</i>)-1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-isobutyl-5-methyl-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylic acid (<b>8</b>) as a white lyophilized powder (45 mg, 93% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.55 (d, <i>J</i> = 9 Hz, 1H), 6.87 (d, <i>J</i> = 9.1 Hz, 1H), 6.44 (d, <i>J</i> = 3.6 Hz, 1H), 5.84 (d, <i>J</i> = 3.5 Hz, 1H), 4.30–4.20 (m, 2H), 4.17–4.06 (m, 2H), 3.52–3.43 (m, 1H), 2.37 (d, <i>J</i> = 7.1 Hz, 2H), 2.0–1.88 (m, 1H), 1.87 (s, 3H), 0.89 (d, <i>J</i> = 6.6 Hz, 6H) ppm. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 173.88, 147.54, 144.54, 137.67, 134.71, 132.45, 128.44, 124.79, 121.42, 118.75, 116.36, 108.59, 56.07, 35.71, 32.87, 28.78, 22.12, 8.15 UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.17 min, 482.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>S</i>)-1-(2,4-Dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylic Acid (<b>9</b>)</h4><div class="NLM_p">5-Chloro-<i>N</i>′-(2,6-dichloro-3-iodophenyl)furan-2-carboximidamide (1.6 g, 3.85 mmol) was dissolved in <i>i</i>-PrOH (10 mL) under argon. NaHCO<sub>3</sub> (0.647 g, 7.70 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (0.671 g, 3.85 mmol) were added, and the reaction mixture was heated to 70 °C. At that temperature, 3-bromo-1,1,1-trifluorobutan-2-one (1.316 mL, 11.55 mmol) was added in one portion, and the resulting suspension was heated to 95 °C for 16 h. After that, an additional batch of 3-bromo-1,1,1-trifluorobutan-2-one (1.096 mL, 9.63 mmol) was added, and the resulting mixture was stirred 48 h at 110 °C. After that, an additional batch of 3-bromo-1,1,1-trifluorobutan-2-one (1.096 mL, 9.63 mmol) was added, and the resulting mixture was stirred for another 5 days at 110 °C. The mixture was cooled to room temperature, partitioned between EtOAc and water, and extracted with EtOAc. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash chromatography eluting with heptane/EtOAc (3/1) to give 1.9 g of a mixture of the starting material and product as yellow oil. This material was then purified by preparative reverse-phase HPLC (method A). The combined fractions were concentrated under reduced pressure, and the remaining aqueous solution was adjusted to pH = 9 with a saturated aqueous NaHCO<sub>3</sub> solution and extracted with EtOAc (2×). The combined organic layers were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole as yellow foam (860 mg, 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 8.29 (d, <i>J</i> = 8.7 Hz, 1H), 7.64 (d, <i>J</i> = 8.7 Hz, 1H), 6.58 (d, <i>J</i> = 3.7 Hz, 1H), 6.35 (d, <i>J</i> = 3.7 Hz, 1H), 2.10 (m, 3H) <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 59.48 <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 143.3, 142.9, 138.8, 137.0, 134.1, 130.8, 130.5, 112.6, 109.7, 8.8 UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.40 min, 520.6 [M + H]<sup>+</sup>.</div><div class="NLM_p">Methyl azetidine-3-carboxylate (371 mg, 2.445 mmol), cesium carbonate (1328 mg, 4.07 mmol), Ruphos (76 mg, 0.163 mmol), and Ruphos palladacyle (133 mg, 0.163 mmol) were mixed in a dry microwave vial. The vial was sealed and purged with argon (3×). Then, a solution of 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole (850 mg, 1.630 mmol) in dry dioxane (16 mL) was added, and the resulting suspension was heated at 95 °C for 18 h. After cooling to room temperature, the reaction mixture was filtered through a pad of Celite rinsing with EtOAc. The filtrate was washed with water and a saturated aqueous NaHCO<sub>3</sub> solution. The organic layer was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give yellow oil. The crude product was purified by preparative reverse-phase HPLC (method A). The combined fractions were adjusted to pH = 9 with a saturated aqueous NaHCO<sub>3</sub> solution and extracted with EtOAc. The combined organic layers were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate as white foam (575 mg. 69% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.63 (d, <i>J</i> = 9.1 Hz, 1H), 6.96 (d, <i>J</i> = 9.1 Hz, 1H), 6.56 (d, <i>J</i> = 3.7 Hz, 1H), 6.13 (d, <i>J</i> = 3.5 Hz, 1H), 4.34–4.28 (dd, <i>J</i> = 8.4, 16.8 Hz, 2H), 4.23–4.11 (m, 2H), 3.68 (s, 3H), 3.66–3.57 (m, 1H), 2.09 (m, 3H) ppm. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.33 min, 508.0 [M + H]<sup>+</sup>.</div><div class="NLM_p">A sample of methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (mixture of 2 atropisomers) (575 mg, 0.523 mmol) was separated by preparative chiral HPLC to give two single, stable atropisomers: Gilson PLC 2020; column, Chiralcel Oz-H 5 μm phenomenex Lux cellulose 2 20 × 250 mm; flow rate, 11 mL/min; mobile phase, heptane/<i>i</i>-PrOH 92:8; detector wavelength, 210, 254 nm; injection, 4.5 mL; sample preparation, 20 mg/2 mL MeOH. The second peak eluting from the column provided (<i>S</i>)-methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (210 mg) as a single, stable atropisomer as a white solid. Chiral purity by chiral HPLC was determined using a Shimadzu AD10vp; column, CHIRALPAK AD-H 5 μm, 4.6 × 250 mm; BPR pressure, 150 bar; flow rate, 1.2 mL/min; mobile phase, Heptane/<i>i</i>-PrOH 95:5; detector wavelength, 210 nm; injection volume, 10 μL; retention time: 12.33 min, chiral purity: >99.0%. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.33 min, 508.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">A solution of LiOH·H<sub>2</sub>O (34.6 mg, 0.826 mmol) in water (1.5 mL) was added to a solution of (<i>S</i>)-methyl 1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylate (210 mg, 0.413 mmol) in THF (3 mL). The reaction mixture was stirred at room temperature for 1 h and then diluted with CH<sub>2</sub>Cl<sub>2</sub>. The solution was adjusted to pH = 3 with HCl (1 M, aqueous). The organic layer was washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The residue was lyophilized to give (<i>S</i>)-1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylic acid (<b>9</b>) as a white powder (198 mg, 97%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.62 (d, <i>J</i> = 9.1 Hz, 1H), 6.95 (d, <i>J</i> = 9.1 Hz, 1H), 6.55 (d, <i>J</i> = 3.7 Hz, 1H), 6.13 (d, <i>J</i> = 3.7 Hz, 1H), 4.32–4.22 (m, 2H), 4.21–4.07 (m, 2H), 3.55–3.45 (m, 1H), 2.09 (m, 3H) ppm. <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm −59.4 <sup>13</sup>C NMR (100 MHz, v): δ ppm 147.28, 142.16, 136.07, 130.00, 128.36, 120.36, 117.70, 116.79, 111.14, 108.67, 65.96, 55.77, 32.57, 7.94. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.16 min, 494.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>19</sub>H<sub>14</sub>Cl<sub>3</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>: 494.0047. Found: 494.0049.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>S</i>)-1-(2,4-Dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxamide (<b>10</b>)</h4><div class="NLM_p last">(<i>S</i>)-1-(2,4-Dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxylic acid (<b>9</b>) (40 mg, 0.081 mmol) and di(1<i>H</i>-imidazole-1-yl)methanone (15.73 mg, 0.097 mmol) were dissolved in THF (0.5 mL) and stirred at room temperature for 1 h. Then, a solution of NH<sub>3</sub> (7 M in MeOH, 0.035 mL, 0.243 mmol) was added, and the reaction mixture was stirred at room temperature for 1 h. The solvent was removed under reduced pressure, and the residue was dissolved in EtOAc and washed with a citric acid solution (1 M, aqueous), NaHCO<sub>3</sub> (10%, aqueous), and brine. The organic layer was dried (MgSO<sub>4</sub>) and concentrated under reduced pressure. The crude product was purified by preparative reverse-phase HPLC (method A). The combined fractions were treated with saturated NaHCO<sub>3</sub> solution and extracted twice with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give the title compound as a white solid (28 mg, 70% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.60 (d, <i>J</i> = 9.1 Hz, 1H), 7.48 (br s, 1H), 7.04(br s, 1H), 6.91 (d, <i>J</i> = 9.1 Hz, 1H), 6.54 (d, <i>J</i> = 3.6 Hz, 1H), 6.11 (d, <i>J</i> = 3.6 Hz, 1H), 4.25–4.18 (m, 2H), 4.13–4.03 (m, 2H), 3.45–3.35 (m, 1H), 2.53–2.46 (m, 3H) ppm. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.09 min, 493.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>19</sub>H<sub>15</sub>Cl<sub>3</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>: 493.0207. Found: 493.0210.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>S</i>)-5-(1-(2,4-Dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidin-3-yl)-1<i>H</i>-tetrazole (<b>11</b>)</h4><div class="NLM_p">A solution of (<i>S</i>)-1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carboxamide (<b>10</b>) (99 mg, 0.2 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (4 mL) was cooled to 0 °C and then treated with pyridine (55 mg, 0.68 mmol) followed by trifluoroacetic anhydride (147 mg, 0.70 mmol). The reaction mixture was stirred at 0 °C for 1 h and then partitioned between CH<sub>2</sub>Cl<sub>2</sub> and a half-saturated NaHCO<sub>3</sub> solution. The aqueous layer was extracted twice with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were dried (MgSO<sub>4</sub>) and concentrated under reduced pressure to give orange oil. The crude product was purified by flash chromatography eluting with cyclohexane/EtOAc, 99/1 to 75/25, to give (<i>S</i>)-1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carbonitrile as pale yellow foam (91 mg, 92%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.61 (d, <i>J</i> = 9.0 Hz, 1H), 6.94 (d, <i>J</i> = 9.1 Hz, 1H), 6.50 (d, <i>J</i> = 3.6 Hz, 1H), 6.11 (d, <i>J</i> = 3.6 Hz, 1H), 4.39–4.15 (m, 4H), 3.79 (tt, <i>J</i> = 8.7, 5.8 Hz, 1H), 2.04 (d, <i>J</i> = 1.2 Hz, 3 H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.26 min, 474.8 [M + H]<sup>+</sup>.</div><div class="NLM_p last">A solution of (<i>S</i>)-1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidine-3-carbonitrile (40 mg, 0.084 mmol) in toluene (1 mL) was treated with TMS-azide (47 mg, 0.22 mmol). The reaction mixture was stirred at 100 °C for 24 h. Another batch of TMS-azide (18 mg, 0.084 mmol) was added, and the reaction mixture was stirred at 100 °C for 16 h. The reaction mixture was concentrated under reduced pressure and then partitioned between CH<sub>2</sub>Cl<sub>2</sub> and half-saturated NaHCO<sub>3</sub> solution. The organic layer was dried (MgSO<sub>4</sub>) and concentrated under reduced pressure to give a white solid. The crude product was purified by preparative reverse-phase HPLC (method A). The combined product fractions were treated with saturated NaHCO<sub>3</sub> solution and extracted twice with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give (<i>S</i>)-5-(1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidin-3-yl)-1<i>H</i>-tetrazole (<b>11</b>) as a white solid (35 mg, 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.66 (d, <i>J</i> = 9.0 Hz, 1H), 7.01 (d, <i>J</i> = 9.0 Hz, 1H), 6.55 (d, <i>J</i> = 3.6 Hz, 1H), 6.15 (d, <i>J</i> = 3.6 Hz, 1H), 4.59 (hept, <i>J</i> = 4.8, 4.4 Hz, 2H), 4.37–4.25 (m, 3H), 2.10 (d, <i>J</i> = 1.71 Hz, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.11 min, 518.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>19</sub>H<sub>13</sub>Cl<sub>3</sub>F<sub>3</sub>N<sub>7</sub>O<sup>+</sup> [M + H]<sup>+</sup>: 518.0272. Found: 518.0281.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>S</i>)-2-(1-(2,4-Dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidin-3-yl)acetic Acid (<b>12</b>)</h4><div class="NLM_p">A sample of 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole (mixture of two atropisomers) (5.35 g, 10.25 mmol) was separated by preparative chiral HPLC to give two single, stable atropisomers: column, CHIRALPAK AS 20 μm 50 × 5 cm; flow rate, 60 mL/min; mobile phase, heptane/<i>i</i>-PrOH 92:8; detector wavelength, 275 nm; injection, 4.5 mL; sample preparation, 5.35 g/25 mL <i>i</i>-PrOH (6 separate injections). The first peak eluting from the column provided (<i>S</i>)-2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole (2.25 g) as a single, stable atropisomer as a yellow solid. Chiral purity by chiral HPLC was determined using column, CHIRALPAK AS-H 5 pm 4.6 × 250 m; BPR pressure, 150 bar; flow rate, 1.0 mL/min; mobile phase, heptane/<i>i</i>-PrOH 92:8; detector wavelength, 275 nm; injection volume, 20 μL; retention time: 4.95 min, chiral purity: >99.0%. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.40 min, 520.6 [M + H]<sup>+</sup>.</div><div class="NLM_p">In a glass vial, Cs<sub>2</sub>CO<sub>3</sub> (1.855 g, 5.58 mmol), Ruphos (0.196 g, 0.400 mmol), and Ruphos palladacyle (0.155 g, 0.186 mmol) were added to a suspension of (<i>S</i>)-2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole (1 g, 1.860 mmol) and methyl 2-(azetidin-3-yl)acetate trifluoroacetate (0.678 g, 2.79 mmol) in toluene (15 mL). The vial was capped and purged with argon (3×). The mixture was stirred at 90 °C for 18 h. The vial was opened, and the sticky precipitate was scraped off the glass wall and sonicated for 1 min. The vial was then recapped and purged with argon (3×) and heated at 90 °C for 72 h. After cooling to room temperature, the reaction mixture was partitioned between EtOAc (300 mL) and a saturated aqueous NaHCO<sub>3</sub> solution (100 mL). The organic layer was washed with brine (2 × 50 mL), dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash chromatography eluting with cyclohexane/EtOAc, 75/25, to give (<i>S</i>)-methyl 2-(1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidin-3-yl)acetate as a yellow solid (650 mg, 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.48 (d, <i>J</i> = 9.05 Hz, 1H), 6.84 (d, <i>J</i> = 9.05 Hz, 1H), 6.31 (d, <i>J</i> = 3.67 Hz, 1H), 6.25 (d, <i>J</i> = 3.42 Hz, 1H), 4.31 (d, <i>J</i> = 8.80 Hz, 2H), 3.83 (s, 2H), 3.68 (s, 3H), 2.95–3.08 (m, 1H), 2.75 (d, <i>J</i> = 7.58 Hz, 2H), 2.08–2.15 (m, 3H) UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.37 min, 523.8 [M + H]<sup>+</sup>.</div><div class="NLM_p last">A solution of LiOH·H<sub>2</sub>O (7.4 mg, 0.18 mmol) in water (0.3 mL) was added to a solution of (<i>S</i>)-methyl 2-(1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidin-3-yl)acetate (48 mg, 0.09 mmol) in THF (0.6 mL). The reaction mixture was stirred at room temperature for 3 h and then diluted with CH<sub>2</sub>Cl<sub>2</sub> (8 mL) and water (1 mL). The aqueous layer was adjusted to pH = 3 with HCl (1 M, aqueous) and extracted with EtOAc (2 × 10 mL). The combined organic layers were dried (MgSO<sub>4</sub>) and concentrated under reduced pressure. The residue was lyophilized to give (<i>S</i>)-2-(1-(2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-5-methyl-4-(trifluoromethyl)-1<i>H</i>-imidazole-1-yl)phenyl)azetidin-3-yl)acetic acid (<b>12</b>) as a yellow powder (43 mg, 92%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 12.61–11.98 (br s, 1H), 7.59 (d, <i>J</i> = 9.05 Hz, 1H), 6.88 (d, <i>J</i> = 9.17 Hz, 1H), 6.55 (d, <i>J</i> = 3.67 Hz, 1H), 6.10 (d, <i>J</i> = 3.55 Hz, 1H), 4.32–4.20 (m, 2H), 3.85–3.71 (m, 2H), 2.98–2.84 (m, 1H), 2.64 (d, <i>J</i> = 7.82 Hz, 2H), 2.09 (s, 3H) UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.19 min, 508.3 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>20</sub>H<sub>16</sub>Cl<sub>3</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>: 508.0204. Found: 508.0209.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>S</i>)-3-((2,4-Dichloro-3-(5-methyl-2-(5-methylfuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)amino)-2,2-dimethylpropanoic Acid (<b>13</b>)</h4><div class="NLM_p">5-Chloro-<i>N</i>′-(2,6-dichloro-3-iodophenyl)furan-2-carboximidamide (3.0 g, 7.22 mmol) and NaHCO<sub>3</sub> (1.82 g, 21.66 mmol) were added to a solution of 2-bromo-1-[1-(trifluoromethyl)cyclopropyl]ethanone (6.10 g, 21.66 mmol) in <i>i</i>-PrOH (60 mL). The resulting mixture was stirred at 90 °C for 48 h. Another batch of 2-bromo-1-[1-(trifluoromethyl)cyclopropyl]ethanone (6.10 g, 21.66 mmol) in <i>i</i>-PrOH (5 mL) and NaHCO<sub>3</sub> (1.820 g, 21.66 mmol) was added, and stirring continued for 4 days at 90 °C. After cooling to room temperature, the mixture was filtered, and the yellow filtrate was concentrated under reduced pressure. The residue was dissolved in AcOH (10 mL) and then stirred at 120 °C for 1 h. The reaction mixture was cooled to 5–10 °C and treated with an aqueous saturated NaHCO<sub>3</sub> until gas evolution had stopped. The mixture was then extracted with EtOAc (2 × 300 mL), and the combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash chromatography eluting with cyclohexane/CH<sub>2</sub>Cl<sub>2</sub>, 1/1 to give 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole as a yellow resin (3.1 g, 76%) 1H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 8.20 (d, <i>J</i> = 8.6 Hz, 1H), 7.59–7.50 (m, 2H), 6.54 (d, <i>J</i> = 3.6 Hz, 1H), 6.23 (d, <i>J</i> = 3.6 Hz, 1H), 2.01 (s, 0H), 1.41–1.27 (m, 4H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.42 min, 546.6 [M + H]<sup>+</sup>.</div><div class="NLM_p">A sample of 2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (3.10 g, 5.66 mmol) was separated by preparative chiral HPLC to give two single, stable atropisomers: column, CHIRALPAK AS 20 μm 50 × 5 cm; flow rate, 70 mL/min; mobile phase, heptane/<i>i</i>-PrOH 96:4; detector wavelength, 254 nm; injection, 6 mL; sample preparation, 3.01 g/12 mL <i>n</i>-heptane/<i>i</i>-PrOH (96/4) (2 separate injections). The first peak eluting from the column provided (<i>S</i>)-2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (1.46 g) as a single, stable atropisomer as a yellow solid. Chiral purity by chiral HPLC was determined using column, CHIRALPAK AS-H 5 pm, 250 × 4.6 mm; flow rate, 1.0 mL/min; mobile phase, heptane/<i>i</i>-PrOH 96:4; detector wavelength, 254 nm; injection volume, 10 μL; retention time: 4.40 min, chiral purity: >99.0%. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.42 min, 546.6 [M + H]<sup>+</sup>.</div><div class="NLM_p">In a microwave vial, (<i>S</i>)-2-(5-chlorofuran-2-yl)-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (500 mg, 0.91 mmol), ethyl 3-amino-2,2-dimethylpropanoate (249 mg, 1.37 mmol), Cs<sub>2</sub>CO<sub>3</sub> (890 mg, 2.74 mmol), BrettPhos (49 mg, 0.091 mmol), RuPhos (42.6 mg, 0.091 mmol), and RuPhos precatalyst (74.6 mg, 0.091 mmol) were dissolved in toluene (10 mL). The mixture was purged with argon (3×), and the vial was sealed and stirred overnight at 110 °C. The reaction mixture was cooled to room temperature and filtered. The residue was washed with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (95:5). The filtrate was concentrated under reduced pressure, and the crude product was purified by flash chromatography eluting with cyclohexane/EtOAc, 100/0 to 60/40, to give (<i>S</i>)-ethyl 3-((2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)amino)-2,2-dimethylpropanoate (336 mg, 63%) as a yellow resin. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.50–7.41 (m, 1H), 7.11 (d, <i>J</i> = 9.2 Hz, 1H), 6.50 (d, <i>J</i> = 3.5 Hz, 1H), 5.99 (d, <i>J</i> = 3.5 Hz, 1H), 5.50 (t, <i>J</i> = 6.5 Hz, 1H), 4.14–3.98 (m, 3H), 3.41 (d, <i>J</i> = 6.4 Hz, 2H), 3.17 (dd, <i>J</i> = 5.3, 1.0 Hz, 2H), 1.43–1.29 (m, 4H), 1.23–1.10 (m, 9H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.47 min, 564.4 [M + H]<sup>+</sup>.</div><div class="NLM_p">NBS (126 mg, 0.710 mmol) was added to a solution of (<i>S</i>)-ethyl 3-((2,4-dichloro-3-(2-(5-chlorofuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)amino)-2,2-dimethylpropanoate (334 mg, 0.591 mmol) in acetonitrile (5 mL). The resulting solution was stirred at room temperature for 1 h. The reaction mixture was partitioned between saturated aqueous NaHCO<sub>3</sub> solution and EtOAc. The aqueous layer was extracted with EtOAc. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash chromatography eluting with cyclohexane/EtOAc, 100/0 to 55/45, to give (<i>S</i>)-ethyl 3-((3-(5-bromo-2-(5-chlorofuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)-2,4-dichlorophenyl)amino)-2,2-dimethylpropanoate as a yellow resin (318 mg, 84%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.53 (d, <i>J</i> = 9.1 Hz, 1H), 7.18 (d, <i>J</i> = 9.2 Hz, 1H), 6.54 (d, <i>J</i> = 3.6 Hz, 1H), 6.09 (d, <i>J</i> = 3.7 Hz, 1H), 5.58 (t, <i>J</i> = 6.5 Hz, 1H), 4.03 (q, <i>J</i> = 7.1 Hz, 2H), 3.42 (dd, <i>J</i> = 6.5, 2.3 Hz, 2H), 1.48–1.38 (m, 4H), 1.19 (s, 6H), 1.14 (t, <i>J</i> = 7.1 Hz, 3H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.51 min, 641.8 [M + H]<sup>+</sup>.</div><div class="NLM_p">K<sub>2</sub>CO<sub>3</sub> (376 mg, 2.72 mmol) and tetrakis(triphenylphosphine)palladium (105 mg, 0.09 mmol) were added to a solution of (<i>S</i>)-ethyl-3-((3-(5-bromo-2-(5-chlorofuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)-2,4-dichlorophenyl)amino)-2,2-dimethylpropanoate (569 mg, 0.907 mmol) and trimethylboroxine (124 mg, 1.813 mmol) in DMF (6 mL) under argon. The resulting mixture was purged with argon (3×) and then placed in a 90 °C heating block and stirred overnight. The reaction mixture was cooled to room temperature and then partitioned between EtOAc and water. The aqueous layer was extracted twice with EtOAc. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The crude product was purified by flash chromatography using cyclohexane/EtOAc, gradient 0–55% EtOAc, followed by preparative reverse-phase HPLC (method A). The combined fractions were treated with saturated NaHCO<sub>3</sub> solution and extracted twice with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were washed with brine, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure to give (<i>S</i>)-ethyl 3-((2,4-dichloro-3-(5-methyl-2-(5-methylfuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)amino)-2,2-dimethylpropanoate as a colorless resin (50 mg, 18%) <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ ppm 7.43 (d, <i>J</i> = 9.1 Hz, 1H), 7.09 (d, <i>J</i> = 9.1 Hz, 1H), 5.98 (dq, <i>J</i> = 3.2, 1.0 Hz, 1H), 5.93 (d, <i>J</i> = 3.4 Hz, 1H), 4.13 (q, <i>J</i> = 7.1 Hz, 2H), 3.50–3.40 (m, 2H), 2.21 (d, <i>J</i> = 1.0 Hz, 3H), 2.05 (s, 3H), 1.47–1.39 (m, 2H), 1.29 (s, 6H), 1.24 (t, <i>J</i> = 7.1 Hz, 3H), 1.22–1.10 (m, 2H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.43 min, 558.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">LiOH·H<sub>2</sub>O (7.5 mg, 0.180 mmol) was added to a solution of (<i>S</i>)-ethyl 3-((2,4-dichloro-3-(5-methyl-2-(5-methylfuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)amino)-2,2-dimethylpropanoate (50 mg, 0.09 mmol) in THF (1 mL) and water (0.5 mL). The resulting solution was stirred at 60 °C for 24 h. The reaction mixture was then acidified with HCl (1 M, aqueous, 0.180 mL) and extracted with EtOAc. The combined organic layers were dried (MgSO<sub>4</sub>) and concentrated under reduced pressure to give (<i>S</i>)-3-((2,4-dichloro-3-(5-methyl-2-(5-methylfuran-2-yl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)amino)-2,2-dimethylpropanoic acid (<b>13</b>) as an off-white solid (42 mg, 85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 12.58 (s, 1H), 7.49 (d, <i>J</i> = 9.0 Hz, 1H), 7.15 (d, <i>J</i> = 9.2 Hz, 1H), 6.03 (d, <i>J</i> = 3.3 Hz, 1H), 5.64 (d, <i>J</i> = 3.4 Hz, 1H), 5.47 (s, 1H), 2.20 (s, 3H), 1.95 (s, 3H), 1.36 (q, <i>J</i> = 4.9 Hz, 2H), 1.22–1.06 (m, 8H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.22 min, 530.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>S</i>)-2-(2,4-Dichloro-3-(2-cyclopropyl-5-methyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylic Acid (<b>14</b>)</h4><div class="NLM_p">2,6-Dichloro-3-iodoaniline (<b>15</b>) (15 g, 52.1 mmol) was dissolved in toluene (90 mL) and cooled to 0 °C under argon. Trimethylaluminum in toluene (31.3 mL, 62.5 mmol) was added, and the reaction mixture was then warmed to room temperature. After 20 min, cyclopropanecarbonitrile (5.76 mL, 78 mmol) was added, and the reaction mixture was stirred at room temperature overnight. The mixture was quenched dropwise (caution, violent reaction) with an aqueous saturated NaHCO<sub>3</sub> solution. The resulting white solid was filtered off, and the filtrate was diluted with EtOAc. The solution was washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated under reduced pressure to give light yellow oil. The crude product was purified by flash chromatography eluting with heptane/EtOAc (3/1) to give <i>N</i>-(2,6-dichloro-3-iodophenyl)cyclopropanecarboximidamide (<b>16</b>) as a beige powder/solid (7.43 g, 39%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.41 (d, <i>J</i> = 8.4 Hz, 1H), 7.08 (d, <i>J</i> = 8.4 Hz, 1H), 0.95–0.87 (m, 2H), 0.77–0.68 (m, 2H) ppm. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 0.54 min, 354.7 [M + H]<sup>+</sup>.</div><div class="NLM_p">2-Bromo-1-[1-(trifluoromethyl)cyclopropyl]ethanone (8.13 g, 25.40 mmol) and NaHCO<sub>3</sub> (2.37 g, 28.2 mmol) were added to <i>N</i>-(2,6-dichloro-3-iodophenyl)cyclopropanecarboximidamide (<b>16</b>) (5.0 g, 14.08 mmol) in <i>i</i>-PrOH (50 mL). The mixture was heated to 90 °C under argon for 42 h. The mixture was cooled to room temperature, filtered, and washed with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (95/5). The filtrate was concentrated under reduced pressure to give brown oil. The crude product was purified by flash chromatography eluting with cyclohexane/EtOAc, 99/1 to 78/22 then 70/30 to give 2-cyclopropyl-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (±<b>17</b>) as orange oil (6.34 g, 97%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 8.13 (d, <i>J</i> = 8.56 Hz, 1H), 7.51 (d, <i>J</i> = 8.68 Hz, 1H), 7.16 (s, 1H), 1.37–1.29 (m, 1H), 1.27–1.16 (m, 4H), 0.83–0.72 (m, 4H) UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.37 min, 487.0 [M + H]<sup>+</sup>.</div><div class="NLM_p">A sample of 2-cyclopropyl-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (±<b>17</b>) (22.0 g, 45.2 mmol) was separated by preparative chiral HPLC to give two single, stable atropisomers: column, Chiralcel OJ 20 μm 10 × 50 cm; flow rate, 120 mL/min; mobile phase, heptane/<i>i</i>-PrOH 97:3; detector wavelength, 275 nm; injection, 3.6 mL; sample preparation, 22 g/50 mL in <i>n</i>-heptane/<i>i</i>-PrOH (80/20) (14 separate injections). The first peak eluting from the column provided (<i>S</i>)-2-cyclopropyl-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (<b>17a</b>) (9.92 g) as a single, stable atropisomer as a yellow solid. Chiral purity by chiral HPLC was determined using column, CHIRALPAK AS-H 5 μm 4.6 × 250 mm flow rate, 1.0 mL/min; mobile phase, heptane/<i>i</i>-PrOH 92:8; detector wavelength, 275 nm; injection volume, 5.0 μL; retention time: 4.95 min, chiral purity: >99.0%. UPLC/MS (ESI) <i>m</i>/<i>z</i> method 4: 1.40 min, 520.6 [M + H]<sup>+</sup>. The second peak eluting from the column provided (<i>R</i>)-2-cyclopropyl-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (<b>17b</b>) (9.80 g). (<i>R</i>)-2-cyclopropyl-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole could be racemized by heating a 0.16 M solution in <i>N</i>,<i>N</i>-dimethylacetamide in the microwave with stirring at 250 °C for 30 min. The solution was concentrated under reduced pressure and repurified using the chiral HPLC method stated above.</div><div class="NLM_p">Thionyl chloride (8.58 mL, 118.0 mmol) was added dropwise to a solution of 2-azaspiro[3.3]heptane-6-carboxylic acid trifluoroacetate (10 g, 39.2 mmol) in MeOH (143 mL) at −5 °C, maintaining the internal temperature below 0 °C. The solution was warmed to room temperature and stirred for 20 min and then heated at 60 °C for 3 h. The mixture was concentrated under reduced pressure to give methyl 2-azaspiro[3.3]heptane-6-carboxylate as an off-white solid (7.5 g, 100%) that was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 9.16 (br s, 2H), 3.92 (t, <i>J</i> = 6.24 Hz, 2H), 3.84 (t, <i>J</i> = 6.24 Hz, 2H), 3.58 (s, 3H), 2.91–3.05 (m, 1H), 2.39–2.47 (m, 2H), 2.26–2.35 (m, 2H) flow injection mass spectrometry method 1. Found: 155.9 [M + H]<sup>+</sup>.</div><div class="NLM_p">Cs<sub>2</sub>CO<sub>3</sub> (38.5 g, 118 mmol), Ruphos (1.53 g, 3.28 mmol), and Ruphos palladacyle (2.68 g, 3.28 mmol) were added to (<i>S</i>)-2-cyclopropyl-1-(2,6-dichloro-3-iodophenyl)-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole (<b>17a</b>) (16.0 g, 32.8 mmol) and methyl 2-azaspiro[3.3]heptane-6-carboxylate (6.12 g, 39.4 mmol) in toluene (250 mL). The resulting mixture was heated under nitrogen at 100 °C for 3 h. After cooling to room temperature, the reaction mixture was filtered and concentrated under reduced pressure. The crude product was purified by flash chromatography eluting with <i>n</i>-heptane/EtOAc, 100/0 to 75/25, to give (<i>S</i>)-methyl 2-(2,4-dichloro-3-(2-cyclopropyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylate (<b>18</b>) as brown oil (13.6 g, 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.47 (d, <i>J</i> = 9.0 Hz, 1H), 7.02 (s, 1H), 6.73 (d, <i>J</i> = 9.0 Hz, 1H), 4.11 (q, <i>J</i> = 8.0 Hz, 2H), 4.00 (q, <i>J</i> = 8.3 Hz, 2H), 3.61 (s, 3H), 3.13–3.04 (m, 1H), 2.52–2.31 (m, 4H), 1.36–1.13 (m, 5H), 0.86–0.68 (m, 4H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.40 min, 514.2 [M + H]<sup>+</sup>.</div><div class="NLM_p"><i>N</i>-Bromosuccinimide (1.37 g, 7.7 mmol) was added to (<i>S</i>)-methyl 2-(2,4-dichloro-3-(2-cyclopropyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylate (<b>18</b>) (3.6 g, 7.0 mmol) in acetonitrile (100 mL) under nitrogen, and the resulting mixture was stirred at room temperature for 1 h. A saturated aqueous NaHCO<sub>3</sub> solution (100 mL) was added, and the solution was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 200 mL). The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated under reduced pressure. The crude product was purified by flash chromatography over silica gel eluting with <i>n</i>-heptane/EtOAc, 100/0 to 75/25, to give (<i>S</i>)-methyl 2-(3-(5-bromo-2-cyclopropyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)-2,4-dichlorophenyl)-2-azaspiro[3.3]heptane-6-carboxylate (<b>19</b>) as yellow oil (3.2 g, 77%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.53 (d, <i>J</i> = 9.0 Hz, 1H), 6.78 (d, <i>J</i> = 9.1 Hz, 1H), 4.14 (q, <i>J</i> = 8.0 Hz, 2H), 4.02 (q, <i>J</i> = 8.3 Hz, 2H), 3.62 (s, 3H), 3.13–3.04 (m, 1H), 2.51–2.40 (m, 2H), 2.40–2.31 (m, 2H), 1.42–1.28 (m, 3H), 1.08–1.01 (m, 2H), 0.83–0.69 (m, 4H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.42 min, 592.1 [M + H]<sup>+</sup>.</div><div class="NLM_p">Argon was bubbled through a solution of (<i>S</i>)-methyl 2-(3-(5-bromo-2-cyclopropyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)-2,4-dichlorophenyl)-2-azaspiro[3.3]heptane-6-carboxylate (<b>19</b>) (10.0 g, 16.86 mmol) and 2,4,6-trimethyl-1,3,5,2,4,6-trioxatriborinane (4.71 mL, 33.7 mmol) in DMF (200 mL) for 15 min. Then, tetrakis(triphenylphosphine)palladium(0) (1.948 g, 1.686 mmol) and potassium carbonate (6.99 g, 50.6 mmol) were added, and the mixture was heated in a preheated oil bath at 100 °C for 18 h. The reaction mixture was cooled to room temperature, filtered through Celite, and washed with TBME. The filtrate was extracted with ice water (800 mL) and TBME (800 mL). The combined organic layers were washed with brine/water (2/3, 800 mL) and brine (1 × 800 mL). The aqueous layer was extracted with TBME (2 × 500 mL). The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated under reduced pressure to give a brown resin. The crude product was purified by flash chromatography eluting with <i>n</i>-heptane/EtOAc, 90/10 to 40/60, to give (<i>S</i>)-methyl 2-(2,4-dichloro-3-(2-cyclopropyl-5-methyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylate as yellow oil (7.65 g, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.48 (d, <i>J</i> = 8.9 Hz, 1H), 6.72 (d, <i>J</i> = 9.0 Hz, 1H), 4.12–4.08 (m, 2H), 4.02–3.98 (m, 2H), 3.59 (s, 3H), 3.13–3.00 (m, 1H), 2.50–2.31 (m, 4H), 1.85 (s, 3H), 1.32–1.21 (m, 3H), 1.02–0.95 (m, 2H), 0.79–0.62 (m, 4H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.31 min, 528.2 [M + H]<sup>+</sup>.</div><div class="NLM_p last">NaOH (1.0 M, aqueous, 7.42 mL, 7.42 mmol) was added dropwise to (<i>S</i>)-methyl 2-(2,4-dichloro-3-(2-cyclopropyl-5-methyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylate (2.0 g, 3.71 mmol) in MeOH (55 mL), and the mixture was stirred at room temperature for 18 h. The reaction mixture was diluted with water (100 mL) and washed with TBME (4 × 100 mL). The combined organic layers were extracted with brine/water (1/2, 2 × 50 mL) and brine (50 mL). TBME (250 mL) was added to the combined aqueous layers adjusted to pH = 3 by the addition of 10% aq citric acid (25 mL). The biphasic mixture was stirred for 10 min, and the organic phase was separated. The aqueous phase was extracted with TBME (150 mL). The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated under reduced pressure to give the crude product as a white solid (1.66 g). CH<sub>2</sub>Cl<sub>2</sub> (150 mL) was added, and the mixture was stirred and filtered. To the filtrate was added <i>n</i>-heptane (50 mL) to give a pale yellow solution. CH<sub>2</sub>Cl<sub>2</sub> was removed under reduced pressure until precipitation started, and the resulting suspension was stirred at room temperature for 1 h. The mixture was filtered, and the residual solid was washed with <i>n</i>-heptane to give (<i>S</i>)-2-(2,4-dichloro-3-(2-cyclopropyl-5-methyl-4-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-imidazole-1-yl)phenyl)-2-azaspiro[3.3]heptane-6-carboxylic acid (<b>14</b>) as a white solid (1.48 g, 76%) <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ ppm 7.48 (d, <i>J</i> = 8.9 Hz, 1H), 6.72 (d, <i>J</i> = 9.0 Hz, 1H), 4.12–4.08 (m, 2H), 4.02–3.98 (m, 2H), 3.00–2.89 (m, 1H), 2.47–2.29 (m, 4H), 1.85 (s, 3H), 1.32–1.19 (m, 3H), 1.02–0.95 (m, 2H), 0.79–0.62 (m, 4H). UPLC/MS (ESI) <i>m</i>/<i>z</i> method 1: 1.09 min, 514.5 [M + H]<sup>+</sup>; HRMS (ESI) calcd for C<sub>24</sub>H<sub>25</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>: 514.1270. Found: 514.1270.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Biology and Pharmacology</h3><div class="NLM_p last">All in vitro and in vivo assay methods have already been described in detail in previous publications.<a onclick="showRef(event, 'ref9 ref11 ref27'); return false;" href="javascript:void(0);" class="ref ref9 ref11 ref27">(9,11,27)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01291" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01291?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01291</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Kinase and safety panel assays for compound <b>14</b>; structural biology specifics (expression and purification, cocrystallization, and X-ray diffraction data collection and processing); X-ray crystal data and structure refinement for compound <b>14</b>; design based on the WATMD model; and cartoon illustrating nuclear receptor LBD conformations (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings including assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf">jm9b01291_si_001.pdf (1.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_002.csv">jm9b01291_si_002.csv (1.18 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors for cocrystal structures have been deposited in the Protein Data Bank with the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6M">6Q6M</a> (compound <b>1</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O">6Q6O</a> (compound <b>2</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7A">6Q7A</a> (compound <b>4</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H">6Q7H</a> (compound <b>9</b>). Authors will release the atomic coordinates and experimental data upon article publication. Crystallographic data of compound <b>14</b> (excluding structure factors) have been deposited with the Cambridge Crystallographic Data Centre as supplementary publication number CCDC 1944604. Copies of the data can be obtained free of charge on application to CCDC, 12 Union Road, Cambridge CB2 1EZ, UK [fax (+44) 1223 336033, email: <a href="/cdn-cgi/l/email-protection#630706130c100a1723000007004d00020e4d02004d1608"><span class="__cf_email__" data-cfemail="95f1f0e5fae6fce1d5f6f6f1f6bbf6f4f8bbf4f6bbe0fe">[email protected]</span></a>]. This material is available free of charge via the internet at http://pubs.acs.org.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01291" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81432" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81432" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Klemens Hoegenauer</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7266-0137" title="Orcid link">http://orcid.org/0000-0001-7266-0137</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e48f888189818a97ca8c8b8183818a85918196a48a8b928596908d97ca878b89"><span class="__cf_email__" data-cfemail="513a3d343c343f227f393e3436343f30243423113f3e2730232538227f323e3c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joerg Kallen</span> - <span class="hlFld-Affiliation affiliation">Chemical Biology & Therapeutics, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eloísa Jiménez-Núñez</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ross Strang</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Ertl</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6496-4448" title="Orcid link">http://orcid.org/0000-0001-6496-4448</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nigel G. Cooke</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samuel Hintermann</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6853-6803" title="Orcid link">http://orcid.org/0000-0001-6853-6803</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Voegtle</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claudia Betschart</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel J. J. McKay</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, 181 Massachusetts Avenue, 02139 Cambridge, Massachusetts, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juergen Wagner</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johannes Ottl</span> - <span class="hlFld-Affiliation affiliation">Chemical Biology & Therapeutics, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Beerli</span> - <span class="hlFld-Affiliation affiliation">Autoimmunity, Transplantation
and Inflammation, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Billich</span> - <span class="hlFld-Affiliation affiliation">Autoimmunity, Transplantation
and Inflammation, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janet Dawson</span> - <span class="hlFld-Affiliation affiliation">Autoimmunity, Transplantation
and Inflammation, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Klemens Kaupmann</span> - <span class="hlFld-Affiliation affiliation">Autoimmunity, Transplantation
and Inflammation, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Streiff</span> - <span class="hlFld-Affiliation affiliation">Autoimmunity, Transplantation
and Inflammation, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nathalie Gobeau</span> - <span class="hlFld-Affiliation affiliation">PK Sciences, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span>; 
    <span>Present Address:
                        Medicines for Malaria Venture, 20 Route de Pré-Bois, 1215 Geneva 15, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephanie Harlfinger</span> - <span class="hlFld-Affiliation affiliation">PK Sciences, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span>; 
    <span>Present Address:
                        Astra Zeneca, Hodgkin Building, Chesterford Research Park, Little Chesterford, Saffron Walden, CB10 1XL</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rowan Stringer</span> - <span class="hlFld-Affiliation affiliation">PK Sciences, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Guntermann</span> - <span class="hlFld-Affiliation affiliation">Autoimmunity, Transplantation
and Inflammation, , , and , Novartis Institutes
for BioMedical Research, 4002 Basel, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24845" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24845" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank A. von Matt, T. Ullrich, and P. Lehr for their synthetic and medicinal chemistry contributions; N. Graveleau, I. Jeulin, J. Sauer, P. Scheifele, and B. Thai for their contributions in compound synthesis; J. Kuehnoel for analytical support; E. Arduin, R. Brunner, F. Ecoeur, Y. Henriquez, and J. Weiss for their efforts toward developing and performing biochemical and cellular assays; B. Jost for in vivo pharmacology; S. Kapps, J. Klopp, E. Koch, C. Be, and A. Izaac for protein expression, purification, and crystallization; and I. Dix and T. Wagner for crystallography support for compound <b>14</b>. We are grateful to the MX beamline team at the Swiss Light Source (PSI Villigen, Switzerland) for outstanding support at the beamline and Expose GmbH (Switzerland) for diffraction data collection.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CDI</td><td class="NLM_def"><p class="first last">1,1′-carbonyldiimidazole</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">ConA</td><td class="NLM_def"><p class="first last">concanavalin A</p></td></tr><tr><td class="NLM_term">DTH</td><td class="NLM_def"><p class="first last">delayed-type hypersensitivity</p></td></tr><tr><td class="NLM_term">LBD</td><td class="NLM_def"><p class="first last">ligand-binding domain</p></td></tr><tr><td class="NLM_term">LipE</td><td class="NLM_def"><p class="first last">lipophilic efficiency</p></td></tr><tr><td class="NLM_term">NIS</td><td class="NLM_def"><p class="first last"><i>N</i>-iodosuccinimide</p></td></tr><tr><td class="NLM_term">RLM</td><td class="NLM_def"><p class="first last">rat liver microsomes</p></td></tr><tr><td class="NLM_term">TDI</td><td class="NLM_def"><p class="first last">time-dependent CYP3A4 inhibition</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02051" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02051" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadokoro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepelley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafaille, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cua, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">The Orphan Nuclear Receptor RORγt Directs the Differentiation Program of Proinflammatory IL-17+ T Helper Cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.07.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.cell.2006.07.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=16990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=1121-1133&author=I.+I.+Ivanovauthor=B.+S.+McKenzieauthor=L.+Zhouauthor=C.+E.+Tadokoroauthor=A.+Lepelleyauthor=J.+J.+Lafailleauthor=D.+J.+Cuaauthor=D.+R.+Littman&title=The+Orphan+Nuclear+Receptor+ROR%CE%B3t+Directs+the+Differentiation+Program+of+Proinflammatory+IL-17%2B+T+Helper+Cells&doi=10.1016%2Fj.cell.2006.07.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17+ T helper cells</span></div><div class="casAuthors">Ivanov, Ivaylo I.; McKenzie, Brent S.; Zhou, Liang; Tadokoro, Carlos E.; Lepelley, Alice; Lafaille, Juan J.; Cua, Daniel J.; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1133</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IL-17-producing T lymphocytes have been recently shown to comprise a distinct lineage of proinflammatory T helper cells, termed Th17 cells, that are major contributors to autoimmune disease.  The authors show here that the orphan nuclear receptor RORγt is the key transcription factor that orchestrates the differentiation of this effector cell lineage.  RORγt induces transcription of the genes encoding IL-17 and the related cytokine IL-17F in naive CD4+ T helper cells and is required for their expression in response to IL-6 and TGF-β, the cytokines known to induce IL-17.  Th17 cells are constitutively present throughout the intestinal lamina propria, express RORγt, and are absent in mice deficient for RORγt or IL-6.  Mice with RORγt-deficient T cells have attenuated autoimmune disease and lack tissue-infiltrating Th17 cells.  Together, these studies suggest that RORγt is a key regulator of immune homeostasis and highlight its potential as a therapeutic target in inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsJd6RyFqWrVg90H21EOLACvtfcHk0liuoDLxj0zPdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE&md5=4f5693603c30dca4f6a2d0994aeaf9b0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.07.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.07.035%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DI.%2BI.%26aulast%3DMcKenzie%26aufirst%3DB.%2BS.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DTadokoro%26aufirst%3DC.%2BE.%26aulast%3DLepelley%26aufirst%3DA.%26aulast%3DLafaille%26aufirst%3DJ.%2BJ.%26aulast%3DCua%26aufirst%3DD.%2BJ.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520Orphan%2520Nuclear%2520Receptor%2520ROR%25CE%25B3t%2520Directs%2520the%2520Differentiation%2520Program%2520of%2520Proinflammatory%2520IL-17%252B%2520T%2520Helper%2520Cells%26jtitle%3DCell%26date%3D2006%26volume%3D126%26spage%3D1121%26epage%3D1133%26doi%3D10.1016%2Fj.cell.2006.07.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappu, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurieva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akimzhanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panopoulos, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schluns, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetten, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span> <span> </span><span class="NLM_article-title">T Helper 17 Lineage Differentiation Is Programmed by Orphan Nuclear Receptors RORα and RORγ</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2007.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.immuni.2007.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=18164222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=29-39&author=X.+O.+Yangauthor=B.+P.+Pappuauthor=R.+Nurievaauthor=A.+Akimzhanovauthor=H.+S.+Kangauthor=Y.+Chungauthor=L.+Maauthor=B.+Shahauthor=A.+D.+Panopoulosauthor=K.+S.+Schlunsauthor=S.+S.+Watowichauthor=Q.+Tianauthor=A.+M.+Jettenauthor=C.+Dong&title=T+Helper+17+Lineage+Differentiation+Is+Programmed+by+Orphan+Nuclear+Receptors+ROR%CE%B1+and+ROR%CE%B3&doi=10.1016%2Fj.immuni.2007.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ</span></div><div class="casAuthors">Yang, Xuexian O.; Pappu, Bhanu P.; Nurieva, Roza; Akimzhanov, Askar; Kang, Hong Soon; Chung, Yeonseok; Ma, Li; Shah, Bhavin; Panopoulos, Athanasia D.; Schluns, Kimberly S.; Watowich, Stephanie S.; Tian, Qiang; Jetten, Anton M.; Dong, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">T cell functional differentiation is mediated by lineage-specific transcription factors.  T helper 17 (Th17) has been recently identified as a distinct Th lineage mediating tissue inflammation.  Retinoic acid receptor-related orphan receptor γ (RORγ) was shown to regulate Th17 differentiation; RORγ deficiency, however, did not completely abolish Th17 cytokine expression.  Here, we report Th17 cells highly expressed another related nuclear receptor, RORα, induced by transforming growth factor-β and interleukin-6 (IL-6), which is dependent on signal transducer and activator of transcription 3.  Overexpression of RORα promoted Th17 differentiation, possibly through the conserved noncoding sequence 2 in II17-II17f locus.  RORα deficiency resulted in reduced IL-17 expression in vitro and in vivo.  Furthermore, RORα and RORγ coexpression synergistically led to greater Th17 differentiation.  Double deficiencies in RORα and RORγ globally impaired Th17 generation and completely protected mice against exptl. autoimmune encephalomyelitis.  Therefore, Th17 differentiation is directed by two lineage-specific nuclear receptors, RORα and RORγ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBQRyY1XQpJbVg90H21EOLACvtfcHk0lirApyP_3qvlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D&md5=436602649095b96d366c983addf526f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2007.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2007.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BO.%26aulast%3DPappu%26aufirst%3DB.%2BP.%26aulast%3DNurieva%26aufirst%3DR.%26aulast%3DAkimzhanov%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DH.%2BS.%26aulast%3DChung%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DShah%26aufirst%3DB.%26aulast%3DPanopoulos%26aufirst%3DA.%2BD.%26aulast%3DSchluns%26aufirst%3DK.%2BS.%26aulast%3DWatowich%26aufirst%3DS.%2BS.%26aulast%3DTian%26aufirst%3DQ.%26aulast%3DJetten%26aufirst%3DA.%2BM.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DT%2520Helper%252017%2520Lineage%2520Differentiation%2520Is%2520Programmed%2520by%2520Orphan%2520Nuclear%2520Receptors%2520ROR%25CE%25B1%2520and%2520ROR%25CE%25B3%26jtitle%3DImmunity%26date%3D2008%26volume%3D28%26spage%3D29%26epage%3D39%26doi%3D10.1016%2Fj.immuni.2007.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oukka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">IL-17 and Th17 Cells</span>. <i>Annu. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1146/annurev.immunol.021908.132710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1146%2Fannurev.immunol.021908.132710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=19132915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFSltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=485-517&author=T.+Kornauthor=E.+Bettelliauthor=M.+Oukkaauthor=V.+K.+Kuchroo&title=IL-17+and+Th17+Cells&doi=10.1146%2Fannurev.immunol.021908.132710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">IL-17 and Th17 cells</span></div><div class="casAuthors">Korn, Thomas; Bettelli, Estelle; Oukka, Mohamed; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">485-517</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  CD4+ T cells, upon activation and expansion, develop into different T helper cell subsets with different cytokine profiles and distinct effector functions.  Until recently, T cells were divided into Th1 or Th2 cells, depending on the cytokines they produce.  A third subset of IL-17-producing effector T helper cells, called Th17 cells, has now been discovered and characterized.  Here, the authors summarize the current information on the differentiation and effector functions of the Th17 lineage.  Th17 cells produce IL-17, IL-17F, and IL-22, thereby inducing a massive tissue reaction owing to the broad distribution of the IL-17 and IL-22 receptors.  Th17 cells also secrete IL-21 to communicate with the cells of the immune system.  The differentiation factors (TGF-β plus IL-6 or IL-21), the growth and stabilization factor (IL-23), and the transcription factors (STAT3, RORγt, and RORα) involved in the development of Th17 cells have just been identified.  The participation of TGF-β in the differentiation of Th17 cells places the Th17 lineage in close relation with CD4+CD25+Foxp3+ regulatory T cells (Tregs), as TGF-β also induces differentiation of native T cells into Foxp3+ Tregs in the peripheral immune compartment.  The investigation of the differentiation, effector function, and regulation of Th17 cells has opened up a new framework for understanding T cell differentiation.  Furthermore, the importance of Th17 cells is now appreciated in clearing pathogens during host defense reactions and in inducing tissue inflammation in autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAJvCEP9y1fLVg90H21EOLACvtfcHk0lirApyP_3qvlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFSltrY%253D&md5=fffce537e2b70751fd9113e25962419e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.immunol.021908.132710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.immunol.021908.132710%26sid%3Dliteratum%253Aachs%26aulast%3DKorn%26aufirst%3DT.%26aulast%3DBettelli%26aufirst%3DE.%26aulast%3DOukka%26aufirst%3DM.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DIL-17%2520and%2520Th17%2520Cells%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D27%26spage%3D485%26epage%3D517%26doi%3D10.1146%2Fannurev.immunol.021908.132710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">Th17 Cell Pathway in Human Immunity: Lessons from Genetics and Therapeutic Interventions</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1040</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2015.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.immuni.2015.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=26682981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVajtb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=1040-1051&author=D.+D.+Patelauthor=V.+K.+Kuchroo&title=Th17+Cell+Pathway+in+Human+Immunity%3A+Lessons+from+Genetics+and+Therapeutic+Interventions&doi=10.1016%2Fj.immuni.2015.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Th17 Cell Pathway in Human Immunity: Lessons from Genetics and Therapeutic Interventions</span></div><div class="casAuthors">Patel, Dhavalkumar D.; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1040-1051</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The T helper 17 (Th17) cell pathway has been linked by genome-wide assocn. studies to multiple autoimmune diseases.  Identification of the genetic causes of primary immunodeficiency diseases revealed that Th17 cells are also crit. in host immunity to mucocutaneous candida infections and Staphylococcus aureus.  Therapeutic interventions with inhibitors of the different components of the pathway such as interleukin-12 (IL-12), IL-23, IL-17A, and IL-17RA have variably beneficial effects in psoriasis, Crohn's disease, rheumatoid arthritis, psoriatic arthritis, ankylosing spondylitis, non-infectious uveitis, and multiple sclerosis.  Thus, whereas Th17 cells are protective against Candida albicans and to a lesser degree Staphylococcus aureus, they are pathogenic in many autoimmune diseases.  Here, we compare and contrast the effects of human genetic mutations of and therapeutic interventions targeted at Th17 cell mols.  We discuss that although there are similarities when Th17 cell pathway mols. are modulated, each mol. has unique non-Th17 cell features that lead to different functional outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb--L2SA4SpLVg90H21EOLACvtfcHk0lirApyP_3qvlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVajtb7I&md5=5af767afe2a213dafa7c07f2b4271242</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2015.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2015.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DTh17%2520Cell%2520Pathway%2520in%2520Human%2520Immunity%253A%2520Lessons%2520from%2520Genetics%2520and%2520Therapeutic%2520Interventions%26jtitle%3DImmunity%26date%3D2015%26volume%3D43%26spage%3D1040%26epage%3D1051%26doi%3D10.1016%2Fj.immuni.2015.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jetten, A. M.</span></span> <span> </span><span class="NLM_article-title">Retinoid-Related Orphan Receptors (RORs): Critical Roles in Development, Immunity, Circadian Rhythm, and Cellular Metabolism</span>. <i>Nucl. Recept. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">nrs.07003</span>, <span class="refDoi"> DOI: 10.1621/nrs.07003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1621%2Fnrs.07003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=nrs.07003&author=A.+M.+Jetten&title=Retinoid-Related+Orphan+Receptors+%28RORs%29%3A+Critical+Roles+in+Development%2C+Immunity%2C+Circadian+Rhythm%2C+and+Cellular+Metabolism&doi=10.1621%2Fnrs.07003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1621%2Fnrs.07003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1621%252Fnrs.07003%26sid%3Dliteratum%253Aachs%26aulast%3DJetten%26aufirst%3DA.%2BM.%26atitle%3DRetinoid-Related%2520Orphan%2520Receptors%2520%2528RORs%2529%253A%2520Critical%2520Roles%2520in%2520Development%252C%2520Immunity%252C%2520Circadian%2520Rhythm%252C%2520and%2520Cellular%2520Metabolism%26jtitle%3DNucl.%2520Recept.%2520Signaling%26date%3D2009%26volume%3D7%26spage%3Dnrs.07003%26doi%3D10.1621%2Fnrs.07003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachchidanand</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, R. C.</span></span> <span> </span><span class="NLM_article-title">Combating Autoimmune Diseases With Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc) Inhibitors: Hits and Misses</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10976</span>– <span class="NLM_lpage">10995</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00588</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00588" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10976-10995&author=V.+B.+Pandyaauthor=S.+Kumarauthor=+Sachchidanandauthor=R.+Sharmaauthor=R.+C.+Desai&title=Combating+Autoimmune+Diseases+With+Retinoic+Acid+Receptor-Related+Orphan+Receptor-%CE%B3+%28ROR%CE%B3+or+RORc%29+Inhibitors%3A+Hits+and+Misses&doi=10.1021%2Facs.jmedchem.8b00588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Autoimmune Diseases With Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc) Inhibitors: Hits and Misses</span></div><div class="casAuthors">Pandya, Vrajesh B.; Kumar, Sanjay; Sachchidanand; Sharma, Rajiv; Desai, Ranjit C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10976-10995</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclear receptor retinoic acid receptor-related orphan receptor gamma (RORγ or RORc) is a key transcription factor for the prodn. of pro-inflammatory cytokines implicated in the pathogenesis of autoimmune diseases.  Recently, small mol. inhibitors of RORc drew the enormous attention of the research community worldwide as a possible therapy for autoimmune diseases, mediated by the IL-17 cytokine.  With the clin. proof-of-concept inferred from a small mol. inhibitor VTP-43742 for psoriasis and recent inflow of several RORc inhibitors into the clinic for therapeutic interventions in autoimmune diseases, this field continues to evolve.  This review briefly summarizes the RORc inhibitors disclosed in the literature and discusses the progress made by these inhibitors in combating autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKGqZl3bPbtLVg90H21EOLACvtfcHk0liJZOHfFsyx-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E&md5=4113d22f73ab2911535eef053801994d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00588%26sid%3Dliteratum%253Aachs%26aulast%3DPandya%26aufirst%3DV.%2BB.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DSachchidanand%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DDesai%26aufirst%3DR.%2BC.%26atitle%3DCombating%2520Autoimmune%2520Diseases%2520With%2520Retinoic%2520Acid%2520Receptor-Related%2520Orphan%2520Receptor-%25CE%25B3%2520%2528ROR%25CE%25B3%2520or%2520RORc%2529%2520Inhibitors%253A%2520Hits%2520and%2520Misses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10976%26epage%3D10995%26doi%3D10.1021%2Facs.jmedchem.8b00588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosef, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baloglu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundrud, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretschmer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafler, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule RORγt Antagonists Inhibit T Helper 17 Cell Transcriptional Network by Divergent Mechanisms</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2014.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.immuni.2014.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=24745332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVKltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2014&pages=477-489&author=S.+Xiaoauthor=N.+Yosefauthor=J.+Yangauthor=Y.+Wangauthor=L.+Zhouauthor=C.+Zhuauthor=C.+Wuauthor=E.+Balogluauthor=D.+Schmidtauthor=R.+Rameshauthor=M.+Loberaauthor=M.+S.+Sundrudauthor=P.-Y.+Tsaiauthor=Z.+Xiangauthor=J.+Wangauthor=Y.+Xuauthor=X.+Linauthor=K.+Kretschmerauthor=P.+B.+Rahlauthor=R.+A.+Youngauthor=Z.+Zhongauthor=D.+A.+Haflerauthor=A.+Regevauthor=S.+Ghoshauthor=A.+Marsonauthor=V.+K.+Kuchroo&title=Small-Molecule+ROR%CE%B3t+Antagonists+Inhibit+T+Helper+17+Cell+Transcriptional+Network+by+Divergent+Mechanisms&doi=10.1016%2Fj.immuni.2014.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule RORγt Antagonists Inhibit T Helper 17 Cell Transcriptional Network by Divergent Mechanisms</span></div><div class="casAuthors">Xiao, Sheng; Yosef, Nir; Yang, Jianfei; Wang, Yonghui; Zhou, Ling; Zhu, Chen; Wu, Chuan; Baloglu, Erkan; Schmidt, Darby; Ramesh, Radha; Lobera, Mercedes; Sundrud, Mark S.; Tsai, Pei-Yun; Xiang, Zhijun; Wang, Jinsong; Xu, Yan; Lin, Xichen; Kretschmer, Karsten; Rahl, Peter B.; Young, Richard A.; Zhong, Zhong; Hafler, David A.; Regev, Aviv; Ghosh, Shomir; Marson, Alexander; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-489</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We identified three retinoid-related orphan receptor gamma t (RORγt)-specific inhibitors that suppress T helper 17 (Th17) cell responses, including Th17-cell-mediated autoimmune disease.  We systemically characterized RORγt binding in the presence and absence of drugs with corresponding whole-genome transcriptome sequencing.  RORγt acts as a direct activator of Th17 cell signature genes and a direct repressor of signature genes from other T cell lineages; its strongest transcriptional effects are on cis-regulatory sites contg. the RORα binding motif.  RORγt is central in a densely interconnected regulatory network that shapes the balance of T cell differentiation.  Here, the three inhibitors modulated the RORγt-dependent transcriptional network to varying extents and through distinct mechanisms.  Whereas one inhibitor displaced RORγt from its target loci, the other two inhibitors affected transcription predominantly without removing DNA binding.  Our work illustrates the power of a system-scale anal. of transcriptional regulation to characterize potential therapeutic compds. that inhibit pathogenic Th17 cells and suppress autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo23p8R83CDDbVg90H21EOLACvtfcHk0liJZOHfFsyx-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVKltr0%253D&md5=940074c565211bc00832179574e5231a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2014.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2014.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DYosef%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DRamesh%26aufirst%3DR.%26aulast%3DLobera%26aufirst%3DM.%26aulast%3DSundrud%26aufirst%3DM.%2BS.%26aulast%3DTsai%26aufirst%3DP.-Y.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DKretschmer%26aufirst%3DK.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DZhong%26aufirst%3DZ.%26aulast%3DHafler%26aufirst%3DD.%2BA.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DMarson%26aufirst%3DA.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DSmall-Molecule%2520ROR%25CE%25B3t%2520Antagonists%2520Inhibit%2520T%2520Helper%252017%2520Cell%2520Transcriptional%2520Network%2520by%2520Divergent%2520Mechanisms%26jtitle%3DImmunity%26date%3D2014%26volume%3D40%26spage%3D477%26epage%3D489%26doi%3D10.1016%2Fj.immuni.2014.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skepner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trocha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baloglu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orband-Miller, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudjelal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundrud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic Inhibition of RORγt Regulates Th17 Signature Gene Expression and Suppresses Cutaneous Inflammation In Vivo</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">2564</span>– <span class="NLM_lpage">2575</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1302190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.4049%2Fjimmunol.1302190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=24516202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslGls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2014&pages=2564-2575&author=J.+Skepnerauthor=R.+Rameshauthor=M.+Trochaauthor=D.+Schmidtauthor=E.+Balogluauthor=M.+Loberaauthor=T.+Carlsonauthor=J.+Hillauthor=L.+A.+Orband-Millerauthor=A.+Barnesauthor=M.+Boudjelalauthor=M.+Sundrudauthor=S.+Ghoshauthor=J.+Yang&title=Pharmacologic+Inhibition+of+ROR%CE%B3t+Regulates+Th17+Signature+Gene+Expression+and+Suppresses+Cutaneous+Inflammation+In+Vivo&doi=10.4049%2Fjimmunol.1302190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of RORγt Regulates Th17 Signature Gene Expression and Suppresses Cutaneous Inflammation In Vivo</span></div><div class="casAuthors">Skepner, Jill; Ramesh, Radha; Trocha, Mark; Schmidt, Darby; Baloglu, Erkan; Lobera, Mercedes; Carlson, Thaddeus; Hill, Jonathan; Orband-Miller, Lisa A.; Barnes, Ashley; Boudjelal, Mohamed; Sundrud, Mark; Ghosh, Shomir; Yang, Jianfei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2564-2575</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">IL-17-producing CD4+Th17 cells, CD8+Tc17 cells, and γδ T cells play crit. roles in the pathogenesis of autoimmune psoriasis.  RORγt is required for the differentiation of Th17 cells and expression of IL-17.  In this article, we describe a novel, potent, and selective RORγt inverse agonist (TMP778), and its inactive diastereomer (TMP776).  This chem., for the first time to our knowledge, provides a unique and powerful set of tools to probe RORγt-dependent functions.  TMP778, but not TMP776, blocked human Th17 and Tc17 cell differentiation and also acutely modulated IL-17A prodn. and inflammatory Th17-signature gene expression (Il17a, Il17f, Il22, Il26, Ccr6, and Il23) in mature human Th17 effector/memory T cells.  In addn., TMP778, but not TMP776, inhibited IL-17A prodn. in both human and mouse γδ T cells.  IL-23-induced IL-17A prodn. was also blocked by TMP778 treatment.  In vivo targeting of RORγt in mice via TMP778 administration reduced imiquimod-induced psoriasis-like cutaneous inflammation.  Further, TMP778 selectively regulated Th17-signature gene expression in mononuclear cells isolated from both the blood and affected skin of psoriasis patients.  In summary, to our knowledge, we are the first to demonstrate that RORγt inverse agonists: (1) inhibit Tc17 cell differentiation, as well as IL-17 prodn. by γδ T cells and CD8+ Tc17 cells; (2) block imiquimod-induced cutaneous inflammation; (3) inhibit Th17 signature gene expression by cells isolated from psoriatic patient samples; and (4) block IL-23-induced IL-17A expression.  Thus, RORγt is a tractable drug target for the treatment of cutaneous inflammatory disorders, which may afford addnl. therapeutic benefit over existing modalities that target only IL-17A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu4CFK2lBK6bVg90H21EOLACvtfcHk0liVSSfPZ0_QAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslGls7w%253D&md5=d72bc88b09a34874cf7b89f993977d35</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1302190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1302190%26sid%3Dliteratum%253Aachs%26aulast%3DSkepner%26aufirst%3DJ.%26aulast%3DRamesh%26aufirst%3DR.%26aulast%3DTrocha%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DLobera%26aufirst%3DM.%26aulast%3DCarlson%26aufirst%3DT.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DOrband-Miller%26aufirst%3DL.%2BA.%26aulast%3DBarnes%26aufirst%3DA.%26aulast%3DBoudjelal%26aufirst%3DM.%26aulast%3DSundrud%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DJ.%26atitle%3DPharmacologic%2520Inhibition%2520of%2520ROR%25CE%25B3t%2520Regulates%2520Th17%2520Signature%2520Gene%2520Expression%2520and%2520Suppresses%2520Cutaneous%2520Inflammation%2520In%2520Vivo%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D192%26spage%3D2564%26epage%3D2575%26doi%3D10.4049%2Fjimmunol.1302190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guendisch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecoeur, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riker, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Inhibition of RORγt Suppresses the Th17 Pathway and Alleviates Arthritis in Vivo</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0188391</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0188391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1371%2Fjournal.pone.0188391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=29155882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWrs7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=U.+Guendischauthor=J.+Weissauthor=F.+Ecoeurauthor=J.+C.+Rikerauthor=K.+Kaupmannauthor=J.+Kallenauthor=S.+Hintermannauthor=D.+Orainauthor=J.+Dawsonauthor=A.+Billichauthor=C.+Guntermann&title=Pharmacological+Inhibition+of+ROR%CE%B3t+Suppresses+the+Th17+Pathway+and+Alleviates+Arthritis+in+Vivo&doi=10.1371%2Fjournal.pone.0188391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of RORγt suppresses the Th17 pathway and alleviates arthritis in vivo</span></div><div class="casAuthors">Guendisch, Ulf; Weiss, Jessica; Ecoeur, Florence; Riker, Julia Christina; Kaupmann, Klemens; Kallen, Joerg; Hintermann, Samuel; Orain, David; Dawson, Janet; Billich, Andreas; Guntermann, Christine</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0188391/1-e0188391/25</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related-orphan-receptor-C (RORγt) is the key transcription factor that is driving the differentiation of IL-17 producing T-helper 17 (Th17) cells that are implicated in the pathol. of various autoimmune and inflammatory diseases.  Based on the importance of RORγt in promoting Th17-driven pathol., there is considerable interest to develop lowmol.- wt. compds. with the aim of inhibiting the transcriptional activity of this nuclear hormone receptor.  In this article, we describe the in vitro and in vivo pharmacol. of a potent and selective small-mol.-wt. RORγt inverse agonist.  The compd. binds to the ligand binding domain (LBD) of RORγt leading to displacement of a co-activator peptide.  We show for the first time that a RORγt inverse agonist down-regulates permissive histone H3 acetylation and methylation at the IL17A and IL23R promoter regions, thereby providing insight into the transcriptional inhibition of RORγt-dependent genes.  Consistent with this, the compd. effectively reduced IL-17A prodn. by polarized human T-cells and γδT-cells and attenuated transcription of RORγt target genes.  The inhibitor showed good in vivo efficacy in an antigen-induced arthritis model in rats and reduced the frequencies of IL-17A producing cells in ex vivo recall assays.  In summary, we demonstrate that inhibiting RORγt by a low-mol.-wt. inhibitor results in efficient and selective blockade of the pro-inflammatory Th17/IL-17A pathway making it an attractive target for Th17- mediated disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGa7wQ5ixe8LVg90H21EOLACvtfcHk0liVSSfPZ0_QAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWrs7zO&md5=0bb658eef161b17729676ec3358a60d7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0188391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0188391%26sid%3Dliteratum%253Aachs%26aulast%3DGuendisch%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DJ.%26aulast%3DEcoeur%26aufirst%3DF.%26aulast%3DRiker%26aufirst%3DJ.%2BC.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DHintermann%26aufirst%3DS.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DBillich%26aufirst%3DA.%26aulast%3DGuntermann%26aufirst%3DC.%26atitle%3DPharmacological%2520Inhibition%2520of%2520ROR%25CE%25B3t%2520Suppresses%2520the%2520Th17%2520Pathway%2520and%2520Alleviates%2520Arthritis%2520in%2520Vivo%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0188391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kono, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yukawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokuhara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of [Cis-3-({(5R)-5-[(7-Fluoro-1,1-Dimethyl-2,3-Dihydro-1H-Inden-5-Yl)Carbamoyl]-2-Methoxy-7,8-Dihydro-1,6-Naphthyridin-6(5H)-Yl}carbonyl)Cyclobutyl]Acetic Acid (TAK-828F) as a Potent, Selective, and Orally Available Novel Retinoic Acid Receptor-Related Orphan Receptor γt Inverse Agonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2973</span>– <span class="NLM_lpage">2988</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00061</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00061" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvFWns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2973-2988&author=M.+Konoauthor=A.+Ochidaauthor=T.+Odaauthor=T.+Imadaauthor=Y.+Bannoauthor=N.+Tayaauthor=S.+Masadaauthor=T.+Kawamotoauthor=K.+Yonemoriauthor=Y.+Naraauthor=Y.+Fukaseauthor=T.+Yukawaauthor=H.+Tokuharaauthor=R.+Skeneauthor=B.-C.+Sangauthor=I.+D.+Hoffmanauthor=G.+P.+Snellauthor=K.+Ugaauthor=A.+Shibataauthor=K.+Igakiauthor=Y.+Nakamuraauthor=H.+Nakagawaauthor=N.+Tsuchimoriauthor=M.+Yamasakiauthor=J.+Shiraiauthor=S.+Yamamoto&title=Discovery+of+%5BCis-3-%28%7B%285R%29-5-%5B%287-Fluoro-1%2C1-Dimethyl-2%2C3-Dihydro-1H-Inden-5-Yl%29Carbamoyl%5D-2-Methoxy-7%2C8-Dihydro-1%2C6-Naphthyridin-6%285H%29-Yl%7Dcarbonyl%29Cyclobutyl%5DAcetic+Acid+%28TAK-828F%29+as+a+Potent%2C+Selective%2C+and+Orally+Available+Novel+Retinoic+Acid+Receptor-Related+Orphan+Receptor+%CE%B3t+Inverse+Agonist&doi=10.1021%2Facs.jmedchem.8b00061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of [cis-3-({(5R)-5-[(7-Fluoro-1,1-dimethyl-2,3-dihydro-1H-inden-5-yl)carbamoyl]-2-methoxy-7,8-dihydro-1,6-naphthyridin-6(5H)-yl}carbonyl)cyclobutyl]acetic Acid (TAK-828F) as a Potent, Selective, and Orally Available Novel Retinoic Acid Receptor-Related Orphan Receptor γt Inverse Agonist</span></div><div class="casAuthors">Kono, Mitsunori; Ochida, Atsuko; Oda, Tsuneo; Imada, Takashi; Banno, Yoshihiro; Taya, Naohiro; Masada, Shinichi; Kawamoto, Tetsuji; Yonemori, Kazuko; Nara, Yoshi; Fukase, Yoshiyuki; Yukawa, Tomoya; Tokuhara, Hidekazu; Skene, Robert; Sang, Bi-Ching; Hoffman, Isaac D.; Snell, Gyorgy P.; Uga, Keiko; Shibata, Akira; Igaki, Keiko; Nakamura, Yoshiki; Nakagawa, Hideyuki; Tsuchimori, Noboru; Yamasaki, Masashi; Shirai, Junya; Yamamoto, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2973-2988</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of tetrahydronaphthyridine derivs. as novel RORγt inverse agonists were designed and synthesized.  We reduced the lipophilicity of tetrahydroisoquinoline compd. 1 by replacement of the trimethylsilyl group and SBDD-guided scaffold exchange, which successfully afforded compd. 7 (VII) with a lower log D value and tolerable in vitro activity.  Consideration of LLE (lipophilic ligand efficiency) values in the subsequent optimization of the carboxylate tether led to the discovery of [cis-3-({(5R)-5-[(7-fluoro-1,1-dimethyl-2,3-dihydro-1H-inden-5-yl)carbamoyl]-2-methoxy-7,8-dihydro-1,6-naphthyridin-6(5H)-yl}carbonyl)cyclobutyl]acetic acid, TAK-828F (10), which showed potent RORγt inverse agonistic activity, excellent selectivity against other ROR isoforms and nuclear receptors, and a good pharmacokinetic profile.  In animal studies, oral administration of compd. 10 exhibited robust and dose-dependent inhibition of IL-17A cytokine expression in a mouse IL23-induced gene expression assay.  Furthermore, development of clin. symptoms in a mouse exptl. autoimmune encephalomyelitis model was significantly reduced.  Compd. 10 was selected as a clin. compd. for the treatment of Th17-driven autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqraNPFfTkC1rVg90H21EOLACvtfcHk0liVSSfPZ0_QAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvFWns7Y%253D&md5=966c20bbdf825d1ff6eb91c5950a33c8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00061%26sid%3Dliteratum%253Aachs%26aulast%3DKono%26aufirst%3DM.%26aulast%3DOchida%26aufirst%3DA.%26aulast%3DOda%26aufirst%3DT.%26aulast%3DImada%26aufirst%3DT.%26aulast%3DBanno%26aufirst%3DY.%26aulast%3DTaya%26aufirst%3DN.%26aulast%3DMasada%26aufirst%3DS.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DYonemori%26aufirst%3DK.%26aulast%3DNara%26aufirst%3DY.%26aulast%3DFukase%26aufirst%3DY.%26aulast%3DYukawa%26aufirst%3DT.%26aulast%3DTokuhara%26aufirst%3DH.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DSang%26aufirst%3DB.-C.%26aulast%3DHoffman%26aufirst%3DI.%2BD.%26aulast%3DSnell%26aufirst%3DG.%2BP.%26aulast%3DUga%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DA.%26aulast%3DIgaki%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DTsuchimori%26aufirst%3DN.%26aulast%3DYamasaki%26aufirst%3DM.%26aulast%3DShirai%26aufirst%3DJ.%26aulast%3DYamamoto%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520%255BCis-3-%2528%257B%25285R%2529-5-%255B%25287-Fluoro-1%252C1-Dimethyl-2%252C3-Dihydro-1H-Inden-5-Yl%2529Carbamoyl%255D-2-Methoxy-7%252C8-Dihydro-1%252C6-Naphthyridin-6%25285H%2529-Yl%257Dcarbonyl%2529Cyclobutyl%255DAcetic%2520Acid%2520%2528TAK-828F%2529%2520as%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Orally%2520Available%2520Novel%2520Retinoic%2520Acid%2520Receptor-Related%2520Orphan%2520Receptor%2520%25CE%25B3t%2520Inverse%2520Agonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2973%26epage%3D2988%26doi%3D10.1021%2Facs.jmedchem.8b00061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carcache, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulpetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vangrevelinghe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegenauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span> <span> </span><span class="NLM_article-title">Optimizing a Weakly Binding Fragment into a Potent RORγt Inverse Agonist with Efficacy in an in Vivo Inflammation Model</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6724</span>– <span class="NLM_lpage">6735</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00529</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00529" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht12jsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6724-6735&author=D.+A.+Carcacheauthor=A.+Vulpettiauthor=J.+Kallenauthor=H.+Mattesauthor=D.+Orainauthor=R.+Stringerauthor=E.+Vangrevelingheauthor=R.+M.+Wolfauthor=K.+Kaupmannauthor=J.+Ottlauthor=J.+Dawsonauthor=N.+G.+Cookeauthor=K.+Hoegenauerauthor=A.+Billichauthor=J.+Wagnerauthor=C.+Guntermannauthor=S.+Hintermann&title=Optimizing+a+Weakly+Binding+Fragment+into+a+Potent+ROR%CE%B3t+Inverse+Agonist+with+Efficacy+in+an+in+Vivo+Inflammation+Model&doi=10.1021%2Facs.jmedchem.8b00529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing a Weakly Binding Fragment into a Potent RORγt Inverse Agonist with Efficacy in an in Vivo Inflammation Model</span></div><div class="casAuthors">Carcache, David A.; Vulpetti, Anna; Kallen, Joerg; Mattes, Henri; Orain, David; Stringer, Rowan; Vangrevelinghe, Eric; Wolf, Romain M.; Kaupmann, Klemens; Ottl, Johannes; Dawson, Janet; Cooke, Nigel G.; Hoegenauer, Klemens; Billich, Andreas; Wagner, Juergen; Guntermann, Christine; Hintermann, Samuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6724-6735</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor RORγt is an attractive drug-target due to its role in the differentiation of IL-17 producing Th17 cells that play a crit. role in the etiopathol. of several autoimmune diseases.  Identification of starting points for RORγt inverse agonists with good properties has been a challenge.  We report the identification of a fragment hit and its conversion into a potent inverse agonist through fragment optimization, growing and merging efforts.  Further anal. of the binding mode revealed that inverse agonism was achieved by an unusual mechanism.  In contrast to other reported inverse agonists, there is no direct interaction or displacement of helix 12 obsd. in the crystal structure.  Nevertheless, compd. 9 proved to be efficacious in a delayed-type hypersensitivity (DTH) inflammation model in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgNfQTVtmUt7Vg90H21EOLACvtfcHk0lg4ASYl7FAmBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht12jsbjK&md5=6032c6b6a85377ea247a92314736db06</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00529%26sid%3Dliteratum%253Aachs%26aulast%3DCarcache%26aufirst%3DD.%2BA.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DMattes%26aufirst%3DH.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DStringer%26aufirst%3DR.%26aulast%3DVangrevelinghe%26aufirst%3DE.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DCooke%26aufirst%3DN.%2BG.%26aulast%3DHoegenauer%26aufirst%3DK.%26aulast%3DBillich%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DJ.%26aulast%3DGuntermann%26aufirst%3DC.%26aulast%3DHintermann%26aufirst%3DS.%26atitle%3DOptimizing%2520a%2520Weakly%2520Binding%2520Fragment%2520into%2520a%2520Potent%2520ROR%25CE%25B3t%2520Inverse%2520Agonist%2520with%2520Efficacy%2520in%2520an%2520in%2520Vivo%2520Inflammation%2520Model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6724%26epage%3D6735%26doi%3D10.1021%2Facs.jmedchem.8b00529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="note"><p class="first last">The terminology for compounds displaying such a functional response is heterogeneous. While many reports use receptor pharmacology vocabulary such as “inverse agonist” or “antagonist”, this analogy is problematic as binding partners to nuclear hormone receptors can be co-activators as well as co-repressors.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izaac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Be, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arista, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegenauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span> <span> </span><span class="NLM_article-title">Structural States of RORγt: X-ray Elucidation of Molecular Mechanisms and Binding Interactions for Natural and Synthetic Compounds</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1021</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1002%2Fcmdc.201700278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=28590087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFamsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1014-1021&author=J.+Kallenauthor=A.+Izaacauthor=C.+Beauthor=L.+Aristaauthor=D.+Orainauthor=K.+Kaupmannauthor=C.+Guntermannauthor=K.+Hoegenauerauthor=S.+Hintermann&title=Structural+States+of+ROR%CE%B3t%3A+X-ray+Elucidation+of+Molecular+Mechanisms+and+Binding+Interactions+for+Natural+and+Synthetic+Compounds&doi=10.1002%2Fcmdc.201700278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural States of RORγt: X-ray Elucidation of Molecular Mechanisms and Binding Interactions for Natural and Synthetic Compounds</span></div><div class="casAuthors">Kallen, Joerg; Izaac, Aude; Be, Celine; Arista, Luca; Orain, David; Kaupmann, Klemens; Guntermann, Christine; Hoegenauer, Klemens; Hintermann, Samuel</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1014-1021</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The T-cell-specific retinoic acid receptor (RAR)-related orphan receptor-γ (RORγt) is a key transcription factor for the prodn. of pro-inflammatory Th17 cytokines, which are implicated in the pathogenesis of autoimmune diseases.  Over the years, several structurally diverse RORγt inverse agonists have been reported, but combining high potency and good physicochem. properties has remained a challenging task.  We recently reported a new series of inverse agonists based on an imidazopyridine core with good physicochem. properties and excellent selectivity.  Herein we report eight new X-ray crystal structures for different classes of natural and synthetic compds., including examples selected from the patent literature.  Anal. of their resp. binding modes revealed insight into the mol. mechanisms that lead to agonism, antagonism, or inverse agonism.  We report new mol. mechanisms for RORγt agonism and propose a sepn. of the inverse agonists into two classes: those that act via steric clash and those that act via other mechanisms (for the latter, co-crystn. with a co-activator peptide and helix 12 in the agonist position is still possible).  For the non-steric clash inverse agonists, we propose a new mechanism ("water trapping") which can be combined with other mechanisms (e.g., close contacts with H479).  In addn., we compare the interactions made for selected compds. in the "back pocket" near S404 and in the "sulfate pocket" near R364 and R367.  Taken together, these new mechanistic insights should prove useful for the design and optimization of further RORγt modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2bJdGLB1okrVg90H21EOLACvtfcHk0lg4ASYl7FAmBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFamsrvK&md5=c49d7d9710f6c4f7baed7e7b207a0bf7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700278%26sid%3Dliteratum%253Aachs%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DIzaac%26aufirst%3DA.%26aulast%3DBe%26aufirst%3DC.%26aulast%3DArista%26aufirst%3DL.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DGuntermann%26aufirst%3DC.%26aulast%3DHoegenauer%26aufirst%3DK.%26aulast%3DHintermann%26aufirst%3DS.%26atitle%3DStructural%2520States%2520of%2520ROR%25CE%25B3t%253A%2520X-ray%2520Elucidation%2520of%2520Molecular%2520Mechanisms%2520and%2520Binding%2520Interactions%2520for%2520Natural%2520and%2520Synthetic%2520Compounds%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1014%26epage%3D1021%26doi%3D10.1002%2Fcmdc.201700278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Questel, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renault, E.</span></span> <span> </span><span class="NLM_article-title">The PKBHX Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4073</span>– <span class="NLM_lpage">4086</span>, <span class="refDoi"> DOI: 10.1021/jm801331y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801331y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4073-4086&author=C.+Laurenceauthor=K.+A.+Brameldauthor=J.+Gratonauthor=J.-Y.+Le+Questelauthor=E.+Renault&title=The+PKBHX+Database%3A+Toward+a+Better+Understanding+of+Hydrogen-Bond+Basicity+for+Medicinal+Chemists&doi=10.1021%2Fjm801331y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The pKBHX Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span></div><div class="casAuthors">Laurence, Christian; Brameld, Ken A.; Graton, Jerome; Le Questel, Jean-Yves; Renault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4073-4086</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The pKBHX database is presented, which corresponds to three main fields of data: hydrogen bond donor identification, thermodn. data and spectroscopic data.  The pKBHX scale of hydrogen donor basicity differs considerable from the pKBH+ scale of proton transfer basicity.  The hydrogen bond basicities of functional groups relevant to medicinal chem. and drug design is reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql9OhmMu2VILVg90H21EOLACvtfcHk0lgAuuSCur6oIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D&md5=9ce4bea3e810752baba6ea5784b55ea0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm801331y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801331y%26sid%3Dliteratum%253Aachs%26aulast%3DLaurence%26aufirst%3DC.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DGraton%26aufirst%3DJ.%26aulast%3DLe%2BQuestel%26aufirst%3DJ.-Y.%26aulast%3DRenault%26aufirst%3DE.%26atitle%3DThe%2520PKBHX%2520Database%253A%2520Toward%2520a%2520Better%2520Understanding%2520of%2520Hydrogen-Bond%2520Basicity%2520for%2520Medicinal%2520Chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4073%26epage%3D4086%26doi%3D10.1021%2Fjm801331y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeuillet-Ritzler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briand, X.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Lipophilicity Measurement with Immobilized Artificial Membranes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2571</span>– <span class="NLM_lpage">2576</span>, <span class="refDoi"> DOI: 10.1021/jm049377w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049377w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsF2isLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2571-2576&author=B.+Fallerauthor=H.+P.+Grimmauthor=F.+Loeuillet-Ritzlerauthor=S.+Arnoldauthor=X.+Briand&title=High-Throughput+Lipophilicity+Measurement+with+Immobilized+Artificial+Membranes&doi=10.1021%2Fjm049377w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Lipophilicity Measurement with Immobilized Artificial Membranes</span></div><div class="casAuthors">Faller, Bernard; Grimm, Hans Peter; Loeuillet-Ritzler, Frederique; Arnold, Sabine; Briand, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2571-2576</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report on a new, high-throughput assay designed to measure octanol/water partition coeffs. in early drug discovery.  The assay is carried out in 96-well microtiter plates and measures the diffusion of compds. between two aq. compartments sepd. by a thin octanol liq. layer. Octanol/water partition coeffs. are derived from the apparent permeability (Pa) values using a calibration curve.  The assay can measure partition coeffs. within the range -2 to + 8; thus, a dynamic range of 10 log units can be covered in one single run.  Unlike chromatog. methods, the technol. is not restricted to neutral and weakly basic compds., and, as no stationary phase is involved, the data can be strictly compared with values obtained from traditional methods such as shake-flask/HPLC or dual-phase potentiometric titrn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCq4XWfzWZ2bVg90H21EOLACvtfcHk0lgAuuSCur6oIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsF2isLc%253D&md5=dacee7abc2349eb0d595133dff427c48</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm049377w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049377w%26sid%3Dliteratum%253Aachs%26aulast%3DFaller%26aufirst%3DB.%26aulast%3DGrimm%26aufirst%3DH.%2BP.%26aulast%3DLoeuillet-Ritzler%26aufirst%3DF.%26aulast%3DArnold%26aufirst%3DS.%26aulast%3DBriand%26aufirst%3DX.%26atitle%3DHigh-Throughput%2520Lipophilicity%2520Measurement%2520with%2520Immobilized%2520Artificial%2520Membranes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2571%26epage%3D2576%26doi%3D10.1021%2Fjm049377w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic Efficiency as an Important Metric in Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+Efficiency+as+an+Important+Metric+in+Drug+Design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0lgAuuSCur6oIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520Efficiency%2520as%2520an%2520Important%2520Metric%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. A.</span></span> <span> </span><span class="NLM_article-title">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">6</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1021/tx3003824</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx3003824" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=6-25&author=L.+A.+Peterson&title=Reactive+Metabolites+in+the+Biotransformation+of+Molecules+Containing+a+Furan+Ring&doi=10.1021%2Ftx3003824"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span></div><div class="casAuthors">Peterson, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-25</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many xenobiotics contg. a furan ring are toxic and/or carcinogenic.  The harmful effects of these compds. require furan ring oxidn.  This reaction generates an electrophilic intermediate.  Depending on the furan ring substituents, the intermediate is either an epoxide or a cis-enedione with more ring substitution favoring epoxide formation.  Either intermediate reacts with cellular nucleophiles such as protein or DNA to trigger toxicities.  The reactivity of the metabolite dets. which cellular nucleophiles are targeted.  The toxicity of a particular furan is also influenced by the presence of competing metabolic pathways or efficient detoxification routes.  GSH plays an important role in modulating the harmful effects of this class of compd. by reacting with the reactive metabolite.  However, this may not represent a detoxification step in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp4Dr5stLUSrVg90H21EOLACvtfcHk0lhhhFXa1n1ErQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN&md5=fe3a280d0b7a99a436ec619abc1e1fa0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Ftx3003824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx3003824%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DReactive%2520Metabolites%2520in%2520the%2520Biotransformation%2520of%2520Molecules%2520Containing%2520a%2520Furan%2520Ring%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D6%26epage%3D25%26doi%3D10.1021%2Ftx3003824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caceres-Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span> <span> </span><span class="NLM_article-title">Bioactivation of Substituted Thiophenes Including α-Chlorothiophene-Containing Compounds in Human Liver Microsomes</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">669</span>, <span class="refDoi"> DOI: 10.1021/tx100386z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx100386z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFOqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=663-669&author=W.+Chenauthor=J.+Caceres-Cortesauthor=H.+Zhangauthor=D.+Zhangauthor=W.+G.+Humphreysauthor=J.+Gan&title=Bioactivation+of+Substituted+Thiophenes+Including+%CE%B1-Chlorothiophene-Containing+Compounds+in+Human+Liver+Microsomes&doi=10.1021%2Ftx100386z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivation of Substituted Thiophenes Including α-Chlorothiophene-Containing Compounds in Human Liver Microsomes</span></div><div class="casAuthors">Chen, Weiqi; Caceres-Cortes, Janet; Zhang, Haiying; Zhang, Donglu; Humphreys, W. Griffith; Gan, Jinping</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">663-669</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The thiophene moiety has been recognized as a toxicophore because of the potential of oxidative bioactivation leading to electrophilic species.  The introduction of bulky or electron-withdrawing groups at the α-carbon to the sulfur atom has the potential to reduce or eliminate bioactivation.  In this article, we describe the bioactivation of a variety of substituted thiophenes.  These compds. were incubated in NADPH-fortified human liver microsomes with or without the addn. of reduced glutathione (GSH) as a trapping agent.  The resulting GSH adducts were characterized by high performance liq. chromatog./high resoln. mass spectrometry with the aid of a background subtraction methodol.  Four of the five α-chlorothiophenes tested formed NADPH-dependent GSH adducts.  Most adducts had masses consistent with the nominal substitution of chlorine by GSH.  LC/MS/MS and proton NMR of the major GSH adduct of 1-(5-chlorothiophen-2-yl)ethanone (1a) confirmed that GSH displaced chlorine.  To further explore the effect of different substitutions on the bioactivation potential, a series of 2-acetylthiophenes substituted at the C4 or C5 positions were tested in a quant. thiol-trapping assay using dansyl glutathione.  Substitutions at the C4 or C5 positions gave adduct levels that decreased in the following order: 4-H, 5-H (no substitution) > 4-Br ∼ 4-Cl > 5-Cl > 5-CN > 4-CH3 > 5-Br > 5-CH3 (no adduct detected).  In conclusion, bioactivation was detected in a series of substituted thiophenes.  Although substitutions on the thiophene ring can reduce the formation of reactive metabolites, the degree of redn. is dependent on the substitution position and substituent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRn2UHQF5AxbVg90H21EOLACvtfcHk0lhhhFXa1n1ErQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFOqtbw%253D&md5=e7aa1532248bff324b084905ea12c6e6</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Ftx100386z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx100386z%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DCaceres-Cortes%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHumphreys%26aufirst%3DW.%2BG.%26aulast%3DGan%26aufirst%3DJ.%26atitle%3DBioactivation%2520of%2520Substituted%2520Thiophenes%2520Including%2520%25CE%25B1-Chlorothiophene-Containing%2520Compounds%2520in%2520Human%2520Liver%2520Microsomes%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2011%26volume%3D24%26spage%3D663%26epage%3D669%26doi%3D10.1021%2Ftx100386z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velez-Vega, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, D. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravamuthan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, J. S.</span></span> <span> </span><span class="NLM_article-title">Time-Averaged Distributions of Solute and Solvent Motions: Exploring Proton Wires of GFP and PfM2DH</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3344</span>– <span class="NLM_lpage">3361</span>, <span class="refDoi"> DOI: 10.1021/ci500571h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500571h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3344-3361&author=C.+Velez-Vegaauthor=D.+J.+J.+McKayauthor=V.+Aravamuthanauthor=R.+Pearlsteinauthor=J.+S.+Duca&title=Time-Averaged+Distributions+of+Solute+and+Solvent+Motions%3A+Exploring+Proton+Wires+of+GFP+and+PfM2DH&doi=10.1021%2Fci500571h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Time-Averaged Distributions of Solute and Solvent Motions: Exploring Proton Wires of GFP and PfM2DH</span></div><div class="casAuthors">Velez-Vega, Camilo; McKay, Daniel J. J.; Aravamuthan, Vibhas; Pearlstein, Robert; Duca, Jose S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3344-3361</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proton translocation pathways of selected variants of the green fluorescent protein (GFP) and Pseudomonas fluorescens mannitol 2-dehydrogenase (PfM2DH) were studied via an explicit solvent mol. dynamics-based anal. protocol that allows for direct quant. relation between a crystal structure and its time-averaged solute-solvent structure obtained from simulation.  The authors' study of GFP is in good agreement with previous research suggesting that the proton released from the chromophore upon photoexcitation can diffuse through an extended internal hydrogen bonding network that allows for the proton to exit to bulk or be recaptured by the anionic chromophore.  Conversely for PfM2DH, the authors identified the most probable ionization states of key residues along the proton escape channel from the catalytic site to bulk solvent, wherein the solute and high-d. solvent crystal structures of binary and ternary complexes were properly reproduced.  Furthermore, the authors proposed a plausible mechanism for this proton translocation process that is consistent with the state-dependent structural shifts obsd. in the authors' anal.  The time-averaged structures generated from the authors' analyses facilitate validation of MD simulation results and provide a comprehensive profile of the dynamic all-occupancy solvation network within and around a flexible solute, from which detailed hydrogen-bonding networks can be inferred.  In this way, potential drawbacks arising from the elucidation of these networks by examn. of static crystal structures or via alternate rigid-protein solvation anal. procedures can be overcome.  Complementary studies aimed at the effective use of the authors' methodol. for alternate implementations (e.g., ligand design) are currently underway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpub7nL0UsqGrVg90H21EOLACvtfcHk0lhhhFXa1n1ErQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO&md5=d34a5696e577371ff4db5630923dedb0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci500571h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500571h%26sid%3Dliteratum%253Aachs%26aulast%3DVelez-Vega%26aufirst%3DC.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%2BJ.%26aulast%3DAravamuthan%26aufirst%3DV.%26aulast%3DPearlstein%26aufirst%3DR.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26atitle%3DTime-Averaged%2520Distributions%2520of%2520Solute%2520and%2520Solvent%2520Motions%253A%2520Exploring%2520Proton%2520Wires%2520of%2520GFP%2520and%2520PfM2DH%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3344%26epage%3D3361%26doi%3D10.1021%2Fci500571h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and Characterizing Binding Sites and Assessing Druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+Characterizing+Binding+Sites+and+Assessing+Druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0linsvXilwlDPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520Characterizing%2520Binding%2520Sites%2520and%2520Assessing%2520Druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein–Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0linsvXilwlDPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Low, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vachaspati, P.</span></span> <span> </span><span class="NLM_article-title">Optimised Method to Estimate Octanol Water Distribution Coefficient (LogD) in a High Throughput Format</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2016.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.ejps.2016.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=27373604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKrsr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2016&pages=110-116&author=Y.+W.+Lowauthor=F.+Blascoauthor=P.+Vachaspati&title=Optimised+Method+to+Estimate+Octanol+Water+Distribution+Coefficient+%28LogD%29+in+a+High+Throughput+Format&doi=10.1016%2Fj.ejps.2016.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Optimised method to estimate octanol water distribution coefficient (logD) in a high throughput format</span></div><div class="casAuthors">Low, Ying Wei; Blasco, Francesca; Vachaspati, Prakash</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110-116</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lipophilicity is one of the mol. properties assessed in early drug discovery.  Direct measurement of the octanol-water distribution coeff. (logD) requires an anal. method with a large dynamic range or multistep dilns., as the analyte's concns. span across several orders of magnitude.  In addn., water/buffer and octanol phases which have very different polarity could lead to matrix effects and affect the LC-MS response, leading to erroneous logD values.  Most compd. libraries use DMSO stocks as it greatly reduces the sample requirement but the presence of DMSO has been shown to underestimate the lipophilicity of the analyte.  The present work describes the development of an optimized shake flask logD method using deepwell 96 well plate that addresses the issues related to matrix effects, DMSO concn. and incubation conditions and is also amenable to high throughput.  Our results indicate that the equil. can be achieved within 30 min by flipping the plate on its side while even 0.5% of DMSO is not tolerated in the assay.  This study uses the matched matrix concept to minimise the errors in analyzing the two phases namely buffer and octanol in LC-MS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHQW7-Y-gkR7Vg90H21EOLACvtfcHk0linsvXilwlDPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKrsr%252FI&md5=97b8c5312efc8b7f535332af1c0d9cab</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2016.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2016.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DLow%26aufirst%3DY.%2BW.%26aulast%3DBlasco%26aufirst%3DF.%26aulast%3DVachaspati%26aufirst%3DP.%26atitle%3DOptimised%2520Method%2520to%2520Estimate%2520Octanol%2520Water%2520Distribution%2520Coefficient%2520%2528LogD%2529%2520in%2520a%2520High%2520Throughput%2520Format%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2016%26volume%3D92%26spage%3D110%26epage%3D116%26doi%3D10.1016%2Fj.ejps.2016.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trunzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span></span> <span> </span><span class="NLM_article-title">CYP3A Time-Dependent Inhibition Risk Assessment Validated with 400 Reference Drugs</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1046</span>, <span class="refDoi"> DOI: 10.1124/dmd.110.037911</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1124%2Fdmd.110.037911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=21383203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFGns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1039-1046&author=A.+Zimmerlinauthor=M.+Trunzerauthor=B.+Faller&title=CYP3A+Time-Dependent+Inhibition+Risk+Assessment+Validated+with+400+Reference+Drugs&doi=10.1124%2Fdmd.110.037911"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">CYP3A time-dependent inhibition risk assessment validated with 400 reference drugs</span></div><div class="casAuthors">Zimmerlin, Alfred; Trunzer, Markus; Faller, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1039-1046</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Although reversible CYP3A inhibition testing is well established for predicting the drug-drug interaction potential of clin. candidates, time-dependent inhibition (TDI) has become the focus of drug designers only recently.  Failure of several late-stage clin. candidates has been attributed to TDI, and this mechanism is also suspected to play a role in liver toxicities often obsd. in preclin. species.  Measurement of enzyme inactivation rates (kinact and KI) is tech. challenging, and a great deal of variability can be found in the literature.  In this article, we have evaluated the TDI potential for 400 registered drugs using a high-throughput assay format based on detn. of the inactivation rate (kobs) at a single concn. of test compd. (10 μM).  The advantages of this new assay format are highlighted by comparison with data generated using the IC50 shift assay, a current std. approach for preliminary assessment of TDI.  With use of an empirically defined pos./neg. kobs bin of 0.02 min-1, only 4% of registered drugs were found to be pos.  This proportion increased to more than 20% when inhouse lead optimization mols. were considered, emphasizing the importance of identifying this property in selection of promising drug candidates.  Finally, it is suggested that the data and technol. described here may be a good basis for building structure-activity relationships and in silico modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYvW4ft03jbVg90H21EOLACvtfcHk0linsvXilwlDPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFGns7w%253D&md5=d21b39e85ab9410b7addfc820c1a4e18</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fdmd.110.037911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.110.037911%26sid%3Dliteratum%253Aachs%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DTrunzer%26aufirst%3DM.%26aulast%3DFaller%26aufirst%3DB.%26atitle%3DCYP3A%2520Time-Dependent%2520Inhibition%2520Risk%2520Assessment%2520Validated%2520with%2520400%2520Reference%2520Drugs%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D1039%26epage%3D1046%26doi%3D10.1124%2Fdmd.110.037911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnes-Seeman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsis, P.</span></span> <span> </span><span class="NLM_article-title">Metabolically Stable Tert-Butyl Replacement</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/ml400045j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400045j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=514-516&author=D.+Barnes-Seemanauthor=M.+Jainauthor=L.+Bellauthor=S.+Ferreiraauthor=S.+Cohenauthor=X.-H.+Chenauthor=J.+Aminauthor=B.+Snodgrassauthor=P.+Hatsis&title=Metabolically+Stable+Tert-Butyl+Replacement&doi=10.1021%2Fml400045j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolically Stable tert-Butyl Replacement</span></div><div class="casAuthors">Barnes-Seeman, David; Jain, Monish; Bell, Leslie; Ferreira, Suzie; Cohen, Scott; Chen, Xiao-Hui; Amin, Jakal; Snodgrass, Brad; Hatsis, Panos</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">514-516</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Susceptibility to metab. is a common issue with the tert-Bu group on compds. of medicinal interest.  The authors demonstrate an approach of removing all the fully Sp3 C-Hs from a tert-Bu group: replacing some C-Hs with C-Fs and increasing the s-character of the remaining C-Hs.  This approach gave a trifluoromethylcyclopropyl group, which increased metabolic stability.  Trifluoromethylcyclopropyl-contg. analogs had consistently higher metabolic stability in vitro and in vivo compared to their tert-butyl-contg. counterparts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrao2I5ukFsTrVg90H21EOLACvtfcHk0lhXbKEN2SsLJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D&md5=156572485ea3a48f5e1b6ea7409f1f61</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fml400045j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400045j%26sid%3Dliteratum%253Aachs%26aulast%3DBarnes-Seeman%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DL.%26aulast%3DFerreira%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.-H.%26aulast%3DAmin%26aufirst%3DJ.%26aulast%3DSnodgrass%26aufirst%3DB.%26aulast%3DHatsis%26aufirst%3DP.%26atitle%3DMetabolically%2520Stable%2520Tert-Butyl%2520Replacement%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D514%26epage%3D516%26doi%3D10.1021%2Fml400045j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for Flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lhXbKEN2SsLJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Absolute Solvation Free Energies Using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1021/ct900587b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900587b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1509-1519&author=D.+Shivakumarauthor=J.+Williamsauthor=Y.+Wuauthor=W.+Dammauthor=J.+Shelleyauthor=W.+Sherman&title=Prediction+of+Absolute+Solvation+Free+Energies+Using+Molecular+Dynamics+Free+Energy+Perturbation+and+the+OPLS+Force+Field&doi=10.1021%2Fct900587b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Williams, Joshua; Wu, Yujie; Damm, Wolfgang; Shelley, John; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1509-1519</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of protein-ligand binding free energies is a primary objective in computer-aided drug design.  The solvation free energy of a small mol. provides a surrogate to the desolvation of the ligand in the thermodn. process of protein-ligand binding.  Here, we use explicit solvent mol. dynamics free energy perturbation to predict the abs. solvation free energies of a set of 239 small mols., spanning diverse chem. functional groups commonly found in drugs and drug-like mols.  We also compare the performance of abs. solvation free energies obtained using the OPLS_2005 force field with two other commonly used small mol. force fields - general AMBER force field (GAFF) with AM1-BCC charges and CHARMm-MSI with CHelpG charges.  Using the OPLS_2005 force field, we obtain high correlation with exptl. solvation free energies (R2 = 0.94) and low av. unsigned errors for a majority of the functional groups compared to AM1-BCC/GAFF or CHelpG/CHARMm-MSI.  However, OPLS_2005 has errors of over 1.3 kcal/mol for certain classes of polar compds.  We show that predictions on these compd. classes can be improved by using a semiempirical charge assignment method with an implicit bond charge correction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_fkbSHqg3bVg90H21EOLACvtfcHk0lhXbKEN2SsLJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D&md5=25d464fb9238e3235e881945c0b50a76</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fct900587b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900587b%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DPrediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520Using%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Perturbation%2520and%2520the%2520OPLS%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1509%26epage%3D1519%26doi%3D10.1021%2Fct900587b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piaia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubic-Schneider, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rio-Espinola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegenauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doelemeyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deurinck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schümann, J.</span></span> <span> </span><span class="NLM_article-title">Retinoic-Acid-Orphan-Receptor-C Inhibition Suppresses Th17 Cells and Induces Thymic Aberrations</span>. <i>JCI Insight</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e91127</span> <span class="refDoi"> DOI: 10.1172/jci.insight.91127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1172%2Fjci.insight.91127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=28289717" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&author=C.+Guntermannauthor=A.+Piaiaauthor=M.-L.+Hamelauthor=D.+Theilauthor=T.+Rubic-Schneiderauthor=A.+del+Rio-Espinolaauthor=L.+Dongauthor=A.+Billichauthor=K.+Kaupmannauthor=J.+Dawsonauthor=K.+Hoegenauerauthor=D.+Orainauthor=S.+Hintermannauthor=R.+Stringerauthor=D.+D.+Patelauthor=A.+Doelemeyerauthor=M.+Deurinckauthor=J.+Sch%C3%BCmann&title=Retinoic-Acid-Orphan-Receptor-C+Inhibition+Suppresses+Th17+Cells+and+Induces+Thymic+Aberrations&doi=10.1172%2Fjci.insight.91127"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1172%2Fjci.insight.91127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci.insight.91127%26sid%3Dliteratum%253Aachs%26aulast%3DGuntermann%26aufirst%3DC.%26aulast%3DPiaia%26aufirst%3DA.%26aulast%3DHamel%26aufirst%3DM.-L.%26aulast%3DTheil%26aufirst%3DD.%26aulast%3DRubic-Schneider%26aufirst%3DT.%26aulast%3Ddel%2BRio-Espinola%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DBillich%26aufirst%3DA.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DHoegenauer%26aufirst%3DK.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DHintermann%26aufirst%3DS.%26aulast%3DStringer%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DDoelemeyer%26aufirst%3DA.%26aulast%3DDeurinck%26aufirst%3DM.%26aulast%3DSch%25C3%25BCmann%26aufirst%3DJ.%26atitle%3DRetinoic-Acid-Orphan-Receptor-C%2520Inhibition%2520Suppresses%2520Th17%2520Cells%2520and%2520Induces%2520Thymic%2520Aberrations%26jtitle%3DJCI%2520Insight%26date%3D2017%26volume%3D2%26doi%3D10.1172%2Fjci.insight.91127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':[],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xishan Wu, Hui Shen, Yan Zhang, Chao Wang, Qiu Li, Cheng Zhang, Xiaoxi Zhuang, Chenchang Li, Yudan Shi, Yanli Xing, Qiuping Xiang, Jinxin Xu, Donghai Wu, Jinsong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Yong Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Characterization of Benzimidazole Derivative XY123 as a Potent, Selective, and Orally Available RORγ Inverse Agonist. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8775-8797. <a href="https://doi.org/10.1021/acs.jmedchem.1c00763" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00763</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00763%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BCharacterization%252Bof%252BBenzimidazole%252BDerivative%252BXY123%252Bas%252Ba%252BPotent%25252C%252BSelective%25252C%252Band%252BOrally%252BAvailable%252BROR%2525CE%2525B3%252BInverse%252BAgonist%26aulast%3DWu%26aufirst%3DXishan%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D27042021%26date%3D14062021%26volume%3D64%26issue%3D12%26spage%3D8775%26epage%3D8797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng‐Jun  Chen</span>, <span class="hlFld-ContribAuthor ">Wei  Liang</span>, <span class="hlFld-ContribAuthor ">Zhuo  Chen</span>, <span class="hlFld-ContribAuthor ">Lin  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Phase‐Transfer Catalytic Strategy: Rapid Synthesis of Spiro‐Fused Heterocycles, Integrated with Four Pharmacophores‐Succinimide, Pyrrolidine, Oxindole, and Trifluoromethyl Group. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>2021 </em>
                                    (5)
                                     , 788-793. <a href="https://doi.org/10.1002/ejoc.202001409" title="DOI URL">https://doi.org/10.1002/ejoc.202001409</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202001409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202001409%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DPhase%2525E2%252580%252590Transfer%252BCatalytic%252BStrategy%25253A%252BRapid%252BSynthesis%252Bof%252BSpiro%2525E2%252580%252590Fused%252BHeterocycles%25252C%252BIntegrated%252Bwith%252BFour%252BPharmacophores%2525E2%252580%252590Succinimide%25252C%252BPyrrolidine%25252C%252BOxindole%25252C%252Band%252BTrifluoromethyl%252BGroup%26aulast%3DChen%26aufirst%3DZheng%25E2%2580%2590Jun%26date%3D2021%26date%3D2020%26volume%3D2021%26issue%3D5%26spage%3D788%26epage%3D793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Superposition of RORγt LBD X-ray cocrystal structures in the complex with cholesterol sulfate (stabilizer of coactivator binding; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NTI">5NTI</a>, carbons in white) and ester <b>2</b> at 2.30 Å resolution (displacer of co-activator binding; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O">6Q6O</a>, carbons in cyan/yellow) explaining the interplay of ligands, namely, W317, H479, and Y502 for helix 12 stabilization; water molecules are omitted for clarity; (A) overall view of the binding pocket illustrating the “business end” and the sterical clash of ligand <b>2</b> with W317 in the helix 12 on position (red circle); (B) different orientation highlighting the differences of the H479 H-bond for both ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Superposition of RORγt LBD X-ray cocrystal structures in the complex with ester <b>2</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O">6Q6O</a>, carbons in white) and imidazole <b>4</b> at 2.20 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7A">6Q7A</a>, carbons in cyan/yellow) illustrates a similar binding mode; water molecules are omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Protein cavity indicating positions of crystallographic waters from 19 X-ray structures revealing possible areas for polar ligand extension. Water molecules that form direct interactions with the protein are mostly conserved but positions do not perfectly overlap between individual structures because of protein flexibility. Networks of water molecules without direct protein contact are more variable and adapt to the ligand. Compound <b>4</b> is shown in orange, and compound <b>9</b> (in cyan) was designed assuming that variable waters can be displaced.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. RORγt LBD X-ray cocrystal structure in the complex with carboxylic acid <b>9</b> at 2.30 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H">6Q7H</a>); selected water molecules are shown as white spheres; in addition to the interactions described for compound <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), the carboxylic acid group of compound <b>9</b> extends into the more hydrophilic part of the LBD and is embedded in a water-mediated hydrogen bond network between CO-F377 and CO-H323.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Box plots illustrate that TDI <i>k</i><sub>obs</sub> values are higher for (A) 2-furyl-substituted imidazoles and (B) 2-furyl-5-trifluoromethyl-substituted imidazoles; values inside the box are between the upper (Q3) and the lower (Q1) quartile; the white line within the box indicates the median value; values in between whiskers are within 1.5 times the interquartile range; outliers are defined as compounds outside 1.5 times the interquartile range and are depicted as separate dots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of the X-ray cocrystal structure of compound <b>9</b> (in cyan) and the docking pose for compound <b>14</b> (brown). Docking was performed using flexible water positioning.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Compound <b>14</b> dose-dependently reduces mBSA-induced DTH responses in female Lewis rats; the mean (+SEM) of the swelling ratios between antigen-challenged and vehicle-injected ears is shown (<i>n</i> = 5); (B) compound <b>14</b> dose-dependently reduces mBSA-induced frequencies of IL-17-producing draining lymph node cells; individual data and mean ± SEM are shown (<i>n</i> = 5); data are a representative from two independent studies; means were compared using ANOVA followed by Dunnett’s test (*<i>p</i> < 0.05, **<i>p</i> < 0.01).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Isopropylester <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) furan-2-carbonyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 40 °C, 18 h; (ii) NaOH (1 M, aq), MeOH, 40 °C, 2 h; (b) (i) ethyl 2-bromopropanoate, NaH, THF, 70 °C, 6 h; (ii) LiOH, H<sub>2</sub>O, MeOH, 70 °C, 18 h; (iii) SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, rt, 1.5 h, then <i>i</i>-PrOH, rt, 18 h.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Imidazole <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) 3-bromo-1-phenylbutan-2-one, NaHCO<sub>3</sub>, <i>i</i>-PrOH, 83 °C, 18 h; (ii) furan-2-carbonyl chloride, 1,2-dichloroethane, 90 °C, 3 d; (b) NH<sub>4</sub>OAc, AcOH, 130 °C, 2 h.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General <i>N</i>-Aryl Imidazole Synthesis via Amidine <b>A1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) AlMe<sub>3</sub> or AlCl<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub> or toluene; (b) NaHCO<sub>3</sub>, Na<sub>2</sub>SO<sub>4,</sub><i>i</i>-PrOH; (c) Zn(CN)<sub>2</sub>, PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub>, DMF, microwave.</p></p></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Imidazoles <b>8</b>, <b>9</b>, <b>12</b>, and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amine, various conditions for Buchwald–Hartwig cross-coupling, see experimental; (b) (i) NBS or <i>N</i>-iodosuccinimide (NIS), acetonitrile; (ii) trimethylboroxine, Cs<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane; (c) LiOH, THF, H<sub>2</sub>O.</p></p></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Amide <b>10</b> and Tetrazole <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,1′-carbonyldiimidazole (CDI), NH<sub>3</sub>, (7 M in MeOH), THF, rt, 2 h; (b) (i) TFAA, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 1 h; (ii) Me<sub>3</sub>SnN<sub>3</sub>, toluene, 100 °C, 2 d.</p></p></figure><figure data-id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Small molecule X-ray structure of compound <b>14</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/medium/jm9b01291_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Preparation of Compound <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01291/20191205/images/large/jm9b01291_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01291&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) AlMe<sub>3</sub>, toluene, 0 °C to rt, 18 h; (b) NaHCO<sub>3</sub>, <i>i</i>-PrOH, 90 °C, 48 h; (c) optical resolution by preparative chiral HPLC; (d) DMA, microwave, 250 °C, 30 min; (e) Cs<sub>2</sub>CO<sub>3</sub>, RuPhos, RuPhos precatalyst, toluene, 100 °C, 3 h; (f) NBS, CH<sub>3</sub>CN, rt, 1 h; (g) (i) trimethylboroxine, K<sub>2</sub>CO<sub>3</sub>, Pd(Ph<sub>3</sub>P)<sub>4</sub>, DMF, 100 °C, 24 h; (ii) NaOH, MeOH, rt, 18 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i47">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12404" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12404" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadokoro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepelley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafaille, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cua, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span> <span> </span><span class="NLM_article-title">The Orphan Nuclear Receptor RORγt Directs the Differentiation Program of Proinflammatory IL-17+ T Helper Cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.07.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.cell.2006.07.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=16990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=1121-1133&author=I.+I.+Ivanovauthor=B.+S.+McKenzieauthor=L.+Zhouauthor=C.+E.+Tadokoroauthor=A.+Lepelleyauthor=J.+J.+Lafailleauthor=D.+J.+Cuaauthor=D.+R.+Littman&title=The+Orphan+Nuclear+Receptor+ROR%CE%B3t+Directs+the+Differentiation+Program+of+Proinflammatory+IL-17%2B+T+Helper+Cells&doi=10.1016%2Fj.cell.2006.07.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The orphan nuclear receptor RORγt directs the differentiation program of proinflammatory IL-17+ T helper cells</span></div><div class="casAuthors">Ivanov, Ivaylo I.; McKenzie, Brent S.; Zhou, Liang; Tadokoro, Carlos E.; Lepelley, Alice; Lafaille, Juan J.; Cua, Daniel J.; Littman, Dan R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1133</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IL-17-producing T lymphocytes have been recently shown to comprise a distinct lineage of proinflammatory T helper cells, termed Th17 cells, that are major contributors to autoimmune disease.  The authors show here that the orphan nuclear receptor RORγt is the key transcription factor that orchestrates the differentiation of this effector cell lineage.  RORγt induces transcription of the genes encoding IL-17 and the related cytokine IL-17F in naive CD4+ T helper cells and is required for their expression in response to IL-6 and TGF-β, the cytokines known to induce IL-17.  Th17 cells are constitutively present throughout the intestinal lamina propria, express RORγt, and are absent in mice deficient for RORγt or IL-6.  Mice with RORγt-deficient T cells have attenuated autoimmune disease and lack tissue-infiltrating Th17 cells.  Together, these studies suggest that RORγt is a key regulator of immune homeostasis and highlight its potential as a therapeutic target in inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsJd6RyFqWrVg90H21EOLACvtfcHk0lh1znRFkf2WfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCnsrrE&md5=4f5693603c30dca4f6a2d0994aeaf9b0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.07.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.07.035%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DI.%2BI.%26aulast%3DMcKenzie%26aufirst%3DB.%2BS.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DTadokoro%26aufirst%3DC.%2BE.%26aulast%3DLepelley%26aufirst%3DA.%26aulast%3DLafaille%26aufirst%3DJ.%2BJ.%26aulast%3DCua%26aufirst%3DD.%2BJ.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26atitle%3DThe%2520Orphan%2520Nuclear%2520Receptor%2520ROR%25CE%25B3t%2520Directs%2520the%2520Differentiation%2520Program%2520of%2520Proinflammatory%2520IL-17%252B%2520T%2520Helper%2520Cells%26jtitle%3DCell%26date%3D2006%26volume%3D126%26spage%3D1121%26epage%3D1133%26doi%3D10.1016%2Fj.cell.2006.07.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappu, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurieva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akimzhanov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panopoulos, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schluns, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetten, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span> <span> </span><span class="NLM_article-title">T Helper 17 Lineage Differentiation Is Programmed by Orphan Nuclear Receptors RORα and RORγ</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2007.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.immuni.2007.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=18164222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=29-39&author=X.+O.+Yangauthor=B.+P.+Pappuauthor=R.+Nurievaauthor=A.+Akimzhanovauthor=H.+S.+Kangauthor=Y.+Chungauthor=L.+Maauthor=B.+Shahauthor=A.+D.+Panopoulosauthor=K.+S.+Schlunsauthor=S.+S.+Watowichauthor=Q.+Tianauthor=A.+M.+Jettenauthor=C.+Dong&title=T+Helper+17+Lineage+Differentiation+Is+Programmed+by+Orphan+Nuclear+Receptors+ROR%CE%B1+and+ROR%CE%B3&doi=10.1016%2Fj.immuni.2007.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">T helper 17 lineage differentiation is programmed by orphan nuclear receptors RORα and RORγ</span></div><div class="casAuthors">Yang, Xuexian O.; Pappu, Bhanu P.; Nurieva, Roza; Akimzhanov, Askar; Kang, Hong Soon; Chung, Yeonseok; Ma, Li; Shah, Bhavin; Panopoulos, Athanasia D.; Schluns, Kimberly S.; Watowich, Stephanie S.; Tian, Qiang; Jetten, Anton M.; Dong, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-39</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">T cell functional differentiation is mediated by lineage-specific transcription factors.  T helper 17 (Th17) has been recently identified as a distinct Th lineage mediating tissue inflammation.  Retinoic acid receptor-related orphan receptor γ (RORγ) was shown to regulate Th17 differentiation; RORγ deficiency, however, did not completely abolish Th17 cytokine expression.  Here, we report Th17 cells highly expressed another related nuclear receptor, RORα, induced by transforming growth factor-β and interleukin-6 (IL-6), which is dependent on signal transducer and activator of transcription 3.  Overexpression of RORα promoted Th17 differentiation, possibly through the conserved noncoding sequence 2 in II17-II17f locus.  RORα deficiency resulted in reduced IL-17 expression in vitro and in vivo.  Furthermore, RORα and RORγ coexpression synergistically led to greater Th17 differentiation.  Double deficiencies in RORα and RORγ globally impaired Th17 generation and completely protected mice against exptl. autoimmune encephalomyelitis.  Therefore, Th17 differentiation is directed by two lineage-specific nuclear receptors, RORα and RORγ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBQRyY1XQpJbVg90H21EOLACvtfcHk0lh1znRFkf2WfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ontrs%253D&md5=436602649095b96d366c983addf526f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2007.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2007.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BO.%26aulast%3DPappu%26aufirst%3DB.%2BP.%26aulast%3DNurieva%26aufirst%3DR.%26aulast%3DAkimzhanov%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DH.%2BS.%26aulast%3DChung%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DShah%26aufirst%3DB.%26aulast%3DPanopoulos%26aufirst%3DA.%2BD.%26aulast%3DSchluns%26aufirst%3DK.%2BS.%26aulast%3DWatowich%26aufirst%3DS.%2BS.%26aulast%3DTian%26aufirst%3DQ.%26aulast%3DJetten%26aufirst%3DA.%2BM.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DT%2520Helper%252017%2520Lineage%2520Differentiation%2520Is%2520Programmed%2520by%2520Orphan%2520Nuclear%2520Receptors%2520ROR%25CE%25B1%2520and%2520ROR%25CE%25B3%26jtitle%3DImmunity%26date%3D2008%26volume%3D28%26spage%3D29%26epage%3D39%26doi%3D10.1016%2Fj.immuni.2007.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oukka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">IL-17 and Th17 Cells</span>. <i>Annu. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1146/annurev.immunol.021908.132710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1146%2Fannurev.immunol.021908.132710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=19132915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFSltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=485-517&author=T.+Kornauthor=E.+Bettelliauthor=M.+Oukkaauthor=V.+K.+Kuchroo&title=IL-17+and+Th17+Cells&doi=10.1146%2Fannurev.immunol.021908.132710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">IL-17 and Th17 cells</span></div><div class="casAuthors">Korn, Thomas; Bettelli, Estelle; Oukka, Mohamed; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">485-517</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  CD4+ T cells, upon activation and expansion, develop into different T helper cell subsets with different cytokine profiles and distinct effector functions.  Until recently, T cells were divided into Th1 or Th2 cells, depending on the cytokines they produce.  A third subset of IL-17-producing effector T helper cells, called Th17 cells, has now been discovered and characterized.  Here, the authors summarize the current information on the differentiation and effector functions of the Th17 lineage.  Th17 cells produce IL-17, IL-17F, and IL-22, thereby inducing a massive tissue reaction owing to the broad distribution of the IL-17 and IL-22 receptors.  Th17 cells also secrete IL-21 to communicate with the cells of the immune system.  The differentiation factors (TGF-β plus IL-6 or IL-21), the growth and stabilization factor (IL-23), and the transcription factors (STAT3, RORγt, and RORα) involved in the development of Th17 cells have just been identified.  The participation of TGF-β in the differentiation of Th17 cells places the Th17 lineage in close relation with CD4+CD25+Foxp3+ regulatory T cells (Tregs), as TGF-β also induces differentiation of native T cells into Foxp3+ Tregs in the peripheral immune compartment.  The investigation of the differentiation, effector function, and regulation of Th17 cells has opened up a new framework for understanding T cell differentiation.  Furthermore, the importance of Th17 cells is now appreciated in clearing pathogens during host defense reactions and in inducing tissue inflammation in autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAJvCEP9y1fLVg90H21EOLACvtfcHk0lh1znRFkf2WfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFSltrY%253D&md5=fffce537e2b70751fd9113e25962419e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.immunol.021908.132710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.immunol.021908.132710%26sid%3Dliteratum%253Aachs%26aulast%3DKorn%26aufirst%3DT.%26aulast%3DBettelli%26aufirst%3DE.%26aulast%3DOukka%26aufirst%3DM.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DIL-17%2520and%2520Th17%2520Cells%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D27%26spage%3D485%26epage%3D517%26doi%3D10.1146%2Fannurev.immunol.021908.132710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">Th17 Cell Pathway in Human Immunity: Lessons from Genetics and Therapeutic Interventions</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1040</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2015.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.immuni.2015.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=26682981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVajtb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=1040-1051&author=D.+D.+Patelauthor=V.+K.+Kuchroo&title=Th17+Cell+Pathway+in+Human+Immunity%3A+Lessons+from+Genetics+and+Therapeutic+Interventions&doi=10.1016%2Fj.immuni.2015.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Th17 Cell Pathway in Human Immunity: Lessons from Genetics and Therapeutic Interventions</span></div><div class="casAuthors">Patel, Dhavalkumar D.; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1040-1051</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The T helper 17 (Th17) cell pathway has been linked by genome-wide assocn. studies to multiple autoimmune diseases.  Identification of the genetic causes of primary immunodeficiency diseases revealed that Th17 cells are also crit. in host immunity to mucocutaneous candida infections and Staphylococcus aureus.  Therapeutic interventions with inhibitors of the different components of the pathway such as interleukin-12 (IL-12), IL-23, IL-17A, and IL-17RA have variably beneficial effects in psoriasis, Crohn's disease, rheumatoid arthritis, psoriatic arthritis, ankylosing spondylitis, non-infectious uveitis, and multiple sclerosis.  Thus, whereas Th17 cells are protective against Candida albicans and to a lesser degree Staphylococcus aureus, they are pathogenic in many autoimmune diseases.  Here, we compare and contrast the effects of human genetic mutations of and therapeutic interventions targeted at Th17 cell mols.  We discuss that although there are similarities when Th17 cell pathway mols. are modulated, each mol. has unique non-Th17 cell features that lead to different functional outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb--L2SA4SpLVg90H21EOLACvtfcHk0lhTyPUqiy9oiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVajtb7I&md5=5af767afe2a213dafa7c07f2b4271242</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2015.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2015.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DTh17%2520Cell%2520Pathway%2520in%2520Human%2520Immunity%253A%2520Lessons%2520from%2520Genetics%2520and%2520Therapeutic%2520Interventions%26jtitle%3DImmunity%26date%3D2015%26volume%3D43%26spage%3D1040%26epage%3D1051%26doi%3D10.1016%2Fj.immuni.2015.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jetten, A. M.</span></span> <span> </span><span class="NLM_article-title">Retinoid-Related Orphan Receptors (RORs): Critical Roles in Development, Immunity, Circadian Rhythm, and Cellular Metabolism</span>. <i>Nucl. Recept. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">nrs.07003</span>, <span class="refDoi"> DOI: 10.1621/nrs.07003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1621%2Fnrs.07003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=nrs.07003&author=A.+M.+Jetten&title=Retinoid-Related+Orphan+Receptors+%28RORs%29%3A+Critical+Roles+in+Development%2C+Immunity%2C+Circadian+Rhythm%2C+and+Cellular+Metabolism&doi=10.1621%2Fnrs.07003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1621%2Fnrs.07003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1621%252Fnrs.07003%26sid%3Dliteratum%253Aachs%26aulast%3DJetten%26aufirst%3DA.%2BM.%26atitle%3DRetinoid-Related%2520Orphan%2520Receptors%2520%2528RORs%2529%253A%2520Critical%2520Roles%2520in%2520Development%252C%2520Immunity%252C%2520Circadian%2520Rhythm%252C%2520and%2520Cellular%2520Metabolism%26jtitle%3DNucl.%2520Recept.%2520Signaling%26date%3D2009%26volume%3D7%26spage%3Dnrs.07003%26doi%3D10.1621%2Fnrs.07003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachchidanand</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, R. C.</span></span> <span> </span><span class="NLM_article-title">Combating Autoimmune Diseases With Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc) Inhibitors: Hits and Misses</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10976</span>– <span class="NLM_lpage">10995</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00588</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00588" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10976-10995&author=V.+B.+Pandyaauthor=S.+Kumarauthor=+Sachchidanandauthor=R.+Sharmaauthor=R.+C.+Desai&title=Combating+Autoimmune+Diseases+With+Retinoic+Acid+Receptor-Related+Orphan+Receptor-%CE%B3+%28ROR%CE%B3+or+RORc%29+Inhibitors%3A+Hits+and+Misses&doi=10.1021%2Facs.jmedchem.8b00588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Autoimmune Diseases With Retinoic Acid Receptor-Related Orphan Receptor-γ (RORγ or RORc) Inhibitors: Hits and Misses</span></div><div class="casAuthors">Pandya, Vrajesh B.; Kumar, Sanjay; Sachchidanand; Sharma, Rajiv; Desai, Ranjit C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10976-10995</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclear receptor retinoic acid receptor-related orphan receptor gamma (RORγ or RORc) is a key transcription factor for the prodn. of pro-inflammatory cytokines implicated in the pathogenesis of autoimmune diseases.  Recently, small mol. inhibitors of RORc drew the enormous attention of the research community worldwide as a possible therapy for autoimmune diseases, mediated by the IL-17 cytokine.  With the clin. proof-of-concept inferred from a small mol. inhibitor VTP-43742 for psoriasis and recent inflow of several RORc inhibitors into the clinic for therapeutic interventions in autoimmune diseases, this field continues to evolve.  This review briefly summarizes the RORc inhibitors disclosed in the literature and discusses the progress made by these inhibitors in combating autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKGqZl3bPbtLVg90H21EOLACvtfcHk0lhTyPUqiy9oiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWrtL7E&md5=4113d22f73ab2911535eef053801994d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00588%26sid%3Dliteratum%253Aachs%26aulast%3DPandya%26aufirst%3DV.%2BB.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DSachchidanand%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DDesai%26aufirst%3DR.%2BC.%26atitle%3DCombating%2520Autoimmune%2520Diseases%2520With%2520Retinoic%2520Acid%2520Receptor-Related%2520Orphan%2520Receptor-%25CE%25B3%2520%2528ROR%25CE%25B3%2520or%2520RORc%2529%2520Inhibitors%253A%2520Hits%2520and%2520Misses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10976%26epage%3D10995%26doi%3D10.1021%2Facs.jmedchem.8b00588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosef, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baloglu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundrud, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretschmer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafler, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchroo, V. K.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule RORγt Antagonists Inhibit T Helper 17 Cell Transcriptional Network by Divergent Mechanisms</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2014.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.immuni.2014.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=24745332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVKltr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2014&pages=477-489&author=S.+Xiaoauthor=N.+Yosefauthor=J.+Yangauthor=Y.+Wangauthor=L.+Zhouauthor=C.+Zhuauthor=C.+Wuauthor=E.+Balogluauthor=D.+Schmidtauthor=R.+Rameshauthor=M.+Loberaauthor=M.+S.+Sundrudauthor=P.-Y.+Tsaiauthor=Z.+Xiangauthor=J.+Wangauthor=Y.+Xuauthor=X.+Linauthor=K.+Kretschmerauthor=P.+B.+Rahlauthor=R.+A.+Youngauthor=Z.+Zhongauthor=D.+A.+Haflerauthor=A.+Regevauthor=S.+Ghoshauthor=A.+Marsonauthor=V.+K.+Kuchroo&title=Small-Molecule+ROR%CE%B3t+Antagonists+Inhibit+T+Helper+17+Cell+Transcriptional+Network+by+Divergent+Mechanisms&doi=10.1016%2Fj.immuni.2014.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule RORγt Antagonists Inhibit T Helper 17 Cell Transcriptional Network by Divergent Mechanisms</span></div><div class="casAuthors">Xiao, Sheng; Yosef, Nir; Yang, Jianfei; Wang, Yonghui; Zhou, Ling; Zhu, Chen; Wu, Chuan; Baloglu, Erkan; Schmidt, Darby; Ramesh, Radha; Lobera, Mercedes; Sundrud, Mark S.; Tsai, Pei-Yun; Xiang, Zhijun; Wang, Jinsong; Xu, Yan; Lin, Xichen; Kretschmer, Karsten; Rahl, Peter B.; Young, Richard A.; Zhong, Zhong; Hafler, David A.; Regev, Aviv; Ghosh, Shomir; Marson, Alexander; Kuchroo, Vijay K.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-489</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We identified three retinoid-related orphan receptor gamma t (RORγt)-specific inhibitors that suppress T helper 17 (Th17) cell responses, including Th17-cell-mediated autoimmune disease.  We systemically characterized RORγt binding in the presence and absence of drugs with corresponding whole-genome transcriptome sequencing.  RORγt acts as a direct activator of Th17 cell signature genes and a direct repressor of signature genes from other T cell lineages; its strongest transcriptional effects are on cis-regulatory sites contg. the RORα binding motif.  RORγt is central in a densely interconnected regulatory network that shapes the balance of T cell differentiation.  Here, the three inhibitors modulated the RORγt-dependent transcriptional network to varying extents and through distinct mechanisms.  Whereas one inhibitor displaced RORγt from its target loci, the other two inhibitors affected transcription predominantly without removing DNA binding.  Our work illustrates the power of a system-scale anal. of transcriptional regulation to characterize potential therapeutic compds. that inhibit pathogenic Th17 cells and suppress autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo23p8R83CDDbVg90H21EOLACvtfcHk0ljH23zBi__IJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVKltr0%253D&md5=940074c565211bc00832179574e5231a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2014.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2014.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DYosef%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DRamesh%26aufirst%3DR.%26aulast%3DLobera%26aufirst%3DM.%26aulast%3DSundrud%26aufirst%3DM.%2BS.%26aulast%3DTsai%26aufirst%3DP.-Y.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DKretschmer%26aufirst%3DK.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DZhong%26aufirst%3DZ.%26aulast%3DHafler%26aufirst%3DD.%2BA.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DMarson%26aufirst%3DA.%26aulast%3DKuchroo%26aufirst%3DV.%2BK.%26atitle%3DSmall-Molecule%2520ROR%25CE%25B3t%2520Antagonists%2520Inhibit%2520T%2520Helper%252017%2520Cell%2520Transcriptional%2520Network%2520by%2520Divergent%2520Mechanisms%26jtitle%3DImmunity%26date%3D2014%26volume%3D40%26spage%3D477%26epage%3D489%26doi%3D10.1016%2Fj.immuni.2014.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skepner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trocha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baloglu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orband-Miller, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudjelal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundrud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic Inhibition of RORγt Regulates Th17 Signature Gene Expression and Suppresses Cutaneous Inflammation In Vivo</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">2564</span>– <span class="NLM_lpage">2575</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1302190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.4049%2Fjimmunol.1302190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=24516202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslGls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2014&pages=2564-2575&author=J.+Skepnerauthor=R.+Rameshauthor=M.+Trochaauthor=D.+Schmidtauthor=E.+Balogluauthor=M.+Loberaauthor=T.+Carlsonauthor=J.+Hillauthor=L.+A.+Orband-Millerauthor=A.+Barnesauthor=M.+Boudjelalauthor=M.+Sundrudauthor=S.+Ghoshauthor=J.+Yang&title=Pharmacologic+Inhibition+of+ROR%CE%B3t+Regulates+Th17+Signature+Gene+Expression+and+Suppresses+Cutaneous+Inflammation+In+Vivo&doi=10.4049%2Fjimmunol.1302190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of RORγt Regulates Th17 Signature Gene Expression and Suppresses Cutaneous Inflammation In Vivo</span></div><div class="casAuthors">Skepner, Jill; Ramesh, Radha; Trocha, Mark; Schmidt, Darby; Baloglu, Erkan; Lobera, Mercedes; Carlson, Thaddeus; Hill, Jonathan; Orband-Miller, Lisa A.; Barnes, Ashley; Boudjelal, Mohamed; Sundrud, Mark; Ghosh, Shomir; Yang, Jianfei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2564-2575</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">IL-17-producing CD4+Th17 cells, CD8+Tc17 cells, and γδ T cells play crit. roles in the pathogenesis of autoimmune psoriasis.  RORγt is required for the differentiation of Th17 cells and expression of IL-17.  In this article, we describe a novel, potent, and selective RORγt inverse agonist (TMP778), and its inactive diastereomer (TMP776).  This chem., for the first time to our knowledge, provides a unique and powerful set of tools to probe RORγt-dependent functions.  TMP778, but not TMP776, blocked human Th17 and Tc17 cell differentiation and also acutely modulated IL-17A prodn. and inflammatory Th17-signature gene expression (Il17a, Il17f, Il22, Il26, Ccr6, and Il23) in mature human Th17 effector/memory T cells.  In addn., TMP778, but not TMP776, inhibited IL-17A prodn. in both human and mouse γδ T cells.  IL-23-induced IL-17A prodn. was also blocked by TMP778 treatment.  In vivo targeting of RORγt in mice via TMP778 administration reduced imiquimod-induced psoriasis-like cutaneous inflammation.  Further, TMP778 selectively regulated Th17-signature gene expression in mononuclear cells isolated from both the blood and affected skin of psoriasis patients.  In summary, to our knowledge, we are the first to demonstrate that RORγt inverse agonists: (1) inhibit Tc17 cell differentiation, as well as IL-17 prodn. by γδ T cells and CD8+ Tc17 cells; (2) block imiquimod-induced cutaneous inflammation; (3) inhibit Th17 signature gene expression by cells isolated from psoriatic patient samples; and (4) block IL-23-induced IL-17A expression.  Thus, RORγt is a tractable drug target for the treatment of cutaneous inflammatory disorders, which may afford addnl. therapeutic benefit over existing modalities that target only IL-17A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu4CFK2lBK6bVg90H21EOLACvtfcHk0ljH23zBi__IJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslGls7w%253D&md5=d72bc88b09a34874cf7b89f993977d35</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1302190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1302190%26sid%3Dliteratum%253Aachs%26aulast%3DSkepner%26aufirst%3DJ.%26aulast%3DRamesh%26aufirst%3DR.%26aulast%3DTrocha%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DD.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DLobera%26aufirst%3DM.%26aulast%3DCarlson%26aufirst%3DT.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DOrband-Miller%26aufirst%3DL.%2BA.%26aulast%3DBarnes%26aufirst%3DA.%26aulast%3DBoudjelal%26aufirst%3DM.%26aulast%3DSundrud%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DJ.%26atitle%3DPharmacologic%2520Inhibition%2520of%2520ROR%25CE%25B3t%2520Regulates%2520Th17%2520Signature%2520Gene%2520Expression%2520and%2520Suppresses%2520Cutaneous%2520Inflammation%2520In%2520Vivo%26jtitle%3DJ.%2520Immunol.%26date%3D2014%26volume%3D192%26spage%3D2564%26epage%3D2575%26doi%3D10.4049%2Fjimmunol.1302190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guendisch, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecoeur, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riker, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Inhibition of RORγt Suppresses the Th17 Pathway and Alleviates Arthritis in Vivo</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0188391</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0188391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1371%2Fjournal.pone.0188391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=29155882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWrs7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=U.+Guendischauthor=J.+Weissauthor=F.+Ecoeurauthor=J.+C.+Rikerauthor=K.+Kaupmannauthor=J.+Kallenauthor=S.+Hintermannauthor=D.+Orainauthor=J.+Dawsonauthor=A.+Billichauthor=C.+Guntermann&title=Pharmacological+Inhibition+of+ROR%CE%B3t+Suppresses+the+Th17+Pathway+and+Alleviates+Arthritis+in+Vivo&doi=10.1371%2Fjournal.pone.0188391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of RORγt suppresses the Th17 pathway and alleviates arthritis in vivo</span></div><div class="casAuthors">Guendisch, Ulf; Weiss, Jessica; Ecoeur, Florence; Riker, Julia Christina; Kaupmann, Klemens; Kallen, Joerg; Hintermann, Samuel; Orain, David; Dawson, Janet; Billich, Andreas; Guntermann, Christine</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0188391/1-e0188391/25</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Retinoic acid receptor-related-orphan-receptor-C (RORγt) is the key transcription factor that is driving the differentiation of IL-17 producing T-helper 17 (Th17) cells that are implicated in the pathol. of various autoimmune and inflammatory diseases.  Based on the importance of RORγt in promoting Th17-driven pathol., there is considerable interest to develop lowmol.- wt. compds. with the aim of inhibiting the transcriptional activity of this nuclear hormone receptor.  In this article, we describe the in vitro and in vivo pharmacol. of a potent and selective small-mol.-wt. RORγt inverse agonist.  The compd. binds to the ligand binding domain (LBD) of RORγt leading to displacement of a co-activator peptide.  We show for the first time that a RORγt inverse agonist down-regulates permissive histone H3 acetylation and methylation at the IL17A and IL23R promoter regions, thereby providing insight into the transcriptional inhibition of RORγt-dependent genes.  Consistent with this, the compd. effectively reduced IL-17A prodn. by polarized human T-cells and γδT-cells and attenuated transcription of RORγt target genes.  The inhibitor showed good in vivo efficacy in an antigen-induced arthritis model in rats and reduced the frequencies of IL-17A producing cells in ex vivo recall assays.  In summary, we demonstrate that inhibiting RORγt by a low-mol.-wt. inhibitor results in efficient and selective blockade of the pro-inflammatory Th17/IL-17A pathway making it an attractive target for Th17- mediated disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGa7wQ5ixe8LVg90H21EOLACvtfcHk0ljH23zBi__IJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWrs7zO&md5=0bb658eef161b17729676ec3358a60d7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0188391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0188391%26sid%3Dliteratum%253Aachs%26aulast%3DGuendisch%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DJ.%26aulast%3DEcoeur%26aufirst%3DF.%26aulast%3DRiker%26aufirst%3DJ.%2BC.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DHintermann%26aufirst%3DS.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DBillich%26aufirst%3DA.%26aulast%3DGuntermann%26aufirst%3DC.%26atitle%3DPharmacological%2520Inhibition%2520of%2520ROR%25CE%25B3t%2520Suppresses%2520the%2520Th17%2520Pathway%2520and%2520Alleviates%2520Arthritis%2520in%2520Vivo%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0188391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kono, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yukawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokuhara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of [Cis-3-({(5R)-5-[(7-Fluoro-1,1-Dimethyl-2,3-Dihydro-1H-Inden-5-Yl)Carbamoyl]-2-Methoxy-7,8-Dihydro-1,6-Naphthyridin-6(5H)-Yl}carbonyl)Cyclobutyl]Acetic Acid (TAK-828F) as a Potent, Selective, and Orally Available Novel Retinoic Acid Receptor-Related Orphan Receptor γt Inverse Agonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2973</span>– <span class="NLM_lpage">2988</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00061</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00061" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvFWns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2973-2988&author=M.+Konoauthor=A.+Ochidaauthor=T.+Odaauthor=T.+Imadaauthor=Y.+Bannoauthor=N.+Tayaauthor=S.+Masadaauthor=T.+Kawamotoauthor=K.+Yonemoriauthor=Y.+Naraauthor=Y.+Fukaseauthor=T.+Yukawaauthor=H.+Tokuharaauthor=R.+Skeneauthor=B.-C.+Sangauthor=I.+D.+Hoffmanauthor=G.+P.+Snellauthor=K.+Ugaauthor=A.+Shibataauthor=K.+Igakiauthor=Y.+Nakamuraauthor=H.+Nakagawaauthor=N.+Tsuchimoriauthor=M.+Yamasakiauthor=J.+Shiraiauthor=S.+Yamamoto&title=Discovery+of+%5BCis-3-%28%7B%285R%29-5-%5B%287-Fluoro-1%2C1-Dimethyl-2%2C3-Dihydro-1H-Inden-5-Yl%29Carbamoyl%5D-2-Methoxy-7%2C8-Dihydro-1%2C6-Naphthyridin-6%285H%29-Yl%7Dcarbonyl%29Cyclobutyl%5DAcetic+Acid+%28TAK-828F%29+as+a+Potent%2C+Selective%2C+and+Orally+Available+Novel+Retinoic+Acid+Receptor-Related+Orphan+Receptor+%CE%B3t+Inverse+Agonist&doi=10.1021%2Facs.jmedchem.8b00061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of [cis-3-({(5R)-5-[(7-Fluoro-1,1-dimethyl-2,3-dihydro-1H-inden-5-yl)carbamoyl]-2-methoxy-7,8-dihydro-1,6-naphthyridin-6(5H)-yl}carbonyl)cyclobutyl]acetic Acid (TAK-828F) as a Potent, Selective, and Orally Available Novel Retinoic Acid Receptor-Related Orphan Receptor γt Inverse Agonist</span></div><div class="casAuthors">Kono, Mitsunori; Ochida, Atsuko; Oda, Tsuneo; Imada, Takashi; Banno, Yoshihiro; Taya, Naohiro; Masada, Shinichi; Kawamoto, Tetsuji; Yonemori, Kazuko; Nara, Yoshi; Fukase, Yoshiyuki; Yukawa, Tomoya; Tokuhara, Hidekazu; Skene, Robert; Sang, Bi-Ching; Hoffman, Isaac D.; Snell, Gyorgy P.; Uga, Keiko; Shibata, Akira; Igaki, Keiko; Nakamura, Yoshiki; Nakagawa, Hideyuki; Tsuchimori, Noboru; Yamasaki, Masashi; Shirai, Junya; Yamamoto, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2973-2988</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of tetrahydronaphthyridine derivs. as novel RORγt inverse agonists were designed and synthesized.  We reduced the lipophilicity of tetrahydroisoquinoline compd. 1 by replacement of the trimethylsilyl group and SBDD-guided scaffold exchange, which successfully afforded compd. 7 (VII) with a lower log D value and tolerable in vitro activity.  Consideration of LLE (lipophilic ligand efficiency) values in the subsequent optimization of the carboxylate tether led to the discovery of [cis-3-({(5R)-5-[(7-fluoro-1,1-dimethyl-2,3-dihydro-1H-inden-5-yl)carbamoyl]-2-methoxy-7,8-dihydro-1,6-naphthyridin-6(5H)-yl}carbonyl)cyclobutyl]acetic acid, TAK-828F (10), which showed potent RORγt inverse agonistic activity, excellent selectivity against other ROR isoforms and nuclear receptors, and a good pharmacokinetic profile.  In animal studies, oral administration of compd. 10 exhibited robust and dose-dependent inhibition of IL-17A cytokine expression in a mouse IL23-induced gene expression assay.  Furthermore, development of clin. symptoms in a mouse exptl. autoimmune encephalomyelitis model was significantly reduced.  Compd. 10 was selected as a clin. compd. for the treatment of Th17-driven autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqraNPFfTkC1rVg90H21EOLACvtfcHk0ljnrFEZBmgUqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvFWns7Y%253D&md5=966c20bbdf825d1ff6eb91c5950a33c8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00061%26sid%3Dliteratum%253Aachs%26aulast%3DKono%26aufirst%3DM.%26aulast%3DOchida%26aufirst%3DA.%26aulast%3DOda%26aufirst%3DT.%26aulast%3DImada%26aufirst%3DT.%26aulast%3DBanno%26aufirst%3DY.%26aulast%3DTaya%26aufirst%3DN.%26aulast%3DMasada%26aufirst%3DS.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DYonemori%26aufirst%3DK.%26aulast%3DNara%26aufirst%3DY.%26aulast%3DFukase%26aufirst%3DY.%26aulast%3DYukawa%26aufirst%3DT.%26aulast%3DTokuhara%26aufirst%3DH.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DSang%26aufirst%3DB.-C.%26aulast%3DHoffman%26aufirst%3DI.%2BD.%26aulast%3DSnell%26aufirst%3DG.%2BP.%26aulast%3DUga%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DA.%26aulast%3DIgaki%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DTsuchimori%26aufirst%3DN.%26aulast%3DYamasaki%26aufirst%3DM.%26aulast%3DShirai%26aufirst%3DJ.%26aulast%3DYamamoto%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520%255BCis-3-%2528%257B%25285R%2529-5-%255B%25287-Fluoro-1%252C1-Dimethyl-2%252C3-Dihydro-1H-Inden-5-Yl%2529Carbamoyl%255D-2-Methoxy-7%252C8-Dihydro-1%252C6-Naphthyridin-6%25285H%2529-Yl%257Dcarbonyl%2529Cyclobutyl%255DAcetic%2520Acid%2520%2528TAK-828F%2529%2520as%2520a%2520Potent%252C%2520Selective%252C%2520and%2520Orally%2520Available%2520Novel%2520Retinoic%2520Acid%2520Receptor-Related%2520Orphan%2520Receptor%2520%25CE%25B3t%2520Inverse%2520Agonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2973%26epage%3D2988%26doi%3D10.1021%2Facs.jmedchem.8b00061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carcache, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulpetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vangrevelinghe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegenauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span> <span> </span><span class="NLM_article-title">Optimizing a Weakly Binding Fragment into a Potent RORγt Inverse Agonist with Efficacy in an in Vivo Inflammation Model</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6724</span>– <span class="NLM_lpage">6735</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00529</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00529" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht12jsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6724-6735&author=D.+A.+Carcacheauthor=A.+Vulpettiauthor=J.+Kallenauthor=H.+Mattesauthor=D.+Orainauthor=R.+Stringerauthor=E.+Vangrevelingheauthor=R.+M.+Wolfauthor=K.+Kaupmannauthor=J.+Ottlauthor=J.+Dawsonauthor=N.+G.+Cookeauthor=K.+Hoegenauerauthor=A.+Billichauthor=J.+Wagnerauthor=C.+Guntermannauthor=S.+Hintermann&title=Optimizing+a+Weakly+Binding+Fragment+into+a+Potent+ROR%CE%B3t+Inverse+Agonist+with+Efficacy+in+an+in+Vivo+Inflammation+Model&doi=10.1021%2Facs.jmedchem.8b00529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing a Weakly Binding Fragment into a Potent RORγt Inverse Agonist with Efficacy in an in Vivo Inflammation Model</span></div><div class="casAuthors">Carcache, David A.; Vulpetti, Anna; Kallen, Joerg; Mattes, Henri; Orain, David; Stringer, Rowan; Vangrevelinghe, Eric; Wolf, Romain M.; Kaupmann, Klemens; Ottl, Johannes; Dawson, Janet; Cooke, Nigel G.; Hoegenauer, Klemens; Billich, Andreas; Wagner, Juergen; Guntermann, Christine; Hintermann, Samuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6724-6735</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor RORγt is an attractive drug-target due to its role in the differentiation of IL-17 producing Th17 cells that play a crit. role in the etiopathol. of several autoimmune diseases.  Identification of starting points for RORγt inverse agonists with good properties has been a challenge.  We report the identification of a fragment hit and its conversion into a potent inverse agonist through fragment optimization, growing and merging efforts.  Further anal. of the binding mode revealed that inverse agonism was achieved by an unusual mechanism.  In contrast to other reported inverse agonists, there is no direct interaction or displacement of helix 12 obsd. in the crystal structure.  Nevertheless, compd. 9 proved to be efficacious in a delayed-type hypersensitivity (DTH) inflammation model in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgNfQTVtmUt7Vg90H21EOLACvtfcHk0ljnrFEZBmgUqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht12jsbjK&md5=6032c6b6a85377ea247a92314736db06</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00529%26sid%3Dliteratum%253Aachs%26aulast%3DCarcache%26aufirst%3DD.%2BA.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DMattes%26aufirst%3DH.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DStringer%26aufirst%3DR.%26aulast%3DVangrevelinghe%26aufirst%3DE.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DCooke%26aufirst%3DN.%2BG.%26aulast%3DHoegenauer%26aufirst%3DK.%26aulast%3DBillich%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DJ.%26aulast%3DGuntermann%26aufirst%3DC.%26aulast%3DHintermann%26aufirst%3DS.%26atitle%3DOptimizing%2520a%2520Weakly%2520Binding%2520Fragment%2520into%2520a%2520Potent%2520ROR%25CE%25B3t%2520Inverse%2520Agonist%2520with%2520Efficacy%2520in%2520an%2520in%2520Vivo%2520Inflammation%2520Model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6724%26epage%3D6735%26doi%3D10.1021%2Facs.jmedchem.8b00529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="note"><p class="first last">The terminology for compounds displaying such a functional response is heterogeneous. While many reports use receptor pharmacology vocabulary such as “inverse agonist” or “antagonist”, this analogy is problematic as binding partners to nuclear hormone receptors can be co-activators as well as co-repressors.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izaac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Be, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arista, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegenauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span> <span> </span><span class="NLM_article-title">Structural States of RORγt: X-ray Elucidation of Molecular Mechanisms and Binding Interactions for Natural and Synthetic Compounds</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1021</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1002%2Fcmdc.201700278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=28590087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFamsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1014-1021&author=J.+Kallenauthor=A.+Izaacauthor=C.+Beauthor=L.+Aristaauthor=D.+Orainauthor=K.+Kaupmannauthor=C.+Guntermannauthor=K.+Hoegenauerauthor=S.+Hintermann&title=Structural+States+of+ROR%CE%B3t%3A+X-ray+Elucidation+of+Molecular+Mechanisms+and+Binding+Interactions+for+Natural+and+Synthetic+Compounds&doi=10.1002%2Fcmdc.201700278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural States of RORγt: X-ray Elucidation of Molecular Mechanisms and Binding Interactions for Natural and Synthetic Compounds</span></div><div class="casAuthors">Kallen, Joerg; Izaac, Aude; Be, Celine; Arista, Luca; Orain, David; Kaupmann, Klemens; Guntermann, Christine; Hoegenauer, Klemens; Hintermann, Samuel</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1014-1021</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The T-cell-specific retinoic acid receptor (RAR)-related orphan receptor-γ (RORγt) is a key transcription factor for the prodn. of pro-inflammatory Th17 cytokines, which are implicated in the pathogenesis of autoimmune diseases.  Over the years, several structurally diverse RORγt inverse agonists have been reported, but combining high potency and good physicochem. properties has remained a challenging task.  We recently reported a new series of inverse agonists based on an imidazopyridine core with good physicochem. properties and excellent selectivity.  Herein we report eight new X-ray crystal structures for different classes of natural and synthetic compds., including examples selected from the patent literature.  Anal. of their resp. binding modes revealed insight into the mol. mechanisms that lead to agonism, antagonism, or inverse agonism.  We report new mol. mechanisms for RORγt agonism and propose a sepn. of the inverse agonists into two classes: those that act via steric clash and those that act via other mechanisms (for the latter, co-crystn. with a co-activator peptide and helix 12 in the agonist position is still possible).  For the non-steric clash inverse agonists, we propose a new mechanism ("water trapping") which can be combined with other mechanisms (e.g., close contacts with H479).  In addn., we compare the interactions made for selected compds. in the "back pocket" near S404 and in the "sulfate pocket" near R364 and R367.  Taken together, these new mechanistic insights should prove useful for the design and optimization of further RORγt modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2bJdGLB1okrVg90H21EOLACvtfcHk0livgxjHQD1TwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFamsrvK&md5=c49d7d9710f6c4f7baed7e7b207a0bf7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700278%26sid%3Dliteratum%253Aachs%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DIzaac%26aufirst%3DA.%26aulast%3DBe%26aufirst%3DC.%26aulast%3DArista%26aufirst%3DL.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DGuntermann%26aufirst%3DC.%26aulast%3DHoegenauer%26aufirst%3DK.%26aulast%3DHintermann%26aufirst%3DS.%26atitle%3DStructural%2520States%2520of%2520ROR%25CE%25B3t%253A%2520X-ray%2520Elucidation%2520of%2520Molecular%2520Mechanisms%2520and%2520Binding%2520Interactions%2520for%2520Natural%2520and%2520Synthetic%2520Compounds%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1014%26epage%3D1021%26doi%3D10.1002%2Fcmdc.201700278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Questel, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renault, E.</span></span> <span> </span><span class="NLM_article-title">The PKBHX Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4073</span>– <span class="NLM_lpage">4086</span>, <span class="refDoi"> DOI: 10.1021/jm801331y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801331y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4073-4086&author=C.+Laurenceauthor=K.+A.+Brameldauthor=J.+Gratonauthor=J.-Y.+Le+Questelauthor=E.+Renault&title=The+PKBHX+Database%3A+Toward+a+Better+Understanding+of+Hydrogen-Bond+Basicity+for+Medicinal+Chemists&doi=10.1021%2Fjm801331y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The pKBHX Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span></div><div class="casAuthors">Laurence, Christian; Brameld, Ken A.; Graton, Jerome; Le Questel, Jean-Yves; Renault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4073-4086</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The pKBHX database is presented, which corresponds to three main fields of data: hydrogen bond donor identification, thermodn. data and spectroscopic data.  The pKBHX scale of hydrogen donor basicity differs considerable from the pKBH+ scale of proton transfer basicity.  The hydrogen bond basicities of functional groups relevant to medicinal chem. and drug design is reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql9OhmMu2VILVg90H21EOLACvtfcHk0livgxjHQD1TwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D&md5=9ce4bea3e810752baba6ea5784b55ea0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm801331y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801331y%26sid%3Dliteratum%253Aachs%26aulast%3DLaurence%26aufirst%3DC.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DGraton%26aufirst%3DJ.%26aulast%3DLe%2BQuestel%26aufirst%3DJ.-Y.%26aulast%3DRenault%26aufirst%3DE.%26atitle%3DThe%2520PKBHX%2520Database%253A%2520Toward%2520a%2520Better%2520Understanding%2520of%2520Hydrogen-Bond%2520Basicity%2520for%2520Medicinal%2520Chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4073%26epage%3D4086%26doi%3D10.1021%2Fjm801331y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeuillet-Ritzler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briand, X.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Lipophilicity Measurement with Immobilized Artificial Membranes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2571</span>– <span class="NLM_lpage">2576</span>, <span class="refDoi"> DOI: 10.1021/jm049377w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049377w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhsF2isLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2571-2576&author=B.+Fallerauthor=H.+P.+Grimmauthor=F.+Loeuillet-Ritzlerauthor=S.+Arnoldauthor=X.+Briand&title=High-Throughput+Lipophilicity+Measurement+with+Immobilized+Artificial+Membranes&doi=10.1021%2Fjm049377w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Lipophilicity Measurement with Immobilized Artificial Membranes</span></div><div class="casAuthors">Faller, Bernard; Grimm, Hans Peter; Loeuillet-Ritzler, Frederique; Arnold, Sabine; Briand, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2571-2576</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report on a new, high-throughput assay designed to measure octanol/water partition coeffs. in early drug discovery.  The assay is carried out in 96-well microtiter plates and measures the diffusion of compds. between two aq. compartments sepd. by a thin octanol liq. layer. Octanol/water partition coeffs. are derived from the apparent permeability (Pa) values using a calibration curve.  The assay can measure partition coeffs. within the range -2 to + 8; thus, a dynamic range of 10 log units can be covered in one single run.  Unlike chromatog. methods, the technol. is not restricted to neutral and weakly basic compds., and, as no stationary phase is involved, the data can be strictly compared with values obtained from traditional methods such as shake-flask/HPLC or dual-phase potentiometric titrn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCq4XWfzWZ2bVg90H21EOLACvtfcHk0livgxjHQD1TwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhsF2isLc%253D&md5=dacee7abc2349eb0d595133dff427c48</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm049377w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049377w%26sid%3Dliteratum%253Aachs%26aulast%3DFaller%26aufirst%3DB.%26aulast%3DGrimm%26aufirst%3DH.%2BP.%26aulast%3DLoeuillet-Ritzler%26aufirst%3DF.%26aulast%3DArnold%26aufirst%3DS.%26aulast%3DBriand%26aufirst%3DX.%26atitle%3DHigh-Throughput%2520Lipophilicity%2520Measurement%2520with%2520Immobilized%2520Artificial%2520Membranes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2571%26epage%3D2576%26doi%3D10.1021%2Fjm049377w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic Efficiency as an Important Metric in Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+Efficiency+as+an+Important+Metric+in+Drug+Design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0ljyHEyRIYdPGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520Efficiency%2520as%2520an%2520Important%2520Metric%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. A.</span></span> <span> </span><span class="NLM_article-title">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">6</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1021/tx3003824</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx3003824" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=6-25&author=L.+A.+Peterson&title=Reactive+Metabolites+in+the+Biotransformation+of+Molecules+Containing+a+Furan+Ring&doi=10.1021%2Ftx3003824"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span></div><div class="casAuthors">Peterson, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-25</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many xenobiotics contg. a furan ring are toxic and/or carcinogenic.  The harmful effects of these compds. require furan ring oxidn.  This reaction generates an electrophilic intermediate.  Depending on the furan ring substituents, the intermediate is either an epoxide or a cis-enedione with more ring substitution favoring epoxide formation.  Either intermediate reacts with cellular nucleophiles such as protein or DNA to trigger toxicities.  The reactivity of the metabolite dets. which cellular nucleophiles are targeted.  The toxicity of a particular furan is also influenced by the presence of competing metabolic pathways or efficient detoxification routes.  GSH plays an important role in modulating the harmful effects of this class of compd. by reacting with the reactive metabolite.  However, this may not represent a detoxification step in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp4Dr5stLUSrVg90H21EOLACvtfcHk0ljyHEyRIYdPGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN&md5=fe3a280d0b7a99a436ec619abc1e1fa0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Ftx3003824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx3003824%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DReactive%2520Metabolites%2520in%2520the%2520Biotransformation%2520of%2520Molecules%2520Containing%2520a%2520Furan%2520Ring%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D6%26epage%3D25%26doi%3D10.1021%2Ftx3003824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caceres-Cortes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span> <span> </span><span class="NLM_article-title">Bioactivation of Substituted Thiophenes Including α-Chlorothiophene-Containing Compounds in Human Liver Microsomes</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">669</span>, <span class="refDoi"> DOI: 10.1021/tx100386z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx100386z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFOqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=663-669&author=W.+Chenauthor=J.+Caceres-Cortesauthor=H.+Zhangauthor=D.+Zhangauthor=W.+G.+Humphreysauthor=J.+Gan&title=Bioactivation+of+Substituted+Thiophenes+Including+%CE%B1-Chlorothiophene-Containing+Compounds+in+Human+Liver+Microsomes&doi=10.1021%2Ftx100386z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivation of Substituted Thiophenes Including α-Chlorothiophene-Containing Compounds in Human Liver Microsomes</span></div><div class="casAuthors">Chen, Weiqi; Caceres-Cortes, Janet; Zhang, Haiying; Zhang, Donglu; Humphreys, W. Griffith; Gan, Jinping</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">663-669</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The thiophene moiety has been recognized as a toxicophore because of the potential of oxidative bioactivation leading to electrophilic species.  The introduction of bulky or electron-withdrawing groups at the α-carbon to the sulfur atom has the potential to reduce or eliminate bioactivation.  In this article, we describe the bioactivation of a variety of substituted thiophenes.  These compds. were incubated in NADPH-fortified human liver microsomes with or without the addn. of reduced glutathione (GSH) as a trapping agent.  The resulting GSH adducts were characterized by high performance liq. chromatog./high resoln. mass spectrometry with the aid of a background subtraction methodol.  Four of the five α-chlorothiophenes tested formed NADPH-dependent GSH adducts.  Most adducts had masses consistent with the nominal substitution of chlorine by GSH.  LC/MS/MS and proton NMR of the major GSH adduct of 1-(5-chlorothiophen-2-yl)ethanone (1a) confirmed that GSH displaced chlorine.  To further explore the effect of different substitutions on the bioactivation potential, a series of 2-acetylthiophenes substituted at the C4 or C5 positions were tested in a quant. thiol-trapping assay using dansyl glutathione.  Substitutions at the C4 or C5 positions gave adduct levels that decreased in the following order: 4-H, 5-H (no substitution) > 4-Br ∼ 4-Cl > 5-Cl > 5-CN > 4-CH3 > 5-Br > 5-CH3 (no adduct detected).  In conclusion, bioactivation was detected in a series of substituted thiophenes.  Although substitutions on the thiophene ring can reduce the formation of reactive metabolites, the degree of redn. is dependent on the substitution position and substituent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRn2UHQF5AxbVg90H21EOLACvtfcHk0ljyHEyRIYdPGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFOqtbw%253D&md5=e7aa1532248bff324b084905ea12c6e6</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Ftx100386z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx100386z%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DCaceres-Cortes%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DHumphreys%26aufirst%3DW.%2BG.%26aulast%3DGan%26aufirst%3DJ.%26atitle%3DBioactivation%2520of%2520Substituted%2520Thiophenes%2520Including%2520%25CE%25B1-Chlorothiophene-Containing%2520Compounds%2520in%2520Human%2520Liver%2520Microsomes%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2011%26volume%3D24%26spage%3D663%26epage%3D669%26doi%3D10.1021%2Ftx100386z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velez-Vega, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, D. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravamuthan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, J. S.</span></span> <span> </span><span class="NLM_article-title">Time-Averaged Distributions of Solute and Solvent Motions: Exploring Proton Wires of GFP and PfM2DH</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3344</span>– <span class="NLM_lpage">3361</span>, <span class="refDoi"> DOI: 10.1021/ci500571h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500571h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3344-3361&author=C.+Velez-Vegaauthor=D.+J.+J.+McKayauthor=V.+Aravamuthanauthor=R.+Pearlsteinauthor=J.+S.+Duca&title=Time-Averaged+Distributions+of+Solute+and+Solvent+Motions%3A+Exploring+Proton+Wires+of+GFP+and+PfM2DH&doi=10.1021%2Fci500571h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Time-Averaged Distributions of Solute and Solvent Motions: Exploring Proton Wires of GFP and PfM2DH</span></div><div class="casAuthors">Velez-Vega, Camilo; McKay, Daniel J. J.; Aravamuthan, Vibhas; Pearlstein, Robert; Duca, Jose S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3344-3361</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proton translocation pathways of selected variants of the green fluorescent protein (GFP) and Pseudomonas fluorescens mannitol 2-dehydrogenase (PfM2DH) were studied via an explicit solvent mol. dynamics-based anal. protocol that allows for direct quant. relation between a crystal structure and its time-averaged solute-solvent structure obtained from simulation.  The authors' study of GFP is in good agreement with previous research suggesting that the proton released from the chromophore upon photoexcitation can diffuse through an extended internal hydrogen bonding network that allows for the proton to exit to bulk or be recaptured by the anionic chromophore.  Conversely for PfM2DH, the authors identified the most probable ionization states of key residues along the proton escape channel from the catalytic site to bulk solvent, wherein the solute and high-d. solvent crystal structures of binary and ternary complexes were properly reproduced.  Furthermore, the authors proposed a plausible mechanism for this proton translocation process that is consistent with the state-dependent structural shifts obsd. in the authors' anal.  The time-averaged structures generated from the authors' analyses facilitate validation of MD simulation results and provide a comprehensive profile of the dynamic all-occupancy solvation network within and around a flexible solute, from which detailed hydrogen-bonding networks can be inferred.  In this way, potential drawbacks arising from the elucidation of these networks by examn. of static crystal structures or via alternate rigid-protein solvation anal. procedures can be overcome.  Complementary studies aimed at the effective use of the authors' methodol. for alternate implementations (e.g., ligand design) are currently underway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpub7nL0UsqGrVg90H21EOLACvtfcHk0ljyHEyRIYdPGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGktbbO&md5=d34a5696e577371ff4db5630923dedb0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci500571h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500571h%26sid%3Dliteratum%253Aachs%26aulast%3DVelez-Vega%26aufirst%3DC.%26aulast%3DMcKay%26aufirst%3DD.%2BJ.%2BJ.%26aulast%3DAravamuthan%26aufirst%3DV.%26aulast%3DPearlstein%26aufirst%3DR.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26atitle%3DTime-Averaged%2520Distributions%2520of%2520Solute%2520and%2520Solvent%2520Motions%253A%2520Exploring%2520Proton%2520Wires%2520of%2520GFP%2520and%2520PfM2DH%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3344%26epage%3D3361%26doi%3D10.1021%2Fci500571h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and Characterizing Binding Sites and Assessing Druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+Characterizing+Binding+Sites+and+Assessing+Druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lh1t3c724oahQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520Characterizing%2520Binding%2520Sites%2520and%2520Assessing%2520Druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein–Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lh1t3c724oahQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Low, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vachaspati, P.</span></span> <span> </span><span class="NLM_article-title">Optimised Method to Estimate Octanol Water Distribution Coefficient (LogD) in a High Throughput Format</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2016.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1016%2Fj.ejps.2016.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=27373604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKrsr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2016&pages=110-116&author=Y.+W.+Lowauthor=F.+Blascoauthor=P.+Vachaspati&title=Optimised+Method+to+Estimate+Octanol+Water+Distribution+Coefficient+%28LogD%29+in+a+High+Throughput+Format&doi=10.1016%2Fj.ejps.2016.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Optimised method to estimate octanol water distribution coefficient (logD) in a high throughput format</span></div><div class="casAuthors">Low, Ying Wei; Blasco, Francesca; Vachaspati, Prakash</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110-116</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lipophilicity is one of the mol. properties assessed in early drug discovery.  Direct measurement of the octanol-water distribution coeff. (logD) requires an anal. method with a large dynamic range or multistep dilns., as the analyte's concns. span across several orders of magnitude.  In addn., water/buffer and octanol phases which have very different polarity could lead to matrix effects and affect the LC-MS response, leading to erroneous logD values.  Most compd. libraries use DMSO stocks as it greatly reduces the sample requirement but the presence of DMSO has been shown to underestimate the lipophilicity of the analyte.  The present work describes the development of an optimized shake flask logD method using deepwell 96 well plate that addresses the issues related to matrix effects, DMSO concn. and incubation conditions and is also amenable to high throughput.  Our results indicate that the equil. can be achieved within 30 min by flipping the plate on its side while even 0.5% of DMSO is not tolerated in the assay.  This study uses the matched matrix concept to minimise the errors in analyzing the two phases namely buffer and octanol in LC-MS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHQW7-Y-gkR7Vg90H21EOLACvtfcHk0lh1t3c724oahQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKrsr%252FI&md5=97b8c5312efc8b7f535332af1c0d9cab</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2016.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2016.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DLow%26aufirst%3DY.%2BW.%26aulast%3DBlasco%26aufirst%3DF.%26aulast%3DVachaspati%26aufirst%3DP.%26atitle%3DOptimised%2520Method%2520to%2520Estimate%2520Octanol%2520Water%2520Distribution%2520Coefficient%2520%2528LogD%2529%2520in%2520a%2520High%2520Throughput%2520Format%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2016%26volume%3D92%26spage%3D110%26epage%3D116%26doi%3D10.1016%2Fj.ejps.2016.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trunzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span></span> <span> </span><span class="NLM_article-title">CYP3A Time-Dependent Inhibition Risk Assessment Validated with 400 Reference Drugs</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1046</span>, <span class="refDoi"> DOI: 10.1124/dmd.110.037911</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1124%2Fdmd.110.037911" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=21383203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFGns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1039-1046&author=A.+Zimmerlinauthor=M.+Trunzerauthor=B.+Faller&title=CYP3A+Time-Dependent+Inhibition+Risk+Assessment+Validated+with+400+Reference+Drugs&doi=10.1124%2Fdmd.110.037911"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">CYP3A time-dependent inhibition risk assessment validated with 400 reference drugs</span></div><div class="casAuthors">Zimmerlin, Alfred; Trunzer, Markus; Faller, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1039-1046</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Although reversible CYP3A inhibition testing is well established for predicting the drug-drug interaction potential of clin. candidates, time-dependent inhibition (TDI) has become the focus of drug designers only recently.  Failure of several late-stage clin. candidates has been attributed to TDI, and this mechanism is also suspected to play a role in liver toxicities often obsd. in preclin. species.  Measurement of enzyme inactivation rates (kinact and KI) is tech. challenging, and a great deal of variability can be found in the literature.  In this article, we have evaluated the TDI potential for 400 registered drugs using a high-throughput assay format based on detn. of the inactivation rate (kobs) at a single concn. of test compd. (10 μM).  The advantages of this new assay format are highlighted by comparison with data generated using the IC50 shift assay, a current std. approach for preliminary assessment of TDI.  With use of an empirically defined pos./neg. kobs bin of 0.02 min-1, only 4% of registered drugs were found to be pos.  This proportion increased to more than 20% when inhouse lead optimization mols. were considered, emphasizing the importance of identifying this property in selection of promising drug candidates.  Finally, it is suggested that the data and technol. described here may be a good basis for building structure-activity relationships and in silico modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYvW4ft03jbVg90H21EOLACvtfcHk0ljTm6zY8QMVjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFGns7w%253D&md5=d21b39e85ab9410b7addfc820c1a4e18</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fdmd.110.037911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.110.037911%26sid%3Dliteratum%253Aachs%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DTrunzer%26aufirst%3DM.%26aulast%3DFaller%26aufirst%3DB.%26atitle%3DCYP3A%2520Time-Dependent%2520Inhibition%2520Risk%2520Assessment%2520Validated%2520with%2520400%2520Reference%2520Drugs%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D1039%26epage%3D1046%26doi%3D10.1124%2Fdmd.110.037911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnes-Seeman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsis, P.</span></span> <span> </span><span class="NLM_article-title">Metabolically Stable Tert-Butyl Replacement</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/ml400045j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400045j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=514-516&author=D.+Barnes-Seemanauthor=M.+Jainauthor=L.+Bellauthor=S.+Ferreiraauthor=S.+Cohenauthor=X.-H.+Chenauthor=J.+Aminauthor=B.+Snodgrassauthor=P.+Hatsis&title=Metabolically+Stable+Tert-Butyl+Replacement&doi=10.1021%2Fml400045j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolically Stable tert-Butyl Replacement</span></div><div class="casAuthors">Barnes-Seeman, David; Jain, Monish; Bell, Leslie; Ferreira, Suzie; Cohen, Scott; Chen, Xiao-Hui; Amin, Jakal; Snodgrass, Brad; Hatsis, Panos</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">514-516</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Susceptibility to metab. is a common issue with the tert-Bu group on compds. of medicinal interest.  The authors demonstrate an approach of removing all the fully Sp3 C-Hs from a tert-Bu group: replacing some C-Hs with C-Fs and increasing the s-character of the remaining C-Hs.  This approach gave a trifluoromethylcyclopropyl group, which increased metabolic stability.  Trifluoromethylcyclopropyl-contg. analogs had consistently higher metabolic stability in vitro and in vivo compared to their tert-butyl-contg. counterparts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrao2I5ukFsTrVg90H21EOLACvtfcHk0ljTm6zY8QMVjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1ymt78%253D&md5=156572485ea3a48f5e1b6ea7409f1f61</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fml400045j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400045j%26sid%3Dliteratum%253Aachs%26aulast%3DBarnes-Seeman%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DL.%26aulast%3DFerreira%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.-H.%26aulast%3DAmin%26aufirst%3DJ.%26aulast%3DSnodgrass%26aufirst%3DB.%26aulast%3DHatsis%26aufirst%3DP.%26atitle%3DMetabolically%2520Stable%2520Tert-Butyl%2520Replacement%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D514%26epage%3D516%26doi%3D10.1021%2Fml400045j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for Flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0ljTm6zY8QMVjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Absolute Solvation Free Energies Using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1021/ct900587b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900587b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1509-1519&author=D.+Shivakumarauthor=J.+Williamsauthor=Y.+Wuauthor=W.+Dammauthor=J.+Shelleyauthor=W.+Sherman&title=Prediction+of+Absolute+Solvation+Free+Energies+Using+Molecular+Dynamics+Free+Energy+Perturbation+and+the+OPLS+Force+Field&doi=10.1021%2Fct900587b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Williams, Joshua; Wu, Yujie; Damm, Wolfgang; Shelley, John; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1509-1519</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of protein-ligand binding free energies is a primary objective in computer-aided drug design.  The solvation free energy of a small mol. provides a surrogate to the desolvation of the ligand in the thermodn. process of protein-ligand binding.  Here, we use explicit solvent mol. dynamics free energy perturbation to predict the abs. solvation free energies of a set of 239 small mols., spanning diverse chem. functional groups commonly found in drugs and drug-like mols.  We also compare the performance of abs. solvation free energies obtained using the OPLS_2005 force field with two other commonly used small mol. force fields - general AMBER force field (GAFF) with AM1-BCC charges and CHARMm-MSI with CHelpG charges.  Using the OPLS_2005 force field, we obtain high correlation with exptl. solvation free energies (R2 = 0.94) and low av. unsigned errors for a majority of the functional groups compared to AM1-BCC/GAFF or CHelpG/CHARMm-MSI.  However, OPLS_2005 has errors of over 1.3 kcal/mol for certain classes of polar compds.  We show that predictions on these compd. classes can be improved by using a semiempirical charge assignment method with an implicit bond charge correction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_fkbSHqg3bVg90H21EOLACvtfcHk0lhRL7KDnKIhEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D&md5=25d464fb9238e3235e881945c0b50a76</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fct900587b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900587b%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DPrediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520Using%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Perturbation%2520and%2520the%2520OPLS%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1509%26epage%3D1519%26doi%3D10.1021%2Fct900587b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guntermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piaia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamel, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubic-Schneider, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rio-Espinola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoegenauer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orain, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doelemeyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deurinck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schümann, J.</span></span> <span> </span><span class="NLM_article-title">Retinoic-Acid-Orphan-Receptor-C Inhibition Suppresses Th17 Cells and Induces Thymic Aberrations</span>. <i>JCI Insight</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e91127</span> <span class="refDoi"> DOI: 10.1172/jci.insight.91127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=10.1172%2Fjci.insight.91127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;key=28289717" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&author=C.+Guntermannauthor=A.+Piaiaauthor=M.-L.+Hamelauthor=D.+Theilauthor=T.+Rubic-Schneiderauthor=A.+del+Rio-Espinolaauthor=L.+Dongauthor=A.+Billichauthor=K.+Kaupmannauthor=J.+Dawsonauthor=K.+Hoegenauerauthor=D.+Orainauthor=S.+Hintermannauthor=R.+Stringerauthor=D.+D.+Patelauthor=A.+Doelemeyerauthor=M.+Deurinckauthor=J.+Sch%C3%BCmann&title=Retinoic-Acid-Orphan-Receptor-C+Inhibition+Suppresses+Th17+Cells+and+Induces+Thymic+Aberrations&doi=10.1172%2Fjci.insight.91127"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1172%2Fjci.insight.91127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci.insight.91127%26sid%3Dliteratum%253Aachs%26aulast%3DGuntermann%26aufirst%3DC.%26aulast%3DPiaia%26aufirst%3DA.%26aulast%3DHamel%26aufirst%3DM.-L.%26aulast%3DTheil%26aufirst%3DD.%26aulast%3DRubic-Schneider%26aufirst%3DT.%26aulast%3Ddel%2BRio-Espinola%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DBillich%26aufirst%3DA.%26aulast%3DKaupmann%26aufirst%3DK.%26aulast%3DDawson%26aufirst%3DJ.%26aulast%3DHoegenauer%26aufirst%3DK.%26aulast%3DOrain%26aufirst%3DD.%26aulast%3DHintermann%26aufirst%3DS.%26aulast%3DStringer%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DD.%2BD.%26aulast%3DDoelemeyer%26aufirst%3DA.%26aulast%3DDeurinck%26aufirst%3DM.%26aulast%3DSch%25C3%25BCmann%26aufirst%3DJ.%26atitle%3DRetinoic-Acid-Orphan-Receptor-C%2520Inhibition%2520Suppresses%2520Th17%2520Cells%2520and%2520Induces%2520Thymic%2520Aberrations%26jtitle%3DJCI%2520Insight%26date%3D2017%26volume%3D2%26doi%3D10.1172%2Fjci.insight.91127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NTI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NTI','PDB','5NTI'); return false;">PDB: 5NTI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O','PDB','6Q6O'); return false;">PDB: 6Q6O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7A','PDB','6Q7A'); return false;">PDB: 6Q7A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H','PDB','6Q7H'); return false;">PDB: 6Q7H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6M','PDB','6Q6M'); return false;">PDB: 6Q6M</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i40"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01291">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_46301"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01291?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01291</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Kinase and safety panel assays for compound <b>14</b>; structural biology specifics (expression and purification, cocrystallization, and X-ray diffraction data collection and processing); X-ray crystal data and structure refinement for compound <b>14</b>; design based on the WATMD model; and cartoon illustrating nuclear receptor LBD conformations (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings including assay data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_001.pdf">jm9b01291_si_001.pdf (1.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01291/suppl_file/jm9b01291_si_002.csv">jm9b01291_si_002.csv (1.18 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors for cocrystal structures have been deposited in the Protein Data Bank with the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6M">6Q6M</a> (compound <b>1</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q6O">6Q6O</a> (compound <b>2</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7A">6Q7A</a> (compound <b>4</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q7H">6Q7H</a> (compound <b>9</b>). Authors will release the atomic coordinates and experimental data upon article publication. Crystallographic data of compound <b>14</b> (excluding structure factors) have been deposited with the Cambridge Crystallographic Data Centre as supplementary publication number CCDC 1944604. Copies of the data can be obtained free of charge on application to CCDC, 12 Union Road, Cambridge CB2 1EZ, UK [fax (+44) 1223 336033, email: <a href="/cdn-cgi/l/email-protection#ee8a8b9e819d879aae8d8d8a8dc08d8f83c08f8dc09b85"><span class="__cf_email__" data-cfemail="1b7f7e6b7468726f5b78787f7835787a76357a78356e70">[email protected]</span></a>]. This material is available free of charge via the internet at http://pubs.acs.org.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01291&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01291%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-23" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01291" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67995b0118093c8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
