
<html lang="en"     class="pb-page"  data-request-id="28cec140-cc8b-46c2-9525-e4d5f4d0fa0f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.5b01283;website:website:acspubs;issue:issue:10.1021/jmcmar.2015.58.issue-24;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Synthesis, Binding and Antiviral Properties of Potent Core-Extended Naphthalene Diimides Targeting the HIV-1 Long Terminal Repeat Promoter G-Quadruplexes" /></meta><meta name="dc.Creator" content="Rosalba  Perrone" /></meta><meta name="dc.Creator" content="Filippo  Doria" /></meta><meta name="dc.Creator" content="Elena  Butovskaya" /></meta><meta name="dc.Creator" content="Ilaria  Frasson" /></meta><meta name="dc.Creator" content="Silvia  Botti" /></meta><meta name="dc.Creator" content="Matteo  Scalabrin" /></meta><meta name="dc.Creator" content="Sara  Lago" /></meta><meta name="dc.Creator" content="Vincenzo  Grande" /></meta><meta name="dc.Creator" content="Matteo  Nadai" /></meta><meta name="dc.Creator" content="Mauro  Freccero" /></meta><meta name="dc.Creator" content="Sara N.  Richter" /></meta><meta name="dc.Description" content="We have previously reported that stabilization of the G-quadruplex structures in the HIV-1 long terminal repeat (LTR) promoter suppresses viral transcription. Here we sought to develop new G-quadru..." /></meta><meta name="Description" content="We have previously reported that stabilization of the G-quadruplex structures in the HIV-1 long terminal repeat (LTR) promoter suppresses viral transcription. Here we sought to develop new G-quadru..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 8, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01283" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01283" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01283" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01283" /></link>
        
    
    

<title>Synthesis, Binding and Antiviral Properties of Potent Core-Extended Naphthalene Diimides Targeting the HIV-1 Long Terminal Repeat Promoter G-Quadruplexes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01283" /></meta><meta property="og:title" content="Synthesis, Binding and Antiviral Properties of Potent Core-Extended Naphthalene Diimides Targeting the HIV-1 Long Terminal Repeat Promoter G-Quadruplexes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0003.jpeg" /></meta><meta property="og:description" content="We have previously reported that stabilization of the G-quadruplex structures in the HIV-1 long terminal repeat (LTR) promoter suppresses viral transcription. Here we sought to develop new G-quadruplex ligands to be exploited as antiviral compounds by enhancing binding toward the viral G-quadruplex structures. We synthesized naphthalene diimide derivatives with a lateral expansion of the aromatic core. The new compounds were able to bind/stabilize the G-quadruplex to a high extent, and some of them displayed clear-cut selectivity toward the viral G-quadruplexes with respect to the human telomeric G-quadruplexes. This feature translated into low nanomolar anti-HIV-1 activity toward two viral strains and encouraging selectivity indexes. The selectivity depended on specific recognition of LTR loop residues; the mechanism of action was ascribed to inhibition of LTR promoter activity in cells. This is the first example of G-quadruplex ligands that show increased selectivity toward the viral G-quadruplexes and display remarkable antiviral activity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01283"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01283">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01283&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01283&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01283&amp;href=/doi/10.1021/acs.jmedchem.5b01283" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 9639-9652</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01611" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01295" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthesis, Binding and Antiviral Properties of Potent Core-Extended Naphthalene Diimides Targeting the HIV-1 Long Terminal Repeat Promoter G-Quadruplexes</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rosalba++Perrone">Rosalba Perrone</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Filippo++Doria">Filippo Doria</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Butovskaya">Elena Butovskaya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ilaria++Frasson">Ilaria Frasson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Silvia++Botti">Silvia Botti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matteo++Scalabrin">Matteo Scalabrin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sara++Lago">Sara Lago</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vincenzo++Grande">Vincenzo Grande</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matteo++Nadai">Matteo Nadai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mauro++Freccero">Mauro Freccero</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sara+N.++Richter">Sara N. Richter</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Chemistry, University of Pavia, V.le Taramelli 10, 27100 Pavia, Italy</span></div><div class="corresp-info"><strong>*</strong>For M.F.: phone, +39 0382 987668; fax, +39 0382 987323; E-mail, <a href="/cdn-cgi/l/email-protection#3b565a4e4954155d495e58585e49547b4e55524b4d15524f"><span class="__cf_email__" data-cfemail="89e4e8fcfbe6a7effbeceaeaecfbe6c9fce7e0f9ffa7e0fd">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For S.N.R.: phone, +39 049 8272346; fax, +39 049 8272355; E-mail, <a href="/cdn-cgi/l/email-protection#94e7f5e6f5bae6fdf7fce0f1e6d4e1fafde4f0bafde0"><span class="__cf_email__" data-cfemail="22514350430c504b414a56475062574c4b52460c4b56">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01283&amp;href=/doi/10.1021%2Facs.jmedchem.5b01283" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 9639–9652</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 24, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 August 2015</li><li><span class="item_label"><b>Published</b> online</span>8 December 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 December 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01283" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01283</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01283"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3761</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">56</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01283" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthesis, Binding and Antiviral Properties of Potent Core-Extended Naphthalene Diimides Targeting the HIV-1 Long Terminal Repeat Promoter G-Quadruplexes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Rosalba&quot;,&quot;last_name&quot;:&quot;Perrone&quot;},{&quot;first_name&quot;:&quot;Filippo&quot;,&quot;last_name&quot;:&quot;Doria&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Butovskaya&quot;},{&quot;first_name&quot;:&quot;Ilaria&quot;,&quot;last_name&quot;:&quot;Frasson&quot;},{&quot;first_name&quot;:&quot;Silvia&quot;,&quot;last_name&quot;:&quot;Botti&quot;},{&quot;first_name&quot;:&quot;Matteo&quot;,&quot;last_name&quot;:&quot;Scalabrin&quot;},{&quot;first_name&quot;:&quot;Sara&quot;,&quot;last_name&quot;:&quot;Lago&quot;},{&quot;first_name&quot;:&quot;Vincenzo&quot;,&quot;last_name&quot;:&quot;Grande&quot;},{&quot;first_name&quot;:&quot;Matteo&quot;,&quot;last_name&quot;:&quot;Nadai&quot;},{&quot;first_name&quot;:&quot;Mauro&quot;,&quot;last_name&quot;:&quot;Freccero&quot;},{&quot;first_name&quot;:&quot;Sara&quot;,&quot;last_name&quot;:&quot;N. Richter&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;9639-9652&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01283&quot;},&quot;abstract&quot;:&quot;We have previously reported that stabilization of the G-quadruplex structures in the HIV-1 long terminal repeat (LTR) promoter suppresses viral transcription. Here we sought to develop new G-quadruplex ligands to be exploited as antiviral compounds by enhancing binding toward the viral G-quadruplex structures. We synthesized naphthalene diimide derivatives with a lateral expansion of the aromatic core. The new compounds were able to bind/stabilize the G-quadruplex to a high extent, and some of them displayed clear-cut selectivity toward the viral G-quadruplexes with respect to the human telomeric G-quadruplexes. This feature translated into low nanomolar anti-HIV-1 activity toward two viral strains and encouraging selectivity indexes. The selectivity depended on specific recognition of LTR loop residues; the mechanism of action was ascribed to inhibition of LTR promoter activity in cells. This is the first example of G-quadruplex ligands that show increased selectivity toward the viral G-quadruplexes and disp&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01283&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01283" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01283&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01283" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01283&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01283" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01283&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01283&amp;href=/doi/10.1021/acs.jmedchem.5b01283" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01283" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01283" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01283%26sid%3Dliteratum%253Aachs%26pmid%3D26599611%26genre%3Darticle%26aulast%3DPerrone%26date%3D2015%26atitle%3DSynthesis%252C%2BBinding%2Band%2BAntiviral%2BProperties%2Bof%2BPotent%2BCore-Extended%2BNaphthalene%2BDiimides%2BTargeting%2Bthe%2BHIV-1%2BLong%2BTerminal%2BRepeat%2BPromoter%2BG-Quadruplexes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D24%26spage%3D9639%26epage%3D9652%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291192" title="Stabilization">Stabilization</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/jmcmar.2015.58.issue-24/20151224/jmcmar.2015.58.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We have previously reported that stabilization of the G-quadruplex structures in the HIV-1 long terminal repeat (LTR) promoter suppresses viral transcription. Here we sought to develop new G-quadruplex ligands to be exploited as antiviral compounds by enhancing binding toward the viral G-quadruplex structures. We synthesized naphthalene diimide derivatives with a lateral expansion of the aromatic core. The new compounds were able to bind/stabilize the G-quadruplex to a high extent, and some of them displayed clear-cut selectivity toward the viral G-quadruplexes with respect to the human telomeric G-quadruplexes. This feature translated into low nanomolar anti-HIV-1 activity toward two viral strains and encouraging selectivity indexes. The selectivity depended on specific recognition of LTR loop residues; the mechanism of action was ascribed to inhibition of LTR promoter activity in cells. This is the first example of G-quadruplex ligands that show increased selectivity toward the viral G-quadruplexes and display remarkable antiviral activity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">G-quadruplexes (G4s) are nucleic acids secondary structures that may form in single-stranded G-rich sequences under physiological conditions.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Four Gs bind via Hoogsteen-type hydrogen bonds base-pairing to yield G-quartets, which in turn stack on top of each other to form the G4. G4 structures may significantly vary, both in terms of strand stoichiometry (forming both inter- and intramolecular structures) and strand orientation/topology. The presence of K<sup>+</sup> cations specifically supports G4 formation and stability.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4-6)</a> In the human genome and in prokaryotes, G4 DNA motifs have been found in telomeres, G-rich micro- and mini-satellites, near oncogene promoters, and within the rDNA.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12">(7-12)</a> Human G4 DNA motifs have been reported to be associated with recombination prone regions<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and to show mutational patterns that preserve the potential to form G4 DNA structures.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Misregulation of these G4 structures has been associated with relevant human neurological disorders.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14-18)</a> The identification of G4 binding proteins<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and their visualization in cells with antibody-based technology<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> have also provided convincing evidence of the existence of cellular G4s in vivo.</div><div class="NLM_p">Recently, the presence of G4s in viruses and their involvement in virus key steps has also been provided.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> G4s have been implicated in pathogenic mechanisms of the Epstein–Barr virus<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> and the SARS coronavirus.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> We and other groups have identified functionally significant G4s in the Nef coding region<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and the unique long terminal repeat (LTR) promoter<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27-29)</a> of the human immunodeficiency virus (HIV), the etiologic agent of the acquired immune deficiency syndrome (AIDS). These studies have shown that G4 folding at the LTR promoter decreased viral transcription with an effect that was augmented by a G4 ligand.<a onclick="showRef(event, 'ref27 ref30'); return false;" href="javascript:void(0);" class="ref ref27 ref30">(27, 30)</a></div><div class="NLM_p">The involvement of G4 structures in several human diseases propelled the development of G4 ligands. Several aromatic cores with protonable side chains bind the G4 conformation, such as porphyrins,<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> acridines, such as <b>1</b> (BRACO-19, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>),<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> fluoroquinolones<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> anthraquinones, phenanthrolines, quinacridines, carbazoles, bis-indole carboxamides, triazoles, and benzimidazoles;<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> natural compound derivatives, such as berberine,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> telomestatin,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> quindolin,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> several tetra-substituted perlyene tetracarboxylic<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and water-soluble naphthalene diimide (NDIs).<a onclick="showRef(event, 'ref41 ref42 ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43 ref44 ref45 ref46">(41-46)</a> So far, the vast majority of molecules has been tested against cellular G4s implicated in tumor pathogenesis: some compounds showed interesting antiproliferative properties;<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> in particular, quarfloxin proceeded into phase II clinical trials, but its limited bioavailability prevented further progress.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Few molecules have been tested also as antiviral agents:<a onclick="showRef(event, 'ref23 ref27 ref30 ref47'); return false;" href="javascript:void(0);" class="ref ref23 ref27 ref30 ref47">(23, 27, 30, 47)</a> despite the low selectivity indexes, these studies have shown the possibility to develop antiviral molecules with a G-quadruplex-mediated mechanism of action.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Structure of BRACO-19 and Core-Extended NDIs (c-exNDIs) Reported in the Literature,<a onclick="showRef(event, 'ref44 ref48 ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref44 ref48 ref49 ref50 ref51">(44, 48-51)</a> and the New Synthesized Water-Soluble c-exNDIs (lower right panel)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The lateral expansion of the NDI core provided by our investigation may offer an additional opportunity to enhance both G4 binding potency and selectivity, discriminating G4 structures with different accessibility to the edge quartets. Examples that comprise the annulation of five- and six-membered heterocyclic rings either on one side of the NDI<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> or on both sides have been reported (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50, 51)</a> Surprisingly, only one class of these core-extended NDIs (<b>c-exNDIs</b>) has been engineered water-soluble to be investigated as G4 ligand,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> the others have found applications in the field of material chemistry and as pH sensors.<a onclick="showRef(event, 'ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51">(49-51)</a></div><div class="NLM_p last">Because of the possibility to block HIV-1 transcription by stabilizing the G4 structures present in the unique viral LTR promoter, we here synthesized and tested a series of novel <b>c-exNDIs</b> for both their binding to the LTR G4s and anti-HIV-1 activity. In these new structures, the NDI core has been merged to the dihydrobenzophenazine polynuclear heterocycle (highlighted in red in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). We proved that these derivatives display binding and stabilization selectivity toward the viral G4s as compared to a telomeric G4 and that this translates into a therapeutic window where the compounds are active against the virus but not toxic for the cell.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11748" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11748" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design and Synthesis of <b>c-exNDIs</b></h3><div class="NLM_p">We have designed and synthesized a small library of new and water-soluble <b>c-exNDIs</b> (<b>2</b>–<b>11</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), where the NDI core has been merged to the dihydrobenzophenazine polynuclear heterocycle (highlighted in red in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The choice has been made based on the p<i>K</i><sub>a</sub> values of the <i>N</i>,<i>N</i>-dimethylpropan-amine side chains: the p<i>K</i><sub>a</sub> values are above 7 (p<i>K</i><sub>a1</sub> = 8.1, p<i>K</i><sub>a2</sub> = 8.6),<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> suggesting that the side chains are mainly protonated under physiological conditions. This aspect ensures good solubilizing properties, reducing self-aggregations. We discharged the shorter <i>N</i>,<i>N</i>-dimethylthyl-amine side chains, as their p<i>K</i><sub>a</sub> values are much lower (p<i>K</i><sub>a1</sub> = 5.6, p<i>K</i><sub>a2</sub> = 7.4). The <b>c-exNDIs</b> are structurally characterized by a larger flat core than the NDIs. This feature should improve π-stacking interaction implementing the binding to the G4. Indeed, DFT calculation performed at B3LYP/6-31+G(d,p) level of theory in gas phase indicated that the dimension of the <b>c-exNDI</b> core supported a more extensive overlap to the quartet of the G4 structure (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S1</a>). Enhancement of the ligand steric hindrance accessing the outside G4 quartets might offer an opportunity to implement selectivity as well. Furthermore, the proposed structural extension on the NDI scaffold should merge the useful optoelectronic properties (absorption, emission, and singlet oxygen sensitizer) of functionalized NDIs to those of substituted dihydrobenzophenazines. Compounds with both H and Br substituents at the Y position were considered to evaluate the effect of a bulky substituent (Br) at this position. To synthesize the <b>c-exNDIs</b>, we developed the synthetic protocol highlighted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. According to published methods, the imidation reaction of the readily available 2,6-dibromo-1,4,5,8-naphthalenetetracarboxylic dianhydride yielded the 2,6-dibromo-substituted NDI, under acidic conditions, in almost quantitative yield. Subsequently, a two-step protocol starting from a nucleophilic aromatic substitution (S<sub>N</sub>Ar) in the presence of an excess (2.5 equiv) of <i>ortho</i>-diaminophenyl derivatives in DMF as solvent at 70 °C for 16 h afforded a 60:40 mixture of monobrominated and debrominated products in an overall quantitative yield. The ring closing reaction following the S<sub>N</sub>Ar in one pot (step a, in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) occurred through a Tchitchibabin mechanism, with the elimination of hydride favored by the formation of a six-membered ring. The second S<sub>N</sub>Ar on the remaining aromatic bromide was carried out by dissolving <b>3</b> in neat <i>N</i><sup>1</sup><i>,N</i><sup>1</sup>-dimethylpropane-1,3-diamine. To improve the reaction yield, we optimized a microwave-assisted protocol (150 °C, 200 psi, 250 bar, 200 W, 3 min, sealed reaction vessels) to give product <b>9</b>, which crystallized from the reaction mixture. We achieved the synthesis of the permanently charged quaternary ammonium salts <b>10</b> and <b>11</b> with a classical exhaustive methylation, in the presence of MeI at rt for 16 h. All the synthesized ligands (<b>2</b>–<b>11</b>) were purified by preparative HPLC (CH<sub>3</sub>CN:H<sub>2</sub>O and 0.1% CF<sub>3</sub>COOH as eluent). Further anion exchange yielded <b>2</b>–<b>9</b> as hydrochloride salts. On the contrary, <b>10</b> and <b>11</b> were investigated as iodide quaternary ammonium salts.</div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Optimized Synthetic Protocol for the Preparation of Water-Soluble <b>2</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) DMF, 70°C, 16 h, under N<sub>2</sub>; (b) <i>N</i>,<i>N</i>-dimethylpropylamine, MW, 150°C 3 min, closed vessel; (c) MeI, CHCl<sub>3</sub>, rt, 16 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> <b>c-exNDIs</b> Greatly Stabilize G4 with a Preference for the LTR Conformations vs the Telomeric Sequence</h3><div class="NLM_p">To detect the ability of the new <b>c-exNDIs</b> to bind and stabilize the G4 conformations of the HIV-1 LTR promoter, two G4-forming LTR sequences were employed: LTR-III and LTR-IV, both composed of four GGG tracts (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). When embedded within the full-length G-rich sequence of the LTR promoter, LTR-III is the most stable G4, while LTR-IV folds when induced by G4 ligands.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> A FRET melting assay was performed to assess the degree of stabilization of the LTR-III and LTR-IV G4 by <b>c-exNDIs</b>. The minimal intramolecular G4-forming human telomeric (hTel) sequence (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A) was employed to evaluate the activity of the <b>c-exNDIs</b> on one of the most highly represented and accessible cellular G4: in fact, the TTAGGG repeat is present as 2000–3000 double-stranded (ds) and 50–200 single-stranded sequences at the human telomeres.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> A double-stranded (ds) DNA was also added to address specificity toward the G4 conformation (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Table S1</a>). While this kind of assay is typically performed using molar excess of the tested compounds, we here initially employed equimolar amounts of the target DNA sequence and compounds (i.e., 0.25 μM) to better differentiate the stabilization efficiency among the different <b>c-exNDIs</b>. The melting temperatures of the LTR-III, LTR-IV, hTel, and dsDNA G4 conformations were sufficiently similar (66.9, 61.9, 66.9, and 66.5 °C, respectively) to allow a meaningful comparison. In the presence of the <b>c-exNDIs</b>, a marked increase in the stabilization of the G4-forming oligonucleotides was observed (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Table S2</a>). Among the tested <b>c-exNDIs</b>, <b>2</b>, <b>3</b>, <b>4</b>, <b>10</b>, and <b>11</b> were the most efficient ligands, with a stabilization on LTR-IV G4 of 10–16 °C at equimolar amounts. At 4-fold <b>c-exNDI</b>:DNA molar ratio, the stabilization increased to above 25 °C, which is the maximum increase measurable in these settings (data not shown). The other <b>c-exNDIs</b> showed intermediate (4 °C ≤ Δ<i>T</i><sub>m</sub> < 7 °C, <b>5</b>, <b>6</b>, and <b>7</b>), or low (Δ<i>T</i><sub>m</sub> < 4 °C, <b>8</b> and <b>9</b>) level of stabilization on LTR-IV G4. Compounds <b>2</b> and <b>10</b> stabilized LTR-III G4 above 10 °C. Among the G4 structures, hTel G4 was the least efficiently stabilized, whereas stabilization on dsDNA was negligible. Only compounds <b>10</b> and <b>11</b> displayed a significant stabilization on dsDNA, indicating that the presence of two permanent charges (the quaternary ammonium moieties) favors an unspecific binding to the negatively charged nucleic acid. In general, the H series of all derivatives (Y = H, <b>2</b>, <b>4</b>, <b>6</b>, <b>10</b>; <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) was more active and more selective for the viral G4s compared to the Br substituted ones (Y = Br, <b>3</b>, <b>5</b>, <b>7</b>, <b>8</b>; <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Compounds <b>2</b>, <b>3</b>, and <b>10</b> exhibited a difference in stabilization of LTR-IV G4s vs hTel G4 equal or above 6 °C, which roughly corresponds to 40% of the overall stabilization efficiency (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Table S2</a>).</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. G4 stabilization by the <b>c-exNDIs</b>. (A) G4-forming sequences used in this study. (B) FRET analysis. Δ<i>T</i><sub>m</sub> values of HIV-1 LTR G4-folded sequences (LTR-III and LTR-IV), the G4-folded telomeric sequence (hTel), and a control double-stranded sequence (dsDNA) obtained upon addition of equimolar amounts of <b>2</b>–<b>11</b> (0.25 μM). (C) <i>Taq</i> polymerase stop assay. <i>Taq</i> polymerization was performed in the presence/absence of K<sup>+</sup> and <b>2</b>, as indicated, on the hTel, LTR-III, and LTR-IV templates, which were previously folded into G4. A control template unable to fold into G4 was also used (non-G4 cnt). Stop regions are highlighted by vertical bars. (D) Quantification of the intensity of the stop bands obtained in the <i>Taq</i> polymerase stop assay. Stop band intensity of the treated samples were normalized vs the untreated sample.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because oligonucleotides used for FRET analysis display fluorophore-modified ends that may influence the measured stabilization, additional techniques using label-free oligonucleotides were employed. Compound <b>2</b>, a representative of the best ligands, was assayed by circular dichroism (CD) on LTR-IV, LTR-III, and hTel G4s. At 4:1 <b>c-exNDI</b>:DNA ratio, a stabilization higher than 27 °C was obtained on both LTR-III and LTR-IV (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figures S2A and S2B</a>), while hTel G4s was stabilized by only 19.9 °C (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S2C</a>), confirming a preference for the viral G4s. A <i>Taq</i> polymerase stop assay was performed on the LTR-III, LTR-IV, and hTel templates. A control template unable to fold into G4 was used to exclude unspecific inhibition of the polymerase enzyme by the <b>c-exNDI</b>. Compound <b>2</b> induced an intense stop site at the most 3′ G tract in each G4-forming template (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C), indicating effective stabilization of the G4 conformation by the compound. This effect was significantly more relevant on the viral sequences vs the telomeric template (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>D), confirming again a preferential stabilization of the viral G4s.</div><div class="NLM_p">The above techniques evaluated stabilization of the G4 conformation as an indicator of the efficiency of the interaction of the ligands with the tested G4 structures. Binding affinity was next directly assessed by two different methods.</div><div class="NLM_p">Competition electrospray ionization mass spectrometry (ESI-MS) experiments<a onclick="showRef(event, 'ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56">(54-56)</a> allowed establishment of the relative affinity of the <b>c-exNDIs</b> toward the G4 sequences. We first confirmed by CD analysis that in the buffer conditions used for the ESI/MS experiments the topology of the G4-folded oligonucleotides did not significantly modify (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S3</a>) Equimolar amounts of LTR-III/hTel, LTR-IV/hTel, and LTR-III/LTR-IV G4s were incubated with 2-fold molar excess of compounds <b>2</b> or <b>4</b>. Because ESI/MS transfers intact biomolecules complexes in the gas phase and signal intensity is proportional to the abundance of each species in solution, the ratio between <i>m</i>/<i>z</i> signals of each bound species and the corresponding bound and free G4 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A) was calculated (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). While LTR-III and LTR-IV were detected mostly as all-bound species, hTel was mostly unbound (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A, and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figures S4A and S4B</a>). This analysis clearly shows that our <b>c-exNDIs</b> preferentially bind the two viral sequences over the telomeric sequence. To corroborate the selectivity toward the viral sequences, two additional cellular G4 folding sequences were evaluated: c-kit2<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and c-myc.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The LTR G4s were preferred over c-kit2 G4 by both compounds; in the case of c-myc G4, compound <b>4</b> preferentially bound to the LTR G4s, whereas compound <b>2</b> showed more intense binding to c-myc (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Therefore, we confirmed a general, even though not absolute, selectivity toward the LTR G4s. It is also possible that the much higher stability of the c-myc G4 (<i>T</i><sub>m</sub> > 95 °C) vs the stability of the other tested G4s (LTR-III 66.9 °C, LTR-IV 61.9 °C, c-kit2 72.1 °C) played a role in this result. When the two LTR G4s were competed with each other, the LTR-III sequence was preferred. At 2-fold excess of compound, LTR-III bound up to two molecules of compounds, whereas only one molecule was obtained for both LTR-IV and hTel (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S5</a>). At saturating concentrations (10-fold molar excess), three molecules were bound to LTR-III G4, whereas up to two molecules could bind LTR-IV and hTel G4s (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S5</a>).</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Affinity and binding site analysis. (A) ESI/MS competition assay. Representative spectrum. Equimolar amounts of LTR-III and hTel were incubated with 2-fold excess of <b>2</b> and analyzed by ESI/MS. The identity of <i>m</i>/<i>z</i> signals is shown above each signal. RI stands for relative intensity. (B) SPR analysis of <b>2</b> binding toward the LTRs and hTel G4s. Representative sensorgram of <b>2</b> binding to LTR-III G4. Biosensor data were collected at 25 °C; the compound was injected at increasing concentrations (26, 40, 59, 89, 133, 200 nM). Sensorgrams are shown as gray lines. Dotted black lines represent local 1:1 binding fits obtained for a calculated <i>R</i><sub>max</sub>. (C,D) CL-mediated footprinting. LTR-IV (C) and LTR-III (D) oligonucleotides were G4 folded, incubated with increasing amounts of <b>2</b> (lanes 5–13), alkylated with CL (CL lanes), and subjected to hot piperidine treatment (PIP lanes) to induce cleavage of the phosphodiester bond at the alkylated bases. (C) Lanes 1–4 are control lanes not treated with the <b>c-exNDI</b>. The symbols * and ○ indicate base protection and exposure, respectively. M is a marker lane obtained by the Maxam and Gilbert sequencing of the oligonucleotides. The oligonucleotide sequence is shown on the left. The vertical dotted bar indicates the region of the LTR-III shared with LTR-IV. (E) Schematics of the LTR-IV and LTR-III G4 conformations. The represented LTR-IV G4 topology has been solved by NMR (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2N4Y">2N4Y</a>); the shown LTR-III G4 conformation derives from previous low resolution structural data.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Relative Binding Affinity of <b>2</b> and <b>4</b> for the LTR-III, LTR-IV, hTel, c-kit2 (kit), and c-myc (myc) G4-Folded Oligonucleotides</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="6" align="center" char=".">binding affinity<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="."><b>c-exNDI</b><b>2</b></th><th class="rowsep1 colsep0" colspan="3" align="center" char="."><b>c-exNDI</b><b>4</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">competing G4s</th><th class="colsep0 rowsep0" align="center" char=".">LTR-III</th><th class="colsep0 rowsep0" align="center" char=".">LTR-IV</th><th class="colsep0 rowsep0" align="center" char=".">cell G4 (hTel, kit, or myc)</th><th class="colsep0 rowsep0" align="center" char=".">LTR-III</th><th class="colsep0 rowsep0" align="center" char=".">LTR-IV</th><th class="colsep0 rowsep0" align="center" char=".">cell G4 (hTel, kit, or myc)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-III/hTel</td><td class="colsep0 rowsep0" align="char" char=".">80</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">82</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-IV/hTel</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">75</td><td class="colsep0 rowsep0" align="char" char=".">18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-III/LTR-IV</td><td class="colsep0 rowsep0" align="char" char=".">71</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">80</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-III/kit</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">54</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-IV/kit</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-III/myc</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LTR-IV/myc</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">44</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">71</td><td class="colsep0 rowsep0" align="char" char=".">45</td></tr></tbody></table></div></div><div class="NLM_p">Next, absolute affinity of <b>2</b> was measured by surface plasmon resonance (SPR) toward the three G4-folded sequences. This analysis confirmed that LTR-III G4 displays the highest binding affinity (<i>K</i><sub>D</sub> 3.2 nM), followed by LTR-IV G4 (<i>K</i><sub>D</sub> 6.1 nM); hTel G4 exhibited the lowest affinity (<i>K</i><sub>D</sub> 13.1 nM) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B, and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S6</a>). The binding affinity of the Br derivatives with each of the substituents in position R against LTR-IV was next assessed. Compounds <b>3</b> and <b>5</b> had <i>K</i><sub>D</sub> values similar to that of <b>2</b> (<i>K</i><sub>D</sub> 6.6 and 5.9 nM, respectively), <b>10</b> displayed an increased binding affinity (<i>K</i><sub>D</sub> 1.7 nM), whereas <b>7</b> and <b>11</b> showed the lowest affinity (<i>K</i><sub>D</sub> 21.6 and 12.9 nM, respectively). These data are in line with the stabilization properties observed for these derivatives by FRET and CD analysis and confirm the improved and reduced recognition properties of the <b>c-exNDIs</b> exhibiting permanent dicationic (<b>10</b>) and neutral or monocationic charge states (<b>7</b>), respectively, under physiological conditions. They also confirm that a bulky substituent in position Y of the NDI core (<b>11</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) highly reduces binding, whereas a NO<sub>2</sub> group in position R maintains excellent binding properties (<b>5</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><div class="NLM_p last">We reasoned that the observed biased affinity for the LTR sequences could derive by specific binding of the <b>c-exNDIs</b> to structural/sequence features shared by the LTR G4s. To this end, we evaluated the protection exerted by compound <b>2</b> over residues of the LTR G4s against clerocidin (CL). This molecule has been shown to alkylate DNA single-stranded G, C, and A bases<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58, 59)</a> and to be able to predict G4 structural folding.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Labeled oligonucleotides were folded, treated with increasing amounts of <b>2</b>, exposed to CL, and further treated with piperidine to highlight CL-alkylated sites. In these settings, we expected to visualize only loop bases bound by the <b>c-exNDI</b> because base-paired Gs involved in G-quartet formation would not be available to CL alkylation, independently of the presence of the compound. The NMR structure of LTR-IV has been solved (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2N4Y">2N4Y</a>), and therefore we were able to assess that nucleotides affected by the presence of the <b>c-exNDI</b> were in two loops of the structure: in particular, in the ACTG loop the A base was protected, whereas C and G were exposed; in the 1-base-loop, the C base was exposed (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>E). The structure of LTR-III has not been solved yet, however, previous data indicated that this G4 can fold into different conformations, the main one of which is likely the one depicted in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>E, as detected by low-resolution structural data.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In this case, we observed protection of the C base in the ACTG loop (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D), which is shared with the LTR-IV sequence (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A); three additional residues (C, C, and G) were protected in the 11-base-loop (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>E). Protection and exposure of residues indicated specific interaction of <b>2</b> at the affected LTR sites. Interestingly, unique LTR loops and in particular the ACTG loop shared by both LTR-III and LTR-IV, and absent in the telomeric sequence, were involved, indicating that these are likely the moieties that induce selectivity. We cannot exclude that other regions are bound by the <b>c-exNDI</b>, but probably the interaction at other sites is less stable or more dynamic so that a clear protection/exposure could not be observed.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> The Extended Core NDIs Display Anti-HIV-1 Activity at Nanomolar Concentrations</h3><div class="NLM_p">We have previously shown that the HIV-1 viral promoter folds into dynamic G4s and that stabilization of these G4 conformations by the G4 ligand <b>1</b> represses viral transcription and virus production in infected cells with IC<sub>50</sub> (the concentration required to inhibit 50% of virus production) in the low μM range.<a onclick="showRef(event, 'ref27 ref30'); return false;" href="javascript:void(0);" class="ref ref27 ref30">(27, 30)</a> The average selectivity index (SI) for <b>1</b> was 12. Given the <b>c-exNDI</b> binding/stabilization selectivity observed in vitro for the viral G4s vs the cellular G4, we set out to investigate the anti-HIV-1 activity of the newly synthesized <b>c-exNDI</b> derivatives. Compound entry was preliminarily measured by phase contrast and fluorescence microscopy, taking advantage of the blue-colored <b>c-exNDI</b> dyes (λ<sub>abs</sub> = 555 and 600 nm, for the unsubstituted <b>2</b>) and their red fluorescence emission (λ<sub>em</sub> = 610 and 655 nm, for <b>2</b>). After 30 min of incubation, compounds concentrated into the cell (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S7</a>). We next treated infected cells with increasing amounts of compounds and measured virus production; concurrently, the cytotoxic concentration (CC<sub>50</sub>) of the <b>c-exNDIs</b> in the same conditions but on uninfected cells was measured by MTT assay. All compounds were tested against the HIV-1 NL4-3 strain, which is a X4 strain. Results are reported in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. Compounds <b>2</b>, <b>3</b>, <b>4</b>, and <b>5</b> displayed inhibitory concentration (IC<sub>50</sub>) values below 25 nM. Compound <b>2</b> displayed a promising SI of 40, the best among the tested <b>c-exNDIs</b>, whereas SI values of compound <b>3</b>, <b>4</b>, and <b>5</b> were around 20, again superior to <b>1</b>. Derivatives <b>6</b> and <b>7</b> exhibited low activity and low cytotoxicity, while <b>9</b> showed the worst combination with low activity, high cytotoxicity, and SI of 2.2. We also evaluated the range of concentration (therapeutic window or TW), at which we obtained antiviral activity paralleled by less than 80% cytotoxicity. As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A, <b>2</b> displayed the widest TW (3.9–250 nM); <b>4</b> and <b>5</b> also showed interesting TW values; TW concentrations for <b>6</b> and <b>7</b>, which are likely to be mainly monoanionic under physiological conditions as both the (CH<sub>2</sub>)<sub>3</sub>NHMe<sub>2</sub><sup>+</sup> moieties exhibit a p<i>K</i><sub>a</sub> >8.1,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> were in the μM range; no TW was observed for <b>8</b> and <b>9</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Finally, <b>10</b> and <b>11</b>, containing two quaternary ammonium moieties, did not display any antiviral or cytotoxic activity likely due to their inability to enter the cells, as already shown for NDIs with permanent positive charges.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Compound <b>2</b> and <b>3</b> were also tested on the HIV-1 BaL strain, a R5 strain with a different tropism. In this case, IC<sub>50</sub> values below 40 nM were obtained, with SI of 14 for the H derivative (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Note that the H series of all derivatives was more active than the Br substituted one. Moreover, all compounds displayed CC<sub>50</sub> values higher than 2 μM against a human fibroblast cell line.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antiviral activity of <b>c-exNDIs</b> and mechanism of action. (A) TZM-bl cells were infected with HIV-1 NL4-3 strain and treated with increasing concentrations (3.9–1000 nM) of the <b>c-exNDIs</b>. Uninfected cells were treated in the same conditions. Antiviral activity, expressed as relative luciferase units (RLU), was measured on infected cells, while cell viability was in parallel obtained with a MTT assay on the uninfected cells. The vertical dotted lines define the therapeutic window (TW) where the compound is active against the virus and not toxic against the cells. This figure is representative of the antiviral/cytotoxicity/TW experiments performed with the whole series of <b>c-exNDI</b> compounds. (B) Entry assay. TZM-bl cells were treated with the compound <b>2</b> (50–100 nM), AZT (0.5 μg/mL), or the entry inhibitor DS (100 μg/mL) at 0, 1, or 2 h pi. Virus production was measured at 31 h pi as RLU. The control depicts cells infected with the virus lacking compound treatment. (C) Effect of <b>2</b> on the LTR promoter activity compared to cytotoxicity. HeLa cells with a stably transfected LTR-EGFP reporter plasmid were treated with increasing concentrations of <b>2</b> (62.5–250 nM). EGFP mean was measured by FACS analysis. Cytotoxicity was estimate in parallel by gating on forward scatter (FSC) vs side scatter (SSC) using untreated cells as reference. (D) Specificity of the <b>2</b>-induced inhibition of LTR promoter activity on the G4 LTR conformation. 293T cells were transiently transfected with plasmids with the wt or M4 + 5 mutant LTR promoter sequence upstream a luciferase reporter gene. Luciferase signals were measured as a function of increasing <b>2</b> amounts and normalized to total protein content.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antiviral Effects of the <b>c-exNDI</b> Derivatives and the Control <b>1</b> against Two HIV-1 Strains in TZM-bl Cells<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">SI</th><th class="rowsep1 colsep0" colspan="2" align="center">TW (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"><b>c-exNDI</b></th><th class="colsep0 rowsep0" align="center">HIV-1 strain NL4-3</th><th class="colsep0 rowsep0" align="center">HIV-1 strain BaL</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">HIV-1 strain NL4-3</th><th class="colsep0 rowsep0" align="center">HIV-1 strain BaL</th><th class="colsep0 rowsep0" align="center">HIV-1 strain NL4-3</th><th class="colsep0 rowsep0" align="center">HIV-1 strain BaL</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="center">13.2 ± 0.1</td><td class="colsep0 rowsep0" align="center">36.2 ± 4.3</td><td class="colsep0 rowsep0" align="center">532.0 ± 11.7</td><td class="colsep0 rowsep0" align="center">40</td><td class="colsep0 rowsep0" align="center">14</td><td class="colsep0 rowsep0" align="center">3.9–250.0</td><td class="colsep0 rowsep0" align="center">15.6–250.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="center">15.7 ± 2.1</td><td class="colsep0 rowsep0" align="center">32.5 ± 7.1</td><td class="colsep0 rowsep0" align="center">343.2 ± 32.2</td><td class="colsep0 rowsep0" align="center">22</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">7.8–125.0</td><td class="colsep0 rowsep0" align="center">15.6–125.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="center">11.0 ± 0.3</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">188.0 ± 4.08</td><td class="colsep0 rowsep0" align="center">17</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">3.9–125.0</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="center">23.0 ± 1.3</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">439.0 ± 10.7</td><td class="colsep0 rowsep0" align="center">19</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">15.6–250.0</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="center">975.3 ± 16.2</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center"> > 5000</td><td class="colsep0 rowsep0" align="center"> >5</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">800–5000</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="center">862.6 ± 31.5</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center"> > 5000</td><td class="colsep0 rowsep0" align="center"> >6</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">800–5000</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="center">63.5 ± 4.8</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">458.1 ± 54.0</td><td class="colsep0 rowsep0" align="center">7.2</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">no TW</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="center">814.3 ± 36.4</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">1759.9 ± 62.5</td><td class="colsep0 rowsep0" align="center">2.2</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">no TW</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="center"> >5000</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center"> >5000</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">no TW</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="center"> > 000</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center"> >5000</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">no TW</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="center">3400 ± 140<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center"> >12000<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center"> >3.5</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">3000–12000</td><td class="colsep0 rowsep0" align="center">nd</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">NL4–3 and BaL are X4 and R5, respectively, HIV-1 strains. IC<sub>50</sub> is the compound concentration required to inhibit 50% of HIV-1 production; CC<sub>50</sub> is the compound concentration at which 50% of cell toxicity is observed; SI is the selectivity index; TW is the therapeutic window concentration. nd stands for “not determined”.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">indicates values previously reported.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> The NDIs Derivatives Inhibit HIV-1 LTR Promoter Activity</h3><div class="NLM_p">It has been reported that compounds with multiple positive charges can block viral entry by interfering with the CXCR4 coreceptor.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> To rule out this unspecific mechanism of action of the <b>c-exNDI</b>, we treated TZM-bl cells at 0, 1, and 2 h post infection (pi) with compound <b>2</b> and tested virus production at 31 h pi. No significant difference in the activity of the compound was observed at the tested times (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Zidovudine (AZT), a reference drug that targets the reverse transcriptase step,<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> displayed a behavior similar to that of the <b>c-exNDI</b>, maintaining activity from 0 to 2 h pi. In contrast, dextran sulfate (DS), a reference entry inhibitor,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> exhibited antiviral effect only when added at 0 h pi. (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). These data indicate that the initial steps are not targeted by <b>2</b>, which is thus active at/after 2 h pi, i.e., after the entry process has occurred.</div><div class="NLM_p last">We next tested the ability of <b>2</b> to reduce the activity of the HIV-1 LTR promoter in an EGFP reporter assay in HeLa-tat-III cells, which is a cell line stably transfected with a plasmid embedding the EGFP coding region under the control of the HIV-1 LTR promoter. Cells were treated with increasing concentrations of the <b>c-exNDI</b>, and the intensity of EGFP fluorescence was assessed by FACS analysis. A steady decrease in the mean levels of EGFP was observed at noncytotoxic concentrations of <b>2</b> (up to 60% of inhibition) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). To check if the activity of the <b>c-exNDI</b> at the LTR promoter was specific for the G4 conformation, two point mutations that abolish G4 folding were introduced in the LTR sequence (M4 + 5 LTR).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The wt LTR and M4 + 5 LTR luciferase reporter plasmids were transiently transfected into 293T cells either alone or in the presence of increasing amounts of compound <b>2</b>, and the luciferase signal was measured. Again, an important inhibition of LTR activity was observed on the wt LTR (up to 65% inhibition), whereas no effect was detected on the non-G4-forming mutant LTR sequence (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). These data indicate a G4-mediated inhibition of the viral promoter by the <b>c-exNDI</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have here shown the synthesis, G4 binding properties, and antiviral activity of a new series of NDI derivatives with an extended aromatic core. First, most of the new compounds displayed improved stabilization on G4 structures, with substantial increments in <i>T</i><sub>m</sub> above values previously reported for NDI derivatives.<a onclick="showRef(event, 'ref41 ref42 ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43 ref44 ref45 ref46">(41-46)</a> This feature is an indication that the expansion of the aromatic surface allows a more effective recognition of the G4 structure. This may be due to a more extensive overlap to the quartet of the G4 structure, as inferred by the dimension of the <b>c-exNDI</b> core, evaluated by DFT calculation (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Figure S1</a>). Second, these compounds exhibited a very promising antiviral activity. Compared to <b>1</b>,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> the best antiviral <b>2</b>–<b>5</b> showed improved IC<sub>50</sub> values (in the low nM vs low μM range). This feature may derive from the selectivity observed in vitro for the LTR G4s over the telomeric G4 structures. The viral and telomeric G4s are likely the two most abundant DNA G4 species in the cell during infection, therefore a preferential effect on the viral G4s would result in higher antiviral activity and lower cytotoxicity, as observed. The <b>c-exNDIs</b> displayed both selective binding affinity and stabilization for the viral G4s. Between the two LTR G4s, the LTR-III sequence was most efficiently bound by <b>2</b>, as measured by ESI/MS and SPR. Even though LTR-III (38 nts) and LTR-IV (29 nts) display an identical number of G-quartets, LTR-III offers longer loops that may accommodate additional molecules and therefore may display increased binding affinity. Indeed, at saturating concentrations, binding of up to three <b>c-exNDI</b> molecules was observed by ESI/MS with LTR-III compared to two molecules for LTR-IV. In addition, two loops in both LTR-III and LTR-IV G4s were shown to be involved in <b>c-exNDI</b> interaction by footprinting analyses. We suggest that the compounds interact with the edge G-quartets through the core NDI planar aromatic surface, while the extended moiety and side chains interact with the side loops, affording selectivity toward the LTR G4s. To our knowledge, this is the first example of this type of selectivity based on chemical recognition of specific residues; in fact, previously reported selectivity rather relied on the recognition of a G4 conformation.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div><div class="NLM_p">Analysis of the binding activity of the series of <b>c-exNDIs</b> led us to the following structure–activity relationship (SAR) conclusions: (i) the nonextended side of the <b>c-exNDI</b> core [position Y (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)], lacking substituents as the introduction of an aminoalkyl side chain, highly hinders stabilization of the G4 structures (i.e., <b>9</b>); (ii) Br is allowed as Y substituent, but it lowers binding and selectivity toward the viral sequences. The Br group may both hinder specific interactions (hence lower selectivity) and partially distort the aromatic core from planarity (hence lower G4 binding); (iii) the pyridine ring in place of the benzene in the extended core is strongly detrimental (compound <b>8</b>). The fair antiviral activity is paralleled by moderate cytotoxicity, allowing no TW, possibly due to off-target activity; (iv) the NO<sub>2</sub> moiety is allowed (compounds <b>4</b> and <b>5</b>) as compound <b>4</b> is the most powerful and selective ligand toward LTR-III; (v) the negatively charged carboxylate group decrease the electrostatic component of the binding to the DNA G4s (compounds <b>6</b> and <b>7</b>); (vi) on the contrary, the permanently positively charged ammonium groups extensively increase binding to the target G4s by adding electrostatic interaction (compounds <b>10</b> and <b>11</b>). However, compounds with permanent charges are not suitable for cell entry and hence cannot be exploited for antiviral purposes. A scheme of the chemical properties that lead to high-affinity recognition and selectivity of the <b>c-exNDI</b> binders is reported in the graphical abstract.</div><div class="NLM_p last">In conclusion, we have reported a new class of G4 ligands able to discriminate G4 structures based on the chemical recognition of structural features that are unique to the selected G4 sequences. On one hand this approach opens the possibility to develop anti-HIV compounds with a new mechanism of action that may complement current clinical AIDS therapies; on the other hand, our work clearly highlights the possibility to selectively recognize G4 structures with small molecules, prompting the search and development of G4 ligands specific for G4s implicated in several important human diseases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43328" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43328" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Materials and General Procedures</h3><div class="NLM_p last">Reagents, oligonucleotides, solvents, and chemicals were purchased from Alfa Aesar or Sigma-Aldrich and were used as supplied without further purification. TLC analysis was carried out on silica gel (Merck 60F-254) with visualization at 254 and 366 nm. HPLC analysis and purifications were performed using two different HPLC: Waters system combining a Delta 600 PUMP, a 2489 UV/vis detector and Fraction Collector III (for preparative and analytical), and an Agilent system series 1260 (for analytical). The analytical column was XSelect CSH Phenyl-Hexyl (150 mm × 4.6 mm) (Waters). The preparative column was XSelect CSH Prep phenyl–hexyl 5 μm (150 mm × 30 mm) (Waters). Flows were 1 or 1.4 mL/min for analytical and 27 mL/min for preparative. For the analytical analysis were used the following method A: (aqueous solvent: 0.1% trifluoroacetic acid in water; organic solvent, acetonitrile); method A = 1 mL/min, isocratic flow over 2 min 95% of aqueous solvent; gradient, 95% aqueous, gradually to 40% aqueous over 12 min and at the end an isocratic flow over 4 min. For the preparative purification was used the following method B: (aqueous solvent: 0.1% trifluoroacetic acid in water; organic solvent, acetonitrile); Method B = 27 mL/min, isocratic flow over 4 min 95% of aqueous solvent; gradient, 95% aqueous, gradually to 70% aqueous over 16 min and at the end an isocratic flow over 4 min). <sup>1</sup>H , <sup>13</sup>C NMR spectra were recorded on a Bruker ADVANCE 300 MHz spectrometer. The potentiometric titrations were made with a Radiometer TitraLab 90 titration system. UV/vis spectra were run on a Varian Cary 100 SCAN spectrophotometer with quartz cuvettes of the appropriate path length (0.1–1 cm) at 25.0 ± 0.1 °C. Emission spectra were recorded on a PerkinElmer LS 50B instrument.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry. General Methods</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Synthesis of Intermediates and Final Ligands</h4><div class="NLM_p last">Br<sub>2</sub>NDI has been synthesized according to the published procedure.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The new ligands (<b>2</b>–<b>11</b>) were analyzed by HPLC (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">HPLC Purity Data</a>), confirming ≥97.6% purity.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Nucleophilic Aromatic Substitution/Tchichibabin Reaction: Synthesis of <b>2</b>–<b>5</b>, <b>8</b></h4><div class="NLM_p">The starting NDI (0.5 mmol) was dissolved into 20 mL of DMF in a round-bottom flask together with 1.0 mmol of <i>o</i>-phenyl-diamine derivatives, the mixture was stirred at 45 °C for 16 h under argon. The resulting dark-violet solution was basified with sodium bicarbonate aqueous solution and extracted with three portion of CHCl<sub>3</sub> (50 mL). The organic layers were collected, dried on NaSO<sub>4</sub>, and evaporated under vacuo. The crude product was analyzed and purified by HPLC chromatography (CH<sub>3</sub>CN:H<sub>2</sub>O 0.1%TFA) according to analytical method A and preparative method B. HCl 1 M solution was added to each chromatographic portion. Solvent evaporation under vacuum afforded the product as hydrochloride salt.</div><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> <b>2</b>·2HCl</h5><div class="NLM_p last">Yield = 25.2%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): 7.74 (s, 2H), 7.11 (dd, <i>J</i> = 3.3, 5.7 Hz, 2H), 6.89 (dd, <i>J</i> = 3.3, 5.7 Hz, 2H), 4.06 (m, 4H); 3.26 (m, 4H), 2.94 (s, 12H), 2.14 (m, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD): 166.4, 164.3, 143.2, 128.7, 128.0, 127.0, 125.9, 123.0, 118.2, 96.0, 57.0, 56.0, 43.9, 38.5, 38.2, 24.9, 24.2. Anal. Calcd for C<sub>30</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>4</sub>: C, 58.73; H, 5.59; Cl, 11.56; N, 13.70; O, 10.43. Found: C, 58.87; H, 5.51; N, 13.52.</div></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> <b>3</b>·2HCl</h5><div class="NLM_p last">Yield = 19.4%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): 8.1 (s, 1H), 7.15 (dd, <i>J</i> = 3.4, 6.0 Hz, 2H), 7.03 (dd, <i>J</i> = 3.4, 6.0 Hz, 2H), 4.17 (m, 4H), 3.25 (m, 4H), 2.94 (s, 12H), 2.16 (m, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD): 166.5, 166.1, 163.9, 163.6, 152.4, 144.3, 132.5, 130.3, 129.0, 128.9, 128.3, 128.1, 126.6, 123.1, 121.1, 120.4, 118.4, 118.2, 97.0, 79.8, 57.1, 43.9, 38.7, 38.5, 24.9, 24.7. Anal. Calcd for C<sub>30</sub>H<sub>33</sub>BrCl<sub>2</sub>N<sub>6</sub>O<sub>4</sub>: C, 52.04; H, 4.80; Br, 11.54; Cl, 10.24; N, 12.14; O, 9.24. Found: C, 52.17; H, 4.81; N, 12.12.</div></div><div id="sec4_2_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> <b>4</b>·2HCl</h5><div class="NLM_p last">Yield = 11.2%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): 8.04 (s, 2H), 7.88 (d, <i>J</i> = 6.6 Hz, 1H), 7.84 (1H, s), 7.07 (d, <i>J</i> = 6.5 Hz, 1H), 4.2 (bs, 4H), 3.27 (m, 4H), 2.94 (s, 12H), 2.17 (m, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD):166.9, 166.7, 164.5, 134.6, 132.1, 129.6, 127.3, 127.2, 126.9, 126.6, 124.3, 124.1, 122.8, 117.8, 112.9, 99.6, 99.4, 57.0, 43.8, 38.4, 24.8. Anal. Calcd for C<sub>30</sub>H<sub>33</sub>Cl<sub>2</sub>N<sub>7</sub>O<sub>6</sub>: C, 54.72; H, 5.05; Cl, 10.77; N, 14.89; O, 14.58. Found: C, 54.80; H, 5.01; N, 14.92.</div></div><div id="sec4_2_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> <b>5</b>·2HCl</h5><div class="NLM_p last">Yield = 18.9%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): 8.00 (s, 1H), 7.87 (d, <i>J</i> = 6.6 Hz, 1H), 7.79 (1H, s), 7.0 (d, <i>J</i> = 6.5 Hz, 1H), 3.98 (bs, 4H), 3.14 (m, 4H), 2.83 (s, 12H), 2.01 (m, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD):162.0, 161.0, 144.6, 141.4, 140.7, 132.1, 127.1, 124.1, 122.9, 122.1, 121.2, 120.7, 118.9, 111.9, 55.1, 48.7, 46.5, 42.6, 37.5, 37.2, 22.5, 22.4. Anal. Calcd for C<sub>30</sub>H<sub>32</sub>BrCl<sub>2</sub>N<sub>7</sub>O<sub>6</sub>: C, 48.86; H, 4.37; Br, 10.84; Cl, 9.61; N, 13.30; O, 13.02. Found: C, 48.79; H, 4.31; N, 13.22.</div></div><div id="sec4_2_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> <b>8</b>·2HCl</h5><div class="NLM_p last">Yield = 18%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 7.85 (s, 1H), 7.76 (s, 1H), 7.20 (m, 1H), 6.95 (bs, 1H), 3.91 (m, 4H), 2.70 (m, 4H), 2.52 (m, 12H), 1.93 (m, 4H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD):165.8, 162.5, 161.5, 146.5, 143.8, 141.9, 133.2, 128.7, 125.5, 125.1, 123.4, 123.1, 121.9, 120.5, 97.8, 57.9, 45.2, 45.1, 39.9, 39.7, 26.0, 25.9. Anal. Calcd for C<sub>29</sub>H<sub>32</sub>BrCl<sub>2</sub>N<sub>7</sub>O<sub>4</sub>: C, 50.23; H, 4.65; Br, 11.52; Cl, 10.22; N, 14.14; O, 9.23. Found: C, 50.17; H, 4.71; N, 14.08.</div></div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Synthesis of <b>6</b> and <b>7</b></h4><div class="NLM_p">NDI <b>1</b> (0.5 mmol) was dissolved into 40 mL of CH<sub>3</sub>CN in a round-bottom flask together with 1.0 mmol of 3,4-diaminobenzoic acid, the mixture was stirred at 85 °C for 72 h under argon. The resulting dark-violet solution was cooled at r.t. to induce precipitation of the product. The crude powder was filtered and purified by HPLC chromatography, (CH<sub>3</sub>CN:H<sub>2</sub>O 0.1%TFA) according to analytical method A and preparative method B, affording the title NDIs <b>6</b> (27%) and <b>7</b> (30.5%).</div><div id="sec4_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> <b>6</b>·2HCl</h5><div class="NLM_p last">Yield = 27%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 7.86 (s, 2H), 7.62 (d, <i>J</i> = 8.0 Hz, 1H), 7.39 (s, 1H), 6.89 (d, <i>J</i> = 8.0 Hz, 1H), 4.16 (m, 4H), 3.26 (m, 4H), 2.95 (s, 12H), 2.16 (m, 4H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD):168.2, 166.6, 166.5, 164.3, 164.2, 143.1, 142.8, 132.6, 129.8, 129.2, 128.6, 127.0, 126.7, 126.6, 126.4, 123.6, 123.5, 119.0, 117.7, 98.3, 97.7, 57.0, 43.8, 40.7, 38.4; 24.8. Anal. Calcd for C<sub>31</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>6</sub>: C, 56.63; H, 5.21; Cl, 10.78; N, 12.78; O, 14.60. Found: C, 56.77; H, 5.31; N, 12.72.</div></div><div id="sec4_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> <b>7</b>·2HCl</h5><div class="NLM_p last">Yield = 30.5%; dark-violet solid; mp dec >200 °C. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): 8.37 (s, 2H), 7.85 (d, <i>J</i> = 8.0 Hz, 1H), 7.73 (s, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 4.35 (m, 4H), 3.41 (m, 4H), 3.05 (s, 12H), 2.29 (m, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD): 168.3, 166.2, 163.5, 161.8, 161.7, 160.8, 152.2, 133.2, 132.6, 130.1, 129.4, 126.5, 123.4, 123.3, 121.6, 119.12, 118.21, 117.9, 98.3, 97.6, 57.04, 43.9, 38.3, 38.5, 24.8, 24.7. Anal. Calcd for C<sub>31</sub>H<sub>33</sub>BrCl<sub>2</sub>N<sub>6</sub>O<sub>6</sub>: C, 50.56; H, 4.52; N, 11.41; O, 13.03. Found: C, 50.34; H, 4.61; N, 11.29</div></div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Microwave Assisted (MW) Nucleophilic Aromatic Substitution: Synthesis of <b>9</b></h4><div class="NLM_p last">Compound <b>3</b> (0.5 mmol) was dissolved into 4 mL of <i>N</i>,<i>N</i>-dimethyl-propan-1,3-diamine. The mixture was stirred and heated in a microwave reactor, according to a closed vessel protocol, at 150 °C, 250 psi, 200 W, for 3 min. The resulting green solution was cooled at rt to induce precipitation of the product. The resulting deep-green powder was filtered and washed by water afforded <b>1</b> (Yield 92%). Further HPLC preparative purification (CH<sub>3</sub>CN:H<sub>2</sub>O and 0.1% CF<sub>3</sub>COOH as eluent), and final trifluoroacetate-chloride anion exchange, by addition of 1 mL HCl 1M, yielded <b>9</b> as trihydrochloride (<b>9</b>·3HCl); deep-green solid; mp dec >200 °C. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): δ = 7.10 (m, 1H), 6.96 (m, 2H), 6.69 (m, 2H), 4.11 (m, 4H), 3.42 (m, 2H), 3.26 (m, 6H), 2.99 (s, 6H), 2.93 (s, 12H), 2.18 (bs, 2H), 2.11 (bs, 4H). <sup>13</sup>C NMR (75 MHz, CD<sub>3</sub>OD): δ = 166.7, 166.4, 164.5, 144.2, 139.7, 129.8, 129.0, 128.9, 127.5, 126.6, 125.0, 117.6, 117.1, 115.8, 108.2, 104.5, 98.1, 97.7, 63.4, 57.0, 54.0, 53.8, 43.8, 41.1, 38.5, 38.1, 25.7, 24.9, 24.8, 24.1. Anal. Calcd for C<sub>35</sub>H<sub>47</sub>Cl<sub>3</sub>N<sub>8</sub>O<sub>4</sub>: C, 56.04; H, 6.32; Cl, 14.18; N, 14.94; O, 8.53. Found: C, 56.17; H, 6.39; N, 14.90.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Exhaustive Methylation: Synthesis of <b>10</b> and <b>11</b></h4><div class="NLM_p">The <b>c-exNDIs</b> (<b>2</b> or <b>3</b>) purified as hydrochlorides were dissolved in a NaHCO<sub>3</sub> solution and extracted 3 times with CH<sub>2</sub>Cl<sub>2</sub>. The recovered organic layers have been dried on Na<sub>2</sub>SO<sub>4</sub> and the solvent evaporated under reduced pressure. The collected amine (2.5 mmol) was suspended in 50 mL of CHCl<sub>3</sub> and 1.2 g (8.5 mmol) of CH<sub>3</sub>I were added. This suspension was stirred under nitrogen atmosphere, at room temperature for 12 h. After this period the resulting dark-violet powder was filtered and dried, afforded the product as pure di-iodide salt.</div><div id="sec4_2_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> <b>10</b></h5><div class="NLM_p last">Yield = 99%; dark-violet crystals; mp dec >200 °C. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ = 7.94 (s, 2H), 7.12 (dd, <i>J</i> = 3.3, 6.0 Hz, 2H), 6.97 (dd, <i>J</i> = 3.3, 6.0 Hz, 2H), 4.14 (m, 4H), 3.49 (m, 4H), 3.15 (m, 18H), 2.21 (m, 4H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): δ = 166.9, 165.8, 143.8, 129.1, 128.1, 127.6, 126.3, 123.5, 118.3, 97.5, 66.1, 54.1, 38.6, 23.6. Anal. Calcd for C<sub>32</sub>H<sub>38</sub>I<sub>2</sub>N<sub>6</sub>O<sub>4</sub>: C, 46.62; H, 4.65; I, 30.78; N, 10.19; O, 7.76. Found: C, 46.53; H, 4.59; N, 10.22.</div></div><div id="sec4_2_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> <b>11</b></h5><div class="NLM_p last">Yield = 99%; dark-violet crystals; mp dec >200 °C. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ = 8.04 (s, 1H), 7.15 (bs, 2H), 7.00 (bs, 2H), 4.12 (m, 4H), 3.51 (m, 4H), 3.15 (s, 18H), 2.20 (bs, 2H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD): δ = 166.3, 165.9, 163.2, 162.2, 145.1, 144.1, 143.0, 132.5, 130.1, 128.8, 128.7, 128.3, 128.2, 126.4, 123.0, 121.1, 120.2, 118.3, 118.2, 96.9, 65.8, 53.9, 38.8, 38.6, 23.3, 23.2. Anal. Calcd for C<sub>32</sub>H<sub>37</sub>BrI<sub>2</sub>N<sub>6</sub>O<sub>4</sub>: C, 42.55; H, 4.13; Br, 8.84; I, 28.09; N, 9.30; O, 7.08. Found: C, 42.60; H, 4.18; N, 9.39.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Spectroscopic Techniques (FRET and CD)</h3><div class="NLM_p">All oligonucleotides used in this study were from Sigma-Aldrich (Milan, Italy). For fluorescence melting curves, equimolar amounts of the compounds were added to each FAM (6-carboxyfluorescein) 5′-end- and TAMRA (6-carboxy-tetramethylrhodamine) 3′-end-labeled oligonucleotide (0.25 uM) folded in the lithium cacodylate buffer supplemented with potassium (100 mM). After stabilization at 4 °C, samples were processed in a LightCycler II (Roche, Milan, Italy) or LightCycler 480 (Roche, Milan, Italy), and the oligonucleotide melting was monitored by observing 6-carboxyfluorescein (6-FAM) emission in the temperature range of 30– 95 °C with 1 °C/min gradient. Melting profiles were normalized as previously described.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a><i>T</i><sub>m</sub> was defined as the temperature corresponding to the 0.5 fraction of the normalized fluorescence.</div><div class="NLM_p last">For CD analysis, oligonucleotides were diluted to a final concentration of 4 μM in lithium cacodylate buffer (10 mM, pH 7.4) and KCl 100 mM. After annealing step (95 °C for 5 min), DNA samples were gradually cooled to room temperature and compounds added from stock at final concentration of 16 μM. CD spectra were recorded on a Chirascan-Plus (Applied Photophysisics, Leatherhead, UK) equipped with a Peltier temperature controller using a quartz cell of 5 mm optical path length and an instrument scanning speed of 50 nm/min over a wavelength range of 230–320 nm. The reported spectrum of each sample represents the average of 2 scans is baseline-corrected for signal contributions due to the buffer. Observed ellipticities were converted to mean residue ellipticity (θ) = deg × cm<sup>2</sup> × dmol<sup>–1</sup> (mol ellip). For the determination of <i>T</i><sub>m</sub>, spectra were recorded over a temperature range of 20–95 °C, with temperature increase of 5 °C/min. <i>T</i><sub>m</sub> values were calculated according to the van’t Hoff equation, applied for a two-state transition from a folded to unfolded state, assuming that the heat capacity of the folded and unfolded states are equal.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>Taq</i> Polymerase Stop Assay</h3><div class="NLM_p last"><i>Taq</i> polymerase stop assay was carried out as previously described.<a onclick="showRef(event, 'ref27 ref30'); return false;" href="javascript:void(0);" class="ref ref27 ref30">(27, 30)</a> Briefly, the 5′-end labeled primer was annealed to its template (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">Table S1</a>) in lithium cacodylate buffer in the presence or absence of KCl 100 mM and by heating at 95 °C for 5 min and gradually cooling to room temperature. Where specified, samples were incubated with <b>2</b> (3.1–25.0 nM). Primer extension was conducted with 2 U of AmpliTaq Gold DNA polymerase (Applied Biosystem, Carlsbad, California, USA) at 47 °C for 30 min. Reactions were stopped by ethanol precipitation, primer extension products were separated on a 15% denaturing gel, and finally visualized by phosphorimaging (Typhoon FLA 9000).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Mass Spectrometric (MS) Competition Assay</h3><div class="NLM_p">Oligonucleotides were heat-denatured and folded in 10 mM KCl, 150 mM trimethylammonium acetate (TMAA), pH 6.8, overnight at room temperature. The oligonucleotides where diluted to final concentration of 10 μM and incubated with the tested compound at ratio DNA:compound 1:1 overnight at 4 °C.</div><div class="NLM_p last">Samples were analyzed by direct infusion electrospray ionization (ESI) on a Xevo G2-XS QTof mass spectrometer (Waters, Manchester, UK). The injection was automatically performed by an Agilent 1290 Infinity HPLC (Agilent Technologies, Santa Clara, CA, US) equipped with an auto sampler; the carrying buffer was TMAA 100 mM. The absence of potassium in the carrying buffer allows the dilution of the salts before MS analysis decreasing the signal suppression. Up to 5 μL samples were typically injected for each analysis. The electrospray capillary was at 1.8 kV, the source and desolvation temperatures were 45 and 65 °C, respectively, and the sampling cone was at 65 V. All these parameters ensured minimal fragmentation of the DNAs complexes. The instrument was calibrated using a 2 mg/mL solution of sodium iodide in 50% 2-propanol. Additionally, the use of the LockSpray during the analysis provided a typical <2 ppm mass accuracy. The internal standard LockSpray consisted in a solution of leu-enkephalin 1 μg/mL in acetonitrile/water (50:50, v/v) containing 0.1% formic acid. Binding affinities were calculated for each experiment using the reconstructed-ion chromatogram area for each species calculated by MassLynx V4.1: this analysis was made possible by the experimental design that used an HPLC system to inject the samples in the mass spectrometer. The binding affinity was calculated with the following formula: [BA = (ΣG4<sub>b</sub>/(ΣG4<sub>f</sub> + ΣG4<sub>b</sub>)) × 100], where BA is the binding affinity, G4<sub>b</sub> is chromatogram area of bound G4 DNA, and G4<sub>f</sub> is the chromatogram area of free G4 DNA.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> G4<sub>b</sub> comprises DNA with one or two bound ligands, where present. Signals with charge states 6<sup>–</sup>, 5<sup>–</sup>, and 4<sup>–</sup> were used for the free and bound hTel DNA and c-myc, charge states 6<sup>–</sup> and 5<sup>–</sup> for LTR-III, charge states 5<sup>–</sup> and 4<sup>–</sup> for c-kit2, and charge state 4<sup>–</sup> for LTR-IV. Free DNA mono and dipotassium adducts were included in the calculation. For stoichiometric calculation, oligonucleotides were folded in 10 mM KCl, 150 mM TMAA pH 6.8 overnight at rt. The oligonucleotides, diluted to final concentration of 10 μM, were incubated with the tested compound (ratio 1:5, 1:10, 1:20, and 1:50) at 4 °C and analyzed by ESI-MS.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> SPR Analysis</h3><div class="NLM_p last">SPR was performed on the Biacore T100 platform (GE Healthcare). 5′-Biotinylated LTR-III, LTR-IV, and hTel oligonucleotides were heated at 95 °C for 5 min and cooled down at room temperature to allow G4 folding. Immobilization was performed in HEPES-KCl running buffer (0.01 M HEPES pH 7.4, 0.2 M KCl, 3 mM EDTA) on a steptavidin coated surface (SA sensor chip, Biacore). Oligonucleotides were diluted in HEPES-KCl running buffer to a concentration of 30 nM and injected to reach the response of around 500 RU. Flow cell 1 was left empty to allow reference subtraction. <b>C-exNDIs</b> binding analysis was performed at a flow rate of 30 μL/min, with contact time of 280 s and dissociation time of 360 s in HEPES-KCl buffer. Sensorgrams were obtained in the concentration range of 25–200 nM. After each compound injection the chip surface was regenerated with glycine 10 mM pH 2.0 solution (GE Healthcare). All sensorgrams were corrected by buffer injection response. Data were fitted to a 1:1 binding model with <i>R</i><sub>max</sub> initial parameter set to theoretical calculated <i>R</i><sub>max</sub> using BIAevaluation software (GE Healthcare). All experiments were performed independently at least twice, and in each instance χ<sup>2</sup> values, which indicate the goodness of fitting, were below 0.2.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> CL-Mediated Footprinting</h3><div class="NLM_p last">All oligonucleotides were gel-purified before use and prepared in desalted/lyophilized form. Oligonucleotides were 5′-end-labeled with [γ- <sup>32</sup>P]ATP by T4 polynucleotide kinase and purified by MicroSpin G-25 columns (GE Healthcare, Europe). They were next resuspended in lithium cacodylate 10 mM, pH 7.4, and KCl 100 mM, heat-denatured, and folded. Reactions of the labeled G-quadruplex folded oligonucleotides (4 pmol/sample) with increasing amounts of compound <b>2</b> (4–100 nM) were performed at 20 °C for 24 h in resuspension buffer. Samples were then reacted with CL (100 μM) at 37 °C for 24 h. Reactions were stopped by ethanol precipitation, resuspended, and either kept on ice or treated at 90 °C for 30 min with 1 M piperidine to complete strand scission according to the Maxam and Gilbert protocol. Samples were then lyophilized, resuspended in formamide gel loading buffer, and heated at 95 °C for 3 min. Reaction products were analyzed on 20% denaturing polyacrylamide gels and visualized by Typhoon FLA 9000 phosphorimaging analysis (GE Healthcare).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Reporter Assays</h3><div class="NLM_p">The HIV-1 promoter activity in live cells in the presence of <b>2</b> was monitored in HeLa-Tat-III/LTR/d1EGFP cells (obtained through the NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH, from Dr. Satoh). This cell line derives from HeLa-tat-III and is transfected with d1EGFP under the control of HIV-1 LTR promoter. Then 8 ×10<sup>4</sup> of HeLa-Tat-III/LTR/d1EGFP cells were seeded in a 12-well plate in 1 mL of DMEM/10% FBS medium supplemented with Geneticin selective antibiotic (1 mg/mL, Life Technologies, Monza, Italy) and incubated overnight. Cells were next treated with increasing concentrations of <b>2</b> (0–62.5–125–250 nM) and incubated at 37 °C. After 48 h, cells were washed with PBS 1× and resuspended in 500 μL of PBS 1×. To evaluate the mean of EGFP fluorescence, a total of 10000 events were acquired for each sample with an LRS 2 instrument using FACS DIVA Software (BD Bioscience) and analyzed with Flow Jo (Tree Star). Cell debris and dead cells were excluded from the analysis by gating on forward scatter (FSC) versus side scatter (SSC) and were quantified to estimate the cytotoxicity.</div><div class="NLM_p last">For the luciferase assay, HIV-1 LTR region (wt or M4 + 5) was inserted into the promoterless luciferase reporter vector pGL4.10-Luc2 (Promega Italia, Italy), as previously reported.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> For assessment of luciferase activity, human embryonic kidney 293T (HEK293T) cells were seeded in a 24-well plate (9 × 10<sup>4</sup> cells/well). Then 24 h later, cells were transfected with 100 ng/well of pGL4.10-LTRwt or of pGL4.10-LTR-M4 + 5 using TransIT-293 transfection reagent (Mirus Bio LLC, Madison, WI, USA), according to the manufacturer’s protocol. After 1 h, cells were treated with <b>2</b> for 24 h at various concentrations. Luciferase activity was measured using the britelite plus Reporter Gene Assay System (PerkinElmer Inc., Milan, Italy) at a Victor X2 multilabel plate reader (PerkinElmer Inc., Milan, Italy), according to the manufacturer’s instructions. Cells were lysed in RIPA buffer (50 mM Tris-HCl pH 7.2, 150 mM NaCl, 1% Igepal, 0.1% SDS) and protein concentration was determined by BCA assay (Thermo Scientific Pierce, Monza, Italy). Luciferase signals were subsequently normalized to total protein content, according to the manufacturer’s protocol (<a href="http://ita.promega.com/~/pdf/resources/pubhub/cellnotes/normalizing-genetic-reporter-assays/" class="extLink">http://ita.promega.com/~/pdf/resources/pubhub/cellnotes/normalizing-genetic-reporter-assays/</a>).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Antiviral Assay in HIV-1 Infected TZM-bl Cells</h3><div class="NLM_p last">HIV-1 infectivity was measured using the TZM-bl reporter cell line (obtained through the NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH, from Dr. J.C. Kappes, Dr. X. Wu, and Tranzyme Inc.). TZM-bl is a HeLa cell line stably expressing large amounts of CD4 and CCR5 and containing integrated copies of the luciferase and β-galactosidase genes under control of the HIV-1 promoter. TZM-bl were grown in DMEM supplemented with 10% FBS. Cells (1 ×10<sup>4</sup> cells/well) were seeded in 96-well plates and grown overnight to permit adherence prior to treatment and viral infection. Cells were next infected with HIV-1 NL4-3 strain or BaL strain at a MOI of 0.5, treated with serial dilutions of tested compounds, and incubated at 37 °C. After 48 h, cells were washed with PBS 1× and HIV-1 production was assessed following the LTR-luciferase signal using the britelite plus Reporter Gene Assay System (PerkinElmer, Waltman, MA, USA) according to the manufacturer’s protocol.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Cytotoxicity Assay</h3><div class="NLM_p last">Cytotoxicity of tested compounds was tested in parallel to antiviral and reporter assays by MTT. The 50% cytotoxic concentration (CC<sub>50</sub>) was defined as the concentration of test compound that was able to reduce the absorbance of the mock-infected cells by 50%. The 50% inhibitory concentration (IC<sub>50</sub>) was defined as the concentration of test compound that inhibit 50% of LTR-Luciferase signal. The selectivity index (SI) is the relative effectiveness of the tested compound in inhibiting viral replication compared to inducing cell death (CC<sub>50</sub> value/IC<sub>50</sub> value). The therapeutic window (TW) is the concentration range at which the compound shows antiviral activity paralleled by less than 80% cytotoxicity.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> HIV-1 Entry Assay</h3><div class="NLM_p last">TZM-bl cells (1 × 10<sup>4</sup> cells/well) were seeded in 96-well plates and grown overnight to permit adherence prior to treatment and viral infection. Cells were next infected with HIV-1 NL4-3 (MOI 0.5) and incubated at 37 °C. Compound <b>2</b> (50–100 nM) and the reference compounds DS (100 μg/mL, Sigma-Aldrich) and AZT (0.5 μg/mL, Sigma-Aldrich) were added at different hours (0, 1, 2 h) pi. After 31 h, cells were washed with PBS 1× and HIV-1 production was assessed following the LTR-luciferase signal using the britelite plus Reporter Gene Assay System (PerkinElmer, Waltman, MA, USA) according to the manufacturer’s protocol.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Computational Method</h3><div class="NLM_p last">DFT calculations were performed with the Gaussian 09, revision B.01 software package.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Gas-phase geometry optimizations were carried out using the B3LYP functional,<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> with 6-31+G(d,p) basis set for all the atoms.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01283">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63864" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63864" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01283" class="ext-link">10.1021/acs.jmedchem.5b01283</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures illustrating all oligonucleotides used in this study, CD spectra, and the complete ESI/MS competition analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf">jm5b01283_si_001.pdf (35.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_002.csv">jm5b01283_si_002.csv (2.07 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01283" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40405" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40405" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mauro Freccero</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Pavia, V.le Taramelli 10, 27100 Pavia, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#35585440475a1b534750565650475a75405b5c45431b5c41"><span class="__cf_email__" data-cfemail="86ebe7f3f4e9a8e0f4e3e5e5e3f4e9c6f3e8eff6f0a8eff2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sara N. Richter</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fd8e9c8f9cd38f949e9589988fbd8893948d99d39489"><span class="__cf_email__" data-cfemail="cab9abb8abe4b8a3a9a2beafb88abfa4a3baaee4a3be">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosalba Perrone</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Filippo Doria</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Pavia, V.le Taramelli 10, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Butovskaya</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ilaria Frasson</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Silvia Botti</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Pavia, V.le Taramelli 10, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matteo Scalabrin</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sara Lago</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vincenzo Grande</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Pavia, V.le Taramelli 10, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matteo Nadai</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, University of Padua, via Gabelli 63, 35121 Padua, Italy</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.P., F.D., and E.B. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d688e2940-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86141" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86141" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Bill and Melinda Gates Foundation (GCE grant numbers OPP1035881, OPP1097238) to S.N.R., the European Research Council (ERC Consolidator grant 615879) to S.N.R. and M.F., and the Italian Ministry of University and Research (FIRB-Ideas RBID082ATK) to S.N.R. and M.F. We thank Prof. Manlio Palumbo, University of Padua, for helpful comments on the manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i42" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i42"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i43" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i43"> Abbreviations Used</h2><tr><td class="NLM_term">G4</td><td class="NLM_def"><p class="first last">G-quadruplex</p></td></tr><tr><td class="NLM_term">NDI</td><td class="NLM_def"><p class="first last">naphthalene diimide</p></td></tr><tr><td class="NLM_term">c-exNDI</td><td class="NLM_def"><p class="first last">core-extended NDI</p></td></tr><tr><td class="NLM_term">LTR</td><td class="NLM_def"><p class="first last">long terminal repeat</p></td></tr><tr><td class="NLM_term">ds</td><td class="NLM_def"><p class="first last">double-stranded</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>m</sub></td><td class="NLM_def"><p class="first last">melting temperature</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 70 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Lipps, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D.</span><span> </span><span class="NLM_article-title">G-quadruplex structures: in vivo evidence and function</span> <span class="citation_source-journal">Trends Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1016/j.tcb.2009.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.tcb.2009.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19589679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=414-422&author=H.+J.+Lippsauthor=D.+Rhodes&title=G-quadruplex+structures%3A+in+vivo+evidence+and+function&doi=10.1016%2Fj.tcb.2009.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex structures: in vivo evidence and function</span></div><div class="casAuthors">Lipps, Hans J.; Rhodes, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although many biochem. and structural studies have demonstrated that DNA sequences contg. runs of adjacent guanines spontaneously fold into G-quadruplex DNA structures in vitro, only recently has evidence started to accumulate for their presence and function in vivo.  Genome-wide analyses have revealed that functional genomic regions from highly divergent organisms are enriched in DNA sequences with G-quadruplex-forming potential, suggesting that G-quadruplexes could provide a nucleic-acid-based mechanism for regulating telomere maintenance, as well as transcription, replication and translation.  Here, we review recent studies aimed at uncovering the in vivo presence and function of G-quadruplexes in genomes and RNA, with a particular focus on telomeric G-quadruplexes and how their formation and resoln. is regulated to permit telomere synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5cRHGDFqMhrVg90H21EOLACvtfcHk0lhtEvAGZMZq4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO&md5=f9f684e81a34d37ec1ae44ef7764c112</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2009.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2009.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DLipps%26aufirst%3DH.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%26atitle%3DG-quadruplex%2520structures%253A%2520in%2520vivo%2520evidence%2520and%2520function%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D414%26epage%3D422%26doi%3D10.1016%2Fj.tcb.2009.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, W.</span><span> </span><span class="NLM_article-title">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">366</span><span class="refDoi"> DOI: 10.1038/334364a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2F334364a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=3393228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=1988&pages=364-366&author=D.+Senauthor=W.+Gilbert&title=Formation+of+parallel+four-stranded+complexes+by+guanine-rich+motifs+in+DNA+and+its+implications+for+meiosis&doi=10.1038%2F334364a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span></div><div class="casAuthors">Sen, Dipankar; Gilbert, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6180</span>),
    <span class="NLM_cas:pages">364-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Single-stranded DNA contg. short guanine-rich motifs will self-assoc. at physiol. salt concns. to make 4-stranded structures in which the strands run in parallel fashion.  These complexes are apparently held together by guanines bonded to each other by Hoogsteen pairing.  It is speculated that this self-recognition of guanine-rich motifs of DNA serves to bring together, and to zipper up in register, the 4 homologous chromatids during meiosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr5smJJlAP6rVg90H21EOLACvtfcHk0lgd3DjnaqE0CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D&md5=96183a2f707c4395862b96c750a0c53c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F334364a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F334364a0%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DW.%26atitle%3DFormation%2520of%2520parallel%2520four-stranded%2520complexes%2520by%2520guanine-rich%2520motifs%2520in%2520DNA%2520and%2520its%2520implications%2520for%2520meiosis%26jtitle%3DNature%26date%3D1988%26volume%3D334%26spage%3D364%26epage%3D366%26doi%3D10.1038%2F334364a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span> </span><span class="NLM_article-title">G-quadruplex nucleic acids and human disease</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">3470</span><span class="NLM_x">–</span> <span class="NLM_lpage">3488</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2010.07760.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1111%2Fj.1742-4658.2010.07760.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=20670277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=3470-3488&author=Y.+Wuauthor=R.+M.+Brosh&title=G-quadruplex+nucleic+acids+and+human+disease&doi=10.1111%2Fj.1742-4658.2010.07760.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex nucleic acids and human disease</span></div><div class="casAuthors">Wu, Yuliang; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3470-3488</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Alternate DNA structures that deviate from B-form double-stranded DNA such as G-quadruplex (G4) DNA can be formed by sequences that are widely distributed throughout the human genome.  G-quadruplex secondary structures, formed by the stacking of planar quartets composed of four guanines that interact by Hoogsteen hydrogen bonding, can affect cellular DNA replication and transcription, and influence genomic stability.  The unique metab. of G-rich chromosomal regions that potentially form quadruplexes may influence a no. of biol. processes including Ig gene rearrangements, promoter activation and telomere maintenance.  A no. of human diseases are characterized by telomere defects, and it is proposed that G-quadruplex structures which form at telomere ends play an important role in telomere stability.  Evidence from cellular studies and model organisms suggests that diseases with known defects in G4 DNA helicases are likely to be perturbed in telomere maintenance and cellular DNA replication.  In this minireview, we discuss the connections of G-quadruplex nucleic acids to human genetic diseases and cancer based on the recent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbqNRG82SQcbVg90H21EOLACvtfcHk0lgd3DjnaqE0CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO&md5=47fd16aa812516caf20b0b46f3544301</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2010.07760.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2010.07760.x%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DG-quadruplex%2520nucleic%2520acids%2520and%2520human%2520disease%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D3470%26epage%3D3488%26doi%3D10.1111%2Fj.1742-4658.2010.07760.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, W.</span><span> </span><span class="NLM_article-title">A sodium-potassium switch in the formation of four-stranded G4-DNA</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">344</span><span class="NLM_x">, </span> <span class="NLM_fpage">410</span><span class="NLM_x">–</span> <span class="NLM_lpage">414</span><span class="refDoi"> DOI: 10.1038/344410a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2F344410a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=2320109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaK3cXitlWnsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=1990&pages=410-414&author=D.+Senauthor=W.+Gilbert&title=A+sodium-potassium+switch+in+the+formation+of+four-stranded+G4-DNA&doi=10.1038%2F344410a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A sodium-potassium switch in the formation of four-stranded G4-DNA</span></div><div class="casAuthors">Sen, Dipankar; Gilbert, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">6265</span>),
    <span class="NLM_cas:pages">410-14</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Single-stranded complex guanine-rich DNA sequences from chromosomal telomeres and elsewhere can assoc. to form stable parallel 4-stranded structures termed G4-DNA by a process that is anomalously dependent on the particular alkali metal cation that is present.  The anomaly, which is not found in the formation of G4-DNA by oligonucleotides contg. short, single runs of three or more guanines, is caused by K+ excessively stabilizing fold-back intermediate structures, or pathway byproducts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJT86WNyFCnrVg90H21EOLACvtfcHk0lgny9jouw4qcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitlWnsL4%253D&md5=bbd66d0e94a0a6b4abf65e9ab85f59f0</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F344410a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F344410a0%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DW.%26atitle%3DA%2520sodium-potassium%2520switch%2520in%2520the%2520formation%2520of%2520four-stranded%2520G4-DNA%26jtitle%3DNature%26date%3D1990%26volume%3D344%26spage%3D410%26epage%3D414%26doi%3D10.1038%2F344410a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Taylor, J. P.</span><span> </span><span class="NLM_article-title">Neurodegenerative diseases: G-quadruplex poses quadruple threat</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1038/nature13067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnature13067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2014&pages=175-177&author=J.+P.+Taylor&title=Neurodegenerative+diseases%3A+G-quadruplex+poses+quadruple+threat&doi=10.1038%2Fnature13067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature13067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13067%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26atitle%3DNeurodegenerative%2520diseases%253A%2520G-quadruplex%2520poses%2520quadruple%2520threat%26jtitle%3DNature%26date%3D2014%26volume%3D507%26spage%3D175%26epage%3D177%26doi%3D10.1038%2Fnature13067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Campbell, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">G-quadruplexes and metal ions</span> <span class="citation_source-journal">Met. Ions Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">119</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1007/978-94-007-2172-2_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1007%2F978-94-007-2172-2_4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22210337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWjtLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=119-134&author=N.+H.+Campbellauthor=S.+Neidle&title=G-quadruplexes+and+metal+ions&doi=10.1007%2F978-94-007-2172-2_4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes and metal ions</span></div><div class="casAuthors">Campbell, Nancy H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Metal Ions in Life Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">Interplay between Metal Ions and Nucleic Acids</span>),
    <span class="NLM_cas:pages">119-134</span>CODEN:
                <span class="NLM_cas:coden">MILSCT</span>;
        ISSN:<span class="NLM_cas:issn">1559-0836</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Metal ions stabilize quadruplex nucleic acids by coordinating the O6 guanine atoms from G-quartets.  These quartets form the basic motif of quadruplex structures.  This article systematically surveys the available crystallog. data on native quadruplexes, their ligand complexes and (in one instance) a protein complex.  Three categories of quadruplex are examd., tetramol., bimol., and intramol.: all are formed by telomeric nucleic acid sequences from human or ciliate organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIHVWG6zI41LVg90H21EOLACvtfcHk0lgny9jouw4qcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWjtLzF&md5=f49615280560f3483cf9c3beda0a03cf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2F978-94-007-2172-2_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-94-007-2172-2_4%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DN.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DG-quadruplexes%2520and%2520metal%2520ions%26jtitle%3DMet.%2520Ions%2520Life%2520Sci.%26date%3D2012%26volume%3D10%26spage%3D119%26epage%3D134%26doi%3D10.1007%2F978-94-007-2172-2_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Eddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maizels, N.</span><span> </span><span class="NLM_article-title">Gene function correlates with potential for G4 DNA formation in the human genome</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">3887</span><span class="NLM_x">–</span> <span class="NLM_lpage">3896</span><span class="refDoi"> DOI: 10.1093/nar/gkl529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=16914419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=3887-3896&author=J.+Eddyauthor=N.+Maizels&title=Gene+function+correlates+with+potential+for+G4+DNA+formation+in+the+human+genome&doi=10.1093%2Fnar%2Fgkl529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Gene function correlates with potential for G4 DNA formation in the human genome</span></div><div class="casAuthors">Eddy, Johanna; Maizels, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3887-3896</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">G-rich genomic regions can form G4 DNA upon transcription or replication.  We have quantified the potential for G4 DNA formation (G4P) of the 16 654 genes in the human RefSeq database, and then correlated gene function with G4P.  We have found that very low and very high G4P correlates with specific functional classes of genes.  Notably, tumor suppressor genes have very low G4P and proto-oncogenes have very high G4P.  G4P of these genes is evenly distributed between exons and introns, and it does not reflect enrichment for CpG islands or local chromosomal environment.  These results show that genomic structure undergoes selection based on gene function.  Selection based on G4P could promote genomic stability (or instability) of specific classes of genes; or reflect mechanisms for global regulation of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPf7ajy8A2MLVg90H21EOLACvtfcHk0ljBEHDWpNtndQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yis7o%253D&md5=c5da07ec9e667a0fbb1843159e773eb8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl529%26sid%3Dliteratum%253Aachs%26aulast%3DEddy%26aufirst%3DJ.%26aulast%3DMaizels%26aufirst%3DN.%26atitle%3DGene%2520function%2520correlates%2520with%2520potential%2520for%2520G4%2520DNA%2520formation%2520in%2520the%2520human%2520genome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D3887%26epage%3D3896%26doi%3D10.1093%2Fnar%2Fgkl529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Huppert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">G-quadruplexes in promoters throughout the human genome</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">406</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1093/nar/gkl1057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl1057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=17169996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOls74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=406-413&author=J.+L.+Huppertauthor=S.+Balasubramanian&title=G-quadruplexes+in+promoters+throughout+the+human+genome&doi=10.1093%2Fnar%2Fgkl1057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes in promoters throughout the human genome</span></div><div class="casAuthors">Huppert, Julian L.; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">406-413</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Certain G-rich DNA sequences readily form four-stranded structures called G-quadruplexes.  These sequence motifs are located in telomeres as a repeated unit, and elsewhere in the genome, where their function is currently unknown.  It has been proposed that G-quadruplexes may be directly involved in gene regulation at the level of transcription.  In support of this hypothesis, we show that the promoter regions (1 kb upstream of the transcription start site TSS) of genes are significantly enriched in quadruplex motifs relative to the rest of the genome, with >40% of human gene promoters contg. one or more quadruplex motif.  Furthermore, these promoter quadruplexes strongly assoc. with nuclease hypersensitive sites identified throughout the genome via biochem. measurement.  Regions of the human genome that are both nuclease hypersensitive and within promoters show a remarkable (230-fold) enrichment of quadruplex elements, compared to the rest of the genome.  These quadruplex motifs identified in promoter regions also show an interesting structural bias towards more stable forms.  These observations support the proposal that promoter G-quadruplexes are directly involved in the regulation of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofaKOSq_qjKrVg90H21EOLACvtfcHk0ljBEHDWpNtndQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOls74%253D&md5=626903207653e1c97d568b9b50c093bd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl1057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl1057%26sid%3Dliteratum%253Aachs%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DG-quadruplexes%2520in%2520promoters%2520throughout%2520the%2520human%2520genome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D406%26epage%3D413%26doi%3D10.1093%2Fnar%2Fgkl1057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Nakken, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hovig, E.</span><span> </span><span class="NLM_article-title">The disruptive positions in human G-quadruplex motifs are less polymorphic and more conserved than their neutral counterparts</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">–</span> <span class="NLM_lpage">5756</span><span class="refDoi"> DOI: 10.1093/nar/gkp590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkp590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=5749-5756&author=S.+Nakkenauthor=T.+Rognesauthor=E.+Hovig&title=The+disruptive+positions+in+human+G-quadruplex+motifs+are+less+polymorphic+and+more+conserved+than+their+neutral+counterparts&doi=10.1093%2Fnar%2Fgkp590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp590%26sid%3Dliteratum%253Aachs%26aulast%3DNakken%26aufirst%3DS.%26aulast%3DRognes%26aufirst%3DT.%26aulast%3DHovig%26aufirst%3DE.%26atitle%3DThe%2520disruptive%2520positions%2520in%2520human%2520G-quadruplex%2520motifs%2520are%2520less%2520polymorphic%2520and%2520more%2520conserved%2520than%2520their%2520neutral%2520counterparts%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D5749%26epage%3D5756%26doi%3D10.1093%2Fnar%2Fgkp590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2536</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1093/nar/gkl286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=16687659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2536-2549&author=S.+Cogoiauthor=L.+E.+Xodo&title=G-quadruplex+formation+within+the+promoter+of+the+KRAS+proto-oncogene+and+its+effect+on+transcription&doi=10.1093%2Fnar%2Fgkl286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2536-2549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In human and mouse, the promoter of the KRAS gene contains a nuclease hypersensitive polypurine-polypyrimidine element (NHPPE) that is essential for transcription.  An interesting feature of the polypurine G-rich strand of NHPPE is its ability to assume an unusual DNA structure that, according to CD and DMS footprinting expts., is attributed to an intramol. parallel G-quadruplex, consisting of three G-tetrads and three loops.  The human and mouse KRAS NHPPE G-rich strands display melting temp. of 64° and 73°, resp., as well as a K+-dependent capacity to arrest DNA polymerase.  Photocleavage and CD expts. showed that the cationic porphyrin TMPyP4 stacks to the external G-tetrads of the KRAS quadruplexes, increasing the Tm by ∼20°.  These findings raise the intriguing question that the G-quadruplex formed within the NHPPE of KRAS may be involved in the regulation of transcription.  Indeed, transfection expts. showed that the activity of the mouse KRAS promoter is reduced to 20% of control, in the presence of the quadruplex-stabilizing TMPyP4.  In addn., we found that G-rich oligonucleotides mimicking the KRAS quadruplex, but not the corresponding 4-base mutant sequences or oligonucleotides forming quadruplexes with different structures, competed with the NHPPE duplex for binding to nuclear proteins.  When vector pKRS-413, contg. CAT driven by the mouse KRAS promoter, and KRAS quadruplex oligonucleotides were co-transfected in 293 cells, the expression of CAT was found to be downregulated to 40% of the control.  On the basis of these data, we propose that the NHPPE of KRAS exists in equil. between a double-stranded form favoring transcription and a folded quadruplex form, which instead inhibits transcription.  Such a mechanism, which is probably adopted by other growth-related genes, provides useful hints for the rational design of anticancer drugs against the KRAS oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfPpw1J4eI7Vg90H21EOLACvtfcHk0ljBEHDWpNtndQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D&md5=b6b7c93fab3ea76e200f3bd21bd61f36</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl286%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG-quadruplex%2520formation%2520within%2520the%2520promoter%2520of%2520the%2520KRAS%2520proto-oncogene%2520and%2520its%2520effect%2520on%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D2536%26epage%3D2549%26doi%3D10.1093%2Fnar%2Fgkl286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Eddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maizels, N.</span><span> </span><span class="NLM_article-title">Conserved elements with potential to form polymorphic G-quadruplex structures in the first intron of human genes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1321</span><span class="NLM_x">–</span> <span class="NLM_lpage">1333</span><span class="refDoi"> DOI: 10.1093/nar/gkm1138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkm1138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=18187510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislKiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=1321-1333&author=J.+Eddyauthor=N.+Maizels&title=Conserved+elements+with+potential+to+form+polymorphic+G-quadruplex+structures+in+the+first+intron+of+human+genes&doi=10.1093%2Fnar%2Fgkm1138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved elements with potential to form polymorphic G-quadruplex structures in the first intron of human genes</span></div><div class="casAuthors">Eddy, Johanna; Maizels, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1321-1333</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">To understand how potential for G-quadruplex formation might influence regulation of gene expression, we examd. the 2 kb spanning the transcription start sites (TSS) of the 18 217 human RefSeq genes, distinguishing contributions of template and nontemplate strands.  Regions both upstream and downstream of the TSS are G-rich, but the downstream region displays a clear bias toward G-richness on the nontemplate strand.  Upstream of the TSS, much of the G-richness and potential for G-quadruplex formation derives from the presence of well-defined canonical regulatory motifs in duplex DNA, including CpG dinucleotides which are sites of regulatory methylation, and motifs recognized by the transcription factor SP1.  This challenges the notion that quadruplex formation upstream of the TSS contributes to regulation of gene expression.  Downstream of the TSS, G-richness is concd. in the first intron, and on the nontemplate strand, where polymorphic sequence elements with potential to form G-quadruplex structures and which cannot be accounted for by known regulatory motifs are found in almost 3000 (16%) of the human RefSeq genes, and are conserved through frogs.  These elements could in principle be recognized either as DNA or as RNA, providing structural targets for regulation at the level of transcription or RNA processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPNriy5euURLVg90H21EOLACvtfcHk0ljFqFxsvS4L-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislKiur0%253D&md5=44f6ad32cdd8cd28c25cbf3d16b17ad2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm1138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm1138%26sid%3Dliteratum%253Aachs%26aulast%3DEddy%26aufirst%3DJ.%26aulast%3DMaizels%26aufirst%3DN.%26atitle%3DConserved%2520elements%2520with%2520potential%2520to%2520form%2520polymorphic%2520G-quadruplex%2520structures%2520in%2520the%2520first%2520intron%2520of%2520human%2520genes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D1321%26epage%3D1333%26doi%3D10.1093%2Fnar%2Fgkm1138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Verma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basundra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">Evidence of genome-wide G4 DNA-mediated gene expression in human cancer cells</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">4194</span><span class="NLM_x">–</span> <span class="NLM_lpage">4204</span><span class="refDoi"> DOI: 10.1093/nar/gkn1076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkn1076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19211664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptFGhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=4194-4204&author=A.+Vermaauthor=V.+K.+Yadavauthor=R.+Basundraauthor=A.+Kumarauthor=S.+Chowdhury&title=Evidence+of+genome-wide+G4+DNA-mediated+gene+expression+in+human+cancer+cells&doi=10.1093%2Fnar%2Fgkn1076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence of genome-wide G4 DNA-mediated gene expression in human cancer cells</span></div><div class="casAuthors">Verma, Anjali; Yadav, Vinod Kumar; Basundra, Richa; Kumar, Akinchan; Chowdhury, Shantanu</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4194-4204</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Guanine-rich DNA of a particular sequence adopts four-stranded structural forms known as G-quadruplex or G4 DNA.  Though in vitro formation of G4 DNA is known for several years, in vivo presence of G4 DNA was only recently noted in eukaryote telomeres.  Recent bioinformatics analyses showing prevalence of G4 DNA within promoters of human and related species seems to implicate G4 DNA in a genome-wide cis-regulatory role.  Herein we demonstrate that G4 DNA may present regulatory sites on a genome-wide scale by showing widespread effect on gene expression in response to the established intracellular G4 DNA-binding ligands.  This is particularly relevant to genes that harbor conserved potential G4 DNA (PG4 DNA) forming sequence across human, mouse and rat promoters of orthologous genes.  Genes with conserved PG4 DNA in promoters show co-regulated expression in 79 human and 61 mouse normal tissues (z-score > 3.5; P < 0.0001).  Conservation of G4 DNA across related species also emphasizes the biol. importance of G4 DNA and its role in transcriptional regulation of genes; shedding light on a relatively novel mechanism of regulation of gene expression in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8IUDB3VYGJLVg90H21EOLACvtfcHk0ljFqFxsvS4L-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptFGhtLg%253D&md5=56c58ede4a7c91710440d731f5bf5fca</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn1076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn1076%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DBasundra%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DEvidence%2520of%2520genome-wide%2520G4%2520DNA-mediated%2520gene%2520expression%2520in%2520human%2520cancer%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D4194%26epage%3D4204%26doi%3D10.1093%2Fnar%2Fgkn1076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Mani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">Genome-wide analyses of recombination prone regions predict role of DNA structural motif in recombination</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e4399</span><span class="refDoi"> DOI: 10.1371/journal.pone.0004399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.pone.0004399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19198658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A280%3ADC%252BD1M7is1CntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e4399&author=P.+Maniauthor=V.+K.+Yadavauthor=S.+K.+Dasauthor=S.+Chowdhury&title=Genome-wide+analyses+of+recombination+prone+regions+predict+role+of+DNA+structural+motif+in+recombination&doi=10.1371%2Fjournal.pone.0004399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide analyses of recombination prone regions predict role of DNA structural motif in recombination</span></div><div class="casAuthors">Mani Prithvi; Yadav Vinod Kumar; Das Swapan Kumar; Chowdhury Shantanu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e4399</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">HapMap findings reveal surprisingly asymmetric distribution of recombinogenic regions.  Short recombinogenic regions (hotspots) are interspersed between large relatively non-recombinogenic regions.  This raises the interesting possibility of DNA sequence and/or other cis- elements as determinants of recombination.  We hypothesized the involvement of non-canonical sequences that can result in local non-B DNA structures and tested this using the G-quadruplex DNA as a model.  G-quadruplex or G4 DNA is a unique form of four-stranded non-B DNA structure that engages certain G-rich sequences, presence of such motifs has been noted within telomeres.  In support of this hypothesis, genome-wide computational analyses presented here reveal enrichment of potential G4 (PG4) DNA forming sequences within 25618 human hotspots relative to 9290 coldspots (p<0.0001).  Furthermore, co-occurrence of PG4 DNA within several short sequence elements that are associated with recombinogenic regions was found to be significantly more than randomly expected.  Interestingly, analyses of more than 50 DNA binding factors revealed that co-occurrence of PG4 DNA with target DNA binding sites of transcription factors c-Rel, NF-kappa B (p50 and p65) and Evi-1 was significantly enriched in recombination-prone regions.  These observations support involvement of G4 DNA in recombination, predicting a functional model that is consistent with duplex-strand separation induced by formation of G4 motifs in supercoiled DNA and/or when assisted by other cellular factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQw2fvA1PJyb-_Q9URG6Dx8fW6udTcc2eZVVGtZr1_16rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M7is1CntA%253D%253D&md5=d3d3f4b1effd792cf676b983a8152000</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0004399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0004399%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DP.%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DDas%26aufirst%3DS.%2BK.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DGenome-wide%2520analyses%2520of%2520recombination%2520prone%2520regions%2520predict%2520role%2520of%2520DNA%2520structural%2520motif%2520in%2520recombination%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De4399%26doi%3D10.1371%2Fjournal.pone.0004399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Haeusler, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donnelly, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Periz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simko, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troncoso, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothstein, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">C9orf72 nucleotide repeat structures initiate molecular cascades of disease</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1038/nature13124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnature13124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24598541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktV2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2014&pages=195-200&author=A.+R.+Haeuslerauthor=C.+J.+Donnellyauthor=G.+Perizauthor=E.+A.+Simkoauthor=P.+G.+Shawauthor=M.+S.+Kimauthor=N.+J.+Maragakisauthor=J.+C.+Troncosoauthor=A.+Pandeyauthor=R.+Sattlerauthor=J.+D.+Rothsteinauthor=J.+Wang&title=C9orf72+nucleotide+repeat+structures+initiate+molecular+cascades+of+disease&doi=10.1038%2Fnature13124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">C9orf72 nucleotide repeat structures initiate molecular cascades of disease</span></div><div class="casAuthors">Haeusler, Aaron R.; Donnelly, Christopher J.; Periz, Goran; Simko, Eric A. J.; Shaw, Patrick G.; Kim, Min-Sik; Maragakis, Nicholas J.; Troncoso, Juan C.; Pandey, Akhilesh; Sattler, Rita; Rothstein, Jeffrey D.; Wang, Jiou</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">507</span>
        (<span class="NLM_cas:issue">7491</span>),
    <span class="NLM_cas:pages">195-200</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A hexanucleotide repeat expansion (HRE), (GGGGCC)n, in C9orf72 is the most common genetic cause of the neurodegenerative diseases amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD).  Here we identify a mol. mechanism by which structural polymorphism of the HRE leads to ALS/FTD pathol. and defects.  The HRE forms DNA and RNA G-quadruplexes with distinct structures and promotes RNA•DNA hybrids (R-loops).  The structural polymorphism causes a repeat-length-dependent accumulation of transcripts aborted in the HRE region.  These transcribed repeats bind to ribonucleoproteins in a conformation-dependent manner.  Specifically, nucleolin, an essential nucleolar protein, preferentially binds the HRE G-quadruplex, and patient cells show evidence of nucleolar stress.  Our results demonstrate that distinct C9orf72 HRE structural polymorphism at both DNA and RNA levels initiates mol. cascades leading to ALS/FTD pathologies, and provide the basis for a mechanistic model for repeat-assocd. neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvSEy_yaVaoLVg90H21EOLACvtfcHk0liEGTh43JdlgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktV2rt7s%253D&md5=cbe8956df515e57926dac896af764c74</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature13124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13124%26sid%3Dliteratum%253Aachs%26aulast%3DHaeusler%26aufirst%3DA.%2BR.%26aulast%3DDonnelly%26aufirst%3DC.%2BJ.%26aulast%3DPeriz%26aufirst%3DG.%26aulast%3DSimko%26aufirst%3DE.%2BA.%26aulast%3DShaw%26aufirst%3DP.%2BG.%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DMaragakis%26aufirst%3DN.%2BJ.%26aulast%3DTroncoso%26aufirst%3DJ.%2BC.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DSattler%26aufirst%3DR.%26aulast%3DRothstein%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DC9orf72%2520nucleotide%2520repeat%2520structures%2520initiate%2520molecular%2520cascades%2520of%2520disease%26jtitle%3DNature%26date%3D2014%26volume%3D507%26spage%3D195%26epage%3D200%26doi%3D10.1038%2Fnature13124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ivanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Day, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emara, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, P.</span><span> </span><span class="NLM_article-title">G-quadruplex structures contribute to the neuroprotective effects of angiogenin-induced tRNA fragments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">18201</span><span class="NLM_x">–</span> <span class="NLM_lpage">18206</span><span class="refDoi"> DOI: 10.1073/pnas.1407361111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.1407361111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18201-18206&author=P.+Ivanovauthor=E.+O%E2%80%99Dayauthor=M.+M.+Emaraauthor=G.+Wagnerauthor=J.+Liebermanauthor=P.+Anderson&title=G-quadruplex+structures+contribute+to+the+neuroprotective+effects+of+angiogenin-induced+tRNA+fragments&doi=10.1073%2Fpnas.1407361111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407361111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407361111%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Day%26aufirst%3DE.%26aulast%3DEmara%26aufirst%3DM.%2BM.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DLieberman%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DP.%26atitle%3DG-quadruplex%2520structures%2520contribute%2520to%2520the%2520neuroprotective%2520effects%2520of%2520angiogenin-induced%2520tRNA%2520fragments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D18201%26epage%3D18206%26doi%3D10.1073%2Fpnas.1407361111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Sket, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohleven, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovanda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stalekar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zupunski, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zalar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plavec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogelj, B.</span><span> </span><span class="NLM_article-title">Characterization of DNA G-quadruplex species forming from C9ORF72 G4C2-expanded repeats associated with amyotrophic lateral sclerosis and frontotemporal lobar degeneration</span> <span class="citation_source-journal">Neurobiol. Aging</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1016/j.neurobiolaging.2014.09.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.neurobiolaging.2014.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25442110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslelsr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1091-1096&author=P.+Sketauthor=J.+Pohlevenauthor=A.+Kovandaauthor=M.+Stalekarauthor=V.+Zupunskiauthor=M.+Zalarauthor=J.+Plavecauthor=B.+Rogelj&title=Characterization+of+DNA+G-quadruplex+species+forming+from+C9ORF72+G4C2-expanded+repeats+associated+with+amyotrophic+lateral+sclerosis+and+frontotemporal+lobar+degeneration&doi=10.1016%2Fj.neurobiolaging.2014.09.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of DNA G-quadruplex species forming from C9ORF72 G4C2-expanded repeats associated with amyotrophic lateral sclerosis and frontotemporal lobar degeneration</span></div><div class="casAuthors">Sket, Primoz; Pohleven, Jure; Kovanda, Anja; Stalekar, Maja; Zupunski, Vera; Zalar, Matja; Plavec, Janez; Rogelj, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Aging</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">NEAGDO</span>;
        ISSN:<span class="NLM_cas:issn">0197-4580</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The G4C2 hexanucleotide repeat expansion, located in the first intron of the C9ORF72 gene, represents a major genetic hallmark of amyotrophic lateral sclerosis and frontotemporal lobar degeneration.  Several hypotheses have been proposed on how the transcribed repeat RNA leads to the development of amyotrophic lateral sclerosis and frontotemporal lobar degeneration.  However, despite their importance, factors affecting the transcription of expanded-repeat RNA are not well known.  As transcription is dependent on the DNA contg. the expanded repeats, it is crucial to understand its structure.  G-quadruplexes are known to affect expression on the level of DNA, therefore whether they form on the expanded-repeat DNA constitutes an important biol. question.  Using NMR and CD spectroscopy we show that DNA G4C2 with varying no. of repeats d(G4C2)n form planar guanine quartets characteristic of G-quadruplexes.  Addnl., we show DNA G-quadruplexes can form inter- and intra-molecularly in either parallel or anti-parallel orientation, based on d(G4C2) sequence length.  This potential structural heterogeneity of longer disease-relevant repeats should therefore be taken into account when studying their role in disease pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm1F5TYUAtvbVg90H21EOLACvtfcHk0lg80kD_VYNOZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslelsr7F&md5=abf45124f3e9098a4eaa7a1acf7a6f2a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.neurobiolaging.2014.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neurobiolaging.2014.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DSket%26aufirst%3DP.%26aulast%3DPohleven%26aufirst%3DJ.%26aulast%3DKovanda%26aufirst%3DA.%26aulast%3DStalekar%26aufirst%3DM.%26aulast%3DZupunski%26aufirst%3DV.%26aulast%3DZalar%26aufirst%3DM.%26aulast%3DPlavec%26aufirst%3DJ.%26aulast%3DRogelj%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520DNA%2520G-quadruplex%2520species%2520forming%2520from%2520C9ORF72%2520G4C2-expanded%2520repeats%2520associated%2520with%2520amyotrophic%2520lateral%2520sclerosis%2520and%2520frontotemporal%2520lobar%2520degeneration%26jtitle%3DNeurobiol.%2520Aging%26date%3D2015%26volume%3D36%26spage%3D1091%26epage%3D1096%26doi%3D10.1016%2Fj.neurobiolaging.2014.09.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Fisette, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montagna, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mihailescu, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, M. S.</span><span> </span><span class="NLM_article-title">A G-rich element forms a G-quadruplex and regulates BACE1 mRNA alternative splicing</span> <span class="citation_source-journal">J. Neurochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">773</span><span class="refDoi"> DOI: 10.1111/j.1471-4159.2012.07680.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1111%2Fj.1471-4159.2012.07680.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22303960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGltrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2012&pages=763-773&author=J.+F.+Fisetteauthor=D.+R.+Montagnaauthor=M.+R.+Mihailescuauthor=M.+S.+Wolfe&title=A+G-rich+element+forms+a+G-quadruplex+and+regulates+BACE1+mRNA+alternative+splicing&doi=10.1111%2Fj.1471-4159.2012.07680.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A G-rich element forms a G-quadruplex and regulates BACE1 mRNA alternative splicing</span></div><div class="casAuthors">Fisette, Jean-Francois; Montagna, Daniel R.; Mihailescu, Mihaela-Rita; Wolfe, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">5&6</span>),
    <span class="NLM_cas:pages">763-773</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">β-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is the transmembrane aspartyl protease that catalyzes the first cleavage step in the proteolysis of the APP to the amyloid β-protein (Aβ), a process involved in the pathogenesis of Alzheimer disease.  BACE1 pre-mRNA undergoes complex alternative splicing, the regulation of which is not well understood.  We identified a G-rich sequence within exon 3 of BACE1 involved in controlling splice site selection.  Mutation of the G-rich sequence decreased use of the normal 5' splice site of exon 3, which leads to full-length and proteolytically active BACE1, and increased use of an alternative splice site, which leads to a shorter, essentially inactive isoform.  Nuclease protection assays, NMR, and CD spectroscopy revealed that this sequence folds into a G-quadruplex structure.  Several proteins were identified as capable of binding to the G-rich sequence, and one of these, heterogeneous nuclear ribonucleoprotein H, was found to regulate BACE1 exon 3 alternative splicing and in a manner dependent on the G-rich sequence.  Knockdown of heterogeneous nuclear ribonucleoprotein H led to a decrease in the full-length BACE1 mRNA isoform as well as a decrease in Aβ prodn. from APP, suggesting new possibilities for therapeutic approaches to Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp19qAAvW5AMrVg90H21EOLACvtfcHk0lg80kD_VYNOZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGltrvL&md5=2d3e2873e4cc643f7b501ad77499b4c7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2012.07680.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2012.07680.x%26sid%3Dliteratum%253Aachs%26aulast%3DFisette%26aufirst%3DJ.%2BF.%26aulast%3DMontagna%26aufirst%3DD.%2BR.%26aulast%3DMihailescu%26aufirst%3DM.%2BR.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DA%2520G-rich%2520element%2520forms%2520a%2520G-quadruplex%2520and%2520regulates%2520BACE1%2520mRNA%2520alternative%2520splicing%26jtitle%3DJ.%2520Neurochem.%26date%3D2012%26volume%3D121%26spage%3D763%26epage%3D773%26doi%3D10.1111%2Fj.1471-4159.2012.07680.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Fry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loeb, L. A.</span><span> </span><span class="NLM_article-title">The fragile X syndrome d(CGG)n nucleotide repeats form a stable tetrahelical structure</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">4950</span><span class="NLM_x">–</span> <span class="NLM_lpage">4954</span><span class="refDoi"> DOI: 10.1073/pnas.91.11.4950</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.91.11.4950" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=4950-4954&author=M.+Fryauthor=L.+A.+Loeb&title=The+fragile+X+syndrome+d%28CGG%29n+nucleotide+repeats+form+a+stable+tetrahelical+structure&doi=10.1073%2Fpnas.91.11.4950"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.11.4950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.11.4950%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DM.%26aulast%3DLoeb%26aufirst%3DL.%2BA.%26atitle%3DThe%2520fragile%2520X%2520syndrome%2520d%2528CGG%2529n%2520nucleotide%2520repeats%2520form%2520a%2520stable%2520tetrahelical%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1994%26volume%3D91%26spage%3D4950%26epage%3D4954%26doi%3D10.1073%2Fpnas.91.11.4950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatto, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span> </span><span class="NLM_article-title">The evolving world of protein-G-quadruplex recognition: a medicinal chemist’s perspective</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">1219</span><span class="NLM_x">–</span> <span class="NLM_lpage">1230</span><span class="refDoi"> DOI: 10.1016/j.biochi.2011.04.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.biochi.2011.04.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21549174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpt1OgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2011&pages=1219-1230&author=C.+Sissiauthor=B.+Gattoauthor=M.+Palumbo&title=The+evolving+world+of+protein-G-quadruplex+recognition%3A+a+medicinal+chemist%E2%80%99s+perspective&doi=10.1016%2Fj.biochi.2011.04.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The evolving world of protein-G-quadruplex recognition: A medicinal chemist's perspective</span></div><div class="casAuthors">Sissi, Claudia; Gatto, Barbara; Palumbo, Manlio</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1219-1230</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The physiol. and pharmacol. role of nucleic acids structures folded into the non canonical G-quadruplex conformation have recently emerged.  Their activities are targeted at vital cellular processes including telomere maintenance, regulation of transcription and processing of the pre-messenger or telomeric RNA.  In addn., severe conditions like cancer, fragile X syndrome, Bloom syndrome, Werner syndrome and Fanconi anemia J are related to genomic defects that involve G-quadruplex forming sequences.  In this connection G-quadruplex recognition and processing by nucleic acid directed proteins and enzymes represents a key event to activate or deactivate physiol. or pathol. pathways.  In this review we examine protein-G-quadruplex recognition in physiol. significant conditions and discuss how to possibly exploit the interactions' selectivity for targeted therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojiEYxee9yTrVg90H21EOLACvtfcHk0liDaA5qX8717Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpt1OgsLs%253D&md5=f9f93b292b845626ef8f65c3e33edddc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2011.04.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2011.04.018%26sid%3Dliteratum%253Aachs%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DGatto%26aufirst%3DB.%26aulast%3DPalumbo%26aufirst%3DM.%26atitle%3DThe%2520evolving%2520world%2520of%2520protein-G-quadruplex%2520recognition%253A%2520a%2520medicinal%2520chemist%25E2%2580%2599s%2520perspective%26jtitle%3DBiochimie%26date%3D2011%26volume%3D93%26spage%3D1219%26epage%3D1230%26doi%3D10.1016%2Fj.biochi.2011.04.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0liDaA5qX8717Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Henderson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chavez, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lansdorp, P. M.</span><span> </span><span class="NLM_article-title">Detection of G-quadruplex DNA in mammalian cells</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">860</span><span class="NLM_x">–</span> <span class="NLM_lpage">869</span><span class="refDoi"> DOI: 10.1093/nar/gkt957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkt957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24163102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlShsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=860-869&author=A.+Hendersonauthor=Y.+Wuauthor=Y.+C.+Huangauthor=E.+A.+Chavezauthor=J.+Plattauthor=F.+B.+Johnsonauthor=R.+M.+Broshauthor=D.+Senauthor=P.+M.+Lansdorp&title=Detection+of+G-quadruplex+DNA+in+mammalian+cells&doi=10.1093%2Fnar%2Fgkt957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of G-quadruplex DNA in mammalian cells</span></div><div class="casAuthors">Henderson, Alexander; Wu, Yuliang; Huang, Yu Chuan; Chavez, Elizabeth A.; Platt, Jesse; Johnson, F. Brad; Brosh, Robert M., Jr.; Sen, Dipankar; Lansdorp, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">860-869</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It has been proposed that guanine-rich DNA forms four-stranded structures in vivo called G-quadruplexes or G4 DNA.  G4 DNA has been implicated in several biol. processes, but tools to study G4 DNA structures in cells are limited.  Here we report the development of novel murine monoclonal antibodies specific for different G4 DNA structures.  We show that one of these antibodies designated 1H6 exhibits strong nuclear staining in most human and murine cells.  Staining intensity increased on treatment of cells with agents that stabilize G4 DNA and, strikingly, cells deficient in FANCJ, a G4 DNA-specific helicase, showed stronger nuclear staining than controls.  Our data strongly support the existence of G4 DNA structures in mammalian cells and indicate that the abundance of such structures is increased in the absence of FANCJ.  We conclude that monoclonal antibody 1H6 is a valuable tool for further studies on the role of G4 DNA in cell and mol. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrli0PqZbZA3bVg90H21EOLACvtfcHk0liDaA5qX8717Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlShsrw%253D&md5=a16e8096cfaa3f7d8d7d045294a6fa3f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt957%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%2BC.%26aulast%3DChavez%26aufirst%3DE.%2BA.%26aulast%3DPlatt%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DF.%2BB.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26aulast%3DSen%26aufirst%3DD.%26aulast%3DLansdorp%26aufirst%3DP.%2BM.%26atitle%3DDetection%2520of%2520G-quadruplex%2520DNA%2520in%2520mammalian%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D860%26epage%3D869%26doi%3D10.1093%2Fnar%2Fgkt957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amrane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Litvak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreola, M. L.</span><span> </span><span class="NLM_article-title">G-quadruplexes in viruses: function and potential therapeutic applications</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">12352</span><span class="NLM_x">–</span> <span class="NLM_lpage">12366</span><span class="refDoi"> DOI: 10.1093/nar/gku999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgku999" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=12352-12366&author=M.+Metifiotauthor=S.+Amraneauthor=S.+Litvakauthor=M.+L.+Andreola&title=G-quadruplexes+in+viruses%3A+function+and+potential+therapeutic+applications&doi=10.1093%2Fnar%2Fgku999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku999%26sid%3Dliteratum%253Aachs%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DAmrane%26aufirst%3DS.%26aulast%3DLitvak%26aufirst%3DS.%26aulast%3DAndreola%26aufirst%3DM.%2BL.%26atitle%3DG-quadruplexes%2520in%2520viruses%253A%2520function%2520and%2520potential%2520therapeutic%2520applications%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D12352%26epage%3D12366%26doi%3D10.1093%2Fnar%2Fgku999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Murat, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lekieffre, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowieson, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clancy, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preiss, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tellam, J.</span><span> </span><span class="NLM_article-title">G-quadruplexes regulate Epstein-Barr virus-encoded nuclear antigen 1 mRNA translation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio.1479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnchembio.1479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24633353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlCktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=358-364&author=P.+Muratauthor=J.+Zhongauthor=L.+Lekieffreauthor=N.+P.+Cowiesonauthor=J.+L.+Clancyauthor=T.+Preissauthor=S.+Balasubramanianauthor=R.+Khannaauthor=J.+Tellam&title=G-quadruplexes+regulate+Epstein-Barr+virus-encoded+nuclear+antigen+1+mRNA+translation&doi=10.1038%2Fnchembio.1479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes regulate Epstein-Barr virus-encoded nuclear antigen 1 mRNA translation</span></div><div class="casAuthors">Murat, Pierre; Zhong, Jie; Lekieffre, Lea; Cowieson, Nathan P.; Clancy, Jennifer L.; Preiss, Thomas; Balasubramanian, Shankar; Khanna, Rajiv; Tellam, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Viruses that establish latent infections have evolved unique mechanisms to avoid host immune recognition.  Maintenance proteins of these viruses regulate their synthesis to levels sufficient for maintaining persistent infection but below threshold levels for host immune detection.  The mechanisms governing this finely tuned regulation of viral latency are unknown.  Here the authors show that mRNAs encoding gammaherpesviral maintenance proteins contain within their open reading frames clusters of unusual structural elements, G-quadruplexes, which are responsible for the cis-acting regulation of viral mRNA translation.  By studying the Epstein-Barr virus-encoded nuclear antigen 1 (EBNA1) mRNA, the authors show that destabilization of G-quadruplexes using antisense oligonucleotides increases EBNA1 mRNA translation.  In contrast, pretreatment with a G-quadruplex-stabilizing small mol., pyridostatin, decreases EBNA1 synthesis, highlighting the importance of G-quadruplexes within virally encoded transcripts as unique regulatory signals for translational control and immune evasion.  Furthermore, these findings suggest alternative therapeutic strategies focused on targeting RNA structure within viral ORFs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppFLmSVv6UerVg90H21EOLACvtfcHk0lgZVJBr1Bw0Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlCktrs%253D&md5=7681817a80afde9bc604b4b7e0dba0a1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1479%26sid%3Dliteratum%253Aachs%26aulast%3DMurat%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DLekieffre%26aufirst%3DL.%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DClancy%26aufirst%3DJ.%2BL.%26aulast%3DPreiss%26aufirst%3DT.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DR.%26aulast%3DTellam%26aufirst%3DJ.%26atitle%3DG-quadruplexes%2520regulate%2520Epstein-Barr%2520virus-encoded%2520nuclear%2520antigen%25201%2520mRNA%2520translation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio.1479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Norseen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, P. M.</span><span> </span><span class="NLM_article-title">Role for G-quadruplex RNA binding by Epstein-Barr virus nuclear antigen 1 in DNA replication and metaphase chromosome attachment</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">10336</span><span class="NLM_x">–</span> <span class="NLM_lpage">10346</span><span class="refDoi"> DOI: 10.1128/JVI.00747-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1128%2FJVI.00747-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19656898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlClsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=10336-10346&author=J.+Norseenauthor=F.+B.+Johnsonauthor=P.+M.+Lieberman&title=Role+for+G-quadruplex+RNA+binding+by+Epstein-Barr+virus+nuclear+antigen+1+in+DNA+replication+and+metaphase+chromosome+attachment&doi=10.1128%2FJVI.00747-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Role for G-quadruplex RNA binding by Epstein-Barr virus nuclear antigen 1 in DNA replication and metaphase chromosome attachment</span></div><div class="casAuthors">Norseen, Julie; Johnson, F. Brad; Lieberman, Paul M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">10336-10346</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Latent infection by Epstein-Barr virus (EBV) requires both replication and maintenance of the viral genome.  EBV nuclear antigen 1 (EBNA1) is a virus-encoded protein that is crit. for the replication and maintenance of the genome during latency in proliferating cells.  The authors have previously demonstrated that EBNA1 recruits the cellular origin recognition complex (ORC) through an RNA-dependent interaction with EBNA1 linking region 1 (LR1) and LR2.  The authors now show that LR1 and LR2 bind to G-rich RNA that is predicted to form G-quadruplex structures.  Several chem. distinct G-quadruplex-interacting drugs disrupted the interaction between EBNA1 and ORC.  The G-quadruplex-interacting compd. BRACO-19 inhibited EBNA1-dependent stimulation of viral DNA replication and preferentially blocked proliferation of EBV-pos. cells relative to EBV-neg. cell lines.  BRACO-19 treatment also disrupted the ability of EBNA1 to tether to metaphase chromosomes, suggesting that maintenance function is also mediated through G-quadruplex recognition.  These findings suggest that the EBNA1 replication and maintenance function uses a common G-quadruplex binding capacity of LR1 and LR2, which may be targetable by small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqiD6hHzR-KrVg90H21EOLACvtfcHk0lgZVJBr1Bw0Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlClsbjK&md5=ec64ae4a8182b80c92a50f1f215ac053</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FJVI.00747-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00747-09%26sid%3Dliteratum%253Aachs%26aulast%3DNorseen%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DF.%2BB.%26aulast%3DLieberman%26aufirst%3DP.%2BM.%26atitle%3DRole%2520for%2520G-quadruplex%2520RNA%2520binding%2520by%2520Epstein-Barr%2520virus%2520nuclear%2520antigen%25201%2520in%2520DNA%2520replication%2520and%2520metaphase%2520chromosome%2520attachment%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D10336%26epage%3D10346%26doi%3D10.1128%2FJVI.00747-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollati, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusov, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesters, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgenfeld, R.</span><span> </span><span class="NLM_article-title">The SARS-unique domain (SUD) of SARS coronavirus contains two macrodomains that bind G-quadruplexes</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000428</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1000428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.ppat.1000428" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000428&author=J.+Tanauthor=C.+Vonrheinauthor=O.+S.+Smartauthor=G.+Bricogneauthor=M.+Bollatiauthor=Y.+Kusovauthor=G.+Hansenauthor=J.+R.+Mestersauthor=C.+L.+Schmidtauthor=R.+Hilgenfeld&title=The+SARS-unique+domain+%28SUD%29+of+SARS+coronavirus+contains+two+macrodomains+that+bind+G-quadruplexes&doi=10.1371%2Fjournal.ppat.1000428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000428%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DSmart%26aufirst%3DO.%2BS.%26aulast%3DBricogne%26aufirst%3DG.%26aulast%3DBollati%26aufirst%3DM.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DSchmidt%26aufirst%3DC.%2BL.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DThe%2520SARS-unique%2520domain%2520%2528SUD%2529%2520of%2520SARS%2520coronavirus%2520contains%2520two%2520macrodomains%2520that%2520bind%2520G-quadruplexes%26jtitle%3DPLoS%2520Pathog.%26date%3D2009%26volume%3D5%26spage%3De1000428%26doi%3D10.1371%2Fjournal.ppat.1000428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poe, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frasson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithgall, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Formation of a unique cluster of G-quadruplex structures in the HIV-1 Nef coding region: implications for antiviral activity</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e73121</span><span class="refDoi"> DOI: 10.1371/journal.pone.0073121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.pone.0073121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24015290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e73121&author=R.+Perroneauthor=M.+Nadaiauthor=J.+A.+Poeauthor=I.+Frassonauthor=M.+Palumboauthor=G.+Paluauthor=T.+E.+Smithgallauthor=S.+N.+Richter&title=Formation+of+a+unique+cluster+of+G-quadruplex+structures+in+the+HIV-1+Nef+coding+region%3A+implications+for+antiviral+activity&doi=10.1371%2Fjournal.pone.0073121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of a unique cluster of G-quadruplex structures in the HIV-1 nef coding region: implications for antiviral activity</span></div><div class="casAuthors">Perrone, Rosalba; Nadai, Matteo; Poe, Jerrod A.; Frasson, Ilaria; Palumbo, Manlio; Palu, Giorgio; Smithgall, Thomas E.; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e73121</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">G-quadruplexes are tetraplex structures of nucleic acids that can form in G-rich sequences.  Their presence and functional role have been established in telomeres, oncogene promoters and coding regions of the human chromosome.  In particular, they have been proposed to be directly involved in gene regulation at the level of transcription.  Because the HIV-1 Nef protein is a fundamental factor for efficient viral replication, infectivity and pathogenesis in vitro and in vivo, we investigated G-quadruplex formation in the HIV-1 nef gene to assess the potential for viral inhibition through G-quadruplex stabilization.  A comprehensive computational anal. of the nef coding region of available strains showed the presence of three conserved sequences that were uniquely clustered.  Biophys. testing proved that G-quadruplex conformations were efficiently stabilized or induced by G-quadruplex ligands in all three sequences.  Upon incubation with a G-quadruplex ligand, Nef expression was reduced in a reporter gene assay and Nef-dependent enhancement of HIV-1 infectivity was significantly repressed in an antiviral assay.  These data constitute the first evidence of the possibility to regulate HIV-1 gene expression and infectivity through G-quadruplex targeting and therefore open a new avenue for viral treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNzHbrQ-pu97Vg90H21EOLACvtfcHk0lhzRdWwTA2D3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrjP&md5=ec3e3ebcb1e84ceac70c4b60c6476e40</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0073121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0073121%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DPoe%26aufirst%3DJ.%2BA.%26aulast%3DFrasson%26aufirst%3DI.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DSmithgall%26aufirst%3DT.%2BE.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DFormation%2520of%2520a%2520unique%2520cluster%2520of%2520G-quadruplex%2520structures%2520in%2520the%2520HIV-1%2520Nef%2520coding%2520region%253A%2520implications%2520for%2520antiviral%2520activity%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De73121%26doi%3D10.1371%2Fjournal.pone.0073121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frasson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poe, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butovskaya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithgall, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">A dynamic G-quadruplex region regulates the HIV-1 long terminal repeat promoter</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">6521</span><span class="NLM_x">–</span> <span class="NLM_lpage">6530</span><span class="refDoi"> DOI: 10.1021/jm400914r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400914r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFeru7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6521-6530&author=R.+Perroneauthor=M.+Nadaiauthor=I.+Frassonauthor=J.+A.+Poeauthor=E.+Butovskayaauthor=T.+E.+Smithgallauthor=M.+Palumboauthor=G.+Paluauthor=S.+N.+Richter&title=A+dynamic+G-quadruplex+region+regulates+the+HIV-1+long+terminal+repeat+promoter&doi=10.1021%2Fjm400914r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Dynamic G-Quadruplex Region Regulates the HIV-1 Long Terminal Repeat Promoter</span></div><div class="casAuthors">Perrone, Rosalba; Nadai, Matteo; Frasson, Ilaria; Poe, Jerrod A.; Butovskaya, Elena; Smithgall, Thomas E.; Palumbo, Manlio; Palu, Giorgio; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6521-6530</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-Quadruplexes, noncanonical nucleic acid structures, act as silencers in the promoter regions of human genes; putative G-quadruplex forming sequences are also present in promoters of other mammals, yeasts, and prokaryotes.  Here we show that also the HIV-1 LTR promoter exploits G-quadruplex-mediated transcriptional regulation with striking similarities to eukaryotic promoters and that treatment with a G-quadruplex ligand inhibits HIV-1 infectivity.  Computational anal. on 953 HIV-1 strains substantiated a highly conserved G-rich sequence corresponding to Sp1 and NF-κB binding sites.  Biophys./biochem. anal. proved that two mutually exclusive parallel-like intramol. G-quadruplexes, stabilized by small mol. ligands, primarily fold in this region.  Mutations disrupting G-quadruplex formation enhanced HIV promoter activity in cells, whereas treatment with a G-quadruplex ligand impaired promoter activity and displayed antiviral effects.  These findings disclose the possibility of inhibiting the HIV-1 LTR promoter by G-quadruplex-interacting small mols., providing a new pathway to development of anti-HIV-1 drugs with unprecedented mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog3AJbPgRilrVg90H21EOLACvtfcHk0ljUkx5fdkFezw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFeru7bP&md5=2ea00d98b5ffef840e1bc80027b35b89</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm400914r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400914r%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DFrasson%26aufirst%3DI.%26aulast%3DPoe%26aufirst%3DJ.%2BA.%26aulast%3DButovskaya%26aufirst%3DE.%26aulast%3DSmithgall%26aufirst%3DT.%2BE.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DA%2520dynamic%2520G-quadruplex%2520region%2520regulates%2520the%2520HIV-1%2520long%2520terminal%2520repeat%2520promoter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6521%26epage%3D6530%26doi%3D10.1021%2Fjm400914r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Amrane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerkour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedrat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vialet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreola, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mergny, J. L.</span><span> </span><span class="NLM_article-title">Topology of a DNA G-quadruplex structure formed in the HIV-1 promoter: a potential target for anti-HIV drug development</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">5249</span><span class="NLM_x">–</span> <span class="NLM_lpage">5252</span><span class="refDoi"> DOI: 10.1021/ja501500c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja501500c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=5249-5252&author=S.+Amraneauthor=A.+Kerkourauthor=A.+Bedratauthor=B.+Vialetauthor=M.+L.+Andreolaauthor=J.+L.+Mergny&title=Topology+of+a+DNA+G-quadruplex+structure+formed+in+the+HIV-1+promoter%3A+a+potential+target+for+anti-HIV+drug+development&doi=10.1021%2Fja501500c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fja501500c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja501500c%26sid%3Dliteratum%253Aachs%26aulast%3DAmrane%26aufirst%3DS.%26aulast%3DKerkour%26aufirst%3DA.%26aulast%3DBedrat%26aufirst%3DA.%26aulast%3DVialet%26aufirst%3DB.%26aulast%3DAndreola%26aufirst%3DM.%2BL.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26atitle%3DTopology%2520of%2520a%2520DNA%2520G-quadruplex%2520structure%2520formed%2520in%2520the%2520HIV-1%2520promoter%253A%2520a%2520potential%2520target%2520for%2520anti-HIV%2520drug%2520development%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D5249%26epage%3D5252%26doi%3D10.1021%2Fja501500c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Piekna-Przybylska, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bambara, R. A.</span><span> </span><span class="NLM_article-title">U3 region in the HIV-1 genome adopts a G-quadruplex structure in its RNA and DNA sequence</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2581</span><span class="NLM_x">–</span> <span class="NLM_lpage">2593</span><span class="refDoi"> DOI: 10.1021/bi4016692</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi4016692" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Cru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=2581-2593&author=D.+Piekna-Przybylskaauthor=M.+A.+Sullivanauthor=G.+Sharmaauthor=R.+A.+Bambara&title=U3+region+in+the+HIV-1+genome+adopts+a+G-quadruplex+structure+in+its+RNA+and+DNA+sequence&doi=10.1021%2Fbi4016692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">U3 region in the HIV-1 genome adopts a G-quadruplex structure in its RNA and DNA sequence</span></div><div class="casAuthors">Piekna-Przybylska, Dorota; Sullivan, Mark A.; Sharma, Gaurav; Bambara, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2581-2593</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Genomic regions rich in G residues are prone to adopt G-quadruplex structure.  Multiple Sp1-binding motifs arranged in tandem have been suggested to form this structure in promoters of cancer-related genes.  Here, we demonstrate that the G-rich proviral DNA sequence of the HIV-1 U3 region, which serves as a promoter of viral transcription, adopts a G-quadruplex structure.  The sequence contains three binding elements for transcription factor Sp1, which is involved in the regulation of HIV-1 latency, reactivation, and high-level virus expression.  We show that the three Sp1 binding motifs can adopt different forms of G-quadruplex structure and that the Sp1 protein can recognize and bind to its site folded into a G-quadruplex.  In addn., a c-kit2 specific antibody, designated hf2, binds to two different G-quadruplexes formed in Sp1 sites.  Since U3 is encoded at both viral genomic ends, the G-rich sequence is also present in the RNA genome.  We demonstrate that the RNA sequence of U3 forms dimers with characteristics known for intermol. G-quadruplexes.  Together with previous reports showing G-quadruplex dimers in the gag and cPPT regions, these results suggest that integrity of the two viral genomes is maintained through numerous intermol. G-quadruplexes formed in different RNA genome locations.  Reconstituted reverse transcription shows that the potassium-dependent structure formed in U3 RNA facilitates RT template switching, suggesting that the G-quadruplex contributes to recombination in U3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8c6X09GeI-LVg90H21EOLACvtfcHk0ljUkx5fdkFezw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Cru7k%253D&md5=42c96411b28a8f0b2ba1e1ce5891de0b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fbi4016692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi4016692%26sid%3Dliteratum%253Aachs%26aulast%3DPiekna-Przybylska%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DM.%2BA.%26aulast%3DSharma%26aufirst%3DG.%26aulast%3DBambara%26aufirst%3DR.%2BA.%26atitle%3DU3%2520region%2520in%2520the%2520HIV-1%2520genome%2520adopts%2520a%2520G-quadruplex%2520structure%2520in%2520its%2520RNA%2520and%2520DNA%2520sequence%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D2581%26epage%3D2593%26doi%3D10.1021%2Fbi4016692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butovskaya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daelemans, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannecouque, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Anti-HIV-1 activity of the G-quadruplex ligand BRACO-19</span> <span class="citation_source-journal">J. Antimicrob. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">3248</span><span class="NLM_x">–</span> <span class="NLM_lpage">3258</span><span class="refDoi"> DOI: 10.1093/jac/dku280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fjac%2Fdku280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25103489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisFaksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=3248-3258&author=R.+Perroneauthor=E.+Butovskayaauthor=D.+Daelemansauthor=G.+Paluauthor=C.+Pannecouqueauthor=S.+N.+Richter&title=Anti-HIV-1+activity+of+the+G-quadruplex+ligand+BRACO-19&doi=10.1093%2Fjac%2Fdku280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV-1 activity of the G-quadruplex ligand BRACO-19</span></div><div class="casAuthors">Perrone, Rosalba; Butovskaya, Elena; Daelemans, Dirk; Palu, Giorgio; Pannecouque, Christophe; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3248-3258</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: A dynamic G-quadruplex region has been previously shown to form in the long terminal repeat (LTR) promoter of the HIV-1 integrated DNA genome.  Inhibition of promoter activity and antiviral effects have been obsd. when this region was stabilized by BRACO-19, a trisubstituted acridine deriv. that binds G-quadruplexes.  Here, we aimed at characterizing the antiviral mechanism of action of BRACO-19 by analyzing its activity towards a broad range of HIV-1 strains, host cells and infection modes.  Methods: The antiviral activity of BRACO-19 in cell lines and primary cells infected or persistently infected by HIV-1 strains was evaluated at different times post-infection.  Virucidal, viral binding, time-dependent and drug-dependent assays were performed to identify the viral target step.  CD, UV spectroscopy and a reverse transcriptase (RT) stop assay were used to assess RNA G-quadruplex folding and inhibition of RT processing.  Results: Thorough virol. assays demonstrated that BRACO-19 acts both at the reverse transcription and the post-integration level during the virus life cycle.  This behavior was rationalized by the observation that a G-quadruplex-forming sequence identical to that of the LTR DNA is present at the 3'-end of the virus RNA genome.  Biophysics and biomol. testing showed that this region has the ability to fold into very stable G-quadruplex structures that are even more stabilized by BRACO-19, therefore inhibiting the reverse transcription process at the template level.  Conclusions: Our findings strongly support the activity of BRACO-19 at the viral G-quadruplex level and therefore strengthen the use of viral G-quadruplexes as new anti-HIV-1 targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobSFff3h6JxLVg90H21EOLACvtfcHk0liWe5VuZ12qBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisFaksr4%253D&md5=1b743fcdb7e70a889bf4835b29b9d4c8</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdku280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdku280%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DButovskaya%26aufirst%3DE.%26aulast%3DDaelemans%26aufirst%3DD.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DAnti-HIV-1%2520activity%2520of%2520the%2520G-quadruplex%2520ligand%2520BRACO-19%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D3248%26epage%3D3258%26doi%3D10.1093%2Fjac%2Fdku280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paramasivam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spolaore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3765</span><span class="NLM_x">–</span> <span class="NLM_lpage">3780</span><span class="refDoi"> DOI: 10.1093/nar/gkn120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkn120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=18490377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=3765-3780&author=S.+Cogoiauthor=M.+Paramasivamauthor=B.+Spolaoreauthor=L.+E.+Xodo&title=Structural+polymorphism+within+a+regulatory+element+of+the+human+KRAS+promoter%3A+formation+of+G4-DNA+recognized+by+nuclear+proteins&doi=10.1093%2Fnar%2Fgkn120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span></div><div class="casAuthors">Cogoi, Susanna; Paramasivam, Manikandan; Spolaore, Barbara; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3765-3780</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The human KRAS proto-oncogene contains a crit. nuclease hypersensitive element (NHE) upstream of the major transcription initiation site.  In this article, we demonstrate by primer-extension expts., PAGE, chem. footprinting, CD, UV and FRET expts. that the G-rich strand of NHE (32R) folds into intra-mol. G-quadruplex structures.  Fluorescence data show that 32R in 100 mM KCl melts with a biphasic profile, showing the formation of two distinct G-quadruplexes with Tm of ∼55°C (Q1) and ∼72°C (Q2).  DMS-footprinting and CD suggest that Q1 can be a parallel and Q2 a mixed parallel/antiparallel G-quadruplex.  When dsNHE (32R hybridized to its complementary) is incubated with a nuclear ext. from Panc-1 cells, three DNA-protein complexes are obsd. by EMSA.  The complex of slower mobility is competed by quadruplex 32R, but not by mutant oligonucleotides, which cannot form a quadruplex structure.  Using paramagnetic beads coupled with 32R, we pulled down from the Panc-1 ext. proteins with affinity for quadruplex 32R.  One of these is the heterogeneous nuclear ribonucleoprotein A1, which was previously reported to unfold quadruplex DNA.  These study suggests a role of quadruplex DNA in KRAS transcription and provides the basis for the rationale design of mol. strategies to inhibit the expression of KRAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5riE_L0077Vg90H21EOLACvtfcHk0liWe5VuZ12qBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D&md5=73fcda8805e5f3989bcf6824805ed046</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn120%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DParamasivam%26aufirst%3DM.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DStructural%2520polymorphism%2520within%2520a%2520regulatory%2520element%2520of%2520the%2520human%2520KRAS%2520promoter%253A%2520formation%2520of%2520G4-DNA%2520recognized%2520by%2520nuclear%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D3765%26epage%3D3780%26doi%3D10.1093%2Fnar%2Fgkn120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0lgPK2Kzq7JG6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Campbell, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Structural basis of DNA quadruplex recognition by an acridine drug</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">6722</span><span class="NLM_x">–</span> <span class="NLM_lpage">6724</span><span class="refDoi"> DOI: 10.1021/ja8016973</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8016973" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsVKmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=6722-6724&author=N.+H.+Campbellauthor=G.+N.+Parkinsonauthor=A.+P.+Reszkaauthor=S.+Neidle&title=Structural+basis+of+DNA+quadruplex+recognition+by+an+acridine+drug&doi=10.1021%2Fja8016973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of DNA Quadruplex Recognition by an Acridine Drug</span></div><div class="casAuthors">Campbell, Nancy H.; Parkinson, Gary N.; Reszka, Anthony P.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6722-6724</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The crystal structure of a complex between the bimol. human telomeric quadruplex d(TAGGGTTAGGGT)2 and the exptl. anticancer drug BRACO-19, has been detd., to 2.5 A resoln.  The binding site for the BRACO-19 mol. is at the interface of two parallel-folded quadruplexes, sandwiched between a G-tetrad surface and a TATA tetrad, and held in the site by networks of water mols.  The structure rationalizes the existing structure-activity data and provides a starting-point for the structure-based design of quadruplex-binding ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod3Mgt4sNRurVg90H21EOLACvtfcHk0li6XCa6mO4BSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsVKmt74%253D&md5=baabf2e38c8fb5b4fb780f09c2659217</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fja8016973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8016973%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DN.%2BH.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520of%2520DNA%2520quadruplex%2520recognition%2520by%2520an%2520acridine%2520drug%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D6722%26epage%3D6724%26doi%3D10.1021%2Fja8016973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Leonetti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amodei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Angelo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benassi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Incalci, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zupi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biroccio, A.</span><span> </span><span class="NLM_article-title">Biological activity of the G-quadruplex ligand RHPS4 (3,11-difluoro-6,8,13-trimethyl-8H-quino[4,3,2-kl]acridinium methosulfate) is associated with telomere capping alteration</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">1138</span><span class="NLM_x">–</span> <span class="NLM_lpage">1146</span><span class="refDoi"> DOI: 10.1124/mol.104.001537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1124%2Fmol.104.001537" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2004&pages=1138-1146&author=C.+Leonettiauthor=S.+Amodeiauthor=C.+D%E2%80%99Angeloauthor=A.+Rizzoauthor=B.+Benassiauthor=A.+Antonelliauthor=R.+Elliauthor=M.+F.+Stevensauthor=M.+D%E2%80%99Incalciauthor=G.+Zupiauthor=A.+Biroccio&title=Biological+activity+of+the+G-quadruplex+ligand+RHPS4+%283%2C11-difluoro-6%2C8%2C13-trimethyl-8H-quino%5B4%2C3%2C2-kl%5Dacridinium+methosulfate%29+is+associated+with+telomere+capping+alteration&doi=10.1124%2Fmol.104.001537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1124%2Fmol.104.001537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.104.001537%26sid%3Dliteratum%253Aachs%26aulast%3DLeonetti%26aufirst%3DC.%26aulast%3DAmodei%26aufirst%3DS.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DC.%26aulast%3DRizzo%26aufirst%3DA.%26aulast%3DBenassi%26aufirst%3DB.%26aulast%3DAntonelli%26aufirst%3DA.%26aulast%3DElli%26aufirst%3DR.%26aulast%3DStevens%26aufirst%3DM.%2BF.%26aulast%3DD%25E2%2580%2599Incalci%26aufirst%3DM.%26aulast%3DZupi%26aufirst%3DG.%26aulast%3DBiroccio%26aufirst%3DA.%26atitle%3DBiological%2520activity%2520of%2520the%2520G-quadruplex%2520ligand%2520RHPS4%2520%25283%252C11-difluoro-6%252C8%252C13-trimethyl-8H-quino%255B4%252C3%252C2-kl%255Dacridinium%2520methosulfate%2529%2520is%2520associated%2520with%2520telomere%2520capping%2520alteration%26jtitle%3DMol.%2520Pharmacol.%26date%3D2004%26volume%3D66%26spage%3D1138%26epage%3D1146%26doi%3D10.1124%2Fmol.104.001537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1038/nrd3428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnrd3428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21455236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=261-275&author=S.+Balasubramanianauthor=L.+H.+Hurleyauthor=S.+Neidle&title=Targeting+G-quadruplexes+in+gene+promoters%3A+a+novel+anticancer+strategy%3F&doi=10.1038%2Fnrd3428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span></div><div class="casAuthors">Balasubramanian, Shankar; Hurley, Laurence H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-275</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are four-stranded DNA structures that are over-represented in gene promoter regions and are viewed as emerging therapeutic targets in oncol., as transcriptional repression of oncogenes through stabilization of these structures could be a novel anticancer strategy.  Many gene promoter G-quadruplexes have physicochem. properties and structural characteristics that might make them druggable, and their structural diversity suggests that a high degree of selectivity might be possible.  Here, we describe the evidence for G-quadruplexes in gene promoters and discuss their potential as therapeutic targets, as well as progress in the development of strategies to harness this potential through intervention with small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0ePjfQEBk7Vg90H21EOLACvtfcHk0li6XCa6mO4BSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D&md5=b3ea27635da9d1ff72d60228f1ef6d76</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrd3428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3428%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DTargeting%2520G-quadruplexes%2520in%2520gene%2520promoters%253A%2520a%2520novel%2520anticancer%2520strategy%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D261%26epage%3D275%26doi%3D10.1038%2Fnrd3428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Ali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattacharya, S.</span><span> </span><span class="NLM_article-title">DNA binders in clinical trials and chemotherapy</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4506</span><span class="NLM_x">–</span> <span class="NLM_lpage">4521</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.05.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.bmc.2014.05.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24947479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvFWltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=4506-4521&author=A.+Aliauthor=S.+Bhattacharya&title=DNA+binders+in+clinical+trials+and+chemotherapy&doi=10.1016%2Fj.bmc.2014.05.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binders in clinical trials and chemotherapy</span></div><div class="casAuthors">Ali, Asfa; Bhattacharya, Santanu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4506-4521</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cancer has always been a dreadful disease and continues to attract extensive research investigations.  Various targets have been identified to restrain cancer.  Among these DNA happens to be the most explored one.  A wide variety of small mols., often referred to as 'ligands', has been synthesized to target numerous structural features of DNA.  The sole purpose of such mol. design has been to interfere with the transcriptional machinery in order to drive the cancer cell toward apoptosis.  The mode of action of the DNA targeting ligands focuses either on the sequence-specificity by groove binding and strand cleavage, or by identifying the morphol. distinct higher order structures like that of the G-quadruplex DNA.  However, in spite of the extensive research, only a tiny fraction of the mols. have been able to reach clin. trials and only a handful are used in chemotherapy.  This review attempts to record the journey of the DNA binding small mols. from its inception to cancer therapy via various modifications at the mol. level.  Nevertheless, factors like limited bioavailability, severe toxicities, unfavorable pharmacokinetics etc. still prove to be the major impediments in the field which warrant considerable scope for further research investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5r0k2LEdJY7Vg90H21EOLACvtfcHk0lgCOWstev81lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvFWltb4%253D&md5=08adad9a7ab77062428d6e697076b56e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.05.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.05.030%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DA.%26aulast%3DBhattacharya%26aufirst%3DS.%26atitle%3DDNA%2520binders%2520in%2520clinical%2520trials%2520and%2520chemotherapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D4506%26epage%3D4521%26doi%3D10.1016%2Fj.bmc.2014.05.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Franceschin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossetti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Ambrosio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirripa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bianco, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortaggi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Natural and synthetic G-quadruplex interactive berberine derivatives</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1707</span><span class="NLM_x">–</span> <span class="NLM_lpage">1711</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2005.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.bmcl.2005.12.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=1707-1711&author=M.+Franceschinauthor=L.+Rossettiauthor=A.+D%E2%80%99Ambrosioauthor=S.+Schirripaauthor=A.+Biancoauthor=G.+Ortaggiauthor=M.+Savinoauthor=C.+Schultesauthor=S.+Neidle&title=Natural+and+synthetic+G-quadruplex+interactive+berberine+derivatives&doi=10.1016%2Fj.bmcl.2005.12.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DFranceschin%26aufirst%3DM.%26aulast%3DRossetti%26aufirst%3DL.%26aulast%3DD%25E2%2580%2599Ambrosio%26aufirst%3DA.%26aulast%3DSchirripa%26aufirst%3DS.%26aulast%3DBianco%26aufirst%3DA.%26aulast%3DOrtaggi%26aufirst%3DG.%26aulast%3DSavino%26aufirst%3DM.%26aulast%3DSchultes%26aufirst%3DC.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DNatural%2520and%2520synthetic%2520G-quadruplex%2520interactive%2520berberine%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D1707%26epage%3D1711%26doi%3D10.1016%2Fj.bmcl.2005.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Kim, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin-Ya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wierzba, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Telomestatin, a potent telomerase inhibitor that interacts quite specifically with the human telomeric intramolecular g-quadruplex</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">2098</span><span class="NLM_x">–</span> <span class="NLM_lpage">2099</span><span class="refDoi"> DOI: 10.1021/ja017308q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja017308q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD38Xht1eksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2002&pages=2098-2099&author=M.+Y.+Kimauthor=H.+Vankayalapatiauthor=K.+Shin-Yaauthor=K.+Wierzbaauthor=L.+H.+Hurley&title=Telomestatin%2C+a+potent+telomerase+inhibitor+that+interacts+quite+specifically+with+the+human+telomeric+intramolecular+g-quadruplex&doi=10.1021%2Fja017308q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Telomestatin, a Potent Telomerase Inhibitor That Interacts Quite Specifically with the Human Telomeric Intramolecular G-Quadruplex</span></div><div class="casAuthors">Kim, Mu-Yong; Vankayalapati, Hariprasad; Shin-ya, Kazuo; Wierzba, Konstanty; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2098-2099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Telomestatin is a natural product isolated from Streptomyces anulatus 3533-SV4 and has been shown to be a very potent telomerase inhibitor.  The structural similarity between telomestatin and a G-tetrad suggested to us that the telomerase inhibition might be due to its ability either to facilitate the formation of or trap out preformed G-quadruplex structures, and thereby sequester single-stranded d[T2AG3]n primer mols. required for telomerase activity.  Significantly, telomestatin appears to be a more potent inhibitor of telomerase (5 nM) than any of the previously described G-quadruplex-interactive mols.  In this communication we provide the first exptl. evidence that telomestatin selectively facilitates the formation of or stabilizes intramol. G-quadruplexes, in particular, that produced from the human telomeric sequence d[T2AG3]4.  A simulated annealing (SA) docking approach was used to study the binding interactions of telomestatin with the intramol. antiparallel G-quadruplex structure.  Each intramol. G-quadruplex mol. was found to bind two telomestatin mols. (unpublished results).  A 2:1 model for the telomestatin bound in the external stacking mode in an energy minimized complex with the human telomeric basket-type G-quadruplex was constructed.  Our observation that a G-quadruplex-interactive mol. without significant groove interactions is able to reorient in a G-quadruplex structure points to the importance of core interaction with an asym. G-quadruplex structure in producing selective binding.  Furthermore, the G-quadruplex interactions of telomestatin are more selective for the intramol. structure in contrast to other G-quadruplex-interactive agents, such as TMPyP4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGHj0hQ7wPOrVg90H21EOLACvtfcHk0lgCOWstev81lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xht1eksLo%253D&md5=a42b84751ccb715bc1a786449de7c13a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja017308q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja017308q%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BY.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DShin-Ya%26aufirst%3DK.%26aulast%3DWierzba%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DTelomestatin%252C%2520a%2520potent%2520telomerase%2520inhibitor%2520that%2520interacts%2520quite%2520specifically%2520with%2520the%2520human%2520telomeric%2520intramolecular%2520g-quadruplex%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D2098%26epage%3D2099%26doi%3D10.1021%2Fja017308q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span> </span><span class="NLM_article-title">Solution structure of a 2:1 quindoline-c-MYC G-quadruplex: insights into G-quadruplex-interactive small molecule drug design</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">17673</span><span class="NLM_x">–</span> <span class="NLM_lpage">17680</span><span class="refDoi"> DOI: 10.1021/ja205646q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja205646q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=17673-17680&author=J.+Daiauthor=M.+Carverauthor=L.+H.+Hurleyauthor=D.+Yang&title=Solution+structure+of+a+2%3A1+quindoline-c-MYC+G-quadruplex%3A+insights+into+G-quadruplex-interactive+small+molecule+drug+design&doi=10.1021%2Fja205646q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Solution Structure of a 2:1 Quindoline-c-MYC G-Quadruplex: Insights into G-Quadruplex-Interactive Small Molecule Drug Design</span></div><div class="casAuthors">Dai, Jixun; Carver, Megan; Hurley, Laurence H.; Yang, Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17673-17680</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Unimol. parallel-stranded G-quadruplex structures are found to be prevalent in gene promoters.  The nuclease hypersensitivity element III1 (NHE III1) of the c-MYC promoter can form transcriptionally active and silenced forms, and the formation of DNA G-quadruplex structures has been shown to be crit. for c-MYC transcriptional silencing.  The soln. structure of a 2:1 quindoline-G-quadruplex complex has been solved and shows unexpected features, including the drug-induced reorientation of the flanking sequences to form a new binding pocket.  While both 3' and 5' complexes show overall similar features, there are identifiable differences that emphasize the importance of both stacking and electronic interactions.  For the first time, we describe the importance of the shape of the ligand as well as the two flanking bases in detg. drug binding specificity.  These structures provide important insights for the structure-based rational design of drugs that bind to unimol. parallel G-quadruplexes commonly found in promoter elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr56uzmCEPLH7Vg90H21EOLACvtfcHk0ljNGyt_9-MfkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fI&md5=4aa27c51128a68f46974dad450e3d4a1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja205646q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja205646q%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DCarver%26aufirst%3DM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DSolution%2520structure%2520of%2520a%25202%253A1%2520quindoline-c-MYC%2520G-quadruplex%253A%2520insights%2520into%2520G-quadruplex-interactive%2520small%2520molecule%2520drug%2520design%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D17673%26epage%3D17680%26doi%3D10.1021%2Fja205646q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Fedoroff, O. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salazar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chemeris, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerwin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">NMR-Based model of a telomerase-inhibiting compound bound to G-quadruplex DNA</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">12367</span><span class="NLM_x">–</span> <span class="NLM_lpage">12374</span><span class="refDoi"> DOI: 10.1021/bi981330n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi981330n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaK1cXltFejtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=12367-12374&author=O.+Y.+Fedoroffauthor=M.+Salazarauthor=H.+Hanauthor=V.+V.+Chemerisauthor=S.+M.+Kerwinauthor=L.+H.+Hurley&title=NMR-Based+model+of+a+telomerase-inhibiting+compound+bound+to+G-quadruplex+DNA&doi=10.1021%2Fbi981330n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">NMR-Based Model of a Telomerase-Inhibiting Compound Bound to G-Quadruplex DNA</span></div><div class="casAuthors">Fedoroff, Oleg Yu.; Salazar, Miguel; Han, Haiyong; Chemeris, Violetta V.; Kerwin, Sean M.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12367-12374</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The single-stranded (TTAGGG)n tail of human telomeric DNA is known to form stable G-quadruplex structures.  Optimal telomerase activity requires the nonfolded single-stranded form of the primer, and stabilization of the G-quadruplex form is known to interfere with telomerase binding.  We have identified 3,4,9,10-perylenetetracarboxylic diimide-based ligands as potent inhibitors of human telomerase by using a primer extension assay that does not use PCR-based amplification of the telomerase primer extension products.  A set of NMR titrns. of the ligand into solns. of G-quadruplexes using various oligonucleotides related to human telomeric DNA showed strong and specific binding of the ligand to the G-quadruplex.  The exchange rate between bound and free DNA forms is slow on the NMR time scale and allows the unequivocal detn. of the binding site and mode of binding.  In the case of the 5'-TTAGGG sequence, the ligand-DNA complex consists of two quadruplexes oriented in a tail-to-tail manner with the ligand sandwiched between terminal G4 planes.  Longer telomeric sequences, such as TTAGGGTT, TTAGGGTTA, and TAGGGTTA, form 1:1 ligand-quadruplex complexes with the ligand bound at the GT step by a threading intercalation mode.  On the basis of 2D NOESY data, a model of the latter complex has been derived that is consistent with the available exptl. data.  The detn. of the soln. structure of this telomerase inhibitor bound to telomeric quadruplex DNA should help in the design of new anticancer agents with a unique and novel mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo94uPgdpduG7Vg90H21EOLACvtfcHk0ljNGyt_9-MfkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltFejtLs%253D&md5=5dfd45e18b66e89d6098e68f5f3b43ed</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fbi981330n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi981330n%26sid%3Dliteratum%253Aachs%26aulast%3DFedoroff%26aufirst%3DO.%2BY.%26aulast%3DSalazar%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DChemeris%26aufirst%3DV.%2BV.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DNMR-Based%2520model%2520of%2520a%2520telomerase-inhibiting%2520compound%2520bound%2520to%2520G-quadruplex%2520DNA%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D12367%26epage%3D12374%26doi%3D10.1021%2Fbi981330n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertipaglia, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mella, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Quinone methides tethered to naphthalene diimides as selective G-quadruplex alkylating agents</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">13132</span><span class="NLM_x">–</span> <span class="NLM_lpage">13141</span><span class="refDoi"> DOI: 10.1021/ja904876q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja904876q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWkurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=13132-13141&author=M.+Di+Antonioauthor=F.+Doriaauthor=S.+N.+Richterauthor=C.+Bertipagliaauthor=M.+Mellaauthor=C.+Sissiauthor=M.+Palumboauthor=M.+Freccero&title=Quinone+methides+tethered+to+naphthalene+diimides+as+selective+G-quadruplex+alkylating+agents&doi=10.1021%2Fja904876q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Quinone Methides Tethered to Naphthalene Diimides as Selective G-Quadruplex Alkylating Agents</span></div><div class="casAuthors">Di Antonio, Marco; Doria, Filippo; Richter, Sara N.; Bertipaglia, Carolina; Mella, Mariella; Sissi, Claudia; Palumbo, Manlio; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">13132-13141</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed novel G-quadruplex (G-4) ligand/alkylating hybrid structures, tethering the naphthalene diimide moiety to quaternary ammonium salts of Mannich bases, as quinone-methide precursors, activatable by mild thermal digestion (40 °C).  The bis-substituted naphthalene diimides were efficiently synthesized, and their reactivity as activatable bis-alkylating agents was investigated in the presence of thiols and amines in aq. buffered solns.  The electrophilic intermediate, quinone-methide, involved in the alkylation process was trapped, in the presence of Et vinyl ether, in a hetero Diels-Alder [4 + 2] cycloaddn. reaction, yielding a substituted 2-ethoxychroman.  The DNA recognition and alkylation properties of these new derivs. were investigated by gel electrophoresis, CD, and enzymic assays.  The alkylation process occurred preferentially on the G-4 structure in comparison to other DNA conformations.  By dissecting reversible recognition and alkylation events, we found that the reversible process is a prerequisite to DNA alkylation, which in turn reinforces the G-quadruplex structural rearrangement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfmiH1zsIlN7Vg90H21EOLACvtfcHk0lgb4Xj3EQ4Ddw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWkurfJ&md5=0d8813397e6dcd04034c9a939e423f55</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja904876q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja904876q%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DBertipaglia%26aufirst%3DC.%26aulast%3DMella%26aufirst%3DM.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DQuinone%2520methides%2520tethered%2520to%2520naphthalene%2520diimides%2520as%2520selective%2520G-quadruplex%2520alkylating%2520agents%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D13132%26epage%3D13141%26doi%3D10.1021%2Fja904876q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Collie, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Promontorio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hampel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span> </span><span class="NLM_article-title">Structural basis for telomeric G-quadruplex targeting by naphthalene diimide ligands</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">2723</span><span class="NLM_x">–</span> <span class="NLM_lpage">2731</span><span class="refDoi"> DOI: 10.1021/ja2102423</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2102423" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFahtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=2723-2731&author=G.+W.+Collieauthor=R.+Promontorioauthor=S.+M.+Hampelauthor=M.+Miccoauthor=S.+Neidleauthor=G.+N.+Parkinson&title=Structural+basis+for+telomeric+G-quadruplex+targeting+by+naphthalene+diimide+ligands&doi=10.1021%2Fja2102423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Telomeric G-Quadruplex Targeting by Naphthalene Diimide Ligands</span></div><div class="casAuthors">Collie, Gavin W.; Promontorio, Rossella; Hampel, Sonja M.; Micco, Marialuisa; Neidle, Stephen; Parkinson, Gary N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2723-2731</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The folding of the single-stranded 3' end of the human telomere into G-quadruplex arrangements inhibits the overhang from hybridizing with the RNA template of telomerase and halts telomere maintenance in cancer cells.  The ability to thermally stabilize human telomeric DNA as a four-stranded G-quadruplex structure by developing selective small mol. compds. is a therapeutic path to regulating telomerase activity and thereby selectively inhibit cancer cell growth.  The development of compds. with the necessary selectivity and affinity to target parallel-stranded G-quadruplex structures has proved particularly challenging to date, relying heavily upon limited structural data.  We report here on a structure-based approach to the design of quadruplex-binding ligands to enhance affinity and selectivity for human telomeric DNA.  Crystal structures have been detd. of complexes between a 22-mer intramol. human telomeric quadruplex and two potent tetra-substituted naphthalene diimide compds., functionalized with pos. charged N-methyl-piperazine side-chains.  These compds. promote parallel-stranded quadruplex topol., binding exclusively to the 3' surface of each quadruplex.  There are significant differences between the complexes in terms of ligand mobility and in the interactions with quadruplex grooves.  One of the two ligands is markedly less mobile in the crystal complex and is more quadruplex-stabilizing, forming multiple electrostatic/hydrogen bond contacts with quadruplex phosphate groups.  The data presented here provides a structural rationale for the biophys. (effects on quadruplex thermal stabilization) and biol. data (inhibition of proliferation in cancer cell lines and evidence of in vivo antitumor activity) on compds. in this series and, thus, for the concept of telomere targeting with DNA quadruplex-binding small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi4fBPG4FU0rVg90H21EOLACvtfcHk0lgb4Xj3EQ4Ddw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFahtA%253D%253D&md5=ccacb043644092d55fec3a75afeae0ad</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja2102423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2102423%26sid%3Dliteratum%253Aachs%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DPromontorio%26aufirst%3DR.%26aulast%3DHampel%26aufirst%3DS.%2BM.%26aulast%3DMicco%26aufirst%3DM.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26atitle%3DStructural%2520basis%2520for%2520telomeric%2520G-quadruplex%2520targeting%2520by%2520naphthalene%2520diimide%2520ligands%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D2723%26epage%3D2731%26doi%3D10.1021%2Fja2102423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Micco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collie, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dale, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnmacht, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pazitna, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunaratnam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Structure-based design and evaluation of naphthalene diimide G-quadruplex ligands as telomere targeting agents in pancreatic cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2959</span><span class="NLM_x">–</span> <span class="NLM_lpage">2974</span><span class="refDoi"> DOI: 10.1021/jm301899y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301899y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlelu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2959-2974&author=M.+Miccoauthor=G.+W.+Collieauthor=A.+G.+Daleauthor=S.+A.+Ohnmachtauthor=I.+Pazitnaauthor=M.+Gunaratnamauthor=A.+P.+Reszkaauthor=S.+Neidle&title=Structure-based+design+and+evaluation+of+naphthalene+diimide+G-quadruplex+ligands+as+telomere+targeting+agents+in+pancreatic+cancer+cells&doi=10.1021%2Fjm301899y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design and Evaluation of Naphthalene Diimide G-Quadruplex Ligands As Telomere Targeting Agents in Pancreatic Cancer Cells</span></div><div class="casAuthors">Micco, Marialuisa; Collie, Gavin W.; Dale, Aaron G.; Ohnmacht, Stephan A.; Pazitna, Ingrida; Gunaratnam, Mekala; Reszka, Anthony P.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2959-2974</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tetra-substituted naphthalene diimide (ND) derivs. with pos. charged termini are potent stabilizers of human telomeric and gene promoter DNA quadruplexes and inhibit the growth of human cancer cells in vitro and in vivo.  The present study reports the enhancement of the pharmacol. properties of earlier ND compds. using structure-based design.  Crystal structures of three complexes with human telomeric intramol. quadruplexes demonstrate that two of the four strongly basic N-methyl-piperazine groups can be replaced by less basic morpholine groups with no loss of intermol. interactions in the grooves of the quadruplex.  The new compds. retain high affinity to human telomeric quadruplex DNA but are 10-fold more potent against the MIA PaCa-2 pancreatic cancer cell line, with IC50 values of ∼10 nM.  The lead compd. induces cellular senescence but does not inhibit telomerase activity at the nanomolar dosage levels required for inhibition of cellular proliferation.  Gene array qPCR anal. of MIA PaCa-2 cells treated with the lead compd. revealed significant dose-dependent modulation of a distinct subset of genes, including strong induction of DNA damage responsive genes CDKN1A, DDIT3, GADD45A/G, and PPM1D, and repression of genes involved in telomere maintenance, including hPOT1 and PARP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC6B5sA4fud7Vg90H21EOLACvtfcHk0ljPNlIqJX8x_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlelu74%253D&md5=0697d89813efa5f1347cbdfddeeabc78</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm301899y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301899y%26sid%3Dliteratum%253Aachs%26aulast%3DMicco%26aufirst%3DM.%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DDale%26aufirst%3DA.%2BG.%26aulast%3DOhnmacht%26aufirst%3DS.%2BA.%26aulast%3DPazitna%26aufirst%3DI.%26aulast%3DGunaratnam%26aufirst%3DM.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520and%2520evaluation%2520of%2520naphthalene%2520diimide%2520G-quadruplex%2520ligands%2520as%2520telomere%2520targeting%2520agents%2520in%2520pancreatic%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2959%26epage%3D2974%26doi%3D10.1021%2Fjm301899y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Water soluble extended naphthalene diimides as pH fluorescent sensors and G-quadruplex ligands</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">3830</span><span class="NLM_x">–</span> <span class="NLM_lpage">3840</span><span class="refDoi"> DOI: 10.1039/c2ob07006e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fc2ob07006e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22469919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFGhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=3830-3840&author=F.+Doriaauthor=M.+Nadaiauthor=G.+Sattinauthor=L.+Pasottiauthor=S.+N.+Richterauthor=M.+Freccero&title=Water+soluble+extended+naphthalene+diimides+as+pH+fluorescent+sensors+and+G-quadruplex+ligands&doi=10.1039%2Fc2ob07006e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Water soluble extended naphthalene diimides as pH fluorescent sensors and G-quadruplex ligands</span></div><div class="casAuthors">Doria, Filippo; Nadai, Matteo; Sattin, Giovanna; Pasotti, Luca; Richter, Sara N.; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3830-3840</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Extended naphthalene diimides (NDIs) fused to 1,4-dihydropyrazine-2,3-dione, contg. two solubilizing moieties, have been synthesized.  Fluorescence spectra of the new NDIs were remarkably affected by pH, as the second deprotonation of the dihydropyrazinedione moiety (pKa 6.9) switched off the emission.  Binding to a G-quadruplex folded oligonucleotide and stoichiometry were evaluated by FRET melting assay and CD anal.  G-quadruplex binding was strongly enhanced shifting from pH 7.4 to pH 6.0 as a consequence of the dihydropyrazinedione moiety protonation.  Cytotoxicity studies using two human telomerase-pos. cell lines (HT29 and A549) revealed that the best G-quadruplex ligand was very active against the colon cell line, with an EC50 of 300 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCoDZL-SHlDbVg90H21EOLACvtfcHk0ljPNlIqJX8x_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFGhtr0%253D&md5=9fa4f30354167482c7a7011491166940</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1039%2Fc2ob07006e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2ob07006e%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DSattin%26aufirst%3DG.%26aulast%3DPasotti%26aufirst%3DL.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DWater%2520soluble%2520extended%2520naphthalene%2520diimides%2520as%2520pH%2520fluorescent%2520sensors%2520and%2520G-quadruplex%2520ligands%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2012%26volume%3D10%26spage%3D3830%26epage%3D3840%26doi%3D10.1039%2Fc2ob07006e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manoli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandoth, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">A naphthalene diimide dyad for fluorescence switch-on detection of G-quadruplexes</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">9105</span><span class="NLM_x">–</span> <span class="NLM_lpage">9108</span><span class="refDoi"> DOI: 10.1039/C5CC01536G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2FC5CC01536G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25940220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFertbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=9105-9108&author=F.+Doriaauthor=A.+Oppiauthor=F.+Manoliauthor=S.+Bottiauthor=N.+Kandothauthor=V.+Grandeauthor=I.+Manetauthor=M.+Freccero&title=A+naphthalene+diimide+dyad+for+fluorescence+switch-on+detection+of+G-quadruplexes&doi=10.1039%2FC5CC01536G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A naphthalene diimide dyad for fluorescence switch-on detection of G-quadruplexes</span></div><div class="casAuthors">Doria, F.; Oppi, A.; Manoli, F.; Botti, S.; Kandoth, N.; Grande, V.; Manet, I.; Freccero, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">9105-9108</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A non-fluorescent naphthalene diimide (NDI) dimer, conjugating red and blue NDI dyes, becomes red/NIR emitting upon G-quadruplex binding.  The fluorescence lifetime which is significantly different for the complexes, the G-quadruplex/dimer and the weakly emitting ds-DNA/dimer is the key feature for the development of new rationally engineered G-quadruplex sensors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnwkfj1IuNNrVg90H21EOLACvtfcHk0lgnbCaGvSwtvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFertbk%253D&md5=27d4ef48c711125bb587c985b3c91f11</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2FC5CC01536G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC01536G%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DOppi%26aufirst%3DA.%26aulast%3DManoli%26aufirst%3DF.%26aulast%3DBotti%26aufirst%3DS.%26aulast%3DKandoth%26aufirst%3DN.%26aulast%3DGrande%26aufirst%3DV.%26aulast%3DManet%26aufirst%3DI.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DA%2520naphthalene%2520diimide%2520dyad%2520for%2520fluorescence%2520switch-on%2520detection%2520of%2520G-quadruplexes%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D51%26spage%3D9105%26epage%3D9108%26doi%3D10.1039%2FC5CC01536G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">A photoreactive G-quadruplex ligand triggered by green light</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">2330</span><span class="NLM_x">–</span> <span class="NLM_lpage">2334</span><span class="refDoi"> DOI: 10.1002/chem.201405215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1002%2Fchem.201405215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25512076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVygt7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2330-2334&author=M.+Nadaiauthor=F.+Doriaauthor=L.+Germaniauthor=S.+N.+Richterauthor=M.+Freccero&title=A+photoreactive+G-quadruplex+ligand+triggered+by+green+light&doi=10.1002%2Fchem.201405215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Photoreactive G-Quadruplex Ligand Triggered by Green Light</span></div><div class="casAuthors">Nadai, Matteo; Doria, Filippo; Germani, Luca; Richter, Sara N.; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2330-2334</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A photoreactive mol. dye targeting the G-quadruplex nucleic acid (G4) of the human telomeric sequence Tel22, and several mutated analogs, was activated by green light (λ=532 nm).  Highly selective covalent modification of G4 vs. single-stranded and double-stranded DNA was achieved with efficiency up to 64%.  The phenoxyl radical was generated and detected by laser-flash photolysis as a reactive intermediate that targeted loop thymine residues.  These insights may suggest a non-invasive tool for selective nucleic acid tagging and pull-down cellular applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbaIIb6jJMNbVg90H21EOLACvtfcHk0lgnbCaGvSwtvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVygt7nK&md5=a056a87113aaab1002956d207b6f000d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fchem.201405215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201405215%26sid%3Dliteratum%253Aachs%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DGermani%26aufirst%3DL.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DA%2520photoreactive%2520G-quadruplex%2520ligand%2520triggered%2520by%2520green%2520light%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2015%26volume%3D21%26spage%3D2330%26epage%3D2334%26doi%3D10.1002%2Fchem.201405215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Artusi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biasolo, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flamand, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calistri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">The Herpes Simplex Virus-1 genome contains multiple clusters of repeated G-quadruplex: Implications for the antiviral activity of a G-quadruplex ligand</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2015.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.antiviral.2015.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25843424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFWls7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2015&pages=123-131&author=S.+Artusiauthor=M.+Nadaiauthor=R.+Perroneauthor=M.+A.+Biasoloauthor=G.+Paluauthor=L.+Flamandauthor=A.+Calistriauthor=S.+N.+Richter&title=The+Herpes+Simplex+Virus-1+genome+contains+multiple+clusters+of+repeated+G-quadruplex%3A+Implications+for+the+antiviral+activity+of+a+G-quadruplex+ligand&doi=10.1016%2Fj.antiviral.2015.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The herpes simplex virus-1 genome contains multiple clusters of repeated G-quadruplex: Implications for the antiviral activity of a G-quadruplex ligand</span></div><div class="casAuthors">Artusi, Sara; Nadai, Matteo; Perrone, Rosalba; Biasolo, Maria Angela; Palu, Giorgio; Flamand, Louis; Calistri, Arianna; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-131</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Guanine-rich nucleic acids can fold into G-quadruplexes, secondary structures implicated in important regulatory functions at the genomic level in humans, prokaryotes and viruses.  The remarkably high guanine content of the Herpes Simplex Virus-1 (HSV-1) genome prompted us to investigate both the presence of G-quadruplex forming sequences in the viral genome and the possibility to target them with G-quadruplex ligands to obtain anti-HSV-1 effects with a novel mechanism of action.  Using biophys., mol. biol. and antiviral assays, we showed that the HSV-1 genome displays multiple clusters of repeated sequences that form very stable G-quadruplexes.  These sequences are mainly located in the inverted repeats of the HSV-1 genome.  Treatment of HSV-1 infected cells with the G-quadruplex ligand BRACO-19 induced inhibition of virus prodn.  BRACO-19 was able to inhibit Taq polymerase processing at G-quadruplex forming sequences in the HSV-1 genome, and decreased intracellular viral DNA in infected cells.  The last step targeted by BRACO-19 was viral DNA replication, while no effect on virus entry in the cells was obsd.  This work, presents the first evidence of extended G-quadruplex sites in key regions of the HSV-1 genome, indicates the possibility to block viral DNA replication by G-quadruplex-ligand and therefore provides a proof of concept for the use of G-quadruplex ligands as new anti-herpetic therapeutic options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdzMRiB-mdprVg90H21EOLACvtfcHk0liFD-1dYI9ZxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFWls7Y%253D&md5=e5478e7cb2c4d276457f8db45a57b54d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DArtusi%26aufirst%3DS.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DBiasolo%26aufirst%3DM.%2BA.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DFlamand%26aufirst%3DL.%26aulast%3DCalistri%26aufirst%3DA.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DThe%2520Herpes%2520Simplex%2520Virus-1%2520genome%2520contains%2520multiple%2520clusters%2520of%2520repeated%2520G-quadruplex%253A%2520Implications%2520for%2520the%2520antiviral%2520activity%2520of%2520a%2520G-quadruplex%2520ligand%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D118%26spage%3D123%26epage%3D131%26doi%3D10.1016%2Fj.antiviral.2015.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benotti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verga, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Substituted heterocyclic naphthalene diimides with unexpected acidity. Synthesis, properties, and reactivity</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">8616</span><span class="NLM_x">–</span> <span class="NLM_lpage">8625</span><span class="refDoi"> DOI: 10.1021/jo9017342</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo9017342" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2009&pages=8616-8625&author=F.+Doriaauthor=M.+di+Antonioauthor=M.+Benottiauthor=D.+Vergaauthor=M.+Freccero&title=Substituted+heterocyclic+naphthalene+diimides+with+unexpected+acidity.+Synthesis%2C+properties%2C+and+reactivity&doi=10.1021%2Fjo9017342"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjo9017342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo9017342%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3Ddi%2BAntonio%26aufirst%3DM.%26aulast%3DBenotti%26aufirst%3DM.%26aulast%3DVerga%26aufirst%3DD.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DSubstituted%2520heterocyclic%2520naphthalene%2520diimides%2520with%2520unexpected%2520acidity.%2520Synthesis%252C%2520properties%252C%2520and%2520reactivity%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2009%26volume%3D74%26spage%3D8616%26epage%3D8625%26doi%3D10.1021%2Fjo9017342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Zhou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title">An unusual addition reaction for constructing a novel pH-controlled fluorescence switch</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">292</span><span class="NLM_x">–</span> <span class="NLM_lpage">295</span><span class="refDoi"> DOI: 10.1021/ol102742f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol102742f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCltLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=292-295&author=C.+Zhouauthor=Y.+Liauthor=Y.+Zhaoauthor=J.+Zhangauthor=W.+Yangauthor=Y.+Li&title=An+unusual+addition+reaction+for+constructing+a+novel+pH-controlled+fluorescence+switch&doi=10.1021%2Fol102742f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">An Unusual Addition Reaction for Constructing a Novel pH-Controlled Fluorescence Switch</span></div><div class="casAuthors">Zhou, Chunjie; Li, Yongjun; Zhao, Yingjie; Zhang, Jianhong; Yang, Wenlong; Li, Yuliang</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-295</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A pH-controlled fluorescence switch based on a core-substituted naphthalenediimide compd. (NDI-DBU) was constructed.  The novel pentacyclic compd. (I; NDI-DBU) was derived from naphthalenediimide (NDI) and 1,8-diazabicyclo[5.4.0]undec-7-ene (DBU) in only one synthetic operation, catalysized by CuI.  The hydroxybutyl-substituted analog was also prepd. indicating the fluorophosre could be used to create further energy transfer systems.  Cyclic voltammetry, UV-visible and pH dependent fluorescence properties are reported for the di-Bu product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWHj7UHy2_QLVg90H21EOLACvtfcHk0liFD-1dYI9ZxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCltLjK&md5=fd1344124ad721f033002a0210ee873a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fol102742f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol102742f%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DAn%2520unusual%2520addition%2520reaction%2520for%2520constructing%2520a%2520novel%2520pH-controlled%2520fluorescence%2520switch%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D292%26epage%3D295%26doi%3D10.1021%2Fol102742f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Suraru, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zschieschang, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauk, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurthner, F.</span><span> </span><span class="NLM_article-title">A core-extended naphthalene diimide as a p-channel semiconductor</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">11504</span><span class="NLM_x">–</span> <span class="NLM_lpage">11506</span><span class="refDoi"> DOI: 10.1039/c1cc15144d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fc1cc15144d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21935548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=11504-11506&author=S.+L.+Suraruauthor=U.+Zschieschangauthor=H.+Klaukauthor=F.+Wurthner&title=A+core-extended+naphthalene+diimide+as+a+p-channel+semiconductor&doi=10.1039%2Fc1cc15144d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A core-extended naphthalene diimide as a p-channel semiconductor</span></div><div class="casAuthors">Suraru, Sabin-Lucian; Zschieschang, Ute; Klauk, Hagen; Wuerthner, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">11504-11506</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A novel naphthalene diimide (I) with a fully conjugated, extended π-core was synthesized in a one-pot, two-step reaction.  This org. semiconductor exhibits ambipolar transport properties with a large hole mobility of 0.56 cm2 V-1 s-1 and a current on/off ratio of 106 in bottom-gate, top-contact thin-film transistors prepd. by vacuum deposition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdbYm4C0mp8LVg90H21EOLACvtfcHk0liheuCAB0gK-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jtLfP&md5=d0e464e666e18b7d024da9b8e030dc0a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2Fc1cc15144d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc15144d%26sid%3Dliteratum%253Aachs%26aulast%3DSuraru%26aufirst%3DS.%2BL.%26aulast%3DZschieschang%26aufirst%3DU.%26aulast%3DKlauk%26aufirst%3DH.%26aulast%3DWurthner%26aufirst%3DF.%26atitle%3DA%2520core-extended%2520naphthalene%2520diimide%2520as%2520a%2520p-channel%2520semiconductor%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2011%26volume%3D47%26spage%3D11504%26epage%3D11506%26doi%3D10.1039%2Fc1cc15144d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Langhals, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzel, S.</span><span> </span><span class="NLM_article-title">Laterally extended naphthalene tetracarboxylic bisimides</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">7781</span><span class="NLM_x">–</span> <span class="NLM_lpage">7784</span><span class="refDoi"> DOI: 10.1021/jo101520a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo101520a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=7781-7784&author=H.+Langhalsauthor=S.+Kinzel&title=Laterally+extended+naphthalene+tetracarboxylic+bisimides&doi=10.1021%2Fjo101520a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjo101520a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo101520a%26sid%3Dliteratum%253Aachs%26aulast%3DLanghals%26aufirst%3DH.%26aulast%3DKinzel%26aufirst%3DS.%26atitle%3DLaterally%2520extended%2520naphthalene%2520tetracarboxylic%2520bisimides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D7781%26epage%3D7784%26doi%3D10.1021%2Fjo101520a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Water-soluble naphthalene diimides as singlet oxygen sensitizers</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">8065</span><span class="NLM_x">–</span> <span class="NLM_lpage">8073</span><span class="refDoi"> DOI: 10.1021/jo401347z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo401347z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFeqsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=8065-8073&author=F.+Doriaauthor=I.+Manetauthor=V.+Grandeauthor=S.+Montiauthor=M.+Freccero&title=Water-soluble+naphthalene+diimides+as+singlet+oxygen+sensitizers&doi=10.1021%2Fjo401347z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Water-Soluble Naphthalene Diimides as Singlet Oxygen Sensitizers</span></div><div class="casAuthors">Doria, Filippo; Manet, Ilse; Grande, Vincenzo; Monti, Sandra; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8065-8073</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromo- and/or alkylamino-substituted and hydrosol. naphthalene diimides (NDIs) were synthesized to study their multimodal photophys. and photochem. properties.  Bromine-contg. NDIs deiv. behaved as both singlet oxygen (1O2) photosensitizers and fluorescent mols. upon irradn. at 532 nm.  Among the NDIs not contg. Br, one exhibited photophys. properties similar to those of Br-NDIs, by irradn. above 610 nm, suggesting that for these NDIs both singlet and triplet excited-state properties are strongly affected by length, structure of the solubilizing moieties, and pH of the soln.  Laser flash photolysis confirmed that the NDI lowest triplet excited state was efficiently populated, upon excitation at both 355 and 532 nm, and that free amine moieties quenched both the singlet and triplet excited states by intramol. electron transfer, with generation of detectable radical anions.  Time-resolved expts., monitoring the 1270 nm 1O2 phosphorescence decay generated upon laser irradn. at 532 nm, allowed a ranking of the NDIs as sensitizers, based on their 1O2 quantum yields (ΦΔ).  The tetrafunctionalized deriv., exhibiting a long-lived triplet state (τ ∼ 32 μs) and the most promising absorptivity for photodynamic therapy application, was tested as efficient photosensitizers in the photooxidns. of 1,5-dihydroxynaphthalene and 9,10-anthracenedipropionic acid in acetonitrile and water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF9kd3DdnyPLVg90H21EOLACvtfcHk0liheuCAB0gK-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFeqsr3N&md5=0949204927d8271a1584da0a89cd0c54</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjo401347z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo401347z%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DManet%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DV.%26aulast%3DMonti%26aufirst%3DS.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DWater-soluble%2520naphthalene%2520diimides%2520as%2520singlet%2520oxygen%2520sensitizers%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26spage%3D8065%26epage%3D8073%26doi%3D10.1021%2Fjo401347z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Palm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Lange, T.</span><span> </span><span class="NLM_article-title">How shelterin protects mammalian telomeres</span> <span class="citation_source-journal">Annu. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">334</span><span class="refDoi"> DOI: 10.1146/annurev.genet.41.110306.130350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1146%2Fannurev.genet.41.110306.130350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=18680434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFemtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2008&pages=301-334&author=W.+Palmauthor=T.+de+Lange&title=How+shelterin+protects+mammalian+telomeres&doi=10.1146%2Fannurev.genet.41.110306.130350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">How shelterin protects mammalian telomeres</span></div><div class="casAuthors">Palm, Wilhelm; de Lange, Titia</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">301-334</span>CODEN:
                <span class="NLM_cas:coden">ARVGB7</span>;
        ISSN:<span class="NLM_cas:issn">0066-4197</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The genomes of prokaryotes and eukaryotic organelles are usually circular as are most plasmids and viral genomes.  In contrast, the nuclear genomes of eukaryotes are organized on linear chromosomes, which require mechanisms to protect and replicate DNA ends.  Eukaryotes navigate these problems with the advent of telomeres, protective nucleoprotein complexes at the ends of linear chromosomes, and telomerase, the enzyme that maintains the DNA in these structures.  Mammalian telomeres contain a specific protein complex, shelterin (I), that functions to protect chromosome ends from all aspects of the DNA damage response and regulates telomere maintenance by telomerase.  Recent expts., discussed here, have revealed how I represses the ATM and ATR kinase signaling pathways and hides chromosome ends from nonhomologous end joining and homol.-directed repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvfcjAIZbU2bVg90H21EOLACvtfcHk0lhQHmyY5Oo3XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFemtbrI&md5=3d3ddc160214d999e23041f5e257791f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1146%2Fannurev.genet.41.110306.130350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.genet.41.110306.130350%26sid%3Dliteratum%253Aachs%26aulast%3DPalm%26aufirst%3DW.%26aulast%3Dde%2BLange%26aufirst%3DT.%26atitle%3DHow%2520shelterin%2520protects%2520mammalian%2520telomeres%26jtitle%3DAnnu.%2520Rev.%2520Genet.%26date%3D2008%26volume%3D42%26spage%3D301%26epage%3D334%26doi%3D10.1146%2Fannurev.genet.41.110306.130350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Rosu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabelica, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houssier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Pauw, E.</span><span> </span><span class="NLM_article-title">Determination of affinity, stoichiometry and sequence selectivity of minor groove binder complexes with double-stranded oligodeoxynucleotides by electrospray ionization mass spectrometry</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">e82</span><span class="refDoi"> DOI: 10.1093/nar/gnf081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgnf081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=e82&author=F.+Rosuauthor=V.+Gabelicaauthor=C.+Houssierauthor=E.+De+Pauw&title=Determination+of+affinity%2C+stoichiometry+and+sequence+selectivity+of+minor+groove+binder+complexes+with+double-stranded+oligodeoxynucleotides+by+electrospray+ionization+mass+spectrometry&doi=10.1093%2Fnar%2Fgnf081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgnf081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgnf081%26sid%3Dliteratum%253Aachs%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DHoussier%26aufirst%3DC.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DDetermination%2520of%2520affinity%252C%2520stoichiometry%2520and%2520sequence%2520selectivity%2520of%2520minor%2520groove%2520binder%2520complexes%2520with%2520double-stranded%2520oligodeoxynucleotides%2520by%2520electrospray%2520ionization%2520mass%2520spectrometry%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26spage%3De82%26doi%3D10.1093%2Fnar%2Fgnf081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Yuan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span> </span><span class="NLM_article-title">Mass spectrometry of G-quadruplex DNA: formation, recognition, property, conversion, and conformation</span> <span class="citation_source-journal">Mass Spectrom. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1121</span><span class="NLM_x">–</span> <span class="NLM_lpage">1142</span><span class="refDoi"> DOI: 10.1002/mas.20315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1002%2Fmas.20315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21520218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyjsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1121-1142&author=G.+Yuanauthor=Q.+Zhangauthor=J.+Zhouauthor=H.+Li&title=Mass+spectrometry+of+G-quadruplex+DNA%3A+formation%2C+recognition%2C+property%2C+conversion%2C+and+conformation&doi=10.1002%2Fmas.20315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Mass spectrometry of G-quadruplex DNA: Formation, recognition, property, conversion, and conformation</span></div><div class="casAuthors">Yuan, Gu; Zhang, Qiang; Zhou, Jiang; Li, Huihui</div><div class="citationInfo"><span class="NLM_cas:title">Mass Spectrometry Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1142</span>CODEN:
                <span class="NLM_cas:coden">MSRVD3</span>;
        ISSN:<span class="NLM_cas:issn">0277-7037</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">G-quadruplexes are special secondary structures formed from G-rich sequences of DNA, and have proven to play important roles in a no. of biol. systems, including the regulation of gene transcription and translation.  The highly distinctive nature of G-quadruplex structures and their functions suggest that G-quadruplexes can act as novel targets for drug development.  As a highly sensitive anal. tool, mass spectrometry has been widely used for the anal. of G-quadruplex structures.  Electrospray-ionization mass spectrometry, in particular, has found captivating applications to probe interactions between small mols. and G-quadruplex DNA.  In this review, we will discuss: (1) mass spectrometry probing of the formation, binding affinity, and stoichiometry between G-quadruplexes and small mols.; (2) stabilization and collision-dissocn. behavior of G-quadruplex DNA; (3) the exploration of the equil. transfer between a G-quadruplex and duplex DNA; and (4) the ESI-MS anal. of the conversion of intramol. and intermol. G-quadruplexes.  Finally, we will also introduce the application of new techniques in the anal. of G-quadruplex conformation, such as ion-mobility and IR multiphoton-dissocn. mass spectrometry.  We believe that, with the new tech. developments, mass spectrometry will play an unparalleled role in the anal. of the G-quadruplex structures. © 2010 Wiley Periodicals, Inc., Mass Spec Rev 30:1121-1142, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP3zw3D_VtGrVg90H21EOLACvtfcHk0lhQHmyY5Oo3XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyjsr7O&md5=0ae9102e7e612f83e402a1614fd03813</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fmas.20315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmas.20315%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMass%2520spectrometry%2520of%2520G-quadruplex%2520DNA%253A%2520formation%252C%2520recognition%252C%2520property%252C%2520conversion%252C%2520and%2520conformation%26jtitle%3DMass%2520Spectrom.%2520Rev.%26date%3D2011%26volume%3D30%26spage%3D1121%26epage%3D1142%26doi%3D10.1002%2Fmas.20315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Turner, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabris, D.</span><span> </span><span class="NLM_article-title">Inhibitory effects of archetypical nucleic acid ligands on the interactions of HIV-1 nucleocapsid protein with elements of Psi-RNA</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1305</span><span class="NLM_x">–</span> <span class="NLM_lpage">1316</span><span class="refDoi"> DOI: 10.1093/nar/gkl004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=1305-1316&author=K.+B.+Turnerauthor=N.+A.+Haganauthor=D.+Fabris&title=Inhibitory+effects+of+archetypical+nucleic+acid+ligands+on+the+interactions+of+HIV-1+nucleocapsid+protein+with+elements+of+Psi-RNA&doi=10.1093%2Fnar%2Fgkl004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl004%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DK.%2BB.%26aulast%3DHagan%26aufirst%3DN.%2BA.%26aulast%3DFabris%26aufirst%3DD.%26atitle%3DInhibitory%2520effects%2520of%2520archetypical%2520nucleic%2520acid%2520ligands%2520on%2520the%2520interactions%2520of%2520HIV-1%2520nucleocapsid%2520protein%2520with%2520elements%2520of%2520Psi-RNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D1305%26epage%3D1316%26doi%3D10.1093%2Fnar%2Fgkl004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Fernando, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladame, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkitaraman, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">A conserved quadruplex motif located in a transcription activation site of the human c-kit oncogene</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7854</span><span class="NLM_x">–</span> <span class="NLM_lpage">7860</span><span class="refDoi"> DOI: 10.1021/bi0601510</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0601510" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD28XltVKgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7854-7860&author=H.+Fernandoauthor=A.+P.+Reszkaauthor=J.+Huppertauthor=S.+Ladameauthor=S.+Rankinauthor=A.+R.+Venkitaramanauthor=S.+Neidleauthor=S.+Balasubramanian&title=A+conserved+quadruplex+motif+located+in+a+transcription+activation+site+of+the+human+c-kit+oncogene&doi=10.1021%2Fbi0601510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Quadruplex Motif Located in a Transcription Activation Site of the Human c-kit Oncogene</span></div><div class="casAuthors">Fernando, Himesh; Reszka, Anthony P.; Huppert, Julian; Ladame, Sylvain; Rankin, Sarah; Venkitaraman, Ashok R.; Neidle, Stephen; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7854-7860</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The c-kit gene encodes a receptor tyrosine kinase, whose engagement by its ligand triggers signals leading to cell proliferation.  C-kit activity is elevated in gastrointestinal stromal tumors (GISTs), and its therapeutic inhibition by small mols. such as imatinib is clin. validated.  We identified a putative quadruplex-forming 21-nucleotide sequence upstream of the c-kit transcription initiation site (c-kit21), on the G-rich strand, which occupies a site required for core promoter activity.  Here, we show by NMR, CD, and UV spectroscopic methods that c-kit21 forms quadruplexes under physiol. conditions.  Mutational anal. of c-kit21 has provided insights into its structural polymorphism.  In particular, one mutated form appears to form a single quadruplex species that adopts a parallel conformation.  The quadruplex-forming sequence shows a high level of sequence conservation across human, mouse, rat, and chimpanzee.  The small variation in sequence between the quadruplex in human/chimpanzee as compared to the rat/mouse was examd. more closely by biophys. methods.  Despite a variation in the sequence and length of loop 2, the quadruplexes showed both comparable CD spectra, indicative of parallel quadruplexes, and also similar thermal stability profiles, suggesting conservation of biophys. characteristics.  Collectively, the evidence suggests that this quadruplex is a serious target for a detailed functional investigation at the cell-biol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_huuxfinFYLVg90H21EOLACvtfcHk0lgps2BTyfoLpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltVKgtrc%253D&md5=4ffebe621ea8ba509e3d10fa1ebb202d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fbi0601510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0601510%26sid%3Dliteratum%253Aachs%26aulast%3DFernando%26aufirst%3DH.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DHuppert%26aufirst%3DJ.%26aulast%3DLadame%26aufirst%3DS.%26aulast%3DRankin%26aufirst%3DS.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DA%2520conserved%2520quadruplex%2520motif%2520located%2520in%2520a%2520transcription%2520activation%2520site%2520of%2520the%2520human%2520c-kit%2520oncogene%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D7854%26epage%3D7860%26doi%3D10.1021%2Fbi0601510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Differential targeting of unpaired bases within duplex DNA by the natural compound clerocidin: a valuable tool to dissect DNA secondary structure</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e52994</span><span class="refDoi"> DOI: 10.1371/journal.pone.0052994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.pone.0052994" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e52994&author=M.+Nadaiauthor=G.+Paluauthor=M.+Palumboauthor=S.+N.+Richter&title=Differential+targeting+of+unpaired+bases+within+duplex+DNA+by+the+natural+compound+clerocidin%3A+a+valuable+tool+to+dissect+DNA+secondary+structure&doi=10.1371%2Fjournal.pone.0052994"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0052994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0052994%26sid%3Dliteratum%253Aachs%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DDifferential%2520targeting%2520of%2520unpaired%2520bases%2520within%2520duplex%2520DNA%2520by%2520the%2520natural%2520compound%2520clerocidin%253A%2520a%2520valuable%2520tool%2520to%2520dissect%2520DNA%2520secondary%2520structure%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De52994%26doi%3D10.1371%2Fjournal.pone.0052994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menegazzo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span> </span><span class="NLM_article-title">Reactivity of clerocidin towards adenine: implications for base-modulated DNA damage</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">976</span><span class="NLM_x">–</span> <span class="NLM_lpage">985</span><span class="refDoi"> DOI: 10.1039/b819049f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fb819049f" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=976-985&author=S.+N.+Richterauthor=I.+Menegazzoauthor=M.+Nadaiauthor=S.+Moroauthor=M.+Palumbo&title=Reactivity+of+clerocidin+towards+adenine%3A+implications+for+base-modulated+DNA+damage&doi=10.1039%2Fb819049f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2Fb819049f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb819049f%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DMenegazzo%26aufirst%3DI.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DMoro%26aufirst%3DS.%26aulast%3DPalumbo%26aufirst%3DM.%26atitle%3DReactivity%2520of%2520clerocidin%2520towards%2520adenine%253A%2520implications%2520for%2520base-modulated%2520DNA%2520damage%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2009%26volume%3D7%26spage%3D976%26epage%3D985%26doi%3D10.1039%2Fb819049f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Clerocidin-mediated DNA footprinting discriminates among different G-quadruplex conformations and detects tetraplex folding in a duplex environment</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1830</span><span class="NLM_x">, </span> <span class="NLM_fpage">4660</span><span class="NLM_x">–</span> <span class="NLM_lpage">4668</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2013.05.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.bbagen.2013.05.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=23747297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emsLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1830&publication_year=2013&pages=4660-4668&author=M.+Nadaiauthor=G.+Sattinauthor=G.+Paluauthor=M.+Palumboauthor=S.+N.+Richter&title=Clerocidin-mediated+DNA+footprinting+discriminates+among+different+G-quadruplex+conformations+and+detects+tetraplex+folding+in+a+duplex+environment&doi=10.1016%2Fj.bbagen.2013.05.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Clerocidin-mediated DNA footprinting discriminates among different G-quadruplex conformations and detects tetraplex folding in a duplex environment</span></div><div class="casAuthors">Nadai, Matteo; Sattin, Giovanna; Palu, Giorgio; Palumbo, Manlio; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1830</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4660-4668</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">G-quadruplexes are polymorphic non-canonical nucleic acid conformations involved both in physiol. and pathol. processes.  Given the high degree of folding heterogeneity and comparable conformational stabilities, different G-quadruplex forms can occur simultaneously, hence rendering the use of basic instrumental methods for structure detn., like X-ray diffraction or NMR, hardly useful.  Footprinting techniques represent valuable and relatively rapid alternative to characterize DNA folding.  The natural diterpenoid clerocidin is an alkylating agent that specifically reacts at single-stranded DNA regions, with different mechanisms depending on the exposed nucleotide.  Clerocidin was used to footprint G-quadruplex structures formed by telomeric and oncogene promoter sequences (c-myc, bcl-2, c-kit2), and by the thrombin binding aptamer.  The easy modulability of CL reactivity towards DNA bases permitted to discriminate fully and partially protected sites, highlights stretched portions of the G-quadruplex conformation, and discriminate among topologies adopted by one sequence in different environmental conditions.  Importantly, CL displayed the unique property to allow detection of G-quadruplex folding within a duplex context.  CL is a finely performing new tool to unveil G-quadruplex arrangements in DNA sequences under genomically relevant conditions.  Nucleic acid G-quadruplex structures are an emerging research field because of the recent indication of their involvement in a series of key biol. functions, in particular in regulation of proliferation-assocd. gene expression.  The use of clerocidin as footprinting agent to identify G-quadruplex structures under genomically relevant conditions may allow detection of new G-quadruplex-based regulatory regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-IrvRFi9AJLVg90H21EOLACvtfcHk0lhbxXuXiRh3lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emsLbM&md5=c7d722997b1941abb18af11c402d3fa1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2013.05.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2013.05.039%26sid%3Dliteratum%253Aachs%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DSattin%26aufirst%3DG.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DClerocidin-mediated%2520DNA%2520footprinting%2520discriminates%2520among%2520different%2520G-quadruplex%2520conformations%2520and%2520detects%2520tetraplex%2520folding%2520in%2520a%2520duplex%2520environment%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2013%26volume%3D1830%26spage%3D4660%26epage%3D4668%26doi%3D10.1016%2Fj.bbagen.2013.05.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Percivalle, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaffaroni, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcaro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artese, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Hybrid ligand-alkylating agents targeting telomeric G-quadruplex structures</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2806</span><span class="refDoi"> DOI: 10.1039/c2ob06816h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fc2ob06816h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22367401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvFKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=2798-2806&author=F.+Doriaauthor=M.+Nadaiauthor=M.+Foliniauthor=M.+Di+Antonioauthor=L.+Germaniauthor=C.+Percivalleauthor=C.+Sissiauthor=N.+Zaffaroniauthor=S.+Alcaroauthor=A.+Arteseauthor=S.+N.+Richterauthor=M.+Freccero&title=Hybrid+ligand-alkylating+agents+targeting+telomeric+G-quadruplex+structures&doi=10.1039%2Fc2ob06816h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Hybrid ligand-alkylating agents targeting telomeric G-quadruplex structures</span></div><div class="casAuthors">Doria, Filippo; Nadai, Matteo; Folini, Marco; Di Antonio, Marco; Germani, Luca; Percivalle, Claudia; Sissi, Claudia; Zaffaroni, Nadia; Alcaro, Stefano; Artese, Anna; Richter, Sara N.; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2798-2806</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The synthesis, physico-chem. properties and biol. effects of a new class of naphthalene diimides (NDIs) capable of reversibly binding telomeric DNA and alkylate it through an electrophilic quinone methide moiety (QM), are reported.  FRET and CD assays showed a marked stabilization and selectivity towards telomeric G4 DNA folded in a hybrid topol.  NDI-QMs' alkylating properties revealed a good reactivity on single nucleosides and selectivity towards telomeric G4.  A selected NDI was able to significantly impair the growth of melanoma cells by causing telomere dysfunction and down-regulation of telomerase expression.  These findings point to the authors' hybrid ligand-alkylating NDIs as possible tools for the development of novel targeted anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDdu_j-tKmYLVg90H21EOLACvtfcHk0lhbxXuXiRh3lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvFKrtrw%253D&md5=43be7fdab1bb56fe7188ab79f98276ba</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1039%2Fc2ob06816h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2ob06816h%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DFolini%26aufirst%3DM.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DGermani%26aufirst%3DL.%26aulast%3DPercivalle%26aufirst%3DC.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DZaffaroni%26aufirst%3DN.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DArtese%26aufirst%3DA.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DHybrid%2520ligand-alkylating%2520agents%2520targeting%2520telomeric%2520G-quadruplex%2520structures%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2012%26volume%3D10%26spage%3D2798%26epage%3D2806%26doi%3D10.1039%2Fc2ob06816h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Wilkinson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pincus, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shepard, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walton, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergin, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teintze, M.</span><span> </span><span class="NLM_article-title">Novel compounds containing multiple guanide groups that bind the HIV coreceptor CXCR4</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">–</span> <span class="NLM_lpage">263</span><span class="refDoi"> DOI: 10.1128/AAC.00709-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1128%2FAAC.00709-10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=255-263&author=R.+A.+Wilkinsonauthor=S.+H.+Pincusauthor=J.+B.+Shepardauthor=S.+K.+Waltonauthor=E.+P.+Berginauthor=M.+Labibauthor=M.+Teintze&title=Novel+compounds+containing+multiple+guanide+groups+that+bind+the+HIV+coreceptor+CXCR4&doi=10.1128%2FAAC.00709-10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1128%2FAAC.00709-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00709-10%26sid%3Dliteratum%253Aachs%26aulast%3DWilkinson%26aufirst%3DR.%2BA.%26aulast%3DPincus%26aufirst%3DS.%2BH.%26aulast%3DShepard%26aufirst%3DJ.%2BB.%26aulast%3DWalton%26aufirst%3DS.%2BK.%26aulast%3DBergin%26aufirst%3DE.%2BP.%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DTeintze%26aufirst%3DM.%26atitle%3DNovel%2520compounds%2520containing%2520multiple%2520guanide%2520groups%2520that%2520bind%2520the%2520HIV%2520coreceptor%2520CXCR4%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D255%26epage%3D263%26doi%3D10.1128%2FAAC.00709-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Furman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fyfe, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Clair, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinhold, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rideout, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehrman, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolognesi, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsuya, H.</span><span> </span><span class="NLM_article-title">Phosphorylation of 3′-azido-3′-deoxythymidine and selective interaction of the 5′-triphosphate with human immunodeficiency virus reverse transcriptase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">8333</span><span class="NLM_x">–</span> <span class="NLM_lpage">8337</span><span class="refDoi"> DOI: 10.1073/pnas.83.21.8333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.83.21.8333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=2430286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaL2sXktlCqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=8333-8337&author=P.+A.+Furmanauthor=J.+A.+Fyfeauthor=M.+H.+St.+Clairauthor=K.+Weinholdauthor=J.+L.+Rideoutauthor=G.+A.+Freemanauthor=S.+N.+Lehrmanauthor=D.+P.+Bolognesiauthor=S.+Broderauthor=H.+Mitsuya&title=Phosphorylation+of+3%E2%80%B2-azido-3%E2%80%B2-deoxythymidine+and+selective+interaction+of+the+5%E2%80%B2-triphosphate+with+human+immunodeficiency+virus+reverse+transcriptase&doi=10.1073%2Fpnas.83.21.8333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of 3'-azido-3'-deoxythymidine and selective interaction of the 5'-triphosphate with human immunodeficiency virus reverse transcriptase</span></div><div class="casAuthors">Furman, Phillip A.; Fyfe, James A.; St. Clair, Marty H.; Weinhold, Kent; Rideout, Janet L.; Freeman, George A.; Lehrman, Sandra Nusinoff; Bolognesi, Dani P.; Broder, Samuel; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8333-7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The thymidine analog 3'-azido-3'-deoxythymidine (azidothymidine)  [30516-87-1] was previously shown to inhibit human immunodeficiency virus (HIV) replication effectively in the 50-500 nM range.  In contrast, inhibition of the growth of uninfected human fibroblasts and lymphocytes was obsd. only at concns. above 1 mM.  The nature of this selectivity was investigated.  Azidothymidine anabolism to the 5'-mono-  [29706-85-2], -di-  [106060-89-3], and -triphosphate  [92586-35-1] derivs. was similar in uninfected and HIV-infected cells.  The level of azidothymidine monophosphate was high, whereas the levels of the di- and triphosphate were low (≤5 μM and ≤2 μM, resp.).  Cytosolic thymidine kinase (EC 2.7.1.21)  [9002-06-6] was responsible for phosphorylation of azidothymidine to its monophosphate.  Purified thymidine kinase catalyzed the phosphorylations of thymidine  [50-89-5] and azidothymidine with apparent Km values of 2.9 μM and 3.0 μM.  The maximal rate of phosphorylation with azidothymidine was equal to 60% of the rate with thymidine.  Phosphorylation of azidothymidine monophosphate to the diphosphate also appeared to be catalyzed by a host-cell enzyme, thymidylate kinase (EC 2.7.4.9)  [9014-43-1].  The apparent Km value for azidothymidine monophosphate was 2-fold greater than the value for dTMP  [365-07-1] (8.6 μM vs. 4.1 μM), but the maximal phosphorylation rate was only 0.3% of the dTMP rate.  These kinetic consts. were consistent with the anabolism results and indicated that azidothymidine monophosphate is an alternative-substrate inhibitor of thymidylate kinase.  This conclusion was reflected in the observation that cell incubated with azidothymidine had reduced intracellular levels of dTTP  [365-08-2].  IC50 (concn. of inhibitor that inhibits enzyme activity 50%) values were detd. for azidothymidine triphosphate with reverse transcriptase  [9068-38-6] and with immortalized human lymphocyte (H9 cell) DNA polymerase  [9012-90-2] α .  Azidothymidine triphosphate competed about 100-fold better for the HIV reverse transcriptase than for the cellular DNA polymerase α.  These results suggest that azidothymidine is nonselectively phosphorylated but that the triphosphate deriv. efficiently and selectively binds to the HIV reverse transcriptase.  Incorporation of azidothymidylate into a growing DNA strand should terminate DNA elongation and thus inhibit DNA synthesis.  These results may have relevance to the antiviral action mechanism for azidothymidine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1_5UurNpyQrVg90H21EOLACvtfcHk0li_46JXpC45_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXktlCqug%253D%253D&md5=2dcdd98db3c5708b2c31058cc47eeb41</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.21.8333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.21.8333%26sid%3Dliteratum%253Aachs%26aulast%3DFurman%26aufirst%3DP.%2BA.%26aulast%3DFyfe%26aufirst%3DJ.%2BA.%26aulast%3DSt.%2BClair%26aufirst%3DM.%2BH.%26aulast%3DWeinhold%26aufirst%3DK.%26aulast%3DRideout%26aufirst%3DJ.%2BL.%26aulast%3DFreeman%26aufirst%3DG.%2BA.%26aulast%3DLehrman%26aufirst%3DS.%2BN.%26aulast%3DBolognesi%26aufirst%3DD.%2BP.%26aulast%3DBroder%26aufirst%3DS.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DPhosphorylation%2520of%25203%25E2%2580%25B2-azido-3%25E2%2580%25B2-deoxythymidine%2520and%2520selective%2520interaction%2520of%2520the%25205%25E2%2580%25B2-triphosphate%2520with%2520human%2520immunodeficiency%2520virus%2520reverse%2520transcriptase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1986%26volume%3D83%26spage%3D8333%26epage%3D8337%26doi%3D10.1073%2Fpnas.83.21.8333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Mitsuya, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Looney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuno, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong-Staal, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, S.</span><span> </span><span class="NLM_article-title">Dextran sulfate suppression of viruses in the HIV family: inhibition of virion binding to CD4+ cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">240</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">649</span><span class="refDoi"> DOI: 10.1126/science.2452480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1126%2Fscience.2452480" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=240&publication_year=1988&pages=646-649&author=H.+Mitsuyaauthor=D.+J.+Looneyauthor=S.+Kunoauthor=R.+Uenoauthor=F.+Wong-Staalauthor=S.+Broder&title=Dextran+sulfate+suppression+of+viruses+in+the+HIV+family%3A+inhibition+of+virion+binding+to+CD4%2B+cells&doi=10.1126%2Fscience.2452480"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1126%2Fscience.2452480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2452480%26sid%3Dliteratum%253Aachs%26aulast%3DMitsuya%26aufirst%3DH.%26aulast%3DLooney%26aufirst%3DD.%2BJ.%26aulast%3DKuno%26aufirst%3DS.%26aulast%3DUeno%26aufirst%3DR.%26aulast%3DWong-Staal%26aufirst%3DF.%26aulast%3DBroder%26aufirst%3DS.%26atitle%3DDextran%2520sulfate%2520suppression%2520of%2520viruses%2520in%2520the%2520HIV%2520family%253A%2520inhibition%2520of%2520virion%2520binding%2520to%2520CD4%252B%2520cells%26jtitle%3DScience%26date%3D1988%26volume%3D240%26spage%3D646%26epage%3D649%26doi%3D10.1126%2Fscience.2452480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Czerwinska, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juskowiak, B.</span><span> </span><span class="NLM_article-title">Photoisomerizable arylstilbazolium ligands recognize parallel and antiparallel structures of G-quadruplexes</span> <span class="citation_source-journal">Int. J. Biol. Macromol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">582</span><span class="refDoi"> DOI: 10.1016/j.ijbiomac.2012.06.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.ijbiomac.2012.06.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22750579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht12lt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=576-582&author=I.+Czerwinskaauthor=B.+Juskowiak&title=Photoisomerizable+arylstilbazolium+ligands+recognize+parallel+and+antiparallel+structures+of+G-quadruplexes&doi=10.1016%2Fj.ijbiomac.2012.06.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Photoisomerizable arylstilbazolium ligands recognize parallel and antiparallel structures of G-quadruplexes</span></div><div class="casAuthors">Czerwinska, Izabella; Juskowiak, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Macromolecules</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">576-582</span>CODEN:
                <span class="NLM_cas:coden">IJBMDR</span>;
        ISSN:<span class="NLM_cas:issn">0141-8130</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The photoisomerization and DNA interaction studies of three arylstilbazolium derivs. with various samples of nucleic acids (duplexes, triplexes and tetraplexes) are reported.  The equil. dialysis study revealed high binding affinities of ligands to tetraplex structures.  The quadruplex-binding affinity could be switched by light, e.g., the E,E and E,Z isomers of 1,4-bis(vinylquinolinium)benzene (1) interacted with parallel and antiparallel tetraplexes exhibiting different binding selectivity.  The E,Z-1 showed higher binding preference for c-myc DNA (a propeller-type quadruplex), whereas the E,E-1 favorably interacted with telomeric DNA (a basket-type quadruplex).  The presence of quadruplex DNA hampered photoisomerization of quadruplex-bound ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAzBIzyDSfcbVg90H21EOLACvtfcHk0ljZj1NrjKhR1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht12lt7vP&md5=ed59d4a08d08ba39a1ae87235dcd42dd</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ijbiomac.2012.06.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijbiomac.2012.06.027%26sid%3Dliteratum%253Aachs%26aulast%3DCzerwinska%26aufirst%3DI.%26aulast%3DJuskowiak%26aufirst%3DB.%26atitle%3DPhotoisomerizable%2520arylstilbazolium%2520ligands%2520recognize%2520parallel%2520and%2520antiparallel%2520structures%2520of%2520G-quadruplexes%26jtitle%3DInt.%2520J.%2520Biol.%2520Macromol.%26date%3D2012%26volume%3D51%26spage%3D576%26epage%3D582%26doi%3D10.1016%2Fj.ijbiomac.2012.06.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Rachwal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, K. R.</span><span> </span><span class="NLM_article-title">Quadruplex melting</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2007.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.ymeth.2007.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=17967699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1amurnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2007&pages=291-301&author=P.+A.+Rachwalauthor=K.+R.+Fox&title=Quadruplex+melting&doi=10.1016%2Fj.ymeth.2007.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex melting</span></div><div class="casAuthors">Rachwal, Phillip A.; Fox, Keith R.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">291-301</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Melting curves are commonly used to det. the stability of folded nucleic acid structures and their interaction with ligands.  This paper describes how the technique can be applied to study the properties of four-stranded nucleic acid structures that are formed by G-rich oligonucleotides.  Changes in the absorbance (at 295 nm), CD (at 260 or 295 nm) or fluorescence of appropriately labeled oligonucleotides, can be used to measure the stability and kinetics of folding.  This paper focuses on a fluorescence melting technique, and explains how this can be used to det. the Tm (T1/2) of intramol. quadruplexes and the effects of quadruplex-binding ligands.  Quant. anal. of these melting curves can be used to det. the thermodn. (ΔH, ΔG, and ΔS) and kinetic (k1, k-1) parameters.  The method can also be adapted to investigate the equil. between quadruplex and duplex DNA and to explore the selectivity of ligands for one or other structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDcGxot8SuB7Vg90H21EOLACvtfcHk0ljZj1NrjKhR1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1amurnP&md5=d36626c59fa1213824577567e46e3810</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2007.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2007.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DRachwal%26aufirst%3DP.%2BA.%26aulast%3DFox%26aufirst%3DK.%2BR.%26atitle%3DQuadruplex%2520melting%26jtitle%3DMethods%26date%3D2007%26volume%3D43%26spage%3D291%26epage%3D301%26doi%3D10.1016%2Fj.ymeth.2007.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Greenfield, N. J.</span><span> </span><span class="NLM_article-title">Using circular dichroism collected as a function of temperature to determine the thermodynamics of protein unfolding and binding interactions</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">2527</span><span class="NLM_x">–</span> <span class="NLM_lpage">2535</span><span class="refDoi"> DOI: 10.1038/nprot.2006.204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnprot.2006.204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=2527-2535&author=N.+J.+Greenfield&title=Using+circular+dichroism+collected+as+a+function+of+temperature+to+determine+the+thermodynamics+of+protein+unfolding+and+binding+interactions&doi=10.1038%2Fnprot.2006.204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.204%26sid%3Dliteratum%253Aachs%26aulast%3DGreenfield%26aufirst%3DN.%2BJ.%26atitle%3DUsing%2520circular%2520dichroism%2520collected%2520as%2520a%2520function%2520of%2520temperature%2520to%2520determine%2520the%2520thermodynamics%2520of%2520protein%2520unfolding%2520and%2520binding%2520interactions%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D1%26spage%3D2527%26epage%3D2535%26doi%3D10.1038%2Fnprot.2006.204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Tan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, G.</span><span> </span><span class="NLM_article-title">Electrospray ionization mass spectrometry probing of binding affinity of berbamine, a flexible cyclic alkaloid from traditional Chinese medicine, with G-quadruplex DNA</span> <span class="citation_source-journal">Rapid Commun. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1002/rcm.6763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1002%2Frcm.6763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24285399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVOisLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=143-147&author=W.+Tanauthor=J.+Zhouauthor=G.+Yuan&title=Electrospray+ionization+mass+spectrometry+probing+of+binding+affinity+of+berbamine%2C+a+flexible+cyclic+alkaloid+from+traditional+Chinese+medicine%2C+with+G-quadruplex+DNA&doi=10.1002%2Frcm.6763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Electrospray ionization mass spectrometry probing of binding affinity of berbamine, a flexible cyclic alkaloid from traditional Chinese medicine, with G-quadruplex DNA</span></div><div class="casAuthors">Tan, Wei; Zhou, Jiang; Yuan, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">143-147</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Classic G-quadruplex binders typically have a large arom. core and interact with G-quadruplexes through π-π stacking with the quartets.  There are rather few reports on natural flexible cyclic mol. from traditional Chinese medicine which has high binding affinity with G-quadruplex.  Electrospray ionization mass spectrometry (ESI-MS) expts. were performed to evaluate the binding affinities of a natural alkaloid, berbamine, with seven G-quadruplexes.  Furthermore, we utilized autodock4 anal. to uncover the binding mode of berbamine with the G-quadruplex.  ESI-MS expts. showed that berbamine has the best binding affinity toward the (GGA)8 G-quadruplex compared with the other six G-quadruplexes.  Autodock4 anal. indicated that berbamine interacted with the (GGA)8 G-quadruplex through hydrogen bonding and van der Waals forces with a binding site at the lateral groove formed by DG8-DA9-DA15-DG16.  In this study, we discovered a novel G-quadruplex binder, berbamine, which has high binding affinity toward the (GGA)8 G-quadruplex.  This study provided important clues regarding the probing of small mol. from traditional Chinese medicine which can target the G-quadruplex with high affinity.  Copyright © 2013 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdYiHudG9GvbVg90H21EOLACvtfcHk0ljLveaUvxI5uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVOisLjO&md5=54025c5a6d95b8096a8814357b1a95e4</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Frcm.6763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.6763%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DG.%26atitle%3DElectrospray%2520ionization%2520mass%2520spectrometry%2520probing%2520of%2520binding%2520affinity%2520of%2520berbamine%252C%2520a%2520flexible%2520cyclic%2520alkaloid%2520from%2520traditional%2520Chinese%2520medicine%252C%2520with%2520G-quadruplex%2520DNA%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2014%26volume%3D28%26spage%3D143%26epage%3D147%26doi%3D10.1002%2Frcm.6763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Mennucci, B.; Petersson, G. A.; Nakatsuji, H.; Caricato, M.; Li, X.; Hratchian, H. P.; Izmaylov, A. F.; Bloino, J.; Zheng, G.; Sonnenberg, J. L.; Hada, M.; Ehara, M.; Toyota, K.; Fukuda, R.; Hasegawa, J.; Ishida, M.; Nakajima, T.; Honda, Y.; Kitao, O.; Nakai, H.; Vreven, T.; Montgomery, J. A.,  Jr.; Peralta, J. E.; Ogliaro, F.; Bearpark, M.; Heyd, J. J.; Brothers, E.; Kudin, K. N.; Staroverov, V. N.; Keith, T.; Kobayashi, R.; Normand, J.; Raghavachari, K.; Rendell, A.; Burant, J. C.; Iyengar, S. S.; Tomasi, J.; Cossi, M.; Rega, N.; Millam, J. M.; Klene, M.; Knox, J. E.; Cross, J. B.; Bakken, V.; Adamo, C.; Jaramillo, J.; Gomperts, R.; Stratmann, R. E.; Yazyev, O.; Austin, A. J.; Cammi, R.; Pomelli, C.; Ochterski, J. W.; Martin, R. L.; Morokuma, K.; Zakrzewski, V. G.; Voth, G. A.; Salvador, P.; Dannenberg, J. J.; Dapprich, S.; Daniels, A. D.; Farkas, O.; Foresman, J. B.; Ortiz, J. V.; Cioslowski, J.; Fox, D. J.</span>Gaussian 09, <span class="NLM_edition">revision B.01</span>; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+Bearpark&author=J.+J.+Heyd&author=E.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=T.+Keith&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=J.+M.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=O.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional thermochemistry. III. The role of exact exchange</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">5648</span><span class="NLM_x">–</span> <span class="NLM_lpage">5652</span><span class="refDoi"> DOI: 10.1063/1.464913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1063%2F1.464913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=5648-5652&author=A.+D.+Becke&title=Density-functional+thermochemistry.+III.+The+role+of+exact+exchange&doi=10.1063%2F1.464913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional thermochemistry. III. The role of exact exchange</span></div><div class="casAuthors">Becke, Axel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5648-52</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Despite the remarkable thermochem. accuracy of Kohn-Sham d.-functional theories with gradient corrections for exchange-correlation, the author believes that further improvements are unlikely unless exact-exchange information is considered.  Arguments to support this view are presented, and a semiempirical exchange-correlation functional (contg. local-spin-d., gradient, and exact-exchange terms) is tested for 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total at. energies of first- and second-row systems.  This functional performs better than previous functionals with gradient corrections only, and fits expt. atomization energies with an impressively small av. abs. deviation of 2.4 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3BqpycUTBwLVg90H21EOLACvtfcHk0lgbSz813Ay66A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D&md5=291bbfc119095338bb1624f0c21c7ca8</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1063%2F1.464913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464913%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520thermochemistry.%2520III.%2520The%2520role%2520of%2520exact%2520exchange%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D5648%26epage%3D5652%26doi%3D10.1063%2F1.464913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 56 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Elena Butovskaya, Paola Soldà, Matteo Scalabrin, Matteo Nadai, <span class="NLM_string-name hlFld-ContribAuthor">Sara N. Richter</span>. </span><span class="cited-content_cbyCitation_article-title">HIV-1 Nucleocapsid Protein Unfolds Stable RNA G-Quadruplexes in the Viral Genome and Is Inhibited by G-Quadruplex Ligands. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2019,</strong> <em>5 </em>
                                    (12)
                                     , 2127-2135. <a href="https://doi.org/10.1021/acsinfecdis.9b00272" title="DOI URL">https://doi.org/10.1021/acsinfecdis.9b00272</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.9b00272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.9b00272%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DHIV-1%252BNucleocapsid%252BProtein%252BUnfolds%252BStable%252BRNA%252BG-Quadruplexes%252Bin%252Bthe%252BViral%252BGenome%252Band%252BIs%252BInhibited%252Bby%252BG-Quadruplex%252BLigands%26aulast%3DButovskaya%26aufirst%3DElena%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22072019%26date%3D06112019%26date%3D24102019%26volume%3D5%26issue%3D12%26spage%3D2127%26epage%3D2135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Emanuela Ruggiero, Martina Tassinari, Rosalba Perrone, Matteo Nadai, <span class="NLM_string-name hlFld-ContribAuthor">Sara N. Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Stable and Conserved G-Quadruplexes in the Long Terminal Repeat Promoter of Retroviruses. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2019,</strong> <em>5 </em>
                                    (7)
                                     , 1150-1159. <a href="https://doi.org/10.1021/acsinfecdis.9b00011" title="DOI URL">https://doi.org/10.1021/acsinfecdis.9b00011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.9b00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.9b00011%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DStable%252Band%252BConserved%252BG-Quadruplexes%252Bin%252Bthe%252BLong%252BTerminal%252BRepeat%252BPromoter%252Bof%252BRetroviruses%26aulast%3DRuggiero%26aufirst%3DEmanuela%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D13012019%26date%3D22052019%26date%3D13052019%26volume%3D5%26issue%3D7%26spage%3D1150%26epage%3D1159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matteo Nadai, Filippo Doria, Matteo Scalabrin, Valentina Pirota, Vincenzo Grande, Greta Bergamaschi, Valeria Amendola, Fernaldo Richtia Winnerdy, Anh Tuân Phan, Sara N. Richter, <span class="NLM_string-name hlFld-ContribAuthor">Mauro Freccero</span>. </span><span class="cited-content_cbyCitation_article-title">A Catalytic and Selective Scissoring Molecular Tool for Quadruplex Nucleic Acids. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2018,</strong> <em>140 </em>
                                    (44)
                                     , 14528-14532. <a href="https://doi.org/10.1021/jacs.8b05337" title="DOI URL">https://doi.org/10.1021/jacs.8b05337</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b05337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b05337%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DA%252BCatalytic%252Band%252BSelective%252BScissoring%252BMolecular%252BTool%252Bfor%252BQuadruplex%252BNucleic%252BAcids%26aulast%3DNadai%26aufirst%3DMatteo%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22052018%26date%3D29102018%26date%3D14102018%26volume%3D140%26issue%3D44%26spage%3D14528%26epage%3D14532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Elena Butovskaya, Brahim Heddi, Blaž Bakalar, Sara N. Richter, <span class="NLM_string-name hlFld-ContribAuthor">Anh Tuân Phan</span>. </span><span class="cited-content_cbyCitation_article-title">Major G-Quadruplex Form of HIV-1 LTR Reveals a (3 + 1) Folding Topology Containing a Stem-Loop. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2018,</strong> <em>140 </em>
                                    (42)
                                     , 13654-13662. <a href="https://doi.org/10.1021/jacs.8b05332" title="DOI URL">https://doi.org/10.1021/jacs.8b05332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b05332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b05332%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DMajor%252BG-Quadruplex%252BForm%252Bof%252BHIV-1%252BLTR%252BReveals%252Ba%252B%2525283%252B%25252B%252B1%252529%252BFolding%252BTopology%252BContaining%252Ba%252BStem-Loop%26aulast%3DButovskaya%26aufirst%3DElena%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22052018%26date%3D09102018%26volume%3D140%26issue%3D42%26spage%3D13654%26epage%3D13662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martina Tassinari, Graziella Cimino-Reale, Matteo Nadai, Filippo Doria, Elena Butovskaya, Marta Recagni, Mauro Freccero, Nadia Zaffaroni, Sara N. Richter, <span class="NLM_string-name hlFld-ContribAuthor">Marco Folini</span>. </span><span class="cited-content_cbyCitation_article-title">Down-Regulation of the Androgen Receptor by G-Quadruplex Ligands Sensitizes Castration-Resistant Prostate Cancer Cells to Enzalutamide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (19)
                                     , 8625-8638. <a href="https://doi.org/10.1021/acs.jmedchem.8b00502" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00502%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDown-Regulation%252Bof%252Bthe%252BAndrogen%252BReceptor%252Bby%252BG-Quadruplex%252BLigands%252BSensitizes%252BCastration-Resistant%252BProstate%252BCancer%252BCells%252Bto%252BEnzalutamide%26aulast%3DTassinari%26aufirst%3DMartina%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30032018%26date%3D20092018%26date%3D06092018%26volume%3D61%26issue%3D19%26spage%3D8625%26epage%3D8638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Efres Belmonte-Reche, Marta Martínez-García, Aurore Guédin, Michela Zuffo, Matilde Arévalo-Ruiz, Filippo Doria, Jenny Campos-Salinas, Marjorie Maynadier, José Juan López-Rubio, Mauro Freccero, Jean-Louis Mergny, José María Pérez-Victoria, <span class="NLM_string-name hlFld-ContribAuthor">Juan Carlos Morales</span>. </span><span class="cited-content_cbyCitation_article-title">G-Quadruplex Identification in the Genome of Protozoan Parasites Points to Naphthalene Diimide Ligands as New Antiparasitic Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 1231-1240. <a href="https://doi.org/10.1021/acs.jmedchem.7b01672" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01672</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01672%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DG-Quadruplex%252BIdentification%252Bin%252Bthe%252BGenome%252Bof%252BProtozoan%252BParasites%252BPoints%252Bto%252BNaphthalene%252BDiimide%252BLigands%252Bas%252BNew%252BAntiparasitic%252BAgents%26aulast%3DBelmonte-Reche%26aufirst%3DEfres%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D15112017%26date%3D27012018%26date%3D11012018%26volume%3D61%26issue%3D3%26spage%3D1231%26epage%3D1240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matteo  Scalabrin</span>, <span class="hlFld-ContribAuthor ">Manlio  Palumbo</span>, and <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>  . </span><span class="cited-content_cbyCitation_article-title">Highly Improved Electrospray Ionization-Mass Spectrometry Detection of G-Quadruplex-Folded Oligonucleotides and Their Complexes with Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2017,</strong> <em>89 </em>
                                    (17)
                                     , 8632-8637. <a href="https://doi.org/10.1021/acs.analchem.7b01282" title="DOI URL">https://doi.org/10.1021/acs.analchem.7b01282</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.7b01282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.7b01282%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DHighly%252BImproved%252BElectrospray%252BIonization-Mass%252BSpectrometry%252BDetection%252Bof%252BG-Quadruplex-Folded%252BOligonucleotides%252Band%252BTheir%252BComplexes%252Bwith%252BSmall%252BMolecules%26aulast%3DScalabrin%26aufirst%3DMatteo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D06042017%26date%3D08082017%26date%3D24082017%26date%3D05092017%26date%3D08082017%26volume%3D89%26issue%3D17%26spage%3D8632%26epage%3D8637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad  Al Kobaisi </span>, <span class="hlFld-ContribAuthor ">Sidhanath V.  Bhosale </span>, <span class="hlFld-ContribAuthor ">Kay  Latham </span>, <span class="hlFld-ContribAuthor ">Aaron M.  Raynor </span>, and <span class="hlFld-ContribAuthor ">Sheshanath V.  Bhosale </span>  . </span><span class="cited-content_cbyCitation_article-title">Functional Naphthalene Diimides: Synthesis, Properties, and Applications. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2016,</strong> <em>116 </em>
                                    (19)
                                     , 11685-11796. <a href="https://doi.org/10.1021/acs.chemrev.6b00160" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00160</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00160%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DFunctional%252BNaphthalene%252BDiimides%25253A%252BSynthesis%25252C%252BProperties%25252C%252Band%252BApplications%26aulast%3DAl%2BKobaisi%26aufirst%3DMohammad%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D28022016%26date%3D26082016%26date%3D12102016%26volume%3D116%26issue%3D19%26spage%3D11685%26epage%3D11796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marc  Lavigne</span>, <span class="hlFld-ContribAuthor ">Olivier  Helynck</span>, <span class="hlFld-ContribAuthor ">Pascal  Rigolet</span>, <span class="hlFld-ContribAuthor ">Rofia  Boudria-Souilah</span>, <span class="hlFld-ContribAuthor ">Mireille  Nowakowski</span>, <span class="hlFld-ContribAuthor ">Bruno  Baron</span>, <span class="hlFld-ContribAuthor ">Sébastien  Brülé</span>, <span class="hlFld-ContribAuthor ">Sylviane  Hoos</span>, <span class="hlFld-ContribAuthor ">Bertrand  Raynal</span>, <span class="hlFld-ContribAuthor ">Lionel  Guittat</span>, <span class="hlFld-ContribAuthor ">Claire  Beauvineau</span>, <span class="hlFld-ContribAuthor ">Stéphane  Petres</span>, <span class="hlFld-ContribAuthor ">Anton  Granzhan</span>, <span class="hlFld-ContribAuthor ">Jean  Guillon</span>, <span class="hlFld-ContribAuthor ">Geneviève  Pratviel</span>, <span class="hlFld-ContribAuthor ">Marie-Paule  Teulade-Fichou</span>, <span class="hlFld-ContribAuthor ">Patrick  England</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Mergny</span>, <span class="hlFld-ContribAuthor ">Hélène  Munier-Lehmann</span>. </span><span class="cited-content_cbyCitation_article-title">SARS-CoV-2 Nsp3 unique domain SUD interacts with guanine quadruplexes and G4-ligands inhibit this interaction. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (13)
                                     , 7695-7712. <a href="https://doi.org/10.1093/nar/gkab571" title="DOI URL">https://doi.org/10.1093/nar/gkab571</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkab571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkab571%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DSARS-CoV-2%252BNsp3%252Bunique%252Bdomain%252BSUD%252Binteracts%252Bwith%252Bguanine%252Bquadruplexes%252Band%252BG4-ligands%252Binhibit%252Bthis%252Binteraction%26aulast%3DLavigne%26aufirst%3DMarc%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D13%26spage%3D7695%26epage%3D7712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ardavan  Abiri</span>, <span class="hlFld-ContribAuthor ">Marc  Lavigne</span>, <span class="hlFld-ContribAuthor ">Masoud  Rezaei</span>, <span class="hlFld-ContribAuthor ">Sanaz  Nikzad</span>, <span class="hlFld-ContribAuthor ">Peyman  Zare</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Mergny</span>, <span class="hlFld-ContribAuthor ">Hamid-Reza  Rahimi</span>, . </span><span class="cited-content_cbyCitation_article-title">Unlocking G-Quadruplexes as Antiviral Targets. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Reviews</span><span> <strong>2021,</strong> <em>73 </em>
                                    (3)
                                     , 897-923. <a href="https://doi.org/10.1124/pharmrev.120.000230" title="DOI URL">https://doi.org/10.1124/pharmrev.120.000230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/pharmrev.120.000230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fpharmrev.120.000230%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Reviews%26atitle%3DUnlocking%252BG-Quadruplexes%252Bas%252BAntiviral%252BTargets%26aulast%3DAbiri%26aufirst%3DArdavan%26date%3D2021%26date%3D2021%26volume%3D73%26issue%3D3%26spage%3D897%26epage%3D923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prativa  Majee</span>, <span class="hlFld-ContribAuthor ">Aryamav  Pattnaik</span>, <span class="hlFld-ContribAuthor ">Bikash R.  Sahoo</span>, <span class="hlFld-ContribAuthor ">Uma  Shankar</span>, <span class="hlFld-ContribAuthor ">Asit K.  Pattnaik</span>, <span class="hlFld-ContribAuthor ">Amit  Kumar</span>, <span class="hlFld-ContribAuthor ">Debasis  Nayak</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Zika virus replication by G-quadruplex-binding ligands. </span><span class="cited-content_cbyCitation_journal-name">Molecular Therapy - Nucleic Acids</span><span> <strong>2021,</strong> <em>23 </em>, 691-701. <a href="https://doi.org/10.1016/j.omtn.2020.12.030" title="DOI URL">https://doi.org/10.1016/j.omtn.2020.12.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.omtn.2020.12.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.omtn.2020.12.030%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Therapy%2520-%2520Nucleic%2520Acids%26atitle%3DInhibition%252Bof%252BZika%252Bvirus%252Breplication%252Bby%252BG-quadruplex-binding%252Bligands%26aulast%3DMajee%26aufirst%3DPrativa%26date%3D2021%26volume%3D23%26spage%3D691%26epage%3D701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Lago</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Emanuela  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Martina  Tassinari</span>, <span class="hlFld-ContribAuthor ">Maja  Marušič</span>, <span class="hlFld-ContribAuthor ">Beatrice  Tosoni</span>, <span class="hlFld-ContribAuthor ">Ilaria  Frasson</span>, <span class="hlFld-ContribAuthor ">Filippo M  Cernilogar</span>, <span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Janez  Plavec</span>, <span class="hlFld-ContribAuthor ">Gunnar  Schotta</span>, <span class="hlFld-ContribAuthor ">Sara N  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">The
              MDM2
              inducible promoter folds into four-tetrad antiparallel G-quadruplexes targetable to fight malignant liposarcoma. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (2)
                                     , 847-863. <a href="https://doi.org/10.1093/nar/gkaa1273" title="DOI URL">https://doi.org/10.1093/nar/gkaa1273</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa1273%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DThe%252BMDM2%252Binducible%252Bpromoter%252Bfolds%252Binto%252Bfour-tetrad%252Bantiparallel%252BG-quadruplexes%252Btargetable%252Bto%252Bfight%252Bmalignant%252Bliposarcoma%26aulast%3DLago%26aufirst%3DSara%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D2%26spage%3D847%26epage%3D863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daria V.  Andreeva</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Tikhomirov</span>, <span class="hlFld-ContribAuthor ">Andrey E.  Shchekotikhin</span>. </span><span class="cited-content_cbyCitation_article-title">Ligands of G-quadruplex nucleic acids. </span><span class="cited-content_cbyCitation_journal-name">Russian Chemical Reviews</span><span> <strong>2021,</strong> <em>90 </em>
                                    (1)
                                     , 1-38. <a href="https://doi.org/10.1070/RCR4968" title="DOI URL">https://doi.org/10.1070/RCR4968</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1070/RCR4968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1070%2FRCR4968%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Chemical%2520Reviews%26atitle%3DLigands%252Bof%252BG-quadruplex%252Bnucleic%252Bacids%26aulast%3DAndreeva%26aufirst%3DDaria%2BV.%26date%3D2021%26volume%3D90%26issue%3D1%26spage%3D1%26epage%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed Ahmed  Abozeid</span>, <span class="hlFld-ContribAuthor ">Aya Atef  El-Sawi</span>, <span class="hlFld-ContribAuthor ">Mohamed  Abdelmoteleb</span>, <span class="hlFld-ContribAuthor ">Hanem  Awad</span>, <span class="hlFld-ContribAuthor ">Marwa Mostafa  Abdel-Aziz</span>, <span class="hlFld-ContribAuthor ">Abdel-Rahman  Hassan Abdel-Rahman</span>, <span class="hlFld-ContribAuthor ">El-Sayed  Ibrahim El-Desoky</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of novel naphthalene-heterocycle hybrids with potent antitumor, anti-inflammatory and antituberculosis activities. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (70)
                                     , 42998-43009. <a href="https://doi.org/10.1039/D0RA08526J" title="DOI URL">https://doi.org/10.1039/D0RA08526J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA08526J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA08526J%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSynthesis%252Bof%252Bnovel%252Bnaphthalene-heterocycle%252Bhybrids%252Bwith%252Bpotent%252Bantitumor%25252C%252Banti-inflammatory%252Band%252Bantituberculosis%252Bactivities%26aulast%3DAbozeid%26aufirst%3DMohamed%2BAhmed%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D70%26spage%3D42998%26epage%3D43009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vyankat A.  Sontakke</span>, <span class="hlFld-ContribAuthor ">Seergazhi G.  Srivatsan</span>. </span><span class="cited-content_cbyCitation_article-title">A dual-app nucleoside probe reports G-quadruplex formation and ligand binding in the long terminal repeat of HIV-1 proviral genome. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (16)
                                     , 127345. <a href="https://doi.org/10.1016/j.bmcl.2020.127345" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127345</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127345%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DA%252Bdual-app%252Bnucleoside%252Bprobe%252Breports%252BG-quadruplex%252Bformation%252Band%252Bligand%252Bbinding%252Bin%252Bthe%252Blong%252Bterminal%252Brepeat%252Bof%252BHIV-1%252Bproviral%252Bgenome%26aulast%3DSontakke%26aufirst%3DVyankat%2BA.%26date%3D2020%26volume%3D30%26issue%3D16%26spage%3D127345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prativa  Majee</span>, <span class="hlFld-ContribAuthor ">Uma  Shankar</span>, <span class="hlFld-ContribAuthor ">Sanjeev  Pasadi</span>, <span class="hlFld-ContribAuthor ">K.  Muniyappa</span>, <span class="hlFld-ContribAuthor ">Debasis  Nayak</span>, <span class="hlFld-ContribAuthor ">Amit  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Genome-wide analysis reveals a regulatory role for G-quadruplexes during Adenovirus multiplication. </span><span class="cited-content_cbyCitation_journal-name">Virus Research</span><span> <strong>2020,</strong> <em>283 </em>, 197960. <a href="https://doi.org/10.1016/j.virusres.2020.197960" title="DOI URL">https://doi.org/10.1016/j.virusres.2020.197960</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.virusres.2020.197960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.virusres.2020.197960%26sid%3Dliteratum%253Aachs%26jtitle%3DVirus%2520Research%26atitle%3DGenome-wide%252Banalysis%252Breveals%252Ba%252Bregulatory%252Brole%252Bfor%252BG-quadruplexes%252Bduring%252BAdenovirus%252Bmultiplication%26aulast%3DMajee%26aufirst%3DPrativa%26date%3D2020%26volume%3D283%26spage%3D197960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivier  Jarjayes</span>, <span class="hlFld-ContribAuthor ">Thomas  Lavergne</span>, <span class="hlFld-ContribAuthor ">Fabrice  Thomas</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction of Metal Complexes with G‐Quadruplexes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 1-24. <a href="https://doi.org/10.1002/9781119951438.eibc2748" title="DOI URL">https://doi.org/10.1002/9781119951438.eibc2748</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781119951438.eibc2748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781119951438.eibc2748%26sid%3Dliteratum%253Aachs%26atitle%3DInteraction%252Bof%252BMetal%252BComplexes%252Bwith%252BG%2525E2%252580%252590Quadruplexes%26aulast%3DJarjayes%26aufirst%3DOlivier%26date%3D2020%26date%3D2020%26spage%3D1%26epage%3D24%26pub%3DWiley%26atitle%3DEncyclopedia%252Bof%252BInorganic%252Band%252BBioinorganic%252BChemistry%26aulast%3DScott%26aufirst%3DRobert%2BA%26date%3D2011%26date%3D2011%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrey M.  Zhirov</span>, <span class="hlFld-ContribAuthor ">Dmitry A.  Kovalev</span>, <span class="hlFld-ContribAuthor ">Diana V.  Ulshina</span>, <span class="hlFld-ContribAuthor ">Sergey V.  Pisarenko</span>, <span class="hlFld-ContribAuthor ">Oleg P.  Demidov</span>, <span class="hlFld-ContribAuthor ">Ivan V.  Borovlev</span>. </span><span class="cited-content_cbyCitation_article-title">Diazapyrenes: interaction with nucleic acids and biological activity. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2020,</strong> <em>56 </em>
                                    (6)
                                     , 674-693. <a href="https://doi.org/10.1007/s10593-020-02717-1" title="DOI URL">https://doi.org/10.1007/s10593-020-02717-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-020-02717-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-020-02717-1%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DDiazapyrenes%25253A%252Binteraction%252Bwith%252Bnucleic%252Bacids%252Band%252Bbiological%252Bactivity%26aulast%3DZhirov%26aufirst%3DAndrey%2BM.%26date%3D2020%26date%3D2020%26volume%3D56%26issue%3D6%26spage%3D674%26epage%3D693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Tassinari</span>, <span class="hlFld-ContribAuthor ">Michela  Zuffo</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Adriana Carolina  Sevilla Montalvo</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>, <span class="hlFld-ContribAuthor ">Sara N  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Selective targeting of mutually exclusive DNA G-quadruplexes: HIV-1 LTR as paradigmatic model. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2020,</strong> <em>48 </em>
                                    (9)
                                     , 4627-4642. <a href="https://doi.org/10.1093/nar/gkaa186" title="DOI URL">https://doi.org/10.1093/nar/gkaa186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa186%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DSelective%252Btargeting%252Bof%252Bmutually%252Bexclusive%252BDNA%252BG-quadruplexes%25253A%252BHIV-1%252BLTR%252Bas%252Bparadigmatic%252Bmodel%26aulast%3DTassinari%26aufirst%3DMartina%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D9%26spage%3D4627%26epage%3D4642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emanuela  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Viral G-quadruplexes: New frontiers in virus pathogenesis and antiviral therapy. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 101-131. <a href="https://doi.org/10.1016/bs.armc.2020.04.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.04.001%26sid%3Dliteratum%253Aachs%26atitle%3DViral%252BG-quadruplexes%25253A%252BNew%252Bfrontiers%252Bin%252Bvirus%252Bpathogenesis%252Band%252Bantiviral%252Btherapy%26aulast%3DRuggiero%26aufirst%3DEmanuela%26date%3D2020%26spage%3D101%26epage%3D131%26pub%3DElsevier%26atitle%3DQuadruplex%252BNucleic%252BAcids%252BAs%252BTargets%252BFor%252BMedicinal%252BChemistry%26date%3D2020%26volume%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jyotirmayee  Dash</span>, <span class="hlFld-ContribAuthor ">Puja  Saha</span>, <span class="hlFld-ContribAuthor ">Khushnood  Fatma</span>. </span><span class="cited-content_cbyCitation_article-title">Click and combinatorial approaches to quadruplex ligand discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 287-324. <a href="https://doi.org/10.1016/bs.armc.2020.04.003" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.04.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.04.003%26sid%3Dliteratum%253Aachs%26atitle%3DClick%252Band%252Bcombinatorial%252Bapproaches%252Bto%252Bquadruplex%252Bligand%252Bdiscovery%26aulast%3DDash%26aufirst%3DJyotirmayee%26date%3D2020%26spage%3D287%26epage%3D324%26pub%3DElsevier%26atitle%3DQuadruplex%252BNucleic%252BAcids%252BAs%252BTargets%252BFor%252BMedicinal%252BChemistry%26date%3D2020%26volume%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Michele  Stasi</span>, <span class="hlFld-ContribAuthor ">Alessandra  Benassi</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>. </span><span class="cited-content_cbyCitation_article-title">An overview of quadruplex ligands: Their common features and chemotype diversity. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 163-196. <a href="https://doi.org/10.1016/bs.armc.2020.04.008" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.04.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.04.008%26sid%3Dliteratum%253Aachs%26atitle%3DAn%252Boverview%252Bof%252Bquadruplex%252Bligands%25253A%252BTheir%252Bcommon%252Bfeatures%252Band%252Bchemotype%252Bdiversity%26aulast%3DPirota%26aufirst%3DValentina%26date%3D2020%26spage%3D163%26epage%3D196%26pub%3DElsevier%26atitle%3DQuadruplex%252BNucleic%252BAcids%252BAs%252BTargets%252BFor%252BMedicinal%252BChemistry%26date%3D2020%26volume%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emanuela  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Sara  Lago</span>, <span class="hlFld-ContribAuthor ">Primož  Šket</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Ilaria  Frasson</span>, <span class="hlFld-ContribAuthor ">Janez  Plavec</span>, <span class="hlFld-ContribAuthor ">Sara N  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">A dynamic i-motif with a duplex stem-loop in the long terminal repeat promoter of the HIV-1 proviral genome modulates viral transcription. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (21)
                                     , 11057-11068. <a href="https://doi.org/10.1093/nar/gkz937" title="DOI URL">https://doi.org/10.1093/nar/gkz937</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz937%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DA%252Bdynamic%252Bi-motif%252Bwith%252Ba%252Bduplex%252Bstem-loop%252Bin%252Bthe%252Blong%252Bterminal%252Brepeat%252Bpromoter%252Bof%252Bthe%252BHIV-1%252Bproviral%252Bgenome%252Bmodulates%252Bviral%252Btranscription%26aulast%3DRuggiero%26aufirst%3DEmanuela%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D21%26spage%3D11057%26epage%3D11068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Recagni</span>, <span class="hlFld-ContribAuthor ">Tassinari</span>, <span class="hlFld-ContribAuthor ">Doria</span>, <span class="hlFld-ContribAuthor ">Cimino-Reale</span>, <span class="hlFld-ContribAuthor ">Zaffaroni</span>, <span class="hlFld-ContribAuthor ">Freccero</span>, <span class="hlFld-ContribAuthor ">Folini</span>, <span class="hlFld-ContribAuthor ">Richter</span>. </span><span class="cited-content_cbyCitation_article-title">The Oncogenic Signaling Pathways in BRAF-Mutant Melanoma Cells are Modulated by Naphthalene Diimide-Like G-Quadruplex Ligands. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2019,</strong> <em>8 </em>
                                    (10)
                                     , 1274. <a href="https://doi.org/10.3390/cells8101274" title="DOI URL">https://doi.org/10.3390/cells8101274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells8101274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells8101274%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DThe%252BOncogenic%252BSignaling%252BPathways%252Bin%252BBRAF-Mutant%252BMelanoma%252BCells%252Bare%252BModulated%252Bby%252BNaphthalene%252BDiimide-Like%252BG-Quadruplex%252BLigands%26aulast%3DRecagni%26date%3D2019%26date%3D2019%26volume%3D8%26issue%3D10%26spage%3D1274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alberto  Insuasty</span>, <span class="hlFld-ContribAuthor ">Serena  Carrara</span>, <span class="hlFld-ContribAuthor ">Linjing  Tang</span>, <span class="hlFld-ContribAuthor ">Craig  Forsyth</span>, <span class="hlFld-ContribAuthor ">Conor F.  Hogan</span>, <span class="hlFld-ContribAuthor ">Christopher R.  McNeill</span>, <span class="hlFld-ContribAuthor ">Steven J.  Langford</span>. </span><span class="cited-content_cbyCitation_article-title">A Family of Heterocyclic Naphthalene Diimide (NDI) Analogues: Comparing Parent Isoquinoline Diimides and Phthalazine Diimides with NDI. </span><span class="cited-content_cbyCitation_journal-name">ChemPlusChem</span><span> <strong>2019,</strong> <em>84 </em>
                                    (10)
                                     , 1638-1642. <a href="https://doi.org/10.1002/cplu.201900596" title="DOI URL">https://doi.org/10.1002/cplu.201900596</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cplu.201900596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcplu.201900596%26sid%3Dliteratum%253Aachs%26jtitle%3DChemPlusChem%26atitle%3DA%252BFamily%252Bof%252BHeterocyclic%252BNaphthalene%252BDiimide%252B%252528NDI%252529%252BAnalogues%25253A%252BComparing%252BParent%252BIsoquinoline%252BDiimides%252Band%252BPhthalazine%252BDiimides%252Bwith%252BNDI%26aulast%3DInsuasty%26aufirst%3DAlberto%26date%3D2019%26date%3D2019%26volume%3D84%26issue%3D10%26spage%3D1638%26epage%3D1642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ilaria  Frasson</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Conserved G-Quadruplexes Regulate the Immediate Early Promoters of Human Alphaherpesviruses. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (13)
                                     , 2375. <a href="https://doi.org/10.3390/molecules24132375" title="DOI URL">https://doi.org/10.3390/molecules24132375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24132375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24132375%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DConserved%252BG-Quadruplexes%252BRegulate%252Bthe%252BImmediate%252BEarly%252BPromoters%252Bof%252BHuman%252BAlphaherpesviruses%26aulast%3DFrasson%26aufirst%3DIlaria%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D13%26spage%3D2375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alberto  Insuasty</span>, <span class="hlFld-ContribAuthor ">Subashani  Maniam</span>, <span class="hlFld-ContribAuthor ">Steven J.  Langford</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in the Core‐Annulation of Naphthalene Diimides. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2019,</strong> <em>25 </em>
                                    (29)
                                     , 7058-7073. <a href="https://doi.org/10.1002/chem.201806009" title="DOI URL">https://doi.org/10.1002/chem.201806009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201806009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201806009%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DRecent%252BAdvances%252Bin%252Bthe%252BCore%2525E2%252580%252590Annulation%252Bof%252BNaphthalene%252BDiimides%26aulast%3DInsuasty%26aufirst%3DAlberto%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D29%26spage%3D7058%26epage%3D7073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chaoyuan  Chen</span>, <span class="hlFld-ContribAuthor ">Yilei  Wu</span>, <span class="hlFld-ContribAuthor ">Hui  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Fine-Tuning Aromatic Stacking and Single-Crystal Photoluminescence Through Coordination Chemistry. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>2019 </em>
                                    (8)
                                     , 1778-1783. <a href="https://doi.org/10.1002/ejoc.201801843" title="DOI URL">https://doi.org/10.1002/ejoc.201801843</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201801843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201801843%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DFine-Tuning%252BAromatic%252BStacking%252Band%252BSingle-Crystal%252BPhotoluminescence%252BThrough%252BCoordination%252BChemistry%26aulast%3DChen%26aufirst%3DChaoyuan%26date%3D2019%26date%3D2019%26volume%3D2019%26issue%3D8%26spage%3D1778%26epage%3D1783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nandhini  Saranathan</span>, <span class="hlFld-ContribAuthor ">Perumal  Vivekanandan</span>. </span><span class="cited-content_cbyCitation_article-title">G-Quadruplexes: More Than Just a Kink in Microbial Genomes. </span><span class="cited-content_cbyCitation_journal-name">Trends in Microbiology</span><span> <strong>2019,</strong> <em>27 </em>
                                    (2)
                                     , 148-163. <a href="https://doi.org/10.1016/j.tim.2018.08.011" title="DOI URL">https://doi.org/10.1016/j.tim.2018.08.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tim.2018.08.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tim.2018.08.011%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Microbiology%26atitle%3DG-Quadruplexes%25253A%252BMore%252BThan%252BJust%252Ba%252BKink%252Bin%252BMicrobial%252BGenomes%26aulast%3DSaranathan%26aufirst%3DNandhini%26date%3D2019%26volume%3D27%26issue%3D2%26spage%3D148%26epage%3D163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Sara  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Naphthalene Diimides as Multimodal G-Quadruplex-Selective Ligands. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (3)
                                     , 426. <a href="https://doi.org/10.3390/molecules24030426" title="DOI URL">https://doi.org/10.3390/molecules24030426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24030426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24030426%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNaphthalene%252BDiimides%252Bas%252BMultimodal%252BG-Quadruplex-Selective%252BLigands%26aulast%3DPirota%26aufirst%3DValentina%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D3%26spage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Puja  Saha</span>, <span class="hlFld-ContribAuthor ">Deepanjan  Panda</span>, <span class="hlFld-ContribAuthor ">Jyotirmayee  Dash</span>. </span><span class="cited-content_cbyCitation_article-title">The application of click chemistry for targeting quadruplex nucleic acids. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (6)
                                     , 731-750. <a href="https://doi.org/10.1039/C8CC07107A" title="DOI URL">https://doi.org/10.1039/C8CC07107A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CC07107A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CC07107A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DThe%252Bapplication%252Bof%252Bclick%252Bchemistry%252Bfor%252Btargeting%252Bquadruplex%252Bnucleic%252Bacids%26aulast%3DSaha%26aufirst%3DPuja%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D6%26spage%3D731%26epage%3D750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">G-Quadruplex Visualization in Cells via Antibody and Fluorescence Probe. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 383-395. <a href="https://doi.org/10.1007/978-1-4939-9666-7_24" title="DOI URL">https://doi.org/10.1007/978-1-4939-9666-7_24</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-9666-7_24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-9666-7_24%26sid%3Dliteratum%253Aachs%26atitle%3DG-Quadruplex%252BVisualization%252Bin%252BCells%252Bvia%252BAntibody%252Band%252BFluorescence%252BProbe%26aulast%3DNadai%26aufirst%3DMatteo%26date%3D2019%26date%3D2019%26spage%3D383%26epage%3D395%26pub%3DSpringer%2520New%2520York%26atitle%3DG-Quadruplex%252BNucleic%252BAcids%26aulast%3DYang%26aufirst%3DDanzhou%26date%3D2019%26volume%3D2035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Subhajit  Makar</span>, <span class="hlFld-ContribAuthor ">Tanmay  Saha</span>, <span class="hlFld-ContribAuthor ">Sushil K.  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Naphthalene, a versatile platform in medicinal chemistry: Sky-high perspective. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>161 </em>, 252-276. <a href="https://doi.org/10.1016/j.ejmech.2018.10.018" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.10.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.10.018%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNaphthalene%25252C%252Ba%252Bversatile%252Bplatform%252Bin%252Bmedicinal%252Bchemistry%25253A%252BSky-high%252Bperspective%26aulast%3DMakar%26aufirst%3DSubhajit%26date%3D2019%26volume%3D161%26spage%3D252%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Enrico  Lavezzo</span>, <span class="hlFld-ContribAuthor ">Michele  Berselli</span>, <span class="hlFld-ContribAuthor ">Ilaria  Frasson</span>, <span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Alessandra R.  Brazzale</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>, <span class="hlFld-ContribAuthor ">Stefano  Toppo</span>, . </span><span class="cited-content_cbyCitation_article-title">G-quadruplex forming sequences in the genome of all known human viruses: A comprehensive guide. </span><span class="cited-content_cbyCitation_journal-name">PLOS Computational Biology</span><span> <strong>2018,</strong> <em>14 </em>
                                    (12)
                                     , e1006675. <a href="https://doi.org/10.1371/journal.pcbi.1006675" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1006675</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1006675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1006675%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Computational%2520Biology%26atitle%3DG-quadruplex%252Bforming%252Bsequences%252Bin%252Bthe%252Bgenome%252Bof%252Ball%252Bknown%252Bhuman%252Bviruses%25253A%252BA%252Bcomprehensive%252Bguide%26aulast%3DLavezzo%26aufirst%3DEnrico%26date%3D2018%26date%3D2018%26volume%3D14%26issue%3D12%26spage%3De1006675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiang-Nan  kong</span>, <span class="hlFld-ContribAuthor ">Chao  Zhang</span>, <span class="hlFld-ContribAuthor ">Yan-Ce  Zhu</span>, <span class="hlFld-ContribAuthor ">Kai  Zhong</span>, <span class="hlFld-ContribAuthor ">Jiang  Wang</span>, <span class="hlFld-ContribAuthor ">Bei-Bei  Chu</span>, <span class="hlFld-ContribAuthor ">Guo-Yu  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and characterization of G-quadruplex formation within the EP0 promoter of pseudorabies virus. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-32222-7" title="DOI URL">https://doi.org/10.1038/s41598-018-32222-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-32222-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-32222-7%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DIdentification%252Band%252Bcharacterization%252Bof%252BG-quadruplex%252Bformation%252Bwithin%252Bthe%252BEP0%252Bpromoter%252Bof%252Bpseudorabies%252Bvirus%26aulast%3Dkong%26aufirst%3DJiang-Nan%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michela  Zuffo</span>, <span class="hlFld-ContribAuthor ">Aurore  Guédin</span>, <span class="hlFld-ContribAuthor ">Emma-Dune  Leriche</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Valérie  Gabelica</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Mergny</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>. </span><span class="cited-content_cbyCitation_article-title">More is not always better: finding the right trade-off between affinity and selectivity of a G-quadruplex ligand. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2018,</strong> <em>46 </em>
                                    (19)
                                     , e115-e115. <a href="https://doi.org/10.1093/nar/gky607" title="DOI URL">https://doi.org/10.1093/nar/gky607</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gky607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgky607%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DMore%252Bis%252Bnot%252Balways%252Bbetter%25253A%252Bfinding%252Bthe%252Bright%252Btrade-off%252Bbetween%252Baffinity%252Band%252Bselectivity%252Bof%252Ba%252BG-quadruplex%252Bligand%26aulast%3DZuffo%26aufirst%3DMichela%26date%3D2018%26date%3D2018%26volume%3D46%26issue%3D19%26spage%3De115%26epage%3De115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Platella</span>, <span class="hlFld-ContribAuthor ">Domenica  Musumeci</span>, <span class="hlFld-ContribAuthor ">Angela  Arciello</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>, <span class="hlFld-ContribAuthor ">Antonio  Randazzo</span>, <span class="hlFld-ContribAuthor ">Jussara  Amato</span>, <span class="hlFld-ContribAuthor ">Bruno  Pagano</span>, <span class="hlFld-ContribAuthor ">Daniela  Montesarchio</span>. </span><span class="cited-content_cbyCitation_article-title">Controlled Pore Glass-based oligonucleotide affinity support: towards High Throughput Screening methods for the identification of conformation-selective G-quadruplex ligands. </span><span class="cited-content_cbyCitation_journal-name">Analytica Chimica Acta</span><span> <strong>2018,</strong> <em>1030 </em>, 133-141. <a href="https://doi.org/10.1016/j.aca.2018.04.071" title="DOI URL">https://doi.org/10.1016/j.aca.2018.04.071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.aca.2018.04.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.aca.2018.04.071%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytica%2520Chimica%2520Acta%26atitle%3DControlled%252BPore%252BGlass-based%252Boligonucleotide%252Baffinity%252Bsupport%25253A%252Btowards%252BHigh%252BThroughput%252BScreening%252Bmethods%252Bfor%252Bthe%252Bidentification%252Bof%252Bconformation-selective%252BG-quadruplex%252Bligands%26aulast%3DPlatella%26aufirst%3DChiara%26date%3D2018%26volume%3D1030%26spage%3D133%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H Steven  Seifert</span>. </span><span class="cited-content_cbyCitation_article-title">Above and Beyond Watson and Crick: Guanine Quadruplex Structures and Microbes. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Microbiology</span><span> <strong>2018,</strong> <em>72 </em>
                                    (1)
                                     , 49-69. <a href="https://doi.org/10.1146/annurev-micro-090817-062629" title="DOI URL">https://doi.org/10.1146/annurev-micro-090817-062629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-micro-090817-062629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-micro-090817-062629%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Microbiology%26atitle%3DAbove%252Band%252BBeyond%252BWatson%252Band%252BCrick%25253A%252BGuanine%252BQuadruplex%252BStructures%252Band%252BMicrobes%26aulast%3DSeifert%26aufirst%3DH%2BSteven%26date%3D2018%26volume%3D72%26issue%3D1%26spage%3D49%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Tassinari</span>, <span class="hlFld-ContribAuthor ">Alberto  Lena</span>, <span class="hlFld-ContribAuthor ">Elena  Butovskaya</span>, <span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Sara  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">A Fragment-Based Approach for the Development of G-Quadruplex Ligands: Role of the Amidoxime Moiety. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (8)
                                     , 1874. <a href="https://doi.org/10.3390/molecules23081874" title="DOI URL">https://doi.org/10.3390/molecules23081874</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23081874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23081874%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DA%252BFragment-Based%252BApproach%252Bfor%252Bthe%252BDevelopment%252Bof%252BG-Quadruplex%252BLigands%25253A%252BRole%252Bof%252Bthe%252BAmidoxime%252BMoiety%26aulast%3DTassinari%26aufirst%3DMartina%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D8%26spage%3D1874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emanuela  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Sara N  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">G-quadruplexes and G-quadruplex ligands: targets and tools in antiviral therapy. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2018,</strong> <em>46 </em>
                                    (7)
                                     , 3270-3283. <a href="https://doi.org/10.1093/nar/gky187" title="DOI URL">https://doi.org/10.1093/nar/gky187</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gky187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgky187%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DG-quadruplexes%252Band%252BG-quadruplex%252Bligands%25253A%252Btargets%252Band%252Btools%252Bin%252Bantiviral%252Btherapy%26aulast%3DRuggiero%26aufirst%3DEmanuela%26date%3D2018%26date%3D2018%26volume%3D46%26issue%3D7%26spage%3D3270%26epage%3D3283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maryam  Zahin</span>, <span class="hlFld-ContribAuthor ">William L.  Dean</span>, <span class="hlFld-ContribAuthor ">Shin-je  Ghim</span>, <span class="hlFld-ContribAuthor ">Joongho  Joh</span>, <span class="hlFld-ContribAuthor ">Robert D.  Gray</span>, <span class="hlFld-ContribAuthor ">Sujita  Khanal</span>, <span class="hlFld-ContribAuthor ">Gregory D.  Bossart</span>, <span class="hlFld-ContribAuthor ">Antonio A.  Mignucci-Giannoni</span>, <span class="hlFld-ContribAuthor ">Eric C.  Rouchka</span>, <span class="hlFld-ContribAuthor ">Alfred B.  Jenson</span>, <span class="hlFld-ContribAuthor ">John O.  Trent</span>, <span class="hlFld-ContribAuthor ">Jonathan B.  Chaires</span>, <span class="hlFld-ContribAuthor ">Julia H.  Chariker</span>, . </span><span class="cited-content_cbyCitation_article-title">Identification of G-quadruplex forming sequences in three manatee papillomaviruses. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (4)
                                     , e0195625. <a href="https://doi.org/10.1371/journal.pone.0195625" title="DOI URL">https://doi.org/10.1371/journal.pone.0195625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0195625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0195625%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIdentification%252Bof%252BG-quadruplex%252Bforming%252Bsequences%252Bin%252Bthree%252Bmanatee%252Bpapillomaviruses%26aulast%3DZahin%26aufirst%3DMaryam%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D4%26spage%3De0195625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Y. V.  Suseela</span>, <span class="hlFld-ContribAuthor ">Nagarjun  Narayanaswamy</span>, <span class="hlFld-ContribAuthor ">Sumon  Pratihar</span>, <span class="hlFld-ContribAuthor ">Thimmaiah  Govindaraju</span>. </span><span class="cited-content_cbyCitation_article-title">Far-red fluorescent probes for canonical and non-canonical nucleic acid structures: current progress and future implications. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2018,</strong> <em>47 </em>
                                    (3)
                                     , 1098-1131. <a href="https://doi.org/10.1039/C7CS00774D" title="DOI URL">https://doi.org/10.1039/C7CS00774D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7CS00774D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7CS00774D%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DFar-red%252Bfluorescent%252Bprobes%252Bfor%252Bcanonical%252Band%252Bnon-canonical%252Bnucleic%252Bacid%252Bstructures%25253A%252Bcurrent%252Bprogress%252Band%252Bfuture%252Bimplications%26aulast%3DSuseela%26aufirst%3DY.%2BV.%26date%3D2018%26date%3D2018%26volume%3D47%26issue%3D3%26spage%3D1098%26epage%3D1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Enrico  Lavezzo</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Conserved presence of G-quadruplex forming sequences in the Long Terminal Repeat Promoter of Lentiviruses. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-02291-1" title="DOI URL">https://doi.org/10.1038/s41598-017-02291-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-02291-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-02291-1%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DConserved%252Bpresence%252Bof%252BG-quadruplex%252Bforming%252Bsequences%252Bin%252Bthe%252BLong%252BTerminal%252BRepeat%252BPromoter%252Bof%252BLentiviruses%26aulast%3DPerrone%26aufirst%3DRosalba%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Callegaro</span>, <span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Matteo  Scalabrin</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">A core extended naphtalene diimide G-quadruplex ligand potently inhibits herpes simplex virus 1 replication. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-02667-3" title="DOI URL">https://doi.org/10.1038/s41598-017-02667-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-02667-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-02667-3%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DA%252Bcore%252Bextended%252Bnaphtalene%252Bdiimide%252BG-quadruplex%252Bligand%252Bpotently%252Binhibits%252Bherpes%252Bsimplex%252Bvirus%252B1%252Breplication%26aulast%3DCallegaro%26aufirst%3DSara%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Enrico  Lavezzo</span>, <span class="hlFld-ContribAuthor ">Erika  Riello</span>, <span class="hlFld-ContribAuthor ">Riccardo  Manganelli</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Stefano  Toppo</span>, <span class="hlFld-ContribAuthor ">Roberta  Provvedi</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Mapping and characterization of G-quadruplexes in Mycobacterium tuberculosis gene promoter regions. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-05867-z" title="DOI URL">https://doi.org/10.1038/s41598-017-05867-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-05867-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-05867-z%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DMapping%252Band%252Bcharacterization%252Bof%252BG-quadruplexes%252Bin%252BMycobacterium%252Btuberculosis%252Bgene%252Bpromoter%252Bregions%26aulast%3DPerrone%26aufirst%3DRosalba%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michela  Zuffo</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Silvia  Botti</span>, <span class="hlFld-ContribAuthor ">Greta  Bergamaschi</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>. </span><span class="cited-content_cbyCitation_article-title">G-quadruplex fluorescence sensing by core-extended naphthalene diimides. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2017,</strong> <em>1861 </em>
                                    (5)
                                     , 1303-1311. <a href="https://doi.org/10.1016/j.bbagen.2016.11.034" title="DOI URL">https://doi.org/10.1016/j.bbagen.2016.11.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2016.11.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2016.11.034%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DG-quadruplex%252Bfluorescence%252Bsensing%252Bby%252Bcore-extended%252Bnaphthalene%252Bdiimides%26aulast%3DZuffo%26aufirst%3DMichela%26date%3D2017%26volume%3D1861%26issue%3D5%26spage%3D1303%26epage%3D1311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Lago</span>, <span class="hlFld-ContribAuthor ">Elena  Tosoni</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Manlio  Palumbo</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">The cellular protein nucleolin preferentially binds long-looped G-quadruplex nucleic acids. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2017,</strong> <em>1861 </em>
                                    (5)
                                     , 1371-1381. <a href="https://doi.org/10.1016/j.bbagen.2016.11.036" title="DOI URL">https://doi.org/10.1016/j.bbagen.2016.11.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2016.11.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2016.11.036%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DThe%252Bcellular%252Bprotein%252Bnucleolin%252Bpreferentially%252Bbinds%252Blong-looped%252BG-quadruplex%252Bnucleic%252Bacids%26aulast%3DLago%26aufirst%3DSara%26date%3D2017%26volume%3D1861%26issue%3D5%26spage%3D1371%26epage%3D1381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matteo  Scalabrin</span>, <span class="hlFld-ContribAuthor ">Ilaria  Frasson</span>, <span class="hlFld-ContribAuthor ">Emanuela  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Elena  Tosoni</span>, <span class="hlFld-ContribAuthor ">Sara  Lago</span>, <span class="hlFld-ContribAuthor ">Martina  Tassinari</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">The cellular protein hnRNP A2/B1 enhances HIV-1 transcription by unfolding LTR promoter G-quadruplexes. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep45244" title="DOI URL">https://doi.org/10.1038/srep45244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep45244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep45244%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DThe%252Bcellular%252Bprotein%252BhnRNP%252BA2%25252FB1%252Benhances%252BHIV-1%252Btranscription%252Bby%252Bunfolding%252BLTR%252Bpromoter%252BG-quadruplexes%26aulast%3DScalabrin%26aufirst%3DMatteo%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laureen  Bonnat</span>, <span class="hlFld-ContribAuthor ">Laure  Bar</span>, <span class="hlFld-ContribAuthor ">Béatrice  Génnaro</span>, <span class="hlFld-ContribAuthor ">Hugues  Bonnet</span>, <span class="hlFld-ContribAuthor ">Olivier  Jarjayes</span>, <span class="hlFld-ContribAuthor ">Fabrice  Thomas</span>, <span class="hlFld-ContribAuthor ">Jérôme  Dejeu</span>, <span class="hlFld-ContribAuthor ">Eric  Defrancq</span>, <span class="hlFld-ContribAuthor ">Thomas  Lavergne</span>. </span><span class="cited-content_cbyCitation_article-title">Template‐Mediated Stabilization of a DNA G‐Quadruplex formed in the HIV‐1 Promoter and Comparative Binding Studies. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (23)
                                     , 5602-5613. <a href="https://doi.org/10.1002/chem.201700417" title="DOI URL">https://doi.org/10.1002/chem.201700417</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201700417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201700417%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DTemplate%2525E2%252580%252590Mediated%252BStabilization%252Bof%252Ba%252BDNA%252BG%2525E2%252580%252590Quadruplex%252Bformed%252Bin%252Bthe%252BHIV%2525E2%252580%2525901%252BPromoter%252Band%252BComparative%252BBinding%252BStudies%26aulast%3DBonnat%26aufirst%3DLaureen%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D23%26spage%3D5602%26epage%3D5613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matilde  Arévalo-Ruiz</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Efres  Belmonte-Reche</span>, <span class="hlFld-ContribAuthor ">Aurore  De Rache</span>, <span class="hlFld-ContribAuthor ">Jenny  Campos-Salinas</span>, <span class="hlFld-ContribAuthor ">Ricardo  Lucas</span>, <span class="hlFld-ContribAuthor ">Eva  Falomir</span>, <span class="hlFld-ContribAuthor ">Miguel  Carda</span>, <span class="hlFld-ContribAuthor ">José María  Pérez-Victoria</span>, <span class="hlFld-ContribAuthor ">Jean-Louis  Mergny</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>, <span class="hlFld-ContribAuthor ">Juan Carlos  Morales</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Binding Properties, and Differences in Cell Uptake of G-Quadruplex Ligands Based on Carbohydrate Naphthalene Diimide Conjugates. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (9)
                                     , 2157-2164. <a href="https://doi.org/10.1002/chem.201604886" title="DOI URL">https://doi.org/10.1002/chem.201604886</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201604886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201604886%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DSynthesis%25252C%252BBinding%252BProperties%25252C%252Band%252BDifferences%252Bin%252BCell%252BUptake%252Bof%2525E2%252580%252585G-Quadruplex%252BLigands%252BBased%252Bon%252BCarbohydrate%252BNaphthalene%252BDiimide%252BConjugates%26aulast%3DAr%25C3%25A9valo-Ruiz%26aufirst%3DMatilde%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D9%26spage%3D2157%26epage%3D2164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Feng-Hua  Zhang</span>, <span class="hlFld-ContribAuthor ">Bikash  Debnath</span>, <span class="hlFld-ContribAuthor ">Zhong-Liang  Xu</span>, <span class="hlFld-ContribAuthor ">Liu-Meng  Yang</span>, <span class="hlFld-ContribAuthor ">Li-Rui  Song</span>, <span class="hlFld-ContribAuthor ">Yong-Tang  Zheng</span>, <span class="hlFld-ContribAuthor ">Nouri  Neamati</span>, <span class="hlFld-ContribAuthor ">Ya-Qiu  Long</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 3-hydroxypicolinamides as selective inhibitors of HIV-1 integrase-LEDGF/p75 interaction. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>125 </em>, 1051-1063. <a href="https://doi.org/10.1016/j.ejmech.2016.10.045" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.10.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.10.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.10.045%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252B3-hydroxypicolinamides%252Bas%252Bselective%252Binhibitors%252Bof%2525C2%2525A0HIV-1%252Bintegrase-LEDGF%25252Fp75%252Binteraction%26aulast%3DZhang%26aufirst%3DFeng-Hua%26date%3D2017%26volume%3D125%26spage%3D1051%26epage%3D1063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">F.  Doria</span>, <span class="hlFld-ContribAuthor ">M.  Nadai</span>, <span class="hlFld-ContribAuthor ">M.  Zuffo</span>, <span class="hlFld-ContribAuthor ">R.  Perrone</span>, <span class="hlFld-ContribAuthor ">M.  Freccero</span>, <span class="hlFld-ContribAuthor ">S. N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">A red-NIR fluorescent dye detecting nuclear DNA G-quadruplexes: in vitro analysis and cell imaging. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2017,</strong> <em>53 </em>
                                    (14)
                                     , 2268-2271. <a href="https://doi.org/10.1039/C6CC08492C" title="DOI URL">https://doi.org/10.1039/C6CC08492C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6CC08492C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6CC08492C%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DA%252Bred-NIR%252Bfluorescent%252Bdye%252Bdetecting%252Bnuclear%252BDNA%252BG-quadruplexes%25253A%252Bin%252Bvitro%252Banalysis%252Band%252Bcell%252Bimaging%26aulast%3DDoria%26aufirst%3DF.%26date%3D2017%26date%3D2017%26volume%3D53%26issue%3D14%26spage%3D2268%26epage%3D2271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Artusi</span>, <span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Sara  Lago</span>, <span class="hlFld-ContribAuthor ">Paolo  Raffa</span>, <span class="hlFld-ContribAuthor ">Enzo  Di Iorio</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Visualization of DNA G-quadruplexes in herpes simplex virus 1-infected cells. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2016,</strong> <em>61 </em>, gkw968. <a href="https://doi.org/10.1093/nar/gkw968" title="DOI URL">https://doi.org/10.1093/nar/gkw968</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkw968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkw968%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DVisualization%252Bof%252BDNA%252BG-quadruplexes%252Bin%252Bherpes%252Bsimplex%252Bvirus%252B1-infected%252Bcells%26aulast%3DArtusi%26aufirst%3DSara%26date%3D2016%26date%3D2016%26volume%3D61%26spage%3Dgkw968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Giosuè  Costa</span>, <span class="hlFld-ContribAuthor ">Giovanna  Sattin</span>, <span class="hlFld-ContribAuthor ">Caroline  Gallati</span>, <span class="hlFld-ContribAuthor ">Greta  Bergamaschi</span>, <span class="hlFld-ContribAuthor ">Federica  Moraca</span>, <span class="hlFld-ContribAuthor ">Stefano  Alcaro</span>, <span class="hlFld-ContribAuthor ">Mauro  Freccero</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Extended Naphthalene Diimides with Donor/Acceptor Hydrogen-Bonding Properties Targeting G-Quadruplex Nucleic Acids. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>2016 </em>
                                    (28)
                                     , 4824-4833. <a href="https://doi.org/10.1002/ejoc.201600757" title="DOI URL">https://doi.org/10.1002/ejoc.201600757</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201600757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201600757%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DExtended%252BNaphthalene%252BDiimides%252Bwith%252BDonor%25252FAcceptor%252BHydrogen-Bonding%252BProperties%252BTargeting%252BG-Quadruplex%252BNucleic%252BAcids%26aulast%3DDoria%26aufirst%3DFilippo%26date%3D2016%26date%3D2016%26volume%3D2016%26issue%3D28%26spage%3D4824%26epage%3D4833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Beatrice  De Nicola</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Lech</span>, <span class="hlFld-ContribAuthor ">Brahim  Heddi</span>, <span class="hlFld-ContribAuthor ">Sagar  Regmi</span>, <span class="hlFld-ContribAuthor ">Ilaria  Frasson</span>, <span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>, <span class="hlFld-ContribAuthor ">Anh Tuân  Phan</span>. </span><span class="cited-content_cbyCitation_article-title">Structure and possible function of a G-quadruplex in the long terminal repeat of the proviral HIV-1 genome. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2016,</strong> <em>44 </em>
                                    (13)
                                     , 6442-6451. <a href="https://doi.org/10.1093/nar/gkw432" title="DOI URL">https://doi.org/10.1093/nar/gkw432</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkw432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkw432%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DStructure%252Band%252Bpossible%252Bfunction%252Bof%252Ba%252BG-quadruplex%252Bin%252Bthe%252Blong%252Bterminal%252Brepeat%252Bof%252Bthe%252Bproviral%252BHIV-1%252Bgenome%26aulast%3DDe%25C2%25A0Nicola%26aufirst%3DBeatrice%26date%3D2016%26date%3D2016%26volume%3D44%26issue%3D13%26spage%3D6442%26epage%3D6451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosalba  Perrone</span>, <span class="hlFld-ContribAuthor ">Elena  Butovskaya</span>, <span class="hlFld-ContribAuthor ">Sara  Lago</span>, <span class="hlFld-ContribAuthor ">Alfredo  Garzino-Demo</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Giorgio  Palù</span>, <span class="hlFld-ContribAuthor ">Sara N.  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">The G-quadruplex-forming aptamer AS1411 potently inhibits HIV-1 attachment to the host cell. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Antimicrobial Agents</span><span> <strong>2016,</strong> <em>47 </em>
                                    (4)
                                     , 311-316. <a href="https://doi.org/10.1016/j.ijantimicag.2016.01.016" title="DOI URL">https://doi.org/10.1016/j.ijantimicag.2016.01.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijantimicag.2016.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijantimicag.2016.01.016%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Antimicrobial%2520Agents%26atitle%3DThe%252BG-quadruplex-forming%252Baptamer%252BAS1411%252Bpotently%252Binhibits%252BHIV-1%252Battachment%252Bto%252Bthe%252Bhost%252Bcell%26aulast%3DPerrone%26aufirst%3DRosalba%26date%3D2016%26volume%3D47%26issue%3D4%26spage%3D311%26epage%3D316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Structure of BRACO-19 and Core-Extended NDIs (c-exNDIs) Reported in the Literature,<a onclick="showRef(event, 'ref44 ref48 ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref44 ref48 ref49 ref50 ref51">(44, 48-51)</a> and the New Synthesized Water-Soluble c-exNDIs (lower right panel)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Optimized Synthetic Protocol for the Preparation of Water-Soluble <b>2</b>–<b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) DMF, 70°C, 16 h, under N<sub>2</sub>; (b) <i>N</i>,<i>N</i>-dimethylpropylamine, MW, 150°C 3 min, closed vessel; (c) MeI, CHCl<sub>3</sub>, rt, 16 h.</p></p></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. G4 stabilization by the <b>c-exNDIs</b>. (A) G4-forming sequences used in this study. (B) FRET analysis. Δ<i>T</i><sub>m</sub> values of HIV-1 LTR G4-folded sequences (LTR-III and LTR-IV), the G4-folded telomeric sequence (hTel), and a control double-stranded sequence (dsDNA) obtained upon addition of equimolar amounts of <b>2</b>–<b>11</b> (0.25 μM). (C) <i>Taq</i> polymerase stop assay. <i>Taq</i> polymerization was performed in the presence/absence of K<sup>+</sup> and <b>2</b>, as indicated, on the hTel, LTR-III, and LTR-IV templates, which were previously folded into G4. A control template unable to fold into G4 was also used (non-G4 cnt). Stop regions are highlighted by vertical bars. (D) Quantification of the intensity of the stop bands obtained in the <i>Taq</i> polymerase stop assay. Stop band intensity of the treated samples were normalized vs the untreated sample.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Affinity and binding site analysis. (A) ESI/MS competition assay. Representative spectrum. Equimolar amounts of LTR-III and hTel were incubated with 2-fold excess of <b>2</b> and analyzed by ESI/MS. The identity of <i>m</i>/<i>z</i> signals is shown above each signal. RI stands for relative intensity. (B) SPR analysis of <b>2</b> binding toward the LTRs and hTel G4s. Representative sensorgram of <b>2</b> binding to LTR-III G4. Biosensor data were collected at 25 °C; the compound was injected at increasing concentrations (26, 40, 59, 89, 133, 200 nM). Sensorgrams are shown as gray lines. Dotted black lines represent local 1:1 binding fits obtained for a calculated <i>R</i><sub>max</sub>. (C,D) CL-mediated footprinting. LTR-IV (C) and LTR-III (D) oligonucleotides were G4 folded, incubated with increasing amounts of <b>2</b> (lanes 5–13), alkylated with CL (CL lanes), and subjected to hot piperidine treatment (PIP lanes) to induce cleavage of the phosphodiester bond at the alkylated bases. (C) Lanes 1–4 are control lanes not treated with the <b>c-exNDI</b>. The symbols * and ○ indicate base protection and exposure, respectively. M is a marker lane obtained by the Maxam and Gilbert sequencing of the oligonucleotides. The oligonucleotide sequence is shown on the left. The vertical dotted bar indicates the region of the LTR-III shared with LTR-IV. (E) Schematics of the LTR-IV and LTR-III G4 conformations. The represented LTR-IV G4 topology has been solved by NMR (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2N4Y">2N4Y</a>); the shown LTR-III G4 conformation derives from previous low resolution structural data.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/medium/jm-2015-012832_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antiviral activity of <b>c-exNDIs</b> and mechanism of action. (A) TZM-bl cells were infected with HIV-1 NL4-3 strain and treated with increasing concentrations (3.9–1000 nM) of the <b>c-exNDIs</b>. Uninfected cells were treated in the same conditions. Antiviral activity, expressed as relative luciferase units (RLU), was measured on infected cells, while cell viability was in parallel obtained with a MTT assay on the uninfected cells. The vertical dotted lines define the therapeutic window (TW) where the compound is active against the virus and not toxic against the cells. This figure is representative of the antiviral/cytotoxicity/TW experiments performed with the whole series of <b>c-exNDI</b> compounds. (B) Entry assay. TZM-bl cells were treated with the compound <b>2</b> (50–100 nM), AZT (0.5 μg/mL), or the entry inhibitor DS (100 μg/mL) at 0, 1, or 2 h pi. Virus production was measured at 31 h pi as RLU. The control depicts cells infected with the virus lacking compound treatment. (C) Effect of <b>2</b> on the LTR promoter activity compared to cytotoxicity. HeLa cells with a stably transfected LTR-EGFP reporter plasmid were treated with increasing concentrations of <b>2</b> (62.5–250 nM). EGFP mean was measured by FACS analysis. Cytotoxicity was estimate in parallel by gating on forward scatter (FSC) vs side scatter (SSC) using untreated cells as reference. (D) Specificity of the <b>2</b>-induced inhibition of LTR promoter activity on the G4 LTR conformation. 293T cells were transiently transfected with plasmids with the wt or M4 + 5 mutant LTR promoter sequence upstream a luciferase reporter gene. Luciferase signals were measured as a function of increasing <b>2</b> amounts and normalized to total protein content.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b01283/20151218/images/large/jm-2015-012832_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01283&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27072" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27072" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 70 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Lipps, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D.</span><span> </span><span class="NLM_article-title">G-quadruplex structures: in vivo evidence and function</span> <span class="citation_source-journal">Trends Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1016/j.tcb.2009.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.tcb.2009.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19589679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=414-422&author=H.+J.+Lippsauthor=D.+Rhodes&title=G-quadruplex+structures%3A+in+vivo+evidence+and+function&doi=10.1016%2Fj.tcb.2009.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex structures: in vivo evidence and function</span></div><div class="casAuthors">Lipps, Hans J.; Rhodes, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although many biochem. and structural studies have demonstrated that DNA sequences contg. runs of adjacent guanines spontaneously fold into G-quadruplex DNA structures in vitro, only recently has evidence started to accumulate for their presence and function in vivo.  Genome-wide analyses have revealed that functional genomic regions from highly divergent organisms are enriched in DNA sequences with G-quadruplex-forming potential, suggesting that G-quadruplexes could provide a nucleic-acid-based mechanism for regulating telomere maintenance, as well as transcription, replication and translation.  Here, we review recent studies aimed at uncovering the in vivo presence and function of G-quadruplexes in genomes and RNA, with a particular focus on telomeric G-quadruplexes and how their formation and resoln. is regulated to permit telomere synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5cRHGDFqMhrVg90H21EOLACvtfcHk0liBotHVE80RPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO&md5=f9f684e81a34d37ec1ae44ef7764c112</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2009.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2009.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DLipps%26aufirst%3DH.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%26atitle%3DG-quadruplex%2520structures%253A%2520in%2520vivo%2520evidence%2520and%2520function%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D414%26epage%3D422%26doi%3D10.1016%2Fj.tcb.2009.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, W.</span><span> </span><span class="NLM_article-title">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">366</span><span class="refDoi"> DOI: 10.1038/334364a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2F334364a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=3393228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=1988&pages=364-366&author=D.+Senauthor=W.+Gilbert&title=Formation+of+parallel+four-stranded+complexes+by+guanine-rich+motifs+in+DNA+and+its+implications+for+meiosis&doi=10.1038%2F334364a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span></div><div class="casAuthors">Sen, Dipankar; Gilbert, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6180</span>),
    <span class="NLM_cas:pages">364-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Single-stranded DNA contg. short guanine-rich motifs will self-assoc. at physiol. salt concns. to make 4-stranded structures in which the strands run in parallel fashion.  These complexes are apparently held together by guanines bonded to each other by Hoogsteen pairing.  It is speculated that this self-recognition of guanine-rich motifs of DNA serves to bring together, and to zipper up in register, the 4 homologous chromatids during meiosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr5smJJlAP6rVg90H21EOLACvtfcHk0ljlKHa4AJafLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D&md5=96183a2f707c4395862b96c750a0c53c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F334364a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F334364a0%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DW.%26atitle%3DFormation%2520of%2520parallel%2520four-stranded%2520complexes%2520by%2520guanine-rich%2520motifs%2520in%2520DNA%2520and%2520its%2520implications%2520for%2520meiosis%26jtitle%3DNature%26date%3D1988%26volume%3D334%26spage%3D364%26epage%3D366%26doi%3D10.1038%2F334364a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span> </span><span class="NLM_article-title">G-quadruplex nucleic acids and human disease</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">3470</span><span class="NLM_x">–</span> <span class="NLM_lpage">3488</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2010.07760.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1111%2Fj.1742-4658.2010.07760.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=20670277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=3470-3488&author=Y.+Wuauthor=R.+M.+Brosh&title=G-quadruplex+nucleic+acids+and+human+disease&doi=10.1111%2Fj.1742-4658.2010.07760.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex nucleic acids and human disease</span></div><div class="casAuthors">Wu, Yuliang; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3470-3488</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Alternate DNA structures that deviate from B-form double-stranded DNA such as G-quadruplex (G4) DNA can be formed by sequences that are widely distributed throughout the human genome.  G-quadruplex secondary structures, formed by the stacking of planar quartets composed of four guanines that interact by Hoogsteen hydrogen bonding, can affect cellular DNA replication and transcription, and influence genomic stability.  The unique metab. of G-rich chromosomal regions that potentially form quadruplexes may influence a no. of biol. processes including Ig gene rearrangements, promoter activation and telomere maintenance.  A no. of human diseases are characterized by telomere defects, and it is proposed that G-quadruplex structures which form at telomere ends play an important role in telomere stability.  Evidence from cellular studies and model organisms suggests that diseases with known defects in G4 DNA helicases are likely to be perturbed in telomere maintenance and cellular DNA replication.  In this minireview, we discuss the connections of G-quadruplex nucleic acids to human genetic diseases and cancer based on the recent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbqNRG82SQcbVg90H21EOLACvtfcHk0ljlKHa4AJafLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO&md5=47fd16aa812516caf20b0b46f3544301</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2010.07760.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2010.07760.x%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DG-quadruplex%2520nucleic%2520acids%2520and%2520human%2520disease%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D3470%26epage%3D3488%26doi%3D10.1111%2Fj.1742-4658.2010.07760.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, W.</span><span> </span><span class="NLM_article-title">A sodium-potassium switch in the formation of four-stranded G4-DNA</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">344</span><span class="NLM_x">, </span> <span class="NLM_fpage">410</span><span class="NLM_x">–</span> <span class="NLM_lpage">414</span><span class="refDoi"> DOI: 10.1038/344410a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2F344410a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=2320109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaK3cXitlWnsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=1990&pages=410-414&author=D.+Senauthor=W.+Gilbert&title=A+sodium-potassium+switch+in+the+formation+of+four-stranded+G4-DNA&doi=10.1038%2F344410a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A sodium-potassium switch in the formation of four-stranded G4-DNA</span></div><div class="casAuthors">Sen, Dipankar; Gilbert, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">6265</span>),
    <span class="NLM_cas:pages">410-14</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Single-stranded complex guanine-rich DNA sequences from chromosomal telomeres and elsewhere can assoc. to form stable parallel 4-stranded structures termed G4-DNA by a process that is anomalously dependent on the particular alkali metal cation that is present.  The anomaly, which is not found in the formation of G4-DNA by oligonucleotides contg. short, single runs of three or more guanines, is caused by K+ excessively stabilizing fold-back intermediate structures, or pathway byproducts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJT86WNyFCnrVg90H21EOLACvtfcHk0ljlKHa4AJafLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitlWnsL4%253D&md5=bbd66d0e94a0a6b4abf65e9ab85f59f0</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2F344410a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F344410a0%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DW.%26atitle%3DA%2520sodium-potassium%2520switch%2520in%2520the%2520formation%2520of%2520four-stranded%2520G4-DNA%26jtitle%3DNature%26date%3D1990%26volume%3D344%26spage%3D410%26epage%3D414%26doi%3D10.1038%2F344410a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Taylor, J. P.</span><span> </span><span class="NLM_article-title">Neurodegenerative diseases: G-quadruplex poses quadruple threat</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1038/nature13067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnature13067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2014&pages=175-177&author=J.+P.+Taylor&title=Neurodegenerative+diseases%3A+G-quadruplex+poses+quadruple+threat&doi=10.1038%2Fnature13067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature13067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13067%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26atitle%3DNeurodegenerative%2520diseases%253A%2520G-quadruplex%2520poses%2520quadruple%2520threat%26jtitle%3DNature%26date%3D2014%26volume%3D507%26spage%3D175%26epage%3D177%26doi%3D10.1038%2Fnature13067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Campbell, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">G-quadruplexes and metal ions</span> <span class="citation_source-journal">Met. Ions Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">119</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1007/978-94-007-2172-2_4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1007%2F978-94-007-2172-2_4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22210337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWjtLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=119-134&author=N.+H.+Campbellauthor=S.+Neidle&title=G-quadruplexes+and+metal+ions&doi=10.1007%2F978-94-007-2172-2_4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes and metal ions</span></div><div class="casAuthors">Campbell, Nancy H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Metal Ions in Life Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">Interplay between Metal Ions and Nucleic Acids</span>),
    <span class="NLM_cas:pages">119-134</span>CODEN:
                <span class="NLM_cas:coden">MILSCT</span>;
        ISSN:<span class="NLM_cas:issn">1559-0836</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Metal ions stabilize quadruplex nucleic acids by coordinating the O6 guanine atoms from G-quartets.  These quartets form the basic motif of quadruplex structures.  This article systematically surveys the available crystallog. data on native quadruplexes, their ligand complexes and (in one instance) a protein complex.  Three categories of quadruplex are examd., tetramol., bimol., and intramol.: all are formed by telomeric nucleic acid sequences from human or ciliate organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIHVWG6zI41LVg90H21EOLACvtfcHk0liRK0sfU-UHmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWjtLzF&md5=f49615280560f3483cf9c3beda0a03cf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2F978-94-007-2172-2_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-94-007-2172-2_4%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DN.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DG-quadruplexes%2520and%2520metal%2520ions%26jtitle%3DMet.%2520Ions%2520Life%2520Sci.%26date%3D2012%26volume%3D10%26spage%3D119%26epage%3D134%26doi%3D10.1007%2F978-94-007-2172-2_4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Eddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maizels, N.</span><span> </span><span class="NLM_article-title">Gene function correlates with potential for G4 DNA formation in the human genome</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">3887</span><span class="NLM_x">–</span> <span class="NLM_lpage">3896</span><span class="refDoi"> DOI: 10.1093/nar/gkl529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl529" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=16914419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=3887-3896&author=J.+Eddyauthor=N.+Maizels&title=Gene+function+correlates+with+potential+for+G4+DNA+formation+in+the+human+genome&doi=10.1093%2Fnar%2Fgkl529"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Gene function correlates with potential for G4 DNA formation in the human genome</span></div><div class="casAuthors">Eddy, Johanna; Maizels, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3887-3896</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">G-rich genomic regions can form G4 DNA upon transcription or replication.  We have quantified the potential for G4 DNA formation (G4P) of the 16 654 genes in the human RefSeq database, and then correlated gene function with G4P.  We have found that very low and very high G4P correlates with specific functional classes of genes.  Notably, tumor suppressor genes have very low G4P and proto-oncogenes have very high G4P.  G4P of these genes is evenly distributed between exons and introns, and it does not reflect enrichment for CpG islands or local chromosomal environment.  These results show that genomic structure undergoes selection based on gene function.  Selection based on G4P could promote genomic stability (or instability) of specific classes of genes; or reflect mechanisms for global regulation of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPf7ajy8A2MLVg90H21EOLACvtfcHk0liRK0sfU-UHmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1yis7o%253D&md5=c5da07ec9e667a0fbb1843159e773eb8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl529%26sid%3Dliteratum%253Aachs%26aulast%3DEddy%26aufirst%3DJ.%26aulast%3DMaizels%26aufirst%3DN.%26atitle%3DGene%2520function%2520correlates%2520with%2520potential%2520for%2520G4%2520DNA%2520formation%2520in%2520the%2520human%2520genome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D3887%26epage%3D3896%26doi%3D10.1093%2Fnar%2Fgkl529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Huppert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">G-quadruplexes in promoters throughout the human genome</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">406</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1093/nar/gkl1057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl1057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=17169996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOls74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=406-413&author=J.+L.+Huppertauthor=S.+Balasubramanian&title=G-quadruplexes+in+promoters+throughout+the+human+genome&doi=10.1093%2Fnar%2Fgkl1057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes in promoters throughout the human genome</span></div><div class="casAuthors">Huppert, Julian L.; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">406-413</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Certain G-rich DNA sequences readily form four-stranded structures called G-quadruplexes.  These sequence motifs are located in telomeres as a repeated unit, and elsewhere in the genome, where their function is currently unknown.  It has been proposed that G-quadruplexes may be directly involved in gene regulation at the level of transcription.  In support of this hypothesis, we show that the promoter regions (1 kb upstream of the transcription start site TSS) of genes are significantly enriched in quadruplex motifs relative to the rest of the genome, with >40% of human gene promoters contg. one or more quadruplex motif.  Furthermore, these promoter quadruplexes strongly assoc. with nuclease hypersensitive sites identified throughout the genome via biochem. measurement.  Regions of the human genome that are both nuclease hypersensitive and within promoters show a remarkable (230-fold) enrichment of quadruplex elements, compared to the rest of the genome.  These quadruplex motifs identified in promoter regions also show an interesting structural bias towards more stable forms.  These observations support the proposal that promoter G-quadruplexes are directly involved in the regulation of gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofaKOSq_qjKrVg90H21EOLACvtfcHk0liRK0sfU-UHmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOls74%253D&md5=626903207653e1c97d568b9b50c093bd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl1057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl1057%26sid%3Dliteratum%253Aachs%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DG-quadruplexes%2520in%2520promoters%2520throughout%2520the%2520human%2520genome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D406%26epage%3D413%26doi%3D10.1093%2Fnar%2Fgkl1057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Nakken, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hovig, E.</span><span> </span><span class="NLM_article-title">The disruptive positions in human G-quadruplex motifs are less polymorphic and more conserved than their neutral counterparts</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">–</span> <span class="NLM_lpage">5756</span><span class="refDoi"> DOI: 10.1093/nar/gkp590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkp590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=5749-5756&author=S.+Nakkenauthor=T.+Rognesauthor=E.+Hovig&title=The+disruptive+positions+in+human+G-quadruplex+motifs+are+less+polymorphic+and+more+conserved+than+their+neutral+counterparts&doi=10.1093%2Fnar%2Fgkp590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp590%26sid%3Dliteratum%253Aachs%26aulast%3DNakken%26aufirst%3DS.%26aulast%3DRognes%26aufirst%3DT.%26aulast%3DHovig%26aufirst%3DE.%26atitle%3DThe%2520disruptive%2520positions%2520in%2520human%2520G-quadruplex%2520motifs%2520are%2520less%2520polymorphic%2520and%2520more%2520conserved%2520than%2520their%2520neutral%2520counterparts%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D5749%26epage%3D5756%26doi%3D10.1093%2Fnar%2Fgkp590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2536</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1093/nar/gkl286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=16687659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2536-2549&author=S.+Cogoiauthor=L.+E.+Xodo&title=G-quadruplex+formation+within+the+promoter+of+the+KRAS+proto-oncogene+and+its+effect+on+transcription&doi=10.1093%2Fnar%2Fgkl286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2536-2549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In human and mouse, the promoter of the KRAS gene contains a nuclease hypersensitive polypurine-polypyrimidine element (NHPPE) that is essential for transcription.  An interesting feature of the polypurine G-rich strand of NHPPE is its ability to assume an unusual DNA structure that, according to CD and DMS footprinting expts., is attributed to an intramol. parallel G-quadruplex, consisting of three G-tetrads and three loops.  The human and mouse KRAS NHPPE G-rich strands display melting temp. of 64° and 73°, resp., as well as a K+-dependent capacity to arrest DNA polymerase.  Photocleavage and CD expts. showed that the cationic porphyrin TMPyP4 stacks to the external G-tetrads of the KRAS quadruplexes, increasing the Tm by ∼20°.  These findings raise the intriguing question that the G-quadruplex formed within the NHPPE of KRAS may be involved in the regulation of transcription.  Indeed, transfection expts. showed that the activity of the mouse KRAS promoter is reduced to 20% of control, in the presence of the quadruplex-stabilizing TMPyP4.  In addn., we found that G-rich oligonucleotides mimicking the KRAS quadruplex, but not the corresponding 4-base mutant sequences or oligonucleotides forming quadruplexes with different structures, competed with the NHPPE duplex for binding to nuclear proteins.  When vector pKRS-413, contg. CAT driven by the mouse KRAS promoter, and KRAS quadruplex oligonucleotides were co-transfected in 293 cells, the expression of CAT was found to be downregulated to 40% of the control.  On the basis of these data, we propose that the NHPPE of KRAS exists in equil. between a double-stranded form favoring transcription and a folded quadruplex form, which instead inhibits transcription.  Such a mechanism, which is probably adopted by other growth-related genes, provides useful hints for the rational design of anticancer drugs against the KRAS oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfPpw1J4eI7Vg90H21EOLACvtfcHk0ljcR-1BogtR1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D&md5=b6b7c93fab3ea76e200f3bd21bd61f36</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl286%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG-quadruplex%2520formation%2520within%2520the%2520promoter%2520of%2520the%2520KRAS%2520proto-oncogene%2520and%2520its%2520effect%2520on%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D2536%26epage%3D2549%26doi%3D10.1093%2Fnar%2Fgkl286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Eddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maizels, N.</span><span> </span><span class="NLM_article-title">Conserved elements with potential to form polymorphic G-quadruplex structures in the first intron of human genes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1321</span><span class="NLM_x">–</span> <span class="NLM_lpage">1333</span><span class="refDoi"> DOI: 10.1093/nar/gkm1138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkm1138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=18187510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislKiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=1321-1333&author=J.+Eddyauthor=N.+Maizels&title=Conserved+elements+with+potential+to+form+polymorphic+G-quadruplex+structures+in+the+first+intron+of+human+genes&doi=10.1093%2Fnar%2Fgkm1138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved elements with potential to form polymorphic G-quadruplex structures in the first intron of human genes</span></div><div class="casAuthors">Eddy, Johanna; Maizels, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1321-1333</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">To understand how potential for G-quadruplex formation might influence regulation of gene expression, we examd. the 2 kb spanning the transcription start sites (TSS) of the 18 217 human RefSeq genes, distinguishing contributions of template and nontemplate strands.  Regions both upstream and downstream of the TSS are G-rich, but the downstream region displays a clear bias toward G-richness on the nontemplate strand.  Upstream of the TSS, much of the G-richness and potential for G-quadruplex formation derives from the presence of well-defined canonical regulatory motifs in duplex DNA, including CpG dinucleotides which are sites of regulatory methylation, and motifs recognized by the transcription factor SP1.  This challenges the notion that quadruplex formation upstream of the TSS contributes to regulation of gene expression.  Downstream of the TSS, G-richness is concd. in the first intron, and on the nontemplate strand, where polymorphic sequence elements with potential to form G-quadruplex structures and which cannot be accounted for by known regulatory motifs are found in almost 3000 (16%) of the human RefSeq genes, and are conserved through frogs.  These elements could in principle be recognized either as DNA or as RNA, providing structural targets for regulation at the level of transcription or RNA processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPNriy5euURLVg90H21EOLACvtfcHk0ljcR-1BogtR1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislKiur0%253D&md5=44f6ad32cdd8cd28c25cbf3d16b17ad2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm1138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm1138%26sid%3Dliteratum%253Aachs%26aulast%3DEddy%26aufirst%3DJ.%26aulast%3DMaizels%26aufirst%3DN.%26atitle%3DConserved%2520elements%2520with%2520potential%2520to%2520form%2520polymorphic%2520G-quadruplex%2520structures%2520in%2520the%2520first%2520intron%2520of%2520human%2520genes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D1321%26epage%3D1333%26doi%3D10.1093%2Fnar%2Fgkm1138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Verma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basundra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">Evidence of genome-wide G4 DNA-mediated gene expression in human cancer cells</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">4194</span><span class="NLM_x">–</span> <span class="NLM_lpage">4204</span><span class="refDoi"> DOI: 10.1093/nar/gkn1076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkn1076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19211664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptFGhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=4194-4204&author=A.+Vermaauthor=V.+K.+Yadavauthor=R.+Basundraauthor=A.+Kumarauthor=S.+Chowdhury&title=Evidence+of+genome-wide+G4+DNA-mediated+gene+expression+in+human+cancer+cells&doi=10.1093%2Fnar%2Fgkn1076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence of genome-wide G4 DNA-mediated gene expression in human cancer cells</span></div><div class="casAuthors">Verma, Anjali; Yadav, Vinod Kumar; Basundra, Richa; Kumar, Akinchan; Chowdhury, Shantanu</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4194-4204</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Guanine-rich DNA of a particular sequence adopts four-stranded structural forms known as G-quadruplex or G4 DNA.  Though in vitro formation of G4 DNA is known for several years, in vivo presence of G4 DNA was only recently noted in eukaryote telomeres.  Recent bioinformatics analyses showing prevalence of G4 DNA within promoters of human and related species seems to implicate G4 DNA in a genome-wide cis-regulatory role.  Herein we demonstrate that G4 DNA may present regulatory sites on a genome-wide scale by showing widespread effect on gene expression in response to the established intracellular G4 DNA-binding ligands.  This is particularly relevant to genes that harbor conserved potential G4 DNA (PG4 DNA) forming sequence across human, mouse and rat promoters of orthologous genes.  Genes with conserved PG4 DNA in promoters show co-regulated expression in 79 human and 61 mouse normal tissues (z-score > 3.5; P < 0.0001).  Conservation of G4 DNA across related species also emphasizes the biol. importance of G4 DNA and its role in transcriptional regulation of genes; shedding light on a relatively novel mechanism of regulation of gene expression in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8IUDB3VYGJLVg90H21EOLACvtfcHk0ljcR-1BogtR1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptFGhtLg%253D&md5=56c58ede4a7c91710440d731f5bf5fca</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn1076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn1076%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DBasundra%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DEvidence%2520of%2520genome-wide%2520G4%2520DNA-mediated%2520gene%2520expression%2520in%2520human%2520cancer%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D4194%26epage%3D4204%26doi%3D10.1093%2Fnar%2Fgkn1076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Mani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">Genome-wide analyses of recombination prone regions predict role of DNA structural motif in recombination</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e4399</span><span class="refDoi"> DOI: 10.1371/journal.pone.0004399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.pone.0004399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19198658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A280%3ADC%252BD1M7is1CntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e4399&author=P.+Maniauthor=V.+K.+Yadavauthor=S.+K.+Dasauthor=S.+Chowdhury&title=Genome-wide+analyses+of+recombination+prone+regions+predict+role+of+DNA+structural+motif+in+recombination&doi=10.1371%2Fjournal.pone.0004399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide analyses of recombination prone regions predict role of DNA structural motif in recombination</span></div><div class="casAuthors">Mani Prithvi; Yadav Vinod Kumar; Das Swapan Kumar; Chowdhury Shantanu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e4399</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">HapMap findings reveal surprisingly asymmetric distribution of recombinogenic regions.  Short recombinogenic regions (hotspots) are interspersed between large relatively non-recombinogenic regions.  This raises the interesting possibility of DNA sequence and/or other cis- elements as determinants of recombination.  We hypothesized the involvement of non-canonical sequences that can result in local non-B DNA structures and tested this using the G-quadruplex DNA as a model.  G-quadruplex or G4 DNA is a unique form of four-stranded non-B DNA structure that engages certain G-rich sequences, presence of such motifs has been noted within telomeres.  In support of this hypothesis, genome-wide computational analyses presented here reveal enrichment of potential G4 (PG4) DNA forming sequences within 25618 human hotspots relative to 9290 coldspots (p<0.0001).  Furthermore, co-occurrence of PG4 DNA within several short sequence elements that are associated with recombinogenic regions was found to be significantly more than randomly expected.  Interestingly, analyses of more than 50 DNA binding factors revealed that co-occurrence of PG4 DNA with target DNA binding sites of transcription factors c-Rel, NF-kappa B (p50 and p65) and Evi-1 was significantly enriched in recombination-prone regions.  These observations support involvement of G4 DNA in recombination, predicting a functional model that is consistent with duplex-strand separation induced by formation of G4 motifs in supercoiled DNA and/or when assisted by other cellular factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQw2fvA1PJyb-_Q9URG6Dx8fW6udTcc2eYqFbHVjAPs4Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M7is1CntA%253D%253D&md5=d3d3f4b1effd792cf676b983a8152000</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0004399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0004399%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DP.%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DDas%26aufirst%3DS.%2BK.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DGenome-wide%2520analyses%2520of%2520recombination%2520prone%2520regions%2520predict%2520role%2520of%2520DNA%2520structural%2520motif%2520in%2520recombination%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De4399%26doi%3D10.1371%2Fjournal.pone.0004399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Haeusler, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donnelly, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Periz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simko, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troncoso, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothstein, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">C9orf72 nucleotide repeat structures initiate molecular cascades of disease</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">507</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">200</span><span class="refDoi"> DOI: 10.1038/nature13124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnature13124" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24598541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktV2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=507&publication_year=2014&pages=195-200&author=A.+R.+Haeuslerauthor=C.+J.+Donnellyauthor=G.+Perizauthor=E.+A.+Simkoauthor=P.+G.+Shawauthor=M.+S.+Kimauthor=N.+J.+Maragakisauthor=J.+C.+Troncosoauthor=A.+Pandeyauthor=R.+Sattlerauthor=J.+D.+Rothsteinauthor=J.+Wang&title=C9orf72+nucleotide+repeat+structures+initiate+molecular+cascades+of+disease&doi=10.1038%2Fnature13124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">C9orf72 nucleotide repeat structures initiate molecular cascades of disease</span></div><div class="casAuthors">Haeusler, Aaron R.; Donnelly, Christopher J.; Periz, Goran; Simko, Eric A. J.; Shaw, Patrick G.; Kim, Min-Sik; Maragakis, Nicholas J.; Troncoso, Juan C.; Pandey, Akhilesh; Sattler, Rita; Rothstein, Jeffrey D.; Wang, Jiou</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">507</span>
        (<span class="NLM_cas:issue">7491</span>),
    <span class="NLM_cas:pages">195-200</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A hexanucleotide repeat expansion (HRE), (GGGGCC)n, in C9orf72 is the most common genetic cause of the neurodegenerative diseases amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD).  Here we identify a mol. mechanism by which structural polymorphism of the HRE leads to ALS/FTD pathol. and defects.  The HRE forms DNA and RNA G-quadruplexes with distinct structures and promotes RNA•DNA hybrids (R-loops).  The structural polymorphism causes a repeat-length-dependent accumulation of transcripts aborted in the HRE region.  These transcribed repeats bind to ribonucleoproteins in a conformation-dependent manner.  Specifically, nucleolin, an essential nucleolar protein, preferentially binds the HRE G-quadruplex, and patient cells show evidence of nucleolar stress.  Our results demonstrate that distinct C9orf72 HRE structural polymorphism at both DNA and RNA levels initiates mol. cascades leading to ALS/FTD pathologies, and provide the basis for a mechanistic model for repeat-assocd. neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvSEy_yaVaoLVg90H21EOLACvtfcHk0lgoYJhsszbjKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktV2rt7s%253D&md5=cbe8956df515e57926dac896af764c74</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature13124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13124%26sid%3Dliteratum%253Aachs%26aulast%3DHaeusler%26aufirst%3DA.%2BR.%26aulast%3DDonnelly%26aufirst%3DC.%2BJ.%26aulast%3DPeriz%26aufirst%3DG.%26aulast%3DSimko%26aufirst%3DE.%2BA.%26aulast%3DShaw%26aufirst%3DP.%2BG.%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DMaragakis%26aufirst%3DN.%2BJ.%26aulast%3DTroncoso%26aufirst%3DJ.%2BC.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DSattler%26aufirst%3DR.%26aulast%3DRothstein%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DC9orf72%2520nucleotide%2520repeat%2520structures%2520initiate%2520molecular%2520cascades%2520of%2520disease%26jtitle%3DNature%26date%3D2014%26volume%3D507%26spage%3D195%26epage%3D200%26doi%3D10.1038%2Fnature13124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ivanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Day, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emara, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, P.</span><span> </span><span class="NLM_article-title">G-quadruplex structures contribute to the neuroprotective effects of angiogenin-induced tRNA fragments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">18201</span><span class="NLM_x">–</span> <span class="NLM_lpage">18206</span><span class="refDoi"> DOI: 10.1073/pnas.1407361111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.1407361111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18201-18206&author=P.+Ivanovauthor=E.+O%E2%80%99Dayauthor=M.+M.+Emaraauthor=G.+Wagnerauthor=J.+Liebermanauthor=P.+Anderson&title=G-quadruplex+structures+contribute+to+the+neuroprotective+effects+of+angiogenin-induced+tRNA+fragments&doi=10.1073%2Fpnas.1407361111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407361111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407361111%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Day%26aufirst%3DE.%26aulast%3DEmara%26aufirst%3DM.%2BM.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DLieberman%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DP.%26atitle%3DG-quadruplex%2520structures%2520contribute%2520to%2520the%2520neuroprotective%2520effects%2520of%2520angiogenin-induced%2520tRNA%2520fragments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D18201%26epage%3D18206%26doi%3D10.1073%2Fpnas.1407361111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Sket, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pohleven, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovanda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stalekar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zupunski, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zalar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plavec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogelj, B.</span><span> </span><span class="NLM_article-title">Characterization of DNA G-quadruplex species forming from C9ORF72 G4C2-expanded repeats associated with amyotrophic lateral sclerosis and frontotemporal lobar degeneration</span> <span class="citation_source-journal">Neurobiol. Aging</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1016/j.neurobiolaging.2014.09.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.neurobiolaging.2014.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25442110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslelsr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1091-1096&author=P.+Sketauthor=J.+Pohlevenauthor=A.+Kovandaauthor=M.+Stalekarauthor=V.+Zupunskiauthor=M.+Zalarauthor=J.+Plavecauthor=B.+Rogelj&title=Characterization+of+DNA+G-quadruplex+species+forming+from+C9ORF72+G4C2-expanded+repeats+associated+with+amyotrophic+lateral+sclerosis+and+frontotemporal+lobar+degeneration&doi=10.1016%2Fj.neurobiolaging.2014.09.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of DNA G-quadruplex species forming from C9ORF72 G4C2-expanded repeats associated with amyotrophic lateral sclerosis and frontotemporal lobar degeneration</span></div><div class="casAuthors">Sket, Primoz; Pohleven, Jure; Kovanda, Anja; Stalekar, Maja; Zupunski, Vera; Zalar, Matja; Plavec, Janez; Rogelj, Boris</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Aging</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">NEAGDO</span>;
        ISSN:<span class="NLM_cas:issn">0197-4580</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The G4C2 hexanucleotide repeat expansion, located in the first intron of the C9ORF72 gene, represents a major genetic hallmark of amyotrophic lateral sclerosis and frontotemporal lobar degeneration.  Several hypotheses have been proposed on how the transcribed repeat RNA leads to the development of amyotrophic lateral sclerosis and frontotemporal lobar degeneration.  However, despite their importance, factors affecting the transcription of expanded-repeat RNA are not well known.  As transcription is dependent on the DNA contg. the expanded repeats, it is crucial to understand its structure.  G-quadruplexes are known to affect expression on the level of DNA, therefore whether they form on the expanded-repeat DNA constitutes an important biol. question.  Using NMR and CD spectroscopy we show that DNA G4C2 with varying no. of repeats d(G4C2)n form planar guanine quartets characteristic of G-quadruplexes.  Addnl., we show DNA G-quadruplexes can form inter- and intra-molecularly in either parallel or anti-parallel orientation, based on d(G4C2) sequence length.  This potential structural heterogeneity of longer disease-relevant repeats should therefore be taken into account when studying their role in disease pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm1F5TYUAtvbVg90H21EOLACvtfcHk0lh6QUjYGz_Q8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslelsr7F&md5=abf45124f3e9098a4eaa7a1acf7a6f2a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.neurobiolaging.2014.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neurobiolaging.2014.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DSket%26aufirst%3DP.%26aulast%3DPohleven%26aufirst%3DJ.%26aulast%3DKovanda%26aufirst%3DA.%26aulast%3DStalekar%26aufirst%3DM.%26aulast%3DZupunski%26aufirst%3DV.%26aulast%3DZalar%26aufirst%3DM.%26aulast%3DPlavec%26aufirst%3DJ.%26aulast%3DRogelj%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520DNA%2520G-quadruplex%2520species%2520forming%2520from%2520C9ORF72%2520G4C2-expanded%2520repeats%2520associated%2520with%2520amyotrophic%2520lateral%2520sclerosis%2520and%2520frontotemporal%2520lobar%2520degeneration%26jtitle%3DNeurobiol.%2520Aging%26date%3D2015%26volume%3D36%26spage%3D1091%26epage%3D1096%26doi%3D10.1016%2Fj.neurobiolaging.2014.09.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Fisette, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montagna, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mihailescu, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, M. S.</span><span> </span><span class="NLM_article-title">A G-rich element forms a G-quadruplex and regulates BACE1 mRNA alternative splicing</span> <span class="citation_source-journal">J. Neurochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">773</span><span class="refDoi"> DOI: 10.1111/j.1471-4159.2012.07680.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1111%2Fj.1471-4159.2012.07680.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22303960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGltrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2012&pages=763-773&author=J.+F.+Fisetteauthor=D.+R.+Montagnaauthor=M.+R.+Mihailescuauthor=M.+S.+Wolfe&title=A+G-rich+element+forms+a+G-quadruplex+and+regulates+BACE1+mRNA+alternative+splicing&doi=10.1111%2Fj.1471-4159.2012.07680.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A G-rich element forms a G-quadruplex and regulates BACE1 mRNA alternative splicing</span></div><div class="casAuthors">Fisette, Jean-Francois; Montagna, Daniel R.; Mihailescu, Mihaela-Rita; Wolfe, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">5&6</span>),
    <span class="NLM_cas:pages">763-773</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">β-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is the transmembrane aspartyl protease that catalyzes the first cleavage step in the proteolysis of the APP to the amyloid β-protein (Aβ), a process involved in the pathogenesis of Alzheimer disease.  BACE1 pre-mRNA undergoes complex alternative splicing, the regulation of which is not well understood.  We identified a G-rich sequence within exon 3 of BACE1 involved in controlling splice site selection.  Mutation of the G-rich sequence decreased use of the normal 5' splice site of exon 3, which leads to full-length and proteolytically active BACE1, and increased use of an alternative splice site, which leads to a shorter, essentially inactive isoform.  Nuclease protection assays, NMR, and CD spectroscopy revealed that this sequence folds into a G-quadruplex structure.  Several proteins were identified as capable of binding to the G-rich sequence, and one of these, heterogeneous nuclear ribonucleoprotein H, was found to regulate BACE1 exon 3 alternative splicing and in a manner dependent on the G-rich sequence.  Knockdown of heterogeneous nuclear ribonucleoprotein H led to a decrease in the full-length BACE1 mRNA isoform as well as a decrease in Aβ prodn. from APP, suggesting new possibilities for therapeutic approaches to Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp19qAAvW5AMrVg90H21EOLACvtfcHk0lh6QUjYGz_Q8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGltrvL&md5=2d3e2873e4cc643f7b501ad77499b4c7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2012.07680.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2012.07680.x%26sid%3Dliteratum%253Aachs%26aulast%3DFisette%26aufirst%3DJ.%2BF.%26aulast%3DMontagna%26aufirst%3DD.%2BR.%26aulast%3DMihailescu%26aufirst%3DM.%2BR.%26aulast%3DWolfe%26aufirst%3DM.%2BS.%26atitle%3DA%2520G-rich%2520element%2520forms%2520a%2520G-quadruplex%2520and%2520regulates%2520BACE1%2520mRNA%2520alternative%2520splicing%26jtitle%3DJ.%2520Neurochem.%26date%3D2012%26volume%3D121%26spage%3D763%26epage%3D773%26doi%3D10.1111%2Fj.1471-4159.2012.07680.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Fry, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loeb, L. A.</span><span> </span><span class="NLM_article-title">The fragile X syndrome d(CGG)n nucleotide repeats form a stable tetrahelical structure</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">4950</span><span class="NLM_x">–</span> <span class="NLM_lpage">4954</span><span class="refDoi"> DOI: 10.1073/pnas.91.11.4950</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.91.11.4950" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=4950-4954&author=M.+Fryauthor=L.+A.+Loeb&title=The+fragile+X+syndrome+d%28CGG%29n+nucleotide+repeats+form+a+stable+tetrahelical+structure&doi=10.1073%2Fpnas.91.11.4950"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.11.4950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.11.4950%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DM.%26aulast%3DLoeb%26aufirst%3DL.%2BA.%26atitle%3DThe%2520fragile%2520X%2520syndrome%2520d%2528CGG%2529n%2520nucleotide%2520repeats%2520form%2520a%2520stable%2520tetrahelical%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1994%26volume%3D91%26spage%3D4950%26epage%3D4954%26doi%3D10.1073%2Fpnas.91.11.4950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatto, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span> </span><span class="NLM_article-title">The evolving world of protein-G-quadruplex recognition: a medicinal chemist’s perspective</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">1219</span><span class="NLM_x">–</span> <span class="NLM_lpage">1230</span><span class="refDoi"> DOI: 10.1016/j.biochi.2011.04.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.biochi.2011.04.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21549174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpt1OgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2011&pages=1219-1230&author=C.+Sissiauthor=B.+Gattoauthor=M.+Palumbo&title=The+evolving+world+of+protein-G-quadruplex+recognition%3A+a+medicinal+chemist%E2%80%99s+perspective&doi=10.1016%2Fj.biochi.2011.04.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The evolving world of protein-G-quadruplex recognition: A medicinal chemist's perspective</span></div><div class="casAuthors">Sissi, Claudia; Gatto, Barbara; Palumbo, Manlio</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1219-1230</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The physiol. and pharmacol. role of nucleic acids structures folded into the non canonical G-quadruplex conformation have recently emerged.  Their activities are targeted at vital cellular processes including telomere maintenance, regulation of transcription and processing of the pre-messenger or telomeric RNA.  In addn., severe conditions like cancer, fragile X syndrome, Bloom syndrome, Werner syndrome and Fanconi anemia J are related to genomic defects that involve G-quadruplex forming sequences.  In this connection G-quadruplex recognition and processing by nucleic acid directed proteins and enzymes represents a key event to activate or deactivate physiol. or pathol. pathways.  In this review we examine protein-G-quadruplex recognition in physiol. significant conditions and discuss how to possibly exploit the interactions' selectivity for targeted therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojiEYxee9yTrVg90H21EOLACvtfcHk0lh6QUjYGz_Q8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpt1OgsLs%253D&md5=f9f93b292b845626ef8f65c3e33edddc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2011.04.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2011.04.018%26sid%3Dliteratum%253Aachs%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DGatto%26aufirst%3DB.%26aulast%3DPalumbo%26aufirst%3DM.%26atitle%3DThe%2520evolving%2520world%2520of%2520protein-G-quadruplex%2520recognition%253A%2520a%2520medicinal%2520chemist%25E2%2580%2599s%2520perspective%26jtitle%3DBiochimie%26date%3D2011%26volume%3D93%26spage%3D1219%26epage%3D1230%26doi%3D10.1016%2Fj.biochi.2011.04.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0li266g0E3GG-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Henderson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chavez, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lansdorp, P. M.</span><span> </span><span class="NLM_article-title">Detection of G-quadruplex DNA in mammalian cells</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">860</span><span class="NLM_x">–</span> <span class="NLM_lpage">869</span><span class="refDoi"> DOI: 10.1093/nar/gkt957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkt957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24163102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlShsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=860-869&author=A.+Hendersonauthor=Y.+Wuauthor=Y.+C.+Huangauthor=E.+A.+Chavezauthor=J.+Plattauthor=F.+B.+Johnsonauthor=R.+M.+Broshauthor=D.+Senauthor=P.+M.+Lansdorp&title=Detection+of+G-quadruplex+DNA+in+mammalian+cells&doi=10.1093%2Fnar%2Fgkt957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of G-quadruplex DNA in mammalian cells</span></div><div class="casAuthors">Henderson, Alexander; Wu, Yuliang; Huang, Yu Chuan; Chavez, Elizabeth A.; Platt, Jesse; Johnson, F. Brad; Brosh, Robert M., Jr.; Sen, Dipankar; Lansdorp, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">860-869</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It has been proposed that guanine-rich DNA forms four-stranded structures in vivo called G-quadruplexes or G4 DNA.  G4 DNA has been implicated in several biol. processes, but tools to study G4 DNA structures in cells are limited.  Here we report the development of novel murine monoclonal antibodies specific for different G4 DNA structures.  We show that one of these antibodies designated 1H6 exhibits strong nuclear staining in most human and murine cells.  Staining intensity increased on treatment of cells with agents that stabilize G4 DNA and, strikingly, cells deficient in FANCJ, a G4 DNA-specific helicase, showed stronger nuclear staining than controls.  Our data strongly support the existence of G4 DNA structures in mammalian cells and indicate that the abundance of such structures is increased in the absence of FANCJ.  We conclude that monoclonal antibody 1H6 is a valuable tool for further studies on the role of G4 DNA in cell and mol. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrli0PqZbZA3bVg90H21EOLACvtfcHk0li266g0E3GG-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlShsrw%253D&md5=a16e8096cfaa3f7d8d7d045294a6fa3f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt957%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%2BC.%26aulast%3DChavez%26aufirst%3DE.%2BA.%26aulast%3DPlatt%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DF.%2BB.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26aulast%3DSen%26aufirst%3DD.%26aulast%3DLansdorp%26aufirst%3DP.%2BM.%26atitle%3DDetection%2520of%2520G-quadruplex%2520DNA%2520in%2520mammalian%2520cells%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D860%26epage%3D869%26doi%3D10.1093%2Fnar%2Fgkt957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amrane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Litvak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreola, M. L.</span><span> </span><span class="NLM_article-title">G-quadruplexes in viruses: function and potential therapeutic applications</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">12352</span><span class="NLM_x">–</span> <span class="NLM_lpage">12366</span><span class="refDoi"> DOI: 10.1093/nar/gku999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgku999" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=12352-12366&author=M.+Metifiotauthor=S.+Amraneauthor=S.+Litvakauthor=M.+L.+Andreola&title=G-quadruplexes+in+viruses%3A+function+and+potential+therapeutic+applications&doi=10.1093%2Fnar%2Fgku999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku999%26sid%3Dliteratum%253Aachs%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DAmrane%26aufirst%3DS.%26aulast%3DLitvak%26aufirst%3DS.%26aulast%3DAndreola%26aufirst%3DM.%2BL.%26atitle%3DG-quadruplexes%2520in%2520viruses%253A%2520function%2520and%2520potential%2520therapeutic%2520applications%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D12352%26epage%3D12366%26doi%3D10.1093%2Fnar%2Fgku999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Murat, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lekieffre, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowieson, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clancy, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Preiss, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tellam, J.</span><span> </span><span class="NLM_article-title">G-quadruplexes regulate Epstein-Barr virus-encoded nuclear antigen 1 mRNA translation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio.1479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnchembio.1479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24633353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlCktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=358-364&author=P.+Muratauthor=J.+Zhongauthor=L.+Lekieffreauthor=N.+P.+Cowiesonauthor=J.+L.+Clancyauthor=T.+Preissauthor=S.+Balasubramanianauthor=R.+Khannaauthor=J.+Tellam&title=G-quadruplexes+regulate+Epstein-Barr+virus-encoded+nuclear+antigen+1+mRNA+translation&doi=10.1038%2Fnchembio.1479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplexes regulate Epstein-Barr virus-encoded nuclear antigen 1 mRNA translation</span></div><div class="casAuthors">Murat, Pierre; Zhong, Jie; Lekieffre, Lea; Cowieson, Nathan P.; Clancy, Jennifer L.; Preiss, Thomas; Balasubramanian, Shankar; Khanna, Rajiv; Tellam, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Viruses that establish latent infections have evolved unique mechanisms to avoid host immune recognition.  Maintenance proteins of these viruses regulate their synthesis to levels sufficient for maintaining persistent infection but below threshold levels for host immune detection.  The mechanisms governing this finely tuned regulation of viral latency are unknown.  Here the authors show that mRNAs encoding gammaherpesviral maintenance proteins contain within their open reading frames clusters of unusual structural elements, G-quadruplexes, which are responsible for the cis-acting regulation of viral mRNA translation.  By studying the Epstein-Barr virus-encoded nuclear antigen 1 (EBNA1) mRNA, the authors show that destabilization of G-quadruplexes using antisense oligonucleotides increases EBNA1 mRNA translation.  In contrast, pretreatment with a G-quadruplex-stabilizing small mol., pyridostatin, decreases EBNA1 synthesis, highlighting the importance of G-quadruplexes within virally encoded transcripts as unique regulatory signals for translational control and immune evasion.  Furthermore, these findings suggest alternative therapeutic strategies focused on targeting RNA structure within viral ORFs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppFLmSVv6UerVg90H21EOLACvtfcHk0lgUfSJDF_ZLGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlCktrs%253D&md5=7681817a80afde9bc604b4b7e0dba0a1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1479%26sid%3Dliteratum%253Aachs%26aulast%3DMurat%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DLekieffre%26aufirst%3DL.%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DClancy%26aufirst%3DJ.%2BL.%26aulast%3DPreiss%26aufirst%3DT.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DR.%26aulast%3DTellam%26aufirst%3DJ.%26atitle%3DG-quadruplexes%2520regulate%2520Epstein-Barr%2520virus-encoded%2520nuclear%2520antigen%25201%2520mRNA%2520translation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio.1479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Norseen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, F. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, P. M.</span><span> </span><span class="NLM_article-title">Role for G-quadruplex RNA binding by Epstein-Barr virus nuclear antigen 1 in DNA replication and metaphase chromosome attachment</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">10336</span><span class="NLM_x">–</span> <span class="NLM_lpage">10346</span><span class="refDoi"> DOI: 10.1128/JVI.00747-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1128%2FJVI.00747-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=19656898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlClsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=10336-10346&author=J.+Norseenauthor=F.+B.+Johnsonauthor=P.+M.+Lieberman&title=Role+for+G-quadruplex+RNA+binding+by+Epstein-Barr+virus+nuclear+antigen+1+in+DNA+replication+and+metaphase+chromosome+attachment&doi=10.1128%2FJVI.00747-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Role for G-quadruplex RNA binding by Epstein-Barr virus nuclear antigen 1 in DNA replication and metaphase chromosome attachment</span></div><div class="casAuthors">Norseen, Julie; Johnson, F. Brad; Lieberman, Paul M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">10336-10346</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Latent infection by Epstein-Barr virus (EBV) requires both replication and maintenance of the viral genome.  EBV nuclear antigen 1 (EBNA1) is a virus-encoded protein that is crit. for the replication and maintenance of the genome during latency in proliferating cells.  The authors have previously demonstrated that EBNA1 recruits the cellular origin recognition complex (ORC) through an RNA-dependent interaction with EBNA1 linking region 1 (LR1) and LR2.  The authors now show that LR1 and LR2 bind to G-rich RNA that is predicted to form G-quadruplex structures.  Several chem. distinct G-quadruplex-interacting drugs disrupted the interaction between EBNA1 and ORC.  The G-quadruplex-interacting compd. BRACO-19 inhibited EBNA1-dependent stimulation of viral DNA replication and preferentially blocked proliferation of EBV-pos. cells relative to EBV-neg. cell lines.  BRACO-19 treatment also disrupted the ability of EBNA1 to tether to metaphase chromosomes, suggesting that maintenance function is also mediated through G-quadruplex recognition.  These findings suggest that the EBNA1 replication and maintenance function uses a common G-quadruplex binding capacity of LR1 and LR2, which may be targetable by small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqiD6hHzR-KrVg90H21EOLACvtfcHk0lgUfSJDF_ZLGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlClsbjK&md5=ec64ae4a8182b80c92a50f1f215ac053</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FJVI.00747-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00747-09%26sid%3Dliteratum%253Aachs%26aulast%3DNorseen%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DF.%2BB.%26aulast%3DLieberman%26aufirst%3DP.%2BM.%26atitle%3DRole%2520for%2520G-quadruplex%2520RNA%2520binding%2520by%2520Epstein-Barr%2520virus%2520nuclear%2520antigen%25201%2520in%2520DNA%2520replication%2520and%2520metaphase%2520chromosome%2520attachment%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D10336%26epage%3D10346%26doi%3D10.1128%2FJVI.00747-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollati, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusov, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesters, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgenfeld, R.</span><span> </span><span class="NLM_article-title">The SARS-unique domain (SUD) of SARS coronavirus contains two macrodomains that bind G-quadruplexes</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000428</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1000428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.ppat.1000428" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000428&author=J.+Tanauthor=C.+Vonrheinauthor=O.+S.+Smartauthor=G.+Bricogneauthor=M.+Bollatiauthor=Y.+Kusovauthor=G.+Hansenauthor=J.+R.+Mestersauthor=C.+L.+Schmidtauthor=R.+Hilgenfeld&title=The+SARS-unique+domain+%28SUD%29+of+SARS+coronavirus+contains+two+macrodomains+that+bind+G-quadruplexes&doi=10.1371%2Fjournal.ppat.1000428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000428%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DSmart%26aufirst%3DO.%2BS.%26aulast%3DBricogne%26aufirst%3DG.%26aulast%3DBollati%26aufirst%3DM.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DSchmidt%26aufirst%3DC.%2BL.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DThe%2520SARS-unique%2520domain%2520%2528SUD%2529%2520of%2520SARS%2520coronavirus%2520contains%2520two%2520macrodomains%2520that%2520bind%2520G-quadruplexes%26jtitle%3DPLoS%2520Pathog.%26date%3D2009%26volume%3D5%26spage%3De1000428%26doi%3D10.1371%2Fjournal.ppat.1000428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poe, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frasson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithgall, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Formation of a unique cluster of G-quadruplex structures in the HIV-1 Nef coding region: implications for antiviral activity</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e73121</span><span class="refDoi"> DOI: 10.1371/journal.pone.0073121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.pone.0073121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24015290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e73121&author=R.+Perroneauthor=M.+Nadaiauthor=J.+A.+Poeauthor=I.+Frassonauthor=M.+Palumboauthor=G.+Paluauthor=T.+E.+Smithgallauthor=S.+N.+Richter&title=Formation+of+a+unique+cluster+of+G-quadruplex+structures+in+the+HIV-1+Nef+coding+region%3A+implications+for+antiviral+activity&doi=10.1371%2Fjournal.pone.0073121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of a unique cluster of G-quadruplex structures in the HIV-1 nef coding region: implications for antiviral activity</span></div><div class="casAuthors">Perrone, Rosalba; Nadai, Matteo; Poe, Jerrod A.; Frasson, Ilaria; Palumbo, Manlio; Palu, Giorgio; Smithgall, Thomas E.; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e73121</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">G-quadruplexes are tetraplex structures of nucleic acids that can form in G-rich sequences.  Their presence and functional role have been established in telomeres, oncogene promoters and coding regions of the human chromosome.  In particular, they have been proposed to be directly involved in gene regulation at the level of transcription.  Because the HIV-1 Nef protein is a fundamental factor for efficient viral replication, infectivity and pathogenesis in vitro and in vivo, we investigated G-quadruplex formation in the HIV-1 nef gene to assess the potential for viral inhibition through G-quadruplex stabilization.  A comprehensive computational anal. of the nef coding region of available strains showed the presence of three conserved sequences that were uniquely clustered.  Biophys. testing proved that G-quadruplex conformations were efficiently stabilized or induced by G-quadruplex ligands in all three sequences.  Upon incubation with a G-quadruplex ligand, Nef expression was reduced in a reporter gene assay and Nef-dependent enhancement of HIV-1 infectivity was significantly repressed in an antiviral assay.  These data constitute the first evidence of the possibility to regulate HIV-1 gene expression and infectivity through G-quadruplex targeting and therefore open a new avenue for viral treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNzHbrQ-pu97Vg90H21EOLACvtfcHk0liD8pY296Y7eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrjP&md5=ec3e3ebcb1e84ceac70c4b60c6476e40</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0073121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0073121%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DPoe%26aufirst%3DJ.%2BA.%26aulast%3DFrasson%26aufirst%3DI.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DSmithgall%26aufirst%3DT.%2BE.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DFormation%2520of%2520a%2520unique%2520cluster%2520of%2520G-quadruplex%2520structures%2520in%2520the%2520HIV-1%2520Nef%2520coding%2520region%253A%2520implications%2520for%2520antiviral%2520activity%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De73121%26doi%3D10.1371%2Fjournal.pone.0073121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frasson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poe, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butovskaya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithgall, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">A dynamic G-quadruplex region regulates the HIV-1 long terminal repeat promoter</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">6521</span><span class="NLM_x">–</span> <span class="NLM_lpage">6530</span><span class="refDoi"> DOI: 10.1021/jm400914r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400914r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFeru7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6521-6530&author=R.+Perroneauthor=M.+Nadaiauthor=I.+Frassonauthor=J.+A.+Poeauthor=E.+Butovskayaauthor=T.+E.+Smithgallauthor=M.+Palumboauthor=G.+Paluauthor=S.+N.+Richter&title=A+dynamic+G-quadruplex+region+regulates+the+HIV-1+long+terminal+repeat+promoter&doi=10.1021%2Fjm400914r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Dynamic G-Quadruplex Region Regulates the HIV-1 Long Terminal Repeat Promoter</span></div><div class="casAuthors">Perrone, Rosalba; Nadai, Matteo; Frasson, Ilaria; Poe, Jerrod A.; Butovskaya, Elena; Smithgall, Thomas E.; Palumbo, Manlio; Palu, Giorgio; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6521-6530</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-Quadruplexes, noncanonical nucleic acid structures, act as silencers in the promoter regions of human genes; putative G-quadruplex forming sequences are also present in promoters of other mammals, yeasts, and prokaryotes.  Here we show that also the HIV-1 LTR promoter exploits G-quadruplex-mediated transcriptional regulation with striking similarities to eukaryotic promoters and that treatment with a G-quadruplex ligand inhibits HIV-1 infectivity.  Computational anal. on 953 HIV-1 strains substantiated a highly conserved G-rich sequence corresponding to Sp1 and NF-κB binding sites.  Biophys./biochem. anal. proved that two mutually exclusive parallel-like intramol. G-quadruplexes, stabilized by small mol. ligands, primarily fold in this region.  Mutations disrupting G-quadruplex formation enhanced HIV promoter activity in cells, whereas treatment with a G-quadruplex ligand impaired promoter activity and displayed antiviral effects.  These findings disclose the possibility of inhibiting the HIV-1 LTR promoter by G-quadruplex-interacting small mols., providing a new pathway to development of anti-HIV-1 drugs with unprecedented mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog3AJbPgRilrVg90H21EOLACvtfcHk0liD8pY296Y7eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFeru7bP&md5=2ea00d98b5ffef840e1bc80027b35b89</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm400914r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400914r%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DFrasson%26aufirst%3DI.%26aulast%3DPoe%26aufirst%3DJ.%2BA.%26aulast%3DButovskaya%26aufirst%3DE.%26aulast%3DSmithgall%26aufirst%3DT.%2BE.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DA%2520dynamic%2520G-quadruplex%2520region%2520regulates%2520the%2520HIV-1%2520long%2520terminal%2520repeat%2520promoter%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6521%26epage%3D6530%26doi%3D10.1021%2Fjm400914r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Amrane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerkour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedrat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vialet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andreola, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mergny, J. L.</span><span> </span><span class="NLM_article-title">Topology of a DNA G-quadruplex structure formed in the HIV-1 promoter: a potential target for anti-HIV drug development</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">5249</span><span class="NLM_x">–</span> <span class="NLM_lpage">5252</span><span class="refDoi"> DOI: 10.1021/ja501500c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja501500c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=5249-5252&author=S.+Amraneauthor=A.+Kerkourauthor=A.+Bedratauthor=B.+Vialetauthor=M.+L.+Andreolaauthor=J.+L.+Mergny&title=Topology+of+a+DNA+G-quadruplex+structure+formed+in+the+HIV-1+promoter%3A+a+potential+target+for+anti-HIV+drug+development&doi=10.1021%2Fja501500c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fja501500c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja501500c%26sid%3Dliteratum%253Aachs%26aulast%3DAmrane%26aufirst%3DS.%26aulast%3DKerkour%26aufirst%3DA.%26aulast%3DBedrat%26aufirst%3DA.%26aulast%3DVialet%26aufirst%3DB.%26aulast%3DAndreola%26aufirst%3DM.%2BL.%26aulast%3DMergny%26aufirst%3DJ.%2BL.%26atitle%3DTopology%2520of%2520a%2520DNA%2520G-quadruplex%2520structure%2520formed%2520in%2520the%2520HIV-1%2520promoter%253A%2520a%2520potential%2520target%2520for%2520anti-HIV%2520drug%2520development%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D5249%26epage%3D5252%26doi%3D10.1021%2Fja501500c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Piekna-Przybylska, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bambara, R. A.</span><span> </span><span class="NLM_article-title">U3 region in the HIV-1 genome adopts a G-quadruplex structure in its RNA and DNA sequence</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2581</span><span class="NLM_x">–</span> <span class="NLM_lpage">2593</span><span class="refDoi"> DOI: 10.1021/bi4016692</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi4016692" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Cru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=2581-2593&author=D.+Piekna-Przybylskaauthor=M.+A.+Sullivanauthor=G.+Sharmaauthor=R.+A.+Bambara&title=U3+region+in+the+HIV-1+genome+adopts+a+G-quadruplex+structure+in+its+RNA+and+DNA+sequence&doi=10.1021%2Fbi4016692"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">U3 region in the HIV-1 genome adopts a G-quadruplex structure in its RNA and DNA sequence</span></div><div class="casAuthors">Piekna-Przybylska, Dorota; Sullivan, Mark A.; Sharma, Gaurav; Bambara, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2581-2593</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Genomic regions rich in G residues are prone to adopt G-quadruplex structure.  Multiple Sp1-binding motifs arranged in tandem have been suggested to form this structure in promoters of cancer-related genes.  Here, we demonstrate that the G-rich proviral DNA sequence of the HIV-1 U3 region, which serves as a promoter of viral transcription, adopts a G-quadruplex structure.  The sequence contains three binding elements for transcription factor Sp1, which is involved in the regulation of HIV-1 latency, reactivation, and high-level virus expression.  We show that the three Sp1 binding motifs can adopt different forms of G-quadruplex structure and that the Sp1 protein can recognize and bind to its site folded into a G-quadruplex.  In addn., a c-kit2 specific antibody, designated hf2, binds to two different G-quadruplexes formed in Sp1 sites.  Since U3 is encoded at both viral genomic ends, the G-rich sequence is also present in the RNA genome.  We demonstrate that the RNA sequence of U3 forms dimers with characteristics known for intermol. G-quadruplexes.  Together with previous reports showing G-quadruplex dimers in the gag and cPPT regions, these results suggest that integrity of the two viral genomes is maintained through numerous intermol. G-quadruplexes formed in different RNA genome locations.  Reconstituted reverse transcription shows that the potassium-dependent structure formed in U3 RNA facilitates RT template switching, suggesting that the G-quadruplex contributes to recombination in U3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8c6X09GeI-LVg90H21EOLACvtfcHk0liD8pY296Y7eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Cru7k%253D&md5=42c96411b28a8f0b2ba1e1ce5891de0b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fbi4016692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi4016692%26sid%3Dliteratum%253Aachs%26aulast%3DPiekna-Przybylska%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DM.%2BA.%26aulast%3DSharma%26aufirst%3DG.%26aulast%3DBambara%26aufirst%3DR.%2BA.%26atitle%3DU3%2520region%2520in%2520the%2520HIV-1%2520genome%2520adopts%2520a%2520G-quadruplex%2520structure%2520in%2520its%2520RNA%2520and%2520DNA%2520sequence%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D2581%26epage%3D2593%26doi%3D10.1021%2Fbi4016692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butovskaya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daelemans, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannecouque, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Anti-HIV-1 activity of the G-quadruplex ligand BRACO-19</span> <span class="citation_source-journal">J. Antimicrob. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">3248</span><span class="NLM_x">–</span> <span class="NLM_lpage">3258</span><span class="refDoi"> DOI: 10.1093/jac/dku280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fjac%2Fdku280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25103489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisFaksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=3248-3258&author=R.+Perroneauthor=E.+Butovskayaauthor=D.+Daelemansauthor=G.+Paluauthor=C.+Pannecouqueauthor=S.+N.+Richter&title=Anti-HIV-1+activity+of+the+G-quadruplex+ligand+BRACO-19&doi=10.1093%2Fjac%2Fdku280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV-1 activity of the G-quadruplex ligand BRACO-19</span></div><div class="casAuthors">Perrone, Rosalba; Butovskaya, Elena; Daelemans, Dirk; Palu, Giorgio; Pannecouque, Christophe; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3248-3258</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: A dynamic G-quadruplex region has been previously shown to form in the long terminal repeat (LTR) promoter of the HIV-1 integrated DNA genome.  Inhibition of promoter activity and antiviral effects have been obsd. when this region was stabilized by BRACO-19, a trisubstituted acridine deriv. that binds G-quadruplexes.  Here, we aimed at characterizing the antiviral mechanism of action of BRACO-19 by analyzing its activity towards a broad range of HIV-1 strains, host cells and infection modes.  Methods: The antiviral activity of BRACO-19 in cell lines and primary cells infected or persistently infected by HIV-1 strains was evaluated at different times post-infection.  Virucidal, viral binding, time-dependent and drug-dependent assays were performed to identify the viral target step.  CD, UV spectroscopy and a reverse transcriptase (RT) stop assay were used to assess RNA G-quadruplex folding and inhibition of RT processing.  Results: Thorough virol. assays demonstrated that BRACO-19 acts both at the reverse transcription and the post-integration level during the virus life cycle.  This behavior was rationalized by the observation that a G-quadruplex-forming sequence identical to that of the LTR DNA is present at the 3'-end of the virus RNA genome.  Biophysics and biomol. testing showed that this region has the ability to fold into very stable G-quadruplex structures that are even more stabilized by BRACO-19, therefore inhibiting the reverse transcription process at the template level.  Conclusions: Our findings strongly support the activity of BRACO-19 at the viral G-quadruplex level and therefore strengthen the use of viral G-quadruplexes as new anti-HIV-1 targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobSFff3h6JxLVg90H21EOLACvtfcHk0li8VoFfA6N5nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisFaksr4%253D&md5=1b743fcdb7e70a889bf4835b29b9d4c8</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdku280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdku280%26sid%3Dliteratum%253Aachs%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DButovskaya%26aufirst%3DE.%26aulast%3DDaelemans%26aufirst%3DD.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DAnti-HIV-1%2520activity%2520of%2520the%2520G-quadruplex%2520ligand%2520BRACO-19%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D3248%26epage%3D3258%26doi%3D10.1093%2Fjac%2Fdku280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paramasivam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spolaore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3765</span><span class="NLM_x">–</span> <span class="NLM_lpage">3780</span><span class="refDoi"> DOI: 10.1093/nar/gkn120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkn120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=18490377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=3765-3780&author=S.+Cogoiauthor=M.+Paramasivamauthor=B.+Spolaoreauthor=L.+E.+Xodo&title=Structural+polymorphism+within+a+regulatory+element+of+the+human+KRAS+promoter%3A+formation+of+G4-DNA+recognized+by+nuclear+proteins&doi=10.1093%2Fnar%2Fgkn120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural polymorphism within a regulatory element of the human KRAS promoter: formation of G4-DNA recognized by nuclear proteins</span></div><div class="casAuthors">Cogoi, Susanna; Paramasivam, Manikandan; Spolaore, Barbara; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3765-3780</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The human KRAS proto-oncogene contains a crit. nuclease hypersensitive element (NHE) upstream of the major transcription initiation site.  In this article, we demonstrate by primer-extension expts., PAGE, chem. footprinting, CD, UV and FRET expts. that the G-rich strand of NHE (32R) folds into intra-mol. G-quadruplex structures.  Fluorescence data show that 32R in 100 mM KCl melts with a biphasic profile, showing the formation of two distinct G-quadruplexes with Tm of ∼55°C (Q1) and ∼72°C (Q2).  DMS-footprinting and CD suggest that Q1 can be a parallel and Q2 a mixed parallel/antiparallel G-quadruplex.  When dsNHE (32R hybridized to its complementary) is incubated with a nuclear ext. from Panc-1 cells, three DNA-protein complexes are obsd. by EMSA.  The complex of slower mobility is competed by quadruplex 32R, but not by mutant oligonucleotides, which cannot form a quadruplex structure.  Using paramagnetic beads coupled with 32R, we pulled down from the Panc-1 ext. proteins with affinity for quadruplex 32R.  One of these is the heterogeneous nuclear ribonucleoprotein A1, which was previously reported to unfold quadruplex DNA.  These study suggests a role of quadruplex DNA in KRAS transcription and provides the basis for the rationale design of mol. strategies to inhibit the expression of KRAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5riE_L0077Vg90H21EOLACvtfcHk0li8VoFfA6N5nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsF2rtb8%253D&md5=73fcda8805e5f3989bcf6824805ed046</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn120%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DParamasivam%26aufirst%3DM.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DStructural%2520polymorphism%2520within%2520a%2520regulatory%2520element%2520of%2520the%2520human%2520KRAS%2520promoter%253A%2520formation%2520of%2520G4-DNA%2520recognized%2520by%2520nuclear%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D3765%26epage%3D3780%26doi%3D10.1093%2Fnar%2Fgkn120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0li8VoFfA6N5nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Campbell, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Structural basis of DNA quadruplex recognition by an acridine drug</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">6722</span><span class="NLM_x">–</span> <span class="NLM_lpage">6724</span><span class="refDoi"> DOI: 10.1021/ja8016973</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8016973" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsVKmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=6722-6724&author=N.+H.+Campbellauthor=G.+N.+Parkinsonauthor=A.+P.+Reszkaauthor=S.+Neidle&title=Structural+basis+of+DNA+quadruplex+recognition+by+an+acridine+drug&doi=10.1021%2Fja8016973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of DNA Quadruplex Recognition by an Acridine Drug</span></div><div class="casAuthors">Campbell, Nancy H.; Parkinson, Gary N.; Reszka, Anthony P.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6722-6724</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The crystal structure of a complex between the bimol. human telomeric quadruplex d(TAGGGTTAGGGT)2 and the exptl. anticancer drug BRACO-19, has been detd., to 2.5 A resoln.  The binding site for the BRACO-19 mol. is at the interface of two parallel-folded quadruplexes, sandwiched between a G-tetrad surface and a TATA tetrad, and held in the site by networks of water mols.  The structure rationalizes the existing structure-activity data and provides a starting-point for the structure-based design of quadruplex-binding ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod3Mgt4sNRurVg90H21EOLACvtfcHk0lh3npQOWmvxMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsVKmt74%253D&md5=baabf2e38c8fb5b4fb780f09c2659217</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fja8016973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8016973%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DN.%2BH.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520of%2520DNA%2520quadruplex%2520recognition%2520by%2520an%2520acridine%2520drug%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D6722%26epage%3D6724%26doi%3D10.1021%2Fja8016973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Leonetti, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amodei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Angelo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benassi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Incalci, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zupi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biroccio, A.</span><span> </span><span class="NLM_article-title">Biological activity of the G-quadruplex ligand RHPS4 (3,11-difluoro-6,8,13-trimethyl-8H-quino[4,3,2-kl]acridinium methosulfate) is associated with telomere capping alteration</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">1138</span><span class="NLM_x">–</span> <span class="NLM_lpage">1146</span><span class="refDoi"> DOI: 10.1124/mol.104.001537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1124%2Fmol.104.001537" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2004&pages=1138-1146&author=C.+Leonettiauthor=S.+Amodeiauthor=C.+D%E2%80%99Angeloauthor=A.+Rizzoauthor=B.+Benassiauthor=A.+Antonelliauthor=R.+Elliauthor=M.+F.+Stevensauthor=M.+D%E2%80%99Incalciauthor=G.+Zupiauthor=A.+Biroccio&title=Biological+activity+of+the+G-quadruplex+ligand+RHPS4+%283%2C11-difluoro-6%2C8%2C13-trimethyl-8H-quino%5B4%2C3%2C2-kl%5Dacridinium+methosulfate%29+is+associated+with+telomere+capping+alteration&doi=10.1124%2Fmol.104.001537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1124%2Fmol.104.001537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.104.001537%26sid%3Dliteratum%253Aachs%26aulast%3DLeonetti%26aufirst%3DC.%26aulast%3DAmodei%26aufirst%3DS.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DC.%26aulast%3DRizzo%26aufirst%3DA.%26aulast%3DBenassi%26aufirst%3DB.%26aulast%3DAntonelli%26aufirst%3DA.%26aulast%3DElli%26aufirst%3DR.%26aulast%3DStevens%26aufirst%3DM.%2BF.%26aulast%3DD%25E2%2580%2599Incalci%26aufirst%3DM.%26aulast%3DZupi%26aufirst%3DG.%26aulast%3DBiroccio%26aufirst%3DA.%26atitle%3DBiological%2520activity%2520of%2520the%2520G-quadruplex%2520ligand%2520RHPS4%2520%25283%252C11-difluoro-6%252C8%252C13-trimethyl-8H-quino%255B4%252C3%252C2-kl%255Dacridinium%2520methosulfate%2529%2520is%2520associated%2520with%2520telomere%2520capping%2520alteration%26jtitle%3DMol.%2520Pharmacol.%26date%3D2004%26volume%3D66%26spage%3D1138%26epage%3D1146%26doi%3D10.1124%2Fmol.104.001537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1038/nrd3428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnrd3428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21455236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=261-275&author=S.+Balasubramanianauthor=L.+H.+Hurleyauthor=S.+Neidle&title=Targeting+G-quadruplexes+in+gene+promoters%3A+a+novel+anticancer+strategy%3F&doi=10.1038%2Fnrd3428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span></div><div class="casAuthors">Balasubramanian, Shankar; Hurley, Laurence H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-275</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are four-stranded DNA structures that are over-represented in gene promoter regions and are viewed as emerging therapeutic targets in oncol., as transcriptional repression of oncogenes through stabilization of these structures could be a novel anticancer strategy.  Many gene promoter G-quadruplexes have physicochem. properties and structural characteristics that might make them druggable, and their structural diversity suggests that a high degree of selectivity might be possible.  Here, we describe the evidence for G-quadruplexes in gene promoters and discuss their potential as therapeutic targets, as well as progress in the development of strategies to harness this potential through intervention with small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0ePjfQEBk7Vg90H21EOLACvtfcHk0lh3npQOWmvxMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D&md5=b3ea27635da9d1ff72d60228f1ef6d76</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrd3428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3428%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DTargeting%2520G-quadruplexes%2520in%2520gene%2520promoters%253A%2520a%2520novel%2520anticancer%2520strategy%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D261%26epage%3D275%26doi%3D10.1038%2Fnrd3428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Ali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattacharya, S.</span><span> </span><span class="NLM_article-title">DNA binders in clinical trials and chemotherapy</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4506</span><span class="NLM_x">–</span> <span class="NLM_lpage">4521</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.05.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.bmc.2014.05.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24947479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvFWltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=4506-4521&author=A.+Aliauthor=S.+Bhattacharya&title=DNA+binders+in+clinical+trials+and+chemotherapy&doi=10.1016%2Fj.bmc.2014.05.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">DNA binders in clinical trials and chemotherapy</span></div><div class="casAuthors">Ali, Asfa; Bhattacharya, Santanu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4506-4521</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cancer has always been a dreadful disease and continues to attract extensive research investigations.  Various targets have been identified to restrain cancer.  Among these DNA happens to be the most explored one.  A wide variety of small mols., often referred to as 'ligands', has been synthesized to target numerous structural features of DNA.  The sole purpose of such mol. design has been to interfere with the transcriptional machinery in order to drive the cancer cell toward apoptosis.  The mode of action of the DNA targeting ligands focuses either on the sequence-specificity by groove binding and strand cleavage, or by identifying the morphol. distinct higher order structures like that of the G-quadruplex DNA.  However, in spite of the extensive research, only a tiny fraction of the mols. have been able to reach clin. trials and only a handful are used in chemotherapy.  This review attempts to record the journey of the DNA binding small mols. from its inception to cancer therapy via various modifications at the mol. level.  Nevertheless, factors like limited bioavailability, severe toxicities, unfavorable pharmacokinetics etc. still prove to be the major impediments in the field which warrant considerable scope for further research investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5r0k2LEdJY7Vg90H21EOLACvtfcHk0lg57r9qWo7Lmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvFWltb4%253D&md5=08adad9a7ab77062428d6e697076b56e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.05.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.05.030%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DA.%26aulast%3DBhattacharya%26aufirst%3DS.%26atitle%3DDNA%2520binders%2520in%2520clinical%2520trials%2520and%2520chemotherapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D4506%26epage%3D4521%26doi%3D10.1016%2Fj.bmc.2014.05.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Franceschin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossetti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Ambrosio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schirripa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bianco, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortaggi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Natural and synthetic G-quadruplex interactive berberine derivatives</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1707</span><span class="NLM_x">–</span> <span class="NLM_lpage">1711</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2005.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.bmcl.2005.12.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=1707-1711&author=M.+Franceschinauthor=L.+Rossettiauthor=A.+D%E2%80%99Ambrosioauthor=S.+Schirripaauthor=A.+Biancoauthor=G.+Ortaggiauthor=M.+Savinoauthor=C.+Schultesauthor=S.+Neidle&title=Natural+and+synthetic+G-quadruplex+interactive+berberine+derivatives&doi=10.1016%2Fj.bmcl.2005.12.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DFranceschin%26aufirst%3DM.%26aulast%3DRossetti%26aufirst%3DL.%26aulast%3DD%25E2%2580%2599Ambrosio%26aufirst%3DA.%26aulast%3DSchirripa%26aufirst%3DS.%26aulast%3DBianco%26aufirst%3DA.%26aulast%3DOrtaggi%26aufirst%3DG.%26aulast%3DSavino%26aufirst%3DM.%26aulast%3DSchultes%26aufirst%3DC.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DNatural%2520and%2520synthetic%2520G-quadruplex%2520interactive%2520berberine%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D1707%26epage%3D1711%26doi%3D10.1016%2Fj.bmcl.2005.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Kim, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin-Ya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wierzba, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Telomestatin, a potent telomerase inhibitor that interacts quite specifically with the human telomeric intramolecular g-quadruplex</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">2098</span><span class="NLM_x">–</span> <span class="NLM_lpage">2099</span><span class="refDoi"> DOI: 10.1021/ja017308q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja017308q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD38Xht1eksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2002&pages=2098-2099&author=M.+Y.+Kimauthor=H.+Vankayalapatiauthor=K.+Shin-Yaauthor=K.+Wierzbaauthor=L.+H.+Hurley&title=Telomestatin%2C+a+potent+telomerase+inhibitor+that+interacts+quite+specifically+with+the+human+telomeric+intramolecular+g-quadruplex&doi=10.1021%2Fja017308q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Telomestatin, a Potent Telomerase Inhibitor That Interacts Quite Specifically with the Human Telomeric Intramolecular G-Quadruplex</span></div><div class="casAuthors">Kim, Mu-Yong; Vankayalapati, Hariprasad; Shin-ya, Kazuo; Wierzba, Konstanty; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2098-2099</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Telomestatin is a natural product isolated from Streptomyces anulatus 3533-SV4 and has been shown to be a very potent telomerase inhibitor.  The structural similarity between telomestatin and a G-tetrad suggested to us that the telomerase inhibition might be due to its ability either to facilitate the formation of or trap out preformed G-quadruplex structures, and thereby sequester single-stranded d[T2AG3]n primer mols. required for telomerase activity.  Significantly, telomestatin appears to be a more potent inhibitor of telomerase (5 nM) than any of the previously described G-quadruplex-interactive mols.  In this communication we provide the first exptl. evidence that telomestatin selectively facilitates the formation of or stabilizes intramol. G-quadruplexes, in particular, that produced from the human telomeric sequence d[T2AG3]4.  A simulated annealing (SA) docking approach was used to study the binding interactions of telomestatin with the intramol. antiparallel G-quadruplex structure.  Each intramol. G-quadruplex mol. was found to bind two telomestatin mols. (unpublished results).  A 2:1 model for the telomestatin bound in the external stacking mode in an energy minimized complex with the human telomeric basket-type G-quadruplex was constructed.  Our observation that a G-quadruplex-interactive mol. without significant groove interactions is able to reorient in a G-quadruplex structure points to the importance of core interaction with an asym. G-quadruplex structure in producing selective binding.  Furthermore, the G-quadruplex interactions of telomestatin are more selective for the intramol. structure in contrast to other G-quadruplex-interactive agents, such as TMPyP4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGHj0hQ7wPOrVg90H21EOLACvtfcHk0lg57r9qWo7Lmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xht1eksLo%253D&md5=a42b84751ccb715bc1a786449de7c13a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja017308q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja017308q%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BY.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DShin-Ya%26aufirst%3DK.%26aulast%3DWierzba%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DTelomestatin%252C%2520a%2520potent%2520telomerase%2520inhibitor%2520that%2520interacts%2520quite%2520specifically%2520with%2520the%2520human%2520telomeric%2520intramolecular%2520g-quadruplex%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D2098%26epage%3D2099%26doi%3D10.1021%2Fja017308q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span> </span><span class="NLM_article-title">Solution structure of a 2:1 quindoline-c-MYC G-quadruplex: insights into G-quadruplex-interactive small molecule drug design</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">17673</span><span class="NLM_x">–</span> <span class="NLM_lpage">17680</span><span class="refDoi"> DOI: 10.1021/ja205646q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja205646q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=17673-17680&author=J.+Daiauthor=M.+Carverauthor=L.+H.+Hurleyauthor=D.+Yang&title=Solution+structure+of+a+2%3A1+quindoline-c-MYC+G-quadruplex%3A+insights+into+G-quadruplex-interactive+small+molecule+drug+design&doi=10.1021%2Fja205646q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Solution Structure of a 2:1 Quindoline-c-MYC G-Quadruplex: Insights into G-Quadruplex-Interactive Small Molecule Drug Design</span></div><div class="casAuthors">Dai, Jixun; Carver, Megan; Hurley, Laurence H.; Yang, Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17673-17680</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Unimol. parallel-stranded G-quadruplex structures are found to be prevalent in gene promoters.  The nuclease hypersensitivity element III1 (NHE III1) of the c-MYC promoter can form transcriptionally active and silenced forms, and the formation of DNA G-quadruplex structures has been shown to be crit. for c-MYC transcriptional silencing.  The soln. structure of a 2:1 quindoline-G-quadruplex complex has been solved and shows unexpected features, including the drug-induced reorientation of the flanking sequences to form a new binding pocket.  While both 3' and 5' complexes show overall similar features, there are identifiable differences that emphasize the importance of both stacking and electronic interactions.  For the first time, we describe the importance of the shape of the ligand as well as the two flanking bases in detg. drug binding specificity.  These structures provide important insights for the structure-based rational design of drugs that bind to unimol. parallel G-quadruplexes commonly found in promoter elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr56uzmCEPLH7Vg90H21EOLACvtfcHk0lg57r9qWo7Lmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWrs7fI&md5=4aa27c51128a68f46974dad450e3d4a1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja205646q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja205646q%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DCarver%26aufirst%3DM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DSolution%2520structure%2520of%2520a%25202%253A1%2520quindoline-c-MYC%2520G-quadruplex%253A%2520insights%2520into%2520G-quadruplex-interactive%2520small%2520molecule%2520drug%2520design%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D17673%26epage%3D17680%26doi%3D10.1021%2Fja205646q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Fedoroff, O. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salazar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chemeris, V. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerwin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">NMR-Based model of a telomerase-inhibiting compound bound to G-quadruplex DNA</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">12367</span><span class="NLM_x">–</span> <span class="NLM_lpage">12374</span><span class="refDoi"> DOI: 10.1021/bi981330n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi981330n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaK1cXltFejtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=12367-12374&author=O.+Y.+Fedoroffauthor=M.+Salazarauthor=H.+Hanauthor=V.+V.+Chemerisauthor=S.+M.+Kerwinauthor=L.+H.+Hurley&title=NMR-Based+model+of+a+telomerase-inhibiting+compound+bound+to+G-quadruplex+DNA&doi=10.1021%2Fbi981330n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">NMR-Based Model of a Telomerase-Inhibiting Compound Bound to G-Quadruplex DNA</span></div><div class="casAuthors">Fedoroff, Oleg Yu.; Salazar, Miguel; Han, Haiyong; Chemeris, Violetta V.; Kerwin, Sean M.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12367-12374</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The single-stranded (TTAGGG)n tail of human telomeric DNA is known to form stable G-quadruplex structures.  Optimal telomerase activity requires the nonfolded single-stranded form of the primer, and stabilization of the G-quadruplex form is known to interfere with telomerase binding.  We have identified 3,4,9,10-perylenetetracarboxylic diimide-based ligands as potent inhibitors of human telomerase by using a primer extension assay that does not use PCR-based amplification of the telomerase primer extension products.  A set of NMR titrns. of the ligand into solns. of G-quadruplexes using various oligonucleotides related to human telomeric DNA showed strong and specific binding of the ligand to the G-quadruplex.  The exchange rate between bound and free DNA forms is slow on the NMR time scale and allows the unequivocal detn. of the binding site and mode of binding.  In the case of the 5'-TTAGGG sequence, the ligand-DNA complex consists of two quadruplexes oriented in a tail-to-tail manner with the ligand sandwiched between terminal G4 planes.  Longer telomeric sequences, such as TTAGGGTT, TTAGGGTTA, and TAGGGTTA, form 1:1 ligand-quadruplex complexes with the ligand bound at the GT step by a threading intercalation mode.  On the basis of 2D NOESY data, a model of the latter complex has been derived that is consistent with the available exptl. data.  The detn. of the soln. structure of this telomerase inhibitor bound to telomeric quadruplex DNA should help in the design of new anticancer agents with a unique and novel mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo94uPgdpduG7Vg90H21EOLACvtfcHk0lh7g12rlDjgfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltFejtLs%253D&md5=5dfd45e18b66e89d6098e68f5f3b43ed</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fbi981330n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi981330n%26sid%3Dliteratum%253Aachs%26aulast%3DFedoroff%26aufirst%3DO.%2BY.%26aulast%3DSalazar%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DChemeris%26aufirst%3DV.%2BV.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DNMR-Based%2520model%2520of%2520a%2520telomerase-inhibiting%2520compound%2520bound%2520to%2520G-quadruplex%2520DNA%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D12367%26epage%3D12374%26doi%3D10.1021%2Fbi981330n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertipaglia, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mella, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Quinone methides tethered to naphthalene diimides as selective G-quadruplex alkylating agents</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">13132</span><span class="NLM_x">–</span> <span class="NLM_lpage">13141</span><span class="refDoi"> DOI: 10.1021/ja904876q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja904876q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWkurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=13132-13141&author=M.+Di+Antonioauthor=F.+Doriaauthor=S.+N.+Richterauthor=C.+Bertipagliaauthor=M.+Mellaauthor=C.+Sissiauthor=M.+Palumboauthor=M.+Freccero&title=Quinone+methides+tethered+to+naphthalene+diimides+as+selective+G-quadruplex+alkylating+agents&doi=10.1021%2Fja904876q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Quinone Methides Tethered to Naphthalene Diimides as Selective G-Quadruplex Alkylating Agents</span></div><div class="casAuthors">Di Antonio, Marco; Doria, Filippo; Richter, Sara N.; Bertipaglia, Carolina; Mella, Mariella; Sissi, Claudia; Palumbo, Manlio; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">13132-13141</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed novel G-quadruplex (G-4) ligand/alkylating hybrid structures, tethering the naphthalene diimide moiety to quaternary ammonium salts of Mannich bases, as quinone-methide precursors, activatable by mild thermal digestion (40 °C).  The bis-substituted naphthalene diimides were efficiently synthesized, and their reactivity as activatable bis-alkylating agents was investigated in the presence of thiols and amines in aq. buffered solns.  The electrophilic intermediate, quinone-methide, involved in the alkylation process was trapped, in the presence of Et vinyl ether, in a hetero Diels-Alder [4 + 2] cycloaddn. reaction, yielding a substituted 2-ethoxychroman.  The DNA recognition and alkylation properties of these new derivs. were investigated by gel electrophoresis, CD, and enzymic assays.  The alkylation process occurred preferentially on the G-4 structure in comparison to other DNA conformations.  By dissecting reversible recognition and alkylation events, we found that the reversible process is a prerequisite to DNA alkylation, which in turn reinforces the G-quadruplex structural rearrangement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfmiH1zsIlN7Vg90H21EOLACvtfcHk0lh7g12rlDjgfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWkurfJ&md5=0d8813397e6dcd04034c9a939e423f55</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja904876q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja904876q%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DBertipaglia%26aufirst%3DC.%26aulast%3DMella%26aufirst%3DM.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DQuinone%2520methides%2520tethered%2520to%2520naphthalene%2520diimides%2520as%2520selective%2520G-quadruplex%2520alkylating%2520agents%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D13132%26epage%3D13141%26doi%3D10.1021%2Fja904876q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Collie, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Promontorio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hampel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span> </span><span class="NLM_article-title">Structural basis for telomeric G-quadruplex targeting by naphthalene diimide ligands</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">2723</span><span class="NLM_x">–</span> <span class="NLM_lpage">2731</span><span class="refDoi"> DOI: 10.1021/ja2102423</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja2102423" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFahtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=2723-2731&author=G.+W.+Collieauthor=R.+Promontorioauthor=S.+M.+Hampelauthor=M.+Miccoauthor=S.+Neidleauthor=G.+N.+Parkinson&title=Structural+basis+for+telomeric+G-quadruplex+targeting+by+naphthalene+diimide+ligands&doi=10.1021%2Fja2102423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Telomeric G-Quadruplex Targeting by Naphthalene Diimide Ligands</span></div><div class="casAuthors">Collie, Gavin W.; Promontorio, Rossella; Hampel, Sonja M.; Micco, Marialuisa; Neidle, Stephen; Parkinson, Gary N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2723-2731</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The folding of the single-stranded 3' end of the human telomere into G-quadruplex arrangements inhibits the overhang from hybridizing with the RNA template of telomerase and halts telomere maintenance in cancer cells.  The ability to thermally stabilize human telomeric DNA as a four-stranded G-quadruplex structure by developing selective small mol. compds. is a therapeutic path to regulating telomerase activity and thereby selectively inhibit cancer cell growth.  The development of compds. with the necessary selectivity and affinity to target parallel-stranded G-quadruplex structures has proved particularly challenging to date, relying heavily upon limited structural data.  We report here on a structure-based approach to the design of quadruplex-binding ligands to enhance affinity and selectivity for human telomeric DNA.  Crystal structures have been detd. of complexes between a 22-mer intramol. human telomeric quadruplex and two potent tetra-substituted naphthalene diimide compds., functionalized with pos. charged N-methyl-piperazine side-chains.  These compds. promote parallel-stranded quadruplex topol., binding exclusively to the 3' surface of each quadruplex.  There are significant differences between the complexes in terms of ligand mobility and in the interactions with quadruplex grooves.  One of the two ligands is markedly less mobile in the crystal complex and is more quadruplex-stabilizing, forming multiple electrostatic/hydrogen bond contacts with quadruplex phosphate groups.  The data presented here provides a structural rationale for the biophys. (effects on quadruplex thermal stabilization) and biol. data (inhibition of proliferation in cancer cell lines and evidence of in vivo antitumor activity) on compds. in this series and, thus, for the concept of telomere targeting with DNA quadruplex-binding small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi4fBPG4FU0rVg90H21EOLACvtfcHk0lh7g12rlDjgfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFahtA%253D%253D&md5=ccacb043644092d55fec3a75afeae0ad</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja2102423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja2102423%26sid%3Dliteratum%253Aachs%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DPromontorio%26aufirst%3DR.%26aulast%3DHampel%26aufirst%3DS.%2BM.%26aulast%3DMicco%26aufirst%3DM.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26atitle%3DStructural%2520basis%2520for%2520telomeric%2520G-quadruplex%2520targeting%2520by%2520naphthalene%2520diimide%2520ligands%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D2723%26epage%3D2731%26doi%3D10.1021%2Fja2102423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Micco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collie, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dale, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnmacht, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pazitna, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunaratnam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Structure-based design and evaluation of naphthalene diimide G-quadruplex ligands as telomere targeting agents in pancreatic cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2959</span><span class="NLM_x">–</span> <span class="NLM_lpage">2974</span><span class="refDoi"> DOI: 10.1021/jm301899y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301899y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlelu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2959-2974&author=M.+Miccoauthor=G.+W.+Collieauthor=A.+G.+Daleauthor=S.+A.+Ohnmachtauthor=I.+Pazitnaauthor=M.+Gunaratnamauthor=A.+P.+Reszkaauthor=S.+Neidle&title=Structure-based+design+and+evaluation+of+naphthalene+diimide+G-quadruplex+ligands+as+telomere+targeting+agents+in+pancreatic+cancer+cells&doi=10.1021%2Fjm301899y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design and Evaluation of Naphthalene Diimide G-Quadruplex Ligands As Telomere Targeting Agents in Pancreatic Cancer Cells</span></div><div class="casAuthors">Micco, Marialuisa; Collie, Gavin W.; Dale, Aaron G.; Ohnmacht, Stephan A.; Pazitna, Ingrida; Gunaratnam, Mekala; Reszka, Anthony P.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2959-2974</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tetra-substituted naphthalene diimide (ND) derivs. with pos. charged termini are potent stabilizers of human telomeric and gene promoter DNA quadruplexes and inhibit the growth of human cancer cells in vitro and in vivo.  The present study reports the enhancement of the pharmacol. properties of earlier ND compds. using structure-based design.  Crystal structures of three complexes with human telomeric intramol. quadruplexes demonstrate that two of the four strongly basic N-methyl-piperazine groups can be replaced by less basic morpholine groups with no loss of intermol. interactions in the grooves of the quadruplex.  The new compds. retain high affinity to human telomeric quadruplex DNA but are 10-fold more potent against the MIA PaCa-2 pancreatic cancer cell line, with IC50 values of ∼10 nM.  The lead compd. induces cellular senescence but does not inhibit telomerase activity at the nanomolar dosage levels required for inhibition of cellular proliferation.  Gene array qPCR anal. of MIA PaCa-2 cells treated with the lead compd. revealed significant dose-dependent modulation of a distinct subset of genes, including strong induction of DNA damage responsive genes CDKN1A, DDIT3, GADD45A/G, and PPM1D, and repression of genes involved in telomere maintenance, including hPOT1 and PARP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC6B5sA4fud7Vg90H21EOLACvtfcHk0lgJYUmd5gA-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlelu74%253D&md5=0697d89813efa5f1347cbdfddeeabc78</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm301899y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301899y%26sid%3Dliteratum%253Aachs%26aulast%3DMicco%26aufirst%3DM.%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DDale%26aufirst%3DA.%2BG.%26aulast%3DOhnmacht%26aufirst%3DS.%2BA.%26aulast%3DPazitna%26aufirst%3DI.%26aulast%3DGunaratnam%26aufirst%3DM.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DStructure-based%2520design%2520and%2520evaluation%2520of%2520naphthalene%2520diimide%2520G-quadruplex%2520ligands%2520as%2520telomere%2520targeting%2520agents%2520in%2520pancreatic%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2959%26epage%3D2974%26doi%3D10.1021%2Fjm301899y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Water soluble extended naphthalene diimides as pH fluorescent sensors and G-quadruplex ligands</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">3830</span><span class="NLM_x">–</span> <span class="NLM_lpage">3840</span><span class="refDoi"> DOI: 10.1039/c2ob07006e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fc2ob07006e" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22469919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFGhtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=3830-3840&author=F.+Doriaauthor=M.+Nadaiauthor=G.+Sattinauthor=L.+Pasottiauthor=S.+N.+Richterauthor=M.+Freccero&title=Water+soluble+extended+naphthalene+diimides+as+pH+fluorescent+sensors+and+G-quadruplex+ligands&doi=10.1039%2Fc2ob07006e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Water soluble extended naphthalene diimides as pH fluorescent sensors and G-quadruplex ligands</span></div><div class="casAuthors">Doria, Filippo; Nadai, Matteo; Sattin, Giovanna; Pasotti, Luca; Richter, Sara N.; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3830-3840</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Extended naphthalene diimides (NDIs) fused to 1,4-dihydropyrazine-2,3-dione, contg. two solubilizing moieties, have been synthesized.  Fluorescence spectra of the new NDIs were remarkably affected by pH, as the second deprotonation of the dihydropyrazinedione moiety (pKa 6.9) switched off the emission.  Binding to a G-quadruplex folded oligonucleotide and stoichiometry were evaluated by FRET melting assay and CD anal.  G-quadruplex binding was strongly enhanced shifting from pH 7.4 to pH 6.0 as a consequence of the dihydropyrazinedione moiety protonation.  Cytotoxicity studies using two human telomerase-pos. cell lines (HT29 and A549) revealed that the best G-quadruplex ligand was very active against the colon cell line, with an EC50 of 300 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCoDZL-SHlDbVg90H21EOLACvtfcHk0lgJYUmd5gA-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFGhtr0%253D&md5=9fa4f30354167482c7a7011491166940</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1039%2Fc2ob07006e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2ob07006e%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DSattin%26aufirst%3DG.%26aulast%3DPasotti%26aufirst%3DL.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DWater%2520soluble%2520extended%2520naphthalene%2520diimides%2520as%2520pH%2520fluorescent%2520sensors%2520and%2520G-quadruplex%2520ligands%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2012%26volume%3D10%26spage%3D3830%26epage%3D3840%26doi%3D10.1039%2Fc2ob07006e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manoli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandoth, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">A naphthalene diimide dyad for fluorescence switch-on detection of G-quadruplexes</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">9105</span><span class="NLM_x">–</span> <span class="NLM_lpage">9108</span><span class="refDoi"> DOI: 10.1039/C5CC01536G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2FC5CC01536G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25940220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFertbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=9105-9108&author=F.+Doriaauthor=A.+Oppiauthor=F.+Manoliauthor=S.+Bottiauthor=N.+Kandothauthor=V.+Grandeauthor=I.+Manetauthor=M.+Freccero&title=A+naphthalene+diimide+dyad+for+fluorescence+switch-on+detection+of+G-quadruplexes&doi=10.1039%2FC5CC01536G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A naphthalene diimide dyad for fluorescence switch-on detection of G-quadruplexes</span></div><div class="casAuthors">Doria, F.; Oppi, A.; Manoli, F.; Botti, S.; Kandoth, N.; Grande, V.; Manet, I.; Freccero, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">9105-9108</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A non-fluorescent naphthalene diimide (NDI) dimer, conjugating red and blue NDI dyes, becomes red/NIR emitting upon G-quadruplex binding.  The fluorescence lifetime which is significantly different for the complexes, the G-quadruplex/dimer and the weakly emitting ds-DNA/dimer is the key feature for the development of new rationally engineered G-quadruplex sensors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnwkfj1IuNNrVg90H21EOLACvtfcHk0lgJYUmd5gA-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFertbk%253D&md5=27d4ef48c711125bb587c985b3c91f11</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2FC5CC01536G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC01536G%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DOppi%26aufirst%3DA.%26aulast%3DManoli%26aufirst%3DF.%26aulast%3DBotti%26aufirst%3DS.%26aulast%3DKandoth%26aufirst%3DN.%26aulast%3DGrande%26aufirst%3DV.%26aulast%3DManet%26aufirst%3DI.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DA%2520naphthalene%2520diimide%2520dyad%2520for%2520fluorescence%2520switch-on%2520detection%2520of%2520G-quadruplexes%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2015%26volume%3D51%26spage%3D9105%26epage%3D9108%26doi%3D10.1039%2FC5CC01536G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">A photoreactive G-quadruplex ligand triggered by green light</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">2330</span><span class="NLM_x">–</span> <span class="NLM_lpage">2334</span><span class="refDoi"> DOI: 10.1002/chem.201405215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1002%2Fchem.201405215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25512076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVygt7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2330-2334&author=M.+Nadaiauthor=F.+Doriaauthor=L.+Germaniauthor=S.+N.+Richterauthor=M.+Freccero&title=A+photoreactive+G-quadruplex+ligand+triggered+by+green+light&doi=10.1002%2Fchem.201405215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Photoreactive G-Quadruplex Ligand Triggered by Green Light</span></div><div class="casAuthors">Nadai, Matteo; Doria, Filippo; Germani, Luca; Richter, Sara N.; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2330-2334</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A photoreactive mol. dye targeting the G-quadruplex nucleic acid (G4) of the human telomeric sequence Tel22, and several mutated analogs, was activated by green light (λ=532 nm).  Highly selective covalent modification of G4 vs. single-stranded and double-stranded DNA was achieved with efficiency up to 64%.  The phenoxyl radical was generated and detected by laser-flash photolysis as a reactive intermediate that targeted loop thymine residues.  These insights may suggest a non-invasive tool for selective nucleic acid tagging and pull-down cellular applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbaIIb6jJMNbVg90H21EOLACvtfcHk0lghoIxr_G7wow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVygt7nK&md5=a056a87113aaab1002956d207b6f000d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fchem.201405215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201405215%26sid%3Dliteratum%253Aachs%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DGermani%26aufirst%3DL.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DA%2520photoreactive%2520G-quadruplex%2520ligand%2520triggered%2520by%2520green%2520light%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2015%26volume%3D21%26spage%3D2330%26epage%3D2334%26doi%3D10.1002%2Fchem.201405215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Artusi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biasolo, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flamand, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calistri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">The Herpes Simplex Virus-1 genome contains multiple clusters of repeated G-quadruplex: Implications for the antiviral activity of a G-quadruplex ligand</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2015.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.antiviral.2015.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=25843424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFWls7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2015&pages=123-131&author=S.+Artusiauthor=M.+Nadaiauthor=R.+Perroneauthor=M.+A.+Biasoloauthor=G.+Paluauthor=L.+Flamandauthor=A.+Calistriauthor=S.+N.+Richter&title=The+Herpes+Simplex+Virus-1+genome+contains+multiple+clusters+of+repeated+G-quadruplex%3A+Implications+for+the+antiviral+activity+of+a+G-quadruplex+ligand&doi=10.1016%2Fj.antiviral.2015.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The herpes simplex virus-1 genome contains multiple clusters of repeated G-quadruplex: Implications for the antiviral activity of a G-quadruplex ligand</span></div><div class="casAuthors">Artusi, Sara; Nadai, Matteo; Perrone, Rosalba; Biasolo, Maria Angela; Palu, Giorgio; Flamand, Louis; Calistri, Arianna; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-131</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Guanine-rich nucleic acids can fold into G-quadruplexes, secondary structures implicated in important regulatory functions at the genomic level in humans, prokaryotes and viruses.  The remarkably high guanine content of the Herpes Simplex Virus-1 (HSV-1) genome prompted us to investigate both the presence of G-quadruplex forming sequences in the viral genome and the possibility to target them with G-quadruplex ligands to obtain anti-HSV-1 effects with a novel mechanism of action.  Using biophys., mol. biol. and antiviral assays, we showed that the HSV-1 genome displays multiple clusters of repeated sequences that form very stable G-quadruplexes.  These sequences are mainly located in the inverted repeats of the HSV-1 genome.  Treatment of HSV-1 infected cells with the G-quadruplex ligand BRACO-19 induced inhibition of virus prodn.  BRACO-19 was able to inhibit Taq polymerase processing at G-quadruplex forming sequences in the HSV-1 genome, and decreased intracellular viral DNA in infected cells.  The last step targeted by BRACO-19 was viral DNA replication, while no effect on virus entry in the cells was obsd.  This work, presents the first evidence of extended G-quadruplex sites in key regions of the HSV-1 genome, indicates the possibility to block viral DNA replication by G-quadruplex-ligand and therefore provides a proof of concept for the use of G-quadruplex ligands as new anti-herpetic therapeutic options.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdzMRiB-mdprVg90H21EOLACvtfcHk0lghoIxr_G7wow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFWls7Y%253D&md5=e5478e7cb2c4d276457f8db45a57b54d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DArtusi%26aufirst%3DS.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DPerrone%26aufirst%3DR.%26aulast%3DBiasolo%26aufirst%3DM.%2BA.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DFlamand%26aufirst%3DL.%26aulast%3DCalistri%26aufirst%3DA.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DThe%2520Herpes%2520Simplex%2520Virus-1%2520genome%2520contains%2520multiple%2520clusters%2520of%2520repeated%2520G-quadruplex%253A%2520Implications%2520for%2520the%2520antiviral%2520activity%2520of%2520a%2520G-quadruplex%2520ligand%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D118%26spage%3D123%26epage%3D131%26doi%3D10.1016%2Fj.antiviral.2015.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benotti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verga, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Substituted heterocyclic naphthalene diimides with unexpected acidity. Synthesis, properties, and reactivity</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">8616</span><span class="NLM_x">–</span> <span class="NLM_lpage">8625</span><span class="refDoi"> DOI: 10.1021/jo9017342</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo9017342" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2009&pages=8616-8625&author=F.+Doriaauthor=M.+di+Antonioauthor=M.+Benottiauthor=D.+Vergaauthor=M.+Freccero&title=Substituted+heterocyclic+naphthalene+diimides+with+unexpected+acidity.+Synthesis%2C+properties%2C+and+reactivity&doi=10.1021%2Fjo9017342"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjo9017342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo9017342%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3Ddi%2BAntonio%26aufirst%3DM.%26aulast%3DBenotti%26aufirst%3DM.%26aulast%3DVerga%26aufirst%3DD.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DSubstituted%2520heterocyclic%2520naphthalene%2520diimides%2520with%2520unexpected%2520acidity.%2520Synthesis%252C%2520properties%252C%2520and%2520reactivity%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2009%26volume%3D74%26spage%3D8616%26epage%3D8625%26doi%3D10.1021%2Fjo9017342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Zhou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span> </span><span class="NLM_article-title">An unusual addition reaction for constructing a novel pH-controlled fluorescence switch</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">292</span><span class="NLM_x">–</span> <span class="NLM_lpage">295</span><span class="refDoi"> DOI: 10.1021/ol102742f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol102742f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCltLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=292-295&author=C.+Zhouauthor=Y.+Liauthor=Y.+Zhaoauthor=J.+Zhangauthor=W.+Yangauthor=Y.+Li&title=An+unusual+addition+reaction+for+constructing+a+novel+pH-controlled+fluorescence+switch&doi=10.1021%2Fol102742f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">An Unusual Addition Reaction for Constructing a Novel pH-Controlled Fluorescence Switch</span></div><div class="casAuthors">Zhou, Chunjie; Li, Yongjun; Zhao, Yingjie; Zhang, Jianhong; Yang, Wenlong; Li, Yuliang</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-295</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A pH-controlled fluorescence switch based on a core-substituted naphthalenediimide compd. (NDI-DBU) was constructed.  The novel pentacyclic compd. (I; NDI-DBU) was derived from naphthalenediimide (NDI) and 1,8-diazabicyclo[5.4.0]undec-7-ene (DBU) in only one synthetic operation, catalysized by CuI.  The hydroxybutyl-substituted analog was also prepd. indicating the fluorophosre could be used to create further energy transfer systems.  Cyclic voltammetry, UV-visible and pH dependent fluorescence properties are reported for the di-Bu product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWHj7UHy2_QLVg90H21EOLACvtfcHk0ljOOiJlS1t36g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCltLjK&md5=fd1344124ad721f033002a0210ee873a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fol102742f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol102742f%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DAn%2520unusual%2520addition%2520reaction%2520for%2520constructing%2520a%2520novel%2520pH-controlled%2520fluorescence%2520switch%26jtitle%3DOrg.%2520Lett.%26date%3D2011%26volume%3D13%26spage%3D292%26epage%3D295%26doi%3D10.1021%2Fol102742f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Suraru, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zschieschang, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauk, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wurthner, F.</span><span> </span><span class="NLM_article-title">A core-extended naphthalene diimide as a p-channel semiconductor</span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">11504</span><span class="NLM_x">–</span> <span class="NLM_lpage">11506</span><span class="refDoi"> DOI: 10.1039/c1cc15144d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fc1cc15144d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21935548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=11504-11506&author=S.+L.+Suraruauthor=U.+Zschieschangauthor=H.+Klaukauthor=F.+Wurthner&title=A+core-extended+naphthalene+diimide+as+a+p-channel+semiconductor&doi=10.1039%2Fc1cc15144d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A core-extended naphthalene diimide as a p-channel semiconductor</span></div><div class="casAuthors">Suraru, Sabin-Lucian; Zschieschang, Ute; Klauk, Hagen; Wuerthner, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">11504-11506</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A novel naphthalene diimide (I) with a fully conjugated, extended π-core was synthesized in a one-pot, two-step reaction.  This org. semiconductor exhibits ambipolar transport properties with a large hole mobility of 0.56 cm2 V-1 s-1 and a current on/off ratio of 106 in bottom-gate, top-contact thin-film transistors prepd. by vacuum deposition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdbYm4C0mp8LVg90H21EOLACvtfcHk0ljOOiJlS1t36g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jtLfP&md5=d0e464e666e18b7d024da9b8e030dc0a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1039%2Fc1cc15144d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc15144d%26sid%3Dliteratum%253Aachs%26aulast%3DSuraru%26aufirst%3DS.%2BL.%26aulast%3DZschieschang%26aufirst%3DU.%26aulast%3DKlauk%26aufirst%3DH.%26aulast%3DWurthner%26aufirst%3DF.%26atitle%3DA%2520core-extended%2520naphthalene%2520diimide%2520as%2520a%2520p-channel%2520semiconductor%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2011%26volume%3D47%26spage%3D11504%26epage%3D11506%26doi%3D10.1039%2Fc1cc15144d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Langhals, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzel, S.</span><span> </span><span class="NLM_article-title">Laterally extended naphthalene tetracarboxylic bisimides</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">7781</span><span class="NLM_x">–</span> <span class="NLM_lpage">7784</span><span class="refDoi"> DOI: 10.1021/jo101520a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo101520a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=7781-7784&author=H.+Langhalsauthor=S.+Kinzel&title=Laterally+extended+naphthalene+tetracarboxylic+bisimides&doi=10.1021%2Fjo101520a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjo101520a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo101520a%26sid%3Dliteratum%253Aachs%26aulast%3DLanghals%26aufirst%3DH.%26aulast%3DKinzel%26aufirst%3DS.%26atitle%3DLaterally%2520extended%2520naphthalene%2520tetracarboxylic%2520bisimides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D7781%26epage%3D7784%26doi%3D10.1021%2Fjo101520a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Water-soluble naphthalene diimides as singlet oxygen sensitizers</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">8065</span><span class="NLM_x">–</span> <span class="NLM_lpage">8073</span><span class="refDoi"> DOI: 10.1021/jo401347z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo401347z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFeqsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=8065-8073&author=F.+Doriaauthor=I.+Manetauthor=V.+Grandeauthor=S.+Montiauthor=M.+Freccero&title=Water-soluble+naphthalene+diimides+as+singlet+oxygen+sensitizers&doi=10.1021%2Fjo401347z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Water-Soluble Naphthalene Diimides as Singlet Oxygen Sensitizers</span></div><div class="casAuthors">Doria, Filippo; Manet, Ilse; Grande, Vincenzo; Monti, Sandra; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8065-8073</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromo- and/or alkylamino-substituted and hydrosol. naphthalene diimides (NDIs) were synthesized to study their multimodal photophys. and photochem. properties.  Bromine-contg. NDIs deiv. behaved as both singlet oxygen (1O2) photosensitizers and fluorescent mols. upon irradn. at 532 nm.  Among the NDIs not contg. Br, one exhibited photophys. properties similar to those of Br-NDIs, by irradn. above 610 nm, suggesting that for these NDIs both singlet and triplet excited-state properties are strongly affected by length, structure of the solubilizing moieties, and pH of the soln.  Laser flash photolysis confirmed that the NDI lowest triplet excited state was efficiently populated, upon excitation at both 355 and 532 nm, and that free amine moieties quenched both the singlet and triplet excited states by intramol. electron transfer, with generation of detectable radical anions.  Time-resolved expts., monitoring the 1270 nm 1O2 phosphorescence decay generated upon laser irradn. at 532 nm, allowed a ranking of the NDIs as sensitizers, based on their 1O2 quantum yields (ΦΔ).  The tetrafunctionalized deriv., exhibiting a long-lived triplet state (τ ∼ 32 μs) and the most promising absorptivity for photodynamic therapy application, was tested as efficient photosensitizers in the photooxidns. of 1,5-dihydroxynaphthalene and 9,10-anthracenedipropionic acid in acetonitrile and water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF9kd3DdnyPLVg90H21EOLACvtfcHk0ljOOiJlS1t36g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFeqsr3N&md5=0949204927d8271a1584da0a89cd0c54</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjo401347z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo401347z%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DManet%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DV.%26aulast%3DMonti%26aufirst%3DS.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DWater-soluble%2520naphthalene%2520diimides%2520as%2520singlet%2520oxygen%2520sensitizers%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26spage%3D8065%26epage%3D8073%26doi%3D10.1021%2Fjo401347z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Palm, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Lange, T.</span><span> </span><span class="NLM_article-title">How shelterin protects mammalian telomeres</span> <span class="citation_source-journal">Annu. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">334</span><span class="refDoi"> DOI: 10.1146/annurev.genet.41.110306.130350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1146%2Fannurev.genet.41.110306.130350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=18680434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFemtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2008&pages=301-334&author=W.+Palmauthor=T.+de+Lange&title=How+shelterin+protects+mammalian+telomeres&doi=10.1146%2Fannurev.genet.41.110306.130350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">How shelterin protects mammalian telomeres</span></div><div class="casAuthors">Palm, Wilhelm; de Lange, Titia</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">301-334</span>CODEN:
                <span class="NLM_cas:coden">ARVGB7</span>;
        ISSN:<span class="NLM_cas:issn">0066-4197</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The genomes of prokaryotes and eukaryotic organelles are usually circular as are most plasmids and viral genomes.  In contrast, the nuclear genomes of eukaryotes are organized on linear chromosomes, which require mechanisms to protect and replicate DNA ends.  Eukaryotes navigate these problems with the advent of telomeres, protective nucleoprotein complexes at the ends of linear chromosomes, and telomerase, the enzyme that maintains the DNA in these structures.  Mammalian telomeres contain a specific protein complex, shelterin (I), that functions to protect chromosome ends from all aspects of the DNA damage response and regulates telomere maintenance by telomerase.  Recent expts., discussed here, have revealed how I represses the ATM and ATR kinase signaling pathways and hides chromosome ends from nonhomologous end joining and homol.-directed repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvfcjAIZbU2bVg90H21EOLACvtfcHk0lhHnZQjhzh5nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFemtbrI&md5=3d3ddc160214d999e23041f5e257791f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1146%2Fannurev.genet.41.110306.130350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.genet.41.110306.130350%26sid%3Dliteratum%253Aachs%26aulast%3DPalm%26aufirst%3DW.%26aulast%3Dde%2BLange%26aufirst%3DT.%26atitle%3DHow%2520shelterin%2520protects%2520mammalian%2520telomeres%26jtitle%3DAnnu.%2520Rev.%2520Genet.%26date%3D2008%26volume%3D42%26spage%3D301%26epage%3D334%26doi%3D10.1146%2Fannurev.genet.41.110306.130350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Rosu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabelica, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houssier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Pauw, E.</span><span> </span><span class="NLM_article-title">Determination of affinity, stoichiometry and sequence selectivity of minor groove binder complexes with double-stranded oligodeoxynucleotides by electrospray ionization mass spectrometry</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">e82</span><span class="refDoi"> DOI: 10.1093/nar/gnf081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgnf081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=e82&author=F.+Rosuauthor=V.+Gabelicaauthor=C.+Houssierauthor=E.+De+Pauw&title=Determination+of+affinity%2C+stoichiometry+and+sequence+selectivity+of+minor+groove+binder+complexes+with+double-stranded+oligodeoxynucleotides+by+electrospray+ionization+mass+spectrometry&doi=10.1093%2Fnar%2Fgnf081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgnf081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgnf081%26sid%3Dliteratum%253Aachs%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DHoussier%26aufirst%3DC.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DDetermination%2520of%2520affinity%252C%2520stoichiometry%2520and%2520sequence%2520selectivity%2520of%2520minor%2520groove%2520binder%2520complexes%2520with%2520double-stranded%2520oligodeoxynucleotides%2520by%2520electrospray%2520ionization%2520mass%2520spectrometry%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26spage%3De82%26doi%3D10.1093%2Fnar%2Fgnf081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Yuan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span> </span><span class="NLM_article-title">Mass spectrometry of G-quadruplex DNA: formation, recognition, property, conversion, and conformation</span> <span class="citation_source-journal">Mass Spectrom. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1121</span><span class="NLM_x">–</span> <span class="NLM_lpage">1142</span><span class="refDoi"> DOI: 10.1002/mas.20315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1002%2Fmas.20315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=21520218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyjsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1121-1142&author=G.+Yuanauthor=Q.+Zhangauthor=J.+Zhouauthor=H.+Li&title=Mass+spectrometry+of+G-quadruplex+DNA%3A+formation%2C+recognition%2C+property%2C+conversion%2C+and+conformation&doi=10.1002%2Fmas.20315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Mass spectrometry of G-quadruplex DNA: Formation, recognition, property, conversion, and conformation</span></div><div class="casAuthors">Yuan, Gu; Zhang, Qiang; Zhou, Jiang; Li, Huihui</div><div class="citationInfo"><span class="NLM_cas:title">Mass Spectrometry Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1142</span>CODEN:
                <span class="NLM_cas:coden">MSRVD3</span>;
        ISSN:<span class="NLM_cas:issn">0277-7037</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">G-quadruplexes are special secondary structures formed from G-rich sequences of DNA, and have proven to play important roles in a no. of biol. systems, including the regulation of gene transcription and translation.  The highly distinctive nature of G-quadruplex structures and their functions suggest that G-quadruplexes can act as novel targets for drug development.  As a highly sensitive anal. tool, mass spectrometry has been widely used for the anal. of G-quadruplex structures.  Electrospray-ionization mass spectrometry, in particular, has found captivating applications to probe interactions between small mols. and G-quadruplex DNA.  In this review, we will discuss: (1) mass spectrometry probing of the formation, binding affinity, and stoichiometry between G-quadruplexes and small mols.; (2) stabilization and collision-dissocn. behavior of G-quadruplex DNA; (3) the exploration of the equil. transfer between a G-quadruplex and duplex DNA; and (4) the ESI-MS anal. of the conversion of intramol. and intermol. G-quadruplexes.  Finally, we will also introduce the application of new techniques in the anal. of G-quadruplex conformation, such as ion-mobility and IR multiphoton-dissocn. mass spectrometry.  We believe that, with the new tech. developments, mass spectrometry will play an unparalleled role in the anal. of the G-quadruplex structures. © 2010 Wiley Periodicals, Inc., Mass Spec Rev 30:1121-1142, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP3zw3D_VtGrVg90H21EOLACvtfcHk0lhHnZQjhzh5nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyjsr7O&md5=0ae9102e7e612f83e402a1614fd03813</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fmas.20315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmas.20315%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMass%2520spectrometry%2520of%2520G-quadruplex%2520DNA%253A%2520formation%252C%2520recognition%252C%2520property%252C%2520conversion%252C%2520and%2520conformation%26jtitle%3DMass%2520Spectrom.%2520Rev.%26date%3D2011%26volume%3D30%26spage%3D1121%26epage%3D1142%26doi%3D10.1002%2Fmas.20315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Turner, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagan, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabris, D.</span><span> </span><span class="NLM_article-title">Inhibitory effects of archetypical nucleic acid ligands on the interactions of HIV-1 nucleocapsid protein with elements of Psi-RNA</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1305</span><span class="NLM_x">–</span> <span class="NLM_lpage">1316</span><span class="refDoi"> DOI: 10.1093/nar/gkl004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1093%2Fnar%2Fgkl004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=1305-1316&author=K.+B.+Turnerauthor=N.+A.+Haganauthor=D.+Fabris&title=Inhibitory+effects+of+archetypical+nucleic+acid+ligands+on+the+interactions+of+HIV-1+nucleocapsid+protein+with+elements+of+Psi-RNA&doi=10.1093%2Fnar%2Fgkl004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl004%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DK.%2BB.%26aulast%3DHagan%26aufirst%3DN.%2BA.%26aulast%3DFabris%26aufirst%3DD.%26atitle%3DInhibitory%2520effects%2520of%2520archetypical%2520nucleic%2520acid%2520ligands%2520on%2520the%2520interactions%2520of%2520HIV-1%2520nucleocapsid%2520protein%2520with%2520elements%2520of%2520Psi-RNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D1305%26epage%3D1316%26doi%3D10.1093%2Fnar%2Fgkl004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Fernando, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladame, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkitaraman, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">A conserved quadruplex motif located in a transcription activation site of the human c-kit oncogene</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7854</span><span class="NLM_x">–</span> <span class="NLM_lpage">7860</span><span class="refDoi"> DOI: 10.1021/bi0601510</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0601510" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD28XltVKgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7854-7860&author=H.+Fernandoauthor=A.+P.+Reszkaauthor=J.+Huppertauthor=S.+Ladameauthor=S.+Rankinauthor=A.+R.+Venkitaramanauthor=S.+Neidleauthor=S.+Balasubramanian&title=A+conserved+quadruplex+motif+located+in+a+transcription+activation+site+of+the+human+c-kit+oncogene&doi=10.1021%2Fbi0601510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Quadruplex Motif Located in a Transcription Activation Site of the Human c-kit Oncogene</span></div><div class="casAuthors">Fernando, Himesh; Reszka, Anthony P.; Huppert, Julian; Ladame, Sylvain; Rankin, Sarah; Venkitaraman, Ashok R.; Neidle, Stephen; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7854-7860</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The c-kit gene encodes a receptor tyrosine kinase, whose engagement by its ligand triggers signals leading to cell proliferation.  C-kit activity is elevated in gastrointestinal stromal tumors (GISTs), and its therapeutic inhibition by small mols. such as imatinib is clin. validated.  We identified a putative quadruplex-forming 21-nucleotide sequence upstream of the c-kit transcription initiation site (c-kit21), on the G-rich strand, which occupies a site required for core promoter activity.  Here, we show by NMR, CD, and UV spectroscopic methods that c-kit21 forms quadruplexes under physiol. conditions.  Mutational anal. of c-kit21 has provided insights into its structural polymorphism.  In particular, one mutated form appears to form a single quadruplex species that adopts a parallel conformation.  The quadruplex-forming sequence shows a high level of sequence conservation across human, mouse, rat, and chimpanzee.  The small variation in sequence between the quadruplex in human/chimpanzee as compared to the rat/mouse was examd. more closely by biophys. methods.  Despite a variation in the sequence and length of loop 2, the quadruplexes showed both comparable CD spectra, indicative of parallel quadruplexes, and also similar thermal stability profiles, suggesting conservation of biophys. characteristics.  Collectively, the evidence suggests that this quadruplex is a serious target for a detailed functional investigation at the cell-biol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_huuxfinFYLVg90H21EOLACvtfcHk0lgjWw4aM3Xw8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltVKgtrc%253D&md5=4ffebe621ea8ba509e3d10fa1ebb202d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fbi0601510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0601510%26sid%3Dliteratum%253Aachs%26aulast%3DFernando%26aufirst%3DH.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DHuppert%26aufirst%3DJ.%26aulast%3DLadame%26aufirst%3DS.%26aulast%3DRankin%26aufirst%3DS.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DA%2520conserved%2520quadruplex%2520motif%2520located%2520in%2520a%2520transcription%2520activation%2520site%2520of%2520the%2520human%2520c-kit%2520oncogene%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D7854%26epage%3D7860%26doi%3D10.1021%2Fbi0601510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Differential targeting of unpaired bases within duplex DNA by the natural compound clerocidin: a valuable tool to dissect DNA secondary structure</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e52994</span><span class="refDoi"> DOI: 10.1371/journal.pone.0052994</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1371%2Fjournal.pone.0052994" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e52994&author=M.+Nadaiauthor=G.+Paluauthor=M.+Palumboauthor=S.+N.+Richter&title=Differential+targeting+of+unpaired+bases+within+duplex+DNA+by+the+natural+compound+clerocidin%3A+a+valuable+tool+to+dissect+DNA+secondary+structure&doi=10.1371%2Fjournal.pone.0052994"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0052994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0052994%26sid%3Dliteratum%253Aachs%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DDifferential%2520targeting%2520of%2520unpaired%2520bases%2520within%2520duplex%2520DNA%2520by%2520the%2520natural%2520compound%2520clerocidin%253A%2520a%2520valuable%2520tool%2520to%2520dissect%2520DNA%2520secondary%2520structure%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De52994%26doi%3D10.1371%2Fjournal.pone.0052994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menegazzo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span> </span><span class="NLM_article-title">Reactivity of clerocidin towards adenine: implications for base-modulated DNA damage</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">976</span><span class="NLM_x">–</span> <span class="NLM_lpage">985</span><span class="refDoi"> DOI: 10.1039/b819049f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fb819049f" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=976-985&author=S.+N.+Richterauthor=I.+Menegazzoauthor=M.+Nadaiauthor=S.+Moroauthor=M.+Palumbo&title=Reactivity+of+clerocidin+towards+adenine%3A+implications+for+base-modulated+DNA+damage&doi=10.1039%2Fb819049f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2Fb819049f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb819049f%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DMenegazzo%26aufirst%3DI.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DMoro%26aufirst%3DS.%26aulast%3DPalumbo%26aufirst%3DM.%26atitle%3DReactivity%2520of%2520clerocidin%2520towards%2520adenine%253A%2520implications%2520for%2520base-modulated%2520DNA%2520damage%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2009%26volume%3D7%26spage%3D976%26epage%3D985%26doi%3D10.1039%2Fb819049f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palumbo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span> </span><span class="NLM_article-title">Clerocidin-mediated DNA footprinting discriminates among different G-quadruplex conformations and detects tetraplex folding in a duplex environment</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1830</span><span class="NLM_x">, </span> <span class="NLM_fpage">4660</span><span class="NLM_x">–</span> <span class="NLM_lpage">4668</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2013.05.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.bbagen.2013.05.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=23747297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emsLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1830&publication_year=2013&pages=4660-4668&author=M.+Nadaiauthor=G.+Sattinauthor=G.+Paluauthor=M.+Palumboauthor=S.+N.+Richter&title=Clerocidin-mediated+DNA+footprinting+discriminates+among+different+G-quadruplex+conformations+and+detects+tetraplex+folding+in+a+duplex+environment&doi=10.1016%2Fj.bbagen.2013.05.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Clerocidin-mediated DNA footprinting discriminates among different G-quadruplex conformations and detects tetraplex folding in a duplex environment</span></div><div class="casAuthors">Nadai, Matteo; Sattin, Giovanna; Palu, Giorgio; Palumbo, Manlio; Richter, Sara N.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1830</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4660-4668</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">G-quadruplexes are polymorphic non-canonical nucleic acid conformations involved both in physiol. and pathol. processes.  Given the high degree of folding heterogeneity and comparable conformational stabilities, different G-quadruplex forms can occur simultaneously, hence rendering the use of basic instrumental methods for structure detn., like X-ray diffraction or NMR, hardly useful.  Footprinting techniques represent valuable and relatively rapid alternative to characterize DNA folding.  The natural diterpenoid clerocidin is an alkylating agent that specifically reacts at single-stranded DNA regions, with different mechanisms depending on the exposed nucleotide.  Clerocidin was used to footprint G-quadruplex structures formed by telomeric and oncogene promoter sequences (c-myc, bcl-2, c-kit2), and by the thrombin binding aptamer.  The easy modulability of CL reactivity towards DNA bases permitted to discriminate fully and partially protected sites, highlights stretched portions of the G-quadruplex conformation, and discriminate among topologies adopted by one sequence in different environmental conditions.  Importantly, CL displayed the unique property to allow detection of G-quadruplex folding within a duplex context.  CL is a finely performing new tool to unveil G-quadruplex arrangements in DNA sequences under genomically relevant conditions.  Nucleic acid G-quadruplex structures are an emerging research field because of the recent indication of their involvement in a series of key biol. functions, in particular in regulation of proliferation-assocd. gene expression.  The use of clerocidin as footprinting agent to identify G-quadruplex structures under genomically relevant conditions may allow detection of new G-quadruplex-based regulatory regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-IrvRFi9AJLVg90H21EOLACvtfcHk0lgjWw4aM3Xw8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emsLbM&md5=c7d722997b1941abb18af11c402d3fa1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2013.05.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2013.05.039%26sid%3Dliteratum%253Aachs%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DSattin%26aufirst%3DG.%26aulast%3DPalu%26aufirst%3DG.%26aulast%3DPalumbo%26aufirst%3DM.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26atitle%3DClerocidin-mediated%2520DNA%2520footprinting%2520discriminates%2520among%2520different%2520G-quadruplex%2520conformations%2520and%2520detects%2520tetraplex%2520folding%2520in%2520a%2520duplex%2520environment%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2013%26volume%3D1830%26spage%3D4660%26epage%3D4668%26doi%3D10.1016%2Fj.bbagen.2013.05.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Doria, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Percivalle, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sissi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaffaroni, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcaro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artese, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freccero, M.</span><span> </span><span class="NLM_article-title">Hybrid ligand-alkylating agents targeting telomeric G-quadruplex structures</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2798</span><span class="NLM_x">–</span> <span class="NLM_lpage">2806</span><span class="refDoi"> DOI: 10.1039/c2ob06816h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1039%2Fc2ob06816h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22367401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvFKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=2798-2806&author=F.+Doriaauthor=M.+Nadaiauthor=M.+Foliniauthor=M.+Di+Antonioauthor=L.+Germaniauthor=C.+Percivalleauthor=C.+Sissiauthor=N.+Zaffaroniauthor=S.+Alcaroauthor=A.+Arteseauthor=S.+N.+Richterauthor=M.+Freccero&title=Hybrid+ligand-alkylating+agents+targeting+telomeric+G-quadruplex+structures&doi=10.1039%2Fc2ob06816h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Hybrid ligand-alkylating agents targeting telomeric G-quadruplex structures</span></div><div class="casAuthors">Doria, Filippo; Nadai, Matteo; Folini, Marco; Di Antonio, Marco; Germani, Luca; Percivalle, Claudia; Sissi, Claudia; Zaffaroni, Nadia; Alcaro, Stefano; Artese, Anna; Richter, Sara N.; Freccero, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2798-2806</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The synthesis, physico-chem. properties and biol. effects of a new class of naphthalene diimides (NDIs) capable of reversibly binding telomeric DNA and alkylate it through an electrophilic quinone methide moiety (QM), are reported.  FRET and CD assays showed a marked stabilization and selectivity towards telomeric G4 DNA folded in a hybrid topol.  NDI-QMs' alkylating properties revealed a good reactivity on single nucleosides and selectivity towards telomeric G4.  A selected NDI was able to significantly impair the growth of melanoma cells by causing telomere dysfunction and down-regulation of telomerase expression.  These findings point to the authors' hybrid ligand-alkylating NDIs as possible tools for the development of novel targeted anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDdu_j-tKmYLVg90H21EOLACvtfcHk0lgzhE-EX-Laxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvFKrtrw%253D&md5=43be7fdab1bb56fe7188ab79f98276ba</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1039%2Fc2ob06816h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2ob06816h%26sid%3Dliteratum%253Aachs%26aulast%3DDoria%26aufirst%3DF.%26aulast%3DNadai%26aufirst%3DM.%26aulast%3DFolini%26aufirst%3DM.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DGermani%26aufirst%3DL.%26aulast%3DPercivalle%26aufirst%3DC.%26aulast%3DSissi%26aufirst%3DC.%26aulast%3DZaffaroni%26aufirst%3DN.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DArtese%26aufirst%3DA.%26aulast%3DRichter%26aufirst%3DS.%2BN.%26aulast%3DFreccero%26aufirst%3DM.%26atitle%3DHybrid%2520ligand-alkylating%2520agents%2520targeting%2520telomeric%2520G-quadruplex%2520structures%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2012%26volume%3D10%26spage%3D2798%26epage%3D2806%26doi%3D10.1039%2Fc2ob06816h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Wilkinson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pincus, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shepard, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walton, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergin, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labib, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teintze, M.</span><span> </span><span class="NLM_article-title">Novel compounds containing multiple guanide groups that bind the HIV coreceptor CXCR4</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">–</span> <span class="NLM_lpage">263</span><span class="refDoi"> DOI: 10.1128/AAC.00709-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1128%2FAAC.00709-10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=255-263&author=R.+A.+Wilkinsonauthor=S.+H.+Pincusauthor=J.+B.+Shepardauthor=S.+K.+Waltonauthor=E.+P.+Berginauthor=M.+Labibauthor=M.+Teintze&title=Novel+compounds+containing+multiple+guanide+groups+that+bind+the+HIV+coreceptor+CXCR4&doi=10.1128%2FAAC.00709-10"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1128%2FAAC.00709-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00709-10%26sid%3Dliteratum%253Aachs%26aulast%3DWilkinson%26aufirst%3DR.%2BA.%26aulast%3DPincus%26aufirst%3DS.%2BH.%26aulast%3DShepard%26aufirst%3DJ.%2BB.%26aulast%3DWalton%26aufirst%3DS.%2BK.%26aulast%3DBergin%26aufirst%3DE.%2BP.%26aulast%3DLabib%26aufirst%3DM.%26aulast%3DTeintze%26aufirst%3DM.%26atitle%3DNovel%2520compounds%2520containing%2520multiple%2520guanide%2520groups%2520that%2520bind%2520the%2520HIV%2520coreceptor%2520CXCR4%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D255%26epage%3D263%26doi%3D10.1128%2FAAC.00709-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Furman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fyfe, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Clair, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinhold, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rideout, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehrman, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolognesi, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsuya, H.</span><span> </span><span class="NLM_article-title">Phosphorylation of 3′-azido-3′-deoxythymidine and selective interaction of the 5′-triphosphate with human immunodeficiency virus reverse transcriptase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">8333</span><span class="NLM_x">–</span> <span class="NLM_lpage">8337</span><span class="refDoi"> DOI: 10.1073/pnas.83.21.8333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1073%2Fpnas.83.21.8333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=2430286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaL2sXktlCqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=8333-8337&author=P.+A.+Furmanauthor=J.+A.+Fyfeauthor=M.+H.+St.+Clairauthor=K.+Weinholdauthor=J.+L.+Rideoutauthor=G.+A.+Freemanauthor=S.+N.+Lehrmanauthor=D.+P.+Bolognesiauthor=S.+Broderauthor=H.+Mitsuya&title=Phosphorylation+of+3%E2%80%B2-azido-3%E2%80%B2-deoxythymidine+and+selective+interaction+of+the+5%E2%80%B2-triphosphate+with+human+immunodeficiency+virus+reverse+transcriptase&doi=10.1073%2Fpnas.83.21.8333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of 3'-azido-3'-deoxythymidine and selective interaction of the 5'-triphosphate with human immunodeficiency virus reverse transcriptase</span></div><div class="casAuthors">Furman, Phillip A.; Fyfe, James A.; St. Clair, Marty H.; Weinhold, Kent; Rideout, Janet L.; Freeman, George A.; Lehrman, Sandra Nusinoff; Bolognesi, Dani P.; Broder, Samuel; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8333-7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The thymidine analog 3'-azido-3'-deoxythymidine (azidothymidine)  [30516-87-1] was previously shown to inhibit human immunodeficiency virus (HIV) replication effectively in the 50-500 nM range.  In contrast, inhibition of the growth of uninfected human fibroblasts and lymphocytes was obsd. only at concns. above 1 mM.  The nature of this selectivity was investigated.  Azidothymidine anabolism to the 5'-mono-  [29706-85-2], -di-  [106060-89-3], and -triphosphate  [92586-35-1] derivs. was similar in uninfected and HIV-infected cells.  The level of azidothymidine monophosphate was high, whereas the levels of the di- and triphosphate were low (≤5 μM and ≤2 μM, resp.).  Cytosolic thymidine kinase (EC 2.7.1.21)  [9002-06-6] was responsible for phosphorylation of azidothymidine to its monophosphate.  Purified thymidine kinase catalyzed the phosphorylations of thymidine  [50-89-5] and azidothymidine with apparent Km values of 2.9 μM and 3.0 μM.  The maximal rate of phosphorylation with azidothymidine was equal to 60% of the rate with thymidine.  Phosphorylation of azidothymidine monophosphate to the diphosphate also appeared to be catalyzed by a host-cell enzyme, thymidylate kinase (EC 2.7.4.9)  [9014-43-1].  The apparent Km value for azidothymidine monophosphate was 2-fold greater than the value for dTMP  [365-07-1] (8.6 μM vs. 4.1 μM), but the maximal phosphorylation rate was only 0.3% of the dTMP rate.  These kinetic consts. were consistent with the anabolism results and indicated that azidothymidine monophosphate is an alternative-substrate inhibitor of thymidylate kinase.  This conclusion was reflected in the observation that cell incubated with azidothymidine had reduced intracellular levels of dTTP  [365-08-2].  IC50 (concn. of inhibitor that inhibits enzyme activity 50%) values were detd. for azidothymidine triphosphate with reverse transcriptase  [9068-38-6] and with immortalized human lymphocyte (H9 cell) DNA polymerase  [9012-90-2] α .  Azidothymidine triphosphate competed about 100-fold better for the HIV reverse transcriptase than for the cellular DNA polymerase α.  These results suggest that azidothymidine is nonselectively phosphorylated but that the triphosphate deriv. efficiently and selectively binds to the HIV reverse transcriptase.  Incorporation of azidothymidylate into a growing DNA strand should terminate DNA elongation and thus inhibit DNA synthesis.  These results may have relevance to the antiviral action mechanism for azidothymidine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1_5UurNpyQrVg90H21EOLACvtfcHk0lgzhE-EX-Laxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXktlCqug%253D%253D&md5=2dcdd98db3c5708b2c31058cc47eeb41</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.21.8333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.21.8333%26sid%3Dliteratum%253Aachs%26aulast%3DFurman%26aufirst%3DP.%2BA.%26aulast%3DFyfe%26aufirst%3DJ.%2BA.%26aulast%3DSt.%2BClair%26aufirst%3DM.%2BH.%26aulast%3DWeinhold%26aufirst%3DK.%26aulast%3DRideout%26aufirst%3DJ.%2BL.%26aulast%3DFreeman%26aufirst%3DG.%2BA.%26aulast%3DLehrman%26aufirst%3DS.%2BN.%26aulast%3DBolognesi%26aufirst%3DD.%2BP.%26aulast%3DBroder%26aufirst%3DS.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DPhosphorylation%2520of%25203%25E2%2580%25B2-azido-3%25E2%2580%25B2-deoxythymidine%2520and%2520selective%2520interaction%2520of%2520the%25205%25E2%2580%25B2-triphosphate%2520with%2520human%2520immunodeficiency%2520virus%2520reverse%2520transcriptase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1986%26volume%3D83%26spage%3D8333%26epage%3D8337%26doi%3D10.1073%2Fpnas.83.21.8333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Mitsuya, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Looney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuno, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong-Staal, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, S.</span><span> </span><span class="NLM_article-title">Dextran sulfate suppression of viruses in the HIV family: inhibition of virion binding to CD4+ cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">240</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">649</span><span class="refDoi"> DOI: 10.1126/science.2452480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1126%2Fscience.2452480" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=240&publication_year=1988&pages=646-649&author=H.+Mitsuyaauthor=D.+J.+Looneyauthor=S.+Kunoauthor=R.+Uenoauthor=F.+Wong-Staalauthor=S.+Broder&title=Dextran+sulfate+suppression+of+viruses+in+the+HIV+family%3A+inhibition+of+virion+binding+to+CD4%2B+cells&doi=10.1126%2Fscience.2452480"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1126%2Fscience.2452480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2452480%26sid%3Dliteratum%253Aachs%26aulast%3DMitsuya%26aufirst%3DH.%26aulast%3DLooney%26aufirst%3DD.%2BJ.%26aulast%3DKuno%26aufirst%3DS.%26aulast%3DUeno%26aufirst%3DR.%26aulast%3DWong-Staal%26aufirst%3DF.%26aulast%3DBroder%26aufirst%3DS.%26atitle%3DDextran%2520sulfate%2520suppression%2520of%2520viruses%2520in%2520the%2520HIV%2520family%253A%2520inhibition%2520of%2520virion%2520binding%2520to%2520CD4%252B%2520cells%26jtitle%3DScience%26date%3D1988%26volume%3D240%26spage%3D646%26epage%3D649%26doi%3D10.1126%2Fscience.2452480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Czerwinska, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juskowiak, B.</span><span> </span><span class="NLM_article-title">Photoisomerizable arylstilbazolium ligands recognize parallel and antiparallel structures of G-quadruplexes</span> <span class="citation_source-journal">Int. J. Biol. Macromol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">582</span><span class="refDoi"> DOI: 10.1016/j.ijbiomac.2012.06.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.ijbiomac.2012.06.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=22750579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht12lt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=576-582&author=I.+Czerwinskaauthor=B.+Juskowiak&title=Photoisomerizable+arylstilbazolium+ligands+recognize+parallel+and+antiparallel+structures+of+G-quadruplexes&doi=10.1016%2Fj.ijbiomac.2012.06.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Photoisomerizable arylstilbazolium ligands recognize parallel and antiparallel structures of G-quadruplexes</span></div><div class="casAuthors">Czerwinska, Izabella; Juskowiak, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biological Macromolecules</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">576-582</span>CODEN:
                <span class="NLM_cas:coden">IJBMDR</span>;
        ISSN:<span class="NLM_cas:issn">0141-8130</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The photoisomerization and DNA interaction studies of three arylstilbazolium derivs. with various samples of nucleic acids (duplexes, triplexes and tetraplexes) are reported.  The equil. dialysis study revealed high binding affinities of ligands to tetraplex structures.  The quadruplex-binding affinity could be switched by light, e.g., the E,E and E,Z isomers of 1,4-bis(vinylquinolinium)benzene (1) interacted with parallel and antiparallel tetraplexes exhibiting different binding selectivity.  The E,Z-1 showed higher binding preference for c-myc DNA (a propeller-type quadruplex), whereas the E,E-1 favorably interacted with telomeric DNA (a basket-type quadruplex).  The presence of quadruplex DNA hampered photoisomerization of quadruplex-bound ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAzBIzyDSfcbVg90H21EOLACvtfcHk0ljQtJOJeR0aCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht12lt7vP&md5=ed59d4a08d08ba39a1ae87235dcd42dd</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.ijbiomac.2012.06.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijbiomac.2012.06.027%26sid%3Dliteratum%253Aachs%26aulast%3DCzerwinska%26aufirst%3DI.%26aulast%3DJuskowiak%26aufirst%3DB.%26atitle%3DPhotoisomerizable%2520arylstilbazolium%2520ligands%2520recognize%2520parallel%2520and%2520antiparallel%2520structures%2520of%2520G-quadruplexes%26jtitle%3DInt.%2520J.%2520Biol.%2520Macromol.%26date%3D2012%26volume%3D51%26spage%3D576%26epage%3D582%26doi%3D10.1016%2Fj.ijbiomac.2012.06.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Rachwal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, K. R.</span><span> </span><span class="NLM_article-title">Quadruplex melting</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2007.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1016%2Fj.ymeth.2007.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=17967699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1amurnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2007&pages=291-301&author=P.+A.+Rachwalauthor=K.+R.+Fox&title=Quadruplex+melting&doi=10.1016%2Fj.ymeth.2007.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex melting</span></div><div class="casAuthors">Rachwal, Phillip A.; Fox, Keith R.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">291-301</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Melting curves are commonly used to det. the stability of folded nucleic acid structures and their interaction with ligands.  This paper describes how the technique can be applied to study the properties of four-stranded nucleic acid structures that are formed by G-rich oligonucleotides.  Changes in the absorbance (at 295 nm), CD (at 260 or 295 nm) or fluorescence of appropriately labeled oligonucleotides, can be used to measure the stability and kinetics of folding.  This paper focuses on a fluorescence melting technique, and explains how this can be used to det. the Tm (T1/2) of intramol. quadruplexes and the effects of quadruplex-binding ligands.  Quant. anal. of these melting curves can be used to det. the thermodn. (ΔH, ΔG, and ΔS) and kinetic (k1, k-1) parameters.  The method can also be adapted to investigate the equil. between quadruplex and duplex DNA and to explore the selectivity of ligands for one or other structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDcGxot8SuB7Vg90H21EOLACvtfcHk0ljQtJOJeR0aCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1amurnP&md5=d36626c59fa1213824577567e46e3810</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2007.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2007.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DRachwal%26aufirst%3DP.%2BA.%26aulast%3DFox%26aufirst%3DK.%2BR.%26atitle%3DQuadruplex%2520melting%26jtitle%3DMethods%26date%3D2007%26volume%3D43%26spage%3D291%26epage%3D301%26doi%3D10.1016%2Fj.ymeth.2007.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Greenfield, N. J.</span><span> </span><span class="NLM_article-title">Using circular dichroism collected as a function of temperature to determine the thermodynamics of protein unfolding and binding interactions</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">2527</span><span class="NLM_x">–</span> <span class="NLM_lpage">2535</span><span class="refDoi"> DOI: 10.1038/nprot.2006.204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1038%2Fnprot.2006.204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=2527-2535&author=N.+J.+Greenfield&title=Using+circular+dichroism+collected+as+a+function+of+temperature+to+determine+the+thermodynamics+of+protein+unfolding+and+binding+interactions&doi=10.1038%2Fnprot.2006.204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.204%26sid%3Dliteratum%253Aachs%26aulast%3DGreenfield%26aufirst%3DN.%2BJ.%26atitle%3DUsing%2520circular%2520dichroism%2520collected%2520as%2520a%2520function%2520of%2520temperature%2520to%2520determine%2520the%2520thermodynamics%2520of%2520protein%2520unfolding%2520and%2520binding%2520interactions%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D1%26spage%3D2527%26epage%3D2535%26doi%3D10.1038%2Fnprot.2006.204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Tan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, G.</span><span> </span><span class="NLM_article-title">Electrospray ionization mass spectrometry probing of binding affinity of berbamine, a flexible cyclic alkaloid from traditional Chinese medicine, with G-quadruplex DNA</span> <span class="citation_source-journal">Rapid Commun. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1002/rcm.6763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1002%2Frcm.6763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=24285399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVOisLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=143-147&author=W.+Tanauthor=J.+Zhouauthor=G.+Yuan&title=Electrospray+ionization+mass+spectrometry+probing+of+binding+affinity+of+berbamine%2C+a+flexible+cyclic+alkaloid+from+traditional+Chinese+medicine%2C+with+G-quadruplex+DNA&doi=10.1002%2Frcm.6763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Electrospray ionization mass spectrometry probing of binding affinity of berbamine, a flexible cyclic alkaloid from traditional Chinese medicine, with G-quadruplex DNA</span></div><div class="casAuthors">Tan, Wei; Zhou, Jiang; Yuan, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">143-147</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Classic G-quadruplex binders typically have a large arom. core and interact with G-quadruplexes through π-π stacking with the quartets.  There are rather few reports on natural flexible cyclic mol. from traditional Chinese medicine which has high binding affinity with G-quadruplex.  Electrospray ionization mass spectrometry (ESI-MS) expts. were performed to evaluate the binding affinities of a natural alkaloid, berbamine, with seven G-quadruplexes.  Furthermore, we utilized autodock4 anal. to uncover the binding mode of berbamine with the G-quadruplex.  ESI-MS expts. showed that berbamine has the best binding affinity toward the (GGA)8 G-quadruplex compared with the other six G-quadruplexes.  Autodock4 anal. indicated that berbamine interacted with the (GGA)8 G-quadruplex through hydrogen bonding and van der Waals forces with a binding site at the lateral groove formed by DG8-DA9-DA15-DG16.  In this study, we discovered a novel G-quadruplex binder, berbamine, which has high binding affinity toward the (GGA)8 G-quadruplex.  This study provided important clues regarding the probing of small mol. from traditional Chinese medicine which can target the G-quadruplex with high affinity.  Copyright © 2013 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdYiHudG9GvbVg90H21EOLACvtfcHk0ljQtJOJeR0aCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVOisLjO&md5=54025c5a6d95b8096a8814357b1a95e4</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Frcm.6763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.6763%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DG.%26atitle%3DElectrospray%2520ionization%2520mass%2520spectrometry%2520probing%2520of%2520binding%2520affinity%2520of%2520berbamine%252C%2520a%2520flexible%2520cyclic%2520alkaloid%2520from%2520traditional%2520Chinese%2520medicine%252C%2520with%2520G-quadruplex%2520DNA%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2014%26volume%3D28%26spage%3D143%26epage%3D147%26doi%3D10.1002%2Frcm.6763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Mennucci, B.; Petersson, G. A.; Nakatsuji, H.; Caricato, M.; Li, X.; Hratchian, H. P.; Izmaylov, A. F.; Bloino, J.; Zheng, G.; Sonnenberg, J. L.; Hada, M.; Ehara, M.; Toyota, K.; Fukuda, R.; Hasegawa, J.; Ishida, M.; Nakajima, T.; Honda, Y.; Kitao, O.; Nakai, H.; Vreven, T.; Montgomery, J. A.,  Jr.; Peralta, J. E.; Ogliaro, F.; Bearpark, M.; Heyd, J. J.; Brothers, E.; Kudin, K. N.; Staroverov, V. N.; Keith, T.; Kobayashi, R.; Normand, J.; Raghavachari, K.; Rendell, A.; Burant, J. C.; Iyengar, S. S.; Tomasi, J.; Cossi, M.; Rega, N.; Millam, J. M.; Klene, M.; Knox, J. E.; Cross, J. B.; Bakken, V.; Adamo, C.; Jaramillo, J.; Gomperts, R.; Stratmann, R. E.; Yazyev, O.; Austin, A. J.; Cammi, R.; Pomelli, C.; Ochterski, J. W.; Martin, R. L.; Morokuma, K.; Zakrzewski, V. G.; Voth, G. A.; Salvador, P.; Dannenberg, J. J.; Dapprich, S.; Daniels, A. D.; Farkas, O.; Foresman, J. B.; Ortiz, J. V.; Cioslowski, J.; Fox, D. J.</span>Gaussian 09, <span class="NLM_edition">revision B.01</span>; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+Bearpark&author=J.+J.+Heyd&author=E.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=T.+Keith&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=J.+M.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=O.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional thermochemistry. III. The role of exact exchange</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">5648</span><span class="NLM_x">–</span> <span class="NLM_lpage">5652</span><span class="refDoi"> DOI: 10.1063/1.464913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=10.1063%2F1.464913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;key=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=5648-5652&author=A.+D.+Becke&title=Density-functional+thermochemistry.+III.+The+role+of+exact+exchange&doi=10.1063%2F1.464913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional thermochemistry. III. The role of exact exchange</span></div><div class="casAuthors">Becke, Axel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5648-52</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Despite the remarkable thermochem. accuracy of Kohn-Sham d.-functional theories with gradient corrections for exchange-correlation, the author believes that further improvements are unlikely unless exact-exchange information is considered.  Arguments to support this view are presented, and a semiempirical exchange-correlation functional (contg. local-spin-d., gradient, and exact-exchange terms) is tested for 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total at. energies of first- and second-row systems.  This functional performs better than previous functionals with gradient corrections only, and fits expt. atomization energies with an impressively small av. abs. deviation of 2.4 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3BqpycUTBwLVg90H21EOLACvtfcHk0ljhPo5Z_lRRVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D&md5=291bbfc119095338bb1624f0c21c7ca8</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1063%2F1.464913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464913%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520thermochemistry.%2520III.%2520The%2520role%2520of%2520exact%2520exchange%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D5648%26epage%3D5652%26doi%3D10.1063%2F1.464913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2N4Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2N4Y','PDB','2N4Y'); return false;">PDB: 2N4Y</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i39"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01283">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_57541"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01283">10.1021/acs.jmedchem.5b01283</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures illustrating all oligonucleotides used in this study, CD spectra, and the complete ESI/MS competition analysis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_001.pdf">jm5b01283_si_001.pdf (35.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01283/suppl_file/jm5b01283_si_002.csv">jm5b01283_si_002.csv (2.07 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01283&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01283%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-24%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01283" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998abe895f3dc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
