
<html lang="en"     class="pb-page"  data-request-id="47816252-0444-4898-8421-a1e9bc8dd81a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b01604;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2020.63.issue-1;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Novel Inhibitors of LpxC Displaying Potent in Vitro Activity against Gram-Negative Bacteria" /></meta><meta name="dc.Creator" content="Jean-Philippe  Surivet" /></meta><meta name="dc.Creator" content="Philippe  Panchaud" /></meta><meta name="dc.Creator" content="Jean-Luc  Specklin" /></meta><meta name="dc.Creator" content="Stefan  Diethelm" /></meta><meta name="dc.Creator" content="Anne-Catherine  Blumstein" /></meta><meta name="dc.Creator" content="Jean-Christophe  Gauvin" /></meta><meta name="dc.Creator" content="Loïc  Jacob" /></meta><meta name="dc.Creator" content="Florence  Masse" /></meta><meta name="dc.Creator" content="Gaëlle  Mathieu" /></meta><meta name="dc.Creator" content="Azely  Mirre" /></meta><meta name="dc.Creator" content="Christine  Schmitt" /></meta><meta name="dc.Creator" content="Roland  Lange" /></meta><meta name="dc.Creator" content="Naomi  Tidten-Luksch" /></meta><meta name="dc.Creator" content="Carmela  Gnerre" /></meta><meta name="dc.Creator" content="Swen  Seeland" /></meta><meta name="dc.Creator" content="Charlyse  Herrmann" /></meta><meta name="dc.Creator" content="Peter  Seiler" /></meta><meta name="dc.Creator" content="Michel  Enderlin-Paput" /></meta><meta name="dc.Creator" content="Aengus  Mac Sweeney" /></meta><meta name="dc.Creator" content="Micha  Wicki" /></meta><meta name="dc.Creator" content="Christian  Hubschwerlen" /></meta><meta name="dc.Creator" content="Daniel  Ritz" /></meta><meta name="dc.Creator" content="Georg  Rueedi" /></meta><meta name="dc.Description" content="UDP-3-O-((R)-3-hydroxymyristoyl)-N-glucosamine deacetylase (LpxC) is as an attractive target for the discovery and development of novel antibacterial drugs to address the critical medical need crea..." /></meta><meta name="Description" content="UDP-3-O-((R)-3-hydroxymyristoyl)-N-glucosamine deacetylase (LpxC) is as an attractive target for the discovery and development of novel antibacterial drugs to address the critical medical need crea..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 5, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01604" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01604" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01604" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01604" /></link>
        
    
    

<title>Discovery of Novel Inhibitors of LpxC Displaying Potent in Vitro Activity against Gram-Negative Bacteria | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01604" /></meta><meta property="og:title" content="Discovery of Novel Inhibitors of LpxC Displaying Potent in Vitro Activity against Gram-Negative Bacteria" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0020.jpeg" /></meta><meta property="og:description" content="UDP-3-O-((R)-3-hydroxymyristoyl)-N-glucosamine deacetylase (LpxC) is as an attractive target for the discovery and development of novel antibacterial drugs to address the critical medical need created by multidrug resistant Gram-negative bacteria. By using a scaffold hopping approach on a known family of methylsulfone hydroxamate LpxC inhibitors, several hit series eliciting potent antibacterial activities against Enterobacteriaceae and Pseudomonas aeruginosa were identified. Subsequent hit-to-lead optimization, using cocrystal structures of inhibitors bound to Pseudomonas aeruginosa LpxC as guides, resulted in the discovery of multiple chemical series based on (i) isoindolin-1-ones, (ii) 4,5-dihydro-6H-thieno[2,3-c]pyrrol-6-ones, and (iii) 1,2-dihydro-3H-pyrrolo[1,2-c]imidazole-3-ones. Synthetic methods, antibacterial activities and relative binding affinities, as well as physicochemical properties that allowed compound prioritization are presented. Finally, in vivo properties of lead molecules which belong to the most promising pyrrolo-imidazolone series, such as 18d, are discussed." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01604"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01604">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01604&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01604&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01604&amp;href=/doi/10.1021/acs.jmedchem.9b01604" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 66-87</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01180" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01605" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Novel Inhibitors of LpxC Displaying Potent in Vitro Activity against Gram-Negative Bacteria</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jean-Philippe Surivet</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jean-Philippe Surivet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +41 61 588 02 77. E-mail: <a href="/cdn-cgi/l/email-protection#bfd5daded192cfd7d6d3d6cfcfda91cccacdd6c9dacbffd6dbd0cdccd6de91dcd0d2"><span class="__cf_email__" data-cfemail="adc7c8ccc380ddc5c4c1c4ddddc883ded8dfc4dbc8d9edc4c9c2dfdec4cc83cec2c0">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jean-Philippe++Surivet">Jean-Philippe Surivet</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3505-2025" title="Orcid link">http://orcid.org/0000-0002-3505-2025</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philippe Panchaud</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philippe Panchaud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philippe++Panchaud">Philippe Panchaud</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7823-7639" title="Orcid link">http://orcid.org/0000-0001-7823-7639</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jean-Luc Specklin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jean-Luc Specklin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jean-Luc++Specklin">Jean-Luc Specklin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Diethelm</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Diethelm</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Diethelm">Stefan Diethelm</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne-Catherine Blumstein</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Catherine Blumstein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Catherine++Blumstein">Anne-Catherine Blumstein</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jean-Christophe Gauvin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jean-Christophe Gauvin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jean-Christophe++Gauvin">Jean-Christophe Gauvin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Loïc Jacob</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Loïc Jacob</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lo%C3%AFc++Jacob">Loïc Jacob</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Florence Masse</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Florence Masse</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Florence++Masse">Florence Masse</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gaëlle Mathieu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gaëlle Mathieu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ga%C3%ABlle++Mathieu">Gaëlle Mathieu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Azely Mirre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Azely Mirre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Azely++Mirre">Azely Mirre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christine Schmitt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine Schmitt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Schmitt">Christine Schmitt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roland Lange</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roland Lange</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roland++Lange">Roland Lange</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Naomi Tidten-Luksch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Naomi Tidten-Luksch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Naomi++Tidten-Luksch">Naomi Tidten-Luksch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carmela Gnerre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carmela Gnerre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carmela++Gnerre">Carmela Gnerre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Swen Seeland</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Swen Seeland</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Swen++Seeland">Swen Seeland</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charlyse Herrmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charlyse Herrmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charlyse++Herrmann">Charlyse Herrmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Seiler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Seiler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Seiler">Peter Seiler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michel Enderlin-Paput</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michel Enderlin-Paput</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michel++Enderlin-Paput">Michel Enderlin-Paput</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aengus Mac Sweeney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aengus Mac Sweeney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aengus++Mac+Sweeney">Aengus Mac Sweeney</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7250-9541" title="Orcid link">http://orcid.org/0000-0001-7250-9541</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Micha Wicki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Micha Wicki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Micha++Wicki">Micha Wicki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Hubschwerlen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Hubschwerlen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Hubschwerlen">Christian Hubschwerlen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel Ritz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Ritz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Ritz">Daniel Ritz</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Georg Rueedi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Georg Rueedi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Idorsia Pharmaceuticals Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Georg++Rueedi">Georg Rueedi</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01604&amp;href=/doi/10.1021%2Facs.jmedchem.9b01604" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 66–87</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 5, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 September 2019</li><li><span class="item_label"><b>Published</b> online</span>5 December 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 January 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01604" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01604</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D66%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJean-Philippe%2BSurivet%252C%2BPhilippe%2BPanchaud%252C%2BJean-Luc%2BSpecklin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D1%26contentID%3Dacs.jmedchem.9b01604%26title%3DDiscovery%2Bof%2BNovel%2BInhibitors%2Bof%2BLpxC%2BDisplaying%2BPotent%2Bin%2BVitro%2BActivity%2Bagainst%2BGram-Negative%2BBacteria%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D87%26publicationDate%3DJanuary%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01604"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2350</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01604" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Novel Inhibitors of LpxC Displaying Potent in Vitro Activity against Gram-Negative Bacteria&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jean-Philippe&quot;,&quot;last_name&quot;:&quot;Surivet&quot;},{&quot;first_name&quot;:&quot;Philippe&quot;,&quot;last_name&quot;:&quot;Panchaud&quot;},{&quot;first_name&quot;:&quot;Jean-Luc&quot;,&quot;last_name&quot;:&quot;Specklin&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Diethelm&quot;},{&quot;first_name&quot;:&quot;Anne-Catherine&quot;,&quot;last_name&quot;:&quot;Blumstein&quot;},{&quot;first_name&quot;:&quot;Jean-Christophe&quot;,&quot;last_name&quot;:&quot;Gauvin&quot;},{&quot;first_name&quot;:&quot;Loïc&quot;,&quot;last_name&quot;:&quot;Jacob&quot;},{&quot;first_name&quot;:&quot;Florence&quot;,&quot;last_name&quot;:&quot;Masse&quot;},{&quot;first_name&quot;:&quot;Gaëlle&quot;,&quot;last_name&quot;:&quot;Mathieu&quot;},{&quot;first_name&quot;:&quot;Azely&quot;,&quot;last_name&quot;:&quot;Mirre&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Schmitt&quot;},{&quot;first_name&quot;:&quot;Roland&quot;,&quot;last_name&quot;:&quot;Lange&quot;},{&quot;first_name&quot;:&quot;Naomi&quot;,&quot;last_name&quot;:&quot;Tidten-Luksch&quot;},{&quot;first_name&quot;:&quot;Carmela&quot;,&quot;last_name&quot;:&quot;Gnerre&quot;},{&quot;first_name&quot;:&quot;Swen&quot;,&quot;last_name&quot;:&quot;Seeland&quot;},{&quot;first_name&quot;:&quot;Charlyse&quot;,&quot;last_name&quot;:&quot;Herrmann&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Seiler&quot;},{&quot;first_name&quot;:&quot;Michel&quot;,&quot;last_name&quot;:&quot;Enderlin-Paput&quot;},{&quot;first_name&quot;:&quot;Aengus&quot;,&quot;last_name&quot;:&quot;Mac Sweeney&quot;},{&quot;first_name&quot;:&quot;Micha&quot;,&quot;last_name&quot;:&quot;Wicki&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Hubschwerlen&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Ritz&quot;},{&quot;first_name&quot;:&quot;Georg&quot;,&quot;last_name&quot;:&quot;Rueedi&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;66-87&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01604&quot;},&quot;abstract&quot;:&quot;UDP-3-O-((R)-3-hydroxymyristoyl)-N-glucosamine deacetylase (LpxC) is as an attractive target for the discovery and development of novel antibacterial drugs to address the critical medical need created by multidrug resistant Gram-negative bacteria. By using a scaffold hopping approach on a known family of methylsulfone hydroxamate LpxC inhibitors, several hit series eliciting potent antibacterial activities against Enterobacteriaceae and Pseudomonas aeruginosa were identified. Subsequent hit-to-lead optimization, using cocrystal structures of inhibitors bound to Pseudomonas aeruginosa LpxC as guides, resulted in the discovery of multiple chemical series based on (i) isoindolin-1-ones, (ii) 4,5-dihydro-6H-thieno[2,3-c]pyrrol-6-ones, and (iii) 1,2-dihydro-3H-pyrrolo[1,2-c]imidazole-3-ones. Synthetic methods, antibacterial activities and relative binding affinities, as well as physicochemical properties that allowed compound prioritization are presented. Finally, in vivo properties of lead molecules which belong &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01604&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01604" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01604&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01604" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01604&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01604" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01604&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01604&amp;href=/doi/10.1021/acs.jmedchem.9b01604" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01604" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01604" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01604%26sid%3Dliteratum%253Aachs%26pmid%3D31804826%26genre%3Darticle%26aulast%3DSurivet%26date%3D2020%26atitle%3DDiscovery%2Bof%2BNovel%2BInhibitors%2Bof%2BLpxC%2BDisplaying%2BPotent%2Bin%2BVitro%2BActivity%2Bagainst%2BGram-Negative%2BBacteria%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D1%26spage%3D66%26epage%3D87%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291676" title="Antibacterial activity">Antibacterial activity</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/jmcmar.2020.63.issue-1/20200109/jmcmar.2020.63.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">UDP-3-<i>O</i>-((<i>R</i>)-3-hydroxymyristoyl)-<i>N</i>-glucosamine deacetylase (LpxC) is as an attractive target for the discovery and development of novel antibacterial drugs to address the critical medical need created by multidrug resistant Gram-negative bacteria. By using a scaffold hopping approach on a known family of methylsulfone hydroxamate LpxC inhibitors, several hit series eliciting potent antibacterial activities against Enterobacteriaceae and <i>Pseudomonas aeruginosa</i> were identified. Subsequent hit-to-lead optimization, using cocrystal structures of inhibitors bound to <i>Pseudomonas aeruginosa</i> LpxC as guides, resulted in the discovery of multiple chemical series based on (i) isoindolin-1-ones, (ii) 4,5-dihydro-6<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-6-ones, and (iii) 1,2-dihydro-3<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazole-3-ones. Synthetic methods, antibacterial activities and relative binding affinities, as well as physicochemical properties that allowed compound prioritization are presented. Finally, in vivo properties of lead molecules which belong to the most promising pyrrolo-imidazolone series, such as <b>18d</b>, are discussed.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16818" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16818" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The World Health Organization (WHO) classifies antimicrobial resistance (AMR) as one of the biggest threats to global health, acknowledging that AMR steadily erodes the power of antibiotherapy used to cure life-threatening infections.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> WHO has established a global priority pathogens list where multidrug- and extensively drug-resistant (MDR and XDR) <i>Acinetobacter</i>, <i>Pseudomonas aeruginosa</i> (Pae), Enterobacteriaceae strains (including <i>Klebsiella pneumoniae</i> (Kpn), <i>Escherichia coli</i> (Eco), <i>Serratia</i>, and <i>Proteus</i>) ranked high.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> These Gram-negative bacteria (GNB) are not only refractory to recommended empiric treatment options (carbapenems and third-generation cephalosporins) but are also becoming increasingly resistant to some of the last resort drugs such as colistin,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> creating a critical medical need for novel, effective, and safe antibacterial agents. Public health experts as well as authorities have recognized the urgency of the situation and economic incentives have been awarded to boost innovation.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">The success of incremental improvement of old antibacterial classes will soon come to an end as it becomes increasingly difficult to circumvent cross-resistance issues. The level of innovation in the field of anti-Gram-negative bacteria drugs has been low, and this can mainly be attributed to the scarcity of drug targets, novel or validated. Expanding the spectrum of activity of drugs effective against Gram-positive bacteria to include GNB has therefore recently gained a lot of attention.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In this context, efforts to identify rationally the processes that drive cytoplasmic drug accumulation in Gram-negative bacteria have been reported.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Over the past decade, only a handful of novel targets amenable to the treatment of Gram-negative infections have been studied, resulting in only a few chemical entities suitable for entry-into-man.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Among them, LpxC is a biological target that holds much promise and has captured the interest of many biopharmaceutical companies, leading to the first phase I trial of Achaogen’s ACHN-975 <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Motivated by these successes, we sought to develop an independent program to identify a novel class of LpxC inhibitors.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. LpxC, ubiquitously found in GNB, catalyzes the irreversible deacetylation of UDP-(3-<i>O</i>-((<i>R</i>)-3-hydroxymyristoyl)-<i>N</i>-acetylglucosamine. Representative inhibitors of LpxC feature a hydroxamate as a zinc-binding group (box).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">LpxC is a cytosolic metal-dependent enzyme found only in Gram-negative bacteria. It catalyzes the first committed step in the biosynthesis of lipid A, a key component of the Gram-negative outer membrane. Utilizing a single active-site metal ion, LpxC hydrolyses the 2-acetamide of UDP-(3-<i>O</i>-((<i>R</i>)-3-hydroxymyristoyl)-<i>N</i>-acetylglucosamine to produce acetate and UDP-(3-<i>O</i>-((<i>R</i>)-3-hydroxymyristoyl)-glucosamine (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a></div><div class="NLM_p">Extensive X-ray crystallographic studies of LpxC homologues from several bacterial species revealed a unique β–α–α–β sandwich fold along with a unique metal (Zn<sup>2+</sup> or Fe<sup>2+</sup>)-binding motif defining the catalytic site of the enzyme. These structural investigations further provided detailed information on the binding mode of the deacetylation reaction product, as well as small molecule inhibitors and other ligands.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The key attributes that make LpxC an attractive drug target are its essentiality for bacterial survival, its gene conservation across a spectrum of bacterial pathogens, the absence of a human homologue and the availability of specific antibacterial inhibitors with proven efficacy in animal models of infection. Inhibition of LpxC by small molecules represents a new mode of antibacterial action with low chance of being compromised by pre-existing target-specific mechanisms of resistance.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">Since the groundbreaking discovery at Merck that L-573,655 <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) exerted antibacterial activity against Eco by inhibiting lipid A biosynthesis,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> all reported inhibitors of LpxC that exhibited potent antibacterial activity against Enterobacteriaceae and Pae have shared two structural elements: (1) a hydroxamic acid moiety as metal binding unit<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> and (2) a linear and rigid lipophilic tail that interacts with the hydrophobic tunnel otherwise occupied by the fatty acyl chain of the natural substrate (hereafter dubbed substrate-binding passage or passage).<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> Families of LpxC inhibitors differ from one another mainly in the nature of the chemical scaffold that links those two recurrent elements. At the time when our research program was initiated, optimization efforts on threonyl-hydroxamate based LpxC inhibitors deriving from CHIR-090 <b>3</b> had culminated in the identification of phase I candidate <b>1</b> at Achaogen and lead compound <b>4</b> at Novartis,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> offering only limited opportunity for a fast-follower approach. However, the concurrent disclosure of the structurally novel methylsulfone hydroxamate derivatives such as <b>5</b> and <b>6</b> with potent antibacterial activities,<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> made this motif an excellent starting point for our investigations. Herein, we report the scaffold hopping approach that eventually led to the identification of novel methylsulfone hydroxamate LpxC inhibitors articulated around isoindolin-1-one (core I),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> 1,2-dihydro-3<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazole-3-one (core II)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and 4,5-dihydro-6<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-6-one<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (core III) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Synthesis, crystallographic, biological, and pharmacological data of compounds generated during the subsequent hit-to-lead optimization program are reported, with a focus on derivatives of the pyrrolo[1,2-<i>c</i>]imidazole-3-one series (core II).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Core structures of novel methylsulfone hydroxamate LpxC inhibitors based on bicyclic heteroaromatic systems.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41772" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41772" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Experimental procedures leading to compounds <b>8</b>–<b>14</b> are reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Supporting Information (SI)</a>. All LpxC inhibitors <b>15</b>–<b>20</b> featuring either phenyl or alkyne-based side chains were prepared from terminal alkynes <b>29</b>, <b>34</b>, and <b>40</b>.</div><div class="NLM_p">As described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, the preparation of isoindolin-1-one alkyne <b>29</b> began with a reductive amination involving known aldehyde <b>21</b> and chiral amine <b>22</b> of (<i>R</i>) absolute stereochemistry (for preparation of amine <b>22</b>, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">SI</a>), which after spontaneous lactamization afforded bromo-isoindolinone <b>23</b>. A bromine–iodine exchange was performed using NaI in the presence of a copper catalyst (obtained by mixing CuI and a <i>trans</i>-<i>N</i>,<i>N</i>′-dimethylcyclohexane-1,2-diamine) to afford iodo-derivative <b>24</b>. Acidic hydrolysis of <i>t</i>-butyl ester <b>24</b> using trifluoroacetic acid (TFA) in the presence of triethylsilane yielded the corresponding free acid <b>25</b>, which was further converted to the THP-protected hydroxamic acid <b>26</b> by reaction with <i>O</i>-(tetrahydro-<i>2H</i>-pyran-2-yl)hydroxylamine (NH<sub>2</sub>OTHP) in the presence of 1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide (EDC), 1-hydroxybenzotriazole (HOBt), and triethylamine (TEA). Subsequent Sonogashira coupling<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> performed with trimethylsilylacetylene <b>27</b> in the presence of PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> and CuI afforded intermediate <b>28</b> as an equimolar mixture of THP anomers. Treatment with potassium carbonate in methanol (MeOH) of TMS-protected <b>28</b> gave rise to terminal alkyne <b>29</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0011.jpeg" id="GRAPHIC-d7e643-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 4 h; (b) NaI, CuI, <i>trans</i>-<i>N</i>,<i>N</i>′-dimethylcyclohexane-1,2-diamine, dioxane, reflux, 3 h; (c) TFA, Et<sub>3</sub>SiH, rt, 16 h; (d) NH<sub>2</sub>OTHP, EDC·HCl, HOBt, TEA, DMF, rt, 16 h; (e) <b>27</b>, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, reflux, 1 h; (f) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt, 1 h; (g) PPTS, EtOH, 70 °C.</p></p></figure><div class="NLM_p">As described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, the synthesis of pyrrolo-imidazolone intermediate <b>34</b> was realized from commercially available pyrrole-aldehyde <b>30</b>. A reductive amination between <b>30</b> and chiral amine <b>22</b> gave rise to an alkylated amine <b>31</b>, which upon treatment with carbonyl-diimidazole (CDI) in the presence of sodium hydride afforded the bicyclic compound <b>32</b>. The <i>t</i>-butyl ester was then transformed to the intermediate TMS–alkyne <b>33</b> via a hydrolysis-amidation-Sonogashira coupling sequence using methods described previously in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, except that propylphosphonic anhydride (T3P) and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) were used as amide coupling reagents. From intermediate <b>33</b>, TMS-cleavage using tetrabutylammonium fluoride (TBAF) in tetrahydrofuran (THF) afforded the terminal alkyne <b>34</b> as an equimolar mixture of THP anomers.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0012.jpeg" id="GRAPHIC-d7e742-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>22</b>, 3 Å molecular sieves, DCM, rt then NaBH<sub>4</sub>, MeOH, 0 °C; (b) CDI, NaH, THF, 0 °C; (c) TFA, Et<sub>3</sub>SiH, DCM, 0 °C to rt, 5 h; (d) THPONH<sub>2</sub>, T<sub>3</sub>P, DIPEA, THF, rt, 2 h; (e) TMS-CCH, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, rt, 1 h; (f) TBAF, THF, rt.</p></p></figure><div class="NLM_p">The synthesis of thienopyrrolone precursor <b>40</b>, described in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, started with a cyclization process that took place between commercially available dialdehyde <b>35</b> and chiral amine <b>22</b> in acetic acid, affording thienopyrrolone <b>36</b> as the sole product. Subsequent regioselective bromination using bromine in aqueous acetic acid gave rise to bromo-intermediate <b>37</b>. After <i>t</i>-butyl ester cleavage under acid conditions, the free acid <b>38</b> was coupled with NH<sub>2</sub>OTHP and the resulting THP-protected hydroxamic acid was converted to TMS–alkyne <b>39</b>. Both transformations employed methods similarly to those reported for the preparation of alkyne <b>29</b>. The TMS-group of intermediate <b>39</b> was cleaved using TBAF, leading to terminal alkyne <b>40</b>, recovered as an equimolar mixture of THP anomers.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0013.jpeg" id="GRAPHIC-d7e824-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>22</b>, AcOH, DCM, rt, 4 h; (b) Br<sub>2</sub>, aq AcOH, rt, 16 h; (c) 4 N HCl, dioxane, rt, 16 h; (d) THPONH<sub>2</sub>, EDC, HOBt, TEA, DMF, rt, 16 h; (e) TMS-CCH, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, rt, 1 h; (f) TBAF, THF, rt.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, the synthesis of hydroxamic acids <b>15a</b>,<b>b</b>, <b>17a</b>–<b>j</b>, and <b>19a</b>–<b>d</b> featuring a phenyl-based side chains were generally performed via a Sonogashira cross-coupling reaction involving one of the terminal alkynes <b>29</b>, <b>34</b>, or <b>40</b> and a 4-iodo-substituted phenyl derivative <b>41</b>–<b>50</b>. Standard coupling conditions (vide supra) were employed. Subsequently, a final THP-deprotection using suitable acidic conditions afforded hydroxamic acids <b>15a</b>,<b>b</b>, <b>17a</b>–<b>j</b>, and <b>19a</b>–<b>d</b>, respectively. The synthesis of hydroxamic acids <b>16a</b>,<b>b</b>, <b>18a</b>–<b>j</b>, and <b>20a</b>–<b>f</b> featuring acetylene-based side chains was achieved via a Cadiot–Chodkiewicz coupling reaction<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> between one of the terminal alkynes <b>29</b>, <b>34</b>, or <b>40</b> and halo-alkynes <b>51</b>–<b>60</b>. Reaction conditions utilizing copper chloride as a catalyst and hydroxylamine as a base were found suitable for all coupling experiments. THP-deprotection under acidic conditions afforded hydroxamic acids <b>16a</b>,<b>b</b>, <b>18a</b>–<b>j</b>, and <b>20a</b>–<b>f</b>. Depending on the nature of the group R<sup>2</sup> present in coupling partners <b>51</b>–<b>60</b>, an intermediate deprotection step (TBAF in THF when a silyl protecting group is present or potassium carbonate in MeOH in the case of an ester) was required prior to the acid-mediated THP-deprotection. The detailed synthesis of iodophenyl derivatives <b>41</b>–<b>50</b> and halo alkynes <b>51</b>–<b>60</b> are available in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">SI</a>.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0014.jpeg" id="GRAPHIC-d7e997-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>41</b>–<b>50</b>, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, reflux; (b) method A, PPTS, EtOH, 70 °C; method B, PPTS, aq dioxane, 70 °C; method C, aq HCl, MeCN, rt; (c) <b>51</b>–<b>60</b>, CuCl, NH<sub>2</sub>OH·HCl, BuNH<sub>2</sub>, rt; (d) TBAF, THF, rt; (e) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58315" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58315" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Scaffold Hopping</h3><div class="NLM_p">With the methylsulfone hydroxamate inhibitor family as a starting point for scaffold hopping, we analyzed the published X-ray cocrystal structures of Pfizer’s inhibitors <b>5</b> and <b>6</b> in complex with Pae LpxC (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U1Y">3U1Y</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UHM">3UHM</a>). While the methylsulfone hydroxamate headgroups of <b>5</b> and <b>6</b> establish crucial H-bonding interactions with the catalytic metal and its immediate environment, the aromatic components, i.e., phenyl or pyridone cores of these inhibitors, together with their respective side lipophilic tails appeared to be only involved in van der Waals interactions with the LpxC protein. Furthermore, the central aromatic cores, seated at the entrance of the substrate-binding passage, played an essential role in maintaining lipophilic side chains at the right distance from the hydroxamate anchor and in directing these tails toward the hydrophobic passage. Therefore, the search for replacements of the central scaffold was initiated using pyridone <b>7</b> as a starting point.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Three design criteria had to be fulfilled. A novel core scaffold had to (1) be planar, (2) provide two exit vectors projecting in opposite direction, and (3) be structurally dissimilar from the pyridone ring. The oxazol-2-one ring met these constraints (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), although both exit vectors around the five-membered core formed a marked obtuse angle (∼139°), in clear contrast to the one observed in the original six-membered pyridone ring (∼180°). As a side aromatic ring, we chose the 2-fluoro-4-methoxyphenyl residue, which had previously provided superior results in the 2-pyridone series. (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The absolute stereochemistry of the chiral methylsulfone hydroxamate headgroup was set to (<i>R</i>), following Pfizer’s findings.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Scaffold hopping approach from pyridone <b>7</b> leading to novel oxazol-2-one <b>8</b>. The numbers indicated below the dash-lined boxes represent the angle at which both exit vectors of the core intersect.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Oxazol-2-one (<i>R</i>)-<b>8</b> and (<i>S</i>)-<b>8</b> were prepared as reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">SI</a>. The in vitro antibacterial activity of compounds (<i>R</i>)-<b>8</b> and (<i>S</i>)-<b>8</b> were determined against a panel of clinically relevant Gram-negative pathogens including wild-type Eco ATCC 25922 and Pae ATCC 27853 strains, and Kpn T 6474, an MDR strain.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Wild-type (wt) Pae PAO1 and an isogenic efflux mutant (PA2616, Δ(<i>mexAB</i>-<i>OprM</i>)), were also employed to assess the impact of compound efflux processes on the antibacterial activity. Binding affinities to Eco and Pae LpxC enzymes were estimated by differential scanning fluorimetry (DSF) and are expressed as thermal shifts (Δ<i>T</i><sub>m</sub> in K).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">As indicated in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, MICs (minimal inhibitory concentration(s), [μg/mL]) of both enantiomers of oxazol-2-one <b>8</b> measured against wild-type Gram-negative bacterial strains were significantly apart, allowing to assign the absolute configuration of the most potent one to (<i>R</i>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Nonetheless, (<i>R</i>)-<b>8</b> elicited moderate to weak MICs (≥8 μg/mL), much higher than that of pyridone <b>7</b>. Binding affinities of compound (<i>R</i>)-<b>8</b> to Eco and Pae LpxC, determined at 25 μM compound concentration, were lower than those of <b>7</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), coinciding well with the observed differences in antibacterial activity. Compounds <b>7</b> and (<i>R</i>)-<b>8</b> were more potent against the efflux-defective Pae strain PA2616 than against its wild-type (wt) PAO1 parent. MIC PAO1/MIC PA2616 ratios (henceforth MIC efflux shifts) calculated for compounds <b>7</b> and (<i>R</i>)-<b>8</b> were found to be ≥8-fold, suggesting that both compounds were efflux pump substrates. Despite its moderate binding affinity to Pae LpxC, an X-ray cocrystal structure of oxazol-2-one <b>8</b> bound to Pae LpxC could be obtained (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Antibacterial Activity against Three Gram-Negative Pathogens and Binding Affinity for Eco and Pae LpxC Enzymes of Compounds <b>7</b>, (<i>R</i>)-<b>8</b> and (<i>S</i>)-<b>8</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" align="center">Compds</th><th class="rowsep1 colsep0" colspan="6" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center">Δ<i>T</i><i><sub>m</sub></i> (K)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Kpn</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center">PAO1</th><th class="colsep0 rowsep0" align="center">PA2616</th><th class="colsep0 rowsep0" align="center">Shift</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Pae</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">≥8</td><td class="colsep0 rowsep0" align="center">7.8</td><td class="colsep0 rowsep0" align="center">11.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>R</i>)-<b>8</b></td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">16</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">5.2</td><td class="colsep0 rowsep0" align="center">9.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(<i>S</i>)-<b>8</b></td><td class="colsep0 rowsep0" align="center">>32</td><td class="colsep0 rowsep0" align="center">>32</td><td class="colsep0 rowsep0" align="center">>32</td><td class="colsep0 rowsep0" align="center">>32</td><td class="colsep0 rowsep0" align="center">16</td><td class="colsep0 rowsep0" align="center">NM</td><td class="colsep0 rowsep0" align="center">1.9</td><td class="colsep0 rowsep0" align="center">3.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Eco ATCC 25922 (wt); Kpn T 6474 (MDR); Pae ATCC 27853 (wt); Pae PAO1 (wt); Pae PA2616, laboratory Δ(<i>mexAB</i>-OprM) mutant with PAO1 background; MIC efflux shift: calculated as the ratio of MIC PAO1/MIC PA2616.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">DSF on isolated enzyme, determined at 25 μM compound concentration. NM, not meaningful.</p></div></div></div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Oxazol-2-one <b>8</b> (white carbon atoms) bound in the active site of Pae LpxC (gray ribbon) (1.9 Å) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I46">6I46</a>). Key interactions of oxazol-2-one <b>8</b> with metal (blue sphere) and key amino acid side chains (gray carbon atoms) are displayed as dotted lines. Electron density of the ligand is shown as a blue mesh (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> density contoured at 1σ). While <i>rac</i>-<b>8</b> was used for cocrystallization experiments with Pae LpxC, only active enantiomer (<i>R</i>)-<b>8</b> was observed.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The methylsulfone hydroxamate headgroup of oxazol-2-one (<i>R</i>)-<b>8</b> is engaged in several metal- or hydrogen-bonding interactions, either with the Zn atom or amino acid side chains (Glu77, Thr190, Lys238, and His264), similar to those previously reported for methylsulfone hydroxamate inhibitors <b>5</b> and <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The sulfone methyl was positioned within a small hydrophobic pocket, pointing toward the phenyl ring of Phe191. As observed in case of the pyridone core, the oxazol-2-one oxygen pointed toward the side chain of Met62. The aromatic tail projected into the substrate-binding passage, i.e., the inner open space of the LpxC insert II motif.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The side aromatic ring and the oxazol-2-one were arranged in a nearly coplanar fashion, forming a dihedral angle of around −10° (37° in case of pyridone <b>6</b>). The low affinity of inhibitor (<i>R</i>)-<b>8</b> could be explained by the smaller angle at which both exit vectors present on its oxazol-2-one core intersect compared to the one of the pyridone ring. Moreover, the comparison between the X-ray structure of pyridone <b>7</b> bound to Pae LpxC generated with the one of compound (<i>R</i>)-<b>8</b>, showed that the inset II α-helix moved by 2.2 Å toward pyridone <b>7</b>, probably leading to its tighter fit in LpxC compared to oxazol-2-one (<i>R</i>)-<b>8</b>. Concomitant with the characterization of oxazol-2-one (<i>R</i>)-<b>8</b>, several heterocyclic methylsulfone hydroxamate LpxC inhibitors with promising antibacterial activities were published.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The scaffold hopping approach thus shifted toward the exploration of bicyclic ring systems as core motifs to maximally diverge from those five-membered heterocyclic templates while taking advantage of our preliminary work. The expectation was that larger bicyclic ring systems would better occupy the vacant space within the substrate-binding passage, hence be less prone to thermal motion and therefore more potent at inhibiting LpxC.</div><div class="NLM_p">From compound (<i>R</i>)-<b>8</b>, different types of bicyclic cores were designed by fusing one additional aromatic ring to the oxazol-2-one motif. The carbonyl group was found beneficial to attain drug-like properties,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> leading to the exploration of either benzoxazolone rings (path <i>a</i>) or isoindolin-1-one-type rings (path <i>b</i>), as depicted in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Paths followed to generate novel bicyclic cores from oxazol-2-one (<i>R</i>)-<b>8</b>. Side aromatic ring (Ar) remained constant (Ar = 2-fluoro-4-methoxyphenyl). The numbers indicated below the dash-lined boxes represent the angle at which both exit vectors of the core intersect. The orientation of Ar relative to the central core was not considered.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The design was further refined by integrating the hypothesis that both exit vectors of the central core intersect with an angle close to 180° (as for pyridones <b>6</b> and <b>7</b>). In the case of benzoxazolone and isoindolin-1-one rings, both exit vectors intersect with angles that are markedly obtuse (around 140°–150°). To restore an overall linear geometry, the phenyl ring was replaced by a five-membered aromatic ring, giving rise to substituted 4,5-dihydro-6<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-6-one (hereafter dubbed thienopyrrolones), 1,2-dihydro-3<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-3-one (or pyrrolo-imidazolones, X = CH), and 6,7-dihydro-5<i>H</i>-imidazo[1,5-<i>a</i>]imidazol-5-one ring systems (or imidazo–imidazolones, X = N), for which the vectors form an angle closer to 180° (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><div class="NLM_p">Seven representative derivatives featuring these designed cores were prepared as reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">SI</a>. Isoindolin-1-one <b>10</b>,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> pyrrolo-imidazolone <b>12</b>,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and thienopyrrolone <b>14</b><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> exhibited potent antibacterial activities against Eco and Kpn strains (MICs ≤ 0.25 μg/mL) and moderate activity against Pae strains with MICs ≤ 2 μg/mL (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Binding affinities of compounds <b>10</b>, <b>12</b>, and <b>14</b>, determined by DSF (Δ<i>T</i><sub>m</sub> recorded on isolated enzyme at 25 μM compound concentration), were comparable on Eco (Δ<i>T</i><sub>m</sub> = 8–9 K) and Pae (Δ<i>T</i><sub>m</sub> = 11–12 K) enzymes and comparable with Δ<i>T</i><sub>m</sub> values obtained with pyridone <b>7</b>. MIC efflux shifts determined for these three derivatives were about 4–8-fold, similarly to that of pyridone <b>7</b>. These efflux shifts indicated that these LpxC inhibitors were to some extent substrates of the Pae MexAB or OprM-dependent efflux systems. Benzoxazolone <b>9</b><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> exhibited weaker antibacterial activity against Gram-negative strains than derivatives <b>10</b>, <b>12</b>, and <b>14</b>, although its binding affinities to both Eco and Pae LpxC (Δ<i>T</i><sub>m</sub> = 8.2 and 11.0 K, respectively) were comparable. The large MIC efflux shift of benzoxazolone <b>9</b> (≥64-fold) suggested that the weaker antibacterial activity against Pae resulted from extensive efflux. Derivatives <b>11</b> and <b>13</b> were inactive against Kpn and Pae strains, and their MICs against Eco were at least 32-fold higher than those of their direct analogues <b>10</b> and <b>12</b>. The poor antibacterial profile of compounds <b>11</b> and <b>13</b> could be explained by their lower binding affinities to Eco (Δ<i>T</i><sub>m</sub> = 5–6 K) and Pae (Δ<i>T</i><sub>m</sub> ∼ 7 K) enzymes, suggesting a negative effect of additional nitrogen atoms present in the annulated aromatic ring. For compounds displaying MICs ≤ 4 μg/mL against Pae, solubility at pH 7 and distribution coefficient (log <i>D</i><sub>7.4</sub>) were determined. Compound <b>12</b> appeared to be less soluble at pH 7 and more lipophilic than pyridone <b>7</b> (log <i>D</i><sub>7.4</sub> 1.8 and 1.1 for compounds <b>12</b> and <b>7</b>, respectively). Compounds <b>10</b> and <b>14</b> were poorly soluble at pH 7 (∼5 μg/mL) and displayed high lipophilicity (log <i>D</i><sub>7.4</sub> ∼2) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). At pH 9, all compounds exhibited higher solubilities as the hydroxamate moiety is predominantly ionized (measured p<i>K</i><sub>a</sub>s of hydroxamic acids usually range between 7.3 and 7.7).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> However, compound <b>12</b> remained less soluble than pyridone <b>7</b> as this pH.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antibacterial Activities, Binding Affinities, and Physicochemical Properties of Compounds <b>7</b>–<b>14</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0015.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0016.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Eco ATCC 25922 (wt); Kpn T 6474 (MDR); Pae ATCC 27853 (wt); Pae PAO1 (wt); Pae PA2616, laboratory Δ(<i>mexAB</i>-OprM) mutant with PAO1 background; MIC efflux shift: calculated as the ratio of MIC PAO1/MIC PA2616.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">DSF on isolated enzyme, determined at 25 μM compound concentration.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Tested as a racemate. ND, not determined; NM, not meaningful</p></div></div><div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Hit-to-Lead Optimization</h3><div class="NLM_p">The next stages toward identifying an LpxC inhibitor suitable for the treatment of GNB infections consisted of chemical exploration of the hits generated by scaffold hopping (i.e., hits <b>10</b>, <b>12</b>, and <b>14</b>) and then selection of the most suitable series for further lead optimization. Antibacterial potency and spectrum, as well as aqueous solubility, were selected as main parameters to guide the hit-to-lead optimization.</div><div class="NLM_p">Upon examination of the cocrystal structures of isoindolin-1-one <b>10</b> and pyrrolo-imidazolone <b>12</b> (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Figures S-6 and S-7</a>, respectively) bound to Pae LpxC,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> identical electrostatic interactions of the methylsulfone hydroxamate headgroup of both entities to the catalytic site were found to those previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> As expected, both bicyclic cores of compounds <b>10</b> and <b>12</b> are located at the entrance of the hydrophobic substrate-binding passage, projecting the side aromatic rings into the space defined by LpxC insert II. The superimposition of LpxC structures bound to oxazol-2-one (<i>R</i>)-<b>8</b> (binding affinity to Pae LpxC Δ<i>T</i><sub>m</sub>: 9.9 K) and <b>10</b> (Δ<i>T</i><sub>m</sub>: 11.6 K) showed that the tighter complex formed with isoindolin-1-one <b>10</b> arose mainly from stronger interactions between the α-helix of insert II and the inhibitor (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The entire α-helix was pulled closer toward the inhibitor by 1.9 Å (distance measured between Cα atoms of Ile197, relatively to an all-Cα-fit of both structures with a Cα-RMSD of 0.239 Å) and therefore closed the hydrophobic tunnel to a tighter fit around the molecule compared to what had been observed for compound (<i>R</i>)<b>-8</b>.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Superimposition of oxazol-2-one (<i>R</i>)-<b>8</b> (white carbons) and isoindolin-1-one <b>10</b> (cyan carbons) bound to Pae LpxC. The α-helix of insert II (and Ile197 side chain, colored in cyan) moved closer to the inhibitor in case of <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From these observations and previous SAR studies conducted on CHIR-090 <b>3</b> analogues,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we anticipated that the composition of the lipophilic tail that spans the hydrophobic passage would greatly influence the binding affinity and therefore the antibacterial activity of LpxC inhibitors. The lipophilic tails of the disclosed LpxC inhibitors with antibacterial activity can be divided in three segments (<b>A</b>, <b>B</b>, and <b>R</b>), each occupying well-defined binding regions in or outside the substrate-binding passage of LpxC enzymes (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). The substructure element directly linked to the bicyclic core, henceforth dubbed segment <b>A</b>, is located within the narrowest part of insert II hydrophobic passage. In the threonyl-hydroxamate series, broad and enhanced antibiotic activity could be achieved for inhibitors featuring a small acetylenic unit (van der Waals (vdW) volume: 38 Å<sup>3</sup> per C<sub>2</sub>H<sub>2</sub> unit)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> in place of a larger phenyl ring (vdW volume: 80.3 Å<sup>3</sup> per C<sub>6</sub>H<sub>6</sub> unit).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> To similarly maximize vdW interactions with LpxC insert II, an acetylenic unit was set as a recurrent segment <b>A</b> for the remainder of the present study.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The second segment of the lipophilic tail (chemical substructure element <b>B</b>, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a) protrudes from the hydrophobic passage and resides at the interphase of the distal end of the hydrophobic passage and the surface of the enzyme (see <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). Only two previously exemplified hydrophobic groups (ACHN-975 <b>1</b> and CHIR-090 <b>3</b>; see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) varying in volume and length were initially considered: a phenyl ring (2.83 Å) and an acetylene unit (1.22 Å). The third substructure element <b>R</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a,b), positioned at the distal end of the inhibitor is partly or totally exposed to solvent, thus providing an excellent means to increase chemical diversity and to modulate physicochemical properties of inhibitors. Efficient synthetic access to designed LpxC inhibitors could be achieved by coupling reactions involving appropriate partners carrying side chains (defined as the combination of structural elements <b>B</b> and <b>R</b>) and three terminal alkyne building blocks <b>29</b>, <b>34</b>, and <b>40</b>, differing from the nature of their core I, II, and III, respectively (see <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). To initiate a study on the effects of segments <b>B</b> and <b>R</b> on antibacterial activity, a few archetypical side chains (<b>SC1</b>–<b>4</b>) were selected as they were easily accessible from commercially available building blocks (including CHIR090 <b>3</b> morpholine-based side chain <b>SC1</b>). For all compounds reported in the following tables, antibacterial activities on wild-type Eco and Pae, MDR Kpn and PAO1 strains, as well as binding affinities (thermal shift, Δ<i>T</i><sub>m</sub> in K) to Eco and Pae LpxC enzymes were determined. For all compounds displaying MIC ≤ 2 μg/mL against Eco, log <i>D</i><sub>7.4</sub> and solubilities at pH 4, 7, and 9 were also measured from DMSO stock solutions.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0007.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a,b) Views of inhibitor <b>17a</b> bound to <i>P. aeruginosa</i> LpxC (2.0 Å) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I49">6I49</a>). The lipophilic tail is subdivided into three segments <b>A</b>, <b>B</b>, and <b>R</b>. From the acetylenic unit <b>A</b>, the side chain resulting from the combination of segment <b>B</b> (here a phenyl ring) and segment <b>R</b>, here an <i>N</i>-methyl-morpholine group protruding out of the substrate-binding passage of Pae LpxC. Several conformations/positions for the morpholine ring of <b>17a</b> are possible, the depicted one serving as an example. Partial ligand electron density is represented as a blue mesh (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> density contoured at 1 sigma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All derivatives featuring side chains <b>SC1</b> and <b>SC2</b> (B = phenyl) displayed good binding affinity to Eco LpxC (Δ<i>T</i><sub>m</sub> ranging from 9.5 to 13 K), translating to good antibacterial activity against Eco and Kpn strains with MICs ≤ 0.5 μg/mL (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Derivatives <b>17a</b>,<b>b</b> and <b>19a</b>,<b>b</b> elicited good MICs against Pae as well (0.5–1 μg/mL range), correlating with good binding affinities to Pae LpxC (Δ<i>T</i><sub>m</sub> ranging from 13.6 to 15.5 K). While featuring different cores, pairs of derivatives <b>17a</b>/<b>19a</b> and <b>17b</b>/<b>19b</b> displayed identical antibacterial profiles against GNBs, even though larger Δ<i>T</i><sub>m</sub> values (+1.9 K) were obtained for thienopyrrolones on Eco and Pae LpxC enzymes. These larger Δ<i>T</i><sub>m</sub> values of thienopyrrolones <b>19a</b> and <b>19b</b> tracked with their higher lipophilicity (higher log <i>D</i><sub>7.4</sub>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Groups <b>R</b> present in <b>SC1</b> and <b>SC2</b> had a negligible effect on biochemical and antibacterial activities when comparing <b>17a</b> to <b>17b</b> and <b>19a</b> to <b>19b</b>. Isoindolin-1-one derivatives <b>15a</b>,<b>b</b> elicited slightly higher MICs (2-fold elevation) against Eco, Kpn, and Pae strains than their analogues featuring either core II or III. The weaker antibacterial activity of isoindolin-1-ones <b>15a</b>,<b>b</b> may be due to weaker binding to LpxC enzymes of these compounds compared to their analoguous inhibitors <b>17a</b>,<b>b</b> or <b>19a</b>,<b>b</b> (featuring cores II or III) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This may result from the marked difference that exists between the orientation of the two exit vectors around core I relative to cores II and III (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). No efforts to further rationalize this observation were undertaken. Solubilities of compounds <b>15a</b> and <b>19a</b> featuring morpholine-containing side chain <b>SC1</b> were higher within the assessed pH range than those of derivatives <b>15b</b> and <b>19b</b> featuring alcohol-based side chain <b>SC2</b>. Pyrrolo-imidazolone derivatives <b>17a</b> and <b>17b</b> featuring side chains <b>SC1</b> and <b>SC2</b>, respectively, shared similar solubility profiles. Morpholine-derivatives <b>15a</b>, <b>17a</b>, and <b>19a</b> had consistently lower log <i>D</i><sub>7.4</sub> than alcohols <b>15b</b>, <b>17b</b>, and <b>19b</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antibacterial Activities, Binding Affinities, and Physicochemical Properties of Compounds <b>15</b>–<b>20a</b>,<b>b</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0017.gif" alt="" id="gr13" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center">Compounds</th><th class="rowsep1 colsep0" colspan="5" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center">Δ<i>T</i><sub>m</sub> (K)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" align="center">log <i>D</i><sub>7.4</sub></th><th class="rowsep1 colsep0" colspan="3" align="center">Solubility (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">core</th><th class="colsep0 rowsep0" align="center">side chain</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Kpn</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center">PAO1</th><th class="colsep0 rowsep0" align="center">Shift</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">pH 4</th><th class="colsep0 rowsep0" align="center">pH 7</th><th class="colsep0 rowsep0" align="center">pH 9</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15a</b></td><td class="colsep0 rowsep0" align="center">I</td><td class="colsep0 rowsep0" align="center">SC1</td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">4×</td><td class="colsep0 rowsep0" align="center">11.0</td><td class="colsep0 rowsep0" align="center">13.8</td><td class="colsep0 rowsep0" align="center">1.8</td><td class="colsep0 rowsep0" align="center">>890</td><td class="colsep0 rowsep0" align="center">148</td><td class="colsep0 rowsep0" align="center">>890</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15b</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC2</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">9.5</td><td class="colsep0 rowsep0" align="center">12.8</td><td class="colsep0 rowsep0" align="center">2.3</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">7</td><td class="colsep0 rowsep0" align="center">136</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16a</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC3</td><td class="colsep0 rowsep0" align="center">16</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">7.0</td><td class="colsep0 rowsep0" align="center">11.6</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16b</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC4</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">8.5</td><td class="colsep0 rowsep0" align="center">12.6</td><td class="colsep0 rowsep0" align="center">1.2</td><td class="colsep0 rowsep0" align="center">71</td><td class="colsep0 rowsep0" align="center">97</td><td class="colsep0 rowsep0" align="center">564</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17a</b></td><td class="colsep0 rowsep0" align="center">II</td><td class="colsep0 rowsep0" align="center">SC1</td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">>16×</td><td class="colsep0 rowsep0" align="center">11.9</td><td class="colsep0 rowsep0" align="center">13.6</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center">802</td><td class="colsep0 rowsep0" align="center">68</td><td class="colsep0 rowsep0" align="center">621</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17b</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC2</td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">11.8</td><td class="colsep0 rowsep0" align="center">13.7</td><td class="colsep0 rowsep0" align="center">1.9</td><td class="colsep0 rowsep0" align="center">70</td><td class="colsep0 rowsep0" align="center">110</td><td class="colsep0 rowsep0" align="center">556</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18a</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC3</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">8.8</td><td class="colsep0 rowsep0" align="center">11.3</td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">>740</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18b</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC4</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">9.9</td><td class="colsep0 rowsep0" align="center">12.3</td><td class="colsep0 rowsep0" align="center">0.9</td><td class="colsep0 rowsep0" align="center">398</td><td class="colsep0 rowsep0" align="center">>710</td><td class="colsep0 rowsep0" align="center">>710</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19a</b></td><td class="colsep0 rowsep0" align="center">III</td><td class="colsep0 rowsep0" align="center">SC1</td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">13.0</td><td class="colsep0 rowsep0" align="center">15.5</td><td class="colsep0 rowsep0" align="center">1.8</td><td class="colsep0 rowsep0" align="center">>900</td><td class="colsep0 rowsep0" align="center">155</td><td class="colsep0 rowsep0" align="center">571</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC2</td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">13.0</td><td class="colsep0 rowsep0" align="center">15.5</td><td class="colsep0 rowsep0" align="center">2.1</td><td class="colsep0 rowsep0" align="center">69</td><td class="colsep0 rowsep0" align="center">83</td><td class="colsep0 rowsep0" align="center">613</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20a</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC3</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">7.4</td><td class="colsep0 rowsep0" align="center">12.2</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">>760</td><td class="colsep0 rowsep0" align="center">>760</td><td class="colsep0 rowsep0" align="center">>760</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">SC4</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">9.9</td><td class="colsep0 rowsep0" align="center">13.6</td><td class="colsep0 rowsep0" align="center">1.2</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">>740</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Eco ATCC 25922 (wt); Kpn T 6474 (MDR); Pae ATCC 27853 (wt); Pae PAO1 (wt); MIC efflux shift: calculated as the ratio of MIC PAO1/MIC PA2616.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">DSF on isolated enzyme, determined at 25 μM compound concentration. ND, not determined.</p></div></div></div><div class="NLM_p">For derivatives <b>16a</b>,<b>b</b>, <b>18a</b>,<b>b</b>, and <b>20a</b>,<b>b</b> featuring alkyne-based side chains <b>SC3</b> and <b>SC4</b> (<b>B</b> = acetylene), MICs determined against Eco and Kpn were within a wider range (0.5–16 μg/mL) than observed for analogues <b>15a</b>,<b>b</b>, <b>17a</b>,<b>b</b>, and <b>19a</b>,<b>b</b> (<b>B</b> = phenyl ring). Derivatives <b>16a</b>, <b>18a</b>, and <b>20a</b> featuring the side chain <b>SC3</b> displayed weaker antibacterial activities with MICs usually ≥2 μg/mL. Binding affinities of these derivatives to Eco (Δ<i>T</i><sub>m</sub> ranging from 7 to 8.8 K) and Pae (Δ<i>T</i><sub>m</sub> ranging from 11.3 to 12.2 K) LpxC enzymes were lower than those of analogues featuring side chain <b>SC4</b>. The comparison of antibacterial activities recorded for pairs of derivatives <b>15b</b>–<b>16b</b>, <b>17b</b>–<b>18b</b>, and <b>19b</b>–<b>20b</b> featuring side chains <b>SC2</b> and <b>SC4</b> (differing only from the nature of segment <b>B</b>) showed that MICs against Eco and Kpn strains were about 4-fold higher for analogues featuring a bis-acetylenic tail (wherein <b>B</b> = acetylene). Similarly, binding affinities (Δ<i>T</i><sub>m</sub>) recorded for compounds <b>16b</b>, <b>18b</b>, and <b>20b</b> to Eco LpxC were lower than those featuring side chain <b>SC2</b>. For the Pae strain, no significantly different antibacterial activities could be observed for compounds differing only in the nature of segment <b>B</b>. Bis-acetylenic derivatives were also sensitive to efflux as indicated by MIC shifts (8 and 16-fold). While the preliminary antibacterial activity profiles of bis-acetylenic derivatives determined against three representative GNB strains seemed slightly inferior to those of phenyl-containing analogues, their solubilities, determined between pH 4 and 9, were consistently higher, matching their lower lipophilicity (lower log <i>D</i><sub>7.4</sub> values). With the exception of derivative <b>15a</b>, isoindolin-1-ones were less soluble at pH 7 than their direct analogues based on cores II and III. Because of their inferior antibacterial and solubility profiles, the isoindolin-1-one series was deprioritized.</div><div class="NLM_p">The binding mode of compound <b>17a</b> to Pae LpxC was determined by X-ray crystallography.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The hydroxamate-based headgroup of <b>17a</b> was bound to zinc in an identical mode to that of previous analogues. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a, the acetylenic unit (segment A) of compound <b>17a</b> occupied the hydrophobic passage, while the phenyl ring (segment B) was located close to the surface of the protein at the exit to the hydrophobic passage. The morpholine ring of <b>17a</b> was located on the surface of the enzyme, fully exposed to solvent, turnable, and not visible in the electron density map (see <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). The analysis of the crystal structure of <b>17a</b> revealed one significant difference in comparison to the structures of hits <b>8</b>, <b>10</b>, and <b>12</b>. While the cores of compounds <b>8</b>, <b>10</b>, and <b>12</b> were coplanar with the peptide unit connecting Phe191 and Gly192, forming a π–π stacking interaction, the core of <b>17a</b> was inclined and therefore not able to stack on these peptide bonds (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). However, another π–π stacking interaction could be observed for <b>17a</b> involving the phenyl moiety and the peptide bond between Gly209 and Ser210 at the outer part of the hydrophobic tunnel, close to the protein surface (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b). This additional interaction could compensate for the former loss of the stacking, leading to decent binding affinity (Eco Δ<i>T</i><sub>m</sub> ∼11.9 K and Pae Δ<i>T</i><sub>m</sub> ∼13.9 K for <b>17a</b>).</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0008.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Superimposition of cores of compound <b>17a</b> (green carbons) and compounds <b>8</b>, <b>10</b>, <b>12</b>, and <b>18d</b> (all gray carbons). Phe191 and Gly192 peptide bonds on right-hand side. (b) View of the <b>B</b><sub>1</sub> phenyl moiety of <b>17a</b> and its stacking over the peptide bond between Gly209 and Ser210.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The superimposition of <b>17a</b> and CHIR-090 <b>3</b> structures (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VWM">5VWM</a>) showed that the core of compound <b>17a</b> resided at the entrance of the hydrophobic passage, while the phenyl core of CHIR-090 <b>3</b> was seated deeper inside the tunnel (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). As the morpholine ring of <b>17a</b> was pushed 0.6 Å further outward, we synthesized shorter bis-acetylenic derivatives to take advantage of the shallow hydrophobic pocket filled by the morpholine ring of CHIR-090 <b>3</b>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Pairs of derivatives differing only in the nature of the structural element <b>B</b> (either a phenyl or and acetylenic unit) were thus produced. Diverse <b>R</b>-groups were attached according to previously reported favorable properties such as substituted cyclopropyl rings (featured in ACHN-975 and analogues).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The most readily accessible pyrrolo-imidazolone series was selected to conduct the study.</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0009.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Superimposition of inhibitor <b>17a</b> (orange carbons) with CHIR-090 <b>3</b> (gray carbons) bound to Pae LpxC (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VWM">5VWM</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All derivatives <b>17a</b>,<b>c</b>–<b>j</b>, wherein <b>B</b> is a phenyl ring displayed potent in vitro antibacterial activity against Eco, Kpn and Pae, with MICs ≤ 2 μg/mL (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). A wide range of functional groups were tolerated as <b>R</b>-groups: alcohols (compounds <b>17c</b>–<b>f</b>), diol (<b>17h</b>), primary and tertiary amines (<b>17g</b> and <b>17a</b>), sulfonamide (<b>17i</b>), and carbamate (<b>17j</b>). All these derivatives were potent LpxC inhibitors as indicated by their strong binding affinities recorded on Eco (Δ<i>T</i><sub>m</sub> ∼11–12 K) and Pae (Δ<i>T</i><sub>m</sub> ∼13–14 K) LpxC enzymes, suggesting that combining the <b>R</b> groups with a phenyl ring has little effect on biochemical potency. Compounds <b>17c</b> and <b>17h</b> displayed good solubilities at pH 7 (>600 μg/mL). In contrast to phenyl-acetylenes, the nature of <b>R</b> greatly influenced antibacterial activity of bis-alkyne derivatives. For instance, sulfonamide <b>18i</b> and diol <b>18h</b> exhibited elevated MICs against Eco, Kpn, and Pae (MICs ≥ 4 μg/mL), roughly 4–8-fold higher than that of their phenyl-containing analogues <b>17i</b> and <b>17h</b>. The weaker antibacterial activity of these derivatives was attributed to their lower Δ<i>T</i><sub>m</sub>s on LpxC enzymes (especially on Eco, with Δ<i>T</i><sub>m</sub> ≤ 8 K). For alcohol <b>18a</b> and tertiary amine <b>18c</b>, antibacterial activities determined against Eco and Kpn were also significantly weaker than that of their phenyl-containing analogues <b>17c</b> and <b>17a</b>, respectively (8–16-fold higher MICs), corresponding to the modest biochemical potencies (Δ<i>T</i><sub>m</sub> values recorded on Eco LpxC ranged from 8.8 to 9.9 K). The antibacterial activity of bis-acetylenic derivatives <b>18a</b> and <b>18c</b> against Pae (MICs 2 μg/mL) was almost as potent as their analogues <b>17c</b> and <b>17a</b>, respectively. Derivatives <b>18d</b>–<b>g</b> displayed potent antibacterial activities against all tested GNB strains (MICs ranging from 0.12 to 1 μg/mL), and only minor differences with their respective phenyl analogues <b>17d</b>–<b>g</b> were observed. As had already been seen with our previous set of data, solubilities at pH 7 of bis-acetylenic derivatives were significantly higher than those of their phenyl-containing analogues. Derivatives <b>18c</b>–<b>g</b> showed good solubility at pH 7 (>350 μg/mL), whereas their respective phenyl-based analogues <b>17a</b> and <b>17d</b>–<b>g</b> displayed limited solubility at the same pH (ranging between 7 and 112 μg/mL). In general, lower solubility and better biochemical potency of phenyl-based derivatives matched with higher log <i>D</i><sub>7.4</sub>. The majority of derivatives presented in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> suffered from being substrates of efflux pumps as indicated by MIC shifts (≥8-fold, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Supporting Information</a> for full MIC PA2616 data set).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Antibacterial Activities, Binding Affinities, and Physicochemical Properties of Novel LpxC Inhibitors <b>17a</b>,<b>c</b>–<b>g</b> and <b>18a</b>,<b>c</b>–<b>g</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0018.gif" alt="" id="gr17" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center">Compounds</th><th class="rowsep1 colsep0" colspan="5" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center">Δ<i>T</i><sub>m</sub> (K)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" align="center">log <i>D</i><sub>7.4</sub></th><th class="rowsep1 colsep0" colspan="3" align="center">Solubility (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">B</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Kpn</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center">PAO1</th><th class="colsep0 rowsep0" align="center">Shift</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">pH 4</th><th class="colsep0 rowsep0" align="center">pH 7</th><th class="colsep0 rowsep0" align="center">pH 9</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17c</b></td><td class="colsep0 rowsep0" align="center">R1</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">10.8</td><td class="colsep0 rowsep0" align="center">13.0</td><td class="colsep0 rowsep0" align="center">1.2</td><td class="colsep0 rowsep0" align="center">195</td><td class="colsep0 rowsep0" align="center">>820</td><td class="colsep0 rowsep0" align="center">>820</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18a</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">8.8</td><td class="colsep0 rowsep0" align="center">11.3</td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">>740</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17a</b></td><td class="colsep0 rowsep0" align="center">R2</td><td class="colsep0 rowsep0" align="center">B<sub><b>1</b></sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">≥8×</td><td class="colsep0 rowsep0" align="center">11.9</td><td class="colsep0 rowsep0" align="center">13.9</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center">802</td><td class="colsep0 rowsep0" align="center">68</td><td class="colsep0 rowsep0" align="center">621</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18c</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">4×</td><td class="colsep0 rowsep0" align="center">9.6</td><td class="colsep0 rowsep0" align="center">11.8</td><td class="colsep0 rowsep0" align="center">0.7</td><td class="colsep0 rowsep0" align="center">>780</td><td class="colsep0 rowsep0" align="center">>780</td><td class="colsep0 rowsep0" align="center">>780</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17d</b></td><td class="colsep0 rowsep0" align="center">R3</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">4×</td><td class="colsep0 rowsep0" align="center">11.8</td><td class="colsep0 rowsep0" align="center">13.6</td><td class="colsep0 rowsep0" align="center">2.0</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center">7</td><td class="colsep0 rowsep0" align="center">56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18d</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">9.9</td><td class="colsep0 rowsep0" align="center">12.5</td><td class="colsep0 rowsep0" align="center">0.9</td><td class="colsep0 rowsep0" align="center">149</td><td class="colsep0 rowsep0" align="center">384</td><td class="colsep0 rowsep0" align="center">553</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17e</b></td><td class="colsep0 rowsep0" align="center">R4</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">≥16×</td><td class="colsep0 rowsep0" align="center">12.1</td><td class="colsep0 rowsep0" align="center">13.4</td><td class="colsep0 rowsep0" align="center">2.0</td><td class="colsep0 rowsep0" align="center">73</td><td class="colsep0 rowsep0" align="center">112</td><td class="colsep0 rowsep0" align="center">568</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18e</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">≥8×</td><td class="colsep0 rowsep0" align="center">10.8</td><td class="colsep0 rowsep0" align="center">13.6</td><td class="colsep0 rowsep0" align="center">1.1</td><td class="colsep0 rowsep0" align="center">600</td><td class="colsep0 rowsep0" align="center">602</td><td class="colsep0 rowsep0" align="center">552</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">       </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17f</b></td><td class="colsep0 rowsep0" align="center">R5</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">≤0.06</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">≥16×</td><td class="colsep0 rowsep0" align="center">12.1</td><td class="colsep0 rowsep0" align="center">13.4</td><td class="colsep0 rowsep0" align="center">1.4</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="center">24</td><td class="colsep0 rowsep0" align="center">170</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18f</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">≥8×</td><td class="colsep0 rowsep0" align="center">11.2</td><td class="colsep0 rowsep0" align="center">13.2</td><td class="colsep0 rowsep0" align="center">1.3</td><td class="colsep0 rowsep0" align="center">600</td><td class="colsep0 rowsep0" align="center">588</td><td class="colsep0 rowsep0" align="center">628</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17g</b></td><td class="colsep0 rowsep0" align="center">R6</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">4×</td><td class="colsep0 rowsep0" align="center">11.3</td><td class="colsep0 rowsep0" align="center">12.6</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center">589</td><td class="colsep0 rowsep0" align="center">32</td><td class="colsep0 rowsep0" align="center">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18g</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">≥8×</td><td class="colsep0 rowsep0" align="center">10.0</td><td class="colsep0 rowsep0" align="center">11.6</td><td class="colsep0 rowsep0" align="center">0.7</td><td class="colsep0 rowsep0" align="center">571</td><td class="colsep0 rowsep0" align="center">418</td><td class="colsep0 rowsep0" align="center">536</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17h</b></td><td class="colsep0 rowsep0" align="center">R7</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">>8×</td><td class="colsep0 rowsep0" align="center">11.1</td><td class="colsep0 rowsep0" align="center">12.7</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">167</td><td class="colsep0 rowsep0" align="center">605</td><td class="colsep0 rowsep0" align="center">660</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18h</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">7.3</td><td class="colsep0 rowsep0" align="center">10.3</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17i</b></td><td class="colsep0 rowsep0" align="center">R8</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">11.2</td><td class="colsep0 rowsep0" align="center">13.2</td><td class="colsep0 rowsep0" align="center">1.0</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">3</td><td class="colsep0 rowsep0" align="center">56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18i</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">4×</td><td class="colsep0 rowsep0" align="center">8.0</td><td class="colsep0 rowsep0" align="center">10.8</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="14" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17j</b></td><td class="colsep0 rowsep0" align="center">R9</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">11.8</td><td class="colsep0 rowsep0" align="center">14.2</td><td class="colsep0 rowsep0" align="center">1.7</td><td class="colsep0 rowsep0" align="center"><1</td><td class="colsep0 rowsep0" align="center">15</td><td class="colsep0 rowsep0" align="center">251</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18j</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">9.4</td><td class="colsep0 rowsep0" align="center">12.9</td><td class="colsep0 rowsep0" align="center">0.9</td><td class="colsep0 rowsep0" align="center">641</td><td class="colsep0 rowsep0" align="center">785</td><td class="colsep0 rowsep0" align="center">>880</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Eco ATCC 25922 (wt); Kpn T 6474 (MDR); Pae ATCC 27853 (wt); Pae PAO1 (wt); MIC efflux shift, calculated as the ratio of MIC PAO1/MIC PA2616.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">DSF on isolated enzyme, determined at 25 μM compound concentration. ND, not determined.</p></div></div></div><div class="NLM_p">The binding mode of bis-acetylenic compound <b>18d</b> to Pae LpxC was also determined by X-ray crystallography.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The methylsulfone hydroxamate headgroup and the pyrrolo-imidazolone core were bound to Pae LpxC similarly as all related derivatives (for details, see SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Figure S-8</a>). As illustrated in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>, the bis-acetylenic needle filled the substrate passage, projecting the group <b>R</b> at the rim of the protein. As postulated earlier, the cyclopropyl ring occupied the shallow hydrophobic side pocket in two refined conformations. The hydroxymethyl group pointed toward the solvent in the opposite direction of cyclopropyl unit (in both conformations). Improved binding affinities to Pae LpxC enzyme were recorded for compounds <b>18e</b> and <b>18f</b> (Δ<i>T</i><sub>m</sub> +1.1 and +0.7 K, respectively) comparatively to compound <b>18d</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). This would suggest that the hydroxymethyl group is better accommodated on either of the two terminal carbons of the cyclopropyl ring, when extending toward the solvent.</div><figure id="fig10" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0010.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. View of compound <b>18d</b> bound to Pae LpxC (2.2 Å) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I4A">6I4A</a>). The 1-hydromethyl-1-cyclopropyl side chain is located outside the lipophilic tunnel. Two conformations are possible.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From the set of pyrrolo-imidazolone derivatives <b>17a</b>–<b>j</b> and <b>18a</b>–<b>j</b>, presented in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, a few side chains that resulted in potent antibacterial activity (MIC ≤ 1 μg/mL) and good solubility profiles (>150 μg/mL at 4 < pH < 9) were selected to prepare and evaluate the corresponding thienopyrrolone derivatives featuring the same combinations of <b>R</b> and <b>B</b> elements.</div><div class="NLM_p">Thienopyrrolone derivatives <b>19c</b>,<b>d</b> and <b>20c</b>–<b>f</b> exhibited good antibacterial activities against GNBs (all MICs were ≤2 μg/mL), nearly identical to those of their respective pyrrolo-imidazolone analogues (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The solubilities of thienopyrrolone derivatives <b>19d</b> and <b>20d</b>,<b>f</b> were roughly 2-fold lower than those of their pyrrolo-imidazolone analogues <b>17h</b> and <b>18e</b>,<b>g</b> but still >200 μg/mL at pH 7. Thienopyrrolones <b>19c</b>, <b>20c</b>, and <b>20e</b> displayed comparable solubilities to those of their analogues <b>17c</b>, <b>18d</b>, and <b>18f</b>. In general, thienopyrrolone derivatives showed higher log <i>D</i><sub>7.4</sub> values (around +0.3 unit) and higher biochemical potencies, with Δ<i>T</i><sub>m</sub>s ranging from 12.3 to 15 K (to be compared with Δ<i>T</i><sub>m</sub>s ranging from 11.6 to 13.6 for corresponding pyrrolo-imidazolone analogues).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Antibacterial Activities, Binding, And Physicochemical Properties of Novel LpxC Inhibitors <b>19c</b>,<b>d</b> and <b>20c</b>–<b>f</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0019.gif" alt="" id="gr19" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center">Compounds</th><th class="rowsep1 colsep0" colspan="5" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="2" align="center">Δ<i>T</i><sub>m</sub> (K)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" align="center">log <i>D</i><sub>7.4</sub></th><th class="rowsep1 colsep0" colspan="3" align="center">Solubility (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">B</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Kpn</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center">PAO1</th><th class="colsep0 rowsep0" align="center">Shift</th><th class="colsep0 rowsep0" align="center">Eco</th><th class="colsep0 rowsep0" align="center">Pae</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">pH 4</th><th class="colsep0 rowsep0" align="center">pH 7</th><th class="colsep0 rowsep0" align="center">pH 9</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19c</b></td><td class="colsep0 rowsep0" align="center">R1</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">11.4</td><td class="colsep0 rowsep0" align="center">14.3</td><td class="colsep0 rowsep0">1.5</td><td class="colsep0 rowsep0" align="center">114</td><td class="colsep0 rowsep0" align="center">>850</td><td class="colsep0 rowsep0" align="center">>850</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19d</b></td><td class="colsep0 rowsep0" align="center">R7</td><td class="colsep0 rowsep0" align="center">B<sub>1</sub></td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">16×</td><td class="colsep0 rowsep0" align="center">12.1</td><td class="colsep0 rowsep0" align="center">14.3</td><td class="colsep0 rowsep0">0.8</td><td class="colsep0 rowsep0" align="center">230</td><td class="colsep0 rowsep0" align="center">300</td><td class="colsep0 rowsep0" align="center">814</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20c</b></td><td class="colsep0 rowsep0" align="center">R3</td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">10.9</td><td class="colsep0 rowsep0" align="center">14.2</td><td class="colsep0 rowsep0">1.2</td><td class="colsep0 rowsep0" align="center">443</td><td class="colsep0 rowsep0" align="center">>760</td><td class="colsep0 rowsep0" align="center">>760</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20d</b></td><td class="colsep0 rowsep0" align="center">R4</td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.12</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">>8×</td><td class="colsep0 rowsep0" align="center">11.2</td><td class="colsep0 rowsep0" align="center">15.0</td><td class="colsep0 rowsep0">1.3</td><td class="colsep0 rowsep0" align="center">464</td><td class="colsep0 rowsep0" align="center">377</td><td class="colsep0 rowsep0" align="center">380</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20e</b></td><td class="colsep0 rowsep0" align="center">R5</td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.15</td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">>8×</td><td class="colsep0 rowsep0" align="center">12.2</td><td class="colsep0 rowsep0" align="center">15.0</td><td class="colsep0 rowsep0">1.3</td><td class="colsep0 rowsep0" align="center">522</td><td class="colsep0 rowsep0" align="center">603</td><td class="colsep0 rowsep0" align="center">>760</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20f</b></td><td class="colsep0 rowsep0" align="center">R6</td><td class="colsep0 rowsep0" align="center">B<sub>2</sub></td><td class="colsep0 rowsep0" align="center">0.25</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">1</td><td class="colsep0 rowsep0" align="center">8×</td><td class="colsep0 rowsep0" align="center">10.0</td><td class="colsep0 rowsep0" align="center">12.5</td><td class="colsep0 rowsep0">1.0</td><td class="colsep0 rowsep0" align="center">>740</td><td class="colsep0 rowsep0" align="center">207</td><td class="colsep0 rowsep0" align="center">>740</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Eco ATCC 25922 (wt); Kpn T 6474 (MDR); Pae ATCC 27853 (wt); Pae PAO1 (wt); MIC efflux shift, calculated as the ratio of MIC PAO1/MIC PA2616.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">DSF on isolated enzyme, determined at 25 μM compound concentration.</p></div></div></div><div class="NLM_p last">In vitro metabolic stability determined either in microsomal preparations or in hepatocytes for representative pyrrolo-imidazolone and thienopyrrolone derivatives were largely similar and did not allow prioritization of one series (data not shown). Eventually, the pyrrolo-imidazolone series was selected over the thienopyrrolone series to enter lead optimization for chemical reasons, considering the much better tractability of key alkyne intermediate <b>34</b> (see <a class="ref internalNav" href="#sec2" aria-label="Chemistry section">Chemistry section</a>).</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Mouse Efficacy and Pharmacokinetic Parameters.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></h3><div class="NLM_p">The pharmacological profile of representative pyrrolo-imidazolone derivatives eliciting MICs ≤ 1 μg/mL against tested GNBs, and solubility >200 μg/mL at pH 7, was investigated in more depth.</div><div class="NLM_p">The activities of <b>17h</b>, <b>18d</b>, <b>18f</b>, and <b>18g</b> against <i>Escherichia coli</i> (Eco) were explored in vivo in a neutropenic murine thigh infection model.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Mice were infected with Eco (inoculum: 1.10<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> CFU/thigh), and 2 h later they were subcutaneously (sc) treated with vehicle or 50 or 150 mg/kg of the compounds (one additional group was not treated and was used to determine the bacterial load 2 h after infection). After 22 h (24 h post infection), the bacterial load had increased to 1 × 10<sup>10</sup> units/thigh in vehicle treated mice. The increase in bacterial load in those treated with the different compounds was lower (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">6</a></a>). All derivatives displayed a dose-dependent increase in efficacy (i.e., reduction in bacterial load with increasing doses) (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">6</a></a>). None of these derivatives were efficacious enough at 150 mg/kg to elicit net static efficacy (i.e., difference in CFU/thigh between treatment start and end = 0, <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">6</a></a>). The pharmacokinetic properties of <b>17h</b>, <b>18d</b>, <b>18f</b>, and <b>18g</b>, especially AUCs in plasma and free AUCs reached over 24 h were determined experimentally after subcutaneous administration of 50 mg/kg (except for compound <b>18g</b> for which data were extrapolated from a 150 mg/kg experiment). Whereas compounds <b>17h</b>, <b>18d</b>, and <b>18g</b> displayed higher free AUC values, ranging from 1.16 to 2.81 (μg·h)/mL, corresponding to calculated unbound clearance values Cl<sub>u</sub>/<i>F</i> ranging between 295 and 716 mL/min/kg, <b>18f</b> was more rapidly cleared, resulting in a lower free AUC (0.37 (μg·h)/mL). Notably, derivatives <b>17h</b> had a free AUC comparable to that of <b>18g</b> despite being highly protein bound in mouse plasma. Derivative <b>18d</b> had a 7-fold higher free AUC than <b>18f</b> because of its higher unbound plasma fraction. All four derivatives showed higher unbound clearances and consequently lower free AUC (at 50 mg/kg) than the optimized pyridone <b>7</b>. With this limited set of data, no good predictor for efficacy could be derived from the MIC in vitro and the PK parameters. Indeed, while compounds <b>17h</b> and <b>18g</b> elicited similar free AUC<sub>50</sub>s and roughly similar MICs, <b>17h</b> was more efficacious against Eco in vivo (1.4 log difference in CFU reduction at a dose of 150 mg/kg) than compound <b>18g</b>. Moreover, compounds <b>18d</b> and <b>18f</b> elicited a similar efficacy profile (Δlog<sub>10</sub> ∼+2 and +0.8 CFU/thigh at 50 or 150 mg/kg) even though free AUCs and free AUCs/MIC were much different. As derivative <b>18d</b> elicited a PK profile in mice at 50 mg/kg (free AUC<sub>50</sub>= 2.81 μg·h/mL) close to the one of reference pyridone <b>7</b>,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> further experiments were conducted with <b>18d</b> to determine a dosing regimen that would elicit net static efficacy. A single dose of 450 mg/kg, resulted in a net killing of 1 log. In addition, in mice dosed four times a day with <b>18d</b> (at 112.54 mg/kg per dose), 2, 7.5, 13 and 18.5 h post infection, efficacy improved slightly (1.5 log net reduction in CFU/thigh). To ascertain the origin of this favorable outcome, further PK experiments were conducted in mice, with subcutaneous dosing at 150 and 450 mg/kg. Free exposures derived from these experiments were AUC<sub>150</sub>= 7.16 and AUC<sub>450</sub> = 8.64 μg·h/mL, respectively, clearly demonstrating that free exposures did not dose-proportionally increase between 150 and 450 mg/kg doses. Strikingly, dosing at 450 mg/kg slightly increased free AUC (+1.5 μg·h/mL) but led to a significant improvement in efficacy (nearly 2 log reduction of CFUs). Even if net static efficacy with derivative <b>18d</b> could only be reached against a wild-type GNB (MIC Eco ATCC 25922 = 0.25 μg/mL) by applying high doses and would likely translate to a nonachievable dosing regimen against less sensitive bacteria in clinics, the significant improvement in efficacy resulting from only a modest increase in free exposure prompted us to pursue the optimization work on this chemotype toward analogues more potent in vivo.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Efficacy in Murine Thigh Model (Eco) and Pharmacokinetic Parameters of LpxC Inhibitors <b>17h</b>, <b>18d</b>, <b>18f</b>, and <b>18g</b> Administered Subcutaneously</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compds</th><th class="colsep0 rowsep0" align="center"><b>17h</b></th><th class="colsep0 rowsep0" align="center"><b>18d</b></th><th class="colsep0 rowsep0" align="center"><b>18f</b></th><th class="colsep0 rowsep0" align="center"><b>18g</b></th><th class="colsep0 rowsep0" align="center"><b>7</b><a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MIC Eco (μg/mL)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">≤0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Δlog<sub>10</sub>CFU/Thigh<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a> Mouse</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">50 mg/kg<a class="ref internalNav" href="#t7fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">+2.4</td><td class="colsep0 rowsep0" align="left">+2.1</td><td class="colsep0 rowsep0" align="left">+1.8</td><td class="colsep0 rowsep0" align="left">+2.8</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">150 mg/kg<a class="ref internalNav" href="#t7fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">+0.4</td><td class="colsep0 rowsep0" align="left">+0.8</td><td class="colsep0 rowsep0" align="left">+0.8</td><td class="colsep0 rowsep0" align="left">+1.8</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">PK Parameters<a class="ref internalNav" href="#t7fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0→24h</sub> (μg·h/mL)</td><td class="colsep0 rowsep0" align="left">76.5</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">14.9<a class="ref internalNav" href="#t7fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">11.73<a class="ref internalNav" href="#t7fn6" aria-label="f">f</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>f</i><sub>u</sub></td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.085</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="left">0.078</td><td class="colsep0 rowsep0" align="left">0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">free AUC<sub>0→24h</sub> (μg·h/mL)</td><td class="colsep0 rowsep0" align="left">1.23</td><td class="colsep0 rowsep0" align="left">2.81</td><td class="colsep0 rowsep0" align="left">0.37</td><td class="colsep0 rowsep0" align="left">1.16<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">3.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cl/F (mL/min/kg)</td><td class="colsep0 rowsep0" align="left">10.8</td><td class="colsep0 rowsep0" align="left">25.1</td><td class="colsep0 rowsep0" align="left">20.2</td><td class="colsep0 rowsep0" align="left">55.9</td><td class="colsep0 rowsep0" align="left">72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cl<sub>u</sub>/<i>F</i> (mL/min/kg)</td><td class="colsep0 rowsep0" align="left">680</td><td class="colsep0 rowsep0" align="left">295</td><td class="colsep0 rowsep0" align="left">2300</td><td class="colsep0 rowsep0" align="left">716</td><td class="colsep0 rowsep0" align="left">231</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (μg/mL)</td><td class="colsep0 rowsep0" align="left">41.3</td><td class="colsep0 rowsep0" align="left">20.2</td><td class="colsep0 rowsep0" align="left">22.6</td><td class="colsep0 rowsep0" align="left">8.6<a class="ref internalNav" href="#t7fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">10.3<a class="ref internalNav" href="#t7fn6" aria-label="f">f</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.33</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Eco ATCC 25922 (wt).</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">Calculated from sc data reported at 75 mg/kg (CD-1 mouse strain).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">Median difference in bacterial load (log<sub>10</sub> CFU/thigh) between start (2 h post infection) and end of treatment period.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">Mice were treated sc (50 or 150 mg/kg) starting at 2 h post infection and sacrificed at 24 h post infection.</p></div><div class="footnote" id="t7fn5"><sup><sup>e</sup></sup><p class="last">Mice dosed sc at 50 mg/kg for <b>17h</b>, <b>18d</b> and <b>18f</b>, at 150 mg/kg for <b>18g</b>.</p></div><div class="footnote" id="t7fn6"><sup><sup>f</sup></sup><p class="last">Linear extrapolation for a sc administered dose of 50 mg/kg; ND, not determined</p></div></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have reported herein the discovery of novel chemical series of LpxC inhibitors based on bicyclic cores using structural information to guide design and optimization processes starting from a known methyl sulfone hydroxamate inhibitor. Along this path, 4,5-dihydro-6<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-6-ones or 1,2-dihydro-3<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazole-3-ones emerged as the most promising hit series. Further investigations performed on the lipophilic side chains led to the identification of derivatives with potent antibacterial activities against Enterobacteriaceae and <i>Pseudomonas aeruginosa</i> strains. The pyrrolo-imidazolone series was subsequently prioritized because of its better scalability, offering lower technical costs in the longer run. Four pyrrolo-imidazolone LpxC inhibitors were shown to be efficacious against <i>Escherichia coli</i> in a clinically relevant murine infection model. However, in vivo efficacy of these novel antibacterial agents could only be achieved at high doses that would likely require unattainable human doses to be efficacious. Moreover, the biopharmaceutical profiles of the most advanced derivatives, <b>18d</b> and <b>18g</b>, were still suboptimal, with insufficient solubility in a pH range suitable for human intravenous administration.</div><div class="NLM_p last">After these initial promising results, our future lead optimization program will focus on identifying compounds eliciting higher intrinsic solubility while achieving in vivo efficacy at lower doses. Strategies and subsequent synthetic medicinal chemistry on the pyrrolo-imidazolone series aimed at solving the issues encountered in this preliminary study are discussed in the <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.9b01605">companion paper</a>.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32381" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32381" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry</h3><div class="NLM_p last">Preparations of iodo-phenyls <b>41</b>–<b>50</b> and halo-alkynes <b>51</b>–<b>60</b> are detailed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">SI</a>. All solvents and chemicals were used as purchased without further purification. Analytical TLC characterizations were performed with 0.2 mm plates: Merck, Silica gel 60 F254. Column chromatography (CC) were performed using Brunschwig 60A silica gel (0.032–0.63 mm) or using an ISCO CombiFlash system and prepacked SiO<sub>2</sub> cartridges, and the elution was performed with an appropriate gradient of the indicated solvents. Unless indicated otherwise, the gradient started from 100% of apolar solvent and stopped at 100% of the polar solvent. When the polar solvent was MeOH, 100% corresponded to a 90:10 mixture of DCM/MeOH. In cases of compounds containing a basic function (e.g., amine), 1% of NH<sub>4</sub>OH (25% aq) was added to the polar eluent(s). Prep-HPLCs were performed on XBridge Prep C18 columns from Waters. Eluents: A, H<sub>2</sub>O + 0.5% HCO<sub>2</sub>H; B, MeCN; gradient, 5% B to 95% B over 5 min. Detection: UV/vis and/or MS and/or ELSD. Melting points were determined using a Büchi A-540 apparatus and are uncorrected. NMR spectra were recorded on a Bruker Ascend 500 (500 MHz for <sup>1</sup>H; 125 MHz for <sup>13</sup>C) spectrometer. Chemical shifts (δ) are reported in parts per million (ppm) relative to deuterated solvent as the internal standard (δH: CDCl<sub>3</sub> 7.26 ppm, DMSO-<i>d</i><sub>6</sub> 2.50 ppm); multiplicities, s = singlet, d = doublet, t = triplet, q = quadruplet, m = multiplet, br = broad signal; coupling constants are given in Hz. LC-HRMS were performed on Waters Acquity UPLC-MS; pump, Waters Acquity Binary, Solvent Manager; MS, SYNAPT G2MS; DAD, Acquity UPLC PDA detector; column, Acquity UPLC CSH C18 1.7 μm 2.1 mm × 50 mm from Waters, thermostated in the Acquity UPLC Column Manager at 60 °C. Eluents: A, H<sub>2</sub>O + 0.05% HCO<sub>2</sub>H; B, MeCN + 0.05% HCO<sub>2</sub>H; gradient, 2% B to 98% B over 2.0 min; flow, 0.6 mL/min; detection, UV 214 nm and MS; the retention time <i>t</i><sub><i>R</i></sub> is given in min. The purity of all final compounds was ≥95%. Chiral LC (method 1) were performed on a ChiralPak AY-H column from Daicel. Column, 5 μm 4.6 mm × 250 mm. Eluent, heptane/EtOH 3:7; flow, 0.8 mL/min; detection, UV 210 nm. Chiral LC (method 2) were performed on a ChiralPak IF column from Daicel. Column, 5 μm 4.6 mm × 250 mm. Eluent, MeCN/EtOH + 0.2% TFA 1:4; flow, 1.2 mL/min; detection, UV 210 nm. Other LC-MS were obtained under the following conditions: pump, Agilent G4220A binary gradient pump; MS detector, Thermo Finnigan MSQPlus or equivalent; DAD detector, Agilent G4212A; ELS detector, Sedere SEDEX 90; column, Zorbax SB-aq 3.5 μm 4.6 mm × 50 mm from Agilent, thermostated at 40 °C. Eluents: A, H<sub>2</sub>O + 0.04% TFA; B, MeCN; gradient, 25% B to 95% B (0.0–1.0 min), 95% B (1.0–1.45 min); flow, 4.5 mL/min; detection, UV 214 nm and MS; the retention time <i>t</i><sub><i>R</i></sub> is given in min. The number of decimals given for the corresponding [M + H<sup>+</sup>] peak(s) of each tested compound depends upon the accuracy of the LC-MS device used.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis of Final Compounds</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> General Procedure A: Amide Coupling with NH<sub>2</sub>OTHP</h4><div class="NLM_p last">Method A: To a solution of carboxylic acid (1 equiv) in DMF (1M) were added NH<sub>2</sub>OTHP (1.5 equiv), HOBT (1.8 equiv), EDC·HCl (1.4 equiv), and TEA (1.8 equiv). The resulting solution was stirred at rt for 16 h. The reaction mixture was partitioned between water (10 mL/mmol) and EtOAc (10 mL/mmol). The organic layer was washed with 10% aq NaHSO<sub>4</sub> (10 mL/mmol), satd aq NaHCO<sub>3</sub> (10 mL/mmol), and brine (10 mL/mmol). If necessary, the evaporation residue was purified via CC using an appropriate gradient of eluent. Method B: To a solution of carboxylic acid (1 equiv) in THF (5 mL/mmol) were added DIPEA (5.5 equiv) and NH<sub>2</sub>OTHP (1.5 equiv). After cooling to 0 °C, a solution of propylphosphonic anhydride (T3P 50% in solution in EtOAc, 1.5 equiv) was added over 30 min. The reaction proceeded at 0 °C for 1 h and was warmed to rt over 2 h. After cooling to 0 °C, satd aq NaHCO<sub>3</sub> (10 mL/mmol) was slowly added, followed by water (10 mL/mmol) and EtOAc (20 mL/mmol). The aq layer was extracted twice with EtOAc. The evaporation residue was purified via CC using an appropriate gradient of eluent.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Procedure B: Sonogashira Coupling</h4><div class="NLM_p last">CuI (0.2 mmol), PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (0.1 mmol), the halo derivative (1.3 mmol) and the terminal alkyne derivative (1 mmol) were introduced in a two-neck round-bottom flask. The atmosphere was flushed with nitrogen during 30 min, then degassed THF (5 mL) and degassed TEA (2.5 mmol) were added. The reaction proceeded at 50 °C until total consumption of the alkyne. After concentration to dryness, the evaporation residue was purified via CC using an appropriate eluent.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure C: Cadiot–Chodkiewicz Coupling</h4><div class="NLM_p last">CuCl (0.1 mmol) was added to a 30% aqueous solution of nBuNH<sub>2</sub> (0.7 mL) at rt. NH<sub>2</sub>OH·HCl (1.42 mmol) was added. The terminal alkyne (0.0.95 mmol) was added, and the solution was cooled to 0 °C. A solution of halo-alkyne (1.42 mmol) in Et<sub>2</sub>O (0.5 mL) was added in one portion. The reaction proceeded at rt until completion. The reaction mixture was diluted with water (40 mL) and extracted with EtOAc (4 × 60 mL). The evaporation residue was purified by CC using an appropriate gradient of solvent.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Procedure D: Acidic THP-Deprotection</h4><div class="NLM_p last">Method A: To the THP-protected hydroxamic acid derivative (0.1 mmol) in EtOH (5 mL), was added PPTS (0.090 g; 0.11 mmol). The reaction proceeded at reflux for 2 h, cooled to rt, and directly purified via precipitation, trituration, or CC using a DCM–MeOH gradient or via prep-HPLC. Method B: Same protocol but using dioxane–water (5–1) as a solvent. Method C: To the THP-protected hydroxamic acid derivative (0.1 mmol) in MeOH (1.1 mL) and water (0.1 mL) was added 2 M HCl (0.5 mL). The reaction proceeded at rt for 1 h. Aqueous NH<sub>4</sub>OH was added until pH 8 was reached, and the resulting mixture was purified via prep-HPLC.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> General Procedure E: Cleavage of a Silyl Protecting Group</h4><div class="NLM_p last">To a solution of silyl protected alcohol (0.154 mmol) in THF (3 mL) was added TBAF (1 M in THF, 0.3 mL; 0.3 mmol). The reaction proceeded at rt for 1 h. EtOAc (20 mL) and brine (10 mL) were added. The evaporation residue was purified via CC using an appropriate gradient of solvent.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Procedure F: Cleavage of an Ester Protecting Group</h4><div class="NLM_p last">To a solution of ester (0.53 mmol) in MeOH (3 mL) was added K<sub>2</sub>CO<sub>3</sub> (0.146 g; 1.06 mmol). The reaction proceeded at rt for 30 min. The reaction mixture was partitioned between DCM (30 mL) and water (10 mL). The two layers were separated and the organic layer was extracted with DCM–MeOH (9–1, 30 mL). The evaporation residue was purified via CC using an appropriate gradient of solvent. Alternatively, the reaction mixture was diluted with water (1 mL) and directly purified via prep-HPLC.</div></div><div id="sec5_2_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (2<i>R</i>)-<i>N</i>-Hydroxy-2-methyl-2-(methylsulfonyl)-4-(5-((4-(morpholinomethyl)phenyl)ethynyl)-1-oxoisoindolin-2-yl)butanamide (<b>15a</b>)</h4><div class="NLM_p last">Alkyne <b>29</b> (0.090 g; 0.21 mmol) and 4-(4-iodobenzyl)morpholine <b>41</b> (0.082 g; 0.27 mmol) were reacted following general procedure B (59% yield). The resulting product was then reacted following general procedure D (method A) to afford <b>15a</b> (0.033 g; 54% yield) as a white solid; mp 153.4 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.97 (d, <i>J</i> = 1.8 Hz, 1H), 9.18 (d, <i>J</i> = 1.9 Hz, 1H), 7.79 (s, 1H), 7.69 (d, <i>J</i> = 7.8 Hz, 1H), 7.64 (d, <i>J</i> = 7.8 Hz, 1H), 7.56 (d, <i>J</i> = 8.2 Hz, 2H), 7.39 (d, <i>J</i> = 8.2 Hz, 2H), 4.57–4.48 (m, 2H), 3.63 (overlaid m, 1H), 3.60–3.56 (m, 4H), 3.50 (s, 2H), 3.48 (overlaid m, 1H), 3.07 (s, 3H), 2.59 (m, 1H), 2.40–2.32 (m, 4H), 1.96 (m, 1H), 1.56 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 166.9, 163.7, 142.9, 139.8, 132.6, 131.9 (2C), 131.5, 129.7 (2C), 126.7, 125.5, 123.5, 120.9, 91.5, 89.2, 68.5, 66.6 (2C), 62.4, 53.6 (2C), 49.7, 38.1, 37.1, 31.2, 13.6. MS (ESI, <i>m</i>/<i>z</i>): 526.27 [M + H<sup>+</sup>] for C<sub>27</sub>H<sub>32</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.57 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>27</sub>H<sub>32</sub>N<sub>3</sub>O<sub>6</sub>S 526.2011, found 526.2012; <i>t</i><sub><i>R</i></sub> = 0.44 min.</div></div><div id="sec5_2_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(5-((4-(2-hydroxypropan-2-yl)phenyl)ethynyl)-1-oxoisoindolin-2-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>15b</b>)</h4><div class="NLM_p last">Alkyne <b>29</b> (0.098 g; 0.22 mmol) and 2-(4-iodophenyl)propan-2-ol <b>42</b> (0.083 g; 0.317 mmol) were reacted following general procedure B (58% yield). The resulting product was then reacted following general procedure D (method C) to afford <b>15b</b> (0.033 g; 54% yield) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.96 (s, 1H), 9.17 (s, 1H), 7.79 (m, 1H), 7.69–7.62 (m, 2H), 7.55–7.51 (m, 4H), 5.14 (s, 1H), 4.52 (d, <i>J</i> = 2.0 Hz, 2H), 3.61 (m, 1H), 3.47 (m, 1H), 3.07 (s, 3H), 2.60 (m, 1H), 1.98 (m, 1H), 1.55 (s, 3H), 1.43 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 167.0, 163.9, 152.3, 142.9, 132.5, 131.5 (2C), 131.4, 126.7, 125.7, 125.6 (2C), 123.5, 119.9, 91.8, 88.9, 71.1, 68.5, 49.7, 38.1, 37.0, 32.2 (2C), 31.3, 13.5. MS (ESI, <i>m</i>/<i>z</i>): 485.161 [M + H<sup>+</sup>] for C<sub>25</sub>H<sub>29</sub>N<sub>2</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.76 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>25</sub>H<sub>29</sub>N<sub>2</sub>O<sub>6</sub>S 485.174, found 485.1754; <i>t</i><sub><i>R</i></sub> = 0.77 min.</div></div><div id="sec5_2_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(5-((3-hydroxyoxetan-3-yl)buta-1,3-diyn-1-yl)-1-oxoisoindolin-2-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>16a</b>)</h4><div class="NLM_p last">Alkyne <b>29</b> (0.105 g; 0.24 mmol) and halo-alkyne derivative <b>51</b> (0.088 g; 0.39 mmol) were reacted following general procedure C (71% yield). The resulting product was then reacted following general procedure A (method C) to afford compound <b>16a</b> (0.061 g; 84% yield) as a white solid; mp 172.1 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (d, <i>J</i> = 1.6 Hz, 1H), 9.16 (d, <i>J</i> = 1.8 Hz, 1H), 7.86 (s, 1H), 7.70–7.67 (m, 2H), 6.83 (s, 1H), 4.74 (d, <i>J</i> = 6.8 Hz, 2H), 4.57 (d, <i>J</i> = 6.8 Hz, 2H), 4.51 (s, 2H), 3.61 (m, 1H), 3.50 (m, 1H), 3.07 (s, 3H), 2.62 (m, 1H), 1.97 (m, 1H), 1.55 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 166.7, 163.9, 142.9, 133.7, 132.7, 128.0, 123.7, 123.3, 85.1, 83.7 (2C), 79.5, 74.7, 69.0, 68.5, 66.5, 49.7, 38.1, 37.0, 31.3, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 447.1 [M + H<sup>+</sup>] for C<sub>21</sub>H<sub>23</sub>N<sub>2</sub>O<sub>7</sub>S; <i>t</i><sub><i>R</i></sub> = 0.64 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>21</sub>H<sub>23</sub>N<sub>2</sub>O<sub>7</sub>S 447.1226, found 447.1230; <i>t</i><sub><i>R</i></sub> = 1.1 min.</div></div><div id="sec5_2_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(5-(5-hydroxy-5-methylhexa-1,3-diyn-1-yl)-1-oxoisoindolin-2-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>16b</b>)</h4><div class="NLM_p last">Alkyne <b>29</b> (0.095 g; 0.21 mmol) and halo-alkyne derivative <b>52</b> (0.051 g; 0.31 mmol) were reacted following general procedure C (59% yield). The resulting product was then reacted following general procedure D (method C) to afford compound <b>16b</b> (0.046 g; 84% yield) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (s, 1H), 9.15 (s, 1H), 7.81 (s, 1H), 7.68–7.62 (m, 2H), 5.71 (s, 1H), 4.50 (s, 2H), 3.59 (m, 1H), 3.48 (m, 1H), 3.06 (s, 3H), 2.58 (m, 1H), 1.96 (m, 1H), 1.54 (s, 3H), 1.43 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 166.8, 163.9, 142.9, 133.5, 132.5, 127.9, 123.7, 123.6, 91.1, 78.0, 75.3, 68.5, 65.5, 64.4, 49.7, 38.1, 37.0, 31.5 (2C), 31.3, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 433.08 [M + H<sup>+</sup>] for C<sub>21</sub>H<sub>25</sub>N<sub>2</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.69 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>21</sub>H<sub>25</sub>N<sub>2</sub>O<sub>6</sub>S 433.1433, found 433.1442; <i>t</i><sub><i>R</i></sub> = 1.11 min.</div></div><div id="sec5_2_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>R</i>)-<i>N</i>-Hydroxy-2-methyl-2-(methylsulfonyl)-4-(6-((4-(morpholinomethyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)butanamide (<b>17a</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.133 g; 0.315 mmol) and 4-(4-iodobenzyl)morpholine <b>41</b> (0.105 g; 0.347 mmol) were reacted following general procedure B (84% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>17a</b> (0.033 g; 24% yield) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.93 (br s, 1H), 9.18 (br s, 1H), 7.45 (s, 1H), 7.44 (d, <i>J</i> = 8.1 Hz, 2H), 7.33 (d, <i>J</i> = 8.1 Hz, 2H), 6.26 (s, 1H), 4.45 (s, 2H), 3.60–3.54 (m, 4H), 3.523.47 (overlaid m, 1H), 3.47 (s, 2H), 3.43–3.35 (m, 1H), 3.07 (s, 3H), 2.65–2.53 (m, 1H), 2.39–2.29 (m, 4H), 2.02–1.94 (m, 1H), 1.53 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 514.95 [M + H<sup>+</sup>] for C<sub>25</sub>H<sub>31</sub>N<sub>4</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.54 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>25</sub>H<sub>31</sub>N<sub>4</sub>O<sub>6</sub>S 515.1964, found 515.1970; <i>t</i><sub><i>R</i></sub> = 0.90 min.</div></div><div id="sec5_2_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((4-(2-hydroxypropan-2-yl)phenyl)ethynyl)-3oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2(methylsulfonyl)butanamide (<b>17b</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.235 g; 0.556 mmol) and 2-(4-iodophenyl)propan-2-ol <b>42</b> (0.16 g; 0.61 mmol) were reacted following general procedure B (47% yield). The resulting product was then reacted following general procedure D (method B) to afford compound <b>17b</b> (0.076 g; 61% yield) as a beige solid; mp 158.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.96 (br s, 1H), 9.16 (br s, 1H), 7.50–7.46 (m, 2H), 7.45 (d, <i>J</i> = 1.2 Hz, 1H), 7.43–7.40 (m, 2H), 6.26 (d, <i>J</i> = 1.2 Hz, 1H), 5.09 (s, 1H), 4.46 (s, 2H), 3.53–3.46 (m, 1H), 3.43–3.35 (m, 1H), 3.07 (s, 3H), 2.59 (m, 1H), 1.96 (m, 1H), 1.52 (s, 3H), 1.42 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.7, 151.3, 150.7, 131.0 (2C), 131.0 (2C), 125.4, 120.7, 115.2, 110.5, 105.4, 89.6, 84.4, 71.1, 68.4, 45.2, 39.5, 37.0, 32.2 (2C), 31.1, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 473.96 [M + H<sup>+</sup>] for C<sub>23</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.70 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>23</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S 474.169, found 474.1704; <i>t</i><sub><i>R</i></sub> = 0.76 min.</div></div><div id="sec5_2_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((4-(3-hydroxyoxetan-3-yl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>17c</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.3 g; 0.71 mmol) and iodo-phenyl derivative <b>43</b> (0.293 g; 1.06 mmol) were reacted following general procedure B (81% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>17c</b> (0.096 g; 34% yield) as a beige solid; mp 190.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.97 (s, 1H), 9.20 (br s, 1H), 7.66–7.62 (m, 2H), 7.55–7.51 (m, 2H), 7.48 (d, <i>J</i> = 0.8 Hz, 1H), 6.47 (m, 1H), 6.29 (d, <i>J</i> = 1.2 Hz, 1H), 4.78 (d, <i>J</i> = 6.8 Hz, 2H), 4.67 (d, <i>J</i> = 6.8 Hz, 2H), 4.47 (s, 2H), 3.50 (m, 1H), 3.43 (m, 1H), 3.08 (s, 3H), 2.62 (m, 1H), 1.98 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.6, 144.8, 131.4 (2C), 130.9, 125.3 (2C), 121.9, 115.3, 110.4, 105.4, 89.3, 85.6 (2C), 85.0, 74.3, 68.3, 45.2, 39.5, 36.9, 31.0, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 488.11 [M + H<sup>+</sup>] for C<sub>23</sub>H<sub>26</sub>N<sub>3</sub>O<sub>7</sub>S; <i>t</i><sub><i>R</i></sub> = 0.69 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>23</sub>H<sub>26</sub>N<sub>3</sub>O<sub>7</sub>S 488.1491, found 488.1503; <i>t</i><sub><i>R</i></sub> = 1.07 min.</div></div><div id="sec5_2_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((4-(1-(hydroxymethyl)cyclopropyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>17d</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.120 g; 0.286 mmol) and iodo-phenyl derivative <b>44</b> (0.085 g; 0.31 mmol) were reacted following general procedure D (55% yield). The resulting product was then reacted following general procedure C (method B) to afford compound <b>17d</b> (0.048 g; 63% yield) as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.92 (br s, 1H), 9.15 (br s, 1H), 7.44 (s, 1H), 7.38 (d, <i>J</i> = 8.2 Hz, 2H), 7.30 (d, <i>J</i> = 8.2 Hz, 2H), 6.25 (s, 1H), 4.71 (t, <i>J</i> = 5.6 Hz, 1H), 4.45 (s, 2H), 3.54 (d, <i>J</i> = 5.6 Hz, 2H), 3.50 (overlaid m, 1H), 3.40 (m, 1H), 3.07 (s, 3H), 2.60–2.55 (overlaid m, 1H), 1.97 (m, 1H), 1.52 (s, 3H), 0.89–0.85 (m, 2H), 0.78–0.74 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 150.7, 145.1, 131.2 (2C), 130.9, 128.5 (2C), 120.4, 115.2, 110.6, 105.4, 89.6, 84.6, 68.4, 67.7, 45.2, 39.5, 37.0, 31.1, 27.2, 13.3, 12.7 (2C). MS (ESI, <i>m</i>/<i>z</i>): 485.94 [M + H<sup>+</sup>] for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.72 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S 486.1698, found 486.1709; <i>t</i><sub><i>R</i></sub> = 0.78 min.</div></div><div id="sec5_2_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((4-((1<i>R</i>,2<i>R</i>)-2-(hydroxymethyl)cyclopropyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>17e</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.097 g; 0.23 mmol) and iodo-phenyl derivative <b>45</b> (0.070 g; 0.25 mmol) were reacted following general procedure D (93% yield). The resulting product was then reacted following general procedure C (method B) to afford compound <b>17e</b> (0.023 g; 22% yield) as a beige solid; mp 162.3 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.92 (br s, 1H), 9.19 (br s, 1H), 7.43 (s, 1H), 7.35 (d, <i>J</i> = 7.9 Hz, 2H), 7.08 (d, <i>J</i> = 7.9 Hz, 2H), 6.25 (s, 1H), 4.63 (t, <i>J</i> = 5.6 Hz, 1H); 4.45 (s, 2H), 3.533.42 (m, 2H), 3.42–3.30 (overlaid m, 2H), 3.07 (s, 3H), 2.61 (m, 1H), 1.98 (m, 1H), 1.81 (m, 1H), 1.53 (s, 3H), 1.30 (m, 1H), 0.93–0.83 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.6, 144.4, 131.4 (2C), 130.9, 126.2 (2C), 119.8, 115.1, 110.6, 105.4, 89.6, 84.4, 68.3, 64.1, 45.1, 39.5, 36.9, 31.0, 26.7, 21.1, 14.6, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 485.92 [M + H<sup>+</sup>] for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.71 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S 486.1698, found 486.1700; <i>t</i><sub><i>R</i></sub> = 0.75 min.</div></div><div id="sec5_2_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((4-((1<i>S</i>,2<i>S</i>)-2-(hydroxymethyl)cyclopropyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>17f</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.118 g; 0.28 mmol) and iodo-phenyl derivative <b>46</b> (0.082 g; 0.30 mmol) were reacted following general procedure D (99% yield). The resulting product was then reacted following general procedure C (method B) to afford compound <b>17f</b> (0.027 g; 21% yield) as a beige solid; mp 179.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.92 (br s, 1H), 9.19 (br s, 1H), 7.43 (s, 1H), 7.34 (d, <i>J</i> = 7.9 Hz, 2H), 7.08 (d, <i>J</i> = 7.9 Hz, 2H), 6.25 (s, 1H), 4.64 (m, 1H), 4.45 (s, 2H), 3.53–3.42 (m, 2H), 3.423.30 (overlaid m, 2H), 3.06 (s, 3H), 2.61 (m, 1H), 1.98 (m, 1H), 1.81 (m, 1H), 1.52 (s, 3H), 1.29 (m, 1H), 0.93–0.83 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 150.7, 144.5, 131.4 (2C), 130.9, 126.3, 119.8, 115.1, 110.6, 105.4, 89.7, 84.5, 68.4, 64.1, 45.2, 37.0, 31.1, 26.8, 21.2, 14.7, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 485.91.1 [M + H<sup>+</sup>] for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.71 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub>S 486.1698, found 486.1703; <i>t</i><sub><i>R</i></sub> = 0.75 min.</div></div><div id="sec5_2_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>R</i>)-4-(6-((4-(1-Aminocyclopropyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-<i>N</i>-hydroxy-2-methyl-2-(methylsulfonyl)butanamide Hydrochloride (<b>17g</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.444 g; 1.05 mmol) and iodo-phenyl derivative <b>47</b> (0.341 g; 1.15 mmol) were reacted following general procedure D (48% yield). The resulting product was then reacted following general procedure C (method C) to afford compound <b>17g</b> (0.166 g; 63% yield) as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (br s, 1H), 9.19 (br s, 1H), 8.81 (br s, 3H), 7.55–7.52 (m, 2H), 7.48 (d, <i>J</i> = 1.2 Hz, 1H), 7.43–7.40 (m, 2H), 6.27 (d, <i>J</i> = 1.2 Hz, 1H), 4.46 (s, 2H), 3.50 (m, 1H), 3.40 (m, 1H), 3.07 (s, 3H), 2.60 (m, 1H), 1.98 (m, 1H), 1.53 (s, 3H), 1.42–1.38 (m, 2H), 1.26–1.22 (m, 2H). MS (ESI, <i>m</i>/<i>z</i>): 471.07 [M + H<sup>+</sup>] for C<sub>23</sub>H<sub>27</sub>N<sub>4</sub>O<sub>5</sub>S; <i>t</i><sub><i>R</i></sub> = 0.56 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>23</sub>H<sub>27</sub>N<sub>4</sub>O<sub>5</sub>S 471.1702, found 471.1710; <i>t</i><sub><i>R</i></sub> = 0.92 min.</div></div><div id="sec5_2_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (2<i>R</i>)-4-(6-((4-((1<i>S</i>)-1,2-Dihydroxyethyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-<i>N</i>-hydroxy-2-methyl-2(methylsulfonyl)butanamide (<b>17h</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.6 g; 1.42 mmol) and iodo-phenyl derivative <b>48</b> (0.412 g; 1.56 mmol) were reacted following general procedure C (60% yield). The resulting product was then reacted following general procedure D (method B) to afford compound <b>17h</b> (0.473 g; 60% yield) as an off-white solid; mp 167.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.93 (br s, 1H), 9.19 (br s, 1H), 7.45 (s, 1H), 7.42 (d, <i>J</i> = 8.2 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 6.27 (s, 1H), 5.32 (d, <i>J</i> = 4.3 Hz, 1H), 4.74 (t, <i>J</i> = 5.9 Hz, 1H), 4.55 (m, 1H), 4.46 (s, 2H), 3.50 (m, 1H), 3.453.35 (m, 3H), 3.07 (s, 3H), 2.61 (m, 1H), 1.98 (m, 1H), 1.53 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 150.7, 144.4, 131.1 (2C), 131.0, 127.1 (2C), 121.6, 115.2, 110.5, 105.4, 89.6, 84.6, 73.9, 68.4, 67.7, 45.2, 39.5, 37.0, 31.1, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 475.97 [M + H<sup>+</sup>] for C<sub>22</sub>H<sub>26</sub>N<sub>3</sub>O<sub>7</sub>S S; <i>t</i><sub><i>R</i></sub> = 0.58 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>22</sub>H<sub>26</sub>N<sub>3</sub>O<sub>7</sub>S 476.1491, found 476.1501; <i>t</i><sub><i>R</i></sub> = 0.99 min.</div></div><div id="sec5_2_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>R</i>)-<i>N</i>-Hydroxy-2-methyl-4-(6-((4-(methylsulfonamidomethyl)phenyl)ethynyl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-(methylsulfonyl)butanamide (<b>17i</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.2 g; 0.47 mmol) and iodo-phenyl derivative <b>49</b> (0.237 g; 0.70 mmol) were reacted following general procedure D (55% yield). The resulting product was then reacted following general procedure C (method A) to afford compound <b>17i</b> (0.11 g; 81% yield) as a greyish solid; mp 196.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.96 (s, 1H), 9.19 (s, 1H), 7.62 (t, <i>J</i> = 6.4 Hz, 1H), 7.48 (d, <i>J</i> = 8.2 Hz, 2H), 7.47 (overlaid s, 1H), 7.38 (d, <i>J</i> = 8.2 Hz, 2H), 6.28 (d, <i>J</i> = 1.2 Hz, 1H), 4.47 (s, 2H), 4.18 (d, <i>J</i> = 6.3 Hz, 2H), 3.50 (m, 1H), 3.41 (m, 1H), 3.08 (s, 3H), 2.88 (s, 3H), 2.60 (m, 1H), 1.99 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.6, 139.1, 131.5 (2C), 130.9, 128.4 (2C), 122.0, 115.3, 110.3, 105.4, 89.3, 85.0, 68.3, 46.1, 45.2, 40.4, 39.5, 36.9, 31.0, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 523.00 [M + H<sup>+</sup>] for C<sub>22</sub>H<sub>27</sub>N<sub>4</sub>O<sub>7</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.67 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>22</sub>H<sub>27</sub>N<sub>4</sub>O<sub>7</sub>S<sub>2</sub> 523.1321, found 523.1335; <i>t</i><sub><i>R</i></sub> = 1.11 min.</div></div><div id="sec5_2_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (3<i>R</i>)-4-((2-(4-(Hydroxyamino)-3-methyl-3-(methylsulfonyl)-4-oxobutyl)-3-oxo-2,3-dihydro-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-6-yl)-ethynyl)benzyl 4-hydroxypiperidine-1-carboxylate (<b>17j</b>).<a onclick="showRef(event, 'cit25b'); return false;" href="javascript:void(0);" class="ref cit25b">(25b)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.2 g; 0.47 mmol) and halo-phenyl derivative <b>50</b> (0.256 g; 0.70 mmol) were reacted following general procedure D (61% yield). The resulting product was then reacted following general procedure C (method C) to afford compound <b>17j</b> (0.137 g; 86% yield) as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.96 (s, 1H), 9.19 (s, 1H), 7.50–7.48 (m, 3H), 7.37 (d, <i>J</i> = 8.2 Hz, 2H), 6.28 (d, <i>J</i> = 0.8 Hz, 1H), 5.09 (s, 2H), 4.75 (d, <i>J</i> = 4.1 Hz, 1H), 4.47 (s, 2H), 4.10 (q, <i>J</i> = 5.2 Hz, 1H), 3.74–3.72 (m, 2H), 3.66 (m, 1H), 3.51 (m, 1H), 3.40 (m, 1H), 3.08 (overlaid m, 1H); 3.08 (s, 3H), 2.60 (m, 1H), 1.98 (m, 1H), 1.75–1.68 (m, 2H), 1.54 (s, 3H), 1.34–1.24 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 154.8, 150.6, 137.7, 131.5 (2C), 131.0, 128.1 (2C), 122.6, 115.4, 110.3, 105.4, 89.2, 85.2, 68.3, 66.1, 65.8, 45.2, 41.7 (2C), 39.5, 36.9, 34.4 (2C), 31.0, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 573.09 [M + H<sup>+</sup>] for C<sub>27</sub>H<sub>33</sub>N<sub>4</sub>O<sub>8</sub>S S; <i>t</i><sub><i>R</i></sub> = 0.70 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>27</sub>H<sub>33</sub>N<sub>4</sub>O<sub>8</sub>S 573.2019, found 573.2025; <i>t</i><sub><i>R</i></sub> = 0.75 min.</div></div><div id="sec5_2_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((3-hydroxyoxetan-3-yl)buta-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2-(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>18a</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.285 g; 0.586 mmol) and iodo-alkyne <b>51</b> (0.143 g; 0.64 mmol) were reacted following general procedure C (51% yield). The resulting product was then reacted following general procedure D (method B) to afford compound <b>18a</b> (0.048 g; 37% yield) as a beige solid; mp 135.9 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (br s, 1H), 9.17 (br s, 1H), 7.62 (m, 1H), 6.71 (m, 1H), 6.29 (m, 1H), 4.69 (d, <i>J</i> = 6.4 Hz, 2H), 4.54 (d, <i>J</i> = 6.4 Hz, 2H), 4.44 (m, 2H), 3.38 (m, 1H), 3.06 (s, 3H), 2.55 (overlaid m, 1H), 2.34 (m, 1H), 1.93 (m, 1H), 1.52 (s, 3H). ESI-MS (M + H)<sup>+</sup><i>m</i>/<i>z</i>: 436. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.3, 131.0, 118.2, 108.1, 105.9, 83.9 (2C), 83.4, 75.7, 73.2, 69.6, 68.3, 66.4, 45.2, 39.5, 36.9, 30.9, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 435.86 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>22</sub>N<sub>3</sub>O<sub>7</sub>S; <i>t</i><sub><i>R</i></sub> = 0.60 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>19</sub>H<sub>22</sub>N<sub>3</sub>O<sub>7</sub>S 436.1178, found 436.1188; <i>t</i><sub><i>R</i></sub> = 1.00 min.</div></div><div id="sec5_2_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-(5-hydroxy-5-methylhexa-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>18b</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.25 g; 0.472 mmol) and iodo-alkyne <b>52</b> (0.114 g; 0.567 mmol) were reacted following general procedure C (89% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>18b</b> (0.118 g; 54% yield) as a yellowish solid; mp 155.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (br s, 1H), 9.19 (br s, 1H), 7.58 (d, <i>J</i> = 1.0 Hz, 1H), 6.27 (m, 1H), 4.44 (s, 2H), 3.49 (m, 1H), 3.39 (m, 1H), 3.06 (s, 3H), 2.60 (m, 1H), 1.98 (m, 1H), 1.53 (s, 3H), 1.41 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.4, 130.9, 117.8, 108.6, 105.9, 89.3, 74.1, 73.7, 68.4, 66.0, 64.3, 45.2, 39.5, 36.9, 31.6 (2C), 30.9, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 422.2 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.70 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>19</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S 422.1385, found 422.1384; <i>t</i><sub><i>R</i></sub> = 1.11 min.</div></div><div id="sec5_2_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (2<i>R</i>)-<i>N</i>-Hydroxy-2-methyl-2-(methylsulfonyl)-4-(6-(5-morpholinopenta-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)butanamide (<b>18c</b>)</h4><div class="NLM_p last">Alkyne <b>34</b> (0.1 g; 0.234 mmol) and iodo-alkyne <b>53</b> (0.08 g; 0.319 mmol) were reacted following general procedure E (59% yield). The resulting product was then reacted following general procedure C (method C) to afford compound <b>18c</b> (0.047 g; 77% yield two steps) as a white solid; mp 129.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.62 (m, 1H), 9.21 (m, 1H), 7.60 (d, <i>J</i> = 0.7 Hz, 1H), 6.29 (d, <i>J</i> = 1.2 Hz, 1H), 4.44 (s, 2H), 3.62–3.56 (m, 4H), 3.51 (s, 2H), 3.51–3.46 (overlaid m, 1H), 3.42–3.35 (overlaid m, 1H), 3.07 (s, 3H), 2.61 (m, 1H), 2.49–2.44 (m, 4H), 1.96 (m, 1H), 1.52 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 150.4, 131.0, 118.0, 108.5, 106.0, 79.9, 73.9, 72.5, 70.1, 68.3, 66.5 (2C), 52.0 (2C), 47.5, 45.2, 39.5, 37.0, 31.0, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 463.18 [M + H<sup>+</sup>] for C<sub>21</sub>H<sub>27</sub>N<sub>4</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.51 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>21</sub>H<sub>27</sub>N<sub>4</sub>O<sub>6</sub>S 463.1651, found 463.1656; <i>t</i><sub><i>R</i></sub> = 0.87 min.</div></div><div id="sec5_2_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-((1-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>18d</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (2 g; 4.72 mmol) and bromo-alkyne <b>54</b> (2.55 g; 6.16 mmol) were reacted following general procedure E (77% yield). After two more repetitions, the pooled products were then reacted following general procedures E and D (method A) to afford compound <b>18d</b> (3.3 g; 53% yield two steps) as a white solid; mp 180.3 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (s, 1H), 9.19 (s, 1H), 7.53 (d, <i>J</i> = 0.9 Hz, 1H), 6.24 (d, <i>J</i> = 1.2 Hz, 1H), 5.00 (t, <i>J</i> = 6.1 Hz, 1H), 4.43 (s, 2H), 3.47 (m, 1H), 3.42–3.36 (overlaid m, 1H), 3.38 (d, <i>J</i> = 6.1 Hz, 2H), 3.06 (s, 3H), 2.59 (m, 1H), 1.96 (m, 1H), 1.53 (s, 3H), 0.92–0.89 (m, 2H), 0.88–0.85 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 150.4, 130.9, 117.6, 108.9, 105.9, 89.1, 74.9, 70.6, 68.4, 65.7, 62.3, 45.2, 39.5, 37.0, 31.0, 15.6, 13.5 (2C), 13.3. MS (ESI, <i>m</i>/<i>z</i>): 434.01 [M + H<sup>+</sup>] for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.66 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S 434.1385, found 434.1389; <i>t</i><sub><i>R</i></sub> = 1.11 min.</div></div><div id="sec5_2_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-(((1<i>R</i>,2<i>R</i>)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2(methylsulfonyl)butanamide (<b>18e</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (1.0 g; 2.36 mmol) and bromo-alkyne <b>56</b> (0.769 g; 3.54 mmol) were reacted following general procedure E (54% yield). The resulting product was then reacted following general procedures F and C (method B) to afford compound <b>18e</b> (0.323 g; 57% yield two steps) as a yellowish solid; mp 92.8 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (br s, 1H), 9.20 (br s, 1H), 7.52 (s, 1H), 6.23 (s,, 1H), 4.70 (t, <i>J</i> = 5.6 Hz, 1H), 4.42 (s, 2H), 3.48 (m, 1H), 3.43–3.37 (overlaid m, 2H), 3.24 (m, 1H), 3.06 (s, 3H), 2.59 (m, 1H), 1.96 (m, 1H), 1.51 (s, 3H), 1.43–1.35 (m, 2H), 0.92–0.82 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.4, 130.9, 117.6, 108.9, 106.0, 87.7, 74.9, 70.2, 68.3, 62.5, 61.3, 45.2, 39.5, 37.0, 31.0, 25.6, 13.5, 13.4, 5.0. MS (ESI, <i>m</i>/<i>z</i>): 433.99 [M + H<sup>+</sup>] for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.66 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S 434.1385, found 434.1389; <i>t</i><sub><i>R</i></sub> = 1.09 min.</div></div><div id="sec5_2_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(6-(((1<i>S</i>,2<i>S</i>)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2(methylsulfonyl)butanamide (<b>18f</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.75 g; 1.42 mmol) and bromo-alkyne <b>55</b> (0.461 g; 2.13 mmol) were reacted following general procedure E (78% yield). The resulting product was then reacted following general procedures G and C (method B) to afford compound <b>18f</b> (0.298 g; 89% yield two steps) as a white solid; mp 116.3 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (br s, 1H), 9.20 (br s, 1H), 7.52 (s, 1H), 6.24 (d, <i>J</i> = 1.1 Hz, 1H), 4.69 (t, <i>J</i> = 5.8 Hz, 1H), 4.42 (s, 2H), 3.49 (m, 1H), 3.43–3.37 (m, 2H), 3.25 (td, <i>J</i> = 7.8, 11.4 Hz, 1H), 3.05 (s, 3H), 2.59 (m, 1H), 1.95 (m, 1H), 1.51 (s, 3H), 1.43–1.35 (m, 2H), 0.91–0.81 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 150.4, 130.9, 117.6, 108.9, 106.0, 87.7, 74.9, 70.2, 68.4, 62.5, 61.3, 45.2, 39.5, 37.0, 31.0, 25.6, 13.5, 13.3, 5.0. MS (ESI, <i>m</i>/<i>z</i>): 433.95 [M + H<sup>+</sup>] for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.66 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>S 434.1385, found 434.1389; <i>t</i><sub><i>R</i></sub> = 1.09 min.</div></div><div id="sec5_2_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (2<i>R</i>)-4-(6-((1-Aminocyclopropyl)buta-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-<i>N</i>-hydroxy-2-methyl-2-(methylsulfonyl)butanamide (<b>18g</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (2 g; 4.72 mmol) and bromo-alkyne <b>57</b> (1.05 g; 0.495 mmol) were reacted following general procedure E (77% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>18g</b> (0.542 g; 87% yield) as a yellowish solid; mp 255.1 °C (decomp). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (m, 1H), 9.20 (m, 1H), 7.53 (m, 1H), 6.24 (m, 1H), 4.43 (br s, 2H), 3.48 (m, 1H), 3.39 (m, 1H), 3.06 (s, 3H), 2.59 (m, 1H), 1.96 (m, 1H), 1.53 (s, 3H), 0.89–0.95 (m, 2H), 0.89–0.83 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.4, 130.9, 117.5, 109.0, 105.9, 89.9, 74.5, 72.6, 68.3, 63.3, 45.2, 39.5, 37.0, 31.0, 25.1, 18.8 (2C), 13.4. MS (ESI, <i>m</i>/<i>z</i>): 419.1 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>5</sub>S; <i>t</i><sub><i>R</i></sub> = 0.49 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>5</sub>S 419.1389, found 419.1389; <i>t</i><sub><i>R</i></sub> = 0.83 min.</div></div><div id="sec5_2_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (2<i>R</i>)-4-(6-((5<i>S</i>)-5,6-Dihydroxyhexa-1,3-diyn-1-yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-<i>N</i>-hydroxy-2-methyl-2(methylsulfonyl)butanamide (<b>18h</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.128 g; 0.30 mmol) and iodo-alkyne <b>58</b> (0.09 g; 0.42 mmol) were reacted following general procedure E (27% yield). The resulting product was then reacted following general procedure C (method A) to afford compound <b>18h</b> (0.01 g; 8% yield) as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.9 (br s, 1H), 9.19 (br s, 1H), 7.59 (s, 1H), 6.28 (d, <i>J</i> = 1.2 Hz, 1H), 5.66 (br s, 1H), 5.04 (br s, 1H), 4.43 (s, 2H), 4.31 (t, <i>J</i> = 6.1 Hz, 1H), 3.52–3.40 (overlaid m, 2H), 3.44 (d, <i>J</i> = 6.1 Hz, 2H), 3.06 (s, 3H), 2.57 (overlapped m,1H), 1.97 (m, 1H), 1.52 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 423.9 [M + H<sup>+</sup>] for C<sub>18</sub>H<sub>22</sub>N<sub>3</sub>O<sub>7</sub>S; <i>t</i><sub><i>R</i></sub> = 0.53 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>18</sub>H<sub>22</sub>N<sub>3</sub>O<sub>7</sub>S 424.1178, found 424.1187; <i>t</i><sub><i>R</i></sub> = 0.91 min.</div></div><div id="sec5_2_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (2<i>R</i>)-<i>N</i>-Hydroxy-2-methyl-4-(6-(5-(methylsulfonamido)penta-1,3-diyn-1yl)-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2(methylsulfonyl)butanamide (<b>18i</b>).<a onclick="showRef(event, 'cit25a'); return false;" href="javascript:void(0);" class="ref cit25a">(25a)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.165 g; 0.39 mmol) and bromo-alkyne <b>59</b> (0.124 g; 0.58 mmol) were reacted following general procedure E (68% yield). The resulting product was then reacted following general procedure C (method A) to afford compound <b>18i</b> (0.08 g; 65% yield) as a gray solid; mp 191.1 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.91 (s, 1H), 9.15 (s, 1H), 7.67 (t, <i>J</i> = 6.0 Hz, 1H), 7.62 (d, <i>J</i> = 0.7 Hz, 1H), 6.29 (d, <i>J</i> = 1.2 Hz, 1H), 4.44 (s, 2H), 4.04 (d, <i>J</i> = 5.9 Hz, 2H), 3.48 (m, 1H), 3.38 (m, 1H), 3.06 (s, 3H), 2.98 (s, 3H), 2.58 (m, 1H), 1.96 (m, 1H), 1.52 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 150.3, 131.0, 118.2, 108.2, 105.9, 80.2, 73.6, 73.5, 68.3, 56.5, 45.1, 40.8, 39.5, 36.9, 33.1, 30.9, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 470.94 [M + H<sup>+</sup>] for C<sub>18</sub>H<sub>23</sub>N<sub>4</sub>O<sub>7</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.63 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>18</sub>H<sub>23</sub>N<sub>4</sub>O<sub>7</sub>S<sub>2</sub> 471.1008, found 471.1016; <i>t</i><sub><i>R</i></sub> = 1.03 min.</div></div><div id="sec5_2_6_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (3<i>R</i>)-5-(2-(4-(Hydroxyamino)-3-methyl-3-(methylsulfonyl)-4-oxobutyl)-3-oxo-2,3-dihydro-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-6-yl)penta-2,4-diyn-1yl-4-hydroxypiperidine-1-carboxylate (<b>18j</b>).<a onclick="showRef(event, 'cit25b'); return false;" href="javascript:void(0);" class="ref cit25b">(25b)</a></h4><div class="NLM_p last">Alkyne <b>34</b> (0.200 g; 0.47 mmol) and iodo-alkyne <b>18j</b> (0.161 g; 0.52 mmol) were reacted following general procedure E (41% yield). The resulting product was then reacted following general procedure C (method A) to afford compound <b>18j</b> (0.077 g; 77% yield) as a gray solid; mp 101.3 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.97 (s, 1H), 9.21 (s, 1H), 7.66 (s, 1H), 6.32 (s, 1H), 4.87 (s, 2H), 4.79 (d, <i>J</i> = 4.0 Hz, 1H), 4.46 (s, 2H), 3.73–3.65 (m, 3H), 3.51 (m, 1H), 3.42 (overlaid m, 1H), 3.16–3.09 (overlaid m, 2H), 3.08 (s, 3H), 2.63 (overlaid m, 1H), 1.95 (m, 1H), 1.75–1.71 (m, 2H), 1.54 (s, 3H), 1.381.25 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 153.9, 150.3, 131.0, 118.4, 108.0, 105.9, 78.8, 74.8, 73.3, 70.5, 68.3, 65.7, 53.5, 45.2, 41.7 (2C), 39.5, 36.9, 34.4 (2C), 34.2, 30.9, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 521.07 [M + H<sup>+</sup>] for C<sub>23</sub>H<sub>29</sub>N<sub>4</sub>O<sub>8</sub>S; <i>t</i><sub><i>R</i></sub> = 0.66 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>23</sub>H<sub>29</sub>N<sub>4</sub>O<sub>8</sub>S 521.1706, found 521.1711; <i>t</i><sub><i>R</i></sub> = 1.10 min.</div></div><div id="sec5_2_6_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (2<i>R</i>)-<i>N</i>-Hydroxy-2-methyl-2-(methylsulfonyl)-4-(2-((4-(morpholinomethyl)phenyl)ethynyl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)butanamide (<b>19a</b>)</h4><div class="NLM_p last">Alkyne <b>40</b> (0.1 g; 0.22 mmol) and iodo-phenyl <b>41</b> (0.090 g; 0.29 mmol) were reacted following general procedure B (53% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>19a</b> (0.042 g; 66% yield) as a yellowish solid; mp 160.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (s, 1H), 9.18 (s, 1H), 7.57–7.54 (m, 3H), 7.40 (d, <i>J</i> = 8.3 Hz, 2H), 4.47 (s, 2H), 3.59–3.54 (m, 5H), 3.51 (s, 2H), 3.45 (m, 1H), 3.08 (s, 3H), 2.59 (m, 1H), 2.39–2.33 (m, 4H), 1.97 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.3, 152.1, 140.3, 135.6, 131.8 (2C), 130.5, 129.8 (2C), 127.5, 120.2, 96.1, 82.6, 68.5, 66.7 (2C), 62.4, 53.7 (2C), 49.2, 38.7, 37.0, 31.6, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 532.25 [M + H<sup>+</sup>] for C<sub>25</sub>H<sub>30</sub>N<sub>3</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.57 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>25</sub>H<sub>30</sub>N<sub>3</sub>O<sub>6</sub>S<sub>2</sub> 532.1576, found 532.1569; <i>t</i><sub><i>R</i></sub> = 0.44 min.</div></div><div id="sec5_2_6_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-((4-(2-hydroxypropan-2-yl)phenyl)ethynyl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>19b</b>)</h4><div class="NLM_p last">Alkyne <b>40</b> (0.14 g; 0.32 mmol) and iodo-phenyl <b>42</b> (0.055 g; 0.27 mmol) were reacted following general procedure B (51% yield). The resulting product was then reacted following general procedure D to afford compound <b>19b</b> (0.041 g; 52% yield) as a yellowish solid; mp 159.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (s, 1H), 9.17 (s, 1H), 7.56–7.51 (m, 5H), 5.15 (s, 1H), 4.50–4.42 (m, 2H), 3.55 (m, 1H), 3.45 (m, 1H), 3.07 (s, 3H), 2.58 (m, 1H), 1.96 (m, 1H), 1.54 (s, 3H), 1.42 (s, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.3, 152.7, 152.1, 135.5, 131.4 (2C), 130.7, 127.4, 125.6 (2C), 119.1, 96.4, 82.2, 71.2, 68.5, 49.2, 38.7, 32.2 (2C), 31.6 (s), 13.4. MS (ESI, <i>m</i>/<i>z</i>): 491.21 [M + H<sup>+</sup>] for C<sub>23</sub>H<sub>27</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.76 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>23</sub>H<sub>27</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub> 491.1310, found 491.1314; <i>t</i><sub><i>R</i></sub> = 0.77 min.</div></div><div id="sec5_2_6_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-((4-(3-hydroxyoxetan-3-yl)phenyl)ethynyl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>19c</b>)</h4><div class="NLM_p last">Alkyne <b>40</b> (0.07 g; 0.16 mmol) and iodo-phenyl <b>43</b> (0.064 g; 0.23 mmol) were reacted following general procedure B (42% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>19c</b> (0.024 g; 81% yield) as a yellowish solid; mp 150.7 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (s, 1H), 9.18 (s, 1H), 7.71–7.68 (m, 2H), 7.65–7.62 (m, 2H), 7.56 (s, 1H), 6.52 (s, 1H), 4.79 (d, <i>J</i> = 6.9 Hz, 2H), 4.68 (d, <i>J</i> = 6.8 Hz, 2H), 4.47 (s, 2H), 3.57 (m, 1H), 3.45 (m, 1H), 3.08 (s, 3H), 2.61 (m, 1H), 1.97 (m, 1H), 1.55 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.2, 152.1, 146.1, 135.7, 131.8 (2C), 130.4, 127.6, 125.5 (2C), 120.3, 96.0, 85.5 (2C), 82.7, 74.2, 68.5, 49.2, 38.7, 37.0, 31.5, 13.3. MS (ESI, <i>m</i>/<i>z</i>): 505.21 [M + H<sup>+</sup>] for C<sub>23</sub>H<sub>25</sub>N<sub>2</sub>O<sub>7</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.69 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>23</sub>H<sub>25</sub>N<sub>2</sub>O<sub>7</sub>S<sub>2</sub> 505.1103, found 505.1106; <i>t</i><sub><i>R</i></sub> = 1.07 min.</div></div><div id="sec5_2_6_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (2<i>R</i>)-4-(2-((4-((2<i>S</i>)-1,2-Dihydroxyethyl)phenyl)ethynyl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-<i>N</i>-hydroxy-2-methyl-2-(methylsulfonyl)butanamide (<b>19d</b>)</h4><div class="NLM_p last">Alkyne <b>40</b> (0.11 g; 0.24 mmol) and iodo-phenyl <b>48</b> (0.088 g; 0.33 mmol) were reacted following general procedure B (35% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>19d</b> (0.022 g; 52% yield) as a yellowish solid; mp 148.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (s, 1H), 9.18 (s, 1H), 7.57–7.54 (m, 3H), 7.40 (d, <i>J</i> = 8.3 Hz, 2H), 4.47 (s, 2H), 3.59–3.54 (m, 5H), 3.51 (s, 2H), 3.45 (m, 1H), 3.08 (s, 3H), 2.59 (m, 1H), 2.39–2.33 (m, 4H), 1.97 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.3, 152.1, 145.8, 135.6, 131.4 (2C), 130.6, 127.4, 127.3 (2C), 119.9, 96.3, 82.3, 73.9, 68.5, 67.6, 49.2, 38.7, 37.0, 31.6, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 493.15 [M + H<sup>+</sup>] for C<sub>22</sub>H<sub>25</sub>N<sub>2</sub>O<sub>7</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.61 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>22</sub>H<sub>25</sub>N<sub>2</sub>O<sub>7</sub>S<sub>2</sub> 493.1103, found 493.1105; <i>t</i><sub><i>R</i></sub> = 0.99 min.</div></div><div id="sec5_2_6_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-((3-hydroxyoxetan-3-yl)buta-1,3-diyn-1-yl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>20a</b>)</h4><div class="NLM_p last">Alkyne <b>40</b> (0.110 g; 0.25 mmol) and iodo-alkyne <b>51</b> (0.092 g; 0.41 mmol) were reacted following general procedure C (68% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>20a</b> (0.042 g; 57% yield) as a yellowish solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (s, 1H), 9.16 (s, 1H), 7.69 (s, 1H), 6.87 (s, 1H), 4.73 (d, <i>J</i> = 7.0 Hz, 2H), 4.57 (d, <i>J</i> = 7.0 Hz, 2H), 4.46 (s, 2H), 3.56 (m, 1H), 3.45 (m, 1H), 3.07 (s, 3H), 2.59 (m, 1H), 1.95 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 162.9, 151.8, 137.3, 130.6, 128.2, 88.1, 83.7 (2C), 79.6, 72.5, 68.5, 66.5, 49.2, 38.8, 37.0, 31.5, 21.3, 13.4. MS (ESI, <i>m</i>/<i>z</i>): 453.13 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>21</sub>N<sub>2</sub>O<sub>7</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.63 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>19</sub>H<sub>21</sub>N<sub>2</sub>O<sub>7</sub>S<sub>2</sub> 453.0790, found 453.0800; <i>t</i><sub><i>R</i></sub> = 1.00 min.</div></div><div id="sec5_2_6_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-(5-hydroxy-5-methylhexa-1,3-diyn-1-yl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>20b</b>)</h4><div class="NLM_p last">Alkyne <b>40</b> (0.285 g; 0.586 mmol) and crude bromo-alkyne <b>52</b> (0.143 g; 0.64 mmol, theoritical) were reacted following general procedure C (51% yield). The resulting product was then reacted following general procedure D (method A) to afford compound <b>20b</b> (0.048 g; 37% yield) as a beige solid; mp 142.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.94 (s, 1H), 9.17 (s, 1H), 7.55–7.52 (m, 3H), 7.43–7.42 (m, 2H), 5.38 (d, <i>J</i> = 4.4 Hz, 1H), 4.78 (t, <i>J</i> = 6.0 Hz, 1H), 4.57 (m, 1H), 4.46 (s, 2H), 3.56 (m, 1H), 3.48–3.40 (m, 3H), 3.07 (s, 3H), 2.59 (m, 1H), 1.96 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.0, 151.8, 136.8, 130.1, 128.7, 94.1, 80.2, 71.0, 68.5, 65.1, 64.5, 49.2, 38.8, 37.0, 31.5, 31.3 (2C), 13.4. MS (ESI, <i>m</i>/<i>z</i>): 439.17 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.70 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>19</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub> 439.0997, found 439.0999; <i>t</i><sub><i>R</i></sub> = 1.11 min.</div></div><div id="sec5_2_6_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-((1-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>20c</b>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h4><div class="NLM_p last">Alkyne <b>40</b> (0.099 g; 0.22 mmol) and crude bromo-alkyne <b>54</b> (0.104 g; 0.25 mmol, theritical) were reacted following general procedure D (61% yield). The resulting product was then reacted following general procedures E and D (method C) to afford compound <b>20c</b> (0.027 g; 58% yield) as a yellowish solid; mp 160.9 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.92 (s, 1H), 9.16 (s, 1H), 7.57 (s, 1H), 5.07 (t, <i>J</i> = 6.0 Hz, 1H), 4.43 (s, 2H), 3.55 (m, 1H), 3.46–3.38 (overlapped m, 1H), 3.39 (d, <i>J</i> = 5.9 Hz, 2H), 3.05 (s, 3H), 2.62–2.43 (overlapped m, 1H), 1.94 (m, 1H), 1.52 (s, 3H), 0.99–0.95 (m, 2H), 0.94–0.90 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.0, 151.8, 136.4, 129.7, 129.3, 94.5, 81.5, 68.5, 67.5, 65.7, 61.4, 49.2, 38.8, 37.0, 31.5, 15.8, 14.0 (2C), 13.4. MS (ESI, <i>m</i>/<i>z</i>): 451.13 [M + H<sup>+</sup>] for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.70 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub> 451.0997, found 451.1005; <i>t</i><sub><i>R</i></sub> = 1.10 min.</div></div><div id="sec5_2_6_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-(((1<i>R</i>,2<i>R</i>)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>20d</b>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h4><div class="NLM_p last">Alkyne <b>40</b> (0.08 g; 0.18 mmol) and bromo-alkyne <b>56</b> (0.049 g; 0.22 mmol) were reacted following general procedure C (96% yield). The resulting product was then reacted following general procedures F (55% yield) and D (method A) to afford compound <b>20e</b> (0.27 g; 63% yield) as a yellowish solid; mp 135.1 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.95 (br s, 1H), 9.20 (br s, 1H), 7.52 (s, 1H), 6.23 (s, 1H), 4.70 (t, <i>J</i> = 5.6 Hz, 1H), 4.42 (s, 2H), 3.48 (m, 1H), 3.43–3.37 (overlaid m, 2H), 3.24 (m, 1H), 3.06 (s, 3H), 2.59 (m, 1H), 1.96 (m, 1H), 1.51 (s, 3H), 1.43–1.35 (m, 2H), 0.92–0.82 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.8, 163.0, 151.8, 136.3, 129.7, 129.3, 93.2, 81.5, 68.4, 67.1, 62.3, 60.3, 49.1, 38.7, 36.9, 31.5, 26.0, 13.8, 13.3, 5.1. MS (ESI, <i>m</i>/<i>z</i>): 451.0 [M + H<sup>+</sup>] for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.66 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub> 451.0997, found 451.1002; <i>t</i><sub><i>R</i></sub> = 1.09 min.</div></div><div id="sec5_2_6_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (2<i>R</i>)-<i>N</i>-Hydroxy-4-(2-(((1<i>S</i>,2<i>S</i>)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanamide (<b>20e</b>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h4><div class="NLM_p last">Alkyne <b>40</b> (0.297 g; 0.67 mmol) and bromo-alkyne <b>55</b> (0.156 g; 0.72 mmol) were reacted following general procedure C (79% yield). The resulting product (0.093 g; 0.17 mmol) was then reacted following general procedure D (method A) to afford compound <b>20d</b> (0.043 g; 56% yield) as a yellowish solid; mp 124.6 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.92 (s, 1H), 9.14 (s, 1H), 7.56 (s, 1H), 4.72 (t, <i>J</i> = 5.7 Hz, 1H), 4.43 (s, 2H), 3.54 (m, 1H), 3.46–3.39 (m, 2H), 3.26 (m, 1H), 3.05 (s, 3H), 2.54 (overlaid m, 1H), 1.94 (m, 1H), 1.56–1.43 (overlaid m, 2H), 1.52 (s, 3H), 0.97 (m, 1H), 0.92 (m, 1H). MS (ESI, <i>m</i>/<i>z</i>): 451.09 [M + H<sup>+</sup>] for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.65 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>20</sub>H<sub>23</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub> 451.0997, found 451.1011; <i>t</i><sub><i>R</i></sub> = 1.09 min.</div></div><div id="sec5_2_6_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (2<i>R</i>)-4-(2-((1-Aminocyclopropyl)buta-1,3-diyn-1-yl)-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-<i>N</i>-hydroxy-2-methyl-2-(methylsulfonyl)butanamide (<b>20f</b>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h4><div class="NLM_p last">Alkyne <b>40</b> (0.091 g; 0.21 mmol) and bromo-alkyne <b>57</b> (0.048 g; 0.24 mmol) were reacted following general procedure C (74% yield). The resulting product was then reacted following general procedure D (method C) to afford compound <b>20f</b> (0.027 g; 42% yield) as a yellowish solid; mp 135.0 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.92 (s, 1H), 9.15 (s, 1H), 7.57 (s, 1H), 4.43 (s, 2H), 3.54 (m, 1H), 3.43 (m, 1H), 3.05 (s, 3H), 2.57 (ddd, <i>J</i> = 6.6, 9.5, 13.2 Hz, 1H), 1.94 (m, 1H), 1.52 (s, 3H), 1.01–0.98 (m, 2H), 0.91–0.87 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 163.9, 163.0, 151.8, 136.4, 129.6, 129.2, 95.2, 81.1, 69.6, 68.4, 62.4, 49.1, 38.7, 36.9, 31.5, 25.1, 19.1 (2C), 13.4. MS (ESI, <i>m</i>/<i>z</i>): 476.99 [M + MeCN<sup>+</sup>] for C<sub>21</sub>H<sub>24</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.50 min. LC-HRMS (ESI, <i>m</i>/<i>z</i>): [M + H<sup>+</sup>] calcd for C<sub>19</sub>H<sub>22</sub>N<sub>3</sub>O<sub>5</sub>S<sub>2</sub> 436.1000, found 436.1007; <i>t</i><sub><i>R</i></sub> = 0.82 min.</div></div><div id="sec5_2_6_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>tert</i>-Butyl (2<i>R</i>)-4-(5-Bromo-1-oxoisoindolin-2-yl)-2-methyl-2(methylsulfonyl)butanoate (<b>23</b>)</h4><div class="NLM_p last">NaBH(OAc)<sub>3</sub> (2.4 g; 11.14 mmol) was added to a solution of amine <b>22</b> (see preparation in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">SI</a>, 1.8 g; 7.15 mmol), methyl 4-bromo-2-formylbenzoate <b>21</b> (2.12 g, 8.6 mmol), and AcOH (0.431 mL, 7.17 mmol) in DCM (50 mL). The reaction mixture was stirred at rt for 24 h. DCM (100 mL) and satd NaHCO<sub>3</sub> (100 mL) were added. The two layers were separated. The evaporation residue was purified via CC (heptane–EtOAc gradient) to afford ester <b>23</b> (2.84 g; 89% yield) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 7.90 (d, <i>J</i> = 1.0 Hz, 1H), 7.67 (dd, <i>J</i> = 1.7, 8.1 Hz, 1H), 7.60 (m, 1H), 4.53–4.43 (m, 2H), 3.67 (m, 1H), 3.54 (m, 1H), 3.11 (s, 3H), 2.51 (overlaid m, 1H), 2.05 (m, 1H), 1.53 (s, 3H), 1.33 (s, 9H). MS (ESI, <i>m</i>/<i>z</i>): 446.17/448.15 [M + H<sup>+</sup>] for C<sub>18</sub>HNOBrS; <i>t</i><sub><i>R</i></sub> = 0.94 min.</div></div><div id="sec5_2_6_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>tert</i>-Butyl (2<i>R</i>)-4-(5-Iodo-1-oxoisoindolin-2-yl)-2-methyl-2(methylsulfonyl)butanoate (<b>24</b>)</h4><div class="NLM_p last">To a solution of <b>23</b> (2.84 g; 6.38 mmol) in dioxane (20 mL) were added <i>trans</i>-<i>N</i>,<i>N</i>′-dimethylcyclohexane-1,2-diamine (0.6 mL; 3.8 mmol), NaI (2.91 g; 19.39 mmol), and CuI (0.37 g; 1.935 mmol). The reaction mixture was refluxed for 16 h. After cooling, the solvent was evaporated and the residue was taken up in water (200 mL) and EtOAc (250 mL). The aq layer was extracted with EtOAc (2 × 200 mL). The evaporation residue was purified via CC (heptane–EtOAc gradient) to afford iodide <b>24</b> (1.94 g; 62% yield) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 8.07 (d, <i>J</i> = 0.7 Hz, 1H), 7.85 (dd, <i>J</i> = 1.4, 7.9 Hz, 1H), 7.46 (d, <i>J</i> = 7.9 Hz, 1H), 4.48–4.44 (m, 2H), 3.67 (m, 1H), 3.53 (m, 1H), 3.12 (s, 3H), 2.51 (overlaid m, 1H), 2.05 (m, 1H), 1.53 (s, 3H), 1.34 (s, 9H). MS (ESI, <i>m</i>/<i>z</i>): 493.9 [M + H<sup>+</sup>] for C<sub>18</sub>H<sub>24</sub>NO<sub>5</sub>IS; <i>t</i><sub><i>R</i></sub> = 0.87 min.</div></div><div id="sec5_2_6_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (2<i>R</i>)-4-(5-Iodo-1-oxoisoindolin-2-yl)-2-methyl-2-(methylsulfonyl)butanoic Acid (<b>25</b>)</h4><div class="NLM_p last">To an ice-chilled solution of <b>24</b> (1.93 g; 3.91 mmol) in DCM (10 mL) were added Et<sub>3</sub>SiH (0.69 mL, 4.32 mmol) and TFA (8.7 mL; 114 mmol). After stirring for 4 h, the reaction mixture was cooled to 0 °C and Et<sub>2</sub>O (20 mL) was added dropwise. The solid was filtered off and scarcely washed with Et<sub>2</sub>O to afford, after drying to a constant weight, acid <b>25</b> (1.47 g; 86% yield) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 13.69 (br s, 1H), 8.05 (d, <i>J</i> = 0.7 Hz, 1H), 7.84 (dd, <i>J</i> = 1.3, 7.9 Hz, 1H), 7.45 (d, <i>J</i> = 7.9 Hz, 1H), 4.48–4.39 (m, 2H), 3.71 (m, 1H), 3.55 (m, 1H), 3.11 (s, 3H), 2.57–2.47 (overlapped m, 1H), 2.04 (m, 1H), 1.54 (s, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ: 169.9, 167.1, 144.7, 137.1, 132.8, 132.1, 124.6, 99.3, 69.6, 49.3, 37.9, 37.2, 31.3, 14.7. MS (ESI, <i>m</i>/<i>z</i>): 437.86 [M + H<sup>+</sup>] for C<sub>14</sub>H<sub>16</sub>NO<sub>5</sub>IS; <i>t</i><sub><i>R</i></sub> = 0.68 min.</div></div><div id="sec5_2_6_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (2<i>R</i>)-4-(5-Iodo-1-oxoisoindolin-2-yl)-2-methyl-2-(methylsulfonyl)-<i>N</i>-((2<i>RS</i>)-(tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>26</b>)</h4><div class="NLM_p last">Acid <b>25</b> (0.798 g; 1.83 mmol) was reacted following general procedure A (method A). After purification via CC (heptane–EtOAC gradient), <b>26</b> (0.982 g; >95% yield, ∼90% purity, unknown contaminant) was obtained as a yellowish foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) mixture of diastereomers δ: 11.37 (s, 1H), 11.34 (s, 1H), 8.05 (d, <i>J</i> = 5.0 Hz, 2H), 7.84 (m, 2H), 7.44 (dd, <i>J</i> = 3.7, 7.9 Hz, 2H), 4.85 (s, 1H), 4.51–4.40 (m, 5H), 4.02 (m, 1H), 3.93 (m, 1H), 3.64–3.54 (m, 3H), 3.46–3.34 (m, 3H), 3.06 (s, 3H), 3.03 (s, 3H), 2.67–2.45 (overlaid m, 2H), 1.99–1.93 (m, 2H), 1.65–1.59 (m, 4H), 1.57 (s, 3H), 1.55 (s, 3H), 1.57–1.40 (overlaid m, 8H). MS (ESI, <i>m</i>/<i>z</i>): 536.97 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>25</sub>N<sub>2</sub>O<sub>6</sub>IS; <i>t</i><sub><i>R</i></sub> = 0.77 min.</div></div><div id="sec5_2_6_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (2<i>R</i>)-2-Methyl-2-(methylsulfonyl)-4-(1-oxo-5-((trimethylsilyl)ethynyl)isoindolin-2-yl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>28</b>)</h4><div class="NLM_p last">To a mixture of iodide <b>26</b> (0.982 g; 1.83 mmol, theoritical), CuI (0.073 g; 0.38 mmol), and PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (0.14 g; 0.20 mmol) in degassed THF (12 mL) were added trimethylsilylacetylene <b>27</b> (0.52 mL; 3.65 mmol) and TEA (1.1 mL; 7.89 mmol). The reaction proceeded at 50 °C for 1 h. The reaction mixture was concentrated to dryness, and the residue was purified via CC (heptane–EtOAc–MeOH gradient) to afford TMS–alkyne <b>28</b> (0.767 g, 83% yield) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) mixture of diastereomers δ: 11.38 (s, 1H), 11.35 (s, 1H), 7.71 (m, 2H), 7.64–7.62 (m, 2H), 7.53 (m, 2H), 4.86 (m, 1H), 4.59 (m, 1H), 4.54–4.41 (m, 4H), 4.03 (m, 1H), 3.95 (m, 1H), 3.63 (m, 1H), 3.57 (m, 2H), 3.45–3.39 (m, 3H), 3.06 (s, 3H), 3.04 (s, 3H), 2.67–2.47 (overlaid m, 2H), 1.98 (m, 2H), 1.63–1.60 (m, 4H), 1.58 (s, 3H), 1.56 (s, 3H), 1.58–1.42 (overlaid m, 8H), 0.25 (s, 18H). MS (ESI, <i>m</i>/<i>z</i>): 507.06 [M + H<sup>+</sup>] for C<sub>24</sub>H<sub>34</sub>N<sub>2</sub>O<sub>6</sub>SSi; <i>t</i><sub><i>R</i></sub> = 0.90 min.</div></div><div id="sec5_2_6_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (2<i>R</i>)-4-(5-Ethynyl-1-oxoisoindolin-2-yl)-2-methyl-2-(methylsulfonyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>29</b>)</h4><div class="NLM_p last">To a solution of compound <b>28</b> (0.762 g; 1.5 mmol) in MeOH (8.8 mL) was added K<sub>2</sub>CO<sub>3</sub> (0.378 g; 2.74 mmol). The reaction proceeded at rt for 1 h. The reaction mixture was concentrated to dryness, and the residue was directly subjected to CC (heptane–EtOAc–MeOH gradient) to afford alkyne <b>29</b> (0.587 g, 79% yield) as a beige foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) mixture of diastereomers δ: 11.36 (d, <i>J</i> = 16.0 Hz, 2H), 7.74 (d, <i>J</i> = 3.4 Hz, 2H), 7.66 (dd, <i>J</i> = 7.8, 3.3 Hz, 2H), 7.57 (d, <i>J</i> = 7.8 Hz, 2H), 4.86 (s, 1H), 4.55 (m, 1H), 4.51 (m, 1H), 4.49 (m, 2H), 4.44–4.48 (m, 2H), 4.40 (s, 2H), 3.95 (m, 1H), 3.64 (m, 1H), 3.59 (t, <i>J</i> = 7.3 Hz, 2H), 3.49–3.38 (m, 3H), 3.07 (s, 3H), 3.04 (s, 3H), 2.68–2.55 (m, 2H), 2.00 (s, 2H), 1.64 (s, 3H), 1.59 (s, 3H), 1.58 (s, 3H),1.54–1.42 (m, 8H). MS (ESI, <i>m</i>/<i>z</i>): 435.01 [M + H<sup>+</sup>] for C<sub>21</sub>H<sub>26</sub>N<sub>2</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.74 min.</div></div><div id="sec5_2_6_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>tert</i>-Butyl (2<i>R</i>)-4-(((4-Iodo-1<i>H</i>-pyrrol-2-yl)methyl)amino)-2-methyl-2-(methylsulfonyl)butanoate (<b>31</b>)</h4><div class="NLM_p last">To a solution of aldehyde <b>30</b> (8.53 g; 38.6 mmol) in MeOH (192 mL) was added amine <b>22</b> (9.47 g; 37.7 mmol). After 2 h stirring at rt, the solids were filtered off, and MeOH (25 mL) was added to the filtrate. The solution was ice-chilled, and NaBH<sub>4</sub> (1.45 g; 37.7 mmol) was added. The reaction proceeded 1 h at 0 °C. Water (20 mL) was added, and the volatiles were removed in vacuo. The residue dissolved in DCM (150 mL). The organic layer was washed with satd aq NaHCO<sub>3</sub> (50 mL), water (50 mL), and brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to dryness to afford crude <b>31</b> (17.2 g) as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 10.88 (s, 1H), 6.76 (m, 1H), 5.97 (m, 1H), 3.63–3.49 (m, 2H), 3.06 (s, 3H), 2.57 (overlaid m, 1H), 2.37 (m, 1H), 2.29 (m, 1H), 2.00 (br s, 1H), 1.80 (m, 1H), 1.40 (s, 9H), 1.38 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 456.89 [M + H<sup>+</sup>] for C<sub>15</sub>H<sub>25</sub>N<sub>2</sub>O<sub>4</sub>IS; <i>t</i><sub><i>R</i></sub> = 0.68 min.</div></div><div id="sec5_2_6_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>tert</i>-Butyl (2<i>R</i>)-4-(6-Iodo-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)butanoate (<b>32</b>)</h4><div class="NLM_p last">To a solution of amine <b>31</b> (17.2 g) in THF (150 mL) were added at rt CDI (5.99 g, 33.2 mmol) and NaH (60% in mineral oil, 0.340 g; 8.45 mmol). The reaction proceeded at rt for 4 h. Water (90 mL), satd NH<sub>4</sub>Cl (90 mL), and EtOAc (150 mL) were added. The organic layer was washed with brine (40 mL). The evaporation residue was purified via CC to afford iodide <b>32</b> (8.35 g; 57% yield) as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 7.33 (d, <i>J</i> = 1.0 Hz, 1H), 6.23 (d, <i>J</i> = 1.2 Hz, 1H), 4.41 (m, 2H), 3.55 (m, 1H), 3.46 (m, 1H), 3.11 (s, 3H), 2.54–2.47 (overlapped m, 1H), 2.04 (m, 1H), 1.51 (s, 3H), 1.34 (s, 9H). MS (ESI, <i>m</i>/<i>z</i>): 482.97 [M + H<sup>+</sup>] for C<sub>16</sub>H<sub>23</sub>N<sub>2</sub>O<sub>5</sub>IS; <i>t</i><sub><i>R</i></sub> = 0.88 min</div></div><div id="sec5_2_6_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (2<i>R</i>)-2-Methyl-2-(methylsulfonyl)-4-(3-oxo-6-((trimethylsilyl)ethynyl)-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>33</b>)</h4><div class="NLM_p last"><i>t</i>-Butyl ester <b>32</b> (40 g; 82.9 mmol) was reacted successively following the procedures described for the preparation of intermediate <b>25</b> (95% yield), general procedure A (method B) (90% yield), and procedure described for the preparation of intermediate <b>28</b> (performed in two identical batches, average yield 75%), to afford compound <b>33</b> (23 g) as a yellowish foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) (mixture of epimers) δ: 11.38 (s, 1H), 11.33 (s, 1H), 7.39 (d, <i>J</i> = 0.9 Hz, 1H), 7.38 (d, <i>J</i> = 0.9 Hz, 1H), 6.19 (d, <i>J</i> = 1.3 Hz, 1H), 6.18 (d, <i>J</i> = 1.3 Hz, 1H), 4.87 (m, 1H), 4.47 (m, 1H), 4.44–4.37 (m, 4H), 4.03 (m, 1H), 3.96 (m, 1H), 3.55–3.36 (m, 6H), 3.07 (s, 3H), 3.04 (s, 3H), 2.62 (m, 2H), 1.99 (m, 2H), 1.71–1.61 (m, 4H), 1.45–1.59 (overlapped m, 8H), 1.56 (s, 3H), 1.55 (s, 3H), 0.20 (s, 18H). MS (ESI, <i>m</i>/<i>z</i>): 496.07 [M + H<sup>+</sup>] for C<sub>22</sub>H<sub>33</sub>N<sub>3</sub>O<sub>6</sub>SSi; <i>t</i><sub><i>R</i></sub> = 0.90 min.</div></div><div id="sec5_2_6_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (2<i>R</i>)-4-(6-Ethynyl-3-oxo-1<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-2(3<i>H</i>)-yl)-2-methyl-2-(methylsulfonyl)-<i>N</i>-((2<i>RS</i>)-tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>34</b>)</h4><div class="NLM_p last">To a solution of TMS–alkyne <b>33</b> (23 g; 46.4 mmol) in THF (180 mL) was added TBAF (1 M in THF, 46.5 mL; 46.5 mmol). The reaction proceeded at rt for 45 min. Water (200 mL) was added, and the volatiles were removed in vacuo. The residue was diluted with EtOAc (200 mL). The two layers were separated, and the aq phase was extracted with EtOAc (2 × 100 mL). The evaporation residue was purified via CC (DCM–MeOH) to afford alkyne <b>34</b> (19.7 g) as a yellowish foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) (mixture of diastereomers) δ: 11.37 (m, 1H), 11.33 (m, 1H), 7.40 (d, <i>J</i> = 0.7 Hz, 1H), 7.39 (d, <i>J</i> = 0.6 Hz, 1H), 6.19 (m, 2H), 4.86 (m, 1H), 4.47 (m, 1H), 4.45–4.36 (m, 4H), 4.03 (m, 1H), 3.98 (s, 2H), 3.95 (m, 1H), 3.56–3.36 (m, 6H), 3.07 (s, 3H), 3.04 (s, 3H), 2.63 (m, 2H), 1.98 (m, 2H), 1.70–1.60 (m, 4H), 1.59–1.45 (overlaid m, 8H), 1.57 (s, 3H), 1.55 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 424.02 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>25</sub>N<sub>3</sub>O<sub>6</sub>S; <i>t</i><sub><i>R</i></sub> = 0.74 min.</div></div><div id="sec5_2_6_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>tert</i>-Butyl (2<i>R</i>)-2-Methyl-2-(methylsulfonyl)-4-(6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)butanoate (<b>36</b>)</h4><div class="NLM_p last">To an ice-chilled solution of thiophenedicarboxaldehyde <b>35</b> (4.69 g; 32.5 mmol) and amine <b>22</b> (8.16 g; 32.5 mmol) in DCM (280 mL) was added AcOH (8 mL; 140 mmol). The reaction proceeded at rt for 3 h 30 min. The solvent was removed in vacuo, and the residue was purified via CC (heptane–EtOAC–MeOH gradient) to afford compound <b>36</b> as a gray solid (9 g; 74% yield). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 7.96 (d, <i>J</i> = 4.7 Hz, 1H), 7.23 (d, <i>J</i> = 4.7 Hz, 1H), 4.46–4.36 (m, 2H), 3.62 (m, 1H), 3.52 (m, 1H), 3.12 (s, 3H), 2.56–2.47 (overlaid m, 1H), 2.04 (m, 1H), 1.53 (s, 3H), 1.34 (s, 9H). MS (ESI, <i>m</i>/<i>z</i>): 373.8 [M + H<sup>+</sup>] for C<sub>16</sub>H<sub>23</sub>NO<sub>5</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.76 min.</div></div><div id="sec5_2_6_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>tert</i>-Butyl (2<i>R</i>)-4-(2-Bromo-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanoate (<b>37</b>)</h4><div class="NLM_p last">To suspension of intermediate <b>36</b> (8.34 g, 22.3 mmol) in AcOH (45 mL) and water (20 mL), cooled at 0 °C, was added dropwise Br<sub>2</sub> (1.5 mL, 29.1 mmol). The reaction proceeded for 16 h at rt. Water (50 mL) was added, and the mixture was extracted with EtOAc (2 × 75 mL). The combined extracts were washed with 15% aq NaHSO<sub>4</sub> (50 mL), 10% aq NaHSO<sub>3</sub> (50 mL), satd aq NaHCO<sub>3</sub> (50 mL), and brine (50 mL). The evaporation residue was purified via CC (heptane–EtOAc–MeOH gradient) to afford bromide <b>37</b> (4.65g, 46% yield) as a yellowish solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 7.48 (s, 1H), 4.46–4.37 (m, 2H), 3.60 (m, 1H), 3.50 (m, 1H), 3.11 (s, 3H), 2.53 (overlaid m, 1H), 2.03 (m, 1H), 1.52 (s, 3H), 1.36 (s, 9H). MS (ESI, <i>m</i>/<i>z</i>): 451.88/453.88 [M + H<sup>+</sup>] for C<sub>16</sub>H<sub>22</sub>NO<sub>5</sub>BrS<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.84 min.</div></div><div id="sec5_2_6_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (2<i>R</i>)-4-(2-Bromo-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)butanoic Acid (<b>38</b>)</h4><div class="NLM_p last">To a mixture of intermediate <b>37</b> (4.94 g; 10.9 mmol) in 4 N HCl in dioxane (100 mL) was added water (2.4 mL). The reaction proceeded for 16 h at rt. The resulting slurry was diluted with Et<sub>2</sub>O (100 mL) and filtered. The solid was dried to a constant weight to afford crude acid <b>38</b> (4.57 g) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ: 13.74 (m, 1H), 7.48 (s, 1H), 4.46–4.37 (m, 2H), 3.65 (m, 1H), 3.52 (m, 1H), 3.12 (s, 3H), 2.52 (overlaid m, 1H), 2.03 (m, 1H), 1.54 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 395.78/397.8 [M + H<sup>+</sup>] for C<sub>12</sub>H<sub>14</sub>NO<sub>5</sub>BrS<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.64 min.</div></div><div id="sec5_2_6_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (2<i>R</i>)-2-Methyl-2-(methylsulfonyl)-4-(6-oxo-2-((trimethylsilyl)ethynyl)-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-<i>N</i>-((2<i>RS</i>)tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>39</b>)</h4><div class="NLM_p last">Acid <b>38</b> (4.32 g; 10.9 mmol, theoritical) was reacted with NH<sub>2</sub>OTHP (4.78 g; 38.8 mmol) following general procedure A (method A) (81% yield). The resulting product was then reacted following the procedure described for the preparation of <b>28</b>. After purification via CC (heptane–EtOAc–MeOH gradient), TMS–alkyne <b>39</b> (3.67 g, 82% yield) was obtained as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) (mixture of isomers) δ: 11.36 (s, 1H), 11.33 (m, 1H), 7.49 (s, 1H), 7.49 (s, 1H), 4.87 (m, 1H), 4.66 (m, 1H), 4.49–4.36 (m, 4H), 4.02 (m, 2H), 3.63–3.38 (m, 6H), 3.07 (s, 3H), 3.04 (s, 3H), 2.58 (m, 2H), 1.98 (m, 2H), 1.71–1.61 (m, 4H), 1.61–1.45 (overlaid m, 8H), 1.56 (s, 3H), 1.55 (s, 3H), 0.26 (s, 18H). MS (ESI, <i>m</i>/<i>z</i>): 513.0 [M + H<sup>+</sup>] for C<sub>22</sub>H<sub>32</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>Si; <i>t</i><sub><i>R</i></sub> = 0.90 min.</div></div><div id="sec5_2_6_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (2<i>R</i>)-4-(2-Ethynyl-6-oxo-4,6-dihydro-5<i>H</i>-thieno[2,3-<i>c</i>]pyrrol-5-yl)-2-methyl-2-(methylsulfonyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)butanamide (<b>40</b>)</h4><div class="NLM_p last">To a solution of compound <b>39</b> (3.67 g, 7.16 mmol) in THF (19 mL) was added TBAF (1 M in THF, 7.2 mL). The reaction proceeded at rt for 1 h. The solvent was removed in vacuo. The residue was purified via CC (DCM–MeOH gradient) to afford alkyne <b>40</b> (2.9 g, 92% yield) as a yellow foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) (mixture of isomers) δ: 11.36 (s, 1H), 11.33 (m, 1H), 7.51 (s, 1H), 7.51 (s, 1H), 4.88 (s, 2H), 4.87 (m, 1H), 4.62 (m, 1H), 4.49–4.37 (m, 4H), 4.04 (m, 1H), 3.97 (m, 1H), 3.62–3.51 (m, 3H), 3.39–3.47 (m, 3H), 3.07 (s, 3H), 3.04 (s, 3H), 2.61 (m, 2H), 1.98 (m, 2H), 1.61–1.71 (m, 4H), 1.44–1.60 (overlaid m, 8H), 1.57 (s, 3H), 1.55 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 440.95 [M + H<sup>+</sup>] for C<sub>19</sub>H<sub>24</sub>N<sub>2</sub>O<sub>6</sub>S<sub>2</sub>; <i>t</i><sub><i>R</i></sub> = 0.72 min.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i73"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01604" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61863" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61863" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01604?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01604</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental details on methods, data collection and refinements statistics for the structures of compounds <b>8</b>, <b>10</b>, <b>12</b>, <b>17a</b>, and <b>18d</b>; MIC testing, log <i>D</i>, and solubility determination; MICs determined against PAO1 and PA2616 strains for compounds <b>7</b>–<b>20</b>; preparation of compounds <b>8</b>–<b>14</b>, <b>22</b>, and <b>41</b>–<b>60</b>; preparation of recombinant <i>Pseudomonas aeruginosa</i> (Pae) and <i>Escherichia coli</i> (Eco) LpxC proteins for X-ray and DSF; neutropenic murine thigh infection model against <i>Escherichia coli</i>, graphical representations and descriptive statistics; pharmacokinetics in the mouse; views of compounds <b>10</b>, <b>12</b>, and <b>18d</b> bound to Pae LpxC (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings(<a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_001.csv">jm9b01604_si_001.csv (4.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf">jm9b01604_si_002.pdf (1.6 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>8</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I46">6I46</a>; <b>10</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I47">6I47</a>; <b>12</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I48">6I48</a>; <b>17a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I49">6I49</a>; <b>18d</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I4A">6I4A</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01604" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56155" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56155" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jean-Philippe Surivet</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3505-2025" title="Orcid link">http://orcid.org/0000-0002-3505-2025</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#751f10141b58051d1c191c0505105b0600071c031001351c111a07061c145b161a18"><span class="__cf_email__" data-cfemail="86ece3e7e8abf6eeefeaeff6f6e3a8f5f3f4eff0e3f2c6efe2e9f4f5efe7a8e5e9eb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philippe Panchaud</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7823-7639" title="Orcid link">http://orcid.org/0000-0001-7823-7639</a></span></div></li><li><div class="authorItemInformation"><sup>†</sup><span class="hlFld-ContribAuthor strong">Jean-Luc Specklin</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Diethelm</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Catherine Blumstein</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jean-Christophe Gauvin</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Loïc Jacob</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florence Masse</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gaëlle Mathieu</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Azely Mirre</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Schmitt</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roland Lange</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naomi Tidten-Luksch</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carmela Gnerre</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Swen Seeland</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charlyse Herrmann</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Seiler</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michel Enderlin-Paput</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aengus Mac Sweeney</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7250-9541" title="Orcid link">http://orcid.org/0000-0001-7250-9541</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Micha Wicki</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><sup>‡</sup><span class="hlFld-ContribAuthor strong">Christian Hubschwerlen</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Ritz</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Georg Rueedi</span> - <span class="hlFld-Affiliation affiliation">Idorsia Pharmaceuticals
Ltd., Hegenheimermattweg 91, CH-4123 Allschwil, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the present version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">During the preparation of the present manuscript, new LpxC inhibitors chemotypes identified by application of virtual screening were disclosed.(41)<br /></br>The authors declare no competing financial interest.<br /></br><sup>†</sup>Jean-Luc Specklin: deceased on November 5, 2015.<br /></br><sup>‡</sup>Christian Hubschwerlen: retired on December 31, 2012.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i78">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Hans Locher, Jonathan Delers, Mika Gaertner, Rachel Guilleux, Nadia Ekambaran, Marina Dos Santos, Milène Walter, François Le Goff, Marco Calderone, Maria Weiss,and Ulrike Babilonski are warmly thanked for the technical assistance they provided during the work described in the manuscript. We are grateful to the DMPK, Preformulation and Preclinical Galenics teams as well as to our colleagues from Process Research for their dedicated and continuous support. We thank the contract synthesis group at the Latvian Institute of Organic Synthesis led by Dr. Ronalds Zemribo for assistance in synthesis of building blocks. We also acknowledge the Paul Scherrer Institut, Villigen, Switzerland, for provision of synchrotron radiation beamtime (Swiss Light Source). We thank Dr. Makda Fisseha for her helpful comments during the preparation of the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the plasma concentration–time curve</p></td></tr><tr><td class="NLM_term">Cl</td><td class="NLM_def"><p class="first last">total clearance</p></td></tr><tr><td class="NLM_term">Cl<sub>u</sub></td><td class="NLM_def"><p class="first last">unbound clearance</p></td></tr><tr><td class="NLM_term"><i>F</i></td><td class="NLM_def"><p class="first last">fraction of dose reaching systemic circulation unchanged</p></td></tr><tr><td class="NLM_term"><i>f</i><sub>u</sub></td><td class="NLM_def"><p class="first last">unbound fraction</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum plasma concentration</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>max</sub></td><td class="NLM_def"><p class="first last">time of <i>C</i><sub>max</sub></p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">GNB</td><td class="NLM_def"><p class="first last">Gram-negative bacteria</p></td></tr><tr><td class="NLM_term">nBuNH<sub>2</sub></td><td class="NLM_def"><p class="first last"><i>n</i>-butyl amine</p></td></tr><tr><td class="NLM_term">CDI</td><td class="NLM_def"><p class="first last">carbonyl-diimidazole DIPEA, <i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide</p></td></tr><tr><td class="NLM_term">CFU</td><td class="NLM_def"><p class="first last">colony forming unit</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">Et<sub>2</sub>O</td><td class="NLM_def"><p class="first last">diethyl ether</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">NH<sub>2</sub>OTHP</td><td class="NLM_def"><p class="first last"><i>O</i>-(tetrahydro-2<i>H</i>-pyran-2-yl)hydroxylamine</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">T3P</td><td class="NLM_def"><p class="first last">propylphosphonic anhydride</p></td></tr><tr><td class="NLM_term">LpxC</td><td class="NLM_def"><p class="first last">UDP-3-<i>O</i>-((<i>R</i>)-3-hydroxymyristoyl)-<i>N</i>-glucosamine deacetylase</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i80">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 41 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span> <i>Antimicrobial
Resistance: Global Report on Surveillance 2014</i>; <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Antimicrobial%0AResistance%3A+Global+Report+on+Surveillance+2014%3B+World+Health+Organization%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%250AResistance%253A%2520Global%2520Report%2520on%2520Surveillance%25202014%26pub%3DWorld%2520Health%2520Organization%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span> <i>Global
Priority List of Antibiotic-Resistant Bacteria to Guide Research,
Discovery, and Development of New Antibiotics</i>; <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Global%0APriority+List+of+Antibiotic-Resistant+Bacteria+to+Guide+Research%2C%0ADiscovery%2C+and+Development+of+New+Antibiotics%3B+World+Health+Organization%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26jtitle%3DGlobal%250APriority%2520List%2520of%2520Antibiotic-Resistant%2520Bacteria%2520to%2520Guide%2520Research%252C%250ADiscovery%252C%2520and%2520Development%2520of%2520New%2520Antibiotics%26pub%3DWorld%2520Health%2520Organization%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granata, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrosillo, N.</span></span> <span> </span><span class="NLM_article-title">Resistance to Colistin in <i>Klebsiella Pneumoniae</i>: A 4.0 Strain?</span>. <i>Infect. Dis. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">7104</span>, <span class="refDoi"> DOI: 10.4081/idr.2017.7104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.4081%2Fidr.2017.7104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=28626539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A280%3ADC%252BC1cnpsVKqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=7104&author=G.+Granataauthor=N.+Petrosillo&title=Resistance+to+Colistin+in+Klebsiella+Pneumoniae%3A+A+4.0+Strain%3F&doi=10.4081%2Fidr.2017.7104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to Colistin in Klebsiella Pneumoniae: A 4.0 Strain?</span></div><div class="casAuthors">Granata Guido; Petrosillo Nicola</div><div class="citationInfo"><span class="NLM_cas:title">Infectious disease reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">7104</span>
        ISSN:<span class="NLM_cas:issn">2036-7430</span>.
    </div><div class="casAbstract">The global rise of multidrug-resistant gram-negative bacteria represents an increasing threat to patient safety.  From the first observation of a carbapenem-resistant gram-negative bacteria a global spread of extended-spectrum beta-lactamases and carbapenemases producing Klebsiella pneumoniae has been observed.  Treatment options for multidrug-resistant K. pneumoniae are actually limited to combination therapy with some aminoglycosides, tigecycline and to older antimicrobial agents.  Unfortunately, the prevalence of colistin-resistant and tigecycline-resistant K. pneumoniae is increasing globally.  Infection due to colistin-resistant K. pneumoniae represents an independent risk factor for mortality.  Resistance to colistin in K. pneumoniae may be multifactorial, as it is mediated by chromosomal genes or plasmids.  The emergence of transmissible, plasmid-mediated colistin resistance is an alarming finding.  The absence of new agents effective against resistant Gram-negative pathogens means that enhanced surveillance, compliance with infection prevention procedures, and antimicrobial stewardship programs will be required to limit the spread of colistin-resistant K. pneumoniae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWTAXmmdHFs5QHPYJuLg8TfW6udTcc2eby5R0AgDDLRbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnpsVKqug%253D%253D&md5=28fe14c9d0962b16c46931c5a4da2149</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4081%2Fidr.2017.7104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4081%252Fidr.2017.7104%26sid%3Dliteratum%253Aachs%26aulast%3DGranata%26aufirst%3DG.%26aulast%3DPetrosillo%26aufirst%3DN.%26atitle%3DResistance%2520to%2520Colistin%2520in%2520Klebsiella%2520Pneumoniae%253A%2520A%25204.0%2520Strain%253F%26jtitle%3DInfect.%2520Dis.%2520Rep.%26date%3D2017%26volume%3D9%26spage%3D7104%26doi%3D10.4081%2Fidr.2017.7104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guidos, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheld, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J. G.</span></span> <span> </span><span class="NLM_article-title">The 10 x ’20 Initiative: Pursuing a Global Commitment to Develop 10 New Antibacterial Drugs by 2020</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1081</span>– <span class="NLM_lpage">1083</span>, <span class="refDoi"> DOI: 10.1086/652237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1086%2F652237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=20214473" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1081-1083&author=D.+N.+Gilbertauthor=R.+J.+Guidosauthor=H.+W.+Boucherauthor=G.+H.+Talbotauthor=B.+Spellbergauthor=J.+E.+Edwardsauthor=W.+M.+Scheldauthor=J.+S.+Bradleyauthor=J.+G.+Bartlett&title=The+10+x+%E2%80%9920+Initiative%3A+Pursuing+a+Global+Commitment+to+Develop+10+New+Antibacterial+Drugs+by+2020&doi=10.1086%2F652237"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1086%2F652237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F652237%26sid%3Dliteratum%253Aachs%26aulast%3DGilbert%26aufirst%3DD.%2BN.%26aulast%3DGuidos%26aufirst%3DR.%2BJ.%26aulast%3DBoucher%26aufirst%3DH.%2BW.%26aulast%3DTalbot%26aufirst%3DG.%2BH.%26aulast%3DSpellberg%26aufirst%3DB.%26aulast%3DEdwards%26aufirst%3DJ.%2BE.%26aulast%3DScheld%26aufirst%3DW.%2BM.%26aulast%3DBradley%26aufirst%3DJ.%2BS.%26aulast%3DBartlett%26aufirst%3DJ.%2BG.%26atitle%3DThe%252010%2520x%2520%25E2%2580%259920%2520Initiative%253A%2520Pursuing%2520a%2520Global%2520Commitment%2520to%2520Develop%252010%2520New%2520Antibacterial%2520Drugs%2520by%25202020%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2010%26volume%3D50%26spage%3D1081%26epage%3D1083%26doi%3D10.1086%2F652237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lgFTvn15dlTbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surivet, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zumbrunn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruyere, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seiler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enderlin-Paput, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enderlin-Paput, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauvin, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubschwerlen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueedi, G.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Characterization of Tetrahydropyran-Based Bacterial Topoisomerase Inhibitors With Antibacterial Activity Against Gram-Negative Bacteria</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3776</span>– <span class="NLM_lpage">3794</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVWktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3776-3794&author=J.-P.+Surivetauthor=C.+Zumbrunnauthor=T.+Bruyereauthor=D.+Burauthor=C.+Kohlauthor=H.+H.+Locherauthor=P.+Seilerauthor=E.+A.+Ertelauthor=P.+Hessauthor=M.+Enderlin-Paputauthor=S.+Enderlin-Paputauthor=J.-C.+Gauvinauthor=A.+Mirreauthor=C.+Hubschwerlenauthor=D.+Ritzauthor=G.+Rueedi&title=Synthesis+and+Characterization+of+Tetrahydropyran-Based+Bacterial+Topoisomerase+Inhibitors+With+Antibacterial+Activity+Against+Gram-Negative+Bacteria&doi=10.1021%2Facs.jmedchem.6b01831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Characterization of Tetrahydropyran-Based Bacterial Topoisomerase Inhibitors with Antibacterial Activity against Gram-Negative Bacteria</span></div><div class="casAuthors">Surivet, Jean-Philippe; Zumbrunn, Cornelia; Bruyere, Thierry; Bur, Daniel; Kohl, Christopher; Locher, Hans H.; Seiler, Peter; Ertel, Eric A.; Hess, Patrick; Enderlin-Paput, Michel; Enderlin-Paput, Stephanie; Gauvin, Jean-Christophe; Mirre, Azely; Hubschwerlen, Christian; Ritz, Daniel; Rueedi, Georg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3776-3794</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is an urgent unmet medical need for novel antibiotics that are effective against a broad range of bacterial species, esp. multidrug resistant ones.  Tetrahydropyran-based inhibitors of bacterial type II topoisomerases (DNA gyrase and topoisomerase IV) display potent activity against Gram-pos. pathogens and no target-mediated cross-resistance with fluoroquinolones.  We report our research efforts aimed at expanding the antibacterial spectrum of this class of mols. toward difficult-to-treat Gram-neg. pathogens.  Physicochem. properties (polarity and basicity) were considered to guide the design process.  Dibasic tetrahydropyran-based compds. such as 6 and 21 are potent inhibitors of both DNA gyrase and topoisomerase IV, displaying antibacterial activities against Gram-pos. and Gram-neg. pathogens (Staphylococcus aureus, Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii).  Compds. 6 and 21 are efficacious in clin. relevant murine infection models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWIubQLqfwWbVg90H21EOLACvtfcHk0liWK6ANfzHn7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVWktrw%253D&md5=0c2f17fda10b44b6c261c0cc86279564</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01831%26sid%3Dliteratum%253Aachs%26aulast%3DSurivet%26aufirst%3DJ.-P.%26aulast%3DZumbrunn%26aufirst%3DC.%26aulast%3DBruyere%26aufirst%3DT.%26aulast%3DBur%26aufirst%3DD.%26aulast%3DKohl%26aufirst%3DC.%26aulast%3DLocher%26aufirst%3DH.%2BH.%26aulast%3DSeiler%26aufirst%3DP.%26aulast%3DErtel%26aufirst%3DE.%2BA.%26aulast%3DHess%26aufirst%3DP.%26aulast%3DEnderlin-Paput%26aufirst%3DM.%26aulast%3DEnderlin-Paput%26aufirst%3DS.%26aulast%3DGauvin%26aufirst%3DJ.-C.%26aulast%3DMirre%26aufirst%3DA.%26aulast%3DHubschwerlen%26aufirst%3DC.%26aulast%3DRitz%26aufirst%3DD.%26aulast%3DRueedi%26aufirst%3DG.%26atitle%3DSynthesis%2520and%2520Characterization%2520of%2520Tetrahydropyran-Based%2520Bacterial%2520Topoisomerase%2520Inhibitors%2520With%2520Antibacterial%2520Activity%2520Against%2520Gram-Negative%2520Bacteria%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3776%26epage%3D3794%26doi%3D10.1021%2Facs.jmedchem.6b01831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bush, K.</span></span> <span> </span><span class="NLM_article-title">Investigational Agents for the Treatment of Gram-Negative Bacterial Infections: A Reality Check</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.5b00100</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.5b00100" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1GksrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=509-511&author=K.+Bush&title=Investigational+Agents+for+the+Treatment+of+Gram-Negative+Bacterial+Infections%3A+A+Reality+Check&doi=10.1021%2Facsinfecdis.5b00100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Investigational Agents for the Treatment of Gram-Negative Bacterial Infections: A Reality Check</span></div><div class="casAuthors">Bush, Karen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">509-511</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antibiotic-resistant Gram-neg. bacteria are, arguably, the most difficult organisms to treat, with a limited no. of new antibiotics in the development pipeline.  Currently 24 new agents in phase 1, phase 2, or phase 3 clin. development were identified for the potential treatment of infections caused by Gram-neg. bacteria.  Of these agents, most are improved iterations of known antibiotic classes, including new aminoglycosides, β-lactams, β-lactamase inhibitors, quinolones, and tetracyclines with greater potency or a broader spectrum of activity.  However, novel structures also appear, with host defense peptide mimetics, boronic acid, and bridged diazabicyclooctane β-lactamase inhibitors and unique bacterial topoisomerase inhibitors.  Most of the new agents have received a Qualified Infectious Disease Product (QIDP) designation that may help to accelerate FDA drug approvals.  Because resistance will inevitably arise to any antibacterial agent, it will be necessary to continue to identify addnl. new agents in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP0Z7V8fADjLVg90H21EOLACvtfcHk0lh1760wUJnJgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1GksrvM&md5=b5833cbe0a6e9abe92b4b292a219dd59</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.5b00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.5b00100%26sid%3Dliteratum%253Aachs%26aulast%3DBush%26aufirst%3DK.%26atitle%3DInvestigational%2520Agents%2520for%2520the%2520Treatment%2520of%2520Gram-Negative%2520Bacterial%2520Infections%253A%2520A%2520Reality%2520Check%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D509%26epage%3D511%26doi%3D10.1021%2Facsinfecdis.5b00100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span>; <span class="NLM_string-name">Kubo, A.</span>; <span class="NLM_string-name">Lopez, S.</span>; <span class="NLM_string-name">Gomez, M.</span>; <span class="NLM_string-name">Corey, V. C.</span>; <span class="NLM_string-name">Andrews, L.</span>; <span class="NLM_string-name">Schwartz, M. A.</span>; <span class="NLM_string-name">Kasar, R.</span>; <span class="NLM_string-name">McEnroe, G.</span>; <span class="NLM_string-name">Aggen, J.</span>; <span class="NLM_string-name">Miller, G.</span>; <span class="NLM_string-name">Armstrong, E. S.</span></span> <span> </span><span class="NLM_article-title">Structure, Potency and Bactericidal Activity of ACHN-975, a First-in-Class LpxC Inhibitor</span>. In  <i>53rd Interscience Conference on Antimicrobial Agents and Chemotherapy, Denver, Colorado, United States</i>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+W.+Serio&author=A.+Kubo&author=S.+Lopez&author=M.+Gomez&author=V.+C.+Corey&author=L.+Andrews&author=M.+A.+Schwartz&author=R.+Kasar&author=G.+McEnroe&author=J.+Aggen&author=G.+Miller&author=E.+S.+Armstrong&title=Structure%2C+Potency+and+Bactericidal+Activity+of+ACHN-975%2C+a+First-in-Class+LpxC+Inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DSerio%26aufirst%3DA.%2BW.%26atitle%3DStructure%252C%2520Potency%2520and%2520Bactericidal%2520Activity%2520of%2520ACHN-975%252C%2520a%2520First-in-Class%2520LpxC%2520Inhibitor%26jtitle%3D53rd%2520Interscience%2520Conference%2520on%2520Antimicrobial%2520Agents%2520and%2520Chemotherapy%252C%2520Denver%252C%2520Colorado%252C%2520United%2520States%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierke, C. A.</span></span> <span> </span><span class="NLM_article-title">UDP-3-O-(<i>R</i>-3-hydroxymyristoyl)-<i>N</i>-acetylglucosamine Deacetylase of <i>Escherichia coli</i> Is a Zinc Metalloenzyme</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1902</span>– <span class="NLM_lpage">1911</span>, <span class="refDoi"> DOI: 10.1021/bi982339s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi982339s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADyaK1MXkvFymsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1902-1911&author=J.+E.+Jackmanauthor=C.+R.+Raetzauthor=C.+A.+Fierke&title=UDP-3-O-%28R-3-hydroxymyristoyl%29-N-acetylglucosamine+Deacetylase+of+Escherichia+coli+Is+a+Zinc+Metalloenzyme&doi=10.1021%2Fbi982339s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">UDP-3-O-(R-3-Hydroxymyristoyl)-N-acetylglucosamine Deacetylase of Escherichia coli Is a Zinc Metalloenzyme</span></div><div class="casAuthors">Jackman, Jane E.; Raetz, Christian R. H.; Fierke, Carol A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1902-1911</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The enzyme UDP-3-O-(R-3-hydroxymyristoyl)-GlcNAc deacetylase (LpxC) catalyzes the committed step in the biosynthesis of lipid A and is therefore a potential antibiotic target.  Inhibition of this enzyme by hydroxamate compds. suggested the presence of a metal ion cofactor.  We have investigated the substrate specificity and metal dependence of the deacetylase using spectroscopic and kinetic analyses.  Comparison of the steady-state kinetic parameters for the physiol. substrate UDP-3-O-(R-3-hydroxymyristoyl)-GlcNAc and an alternative substrate, UDP-GlcNAc, demonstrates that the ester-linked R-3-hydroxymyristoyl chain increases kcat/KM (5 × 106)-fold.  Metal-chelating reagents, such as dipicolinic acid (DPA) and EDTA, completely inhibit LpxC activity, implicating an essential metal ion.  Plasma emission spectroscopy and colorimetric assays directly demonstrate that purified LpxC contains bound Zn2+.  This Zn2+ can be removed by incubation with DPA, causing a decrease in the LpxC activity that can be restored by subsequent addn. of Zn2+.  However, high concns. of Zn2+ also inhibit LpxC.  Addn. of Co2+, Ni2+, or Mn2+ to apo-LpxC also activates the enzyme to varying degrees while no addnl. activity is obsd. upon the addn. of Cd2+, Ca2+, Mg2+, or Cu2+.  This is consistent with the profile of metals that substitute for catalytic zinc ions in metalloproteinases.  Co2+ ions stimulate LpxC activity maximally at a stoichiometry of 1:1.  These data demonstrate that E. coli LpxC is a metalloenzyme that requires bound Zn2+ for optimal activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoETi4nWY-kpLVg90H21EOLACvtfcHk0lgz72NZt7j9Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXkvFymsw%253D%253D&md5=43e4ad7d68cbbd04a202c898c7bb667a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fbi982339s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi982339s%26sid%3Dliteratum%253Aachs%26aulast%3DJackman%26aufirst%3DJ.%2BE.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DFierke%26aufirst%3DC.%2BA.%26atitle%3DUDP-3-O-%2528R-3-hydroxymyristoyl%2529-N-acetylglucosamine%2520Deacetylase%2520of%2520Escherichia%2520coli%2520Is%2520a%2520Zinc%2520Metalloenzyme%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D1902%26epage%3D1911%26doi%3D10.1021%2Fbi982339s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierke, C. A.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism and molecular recognition of <i>E. coli</i> UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine Deacetylase Probed by Mutagenesis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">15240</span>– <span class="NLM_lpage">15248</span>, <span class="refDoi"> DOI: 10.1021/bi061405k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi061405k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlChurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=15240-15248&author=M.+Hernickauthor=C.+A.+Fierke&title=Catalytic+mechanism+and+molecular+recognition+of+E.+coli+UDP-3-O-%28R-3-hydroxymyristoyl%29-N-acetylglucosamine+Deacetylase+Probed+by+Mutagenesis&doi=10.1021%2Fbi061405k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic Mechanism and Molecular Recognition of E. coli UDP-3-O-(R-3-Hydroxymyristoyl)-N-acetylglucosamine Deacetylase Probed by Mutagenesis</span></div><div class="casAuthors">Hernick, Marcy; Fierke, Carol A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15240-15248</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a metal-dependent deacetylase that catalyzes the hydrolysis of UDP-3-O-myristoyl-N-acetyl-glucosamine to form UDP-3-O-myristoyl-glucosamine and acetate.  This is the committed step in the biosynthesis of lipid A, and therefore, LpxC is a target for the development of antimicrobial agents in the treatment of Gram-neg. infections.  To facilitate the development of potent and specific inhibitors of LpxC, the mol. determinants of binding and specificity and the catalytic mechanism for this enzyme have been probed.  The functions of active site residues have been classified on the basis of changes in steady-state turnover (kcat, KM, and kcat/KM) and product binding affinity (KDProduct).  We have identified side chains that enhance product affinity and reactivity (F192, K239, D246, and H265), destabilize product affinity (E78 and D197), and preferentially enhance catalytic efficiency (H19, T191, K143, and N162).  In addn., the affinity of LpxC for myrUDP-GlcNH2 is dependent on two ionizations, one deprotonation and one protonation, with apparent pKa values of 6.5±0.1 and 7.4±0.1, resp.  The UDP moiety of the product contributes significantly to recognition by LpxC, suggesting that this region can be targeted in drug development.  These data provide a map of the active site features essential for catalysis and mol. recognition by LpxC that can be used for developing more potent LpxC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon6LHZu-oUlLVg90H21EOLACvtfcHk0liXH3aMAT4rfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlChurvI&md5=47e028e85adf93ac8975001ecbff1e1a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fbi061405k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi061405k%26sid%3Dliteratum%253Aachs%26aulast%3DHernick%26aufirst%3DM.%26aulast%3DFierke%26aufirst%3DC.%2BA.%26atitle%3DCatalytic%2520mechanism%2520and%2520molecular%2520recognition%2520of%2520E.%2520coli%2520UDP-3-O-%2528R-3-hydroxymyristoyl%2529-N-acetylglucosamine%2520Deacetylase%2520Probed%2520by%2520Mutagenesis%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D15240%26epage%3D15248%26doi%3D10.1021%2Fbi061405k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure of the Bacterial Deacetylase LpxC Bound to the Nucleotide Reaction Product Reveals Mechanisms of Oxyanion Stabilization and Proton Transfer</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">34073</span>– <span class="NLM_lpage">34080</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.513028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1074%2Fjbc.M113.513028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=24108127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=34073-34080&author=G.+M.+Claytonauthor=D.+J.+Kleinauthor=K.+W.+Rickertauthor=S.+B.+Patelauthor=M.+Kornienkoauthor=J.+Zugay-Murphyauthor=J.+C.+Reidauthor=S.+Tummalaauthor=S.+Sharmaauthor=S.+B.+Singhauthor=L.+Mieselauthor=K.+J.+Lumbauthor=S.+M.+Soisson&title=Structure+of+the+Bacterial+Deacetylase+LpxC+Bound+to+the+Nucleotide+Reaction+Product+Reveals+Mechanisms+of+Oxyanion+Stabilization+and+Proton+Transfer&doi=10.1074%2Fjbc.M113.513028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the bacterial deacetylase LpxC bound to the nucleotide reaction product reveals mechanisms of oxyanion stabilization and proton transfer</span></div><div class="casAuthors">Clayton, Gina M.; Klein, Daniel J.; Rickert, Keith W.; Patel, Sangita B.; Kornienko, Maria; Zugay-Murphy, Joan; Reid, John C.; Tummala, Srivanya; Sharma, Sujata; Singh, Sheo B.; Miesel, Lynn; Lumb, Kevin J.; Soisson, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">34073-34080</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The emergence of antibiotic-resistant strains of pathogenic bacteria is an increasing threat to global health that underscores an urgent need for an expanded antibacterial armamentarium.  Gram-neg. bacteria, such as Escherichia coli, have become increasingly important clin. pathogens with limited treatment options.  This is due in part to their lipopolysaccharide (LPS) outer membrane components, which dually serve as endotoxins while also protecting Gram-neg. bacteria from antibiotic entry.  The LpxC enzyme, UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase, catalyzes the committed step of LPS biosynthesis, making LpxC a promising target for new antibacterials.  Here, the authors present the 1st structure of E. coli LpxC in complex with the deacetylation reaction product, UDP-[3-O-(R-3-hydroxymyristoyl)]glucosamine.  These studies provide valuable insight into the recognition of substrates and products by LpxC and a platform for structure-guided drug discovery of broad spectrum Gram-neg. antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm8dRnYgccKbVg90H21EOLACvtfcHk0lj7kuR4m6DnmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO&md5=ac4adc79fe696089087d35a29e39f892</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.513028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.513028%26sid%3Dliteratum%253Aachs%26aulast%3DClayton%26aufirst%3DG.%2BM.%26aulast%3DKlein%26aufirst%3DD.%2BJ.%26aulast%3DRickert%26aufirst%3DK.%2BW.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DTummala%26aufirst%3DS.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26atitle%3DStructure%2520of%2520the%2520Bacterial%2520Deacetylase%2520LpxC%2520Bound%2520to%2520the%2520Nucleotide%2520Reaction%2520Product%2520Reveals%2520Mechanisms%2520of%2520Oxyanion%2520Stabilization%2520and%2520Proton%2520Transfer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D34073%26epage%3D34080%26doi%3D10.1074%2Fjbc.M113.513028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, N. B.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of <i>A. Aeolicu</i>s LpxC With Bound Product Suggests Alternate Deacetylation Mechanism</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">1706</span>– <span class="NLM_lpage">1719</span>, <span class="refDoi"> DOI: 10.1002/prot.24856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1002%2Fprot.24856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=26177919" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2015&pages=1706-1719&author=M.+D.+Millerauthor=N.+Gaoauthor=P.+L.+Rossauthor=N.+B.+Olivier&title=Crystal+Structure+of+A.+Aeolicus+LpxC+With+Bound+Product+Suggests+Alternate+Deacetylation+Mechanism&doi=10.1002%2Fprot.24856"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fprot.24856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24856%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DOlivier%26aufirst%3DN.%2BB.%26atitle%3DCrystal%2520Structure%2520of%2520A.%2520Aeolicus%2520LpxC%2520With%2520Bound%2520Product%2520Suggests%2520Alternate%2520Deacetylation%2520Mechanism%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2015%26volume%3D83%26spage%3D1706%26epage%3D1719%26doi%3D10.1002%2Fprot.24856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalinin, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holl, R.</span></span> <span> </span><span class="NLM_article-title">Insights Into the Zinc-Dependent Deacetylase LpxC: Biochemical Properties and Inhibitor Design</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2379</span>– <span class="NLM_lpage">2430</span>, <span class="refDoi"> DOI: 10.2174/1568026616666160413135835</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.2174%2F1568026616666160413135835" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=27072691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFOgs7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=2379-2430&author=D.+V.+Kalininauthor=R.+Holl&title=Insights+Into+the+Zinc-Dependent+Deacetylase+LpxC%3A+Biochemical+Properties+and+Inhibitor+Design&doi=10.2174%2F1568026616666160413135835"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Zinc-Dependent Deacetylase LpxC: Biochemical Properties and Inhibitor Design</span></div><div class="casAuthors">Kalinin, Dmitrii V.; Holl, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2379-2430</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The bacterial enzyme UDP-3-O-[(R)-3-hydroxymyristoyl]-N-acetylglucosamine deacetylase (LpxC), catalyzing the first committed step of lipid A biosynthesis, represents a promising target in the development of novel antibiotics against Gram-neg. bacteria.  Structure, catalytic reaction mechanism and regulation of the Zn2+-dependent metalloamidase have been intensively investigated.  The enzyme is required for growth and viability of Gram-neg. bacteria, displays no sequence homol. with any mammalian protein, but is highly conserved in Gram-neg. bacteria, thus permitting the development of Gram-neg. selective antibacterial agents with limited off-target effects.  Several small mol. LpxC inhibitors have been developed, like the substrate analog TU-514 (12a), the aryloxazoline L-161,240 (13w), the sulfonamide BB-78485 (23a), the N-aroyl-L-threonine deriv. CHIR-090 (24a), the sulfone-contg. pyridone LpxC-3 (43e), and the uridine-based inhibitor 1-68A (47a), displaying diverse inhibitory and antibacterial activities.  Most of these compds. share a Zn2+-binding hydroxamate moiety attached to a structural element addressing the hydrophobic tunnel or the UDP binding site.  The butadiynyl deriv. ACHN-975 (28) is the first LpxC inhibitor entering clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod_JborvvzXbVg90H21EOLACvtfcHk0ljD8PLfK_UMZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFOgs7fE&md5=82abc89ee4dab91b95bbb9dabb23f0b3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F1568026616666160413135835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026616666160413135835%26sid%3Dliteratum%253Aachs%26aulast%3DKalinin%26aufirst%3DD.%2BV.%26aulast%3DHoll%26aufirst%3DR.%26atitle%3DInsights%2520Into%2520the%2520Zinc-Dependent%2520Deacetylase%2520LpxC%253A%2520Biochemical%2520Properties%2520and%2520Inhibitor%2520Design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2016%26volume%3D16%26spage%3D2379%26epage%3D2430%26doi%3D10.2174%2F1568026616666160413135835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erwin, A. L.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Drug Discovery Targeting the Lipopolysaccharide Biosynthetic Enzyme LpxC</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a025304</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a025304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1101%2Fcshperspect.a025304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=27235477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a025304&author=A.+L.+Erwin&title=Antibacterial+Drug+Discovery+Targeting+the+Lipopolysaccharide+Biosynthetic+Enzyme+LpxC&doi=10.1101%2Fcshperspect.a025304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery targeting the lipopolysaccharide biosynthetic enzyme LpxC</span></div><div class="casAuthors">Erwin, Alice L.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a025304/1-a025304/15</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The enzyme LpxC (UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase) is broadly conserved across Gram-neg. bacteria and is essential for synthesis of lipid A, the membrane anchor of the lipopolysaccharides (LPSs), which are a major component of the outer membrane in nearly all Gram-neg. bacteria.  LpxC has been the focus of targetdirected antibiotic discovery projects in numerous pharmaceutical and academic groups for more than 20 years.  Despite intense effort, no LpxC inhibitor has been approved for therapeutic use, and only one has yet reached human studies.  This article will summarize the history of LpxC as a drug target and the parallel history of research on LpxC biol.  Both academic and industrial researchers have used LpxC inhibitors as tool compds., leading to increased understanding of the differing mechanisms for regulation of LPS synthesis in Escherichia coli and Pseudomonas aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-AZMwFme-97Vg90H21EOLACvtfcHk0lgM7YTnIY_l_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D&md5=a8ca24f54a9aaad079b94a41cfab62a4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a025304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a025304%26sid%3Dliteratum%253Aachs%26aulast%3DErwin%26aufirst%3DA.%2BL.%26atitle%3DAntibacterial%2520Drug%2520Discovery%2520Targeting%2520the%2520Lipopolysaccharide%2520Biosynthetic%2520Enzyme%2520LpxC%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da025304%26doi%3D10.1101%2Fcshperspect.a025304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Onishi, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelak, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerckens, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patchett, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galloway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Agents That Inhibit Lipid A Biosynthesis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">980</span>– <span class="NLM_lpage">982</span>, <span class="refDoi"> DOI: 10.1126/science.274.5289.980</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1126%2Fscience.274.5289.980" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=8875939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADyaK28XmslKmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=980-982&author=H.+R.+Onishiauthor=B.+A.+Pelakauthor=L.+S.+Gerckensauthor=L.+L.+Silverauthor=F.+M.+Kahanauthor=M.+H.+Chenauthor=A.+A.+Patchettauthor=S.+M.+Gallowayauthor=S.+A.+Hylandauthor=M.+S.+Andersonauthor=C.+R.+Raetz&title=Antibacterial+Agents+That+Inhibit+Lipid+A+Biosynthesis&doi=10.1126%2Fscience.274.5289.980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial agents that inhibit lipid A biosynthesis</span></div><div class="casAuthors">Onishi, H. Russell; Pelak, Barbara A.; Gerckens, Lynn S.; Silver, Lynn L.; Kahan, Frederick M.; Chen, Meng-Hsin; Patchett, Arthur A.; Galloway, Susan M.; Hyland, Sheryl A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5289</span>),
    <span class="NLM_cas:pages">980-982</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lipid A constitutes the outer monolayer of the outer membrane of Gram-neg. bacteria and is essential for bacterial growth.  Synthetic antibacterials were identified that inhibit the second enzyme (a unique deacetylase) of lipid A biosynthesis.  The inhibitors are chiral hydroxamic acids bearing certain hydrophobic arom. moieties.  They may bind to a metal in the active site of the deacetylase.  The most potent analog (with an inhibition const. of about 50 nM) displayed a minimal inhibitory concn. of about 1 μg per mL against Escherichia coli, causes three logs of bacterial killing in 4 h, and cured mice infected with a lethal i.p. dose of E. coli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVKgUe_5zI9rVg90H21EOLACvtfcHk0lgM7YTnIY_l_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmslKmsLw%253D&md5=d5098756849325110d7f1f4850322cba</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5289.980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5289.980%26sid%3Dliteratum%253Aachs%26aulast%3DOnishi%26aufirst%3DH.%2BR.%26aulast%3DPelak%26aufirst%3DB.%2BA.%26aulast%3DGerckens%26aufirst%3DL.%2BS.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26aulast%3DKahan%26aufirst%3DF.%2BM.%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DGalloway%26aufirst%3DS.%2BM.%26aulast%3DHyland%26aufirst%3DS.%2BA.%26aulast%3DAnderson%26aufirst%3DM.%2BS.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26atitle%3DAntibacterial%2520Agents%2520That%2520Inhibit%2520Lipid%2520A%2520Biosynthesis%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D980%26epage%3D982%26doi%3D10.1126%2Fscience.274.5289.980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span> <i>Hydroxamic Acids</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S. P.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1007%2F978-3-642-38111-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=S.+P.+Gupta&title=Hydroxamic+Acids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-38111-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-38111-9%26sid%3Dliteratum%253Aachs%26btitle%3DHydroxamic%2520Acids%26aulast%3DGupta%26aufirst%3DS.%2BP.%26pub%3DSpringer%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span> <span> </span><span class="NLM_article-title">Syntheses, Structures and Antibiotic Activities of LpxC Inhibitors Based on the Diacetylene Scaffold</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2Fj.bmc.2010.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=21194954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=852-860&author=X.+Liangauthor=C.+J.+Leeauthor=X.+Chenauthor=H.+S.+Chungauthor=D.+Zengauthor=C.+R.+Raetzauthor=Y.+Liauthor=P.+Zhouauthor=E.+J.+Toone&title=Syntheses%2C+Structures+and+Antibiotic+Activities+of+LpxC+Inhibitors+Based+on+the+Diacetylene+Scaffold&doi=10.1016%2Fj.bmc.2010.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Syntheses, structures and antibiotic activities of LpxC inhibitors based on the diacetylene scaffold</span></div><div class="casAuthors">Liang, Xiaofei; Lee, Chul-Jin; Chen, Xin; Chung, Hak Suk; Zeng, Daina; Raetz, Christian R. H.; Li, Yaoxian; Zhou, Pei; Toone, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">852-860</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Compds. inhibiting LpxC in the lipid A biosynthetic pathway are promising leads for novel antibiotics against multidrug-resistant Gram-neg. pathogens.  The authors report the syntheses and structural and biochem. characterizations of LpxC inhibitors based on a diphenyl-diacetylene (1,4-diphenyl-1,3-butadiyne) threonyl-hydroxamate scaffold.  These studies provide a mol. interpretation for the differential antibiotic activities of compds. with a substituted distal Ph ring as well as the abs. stereochem. requirement at the C2, but not C3, position of the threonyl group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWL2QPzfjArLVg90H21EOLACvtfcHk0lgeominZzo1cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVCnuw%253D%253D&md5=10326af21364b0785a67692687b21385</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChung%26aufirst%3DH.%2BS.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26atitle%3DSyntheses%252C%2520Structures%2520and%2520Antibiotic%2520Activities%2520of%2520LpxC%2520Inhibitors%2520Based%2520on%2520the%2520Diacetylene%2520Scaffold%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D852%26epage%3D860%26doi%3D10.1016%2Fj.bmc.2010.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barb, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholas, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span> <span> </span><span class="NLM_article-title">Species-Specific and Inhibitor-Dependent Conformations of LpxC: Implications for Antibiotic Design</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2Fj.chembiol.2010.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=21167751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlars70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=38-47&author=C.+J.+Leeauthor=X.+Liangauthor=X.+Chenauthor=D.+Zengauthor=S.+H.+Jooauthor=H.+S.+Chungauthor=A.+W.+Barbauthor=S.+M.+Swansonauthor=R.+A.+Nicholasauthor=Y.+Liauthor=E.+J.+Tooneauthor=C.+R.+Raetzauthor=P.+Zhou&title=Species-Specific+and+Inhibitor-Dependent+Conformations+of+LpxC%3A+Implications+for+Antibiotic+Design&doi=10.1016%2Fj.chembiol.2010.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Species-specific and inhibitor-dependent conformations of LpxC: implications for antibiotic design</span></div><div class="casAuthors">Lee, Chul-Jin; Liang, Xiao-Fei; Chen, Xin; Zeng, Dai-Na; Joo, Sang-Hoon; Chung, Hak-Suk; Barb, Adam W.; Swanson, Shauna M.; Nicholas, Robert A.; Li, Yao-Xian; Toone, Eric J.; Raetz, Christian R. H.; Zhou, Pei</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-47</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LpxC is an essential enzyme in the lipid A biosynthetic pathway in gram-neg. bacteria.  Several promising antimicrobial lead compds. targeting LpxC have been reported, though they typically display a large variation in potency against different gram-neg. pathogens.  We report that inhibitors with a diacetylene scaffold effectively overcome the resistance caused by sequence variation in the LpxC substrate-binding passage.  Compd. binding is captured in complex with representative LpxC orthologs, and structural anal. reveals large conformational differences that mostly reflect inherent mol. features of distinct LpxC orthologs, whereas ligand-induced structural adaptations occur at a smaller scale.  These observations highlight the need for a mol. understanding of inherent structural features and conformational plasticity of LpxC enzymes for optimizing LpxC inhibitors as broad-spectrum antibiotics against gram-neg. infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbcwncn731jrVg90H21EOLACvtfcHk0lgc3jr6FMkaOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlars70%253D&md5=7aa1222210dfec8ad8675194b0b13731</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DJoo%26aufirst%3DS.%2BH.%26aulast%3DChung%26aufirst%3DH.%2BS.%26aulast%3DBarb%26aufirst%3DA.%2BW.%26aulast%3DSwanson%26aufirst%3DS.%2BM.%26aulast%3DNicholas%26aufirst%3DR.%2BA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DSpecies-Specific%2520and%2520Inhibitor-Dependent%2520Conformations%2520of%2520LpxC%253A%2520Implications%2520for%2520Antibiotic%2520Design%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D38%26epage%3D47%26doi%3D10.1016%2Fj.chembiol.2010.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis, Structure, and Antibiotic Activity of Aryl-Substituted LpxC Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6954</span>– <span class="NLM_lpage">6966</span>, <span class="refDoi"> DOI: 10.1021/jm4007774</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4007774" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WltrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6954-6966&author=X.+Liangauthor=C.+J.+Leeauthor=J.+Zhaoauthor=E.+J.+Tooneauthor=P.+Zhou&title=Synthesis%2C+Structure%2C+and+Antibiotic+Activity+of+Aryl-Substituted+LpxC+Inhibitors&doi=10.1021%2Fjm4007774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, Structure, and Antibiotic Activity of Aryl-Substituted LpxC Inhibitors</span></div><div class="casAuthors">Liang, Xiaofei; Lee, Chul-Jin; Zhao, Jinshi; Toone, Eric J.; Zhou, Pei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6954-6966</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The zinc-dependent deacetylase LpxC catalyzes the committed step of lipid A biosynthesis in Gram-neg. bacteria and is a validated target for the development of novel antibiotics to combat multidrug-resistant Gram-neg. infections.  Many potent LpxC inhibitors contain an essential threonyl-hydroxamate headgroup for high-affinity interaction with LpxC.  The authors report the synthesis, antibiotic activity, and structural and enzymic characterization of novel LpxC inhibitors contg. an addnl. aryl group in the threonyl-hydroxamate moiety, which expands the inhibitor-binding surface in LpxC.  These compds. display enhanced potency against LpxC in enzymic assays and superior antibiotic activity against Francisella novicida in cell culture.  The comparison of the antibiotic activities of these compds. against a leaky Escherichia coli strain and the wild-type strain reveals the contribution of the formidable outer-membrane permeability barrier that reduces the compds. efficacy in cell culture and emphasizes the importance of maintaining a balanced hydrophobicity and hydrophilicity profile in developing effective LpxC-targeting antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUuKY9tg3BU7Vg90H21EOLACvtfcHk0lgc3jr6FMkaOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WltrrJ&md5=d81226a4e668a5f819ea7248ce76c7ae</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm4007774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4007774%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DSynthesis%252C%2520Structure%252C%2520and%2520Antibiotic%2520Activity%2520of%2520Aryl-Substituted%2520LpxC%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6954%26epage%3D6966%26doi%3D10.1021%2Fjm4007774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barb, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span> <span> </span><span class="NLM_article-title">Structure of the Deacetylase LpxC Bound to the Antibiotic CHIR-090: Time-Dependent Inhibition and Specificity in Ligand Binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">18433</span>– <span class="NLM_lpage">18438</span>, <span class="refDoi"> DOI: 10.1073/pnas.0709412104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1073%2Fpnas.0709412104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=18025458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2ku7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18433-18438&author=A.+W.+Barbauthor=L.+Jiangauthor=C.+R.+Raetzauthor=P.+Zhou&title=Structure+of+the+Deacetylase+LpxC+Bound+to+the+Antibiotic+CHIR-090%3A+Time-Dependent+Inhibition+and+Specificity+in+Ligand+Binding&doi=10.1073%2Fpnas.0709412104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the deacetylase LpxC bound to the antibiotic CHIR-090: time-dependent inhibition and specificity in ligand binding</span></div><div class="casAuthors">Barb, Adam W.; Jiang, Ling; Raetz, Christian R. H.; Zhou, Pei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">18433-18438, S18433/1-S18433/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The UDP-3-O-(R-3-hydroxyacyl)-N-acetylglucosamine deacetylase LpxC is an essential enzyme of lipid A biosynthesis in Gram-neg. bacteria and a promising antibiotic target.  CHIR-090, the most potent LpxC inhibitor discovered to date, displays two-step time-dependent inhibition and kills a wide range of Gram-neg. pathogens as effectively as ciprofloxacin or tobramycin.  In this study, we report the soln. structure of the LpxC-CHIR-090 complex.  CHIR-090 exploits conserved features of LpxC that are crit. for catalysis, including the hydrophobic passage and essential active-site residues.  CHIR-090 is adjacent to, but does not occupy, the UDP-binding pocket of LpxC, suggesting that a fragment-based approach may facilitate further optimization of LpxC inhibitors.  Addnl., we identified key residues in the Insert II hydrophobic passage that modulate time-dependent inhibition and CHIR-090 resistance.  CHIR-090 shares a similar, although previously unrecognized, chem. scaffold with other small-mol. antibiotics such as L-161,240 targeting LpxC, and provides a template for understanding the binding mode of these inhibitors.  Consistent with this model, we provide evidence that L-161,240 also occupies the hydrophobic passage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraWHV0AFZnM7Vg90H21EOLACvtfcHk0lgtxSHbZuwHnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2ku7rP&md5=e54a3a1c978536947380f6f1bbdbc4bf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709412104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709412104%26sid%3Dliteratum%253Aachs%26aulast%3DBarb%26aufirst%3DA.%2BW.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DStructure%2520of%2520the%2520Deacetylase%2520LpxC%2520Bound%2520to%2520the%2520Antibiotic%2520CHIR-090%253A%2520Time-Dependent%2520Inhibition%2520and%2520Specificity%2520in%2520Ligand%2520Binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D18433%26epage%3D18438%26doi%3D10.1073%2Fpnas.0709412104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piizzi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wattanasin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanieri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojkovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzink-Fox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lilly, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranjitkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, C. R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Properties of a Potent Inhibitor of <i>Pseudomonas aeruginosa</i> Deacetylase LpxC</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5002</span>– <span class="NLM_lpage">5014</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00377</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00377" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1SgsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5002-5014&author=G.+Piizziauthor=D.+T.+Parkerauthor=Y.+Pengauthor=M.+Doblerauthor=A.+Patnaikauthor=S.+Wattanasinauthor=E.+Liuauthor=F.+Lenoirauthor=J.+Nunezauthor=J.+Kerriganauthor=D.+McKenneyauthor=C.+Osborneauthor=D.+Yuauthor=L.+Lanieriauthor=J.+Bojkovicauthor=J.+Dzink-Foxauthor=M.+D.+Lillyauthor=E.+R.+Spragueauthor=Y.+Luauthor=H.+Wangauthor=S.+Ranjitkarauthor=L.+Xieauthor=B.+Wangauthor=M.+Glickauthor=L.+G.+Hamannauthor=R.+Tommasiauthor=X.+Yangauthor=C.+R.+Dean&title=Design%2C+Synthesis%2C+and+Properties+of+a+Potent+Inhibitor+of+Pseudomonas+aeruginosa+Deacetylase+LpxC&doi=10.1021%2Facs.jmedchem.7b00377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Properties of a Potent Inhibitor of Pseudomonas aeruginosa Deacetylase LpxC</span></div><div class="casAuthors">Piizzi, Grazia; Parker, David T.; Peng, Yunshan; Dobler, Markus; Patnaik, Anup; Wattanasin, Som; Liu, Eugene; Lenoir, Francois; Nunez, Jill; Kerrigan, John; McKenney, David; Osborne, Colin; Yu, Donghui; Lanieri, Leanne; Bojkovic, Jade; Dzink-Fox, JoAnn; Lilly, Maria-Dawn; Sprague, Elizabeth R.; Lu, Yipin; Wang, Hongming; Ranjitkar, Srijan; Xie, Lili; Wang, Bing; Glick, Meir; Hamann, Lawrence G.; Tommasi, Ruben; Yang, Xia; Dean, Charles R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5002-5014</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Over the past several decades, the frequency of antibacterial resistance in hospitals, including multidrug resistance (MDR) and its assocn. with serious infectious diseases, has increased at alarming rates.  Pseudomonas aeruginosa is a leading cause of nosocomial infections, and resistance to virtually all approved antibacterial agents is emerging in this pathogen.  To address the need for new agents to treat MDR P. aeruginosa, we focused on inhibiting the first committed step in the biosynthesis of lipid A, the deacetylation of uridyldiphospho-3-O-(R-hydroxydecanoyl)-N-acetylglucosamine by the enzyme LpxC.  We approached this through the design, synthesis, and biol. evaluation of novel hydroxamic acid LpxC inhibitors, exemplified by I, where cytotoxicity against mammalian cell lines was reduced, soly. and plasma-protein binding were improved while retaining potent anti-pseudomonal activity in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbG9H5F_ITrVg90H21EOLACvtfcHk0lj4p-CPqhyvSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1SgsLk%253D&md5=ea3fc92a1b15ffed9edd3702395a2b35</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00377%26sid%3Dliteratum%253Aachs%26aulast%3DPiizzi%26aufirst%3DG.%26aulast%3DParker%26aufirst%3DD.%2BT.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DDobler%26aufirst%3DM.%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DWattanasin%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DE.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DNunez%26aufirst%3DJ.%26aulast%3DKerrigan%26aufirst%3DJ.%26aulast%3DMcKenney%26aufirst%3DD.%26aulast%3DOsborne%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DLanieri%26aufirst%3DL.%26aulast%3DBojkovic%26aufirst%3DJ.%26aulast%3DDzink-Fox%26aufirst%3DJ.%26aulast%3DLilly%26aufirst%3DM.%2BD.%26aulast%3DSprague%26aufirst%3DE.%2BR.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DRanjitkar%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DTommasi%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DDean%26aufirst%3DC.%2BR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Properties%2520of%2520a%2520Potent%2520Inhibitor%2520of%2520Pseudomonas%2520aeruginosa%2520Deacetylase%2520LpxC%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5002%26epage%3D5014%26doi%3D10.1021%2Facs.jmedchem.7b00377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcari, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collantes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engtrakul, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knafels, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishka, D. G.</span></span> <span> </span><span class="NLM_article-title">Potent Inhibitors of LpxC for the Treatment of Gram-Negative Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">914</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1021/jm2014748</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2014748" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ers7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=914-923&author=M.+F.+Brownauthor=U.+Reillyauthor=J.+A.+Abramiteauthor=J.+T.+Arcariauthor=R.+Oliverauthor=R.+A.+Barhamauthor=Y.+Cheauthor=J.+M.+Chenauthor=E.+M.+Collantesauthor=S.+W.+Chungauthor=C.+Desbonnetauthor=J.+Dotyauthor=M.+Doroskiauthor=J.+J.+Engtrakulauthor=T.+M.+Harrisauthor=M.+Hubandauthor=J.+D.+Knafelsauthor=K.+L.+Leachauthor=S.+Liuauthor=A.+Marfatauthor=A.+Marraauthor=E.+McElroyauthor=M.+Melnickauthor=C.+A.+Menardauthor=J.+I.+Montgomeryauthor=L.+Mullinsauthor=M.+C.+Noeauthor=J.+O%E2%80%99Donnellauthor=J.+Penzienauthor=M.+S.+Plummerauthor=L.+M.+Priceauthor=V.+Shanmugasundaramauthor=C.+Thomaauthor=D.+P.+Uccelloauthor=J.+S.+Warmusauthor=D.+G.+Wishka&title=Potent+Inhibitors+of+LpxC+for+the+Treatment+of+Gram-Negative+Infections&doi=10.1021%2Fjm2014748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Inhibitors of LpxC for the Treatment of Gram-Negative Infections</span></div><div class="casAuthors">Brown, Matthew F.; Reilly, Usa; Abramite, Joseph A.; Arcari, Joel T.; Oliver, Robert; Barham, Rose A.; Che, Ye; Chen, Jinshan Michael; Collantes, Elizabeth M.; Chung, Seung Won; Desbonnet, Charlene; Doty, Jonathan; Doroski, Matthew; Engtrakul, Juntyma J.; Harris, Thomas M.; Huband, Michael; Knafels, John D.; Leach, Karen L.; Liu, Shenping; Marfat, Anthony; Marra, Andrea; McElroy, Eric; Melnick, Michael; Menard, Carol A.; Montgomery, Justin I.; Mullins, Lisa; Noe, Mark. C.; O'Donnell, John; Penzien, Joseph; Plummer, Mark S.; Price, Loren M.; Shanmugasundaram, Veerabahu; Thoma, Christy; Uccello, Daniel P.; Warmus, Joseph S.; Wishka, Donn G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">914-923</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the synthesis and SAR as well as selectivity, pharmacokinetic, and infection model data for representative analogs of a novel series of potent antibacterial LpxC inhibitors represented by hydroxamic acid I, is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpizPNDLsgyIbVg90H21EOLACvtfcHk0lgO9Y4GzjJH9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1ers7rE&md5=fda4af453d09a4baa25cac35f69f5ac3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm2014748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2014748%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DArcari%26aufirst%3DJ.%2BT.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DBarham%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DCollantes%26aufirst%3DE.%2BM.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DDoroski%26aufirst%3DM.%26aulast%3DEngtrakul%26aufirst%3DJ.%2BJ.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKnafels%26aufirst%3DJ.%2BD.%26aulast%3DLeach%26aufirst%3DK.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMarra%26aufirst%3DA.%26aulast%3DMcElroy%26aufirst%3DE.%26aulast%3DMelnick%26aufirst%3DM.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DPenzien%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DM.%2BS.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DThoma%26aufirst%3DC.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26aulast%3DWishka%26aufirst%3DD.%2BG.%26atitle%3DPotent%2520Inhibitors%2520of%2520LpxC%2520for%2520the%2520Treatment%2520of%2520Gram-Negative%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D914%26epage%3D923%26doi%3D10.1021%2Fjm2014748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collantes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engtrakul, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knafels, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishka, D. G.</span></span> <span> </span><span class="NLM_article-title">Pyridone Methylsulfone Hydroxamate LpxC Inhibitors for the Treatment of Serious Gram-Negative Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1670</span>, <span class="refDoi"> DOI: 10.1021/jm2014875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2014875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1662-1670&author=J.+I.+Montgomeryauthor=M.+F.+Brownauthor=U.+Reillyauthor=L.+M.+Priceauthor=J.+A.+Abramiteauthor=J.+Arcariauthor=R.+Barhamauthor=Y.+Cheauthor=J.+M.+Chenauthor=S.+W.+Chungauthor=E.+M.+Collantesauthor=C.+Desbonnetauthor=M.+Doroskiauthor=J.+Dotyauthor=J.+J.+Engtrakulauthor=T.+M.+Harrisauthor=M.+Hubandauthor=J.+D.+Knafelsauthor=K.+L.+Leachauthor=S.+Liuauthor=A.+Marfatauthor=L.+McAllisterauthor=E.+McElroyauthor=C.+A.+Menardauthor=M.+Mitton-Fryauthor=L.+Mullinsauthor=M.+C.+Noeauthor=J.+O%E2%80%99Donnellauthor=R.+Oliverauthor=J.+Penzienauthor=M.+Plummerauthor=V.+Shanmugasundaramauthor=C.+Thomaauthor=A.+P.+Tomarasauthor=D.+P.+Uccelloauthor=A.+Vazauthor=D.+G.+Wishka&title=Pyridone+Methylsulfone+Hydroxamate+LpxC+Inhibitors+for+the+Treatment+of+Serious+Gram-Negative+Infections&doi=10.1021%2Fjm2014875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridone methylsulfone hydroxamate LpxC inhibitors for the treatment of serious gram-negative infections</span></div><div class="casAuthors">Montgomery, Justin I.; Brown, Matthew F.; Reilly, Usa; Price, Loren M.; Abramite, Joseph A.; Arcari, Joel; Barham, Rose; Che, Ye; Chen, Jinshan Michael; Chung, Seung Won; Collantes, Elizabeth M.; Desbonnet, Charlene; Doroski, Matthew; Doty, Jonathan; Engtrakul, Juntyma J.; Harris, Thomas M.; Huband, Michael; Knafels, John D.; Leach, Karen L.; Liu, Shenping; Marfat, Anthony; McAllister, Laura; McElroy, Eric; Menard, Carol A.; Mitton-Fry, Mark; Mullins, Lisa; Noe, Mark C.; O'Donnell, John; Oliver, Robert; Penzien, Joseph; Plummer, Mark; Shanmugasundaram, Veerabahu; Thoma, Christy; Tomaras, Andrew P.; Uccello, Daniel P.; Vaz, Alfin; Wishka, Donn G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1662-1670</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and biol. activity of a new series of LpxC inhibitors represented by pyridone methylsulfone hydroxamate 2a (I) is presented.  Members of this series have improved soly. and free fraction when compared to compds. in the previously described biphenyl methylsulfone hydroxamate series, and they maintain superior Gram-neg. antibacterial activity to comparator agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgNGd-bEXEabVg90H21EOLACvtfcHk0lggCeU9WRwemg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D&md5=f2410aae177b2fa24d8da72942b95564</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm2014875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2014875%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DArcari%26aufirst%3DJ.%26aulast%3DBarham%26aufirst%3DR.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DCollantes%26aufirst%3DE.%2BM.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DDoroski%26aufirst%3DM.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DEngtrakul%26aufirst%3DJ.%2BJ.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKnafels%26aufirst%3DJ.%2BD.%26aulast%3DLeach%26aufirst%3DK.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcAllister%26aufirst%3DL.%26aulast%3DMcElroy%26aufirst%3DE.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMitton-Fry%26aufirst%3DM.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DPenzien%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DThoma%26aufirst%3DC.%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DVaz%26aufirst%3DA.%26aulast%3DWishka%26aufirst%3DD.%2BG.%26atitle%3DPyridone%2520Methylsulfone%2520Hydroxamate%2520LpxC%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Serious%2520Gram-Negative%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1662%26epage%3D1670%26doi%3D10.1021%2Fjm2014875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Schmitt, C.</span>; <span class="NLM_string-name">Surivet, J.-P.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrrolidinone Derivatives for Use as Antibacterial Agents</span>. World Patent <span class="NLM_patent">WO 2017037039</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=P.+Panchaud&author=C.+Schmitt&author=J.-P.+Surivet&title=Preparation+of+Pyrrolidinone+Derivatives+for+Use+as+Antibacterial+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPanchaud%26aufirst%3DP.%26atitle%3DPreparation%2520of%2520Pyrrolidinone%2520Derivatives%2520for%2520Use%2520as%2520Antibacterial%2520Agents%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Specklin, J.-L.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Schmitt, C.</span></span> <span> </span><span class="NLM_article-title">Preparation of Dihydropyrrolo[1, 2-<i>c</i>]imidazol-3-one Derivatives Useful as Antibacterial Agents</span>. World Patent <span class="NLM_patent">WO 2015173329</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=G.+Chapoux&author=J.-C.+Gauvin&author=P.+Panchaud&author=J.-L.+Specklin&author=J.-P.+Surivet&author=C.+Schmitt&title=Preparation+of+Dihydropyrrolo%5B1%2C+2-c%5Dimidazol-3-one+Derivatives+Useful+as+Antibacterial+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChapoux%26aufirst%3DG.%26atitle%3DPreparation%2520of%2520Dihydropyrrolo%255B1%252C%25202-c%255Dimidazol-3-one%2520Derivatives%2520Useful%2520as%2520Antibacterial%2520Agents%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit25b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Blumstein, A. C.</span>; <span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Jacob, L.</span>; <span class="NLM_string-name">Masse, F.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Schmitt, C.</span>; <span class="NLM_string-name">Surivet, J. P.</span>; <span class="NLM_string-name">Specklin, J. L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Substituted 1,2-Dihydro-3<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-3-one Antibacterial Compounds</span>. World Patent <span class="NLM_patent">WO 2017037221</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=A.+C.+Blumstein&author=G.+Chapoux&author=L.+Jacob&author=F.+Masse&author=A.+Mirre&author=P.+Panchaud&author=C.+Schmitt&author=J.+P.+Surivet&author=J.+L.+Specklin&title=Preparation+of+Substituted+1%2C2-Dihydro-3H-pyrrolo%5B1%2C2-c%5Dimidazol-3-one+Antibacterial+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBlumstein%26aufirst%3DA.%2BC.%26atitle%3DPreparation%2520of%2520Substituted%25201%252C2-Dihydro-3H-pyrrolo%255B1%252C2-c%255Dimidazol-3-one%2520Antibacterial%2520Compounds%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chinchilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nájera, C.</span></span> <span> </span><span class="NLM_article-title">The Sonogashira Reaction: A Booming Methodology in Synthetic Organic Chemistry</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">874</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1021/cr050992x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr050992x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2007&pages=874-922&author=R.+Chinchillaauthor=C.+N%C3%A1jera&title=The+Sonogashira+Reaction%3A+A+Booming+Methodology+in+Synthetic+Organic+Chemistry&doi=10.1021%2Fcr050992x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Sonogashira reaction: a booming methodology in synthetic organic chemistry</span></div><div class="casAuthors">Chinchilla, Rafael; Najera, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">874-922</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The palladium-catalyzed sp2-sp coupling reaction between aryl or alkenyl halides or triflates and terminal alkynes, with or without the presence of copper(I) cocatalyst, was reviewed.  The mechanism of Sonogashira reaction was discussed, and the catalysts and reaction conditions were explored as well.  In addn., the application of Sonogashira cross-coupling reaction was also introduced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ8ePd8RdbSrVg90H21EOLACvtfcHk0lgUA1YgG2yvLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbc%253D&md5=91f627efbf940c60809c2acfe597d3b7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcr050992x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050992x%26sid%3Dliteratum%253Aachs%26aulast%3DChinchilla%26aufirst%3DR.%26aulast%3DN%25C3%25A1jera%26aufirst%3DC.%26atitle%3DThe%2520Sonogashira%2520Reaction%253A%2520A%2520Booming%2520Methodology%2520in%2520Synthetic%2520Organic%2520Chemistry%26jtitle%3DChem.%2520Rev.%26date%3D2007%26volume%3D107%26spage%3D874%26epage%3D922%26doi%3D10.1021%2Fcr050992x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cadiot, P.</span>; <span class="NLM_string-name">Chodkiewicz, W.</span></span> In  <i>Chemistry of Acetylenes</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Viehe, H. G.</span></span>, Ed.; <span class="NLM_publisher-name">Marcel Dekker</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1969</span>; pp  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">647</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1969&pages=597-647&author=P.+Cadiot&author=W.+Chodkiewiczauthor=H.+G.+Viehe&title=Chemistry+of+Acetylenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCadiot%26aufirst%3DP.%26btitle%3DChemistry%2520of%2520Acetylenes%26aulast%3DViehe%26aufirst%3DH.%2BG.%26pub%3DMarcel%2520Dekker%26date%3D1969%26spage%3D597%26epage%3D647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="note"><p class="first last"><i>Klebsiella pneumoniae</i> T6474 is a clinical isolate expressing ESBL and with mutations in the Quinolone Resistance Determining Region of gyrA (Ser83-Ile) and ParC (Ser80-Ile). It is resistant to ampicillin, aztreonam, ceftriaxone, norfloxacin, and trimethoprim.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="note"><p class="first last">For details on assay conditions, calibration and relevance, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The Use of Differential Scanning Fluorimetry to Detect Ligand Interactions That Promote Protein Stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+Use+of+Differential+Scanning+Fluorimetry+to+Detect+Ligand+Interactions+That+Promote+Protein+Stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lhiGcJoCF4-0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520Use%2520of%2520Differential%2520Scanning%2520Fluorimetry%2520to%2520Detect%2520Ligand%2520Interactions%2520That%2520Promote%2520Protein%2520Stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="note"><p class="first last">Data collection and refinements statistics for all structures are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span> <span> </span><span class="NLM_article-title">Heterocyclic Methylsulfone Hydroxamic Acid LpxC Inhibitors as Gram-Negative Antibacterial Agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">6832</span>– <span class="NLM_lpage">6838</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.09.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2Fj.bmcl.2012.09.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=23046961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVykur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=6832-6838&author=L.+A.+McAllisterauthor=J.+I.+Montgomeryauthor=J.+A.+Abramiteauthor=U.+Reillyauthor=M.+F.+Brownauthor=J.+M.+Chenauthor=R.+A.+Barhamauthor=Y.+Cheauthor=S.+W.+Chungauthor=C.+A.+Menardauthor=M.+Mitton-Fryauthor=L.+M.+Mullinsauthor=M.+C.+Noeauthor=J.+P.+O%E2%80%99Donnellauthor=R.+M.+Oliverauthor=J.+B.+Penzienauthor=M.+Plummerauthor=L.+M.+Priceauthor=V.+Shanmugasundaramauthor=A.+P.+Tomarasauthor=D.+P.+Uccello&title=Heterocyclic+Methylsulfone+Hydroxamic+Acid+LpxC+Inhibitors+as+Gram-Negative+Antibacterial+Agents&doi=10.1016%2Fj.bmcl.2012.09.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Heterocyclic methylsulfone hydroxamic acid LpxC inhibitors as Gram-negative antibacterial agents</span></div><div class="casAuthors">McAllister, Laura A.; Montgomery, Justin I.; Abramite, Joseph A.; Reilly, Usa; Brown, Matthew F.; Chen, Jinshan M.; Barham, Rose A.; Che, Ye; Chung, Seung Won; Menard, Carol A.; Mitton-Fry, Mark; Mullins, Lisa M.; Noe, Mark C.; O'Donnell, John P.; Oliver, Robert M.; Penzien, Joseph B.; Plummer, Mark; Price, Loren M.; Shanmugasundaram, Veerabahu; Tomaras, Andrew P.; Uccello, Daniel P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6832-6838</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis and antibacterial activity of heterocyclic methylsulfone hydroxamates is presented.  Compds. in this series are potent inhibitors of the LpxC enzyme, a key enzyme involved in the prodn. of lipopolysaccharide (LPS) found in the outer membrane of Gram-neg. bacteria.  SAR evaluation of compds. in this series revealed analogs, e.g., I, with potent antibacterial activity against challenging Gram-neg. species such as Pseudomonas aeruginosa and Klebsiella pneumoniae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_EQM-fls8LVg90H21EOLACvtfcHk0ljug0DBQK9ZIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVykur7P&md5=93dbfffc47f4eab6b67111ef7e67fa2e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.09.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.09.058%26sid%3Dliteratum%253Aachs%26aulast%3DMcAllister%26aufirst%3DL.%2BA.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DBarham%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMitton-Fry%26aufirst%3DM.%26aulast%3DMullins%26aufirst%3DL.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26aulast%3DOliver%26aufirst%3DR.%2BM.%26aulast%3DPenzien%26aufirst%3DJ.%2BB.%26aulast%3DPlummer%26aufirst%3DM.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26atitle%3DHeterocyclic%2520Methylsulfone%2520Hydroxamic%2520Acid%2520LpxC%2520Inhibitors%2520as%2520Gram-Negative%2520Antibacterial%2520Agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D6832%26epage%3D6838%26doi%3D10.1016%2Fj.bmcl.2012.09.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32a"><span>Related
2<i>H</i>-indazoles and benzothiazoles derivatives that
were synthesiszed and characterized during the course of our program
elicited somewhat lower solubilites and higher log <i>D</i><sub>7.4</sub> (data not shown). <span class="NLM_contrib-group"><span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Ochala, E.</span>; <span class="NLM_string-name">Surivet, J.-P.</span></span> <span> </span><span class="NLM_article-title">2<i>H</i>-Indazole Derivatives as Antibacterial and Their Preparation and Use</span>. World Patent <span class="NLM_patent">WO 2015036964</span>, <span class="NLM_year">2015</span>;.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.-C.+Gauvin&author=A.+Mirre&author=E.+Ochala&author=J.-P.+Surivet&title=2H-Indazole+Derivatives+as+Antibacterial+and+Their+Preparation+and+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGauvin%26aufirst%3DJ.-C.%26atitle%3D2H-Indazole%2520Derivatives%2520as%2520Antibacterial%2520and%2520Their%2520Preparation%2520and%2520Use%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit32b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, C.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Specklin, J.-L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Benzothiazole Derivatives as Antibacterial Agents</span>. World Patent <span class="NLM_patent">WO 2016079688</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=C.+Schmitt&author=J.-P.+Surivet&author=G.+Chapoux&author=A.+Mirre&author=J.-L.+Specklin&title=Preparation+of+Benzothiazole+Derivatives+as+Antibacterial+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DC.%26atitle%3DPreparation%2520of%2520Benzothiazole%2520Derivatives%2520as%2520Antibacterial%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Diethelm, S.</span>; <span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Surivet, J.-P.</span></span> <span> </span><span class="NLM_article-title">Preparation of Functionalized Benzoxazolones and Benzothiazolones as Antibacterial Compounds</span>. World Patent <span class="NLM_patent">WO 2017037039</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Chapoux&author=S.+Diethelm&author=J.-C.+Gauvin&author=P.+Panchaud&author=J.-P.+Surivet&title=Preparation+of+Functionalized+Benzoxazolones+and+Benzothiazolones+as+Antibacterial+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChapoux%26aufirst%3DG.%26atitle%3DPreparation%2520of%2520Functionalized%2520Benzoxazolones%2520and%2520Benzothiazolones%2520as%2520Antibacterial%2520Compounds%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Diethelm, S.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumstein, A.-C.</span>; <span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Jacob, L.</span>; <span class="NLM_string-name">Masse, F.</span>; <span class="NLM_string-name">Mathieu, G.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Schmitt, C.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Enderlin-Paput, M.</span>; <span class="NLM_string-name">Lange, R.</span>; <span class="NLM_string-name">Gnerre, C.</span>; <span class="NLM_string-name">Seeland, S.</span>; <span class="NLM_string-name">Herrmann, C.</span>; <span class="NLM_string-name">Locher, H. H.</span>; <span class="NLM_string-name">Seiler, P.</span>; <span class="NLM_string-name">Ritz, D.</span>; <span class="NLM_string-name">Rueedi, G.</span></span> <span> </span><span class="NLM_article-title">Optimization of LpxC Inhibitor Lead Compounds Focusing on Efficacy and Formulation for High Dose Intravenous Administration</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01605</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+Panchaud&author=J.-P.+Surivet&author=S.+Diethelmauthor=A.-C.+Blumstein&author=J.-C.+Gauvin&author=L.+Jacob&author=F.+Masse&author=G.+Mathieu&author=A.+Mirre&author=C.+Schmittauthor=M.+Enderlin-Paput&author=R.+Lange&author=C.+Gnerre&author=S.+Seeland&author=C.+Herrmann&author=H.+H.+Locher&author=P.+Seiler&author=D.+Ritz&author=G.+Rueedi&title=Optimization+of+LpxC+Inhibitor+Lead+Compounds+Focusing+on+Efficacy+and+Formulation+for+High+Dose+Intravenous+Administration&doi=10.1021%2Facs.jmedchem.9b01605"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01605%26sid%3Dliteratum%253Aachs%26aulast%3DPanchaud%26aufirst%3DP.%26aulast%3DBlumstein%26aufirst%3DA.-C.%26aulast%3DEnderlin-Paput%26aufirst%3DM.%26atitle%3DOptimization%2520of%2520LpxC%2520Inhibitor%2520Lead%2520Compounds%2520Focusing%2520on%2520Efficacy%2520and%2520Formulation%2520for%2520High%2520Dose%2520Intravenous%2520Administration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.9b01605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zissimos, A. M.</span></span> <span> </span><span class="NLM_article-title">Fast Calculation of van der Waals Volume as a Sum of Atomic and Bond Contributions and Its Application to Drug Compounds</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">7368</span>– <span class="NLM_lpage">7373</span>, <span class="refDoi"> DOI: 10.1021/jo034808o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo034808o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=7368-7373&author=Y.+H.+Zhaoauthor=M.+H.+Abrahamauthor=A.+M.+Zissimos&title=Fast+Calculation+of+van+der+Waals+Volume+as+a+Sum+of+Atomic+and+Bond+Contributions+and+Its+Application+to+Drug+Compounds&doi=10.1021%2Fjo034808o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of van der Waals Volume as a Sum of Atomic and Bond Contributions and Its Application to Drug Compounds</span></div><div class="casAuthors">Zhao, Yuan H.; Abraham, Michael H.; Zissimos, Andreas M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7368-7373</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The van der Waals vol. is a widely used descriptor in modeling physicochem. properties. However, the calcn. of the van der Waals vol. (VvdW) is rather time-consuming, from Bondi group contributions, for a large data set.  A new method for calcg. van der Waals vol. has been developed, based on Bondi radii.  The method, termed At. and Bond Contributions of van der Waals vol. (VABC), is very simple and fast.  The only information needed for calcg. VABC is at. contributions and the no. of atoms, bonds, and rings. Then, the van der Waals vol. (Å3/mol.) can be calcd. from the following formula: VvdW = .sum. all atom contributions - 5.92NB - 14.7RA - 3.8RNR (NB is the no. of bonds, RA is the no. of arom. rings, and RNA is the no. of nonarom. rings).  The no. of bonds present (NB) can be simply calcd. by NB = N - 1 + RA + RNA (where N is the total no. of atoms).  A simple Excel spread sheet has been made to calc. van der Waals vols. for a wide range of 677 org. compds., including 237 drug compds.  The results show that the van der Waals vols. calcd. from VABC are equiv. to the computer-calcd. van der Waals vols. for org. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojBiFlWWZKjbVg90H21EOLACvtfcHk0lj5KefSjzURbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D&md5=853cd54edc7a8c07e427196ee30d198f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjo034808o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034808o%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%2BH.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DZissimos%26aufirst%3DA.%2BM.%26atitle%3DFast%2520Calculation%2520of%2520van%2520der%2520Waals%2520Volume%2520as%2520a%2520Sum%2520of%2520Atomic%2520and%2520Bond%2520Contributions%2520and%2520Its%2520Application%2520to%2520Drug%2520Compounds%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D7368%26epage%3D7373%26doi%3D10.1021%2Fjo034808o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="note"><p class="first last">Structure–activity relationship studies focused on alternative segments <b>A</b> will be presented elsewhere.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasar, R.</span>; <span class="NLM_string-name">Linsell, M. S.</span>; <span class="NLM_string-name">Aggen, J. B.</span>; <span class="NLM_string-name">Lu, Q.</span>; <span class="NLM_string-name">Wang, D.</span>; <span class="NLM_string-name">Church, T.</span>; <span class="NLM_string-name">Moser, H. E.</span>; <span class="NLM_string-name">Patten, P. A.</span></span> <span> </span><span class="NLM_article-title">Hydroxamic Acid Derivatives and Their Use in the Treatment of Bacterial Infections</span>. World Patent <span class="NLM_patent">WO 2012154204</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=R.+Kasar&author=M.+S.+Linsell&author=J.+B.+Aggen&author=Q.+Lu&author=D.+Wang&author=T.+Church&author=H.+E.+Moser&author=P.+A.+Patten&title=Hydroxamic+Acid+Derivatives+and+Their+Use+in+the+Treatment+of+Bacterial+Infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKasar%26aufirst%3DR.%26atitle%3DHydroxamic%2520Acid%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Bacterial%2520Infections%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">See also</p></div><div class="NLM_citation" id="cit37b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assar, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dozzo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easterday, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haglund, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrandt, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konradi, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machajewski, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miroshnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plato, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seroogy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhmin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviridov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlasichuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirz, R. T.</span></span> <span> </span><span class="NLM_article-title">Optimization of LpxC Inhibitors for Antibacterial Activity and Cardiovascular Safety</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1002%2Fcmdc.201900287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=31283109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1560-1572&author=F.+Cohenauthor=J.+B.+Aggenauthor=L.+D.+Andrewsauthor=Z.+Assarauthor=J.+Boggsauthor=T.+Choiauthor=P.+Dozzoauthor=A.+N.+Easterdayauthor=C.+M.+Haglundauthor=D.+J.+Hildebrandtauthor=M.+C.+Holtauthor=K.+Jolyauthor=A.+Jubbauthor=Z.+Kamalauthor=T.+R.+Kaneauthor=A.+W.+Konradiauthor=K.+M.+Krauseauthor=M.+S.+Linsellauthor=T.+D.+Machajewskiauthor=O.+Miroshnikovaauthor=H.+E.+Moserauthor=V.+Nietoauthor=T.+Phanauthor=C.+Platoauthor=A.+W.+Serioauthor=J.+Seroogyauthor=A.+Shakhminauthor=A.+J.+Steinauthor=A.+D.+Sunauthor=S.+Sviridovauthor=Z.+Wangauthor=K.+Wlasichukauthor=W.+Yangauthor=X.+Zhouauthor=H.+Zhuauthor=R.+T.+Cirz&title=Optimization+of+LpxC+Inhibitors+for+Antibacterial+Activity+and+Cardiovascular+Safety&doi=10.1002%2Fcmdc.201900287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37bR"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of LpxC Inhibitors for Antibacterial Activity and Cardiovascular Safety</span></div><div class="casAuthors">Cohen, Frederick; Aggen, James B.; Andrews, Logan D.; Assar, Zahra; Boggs, Jen; Choi, Taylor; Dozzo, Paola; Easterday, Ashton N.; Haglund, Cat M.; Hildebrandt, Darin J.; Holt, Melissa C.; Joly, Kristin; Jubb, Adrian; Kamal, Zeeshan; Kane, Timothy R.; Konradi, Andrei W.; Krause, Kevin M.; Linsell, Martin S.; Machajewski, Timothy D.; Miroshnikova, Olga; Moser, Heinz E.; Nieto, Vincent; Phan, Thu; Plato, Craig; Serio, Alisa W.; Seroogy, Julie; Shakhmin, Anton; Stein, Adam J.; Sun, Alex D.; Sviridov, Serguei; Wang, Zhan; Wlasichuk, Kenneth; Yang, Wen; Zhou, Xiaoming; Zhu, Hai; Cirz, Ryan T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1560-1572</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a Zn2+ deacetylase that is essential for the survival of most pathogenic Gram-neg. bacteria.  ACHN-975 (N-((S)-3-amino-1-(hydroxyamino)-3-methyl-1-oxobutan-2-yl)-4-(((1R,2R)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)benzamide) was the first LpxC inhibitor to reach human clin. testing and was discovered to have a dose-limiting cardiovascular toxicity of transient hypotension without compensatory tachycardia.  Herein we report the effort beyond ACHN-975 to discover LpxC inhibitors optimized for enzyme potency, antibacterial activity, pharmacokinetics, and cardiovascular safety.  Based on its overall profile, compd. 26 (LPXC-516, (S)-N-(2-(hydroxyamino)-1-(3-methoxy-1,1-dioxidothietan-3-yl)-2-oxoethyl)-4-(6-hydroxyhexa-1,3-diyn-1-yl)benzamide) was chosen for further development.  A phosphate prodrug of 26 was developed that provided a soly. of >30 mg mL-1 for parenteral administration and conversion into the active drug with a t1/2 of approx. two minutes.  Unexpectedly, and despite our optimization efforts, the prodrug of 26 still possesses a therapeutic window insufficient to support further clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNhe0SbdQ2NbVg90H21EOLACvtfcHk0lgde8MXd8gTBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N&md5=b8cc87e1ebb76dfe8d330729a9d048d6</span></div><a href="/servlet/linkout?suffix=cit37b&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900287%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DAggen%26aufirst%3DJ.%2BB.%26aulast%3DAndrews%26aufirst%3DL.%2BD.%26aulast%3DAssar%26aufirst%3DZ.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DT.%26aulast%3DDozzo%26aufirst%3DP.%26aulast%3DEasterday%26aufirst%3DA.%2BN.%26aulast%3DHaglund%26aufirst%3DC.%2BM.%26aulast%3DHildebrandt%26aufirst%3DD.%2BJ.%26aulast%3DHolt%26aufirst%3DM.%2BC.%26aulast%3DJoly%26aufirst%3DK.%26aulast%3DJubb%26aufirst%3DA.%26aulast%3DKamal%26aufirst%3DZ.%26aulast%3DKane%26aufirst%3DT.%2BR.%26aulast%3DKonradi%26aufirst%3DA.%2BW.%26aulast%3DKrause%26aufirst%3DK.%2BM.%26aulast%3DLinsell%26aufirst%3DM.%2BS.%26aulast%3DMachajewski%26aufirst%3DT.%2BD.%26aulast%3DMiroshnikova%26aufirst%3DO.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26aulast%3DNieto%26aufirst%3DV.%26aulast%3DPhan%26aufirst%3DT.%26aulast%3DPlato%26aufirst%3DC.%26aulast%3DSerio%26aufirst%3DA.%2BW.%26aulast%3DSeroogy%26aufirst%3DJ.%26aulast%3DShakhmin%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DA.%2BJ.%26aulast%3DSun%26aufirst%3DA.%2BD.%26aulast%3DSviridov%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWlasichuk%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DCirz%26aufirst%3DR.%2BT.%26atitle%3DOptimization%2520of%2520LpxC%2520Inhibitors%2520for%2520Antibacterial%2520Activity%2520and%2520Cardiovascular%2520Safety%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D1560%26epage%3D1572%26doi%3D10.1002%2Fcmdc.201900287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="note"><p class="first last">All animal housing and experiments were performed in agreement with the Swiss Federal Ordinance for Animal Protection, the animal welfare guidelines from the Cantonal Veterinary Office Basellandschaft, and the Idorsia Pharmaceuticals Ltd. internal animal welfare guidelines.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, W. A.</span></span> <span> </span><span class="NLM_article-title">Animal Model Pharmacokinetics and Pharmacodynamics: a Critical Review</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/S0924-8579(02)00022-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2FS0924-8579%2802%2900022-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=11978497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtVeis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2002&pages=261-268&author=D.+Andesauthor=W.+A.+Craig&title=Animal+Model+Pharmacokinetics+and+Pharmacodynamics%3A+a+Critical+Review&doi=10.1016%2FS0924-8579%2802%2900022-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Animal model pharmacokinetics and pharmacodynamics: a critical review</span></div><div class="casAuthors">Andes, D.; Craig, W. A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-268</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  Animals have been extensively used in the evaluation of antimicrobials.  The value of animals in the pharmacokinetic and pharmacodynamic characterization of antimicrobials is critically reviewed.  Animal studies have demonstrated that the pharmacokinetic/pharmacodynamic (PK/PD) target detg. efficacy can vary for different classes of antimicrobials.  However, the magnitude of the target required for bacteriol. efficacy is relatively similar for various sites of infection, various pathogens and various drugs within the same class, provided free drug levels are used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEIxgWGnD1TrVg90H21EOLACvtfcHk0lhYUmpxTBDgSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtVeis7c%253D&md5=9167c5428197cce09400e0a1a438b02e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0924-8579%2802%2900022-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0924-8579%252802%252900022-5%26sid%3Dliteratum%253Aachs%26aulast%3DAndes%26aufirst%3DD.%26aulast%3DCraig%26aufirst%3DW.%2BA.%26atitle%3DAnimal%2520Model%2520Pharmacokinetics%2520and%2520Pharmacodynamics%253A%2520a%2520Critical%2520Review%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2002%26volume%3D19%26spage%3D261%26epage%3D268%26doi%3D10.1016%2FS0924-8579%2802%2900022-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carifa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidem, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. P.</span></span> <span> </span><span class="NLM_article-title">LpxC Inhibitors as New Antibacterial Agents and Tools for Studying Regulation of Lipid A Biosynthesis in Gram-Negative Pathogens</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e01551-14</span>, <span class="refDoi"> DOI: 10.1128/mBio.01551-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1128%2FmBio.01551-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=25271285" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=e01551-14&author=A.+P.+Tomarasauthor=C.+J.+McPhersonauthor=M.+Kuhnauthor=A.+Carifaauthor=L.+Mullinsauthor=D.+Georgeauthor=C.+Desbonnetauthor=T.+M.+Eidemauthor=J.+I.+Montgomeryauthor=M.+F.+Brownauthor=U.+Reillyauthor=A.+A.+Millerauthor=J.+P.+O%E2%80%99Donnell&title=LpxC+Inhibitors+as+New+Antibacterial+Agents+and+Tools+for+Studying+Regulation+of+Lipid+A+Biosynthesis+in+Gram-Negative+Pathogens&doi=10.1128%2FmBio.01551-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FmBio.01551-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.01551-14%26sid%3Dliteratum%253Aachs%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DMcPherson%26aufirst%3DC.%2BJ.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DCarifa%26aufirst%3DA.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DGeorge%26aufirst%3DD.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DEidem%26aufirst%3DT.%2BM.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DMiller%26aufirst%3DA.%2BA.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26atitle%3DLpxC%2520Inhibitors%2520as%2520New%2520Antibacterial%2520Agents%2520and%2520Tools%2520for%2520Studying%2520Regulation%2520of%2520Lipid%2520A%2520Biosynthesis%2520in%2520Gram-Negative%2520Pathogens%26jtitle%3DmBio%26date%3D2014%26volume%3D5%26spage%3De01551-14%26doi%3D10.1128%2FmBio.01551-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapointe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madera, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yifru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjandra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojkovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uehara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellamacina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonn, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benton, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, Z. K.</span></span> <span> </span><span class="NLM_article-title">Application of Virtual Screening to the Identification of New LpxC Inhibitor Chemotypes, Oxazolidinone and Isoxazoline</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9360</span>– <span class="NLM_lpage">9370</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01287</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01287" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslaqtrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9360-9370&author=P.+S.+Leeauthor=G.+Lapointeauthor=A.+M.+Maderaauthor=R.+Simmonsauthor=W.+Xuauthor=A.+Yifruauthor=M.+Tjandraauthor=S.+Karurauthor=A.+Ricoauthor=K.+Thompsonauthor=J.+Bojkovicauthor=L.+Xieauthor=K.+Ueharaauthor=A.+Liuauthor=W.+Shuauthor=C.+Bellamacinaauthor=D.+McKenneyauthor=L.+Morrisauthor=G.+R.+Tonnauthor=C.+S.+Osborneauthor=B.+Bentonauthor=L.+McDowellauthor=J.+Fuauthor=Z.+K.+Sweeney&title=Application+of+Virtual+Screening+to+the+Identification+of+New+LpxC+Inhibitor+Chemotypes%2C+Oxazolidinone+and+Isoxazoline&doi=10.1021%2Facs.jmedchem.8b01287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Virtual Screening to the Identification of New LpxC Inhibitor Chemotypes, Oxazolidinone and Isoxazoline</span></div><div class="casAuthors">Lee, Patrick S.; Lapointe, Guillaume; Madera, Ann Marie; Simmons, Robert L.; Xu, Wenjian; Yifru, Aregahegn; Tjandra, Meiliana; Karur, Subramanian; Rico, Alice; Thompson, Katherine; Bojkovic, Jade; Xie, Lili; Uehara, Kyoko; Liu, Amy; Shu, Wei; Bellamacina, Cornelia; McKenney, David; Morris, Laura; Tonn, George R.; Osborne, Colin; Benton, Bret M.; McDowell, Laura; Fu, Jiping; Sweeney, Zachary K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9360-9370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report summarizes the identification and synthesis of novel LpxC inhibitors aided by computational methods that leveraged numerous crystal structures.  This effort led to the identification of oxazolidinone and isoxazoline inhibitors with potent in vitro activity against P. aeruginosa and other Gram-neg. bacteria.  Representative compd. 13f demonstrated efficacy against P. aeruginosa in a mouse neutropenic thigh infection model.  The antibacterial activity against K. pneumoniae could be potentiated by Gram-pos. antibiotics rifampicin (RIF) and vancomycin (VAN) in both in vitro and in vivo models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphGwLR2DR7VrVg90H21EOLACvtfcHk0liNYfUXZlTRRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslaqtrvF&md5=73c59bf77a4a9ea5c993d7f935955307</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01287%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DP.%2BS.%26aulast%3DLapointe%26aufirst%3DG.%26aulast%3DMadera%26aufirst%3DA.%2BM.%26aulast%3DSimmons%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DYifru%26aufirst%3DA.%26aulast%3DTjandra%26aufirst%3DM.%26aulast%3DKarur%26aufirst%3DS.%26aulast%3DRico%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DK.%26aulast%3DBojkovic%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DUehara%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DShu%26aufirst%3DW.%26aulast%3DBellamacina%26aufirst%3DC.%26aulast%3DMcKenney%26aufirst%3DD.%26aulast%3DMorris%26aufirst%3DL.%26aulast%3DTonn%26aufirst%3DG.%2BR.%26aulast%3DOsborne%26aufirst%3DC.%2BS.%26aulast%3DBenton%26aufirst%3DB.%26aulast%3DMcDowell%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DSweeney%26aufirst%3DZ.%2BK.%26atitle%3DApplication%2520of%2520Virtual%2520Screening%2520to%2520the%2520Identification%2520of%2520New%2520LpxC%2520Inhibitor%2520Chemotypes%252C%2520Oxazolidinone%2520and%2520Isoxazoline%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9360%26epage%3D9370%26doi%3D10.1021%2Facs.jmedchem.8b01287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25a','cit25b'],'ref26':['cit26'],'ref27':['cit27'],'ref28':[],'ref29':['cit29'],'ref30':[],'ref31':['cit31'],'ref32':['cit32a','cit32b'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':[],'ref37':['cit37a','cit37b'],'ref38':[],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Azza Troudi, Jean Marie Pagès, <span class="NLM_string-name hlFld-ContribAuthor">Jean Michel Brunel</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Highlights Supporting the Role of Lipid A in Efficient Biological Adaptation of Gram-Negative Bacteria to External Stresses. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (4)
                                     , 1816-1834. <a href="https://doi.org/10.1021/acs.jmedchem.0c02185" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02185</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02185%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChemical%252BHighlights%252BSupporting%252Bthe%252BRole%252Bof%252BLipid%252BA%252Bin%252BEfficient%252BBiological%252BAdaptation%252Bof%252BGram-Negative%252BBacteria%252Bto%252BExternal%252BStresses%26aulast%3DTroudi%26aufirst%3DAzza%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D18122020%26date%3D04022021%26volume%3D64%26issue%3D4%26spage%3D1816%26epage%3D1834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Masahiro Abe, Baptiste Picard, <span class="NLM_string-name hlFld-ContribAuthor">Michaël De Paolis</span>. </span><span class="cited-content_cbyCitation_article-title">Convergent Synthesis of 2-Oxazolone-4-carboxylates Esters by Reaction of Aldehydes with Ambivalent N-Cbz-α-Tosylglycinate Ester. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (12)
                                     , 4864-4867. <a href="https://doi.org/10.1021/acs.orglett.0c01703" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c01703</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c01703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c01703%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DConvergent%252BSynthesis%252Bof%252B2-Oxazolone-4-carboxylates%252BEsters%252Bby%252BReaction%252Bof%252BAldehydes%252Bwith%252BAmbivalent%252BN-Cbz-%2525CE%2525B1-Tosylglycinate%252BEster%26aulast%3DAbe%26aufirst%3DMasahiro%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19052020%26date%3D10062020%26volume%3D22%26issue%3D12%26spage%3D4864%26epage%3D4867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Philippe Panchaud, Jean-Philippe Surivet, Stefan Diethelm, Anne-Catherine Blumstein, Jean-Christophe Gauvin, Loïc Jacob, Florence Masse, Gaëlle Mathieu, Azely Mirre, Christine Schmitt, Michel Enderlin-Paput, Roland Lange, Carmela Gnerre, Swen Seeland, Charlyse Herrmann, Hans H. Locher, Peter Seiler, Daniel Ritz, <span class="NLM_string-name hlFld-ContribAuthor">Georg Rueedi</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of LpxC Inhibitor Lead Compounds Focusing on Efficacy and Formulation for High Dose Intravenous Administration. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (1)
                                     , 88-102. <a href="https://doi.org/10.1021/acs.jmedchem.9b01605" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01605</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01605%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252BLpxC%252BInhibitor%252BLead%252BCompounds%252BFocusing%252Bon%252BEfficacy%252Band%252BFormulation%252Bfor%252BHigh%252BDose%252BIntravenous%252BAdministration%26aulast%3DPanchaud%26aufirst%3DPhilippe%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D27092019%26date%3D23122019%26date%3D05122019%26volume%3D63%26issue%3D1%26spage%3D88%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Randi L.  Guest</span>, <span class="hlFld-ContribAuthor ">Steven T.  Rutherford</span>, <span class="hlFld-ContribAuthor ">Thomas J.  Silhavy</span>. </span><span class="cited-content_cbyCitation_article-title">Border Control: Regulating LPS Biogenesis. </span><span class="cited-content_cbyCitation_journal-name">Trends in Microbiology</span><span> <strong>2021,</strong> <em>29 </em>
                                    (4)
                                     , 334-345. <a href="https://doi.org/10.1016/j.tim.2020.09.008" title="DOI URL">https://doi.org/10.1016/j.tim.2020.09.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tim.2020.09.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tim.2020.09.008%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Microbiology%26atitle%3DBorder%252BControl%25253A%252BRegulating%252BLPS%252BBiogenesis%26aulast%3DGuest%26aufirst%3DRandi%2BL.%26date%3D2021%26volume%3D29%26issue%3D4%26spage%3D334%26epage%3D345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergi  Pascual</span>, <span class="hlFld-ContribAuthor ">Antoni  Planas</span>. </span><span class="cited-content_cbyCitation_article-title">Carbohydrate de-N-acetylases acting on structural polysaccharides and glycoconjugates. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>61 </em>, 9-18. <a href="https://doi.org/10.1016/j.cbpa.2020.09.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.09.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.09.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DCarbohydrate%252Bde-N-acetylases%252Bacting%252Bon%252Bstructural%252Bpolysaccharides%252Band%252Bglycoconjugates%26aulast%3DPascual%26aufirst%3DSergi%26date%3D2021%26volume%3D61%26spage%3D9%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fumihito  Ushiyama</span>, <span class="hlFld-ContribAuthor ">Hajime  Takashima</span>, <span class="hlFld-ContribAuthor ">Yohei  Matsuda</span>, <span class="hlFld-ContribAuthor ">Yuya  Ogata</span>, <span class="hlFld-ContribAuthor ">Naoki  Sasamoto</span>, <span class="hlFld-ContribAuthor ">Risa  Kurimoto-Tsuruta</span>, <span class="hlFld-ContribAuthor ">Kaori  Ueki</span>, <span class="hlFld-ContribAuthor ">Nozomi  Tanaka-Yamamoto</span>, <span class="hlFld-ContribAuthor ">Mayumi  Endo</span>, <span class="hlFld-ContribAuthor ">Masashi  Mima</span>, <span class="hlFld-ContribAuthor ">Kiyoko  Fujita</span>, <span class="hlFld-ContribAuthor ">Iichiro  Takata</span>, <span class="hlFld-ContribAuthor ">Satoshi  Tsuji</span>, <span class="hlFld-ContribAuthor ">Haruhiro  Yamashita</span>, <span class="hlFld-ContribAuthor ">Hirotoshi  Okumura</span>, <span class="hlFld-ContribAuthor ">Katsumasa  Otake</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Sugiyama</span>. </span><span class="cited-content_cbyCitation_article-title">Lead optimization of 2-hydroxymethyl imidazoles as non-hydroxamate LpxC inhibitors: Discovery of TP0586532. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>30 </em>, 115964. <a href="https://doi.org/10.1016/j.bmc.2020.115964" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115964</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115964%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DLead%252Boptimization%252Bof%252B2-hydroxymethyl%252Bimidazoles%252Bas%252Bnon-hydroxamate%252BLpxC%252Binhibitors%25253A%252BDiscovery%252Bof%252BTP0586532%26aulast%3DUshiyama%26aufirst%3DFumihito%26date%3D2021%26volume%3D30%26spage%3D115964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takeru  Furuya</span>, <span class="hlFld-ContribAuthor ">Adam B.  Shapiro</span>, <span class="hlFld-ContribAuthor ">Janelle  Comita-Prevoir</span>, <span class="hlFld-ContribAuthor ">Eric J.  Kuenstner</span>, <span class="hlFld-ContribAuthor ">Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Seth D.  Ribe</span>, <span class="hlFld-ContribAuthor ">April  Chen</span>, <span class="hlFld-ContribAuthor ">Daniel  Hines</span>, <span class="hlFld-ContribAuthor ">Samir H.  Moussa</span>, <span class="hlFld-ContribAuthor ">Nicole M.  Carter</span>, <span class="hlFld-ContribAuthor ">Mark A.  Sylvester</span>, <span class="hlFld-ContribAuthor ">Jan A.C.  Romero</span>, <span class="hlFld-ContribAuthor ">Camilo V.  Vega</span>, <span class="hlFld-ContribAuthor ">Michael D.  Sacco</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">John P.  O'Donnell</span>, <span class="hlFld-ContribAuthor ">Thomas F.  Durand-Reville</span>, <span class="hlFld-ContribAuthor ">Alita A.  Miller</span>, <span class="hlFld-ContribAuthor ">Ruben A.  Tommasi</span>. </span><span class="cited-content_cbyCitation_article-title">N-Hydroxyformamide LpxC inhibitors, their in vivo efficacy in a mouse Escherichia coli infection model, and their safety in a rat hemodynamic assay. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (24)
                                     , 115826. <a href="https://doi.org/10.1016/j.bmc.2020.115826" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115826</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115826%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DN-Hydroxyformamide%252BLpxC%252Binhibitors%25252C%252Btheir%252Bin%252Bvivo%252Befficacy%252Bin%252Ba%252Bmouse%252BEscherichia%252Bcoli%252Binfection%252Bmodel%25252C%252Band%252Btheir%252Bsafety%252Bin%252Ba%252Brat%252Bhemodynamic%252Bassay%26aulast%3DFuruya%26aufirst%3DTakeru%26date%3D2020%26volume%3D28%26issue%3D24%26spage%3D115826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naveen</span>, <span class="hlFld-ContribAuthor ">Ram  Kumar Tittal</span>, <span class="hlFld-ContribAuthor ">Ghule D.  Vikas</span>, <span class="hlFld-ContribAuthor ">Poonam  Rani</span>, <span class="hlFld-ContribAuthor ">Kashmiri  Lal</span>, <span class="hlFld-ContribAuthor ">Ashwani  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Antimicrobial Activity, Molecular Docking and DFT Study: Aryl‐Carbamic Acid 1‐Benzyl‐1
              H
              ‐[1,2,3]Triazol‐4‐ylmethyl Esters. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (22)
                                     , 6723-6729. <a href="https://doi.org/10.1002/slct.202001547" title="DOI URL">https://doi.org/10.1002/slct.202001547</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202001547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202001547%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DSynthesis%25252C%252BAntimicrobial%252BActivity%25252C%252BMolecular%252BDocking%252Band%252BDFT%252BStudy%25253A%252BAryl%2525E2%252580%252590Carbamic%252BAcid%252B1%2525E2%252580%252590Benzyl%2525E2%252580%2525901%252BH%252B%2525E2%252580%252590%25255B1%25252C2%25252C3%25255DTriazol%2525E2%252580%2525904%2525E2%252580%252590ylmethyl%252BEsters%26aulast%3DNaveen%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D22%26spage%3D6723%26epage%3D6729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. LpxC, ubiquitously found in GNB, catalyzes the irreversible deacetylation of UDP-(3-<i>O</i>-((<i>R</i>)-3-hydroxymyristoyl)-<i>N</i>-acetylglucosamine. Representative inhibitors of LpxC feature a hydroxamate as a zinc-binding group (box).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Core structures of novel methylsulfone hydroxamate LpxC inhibitors based on bicyclic heteroaromatic systems.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0011.jpeg" id="rightTab-GRAPHIC-d7e643-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, AcOH, DCM, rt, 4 h; (b) NaI, CuI, <i>trans</i>-<i>N</i>,<i>N</i>′-dimethylcyclohexane-1,2-diamine, dioxane, reflux, 3 h; (c) TFA, Et<sub>3</sub>SiH, rt, 16 h; (d) NH<sub>2</sub>OTHP, EDC·HCl, HOBt, TEA, DMF, rt, 16 h; (e) <b>27</b>, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, reflux, 1 h; (f) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt, 1 h; (g) PPTS, EtOH, 70 °C.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0012.jpeg" id="rightTab-GRAPHIC-d7e742-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>22</b>, 3 Å molecular sieves, DCM, rt then NaBH<sub>4</sub>, MeOH, 0 °C; (b) CDI, NaH, THF, 0 °C; (c) TFA, Et<sub>3</sub>SiH, DCM, 0 °C to rt, 5 h; (d) THPONH<sub>2</sub>, T<sub>3</sub>P, DIPEA, THF, rt, 2 h; (e) TMS-CCH, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, rt, 1 h; (f) TBAF, THF, rt.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0013.jpeg" id="rightTab-GRAPHIC-d7e824-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>22</b>, AcOH, DCM, rt, 4 h; (b) Br<sub>2</sub>, aq AcOH, rt, 16 h; (c) 4 N HCl, dioxane, rt, 16 h; (d) THPONH<sub>2</sub>, EDC, HOBt, TEA, DMF, rt, 16 h; (e) TMS-CCH, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, rt, 1 h; (f) TBAF, THF, rt.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0014.jpeg" id="rightTab-GRAPHIC-d7e997-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>41</b>–<b>50</b>, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, CuI, TEA, THF, reflux; (b) method A, PPTS, EtOH, 70 °C; method B, PPTS, aq dioxane, 70 °C; method C, aq HCl, MeCN, rt; (c) <b>51</b>–<b>60</b>, CuCl, NH<sub>2</sub>OH·HCl, BuNH<sub>2</sub>, rt; (d) TBAF, THF, rt; (e) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt.</p></p></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Scaffold hopping approach from pyridone <b>7</b> leading to novel oxazol-2-one <b>8</b>. The numbers indicated below the dash-lined boxes represent the angle at which both exit vectors of the core intersect.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Oxazol-2-one <b>8</b> (white carbon atoms) bound in the active site of Pae LpxC (gray ribbon) (1.9 Å) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I46">6I46</a>). Key interactions of oxazol-2-one <b>8</b> with metal (blue sphere) and key amino acid side chains (gray carbon atoms) are displayed as dotted lines. Electron density of the ligand is shown as a blue mesh (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> density contoured at 1σ). While <i>rac</i>-<b>8</b> was used for cocrystallization experiments with Pae LpxC, only active enantiomer (<i>R</i>)-<b>8</b> was observed.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Paths followed to generate novel bicyclic cores from oxazol-2-one (<i>R</i>)-<b>8</b>. Side aromatic ring (Ar) remained constant (Ar = 2-fluoro-4-methoxyphenyl). The numbers indicated below the dash-lined boxes represent the angle at which both exit vectors of the core intersect. The orientation of Ar relative to the central core was not considered.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Superimposition of oxazol-2-one (<i>R</i>)-<b>8</b> (white carbons) and isoindolin-1-one <b>10</b> (cyan carbons) bound to Pae LpxC. The α-helix of insert II (and Ile197 side chain, colored in cyan) moved closer to the inhibitor in case of <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0007.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (a,b) Views of inhibitor <b>17a</b> bound to <i>P. aeruginosa</i> LpxC (2.0 Å) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I49">6I49</a>). The lipophilic tail is subdivided into three segments <b>A</b>, <b>B</b>, and <b>R</b>. From the acetylenic unit <b>A</b>, the side chain resulting from the combination of segment <b>B</b> (here a phenyl ring) and segment <b>R</b>, here an <i>N</i>-methyl-morpholine group protruding out of the substrate-binding passage of Pae LpxC. Several conformations/positions for the morpholine ring of <b>17a</b> are possible, the depicted one serving as an example. Partial ligand electron density is represented as a blue mesh (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> density contoured at 1 sigma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0008.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Superimposition of cores of compound <b>17a</b> (green carbons) and compounds <b>8</b>, <b>10</b>, <b>12</b>, and <b>18d</b> (all gray carbons). Phe191 and Gly192 peptide bonds on right-hand side. (b) View of the <b>B</b><sub>1</sub> phenyl moiety of <b>17a</b> and its stacking over the peptide bond between Gly209 and Ser210.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0009.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Superimposition of inhibitor <b>17a</b> (orange carbons) with CHIR-090 <b>3</b> (gray carbons) bound to Pae LpxC (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VWM">5VWM</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/medium/jm9b01604_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0010.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. View of compound <b>18d</b> bound to Pae LpxC (2.2 Å) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I4A">6I4A</a>). The 1-hydromethyl-1-cyclopropyl side chain is located outside the lipophilic tunnel. Two conformations are possible.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-1/acs.jmedchem.9b01604/20200106/images/large/jm9b01604_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01604&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i80">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11386" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11386" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 41 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span> <i>Antimicrobial
Resistance: Global Report on Surveillance 2014</i>; <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Antimicrobial%0AResistance%3A+Global+Report+on+Surveillance+2014%3B+World+Health+Organization%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%250AResistance%253A%2520Global%2520Report%2520on%2520Surveillance%25202014%26pub%3DWorld%2520Health%2520Organization%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span> <i>Global
Priority List of Antibiotic-Resistant Bacteria to Guide Research,
Discovery, and Development of New Antibiotics</i>; <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Global%0APriority+List+of+Antibiotic-Resistant+Bacteria+to+Guide+Research%2C%0ADiscovery%2C+and+Development+of+New+Antibiotics%3B+World+Health+Organization%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26jtitle%3DGlobal%250APriority%2520List%2520of%2520Antibiotic-Resistant%2520Bacteria%2520to%2520Guide%2520Research%252C%250ADiscovery%252C%2520and%2520Development%2520of%2520New%2520Antibiotics%26pub%3DWorld%2520Health%2520Organization%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granata, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrosillo, N.</span></span> <span> </span><span class="NLM_article-title">Resistance to Colistin in <i>Klebsiella Pneumoniae</i>: A 4.0 Strain?</span>. <i>Infect. Dis. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">7104</span>, <span class="refDoi"> DOI: 10.4081/idr.2017.7104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.4081%2Fidr.2017.7104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=28626539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A280%3ADC%252BC1cnpsVKqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=7104&author=G.+Granataauthor=N.+Petrosillo&title=Resistance+to+Colistin+in+Klebsiella+Pneumoniae%3A+A+4.0+Strain%3F&doi=10.4081%2Fidr.2017.7104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to Colistin in Klebsiella Pneumoniae: A 4.0 Strain?</span></div><div class="casAuthors">Granata Guido; Petrosillo Nicola</div><div class="citationInfo"><span class="NLM_cas:title">Infectious disease reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">7104</span>
        ISSN:<span class="NLM_cas:issn">2036-7430</span>.
    </div><div class="casAbstract">The global rise of multidrug-resistant gram-negative bacteria represents an increasing threat to patient safety.  From the first observation of a carbapenem-resistant gram-negative bacteria a global spread of extended-spectrum beta-lactamases and carbapenemases producing Klebsiella pneumoniae has been observed.  Treatment options for multidrug-resistant K. pneumoniae are actually limited to combination therapy with some aminoglycosides, tigecycline and to older antimicrobial agents.  Unfortunately, the prevalence of colistin-resistant and tigecycline-resistant K. pneumoniae is increasing globally.  Infection due to colistin-resistant K. pneumoniae represents an independent risk factor for mortality.  Resistance to colistin in K. pneumoniae may be multifactorial, as it is mediated by chromosomal genes or plasmids.  The emergence of transmissible, plasmid-mediated colistin resistance is an alarming finding.  The absence of new agents effective against resistant Gram-negative pathogens means that enhanced surveillance, compliance with infection prevention procedures, and antimicrobial stewardship programs will be required to limit the spread of colistin-resistant K. pneumoniae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWTAXmmdHFs5QHPYJuLg8TfW6udTcc2eZf6JQ0_8CBq7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnpsVKqug%253D%253D&md5=28fe14c9d0962b16c46931c5a4da2149</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4081%2Fidr.2017.7104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4081%252Fidr.2017.7104%26sid%3Dliteratum%253Aachs%26aulast%3DGranata%26aufirst%3DG.%26aulast%3DPetrosillo%26aufirst%3DN.%26atitle%3DResistance%2520to%2520Colistin%2520in%2520Klebsiella%2520Pneumoniae%253A%2520A%25204.0%2520Strain%253F%26jtitle%3DInfect.%2520Dis.%2520Rep.%26date%3D2017%26volume%3D9%26spage%3D7104%26doi%3D10.4081%2Fidr.2017.7104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guidos, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbot, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheld, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J. G.</span></span> <span> </span><span class="NLM_article-title">The 10 x ’20 Initiative: Pursuing a Global Commitment to Develop 10 New Antibacterial Drugs by 2020</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1081</span>– <span class="NLM_lpage">1083</span>, <span class="refDoi"> DOI: 10.1086/652237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1086%2F652237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=20214473" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1081-1083&author=D.+N.+Gilbertauthor=R.+J.+Guidosauthor=H.+W.+Boucherauthor=G.+H.+Talbotauthor=B.+Spellbergauthor=J.+E.+Edwardsauthor=W.+M.+Scheldauthor=J.+S.+Bradleyauthor=J.+G.+Bartlett&title=The+10+x+%E2%80%9920+Initiative%3A+Pursuing+a+Global+Commitment+to+Develop+10+New+Antibacterial+Drugs+by+2020&doi=10.1086%2F652237"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1086%2F652237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F652237%26sid%3Dliteratum%253Aachs%26aulast%3DGilbert%26aufirst%3DD.%2BN.%26aulast%3DGuidos%26aufirst%3DR.%2BJ.%26aulast%3DBoucher%26aufirst%3DH.%2BW.%26aulast%3DTalbot%26aufirst%3DG.%2BH.%26aulast%3DSpellberg%26aufirst%3DB.%26aulast%3DEdwards%26aufirst%3DJ.%2BE.%26aulast%3DScheld%26aufirst%3DW.%2BM.%26aulast%3DBradley%26aufirst%3DJ.%2BS.%26aulast%3DBartlett%26aufirst%3DJ.%2BG.%26atitle%3DThe%252010%2520x%2520%25E2%2580%259920%2520Initiative%253A%2520Pursuing%2520a%2520Global%2520Commitment%2520to%2520Develop%252010%2520New%2520Antibacterial%2520Drugs%2520by%25202020%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2010%26volume%3D50%26spage%3D1081%26epage%3D1083%26doi%3D10.1086%2F652237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lgickHjj25-Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surivet, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zumbrunn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruyere, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locher, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seiler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ertel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enderlin-Paput, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enderlin-Paput, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauvin, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubschwerlen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueedi, G.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Characterization of Tetrahydropyran-Based Bacterial Topoisomerase Inhibitors With Antibacterial Activity Against Gram-Negative Bacteria</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3776</span>– <span class="NLM_lpage">3794</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVWktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3776-3794&author=J.-P.+Surivetauthor=C.+Zumbrunnauthor=T.+Bruyereauthor=D.+Burauthor=C.+Kohlauthor=H.+H.+Locherauthor=P.+Seilerauthor=E.+A.+Ertelauthor=P.+Hessauthor=M.+Enderlin-Paputauthor=S.+Enderlin-Paputauthor=J.-C.+Gauvinauthor=A.+Mirreauthor=C.+Hubschwerlenauthor=D.+Ritzauthor=G.+Rueedi&title=Synthesis+and+Characterization+of+Tetrahydropyran-Based+Bacterial+Topoisomerase+Inhibitors+With+Antibacterial+Activity+Against+Gram-Negative+Bacteria&doi=10.1021%2Facs.jmedchem.6b01831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Characterization of Tetrahydropyran-Based Bacterial Topoisomerase Inhibitors with Antibacterial Activity against Gram-Negative Bacteria</span></div><div class="casAuthors">Surivet, Jean-Philippe; Zumbrunn, Cornelia; Bruyere, Thierry; Bur, Daniel; Kohl, Christopher; Locher, Hans H.; Seiler, Peter; Ertel, Eric A.; Hess, Patrick; Enderlin-Paput, Michel; Enderlin-Paput, Stephanie; Gauvin, Jean-Christophe; Mirre, Azely; Hubschwerlen, Christian; Ritz, Daniel; Rueedi, Georg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3776-3794</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is an urgent unmet medical need for novel antibiotics that are effective against a broad range of bacterial species, esp. multidrug resistant ones.  Tetrahydropyran-based inhibitors of bacterial type II topoisomerases (DNA gyrase and topoisomerase IV) display potent activity against Gram-pos. pathogens and no target-mediated cross-resistance with fluoroquinolones.  We report our research efforts aimed at expanding the antibacterial spectrum of this class of mols. toward difficult-to-treat Gram-neg. pathogens.  Physicochem. properties (polarity and basicity) were considered to guide the design process.  Dibasic tetrahydropyran-based compds. such as 6 and 21 are potent inhibitors of both DNA gyrase and topoisomerase IV, displaying antibacterial activities against Gram-pos. and Gram-neg. pathogens (Staphylococcus aureus, Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii).  Compds. 6 and 21 are efficacious in clin. relevant murine infection models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWIubQLqfwWbVg90H21EOLACvtfcHk0lioaBVTPBZOlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVWktrw%253D&md5=0c2f17fda10b44b6c261c0cc86279564</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01831%26sid%3Dliteratum%253Aachs%26aulast%3DSurivet%26aufirst%3DJ.-P.%26aulast%3DZumbrunn%26aufirst%3DC.%26aulast%3DBruyere%26aufirst%3DT.%26aulast%3DBur%26aufirst%3DD.%26aulast%3DKohl%26aufirst%3DC.%26aulast%3DLocher%26aufirst%3DH.%2BH.%26aulast%3DSeiler%26aufirst%3DP.%26aulast%3DErtel%26aufirst%3DE.%2BA.%26aulast%3DHess%26aufirst%3DP.%26aulast%3DEnderlin-Paput%26aufirst%3DM.%26aulast%3DEnderlin-Paput%26aufirst%3DS.%26aulast%3DGauvin%26aufirst%3DJ.-C.%26aulast%3DMirre%26aufirst%3DA.%26aulast%3DHubschwerlen%26aufirst%3DC.%26aulast%3DRitz%26aufirst%3DD.%26aulast%3DRueedi%26aufirst%3DG.%26atitle%3DSynthesis%2520and%2520Characterization%2520of%2520Tetrahydropyran-Based%2520Bacterial%2520Topoisomerase%2520Inhibitors%2520With%2520Antibacterial%2520Activity%2520Against%2520Gram-Negative%2520Bacteria%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3776%26epage%3D3794%26doi%3D10.1021%2Facs.jmedchem.6b01831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bush, K.</span></span> <span> </span><span class="NLM_article-title">Investigational Agents for the Treatment of Gram-Negative Bacterial Infections: A Reality Check</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.5b00100</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.5b00100" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1GksrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=509-511&author=K.+Bush&title=Investigational+Agents+for+the+Treatment+of+Gram-Negative+Bacterial+Infections%3A+A+Reality+Check&doi=10.1021%2Facsinfecdis.5b00100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Investigational Agents for the Treatment of Gram-Negative Bacterial Infections: A Reality Check</span></div><div class="casAuthors">Bush, Karen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">509-511</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antibiotic-resistant Gram-neg. bacteria are, arguably, the most difficult organisms to treat, with a limited no. of new antibiotics in the development pipeline.  Currently 24 new agents in phase 1, phase 2, or phase 3 clin. development were identified for the potential treatment of infections caused by Gram-neg. bacteria.  Of these agents, most are improved iterations of known antibiotic classes, including new aminoglycosides, β-lactams, β-lactamase inhibitors, quinolones, and tetracyclines with greater potency or a broader spectrum of activity.  However, novel structures also appear, with host defense peptide mimetics, boronic acid, and bridged diazabicyclooctane β-lactamase inhibitors and unique bacterial topoisomerase inhibitors.  Most of the new agents have received a Qualified Infectious Disease Product (QIDP) designation that may help to accelerate FDA drug approvals.  Because resistance will inevitably arise to any antibacterial agent, it will be necessary to continue to identify addnl. new agents in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP0Z7V8fADjLVg90H21EOLACvtfcHk0lj8PsN03-XlGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1GksrvM&md5=b5833cbe0a6e9abe92b4b292a219dd59</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.5b00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.5b00100%26sid%3Dliteratum%253Aachs%26aulast%3DBush%26aufirst%3DK.%26atitle%3DInvestigational%2520Agents%2520for%2520the%2520Treatment%2520of%2520Gram-Negative%2520Bacterial%2520Infections%253A%2520A%2520Reality%2520Check%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D509%26epage%3D511%26doi%3D10.1021%2Facsinfecdis.5b00100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span>; <span class="NLM_string-name">Kubo, A.</span>; <span class="NLM_string-name">Lopez, S.</span>; <span class="NLM_string-name">Gomez, M.</span>; <span class="NLM_string-name">Corey, V. C.</span>; <span class="NLM_string-name">Andrews, L.</span>; <span class="NLM_string-name">Schwartz, M. A.</span>; <span class="NLM_string-name">Kasar, R.</span>; <span class="NLM_string-name">McEnroe, G.</span>; <span class="NLM_string-name">Aggen, J.</span>; <span class="NLM_string-name">Miller, G.</span>; <span class="NLM_string-name">Armstrong, E. S.</span></span> <span> </span><span class="NLM_article-title">Structure, Potency and Bactericidal Activity of ACHN-975, a First-in-Class LpxC Inhibitor</span>. In  <i>53rd Interscience Conference on Antimicrobial Agents and Chemotherapy, Denver, Colorado, United States</i>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+W.+Serio&author=A.+Kubo&author=S.+Lopez&author=M.+Gomez&author=V.+C.+Corey&author=L.+Andrews&author=M.+A.+Schwartz&author=R.+Kasar&author=G.+McEnroe&author=J.+Aggen&author=G.+Miller&author=E.+S.+Armstrong&title=Structure%2C+Potency+and+Bactericidal+Activity+of+ACHN-975%2C+a+First-in-Class+LpxC+Inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DSerio%26aufirst%3DA.%2BW.%26atitle%3DStructure%252C%2520Potency%2520and%2520Bactericidal%2520Activity%2520of%2520ACHN-975%252C%2520a%2520First-in-Class%2520LpxC%2520Inhibitor%26jtitle%3D53rd%2520Interscience%2520Conference%2520on%2520Antimicrobial%2520Agents%2520and%2520Chemotherapy%252C%2520Denver%252C%2520Colorado%252C%2520United%2520States%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierke, C. A.</span></span> <span> </span><span class="NLM_article-title">UDP-3-O-(<i>R</i>-3-hydroxymyristoyl)-<i>N</i>-acetylglucosamine Deacetylase of <i>Escherichia coli</i> Is a Zinc Metalloenzyme</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1902</span>– <span class="NLM_lpage">1911</span>, <span class="refDoi"> DOI: 10.1021/bi982339s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi982339s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADyaK1MXkvFymsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1902-1911&author=J.+E.+Jackmanauthor=C.+R.+Raetzauthor=C.+A.+Fierke&title=UDP-3-O-%28R-3-hydroxymyristoyl%29-N-acetylglucosamine+Deacetylase+of+Escherichia+coli+Is+a+Zinc+Metalloenzyme&doi=10.1021%2Fbi982339s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">UDP-3-O-(R-3-Hydroxymyristoyl)-N-acetylglucosamine Deacetylase of Escherichia coli Is a Zinc Metalloenzyme</span></div><div class="casAuthors">Jackman, Jane E.; Raetz, Christian R. H.; Fierke, Carol A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1902-1911</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The enzyme UDP-3-O-(R-3-hydroxymyristoyl)-GlcNAc deacetylase (LpxC) catalyzes the committed step in the biosynthesis of lipid A and is therefore a potential antibiotic target.  Inhibition of this enzyme by hydroxamate compds. suggested the presence of a metal ion cofactor.  We have investigated the substrate specificity and metal dependence of the deacetylase using spectroscopic and kinetic analyses.  Comparison of the steady-state kinetic parameters for the physiol. substrate UDP-3-O-(R-3-hydroxymyristoyl)-GlcNAc and an alternative substrate, UDP-GlcNAc, demonstrates that the ester-linked R-3-hydroxymyristoyl chain increases kcat/KM (5 × 106)-fold.  Metal-chelating reagents, such as dipicolinic acid (DPA) and EDTA, completely inhibit LpxC activity, implicating an essential metal ion.  Plasma emission spectroscopy and colorimetric assays directly demonstrate that purified LpxC contains bound Zn2+.  This Zn2+ can be removed by incubation with DPA, causing a decrease in the LpxC activity that can be restored by subsequent addn. of Zn2+.  However, high concns. of Zn2+ also inhibit LpxC.  Addn. of Co2+, Ni2+, or Mn2+ to apo-LpxC also activates the enzyme to varying degrees while no addnl. activity is obsd. upon the addn. of Cd2+, Ca2+, Mg2+, or Cu2+.  This is consistent with the profile of metals that substitute for catalytic zinc ions in metalloproteinases.  Co2+ ions stimulate LpxC activity maximally at a stoichiometry of 1:1.  These data demonstrate that E. coli LpxC is a metalloenzyme that requires bound Zn2+ for optimal activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoETi4nWY-kpLVg90H21EOLACvtfcHk0lhaxBI3e6AiIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXkvFymsw%253D%253D&md5=43e4ad7d68cbbd04a202c898c7bb667a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fbi982339s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi982339s%26sid%3Dliteratum%253Aachs%26aulast%3DJackman%26aufirst%3DJ.%2BE.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DFierke%26aufirst%3DC.%2BA.%26atitle%3DUDP-3-O-%2528R-3-hydroxymyristoyl%2529-N-acetylglucosamine%2520Deacetylase%2520of%2520Escherichia%2520coli%2520Is%2520a%2520Zinc%2520Metalloenzyme%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D1902%26epage%3D1911%26doi%3D10.1021%2Fbi982339s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierke, C. A.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism and molecular recognition of <i>E. coli</i> UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine Deacetylase Probed by Mutagenesis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">15240</span>– <span class="NLM_lpage">15248</span>, <span class="refDoi"> DOI: 10.1021/bi061405k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi061405k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlChurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=15240-15248&author=M.+Hernickauthor=C.+A.+Fierke&title=Catalytic+mechanism+and+molecular+recognition+of+E.+coli+UDP-3-O-%28R-3-hydroxymyristoyl%29-N-acetylglucosamine+Deacetylase+Probed+by+Mutagenesis&doi=10.1021%2Fbi061405k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic Mechanism and Molecular Recognition of E. coli UDP-3-O-(R-3-Hydroxymyristoyl)-N-acetylglucosamine Deacetylase Probed by Mutagenesis</span></div><div class="casAuthors">Hernick, Marcy; Fierke, Carol A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15240-15248</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a metal-dependent deacetylase that catalyzes the hydrolysis of UDP-3-O-myristoyl-N-acetyl-glucosamine to form UDP-3-O-myristoyl-glucosamine and acetate.  This is the committed step in the biosynthesis of lipid A, and therefore, LpxC is a target for the development of antimicrobial agents in the treatment of Gram-neg. infections.  To facilitate the development of potent and specific inhibitors of LpxC, the mol. determinants of binding and specificity and the catalytic mechanism for this enzyme have been probed.  The functions of active site residues have been classified on the basis of changes in steady-state turnover (kcat, KM, and kcat/KM) and product binding affinity (KDProduct).  We have identified side chains that enhance product affinity and reactivity (F192, K239, D246, and H265), destabilize product affinity (E78 and D197), and preferentially enhance catalytic efficiency (H19, T191, K143, and N162).  In addn., the affinity of LpxC for myrUDP-GlcNH2 is dependent on two ionizations, one deprotonation and one protonation, with apparent pKa values of 6.5±0.1 and 7.4±0.1, resp.  The UDP moiety of the product contributes significantly to recognition by LpxC, suggesting that this region can be targeted in drug development.  These data provide a map of the active site features essential for catalysis and mol. recognition by LpxC that can be used for developing more potent LpxC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon6LHZu-oUlLVg90H21EOLACvtfcHk0ljLxGYbH9X_ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlChurvI&md5=47e028e85adf93ac8975001ecbff1e1a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fbi061405k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi061405k%26sid%3Dliteratum%253Aachs%26aulast%3DHernick%26aufirst%3DM.%26aulast%3DFierke%26aufirst%3DC.%2BA.%26atitle%3DCatalytic%2520mechanism%2520and%2520molecular%2520recognition%2520of%2520E.%2520coli%2520UDP-3-O-%2528R-3-hydroxymyristoyl%2529-N-acetylglucosamine%2520Deacetylase%2520Probed%2520by%2520Mutagenesis%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D15240%26epage%3D15248%26doi%3D10.1021%2Fbi061405k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span> <span> </span><span class="NLM_article-title">Structure of the Bacterial Deacetylase LpxC Bound to the Nucleotide Reaction Product Reveals Mechanisms of Oxyanion Stabilization and Proton Transfer</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">34073</span>– <span class="NLM_lpage">34080</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.513028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1074%2Fjbc.M113.513028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=24108127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=34073-34080&author=G.+M.+Claytonauthor=D.+J.+Kleinauthor=K.+W.+Rickertauthor=S.+B.+Patelauthor=M.+Kornienkoauthor=J.+Zugay-Murphyauthor=J.+C.+Reidauthor=S.+Tummalaauthor=S.+Sharmaauthor=S.+B.+Singhauthor=L.+Mieselauthor=K.+J.+Lumbauthor=S.+M.+Soisson&title=Structure+of+the+Bacterial+Deacetylase+LpxC+Bound+to+the+Nucleotide+Reaction+Product+Reveals+Mechanisms+of+Oxyanion+Stabilization+and+Proton+Transfer&doi=10.1074%2Fjbc.M113.513028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the bacterial deacetylase LpxC bound to the nucleotide reaction product reveals mechanisms of oxyanion stabilization and proton transfer</span></div><div class="casAuthors">Clayton, Gina M.; Klein, Daniel J.; Rickert, Keith W.; Patel, Sangita B.; Kornienko, Maria; Zugay-Murphy, Joan; Reid, John C.; Tummala, Srivanya; Sharma, Sujata; Singh, Sheo B.; Miesel, Lynn; Lumb, Kevin J.; Soisson, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">34073-34080</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The emergence of antibiotic-resistant strains of pathogenic bacteria is an increasing threat to global health that underscores an urgent need for an expanded antibacterial armamentarium.  Gram-neg. bacteria, such as Escherichia coli, have become increasingly important clin. pathogens with limited treatment options.  This is due in part to their lipopolysaccharide (LPS) outer membrane components, which dually serve as endotoxins while also protecting Gram-neg. bacteria from antibiotic entry.  The LpxC enzyme, UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase, catalyzes the committed step of LPS biosynthesis, making LpxC a promising target for new antibacterials.  Here, the authors present the 1st structure of E. coli LpxC in complex with the deacetylation reaction product, UDP-[3-O-(R-3-hydroxymyristoyl)]glucosamine.  These studies provide valuable insight into the recognition of substrates and products by LpxC and a platform for structure-guided drug discovery of broad spectrum Gram-neg. antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm8dRnYgccKbVg90H21EOLACvtfcHk0lgU9EOYrfeRug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2qsrjO&md5=ac4adc79fe696089087d35a29e39f892</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.513028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.513028%26sid%3Dliteratum%253Aachs%26aulast%3DClayton%26aufirst%3DG.%2BM.%26aulast%3DKlein%26aufirst%3DD.%2BJ.%26aulast%3DRickert%26aufirst%3DK.%2BW.%26aulast%3DPatel%26aufirst%3DS.%2BB.%26aulast%3DKornienko%26aufirst%3DM.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DReid%26aufirst%3DJ.%2BC.%26aulast%3DTummala%26aufirst%3DS.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DMiesel%26aufirst%3DL.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26atitle%3DStructure%2520of%2520the%2520Bacterial%2520Deacetylase%2520LpxC%2520Bound%2520to%2520the%2520Nucleotide%2520Reaction%2520Product%2520Reveals%2520Mechanisms%2520of%2520Oxyanion%2520Stabilization%2520and%2520Proton%2520Transfer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D34073%26epage%3D34080%26doi%3D10.1074%2Fjbc.M113.513028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivier, N. B.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of <i>A. Aeolicu</i>s LpxC With Bound Product Suggests Alternate Deacetylation Mechanism</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">1706</span>– <span class="NLM_lpage">1719</span>, <span class="refDoi"> DOI: 10.1002/prot.24856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1002%2Fprot.24856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=26177919" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2015&pages=1706-1719&author=M.+D.+Millerauthor=N.+Gaoauthor=P.+L.+Rossauthor=N.+B.+Olivier&title=Crystal+Structure+of+A.+Aeolicus+LpxC+With+Bound+Product+Suggests+Alternate+Deacetylation+Mechanism&doi=10.1002%2Fprot.24856"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fprot.24856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24856%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DRoss%26aufirst%3DP.%2BL.%26aulast%3DOlivier%26aufirst%3DN.%2BB.%26atitle%3DCrystal%2520Structure%2520of%2520A.%2520Aeolicus%2520LpxC%2520With%2520Bound%2520Product%2520Suggests%2520Alternate%2520Deacetylation%2520Mechanism%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2015%26volume%3D83%26spage%3D1706%26epage%3D1719%26doi%3D10.1002%2Fprot.24856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalinin, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holl, R.</span></span> <span> </span><span class="NLM_article-title">Insights Into the Zinc-Dependent Deacetylase LpxC: Biochemical Properties and Inhibitor Design</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2379</span>– <span class="NLM_lpage">2430</span>, <span class="refDoi"> DOI: 10.2174/1568026616666160413135835</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.2174%2F1568026616666160413135835" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=27072691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFOgs7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=2379-2430&author=D.+V.+Kalininauthor=R.+Holl&title=Insights+Into+the+Zinc-Dependent+Deacetylase+LpxC%3A+Biochemical+Properties+and+Inhibitor+Design&doi=10.2174%2F1568026616666160413135835"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Zinc-Dependent Deacetylase LpxC: Biochemical Properties and Inhibitor Design</span></div><div class="casAuthors">Kalinin, Dmitrii V.; Holl, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2379-2430</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The bacterial enzyme UDP-3-O-[(R)-3-hydroxymyristoyl]-N-acetylglucosamine deacetylase (LpxC), catalyzing the first committed step of lipid A biosynthesis, represents a promising target in the development of novel antibiotics against Gram-neg. bacteria.  Structure, catalytic reaction mechanism and regulation of the Zn2+-dependent metalloamidase have been intensively investigated.  The enzyme is required for growth and viability of Gram-neg. bacteria, displays no sequence homol. with any mammalian protein, but is highly conserved in Gram-neg. bacteria, thus permitting the development of Gram-neg. selective antibacterial agents with limited off-target effects.  Several small mol. LpxC inhibitors have been developed, like the substrate analog TU-514 (12a), the aryloxazoline L-161,240 (13w), the sulfonamide BB-78485 (23a), the N-aroyl-L-threonine deriv. CHIR-090 (24a), the sulfone-contg. pyridone LpxC-3 (43e), and the uridine-based inhibitor 1-68A (47a), displaying diverse inhibitory and antibacterial activities.  Most of these compds. share a Zn2+-binding hydroxamate moiety attached to a structural element addressing the hydrophobic tunnel or the UDP binding site.  The butadiynyl deriv. ACHN-975 (28) is the first LpxC inhibitor entering clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod_JborvvzXbVg90H21EOLACvtfcHk0liKz2Csa2F-RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFOgs7fE&md5=82abc89ee4dab91b95bbb9dabb23f0b3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F1568026616666160413135835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026616666160413135835%26sid%3Dliteratum%253Aachs%26aulast%3DKalinin%26aufirst%3DD.%2BV.%26aulast%3DHoll%26aufirst%3DR.%26atitle%3DInsights%2520Into%2520the%2520Zinc-Dependent%2520Deacetylase%2520LpxC%253A%2520Biochemical%2520Properties%2520and%2520Inhibitor%2520Design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2016%26volume%3D16%26spage%3D2379%26epage%3D2430%26doi%3D10.2174%2F1568026616666160413135835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erwin, A. L.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Drug Discovery Targeting the Lipopolysaccharide Biosynthetic Enzyme LpxC</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">a025304</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a025304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1101%2Fcshperspect.a025304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=27235477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=a025304&author=A.+L.+Erwin&title=Antibacterial+Drug+Discovery+Targeting+the+Lipopolysaccharide+Biosynthetic+Enzyme+LpxC&doi=10.1101%2Fcshperspect.a025304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial drug discovery targeting the lipopolysaccharide biosynthetic enzyme LpxC</span></div><div class="casAuthors">Erwin, Alice L.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a025304/1-a025304/15</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The enzyme LpxC (UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase) is broadly conserved across Gram-neg. bacteria and is essential for synthesis of lipid A, the membrane anchor of the lipopolysaccharides (LPSs), which are a major component of the outer membrane in nearly all Gram-neg. bacteria.  LpxC has been the focus of targetdirected antibiotic discovery projects in numerous pharmaceutical and academic groups for more than 20 years.  Despite intense effort, no LpxC inhibitor has been approved for therapeutic use, and only one has yet reached human studies.  This article will summarize the history of LpxC as a drug target and the parallel history of research on LpxC biol.  Both academic and industrial researchers have used LpxC inhibitors as tool compds., leading to increased understanding of the differing mechanisms for regulation of LPS synthesis in Escherichia coli and Pseudomonas aeruginosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-AZMwFme-97Vg90H21EOLACvtfcHk0lg3mNJZZNtkfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1ymurc%253D&md5=a8ca24f54a9aaad079b94a41cfab62a4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a025304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a025304%26sid%3Dliteratum%253Aachs%26aulast%3DErwin%26aufirst%3DA.%2BL.%26atitle%3DAntibacterial%2520Drug%2520Discovery%2520Targeting%2520the%2520Lipopolysaccharide%2520Biosynthetic%2520Enzyme%2520LpxC%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2016%26volume%3D6%26spage%3Da025304%26doi%3D10.1101%2Fcshperspect.a025304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Onishi, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelak, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerckens, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patchett, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galloway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Agents That Inhibit Lipid A Biosynthesis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">980</span>– <span class="NLM_lpage">982</span>, <span class="refDoi"> DOI: 10.1126/science.274.5289.980</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1126%2Fscience.274.5289.980" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=8875939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADyaK28XmslKmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=980-982&author=H.+R.+Onishiauthor=B.+A.+Pelakauthor=L.+S.+Gerckensauthor=L.+L.+Silverauthor=F.+M.+Kahanauthor=M.+H.+Chenauthor=A.+A.+Patchettauthor=S.+M.+Gallowayauthor=S.+A.+Hylandauthor=M.+S.+Andersonauthor=C.+R.+Raetz&title=Antibacterial+Agents+That+Inhibit+Lipid+A+Biosynthesis&doi=10.1126%2Fscience.274.5289.980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial agents that inhibit lipid A biosynthesis</span></div><div class="casAuthors">Onishi, H. Russell; Pelak, Barbara A.; Gerckens, Lynn S.; Silver, Lynn L.; Kahan, Frederick M.; Chen, Meng-Hsin; Patchett, Arthur A.; Galloway, Susan M.; Hyland, Sheryl A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5289</span>),
    <span class="NLM_cas:pages">980-982</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lipid A constitutes the outer monolayer of the outer membrane of Gram-neg. bacteria and is essential for bacterial growth.  Synthetic antibacterials were identified that inhibit the second enzyme (a unique deacetylase) of lipid A biosynthesis.  The inhibitors are chiral hydroxamic acids bearing certain hydrophobic arom. moieties.  They may bind to a metal in the active site of the deacetylase.  The most potent analog (with an inhibition const. of about 50 nM) displayed a minimal inhibitory concn. of about 1 μg per mL against Escherichia coli, causes three logs of bacterial killing in 4 h, and cured mice infected with a lethal i.p. dose of E. coli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVKgUe_5zI9rVg90H21EOLACvtfcHk0lj99mkLDwgLSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmslKmsLw%253D&md5=d5098756849325110d7f1f4850322cba</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5289.980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5289.980%26sid%3Dliteratum%253Aachs%26aulast%3DOnishi%26aufirst%3DH.%2BR.%26aulast%3DPelak%26aufirst%3DB.%2BA.%26aulast%3DGerckens%26aufirst%3DL.%2BS.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26aulast%3DKahan%26aufirst%3DF.%2BM.%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DPatchett%26aufirst%3DA.%2BA.%26aulast%3DGalloway%26aufirst%3DS.%2BM.%26aulast%3DHyland%26aufirst%3DS.%2BA.%26aulast%3DAnderson%26aufirst%3DM.%2BS.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26atitle%3DAntibacterial%2520Agents%2520That%2520Inhibit%2520Lipid%2520A%2520Biosynthesis%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D980%26epage%3D982%26doi%3D10.1126%2Fscience.274.5289.980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span> <i>Hydroxamic Acids</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S. P.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1007%2F978-3-642-38111-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=S.+P.+Gupta&title=Hydroxamic+Acids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-38111-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-38111-9%26sid%3Dliteratum%253Aachs%26btitle%3DHydroxamic%2520Acids%26aulast%3DGupta%26aufirst%3DS.%2BP.%26pub%3DSpringer%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span> <span> </span><span class="NLM_article-title">Syntheses, Structures and Antibiotic Activities of LpxC Inhibitors Based on the Diacetylene Scaffold</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2Fj.bmc.2010.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=21194954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=852-860&author=X.+Liangauthor=C.+J.+Leeauthor=X.+Chenauthor=H.+S.+Chungauthor=D.+Zengauthor=C.+R.+Raetzauthor=Y.+Liauthor=P.+Zhouauthor=E.+J.+Toone&title=Syntheses%2C+Structures+and+Antibiotic+Activities+of+LpxC+Inhibitors+Based+on+the+Diacetylene+Scaffold&doi=10.1016%2Fj.bmc.2010.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Syntheses, structures and antibiotic activities of LpxC inhibitors based on the diacetylene scaffold</span></div><div class="casAuthors">Liang, Xiaofei; Lee, Chul-Jin; Chen, Xin; Chung, Hak Suk; Zeng, Daina; Raetz, Christian R. H.; Li, Yaoxian; Zhou, Pei; Toone, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">852-860</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Compds. inhibiting LpxC in the lipid A biosynthetic pathway are promising leads for novel antibiotics against multidrug-resistant Gram-neg. pathogens.  The authors report the syntheses and structural and biochem. characterizations of LpxC inhibitors based on a diphenyl-diacetylene (1,4-diphenyl-1,3-butadiyne) threonyl-hydroxamate scaffold.  These studies provide a mol. interpretation for the differential antibiotic activities of compds. with a substituted distal Ph ring as well as the abs. stereochem. requirement at the C2, but not C3, position of the threonyl group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWL2QPzfjArLVg90H21EOLACvtfcHk0ljT6tXta5x1Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVCnuw%253D%253D&md5=10326af21364b0785a67692687b21385</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChung%26aufirst%3DH.%2BS.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26atitle%3DSyntheses%252C%2520Structures%2520and%2520Antibiotic%2520Activities%2520of%2520LpxC%2520Inhibitors%2520Based%2520on%2520the%2520Diacetylene%2520Scaffold%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D852%26epage%3D860%26doi%3D10.1016%2Fj.bmc.2010.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joo, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barb, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholas, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span> <span> </span><span class="NLM_article-title">Species-Specific and Inhibitor-Dependent Conformations of LpxC: Implications for Antibiotic Design</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2Fj.chembiol.2010.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=21167751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlars70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=38-47&author=C.+J.+Leeauthor=X.+Liangauthor=X.+Chenauthor=D.+Zengauthor=S.+H.+Jooauthor=H.+S.+Chungauthor=A.+W.+Barbauthor=S.+M.+Swansonauthor=R.+A.+Nicholasauthor=Y.+Liauthor=E.+J.+Tooneauthor=C.+R.+Raetzauthor=P.+Zhou&title=Species-Specific+and+Inhibitor-Dependent+Conformations+of+LpxC%3A+Implications+for+Antibiotic+Design&doi=10.1016%2Fj.chembiol.2010.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Species-specific and inhibitor-dependent conformations of LpxC: implications for antibiotic design</span></div><div class="casAuthors">Lee, Chul-Jin; Liang, Xiao-Fei; Chen, Xin; Zeng, Dai-Na; Joo, Sang-Hoon; Chung, Hak-Suk; Barb, Adam W.; Swanson, Shauna M.; Nicholas, Robert A.; Li, Yao-Xian; Toone, Eric J.; Raetz, Christian R. H.; Zhou, Pei</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-47</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LpxC is an essential enzyme in the lipid A biosynthetic pathway in gram-neg. bacteria.  Several promising antimicrobial lead compds. targeting LpxC have been reported, though they typically display a large variation in potency against different gram-neg. pathogens.  We report that inhibitors with a diacetylene scaffold effectively overcome the resistance caused by sequence variation in the LpxC substrate-binding passage.  Compd. binding is captured in complex with representative LpxC orthologs, and structural anal. reveals large conformational differences that mostly reflect inherent mol. features of distinct LpxC orthologs, whereas ligand-induced structural adaptations occur at a smaller scale.  These observations highlight the need for a mol. understanding of inherent structural features and conformational plasticity of LpxC enzymes for optimizing LpxC inhibitors as broad-spectrum antibiotics against gram-neg. infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbcwncn731jrVg90H21EOLACvtfcHk0ljWI7Rt2b1RqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlars70%253D&md5=7aa1222210dfec8ad8675194b0b13731</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DJoo%26aufirst%3DS.%2BH.%26aulast%3DChung%26aufirst%3DH.%2BS.%26aulast%3DBarb%26aufirst%3DA.%2BW.%26aulast%3DSwanson%26aufirst%3DS.%2BM.%26aulast%3DNicholas%26aufirst%3DR.%2BA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DSpecies-Specific%2520and%2520Inhibitor-Dependent%2520Conformations%2520of%2520LpxC%253A%2520Implications%2520for%2520Antibiotic%2520Design%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D38%26epage%3D47%26doi%3D10.1016%2Fj.chembiol.2010.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis, Structure, and Antibiotic Activity of Aryl-Substituted LpxC Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6954</span>– <span class="NLM_lpage">6966</span>, <span class="refDoi"> DOI: 10.1021/jm4007774</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4007774" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WltrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6954-6966&author=X.+Liangauthor=C.+J.+Leeauthor=J.+Zhaoauthor=E.+J.+Tooneauthor=P.+Zhou&title=Synthesis%2C+Structure%2C+and+Antibiotic+Activity+of+Aryl-Substituted+LpxC+Inhibitors&doi=10.1021%2Fjm4007774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, Structure, and Antibiotic Activity of Aryl-Substituted LpxC Inhibitors</span></div><div class="casAuthors">Liang, Xiaofei; Lee, Chul-Jin; Zhao, Jinshi; Toone, Eric J.; Zhou, Pei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6954-6966</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The zinc-dependent deacetylase LpxC catalyzes the committed step of lipid A biosynthesis in Gram-neg. bacteria and is a validated target for the development of novel antibiotics to combat multidrug-resistant Gram-neg. infections.  Many potent LpxC inhibitors contain an essential threonyl-hydroxamate headgroup for high-affinity interaction with LpxC.  The authors report the synthesis, antibiotic activity, and structural and enzymic characterization of novel LpxC inhibitors contg. an addnl. aryl group in the threonyl-hydroxamate moiety, which expands the inhibitor-binding surface in LpxC.  These compds. display enhanced potency against LpxC in enzymic assays and superior antibiotic activity against Francisella novicida in cell culture.  The comparison of the antibiotic activities of these compds. against a leaky Escherichia coli strain and the wild-type strain reveals the contribution of the formidable outer-membrane permeability barrier that reduces the compds. efficacy in cell culture and emphasizes the importance of maintaining a balanced hydrophobicity and hydrophilicity profile in developing effective LpxC-targeting antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUuKY9tg3BU7Vg90H21EOLACvtfcHk0licC61mTh4U8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WltrrJ&md5=d81226a4e668a5f819ea7248ce76c7ae</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm4007774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4007774%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DC.%2BJ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DSynthesis%252C%2520Structure%252C%2520and%2520Antibiotic%2520Activity%2520of%2520Aryl-Substituted%2520LpxC%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6954%26epage%3D6966%26doi%3D10.1021%2Fjm4007774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barb, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span> <span> </span><span class="NLM_article-title">Structure of the Deacetylase LpxC Bound to the Antibiotic CHIR-090: Time-Dependent Inhibition and Specificity in Ligand Binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">18433</span>– <span class="NLM_lpage">18438</span>, <span class="refDoi"> DOI: 10.1073/pnas.0709412104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1073%2Fpnas.0709412104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=18025458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2ku7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18433-18438&author=A.+W.+Barbauthor=L.+Jiangauthor=C.+R.+Raetzauthor=P.+Zhou&title=Structure+of+the+Deacetylase+LpxC+Bound+to+the+Antibiotic+CHIR-090%3A+Time-Dependent+Inhibition+and+Specificity+in+Ligand+Binding&doi=10.1073%2Fpnas.0709412104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the deacetylase LpxC bound to the antibiotic CHIR-090: time-dependent inhibition and specificity in ligand binding</span></div><div class="casAuthors">Barb, Adam W.; Jiang, Ling; Raetz, Christian R. H.; Zhou, Pei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">18433-18438, S18433/1-S18433/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The UDP-3-O-(R-3-hydroxyacyl)-N-acetylglucosamine deacetylase LpxC is an essential enzyme of lipid A biosynthesis in Gram-neg. bacteria and a promising antibiotic target.  CHIR-090, the most potent LpxC inhibitor discovered to date, displays two-step time-dependent inhibition and kills a wide range of Gram-neg. pathogens as effectively as ciprofloxacin or tobramycin.  In this study, we report the soln. structure of the LpxC-CHIR-090 complex.  CHIR-090 exploits conserved features of LpxC that are crit. for catalysis, including the hydrophobic passage and essential active-site residues.  CHIR-090 is adjacent to, but does not occupy, the UDP-binding pocket of LpxC, suggesting that a fragment-based approach may facilitate further optimization of LpxC inhibitors.  Addnl., we identified key residues in the Insert II hydrophobic passage that modulate time-dependent inhibition and CHIR-090 resistance.  CHIR-090 shares a similar, although previously unrecognized, chem. scaffold with other small-mol. antibiotics such as L-161,240 targeting LpxC, and provides a template for understanding the binding mode of these inhibitors.  Consistent with this model, we provide evidence that L-161,240 also occupies the hydrophobic passage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraWHV0AFZnM7Vg90H21EOLACvtfcHk0lgphIbu5I2q0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2ku7rP&md5=e54a3a1c978536947380f6f1bbdbc4bf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0709412104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0709412104%26sid%3Dliteratum%253Aachs%26aulast%3DBarb%26aufirst%3DA.%2BW.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DRaetz%26aufirst%3DC.%2BR.%26aulast%3DZhou%26aufirst%3DP.%26atitle%3DStructure%2520of%2520the%2520Deacetylase%2520LpxC%2520Bound%2520to%2520the%2520Antibiotic%2520CHIR-090%253A%2520Time-Dependent%2520Inhibition%2520and%2520Specificity%2520in%2520Ligand%2520Binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D18433%26epage%3D18438%26doi%3D10.1073%2Fpnas.0709412104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piizzi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wattanasin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanieri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojkovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzink-Fox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lilly, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranjitkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, C. R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Properties of a Potent Inhibitor of <i>Pseudomonas aeruginosa</i> Deacetylase LpxC</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5002</span>– <span class="NLM_lpage">5014</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00377</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00377" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1SgsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5002-5014&author=G.+Piizziauthor=D.+T.+Parkerauthor=Y.+Pengauthor=M.+Doblerauthor=A.+Patnaikauthor=S.+Wattanasinauthor=E.+Liuauthor=F.+Lenoirauthor=J.+Nunezauthor=J.+Kerriganauthor=D.+McKenneyauthor=C.+Osborneauthor=D.+Yuauthor=L.+Lanieriauthor=J.+Bojkovicauthor=J.+Dzink-Foxauthor=M.+D.+Lillyauthor=E.+R.+Spragueauthor=Y.+Luauthor=H.+Wangauthor=S.+Ranjitkarauthor=L.+Xieauthor=B.+Wangauthor=M.+Glickauthor=L.+G.+Hamannauthor=R.+Tommasiauthor=X.+Yangauthor=C.+R.+Dean&title=Design%2C+Synthesis%2C+and+Properties+of+a+Potent+Inhibitor+of+Pseudomonas+aeruginosa+Deacetylase+LpxC&doi=10.1021%2Facs.jmedchem.7b00377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Properties of a Potent Inhibitor of Pseudomonas aeruginosa Deacetylase LpxC</span></div><div class="casAuthors">Piizzi, Grazia; Parker, David T.; Peng, Yunshan; Dobler, Markus; Patnaik, Anup; Wattanasin, Som; Liu, Eugene; Lenoir, Francois; Nunez, Jill; Kerrigan, John; McKenney, David; Osborne, Colin; Yu, Donghui; Lanieri, Leanne; Bojkovic, Jade; Dzink-Fox, JoAnn; Lilly, Maria-Dawn; Sprague, Elizabeth R.; Lu, Yipin; Wang, Hongming; Ranjitkar, Srijan; Xie, Lili; Wang, Bing; Glick, Meir; Hamann, Lawrence G.; Tommasi, Ruben; Yang, Xia; Dean, Charles R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5002-5014</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Over the past several decades, the frequency of antibacterial resistance in hospitals, including multidrug resistance (MDR) and its assocn. with serious infectious diseases, has increased at alarming rates.  Pseudomonas aeruginosa is a leading cause of nosocomial infections, and resistance to virtually all approved antibacterial agents is emerging in this pathogen.  To address the need for new agents to treat MDR P. aeruginosa, we focused on inhibiting the first committed step in the biosynthesis of lipid A, the deacetylation of uridyldiphospho-3-O-(R-hydroxydecanoyl)-N-acetylglucosamine by the enzyme LpxC.  We approached this through the design, synthesis, and biol. evaluation of novel hydroxamic acid LpxC inhibitors, exemplified by I, where cytotoxicity against mammalian cell lines was reduced, soly. and plasma-protein binding were improved while retaining potent anti-pseudomonal activity in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbG9H5F_ITrVg90H21EOLACvtfcHk0lgw3His_fqSSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1SgsLk%253D&md5=ea3fc92a1b15ffed9edd3702395a2b35</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00377%26sid%3Dliteratum%253Aachs%26aulast%3DPiizzi%26aufirst%3DG.%26aulast%3DParker%26aufirst%3DD.%2BT.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DDobler%26aufirst%3DM.%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DWattanasin%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DE.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DNunez%26aufirst%3DJ.%26aulast%3DKerrigan%26aufirst%3DJ.%26aulast%3DMcKenney%26aufirst%3DD.%26aulast%3DOsborne%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DLanieri%26aufirst%3DL.%26aulast%3DBojkovic%26aufirst%3DJ.%26aulast%3DDzink-Fox%26aufirst%3DJ.%26aulast%3DLilly%26aufirst%3DM.%2BD.%26aulast%3DSprague%26aufirst%3DE.%2BR.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DRanjitkar%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DTommasi%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DDean%26aufirst%3DC.%2BR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Properties%2520of%2520a%2520Potent%2520Inhibitor%2520of%2520Pseudomonas%2520aeruginosa%2520Deacetylase%2520LpxC%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5002%26epage%3D5014%26doi%3D10.1021%2Facs.jmedchem.7b00377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcari, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collantes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engtrakul, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knafels, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishka, D. G.</span></span> <span> </span><span class="NLM_article-title">Potent Inhibitors of LpxC for the Treatment of Gram-Negative Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">914</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1021/jm2014748</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2014748" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ers7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=914-923&author=M.+F.+Brownauthor=U.+Reillyauthor=J.+A.+Abramiteauthor=J.+T.+Arcariauthor=R.+Oliverauthor=R.+A.+Barhamauthor=Y.+Cheauthor=J.+M.+Chenauthor=E.+M.+Collantesauthor=S.+W.+Chungauthor=C.+Desbonnetauthor=J.+Dotyauthor=M.+Doroskiauthor=J.+J.+Engtrakulauthor=T.+M.+Harrisauthor=M.+Hubandauthor=J.+D.+Knafelsauthor=K.+L.+Leachauthor=S.+Liuauthor=A.+Marfatauthor=A.+Marraauthor=E.+McElroyauthor=M.+Melnickauthor=C.+A.+Menardauthor=J.+I.+Montgomeryauthor=L.+Mullinsauthor=M.+C.+Noeauthor=J.+O%E2%80%99Donnellauthor=J.+Penzienauthor=M.+S.+Plummerauthor=L.+M.+Priceauthor=V.+Shanmugasundaramauthor=C.+Thomaauthor=D.+P.+Uccelloauthor=J.+S.+Warmusauthor=D.+G.+Wishka&title=Potent+Inhibitors+of+LpxC+for+the+Treatment+of+Gram-Negative+Infections&doi=10.1021%2Fjm2014748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Inhibitors of LpxC for the Treatment of Gram-Negative Infections</span></div><div class="casAuthors">Brown, Matthew F.; Reilly, Usa; Abramite, Joseph A.; Arcari, Joel T.; Oliver, Robert; Barham, Rose A.; Che, Ye; Chen, Jinshan Michael; Collantes, Elizabeth M.; Chung, Seung Won; Desbonnet, Charlene; Doty, Jonathan; Doroski, Matthew; Engtrakul, Juntyma J.; Harris, Thomas M.; Huband, Michael; Knafels, John D.; Leach, Karen L.; Liu, Shenping; Marfat, Anthony; Marra, Andrea; McElroy, Eric; Melnick, Michael; Menard, Carol A.; Montgomery, Justin I.; Mullins, Lisa; Noe, Mark. C.; O'Donnell, John; Penzien, Joseph; Plummer, Mark S.; Price, Loren M.; Shanmugasundaram, Veerabahu; Thoma, Christy; Uccello, Daniel P.; Warmus, Joseph S.; Wishka, Donn G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">914-923</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper, the synthesis and SAR as well as selectivity, pharmacokinetic, and infection model data for representative analogs of a novel series of potent antibacterial LpxC inhibitors represented by hydroxamic acid I, is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpizPNDLsgyIbVg90H21EOLACvtfcHk0lhVzk0vLBw_DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1ers7rE&md5=fda4af453d09a4baa25cac35f69f5ac3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm2014748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2014748%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DArcari%26aufirst%3DJ.%2BT.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DBarham%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DCollantes%26aufirst%3DE.%2BM.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DDoroski%26aufirst%3DM.%26aulast%3DEngtrakul%26aufirst%3DJ.%2BJ.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKnafels%26aufirst%3DJ.%2BD.%26aulast%3DLeach%26aufirst%3DK.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMarra%26aufirst%3DA.%26aulast%3DMcElroy%26aufirst%3DE.%26aulast%3DMelnick%26aufirst%3DM.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DPenzien%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DM.%2BS.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DThoma%26aufirst%3DC.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26aulast%3DWishka%26aufirst%3DD.%2BG.%26atitle%3DPotent%2520Inhibitors%2520of%2520LpxC%2520for%2520the%2520Treatment%2520of%2520Gram-Negative%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D914%26epage%3D923%26doi%3D10.1021%2Fjm2014748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arcari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collantes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doty, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engtrakul, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knafels, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishka, D. G.</span></span> <span> </span><span class="NLM_article-title">Pyridone Methylsulfone Hydroxamate LpxC Inhibitors for the Treatment of Serious Gram-Negative Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1670</span>, <span class="refDoi"> DOI: 10.1021/jm2014875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2014875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1662-1670&author=J.+I.+Montgomeryauthor=M.+F.+Brownauthor=U.+Reillyauthor=L.+M.+Priceauthor=J.+A.+Abramiteauthor=J.+Arcariauthor=R.+Barhamauthor=Y.+Cheauthor=J.+M.+Chenauthor=S.+W.+Chungauthor=E.+M.+Collantesauthor=C.+Desbonnetauthor=M.+Doroskiauthor=J.+Dotyauthor=J.+J.+Engtrakulauthor=T.+M.+Harrisauthor=M.+Hubandauthor=J.+D.+Knafelsauthor=K.+L.+Leachauthor=S.+Liuauthor=A.+Marfatauthor=L.+McAllisterauthor=E.+McElroyauthor=C.+A.+Menardauthor=M.+Mitton-Fryauthor=L.+Mullinsauthor=M.+C.+Noeauthor=J.+O%E2%80%99Donnellauthor=R.+Oliverauthor=J.+Penzienauthor=M.+Plummerauthor=V.+Shanmugasundaramauthor=C.+Thomaauthor=A.+P.+Tomarasauthor=D.+P.+Uccelloauthor=A.+Vazauthor=D.+G.+Wishka&title=Pyridone+Methylsulfone+Hydroxamate+LpxC+Inhibitors+for+the+Treatment+of+Serious+Gram-Negative+Infections&doi=10.1021%2Fjm2014875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridone methylsulfone hydroxamate LpxC inhibitors for the treatment of serious gram-negative infections</span></div><div class="casAuthors">Montgomery, Justin I.; Brown, Matthew F.; Reilly, Usa; Price, Loren M.; Abramite, Joseph A.; Arcari, Joel; Barham, Rose; Che, Ye; Chen, Jinshan Michael; Chung, Seung Won; Collantes, Elizabeth M.; Desbonnet, Charlene; Doroski, Matthew; Doty, Jonathan; Engtrakul, Juntyma J.; Harris, Thomas M.; Huband, Michael; Knafels, John D.; Leach, Karen L.; Liu, Shenping; Marfat, Anthony; McAllister, Laura; McElroy, Eric; Menard, Carol A.; Mitton-Fry, Mark; Mullins, Lisa; Noe, Mark C.; O'Donnell, John; Oliver, Robert; Penzien, Joseph; Plummer, Mark; Shanmugasundaram, Veerabahu; Thoma, Christy; Tomaras, Andrew P.; Uccello, Daniel P.; Vaz, Alfin; Wishka, Donn G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1662-1670</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and biol. activity of a new series of LpxC inhibitors represented by pyridone methylsulfone hydroxamate 2a (I) is presented.  Members of this series have improved soly. and free fraction when compared to compds. in the previously described biphenyl methylsulfone hydroxamate series, and they maintain superior Gram-neg. antibacterial activity to comparator agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgNGd-bEXEabVg90H21EOLACvtfcHk0lj5QFpixdfNCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsVyjtw%253D%253D&md5=f2410aae177b2fa24d8da72942b95564</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm2014875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2014875%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DArcari%26aufirst%3DJ.%26aulast%3DBarham%26aufirst%3DR.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DCollantes%26aufirst%3DE.%2BM.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DDoroski%26aufirst%3DM.%26aulast%3DDoty%26aufirst%3DJ.%26aulast%3DEngtrakul%26aufirst%3DJ.%2BJ.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKnafels%26aufirst%3DJ.%2BD.%26aulast%3DLeach%26aufirst%3DK.%2BL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMarfat%26aufirst%3DA.%26aulast%3DMcAllister%26aufirst%3DL.%26aulast%3DMcElroy%26aufirst%3DE.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMitton-Fry%26aufirst%3DM.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DPenzien%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DThoma%26aufirst%3DC.%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DVaz%26aufirst%3DA.%26aulast%3DWishka%26aufirst%3DD.%2BG.%26atitle%3DPyridone%2520Methylsulfone%2520Hydroxamate%2520LpxC%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Serious%2520Gram-Negative%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1662%26epage%3D1670%26doi%3D10.1021%2Fjm2014875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Schmitt, C.</span>; <span class="NLM_string-name">Surivet, J.-P.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrrolidinone Derivatives for Use as Antibacterial Agents</span>. World Patent <span class="NLM_patent">WO 2017037039</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=P.+Panchaud&author=C.+Schmitt&author=J.-P.+Surivet&title=Preparation+of+Pyrrolidinone+Derivatives+for+Use+as+Antibacterial+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPanchaud%26aufirst%3DP.%26atitle%3DPreparation%2520of%2520Pyrrolidinone%2520Derivatives%2520for%2520Use%2520as%2520Antibacterial%2520Agents%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Specklin, J.-L.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Schmitt, C.</span></span> <span> </span><span class="NLM_article-title">Preparation of Dihydropyrrolo[1, 2-<i>c</i>]imidazol-3-one Derivatives Useful as Antibacterial Agents</span>. World Patent <span class="NLM_patent">WO 2015173329</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=G.+Chapoux&author=J.-C.+Gauvin&author=P.+Panchaud&author=J.-L.+Specklin&author=J.-P.+Surivet&author=C.+Schmitt&title=Preparation+of+Dihydropyrrolo%5B1%2C+2-c%5Dimidazol-3-one+Derivatives+Useful+as+Antibacterial+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChapoux%26aufirst%3DG.%26atitle%3DPreparation%2520of%2520Dihydropyrrolo%255B1%252C%25202-c%255Dimidazol-3-one%2520Derivatives%2520Useful%2520as%2520Antibacterial%2520Agents%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit25b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Blumstein, A. C.</span>; <span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Jacob, L.</span>; <span class="NLM_string-name">Masse, F.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Schmitt, C.</span>; <span class="NLM_string-name">Surivet, J. P.</span>; <span class="NLM_string-name">Specklin, J. L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Substituted 1,2-Dihydro-3<i>H</i>-pyrrolo[1,2-<i>c</i>]imidazol-3-one Antibacterial Compounds</span>. World Patent <span class="NLM_patent">WO 2017037221</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=A.+C.+Blumstein&author=G.+Chapoux&author=L.+Jacob&author=F.+Masse&author=A.+Mirre&author=P.+Panchaud&author=C.+Schmitt&author=J.+P.+Surivet&author=J.+L.+Specklin&title=Preparation+of+Substituted+1%2C2-Dihydro-3H-pyrrolo%5B1%2C2-c%5Dimidazol-3-one+Antibacterial+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBlumstein%26aufirst%3DA.%2BC.%26atitle%3DPreparation%2520of%2520Substituted%25201%252C2-Dihydro-3H-pyrrolo%255B1%252C2-c%255Dimidazol-3-one%2520Antibacterial%2520Compounds%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chinchilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nájera, C.</span></span> <span> </span><span class="NLM_article-title">The Sonogashira Reaction: A Booming Methodology in Synthetic Organic Chemistry</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">874</span>– <span class="NLM_lpage">922</span>, <span class="refDoi"> DOI: 10.1021/cr050992x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr050992x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2007&pages=874-922&author=R.+Chinchillaauthor=C.+N%C3%A1jera&title=The+Sonogashira+Reaction%3A+A+Booming+Methodology+in+Synthetic+Organic+Chemistry&doi=10.1021%2Fcr050992x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The Sonogashira reaction: a booming methodology in synthetic organic chemistry</span></div><div class="casAuthors">Chinchilla, Rafael; Najera, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">874-922</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The palladium-catalyzed sp2-sp coupling reaction between aryl or alkenyl halides or triflates and terminal alkynes, with or without the presence of copper(I) cocatalyst, was reviewed.  The mechanism of Sonogashira reaction was discussed, and the catalysts and reaction conditions were explored as well.  In addn., the application of Sonogashira cross-coupling reaction was also introduced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ8ePd8RdbSrVg90H21EOLACvtfcHk0liMXEwa-vjFjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbc%253D&md5=91f627efbf940c60809c2acfe597d3b7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcr050992x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr050992x%26sid%3Dliteratum%253Aachs%26aulast%3DChinchilla%26aufirst%3DR.%26aulast%3DN%25C3%25A1jera%26aufirst%3DC.%26atitle%3DThe%2520Sonogashira%2520Reaction%253A%2520A%2520Booming%2520Methodology%2520in%2520Synthetic%2520Organic%2520Chemistry%26jtitle%3DChem.%2520Rev.%26date%3D2007%26volume%3D107%26spage%3D874%26epage%3D922%26doi%3D10.1021%2Fcr050992x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cadiot, P.</span>; <span class="NLM_string-name">Chodkiewicz, W.</span></span> In  <i>Chemistry of Acetylenes</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Viehe, H. G.</span></span>, Ed.; <span class="NLM_publisher-name">Marcel Dekker</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1969</span>; pp  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">647</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1969&pages=597-647&author=P.+Cadiot&author=W.+Chodkiewiczauthor=H.+G.+Viehe&title=Chemistry+of+Acetylenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCadiot%26aufirst%3DP.%26btitle%3DChemistry%2520of%2520Acetylenes%26aulast%3DViehe%26aufirst%3DH.%2BG.%26pub%3DMarcel%2520Dekker%26date%3D1969%26spage%3D597%26epage%3D647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="note"><p class="first last"><i>Klebsiella pneumoniae</i> T6474 is a clinical isolate expressing ESBL and with mutations in the Quinolone Resistance Determining Region of gyrA (Ser83-Ile) and ParC (Ser80-Ile). It is resistant to ampicillin, aztreonam, ceftriaxone, norfloxacin, and trimethoprim.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="note"><p class="first last">For details on assay conditions, calibration and relevance, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The Use of Differential Scanning Fluorimetry to Detect Ligand Interactions That Promote Protein Stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+Use+of+Differential+Scanning+Fluorimetry+to+Detect+Ligand+Interactions+That+Promote+Protein+Stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lgJi5DZm3a1UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520Use%2520of%2520Differential%2520Scanning%2520Fluorimetry%2520to%2520Detect%2520Ligand%2520Interactions%2520That%2520Promote%2520Protein%2520Stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="note"><p class="first last">Data collection and refinements statistics for all structures are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barham, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitton-Fry, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R. M.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penzien, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span> <span> </span><span class="NLM_article-title">Heterocyclic Methylsulfone Hydroxamic Acid LpxC Inhibitors as Gram-Negative Antibacterial Agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">6832</span>– <span class="NLM_lpage">6838</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.09.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2Fj.bmcl.2012.09.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=23046961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVykur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=6832-6838&author=L.+A.+McAllisterauthor=J.+I.+Montgomeryauthor=J.+A.+Abramiteauthor=U.+Reillyauthor=M.+F.+Brownauthor=J.+M.+Chenauthor=R.+A.+Barhamauthor=Y.+Cheauthor=S.+W.+Chungauthor=C.+A.+Menardauthor=M.+Mitton-Fryauthor=L.+M.+Mullinsauthor=M.+C.+Noeauthor=J.+P.+O%E2%80%99Donnellauthor=R.+M.+Oliverauthor=J.+B.+Penzienauthor=M.+Plummerauthor=L.+M.+Priceauthor=V.+Shanmugasundaramauthor=A.+P.+Tomarasauthor=D.+P.+Uccello&title=Heterocyclic+Methylsulfone+Hydroxamic+Acid+LpxC+Inhibitors+as+Gram-Negative+Antibacterial+Agents&doi=10.1016%2Fj.bmcl.2012.09.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Heterocyclic methylsulfone hydroxamic acid LpxC inhibitors as Gram-negative antibacterial agents</span></div><div class="casAuthors">McAllister, Laura A.; Montgomery, Justin I.; Abramite, Joseph A.; Reilly, Usa; Brown, Matthew F.; Chen, Jinshan M.; Barham, Rose A.; Che, Ye; Chung, Seung Won; Menard, Carol A.; Mitton-Fry, Mark; Mullins, Lisa M.; Noe, Mark C.; O'Donnell, John P.; Oliver, Robert M.; Penzien, Joseph B.; Plummer, Mark; Price, Loren M.; Shanmugasundaram, Veerabahu; Tomaras, Andrew P.; Uccello, Daniel P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6832-6838</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The synthesis and antibacterial activity of heterocyclic methylsulfone hydroxamates is presented.  Compds. in this series are potent inhibitors of the LpxC enzyme, a key enzyme involved in the prodn. of lipopolysaccharide (LPS) found in the outer membrane of Gram-neg. bacteria.  SAR evaluation of compds. in this series revealed analogs, e.g., I, with potent antibacterial activity against challenging Gram-neg. species such as Pseudomonas aeruginosa and Klebsiella pneumoniae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_EQM-fls8LVg90H21EOLACvtfcHk0ljMXWmCyxHVDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVykur7P&md5=93dbfffc47f4eab6b67111ef7e67fa2e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.09.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.09.058%26sid%3Dliteratum%253Aachs%26aulast%3DMcAllister%26aufirst%3DL.%2BA.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DBarham%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DS.%2BW.%26aulast%3DMenard%26aufirst%3DC.%2BA.%26aulast%3DMitton-Fry%26aufirst%3DM.%26aulast%3DMullins%26aufirst%3DL.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26aulast%3DOliver%26aufirst%3DR.%2BM.%26aulast%3DPenzien%26aufirst%3DJ.%2BB.%26aulast%3DPlummer%26aufirst%3DM.%26aulast%3DPrice%26aufirst%3DL.%2BM.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26atitle%3DHeterocyclic%2520Methylsulfone%2520Hydroxamic%2520Acid%2520LpxC%2520Inhibitors%2520as%2520Gram-Negative%2520Antibacterial%2520Agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D6832%26epage%3D6838%26doi%3D10.1016%2Fj.bmcl.2012.09.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32a"><span>Related
2<i>H</i>-indazoles and benzothiazoles derivatives that
were synthesiszed and characterized during the course of our program
elicited somewhat lower solubilites and higher log <i>D</i><sub>7.4</sub> (data not shown). <span class="NLM_contrib-group"><span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Ochala, E.</span>; <span class="NLM_string-name">Surivet, J.-P.</span></span> <span> </span><span class="NLM_article-title">2<i>H</i>-Indazole Derivatives as Antibacterial and Their Preparation and Use</span>. World Patent <span class="NLM_patent">WO 2015036964</span>, <span class="NLM_year">2015</span>;.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.-C.+Gauvin&author=A.+Mirre&author=E.+Ochala&author=J.-P.+Surivet&title=2H-Indazole+Derivatives+as+Antibacterial+and+Their+Preparation+and+Use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGauvin%26aufirst%3DJ.-C.%26atitle%3D2H-Indazole%2520Derivatives%2520as%2520Antibacterial%2520and%2520Their%2520Preparation%2520and%2520Use%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit32b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, C.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Specklin, J.-L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Benzothiazole Derivatives as Antibacterial Agents</span>. World Patent <span class="NLM_patent">WO 2016079688</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=C.+Schmitt&author=J.-P.+Surivet&author=G.+Chapoux&author=A.+Mirre&author=J.-L.+Specklin&title=Preparation+of+Benzothiazole+Derivatives+as+Antibacterial+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DC.%26atitle%3DPreparation%2520of%2520Benzothiazole%2520Derivatives%2520as%2520Antibacterial%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chapoux, G.</span>; <span class="NLM_string-name">Diethelm, S.</span>; <span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Surivet, J.-P.</span></span> <span> </span><span class="NLM_article-title">Preparation of Functionalized Benzoxazolones and Benzothiazolones as Antibacterial Compounds</span>. World Patent <span class="NLM_patent">WO 2017037039</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Chapoux&author=S.+Diethelm&author=J.-C.+Gauvin&author=P.+Panchaud&author=J.-P.+Surivet&title=Preparation+of+Functionalized+Benzoxazolones+and+Benzothiazolones+as+Antibacterial+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChapoux%26aufirst%3DG.%26atitle%3DPreparation%2520of%2520Functionalized%2520Benzoxazolones%2520and%2520Benzothiazolones%2520as%2520Antibacterial%2520Compounds%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panchaud, P.</span>; <span class="NLM_string-name">Surivet, J.-P.</span>; <span class="NLM_string-name">Diethelm, S.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumstein, A.-C.</span>; <span class="NLM_string-name">Gauvin, J.-C.</span>; <span class="NLM_string-name">Jacob, L.</span>; <span class="NLM_string-name">Masse, F.</span>; <span class="NLM_string-name">Mathieu, G.</span>; <span class="NLM_string-name">Mirre, A.</span>; <span class="NLM_string-name">Schmitt, C.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Enderlin-Paput, M.</span>; <span class="NLM_string-name">Lange, R.</span>; <span class="NLM_string-name">Gnerre, C.</span>; <span class="NLM_string-name">Seeland, S.</span>; <span class="NLM_string-name">Herrmann, C.</span>; <span class="NLM_string-name">Locher, H. H.</span>; <span class="NLM_string-name">Seiler, P.</span>; <span class="NLM_string-name">Ritz, D.</span>; <span class="NLM_string-name">Rueedi, G.</span></span> <span> </span><span class="NLM_article-title">Optimization of LpxC Inhibitor Lead Compounds Focusing on Efficacy and Formulation for High Dose Intravenous Administration</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01605</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+Panchaud&author=J.-P.+Surivet&author=S.+Diethelmauthor=A.-C.+Blumstein&author=J.-C.+Gauvin&author=L.+Jacob&author=F.+Masse&author=G.+Mathieu&author=A.+Mirre&author=C.+Schmittauthor=M.+Enderlin-Paput&author=R.+Lange&author=C.+Gnerre&author=S.+Seeland&author=C.+Herrmann&author=H.+H.+Locher&author=P.+Seiler&author=D.+Ritz&author=G.+Rueedi&title=Optimization+of+LpxC+Inhibitor+Lead+Compounds+Focusing+on+Efficacy+and+Formulation+for+High+Dose+Intravenous+Administration&doi=10.1021%2Facs.jmedchem.9b01605"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01605%26sid%3Dliteratum%253Aachs%26aulast%3DPanchaud%26aufirst%3DP.%26aulast%3DBlumstein%26aufirst%3DA.-C.%26aulast%3DEnderlin-Paput%26aufirst%3DM.%26atitle%3DOptimization%2520of%2520LpxC%2520Inhibitor%2520Lead%2520Compounds%2520Focusing%2520on%2520Efficacy%2520and%2520Formulation%2520for%2520High%2520Dose%2520Intravenous%2520Administration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.9b01605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zissimos, A. M.</span></span> <span> </span><span class="NLM_article-title">Fast Calculation of van der Waals Volume as a Sum of Atomic and Bond Contributions and Its Application to Drug Compounds</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">7368</span>– <span class="NLM_lpage">7373</span>, <span class="refDoi"> DOI: 10.1021/jo034808o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo034808o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=7368-7373&author=Y.+H.+Zhaoauthor=M.+H.+Abrahamauthor=A.+M.+Zissimos&title=Fast+Calculation+of+van+der+Waals+Volume+as+a+Sum+of+Atomic+and+Bond+Contributions+and+Its+Application+to+Drug+Compounds&doi=10.1021%2Fjo034808o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of van der Waals Volume as a Sum of Atomic and Bond Contributions and Its Application to Drug Compounds</span></div><div class="casAuthors">Zhao, Yuan H.; Abraham, Michael H.; Zissimos, Andreas M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7368-7373</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The van der Waals vol. is a widely used descriptor in modeling physicochem. properties. However, the calcn. of the van der Waals vol. (VvdW) is rather time-consuming, from Bondi group contributions, for a large data set.  A new method for calcg. van der Waals vol. has been developed, based on Bondi radii.  The method, termed At. and Bond Contributions of van der Waals vol. (VABC), is very simple and fast.  The only information needed for calcg. VABC is at. contributions and the no. of atoms, bonds, and rings. Then, the van der Waals vol. (Å3/mol.) can be calcd. from the following formula: VvdW = .sum. all atom contributions - 5.92NB - 14.7RA - 3.8RNR (NB is the no. of bonds, RA is the no. of arom. rings, and RNA is the no. of nonarom. rings).  The no. of bonds present (NB) can be simply calcd. by NB = N - 1 + RA + RNA (where N is the total no. of atoms).  A simple Excel spread sheet has been made to calc. van der Waals vols. for a wide range of 677 org. compds., including 237 drug compds.  The results show that the van der Waals vols. calcd. from VABC are equiv. to the computer-calcd. van der Waals vols. for org. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojBiFlWWZKjbVg90H21EOLACvtfcHk0ljqRBEEqbXu1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D&md5=853cd54edc7a8c07e427196ee30d198f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjo034808o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034808o%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%2BH.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DZissimos%26aufirst%3DA.%2BM.%26atitle%3DFast%2520Calculation%2520of%2520van%2520der%2520Waals%2520Volume%2520as%2520a%2520Sum%2520of%2520Atomic%2520and%2520Bond%2520Contributions%2520and%2520Its%2520Application%2520to%2520Drug%2520Compounds%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D7368%26epage%3D7373%26doi%3D10.1021%2Fjo034808o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="note"><p class="first last">Structure–activity relationship studies focused on alternative segments <b>A</b> will be presented elsewhere.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasar, R.</span>; <span class="NLM_string-name">Linsell, M. S.</span>; <span class="NLM_string-name">Aggen, J. B.</span>; <span class="NLM_string-name">Lu, Q.</span>; <span class="NLM_string-name">Wang, D.</span>; <span class="NLM_string-name">Church, T.</span>; <span class="NLM_string-name">Moser, H. E.</span>; <span class="NLM_string-name">Patten, P. A.</span></span> <span> </span><span class="NLM_article-title">Hydroxamic Acid Derivatives and Their Use in the Treatment of Bacterial Infections</span>. World Patent <span class="NLM_patent">WO 2012154204</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=R.+Kasar&author=M.+S.+Linsell&author=J.+B.+Aggen&author=Q.+Lu&author=D.+Wang&author=T.+Church&author=H.+E.+Moser&author=P.+A.+Patten&title=Hydroxamic+Acid+Derivatives+and+Their+Use+in+the+Treatment+of+Bacterial+Infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKasar%26aufirst%3DR.%26atitle%3DHydroxamic%2520Acid%2520Derivatives%2520and%2520Their%2520Use%2520in%2520the%2520Treatment%2520of%2520Bacterial%2520Infections%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">See also</p></div><div class="NLM_citation" id="rightTab-cit37b"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assar, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dozzo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easterday, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haglund, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrandt, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konradi, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machajewski, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miroshnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plato, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seroogy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhmin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviridov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlasichuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirz, R. T.</span></span> <span> </span><span class="NLM_article-title">Optimization of LpxC Inhibitors for Antibacterial Activity and Cardiovascular Safety</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1002%2Fcmdc.201900287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=31283109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1560-1572&author=F.+Cohenauthor=J.+B.+Aggenauthor=L.+D.+Andrewsauthor=Z.+Assarauthor=J.+Boggsauthor=T.+Choiauthor=P.+Dozzoauthor=A.+N.+Easterdayauthor=C.+M.+Haglundauthor=D.+J.+Hildebrandtauthor=M.+C.+Holtauthor=K.+Jolyauthor=A.+Jubbauthor=Z.+Kamalauthor=T.+R.+Kaneauthor=A.+W.+Konradiauthor=K.+M.+Krauseauthor=M.+S.+Linsellauthor=T.+D.+Machajewskiauthor=O.+Miroshnikovaauthor=H.+E.+Moserauthor=V.+Nietoauthor=T.+Phanauthor=C.+Platoauthor=A.+W.+Serioauthor=J.+Seroogyauthor=A.+Shakhminauthor=A.+J.+Steinauthor=A.+D.+Sunauthor=S.+Sviridovauthor=Z.+Wangauthor=K.+Wlasichukauthor=W.+Yangauthor=X.+Zhouauthor=H.+Zhuauthor=R.+T.+Cirz&title=Optimization+of+LpxC+Inhibitors+for+Antibacterial+Activity+and+Cardiovascular+Safety&doi=10.1002%2Fcmdc.201900287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37bR"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of LpxC Inhibitors for Antibacterial Activity and Cardiovascular Safety</span></div><div class="casAuthors">Cohen, Frederick; Aggen, James B.; Andrews, Logan D.; Assar, Zahra; Boggs, Jen; Choi, Taylor; Dozzo, Paola; Easterday, Ashton N.; Haglund, Cat M.; Hildebrandt, Darin J.; Holt, Melissa C.; Joly, Kristin; Jubb, Adrian; Kamal, Zeeshan; Kane, Timothy R.; Konradi, Andrei W.; Krause, Kevin M.; Linsell, Martin S.; Machajewski, Timothy D.; Miroshnikova, Olga; Moser, Heinz E.; Nieto, Vincent; Phan, Thu; Plato, Craig; Serio, Alisa W.; Seroogy, Julie; Shakhmin, Anton; Stein, Adam J.; Sun, Alex D.; Sviridov, Serguei; Wang, Zhan; Wlasichuk, Kenneth; Yang, Wen; Zhou, Xiaoming; Zhu, Hai; Cirz, Ryan T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1560-1572</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylase (LpxC) is a Zn2+ deacetylase that is essential for the survival of most pathogenic Gram-neg. bacteria.  ACHN-975 (N-((S)-3-amino-1-(hydroxyamino)-3-methyl-1-oxobutan-2-yl)-4-(((1R,2R)-2-(hydroxymethyl)cyclopropyl)buta-1,3-diyn-1-yl)benzamide) was the first LpxC inhibitor to reach human clin. testing and was discovered to have a dose-limiting cardiovascular toxicity of transient hypotension without compensatory tachycardia.  Herein we report the effort beyond ACHN-975 to discover LpxC inhibitors optimized for enzyme potency, antibacterial activity, pharmacokinetics, and cardiovascular safety.  Based on its overall profile, compd. 26 (LPXC-516, (S)-N-(2-(hydroxyamino)-1-(3-methoxy-1,1-dioxidothietan-3-yl)-2-oxoethyl)-4-(6-hydroxyhexa-1,3-diyn-1-yl)benzamide) was chosen for further development.  A phosphate prodrug of 26 was developed that provided a soly. of >30 mg mL-1 for parenteral administration and conversion into the active drug with a t1/2 of approx. two minutes.  Unexpectedly, and despite our optimization efforts, the prodrug of 26 still possesses a therapeutic window insufficient to support further clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNhe0SbdQ2NbVg90H21EOLACvtfcHk0lgX7biEEEUZCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFSgtL7N&md5=b8cc87e1ebb76dfe8d330729a9d048d6</span></div><a href="/servlet/linkout?suffix=cit37b&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900287%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DAggen%26aufirst%3DJ.%2BB.%26aulast%3DAndrews%26aufirst%3DL.%2BD.%26aulast%3DAssar%26aufirst%3DZ.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DT.%26aulast%3DDozzo%26aufirst%3DP.%26aulast%3DEasterday%26aufirst%3DA.%2BN.%26aulast%3DHaglund%26aufirst%3DC.%2BM.%26aulast%3DHildebrandt%26aufirst%3DD.%2BJ.%26aulast%3DHolt%26aufirst%3DM.%2BC.%26aulast%3DJoly%26aufirst%3DK.%26aulast%3DJubb%26aufirst%3DA.%26aulast%3DKamal%26aufirst%3DZ.%26aulast%3DKane%26aufirst%3DT.%2BR.%26aulast%3DKonradi%26aufirst%3DA.%2BW.%26aulast%3DKrause%26aufirst%3DK.%2BM.%26aulast%3DLinsell%26aufirst%3DM.%2BS.%26aulast%3DMachajewski%26aufirst%3DT.%2BD.%26aulast%3DMiroshnikova%26aufirst%3DO.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26aulast%3DNieto%26aufirst%3DV.%26aulast%3DPhan%26aufirst%3DT.%26aulast%3DPlato%26aufirst%3DC.%26aulast%3DSerio%26aufirst%3DA.%2BW.%26aulast%3DSeroogy%26aufirst%3DJ.%26aulast%3DShakhmin%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DA.%2BJ.%26aulast%3DSun%26aufirst%3DA.%2BD.%26aulast%3DSviridov%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWlasichuk%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DCirz%26aufirst%3DR.%2BT.%26atitle%3DOptimization%2520of%2520LpxC%2520Inhibitors%2520for%2520Antibacterial%2520Activity%2520and%2520Cardiovascular%2520Safety%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D1560%26epage%3D1572%26doi%3D10.1002%2Fcmdc.201900287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="note"><p class="first last">All animal housing and experiments were performed in agreement with the Swiss Federal Ordinance for Animal Protection, the animal welfare guidelines from the Cantonal Veterinary Office Basellandschaft, and the Idorsia Pharmaceuticals Ltd. internal animal welfare guidelines.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, W. A.</span></span> <span> </span><span class="NLM_article-title">Animal Model Pharmacokinetics and Pharmacodynamics: a Critical Review</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/S0924-8579(02)00022-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1016%2FS0924-8579%2802%2900022-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=11978497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtVeis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2002&pages=261-268&author=D.+Andesauthor=W.+A.+Craig&title=Animal+Model+Pharmacokinetics+and+Pharmacodynamics%3A+a+Critical+Review&doi=10.1016%2FS0924-8579%2802%2900022-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Animal model pharmacokinetics and pharmacodynamics: a critical review</span></div><div class="casAuthors">Andes, D.; Craig, W. A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-268</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  Animals have been extensively used in the evaluation of antimicrobials.  The value of animals in the pharmacokinetic and pharmacodynamic characterization of antimicrobials is critically reviewed.  Animal studies have demonstrated that the pharmacokinetic/pharmacodynamic (PK/PD) target detg. efficacy can vary for different classes of antimicrobials.  However, the magnitude of the target required for bacteriol. efficacy is relatively similar for various sites of infection, various pathogens and various drugs within the same class, provided free drug levels are used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEIxgWGnD1TrVg90H21EOLACvtfcHk0lhhQ2BYoBgLDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtVeis7c%253D&md5=9167c5428197cce09400e0a1a438b02e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0924-8579%2802%2900022-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0924-8579%252802%252900022-5%26sid%3Dliteratum%253Aachs%26aulast%3DAndes%26aufirst%3DD.%26aulast%3DCraig%26aufirst%3DW.%2BA.%26atitle%3DAnimal%2520Model%2520Pharmacokinetics%2520and%2520Pharmacodynamics%253A%2520a%2520Critical%2520Review%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2002%26volume%3D19%26spage%3D261%26epage%3D268%26doi%3D10.1016%2FS0924-8579%2802%2900022-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomaras, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carifa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbonnet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidem, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. P.</span></span> <span> </span><span class="NLM_article-title">LpxC Inhibitors as New Antibacterial Agents and Tools for Studying Regulation of Lipid A Biosynthesis in Gram-Negative Pathogens</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e01551-14</span>, <span class="refDoi"> DOI: 10.1128/mBio.01551-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=10.1128%2FmBio.01551-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=25271285" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=e01551-14&author=A.+P.+Tomarasauthor=C.+J.+McPhersonauthor=M.+Kuhnauthor=A.+Carifaauthor=L.+Mullinsauthor=D.+Georgeauthor=C.+Desbonnetauthor=T.+M.+Eidemauthor=J.+I.+Montgomeryauthor=M.+F.+Brownauthor=U.+Reillyauthor=A.+A.+Millerauthor=J.+P.+O%E2%80%99Donnell&title=LpxC+Inhibitors+as+New+Antibacterial+Agents+and+Tools+for+Studying+Regulation+of+Lipid+A+Biosynthesis+in+Gram-Negative+Pathogens&doi=10.1128%2FmBio.01551-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FmBio.01551-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.01551-14%26sid%3Dliteratum%253Aachs%26aulast%3DTomaras%26aufirst%3DA.%2BP.%26aulast%3DMcPherson%26aufirst%3DC.%2BJ.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DCarifa%26aufirst%3DA.%26aulast%3DMullins%26aufirst%3DL.%26aulast%3DGeorge%26aufirst%3DD.%26aulast%3DDesbonnet%26aufirst%3DC.%26aulast%3DEidem%26aufirst%3DT.%2BM.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DReilly%26aufirst%3DU.%26aulast%3DMiller%26aufirst%3DA.%2BA.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26atitle%3DLpxC%2520Inhibitors%2520as%2520New%2520Antibacterial%2520Agents%2520and%2520Tools%2520for%2520Studying%2520Regulation%2520of%2520Lipid%2520A%2520Biosynthesis%2520in%2520Gram-Negative%2520Pathogens%26jtitle%3DmBio%26date%3D2014%26volume%3D5%26spage%3De01551-14%26doi%3D10.1128%2FmBio.01551-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapointe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madera, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yifru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjandra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rico, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojkovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uehara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellamacina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonn, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benton, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, Z. K.</span></span> <span> </span><span class="NLM_article-title">Application of Virtual Screening to the Identification of New LpxC Inhibitor Chemotypes, Oxazolidinone and Isoxazoline</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9360</span>– <span class="NLM_lpage">9370</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01287</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01287" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslaqtrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9360-9370&author=P.+S.+Leeauthor=G.+Lapointeauthor=A.+M.+Maderaauthor=R.+Simmonsauthor=W.+Xuauthor=A.+Yifruauthor=M.+Tjandraauthor=S.+Karurauthor=A.+Ricoauthor=K.+Thompsonauthor=J.+Bojkovicauthor=L.+Xieauthor=K.+Ueharaauthor=A.+Liuauthor=W.+Shuauthor=C.+Bellamacinaauthor=D.+McKenneyauthor=L.+Morrisauthor=G.+R.+Tonnauthor=C.+S.+Osborneauthor=B.+Bentonauthor=L.+McDowellauthor=J.+Fuauthor=Z.+K.+Sweeney&title=Application+of+Virtual+Screening+to+the+Identification+of+New+LpxC+Inhibitor+Chemotypes%2C+Oxazolidinone+and+Isoxazoline&doi=10.1021%2Facs.jmedchem.8b01287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Virtual Screening to the Identification of New LpxC Inhibitor Chemotypes, Oxazolidinone and Isoxazoline</span></div><div class="casAuthors">Lee, Patrick S.; Lapointe, Guillaume; Madera, Ann Marie; Simmons, Robert L.; Xu, Wenjian; Yifru, Aregahegn; Tjandra, Meiliana; Karur, Subramanian; Rico, Alice; Thompson, Katherine; Bojkovic, Jade; Xie, Lili; Uehara, Kyoko; Liu, Amy; Shu, Wei; Bellamacina, Cornelia; McKenney, David; Morris, Laura; Tonn, George R.; Osborne, Colin; Benton, Bret M.; McDowell, Laura; Fu, Jiping; Sweeney, Zachary K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9360-9370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report summarizes the identification and synthesis of novel LpxC inhibitors aided by computational methods that leveraged numerous crystal structures.  This effort led to the identification of oxazolidinone and isoxazoline inhibitors with potent in vitro activity against P. aeruginosa and other Gram-neg. bacteria.  Representative compd. 13f demonstrated efficacy against P. aeruginosa in a mouse neutropenic thigh infection model.  The antibacterial activity against K. pneumoniae could be potentiated by Gram-pos. antibiotics rifampicin (RIF) and vancomycin (VAN) in both in vitro and in vivo models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphGwLR2DR7VrVg90H21EOLACvtfcHk0ljmLnrp72Nt9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslaqtrvF&md5=73c59bf77a4a9ea5c993d7f935955307</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01287%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DP.%2BS.%26aulast%3DLapointe%26aufirst%3DG.%26aulast%3DMadera%26aufirst%3DA.%2BM.%26aulast%3DSimmons%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DYifru%26aufirst%3DA.%26aulast%3DTjandra%26aufirst%3DM.%26aulast%3DKarur%26aufirst%3DS.%26aulast%3DRico%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DK.%26aulast%3DBojkovic%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DUehara%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DShu%26aufirst%3DW.%26aulast%3DBellamacina%26aufirst%3DC.%26aulast%3DMcKenney%26aufirst%3DD.%26aulast%3DMorris%26aufirst%3DL.%26aulast%3DTonn%26aufirst%3DG.%2BR.%26aulast%3DOsborne%26aufirst%3DC.%2BS.%26aulast%3DBenton%26aufirst%3DB.%26aulast%3DMcDowell%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DSweeney%26aufirst%3DZ.%2BK.%26atitle%3DApplication%2520of%2520Virtual%2520Screening%2520to%2520the%2520Identification%2520of%2520New%2520LpxC%2520Inhibitor%2520Chemotypes%252C%2520Oxazolidinone%2520and%2520Isoxazoline%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9360%26epage%3D9370%26doi%3D10.1021%2Facs.jmedchem.8b01287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U1Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U1Y','PDB','3U1Y'); return false;">PDB: 3U1Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UHM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UHM','PDB','3UHM'); return false;">PDB: 3UHM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I46" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I46','PDB','6I46'); return false;">PDB: 6I46</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I49" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I49','PDB','6I49'); return false;">PDB: 6I49</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VWM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VWM','PDB','5VWM'); return false;">PDB: 5VWM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I4A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I4A','PDB','6I4A'); return false;">PDB: 6I4A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I47" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I47','PDB','6I47'); return false;">PDB: 6I47</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I48" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I48','PDB','6I48'); return false;">PDB: 6I48</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i73"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01604">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33163"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01604?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01604</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental details on methods, data collection and refinements statistics for the structures of compounds <b>8</b>, <b>10</b>, <b>12</b>, <b>17a</b>, and <b>18d</b>; MIC testing, log <i>D</i>, and solubility determination; MICs determined against PAO1 and PA2616 strains for compounds <b>7</b>–<b>20</b>; preparation of compounds <b>8</b>–<b>14</b>, <b>22</b>, and <b>41</b>–<b>60</b>; preparation of recombinant <i>Pseudomonas aeruginosa</i> (Pae) and <i>Escherichia coli</i> (Eco) LpxC proteins for X-ray and DSF; neutropenic murine thigh infection model against <i>Escherichia coli</i>, graphical representations and descriptive statistics; pharmacokinetics in the mouse; views of compounds <b>10</b>, <b>12</b>, and <b>18d</b> bound to Pae LpxC (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings(<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_001.csv">jm9b01604_si_001.csv (4.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01604/suppl_file/jm9b01604_si_002.pdf">jm9b01604_si_002.pdf (1.6 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>8</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I46">6I46</a>; <b>10</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I47">6I47</a>; <b>12</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I48">6I48</a>; <b>17a</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I49">6I49</a>; <b>18d</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I4A">6I4A</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01604&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01604%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-1%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01604" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992dfe6f6a1991","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
