
<html lang="en"     class="pb-page"  data-request-id="47bb06b2-5d39-4f09-add5-f5fbaa200fc3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.5b01631;requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2016.59.issue-2;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome" /></meta><meta name="dc.Creator" content="Yu  Zhou" /></meta><meta name="dc.Creator" content="Jing  Ma" /></meta><meta name="dc.Creator" content="Xingyu  Lin" /></meta><meta name="dc.Creator" content="Xi-Ping  Huang" /></meta><meta name="dc.Creator" content="Kaichun  Wu" /></meta><meta name="dc.Creator" content="Niu  Huang" /></meta><meta name="dc.Description" content="Here we employed structure-based ligand discovery techniques to explore a recently determined crystal structure of the 5-hydroxytryptamine 2B (5-HT2B) receptor. Ten compounds containing a novel che..." /></meta><meta name="Description" content="Here we employed structure-based ligand discovery techniques to explore a recently determined crystal structure of the 5-hydroxytryptamine 2B (5-HT2B) receptor. Ten compounds containing a novel che..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 7, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01631" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01631" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01631" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01631" /></link>
        
    
    

<title>Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01631" /></meta><meta property="og:title" content="Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0005.jpeg" /></meta><meta property="og:description" content="Here we employed structure-based ligand discovery techniques to explore a recently determined crystal structure of the 5-hydroxytryptamine 2B (5-HT2B) receptor. Ten compounds containing a novel chemical scaffold were identified; among them, seven molecules were active in cellular function assays with the most potent one exhibiting an IC50 value of 27.3 nM. We then systematically probed the binding characteristics of this scaffold by designing, synthesizing, and testing a series of structural modifications. The structure–activity relationship studies strongly support our predicted binding model. The binding profiling across a panel of 11 5-HT receptors indicated that these compounds are highly selective for the 5-HT2B receptor. Oral administration of compound 15 (30 mg/kg) produced significant attenuation of visceral hypersensitivity in a rat model of irritable bowel syndrome (IBS). We expect this novel scaffold will serve as the foundation for the development of 5-HT2B antagonists for the treatment of IBS." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01631"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01631">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01631&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01631&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01631&amp;href=/doi/10.1021/acs.jmedchem.5b01631" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 707-720</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01594" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01771" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Zhou">Yu Zhou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Ma">Jing Ma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xingyu++Lin">Xingyu Lin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xi-Ping++Huang">Xi-Ping Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kaichun++Wu">Kaichun Wu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niu++Huang">Niu Huang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">National Institute of Biological Sciences, Beijing, No. 7 Science Park Road, Zhongguancun Life Science Park, Beijing 102206, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmacology and Pharmaceutical Sciences, School of Medicine, Tsinghua University, Beijing 100084, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">State Key Laboratory of Cancer Biology and Xijing Hospital of Digestive Diseases, Fourth Military Medical University, 127 West Changle Road, Xi’an, Shaanxi Province 710032, China</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Pharmacology, The National Institute of Mental Health Psychoactive Drug Screening Program (NIMH PDSP), The University of North Carolina, Chapel Hill, North Carolina 27759, United States</span></div><div class="corresp-info"><strong>*</strong>Tel, 86-10-80720645; fax, 86-10-80720813; e-mail, <a href="/cdn-cgi/l/email-protection#4f273a2e212821263a0f21262d3c612e2c612c21"><span class="__cf_email__" data-cfemail="e8809d89868f86819da886818a9bc6898bc68b86">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Tel, 86-29-84771502; fax, 86-29-82539041; e-mail, <a href="/cdn-cgi/l/email-protection#cba0aaa2a8bcbe8bada6a6bee5aeafbee5a8a5"><span class="__cf_email__" data-cfemail="6209030b01151722040f0f174c0706174c010c">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01631&amp;href=/doi/10.1021%2Facs.jmedchem.5b01631" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 707–720</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 23, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 October 2015</li><li><span class="item_label"><b>Published</b> online</span>7 January 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 January 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01631" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01631</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D707%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYu%2BZhou%252C%2BJing%2BMa%252C%2BXingyu%2BLin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D2%26contentID%3Dacs.jmedchem.5b01631%26title%3DStructure-Based%2BDiscovery%2Bof%2BNovel%2Band%2BSelective%2B5-Hydroxytryptamine%2B2B%2BReceptor%2BAntagonists%2Bfor%2Bthe%2BTreatment%2Bof%2BIrritable%2BBowel%2BSyndrome%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D720%26publicationDate%3DJanuary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01631"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2009</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">23</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01631" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Discovery of Novel and Selective 5-Hydroxytryptamine 2B Receptor Antagonists for the Treatment of Irritable Bowel Syndrome&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Xingyu&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Xi-Ping&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Kaichun&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Niu&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;707-720&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01631&quot;},&quot;abstract&quot;:&quot;Here we employed structure-based ligand discovery techniques to explore a recently determined crystal structure of the 5-hydroxytryptamine 2B (5-HT2B) receptor. Ten compounds containing a novel chemical scaffold were identified; among them, seven molecules were active in cellular function assays with the most potent one exhibiting an IC50 value of 27.3 nM. We then systematically probed the binding characteristics of this scaffold by designing, synthesizing, and testing a series of structural modifications. The structure–activity relationship studies strongly support our predicted binding model. The binding profiling across a panel of 11 5-HT receptors indicated that these compounds are highly selective for the 5-HT2B receptor. Oral administration of compound 15 (30 mg/kg) produced significant attenuation of visceral hypersensitivity in a rat model of irritable bowel syndrome (IBS). We expect this novel scaffold will serve as the foundation for the development of 5-HT2B antagonists for the treatment of IBS.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01631&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01631" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01631&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01631" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01631&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01631" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01631&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01631&amp;href=/doi/10.1021/acs.jmedchem.5b01631" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01631" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01631" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01631%26sid%3Dliteratum%253Aachs%26pmid%3D26700945%26genre%3Darticle%26aulast%3DZhou%26date%3D2016%26atitle%3DStructure-Based%2BDiscovery%2Bof%2BNovel%2Band%2BSelective%2B5-Hydroxytryptamine%2B2B%2BReceptor%2BAntagonists%2Bfor%2Bthe%2BTreatment%2Bof%2BIrritable%2BBowel%2BSyndrome%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D2%26spage%3D707%26epage%3D720%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/jmcmar.2016.59.issue-2/20160128/jmcmar.2016.59.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Here we employed structure-based ligand discovery techniques to explore a recently determined crystal structure of the 5-hydroxytryptamine 2B (5-HT<sub>2B</sub>) receptor. Ten compounds containing a novel chemical scaffold were identified; among them, seven molecules were active in cellular function assays with the most potent one exhibiting an IC<sub>50</sub> value of 27.3 nM. We then systematically probed the binding characteristics of this scaffold by designing, synthesizing, and testing a series of structural modifications. The structure–activity relationship studies strongly support our predicted binding model. The binding profiling across a panel of 11 5-HT receptors indicated that these compounds are highly selective for the 5-HT<sub>2B</sub> receptor. Oral administration of compound <b>15</b> (30 mg/kg) produced significant attenuation of visceral hypersensitivity in a rat model of irritable bowel syndrome (IBS). We expect this novel scaffold will serve as the foundation for the development of 5-HT<sub>2B</sub> antagonists for the treatment of IBS.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13865" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13865" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">G-protein coupled receptors (GPCRs) represent the most prominent pharmaceutical targets in drug discovery, and over 40% of marketed drugs act through them.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> GPCR drugs have typically been developed based on the traditional medicinal chemistry approach, which has restricted the diversity of the chemical scaffolds considered as candidate drugs. Recent progress in the determination of GPCR structures has opened up great opportunities for structure-based discovery efforts seeking novel GPCR ligands. Docking screens against these crystal structures have been successfully applied in identification of novel potent ligands.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7 ref8">(2-8)</a></div><div class="NLM_p">The 5-HT<sub>2B</sub> receptor is one of the recently determined GPCR structures.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> It belongs to 5-HT<sub>2</sub> receptor family, which includes three subtypes: 5-HT<sub>2A</sub>, 5-HT<sub>2B</sub>, and 5-HT<sub>2C</sub>. Although both the 5-HT<sub>2A</sub> and 5HT<sub>2C</sub> receptors have been widely studied as therapeutic targets, drug discovery efforts focused on the 5-HT<sub>2B</sub> receptor have been relatively limited. 5-HT<sub>2B</sub> agonism has been regarded as a side effect for drug molecules, as its activation is related to cardiac hypertrophy and pulmonary hypertension.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> More recent studies have focused on the possible application of 5-HT<sub>2B</sub> antagonists.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Several compounds have been advanced to clinical trials in several disease areas, including migraine disorder,<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> irritable bowel syndrome (IBS),<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> and pulmonary arterial hypertension (PAH).<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a></div><div class="NLM_p">IBS is a common functional gastrointestinal (GI) disorder worldwide that is characterized by abdominal pain and alterations in bowel habits. Despite these impacts, the pathogenic mechanism of this disease remains incompletely understood and seems to be multifactorial.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Visceral hypersensitivity is now widely accepted as one of the core characteristics, and this probably plays a major role in the development of disease symptoms.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Currently, there is no effective cure for IBS. Traditional management has mainly focused on the relief of symptoms. The efficacy of these management approaches has been limited, and the side effects of various drugs used to relieve symptoms have been problematic. For example, the 5-HT<sub>3</sub> antagonist alosetron was approved by the US FDA in 2000 and is used to treat the diarrhea-predominate form of IBS.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Unfortunately, severe adverse GI reactions were reported soon after its introduction to the market. In 2015, the US FDA approved two new drugs for the diarrhea-predominate form of IBS. The effects of rifaximin, an antibiobic derived from refampin, are thought to be related to changes in the bacterial content in the GI tract. Eluxadoline, a mixed opioid receptor ligand, acts on the nervous system to reduce bowel contractions. However, the safety of these two drugs is still of great concern given reported side effects that include constipation, nausea, spasm, and increased alanine aminotransferase levels.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p">Recent studies have strongly suggested that the 5-HT<sub>2B</sub> receptor plays an important role in the pathogenesis of IBS; it is therefore a promising target for the development of novel therapies. The 5-HT<sub>2B</sub> receptor is widely expressed throughout the human intestinal tract.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> It may function in regulating the contraction of smooth muscle to mediate GI motility and visceral perception. The activation of the 5-HT<sub>2B</sub> receptor was shown to increase the proliferation of interstitial cells of Cajal through protein kinase Cγ, a process that may lead to motility disorder and visceral hypersensitivity in the colon.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In addition, increased 5-HT<sub>2B</sub> receptor expression has been reported for rat models of diarrhea-predominant IBS.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Selective antagonism of the 5-HT<sub>2B</sub> receptor in IBS rats has been shown to produce significant attenuation of visceral hypersensitivity and improvement with defecation.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26-29)</a></div><div class="NLM_p last">Currently, only a few selective antagonists of the 5-HT<sub>2B</sub> receptor have been reported, due to the high degree of homology with its closely related protein family members.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Therefore, the discovery of novel and selective 5-HT<sub>2B</sub> antagonists would be of great interest to further explore the physiological function and therapeutic application of the 5-HT<sub>2B</sub> receptor. In the present study, we used structure-based virtual screening to explore the opportunity presented by a recently published crystal structure of the 5-HT<sub>2B</sub> receptor to identify novel antagonists against this protein. One class of compounds with a novel structural scaffold was enriched among the top of the scoring list. Compounds from this class were validated in cellular functional assays and binding profiling assays. We further performed structure–activity relationship (SAR) studies to assess the predicted binding pose of this scaffold, which led to the selection of a promising candidate with high binding affinity and high selectivity for the 5-HT<sub>2B</sub> receptor. In vivo studies of this drug candidate suggested that this class of 5-HT<sub>2B</sub> antagonists can serve as a promising starting point for the development of new drugs to treat IBS.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42381" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42381" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Binding-Site Refinement of the 5-HT<sub>2B</sub> Crystal Structure</h3><div class="NLM_p">GPCR undergoes conformational changes to accommodate different ligands, and these conformational changes are especially pronounced when making comparisons between agonist-bound structures and antagonist-bound structures. The only currently available 5-HT<sub>2B</sub> crystal structure is bound with a biased agonist (ergotamine), and it has been suggested that this structure exhibits conformational characteristics of both the active and the inactive states (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1A">Figure S1A</a>, Supporting Information).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compared with other inactive aminergic GPCR structures, the conserved anchor residue Asp<sup>3.32</sup>, which is critical for ligand binding, adopts a different orientation in the 5-HT<sub>2B</sub> structure (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1B">Figure S1B</a>). Accordingly, rigid-receptor docking against this conformation might be of limited utility in identifying efficacious antagonists.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Thus, instead of docking to the crystal conformation directly, we deliberately modified the structure to incorporate the induced-fit effects associated with 5-HT<sub>2B</sub> antagonists.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> To obtain a reliable induced-fit structural model, we selected two promiscuous GPCR ligands: doxepin and carvedilol (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). Both of them exhibit potent 5-HT<sub>2B</sub> antagonist activity;<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> they also bind to other GPCRs and have corresponding crystal complex structures available (with histamine H<sub>1</sub> and β<sub>1</sub>-adrenergic receptors, respectively).<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding site refinement of the 5-HT<sub>2B</sub> crystal structure. (A) Chemical structures and biological activities of doxepin and carvedilol. (B) Comparison of the doxepin induced-fit model (cyan) with the 5-HT<sub>2B</sub> crystal structure (tan, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4">4IB4</a>). (C) Solvent property analysis of the 5-HT<sub>2B</sub> binding pocket. Predicted water clusters are shown as spheres. The volume of spheres represents their occupancy in the simulation, and the color represents their free energy. Blue indicates that the water molecule is energy favorable, while red indicates it is energy unfavorable. (D) The enrichment profile of the doxepin induced-fit model. The percentage of ligands found (<i>y</i>-axis) plotted as a function of the percentage of the docking ranked database (<i>x</i>-axis in logarithmic scale).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B illustrates the doxepin induced-fit model, in which doxepin maintains the dominant interactions that form in the histamine H<sub>1</sub> receptor (<a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>, Supporting Information). The amine moiety forms an anchor salt bridge with Asp<sup>3.32</sup>. The tricyclic rings of doxepin sit deeply in the hydrophobic binding pocket and are surrounded by the following highly conserved residues: Ile<sup>3.40</sup>, Trp<sup>6.48</sup>, Phe<sup>6.51</sup>, and Phe<sup>6.52</sup>. The oxygen atom of doxepin (E isomer) forms a hydrogen bond with Thr<sup>3.37</sup>, which is consistent with that in the histamine H<sub>1</sub>–doxepin crystal complex structure. This result significantly bolstered our confidence about the properties of our induced-fit model. Compared with the initial crystal configuration, we found that three residues experience significant conformational changes in the doxepin induced-fit conformation. Asp<sup>3.32</sup> orientates toward the binding pocket, forming a salt bridge with the ligand, an orientation consistent with that of other inactive monoamine GPCRs. Yet both Met<sup>5.39</sup> and Ile<sup>3.40</sup> rotate away from the ligand, resulting in a larger binding pocket. Notably, the conformation of Ile<sup>3.40</sup> is switched back to the inactive state as shown in other inactive aminergic GPCR structures. We observed similar (but less pronounced) conformational changes in the carvedilol induced-fit model (<a class="ref internalNav" href="#notes-1" aria-label="Figure S3A">Figure S3A</a>, Supporting Information).</div><div class="NLM_p">We further analyzed the binding-site solvent properties of the 5-HT<sub>2B</sub> receptor. The displacement of water molecules from the binding pocket is a key component of ligand binding. Here we calculated the thermodynamic properties of binding-site water molecules in the 5-HT<sub>2B</sub> receptor using the Solvent Property Analysis (SPA) program.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The predictions for occupancy and free energy of the binding-site waters enabled us to generally summarize the potential interaction patterns of 5-HT<sub>2B</sub> ligands (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C). First, a cluster of highly energy-favorable waters surround the negatively charged residue Asp<sup>3.32</sup>, the displacement of which would result in a large enthalpy penalty. Thus, a positively charged amine group is usually required here for the ligand to form a salt bridge interaction with Asp<sup>3.32</sup> to compensate for the energy penalty. Second, three less energy-favorable waters form hydrogen bond interactions with Asn<sup>6.55</sup>, implying a potential hydrogen bond interaction with the ligand in this position. We also identified a group of energy-unfavorable waters bound deeply within the binding pocket that were restricted by the aromatic rings of nearby residues. A nonpolar planar group here would generate significant potency from the entropy gain of the water displacement. The distribution of these binding-site waters represents the chemical composition of an ideal ligand. On the basis of the above analysis, we derived three structural filters for the screening of 5-HT<sub>2B</sub> antagonists: (a) a salt bridge interaction must be formed between the positively charged amine group of the ligand and residue Asp<sup>3.32</sup>; (b) the region of unfavorable waters should be occupied by a hydrophobic group of the ligand (here we define the occupation as at least one carbon atom having a distance <3 Å from the oxygen atom of each water molecule); (c) considering the overall hydrophobic nature of the binding site, there should not be many polar atoms that do not form a hydrogen bond interaction. Thus, a ligand with three or more polar atoms buried in the pocket should form at least one hydrogen bond with receptor residues.</div><div class="NLM_p last">We next estimated the ability of our induced-fit models to enrich known antagonists of the 5-HT<sub>2B</sub> receptor from a background of physicochemically similar, but structurally dissimilar, decoys. The derived structural filters were also applied. The docking results suggested that both of the induced-fit models provide better enrichment performance than does the original crystal structure; this was especially the case when the models were combined with structural filters (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>D and <a class="ref internalNav" href="#notes-1" aria-label="Figure S3B">Figure S3B</a>). The EF1 values (enrichment factor at 1% of the ranked database) were 2.4, 8.2, and 5.8 for the crystal structure, the doxepin model, and the carvedilol model, respectively. Structural filtering further improved the results, yielding EF1 values of 4.6, 14.9, and 10.5, respectively. The doxepin induced-fit model combined with structural filtering exhibited the best performance. Thus, we finally selected the doxepin induced-fit model for use in further virtual screening efforts.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of a Novel Scaffold in Prospective Virtual Screening</h3><div class="NLM_p">A structure-based hierarchical virtual screening strategy<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a> was applied to identify novel 5-HT<sub>2B</sub> antagonists. To balance efficiency and accuracy, this protocol combines molecular docking-based high throughput screening and minimization-based MM-GB/SA rescoring (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). We initially docked our in-house compound library that contains 100 413 diverse drug-like compounds. More than 300 000 binding conformations were generated, and the 10 000 top-ranked poses were further subjected to minimization-based MM-GB/SA refining and rescoring. We then saved the 1000 top-ranked nonredundant compounds for structural analysis; only 253 of the hits passed the structural filters. With the aim of identifying a novel 5-HT<sub>2B</sub> ligand scaffold, we subjected these surviving molecules to the Similarity Ensemble Approach<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (SEA, <a href="http://sea.bkslab.org" class="extLink">http://sea.bkslab.org</a>) server in an effort to avoid the selection of compounds that are structurally similar to known GPCR ligands. Finally, we obtained 169 molecules of interest (<a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>, Supporting Information). Among these, one scaffold in particular stood out, as ten molecules with this scaffold were enriched in the hit list, and all ten of these molecules adopted a similar binding mode (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A and <a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a>, Supporting Information).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Flowchart of the structure-based hierarchical virtual screening strategy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Experimental validation of the identified compounds. (A) Scaffold of the identified compounds. (B) Inhibitory activity on the 5-HT<sub>2B</sub> receptor at 10 μM and 1 μM concentrations. (C) Chemical structure of compound <b>9</b>. (D) The dose-dependent inhibitory response curve of compound <b>9</b> on the 5-HT<sub>2B</sub> receptor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Cellular functional assays were performed to validate the antagonist activity of the ten selected molecules. Encouragingly, seven compounds exhibited >50% inhibition at a 10 μM concentration and five of these seven exhibited >50% inhibition at a 1 μM concentration (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B), results that clearly suggested that a common mode of action was shared by this series of compounds. We further determined the dose–response inhibition activity of these five compounds (<a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a>). The most potent one, compound <b>9</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C), exhibits an IC<sub>50</sub> value of 27.3 nM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Assessment of the Predicted Binding Mode</h3><div class="NLM_p">These compounds represent a novel class of 5-HT<sub>2B</sub> antagonists. The triazine ring in trizazspiro is basic (with experimental p<i>K</i><sub>a</sub> of 10–11).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Thus, it is protonated in solution and forms the salt bridge interaction with conserved Asp<sup>3.32</sup> through N<sup>1</sup> and the nitrogen atom on the 2-amine group (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). The nitrogen atom on the 4-amine group forms a hydrogen bond with residue Asn<sup>6.55</sup>. The saturated cyclohexene group is located in the entrance of the binding pocket and forms hydrophobic contacts with extracellular loop II (ECL II). The phenyl ring is deeply buried into the inner part of the hydrophobic pocket and is surrounded by the aromatic side chains of residues Trp<sup>6.48</sup>, Phe<sup>6.51</sup>, and Phe<sup>6.52</sup>. These interactions are quite consistent with the above solvent property analysis. Specifically, for compound <b>9</b>, there is an additional hydrogen bond interaction with residue Thr<sup>3.37</sup> that may further contribute to its strong binding. Compared with the crystal structure of 5-HT<sub>2B</sub>–ergotamine complex, our compounds occupy the orthosteric binding pocket, without completely overlapping with the biased agonist ergotamine binding site (<a class="ref internalNav" href="#notes-1" aria-label="Figure S4">Figure S4</a>, Supporting Information).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Predicted binding mode of compound <b>9</b>. (B) Correlation between the experimental pIC<sub>50</sub> and the computed Δ<i>E</i> based on the predicted binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of this predicted binding mode and potential chemical modifications, we next probed the binding characteristics by envisioning and testing a series of structural analogues (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). To confirm the salt bridge interaction, we synthesized compound <b>11</b>, in which a methyl group was added to the nitrogen atom on the 2-amine group. As we expected, this modification destabilized the salt bridge with residue Asp<sup>3.32</sup> and led to a remarkable decrease in potency. Further, compound <b>12</b>, with a cyclized N<sup>1</sup> nitrogen atom, could not form the salt bridge interaction and lost all activity. Compounds <b>13</b> and <b>14</b> were designed to probe the hydrogen bond to residue Asn<sup>6.55</sup>. A single methyl group added on the nitrogen atom of 4-amine group preserves the hydrogen bond. Therefore, compound <b>13</b> exhibited comparable inhibitory activity with compound <b>9</b>, with an IC<sub>50</sub> value of 49.4 nM. However, the substitution of two methyl groups totally abolished the hydrogen bond interaction, resulting in a >10-fold decrease in the potency of compound <b>14</b>. The saturated cyclohexane ring forms hydrophobic contacts with residues on ECL II. The effects of subtle variations to this region were also explored. Replacing the cyclohexane subunit with a smaller germinal dimethyl or saturated cyclopentane group yielded, respectively, 10-fold and 4-fold decreased potency (compounds <b>1</b>, <b>5</b>, <b>15</b>, and <b>16</b>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Validation of the Predicted Binding Mode<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0003.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">N.D. means not determined.</p></div></div><div></div></div><div class="NLM_p">We then focused on the R-substituents on the phenyl group, which sits deeply in the hydrophobic binding pocket and makes extensive contacts with nearby residues. A series of structural analogues were tested (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Compound <b>17</b>, without any modifications on the phenyl ring group, exhibited no inhibition of the 5-HT<sub>2B</sub> receptor, indicating that the hydrophobic interaction in the binding pocket might be essential for the antagonist activity. The R<sup>2</sup> position is restricted by the ligand itself or by nearby residues in the protein. Modifications on this site contribute little to binding. Thus, compound <b>18</b> and <b>19</b> yielded minimal increase in potency. The R<sup>3</sup> position is buried deeply into the pocket, where bulky groups are tolerable. We surmised that nonpolar substitutions on this site would introduce favorable hydrophobic interactions. As expected, the potency was significantly enhanced by various monosubstitutions at the R<sup>3</sup> position. For example, compounds <b>15</b> and <b>16</b>, modified by a bromine atom or methoxy group, respectively, exhibited potent inhibition; the IC<sub>50</sub> values for these two compounds were 33.4 nM and 280 nM. Considering the hydrolytic lability of the ester group in compound <b>9</b>, we also synthesized and tested its hydrolysis product, compound <b>20</b>. However, no inhibitory activity was detected. This may be because the negatively charged group is unfavorable for binding in the hydrophobic pocket surrounded by aromatic rings, owing to a large desolvation penalty. This led us to synthesize compounds <b>21</b> and <b>22</b>, with the replacement of the ester group with, respectively, a carbonyl group and an amide group to pursue improved stability; their IC<sub>50</sub> values were 35.8 nM and 104.0 nM. Compound <b>23</b>, with an additional methyl group added to compound <b>9</b>, exhibits a slightly stronger inhibition, with an IC<sub>50</sub> value of 18.1 nM. Compounds <b>24</b> and <b>25</b>, substituted with relatively bulky groups, exhibited no activity or very low activity, consistent with a narrow and deep cleft in this position in the receptor binding site (<a class="ref internalNav" href="#notes-1" aria-label="Figure S5">Figure S5</a>, Supporting Information). As for the R<sup>4</sup> position, the space is relatively small (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). Only small substitutions such as methyl groups or a chlorine atom (compound <b>26</b> and <b>27</b>) are tolerable. Larger groups here, such as that in compound <b>28,</b> introduce steric clashes and diminish binding.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the R Substituent<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0004.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">inhibition (%) at 1 μM</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="char" char="±">–11.1 ± 3.2</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="center">2-Me</td><td class="colsep0 rowsep0" align="char" char="±">39.9 ± 3.4</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="center">2-OMe</td><td class="colsep0 rowsep0" align="char" char="±">20.7 ± 3.6</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="center">3-Br</td><td class="colsep0 rowsep0" align="char" char="±">97.4 ± 2.2</td><td class="colsep0 rowsep0" align="char" char=".">33.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="center">3-OMe</td><td class="colsep0 rowsep0" align="char" char="±">84.6 ± 1.2</td><td class="colsep0 rowsep0" align="char" char=".">280.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="center">3-COOEt</td><td class="colsep0 rowsep0" align="char" char="±">107.2 ± 0.6</td><td class="colsep0 rowsep0" align="char" char=".">27.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="center">3-COOH</td><td class="colsep0 rowsep0" align="char" char="±">4.4 ± 3.3</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="center">3-CO-n-Pr</td><td class="colsep0 rowsep0" align="char" char="±">106.0 ± 0.8</td><td class="colsep0 rowsep0" align="char" char=".">35.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="center">3-CONHEt</td><td class="colsep0 rowsep0" align="char" char="±">95.5 ± 4.8</td><td class="colsep0 rowsep0" align="char" char=".">104.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="center">3-COO-n-Pr</td><td class="colsep0 rowsep0" align="char" char="±">110.5 ± 2.7</td><td class="colsep0 rowsep0" align="char" char=".">18.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="center">3-CO-c-Pr</td><td class="colsep0 rowsep0" align="char" char="±">4.0 ± 1.1</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="center">3-CON(Et)<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">40.7 ± 1.2</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="center">4-Me</td><td class="colsep0 rowsep0" align="char" char="±">50.2 ± 8.0</td><td class="colsep0 rowsep0" align="char" char=".">485.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="center">4-Cl</td><td class="colsep0 rowsep0" align="char" char="±">57.1 ± 5.4</td><td class="colsep0 rowsep0" align="char" char=".">531.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="center">4-N(Me)<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">17.6 ± 0.4</td><td class="colsep0 rowsep0" align="char" char=".">N.D.</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">N.D. means not determined.</p></div></div></div><div class="NLM_p last">The above SAR results strongly support our predicted binding mode. On the basis of this binding mode, we further calculated the binding energies of 14 compounds with experimentally determined IC<sub>50</sub> values, using the MM-GB/SA rescoring method. In our previous work,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> the target protein is usually kept rigid during the minimization process of the ligand–protein complex. Here we also attempted to include side-chain flexibility during the rescoring stage by simply minimizing side-chain conformations along with the ligand.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> There were good correlations between the experimental pIC<sub>50</sub> and the computed Δ<i>E</i> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). This further supports the assertion that our predicted binding mode is reasonable. Of particular note was the improved correlation (<i>R</i><sup>2</sup> = 0.79) between the experimental pIC<sub>50</sub> and the computed Δ<i>E</i> that resulted from the flexible treatment of the side chains of the binding-site residues, emphasizing the importance of incorporating protein flexibility into ligand binding energy calculation.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Binding Selectivity over Other 5-HT Receptors</h3><div class="NLM_p">We performed binding profiling tests for five compounds on a panel of 11 5-HT receptors. The results suggested that this class of compounds are relatively specific to the 5-HT<sub>2B</sub> receptor (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). They strongly bound to the 5-HT<sub>2B</sub> receptor, with <i>K</i><sub>i</sub> values ranging from 6.2 nM to 38.5 nM, and bound only weakly to the other six 5-HT receptors (5-HT<sub>1A</sub>, 5-HT<sub>1B</sub>, 5-HT<sub>1D</sub>, 5-HT<sub>5</sub>, 5-HT<sub>6</sub>, and 5-HT<sub>7</sub>); they did not bind to 5-HT<sub>1E</sub> and 5HT<sub>3</sub>. Even for 5-HT<sub>2A</sub> and 5-HT<sub>2C</sub>, which are the two most highly related family members of 5-HT<sub>2B</sub>, these compounds also exhibited reasonably good selectivity. For example, compound <b>15</b>, bound to the 5-HT<sub>2B</sub>, 5-HT<sub>2A</sub>, and 5-HT<sub>2C</sub> with <i>K</i><sub>i</sub> values of 6.2, 211, and 156 nM, respectively.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. 5-HTRs Binding Profile Results (<i>K</i><sub>i</sub>, nM)</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>1A</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>1B</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>1D</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>1E</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>2A</sub></th><th class="colsep0 rowsep0" align="center" char="."><b>5</b>-HT<sub>2B</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub><b>2C</b></sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>3</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>5A</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>6</sub></th><th class="colsep0 rowsep0" align="center" char=".">5-HT<sub>7A</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">660</td><td class="colsep0 rowsep0" align="char" char=".">1531</td><td class="colsep0 rowsep0" align="char" char=".">2655</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">783</td><td class="colsep0 rowsep0" align="char" char=".">20.5</td><td class="colsep0 rowsep0" align="char" char=".">265</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">389</td><td class="colsep0 rowsep0" align="char" char=".">696</td><td class="colsep0 rowsep0" align="char" char=".">2083</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">320</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">2904</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">211</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">156</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">227</td><td class="colsep0 rowsep0" align="char" char=".">3031</td><td class="colsep0 rowsep0" align="char" char=".">401</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">714</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">8590</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">174</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">851</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">457</td><td class="colsep0 rowsep0" align="char" char=".">3587</td><td class="colsep0 rowsep0" align="char" char=".">472</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">6764</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">1313</td><td class="colsep0 rowsep0" align="char" char=".">27.5</td><td class="colsep0 rowsep0" align="char" char=".">1897</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">994</td><td class="colsep0 rowsep0" align="char" char=".">5909</td><td class="colsep0 rowsep0" align="char" char=".">427</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">116</td><td class="colsep0 rowsep0" align="char" char=".">9653</td><td class="colsep0 rowsep0" align="char" char=".">3830</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">1541</td><td class="colsep0 rowsep0" align="char" char=".">38.5</td><td class="colsep0 rowsep0" align="char" char=".">1201</td><td class="colsep0 rowsep0" align="char" char="."> >10000</td><td class="colsep0 rowsep0" align="char" char=".">595</td><td class="colsep0 rowsep0" align="char" char=".">4777</td><td class="colsep0 rowsep0" align="char" char=".">516</td></tr></tbody></table></div></div><div class="NLM_p">Currently, only crystal structures of 5-HT<sub>2B</sub> and 5-HT<sub>1B</sub> receptors are determined in the 5HT receptor family.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Structural comparison suggests that most of their binding-site residues are conserved, with the exception of residue 6.55 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). Substitution of Asn<sup>6.55</sup> for Ser<sup>6.55</sup> in the 5-HT<sub>1B</sub> receptor abolishes the hydrogen bond interaction with our identified compounds, which should result in decreased binding affinity. On the basis of the sequence alignment of the 5-HTRs, three additional residues located in the ECL II region of 5-HT<sub>2B</sub> are unique to the 5-HT<sub>2B</sub> receptor (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). These three additional residues form a helix turn and elevate the ECL II to an upper position, which facilitate the formation of an adjacent binding site in 5-HT<sub>2B</sub> receptor, consisting of hydrophobic residues Trp<sup>3.28</sup>, Leu<sup>3.29</sup>, Val<sup>7.39</sup>, Val208, and Leu209 in ECL II. This hydrophobic site may accommodate the cyclohexene group and contribute to the selectivity of this type of compounds (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). For the 5-HT<sub>1B</sub> receptor, only two additional residues are located in the ECL II region and no helix turn is formed. Thus, the cyclohexene group of our compounds may not be accommodated into the reduced pocket created by this short loop region. This supposition is consistent with what is known about other aminergic GPCRs with solved crystal structures. The histamine H<sub>1</sub> and M<sub>2</sub> muscarinic acetylcholine receptors with three additional residues located in the ECL II region, exhibit similar conformations to the 5-HT<sub>2B</sub> receptor. For the β<sub>2</sub>-adrenergic and dopamine D<sub>3</sub> receptors, however, only one or no residue located in this region, results in a closed conformation for ECL II and a reduced binding pocket (<a class="ref internalNav" href="#notes-1" aria-label="Figure S6">Figure S6</a>, Supporting Information).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Potential ligand binding selectivity mechanism of the 5-HT<sub>2B</sub> receptor. (A) Crystal structure comparsion between the 5-HT<sub>2B</sub> (green, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4">4IB4</a>) and 5-HT<sub>1B</sub> (cyan, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IAR">4IAR</a>) receptors. (B) Sequence alignment of the ECL II and TM6 regions among 5-HTRs and other GPCRs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Inhibition of Visceral Hypersensitivity in an IBS Rat Model</h3><div class="NLM_p">We selected one of the most potent compounds from the in vitro binding and functional assays, compound <b>15</b>, and evaluated its in vivo therapeutic efficiency in an IBS rat model. Compared to its 5-HT<sub>2B</sub> receptor antagonist activity, compound <b>15</b> exhibited relatively weak inhibitory activity against the hERG channel (IC<sub>50</sub> > 1 μM, <a class="ref internalNav" href="#notes-1" aria-label="Figure S7">Figure S7</a>, Supporting Information), and acute oral toxicity evaluation in rat showed no toxic response at a dose of 500 mg/kg. The visceral sensitivity of rats was evaluated by abdominal withdrawal reflex (AWR).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> As shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, the recorded pain thresholds significantly decreased in IBS rats compared with control rats (score 3:32 mmHg vs 57 mmHg, <i>P</i> < 0.001; score 4:57 mmHg vs 82 mmHg, <i>P</i> < 0.001). However, oral administration of compound <b>15</b> at 30 mg/kg produced a significant attenuation in visceral hypersensitivity (68% inhibition for score 3, <i>P</i> < 0.001 and 63% inhibition for score 4, <i>P</i> < 0.01). The mRNA expression level of the 5-HT<sub>2B</sub> receptor was relatively higher for the IBS rats, in both distal and proximal colons (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C), further indicating that blocking the 5-HT<sub>2B</sub> receptor may contribute to the inhibition of visceral hypersensitivity in IBS rats.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of visceral hypersensitivity in IBS rat model. (A and B) Therapeutic effects of compound <b>15</b> in IBS rats. The pain threshold decreased in response to CRD (at score 3 and score 4). (C) Overexpression of mRNA for the 5-HT<sub>2B</sub> receptor in IBS rat colons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32027" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32027" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The great progress in the determination GPCR structures has inspired a surge in structure-based ligand discovery efforts. Previous docking studies have indicated the functional fidelity of the docking screen hits to the conformation of the GPCR targets. The activity of the hits has recapitulated the activity of the cocrystallized ligands.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The only currently available 5-HT<sub>2B</sub> structure is bound with a biased agonist; it is thought to exhibit conformational characteristics of both its active and inactive states. Our induced-fit models have incorporated the knowledge of antagonist bound induced-fit effects and yielded structures with inactive-like conformations as in other inactive aminergic GPCR structures (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B, <a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>). Therefore, on this basis, we propose that our induced-fit models are likely more reasonable for 5-HT<sub>2B</sub> antagonist screening than using the crystal structure bound with a biased agonist. The retrospective docking screens of known 5-HT<sub>2B</sub> antagonists indicated improved enrichment performance using our induced-fit models. Actually, the prospective docking screen against the crystal structure could not find any of our identified 5-HT<sub>2B</sub> antagonists in the top 1000 compounds of scoring list (e.g., the rank of compound <b>9</b> is 3538); therefore, the optimization of the crystal conformation facilitated the successful identification of novel 5-HT<sub>2B</sub> antagonists. Besides, we also tested the agonist activity for several of our compounds. None of them increased or decreased the basal-level activity of 5-HT<sub>2B</sub> receptor at 1 μM. Thus, our compounds shall be pure antagonist, without possessing partial agonist or inverse agonist activity.</div><div class="NLM_p">Further, the binding site also experiences subtle conformation changes when binding with different antagonists. Our MM-GB/SA calculation treated the binding site with flexible side-chain conformations and yielded improved correlation between the experimental pIC<sub>50</sub> and computed Δ<i>E</i> compared with the rigid-receptor approximation, a result that emphasized the importance to incorporate binding-site flexibility into binding energy calculations.</div><div class="NLM_p last">As off-target interactions are responsible for the majority of adverse drug effects and failures in clinical trials, the identification of highly selective ligands is a critical aspect in modern drug discovery. Currently, very few 5-HTR ligands are subtype-selective, and the development of novel 5-HT<sub>2B</sub> antagonists with better specificity is highly desirable. In this study, we identified a series of potent 5-HT<sub>2B</sub> antagonists. These compounds represent a novel scaffold with distinct structural characteristics. Instead of containing the classical tertiary amine group, they utilize a protonated triazine ring to form a salt bridge interaction with the carboxylate group of conserved Asp<sup>3.32</sup> and may also form additional hydrogen bonds with the Asn<sup>6.55</sup> residue. Encouragingly, these compounds exhibited high selectivity for the 5-HT<sub>2B</sub> receptor over other 5HTRs. In vivo assessment has further indicated that compound <b>15</b> could produce significant attenuation of visceral hypersensitivity in IBS rat models. Thus, this scaffold represents a new avenue for the development of novel and selective 5-HT<sub>2B</sub> antagonists for the treatment of 5-HT<sub>2B</sub> receptor-related disease.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06555" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06555" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Binding Site Refinement and Analysis</h3><div class="NLM_p">Crystal structures of the histamine H<sub>1</sub>–doxepin complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZE">3RZE</a>) and the β<sub>1</sub>-carvedilol complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AMJ">4AMJ</a>) were utilized to generate the initial binding modes of doxepin and carvedilol to the 5-HT<sub>2B</sub> receptor. Considering the homology of the transmembrane regions, we assumed that both ligands would adopt a binding mode in the 5-HT<sub>2B</sub> receptor similar to that in other GPCRs. Thus, we superimposed the ligand-bound GPCR structures on the 5-HT<sub>2B</sub> structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4">4IB4</a>) and extracted the coordinates of the ligands as their initial binding poses. Subsequently, the conformations of the binding-site residues in the 5-HT<sub>2B</sub> receptor were sampled along with the placed ligand, using the Protein Local Optimization Program (PLOP).<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43-45)</a> The ligand was minimized first, followed by side-chain sampling of the binding site residues within 10 Å of the ligand. We implemented three rounds of such iterations to fully accommodate the ligand by sampling the surrounding residues.</div><div class="NLM_p">To analyze the thermodynamic properties of the binding-site waters, we carried out MD simulations of the 5-HT<sub>2B</sub> structure with the Gromacs v4.5.3 program,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> applying the AMBER99SB force field<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> for the protein and the TIP3P water model.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Water molecules and ligand in the crystal structure of the 5-HT<sub>2B</sub> receptor were removed. All calculations applied an atom-based truncation scheme that was updated heuristically with a list cutoff of 14 Å, a nonbond cutoff of 12 Å, and the Lennard–Jones (LJ) smoothing function initiated at 10 Å. Long-range electrostatic interactions were computed using the Particle Mesh Ewald (PME) method.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> To equilibrate the solvent molecules around the solute, the system was minimized with 500 steps of steepest descent, followed by 200 ps of NVT MD simulation and a subsequent 200 ps of NPT simulation. The 12 ns of NVT MD production simulation was performed at 300 K via the v-rescale temperature coupling scheme. The LINCS algorithm<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> was applied, with a time step of 2 fs, to constrain all bonds involving hydrogen atoms. All heavy atoms of the protein were harmonically restrained during the equilibration and production simulation with a force constant of 1000 kJ/mol·nm<sup>2</sup>. The trajectories of the last 5 ns of the MD simulations were submitted as input to the Solvent Property Analysis (SPA)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> program to compute the structural and energetic properties of the binding-site water molecules.</div><div class="NLM_p last">The induced-fit models were evaluated based on their ability to enrich known 5-HT<sub>2B</sub> ligands. In total, 206 5-HT<sub>2B</sub> antagonists were collected from the GLIDA database.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> We then further clustered these antagonists by their Bemis–Murcko atomic frameworks<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> to obtain 84 structurally diverse ligands (<a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>, Supporting Information). For each ligand, 50 decoy compounds were generated based on the DUD-E protocol,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> leading finally to 4096 nonredundant decoys that were physically similar but topologically dissimilar to the annotated ligands (<a class="ref internalNav" href="#notes-1" aria-label="Figure S8">Figure S8</a>, Supporting Information). Enrichment performance is one common metric used to measure docking performance; it represents the prioritization of true ligands among the top ranking hits of a docking-ordered library, which is calculated as EF<sub>subset</sub> = (ligands<sub>selected</sub>/<i>N</i><sub>subset</sub>)/(ligand<i>s</i><sub>tota</sub>l/<i>N</i><sub>total</sub>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Structure-Based Hierarchical Virtual Screening</h3><div class="NLM_p">Molecular docking was performed with the DOCK 3.5.54,<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56, 57)</a> using the doxepin-induced model as the receptor structure. Binding-site residues were identified within 12 Å of doxepin. Then the solvent-accessible molecular surface was calculated with the DMS<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> program using a probe radius of 1.4 Å. Receptor-derived matching spheres were generated with the SPHGEN program,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> and the ligand-derived spheres were generated from the positions of the heavy atoms of doxepin. Four sets of grids were generated: an excluded volume grid using DISMAP,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> a united atom AMBER-based van der Waals potential grid using CHEMGRID,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> an electrostatic potential grid using Delphi,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> and a solvent occlusion map using SOLVMAP.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> All of compounds in the library were prepared in the flexibase format with the latest version of the ZINC protocol.<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62, 63)</a> For sampling, the minimum number of graph matching nodes was set to 3, and ligand overlap was set to 0.1. Ligand conformations were scored on the basis of the total energy (<i>E</i><sub>tot</sub> = <i>E</i><sub>vdw</sub> + <i>E</i><sub>ele</sub> – Δ<i>G</i><sub>lig-solv</sub>) after 200 steps of rigid-body minimization, with initial translations of 0.2 Å and initial rotations of 5°.</div><div class="NLM_p last">MM-GB/SA refinement and rescoring was performed using PLOP. Briefly, the docked ligand was subjected to the fully flexible minimization in the receptor and in the solvent with the all-atom OPLS force field and the Surface Generalized Born implicit solvent as described previously.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The same protein structure used in the docking was used for rescoring. Hydrogen atoms were added in standard geometries as defined by the all-atom OPLS 2005 force field.<a onclick="showRef(event, 'ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref64 ref65">(64, 65)</a> The protein was kept rigid during protein–ligand complex minimization. The binding energy (<i>E</i><sub>bind</sub> = <i>E</i><sup>R*L</sup> – <i>E</i><sup>L</sup> – <i>E</i><sup>R</sup>) was calculated by subtracting the energies of the optimized free ligand in solution (E<sup>L</sup>) and the free protein in solution (<i>E</i><sup>R</sup>) from the optimized protein–ligand complex’s energy in solution (<i>E</i><sup>R*L</sup>). For the flexible rescoring procedure, the binding-site residues within 10 Å of the ligand were subjected to side-chain prediction and energy minimization.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry</h3><div class="NLM_p">Our in-house screening compound library was constructed from the commercially available ChemDiv compound database (<a href="http://eu.chemdiv.com" class="extLink">http://eu.chemdiv.com</a>) based on maximizing structural diversity.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Several analogues (<b>12</b>, <b>16</b>–<b>19</b>, and <b>26</b>–<b>28</b>) for SAR studies were purchased from Vitas-M (<a href="http://www.vitasmlab.com" class="extLink">http://www.vitasmlab.com</a>), Chembridge (<a href="http://www.chembridge.com" class="extLink">http://www.chembridge.com</a>), and Enamine (<a href="http://www.enamine.net" class="extLink">http://www.enamine.net</a>), respectively. For each of these compounds, the vendor had verified the compound purity with liquid chromatography–mass spectrometry (LC-MS) and/or nuclear magnetic resonance (NMR) experiments. The <sup>1</sup>H NMR spectrum and the MS data for the initially identified active compound <b>9</b> are included in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p">The other compounds (<b>11</b>, <b>13</b>–<b>15</b>, and <b>20</b>–<b>25</b>) were prepared with reference to the general procedure described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> All solvents, chemicals, and reagents were obtained commercially and used without purification. The <sup>1</sup>H NMR spectra were obtained in CDCl<sub>3</sub>, <i>d</i><sub>6</sub>-DMSO, CD<sub>3</sub>OD, or <i>d</i><sub>6</sub>-acetone at 25 °C at 300 MHz on an OXFORD instrument (Varian), with chemical shift (δ, ppm) reported relative to TMS as an internal standard. HPLC-MS chromatograms and mass spectra were obtained with a Shimadzu LC-MS-2020 system. The prep-HPLC instruments used included a Gilson GX-281(Gilson) and an Elite P230 Preparative Gradient System (Elite). Chiral Prep-HPLC was performed with the Elite P230 Preparative Gradient System using Thar Prep-80 and Thar SFC X-5 columns. The microwave instrument used was a CEM Discover SP. All tested compounds had a purity >95%.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedure for Compound Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reaction conditions: (i) concd HCl, EtOH, reflux; (ii) pyridine, EtOH, 120 °C.</p></p></figure></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Ethyl 3-((4-Amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-yl)(methyl)amino)benzoate (<b>11</b>)</h3><div class="NLM_p">Ethyl 3-aminobenzoate (4.1 g, 24.8 mmol) and di-<i>tert</i>-butyl dicarbonate (8.1 g, 37.2 mmol) were dissolved in anhydrous THF (60 mL). Then <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (6.4 g, 49.6 mmol) was added. The reaction mixture was stirred at room temperature. After completion of the reaction, it was concentrated in vacuo, and the residue was extracted with ethyl acetate. The organic layer was concentrated and triturated with hexanes to provide ethyl 3-(<i>tert</i>-butoxycarbonylamino)benzoate as a white solid (5.6 g, 21.1 mmol, 85%). MS [M + H]<sup>+</sup> calculated for C<sub>14</sub>H<sub>20</sub>NO<sub>4</sub>: 266.13, found: 266.10.</div><div class="NLM_p">Ethyl 3-(<i>tert</i>-butoxycarbonylamino)benzoate was dissolved in anhydrous THF (20 mL), and NaH (1.7 g, 42.2 mmol) was subsequently added under N<sub>2</sub> atmosphere. The mixture was stirred at room temperature for 15 min. MeI (2.0 mL, 31.7 mmol) was then added slowly in a dropwise manner. After completion, the reaction was quenched with saturated aqueous NH<sub>4</sub>Cl and extracted three times with ethyl acetate. The organic layers were combined and concentrated in vacuo to provide a residue; the residue was purified by column chromatography to give ethyl 3-(<i>tert</i>-butoxycarbonyl(methyl)amino)benzoate (2.4 g, 41%).</div><div class="NLM_p">Ethyl 3-(<i>tert</i>-butoxycarbonyl(methyl)amino)benzoate (2.4 g, 8.59 mmol) was dissolved in HCl (10 mL, 4 M) and THF (10 mL), and the mixture was refluxed overnight. It was then concentrated in vacuo, and the residue was purified by column chromatography to provide the ethyl 3-(methylamino)benzoate (1.2 g, 80%). MS [M + H]<sup>+</sup> calculated for C<sub>10</sub>H<sub>14</sub>NO<sub>2</sub>: 180.09, found: 180.10.</div><div class="NLM_p last">A mixture of ethyl 3-(methylamino)benzoate (606 mg, 3.4 mmol), cyanoguanidine (313 mg, 3.7 mmol), concentrated HCl (0.28 mL, 3.4 mmol), and cyclohexanone (334 mg, 3.4 mmol) in EtOH (2 mL) was refluxed for 2 days. The reaction mixture was neutralized with saturated NaHCO<sub>3</sub> to pH 8 and then extracted with DCM (50 mL, three times). The organic layers were concentrated in vacuo to provide a crude product, which was purified by column chromatography to give the compound <b>11</b> (28 mg, 2.5%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 8.77 (s, 1H), 8.05–7.80 (m, 3H), 7.63 (d, <i>J</i> = 5.1 Hz, 2H), 7.00 (s, 1H), 4.40–4.29 (m, 2H), 3.38 (s, 3H), 2.02–1.38 (m, 9H), 1.35–1.25 (m, 4H). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub>: 344.20, found: 344.20.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Ethyl 3-(4-(Methylamino)-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-ylamino)benzoate (<b>13</b>)</h3><div class="NLM_p">Sodium dicyanoamide (5 g, 56.2 mmol) and methylamine hydrochloride (3.8 g, 56.2 mmol) were dissolved in <i>n</i>-butanol (25 mL) and H<sub>2</sub>O (10 mL). The mixture was then refluxed. After completion of the reaction, the mixture was concentrated in vacuo and the residue was purified further by column chromatography to give 1-cyano-3-methylguanidine (1.7 g, 31%). MS [M + H]<sup>+</sup> calculated for C<sub>3</sub>H<sub>7</sub>N<sub>4</sub>: 99.06, found: 99.10.</div><div class="NLM_p">A mixture of 1-cyano-3-methylguanidine (500 mg, 5.1 mmol), ethyl 3-aminobenzoate (765 mg, 4.63 mmol), and concentrated HCl (0.39 mL, 4.63 mmol) in EtOH (2 mL) was refluxed for 6 h. Then the ethyl 3-(3-(<i>N</i>-methylcarbamimidoyl)guanidino)benzoate was obtained by filtration (510 mg, 37%). MS [M + H]<sup>+</sup> calculated for C<sub>12</sub>H<sub>18</sub>N<sub>5</sub>O<sub>2</sub>: 264.14, found: 264.14.</div><div class="NLM_p last">Ethyl 3-(3-(<i>N</i>-methylcarbamimidoyl)guanidino)benzoate (510 mg, 1.7 mmol) in cyclohexanone (15 mL) and EtOH (5 mL) was refluxed overnight and then cooled to room temperature. Saturated aqueous NaHCO<sub>3</sub> (25 mL) was then added. The mixture was extracted with DCM (50 mL, three times) and washed with brine (30 mL). The organic layers were combined and concentrated in vacuo to afford a crude product, which was purified by column chromatography to provide the compound <b>13</b> (230 mg, 39%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 8.89 (s, 1H), 7.96 (s, 1H), 7.75 (d, <i>J</i> = 7.9 Hz, 1H), 7.33 (d, <i>J</i> = 7.6 Hz, 1H), 7.22 (t, <i>J</i> = 7.8 Hz, 1H), 6.41 (s, 1H), 5.76 (s, 1H), 4.26 (q, <i>J</i> = 7.1 Hz, 2H), 2.72 (d, <i>J</i> = 4.2 Hz, 3H), 1.73–1.87 (m, 2H), 1.67–1.23 (m, 11H). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub>: 344.20, found: 344.20.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Ethyl 3-(4-(Dimethylamino)-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-ylamino)benzoate (<b>14</b>)</h3><div class="NLM_p">Sodium dicyanoamide (5 g, 56.2 mmol) and dimethylamine hydrochloride (4.58 g, 56.2 mmol) were dissolved in <i>n</i>-butanol and H<sub>2</sub>O, and this mixture was then refluxed. After completion of the reaction, the mixture was concentrated in vacuo and the residue was purified further by column chromatography to obtain the product 2-cyano-1,1-dimethylguanidine (1.5 g, 24%). MS [M + H]<sup>+</sup> calculated for C<sub>4</sub>H<sub>9</sub>N<sub>4</sub>: 113.07, found: 113.10.</div><div class="NLM_p last">A mixture of 2-cyano-1,1-dimethylguanidine (500 mg, 4.46 mmol), ethyl 3-aminobenzoate (670 mg, 4.1 mmol), concentrated HCl (0.34 mL, 4.1 mmol), and cyclohexanone (402 mg, 4.1 mmol) in EtOH (2 mL) was refluxed. After completion of the reaction, it was cooled to room temperature. The reaction mixture was then neutralized with saturated NaHCO<sub>3</sub> to pH 8 and then extracted with DCM (50 mL, three times). The organic layers were combined and concentrated in vacuo to afford a crude product, which was purified by column chromatography to give compound <b>14</b> (60 mg, 4%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.63 (s, 1H), 9.14 (s, 1H), 8.46 (s, 1H), 8.22 (s, 1H), 7.69 (d, <i>J</i> = 7.6 Hz, 1H), 7.59 (d, <i>J</i> = 8.7 Hz, 1H), 7.50 (t, <i>J</i> = 7.9 Hz, 1H), 4.31 (q, <i>J</i> = 7.1 Hz, 2H), 3.30–2.97 (m, 6H), 1.94–1.61 (m, 9H), 1.36–1.16 (m, 4H). MS [M + H]<sup>+</sup> calculated for C<sub>19</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub>: 358.22, found: 358.20.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>N</i><sup>4</sup>-(3-Bromophenyl)-1,3,5-triazaspiro[5.5]undeca-1,3-diene-2,4-diamine (<b>15</b>)</h3><div class="NLM_p">A mixture of 3-bromobenzenamine (2 g, 11.6 mmol), cyanoguanidine (1.08 g, 12.8 mmol), concentrated HCl (1.0 mL, 11.6 mmol), and cyclohexanone (1.1 g, 11.6 mmol) in EtOH (2 mL) was refluxed. After completion of the reaction, the solid was filtered to obtain 5-(3-bromophenyl)-1,3,5-triazaspiro[5.5]undeca-1,3-diene-2,4-diamine (1.6 g, 41%). MS [M + H]<sup>+</sup> calculated for C<sub>14</sub>H<sub>19</sub>BrN<sub>5</sub>: 336.07, found: 336.0.</div><div class="NLM_p last">5-(3-Bromophenyl)-1,3,5-triazaspiro[5.5]undeca-1,3-diene-2,4-diamine was dissolved in pyridine (2 mL) and EtOH (3 mL), and the mixture was heated to and held at 120 °C overnight. It was then concentrated in vacuo to dryness. After water (10 mL) was added, the mixture was stirred at room temperature for 30 min. Then the solid was filtered and dried in vacuo to afford compound <b>15</b> (310 mg, 62%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.28 (s, 1H), 8.91 (s, 1H), 8.58 (d, <i>J</i> = 4.2 Hz, 1H), 7.82–7.78 (m, 1H), 7.58–7.49 (m, 1H), 7.39 (dd, <i>J</i> = 7.6 Hz, <i>J</i> = 5.7 Hz, 1H), 7.31–7.27 (m, 2H), 1.88–1.16 (m, 10H). MS [M + H]<sup>+</sup> calculated for C<sub>14</sub>H<sub>19</sub>BrN<sub>5</sub>: 336.07, found: 336.0.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 3-(4-Amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-ylamino)benzoic Acid (<b>20</b>)</h3><div class="NLM_p last">A solution of ethyl 3-(4-amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-ylamino)benzoate (25.0 mg, 0.09 mmol) in MeOH (1.00 mL) was added to NaOH (2.00 mL). Then the reaction mixture was stirred at 30 °C overnight, diluted with water, and slowly acidified with trifluoroacetic acid to adjust the reaction mixture to pH < 4. The mixture was concentrated, and the resulting residue was then purified using reverse-phase chromatography to give a white solid (3.0 mg, 13%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 1.37–1.70 (m, 10H), 6.97–7.08 (m, 1H), 7.44 (t, <i>J</i> = 8.0 Hz, 1H), 7.65–7.67(m, 1H), 7.82 (s, 1H), 7.85–7.87 (m, 1H), 8.57–8.71 (m, 2H), 9.69–9.82 (m, 1H). MS [M + H]<sup>+</sup> calculated for C<sub>15</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub>: 302.16, found: 302.30.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 1-(3-((4-Amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-yl)amino)phenyl)butan-1-one (<b>21</b>)</h3><div class="NLM_p">A solution of 1-phenylbutan-1-one (2500 mg, 16.9 mmol) was added to HNO<sub>3</sub> (5 mL) at −5 °C dropwise in flask. The resulting suspension was stirred at −5 °C for about 4 h and monitored by TLC until the starting materials were completely consumed. Ice– water was added to the reaction solution and filtered to give the crude product 1-(3-nitrophenyl)butan-1-one as a yellow solid, which was then used in the next step without further purification (900 mg, 28%). MS [M + H]<sup>+</sup> calculated for C<sub>10</sub>H<sub>12</sub>NO<sub>3</sub>: 194.2, found: 194.2.</div><div class="NLM_p">A solution of 1-(3-nitrophenyl)butan-1-one (900 mg, 4.7 mmol) in EtOH (50 mL) was added to Zn powder (0.6 g, 9.4 mmol) and concentrated HCl (1.8 g, 18.8 mmol) at 0 °C. The resulting suspension was allowed to warm to room temperature and monitored by TLC until the starting materials were completely consumed. The reaction solution was filtered and the solvent was eliminated under reduced pressure to give the crude product, which was purified by chromatographic column to gain the desired product 1-(3-aminophenyl)butan-1-one (450 mg, 59%). MS [M + H]<sup>+</sup> calculated for C<sub>10</sub>H<sub>14</sub>NO: 164.2, found: 164.2.</div><div class="NLM_p">A solution of 1-(3-aminophenyl)butan-1-one (450 mg, 2.8 mmol) in EtOH (50 mL) was added to cyanoguanidine (235 mg, 2.8 mmol) and concentrated HCl (2.8 g, 28 mmol) at room temperature. The resulting suspension was warmed to reflux overnight and monitored by TLC until the starting materials were completely consumed. The solvent was eliminated under reduced pressure to give the crude product 1-(3-(3-(carbamimidoyl)guanidino)phenyl)butan-1-one, which was used in the next step without further purification (180 mg, 26%). MS [M + H]<sup>+</sup> calculated for C<sub>12</sub>H<sub>18</sub>N<sub>5</sub>O: 248.3, found: 248.3.</div><div class="NLM_p">A solution of 1-(3-(3-(carbamimidoyl)guanidino)phenyl)butan-1-one (180 mg, 0.7 mmol) in EtOH (50 mL) was added to cyclohexanone (137 mg, 1.4 mmol) and triethyl orthoformate (208 mg, 1.4 mmol) at 0 °C. The resulting suspension was allowed to warm to room temperature overnight and monitored by TLC until the starting materials were completely consumed. The solvent was eliminated under reduced pressure to give the crude product 1-(3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)phenyl)butan-1-one, which was used in the next step without further purification (120 mg, 51%). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>26</sub>N<sub>5</sub>O: 328.4, found: 328.4.</div><div class="NLM_p last">A solution of 1-(3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)phenyl)butan-1-one (120 mg, 0.37 mmol) in EtOH (15 mL) was added to pyridine (315 mg, 3.7 mmol) at room temperature. The resulting suspension was allowed to warm to reflux overnight and monitored by TLC until the starting materials were completely consumed. The solvent was eliminated under reduced pressure to give the crude product, which was washed with MeOH to afford the compound <b>21</b> (27 mg, 22%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.44 (s, 1H), 9.08 (s, 2H), 8.92 (s, 1H), 8.53 (t, <i>J</i> = 8.0 Hz, 1H), 7.99–8.05 (m, 2H), 7.86–7.97 (m, 1H), 7.65–7.75 (m, 1H), 7.46–7.50 (m, 1H), 3.44 (q, <i>J</i> = 8.0 Hz, 1H), 3.00 (t, <i>J</i> = 8.0 Hz, 2H), 1.60–1.70 (m, 9H), 1.28–1.51 (m, 2H), 1.06 (t, <i>J</i> = 8.0 Hz, 2H), 0.94 (t, <i>J</i> = 8.0 Hz, 2H). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>26</sub>N<sub>5</sub>O: 328.21, found: 328.2.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 3-(4-Amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-ylamino)-<i>N</i>-ethylbenzamide (<b>22</b>)</h3><div class="NLM_p">A solution of 3-aminobenzoic acid (4.4 g, 32.1 mmol) and di-<i>tert</i>-butyl dicarbonate (10.5 g, 48.2 mmol) in anhydrous THF (60 mL) was added to <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (8.3 g, 64.2 mmol). The reaction mixture was stirred at room temperature. After completion of the reaction, it was concentrated in vacuo and the residue was extracted with ethyl acetate. The organic layer was concentrated, and the residue was purified by column chromatography to provide 3-(<i>tert</i>-butoxycarbonylamino)benzoic acid (6.2 g, 82%). MS [M – H]<sup>−</sup> calculated for C<sub>12</sub>H<sub>14</sub>NO<sub>4</sub>: 236.10, found: 236.10.</div><div class="NLM_p">A mixture of 3-(<i>tert</i>-butoxycarbonylamino)benzoic acid (2 g, 8.43 mmol), HATU (6.4 g, 16.9 mmol), and <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (3.27 g, 25.3 mmol) was dissolved in DMF (40 mL), and then the mixture was stirred at room temperature for 30 min. Then ethanamine hydrochloride (2.1 g, 25.3 mmol) was added. After completion of the reaction, water was added and the mixture was extracted three times with EtOAc. The organic layer was concentrated to give the crude product <i>tert</i>-butyl 3-(ethylcarbamoyl)phenylcarbamate (2.24 g, 100%) which was used without further purification.</div><div class="NLM_p"><i>tert</i>-Butyl 3-(ethylcarbamoyl)phenylcarbamate (3.24 g, 12.3 mmol) was dissolved in dioxane (40 mL), and then concentrated HCl (10 mL) was added. The mixture was stirred at room temperature for 8 h. After completion of the reaction, it was concentrated in vacuo and the residue was purified by column chromatography to give 3-amino-<i>N</i>-ethylbenzamide (1.4 g, 70%). MS [M + H]<sup>+</sup> calculated for C<sub>9</sub>H<sub>13</sub>N<sub>2</sub>O: 165.09, found: 165.10.</div><div class="NLM_p">A mixture of 3-amino-<i>N</i>-ethylbenzamide (1 g, 6.1 mmol), cyanoguanidine (563 mg, 6.7 mmol), concentrated HCl (0.51 mL, 6.1 mmol), and cyclohexanone (599 mg, 6.1 mmol) in EtOH (2 mL) was refluxed. After completion of the reaction, the solid was filtered to give the desired product 3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)-<i>N</i>-ethylbenzamide (1.18 g, 59%) as a white solid. MS [M + H]<sup>+</sup> calculated for C<sub>17</sub>H<sub>25</sub>N<sub>6</sub>O: 329.20, found: 329.20.</div><div class="NLM_p last">3-(2,4-Diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)-<i>N</i>-ethylbenzamide (300 mg, 0.91 mmol) was dissolved in pyridine (3 mL), the mixture was heated to 120 °C overnight, and then it was concentrated in vacuo to give the crude product as a salt. Then the solid was neutralized by sat. NaHCO<sub>3</sub> to pH = 8. The solid was filtered and dried in vacuo to give the compound <b>22</b> (232 mg, 77%) as a pale pink solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.17 (s, 1H), 8.90–9.01(m, 2H), 8.48 (t, <i>J</i> = 5.3 Hz, 1H), 7.76–7.81 (m, 2H), 7.57 (d, <i>J</i> = 7.7 Hz, 1H), 7.41 (t, <i>J</i> = 7.8 Hz, 1H), 7.18 (s, 1H), 3.33–3.24 (m, 2H), 1.89–1.25 (m, 10H), 1.12 (t, <i>J</i> = 7.2 Hz, 3H). MS [M + H]<sup>+</sup> calculated for C<sub>17</sub>H<sub>25</sub>N<sub>6</sub>O: 329.20, found: 329.20.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Propyl 3-(4-Amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-ylamino)benzoate (<b>23</b>)</h3><div class="NLM_p">3-Aminobenzoic acid (2 g, 14.6 mmol) was dissolved in propan-1-ol (20 mL), concentrated sulfuric acid (0.5 mL) was added, and the mixture was refluxed overnight. After completion of the reaction, it was concentrated in vacuo to obtain propyl 3-aminobenzoate (2.4 g, 92%) and used in the next step without further purification. MS [M + H]<sup>+</sup> calculated for C<sub>10</sub>H<sub>14</sub>NO<sub>2</sub>: 180.09, found: 180.10.</div><div class="NLM_p">A mixture of propyl 3-aminobenzoate (2 g, 11.2 mmol), cyanoguanidine (1 g, 12.3 mmol), concentrated HCl (0.93 mL, 11.2 mmol), and cyclohexanone (1.1 g, 11.2 mmol) in propan-1-ol (2 mL) was refluxed. After completion of the reaction, the solid was filtered to provide propyl 3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)benzoate (1.1 g, 29%). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub>: 344.20, found: 344.20.</div><div class="NLM_p last">Propyl 3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)benzoate (500 mg, 1.46 mmol) was dissolved in pyridine (2 mL) and propan-1-ol (3 mL), and then the mixture was heated to 120 °C overnight. After LC-MS showed that the reaction was completed, the mixture was concentrated in vacuo to give the crude product as a salt. Then the solid was neutralized by sat. NaHCO3 to pH = 8. The solid was filtered and dried in vacuo to give the desired product (410 mg, 82%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.33 (s, 1H), 8.97 (s, 2H), 8.06–8.02 (m, 1H), 7.88 (s, 1H), 7.70 (d, <i>J</i> = 7.8 Hz, 1H), 7.49 (t, <i>J</i> = 7.9 Hz, 1H), 7.22 (s, 1H), 4.25 (t, <i>J</i> = 6.6 Hz, 2H), 1.78–1.66 (m, 10H), 1.49–1.31 (m, 2H), 0.98 (t, <i>J</i> = 7.4 Hz, 3H). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub>: 344.20, found: 344.20.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> (3-((4-Amino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-2-yl)amino)phenyl)(cyclopropyl)methanone (<b>24</b>)</h3><div class="NLM_p">A mixture of (3-aminophenyl) (cyclopropyl)methanone (350 mg, 2.0 mmol), cyanoguanidine (185 mg, 2.2 mmol), concentrated HCl (0.21 mL, 2.5 mmol), and cyclohexanone (216 mg, 2.2 mmol) in EtOH (5 mL) was refluxed. After the reaction was complete, the solid was filtered to obtain cyclopropyl(3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)phenyl)methanone (160 mg, 25%). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>23</sub>N<sub>5</sub>O: 326.1, found: 326.1.</div><div class="NLM_p last">Cyclopropyl(3-(2,4-diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)phenyl)methanone (80 mg, 0.25 mmol) was dissolved in pyridine (1 mL) and EtOH (1.5 mL), and then the mixture was heated to 120 °C overnight. After the reaction was complete, the mixture was concentrated in vacuo, and water (2 mL) was added. The mixture was stirred at room temperature for 30 min. Then the solid was filtered and dried in vacuo to afford the compound <b>24</b> (68 mg, 85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 8.93 (s, 1H), 8.19 (d, <i>J</i> = 7.7 Hz, 1H), 7.95 (d, <i>J</i> = 1.6 Hz, 1H), 7.60–7.69 (m, 3H), 7.38 (s, 1H), 6.44 (s, 1H), 2.93–3.10 (m, 1H), 1.93 (t, <i>J</i> = 12.0 Hz, 2H), 1.56–1.63 (m, 5H), 1.39–1.23 (m, 2H), 1.14–0.88 (m, 5H). MS [M + H]<sup>+</sup> calculated for C<sub>18</sub>H<sub>23</sub>N<sub>5</sub>O: 326.1, found: 326.1.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 3-(2,4-Diamino-1,3,5-triazaspiro[5.5]undeca-2,4-dien-1-yl)-<i>N</i>,<i>N</i>-diethylbenzamide (<b>25</b>)</h3><div class="NLM_p">3-Aminobenzoic acid (4.4 g, 32.1 mmol) and di-<i>tert</i>-butyl dicarbonate (10.5 g, 48.2 mmol) were dissolved in anhydrous THF (60 mL). Then <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (8.3 g, 64.2 mmol) was added. The reaction mixture was stirred at room temperature. After completion of the reaction, it was concentrated in vacuo and the residue was extracted with ethyl acetate. The organic layer was concentrated, and the residue was purified by column chromatograph to provide 3-(<i>tert</i>-butoxycarbonylamino)benzoic acid (6.2 g, 82%); MS [M – H]<sup>−</sup> calculated for C<sub>12</sub>H<sub>14</sub>NO<sub>4</sub> 236.10, found 236.10.</div><div class="NLM_p">A mixture of 3-(<i>tert</i>-butoxycarbonylamino)benzoic acid (2 g, 8.43 mmol), HATU (6.4 g, 16.9 mmol), and <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (3.27 g, 25.3 mmol) was dissolved in DMF (40 mL), and then the mixture was stirred at room temperature for 30 min. Then diethanamine hydrochloride (2.77 g, 25.3 mmol) was added. After completion of the reaction, water was added and the mixture was extracted with EtOAc for three times. The organic layer was concentrated to give <i>tert</i>-butyl 3-(ethylcarbamoyl)phenylcarbamate (2.2 g, 89%) which was used in the next step without further purification.</div><div class="NLM_p"><i>tert</i>-Butyl 3-(ethylcarbamoyl)phenylcarbamate (2.2 g, 7.52 mmol) was dissolved in dioxane (40 mL), and then concentrated HCl (10 mL) was added. The mixture was stirred at room temperature for 8 h. After completion of the reaction, it was concentrated in vacuo and the residue was purified by column chromatography (silica gel, PE:EA = 4:1) to give 3-amino-<i>N</i>,<i>N</i>-diethylbenzamide (1.45 g, 100%); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ(ppm) 7.16 (t, <i>J</i> = 7.7 Hz, 1H), 6.82–6.53 (m, 3H), 3.53 (m, 2H), 3.28 (m, 2H), 1.22 (m, 6H). MS [M + H]<sup>+</sup> calculated for C<sub>11</sub>H<sub>16</sub>N<sub>2</sub>O: 193.13, found: 193.20.</div><div class="NLM_p last">A mixture of 3-amino-<i>N</i>,<i>N</i>-diethylbenzamide (500 mg, 2.6 mmol), cyanoguanidine (241 mg, 2.86 mmol), concentrated HCl (0.22 mL, 2.6 mmol), and cyclohexanone (255 mg, 2.6 mmol) in EtOH (2 mL) was refluxed for 6 h. LC-MS showed that two isomer products were formed. The desired product compound <b>25</b> was obtained by Prep-HPLC (52 mg, 6%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 10.24 (s, 1H), 9.17 (s, 1H), 8.83 (s, 1H), 7.82 (s, 1H), 7.71 (m, 1H), 7.49–7.35 (m, 2H), 7.36 (s, 1H), 7.05 (d, <i>J</i> = 7.4 Hz, 1H), 3.43 (m, 2H), 3.19 (m, 2H), 1.33 (m, 16H). MS [M + H]<sup>+</sup> calculated for C<sub>17</sub>H<sub>25</sub>N<sub>6</sub>O: 357.23, found: 357.20.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Biological Assays</h3><div class="NLM_p last">Detailed descriptions of cellular function assay, binding assay, and animal assay are included in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. Briefly, cellular function assays were performed with the CHO-K1/5-HT<sub>2B</sub> cell line, using a calcium flux assay method. The binding assays were performed by staff at the National Institute of Mental Health’s Psychoactive Drug Screening Program (NIMH-PDSP), following the standard protocol.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> The radiolabeled reference compounds ([<sup>3</sup>H]8-OH-DPAT for 5-HT<sub>1A</sub>; [<sup>3</sup>H]GR127543 for 5-HT<sub>1B</sub> and 5-HT<sub>1D</sub>; [<sup>3</sup>H]5-HT for 5-HT<sub>1E</sub>; [<sup>3</sup>H]ketanserin for 5-HT<sub>2A</sub>; [<sup>3</sup>H]LSD for 5-HT<sub>2B</sub> and 5-HT<sub>2C</sub>, 5-HT<sub>5a</sub>, 5-HT<sub>6</sub>, and 5-HT<sub>7</sub>; [<sup>3</sup>H]LY278584 for 5-HT<sub>3</sub>) were used in the <i>K</i><sub>i</sub> determination assays. The PDSP online data entry and analysis system calculates the variance of the quadruplicate measurements (for the total, nonspecific, and test compound binding values); variances greater than 20% are flagged for further inspection, and assays are repeated if necessary. The IBS rat models for the visceral hypersensitivity assays were induced by the injection of 2,4,6-trinitrobenzenesulfonic acid (TNBS) and then treated with chronic unpredictable mild stress (CUMS).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Haematoxylin–eosin (HE) stains of the proximal and distal colon were compared to confirm the lack of histological changes in the IBS rat model (<a class="ref internalNav" href="#notes-1" aria-label="Figure S9">Figure S9</a>, Supporting Information). Behavioral responses to colorectal distension (CRD) were used to assess visceral sensitivity by observing the AWR. The pain threshold to CRD was determined as the minimum pressure required to evoke AWR score 3 and AWR score 4.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01631">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01631" class="ext-link">10.1021/acs.jmedchem.5b01631</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures and tables illustrating all computational and biological results, detailed experimental procedures, and <sup>1</sup>H NMR spectra and LC-MS spectra data of all synthesized compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Table illustrating biological results (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_001.pdf">jm5b01631_si_001.pdf (2.49 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_002.csv">jm5b01631_si_002.csv (3.96 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01631" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54525" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54525" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaichun Wu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Cancer Biology and Xijing Hospital of Digestive
Diseases, Fourth Military Medical University, 127 West Changle Road, Xi’an, Shaanxi Province 710032, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a2cad7c3ccc5cccbd7e2cccbc0d18cc3c18cc1cc"><span class="__cf_email__" data-cfemail="fd95889c939a939488bd93949f8ed39c9ed39e93">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niu Huang</span> - <span class="hlFld-Affiliation affiliation">National Institute
of Biological Sciences, Beijing, No.
7 Science Park Road, Zhongguancun Life Science Park, Beijing 102206, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#503b313933272510363d3d257e3534257e333e"><span class="__cf_email__" data-cfemail="f299939b918587b2949f9f87dc979687dc919c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Zhou</span> - <span class="hlFld-Affiliation affiliation">National Institute
of Biological Sciences, Beijing, No.
7 Science Park Road, Zhongguancun Life Science Park, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Pharmaceutical Sciences, School of Medicine, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Ma</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Cancer Biology and Xijing Hospital of Digestive
Diseases, Fourth Military Medical University, 127 West Changle Road, Xi’an, Shaanxi Province 710032, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xingyu Lin</span> - <span class="hlFld-Affiliation affiliation">National Institute
of Biological Sciences, Beijing, No.
7 Science Park Road, Zhongguancun Life Science Park, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xi-Ping Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, The National Institute of Mental Health Psychoactive
Drug Screening Program (NIMH PDSP), The
University of North Carolina, Chapel
Hill, North Carolina 27759, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d160e2914-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Financial support from the Chinese Ministry of Science and Technology “973” Grant (2014CB849802) and the National Natural Science Foundation of China (81470862) (to N.H.) is gratefully acknowledged. Y.Z. was supported by the AstraZeneca Open Innovation Postdoctoral Research Fellowship, the China Postdoctoral Science Foundation (2015M570948), and the Beijing Postdoctoral Research Foundation (2015ZZ-118). X-P.H. was supported by the National Institute of Mental Health Psychoactive Drug Screening Program (NIMH-PDSP, HHSN-271-2013-00017-C) directed by Bryan L. Roth (MD, Ph.D.) at the University of North Carolina at Chapel Hill and Program Officer Jamie Driscoll at NIMH (Bethesda, MD). Computational support was provided by the Supercomputing Center of the Chinese Academy of Sciences (SCCAS). We also thank the Chemical Center at NIBS for providing the in-house screening compounds.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i31" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i31"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i32" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i32"> Abbreviations Used</h2><tr><td class="NLM_term">PAH</td><td class="NLM_def"><p class="first last">pulmonary arterial hypertension</p></td></tr><tr><td class="NLM_term">ECL</td><td class="NLM_def"><p class="first last">extracellular loop</p></td></tr><tr><td class="NLM_term">TM</td><td class="NLM_def"><p class="first last">transmembrane</p></td></tr><tr><td class="NLM_term">TNBS</td><td class="NLM_def"><p class="first last">2,4,6-trinitrobenzenesulfonic acid</p></td></tr><tr><td class="NLM_term">HE</td><td class="NLM_def"><p class="first last">haematoxylin–eosin</p></td></tr><tr><td class="NLM_term">AWR</td><td class="NLM_def"><p class="first last">abdominal withdrawal reflex</p></td></tr><tr><td class="NLM_term">CRD</td><td class="NLM_def"><p class="first last">colorectal distension</p></td></tr><tr><td class="NLM_term">CUMS</td><td class="NLM_def"><p class="first last">chronic unpredictable mild stress</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41845" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41845" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 68 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Filmore, D.</span><span> </span><span class="NLM_article-title">It’s a GPCR world</span> <span class="citation_source-journal">Mod. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVKnsbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=24-28&author=D.+Filmore&title=It%E2%80%99s+a+GPCR+world"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">It's a GPCR world</span></div><div class="casAuthors">Filmore, David</div><div class="citationInfo"><span class="NLM_cas:title">Modern Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">24-26, 28</span>CODEN:
                <span class="NLM_cas:coden">MDDIFT</span>;
        ISSN:<span class="NLM_cas:issn">1099-8209</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with refs.  Cell-based screening assays and structural studies are fueling G-protein coupled receptors (GPCR) as one of the most popular classes of investigational drug targets.  Besides the many GPCRs that function as basic receptors for sensory functions like seeing and smelling, which are not prime therapeutic targets, there are more than 300 other GPCRs up for grabs for drug discovery initiatives.  About 200 of these, a portion of which account for currently marketed GPCR drug targets, have known natural ligands.  And ligands are not known for about 150 nonsensory receptors identified as GPCRs from the Human Genome Project.  Several firms have completely focus on GPCRs because of the largely uncharted path of discovery they offer.  GPCR seem to be a good bet for future drug discovery successes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUWfG2qMCe7rVg90H21EOLACvtfcHk0liIurEoiNi_hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVKnsbfN&md5=41753a459c3c8a63bf1ff48d6e2585fd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFilmore%26aufirst%3DD.%26atitle%3DIt%25E2%2580%2599s%2520a%2520GPCR%2520world%26jtitle%3DMod.%2520Drug%2520Discovery%26date%3D2004%26volume%3D7%26spage%3D24%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of beta<sub>2</sub>-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span><span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta2-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0liIurEoiNi_hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta2-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of A<sub>2A</sub> adenosine receptor ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3748</span><span class="NLM_x">–</span> <span class="NLM_lpage">3755</span><span class="refDoi"> DOI: 10.1021/jm100240h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.+G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-based+discovery+of+A2A+adenosine+receptor+ligands&doi=10.1021%2Fjm100240h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0liafe1avkTRug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520A2A%2520adenosine%2520receptor%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755%26doi%3D10.1021%2Fjm100240h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D<sub>3</sub> receptor homology model and crystal structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0liafe1avkTRug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziarek, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heveker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkman, B. F.</span><span> </span><span class="NLM_article-title">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5517</span><span class="NLM_x">–</span> <span class="NLM_lpage">5522</span><span class="refDoi"> DOI: 10.1073/pnas.1120431109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1073%2Fpnas.1120431109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=22431600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5517-5522&author=M.+M.+Mysingerauthor=D.+R.+Weissauthor=J.+J.+Ziarekauthor=S.+Gravelauthor=A.+K.+Doakauthor=J.+Karpiakauthor=N.+Hevekerauthor=B.+K.+Shoichetauthor=B.+F.+Volkman&title=Structure-based+ligand+discovery+for+the+protein-protein+interface+of+chemokine+receptor+CXCR4&doi=10.1073%2Fpnas.1120431109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span></div><div class="casAuthors">Mysinger, Michael M.; Weiss, Dahlia R.; Ziarek, Joshua J.; Gravel, Stephanie; Doak, Allison K.; Karpiak, Joel; Heveker, Nikolaus; Shoichet, Brian K.; Volkman, Brian F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5517-5522, S5517/1-S5517/28</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are key signaling mols. and are intensely studied.  Whereas GPCRs recognizing small-mols. have been successfully targeted for drug discovery, protein-recognizing GPCRs, such as the chemokine receptors, claim few drugs or even useful small mol. reagents.  This reflects both the difficulties that attend protein-protein interface inhibitor discovery, and the lack of structures for these targets.  Imminent structure detn. of chemokine receptor CXCR4 motivated docking screens for new ligands against a homol. model and subsequently the crystal structure.  More than 3 million mols. were docked against the model and then against the crystal structure; 24 and 23 high-scoring compds. from the resp. screens were tested exptl.  Docking against the model yielded only one antagonist, which resembled known ligands and lacked specificity, whereas the crystal structure docking yielded four that were dissimilar to previously known scaffolds and apparently specific.  Intriguingly, several were potent and relatively small, with IC50 values as low as 306 nM, ligand efficiencies as high as 0.36, and with efficacy in cellular chemotaxis.  The potency and efficiency of these mols. has few precedents among protein-protein interface inhibitors, and supports structure-based efforts to discover leads for chemokine GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeJKWh0aH7FLVg90H21EOLACvtfcHk0liKnVMu8xyFhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D&md5=762616e6eb0eebc0d983964d71499e75</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120431109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120431109%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DZiarek%26aufirst%3DJ.%2BJ.%26aulast%3DGravel%26aufirst%3DS.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DHeveker%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DVolkman%26aufirst%3DB.%2BF.%26atitle%3DStructure-based%2520ligand%2520discovery%2520for%2520the%2520protein-protein%2520interface%2520of%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5517%26epage%3D5522%26doi%3D10.1073%2Fpnas.1120431109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eitel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmeiner, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Muscarinic receptors as model targets and antitargets for structure-based ligand discovery</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">528</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1124/mol.113.087551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1124%2Fmol.113.087551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=528-540&author=A.+C.+Kruseauthor=D.+R.+Weissauthor=M.+Rossiauthor=J.+Huauthor=K.+Huauthor=K.+Eitelauthor=P.+Gmeinerauthor=J.+Wessauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Muscarinic+receptors+as+model+targets+and+antitargets+for+structure-based+ligand+discovery&doi=10.1124%2Fmol.113.087551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.087551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.087551%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DRossi%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DK.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMuscarinic%2520receptors%2520as%2520model%2520targets%2520and%2520antitargets%2520for%2520structure-based%2520ligand%2520discovery%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D528%26epage%3D540%26doi%3D10.1124%2Fmol.113.087551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strachan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Conformation guides molecular efficacy in docking screens of activated beta-2 adrenergic G protein coupled receptor</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1018</span><span class="NLM_x">–</span> <span class="NLM_lpage">1026</span><span class="refDoi"> DOI: 10.1021/cb400103f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400103f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1018-1026&author=D.+R.+Weissauthor=S.+Ahnauthor=M.+F.+Sassanoauthor=A.+Kleistauthor=X.+Zhuauthor=R.+Strachanauthor=B.+L.+Rothauthor=R.+J.+Lefkowitzauthor=B.+K.+Shoichet&title=Conformation+guides+molecular+efficacy+in+docking+screens+of+activated+beta-2+adrenergic+G+protein+coupled+receptor&doi=10.1021%2Fcb400103f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb400103f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400103f%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DAhn%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DKleist%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DStrachan%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DConformation%2520guides%2520molecular%2520efficacy%2520in%2520docking%2520screens%2520of%2520activated%2520beta-2%2520adrenergic%2520G%2520protein%2520coupled%2520receptor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1018%26epage%3D1026%26doi%3D10.1021%2Fcb400103f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Rodríguez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brea, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loza, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, J.</span><span> </span><span class="NLM_article-title">Structure-based discovery of selective serotonin 5-HT<sub>1B</sub> receptor ligands</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1140</span><span class="NLM_x">–</span> <span class="NLM_lpage">1151</span><span class="refDoi"> DOI: 10.1016/j.str.2014.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.str.2014.05.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=1140-1151&author=D.+Rodr%C3%ADguezauthor=J.+Breaauthor=M.+I.+Lozaauthor=J.+Carlsson&title=Structure-based+discovery+of+selective+serotonin+5-HT1B+receptor+ligands&doi=10.1016%2Fj.str.2014.05.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DBrea%26aufirst%3DJ.%26aulast%3DLoza%26aufirst%3DM.%2BI.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DStructure-based%2520discovery%2520of%2520selective%2520serotonin%25205-HT1B%2520receptor%2520ligands%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D1140%26epage%3D1151%26doi%3D10.1016%2Fj.str.2014.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vardy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCorvy, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siu, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Structural features for functional selectivity at serotonin receptors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">340</span><span class="NLM_x">, </span> <span class="NLM_fpage">615</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1126/science.1232808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1126%2Fscience.1232808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23519215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslWksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2013&pages=615-619&author=D.+Wackerauthor=C.+Wangauthor=V.+Katritchauthor=G.+W.+Hanauthor=X.+P.+Huangauthor=E.+Vardyauthor=J.+D.+McCorvyauthor=Y.+Jiangauthor=M.+Chuauthor=F.+Y.+Siuauthor=W.+Liuauthor=H.+E.+Xuauthor=V.+Cherezovauthor=B.+L.+Rothauthor=R.+C.+Stevens&title=Structural+features+for+functional+selectivity+at+serotonin+receptors&doi=10.1126%2Fscience.1232808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Features for Functional Selectivity at Serotonin Receptors</span></div><div class="casAuthors">Wacker, Daniel; Wang, Chong; Katritch, Vsevolod; Han, Gye Won; Huang, Xi-Ping; Vardy, Eyal; McCorvy, John D.; Jiang, Yi; Chu, Meihua; Siu, Fai Yiu; Liu, Wei; Xu, H. Eric; Cherezov, Vadim; Roth, Bryan L.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">340</span>
        (<span class="NLM_cas:issue">6132</span>),
    <span class="NLM_cas:pages">615-619</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Drugs active at G protein-coupled receptors (GPCRs) can differentially modulate either canonical or noncanonical signaling pathways via a phenomenon known as functional selectivity or biased signaling.  The authors report biochem. studies showing that the hallucinogen lysergic acid diethylamide, its precursor ergotamine (ERG), and related ergolines display strong functional selectivity for β-arrestin signaling at the 5-HT2B 5-hydroxytryptamine (5-HT) receptor, whereas they are relatively unbiased at the 5-HT1B receptor.  To investigate the structural basis for biased signaling, the authors detd. the crystal structure of the human 5-HT2B receptor bound to ERG and compared it with the 5-HT1B/ERG structure.  Given the relatively poor understanding of GPCR structure and function to date, insight into different GPCR signaling pathways is important to better understand both adverse and favorable therapeutic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreo_QegM-j7bVg90H21EOLACvtfcHk0lhPNKG3JcznfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslWksb0%253D&md5=16e96d0255be3ac367aeda7187aa4c15</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1232808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1232808%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DVardy%26aufirst%3DE.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DSiu%26aufirst%3DF.%2BY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DH.%2BE.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520features%2520for%2520functional%2520selectivity%2520at%2520serotonin%2520receptors%26jtitle%3DScience%26date%3D2013%26volume%3D340%26spage%3D615%26epage%3D619%26doi%3D10.1126%2Fscience.1232808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gati, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weierstall, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zatsepin, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messerschmidt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koglin, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seibert, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromme, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupitz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rendek, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotjohann, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromme, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirian, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beyerlein, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spence, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span> </span><span class="NLM_article-title">Serial femtosecond crystallography of G protein-coupled receptors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">1521</span><span class="NLM_x">–</span> <span class="NLM_lpage">1524</span><span class="refDoi"> DOI: 10.1126/science.1244142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1126%2Fscience.1244142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=24357322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2013&pages=1521-1524&author=W.+Liuauthor=D.+Wackerauthor=C.+Gatiauthor=G.+W.+Hanauthor=D.+Jamesauthor=D.+Wangauthor=G.+Nelsonauthor=U.+Weierstallauthor=V.+Katritchauthor=A.+Bartyauthor=N.+A.+Zatsepinauthor=D.+Liauthor=M.+Messerschmidtauthor=S.+Boutetauthor=G.+J.+Williamsauthor=J.+E.+Koglinauthor=M.+M.+Seibertauthor=C.+Wangauthor=S.+T.+Shahauthor=S.+Basuauthor=R.+Frommeauthor=C.+Kupitzauthor=K.+N.+Rendekauthor=I.+Grotjohannauthor=P.+Frommeauthor=R.+A.+Kirianauthor=K.+R.+Beyerleinauthor=T.+A.+Whiteauthor=H.+N.+Chapmanauthor=M.+Caffreyauthor=J.+C.+Spenceauthor=R.+C.+Stevensauthor=V.+Cherezov&title=Serial+femtosecond+crystallography+of+G+protein-coupled+receptors&doi=10.1126%2Fscience.1244142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Serial Femtosecond Crystallography of G Protein-Coupled Receptors</span></div><div class="casAuthors">Liu, Wei; Wacker, Daniel; Gati, Cornelius; Han, Gye Won; James, Daniel; Wang, Dingjie; Nelson, Garrett; Weierstall, Uwe; Katritch, Vsevolod; Barty, Anton; Zatsepin, Nadia A.; Li, Dianfan; Messerschmidt, Marc; Boutet, Sebastien; Williams, Garth J.; Koglin, Jason E.; Seibert, M. Marvin; Wang, Chong; Shah, Syed T. A.; Basu, Shibom; Fromme, Raimund; Kupitz, Christopher; Rendek, Kimberley N.; Grotjohann, Ingo; Fromme, Petra; Kirian, Richard A.; Beyerlein, Kenneth R.; White, Thomas A.; Chapman, Henry N.; Caffrey, Martin; Spence, John C. H.; Stevens, Raymond C.; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">6165</span>),
    <span class="NLM_cas:pages">1521-1524</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">X-ray crystallog. of G protein-coupled receptors and other membrane proteins is hampered by difficulties assocd. with growing sufficiently large crystals that withstand radiation damage and yield high-resoln. data at synchrotron sources.  The authors used an x-ray free-electron laser (XFEL) with individual 50-fs-duration x-ray pulses to minimize radiation damage and obtained a high-resoln. room-temp. structure of a human serotonin receptor using sub-10-μm microcrystals grown in a membrane mimetic matrix known as lipidic cubic phase.  Compared with the structure solved by using traditional microcrystallog. from cryo-cooled crystals of about two orders of magnitude larger vol., the room-temp. XFEL structure displays a distinct distribution of thermal motions and conformations of residues that likely more accurately represent the receptor structure and dynamics in a cellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5GiymqAvRprVg90H21EOLACvtfcHk0li4rNicKVCqRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE&md5=a632160c4265cddfebb28ef9b002b38a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244142%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DKoglin%26aufirst%3DJ.%2BE.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShah%26aufirst%3DS.%2BT.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DRendek%26aufirst%3DK.%2BN.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DKirian%26aufirst%3DR.%2BA.%26aulast%3DBeyerlein%26aufirst%3DK.%2BR.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DSerial%2520femtosecond%2520crystallography%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DScience%26date%3D2013%26volume%3D342%26spage%3D1521%26epage%3D1524%26doi%3D10.1126%2Fscience.1244142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Rothman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savage, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauser, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBride, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Evidence for possible involvement of 5-HT<sub>2B</sub> receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span> <span class="citation_source-journal">Circulation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">2836</span><span class="NLM_x">–</span> <span class="NLM_lpage">2841</span><span class="refDoi"> DOI: 10.1161/01.CIR.102.23.2836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1161%2F01.CIR.102.23.2836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=11104741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2000&pages=2836-2841&author=R.+B.+Rothmanauthor=M.+H.+Baumannauthor=J.+E.+Savageauthor=L.+Rauserauthor=A.+McBrideauthor=S.+J.+Hufeisenauthor=B.+L.+Roth&title=Evidence+for+possible+involvement+of+5-HT2B+receptors+in+the+cardiac+valvulopathy+associated+with+fenfluramine+and+other+serotonergic+medications&doi=10.1161%2F01.CIR.102.23.2836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for possible involvement of 5-HT2B receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span></div><div class="casAuthors">Rothman, Richard B.; Baumann, Michael H.; Savage, Jason E.; Rauser, Laura; McBride, Ace; Hufeisen, Sandra J.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2836-2841</span>CODEN:
                <span class="NLM_cas:coden">CIRCAZ</span>;
        ISSN:<span class="NLM_cas:issn">0009-7322</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Serotonergic medications with various mechanisms of action are used to treat psychiatric disorders and are being investigated as treatments for drug dependence.  The occurrence of fenfluramine-assocd. valvular heart disease (VHD) has raised concerns that other serotonergic medications might also increase the risk of developing VHD.  We hypothesized that fenfluramine or its metabolite norfenfluramine and other medications known to produce VHD have preferentially high affinities for a particular serotonin receptor subtype capable of stimulating mitogenesis.  Medications known or suspected to cause VHD (pos. controls) and medications not assocd. with VHD (neg. controls) were screened for activity at 11 cloned serotonin receptor subtypes by use of ligand-binding methods and functional assays.  The pos. control drugs were (±)-fenfluramine; (+)-fenfluramine; (-)-fenfluramine; its metabolites (±)-norfenfluramine, (+)-norfenfluramine, and (-)-norfenfluramine; ergotamine; and methysergide and its metabolite methylergonovine.  The neg. control drugs were phentermine, fluoxetine, its metabolite norfluoxetine, and trazodone and its active metabolite m-chlorophenylpiperazine.  (±)-, (+)-, And (-)-Norfenfluramine, ergotamine, and methylergonovine all had preferentially high affinities for the cloned human serotonin 5-HT2B receptor and were partial to full agonists at the 5-HT2B receptor.  Our data imply that activation of 5-HT2B receptors is necessary to produce VHD and that serotonergic medications that do not activate 5-HT2B receptors are unlikely to produce VHD.  We suggest that all clin. available medications with serotonergic activity and their active metabolites be screened for agonist activity at 5-HT2B receptors and that clinicians should consider suspending their use of medications with significant activity at 5-HT2B receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6spUXOCLtLrVg90H21EOLACvtfcHk0li4rNicKVCqRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D&md5=6b8c6059885dd9b058859089914c61b3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1161%2F01.CIR.102.23.2836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.CIR.102.23.2836%26sid%3Dliteratum%253Aachs%26aulast%3DRothman%26aufirst%3DR.%2BB.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26aulast%3DSavage%26aufirst%3DJ.%2BE.%26aulast%3DRauser%26aufirst%3DL.%26aulast%3DMcBride%26aufirst%3DA.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DEvidence%2520for%2520possible%2520involvement%2520of%25205-HT2B%2520receptors%2520in%2520the%2520cardiac%2520valvulopathy%2520associated%2520with%2520fenfluramine%2520and%2520other%2520serotonergic%2520medications%26jtitle%3DCirculation%26date%3D2000%26volume%3D102%26spage%3D2836%26epage%3D2841%26doi%3D10.1161%2F01.CIR.102.23.2836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Hutcheson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merryman, W. D.</span><span> </span><span class="NLM_article-title">Serotonin receptors and heart valve disease--it was meant 2B</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">157</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.pharmthera.2011.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=21440001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaksLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2011&pages=146-157&author=J.+D.+Hutchesonauthor=V.+Setolaauthor=B.+L.+Rothauthor=W.+D.+Merryman&title=Serotonin+receptors+and+heart+valve+disease%2D%2Dit+was+meant+2B&doi=10.1016%2Fj.pharmthera.2011.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Serotonin receptors and heart valve disease-It was meant 2B</span></div><div class="casAuthors">Hutcheson, Joshua D.; Setola, Vincent; Roth, Bryan L.; Merryman, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-157</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Carcinoid heart disease was one of the first valvular pathologies studied in mol. detail, and early research identified serotonin produced by oncogenic enterochromaffin cells as the likely culprit in causing changes in heart valve tissue.  Researchers and physicians in the mid-1960s noted a connection between the use of several ergot-derived medications with structures similar to serotonin and the development of heart valve pathologies similar to those obsd. in carcinoid patients.  The exact serotonergic target that mediated valvular pathogenesis remained a mystery for many years until similar cases were reported in patients using the popular diet drug Fen-Phen in the late 1990s.  The Fen-Phen episode sparked renewed interest in serotonin-mediated valve disease, and studies led to the identification of the 5-HT2B receptor as the likely mol. target leading to heart valve tissue fibrosis.  Subsequent studies have identified numerous other activators of the 5-HT2B receptor, and consequently, the use of many of these mols. has been linked to heart valve disease.  Herein, we: review the mol. properties of the 5-HT2B receptor including factors that differentiate the 5-HT2B receptor from other 5-HT receptor subtypes, discuss the studies that led to the identification of the 5-HT2B receptor as the mediator of heart valve disease, present current efforts to identify potential valvulopathogens by screening for 5-HT2B receptor activity, and speculate on potential therapeutic benefits of 5-HT2B receptor targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4R8A9Ia-r3LVg90H21EOLACvtfcHk0lg2NCkGxSbWgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaksLnO&md5=b97c1992208a501b1959c2f48f3ca9cd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DHutcheson%26aufirst%3DJ.%2BD.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DMerryman%26aufirst%3DW.%2BD.%26atitle%3DSerotonin%2520receptors%2520and%2520heart%2520valve%2520disease--it%2520was%2520meant%25202B%26jtitle%3DPharmacol.%2520Ther.%26date%3D2011%26volume%3D132%26spage%3D146%26epage%3D157%26doi%3D10.1016%2Fj.pharmthera.2011.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Brea, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro-Palomino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeste, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cubero, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parraga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominguez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loza, M. I.</span><span> </span><span class="NLM_article-title">Emerging opportunities and concerns for drug discovery at serotonin 5-HT<sub>2B</sub> receptors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">493</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.2174/156802610791111524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.2174%2F156802610791111524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=20166944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFKntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=493-503&author=J.+Breaauthor=J.+Castro-Palominoauthor=S.+Yesteauthor=E.+Cuberoauthor=A.+Parragaauthor=E.+Dominguezauthor=M.+I.+Loza&title=Emerging+opportunities+and+concerns+for+drug+discovery+at+serotonin+5-HT2B+receptors&doi=10.2174%2F156802610791111524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging opportunities and concerns for drug discovery at serotonin 5-HT2B receptors</span></div><div class="casAuthors">Brea, Jose; Castro-Palomino, Julio; Yeste, Sandra; Cubero, Elena; Parraga, Antonio; Dominguez, Eduardo; Loza, Maria Isabel</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">493-503</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  5-HT2B receptors have been reported to play an important role at cardiac, intestinal and central levels, as well as in bone marrow formation and growth.  In the last decade, 5-HT2B receptors have also gained much attention as new targets in therapeutics, but also as off-targets because their activation along with the inhibition of serotonin transporters plays a significant role in the pathogenesis of 5-HT induced valvulopathy.  Taking this into account, the present review focuses on the new therapeutic applications of 5-HT2B receptor ligands as well as on the potential concerns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB-wKEI1AFFLVg90H21EOLACvtfcHk0ljJsr9kfIQ69g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFKntbk%253D&md5=6c92f93e70cc97f377881d198d9eaf16</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F156802610791111524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610791111524%26sid%3Dliteratum%253Aachs%26aulast%3DBrea%26aufirst%3DJ.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%26aulast%3DYeste%26aufirst%3DS.%26aulast%3DCubero%26aufirst%3DE.%26aulast%3DParraga%26aufirst%3DA.%26aulast%3DDominguez%26aufirst%3DE.%26aulast%3DLoza%26aufirst%3DM.%2BI.%26atitle%3DEmerging%2520opportunities%2520and%2520concerns%2520for%2520drug%2520discovery%2520at%2520serotonin%25205-HT2B%2520receptors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D493%26epage%3D503%26doi%3D10.2174%2F156802610791111524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Panconesi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicuteri, R.</span><span> </span><span class="NLM_article-title">Headache induced by serotonergic agonists--a key to the interpretation of migraine pathogenesis?</span> <span class="citation_source-journal">Cephalalgia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.1046/j.1468-2982.1997.1701003.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1046%2Fj.1468-2982.1997.1701003.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=3-14&author=A.+Panconesiauthor=R.+Sicuteri&title=Headache+induced+by+serotonergic+agonists%2D%2Da+key+to+the+interpretation+of+migraine+pathogenesis%3F&doi=10.1046%2Fj.1468-2982.1997.1701003.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1046%2Fj.1468-2982.1997.1701003.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1468-2982.1997.1701003.x%26sid%3Dliteratum%253Aachs%26aulast%3DPanconesi%26aufirst%3DA.%26aulast%3DSicuteri%26aufirst%3DR.%26atitle%3DHeadache%2520induced%2520by%2520serotonergic%2520agonists--a%2520key%2520to%2520the%2520interpretation%2520of%2520migraine%2520pathogenesis%253F%26jtitle%3DCephalalgia%26date%3D1997%26volume%3D17%26spage%3D3%26epage%3D14%26doi%3D10.1046%2Fj.1468-2982.1997.1701003.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Mucke, H.</span><span> </span><span class="NLM_article-title">MT-500 (POZEN)</span> <span class="citation_source-journal">Curr. Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">421</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=419-421&author=H.+Mucke&title=MT-500+%28POZEN%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMucke%26aufirst%3DH.%26atitle%3DMT-500%2520%2528POZEN%2529%26jtitle%3DCurr.%2520Opin.%2520Invest.%2520Drugs%26date%3D2001%26volume%3D2%26spage%3D419%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Borman, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilford, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmer, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheldrick, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, G. S.</span><span> </span><span class="NLM_article-title">5-HT<sub>2B</sub> receptors play a key role in mediating the excitatory effects of 5-HT in human colon in vitro</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">1144</span><span class="NLM_x">–</span> <span class="NLM_lpage">1151</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0704571</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fsj.bjp.0704571" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=11877320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1ait7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2002&pages=1144-1151&author=R.+A.+Bormanauthor=N.+S.+Tilfordauthor=D.+W.+Harmerauthor=N.+Dayauthor=E.+S.+Ellisauthor=R.+L.+Sheldrickauthor=J.+Careyauthor=R.+A.+Colemanauthor=G.+S.+Baxter&title=5-HT2B+receptors+play+a+key+role+in+mediating+the+excitatory+effects+of+5-HT+in+human+colon+in+vitro&doi=10.1038%2Fsj.bjp.0704571"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">5-HT2B receptors play a key role in mediating the excitatory effects of 5-HT in human colon in vitro</span></div><div class="casAuthors">Borman, R. A.; Tilford, N. S.; Harmer, D. W.; Day, N.; Ellis, E. S.; Sheldrick, R. L. G.; Carey, J.; Coleman, R. A.; Baxter, G. S.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1144-1151</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">5-Hydroxytryptamine (5-HT) is known to produce a no. of different effects in the gastrointestinal tract of various species, and has been proposed to play a key role in a no. of intestinal disorders in man, including irritable bowel syndrome (IBS), although the receptors involved have yet to be established.  The aim of the present study was to investigate the distribution and function of 5-HT2B receptors in human colon, and to establish their possible role in the etiol. of IBS.  The distribution of 5-HT2B receptor mRNA and protein were investigated by quant. RT-PCR, Western anal. and immunocytochem.  High levels of both mRNA and protein for 5-HT2B receptors were found throughout the human gastrointestinal tract, and in particular in colon, where 5-HT2B receptors were found predominantly in the longitudinal and circular smooth muscle layers within the muscularis externa, and in the myenteric nerve plexus lying between these two layers.  Elec. field stimulation of longitudinal muscle prepns. of human colon mounted in organ baths resulted in neuronally mediated contractile responses, that were significantly potentiated by application of 5-HT (up to 10-7 M), with a pEC50 of 8.2.  The response to 5-HT was inhibited by a no. of selective 5-HT2B receptor antagonists.  This study has shown for the first time that, in contrast to animal studies, the excitatory effects of 5-HT in human colon are mediated by 5-HT2B receptors.  It is proposed that these receptors contribute to the putative 5-HT-induced colonic smooth muscle hypersensitivity assocd. with IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruU5R-UDwGebVg90H21EOLACvtfcHk0lhaL-q2uP1OhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1ait7k%253D&md5=33fb1447a886d3313bc4aa0b2e8872e5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0704571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0704571%26sid%3Dliteratum%253Aachs%26aulast%3DBorman%26aufirst%3DR.%2BA.%26aulast%3DTilford%26aufirst%3DN.%2BS.%26aulast%3DHarmer%26aufirst%3DD.%2BW.%26aulast%3DDay%26aufirst%3DN.%26aulast%3DEllis%26aufirst%3DE.%2BS.%26aulast%3DSheldrick%26aufirst%3DR.%2BL.%26aulast%3DCarey%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BA.%26aulast%3DBaxter%26aufirst%3DG.%2BS.%26atitle%3D5-HT2B%2520receptors%2520play%2520a%2520key%2520role%2520in%2520mediating%2520the%2520excitatory%2520effects%2520of%25205-HT%2520in%2520human%2520colon%2520in%2520vitro%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2002%26volume%3D135%26spage%3D1144%26epage%3D1151%26doi%3D10.1038%2Fsj.bjp.0704571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Spiller, R.</span><span> </span><span class="NLM_article-title">Serotonergic agents and the irritable bowel syndrome: what goes wrong?</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">709</span><span class="NLM_x">–</span> <span class="NLM_lpage">714</span><span class="refDoi"> DOI: 10.1016/j.coph.2008.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.coph.2008.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=18672092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2qurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=709-714&author=R.+Spiller&title=Serotonergic+agents+and+the+irritable+bowel+syndrome%3A+what+goes+wrong%3F&doi=10.1016%2Fj.coph.2008.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Serotonergic agents and the irritable bowel syndrome: what goes wrong?</span></div><div class="casAuthors">Spiller, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">709-714</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  It is now nine years since the first large randomized controlled trials showed the benefit of a 5HT3 antagonist in irritable bowel syndrome (IBS) with diarrhea (IBS-D) and a 5HT4 partial agonist in IBS with constipation (IBS-C).  Although both of these drugs have now been withdrawn because of rare complications (ischemic colitis and thrombotic episodes resp.), their clin. effectiveness has stimulated substantial advances in our understanding of the physiol. role of serotonin and its disturbance in IBS. 5HT-contg. enteroendocrine cells are most numerous in the duodenum and colon and are responsive to a range of stimuli including tastants, mechanic force and short chain fatty acids.  Several studies have described an excess of 5HT in IBS-D and abnormally low 5HT availability in IBS-C.  Serotonin transporter (SERT) is only weakly expressed in the colon and whether this is further reduced in IBS is unclear with conflicting reports.  SERT promoter polymorphisms are inconsistently related to IBS subtypes but appear to partly predict response to both 5HT3 antagonists and 5HT4 agonists.  Stressors release serotonin and 5HT3 antagonists have been shown to inhibit the assocd. acceleration of colonic transit.  Newer 5HT3 antagonists and 5HT4 agonists are under development.  Their undoubted effectiveness in subgroups of IBS should stimulate further work to define biomarkers of both responsiveness and of risk of developing adverse events to improve the risk benefit balance for these potentially useful therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEsYzpKY4CtrVg90H21EOLACvtfcHk0lhaL-q2uP1OhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2qurnE&md5=dbcbe1675441b53b62ad5ec40b3053bc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2008.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2008.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DSpiller%26aufirst%3DR.%26atitle%3DSerotonergic%2520agents%2520and%2520the%2520irritable%2520bowel%2520syndrome%253A%2520what%2520goes%2520wrong%253F%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2008%26volume%3D8%26spage%3D709%26epage%3D714%26doi%3D10.1016%2Fj.coph.2008.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Launay, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herve, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peoc’h, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tournois, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callebert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebigil, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etienne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drouet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humbert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonneau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maroteaux, L.</span><span> </span><span class="NLM_article-title">Function of the serotonin 5-hydroxytryptamine 2B receptor in pulmonary hypertension</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1135</span><span class="refDoi"> DOI: 10.1038/nm764</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnm764" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=12244304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGrt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2002&pages=1129-1135&author=J.+M.+Launayauthor=P.+Herveauthor=K.+Peoc%E2%80%99hauthor=C.+Tournoisauthor=J.+Callebertauthor=C.+G.+Nebigilauthor=N.+Etienneauthor=L.+Drouetauthor=M.+Humbertauthor=G.+Simonneauauthor=L.+Maroteaux&title=Function+of+the+serotonin+5-hydroxytryptamine+2B+receptor+in+pulmonary+hypertension&doi=10.1038%2Fnm764"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Function of the serotonin 5-hydroxytryptamine 2B receptor in pulmonary hypertension</span></div><div class="casAuthors">Launay, J.-M.; Herve, P.; Peoc'h, K.; Tournois, C.; Callebert, J.; Nebigil, C. G.; Etienne, N.; Drouet, L.; Humbert, M.; Simonneau, G.; Maroteaux, L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1129-1135</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Primary pulmonary hypertension is a progressive and often fatal disorder in humans that results from an increase in pulmonary blood pressure assocd. with abnormal vascular proliferation.  Dexfenfluramine increases the risk of pulmonary hypertension in humans, and its active metabolite is a selective serotonin 5-hydroxytryptamine 2B (5-HT2B) receptor agonist.  Thus, the authors investigated the contribution of the 5-HT2B receptor to the pathogenesis of pulmonary hypertension.  Using the chronic-hypoxic-mouse model of pulmonary hypertension, the authors found that the hypoxia-dependent increase in pulmonary blood pressure and lung remodeling are assocd. with an increase in vascular proliferation, elastase activity and transforming growth factor-β levels, and that these parameters are potentiated by dexfenfluramine treatment.  In contrast, hypoxic mice with genetically or pharmacol. inactive 5-HT2B receptors manifested no change in any of these parameters.  In both humans and mice, pulmonary hypertension is assocd. with a substantial increase in 5-HT2B receptor expression in pulmonary arteries.  These data show that activation of 5-HT2B receptors is a limiting step in the development of pulmonary hypertension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP9zEi5XRykbVg90H21EOLACvtfcHk0lgIhnq9G-JzTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGrt7Y%253D&md5=01bde279d3503f827598478602ccec25</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnm764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm764%26sid%3Dliteratum%253Aachs%26aulast%3DLaunay%26aufirst%3DJ.%2BM.%26aulast%3DHerve%26aufirst%3DP.%26aulast%3DPeoc%25E2%2580%2599h%26aufirst%3DK.%26aulast%3DTournois%26aufirst%3DC.%26aulast%3DCallebert%26aufirst%3DJ.%26aulast%3DNebigil%26aufirst%3DC.%2BG.%26aulast%3DEtienne%26aufirst%3DN.%26aulast%3DDrouet%26aufirst%3DL.%26aulast%3DHumbert%26aufirst%3DM.%26aulast%3DSimonneau%26aufirst%3DG.%26aulast%3DMaroteaux%26aufirst%3DL.%26atitle%3DFunction%2520of%2520the%2520serotonin%25205-hydroxytryptamine%25202B%2520receptor%2520in%2520pulmonary%2520hypertension%26jtitle%3DNat.%2520Med.%26date%3D2002%26volume%3D8%26spage%3D1129%26epage%3D1135%26doi%3D10.1038%2Fnm764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Porvasnik, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embury, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gannon, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shacham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Mousily, F.</span><span> </span><span class="NLM_article-title">PRX-08066, a novel 5-hydroxytryptamine receptor 2B antagonist, reduces monocrotaline-induced pulmonary arterial hypertension and right ventricular hypertrophy in rats</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1124/jpet.109.165001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1124%2Fjpet.109.165001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=20430844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1KisLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2010&pages=364-372&author=S.+L.+Porvasnikauthor=S.+Germainauthor=J.+Emburyauthor=K.+S.+Gannonauthor=V.+Jacquesauthor=J.+Murrayauthor=B.+J.+Byrneauthor=S.+Shachamauthor=F.+Al-Mousily&title=PRX-08066%2C+a+novel+5-hydroxytryptamine+receptor+2B+antagonist%2C+reduces+monocrotaline-induced+pulmonary+arterial+hypertension+and+right+ventricular+hypertrophy+in+rats&doi=10.1124%2Fjpet.109.165001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">PRX-08066, a novel 5-hydroxytryptamine receptor 2B antagonist, reduces monocrotaline-induced pulmonary arterial hypertension and right ventricular hypertrophy in rats</span></div><div class="casAuthors">Porvasnik, Stacy L.; Germain, Sean; Embury, Jennifer; Gannon, Kimberley S.; Jacques, Vincent; Murray, Justin; Byrne, Barry J.; Shacham, Sharon; Al-Mousily, Faris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">364-372</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Pulmonary arterial hypertension (PAH) is a life-threatening disease that results in right ventricular failure.  5-((4-(6-Chlorothieno[2,3-d]pyrimidin-4-ylamino)piperidin-1-yl)methyl)-2-fluorobenzonitrile monofumarate (PRX-08066) is a selective 5-hydroxytryptamine receptor 2B (5-HT2BR) antagonist that causes selective vasodilation of pulmonary arteries.  In the current study, the effects of PRX-08066 were assessed by using the monocrotaline (MCT)-induced PAH rat model.  Male rats received 40 mg/kg MCT or phosphate-buffered saline and were treated orally twice a day with vehicle or 50 or 100 mg/kg PRX-08066 for 5 wk.  Pulmonary and cardiac functions were evaluated by hemodynamics, heart wt., magnetic resonance imaging (MRI), pulmonary artery (PA) morphol., and histol.  Cardiac MRI demonstrated that PRX-08066 (100 mg/kg) significantly (P < 0.05) improved right ventricular ejection fraction.  PRX-08066 significantly reduced peak PA pressure at 50 and 100 mg/kg (P < 0.05 and < 0.01, resp.) compared with MCT control animals.  PRX-08066 therapy also significantly reduced right ventricle (RV)/body wt. and RV/left ventricle + septum (P < 0.01 and < 0.001, resp.) compared with MCT-treated animals.  Morphometric assessment of pulmonary arterioles revealed a significant redn. in medial wall thickening and lumen occlusion assocd. with both doses of PRX-08066 (P < 0.01).  The 5-HT2BR antagonist PRX-08066 significantly attenuated the elevation in PA pressure and RV hypertrophy and maintained cardiac function.  Pulmonary vascular remodeling was also diminished compared with MCT control rats.  PRX-08066 prevents the severity of PAH in the MCT rat model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_CaIqQmpIS7Vg90H21EOLACvtfcHk0lgIhnq9G-JzTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1KisLbJ&md5=c00235e9c830eb3a3937fa3696dac88b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fjpet.109.165001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.109.165001%26sid%3Dliteratum%253Aachs%26aulast%3DPorvasnik%26aufirst%3DS.%2BL.%26aulast%3DGermain%26aufirst%3DS.%26aulast%3DEmbury%26aufirst%3DJ.%26aulast%3DGannon%26aufirst%3DK.%2BS.%26aulast%3DJacques%26aufirst%3DV.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DByrne%26aufirst%3DB.%2BJ.%26aulast%3DShacham%26aufirst%3DS.%26aulast%3DAl-Mousily%26aufirst%3DF.%26atitle%3DPRX-08066%252C%2520a%2520novel%25205-hydroxytryptamine%2520receptor%25202B%2520antagonist%252C%2520reduces%2520monocrotaline-induced%2520pulmonary%2520arterial%2520hypertension%2520and%2520right%2520ventricular%2520hypertrophy%2520in%2520rats%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2010%26volume%3D334%26spage%3D364%26epage%3D372%26doi%3D10.1124%2Fjpet.109.165001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Soares, R. L.</span><span> </span><span class="NLM_article-title">Irritable bowel syndrome: A clinical review</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">12144</span><span class="NLM_x">–</span> <span class="NLM_lpage">12160</span><span class="refDoi"> DOI: 10.3748/wjg.v20.i34.12144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.3748%2Fwjg.v20.i34.12144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=25232249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ktleltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=12144-12160&author=R.+L.+Soares&title=Irritable+bowel+syndrome%3A+A+clinical+review&doi=10.3748%2Fwjg.v20.i34.12144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Irritable bowel syndrome: a clinical review</span></div><div class="casAuthors">Soares Rosa L S</div><div class="citationInfo"><span class="NLM_cas:title">World journal of gastroenterology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12144-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Irritable bowel syndrome (IBS) remains a clinical challenge in the 21(st) century.  It's the most commonly diagnosed gastrointestinal condition and also the most common reason for referral to gastroenterology clinics.  Its can affect up to one in five people at some point in their lives, and has a significantly impact of life quality and health care utilization.  The prevalence varies according to country and criteria used to define IBS.  Various mechanisms and theories have been proposed about its etiology, but the biopsychosocial model is the most currently accepted for IBS.  The complex of symptoms would be the result of the interaction between psychological, behavioral, psychosocial and environmental factors.  The diagnosis of IBS is not confirmed by a specific test or structural abnormality.  It is made using criteria based on clinical symptoms such as Rome criteria, unless the symptoms are thought to be atypical.  Today the Rome Criteria III is the current gold-standard for the diagnoses of IBS.  Secure positive evidence of IBS by means of specific disease marker is currently not possible and cannot be currently recommended for routine diagnosis.  There is still no clinical evidence to recommend the use of biomarkers in blood to diagnose IBS.  However, a number of different changes in IBS patients were demonstrated in recent years, some of which can be used in the future as a diagnostic support.  IBS has no definitive treatment but could be controlled by non-pharmacologic management eliminating of some exacerbating factors such certain drugs, stressor conditions and changes in dietary habits.The traditional pharmacologic management of IBS has been symptom based and several drugs have been used.  However, the cornerstone of its therapy is a solid patient physician relationship.  This review will provide a summary of pathophysiology, diagnostic criteria and current and emerging therapies for IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQiMlaT5g6za2Zyxfu64m4ffW6udTcc2eZSr_Jh3Hy3jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ktleltQ%253D%253D&md5=2cc95ea97567a60537ad0b52d04180fb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v20.i34.12144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v20.i34.12144%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DR.%2BL.%26atitle%3DIrritable%2520bowel%2520syndrome%253A%2520A%2520clinical%2520review%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2014%26volume%3D20%26spage%3D12144%26epage%3D12160%26doi%3D10.3748%2Fwjg.v20.i34.12144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Occhipinti, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. W.</span><span> </span><span class="NLM_article-title">Irritable bowel syndrome: a review and update</span> <span class="citation_source-journal">Clin. Colon Rectal Surg.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1055/s-0032-1301759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1055%2Fs-0032-1301759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23449495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A280%3ADC%252BC3svit12hsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=46-52&author=K.+Occhipintiauthor=J.+W.+Smith&title=Irritable+bowel+syndrome%3A+a+review+and+update&doi=10.1055%2Fs-0032-1301759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Irritable bowel syndrome: a review and update</span></div><div class="casAuthors">Occhipinti Kaitlin; Smith James W</div><div class="citationInfo"><span class="NLM_cas:title">Clinics in colon and rectal surgery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-52</span>
        ISSN:<span class="NLM_cas:issn">1531-0043</span>.
    </div><div class="casAbstract">The understanding of irritable bowel syndrome (IBS) has undergone a rapid evolution with scientific advancement.  IBS is a common functional bowel disorder that generates a significant health care burden and is the most commonly diagnosed gastrointestinal condition.  There are well-established diagnostic criteria and algorithms for the initial evaluation of patients presenting with the symptoms of IBS.  The symptoms can be targeted for therapy with a variety of pharmaceutical and nonpharmaceutical agents.  Therapy should be individualized for the patient, and the cornerstone for any effective treatment strategy should be the solid patient-physician relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTd2CLvsyvO3_6xYuD7tu2XfW6udTcc2eZSr_Jh3Hy3jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svit12hsg%253D%253D&md5=edf8a4c80999a907b7922f4b3638cb44</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1055%2Fs-0032-1301759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0032-1301759%26sid%3Dliteratum%253Aachs%26aulast%3DOcchipinti%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%2BW.%26atitle%3DIrritable%2520bowel%2520syndrome%253A%2520a%2520review%2520and%2520update%26jtitle%3DClin.%2520Colon%2520Rectal%2520Surg.%26date%3D2012%26volume%3D25%26spage%3D46%26epage%3D52%26doi%3D10.1055%2Fs-0032-1301759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Ashburn, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, M. S.</span><span> </span><span class="NLM_article-title">The IBS market</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">100</span><span class="refDoi"> DOI: 10.1038/nrd1961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnrd1961" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=99-100&author=T.+T.+Ashburnauthor=M.+S.+Gupta&title=The+IBS+market&doi=10.1038%2Fnrd1961"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd1961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1961%26sid%3Dliteratum%253Aachs%26aulast%3DAshburn%26aufirst%3DT.%2BT.%26aulast%3DGupta%26aufirst%3DM.%2BS.%26atitle%3DThe%2520IBS%2520market%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D99%26epage%3D100%26doi%3D10.1038%2Fnrd1961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Mullard, A.</span><span> </span><span class="NLM_article-title">FDA approves two IBS drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">449</span><span class="refDoi"> DOI: 10.1038/nrd4688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnrd4688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=449&author=A.+Mullard&title=FDA+approves+two+IBS+drugs&doi=10.1038%2Fnrd4688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd4688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4688%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFDA%2520approves%2520two%2520IBS%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D449%26doi%3D10.1038%2Fnrd4688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Wouters, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tharayil, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanich, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strege, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardsley, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ordog, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbons, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrugia, G.</span><span> </span><span class="NLM_article-title">Protein kinase C<sub>γ</sub> mediates regulation of proliferation by the serotonin 5-hydroxytryptamine receptor 2B</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">21177</span><span class="NLM_x">–</span> <span class="NLM_lpage">21184</span><span class="refDoi"> DOI: 10.1074/jbc.M109.015859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1074%2Fjbc.M109.015859" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=21177-21184&author=M.+M.+Woutersauthor=J.+L.+Roederauthor=V.+S.+Tharayilauthor=J.+E.+Stanichauthor=P.+R.+Stregeauthor=S.+Leiauthor=M.+R.+Bardsleyauthor=T.+Ordogauthor=S.+J.+Gibbonsauthor=G.+Farrugia&title=Protein+kinase+C%CE%B3+mediates+regulation+of+proliferation+by+the+serotonin+5-hydroxytryptamine+receptor+2B&doi=10.1074%2Fjbc.M109.015859"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.015859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.015859%26sid%3Dliteratum%253Aachs%26aulast%3DWouters%26aufirst%3DM.%2BM.%26aulast%3DRoeder%26aufirst%3DJ.%2BL.%26aulast%3DTharayil%26aufirst%3DV.%2BS.%26aulast%3DStanich%26aufirst%3DJ.%2BE.%26aulast%3DStrege%26aufirst%3DP.%2BR.%26aulast%3DLei%26aufirst%3DS.%26aulast%3DBardsley%26aufirst%3DM.%2BR.%26aulast%3DOrdog%26aufirst%3DT.%26aulast%3DGibbons%26aufirst%3DS.%2BJ.%26aulast%3DFarrugia%26aufirst%3DG.%26atitle%3DProtein%2520kinase%2520C%25CE%25B3%2520mediates%2520regulation%2520of%2520proliferation%2520by%2520the%2520serotonin%25205-hydroxytryptamine%2520receptor%25202B%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D21177%26epage%3D21184%26doi%3D10.1074%2Fjbc.M109.015859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span> </span><span class="NLM_article-title">Effect of 5-HT<sub>2B</sub> receptor on the pathogenetic mechanism of irritable bowel syndrome subgroups in rat models</span> <span class="citation_source-journal">Acta Univ. Med. Nanjing</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=609-615&author=Y.+Chenauthor=Z.+Liauthor=Y.+Yangauthor=L.+Linauthor=H.+Zhang&title=Effect+of+5-HT2B+receptor+on+the+pathogenetic+mechanism+of+irritable+bowel+syndrome+subgroups+in+rat+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DEffect%2520of%25205-HT2B%2520receptor%2520on%2520the%2520pathogenetic%2520mechanism%2520of%2520irritable%2520bowel%2520syndrome%2520subgroups%2520in%2520rat%2520models%26jtitle%3DActa%2520Univ.%2520Med.%2520Nanjing%26date%3D2012%26volume%3D32%26spage%3D609%26epage%3D615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bassil, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolton, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summerfield, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winchester, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanger, G. J.</span><span> </span><span class="NLM_article-title">Inhibition of colonic motility and defecation by RS-127445 suggests an involvement of the 5-HT<sub>2B</sub> receptor in rodent large bowel physiology</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">258</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00155.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1111%2Fj.1476-5381.2009.00155.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=19371340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2009&pages=252-258&author=A.+Bassilauthor=C.+Taylorauthor=V.+Boltonauthor=K.+Grayauthor=J.+Brownauthor=L.+Cutlerauthor=S.+Summerfieldauthor=G.+Brutonauthor=W.+Winchesterauthor=K.+Leeauthor=G.+J.+Sanger&title=Inhibition+of+colonic+motility+and+defecation+by+RS-127445+suggests+an+involvement+of+the+5-HT2B+receptor+in+rodent+large+bowel+physiology&doi=10.1111%2Fj.1476-5381.2009.00155.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of colonic motility and defecation by RS-127445 suggests an involvement of the 5-HT2B receptor in rodent large bowel physiology</span></div><div class="casAuthors">Bassil, A. K.; Taylor, C. M.; Bolton, V. J. N.; Gray, K. M.; Brown, J. D.; Cutler, L.; Summerfield, S. G.; Bruton, G.; Winchester, W. J.; Lee, K.; Sanger, G. J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">252-258</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">5-HT2B receptors are localized within the myenteric nervous system, but their functions on motor/sensory neurons are unclear.  To explore the role of these receptors, we further characterized the 5-HT2B receptor antagonist RS-127445 and studied its effects on peristalsis and defecation.  Although reported as a selective 5-HT2B receptor antagonist, any interactions of RS-127445 with 5-HT4 receptors are unknown; this was examd. using the recombinant receptor and Biomol. Interaction Detection technol.  Mouse isolated colon was mounted in tissue baths for isometric recording of neuronal contractions evoked by elec. field stimulation (EFS), or under an intraluminal pressure gradient to induce peristalsis; the effects of RS-127445 on EFS-induced and on peristaltic contractions were measured.  Faecal output of rats in grid-bottom cages was measured over 3 h following i.p. RS-127445 and sep., validation of the EDs was achieved by detg. the free, unbound fraction of RS-127445 in blood and brain.  RS-127445 (up to 1 μmol/L-1) did not interact with the 5-HT4 receptor.  RS-127445 (0.001-1 μmol/L-1) did not affect EFS-induced contractions of the colon, although at 10 μmol/L-1 the contractions were reduced (to 36 ± 8% of control, n = 4).  RS-127445 (0.1-10 μmol/L-1) concn.-dependently reduced peristaltic frequency (n = 4).  RS-127445 (1-30 mg/kg-1), dose-dependently reduced faecal output, reaching significance at 10 and 30 mg/kg-1 (n = 6-11).  In blood and brain, >98% of RS-127445 was protein-bound.  High-protein binding of RS-127445 indicates that relatively high doses are required for efficacy.  The results suggest that 5-HT2B receptors tonically regulate colonic motility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5jssIHDufubVg90H21EOLACvtfcHk0lho6AYbqP6iEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr%252FK&md5=411ce4124f41fca8bdb468defcb19dfc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00155.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00155.x%26sid%3Dliteratum%253Aachs%26aulast%3DBassil%26aufirst%3DA.%26aulast%3DTaylor%26aufirst%3DC.%26aulast%3DBolton%26aufirst%3DV.%26aulast%3DGray%26aufirst%3DK.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DSummerfield%26aufirst%3DS.%26aulast%3DBruton%26aufirst%3DG.%26aulast%3DWinchester%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSanger%26aufirst%3DG.%2BJ.%26atitle%3DInhibition%2520of%2520colonic%2520motility%2520and%2520defecation%2520by%2520RS-127445%2520suggests%2520an%2520involvement%2520of%2520the%25205-HT2B%2520receptor%2520in%2520rodent%2520large%2520bowel%2520physiology%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D158%26spage%3D252%26epage%3D258%26doi%3D10.1111%2Fj.1476-5381.2009.00155.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">O’mahony, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulmer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coelho, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bongiovanni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winchester, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinan, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J.</span><span> </span><span class="NLM_article-title">5-HT<sub>2B</sub> receptors modulate visceral hypersensitivity in a stress-sensitive animal model of brain-gut axis dysfunction</span> <span class="citation_source-journal">Neurogastroenterol. Motil.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">573</span><span class="NLM_x">–</span> <span class="NLM_lpage">e124</span><span class="refDoi"> DOI: 10.1111/j.1365-2982.2009.01432.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1111%2Fj.1365-2982.2009.01432.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=20003079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslCitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=573-e124&author=S.+O%E2%80%99mahonyauthor=D.+Bulmerauthor=A.+Coelhoauthor=P.+Fitzgeraldauthor=C.+Bongiovanniauthor=K.+Leeauthor=W.+Winchesterauthor=T.+Dinanauthor=J.+Cryan&title=5-HT2B+receptors+modulate+visceral+hypersensitivity+in+a+stress-sensitive+animal+model+of+brain-gut+axis+dysfunction&doi=10.1111%2Fj.1365-2982.2009.01432.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">5-HT2B receptors modulate visceral hypersensitivity in a stress-sensitive animal model of brain-gut axis dysfunction</span></div><div class="casAuthors">O'Mahony, S. M.; Bulmer, D. C.; Coelho, A.-M.; Fitzgerald, P.; Bongiovanni, C.; Lee, K.; Winchester, W.; Dinan, T. G.; Cryan, J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Neurogastroenterology & Motility</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">573-578</span>CODEN:
                <span class="NLM_cas:coden">NMOTEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-1925</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Irritable bowel syndrome (IBS) is assocd. with an enhanced perception to visceral stimuli and exaggerated stress response.  The serotonergic neurotransmitter system has been strongly implicated as a key player in the manifestation of IBS symptomatol. including visceral hypersensitivity.  However the role of 5-HT2B receptors in visceral pain, although speculated, is currently unclear.  Thus we assessed the impact of a selective 5-HT2B receptor antagonist, RS-127445, on visceral hypersensitivity in a model of brain gut axis dysfunction the Wistar Kyoto (WKY) rat.  Colorectal distension (CRD) was used to assess the visceral sensitivity of the WKY rat compared to normosensitive Sprague Dawley (SD) rats.  Once we verified the visceral sensitivity of the WKY rat we assessed the efficacy of RS-127445 in pain signalling from the colorectum.  We administered the compd. peripherally (i.p.) and centrally (i.c.v.) in order to ascertain the site of action of RS 127445.  Behavioral responses to colorectal distention were then monitored.  The WKY rats were more viscerally hypersensitive than the SD as previously shown.  RS-127445 (5 mg/kg-1, i.p.) significantly reversed visceral hypersensitivity in WKY animals.  Moreover, when administered intracerebroventricularly RS-127445 (100 nM) also decreased the no. of pain behaviors during noxious CRD in the WKY animals.  Taken together, blockade of 5-HT2B receptors offers an exciting novel therapeutic target for pain relief in stress-related gastrointestinal disorders such as IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVsA5ZZFutFrVg90H21EOLACvtfcHk0liVIOca7KTuew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslCitLw%253D&md5=404d887aa980c14c5742be450de7dcba</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2982.2009.01432.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2982.2009.01432.x%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599mahony%26aufirst%3DS.%26aulast%3DBulmer%26aufirst%3DD.%26aulast%3DCoelho%26aufirst%3DA.%26aulast%3DFitzgerald%26aufirst%3DP.%26aulast%3DBongiovanni%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWinchester%26aufirst%3DW.%26aulast%3DDinan%26aufirst%3DT.%26aulast%3DCryan%26aufirst%3DJ.%26atitle%3D5-HT2B%2520receptors%2520modulate%2520visceral%2520hypersensitivity%2520in%2520a%2520stress-sensitive%2520animal%2520model%2520of%2520brain-gut%2520axis%2520dysfunction%26jtitle%3DNeurogastroenterol.%2520Motil.%26date%3D2010%26volume%3D22%26spage%3D573%26epage%3De124%26doi%3D10.1111%2Fj.1365-2982.2009.01432.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Ohashi-doi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himaki, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gale, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furness, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurebayashi, Y.</span><span> </span><span class="NLM_article-title">A selective, high affinity 5-HT<sub>2B</sub> receptor antagonist inhibits visceral hypersensitivity in rats</span> <span class="citation_source-journal">Neurogastroenterol. Motil.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">e69</span><span class="NLM_x">–</span> <span class="NLM_lpage">e76</span><span class="refDoi"> DOI: 10.1111/j.1365-2982.2009.01395.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1111%2Fj.1365-2982.2009.01395.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=e69-e76&author=K.+Ohashi-doiauthor=D.+Himakiauthor=K.+Nagaoauthor=M.+Kawaiauthor=J.+Galeauthor=J.+Furnessauthor=Y.+Kurebayashi&title=A+selective%2C+high+affinity+5-HT2B+receptor+antagonist+inhibits+visceral+hypersensitivity+in+rats&doi=10.1111%2Fj.1365-2982.2009.01395.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2982.2009.01395.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2982.2009.01395.x%26sid%3Dliteratum%253Aachs%26aulast%3DOhashi-doi%26aufirst%3DK.%26aulast%3DHimaki%26aufirst%3DD.%26aulast%3DNagao%26aufirst%3DK.%26aulast%3DKawai%26aufirst%3DM.%26aulast%3DGale%26aufirst%3DJ.%26aulast%3DFurness%26aufirst%3DJ.%26aulast%3DKurebayashi%26aufirst%3DY.%26atitle%3DA%2520selective%252C%2520high%2520affinity%25205-HT2B%2520receptor%2520antagonist%2520inhibits%2520visceral%2520hypersensitivity%2520in%2520rats%26jtitle%3DNeurogastroenterol.%2520Motil.%26date%3D2010%26volume%3D22%26spage%3De69%26epage%3De76%26doi%3D10.1111%2Fj.1365-2982.2009.01395.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Takahashi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inagaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taniguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakaguchi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, K.</span><span> </span><span class="NLM_article-title">The novel 5-HT<sub>2B</sub> receptor antagonist, RQ-00310941, attenuates visceral hypersensitivity and abnormal defecation in rat models</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x">, </span> <span class="NLM_fpage">S</span><span class="NLM_x">–</span> <span class="NLM_lpage">607</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2011&pages=S-607&author=N.+Takahashiauthor=K.+Inagakiauthor=K.+Taniguchiauthor=Y.+Sakaguchiauthor=K.+Kawamura&title=The+novel+5-HT2B+receptor+antagonist%2C+RQ-00310941%2C+attenuates+visceral+hypersensitivity+and+abnormal+defecation+in+rat+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DN.%26aulast%3DInagaki%26aufirst%3DK.%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DSakaguchi%26aufirst%3DY.%26aulast%3DKawamura%26aufirst%3DK.%26atitle%3DThe%2520novel%25205-HT2B%2520receptor%2520antagonist%252C%2520RQ-00310941%252C%2520attenuates%2520visceral%2520hypersensitivity%2520and%2520abnormal%2520defecation%2520in%2520rat%2520models%26jtitle%3DGastroenterology%26date%3D2011%26volume%3D140%26spage%3DS%26epage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Poissonnet, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parmentier, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutin, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, S.</span><span> </span><span class="NLM_article-title">The emergence of selective 5-HT<sub>2B</sub> antagonists structures, activities and potential therapeutic applications</span> <span class="citation_source-journal">Mini-Rev. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">330</span><span class="refDoi"> DOI: 10.2174/1389557043487312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.2174%2F1389557043487312" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=325-330&author=G.+Poissonnetauthor=J.+G.+Parmentierauthor=J.+A.+Boutinauthor=S.+Goldstein&title=The+emergence+of+selective+5-HT2B+antagonists+structures%2C+activities+and+potential+therapeutic+applications&doi=10.2174%2F1389557043487312"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.2174%2F1389557043487312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557043487312%26sid%3Dliteratum%253Aachs%26aulast%3DPoissonnet%26aufirst%3DG.%26aulast%3DParmentier%26aufirst%3DJ.%2BG.%26aulast%3DBoutin%26aufirst%3DJ.%2BA.%26aulast%3DGoldstein%26aufirst%3DS.%26atitle%3DThe%2520emergence%2520of%2520selective%25205-HT2B%2520antagonists%2520structures%252C%2520activities%2520and%2520potential%2520therapeutic%2520applications%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2004%26volume%3D4%26spage%3D325%26epage%3D330%26doi%3D10.2174%2F1389557043487312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Life beyond kinases: Structure-based discovery of sorafenib as nanomolar antagonist of 5-HT receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">–</span> <span class="NLM_lpage">5759</span><span class="refDoi"> DOI: 10.1021/jm300338m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300338m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=5749-5759&author=X.+Linauthor=X.-P.+Huangauthor=G.+Chenauthor=R.+Whaleyauthor=S.+Pengauthor=Y.+Wangauthor=G.+Zhangauthor=S.+X.+Wangauthor=S.+Wangauthor=B.+L.+Rothauthor=N.+Huang&title=Life+beyond+kinases%3A+Structure-based+discovery+of+sorafenib+as+nanomolar+antagonist+of+5-HT+receptors&doi=10.1021%2Fjm300338m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm300338m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300338m%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DS.%2BX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DLife%2520beyond%2520kinases%253A%2520Structure-based%2520discovery%2520of%2520sorafenib%2520as%2520nanomolar%2520antagonist%2520of%25205-HT%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D5749%26epage%3D5759%26doi%3D10.1021%2Fjm300338m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Cusack, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richelson, E.</span><span> </span><span class="NLM_article-title">Binding of antidepressants to human brain receptors: Focus on newer generation compounds</span> <span class="citation_source-journal">Psychopharmacology (Berl)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">565</span><span class="refDoi"> DOI: 10.1007/BF02244985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1007%2FBF02244985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=7855217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK2cXksV2qtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1994&pages=559-565&author=B.+Cusackauthor=A.+Nelsonauthor=E.+Richelson&title=Binding+of+antidepressants+to+human+brain+receptors%3A+Focus+on+newer+generation+compounds&doi=10.1007%2FBF02244985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of antidepressants to human brain receptors: focus on newer generation compounds</span></div><div class="casAuthors">Cusack, Bernadette; Nelson, Albert; Richelson, Elliott</div><div class="citationInfo"><span class="NLM_cas:title">Psychopharmacology (Berlin, Germany)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">559-65</span>CODEN:
                <span class="NLM_cas:coden">PSCHDL</span>;
        ISSN:<span class="NLM_cas:issn">0033-3158</span>.
    </div><div class="casAbstract">Using radioligand binding assays and post-mortem normal human brain tissue, the authors obtained equil. dissocn. consts. (Kds) for 17 antidepressants and two of their metabolites at histamine H1, muscarinic, α1-adrenergic, α2-adrenergic, dopamine D2, serotonin 5-HT1A, and serotonin 5-HT2 receptors.  Several newer antidepressants were compared with older drugs.  In addn., the authors studied some antimuscarinic, antiparkinson, antihistamine, and neuroleptic compds. at some of these receptors.  For the antidepressants, classical tricyclic antidepressants were the most potent drugs at five of the seven receptors (all but α2-adrenergic and 5-HT1A receptors).  The chlorophenylpiperazine deriv. antidepressants (etoperidone, nefazodone, trazodone) were the most potent antidepressants at α2-adrenergic and 5-HT1A receptors.  Of ten antihistamines tested, none was more potent than doxepin at histamine H1 receptors.  At muscarinic receptors antidepressants and antihistamines had a range of potencies, which were mostly weaker than those for antimuscarinics.  From the in vitro data, the authors expect adinazolam, bupropion, fluoxetine, sertraline, tomoxetine, and venlafaxine not to block any of these five receptors in vivo.  An antidepressant's potency for blocking a specific receptor is predictive of certain side effects and drug-drug interactions.  These studies can provide guidelines for the clinician in the choice of antidepressant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4C7IsG-1gh7Vg90H21EOLACvtfcHk0lgzxVZtiNkA4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXksV2qtbw%253D&md5=7be0af5b1e8e5abbca3412d51606da76</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2FBF02244985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02244985%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DB.%26aulast%3DNelson%26aufirst%3DA.%26aulast%3DRichelson%26aufirst%3DE.%26atitle%3DBinding%2520of%2520antidepressants%2520to%2520human%2520brain%2520receptors%253A%2520Focus%2520on%2520newer%2520generation%2520compounds%26jtitle%3DPsychopharmacology%2520%2528Berl%2529%26date%3D1994%26volume%3D114%26spage%3D559%26epage%3D565%26doi%3D10.1007%2FBF02244985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiroishi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weyand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwata, S.</span><span> </span><span class="NLM_article-title">Structure of the human histamine H<sub>1</sub> receptor complex with doxepin</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">475</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1038/nature10236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnature10236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=21697825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVekt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=475&publication_year=2011&pages=65-70&author=T.+Shimamuraauthor=M.+Shiroishiauthor=S.+Weyandauthor=H.+Tsujimotoauthor=G.+Winterauthor=V.+Katritchauthor=R.+Abagyanauthor=V.+Cherezovauthor=W.+Liuauthor=G.+W.+Hanauthor=T.+Kobayashiauthor=R.+C.+Stevensauthor=S.+Iwata&title=Structure+of+the+human+histamine+H1+receptor+complex+with+doxepin&doi=10.1038%2Fnature10236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human histamine H1 receptor complex with doxepin</span></div><div class="casAuthors">Shimamura, Tatsuro; Shiroishi, Mitsunori; Weyand, Simone; Tsujimoto, Hirokazu; Winter, Graeme; Katritch, Vsevolod; Abagyan, Ruben; Cherezov, Vadim; Liu, Wei; Han, Gye Won; Kobayashi, Takuya; Stevens, Raymond C.; Iwata, So</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">475</span>
        (<span class="NLM_cas:issue">7354</span>),
    <span class="NLM_cas:pages">65-70</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The biogenic amine histamine is an important pharmacol. mediator involved in pathophysiol. processes such as allergies and inflammations.  Histamine H1 receptor (H1R) antagonists are very effective drugs alleviating the symptoms of allergic reactions.  Here, we show the crystal structure of the H1R complex with doxepin, a first-generation H1R antagonist.  Doxepin sits deep in the ligand-binding pocket and directly interacts with Trp 428, a highly conserved key residue in G-protein-coupled-receptor activation.  This well-conserved pocket with mostly hydrophobic nature contributes to the low selectivity of the first-generation compds.  The pocket is assocd. with an anion-binding region occupied by a phosphate ion.  Docking of various second-generation H1R antagonists reveals that the unique carboxyl group present in this class of compds. interacts with Lys 191 and/or Lys 179ECL2, both of which form part of the anion-binding region.  This region is not conserved in other aminergic receptors, demonstrating how minor differences in receptors lead to pronounced selectivity differences with small mols.  Our study sheds light on the mol. basis of H1R antagonist specificity against H1R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN1obci6QtYbVg90H21EOLACvtfcHk0lgtDzEYVjR3Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVekt7k%253D&md5=7116228016caa1f3afbfed11468c0154</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature10236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10236%26sid%3Dliteratum%253Aachs%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DWeyand%26aufirst%3DS.%26aulast%3DTsujimoto%26aufirst%3DH.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DIwata%26aufirst%3DS.%26atitle%3DStructure%2520of%2520the%2520human%2520histamine%2520H1%2520receptor%2520complex%2520with%2520doxepin%26jtitle%3DNature%26date%3D2011%26volume%3D475%26spage%3D65%26epage%3D70%26doi%3D10.1038%2Fnature10236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Warne, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tate, C. G.</span><span> </span><span class="NLM_article-title">Crystal structures of a stabilized beta<sub>1</sub>-adrenoceptor bound to the biased agonists bucindolol and carvedilol</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span><span class="refDoi"> DOI: 10.1016/j.str.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.str.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=22579251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=841-849&author=T.+Warneauthor=P.+C.+Edwardsauthor=A.+G.+Leslieauthor=C.+G.+Tate&title=Crystal+structures+of+a+stabilized+beta1-adrenoceptor+bound+to+the+biased+agonists+bucindolol+and+carvedilol&doi=10.1016%2Fj.str.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of a Stabilized β1-Adrenoceptor Bound to the Biased Agonists Bucindolol and Carvedilol</span></div><div class="casAuthors">Warne, Tony; Edwards, Patricia C.; Leslie, Andrew G. W.; Tate, Christopher G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">841-849</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The β1-adrenoceptor (β1AR) is the site of action of beta blockers used in the treatment of cardiac-related illnesses.  Two beta blockers, carvedilol and bucindolol, show distinctive activities compared to other beta blockers and have been proposed as treatments tailored to the Arg/Gly3898.56 polymorphism of the human β1AR.  Both carvedilol and bucindolol are classified as biased agonists, because they stimulate G protein-independent signaling, while acting as either inverse or partial agonists of the G protein pathway.  We have detd. the crystal structures of a thermostabilized avian β1AR mutant bound to bucindolol and to carvedilol at 3.2 and 2.3 Å resoln., resp.  In comparison to other beta blockers, bucindolol and carvedilol interact with addnl. residues, in extracellular loop 2 and transmembrane helix 7, which may promote G protein-independent signaling.  The structures also suggest that there may be a structural explanation for the pharmacol. differences arising from the Arg/Gly3898.56 polymorphism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9aCm9IL2y37Vg90H21EOLACvtfcHk0lgtDzEYVjR3Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsLw%253D&md5=86542cd6a868477a9aabcbe25b1db8bb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DWarne%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DP.%2BC.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DTate%26aufirst%3DC.%2BG.%26atitle%3DCrystal%2520structures%2520of%2520a%2520stabilized%2520beta1-adrenoceptor%2520bound%2520to%2520the%2520biased%2520agonists%2520bucindolol%2520and%2520carvedilol%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D841%26epage%3D849%26doi%3D10.1016%2Fj.str.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Incorporating replacement free energy of binding-site waters in molecular docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">1765</span><span class="NLM_x">–</span> <span class="NLM_lpage">1776</span><span class="refDoi"> DOI: 10.1002/prot.24530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.24530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=24549784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislCgtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2014&pages=1765-1776&author=H.+Sunauthor=L.+Zhaoauthor=S.+Pengauthor=N.+Huang&title=Incorporating+replacement+free+energy+of+binding-site+waters+in+molecular+docking&doi=10.1002%2Fprot.24530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporating replacement free energy of binding-site waters in molecular docking</span></div><div class="casAuthors">Sun, Hanzi; Zhao, Lifeng; Peng, Shiming; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1765-1776</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Binding-site water mols. play a crucial role in protein-ligand recognition, either being displaced upon ligand binding or forming water bridges to stabilize the complex.  However, rigorously treating explicit binding-site waters is challenging in mol. docking, which requires to fully sample ensembles of waters and to consider the free energy cost of replacing waters.  Here, the authors describe a method to incorporate structural and energetic properties of binding-site waters into mol. docking.  The authors first developed a solvent property anal. (SPA) program to compute the replacement free energies of binding-site water mols. by post-processing mol. dynamics trajectories obtained from ligand-free protein structure simulation in explicit water.  Next, the authors implemented a distance-dependent scoring term into DOCK scoring function to take account of the water replacement free energy cost upon ligand binding.  The authors assessed this approach in protein targets contg. important binding-site waters, and the authors' approach is reliable in reproducing the crystal binding geometries of protein-ligand-water complexes, as well as moderately improving the ligand docking enrichment performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwQA_EiIKYHbVg90H21EOLACvtfcHk0lhk3LB36uMSVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislCgtL0%253D&md5=a8703dd3206123760dc6ffcd2d30fd41</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fprot.24530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24530%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DIncorporating%2520replacement%2520free%2520energy%2520of%2520binding-site%2520waters%2520in%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2014%26volume%3D82%26spage%3D1765%26epage%3D1776%26doi%3D10.1002%2Fprot.24530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based scoring of protein–ligand complexes: Enrichment of known inhibitors in large-scale virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span><span class="refDoi"> DOI: 10.1021/ci0502855</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0502855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=243-253&author=N.+Huangauthor=C.+Kalyanaramanauthor=J.+J.+Irwinauthor=M.+P.+Jacobson&title=Physics-based+scoring+of+protein%E2%80%93ligand+complexes%3A+Enrichment+of+known+inhibitors+in+large-scale+virtual+screening&doi=10.1021%2Fci0502855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-Based Scoring of Protein-Ligand Complexes: Enrichment of Known Inhibitors in Large-Scale Virtual Screening</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Irwin, John J.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors demonstrate that using an all-atom mol. mechanics force field combined with an implicit solvent model for scoring protein-ligand complexes is a promising approach for improving inhibitor enrichment in the virtual screening of large compd. databases.  The rescoring method is evaluated by the extent to which known binders for nine diverse, therapeutically relevant enzymes are enriched against a background of ∼100 000 drug-like decoys.  The improvement in enrichment is most robust and dramatic within the top 1% of the ranked database, i.e., the first thousand compds.; below the first few percent of the ranked database, there is little overall improvement.  The improved early enrichment is likely due to the more realistic treatment of ligand and receptor desolvation in the rescoring procedure.  We also present anecdotal but encouraging results assessing the ability of the rescoring method to predict specificity of inhibitors for structurally related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUqkz9N6IErLVg90H21EOLACvtfcHk0lhk3LB36uMSVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N&md5=cf636695aca21a87e050e0f39286ea1d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci0502855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0502855%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520scoring%2520of%2520protein%25E2%2580%2593ligand%2520complexes%253A%2520Enrichment%2520of%2520known%2520inhibitors%2520in%2520large-scale%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D243%26epage%3D253%26doi%3D10.1021%2Fci0502855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Discovery of novel tubulin inhibitors via structure-based hierarchical virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2730</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/ci300302c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300302c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlKltrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2730-2740&author=R.+Caoauthor=M.+Liuauthor=M.+Yinauthor=Q.+Liuauthor=Y.+Wangauthor=N.+Huang&title=Discovery+of+novel+tubulin+inhibitors+via+structure-based+hierarchical+virtual+screening&doi=10.1021%2Fci300302c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Tubulin Inhibitors via Structure-Based Hierarchical Virtual Screening</span></div><div class="casAuthors">Cao, Ran; Liu, Minyu; Yin, Min; Liu, Quanhai; Wang, Yanli; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2730-2740</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To discover novel tubulin inhibitors, we performed structure-based virtual screening against the colchicine binding pocket.  In combination with a hierarchical docking and scoring procedure, the structural information of an addnl. subpocket in colchicine site was applied to filter out the undesired docking hits.  This strategy automatically resulted in 63 candidates meeting the structural and energetic criteria from a screening library contg. approx. 100 000 diverse druglike compds.  Among them, nine mols. were chosen for exptl. validation, which all share the similar binding pose and contain an enriched scaffold bearing thiophene core.  Encouragingly, five compds. are active in tubulin polymn. assay.  The most potent inhibitor, 2-(2-fluorobenzamido)-3-carboxamide-4,5-dimethylthiophene, is structurally distinct to any known colchicine site binders and has higher ligand efficiency than colchicine.  On the basis of its predicted binding pose, we systematically probed its binding characteristics by testing series of structural modifications.  The obtained structure-activity relationship results are consistent with our binding model, and the inhibition activities of two analogs are improved by 2-fold.  We expect that the novel structure discovered in the present study may serve as a starting point for developing tubulin inhibitors with improved efficacy and fewer side effects.  We also expect that our hierarchical strategy may be generally applicable in structure-based virtual screening campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeoH-2Mo3J0LVg90H21EOLACvtfcHk0lhZm0t0Gt8s5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlKltrzJ&md5=ddc129d0a8e5baa601f3e0076a184bbf</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300302c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300302c%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520novel%2520tubulin%2520inhibitors%2520via%2520structure-based%2520hierarchical%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2730%26epage%3D2740%26doi%3D10.1021%2Fci300302c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lhZm0t0Gt8s5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Yuthavong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarnchompoo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vilaivan, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chitnumsub, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamchonwongpaisan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLennan, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vivas, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bongard, E.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">Malarial dihydrofolate reductase as a paradigm for drug development against a resistance-compromised target</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">16823</span><span class="NLM_x">–</span> <span class="NLM_lpage">16828</span><span class="refDoi"> DOI: 10.1073/pnas.1204556109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1073%2Fpnas.1204556109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23035243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WksL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16823-16828&author=Y.+Yuthavongauthor=B.+Tarnchompooauthor=T.+Vilaivanauthor=P.+Chitnumsubauthor=S.+Kamchonwongpaisanauthor=S.+A.+Charmanauthor=D.+N.+McLennanauthor=K.+L.+Whiteauthor=L.+Vivasauthor=E.+Bongard&title=Malarial+dihydrofolate+reductase+as+a+paradigm+for+drug+development+against+a+resistance-compromised+target&doi=10.1073%2Fpnas.1204556109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Malarial dihydrofolate reductase as a paradigm for drug development against a resistance-compromised target</span></div><div class="casAuthors">Yuthavong, Yongyuth; Tarnchompoo, Bongkoch; Vilaivan, Tirayut; Chitnumsub, Penchit; Kamchonwongpaisan, Sumalee; Charman, Susan A.; McLennan, Danielle N.; White, Karen L.; Vivas, Livia; Bongard, Emily; Thongphanchang, Chawanee; Taweechai, Supannee; Vanichtanankul, Jarunee; Rattanajak, Roonglawan; Arwon, Uthai; Fantauzzi, Pascal; Yuvaniyama, Jirundon; Charman, William N.; Matthews, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">16823-16828, S16823/1-S16823/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Malarial dihydrofolate reductase (DHFR) is the target of antifolate antimalarial drugs such as pyrimethamine and cycloguanil, the clin. efficacy of which have been compromised by resistance arising through mutations at various sites on the enzyme.  Here, we describe the use of cocrystal structures with inhibitors and substrates, along with efficacy and pharmacokinetic profiling for the design, characterization, and preclin. development of a selective, highly efficacious, and orally available antimalarial drug candidate (P218) that potently inhibits both wild-type and clin. relevant mutated forms of Plasmodium falciparum (Pf) DHFR.  Important structural characteristics of P218 include pyrimidine side-chain flexibility and a carboxylate group that makes charge-mediated hydrogen bonds with conserved Arg122 (PfDHFR-TS amino acid numbering).  An analogous interaction of P218 with human DHFR is disfavored because of three species-dependent amino acid substitutions in the vicinity of the conserved Arg.  Thus, P218 binds to the active site of PfDHFR in a substantially different fashion from the human enzyme, which is the basis for its high selectivity.  Unlike pyrimethamine, P218 binds both wild-type and mutant PfDHFR in a slow-on/slow-off tight-binding mode, which prolongs the target residence time.  P218, when bound to PfDHFR-TS, resides almost entirely within the envelope mapped out by the dihydrofolate substrate, which may make it less susceptible to resistance mutations.  The high in vivo efficacy in a SCID mouse model of P. falciparum malaria, good oral bioavailability, favorable enzyme selectivity, and good safety characteristics of P218 make it a potential candidate for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIVM7josnUBLVg90H21EOLACvtfcHk0li3JsO_GC_Bcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WksL7M&md5=7ef675391becc919f718de74fa61550b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1204556109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1204556109%26sid%3Dliteratum%253Aachs%26aulast%3DYuthavong%26aufirst%3DY.%26aulast%3DTarnchompoo%26aufirst%3DB.%26aulast%3DVilaivan%26aufirst%3DT.%26aulast%3DChitnumsub%26aufirst%3DP.%26aulast%3DKamchonwongpaisan%26aufirst%3DS.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DMcLennan%26aufirst%3DD.%2BN.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DVivas%26aufirst%3DL.%26aulast%3DBongard%26aufirst%3DE.%26atitle%3DMalarial%2520dihydrofolate%2520reductase%2520as%2520a%2520paradigm%2520for%2520drug%2520development%2520against%2520a%2520resistance-compromised%2520target%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D16823%26epage%3D16828%26doi%3D10.1073%2Fpnas.1204556109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Wan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">A new target for an old drug: Identifying mitoxantrone as a nanomolar inhibitor of PIM1 kinase via kinome-wide selectivity modeling</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2619</span><span class="NLM_x">–</span> <span class="NLM_lpage">2629</span><span class="refDoi"> DOI: 10.1021/jm400045y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400045y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2619-2629&author=X.+Wanauthor=W.+Zhangauthor=L.+Liauthor=Y.+Xieauthor=W.+Liauthor=N.+Huang&title=A+new+target+for+an+old+drug%3A+Identifying+mitoxantrone+as+a+nanomolar+inhibitor+of+PIM1+kinase+via+kinome-wide+selectivity+modeling&doi=10.1021%2Fjm400045y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling</span></div><div class="casAuthors">Wan, Xiaobo; Zhang, Wei; Li, Li; Xie, Yuting; Li, Wei; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2619-2629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rational design of selective kinase inhibitors remains a great challenge.  Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile.  We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases.  Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays.  Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells.  We further detd. the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition.  Although mitoxantrone's mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute to mitoxantrone's therapeutic efficacy and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCdODeUj1frVg90H21EOLACvtfcHk0lgCzUtCxBatiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D&md5=ff53cfe69b80b0b28821a4bfb4583378</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm400045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400045y%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DA%2520new%2520target%2520for%2520an%2520old%2520drug%253A%2520Identifying%2520mitoxantrone%2520as%2520a%2520nanomolar%2520inhibitor%2520of%2520PIM1%2520kinase%2520via%2520kinome-wide%2520selectivity%2520modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2619%26epage%3D2629%26doi%3D10.1021%2Fjm400045y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">McCorvy, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Structure and function of serotonin G protein-coupled receptors</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">142</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.pharmthera.2015.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=25601315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVeisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=129-142&author=J.+D.+McCorvyauthor=B.+L.+Roth&title=Structure+and+function+of+serotonin+G+protein-coupled+receptors&doi=10.1016%2Fj.pharmthera.2015.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of serotonin G protein-coupled receptors</span></div><div class="casAuthors">McCorvy, John D.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">129-142</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Serotonin receptors are prevalent throughout the nervous system and the periphery, and remain one of the most lucrative and promising drug discovery targets for disorders ranging from migraine headaches to neuropsychiatric disorders such as schizophrenia and depression.  There are 14 distinct serotonin receptors, of which 13 are G protein-coupled receptors (GPCRs), which are targets for approx. 40% of the approved medicines.  Recent crystallog. and biochem. evidence has provided a converging understanding of the basic structure and functional mechanics of GPCR activation.  Currently, two GPCR crystal structures exist for the serotonin family, the 5-HT1B and 5-HT2B receptor, with the antimigraine and valvulopathic drug ergotamine bound.  The first serotonin crystal structures not only provide the first evidence of serotonin receptor topog. but also provide mechanistic explanations into functional selectivity or biased agonism.  This review will detail the findings of these crystal structures from a mol. and mutagenesis perspective for driving rational drug design for novel therapeutics incorporating biased signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpapliz37oS9bVg90H21EOLACvtfcHk0lgCzUtCxBatiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVeisbs%253D&md5=0fcfbc57dfb7047f6ffb7c74d437b5b4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructure%2520and%2520function%2520of%2520serotonin%2520G%2520protein-coupled%2520receptors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D150%26spage%3D129%26epage%3D142%26doi%3D10.1016%2Fj.pharmthera.2015.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Al-Chaer, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawasaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasricha, P. J.</span><span> </span><span class="NLM_article-title">A new model of chronic visceral hypersensitivity in adult rats induced by colon irritation during postnatal development</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">1276</span><span class="NLM_x">–</span> <span class="NLM_lpage">1285</span><span class="refDoi"> DOI: 10.1053/gast.2000.19576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1053%2Fgast.2000.19576" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2000&pages=1276-1285&author=E.+D.+Al-Chaerauthor=M.+Kawasakiauthor=P.+J.+Pasricha&title=A+new+model+of+chronic+visceral+hypersensitivity+in+adult+rats+induced+by+colon+irritation+during+postnatal+development&doi=10.1053%2Fgast.2000.19576"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1053%2Fgast.2000.19576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fgast.2000.19576%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Chaer%26aufirst%3DE.%2BD.%26aulast%3DKawasaki%26aufirst%3DM.%26aulast%3DPasricha%26aufirst%3DP.%2BJ.%26atitle%3DA%2520new%2520model%2520of%2520chronic%2520visceral%2520hypersensitivity%2520in%2520adult%2520rats%2520induced%2520by%2520colon%2520irritation%2520during%2520postnatal%2520development%26jtitle%3DGastroenterology%26date%3D2000%26volume%3D119%26spage%3D1276%26epage%3D1285%26doi%3D10.1053%2Fgast.2000.19576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Jacobson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span> </span><span class="NLM_article-title">Force field validation using protein side chain prediction</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">11673</span><span class="NLM_x">–</span> <span class="NLM_lpage">11680</span><span class="refDoi"> DOI: 10.1021/jp021564n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp021564n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvVOisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2002&pages=11673-11680&author=M.+A.+Jacobsonauthor=G.+A.+Kaminskiauthor=G.+A.+Friesnerauthor=C.+S.+Rapp&title=Force+field+validation+using+protein+side+chain+prediction&doi=10.1021%2Fjp021564n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Force Field Validation Using Protein Side Chain Prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Kaminski, George A.; Friesner, Richard A.; Rapp, Chaya S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">11673-11680</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The prediction of protein side chain conformations is used to evaluate the accuracy of force field parameters.  Specifically, new torsional parameters have recently been reported for the OPLS-AA force field, which achieved substantially better accuracy with respect to high level gas-phase quantum chem. calcns. [J. Phys. Chem. B 2001, 105, 6474].  Here we demonstrate that these new parameters also lead to qual. improved side chain prediction accuracy.  The primary emphasis is on the prediction of single side chain conformations, with the rest of the protein held fixed at the native configuration.  Errors due to incomplete sampling can thus be essentially eliminated, using a combination of rotamer search and energy minimization.  In addn., the protein environment is modeled realistically using implicit solvation and an explicit representation of crystal packing effects.  Aided by the development of new algorithms, these calcns. have been performed with modest computational requirements (a cluster of PCs) on a database of 36 proteins (∼5000 total residues).  The side chain prediction tests that we employ are quite general and can be used to evaluate nonbonded or solvation parameters as well.  As such, they provide a useful complement to decoy studies for force field validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZJvh-DVauJrVg90H21EOLACvtfcHk0lgygDtd5N4pPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvVOisrw%253D&md5=1cc23c97485ac031d3f2524f672370a4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjp021564n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp021564n%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BA.%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DG.%2BA.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26atitle%3DForce%2520field%2520validation%2520using%2520protein%2520side%2520chain%2520prediction%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2002%26volume%3D106%26spage%3D11673%26epage%3D11680%26doi%3D10.1021%2Fjp021564n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pincus, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0lgygDtd5N4pPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">High-resolution prediction of protein helix positions and orientations</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">368</span><span class="NLM_x">–</span> <span class="NLM_lpage">382</span><span class="refDoi"> DOI: 10.1002/prot.20014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.20014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=368-382&author=X.+Liauthor=M.+P.+Jacobsonauthor=R.+A.+Friesner&title=High-resolution+prediction+of+protein+helix+positions+and+orientations&doi=10.1002%2Fprot.20014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fprot.20014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20014%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DHigh-resolution%2520prediction%2520of%2520protein%2520helix%2520positions%2520and%2520orientations%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26spage%3D368%26epage%3D382%26doi%3D10.1002%2Fprot.20014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0ljjU55rLEyuKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strockbine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1002/prot.21123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=712-725&author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&title=Comparison+of+multiple+Amber+force+fields+and+development+of+improved+protein+backbone+parameters&doi=10.1002%2Fprot.21123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of α-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./ψ dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./ψ dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0lj6O1x3_kbIFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520multiple%2520Amber%2520force%2520fields%2520and%2520development%2520of%2520improved%2520protein%2520backbone%2520parameters%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725%26doi%3D10.1002%2Fprot.21123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lj6O1x3_kbIFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Sciortino, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essmann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmati, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angell, C. A.</span><span> </span><span class="NLM_article-title">Crystal stability limits at positive and negative pressures, and crystal-to-glass transitions</span> <span class="citation_source-journal">Phys. Rev. E: Stat. Phys., Plasmas, Fluids, Relat. Interdiscip. Top.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6484</span><span class="NLM_x">–</span> <span class="NLM_lpage">6491</span><span class="refDoi"> DOI: 10.1103/PhysRevE.52.6484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1103%2FPhysRevE.52.6484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=9964167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSnsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1995&pages=6484-6491&author=F.+Sciortinoauthor=U.+Essmannauthor=H.+E.+Stanleyauthor=M.+Hemmatiauthor=J.+Shaoauthor=G.+H.+Wolfauthor=C.+A.+Angell&title=Crystal+stability+limits+at+positive+and+negative+pressures%2C+and+crystal-to-glass+transitions&doi=10.1103%2FPhysRevE.52.6484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal stability limits at positive and negative pressures, and crystal-to-glass transitions</span></div><div class="casAuthors">Sciortino, Francesco; Essmann, Ulrich; Stanley, H. Eugene; Hemmati, Mahin; Shao, Jun; Wolf, George H.; Angell, C. Austen</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review E: Statistical Physics, Plasmas, Fluids, and Related Interdisciplinary Topics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6-B</span>),
    <span class="NLM_cas:pages">6484-91</span>CODEN:
                <span class="NLM_cas:coden">PLEEE8</span>;
        ISSN:<span class="NLM_cas:issn">1063-651X</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">The direct crystal-to-glass transformation, i.e., spontaneous amorphization, which was first obsd. by thermal annealing of stishovite SiO2 at ambient pressure, has now been obsd. as an isothermal phenomenon during both compression and decompression of initially stable crystals.  While counterintuitive, and dependent on kinetically controlled metastable events, the phenomenon is of broad interest and potential importance in materials science and geophysics.  The authors use a combination of mol. dynamics simulations and analyses of lab. data to explore the metastable crystal ranges, including the neg. pressure range, for key compds. such as the ices, silicas, and alk. earth perovskites.  The focus is on the establishment of phenomenol. patterns rather than on specific metastability-terminating mechanisms.  A simple quadratic law, P-Ps ∼ (V-Vs)2 (where Ps and Vs are the values of the pressure P and vol. V on the spinodal), well approximates the equations of state over much of the metastable and even the stable range - and implies the existence of an isochoric boundary line for stability to isotropic d. fluctuations.  The authors delineate the conditions under which amorphization occurs, usually substantially before the stability limit is reached.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPt_Pd-cyJRrVg90H21EOLACvtfcHk0li37LLG3faYoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSnsr3N&md5=dd10f8c6ef5e00ec0a7903bf17df84e9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1103%2FPhysRevE.52.6484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevE.52.6484%26sid%3Dliteratum%253Aachs%26aulast%3DSciortino%26aufirst%3DF.%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DStanley%26aufirst%3DH.%2BE.%26aulast%3DHemmati%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DG.%2BH.%26aulast%3DAngell%26aufirst%3DC.%2BA.%26atitle%3DCrystal%2520stability%2520limits%2520at%2520positive%2520and%2520negative%2520pressures%252C%2520and%2520crystal-to-glass%2520transitions%26jtitle%3DPhys.%2520Rev.%2520E%253A%2520Stat.%2520Phys.%252C%2520Plasmas%252C%2520Fluids%252C%2520Relat.%2520Interdiscip.%2520Top.%26date%3D1995%26volume%3D52%26spage%3D6484%26epage%3D6491%26doi%3D10.1103%2FPhysRevE.52.6484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bekker, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, J.</span><span> </span><span class="NLM_article-title">LINCS: A linear constraint solver for molecular simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1463</span><span class="NLM_x">–</span> <span class="NLM_lpage">1472</span><span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.3.CO;2-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+Fraaije&title=LINCS%3A+A+linear+constraint+solver+for+molecular+simulations&doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.3.CO%253B2-L%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%26atitle%3DLINCS%253A%2520A%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26spage%3D1463%26epage%3D1472%26doi%3D10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taneishi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, G.</span><span> </span><span class="NLM_article-title">GLIDA: GPCR-ligand database for chemical genomic drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">D673</span><span class="NLM_x">–</span> <span class="NLM_lpage">D677</span><span class="refDoi"> DOI: 10.1093/nar/gkj028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1093%2Fnar%2Fgkj028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=D673-D677&author=Y.+Okunoauthor=J.+Yangauthor=K.+Taneishiauthor=H.+Yabuuchiauthor=G.+Tsujimoto&title=GLIDA%3A+GPCR-ligand+database+for+chemical+genomic+drug+discovery&doi=10.1093%2Fnar%2Fgkj028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkj028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkj028%26sid%3Dliteratum%253Aachs%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTaneishi%26aufirst%3DK.%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DTsujimoto%26aufirst%3DG.%26atitle%3DGLIDA%253A%2520GPCR-ligand%2520database%2520for%2520chemical%2520genomic%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DD673%26epage%3DD677%26doi%3D10.1093%2Fnar%2Fgkj028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niijima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minowa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonomura, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunimoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, C.</span><span> </span><span class="NLM_article-title">GLIDA: GPCR—ligand database for chemical genomics drug discovery—database and tools update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D907</span><span class="NLM_x">–</span> <span class="NLM_lpage">D912</span><span class="refDoi"> DOI: 10.1093/nar/gkm948</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1093%2Fnar%2Fgkm948" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D907-D912&author=Y.+Okunoauthor=A.+Tamonauthor=H.+Yabuuchiauthor=S.+Niijimaauthor=Y.+Minowaauthor=K.+Tonomuraauthor=R.+Kunimotoauthor=C.+Feng&title=GLIDA%3A+GPCR%E2%80%94ligand+database+for+chemical+genomics+drug+discovery%E2%80%94database+and+tools+update&doi=10.1093%2Fnar%2Fgkm948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm948%26sid%3Dliteratum%253Aachs%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DTamon%26aufirst%3DA.%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTonomura%26aufirst%3DK.%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DC.%26atitle%3DGLIDA%253A%2520GPCR%25E2%2580%2594ligand%2520database%2520for%2520chemical%2520genomics%2520drug%2520discovery%25E2%2580%2594database%2520and%2520tools%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD907%26epage%3DD912%26doi%3D10.1093%2Fnar%2Fgkm948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span><span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lhBMpHnyfkz5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carchia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Directory of useful decoys, enhanced (DUD-E): Better ligands and decoys for better benchmarking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6582</span><span class="NLM_x">–</span> <span class="NLM_lpage">6594</span><span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&author=M.+M.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+useful+decoys%2C+enhanced+%28DUD-E%29%3A+Better+ligands+and+decoys+for+better+benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0lh6w0t_oCXu8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520useful%2520decoys%252C%2520enhanced%2520%2528DUD-E%2529%253A%2520Better%2520ligands%2520and%2520decoys%2520for%2520better%2520benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span><span class="refDoi"> DOI: 10.1021/jm0608356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking&doi=10.1021%2Fjm0608356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0lh6w0t_oCXu8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801%26doi%3D10.1021%2Fjm0608356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baase, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A model binding site for testing scoring functions in molecular docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">355</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00777-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2FS0022-2836%2802%2900777-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=12217695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=339-355&author=B.+Q.+Weiauthor=W.+A.+Baaseauthor=L.+H.+Weaverauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking&doi=10.1016%2FS0022-2836%2802%2900777-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Model Binding Site for Testing Scoring Functions in Molecular Docking</span></div><div class="casAuthors">Wei, Binqing Q.; Baase, Walter A.; Weaver, Larry H.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-355</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Prediction of interaction energies between ligands and their receptors remains a major challenge for structure-based inhibitor discovery.  Much effort has been devoted to developing scoring schemes that can successfully rank the affinities of a diverse set of possible ligands to a binding site for which the structure is known.  To test these scoring functions, well-characterized exptl. systems can be very useful.  Here, mutation-created binding sites in T4 lysozyme were used to investigate how the quality of at. charges and solvation energies affects mol. docking.  At. charges and solvation energies were calcd. for 172,118 mols. in the Available Chems.  Directory using a semi-empirical quantum mech. approach by the program AMSOL.  The database was first screened against the apolar cavity site created by the mutation Leu99Ala (L99A).  Compared to the electronegativity-based charges that are widely used, the new charges and desolvation energies improved ranking of known apolar ligands, and better distinguished them from more polar isosteres that are not obsd. to bind.  To investigate whether the new charges had predictive value, the non-polar residue Met102, which forms part of the binding site, was changed to the polar residue glutamine.  The structure of the resulting Leu99 Ala and Met102 Gln double mutant of T4 lysozyme (L99A/M102Q) was detd. and the docking calcn. was repeated for the new site.  Seven representative polar mols. that preferentially docked to the polar vs. the apolar binding site were tested exptl.  All seven bind to the polar cavity (L99A/M102Q) but do not detectably bind to the apolar cavity (L99A).  Five ligand-bound structures of L99A/M102Q were detd. by X-ray crystallog.  Docking predictions corresponded to the crystallog. results to within 0.4 A RMSD.  Improved treatment of partial at. charges and desolvation energies in database docking appears feasible and leads to better distinction of true ligands.  Simple model binding sites, such as L99A and its more polar variants, may find broad use in the development and testing of docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGral4WDhJW6jbVg90H21EOLACvtfcHk0liZPiuSYHn-WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D&md5=040e009b33f125ab4abbb322cf32d2c9</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900777-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900777-5%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBaase%26aufirst%3DW.%2BA.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D339%26epage%3D355%26doi%3D10.1016%2FS0022-2836%2802%2900777-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Lorber, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Hierarchical docking of databases of multiple ligand conformations</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">749</span><span class="refDoi"> DOI: 10.2174/1568026054637683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.2174%2F1568026054637683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=16101414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=739-749&author=D.+M.+Lorberauthor=B.+K.+Shoichet&title=Hierarchical+docking+of+databases+of+multiple+ligand+conformations&doi=10.2174%2F1568026054637683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical docking of databases of multiple ligand conformations</span></div><div class="casAuthors">Lorber, David M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">739-749</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Ligand flexibility is an important problem in mol. docking and virtual screening.  To address this challenge, we investigate a hierarchical pre-organization of multiple conformations of small mols.  Such organization of pre-calcd. conformations removes the exploration of ligand conformational space from the docking calcn. and allows for concise representation of what can be thousands of conformations.  The hierarchy also recognizes and prunes incompatible conformations early in the calcn., eliminating redundant calcns. of fit.  We investigate the method by docking the MDL Drug Data Report (MDDR), an annotated database of 100,000 mols., into apo and holo forms of 7 unrelated targets.  This annotated database allows us to track the ranking of tens to hundreds of annotated ligands in each of the docking systems.  The binding sites and database are prepd. in an automated fashion in an attempt to remove some human bias from the calcns.  Many thousands of explicit and implicit ligand conformations may be docked in calcns. not much longer than required for single conformer docking.  As long as internal energies are not considered, recombination with the hierarchy is additive as the no. of degrees of freedom is increased.  Mols. with even millions of conformations can be docked in a few minutes on a single desktop computer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ_iMnNjm9LVg90H21EOLACvtfcHk0liZPiuSYHn-WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D&md5=25c833b5608e5faedc1af3ad38fdfa06</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.2174%2F1568026054637683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054637683%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DHierarchical%2520docking%2520of%2520databases%2520of%2520multiple%2520ligand%2520conformations%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D739%26epage%3D749%26doi%3D10.2174%2F1568026054637683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Ferrin, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvis, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span> </span><span class="NLM_article-title">The MIDAS display system</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1016/0263-7855(88)80054-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2F0263-7855%2888%2980054-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaL1cXktVGrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1988&pages=13-27&author=T.+E.+Ferrinauthor=C.+C.+Huangauthor=L.+E.+Jarvisauthor=R.+Langridge&title=The+MIDAS+display+system&doi=10.1016%2F0263-7855%2888%2980054-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The MIDAS display system</span></div><div class="casAuthors">Ferrin, Thomas E.; Huang, Conrad C.; Jarvis, Laurie E.; Langridge, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-27, 36-7</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    </div><div class="casAbstract">A description is given of the Mol. Interactive Display and Simulation (MIDAS) system for the manipulation and display of large macromols., such as proteins and nucleic acids.  Several ancillary programs are described that allow for such features as computing the surface of a mol., and computing electrostatic charge potentials.  The core of MIDAS is a hierarchical database system, designed specifically for macromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGxkghePxXQ7Vg90H21EOLACvtfcHk0ljRiTbGCUjkxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktVGrsrs%253D&md5=86a9a293a0b51feb7a7310317bb99662</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2888%2980054-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252888%252980054-7%26sid%3Dliteratum%253Aachs%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DJarvis%26aufirst%3DL.%2BE.%26aulast%3DLangridge%26aufirst%3DR.%26atitle%3DThe%2520MIDAS%2520display%2520system%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1988%26volume%3D6%26spage%3D13%26epage%3D27%26doi%3D10.1016%2F0263-7855%2888%2980054-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oatley, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">A geometric approach to macromolecule-ligand interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1982</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">288</span><span class="refDoi"> DOI: 10.1016/0022-2836(82)90153-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2F0022-2836%2882%2990153-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=7154081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=1982&pages=269-288&author=I.+D.+Kuntzauthor=J.+M.+Blaneyauthor=S.+J.+Oatleyauthor=R.+Langridgeauthor=T.+E.+Ferrin&title=A+geometric+approach+to+macromolecule-ligand+interactions&doi=10.1016%2F0022-2836%2882%2990153-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A geometric approach to macromolecule-ligand interactions</span></div><div class="casAuthors">Kuntz, Irwin D.; Blaney, Jeffrey M.; Oatley, Stuart J.; Langridge, Robert; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-88</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A method is described to explore geometrically feasible alignments of ligands and receptors of known structure.  Algorithms are presented that examine many binding geometries and evaluate them in terms of steric overlap.  The procedure uses specific mol. conformations.  A method is included for finding putative binding sites on a macromol. surface.  Results are reported for heme-myoglobin interaction and the binding of thyroid hormone analogs to prealbumin.  In each case, the program finds structures within 1 Å of the x-ray results and also finds distinctly different geometries that provide good steric fits.  The approach seems well-suited for generating conformations for energy refinement programs and interactive computer graphics routines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWsp3KKxuQf7Vg90H21EOLACvtfcHk0ljRiTbGCUjkxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D&md5=8a4234b24356ea5340f33d906cd71d3e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2882%2990153-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252882%252990153-X%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DOatley%26aufirst%3DS.%2BJ.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DA%2520geometric%2520approach%2520to%2520macromolecule-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D161%26spage%3D269%26epage%3D288%26doi%3D10.1016%2F0022-2836%2882%2990153-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span><span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0ljRiTbGCUjkxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Nicholls, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">A rapid finite difference algorithm, utilizing successive over-relaxation to solve the Poisson-Boltzmann equation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">445</span><span class="refDoi"> DOI: 10.1002/jcc.540120405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fjcc.540120405" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1991&pages=435-445&author=A.+Nichollsauthor=B.+Honig&title=A+rapid+finite+difference+algorithm%2C+utilizing+successive+over-relaxation+to+solve+the+Poisson-Boltzmann+equation&doi=10.1002%2Fjcc.540120405"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540120405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540120405%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DA%2520rapid%2520finite%2520difference%2520algorithm%252C%2520utilizing%2520successive%2520over-relaxation%2520to%2520solve%2520the%2520Poisson-Boltzmann%2520equation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1991%26volume%3D12%26spage%3D435%26epage%3D445%26doi%3D10.1002%2Fjcc.540120405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC-a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC-a+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eaRgAcVZlVHJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC-a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: A free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lgj2fu4mfgcCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span> </span><span class="NLM_article-title">Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11225</span><span class="NLM_x">–</span> <span class="NLM_lpage">11236</span><span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+testing+of+the+OPLS+all-atom+force+field+on+conformational+energetics+and+properties+of+organic+liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0lgj2fu4mfgcCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520the%2520OPLS%2520all-atom%2520force%2520field%2520on%2520conformational%2520energetics%2520and%2520properties%2520of%2520organic%2520liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Evaluation and reparametrization of the OPLS-AA force field for proteins via comparison with accurate quantum chemical calculations on peptides</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6474</span><span class="NLM_x">–</span> <span class="NLM_lpage">6487</span><span class="refDoi"> DOI: 10.1021/jp003919d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp003919d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=6474-6487&author=G.+A.+Kaminskiauthor=R.+A.+Friesnerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Evaluation+and+reparametrization+of+the+OPLS-AA+force+field+for+proteins+via+comparison+with+accurate+quantum+chemical+calculations+on+peptides&doi=10.1021%2Fjp003919d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span></div><div class="casAuthors">Kaminski, George A.; Friesner, Richard A.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6474-6487</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present results of improving the OPLS-AA force field for peptides by means of refitting the key Fourier torsional coeffs.  The fitting technique combines using accurate ab initio data as the target, choosing an efficient fitting subspace of the whole potential-energy surface, and detg. wts. for each of the fitting points based on magnitudes of the potential-energy gradient.  The av. energy RMS deviation from the LMP2/cc-pVTZ(-f)//HF/6-31G** data is reduced by ∼40% from 0.81 to 0.47 kcal/mol as a result of the fitting for the electrostatically uncharged dipeptides.  Transferability of the parameters is demonstrated by using the same alanine dipeptide-fitted backbone torsional parameters for all of the other dipeptides (with the appropriate side-chain refitting) and the alanine tetrapeptide.  Parameters of nonbonded interactions have also been refitted for the sulfur-contg. dipeptides (cysteine and methionine), and the validity of the new Coulombic charges and the van der Waals σ's and ε's is proved through reproducing gas-phase energies of complex formation heats of vaporization and densities of pure model liqs.  Moreover, a novel approach to fitting torsional parameters for electrostatically charged mol. systems has been presented and successfully tested on five dipeptides with charged side chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSHwmffIQePbVg90H21EOLACvtfcHk0ljfkpgp3fbfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D&md5=3ff059626977ee7f6342466f5820f5b7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjp003919d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp003919d%26sid%3Dliteratum%253Aachs%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEvaluation%2520and%2520reparametrization%2520of%2520the%2520OPLS-AA%2520force%2520field%2520for%2520proteins%2520via%2520comparison%2520with%2520accurate%2520quantum%2520chemical%2520calculations%2520on%2520peptides%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D105%26spage%3D6474%26epage%3D6487%26doi%3D10.1021%2Fjp003919d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Ma, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poon, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chui, W. K.</span><span> </span><span class="NLM_article-title">Synthesis and in vitro evaluation of 2,4-diamino-1,3,5-triazine derivatives as neuronal voltage-gated sodium channel blockers</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">5644</span><span class="NLM_x">–</span> <span class="NLM_lpage">5647</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.bmcl.2009.08.052" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=5644-5647&author=X.+Maauthor=T.+Y.+Poonauthor=P.+T.+Wongauthor=W.+K.+Chui&title=Synthesis+and+in+vitro+evaluation+of+2%2C4-diamino-1%2C3%2C5-triazine+derivatives+as+neuronal+voltage-gated+sodium+channel+blockers&doi=10.1016%2Fj.bmcl.2009.08.052"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DX.%26aulast%3DPoon%26aufirst%3DT.%2BY.%26aulast%3DWong%26aufirst%3DP.%2BT.%26aulast%3DChui%26aufirst%3DW.%2BK.%26atitle%3DSynthesis%2520and%2520in%2520vitro%2520evaluation%2520of%25202%252C4-diamino-1%252C3%252C5-triazine%2520derivatives%2520as%2520neuronal%2520voltage-gated%2520sodium%2520channel%2520blockers%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D5644%26epage%3D5647%26doi%3D10.1016%2Fj.bmcl.2009.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lgaYvRAeBdSbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Papp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moryl, E.</span><span> </span><span class="NLM_article-title">Antidepressant activity of non-competitive and competitive NMDA receptor antagonists in a chronic mild stress model of depression</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">263</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/0014-2999(94)90516-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2F0014-2999%2894%2990516-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=7821340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK2cXmsVKitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1994&pages=1-7&author=M.+Pappauthor=E.+Moryl&title=Antidepressant+activity+of+non-competitive+and+competitive+NMDA+receptor+antagonists+in+a+chronic+mild+stress+model+of+depression&doi=10.1016%2F0014-2999%2894%2990516-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Antidepressant activity of non-competitive and competitive NMDA receptor antagonists in a chronic mild stress model of depression</span></div><div class="casAuthors">Papp, Mariusz; Moryl, Elzbieta</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">1/2</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The antidepressant properties of the non-competitive NMDA receptor antagonist, MK-801 (dizocilpine), and the competitive NMDA receptor antagonist, CGP 37849 (DL-(E)-2-amino-4-methyl-5-phosphono-3-pentonoic acid) and its (R)-enantiomer CGP 40116, were studied in a chronic mild stress model of depression.  In this model, animals subjected to a variety of mild stressors for a prolonged period of time show a substantial decrease in the consumption of palatable sucrose soln. (anhedonia).  It was previously demonstrated that the chronic mild stress-induced anhedonia can be reversed by chronic treatment with various antidepressant drugs.  In this study we found that the stress-induced deficit in sucrose intake was gradually reversed by chronic (4-5 wk) treatment with MK-801 (0.3 mg/kg i.p.), CGP 37849 (5 mg/kg i.p.) and CGP (25 mg/kg p.o.).  The magnitude of this effect and its time course were comparable to those obsd. following similar administration of imipramine (10 mg/kg i.p. or p.o.).  The increase in sucrose intake following chronic administration of imipramine and NMDA receptor antagonists was specific to stressed animals; the behavior of non-stressed controls was unchanged by any of the drugs tested.  These results confirm those of previous studies, carried out on 'normal' animals, suggesting that antagonists of NMDA receptors may have antidepressant properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXAZ-Pbbeub7Vg90H21EOLACvtfcHk0lgaYvRAeBdSbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsVKitL8%253D&md5=bae5b93e49fd2ef9054fbbe7470bb0a3</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2894%2990516-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252894%252990516-9%26sid%3Dliteratum%253Aachs%26aulast%3DPapp%26aufirst%3DM.%26aulast%3DMoryl%26aufirst%3DE.%26atitle%3DAntidepressant%2520activity%2520of%2520non-competitive%2520and%2520competitive%2520NMDA%2520receptor%2520antagonists%2520in%2520a%2520chronic%2520mild%2520stress%2520model%2520of%2520depression%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1994%26volume%3D263%26spage%3D1%26epage%3D7%26doi%3D10.1016%2F0014-2999%2894%2990516-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 23 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Reed M. Stein, Ying Yang, Trent E. Balius, Matt J. O’Meara, Jiankun Lyu, Jennifer Young, Khanh Tang, Brian K. Shoichet, <span class="NLM_string-name hlFld-ContribAuthor">John J. Irwin</span>. </span><span class="cited-content_cbyCitation_article-title">Property-Unmatched Decoys in Docking Benchmarks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (2)
                                     , 699-714. <a href="https://doi.org/10.1021/acs.jcim.0c00598" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00598%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProperty-Unmatched%252BDecoys%252Bin%252BDocking%252BBenchmarks%26aulast%3DStein%26aufirst%3DReed%2BM.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29052020%26date%3D25012021%26volume%3D61%26issue%3D2%26spage%3D699%26epage%3D714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gudong Li, Wei Zhang, Yuting Xie, Yang Li, Rao Cao, Guojun Zheng, Niu Huang, <span class="NLM_string-name hlFld-ContribAuthor">Yu Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Optimization of 10-DEBC Derivatives as Potent and Selective Pim-1 Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 3287-3294. <a href="https://doi.org/10.1021/acs.jcim.0c00245" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00245</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00245%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructure-Based%252BOptimization%252Bof%252B10-DEBC%252BDerivatives%252Bas%252BPotent%252Band%252BSelective%252BPim-1%252BKinase%252BInhibitors%26aulast%3DLi%26aufirst%3DGudong%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10032020%26date%3D29052020%26date%3D14052020%26volume%3D60%26issue%3D6%26spage%3D3287%26epage%3D3294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gudong Li, Wei Li, Yuting Xie, Xiaobo Wan, Guojun Zheng, Niu Huang, <span class="NLM_string-name hlFld-ContribAuthor">Yu Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Pim-1 Kinase Inhibitors with a Flexible-Receptor Docking Protocol. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (10)
                                     , 4116-4119. <a href="https://doi.org/10.1021/acs.jcim.9b00494" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00494</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00494%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BNovel%252BPim-1%252BKinase%252BInhibitors%252Bwith%252Ba%252BFlexible-Receptor%252BDocking%252BProtocol%26aulast%3DLi%26aufirst%3DGudong%26date%3D2019%26date%3D2019%26date%3D17102019%26date%3D14102019%26volume%3D59%26issue%3D10%26spage%3D4116%26epage%3D4119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuanxun Wang, Qiuyu Fu, Yu Zhou, Yunfei Du, <span class="NLM_string-name hlFld-ContribAuthor">Niu Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Replacement of Protein Binding-Site Waters Contributes to Favorable Halogen Bond Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (7)
                                     , 3136-3143. <a href="https://doi.org/10.1021/acs.jcim.9b00128" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00128%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DReplacement%252Bof%252BProtein%252BBinding-Site%252BWaters%252BContributes%252Bto%252BFavorable%252BHalogen%252BBond%252BInteractions%26aulast%3DWang%26aufirst%3DYuanxun%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12022019%26date%3D25062019%26date%3D12062019%26volume%3D59%26issue%3D7%26spage%3D3136%26epage%3D3143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yu Zhou, Yuanxun Wang, Pengfei Li, Xi-Ping Huang, Xiangbin Qi, Yunfei Du, <span class="NLM_string-name hlFld-ContribAuthor">Niu Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Halogen Bonds in 5-Hydroxytryptamine 2B Receptor–Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (10)
                                     , 1019-1024. <a href="https://doi.org/10.1021/acsmedchemlett.8b00300" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00300</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00300%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DExploring%252BHalogen%252BBonds%252Bin%252B5-Hydroxytryptamine%252B2B%252BReceptor%2525E2%252580%252593Ligand%252BInteractions%26aulast%3DZhou%26aufirst%3DYu%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30062018%26date%3D24092018%26date%3D01102018%26volume%3D9%26issue%3D10%26spage%3D1019%26epage%3D1024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dilip K.  Tosh</span>, <span class="hlFld-ContribAuthor ">Antonella  Ciancetta</span>, <span class="hlFld-ContribAuthor ">Eugene  Warnick</span>, <span class="hlFld-ContribAuthor ">Steven  Crane</span>, <span class="hlFld-ContribAuthor ">Zhan-Guo  Gao</span>, and <span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Scaffold Repurposing for G Protein-Coupled Receptors: Transformation of Adenosine Derivatives into 5HT2B/5HT2C Serotonin Receptor Antagonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (24)
                                     , 11006-11026. <a href="https://doi.org/10.1021/acs.jmedchem.6b01183" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01183</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01183%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BScaffold%252BRepurposing%252Bfor%252BG%252BProtein-Coupled%252BReceptors%25253A%252BTransformation%252Bof%252BAdenosine%252BDerivatives%252Binto%252B5HT2B%25252F5HT2C%252BSerotonin%252BReceptor%252BAntagonists%26aulast%3DTosh%26aufirst%3DDilip%2BK.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D05082016%26date%3D09122016%26date%3D22122016%26volume%3D59%26issue%3D24%26spage%3D11006%26epage%3D11026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span> and <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Novel Ligands Conferring New Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4103-4120. <a href="https://doi.org/10.1021/acs.jmedchem.5b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BNovel%252BLigands%252BConferring%252BNew%252BBiology%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D25122015%26date%3D15032016%26date%3D12052016%26date%3D25022016%26volume%3D59%26issue%3D9%26spage%3D4103%26epage%3D4120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dehua  Yang</span>, <span class="hlFld-ContribAuthor ">Qingtong  Zhou</span>, <span class="hlFld-ContribAuthor ">Viktorija  Labroska</span>, <span class="hlFld-ContribAuthor ">Shanshan  Qin</span>, <span class="hlFld-ContribAuthor ">Sanaz  Darbalaei</span>, <span class="hlFld-ContribAuthor ">Yiran  Wu</span>, <span class="hlFld-ContribAuthor ">Elita  Yuliantie</span>, <span class="hlFld-ContribAuthor ">Linshan  Xie</span>, <span class="hlFld-ContribAuthor ">Houchao  Tao</span>, <span class="hlFld-ContribAuthor ">Jianjun  Cheng</span>, <span class="hlFld-ContribAuthor ">Qing  Liu</span>, <span class="hlFld-ContribAuthor ">Suwen  Zhao</span>, <span class="hlFld-ContribAuthor ">Wenqing  Shui</span>, <span class="hlFld-ContribAuthor ">Yi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ming-Wei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">G protein-coupled receptors: structure- and function-based drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2021,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-020-00435-w" title="DOI URL">https://doi.org/10.1038/s41392-020-00435-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-020-00435-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-020-00435-w%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DG%252Bprotein-coupled%252Breceptors%25253A%252Bstructure-%252Band%252Bfunction-based%252Bdrug%252Bdiscovery%26aulast%3DYang%26aufirst%3DDehua%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qing  Wang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Jianhui  Huang</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, Function, and Pharmaceutical Ligands of 5-Hydroxytryptamine 2B Receptor. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (2)
                                     , 76. <a href="https://doi.org/10.3390/ph14020076" title="DOI URL">https://doi.org/10.3390/ph14020076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14020076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14020076%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DStructure%25252C%252BFunction%25252C%252Band%252BPharmaceutical%252BLigands%252Bof%252B5-Hydroxytryptamine%252B2B%252BReceptor%26aulast%3DWang%26aufirst%3DQing%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D2%26spage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oleksandr  Grytsai</span>, <span class="hlFld-ContribAuthor ">Cyril  Ronco</span>, <span class="hlFld-ContribAuthor ">Rachid  Benhida</span>. </span><span class="cited-content_cbyCitation_article-title">Synthetic accesses to biguanide compounds. </span><span class="cited-content_cbyCitation_journal-name">Beilstein Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>17 </em>, 1001-1040. <a href="https://doi.org/10.3762/bjoc.17.82" title="DOI URL">https://doi.org/10.3762/bjoc.17.82</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3762/bjoc.17.82&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3762%2Fbjoc.17.82%26sid%3Dliteratum%253Aachs%26jtitle%3DBeilstein%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthetic%252Baccesses%252Bto%252Bbiguanide%252Bcompounds%26aulast%3DGrytsai%26aufirst%3DOleksandr%26date%3D2021%26date%3D2021%26volume%3D17%26spage%3D1001%26epage%3D1040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Viktor  Dmitriev</span>, <span class="hlFld-ContribAuthor ">Tatiana  Vedekhina</span>, <span class="hlFld-ContribAuthor ">Alexander  Sapegin</span>, <span class="hlFld-ContribAuthor ">Mikhail  Krasavin</span>. </span><span class="cited-content_cbyCitation_article-title">Convenient Assembly of Privileged (Hetero)Arene‐Fused Benzo[1.4]oxazepines via Two Tandem S
              
                N
              
              Ar Events. Part 3 – (Hetero)arene‐Fused benzo[
              f
              ]imidazo[1,2‐
              d
              ][1,4]oxazepines. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (26)
                                     , 7873-7879. <a href="https://doi.org/10.1002/slct.202002033" title="DOI URL">https://doi.org/10.1002/slct.202002033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202002033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202002033%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DConvenient%252BAssembly%252Bof%252BPrivileged%252B%252528Hetero%252529Arene%2525E2%252580%252590Fused%252BBenzo%25255B1.4%25255Doxazepines%252Bvia%252BTwo%252BTandem%252BS%252BN%252BAr%252BEvents.%252BPart%2525E2%252580%2525853%252B%2525E2%252580%252593%252B%252528Hetero%252529arene%2525E2%252580%252590Fused%252Bbenzo%25255B%252Bf%252B%25255Dimidazo%25255B1%25252C2%2525E2%252580%252590%252Bd%252B%25255D%25255B1%25252C4%25255Doxazepines%26aulast%3DDmitriev%26aufirst%3DViktor%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D26%26spage%3D7873%26epage%3D7879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jincai  Yang</span>, <span class="hlFld-ContribAuthor ">Cheng  Shen</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting or Pretending: Artificial Intelligence for Protein-Ligand Interactions Lack of Sufficiently Large and Unbiased Datasets. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.00069" title="DOI URL">https://doi.org/10.3389/fphar.2020.00069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.00069%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DPredicting%252Bor%252BPretending%25253A%252BArtificial%252BIntelligence%252Bfor%252BProtein-Ligand%252BInteractions%252BLack%252Bof%252BSufficiently%252BLarge%252Band%252BUnbiased%252BDatasets%26aulast%3DYang%26aufirst%3DJincai%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhe  Wang</span>, <span class="hlFld-ContribAuthor ">Huiyong  Sun</span>, <span class="hlFld-ContribAuthor ">Chao  Shen</span>, <span class="hlFld-ContribAuthor ">Xueping  Hu</span>, <span class="hlFld-ContribAuthor ">Junbo  Gao</span>, <span class="hlFld-ContribAuthor ">Dan  Li</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Cao</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Combined strategies in structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2020,</strong> <em>22 </em>
                                    (6)
                                     , 3149-3159. <a href="https://doi.org/10.1039/C9CP06303J" title="DOI URL">https://doi.org/10.1039/C9CP06303J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CP06303J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CP06303J%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DCombined%252Bstrategies%252Bin%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DWang%26aufirst%3DZhe%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D6%26spage%3D3149%26epage%3D3159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander  Sapegin</span>, <span class="hlFld-ContribAuthor ">Mikhail  Krasavin</span>. </span><span class="cited-content_cbyCitation_article-title">Convenient Assembly of Privileged (Hetero)Arene-Fused Benzo[1.4]oxazepines via Two Tandem S
              
                N
              
              Ar Events. Part 1 - On the Importance of the Intermittent Smiles Rearrangement. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>2019 </em>
                                    (31-32)
                                     , 5234-5241. <a href="https://doi.org/10.1002/ejoc.201900263" title="DOI URL">https://doi.org/10.1002/ejoc.201900263</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201900263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201900263%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DConvenient%252BAssembly%252Bof%252BPrivileged%252B%252528Hetero%252529Arene-Fused%252BBenzo%25255B1.4%25255Doxazepines%252Bvia%252BTwo%252BTandem%252BS%252BN%252BAr%252BEvents.%252BPart%252B1%252B-%252BOn%252Bthe%252BImportance%252Bof%252Bthe%252BIntermittent%252BSmiles%252BRearrangement%26aulast%3DSapegin%26aufirst%3DAlexander%26date%3D2019%26date%3D2019%26volume%3D2019%26issue%3D31-32%26spage%3D5234%26epage%3D5241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yin-Shu  Wang</span>, <span class="hlFld-ContribAuthor ">En-Kang  Wang</span>, <span class="hlFld-ContribAuthor ">Yang-Yang  Meng</span>, <span class="hlFld-ContribAuthor ">Zi-Juan  Bi</span>, <span class="hlFld-ContribAuthor ">Jian-Ye  Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in understanding relationship between 5-hydroxytryptamine and its receptors and intestinal dysmotility in irritable bowel syndrome. </span><span class="cited-content_cbyCitation_journal-name">World Chinese Journal of Digestology</span><span> <strong>2019,</strong> <em>27 </em>
                                    (15)
                                     , 954-960. <a href="https://doi.org/10.11569/wcjd.v27.i15.954" title="DOI URL">https://doi.org/10.11569/wcjd.v27.i15.954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.11569/wcjd.v27.i15.954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.11569%2Fwcjd.v27.i15.954%26sid%3Dliteratum%253Aachs%26jtitle%3DWorld%2520Chinese%2520Journal%2520of%2520Digestology%26atitle%3DAdvances%252Bin%252Bunderstanding%252Brelationship%252Bbetween%252B5-hydroxytryptamine%252Band%252Bits%252Breceptors%252Band%252Bintestinal%252Bdysmotility%252Bin%252Birritable%252Bbowel%252Bsyndrome%26aulast%3DWang%26aufirst%3DYin-Shu%26date%3D2019%26volume%3D27%26issue%3D15%26spage%3D954%26epage%3D960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paulo J.  Costa</span>, <span class="hlFld-ContribAuthor ">Rafael  Nunes</span>, <span class="hlFld-ContribAuthor ">Diogo  Vila-Viçosa</span>. </span><span class="cited-content_cbyCitation_article-title">Halogen bonding in halocarbon-protein complexes and computational tools for rational drug design. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (8)
                                     , 805-820. <a href="https://doi.org/10.1080/17460441.2019.1619692" title="DOI URL">https://doi.org/10.1080/17460441.2019.1619692</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1619692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1619692%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DHalogen%252Bbonding%252Bin%252Bhalocarbon-protein%252Bcomplexes%252Band%252Bcomputational%252Btools%252Bfor%252Brational%252Bdrug%252Bdesign%26aulast%3DCosta%26aufirst%3DPaulo%2BJ.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D8%26spage%3D805%26epage%3D820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sheyda  Ranjbar</span>, <span class="hlFld-ContribAuthor ">Seyed Afshin  Seyednejad</span>, <span class="hlFld-ContribAuthor ">Shekoufeh  Nikfar</span>, <span class="hlFld-ContribAuthor ">Roja  Rahimi</span>, <span class="hlFld-ContribAuthor ">Mohammad  Abdollahi</span>. </span><span class="cited-content_cbyCitation_article-title">How can we develop better antispasmodics for irritable bowel syndrome?. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (6)
                                     , 549-562. <a href="https://doi.org/10.1080/17460441.2019.1593369" title="DOI URL">https://doi.org/10.1080/17460441.2019.1593369</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1593369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1593369%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DHow%252Bcan%252Bwe%252Bdevelop%252Bbetter%252Bantispasmodics%252Bfor%252Birritable%252Bbowel%252Bsyndrome%25253F%26aulast%3DRanjbar%26aufirst%3DSheyda%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D6%26spage%3D549%26epage%3D562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wanchao  Yin</span>, <span class="hlFld-ContribAuthor ">X. Edward  Zhou</span>, <span class="hlFld-ContribAuthor ">Dehua  Yang</span>, <span class="hlFld-ContribAuthor ">Parker W.  de Waal</span>, <span class="hlFld-ContribAuthor ">Meitian  Wang</span>, <span class="hlFld-ContribAuthor ">Antao  Dai</span>, <span class="hlFld-ContribAuthor ">Xiaoqing  Cai</span>, <span class="hlFld-ContribAuthor ">Chia-Ying  Huang</span>, <span class="hlFld-ContribAuthor ">Ping  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaoxi  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Yin</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Jiang  Wang</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>, <span class="hlFld-ContribAuthor ">Martin  Caffrey</span>, <span class="hlFld-ContribAuthor ">Karsten  Melcher</span>, <span class="hlFld-ContribAuthor ">Yechun  Xu</span>, <span class="hlFld-ContribAuthor ">Ming-Wei  Wang</span>, <span class="hlFld-ContribAuthor ">H. Eric  Xu</span>, <span class="hlFld-ContribAuthor ">Yi  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structure of the human 5-HT1B serotonin receptor bound to an inverse agonist. </span><span class="cited-content_cbyCitation_journal-name">Cell Discovery</span><span> <strong>2018,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41421-018-0009-2" title="DOI URL">https://doi.org/10.1038/s41421-018-0009-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41421-018-0009-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41421-018-0009-2%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Discovery%26atitle%3DCrystal%252Bstructure%252Bof%252Bthe%252Bhuman%252B5-HT1B%252Bserotonin%252Breceptor%252Bbound%252Bto%252Ban%252Binverse%252Bagonist%26aulast%3DYin%26aufirst%3DWanchao%26date%3D2018%26date%3D2018%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krzysztof  Rataj</span>, <span class="hlFld-ContribAuthor ">Ádám  Kelemen</span>, <span class="hlFld-ContribAuthor ">José  Brea</span>, <span class="hlFld-ContribAuthor ">María  Loza</span>, <span class="hlFld-ContribAuthor ">Andrzej  Bojarski</span>, <span class="hlFld-ContribAuthor ">György  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Fingerprint-Based Machine Learning Approach to Identify Potent and Selective 5-HT2BR Ligands. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (5)
                                     , 1137. <a href="https://doi.org/10.3390/molecules23051137" title="DOI URL">https://doi.org/10.3390/molecules23051137</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23051137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23051137%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DFingerprint-Based%252BMachine%252BLearning%252BApproach%252Bto%252BIdentify%252BPotent%252Band%252BSelective%252B5-HT2BR%252BLigands%26aulast%3DRataj%26aufirst%3DKrzysztof%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D5%26spage%3D1137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moustafa T.  Gabr</span>, <span class="hlFld-ContribAuthor ">Mohammed S.  Abdel-Raziq</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore-based tailoring of biphenyl amide derivatives as selective 5-hydroxytryptamine 2B receptor antagonists. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2018,</strong> <em>9 </em>
                                    (6)
                                     , 1069-1075. <a href="https://doi.org/10.1039/C8MD00204E" title="DOI URL">https://doi.org/10.1039/C8MD00204E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8MD00204E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8MD00204E%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DPharmacophore-based%252Btailoring%252Bof%252Bbiphenyl%252Bamide%252Bderivatives%252Bas%252Bselective%252B5-hydroxytryptamine%252B2B%252Breceptor%252Bantagonists%26aulast%3DGabr%26aufirst%3DMoustafa%2BT.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D6%26spage%3D1069%26epage%3D1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Congreve</span>, <span class="hlFld-ContribAuthor ">A.  Bortolato</span>, <span class="hlFld-ContribAuthor ">G.  Brown</span>, <span class="hlFld-ContribAuthor ">R.M.  Cooke</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling and Design for Membrane Protein Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 145-188. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12358-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12358-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12358-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12358-3%26sid%3Dliteratum%253Aachs%26atitle%3DModeling%252Band%252BDesign%252Bfor%252BMembrane%252BProtein%252BTargets%26aulast%3DCongreve%26aufirst%3DM.%26date%3D2017%26spage%3D145%26epage%3D188%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander  Sapegin</span>, <span class="hlFld-ContribAuthor ">Elena  Reutskaya</span>, <span class="hlFld-ContribAuthor ">Alexey  Smirnov</span>, <span class="hlFld-ContribAuthor ">Mikhail  Korsakov</span>, <span class="hlFld-ContribAuthor ">Mikhail  Krasavin</span>. </span><span class="cited-content_cbyCitation_article-title">Facile entry into structurally diverse, privileged, (hetero)arene-fused N-alkoxy 3,4-dihydrobenzo[f][1,4]oxazepin-5(2H)-ones. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2016,</strong> <em>57 </em>
                                    (52)
                                     , 5877-5880. <a href="https://doi.org/10.1016/j.tetlet.2016.11.064" title="DOI URL">https://doi.org/10.1016/j.tetlet.2016.11.064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2016.11.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2016.11.064%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DFacile%252Bentry%252Binto%252Bstructurally%252Bdiverse%25252C%252Bprivileged%25252C%252B%252528hetero%252529arene-fused%252BN-alkoxy%252B3%25252C4-dihydrobenzo%25255Bf%25255D%25255B1%25252C4%25255Doxazepin-5%2525282H%252529-ones%26aulast%3DSapegin%26aufirst%3DAlexander%26date%3D2016%26volume%3D57%26issue%3D52%26spage%3D5877%26epage%3D5880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maura  Corsetti</span>, <span class="hlFld-ContribAuthor ">Peter  Whorwell</span>. </span><span class="cited-content_cbyCitation_article-title">Novel pharmacological therapies for irritable bowel syndrome. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Gastroenterology & Hepatology</span><span> <strong>2016,</strong> <em>10 </em>
                                    (7)
                                     , 807-815. <a href="https://doi.org/10.1586/17474124.2016.1158099" title="DOI URL">https://doi.org/10.1586/17474124.2016.1158099</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1586/17474124.2016.1158099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1586%2F17474124.2016.1158099%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Gastroenterology%2520%2526%2520Hepatology%26atitle%3DNovel%252Bpharmacological%252Btherapies%252Bfor%252Birritable%252Bbowel%252Bsyndrome%26aulast%3DCorsetti%26aufirst%3DMaura%26date%3D2016%26date%3D2016%26volume%3D10%26issue%3D7%26spage%3D807%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding site refinement of the 5-HT<sub>2B</sub> crystal structure. (A) Chemical structures and biological activities of doxepin and carvedilol. (B) Comparison of the doxepin induced-fit model (cyan) with the 5-HT<sub>2B</sub> crystal structure (tan, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4">4IB4</a>). (C) Solvent property analysis of the 5-HT<sub>2B</sub> binding pocket. Predicted water clusters are shown as spheres. The volume of spheres represents their occupancy in the simulation, and the color represents their free energy. Blue indicates that the water molecule is energy favorable, while red indicates it is energy unfavorable. (D) The enrichment profile of the doxepin induced-fit model. The percentage of ligands found (<i>y</i>-axis) plotted as a function of the percentage of the docking ranked database (<i>x</i>-axis in logarithmic scale).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Flowchart of the structure-based hierarchical virtual screening strategy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Experimental validation of the identified compounds. (A) Scaffold of the identified compounds. (B) Inhibitory activity on the 5-HT<sub>2B</sub> receptor at 10 μM and 1 μM concentrations. (C) Chemical structure of compound <b>9</b>. (D) The dose-dependent inhibitory response curve of compound <b>9</b> on the 5-HT<sub>2B</sub> receptor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Predicted binding mode of compound <b>9</b>. (B) Correlation between the experimental pIC<sub>50</sub> and the computed Δ<i>E</i> based on the predicted binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Potential ligand binding selectivity mechanism of the 5-HT<sub>2B</sub> receptor. (A) Crystal structure comparsion between the 5-HT<sub>2B</sub> (green, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4">4IB4</a>) and 5-HT<sub>1B</sub> (cyan, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IAR">4IAR</a>) receptors. (B) Sequence alignment of the ECL II and TM6 regions among 5-HTRs and other GPCRs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of visceral hypersensitivity in IBS rat model. (A and B) Therapeutic effects of compound <b>15</b> in IBS rats. The pain threshold decreased in response to CRD (at score 3 and score 4). (C) Overexpression of mRNA for the 5-HT<sub>2B</sub> receptor in IBS rat colons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/medium/jm-2015-01631t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedure for Compound Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01631/20160122/images/large/jm-2015-01631t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01631&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reaction conditions: (i) concd HCl, EtOH, reflux; (ii) pyridine, EtOH, 120 °C.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78752" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78752" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 68 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Filmore, D.</span><span> </span><span class="NLM_article-title">It’s a GPCR world</span> <span class="citation_source-journal">Mod. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVKnsbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=24-28&author=D.+Filmore&title=It%E2%80%99s+a+GPCR+world"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">It's a GPCR world</span></div><div class="casAuthors">Filmore, David</div><div class="citationInfo"><span class="NLM_cas:title">Modern Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">24-26, 28</span>CODEN:
                <span class="NLM_cas:coden">MDDIFT</span>;
        ISSN:<span class="NLM_cas:issn">1099-8209</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with refs.  Cell-based screening assays and structural studies are fueling G-protein coupled receptors (GPCR) as one of the most popular classes of investigational drug targets.  Besides the many GPCRs that function as basic receptors for sensory functions like seeing and smelling, which are not prime therapeutic targets, there are more than 300 other GPCRs up for grabs for drug discovery initiatives.  About 200 of these, a portion of which account for currently marketed GPCR drug targets, have known natural ligands.  And ligands are not known for about 150 nonsensory receptors identified as GPCRs from the Human Genome Project.  Several firms have completely focus on GPCRs because of the largely uncharted path of discovery they offer.  GPCR seem to be a good bet for future drug discovery successes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUWfG2qMCe7rVg90H21EOLACvtfcHk0liEC9_d1vCoZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVKnsbfN&md5=41753a459c3c8a63bf1ff48d6e2585fd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFilmore%26aufirst%3DD.%26atitle%3DIt%25E2%2580%2599s%2520a%2520GPCR%2520world%26jtitle%3DMod.%2520Drug%2520Discovery%26date%3D2004%26volume%3D7%26spage%3D24%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of beta<sub>2</sub>-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span><span class="refDoi"> DOI: 10.1073/pnas.0812657106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1073%2Fpnas.0812657106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=19342484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta2-adrenergic+receptor+ligands&doi=10.1073%2Fpnas.0812657106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of β2-adrenergic receptor ligands</span></div><div class="casAuthors">Kolb, Peter; Rosenbaum, Daniel M.; Irwin, John J.; Fung, Juan Jose; Kobilka, Brian K.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6843-6848</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aminergic G protein-coupled receptors (GPCRs) have been a major focus of pharmaceutical research for many years.  Due partly to the lack of reliable receptor structures, drug discovery efforts have been largely ligand-based.  The recently detd. X-ray structure of the β2-adrenergic receptor offers an opportunity to investigate the advantages and limitations inherent in a structure-based approach to ligand discovery against this and related GPCR targets.  Approx. 1 million com. available, "lead-like" mols. were docked against the β2-adrenergic receptor structure.  On testing of 25 high-ranking mols., 6 were active with binding affinities <4 μM, with the best mol. binding with a Ki of 9 nM (95% confidence interval 7-10 nM).  Five of these mols. were inverse agonists.  The high hit rate, the high affinity of the most potent mol., the discovery of unprecedented chemotypes among the new inhibitors, and the apparent bias toward inverse agonists among the docking hits, have implications for structure-based approaches against GPCRs that recognize small org. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo89kM84Li_H7Vg90H21EOLACvtfcHk0liEC9_d1vCoZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsV2qsro%253D&md5=4d1a4cb2aa3925aa4c99c6b0496417a7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0812657106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0812657106%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta2-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848%26doi%3D10.1073%2Fpnas.0812657106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of A<sub>2A</sub> adenosine receptor ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3748</span><span class="NLM_x">–</span> <span class="NLM_lpage">3755</span><span class="refDoi"> DOI: 10.1021/jm100240h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.+G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-based+discovery+of+A2A+adenosine+receptor+ligands&doi=10.1021%2Fjm100240h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0lh2Y2Wy8fX5Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520A2A%2520adenosine%2520receptor%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755%26doi%3D10.1021%2Fjm100240h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D<sub>3</sub> receptor homology model and crystal structure</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span><span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lh2Y2Wy8fX5Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziarek, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heveker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkman, B. F.</span><span> </span><span class="NLM_article-title">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5517</span><span class="NLM_x">–</span> <span class="NLM_lpage">5522</span><span class="refDoi"> DOI: 10.1073/pnas.1120431109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1073%2Fpnas.1120431109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=22431600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5517-5522&author=M.+M.+Mysingerauthor=D.+R.+Weissauthor=J.+J.+Ziarekauthor=S.+Gravelauthor=A.+K.+Doakauthor=J.+Karpiakauthor=N.+Hevekerauthor=B.+K.+Shoichetauthor=B.+F.+Volkman&title=Structure-based+ligand+discovery+for+the+protein-protein+interface+of+chemokine+receptor+CXCR4&doi=10.1073%2Fpnas.1120431109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span></div><div class="casAuthors">Mysinger, Michael M.; Weiss, Dahlia R.; Ziarek, Joshua J.; Gravel, Stephanie; Doak, Allison K.; Karpiak, Joel; Heveker, Nikolaus; Shoichet, Brian K.; Volkman, Brian F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5517-5522, S5517/1-S5517/28</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are key signaling mols. and are intensely studied.  Whereas GPCRs recognizing small-mols. have been successfully targeted for drug discovery, protein-recognizing GPCRs, such as the chemokine receptors, claim few drugs or even useful small mol. reagents.  This reflects both the difficulties that attend protein-protein interface inhibitor discovery, and the lack of structures for these targets.  Imminent structure detn. of chemokine receptor CXCR4 motivated docking screens for new ligands against a homol. model and subsequently the crystal structure.  More than 3 million mols. were docked against the model and then against the crystal structure; 24 and 23 high-scoring compds. from the resp. screens were tested exptl.  Docking against the model yielded only one antagonist, which resembled known ligands and lacked specificity, whereas the crystal structure docking yielded four that were dissimilar to previously known scaffolds and apparently specific.  Intriguingly, several were potent and relatively small, with IC50 values as low as 306 nM, ligand efficiencies as high as 0.36, and with efficacy in cellular chemotaxis.  The potency and efficiency of these mols. has few precedents among protein-protein interface inhibitors, and supports structure-based efforts to discover leads for chemokine GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeJKWh0aH7FLVg90H21EOLACvtfcHk0ljCzCOG_lP3Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D&md5=762616e6eb0eebc0d983964d71499e75</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120431109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120431109%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DZiarek%26aufirst%3DJ.%2BJ.%26aulast%3DGravel%26aufirst%3DS.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DHeveker%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DVolkman%26aufirst%3DB.%2BF.%26atitle%3DStructure-based%2520ligand%2520discovery%2520for%2520the%2520protein-protein%2520interface%2520of%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5517%26epage%3D5522%26doi%3D10.1073%2Fpnas.1120431109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Kruse, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eitel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gmeiner, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Muscarinic receptors as model targets and antitargets for structure-based ligand discovery</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">528</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1124/mol.113.087551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1124%2Fmol.113.087551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=528-540&author=A.+C.+Kruseauthor=D.+R.+Weissauthor=M.+Rossiauthor=J.+Huauthor=K.+Huauthor=K.+Eitelauthor=P.+Gmeinerauthor=J.+Wessauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Muscarinic+receptors+as+model+targets+and+antitargets+for+structure-based+ligand+discovery&doi=10.1124%2Fmol.113.087551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.087551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.087551%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DRossi%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DK.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMuscarinic%2520receptors%2520as%2520model%2520targets%2520and%2520antitargets%2520for%2520structure-based%2520ligand%2520discovery%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26spage%3D528%26epage%3D540%26doi%3D10.1124%2Fmol.113.087551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strachan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefkowitz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Conformation guides molecular efficacy in docking screens of activated beta-2 adrenergic G protein coupled receptor</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1018</span><span class="NLM_x">–</span> <span class="NLM_lpage">1026</span><span class="refDoi"> DOI: 10.1021/cb400103f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400103f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1018-1026&author=D.+R.+Weissauthor=S.+Ahnauthor=M.+F.+Sassanoauthor=A.+Kleistauthor=X.+Zhuauthor=R.+Strachanauthor=B.+L.+Rothauthor=R.+J.+Lefkowitzauthor=B.+K.+Shoichet&title=Conformation+guides+molecular+efficacy+in+docking+screens+of+activated+beta-2+adrenergic+G+protein+coupled+receptor&doi=10.1021%2Fcb400103f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb400103f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400103f%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DAhn%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DKleist%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DStrachan%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DConformation%2520guides%2520molecular%2520efficacy%2520in%2520docking%2520screens%2520of%2520activated%2520beta-2%2520adrenergic%2520G%2520protein%2520coupled%2520receptor%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1018%26epage%3D1026%26doi%3D10.1021%2Fcb400103f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Rodríguez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brea, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loza, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlsson, J.</span><span> </span><span class="NLM_article-title">Structure-based discovery of selective serotonin 5-HT<sub>1B</sub> receptor ligands</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1140</span><span class="NLM_x">–</span> <span class="NLM_lpage">1151</span><span class="refDoi"> DOI: 10.1016/j.str.2014.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.str.2014.05.017" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=1140-1151&author=D.+Rodr%C3%ADguezauthor=J.+Breaauthor=M.+I.+Lozaauthor=J.+Carlsson&title=Structure-based+discovery+of+selective+serotonin+5-HT1B+receptor+ligands&doi=10.1016%2Fj.str.2014.05.017"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DBrea%26aufirst%3DJ.%26aulast%3DLoza%26aufirst%3DM.%2BI.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DStructure-based%2520discovery%2520of%2520selective%2520serotonin%25205-HT1B%2520receptor%2520ligands%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D1140%26epage%3D1151%26doi%3D10.1016%2Fj.str.2014.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vardy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCorvy, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siu, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Structural features for functional selectivity at serotonin receptors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">340</span><span class="NLM_x">, </span> <span class="NLM_fpage">615</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1126/science.1232808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1126%2Fscience.1232808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23519215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslWksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2013&pages=615-619&author=D.+Wackerauthor=C.+Wangauthor=V.+Katritchauthor=G.+W.+Hanauthor=X.+P.+Huangauthor=E.+Vardyauthor=J.+D.+McCorvyauthor=Y.+Jiangauthor=M.+Chuauthor=F.+Y.+Siuauthor=W.+Liuauthor=H.+E.+Xuauthor=V.+Cherezovauthor=B.+L.+Rothauthor=R.+C.+Stevens&title=Structural+features+for+functional+selectivity+at+serotonin+receptors&doi=10.1126%2Fscience.1232808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Features for Functional Selectivity at Serotonin Receptors</span></div><div class="casAuthors">Wacker, Daniel; Wang, Chong; Katritch, Vsevolod; Han, Gye Won; Huang, Xi-Ping; Vardy, Eyal; McCorvy, John D.; Jiang, Yi; Chu, Meihua; Siu, Fai Yiu; Liu, Wei; Xu, H. Eric; Cherezov, Vadim; Roth, Bryan L.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">340</span>
        (<span class="NLM_cas:issue">6132</span>),
    <span class="NLM_cas:pages">615-619</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Drugs active at G protein-coupled receptors (GPCRs) can differentially modulate either canonical or noncanonical signaling pathways via a phenomenon known as functional selectivity or biased signaling.  The authors report biochem. studies showing that the hallucinogen lysergic acid diethylamide, its precursor ergotamine (ERG), and related ergolines display strong functional selectivity for β-arrestin signaling at the 5-HT2B 5-hydroxytryptamine (5-HT) receptor, whereas they are relatively unbiased at the 5-HT1B receptor.  To investigate the structural basis for biased signaling, the authors detd. the crystal structure of the human 5-HT2B receptor bound to ERG and compared it with the 5-HT1B/ERG structure.  Given the relatively poor understanding of GPCR structure and function to date, insight into different GPCR signaling pathways is important to better understand both adverse and favorable therapeutic activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreo_QegM-j7bVg90H21EOLACvtfcHk0ljijHVhluyn-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslWksb0%253D&md5=16e96d0255be3ac367aeda7187aa4c15</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1232808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1232808%26sid%3Dliteratum%253Aachs%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DVardy%26aufirst%3DE.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DSiu%26aufirst%3DF.%2BY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DH.%2BE.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520features%2520for%2520functional%2520selectivity%2520at%2520serotonin%2520receptors%26jtitle%3DScience%26date%3D2013%26volume%3D340%26spage%3D615%26epage%3D619%26doi%3D10.1126%2Fscience.1232808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wacker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gati, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weierstall, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barty, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zatsepin, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messerschmidt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koglin, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seibert, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromme, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupitz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rendek, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotjohann, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromme, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirian, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beyerlein, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caffrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spence, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span> </span><span class="NLM_article-title">Serial femtosecond crystallography of G protein-coupled receptors</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">342</span><span class="NLM_x">, </span> <span class="NLM_fpage">1521</span><span class="NLM_x">–</span> <span class="NLM_lpage">1524</span><span class="refDoi"> DOI: 10.1126/science.1244142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1126%2Fscience.1244142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=24357322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2013&pages=1521-1524&author=W.+Liuauthor=D.+Wackerauthor=C.+Gatiauthor=G.+W.+Hanauthor=D.+Jamesauthor=D.+Wangauthor=G.+Nelsonauthor=U.+Weierstallauthor=V.+Katritchauthor=A.+Bartyauthor=N.+A.+Zatsepinauthor=D.+Liauthor=M.+Messerschmidtauthor=S.+Boutetauthor=G.+J.+Williamsauthor=J.+E.+Koglinauthor=M.+M.+Seibertauthor=C.+Wangauthor=S.+T.+Shahauthor=S.+Basuauthor=R.+Frommeauthor=C.+Kupitzauthor=K.+N.+Rendekauthor=I.+Grotjohannauthor=P.+Frommeauthor=R.+A.+Kirianauthor=K.+R.+Beyerleinauthor=T.+A.+Whiteauthor=H.+N.+Chapmanauthor=M.+Caffreyauthor=J.+C.+Spenceauthor=R.+C.+Stevensauthor=V.+Cherezov&title=Serial+femtosecond+crystallography+of+G+protein-coupled+receptors&doi=10.1126%2Fscience.1244142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Serial Femtosecond Crystallography of G Protein-Coupled Receptors</span></div><div class="casAuthors">Liu, Wei; Wacker, Daniel; Gati, Cornelius; Han, Gye Won; James, Daniel; Wang, Dingjie; Nelson, Garrett; Weierstall, Uwe; Katritch, Vsevolod; Barty, Anton; Zatsepin, Nadia A.; Li, Dianfan; Messerschmidt, Marc; Boutet, Sebastien; Williams, Garth J.; Koglin, Jason E.; Seibert, M. Marvin; Wang, Chong; Shah, Syed T. A.; Basu, Shibom; Fromme, Raimund; Kupitz, Christopher; Rendek, Kimberley N.; Grotjohann, Ingo; Fromme, Petra; Kirian, Richard A.; Beyerlein, Kenneth R.; White, Thomas A.; Chapman, Henry N.; Caffrey, Martin; Spence, John C. H.; Stevens, Raymond C.; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">6165</span>),
    <span class="NLM_cas:pages">1521-1524</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">X-ray crystallog. of G protein-coupled receptors and other membrane proteins is hampered by difficulties assocd. with growing sufficiently large crystals that withstand radiation damage and yield high-resoln. data at synchrotron sources.  The authors used an x-ray free-electron laser (XFEL) with individual 50-fs-duration x-ray pulses to minimize radiation damage and obtained a high-resoln. room-temp. structure of a human serotonin receptor using sub-10-μm microcrystals grown in a membrane mimetic matrix known as lipidic cubic phase.  Compared with the structure solved by using traditional microcrystallog. from cryo-cooled crystals of about two orders of magnitude larger vol., the room-temp. XFEL structure displays a distinct distribution of thermal motions and conformations of residues that likely more accurately represent the receptor structure and dynamics in a cellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5GiymqAvRprVg90H21EOLACvtfcHk0li7uDv8QPbtJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE&md5=a632160c4265cddfebb28ef9b002b38a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244142%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DKoglin%26aufirst%3DJ.%2BE.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShah%26aufirst%3DS.%2BT.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DRendek%26aufirst%3DK.%2BN.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DKirian%26aufirst%3DR.%2BA.%26aulast%3DBeyerlein%26aufirst%3DK.%2BR.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DSerial%2520femtosecond%2520crystallography%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DScience%26date%3D2013%26volume%3D342%26spage%3D1521%26epage%3D1524%26doi%3D10.1126%2Fscience.1244142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Rothman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savage, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauser, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBride, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Evidence for possible involvement of 5-HT<sub>2B</sub> receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span> <span class="citation_source-journal">Circulation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">2836</span><span class="NLM_x">–</span> <span class="NLM_lpage">2841</span><span class="refDoi"> DOI: 10.1161/01.CIR.102.23.2836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1161%2F01.CIR.102.23.2836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=11104741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2000&pages=2836-2841&author=R.+B.+Rothmanauthor=M.+H.+Baumannauthor=J.+E.+Savageauthor=L.+Rauserauthor=A.+McBrideauthor=S.+J.+Hufeisenauthor=B.+L.+Roth&title=Evidence+for+possible+involvement+of+5-HT2B+receptors+in+the+cardiac+valvulopathy+associated+with+fenfluramine+and+other+serotonergic+medications&doi=10.1161%2F01.CIR.102.23.2836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for possible involvement of 5-HT2B receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span></div><div class="casAuthors">Rothman, Richard B.; Baumann, Michael H.; Savage, Jason E.; Rauser, Laura; McBride, Ace; Hufeisen, Sandra J.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2836-2841</span>CODEN:
                <span class="NLM_cas:coden">CIRCAZ</span>;
        ISSN:<span class="NLM_cas:issn">0009-7322</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Serotonergic medications with various mechanisms of action are used to treat psychiatric disorders and are being investigated as treatments for drug dependence.  The occurrence of fenfluramine-assocd. valvular heart disease (VHD) has raised concerns that other serotonergic medications might also increase the risk of developing VHD.  We hypothesized that fenfluramine or its metabolite norfenfluramine and other medications known to produce VHD have preferentially high affinities for a particular serotonin receptor subtype capable of stimulating mitogenesis.  Medications known or suspected to cause VHD (pos. controls) and medications not assocd. with VHD (neg. controls) were screened for activity at 11 cloned serotonin receptor subtypes by use of ligand-binding methods and functional assays.  The pos. control drugs were (±)-fenfluramine; (+)-fenfluramine; (-)-fenfluramine; its metabolites (±)-norfenfluramine, (+)-norfenfluramine, and (-)-norfenfluramine; ergotamine; and methysergide and its metabolite methylergonovine.  The neg. control drugs were phentermine, fluoxetine, its metabolite norfluoxetine, and trazodone and its active metabolite m-chlorophenylpiperazine.  (±)-, (+)-, And (-)-Norfenfluramine, ergotamine, and methylergonovine all had preferentially high affinities for the cloned human serotonin 5-HT2B receptor and were partial to full agonists at the 5-HT2B receptor.  Our data imply that activation of 5-HT2B receptors is necessary to produce VHD and that serotonergic medications that do not activate 5-HT2B receptors are unlikely to produce VHD.  We suggest that all clin. available medications with serotonergic activity and their active metabolites be screened for agonist activity at 5-HT2B receptors and that clinicians should consider suspending their use of medications with significant activity at 5-HT2B receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6spUXOCLtLrVg90H21EOLACvtfcHk0ljv1ufnxGi7cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D&md5=6b8c6059885dd9b058859089914c61b3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1161%2F01.CIR.102.23.2836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.CIR.102.23.2836%26sid%3Dliteratum%253Aachs%26aulast%3DRothman%26aufirst%3DR.%2BB.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26aulast%3DSavage%26aufirst%3DJ.%2BE.%26aulast%3DRauser%26aufirst%3DL.%26aulast%3DMcBride%26aufirst%3DA.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DEvidence%2520for%2520possible%2520involvement%2520of%25205-HT2B%2520receptors%2520in%2520the%2520cardiac%2520valvulopathy%2520associated%2520with%2520fenfluramine%2520and%2520other%2520serotonergic%2520medications%26jtitle%3DCirculation%26date%3D2000%26volume%3D102%26spage%3D2836%26epage%3D2841%26doi%3D10.1161%2F01.CIR.102.23.2836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Hutcheson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merryman, W. D.</span><span> </span><span class="NLM_article-title">Serotonin receptors and heart valve disease--it was meant 2B</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">157</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.pharmthera.2011.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=21440001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaksLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2011&pages=146-157&author=J.+D.+Hutchesonauthor=V.+Setolaauthor=B.+L.+Rothauthor=W.+D.+Merryman&title=Serotonin+receptors+and+heart+valve+disease%2D%2Dit+was+meant+2B&doi=10.1016%2Fj.pharmthera.2011.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Serotonin receptors and heart valve disease-It was meant 2B</span></div><div class="casAuthors">Hutcheson, Joshua D.; Setola, Vincent; Roth, Bryan L.; Merryman, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-157</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Carcinoid heart disease was one of the first valvular pathologies studied in mol. detail, and early research identified serotonin produced by oncogenic enterochromaffin cells as the likely culprit in causing changes in heart valve tissue.  Researchers and physicians in the mid-1960s noted a connection between the use of several ergot-derived medications with structures similar to serotonin and the development of heart valve pathologies similar to those obsd. in carcinoid patients.  The exact serotonergic target that mediated valvular pathogenesis remained a mystery for many years until similar cases were reported in patients using the popular diet drug Fen-Phen in the late 1990s.  The Fen-Phen episode sparked renewed interest in serotonin-mediated valve disease, and studies led to the identification of the 5-HT2B receptor as the likely mol. target leading to heart valve tissue fibrosis.  Subsequent studies have identified numerous other activators of the 5-HT2B receptor, and consequently, the use of many of these mols. has been linked to heart valve disease.  Herein, we: review the mol. properties of the 5-HT2B receptor including factors that differentiate the 5-HT2B receptor from other 5-HT receptor subtypes, discuss the studies that led to the identification of the 5-HT2B receptor as the mediator of heart valve disease, present current efforts to identify potential valvulopathogens by screening for 5-HT2B receptor activity, and speculate on potential therapeutic benefits of 5-HT2B receptor targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4R8A9Ia-r3LVg90H21EOLACvtfcHk0ljv1ufnxGi7cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaksLnO&md5=b97c1992208a501b1959c2f48f3ca9cd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DHutcheson%26aufirst%3DJ.%2BD.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DMerryman%26aufirst%3DW.%2BD.%26atitle%3DSerotonin%2520receptors%2520and%2520heart%2520valve%2520disease--it%2520was%2520meant%25202B%26jtitle%3DPharmacol.%2520Ther.%26date%3D2011%26volume%3D132%26spage%3D146%26epage%3D157%26doi%3D10.1016%2Fj.pharmthera.2011.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Brea, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro-Palomino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeste, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cubero, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parraga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominguez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loza, M. I.</span><span> </span><span class="NLM_article-title">Emerging opportunities and concerns for drug discovery at serotonin 5-HT<sub>2B</sub> receptors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">493</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.2174/156802610791111524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.2174%2F156802610791111524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=20166944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFKntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=493-503&author=J.+Breaauthor=J.+Castro-Palominoauthor=S.+Yesteauthor=E.+Cuberoauthor=A.+Parragaauthor=E.+Dominguezauthor=M.+I.+Loza&title=Emerging+opportunities+and+concerns+for+drug+discovery+at+serotonin+5-HT2B+receptors&doi=10.2174%2F156802610791111524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging opportunities and concerns for drug discovery at serotonin 5-HT2B receptors</span></div><div class="casAuthors">Brea, Jose; Castro-Palomino, Julio; Yeste, Sandra; Cubero, Elena; Parraga, Antonio; Dominguez, Eduardo; Loza, Maria Isabel</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">493-503</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  5-HT2B receptors have been reported to play an important role at cardiac, intestinal and central levels, as well as in bone marrow formation and growth.  In the last decade, 5-HT2B receptors have also gained much attention as new targets in therapeutics, but also as off-targets because their activation along with the inhibition of serotonin transporters plays a significant role in the pathogenesis of 5-HT induced valvulopathy.  Taking this into account, the present review focuses on the new therapeutic applications of 5-HT2B receptor ligands as well as on the potential concerns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB-wKEI1AFFLVg90H21EOLACvtfcHk0ljv1ufnxGi7cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFKntbk%253D&md5=6c92f93e70cc97f377881d198d9eaf16</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F156802610791111524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610791111524%26sid%3Dliteratum%253Aachs%26aulast%3DBrea%26aufirst%3DJ.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%26aulast%3DYeste%26aufirst%3DS.%26aulast%3DCubero%26aufirst%3DE.%26aulast%3DParraga%26aufirst%3DA.%26aulast%3DDominguez%26aufirst%3DE.%26aulast%3DLoza%26aufirst%3DM.%2BI.%26atitle%3DEmerging%2520opportunities%2520and%2520concerns%2520for%2520drug%2520discovery%2520at%2520serotonin%25205-HT2B%2520receptors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D493%26epage%3D503%26doi%3D10.2174%2F156802610791111524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Panconesi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicuteri, R.</span><span> </span><span class="NLM_article-title">Headache induced by serotonergic agonists--a key to the interpretation of migraine pathogenesis?</span> <span class="citation_source-journal">Cephalalgia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span><span class="refDoi"> DOI: 10.1046/j.1468-2982.1997.1701003.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1046%2Fj.1468-2982.1997.1701003.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=3-14&author=A.+Panconesiauthor=R.+Sicuteri&title=Headache+induced+by+serotonergic+agonists%2D%2Da+key+to+the+interpretation+of+migraine+pathogenesis%3F&doi=10.1046%2Fj.1468-2982.1997.1701003.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1046%2Fj.1468-2982.1997.1701003.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1468-2982.1997.1701003.x%26sid%3Dliteratum%253Aachs%26aulast%3DPanconesi%26aufirst%3DA.%26aulast%3DSicuteri%26aufirst%3DR.%26atitle%3DHeadache%2520induced%2520by%2520serotonergic%2520agonists--a%2520key%2520to%2520the%2520interpretation%2520of%2520migraine%2520pathogenesis%253F%26jtitle%3DCephalalgia%26date%3D1997%26volume%3D17%26spage%3D3%26epage%3D14%26doi%3D10.1046%2Fj.1468-2982.1997.1701003.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Mucke, H.</span><span> </span><span class="NLM_article-title">MT-500 (POZEN)</span> <span class="citation_source-journal">Curr. Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">419</span><span class="NLM_x">–</span> <span class="NLM_lpage">421</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=419-421&author=H.+Mucke&title=MT-500+%28POZEN%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMucke%26aufirst%3DH.%26atitle%3DMT-500%2520%2528POZEN%2529%26jtitle%3DCurr.%2520Opin.%2520Invest.%2520Drugs%26date%3D2001%26volume%3D2%26spage%3D419%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Borman, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilford, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmer, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheldrick, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, G. S.</span><span> </span><span class="NLM_article-title">5-HT<sub>2B</sub> receptors play a key role in mediating the excitatory effects of 5-HT in human colon in vitro</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">1144</span><span class="NLM_x">–</span> <span class="NLM_lpage">1151</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0704571</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fsj.bjp.0704571" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=11877320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38Xis1ait7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2002&pages=1144-1151&author=R.+A.+Bormanauthor=N.+S.+Tilfordauthor=D.+W.+Harmerauthor=N.+Dayauthor=E.+S.+Ellisauthor=R.+L.+Sheldrickauthor=J.+Careyauthor=R.+A.+Colemanauthor=G.+S.+Baxter&title=5-HT2B+receptors+play+a+key+role+in+mediating+the+excitatory+effects+of+5-HT+in+human+colon+in+vitro&doi=10.1038%2Fsj.bjp.0704571"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">5-HT2B receptors play a key role in mediating the excitatory effects of 5-HT in human colon in vitro</span></div><div class="casAuthors">Borman, R. A.; Tilford, N. S.; Harmer, D. W.; Day, N.; Ellis, E. S.; Sheldrick, R. L. G.; Carey, J.; Coleman, R. A.; Baxter, G. S.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1144-1151</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">5-Hydroxytryptamine (5-HT) is known to produce a no. of different effects in the gastrointestinal tract of various species, and has been proposed to play a key role in a no. of intestinal disorders in man, including irritable bowel syndrome (IBS), although the receptors involved have yet to be established.  The aim of the present study was to investigate the distribution and function of 5-HT2B receptors in human colon, and to establish their possible role in the etiol. of IBS.  The distribution of 5-HT2B receptor mRNA and protein were investigated by quant. RT-PCR, Western anal. and immunocytochem.  High levels of both mRNA and protein for 5-HT2B receptors were found throughout the human gastrointestinal tract, and in particular in colon, where 5-HT2B receptors were found predominantly in the longitudinal and circular smooth muscle layers within the muscularis externa, and in the myenteric nerve plexus lying between these two layers.  Elec. field stimulation of longitudinal muscle prepns. of human colon mounted in organ baths resulted in neuronally mediated contractile responses, that were significantly potentiated by application of 5-HT (up to 10-7 M), with a pEC50 of 8.2.  The response to 5-HT was inhibited by a no. of selective 5-HT2B receptor antagonists.  This study has shown for the first time that, in contrast to animal studies, the excitatory effects of 5-HT in human colon are mediated by 5-HT2B receptors.  It is proposed that these receptors contribute to the putative 5-HT-induced colonic smooth muscle hypersensitivity assocd. with IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruU5R-UDwGebVg90H21EOLACvtfcHk0liKgr_tnKSe_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xis1ait7k%253D&md5=33fb1447a886d3313bc4aa0b2e8872e5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0704571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0704571%26sid%3Dliteratum%253Aachs%26aulast%3DBorman%26aufirst%3DR.%2BA.%26aulast%3DTilford%26aufirst%3DN.%2BS.%26aulast%3DHarmer%26aufirst%3DD.%2BW.%26aulast%3DDay%26aufirst%3DN.%26aulast%3DEllis%26aufirst%3DE.%2BS.%26aulast%3DSheldrick%26aufirst%3DR.%2BL.%26aulast%3DCarey%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BA.%26aulast%3DBaxter%26aufirst%3DG.%2BS.%26atitle%3D5-HT2B%2520receptors%2520play%2520a%2520key%2520role%2520in%2520mediating%2520the%2520excitatory%2520effects%2520of%25205-HT%2520in%2520human%2520colon%2520in%2520vitro%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2002%26volume%3D135%26spage%3D1144%26epage%3D1151%26doi%3D10.1038%2Fsj.bjp.0704571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Spiller, R.</span><span> </span><span class="NLM_article-title">Serotonergic agents and the irritable bowel syndrome: what goes wrong?</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">709</span><span class="NLM_x">–</span> <span class="NLM_lpage">714</span><span class="refDoi"> DOI: 10.1016/j.coph.2008.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.coph.2008.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=18672092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2qurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=709-714&author=R.+Spiller&title=Serotonergic+agents+and+the+irritable+bowel+syndrome%3A+what+goes+wrong%3F&doi=10.1016%2Fj.coph.2008.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Serotonergic agents and the irritable bowel syndrome: what goes wrong?</span></div><div class="casAuthors">Spiller, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">709-714</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  It is now nine years since the first large randomized controlled trials showed the benefit of a 5HT3 antagonist in irritable bowel syndrome (IBS) with diarrhea (IBS-D) and a 5HT4 partial agonist in IBS with constipation (IBS-C).  Although both of these drugs have now been withdrawn because of rare complications (ischemic colitis and thrombotic episodes resp.), their clin. effectiveness has stimulated substantial advances in our understanding of the physiol. role of serotonin and its disturbance in IBS. 5HT-contg. enteroendocrine cells are most numerous in the duodenum and colon and are responsive to a range of stimuli including tastants, mechanic force and short chain fatty acids.  Several studies have described an excess of 5HT in IBS-D and abnormally low 5HT availability in IBS-C.  Serotonin transporter (SERT) is only weakly expressed in the colon and whether this is further reduced in IBS is unclear with conflicting reports.  SERT promoter polymorphisms are inconsistently related to IBS subtypes but appear to partly predict response to both 5HT3 antagonists and 5HT4 agonists.  Stressors release serotonin and 5HT3 antagonists have been shown to inhibit the assocd. acceleration of colonic transit.  Newer 5HT3 antagonists and 5HT4 agonists are under development.  Their undoubted effectiveness in subgroups of IBS should stimulate further work to define biomarkers of both responsiveness and of risk of developing adverse events to improve the risk benefit balance for these potentially useful therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEsYzpKY4CtrVg90H21EOLACvtfcHk0liKgr_tnKSe_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2qurnE&md5=dbcbe1675441b53b62ad5ec40b3053bc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2008.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2008.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DSpiller%26aufirst%3DR.%26atitle%3DSerotonergic%2520agents%2520and%2520the%2520irritable%2520bowel%2520syndrome%253A%2520what%2520goes%2520wrong%253F%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2008%26volume%3D8%26spage%3D709%26epage%3D714%26doi%3D10.1016%2Fj.coph.2008.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Launay, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herve, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peoc’h, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tournois, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callebert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebigil, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Etienne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drouet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humbert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simonneau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maroteaux, L.</span><span> </span><span class="NLM_article-title">Function of the serotonin 5-hydroxytryptamine 2B receptor in pulmonary hypertension</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1135</span><span class="refDoi"> DOI: 10.1038/nm764</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnm764" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=12244304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGrt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2002&pages=1129-1135&author=J.+M.+Launayauthor=P.+Herveauthor=K.+Peoc%E2%80%99hauthor=C.+Tournoisauthor=J.+Callebertauthor=C.+G.+Nebigilauthor=N.+Etienneauthor=L.+Drouetauthor=M.+Humbertauthor=G.+Simonneauauthor=L.+Maroteaux&title=Function+of+the+serotonin+5-hydroxytryptamine+2B+receptor+in+pulmonary+hypertension&doi=10.1038%2Fnm764"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Function of the serotonin 5-hydroxytryptamine 2B receptor in pulmonary hypertension</span></div><div class="casAuthors">Launay, J.-M.; Herve, P.; Peoc'h, K.; Tournois, C.; Callebert, J.; Nebigil, C. G.; Etienne, N.; Drouet, L.; Humbert, M.; Simonneau, G.; Maroteaux, L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1129-1135</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Primary pulmonary hypertension is a progressive and often fatal disorder in humans that results from an increase in pulmonary blood pressure assocd. with abnormal vascular proliferation.  Dexfenfluramine increases the risk of pulmonary hypertension in humans, and its active metabolite is a selective serotonin 5-hydroxytryptamine 2B (5-HT2B) receptor agonist.  Thus, the authors investigated the contribution of the 5-HT2B receptor to the pathogenesis of pulmonary hypertension.  Using the chronic-hypoxic-mouse model of pulmonary hypertension, the authors found that the hypoxia-dependent increase in pulmonary blood pressure and lung remodeling are assocd. with an increase in vascular proliferation, elastase activity and transforming growth factor-β levels, and that these parameters are potentiated by dexfenfluramine treatment.  In contrast, hypoxic mice with genetically or pharmacol. inactive 5-HT2B receptors manifested no change in any of these parameters.  In both humans and mice, pulmonary hypertension is assocd. with a substantial increase in 5-HT2B receptor expression in pulmonary arteries.  These data show that activation of 5-HT2B receptors is a limiting step in the development of pulmonary hypertension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP9zEi5XRykbVg90H21EOLACvtfcHk0lgYAsZOeFxlvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGrt7Y%253D&md5=01bde279d3503f827598478602ccec25</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnm764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm764%26sid%3Dliteratum%253Aachs%26aulast%3DLaunay%26aufirst%3DJ.%2BM.%26aulast%3DHerve%26aufirst%3DP.%26aulast%3DPeoc%25E2%2580%2599h%26aufirst%3DK.%26aulast%3DTournois%26aufirst%3DC.%26aulast%3DCallebert%26aufirst%3DJ.%26aulast%3DNebigil%26aufirst%3DC.%2BG.%26aulast%3DEtienne%26aufirst%3DN.%26aulast%3DDrouet%26aufirst%3DL.%26aulast%3DHumbert%26aufirst%3DM.%26aulast%3DSimonneau%26aufirst%3DG.%26aulast%3DMaroteaux%26aufirst%3DL.%26atitle%3DFunction%2520of%2520the%2520serotonin%25205-hydroxytryptamine%25202B%2520receptor%2520in%2520pulmonary%2520hypertension%26jtitle%3DNat.%2520Med.%26date%3D2002%26volume%3D8%26spage%3D1129%26epage%3D1135%26doi%3D10.1038%2Fnm764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Porvasnik, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embury, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gannon, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shacham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Mousily, F.</span><span> </span><span class="NLM_article-title">PRX-08066, a novel 5-hydroxytryptamine receptor 2B antagonist, reduces monocrotaline-induced pulmonary arterial hypertension and right ventricular hypertrophy in rats</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1124/jpet.109.165001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1124%2Fjpet.109.165001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=20430844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1KisLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2010&pages=364-372&author=S.+L.+Porvasnikauthor=S.+Germainauthor=J.+Emburyauthor=K.+S.+Gannonauthor=V.+Jacquesauthor=J.+Murrayauthor=B.+J.+Byrneauthor=S.+Shachamauthor=F.+Al-Mousily&title=PRX-08066%2C+a+novel+5-hydroxytryptamine+receptor+2B+antagonist%2C+reduces+monocrotaline-induced+pulmonary+arterial+hypertension+and+right+ventricular+hypertrophy+in+rats&doi=10.1124%2Fjpet.109.165001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">PRX-08066, a novel 5-hydroxytryptamine receptor 2B antagonist, reduces monocrotaline-induced pulmonary arterial hypertension and right ventricular hypertrophy in rats</span></div><div class="casAuthors">Porvasnik, Stacy L.; Germain, Sean; Embury, Jennifer; Gannon, Kimberley S.; Jacques, Vincent; Murray, Justin; Byrne, Barry J.; Shacham, Sharon; Al-Mousily, Faris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">364-372</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Pulmonary arterial hypertension (PAH) is a life-threatening disease that results in right ventricular failure.  5-((4-(6-Chlorothieno[2,3-d]pyrimidin-4-ylamino)piperidin-1-yl)methyl)-2-fluorobenzonitrile monofumarate (PRX-08066) is a selective 5-hydroxytryptamine receptor 2B (5-HT2BR) antagonist that causes selective vasodilation of pulmonary arteries.  In the current study, the effects of PRX-08066 were assessed by using the monocrotaline (MCT)-induced PAH rat model.  Male rats received 40 mg/kg MCT or phosphate-buffered saline and were treated orally twice a day with vehicle or 50 or 100 mg/kg PRX-08066 for 5 wk.  Pulmonary and cardiac functions were evaluated by hemodynamics, heart wt., magnetic resonance imaging (MRI), pulmonary artery (PA) morphol., and histol.  Cardiac MRI demonstrated that PRX-08066 (100 mg/kg) significantly (P < 0.05) improved right ventricular ejection fraction.  PRX-08066 significantly reduced peak PA pressure at 50 and 100 mg/kg (P < 0.05 and < 0.01, resp.) compared with MCT control animals.  PRX-08066 therapy also significantly reduced right ventricle (RV)/body wt. and RV/left ventricle + septum (P < 0.01 and < 0.001, resp.) compared with MCT-treated animals.  Morphometric assessment of pulmonary arterioles revealed a significant redn. in medial wall thickening and lumen occlusion assocd. with both doses of PRX-08066 (P < 0.01).  The 5-HT2BR antagonist PRX-08066 significantly attenuated the elevation in PA pressure and RV hypertrophy and maintained cardiac function.  Pulmonary vascular remodeling was also diminished compared with MCT control rats.  PRX-08066 prevents the severity of PAH in the MCT rat model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_CaIqQmpIS7Vg90H21EOLACvtfcHk0lgYAsZOeFxlvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1KisLbJ&md5=c00235e9c830eb3a3937fa3696dac88b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fjpet.109.165001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.109.165001%26sid%3Dliteratum%253Aachs%26aulast%3DPorvasnik%26aufirst%3DS.%2BL.%26aulast%3DGermain%26aufirst%3DS.%26aulast%3DEmbury%26aufirst%3DJ.%26aulast%3DGannon%26aufirst%3DK.%2BS.%26aulast%3DJacques%26aufirst%3DV.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DByrne%26aufirst%3DB.%2BJ.%26aulast%3DShacham%26aufirst%3DS.%26aulast%3DAl-Mousily%26aufirst%3DF.%26atitle%3DPRX-08066%252C%2520a%2520novel%25205-hydroxytryptamine%2520receptor%25202B%2520antagonist%252C%2520reduces%2520monocrotaline-induced%2520pulmonary%2520arterial%2520hypertension%2520and%2520right%2520ventricular%2520hypertrophy%2520in%2520rats%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2010%26volume%3D334%26spage%3D364%26epage%3D372%26doi%3D10.1124%2Fjpet.109.165001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Soares, R. L.</span><span> </span><span class="NLM_article-title">Irritable bowel syndrome: A clinical review</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">12144</span><span class="NLM_x">–</span> <span class="NLM_lpage">12160</span><span class="refDoi"> DOI: 10.3748/wjg.v20.i34.12144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.3748%2Fwjg.v20.i34.12144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=25232249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ktleltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=12144-12160&author=R.+L.+Soares&title=Irritable+bowel+syndrome%3A+A+clinical+review&doi=10.3748%2Fwjg.v20.i34.12144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Irritable bowel syndrome: a clinical review</span></div><div class="casAuthors">Soares Rosa L S</div><div class="citationInfo"><span class="NLM_cas:title">World journal of gastroenterology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12144-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Irritable bowel syndrome (IBS) remains a clinical challenge in the 21(st) century.  It's the most commonly diagnosed gastrointestinal condition and also the most common reason for referral to gastroenterology clinics.  Its can affect up to one in five people at some point in their lives, and has a significantly impact of life quality and health care utilization.  The prevalence varies according to country and criteria used to define IBS.  Various mechanisms and theories have been proposed about its etiology, but the biopsychosocial model is the most currently accepted for IBS.  The complex of symptoms would be the result of the interaction between psychological, behavioral, psychosocial and environmental factors.  The diagnosis of IBS is not confirmed by a specific test or structural abnormality.  It is made using criteria based on clinical symptoms such as Rome criteria, unless the symptoms are thought to be atypical.  Today the Rome Criteria III is the current gold-standard for the diagnoses of IBS.  Secure positive evidence of IBS by means of specific disease marker is currently not possible and cannot be currently recommended for routine diagnosis.  There is still no clinical evidence to recommend the use of biomarkers in blood to diagnose IBS.  However, a number of different changes in IBS patients were demonstrated in recent years, some of which can be used in the future as a diagnostic support.  IBS has no definitive treatment but could be controlled by non-pharmacologic management eliminating of some exacerbating factors such certain drugs, stressor conditions and changes in dietary habits.The traditional pharmacologic management of IBS has been symptom based and several drugs have been used.  However, the cornerstone of its therapy is a solid patient physician relationship.  This review will provide a summary of pathophysiology, diagnostic criteria and current and emerging therapies for IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQiMlaT5g6za2Zyxfu64m4ffW6udTcc2eZg5H7pORM2QLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ktleltQ%253D%253D&md5=2cc95ea97567a60537ad0b52d04180fb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v20.i34.12144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v20.i34.12144%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DR.%2BL.%26atitle%3DIrritable%2520bowel%2520syndrome%253A%2520A%2520clinical%2520review%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2014%26volume%3D20%26spage%3D12144%26epage%3D12160%26doi%3D10.3748%2Fwjg.v20.i34.12144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Occhipinti, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. W.</span><span> </span><span class="NLM_article-title">Irritable bowel syndrome: a review and update</span> <span class="citation_source-journal">Clin. Colon Rectal Surg.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1055/s-0032-1301759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1055%2Fs-0032-1301759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23449495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A280%3ADC%252BC3svit12hsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=46-52&author=K.+Occhipintiauthor=J.+W.+Smith&title=Irritable+bowel+syndrome%3A+a+review+and+update&doi=10.1055%2Fs-0032-1301759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Irritable bowel syndrome: a review and update</span></div><div class="casAuthors">Occhipinti Kaitlin; Smith James W</div><div class="citationInfo"><span class="NLM_cas:title">Clinics in colon and rectal surgery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-52</span>
        ISSN:<span class="NLM_cas:issn">1531-0043</span>.
    </div><div class="casAbstract">The understanding of irritable bowel syndrome (IBS) has undergone a rapid evolution with scientific advancement.  IBS is a common functional bowel disorder that generates a significant health care burden and is the most commonly diagnosed gastrointestinal condition.  There are well-established diagnostic criteria and algorithms for the initial evaluation of patients presenting with the symptoms of IBS.  The symptoms can be targeted for therapy with a variety of pharmaceutical and nonpharmaceutical agents.  Therapy should be individualized for the patient, and the cornerstone for any effective treatment strategy should be the solid patient-physician relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTd2CLvsyvO3_6xYuD7tu2XfW6udTcc2eZg5H7pORM2QLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svit12hsg%253D%253D&md5=edf8a4c80999a907b7922f4b3638cb44</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1055%2Fs-0032-1301759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0032-1301759%26sid%3Dliteratum%253Aachs%26aulast%3DOcchipinti%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%2BW.%26atitle%3DIrritable%2520bowel%2520syndrome%253A%2520a%2520review%2520and%2520update%26jtitle%3DClin.%2520Colon%2520Rectal%2520Surg.%26date%3D2012%26volume%3D25%26spage%3D46%26epage%3D52%26doi%3D10.1055%2Fs-0032-1301759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Ashburn, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, M. S.</span><span> </span><span class="NLM_article-title">The IBS market</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">100</span><span class="refDoi"> DOI: 10.1038/nrd1961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnrd1961" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=99-100&author=T.+T.+Ashburnauthor=M.+S.+Gupta&title=The+IBS+market&doi=10.1038%2Fnrd1961"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd1961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1961%26sid%3Dliteratum%253Aachs%26aulast%3DAshburn%26aufirst%3DT.%2BT.%26aulast%3DGupta%26aufirst%3DM.%2BS.%26atitle%3DThe%2520IBS%2520market%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D99%26epage%3D100%26doi%3D10.1038%2Fnrd1961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Mullard, A.</span><span> </span><span class="NLM_article-title">FDA approves two IBS drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">449</span><span class="refDoi"> DOI: 10.1038/nrd4688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnrd4688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=449&author=A.+Mullard&title=FDA+approves+two+IBS+drugs&doi=10.1038%2Fnrd4688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd4688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4688%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFDA%2520approves%2520two%2520IBS%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D449%26doi%3D10.1038%2Fnrd4688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Wouters, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tharayil, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanich, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strege, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardsley, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ordog, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbons, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrugia, G.</span><span> </span><span class="NLM_article-title">Protein kinase C<sub>γ</sub> mediates regulation of proliferation by the serotonin 5-hydroxytryptamine receptor 2B</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">21177</span><span class="NLM_x">–</span> <span class="NLM_lpage">21184</span><span class="refDoi"> DOI: 10.1074/jbc.M109.015859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1074%2Fjbc.M109.015859" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=21177-21184&author=M.+M.+Woutersauthor=J.+L.+Roederauthor=V.+S.+Tharayilauthor=J.+E.+Stanichauthor=P.+R.+Stregeauthor=S.+Leiauthor=M.+R.+Bardsleyauthor=T.+Ordogauthor=S.+J.+Gibbonsauthor=G.+Farrugia&title=Protein+kinase+C%CE%B3+mediates+regulation+of+proliferation+by+the+serotonin+5-hydroxytryptamine+receptor+2B&doi=10.1074%2Fjbc.M109.015859"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.015859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.015859%26sid%3Dliteratum%253Aachs%26aulast%3DWouters%26aufirst%3DM.%2BM.%26aulast%3DRoeder%26aufirst%3DJ.%2BL.%26aulast%3DTharayil%26aufirst%3DV.%2BS.%26aulast%3DStanich%26aufirst%3DJ.%2BE.%26aulast%3DStrege%26aufirst%3DP.%2BR.%26aulast%3DLei%26aufirst%3DS.%26aulast%3DBardsley%26aufirst%3DM.%2BR.%26aulast%3DOrdog%26aufirst%3DT.%26aulast%3DGibbons%26aufirst%3DS.%2BJ.%26aulast%3DFarrugia%26aufirst%3DG.%26atitle%3DProtein%2520kinase%2520C%25CE%25B3%2520mediates%2520regulation%2520of%2520proliferation%2520by%2520the%2520serotonin%25205-hydroxytryptamine%2520receptor%25202B%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D21177%26epage%3D21184%26doi%3D10.1074%2Fjbc.M109.015859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span> </span><span class="NLM_article-title">Effect of 5-HT<sub>2B</sub> receptor on the pathogenetic mechanism of irritable bowel syndrome subgroups in rat models</span> <span class="citation_source-journal">Acta Univ. Med. Nanjing</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=609-615&author=Y.+Chenauthor=Z.+Liauthor=Y.+Yangauthor=L.+Linauthor=H.+Zhang&title=Effect+of+5-HT2B+receptor+on+the+pathogenetic+mechanism+of+irritable+bowel+syndrome+subgroups+in+rat+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DEffect%2520of%25205-HT2B%2520receptor%2520on%2520the%2520pathogenetic%2520mechanism%2520of%2520irritable%2520bowel%2520syndrome%2520subgroups%2520in%2520rat%2520models%26jtitle%3DActa%2520Univ.%2520Med.%2520Nanjing%26date%3D2012%26volume%3D32%26spage%3D609%26epage%3D615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bassil, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolton, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summerfield, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winchester, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanger, G. J.</span><span> </span><span class="NLM_article-title">Inhibition of colonic motility and defecation by RS-127445 suggests an involvement of the 5-HT<sub>2B</sub> receptor in rodent large bowel physiology</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">158</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">258</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00155.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1111%2Fj.1476-5381.2009.00155.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=19371340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2009&pages=252-258&author=A.+Bassilauthor=C.+Taylorauthor=V.+Boltonauthor=K.+Grayauthor=J.+Brownauthor=L.+Cutlerauthor=S.+Summerfieldauthor=G.+Brutonauthor=W.+Winchesterauthor=K.+Leeauthor=G.+J.+Sanger&title=Inhibition+of+colonic+motility+and+defecation+by+RS-127445+suggests+an+involvement+of+the+5-HT2B+receptor+in+rodent+large+bowel+physiology&doi=10.1111%2Fj.1476-5381.2009.00155.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of colonic motility and defecation by RS-127445 suggests an involvement of the 5-HT2B receptor in rodent large bowel physiology</span></div><div class="casAuthors">Bassil, A. K.; Taylor, C. M.; Bolton, V. J. N.; Gray, K. M.; Brown, J. D.; Cutler, L.; Summerfield, S. G.; Bruton, G.; Winchester, W. J.; Lee, K.; Sanger, G. J.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">252-258</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">5-HT2B receptors are localized within the myenteric nervous system, but their functions on motor/sensory neurons are unclear.  To explore the role of these receptors, we further characterized the 5-HT2B receptor antagonist RS-127445 and studied its effects on peristalsis and defecation.  Although reported as a selective 5-HT2B receptor antagonist, any interactions of RS-127445 with 5-HT4 receptors are unknown; this was examd. using the recombinant receptor and Biomol. Interaction Detection technol.  Mouse isolated colon was mounted in tissue baths for isometric recording of neuronal contractions evoked by elec. field stimulation (EFS), or under an intraluminal pressure gradient to induce peristalsis; the effects of RS-127445 on EFS-induced and on peristaltic contractions were measured.  Faecal output of rats in grid-bottom cages was measured over 3 h following i.p. RS-127445 and sep., validation of the EDs was achieved by detg. the free, unbound fraction of RS-127445 in blood and brain.  RS-127445 (up to 1 μmol/L-1) did not interact with the 5-HT4 receptor.  RS-127445 (0.001-1 μmol/L-1) did not affect EFS-induced contractions of the colon, although at 10 μmol/L-1 the contractions were reduced (to 36 ± 8% of control, n = 4).  RS-127445 (0.1-10 μmol/L-1) concn.-dependently reduced peristaltic frequency (n = 4).  RS-127445 (1-30 mg/kg-1), dose-dependently reduced faecal output, reaching significance at 10 and 30 mg/kg-1 (n = 6-11).  In blood and brain, >98% of RS-127445 was protein-bound.  High-protein binding of RS-127445 indicates that relatively high doses are required for efficacy.  The results suggest that 5-HT2B receptors tonically regulate colonic motility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5jssIHDufubVg90H21EOLACvtfcHk0liwelyS_6kT5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKlsr%252FK&md5=411ce4124f41fca8bdb468defcb19dfc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00155.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00155.x%26sid%3Dliteratum%253Aachs%26aulast%3DBassil%26aufirst%3DA.%26aulast%3DTaylor%26aufirst%3DC.%26aulast%3DBolton%26aufirst%3DV.%26aulast%3DGray%26aufirst%3DK.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DSummerfield%26aufirst%3DS.%26aulast%3DBruton%26aufirst%3DG.%26aulast%3DWinchester%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSanger%26aufirst%3DG.%2BJ.%26atitle%3DInhibition%2520of%2520colonic%2520motility%2520and%2520defecation%2520by%2520RS-127445%2520suggests%2520an%2520involvement%2520of%2520the%25205-HT2B%2520receptor%2520in%2520rodent%2520large%2520bowel%2520physiology%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2009%26volume%3D158%26spage%3D252%26epage%3D258%26doi%3D10.1111%2Fj.1476-5381.2009.00155.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">O’mahony, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulmer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coelho, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzgerald, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bongiovanni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winchester, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinan, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cryan, J.</span><span> </span><span class="NLM_article-title">5-HT<sub>2B</sub> receptors modulate visceral hypersensitivity in a stress-sensitive animal model of brain-gut axis dysfunction</span> <span class="citation_source-journal">Neurogastroenterol. Motil.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">573</span><span class="NLM_x">–</span> <span class="NLM_lpage">e124</span><span class="refDoi"> DOI: 10.1111/j.1365-2982.2009.01432.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1111%2Fj.1365-2982.2009.01432.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=20003079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslCitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=573-e124&author=S.+O%E2%80%99mahonyauthor=D.+Bulmerauthor=A.+Coelhoauthor=P.+Fitzgeraldauthor=C.+Bongiovanniauthor=K.+Leeauthor=W.+Winchesterauthor=T.+Dinanauthor=J.+Cryan&title=5-HT2B+receptors+modulate+visceral+hypersensitivity+in+a+stress-sensitive+animal+model+of+brain-gut+axis+dysfunction&doi=10.1111%2Fj.1365-2982.2009.01432.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">5-HT2B receptors modulate visceral hypersensitivity in a stress-sensitive animal model of brain-gut axis dysfunction</span></div><div class="casAuthors">O'Mahony, S. M.; Bulmer, D. C.; Coelho, A.-M.; Fitzgerald, P.; Bongiovanni, C.; Lee, K.; Winchester, W.; Dinan, T. G.; Cryan, J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Neurogastroenterology & Motility</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">573-578</span>CODEN:
                <span class="NLM_cas:coden">NMOTEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-1925</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Irritable bowel syndrome (IBS) is assocd. with an enhanced perception to visceral stimuli and exaggerated stress response.  The serotonergic neurotransmitter system has been strongly implicated as a key player in the manifestation of IBS symptomatol. including visceral hypersensitivity.  However the role of 5-HT2B receptors in visceral pain, although speculated, is currently unclear.  Thus we assessed the impact of a selective 5-HT2B receptor antagonist, RS-127445, on visceral hypersensitivity in a model of brain gut axis dysfunction the Wistar Kyoto (WKY) rat.  Colorectal distension (CRD) was used to assess the visceral sensitivity of the WKY rat compared to normosensitive Sprague Dawley (SD) rats.  Once we verified the visceral sensitivity of the WKY rat we assessed the efficacy of RS-127445 in pain signalling from the colorectum.  We administered the compd. peripherally (i.p.) and centrally (i.c.v.) in order to ascertain the site of action of RS 127445.  Behavioral responses to colorectal distention were then monitored.  The WKY rats were more viscerally hypersensitive than the SD as previously shown.  RS-127445 (5 mg/kg-1, i.p.) significantly reversed visceral hypersensitivity in WKY animals.  Moreover, when administered intracerebroventricularly RS-127445 (100 nM) also decreased the no. of pain behaviors during noxious CRD in the WKY animals.  Taken together, blockade of 5-HT2B receptors offers an exciting novel therapeutic target for pain relief in stress-related gastrointestinal disorders such as IBS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVsA5ZZFutFrVg90H21EOLACvtfcHk0liwelyS_6kT5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslCitLw%253D&md5=404d887aa980c14c5742be450de7dcba</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2982.2009.01432.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2982.2009.01432.x%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599mahony%26aufirst%3DS.%26aulast%3DBulmer%26aufirst%3DD.%26aulast%3DCoelho%26aufirst%3DA.%26aulast%3DFitzgerald%26aufirst%3DP.%26aulast%3DBongiovanni%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWinchester%26aufirst%3DW.%26aulast%3DDinan%26aufirst%3DT.%26aulast%3DCryan%26aufirst%3DJ.%26atitle%3D5-HT2B%2520receptors%2520modulate%2520visceral%2520hypersensitivity%2520in%2520a%2520stress-sensitive%2520animal%2520model%2520of%2520brain-gut%2520axis%2520dysfunction%26jtitle%3DNeurogastroenterol.%2520Motil.%26date%3D2010%26volume%3D22%26spage%3D573%26epage%3De124%26doi%3D10.1111%2Fj.1365-2982.2009.01432.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Ohashi-doi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himaki, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gale, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furness, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurebayashi, Y.</span><span> </span><span class="NLM_article-title">A selective, high affinity 5-HT<sub>2B</sub> receptor antagonist inhibits visceral hypersensitivity in rats</span> <span class="citation_source-journal">Neurogastroenterol. Motil.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">e69</span><span class="NLM_x">–</span> <span class="NLM_lpage">e76</span><span class="refDoi"> DOI: 10.1111/j.1365-2982.2009.01395.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1111%2Fj.1365-2982.2009.01395.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2010&pages=e69-e76&author=K.+Ohashi-doiauthor=D.+Himakiauthor=K.+Nagaoauthor=M.+Kawaiauthor=J.+Galeauthor=J.+Furnessauthor=Y.+Kurebayashi&title=A+selective%2C+high+affinity+5-HT2B+receptor+antagonist+inhibits+visceral+hypersensitivity+in+rats&doi=10.1111%2Fj.1365-2982.2009.01395.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2982.2009.01395.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2982.2009.01395.x%26sid%3Dliteratum%253Aachs%26aulast%3DOhashi-doi%26aufirst%3DK.%26aulast%3DHimaki%26aufirst%3DD.%26aulast%3DNagao%26aufirst%3DK.%26aulast%3DKawai%26aufirst%3DM.%26aulast%3DGale%26aufirst%3DJ.%26aulast%3DFurness%26aufirst%3DJ.%26aulast%3DKurebayashi%26aufirst%3DY.%26atitle%3DA%2520selective%252C%2520high%2520affinity%25205-HT2B%2520receptor%2520antagonist%2520inhibits%2520visceral%2520hypersensitivity%2520in%2520rats%26jtitle%3DNeurogastroenterol.%2520Motil.%26date%3D2010%26volume%3D22%26spage%3De69%26epage%3De76%26doi%3D10.1111%2Fj.1365-2982.2009.01395.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Takahashi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inagaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taniguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakaguchi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, K.</span><span> </span><span class="NLM_article-title">The novel 5-HT<sub>2B</sub> receptor antagonist, RQ-00310941, attenuates visceral hypersensitivity and abnormal defecation in rat models</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x">, </span> <span class="NLM_fpage">S</span><span class="NLM_x">–</span> <span class="NLM_lpage">607</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2011&pages=S-607&author=N.+Takahashiauthor=K.+Inagakiauthor=K.+Taniguchiauthor=Y.+Sakaguchiauthor=K.+Kawamura&title=The+novel+5-HT2B+receptor+antagonist%2C+RQ-00310941%2C+attenuates+visceral+hypersensitivity+and+abnormal+defecation+in+rat+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DN.%26aulast%3DInagaki%26aufirst%3DK.%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DSakaguchi%26aufirst%3DY.%26aulast%3DKawamura%26aufirst%3DK.%26atitle%3DThe%2520novel%25205-HT2B%2520receptor%2520antagonist%252C%2520RQ-00310941%252C%2520attenuates%2520visceral%2520hypersensitivity%2520and%2520abnormal%2520defecation%2520in%2520rat%2520models%26jtitle%3DGastroenterology%26date%3D2011%26volume%3D140%26spage%3DS%26epage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Poissonnet, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parmentier, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutin, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, S.</span><span> </span><span class="NLM_article-title">The emergence of selective 5-HT<sub>2B</sub> antagonists structures, activities and potential therapeutic applications</span> <span class="citation_source-journal">Mini-Rev. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">330</span><span class="refDoi"> DOI: 10.2174/1389557043487312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.2174%2F1389557043487312" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=325-330&author=G.+Poissonnetauthor=J.+G.+Parmentierauthor=J.+A.+Boutinauthor=S.+Goldstein&title=The+emergence+of+selective+5-HT2B+antagonists+structures%2C+activities+and+potential+therapeutic+applications&doi=10.2174%2F1389557043487312"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.2174%2F1389557043487312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557043487312%26sid%3Dliteratum%253Aachs%26aulast%3DPoissonnet%26aufirst%3DG.%26aulast%3DParmentier%26aufirst%3DJ.%2BG.%26aulast%3DBoutin%26aufirst%3DJ.%2BA.%26aulast%3DGoldstein%26aufirst%3DS.%26atitle%3DThe%2520emergence%2520of%2520selective%25205-HT2B%2520antagonists%2520structures%252C%2520activities%2520and%2520potential%2520therapeutic%2520applications%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2004%26volume%3D4%26spage%3D325%26epage%3D330%26doi%3D10.2174%2F1389557043487312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Life beyond kinases: Structure-based discovery of sorafenib as nanomolar antagonist of 5-HT receptors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5749</span><span class="NLM_x">–</span> <span class="NLM_lpage">5759</span><span class="refDoi"> DOI: 10.1021/jm300338m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300338m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=5749-5759&author=X.+Linauthor=X.-P.+Huangauthor=G.+Chenauthor=R.+Whaleyauthor=S.+Pengauthor=Y.+Wangauthor=G.+Zhangauthor=S.+X.+Wangauthor=S.+Wangauthor=B.+L.+Rothauthor=N.+Huang&title=Life+beyond+kinases%3A+Structure-based+discovery+of+sorafenib+as+nanomolar+antagonist+of+5-HT+receptors&doi=10.1021%2Fjm300338m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm300338m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300338m%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DS.%2BX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DLife%2520beyond%2520kinases%253A%2520Structure-based%2520discovery%2520of%2520sorafenib%2520as%2520nanomolar%2520antagonist%2520of%25205-HT%2520receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D5749%26epage%3D5759%26doi%3D10.1021%2Fjm300338m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Cusack, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richelson, E.</span><span> </span><span class="NLM_article-title">Binding of antidepressants to human brain receptors: Focus on newer generation compounds</span> <span class="citation_source-journal">Psychopharmacology (Berl)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">559</span><span class="NLM_x">–</span> <span class="NLM_lpage">565</span><span class="refDoi"> DOI: 10.1007/BF02244985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1007%2FBF02244985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=7855217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK2cXksV2qtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1994&pages=559-565&author=B.+Cusackauthor=A.+Nelsonauthor=E.+Richelson&title=Binding+of+antidepressants+to+human+brain+receptors%3A+Focus+on+newer+generation+compounds&doi=10.1007%2FBF02244985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of antidepressants to human brain receptors: focus on newer generation compounds</span></div><div class="casAuthors">Cusack, Bernadette; Nelson, Albert; Richelson, Elliott</div><div class="citationInfo"><span class="NLM_cas:title">Psychopharmacology (Berlin, Germany)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">559-65</span>CODEN:
                <span class="NLM_cas:coden">PSCHDL</span>;
        ISSN:<span class="NLM_cas:issn">0033-3158</span>.
    </div><div class="casAbstract">Using radioligand binding assays and post-mortem normal human brain tissue, the authors obtained equil. dissocn. consts. (Kds) for 17 antidepressants and two of their metabolites at histamine H1, muscarinic, α1-adrenergic, α2-adrenergic, dopamine D2, serotonin 5-HT1A, and serotonin 5-HT2 receptors.  Several newer antidepressants were compared with older drugs.  In addn., the authors studied some antimuscarinic, antiparkinson, antihistamine, and neuroleptic compds. at some of these receptors.  For the antidepressants, classical tricyclic antidepressants were the most potent drugs at five of the seven receptors (all but α2-adrenergic and 5-HT1A receptors).  The chlorophenylpiperazine deriv. antidepressants (etoperidone, nefazodone, trazodone) were the most potent antidepressants at α2-adrenergic and 5-HT1A receptors.  Of ten antihistamines tested, none was more potent than doxepin at histamine H1 receptors.  At muscarinic receptors antidepressants and antihistamines had a range of potencies, which were mostly weaker than those for antimuscarinics.  From the in vitro data, the authors expect adinazolam, bupropion, fluoxetine, sertraline, tomoxetine, and venlafaxine not to block any of these five receptors in vivo.  An antidepressant's potency for blocking a specific receptor is predictive of certain side effects and drug-drug interactions.  These studies can provide guidelines for the clinician in the choice of antidepressant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4C7IsG-1gh7Vg90H21EOLACvtfcHk0lgsFcLoRUr-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXksV2qtbw%253D&md5=7be0af5b1e8e5abbca3412d51606da76</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2FBF02244985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02244985%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DB.%26aulast%3DNelson%26aufirst%3DA.%26aulast%3DRichelson%26aufirst%3DE.%26atitle%3DBinding%2520of%2520antidepressants%2520to%2520human%2520brain%2520receptors%253A%2520Focus%2520on%2520newer%2520generation%2520compounds%26jtitle%3DPsychopharmacology%2520%2528Berl%2529%26date%3D1994%26volume%3D114%26spage%3D559%26epage%3D565%26doi%3D10.1007%2FBF02244985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiroishi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weyand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwata, S.</span><span> </span><span class="NLM_article-title">Structure of the human histamine H<sub>1</sub> receptor complex with doxepin</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">475</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1038/nature10236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnature10236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=21697825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVekt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=475&publication_year=2011&pages=65-70&author=T.+Shimamuraauthor=M.+Shiroishiauthor=S.+Weyandauthor=H.+Tsujimotoauthor=G.+Winterauthor=V.+Katritchauthor=R.+Abagyanauthor=V.+Cherezovauthor=W.+Liuauthor=G.+W.+Hanauthor=T.+Kobayashiauthor=R.+C.+Stevensauthor=S.+Iwata&title=Structure+of+the+human+histamine+H1+receptor+complex+with+doxepin&doi=10.1038%2Fnature10236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human histamine H1 receptor complex with doxepin</span></div><div class="casAuthors">Shimamura, Tatsuro; Shiroishi, Mitsunori; Weyand, Simone; Tsujimoto, Hirokazu; Winter, Graeme; Katritch, Vsevolod; Abagyan, Ruben; Cherezov, Vadim; Liu, Wei; Han, Gye Won; Kobayashi, Takuya; Stevens, Raymond C.; Iwata, So</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">475</span>
        (<span class="NLM_cas:issue">7354</span>),
    <span class="NLM_cas:pages">65-70</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The biogenic amine histamine is an important pharmacol. mediator involved in pathophysiol. processes such as allergies and inflammations.  Histamine H1 receptor (H1R) antagonists are very effective drugs alleviating the symptoms of allergic reactions.  Here, we show the crystal structure of the H1R complex with doxepin, a first-generation H1R antagonist.  Doxepin sits deep in the ligand-binding pocket and directly interacts with Trp 428, a highly conserved key residue in G-protein-coupled-receptor activation.  This well-conserved pocket with mostly hydrophobic nature contributes to the low selectivity of the first-generation compds.  The pocket is assocd. with an anion-binding region occupied by a phosphate ion.  Docking of various second-generation H1R antagonists reveals that the unique carboxyl group present in this class of compds. interacts with Lys 191 and/or Lys 179ECL2, both of which form part of the anion-binding region.  This region is not conserved in other aminergic receptors, demonstrating how minor differences in receptors lead to pronounced selectivity differences with small mols.  Our study sheds light on the mol. basis of H1R antagonist specificity against H1R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN1obci6QtYbVg90H21EOLACvtfcHk0lgsFcLoRUr-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVekt7k%253D&md5=7116228016caa1f3afbfed11468c0154</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature10236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10236%26sid%3Dliteratum%253Aachs%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DWeyand%26aufirst%3DS.%26aulast%3DTsujimoto%26aufirst%3DH.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DIwata%26aufirst%3DS.%26atitle%3DStructure%2520of%2520the%2520human%2520histamine%2520H1%2520receptor%2520complex%2520with%2520doxepin%26jtitle%3DNature%26date%3D2011%26volume%3D475%26spage%3D65%26epage%3D70%26doi%3D10.1038%2Fnature10236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Warne, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tate, C. G.</span><span> </span><span class="NLM_article-title">Crystal structures of a stabilized beta<sub>1</sub>-adrenoceptor bound to the biased agonists bucindolol and carvedilol</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span><span class="refDoi"> DOI: 10.1016/j.str.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.str.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=22579251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=841-849&author=T.+Warneauthor=P.+C.+Edwardsauthor=A.+G.+Leslieauthor=C.+G.+Tate&title=Crystal+structures+of+a+stabilized+beta1-adrenoceptor+bound+to+the+biased+agonists+bucindolol+and+carvedilol&doi=10.1016%2Fj.str.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of a Stabilized β1-Adrenoceptor Bound to the Biased Agonists Bucindolol and Carvedilol</span></div><div class="casAuthors">Warne, Tony; Edwards, Patricia C.; Leslie, Andrew G. W.; Tate, Christopher G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">841-849</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The β1-adrenoceptor (β1AR) is the site of action of beta blockers used in the treatment of cardiac-related illnesses.  Two beta blockers, carvedilol and bucindolol, show distinctive activities compared to other beta blockers and have been proposed as treatments tailored to the Arg/Gly3898.56 polymorphism of the human β1AR.  Both carvedilol and bucindolol are classified as biased agonists, because they stimulate G protein-independent signaling, while acting as either inverse or partial agonists of the G protein pathway.  We have detd. the crystal structures of a thermostabilized avian β1AR mutant bound to bucindolol and to carvedilol at 3.2 and 2.3 Å resoln., resp.  In comparison to other beta blockers, bucindolol and carvedilol interact with addnl. residues, in extracellular loop 2 and transmembrane helix 7, which may promote G protein-independent signaling.  The structures also suggest that there may be a structural explanation for the pharmacol. differences arising from the Arg/Gly3898.56 polymorphism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9aCm9IL2y37Vg90H21EOLACvtfcHk0lgXgT7KK-ZIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsLw%253D&md5=86542cd6a868477a9aabcbe25b1db8bb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DWarne%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DP.%2BC.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DTate%26aufirst%3DC.%2BG.%26atitle%3DCrystal%2520structures%2520of%2520a%2520stabilized%2520beta1-adrenoceptor%2520bound%2520to%2520the%2520biased%2520agonists%2520bucindolol%2520and%2520carvedilol%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D841%26epage%3D849%26doi%3D10.1016%2Fj.str.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Incorporating replacement free energy of binding-site waters in molecular docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">82</span><span class="NLM_x">, </span> <span class="NLM_fpage">1765</span><span class="NLM_x">–</span> <span class="NLM_lpage">1776</span><span class="refDoi"> DOI: 10.1002/prot.24530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.24530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=24549784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislCgtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2014&pages=1765-1776&author=H.+Sunauthor=L.+Zhaoauthor=S.+Pengauthor=N.+Huang&title=Incorporating+replacement+free+energy+of+binding-site+waters+in+molecular+docking&doi=10.1002%2Fprot.24530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporating replacement free energy of binding-site waters in molecular docking</span></div><div class="casAuthors">Sun, Hanzi; Zhao, Lifeng; Peng, Shiming; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1765-1776</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Binding-site water mols. play a crucial role in protein-ligand recognition, either being displaced upon ligand binding or forming water bridges to stabilize the complex.  However, rigorously treating explicit binding-site waters is challenging in mol. docking, which requires to fully sample ensembles of waters and to consider the free energy cost of replacing waters.  Here, the authors describe a method to incorporate structural and energetic properties of binding-site waters into mol. docking.  The authors first developed a solvent property anal. (SPA) program to compute the replacement free energies of binding-site water mols. by post-processing mol. dynamics trajectories obtained from ligand-free protein structure simulation in explicit water.  Next, the authors implemented a distance-dependent scoring term into DOCK scoring function to take account of the water replacement free energy cost upon ligand binding.  The authors assessed this approach in protein targets contg. important binding-site waters, and the authors' approach is reliable in reproducing the crystal binding geometries of protein-ligand-water complexes, as well as moderately improving the ligand docking enrichment performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwQA_EiIKYHbVg90H21EOLACvtfcHk0lgXgT7KK-ZIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislCgtL0%253D&md5=a8703dd3206123760dc6ffcd2d30fd41</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fprot.24530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.24530%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DIncorporating%2520replacement%2520free%2520energy%2520of%2520binding-site%2520waters%2520in%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2014%26volume%3D82%26spage%3D1765%26epage%3D1776%26doi%3D10.1002%2Fprot.24530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based scoring of protein–ligand complexes: Enrichment of known inhibitors in large-scale virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span><span class="refDoi"> DOI: 10.1021/ci0502855</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci0502855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=243-253&author=N.+Huangauthor=C.+Kalyanaramanauthor=J.+J.+Irwinauthor=M.+P.+Jacobson&title=Physics-based+scoring+of+protein%E2%80%93ligand+complexes%3A+Enrichment+of+known+inhibitors+in+large-scale+virtual+screening&doi=10.1021%2Fci0502855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-Based Scoring of Protein-Ligand Complexes: Enrichment of Known Inhibitors in Large-Scale Virtual Screening</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Irwin, John J.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors demonstrate that using an all-atom mol. mechanics force field combined with an implicit solvent model for scoring protein-ligand complexes is a promising approach for improving inhibitor enrichment in the virtual screening of large compd. databases.  The rescoring method is evaluated by the extent to which known binders for nine diverse, therapeutically relevant enzymes are enriched against a background of ∼100 000 drug-like decoys.  The improvement in enrichment is most robust and dramatic within the top 1% of the ranked database, i.e., the first thousand compds.; below the first few percent of the ranked database, there is little overall improvement.  The improved early enrichment is likely due to the more realistic treatment of ligand and receptor desolvation in the rescoring procedure.  We also present anecdotal but encouraging results assessing the ability of the rescoring method to predict specificity of inhibitors for structurally related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUqkz9N6IErLVg90H21EOLACvtfcHk0lgXgT7KK-ZIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N&md5=cf636695aca21a87e050e0f39286ea1d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci0502855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0502855%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520scoring%2520of%2520protein%25E2%2580%2593ligand%2520complexes%253A%2520Enrichment%2520of%2520known%2520inhibitors%2520in%2520large-scale%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D243%26epage%3D253%26doi%3D10.1021%2Fci0502855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">Discovery of novel tubulin inhibitors via structure-based hierarchical virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2730</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/ci300302c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300302c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlKltrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2730-2740&author=R.+Caoauthor=M.+Liuauthor=M.+Yinauthor=Q.+Liuauthor=Y.+Wangauthor=N.+Huang&title=Discovery+of+novel+tubulin+inhibitors+via+structure-based+hierarchical+virtual+screening&doi=10.1021%2Fci300302c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Tubulin Inhibitors via Structure-Based Hierarchical Virtual Screening</span></div><div class="casAuthors">Cao, Ran; Liu, Minyu; Yin, Min; Liu, Quanhai; Wang, Yanli; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2730-2740</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To discover novel tubulin inhibitors, we performed structure-based virtual screening against the colchicine binding pocket.  In combination with a hierarchical docking and scoring procedure, the structural information of an addnl. subpocket in colchicine site was applied to filter out the undesired docking hits.  This strategy automatically resulted in 63 candidates meeting the structural and energetic criteria from a screening library contg. approx. 100 000 diverse druglike compds.  Among them, nine mols. were chosen for exptl. validation, which all share the similar binding pose and contain an enriched scaffold bearing thiophene core.  Encouragingly, five compds. are active in tubulin polymn. assay.  The most potent inhibitor, 2-(2-fluorobenzamido)-3-carboxamide-4,5-dimethylthiophene, is structurally distinct to any known colchicine site binders and has higher ligand efficiency than colchicine.  On the basis of its predicted binding pose, we systematically probed its binding characteristics by testing series of structural modifications.  The obtained structure-activity relationship results are consistent with our binding model, and the inhibition activities of two analogs are improved by 2-fold.  We expect that the novel structure discovered in the present study may serve as a starting point for developing tubulin inhibitors with improved efficacy and fewer side effects.  We also expect that our hierarchical strategy may be generally applicable in structure-based virtual screening campaigns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeoH-2Mo3J0LVg90H21EOLACvtfcHk0lhj24uXjxHsdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlKltrzJ&md5=ddc129d0a8e5baa601f3e0076a184bbf</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci300302c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300302c%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520novel%2520tubulin%2520inhibitors%2520via%2520structure-based%2520hierarchical%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2730%26epage%3D2740%26doi%3D10.1021%2Fci300302c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armbruster, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernsberger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">206</span><span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lhj24uXjxHsdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Yuthavong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarnchompoo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vilaivan, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chitnumsub, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamchonwongpaisan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLennan, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vivas, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bongard, E.</span><span class="NLM_etal"></span><span> </span><span class="NLM_article-title">Malarial dihydrofolate reductase as a paradigm for drug development against a resistance-compromised target</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">16823</span><span class="NLM_x">–</span> <span class="NLM_lpage">16828</span><span class="refDoi"> DOI: 10.1073/pnas.1204556109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1073%2Fpnas.1204556109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23035243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WksL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16823-16828&author=Y.+Yuthavongauthor=B.+Tarnchompooauthor=T.+Vilaivanauthor=P.+Chitnumsubauthor=S.+Kamchonwongpaisanauthor=S.+A.+Charmanauthor=D.+N.+McLennanauthor=K.+L.+Whiteauthor=L.+Vivasauthor=E.+Bongard&title=Malarial+dihydrofolate+reductase+as+a+paradigm+for+drug+development+against+a+resistance-compromised+target&doi=10.1073%2Fpnas.1204556109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Malarial dihydrofolate reductase as a paradigm for drug development against a resistance-compromised target</span></div><div class="casAuthors">Yuthavong, Yongyuth; Tarnchompoo, Bongkoch; Vilaivan, Tirayut; Chitnumsub, Penchit; Kamchonwongpaisan, Sumalee; Charman, Susan A.; McLennan, Danielle N.; White, Karen L.; Vivas, Livia; Bongard, Emily; Thongphanchang, Chawanee; Taweechai, Supannee; Vanichtanankul, Jarunee; Rattanajak, Roonglawan; Arwon, Uthai; Fantauzzi, Pascal; Yuvaniyama, Jirundon; Charman, William N.; Matthews, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">16823-16828, S16823/1-S16823/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Malarial dihydrofolate reductase (DHFR) is the target of antifolate antimalarial drugs such as pyrimethamine and cycloguanil, the clin. efficacy of which have been compromised by resistance arising through mutations at various sites on the enzyme.  Here, we describe the use of cocrystal structures with inhibitors and substrates, along with efficacy and pharmacokinetic profiling for the design, characterization, and preclin. development of a selective, highly efficacious, and orally available antimalarial drug candidate (P218) that potently inhibits both wild-type and clin. relevant mutated forms of Plasmodium falciparum (Pf) DHFR.  Important structural characteristics of P218 include pyrimidine side-chain flexibility and a carboxylate group that makes charge-mediated hydrogen bonds with conserved Arg122 (PfDHFR-TS amino acid numbering).  An analogous interaction of P218 with human DHFR is disfavored because of three species-dependent amino acid substitutions in the vicinity of the conserved Arg.  Thus, P218 binds to the active site of PfDHFR in a substantially different fashion from the human enzyme, which is the basis for its high selectivity.  Unlike pyrimethamine, P218 binds both wild-type and mutant PfDHFR in a slow-on/slow-off tight-binding mode, which prolongs the target residence time.  P218, when bound to PfDHFR-TS, resides almost entirely within the envelope mapped out by the dihydrofolate substrate, which may make it less susceptible to resistance mutations.  The high in vivo efficacy in a SCID mouse model of P. falciparum malaria, good oral bioavailability, favorable enzyme selectivity, and good safety characteristics of P218 make it a potential candidate for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIVM7josnUBLVg90H21EOLACvtfcHk0lhAu_h18dq6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WksL7M&md5=7ef675391becc919f718de74fa61550b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1204556109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1204556109%26sid%3Dliteratum%253Aachs%26aulast%3DYuthavong%26aufirst%3DY.%26aulast%3DTarnchompoo%26aufirst%3DB.%26aulast%3DVilaivan%26aufirst%3DT.%26aulast%3DChitnumsub%26aufirst%3DP.%26aulast%3DKamchonwongpaisan%26aufirst%3DS.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DMcLennan%26aufirst%3DD.%2BN.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DVivas%26aufirst%3DL.%26aulast%3DBongard%26aufirst%3DE.%26atitle%3DMalarial%2520dihydrofolate%2520reductase%2520as%2520a%2520paradigm%2520for%2520drug%2520development%2520against%2520a%2520resistance-compromised%2520target%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D16823%26epage%3D16828%26doi%3D10.1073%2Fpnas.1204556109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Wan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span> </span><span class="NLM_article-title">A new target for an old drug: Identifying mitoxantrone as a nanomolar inhibitor of PIM1 kinase via kinome-wide selectivity modeling</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2619</span><span class="NLM_x">–</span> <span class="NLM_lpage">2629</span><span class="refDoi"> DOI: 10.1021/jm400045y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400045y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2619-2629&author=X.+Wanauthor=W.+Zhangauthor=L.+Liauthor=Y.+Xieauthor=W.+Liauthor=N.+Huang&title=A+new+target+for+an+old+drug%3A+Identifying+mitoxantrone+as+a+nanomolar+inhibitor+of+PIM1+kinase+via+kinome-wide+selectivity+modeling&doi=10.1021%2Fjm400045y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A New Target for an Old Drug: Identifying Mitoxantrone as a Nanomolar Inhibitor of PIM1 Kinase via Kinome-Wide Selectivity Modeling</span></div><div class="casAuthors">Wan, Xiaobo; Zhang, Wei; Li, Li; Xie, Yuting; Li, Wei; Huang, Niu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2619-2629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rational design of selective kinase inhibitors remains a great challenge.  Here we describe a physics-based approach to computationally modeling the kinase inhibitor selectivity profile.  We retrospectively assessed this protocol by computing the binding profiles of 17 well-known kinase inhibitors against 143 kinases.  Next, we predicted the binding profile of the chemotherapy drug mitoxantrone, and chose the predicted top five kinase targets for in vitro kinase assays.  Remarkably, mitoxantrone was shown to possess low nanomolar inhibitory activity against PIM1 kinase and to inhibit the PIM1-mediated phosphorylation in cancer cells.  We further detd. the crystal complex structure of PIM1 bound with mitoxantrone, which reveals the structural and mechanistic basis for a novel mode of PIM1 inhibition.  Although mitoxantrone's mechanism of action had been originally thought to act through DNA intercalation and type II topoisomerase inhibition, we hypothesize that PIM1 kinase inhibition might also contribute to mitoxantrone's therapeutic efficacy and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCdODeUj1frVg90H21EOLACvtfcHk0lhAu_h18dq6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCit78%253D&md5=ff53cfe69b80b0b28821a4bfb4583378</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm400045y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400045y%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DN.%26atitle%3DA%2520new%2520target%2520for%2520an%2520old%2520drug%253A%2520Identifying%2520mitoxantrone%2520as%2520a%2520nanomolar%2520inhibitor%2520of%2520PIM1%2520kinase%2520via%2520kinome-wide%2520selectivity%2520modeling%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2619%26epage%3D2629%26doi%3D10.1021%2Fjm400045y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">McCorvy, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Structure and function of serotonin G protein-coupled receptors</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">142</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.pharmthera.2015.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=25601315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVeisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=129-142&author=J.+D.+McCorvyauthor=B.+L.+Roth&title=Structure+and+function+of+serotonin+G+protein-coupled+receptors&doi=10.1016%2Fj.pharmthera.2015.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of serotonin G protein-coupled receptors</span></div><div class="casAuthors">McCorvy, John D.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">129-142</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Serotonin receptors are prevalent throughout the nervous system and the periphery, and remain one of the most lucrative and promising drug discovery targets for disorders ranging from migraine headaches to neuropsychiatric disorders such as schizophrenia and depression.  There are 14 distinct serotonin receptors, of which 13 are G protein-coupled receptors (GPCRs), which are targets for approx. 40% of the approved medicines.  Recent crystallog. and biochem. evidence has provided a converging understanding of the basic structure and functional mechanics of GPCR activation.  Currently, two GPCR crystal structures exist for the serotonin family, the 5-HT1B and 5-HT2B receptor, with the antimigraine and valvulopathic drug ergotamine bound.  The first serotonin crystal structures not only provide the first evidence of serotonin receptor topog. but also provide mechanistic explanations into functional selectivity or biased agonism.  This review will detail the findings of these crystal structures from a mol. and mutagenesis perspective for driving rational drug design for novel therapeutics incorporating biased signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpapliz37oS9bVg90H21EOLACvtfcHk0lhAu_h18dq6ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVeisbs%253D&md5=0fcfbc57dfb7047f6ffb7c74d437b5b4</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructure%2520and%2520function%2520of%2520serotonin%2520G%2520protein-coupled%2520receptors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D150%26spage%3D129%26epage%3D142%26doi%3D10.1016%2Fj.pharmthera.2015.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Al-Chaer, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawasaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasricha, P. J.</span><span> </span><span class="NLM_article-title">A new model of chronic visceral hypersensitivity in adult rats induced by colon irritation during postnatal development</span> <span class="citation_source-journal">Gastroenterology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">1276</span><span class="NLM_x">–</span> <span class="NLM_lpage">1285</span><span class="refDoi"> DOI: 10.1053/gast.2000.19576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1053%2Fgast.2000.19576" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2000&pages=1276-1285&author=E.+D.+Al-Chaerauthor=M.+Kawasakiauthor=P.+J.+Pasricha&title=A+new+model+of+chronic+visceral+hypersensitivity+in+adult+rats+induced+by+colon+irritation+during+postnatal+development&doi=10.1053%2Fgast.2000.19576"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1053%2Fgast.2000.19576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fgast.2000.19576%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Chaer%26aufirst%3DE.%2BD.%26aulast%3DKawasaki%26aufirst%3DM.%26aulast%3DPasricha%26aufirst%3DP.%2BJ.%26atitle%3DA%2520new%2520model%2520of%2520chronic%2520visceral%2520hypersensitivity%2520in%2520adult%2520rats%2520induced%2520by%2520colon%2520irritation%2520during%2520postnatal%2520development%26jtitle%3DGastroenterology%26date%3D2000%26volume%3D119%26spage%3D1276%26epage%3D1285%26doi%3D10.1053%2Fgast.2000.19576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Jacobson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span> </span><span class="NLM_article-title">Force field validation using protein side chain prediction</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">11673</span><span class="NLM_x">–</span> <span class="NLM_lpage">11680</span><span class="refDoi"> DOI: 10.1021/jp021564n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp021564n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvVOisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2002&pages=11673-11680&author=M.+A.+Jacobsonauthor=G.+A.+Kaminskiauthor=G.+A.+Friesnerauthor=C.+S.+Rapp&title=Force+field+validation+using+protein+side+chain+prediction&doi=10.1021%2Fjp021564n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Force Field Validation Using Protein Side Chain Prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Kaminski, George A.; Friesner, Richard A.; Rapp, Chaya S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">11673-11680</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The prediction of protein side chain conformations is used to evaluate the accuracy of force field parameters.  Specifically, new torsional parameters have recently been reported for the OPLS-AA force field, which achieved substantially better accuracy with respect to high level gas-phase quantum chem. calcns. [J. Phys. Chem. B 2001, 105, 6474].  Here we demonstrate that these new parameters also lead to qual. improved side chain prediction accuracy.  The primary emphasis is on the prediction of single side chain conformations, with the rest of the protein held fixed at the native configuration.  Errors due to incomplete sampling can thus be essentially eliminated, using a combination of rotamer search and energy minimization.  In addn., the protein environment is modeled realistically using implicit solvation and an explicit representation of crystal packing effects.  Aided by the development of new algorithms, these calcns. have been performed with modest computational requirements (a cluster of PCs) on a database of 36 proteins (∼5000 total residues).  The side chain prediction tests that we employ are quite general and can be used to evaluate nonbonded or solvation parameters as well.  As such, they provide a useful complement to decoy studies for force field validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZJvh-DVauJrVg90H21EOLACvtfcHk0ljpjLaeA5xJQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvVOisrw%253D&md5=1cc23c97485ac031d3f2524f672370a4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjp021564n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp021564n%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BA.%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DG.%2BA.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26atitle%3DForce%2520field%2520validation%2520using%2520protein%2520side%2520chain%2520prediction%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2002%26volume%3D106%26spage%3D11673%26epage%3D11680%26doi%3D10.1021%2Fjp021564n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pincus, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0ljpjLaeA5xJQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">High-resolution prediction of protein helix positions and orientations</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">368</span><span class="NLM_x">–</span> <span class="NLM_lpage">382</span><span class="refDoi"> DOI: 10.1002/prot.20014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.20014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=368-382&author=X.+Liauthor=M.+P.+Jacobsonauthor=R.+A.+Friesner&title=High-resolution+prediction+of+protein+helix+positions+and+orientations&doi=10.1002%2Fprot.20014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fprot.20014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20014%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DHigh-resolution%2520prediction%2520of%2520protein%2520helix%2520positions%2520and%2520orientations%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26spage%3D368%26epage%3D382%26doi%3D10.1002%2Fprot.20014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0ljDQ2InJ0neCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Hornak, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strockbine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roitberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmerling, C.</span><span> </span><span class="NLM_article-title">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1002/prot.21123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=712-725&author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&title=Comparison+of+multiple+Amber+force+fields+and+development+of+improved+protein+backbone+parameters&doi=10.1002%2Fprot.21123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of α-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./ψ dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./ψ dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0ljDQ2InJ0neCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520multiple%2520Amber%2520force%2520fields%2520and%2520development%2520of%2520improved%2520protein%2520backbone%2520parameters%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725%26doi%3D10.1002%2Fprot.21123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0ljDQ2InJ0neCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Sciortino, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essmann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hemmati, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angell, C. A.</span><span> </span><span class="NLM_article-title">Crystal stability limits at positive and negative pressures, and crystal-to-glass transitions</span> <span class="citation_source-journal">Phys. Rev. E: Stat. Phys., Plasmas, Fluids, Relat. Interdiscip. Top.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6484</span><span class="NLM_x">–</span> <span class="NLM_lpage">6491</span><span class="refDoi"> DOI: 10.1103/PhysRevE.52.6484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1103%2FPhysRevE.52.6484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=9964167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSnsr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1995&pages=6484-6491&author=F.+Sciortinoauthor=U.+Essmannauthor=H.+E.+Stanleyauthor=M.+Hemmatiauthor=J.+Shaoauthor=G.+H.+Wolfauthor=C.+A.+Angell&title=Crystal+stability+limits+at+positive+and+negative+pressures%2C+and+crystal-to-glass+transitions&doi=10.1103%2FPhysRevE.52.6484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal stability limits at positive and negative pressures, and crystal-to-glass transitions</span></div><div class="casAuthors">Sciortino, Francesco; Essmann, Ulrich; Stanley, H. Eugene; Hemmati, Mahin; Shao, Jun; Wolf, George H.; Angell, C. Austen</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review E: Statistical Physics, Plasmas, Fluids, and Related Interdisciplinary Topics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6-B</span>),
    <span class="NLM_cas:pages">6484-91</span>CODEN:
                <span class="NLM_cas:coden">PLEEE8</span>;
        ISSN:<span class="NLM_cas:issn">1063-651X</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">The direct crystal-to-glass transformation, i.e., spontaneous amorphization, which was first obsd. by thermal annealing of stishovite SiO2 at ambient pressure, has now been obsd. as an isothermal phenomenon during both compression and decompression of initially stable crystals.  While counterintuitive, and dependent on kinetically controlled metastable events, the phenomenon is of broad interest and potential importance in materials science and geophysics.  The authors use a combination of mol. dynamics simulations and analyses of lab. data to explore the metastable crystal ranges, including the neg. pressure range, for key compds. such as the ices, silicas, and alk. earth perovskites.  The focus is on the establishment of phenomenol. patterns rather than on specific metastability-terminating mechanisms.  A simple quadratic law, P-Ps ∼ (V-Vs)2 (where Ps and Vs are the values of the pressure P and vol. V on the spinodal), well approximates the equations of state over much of the metastable and even the stable range - and implies the existence of an isochoric boundary line for stability to isotropic d. fluctuations.  The authors delineate the conditions under which amorphization occurs, usually substantially before the stability limit is reached.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPt_Pd-cyJRrVg90H21EOLACvtfcHk0lhpPEbSQ0vWQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSnsr3N&md5=dd10f8c6ef5e00ec0a7903bf17df84e9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1103%2FPhysRevE.52.6484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevE.52.6484%26sid%3Dliteratum%253Aachs%26aulast%3DSciortino%26aufirst%3DF.%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DStanley%26aufirst%3DH.%2BE.%26aulast%3DHemmati%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DG.%2BH.%26aulast%3DAngell%26aufirst%3DC.%2BA.%26atitle%3DCrystal%2520stability%2520limits%2520at%2520positive%2520and%2520negative%2520pressures%252C%2520and%2520crystal-to-glass%2520transitions%26jtitle%3DPhys.%2520Rev.%2520E%253A%2520Stat.%2520Phys.%252C%2520Plasmas%252C%2520Fluids%252C%2520Relat.%2520Interdiscip.%2520Top.%26date%3D1995%26volume%3D52%26spage%3D6484%26epage%3D6491%26doi%3D10.1103%2FPhysRevE.52.6484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bekker, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, J.</span><span> </span><span class="NLM_article-title">LINCS: A linear constraint solver for molecular simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1463</span><span class="NLM_x">–</span> <span class="NLM_lpage">1472</span><span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.3.CO;2-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+Fraaije&title=LINCS%3A+A+linear+constraint+solver+for+molecular+simulations&doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.3.CO%253B2-L%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%26atitle%3DLINCS%253A%2520A%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26spage%3D1463%26epage%3D1472%26doi%3D10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.3.CO%3B2-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taneishi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujimoto, G.</span><span> </span><span class="NLM_article-title">GLIDA: GPCR-ligand database for chemical genomic drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">D673</span><span class="NLM_x">–</span> <span class="NLM_lpage">D677</span><span class="refDoi"> DOI: 10.1093/nar/gkj028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1093%2Fnar%2Fgkj028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=D673-D677&author=Y.+Okunoauthor=J.+Yangauthor=K.+Taneishiauthor=H.+Yabuuchiauthor=G.+Tsujimoto&title=GLIDA%3A+GPCR-ligand+database+for+chemical+genomic+drug+discovery&doi=10.1093%2Fnar%2Fgkj028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkj028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkj028%26sid%3Dliteratum%253Aachs%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTaneishi%26aufirst%3DK.%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DTsujimoto%26aufirst%3DG.%26atitle%3DGLIDA%253A%2520GPCR-ligand%2520database%2520for%2520chemical%2520genomic%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DD673%26epage%3DD677%26doi%3D10.1093%2Fnar%2Fgkj028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niijima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minowa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonomura, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunimoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, C.</span><span> </span><span class="NLM_article-title">GLIDA: GPCR—ligand database for chemical genomics drug discovery—database and tools update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D907</span><span class="NLM_x">–</span> <span class="NLM_lpage">D912</span><span class="refDoi"> DOI: 10.1093/nar/gkm948</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1093%2Fnar%2Fgkm948" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D907-D912&author=Y.+Okunoauthor=A.+Tamonauthor=H.+Yabuuchiauthor=S.+Niijimaauthor=Y.+Minowaauthor=K.+Tonomuraauthor=R.+Kunimotoauthor=C.+Feng&title=GLIDA%3A+GPCR%E2%80%94ligand+database+for+chemical+genomics+drug+discovery%E2%80%94database+and+tools+update&doi=10.1093%2Fnar%2Fgkm948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm948%26sid%3Dliteratum%253Aachs%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DTamon%26aufirst%3DA.%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTonomura%26aufirst%3DK.%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DC.%26atitle%3DGLIDA%253A%2520GPCR%25E2%2580%2594ligand%2520database%2520for%2520chemical%2520genomics%2520drug%2520discovery%25E2%2580%2594database%2520and%2520tools%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD907%26epage%3DD912%26doi%3D10.1093%2Fnar%2Fgkm948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span><span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lhnlwX8lZiROQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carchia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Directory of useful decoys, enhanced (DUD-E): Better ligands and decoys for better benchmarking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6582</span><span class="NLM_x">–</span> <span class="NLM_lpage">6594</span><span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&author=M.+M.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+useful+decoys%2C+enhanced+%28DUD-E%29%3A+Better+ligands+and+decoys+for+better+benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0lhnlwX8lZiROQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520useful%2520decoys%252C%2520enhanced%2520%2528DUD-E%2529%253A%2520Better%2520ligands%2520and%2520decoys%2520for%2520better%2520benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span><span class="refDoi"> DOI: 10.1021/jm0608356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking&doi=10.1021%2Fjm0608356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0liWjaQNrAiIQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801%26doi%3D10.1021%2Fjm0608356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baase, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A model binding site for testing scoring functions in molecular docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">355</span><span class="refDoi"> DOI: 10.1016/S0022-2836(02)00777-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2FS0022-2836%2802%2900777-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=12217695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=339-355&author=B.+Q.+Weiauthor=W.+A.+Baaseauthor=L.+H.+Weaverauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking&doi=10.1016%2FS0022-2836%2802%2900777-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Model Binding Site for Testing Scoring Functions in Molecular Docking</span></div><div class="casAuthors">Wei, Binqing Q.; Baase, Walter A.; Weaver, Larry H.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-355</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Prediction of interaction energies between ligands and their receptors remains a major challenge for structure-based inhibitor discovery.  Much effort has been devoted to developing scoring schemes that can successfully rank the affinities of a diverse set of possible ligands to a binding site for which the structure is known.  To test these scoring functions, well-characterized exptl. systems can be very useful.  Here, mutation-created binding sites in T4 lysozyme were used to investigate how the quality of at. charges and solvation energies affects mol. docking.  At. charges and solvation energies were calcd. for 172,118 mols. in the Available Chems.  Directory using a semi-empirical quantum mech. approach by the program AMSOL.  The database was first screened against the apolar cavity site created by the mutation Leu99Ala (L99A).  Compared to the electronegativity-based charges that are widely used, the new charges and desolvation energies improved ranking of known apolar ligands, and better distinguished them from more polar isosteres that are not obsd. to bind.  To investigate whether the new charges had predictive value, the non-polar residue Met102, which forms part of the binding site, was changed to the polar residue glutamine.  The structure of the resulting Leu99 Ala and Met102 Gln double mutant of T4 lysozyme (L99A/M102Q) was detd. and the docking calcn. was repeated for the new site.  Seven representative polar mols. that preferentially docked to the polar vs. the apolar binding site were tested exptl.  All seven bind to the polar cavity (L99A/M102Q) but do not detectably bind to the apolar cavity (L99A).  Five ligand-bound structures of L99A/M102Q were detd. by X-ray crystallog.  Docking predictions corresponded to the crystallog. results to within 0.4 A RMSD.  Improved treatment of partial at. charges and desolvation energies in database docking appears feasible and leads to better distinction of true ligands.  Simple model binding sites, such as L99A and its more polar variants, may find broad use in the development and testing of docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGral4WDhJW6jbVg90H21EOLACvtfcHk0liWjaQNrAiIQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D&md5=040e009b33f125ab4abbb322cf32d2c9</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900777-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900777-5%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBaase%26aufirst%3DW.%2BA.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D339%26epage%3D355%26doi%3D10.1016%2FS0022-2836%2802%2900777-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Lorber, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Hierarchical docking of databases of multiple ligand conformations</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">749</span><span class="refDoi"> DOI: 10.2174/1568026054637683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.2174%2F1568026054637683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=16101414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=739-749&author=D.+M.+Lorberauthor=B.+K.+Shoichet&title=Hierarchical+docking+of+databases+of+multiple+ligand+conformations&doi=10.2174%2F1568026054637683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical docking of databases of multiple ligand conformations</span></div><div class="casAuthors">Lorber, David M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">739-749</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Ligand flexibility is an important problem in mol. docking and virtual screening.  To address this challenge, we investigate a hierarchical pre-organization of multiple conformations of small mols.  Such organization of pre-calcd. conformations removes the exploration of ligand conformational space from the docking calcn. and allows for concise representation of what can be thousands of conformations.  The hierarchy also recognizes and prunes incompatible conformations early in the calcn., eliminating redundant calcns. of fit.  We investigate the method by docking the MDL Drug Data Report (MDDR), an annotated database of 100,000 mols., into apo and holo forms of 7 unrelated targets.  This annotated database allows us to track the ranking of tens to hundreds of annotated ligands in each of the docking systems.  The binding sites and database are prepd. in an automated fashion in an attempt to remove some human bias from the calcns.  Many thousands of explicit and implicit ligand conformations may be docked in calcns. not much longer than required for single conformer docking.  As long as internal energies are not considered, recombination with the hierarchy is additive as the no. of degrees of freedom is increased.  Mols. with even millions of conformations can be docked in a few minutes on a single desktop computer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ_iMnNjm9LVg90H21EOLACvtfcHk0liWjaQNrAiIQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D&md5=25c833b5608e5faedc1af3ad38fdfa06</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.2174%2F1568026054637683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054637683%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DHierarchical%2520docking%2520of%2520databases%2520of%2520multiple%2520ligand%2520conformations%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D739%26epage%3D749%26doi%3D10.2174%2F1568026054637683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Ferrin, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvis, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span> </span><span class="NLM_article-title">The MIDAS display system</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1016/0263-7855(88)80054-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2F0263-7855%2888%2980054-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaL1cXktVGrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1988&pages=13-27&author=T.+E.+Ferrinauthor=C.+C.+Huangauthor=L.+E.+Jarvisauthor=R.+Langridge&title=The+MIDAS+display+system&doi=10.1016%2F0263-7855%2888%2980054-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The MIDAS display system</span></div><div class="casAuthors">Ferrin, Thomas E.; Huang, Conrad C.; Jarvis, Laurie E.; Langridge, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-27, 36-7</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    </div><div class="casAbstract">A description is given of the Mol. Interactive Display and Simulation (MIDAS) system for the manipulation and display of large macromols., such as proteins and nucleic acids.  Several ancillary programs are described that allow for such features as computing the surface of a mol., and computing electrostatic charge potentials.  The core of MIDAS is a hierarchical database system, designed specifically for macromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGxkghePxXQ7Vg90H21EOLACvtfcHk0ljjoIdDdCej3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktVGrsrs%253D&md5=86a9a293a0b51feb7a7310317bb99662</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2888%2980054-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252888%252980054-7%26sid%3Dliteratum%253Aachs%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DJarvis%26aufirst%3DL.%2BE.%26aulast%3DLangridge%26aufirst%3DR.%26atitle%3DThe%2520MIDAS%2520display%2520system%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1988%26volume%3D6%26spage%3D13%26epage%3D27%26doi%3D10.1016%2F0263-7855%2888%2980054-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oatley, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">A geometric approach to macromolecule-ligand interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1982</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">288</span><span class="refDoi"> DOI: 10.1016/0022-2836(82)90153-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2F0022-2836%2882%2990153-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=7154081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=1982&pages=269-288&author=I.+D.+Kuntzauthor=J.+M.+Blaneyauthor=S.+J.+Oatleyauthor=R.+Langridgeauthor=T.+E.+Ferrin&title=A+geometric+approach+to+macromolecule-ligand+interactions&doi=10.1016%2F0022-2836%2882%2990153-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A geometric approach to macromolecule-ligand interactions</span></div><div class="casAuthors">Kuntz, Irwin D.; Blaney, Jeffrey M.; Oatley, Stuart J.; Langridge, Robert; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-88</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A method is described to explore geometrically feasible alignments of ligands and receptors of known structure.  Algorithms are presented that examine many binding geometries and evaluate them in terms of steric overlap.  The procedure uses specific mol. conformations.  A method is included for finding putative binding sites on a macromol. surface.  Results are reported for heme-myoglobin interaction and the binding of thyroid hormone analogs to prealbumin.  In each case, the program finds structures within 1 Å of the x-ray results and also finds distinctly different geometries that provide good steric fits.  The approach seems well-suited for generating conformations for energy refinement programs and interactive computer graphics routines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWsp3KKxuQf7Vg90H21EOLACvtfcHk0ljjoIdDdCej3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D&md5=8a4234b24356ea5340f33d906cd71d3e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2882%2990153-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252882%252990153-X%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DOatley%26aufirst%3DS.%2BJ.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DA%2520geometric%2520approach%2520to%2520macromolecule-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D161%26spage%3D269%26epage%3D288%26doi%3D10.1016%2F0022-2836%2882%2990153-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span><span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0ljjoIdDdCej3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Nicholls, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">A rapid finite difference algorithm, utilizing successive over-relaxation to solve the Poisson-Boltzmann equation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">445</span><span class="refDoi"> DOI: 10.1002/jcc.540120405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1002%2Fjcc.540120405" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1991&pages=435-445&author=A.+Nichollsauthor=B.+Honig&title=A+rapid+finite+difference+algorithm%2C+utilizing+successive+over-relaxation+to+solve+the+Poisson-Boltzmann+equation&doi=10.1002%2Fjcc.540120405"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540120405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540120405%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DA%2520rapid%2520finite%2520difference%2520algorithm%252C%2520utilizing%2520successive%2520over-relaxation%2520to%2520solve%2520the%2520Poisson-Boltzmann%2520equation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1991%26volume%3D12%26spage%3D435%26epage%3D445%26doi%3D10.1002%2Fjcc.540120405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC-a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC-a+free+database+of+commercially+available+compounds+for+virtual+screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eauvI5sFXW3rLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC-a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: A free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+A+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lgDkw4qHs4-3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520A%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span> </span><span class="NLM_article-title">Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11225</span><span class="NLM_x">–</span> <span class="NLM_lpage">11236</span><span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+testing+of+the+OPLS+all-atom+force+field+on+conformational+energetics+and+properties+of+organic+liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0ljfBOf6xrffSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520the%2520OPLS%2520all-atom%2520force%2520field%2520on%2520conformational%2520energetics%2520and%2520properties%2520of%2520organic%2520liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Evaluation and reparametrization of the OPLS-AA force field for proteins via comparison with accurate quantum chemical calculations on peptides</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6474</span><span class="NLM_x">–</span> <span class="NLM_lpage">6487</span><span class="refDoi"> DOI: 10.1021/jp003919d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp003919d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=6474-6487&author=G.+A.+Kaminskiauthor=R.+A.+Friesnerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Evaluation+and+reparametrization+of+the+OPLS-AA+force+field+for+proteins+via+comparison+with+accurate+quantum+chemical+calculations+on+peptides&doi=10.1021%2Fjp003919d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span></div><div class="casAuthors">Kaminski, George A.; Friesner, Richard A.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6474-6487</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present results of improving the OPLS-AA force field for peptides by means of refitting the key Fourier torsional coeffs.  The fitting technique combines using accurate ab initio data as the target, choosing an efficient fitting subspace of the whole potential-energy surface, and detg. wts. for each of the fitting points based on magnitudes of the potential-energy gradient.  The av. energy RMS deviation from the LMP2/cc-pVTZ(-f)//HF/6-31G** data is reduced by ∼40% from 0.81 to 0.47 kcal/mol as a result of the fitting for the electrostatically uncharged dipeptides.  Transferability of the parameters is demonstrated by using the same alanine dipeptide-fitted backbone torsional parameters for all of the other dipeptides (with the appropriate side-chain refitting) and the alanine tetrapeptide.  Parameters of nonbonded interactions have also been refitted for the sulfur-contg. dipeptides (cysteine and methionine), and the validity of the new Coulombic charges and the van der Waals σ's and ε's is proved through reproducing gas-phase energies of complex formation heats of vaporization and densities of pure model liqs.  Moreover, a novel approach to fitting torsional parameters for electrostatically charged mol. systems has been presented and successfully tested on five dipeptides with charged side chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSHwmffIQePbVg90H21EOLACvtfcHk0ljfBOf6xrffSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D&md5=3ff059626977ee7f6342466f5820f5b7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjp003919d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp003919d%26sid%3Dliteratum%253Aachs%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEvaluation%2520and%2520reparametrization%2520of%2520the%2520OPLS-AA%2520force%2520field%2520for%2520proteins%2520via%2520comparison%2520with%2520accurate%2520quantum%2520chemical%2520calculations%2520on%2520peptides%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D105%26spage%3D6474%26epage%3D6487%26doi%3D10.1021%2Fjp003919d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Ma, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poon, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chui, W. K.</span><span> </span><span class="NLM_article-title">Synthesis and in vitro evaluation of 2,4-diamino-1,3,5-triazine derivatives as neuronal voltage-gated sodium channel blockers</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">5644</span><span class="NLM_x">–</span> <span class="NLM_lpage">5647</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2Fj.bmcl.2009.08.052" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=5644-5647&author=X.+Maauthor=T.+Y.+Poonauthor=P.+T.+Wongauthor=W.+K.+Chui&title=Synthesis+and+in+vitro+evaluation+of+2%2C4-diamino-1%2C3%2C5-triazine+derivatives+as+neuronal+voltage-gated+sodium+channel+blockers&doi=10.1016%2Fj.bmcl.2009.08.052"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DX.%26aulast%3DPoon%26aufirst%3DT.%2BY.%26aulast%3DWong%26aufirst%3DP.%2BT.%26aulast%3DChui%26aufirst%3DW.%2BK.%26atitle%3DSynthesis%2520and%2520in%2520vitro%2520evaluation%2520of%25202%252C4-diamino-1%252C3%252C5-triazine%2520derivatives%2520as%2520neuronal%2520voltage-gated%2520sodium%2520channel%2520blockers%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D5644%26epage%3D5647%26doi%3D10.1016%2Fj.bmcl.2009.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lg3UY_AVg83Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Papp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moryl, E.</span><span> </span><span class="NLM_article-title">Antidepressant activity of non-competitive and competitive NMDA receptor antagonists in a chronic mild stress model of depression</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">263</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/0014-2999(94)90516-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=10.1016%2F0014-2999%2894%2990516-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=7821340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;key=1%3ACAS%3A528%3ADyaK2cXmsVKitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1994&pages=1-7&author=M.+Pappauthor=E.+Moryl&title=Antidepressant+activity+of+non-competitive+and+competitive+NMDA+receptor+antagonists+in+a+chronic+mild+stress+model+of+depression&doi=10.1016%2F0014-2999%2894%2990516-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Antidepressant activity of non-competitive and competitive NMDA receptor antagonists in a chronic mild stress model of depression</span></div><div class="casAuthors">Papp, Mariusz; Moryl, Elzbieta</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">1/2</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The antidepressant properties of the non-competitive NMDA receptor antagonist, MK-801 (dizocilpine), and the competitive NMDA receptor antagonist, CGP 37849 (DL-(E)-2-amino-4-methyl-5-phosphono-3-pentonoic acid) and its (R)-enantiomer CGP 40116, were studied in a chronic mild stress model of depression.  In this model, animals subjected to a variety of mild stressors for a prolonged period of time show a substantial decrease in the consumption of palatable sucrose soln. (anhedonia).  It was previously demonstrated that the chronic mild stress-induced anhedonia can be reversed by chronic treatment with various antidepressant drugs.  In this study we found that the stress-induced deficit in sucrose intake was gradually reversed by chronic (4-5 wk) treatment with MK-801 (0.3 mg/kg i.p.), CGP 37849 (5 mg/kg i.p.) and CGP (25 mg/kg p.o.).  The magnitude of this effect and its time course were comparable to those obsd. following similar administration of imipramine (10 mg/kg i.p. or p.o.).  The increase in sucrose intake following chronic administration of imipramine and NMDA receptor antagonists was specific to stressed animals; the behavior of non-stressed controls was unchanged by any of the drugs tested.  These results confirm those of previous studies, carried out on 'normal' animals, suggesting that antagonists of NMDA receptors may have antidepressant properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXAZ-Pbbeub7Vg90H21EOLACvtfcHk0lg3UY_AVg83Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsVKitL8%253D&md5=bae5b93e49fd2ef9054fbbe7470bb0a3</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2894%2990516-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252894%252990516-9%26sid%3Dliteratum%253Aachs%26aulast%3DPapp%26aufirst%3DM.%26aulast%3DMoryl%26aufirst%3DE.%26atitle%3DAntidepressant%2520activity%2520of%2520non-competitive%2520and%2520competitive%2520NMDA%2520receptor%2520antagonists%2520in%2520a%2520chronic%2520mild%2520stress%2520model%2520of%2520depression%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1994%26volume%3D263%26spage%3D1%26epage%3D7%26doi%3D10.1016%2F0014-2999%2894%2990516-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB4','PDB','4IB4'); return false;">PDB: 4IB4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IAR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IAR','PDB','4IAR'); return false;">PDB: 4IAR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RZE','PDB','3RZE'); return false;">PDB: 3RZE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AMJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AMJ','PDB','4AMJ'); return false;">PDB: 4AMJ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01631">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_71311"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01631">10.1021/acs.jmedchem.5b01631</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures and tables illustrating all computational and biological results, detailed experimental procedures, and <sup>1</sup>H NMR spectra and LC-MS spectra data of all synthesized compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Table illustrating biological results (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_001.pdf">jm5b01631_si_001.pdf (2.49 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01631/suppl_file/jm5b01631_si_002.csv">jm5b01631_si_002.csv (3.96 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01631&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01631%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01631" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992630a9673cb2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
