
<html lang="en"     class="pb-page"  data-request-id="c8ed206a-3dd3-4aee-a5df-5b6404898e71"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01303;issue:issue:10.1021/jmcmar.2016.59.issue-24;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Design, Synthesis, and Biological Activity of Substrate Competitive SMYD2 Inhibitors" /></meta><meta name="dc.Creator" content="Scott D.  Cowen" /></meta><meta name="dc.Creator" content="Daniel  Russell" /></meta><meta name="dc.Creator" content="Leslie A.  Dakin" /></meta><meta name="dc.Creator" content="Huawei  Chen" /></meta><meta name="dc.Creator" content="Nicholas A.  Larsen" /></meta><meta name="dc.Creator" content="Robert  Godin" /></meta><meta name="dc.Creator" content="Scott  Throner" /></meta><meta name="dc.Creator" content="Xiaolan  Zheng" /></meta><meta name="dc.Creator" content="Audrey  Molina" /></meta><meta name="dc.Creator" content="Jiaquan  Wu" /></meta><meta name="dc.Creator" content="Tony  Cheung" /></meta><meta name="dc.Creator" content="Tina  Howard" /></meta><meta name="dc.Creator" content="Renee  Garcia-Arenas" /></meta><meta name="dc.Creator" content="Nicholas  Keen" /></meta><meta name="dc.Creator" content="Christopher S.  Pendleton" /></meta><meta name="dc.Creator" content="Jennifer A.  Pietenpol" /></meta><meta name="dc.Creator" content="Andrew D.  Ferguson" /></meta><meta name="dc.Description" content="Protein lysine methyltransferases (KMTs) have emerged as important regulators of epigenetic signaling. These enzymes catalyze the transfer of donor methyl groups from the cofactor S-adenosylmethion..." /></meta><meta name="Description" content="Protein lysine methyltransferases (KMTs) have emerged as important regulators of epigenetic signaling. These enzymes catalyze the transfer of donor methyl groups from the cofactor S-adenosylmethion..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 1, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01303" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01303" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01303" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01303" /></link>
        
    
    

<title>Design, Synthesis, and Biological Activity of Substrate Competitive SMYD2 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01303" /></meta><meta property="og:title" content="Design, Synthesis, and Biological Activity of Substrate Competitive SMYD2 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0022.jpeg" /></meta><meta property="og:description" content="Protein lysine methyltransferases (KMTs) have emerged as important regulators of epigenetic signaling. These enzymes catalyze the transfer of donor methyl groups from the cofactor S-adenosylmethionine to specific acceptor lysine residues on histones, leading to changes in chromatin structure and transcriptional regulation. These enzymes also methylate an array of nonhistone proteins, suggesting additional mechanisms by which they influence cellular physiology. SMYD2 is reported to be an oncogenic methyltransferase that represses the functional activity of the tumor suppressor proteins p53 and RB. HTS screening led to identification of five distinct substrate-competitive chemical series. Determination of liganded crystal structures of SMYD2 contributed significantly to “hit-to-lead” design efforts, culminating in the creation of potent and selective inhibitors that were used to understand the functional consequences of SMYD2 inhibition. Taken together, these results have broad implications for inhibitor design against KMTs and clearly demonstrate the potential for developing novel therapies against these enzymes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01303"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01303">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01303&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01303&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01303&amp;href=/doi/10.1021/acs.jmedchem.6b01303" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 11079-11097</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01277" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01333" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Biological Activity of Substrate Competitive SMYD2 Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Scott+D.++Cowen">Scott D. Cowen</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Russell">Daniel Russell</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Leslie+A.++Dakin">Leslie A. Dakin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Huawei++Chen">Huawei Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+A.++Larsen">Nicholas A. Larsen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Godin">Robert Godin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Throner">Scott Throner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaolan++Zheng">Xiaolan Zheng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Audrey++Molina">Audrey Molina</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiaquan++Wu">Jiaquan Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tony++Cheung">Tony Cheung</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tina++Howard">Tina Howard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Renee++Garcia-Arenas">Renee Garcia-Arenas</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Keen">Nicholas Keen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+S.++Pendleton">Christopher S. Pendleton</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+A.++Pietenpol">Jennifer A. Pietenpol</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+D.++Ferguson">Andrew D. Ferguson</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-1018-9631" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Department of Chemistry, <sup>‡</sup>Department of Bioscience, Oncology Innovative Medicines Unit, <sup>§</sup>Structure and Biophysics, Discovery Sciences, AstraZeneca R&D Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">∥</span> <span class="aff-text">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div><div class="corresp-info"><strong>*</strong>For A.D.F.: phone, 908-528-7078; E-mail, <a href="/cdn-cgi/l/email-protection#3f5e5b595a4d587f58525e5653115c5052"><span class="__cf_email__" data-cfemail="b1d0d5d7d4c3d6f1d6dcd0d8dd9fd2dedc">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For S.D.C.: phone, 774-278-8014); E-mail, <a href="/cdn-cgi/l/email-protection#d8bbb7afbdb6abbbe998b1bbb4b7adbcf6bbb7b5"><span class="__cf_email__" data-cfemail="7a19150d1f1409194b3a131916150f1e54191517">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01303&amp;href=/doi/10.1021%2Facs.jmedchem.6b01303" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 11079–11097</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 14, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 September 2016</li><li><span class="item_label"><b>Published</b> online</span>1 December 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 December 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01303" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01303</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11079%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DScott%2BD.%2BCowen%252C%2BDaniel%2BRussell%252C%2BLeslie%2BA.%2BDakin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D24%26contentID%3Dacs.jmedchem.6b01303%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BActivity%2Bof%2BSubstrate%2BCompetitive%2BSMYD2%2BInhibitors%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11097%26publicationDate%3DDecember%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01303"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2293</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">17</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01303" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Biological Activity of Substrate Competitive SMYD2 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;D. Cowen&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Russell&quot;},{&quot;first_name&quot;:&quot;Leslie&quot;,&quot;last_name&quot;:&quot;A. Dakin&quot;},{&quot;first_name&quot;:&quot;Huawei&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;A. Larsen&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Godin&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Throner&quot;},{&quot;first_name&quot;:&quot;Xiaolan&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Audrey&quot;,&quot;last_name&quot;:&quot;Molina&quot;},{&quot;first_name&quot;:&quot;Jiaquan&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Tony&quot;,&quot;last_name&quot;:&quot;Cheung&quot;},{&quot;first_name&quot;:&quot;Tina&quot;,&quot;last_name&quot;:&quot;Howard&quot;},{&quot;first_name&quot;:&quot;Renee&quot;,&quot;last_name&quot;:&quot;Garcia-Arenas&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Keen&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;S. Pendleton&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;A. Pietenpol&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;D. Ferguson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;11079-11097&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01303&quot;},&quot;abstract&quot;:&quot;Protein lysine methyltransferases (KMTs) have emerged as important regulators of epigenetic signaling. These enzymes catalyze the transfer of donor methyl groups from the cofactor S-adenosylmethionine to specific acceptor lysine residues on histones, leading to changes in chromatin structure and transcriptional regulation. These enzymes also methylate an array of nonhistone proteins, suggesting additional mechanisms by which they influence cellular physiology. SMYD2 is reported to be an oncogenic methyltransferase that represses the functional activity of the tumor suppressor proteins p53 and RB. HTS screening led to identification of five distinct substrate-competitive chemical series. Determination of liganded crystal structures of SMYD2 contributed significantly to “hit-to-lead” design efforts, culminating in the creation of potent and selective inhibitors that were used to understand the functional consequences of SMYD2 inhibition. Taken together, these results have broad implications for inhibitor design&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01303&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01303" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01303&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01303" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01303&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01303" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01303&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01303&amp;href=/doi/10.1021/acs.jmedchem.6b01303" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01303" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01303" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01303%26sid%3Dliteratum%253Aachs%26pmid%3D28002961%26genre%3Darticle%26aulast%3DCowen%26date%3D2016%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BActivity%2Bof%2BSubstrate%2BCompetitive%2BSMYD2%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D24%26spage%3D11079%26epage%3D11097%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/jmcmar.2016.59.issue-24/20161222/jmcmar.2016.59.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein lysine methyltransferases (KMTs) have emerged as important regulators of epigenetic signaling. These enzymes catalyze the transfer of donor methyl groups from the cofactor <i>S</i>-adenosylmethionine to specific acceptor lysine residues on histones, leading to changes in chromatin structure and transcriptional regulation. These enzymes also methylate an array of nonhistone proteins, suggesting additional mechanisms by which they influence cellular physiology. SMYD2 is reported to be an oncogenic methyltransferase that represses the functional activity of the tumor suppressor proteins p53 and RB. HTS screening led to identification of five distinct substrate-competitive chemical series. Determination of liganded crystal structures of SMYD2 contributed significantly to “<i>hit-to-lead</i>” design efforts, culminating in the creation of potent and selective inhibitors that were used to understand the functional consequences of SMYD2 inhibition. Taken together, these results have broad implications for inhibitor design against KMTs and clearly demonstrate the potential for developing novel therapies against these enzymes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38614" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38614" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein lysine methyltransferases (KMTs) are important regulators of epigenetic signaling. This enzyme class catalyzes the transfer of donor methyl groups from the cofactor <i>S</i>-adenosylmethionine to specific acceptor lysine residues on histones, resulting in discrete patterns of mono-, di-, or trimethylation on target substrates. Methylation marks act as recognition elements for additional effector proteins that induce changes in chromatin structure and transcriptional regulation. KMTs are also known to methylate numerous nonhistone protein substrates, revealing additional mechanisms to regulate cellular physiology, and this area has been the subject of several reviews.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">The KMT protein, SMYD2, has been implicated in a variety of cancers. Komatsu  et al.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> have shown that overexpression of SMYD2 is observed in KYSE150 cells with amplification at 1q32–41.1 and in other esophageal squamous cell carcinoma (ESCC) cell lines. Overexpression of SMYD2 mRNA and protein has also been detected in cell lines and primary tumor tissues.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> This study also showed that patients with SMYD2-overexpressing ESCC tumors had a lower survival rate when compared to those with non-SMYD2 expressing ESCC tumors. Knockdown of <i>SMYD2</i> gene expression inhibits ESCC cell proliferation in a <i>TP53</i> mutation-independent manner.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">p53 is a transcription factor that functions as a tumor suppressor that regulates cell cycle arrest through both transcription-dependent and independent signaling pathways. Inactivating mutations in the <i>p53</i> gene are observed in roughly 50% of all human cancers. SMYD2 preferentially methylates residue K370 of p53 when compared to histone substrates. Endogenous levels of monomethylated p53 are significantly elevated when SMYD2 is overexpressed in vivo,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> which decreases the ability of p53 to transcriptionally activate target genes involved in cell cycle arrest. These findings suggest that SMYD2 inhibition may provide a potential therapeutic option for cancers that utilize SMYD2 to inactivate p53 signaling.</div><div class="NLM_p">Recent studies have shown that SMYD2 acts upon additional nonhistone targets, including estrogen receptor α (ERα),<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> heat shock protein 90 (HSP90),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> poly(ADP-ribose) polymerase 1 (PARP1),<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and the retinoblastoma tumor suppressor (RB).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> These target proteins are involved in a variety of cellular processes, including epigenetic gene regulation, cellular differentiation, apoptosis, the DNA damage response, and cell cycle progression. In each of these cases, evidence has been presented that SMYD2-mediated methylation represses the functional activities of these proteins. A recent SILAC-based proteomics study<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> indicates that the number of target proteins that are methylated by SMYD2 is even larger than previously appreciated, thus the ability to discern which SMYD2 substrates may contribute to a given cellular phenotype has become increasingly complex.</div><div class="NLM_p">Here we report the discovery and “<i>hit-to-lead</i>” efforts leading to the development of potent and selective SMYD2 inhibitors (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) that will serve as valuable tools to further elucidate the physiological role of SMYD2.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of SMYD2 inhibitors<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70568" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70568" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Screening</h3><div class="NLM_p">We undertook an internal hit identification campaign to identify chemical equity against a panel of histone methyltransferases using a combination of high throughput screening and high concentration fragment screening. We previously reported the details of that screening campaign.<a onclick="showRef(event, 'ref9 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref11">(9, 11)</a> Those screening efforts identified a variety of chemotypes with activity against SMYD2 with IC<sub>50</sub> values ranging from 0.3 to 20 μM. Two generalized modes of inhibition are possible with KMTs, substrate competitive as observed with G9a and GLP,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> or cofactor competitive as seen with DOT1L (overlaps with the cofactor) and EZH2 (partially overlaps the cofactor).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> However, allosteric binding modes are also possible. Isothermal titration calorimetry (ITC) was used to validate the biophysical binding properties of exemplar compounds from the most chemically attractive clusters. Those experiments demonstrated clear binding for five distinct substrate competitive chemotypes (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). Intriguingly, these five series do not share any obvious structural or chemical similarities that would define a pharmacophore.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ITC validated hits identified from the screening campaign.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Hit Evaluation</h3><div class="NLM_p">With these validated hits, we initiated a “<i>hit-to-lead</i>” chemistry program with the goal of improving biochemical and cellular potency and to assess the associated pharmacokinetic profiles. We initially expanded the scope of the SAR of these series by evaluating near neighbors from the AstraZeneca corporate collection and through the synthesis of focused SAR expansion sets. Compound <b>5</b> (IC<sub>50</sub> = 5.6 μM) was particularly attractive as a starting point. It had modest enzymatic activity and a favorable physicochemical profile with excellent solubility but had a higher than desirable rat microsomal clearance (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). The other hits were also pursued. The series related to compound <b>6</b> was pursued and eventually furnished potent SMYD2 inhibitors, which will be discussed in another publication. Analogues of compound <b>7</b> were found to be unstable as the piperidine ring was prone to aromatization so that series was discontinued. Compound <b>8</b> and its analogues were also deprioritized as they are potent kinase inhibitors.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ITC titration of compound <b>5</b>: 34 μM SMYD2 protein solution and 400 μM compound <b>5</b> were used; <i>K</i><sub>d</sub> of 1.73 μM, Δ<i>H</i>° = −6.90 kCal/mol, <i>T</i>Δ<i>S</i>° = 3.14 kCal/mol, and <i>N</i> = 0.92.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">At the onset of the chemistry campaign, no crystallographic data was available for SMYD2. However, by analogy to the published structures of G9a and EHMT1<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> inhibitors, we hypothesized that the propyl pyrrolidine moiety of <b>5</b> functioned as a methyl lysine mimetic that would be positioned in the lysine binding channel of SMYD2. The synthetic route to generating <b>5</b> and analogues thereof was conveniently carried out in a two-step, one-pot reaction starting from pyruvic acid and the appropriate aldehydes and aminopyrazoles (see Chemistry section). Although the cyclocondensation step was low yielding, the simplicity of the reaction sequence allowed for rapid preparation of multiple intermediates. The resulting intermediates were then functionalized with a variety of amines (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). After preparing more than 50 analogues within this chemical series, we were unable to improve the biochemical potency of the initial hit. The IC<sub>50</sub> values for this series of analogues demonstrated steep SAR at this position, and the superiority of the propyl pyrrolidine substituent relative to other analogues. In parallel, we also examined the SAR of the C6 aryl functionality (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) and likewise observed that the orthomethyl phenyl seen in <b>5</b> was the preferred substituent at this position. Finally, we performed a simple scan of heteroatom and substituents on the benzamide ring (A-ring in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Although many combinations were evaluated, only the most potent compounds are included here in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>. We observed that incorporation of an electron withdrawing group such as cyano at R1 or a nitrogen at para to the carboxamide improved activity. These results were used to inform the design of subsequent analogues featured in <a class="ref internalNav" href="#tbl4" aria-label="Tables 4">Tables 4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Amide SAR</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0016.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0017.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Aryl SAR</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0018.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">LE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">16.8</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">14.5</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">8.9</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">cPro</td><td class="colsep0 rowsep0" align="char" char=".">13.3</td><td class="colsep0 rowsep0" align="char" char=".">0.22</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. A-Ring SAR</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0019.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">LE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">17.4</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="char" char=".">22.1</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="char" char=".">9.9</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="char" char=".">17.7</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure-Guided Design</h3><div class="NLM_p">The determination of the X-ray crystal structures of the five ITC validated substrate competitive chemical series in complex with SMYD2 illustrated how these inhibitors achieved enzymatic inhibition, which proved to be invaluable in informing the chemistry plan. In the following section, we discuss the structure-guided design, synthesis, and biological activity of our <i>hit-to lead</i> efforts based on compounds <b>1</b> and <b>5</b>.</div><div class="NLM_p">The crystal structure of SMYD2 in complex with <b>5</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) confirmed our initial hypothesis that the propyl pyrrolidine moiety would act as a methyl lysine mimetic (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf" class="ext-link">Table S1</a>). Structural overlay of <b>5</b> and the p53 substrate peptide show that the pyrrolidine moiety occupies the same region as the p53 peptide K370 terminus within the lysine binding channel.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overlay of the X-ray crystal structures of SMYD2 in complex with <b>5</b> (green) and a peptide derived from p53 (magenta). The bound inhibitor molecule and peptide are shown as stick models, and the protein backbone is presented as a white ribbon (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJL">5KJL</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We previously described the binding mode of <b>1</b><a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a). Compound <b>1</b> is composed of three distinct moieties: benzooxazinone, cyclohexyl, and dichlorophenethyl substituents. The benzooxazinone group is positioned deep within the lysine binding channel of SMYD2 and forms an intricate series of electrostatic and hydrophobic packing interactions with the donor methyl group of the cofactor SAM and main chain and side chain atoms of SMYD2. The ketone oxygen of the benzooxazinone moiety is placed 2.8 Å from the donor methyl of SAM. The hydroxyl substituent and the ring amine nitrogen jointly coordinate a water molecule, which mediates electrostatic interactions with the backbone carbonyl oxygens of residues V202 and M205. On the opposing side of the benzooxazinone ring, the ether oxygen is coordinated by residue Y258. A highly coordinated water molecule bridges additional interactions between the ether oxygen and the amine linker of <b>1</b> and the backbone carbonyls of residues N180, G183, and Y258. The cyclohexyl group of <b>1</b> is positioned in a primary hydrophobic pocket. The amine linker connecting the cyclohexyl and dichlorophenethyl elements of <b>1</b> forms a hydrogen bond with the side chain of residue N180.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Hybridization strategy. (a) Overlay of the X-ray crystal structures of SMYD2 in complex with <b>1</b> (light blue) and <b>5</b> (green). The bound inhibitor molecules are shown as stick models. SMYD2 is colored white. (b) Chemistry schematic describing the structure-based hybridization approach.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Perhaps the most interesting feature of the compound <b>1</b> binding mode is that the dichlorophenethyl moiety of the inhibitor extends across the peptide binding groove of SMYD2 and is inserted into a secondary hydrophobic pocket adjacent to the cofactor binding site. The secondary hydrophobic pocket is adjacent to the SAM binding site, and residues that form electrostatic interactions with the adenosyl group of SAM also make contacts with the dichlorophenethyl moiety of <b>1</b>. This observation suggests the possibility of cooperativity, as the presence of SAM may contribute to inhibitor binding.</div><div class="NLM_p">The overlay of compounds <b>1</b> and <b>5</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a) revealed opportunities for scaffold hybridization. The methyl group of <b>5</b> presents a convenient functional handle and is ideally situated to append a spacer to enable the placement of a hydrophobic moiety into the secondary hydrophobic pocket that is occupied by the dichlorophenyl moiety of compound <b>1</b>. To evaluate possible analogues, we enumerated a virtual library of compounds combining features of both <b>1</b> and <b>5</b>. Docking studies using GLIDE helped to select compounds that were likely to span the binding site. Compound <b>30</b> illustrates a representative hybrid (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b). The basic nitrogen of the piperazine moiety in the docked binding pose is positioned to form a hydrogen bond with residue N180 and would also present the phenethyl group in the appropriate direction to occupy the secondary hydrophobic binding pocket.</div><div class="NLM_p">We synthesized a variety of analogues including <b>30</b> and were pleased to find that the new compounds had improved enzymatic and cellular activity relative to the progenitor compounds. Indeed, <b>30</b> showed an IC<sub>50</sub> value of 65 nM with cellular activity that was superior to either compound <b>1</b> or <b>5</b>. In addition, determination of the crystal structure of SMYD2 with <b>30</b> confirmed the expected binding mode (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). However, this compound has a poor in vitro DMPK profile with a Log <i>D</i> value >3.5, huMic Cl<sub>int</sub> >300 and high plasma protein binding. To address these issues, we designed subsequent analogues with the intention of reducing polarity and metabolic clearance (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0007.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure of SMYD2 in complex with <b>30</b>. The bound inhibitor is colored magenta, and cofactor SAM is colored orange. All residues within 3.2 Å of the inhibitor are shown as sticks, and hydrogen bonds are shown as black dashed lines (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJM">5KJM</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The SAR from compounds <b>24</b>–<b>29</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) taken together with the crystal structure of SMYD2 in complex with <b>30</b>, suggest that the fused pyrazole ring in this series does not contribute significantly to binding affinity. We therefore, focused analogue design on a tricyclic ring system exemplified in <a class="ref internalNav" href="#tbl4" aria-label="Tables 4">Tables 4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR and In Vitro DMPK Values for Bis-Aryl Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0020.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">cell EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>int</sub> huMics<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">sol (μM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">hu PPB (% free)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3-Me-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.211</td><td class="colsep0 rowsep0" align="char" char=".">4.14</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">722</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3-MeO-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.273</td><td class="colsep0 rowsep0" align="char" char=".">3.50</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">815</td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-Cl-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.114</td><td class="colsep0 rowsep0" align="char" char=".">3.12</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">332</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.339</td><td class="colsep0 rowsep0" align="char" char=".">4.91</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">851</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3-(2-ethyl)-indole</td><td class="colsep0 rowsep0" align="char" char=".">0.041</td><td class="colsep0 rowsep0" align="char" char=".">3.46</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">246</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3,4-di-Cl-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.165</td><td class="colsep0 rowsep0" align="char" char=".">3.23</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char="."><1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">3-Me-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.031</td><td class="colsep0 rowsep0" align="char" char=".">1.21</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">588</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">3-MeO-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.059</td><td class="colsep0 rowsep0" align="char" char=".">2.69</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="char" char=".">602</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">3-(2-ethyl)-indole</td><td class="colsep0 rowsep0" align="char" char=".">0.028</td><td class="colsep0 rowsep0" align="char" char=".">0.270</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">396</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">3,4-di-Cl-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.138</td><td class="colsep0 rowsep0" align="char" char=".">0.425</td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char="."><1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Immunofluorescence assay measuring inhibition of SMYD2 mediated methylation of monomethyl p53 peptide in U2OS cells.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">In vitro clearance in human microsomes (μL/min/mg).</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">24 h solubility of solid sample determined in 0.1 M phosphate buffer (pH 7.4) at 25 °C.</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last">Fraction unbound (<i>f</i><sub>u</sub>%) in human plasma, determined from DMSO stock solution of compound by equilibrium dialysis.</p></div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Biochemical Potency and in Vitro DMPK Values for Pyridyl Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0021.gif" alt="" id="fx6" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">cell EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">sol (μM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">hu PPB (% free)<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3-MeO-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.440</td><td class="colsep0 rowsep0" align="char" char=".">1.22</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="char" char=".">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4-MeO-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">1.05</td><td class="colsep0 rowsep0" align="char" char=".">2.44</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="char" char=".">34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4-Cl-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.305</td><td class="colsep0 rowsep0" align="char" char=".">1.62</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="char" char=".">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3,4-di-Cl-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.120</td><td class="colsep0 rowsep0" align="char" char=".">5.57</td><td class="colsep0 rowsep0" align="char" char=".">790</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3-(2-ethyl)-indole</td><td class="colsep0 rowsep0" align="char" char=".">0.062</td><td class="colsep0 rowsep0" align="char" char=".">5.61</td><td class="colsep0 rowsep0" align="char" char=".">923</td><td class="colsep0 rowsep0" align="char" char=".">34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">3,4-di-Cl-phenethyl</td><td class="colsep0 rowsep0" align="char" char=".">0.038</td><td class="colsep0 rowsep0" align="char" char=".">0.450</td><td class="colsep0 rowsep0" align="char" char=".">393</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">3-(2-ethyl)-indole</td><td class="colsep0 rowsep0" align="char" char=".">0.017</td><td class="colsep0 rowsep0" align="char" char=".">1.02</td><td class="colsep0 rowsep0" align="char" char=".">858</td><td class="colsep0 rowsep0" align="char" char=".">17</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Immunofluorescence assay measuring inhibition of SMYD2 mediated methylation of monomethyl p53 peptide in U2OS cells.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">24 h solubility of solid sample determined in 0.1 M phosphate buffer (pH 7.4) at 25 °C.</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">Fraction unbound (<i>f</i><sub>u</sub>%) in human plasma, determined from DMSO stock solution of compound by equilibrium dialysis.</p></div></div></div><div class="NLM_p">In addition to the SAR findings summarized in <a class="ref internalNav" href="#tbl4" aria-label="Tables 4">Tables 4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>, we also examined how replacement of the piperazine linker might affect SAR. We prepared analogues with various alternatives to the piperazine and observed that some of these modifications were tolerated, although in general, they did not offer any advantage with respect to cellular activity or DMPK profile. The crystal structures of SMYD2 in complex with two of the more potent examples from this effort, compounds <b>47</b> and <b>48</b>, were determined (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf" class="ext-link">Table S1</a>). The superposition of SMYD2 in complex with the indole compound <b>47</b> with <b>48</b> (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) showed that the binding mode is shared with the piperazine analogues.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0008.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structural superposition. (a) Overlay of the X-ray crystal structures of SMYD2 in complex with <b>48</b> (white) and <b>47</b> (green) illustrating alternate linkers. The bound inhibitors are shown as stick models, and the binding groove of SMYD2 is shown as a surface and has been colored by electrostatic potential (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJN">5KJN</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJK">5KJK</a>). (b) Chemical structures of compounds <b>48</b> and <b>49</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> In Vivo Pharmacokinetics</h3><div class="NLM_p">Two compounds, <b>39</b> and <b>40</b>, with good cellular activity and reasonable in vitro DMPK profiles, were selected for in vivo pharmacokinetic studies. The results from this study are presented in <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>. Although they are dibasic, both compounds have good oral bioavailability (<i>F</i> = 18% and 81%, respectively). Compound <b>39</b> has a relatively higher clearance and a correspondingly shorter half-life. The half-life of <b>40</b> is estimated to be 28 h based on extrapolation from the final time point at 24 h.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Rat Pharmacokinetics of Compounds <b>39</b> and <b>40</b><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th><th class="colsep0 rowsep0" align="center" char=".">Cl (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">Vd<sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="char" char=".">81</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">>24</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">Rat pharmacokinetics in Han Wistar rats followed a dose of 14 μM/kg (po) and 4 μM/kg (iv).</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Selectivity</h3><div class="NLM_p last">The selectivity of a set of compounds was evaluated against a focused panel of KMTs with previously defined selective tool compounds: EZH2, NSD1, G9a SMYD2, and DOT1L. The data shows that these hybrid compounds are selective for SMYD2 but do possess limited activity against DOT1L at high inhibitor concentrations (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf" class="ext-link">Table S2</a>).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cell Data</h3><div class="NLM_p">To verify that compounds inhibited SMYD2 activity in cells, a polyclonal antibody against a p53 peptide containing the monomethylated K370 epitope<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> was used in an immunofluorescence assay as described previously.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> U2OS cells transfected with a SMYD2 expression vector were treated with compound for 24 h and a reduction in the SMYD2-mediated methylation signal was observed (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>). Staining with the anti-FLAG tag antibody was used to control for loss of cell number during compound treatment. Quantification of fluorescent intensity per cell was determined, and the IC<sub>50</sub> values for these compounds are shown in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>. There was at least a >2-fold shift in the IC<sub>50</sub> values relative to the direct effect on SMYD2 methylation. These results demonstrate that these compounds do inhibit SMYD2 methyltransferase activity in cells, leading to a decrease in the SMYD2-mediated methylation signal.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Inhibition of SMYD2-Mediated Methylation<a class="ref internalNav" href="#tbl7-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">p53-me cell assay IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">cell count IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">9.98</td><td class="colsep0 rowsep0" align="char" char=".">23.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="char" char=".">0.67</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="char" char=".">2.43</td><td class="colsep0 rowsep0" align="char" char=".">13.26</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl7-fn1"><div class="footnote" id="tbl7-fn1"><sup>a</sup><p class="last">U2OS cells were transfected with a SMYD2 expression vector and treated with compound for 24 h. The calculated IC<sub>50</sub> values for each compound are based on fluorescent intensity per cell. DAPI staining was used to calculate the cell count IC<sub>50</sub> value.</p></div></div></div><div class="NLM_p">To evaluate if the SMYD2 inhibitors affect tumor cell viability in a p53-dependent manner, IC<sub>50</sub> values were determined using a panel of matched paired p53 wild-type and p53-null colorectal cancer cell lines (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). The parental HCT116 cell line (+/+) contains a wild-type p53 gene, whereas the <i>TP53</i> gene has been removed through homologous recombination in the isogenic derivative HCT 116 (−/−) cell line.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The colorectal cancer cell line RKO-E6a expresses a human papillomavirus E6 gene that binds p53 and targets it for degradation.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The RKO-neo cell line is the empty vector control pair to the RKO-E6a line. There was not an appreciable difference in the sensitivity of the p53 wild-type or p53-deficient cell lines to the SMYD2 inhibitors evaluated (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). There was a strong correlation between the IC<sub>50</sub> values obtained from the 24 h p53 methylation and cell viability assays across all cell lines (either p53 wild-type or p53-deficient cells), suggesting that the effects on viability are not due to SMYD2-mediated modulation of p53 activity by the inhibitors. These results do not support the hypothesis that SMYD2 inhibitors work in a p53-dependent manner to affect cell proliferation under the conditions evaluated. However, a recent study has shown that SMYD2 suppresses p53-dependent cardiomyocyte apoptosis, indicating that SMYD2 functions as a cardioprotective protein by methylating p53 in cardiomyocytes.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0009.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of the indicated compounds on cell proliferation is not p53-dependent. HCT116+/+ and RKO-neo represent p53 wild-type cells; HCT116–/– and RKO-E6a represent p53-deficient cells. Cells were treated with the indicated compounds (<b>1</b>, <b>40</b>, and <b>47</b>; listed in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) between 0.03 and 30 μM for 72 h, and cell viability was determined using Alamar Blue. No significant differences in IC<sub>50</sub> values were observed between p53 wild-type and p53-deficient cell lines. The assay was performed in triplicate in two separate experiments. Values shown are mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry</h3><div class="NLM_p">The synthesis of the pyrazolopyridine <b>5</b> and related analogues is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The cyclocondensation reaction of an appropriately substituted benzaldehyde with pyruvic acid and 5-aminopyrazoles readily afforded the 6-aryl-pyrazolopyridine-4-carboxylic acid<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> intermediates <b>50a</b>–<b>h</b>. Intermediate <b>50a</b> was coupled with a variety of amines to give compounds <b>9</b>–<b>16</b> found in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. Compounds <b>17</b>–<b>23</b> in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> were similarly prepared from the acids <b>50b</b>–<b>h</b> by coupling with 3-(pyrrolidin-1-yl)propan-1-amine. Although the initial cyclocondensation step is low yielding, the simplicity of the reaction sequence allowed for rapid preparation of analogues to investigate the SAR of this series.</div><figure id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrazolopyridine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) pyruvic acid, AcOH, reflux, 8–24%; (b) amine, HATU or EDC, DIPEA, DMF, 50 °C, 24–90%. See <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> for amines for <b>9</b>–<b>16</b>.</p></p></figure><div class="NLM_p">Entry into the truncated analogues lacking the fused pyrazole ring is described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Amide formation starting with commercially available 3-bromobenzoic acids <b>51a</b>–<b>c</b> or isonicotinic acids <b>51d</b>–<b>f</b> proceeded in good yield to give carboxamides <b>52a</b>–<b>f</b>. Palladium mediated cross-coupling with <i>o</i>-tolylboronic acid gave rise to final compounds <b>24</b>–<b>29</b> in the range of yields indicated.</div><figure id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Truncated Biaryl Amides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, TEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 80–84%; (b) <i>o</i>-tolylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O, 80 °C, 14–84%.</p></p></figure><div class="NLM_p">The preparation of the extended pyrazolopyridine <b>30</b> is outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The requisite benzaldehyde <b>54</b> was prepared in two steps starting from commercially available <b>53a</b>. Boc-deprotection with HCl in dioxane afforded the piperazine <b>53b</b>, which was subsequently alkylated with 3,4-dichlorophenethylmethanesulfonate.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> With <b>54</b> in hand, the stage was set to perform the cyclocondensation reaction to afford the elaborated pyrazolopyridine-4-carboxylic acid <b>55</b> in moderate yield. The propyl pyrrolidine moiety was introduced via amide coupling to afford the desired compound <b>30</b>. Although the amide coupling occurred with good conversion to the carboxamide, material was lost under purification, resulting in the poor overall recovery.</div><figure id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0012.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Extended Pyrazolopyridine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) HCl in dioxane, rt, quant; (b) K<sub>2</sub>CO<sub>3</sub>, ACN, 70 °C, 65%; (c) 1<i>H</i>-pyrazol-3-amine, pyruvic acid, AcOH, reflux, 24%; (d) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, DMF, 60 °C, 5%.</p></p></figure><div class="NLM_p">Biphenylpiperazine analogues <b>31</b>–<b>36</b> were synthesized according to the route outlined in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The aryl piperazine <b>56</b> was Boc-protected to generate <b>57</b>, which was then subject to a Suzuki coupling reaction with the 3-(ethoxycarbonyl)phenylboronic acid, giving rise to <b>58a</b> in high yield over the two steps. The acid <b>58b</b> was obtained by ester hydrolysis under basic conditions, and the product was subjected to amide coupling to produce <b>59</b>. Removal of the Boc group provided piperazine <b>60</b>, which could be alkylated under a variety of conditions (f or g in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) to afford the final compounds <b>31</b>–<b>36</b>.</div><figure id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0013.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Biphenylpiperazines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Boc<sub>2</sub>O, TEA, Et<sub>2</sub>O/THF, rt, 98%; (b) 3-(ethoxycarbonyl)phenyl-boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 98%; (c) LiOH, MeOH/THF/H<sub>2</sub>O, rt, 92%; (d) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 91%; (e) HCl in dioxane, MeOH, rt, 90%; (f) R-Br, TEA, ACN, reflux, 31–50%; (g) 3,4-dichlorophenethylmethanesulfonate, K<sub>2</sub>CO<sub>3</sub>, ACN, 72%.</p></p></figure><div class="NLM_p">The 3-cyano-biphenylpiperazine derivatives were synthesized as shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Palladium mediate cross-coupling of aryl bromide <b>52b</b> (previously described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) with 2-[4-(<i>N</i>-Boc)piperazin-1-yl]phenylboronic acid pinacol ester generated <b>62a</b> in high yield. Subsequent Boc-deprotection with TFA in DCM gave piperazine intermediate <b>62b</b>. This was then treated with the appropriate alkyl bromides in the presence of potassium carbonate in acetonitrile at high temperatures under microwave heating or reflux conditions to afford the final compounds <b>37</b>–<b>39</b>. Likewise, alkylation with 3,4-dichlorophenethylmethanesulfonate in the presence of sodium iodide gave rise to <b>40</b>. The 2-aryl isonicotinic acid derivatives <b>41</b>–<b>45</b> were prepared in the same fashion starting from the pyridyl bromide <b>52d</b>.</div><figure id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0014.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 3-Cyanophenyl and Pyridyl Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>52b</b> or <b>52d</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O, 70 °C, 86%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, quant; (c) R-Br, K<sub>2</sub>CO<sub>3</sub>, ACN, microwave 100–150 °C, 36–42%; (d) R-Br 3-(2-bromoethyl)-1<i>H</i>-indole, K<sub>2</sub>CO<sub>3</sub>, ACN, reflux, 18–36%; (e) 3,4-dichlorophenethylmethanesulfonate, NaI, K<sub>2</sub>CO<sub>3</sub>, ACN, 70 °C, 69%.</p></p></figure><div class="NLM_p">The 3,5-disubstituted pyridyl analogues <b>46</b> and <b>47</b> were synthesized as described in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. Palladium mediated cross-coupling between 3-(ethoxycarbonyl)pyridine-5-boronic acid pinacol ester <b>64</b> and piperazinyl aryl bromide <b>57</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) gave the pyridyl ester <b>65</b>. Direct amidation of the ester facilitated by catalytic TBD<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> in hot toluene gave the amide <b>66a</b> in high yield, followed by Boc-deprotection to furnish <b>66b</b>. Alternatively, <b>66a</b> could be obtained starting from 5-bromonicotinic acid <b>67</b> via amide coupling to afford <b>68</b> and subsequent Suzuki coupling with boronate ester <b>61</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). Alkylation of the piperazine in a manner previously described was then performed to obtain <b>46</b> and <b>47</b>.</div><figure id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0015.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of 3,5-Disubstituted-Pyridyl A-Ring Biarylpiperazines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>57</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 75 °C, 69%; (b) 3-(pyrrolidin-1-yl)propan-1-amine, TBD, toluene, 90 °C, 94%; (c) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, TEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 85%; (d) <b>61</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O, 75 °C, 99%; (e) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, quant; (f) mesylate, K<sub>2</sub>CO<sub>3</sub>, ACN, 70 °C, 46%; (g) 3-(2-bromoethyl)-1<i>H</i>-indole, TEA, ACN, 75 °C, 20%.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52752" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52752" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our lead generation efforts have led to the discovery of five novel chemical series of SMYD2 inhibitors. Our original disclosure of compounds <b>1</b><a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and <b>39</b>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> representing the first potent and selective SMYD2 inhibitors, paved the way for further studies,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> leading to the development of <b>2</b> (LLY-507)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (an analogue of <b>39</b>), <b>3</b> (A-893) (an analogue of <b>1</b>),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and <b>4</b> (BAY-598).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">In this study, we used structure-guided design and medicinal chemistry to understand the SAR in the lysine binding channel and the selectivity pockets of the substrate binding groove of SMYD2. Importantly, we utilized scaffold hybridization techniques to combine the attractive features from two distinct chemical scaffolds to improve biochemical and cellular potency as well as the pharmocokinetic profile of the resulting piperazine-based hybrid molecules. X-ray crystallography was used to demonstrate that these compounds form optimal interactions in the lysine binding channel and form a favorable series of hydrophobic interactions in both selectivity pockets of SMYD2.</div><div class="NLM_p last">Despite having highly potent biochemical inhibitors with attractive physicochemical properties and cellular activity, our data does not support the hypothesis that SMYD2 inhibitors work in a p53-dependent manner to affect cell proliferation. These findings agree with recent in vivo xenograft and DMPK data on <b>4</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> which failed also to show a direct effect on p53-dependent cell proliferation.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The utility and value of SMYD2 inhibitors to understand the complex cell biology of SMYD2 is best exemplified by the recent findings of Reynoird et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In this study, it was found that levels of SMYD2 are elevated in pancreatic ductal adenocarcinoma (PDAC), indicating that SMYD2 promotes pancreatic cancer. Pharmalogical inhibition of SMYD2 was shown to restrict PDAC proliferation. Moreover, inhibition of SMYD2 enhances the efficacy when used in combination with standard chemotherapy to treat PDAC.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Experimental</h3><div class="NLM_p">All reagents and solvents used were purchased from commercial sources and used without further purification (the 3-(pyrrolidin-1-yl)propan-1-amine was purchased from Acros; 3-(ethoxycarbonyl)pyridine-5-boronic acid pinacol ester was purchased from Frontier Scientific; 5-bromo-2-fluoronicotinic acid was purchased from Ark Pharma; 5-bromo-2-fluorobenzoic acid was purchased from Combi-Blocks; 2-chloro-6-methoxyisonicotinic acid was purchased from Maybridge. All other solvents and chemicals were purchased from Sigma-Aldrich). All reactions were performed under a nitrogen or argon atmosphere unless otherwise noted. <sup>1</sup>H NMR spectra were obtained using a Bruker 300 or 400 MHz spectrometer at 27 °C unless otherwise noted; chemical shifts are expressed in parts per million (ppm, δ units) and are referenced to the residual protons in the deuterated solvent used. Coupling constants are given in units of hertz (Hz). Splitting patterns describe apparent multiplicities and are designated as s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), and br s (broad singlet). Mass spectrometry analyses were performed with a Waters ZQ mass spectrometer connected to an Agilent 1100 equipped with Waters (Atlantis T3, 2.1 mm × 50 mm, 3 μm, or Atlantis C18, 2.1 mm × 50 mm, 5 μm) or Varian (Polaris C18A, 2 mm × 50 mm, 3 μm) columns eluted with a gradient mixture of water and acetonitrile with either formic acid or ammonium acetate added as a modifier: overall run length correspond to LCMS procedure A = 5–95% of acetonitrile over 4.5 min, LCMS procedure B = 5–95% of acetonitrile over 2.5 min. High resolution mass spectra (HRMS) were recorded on a Bruker Daltonics microTOF-Q II hybrid quadrupole time-of-flight mass spectrometer using positive electrospray ionization. Reverse-phase chromatography was performed on a Gilson system using an Atlantis Prep T3 OBD, Xbridge C18 or Phenyl OBD column: reverse-phase HPLC column (19 mm × 100 mm) in water/acetonitrile with 0.1% trifluoroacetic acid as mobile phase unless otherwise noted. The reported molecular ion corresponds to the [M + H]<sup>+</sup>; for molecules with multiple isotopic patterns (Br, Cl, etc.), the reported value is the one obtained for the lowest isotope mass unless otherwise specified. Thin layer chromatography (TLC) was performed using EMD silica gel 60 F<sub>254</sub> plates, which were visualized using either UV light, reversibly stained with iodine (I<sub>2</sub> absorbed on silica) or a stain prepared by dissolving 2 g of KMnO<sub>4</sub> and 12 g of Na<sub>2</sub>CO<sub>3</sub> in 200 mL of H<sub>2</sub>O. Column chromatography was performed using SiliCycle SiliaSep or RediSep Rf preloaded silica gel cartridges on Teledyne ISCO CombiFlash Companion automated purification systems. Unless otherwise indicated, all final compounds were purified to ≥95% purity as assessed by analytical HPLC using an Agilent 1100 equipped with Waters columns (Atlantis T3, 2.1 mm × 50 mm, 3 μm, or Atlantis C18, 2.1 mm × 50 mm, 5 μm) eluted for >10 min with a gradient mixture of water and acetonitrile with either formic acid or ammonium acetate added as a modifier, monitored at wavelengths of 220, 254, and 280 nm.</div><div class="NLM_p">Compounds are listed in the order in which they appear in the tables or chemistry schemes.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(3-(Pyrrolidin-1-yl)propyl)-6-(<i>o</i>-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>5</b>)</h4><div class="NLM_p">A stirred mixture of the 6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxylic acid <b>50a</b> (0.143 g, 0.56 mmol), DIPEA (0.394 mL, 2.26 mmol), 3-(pyrrolidin-1-yl)propan-1-amine (0.145 g, 1.13 mmol), and HATU (0.537 g, 1.41 mmol) in DMF (2 mL) was warmed to 50 °C for 48 h. The reaction was cooled to rt, diluted with water (100 mL), and extracted with ethyl acetate (2 × 100 mL). The combined organic extracts were washed with brine, dried with MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by reverse-phase HPLC (gradient elution: 12–70% ACN in water containing 0.1% trifluoroacetic acid) to afford title compound after lyophilization as a white solid (0.155 g, 58%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.64–2.14 (m, 6H), 2.21–2.44 (m, 3H), 2.82–3.09 (m, 2H), 3.09–3.28 (m, 2H), 3.43 (q, <i>J</i> = 6.4 Hz, 2H), 3.57 (dd, <i>J</i> = 10.6, 5.3 Hz, 2H), 7.13–7.43 (m, 3H), 7.52 (d, <i>J</i> = 7.1 Hz, 1H), 7.62–7.87 (m, 1H), 8.43 (s, 1H), 8.99 (t, <i>J</i> = 5.6 Hz, 1H), 9.27–9.84 (m, 1H), 13.88 (br, s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 364.2132; found, 364.2143.</div><div class="NLM_p last">The following compounds were prepared in an analogous manner to compound <b>5</b> using the appropriate amine.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(3-(Dimethylamino)propyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>9</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.87–1.98 (m, 2H), 2.37 (s, 3H), 2.78 (d, <i>J</i> = 4.8 Hz, 6H), 3.07–3.17 (m, 2H), 3.40 (q, <i>J</i> = 6.4 Hz, 2H), 7.31–7.41 (m, 3H), 7.50 (d, <i>J</i> = 7.1 Hz, 1H), 7.70 (s, 1H), 8.41 (s, 1H), 8.98 (t, <i>J</i> = 5.81 Hz, 1H), 9.30–9.44 (m, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 338.2. <i>t</i><sub>R</sub> = 1.53 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>24</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 338.1975; found, 338.1989.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-(3-(Methylamino)propyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide hydrochloride (<b>10</b>)</h4><div class="NLM_p last">A vial was charged with <i>tert</i>-butyl methyl(3-(6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamido)propyl)carbamate (85 mg, 0.20 mmol) and 0.5 M HCl in MeOH (5 mL). The reaction was stirred at room temperature for 2 h then concentrated in vacuo to afford the hydrochloride salt of the title compound as white solid (69 mg, 96%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.92 (quin, <i>J</i> = 7.1 Hz, 2H), 2.38 (s, 3H), 2.50–2.55 (m, 3H), 2.87–3.01 (m, 2H), 3.42 (q, <i>J</i> = 6.5 Hz, 2H), 7.30–7.41 (m, 3H), 7.53 (d, <i>J</i> = 7.1 Hz, 1H), 7.77 (s, 1H), 8.42 (s, 1H), 8.97 (br s, 2H), 9.13 (t, <i>J</i> = 5.7 Hz, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 324.2. <i>t</i><sub>R</sub> = 1.48 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>22</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 324.1819; found, 324.1831.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(3-Morpholinopropyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>11</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.85–1.96 (m, 2H), 2.31 (s, 3H), 2.93–3.06 (m, 2H), 3.12 (dd, <i>J</i> = 10.5, 5.9 Hz, 2H), 3.32–3.42 (m, 4H), 3.57 (t, <i>J</i> = 11.9 Hz, 2H), 3.91 (d, <i>J</i> = 11.6 Hz, 2H), 7.25–7.37 (m, 3H), 7.44 (d, <i>J</i> = 7.1 Hz, 1H), 7.64 (s, 1H), 8.35 (s, 1H), 8.93 (t, <i>J</i> = 5.8 Hz, 1H), 9.57 (br s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 380.1. <i>t</i><sub>R</sub> = 1.54 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 380.2081; found, 380.2089.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i>-(3-(Piperidin-1-yl)propyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>12</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.52–1.72 (m, 4H), 1.75–1.87 (m, 2H), 1.89–2.01 (m, 2H), 2.37 (s, 3H), 2.79–2.95 (m, 2H), 3.04–3.17 (m, 2H), 3.35–3.51 (m, 4H), 7.33–7.39 (m, 3H), 7.47–7.53 (m, 1H), 7.69 (s, 1H), 8.41 (s, 1H), 8.92–9.05 (m, 2H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 378.2. <i>t</i><sub>R</sub> = 1.64 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 378.2288; found, 378.2285.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(2-Hydroxy-3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>13</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.79–1.90 (m, 2H), 1.93–2.03 (m, 2H), 2.37 (s, 3 H), 2.96–3.09 (m, 2H), 3.11–3.20 (m, 1H), 3.23–3.32 (m, 1H), 3.40 (q, <i>J</i> = 5.8 Hz, 2H), 3.47–3.60 (m, 2H), 4.00–4.11 (m, 1H), 5.95 (br s, 1H), 7.33–7.42 (m, 3H), 7.51 (d, <i>J</i> = 7.1 Hz, 1H), 7.75 (s, 1H), 8.41 (s, 1H), 9.00 (t, <i>J</i> = 5.8 Hz, 1H), 9.46 (br s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 380.2. <i>t</i><sub>R</sub> = 1.54 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 380.2081; found, 380.2094.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(2-Benzyl-3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>14</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.85–2.02 (m, 4H), 2.38 (s, 3H), 2.39–2.44 (m, 1H), 2.64–2.84 (m, 2H), 2.93 (br d, <i>J</i> = 7.0 Hz, 1H), 2.97–3.13 (m, 2H), 3.15–3.30 (m, 1H), 3.36–3.47 (m, 2H), 3.68 (d, <i>J</i> = 4.5 Hz, 2H), 7.20 (app dq, <i>J</i> = 8.5, 4.3 Hz, 1H), 7.30 (d, <i>J</i> = 4.3 Hz, 4H), 7.33–7.40 (m, 3H), 7.47–7.55 (m, 1H), 7.70 (s, 1H), 8.40 (s, 1H), 9.00 (t, <i>J</i> = 5.7 Hz, 1H), 9.44–9.62 (m, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 454.1. <i>t</i><sub>R</sub> = 2.11 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>32</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 454.2601; found, 454.2611.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1,4′-Bipiperidin-1′-yl(6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridin-4-yl)methanone (<b>15</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.22–1.43 (m, 1H), 1.44–1.69 (m, 5H), 1.70–1.89 (m, 3H), 2.00–2.14 (m, 1H), 2.32 (s, 3H), 2.75–2.96 (m, 3H), 3.00–3.18 (m, 1H), 3.23–3.64 (m, 4H), 4.58–4.74 (m, 1H), 7.21–7.35 (m, 4H), 7.42 (d, <i>J</i> = 7.0 Hz, 1H), 8.11 (s, 1H), 9.06 (br s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 404.3. <i>t</i><sub>R</sub> = 1.62 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>30</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 404.2445; found, 404.2462.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-Methyl-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolyl-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>16</b>)</h4><div class="NLM_p"><sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 2.06–2.28 (m, 6H), 2.41 (s, 3H), 3.08 (s, 3H), 3.13 (m, 2H), 3.32–3.41 (m, 2H), 3.70–3.82 (m, 4H), 7.33–7.49 (m, 4H), 7.61 (s, 1H), 8.52 (s, 1H); NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M+H)<sup>+</sup> = 378.2. <i>t</i><sub>R</sub> = 1.54 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 378.2288; found, 378.2292.</div><div class="NLM_p last">The following compounds were prepared in an analogous manner to compound <b>5</b> using the appropriate benzaldehyde.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 6-(2-Bromophenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>17</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.69–2.09 (m, 6H), 2.87–3.07 (m, 2H), 3.15–3.30 (m, 2H), 3.42 (q, <i>J</i> = 6.3 Hz, 2H), 3.50–3.68 (m, 2H), 7.06–7.49 (m, 1H), 7.54–7.66 (m, 2H), 7.74–7.92 (m, 2H), 8.22–8.60 (m, 1H), 8.97 (t, <i>J</i> = 5.6 Hz, 1H), 9.65 (br s, 1H), 13.99 (br s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>BrN<sub>5</sub>O (M + H)<sup>+</sup>, 428.1080; found, 428.1091.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-(2-Chlorophenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>18</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.55–2.08 (m, 6H), 2.81–3.01 (m, 2H), 3.06–3.19 (m, 2H), 3.36 (q, <i>J</i> = 6.3 Hz, 2H), 3.41–3.57 (m, 2H), 7.26–7.51 (m, 2H), 7.51–7.69 (m, 2H), 7.72 (s, 1H), 7.77 (s, 1H), 7.68–7.89 (m, 1H), 8.38 (s, 1H), 8.91(t, <i>J</i> = 5.9 Hz, 1H), 9.49 (br s, 1H), 13.99 (br s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>5</sub>O (M + H)<sup>+</sup>, 384.1586; found, 364.1593.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 6-(2-Fluorophenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>19</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.67–2.23 (m, 6H), 2.82–3.11 (m, 2H), 3.11–3.31 (m, 2H), 3.44 (q, <i>J</i> = 6.3 Hz, 2H), 3.50–3.67 (m, 2H), 7.22–7.49 (m, 2H), 7.49–7.67 (m, 2H), 7.82–8.06 (m, 2H), 8.43 (s, 1H), 8.85–9.11 (m, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>FN<sub>5</sub>O (M + H)<sup>+</sup>, 368.1881; found, 368.1889.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 6-(2-Ethylphenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>20</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.05 (t, <i>J</i> = 7.4 Hz, 3H), 1.72–2.22 (m, 6H), 2.67–2.83 (m, 2H), 2.90–3.05 (m, 2H), 3.18–3.25 (m, 2H), 3.42 (q, <i>J</i> = 7.4 Hz, 2H), 3.56 (dd, <i>J</i> = 10.4, 5.3 Hz, 2H), 7.04–7.54 (m, 4H), 7.64–7.82 (m, 1H), 8.22–8.58 (m, 1H), 9.00 (t, <i>J</i> = 5.7 Hz, 1H), 9.74 (br s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 378.2288; found, 378.2287.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 6-(2-Methoxyphenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>21</b>)</h4><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.75 (m, 4H), 1.80–1.90 (m, 2H), 2.76–2.86 (m, 6H), 3.39 (q, <i>J</i> = 6.7 Hz, 2H), 3.82 (s, 3H), 7.10 (td, <i>J</i> = 7.5, 1.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.3 Hz, 1H), 7.47 (ddd, <i>J</i> = 8.3, 7.4, 1.8 Hz, 1H), 7.65 (dd, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.87 (s, 1H), 8.34 (s, 1H), 8.82–8.90 (m, 1H), 13.82 (br s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 380.2081; found, 380.2085.</div><div class="NLM_p last">The following compounds were prepared in an analogous manner to compound <b>5</b> using the appropriate pyrazoles.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 3-Methyl-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-(<i>o</i>-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>22</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.70–2.00 (m, 6H), 2.31 (s, 3H), 2.42 (s, 3H), 2.81–3.01 (m, 2H), 3.03–3.21 (m, 2H), 3.33 (q, <i>J</i> = 6.6 Hz, 2H), 3.50 (dd, <i>J</i> = 10.4, 5.1 Hz, 2H), 7.06–7.35 (m, 4H), 7.42 (d, <i>J</i> = 7.1 Hz, 1H), 8.80 (t, <i>J</i> = 5.7 Hz, 1H), 9.68 (br s, 1 H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 378.2288; found, 378.2299.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 3-Cyclopropyl-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-(<i>o</i>-tolyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>23</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 0.71–0.95 (m, 4H), 1.71–2.01 (m, 6H), 2.16–2.37 (m, 4H), 2.79–3.02 (m, 2H), 3.03–3.23 (m, 2H), 3.34 (q, <i>J</i> = 6.6 Hz, 2H), 3.49 (dd, <i>J</i> = 10.4, 5.3 Hz, 2H), 7.07–7.35 (m, 4H), 7.42 (d, <i>J</i> = 7.3 Hz, 1H), 8.82 (t, <i>J</i> = 5.7 Hz, 1H), 9.81 (br s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>30</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 404.2445; found, 404.2454.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(3-(Pyrrolidin-1-yl)propyl)-2-(<i>o</i>-tolyl)isonicotinamide (<b>24</b>)</h4><div class="NLM_p last">A 50 mL vial was charged with the 2-bromo-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide <b>52a</b> (280 mg, 0.90 mmol), <i>o</i>-tolylboronic acid (183 mg, 1.35 mmol), K<sub>2</sub>CO<sub>3</sub> (372 mg, 2.69 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (155 mg, 0.13 mmol) and a mixture of dioxane/water (10:1, 3 mL). The vial was purged and placed under argon then the stirred mixture was heated to 90 °C for 24 h. The reaction was diluted with water (50 mL) and extracted with EtOAc (2 × 50 mL). The combined organic extract was dried with MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by reverse-phase HPLC (gradient elution: 0–65% ACN in water containing 0.1% trifluoroacetic acid) to provide the trifluoroacetate of the title compound as a white solid (55 mg, 14%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.79–1.91 (m, 4H), 1.92–2.05 (m, 2H), 2.18 (s, 3H), 2.88–3.00 (m, 2H), 3.07–3.17 (m, 2H), 3.32 (t, <i>J</i> = 6.6 Hz, 2H), 3.44–3.56 (m, 2H), 3.83 (m, 1H), 7.13–7.22 (m, 1H), 7.22–7.31 (m, 3H), 7.44–7.49 (m, 1H), 7.50–7.58 (m, 1H), 7.73 (s, 1H), 7.76–7.84 (m, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 323.3. <i>t</i><sub>R</sub> = 2.07 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>27</sub>N<sub>2</sub>O (M + H)<sup>+</sup>, 323.2118; found, 323.2134.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 5-Cyano-2′-methyl-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>25</b>)</h4><div class="NLM_p last">A solution of 3-bromo-5-cyano-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)benzamide <b>52b</b> (477 mg, 1.42 mmol), <i>o</i>-tolylboronic acid (251 mg, 1.85 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (164 mg, 0.14 mmol), and K<sub>2</sub>CO<sub>3</sub> (589 mg, 4.26 mmol) in DME/water (10:1, 11 mL) was at 80 °C for 3 h. The reaction mixture was concentrated in vacuo and the residue purified directly by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc). The desired fractions were collected and concentrated in vacuo. The residue obtained was then treated with 0.5 M HCl in MeOH and concentrated under reduced pressure. Drying on high vacuum overnight afforded the hydrochloride of the title compound as a white solid (389 mg, 71%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.99–2.21 (m, 6H), 2.26 (s, 3H), 3.00–3.17 (m, 2H), 3.27 (d, <i>J</i> = 8.1 Hz, 2H), 3.53 (t, <i>J</i> = 6.6 Hz, 2H), 3.60–3.78 (m, 2H), 7.17–7.41 (m, 4H), 7.88 (d, <i>J</i> = 1.5 Hz, 1H), 8.09 (t, <i>J</i> = 1.4 Hz, 1H), 8.21 (t, <i>J</i> = 1.3 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 348.1. <i>t</i><sub>R</sub> = 2.08 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>26</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 348.2070; found, 348.2082.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 5-Methoxy-2′-methyl-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>26</b>)</h4><div class="NLM_p last">A solution of 3-bromo-5-methoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)benzamide <b>52c</b> (200 mg, 0.59 mmol), <i>o</i>-tolylboronic acid (104 mg, 0.76 mmol), K<sub>2</sub>CO<sub>3</sub> (243 mg, 1.76 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (67.7 mg, 0.06 mmol) in a mixture of DME/water (10:1, 6 mL) was allowed to stir for 3 h under an argon atmosphere at 80 °C. The reaction mixture was cooled then concentrated under reduced pressure. The crude product was purified by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc). The desired fractions were collected and concentrated under reduced pressure, and the product residue was then treated with 0.5 M HCl in MeOH and concentrated to dryness. The residue was treated with water and subject to lyophilization to afford the hydrochloride salt of the title compound as a tacky solid (50 mg, 22%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 2.06 (m, 6H), 2.24 (s, 3H), 3.23–3.27 (m, 2H), 3.28–3.30 (m, 4H), 3.50 (t, <i>J</i> = 6.4 Hz, 2H), 3.87 (s, 3H), 7.03 (dd, <i>J</i> = 2.4, 1.4 Hz, 1H), 7.17–7.25 (m, 2H), 7.25–7.29 (m, 2H), 7.35 (t, <i>J</i> = 1.4 Hz, 1H), 7.39–7.42 (m, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 353.3. <i>t</i><sub>R</sub> = 2.13 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>29</sub>N<sub>2</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 353.2224; found, 353.2237.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-(3-(Pyrrolidin-1-yl)propyl)-2-(<i>o</i>-tolyl)isonicotinamide (<b>27</b>)</h4><div class="NLM_p last">A 50 mL vial was charged with the 2-bromo-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide <b>52d</b> (280 mg, 0.90 mmol), <i>o</i>-tolylboronic acid (183 mg, 1.35 mmol), K<sub>2</sub>CO<sub>3</sub> (372 mg, 2.69 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (155 mg, 0.13 mmol) and a mixture of dioxane/water (10:1, 3 mL). The vial was purged and placed under an argon atmosphere, and the stirred mixture was heated to 90 °C for 24 h. The reaction was diluted with water (50 mL) and extracted with ethyl acetate (2 × 50 mL). The combined organic extract was dried with MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by reverse-phase HPLC (gradient elution: 0–65% ACN in water containing 0.1% trifluoroacetic acid) to provide the trifluoroacetate of the title compound as a white solid (55 mg, 14%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.97 (br m, 6H), 2.34 (br s, 3H), 2.95 (m, 2H), 3.16 (m, 2H), 3.39 (m, 2H), 3.50 (m, 2H), 7.19–7.59 (m, 4H), 7.90 (br s, 1H), 8.03 (br s, 1H), 8.86 (m, 1H), 9.18 (s., 1H), 10.75 (m, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 324.2070; found, 324.2083</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 2-Methoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolylisonicotinamide (<b>28</b>)</h4><div class="NLM_p last">A solution of 2-chloro-6-methoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide <b>52e</b> (353 mg, 1.19 mmol), <i>o</i>-tolylboronic acid (210 mg, 1.54 mmol), K<sub>2</sub>CO<sub>3</sub> (492 mg, 3.56 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (137 mg, 0.22 mmol) in a mixture of DME/water (10:1, 5.5 mL) was stirred at 80 °C for 3 h. The reaction mixture was concentrated in vacuo and purified directly by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc). The desired fractions were collected and concentrated in vacuo. The residue obtained was then treated with 0.5 M HCl in MeOH and concentrated under reduced pressure. Drying on high vacuum overnight afforded the hydrochloride of the title compound as a white solid (388 mg, 84%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.98–2.27 (m, 6H), 2.40 (s, 3H), 2.97–3.18 (m, 2H), 3.26 (d, <i>J</i> = 7.7 Hz, 2H), 3.52 (t, <i>J</i> = 6.6 Hz, 2H), 3.60–3.79 (m, 2H), 3.89–4.08 (m, 3H), 7.17 (d, <i>J</i> = 1.3 Hz, 1H), 7.24–7.38 (m, 3H), 7.39–7.52 (m, 2 H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 354.3. <i>t</i><sub>R</sub> = 2.07 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>28</sub>N<sub>3</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 345.2176; found, 345.2177.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 2-(Methylamino)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolylisonicotinamide (<b>29</b>)</h4><div class="NLM_p last">A mixture of 2-chloro-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolylisonicotinamide (180 mg, 0.50 mmol) and methylamine (102 mg, 1.51 mmol) in NMP (1.2 mL) was subject to microwave heating at 200 °C for 1 h. The crude mixture was directly subject to reverse-phase HPLC purification (gradient elution: 10–30% ACN in water containing 0.1% trifluoroacetic acid) to afford the bis-trifluoroacetate salt of the title compound as pale-yellow solid (132 mg, 46%) following lyophilization. <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 2.01–2.18 (m, 6H), 2.36 (s, 3H), 3.06–3.11 (m, 2H), 3.12 (s, 3H), 3.27–3.30 (m, 2H), 3.53 (t, <i>J</i> = 6.2 Hz, 2H), 3.64–3.71 (m, 2H), 7.14 (s, 1H), 7.33–7.41 (m, 2H), 7.42 (s, 1H), 7.43–7.52 (m, 3H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 353.2. <i>t</i><sub>R</sub> = 1.06 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>29</sub>N<sub>4</sub>O (M + H)<sup>+</sup>, 353.2336; found, 353.2338.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 6-(2-(4-(3,4-Dichlorophenethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxamide (<b>30</b>)</h4><div class="NLM_p last">A mixture of EDC (145 mg, 0.81 mmol), HOBt (223 mg, 0.81 mmol), 6-(2-(4-(3,4-dichlorophenethyl)piperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxylic acid <b>55</b> (200 mg, 0.40 mmol), and 3-(pyrrolidin-1-yl)propan-1-amine (207 mg, 1.61 mmol) in DMF (2 mL) was stirred at 60 °C for 2 h. The mixture was directly purified by reverse-phase HPLC (gradient elution: 5–85% ACN in water containing 0.1% ammonium acetate) to afford the title compound as a dry film (8 mg, 5%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.53 (br s, 4H), 1.60–1.74 (m, 2H), 2.18–2.32 (m, 5H), 2.32–2.40 (m, 5H), 2.61 (t, <i>J</i> = 7.4 Hz, 3H), 2.72 (br s, 4H), 3.27–3.38 (m, 3H), 6.98–7.16 (m, 3H), 7.21–7.38 (m, 1H), 7.38–7.48 (m, 2H), 7.53 (dd, <i>J</i> = 7.9, 1.6 Hz, 1H), 8.22–8.30 (m, 2H), 8.82 (t, <i>J</i> = 5.4 Hz, 1H), 13.74 (s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 606.2. <i>t</i><sub>R</sub> = 1.68 min.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2′-(4-(3-Methylphenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>31</b>)</h4><div class="NLM_p last">To a stirred solution of 2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide dihydrochloride <b>60</b> (220 mg, 0.26 mmol) and triethylamine (0.180 mL, 1.29 mmol) in ACN (1.5 mL) was added the 1-(2-bromoethyl)-3-methylbenzene (0.043 mL, 0.28 mmol). The solution was then brought to reflux for 16 h. Upon cooling, the mixture was diluted with MeOH, filtered, and purified by reverse-phase HPLC (gradient elution: 15–40% ACN in water containing 0.1% trifluoroacetic acid). Lyophilization gave rise to bis-trifluoroacetate of the title compound as a white solid (97 mg, 50%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.95–2.08 (m, 4H), 2.10–2.19 (m, 2H), 2.32 (s, 3H), 2.94–3.09 (m, 8H), 3.23–3.28 (m, 4H), 3.32–3.38 (m, 2H), 3.51 (app t, <i>J</i> = 6.6 Hz, 4 H), 3.65 (m, 2 H), 7.00–7.08 (m, 2H), 7.09 (s, 1H), 7.17–7.25 (m, 3H), 7.33 (dd, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.35–7.44 (m, 1H), 7.56 (t, <i>J</i> = 7.7 Hz, 1H), 7.78–7.82 (m, 1H), 7.82–7.86 (m, 1H), 8.11 (t, <i>J</i> = 1.6 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 511.3. <i>t</i><sub>R</sub> = 1.92 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>43</sub>N<sub>4</sub>O (M + H)<sup>+</sup>, 511.3431; found, 511.3432.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2′-(4-(3-Methoxyphenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>32</b>)</h4><div class="NLM_p last">The compound was prepared in an analogous manner to compound <b>31</b> using 3-methoxyphenethyl bromide to afford the bis-trifluoroacetate salt of the title compound as a colorless semisolid (44 mg, 41%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.87–2.15 (m, 6H), 2.82–3.06 (m, 8H), 3.09–3.21 (m, 4H), 3.25–3.34 (m, 2H), 3.36–3.49 (m, 4H), 3.57 (d, <i>J</i> = 5.1 Hz, 2H), 3.70 (s, 3H), 6.71–6.81 (m, 3H), 7.07–7.20 (m, 3H), 7.25 (dd, <i>J</i> = 7.5, 1.6 Hz, 1H), 7.28–7.35 (m, 1H), 7.49 (t, <i>J</i> = 7.7 Hz, 1 H), 7.68–7.79 (m, 2H), 8.03 (t, <i>J</i> = 1.6 Hz, 1H), 8.71 (t, <i>J</i> = 5.7 Hz, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 527.3. <i>t</i><sub>R</sub> = 1.73 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 527.3381; found, 527.3379.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2′-(4-(4-Chlorophenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>33</b>)</h4><div class="NLM_p last">The material was prepared in an analogous manner to compound <b>31</b> using 1-(2-bromoethyl)-4-chlorobenzene to afford the title compound as the bis-trifluoroacetate salt (97 mg, 50%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.97–2.23 (m, 6H), 2.95–3.15 (m, 8H), 3.19–3.31 (m, 4H), 3.35–3.46 (m, 2H), 3.54 (t, <i>J</i> = 6.6 Hz, 4H), 3.63–3.74 (m, 2H), 7.18–7.25 (m, 2H), 7.26–7.31 (m, 2H), 7.32–7.44 (m, 4H), 7.58 (t, <i>J</i> = 7.8 Hz, 1H), 7.81–7.88 (m, 2H), 8.14 (t, <i>J</i> = 1.6 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 531.2. <i>t</i><sub>R</sub> = 1.72 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>32</sub>H<sub>40</sub>ClN<sub>4</sub>O (M + H)<sup>+</sup>, 531.2885; found, 531.2888.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 2′-(4-Phenethylpiperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>34</b>)</h4><div class="NLM_p last">The material was prepared in an analogous manner to compound <b>31</b> using (2-bromoethyl)benzene to afford the bis-trifluoroacetate salt of the title compound as a gum (65 mg, 35%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.94–2.28 (m, 6H), 2.96–3.14 (m, 8H), 3.18–3.31 (m, 4H), 3.35–3.43 (m, 2H), 3.54 (app t, <i>J</i> = 6.6 Hz, 4H), 3.62–3.77 (m, 2H), 7.17–7.25 (m, 2H), 7.25–7.31 (m, 3H), 7.32–7.43 (m, 4H), 7.58 (t, <i>J</i> = 7.7 Hz, 1H), 7.84 (app tt, <i>J</i> = 7.8, 1.2 Hz, 2H), 8.14 (t, <i>J</i> = 1.6 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 497.3. <i>t</i><sub>R</sub> = 1.67 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>32</sub>H<sub>41</sub>N<sub>4</sub>O (M + H)<sup>+</sup>, 497.3275; found, 497.3273.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 2′-(4-(2-(1<i>H</i>-Indol-3-yl)ethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>35</b>)</h4><div class="NLM_p last">The title compound was prepared in an analogous manner to compound <b>31</b> using 3-(2-bromoethyl)-1<i>H</i>-indole. Following reverse-phase HPLC purification (gradient elution: 5–55% ACN in water containing 0.1% trifluoroacetic acid), the fractions containing the desired product were combined and concentrated under reduced pressure to remove the organics. The aqueous remains were treated with NH<sub>4</sub>OH (aq) dropwise and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give rise to the title compound (34 mg, 31%) as an off-white solid. <sup>1</sup>H NMR (400 MHz CD<sub>2</sub>Cl<sub>2</sub>) δ 1.18 (s, 2H), 1.61 (app ddd, <i>J</i> = 6.3, 3.5, 3.3 Hz, 4H), 1.76 (quin, <i>J</i> = 6.0 Hz, 2H), 2.35 (br s, 3H), 2.49–2.62 (m, 5H), 2.65–2.72 (m, 2H), 2.73–2.83 (m, 6H), 3.40–3.51 (m, 2H), 6.91–7.01 (m, 4H), 7.05 (td, <i>J</i> = 7.6, 1.3 Hz, 1H), 7.14–7.28 (m, 3H), 7.37 (t, <i>J</i> = 7.7 Hz, 1H), 7.46 (d, <i>J</i> = 7.8 Hz, 1H), 7.66 (dt, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.73 (ddd, <i>J</i> = 7.7, 1.4, 1.3 Hz, 1H), 7.92 (t, <i>J</i> = 1.5 Hz, 1H), 8.27 (br s, 1H), 8.47 (br s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 536.3. <i>t</i><sub>R</sub> = 1.86 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>42</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 536.3384; found, 536.3380.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2′-(4-(3,4-Dichlorophenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>36</b>)</h4><div class="NLM_p last">The 3,4-dichlorophenethylmethanesulfonate (211 mg, 0.78 mmol) was added to a stirred mixture of 2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide dihydrochloride <b>60</b> (243 mg, 0.39 mmol) and K<sub>2</sub>CO<sub>3</sub> (433 mg, 3.13 mmol) in ACN (3 mL). The mixture was heated to 70 °C for 20 h. The mixture was cooled, filtered, and the filtrate concentrated under reduced pressure. The crude product was subject to silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc). The resultant impure solid was again subject to silica gel chromatography (isocratic: 5% of 3% NH<sub>4</sub>OH/MeOH in EtOAc), and the resultant product was treated with 4 M HCl in dioxane. Concentration and vacuum drying yielded the dihydrochloride of the title compound as a pale-tan solid (159 mg, 72%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 2.01–2.19 (m, 6H), 3.00–3.17 (m, 8H), 3.25–3.30 (m, 2H), 3.34–3.42 (m, 2H), 3.49–3.56 (m, 4H), 3.63–3.71 (m, 4H), 7.16–7.22 (m, 2H), 7.27 (d, <i>J</i> = 8.1 Hz, 1H), 7.31–7.40 (m, 2H), 7.48 (d, <i>J</i> = 8.3 Hz, 1H), 7.52–7.61 (m, 2H), 7.84 (t, <i>J</i> = 8.0 Hz, 2H), 8.14 (s, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 565.2. <i>t</i><sub>R</sub> = 1.50 min.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 5-Cyano-2′-(4-(3-methylphenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>37</b>)</h4><div class="NLM_p last">A mixture of 5-cyano-2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide bis-trifluoroacetate <b>62b</b> (75 mg, 0.18 mmol), 1-(2-bromoethyl)-3-methylbenzene (34 mg, 0.2 mmol), and K<sub>2</sub>CO<sub>3</sub> (99 mg, 0.072 mmol) in ACN (1 mL) was subject to microwave heating at 100 °C for 20 min then 150 °C for 30 min. The mixture was filtered then concentrated under reduced pressure. The crude product was subject to silica gel chromatography (gradient elution: 0–10% of 1% NH<sub>4</sub>OH/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to afford the impure desired compound. The residue was purified via reverse-phase HPLC (gradient elution: 10–70% ACN in water containing 0.1% trifluoroacetic acid). The appropriate fractions were collected and set to lyophilize to afford the bis-trifluoroacetate of title compound as white solid (57 mg, 42%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 1.96–2.20 (m, 6H), 2.32 (s, 3H), 2.94–3.15 (m, 8H), 3.16–3.30 (m, 4H), 3.33–3.41 (m, 2H), 3.48–3.59 (m, 4H), 3.61–3.71 (m, 2H), 7.00–7.14 (m, 3H), 7.34–7.40 (m, 1H), 7.16–7.31 (m, 3H), 7.41–7.51 (m, 1H), 8.15 (dt, <i>J</i> = 10.3, 1.6 Hz, 2H), 8.46 (t, <i>J</i> = 1.8 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 536.2. <i>t</i><sub>R</sub> = 1.86 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>42</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 536.3384; found, 536.3378.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 5-Cyano-2′-(4-(3-methoxyphenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>38</b>)</h4><div class="NLM_p last">A vial was charged with 5-cyano-2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide bis-trifluoroacetate <b>62b</b> (74 mg, 0.18 mmol), 3-methoxyphenethyl bromide (0.033 mL, 0.21 mmol), K<sub>2</sub>CO<sub>3</sub> (98 mg, 0.71 mmol), and ACN (2 mL). The stirred mixture was subject to microwave heating at 150 °C for 1 h. The mixture was filtered and purified by reverse-phase HPLC (gradient elution: 10–70% ACN in water containing 0.1% trifluoroacetic acid) to afford the desired product as the bis-trifluoroacetate following lyophilization. The resulting gum was treated with excess 4 M HCl in dioxane to afford the dihydrochloride salt of the title compound (40 mg, 36%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.96–2.22 (m, 6H), 2.96–3.17 (m, 8H), 3.18–3.29 (m, 4H), 3.35–3.43 (m, 2H), 3.47–3.59 (m, 4H), 3.61–3.70 (m, 2H), 3.79 (s, 3H), 6.71–6.92 (m, 3H), 7.22–7.31 (m, 3H), 7.35–7.51 (m, 2H), 8.15 (dt, <i>J</i> = 19.7, 1.5 Hz, 2H), 8.47 (t, <i>J</i> = 1.6 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 552.3. <i>t</i><sub>R</sub> = 2.1 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>42</sub>N<sub>5</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 552.3333; found, 552.3326.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2′-(4-(2-(1<i>H</i>-Indol-3-yl)ethyl)piperazin-1-yl)-5-cyano-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-[1,1′-biphenyl]-3-carboxamide (<b>39</b>)</h4><div class="NLM_p last">Potassium carbonate (596 mg, 4.31 mmol) was added to a stirred solution of 5-cyano-2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-[1,1′-biphenyl]-3-carboxamide bis-trifluoroacetate <b>62b</b> (655 mg, 0.86 mmol) in ACN (7 mL). The mixture was stirred for 3 min then 3-(2-bromoethyl)-1<i>H</i>-indole (199 mg, 0.86 mmol) was added and the mixture was heated at 80 °C for 14 h. The mixture was cooled to ambient temperature, diluted with EtOAc, filtered, and concentrated under reduced pressure. The resultant gum was subject to silica gel chromatography (40 g; gradient elution: 0.5–10% 2 M NH<sub>3</sub>/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to afford the impure product as a white solid. The resulting solid was dissolved in DMSO and purified by reverse-phase HPLC (gradient elution: 25–50% ACN in water containing 10 mM NH<sub>4</sub>OAc). The appropriate fractions were dried and subjected to lyophilization to afford the title compound (111 mg, 18%) as the white solid of the acetate salt (∼2.5 equiv). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.61 (td, <i>J</i> = 6.8, 3.1 Hz, 4H), 1.66–1.77 (m, 2H), 1.90 (s, 2.5H, acetate salt), 2.34–2.40 (m, 7H), 2.79 (d, <i>J</i> = 5.0 Hz, 6H), 3.32–3.39 (m, 7H), 6.89–6.98 (m, 1H), 7.00–7.08 (m, 1H), 7.22 (d, <i>J</i> = 2.3 Hz, 1H), 7.14–7.23 (m, 2H), 7.32 (d, <i>J</i> = 8.0 Hz, 1H), 7.34–7.43 (m, 2H), 7.47 (d, <i>J</i> = 7.8 Hz, 1H), 8.17 (t, <i>J</i> = 1.5 Hz, 1H), 8.23 (t, <i>J</i> = 1.5 Hz, 1H), 8.48 (t, <i>J</i> = 1.6 Hz, 1H), 8.77 (t, <i>J</i> = 5.3 Hz, 1H), 10.75 (s, 1H). LCMS using procedure B: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 562. <i>t</i><sub>R</sub> = 0.60 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>35</sub>H<sub>41</sub>N<sub>6</sub>O (M + H)<sup>+</sup>, 561.3336; found, 561.3323.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 5-Cyano-2′-(4-(3,4-dichlorophenethyl)piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>40</b>)</h4><div class="NLM_p last">A mixture of 5-cyano-2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide bis-trifluoroacetate <b>62b</b> (183 mg, 0.44 mmol), 3,4-dichlorophenethylmethanesulfonate (236 mg, 0.88 mmol), and K<sub>2</sub>CO<sub>3</sub> (484 mg, 3.50 mmol) in ACN (3 mL) was allowed to stir at 70 °C for 20 h. The reaction was cooled to ambient temperature, sodium iodide (66 mg, 0.44 mmol) was added, and heating resumed. After 24 h, the reaction mixture was directly subject to silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) and the desired fractions were collected and concentrated in vacuo. The resulting gum was treated with 4 M HCl in dioxane, and the mixture was concentrated under reduced pressure. Drying in vacuo gave rise to the dihydrochloride salt of title compound as a tan solid (200 mg, 69%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.98–2.25 (m, 6H), 2.94–3.27 (m, 10 H), 3.35–3.46 (m, 2H), 3.46–3.85 (m, 8H), 7.28 (dd, <i>J</i> = 6.2, 3.8 Hz, 3H), 7.34–7.61 (m, 4H), 8.07–8.25 (m, 2H), 8.48 (br s, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 590.3. <i>t</i><sub>R</sub> = 1.98 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>38</sub>Cl<sub>2</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 590.2448; found, 590.2448.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-(2-(4-(3-Methoxyphenethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>41</b>)</h4><div class="NLM_p last">A solution of 2-(2-(piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide dihydrochloride <b>63b</b> (100 mg, 0.21 mmol), 1-(2-bromoethyl)-3-methoxybenezene (0.050 mL, 0.32 mmol), and DIPEA (0.150 mL, 0.86 mmol) in DMF (1 mL) was stirred at 90 °C overnight. The crude reaction was directly subject to silica gel chromatography (gradient elution: 0–10% of 1% NH<sub>4</sub>OH/MeOH in CH<sub>2</sub>Cl<sub>2</sub>). The resultant was dissolved in MeOH and treated with 1 M HCl in Et<sub>2</sub>O. The solution was concentrated in vacuo, and the residue was dissolved in water. Lyophilization afforded the dihydrochloride salt of the title compound (23 mg, 16%) as a white solid. <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.98–2.21 (m, 6H), 3.03–3.18 (m, 4H), 3.20–3.29 (m, 6H), 3.34–3.40 (m, 2H), 3.42–3.50 (m, 2H), 3.54–3.63 (m, 4H), 3.65–3.72 (m, 2H), 3.79 (s, 3H), 6.79–6.93 (m, 3H), 7.25 (t, <i>J</i> = 7.8 Hz, 1H), 7.45 (t, <i>J</i> = 7.4 Hz, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 1H), 7.66–7.74 (m, 1H), 7.77 (d, <i>J</i> = 7.6 Hz, 1H), 8.41 (d, <i>J</i> = 5.3 Hz, 1H), 8.77 (s, 1H), 9.11 (d, <i>J</i> = 5.8 Hz, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 528.2. <i>t</i><sub>R</sub> = 1.46 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>32</sub>H<sub>42</sub>N<sub>5</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 528.3333; found, 528.3326.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-(2-(4-(4-Methoxyphenethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>42</b>)</h4><div class="NLM_p last">The title compound was prepared in an analogous manner to compound <b>41</b> using 1-(2-bromoethyl)-4-methoxybenezene to yield the dihydrochloride as a white solid (40 mg, 30%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.92–2.24 (m, 6H), 2.99–3.06 (m, 2H), 3.07–3.16 (m, 2H), 3.16–3.28 (m, 4H), 3.33–3.46 (m, 2H), 3.53–3.64 (m, 4H), 3.53–3.64 (m, 4H), 3.68 (m, 2H), 3.77 (s, 3H), 6.90 (d, <i>J</i> = 8.6 Hz, 2H), 7.23 (d, <i>J</i> = 8.6 Hz, 2H), 7.40–7.57 (m, 2H), 7.67–7.82 (m, 2H), 8.34–8.47 (m, 1H), 8.77 (s, 1H), 9.09 (d, <i>J</i> = 6.1 Hz, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 528.3. <i>t</i><sub>R</sub> = 1.47 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>32</sub>H<sub>42</sub>N<sub>5</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 528.3333; found, 528.3321.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-(2-(4-(4-Chlorophenethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>43</b>)</h4><div class="NLM_p last">A solution of 2-(2-(piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)iso-nicotinamide dihydrochloride <b>63b</b> (150 mg, 0.32 mmol), 1-(2-bromoethyl)-4-chlorobenzene (0.049 mL, 0.34 mmol), and DIPEA (0.225 mL, 1.29 mmol) in DMF (2 mL) was heated to 70 °C overnight. Additional 1-(2-bromoethyl)-4-chlorobenzene (50 μL) and DIPEA (200 μL) were added, and the reaction was heated 95 °C for 3 h. The solution was allowed to cool and directly subjected to silica gel chromatography (gradient elution: 0–10% of 1% NH<sub>4</sub>OH/MeOH in CH<sub>2</sub>Cl<sub>2</sub>). The resultant residue was treated with 4 M HCl in dioxane and concentrated under reduced pressure. The solid was triturated with Et<sub>2</sub>O/hexanes to afford the dihydrochloride of the title compound as a white solid (80 mg, 41%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 2.17 (m, 6H), 3.11 (d, <i>J</i> = 4.6 Hz, 4H), 3.20–3.29 (m, 6H), 3.35 (s, 2H), 3.44 (br s, 2H), 3.52–3.64 (m, 4H), 3.68 (d, <i>J</i> = 4.0 Hz, 2H), 7.34 (d, <i>J</i> = 4.8 Hz, 4H), 7.40–7.55 (m, 2H), 7.71 (d, <i>J</i> = 10.6 Hz, 2H), 8.28–8.40 (m, 1H), 8.72 (s, 1H), 8.98–9.13 (m, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 532.2. <i>t</i><sub>R</sub> = 1.58 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>39</sub>ClN<sub>5</sub>O (M + H)<sup>+</sup>, 532.2838; found, 532.2832.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-(2-(4-(3,4-Dichlorophenethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>44</b>)</h4><div class="NLM_p last">A suspension of 2-(2-(piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide dihydrochloride <b>63b</b> (100 mg, 0.21 mmol), 3,4-dichlorophenethylmethanesulfonate (115 mg, 0.43 mmol), and K<sub>2</sub>CO<sub>3</sub> (237 mg, 1.72 mmol) in ACN (15 mL) was stirred at 70 °C for 24 h then at reflux for 16 h. The reaction mixture was cooled, filtered, and the filter cake washed with MeOH. The filtrate was concentrated and then purified by reverse-phase HPLC (gradient elution of ACN in water containing 0.1% trifluoroacetic acid) to afford the title compound as a pale-yellow solid (41 mg, 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.72–1.97 (m, 6H), 2.75–3.02 (m, 8H), 3.10 (dt, <i>J</i> = 10.4, 5.4 Hz, 4H), 3.19–3.35 (m, 4H), 3.35–3.53 (m, 4H), 7.01–7.29 (m, 3H), 7.29–7.44 (m, 1H), 7.44–7.62 (m, 3H), 7.68 (dd, <i>J</i> = 5.1, 1.5 Hz, 1H), 8.32 (s, 1H), 8.77 (d, <i>J</i> = 5.31 Hz, 1H), 8.89 (t, <i>J</i> = 5.7 Hz, 1H), 9.78 (br s, 1H), 10.06 (br s, 1H). HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>38</sub>Cl<sub>2</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 566.2448; found, 566.2445.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 2-(2-(4-(2-(1<i>H</i>-Indol-3-yl)ethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>45</b>)</h4><div class="NLM_p last">A mixture of 2-(2-(piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide dihydrochloride <b>63b</b> (100 mg, 0.21 mmol), 3-(2-bromoethyl)-1<i>H</i>-indole (29 mg, 0.13 mmol), DIPEA (0.75 mL, 0.43 mmol), and K<sub>2</sub>CO<sub>3</sub> (59 mg, 0.43 mmol) in DMF (1 mL) was subject to microwave heating at 100 °C for 30 min then at 150 °C for 30 min. The mixture was allowed to cool, filtered, and the crude product solution purified by silica gel chromatography (gradient elution: 0–10% of 1% NH<sub>4</sub>OH/MeOH in CH<sub>2</sub>Cl<sub>2</sub>). The resultant product was dissolved in MeOH and treated with 1 M HCl in Et<sub>2</sub>O. The solution was concentrated in vacuo and the residue was dissolved in water. Lyophilization afforded the dihydrochloride salt of the title compound as a white solid (27 mg, 40%). <sup>1</sup>H NMR (400 MHz CD<sub>3</sub>OD) δ 1.20–1.43 (m, 2H), 1.97–2.03 (m, 2H), 2.11 (br s, 4H), 3.03 (br s, 2H), 3.26 (br s, 8H), 3.43–3.53 (m, 2H), 3.55–3.72 (m, 6H), 6.99–7.08 (m, 1H), 7.12 (t, <i>J</i> = 7.6 Hz, 1H), 7.22 (s, 1H), 7.37 (d, <i>J</i> = 8.1 Hz, 1H), 7.40–7.54 (m, 2H), 7.55–7.82 (m, 3H), 8.35 (br s, 1H), 8.77 (br s, 1H), 9.04 (br s, 1H), 9.38 (br s, 1H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 537.2. <i>t</i><sub>R</sub> = 1.50 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>41</sub>N<sub>6</sub>O (M + H)<sup>+</sup>, 537.3336; found, 537.3324.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 5-(2-(4-(3,4-Dichlorophenethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)nicotinamide (<b>46</b>)</h4><div class="NLM_p last">A mixture of 5-(2-(piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)nicotinamide bis-trifluoroacetate <b>66b</b> (727 mg, 1.17 mmol) in ACN (4 mL) was treated with a solution of 3,4-dichlorophenethylmethanesulfonate (630 mg, 2.34 mmol) in ACN (1 mL), followed by K<sub>2</sub>CO<sub>3</sub> (1.29 g, 9.36 mmol). The mixture was heated to 70 °C for 4 h. The mixture was cooled, filtered, and concentrated. The residue was purified by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the desired product. The residue was then treated with 4 M HCl in dioxane, filtered, and dried in vacuo to afford the dihydrochloride salt of title compound (344 mg, 46%) as a white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 1.97–2.06 (m, 2H), 2.08–2.17 (m, 4H), 3.05–3.15 (m, 4H), 3.15–3.27 (m, 6H), 3.31–3.37 (m, 2H), 3.38–3.48 (m, 2H), 3.52–3.62 (m, 4H), 3.65–3.73 (m, 2H), 7.28 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 7.37–7.45 (m, 2H), 7.49 (d, <i>J</i> = 8.3 Hz, 1H), 7.53–7.61 (m, 3H), 9.25 (d, <i>J</i> = 1.5 Hz, 1H), 9.27–9.32 (m, 2H); exchangeable NH not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 566.3. <i>t</i><sub>R</sub> = 1.66 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>38</sub>Cl<sub>2</sub>N<sub>5</sub>O (M + H)<sup>+</sup>, 566.2448; found, 566.2453.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 5-(2-(4-(2-(1<i>H</i>-Indol-3-yl)ethyl)piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)nicotinamide (<b>47</b>)</h4><div class="NLM_p last">The title compound was prepared in a similar manner as described.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Briefly, to a stirred mixture of 5-(2-(piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)nicotinamide, bis-trifluoroacetate <b>66b</b> (605 mg, 0.97 mmol), TEA (0.723 mL, 5.19 mmol), and 4 Å molecular sieves (powdered) in anhydrous ACN (4 mL) were added the 3-(2-bromoethyl)-1<i>H</i>-indole (320 mg, 1.43 mmol) and sodium iodide (16 mg, 0.11 mmol). The reaction was warmed to 75 °C for 40 h then was cooled, diluted with MeOH, filtered, and concentrated under reduced pressure. The resultant residue was subject to silica gel chromatography (gradient elution: 1–10% 2 M NH<sub>3</sub>/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound (115 mg) in 94% purity (by HPLC). The solid was dissolved in MeOH and purified by reverse-phase HPLC (gradient elution: 10–45% ACN in water containing 0.1% trifluoroacetic acid). After lyophilization, the resultant trifluoroacetate salt was dissolved in CH<sub>2</sub>Cl<sub>2</sub> and washed successively with saturated K<sub>2</sub>CO<sub>3</sub> (aq), water, and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford the title compound. The resultant free-base was then dissolved in MeOH and added to a methanolic solution of citric acid (1 equiv, 37 mg). The solution was concentrated via a N<sub>2</sub> stream, then dried in vacuo overnight, giving rise to a white solid of the title compound as the citrate salt (141 mg, 20%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 1.95–2.22 (m, 6H), 2.66–2.82 (m, 4H; citric acid), 2.83–2.89 (br s, 2H) 2.97 (m, 2H), 2.98–3.10 (m, 6H), 3.14–3.19 (m, 2H), 3.24–3.31 (m, 6H), 3.55 (t, <i>J</i> = 6.5 Hz, 2H), 6.98–7.06 (m, 1H), 7.08–7.16 (m, 2H), 7.21–7.31 (m, 2H), 7.36 (d, <i>J</i> = 8.1 Hz, 1H), 7.40 (dd, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.42–7.49 (m, 1H), 7.56 (d, <i>J</i> = 7.6 Hz, 1H), 8.61 (t, <i>J</i> = 2.2 Hz, 1H), 8.97 (dd, <i>J</i> = 7.2, 2.2 Hz, 2H); two exchangeable NHs and citric acid OHs not observed. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 537.2. <i>t</i><sub>R</sub> = 1.59 min. HRMS (ES+, TOF) <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>41</sub>N<sub>6</sub>O (M + H)<sup>+</sup>, 537.3336; found, 537.3340.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 6-(<i>o</i>-Tolyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxylic Acid (<b>50a</b>)</h4><div class="NLM_p last">The title compound was prepared as described.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Briefly, a round-bottom flask was charged with a magnetic stir bar, 1<i>H</i>-pyrazol-5-amine (1.000 g, 12.03 mmol), 2-oxopropanoic acid (0.841 mL, 12.03 mmol), acetic acid (17 mL), and 2-methylbenzaldehyde (1.446 g, 12.03 mmol). The reaction was heated to reflux for 12 h then allowed to cool to ambient temperature. Water (400 mL) was added, and the resulting tan precipitate was collected via vacuum filtration. The solid was washed with ethanol, collected, and dried in vacuo to afford the title compound as an off-white solid (0.655 g, 21.5%). This material was used directly without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.36 (s, 3H), 7.31–7.40 (m, 3H), 7.49 (d, 1H), 7.72 (s, 1H), 8.39 (s, 1H), 13.98 (s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 254.2. <i>t</i><sub>R</sub> = 2.23 min.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2,6-Dichloro-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>51f</b>)</h4><div class="NLM_p last">A solution of 2,6-dichloroisonicotinic acid (3.0 g, 15.622 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) was treated in succession with 3-(pyrrolidin-1-yl)propan-1-amine (2.5 mL, 17.19 mmol), EDC (3.59 g, 1.8.75 mmol), TEA (6.53 mL, 46.88 mmol), and HOBt (2.87 g, 18.75 mmol). The reaction mixture was stirred at room temperature for 12 h. The mixture was concentrated and the crude product purified by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the title compound as an orange solid (3.367 g, 71%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 303.9. <i>t</i><sub>R</sub> = 1.28 min.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 2-Bromo-<i>N</i>-(3-pyrrolidin-1-ylpropyl)pyridine-4-carboxamide (<b>52a</b>)</h4><div class="NLM_p last">A mixture of 3-(pyrrolidin-1-yl)propan-1-amine (3.9 mL, 27.2 mmol), 2-bromoisonicotinic acid (5 mL, 24.8 mmol), EDC (5.22 g, 27.2 mmol), HOBt (4.17 mg, 27.2 mmol), and TEA (10.35 mL, 74.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (70 mL) was stirred overnight. The solution was further diluted with CH<sub>2</sub>Cl<sub>2</sub> and washed with saturated aq NaHCO<sub>3</sub>. The phases were separated, and the organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the title compound as a white solid upon standing (6.34 g, 82%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 311.9. <i>t</i><sub>R</sub> = 1.02 min.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 3-Bromo-5-cyano-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)benzamide (<b>52b</b>)</h4><div class="NLM_p last">Triethylamine (1.9 mL, 13.63 mmol) was added to a stirred solution of 3-bromo-5-cyanobenzoic acid <b>51b</b> (1.026 g, 4.54 mmol), 3-(pyrrolidin-1-yl)propan-1-amine (0.621 mL, 4.77 mmol), and EDC (1.044 g, 5.45 mmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (15 mL). After 17 h, the reaction was concentrated to ∼<sup>1</sup>/<sub>3</sub> volume and directly subject to silica gel chromatography (gradient elution: 2–7.5% of 2 M NH<sub>3</sub>/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to afford the titled compound as a clear gum (0.75 g, 49%). This material used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.65–1.75 (m, 6H), 2.37–2.47 (m, 4H), 3.27–3.36 (m, 4H), 8.25 (t, <i>J</i> = 1.4 Hz, 1H), 8.30 (t, <i>J</i> = 1.8 Hz, 1H), 8.32–8.37 (m, 1H), 8.81 (t, <i>J</i> = 5.1 Hz, 1H). LCMS using procedure B: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 336.0. <i>t</i><sub>R</sub> = 0.58 min.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 3-Bromo-5-methoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)benzamide (<b>52c</b>)</h4><div class="NLM_p last">The triethylamine (0.593 mL, 4.25 mmol) was added to a stirred solution of 3-bromo-5-methoxybenzoic acid <b>51c</b> (328 mg, 1.42 mmol), 3-(pyrrolidin-1-yl)propan-1-amine (0.22 mL, 1.56 mmol), and EDC (326 mg, 1.70 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (15 mL) under a nitrogen atmosphere. After 12 h, the reaction was concentrated to ∼<sup>1</sup>/<sub>3</sub> volume and directly subject to silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the titled compound as a clear gum (485 mg, quant). This material used as-is without further purification. LCMS using procedure B: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 342.9. <i>t</i><sub>R</sub> = 1.56 min.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-Bromo-<i>N</i>-(3-pyrrolidin-1-ylpropyl)pyridine-4-carboxamide (<b>52d</b>)</h4><div class="NLM_p last">A mixture of 3-(pyrrolidin-1-yl)propan-1-amine (3.9 mL, 27.2 mmol), 2-bromoisonicotinic acid <b>51d</b> (5 mL, 24.8 mmol), EDC (5.22 g, 27.2 mmol), HOBt (4.17 mg, 27.2 mmol), and TEA (10.35 mL, 74.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (70 mL) was stirred overnight. The solution was further diluted with CH<sub>2</sub>Cl<sub>2</sub> and washed with saturated aq NaHCO<sub>3</sub>. The phases were separated, and the organics were dried over dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the title compound as a white solid upon standing (6.34 g, 82%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 311.9. <i>t</i><sub>R</sub> = 1.02 min.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2-Chloro-6-methoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide (<b>52e</b>)</h4><div class="NLM_p last">A solution of 2-chloro-6-methoxyisonicotinic acid <b>51e</b> (266 mg, 1.42 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was treated in succession with 3-(pyrrolidin-1-yl)propan-1-amine (0.222 mL, 1.56 mmol), EDC (326 mg, 1.70 mmol), TEA (0.593 mL, 4.25 mmol), and HOBt (261 mg, 1.70 mmol). The reaction mixture was stirred at room temperature for 16 h. The mixture was subject to silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the title compound as a white solid (353 mg, 84%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 298.0. <i>t</i><sub>R</sub> = 2.08 min.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 2-Chloro-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-6-<i>o</i>-tolylisonicotinamide (<b>52f</b>)</h4><div class="NLM_p last">A solution of 2,6-dichloro-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)isonicotinamide <b>51f</b> (1.89 g, 6.24 mmol), <i>o</i>-tolylboronic acid (0.81 g, 5.93 mmol), K<sub>2</sub>CO<sub>3</sub> (2.59 g, 18.71 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.72 g, 0.62 mmol) in a mixture of DME/water (10:1, 5.5 mL) was stirred at 80 °C for 12 h. The reaction mixture was concentrated in vacuo and the crude product purified by silica gel chromatography (gradient elution: 0–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the title compound as a solid (371 mg, 17%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 358.1. <i>t</i><sub>R</sub> = 1.92 min.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 2-(Piperazin-1-yl)benzaldehyde (<b>53b</b>)</h4><div class="NLM_p last">A solution of 4 M HCl in dioxane (10 mL, 288 mmol) was added to the solid <i>tert</i>-butyl 4-(2-formylphenyl)piperazine-1-carboxylate <b>53a</b> (1 g, 3.44 mmol). The resulting suspension was stirred at ambient temperature for 16 h. The reaction mixture was diluted with hexane (20 mL), and the solid from the reaction mixture was collected by filtration and dried in vacuo to yield the hydrochloride salt of the title compound as a yellow solid (0.780 g, 100%). LCMS using procedure B: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 191.2. <i>t</i><sub>R</sub> = 0.36 min.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-(4-(3,4-Dichlorophenethyl)piperazin-1-yl)benzaldehyde (<b>54</b>)</h4><div class="NLM_p last">To a mixture of 3,4-dichlorophenethylmethanesulfonate (722 mg, 2.65 mmol), prepared as described<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and 2-(piperazin-1-yl)benzaldehyde <b>53b</b> (600 mg, 2.65 mmol) in ACN (30 mL) was added K<sub>2</sub>CO<sub>3</sub> (1829 mg, 13.23 mmol) in one portion. The suspension was stirred at 70 °C for overnight. Water was added to the mixture, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub>. The organics were dried over MgSO<sub>4</sub>, filtered, and concentrated. The residue was purified by silica gel chromatography (gradient elution: 15–50% EtOAc in hexanes) to yield the title compound as a light-yellow oil (620 mg, 65%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.58–2.71 (m, 6H), 2.72–2.88 (m, 2H), 2.91–3.18 (m, 4H), 7.15 (t, <i>J</i> = 7.4 Hz, 1H), 7.22 (d, <i>J</i> = 7.8 Hz, 1H), 7.28 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 7.45–7.64 (m, 3H), 7.70 (dd, <i>J</i> = 7.7, 1.6 Hz, 1H), 10.21 (s, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> = 364.9. <i>t</i><sub>R</sub> = 2.16 min.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 6-(2-(4-(3,4-Dichlorophenethyl)piperazin-1-yl)phenyl)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridine-4-carboxylic Acid (<b>55</b>)</h4><div class="NLM_p last">A mixture of 2-(4-(3,4-dichlorophenethyl)piperazin-1-yl)benzaldehyde <b>54</b> (611 mg, 1.68 mmol), 1<i>H</i>-pyrazol-3-amine (140 mg, 1.68 mmol), and 2-oxopropanoic acid (148 mg, 1.68 mmol) in acetic acid (5 mL) was stirred at reflux for 16 h. The reaction was concentrated under reduced pressure. The residue was treated with Et<sub>2</sub>O and sonicated. The resultant solid was then collected by filtration. The solid was dissolved in DMSO and purified by reverse-phase HPLC (gradient elution: 10–75% ACN in water containing 0.1% trifluoroacetic acid) to afford the title compound (200 mg, 24%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+)<sup>+</sup> (M + H)<sup>+</sup> = 496.1. <i>t</i><sub>R</sub> = 2.17 min.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>tert</i>-Butyl 4-(2-Bromophenyl)piperazine-1-carboxylate (<b>57</b>)</h4><div class="NLM_p last">To a stirred solution of 1-(2-bromophenyl)piperazine <b>56</b> (650 mg, 2.70 mmol) and TEA (0.451 mL, 3.23 mmol) in a mixture of Et<sub>2</sub>O/THF (1:1, 10 mL) was added a solution of Boc<sub>2</sub>O (706 mg, 3.23 mmol) in Et<sub>2</sub>O (2 mL) dropwise. After 30 min, the mixture was diluted with Et<sub>2</sub>O (10 mL) and washed with water (10 mL) then brine (10 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered through a plug of silica, and concentrated to dryness to give rise to the desire product as a white semisolid (898 mg, 98%). This material was used directly without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.44–1.53 (m, 10H), 2.90–3.05 (m, 4H), 3.55–3.67 (m, 4H), 6.90–6.98 (m, 1H), 7.00–7.08 (m, 1H), 7.23–7.32 (m, 1H), 7.58 (dd, <i>J</i> = 8.1, 1.5 Hz, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 340.9. <i>t</i><sub>R</sub> = 3.40 min.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>tert</i>-Butyl 4-(3′-(Ethoxycarbonyl)biphenyl-2-yl)piperazine-1-carboxylate (<b>58a</b>)</h4><div class="NLM_p last">To a stirred mixture of <i>tert</i>-butyl 4-(2-bromophenyl)piperazine-1-carboxylate <b>57</b> (5.00 g, 14.65 mmol), 3-(ethoxycarbonyl)phenylboronic acid (3.55 g, 18.32 mmol), and K<sub>2</sub>CO<sub>3</sub> (4.05 g, 29.30 mmol) in dioxane/water (10:1, 80 mL) was added Pd(PPh<sub>3</sub>)<sub>4</sub> (1.693 g, 1.47 mmol). The reaction vessel was capped with a septa, purged with argon, and heated to 90 °C for 22 h. The reaction was diluted with water (100 mL) and extracted with ethyl acetate (2 × 100 mL). The combined organic extract was dried with MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product purified via silica gel chromatography (80 g; 20% EtOAc/hexanes as the eluent) to provide the title compound (5.90 g, 98%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 411.0. <i>t</i><sub>R</sub> = 3.62 min.</div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 2′-(4-(<i>tert</i>-Butoxycarbonyl)piperazin-1-yl)biphenyl-3-carboxylic Acid (<b>58b</b>)</h4><div class="NLM_p last">To a stirred solution of <i>tert</i>-butyl 4-(3′-(ethoxycarbonyl)-biphenyl-2-yl)piperazine-1-carboxylate <b>58a</b> (4.90 g, 11.94 mmol) in a mixture of THF/MeOH/water (2:2:1, 50 mL) was added the LiOH (0.858 g, 35.81 mmol). The reaction was stirred at room temperature overnight. The volatiles were removed in vacuo, and water (50 mL) was added. The flask was cooled in an ice bath and acidified with 1 M HCl (35 mL). A precipitate formed which was collected via vacuum filtration and the solid dried in vacuo to afford the desired compound as an off-white solid (4.18 g, 92%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.18–1.30 (m, 11H), 2.62 (br s, 5H), 3.03–3.15 (m, 5H), 6.96–7.10 (m, 3H), 7.20 (dd, <i>J</i> = 7.5, 1.6 Hz, 1H), 7.22–7.36 (m, 1H), 7.48 (t, <i>J</i> = 7.7 Hz, 1H), 7.62–7.87 (m, 3H), 8.19 (t, <i>J</i> = 1.6 Hz, 1 H), 22.89 (br s, 1 H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 383.0. <i>t</i><sub>R</sub> = 3.42 min.</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>tert</i>-Butyl 4-(3′-(3-(Pyrrolidin-1-yl)propylcarbamoyl)biphenyl-2-yl)piperazine-1-carboxylate (<b>59</b>)</h4><div class="NLM_p last">HOBt (2.51 g, 16.39 mmol) and EDC (3.14 g, 16.39 mmol) were added to a stirred solution of 2′-(4-(<i>tert</i>-butoxycarbonyl)piperazin-1-yl)biphenyl-3-carboxylic acid <b>58b</b> (4.18 g, 10.93 mmol), DIPEA (5.7 mL, 32.8 mmol), and 3-(pyrrolidin-1-yl)propan-1-amine (4.20 g, 32.8 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (38 mL). The reaction was stirred at room temperature overnight. The mixture was poured into water (250 mL) and extracted with CHCl<sub>3</sub>/IPA (10:1, 2 × 250 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified via silica gel chromatography (10% MeOH/CH<sub>2</sub>Cl<sub>2</sub> as eluent) to provide the title compound as a white solid (4.90 g, 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.37 (s, 9H), 1.58–1.79 (m, 6H), 2.51–2.59 (m, 5H), 2.70 (br s, 4H), 3.25–3.37 (m, 4H), 6.64 (s, 1H), 7.04–7.17 (m, 2 H), 7.25–7.38 (m, 2H), 7.44–7.60 (m, 2H), 7.74–7.82 (m, 2H), 8.09–8.20 (m, 2H), 8.48–8.67 (m, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 493.2. <i>t</i><sub>R</sub> = 2.47 min.</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 2′-(Piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)biphenyl-3-carboxamide (<b>60</b>)</h4><div class="NLM_p last">The 4 M HCl in dioxane (10 mL, 40 mmol) was added to a stirred solution of <i>tert</i>-butyl 4-(3′-(3-(pyrrolidin-1-yl)propylcarbamoyl)biphenyl-2-yl)piperazine-1-carboxylate <b>59</b> (2.0 g, 4.06 mmol) in MeOH (6 mL). The mixture was stirred overnight at ambient temperature before being concentrated in vacuo. This material was triturated with Et<sub>2</sub>O, sonicated, and filtered to afford the dihydrochloride salt of the title compound as a white solid upon drying (1.710 g, 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.79–2.06 (m, 4H), 2.83–3.05 (m, 7H), 3.08–3.22 (m, 2H), 3.30–3.42 (m, 2H), 3.45–3.57 (m, 2H), 4.80 (br s, 2H), 7.22–7.22 (m, 1H), 7.31–7.40 (m, 1H), 7.50–7.60 (m, 1H), 7.85 (dd, <i>J</i> = 19.7, 7.8 Hz, 1H), 8.20 (s, 1H), 8.64–8.92 (m, 1H), 8.79 (t, <i>J</i> = 5.8 Hz, 1H), 9.10–9.37 (m, 2H), 9.24 (br s, 2H), 10.73 (br s, 1H), 10.65–10.86 (m, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 393.1. <i>t</i><sub>R</sub> = 1.50 min.</div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>tert</i>-Butyl 4-(3′-Cyano-5′-((3-(pyrrolidin-1-yl)propyl)carbamoyl)-[1,1′-biphenyl]-2-yl)piperazine-1-carboxylate (<b>62a</b>)</h4><div class="NLM_p last">A round-bottom flask was charged with the 3-bromo-5-cyano-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)benzamide <b>52b</b> (745 mg, 2.22 mmol), 2-[4-(<i>N</i>-Boc)piperazin-1-yl]phenylboronic acid pinacol ester <b>61</b> (946 mg, 2.44 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (77 mg, 0.07 mmol), K<sub>2</sub>CO<sub>3</sub> (919 mg, 6.65 mmol), and a stir bar. The flask was evacuated and backfilled with N<sub>2</sub> three times. A mixture of DME/water (5:1, 22 mL; previously degassed via N<sub>2</sub> sparge) was added, and the mixture was placed in a preheated oil bath at 70 °C for 16 h. The reaction was cooled to ambient temperature, diluted with ethyl acetate, and the aqueous phase separated. The organics were washed with brine and concentrated under reduced pressure. The resultant gum was subject to silica gel chromatography (gradient elution: 2–10% of 2 M NH<sub>3</sub>/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound as a tacky solid (986 mg, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.37 (s, 9H), 1.59–1.79 (m, 6H), 2.43 (br s, 6H), 2.70 (br s, 4H), 3.19 (br s, 4H), 3.41 (br s, 2H), 7.07–7.23 (m, 2H), 7.33–7.47 (m, 2H), 8.19 (d, <i>J</i> = 9.3 Hz, 2H), 8.49 (s, 1H), 8.76 (br s, 1H). LCMS using procedure B: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 518.2. <i>t</i><sub>R</sub> = 0.94 min.</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 5-Cyano-2′-(piperazin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)-[1,1′-biphenyl]-3-carboxamide (<b>62b</b>)</h4><div class="NLM_p last">Trifluoroacetic acid (3 mL, 38.94 mmol) was added to a stirred solution of <i>tert</i>-butyl 4-(3′-cyano-5′-((3-(pyrrolidin-1-yl)propyl)carbamoyl)-[1,1′-biphenyl]-2-yl)piperazine-1-carboxylate <b>62a</b> (960 mg, 1.85 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at ambient temperature. After 45 min, the reaction was concentrated under reduced pressure and the gum reconstituted in CH<sub>2</sub>Cl<sub>2</sub> and concentrated again. Drying in vacuo afforded the title compound as the bis-trifluoroacetate salt (1.725 g). This material was used directly without further purification. LCMS using procedure B: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 418.3. <i>t</i><sub>R</sub> = 0.52 min.</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>tert</i>-Butyl 4-(2-(4-(3-(Pyrrolidin-1-yl)propylcarbamoyl)pyridin-2-yl)phenyl piperazine-1-carboxylate (<b>63a</b>)</h4><div class="NLM_p last">A mixture of 2-bromo-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)iso-nicotinamide <b>52d</b> (2.475 g, 7.93 mmol), 2-[4-(<i>N</i>-Boc)piperazin-1-yl]phenylboronic acid pinacol ester (3.14 g, 8.09 mmol), and K<sub>2</sub>CO<sub>3</sub> (3.29 g, 23.78 mmol) in DME/water (10:1, 40 mL) was charged with Pd(PPh<sub>3</sub>)<sub>4</sub> (0.733 g, 0.63 mmol) under a N<sub>2</sub> sparge. The stirred mixture was heated at 75 °C for 8 h, at which time TLC (10% MeOH + 3% NH<sub>4</sub>OH/EtOAc) revealed that some starting bromide was still present. Additional 2-[4-(<i>N</i>-Boc)piperazin-1-yl]phenylboronic acid pinacol ester <b>61</b> (0.5 equiv), K<sub>2</sub>CO<sub>3</sub> (1.5 equiv), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.04 equiv) were added, and the mixture was set to heat 80 °C for an additional 22 h. The reaction was cooled to room temperature, filtered, and the crude product purified by silica gel chromatography (gradient elution: 1–10% of 1% NH<sub>4</sub>OH/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound as a white waxy solid (2.02 g, 52%). This material was used directly without further characterization.</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 2-(2-Piperazin-1-ylphenyl)-<i>N</i>-(3-pyrrolidin-1-ylpropyl)pyridine-4-carboxamide (<b>63b</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-(2-(4-(3-(pyrrolidin-1-yl)propylcarbamoyl)pyridin-2-yl)phenyl)piperazine-1-carboxylate <b>63a</b> (2.02 g, 4.09 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (25 mL) was added TFA (10 mL). The solution was stirred at room temperature until the complete consumption of the starting material as judged by TLC analysis (15% MeOH/CH<sub>2</sub>Cl<sub>2</sub>). The solution was concentrated under reduced pressure, reconstituted in CH<sub>2</sub>Cl<sub>2</sub>, and concentrated again. The residue was treated with 4 M HCl in dioxane to afford the dihydrochloride salt of the title compound as a tacky gum (2.385 g, quant). This material was taken forward without further purification. LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 394.2. <i>t</i><sub>R</sub> = 1.02 min.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> <i>tert</i>-Butyl 4-(2-(5-(Ethoxycarbonyl)pyridin-3-yl)phenyl)piperazine-1-carboxylate (<b>65</b>)</h4><div class="NLM_p last">A mixture of 3-(ethoxycarbonyl)pyridine-5-boronic acid pinacol ester <b>64</b> (526 mg, 1.90 mmol), <i>tert</i>-butyl 4-(2-bromophenyl)piperazine-1-carboxylate <b>57</b> (648 mg, 1.90 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (928 mg, 2.85 mmol) was stirred in a mixture of dioxane/water (4:1, 15 mL). The stirred solution was degassed by nitrogen (g) sparge at room temperature for 5 min prior to the addition of Pd(PPh<sub>3</sub>)<sub>4</sub> (43.9 mg, 0.04 mmol). The mixture was then heated to 75 °C overnight, cooled to room temperature, and concentrated under reduced pressure. The residue was partitioned between water (10 mL) and CH<sub>2</sub>Cl<sub>2</sub> (30 mL). The organic phase was separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL). The combined organics were washed with water (10 mL) and brine (10 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to dryness. The crude product mixture was purified by silica gel chromatography (gradient elution of 1–5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound as an off-white powder (540 mg, 69%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.40–1.47 (m, 12H), 2.78 (br s, 4H), 3.32 (br s, 4H), 4.45 (q, <i>J</i> = 7.2 Hz, 2H), 7.09–7.14 (m, 1H), 7.15–7.23 (m, 1H), 7.31 (dd, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.39 (td, <i>J</i> = 7.7, 1.8 Hz, 1 H), 8.65 (t, <i>J</i> = 2.2 Hz, 1H), 9.03 (d, <i>J</i> = 2.0 Hz, 1H), 9.16 (d, <i>J</i> = 2.0 Hz, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 422.1. <i>t</i><sub>R</sub> = 3.19 min.</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>tert</i>-Butyl 4-(2-(5-(3-(Pyrrolidin-1-yl)propylcarbamoyl)pyridin-3-yl)phenyl)piperazine-1-carboxylate (<b>66a</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-(2-(5-(ethoxycarbonyl)pyridin-3-yl)phenyl)piperazine-1-carboxylate <b>65</b> (400 mg, 0.97 mmol) and 3-(pyrrolidin-1-yl)propan-1-amine (374 mg, 2.92 mmol) were stirred in toluene (10 mL). A catalytic amount of TBD (∼15 mg) was added, and the reaction was warmed to 90 °C. After 15 h, the solvent was removed under reduced pressure. The residue was purified by silica gel chromatography (5% MeOH in CH<sub>2</sub>Cl<sub>2</sub> as the eluent) to give rise to the title compound as a light-tan solid (450 mg, 94%). <sup>1</sup>H NMR (400 MHz, C<sub>6</sub>D<sub>6</sub>) δ 1.34 (dt, <i>J</i> = 11.5, 5.9 Hz, 3H), 1.41 (s, 9H), 1.46–1.55 (m, 4H), 2.15 (br s, 4H), 2.17–2.24 (m, 2H), 2.38–2.41 (m, 4H), 3.13–3.45 (m, 4H), 3.52–3.59 (m, 2H), 6.62–6.67 (m, 1H), 6.83–6.91 (m, 1H), 7.02–7.08 (m, 2H), 8.82 (t, <i>J</i> = 2.0 Hz, 1H), 8.89 (br s, 1H) 9.08 (d, <i>J</i> = 2.3 Hz, 1H), 9.18 (d, <i>J</i> = 2.0 Hz, 1H). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 493. <i>t</i><sub>R</sub> = 2.03 min.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 5-(2-(Piperazin-1-yl)phenyl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)nicotinamide (<b>66b</b>)</h4><div id="sec4_1_63_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> Method A</h5><div class="NLM_p last">To a stirred solution of <i>tert</i>-butyl 4-(2-(5-(3-(pyrrolidin-1-yl)propylcarbamoyl)pyridin-3-yl)phenyl)piperazine-1-carboxylate <b>66a</b> (400 mg, 0.81 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was added a mixture of TFA/CH<sub>2</sub>Cl<sub>2</sub> (1:1, 6 mL) dropwise at ambient temperature. After 15 h, the solvent was removed in vacuo and the resulting gum was purified by reverse-phase HPLC (C18, gradient elution, 10–75% ACN in water with 0.1% trifluoroacetic acid). The fractions were lyophylized to yield the bis-trifluoroacetate salt of the title compound as a glassy solid (520 mg, quant). This material carried forward without further characterization.</div></div><div id="sec4_1_63_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> Method B</h5><div class="NLM_p last">A mixture of 5-bromo-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)nicotinamide <b>68</b> (365 mg, 1.17 mmol), 2-[4-(<i>N</i>-Boc)piperazin-1-yl]phenylboronic acid pinacol ester <b>61</b> (500 mg, 1.29 mmol), and K<sub>2</sub>CO<sub>3</sub> (485 mg, 3.51 mmol) in DME/water (10:1, 3.3 mL) was treated with Pd(PPh<sub>3</sub>)<sub>4</sub> (135 mg, 0.22 mmol) and set to heat at 70 °C for 16 h. The reaction was cooled to room temperature and partitioned between water (10 mL) and CH<sub>2</sub>Cl<sub>2</sub> (30 mL). The organic phase was separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were dried over MgSO<sub>4</sub>, filtered, and concentrated to dryness. The crude product mixture was purified by silica gel chromatography (gradient elution: 1–10% of 1% NH<sub>4</sub>OH/MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to give rise to the intermediate <i>tert</i>-butyl 4-(2-(5-(3-(pyrrolidin-1-yl)propyl-carbamoyl)pyridin-3-yl)phenyl) piperazine-1-carboxylate <b>66a</b>. The residue was reconstituted in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and treated with TFA (10 mL) at ambient temperature. Concentration under reduced pressure and drying in vacuo afforded the bis-trifluoroacetate salt title compound as a semisolid (727 mg, 99%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 494.4. <i>t</i><sub>R</sub> = 2.14 min.</div></div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 5-Bromo-<i>N</i>-(3-pyrrolidin-1-ylpropyl)pyridine-3-carboxamide (<b>68</b>)</h4><div class="NLM_p last">A mixture of 3-(pyrrolidin-1-yl)propan-1-amine (0.222 mL, 1.56 mmol), 5-bromopyridine-3-carboxylic acid <b>67</b> (286 mg, 1.42 mmol), EDC (326 mg, 1.70 mmol), HOBt (260 mg, 1.70 mmol), and TEA (0.592 mL, 4.25 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was stirred overnight at room temperature. The reaction was further diluted with CH<sub>2</sub>Cl<sub>2</sub> and washed with saturated aq NaHCO<sub>3</sub>. The organics were separated and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by silica gel chromatography (gradient elution: 1–10% of 3% NH<sub>4</sub>OH/MeOH in EtOAc) to afford the title compound (1.31 g, 85%). LCMS using procedure A: MS <i>m</i>/<i>z</i> (ES+) (M + H)<sup>+</sup> = 311.9. <i>t</i><sub>R</sub> = 1.10 min.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> ITC</h3><div class="NLM_p last">ITC experiments were conducted using a Microcal ITC200 at 23 °C in 25 mM Tris (pH 8.5), 50 mM NaCl, and 2% DMSO. Approximately 300 μL of a 15 μM SMYD2 protein solution was loaded into the sample cell, and a 200 μM compound solubilized in DMSO was loaded into the injection syringe. Then eight 4 μL injections of compound were sequentially added to the sample cell, producing a final compound-to-protein molar ratio of approximately 1.4:1. Thermodynamic parameters <i>N</i> (stoichiometry), <i>K</i><sub>a</sub> (association constant), and Δ<i>H</i>° (enthalpic change) were obtained by nonlinear least-squares fitting using Origin software.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Crystallography</h3><div class="NLM_p last">SMYD2 protein expressed and purified as described.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> All compounds were cocrystallized with SMYD2 using previously established crystallization conditions.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Crystals typically appeared within 3 days and achieved their full size in 1 week. All crystals were vitrified in liquid nitrogen using a cryoprotectant solution consisting of a 4:1 ratio of reservoir solution and <span class="smallcaps smallerCapital">l</span>-(+)-2,3-butanediol. All X-ray diffraction data were collected at cryogenic temperature using synchrotron radiation at beamline ID 23.1 at the European Radiation Synchrotron Facility (Grenoble, France) (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf" class="ext-link">Table S1</a>). Diffraction data was processed with XDS and scaled using SCALA<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> as implemented in the autoPROC routines from Global Phasing. These crystals belong to the tetragonal space group <i>I</i>4 and contain one molecule per asymmetric unit. All liganded structures were solved by molecular replacement. The resulting difference density maps unambiguously showed connected difference density with the expected molecular features for these inhibitors. Sequential rounds of model building using Coot and refinement using autoBUSTER<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> produced models that contain all residues except for the first four N-terminal residues. The final refinement statistics are shown in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf" class="ext-link">Table S1</a>. All figures were prepared using PyMOL (The PyMOL Molecular Graphics System, Schrödinger, LLC.).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Cell Assay</h3><div class="NLM_p last">U2OS cells were plated at a density of 8000 cells/well in 96-well plates and transfected using the Fugene system with constructs expressing wild-type SMYD2. Compounds were solubilized in 100% DMSO and diluted with cell culture media (final DMSO concentration of 0.2%) before being added to cells. Cells were incubated for 24 h following transfection, washed, and fixed in 4% PFA, then stained with anti-FLAG Tag antibody at a dilution of 1:800 and a custom polyclonal antibody (21st Century, Marlboro, MA) generated against the monomethylated p53 epitope at a dilution of 1:1000. Cells were stained with DAPI, and images were captured on an Image Express instrument. Fluorescent intensity and cell number was quantified using Accuity and MetaExpress software.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Proliferation Assay</h3><div class="NLM_p last">HCT116 p53+/+, HCT116 p53–/–, RKO-neo, and RKO-E6a cells were seeded in triplicate in 96-well plates and allowed to attach overnight. Media was then replaced with either DMSO control media or media containing compounds <b>1</b>, <b>40</b>, or <b>47</b> at concentrations ranging from 0.03 to 30 μM. Cells were incubated in the presence of compound for 72 h. Viability was determined by measuring fluorescent intensity following incubation with AlamarBlue. IC<sub>50</sub> values were determined as previously described.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Graphical representations were created using Prism.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01303">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17480" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17480" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01303" class="ext-link">10.1021/acs.jmedchem.6b01303</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallographic and refinement statistics, KMT selectivity, preparation of <b>48</b>, preparation of <b>49</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf">jm6b01303_si_001.pdf (214.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_002.csv">jm6b01303_si_002.csv (3.09 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factor amplitudes of the SMYD2 inhibitor complexes have been deposited in the Protein Data Bank with the following accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJM">5KJM</a> (<b>30</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJK">5KJK</a> (<b>48</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJN">5KJN</a> (<b>47</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJL">5KJL</a> (<b>5</b>), and will be released upon publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01303" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70288" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70288" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott D. Cowen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f3909c84969d8090c2b39a909f9c8697dd909c9e"><span class="__cf_email__" data-cfemail="3c5f534b59524f5f0d7c555f50534958125f5351">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew D. Ferguson</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1018-9631" title="Orcid link">http://orcid.org/0000-0002-1018-9631</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#19787d7f7c6b7e597e74787075377a7674"><span class="__cf_email__" data-cfemail="4120252724332601262c20282d6f222e2c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Russell</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leslie A. Dakin</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huawei Chen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas A. Larsen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Godin</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Throner</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaolan Zheng</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Audrey Molina</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiaquan Wu</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tony Cheung</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tina Howard</span> - <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renee Garcia-Arenas</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Keen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Chemistry, ‡Department of Bioscience, Oncology Innovative Medicines Unit, §Structure and Biophysics,
Discovery Sciences, AstraZeneca R&D
Boston, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structure and Biophysics, Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Cheshire, SK10 4TG United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher S. Pendleton</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer A. Pietenpol</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Vanderbilt-Ingram Cancer Center, Vanderbilt University, Nashville, Tennessee 37232 United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): With the exception of Jennifer Pietnepol and Christopher Pendleton, all co-authors are (or were) employees of AstraZeneca.<br /></br></div></li></ul></div><div class="ack" id="ACK-d145e6794-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27245" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27245" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank J. Debreczeni and J. Tucker for their assistance with data collection and E. Code, H. Pollard, and X. Zhu for protein preparation and A. Nielsen for additional chemical synthesis support. We thank the staff of the European Radiation Synchrotron Facility for their assistance. Use of the IMCA-CAT beamline 17-ID at the Advanced Photon Source was supported by the companies of the Industrial Macromolecular Crystallography Association through a contract with Hauptman-Woodward Medical Research Institute. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under contract no. DE-AC02-06CH11357.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i100" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i100"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i101" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i101"> Abbreviations Used</h2><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">adsorption, distribution, metabolism and excretion</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">ACN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DME</td><td class="NLM_def"><p class="first last">dimethoxyethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylacetamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">GMF</td><td class="NLM_def"><p class="first last">glass membrane filter</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">(dimethylamino)-<i>N</i>,<i>N</i>-dimethyl(3<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-3-loxy)methaniminium hexafluoro-phosphate</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">IPA</td><td class="NLM_def"><p class="first last">isopropyl alcohol</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl-2-pyrrolidone</p></td></tr><tr><td class="NLM_term">TBD</td><td class="NLM_def"><p class="first last">1,5,7-triazabicyclo[4.4.0]dec-5-ene</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07107" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07107" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0lhzUtSEn6VwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saloura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1038/nrc3884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnrc3884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25614009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=110-124&author=R.+Hamamotoauthor=V.+Salouraauthor=Y.+Nakamura&title=Critical+roles+of+non-histone+protein+lysine+methylation+in+human+tumorigenesis&doi=10.1038%2Fnrc3884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span></div><div class="casAuthors">Hamamoto, Ryuji; Saloura, Vassiliki; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-124</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several protein lysine methyltransferases and demethylases have been identified to have crit. roles in histone modification.  A large body of evidence has indicated that their dysregulation is involved in the development and progression of various diseases, including cancer, and these enzymes are now considered to be potential therapeutic targets.  Although most studies have focused on histone methylation, many reports have revealed that these enzymes also regulate the methylation dynamics of non-histone proteins such as p53, RB1 and STAT3 (signal transducer and activator of transcription 3), which have important roles in human tumorigenesis.  In this Review, we summarize the mol. functions of protein lysine methylation and its involvement in human cancer, with a particular focus on lysine methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0wn_Jk7jSGbVg90H21EOLACvtfcHk0lhzUtSEn6VwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D&md5=9d998a4efb3c1a069e14747341ceca5e</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrc3884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3884%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DSaloura%26aufirst%3DV.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DCritical%2520roles%2520of%2520non-histone%2520protein%2520lysine%2520methylation%2520in%2520human%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D110%26epage%3D124%26doi%3D10.1038%2Fnrc3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Biggar, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span> </span><span class="NLM_article-title">Non-histone protein methylation as a regulator of cellular signalling and function</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1038/nrm3915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnrm3915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25491103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=5-17&author=K.+K.+Biggarauthor=S.+Li&title=Non-histone+protein+methylation+as+a+regulator+of+cellular+signalling+and+function&doi=10.1038%2Fnrm3915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1cR"><div class="casContent"><span class="casTitleNuber">1c</span><div class="casTitle"><span class="NLM_cas:atitle">Non-histone protein methylation as a regulator of cellular signalling and function</span></div><div class="casAuthors">Biggar, Kyle K.; Li, Shawn S.-C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-17</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Methylation of Lys and Arg residues on non-histone proteins has emerged as a prevalent post-translational modification and as an important regulator of cellular signal transduction mediated by the MAPK, WNT, BMP, Hippo and JAK-STAT signalling pathways.  Crosstalk between methylation and other types of post-translational modifications, and between histone and non-histone protein methylation frequently occurs and affects cellular functions such as chromatin remodelling, gene transcription, protein synthesis, signal transduction and DNA repair.  With recent advances in proteomic techniques, in particular mass spectrometry, the stage is now set to decode the methylproteome and define its functions in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5E-jYGUpGCbVg90H21EOLACvtfcHk0lhzUtSEn6VwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM&md5=3f60179eadac0ad72295b516003ec048</span></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1038%2Fnrm3915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3915%26sid%3Dliteratum%253Aachs%26aulast%3DBiggar%26aufirst%3DK.%2BK.%26aulast%3DLi%26aufirst%3DS.%26atitle%3DNon-histone%2520protein%2520methylation%2520as%2520a%2520regulator%2520of%2520cellular%2520signalling%2520and%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D5%26epage%3D17%26doi%3D10.1038%2Fnrm3915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">The emerging field of dynamic lysine methylation of non-histone proteins</span> <span class="citation_source-journal">Curr. Opin. Genet. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1016/j.gde.2008.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.gde.2008.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=18339539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsFKgs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=152-158&author=J.+Huangauthor=S.+L.+Berger&title=The+emerging+field+of+dynamic+lysine+methylation+of+non-histone+proteins&doi=10.1016%2Fj.gde.2008.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1dR"><div class="casContent"><span class="casTitleNuber">1d</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging field of dynamic lysine methylation of non-histone proteins</span></div><div class="casAuthors">Huang, Jing; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Genetics & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">152-158</span>CODEN:
                <span class="NLM_cas:coden">COGDET</span>;
        ISSN:<span class="NLM_cas:issn">0959-437X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) regulate protein structure and function.  Lys residue methylation abundantly decorates histone proteins and has recently been detected on non-histone proteins.  In particular, tumor suppressor and transcription factor p53 has provided a model for Lys methylation on a non-histone protein.  As found for histones, Lys methylation is dynamic and can be reversed by demethylation.  Lys methylation regulates function via several distinct mechanisms.  Me-Lys residues provide docking sites for the binding of effector proteins.  Methylation can serve to inhibit alternate PTMs on the same Lys residue.  In addn., Lys residues can be monomethylated, dimethylated, or trimethylated, and these levels of methylation correlate with distinct genomic locations and functions.  Taking into account combinatorial activity with numerous other PTMs, Lys residue methylation provides enormous functional diversity and regulatory complexity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNOzzPE2XZ-LVg90H21EOLACvtfcHk0lhzUtSEn6VwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsFKgs74%253D&md5=34b9d26dfcd6708a828042e0df47cb55</span></div><a href="/servlet/linkout?suffix=cit1d&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2008.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2008.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DThe%2520emerging%2520field%2520of%2520dynamic%2520lysine%2520methylation%2520of%2520non-histone%2520proteins%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2008%26volume%3D18%26spage%3D152%26epage%3D158%26doi%3D10.1016%2Fj.gde.2008.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein methylation at the surface and buried deep: thinking outside the histone box</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.1016/j.tibs.2013.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.tibs.2013.02.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=243-252&author=S.+G.+Clarke&title=Protein+methylation+at+the+surface+and+buried+deep%3A+thinking+outside+the+histone+box&doi=10.1016%2Fj.tibs.2013.02.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1e&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2013.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2013.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520methylation%2520at%2520the%2520surface%2520and%2520buried%2520deep%253A%2520thinking%2520outside%2520the%2520histone%2520box%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2013%26volume%3D38%26spage%3D243%26epage%3D252%26doi%3D10.1016%2Fj.tibs.2013.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Komatsu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aiko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshizumi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otsuji, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span> </span><span class="NLM_article-title">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1139</span><span class="NLM_x">–</span> <span class="NLM_lpage">1146</span><span class="refDoi"> DOI: 10.1093/carcin/bgp116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1093%2Fcarcin%2Fbgp116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=19423649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1139-1146&author=S.+Komatsuauthor=I.+Imotoauthor=H.+Tsudaauthor=K.+Kozakiauthor=T.+Muramatsuauthor=Y.+Shimadaauthor=S.+Aikoauthor=Y.+Yoshizumiauthor=D.+Ichikawaauthor=E.+Otsujiauthor=J.+Inazawa&title=Overexpression+of+SMYD2+relates+to+tumor+cell+proliferation+and+malignant+outcome+of+esophageal+squamous+cell+carcinoma&doi=10.1093%2Fcarcin%2Fbgp116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span></div><div class="casAuthors">Komatsu, Shuhei; Imoto, Issei; Tsuda, Hitoshi; Kozaki, Ken-ich; Muramatsu, Tomoki; Shimada, Yutaka; Aiko, Satoshi; Yoshizumi, Yutaka; Ichikawa, Daisuke; Otsuji, Eigo; Inazawa, Johji</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although we have identified two putative targets, ATF3 and CENPF, for a frequently gained/amplified region around 1q32-q41 in esophageal squamous cell carcinoma (ESCC), it is possible that other amplification targets remain to be identified.  In this study, we tested whether SET and MYND domain-contg. protein 2 (SMYD2), located between those two genes and encoding a lysine methyltransferase for histone H3K36 and p53K370 that regulates transcription and inhibits transactivation activity, resp., acts as a cancer-promoting gene through activation/overexpression in ESCC.  Frequent overexpression of SMYD2 mRNA and protein was obsd. in KYSE150 cells with remarkable amplification at 1q32-41.1 and other ESCC cell lines (11/43 lines, 25.6%).  Overexpression of SMYD2 protein was frequently detected in primary tumor samples of ESCC (117/153 cases, 76.5%) as well and significantly correlated with gender, venous invasion, the pT category in the tumor-lymph node-metastases classification and status of recurrence.  Patients with SMYD2-overexpressing tumors had a worse overall rate of survival than those with non-expressing tumors, and SMYD2 positivity was independently assocd. with a worse outcome in the multivariate anal.  Knockdown of SMYD2 expression inhibited and ectopic overexpression of SMYD2 promoted the proliferation of ESCC cells in a TP53 mutation-independent but SMYD2 expression-dependent manner.  These findings suggest that SMYD2 plays an important role in tumor cell proliferation through its activation/overexpression and highlight its usefulness as a prognosticator and potential therapeutic target in ESCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGuuMnL7IjzbVg90H21EOLACvtfcHk0lh0RqIo2ndkvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D&md5=7eb396b256c9ec2b182fd999796527b5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp116%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DS.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DKozaki%26aufirst%3DK.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DAiko%26aufirst%3DS.%26aulast%3DYoshizumi%26aufirst%3DY.%26aulast%3DIchikawa%26aufirst%3DD.%26aulast%3DOtsuji%26aufirst%3DE.%26aulast%3DInazawa%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520SMYD2%2520relates%2520to%2520tumor%2520cell%2520proliferation%2520and%2520malignant%2520outcome%2520of%2520esophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DCarcinogenesis%26date%3D2009%26volume%3D30%26spage%3D1139%26epage%3D1146%26doi%3D10.1093%2Fcarcin%2Fbgp116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Placek, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">Repression of p53 activity by SMYD2 mediated methylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">444</span><span class="NLM_x">, </span> <span class="NLM_fpage">629</span><span class="NLM_x">–</span> <span class="NLM_lpage">632</span><span class="refDoi"> DOI: 10.1038/nature05287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnature05287" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=444&publication_year=2006&pages=629-632&author=J.+Huangauthor=L.+Perez-Burgosauthor=B.+J.+Placekauthor=R.+Senguptaauthor=M.+Richterauthor=J.+A.+Dorseyauthor=S.+Kubicekauthor=S.+Opravilauthor=T.+Jenuweinauthor=S.+L.+Berger&title=Repression+of+p53+activity+by+SMYD2+mediated+methylation&doi=10.1038%2Fnature05287"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature05287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05287%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DPlacek%26aufirst%3DB.%2BJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DRepression%2520of%2520p53%2520activity%2520by%2520SMYD2%2520mediated%2520methylation%26jtitle%3DNature%26date%3D2006%26volume%3D444%26spage%3D629%26epage%3D632%26doi%3D10.1038%2Fnature05287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trescott, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holcomb, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirinupong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span> </span><span class="NLM_article-title">Structural insights into estrogen receptor α methylation by histone methyltransferase SMYD2, a cellular event implicated in estrogen signaling regulation</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">426</span><span class="NLM_x">, </span> <span class="NLM_fpage">3413</span><span class="NLM_x">–</span> <span class="NLM_lpage">3425</span><span class="refDoi"> DOI: 10.1016/j.jmb.2014.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.jmb.2014.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=24594358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Shuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3413-3425&author=Y.+Jiangauthor=L.+Trescottauthor=J.+Holcombauthor=X.+Zhangauthor=J.+Brunzelleauthor=N.+Sirinupongauthor=X.+Shiauthor=Z.+Yang&title=Structural+insights+into+estrogen+receptor+%CE%B1+methylation+by+histone+methyltransferase+SMYD2%2C+a+cellular+event+implicated+in+estrogen+signaling+regulation&doi=10.1016%2Fj.jmb.2014.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into Estrogen Receptor α Methylation by Histone Methyltransferase SMYD2, a Cellular Event Implicated in Estrogen Signaling Regulation</span></div><div class="casAuthors">Jiang, Yuanyuan; Trescott, Laura; Holcomb, Joshua; Zhang, Xi; Brunzelle, Joseph; Sirinupong, Nualpun; Shi, Xiaobing; Yang, Zhe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3413-3425</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Estrogen receptor (ER) signaling plays a pivotal role in many developmental processes and has been implicated in numerous diseases including cancers.  We recently showed that direct ERα methylation by the multi-specificity histone lysine methyltransferase SMYD2 regulates estrogen signaling through repressing ERα-dependent transactivation.  However, the mechanism controlling the specificity of the SMYD2-ERα interaction and the structural basis of SMYD2 substrate binding diversity are unknown.  Here we present the crystal structure of SMYD2 in complex with a target lysine (Lys266)-contg. ERα peptide.  The structure reveals that ERα binds SMYD2 in a U-shaped conformation with the binding specificity detd. mainly by residues C-terminal to the target lysine.  The structure also reveals numerous intrapeptide contacts that ensure shape complementarity between the substrate and the active site of the enzyme, thereby likely serving as an addnl. structural determinant of substrate specificity.  In addn., comparison of the SMYD2-ERα and SMYD2-p53 structures provides the first structural insight into the diverse nature of SMYD2 substrate recognition and suggests that the broad specificity of SMYD2 is achieved by multiple mol. mechanisms such as distinct peptide binding modes and the intrinsic dynamics of peptide ligands.  Strikingly, a novel potentially SMYD2-specific polyethylene glycol binding site is identified in the CTD domain, implicating possible functions in extended substrate binding or protein-protein interactions.  Our study thus provides the structural basis for the SMYD2-mediated ERα methylation, and the resulting knowledge of SMYD2 substrate specificity and target binding diversity could have important implications in selective drug design against a wide range of ERα-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeCcweXaG62LVg90H21EOLACvtfcHk0lh0RqIo2ndkvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Shuro%253D&md5=769725ae4bb4d43bc33fdc717ffad56c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTrescott%26aufirst%3DL.%26aulast%3DHolcomb%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DSirinupong%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DStructural%2520insights%2520into%2520estrogen%2520receptor%2520%25CE%25B1%2520methylation%2520by%2520histone%2520methyltransferase%2520SMYD2%252C%2520a%2520cellular%2520event%2520implicated%2520in%2520estrogen%2520signaling%2520regulation%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3413%26epage%3D3425%26doi%3D10.1016%2Fj.jmb.2014.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakakido, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1016/j.canlet.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.canlet.2014.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=24880080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2014&pages=126-133&author=R.+Hamamotoauthor=R.+Toyokawaauthor=M.+Nakakidoauthor=K.+Uedaauthor=Y.+Nakamura&title=SMYD2-dependent+HSP90+methylation+promotes+cancer+cell+proliferation+by+regulating+the+chaperone+complex+formation&doi=10.1016%2Fj.canlet.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span></div><div class="casAuthors">Hamamoto, Ryuji; Toyokawa, Gouji; Nakakido, Makoto; Ueda, Koji; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-133</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Heat shock protein 90 (HSP90) is a highly conserved mol. chaperone that facilitates the maturation of a wide range of proteins, and it has been recognized as a crucial facilitator of oncogene addiction and cancer cell survival.  Although HSP90 function is regulated by a variety of post-translational modifications, the physiol. significance of methylation has not fully been elucidated.  Here we demonstrate that HSP90AB1 is methylated by the histone methyltransferase SMYD2 and that it plays a crit. role in human carcinogenesis.  HSP90AB1 and SMYD2 can interact through the C-terminal region of HSP90AB1 and the SET domain of SMYD2.  Both in vitro and in vivo methyltransferase assays revealed that SMYD2 could methylate HSP90AB1 and mass spectrometry anal. indicated lysines 531 and 574 of HSP90AB1 to be methylated.  These methylation sites were shown to be important for the dimerization and chaperone complex formation of HSP90AB1.  Furthermore, methylated HSP90AB1 accelerated the proliferation of cancer cells.  Our study reveals a novel mechanism for human carcinogenesis via methylation of HSP90AB1 by SMYD2, and addnl. functional studies may assist in developing novel strategies for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhJR6mltsK1LVg90H21EOLACvtfcHk0ljaCIvItammSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D&md5=c2832d0fd7c14091ed8098a466f13122</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DToyokawa%26aufirst%3DR.%26aulast%3DNakakido%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DSMYD2-dependent%2520HSP90%2520methylation%2520promotes%2520cancer%2520cell%2520proliferation%2520by%2520regulating%2520the%2520chaperone%2520complex%2520formation%26jtitle%3DCancer%2520Lett.%26date%3D2014%26volume%3D351%26spage%3D126%26epage%3D133%26doi%3D10.1016%2Fj.canlet.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Piao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">The histone methyltransferase SMYD2 methylates PARP1 and promotes Poly(ADP-ribosyl)ation activity in cancer cells</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1016/j.neo.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.neo.2014.03.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=257-264&author=L.+Piaoauthor=D.+Kangauthor=T.+Suzukiauthor=A.+Masudaauthor=N.+Dohmaeauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=The+histone+methyltransferase+SMYD2+methylates+PARP1+and+promotes+Poly%28ADP-ribosyl%29ation+activity+in+cancer+cells&doi=10.1016%2Fj.neo.2014.03.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.neo.2014.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neo.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DPiao%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DThe%2520histone%2520methyltransferase%2520SMYD2%2520methylates%2520PARP1%2520and%2520promotes%2520Poly%2528ADP-ribosyl%2529ation%2520activity%2520in%2520cancer%2520cells%26jtitle%3DNeoplasia%26date%3D2014%26volume%3D16%26spage%3D257%26epage%3D264%26doi%3D10.1016%2Fj.neo.2014.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Saddic, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aslanian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, J. R.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span> </span><span class="NLM_article-title">Methylation of the retinoblastoma tumor suppressor by SMYD2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">37733</span><span class="NLM_x">–</span> <span class="NLM_lpage">37740</span><span class="refDoi"> DOI: 10.1074/jbc.M110.137612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1074%2Fjbc.M110.137612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20870719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=37733-37740&author=L.+A.+Saddicauthor=L.+E.+Westauthor=A.+Aslanianauthor=J.+R.+Yatesauthor=S.+M.+Rubinauthor=O.+Gozaniauthor=J.+Sage&title=Methylation+of+the+retinoblastoma+tumor+suppressor+by+SMYD2&doi=10.1074%2Fjbc.M110.137612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of the Retinoblastoma Tumor Suppressor by SMYD2</span></div><div class="casAuthors">Saddic, Louis A.; West, Lisandra E.; Aslanian, Aaron; Yates, John R., III; Rubin, Seth M.; Gozani, Or; Sage, Julien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">37733-37740</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor.  RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells.  Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification.  This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage.  Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1.  These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXUFwS6LhLybVg90H21EOLACvtfcHk0ljaCIvItammSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt7rM&md5=16dd8b7837d4d623dd553f11b0c5e2df</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.137612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.137612%26sid%3Dliteratum%253Aachs%26aulast%3DSaddic%26aufirst%3DL.%2BA.%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DAslanian%26aufirst%3DA.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DRubin%26aufirst%3DS.%2BM.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DMethylation%2520of%2520the%2520retinoblastoma%2520tumor%2520suppressor%2520by%2520SMYD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D37733%26epage%3D37740%26doi%3D10.1074%2Fjbc.M110.137612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">RB1 methylation by SMYD2 enhances cell cycle progression through an increase of RB1 phosphorylation</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">476</span><span class="NLM_x">–</span> <span class="NLM_lpage">486</span><span class="refDoi"> DOI: 10.1593/neo.12656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1593%2Fneo.12656" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=476-486&author=H.+S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=K.+Maejimaauthor=Y.+Yamaneauthor=T.+Suzukiauthor=N.+Dohmaeauthor=M.+Kogureauthor=D.+Kangauthor=D.+E.+Nealauthor=B.+A.+Ponderauthor=H.+Yamaueauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=RB1+methylation+by+SMYD2+enhances+cell+cycle+progression+through+an+increase+of+RB1+phosphorylation&doi=10.1593%2Fneo.12656"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1593%2Fneo.12656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1593%252Fneo.12656%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DRB1%2520methylation%2520by%2520SMYD2%2520enhances%2520cell%2520cycle%2520progression%2520through%2520an%2520increase%2520of%2520RB1%2520phosphorylation%26jtitle%3DNeoplasia%26date%3D2012%26volume%3D14%26spage%3D476%26epage%3D486%26doi%3D10.1593%2Fneo.12656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span> </span><span class="NLM_article-title">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">892</span><span class="NLM_x">–</span> <span class="NLM_lpage">905</span><span class="refDoi"> DOI: 10.1074/mcp.M115.053280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1074%2Fmcp.M115.053280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=26750096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=892-905&author=J.+B.+Olsenauthor=X.+Caoauthor=B.+Hanauthor=L.+H.+Chenauthor=A.+Horvathauthor=T.+I.+Richardsonauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=H.+Nguyen&title=Quantitative+profiling+of+the+activity+of+protein+lysine+methyltransferase+SMYD2+using+SILAC-based+proteomics&doi=10.1074%2Fmcp.M115.053280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span></div><div class="casAuthors">Olsen, Jonathan B.; Cao, Xing-Jun; Han, Bomie; Chen, Lisa Hong; Horvath, Alexander; Richardson, Timothy I.; Campbell, Robert M.; Garcia, Benjamin A.; Nguyen, Hannah</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-905</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The significance of non-histone lysine methylation in cell biol. and human disease is an emerging area of research exploration.  The development of small mol. inhibitors that selectively and potently target enzymes that catalyze the addn. of methyl-groups to lysine residues, such as the protein lysine mono-methyltransferase SMYD2, is an active area of drug discovery.  Crit. to the accurate assessment of biol. function is the ability to identify target enzyme substrates and to define enzyme substrate specificity within the context of the cell.  Here, using stable isotopic labeling with amino acids in cell culture (SILAC) coupled with immunoaffinity enrichment of mono-methyl-lysine (Kme1) peptides and mass spectrometry, we report a comprehensive, large-scale proteomic study of lysine mono-methylation, comprising a total of 1032 Kme1 sites in esophageal squamous cell carcinoma (ESCC) cells and 1861 Kme1 sites in ESCC cells overexpressing SMYD2.  Among these Kme1 sites is a subset of 35 found to be potently down-regulated by both shRNA-mediated knockdown of SMYD2 and LLY-507, a selective small mol. inhibitor of SMYD2.  In addn., we report specific protein sequence motifs enriched in Kme1 sites that are directly regulated by endogenous SMYD2 activity, revealing that SMYD2 substrate specificity is more diverse than expected.  We further show direct activity of SMYD2 toward BTF3-K2, PDAP1-K126 as well as numerous sites within the repetitive units of two unique and exceptionally large proteins, AHNAK and AHNAK2.  Collectively, our findings provide quant. insights into the cellular activity and substrate recognition of SMYD2 as well as the global landscape and regulation of protein mono-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIbAGzEvpHn7Vg90H21EOLACvtfcHk0ljVrMPG-WxzfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D&md5=ebc0c9ca87e73f98b82cb1e3c10d1c71</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M115.053280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M115.053280%26sid%3Dliteratum%253Aachs%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DHorvath%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DT.%2BI.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DNguyen%26aufirst%3DH.%26atitle%3DQuantitative%2520profiling%2520of%2520the%2520activity%2520of%2520protein%2520lysine%2520methyltransferase%2520SMYD2%2520using%2520SILAC-based%2520proteomics%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2016%26volume%3D15%26spage%3D892%26epage%3D905%26doi%3D10.1074%2Fmcp.M115.053280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1016/j.str.2011.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2&doi=10.1016%2Fj.str.2011.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0ljVrMPG-WxzfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273%26doi%3D10.1016%2Fj.str.2011.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonysamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emtage, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gheyi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">LLY-507, a cell-active, potent, and selective inhibitor of protein-lysine methyltransferase SMYD2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">13641</span><span class="NLM_x">–</span> <span class="NLM_lpage">13653</span><span class="refDoi"> DOI: 10.1074/jbc.M114.626861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1074%2Fjbc.M114.626861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25825497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=13641-13653&author=H.+Nguyenauthor=A.+Allali-Hassaniauthor=S.+Antonysamyauthor=S.+Changauthor=L.+H.+Chenauthor=C.+Curtisauthor=S.+Emtageauthor=L.+Fanauthor=T.+Gheyiauthor=F.+Liauthor=S.+Liuauthor=J.+R.+Martinauthor=D.+Mendelauthor=J.+B.+Olsenauthor=L.+Pelletierauthor=T.+Shatsevaauthor=S.+Wuauthor=F.+F.+Zhangauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=D.+Barsyte-Lovejoyauthor=M.+Maderauthor=M.+Vedadi&title=LLY-507%2C+a+cell-active%2C+potent%2C+and+selective+inhibitor+of+protein-lysine+methyltransferase+SMYD2&doi=10.1074%2Fjbc.M114.626861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-507, a Cell-active, Potent, and Selective Inhibitor of Protein-lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Nguyen, Hannah; Allali-Hassani, Abdellah; Antonysamy, Stephen; Chang, Shawn; Chen, Lisa Hong; Curtis, Carmen; Emtage, Spencer; Fan, Li; Gheyi, Tarun; Li, Fengling; Liu, Shichong; Martin, Joseph R.; Mendel, David; Olsen, Jonathan B.; Pelletier, Laura; Shatseva, Tatiana; Wu, Song; Zhang, Feiyu Fred; Arrowsmith, Cheryl H.; Brown, Peter J.; Campbell, Robert M.; Garcia, Benjamin A.; Barsyte-Lovejoy, Dalia; Mader, Mary; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">13641-13653</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 is a lysine methyltransferase that catalyzes the monomethylation of several protein substrates including p53.  SMYD2 is overexpressed in a significant percentage of esophageal squamous primary carcinomas, and that overexpression correlates with poor patient survival.  However, the mechanism(s) by which SMYD2 promotes oncogenesis is not understood.  A small mol. probe for SMYD2 would allow for the pharmacol. dissection of this biol.  In this report, we disclose LLY-507, a cell-active, potent small mol. inhibitor of SMYD2.  LLY-507 is >100-fold selective for SMYD2 over a broad range of methyltransferase and non-methyltransferase targets.  A 1.63-Å resoln. crystal structure of SMYD2 in complex with LLY-507 shows the inhibitor binding in the substrate peptide binding pocket.  LLY-507 is active in cells as measured by redn. of SMYD2-induced monomethylation of p53 Lys370 at submicromolar concns.  We used LLY-507 to further test other potential roles of SMYD2.  Mass spectrometry-based proteomics showed that cellular global histone methylation levels were not significantly affected by SMYD2 inhibition with LLY-507, and subcellular fractionation studies indicate that SMYD2 is primarily cytoplasmic, suggesting that SMYD2 targets a very small subset of histones at specific chromatin loci and/or non-histone substrates.  Breast and liver cancers were identified through in silico data mining as tumor types that display amplification and/or overexpression of SMYD2.  LLY-507 inhibited the proliferation of several esophageal, liver, and breast cancer cell lines in a dose-dependent manner.  These findings suggest that LLY-507 serves as a valuable chem. probe to aid in the dissection of SMYD2 function in cancer and other biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb5tVOe0M2gLVg90H21EOLACvtfcHk0lhmaAX_6VXT2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D&md5=a75f59a900cfe7a6a04d35498caae78b</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626861%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DCurtis%26aufirst%3DC.%26aulast%3DEmtage%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BR.%26aulast%3DMendel%26aufirst%3DD.%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DPelletier%26aufirst%3DL.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%2BF.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DLLY-507%252C%2520a%2520cell-active%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520protein-lysine%2520methyltransferase%2520SMYD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D13641%26epage%3D13653%26doi%3D10.1074%2Fjbc.M114.626861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit10b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algire, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakob, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery of A-893, a new cell-active benzoxazinone inhibitor of lysine methyltransferase SMYD2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">695</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=695-700&author=R.+F.+Sweisauthor=Z.+Wangauthor=M.+Algireauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=G.+G.+Chiangauthor=J.+Guoauthor=C.+G.+Jakobauthor=S.+Kennedyauthor=F.+Liauthor=D.+Maagauthor=B.+Shawauthor=N.+B.+Soniauthor=M.+Vedadiauthor=W.+N.+Pappano&title=Discovery+of+A-893%2C+a+new+cell-active+benzoxazinone+inhibitor+of+lysine+methyltransferase+SMYD2&doi=10.1021%2Facsmedchemlett.5b00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of A-893, A New Cell-Active Benzoxazinone Inhibitor of Lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Sweis, Ramzi F.; Wang, Zhi; Algire, Mikkel; Arrowsmith, Cheryl H.; Brown, Peter J.; Chiang, Gary G.; Guo, Jun; Jakob, Clarissa G.; Kennedy, Steven; Li, Fengling; Maag, David; Shaw, Bailin; Soni, Nirupama B.; Vedadi, Masoud; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">695-700</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A lack of useful small mol. tools has precluded thorough interrogation of the biol. function of SMYD2, a lysine methyltransferase with known tumor-suppressor substrates.  Systematic exploration of the structure-activity relationships of a previously known benzoxazinone compd. led to the synthesis of A-893 I, a potent and selective SMYD2 inhibitor (IC50: 2.8 nM).  A cocrystal structure reveals the origin of enhanced potency, and effective suppression of p53K370 methylation is obsd. in a lung carcinoma (A549) cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI0i3TmEl2DLVg90H21EOLACvtfcHk0lhmaAX_6VXT2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D&md5=4e37d0d524f1b1066a4ef7ba03dc4695</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00124%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520of%2520A-893%252C%2520a%2520new%2520cell-active%2520benzoxazinone%2520inhibitor%2520of%2520lysine%2520methyltransferase%2520SMYD2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D695%26epage%3D700%26doi%3D10.1021%2Facsmedchemlett.5b00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit10c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Eggert, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillig, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohr, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockigt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiske, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barak, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mowat, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brumby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christ, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ter Laak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Montalvan, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badock, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartung, I. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stellfeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stresemann, C.</span><span> </span><span class="NLM_article-title">Discovery and characterization of a highly potent and selective aminopyrazoline-based in vivo probe (BAY-598) for the protein lysine methyltransferase SMYD2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4578</span><span class="NLM_x">–</span> <span class="NLM_lpage">4600</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01890</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01890" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVGmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4578-4600&author=E.+Eggertauthor=R.+C.+Hilligauthor=S.+Kohrauthor=D.+Stockigtauthor=J.+Weiskeauthor=N.+Barakauthor=J.+Mowatauthor=T.+Brumbyauthor=C.+D.+Christauthor=A.+Ter+Laakauthor=T.+Langauthor=A.+E.+Fernandez-Montalvanauthor=V.+Badockauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=D.+Barsyte-Lovejoyauthor=M.+Szewczykauthor=S.+Kennedyauthor=F.+Liauthor=M.+Vedadiauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=C.+H.+Arrowsmithauthor=T.+Stellfeldauthor=C.+Stresemann&title=Discovery+and+characterization+of+a+highly+potent+and+selective+aminopyrazoline-based+in+vivo+probe+%28BAY-598%29+for+the+protein+lysine+methyltransferase+SMYD2&doi=10.1021%2Facs.jmedchem.5b01890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10cR"><div class="casContent"><span class="casTitleNuber">10c</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Eggert, Erik; Hillig, Roman C.; Koehr, Silke; Stoeckigt, Detlef; Weiske, Joerg; Barak, Naomi; Mowat, Jeffrey; Brumby, Thomas; Christ, Clara D.; ter Laak, Antonius; Lang, Tina; Fernandez-Montalvan, Amaury E.; Badock, Volker; Weinmann, Hilmar; Hartung, Ingo V.; Barsyte-Lovejoy, Dalia; Szewczyk, Magdalena; Kennedy, Steven; Li, Fengling; Vedadi, Masoud; Brown, Peter J.; Santhakumar, Vijayaratnam; Arrowsmith, Cheryl H.; Stellfeld, Timo; Stresemann, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4578-4600</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways.  SET and MYND domain contg. protein 2 (SMYD2) is a catalytic SET domain contg. methyltransferase reported to monomethylate lysine residues on histone and nonhistone proteins.  Although several studies have uncovered an important role of SMYD2 in promoting cancer by protein methylation, the biol. of SMYD2 is far from being fully understood.  Utilization of highly potent and selective chem. probes for target validation has emerged as a concept which circumvents possible limitations of knockdown expts. and, in particular, could result in an improved exploration of drug targets with a complex underlying biol.  Here, we report the development of a potent, selective, and cell-active, substrate-competitive inhibitor of SMYD2, which is the first reported inhibitor suitable for in vivo target validation studies in rodents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAuNehWDd3N7Vg90H21EOLACvtfcHk0lgiWLbTEYBCDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVGmsLc%253D&md5=d0944b0a6870b9790f9f4c429290f81e</span></div><a href="/servlet/linkout?suffix=cit10c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01890%26sid%3Dliteratum%253Aachs%26aulast%3DEggert%26aufirst%3DE.%26aulast%3DHillig%26aufirst%3DR.%2BC.%26aulast%3DKohr%26aufirst%3DS.%26aulast%3DStockigt%26aufirst%3DD.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DBarak%26aufirst%3DN.%26aulast%3DMowat%26aufirst%3DJ.%26aulast%3DBrumby%26aufirst%3DT.%26aulast%3DChrist%26aufirst%3DC.%2BD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DLang%26aufirst%3DT.%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DStellfeld%26aufirst%3DT.%26aulast%3DStresemann%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520highly%2520potent%2520and%2520selective%2520aminopyrazoline-based%2520in%2520vivo%2520probe%2520%2528BAY-598%2529%2520for%2520the%2520protein%2520lysine%2520methyltransferase%2520SMYD2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4578%26epage%3D4600%26doi%3D10.1021%2Facs.jmedchem.5b01890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theriault, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Code, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birr, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span> </span><span class="NLM_article-title">Biochemical characterization of human SET and MYND domain-containing protein 2 methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">6488</span><span class="NLM_x">–</span> <span class="NLM_lpage">6497</span><span class="refDoi"> DOI: 10.1021/bi200725p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200725p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6488-6497&author=J.+Wuauthor=T.+Cheungauthor=C.+Grandeauthor=A.+D.+Fergusonauthor=X.+Zhuauthor=K.+Theriaultauthor=E.+Codeauthor=C.+Birrauthor=N.+Keenauthor=H.+Chen&title=Biochemical+characterization+of+human+SET+and+MYND+domain-containing+protein+2+methyltransferase&doi=10.1021%2Fbi200725p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fbi200725p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200725p%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DTheriault%26aufirst%3DK.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DBirr%26aufirst%3DC.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DBiochemical%2520characterization%2520of%2520human%2520SET%2520and%2520MYND%2520domain-containing%2520protein%25202%2520methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D6488%26epage%3D6497%26doi%3D10.1021%2Fbi200725p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.jmb.2010.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.jmb.2010.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20434463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases&doi=10.1016%2Fj.jmb.2010.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Adding a Lysine Mimic in the Design of Potent Inhibitors of Histone Lysine Methyltransferases</span></div><div class="casAuthors">Chang, Yanqi; Ganesh, Thota; Horton, John R.; Spannhoff, Astrid; Liu, Jin; Sun, Aiming; Zhang, Xing; Bedford, Mark T.; Shinkai, Yoichi; Snyder, James P.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dynamic histone lysine methylation involves the activities of modifying enzymes (writers), enzymes removing modifications (erasers), and readers of the histone code.  One common feature of these activities is the recognition of lysines in methylated and unmethylated states, whether they are substrates, reaction products, or binding partners.  The authors applied the concept of adding a lysine mimic to an established inhibitor (BIX-01294) of histone H3 lysine 9 methyltransferases G9a and G9a-like protein by including a 5-aminopentyloxy moiety, which is inserted into the target lysine-binding channel and becomes methylated by G9a-like protein, albeit slowly.  The compd. enhances its potency in vitro and reduces cell toxicity in vivo.  The authors suggest that adding a lysine or methyl-lysine mimic should be considered in the design of small-mol. inhibitors for other methyl-lysine writers, erasers, and readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsrNWhUZuRrLVg90H21EOLACvtfcHk0lgiWLbTEYBCDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D&md5=7ec7ce81db069cd6c1c6637b3ad57007</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.jmb.2010.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span><span class="refDoi"> DOI: 10.1021/jm901543m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a&doi=10.1021%2Fjm901543m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lhk5A1_TPHPHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953%26doi%3D10.1021%2Fjm901543m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span> </span><span class="NLM_article-title">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">971</span><span class="NLM_x">–</span> <span class="NLM_lpage">980</span><span class="refDoi"> DOI: 10.1111/cbdd.12050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1111%2Fcbdd.12050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=22978415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2012&pages=971-980&author=A.+Basavapathruniauthor=L.+Jinauthor=S.+R.+Daigleauthor=C.+R.+Majerauthor=C.+A.+Therkelsenauthor=T.+J.+Wigleauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=R.+M.+Pollockauthor=M.+P.+Scottauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=E.+J.+Olhava&title=Conformational+adaptation+drives+potent%2C+selective+and+durable+inhibition+of+the+human+protein+methyltransferase+DOT1L&doi=10.1111%2Fcbdd.12050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span></div><div class="casAuthors">Basavapathruni, Aravind; Jin, Lei; Daigle, Scott R.; Majer, Christina R. A.; Therkelsen, Carly A.; Wigle, Tim J.; Kuntz, Kevin W.; Chesworth, Richard; Pollock, Roy M.; Scott, Margaret P.; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Olhava, Edward J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">971-980</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">DOT1L is the human protein methyltransferase responsible for catalyzing the methylation of histone H3 on lysine 79 (H3K79).  The ectopic activity of DOT1L, assocd. with the chromosomal translocation that is a universal hallmark of MLL-rearranged leukemia, is a required driver of leukemogenesis in this malignancy.  Here, we present studies on the structure-activity relation of aminonucleoside-based DOT1L inhibitors.  Within this series, we find that improvements in target enzyme affinity and selectivity are driven entirely by diminution of the dissocn. rate const. for the enzyme-inhibitor complex, leading to long residence times for the binary complex.  The biochem. Ki and residence times measured for these inhibitors correlate well with their effects on intracellular H3K79 methylation and MLL-rearranged leukemic cell killing.  Crystallog. studies reveal a conformational adaptation mechanism assocd. with high-affinity inhibitor binding and prolonged residence time; these studies also suggest that conformational adaptation likewise plays a crit. role in natural ligand interactions with the enzyme, hence, facilitating enzyme turnover.  These results provide crit. insights into the role of conformational adaptation in the enzymic mechanism of catalysis and in pharmacol. intervention for DOT1L and other members of this enzyme class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJm_d3iX3p77Vg90H21EOLACvtfcHk0lhk5A1_TPHPHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O&md5=de628fcd2c37b5f6496cc3b483125309</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12050%26sid%3Dliteratum%253Aachs%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26atitle%3DConformational%2520adaptation%2520drives%2520potent%252C%2520selective%2520and%2520durable%2520inhibition%2520of%2520the%2520human%2520protein%2520methyltransferase%2520DOT1L%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2012%26volume%3D80%26spage%3D971%26epage%3D980%26doi%3D10.1111%2Fcbdd.12050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+O.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lhk5A1_TPHPHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BO.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Bunz, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutriaux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedivy, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span> </span><span class="NLM_article-title">Requirement for p53 and p21 to sustain G2 arrest after DNA damage</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1501</span><span class="refDoi"> DOI: 10.1126/science.282.5393.1497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1126%2Fscience.282.5393.1497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=9822382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADyaK1cXns1yqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1998&pages=1497-1501&author=F.+Bunzauthor=A.+Dutriauxauthor=C.+Lengauerauthor=T.+Waldmanauthor=S.+Zhouauthor=J.+P.+Brownauthor=J.+M.+Sedivyauthor=K.+W.+Kinzlerauthor=B.+Vogelstein&title=Requirement+for+p53+and+p21+to+sustain+G2+arrest+after+DNA+damage&doi=10.1126%2Fscience.282.5393.1497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement for p53 and p21 to sustain G2 arrest after DNA damage</span></div><div class="casAuthors">Bunz, F.; Dutriaux, A.; Lengauer, C.; Waldman, T.; Zhou, S.; Brown, J. P.; Sedivy, J. M.; Kinzler, K. W.; Vogelstein, B.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">5393</span>),
    <span class="NLM_cas:pages">1497-1501</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">After DNA damage, many cells appear to enter a sustained arrest in the G2 phase of the cell cycle.  It is shown here that this arrest could be sustained only when p53 was present in the cell and capable of transcriptionally activating the cyclin-dependent kinase inhibitor p21.  After disruption of either the p53 or the p21 gene, γ radiated cells progressed into mitosis and exhibited a G2 DNA content only because of a failure of cytokinesis.  Thus, p53 and p21 appear to be essential for maintaining the G2 checkpoint in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO6NPwYqRmK7Vg90H21EOLACvtfcHk0lhOmceE4PzzJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1yqurc%253D&md5=e6f0dc856f71915dbd2b8cfb11de4bad</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1126%2Fscience.282.5393.1497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.282.5393.1497%26sid%3Dliteratum%253Aachs%26aulast%3DBunz%26aufirst%3DF.%26aulast%3DDutriaux%26aufirst%3DA.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DWaldman%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DJ.%2BP.%26aulast%3DSedivy%26aufirst%3DJ.%2BM.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26aulast%3DVogelstein%26aufirst%3DB.%26atitle%3DRequirement%2520for%2520p53%2520and%2520p21%2520to%2520sustain%2520G2%2520arrest%2520after%2520DNA%2520damage%26jtitle%3DScience%26date%3D1998%26volume%3D282%26spage%3D1497%26epage%3D1501%26doi%3D10.1126%2Fscience.282.5393.1497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Smith, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fornace, A. J.,  Jr.</span><span> </span><span class="NLM_article-title">Involvement of the p53 tumor suppressor in repair of u.v.-type DNA damage</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1053</span><span class="NLM_x">–</span> <span class="NLM_lpage">1059</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1995&pages=1053-1059&author=M.+L.+Smithauthor=I.+T.+Chenauthor=Q.+Zhanauthor=P.+M.+O%E2%80%99Connorauthor=A.+J.+Fornace&title=Involvement+of+the+p53+tumor+suppressor+in+repair+of+u.v.-type+DNA+damage"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DChen%26aufirst%3DI.%2BT.%26aulast%3DZhan%26aufirst%3DQ.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DP.%2BM.%26aulast%3DFornace%26aufirst%3DA.%2BJ.%26atitle%3DInvolvement%2520of%2520the%2520p53%2520tumor%2520suppressor%2520in%2520repair%2520of%2520u.v.-type%2520DNA%2520damage%26jtitle%3DOncogene%26date%3D1995%26volume%3D10%26spage%3D1053%26epage%3D1059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit14c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Bauer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehmann, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendleton, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shyr, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arteaga, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietenpol, J. A.</span><span> </span><span class="NLM_article-title">RNA interference (RNAi) screening approach identifies agents that enhance paclitaxel activity in breast cancer cells</span> <span class="citation_source-journal">Breast Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">R41</span><span class="refDoi"> DOI: 10.1186/bcr2595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1186%2Fbcr2595" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=R41&author=J.+A.+Bauerauthor=F.+Yeauthor=C.+B.+Marshallauthor=B.+D.+Lehmannauthor=C.+S.+Pendletonauthor=Y.+Shyrauthor=C.+L.+Arteagaauthor=J.+A.+Pietenpol&title=RNA+interference+%28RNAi%29+screening+approach+identifies+agents+that+enhance+paclitaxel+activity+in+breast+cancer+cells&doi=10.1186%2Fbcr2595"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14c&amp;dbid=16384&amp;doi=10.1186%2Fbcr2595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2595%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DJ.%2BA.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DMarshall%26aufirst%3DC.%2BB.%26aulast%3DLehmann%26aufirst%3DB.%2BD.%26aulast%3DPendleton%26aufirst%3DC.%2BS.%26aulast%3DShyr%26aufirst%3DY.%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DPietenpol%26aufirst%3DJ.%2BA.%26atitle%3DRNA%2520interference%2520%2528RNAi%2529%2520screening%2520approach%2520identifies%2520agents%2520that%2520enhance%2520paclitaxel%2520activity%2520in%2520breast%2520cancer%2520cells%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2010%26volume%3D12%26spage%3DR41%26doi%3D10.1186%2Fbcr2595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sajjad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novoyatleva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vergarajauregui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troidl, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schermuly, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, H. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, F. B.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1843</span><span class="NLM_x">, </span> <span class="NLM_fpage">2556</span><span class="NLM_x">–</span> <span class="NLM_lpage">2562</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.bbamcr.2014.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25014164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=2556-2562&author=A.+Sajjadauthor=T.+Novoyatlevaauthor=S.+Vergarajaureguiauthor=C.+Troidlauthor=R.+T.+Schermulyauthor=H.+O.+Tuckerauthor=F.+B.+Engel&title=Lysine+methyltransferase+Smyd2+suppresses+p53-dependent+cardiomyocyte+apoptosis&doi=10.1016%2Fj.bbamcr.2014.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span></div><div class="casAuthors">Sajjad, Amna; Novoyatleva, Tatyana; Vergarajauregui, Silvia; Troidl, Christian; Schermuly, Ralph T.; Tucker, Haley O.; Engel, Felix B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2556-2562</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apoptosis, or programmed cell death, is an essential physiol. process for proper embryogenesis as well as for homeostasis during aging.  In addn., apoptosis is one of the major mechanisms causing cell loss in pathophysiol. conditions such as heart failure.  Thus, inhibition of apoptosis is an important approach for preventive and therapeutic strategies.  Here we show that the histone 3 lysine 4- and lysine 36-specific methyltransferase Smyd2 acts as an endogenous antagonistic player of p53-dependent cardiomyocyte apoptosis.  Smyd2 protein levels were significantly decreased in cardiomyocytes upon cobalt chloride-induced apoptosis or myocardial infarction, while p53 expression was enhanced.  SiRNA-mediated knockdown of Smyd2 in cultured cardiomyocytes further enhanced cobalt chloride-induced cardiomyocyte apoptosis.  In contrast, Smyd2 overexpression resulted in marked methylation of p53 and prevented its accumulation as well as apoptotic cell death in an Hsp90-independent manner.  Moreover, overexpression, of Smyd2, but not Smyd2Y240F lacking a Me transferase activity, significantly rescued CoCl2-induced apoptosis in H9c2 cardioblasts.  Finally, Smyd2 cardiomyocyte-specific deletion in vivo promoted apoptotic cell death upon myocardial infarction, which correlated with enhanced expression of p53 and pro-apoptotic Bax.  Collectively, our data indicate Smyd2 as a cardioprotective protein by methylating p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfw3CFurADyLVg90H21EOLACvtfcHk0lhOmceE4PzzJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J&md5=327a9367f05026a05ec527e18f71b26f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DSajjad%26aufirst%3DA.%26aulast%3DNovoyatleva%26aufirst%3DT.%26aulast%3DVergarajauregui%26aufirst%3DS.%26aulast%3DTroidl%26aufirst%3DC.%26aulast%3DSchermuly%26aufirst%3DR.%2BT.%26aulast%3DTucker%26aufirst%3DH.%2BO.%26aulast%3DEngel%26aufirst%3DF.%2BB.%26atitle%3DLysine%2520methyltransferase%2520Smyd2%2520suppresses%2520p53-dependent%2520cardiomyocyte%2520apoptosis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D2556%26epage%3D2562%26doi%3D10.1016%2Fj.bbamcr.2014.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Chebanov, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakhno, Y. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desenko, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernenko, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musatov, V. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shishkina, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shishkin, O. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kappe, C. O.</span><span> </span><span class="NLM_article-title">Cyclocondensation reactions of 5-aminopyrazoles, pyruvic acids and aldehydes. Multicomponent approaches to pyrazolopyridines and related products</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">1229</span><span class="NLM_x">–</span> <span class="NLM_lpage">1242</span><span class="refDoi"> DOI: 10.1016/j.tet.2006.11.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.tet.2006.11.048" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=1229-1242&author=V.+A.+Chebanovauthor=Y.+I.+Sakhnoauthor=S.+M.+Desenkoauthor=V.+N.+Chernenkoauthor=V.+I.+Musatovauthor=S.+V.+Shishkinaauthor=O.+V.+Shishkinauthor=C.+O.+Kappe&title=Cyclocondensation+reactions+of+5-aminopyrazoles%2C+pyruvic+acids+and+aldehydes.+Multicomponent+approaches+to+pyrazolopyridines+and+related+products&doi=10.1016%2Fj.tet.2006.11.048"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2006.11.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2006.11.048%26sid%3Dliteratum%253Aachs%26aulast%3DChebanov%26aufirst%3DV.%2BA.%26aulast%3DSakhno%26aufirst%3DY.%2BI.%26aulast%3DDesenko%26aufirst%3DS.%2BM.%26aulast%3DChernenko%26aufirst%3DV.%2BN.%26aulast%3DMusatov%26aufirst%3DV.%2BI.%26aulast%3DShishkina%26aufirst%3DS.%2BV.%26aulast%3DShishkin%26aufirst%3DO.%2BV.%26aulast%3DKappe%26aufirst%3DC.%2BO.%26atitle%3DCyclocondensation%2520reactions%2520of%25205-aminopyrazoles%252C%2520pyruvic%2520acids%2520and%2520aldehydes.%2520Multicomponent%2520approaches%2520to%2520pyrazolopyridines%2520and%2520related%2520products%26jtitle%3DTetrahedron%26date%3D2007%26volume%3D63%26spage%3D1229%26epage%3D1242%26doi%3D10.1016%2Fj.tet.2006.11.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Hardy, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sang, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpkin, D. S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Follenfant, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, T. W.</span><span> </span><span class="NLM_article-title">Peripherally acting enkephalin analogues. Polar tri- and tetrapeptides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">1108</span><span class="NLM_x">–</span> <span class="NLM_lpage">1118</span><span class="refDoi"> DOI: 10.1021/jm00125a028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00125a028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1989&pages=1108-1118&author=G.+W.+Hardyauthor=L.+A.+Loweauthor=G.+Millsauthor=P.+Y.+Sangauthor=D.+S.+A.+Simpkinauthor=R.+L.+Follenfantauthor=C.+Shankleyauthor=T.+W.+Smith&title=Peripherally+acting+enkephalin+analogues.+Polar+tri-+and+tetrapeptides&doi=10.1021%2Fjm00125a028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm00125a028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00125a028%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DG.%2BW.%26aulast%3DLowe%26aufirst%3DL.%2BA.%26aulast%3DMills%26aufirst%3DG.%26aulast%3DSang%26aufirst%3DP.%2BY.%26aulast%3DSimpkin%26aufirst%3DD.%2BS.%2BA.%26aulast%3DFollenfant%26aufirst%3DR.%2BL.%26aulast%3DShankley%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DT.%2BW.%26atitle%3DPeripherally%2520acting%2520enkephalin%2520analogues.%2520Polar%2520tri-%2520and%2520tetrapeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1989%26volume%3D32%26spage%3D1108%26epage%3D1118%26doi%3D10.1021%2Fjm00125a028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Kiesewetter, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholten, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedrick, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waymouth, R. M.</span><span> </span><span class="NLM_article-title">Cyclic guanidine organic catalysts: What is magic about triazabicyclodecene?</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">9490</span><span class="NLM_x">–</span> <span class="NLM_lpage">9496</span><span class="refDoi"> DOI: 10.1021/jo902369g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo902369g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKhs7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2009&pages=9490-9496&author=M.+K.+Kiesewetterauthor=M.+D.+Scholtenauthor=N.+Kirnauthor=R.+L.+Weberauthor=J.+L.+Hedrickauthor=R.+M.+Waymouth&title=Cyclic+guanidine+organic+catalysts%3A+What+is+magic+about+triazabicyclodecene%3F&doi=10.1021%2Fjo902369g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic guanidine organic catalysts: What is magic about triazabicyclodecene?</span></div><div class="casAuthors">Kiesewetter, Matthew K.; Scholten, Marc D.; Kirn, Nicole; Weber, Ryan L.; Hedrick, James L.; Waymouth, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9490-9496</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bicyclic guanidine 1,5,7-triazabicyclo[4.4.0]dec-5-ene (TBD) is an effective organocatalyst for the formation of amides from esters and primary amines.  Mechanistic and kinetic studies support a nucleophilic mechanism where TBD reacts reversibly with esters to generate an acyl-TBD intermediate that acylates amines to generate the amides.  Comparative studies of the analogous bicyclic guanidine 1,4,6-triazabicyclo[3.3.0]oct-4-ene (TBO) reveal it to be a much less active acylation catalyst than TBD.  Theor. and mechanistic studies imply that the higher reactivity of TBD is a consequence of both its higher basicity and nucleophilicity than TBO and the high reactivity of the acyl-TBD intermediate, which is sterically prevented from adopting a planar amide structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9dztZtVWNcrVg90H21EOLACvtfcHk0lgUTmxgN9RHsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKhs7nP&md5=0f263f5a0f48749b17ed6bbd486e7f32</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjo902369g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo902369g%26sid%3Dliteratum%253Aachs%26aulast%3DKiesewetter%26aufirst%3DM.%2BK.%26aulast%3DScholten%26aufirst%3DM.%2BD.%26aulast%3DKirn%26aufirst%3DN.%26aulast%3DWeber%26aufirst%3DR.%2BL.%26aulast%3DHedrick%26aufirst%3DJ.%2BL.%26aulast%3DWaymouth%26aufirst%3DR.%2BM.%26atitle%3DCyclic%2520guanidine%2520organic%2520catalysts%253A%2520What%2520is%2520magic%2520about%2520triazabicyclodecene%253F%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2009%26volume%3D74%26spage%3D9490%26epage%3D9496%26doi%3D10.1021%2Fjo902369g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Cowen, S. D.</span><span> </span><span class="NLM_article-title">Targeting the substrate binding site of methyltransferases: structure-based design of SMYD2 inhibitors</span>. EpiCongress 2013, Boston, MA, July 23–24, 2013,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=S.+D.+Cowen&title=Targeting+the+substrate+binding+site+of+methyltransferases%3A+structure-based+design+of+SMYD2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DCowen%26aufirst%3DS.%2BD.%26atitle%3DTargeting%2520the%2520substrate%2520binding%2520site%2520of%2520methyltransferases%253A%2520structure-based%2520design%2520of%2520SMYD2%2520inhibitors%26jtitle%3DEpiCongress%25202013%252C%2520Boston%252C%2520MA%252C%2520July%252023%25E2%2580%259324%252C%25202013%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit19b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Throner, S.; Cowen, S.; Russell, D. J.; Dakin, L.; Chen, H.; Larsen, N. A.; Godin, R. E.; Zheng, X.; Molina, A.; Wu, J.; Cheung, T.; Howard, T.; Garcia-Arenas, R.; Keen, N.; Ferguson, A. D.</span>Abstracts of Papers, 250th National Meeting of the American Chemical Society, Boston, MA, Aug 16–20, 2015; <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington, DC</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span> <span class="NLM_fpage">MEDI 513</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=MEDI+513&author=S.+Throner&author=S.+Cowen&author=D.+J.+Russell&author=L.+Dakin&author=H.+Chen&author=N.+A.+Larsen&author=R.+E.+Godin&author=X.+Zheng&author=A.+Molina&author=J.+Wu&author=T.+Cheung&author=T.+Howard&author=R.+Garcia-Arenas&author=N.+Keen&author=A.+D.+Ferguson&title=Abstracts+of+Papers%2C+250th+National+Meeting+of+the+American+Chemical+Society%2C+Boston%2C+MA%2C+Aug+16%E2%80%9320%2C+2015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DThroner%26aufirst%3DS.%26jtitle%3DAbstracts%2520of%2520Papers%252C%2520250th%2520National%2520Meeting%2520of%2520the%2520American%2520Chemical%2520Society%252C%2520Boston%252C%2520MA%252C%2520Aug%252016%25E2%2580%259320%252C%25202015%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2015%26spage%3DMEDI%2520513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Reynoird, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazur, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stellfeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flores, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lofgren, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hainaut, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaznowska, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khatri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stresemann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span> </span><span class="NLM_article-title">Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.1101/gad.275529.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1101%2Fgad.275529.115" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=772-785&author=N.+Reynoirdauthor=P.+K.+Mazurauthor=T.+Stellfeldauthor=N.+M.+Floresauthor=S.+M.+Lofgrenauthor=S.+M.+Carlsonauthor=E.+Brambillaauthor=P.+Hainautauthor=E.+B.+Kaznowskaauthor=C.+H.+Arrowsmithauthor=P.+Khatriauthor=C.+Stresemannauthor=O.+Gozaniauthor=J.+Sage&title=Coordination+of+stress+signals+by+the+lysine+methyltransferase+SMYD2+promotes+pancreatic+cancer&doi=10.1101%2Fgad.275529.115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1101%2Fgad.275529.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.275529.115%26sid%3Dliteratum%253Aachs%26aulast%3DReynoird%26aufirst%3DN.%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DStellfeld%26aufirst%3DT.%26aulast%3DFlores%26aufirst%3DN.%2BM.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DHainaut%26aufirst%3DP.%26aulast%3DKaznowska%26aufirst%3DE.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKhatri%26aufirst%3DP.%26aulast%3DStresemann%26aufirst%3DC.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DCoordination%2520of%2520stress%2520signals%2520by%2520the%2520lysine%2520methyltransferase%2520SMYD2%2520promotes%2520pancreatic%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2016%26volume%3D30%26spage%3D772%26epage%3D785%26doi%3D10.1101%2Fgad.275529.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Smid, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coolen, H. K. A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keizer, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Moes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Hartog, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stork, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plekkenpol, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niemann, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroomer, C. N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Stuivenberg, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCreary, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hesselink, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herremans, A. H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, C. G.</span><span> </span><span class="NLM_article-title">Synthesis, structure-activity relationships, and biological properties of 1-heteroaryl-4-[w-(1H-indol-3-yl)alkyl] piperazines, novel potential antipsychotics combining potent dopamine D<sub>2</sub> receptor antagonism with potent serotonin reuptake inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6855</span><span class="NLM_x">–</span> <span class="NLM_lpage">6869</span><span class="refDoi"> DOI: 10.1021/jm050148z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050148z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6855-6869&author=P.+Smidauthor=H.+K.+A.+C.+Coolenauthor=H.+G.+Keizerauthor=R.+van+Hesauthor=J.+de+Moesauthor=A.+P.+den+Hartogauthor=B.+Storkauthor=R.+H.+Plekkenpolauthor=L.+C.+Niemannauthor=C.+N.+J.+Stroomerauthor=M.+Tulpauthor=H.+H.+van+Stuivenbergauthor=A.+C.+McCrearyauthor=M.+B.+Hesselinkauthor=A.+H.+J.+Herremansauthor=C.+G.+Kruse&title=Synthesis%2C+structure-activity+relationships%2C+and+biological+properties+of+1-heteroaryl-4-%5Bw-%281H-indol-3-yl%29alkyl%5D+piperazines%2C+novel+potential+antipsychotics+combining+potent+dopamine+D2+receptor+antagonism+with+potent+serotonin+reuptake+inhibition&doi=10.1021%2Fjm050148z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm050148z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050148z%26sid%3Dliteratum%253Aachs%26aulast%3DSmid%26aufirst%3DP.%26aulast%3DCoolen%26aufirst%3DH.%2BK.%2BA.%2BC.%26aulast%3DKeizer%26aufirst%3DH.%2BG.%26aulast%3Dvan%2BHes%26aufirst%3DR.%26aulast%3Dde%2BMoes%26aufirst%3DJ.%26aulast%3Dden%2BHartog%26aufirst%3DA.%2BP.%26aulast%3DStork%26aufirst%3DB.%26aulast%3DPlekkenpol%26aufirst%3DR.%2BH.%26aulast%3DNiemann%26aufirst%3DL.%2BC.%26aulast%3DStroomer%26aufirst%3DC.%2BN.%2BJ.%26aulast%3DTulp%26aufirst%3DM.%26aulast%3Dvan%2BStuivenberg%26aufirst%3DH.%2BH.%26aulast%3DMcCreary%26aufirst%3DA.%2BC.%26aulast%3DHesselink%26aufirst%3DM.%2BB.%26aulast%3DHerremans%26aufirst%3DA.%2BH.%2BJ.%26aulast%3DKruse%26aufirst%3DC.%2BG.%26atitle%3DSynthesis%252C%2520structure-activity%2520relationships%252C%2520and%2520biological%2520properties%2520of%25201-heteroaryl-4-%255Bw-%25281H-indol-3-yl%2529alkyl%255D%2520piperazines%252C%2520novel%2520potential%2520antipsychotics%2520combining%2520potent%2520dopamine%2520D2%2520receptor%2520antagonism%2520with%2520potent%2520serotonin%2520reuptake%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6855%26epage%3D6869%26doi%3D10.1021%2Fjm050148z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22aR"><div class="casContent"><span class="casTitleNuber">22a</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhJwdzh0AOO5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit22b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Bailey, S.</span><span> </span><span class="NLM_article-title">The CCP4 suite - programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+CCP4+suite+-+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520CCP4%2520suite%2520-%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit22c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22cR"><div class="casContent"><span class="casTitleNuber">22c</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0ljelDzzc9P_ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit22c&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit22d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22dR"><div class="casContent"><span class="casTitleNuber">22d</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljelDzzc9P_ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit22d&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit22e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Bricogne, G.; Blanc, E.; Brandl, M.; Flensburg, C.; Keller, P.; Paciorek, W.; Roversi, P.; Smart, O. S.; Vonrhein, C.; Womack, T. O.</span>BUSTER<span class="NLM_edition">version 2.8.0</span>; <span class="NLM_publisher-name">Global Phasing Ltd</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>.<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22e&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b','cit1c','cit1d','cit1e'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7a','cit7b'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10a','cit10b','cit10c'],'ref11':['cit11'],'ref12':['cit12a','cit12b'],'ref13':['cit13a','cit13b'],'ref14':['cit14a','cit14b','cit14c'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19a','cit19b'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22a','cit22b','cit22c','cit22d','cit22e']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 17 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Alexandria
P. Taylor, Magdalena Swewczyk, Steven Kennedy, Viacheslav V. Trush, Hong Wu, Hong Zeng, Aiping Dong, Renato Ferreira de Freitas, John Tatlock, Robert A. Kumpf, Martin Wythes, Agustin Casimiro-Garcia, Rajiah Aldrin Denny, Mihir D. Parikh, Fengling Li, Dalia Barsyte-Lovejoy, Matthieu Schapira, Masoud Vedadi, Peter J. Brown, Cheryl H. Arrowsmith, <span class="NLM_string-name hlFld-ContribAuthor">Dafydd R. Owen</span>. </span><span class="cited-content_cbyCitation_article-title">Selective, Small-Molecule Co-Factor Binding Site Inhibition of a Su(var)3–9, Enhancer of Zeste, Trithorax Domain Containing Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 7669-7683. <a href="https://doi.org/10.1021/acs.jmedchem.9b00112" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00112</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00112%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%25252C%252BSmall-Molecule%252BCo-Factor%252BBinding%252BSite%252BInhibition%252Bof%252Ba%252BSu%252528var%2525293%2525E2%252580%2525939%25252C%252BEnhancer%252Bof%252BZeste%25252C%252BTrithorax%252BDomain%252BContaining%252BLysine%252BMethyltransferase%26aulast%3DTaylor%26aufirst%3DAlexandria%2BP.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17012019%26date%3D03092019%26date%3D15082019%26volume%3D62%26issue%3D17%26spage%3D7669%26epage%3D7683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Ge  Jin</span>, <span class="hlFld-ContribAuthor ">Yunfeng  Guo</span>, <span class="hlFld-ContribAuthor ">Yuan  Cao</span>, <span class="hlFld-ContribAuthor ">Shuhuai  Niu</span>, <span class="hlFld-ContribAuthor ">Xiaomei  Fan</span>, <span class="hlFld-ContribAuthor ">Jun  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">SMYD2 suppresses p53 activity to promote glucose metabolism in cervical cancer. </span><span class="cited-content_cbyCitation_journal-name">Experimental Cell Research</span><span> <strong>2021,</strong> <em>404 </em>
                                    (2)
                                     , 112649. <a href="https://doi.org/10.1016/j.yexcr.2021.112649" title="DOI URL">https://doi.org/10.1016/j.yexcr.2021.112649</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.yexcr.2021.112649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.yexcr.2021.112649%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Cell%2520Research%26atitle%3DSMYD2%252Bsuppresses%252Bp53%252Bactivity%252Bto%252Bpromote%252Bglucose%252Bmetabolism%252Bin%252Bcervical%252Bcancer%26aulast%3DWang%26aufirst%3DYing%26date%3D2021%26volume%3D404%26issue%3D2%26spage%3D112649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamakoti P.  Bhat</span>, <span class="hlFld-ContribAuthor ">H.  Ümit Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Or  Gozani</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics and beyond: targeting writers of protein lysine methylation to treat disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (4)
                                     , 265-286. <a href="https://doi.org/10.1038/s41573-020-00108-x" title="DOI URL">https://doi.org/10.1038/s41573-020-00108-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-00108-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-00108-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DEpigenetics%252Band%252Bbeyond%25253A%252Btargeting%252Bwriters%252Bof%252Bprotein%252Blysine%252Bmethylation%252Bto%252Btreat%252Bdisease%26aulast%3DBhat%26aufirst%3DKamakoti%2BP.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D4%26spage%3D265%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Zhang</span>, <span class="hlFld-ContribAuthor ">Liping  Liao</span>, <span class="hlFld-ContribAuthor ">Fan  Wu</span>, <span class="hlFld-ContribAuthor ">Fengcai  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhongya  Sun</span>, <span class="hlFld-ContribAuthor ">Haijun  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and structure–activity relationship studies of LLY-507 analogues as SMYD2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (22)
                                     , 127598. <a href="https://doi.org/10.1016/j.bmcl.2020.127598" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127598%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bof%252BLLY-507%252Banalogues%252Bas%252BSMYD2%252Binhibitors%26aulast%3DZhang%26aufirst%3DBin%26date%3D2020%26volume%3D30%26issue%3D22%26spage%3D127598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hope S.  Rugo</span>, <span class="hlFld-ContribAuthor ">Ira  Jacobs</span>, <span class="hlFld-ContribAuthor ">Shikhar  Sharma</span>, <span class="hlFld-ContribAuthor ">Frank  Scappaticci</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Paul</span>, <span class="hlFld-ContribAuthor ">Kristen  Jensen-Pergakes</span>, <span class="hlFld-ContribAuthor ">Gabriel G.  Malouf</span>. </span><span class="cited-content_cbyCitation_article-title">The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review. </span><span class="cited-content_cbyCitation_journal-name">Advances in Therapy</span><span> <strong>2020,</strong> <em>37 </em>
                                    (7)
                                     , 3059-3082. <a href="https://doi.org/10.1007/s12325-020-01379-x" title="DOI URL">https://doi.org/10.1007/s12325-020-01379-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12325-020-01379-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12325-020-01379-x%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvances%2520in%2520Therapy%26atitle%3DThe%252BPromise%252Bfor%252BHistone%252BMethyltransferase%252BInhibitors%252Bfor%252BEpigenetic%252BTherapy%252Bin%252BClinical%252BOncology%25253A%252BA%252BNarrative%252BReview%26aulast%3DRugo%26aufirst%3DHope%2BS.%26date%3D2020%26date%3D2020%26volume%3D37%26issue%3D7%26spage%3D3059%26epage%3D3082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dillon K.  Jarrell</span>, <span class="hlFld-ContribAuthor ">Kelly N.  Hassell</span>, <span class="hlFld-ContribAuthor ">Debbie C.  Crans</span>, <span class="hlFld-ContribAuthor ">Shari  Lanning</span>, <span class="hlFld-ContribAuthor ">Mark A.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Characterizing the Role of SMYD2 in Mammalian Embryogenesis—Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Veterinary Sciences</span><span> <strong>2020,</strong> <em>7 </em>
                                    (2)
                                     , 63. <a href="https://doi.org/10.3390/vetsci7020063" title="DOI URL">https://doi.org/10.3390/vetsci7020063</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/vetsci7020063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fvetsci7020063%26sid%3Dliteratum%253Aachs%26jtitle%3DVeterinary%2520Sciences%26atitle%3DCharacterizing%252Bthe%252BRole%252Bof%252BSMYD2%252Bin%252BMammalian%252BEmbryogenesis%2525E2%252580%252594Future%252BDirections%26aulast%3DJarrell%26aufirst%3DDillon%2BK.%26date%3D2020%26date%3D2020%26volume%3D7%26issue%3D2%26spage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Yi</span>, <span class="hlFld-ContribAuthor ">Xue-Jun  Jiang</span>, <span class="hlFld-ContribAuthor ">Ze-Min  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Histone methyltransferase SMYD2: ubiquitous regulator of disease. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-019-0711-4" title="DOI URL">https://doi.org/10.1186/s13148-019-0711-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-019-0711-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-019-0711-4%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DHistone%252Bmethyltransferase%252BSMYD2%25253A%252Bubiquitous%252Bregulator%252Bof%252Bdisease%26aulast%3DYi%26aufirst%3DXin%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi  Zeng</span>, <span class="hlFld-ContribAuthor ">Rongfang  Qiu</span>, <span class="hlFld-ContribAuthor ">Yang  Yang</span>, <span class="hlFld-ContribAuthor ">Tianyang  Gao</span>, <span class="hlFld-ContribAuthor ">Yu  Zheng</span>, <span class="hlFld-ContribAuthor ">Wei  Huang</span>, <span class="hlFld-ContribAuthor ">Jie  Gao</span>, <span class="hlFld-ContribAuthor ">Kai  Zhang</span>, <span class="hlFld-ContribAuthor ">Ruiqiong  Liu</span>, <span class="hlFld-ContribAuthor ">Shuang  Wang</span>, <span class="hlFld-ContribAuthor ">Yongqiang  Hou</span>, <span class="hlFld-ContribAuthor ">Wenqian  Yu</span>, <span class="hlFld-ContribAuthor ">Shuai  Leng</span>, <span class="hlFld-ContribAuthor ">Dandan  Feng</span>, <span class="hlFld-ContribAuthor ">Wei  Liu</span>, <span class="hlFld-ContribAuthor ">Xi  Zhang</span>, <span class="hlFld-ContribAuthor ">Yan  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Regulation of EZH2 by SMYD2-Mediated Lysine Methylation Is Implicated in Tumorigenesis. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2019,</strong> <em>29 </em>
                                    (6)
                                     , 1482-1498.e4. <a href="https://doi.org/10.1016/j.celrep.2019.10.004" title="DOI URL">https://doi.org/10.1016/j.celrep.2019.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2019.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2019.10.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DRegulation%252Bof%252BEZH2%252Bby%252BSMYD2-Mediated%252BLysine%252BMethylation%252BIs%252BImplicated%252Bin%252BTumorigenesis%26aulast%3DZeng%26aufirst%3DYi%26date%3D2019%26volume%3D29%26issue%3D6%26spage%3D1482%26epage%3D1498.e4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jodie  Hay</span>, <span class="hlFld-ContribAuthor ">Kathryn  Gilroy</span>, <span class="hlFld-ContribAuthor ">Camille  Huser</span>, <span class="hlFld-ContribAuthor ">Anna  Kilbey</span>, <span class="hlFld-ContribAuthor ">Alma  Mcdonald</span>, <span class="hlFld-ContribAuthor ">Amanda  MacCallum</span>, <span class="hlFld-ContribAuthor ">Ailsa  Holroyd</span>, <span class="hlFld-ContribAuthor ">Ewan  Cameron</span>, <span class="hlFld-ContribAuthor ">James C.  Neil</span>. </span><span class="cited-content_cbyCitation_article-title">Collaboration of MYC and RUNX2 in lymphoma simulates T‐cell receptor signaling and attenuates p53 pathway activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Biochemistry</span><span> <strong>2019,</strong> <em>120 </em>
                                    (10)
                                     , 18332-18345. <a href="https://doi.org/10.1002/jcb.29143" title="DOI URL">https://doi.org/10.1002/jcb.29143</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcb.29143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcb.29143%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Biochemistry%26atitle%3DCollaboration%252Bof%252BMYC%252Band%252BRUNX2%252Bin%252Blymphoma%252Bsimulates%252BT%2525E2%252580%252590cell%252Breceptor%252Bsignaling%252Band%252Battenuates%252Bp53%252Bpathway%252Bactivity%26aulast%3DHay%26aufirst%3DJodie%26date%3D2019%26date%3D2019%26volume%3D120%26issue%3D10%26spage%3D18332%26epage%3D18345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Dilworth</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein methylation: from chemical tools to precision medicines. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2967-2985. <a href="https://doi.org/10.1007/s00018-019-03147-9" title="DOI URL">https://doi.org/10.1007/s00018-019-03147-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03147-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03147-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DTargeting%252Bprotein%252Bmethylation%25253A%252Bfrom%252Bchemical%252Btools%252Bto%252Bprecision%252Bmedicines%26aulast%3DDilworth%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2967%26epage%3D2985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edoardo  Fabini</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Manoni</span>, <span class="hlFld-ContribAuthor ">Claudia  Ferroni</span>, <span class="hlFld-ContribAuthor ">Alberto Del  Rio</span>, <span class="hlFld-ContribAuthor ">Manuela  Bartolini</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule inhibitors of lysine methyltransferases SMYD2 and SMYD3: current trends. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (8)
                                     , 901-921. <a href="https://doi.org/10.4155/fmc-2018-0380" title="DOI URL">https://doi.org/10.4155/fmc-2018-0380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0380%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSmall-molecule%252Binhibitors%252Bof%252Blysine%252Bmethyltransferases%252BSMYD2%252Band%252BSMYD3%25253A%252Bcurrent%252Btrends%26aulast%3DFabini%26aufirst%3DEdoardo%26date%3D2019%26volume%3D11%26issue%3D8%26spage%3D901%26epage%3D921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maryna V  Murlykina</span>, <span class="hlFld-ContribAuthor ">Oleksandr V  Kolomiets</span>, <span class="hlFld-ContribAuthor ">Maryna M  Kornet</span>, <span class="hlFld-ContribAuthor ">Yana I  Sakhno</span>, <span class="hlFld-ContribAuthor ">Sergey M  Desenko</span>, <span class="hlFld-ContribAuthor ">Victoriya V  Dyakonenko</span>, <span class="hlFld-ContribAuthor ">Svetlana V  Shishkina</span>, <span class="hlFld-ContribAuthor ">Oleksandr A  Brazhko</span>, <span class="hlFld-ContribAuthor ">Vladimir I  Musatov</span>, <span class="hlFld-ContribAuthor ">Alexander V  Tsygankov</span>, <span class="hlFld-ContribAuthor ">Erik V  Van der Eycken</span>, <span class="hlFld-ContribAuthor ">Valentyn A  Chebanov</span>. </span><span class="cited-content_cbyCitation_article-title">Doebner-type pyrazolopyridine carboxylic acids in an Ugi four-component reaction. </span><span class="cited-content_cbyCitation_journal-name">Beilstein Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>15 </em>, 1281-1288. <a href="https://doi.org/10.3762/bjoc.15.126" title="DOI URL">https://doi.org/10.3762/bjoc.15.126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3762/bjoc.15.126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3762%2Fbjoc.15.126%26sid%3Dliteratum%253Aachs%26jtitle%3DBeilstein%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DDoebner-type%252Bpyrazolopyridine%252Bcarboxylic%252Bacids%252Bin%252Ban%252BUgi%252Bfour-component%252Breaction%26aulast%3DMurlykina%26aufirst%3DMaryna%2BV%26date%3D2019%26date%3D2019%26volume%3D15%26spage%3D1281%26epage%3D1288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maryna V.  Murlykina</span>, <span class="hlFld-ContribAuthor ">Alisa D.  Morozova</span>, <span class="hlFld-ContribAuthor ">Ievgen M.  Zviagin</span>, <span class="hlFld-ContribAuthor ">Yana I.  Sakhno</span>, <span class="hlFld-ContribAuthor ">Sergey M.  Desenko</span>, <span class="hlFld-ContribAuthor ">Valentyn A.  Chebanov</span>. </span><span class="cited-content_cbyCitation_article-title">Aminoazole-Based Diversity-Oriented Synthesis of Heterocycles. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00527" title="DOI URL">https://doi.org/10.3389/fchem.2018.00527</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00527%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DAminoazole-Based%252BDiversity-Oriented%252BSynthesis%252Bof%252BHeterocycles%26aulast%3DMurlykina%26aufirst%3DMaryna%2BV.%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Evan M.  Cornett</span>, <span class="hlFld-ContribAuthor ">Bradley M.  Dickson</span>, <span class="hlFld-ContribAuthor ">Krzysztof  Krajewski</span>, <span class="hlFld-ContribAuthor ">Nicholas  Spellmon</span>, <span class="hlFld-ContribAuthor ">Andrew  Umstead</span>, <span class="hlFld-ContribAuthor ">Robert M.  Vaughan</span>, <span class="hlFld-ContribAuthor ">Kevin M.  Shaw</span>, <span class="hlFld-ContribAuthor ">Philip P.  Versluis</span>, <span class="hlFld-ContribAuthor ">Martis W.  Cowles</span>, <span class="hlFld-ContribAuthor ">Joseph  Brunzelle</span>, <span class="hlFld-ContribAuthor ">Zhe  Yang</span>, <span class="hlFld-ContribAuthor ">Irving E.  Vega</span>, <span class="hlFld-ContribAuthor ">Zu-Wen  Sun</span>, <span class="hlFld-ContribAuthor ">Scott B.  Rothbart</span>. </span><span class="cited-content_cbyCitation_article-title">A functional proteomics platform to reveal the sequence determinants of lysine methyltransferase substrate selectivity. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2018,</strong> <em>4 </em>
                                    (11)
                                     , eaav2623. <a href="https://doi.org/10.1126/sciadv.aav2623" title="DOI URL">https://doi.org/10.1126/sciadv.aav2623</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.aav2623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.aav2623%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DA%252Bfunctional%252Bproteomics%252Bplatform%252Bto%252Breveal%252Bthe%252Bsequence%252Bdeterminants%252Bof%252Blysine%252Bmethyltransferase%252Bsubstrate%252Bselectivity%26aulast%3DCornett%26aufirst%3DEvan%2BM.%26date%3D2018%26date%3D2018%26volume%3D4%26issue%3D11%26spage%3Deaav2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J.  Morrison</span>, <span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>, <span class="hlFld-ContribAuthor ">Tim J.  Wigle</span>, <span class="hlFld-ContribAuthor ">Dipti  Sadalge</span>, <span class="hlFld-ContribAuthor ">Kevin W.  Kuntz</span>, <span class="hlFld-ContribAuthor ">Margaret Porter  Scott</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">Darren M.  Harvey</span>, <span class="hlFld-ContribAuthor ">John W.  Lampe</span>, <span class="hlFld-ContribAuthor ">Lorna H.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>, . </span><span class="cited-content_cbyCitation_article-title">Identification of a peptide inhibitor for the histone methyltransferase WHSC1. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , e0197082. <a href="https://doi.org/10.1371/journal.pone.0197082" title="DOI URL">https://doi.org/10.1371/journal.pone.0197082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0197082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0197082%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIdentification%252Bof%252Ba%252Bpeptide%252Binhibitor%252Bfor%252Bthe%252Bhistone%252Bmethyltransferase%252BWHSC1%26aulast%3DMorrison%26aufirst%3DMichael%2BJ.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0197082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bárbara I.  Díaz-Eufracio</span>, <span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Protein–Protein Interaction Modulators for Epigenetic Therapies. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 65-84. <a href="https://doi.org/10.1016/bs.apcsb.2017.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2017.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2017.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2017.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DProtein%2525E2%252580%252593Protein%252BInteraction%252BModulators%252Bfor%252BEpigenetic%252BTherapies%26aulast%3DD%25C3%25ADaz-Eufracio%26aufirst%3DB%25C3%25A1rbara%2BI.%26date%3D2018%26spage%3D65%26epage%3D84%26pub%3DElsevier%26atitle%3DProtein-Protein%252BInteractions%252Bin%252BHuman%252BDisease%25252C%252BPart%252BA%26date%3D2018%26volume%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of SMYD2 inhibitors<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ITC validated hits identified from the screening campaign.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ITC titration of compound <b>5</b>: 34 μM SMYD2 protein solution and 400 μM compound <b>5</b> were used; <i>K</i><sub>d</sub> of 1.73 μM, Δ<i>H</i>° = −6.90 kCal/mol, <i>T</i>Δ<i>S</i>° = 3.14 kCal/mol, and <i>N</i> = 0.92.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overlay of the X-ray crystal structures of SMYD2 in complex with <b>5</b> (green) and a peptide derived from p53 (magenta). The bound inhibitor molecule and peptide are shown as stick models, and the protein backbone is presented as a white ribbon (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJL">5KJL</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Hybridization strategy. (a) Overlay of the X-ray crystal structures of SMYD2 in complex with <b>1</b> (light blue) and <b>5</b> (green). The bound inhibitor molecules are shown as stick models. SMYD2 is colored white. (b) Chemistry schematic describing the structure-based hybridization approach.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0007.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure of SMYD2 in complex with <b>30</b>. The bound inhibitor is colored magenta, and cofactor SAM is colored orange. All residues within 3.2 Å of the inhibitor are shown as sticks, and hydrogen bonds are shown as black dashed lines (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJM">5KJM</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0008.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structural superposition. (a) Overlay of the X-ray crystal structures of SMYD2 in complex with <b>48</b> (white) and <b>47</b> (green) illustrating alternate linkers. The bound inhibitors are shown as stick models, and the binding groove of SMYD2 is shown as a surface and has been colored by electrostatic potential (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJN">5KJN</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJK">5KJK</a>). (b) Chemical structures of compounds <b>48</b> and <b>49</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0009.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Effect of the indicated compounds on cell proliferation is not p53-dependent. HCT116+/+ and RKO-neo represent p53 wild-type cells; HCT116–/– and RKO-E6a represent p53-deficient cells. Cells were treated with the indicated compounds (<b>1</b>, <b>40</b>, and <b>47</b>; listed in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) between 0.03 and 30 μM for 72 h, and cell viability was determined using Alamar Blue. No significant differences in IC<sub>50</sub> values were observed between p53 wild-type and p53-deficient cell lines. The assay was performed in triplicate in two separate experiments. Values shown are mean ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrazolopyridine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) pyruvic acid, AcOH, reflux, 8–24%; (b) amine, HATU or EDC, DIPEA, DMF, 50 °C, 24–90%. See <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> for amines for <b>9</b>–<b>16</b>.</p></p></figure><figure data-id="sch2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Truncated Biaryl Amides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, TEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 80–84%; (b) <i>o</i>-tolylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O, 80 °C, 14–84%.</p></p></figure><figure data-id="sch3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0012.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Extended Pyrazolopyridine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) HCl in dioxane, rt, quant; (b) K<sub>2</sub>CO<sub>3</sub>, ACN, 70 °C, 65%; (c) 1<i>H</i>-pyrazol-3-amine, pyruvic acid, AcOH, reflux, 24%; (d) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, DMF, 60 °C, 5%.</p></p></figure><figure data-id="sch4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0013.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Biphenylpiperazines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Boc<sub>2</sub>O, TEA, Et<sub>2</sub>O/THF, rt, 98%; (b) 3-(ethoxycarbonyl)phenyl-boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 90 °C, 98%; (c) LiOH, MeOH/THF/H<sub>2</sub>O, rt, 92%; (d) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 91%; (e) HCl in dioxane, MeOH, rt, 90%; (f) R-Br, TEA, ACN, reflux, 31–50%; (g) 3,4-dichlorophenethylmethanesulfonate, K<sub>2</sub>CO<sub>3</sub>, ACN, 72%.</p></p></figure><figure data-id="sch5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0014.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 3-Cyanophenyl and Pyridyl Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>52b</b> or <b>52d</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O, 70 °C, 86%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, quant; (c) R-Br, K<sub>2</sub>CO<sub>3</sub>, ACN, microwave 100–150 °C, 36–42%; (d) R-Br 3-(2-bromoethyl)-1<i>H</i>-indole, K<sub>2</sub>CO<sub>3</sub>, ACN, reflux, 18–36%; (e) 3,4-dichlorophenethylmethanesulfonate, NaI, K<sub>2</sub>CO<sub>3</sub>, ACN, 70 °C, 69%.</p></p></figure><figure data-id="sch6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/medium/jm-2016-013036_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0015.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of 3,5-Disubstituted-Pyridyl A-Ring Biarylpiperazines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-24/acs.jmedchem.6b01303/20161216/images/large/jm-2016-013036_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01303&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>57</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O, 75 °C, 69%; (b) 3-(pyrrolidin-1-yl)propan-1-amine, TBD, toluene, 90 °C, 94%; (c) 3-(pyrrolidin-1-yl)propan-1-amine, EDC, HOBt, TEA, CH<sub>2</sub>Cl<sub>2</sub>, rt, 85%; (d) <b>61</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O, 75 °C, 99%; (e) TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt, quant; (f) mesylate, K<sub>2</sub>CO<sub>3</sub>, ACN, 70 °C, 46%; (g) 3-(2-bromoethyl)-1<i>H</i>-indole, TEA, ACN, 75 °C, 20%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01711" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01711" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0ljpMzLFrBGMWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saloura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1038/nrc3884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnrc3884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25614009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=110-124&author=R.+Hamamotoauthor=V.+Salouraauthor=Y.+Nakamura&title=Critical+roles+of+non-histone+protein+lysine+methylation+in+human+tumorigenesis&doi=10.1038%2Fnrc3884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span></div><div class="casAuthors">Hamamoto, Ryuji; Saloura, Vassiliki; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-124</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several protein lysine methyltransferases and demethylases have been identified to have crit. roles in histone modification.  A large body of evidence has indicated that their dysregulation is involved in the development and progression of various diseases, including cancer, and these enzymes are now considered to be potential therapeutic targets.  Although most studies have focused on histone methylation, many reports have revealed that these enzymes also regulate the methylation dynamics of non-histone proteins such as p53, RB1 and STAT3 (signal transducer and activator of transcription 3), which have important roles in human tumorigenesis.  In this Review, we summarize the mol. functions of protein lysine methylation and its involvement in human cancer, with a particular focus on lysine methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0wn_Jk7jSGbVg90H21EOLACvtfcHk0ljpMzLFrBGMWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D&md5=9d998a4efb3c1a069e14747341ceca5e</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrc3884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3884%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DSaloura%26aufirst%3DV.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DCritical%2520roles%2520of%2520non-histone%2520protein%2520lysine%2520methylation%2520in%2520human%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D110%26epage%3D124%26doi%3D10.1038%2Fnrc3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Biggar, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span> </span><span class="NLM_article-title">Non-histone protein methylation as a regulator of cellular signalling and function</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1038/nrm3915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnrm3915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25491103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=5-17&author=K.+K.+Biggarauthor=S.+Li&title=Non-histone+protein+methylation+as+a+regulator+of+cellular+signalling+and+function&doi=10.1038%2Fnrm3915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1cR"><div class="casContent"><span class="casTitleNuber">1c</span><div class="casTitle"><span class="NLM_cas:atitle">Non-histone protein methylation as a regulator of cellular signalling and function</span></div><div class="casAuthors">Biggar, Kyle K.; Li, Shawn S.-C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-17</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Methylation of Lys and Arg residues on non-histone proteins has emerged as a prevalent post-translational modification and as an important regulator of cellular signal transduction mediated by the MAPK, WNT, BMP, Hippo and JAK-STAT signalling pathways.  Crosstalk between methylation and other types of post-translational modifications, and between histone and non-histone protein methylation frequently occurs and affects cellular functions such as chromatin remodelling, gene transcription, protein synthesis, signal transduction and DNA repair.  With recent advances in proteomic techniques, in particular mass spectrometry, the stage is now set to decode the methylproteome and define its functions in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5E-jYGUpGCbVg90H21EOLACvtfcHk0lgjhMPebPSKtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitV2nsrzM&md5=3f60179eadac0ad72295b516003ec048</span></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1038%2Fnrm3915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3915%26sid%3Dliteratum%253Aachs%26aulast%3DBiggar%26aufirst%3DK.%2BK.%26aulast%3DLi%26aufirst%3DS.%26atitle%3DNon-histone%2520protein%2520methylation%2520as%2520a%2520regulator%2520of%2520cellular%2520signalling%2520and%2520function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D5%26epage%3D17%26doi%3D10.1038%2Fnrm3915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">The emerging field of dynamic lysine methylation of non-histone proteins</span> <span class="citation_source-journal">Curr. Opin. Genet. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1016/j.gde.2008.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.gde.2008.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=18339539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsFKgs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=152-158&author=J.+Huangauthor=S.+L.+Berger&title=The+emerging+field+of+dynamic+lysine+methylation+of+non-histone+proteins&doi=10.1016%2Fj.gde.2008.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1dR"><div class="casContent"><span class="casTitleNuber">1d</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging field of dynamic lysine methylation of non-histone proteins</span></div><div class="casAuthors">Huang, Jing; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Genetics & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">152-158</span>CODEN:
                <span class="NLM_cas:coden">COGDET</span>;
        ISSN:<span class="NLM_cas:issn">0959-437X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) regulate protein structure and function.  Lys residue methylation abundantly decorates histone proteins and has recently been detected on non-histone proteins.  In particular, tumor suppressor and transcription factor p53 has provided a model for Lys methylation on a non-histone protein.  As found for histones, Lys methylation is dynamic and can be reversed by demethylation.  Lys methylation regulates function via several distinct mechanisms.  Me-Lys residues provide docking sites for the binding of effector proteins.  Methylation can serve to inhibit alternate PTMs on the same Lys residue.  In addn., Lys residues can be monomethylated, dimethylated, or trimethylated, and these levels of methylation correlate with distinct genomic locations and functions.  Taking into account combinatorial activity with numerous other PTMs, Lys residue methylation provides enormous functional diversity and regulatory complexity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNOzzPE2XZ-LVg90H21EOLACvtfcHk0lgjhMPebPSKtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsFKgs74%253D&md5=34b9d26dfcd6708a828042e0df47cb55</span></div><a href="/servlet/linkout?suffix=cit1d&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2008.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2008.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DThe%2520emerging%2520field%2520of%2520dynamic%2520lysine%2520methylation%2520of%2520non-histone%2520proteins%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2008%26volume%3D18%26spage%3D152%26epage%3D158%26doi%3D10.1016%2Fj.gde.2008.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein methylation at the surface and buried deep: thinking outside the histone box</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">243</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.1016/j.tibs.2013.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.tibs.2013.02.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=243-252&author=S.+G.+Clarke&title=Protein+methylation+at+the+surface+and+buried+deep%3A+thinking+outside+the+histone+box&doi=10.1016%2Fj.tibs.2013.02.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1e&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2013.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2013.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520methylation%2520at%2520the%2520surface%2520and%2520buried%2520deep%253A%2520thinking%2520outside%2520the%2520histone%2520box%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2013%26volume%3D38%26spage%3D243%26epage%3D252%26doi%3D10.1016%2Fj.tibs.2013.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Komatsu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozaki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aiko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshizumi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otsuji, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span> </span><span class="NLM_article-title">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1139</span><span class="NLM_x">–</span> <span class="NLM_lpage">1146</span><span class="refDoi"> DOI: 10.1093/carcin/bgp116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1093%2Fcarcin%2Fbgp116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=19423649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1139-1146&author=S.+Komatsuauthor=I.+Imotoauthor=H.+Tsudaauthor=K.+Kozakiauthor=T.+Muramatsuauthor=Y.+Shimadaauthor=S.+Aikoauthor=Y.+Yoshizumiauthor=D.+Ichikawaauthor=E.+Otsujiauthor=J.+Inazawa&title=Overexpression+of+SMYD2+relates+to+tumor+cell+proliferation+and+malignant+outcome+of+esophageal+squamous+cell+carcinoma&doi=10.1093%2Fcarcin%2Fbgp116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span></div><div class="casAuthors">Komatsu, Shuhei; Imoto, Issei; Tsuda, Hitoshi; Kozaki, Ken-ich; Muramatsu, Tomoki; Shimada, Yutaka; Aiko, Satoshi; Yoshizumi, Yutaka; Ichikawa, Daisuke; Otsuji, Eigo; Inazawa, Johji</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although we have identified two putative targets, ATF3 and CENPF, for a frequently gained/amplified region around 1q32-q41 in esophageal squamous cell carcinoma (ESCC), it is possible that other amplification targets remain to be identified.  In this study, we tested whether SET and MYND domain-contg. protein 2 (SMYD2), located between those two genes and encoding a lysine methyltransferase for histone H3K36 and p53K370 that regulates transcription and inhibits transactivation activity, resp., acts as a cancer-promoting gene through activation/overexpression in ESCC.  Frequent overexpression of SMYD2 mRNA and protein was obsd. in KYSE150 cells with remarkable amplification at 1q32-41.1 and other ESCC cell lines (11/43 lines, 25.6%).  Overexpression of SMYD2 protein was frequently detected in primary tumor samples of ESCC (117/153 cases, 76.5%) as well and significantly correlated with gender, venous invasion, the pT category in the tumor-lymph node-metastases classification and status of recurrence.  Patients with SMYD2-overexpressing tumors had a worse overall rate of survival than those with non-expressing tumors, and SMYD2 positivity was independently assocd. with a worse outcome in the multivariate anal.  Knockdown of SMYD2 expression inhibited and ectopic overexpression of SMYD2 promoted the proliferation of ESCC cells in a TP53 mutation-independent but SMYD2 expression-dependent manner.  These findings suggest that SMYD2 plays an important role in tumor cell proliferation through its activation/overexpression and highlight its usefulness as a prognosticator and potential therapeutic target in ESCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGuuMnL7IjzbVg90H21EOLACvtfcHk0lgjhMPebPSKtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D&md5=7eb396b256c9ec2b182fd999796527b5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp116%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DS.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DKozaki%26aufirst%3DK.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DAiko%26aufirst%3DS.%26aulast%3DYoshizumi%26aufirst%3DY.%26aulast%3DIchikawa%26aufirst%3DD.%26aulast%3DOtsuji%26aufirst%3DE.%26aulast%3DInazawa%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520SMYD2%2520relates%2520to%2520tumor%2520cell%2520proliferation%2520and%2520malignant%2520outcome%2520of%2520esophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DCarcinogenesis%26date%3D2009%26volume%3D30%26spage%3D1139%26epage%3D1146%26doi%3D10.1093%2Fcarcin%2Fbgp116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Placek, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">Repression of p53 activity by SMYD2 mediated methylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">444</span><span class="NLM_x">, </span> <span class="NLM_fpage">629</span><span class="NLM_x">–</span> <span class="NLM_lpage">632</span><span class="refDoi"> DOI: 10.1038/nature05287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnature05287" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=444&publication_year=2006&pages=629-632&author=J.+Huangauthor=L.+Perez-Burgosauthor=B.+J.+Placekauthor=R.+Senguptaauthor=M.+Richterauthor=J.+A.+Dorseyauthor=S.+Kubicekauthor=S.+Opravilauthor=T.+Jenuweinauthor=S.+L.+Berger&title=Repression+of+p53+activity+by+SMYD2+mediated+methylation&doi=10.1038%2Fnature05287"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature05287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05287%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DPlacek%26aufirst%3DB.%2BJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DRepression%2520of%2520p53%2520activity%2520by%2520SMYD2%2520mediated%2520methylation%26jtitle%3DNature%26date%3D2006%26volume%3D444%26spage%3D629%26epage%3D632%26doi%3D10.1038%2Fnature05287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trescott, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holcomb, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirinupong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span> </span><span class="NLM_article-title">Structural insights into estrogen receptor α methylation by histone methyltransferase SMYD2, a cellular event implicated in estrogen signaling regulation</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">426</span><span class="NLM_x">, </span> <span class="NLM_fpage">3413</span><span class="NLM_x">–</span> <span class="NLM_lpage">3425</span><span class="refDoi"> DOI: 10.1016/j.jmb.2014.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.jmb.2014.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=24594358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkt1Shuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3413-3425&author=Y.+Jiangauthor=L.+Trescottauthor=J.+Holcombauthor=X.+Zhangauthor=J.+Brunzelleauthor=N.+Sirinupongauthor=X.+Shiauthor=Z.+Yang&title=Structural+insights+into+estrogen+receptor+%CE%B1+methylation+by+histone+methyltransferase+SMYD2%2C+a+cellular+event+implicated+in+estrogen+signaling+regulation&doi=10.1016%2Fj.jmb.2014.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into Estrogen Receptor α Methylation by Histone Methyltransferase SMYD2, a Cellular Event Implicated in Estrogen Signaling Regulation</span></div><div class="casAuthors">Jiang, Yuanyuan; Trescott, Laura; Holcomb, Joshua; Zhang, Xi; Brunzelle, Joseph; Sirinupong, Nualpun; Shi, Xiaobing; Yang, Zhe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3413-3425</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Estrogen receptor (ER) signaling plays a pivotal role in many developmental processes and has been implicated in numerous diseases including cancers.  We recently showed that direct ERα methylation by the multi-specificity histone lysine methyltransferase SMYD2 regulates estrogen signaling through repressing ERα-dependent transactivation.  However, the mechanism controlling the specificity of the SMYD2-ERα interaction and the structural basis of SMYD2 substrate binding diversity are unknown.  Here we present the crystal structure of SMYD2 in complex with a target lysine (Lys266)-contg. ERα peptide.  The structure reveals that ERα binds SMYD2 in a U-shaped conformation with the binding specificity detd. mainly by residues C-terminal to the target lysine.  The structure also reveals numerous intrapeptide contacts that ensure shape complementarity between the substrate and the active site of the enzyme, thereby likely serving as an addnl. structural determinant of substrate specificity.  In addn., comparison of the SMYD2-ERα and SMYD2-p53 structures provides the first structural insight into the diverse nature of SMYD2 substrate recognition and suggests that the broad specificity of SMYD2 is achieved by multiple mol. mechanisms such as distinct peptide binding modes and the intrinsic dynamics of peptide ligands.  Strikingly, a novel potentially SMYD2-specific polyethylene glycol binding site is identified in the CTD domain, implicating possible functions in extended substrate binding or protein-protein interactions.  Our study thus provides the structural basis for the SMYD2-mediated ERα methylation, and the resulting knowledge of SMYD2 substrate specificity and target binding diversity could have important implications in selective drug design against a wide range of ERα-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeCcweXaG62LVg90H21EOLACvtfcHk0lgyFr84IhsFwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkt1Shuro%253D&md5=769725ae4bb4d43bc33fdc717ffad56c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTrescott%26aufirst%3DL.%26aulast%3DHolcomb%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DSirinupong%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DStructural%2520insights%2520into%2520estrogen%2520receptor%2520%25CE%25B1%2520methylation%2520by%2520histone%2520methyltransferase%2520SMYD2%252C%2520a%2520cellular%2520event%2520implicated%2520in%2520estrogen%2520signaling%2520regulation%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3413%26epage%3D3425%26doi%3D10.1016%2Fj.jmb.2014.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakakido, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1016/j.canlet.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.canlet.2014.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=24880080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2014&pages=126-133&author=R.+Hamamotoauthor=R.+Toyokawaauthor=M.+Nakakidoauthor=K.+Uedaauthor=Y.+Nakamura&title=SMYD2-dependent+HSP90+methylation+promotes+cancer+cell+proliferation+by+regulating+the+chaperone+complex+formation&doi=10.1016%2Fj.canlet.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span></div><div class="casAuthors">Hamamoto, Ryuji; Toyokawa, Gouji; Nakakido, Makoto; Ueda, Koji; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-133</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Heat shock protein 90 (HSP90) is a highly conserved mol. chaperone that facilitates the maturation of a wide range of proteins, and it has been recognized as a crucial facilitator of oncogene addiction and cancer cell survival.  Although HSP90 function is regulated by a variety of post-translational modifications, the physiol. significance of methylation has not fully been elucidated.  Here we demonstrate that HSP90AB1 is methylated by the histone methyltransferase SMYD2 and that it plays a crit. role in human carcinogenesis.  HSP90AB1 and SMYD2 can interact through the C-terminal region of HSP90AB1 and the SET domain of SMYD2.  Both in vitro and in vivo methyltransferase assays revealed that SMYD2 could methylate HSP90AB1 and mass spectrometry anal. indicated lysines 531 and 574 of HSP90AB1 to be methylated.  These methylation sites were shown to be important for the dimerization and chaperone complex formation of HSP90AB1.  Furthermore, methylated HSP90AB1 accelerated the proliferation of cancer cells.  Our study reveals a novel mechanism for human carcinogenesis via methylation of HSP90AB1 by SMYD2, and addnl. functional studies may assist in developing novel strategies for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhJR6mltsK1LVg90H21EOLACvtfcHk0lgyFr84IhsFwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D&md5=c2832d0fd7c14091ed8098a466f13122</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DToyokawa%26aufirst%3DR.%26aulast%3DNakakido%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DSMYD2-dependent%2520HSP90%2520methylation%2520promotes%2520cancer%2520cell%2520proliferation%2520by%2520regulating%2520the%2520chaperone%2520complex%2520formation%26jtitle%3DCancer%2520Lett.%26date%3D2014%26volume%3D351%26spage%3D126%26epage%3D133%26doi%3D10.1016%2Fj.canlet.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Piao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">The histone methyltransferase SMYD2 methylates PARP1 and promotes Poly(ADP-ribosyl)ation activity in cancer cells</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1016/j.neo.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.neo.2014.03.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=257-264&author=L.+Piaoauthor=D.+Kangauthor=T.+Suzukiauthor=A.+Masudaauthor=N.+Dohmaeauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=The+histone+methyltransferase+SMYD2+methylates+PARP1+and+promotes+Poly%28ADP-ribosyl%29ation+activity+in+cancer+cells&doi=10.1016%2Fj.neo.2014.03.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.neo.2014.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neo.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DPiao%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DThe%2520histone%2520methyltransferase%2520SMYD2%2520methylates%2520PARP1%2520and%2520promotes%2520Poly%2528ADP-ribosyl%2529ation%2520activity%2520in%2520cancer%2520cells%26jtitle%3DNeoplasia%26date%3D2014%26volume%3D16%26spage%3D257%26epage%3D264%26doi%3D10.1016%2Fj.neo.2014.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Saddic, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aslanian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, J. R.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span> </span><span class="NLM_article-title">Methylation of the retinoblastoma tumor suppressor by SMYD2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">37733</span><span class="NLM_x">–</span> <span class="NLM_lpage">37740</span><span class="refDoi"> DOI: 10.1074/jbc.M110.137612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1074%2Fjbc.M110.137612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20870719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=37733-37740&author=L.+A.+Saddicauthor=L.+E.+Westauthor=A.+Aslanianauthor=J.+R.+Yatesauthor=S.+M.+Rubinauthor=O.+Gozaniauthor=J.+Sage&title=Methylation+of+the+retinoblastoma+tumor+suppressor+by+SMYD2&doi=10.1074%2Fjbc.M110.137612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of the Retinoblastoma Tumor Suppressor by SMYD2</span></div><div class="casAuthors">Saddic, Louis A.; West, Lisandra E.; Aslanian, Aaron; Yates, John R., III; Rubin, Seth M.; Gozani, Or; Sage, Julien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">37733-37740</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The retinoblastoma tumor suppressor (RB) is a central cell cycle regulator and tumor suppressor.  RB cellular functions are known to be regulated by a diversity of post-translational modifications such as phosphorylation and acetylation, raising the possibility that RB may also be methylated in cells.  Here we demonstrate that RB can be methylated by SMYD2 at lysine 860, a highly conserved and novel site of modification.  This methylation event occurs in vitro and in cells, and it is regulated during cell cycle progression, cellular differentiation, and in response to DNA damage.  Furthermore, we show that RB monomethylation at lysine 860 provides a direct binding site for the methyl-binding domain of the transcriptional repressor L3MBTL1.  These results support the idea that a code of post-translational modifications exists for RB and helps guide its functions in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXUFwS6LhLybVg90H21EOLACvtfcHk0lgFWlDpsfqCKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt7rM&md5=16dd8b7837d4d623dd553f11b0c5e2df</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.137612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.137612%26sid%3Dliteratum%253Aachs%26aulast%3DSaddic%26aufirst%3DL.%2BA.%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DAslanian%26aufirst%3DA.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DRubin%26aufirst%3DS.%2BM.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DMethylation%2520of%2520the%2520retinoblastoma%2520tumor%2520suppressor%2520by%2520SMYD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D37733%26epage%3D37740%26doi%3D10.1074%2Fjbc.M110.137612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">RB1 methylation by SMYD2 enhances cell cycle progression through an increase of RB1 phosphorylation</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">476</span><span class="NLM_x">–</span> <span class="NLM_lpage">486</span><span class="refDoi"> DOI: 10.1593/neo.12656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1593%2Fneo.12656" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=476-486&author=H.+S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=K.+Maejimaauthor=Y.+Yamaneauthor=T.+Suzukiauthor=N.+Dohmaeauthor=M.+Kogureauthor=D.+Kangauthor=D.+E.+Nealauthor=B.+A.+Ponderauthor=H.+Yamaueauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=RB1+methylation+by+SMYD2+enhances+cell+cycle+progression+through+an+increase+of+RB1+phosphorylation&doi=10.1593%2Fneo.12656"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1593%2Fneo.12656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1593%252Fneo.12656%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DRB1%2520methylation%2520by%2520SMYD2%2520enhances%2520cell%2520cycle%2520progression%2520through%2520an%2520increase%2520of%2520RB1%2520phosphorylation%26jtitle%3DNeoplasia%26date%3D2012%26volume%3D14%26spage%3D476%26epage%3D486%26doi%3D10.1593%2Fneo.12656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span> </span><span class="NLM_article-title">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">892</span><span class="NLM_x">–</span> <span class="NLM_lpage">905</span><span class="refDoi"> DOI: 10.1074/mcp.M115.053280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1074%2Fmcp.M115.053280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=26750096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=892-905&author=J.+B.+Olsenauthor=X.+Caoauthor=B.+Hanauthor=L.+H.+Chenauthor=A.+Horvathauthor=T.+I.+Richardsonauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=H.+Nguyen&title=Quantitative+profiling+of+the+activity+of+protein+lysine+methyltransferase+SMYD2+using+SILAC-based+proteomics&doi=10.1074%2Fmcp.M115.053280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span></div><div class="casAuthors">Olsen, Jonathan B.; Cao, Xing-Jun; Han, Bomie; Chen, Lisa Hong; Horvath, Alexander; Richardson, Timothy I.; Campbell, Robert M.; Garcia, Benjamin A.; Nguyen, Hannah</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-905</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The significance of non-histone lysine methylation in cell biol. and human disease is an emerging area of research exploration.  The development of small mol. inhibitors that selectively and potently target enzymes that catalyze the addn. of methyl-groups to lysine residues, such as the protein lysine mono-methyltransferase SMYD2, is an active area of drug discovery.  Crit. to the accurate assessment of biol. function is the ability to identify target enzyme substrates and to define enzyme substrate specificity within the context of the cell.  Here, using stable isotopic labeling with amino acids in cell culture (SILAC) coupled with immunoaffinity enrichment of mono-methyl-lysine (Kme1) peptides and mass spectrometry, we report a comprehensive, large-scale proteomic study of lysine mono-methylation, comprising a total of 1032 Kme1 sites in esophageal squamous cell carcinoma (ESCC) cells and 1861 Kme1 sites in ESCC cells overexpressing SMYD2.  Among these Kme1 sites is a subset of 35 found to be potently down-regulated by both shRNA-mediated knockdown of SMYD2 and LLY-507, a selective small mol. inhibitor of SMYD2.  In addn., we report specific protein sequence motifs enriched in Kme1 sites that are directly regulated by endogenous SMYD2 activity, revealing that SMYD2 substrate specificity is more diverse than expected.  We further show direct activity of SMYD2 toward BTF3-K2, PDAP1-K126 as well as numerous sites within the repetitive units of two unique and exceptionally large proteins, AHNAK and AHNAK2.  Collectively, our findings provide quant. insights into the cellular activity and substrate recognition of SMYD2 as well as the global landscape and regulation of protein mono-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIbAGzEvpHn7Vg90H21EOLACvtfcHk0lgFWlDpsfqCKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D&md5=ebc0c9ca87e73f98b82cb1e3c10d1c71</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M115.053280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M115.053280%26sid%3Dliteratum%253Aachs%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DHorvath%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DT.%2BI.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DNguyen%26aufirst%3DH.%26atitle%3DQuantitative%2520profiling%2520of%2520the%2520activity%2520of%2520protein%2520lysine%2520methyltransferase%2520SMYD2%2520using%2520SILAC-based%2520proteomics%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2016%26volume%3D15%26spage%3D892%26epage%3D905%26doi%3D10.1074%2Fmcp.M115.053280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1016/j.str.2011.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2&doi=10.1016%2Fj.str.2011.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0lgFWlDpsfqCKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273%26doi%3D10.1016%2Fj.str.2011.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonysamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emtage, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gheyi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">LLY-507, a cell-active, potent, and selective inhibitor of protein-lysine methyltransferase SMYD2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">13641</span><span class="NLM_x">–</span> <span class="NLM_lpage">13653</span><span class="refDoi"> DOI: 10.1074/jbc.M114.626861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1074%2Fjbc.M114.626861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25825497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=13641-13653&author=H.+Nguyenauthor=A.+Allali-Hassaniauthor=S.+Antonysamyauthor=S.+Changauthor=L.+H.+Chenauthor=C.+Curtisauthor=S.+Emtageauthor=L.+Fanauthor=T.+Gheyiauthor=F.+Liauthor=S.+Liuauthor=J.+R.+Martinauthor=D.+Mendelauthor=J.+B.+Olsenauthor=L.+Pelletierauthor=T.+Shatsevaauthor=S.+Wuauthor=F.+F.+Zhangauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=D.+Barsyte-Lovejoyauthor=M.+Maderauthor=M.+Vedadi&title=LLY-507%2C+a+cell-active%2C+potent%2C+and+selective+inhibitor+of+protein-lysine+methyltransferase+SMYD2&doi=10.1074%2Fjbc.M114.626861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-507, a Cell-active, Potent, and Selective Inhibitor of Protein-lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Nguyen, Hannah; Allali-Hassani, Abdellah; Antonysamy, Stephen; Chang, Shawn; Chen, Lisa Hong; Curtis, Carmen; Emtage, Spencer; Fan, Li; Gheyi, Tarun; Li, Fengling; Liu, Shichong; Martin, Joseph R.; Mendel, David; Olsen, Jonathan B.; Pelletier, Laura; Shatseva, Tatiana; Wu, Song; Zhang, Feiyu Fred; Arrowsmith, Cheryl H.; Brown, Peter J.; Campbell, Robert M.; Garcia, Benjamin A.; Barsyte-Lovejoy, Dalia; Mader, Mary; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">13641-13653</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 is a lysine methyltransferase that catalyzes the monomethylation of several protein substrates including p53.  SMYD2 is overexpressed in a significant percentage of esophageal squamous primary carcinomas, and that overexpression correlates with poor patient survival.  However, the mechanism(s) by which SMYD2 promotes oncogenesis is not understood.  A small mol. probe for SMYD2 would allow for the pharmacol. dissection of this biol.  In this report, we disclose LLY-507, a cell-active, potent small mol. inhibitor of SMYD2.  LLY-507 is >100-fold selective for SMYD2 over a broad range of methyltransferase and non-methyltransferase targets.  A 1.63-Å resoln. crystal structure of SMYD2 in complex with LLY-507 shows the inhibitor binding in the substrate peptide binding pocket.  LLY-507 is active in cells as measured by redn. of SMYD2-induced monomethylation of p53 Lys370 at submicromolar concns.  We used LLY-507 to further test other potential roles of SMYD2.  Mass spectrometry-based proteomics showed that cellular global histone methylation levels were not significantly affected by SMYD2 inhibition with LLY-507, and subcellular fractionation studies indicate that SMYD2 is primarily cytoplasmic, suggesting that SMYD2 targets a very small subset of histones at specific chromatin loci and/or non-histone substrates.  Breast and liver cancers were identified through in silico data mining as tumor types that display amplification and/or overexpression of SMYD2.  LLY-507 inhibited the proliferation of several esophageal, liver, and breast cancer cell lines in a dose-dependent manner.  These findings suggest that LLY-507 serves as a valuable chem. probe to aid in the dissection of SMYD2 function in cancer and other biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb5tVOe0M2gLVg90H21EOLACvtfcHk0lh2Pg0NIRihsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D&md5=a75f59a900cfe7a6a04d35498caae78b</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626861%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DCurtis%26aufirst%3DC.%26aulast%3DEmtage%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BR.%26aulast%3DMendel%26aufirst%3DD.%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DPelletier%26aufirst%3DL.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%2BF.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DLLY-507%252C%2520a%2520cell-active%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520protein-lysine%2520methyltransferase%2520SMYD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D13641%26epage%3D13653%26doi%3D10.1074%2Fjbc.M114.626861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit10b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algire, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakob, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery of A-893, a new cell-active benzoxazinone inhibitor of lysine methyltransferase SMYD2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">695</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=695-700&author=R.+F.+Sweisauthor=Z.+Wangauthor=M.+Algireauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=G.+G.+Chiangauthor=J.+Guoauthor=C.+G.+Jakobauthor=S.+Kennedyauthor=F.+Liauthor=D.+Maagauthor=B.+Shawauthor=N.+B.+Soniauthor=M.+Vedadiauthor=W.+N.+Pappano&title=Discovery+of+A-893%2C+a+new+cell-active+benzoxazinone+inhibitor+of+lysine+methyltransferase+SMYD2&doi=10.1021%2Facsmedchemlett.5b00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of A-893, A New Cell-Active Benzoxazinone Inhibitor of Lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Sweis, Ramzi F.; Wang, Zhi; Algire, Mikkel; Arrowsmith, Cheryl H.; Brown, Peter J.; Chiang, Gary G.; Guo, Jun; Jakob, Clarissa G.; Kennedy, Steven; Li, Fengling; Maag, David; Shaw, Bailin; Soni, Nirupama B.; Vedadi, Masoud; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">695-700</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A lack of useful small mol. tools has precluded thorough interrogation of the biol. function of SMYD2, a lysine methyltransferase with known tumor-suppressor substrates.  Systematic exploration of the structure-activity relationships of a previously known benzoxazinone compd. led to the synthesis of A-893 I, a potent and selective SMYD2 inhibitor (IC50: 2.8 nM).  A cocrystal structure reveals the origin of enhanced potency, and effective suppression of p53K370 methylation is obsd. in a lung carcinoma (A549) cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI0i3TmEl2DLVg90H21EOLACvtfcHk0lh2Pg0NIRihsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D&md5=4e37d0d524f1b1066a4ef7ba03dc4695</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00124%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520of%2520A-893%252C%2520a%2520new%2520cell-active%2520benzoxazinone%2520inhibitor%2520of%2520lysine%2520methyltransferase%2520SMYD2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D695%26epage%3D700%26doi%3D10.1021%2Facsmedchemlett.5b00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit10c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Eggert, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillig, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohr, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockigt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiske, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barak, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mowat, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brumby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christ, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ter Laak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez-Montalvan, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badock, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartung, I. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stellfeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stresemann, C.</span><span> </span><span class="NLM_article-title">Discovery and characterization of a highly potent and selective aminopyrazoline-based in vivo probe (BAY-598) for the protein lysine methyltransferase SMYD2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4578</span><span class="NLM_x">–</span> <span class="NLM_lpage">4600</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01890</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01890" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVGmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4578-4600&author=E.+Eggertauthor=R.+C.+Hilligauthor=S.+Kohrauthor=D.+Stockigtauthor=J.+Weiskeauthor=N.+Barakauthor=J.+Mowatauthor=T.+Brumbyauthor=C.+D.+Christauthor=A.+Ter+Laakauthor=T.+Langauthor=A.+E.+Fernandez-Montalvanauthor=V.+Badockauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=D.+Barsyte-Lovejoyauthor=M.+Szewczykauthor=S.+Kennedyauthor=F.+Liauthor=M.+Vedadiauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=C.+H.+Arrowsmithauthor=T.+Stellfeldauthor=C.+Stresemann&title=Discovery+and+characterization+of+a+highly+potent+and+selective+aminopyrazoline-based+in+vivo+probe+%28BAY-598%29+for+the+protein+lysine+methyltransferase+SMYD2&doi=10.1021%2Facs.jmedchem.5b01890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10cR"><div class="casContent"><span class="casTitleNuber">10c</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Eggert, Erik; Hillig, Roman C.; Koehr, Silke; Stoeckigt, Detlef; Weiske, Joerg; Barak, Naomi; Mowat, Jeffrey; Brumby, Thomas; Christ, Clara D.; ter Laak, Antonius; Lang, Tina; Fernandez-Montalvan, Amaury E.; Badock, Volker; Weinmann, Hilmar; Hartung, Ingo V.; Barsyte-Lovejoy, Dalia; Szewczyk, Magdalena; Kennedy, Steven; Li, Fengling; Vedadi, Masoud; Brown, Peter J.; Santhakumar, Vijayaratnam; Arrowsmith, Cheryl H.; Stellfeld, Timo; Stresemann, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4578-4600</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways.  SET and MYND domain contg. protein 2 (SMYD2) is a catalytic SET domain contg. methyltransferase reported to monomethylate lysine residues on histone and nonhistone proteins.  Although several studies have uncovered an important role of SMYD2 in promoting cancer by protein methylation, the biol. of SMYD2 is far from being fully understood.  Utilization of highly potent and selective chem. probes for target validation has emerged as a concept which circumvents possible limitations of knockdown expts. and, in particular, could result in an improved exploration of drug targets with a complex underlying biol.  Here, we report the development of a potent, selective, and cell-active, substrate-competitive inhibitor of SMYD2, which is the first reported inhibitor suitable for in vivo target validation studies in rodents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAuNehWDd3N7Vg90H21EOLACvtfcHk0lgZzF5XiptQtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVGmsLc%253D&md5=d0944b0a6870b9790f9f4c429290f81e</span></div><a href="/servlet/linkout?suffix=cit10c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01890%26sid%3Dliteratum%253Aachs%26aulast%3DEggert%26aufirst%3DE.%26aulast%3DHillig%26aufirst%3DR.%2BC.%26aulast%3DKohr%26aufirst%3DS.%26aulast%3DStockigt%26aufirst%3DD.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DBarak%26aufirst%3DN.%26aulast%3DMowat%26aufirst%3DJ.%26aulast%3DBrumby%26aufirst%3DT.%26aulast%3DChrist%26aufirst%3DC.%2BD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DLang%26aufirst%3DT.%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DStellfeld%26aufirst%3DT.%26aulast%3DStresemann%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520highly%2520potent%2520and%2520selective%2520aminopyrazoline-based%2520in%2520vivo%2520probe%2520%2528BAY-598%2529%2520for%2520the%2520protein%2520lysine%2520methyltransferase%2520SMYD2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4578%26epage%3D4600%26doi%3D10.1021%2Facs.jmedchem.5b01890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theriault, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Code, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birr, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span> </span><span class="NLM_article-title">Biochemical characterization of human SET and MYND domain-containing protein 2 methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">6488</span><span class="NLM_x">–</span> <span class="NLM_lpage">6497</span><span class="refDoi"> DOI: 10.1021/bi200725p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200725p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6488-6497&author=J.+Wuauthor=T.+Cheungauthor=C.+Grandeauthor=A.+D.+Fergusonauthor=X.+Zhuauthor=K.+Theriaultauthor=E.+Codeauthor=C.+Birrauthor=N.+Keenauthor=H.+Chen&title=Biochemical+characterization+of+human+SET+and+MYND+domain-containing+protein+2+methyltransferase&doi=10.1021%2Fbi200725p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fbi200725p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200725p%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DTheriault%26aufirst%3DK.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DBirr%26aufirst%3DC.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DBiochemical%2520characterization%2520of%2520human%2520SET%2520and%2520MYND%2520domain-containing%2520protein%25202%2520methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D6488%26epage%3D6497%26doi%3D10.1021%2Fbi200725p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.jmb.2010.04.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.jmb.2010.04.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20434463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases&doi=10.1016%2Fj.jmb.2010.04.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Adding a Lysine Mimic in the Design of Potent Inhibitors of Histone Lysine Methyltransferases</span></div><div class="casAuthors">Chang, Yanqi; Ganesh, Thota; Horton, John R.; Spannhoff, Astrid; Liu, Jin; Sun, Aiming; Zhang, Xing; Bedford, Mark T.; Shinkai, Yoichi; Snyder, James P.; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Dynamic histone lysine methylation involves the activities of modifying enzymes (writers), enzymes removing modifications (erasers), and readers of the histone code.  One common feature of these activities is the recognition of lysines in methylated and unmethylated states, whether they are substrates, reaction products, or binding partners.  The authors applied the concept of adding a lysine mimic to an established inhibitor (BIX-01294) of histone H3 lysine 9 methyltransferases G9a and G9a-like protein by including a 5-aminopentyloxy moiety, which is inserted into the target lysine-binding channel and becomes methylated by G9a-like protein, albeit slowly.  The compd. enhances its potency in vitro and reduces cell toxicity in vivo.  The authors suggest that adding a lysine or methyl-lysine mimic should be considered in the design of small-mol. inhibitors for other methyl-lysine writers, erasers, and readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsrNWhUZuRrLVg90H21EOLACvtfcHk0lj9wRigSVwxhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaju7c%253D&md5=7ec7ce81db069cd6c1c6637b3ad57007</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.04.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.04.048%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.jmb.2010.04.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span><span class="refDoi"> DOI: 10.1021/jm901543m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a&doi=10.1021%2Fjm901543m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lj9wRigSVwxhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953%26doi%3D10.1021%2Fjm901543m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span> </span><span class="NLM_article-title">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">971</span><span class="NLM_x">–</span> <span class="NLM_lpage">980</span><span class="refDoi"> DOI: 10.1111/cbdd.12050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1111%2Fcbdd.12050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=22978415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2012&pages=971-980&author=A.+Basavapathruniauthor=L.+Jinauthor=S.+R.+Daigleauthor=C.+R.+Majerauthor=C.+A.+Therkelsenauthor=T.+J.+Wigleauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=R.+M.+Pollockauthor=M.+P.+Scottauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=E.+J.+Olhava&title=Conformational+adaptation+drives+potent%2C+selective+and+durable+inhibition+of+the+human+protein+methyltransferase+DOT1L&doi=10.1111%2Fcbdd.12050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span></div><div class="casAuthors">Basavapathruni, Aravind; Jin, Lei; Daigle, Scott R.; Majer, Christina R. A.; Therkelsen, Carly A.; Wigle, Tim J.; Kuntz, Kevin W.; Chesworth, Richard; Pollock, Roy M.; Scott, Margaret P.; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Olhava, Edward J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">971-980</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">DOT1L is the human protein methyltransferase responsible for catalyzing the methylation of histone H3 on lysine 79 (H3K79).  The ectopic activity of DOT1L, assocd. with the chromosomal translocation that is a universal hallmark of MLL-rearranged leukemia, is a required driver of leukemogenesis in this malignancy.  Here, we present studies on the structure-activity relation of aminonucleoside-based DOT1L inhibitors.  Within this series, we find that improvements in target enzyme affinity and selectivity are driven entirely by diminution of the dissocn. rate const. for the enzyme-inhibitor complex, leading to long residence times for the binary complex.  The biochem. Ki and residence times measured for these inhibitors correlate well with their effects on intracellular H3K79 methylation and MLL-rearranged leukemic cell killing.  Crystallog. studies reveal a conformational adaptation mechanism assocd. with high-affinity inhibitor binding and prolonged residence time; these studies also suggest that conformational adaptation likewise plays a crit. role in natural ligand interactions with the enzyme, hence, facilitating enzyme turnover.  These results provide crit. insights into the role of conformational adaptation in the enzymic mechanism of catalysis and in pharmacol. intervention for DOT1L and other members of this enzyme class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJm_d3iX3p77Vg90H21EOLACvtfcHk0lixOGNwCHaOdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O&md5=de628fcd2c37b5f6496cc3b483125309</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12050%26sid%3Dliteratum%253Aachs%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26atitle%3DConformational%2520adaptation%2520drives%2520potent%252C%2520selective%2520and%2520durable%2520inhibition%2520of%2520the%2520human%2520protein%2520methyltransferase%2520DOT1L%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2012%26volume%3D80%26spage%3D971%26epage%3D980%26doi%3D10.1111%2Fcbdd.12050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+O.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lixOGNwCHaOdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BO.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Bunz, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutriaux, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldman, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedivy, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span> </span><span class="NLM_article-title">Requirement for p53 and p21 to sustain G2 arrest after DNA damage</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1501</span><span class="refDoi"> DOI: 10.1126/science.282.5393.1497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1126%2Fscience.282.5393.1497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=9822382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADyaK1cXns1yqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1998&pages=1497-1501&author=F.+Bunzauthor=A.+Dutriauxauthor=C.+Lengauerauthor=T.+Waldmanauthor=S.+Zhouauthor=J.+P.+Brownauthor=J.+M.+Sedivyauthor=K.+W.+Kinzlerauthor=B.+Vogelstein&title=Requirement+for+p53+and+p21+to+sustain+G2+arrest+after+DNA+damage&doi=10.1126%2Fscience.282.5393.1497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement for p53 and p21 to sustain G2 arrest after DNA damage</span></div><div class="casAuthors">Bunz, F.; Dutriaux, A.; Lengauer, C.; Waldman, T.; Zhou, S.; Brown, J. P.; Sedivy, J. M.; Kinzler, K. W.; Vogelstein, B.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">5393</span>),
    <span class="NLM_cas:pages">1497-1501</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">After DNA damage, many cells appear to enter a sustained arrest in the G2 phase of the cell cycle.  It is shown here that this arrest could be sustained only when p53 was present in the cell and capable of transcriptionally activating the cyclin-dependent kinase inhibitor p21.  After disruption of either the p53 or the p21 gene, γ radiated cells progressed into mitosis and exhibited a G2 DNA content only because of a failure of cytokinesis.  Thus, p53 and p21 appear to be essential for maintaining the G2 checkpoint in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO6NPwYqRmK7Vg90H21EOLACvtfcHk0lixOGNwCHaOdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1yqurc%253D&md5=e6f0dc856f71915dbd2b8cfb11de4bad</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1126%2Fscience.282.5393.1497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.282.5393.1497%26sid%3Dliteratum%253Aachs%26aulast%3DBunz%26aufirst%3DF.%26aulast%3DDutriaux%26aufirst%3DA.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DWaldman%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DJ.%2BP.%26aulast%3DSedivy%26aufirst%3DJ.%2BM.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26aulast%3DVogelstein%26aufirst%3DB.%26atitle%3DRequirement%2520for%2520p53%2520and%2520p21%2520to%2520sustain%2520G2%2520arrest%2520after%2520DNA%2520damage%26jtitle%3DScience%26date%3D1998%26volume%3D282%26spage%3D1497%26epage%3D1501%26doi%3D10.1126%2Fscience.282.5393.1497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Smith, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fornace, A. J.,  Jr.</span><span> </span><span class="NLM_article-title">Involvement of the p53 tumor suppressor in repair of u.v.-type DNA damage</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1053</span><span class="NLM_x">–</span> <span class="NLM_lpage">1059</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1995&pages=1053-1059&author=M.+L.+Smithauthor=I.+T.+Chenauthor=Q.+Zhanauthor=P.+M.+O%E2%80%99Connorauthor=A.+J.+Fornace&title=Involvement+of+the+p53+tumor+suppressor+in+repair+of+u.v.-type+DNA+damage"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DChen%26aufirst%3DI.%2BT.%26aulast%3DZhan%26aufirst%3DQ.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DP.%2BM.%26aulast%3DFornace%26aufirst%3DA.%2BJ.%26atitle%3DInvolvement%2520of%2520the%2520p53%2520tumor%2520suppressor%2520in%2520repair%2520of%2520u.v.-type%2520DNA%2520damage%26jtitle%3DOncogene%26date%3D1995%26volume%3D10%26spage%3D1053%26epage%3D1059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit14c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Bauer, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehmann, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendleton, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shyr, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arteaga, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietenpol, J. A.</span><span> </span><span class="NLM_article-title">RNA interference (RNAi) screening approach identifies agents that enhance paclitaxel activity in breast cancer cells</span> <span class="citation_source-journal">Breast Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">R41</span><span class="refDoi"> DOI: 10.1186/bcr2595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1186%2Fbcr2595" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=R41&author=J.+A.+Bauerauthor=F.+Yeauthor=C.+B.+Marshallauthor=B.+D.+Lehmannauthor=C.+S.+Pendletonauthor=Y.+Shyrauthor=C.+L.+Arteagaauthor=J.+A.+Pietenpol&title=RNA+interference+%28RNAi%29+screening+approach+identifies+agents+that+enhance+paclitaxel+activity+in+breast+cancer+cells&doi=10.1186%2Fbcr2595"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14c&amp;dbid=16384&amp;doi=10.1186%2Fbcr2595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2595%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DJ.%2BA.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DMarshall%26aufirst%3DC.%2BB.%26aulast%3DLehmann%26aufirst%3DB.%2BD.%26aulast%3DPendleton%26aufirst%3DC.%2BS.%26aulast%3DShyr%26aufirst%3DY.%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DPietenpol%26aufirst%3DJ.%2BA.%26atitle%3DRNA%2520interference%2520%2528RNAi%2529%2520screening%2520approach%2520identifies%2520agents%2520that%2520enhance%2520paclitaxel%2520activity%2520in%2520breast%2520cancer%2520cells%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2010%26volume%3D12%26spage%3DR41%26doi%3D10.1186%2Fbcr2595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sajjad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novoyatleva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vergarajauregui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troidl, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schermuly, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, H. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, F. B.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1843</span><span class="NLM_x">, </span> <span class="NLM_fpage">2556</span><span class="NLM_x">–</span> <span class="NLM_lpage">2562</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.bbamcr.2014.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=25014164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=2556-2562&author=A.+Sajjadauthor=T.+Novoyatlevaauthor=S.+Vergarajaureguiauthor=C.+Troidlauthor=R.+T.+Schermulyauthor=H.+O.+Tuckerauthor=F.+B.+Engel&title=Lysine+methyltransferase+Smyd2+suppresses+p53-dependent+cardiomyocyte+apoptosis&doi=10.1016%2Fj.bbamcr.2014.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span></div><div class="casAuthors">Sajjad, Amna; Novoyatleva, Tatyana; Vergarajauregui, Silvia; Troidl, Christian; Schermuly, Ralph T.; Tucker, Haley O.; Engel, Felix B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2556-2562</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apoptosis, or programmed cell death, is an essential physiol. process for proper embryogenesis as well as for homeostasis during aging.  In addn., apoptosis is one of the major mechanisms causing cell loss in pathophysiol. conditions such as heart failure.  Thus, inhibition of apoptosis is an important approach for preventive and therapeutic strategies.  Here we show that the histone 3 lysine 4- and lysine 36-specific methyltransferase Smyd2 acts as an endogenous antagonistic player of p53-dependent cardiomyocyte apoptosis.  Smyd2 protein levels were significantly decreased in cardiomyocytes upon cobalt chloride-induced apoptosis or myocardial infarction, while p53 expression was enhanced.  SiRNA-mediated knockdown of Smyd2 in cultured cardiomyocytes further enhanced cobalt chloride-induced cardiomyocyte apoptosis.  In contrast, Smyd2 overexpression resulted in marked methylation of p53 and prevented its accumulation as well as apoptotic cell death in an Hsp90-independent manner.  Moreover, overexpression, of Smyd2, but not Smyd2Y240F lacking a Me transferase activity, significantly rescued CoCl2-induced apoptosis in H9c2 cardioblasts.  Finally, Smyd2 cardiomyocyte-specific deletion in vivo promoted apoptotic cell death upon myocardial infarction, which correlated with enhanced expression of p53 and pro-apoptotic Bax.  Collectively, our data indicate Smyd2 as a cardioprotective protein by methylating p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfw3CFurADyLVg90H21EOLACvtfcHk0liLtHaePcFPpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J&md5=327a9367f05026a05ec527e18f71b26f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DSajjad%26aufirst%3DA.%26aulast%3DNovoyatleva%26aufirst%3DT.%26aulast%3DVergarajauregui%26aufirst%3DS.%26aulast%3DTroidl%26aufirst%3DC.%26aulast%3DSchermuly%26aufirst%3DR.%2BT.%26aulast%3DTucker%26aufirst%3DH.%2BO.%26aulast%3DEngel%26aufirst%3DF.%2BB.%26atitle%3DLysine%2520methyltransferase%2520Smyd2%2520suppresses%2520p53-dependent%2520cardiomyocyte%2520apoptosis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D2556%26epage%3D2562%26doi%3D10.1016%2Fj.bbamcr.2014.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Chebanov, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakhno, Y. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desenko, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernenko, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musatov, V. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shishkina, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shishkin, O. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kappe, C. O.</span><span> </span><span class="NLM_article-title">Cyclocondensation reactions of 5-aminopyrazoles, pyruvic acids and aldehydes. Multicomponent approaches to pyrazolopyridines and related products</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">1229</span><span class="NLM_x">–</span> <span class="NLM_lpage">1242</span><span class="refDoi"> DOI: 10.1016/j.tet.2006.11.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1016%2Fj.tet.2006.11.048" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=1229-1242&author=V.+A.+Chebanovauthor=Y.+I.+Sakhnoauthor=S.+M.+Desenkoauthor=V.+N.+Chernenkoauthor=V.+I.+Musatovauthor=S.+V.+Shishkinaauthor=O.+V.+Shishkinauthor=C.+O.+Kappe&title=Cyclocondensation+reactions+of+5-aminopyrazoles%2C+pyruvic+acids+and+aldehydes.+Multicomponent+approaches+to+pyrazolopyridines+and+related+products&doi=10.1016%2Fj.tet.2006.11.048"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2006.11.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2006.11.048%26sid%3Dliteratum%253Aachs%26aulast%3DChebanov%26aufirst%3DV.%2BA.%26aulast%3DSakhno%26aufirst%3DY.%2BI.%26aulast%3DDesenko%26aufirst%3DS.%2BM.%26aulast%3DChernenko%26aufirst%3DV.%2BN.%26aulast%3DMusatov%26aufirst%3DV.%2BI.%26aulast%3DShishkina%26aufirst%3DS.%2BV.%26aulast%3DShishkin%26aufirst%3DO.%2BV.%26aulast%3DKappe%26aufirst%3DC.%2BO.%26atitle%3DCyclocondensation%2520reactions%2520of%25205-aminopyrazoles%252C%2520pyruvic%2520acids%2520and%2520aldehydes.%2520Multicomponent%2520approaches%2520to%2520pyrazolopyridines%2520and%2520related%2520products%26jtitle%3DTetrahedron%26date%3D2007%26volume%3D63%26spage%3D1229%26epage%3D1242%26doi%3D10.1016%2Fj.tet.2006.11.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Hardy, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sang, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpkin, D. S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Follenfant, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, T. W.</span><span> </span><span class="NLM_article-title">Peripherally acting enkephalin analogues. Polar tri- and tetrapeptides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">1108</span><span class="NLM_x">–</span> <span class="NLM_lpage">1118</span><span class="refDoi"> DOI: 10.1021/jm00125a028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00125a028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1989&pages=1108-1118&author=G.+W.+Hardyauthor=L.+A.+Loweauthor=G.+Millsauthor=P.+Y.+Sangauthor=D.+S.+A.+Simpkinauthor=R.+L.+Follenfantauthor=C.+Shankleyauthor=T.+W.+Smith&title=Peripherally+acting+enkephalin+analogues.+Polar+tri-+and+tetrapeptides&doi=10.1021%2Fjm00125a028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm00125a028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00125a028%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DG.%2BW.%26aulast%3DLowe%26aufirst%3DL.%2BA.%26aulast%3DMills%26aufirst%3DG.%26aulast%3DSang%26aufirst%3DP.%2BY.%26aulast%3DSimpkin%26aufirst%3DD.%2BS.%2BA.%26aulast%3DFollenfant%26aufirst%3DR.%2BL.%26aulast%3DShankley%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DT.%2BW.%26atitle%3DPeripherally%2520acting%2520enkephalin%2520analogues.%2520Polar%2520tri-%2520and%2520tetrapeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1989%26volume%3D32%26spage%3D1108%26epage%3D1118%26doi%3D10.1021%2Fjm00125a028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Kiesewetter, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholten, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedrick, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waymouth, R. M.</span><span> </span><span class="NLM_article-title">Cyclic guanidine organic catalysts: What is magic about triazabicyclodecene?</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">9490</span><span class="NLM_x">–</span> <span class="NLM_lpage">9496</span><span class="refDoi"> DOI: 10.1021/jo902369g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo902369g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKhs7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2009&pages=9490-9496&author=M.+K.+Kiesewetterauthor=M.+D.+Scholtenauthor=N.+Kirnauthor=R.+L.+Weberauthor=J.+L.+Hedrickauthor=R.+M.+Waymouth&title=Cyclic+guanidine+organic+catalysts%3A+What+is+magic+about+triazabicyclodecene%3F&doi=10.1021%2Fjo902369g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic guanidine organic catalysts: What is magic about triazabicyclodecene?</span></div><div class="casAuthors">Kiesewetter, Matthew K.; Scholten, Marc D.; Kirn, Nicole; Weber, Ryan L.; Hedrick, James L.; Waymouth, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9490-9496</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bicyclic guanidine 1,5,7-triazabicyclo[4.4.0]dec-5-ene (TBD) is an effective organocatalyst for the formation of amides from esters and primary amines.  Mechanistic and kinetic studies support a nucleophilic mechanism where TBD reacts reversibly with esters to generate an acyl-TBD intermediate that acylates amines to generate the amides.  Comparative studies of the analogous bicyclic guanidine 1,4,6-triazabicyclo[3.3.0]oct-4-ene (TBO) reveal it to be a much less active acylation catalyst than TBD.  Theor. and mechanistic studies imply that the higher reactivity of TBD is a consequence of both its higher basicity and nucleophilicity than TBO and the high reactivity of the acyl-TBD intermediate, which is sterically prevented from adopting a planar amide structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9dztZtVWNcrVg90H21EOLACvtfcHk0lgCsM9WjR4yTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKhs7nP&md5=0f263f5a0f48749b17ed6bbd486e7f32</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjo902369g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo902369g%26sid%3Dliteratum%253Aachs%26aulast%3DKiesewetter%26aufirst%3DM.%2BK.%26aulast%3DScholten%26aufirst%3DM.%2BD.%26aulast%3DKirn%26aufirst%3DN.%26aulast%3DWeber%26aufirst%3DR.%2BL.%26aulast%3DHedrick%26aufirst%3DJ.%2BL.%26aulast%3DWaymouth%26aufirst%3DR.%2BM.%26atitle%3DCyclic%2520guanidine%2520organic%2520catalysts%253A%2520What%2520is%2520magic%2520about%2520triazabicyclodecene%253F%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2009%26volume%3D74%26spage%3D9490%26epage%3D9496%26doi%3D10.1021%2Fjo902369g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Cowen, S. D.</span><span> </span><span class="NLM_article-title">Targeting the substrate binding site of methyltransferases: structure-based design of SMYD2 inhibitors</span>. EpiCongress 2013, Boston, MA, July 23–24, 2013,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=S.+D.+Cowen&title=Targeting+the+substrate+binding+site+of+methyltransferases%3A+structure-based+design+of+SMYD2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DCowen%26aufirst%3DS.%2BD.%26atitle%3DTargeting%2520the%2520substrate%2520binding%2520site%2520of%2520methyltransferases%253A%2520structure-based%2520design%2520of%2520SMYD2%2520inhibitors%26jtitle%3DEpiCongress%25202013%252C%2520Boston%252C%2520MA%252C%2520July%252023%25E2%2580%259324%252C%25202013%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit19b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Throner, S.; Cowen, S.; Russell, D. J.; Dakin, L.; Chen, H.; Larsen, N. A.; Godin, R. E.; Zheng, X.; Molina, A.; Wu, J.; Cheung, T.; Howard, T.; Garcia-Arenas, R.; Keen, N.; Ferguson, A. D.</span>Abstracts of Papers, 250th National Meeting of the American Chemical Society, Boston, MA, Aug 16–20, 2015; <span class="NLM_publisher-name">American Chemical Society</span>: <span class="NLM_publisher-loc">Washington, DC</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span> <span class="NLM_fpage">MEDI 513</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=MEDI+513&author=S.+Throner&author=S.+Cowen&author=D.+J.+Russell&author=L.+Dakin&author=H.+Chen&author=N.+A.+Larsen&author=R.+E.+Godin&author=X.+Zheng&author=A.+Molina&author=J.+Wu&author=T.+Cheung&author=T.+Howard&author=R.+Garcia-Arenas&author=N.+Keen&author=A.+D.+Ferguson&title=Abstracts+of+Papers%2C+250th+National+Meeting+of+the+American+Chemical+Society%2C+Boston%2C+MA%2C+Aug+16%E2%80%9320%2C+2015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DThroner%26aufirst%3DS.%26jtitle%3DAbstracts%2520of%2520Papers%252C%2520250th%2520National%2520Meeting%2520of%2520the%2520American%2520Chemical%2520Society%252C%2520Boston%252C%2520MA%252C%2520Aug%252016%25E2%2580%259320%252C%25202015%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2015%26spage%3DMEDI%2520513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Reynoird, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazur, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stellfeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flores, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lofgren, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hainaut, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaznowska, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khatri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stresemann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span> </span><span class="NLM_article-title">Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.1101/gad.275529.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1101%2Fgad.275529.115" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=772-785&author=N.+Reynoirdauthor=P.+K.+Mazurauthor=T.+Stellfeldauthor=N.+M.+Floresauthor=S.+M.+Lofgrenauthor=S.+M.+Carlsonauthor=E.+Brambillaauthor=P.+Hainautauthor=E.+B.+Kaznowskaauthor=C.+H.+Arrowsmithauthor=P.+Khatriauthor=C.+Stresemannauthor=O.+Gozaniauthor=J.+Sage&title=Coordination+of+stress+signals+by+the+lysine+methyltransferase+SMYD2+promotes+pancreatic+cancer&doi=10.1101%2Fgad.275529.115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1101%2Fgad.275529.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.275529.115%26sid%3Dliteratum%253Aachs%26aulast%3DReynoird%26aufirst%3DN.%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DStellfeld%26aufirst%3DT.%26aulast%3DFlores%26aufirst%3DN.%2BM.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DHainaut%26aufirst%3DP.%26aulast%3DKaznowska%26aufirst%3DE.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKhatri%26aufirst%3DP.%26aulast%3DStresemann%26aufirst%3DC.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DCoordination%2520of%2520stress%2520signals%2520by%2520the%2520lysine%2520methyltransferase%2520SMYD2%2520promotes%2520pancreatic%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2016%26volume%3D30%26spage%3D772%26epage%3D785%26doi%3D10.1101%2Fgad.275529.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Smid, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coolen, H. K. A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keizer, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Hes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Moes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Hartog, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stork, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plekkenpol, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niemann, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroomer, C. N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulp, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Stuivenberg, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCreary, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hesselink, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herremans, A. H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, C. G.</span><span> </span><span class="NLM_article-title">Synthesis, structure-activity relationships, and biological properties of 1-heteroaryl-4-[w-(1H-indol-3-yl)alkyl] piperazines, novel potential antipsychotics combining potent dopamine D<sub>2</sub> receptor antagonism with potent serotonin reuptake inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6855</span><span class="NLM_x">–</span> <span class="NLM_lpage">6869</span><span class="refDoi"> DOI: 10.1021/jm050148z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050148z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6855-6869&author=P.+Smidauthor=H.+K.+A.+C.+Coolenauthor=H.+G.+Keizerauthor=R.+van+Hesauthor=J.+de+Moesauthor=A.+P.+den+Hartogauthor=B.+Storkauthor=R.+H.+Plekkenpolauthor=L.+C.+Niemannauthor=C.+N.+J.+Stroomerauthor=M.+Tulpauthor=H.+H.+van+Stuivenbergauthor=A.+C.+McCrearyauthor=M.+B.+Hesselinkauthor=A.+H.+J.+Herremansauthor=C.+G.+Kruse&title=Synthesis%2C+structure-activity+relationships%2C+and+biological+properties+of+1-heteroaryl-4-%5Bw-%281H-indol-3-yl%29alkyl%5D+piperazines%2C+novel+potential+antipsychotics+combining+potent+dopamine+D2+receptor+antagonism+with+potent+serotonin+reuptake+inhibition&doi=10.1021%2Fjm050148z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm050148z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050148z%26sid%3Dliteratum%253Aachs%26aulast%3DSmid%26aufirst%3DP.%26aulast%3DCoolen%26aufirst%3DH.%2BK.%2BA.%2BC.%26aulast%3DKeizer%26aufirst%3DH.%2BG.%26aulast%3Dvan%2BHes%26aufirst%3DR.%26aulast%3Dde%2BMoes%26aufirst%3DJ.%26aulast%3Dden%2BHartog%26aufirst%3DA.%2BP.%26aulast%3DStork%26aufirst%3DB.%26aulast%3DPlekkenpol%26aufirst%3DR.%2BH.%26aulast%3DNiemann%26aufirst%3DL.%2BC.%26aulast%3DStroomer%26aufirst%3DC.%2BN.%2BJ.%26aulast%3DTulp%26aufirst%3DM.%26aulast%3Dvan%2BStuivenberg%26aufirst%3DH.%2BH.%26aulast%3DMcCreary%26aufirst%3DA.%2BC.%26aulast%3DHesselink%26aufirst%3DM.%2BB.%26aulast%3DHerremans%26aufirst%3DA.%2BH.%2BJ.%26aulast%3DKruse%26aufirst%3DC.%2BG.%26atitle%3DSynthesis%252C%2520structure-activity%2520relationships%252C%2520and%2520biological%2520properties%2520of%25201-heteroaryl-4-%255Bw-%25281H-indol-3-yl%2529alkyl%255D%2520piperazines%252C%2520novel%2520potential%2520antipsychotics%2520combining%2520potent%2520dopamine%2520D2%2520receptor%2520antagonism%2520with%2520potent%2520serotonin%2520reuptake%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6855%26epage%3D6869%26doi%3D10.1021%2Fjm050148z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22aR"><div class="casContent"><span class="casTitleNuber">22a</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhX2DP2WRy_tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit22b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Bailey, S.</span><span> </span><span class="NLM_article-title">The CCP4 suite - programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span><span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+CCP4+suite+-+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520CCP4%2520suite%2520-%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit22c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22cR"><div class="casContent"><span class="casTitleNuber">22c</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lhX2DP2WRy_tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit22c&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit22d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22dR"><div class="casContent"><span class="casTitleNuber">22d</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhX2DP2WRy_tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit22d&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit22e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Bricogne, G.; Blanc, E.; Brandl, M.; Flensburg, C.; Keller, P.; Paciorek, W.; Roversi, P.; Smart, O. S.; Vonrhein, C.; Womack, T. O.</span>BUSTER<span class="NLM_edition">version 2.8.0</span>; <span class="NLM_publisher-name">Global Phasing Ltd</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>.<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22e&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJL','PDB','5KJL'); return false;">PDB: 5KJL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJM','PDB','5KJM'); return false;">PDB: 5KJM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJN','PDB','5KJN'); return false;">PDB: 5KJN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJK','PDB','5KJK'); return false;">PDB: 5KJK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i96"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01303">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_50187"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01303">10.1021/acs.jmedchem.6b01303</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallographic and refinement statistics, KMT selectivity, preparation of <b>48</b>, preparation of <b>49</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_001.pdf">jm6b01303_si_001.pdf (214.44 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01303/suppl_file/jm6b01303_si_002.csv">jm6b01303_si_002.csv (3.09 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factor amplitudes of the SMYD2 inhibitor complexes have been deposited in the Protein Data Bank with the following accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJM">5KJM</a> (<b>30</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJK">5KJK</a> (<b>48</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJN">5KJN</a> (<b>47</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KJL">5KJL</a> (<b>5</b>), and will be released upon publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01303&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01303%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01303" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b98a98a13d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
