
<html lang="en"     class="pb-page"  data-request-id="989d842f-0fa7-4965-abb2-3991b6366ec7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.5b00773;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2015.58.issue-15"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Rebecca C.  Furze" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="Anne-Marie  Michon" /></meta><meta name="dc.Creator" content="Robert J.  Sheppard" /></meta><meta name="dc.Creator" content="Heather  Barnett" /></meta><meta name="dc.Creator" content="Hawa  Diallo" /></meta><meta name="dc.Creator" content="David P.  Dixon" /></meta><meta name="dc.Creator" content="Clement  Douault" /></meta><meta name="dc.Creator" content="Emma J.  Jones" /></meta><meta name="dc.Creator" content="Bhumika  Karamshi" /></meta><meta name="dc.Creator" content="Darren J.  Mitchell" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Christina  Rau" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="Thilo  Werner" /></meta><meta name="dc.Creator" content="Emmanuel H.  Demont" /></meta><meta name="dc.Description" content="ATAD2 is a bromodomain-containing protein whose overexpression is linked to poor outcomes in a number of different cancer types. To date, no potent and selective inhibitors of the bromodomain have ..." /></meta><meta name="Description" content="ATAD2 is a bromodomain-containing protein whose overexpression is linked to poor outcomes in a number of different cancer types. To date, no potent and selective inhibitors of the bromodomain have ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 31, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b00773" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b00773" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b00773" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b00773" /></link>
        
    
    

<title>Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b00773" /></meta><meta property="og:title" content="Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0021.jpeg" /></meta><meta property="og:description" content="ATAD2 is a bromodomain-containing protein whose overexpression is linked to poor outcomes in a number of different cancer types. To date, no potent and selective inhibitors of the bromodomain have been reported. This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b00773"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b00773">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b00773&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b00773&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b00773&amp;href=/doi/10.1021/acs.jmedchem.5b00773" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 6151-6178</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b00750" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b00775" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rebecca+C.++Furze">Rebecca C. Furze</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Michon">Anne-Marie Michon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Sheppard">Robert J. Sheppard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Heather++Barnett">Heather Barnett</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hawa++Diallo">Hawa Diallo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+P.++Dixon">David P. Dixon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Clement++Douault">Clement Douault</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emma+J.++Jones">Emma J. Jones</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bhumika++Karamshi">Bhumika Karamshi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Darren+J.++Mitchell">Darren J. Mitchell</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christina++Rau">Christina Rau</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thilo++Werner">Thilo Werner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H.++Demont">Emmanuel H. Demont</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>â </sup>Molecular Discovery Research, <sup>â¡</sup>Epinova Discovery Performance Unit, and <sup>Â§</sup>Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¥</span> <span class="aff-text">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div><div class="corresp-info"><strong>*</strong>(P.B.) E-mail: <a href="/cdn-cgi/l/email-protection#c898a9bda4e689e68aa9a5aaa7baa7bdafa088afbba3e6aba7a5"><span class="__cf_email__" data-cfemail="613100140d4f204f23000c030e130e1406092106120a4f020e0c">[emailÂ protected]</span></a>. Phone: +44 1438 763246. Fax: +44 1438 763352.</div><div class="corresp-info"><strong>*</strong>(E.H.D.; for materials or collaboration) E-mail: <a href="/cdn-cgi/l/email-protection#96d3fbfbf7f8e3f3fab8deb8d2f3fbf9f8e2d6f1e5fdb8f5f9fb"><span class="__cf_email__" data-cfemail="094c646468677c6c652741274d6c6466677d496e7a62276a6664">[emailÂ protected]</span></a>. Phone: +44 1438 764319. Fax +44 1438 768302.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b00773&amp;href=/doi/10.1021%2Facs.jmedchem.5b00773" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 6151â6178</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 31, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 May 2015</li><li><span class="item_label"><b>Published</b> online</span>31 July 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b00773" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00773</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6151%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaul%2BBamborough%252C%2BChun-wa%2BChung%252C%2BRebecca%2BC.%2BFurze%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D15%26contentID%3Dacs.jmedchem.5b00773%26title%3DStructure-Based%2BOptimization%2Bof%2BNaphthyridones%2Binto%2BPotent%2BATAD2%2BBromodomain%2BInhibitors%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6178%26publicationDate%3DAugust%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b00773"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2750</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">56</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b00773" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Rebecca&quot;,&quot;last_name&quot;:&quot;C. Furze&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Michon&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Sheppard&quot;},{&quot;first_name&quot;:&quot;Heather&quot;,&quot;last_name&quot;:&quot;Barnett&quot;},{&quot;first_name&quot;:&quot;Hawa&quot;,&quot;last_name&quot;:&quot;Diallo&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;P. Dixon&quot;},{&quot;first_name&quot;:&quot;Clement&quot;,&quot;last_name&quot;:&quot;Douault&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;J. Jones&quot;},{&quot;first_name&quot;:&quot;Bhumika&quot;,&quot;last_name&quot;:&quot;Karamshi&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Christina&quot;,&quot;last_name&quot;:&quot;Rau&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;Thilo&quot;,&quot;last_name&quot;:&quot;Werner&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H. Demont&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;6151-6178&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b00773&quot;},&quot;abstract&quot;:&quot;ATAD2 is a bromodomain-containing protein whose overexpression is linked to poor outcomes in a number of different cancer types. To date, no potent and selective inhibitors of the bromodomain have been reported. This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00773&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00773" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00773&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00773" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00773&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00773" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b00773&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00773&amp;href=/doi/10.1021/acs.jmedchem.5b00773" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b00773" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b00773" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26pmid%3D26230603%26genre%3Darticle%26aulast%3DBamborough%26date%3D2015%26atitle%3DStructure-Based%2BOptimization%2Bof%2BNaphthyridones%2Binto%2BPotent%2BATAD2%2BBromodomain%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D15%26spage%3D6151%26epage%3D6178%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291074" title="Purification">Purification</a>,</li><li><a href="/action/doSearch?ConceptID=291594" title="Silica">Silica</a>,</li><li><a href="/action/doSearch?ConceptID=292234" title="Chromatography">Chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291774" title="Nuclear magnetic resonance spectroscopy">Nuclear magnetic resonance spectroscopy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/jmcmar.2015.58.issue-15/20150813/jmcmar.2015.58.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">ATAD2 is a bromodomain-containing protein whose overexpression is linked to poor outcomes in a number of different cancer types. To date, no potent and selective inhibitors of the bromodomain have been reported. This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50734" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50734" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain-containing protein ATAD2 (ATPase family, AAA domain containing 2), also known as ANCCA (AAA nuclear coregulator cancer-associated protein), is a promising oncology target. Increased expression correlates with poor outcome in a range of cancers, and ATAD2 knockdown has been shown to modulate multiple relevant tumor cell growth factors.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> The fact that ATAD2 contains a bromodomain makes it an attractive area for research, as similar modules are known to be tractable to drug discovery in the tandem-bromodomain BET (bromodomain and extra terminal) family proteins (BRD2,3,4 and T).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> Despite this, relatively little work has been published, and there is a great need for small molecules with which to investigate phenotypic responses to inhibition of the ATAD2 bromodomain. Until very recently, the most potent inhibitors of the ATAD2 bromodomain were weak (â¥200 Î¼M) fragments, which are not suitable for use as cellular chemical probes and for which no optimization has been reported.</div><div class="NLM_p">In our preceding article, we described the discovery of the first reported low-micromolar inhibitors of the ATAD2 bromodomain (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> A weak hit <b>1</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) was synthesized within an ATAD2-targeted portion of an array based upon a quinolinone template. The crystallographic binding mode to the bromodomain of ATAD2 of analogues such as <b>2</b> was characterized, confirming that the napthyridone acts as an acetyl-lysine (KAc) mimetic via direct and through-water hydrogen-bonded interactions to Asn1064 and Tyr1021 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). The basic piperidine substituent makes an important additional contribution to ATAD2 binding through its interaction with Asp1071. Exploration of SAR at the C5 position of the naphthyridone demonstrated that 3-pyridyl derivatives provided a further boost to ATAD2 activity and resulted in the micromolar ATAD2 lead <b>3</b>.</div><div class="NLM_p">Because of the extensive effects of BET inhibitors,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> high selectivity over these proteins is required in chemical probes of other bromodomains. In a cellular probe for ATAD2, we aimed to achieve at least 100-fold selectivity over the BET family. Attempts to gain this selectivity through further exploration of the C5-position  met with limited success, with most analogues being more active against the BET family, exemplified by the N-terminal bromodomain of BRD4 (BRD4 BD1), than ATAD2. We now continue the story of our attempts to increase ATAD2 potency and introduce selectivity over the BET bromodomains.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. From hit to micromolar ATAD2 inhibitor. pIC<sub>50</sub>s of compounds <b>1</b>â<b>3</b> in ATAD2 peptide- and ligand-based TR-FRET assays and against BRD4 BD1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17477" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17477" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As this work progressed, we moved away from the peptide-based assay we had used to develop compound <b>3</b> to an alternative TR-FRET competition binding assay, in which a fluorescently labeled naphthyridone analogue of <b>3</b> was used instead of a peptide. Data generated for compounds in both assays showed that the two were in good agreement (<i>r</i><sup>2</sup> = 0.94 for 386 compounds over the TR-FRET IC<sub>50</sub> range 100 Î¼M to 100 nM; see Figure S1a, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). ATAD2 pIC<sub>50</sub> data reported here were generated using the ligand-based TR-FRET assay. For completeness, data from the peptide-based assay is also shown in Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p">Throughout the optimization, we also confirmed that the activity against the recombinant bromodomain was retained against ATAD2 protein from a more physiological source. The ATAD2 <i>Bromosphere</i> assay measures competitive binding to endogenous full-length ATAD2 from cell lysate. The correlation between the ATAD2 <i>Bromosphere</i> assay and the recombinant TR-FRET assay was also good (<i>r</i><sup>2</sup> > 0.67 for 237 compounds over a range of TR-FRET IC<sub>50</sub> between 100 Î¼M and 100 nM; Figure S1b, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Exploration of the 3â² Position to Find Lipophilic Interactions with the RVF Shelf</h3><div class="NLM_p">Having found difficulty in improving on the 3-pyridyl substituent at the naphthyridone C5 position of <b>3</b>, we considered ways to interact with other parts of the ATAD2 binding site. For the BET-family bromodomains, some areas outside the immediate KAc site have proved to be valuable in optimizing potency. One such area is the WPF shelf, named after the conserved TrpâProâPhe motif located nearby (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In the BRD4 N-terminal bromodomain (BRD4 BD1), key residues of the WPF shelf include Trp81 and Pro82 of the WPF motif, Met149, and especially Ile146. Potent BET inhibitors typically exploit the WPF shelf using aromatic or lipophilic substituents. In one recent example, aniline substituents projecting from a tetrahydroquinoline core onto the BET WPF shelf gained significant extra potency, and a similarly positioned phenyl ring is an essential part of first-generation BET inhibitors I-BET762 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a) and JQ-1.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9-11)</a></div><div class="NLM_p">A similar shallow shelf exists in ATAD2, but the amino acids surrounding it differ considerably from the BET bromodomains. As the BET WPF motif is replaced by RVF (Arg1007âVal1008âPhe1009) in ATAD2, this region has been called the RVF shelf.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Another important difference is the presence of Arg1077 in ATAD2 instead of Met149 (BRD4 numbering) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b and Figure S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Together, these changes result in a more polar and flexible environment on the RVF shelf in ATAD2 that is more challenging to interact with productively than in BRD4. While the rims of the two bromodomain shelves differ considerably, the floors are more similar. In ATAD2, the floor is formed by gatekeeper residue Ile1074 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b), which is identical to the corresponding residue in the second BET bromodomains, and only conservatively differs from the first (Ile146 and Val439 in BRD4 BD1 and BD2, respectively).</div><div class="NLM_p">We inspected the ATAD2 crystal structure complex with <b>3</b> to find ways to target the RVF shelf. The C3â² equatorial position of the piperidine seemed to offer a suitable vector (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). We first tried to interact with the lipophilic part of the RVF shelf surface produced by Ile1074, which is the closest part of the shelf to the piperidine ring of <b>3</b>. Modeling suggested that an ether linker could direct lipophilic groups toward this position in a low-energy conformation.</div><div class="NLM_p">This hypothesis could be explored rapidly in the C5-H naphthyridone template, via O-alkylation of the trans azido protected hydroxy-piperidine <b>5</b>, which was easily accessible from the known epoxide <b>4</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Reduction of the azide generated protected 3-alkoxy 4-amino piperidines <b>6</b> suitable for palladium-catalyzed coupling with the chloro naphthyridines <b>7a</b>,<b>b</b>, as previously described.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Deprotection then followed under acidic conditions. As well as giving access to racemic products, this route also enabled the isolation of inhibitors as single enantiomers, since the enantiomers of the azides, BOC-protected naphthyridines, or the final compounds could be easily separated by chiral chromatography.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Acetyl-lysine binding site of BRD4 BD1 bound to literature inhibitor I-BET762 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p5o">3p5o</a>)<sup>10</sup> showing detailed interactions (left) and electrostatic charge colored protein surface (right). (B) X-ray structure of ATAD2 bound to <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q">5a5q</a>), in the same orientation. Hydrogen bonds are shown as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Key Trans Piperidines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaN<sub>3</sub>, DMF/acetone/water, 80Â°C, 63%; (b) NaH, DMF, 0Â°C then RI or ROTf, 0 Â°C to room temperature, 10â100%; (c) Pd/C, MeOH, H<sub>2</sub> (1 bar), 69â100% or PPh<sub>3</sub>, THF, room temperature then water, reflux, 69â79%; (d) for R<sub>2</sub> = H: LiHMDS, Caddick catalyst, THF, 80â120 Â°C, microwave, 13â95%; or for R<sub>2</sub> = Bn: BrettPhos, BrettPhos precatalyst, THF, <i>t</i>BuONa, room temperature, 65â95%; (e) for R<sub>2</sub> = H: TFA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 73â92%; (f) for R<sub>2</sub> = Bn: H<sub>2</sub>, Pd/C, MeOH, room temperature, 99% then TFA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 84%.</p></p></figure><div class="NLM_p">The SAR (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) shows a size- and lipophilicity-dependent increase in potency up to a point, with acyclic groups producing relatively ineffective ATAD2 inhibitors (<b>9</b>â<b>12</b>). Alpha branching (as in <b>10</b>) was especially poorly tolerated compared to linear or Î²-branched alkyl chains (e.g., <b>11</b> and <b>12</b>). The cyclohexyl derivative <b>13</b> appeared to be the optimal cycloalkyl group, being marginally more ligand-efficient than the cyclopentyl (<b>14</b>) and comparable to the cycloheptyl (<b>15</b>). After separation of the two enantiomers of compound <b>13</b>, one enantiomer (<b>17</b>) was significantly more active than the other (<b>16</b>) by a factor of 1.3 logs. We judged that although the increase in potency in moving from <b>2</b> to <b>17</b> was achieved at the expense of some ligand efficiency and was driven by lipophilicity, this was acceptable to counter the hydrophilicity of the naphthyridone core, producing more balanced molecules with greater logD to promote cell permeability.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Data for C3â²-Substituents in the C5-H Naphthyridone Series<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0012.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0013.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Ligand efficiency (LE) â 1.37 Ã pIC<sub>50</sub>/(number of heavy atoms). For statistics, see Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div></div><div></div></div><div class="NLM_p">A 2.0 Ã crystal structure of ATAD2 bound to compound <b>13</b> was rapidly solved (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a). The addition of the 3â² substituent does not change the overall binding mode of the naphthyridone, which is unmoved relative to the bound structure of <b>2</b>. Interestingly, only the single (<i>R</i>,<i>R</i>) isomer of the trans racemic compound <b>13</b> was seen to bind in the ATAD2 active site (Figure S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). The structure confirmed our intended binding mode, with the 3â²-substituent cyclohexyl group positioned on the RVF shelf, in direct hydrophobic contact with Ile1074. The addition of the cyclohexyl substituent displaces two water molecules from ATAD2 that were visible in the complex with <b>2</b>. Both appear to be weakly bound. For example, one displaced water molecule, labeled H<sub>2</sub>O<sub>(R1077)</sub> in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a, forms only a single visible hydrogen bond to the Arg1077 guanidine group and is otherwise surrounded by the aliphatic side chains of Val1008, Leu1073, and Ile1074.</div><div class="NLM_p">For comparison, the BRD4 BD1 X-ray crystal structure with compound <b>13</b> was obtained by cocrystallization (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b). The naphthyridone of <b>13</b> binds to the acetyl-lysine pocket of BRD4 in a similar way to that of <b>3</b>.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> As with ATAD2, the electron density in the BRD4 BD1 X-ray structure was consistent with the (<i>R</i>,<i>R</i>) enantiomer. The cyclohexyl ring binds on the BRD4 BD1 WPF shelf, making lipophilic contact with the Ile146 side chain. As this hydrophobic interaction (with Ile1074 and Ile146 in ATAD2 and BRD4 BD1, respectively) is similar in both bromodomains, it is not surprising that the BRD4 BD1 potency tracks the ATAD2 potency closely for these and similar compounds (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Crystal structure of ATAD2 bound to <b>13</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a81">5a81</a>), superimposed on that of <b>2</b> (orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q">5a5q</a>) using alpha-carbon atoms of the active sites. The water molecule labeled H<sub>2</sub>O<sub>(R1077)</sub> on the RVF shelf close to Arg1077 is present in the complex with <b>2</b> and is displaced by <b>13</b>. (B) Crystal structure of BRD4 BD1 bound to <b>13</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a85">5a85</a>). Dashed yellow lines indicate hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">From the mixture of the two trans enantiomers of <b>13</b>, the (<i>R</i>,<i>R</i>) enantiomer was found to bind to both ATAD2 and BRD4 crystallographically. We assumed that this was the more potent of the two separated enantiomers and that, therefore, the more potent of the two separated enantiomers, <b>17</b>, had (<i>R</i>,<i>R</i>) stereochemistry. Later synthesis of enantiomerically pure material via chiral separation of azide intermediate <b>5</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> followed by crystallographic binding mode determination of the most active enantiomer showed that, indeed, the most potent single enantiomer was always the one associated with (<i>R</i>,<i>R</i>) stereochemistry (data not shown). We checked periodically to ensure that this remained true. From hereon, it is assumed that the (<i>R</i>,<i>R</i>) enantiomer is systematically more potent against ATAD2 than (<i>S</i>,<i>S</i>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Combining C5 and Lipophilic C3â² Substituents</h3><div class="NLM_p">Seeing little difference in the position of the naphthyridone between the ATAD2-bound X-ray structures of <b>2</b> and <b>13</b>, we hoped that the improvements in potency from modification at the C3â² position could be boosted further if other changes were additive. In our previous article, we described SAR at the naphthyridone C5-position.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> We introduced some of our favored C5-substituents, including the 3-pyridyl group of <b>3</b>, into C3â²-derivatized molecules according to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The derivatives <b>8</b> were selectively brominated at C5 to give <b>18</b>, which could be coupled with the required boronic acids to give <b>19.</b> Acidic deprotection of these intermediates gave the corresponding inhibitors in moderate to good yields.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, CHCl<sub>3</sub>, â10 Â°C, 84â95%; (b) Pd(OAc)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, cataCXium A, ArB(OH)<sub>2</sub>, 1,4-dioxane/water, 100 Â°C, microwave, 46â87%; (c), TFA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature (R<sub>1</sub> = H, 26â90%) or TFA reflux (R<sub>1</sub> = Bn, 39â99%).</p></p></figure><div class="NLM_p">Data for two example C5, C3â² disubstituted derivatives are presented in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. As reported previously for the 3â²-H piperidine analogues, the addition of 3-pyridyl substituents at the C-5 position improved ATAD2 potency significantly, conserving ligand efficiency (compare <b>20</b> and <b>22</b> to <b>17</b>). Consistent with the data shown in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> for 5-H naphthyridones, in the C5-(3-pyridyl) analogues the (<i>R</i>,<i>R</i>) enantiomer <b>20</b> was 1.3 logs more potent than the (<i>S</i>,<i>S</i>) enantiomer <b>21</b>. For the first time, these compounds clearly demonstrated the possibility of achieving our target ATAD2 activity (pIC<sub>50</sub> >7.0). In the case of compound <b>22</b>, this milestone was confirmed by potent activity measured against endogenous ATAD2 in the chemoproteomic <i>Bromosphere</i> assay (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Effect of C5-(3-Pyridine) Substituents<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0014.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 LE</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 <i>Bromosphere</i> pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">BRD4 BD1 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Î</th><th class="colsep0 rowsep0" align="center">Solubility Î¼M</th><th class="colsep0 rowsep0" align="center" char=".">LogD</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="left">220</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="left">170</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">0.22</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">196</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="left">494</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Î = difference between ATAD2 and BRD4 BD1 FRET pIC<sub>50</sub>. For statistics, see Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div></div></div><div class="NLM_p last">Also for the first time, the combination of C5 and C3â² groups in compounds <b>20</b> or <b>22</b> gave a modest but significant window of selectivity for ATAD2 over BRD4 (â¥0.7 log difference between the two pIC<sub>50</sub> values in TR-FRET assays). While encouraging, we felt that this level of selectivity was still too small to be able to distinguish any phenotypic readout due to ATAD2 from residual BET activity. We therefore looked for ways to widen the selectivity window. Up to this point, the selection of substituents at the 3â² position had been quite limited and did not seek to exploit the differences between the ATAD2 RVF shelf and the BET WPF shelf outlined above. The ATAD2 environment is significantly more polar, due to the presence of Arg1007 and Arg1077, than in BRD4 (Trp81 and Met149). <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> shows the different surroundings of the 4-position of the 3â²-cyclohexyl moiety of <b>13</b> when bound to ATAD2 and BRD4. As mentioned above, the loss of a water molecule H<sub>2</sub>O<sub>(R1077)</sub> bound to the Arg1077 side chain had been seen upon the addition of the cyclohexyl C3â² substituent. We speculated that introducing hydrogen-bond acceptor groups in this region might improve ATAD2 activity either through direct or water-bridged interactions with Arg1077. If direct or indirect H-bonds could not be formed, then we hoped at least that electronegative groups would be more tolerated in ATAD2 because of the more positive electrostatic surface in this area while being detrimental to BET activity by virtue of its greater lipophilicity.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Polar C3â² Substituents</h3><div class="NLM_p">As before, derivatives were first synthesized in the naphthyridone C5-H template. In most cases, these were prepared from the racemic trans hydroxy azido piperidine <b>5</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), and the enantiomers were separated for selected examples. In some cases (examples <b>26</b>, <b>31</b>â<b>33</b>, <b>44</b>), the alkylation with the C3â² substituent was performed after the Buchwald coupling between <b>7b</b> and the protected amine <b>23</b>, which was derived from the hydroxyl derivative <b>5</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Deprotection of the secondary alcohol gave <b>24</b>, which, after alkylation and further deprotection, gave the desired inhibitors.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) TBDMSCl, imidazole, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 71%; (b) H<sub>2</sub>, Pd/C, EtOH, room temperature, 95%; (c) BrettPhos, BrettPhos precatalyst, <i>t</i>BuONa, THF, room temperature to 60 Â°C, 97%; (d) TBAF, THF, room temperature, 71%; (e) chiral separation of enantiomers; (f) ROTf, <i>t</i>BuOK, THF, 0 Â°C to room temperature, 10â65% or NaH, THF (or DMF), 0 Â°C then ROTf, 0 Â°C to room temperature, 37â65%; (g) TFA, 60 Â°C to reflux, 45â94%.</p></p></figure><div class="NLM_p">Examples of such compounds with increased polarity in their C3â² substituents are shown in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>. Most of the racemic mixtures (e.g., <b>25</b>â<b>27</b>) had ATAD2 activity comparable to that of the racemic cyclohexyl benchmark <b>13</b> (pIC<sub>50</sub> 5.5). As before, when the mixtures were separated, most of the activity resided in a single enantiomer (compare <b>29</b> to <b>28</b>). The most promising polar analogue was the cyclic sulfone <b>30</b>. When chirally separated, the more active enantiomer <b>32</b> showed improved ATAD2 activity over the chiral cyclohexyl <b>17</b>. More importantly, consistent with our strategy for increasing selectivity over BRD4 by increasing negative polarity, both the tetrahydropyran <b>29</b> and cyclic sulfone <b>32</b> showed lower BRD4 activity than <b>17</b>. Further analogues using similar ring systems (e.g., <b>33</b>) did not show any advantage, as was also the case when extending the linker between the piperidine and the substituent (e.g., <b>34</b>).</div><div class="NLM_p">To better understand the interactions made by the cyclic sulfone and to confirm the identity of the more active enantiomer, a soak of the racemic cyclic sulfone <b>30</b> into ATAD2 crystals led to the X-ray structure shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. The binding mode is very similar to that of <b>13</b>. Again, the (<i>R</i>,<i>R</i>) isomer can be seen unambiguously binding within the active site (Figure S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). The main differences from the structure of <b>13</b> are the polar interactions made by the sulfone oxygen atoms, each of which accepts hydrogen bonds from the guanidinium group of Arg1077. The first sulfone oxygen lies in the  position occupied by H<sub>2</sub>O<sub>(R1077)</sub>, the Arg1077-bound water molecule in the complex with <b>2</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c). The second sulfone oxygen also hydrogen bonds to the Arg1077 terminal NH<sub>2</sub> group and is able to make a second interaction with the terminal NH<sub>2</sub> group of Arg1007, which moves slightly toward the sulfone.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (AâC) Three different views of the X-ray structure of ATAD2 bound to <b>30</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a82">5a82</a>), superimposed on <b>2</b> (orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q">5a5q</a>). In (C), a water molecule H<sub>2</sub>O<sub>(R1077)</sub> is shown. This lies near Arg1077 in the complex with <b>2</b> and is displaced by <b>30</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The enhanced selectivity over BRD4 of <b>32</b> relative to <b>17</b> is due to both increased ATAD2 potency and decreased BET activity (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The ATAD2 potency gain arises from the new direct hydrogen bonds to the arginines and the displacement of the weakly bound water molecules. The reduction of BET activity presumably results from placing polar sulfone oxygen atoms in an unfavorable lipophilic location near Trp81 and Met149 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b). The tetrahydropyran <b>29</b> is intermediate in polarity between the cyclohexyl <b>17</b> and cyclic sulfone <b>32</b> and shows intermediate selectivity.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Polar C3â² Substituents<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0015.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0016.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">Î = difference between ATAD2 and BRD4 BD1 FRET pIC<sub>50</sub>. For statistics, see Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a></p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Combining C5 and Polar C3â² Substituents</h3><div class="NLM_p">The tetrahydropyran and the cyclic sulfone C3â² substituents offered the best compromise of structural complexity, ATAD2 activity, and BET selectivity. Next, we assessed the impact of reintroduction of C5-(3-pyridine) groups on the overall profile of our fully functionalized inhibitors, prepared according to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. In agreement with earlier results, C5-(3-pyridyl) substituents were about 1 log more potent against ATAD2 than the direct C5-H analogues (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>: compare <b>20</b> to <b>17</b>, <b>37</b> to <b>29</b>, and <b>38</b> to <b>32</b>). As was found for the 5-H analogues, the most potent C5-(3-pyridyl)-substituted compounds were the C3â²-cyclic sulfones (<b>36</b>, <b>38</b>, and <b>40</b>, all with ATAD2 pIC<sub>50</sub> 6.9). The 3â² tetrahydopyran derivatives were uniformly slightly lower in potency than the cyclic sulfones by â¼0.3 logs (compare <b>35</b> to <b>36</b>, <b>37</b> to <b>38</b>, and <b>39</b> to <b>40</b>).</div><div class="NLM_p">Unexpectedly, addition of C5-(3-pyridine) substituents to compounds with polar C3â² groups also improved the selectivity for ATAD2 over BRD4 BD1 (compare, e.g., <b>38</b> to <b>32</b>). <b>38</b> was the first compound whose activity in the ATAD2 FRET assay was over 2 logs greater than in BRD4 BD1. Interestingly, introduction of the (3,5)-pyrimidine at C5 gave somewhat improved BET selectivity (in the case of the tetrahydropyran <b>41</b> compared to <b>35</b>, <b>37</b>, or <b>39</b>), but this came at the expense of some ATAD2 potency. This is in line with the effects of the C5-pyrimidine in the 3â²-unsubstituted naphthyridone (compound 67 in our preceding article).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Compound <b>38</b> almost matched our ATAD2 TR-FRET target activity and possessed even greater activity when measured against endogenous ATAD2 in the <i>Bromosphere</i> assay (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). It also met our probe criteria of 2 logs of selectivity over BRD4 BD1. While this was encouraging, concerns remained over the polarity of the C3â² cyclic sulfone and its possible impact on cell permeability. For early analogues such as <b>3</b> that retained significant BRD4 BD1 potency, IL-6 inhibitory activity in LPS-stimulated PBMCs was consistent with this BET-driven readout, showing that they were cell-permeable.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Less polar analogues, such as C3â²-cyclohexyl <b>20</b> or tetrahydropyran derivatives <b>37</b> and <b>39</b>, possess measurable artificial membrane permeability (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>) and also have expected levels of BET-driven IL-6 activity (Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). However, these have insufficient ATAD2 potency and/or BET selectivity to meet our probe criteria. For selective ATAD2 inhibitors such as <b>36</b>, <b>38</b> and <b>40</b>, BET-driven cellular IL-6 activity would be expected to be minimal, so this could no longer be used as an indication of cell penetration. The low logD and artificial membrane permeability of these sulfone-containing compounds (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>) did suggest that their concentration within cells could be relatively limited.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Combinations of Optimal C5 and C3â² Substituents<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0017.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0018.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">All are the most potent single enantiomers. Î = difference between ATAD2 and BRD4 BD1 FRET pIC<sub>50</sub>. For statistics, see Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a></p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Increasing Permeability</h3><div class="NLM_p">Because of these concerns, we attempted to increase the permeability of the sulfone-bearing inhibitors by reducing the polar surface area and the number of hydrogen bond donors and acceptors. In parallel, we tried to increase logD by finding areas where small lipophilic substituents could be tolerated. <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a> highlights some results of this strategy. First, to remove one heteroatom, the naphthyridone core was changed back to the quinolinone (as in the original hit <b>1</b>). The resulting compounds included the direct quinolinone analogue of <b>38</b> (the racemic <b>42</b> and its most potent single enantiomer, <b>43</b>). Although the removal of the naphthyridone nitrogen did increase logD by â¼0.5, this did not translate into measurable artificial membrane permeability.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Increasing Lipophilicity in the Sulfone Series<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0019.gif" alt="" id="fx8" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">Z</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 LE</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 <i>Bromophere</i> pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">BRD4 BD1 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">Î</th><th class="colsep0 rowsep0" align="center">Solubility Î¼M</th><th class="colsep0 rowsep0" align="center" char=".">LogD</th><th class="colsep0 rowsep0" align="center">Permeability (nm/s, pHÂ 7.4)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="left">179</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="left"><3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="left">217</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="left"><3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="left">â¥266</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="left"><3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="left"><3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="left">323</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="left"><3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="left">â¥341</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="left"><10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">All are single (<i>R</i>,<i>R</i>) enantiomers except for <b>42</b>, which is a racemic mixture of trans enantiomers. Î = difference between ATAD2 and BRD4 BD1 FRET pIC<sub>50</sub>. LE â 1.37 Ã pIC<sub>50</sub>/(number of heavy atoms). For statistics, see Table S1, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div></div></div><div class="NLM_p">A crystal structure of <b>42</b> was solved in ATAD2. As before, the (<i>R</i>,<i>R</i>) enantiomer was observed to bind (Figure S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). The binding mode is very similar to that of <b>30</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). The key protein contacts made by the quinolinone of <b>42</b> and the naphthyridone of <b>30</b> are essentially identical. The piperidines of each compound also overlay perfectly. The increase of potency conferred by the C5-(3-pyridyl) group can be rationalized in the same way as that previously used to explain the increase in potency for <b>3</b> over <b>2</b>.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The pyridine nitrogen forms a direct hydrogen bond with the backbone NH group of Asp1014 in the ZA loop of ATAD2 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A, B) Two orientations of the ATAD2 X-ray complex with <b>42</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a83">5a83</a>), superimposed on the structure of <b>30</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a82">5a82</a>). The position of the mobile side chain of Arg1007 in the complex with <b>30</b> is shown in magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the RVF shelf, the hydrogen bonds between Arg1077 and the sulfone of <b>42</b> are also the same as seen in the complex with <b>30</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). Unexpectedly, the side chain of Arg1007 moves away from the sulfone in the complex with <b>42</b>, suggesting that their direct interaction seen in the complex with <b>30</b> is not critical for sulfone recognition.</div><div class="NLM_p">The ATAD2 activity of <b>43</b>, the (<i>R</i>,<i>R</i>) enantiomer of <b>42</b>, is similar to that of <b>38</b> (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). This is as expected given the lack of any visible interactions in the X-ray structures involving the naphthyridone N7 atom or the corresponding quinolinone C7 atom. For reasons not well understood, a small but consistent decrease in selectivity over BET was observed for the quinolinones relative to the naphthyridones (compare <b>43</b> to <b>38</b>, and other analogues not shown).</div><div class="NLM_p last">Finding no advantage in the quinolinone series over the naphthyridones, we investigated the addition of small lipophilic substituents elsewhere in the structure. These were tolerated at some positions on the C3â² substituent with no impact on activity or selectivity. For example, we tried to mask some of the polarity of the cyclic sulfone by methylation upon either side (<b>44</b>). This change was tolerated in terms of potency, but it did not affect measured logD or permeability, and it was somewhat detrimental to solubility, which was otherwise good in this series. Perhaps more surprisingly, replacement of the naphthyridone C3-methyl group, which binds in the acetyl-lysine methyl pocket, with a bulkier ethyl substituent (<b>45</b>) was also well tolerated. Unfortunately, this did not improve the measured permeability. Finally, small piperidine N-alkyl substituents were introduced. Piperidine N-methylation led to a modest loss of potency at ATAD2 for compound <b>46</b> relative to <b>38</b>, probably due to the close proximity of the N-methyl group to the side chain of Asp1071 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). A similar reduction was seen in the <i>Bromosphere</i> assay (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Although not quite reaching our ATAD2 target potency, <b>46</b> had higher logD than <b>38</b> and lower BRD4 BD1 activity, resulting in a slightly wider selectivity window.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Further Characterization</h3><div class="NLM_p">This work was primarily driven by activity against the ATAD2 bromodomain, measured in the TR-FRET assay, as well as in the <i>Bromosphere</i> assay as described above. For some compounds, p<i>K</i><sub>d</sub> values were also derived by monitoring direct binding to the ATAD2 bromodomain using surface plasmon resonance (SPR). SPR p<i>K</i><sub>d</sub> values were in excellent agreement with the TR-FRET pIC<sub>50</sub>s (<i>r</i><sup>2</sup> > 0.95, <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a> and Figure S1c, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). This confirmed the sub-100 nM (pIC<sub>50</sub> â¥ 7.0) potency of key molecules such as <b>38</b>. Much of the important SAR also tracked, for example, the decrease in potency caused by the piperidine N-methylation of <b>38</b> to give <b>46</b>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. ATAD2 SPR Binding Constants Compared to TR-FRET pIC<sub>50</sub> Values</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 SPR <i>K</i><sub>d</sub> Î¼M</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 SPR p<i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 TR-FRET pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">0.05</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="char" char=".">0.09</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">0.06</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="char" char=".">0.20</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td></tr></tbody></table></div></div><div class="NLM_p">The selectivity of <b>46</b> against the wider bromodomain family was assessed in the BROMO<i>scan</i> panel (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a> and Table S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The ATAD2 p<i>K</i><sub>i</sub> of 7.7 (<i>K</i><sub>i</sub> 20 nM) in this format was slightly more potent than in other assays. Similar activity was found against the closest homologue of ATAD2, ATAD2B (p<i>K</i><sub>i</sub> 7.4), which was expected given the high sequence similarity between the two bromodomains (73% identity, with only conservative changes in the KAc site, see Figure S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Selectivity over the BET bromodomains was confirmed, with a window of at least 2.6 logs (>400-fold), BRD4 BD1 being the most potent (p<i>K</i><sub>i</sub> 5.1). For the other bromodomains tested, the highest activity was with the second bromodomains of TAF1 and TAF1L (p<i>K</i><sub>i</sub> 7.3 and 6.9, respectively). Weaker activity remained against the BRPF family (p<i>K</i><sub>i</sub> 5.3/6.4/6.1 against BRPF1/2/3, respectively) and CECR2 (p<i>K</i><sub>i</sub> 6.1). The cellular consequences of inhibition of these bromodomains is currently unknown, so we were satisfied to find this highly restricted off-target profile among the bromodomain family and to verify the excellent selectivity over the BET bromodomains.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selectivity of <b>46</b> against other bromodomains. The color scale ranges from p<i>K</i><sub>i</sub> 7.7 (red) to <4.3 (blue, triangles denote â<â modifier).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In parallel, we assessed the selectivity of <b>46</b> and other examples against a panel of endogenous bromodomain-containing proteins present in cell lysates using a <i>Bromosphere</i> chemoproteomic assay. A focused panel of immobilized, unselective bromodomain inhibitors was able to pull down 19 of these proteins, as detected using mass spectrometry. The p<i>K</i><sub>d</sub> values of <b>20</b>, <b>43</b>, and <b>46</b> for those full-length proteins were determined using competition doseâresponse experiments. <b>46</b> shows clearly improved selectivity over <b>20</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>c), as also found for the BET bromodomains by TR-FRET. For <b>46</b>, the bromodomains detected to bind most strongly are consistent with those found in the BROMO<i>scan</i> panel, although, pleasingly, it appears that the compound shows greater selectivity for ATAD2 and ATAD2B when measured using endogenous proteins.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Selectivity of (A) <b>20</b>, (B) <b>43</b>, and (C) <b>46</b> for ATAD2 against other bromodomain containing proteins using MS-detected pull downs from cell lysates. Specified values are apparent p<i>K</i><sub>d</sub>s. Values <4 indicate that no p<i>K</i><sub>d</sub> was determined up to 100 Î¼M compound concentration. No bar indicates that protein was not identified.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Here, we have presented the first report of successful optimization of inhibitors of the ATAD2 bromodomain to sub-100 nMaffinity. Potency and BET selectivity were improved with the help of extensive crystallography in ATAD2 and BRD4 BD1. This led to the identification of compounds such as <b>38</b> (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>), which matched our potency and selectivity criteria, showing >100-fold selectivity over the BET bromodomains. This selectivity was achieved by optimizing the 3â² substituent for complementarity with the ATAD2 site and  against the BET site, resulting in the C3â²-cyclic sulfone moiety found within <b>38</b>. However, this came at the expense of greater hydrophilicity, and the passive permeability of the sulfone-containing members of this series remained concerningly low for use in cellular assays. We were partly able to mitigate this in compound <b>46</b>, although the potency was lower than we had hoped and some off-target activity remained against other bromodomains. The conclusion of the optimization of this series to generate a cell-permeable and significantly more selective ATAD2 chemical probe will be the subject of our next publication.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Summary of Properties of <b>38</b> and <b>46</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0020.gif" alt="" id="fx9" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center"><b>38</b></th><th class="colsep0 rowsep0" align="center"><b>46</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 pIC<sub>50</sub> (TR-FRET)</td><td class="colsep0 rowsep0" align="center">6.9</td><td class="colsep0 rowsep0" align="center">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 pIC<sub>50</sub> (Peptide FRET)</td><td class="colsep0 rowsep0" align="center">6.9</td><td class="colsep0 rowsep0" align="center">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 pIC<sub>50</sub> (<i>Bromosphere</i>)</td><td class="colsep0 rowsep0" align="center">7.5</td><td class="colsep0 rowsep0" align="center">7.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 p<i>K</i><sub>d</sub> (SPR)</td><td class="colsep0 rowsep0" align="center">7.1</td><td class="colsep0 rowsep0" align="center">6.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 p<i>K</i><sub>i</sub> (BROMO<i>scan</i>)</td><td class="colsep0 rowsep0" align="center">Â </td><td class="colsep0 rowsep0" align="center">7.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2 BD1/BD2 pIC<sub>50</sub> (TR-FRET)</td><td class="colsep0 rowsep0" align="center">4.5/<4.5</td><td class="colsep0 rowsep0" align="center"><4.3/<4.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3 BD1/BD2 pIC<sub>50</sub> (TR-FRET)</td><td class="colsep0 rowsep0" align="center"><4.3/<4.3</td><td class="colsep0 rowsep0" align="center"><4.3/<4.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1/BD2 pIC<sub>50</sub> (TR-FRET)</td><td class="colsep0 rowsep0" align="center">4.8/5.3</td><td class="colsep0 rowsep0" align="center">4.1/<3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRDT BD1/BD2 pIC<sub>50</sub> (TR-FRET)</td><td class="colsep0 rowsep0" align="center">4.5/<4.3</td><td class="colsep0 rowsep0" align="center"><4.3/<Â 4.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Chrom LogD (pH 7.4)</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="center">2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ArtificialÂ membraneÂ permeabilityÂ (nm/s,Â pHÂ 7.4)</td><td class="colsep0 rowsep0" align="center"><3</td><td class="colsep0 rowsep0" align="center"><10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND solubility (Î¼M)</td><td class="colsep0 rowsep0" align="center">179</td><td class="colsep0 rowsep0" align="center">â¥341</td></tr></tbody></table></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Protein Expression, SPR, BET Bromodomain Assays, PBMC IL6 Assay, and Physicochemical Property Measurement</h3><div class="NLM_p last">These were carried out as described previously.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> ATAD2 Synthetic Ligand Competition TR-FRET Binding Assay</h3><div class="NLM_p last">Compounds were titrated from 10 mM in 100% DMSO, and 100 nL was transferred to a low-volume black 384-well microtiter plate using a Labcyte Echo 555. A Thermo Scientific Multidrop Micro was used to dispense 5 Î¼L of 5 nM FLAG-6His-Tev-ATAD2(981â1121) in 50 mM Hepes, 150 mM NaCl, 5% Glycerol, 1 mM CHAPS, and 1 mM DTT, pH 7.4, in the presence of 100 nM Alexa Fluor 488-labeled ligand. After equilibrating for 30 min in the dark at room temperature, the ATAD2 proteinâfluorescent ligand interaction was detected using TR-FRET following a 5 Î¼L addition of 1.5 nM Lanthascreen Elite Tb-anti His antibody (Invitrogen, PV5863) in assay buffer. Time-resolved fluorescence (TRF) was then detected on a TRF laser equipped PerkinElmer Envision multimode plate reader (excitation = 337 nm; emission 1 = 520 nm; emission 2 = 495 nm; dual wavelength bias dichroic = 400, 505 nm). TR-FRET ratio was calculated using the following equation: ratio = ((acceptor fluorescence at 520 nm)/(donor fluorescence at 495 nm)) Ã 1000. TR FRET ratio data was normalized to a mean of 16 replicates per microtiter plate of both 10 Î¼M GSK3190320 (a potent naphthyridone ATAD2 inhibitor similar to <b>37</b>; see Figure S4, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>) and 1% DMSO controls, and IC<sub>50</sub> values were determined for each of the compounds tested by fitting the fluorescence ratio data to a four-parameter model, <i>y</i> = <i>a</i> + ((<i>b</i> â <i>a</i>)/(1 + (10<i><sup>x</sup></i>/10<i><sup>c</sup></i>)<i><sup>d</sup></i>), where <i>a</i> is the minimum, <i>b</i> is the Hill slope, <i>c</i> is the IC<sub>50</sub>, and <i>d</i> is the maximum.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Preparation of Cell Fractions for Chemoproteomic Profiling</h3><div class="NLM_p last">Nuclear extract was produced from HuT78 cells grown at 1 Ã 10<sup>6</sup> to 5 Ã 10<sup>6</sup> cells/mL in spinner flasks. Cells were collected by centrifugation, washed with PBS, and resuspended in three volumes with hypotonic buffer A (10 mM Tris-Cl, pH 7.4, 1.5 mM MgCl<sub>2</sub>, 10 mM KCl, 25 mM NaF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mM DTT, and 1 Roche protease inhibitor tablet per 25 mL). After 15 min, cells were homogenized with a Dounce homogenizer. Nuclei were collected by centrifugation (2500<i>g</i>), washed with hypotonic buffer A, and homogenized in one volume of extraction buffer B (50 mM Tris-Cl, pH 7.4, 1.5 mM MgCl<sub>2</sub>, 20% glycerol, 420 mM NaCl, 25 mM NaF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mM DTT, 400 units/mL DNase I, and 1 Roche protease inhibitor tablet per 25 mL). Extraction was allowed to proceed under agitation for 30 min at 4 Â°C before the extract was clarified by centrifugation at 13â¯000<i>g</i>. The extract was diluted in buffer D (50 mM Tris-Cl, pH 7.4 (RT), 1.5 mM MgCl<sub>2</sub>, 25 mM NaF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 0.6% NP-40, 1 mM DTT, and Roche protease inhibitors), and aliquots were snap-frozen in liquid nitrogen and stored at â80 Â°C. Tight chromatin-associated protein-enriched fractions were prepared by resuspending the remaining pellet in 10 volumes of high-salt extraction buffer (20 mM Hepes, pH 7.4, 1.5 mM MgCl<sub>2</sub>, 1 M KCl, 10% glycerol, 0.1% NP40, 0.5 mM DTT, and Roche protease inhibitors) with 8â10 cycles of 10 s on/50 s off sonication with an ultrasound homogenizer (Bandelin Sonoplus). After sonication and incubation for 45 min at 4 Â°C, the homogenate was clarified at 8000<i>g</i>. The salt concentration was adjusted to 150 mM KCl through stepwise dialysis before ultracentrifugation at 100â¯000<i>g</i> for 20 min. Aliquots were snap-frozen in liquid nitrogen and stored at â80 Â°C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> ATAD2 <i>Bromosphere</i> Assay.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></h3><div class="NLM_p last">The ATAD2 <i>Bromosphere</i> competition binding assay was performed in a 384-well format using an antibody-based array readout. Briefly, 60â120 Î¼g HuT-78 chromatin lysate and 1.5 Î¼L of ATAD2 capturing matrix (immobilized GSK2998821 on NHS Sepharose, structure not shown) per well (final volume 75 Î¼L) were incubated in the absence or presence of inhibitor compound at 4 Â°C. Compounds were tested in a doseâresponse format with a starting concentration of 100â200 Î¼M, applying 1:3 dilution steps for, in total, 10 or 11 data points. DMSO concentration was 1% (v/v). After 2 h incubation, the nonbound fraction was removed by washing the beads with lysis buffer (50 mM Tris-HCl (pH 7.4), 0.1% (v/v) Igepal-CA630, 5% (v/v) glycerol, 150 mM NaCl, 1.5 mM MgCl<sub>2</sub>, 25 mM NaF, 1 mM sodium vanadate, 1 mM dithiothreitol). Proteins retained on the beads were eluted in SDS sample buffer (100 mM Tris (pH 7.4), 4% (w/v) SDS, 20% (v/v) glycerol, 0.01% (w/v) bromophenol blue, 50 mM dithiothreitol) and spotted on nitrocellulose membranes (400 nL per spot) using an automated pin-tool liquid transfer (Biomek FX, Beckman). After drying, the membranes were rehydrated in 20% (v/v) ethanol and processed for detection with a specific anti-ATAD2 antibody (Sigma; 29424 (1:300) in the presence of 0.4% Tween) followed by incubation with an IRDye 800-labeled secondary antibody for visualization (anti-rabbit 926-32211, LICOR (1:5000)) in the presence of 0.2% Tween. Spot intensities were quantified using a LiCOR Odyssey scanner.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemoproteomic Assay with Dose-Dependent Competition</h3><div class="NLM_p last">Affinity profiling assays were performed as described previously.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> Briefly, sepharose beads were derivatized with nonselective bromodomain inhibitors of various chemotypes containing a primary amine handle (e.g., 0.6 mM GSK2975292 or 2 mM GSK2893910 or 1 mM GSK3040504, structures not shown) via amide-bond formation on activated NHS beads. Increasing concentrations of tested ATAD2 inhibitor compounds were spiked into HuT-78 mixed chromatin and nuclear extracts and incubated for 45 min at 4 Â°C. Derivatized sepharose beads (35 Î¼L beads per sample) were equilibrated in lysis buffer and incubated with cell extract containing compound. Beads were washed with lysis buffer containing 0.2% NP-40 and eluted with 2Ã SDS sample buffer supplemented with DTT.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Mass Spectrometry and Data Analysis for Chemoproteomic Assay</h3><div class="NLM_p last">Sample preparation, labeling with TMT isobaric mass tags, peptide fractionation, and mass spectrometric analyses were performed essentially as described earlier.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The pooled peptide samples were either fractionated with a reverse phase at pH 12 as described previously<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and subsequently analyzed by LC-MS with a 1D plus (Eksigent) LC connected to a Orbitrap XL (Thermo Scientific) or directly analyzed using Nanoacquity UPLC system (Waters) connected to an Orbitrap Elite mass spectrometer (Thermo Scientific). In both cases, the peptides were separated on custom-made 50 cm Ã 100 Î¼M reversed-phase columns (Reprosil) at 40 Â°C. Gradient elution was performed from 2 to 40% acetonitrile in 0.1% formic acid over 2 h. A targeted data acquisition strategy was applied as described<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> for protein identification using a list of 20 peptide charge mass combinations per bromodomain protein. Mascot 2.3.02 (Matrix Science) was used for protein identification, using 10 ppm mass tolerance for peptide precursors and 20 mDa (HCD) or 0.8 Da (CID) mass tolerance for fragment ions. Carbamidomethylation of cysteine residues and TMT modification of lysine residues were set as fixed modifications, and methionine oxidation, N-terminal acetylation of proteins, and TMT modification of peptide N-termini were set as variable modifications. The search database consisted of a customized version of the IPI protein sequence database combined with a decoy version of this database created using a script supplied by Matrix Science. All identified proteins were quantified; FDR for quantified proteins was âª0.1%. For dose-dependent experiments, doseâresponse curves were fitted using R (<a href="http://www.r-project.org/" class="extLink">http://www.r-project.org/</a>) and the drc package (<a href="http://www.bioassay.dk" class="extLink">http://www.bioassay.dk</a>), as described previously.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> IC<sub>50</sub> values were corrected for the influence of the immobilized ligand on the binding equilibrium using the ChengâPrusoff relationship.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> ATAD2 Crystallography</h3><div class="NLM_p last">Apo crystals of ATAD2 981â1108, expressed and detagged as described previously,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> were typically grown at 4 Â°C in 96-well MRC plates using well solutions under a range of conditions spanning 0.1 M Tris-HCl, pH 7.0â8.0, 1.2â1.5 M ammonium sulfate, and 20â25% PEG3350. The sitting drops were made up of 3:1 ratio of protein/well solution, and the hexagonal crystals often grew from phase-separated drops. Compounds were soaked into apo crystals overnight either from solid or using 1â5 mM ligand solution diluted from a 100â200 mM DMSO stock solution. Soaked crystals were harvested in a cryo-loop, briefly transferred into a cryoprotectant buffer of 20% ethylene glycol + 80% well solution, before being plunge-frozen in liquid nitrogen. Data from single crystals were collected at 100 K on an in-house Rigaku FR-E+ SuperBright/Saturn A200 detector/ACTOR robotic system or at the Diamond Light Source (Oxford) or the Europeon Synchrotron Radiation Facility (Genoble). Data processing was performed using DENZO,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> MOSFLM,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> or XDS<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (within AUTOPROC [Global Phasing Limited]) and scaled using either SCALEPACK,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> SCALA,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> or AIMLESS<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> [within the CCP4 programming suite<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>]. Structures were solved by Fourier synthesis using REFMAC<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (via CCP4) starting from a previously determined in-house structure; model-building was performed using COOT,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and refinement, using REFMAC via CCP4. The statistics for the data collection and refined coordinates are given in Table S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, and OMIT maps are provided in <a class="ref internalNav" href="#notes-1" aria-label="Figure S3">Figure S3</a>. The final crystal structures have been deposited in the Protein Data Bank under the accession codes shown in Table S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> BRD4 BD1 Crystallography</h3><div class="NLM_p last">Detagged BRD4-BD1(44â168), generated as previously described,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> was cocrystallized with at least 3:1 excess of compound at 9â10 mg/mL in sitting drops using a 96-well MRC plate with a well solution indicated in <a class="ref internalNav" href="#notes-1" aria-label="Table S4">Table S4</a>. Crystals were briefly transferred into a cryoprotectant solution consisting of well solution with 20% ethylene glycol added prior to flash freezing in liquid nitrogen. Data from single crystals at 100 K on an in-house Rigaku FR-E<sup>+</sup> SuperBright/Saturn A200 detector/ACTOR robotic system or at the Europeon Synchrotron Radiation Facility (Genoble) were processed using XDS<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and scaled using SCALA<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> or AIMLESS<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> within the CCP4 programming suite.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Molecular replacement solutions were determined with a previously collected in-house structure. The <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> cells have a single molecule in the asymmetric unit. Manual model building was performed using COOT<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and refined using REFMAC<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> via CCP4. There was clear difference density for the ligands in the acetylated lysine binding site, allowing the ligand to be unambiguously modeled. The statistics for the data collection and refined coordinates are given in Table S4, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, and electron density maps are shown in Figure S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. The final crystal structures have been deposited in the Protein Data Bank under the accession codes shown in Table S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Chemistry</h3><div class="NLM_p last">All solvents were purchased from Sigma-Aldrich (Hy-Dry anhydrous solvents), and commercially available reagents were used as received. All reactions were followed by TLC analysis (TLC plates GF254, Merck) or LCMS (liquid chromatography mass spectrometry) using a Waters ZQ instrument. NMR spectra were recorded on a Bruker nanobay 400 MHz or a Bruker AVII+ 600 MHz spectrometers and are referenced as follows: <sup>1</sup>H NMR (400 or 600 MHz), internal standard TMS at Î´ = 0.00; <sup>13</sup>C NMR (100.6 or 150.9 MHz), internal standard CDCl<sub>3</sub> at Î´ = 77.23 or DMSO-<i>d</i><sub>6</sub> at Î´ = 39.70. Column chromatography was performed on prepacked silica gel columns (30â90 mesh, IST) using a biotage SP4. Mass spectra were recorded on Waters ZQ (ESI-MS) and Q-Tof 2 (HRMS) spectrometers. Mass-directed auto prep was performed on a Waters 2767 with a MicroMass ZQ mass spectrometer using a Supelco LCABZ++ column.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Synthetic Methods and Characterization of Compounds</h3><div class="NLM_p last">Abbreviations for multiplicities observed in NMR spectra: s; singulet; br s, broad singulet; d, doublet; t, triplet; q, quadruplet; p, pentuplet; spt, septuplet; m, multiplet. The purity of all compounds was determined by LCMS and <sup>1</sup>H NMR and was always >95%. See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> LCMS</h3><div class="NLM_p">UPLC analysis was conducted on an Acquity UPLC BEH C18 column (50 mm Ã 2.1 mm, i.d. 1.7 Î¼m packing diameter) at 40 Â°C. Formate method: solvents employed were A = 0.1% v/v solution of formic acid in water and B = 0.1% v/v solution of formic acid in acetonitrile. High-pH method: the solvents employed were A = 10 mM ammonium hydrogen carbonate in water adjusted to pH 10 with ammonia solution and B = acetonitrile. For both methods, the gradient employed is shown in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. LCMS Gradients Employed</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">time (min)</th><th class="colsep0 rowsep0" align="center">flow rate (mL/min)</th><th class="colsep0 rowsep0" align="center">%Â A</th><th class="colsep0 rowsep0" align="center">%Â B</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">100</td></tr></tbody></table></div></div><div class="NLM_p last">The UV detection was a summed signal from wavelength of 210 to 350 nm. Mass spectra were obtained on a Waters ZQ instrument; ionization mode, alternate-scan positive and negative electrospray; scan range, 100â1000 AMU; scan time, 0.27 s; inter scan delay, 0.10 s.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> MDAP</h3><div class="NLM_p last">Formate method: HPLC analysis was conducted on either a Sunfire C18 column (100 mm Ã 19 mm, i.d. 5 Î¼m packing diameter) or a Sunfire C18 column (150 mm Ã 30 mm, i.d. 5 Î¼m packing diameter) at ambient temperature. The solvents employed were A = 0.1% v/v solution of formic acid in water and B = 0.1% v/v solution of formic acid in acetonitrile. Run as a gradient over either 15 or 25 min (extended run) with a flow rate of 20 mL/min (100 mm Ã 19 mm, i.d 5 Î¼m packing diameter) or 40 mL/min (150 mm Ã 30 mm, i.d. 5 Î¼m packing diameter). High-pH method: HPLC analysis was conducted on either an Xbridge C18 column (100 mm Ã 19 mm, i.d. 5 Î¼m packing diameter) or a Xbridge C18 column (100 mm Ã 30 mm, i.d. 5 Î¼m packing diameter) at ambient temperature. The solvents employed were A = 10 mM ammonium bicarbonate in water, adjusted to pH 10 with ammonia solution, and B = acetonitrile. Run as a gradient over either 15 or 25 min (extended run) with a flow rate of 20 mL/min (100 mm Ã 19 mm, i.d. 5 Î¼m packing diameter) or 40 mL/min (100 mm Ã 30 mm, i.d. 5 Î¼m packing diameter). For both methods, the UV detection was a summed signal from wavelength of 210 to 350 nm. Mass spectra were obtained on a Waters ZQ instrument; ionization mode, alternate-scan positive and negative electrospray; scan range, 100â1000 AMU; scan time, 0.50 s; inter scan delay, 0.20 s.</div></div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (3<i>R</i>,4<i>R</i>)-<i>tert</i>-Butyl 4-Azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>)</h3><div class="NLM_p last">A solution of <i>tert</i>-butyl 7-oxa-3-azabicyclo[4.1.0]heptane-3-carboxylate (<b>4</b><a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a>) (11 g, 55 mmol) in DMF (100 mL) at room temperature was treated with a solution of sodium azide (6 g, 92 mmol) in water (40 mL) and acetone (60 mL), and the resulting mixture was stirred at 80 Â°C for 2 h and was then cooled to room temperature and diluted with EtOAc (200 mL) and water (200 mL). The layers were separated, and the organic phase was washed with water (200 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give a pale yellow liquid. Purification of this residue by flash chromatography on silica gel (340 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (8.4 g, 63%) as a white solid. Later eluting fractions were combined and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-azido-4-hydroxy piperidine-1-carboxylate (1.9 g, 14%) as a colorless solid. <sup>1</sup>H NMR (<i>T</i> = 120 Â°C, 400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 4.97 (br s, 1H) 3.89â4.01 (m, 1H), 3.82 (d, <i>J</i> = 13.3 Hz, 1H), 3.27â3.44 (m, 2H), 2.83â2.90 (m 1H), 2.69 (dd, <i>J</i> = 12.8, 9.1 Hz, 1H), 1.84â2.01 (m, 1H), 1.44 (s, 9H), 1.27â1.40 (m, 1H). The racemic mixture was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralpak AD-H (4.6 mm i.d. Ã 25 cm), using 20% EtOH in heptane at a flow rate of 1 mL/min. The material (4.9 g) was dissolved in a mixture of EtOH/heptane (1:1) (500 mg in 2 mL). The purification was carried out on a Chiralpak AD-H (30 mm Ã 25 cm), using 20% EtOH in heptane at a flow rate of 30 mL/min, and 2 mL of solution was injected at a time. The appropriate fractions were combined and concentrated under reduced pressure to give the fastest running enantiomer, (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (2.3 g), and the slowest running enantiomer, (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (2.4 g).</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 8-Chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) and 2-(Benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>7b</b>)</h3><div class="NLM_p last">The synthesis of these two compounds is described in the preceding article.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div></div><div id="sec4_12_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 8-(((3<i>R</i>,4<i>R</i>)-3-Methoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>9</b>)</h3><div class="NLM_p last">Step 1: A solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (0.54 g, 2.23 mmol) in DMF (10 mL) at 0 Â°C under nitrogen was treated with NaH (60% w/w in mineral oil, 0.116 g, 2.90 mmol), and the resulting mixture was stirred at this temperature for 20 min. Iodomethane (0.153 mL, 2.45 mmol) was then added, and the solution was stirred for 16 h at room temperature. The mixture was then diluted with water (20 mL), and the aqueous phase was extracted with EtOAc (20 mL). The organic phase was washed with water (20 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-methoxypiperidine-1-carboxylate (0.48 g, 84%) as a colorless oil. Step 2: (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-methoxypiperidine-1-carboxylate (0.45 g, 1.76 mmol) was dissolved in MeOH (40 mL) and hydrogenated in the H-Cube (1 mL/min, 10% Pd/C cartridge, full H<sub>2</sub> mode), and then MeOH was concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-methoxypiperidine-1-carboxylate (<b>6</b>, R<sub>1</sub>âMe) (0.38 g, 94%) as a colorless oil. Step 3: A suspension of 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) (0.08 g, 0.41 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-methoxypiperidine-1-carboxylate (<b>6</b>, R<sub>1</sub>âMe) (0.156 mL, 0.617 mmol), and Caddick catalyst (0.024 g, 0.041 mmol) in THF (2 mL) in a microwave vial at room temperature under nitrogen was treated with LiHMDS (1N in THF, 1.23 mL, 1.23 mmol), and the resulting mixture was stirred at 120 Â°C for 30 min under microwave irradiation and then cooled to room temperature. The mixture was diluted with a saturated NH<sub>4</sub>Cl aqueous solution (10 mL), and the aqueous phase was extracted with EtOAc (2 Ã 10 mL). The combined organic phases were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-methoxy-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (63 mg, 39%) as a beige solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.99 min, [M + H]<sup>+</sup> = 389.3. Step 4: A solution of (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-methoxy-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (76 mg, 0.20 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL), and the resulting mixture was stirred at this temperature for 1 h and then concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 8-(((3<i>S</i>,4<i>S</i>)-3-methoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1H)-one (48 mg, 85%) as a pale yellow solid. The racemic mixture was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralpak AD-H (4.6 mm i.d. Ã 25 cm), using 40% EtOH (+0.2% isopropylamine) in heptane at a flow rate of 1 mL/min. The material (34 mg) was dissolved in a mixture of EtOH/heptane (2:1). The purification was carried out on a Chiralpak AD-H (30 mm Ã 25 cm), using 40% EtOH (+ 0.2% isopropylamine) in heptane at a flow rate of 30 mL/min, and 3 mL of solution was injected. The appropriate fractions were combined and concentrated under reduced pressure to give the fastest running enatiomer, 8-(((3<i>R</i>,4<i>R</i>)-3-methoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>9</b>) (12 mg, 0.042 mmol, 21%), as a white solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.62 min, [M + H]<sup>+</sup> = 289.2. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) Î´ ppm 7.78 (d, <i>J</i> = 5.4 Hz, 1H), 7.72 (d, <i>J</i> = 1.0 Hz, 1H), 6.79 (d, <i>J</i> = 5.6 Hz, 1H), 4.12â4.20 (m, 1H), 3.45 (s, 3H), 3.37 (m, 4H), 2.99â3.06 (m, 1H), 2.69â2.78 (m, 1H), 2.51â2.58 (m, 1H), 2.24 (d, <i>J</i> = 1.2 Hz, 3H). Three NH not seen.</div></div><div id="sec4_12_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 8-(((3<i>R</i>,4<i>R</i>)-3-Isopropoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>10</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (0.54 g, 2.2 mmol) in DMF (10 mL) at 0 Â°C under nitrogen was treated with sodium hydride (60% w/w in mineral oil, 0.116 g, 2.90 mmol), and the resulting mixture was stirred for 20 min at this temperature. 2-Iodopropane (0.245 mL, 2.45 mmol) was then added, and the solution was stirred at room temperature for 16 h. The mixture was then stirred at 70 Â°C for 2 h before being cooled to room temperature and treated with water (30 mL). The aqueous phase was extracted with EtOAc (30 mL). The organic phase was then washed with water (30 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-isopropoxypiperidine-1-carboxylate (55 mg, 9%) as a colorless oil. Step 2: (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-isopropoxypiperidine-1-carboxylate (50 mg, 0.18 mmol) was dissolved in MeOH (20 mL) and hydrogenated in the H-Cube (1 mL/min flow rate, 10% Pd/C cartridge, full H<sub>2</sub> mode). The solution was then concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-isopropoxypiperidine-1-carboxylate (<b>6</b>, R<sub>1</sub> = <i>i</i>-Pr) (42 mg, 92%) as a colorless oil. Step 3: a microwave vial was charged with 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) (0.035 g, 0.18 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-isopropoxypiperidine-1-carboxylate (<b>6</b>, R<sub>1</sub> = <i>i</i>-Pr) (0.053 mL, 0.19 mmol), and Caddick catalyst (10.6 mg, 0.018 mmol) and filled with THF (2 mL), and the resulting suspension was treated at room temperature under nitrogen with LiHMDS (1N in THF, 0.54 mL, 0.54 mmol). The resulting mixture was stirred at 120 Â°C for 30 min under microwave irradiation and was then cooled to room temperature. The mixture was then diluted with a saturated NH<sub>4</sub>Cl aqueous solution (10 mL), and the aqueous phase was extracted with EtOAc (2 Ã 10 mL). The combined organic phases were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo.</i> Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave 8-(((3<i>R</i>,4<i>R</i>)-3-isopropoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (12 mg, 21%) (<b>10</b>) as a beige solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.75 min, [M + H]<sup>+</sup> = 317.2. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) Î´ ppm 7.86 (d, <i>J</i> = 5.6 Hz, 1H), 7.75 (d, <i>J</i> = 1.0 Hz, 1H), 6.88 (d, <i>J</i> = 5.4 Hz, 1H), 4.39 (d, <i>J</i> = 4.4 Hz, 1H), 4.05â4.10 (m, 1H), 3.92â4.01 (m, 1H), 3.41â3.51 (m, 2H), 3.21â3.28 (m, 1H), 3.17 (dd, <i>J</i> = 13.1, 4.5 Hz, 1H), 2.41â2.51 (m, 1H), 2.25 (d, <i>J</i> = 0.7 Hz, 3H), 1.97â2.05 (m, 1H), 1.24 (d, <i>J</i> = 6.1 Hz, 6H). Three NH not seen.</div></div><div id="sec4_12_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 3-Methyl-8-(((3<i>R</i>,4<i>R</i>)-3-propoxypiperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>11</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (1.94 g, 8.00 mmol) in DMF (30 mL) at room temperature under nitrogen was treated with sodium hydride (60% w/w in mineral oil, 0.384 g, 9.60 mmol) and then, after 10 min, with allyl bromide (1.04 mL, 12.0 mmol). The solution was stirred at room temperature for 1 h before being quenched with MeOH (1 mL) and diluted with water. The aqueous phase was extracted four times with Et<sub>2</sub>O, and the combined organics were washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(allyloxy)-4-azidopiperidine-1-carboxylate (2.32 g, 103%) as a clear oil, which was used in the next step without further purification. Step 2: a solution of (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(allyloxy)-4-azidopiperidine-1-carboxylate (2.32 g, 8.22 mmol) in THF (40 mL) at room temperature was treated with triphenylphosphine (3.23 g, 12.3 mmol), and the resulting mixture was stirred at this temperature for 16 h. Water (3 mL) was then added, and the solution was refluxed for 2.5 h and then cooled to room temperature and concentrated <i>in vacuo</i>. The residue was dissolved in EtOAc, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give a white solid. This residue was loaded on to a 50 g SCX column, washed with MeOH, and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give the desired amine still contaminated with triphenylphosphine oxide. Purification of this residue by flash chromatography on silica gel (50 g column, gradient: 10 to 40% (20% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(allyloxy)-4-aminopiperidine-1-carboxylate (1.45 g, 69%) as a colorless oil. Step 3: (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(allyloxy)-4-aminopiperidine-1-carboxylate (300 mg, 1.17 mmol) was dissolved in MeOH (40 mL) and hydrogenated in the H-Cube on full hydrogen mode with 10% Pd/C cartridge at 1 mL/min flow rate. The eluent was evaporated <i>in vacuo</i> to give a colorless oil. The product was redissolved in MeOH (40 mL), and hydrogenation was repeated under the same conditions to give (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-amino-3-propoxypiperidine-1-carboxylate (<b>6</b>, R<sub>1</sub> = Pr) (285 mg, 94%). Step 4: a microwave vial was charged with 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) (0.043 g, 0.22 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-propoxypiperidine-1-carboxylate (66 mg, 0.25 mmol), and Caddick catalyst (0.020 g, 0.033 mmol) and was then filled with THF (2 mL). The resulting mixture was treated with LiHMDS (1 N in THF, 0.77 mL, 0.77 mmol) and was then stirred at 80 Â°C for 30 min under nitrogen and microwave irradiation before being cooled to room temperature and diluted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The organic phase was washed with a saturated NH<sub>4</sub>Cl aqueous solution, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-propoxypiperidine-1-carboxylate (73 mg, 79%) as a brown solid. This product was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) and treated at room temperature with TFA (1 mL). The mixture was stirred at this temperature for 1 h and was then concentrated <i>in vacuo</i>. The residue obtained was loaded on to a 5 g SCX-2 cartridge, which was washed with MeOH (10 mL) and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 3-methyl-8-(((3<i>R</i>,4<i>R</i>)-3-propoxypiperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>11</b>) (51 mg, 73%) as a pale yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.72 min, [M + H]<sup>+</sup> = 317.2. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.95 (d, <i>J</i> = 5.4 Hz, 1H), 7.65 (d, <i>J</i> = 1.2 Hz, 1H), 6.69 (d, <i>J</i> = 5.6 Hz, 1H), 6.50 (d, <i>J</i> = 7.3 Hz, 1H), 4.40â4.52 (m, 1H), 3.59 (dt, <i>J</i> = 9.2, 6.5 Hz, 1H), 3.45â3.54 (m, 2H), 3.34â3.42 (m, 1H), 3.09â3.18 (m, 1H), 2.83 (ddd, <i>J</i> = 12.7, 10.1, 2.9 Hz, 1H), 2.67â2.75 (m, 1H), 2.37 (d, <i>J</i> = 0.7 Hz, 3H), 2.21â2.30 (m, 1H), 1.60â1.72 (m, 1H), 1.43â1.54 (m, 2H), 0.74 (t, <i>J</i> = 7.5 Hz, 3H). Two NH not seen.</div></div><div id="sec4_12_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 8-(((3<i>R</i>,4<i>R</i>)-3-Isobutoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>12</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (0.70 g, 2.89 mmol) in DMF (10 mL) at 0 Â°C was treated with NaH (60% w/w in mineral oil, 0.139 g, 3.47 mmol), and the resulting mixture was stirred for 20 min at 0 Â°C before 1-bromo-2-methylpropane (0.396 g, 2.89 mmol) was added. The mixture was stirred at this temperature for 2 h and was then diluted with water (40 mL). The aqueous phase was extracted with EtOAc (40 mL), and the organic layer was washed with water (40 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-isobutoxypiperidine-1-carboxylate (90 mg, 10%). Step 2: a suspension of 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) (0.050 g, 0.26 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-isobutoxypiperidine-1-carboxylate (0.078 g, 0.29 mmol), and Caddick catalyst (0.017 g, 0.029 mmol) in THF (2 mL) at room temperature was treated with LiHMDS (1N in THF, 0.86 mL, 0.86 mmol), and the resulting mixture was stirred at 120 Â°C under microwave irradiation for 1 h, cooled to 0 Â°C, and quenched with a saturated NH<sub>4</sub>Cl aqueous solution (10 mL). The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL). The organic layer was dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-isobutoxy-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (51 mg, 41%) as a pale yellow solid. Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-isobutoxy-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (50 mg, 0.12 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL), and the resulting mixture was stirred at this temperature for 1 h and concentrated <i>in vacuo</i>. The residue was loaded on to a 5 g SCX-2 cartridge, washed with MeOH (20 mL), and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-isobutoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>12</b>) (31 mg, 81%) as a pale yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.80 min, [M + H]<sup>+</sup> = 331.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.72 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (d, <i>J</i> = 1.2 Hz, 1H), 6.68 (d, <i>J</i> = 5.4 Hz, 1H), 6.62 (d, <i>J</i> = 7.3 Hz, 1H), 4.13 (m, 1H), 3.35(br s, 1H), 3.30 (dd, <i>J</i> = 8.9, 6.5 Hz, 2H), 3.13â3.23 (m, 3H), 2.81â2.90 (m, 1H), 2.44â2.50 (m, 1H), 2.25â2.35 (m, 1H), 2.11 (d, <i>J</i> = 0.7 Hz, 3H), 1.97 (dd, <i>J</i> = 13.0, 3.7 Hz, 1H), 1.62 (dt, <i>J</i> = 13.2, 6.6 Hz, 1H), 1.22â1.35 (m, 1H), 0.69 (dd, <i>J</i> = 8.6, 6.8 Hz, 6H).</div></div><div id="sec4_12_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 8-(((3<i>R</i>,4<i>R</i>)-3-(Cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>13</b>)</h3><div class="NLM_p last">Step 1: a solution of cyclohexylmethanol (10 g, 88 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (150 mL) at 0 Â°C under nitrogen was treated with pyridine (7.8 mL, 96 mmol), and the resulting mixture was stirred for 5 min before being treated with the dropwise addition of Tf<sub>2</sub>O (16.3 mL, 96 mmol) over 10 min. The resulting mixture was stirred for 1 h at this temperature and was then warmed to room temperature. The organic phase was washed with water (2 Ã 50 mL) and then brine (50 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give cyclohexylmethyl trifluoromethanesulfonate (20 g, 93%) as a colorless oil, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.35 (d, <i>J</i> = 6.1 Hz, 2H), 1.76â1.89 (m, 5H), 1.69â1.76 (m, 1H), 1.17â1.38 (m, 3H), 0.99â1.12 (m, 2H). Step 2: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (15 g, 62 mmol) in DMF (50 mL) and THF (20 mL) was cooled to 0 Â°C and treated with sodium hydride (60% w/w in mineral oil, 3.2 g, 80 mmol) in small portions over 10 min. The resulting mixture was stirred at this temperature for 30 min. A solution of cyclohexylmethyl trifluoromethanesulfonate (20 g, 81 mmol) in THF (30 mL) was then added dropwise over 20 min, and the resulting mixture was allowed to slowly warm to room temperature over 2 h before being diluted with water (500 mL). The aqueous phase was extracted with Et<sub>2</sub>O (2 Ã 300 mL), and the combined organics were washed with water (2 Ã 400 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give a yellow oil. Purification of this residue by flash chromatography on silica gel (330 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cyclohexylmethoxy) piperidine-1-carboxylate (20.5 g, 98%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 3.97â4.34 (m, 1H), 3.84â3.94 (m, 1H), 3.36â3.38 (m, 2H), 3.32â3.39 (m, 1H), 3.10â3.20 (m, 1H), 2.84â2.96 (m, 1H), 2.60â2.83 (m, 1H), 1.88â1.97 (m, 1H), 1.75 (t, <i>J</i> = 13.1 Hz, 5H), 1.53â1.64 (m, 1H), 1.48 (s, 9H), 1.11â1.45 (m, 4H), 0.97 (d, <i>J</i> = 11.7 Hz, 2H). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cyclohexylmethoxy)piperidine-1-carboxylate (20.5 g, 60.6 mmol) in THF (200 mL) at room temperature was treated with triphenylphosphine (19.1 g, 72.7 mmol), and the resulting solution was allowed to stand at room temperature over 2 days; then, water (30 mL) was added, and the resulting mixture was refluxed for 3 h before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue obtained was partitioned between EtOAc (200 mL) and brine (200 mL), and the layers were separated. The organic phase was dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i> to give a colorless solid. This solid was triturated with Et<sub>2</sub>O (200 mL) for 30 min and then filtered off to remove triphenylphosphine oxide. The filtrate was concentrated <i>in vacuo</i>. Purification of the residue obtained by flash chromatography on silica gel (340 g column, gradient: 0 to 10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(cyclohexylmethoxy)piperidine-1-carboxylate (<b>6</b>, R<sub>1</sub> = cyclohexylmethyl) (14.9 g, 79%) as a pale yellow oil that crystallized on standing overnight. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.13â4.49 (m, 1H), 3.93â4.09 (m, 1H), 3.42â3.50 (m, 1H), 3.25 (dd, <i>J</i> = 9.0, 6.6 Hz, 1H), 2.80â2.90 (m, 1H), 2.73 (br s, 2H), 2.31â2.53 (m, 1H), 1.64â1.87 (m, 6H), 1.53â1.63 (m, 1H), 1.43â1.50 (m, 9H), 1.10â1.38 (m, 6H), 0.87â1.02 (m, 2H). Step 4: a flask was charged with 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>7b</b>) (530 mg, 1.86 mmol), sodium <i>tert</i>-butoxide (537 mg, 5.58 mmol), Brettphos palladacycle (74.3 mg, 0.093 mmol), Brettphos (50.0 mg, 0.093 mmol), and (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-amino-3-(cyclohexylmethoxy)piperidine-1-carboxylate (756 mg, 2.42 mmol) and then filled with THF (10 mL), and the resulting mixture stirred was for 1 h at room temperature. The color of the reaction changed from deep green to orange during this time. The mixture was then diluted with EtOAc (20 mL), and the organic phase was washed with water (20 mL) and then dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-(cyclohexylmethoxy) piperidine-1-carboxylate (1.02 g, 98%) as a pale yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.73 min, [M + H]<sup>+</sup><sub>=</sub> 561. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.91 (d, <i>J</i> = 5.6 Hz, 1H), 7.67 (d, <i>J</i> = 1.2 Hz, 1H), 7.50 (d, <i>J</i> = 7.1 Hz, 2H), 7.39â7.44 (m, 2H), 7.36 (d, <i>J</i> = 7.3 Hz, 1H), 6.74 (d, <i>J</i> = 5.6 Hz, 1H), 6.37 (d, <i>J</i> = 7.3 Hz, 1H), 5.47â5.56 (m, 2H), 4.18â4.32 (m, 1H), 3.85 (s, 1H), 3.40â3.49 (m, 1H), 3.29â3.37 (m, 1H), 3.22 (dd, <i>J</i> = 8.9, 6.5 Hz, 2H), 2.42â2.51 (m, 1H), 2.40 (d, <i>J</i> = 0.7 Hz, 3H), 1.47â1.66 (m, 18H), 1.05 (d, <i>J</i> = 8.3 Hz, 3H), 0.68â0.86 (m, 2H). Step 5: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-(cyclohexylmethoxy) piperidine-1-carboxylate (1.02 g, 1.82 mmol) in MeOH (100 mL) was hydrogenated using a H-Cube apparatus, on full hydrogen mode, running at 1 mL/min with a 10% Pd/C cartridge. The eluent was evaporated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (0.85 g, 99%) as a pale yellow solid which was used in the next step without further purification. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.47 min, [M + H]<sup>+</sup> = 471. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.06â11.22 (m, 1H), 7.76 (d, <i>J</i> = 5.3 Hz, 1H), 7.65 (d, <i>J</i> = 1.0 Hz, 1H), 6.73 (d, <i>J</i> = 5.3 Hz, 1H), 6.46 (d, <i>J</i> = 7.0 Hz, 1H), 4.29â4.38 (m, 1H), 3.76â3.84 (m, 1H), 3.60â3.69 (m, 1H), 3.39 (dd, <i>J</i> = 9.3, 6.5 Hz, 1H), 3.14â3.36 (m, 4H), 2.15 (d, <i>J</i> = 1.0 Hz, 3H), 1.97â2.08 (m, 1H), 1.59 (d, <i>J</i> = 7.5 Hz, 5H), 1.39â1.53 (m, 11H), 1.02â1.22 (m, 3H), 0.89 (br s, 2H). Step 6: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (50 mg, 0.11 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL), and the resulting mixture was stirred at this temperature for 1 h and then concentrated <i>in vacuo</i>. The residue was dissolved in MeOH (5 mL) and loaded onto a 5g SCX-2 cartridge. The cartridge was washed with MeOH (20 mL) and was then eluted with a 2 N NH<sub>3</sub> solution in MeOH (10 mL). The ammoniac fractions were concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-(cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>13</b>) (33 mg, 84%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.95 min, [M + H]<sup>+</sup> = 371.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 7.96 (d, <i>J</i> = 5.4 Hz, 1H), 7.65 (d, <i>J</i> = 1.0 Hz, 1H), 6.69 (d, <i>J</i> = 5.6 Hz, 1H), 6.45 (d, <i>J</i> = 6.4 Hz, 1H), 4.46 (d, <i>J</i> = 5.1 Hz, 1H), 3.34â3.49 (m, 3H), 3.29 (dd, <i>J</i> = 9.2, 6.5 Hz, 1H), 3.09â3.17 (m, 1H), 2.78â2.88 (m, 1H), 2.70 (dd, <i>J</i> = 12.1, 8.4 Hz, 1H), 2.38 (d, <i>J</i> = 0.7 Hz, 3H), 2.20â2.28 (m, 1H), 1.62â1.77 (m, 2H), 1.39â1.61 (m, 5H), 0.88â1.13 (m, 3H), 0.65â0.83 (m, 2H). One NH not seen.</div></div><div id="sec4_12_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 8-(((3<i>R</i>,4<i>R</i>)-3-(Cyclopentylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>14</b>)</h3><div class="NLM_p last">Step 1: a solution of cyclopentylmethanol (1.55 g, 15.5 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (30 mL) was cooled in an ice bath and was then treated with pyridine (1.38 mL, 17.0 mmol) followed by trifluoromethanesulfonic anhydride (2.88 mL, 17.0 mmol). The resulting mixture was stirred at this temperature for 2 h and was then washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give cyclopentylmethyl trifluoromethanesulfonate (3.02 g, 84%) as a colorless oil, which was used in the next step without further purification. Step 2: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (1.8 g, 7.4 mmol) in DMF (5 mL) was cooled in an ice bath and was then treated with NaH (60% w/w in mineral oil, 0.232 g, 9.66 mmol), and the resulting mixture was stirred at this temperature for 20 min before being treated with cyclopentylmethyl trifluoromethanesulfonate (2 g, 8.61 mmol). After stirring at 0 Â°C for 2 h, the reaction mixture was allowed to warm to room temperature. The reaction mixture was then diluted with water (15 mL), and the aqueous phase was extracted with EtOAc (20 mL). The organic layer was washed with water (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cyclopentylmethoxy)piperidine-1-carboxylate (1.01 g, 42%) as a colorless oil. Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cyclopentylmethoxy)piperidine-1-carboxylate (1.01 g, 3.11 mmol) in MeOH (80 mL) was hydrogenated in the H-Cube on full hydrogen mode over a 10% Pd/C cartridge at 1 mL/min flow rate. The eluent was concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(cyclopentylmethoxy) piperidine-1-carboxylate (<b>6</b>, R<sub>1</sub> = cyclopentylmethyl) (0.92 g, 99% yield) as a colorless oil. Step 4: a microwave vial was charged with 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) (65 mg, 0.33 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl4-amino-3-(cyclopentylmethoxy) piperidine-1-carboxylate (130 mg, 0.43 mmol), and Caddick catalyst (15 mg, 0.33 mmol) and flushed with nitrogen for 10 min. THF (2 mL) followed by LiHMDS (1N in THF, 1.34 mL, 1.34 mmol) was added, and the reaction mixture was stirred at 80 Â°C for 30 min under microwave irradiation and then cooled to room temperature. The reaction mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and washed with a saturated NH<sub>4</sub>Cl aqueous solution (10 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclopentylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (145 mg, 95%) as a yellow glass. A solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclopentylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (145 mg, 0.32 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL), and the resulting mixture was stirred at room temperature for 1 h and then concentrated <i>in vacuo</i>. The residue was loaded on to a 5 g SCX-2 cartridge, washed with MeOH, and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were combined and concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-(cyclopentylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>14</b>) (110 mg, 92%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.89 min, [M + H]<sup>+</sup> = 357. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.96 (d, <i>J</i> = 5.4 Hz, 1H), 7.66 (s, 1H), 6.69 (d, <i>J</i> = 5.4 Hz, 1H), 6.48 (d, <i>J</i> = 7.1 Hz, 1H), 4.51â4.40 (m, 1H), 3.54â3.43 (m, 2H), 3.42â3.32 (m, 2H), 3.17â3.07 (m, 1H), 2.87â2.77 (m, 1H), 2.74â2.65 (m, 1H), 2.38 (s, 3H), 2.30â2.21 (m, 1H), 2.11â1.97 (m, 1H), 1.73â1.61 (m, 1H), 1.59â1.43 (m, 2H), 1.43â1.30 (m, 4H), 1.13â1.01 (m, 2H). Two NH not seen.</div></div><div id="sec4_12_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 8-(((3<i>R</i>,4<i>R</i>)-3-(Cycloheptylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>15</b>)</h3><div class="NLM_p last">Step 1: a solution of cycloheptane carboxylic acid (3 g, 21 mmol) in THF (50 mL) was cooled to 0 Â°C using an ice bath and was then treated under nitrogen with LAH (1 N in THF, 36 mL, 36 mmol) dropwise over 10 min. The resulting mixture was stirred at this temperature for 1 h and was then stirred at room temperature for 16 h. The reaction mixture was quenched by the dropwise addition of water (1.4 mL) over 10 min, followed by a 15% w/w NaOH aqueous solution (1.4 mL). After stirring for 20 min, the mixture was filtered, and the filtrate was concentrated <i>in vacuo</i> to give cycloheptylmethanol (2.55 g, 94%) as a colorless oil. Step 2: cycloheptylmethyl trifluoromethanesulfonate was prepared from cycloheptylmethanol using the same procedure as that for the synthesis of cyclopentylmethyl trifluoromethanesulfonate from cyclopentylmethanol (preparation of <b>14</b>, step 1) (5.2 g, 95%, colorless oil). Step 3: (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cycloheptylmethoxy) piperidine-1-carboxylate was prepared from cycloheptylmethyl trifluoromethanesulfonate and (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) using the same procedure as that for the synthesis of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cyclopentylmethoxy)piperidine-1-carboxylate from cyclopentylmethyl trifluoromethanesulfonate (preparation of <b>14</b>, step 2) (3.4 g, 78%, colorless oil). Step 4: (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(cycloheptylmethoxy)piperidine-1-carboxylate was prepared from (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cycloheptylmethoxy)piperidine-1-carboxylate using the same procedure as that for the synthesis of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(cyclopentylmethoxy)piperidine-1-carboxylate from (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(cyclopentylmethoxy)piperidine-1-carboxylate (preparation of <b>14</b>, step 3). Purification of the crude amine by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(cycloheptylmethoxy) piperidine-1-carboxylate (1.35 g, 97%) as a colorless oil. Step 5: 8-(((3<i>R</i>,4<i>R</i>)-3-(cycloheptylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one was prepared from (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(cycloheptylmethoxy) piperidine-1-carboxylate and 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) using the same procedure as that for the synthesis of 8-(((3<i>R</i>,4<i>R</i>)-3-(cyclopentylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (preparation of <b>14</b>, step 4). Purification of the crude amine by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-(cycloheptyl methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (18 mg, 13%) as a yellow oil. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.01 min, [M + H]<sup>+</sup> = 385. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.72 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (s, 1H), 6.67 (d, <i>J</i> = 5.4 Hz, 1H), 6.59 (d, <i>J</i> = 7.6 Hz, 1H), 4.19â4.08 (m, 1H), 3.22â3.12 (m, 3H), 2.90â2.81 (m, 1H), 2.49â2.43 (m, 1H), 2.34â2.24 (m, 1H), 2.11 (s, 3H), 1.99â1.90 (m, 1H), 1.57â1.11 (m, 13H), 1.01â0.87 (m, 2H). Two NH not seen.</div></div><div id="sec4_12_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 8-(((3<i>S</i>,4<i>S</i>)-3-(Cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>16</b>) and 8-(((3<i>R</i>,4<i>R</i>)-3-(Cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>17</b>)</h3><div class="NLM_p last">Step 1: the racemic mixture of the Boc-protected compound <b>13</b> (<b>8</b>, R<sub>1</sub> = methylcyclohexyl, R<sub>2</sub> = H) was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralpak IC (4.6 mm i.d. Ã 25 cm), using 20% EtOH in heptane at a flow rate of 1 mL/min. The material (100 mg) was dissolved in a mixture of EtOH/heptane (1:1) (4 mL). The purification was carried out on a Chiralpak IC (2 cm Ã 25 cm), using 20% EtOH in heptane at a flow rate of 15 mL/min, and 1 mL of solution was injected at a time. The appropriate fractions were combined and concentrated under reduced pressure to give the fastest running enantiomer, (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (28 mg, 0.059 mmol), and the slowest running enantiomer, (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (20 mg, 0.042 mmol). Step 2: (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (28 mg, 0.059 mmol) and (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (20 mg, 0.042 mmol) were each dissolved in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature and then were treated with TFA (1 mL). The mixtures were stirred at this temperature for 30 min and were then concentrated <i>in vacuo</i>. The residue were dissolved in MeOH (3 mL), loaded onto a 5 g SCX-2 cartridge, washed with MeOH (10 mL), and finally eluted with a 2 N NH<sub>3</sub> solution in MeOH (10 mL). The ammoniac fractions were in each case concentrated <i>in vacuo</i> to give 8-(((3<i>S</i>,4<i>S</i>)-3-(cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>16</b>) (21 mg, 95%) as a pale yellow solid and 8-(((3<i>R</i>,4<i>R</i>)-3-(cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>17</b>) (15 mg, 68) as pale yellow solid.</div></div><div id="sec4_12_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 8-(((3<i>R</i>,4<i>R</i>)-3-(Cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>20</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (208 mg, 0.442 mmol) in CHCl<sub>3</sub> (20 mL) at â10 Â°C was treated with NBS (79 mg, 0.44 mmol), and the resulting solution was stirred at this temperature for 1 h, treated with a saturated sodium metabisulfate aqueous solution, and stirred for another 10 min. The layers were separated, and the organic phase was dried using a phase separator and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((5-bromo-3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(cyclohexylmethoxy)piperidine-1-carboxylate (232 mg, 96%) as a pale yellow solid, which was used in the next step without further purification. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.56 min, [M + H]<sup>+</sup> = 549 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 8.08 (s, 1H), 8.03 (d, <i>J</i> = 1.0 Hz, 1H), 6.50â6.63 (m, 1H), 4.38â4.53 (m, 1H), 3.97â4.09 (m, 1H), 3.40â3.50 (m, 2H), 3.29 (d, <i>J</i> = 6.6 Hz, 2H), 3.02â3.18 (m, 2H), 2.36 (s, 3H), 2.15â2.26 (m, 2H), 1.48â1.64 (m, 15H), 1.07 (d, <i>J</i> = 12.6 Hz, 3H), 0.76 (br s, 2H). Step 2: a microwave vial was charged with (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((5-bromo-3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(cyclohexylmethoxy)piperidine-1-carboxylate (100 mg, 0.182 mmol), Pd(OAc)<sub>2</sub> (4.09 mg, 0.018 mmol), cataCXium A (6.53 mg, 0.018 mmol), 5-methylpyridine-5-boronic acid (49.8 mg, 0.364 mmol), and K<sub>2</sub>CO<sub>3</sub> (75 mg, 0.55 mmol) and then filled with 1,4-dioxane (3 mL) and water (0.5 mL). The resulting mixture was stirred at 100 Â°C for 30 min under nitrogen and microwave irradiation and was then cooled to room temperature. The mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL), and the organic phase was washed with water, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave a residue that was further purified by flash chromatography on silica gel (10 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH)) to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-5-(5-methylpyridin-3-yl)-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (<b>19</b>, R<sub>1</sub> = H, X = CH<sub>2</sub>, Y = C, R<sub>2</sub> = Me) (65 mg, 64%) as a yellow gum. LCMS (method formic): <i>t</i><sub>R</sub> 1.11 min, [M + H]<sup>+</sup> = 562. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 8.52 (d, <i>J</i> = 1.5 Hz, 1H), 8.47 (d, <i>J</i> = 1.7 Hz, 1H), 7.91 (s, 1H), 7.71 (d, <i>J</i> = 1.0 Hz, 1H), 7.53 (s, 1H), 6.66â6.77 (m, 1H), 4.50â4.65 (m, 1H), 4.22â4.46 (m, 1H), 3.98â4.08 (m, 1H), 3.41â3.54 (m, 2H), 3.23â3.34 (m, 1H), 2.75â3.19 (m, 2H), 2.47 (s, 2H), 2.28 (d, <i>J</i> = 0.7 Hz, 2H), 2.20â2.26 (m, 1H), 1.99â2.05 (m, 3H), 1.77 (br s, 3H), 1.61â1.73 (m, 3H), 1.39â1.60 (m, 14H). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-5-(5-methylpyridin-3-yl)-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (65 mg, 0.12 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL), and the resulting mixture was stirred for 1 h at this temperature and then concentrated <i>in vacuo</i>. The residue was dissolved in MeOH (10 mL) and loaded onto a 5 g SCX-2 cartridge, which was washed with MeOH (10 mL) and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were combined and concentrated <i>in vacuo</i>. Purification of this residue by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-(cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>20</b>) (14 mg, 26%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.98 min, [M + H]<sup>+</sup> = 462. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.5 (br s, 1H), 8.52 (d, <i>J</i> = 1.5 Hz, 1H), 8.48 (d, <i>J</i> = 1.7 Hz, 1H), 7.91 (s, 1H), 7.71 (d, <i>J</i> = 1.0 Hz, 1H), 7.53 (s, 1H), 6.65â6.74 (m, 1H), 4.46â4.56 (m, 1H), 3.36â3.50 (m, 3H), 3.31 (dd, <i>J</i> = 9.3, 6.6 Hz, 1H), 3.14 (br s, 1H), 2.85 (br s, 1H), 2.71 (dd, <i>J</i> = 12.2, 8.6 Hz, 1H), 2.47 (s, 3H), 2.34 (d, <i>J</i> = 0.7 Hz, 3H), 2.23â2.31 (m, 1H), 1.71 (d, <i>J</i> = 9.8 Hz, 1H), 1.43â1.64 (m, 6H), 0.94â1.19 (m, 3H), 0.68â0.85 (m, 2H). One NH not seen.</div></div><div id="sec4_12_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 8-(((3<i>S</i>,4<i>S</i>)-3-(Cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>21</b>)</h3><div class="NLM_p last">This compound was obtained in an analogous manner as that for compound <b>20</b>, starting from (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 3-(cyclohexylmethoxy)-4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate.</div></div><div id="sec4_12_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 5-(5-Aminopyridin-3-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-(cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>22</b>)</h3><div class="NLM_p last">A 20 mL microwave vial was charged with (5-aminopyridin-2-yl)boronic acid (40.2 mg, 0.291 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((5-bromo-3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(cyclohexylmethoxy)piperidine-1-carboxylate (<b>18</b>, R<sub>1</sub> = H, X = CH<sub>2</sub>) (80 mg, 0.15 mmol), K<sub>2</sub>CO<sub>3</sub> (80 mg, 0.582 mmol), Pd(OAc)<sub>2</sub> (3.3 mg, 0.015 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (5.22 mg, 0.015 mmol) and was then filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was degassed for 20 min with nitrogen and was then stirred at 110 Â°C for 30 min under microwave irradiation before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using an hydrophobic frit and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((5-(5-aminopyridin-3-yl)-3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(cyclohexylmethoxy)piperidine-1-carboxylate (57 mg, 70%) as a yellow solid. This compound was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (3 mL), and the resulting solution was treated with TFA (1 mL). The mixture was stirred at room temperature for 2 h and then concentrated <i>in vacuo</i>. The residue was dissolved in MeOH (5 mL) and loaded onto a 5 g SCX-2 cartridge. This cartridge was washed with MeOH (10 mL) and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 5-(5-aminopyridin-3-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-(cyclohexylmethoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>21</b>) (33 mg, 71%) as a yellow solid. LCMS (method TFA): <i>t</i><sub>R</sub> 0.57 min, [M + H]<sup>+</sup> = 463.4. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) Î´ ppm 8.02 (d, <i>J</i> = 2.7 Hz, 1H), 7.79 (d, <i>J</i> = 2.0 Hz, 1H), 7.68â7.74 (m, 2H), 7.12 (br s, 1H), 4.27â4.37 (m, 1H), 3.43â3.50 (m, 1H), 3.35â3.42 (m, 2H), 3.22â3.29 (m, 1H), 3.03â3.12 (m, 1H), 2.71â2.81 (m, 1H), 2.50â2.63 (m, 1H), 2.21 (d, <i>J</i> = 1.0 Hz, 4H), 1.54 (br s, 6H), 1.34â1.42 (m, 1H), 0.95â1.19 (m, 4H), 0.68â0.85 (m, 2H). Four NH not seen.</div></div><div id="sec4_12_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1-Acetylpiperidin-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>25</b>)</h3><div class="NLM_p last">Step 1: a solution of piperidin-4-ylmethanol (5.00 g, 43.4 mmol) and triethylamine (9.00 mL, 64.6 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (100 mL) was cooled in an ice bath and was then treated with acetic anhydride (4.10 mL, 43.4 mmol), and the resulting mixture was allowed to warm to room temperature and stirred under nitrogen for 16 h. The solution was then washed with brine (100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give 1-(4-(hydroxymethyl)piperidin-1-yl)ethanone (4.56 g, 67%) as a pale yellow gum. Step 2: a solution of 1-(4-(hydroxymethyl)piperidin-1-yl)ethanone (2.00 g, 12.7 mmol) and pyridine (1.24 mL, 15.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL) under nitrogen was cooled in an ice bath and then treated with trifluoromethanesulfonic anhydride (2.36 mL, 14.0 mmol) dropwise over 10 min. The resulting mixture was stirred at this temperature for 30 min and then washed with water (50 mL). The organic layer was then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i> to give (1-acetylpiperidin-4-yl)methyl trifluoromethanesulfonate (1.79 g, 49%) as a brown gum, which was used in the next step without further purification. Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (1.5 g, 6.2 mmol) in DMF (5 mL) was cooled in an ice bath and then treated with NaH (60% w/w in mineral oil, 0.19 g, 8.05 mmol), and the resulting mixture stirred at this temperature for 20 min before (1-acetylpiperidin-4-yl)methyl trifluoromethanesulfonate (1.79 g, 6.19 mmol) was added as a solution in DMF (5 mL). The solution was stirred at 0 Â°C for 2 h and was then allowed to warm to room temperature and diluted with water (15 mL). The aqueous phase was extracted with EtOAc (20 mL). The organic layer was washed with water (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 100% EtOAc in cyclohexane followed by 5 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-((1-acetylpiperidin-4-yl)methoxy)-4-azidopiperidine-1-carboxylate (0.43 g, 18% yield) as a yellow gum. This azide was dissolved in MeOH (20 mL) and hydrogenated in the H-Cube on full hydrogen mode, using a 10% Pd/C cartridge, at 1 mL/min flow rate; then, the eluent was concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl3-((1-acetylpiperidin-4-yl)methoxy)-4-aminopiperidine-1-carboxylate (0.34 g, 15%). Step 4: (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-((1-acetylpiperidin-4-yl)methoxy)-4-aminopiperidine-1-carboxylate (225 mg, 0.632 mmol), sodium <i>tert</i>-butoxide (152 mg, 1.580 mmol), and BrettPhos precatalyst (21 mg, 0.026 mmol) were added to a solution of 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>7b</b>) (150 mg, 0.527 mmol) in THF (3 mL) at 0 Â°C under nitrogen. After stirring at room temperature for 3 h, the mixture was concentrated <i>in vacuo</i>, and the residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL). The layers were separated, and the organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-((1-acetylpiperidin-4-yl)methoxy)-4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (182 mg, 57%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.38 min, [M + H]<sup>+</sup> = 604.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.86 (s, 1H), 7.82 (d, <i>J</i> = 5.5 Hz, 1H), 7.52 (d, <i>J</i> = 7.6 Hz, 2H),7.26â7.43 (m, 3H), 6.80 (d, <i>J</i> = 5.5 Hz, 1H), 6.50 (d, <i>J</i> = 7.6 Hz, 1H), 5.62 (d, <i>J</i> = 2.8 Hz, 2H), 4.20â4.33 (m, 1H), 3.78â3.96 (m, 3H), 3.67â3.76 (m, 1H), 3.42â3.54 (m, 2H), 3.33 (dd, <i>J</i> = 9.3, 6.3 Hz, 1H), 3.14â3.26 (m, 2H), 3.08 (dd, <i>J</i> = 13.3, 8.1 Hz, 1H), 2.82 (br s, 2H), 2.65 (br s, 1H), 2.51 (d, <i>J</i> = 1.5 Hz, 1H), 2.37 (s, 3H), 2.13 (d, <i>J</i> = 4.0 Hz, 1H), 1.58â1.71 (m, 2H), 1.41â1.56 (m, 11H), 0.89â1.04 (m, 2H). Step 5: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-((1-acetylpiperidin-4-yl)methoxy)-4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (180 mg, 0.298 mmol) in TFA (2 mL) was heated at 50 Â°C for 2 h and then cooled to room temperature and concentrated <i>in vacuo.</i> Purification of the residue by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-((1-acetylpiperidin-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (85 mg, 69%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.63 min, [M + H]<sup>+</sup> = 414. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.74 (d, <i>J</i> = 5.3 Hz, 1H), 7.60 (s, 1H), 6.67 (d, <i>J</i> = 5.3 Hz, 1H), 6.39 (d, <i>J</i> = 6.5 Hz, 1H), 4.23 (br s, 1H), 3.88 (br s, 2H), 3.46â3.40 (m, 1H), 3.37â3.18 (m, 3H), 2.98â2.89 (m, 1H), 2.82 (br s, 1H), 2.70 (br s, 2H), 2.62â2.53 (m, 1H), 2.47â2.39 (m, 1H), 2.16 (s, 3H), 2.05â1.97 (m, 1H), 1.90 (s, 3H), 1.71â1.59 (m, 1H), 1.58â1.47 (m, 2H), 1.48â1.38 (m, 1H), 1.05â0.92 (m, 2H). One NH not seen.</div></div><div id="sec4_12_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 8-(((3<i>R</i>,4<i>R</i>)-3-(((<i>trans</i>)-4-Hydroxycyclohexyl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>26</b>)</h3><div class="NLM_p last">Step 1: A solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (1 g, 4.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (30 mL) at room temperature was treated with imidazole (0.309 g, 4.54 mmol) and TBDMSCl (0.684 g, 4.54 mmol), and the resulting mixture was stirred at this temperature for 24 h. The organic phase was then washed with water (30 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (1.05 g, 71%) as a colorless oil. Step 2: (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (0.43 g, 1.2 mmol) was dissolved in EtOH (40 mL) and hydrogenated in the H-Cube (flow rate 1 mL/min, Pd/C cartridge, full H<sub>2</sub> mode). The solvent was then concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (<b>23</b>) (0.38 g, 95%) as a colorless oil. Step 3: A flask was charged with (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((<i>tert</i>-butyldimethylsilyl)oxy) piperidine-1-carboxylate (<b>23</b>) (6.04 g, 18.3 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>7b</b>) (4.0 g, 14 mmol), sodium <i>tert</i>-butoxide (4.05 g, 42.1 mmol), Brettphos (0.649 g, 1.21 mmol), and Brettphos palladacycle (0.4 g, 0.5 mmol) and then filled with THF (10 mL). The resulting mixture was stirred under nitrogen at room temperature for 2 h. Pd<sub>2</sub>(dba)<sub>3</sub> (0.643 g, 0.702 mmol) and Brettphos (0.649 g, 1.21 mmol) were further added, and the mixture was stirred at 50 Â°C for 2 h, cooled to room temperature, and diluted with a saturated NH<sub>4</sub>Cl aqueous solution (20 mL). The aqueous phase was extracted with EtOAc (40 mL). The organic phase was dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (8.05 g, 99%) as a yellow foam. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.79 min, [M + H]<sup>+</sup> = 579.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.90 (d, <i>J</i> = 5.9 Hz, 1H), 7.67 (d, <i>J</i> = 1.0 Hz, 1H), 7.50 (d, <i>J</i> = 7.1 Hz, 2H), 7.39â7.46 (m, 2H), 7.35 (s, 1H), 6.73 (d, <i>J</i> = 5.9 Hz, 1H), 6.16â6.26 (m, 1H), 5.51 (d, <i>J</i> = 16.9 Hz, 2H), 4.23â4.36 (m, 1H), 3.86â4.20 (m, 2H), 3.60â3.72 (m, 1H), 3.11â3.22 (m, 1H), 2.92â3.09 (m, 1H), 2.41 (d, <i>J</i> = 1.0 Hz, 3H), 2.21â2.31 (m, 1H), 1.45 (s, 9H), 0.77 (s, 9H), 0.10 (s, 3H), 0.07 (s, 3H). One NH not seen. Step 4: A solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (8.0 g, 14 mmol) in THF (50 mL) at room temperature was treated with TBAF (1N in THF, 20 mL, 20 mmol), and the resulting mixture was stirred at this temperature for 3 h and then concentrated <i>in vacuo.</i> The residue was partitioned between EtOAc (100 mL) and water (100 mL), and the layers were separated. The organic phase was washed with water (2 Ã 100 mL) and brine (100 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (100 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate. The racemic mixture was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralcel OD-H (4.6 mm i.d. Ã 25 cm), using 25% EtOH in heptane at a flow rate of 1 mL/min. The material (5.3 g) was dissolved in a mixture of EtOH/heptane (1:1) (250 mg in 1 mL). The purification was carried out on a Chiralcel OD (5 cm Ã 25 cm), using 25% EtOH in heptane at a flow rate of 50 mL/min, and 2 mL of solution was injected at a time. The appropriate fractions were combined and concentrated <i>in vacuo</i>. The residues were dissolved with CH<sub>2</sub>Cl<sub>2</sub> and concentrated under reduced pressure to give the fastest running enantiomer, (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (2.4 g), and the slowest running enantiomer, (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (<b>24</b>) (2.2 g). LCMS (high-pH method): <i>t</i><sub>R</sub> 1.40 min, [M + H]<sup>+</sup> = 465.28. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.77 (d, <i>J</i> = 5.9 Hz, 1H), 7.69 (d, <i>J</i> = 1.0 Hz, 1H), 7.50 (d, <i>J</i> = 7.1 Hz, 2H), 7.32â7.44 (m, 3H), 6.81 (d, <i>J</i> = 5.9 Hz, 1H), 6.30 (d, <i>J</i> = 4.2 Hz, 1H), 5.42â5.61 (m, 2H), 4.18â4.49 (m, 2H), 3.74â3.87 (m, 1H), 3.50â3.61 (m, 1H), 2.63â2.90 (m, 2H), 2.42 (d, <i>J</i> = 0.7 Hz, 3H), 1.99â2.06 (m, 1H), 1.72â1.86 (m, 1H), 1.51 (s, 9H). One H not seen. Step 5: 1,4-dioxaspiro[4.5]decan-8-ylmethyl trifluoromethanesulfonate was prepared from 1,4-dioxaspiro[4.5]decan-8-ylmethanol using the same procedure as that for the synthesis of (1-acetylpiperidin-4-yl)methyl trifluoromethanesulfonate from 1-(4-(hydroxymethyl)piperidin-1-yl)ethanone (see synthesis of <b>25</b>, steps 1 and 2) (1.78 g, 100%, pale yellow oil). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.38 (d, <i>J</i> = 6.6 Hz, 2H), 3.86â4.05 (m, 4H), 1.76â1.95 (m, 5H), 1.59 (td, <i>J</i> = 13.4, 4.3 Hz, 2H), 1.32â1.48 (m, 2H). Step 6: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (100 mg, 0.215 mmol; see compounds <b>31</b> and <b>32</b>, steps 1â4) at 0 Â°C under nitrogen in THF (3 mL) was treated with sodium hydride (60% w/w in mineral oil, 21.5 mg, 0.538 mmol), and the resulting mixture was stirred for 20 min at this temperature before being treated with 1,4-dioxaspiro[4.5]decan-8-ylmethyl trifluoromethanesulfonate (131 mg, 0.431 mmol) as a solution in THF (2 mL). After stirring at 0 Â°C for 2 h, the mixture was diluted with water (10 mL) and extracted with EtOAc (2 Ã 10 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(1,4-dioxaspiro[4.5]decan-8-ylmethoxy)-4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (87 mg, 65%) as a colorless gum. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.56 min, [M + H]<sup>+</sup> = 619.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.90 (d, <i>J</i> = 5.6 Hz, 1H), 7.67 (d, <i>J</i> = 0.7 Hz, 1H), 7.51 (d, <i>J</i> = 7.1 Hz, 2H), 7.38â7.44 (m, 2H), 7.35 (d, <i>J</i> = 7.3 Hz, 1H), 6.75 (d, <i>J</i> = 5.9 Hz, 1H), 6.33â6.38 (m, 1H), 5.50 (d, <i>J</i> = 4.4 Hz, 2H), 4.18â4.31 (m, 1H), 3.79â3.92 (m, 5H), 3.43â3.55 (m, 2H), 3.31â3.38 (m, 2H), 3.19â3.28 (m, 2H), 2.40 (d, <i>J</i> = 0.7 Hz, 3H), 1.55â1.85 (m, 6H), 1.45 (s, 9H), 1.26â1.42 (m, 1H), 1.22â1.34 (m, 2H), 1.04â1.21 (m, 2H). Step 7: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-(1,4-dioxaspiro[4.5]decan-8-ylmethoxy)-4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate (40 mg, 0.065 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at room temperature was treated with TFA (0.5 mL), and the resulting mixture was stirred at this temperature for 1 h and then concentrated <i>in vacuo</i>. The residue was dissolved in MeOH (3 mL), cooled to 0 Â°C using an ice bath, and treated with NaBH<sub>4</sub> (12.2 mg, 0.323 mmol). After stirring for 2 h at this temperature, the mixture was diluted with water (10 mL), and the aqueous phase was extracted with EtOAc (2 Ã 10 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i>. The residue was dissolved in MeOH (30 mL) and hydrogenated in the H-cube on full hydrogen mode over a 10% Pd/C cartridge at 1 mL/min flow rate. The eluent was concentrated <i>in vacuo</i>, and the residue was purified by MDAP (high-pH method) to give 8-(((3<i>R</i>,4<i>R</i>)-3-(((1<i>R</i>,4<i>R</i>)-4-hydroxycyclohexyl) methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>26</b>) (4 mg, 16%). LCMS (high-pH method): <i>t</i><sub>R</sub> 0.63 min, [M + H]<sup>+</sup> = 387. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ = 13.2 (br s, 1H), 7.73 (d, <i>J</i> = 5.1 Hz, 1H), 7.67 (s, 1H), 6.68 (d, <i>J</i> = 5.1 Hz, 1H), 6.59 (d, <i>J</i> = 7.7 Hz, 1H), 4.35 (d, <i>J</i> = 4.4 Hz, 1H), 4.17â4.08 (m, 1H), 3.34â3.28 (m, 1H) 3.22â3.13 (m, 4H), 2.89â2.83 (m, 1H), 2.49â2.44 (m, 1H), 2.32â2.26 (m, 1H), 2.12 (s, 3H), 1.99â1.93 (m, 1H), 1.68â1.62 (m, 2H), 1.55â1.45 (m, 2H), 1.34â1.19 (m, 2H), 1.00â0.91 (m, 2H), 0.78â0.69 (m, 2H). One NH not seen.</div></div><div id="sec4_12_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 3-Methyl-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>27</b>)</h3><div class="NLM_p last">Step 1: a solution of (tetrahydro-2<i>H</i>-pyran-4-yl)methanol (7.0 g, 60 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) at 0 Â°C under nitrogen was treated with pyridine (5.36 mL, 66.3 mmol) and, after 5 min, with the dropwise addition of Tf<sub>2</sub>O (11.2 mL, 66.3 mmol) over 10 min. The resulting mixture was stirred at this temperature for 1 h and was then warmed to room temperature. The organic phase was washed with water (2 Ã 50 mL) and then brine (50 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give (tetrahydro-2<i>H</i>-pyran-4-yl)methyl trifluoromethanesulfonate (14.2 g, 95%) as a colorless oil, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.38 (d, <i>J</i> = 6.6 Hz, 2H), 3.99â4.09 (m, 2H), 3.43 (td, <i>J</i> = 11.9, 2.0 Hz, 2H), 2.06â2.18 (m, 1H), 1.65â1.75 (m, 2H), 1.36â1.51 (m, 2H). Step 2: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (10 g, 41 mmol) in DMF (50 mL) and THF (50 mL) at 0 Â°C under nitrogen was treated portionwise with sodium hydride (60% w/w in mineral oil, 2.2 g, 55 mmol). The resulting mixture was stirred at this temperature for 30 min before being treated with the dropwise addition of a solution of (tetrahydro-2<i>H</i>-pyran-4-yl)methyl trifluoromethanesulfonate (13.3 g, 53.7 mmol) in THF (50 mL) over 20 min. The resulting mixture was then slowly allowed to warm to room temperature over 2 h and then diluted with water (500 mL). The aqueous phase was extracted with Et<sub>2</sub>O (2 Ã 300 mL), and the combined organics were washed with water (2 Ã 400 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give a yellow oil. Purification of this residue by flash chromatography on silica gel (330 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (12.8 g, 91%) as a colorless oil, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.09â4.34 (m, 1H), 3.95â4.03 (m, 3H), 3.85â3.94 (m, 1H), 3.48â3.57 (m, 1H), 3.34â3.47 (m, 4H), 3.12â3.22 (m, 1H), 2.85â2.96 (m, 1H), 2.61â2.82 (m, 1H), 1.77â1.99 (m, 2H), 1.61â1.73 (m, 2H), 1.47 (s, 9H), 1.29â1.43 (m, 2H). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (12.8 g, 37.6 mmol) in THF (100 mL) at room temperature was treated with triphenylphosphine (10.8 g, 41.4 mmol), and the resulting mixture was stirred at room temperature for 16 h. Water (30 mL) was then added, and the mixture was refluxed for 24 h before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was dissolved in Et<sub>2</sub>O (200 mL), and the solution was diluted with cyclohexane (150 mL). Some solid separated after standing for 1 h at room temperature, and the solvent was partially evaporated, causing further precipitation of triphenylphosphine oxide. The solid was filtered off, and the filtrate was concentrated <i>in vacuo</i> to give a pale yellow oil. Purification of this residue by flash chromatography on silica gel (2 Ã 100 g column, gradient: 0 to 10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (8.7 g, 74%) as a pale yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.13â4.47 (m, 1H), 3.92â4.09 (m, 3H), 3.49â3.60 (m, 1H), 3.40 (td, <i>J</i> = 11.7, 2.2 Hz, 2H), 3.31 (dd, <i>J</i> = 9.2, 6.5 Hz, 1H), 2.83â2.93 (m, 1H), 2.76 (br s, 2H), 2.35â2.55 (m, 1H), 1.80â1.88 (m, 2H), 1.61â1.72 (m 2H), 1.52 (s, 2H), 1.47 (s, 9H), 1.23â1.43 (m, 3H). Step 4: a mixture of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (2.3 g, 7.3 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>7b</b>) (1.5 g, 5.3 mmol), sodium 2-methylpropan-2-olate (1.52 g, 15.8 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (0.241 g, 263 mmol), and BrettPhos (0.283 g, 0.527 mmol) in THF (20 mL) was stirred at room temperature under nitrogen for 1 h and was then stirred at 60 Â°C for 3 h before being cooled to room temperature and diluted with a saturated NH<sub>4</sub>Cl aqueous solution. The aqueous phase was extracted with EtOAc (2 Ã 50 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0 to 70% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (2.72 g, 92%) as a pale yellow foam. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.50 min, [M + H]<sup>+</sup> = 563. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.90 (d, <i>J</i> = 5.9 Hz, 1H), 7.68 (d, <i>J</i> = 1.0 Hz, 1H), 7.49 (d, <i>J</i> = 7.1 Hz, 2H), 7.38â7.44 (m, 2H), 7.35 (d, <i>J</i> = 7.1 Hz, 1H), 6.75 (d, <i>J</i> = 5.9 Hz, 1H), 6.28â6.36 (m, 1H), 5.51 (s, 2H), 4.25â4.37 (m, 1H), 3.75â3.88 (m 2H), 3.43â3.52 (m, 1H), 3.32 (d, <i>J</i> = 3.9 Hz, 1H), 3.12â3.27 (m, 3H), 2.89â3.11 (m, 1H), 2.41 (d, <i>J</i> = 0.7 Hz, 3H), 2.28â2.38 (m, 1H), 1.58â1.93 (m, 2H), 1.49â1.54 (m, 11H), 1.23â1.48 (m, 3H), 0.96â1.22 (m, 2H). Step 5: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.18 g, 0.320 mmol) in MeOH (40 mL) was hydrogenated in the H-Cube on full mode using a 10% Pd/C cartridge. The eluant was evaporated <i>in vacuo</i> to give a colorless glass. This material was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL), and the resulting solution was treated with TFA (2 mL). The mixture was stirred at room temperature for 1 h and then concentrated <i>in vacuo.</i> Purification of the residue by MDAP (high-pH method) gave 3-methyl-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>27</b>) (35 mg, 29%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.67 min, [M + H]<sup>+</sup> = 373.2. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.96 (d, <i>J</i> = 5.4 Hz, 1H), 7.66 (s, 1H), 6.68â6.74 (m, 1H), 6.29â6.49 (m, 1H), 4.37â4.51 (m, 1H), 3.73â3.87 (m, 2H), 3.43â3.50 (m, 2H), 3.33â3.42 (m, 2H), 3.18â3.29 (m, 2H), 3.09â3.17 (m, 1H), 2.77â2.87 (m, 1H), 2.65â2.74 (m, 1H), 2.38 (s, 3H), 2.19â2.28 (m, 1H), 1.58 (m, 6H), 1.06â1.26 (m, 2H).</div></div><div id="sec4_12_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 3-Methyl-8-(((3<i>S</i>,4<i>S</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>28</b>) and 3-Methyl-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>29</b>)</h3><div class="NLM_p">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (1.76 g, 3.13 mmol) in MeOH (100 mL) was treated with 10% w/w palladium on charcoal (0.30 g, 0.282 mmol), and the resulting mixture was stirred at room temperature under an atmosphere of hydrogen (1 bar) for 3 h. The catalyst was filtered off using a 2.5 g pad of Celite and rinsed with MeOH. The combined organics were concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-hydroxy-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (1.50 g, 100%).</div><div class="NLM_p last">The racemic mixture was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralpak IC (4.6 mm i.d. Ã 25 cm), using 20% EtOH (+0.2% isopropylamine) in heptane at a flow rate of 1 mL/min. The material (40 mg) was dissolved in a mixture of EtOH/heptane (1:1) (1 mL). The purification was carried out on a Chiralpak IC (30 mm Ã 25 cm), using 20% EtOH (+0.2% isopropylamine) in heptane at a flow rate of 30 mL/min, and 1 mL of solution was injected. The appropriate fractions were combined and concentrated under reduced pressure to give the fastest running enantiomer, (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (10 mg, 0.021 mmol), and the slowest running enantiomer, (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (10 mg, 0.021 mmol). LCMS (high-pH method): <i>t</i><sub>R</sub> 1.47 min, [M + H]<sup>+</sup> = 473. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 7.96 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (d, <i>J</i> = 1.0 Hz, 1H), 6.71 (d, <i>J</i> = 5.6 Hz, 1H), 6.49â6.59 (m, 1H), 4.49â4.62 (m, 1H), 3.95â4.44 (m, 2H), 3.70â3.88 (m, 2H), 3.44â3.56 (m, 2H), 3.21 (d, <i>J</i> = 11.7 Hz, 4H), 2.32 (d, <i>J</i> = 0.7 Hz, 3H), 2.13â2.25 (m, 1H), 1.60â1.79 (m, 2H), 1.51 (s, 9H), 0.99â1.26 (m, 5H). Step 2: (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (10 mg, 0.021 mmol) and (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (10 mg, 0.021 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> and treated with TFA (0.5 mL) at room temperature. The solutions were stirred at this temperature for 2 h and then concentrated <i>in vacuo</i>. The residues were dissolved in MeOH (2 mL) and loaded on to two 5 g SCX-2 cartridges. The cartridges were washed with MeOH and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 3-methyl-8-(((3<i>S</i>,4<i>S</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>28</b>) (7 mg, 89%) and 3-methyl-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>29</b>) (7 mg, 89%).</div></div><div id="sec4_12_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>30</b>)</h3><div class="NLM_p last">Step 1: a solution of (tetrahydro-2<i>H</i>-thiopyran-4-yl)methanol (10 g, 76 mmol) in MeOH (200 mL) at room temperature was treated with a solution of sodium periodate (32.4 g, 151 mmol) in water (300 mL), and the resulting suspension was stirred at 50 Â°C for 60 h, cooled to room temperature, and concentrated <i>in vacuo</i> to leave a white, granular solid. This solid was suspended in EtOAc (200 mL), stirred for 10 min, and then filtered off. The solid obtained was then suspended in 10% MeOH in CH<sub>2</sub>Cl<sub>2</sub> (200 mL), stirred for 10 min, and then filtered off. The combined filtrates were concentrated <i>in vacuo</i> to give 4-(hydroxymethyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (11.2 g, 90%) as a colorless, crystalline solid, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 3.59 (d, <i>J</i> = 6.4 Hz, 2H), 3.06â3.15 (m, 2H), 2.93â3.05 (m, 2H), 2.16â2.27 (m, 2H), 1.84â1.97 (m, 2H), 1.71â1.81 (m, 2H). Step 2: a solution of 4-(hydroxymethyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (3.0 g, 18.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) cooled at â10 Â°C using an acetone/ice bath under nitrogen was treated with pyridine (1.62 mL, 20.1 mmol) and then with Tf<sub>2</sub>O (3.4 mL, 20.1 mmol) dropwise, and the resulting mixture was stirred at this temperature for 1 h and then allowed to warm to room temperature. The solution was then washed with brine (10 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give (1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl trifluoromethanesulfonate (5.2 g, 96%) as a colorless solid, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.42 (d, <i>J</i> = 6.1 Hz, 2H), 3.11â3.22 (m, 2H), 2.96â3.10 (m, 2H), 2.19â2.31 (m, 2H), 1.95â2.17 (m, 3H). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (2.28 g, 9.41 mmol) in THF (10 mL) at 0 Â°C under nitrogen was treated with potassium <i>tert</i>-butoxide (1 N in THF, 14.1 mL, 14.1 mmol), and the resulting mixture was stirred at this temperature for 20 min. (1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl trifluoromethanesulfonate (4.18 g, 14.1 mmol) was then added in small portions over 10 min, and the resulting mixture was stirred at 0 Â°C for 2 h. Water (30 mL) was added, and the aqueous phase was extracted with EtOAc (2 Ã 20 mL). The combined organic phases were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i> to give a pale yellow oil. Purification of this residue by flash chromatography on silica gel (50 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (3.32 g, 91%) as a pale yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 3.82â4.00 (m, 1H), 3.53â3.65 (m, 1H), 3.39â3.52 (m, 2H), 3.15â3.23 (m, 1H), 3.04â3.14 (m, 3H), 2.87â3.03 (m, 3H), 2.70â2.81 (m, 2H), 2.19 (d, <i>J</i> = 12.2 Hz, 2H), 1.79â2.01 (m, 4H), 1.38â1.51 (m, 9H). Step 4: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (3.3 g, 8.5 mmol) in MeOH (100 mL) was treated with palladium on charcoal (5% w/w, 50% wet, 1 g), and the resulting mixture was stirred under an atmosphere of hydrogen (1 bar) for 2 h. The catalyst was filtered off using a pad of Celite (2.5 g), and the filtrate was concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (2.34 g, 76%) as a colorless crystalline solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.13â4.43 (m, 1H), 3.92â4.08 (m, 1H), 3.54â3.65 (m, 1H), 3.39 (dd, <i>J</i> = 9.2, 6.0 Hz, 1H), 3.05â3.14 (m, 2H), 2.87â3.04 (m, 3H), 2.71â2.85 (m, 2H), 2.41â2.55 (m, 1H), 2.11â2.23 (m, 2H), 1.91â2.02 (m, 2H), 1.78â1.88 (m, 2H), 1.42â1.61 (m, 11H), 1.25â1.41 (m, 1H). Step 5: a mixture of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (1.7 g, 4.7 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>7b</b>) (1.10 g, 3.86 mmol), sodium 2-methylpropan-2-olate (1.11 g, 11.6 mmol), BrettPhos (0.207 g, 0.386 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.177 g, 0.193 mmol) in THF (20 mL) was stirred under nitrogen at room temperature for 1 h and was then stirred at 60 Â°C for 3 h before being cooled to room temperature and diluted with a saturated NH<sub>4</sub>Cl aqueous solution. The aqueous phase was extracted with EtOAc (2 Ã 50 mL), and the combined organic phases were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0 to 70% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (2.31 g, 98%) as a pale yellow foam. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.38 min, [M + H]<sup>+</sup> = 611. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.90 (d, <i>J</i> = 5.6 Hz, 1H), 7.69 (s, 1H), 7.47â7.52 (m, 2H), 7.41 (t, <i>J</i> = 7.3 Hz, 2H), 7.35 (d, <i>J</i> = 7.3 Hz, 1H), 6.75 (d, <i>J</i> = 5.6 Hz, 1H), 6.22 (d, <i>J</i> = 8.6 Hz, 1H), 5.46â5.58 (m, 2H), 4.30â4.41 (m, 1H), 3.89â4.11 (m, 1H), 3.76â3.87 (m, 1H), 3.46â3.57 (m, 1H), 3.31 (d, <i>J</i> = 3.7 Hz, 4H), 2.67 (br s, 4H), 2.42 (s, 3H), 2.17â2.28 (m, 1H), 1.86â1.99 (m, 1H), 1.74â1.83 (m, 1H), 1.39â1.72 (m, 13H). Step 6: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (26 mg, 0.043 mmol) in TFA (2 mL) was stirred at 50 Â°C for 2 h and then cooled to room temperature and concentrated <i>in vacuo</i>. The residue was dissolved in MeOH and loaded on to a 5 g SCX-2 cartridge. The cartridge was washed with MeOH and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i>. Purification of the residue obtained by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>30</b>) (7.5 mg, 42%) as a beige solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.62 min, [M + H]<sup>+</sup> = 421.1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 7.95 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (d, <i>J</i> = 1.0 Hz, 1H), 6.62â6.77 (m, 2H), 4.45â4.60 (m, 1H), 3.54â3.63 (m, 2H), 3.44 (dd, <i>J</i> = 8.9, 6.0 Hz, 2H), 3.24 (d, <i>J</i> = 12.7 Hz, 1H), 2.71â2.98 (m, 6H), 2.33â2.38 (m, 3H), 2.22â2.31 (m, 1H), 1.93â2.10 (m, 2H), 1.60â1.84 (m, 4H). One NH not seen.</div></div><div id="sec4_12_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 8-(((3<i>S</i>,4<i>S</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>31</b>) and 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>32</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (75 mg, 0.16 mmol) (see steps 1â4 of compound <b>26</b>) and a solution of (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (75 mg, 0.16 mmol) in DMF (5 mL) at 0 Â°C under nitrogen were treated with sodium hydride (60% w/w in mineral oil, 21.5 mg, 0.538 mmol), and the resulting mixtures were stirred at this temperature for 10 min; then, (1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl trifluoromethanesulfonate (96 mg, 0.323 mmol) was added as a solution in DMF (1 mL) to each reaction. The mixtures were allowed to warm to room temperature over 2 and were then diluted with water (10 mL) and extracted with EtOAc (10 mLl). In each case, the organic layer was washed with water (10 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo.</i> Purification of each residue by flash chromatography on silica gel (10 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (40 mg, 41%) as a colorless glass and (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (37 mg, 37%) as a colorless glass. Step 2: (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (40 mg, 0.065 mmol) and (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (35 mg, 0.057 mmol) were each dissolved in TFA (3 mL), and the solutions were stirred at 60 Â°C for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. The residues were dissolved in MeOH (2 mL) and loaded on to two 5 g SCX-2 cartridges. The cartridges were washed with MeOH and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 8-(((3<i>S</i>,4<i>S</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>31</b>) (26 mg, 94%) and 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>32</b>) (12.4 mg, 45%) as beige solids.</div></div><div id="sec4_12_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydrothiophen-3-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>33</b>)</h3><div class="NLM_p last">Step 1: (1,1-dioxidotetrahydrothiophen-3-yl)methyl trifluoromethanesulfonate was prepared from 3-(hydroxymethyl)tetrahydrothiophene 1,1-dioxide using the same procedure as that for the synthesis of (1-acetylpiperidin-4-yl)methyl trifluoromethanesulfonate from 1-(4-(hydroxymethyl)piperidin-1-yl)ethanone (1.72 g, 92%, pale yellow crystalline solid). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.52â4.65 (m, 2H), 3.22â3.38 (m, 2H), 3.07â3.20 (m, 1H), 2.94â3.05 (m, 1H), 2.86â2.94 (m, 1H), 2.41â2.52 (m, 1H), 2.07 (dq, <i>J</i> = 13.6, 9.4 Hz, 1H). Step 2: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (<b>24</b>) (450 mg, 0.969 mmol) in THF (5 mL) under nitrogen was cooled to 0 Â°C using an ice bath and was then treated with potassium <i>tert</i>-butoxide (239 mg, 2.13 mmol). After stirring for 10 min, (1,1-dioxidotetrahydrothiophen-3-yl)methyl trifluoromethanesulfonate (547 mg, 1.94 mmol) was added, and the resulting mixture was stirred for 2 h at this temperature. After standing over 2 days at room temperature under a nitrogen atmosphere, the reaction mixture was concentrated <i>in vacuo</i>. Purification of the residue obtained by flash chromatography on silica gel (25 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydrothiophen-3-yl)methoxy)piperidine-1-carboxylate (55 mg, 10% yield) as a pale yellow gum. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.38 min, [M + H]<sup>+</sup> = 597.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.89 (d, <i>J</i> = 5.6 Hz, 1H), 7.69 (s, 1H), 7.48 (d, <i>J</i> = 7.1 Hz, 2H), 7.39 (t, <i>J</i> = 7.3 Hz, 2H), 7.34 (d, <i>J</i> = 7.3 Hz, 1H), 6.98 (dd, <i>J</i> = 17.1, 10.8 Hz, 1H), 6.77 (d, <i>J</i> = 5.9 Hz, 1H), 5.45â5.54 (m, 2H), 4.32â4.44 (m, 1H), 4.09â4.20 (m, 1H), 3.95â4.05 (m, 1H), 3.59â3.94 (m, 2H), 3.27â3.51 (m, 2H), 3.14â3.26 (m, 2H), 2.99â3.12 (m, 1H), 2.75â2.97 (m, 2H), 2.25â2.43 (m, 4H), 2.14â2.24 (m, 1H), 1.93â2.12 (m, 1H), 1.68â1.84 (m, 1H), 1.51 (s, 9H), 1.23â1.37 (m, 1H). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydrothiophen-3-yl)methoxy)piperidine-1-carboxylate (55 mg, 0.092 mmol) in TFA (1 mL) was refluxed for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. The residue was loaded on to a 5 g SCX cartridge, washed with MeOH, and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were combined and concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydrothiophen-3-yl)methoxy)piperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>33</b>) (17 mg, 45%) as a beige glass. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.61 min, [M + H]<sup>+</sup> = 407. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ = 7.95 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (s, 1H), 6.72 (d, <i>J</i> = 5.4 Hz, 1H), 6.60 (d, <i>J</i> = 7.3 Hz, 1H), 4.55â4.43 (m, 1H), 3.79â3.67 (m, 1H), 3.67â3.59 (m, 1H), 3.58â3.52 (m, 1H), 3.44â3.32 (m, 1H), 3.21â3.11 (m, 1H), 3.06â2.91 (m, 2H), 2.91â2.66 (m, 4H), 2.66â2.56 (m, 1H), 2.36 (s, 3H), 2.28â2.20 (m, 1H), 2.17â2.06 (m, 1H), 1.94â1.77 (m, 1H), 1.70 (d, <i>J</i> = 10.0 Hz, 1H). Two NH not seen.</div></div><div id="sec4_12_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 3-Methyl-8-(((3<i>R</i>,4<i>R</i>)-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>34</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-hydroxypiperidine-1-carboxylate (<b>5</b>) (0.64 g, 2.64 mmol) in DMF (10 mL) under nitrogen was cooled at 0 Â°C using an ice bath and then treated with sodium hydride (60% w/w in mineral oil, 0.14 g, 3.50 mmol). The resulting mixture was stirred at this temperature for 20 min; then, 4-(2-bromoethyl)tetrahydropyran (0.61 g, 3.17 mmol) was added. After stirring at 0 Â°C for 2 h, the reaction mixture was allowed to warm to room temperature and stirred for a further 2 h. The reaction mixture was then treated with water (30 mL), and the aqueous phase was extracted with EtOAc (2 Ã 30 mL). The combined organics were washed with water (2 Ã 30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of this residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidine-1-carboxylate (0.55 g, 59% yield) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.04â4.41 (m, 1H), 3.96 (s, 3H), 3.67â3.80 (m, 1H), 3.53â3.64 (m, 1H), 3.34â3.46 (m, 3H), 3.12â3.22 (m, 1H), 2.81â2.92 (m, 1H), 2.70 (br s, 1H), 1.87â1.99 (m, 1H), 1.52â1.78 (m, 5H), 1.40â1.51 (m, 10H), 1.25â1.39 (m, 2H). Step 2: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-azido-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidine-1-carboxylate (0.52 g, 1.467 mmol) in MeOH (80 mL) was hydrogenated in the H-cube on full hydrogen mode over a 10% Pd/C cartridge at 1 mL/min flow rate. The eluant was concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidine-1-carboxylate (0.48 g, 100%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 4.13â4.55 (m, 1H), 4.00â4.09 (m, 1H), 3.96 (dd, <i>J</i> = 11.2, 3.9 Hz, 2H), 3.68â3.78 (m, 1H), 3.47â3.54 (m, 1H), 3.39 (t, <i>J</i> = 11.6 Hz, 2H), 2.84â2.93 (m, 1H), 2.68â2.80 (m, 2H), 2.35â2.56 (m, 1H), 1.79â1.87 (m, 1H), 1.52â1.74 (m, 6H), 1.45â1.50 (m, 10H), 1.25â1.40 (m, 3H). Step 3: a suspension of 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) (40 mg, 0.21 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidine-1-carboxylate (101 mg, 0.308 mmol) and Caddick catalyst (12 mg, 0.021 mmol) in THF (250 mL) at room temperature under nitrogen was treated with LiHMDS (1 M in THF, 0.62 mL, 0.62 mmol) and was then stirred at 60 Â°C for 3 h before being cooled to 0 Â°C and quenched with a saturated NH<sub>4</sub>Cl aqueous solution. The mixture was diluted with EtOAc (30 mL), and the insoluble material was filtered off and then washed with EtOAc (2 Ã 50 mL) to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidine-1-carboxylate (52 mg, 52%) as a pale yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.07 min, [M + H]<sup>+</sup> = 487.4. Step 4: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidine-1-carboxylate (52 mg, 0.107 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL). The resulting mixture was stirred at room temperature for 1 h and was then concentrated <i>in vacuo</i>. The residue was loaded on to a 5 g SCX cartridge, washed with MeOH (20 mL), and eluted with a 2 N NH<sub>3</sub> solution in MeOH (20 mL). The ammoniac fractions were concentrated <i>in vacuo</i> to give 3-methyl-8-(((3<i>R</i>,4<i>R</i>)-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>34</b>) (35 mg, 85% yield) as a brown solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.68 min, [M + H]<sup>+</sup> = 387. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 0.98â1.18 (m, 2H), 1.27â1.75 (m, 7H), 2.14â2.25 (m, 1H), 2.36 (s, 3H), 2.63â2.73 (m, 1H), 2.80 (br t, <i>J</i> = 10.3 Hz, 1H), 2.94 (br t, <i>J</i> = 12.0 Hz, 1H), 3.04â3.13 (m, 1H), 3.09â3.18 (m, 1H), 3.37â3.47 (m, 2H), 3.51â3.60 (m, 1H), 3.63â3.71 (m, 2H), 3.75 (br dd, <i>J</i> = 11.4, 3.7 Hz, 1H), 4.37â4.52 (m, 1H), 6.41 (br s, 1H), 6.69 (d, <i>J</i> = 5.5 Hz, 1H), 7.65 (q, <i>J</i> = 1.5 Hz, 1H), 7.94 (d, <i>J</i> = 5.4 Hz, 1H), 13.09 (br s, 1H).</div></div><div id="sec4_12_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 3-Methyl-5-(pyridin-3-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>35</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (2.7 g, 4.8 mmol) in CH<sub>2</sub>Cl<sub>2</sub> at 0 Â°C was treated with NBS (0.683 g, 3.84 mmol), and the resulting mixture was stirred at this temperature for 30 min. Further NBS (50 mg, 0.28 mmol) was added, and the mixture was stirred for another 10 min, warmed to room temperature, and washed with a saturated sodium metabisulphite aqueous solution. The organic phase was then dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (2.92 g, 95%) as a beige foam, which was used in the next step without further purification. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.68 min, [M + H]<sup>+</sup> = 641/643 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 8.03 (s, 1H), 7.99 (d, <i>J</i> = 1.2 Hz, 1H), 7.46â7.50 (m, 2H), 7.38â7.44 (m, 2H), 7.36 (d, <i>J</i> = 7.1 Hz, 1H), 6.29 (d, <i>J</i> = 7.8 Hz, 1H), 5.52 (s, 2H), 4.17â4.30 (m, 1H), 3.82 (d, <i>J</i> = 7.6 Hz, 3H), 3.43â3.52 (m, 1H), 3.30 (d, <i>J</i> = 3.9 Hz, 1H), 3.14â3.25 (m, 3H), 2.46 (d, <i>J</i> = 1.0 Hz, 2H), 2.26â2.36 (m, 1H),1.60â1.72 (m, 2H), 1.52 (s, 11H), 1.50 (br s, 4H), 0.96â1.25 (m, 2H). Step 2: a 20 mL microwave vial was charged with K<sub>2</sub>CO<sub>3</sub> (172 mg, 1.25 mmol), Pd(OAc)<sub>2</sub> (7.0 mg, 0.031 mmol), 3-pyridineboronic acid pinacol ester (128 mg, 0.623 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (200 mg, 0.312 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (11.18 mg, 0.031 mmol) and was then filled with 1,4-dioxane (8 mL) and water (4 mL). The reaction mixture was degassed for 20 min with nitrogen and then stirred under microwave irradiation at 100 Â°C for 1 h before being cooled to room temperature. The solvent was removed <i>in vacuo</i>, and the residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL). The layers were separated, and the organic phase was dried using a hydrophobic frit and then concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 8% MeOH in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(pyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (133 mg, 60%). LCMS (high-pH method): <i>t</i><sub>R</sub> 1.45 min, [M + H]<sup>+</sup> = 640. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.59â8.66 (m, 2H), 7.84â7.88 (m, 1H), 7.82 (d, <i>J</i> = 1.2 Hz, 1H), 7.80 (s, 1H), 7.50â7.58 (m, 3H), 7.37â7.43 (m, 2H), 7.33 (s, 1H), 6.93â6.98 (m, 1H), 5.59â5.73 (m, 2H), 4.26â4.36 (m, 1H), 3.76â3.87 (m, 1H), 3.58â3.66 (m, 2H), 3.45â3.54 (m, 1H), 3.38â3.44 (m, 1H), 3.19â3.26 (m, 1H), 3.06 (br s, 3H), 2.31 (d, <i>J</i> = 0.7 Hz, 3H), 2.10â2.23 (m, 1H), 2.01â2.09 (m, 1H), 1.88â1.98 (m, 1H), 1.53â1.77 (m, 2H), 1.44 (s, 9H), 1.26â1.36 (m, 2H), 0.83â1.04 (m, 2H). Step 3: a flask was charged with (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(pyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (102 mg, 0.159 mmol) and then filled with TFA (2 mL), and the resulting mixture was stirred at 80 Â°C for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 3-methyl-5-(pyridin-3-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (34 mg, 47%) as a colorless foam. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.66 min, [M + H]<sup>+</sup> = 450. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.50 (br s, 1H), 8.62 (dd, <i>J</i> = 4.8, 1.6 Hz, 1H), 8.59 (d, <i>J</i> = 1.7 Hz, 1H), 7.80 (s, 1H), 7.72 (s, 1H), 7.50â7.55 (m, 2H), 6.84â6.88 (m, 1H), 4.13â4.27 (m, 1H), 3.62â3.72 (m, 2H), 3.37â3.43 (m, 1H), 3.07â3.29 (m, 5H), 2.84â2.93 (m, 1H), 2.26â2.36 (m, 1H), 2.09 (d, <i>J</i> = 1.2 Hz, 3H), 1.96â2.05 (m, 1H), 1.56â1.69 (m, 1H), 1.32â1.41 (m, 3H), 0.93â1.10 (m, 2H). Two NH not seen.</div></div><div id="sec4_12_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(pyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>36</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (2.6 g, 4.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) at â10 Â°C was treated with NBS (0.66 g, 3.71 mmol), and the resulting mixture stirred at this temperature for 30 min before being treated with a saturated sodium metabisulphite aqueous solution (50 mL). The resulting mixture was stirred for 20 min at room temperature, and the layers were separated. The organic phase was dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (2.71 g, 92%) as an orange solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.54 min, [M + H]<sup>+</sup> = 688/690 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 8.03 (s, 1H), 8.00 (d, <i>J</i> = 1.2 Hz, 1H), 7.46â7.50 (m, 2H), 7.38â7.44 (m, 2H), 7.36 (d, <i>J</i> = 7.3 Hz, 1H), 6.21 (d, <i>J</i> = 8.3 Hz, 1H), 5.47â5.57 (m, 2H), 4.24â4.35 (m, 1H), 3.85â4.10 (m, 1H), 3.76â3.84 (m, 1H), 3.47â3.56 (m, 1H), 3.03â3.34 (m, 3H), 2.88â2.97 (m, 1H), 2.65â2.86 (m, 3H), 2.48 (d, <i>J</i> = 0.7 Hz, 3H), 2.17â2.26 (m, 1H), 1.89â2.01 (m, 1H), 1.77â1.86 (m, 1H), 1.64â1.76 (m, 1H), 1.52 (s, 9H), 1.45â1.65 (m, 4H). Step 2: a 20 mL microwave vial was charged with pyridin-3-ylboronic acid (21.4 mg, 0.174 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (80 mg, 0.12 mmol), K<sub>2</sub>CO<sub>3</sub> (64.1 mg, 0.464 mmol), Pd(OAc)<sub>2</sub> (2.6 mg, 0.012 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (4.2 mg, 0.012 mmol) and then filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was degassed under vacuum, quenched with nitrogen for 20 min, and stirred under nitrogen and microwave irradiation at 100 Â°C for 30 min before being cooled to room temperature and concentrated <i>in vacuo.</i> The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic layer was dried using a hydrophobic frit and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2 N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)-4-((3-methyl-2-oxo-5-(pyridin-3-yl)-1,2-dihydro-1,7-naphthyridin-8-yl)amino)piperidine-1-carboxylate. This derivative was dissolved in TFA (2 mL), and the resulting solution was stirred at reflux for 16 h, cooled to room temperature, and concentrated <i>in vacuo</i>. The residue was loaded on a 5 g SCX cartridge, washed with MeOH, and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(pyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>36</b>) (32 mg, 55%) as yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.61 min, [M + H]<sup>+</sup> = 498. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.50 (br s, 1H), 8.55â8.68 (m, 1H), 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.73 (s, 1H), 7.47â7.58 (m, 2H), 6.85 (d, <i>J</i> = 7.6 Hz, 1H), 4.19 (br s, 1H), 3.46 (dd, <i>J</i> = 9.3, 6.4 Hz, 1H), 3.21â3.28 (m, 2H), 3.17 (d, <i>J</i> = 2.4 Hz, 1H), 2.94â3.05 (m, 2H), 2.82â2.93 (m, 3H), 2.29â2.41 (m, 1H), 2.09 (s, 3H), 1.99â2.06 (m, 1H), 1.82â1.96 (m, 2H), 1.67â1.79 (m, 1H), 1.47â1.49 (m, 1H), 1.41â1.55 (m, 3H), 1.35 (d, <i>J</i> = 9.3 Hz, 2H).</div></div><div id="sec4_12_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 3-Methyl-5-(5-methylpyridin-3-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>37</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (1.5 g, 3.2 mmol) in CHCl<sub>3</sub> (20 mL) at â10 Â°C was treated with NBS (0.508 g, 2.86 mmol), and the resulting mixture was stirred at this temperature for 1 h and then treated with a saturated sodium metabisulphite aqueous solution (20 mL). The biphasic mixture was warmed to room temperature and stirred for 10 min, and the layers were separated. The organic phase was dried using a phase separator and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((5-bromo-3-methyl-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (1.63 g, 93%) as a yellow solid, which was used in the next step without further purification. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.22 min, [M + H]<sup>+</sup> = 551/553 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.21 (br s, 1H), 8.08 (s, 1H), 8.04 (s, 1H), 6.58 (d, <i>J</i> = 7.1 Hz, 1H), 4.41â4.54 (m, 1H), 4.15â4.35 (m, 2H), 3.96â4.06 (m, 1H), 3.82 (br s, 2H), 3.41â3.56 (m, 2H), 3.03â3.35 (m, 4H), 2.36 (s, 3H), 2.15â2.25 (m, 1H), 1.67 (br s, 2H), 1.52 (s, 9H), 1.33â1.48 (m, 2H), 1.03â1.25 (m, 2H). Step 2: a 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (64.6 mg, 0.471 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((5-bromo-2-hydroxy-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (200 mg, 0.363 mmol), K<sub>2</sub>CO<sub>3</sub> (150 mg, 1.09 mmol), Pd(OAc)<sub>2</sub> (8.1 mg, 0.036 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (13 mg, 0.036 mmol) and then filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was degassed for 20 min under vacuum, quenched with nitrogen, and stirred at 100 Â°C for 30 min under microwave irradiation before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a hydrophobic frit and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-5-(5-methylpyridin-3-yl)-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.178 g, 87%) as a pale yellow solid. The racemic mixture was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralpak IC (4.6 mm i.d. Ã 25 cm), using 40% EtOH in heptane at a flow rate of 1 mL/min. The material (120 mg) was dissolved in a mixture of EtOH/heptane (2:1) (1 mL). The purification was carried out on a Chiralpak IC (30 mm Ã 25 cm), using 40% EtOH in heptane at a flow rate of 30 mL/min, and 1 mL of solution was injected at a time. The appropriate fractions were combined and concentrated <i>in vacuo</i> to give the fastest running enantiomer, (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((3-methyl-5-(5-methylpyridin-3-yl)-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (58 mg), and the slowest running enantiomer, (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-5-(5-methylpyridin-3-yl)-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (53 mg). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((3-methyl-5-(5-methylpyridin-3-yl)-2-oxo-1,2-dihydro-1,7-naphthyridin-8-yl)amino)-3-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)piperidine-1-carboxylate (58 mg, 0.10 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) at room temperature was treated with TFA (1 mL), and the resulting mixture was stirred at this temperature for 30 min and then concentrated <i>in vacuo</i>. The residue was loaded on a 2 g SCX-2 cartridge, washed with MeOH, and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were combined and concentrated <i>in vacuo</i> to give 3-methyl-5-(5-methylpyridin-3-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>37</b>) (43 mg, 90%). LCMS (method TFA): <i>t</i><sub>R</sub> 0.48 min, [M + H]<sup>+</sup> = 464. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 8.52 (d, <i>J</i> = 1.5 Hz, 1H), 8.47 (d, <i>J</i> = 1.7 Hz, 1H), 7.91 (s, 1H), 7.72 (s, 1H), 7.53 (s, 1H), 6.69 (d, <i>J</i> = 6.6 Hz, 1H), 4.51 (d, <i>J</i> = 4.4 Hz, 1H), 3.76â3.89 (m, 2H), 3.39 (dd, <i>J</i> = 9.3, 6.6 Hz, 2H), 3.26 (qd, <i>J</i> = 11.8, 2.0 Hz, 2H), 3.10â3.18 (m, 1H), 2.83 (t, <i>J</i> = 10.1 Hz, 1H), 2.70 (dd, <i>J</i> = 12.2, 8.8 Hz, 1H), 2.47 (s, 3H), 2.35 (m, 4H), 2.22â2.30 (m, 1H), 1.64â1.84 (m, 3H), 1.41â1.56 (m, 2H), 1.07â1.29 (m, 2H). One NH not seen.</div></div><div id="sec4_12_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>38</b>)</h3><div class="NLM_p last">A 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (34.2 mg, 0.249 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (see preparation of <b>36</b>) (86 mg, 0.12 mmol), K<sub>2</sub>CO<sub>3</sub> (68.9 mg, 0.499 mmol), Pd(OAc)<sub>2</sub> (2.8 mg, 0.012 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (4.5 mg, 0.012 mmol) and then filled with 1,4-dioxane (4 mL) and water (2 mL). The reaction mixture was stirred and degassed for 20 min with nitrogen and was then stirred at 100 Â°C for 30 min under microwave irradiation before being cooled to room temperature and concentrated <i>in vacuo.</i> The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried through a hydrophobic frit and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub> gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate as a pale yellow solid. This residue was dissolved in TFA (3 mL), and the resulting solution was refluxed for 2 h before being cooled to room temperature and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>38</b>) (47 mg, 74%) as a pale yellow solid. LCMS (method TFA): <i>t</i><sub>R</sub> 0.68 min, [M + H]<sup>+</sup> = 512.3. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) Î´ ppm 8.77â8.80 (m, 1H), 8.73â8.76 (m, 1H), 8.40â8.46 (m, 1H), 7.82â7.86 (m, 1H), 7.77â7.80 (m, 1H), 4.44â4.55 (m, 1H), 3.87â3.95 (m, 1H), 3.64â3.76 (m, 2H), 3.46â3.59 (m, 2H), 3.21â3.30 (m, 2H), 3.11â3.19 (m, 1H), 2.91â3.11 (m, 2H), 2.79â2.89 (m, 1H), 2.64 (s, 3H), 2.39â2.50 (m, 1H), 2.30 (s, 3H), 2.09â2.21 (m, 1H), 2.01â2.09 (m, 1H), 1.91â2.00 (m, 1H), 1.73â1.90 (m, 2H), 1.56â1.71 (m, 1H). Three NH protons not seen.</div></div><div id="sec4_12_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 5-(5-Methoxypyridin-3-yl)-3-methyl-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>39</b>)</h3><div class="NLM_p last">Step 1: a 20 mL microwave vial was charged with K<sub>2</sub>CO<sub>3</sub> (172 mg, 1.25 mmol), Pd(OAc)<sub>2</sub> (7.0 mg, 0.031 mmol), 3-methoxypyridine-5-boronic acid pinacol ester (147 mg, 0.623 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (200 mg, 0.312 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (11.2 mg, 0.031 mmol) and then filled with 1,4-dioxane (8 mL) and water (4 mL). The resulting mixture was degassed under vacuum, quenched with nitrogen over 20 min, and stirred under microwave irradiation at 100 Â°C for 1 h before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a hydrophobic frit and then concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 2 to 8% MeOH in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-(5-methoxypyridin-3-yl)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (120 mg, 46%) as a pale yellow solid. LCMS (method TFA): <i>t</i><sub>R</sub> 1.47 min, [M + H]<sup>+</sup> = 670. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.33 (d, <i>J</i> = 2.9 Hz, 1H), 8.22 (d, <i>J</i> = 1.7 Hz, 1H), 7.87 (d, <i>J</i> = 1.0 Hz, 1H), 7.82 (s, 1H), 7.54 (d, <i>J</i> = 7.1 Hz, 2H), 7.37â7.44 (m, 3H), 7.30â7.36 (m, 1H), 6.92â6.99 (m, 1H), 5.59â5.74 (m, 2H), 4.25â4.36 (m, 1H), 3.90â4.15 (m, 2H), 3.89 (s, 3H), 3.76â3.85 (m, 1H), 3.57â3.66 (m, 2H), 3.45â3.54 (m, 1H), 3.37â3.44 (m, 1H), 3.20â3.26 (m, 1H), 3.07 (m, 2H), 2.70â3.12 (m, 1H), 2.32 (d, <i>J</i> = 0.7 Hz, 3H), 2.00â2.09 (m, 1H), 1.52â1.66 (m, 2H), 1.44 (s, 9H), 1.26â1.37 (m, 2H), 0.88â1.05 (m, 2H). Step 2: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-(5-methoxypyridin-3-yl)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (130 mg, 0.194 mmol) in TFA (2 mL) was stirred at 80 Â°C for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 5-(5-methoxypyridin-3-yl)-3-methyl-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>39</b>) (66 mg, 71%) as a colorless foam. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.70 min, [M + H]<sup>+</sup> = 480. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.50 (br s, 1H), 8.32â8.36 (m, 1H), 8.13â8.20 (m, 1H), 7.72â7.78 (m, 1H), 7.58 (d, <i>J</i> = 1.2 Hz, 1H), 7.37 (dd, <i>J</i> = 2.7, 2.0 Hz, 1H), 6.85 (d, <i>J</i> = 7.6 Hz, 1H), 4.13â4.27 (m, 1H), 3.89 (s, 3H), 3.63â3.73 (m, 2H), 3.40 (dd, <i>J</i> = 9.3, 6.6 Hz, 1H), 3.18â3.29 (m, 3H), 3.14 (tt, <i>J</i> = 11.7, 2.4 Hz, 2H), 2.89 (d, <i>J</i> = 12.5 Hz, 1H), 2.26â2.37 (m, 1H), 2.10 (d, <i>J</i> = 1.0 Hz, 3H), 2.00 (d, <i>J</i> = 8.8 Hz, 1H), 1.63 (br s, 2H), 1.28â1.44 (m, 3H), 0.95â1.10 (m, 2H). One NH not seen.</div></div><div id="sec4_12_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-5-(5-methoxypyridin-3-yl)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>40</b>)</h3><div class="NLM_p last">Step 1: a 20 mL microwave vial was charged with (5-methoxypyridin-3-yl)boronic acid (26.6 mg, 0.174 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (see preparation of <b>36</b>) (80 mg, 0.116 mmol), K<sub>2</sub>CO<sub>3</sub> (64.1 mg, 0.464 mmol), Pd(OAc)<sub>2</sub> (2.60 mg, 0.012 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (4.2 mg, 0.012 mmol) and then filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was degassed under vacuum, quenched with nitrogen for 20 min, and stirred under nitrogen and microwave irradiation at 100 Â°C for 30 min before being cooled to room temperature and concentrated <i>in vacuo.</i> The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic layer was dried using a hydrophobic frit and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave 4-((((3<i>R</i>,4<i>R</i>)-4-((2-(benzyloxy)-5-(5-methoxypyridin-3-yl)-3-methyl-1,7-naphthyridin-8-yl)amino)piperidin-3-yl)oxy)methyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide. This derivative was dissolved in TFA (2 mL), and the resulting solution was refluxed for 16 h, cooled to room temperature, and concentrated <i>in vacuo</i>. The residue was loaded on a 5 g SCX cartridge, washed with MeOH, and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-5-(5-methoxypyridin-3-yl)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (28 mg, 46%) as yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.70 min, [M + H]<sup>+</sup> = 528. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.20 (br s, 1H), 8.39 (d, <i>J</i> = 2.7 Hz, 1H), 8.27 (s, 1H), 7.92 (s, 1H), 7.77 (s, 1H), 7.26 (d, <i>J</i> = 1.5 Hz, 1H), 6.75 (d, <i>J</i> = 7.3 Hz, 1H), 4.76 (br s, 1H), 4.51 (d, <i>J</i> = 3.7 Hz, 1H), 3.95 (s, 3H), 3.51â3.62 (m, 2H), 3.48 (dd, <i>J</i> = 8.7, 6.2 Hz, 1H), 3.38â3.44 (m, 1H), 3.16 (d, <i>J</i> = 12.5 Hz, 1H), 2.75â2.99 (m, 5H), 2.69 (dd, <i>J</i> = 11.7, 9.0 Hz, 1H), 2.34 (s, 3H), 2.25 (d, <i>J</i> = 10.0 Hz, 1H), 2.05 (t, <i>J</i> = 12.3 Hz, 2H), 1.62â1.90 (m, 4H).</div></div><div id="sec4_12_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 3-Methyl-5-(pyrimidin-5-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>41</b>)</h3><div class="NLM_p last">Step 1: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (2.7 g, 4.8 mmol) in CH<sub>2</sub>Cl<sub>2</sub> at 0 Â°C was treated with NBS (0.683 g, 3.84 mmol), and the resulting solution was stirred at this temperature for 30 min. Further NBS (50 mg, 0.28 mmol) was added. The solution was stirred at 0 Â°C for 10 min before being treated with a saturated sodium metabisulphite aqueous solution. The layers were separated, and the organic phase was dried using a phase separator and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (2.92 g, 95%) as a beige foam, which was used in the next step without further purification. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.67 min, [M + H]<sup>+</sup> = 643.5 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 8.03 (s, 1H), 7.99 (d, <i>J</i> = 1.2 Hz, 1H), 7.46â7.50 (m, 2H), 7.38â7.44 (m, 2H), 7.36 (d, <i>J</i> = 7.1 Hz, 1H), 6.29 (d, <i>J</i> = 7.8 Hz, 1H), 5.52 (s, 2H), 4.14â4.31 (m, 1H), 3.82 (d, <i>J</i> = 7.6 Hz, 2H), 3.41â3.54 (m, 1H), 3.30 (d, <i>J</i> = 3.9 Hz, 1H), 2.92â3.25 (m, 4H), 2.46 (d, <i>J</i> = 1.0 Hz, 2H), 2.26â2.36 (m, 1H), 1.61â1.71 (m, 2H), 1.33â1.55 (m, 15H), 1.11â1.24 (m, 1H), 0.98â1.10 (m, 1H). Step 2: a 20 mL microwave vial was charged with K<sub>2</sub>CO<sub>3</sub> (172 mg, 1.25 mmol), Pd(OAc)<sub>2</sub> (7.00 mg, 0.031 mmol), pyrimidine-5-boronic acid (77 mg, 0.62 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (200 mg, 0.312 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (11 mg, 0.031 mmol) and then filled with 1,4-dioxane (8 mL) and water (4 mL). The resulting mixture was stirred, degassed for 20 min with nitrogen, and stirred at 100 Â°C under microwave irradiation for 1 h before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a hydrophobic frit and then concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 2 to 8% MeOH in CH<sub>2</sub>Cl<sub>2</sub>) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(pyrimidin-5-yl)-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (120 mg, 60%). LCMS (high-pH method): <i>t</i><sub>R</sub> 1.40 min, [M + H]<sup>+</sup> = 641.6. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.30 (s, 1H), 9.23 (s, 1H), 8.91 (s, 2H), 7.89 (d, <i>J</i> = 1.0 Hz, 1H), 7.85 (s, 1H), 7.55 (d, <i>J</i> = 7.1 Hz, 2H), 7.38â7.44 (m, 2H), 7.34 (s, 1H), 7.01â7.09 (m, 1H), 5.59â5.74 (m, 2H), 4.26â4.38 (m, 1H), 3.88â4.19 (m, 1H), 3.77â3.87 (m, 1H), 3.57â3.67 (m, 2H), 3.47â3.55 (m, 1H), 3.38â3.45 (m, 1H), 3.20â3.27 (m, 1H), 3.07 (br s, 2H), 2.70â2.97 (m, 1H), 2.32 (s, 3H), 1.99â2.09 (m, 1H), 1.53â1.67 (m, 2H), 1.38â1.48 (m, 9H), 1.27â1.36 (m, 2H), 0.87â1.06 (m, 2H). Step 3: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(pyrimidin-5-yl)-1,7-naphthyridin-8-yl)amino)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (120 mg, 0.187 mmol) in TFA (2 mL) was stirred at 80 Â°C for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 3-methyl-5-(pyrimidin-5-yl)-8-(((3<i>R</i>,4<i>R</i>)-3-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>41</b>) (20 mg, 41%) as a colorless oil. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.57 min, [M + H]<sup>+</sup> = 451.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 9.23 (s, 1H), 8.86 (s, 2H), 7.76 (s, 1H), 7.59 (d, <i>J</i> = 1.2 Hz, 1H), 6.91â6.98 (m, 1H), 4.14â4.25 (m, 1H), 3.63â3.73 (m, 2H), 3.37â3.43 (m, 1H), 3.08â3.29 (m, 6H), 2.84â2.92 (m, 1H), 2.26â2.36 (m, 1H), 2.10 (d, <i>J</i> = 1.0 Hz, 3H), 1.95â2.05 (m, 1H), 1.58â1.70 (m, 1H), 1.28â1.44 (m, 3H), 0.95â1.09 (m, 2H). Two NH not seen.</div></div><div id="sec4_12_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)quinolin-2(1<i>H</i>)-one (<b>42</b>)</h3><div class="NLM_p last">Step 1: a solution of 2-methyl-3-phenylacryloyl chloride (11 g, 60.9 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (300 mL) at 0 Â°C under nitrogen was treated with NEt<sub>3</sub> (9.34 mL, 67.0 mmol) and then, after 10 min, with 2-bromoaniline (10.5 g, 60.9 mmol). The resulting solution was stirred at this temperature for 4 h, washed successively with water (300 mL), a 5% w/w Na<sub>2</sub>CO<sub>3</sub> aqueous solution (300 mL), and a 1 N HCl aqueous solution (300 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give <i>N</i>-(2-bromophenyl)-2-methyl-3-phenylacrylamide (18.5 g, 96%) as a dark brown oil that crystallized on standing and that was used in the next step without further purification. Step 2: a solution of <i>N</i>-(2-bromophenyl)-2-methyl-3-phenylacrylamide (16.0 g, 50.6 mmol) in chlorobenzene (40 mL) at 0 Â°C was treated with AlCl<sub>3</sub> (40.5 g, 304 mmol), and the resulting mixture was stirred at this temperature for 20 min and then at 120 Â°C for 2 h. The reaction mixture was cooled to room temperature and then added very cautiously to ice (500 g). The mixture was agitated thoroughly to ensure complete hydrolysis of aluminum chloride. The mixture was then extracted with CH<sub>2</sub>Cl<sub>2</sub> (200 mL), and the organic layer was dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (330 g column, 0 to 100% gradient EtOAc in cyclohexane) gave 8-bromo-3-methylquinolin-2(1<i>H</i>)-one (6.9 g, 57%) as a pink solid. LCMS (method formate): <i>t</i><sub>R</sub> 0.82 min, [M + H]<sup>+</sup> = 238.0 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 9.04 (br s, 1H), 7.66 (dd, <i>J</i> = 7.8, 1.1 Hz, 1H), 7.58 (d, <i>J</i> = 1.2 Hz, 1H), 7.47 (dd, <i>J</i> = 1.0, 7.8 Hz, 1H), 7.09 (t, <i>J</i> = 7.8 Hz, 1H), 2.29 (d, <i>J</i> = 1.2 Hz, 3H). Step 3: a suspension of 8-bromo-3-methylquinolin-2(1<i>H</i>)-one (1.0 g, 4.2 mmol) DMF (5 mL) at room temperature was treated with K<sub>2</sub>CO<sub>3</sub> (0.71 g, 5.1 mmol), and the resulting mixture was stirred at this temperature for 10 min before (bromomethyl)benzene (0.55 mL, 4.6 mmol) was added. The resulting mixture was then stirred for 72 h at room temperature. The precipitate was filtered off, the mother liquors were diluted with water (50 mL) and EtOAc (50 mL), and the layers were separated. The organic phase was washed with brine (3 Ã 50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, 0 to 100% gradient EtOAc in cyclohexane) gave 2-(benzyloxy)-8-bromo-3-methylquinoline (1.17 g, 85%) as a pink solid and 1-benzyl-8-bromo-3-methylquinolin-2(1<i>H</i>)-one (40 mg, 3%) as a milky white gum. LCMS (method formate): <i>t</i><sub>R</sub> 1.57 min, [M + H]<sup>+</sup> = 328.0 (1 Br).<sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.72 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.55 (d, <i>J</i> = 1.2 Hz, 1H), 7.47 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.18â7.31 (m, 3H), 7.10 (m, 2H), 7.04 (dd, <i>J</i> = 7.7, 7.7 Hz, 1H), 6.06 (s, 2H), 2.29 (d, <i>J</i> = 1.2 Hz, 3H). Step 4: a mixture of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (1.21 g, 3.66 mmol), 2-(benzyloxy)-8-bromo-3-methylquinoline (1.0 g, 3.0 mmol), sodium <i>tert</i>-butoxide (0.878 g, 9.14 mmol), BrettPhos (0.164 g, 0.305 mmol), and BrettPhos precatalyst (0.243 g, 0.305 mmol) in THF (2 mL) was stirred at room temerature for 1 h, stirred at 60 Â°C for 3 h, cooled to room temperature, and diluted with a saturated NH<sub>4</sub>Cl aqueous solution. The aqueous phase was extracted with EtOAc (2 Ã 50 mLl), and the combined organics were dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0â25% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (1.70 g, 97%) as a pale yellow foam. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.90 min, [M + H]<sup>+</sup> = 578. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.76 (d, <i>J</i> = 0.7 Hz, 1H), 7.49 (s, 2H), 7.39 (s, 2H), 7.30â7.35 (m, 1H), 7.17â7.23 (m, 1H), 6.93â6.99 (m, 1H), 6.72â6.77 (m, 1H), 5.62â5.68 (m, 1H), 5.55 (d, <i>J</i> = 8.8 Hz, 2H), 3.72â3.93 (m, 2H), 3.59â3.67 (m, 1H), 3.48â3.57 (m, 1H), 3.16â3.27 (m, 1H), 3.05â3.14 (m, 1H), 2.40 (s, 3H), 1.60â1.76 (m, 2H), 1.52 (s, 9H), 0.84 (s, 9H), â0.09 (s, 6H). Step 5: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-((<i>tert</i>-butyldimethylsilyl)oxy)piperidine-1-carboxylate (1.60 g, 2.77 mmol) in THF (50 mL) at room temperature was treated with TBAF (1 N in THF, 5.54 mL, 5.54 mmol), and the resulting mixture was stirred at this temperature for 2 h and then concentrated <i>in vacuo</i>. The residue was partitioned between EtOAc (100 mL) and water (100 mL), and the layers were separated. The organic phase was dried under MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient 0â50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (0.91g, 71%) as a colorless oil. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.47 min, [M + H]<sup>+</sup> = 464. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 7.77 (d, <i>J</i> = 0.7 Hz, 1H), 7.50 (d, <i>J</i> = 7.3 Hz, 2H), 7.36â7.43 (m, 2H), 7.33 (d, <i>J</i> = 7.3 Hz, 1H), 7.19â7.25 (m, 1H), 7.01â7.07 (m, 1H), 6.84 (d, <i>J</i> = 7.3 Hz, 1H), 5.50â5.59 (m, 2H), 5.42â5.49 (m, 1H), 4.28â4.39 (m, 1H), 3.92â4.22 (m, 1H), 3.58â3.69 (m, 1H), 3.41â3.53 (m, 1H), 2.83â3.02 (m, 2H), 2.59â2.73 (m, 1H), 2.41 (d, <i>J</i> = 1.0 Hz, 3H), 2.03â2.13 (m, 1H), 1.52 (s, 9H), 1.30â1.43 (m, 1H). Step 6: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (420 mg, 0.906 mmol) in THF (10 mL) at 0 Â°C was treated with potassium <i>tert</i>-butoxide (224 mg, 1.99 mmol). The resulting mixture was stirred at this temperature for 10 min, treated with (1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl trifluoromethanesulfonate (537 mg, 1.81 mmol), and allowed to warm to room temperature over 2 h before being diluted with water (10 mL). The aqueous phase was extracted with EtOAc (10 mL). The organic layer was washed with water (10 mL), dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (0.47 g, 85%) as a pale green oil. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.51 min, [M + H]<sup>+</sup> = 610.5. Step 7: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (0.47 g, 0.77 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) at â10 Â°C was treated with NBS (0.123 g, 0.694 mmol), and the resulting mixture was stirred at this temperature for 20 min before being treated with a saturated sodium metabisulphite aqueous solution (20 mL). The biphasic mixture was stirred at room temperature for 10 min, and the layers were separated. The organic phase was dried using a phase separator and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methylquinolin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (370 mg, 70%) as a pale yellow gum. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.55 min, [M + H]<sup>+</sup> = 688/690. Step 8: a 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (53.7 mg, 0.392 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methylquinolin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (180 mg, 0.261 mmol), K<sub>2</sub>CO<sub>3</sub> (144 mg, 1.04 mmol), Pd(OAc)<sub>2</sub> (5.9 mg, 0.026 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (9.4 mg, 0.026 mmol) and then filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was degassed under vacuum over 20 min, quenched with nitrogen, and stirred at 100 Â°C for 30 min under microwave irradiation before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a hydrophobic frit and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (10 g column, gradient: 0 to 100% EtOAc in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)quinolin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (125 mg, 68%) as a pale yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.48 min, [M + H]<sup>+</sup> = 701.6. Step 9: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)quinolin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (125 mg, 0.178 mmol) in TFA (1 mL) was refluxed for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. The residue was dissolved in MeOH and loaded onto a 5 g SCX-2 cartridge, which was washed with MeOH (20 mL) and eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)quinolin-2(1<i>H</i>)-one (<b>42</b>) (90 mg, 99%) as a yellow solid. LCMS (high-pH method): <i>t</i><sub>R</sub> 0.68 min, [M + H]<sup>+</sup> = 511, <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 11.70 (br s, 1H), 8.51 (d, <i>J</i> = 1.5 Hz, 1H), 8.46 (d, <i>J</i> = 2.0 Hz, 1H), 7.65 (d, <i>J</i> = 1.2 Hz, 1H), 7.52 (s, 1H), 6.99â7.06 (m, 2H), 4.24 (br s, 1H), 3.58â3.65 (m, 1H), 3.36â3.56 (m, 4H), 3.20 (br s, 1H), 2.94â3.14 (m, 5H), 2.57â2.66 (m, 1H), 2.55â2.57 (m, 1H), 2.52 (dd, <i>J</i> = 11.5, 9.5 Hz, 1H), 2.46 (s, 3H), 2.07â2.31 (m, 4H), 1.75â1.90 (m, 3H), 1.53â1.62 (m, 2H).</div></div><div id="sec4_12_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)quinolin-2(1<i>H</i>)-one (<b>43</b>)</h3><div class="NLM_p last">This compound was obtained in an anlogous manner as that for compound <b>42</b> following the separation of the two enantiomers of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (step 5, compound <b>42</b>), which was performed as follows: The racemic mixture was separated by chiral HLPC. The HPLC analysis was carried out on a Chiralpak IC (4.6 mm i.d. Ã 25 cm), using 25% EtOH in heptane at a flow rate of 1 mL/min. The material (910 mg) was dissolved in a mixture of EtOH/heptane (2:1) (1 mL). The purification was carried out on a Chiralpak IC (30 mm Ã 25 cm), using 25% EtOH in heptane at a flow rate of 30 mL/min, and 1 mL of solution was injected at a time. The appropriate fractions were combined and concentrated <i>in vacuo</i> to give the fastest running enantiomer, (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (422 mg), and the slowest running enantiomer, (3<i>S</i>,4<i>S</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methylquinolin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (423 mg).</div></div><div id="sec4_12_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 8-(((3<i>R</i>,4<i>R</i>)-3-(((2<i>R</i>,4<i>R</i>,6<i>S</i>)-2,6-Dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>44</b>)</h3><div class="NLM_p last">Step 1: a solution of 4-(hydroxymethyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (1.0 g, 6.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (30 mL) at room temperature was treated with imidazole (0.497 g, 7.31 mmol) and DMAP (0.074 g, 0.61 mmol). The mixture was stirred at this temperature for 20 min; then, TBDMSCl (1.10 g, 7.31 mmol) was added, and the resulting suspension was stirred for 2 h. The organic phase was then washed with water (30 mL), dried using a phase separator, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0 to 50% EtOAc in cyclohexane) gave 4-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (1.62 g, 96%) as a colorless oil. Step 2: a solution of 4-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (1.4 g, 5.0 mmol) in THF (10 mL) at â78 Â°C under nitrogen was treated with LiHMDS (1 N in THF, 6.54 mL, 6.54 mmol). The resulting mixture was stirred at this temperature for 30 min before being treated with MeI (0.503 mL, 8.04 mmol). The mixture was stirred at â78 Â°C for 2 h and was then allowed to warm to room temperature. The solution was diluted with water (30 mL), and the aqueous phase was extracted with EtOAc (20 mL). The organic phase was dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i> to give (2<i>R</i>,6<i>S</i>)-4-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)-2,6-dimethyltetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (contaminated with a mixture of monomethylated adducts) (1.45 g, 95%) as a pale yellow oil, which was used in the next step without further purification. Step 3: a solution of (2<i>R</i>,6<i>S</i>)-4-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)-2,6-dimethyltetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (0.50 g, 1.7 mmol) in THF (10 mL) at room temperature was treated with a 2 N HCl aqueous solution (10 mL), and the resulting mixture was stirred at this temperature for 18 h and then concentrated <i>in vacuo</i> to give (2<i>R</i>,6<i>S</i>)-4-(hydroxymethyl)-2,6-dimethyltetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (0.186 g, 61%) as a colorless solid (contaminated with a mixture of monomethylated adducts), which was used in the next step without further purification. Step 4: a solution of (2<i>R</i>,6<i>S</i>)-4-(hydroxymethyl)-2,6-dimethyltetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (0.186 g, 1.04 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) under nitrogen was cooled using an acetone/ice bath (â10 Â°C) then was treated with pyridine (0.093 mL, 1.15 mmol) and Tf<sub>2</sub>O (0.194 mL, 1.15 mmol) dropwise. The resulting mixture was stirred at this temperature for 1 h and then allowed to warm to room temperature. The organic phase was washed with brine (10 mL), dried using a phase separator, and concentrated <i>in vacuo</i> to give ((2<i>R</i>,6<i>S</i>)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl trifluoromethanesulfonate (0.32 g, 99%) as a pale yellow oil (contaminated with a mixture of monomethylated adducts), which was used in the next step without further purification. Step 5: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-hydroxypiperidine-1-carboxylate (0.30 g, 0.65 mmol) in THF (10 mL) at 0 Â°C under nitrogen was treated with tBuOK (0.152 g, 1.36 mmol), and the resulting mixture was stirred at this temperature for 10 min before being treated with a solution of ((2<i>R</i>,6<i>S</i>)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methyl trifluoromethanesulfonate (0.301 g, 0.969 mmol) in THF (10 mL). The resulting mixture was stirred at 0 Â°C for 1 h and then at room temperature for 30 min before being quenched with water. The aqueous phase was extracted with EtOAc. The organic phase was washed with water, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient 0 to 70% EtOAc in hexanes) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-(((2<i>R</i>,6<i>S</i>)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (0.19 g, 47%) as a pale yellow gum contaminated with a mixture of monomethylated adducts. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.43 min, [M + H]<sup>+</sup> = 639.5. Step 6: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-(((2<i>R</i>,6<i>S</i>)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (0.19 g, 0.30 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) at â10 Â°C was treated with NBS (0.049 g, 0.27 mmol), and the resulting mixture was stirred at this temperature for 40 min and then treated with a saturated sodium metabisulphite aqueous solution. The biphasic mixture was allowed to warm to room temperature and stirred for 10 min. The layers were separated, and the organic phase was dried using a phase separator and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-(((2<i>R</i>,6<i>S</i>)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (180 mg, 84%) as a pale yellow gum (contaminated with a mixture of monomethylated adducts), which was used in the next step without further purification. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.57 min, [M + H]<sup>+</sup> = 719.4 (1 Br). Step 7: a 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (0.070 g, 0.51 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-((2-methyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (0.18 g, 0.26 mmol), K<sub>2</sub>CO<sub>3</sub> (0.141 g, 1.02 mmol), Pd(OAc)<sub>2</sub> (5.74 mg, 0.026 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (9.2 mg, 0.026 mmol) and then filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was degassed under vacuum for 20 min and quenched with nitrogen before being stirred at 100 Â°C for 30 min under microwave irradiation.. The mixture was then cooled to room temperature and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-(((2<i>R</i>,6<i>S</i>)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (140 mg, 76%) contaminated with a mixture of monomethylated adducts. The mixture was further purified by chiral HLPC. The HPLC analysis was carried out on a Chiralpak IB (4.6 mm i.d. Ã 25 cm), using 10% EtOH (+0.2% isopropylamine) in heptane at a flow rate of 1 mL/min. The material (140 mg) was dissolved in a mixture of EtOH (100 mg in mL). The purification was carried out on a Chiralpak IB (2 cm Ã 25 cm), using 10% EtOH (+0.2% isopropylamine) in heptane at a flow rate of 20 mL/min, and 1 mL of solution was injected. The appropriate fractions were combined and concentrated under reduced pressure to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-(((2R,4R,6S)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (80 mg, 43%). LCMS (high-pH method): <i>t</i><sub>R</sub> 1.43 min, [M + H]<sup>+</sup> = 730.6. Step 8: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-(((2R,4r,6S)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (70 mg, 0.096 mmol) in TFA (2 mL) was refluxed for 2 h and then cooled to room temperature and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-(((2R,4r,6S)-2,6-dimethyl-1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>44</b>) (20 mg, 39%) as a bright yellow solid. LCMS (method TFA): <i>t</i><sub>R</sub> 0.47 min, [M + H]<sup>+</sup> = 540. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 8.53 (d, <i>J</i> = 1.5 Hz, 1H), 8.48 (d, <i>J</i> = 1.7 Hz, 1H), 7.90â7.94 (m, 1H), 7.75 (d, <i>J</i> = 1.0 Hz, 1H), 7.54 (s, 1H), 6.75 (d, <i>J</i> = 6.4 Hz, 1H), 4.52 (br s, 1H), 3.46â3.59 (m, 2H), 3.32â3.45 (m, 2H), 3.09â3.17 (m, 1H), 2.77â2.92 (m, 3H), 2.63â2.71 (m, 1H), 2.47 (s, 3H), 2.34 (d, <i>J</i> = 0.7 Hz, 3H), 2.19â2.27 (m, 1H), 1.84â1.94 (m, 3H), 1.60â1.76 (m, 2H), 1.30â1.54 (m, 2H), 1.23 (d, <i>J</i> = 6.8 Hz, 3H), 1.15 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_12_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-ethyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>45</b>)</h3><div class="NLM_p last">Step 1: (<i>E</i>)-ethyl 2-((3-((<i>tert</i>-butoxycarbonyl)amino)-2-chloropyridin-4-yl)methylene)butanoate was obtained from <i>tert</i>-butyl (2-chloro-4-formylpyridin-3-yl)carbamate in an anaogous manner as that for (<i>E</i>)-ethyl 3-(3-((<i>tert</i>-butoxycarbonyl)amino)-2-chloropyridin-4-yl)-2-methyl acrylate (see synthesis of <b>7a</b> in preceding article<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>) using ethyl 2-(diethoxyphosphoryl)butanoate (15 g, 57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 8.33 (d, <i>J</i> = 4.8 Hz, 1H), 7.39 (s, 1H), 7.35 (d, <i>J</i> = 4.8 Hz, 1H), 4.21 (q, <i>J</i> = 7.2 Hz, 2H), 2.27 (d, <i>J</i> = 7.5 Hz, 2H), 1.39 (br s, 7H), 1.25 (t, <i>J</i> = 7.1 Hz, 3H), 1.01 (t, <i>J</i> = 7.3 Hz, 3H). Step 2: (<i>E</i>)-ethyl 2-((3-amino-2-chloropyridin-4-yl)methylene)butanoate was obtained from (<i>E</i>)-ethyl 2-((3-((<i>tert</i>-butoxycarbonyl)amino)-2-chloropyridin-4-yl)methylene)butanoate using the same procedure as that for the synthesis of (<i>E</i>)-ethyl 3-(3-amino-2-chloropyridin-4-yl)-2-methyl acrylate from ethyl 3-(3-((<i>tert</i>-butoxycarbonyl)amino)-2-chloropyridin-4-yl)-2-methyl acrylate (see synthesis of <b>7a</b> in preceding article<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>) (8.1 g, 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 7.63 (d, <i>J</i> = 1.0 Hz, 1H), 7.34 (s, 1H), 6.98 (d, <i>J</i> = 1.0 Hz, 1H), 5.50 (s, 2H), 4.19â4.25 (m, 2H), 2.26â2.34 (m, 2H), 1.29 (t, <i>J</i> = 1.0 Hz, 3H), 1.03 (t, <i>J</i> = 1.0 Hz, 3H). Step 3: 8-chloro-3-ethyl-1,7-naphthyridin-2(1<i>H</i>)-one was obtained from (<i>E</i>)-ethyl 2-((3-amino-2-chloropyridin-4-yl)methylene)butanoate using the same procedure as that for the synthesis of 8-chloro-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>7a</b>) from (<i>E</i>)-ethyl 3-(3-amino-2-chloropyridin-4-yl)-2-methyl acrylate (5 g, 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 11.35 (s, 1H), 8.13 (d, <i>J</i> = 5.0 Hz, 1H), 7.83 (s, 1H), 7.64 (d, <i>J</i> = 5.0 Hz, 1H), 2.53â2.60 (m, 2H), 1.19 (t, <i>J</i> = 7.5 Hz, 3H). Step 4: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-amino-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (249 mg, 0.686 mmol) in THF (4 mL) was treated with 2-(benzyloxy)-8-chloro-3-ethyl-1,7-naphthyridine (248 mg, 0.830 mmol), sodium <i>tert</i>-butoxide (203 mg, 2.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (34 mg, 0.037 mmol), and BrettPhos (39 mg, 0.074 mmol). The resulting mixture was stirred at room temperature for 1 h and was then refluxed for 3 h before being cooled to room temperature and concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 15 to 70% EtOAc in cyclohexane) gave a first residue, which was further purified by flash chromatography on silica gel (50 g column, gradient: 15 to 70% EtOAc in cyclohexane) to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-ethyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (69.1 mg, 16%) as a yellow oil. LCMS (method formic): <i>t</i><sub>R</sub> 1.04 min, [M + H]<sup>+</sup> = 625.6. Step 5: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-ethyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (6.1 mL, 0.096 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was cooled to â10 Â°C using an NaCl ice bath and then treated with NBS (10.6 mg, 0.060 mmol), and the resulting mixture was stirred at this temperature for 30 min. NBS (3.6 mg, 0.020 mmol) was further added, and the mixture was stirred at the same temperature for 30 min before being treated with a saturated sodium metabisulfite aqueous solution (5 mL). The reaction mixture was stirred for 20 min and then warmed to room temperature. The layers were separated, and the organic phase was dried using a hydrophobic frit and concentrated <i>in vacuo</i> to give (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-ethyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (59.4 mg, 88%) as a yellow oil, which was used in the next step without further purification. LCMS (method formic): <i>t</i><sub>R</sub> 1.46 min, [M + H]<sup>+</sup> = 705.6 (1 Br). Step 6: a 5 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (15 mg, 0.11 mmol), (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-ethyl-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (59 mg, 0.084 mmol), Pd(OAc)<sub>2</sub> (2.0 mg, 8.9 Î¼mol), K<sub>2</sub>CO<sub>3</sub> (51 mg, 0.37 mmol), and di((3S,5S,7S)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (3.0 mg, 8.4 Î¼mol) and then filled with 1,4-dioxane (2 mL) and water (1 mL), and the resulting mixture was degassed by bubbling nitrogen through it for 30 min. The mixture was then stirred under microwave irradiation at 100 Â°C for 30 min before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (8 mL) and water (8 mL), and the layers were separated. The organic phase was dried using a hydrophobic frit and then concentrated <i>in vacuo</i>. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 12 to 63% (3:1 EtOAc/EtOH) in cyclohexane) gave (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-ethyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (25.6 mg, 42%) as a yellow oil. LCMS (high-pH method): <i>t</i><sub>R</sub> 1.45 min, [M + H]<sup>+</sup> = 717.7. Step 7: a solution of (3<i>R</i>,4<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-ethyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidine-1-carboxylate (25 mg, 0.036 mmol) in TFA (1 mL) was refluxed at 80 Â°C for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. The residue was again dissolved in TFA (1 mL), and the mixture was refluxed for another 3 h before being cooled to room temperature and concentrated <i>in vacuo</i>. The residue was loaded on to a 500 mg SCX-2 cartridge, which was washed with MeOH (5 mL) and then eluted with a 2 N NH<sub>3</sub> solution in MeOH. The ammoniac fractions were combined and concentrated <i>in vacuo</i> to give 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-ethyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>45</b>) (12.1 mg, 64%) as a yellow solid. LCMS (method formic): <i>t</i><sub>R</sub> 0.51 min, [M + H]<sup>+</sup> = 526. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) Î´ ppm 8.46 (d, <i>J</i> = 1.5 Hz, 1H), 8.41 (d, <i>J</i> = 1.7 Hz, 1H), 7.75â7.79 (m, 2H), 7.61 (s, 1H), 4.30â4.38 (m, 1H), 3.60 (dd, <i>J</i> = 9.2, 5.5 Hz, 1H), 3.38â3.46 (m, 2H), 3.05â3.12 (m, 1H), 2.82â3.01 (m, 4H), 2.72â2.81 (m, 1H), 2.64 (q, <i>J</i> = 7.8 Hz, 2H), 2.54â2.60 (m, 1H), 2.47 (s, 4H), 2.16â2.25 (m, 1H), 1.93â2.02 (m, 2H), 1.75 (ddd, <i>J</i> = 8.4, 5.6, 2.9 Hz, 1H), 1.58â1.70 (m, 3H), 1.19 (t, <i>J</i> = 7.5 Hz, 3H). Three NH not seen.</div></div><div id="sec4_12_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)-1-methylpiperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>46</b>)</h3><div class="NLM_p last">A solution of 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (110 mg, 0.215 mmol) in CHCl<sub>3</sub> was treated with formic acid (11 Î¼L, 0.28 mmol) and with a formaldehyde solution in water, containing 10â15% MeOH as stabilizer (37% w/w, 19 Î¼L, 0.26 mmol). The resulting mixture was stirred under microwave irradiation at 80 Â°C for 2 h, cooled to room temperature, and concentrated <i>in vacuo</i>. Purification of the residue by MDAP (high-pH method) gave 8-(((3<i>R</i>,4<i>R</i>)-3-((1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)methoxy)-1-methylpiperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>46</b>) (67 mg, 59%) as a yellow solid. LCMS (method TFA): <i>t</i><sub>R</sub> 0.77 min, [M + H]<sup>+</sup> = 526. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ ppm 13.2 (br s, 1H), 8.52 (d, <i>J</i> = 1.5 Hz, 1H), 8.46 (d, <i>J</i> = 1.7 Hz, 1H), 7.90 (s, 1H), 7.73 (d, <i>J</i> = 1.2 Hz, 1H), 7.54 (d, <i>J</i> = 0.7 Hz, 1H), 6.64 (d, <i>J</i> = 7.6 Hz, 1H), 4.32â4.43 (m, 1H), 3.70 (td, <i>J</i> = 9.1, 4.3 Hz, 1H), 3.55â3.61 (m, 1H), 3.48â3.54 (m, 1H), 3.17 (dd, <i>J</i> = 10.9, 3.3 Hz, 1H), 2.75â2.98 (m, 6H), 2.47 (s, 3H), 2.38 (s, 3H), 2.34 (d, <i>J</i> = 1.0 Hz, 3H), 2.21â2.32 (m, 3H), 2.16 (t, <i>J</i> = 10.1 Hz, 1H), 1.98â2.11 (m, 2H), 1.66â1.86 (m, 2H).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65"><a href="/doi/suppl/10.1021/acs.jmedchem.5b00773">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80217" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80217" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Table S1, parts 1 and 2: Full data for exemplified compounds in main tables. Table S2: Selectivity profile of compound <b>46</b> in the BromoSCAN panel. Table S3: Data collection and refinement statistics for ATAD2 X-ray structures. Table S4: Data collection and refinement statistics for BRD4 BD1 structure. Figure S1: Correlation plots between ATAD2 assays. Figure S2: KAc site amino acids of ATAD2, ATAD2B, BRD4 BD1, and BRD4 BD2 bromodomains. Figure S3: OMIT maps for ATAD2 and BRD4 BD1 X-ray structures. Figure S4: GSK3190320, the synthetic ligand used in the ATAD2 TR-FRET assay. The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b00773" class="ext-link">10.1021/acs.jmedchem.5b00773</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00773/suppl_file/jm5b00773_si_001.pdf">jm5b00773_si_001.pdf (983.65 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00773/suppl_file/jm5b00773_si_002.csv">jm5b00773_si_002.csv (2.58 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b00773" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f7a796829bd9b6d9b5969a9598859882909fb790849cd994989a"><span class="__cf_email__" data-cfemail="b1e1d0c4dd9ff09ff3d0dcd3dec3dec4d6d9f1d6c2da9fd2dedc">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H. Demont</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#94d1f9f9f5fae1f1f8badcbad0f1f9fbfae0d4f3e7ffbaf7fbf9"><span class="__cf_email__" data-cfemail="2e6b43434f405b4b420066006a4b4341405a6e495d45004d4143">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebecca C. Furze</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Michon</span> - <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Sheppard</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heather Barnett</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hawa Diallo</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David P. Dixon</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clement Douault</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma J. Jones</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bhumika Karamshi</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christina Rau</span> - <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">â Molecular Discovery Research, â¡Epinova Discovery
Performance Unit, and Â§Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>;Â 
    <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thilo Werner</span> - <span class="hlFld-Affiliation affiliation">Cellzome GmbH, Molecular Discovery Research, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are current or previous employees of GSK.<br /></br></div></li></ul></div><div class="ack" id="ACK-d4557e9905-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42515" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42515" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Thanks are extended to Tony Cooper (array design and chemistry) and Antonia Lewis (PBMC assay).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i68" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i68"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i69" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i69"> Abbreviations Used</h2><tr><td class="NLM_term">ANCCA</td><td class="NLM_def"><p class="first last">AAA nuclear coregulator cancer-associated protein</p></td></tr><tr><td class="NLM_term">ATAD2</td><td class="NLM_def"><p class="first last">ATPase family AAA domain containing 2</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra terminal domain</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">BRPF1/2/3</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger containing 1/2/3</p></td></tr><tr><td class="NLM_term">CECR2</td><td class="NLM_def"><p class="first last">cat eye syndrome chromosome region, candidate 2</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetyl lysine. pIC<sub>50</sub>, âlog<sub>10</sub> IC<sub>50</sub></p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TAF</td><td class="NLM_def"><p class="first last">TATA box binding protein (TBP)-associated factor</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80877" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80877" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 31 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Caron, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lestrat, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escoffier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Virolle, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbry, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debernardi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousseaux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khochbin, S.</span><span> </span><span class="NLM_article-title">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">5171</span><span class="NLM_x">â</span> <span class="NLM_lpage">5181</span><span class="refDoi">Â DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+characterization+of+ATAD2+as+a+new+cancer%2Ftestis+factor+and+a+predictor+of+poor+prognosis+in+breast+and+lung+cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0liFkPhVAwZQEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520characterization%2520of%2520ATAD2%2520as%2520a%2520new%2520cancer%252Ftestis%2520factor%2520and%2520a%2520predictor%2520of%2520poor%2520prognosis%2520in%2520breast%2520and%2520lung%2520cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Boussouar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamshidikia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morozumi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousseaux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khochbin, S.</span><span> </span><span class="NLM_article-title">Malignant genome reprogramming by ATAD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1829</span><span class="NLM_x">, </span> <span class="NLM_fpage">1010</span><span class="NLM_x">â</span> <span class="NLM_lpage">1014</span><span class="refDoi">Â DOI: 10.1016/j.bbagrm.2013.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1016%2Fj.bbagrm.2013.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=23831842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=1010-1014&author=F.+Boussouarauthor=M.+Jamshidikiaauthor=Y.+Morozumiauthor=S.+Rousseauxauthor=S.+Khochbin&title=Malignant+genome+reprogramming+by+ATAD2&doi=10.1016%2Fj.bbagrm.2013.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Malignant genome reprogramming by ATAD2</span></div><div class="casAuthors">Boussouar, Faycal; Jamshidikia, Mahya; Morozumi, Yuichi; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1829</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Unscheduled expression of crit. cellular regulators could be central to malignant genome reprogramming and tumor establishment.  One such factor appears to be ATAD2, a remarkably conserved protein normally predominantly expressed in germ cells but almost systematically over-expressed in a variety of unrelated cancers.  The presence of a bromodomain adjacent to an AAA type ATPase domain, points to ATAD2 as a factor preliminarily acting on chromatin structure and function.  Accordingly, ATAD2 has been shown to cooperate with a series of transcription factors and chromatin modifiers to regulate specific set of genes.  Here we discuss our knowledge on ATAD2 to evaluate its role as a cancer driver and its value as a new anti-cancer target.  Upon its activation, ATAD2 through its interaction with defined transcription factors, initiates a loop of transcriptional stimulation of target genes, including ATAD2 itself, leading to enhanced cell proliferation and resistance to apoptosis in an ATAD2-dependent manner.  Approaches aiming at neutralizing ATAD2 activity in cancer, including the use of small mol. inhibitors of its two "druggable" domains, AAA ATPase and bromodomain, could become part of a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp824JMo6Dp4rVg90H21EOLACvtfcHk0liFkPhVAwZQEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF&md5=cba5991a2dc5f96f522e1104ae4e4889</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2013.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2013.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DMalignant%2520genome%2520reprogramming%2520by%2520ATAD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2013%26volume%3D1829%26spage%3D1010%26epage%3D1014%26doi%3D10.1016%2Fj.bbagrm.2013.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Zou, J. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revenko, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tepper, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gemo, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H. W.</span><span> </span><span class="NLM_article-title">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">3339</span><span class="NLM_x">â</span> <span class="NLM_lpage">3346</span><span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-08-3440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1158%2F0008-5472.CAN-08-3440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=19318566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=3339-3346&author=J.+X.+Zouauthor=L.+Guoauthor=A.+S.+Revenkoauthor=C.+G.+Tepperauthor=A.+T.+Gemoauthor=H.+J.+Kungauthor=H.+W.+Chen&title=Androgen-induced+coactivator+ANCCA+mediates+specific+androgen+receptor+signaling+in+prostate+cancer&doi=10.1158%2F0008-5472.CAN-08-3440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span></div><div class="casAuthors">Zou, June X.; Guo, Linlang; Revenko, Alexey S.; Tepper, Clifford G.; Gemo, Abigael T.; Kung, Hsing-Jien; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3339-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Androgen receptor (AR) plays a pivotal role in prostate cancer, primarily by regulating different gene expression programs elicited by androgen, which is important for cancer cell proliferation, survival, and differentiation.  It is believed that the transcriptional function of AR is mediated largely by distinct nuclear coregulators.  We report here the identification of ANCCA (also known as ATAD2), a new member of the AAA+ ATPase family proteins, as a novel AR coactivator.  ANCCA interacts directly with AR and enhances its transcriptional activity, and is required for androgen-stimulated expression of a specific subgroup of genes including IGF1R, IRS-2, SGK1, and survivin.  Upon androgen stimulation, ANCCA together with AR is recruited to the specific AR target genes.  Suppression of ANCCA expression strongly inhibited the proliferation of androgen-responsive or androgen-independent, AR-pos. prostate cancer cells and caused a significant increase of cellular apoptosis.  Strikingly, the ANCCA gene itself, located at chromosome 8q24, is highly induced by androgen in androgen-dependent prostate cancer cells and xenograft tumors.  Although ANCCA is hardly detected in normal human prostate tissue, high levels of ANCCA are found in hormone-independent prostate cancer cell lines, xenograft tumor, and a subset of prostate cancers with high Gleason scores.  Together, these findings suggest that ANCCA plays an important role in prostate cancer by mediating specific AR functions in cancer cell survival and proliferation.  The possession of ATPase and bromodomain by ANCCA makes it an attractive target for the development of therapeutics for the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtieC3cr_jX7Vg90H21EOLACvtfcHk0ljlTFhdmxBp3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D&md5=daa42208c9c7019809bb491e5af1f152</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3440%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DAndrogen-induced%2520coactivator%2520ANCCA%2520mediates%2520specific%2520androgen%2520receptor%2520signaling%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D3339%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-08-3440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J.</span><span> </span><span class="NLM_article-title">Targeting epigenetic readers in cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">367</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">â</span> <span class="NLM_lpage">657</span><span class="refDoi">Â DOI: 10.1056/NEJMra1112635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1056%2FNEJMra1112635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=22894577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=647-657&author=M.+A.+Dawsonauthor=T.+Kouzaridesauthor=B.+J.+Huntly&title=Targeting+epigenetic+readers+in+cancer&doi=10.1056%2FNEJMra1112635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic readers in cancer</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">647-657</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on assocn. between epigenetics and chromatin biol., BET bromodomain inhibitors, and role of BET inhibitors in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnz_YQy6Z2T7Vg90H21EOLACvtfcHk0ljlTFhdmxBp3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F&md5=f5e1e82c8a1105f798b1f7719138721c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1112635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1112635%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26atitle%3DTargeting%2520epigenetic%2520readers%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D647%26epage%3D657%26doi%3D10.1056%2FNEJMra1112635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Jennings, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Measures, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Phenotypic screening and fragment-based approaches to the discovery of small-molecule bromodomain ligands</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">179</span><span class="NLM_x">â</span> <span class="NLM_lpage">204</span><span class="refDoi">Â DOI: 10.4155/fmc.13.197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.4155%2Ffmc.13.197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=24467243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCkuro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=179-204&author=L.+E.+Jenningsauthor=A.+R.+Measuresauthor=B.+G.+Wilsonauthor=S.+J.+Conway&title=Phenotypic+screening+and+fragment-based+approaches+to+the+discovery+of+small-molecule+bromodomain+ligands&doi=10.4155%2Ffmc.13.197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening and fragment-based approaches to the discovery of small-molecule bromodomain ligands</span></div><div class="casAuthors">Jennings, Laura E.; Measures, Angelina R.; Wilson, Brian G.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-204</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues and hence facilitate protein-protein interactions.  These bromodomain-mediated interactions often play key roles in transcriptional regulation and their dysfunction is implicated in a large no. of diseases.  The discovery of potent and selective small-mol. bromodomain and extra C-terminal domain bromodomain ligands, which show promising results for the treatment of cancers and atherosclerosis, has promoted intense interest in this area.  Here we describe the progress that has been made to date in the discovery of small-mol. bromodomain ligands, with particular emphasis on the roles played by phenotypic screening and fragment-based approaches.  In considering the future of the field we discuss the prospects for development of mol. probes and drugs for the non-bromodomain and extra C-terminal domain bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6WhUfE_Ad4rVg90H21EOLACvtfcHk0ljlTFhdmxBp3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCkuro%253D&md5=08c3266ceefe824fde8ef7c400d6bc22</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.197%26sid%3Dliteratum%253Aachs%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DMeasures%26aufirst%3DA.%2BR.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DPhenotypic%2520screening%2520and%2520fragment-based%2520approaches%2520to%2520the%2520discovery%2520of%2520small-molecule%2520bromodomain%2520ligands%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D179%26epage%3D204%26doi%3D10.4155%2Ffmc.13.197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Demont, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furze, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wellaway, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bridges, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craggs, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douault, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmons, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karamshi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of low-micromolar ATAD2 bromodomain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5649</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649&author=E.+H.+Demontauthor=C.+Chungauthor=R.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+Emmonsauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-based+discovery+of+low-micromolar+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0lhWSZlQXEKYLQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DFurze%26aufirst%3DR.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520low-micromolar%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5649%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Discovery of BET bromodomain inhibitors and their role in target validation</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">â</span> <span class="NLM_lpage">296</span><span class="refDoi">Â DOI: 10.1039/c3md00291h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1039%2Fc3md00291h" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=288-296&author=S.+Mullerauthor=S.+Knapp&title=Discovery+of+BET+bromodomain+inhibitors+and+their+role+in+target+validation&doi=10.1039%2Fc3md00291h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2Fc3md00291h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00291h%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520BET%2520bromodomain%2520inhibitors%2520and%2520their%2520role%2520in%2520target%2520validation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D288%26epage%3D296%26doi%3D10.1039%2Fc3md00291h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Chung, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosmini, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delves, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magny, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boursier, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouillot, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellibert, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riou, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uings, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clement, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boullay, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blandel, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3827</span><span class="NLM_x">â</span> <span class="NLM_lpage">3838</span><span class="refDoi">Â DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.+M.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Clementauthor=A.+B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lhWSZlQXEKYLQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Gosmini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krysa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riou-Eymard, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boursier, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trottet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demont, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaur, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schilling, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">The discovery of I-BET726 (GSK1324726A), a potent tetrahydroquinoline ApoA1 up-regulator and selective BET bromodomain inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8111</span><span class="NLM_x">â</span> <span class="NLM_lpage">8131</span><span class="refDoi">Â DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.+M.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.+W.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=The+discovery+of+I-BET726+%28GSK1324726A%29%2C+a+potent+tetrahydroquinoline+ApoA1+up-regulator+and+selective+BET+bromodomain+inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0ljowkBcl1VbzA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DThe%2520discovery%2520of%2520I-BET726%2520%2528GSK1324726A%2529%252C%2520a%2520potent%2520tetrahydroquinoline%2520ApoA1%2520up-regulator%2520and%2520selective%2520BET%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">â</span> <span class="NLM_lpage">1123</span><span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0ljowkBcl1VbzA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">â</span> <span class="NLM_lpage">1073</span><span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljowkBcl1VbzA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Poncet-Montange, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardenhagen, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrocchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geck Do, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladbury, P.</span><span> </span><span class="NLM_article-title">J. E. Observed bromodomain flexibility reveals histone peptide- and small molecule ligand-compatible forms of ATAD2A</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">466</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">â</span> <span class="NLM_lpage">346</span><span class="refDoi">Â DOI: 10.1042/BJ20140933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1042%2FBJ20140933" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2015&pages=337-346&author=G.+Poncet-Montangeauthor=Y.+Zhanauthor=J.+P.+Bardenhagenauthor=A.+Petrocchiauthor=E.+Leoauthor=X.+Shiauthor=G.+R.+Leeauthor=P.+G.+Leonardauthor=M.+D.+Geck+Doauthor=M.+K.+Cardozoauthor=M.+G.+Andersenauthor=J.+N.+Palmerauthor=W.+S.+Jonesauthor=P.+Ladbury&title=J.+E.+Observed+bromodomain+flexibility+reveals+histone+peptide-+and+small+molecule+ligand-compatible+forms+of+ATAD2A&doi=10.1042%2FBJ20140933"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20140933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20140933%26sid%3Dliteratum%253Aachs%26aulast%3DPoncet-Montange%26aufirst%3DG.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DG.%2BR.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DGeck%2BDo%26aufirst%3DM.%2BD.%26aulast%3DCardozo%26aufirst%3DM.%2BK.%26aulast%3DAndersen%26aufirst%3DM.%2BG.%26aulast%3DPalmer%26aufirst%3DJ.%2BN.%26aulast%3DJones%26aufirst%3DW.%2BS.%26aulast%3DLadbury%26aufirst%3DP.%26atitle%3DJ.%2520E.%2520Observed%2520bromodomain%2520flexibility%2520reveals%2520histone%2520peptide-%2520and%2520small%2520molecule%2520ligand-compatible%2520forms%2520of%2520ATAD2A%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D466%26spage%3D337%26epage%3D346%26doi%3D10.1042%2FBJ20140933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballentine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teleha, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branum, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patch, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Player, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, W.</span><span> </span><span class="NLM_article-title">Process Development for Scale-Up of a Novel 3, 5-Substituted Thiazolidine-2, 4-dione Compound as a Potent Inhibitor for Estrogen-Related Receptor 1</span> <span class="citation_source-journal">Org. Process Res. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">321</span><span class="NLM_x">â</span> <span class="NLM_lpage">330</span><span class="refDoi">Â DOI: 10.1021/op400325r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/op400325r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=321-330&author=X.+Liauthor=R.+K.+Russellauthor=J.+Spinkauthor=S.+Ballentineauthor=C.+Telehaauthor=S.+Branumauthor=K.+Wellsauthor=D.+Beauchampauthor=R.+Patchauthor=H.+Huangauthor=M.+Playerauthor=W.+Murray&title=Process+Development+for+Scale-Up+of+a+Novel+3%2C+5-Substituted+Thiazolidine-2%2C+4-dione+Compound+as+a+Potent+Inhibitor+for+Estrogen-Related+Receptor+1&doi=10.1021%2Fop400325r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fop400325r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fop400325r%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRussell%26aufirst%3DR.%2BK.%26aulast%3DSpink%26aufirst%3DJ.%26aulast%3DBallentine%26aufirst%3DS.%26aulast%3DTeleha%26aufirst%3DC.%26aulast%3DBranum%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DK.%26aulast%3DBeauchamp%26aufirst%3DD.%26aulast%3DPatch%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DPlayer%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DW.%26atitle%3DProcess%2520Development%2520for%2520Scale-Up%2520of%2520a%2520Novel%25203%252C%25205-Substituted%2520Thiazolidine-2%252C%25204-dione%2520Compound%2520as%2520a%2520Potent%2520Inhibitor%2520for%2520Estrogen-Related%2520Receptor%25201%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2014%26volume%3D18%26spage%3D321%26epage%3D330%26doi%3D10.1021%2Fop400325r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Fink, B. E.; Gavai, A. V.; Vite, G. D.; Chen, P.; Mastalerz, H.; Norris-Drouin, J.; Tokarski, J. S.; Zhao, Y.; Han, W. C.</span><span> </span><span class="NLM_article-title">Pyrrolotriazine compounds as kinase inhibitors. PCT Int. Appl</span> WO2005066176A1,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=B.+E.+Fink&author=A.+V.+Gavai&author=G.+D.+Vite&author=P.+Chen&author=H.+Mastalerz&author=J.+Norris-Drouin&author=J.+S.+Tokarski&author=Y.+Zhao&author=W.+C.+Han&title=Pyrrolotriazine+compounds+as+kinase+inhibitors.+PCT+Int.+Appl"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFink%26aufirst%3DB.%2BE.%26atitle%3DPyrrolotriazine%2520compounds%2520as%2520kinase%2520inhibitors.%2520PCT%2520Int.%2520Appl%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span>The BROMO<i>scan</i> panel, courtesy of DiscoveRx Corp. <a href="http://www.discoverx.com" class="extLink">www.discoverx.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+BROMOscan+panel%2C+courtesy+of+DiscoveRx+Corp.+www.discoverx.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span>Further details of the application of Cellzome bead-based chemoproteomics technology to bromodomain-containing proteins will be the subject of future publications.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Further+details+of+the+application+of+Cellzome+bead-based+chemoproteomics+technology+to+bromodomain-containing+proteins+will+be+the+subject+of+future+publications."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cansfield, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellard, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doce, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leggate, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâMahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plavec, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rharbaoui, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirsch, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rausch, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span> </span><span class="NLM_article-title">A selective inhibitor reveals PI3Kgamma dependence of T<sub>H</sub>17 cell differentiation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">â</span> <span class="NLM_lpage">582</span><span class="refDoi">Â DOI: 10.1038/nchembio.957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnchembio.957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=22544264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Gqt7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=576-582&author=G.+Bergaminiauthor=K.+Bellauthor=S.+Shimamuraauthor=T.+Wernerauthor=A.+Cansfieldauthor=K.+Mullerauthor=J.+Perrinauthor=C.+Rauauthor=K.+Ellardauthor=C.+Hopfauthor=C.+Doceauthor=D.+Leggateauthor=R.+Manganoauthor=T.+Mathiesonauthor=A.+O%E2%80%99Mahonyauthor=I.+Plavecauthor=F.+Rharbaouiauthor=F.+Reinhardauthor=M.+M.+Savitskiauthor=N.+Ramsdenauthor=E.+Hirschauthor=G.+Drewesauthor=O.+Rauschauthor=M.+Bantscheffauthor=G.+Neubauer&title=A+selective+inhibitor+reveals+PI3Kgamma+dependence+of+TH17+cell+differentiation&doi=10.1038%2Fnchembio.957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor reveals PI3KÎ³ dependence of TH17 cell differentiation</span></div><div class="casAuthors">Bergamini, Giovanna; Bell, Kathryn; Shimamura, Satoko; Werner, Thilo; Cansfield, Andrew; Mueller, Katrin; Perrin, Jessica; Rau, Christina; Ellard, Katie; Hopf, Carsten; Doce, Carola; Leggate, Daniel; Mangano, Raffaella; Mathieson, Toby; O'Mahony, Alison; Plavec, Ivan; Rharbaoui, Faiza; Reinhard, Friedrich; Savitski, Mikhail M.; Ramsden, Nigel; Hirsch, Emilio; Drewes, Gerard; Rausch, Oliver; Bantscheff, Marcus; Neubauer, Gitte</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">576-582</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We devised a high-throughput chemoproteomics method that enabled multiplexed screening of 16,000 compds. against native protein and lipid kinases in cell exts.  Optimization of one chem. series resulted in CZC24832, which is to our knowledge the first selective inhibitor of phosphoinositide 3-kinase Î³ (PI3KÎ³) with efficacy in in vitro and in vivo models of inflammation.  Extensive target- and cell-based profiling of CZC24832 revealed regulation of interleukin-17-producing T helper cell (TH17) differentiation by PI3KÎ³, thus reinforcing selective inhibition of PI3KÎ³ as a potential treatment for inflammatory and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqinUn8kLqw_LVg90H21EOLACvtfcHk0lhEC7AjnmQV8A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Gqt7c%253D&md5=0141d8596ee10d4d796de57fb87a2c19</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.957%26sid%3Dliteratum%253Aachs%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DCansfield%26aufirst%3DA.%26aulast%3DMuller%26aufirst%3DK.%26aulast%3DPerrin%26aufirst%3DJ.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DEllard%26aufirst%3DK.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DLeggate%26aufirst%3DD.%26aulast%3DMangano%26aufirst%3DR.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DPlavec%26aufirst%3DI.%26aulast%3DRharbaoui%26aufirst%3DF.%26aulast%3DReinhard%26aufirst%3DF.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DRamsden%26aufirst%3DN.%26aulast%3DHirsch%26aufirst%3DE.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DRausch%26aufirst%3DO.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DNeubauer%26aufirst%3DG.%26atitle%3DA%2520selective%2520inhibitor%2520reveals%2520PI3Kgamma%2520dependence%2520of%2520TH17%2520cell%2520differentiation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D576%26epage%3D582%26doi%3D10.1038%2Fnchembio.957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesche, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delling, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumpelfeld, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poeckel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reader, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strunk, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Chemoproteomics profiling of HDAC inhibitors reveals selective targeting of HDAC complexes</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">â</span> <span class="NLM_lpage">265</span><span class="refDoi">Â DOI: 10.1038/nbt.1759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnbt.1759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=21258344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1ahug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=255-265&author=M.+Bantscheffauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=A.+Dittmannauthor=P.+Grandiauthor=A.+M.+Michonauthor=J.+Schleglauthor=Y.+Abrahamauthor=I.+Becherauthor=G.+Bergaminiauthor=M.+Boescheauthor=M.+Dellingauthor=B.+Dumpelfeldauthor=D.+Eberhardauthor=C.+Huthmacherauthor=T.+Mathiesonauthor=D.+Poeckelauthor=V.+Readerauthor=K.+Strunkauthor=G.+Sweetmanauthor=U.+Kruseauthor=G.+Neubauerauthor=N.+G.+Ramsdenauthor=G.+Drewes&title=Chemoproteomics+profiling+of+HDAC+inhibitors+reveals+selective+targeting+of+HDAC+complexes&doi=10.1038%2Fnbt.1759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoproteomics profiling of HDAC inhibitors reveals selective targeting of HDAC complexes</span></div><div class="casAuthors">Bantscheff, Marcus; Hopf, Carsten; Savitski, Mikhail M.; Dittmann, Antje; Grandi, Paola; Michon, Anne-Marie; Schlegl, Judith; Abraham, Yann; Becher, Isabelle; Bergamini, Giovanna; Boesche, Markus; Delling, Manja; Duempelfeld, Birgit; Eberhard, Dirk; Huthmacher, Carola; Mathieson, Toby; Poeckel, Daniel; Reader, Valerie; Strunk, Katja; Sweetman, Gavain; Kruse, Ulrich; Neubauer, Gitte; Ramsden, Nigel G.; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">255-265</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The development of selective histone deacetylase (HDAC) inhibitors with anti-cancer and anti-inflammatory properties remains challenging in large part owing to the difficulty of probing the interaction of small mols. with megadalton protein complexes.  A combination of affinity capture and quant. mass spectrometry revealed the selectivity with which 16 HDAC inhibitors target multiple HDAC complexes scaffolded by ELM-SANT domain subunits, including a novel mitotic deacetylase complex (MiDAC).  Inhibitors clustered according to their target profiles with stronger binding of aminobenzamides to the HDAC NCoR complex than to the HDAC Sin3 complex.  We identified several non-HDAC targets for hydroxamate inhibitors.  HDAC inhibitors with distinct profiles have correspondingly different effects on downstream targets.  We also identified the anti-inflammatory drug bufexamac as a class IIb (HDAC6, HDAC10) HDAC inhibitor.  Our approach enables the discovery of novel targets and inhibitors and suggests that the selectivity of HDAC inhibitors should be evaluated in the context of HDAC complexes and not purified catalytic subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdsITjgu14rVg90H21EOLACvtfcHk0lhOd3h3u4680g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1ahug%253D%253D&md5=3a826d143fb793719a56fba3c410d51e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1759%26sid%3Dliteratum%253Aachs%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DAbraham%26aufirst%3DY.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DBoesche%26aufirst%3DM.%26aulast%3DDelling%26aufirst%3DM.%26aulast%3DDumpelfeld%26aufirst%3DB.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DPoeckel%26aufirst%3DD.%26aulast%3DReader%26aufirst%3DV.%26aulast%3DStrunk%26aufirst%3DK.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DRamsden%26aufirst%3DN.%2BG.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DChemoproteomics%2520profiling%2520of%2520HDAC%2520inhibitors%2520reveals%2520selective%2520targeting%2520of%2520HDAC%2520complexes%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D255%26epage%3D265%26doi%3D10.1038%2Fnbt.1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallasch, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frenzel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghidelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maier, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendtner, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Chemoproteomics-based kinome profiling and target deconvolution of clinical multi-kinase inhibitors in primary chronic lymphocytic leukemia cells</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">89</span><span class="NLM_x">â</span> <span class="NLM_lpage">100</span><span class="refDoi">Â DOI: 10.1038/leu.2010.233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fleu.2010.233" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=89-100&author=U.+Kruseauthor=C.+P.+Pallaschauthor=M.+Bantscheffauthor=D.+Eberhardauthor=L.+Frenzelauthor=S.+Ghidelliauthor=S.+K.+Maierauthor=T.+Wernerauthor=C.+M.+Wendtnerauthor=G.+Drewes&title=Chemoproteomics-based+kinome+profiling+and+target+deconvolution+of+clinical+multi-kinase+inhibitors+in+primary+chronic+lymphocytic+leukemia+cells&doi=10.1038%2Fleu.2010.233"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fleu.2010.233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2010.233%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DPallasch%26aufirst%3DC.%2BP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DFrenzel%26aufirst%3DL.%26aulast%3DGhidelli%26aufirst%3DS.%26aulast%3DMaier%26aufirst%3DS.%2BK.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DWendtner%26aufirst%3DC.%2BM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DChemoproteomics-based%2520kinome%2520profiling%2520and%2520target%2520deconvolution%2520of%2520clinical%2520multi-kinase%2520inhibitors%2520in%2520primary%2520chronic%2520lymphocytic%2520leukemia%2520cells%26jtitle%3DLeukemia%26date%3D2011%26volume%3D25%26spage%3D89%26epage%3D100%26doi%3D10.1038%2Fleu.2010.233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span> </span><span class="NLM_article-title">Targeted data acquisition for improved reproducibility and robustness of proteomic mass spectrometry assays</span> <span class="citation_source-journal">J. Am. Soc. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">â</span> <span class="NLM_lpage">1679</span><span class="refDoi">Â DOI: 10.1016/j.jasms.2010.01.012</span> </span><div class="citationLinks">[<a href="/doi/10.1016/j.jasms.2010.01.012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Wmt7bN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=1668-1679&author=M.+M.+Savitskiauthor=F.+Fischerauthor=T.+Mathiesonauthor=G.+Sweetmanauthor=M.+Langauthor=M.+Bantscheff&title=Targeted+data+acquisition+for+improved+reproducibility+and+robustness+of+proteomic+mass+spectrometry+assays&doi=10.1016%2Fj.jasms.2010.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Data Acquisition for Improved Reproducibility and Robustness of Proteomic Mass Spectrometry Assays</span></div><div class="casAuthors">Savitski, Mikhail M.; Fischer, Frank; Mathieson, Toby; Sweetman, Gavain; Lang, Manja; Bantscheff, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1668-1679</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Quant. mass spectrometry-based proteomic assays often suffer from a lack of robustness and reproducibility.  The authors here describe a targeted mass spectrometric data acquisition strategy for affinity enriched subproteomes - in the authors' case the kinome - that enables a substantially improved reproducibility of detection, and improved quantification via isobaric tags.  Inclusion mass lists contg. m/z, charge state, and retention time were created based on a set of 80 shotgun-type expts. performed under identical exptl. conditions.  For each target protein, peptides were selected according to their frequency of observation and isobaric tag for relative and abs. quantitation (iTRAQ) reporter ion quality.  Retention times of selected peptides were aligned using similarity driven pairwise alignment strategy yielding <1 min std. deviation for 4 h gradients.  Multiple fragmentation of the same peptides resulted in better statistics and more precise reporter ion based quantification without any loss in coverage.  Overall, 24% more target proteins were quantified using the targeted data acquisition approach, and precision of quantification improved by >1.5-fold.  The authors also show that a combination of higher energy collisional dissocn. (HCD) with collisional induced dissocn. (CID) outperformed pulsed-Q-dissocn. (PQD) on the Orbitrap XL.  With the CID/HCD based targeted data acquisition approach 10% more quantifiable target proteins were identified and a 2-fold increase in quantification precision was achieved.  The authors have obsd. excellent reproducibility between different instruments, underlining the robustness of the approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyp9JslOo1a7Vg90H21EOLACvtfcHk0lhOd3h3u4680g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Wmt7bN&md5=c7b65735530bbe9978c8c481876610b9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jasms.2010.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jasms.2010.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DFischer%26aufirst%3DF.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DLang%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26atitle%3DTargeted%2520data%2520acquisition%2520for%2520improved%2520reproducibility%2520and%2520robustness%2520of%2520proteomic%2520mass%2520spectrometry%2520assays%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D2010%26volume%3D21%26spage%3D1668%26epage%3D1679%26doi%3D10.1016%2Fj.jasms.2010.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastuck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesche, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reader, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouwmeester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">â</span> <span class="NLM_lpage">1044</span><span class="refDoi">Â DOI: 10.1038/nbt1328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnbt1328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=17721511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1035-1044&author=M.+Bantscheffauthor=D.+Eberhardauthor=Y.+Abrahamauthor=S.+Bastuckauthor=M.+Boescheauthor=S.+Hobsonauthor=T.+Mathiesonauthor=J.+Perrinauthor=M.+Raidaauthor=C.+Rauauthor=V.+Readerauthor=G.+Sweetmanauthor=A.+Bauerauthor=T.+Bouwmeesterauthor=C.+Hopfauthor=U.+Kruseauthor=G.+Neubauerauthor=N.+Ramsdenauthor=J.+Rickauthor=B.+Kusterauthor=G.+Drewes&title=Quantitative+chemical+proteomics+reveals+mechanisms+of+action+of+clinical+ABL+kinase+inhibitors&doi=10.1038%2Fnbt1328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span></div><div class="casAuthors">Bantscheff, Marcus; Eberhard, Dirk; Abraham, Yann; Bastuck, Sonja; Boesche, Markus; Hobson, Scott; Mathieson, Toby; Perrin, Jessica; Raida, Manfred; Rau, Christina; Reader, Valerie; Sweetman, Gavain; Bauer, Andreas; Bouwmeester, Tewis; Hopf, Carsten; Kruse, Ulrich; Neubauer, Gitte; Ramsden, Nigel; Rick, Jens; Kuster, Bernhard; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1035-1044</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors describe a chem. proteomics approach to profile the interaction of small mols. with hundreds of endogenously expressed protein kinases and purine-binding proteins.  This subproteome is captured by immobilized nonselective kinase inhibitors (kinobeads), and the bound proteins are quantified in parallel by mass spectrometry using isobaric tags for relative and abs. quantification (iTRAQ).  By measuring the competition with the affinity matrix, the authors assess the binding of drugs to their targets in cell lysates and in cells.  By mapping drug-induced changes in the phosphorylation state of the captured proteome, the authors also analyze signaling pathways downstream of target kinases.  Quant. profiling of the drugs imatinib (Gleevec), dasatinib (Sprycel) and bosutinib in K562 cells confirms known targets including ABL and SRC family kinases and identifies the receptor tyrosine kinase DDR1 and the oxidoreductase NQO2 as novel targets of imatinib.  The data suggest that the approach is a valuable tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsC3OpIuhl8LVg90H21EOLACvtfcHk0liVJL78ZIz6_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF&md5=ed20d4684a8a6733aba24ca64602bae8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnbt1328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1328%26sid%3Dliteratum%253Aachs%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DY.%26aulast%3DBastuck%26aufirst%3DS.%26aulast%3DBoesche%26aufirst%3DM.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DPerrin%26aufirst%3DJ.%26aulast%3DRaida%26aufirst%3DM.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DReader%26aufirst%3DV.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DRamsden%26aufirst%3DN.%26aulast%3DRick%26aufirst%3DJ.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DQuantitative%2520chemical%2520proteomics%2520reveals%2520mechanisms%2520of%2520action%2520of%2520clinical%2520ABL%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1035%26epage%3D1044%26doi%3D10.1038%2Fnbt1328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Sharma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bairlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greff, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daub, H.</span><span> </span><span class="NLM_article-title">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">â</span> <span class="NLM_lpage">744</span><span class="refDoi">Â DOI: 10.1038/nmeth.1373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnmeth.1373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=19749761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=741-744&author=K.+Sharmaauthor=C.+Weberauthor=M.+Bairleinauthor=Z.+Greffauthor=G.+Keriauthor=J.+Coxauthor=J.+V.+Olsenauthor=H.+Daub&title=Proteomics+strategy+for+quantitative+protein+interaction+profiling+in+cell+extracts&doi=10.1038%2Fnmeth.1373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span></div><div class="casAuthors">Sharma, Kirti; Weber, Christoph; Bairlein, Michaela; Greff, Zoltan; Keri, Gyoergy; Cox, Juergen; Olsen, Jesper V.; Daub, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-744</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors report a proteomics strategy to both identify and quantify cellular target protein interactions with externally introduced ligands.  The authors detd. dissocn. consts. for target proteins interacting with the ligand of interest by combining quant. mass spectrometry with a defined set of affinity purifn. expts.  The authors demonstrate the general utility of this methodol. in interaction studies involving small-mol. kinase inhibitors, a tyrosine-phosphorylated peptide and an antibody as affinity ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9I5plLFxQRbVg90H21EOLACvtfcHk0liVJL78ZIz6_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O&md5=78732562cac1cc2af0938f5536134098</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1373%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DK.%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DBairlein%26aufirst%3DM.%26aulast%3DGreff%26aufirst%3DZ.%26aulast%3DKeri%26aufirst%3DG.%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DDaub%26aufirst%3DH.%26atitle%3DProteomics%2520strategy%2520for%2520quantitative%2520protein%2520interaction%2520profiling%2520in%2520cell%2520extracts%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D741%26epage%3D744%26doi%3D10.1038%2Fnmeth.1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.: Macromol. Crystallogr., Part A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">â</span> <span class="NLM_lpage">326</span><span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0liVJL78ZIz6_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%253A%2520Macromol.%2520Crystallogr.%252C%2520Part%2520A%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">Recent changes to the MOSFLM package for processing film and image plate data</span> <span class="citation_source-journal">Jt. CCP4 ESF-EACBM Newsl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">â</span> <span class="NLM_lpage">35</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1992&pages=29-35&author=A.+G.+W.+Leslie&title=Recent+changes+to+the+MOSFLM+package+for+processing+film+and+image+plate+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DRecent%2520changes%2520to%2520the%2520MOSFLM%2520package%2520for%2520processing%2520film%2520and%2520image%2520plate%2520data%26jtitle%3DJt.%2520CCP4%2520ESF-EACBM%2520Newsl.%26date%3D1992%26volume%3D26%26spage%3D29%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span><span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhBnIvZvF4EPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Evans, P. R.</span><span> </span><span class="NLM_article-title">Data Reduction</span> <span class="citation_source-journal">Proc. CCP4 Study Weekend, Data Collect. Process.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">â</span> <span class="NLM_lpage">122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&pages=114-122&author=P.+R.+Evans&title=Data+Reduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DData%2520Reduction%26jtitle%3DProc.%2520CCP4%2520Study%2520Weekend%252C%2520Data%2520Collect.%2520Process.%26date%3D1993%26spage%3D114%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">â</span> <span class="NLM_lpage">1214</span><span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lhBnIvZvF4EPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Collaborative Computational Project No.4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">D50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">â</span> <span class="NLM_lpage">763</span><span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project+No.4&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3DD50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">â</span> <span class="NLM_lpage">255</span><span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhBnIvZvF4EPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">â</span> <span class="NLM_lpage">2132</span><span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhBnIvZvF4EPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudgin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auger, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">â</span> <span class="NLM_lpage">533</span><span class="refDoi">Â DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0li_bluGdUFwhw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 56 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Michael A. Clegg, Natalie H. Theodoulou, Paul Bamborough, Chun-wa Chung, Peter D. Craggs, Emmanuel H. Demont, Laurie J. Gordon, Gemma M. Liwicki, Alex Phillipou, Nicholas C. O. Tomkinson, Rab K. Prinjha, <span class="NLM_string-name hlFld-ContribAuthor">Philip G. Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Naphthyridones into Selective TATA-Binding Protein Associated Factor 1 (TAF1) Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00294%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DOptimization%252Bof%252BNaphthyridones%252Binto%252BSelective%252BTATA-Binding%252BProtein%252BAssociated%252BFactor%252B1%252B%252528TAF1%252529%252BBromodomain%252BInhibitors%26aulast%3DClegg%26aufirst%3DMichael%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D21052021%26date%3D29072021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francesco Rianjongdee, Stephen J. Atkinson, Chun-wa Chung, Paola Grandi, James R. J. Gray, Laura J. Kaushansky, Patricia Medeiros, Cassie Messenger, Alex Phillipou, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Alexander L. Satz, Simon Taylor, Ian D. Wall, Robert J. Watson, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00412%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BSelective%252BBET%252BBD2%252BInhibitor%252Bfrom%252Ba%252BDNA-Encoded%252BLibrary%252BTechnology%252BScreening%252BHit%26aulast%3DRianjongdee%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26date%3D08032021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan T. Lloyd, Kyle McLaughlin, Mulu Y. Lubula, Jamie C. Gay, Andrea Dest, Cong Gao, Margaret Phillips, Marco Tonelli, Gabriel Cornilescu, Matthew R. Marunde, Chiara M. Evans, Samuel P. Boyson, Samuel Carlson, Michael-Christopher Keogh, John L. Markley, Seth Frietze, <span class="NLM_string-name hlFld-ContribAuthor">Karen C. Glass</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights into the Recognition of Mono- and Diacetylated Histones by the ATAD2B Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12799-12813. <a href="https://doi.org/10.1021/acs.jmedchem.0c01178" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01178</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01178%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructural%252BInsights%252Binto%252Bthe%252BRecognition%252Bof%252BMono-%252Band%252BDiacetylated%252BHistones%252Bby%252Bthe%252BATAD2B%252BBromodomain%26aulast%3DLloyd%26aufirst%3DJonathan%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07072020%26date%3D21102020%26volume%3D63%26issue%3D21%26spage%3D12799%26epage%3D12813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alex Preston, Stephen Atkinson, Paul Bamborough, Chun-wa Chung, Peter D. Craggs, Laurie Gordon, Paola Grandi, James R. J. Gray, Emma J. Jones, Matthew Lindon, Anne-Marie Michon, Darren J. Mitchell, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan Seal, Simon Taylor, Ian Wall, Robert J. Watson, James Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9070-9092. <a href="https://doi.org/10.1021/acs.jmedchem.0c00605" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00605</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252Ba%252BHighly%252BSelective%252Band%252BIn%252BVivo-Capable%252BInhibitor%252Bof%252Bthe%252BSecond%252BBromodomain%252Bof%252Bthe%252BBromodomain%252Band%252BExtra%252BTerminal%252BDomain%252BFamily%252Bof%252BProteins%26aulast%3DPreston%26aufirst%3DAlex%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16042020%26date%3D20082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9070%26epage%3D9092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Watson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Rob Davis, Laurie Gordon, Paola Grandi, Massimo Petretich, Alex Phillipou, Rab K. Prinjha, Inmaculada Rioja, Peter Soden, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9045-9069. <a href="https://doi.org/10.1021/acs.jmedchem.0c00614" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGSK789%25253A%252BA%252BSelective%252BInhibitor%252Bof%252Bthe%252BFirst%252BBromodomains%252B%252528BD1%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DWatson%26aufirst%3DRobert%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D04082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9045%26epage%3D9069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan T. Seal, Stephen J. Atkinson, Helen Aylott, Paul Bamborough, Chun-wa Chung, Royston C. B. Copley, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Matthew Lindon, Cassie Messenger, Anne-Marie Michon, Darren Mitchell, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Simon Taylor, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9093-9126. <a href="https://doi.org/10.1021/acs.jmedchem.0c00796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00796</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BOptimization%252Bof%252Ba%252BNovel%25252C%252BWeak%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BBromodomain%252BFragment%252BLigand%252Bto%252Ba%252BPotent%252Band%252BSelective%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitor%26aulast%3DSeal%26aufirst%3DJonathan%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D30082020%26date%3D23072020%26volume%3D63%26issue%3D17%26spage%3D9093%26epage%3D9126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongdong Chen, Weixia Du, Xufeng Yang, <span class="NLM_string-name hlFld-ContribAuthor">Tao Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Domino Synthetic Strategy for Tetrahydrothiopyran Derivatives from Benzaldehydes, 2-Acetylfuran/2-Acetylthiophene, and Sodium Sulfide. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (14)
                                     , 9088-9095. <a href="https://doi.org/10.1021/acs.joc.0c01006" title="DOI URL">https://doi.org/10.1021/acs.joc.0c01006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.0c01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.0c01006%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DDomino%252BSynthetic%252BStrategy%252Bfor%252BTetrahydrothiopyran%252BDerivatives%252Bfrom%252BBenzaldehydes%25252C%252B2-Acetylfuran%25252F2-Acetylthiophene%25252C%252Band%252BSodium%252BSulfide%26aulast%3DChen%26aufirst%3DDongdong%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D23042020%26date%3D26062020%26date%3D12062020%26volume%3D85%26issue%3D14%26spage%3D9088%26epage%3D9095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Simon C. C. Lucas, Stephen J. Atkinson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Laurie Gordon, Darren J. Mitchell, Alexander Phillipou, Rab K. Prinjha, Robert J. Sheppard, Nicholas C. O. Tomkinson, Robert J. Watson, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5212-5241. <a href="https://doi.org/10.1021/acs.jmedchem.0c00021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00021%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252BPotent%252BATAD2%252Band%252BCECR2%252BBromodomain%252BInhibitors%252Bwith%252Ban%252BAtypical%252BBinding%252BMode%26aulast%3DLucas%26aufirst%3DSimon%2BC.%2BC.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06012020%26date%3D06052020%26date%3D22042020%26volume%3D63%26issue%3D10%26spage%3D5212%26epage%3D5241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul Bamborough, Chun-wa Chung, Emmanuel H. Demont, Angela M. Bridges, Peter D. Craggs, David P. Dixon, Peter Francis, Rebecca C. Furze, Paola Grandi, Emma J. Jones, Bhumika Karamshi, Kelly Locke, Simon C. C. Lucas, Anne-Marie Michon, Darren J. Mitchell, Peter PogÃ¡ny, Rab K. Prinjha, Christina Rau, Ana Maria Roa, Andrew D. Roberts, Robert J. Sheppard, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Watson</span>. </span><span class="cited-content_cbyCitation_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (16)
                                     , 7506-7525. <a href="https://doi.org/10.1021/acs.jmedchem.9b00673" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00673</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BQualified%252BSuccess%25253A%252BDiscovery%252Bof%252Ba%252BNew%252BSeries%252Bof%252BATAD2%252BBromodomain%252BInhibitors%252Bwith%252Ba%252BNovel%252BBinding%252BMode%252BUsing%252BHigh-Throughput%252BScreening%252Band%252BHit%252BQualification%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23042019%26date%3D09082019%26volume%3D62%26issue%3D16%26spage%3D7506%26epage%3D7525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shumei Wang, Vickie Tsui, Terry D. Crawford, James E. Audia, Daniel J. Burdick, Maureen H. Beresini, Alexandre CÃ´tÃ©, Richard Cummings, Martin Duplessis, E. Megan Flynn, Michael C. Hewitt, Hon-Ren Huang, Hariharan Jayaram, Ying Jiang, Shivangi Joshi, Jeremy Murray, Christopher G. Nasveschuk, Eneida Pardo, Florence Poy, F. Anthony Romero, Yong Tang, Alexander M. Taylor, Jian Wang, Zhaowu Xu, Laura E. Zawadzke, Xiaoyu Zhu, Brian K. Albrecht, Steven R. Magnuson, Steve Bellon, <span class="NLM_string-name hlFld-ContribAuthor">Andrea G. Cochran</span>. </span><span class="cited-content_cbyCitation_article-title">GNE-371, a Potent and Selective Chemical Probe for the Second Bromodomains of Human Transcription-Initiation-Factor TFIID Subunit 1 and Transcription-Initiation-Factor TFIID Subunit 1-like. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (20)
                                     , 9301-9315. <a href="https://doi.org/10.1021/acs.jmedchem.8b01225" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01225%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGNE-371%25252C%252Ba%252BPotent%252Band%252BSelective%252BChemical%252BProbe%252Bfor%252Bthe%252BSecond%252BBromodomains%252Bof%252BHuman%252BTranscription-Initiation-Factor%252BTFIID%252BSubunit%252B1%252Band%252BTranscription-Initiation-Factor%252BTFIID%252BSubunit%252B1-like%26aulast%3DWang%26aufirst%3DShumei%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D03082018%26date%3D05102018%26volume%3D61%26issue%3D20%26spage%3D9301%26epage%3D9315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul Bamborough, Chun-wa Chung, Rebecca C. Furze, Paola Grandi, Anne-Marie Michon, Robert J. Watson, Darren J. Mitchell, Heather Barnett, Rab K. Prinjha, Christina Rau, Robert J. Sheppard, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (18)
                                     , 8321-8336. <a href="https://doi.org/10.1021/acs.jmedchem.8b00862" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00862</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00862%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAiming%252Bto%252BMiss%252Ba%252BMoving%252BTarget%25253A%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BSelectivity%252Bin%252BConstrained%252BATAD2%252BInhibitors%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30052018%26date%3D18092018%26volume%3D61%26issue%3D18%26spage%3D8321%26epage%3D8336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laurent  Batiste</span>, <span class="hlFld-ContribAuthor ">Andrea  Unzue</span>, <span class="hlFld-ContribAuthor ">Aymeric  Dolbois</span>, <span class="hlFld-ContribAuthor ">Fabrice  Hassler</span>, <span class="hlFld-ContribAuthor ">Xuan  Wang</span>, <span class="hlFld-ContribAuthor ">Nicholas  Deerain</span>, <span class="hlFld-ContribAuthor ">Jian  Zhu</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Spiliotopoulos</span>, <span class="hlFld-ContribAuthor ">Cristina  Nevado</span>, and <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>  . </span><span class="cited-content_cbyCitation_article-title">Chemical Space Expansion of Bromodomain Ligands Guided by in Silico Virtual Couplings (AutoCouple). </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2018,</strong> <em>4 </em>
                                    (2)
                                     , 180-188. <a href="https://doi.org/10.1021/acscentsci.7b00401" title="DOI URL">https://doi.org/10.1021/acscentsci.7b00401</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.7b00401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.7b00401%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DChemical%252BSpace%252BExpansion%252Bof%252BBromodomain%252BLigands%252BGuided%252Bby%252Bin%252BSilico%252BVirtual%252BCouplings%252B%252528AutoCouple%252529%26aulast%3DBatiste%26aufirst%3DLaurent%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D29082017%26date%3D07022018%26date%3D28022018%26volume%3D4%26issue%3D2%26spage%3D180%26epage%3D188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amaury E. FernÃ¡ndez-MontalvÃ¡n, Markus Berger, Benno Kuropka, Seong Joo Koo, Volker Badock, Joerg Weiske, Vera Puetter, Simon J. Holton, Detlef StÃ¶ckigt, Antonius ter Laak, Paolo A. Centrella, Matthew A. Clark, Christoph E. Dumelin, Eric A. Sigel, Holly H. Soutter, Dawn M. Troast, Ying Zhang, John W. Cuozzo, Anthony D. Keefe, Didier Roche, Vincent Rodeschini, Apirat Chaikuad, Laura DÃ­az-SÃ¡ez, James M. Bennett, Oleg Fedorov, Kilian V. M. Huber, Jan HÃ¼bner, Hilmar Weinmann, Ingo V. Hartung, <span class="NLM_string-name hlFld-ContribAuthor">MÃ¡tyÃ¡s GorjÃ¡nÃ¡cz</span>. </span><span class="cited-content_cbyCitation_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (11)
                                     , 2730-2736. <a href="https://doi.org/10.1021/acschembio.7b00708" title="DOI URL">https://doi.org/10.1021/acschembio.7b00708</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DIsoform-Selective%252BATAD2%252BChemical%252BProbe%252Bwith%252BNovel%252BChemical%252BStructure%252Band%252BUnusual%252BMode%252Bof%252BAction%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DAmaury%2BE.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D15082017%26date%3D18102017%26date%3D24102017%26date%3D18102017%26volume%3D12%26issue%3D11%26spage%3D2730%26epage%3D2736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher N.  Johnson</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Christopher W.  Murray</span>, and <span class="hlFld-ContribAuthor ">David C.  Rees</span>  . </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2015. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (1)
                                     , 89-99. <a href="https://doi.org/10.1021/acs.jmedchem.6b01123" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01123</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01123%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2015%26aulast%3DJohnson%26aufirst%3DChristopher%2BN.%26date%3D2017%26date%3D2016%26date%3D2016%26date%3D28072016%26date%3D14102016%26date%3D12012017%26volume%3D60%26issue%3D1%26spage%3D89%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Terry D.  Crawford</span>, <span class="hlFld-ContribAuthor ">Vickie  Tsui</span>, <span class="hlFld-ContribAuthor ">E. Megan  Flynn</span>, <span class="hlFld-ContribAuthor ">Shumei  Wang</span>, <span class="hlFld-ContribAuthor ">Alexander M.  Taylor</span>, <span class="hlFld-ContribAuthor ">Alexandre  CÃ´tÃ©</span>, <span class="hlFld-ContribAuthor ">James E.  Audia</span>, <span class="hlFld-ContribAuthor ">Maureen H.  Beresini</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Burdick</span>, <span class="hlFld-ContribAuthor ">Richard  Cummings</span>, <span class="hlFld-ContribAuthor ">Les A.  Dakin</span>, <span class="hlFld-ContribAuthor ">Martin  Duplessis</span>, <span class="hlFld-ContribAuthor ">Andrew C.  Good</span>, <span class="hlFld-ContribAuthor ">Michael C.  Hewitt</span>, <span class="hlFld-ContribAuthor ">Hon-Ren  Huang</span>, <span class="hlFld-ContribAuthor ">Hariharan  Jayaram</span>, <span class="hlFld-ContribAuthor ">James R.  Kiefer</span>, <span class="hlFld-ContribAuthor ">Ying  Jiang</span>, <span class="hlFld-ContribAuthor ">Jeremy  Murray</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Nasveschuk</span>, <span class="hlFld-ContribAuthor ">Eneida  Pardo</span>, <span class="hlFld-ContribAuthor ">Florence  Poy</span>, <span class="hlFld-ContribAuthor ">F. Anthony  Romero</span>, <span class="hlFld-ContribAuthor ">Yong  Tang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhaowu  Xu</span>, <span class="hlFld-ContribAuthor ">Laura E.  Zawadzke</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhu</span>, <span class="hlFld-ContribAuthor ">Brian K.  Albrecht</span>, <span class="hlFld-ContribAuthor ">Steven R.  Magnuson</span>, <span class="hlFld-ContribAuthor ">Steve  Bellon</span>, and <span class="hlFld-ContribAuthor ">Andrea G.  Cochran</span>  . </span><span class="cited-content_cbyCitation_article-title">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (11)
                                     , 5391-5402. <a href="https://doi.org/10.1021/acs.jmedchem.6b00264" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00264</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiving%252Binto%252Bthe%252BWater%25253A%252BInducible%252BBinding%252BConformations%252Bfor%252BBRD4%25252C%252BTAF1%2525282%252529%25252C%252BBRD9%25252C%252Band%252BCECR2%252BBromodomains%26aulast%3DCrawford%26aufirst%3DTerry%2BD.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D18022016%26date%3D31052016%26date%3D09062016%26date%3D24052016%26volume%3D59%26issue%3D11%26spage%3D5391%26epage%3D5402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">F. Anthony  Romero</span>, <span class="hlFld-ContribAuthor ">Alexander M.  Taylor</span>, <span class="hlFld-ContribAuthor ">Terry D.  Crawford</span>, <span class="hlFld-ContribAuthor ">Vickie  Tsui</span>, <span class="hlFld-ContribAuthor ">Alexandre  CÃ´tÃ©</span>, and <span class="hlFld-ContribAuthor ">Steven  Magnuson</span>  . </span><span class="cited-content_cbyCitation_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1271-1298. <a href="https://doi.org/10.1021/acs.jmedchem.5b01514" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDisrupting%252BAcetyl-Lysine%252BRecognition%25253A%252BProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BBromodomain%252BInhibitors%26aulast%3DRomero%26aufirst%3DF.%2BAnthony%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D29092015%26date%3D01122015%26date%3D25022016%26date%3D16112015%26volume%3D59%26issue%3D4%26spage%3D1271%26epage%3D1298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Apirat  Chaikuad</span>, <span class="hlFld-ContribAuthor ">Steffen  Lang</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Claudia  Temperini</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Johan  Hollander</span>, <span class="hlFld-ContribAuthor ">Ruta  Nachane</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>, <span class="hlFld-ContribAuthor ">Susanne  MÃ¼ller</span>, <span class="hlFld-ContribAuthor ">Gregg  Siegal</span>, and <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Identification of Inhibitory Fragments Targeting the p300/CBP-Associated Factor Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1648-1653. <a href="https://doi.org/10.1021/acs.jmedchem.5b01719" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01719</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01719%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BIdentification%252Bof%252BInhibitory%252BFragments%252BTargeting%252Bthe%252Bp300%25252FCBP-Associated%252BFactor%252BBromodomain%26aulast%3DChaikuad%26aufirst%3DApirat%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D04112015%26date%3D13012016%26date%3D25022016%26date%3D05012016%26volume%3D59%26issue%3D4%26spage%3D1648%26epage%3D1653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditi  Nayak</span>, <span class="hlFld-ContribAuthor ">Madhuri  Dutta</span>, <span class="hlFld-ContribAuthor ">Anasuya  Roychowdhury</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging oncogene ATAD2: Signaling cascades and therapeutic initiatives. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2021,</strong> <em>276 </em>, 119322. <a href="https://doi.org/10.1016/j.lfs.2021.119322" title="DOI URL">https://doi.org/10.1016/j.lfs.2021.119322</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2021.119322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2021.119322%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DEmerging%252Boncogene%252BATAD2%25253A%252BSignaling%252Bcascades%252Band%252Btherapeutic%252Binitiatives%26aulast%3DNayak%26aufirst%3DAditi%26date%3D2021%26volume%3D276%26spage%3D119322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Zhang</span>, <span class="hlFld-ContribAuthor ">Shan  Li</span>, <span class="hlFld-ContribAuthor ">Kai-Wen  Cheng</span>, <span class="hlFld-ContribAuthor ">Tsui-Fen  Chou</span>. </span><span class="cited-content_cbyCitation_article-title">AAA ATPases as therapeutic targets: Structure, functions, and small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>219 </em>, 113446. <a href="https://doi.org/10.1016/j.ejmech.2021.113446" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113446%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAAA%252BATPases%252Bas%252Btherapeutic%252Btargets%25253A%252BStructure%25252C%252Bfunctions%25252C%252Band%252Bsmall-molecule%252Binhibitors%26aulast%3DZhang%26aufirst%3DGang%26date%3D2021%26volume%3D219%26spage%3D113446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samuel P.  Boyson</span>, <span class="hlFld-ContribAuthor ">Cong  Gao</span>, <span class="hlFld-ContribAuthor ">Kathleen  Quinn</span>, <span class="hlFld-ContribAuthor ">Joseph  Boyd</span>, <span class="hlFld-ContribAuthor ">Hana  Paculova</span>, <span class="hlFld-ContribAuthor ">Seth  Frietze</span>, <span class="hlFld-ContribAuthor ">Karen C.  Glass</span>. </span><span class="cited-content_cbyCitation_article-title">Functional Roles of Bromodomain Proteins in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (14)
                                     , 3606. <a href="https://doi.org/10.3390/cancers13143606" title="DOI URL">https://doi.org/10.3390/cancers13143606</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13143606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13143606%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DFunctional%252BRoles%252Bof%252BBromodomain%252BProteins%252Bin%252BCancer%26aulast%3DBoyson%26aufirst%3DSamuel%2BP.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D14%26spage%3D3606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lianbo  Zhao</span>, <span class="hlFld-ContribAuthor ">Lei  Hao</span>, <span class="hlFld-ContribAuthor ">Yupeng  Fu</span>, <span class="hlFld-ContribAuthor ">Yan  Cheng</span>, <span class="hlFld-ContribAuthor ">Guojun  Pan</span>, <span class="hlFld-ContribAuthor ">Laurent  DÃ©saubry</span>, <span class="hlFld-ContribAuthor ">Peng  Yu</span>, <span class="hlFld-ContribAuthor ">Dong  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Accessing 1,8âNaphthyridones by MetalâFree Regioselective Amination of Pyridine
              N
              âoxides/AcidâMediated Cyclization. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2020,</strong> <em>362 </em>
                                    (18)
                                     , 3841-3845. <a href="https://doi.org/10.1002/adsc.202000769" title="DOI URL">https://doi.org/10.1002/adsc.202000769</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.202000769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.202000769%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DAccessing%252B1%25252C8%2525E2%252580%252590Naphthyridones%252Bby%252BMetal%2525E2%252580%252590Free%252BRegioselective%252BAmination%252Bof%252BPyridine%252BN%252B%2525E2%252580%252590oxides%25252FAcid%2525E2%252580%252590Mediated%252BCyclization%26aulast%3DZhao%26aufirst%3DLianbo%26date%3D2020%26date%3D2020%26volume%3D362%26issue%3D18%26spage%3D3841%26epage%3D3845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunâwa  Chung</span>, <span class="hlFld-ContribAuthor ">Michael M.  Hann</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting ProteinâProtein Interactions Perspective. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 479-502. <a href="https://doi.org/10.1002/9781118681121.ch20" title="DOI URL">https://doi.org/10.1002/9781118681121.ch20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch20%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252BProtein%2525E2%252580%252593Protein%252BInteractions%252BPerspective%26aulast%3DChung%26aufirst%3DChun%25E2%2580%2590wa%26date%3D2020%26date%3D2020%26spage%3D479%26epage%3D502%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dahong  Yao</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinhui  Wang</span>, <span class="hlFld-ContribAuthor ">Dabo  Pan</span>, <span class="hlFld-ContribAuthor ">Zhendan  He</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel ATAD2 bromodomain inhibitors that trigger apoptosis and autophagy in breast cells by structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2020,</strong> <em>35 </em>
                                    (1)
                                     , 713-725. <a href="https://doi.org/10.1080/14756366.2020.1740924" title="DOI URL">https://doi.org/10.1080/14756366.2020.1740924</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2020.1740924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2020.1740924%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252BATAD2%252Bbromodomain%252Binhibitors%252Bthat%252Btrigger%252Bapoptosis%252Band%252Bautophagy%252Bin%252Bbreast%252Bcells%252Bby%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DYao%26aufirst%3DDahong%26date%3D2020%26date%3D2020%26volume%3D35%26issue%3D1%26spage%3D713%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemische Epigenetik: der Einfluss chemischer und chemoâbiologischer Techniken auf die ZielstrukturâValidierung von BromodomÃ¤nen. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2019,</strong> <em>131 </em>
                                    (50)
                                     , 18096-18120. <a href="https://doi.org/10.1002/ange.201812164" title="DOI URL">https://doi.org/10.1002/ange.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DChemische%252BEpigenetik%25253A%252Bder%252BEinfluss%252Bchemischer%252Bund%252Bchemo%2525E2%252580%252590biologischer%252BTechniken%252Bauf%252Bdie%252BZielstruktur%2525E2%252580%252590Validierung%252Bvon%252BBromodom%2525C3%2525A4nen%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26date%3D2019%26volume%3D131%26issue%3D50%26spage%3D18096%26epage%3D18120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2019,</strong> <em>58 </em>
                                    (50)
                                     , 17930-17952. <a href="https://doi.org/10.1002/anie.201812164" title="DOI URL">https://doi.org/10.1002/anie.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DChemical%252BEpigenetics%25253A%252BThe%252BImpact%252Bof%252BChemical%252Band%252BChemical%252BBiology%252BTechniques%252Bon%252BBromodomain%252BTarget%252BValidation%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26volume%3D58%26issue%3D50%26spage%3D17930%26epage%3D17952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea G.  Cochran</span>, <span class="hlFld-ContribAuthor ">Andrew R.  Conery</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sims</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: a new target class forÂ drug development. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2019,</strong> <em>18 </em>
                                    (8)
                                     , 609-628. <a href="https://doi.org/10.1038/s41573-019-0030-7" title="DOI URL">https://doi.org/10.1038/s41573-019-0030-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DBromodomains%25253A%252Ba%252Bnew%252Btarget%252Bclass%252Bfor%2525C2%2525A0drug%252Bdevelopment%26aulast%3DCochran%26aufirst%3DAndrea%2BG.%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D8%26spage%3D609%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael A.  Clegg</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Advancements in the Development of nonâBET Bromodomain Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (4)
                                     , 362-385. <a href="https://doi.org/10.1002/cmdc.201800738" title="DOI URL">https://doi.org/10.1002/cmdc.201800738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DAdvancements%252Bin%252Bthe%252BDevelopment%252Bof%252Bnon%2525E2%252580%252590BET%252BBromodomain%252BChemical%252BProbes%26aulast%3DClegg%26aufirst%3DMichael%2BA.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D4%26spage%3D362%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amy  Donner</span>, <span class="hlFld-ContribAuthor ">Heather  King</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Moses  Moustakim</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 133-152. <a href="https://doi.org/10.1002/9783527809257.ch6" title="DOI URL">https://doi.org/10.1002/9783527809257.ch6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch6%26sid%3Dliteratum%253Aachs%26atitle%3DChemical%252BProbes%26aulast%3DDonner%26aufirst%3DAmy%26date%3D2019%26date%3D2019%26spage%3D133%26epage%3D152%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jamie C.  Gay</span>, <span class="hlFld-ContribAuthor ">Brian E.  Eckenroth</span>, <span class="hlFld-ContribAuthor ">Chiara M.  Evans</span>, <span class="hlFld-ContribAuthor ">Cassiano  Langini</span>, <span class="hlFld-ContribAuthor ">Samuel  Carlson</span>, <span class="hlFld-ContribAuthor ">Jonathan T.  Lloyd</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>, <span class="hlFld-ContribAuthor ">Karen C.  Glass</span>. </span><span class="cited-content_cbyCitation_article-title">Disulfide bridge formation influences ligand recognition by the ATAD2 bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2019,</strong> <em>87 </em>
                                    (2)
                                     , 157-167. <a href="https://doi.org/10.1002/prot.25636" title="DOI URL">https://doi.org/10.1002/prot.25636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.25636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.25636%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DDisulfide%252Bbridge%252Bformation%252Binfluences%252Bligand%252Brecognition%252Bby%252Bthe%252BATAD2%252Bbromodomain%26aulast%3DGay%26aufirst%3DJamie%2BC.%26date%3D2019%26date%3D2018%26volume%3D87%26issue%3D2%26spage%3D157%26epage%3D167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nairong  Liu</span>, <span class="hlFld-ContribAuthor ">Kohei  Funasaka</span>, <span class="hlFld-ContribAuthor ">Tomohiko  Obayashi</span>, <span class="hlFld-ContribAuthor ">Ryoji  Miyahara</span>, <span class="hlFld-ContribAuthor ">Yoshiki  Hirooka</span>, <span class="hlFld-ContribAuthor ">Hidemi  Goto</span>, <span class="hlFld-ContribAuthor ">Takeshi  Senga</span>. </span><span class="cited-content_cbyCitation_article-title">ATAD2 is associated with malignant characteristics of pancreatic cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2019,</strong> <em> </em><a href="https://doi.org/10.3892/ol.2019.9960" title="DOI URL">https://doi.org/10.3892/ol.2019.9960</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2019.9960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2019.9960%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DATAD2%252Bis%252Bassociated%252Bwith%252Bmalignant%252Bcharacteristics%252Bof%252Bpancreatic%252Bcancer%252Bcells%26aulast%3DLiu%26aufirst%3DNairong%26date%3D2019%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan T.  Lloyd</span>, <span class="hlFld-ContribAuthor ">Karen C.  Glass</span>. </span><span class="cited-content_cbyCitation_article-title">Biological function and histone recognition of family IV bromodomain-containing proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Physiology</span><span> <strong>2018,</strong> <em>233 </em>
                                    (3)
                                     , 1877-1886. <a href="https://doi.org/10.1002/jcp.26010" title="DOI URL">https://doi.org/10.1002/jcp.26010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcp.26010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcp.26010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Physiology%26atitle%3DBiological%252Bfunction%252Band%252Bhistone%252Brecognition%252Bof%252Bfamily%252BIV%252Bbromodomain-containing%252Bproteins%26aulast%3DLloyd%26aufirst%3DJonathan%2BT.%26date%3D2018%26date%3D2017%26volume%3D233%26issue%3D3%26spage%3D1877%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muzammal  Hussain</span>, <span class="hlFld-ContribAuthor ">Yang  Zhou</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">H.M. Adnan  Hameed</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Yaoquan  Tu</span>, <span class="hlFld-ContribAuthor ">Jiancun  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">ATAD2 in cancer: a pharmacologically challenging but tractable target. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2018,</strong> <em>22 </em>
                                    (1)
                                     , 85-96. <a href="https://doi.org/10.1080/14728222.2018.1406921" title="DOI URL">https://doi.org/10.1080/14728222.2018.1406921</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DATAD2%252Bin%252Bcancer%25253A%252Ba%252Bpharmacologically%252Bchallenging%252Bbut%252Btractable%252Btarget%26aulast%3DHussain%26aufirst%3DMuzammal%26date%3D2018%26date%3D2017%26volume%3D22%26issue%3D1%26spage%3D85%26epage%3D96" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhenyuan  Miao</span>, <span class="hlFld-ContribAuthor ">Xianghong  Guan</span>, <span class="hlFld-ContribAuthor ">Jiewei  Jiang</span>, <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span>. </span><span class="cited-content_cbyCitation_article-title">BRDT Inhibitors for Male Contraceptive Drug Discovery: Current Status. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 287-315. <a href="https://doi.org/10.1007/978-981-13-0773-7_11" title="DOI URL">https://doi.org/10.1007/978-981-13-0773-7_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-0773-7_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-0773-7_11%26sid%3Dliteratum%253Aachs%26atitle%3DBRDT%252BInhibitors%252Bfor%252BMale%252BContraceptive%252BDrug%252BDiscovery%25253A%252BCurrent%252BStatus%26aulast%3DMiao%26aufirst%3DZhenyuan%26date%3D2018%26date%3D2018%26spage%3D287%26epage%3D315%26pub%3DSpringer%2520Singapore%26atitle%3DTargeting%252BProtein-Protein%252BInteractions%252Bby%252BSmall%252BMolecules%26aulast%3DSheng%26aufirst%3DChunquan%26date%3D2018%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. C.  Runcie</span>, <span class="hlFld-ContribAuthor ">M.  Zengerle</span>, <span class="hlFld-ContribAuthor ">K.-H.  Chan</span>, <span class="hlFld-ContribAuthor ">A.  Testa</span>, <span class="hlFld-ContribAuthor ">L.  van Beurden</span>, <span class="hlFld-ContribAuthor ">M. G. J.  Baud</span>, <span class="hlFld-ContribAuthor ">O.  Epemolu</span>, <span class="hlFld-ContribAuthor ">L. C. J.  Ellis</span>, <span class="hlFld-ContribAuthor ">K. D.  Read</span>, <span class="hlFld-ContribAuthor ">V.  Coulthard</span>, <span class="hlFld-ContribAuthor ">A.  Brien</span>, <span class="hlFld-ContribAuthor ">A.  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of a âbump-and-holeâ approach to allele-selective BET bromodomain inhibition. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2018,</strong> <em>9 </em>
                                    (9)
                                     , 2452-2468. <a href="https://doi.org/10.1039/C7SC02536J" title="DOI URL">https://doi.org/10.1039/C7SC02536J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7SC02536J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7SC02536J%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DOptimization%252Bof%252Ba%252B%2525E2%252580%25259Cbump-and-hole%2525E2%252580%25259D%252Bapproach%252Bto%252Ballele-selective%252BBET%252Bbromodomain%252Binhibition%26aulast%3DRuncie%26aufirst%3DA.%2BC.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D9%26spage%3D2452%26epage%3D2468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yang  Zhou</span>, <span class="hlFld-ContribAuthor ">Muzammal  Hussain</span>, <span class="hlFld-ContribAuthor ">Guanglin  Kuang</span>, <span class="hlFld-ContribAuthor ">Jiancun  Zhang</span>, <span class="hlFld-ContribAuthor ">Yaoquan  Tu</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanistic insights into peptide and ligand binding of the ATAD2-bromodomain
              via
              atomistic simulations disclosing a role of induced fit and conformational selection. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (36)
                                     , 23222-23232. <a href="https://doi.org/10.1039/C8CP03860K" title="DOI URL">https://doi.org/10.1039/C8CP03860K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP03860K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP03860K%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DMechanistic%252Binsights%252Binto%252Bpeptide%252Band%252Bligand%252Bbinding%252Bof%252Bthe%252BATAD2-bromodomain%252Bvia%252Batomistic%252Bsimulations%252Bdisclosing%252Ba%252Brole%252Bof%252Binduced%252Bfit%252Band%252Bconformational%252Bselection%26aulast%3DZhou%26aufirst%3DYang%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D36%26spage%3D23222%26epage%3D23232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duncan C.  Miller</span>, <span class="hlFld-ContribAuthor ">Mathew P.  Martin</span>, <span class="hlFld-ContribAuthor ">Santosh  Adhikari</span>, <span class="hlFld-ContribAuthor ">Alfie  Brennan</span>, <span class="hlFld-ContribAuthor ">Jane A.  Endicott</span>, <span class="hlFld-ContribAuthor ">Bernard T.  Golding</span>, <span class="hlFld-ContribAuthor ">Ian R.  Hardcastle</span>, <span class="hlFld-ContribAuthor ">Amy  Heptinstall</span>, <span class="hlFld-ContribAuthor ">Stephen  Hobson</span>, <span class="hlFld-ContribAuthor ">Claire  Jennings</span>, <span class="hlFld-ContribAuthor ">Lauren  Molyneux</span>, <span class="hlFld-ContribAuthor ">Yvonne  Ng</span>, <span class="hlFld-ContribAuthor ">Stephen R.  Wedge</span>, <span class="hlFld-ContribAuthor ">Martin E. M.  Noble</span>, <span class="hlFld-ContribAuthor ">Celine  Cano</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2018,</strong> <em>16 </em>
                                    (11)
                                     , 1843-1850. <a href="https://doi.org/10.1039/C8OB00099A" title="DOI URL">https://doi.org/10.1039/C8OB00099A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8OB00099A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8OB00099A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DIdentification%252Bof%252Ba%252Bnovel%252Bligand%252Bfor%252Bthe%252BATAD2%252Bbromodomain%252Bwith%252Bselectivity%252Bover%252BBRD4%252Bthrough%252Ba%252Bfragment%252Bgrowing%252Bapproach%26aulast%3DMiller%26aufirst%3DDuncan%2BC.%26date%3D2018%26date%3D2018%26volume%3D16%26issue%3D11%26spage%3D1843%26epage%3D1850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">B.  Sepehri</span>, <span class="hlFld-ContribAuthor ">R.  Ghavami</span>. </span><span class="cited-content_cbyCitation_article-title">The identification of new ATAD2 bromodomain inhibitors: the application of combined ligand and structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2017,</strong> <em>28 </em>
                                    (12)
                                     , 957-971. <a href="https://doi.org/10.1080/1062936X.2017.1385532" title="DOI URL">https://doi.org/10.1080/1062936X.2017.1385532</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1062936X.2017.1385532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1062936X.2017.1385532%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DThe%252Bidentification%252Bof%252Bnew%252BATAD2%252Bbromodomain%252Binhibitors%25253A%252Bthe%252Bapplication%252Bof%252Bcombined%252Bligand%252Band%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DSepehri%26aufirst%3DB.%26date%3D2017%26date%3D2017%26volume%3D28%26issue%3D12%26spage%3D957%26epage%3D971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ghasem Firouzzade  Pasha</span>, <span class="hlFld-ContribAuthor ">Sakineh  Asghari</span>, <span class="hlFld-ContribAuthor ">Mahmoud  Tajbakhsh</span>, <span class="hlFld-ContribAuthor ">Mojtaba  Mohseni</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and characterization of sulfide, sulfoxide and sulfone derivatives of thiopyran: antimicrobial evaluation. </span><span class="cited-content_cbyCitation_journal-name">Research on Chemical Intermediates</span><span> <strong>2017,</strong> <em>43 </em>
                                    (12)
                                     , 7291-7306. <a href="https://doi.org/10.1007/s11164-017-3075-4" title="DOI URL">https://doi.org/10.1007/s11164-017-3075-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11164-017-3075-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11164-017-3075-4%26sid%3Dliteratum%253Aachs%26jtitle%3DResearch%2520on%2520Chemical%2520Intermediates%26atitle%3DSynthesis%252Band%252Bcharacterization%252Bof%252Bsulfide%25252C%252Bsulfoxide%252Band%252Bsulfone%252Bderivatives%252Bof%252Bthiopyran%25253A%252Bantimicrobial%252Bevaluation%26aulast%3DPasha%26aufirst%3DGhasem%2BFirouzzade%26date%3D2017%26date%3D2017%26volume%3D43%26issue%3D12%26spage%3D7291%26epage%3D7306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, , <span class="hlFld-ContribAuthor ">AmandaÂ J.  Price</span>, <span class="hlFld-ContribAuthor ">Steven  Howard</span>, <span class="hlFld-ContribAuthor ">BenjaminÂ D.  Cons</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based drug discovery and its application to challenging drug targets. </span><span class="cited-content_cbyCitation_journal-name">Essays in Biochemistry</span><span> <strong>2017,</strong> <em>61 </em>
                                    (5)
                                     , 475-484. <a href="https://doi.org/10.1042/EBC20170029" title="DOI URL">https://doi.org/10.1042/EBC20170029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/EBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FEBC20170029%26sid%3Dliteratum%253Aachs%26jtitle%3DEssays%2520in%2520Biochemistry%26atitle%3DFragment-based%252Bdrug%252Bdiscovery%252Band%252Bits%252Bapplication%252Bto%252Bchallenging%252Bdrug%252Btargets%26aulast%3Dvan%2BMontfort%26aufirst%3DRob%2BL.M.%26date%3D2017%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D475%26epage%3D484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hermann AM  Mucke</span>. </span><span class="cited-content_cbyCitation_article-title">Patent Highlights JuneâJuly 2017. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Patent Analyst</span><span> <strong>2017,</strong> <em>6 </em>
                                    (6)
                                     , 259-266. <a href="https://doi.org/10.4155/ppa-2017-0034" title="DOI URL">https://doi.org/10.4155/ppa-2017-0034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/ppa-2017-0034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Fppa-2017-0034%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Patent%2520Analyst%26atitle%3DPatent%252BHighlights%252BJune%2525E2%252580%252593July%252B2017%26aulast%3DMucke%26aufirst%3DHermann%2BAM%26date%3D2017%26volume%3D6%26issue%3D6%26spage%3D259%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoxiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Kai  Chen</span>, <span class="hlFld-ContribAuthor ">Yun-Dong  Wu</span>, <span class="hlFld-ContribAuthor ">Olaf  Wiest</span>, . </span><span class="cited-content_cbyCitation_article-title">Protein dynamics and structural waters in bromodomains. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (10)
                                     , e0186570. <a href="https://doi.org/10.1371/journal.pone.0186570" title="DOI URL">https://doi.org/10.1371/journal.pone.0186570</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DProtein%252Bdynamics%252Band%252Bstructural%252Bwaters%252Bin%252Bbromodomains%26aulast%3DZhang%26aufirst%3DXiaoxiao%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D10%26spage%3De0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cassiano  Langini</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>, <span class="hlFld-ContribAuthor ">Andreas  Vitalis</span>. </span><span class="cited-content_cbyCitation_article-title">The ATAD2 bromodomain binds different acetylation marks on the histone H4 in similar fuzzy complexes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2017,</strong> <em>292 </em>
                                    (40)
                                     , 16734-16745. <a href="https://doi.org/10.1074/jbc.M117.786350" title="DOI URL">https://doi.org/10.1074/jbc.M117.786350</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M117.786350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M117.786350%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DThe%252BATAD2%252Bbromodomain%252Bbinds%252Bdifferent%252Bacetylation%252Bmarks%252Bon%252Bthe%252Bhistone%252BH4%252Bin%252Bsimilar%252Bfuzzy%252Bcomplexes%26aulast%3DLangini%26aufirst%3DCassiano%26date%3D2017%26volume%3D292%26issue%3D40%26spage%3D16734%26epage%3D16745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abhinav K  Jain</span>, <span class="hlFld-ContribAuthor ">Michelle C  Barton</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain Histone Readers and Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2017,</strong> <em>429 </em>
                                    (13)
                                     , 2003-2010. <a href="https://doi.org/10.1016/j.jmb.2016.11.020" title="DOI URL">https://doi.org/10.1016/j.jmb.2016.11.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2016.11.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2016.11.020%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DBromodomain%252BHistone%252BReaders%252Band%252BCancer%26aulast%3DJain%26aufirst%3DAbhinav%2BK%26date%3D2017%26volume%3D429%26issue%3D13%26spage%3D2003%26epage%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Montserrat  PÃ©rez-Salvia</span>, <span class="hlFld-ContribAuthor ">Manel  Esteller</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain inhibitors and cancer therapy: From structures to applications. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 323-339. <a href="https://doi.org/10.1080/15592294.2016.1265710" title="DOI URL">https://doi.org/10.1080/15592294.2016.1265710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DBromodomain%252Binhibitors%252Band%252Bcancer%252Btherapy%25253A%252BFrom%252Bstructures%252Bto%252Bapplications%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DMontserrat%26date%3D2017%26date%3D2016%26volume%3D12%26issue%3D5%26spage%3D323%26epage%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bakhtyar  Sepehri</span>, <span class="hlFld-ContribAuthor ">Zolaikha  Rasouli</span>, <span class="hlFld-ContribAuthor ">Zeinabe  Hassanzadeh</span>, <span class="hlFld-ContribAuthor ">Raouf  Ghavami</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking and QSAR analysis of naphthyridone derivatives as ATAD2 bromodomain inhibitors: application of CoMFA, LS-SVM, and RBF neural network. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2016,</strong> <em>25 </em>
                                    (12)
                                     , 2895-2905. <a href="https://doi.org/10.1007/s00044-016-1686-8" title="DOI URL">https://doi.org/10.1007/s00044-016-1686-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-016-1686-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-016-1686-8%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DMolecular%252Bdocking%252Band%252BQSAR%252Banalysis%252Bof%252Bnaphthyridone%252Bderivatives%252Bas%252BATAD2%252Bbromodomain%252Binhibitors%25253A%252Bapplication%252Bof%252BCoMFA%25252C%252BLS-SVM%25252C%252Band%252BRBF%252Bneural%252Bnetwork%26aulast%3DSepehri%26aufirst%3DBakhtyar%26date%3D2016%26date%3D2016%26volume%3D25%26issue%3D12%26spage%3D2895%26epage%3D2905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Weigt</span>, <span class="hlFld-ContribAuthor ">Carsten  Hopf</span>, <span class="hlFld-ContribAuthor ">Guillaume  MÃ©dard</span>. </span><span class="cited-content_cbyCitation_article-title">Studying epigenetic complexes and their inhibitors with the proteomics toolbox. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-016-0244-z" title="DOI URL">https://doi.org/10.1186/s13148-016-0244-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-016-0244-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-016-0244-z%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DStudying%252Bepigenetic%252Bcomplexes%252Band%252Btheir%252Binhibitors%252Bwith%252Bthe%252Bproteomics%252Btoolbox%26aulast%3DWeigt%26aufirst%3DDavid%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Bamborough</span>, <span class="hlFld-ContribAuthor ">Chun-wa  Chung</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Furze</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Bannister</span>, <span class="hlFld-ContribAuthor ">Ka Hing  Che</span>, <span class="hlFld-ContribAuthor ">Hawa  Diallo</span>, <span class="hlFld-ContribAuthor ">Clement  Douault</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>, <span class="hlFld-ContribAuthor ">Tony  Kouzarides</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Michon</span>, <span class="hlFld-ContribAuthor ">Darren J.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Christina  Rau</span>, <span class="hlFld-ContribAuthor ">Samuel  Robson</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sheppard</span>, <span class="hlFld-ContribAuthor ">Richard  Upton</span>, <span class="hlFld-ContribAuthor ">Robert J.  Watson</span>. </span><span class="cited-content_cbyCitation_article-title">A Chemical Probe for the ATAD2 Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2016,</strong> <em>128 </em>
                                    (38)
                                     , 11554-11558. <a href="https://doi.org/10.1002/ange.201603928" title="DOI URL">https://doi.org/10.1002/ange.201603928</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201603928%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DA%252BChemical%252BProbe%252Bfor%252Bthe%252BATAD2%252BBromodomain%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2016%26date%3D2016%26volume%3D128%26issue%3D38%26spage%3D11554%26epage%3D11558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Bamborough</span>, <span class="hlFld-ContribAuthor ">Chun-wa  Chung</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Furze</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Bannister</span>, <span class="hlFld-ContribAuthor ">Ka Hing  Che</span>, <span class="hlFld-ContribAuthor ">Hawa  Diallo</span>, <span class="hlFld-ContribAuthor ">Clement  Douault</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>, <span class="hlFld-ContribAuthor ">Tony  Kouzarides</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Michon</span>, <span class="hlFld-ContribAuthor ">Darren J.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Christina  Rau</span>, <span class="hlFld-ContribAuthor ">Samuel  Robson</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sheppard</span>, <span class="hlFld-ContribAuthor ">Richard  Upton</span>, <span class="hlFld-ContribAuthor ">Robert J.  Watson</span>. </span><span class="cited-content_cbyCitation_article-title">A Chemical Probe for the ATAD2 Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2016,</strong> <em>55 </em>
                                    (38)
                                     , 11382-11386. <a href="https://doi.org/10.1002/anie.201603928" title="DOI URL">https://doi.org/10.1002/anie.201603928</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201603928%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DA%252BChemical%252BProbe%252Bfor%252Bthe%252BATAD2%252BBromodomain%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2016%26date%3D2016%26volume%3D55%26issue%3D38%26spage%3D11382%26epage%3D11386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carles  Galdeano</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1655-1680. <a href="https://doi.org/10.4155/fmc-2016-0059" title="DOI URL">https://doi.org/10.4155/fmc-2016-0059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSelectivity%252Bon-target%252Bof%252Bbromodomain%252Bchemical%252Bprobes%252Bby%252Bstructure-guided%252Bmedicinal%252Bchemistry%252Band%252Bchemical%252Bbiology%26aulast%3DGaldeano%26aufirst%3DCarles%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1655%26epage%3D1680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">R  Kumar</span>, <span class="hlFld-ContribAuthor ">D-Q  Li</span>, <span class="hlFld-ContribAuthor ">S  MÃ¼ller</span>, <span class="hlFld-ContribAuthor ">S  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenomic regulation of oncogenesis by chromatin remodeling. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2016,</strong> <em>35 </em>
                                    (34)
                                     , 4423-4436. <a href="https://doi.org/10.1038/onc.2015.513" title="DOI URL">https://doi.org/10.1038/onc.2015.513</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/onc.2015.513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fonc.2015.513%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DEpigenomic%252Bregulation%252Bof%252Boncogenesis%252Bby%252Bchromatin%252Bremodeling%26aulast%3DKumar%26aufirst%3DR%26date%3D2016%26date%3D2016%26volume%3D35%26issue%3D34%26spage%3D4423%26epage%3D4436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathy A.  Gelato</span>, <span class="hlFld-ContribAuthor ">Zaki  Shaikhibrahim</span>, <span class="hlFld-ContribAuthor ">Matthias  Ocker</span>, <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic regulators for cancer therapy: modulation of bromodomain proteins, methyltransferases, demethylases, and microRNAs. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2016,</strong> <em>20 </em>
                                    (7)
                                     , 783-799. <a href="https://doi.org/10.1517/14728222.2016.1134490" title="DOI URL">https://doi.org/10.1517/14728222.2016.1134490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/14728222.2016.1134490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F14728222.2016.1134490%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DTargeting%252Bepigenetic%252Bregulators%252Bfor%252Bcancer%252Btherapy%25253A%252Bmodulation%252Bof%252Bbromodomain%252Bproteins%25252C%252Bmethyltransferases%25252C%252Bdemethylases%25252C%252Band%252BmicroRNAs%26aulast%3DGelato%26aufirst%3DKathy%2BA.%26date%3D2016%26date%3D2016%26volume%3D20%26issue%3D7%26spage%3D783%26epage%3D799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rongsheng  Ma</span>, <span class="hlFld-ContribAuthor ">Pengchao  Wang</span>, <span class="hlFld-ContribAuthor ">Jihui  Wu</span>, <span class="hlFld-ContribAuthor ">Ke  Ruan</span>. </span><span class="cited-content_cbyCitation_article-title">Process of Fragment-Based Lead DiscoveryâA Perspective from NMR. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2016,</strong> <em>21 </em>
                                    (7)
                                     , 854. <a href="https://doi.org/10.3390/molecules21070854" title="DOI URL">https://doi.org/10.3390/molecules21070854</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules21070854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules21070854%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DProcess%252Bof%252BFragment-Based%252BLead%252BDiscovery%2525E2%252580%252594A%252BPerspective%252Bfrom%252BNMR%26aulast%3DMa%26aufirst%3DRongsheng%26date%3D2016%26date%3D2016%26volume%3D21%26issue%3D7%26spage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paolo  Filipponi</span>, <span class="hlFld-ContribAuthor ">Ian R.  Baxendale</span>. </span><span class="cited-content_cbyCitation_article-title">The Generation of a Library of Â­Bromodomain-Containing Protein Modulators Expedited by Continuous Flow Synthesis. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>2016 </em>
                                    (11)
                                     , 2000-2012. <a href="https://doi.org/10.1002/ejoc.201600222" title="DOI URL">https://doi.org/10.1002/ejoc.201600222</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201600222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201600222%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DThe%252BGeneration%252Bof%252Ba%252BLibrary%252Bof%252B%2525C2%2525ADBromodomain-Containing%252BProtein%252BModulators%252BExpedited%252Bby%252BContinuous%252BFlow%252BSynthesis%26aulast%3DFilipponi%26aufirst%3DPaolo%26date%3D2016%26date%3D2016%26volume%3D2016%26issue%3D11%26spage%3D2000%26epage%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie H.  Theodoulou</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in the Development of non-BET Bromodomain Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (5)
                                     , 477-487. <a href="https://doi.org/10.1002/cmdc.201500540" title="DOI URL">https://doi.org/10.1002/cmdc.201500540</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DProgress%252Bin%252Bthe%252BDevelopment%252Bof%252Bnon-BET%252BBromodomain%252BChemical%252BProbes%26aulast%3DTheodoulou%26aufirst%3DNatalie%2BH.%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D477%26epage%3D487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jamel  Meslamani</span>, <span class="hlFld-ContribAuthor ">Steven G.  Smith</span>, <span class="hlFld-ContribAuthor ">Roberto  Sanchez</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Structural features and inhibitors of bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2016,</strong> <em>19 </em>, 3-15. <a href="https://doi.org/10.1016/j.ddtec.2016.09.001" title="DOI URL">https://doi.org/10.1016/j.ddtec.2016.09.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2016.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2016.09.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DStructural%252Bfeatures%252Band%252Binhibitors%252Bof%252Bbromodomains%26aulast%3DMeslamani%26aufirst%3DJamel%26date%3D2016%26volume%3D19%26spage%3D3%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moses  Moustakim</span>, <span class="hlFld-ContribAuthor ">Peter G. K.  Clark</span>, <span class="hlFld-ContribAuthor ">Duncan A.  Hay</span>, <span class="hlFld-ContribAuthor ">Darren J.  Dixon</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical probes and inhibitors of bromodomains outside the BET family. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2016,</strong> <em>7 </em>
                                    (12)
                                     , 2246-2264. <a href="https://doi.org/10.1039/C6MD00373G" title="DOI URL">https://doi.org/10.1039/C6MD00373G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00373G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00373G%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DChemical%252Bprobes%252Band%252Binhibitors%252Bof%252Bbromodomains%252Boutside%252Bthe%252BBET%252Bfamily%26aulast%3DMoustakim%26aufirst%3DMoses%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D12%26spage%3D2246%26epage%3D2264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. From hit to micromolar ATAD2 inhibitor. pIC<sub>50</sub>s of compounds <b>1</b>â<b>3</b> in ATAD2 peptide- and ligand-based TR-FRET assays and against BRD4 BD1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Acetyl-lysine binding site of BRD4 BD1 bound to literature inhibitor I-BET762 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p5o">3p5o</a>)<sup>10</sup> showing detailed interactions (left) and electrostatic charge colored protein surface (right). (B) X-ray structure of ATAD2 bound to <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q">5a5q</a>), in the same orientation. Hydrogen bonds are shown as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Key Trans Piperidines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaN<sub>3</sub>, DMF/acetone/water, 80Â°C, 63%; (b) NaH, DMF, 0Â°C then RI or ROTf, 0 Â°C to room temperature, 10â100%; (c) Pd/C, MeOH, H<sub>2</sub> (1 bar), 69â100% or PPh<sub>3</sub>, THF, room temperature then water, reflux, 69â79%; (d) for R<sub>2</sub> = H: LiHMDS, Caddick catalyst, THF, 80â120 Â°C, microwave, 13â95%; or for R<sub>2</sub> = Bn: BrettPhos, BrettPhos precatalyst, THF, <i>t</i>BuONa, room temperature, 65â95%; (e) for R<sub>2</sub> = H: TFA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 73â92%; (f) for R<sub>2</sub> = Bn: H<sub>2</sub>, Pd/C, MeOH, room temperature, 99% then TFA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 84%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Crystal structure of ATAD2 bound to <b>13</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a81">5a81</a>), superimposed on that of <b>2</b> (orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q">5a5q</a>) using alpha-carbon atoms of the active sites. The water molecule labeled H<sub>2</sub>O<sub>(R1077)</sub> on the RVF shelf close to Arg1077 is present in the complex with <b>2</b> and is displaced by <b>13</b>. (B) Crystal structure of BRD4 BD1 bound to <b>13</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a85">5a85</a>). Dashed yellow lines indicate hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, CHCl<sub>3</sub>, â10 Â°C, 84â95%; (b) Pd(OAc)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, cataCXium A, ArB(OH)<sub>2</sub>, 1,4-dioxane/water, 100 Â°C, microwave, 46â87%; (c), TFA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature (R<sub>1</sub> = H, 26â90%) or TFA reflux (R<sub>1</sub> = Bn, 39â99%).</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) TBDMSCl, imidazole, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 71%; (b) H<sub>2</sub>, Pd/C, EtOH, room temperature, 95%; (c) BrettPhos, BrettPhos precatalyst, <i>t</i>BuONa, THF, room temperature to 60 Â°C, 97%; (d) TBAF, THF, room temperature, 71%; (e) chiral separation of enantiomers; (f) ROTf, <i>t</i>BuOK, THF, 0 Â°C to room temperature, 10â65% or NaH, THF (or DMF), 0 Â°C then ROTf, 0 Â°C to room temperature, 37â65%; (g) TFA, 60 Â°C to reflux, 45â94%.</p></p></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (AâC) Three different views of the X-ray structure of ATAD2 bound to <b>30</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a82">5a82</a>), superimposed on <b>2</b> (orange, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q">5a5q</a>). In (C), a water molecule H<sub>2</sub>O<sub>(R1077)</sub> is shown. This lies near Arg1077 in the complex with <b>2</b> and is displaced by <b>30</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A, B) Two orientations of the ATAD2 X-ray complex with <b>42</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a83">5a83</a>), superimposed on the structure of <b>30</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a82">5a82</a>). The position of the mobile side chain of Arg1007 in the complex with <b>30</b> is shown in magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selectivity of <b>46</b> against other bromodomains. The color scale ranges from p<i>K</i><sub>i</sub> 7.7 (red) to <4.3 (blue, triangles denote â<â modifier).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/medium/jm-2015-00773g_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Selectivity of (A) <b>20</b>, (B) <b>43</b>, and (C) <b>46</b> for ATAD2 against other bromodomain containing proteins using MS-detected pull downs from cell lysates. Specified values are apparent p<i>K</i><sub>d</sub>s. Values <4 indicate that no p<i>K</i><sub>d</sub> was determined up to 100 Î¼M compound concentration. No bar indicates that protein was not identified.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-15/acs.jmedchem.5b00773/20150807/images/large/jm-2015-00773g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00773&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i70">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01328" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01328" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 31 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Caron, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lestrat, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escoffier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Virolle, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbry, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debernardi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousseaux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khochbin, S.</span><span> </span><span class="NLM_article-title">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">5171</span><span class="NLM_x">â</span> <span class="NLM_lpage">5181</span><span class="refDoi">Â DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+characterization+of+ATAD2+as+a+new+cancer%2Ftestis+factor+and+a+predictor+of+poor+prognosis+in+breast+and+lung+cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0lhY3Fy_digTgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520characterization%2520of%2520ATAD2%2520as%2520a%2520new%2520cancer%252Ftestis%2520factor%2520and%2520a%2520predictor%2520of%2520poor%2520prognosis%2520in%2520breast%2520and%2520lung%2520cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Boussouar, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamshidikia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morozumi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rousseaux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khochbin, S.</span><span> </span><span class="NLM_article-title">Malignant genome reprogramming by ATAD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1829</span><span class="NLM_x">, </span> <span class="NLM_fpage">1010</span><span class="NLM_x">â</span> <span class="NLM_lpage">1014</span><span class="refDoi">Â DOI: 10.1016/j.bbagrm.2013.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1016%2Fj.bbagrm.2013.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=23831842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=1010-1014&author=F.+Boussouarauthor=M.+Jamshidikiaauthor=Y.+Morozumiauthor=S.+Rousseauxauthor=S.+Khochbin&title=Malignant+genome+reprogramming+by+ATAD2&doi=10.1016%2Fj.bbagrm.2013.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Malignant genome reprogramming by ATAD2</span></div><div class="casAuthors">Boussouar, Faycal; Jamshidikia, Mahya; Morozumi, Yuichi; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1829</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Unscheduled expression of crit. cellular regulators could be central to malignant genome reprogramming and tumor establishment.  One such factor appears to be ATAD2, a remarkably conserved protein normally predominantly expressed in germ cells but almost systematically over-expressed in a variety of unrelated cancers.  The presence of a bromodomain adjacent to an AAA type ATPase domain, points to ATAD2 as a factor preliminarily acting on chromatin structure and function.  Accordingly, ATAD2 has been shown to cooperate with a series of transcription factors and chromatin modifiers to regulate specific set of genes.  Here we discuss our knowledge on ATAD2 to evaluate its role as a cancer driver and its value as a new anti-cancer target.  Upon its activation, ATAD2 through its interaction with defined transcription factors, initiates a loop of transcriptional stimulation of target genes, including ATAD2 itself, leading to enhanced cell proliferation and resistance to apoptosis in an ATAD2-dependent manner.  Approaches aiming at neutralizing ATAD2 activity in cancer, including the use of small mol. inhibitors of its two "druggable" domains, AAA ATPase and bromodomain, could become part of a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp824JMo6Dp4rVg90H21EOLACvtfcHk0lhY3Fy_digTgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF&md5=cba5991a2dc5f96f522e1104ae4e4889</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2013.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2013.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DMalignant%2520genome%2520reprogramming%2520by%2520ATAD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2013%26volume%3D1829%26spage%3D1010%26epage%3D1014%26doi%3D10.1016%2Fj.bbagrm.2013.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Zou, J. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revenko, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tepper, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gemo, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H. W.</span><span> </span><span class="NLM_article-title">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">3339</span><span class="NLM_x">â</span> <span class="NLM_lpage">3346</span><span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-08-3440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1158%2F0008-5472.CAN-08-3440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=19318566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=3339-3346&author=J.+X.+Zouauthor=L.+Guoauthor=A.+S.+Revenkoauthor=C.+G.+Tepperauthor=A.+T.+Gemoauthor=H.+J.+Kungauthor=H.+W.+Chen&title=Androgen-induced+coactivator+ANCCA+mediates+specific+androgen+receptor+signaling+in+prostate+cancer&doi=10.1158%2F0008-5472.CAN-08-3440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span></div><div class="casAuthors">Zou, June X.; Guo, Linlang; Revenko, Alexey S.; Tepper, Clifford G.; Gemo, Abigael T.; Kung, Hsing-Jien; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3339-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Androgen receptor (AR) plays a pivotal role in prostate cancer, primarily by regulating different gene expression programs elicited by androgen, which is important for cancer cell proliferation, survival, and differentiation.  It is believed that the transcriptional function of AR is mediated largely by distinct nuclear coregulators.  We report here the identification of ANCCA (also known as ATAD2), a new member of the AAA+ ATPase family proteins, as a novel AR coactivator.  ANCCA interacts directly with AR and enhances its transcriptional activity, and is required for androgen-stimulated expression of a specific subgroup of genes including IGF1R, IRS-2, SGK1, and survivin.  Upon androgen stimulation, ANCCA together with AR is recruited to the specific AR target genes.  Suppression of ANCCA expression strongly inhibited the proliferation of androgen-responsive or androgen-independent, AR-pos. prostate cancer cells and caused a significant increase of cellular apoptosis.  Strikingly, the ANCCA gene itself, located at chromosome 8q24, is highly induced by androgen in androgen-dependent prostate cancer cells and xenograft tumors.  Although ANCCA is hardly detected in normal human prostate tissue, high levels of ANCCA are found in hormone-independent prostate cancer cell lines, xenograft tumor, and a subset of prostate cancers with high Gleason scores.  Together, these findings suggest that ANCCA plays an important role in prostate cancer by mediating specific AR functions in cancer cell survival and proliferation.  The possession of ATPase and bromodomain by ANCCA makes it an attractive target for the development of therapeutics for the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtieC3cr_jX7Vg90H21EOLACvtfcHk0lhY3Fy_digTgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D&md5=daa42208c9c7019809bb491e5af1f152</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3440%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DAndrogen-induced%2520coactivator%2520ANCCA%2520mediates%2520specific%2520androgen%2520receptor%2520signaling%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D3339%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-08-3440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J.</span><span> </span><span class="NLM_article-title">Targeting epigenetic readers in cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">367</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">â</span> <span class="NLM_lpage">657</span><span class="refDoi">Â DOI: 10.1056/NEJMra1112635</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1056%2FNEJMra1112635" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=22894577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2012&pages=647-657&author=M.+A.+Dawsonauthor=T.+Kouzaridesauthor=B.+J.+Huntly&title=Targeting+epigenetic+readers+in+cancer&doi=10.1056%2FNEJMra1112635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic readers in cancer</span></div><div class="casAuthors">Dawson, Mark A.; Kouzarides, Tony; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">647-657</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on assocn. between epigenetics and chromatin biol., BET bromodomain inhibitors, and role of BET inhibitors in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnz_YQy6Z2T7Vg90H21EOLACvtfcHk0lj7HSTD9B27xQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Chu77F&md5=f5e1e82c8a1105f798b1f7719138721c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1112635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1112635%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26atitle%3DTargeting%2520epigenetic%2520readers%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D367%26spage%3D647%26epage%3D657%26doi%3D10.1056%2FNEJMra1112635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Jennings, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Measures, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Phenotypic screening and fragment-based approaches to the discovery of small-molecule bromodomain ligands</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">179</span><span class="NLM_x">â</span> <span class="NLM_lpage">204</span><span class="refDoi">Â DOI: 10.4155/fmc.13.197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.4155%2Ffmc.13.197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=24467243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCkuro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=179-204&author=L.+E.+Jenningsauthor=A.+R.+Measuresauthor=B.+G.+Wilsonauthor=S.+J.+Conway&title=Phenotypic+screening+and+fragment-based+approaches+to+the+discovery+of+small-molecule+bromodomain+ligands&doi=10.4155%2Ffmc.13.197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening and fragment-based approaches to the discovery of small-molecule bromodomain ligands</span></div><div class="casAuthors">Jennings, Laura E.; Measures, Angelina R.; Wilson, Brian G.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-204</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues and hence facilitate protein-protein interactions.  These bromodomain-mediated interactions often play key roles in transcriptional regulation and their dysfunction is implicated in a large no. of diseases.  The discovery of potent and selective small-mol. bromodomain and extra C-terminal domain bromodomain ligands, which show promising results for the treatment of cancers and atherosclerosis, has promoted intense interest in this area.  Here we describe the progress that has been made to date in the discovery of small-mol. bromodomain ligands, with particular emphasis on the roles played by phenotypic screening and fragment-based approaches.  In considering the future of the field we discuss the prospects for development of mol. probes and drugs for the non-bromodomain and extra C-terminal domain bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6WhUfE_Ad4rVg90H21EOLACvtfcHk0lj7HSTD9B27xQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCkuro%253D&md5=08c3266ceefe824fde8ef7c400d6bc22</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.197%26sid%3Dliteratum%253Aachs%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DMeasures%26aufirst%3DA.%2BR.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DPhenotypic%2520screening%2520and%2520fragment-based%2520approaches%2520to%2520the%2520discovery%2520of%2520small-molecule%2520bromodomain%2520ligands%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D179%26epage%3D204%26doi%3D10.4155%2Ffmc.13.197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Demont, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furze, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wellaway, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bridges, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craggs, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Douault, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmons, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karamshi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouzon, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of low-micromolar ATAD2 bromodomain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5649</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649&author=E.+H.+Demontauthor=C.+Chungauthor=R.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+Emmonsauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-based+discovery+of+low-micromolar+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0lgaSTp1xd7ILw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DFurze%26aufirst%3DR.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520low-micromolar%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5649%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Discovery of BET bromodomain inhibitors and their role in target validation</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">â</span> <span class="NLM_lpage">296</span><span class="refDoi">Â DOI: 10.1039/c3md00291h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1039%2Fc3md00291h" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=288-296&author=S.+Mullerauthor=S.+Knapp&title=Discovery+of+BET+bromodomain+inhibitors+and+their+role+in+target+validation&doi=10.1039%2Fc3md00291h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2Fc3md00291h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00291h%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520BET%2520bromodomain%2520inhibitors%2520and%2520their%2520role%2520in%2520target%2520validation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D288%26epage%3D296%26doi%3D10.1039%2Fc3md00291h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Chung, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosmini, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delves, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magny, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boursier, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouillot, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellibert, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riou, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uings, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clement, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boullay, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blandel, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3827</span><span class="NLM_x">â</span> <span class="NLM_lpage">3838</span><span class="refDoi">Â DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.+M.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Clementauthor=A.+B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lgaSTp1xd7ILw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Gosmini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krysa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riou-Eymard, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boursier, E. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trottet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demont, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaur, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schilling, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">The discovery of I-BET726 (GSK1324726A), a potent tetrahydroquinoline ApoA1 up-regulator and selective BET bromodomain inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8111</span><span class="NLM_x">â</span> <span class="NLM_lpage">8131</span><span class="refDoi">Â DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.+M.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.+W.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=The+discovery+of+I-BET726+%28GSK1324726A%29%2C+a+potent+tetrahydroquinoline+ApoA1+up-regulator+and+selective+BET+bromodomain+inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0liOyldqnK5dbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DThe%2520discovery%2520of%2520I-BET726%2520%2528GSK1324726A%2529%252C%2520a%2520potent%2520tetrahydroquinoline%2520ApoA1%2520up-regulator%2520and%2520selective%2520BET%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">â</span> <span class="NLM_lpage">1123</span><span class="refDoi">Â DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0liOyldqnK5dbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">â</span> <span class="NLM_lpage">1073</span><span class="refDoi">Â DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0liOyldqnK5dbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Poncet-Montange, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardenhagen, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrocchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geck Do, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladbury, P.</span><span> </span><span class="NLM_article-title">J. E. Observed bromodomain flexibility reveals histone peptide- and small molecule ligand-compatible forms of ATAD2A</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">466</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">â</span> <span class="NLM_lpage">346</span><span class="refDoi">Â DOI: 10.1042/BJ20140933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1042%2FBJ20140933" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2015&pages=337-346&author=G.+Poncet-Montangeauthor=Y.+Zhanauthor=J.+P.+Bardenhagenauthor=A.+Petrocchiauthor=E.+Leoauthor=X.+Shiauthor=G.+R.+Leeauthor=P.+G.+Leonardauthor=M.+D.+Geck+Doauthor=M.+K.+Cardozoauthor=M.+G.+Andersenauthor=J.+N.+Palmerauthor=W.+S.+Jonesauthor=P.+Ladbury&title=J.+E.+Observed+bromodomain+flexibility+reveals+histone+peptide-+and+small+molecule+ligand-compatible+forms+of+ATAD2A&doi=10.1042%2FBJ20140933"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1042%2FBJ20140933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20140933%26sid%3Dliteratum%253Aachs%26aulast%3DPoncet-Montange%26aufirst%3DG.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DG.%2BR.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DGeck%2BDo%26aufirst%3DM.%2BD.%26aulast%3DCardozo%26aufirst%3DM.%2BK.%26aulast%3DAndersen%26aufirst%3DM.%2BG.%26aulast%3DPalmer%26aufirst%3DJ.%2BN.%26aulast%3DJones%26aufirst%3DW.%2BS.%26aulast%3DLadbury%26aufirst%3DP.%26atitle%3DJ.%2520E.%2520Observed%2520bromodomain%2520flexibility%2520reveals%2520histone%2520peptide-%2520and%2520small%2520molecule%2520ligand-compatible%2520forms%2520of%2520ATAD2A%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D466%26spage%3D337%26epage%3D346%26doi%3D10.1042%2FBJ20140933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballentine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teleha, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Branum, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patch, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Player, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, W.</span><span> </span><span class="NLM_article-title">Process Development for Scale-Up of a Novel 3, 5-Substituted Thiazolidine-2, 4-dione Compound as a Potent Inhibitor for Estrogen-Related Receptor 1</span> <span class="citation_source-journal">Org. Process Res. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">321</span><span class="NLM_x">â</span> <span class="NLM_lpage">330</span><span class="refDoi">Â DOI: 10.1021/op400325r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/op400325r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=321-330&author=X.+Liauthor=R.+K.+Russellauthor=J.+Spinkauthor=S.+Ballentineauthor=C.+Telehaauthor=S.+Branumauthor=K.+Wellsauthor=D.+Beauchampauthor=R.+Patchauthor=H.+Huangauthor=M.+Playerauthor=W.+Murray&title=Process+Development+for+Scale-Up+of+a+Novel+3%2C+5-Substituted+Thiazolidine-2%2C+4-dione+Compound+as+a+Potent+Inhibitor+for+Estrogen-Related+Receptor+1&doi=10.1021%2Fop400325r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fop400325r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fop400325r%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRussell%26aufirst%3DR.%2BK.%26aulast%3DSpink%26aufirst%3DJ.%26aulast%3DBallentine%26aufirst%3DS.%26aulast%3DTeleha%26aufirst%3DC.%26aulast%3DBranum%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DK.%26aulast%3DBeauchamp%26aufirst%3DD.%26aulast%3DPatch%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DPlayer%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DW.%26atitle%3DProcess%2520Development%2520for%2520Scale-Up%2520of%2520a%2520Novel%25203%252C%25205-Substituted%2520Thiazolidine-2%252C%25204-dione%2520Compound%2520as%2520a%2520Potent%2520Inhibitor%2520for%2520Estrogen-Related%2520Receptor%25201%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2014%26volume%3D18%26spage%3D321%26epage%3D330%26doi%3D10.1021%2Fop400325r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Fink, B. E.; Gavai, A. V.; Vite, G. D.; Chen, P.; Mastalerz, H.; Norris-Drouin, J.; Tokarski, J. S.; Zhao, Y.; Han, W. C.</span><span> </span><span class="NLM_article-title">Pyrrolotriazine compounds as kinase inhibitors. PCT Int. Appl</span> WO2005066176A1,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=B.+E.+Fink&author=A.+V.+Gavai&author=G.+D.+Vite&author=P.+Chen&author=H.+Mastalerz&author=J.+Norris-Drouin&author=J.+S.+Tokarski&author=Y.+Zhao&author=W.+C.+Han&title=Pyrrolotriazine+compounds+as+kinase+inhibitors.+PCT+Int.+Appl"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFink%26aufirst%3DB.%2BE.%26atitle%3DPyrrolotriazine%2520compounds%2520as%2520kinase%2520inhibitors.%2520PCT%2520Int.%2520Appl%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span>The BROMO<i>scan</i> panel, courtesy of DiscoveRx Corp. <a href="http://www.discoverx.com" class="extLink">www.discoverx.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+BROMOscan+panel%2C+courtesy+of+DiscoveRx+Corp.+www.discoverx.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span>Further details of the application of Cellzome bead-based chemoproteomics technology to bromodomain-containing proteins will be the subject of future publications.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Further+details+of+the+application+of+Cellzome+bead-based+chemoproteomics+technology+to+bromodomain-containing+proteins+will+be+the+subject+of+future+publications."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cansfield, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellard, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doce, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leggate, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâMahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plavec, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rharbaoui, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirsch, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rausch, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span> </span><span class="NLM_article-title">A selective inhibitor reveals PI3Kgamma dependence of T<sub>H</sub>17 cell differentiation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">â</span> <span class="NLM_lpage">582</span><span class="refDoi">Â DOI: 10.1038/nchembio.957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnchembio.957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=22544264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1Gqt7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=576-582&author=G.+Bergaminiauthor=K.+Bellauthor=S.+Shimamuraauthor=T.+Wernerauthor=A.+Cansfieldauthor=K.+Mullerauthor=J.+Perrinauthor=C.+Rauauthor=K.+Ellardauthor=C.+Hopfauthor=C.+Doceauthor=D.+Leggateauthor=R.+Manganoauthor=T.+Mathiesonauthor=A.+O%E2%80%99Mahonyauthor=I.+Plavecauthor=F.+Rharbaouiauthor=F.+Reinhardauthor=M.+M.+Savitskiauthor=N.+Ramsdenauthor=E.+Hirschauthor=G.+Drewesauthor=O.+Rauschauthor=M.+Bantscheffauthor=G.+Neubauer&title=A+selective+inhibitor+reveals+PI3Kgamma+dependence+of+TH17+cell+differentiation&doi=10.1038%2Fnchembio.957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor reveals PI3KÎ³ dependence of TH17 cell differentiation</span></div><div class="casAuthors">Bergamini, Giovanna; Bell, Kathryn; Shimamura, Satoko; Werner, Thilo; Cansfield, Andrew; Mueller, Katrin; Perrin, Jessica; Rau, Christina; Ellard, Katie; Hopf, Carsten; Doce, Carola; Leggate, Daniel; Mangano, Raffaella; Mathieson, Toby; O'Mahony, Alison; Plavec, Ivan; Rharbaoui, Faiza; Reinhard, Friedrich; Savitski, Mikhail M.; Ramsden, Nigel; Hirsch, Emilio; Drewes, Gerard; Rausch, Oliver; Bantscheff, Marcus; Neubauer, Gitte</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">576-582</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We devised a high-throughput chemoproteomics method that enabled multiplexed screening of 16,000 compds. against native protein and lipid kinases in cell exts.  Optimization of one chem. series resulted in CZC24832, which is to our knowledge the first selective inhibitor of phosphoinositide 3-kinase Î³ (PI3KÎ³) with efficacy in in vitro and in vivo models of inflammation.  Extensive target- and cell-based profiling of CZC24832 revealed regulation of interleukin-17-producing T helper cell (TH17) differentiation by PI3KÎ³, thus reinforcing selective inhibition of PI3KÎ³ as a potential treatment for inflammatory and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqinUn8kLqw_LVg90H21EOLACvtfcHk0lhbSNakxFkfBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1Gqt7c%253D&md5=0141d8596ee10d4d796de57fb87a2c19</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.957%26sid%3Dliteratum%253Aachs%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DCansfield%26aufirst%3DA.%26aulast%3DMuller%26aufirst%3DK.%26aulast%3DPerrin%26aufirst%3DJ.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DEllard%26aufirst%3DK.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DDoce%26aufirst%3DC.%26aulast%3DLeggate%26aufirst%3DD.%26aulast%3DMangano%26aufirst%3DR.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DPlavec%26aufirst%3DI.%26aulast%3DRharbaoui%26aufirst%3DF.%26aulast%3DReinhard%26aufirst%3DF.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DRamsden%26aufirst%3DN.%26aulast%3DHirsch%26aufirst%3DE.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DRausch%26aufirst%3DO.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DNeubauer%26aufirst%3DG.%26atitle%3DA%2520selective%2520inhibitor%2520reveals%2520PI3Kgamma%2520dependence%2520of%2520TH17%2520cell%2520differentiation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D576%26epage%3D582%26doi%3D10.1038%2Fnchembio.957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesche, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delling, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumpelfeld, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poeckel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reader, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strunk, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Chemoproteomics profiling of HDAC inhibitors reveals selective targeting of HDAC complexes</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">â</span> <span class="NLM_lpage">265</span><span class="refDoi">Â DOI: 10.1038/nbt.1759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnbt.1759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=21258344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1ahug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=255-265&author=M.+Bantscheffauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=A.+Dittmannauthor=P.+Grandiauthor=A.+M.+Michonauthor=J.+Schleglauthor=Y.+Abrahamauthor=I.+Becherauthor=G.+Bergaminiauthor=M.+Boescheauthor=M.+Dellingauthor=B.+Dumpelfeldauthor=D.+Eberhardauthor=C.+Huthmacherauthor=T.+Mathiesonauthor=D.+Poeckelauthor=V.+Readerauthor=K.+Strunkauthor=G.+Sweetmanauthor=U.+Kruseauthor=G.+Neubauerauthor=N.+G.+Ramsdenauthor=G.+Drewes&title=Chemoproteomics+profiling+of+HDAC+inhibitors+reveals+selective+targeting+of+HDAC+complexes&doi=10.1038%2Fnbt.1759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoproteomics profiling of HDAC inhibitors reveals selective targeting of HDAC complexes</span></div><div class="casAuthors">Bantscheff, Marcus; Hopf, Carsten; Savitski, Mikhail M.; Dittmann, Antje; Grandi, Paola; Michon, Anne-Marie; Schlegl, Judith; Abraham, Yann; Becher, Isabelle; Bergamini, Giovanna; Boesche, Markus; Delling, Manja; Duempelfeld, Birgit; Eberhard, Dirk; Huthmacher, Carola; Mathieson, Toby; Poeckel, Daniel; Reader, Valerie; Strunk, Katja; Sweetman, Gavain; Kruse, Ulrich; Neubauer, Gitte; Ramsden, Nigel G.; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">255-265</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The development of selective histone deacetylase (HDAC) inhibitors with anti-cancer and anti-inflammatory properties remains challenging in large part owing to the difficulty of probing the interaction of small mols. with megadalton protein complexes.  A combination of affinity capture and quant. mass spectrometry revealed the selectivity with which 16 HDAC inhibitors target multiple HDAC complexes scaffolded by ELM-SANT domain subunits, including a novel mitotic deacetylase complex (MiDAC).  Inhibitors clustered according to their target profiles with stronger binding of aminobenzamides to the HDAC NCoR complex than to the HDAC Sin3 complex.  We identified several non-HDAC targets for hydroxamate inhibitors.  HDAC inhibitors with distinct profiles have correspondingly different effects on downstream targets.  We also identified the anti-inflammatory drug bufexamac as a class IIb (HDAC6, HDAC10) HDAC inhibitor.  Our approach enables the discovery of novel targets and inhibitors and suggests that the selectivity of HDAC inhibitors should be evaluated in the context of HDAC complexes and not purified catalytic subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWdsITjgu14rVg90H21EOLACvtfcHk0lhUwWKm7GkyWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1ahug%253D%253D&md5=3a826d143fb793719a56fba3c410d51e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1759%26sid%3Dliteratum%253Aachs%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DAbraham%26aufirst%3DY.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DBoesche%26aufirst%3DM.%26aulast%3DDelling%26aufirst%3DM.%26aulast%3DDumpelfeld%26aufirst%3DB.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DPoeckel%26aufirst%3DD.%26aulast%3DReader%26aufirst%3DV.%26aulast%3DStrunk%26aufirst%3DK.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DRamsden%26aufirst%3DN.%2BG.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DChemoproteomics%2520profiling%2520of%2520HDAC%2520inhibitors%2520reveals%2520selective%2520targeting%2520of%2520HDAC%2520complexes%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D255%26epage%3D265%26doi%3D10.1038%2Fnbt.1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallasch, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frenzel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghidelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maier, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wendtner, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Chemoproteomics-based kinome profiling and target deconvolution of clinical multi-kinase inhibitors in primary chronic lymphocytic leukemia cells</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">89</span><span class="NLM_x">â</span> <span class="NLM_lpage">100</span><span class="refDoi">Â DOI: 10.1038/leu.2010.233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fleu.2010.233" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=89-100&author=U.+Kruseauthor=C.+P.+Pallaschauthor=M.+Bantscheffauthor=D.+Eberhardauthor=L.+Frenzelauthor=S.+Ghidelliauthor=S.+K.+Maierauthor=T.+Wernerauthor=C.+M.+Wendtnerauthor=G.+Drewes&title=Chemoproteomics-based+kinome+profiling+and+target+deconvolution+of+clinical+multi-kinase+inhibitors+in+primary+chronic+lymphocytic+leukemia+cells&doi=10.1038%2Fleu.2010.233"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fleu.2010.233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2010.233%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DPallasch%26aufirst%3DC.%2BP.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DFrenzel%26aufirst%3DL.%26aulast%3DGhidelli%26aufirst%3DS.%26aulast%3DMaier%26aufirst%3DS.%2BK.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DWendtner%26aufirst%3DC.%2BM.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DChemoproteomics-based%2520kinome%2520profiling%2520and%2520target%2520deconvolution%2520of%2520clinical%2520multi-kinase%2520inhibitors%2520in%2520primary%2520chronic%2520lymphocytic%2520leukemia%2520cells%26jtitle%3DLeukemia%26date%3D2011%26volume%3D25%26spage%3D89%26epage%3D100%26doi%3D10.1038%2Fleu.2010.233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span> </span><span class="NLM_article-title">Targeted data acquisition for improved reproducibility and robustness of proteomic mass spectrometry assays</span> <span class="citation_source-journal">J. Am. Soc. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1668</span><span class="NLM_x">â</span> <span class="NLM_lpage">1679</span><span class="refDoi">Â DOI: 10.1016/j.jasms.2010.01.012</span> </span><div class="citationLinks">[<a href="/doi/10.1016/j.jasms.2010.01.012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Wmt7bN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=1668-1679&author=M.+M.+Savitskiauthor=F.+Fischerauthor=T.+Mathiesonauthor=G.+Sweetmanauthor=M.+Langauthor=M.+Bantscheff&title=Targeted+data+acquisition+for+improved+reproducibility+and+robustness+of+proteomic+mass+spectrometry+assays&doi=10.1016%2Fj.jasms.2010.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Data Acquisition for Improved Reproducibility and Robustness of Proteomic Mass Spectrometry Assays</span></div><div class="casAuthors">Savitski, Mikhail M.; Fischer, Frank; Mathieson, Toby; Sweetman, Gavain; Lang, Manja; Bantscheff, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1668-1679</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Quant. mass spectrometry-based proteomic assays often suffer from a lack of robustness and reproducibility.  The authors here describe a targeted mass spectrometric data acquisition strategy for affinity enriched subproteomes - in the authors' case the kinome - that enables a substantially improved reproducibility of detection, and improved quantification via isobaric tags.  Inclusion mass lists contg. m/z, charge state, and retention time were created based on a set of 80 shotgun-type expts. performed under identical exptl. conditions.  For each target protein, peptides were selected according to their frequency of observation and isobaric tag for relative and abs. quantitation (iTRAQ) reporter ion quality.  Retention times of selected peptides were aligned using similarity driven pairwise alignment strategy yielding <1 min std. deviation for 4 h gradients.  Multiple fragmentation of the same peptides resulted in better statistics and more precise reporter ion based quantification without any loss in coverage.  Overall, 24% more target proteins were quantified using the targeted data acquisition approach, and precision of quantification improved by >1.5-fold.  The authors also show that a combination of higher energy collisional dissocn. (HCD) with collisional induced dissocn. (CID) outperformed pulsed-Q-dissocn. (PQD) on the Orbitrap XL.  With the CID/HCD based targeted data acquisition approach 10% more quantifiable target proteins were identified and a 2-fold increase in quantification precision was achieved.  The authors have obsd. excellent reproducibility between different instruments, underlining the robustness of the approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyp9JslOo1a7Vg90H21EOLACvtfcHk0lhUwWKm7GkyWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Wmt7bN&md5=c7b65735530bbe9978c8c481876610b9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jasms.2010.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jasms.2010.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DFischer%26aufirst%3DF.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DLang%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26atitle%3DTargeted%2520data%2520acquisition%2520for%2520improved%2520reproducibility%2520and%2520robustness%2520of%2520proteomic%2520mass%2520spectrometry%2520assays%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D2010%26volume%3D21%26spage%3D1668%26epage%3D1679%26doi%3D10.1016%2Fj.jasms.2010.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastuck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesche, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reader, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweetman, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouwmeester, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruse, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsden, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuster, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span> </span><span class="NLM_article-title">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">â</span> <span class="NLM_lpage">1044</span><span class="refDoi">Â DOI: 10.1038/nbt1328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnbt1328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=17721511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1035-1044&author=M.+Bantscheffauthor=D.+Eberhardauthor=Y.+Abrahamauthor=S.+Bastuckauthor=M.+Boescheauthor=S.+Hobsonauthor=T.+Mathiesonauthor=J.+Perrinauthor=M.+Raidaauthor=C.+Rauauthor=V.+Readerauthor=G.+Sweetmanauthor=A.+Bauerauthor=T.+Bouwmeesterauthor=C.+Hopfauthor=U.+Kruseauthor=G.+Neubauerauthor=N.+Ramsdenauthor=J.+Rickauthor=B.+Kusterauthor=G.+Drewes&title=Quantitative+chemical+proteomics+reveals+mechanisms+of+action+of+clinical+ABL+kinase+inhibitors&doi=10.1038%2Fnbt1328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative chemical proteomics reveals mechanisms of action of clinical ABL kinase inhibitors</span></div><div class="casAuthors">Bantscheff, Marcus; Eberhard, Dirk; Abraham, Yann; Bastuck, Sonja; Boesche, Markus; Hobson, Scott; Mathieson, Toby; Perrin, Jessica; Raida, Manfred; Rau, Christina; Reader, Valerie; Sweetman, Gavain; Bauer, Andreas; Bouwmeester, Tewis; Hopf, Carsten; Kruse, Ulrich; Neubauer, Gitte; Ramsden, Nigel; Rick, Jens; Kuster, Bernhard; Drewes, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1035-1044</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors describe a chem. proteomics approach to profile the interaction of small mols. with hundreds of endogenously expressed protein kinases and purine-binding proteins.  This subproteome is captured by immobilized nonselective kinase inhibitors (kinobeads), and the bound proteins are quantified in parallel by mass spectrometry using isobaric tags for relative and abs. quantification (iTRAQ).  By measuring the competition with the affinity matrix, the authors assess the binding of drugs to their targets in cell lysates and in cells.  By mapping drug-induced changes in the phosphorylation state of the captured proteome, the authors also analyze signaling pathways downstream of target kinases.  Quant. profiling of the drugs imatinib (Gleevec), dasatinib (Sprycel) and bosutinib in K562 cells confirms known targets including ABL and SRC family kinases and identifies the receptor tyrosine kinase DDR1 and the oxidoreductase NQO2 as novel targets of imatinib.  The data suggest that the approach is a valuable tool for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsC3OpIuhl8LVg90H21EOLACvtfcHk0lhUwWKm7GkyWQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWksLvF&md5=ed20d4684a8a6733aba24ca64602bae8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnbt1328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1328%26sid%3Dliteratum%253Aachs%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DY.%26aulast%3DBastuck%26aufirst%3DS.%26aulast%3DBoesche%26aufirst%3DM.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DMathieson%26aufirst%3DT.%26aulast%3DPerrin%26aufirst%3DJ.%26aulast%3DRaida%26aufirst%3DM.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DReader%26aufirst%3DV.%26aulast%3DSweetman%26aufirst%3DG.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DKruse%26aufirst%3DU.%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DRamsden%26aufirst%3DN.%26aulast%3DRick%26aufirst%3DJ.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DDrewes%26aufirst%3DG.%26atitle%3DQuantitative%2520chemical%2520proteomics%2520reveals%2520mechanisms%2520of%2520action%2520of%2520clinical%2520ABL%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1035%26epage%3D1044%26doi%3D10.1038%2Fnbt1328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Sharma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bairlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greff, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daub, H.</span><span> </span><span class="NLM_article-title">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">â</span> <span class="NLM_lpage">744</span><span class="refDoi">Â DOI: 10.1038/nmeth.1373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnmeth.1373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=19749761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=741-744&author=K.+Sharmaauthor=C.+Weberauthor=M.+Bairleinauthor=Z.+Greffauthor=G.+Keriauthor=J.+Coxauthor=J.+V.+Olsenauthor=H.+Daub&title=Proteomics+strategy+for+quantitative+protein+interaction+profiling+in+cell+extracts&doi=10.1038%2Fnmeth.1373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span></div><div class="casAuthors">Sharma, Kirti; Weber, Christoph; Bairlein, Michaela; Greff, Zoltan; Keri, Gyoergy; Cox, Juergen; Olsen, Jesper V.; Daub, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-744</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors report a proteomics strategy to both identify and quantify cellular target protein interactions with externally introduced ligands.  The authors detd. dissocn. consts. for target proteins interacting with the ligand of interest by combining quant. mass spectrometry with a defined set of affinity purifn. expts.  The authors demonstrate the general utility of this methodol. in interaction studies involving small-mol. kinase inhibitors, a tyrosine-phosphorylated peptide and an antibody as affinity ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9I5plLFxQRbVg90H21EOLACvtfcHk0lg1rMoKiZ0nIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O&md5=78732562cac1cc2af0938f5536134098</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1373%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DK.%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DBairlein%26aufirst%3DM.%26aulast%3DGreff%26aufirst%3DZ.%26aulast%3DKeri%26aufirst%3DG.%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DDaub%26aufirst%3DH.%26atitle%3DProteomics%2520strategy%2520for%2520quantitative%2520protein%2520interaction%2520profiling%2520in%2520cell%2520extracts%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D741%26epage%3D744%26doi%3D10.1038%2Fnmeth.1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.: Macromol. Crystallogr., Part A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">â</span> <span class="NLM_lpage">326</span><span class="refDoi">Â DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lg1rMoKiZ0nIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%253A%2520Macromol.%2520Crystallogr.%252C%2520Part%2520A%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">Recent changes to the MOSFLM package for processing film and image plate data</span> <span class="citation_source-journal">Jt. CCP4 ESF-EACBM Newsl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">â</span> <span class="NLM_lpage">35</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1992&pages=29-35&author=A.+G.+W.+Leslie&title=Recent+changes+to+the+MOSFLM+package+for+processing+film+and+image+plate+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DRecent%2520changes%2520to%2520the%2520MOSFLM%2520package%2520for%2520processing%2520film%2520and%2520image%2520plate%2520data%26jtitle%3DJt.%2520CCP4%2520ESF-EACBM%2520Newsl.%26date%3D1992%26volume%3D26%26spage%3D29%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span><span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg1rMoKiZ0nIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Evans, P. R.</span><span> </span><span class="NLM_article-title">Data Reduction</span> <span class="citation_source-journal">Proc. CCP4 Study Weekend, Data Collect. Process.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">â</span> <span class="NLM_lpage">122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&pages=114-122&author=P.+R.+Evans&title=Data+Reduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DData%2520Reduction%26jtitle%3DProc.%2520CCP4%2520Study%2520Weekend%252C%2520Data%2520Collect.%2520Process.%26date%3D1993%26spage%3D114%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">â</span> <span class="NLM_lpage">1214</span><span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lhn4WBJKAuvIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Collaborative Computational Project No.4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">D50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">â</span> <span class="NLM_lpage">763</span><span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project+No.4&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3DD50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">â</span> <span class="NLM_lpage">255</span><span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhn4WBJKAuvIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">â</span> <span class="NLM_lpage">2132</span><span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liLS9bAyj2p6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudgin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auger, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">â</span> <span class="NLM_lpage">533</span><span class="refDoi">Â DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0liLS9bAyj2p6g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p5o" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3p5o','PDB','3p5o'); return false;">PDB: 3p5o</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a5q','PDB','5a5q'); return false;">PDB: 5a5q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a81" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a81','PDB','5a81'); return false;">PDB: 5a81</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a85" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a85','PDB','5a85'); return false;">PDB: 5a85</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a82" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a82','PDB','5a82'); return false;">PDB: 5a82</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a83" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5a83','PDB','5a83'); return false;">PDB: 5a83</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i65"><a href="/doi/suppl/10.1021/acs.jmedchem.5b00773">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_13771"></div></div></div></div></div><hr /></hr><p class="last">Table S1, parts 1 and 2: Full data for exemplified compounds in main tables. Table S2: Selectivity profile of compound <b>46</b> in the BromoSCAN panel. Table S3: Data collection and refinement statistics for ATAD2 X-ray structures. Table S4: Data collection and refinement statistics for BRD4 BD1 structure. Figure S1: Correlation plots between ATAD2 assays. Figure S2: KAc site amino acids of ATAD2, ATAD2B, BRD4 BD1, and BRD4 BD2 bromodomains. Figure S3: OMIT maps for ATAD2 and BRD4 BD1 X-ray structures. Figure S4: GSK3190320, the synthetic ligand used in the ATAD2 TR-FRET assay. The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b00773">10.1021/acs.jmedchem.5b00773</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00773/suppl_file/jm5b00773_si_001.pdf">jm5b00773_si_001.pdf (983.65 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00773/suppl_file/jm5b00773_si_002.csv">jm5b00773_si_002.csv (2.58 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b00773%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-15" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b00773" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d1570c682367","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
