
<html lang="en"     class="pb-page"  data-request-id="22beca95-1a62-4c7b-9cb0-9a80ef265d55"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.8b00862;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2018.61.issue-18;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Rebecca C.  Furze" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="Anne-Marie  Michon" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="Darren J.  Mitchell" /></meta><meta name="dc.Creator" content="Heather  Barnett" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Christina  Rau" /></meta><meta name="dc.Creator" content="Robert J.  Sheppard" /></meta><meta name="dc.Creator" content="Thilo  Werner" /></meta><meta name="dc.Creator" content="Emmanuel H.  Demont" /></meta><meta name="dc.Description" content="ATAD2 is a cancer-associated protein whose bromodomain has been described as among the least druggable of its class. In our recent disclosure of the first chemical probe against this bromodomain, G..." /></meta><meta name="Description" content="ATAD2 is a cancer-associated protein whose bromodomain has been described as among the least druggable of its class. In our recent disclosure of the first chemical probe against this bromodomain, G..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 18, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00862" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00862" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00862" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00862" /></link>
        
    
    

<title>Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00862" /></meta><meta property="og:title" content="Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0012.jpeg" /></meta><meta property="og:description" content="ATAD2 is a cancer-associated protein whose bromodomain has been described as among the least druggable of its class. In our recent disclosure of the first chemical probe against this bromodomain, GSK8814 (6), we described the use of a conformationally constrained methoxy piperidine to gain selectivity over the BET bromodomains. Here we describe an orthogonal conformational restriction strategy of the piperidine ring to give potent and selective tropane inhibitors and show structural insights into why this was more challenging than expected. Greater understanding of why different rational approaches succeeded or failed should help in the future design of selectivity in the bromodomain family." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00862"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00862">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00862&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00862&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00862&amp;href=/doi/10.1021/acs.jmedchem.8b00862" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 8321-8336</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00838" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00832" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Discovery Research, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>For P.B.: phone, +44 1438 763246; fax, +44 1438 763352; E-mail, <a href="/cdn-cgi/l/email-protection#88d8e9fde4a6c9a6cae9e5eae7fae7fdefe0c8effbe3a6ebe7e5"><span class="__cf_email__" data-cfemail="530332263f7d127d11323e313c213c26343b133420387d303c3e">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Discovery Research, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rebecca C. Furze</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rebecca C. Furze</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rebecca+C.++Furze">Rebecca C. Furze</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Grandi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Grandi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne-Marie Michon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Marie Michon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Michon">Anne-Marie Michon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darren J. Mitchell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren J. Mitchell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren+J.++Mitchell">Darren J. Mitchell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heather Barnett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heather Barnett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Flexible Discovery Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heather++Barnett">Heather Barnett</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christina Rau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christina Rau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christina++Rau">Christina Rau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Sheppard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Sheppard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Sheppard">Robert J. Sheppard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thilo Werner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thilo Werner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thilo++Werner">Thilo Werner</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Emmanuel H. Demont</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emmanuel H. Demont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>For E.H.D.: phone, +44 1438 764319; fax, +44 1438 768302; E-mail, <a href="/cdn-cgi/l/email-protection#f6b39b9b979883939ad8bed8b2939b999882b691859dd895999b"><span class="__cf_email__" data-cfemail="d095bdbdb1bea5b5bcfe98fe94b5bdbfbea490b7a3bbfeb3bfbd">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H.++Demont">Emmanuel H. Demont</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00862&amp;href=/doi/10.1021%2Facs.jmedchem.8b00862" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 8321–8336</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 18, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 May 2018</li><li><span class="item_label"><b>Published</b> online</span>18 September 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 September 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00862" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00862</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8321%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaul%2BBamborough%252C%2BChun-wa%2BChung%252C%2BRebecca%2BC.%2BFurze%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D18%26contentID%3Dacs.jmedchem.8b00862%26title%3DAiming%2Bto%2BMiss%2Ba%2BMoving%2BTarget%253A%2BBromo%2Band%2BExtra%2BTerminal%2BDomain%2B%2528BET%2529%2BSelectivity%2Bin%2BConstrained%2BATAD2%2BInhibitors%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8336%26publicationDate%3DSeptember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00862"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1495</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00862" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Rebecca&quot;,&quot;last_name&quot;:&quot;C. Furze&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Michon&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Heather&quot;,&quot;last_name&quot;:&quot;Barnett&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Christina&quot;,&quot;last_name&quot;:&quot;Rau&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Sheppard&quot;},{&quot;first_name&quot;:&quot;Thilo&quot;,&quot;last_name&quot;:&quot;Werner&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H. Demont&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;8321-8336&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00862&quot;},&quot;abstract&quot;:&quot;ATAD2 is a cancer-associated protein whose bromodomain has been described as among the least druggable of its class. In our recent disclosure of the first chemical probe against this bromodomain, GSK8814 (6), we described the use of a conformationally constrained methoxy piperidine to gain selectivity over the BET bromodomains. Here we describe an orthogonal conformational restriction strategy of the piperidine ring to give potent and selective tropane inhibitors and show structural insights into why this was more challenging than expected. Greater understanding of why different rational approaches succeeded or failed should help in the future design of selectivity in the bromodomain family.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00862&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00862" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00862&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00862" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00862&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00862" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00862&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00862&amp;href=/doi/10.1021/acs.jmedchem.8b00862" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00862" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00862" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00862%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DBamborough%26date%3D2018%26atitle%3DAiming%2Bto%2BMiss%2Ba%2BMoving%2BTarget%253A%2BBromo%2Band%2BExtra%2BTerminal%2BDomain%2B%2528BET%2529%2BSelectivity%2Bin%2BConstrained%2BATAD2%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D18%26spage%3D8321%26epage%3D8336%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291870" title="Piperidines">Piperidines</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/jmcmar.2018.61.issue-18/20180927/jmcmar.2018.61.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">ATAD2 is a cancer-associated protein whose bromodomain has been described as among the least druggable of its class. In our recent disclosure of the first chemical probe against this bromodomain, GSK8814 (<b>6</b>), we described the use of a conformationally constrained methoxy piperidine to gain selectivity over the BET bromodomains. Here we describe an orthogonal conformational restriction strategy of the piperidine ring to give potent and selective tropane inhibitors and show structural insights into why this was more challenging than expected. Greater understanding of why different rational approaches succeeded or failed should help in the future design of selectivity in the bromodomain family.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51154" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51154" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">ATAD2, also known as ATAD2A or ANCCA (AAA nuclear coregulator cancer-associated protein), is a nuclear protein normally expressed in germ cells and embryonic stem cells with a key role in chromatin remodelling.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Since its identification as a target gene in breast cancer,<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> ATAD2 has been shown to be highly expressed in a wide variety of unrelated cancers with increased expression levels correlating with disease recurrence and poor clinical prognosis.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Potential oncology indications include lung adenocarcinoma,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> hepatocellular carcinoma,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> colorectal,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> ovarian,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and endometrial cancer<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> as well as osteosarcoma<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and gastric cancer.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">As an oncology target, ATAD2 is supported by downregulation of protein expression using siRNA technology. This has implicated the role of ATAD2 in pathways including apoptosis, cell survival, proliferation, and migration.<a onclick="showRef(event, 'ref2 ref4 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref2 ref4 ref13 ref14 ref15 ref16">(2,4,13−16)</a> Mechanistically, ATAD2 has been shown to function as a coactivator for a variety of transcription factors including MYC, estrogen/androgen receptors, and E2F family members, directing transcription of a number of genes involved in cell proliferation and survival, including cyclins, Cdk2, and kinesins.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21">(17−21)</a> Furthermore, it has been demonstrated that ATAD2 is recruited to newly synthesized histones during DNA replication via a direct interaction with diacetylation modifications at K5 and K12 on histone H4.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a></div><div class="NLM_p">ATAD2 possesses a bromodomain and an ATPase associated with diverse cellular activities (AAA+) domain. So far, drug discovery efforts for oncology have focused on the ATAD2 bromodomain.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> The discovery that molecules identified through phenotypic screens act by inhibition of the bromodomains of the BET family demonstrated the ligandability of bromodomains and highlighted their potential in a wide range of clinical indications, including oncology and inflammation.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> A number of drug-like BET bromodomain inhibitors have now been reported, and over 20 clinical trials are underway studying their effects in oncology and inflammatory diseases.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> A growing number of tool molecules have been reported to inhibit the bromodomains of non-BET proteins, including but not limited to CBP/EP300, BRD7/9, BRPF1, and SMARCA2/4.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a> Because of the potent pharmacology of even quite weak BET inhibitors,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> cellular chemical probes of other bromodomains require exceptional selectivity. We have typically tried to reach 100–1000-fold selectivity. However, because of the conserved nature of bromodomain KAc sites, this is not always easily achieved. Greater understanding of which rational attempts to engineer greater selectivity succeed, and why they can unexpectedly fail, provides insights to inform future efforts.</div><div class="NLM_p">We<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> and others<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> have identified small molecule inhibitors of the ATAD2/ATAD2B bromodomain to probe its functional role in disease and to assess if its inhibition was sufficient to reproduce the promising results seen with siRNA knockdown. Starting from a fragment hit, we identified potent, cell permeable, highly selective ATAD2 inhibitors such as compound <b>6</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> This uses a methoxy C5′-substituent to constrain the piperidine ring to an equatorial geometry in order to gain selectivity against the BET bromodomains. In this article, we present results from an alternative means of conformational restriction using 2-carbon-bridged piperidine (tropane) derivatives. This approach led to the identification of compounds with good potency and selectivity over the BET bromodomains. It also provided additional rationale for why compounds such as <b>6</b> have exceptional selectivity against BET.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure and Profile of Our Reported ATAD2 Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0008.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">BRD4 BD1 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">selectivity (fold)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="center">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="center">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">CH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="center">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">SO<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="center">125</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="center">OMe</td><td class="colsep0 rowsep0" align="center">SO<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="center">1300</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="center">OMe</td><td class="colsep0 rowsep0" align="center">CF<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="center">500</td></tr></tbody></table></div></div><div class="NLM_p">As previously reported,<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> we used structure-based design to increase the potency and selectivity for ATAD2 over BRD4 BD1 (a representative of the eight bromodomains of this family, see Supporting Information, <a class="ref internalNav" href="#notes-1" aria-label="Tables S2 and S3">Tables S2 and S3</a>) starting from compound <b>1</b> to give inhibitors such as <b>2</b> and <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In particular we exploited the different interactions between the C3′-substituent and the RVF shelf in ATAD2 and the WPF shelf in BET. We reasoned that certain polar analogues of this cyclohexyl ring would interact better with the arginine side chains Arg1007 and Arg1077 found in ATAD2 than with the analogous tryptophan and methionine of the BET family (Trp81 and Met149 in BRD4 BD1), hence providing selectivity. This was indeed found to be the case (compare compounds such as <b>4</b> to <b>3</b>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Crystal structures of the analogous quinolinone series in ATAD2 and BET revealed unexpected features of their binding modes.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In ATAD2, the piperidine lies in a trans-diequatorial conformation, making specific interactions with Arg1077. Unexpectedly, in BRD4 BD1 the piperidine ring adopts a trans-diaxial conformation to remove the polar sulfone C3′-group from the WPF shelf. Selectivity for ATAD2 over BRD4 BD1 was increased further by stabilizing the equatorial conformation versus the axial. This was achieved by the introduction of a C5′-methoxy substituent <i>cis</i> to the C3′ ether, giving <b>5</b>. This group creates a 1,3 steric strain in the putative triaxial conformation, disfavoring binding to BET, with limited impact on the triequatorial conformation seen in ATAD2. A final modification (isosteric substitution of the SO<sub>2</sub> group with CF<sub>2</sub>) dramatically increased the passive permeability of our inhibitors, leading to compound <b>6</b> (GSK8814) the first potent, selective, and permeable ATAD2 chemical probe.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Crystal structures of <b>6</b> showed that it also adopts a trans-equatorial conformation when bound to ATAD2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) and a trans-triaxial conformation when bound to BRD4 BD1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>6</b> bound to ATAD2 and BRD4 (same orientation) illustrating the conformational basis for selectivity. In both proteins, the residues of the RVF/WPF motif are shown in pale yellow and the gatekeeper residue in salmon. In ATAD2 (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0">5LJ0</a>), the piperidine ring is triequatorial, positioning the C3′-substituent upon the RVF shelf where its CF<sub>2</sub> group interacts with R1077. In BRD4 (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ1">5LJ1</a>), the triaxial piperidine geometry moves the C3′ substituent away from the WPF shelf into solvent.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60831" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60831" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">While evaluating the impact of piperidine ring 1,3 sterics on BET selectivity, we also investigated other conformational constraints intended to force the shelf substituent into an equatorial position, including tropane derivatives. The starting points of this work were small analogues unsubstituted at the naphthyridone C5-position. As can be seen in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, both unsubstituted tropane isomers <b>9</b> and <b>10</b> have similar BET activity to the piperidine analogue <b>8</b>. Their potency against ATAD2 is slightly reduced, in particular for <b>9</b>, although this was not thought to be very significant.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Comparison of ATAD2 and BET Activity of Unfunctionalized Tropane and Piperidines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0009.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">BRD4 BD1 pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr></tbody></table></div></div><div class="NLM_p">Crystallographic data (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) showed that the tropane <b>10</b> has a similar ATAD2 binding mode to piperidine compounds such as <b>6</b>. We expected that the 3-pyridinyl substituents previously found to be advantageous at the naphthyridone C5-position could be incorporated directly into this new template. We also thought that the tropane could provide access to substitution vectors similar to the C5′-position of the piperidine of <b>6</b>. As a conformationally constrained alternative to the piperidine, the tropane could direct polar substituents onto the RVF shelf of ATAD2 and the WPF shelf of BET without permitting a conformational escape route and so exploit the differences between the amino acids there to gain selectivity.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ATAD2 crystal structures of the NH piperidine GSK8814 (<b>6</b>, white, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0">5LJ0</a>) and the simplified tropane <b>10</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDN">6HDN</a>). Compound <b>6</b> uses its piperidine C3′-vector to make its CF<sub>2</sub> interaction with R1077, and its naphthyridone 5-position to make its pyridyl interaction with D1014. The analogous vectors are available to the tropane template <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The synthesis of functionalized tropanes is described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The C3′-substituent was introduced via an aldol reaction between ketone <b>14</b> and aldehydes <b>13a</b>–<b>c</b> (readily available from commercial ketones <b>11a</b>,<b>b</b> or acid <b>12c</b>). The intermediate alcohol was activated as a mesylate and eliminated to give alkenes <b>15a</b>,<b>c</b> as single stereoisomers in good yields. Following a stereoselective hydrogenation of the double bond, the ketone was converted to the oxime, which, on reduction with sodium metal, provided racemic amines <b>16a</b>–<b>c</b> as single diastereoisomers. We did not anticipate that changing the ether chain (as used in the piperidine series) to an alkyl chain (tropane series) to access the RVF shelf would have an impact as it did not have any in the piperidine series (see pairwise comparison in <a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>, Supporting Information). The racemic synthesis of the analogous C3′–C5′ substituted derivatives <b>18a</b>–<b>c</b> from commercial <b>17</b> has already been described.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route to Functionalized Tropanes and Piperidines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (Ph)<sub>3</sub>P = CHCOOEt, toluene, 100 °C, 75–99%; (b) X = S, NiCl<sub>2</sub>, NaBH<sub>4</sub>, EtOH, 0 °C, 70%; (c) X = S, <i>m</i>-CPBA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 98%; (d) X = CF<sub>2</sub>, H<sub>2</sub>, Pd/C, MeOH, room temperature, 98%; (e) LiAlH<sub>4</sub>, THF/Et<sub>2</sub>O, −30 °C to room temperature, 88–100%; (f) (COCl)<sub>2</sub>, DMSO, CH<sub>2</sub>Cl<sub>2</sub>, −78 °C then NEt<sub>3</sub>; (g) LDA, THF, −78 °C then <b>13a</b>, <b>b</b>, or <b>c</b>; (h) (CH<sub>3</sub>SO<sub>2</sub>)<sub>2</sub>O, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C; (i) DBU, THF, room temperature, 69–70% over 4 steps; (j) H<sub>2</sub>, Pd/C, MeOH/THF, room temperature, 74–79%; (k) NH<sub>2</sub>OH.HCl, pyridine, EtOH, reflux, 89–100%; Na, EtOH, reflux, 47–76%.</p></p></figure><div class="NLM_p">These amines were then coupled to the known naphthyridone <b>19a</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The derivatives <b>20</b> were further functionalized at the C5 position following bromination and Suzuki coupling. Deprotection in acidic conditions gave the racemate of inhibitors <b>22</b>–<b>26</b>. The chiral separation of the racemic inhibitors or intermediates could be performed at any stage after the Buchwald coupling. In all cases, one enantiomer was significantly more potent at ATAD2 than the other. Co-crystallizations of the most active enantiomers in ATAD2 (or BRD4 BD1/BD2) always showed the same configuration, which is the one exemplified in all the schemes and tables. The analogous racemic quinolinone derivatives <b>27</b> and <b>28</b> (vide infra) could be obtained using the same sequence, starting from the known <b>19b</b> intermediate.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Compound <b>30</b> was obtained by direct deprotection of the corresponding intermediate <b>20</b>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of ATAD2 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>t</i>BuONa, BrettPhos, Pd<sub>2</sub>(dba)<sub>3</sub>, THF, 60 °C, 25–94%; (b) TFA, reflux, 63–96%; (c) NBS, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C; (d) ArB(OH)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, Pd(OAc)<sub>2</sub>, di(1-adamantyl)-<i>n</i>-butylphosphine, 1,4-dioxane/water, 100 °C, 71–93% (2 steps); (e) chiral chromatography.</p></p></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> ATAD2 Activity of Tropane Derivatives</h3><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> compares activities against ATAD2 and BET for the mono- or disubstituted piperidines and their corresponding tropanes. These differ from <b>10</b> by the incorporation of substituents known to be favorable from our previous work, namely the methylpyridine at the naphthyridone C5-position and the introduction of tropane substituents intended to interact with the RVF shelf.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> The combination of both modifications led to highly potent tropane-containing ATAD2 inhibitors such as <b>23</b>, <b>24</b>, and <b>26</b>, which also show excellent improvements in BET selectivity. The improvement in potency of these compounds over <b>10</b> is ∼100-fold, which is comparable to that found relative to <b>7</b> in the piperidine series.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Comparison of Activities between Tropane and Piperidine Derivatives</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0010.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">type</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">BRD4 BD1 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">selectivity (fold)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">A</td><td class="colsep0 rowsep0" align="center">O</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="center">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">B</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="center">160</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">C</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="center">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">A</td><td class="colsep0 rowsep0" align="center">SO<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="center">125</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">B</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="center">1260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">C</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char="."><4.3</td><td class="colsep0 rowsep0" align="center">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">A</td><td class="colsep0 rowsep0" align="center">CF<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="center">150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">B</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="center">500</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="center">N</td><td class="colsep0 rowsep0" align="center">C</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="center">60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">CH</td><td class="colsep0 rowsep0" align="center">C</td><td class="colsep0 rowsep0" align="center">SO<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="center">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">CH</td><td class="colsep0 rowsep0" align="center">A</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="center">25</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Compounds are racemic.</p></div></div></div><div class="NLM_p">Although we were unable to obtain a crystal structure of ATAD2 bound to <b>24</b>, we did manage to solve one in complex with its direct quinolinone N → CH analogue <b>27</b>. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> shows ATAD2 bound to <b>27</b> superimposed on our previously published structure of ATAD2 bound to <b>28</b>, which is the directly comparable quinolinone analogue of <b>4</b>.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In our previous work, we saw no structural differences in binding mode between the naphthyridone and quinolinone templates.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The ATAD2 binding mode of the substituted tropane derivative is very similar to its piperidine counterpart. Both derivatives form a hydrogen bond to Asp1071 via the piperidine or tropane nitrogen, anchoring this part of the molecule to the binding site. In the RVF shelf subpocket, a sulfone oxygen of each inhibitor is positioned appropriately to interact with Arg1077. In this particular example, the slight shift of the shelf substituent seen in the tropane series compared to the piperidine series might explain the minor decreases in ATAD2 potency observed for most pairs of examples made (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, compare <b>24</b> to <b>4</b> and <b>5</b>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Superimposed crystal structures of the ATAD2-bound racemate of compound <b>27</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HD0">6HD0</a>) and its direct piperidine analogue <b>28</b> (white, PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A83">5A83</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> The Selectivity of Constrained Tropane Derivatives over BRD4</h3><div class="NLM_p">From our previous work in the piperidine series, we understood that polar C3′-substituents (tetrahydropyran, difluoro-cyclohexyl, and cyclic sulfonamide) were favored on the RVF shelf of ATAD2 but poorly tolerated on the WPF shelf of BET.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Crystal structures showed that when these compounds bind to BRD4 BD1 their piperidine ring flips to adopt an axial conformation that allows the C3′-substituents to escape the WPF shelf environment by placing their polar functionality into solvent. Presumably this involves some conformational cost. Clearly, it would be impossible for the tropane derivatives to adopt this axial conformation.</div><div class="NLM_p">The tropanes <b>23</b>, <b>24</b>, and <b>26</b> have fairly good BRD4 selectivity, comparable to or better than their monosubstituted piperidine analogues (<b>21</b>, <b>4</b>, and <b>25</b> respectively, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The selectivity for ATAD2 over BRD4 is most pronounced for the sulfone derivative <b>24</b> (>250-fold). This is approaching the selectivity window achieved by the trisubstituted methoxy-piperidine sulfone <b>5</b>. However, the THP and difluoro-cyclohexyl tropanes <b>23</b> and <b>26</b> did not quite achieve the level of selectivity reached by their direct methoxy-piperidine analogues <b>22</b> and <b>6</b>.</div><div class="NLM_p">We used X-ray crystallography to investigate the reason why the BRD4 potency of tropane derivative <b>26</b> is around 5-fold greater than the disubstituted piperidine <b>6</b>. The X-ray structure of BRD4 BD1 bound to compound <b>26</b> is shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> compared to the previously published structure of BRD4 BD1 bound to <b>1</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A85">5A85</a>). The position of the naphthyridone core is not changed by the modifications. As expected, because of its constrained tropane ring, the difluorocyclohexane substituent of <b>26</b> is unable to move away from the WPF shelf of BRD4 unlike that of <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). The CF<sub>2</sub> group of <b>26</b> is forced to sit on the shelf in a region which is unfavorable due to steric or electrostatic interactions with Trp81. In this constrained situation, Trp81 responds to binding of <b>26</b> through a local rearrangement, adopting several different rotamers where the side chain moves away from the CF<sub>2</sub> group. These alternative conformers are rarely present in reported BRD4 BD1 structures in the Protein Data Bank (<a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>, Supporting Information), but this increased disorder is apparently preferable to Trp81 remaining next to the difluorocyclohexane of the ligand. We attribute the lower BRD4 activity of the sulfone <b>24</b> relative to <b>26</b> to the adoption of a similar binding mode leaving the more polar SO<sub>2</sub> functionality in proximity to a rearranged Trp81.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystallographic binding mode of compound <b>26</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ">6HDQ</a>) in BRD4 BD1, showing alternative conformers of Trp81 (orange). This is superimposed on the BRD4 structure of the racemate of <b>1</b> (white, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A85">5A85</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a further effort to understand better why the trisubstituted methoxy piperidines are more selective for ATAD2 over BRD4 BD1 than the tropanes, we made some simplified analogues lacking any C3′ substituents large enough to be able to occupy the WPF shelf (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The C3′–C5′ dimethoxy piperidine is intrinsically less active against BET (compare activity of compound <b>30</b> versus compounds <b>7</b>, <b>29</b>, or even <b>10</b>). The impact of C3′ and C5′ dimethoxy substitution on ATAD2 potency is much smaller than the impact on BET potency.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Activity and Selectivity of Piperidine Bearing Small Substituents</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0011.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">BRD4 BD1 pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr></tbody></table></div></div><div class="NLM_p">This can be understood by examining the crystal structures of <b>6</b> bound to ATAD2 and to BRD4 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The side chains of the ZA loop are closer to the inhibitor in BRD4 than ATAD2, leaving less space available at the piperidine C5′ position. The closest side chain of BRD4 to the C5′ atom of <b>25</b> in the crystal structure is Leu94 (3.8 Å distance). For comparison, in ATAD2 the closest side chain the C5′ atom of <b>6</b> is Val1018 (6.1 Å distance). Therefore, we conclude that in addition to the conformational contribution of the trisubstituted methoxypiperidine to the selectivity of <b>6</b> that we described previously, there also appears to be a steric component arising from the more limited amount of accessible space for the 5′-methoxy group in BRD4.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray structures of ATAD2 bound to <b>6</b> (white, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0">5LJ0</a>) and BRD4 BD1 bound to <b>26</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ">6HDQ</a>). The magenta dashed line shows the closest contact between <b>26</b> and BRD4 Leu94. Molecular surfaces of the closest side chains to the C3′-position (BRD4 Leu94 and ATAD2 Val1018) highlight the relatively greater space that is available for C3′ substituents in ATAD2 compared to BRD4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We set out to test the idea that constrained bridged piperidine derivatives would show greater selectivity over the BET bromodomains by rigidification, removing any possibility for the ligand to avoid placing polar C3′ substituents onto the WPF shelf. This led to the discovery of compounds such as <b>24</b> and <b>26</b> that are structurally different from our previous series. These retain excellent ATAD2 potency and show good selectivity over BET.</div><div class="NLM_p">The tropane derivatives were, however, less selective than their disubstituted piperidine analogues. Our efforts to understand the reasons for this led us to generate SAR with simplified compounds in the latter series, showing that dimethoxy substitution on the piperidine was intrinsically detrimental to BET activity. A closer examination of the crystal structures of this series bound to BRD4 and ATAD2 suggest that this is because the side chains of the ZA loop of BRD4 allow less space for substitution at the C5′ position. This further explains the exceptional selectivity seen with <b>6</b>.</div><div class="NLM_p">The X-ray structure of the complex of BRD4 bound to <b>26</b> showed that the tropane does indeed force the difluorocyclohexyl substituent onto the WPF shelf. However, because this is electrostatically and/or sterically unfavorable, the crystal structure also revealed that as a result the protein undergoes an unexpected ligand-induced conformational change of the Trp81 side chain. This illustrates the complexity that can arise in interpreting the SAR of weak protein–ligand interactions and the difficulties that can complicate efforts to improve selectivity over an antitarget like BRD4. Rational modifications designed to introduce particular negative interactions to reduce binding affinity to the antitarget may be less effective than expected if the system can find ways to evade this interaction. Some systems, like BRD4, may find multiple ways to do this. In our earlier study on the piperidine series, we showed how the ligand escaped an unfavorable interaction with BRD4 Trp81 by adopting an unexpected axial conformational state, which limited the effectiveness of our rational design strategy until steps were taken to eliminate the unwanted conformer.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Here, we have shown that even when this escape route was blocked by rigidification of the ligand, the detrimental effects of the Trp81 interaction were still limited by an unanticipated conformational change, in this case, the ability of a side chain to move by adopting multiple rotameric states.</div><div class="NLM_p last">No doubt there are many cases where rational selectivity design has been attempted and found to be less successful than hoped, for similar reasons to those in our example. The explanation often remains poorly understood because crystallography of the antitargets may not be available or because crystallization of ligands that bind relatively weakly is challenging. However, when successful the structural insights may reveal unexpected ways that proteins or ligands can adapt to minimize unfavorable interactions, and this information can suggest new strategies to improve selectivity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67308" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67308" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Protein Expression, ATAD2 and BET Bromodomain Assays, and Physicochemical Property Measurement</h3><div class="NLM_p last">These were carried out as described previously.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry</h3><div class="NLM_p last">All solvents were purchased from Sigma-Aldrich (Hy-Dry anhydrous solvents) and commercially available reagents were used as received. All reactions were followed by TLC analysis (TLC plates GF254, Merck) or liquid chromatography mass spectrometry (LCMS) using a Waters ZQ instrument. NMR spectra were recorded on a Bruker nanobay 400 MHz or a Bruker AVII+ 600 MHz spectrometers and are referenced as follows: <sup>1</sup>H NMR (400 or 600 MHz), internal standard TMS at δ = 0.00. <sup>13</sup>C NMR (100.6 or 150.9 MHz), internal standard CDCl<sub>3</sub> at δ = 77.23 or DMSO-<i>d</i><sub>6</sub> at δ = 39.70. Column chromatography was performed on prepacked silica gel columns (30–90 mesh, IST) using a biotage SP4. Mass spectra were recorded on Waters ZQ (ESI-MS) and Q-Tof 2 (HRMS) spectrometers. Mass directed auto prep was performed on a Waters 2767 with a MicroMass ZQ mass spectrometer using a Supelco LCABZ++ column.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthetic Methods and Characterization of Compounds</h3><div class="NLM_p last">Abbreviations for multiplicities observed in NMR spectra: s; singulet; br s, broad singulet; d, doublet; t, triplet; q, quadruplet; p, pentuplet; spt, septuplet; m, multiplet. The purity of all compounds was determined by LCMS and <sup>1</sup>H NMR and was always >95%.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> LCMS</h3><div class="NLM_p">UPLC analysis was conducted on an Acquity UPLC BEH C18 column (50 mm × 2.1 mm, i.d. 1.7 μm packing diameter) at 40 °C. Formate method: solvents employed were A = 0.1% v/v solution of formic acid in water; B = 0.1% v/v solution of formic acid in acetonitrile. High pH method: the solvents employed were: A = 10 mM ammonium hydrogen carbonate in water adjusted to pH 10 with ammonia solution; B = acetonitrile. For both methods, the gradient employed was (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>):</div><div class="NLM_table-wrap" id="tbl5">Table 5<div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">time (min)</th><th class="colsep0 rowsep0" align="center" char=".">flow rate (mL/min)</th><th class="colsep0 rowsep0" align="center" char=".">%A</th><th class="colsep0 rowsep0" align="center" char=".">%B</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr></tbody></table></div></div><div class="NLM_p last">The UV detection was a summed signal from wavelength of 210 to 350 nm. Mass spectra were obtained on a Waters ZQ instrument; ionization mode, alternate-scan positive, and negative electrospray; scan range 100 to 1000 AMU; scan time 0.27 s; interscan delay 0.10 s.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> MDAP</h3><div class="NLM_p">Formate method: HPLC analysis was conducted on either a Sunfire C18 column (100 mm × 19 mm, i.d 5 μm packing diameter) or a Sunfire C18 column (150 mm × 30 mm, i.d. 5 μm packing diameter) at ambient temperature. The solvents employed were: A = 0.1% v/v solution of formic acid in water; B = 0.1% v/v solution of formic acid in acetonitrile. Run as a gradient over either 15 or 25 min (extended run) with a flow rate of 20 mL/min (100 mm × 19 mm, i.d. 5 μm packing diameter) or 40 mL/min (150 mm × 30 mm, i.d. 5 μm packing diameter). High pH method: HPLC analysis was conducted on either an Xbridge C18 column (100 mm × 19 mm, i.d. 5 μm packing diameter) or a Xbridge C18 column (100 mm × 30 mm, i.d. 5 μm packing diameter) at ambient temperature. The solvents employed were: A = 10 mM ammonium bicarbonate in water, adjusted to pH10 with ammonia solution; B = acetonitrile. Run as a gradient over either 15 or 25 min (extended run) with a flow rate of 20 mL/min (100 mm × 19 mm, i.d. 5 μm packing diameter) or 40 mL/min (100 mm × 30 mm, i.d. 5 μm packing diameter). For both methods, the UV detection was a summed signal from wavelength of 210–350 nm. Mass spectra were obtained on a Waters ZQ instrument; ionization mode, alternate-scan positive, and negative electrospray; scan range 100–1000 AMU; scan time 0.50 s; interscan delay 0.20 s.</div><div class="NLM_p last">Compounds <b>1</b>–<b>10</b>, <b>19a</b>,<b>b</b>, <b>21</b>, <b>25</b>, and <b>29</b> synthesis have already been published.<a onclick="showRef(event, 'ref33 ref34 ref38'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref38">(33,34,38)</a></div></div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Ethyl 2-(1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)acetate (<b>12a</b>)</h3><div class="NLM_p last">Step 1: A solution of dihydro-2<i>H</i>-thiopyran-4(3<i>H</i>)-one (11.45 g, 99.00 mmol) and ethyl 2-(triphenylphosphoranylidene)acetate (36.0 g, 103 mmol) in toluene (300 mL) was stirred at 100 °C for 8 h. Ethyl 2-(triphenylphosphoranylidene)acetate (4.0 g, 11 mmol) was added, and the mixture was stirred for another 4 h and then was cooled to room temperature. The solvent was removed in vacuo, and the residue was triturated with Et<sub>2</sub>O. The solid Ph<sub>3</sub>PO formed was filtered off and rinsed with Et<sub>2</sub>O, and the combined organics were concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (340 g column, 10% GLOBAL gradient (AcOEt in hexanes)) gave ethyl 2-(dihydro-2<i>H</i>-thiopyran-4(3<i>H</i>)-ylidene)acetate (13.7 g, 75%) as colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 5.69 (s, 1H), 4.16 (q, <i>J</i> = 7.2 Hz, 2H), 3.21 (m, 2H), 2.82–2.73 (m, 4H), 2.59–2.50 (m, 2H), 1.29 (t, <i>J</i> = 7.2 Hz, 3H). Step 2: A solution of ethyl 2-(dihydro-2<i>H</i>-thiopyran-4(3<i>H</i>)-ylidene)acetate (7.88 g, 42.3 mmol) in EtOH (160 mL) at 0 °C was treated with nickel(II) chloride (16.4 g, 127 mmol) then with sodium borohydride (6.40 g, 169 mmol) Portionwise (1 g every 10 min). Then 2 h after the final addition, the solvent was removed in vacuo and the residue was partitioned between EtOAc and water. The biphasic phase was filtered through Celite and the layers were separated. The aqueous phase was extracted twice with EtOAc, and the combined organics were washed twice with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo to give ethyl 2-(tetrahydro-2<i>H</i>-thiopyran-4-yl)acetate (5.6 g, 70%) as a colorless oil, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.17 (q, <i>J</i> = 7.1 Hz, 2H), 3.10–2.97 (m, 4H), 2.34 (d, <i>J</i> = 6.6 Hz, 2H), 2.21–2.05 (m, 3H), 2.02–1.85 (m, 2H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H). Step 3: A solution of ethyl 2-(tetrahydro-2<i>H</i>-thiopyran-4-yl)acetate (5.6 g, 30 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (150 mL) at 0 °C was treated with m-CPBA (<77% w/w, 12.7 g, 62.5 mmol) in two portions, and the resulting mixture was stirred at this temperature. After 30 min, CH<sub>2</sub>Cl<sub>2</sub> (150 mL) was added to help stirring. After 6 h, the mixture was treated with sodium thiosulfate (9.40 g, 59.5 mmol) in water (50 mL). The mixture was vigorously stirred for 10 min, and the layers were separated. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub>, and the combined organic phases were washed three times with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a phase separator, and concentrated in vacuo to give ethyl 2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)acetate (<b>12a</b>) (6.43 g, 98%) as a white solid which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.16 (q, <i>J</i> = 7.1 Hz, 2H), 3.10–2.93 (m, 4H), 2.34 (d, <i>J</i> = 6.8 Hz, 2H), 2.19–2.04 (m, 2H), 2.00–1.85 (m, 2H), 1.78–1.72 (m, 1H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Ethyl 2-(4,4-Difluorocyclohexyl)acetate (<b>12b</b>)</h3><div class="NLM_p last">Step 1: A solution of 4,4-difluorocyclohexanone (7.35 g, 54.8 mmol) and ethyl 2-(triphenylphosphoranylidene)acetate (21.0 g, 60.3 mmol) in toluene (100 mL) was stirred at 100 °C for 16 h then was cooled to room temperature and concentrated in vacuo. The residue was triturated with Et<sub>2</sub>O and the solid formed (Ph<sub>3</sub>PO) removed by filtration and rinsed with Et<sub>2</sub>O. The combined organics were concentrated in vacuo. Purification of the residue obtained by flash chromatography on silica gel (340 g column, 10% GLOBAL gradient (AcOEt in hexanes)) gave ethyl 2-(4,4-difluorocyclohexylidene)acetate (11.0 g, 99%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 5.75 (s, 1H), 4.18 (q, <i>J</i> = 7.1 Hz, 2H), 3.10–3.01 (m, 2H), 2.46–2.37 (m, 2H), 2.13–1.96 (m, 4H), 1.30 (t, <i>J</i> = 7.1 Hz, 3H). Step 2: A solution of ethyl 2-(4,4-difluorocyclohexylidene)acetate (11.0 g, 54.0 mmol) in MeOH (160 mL) at room temperature was treated with palladium on carbon (10% w/w, 50% wet, 1 g), and the resulting mixture was stirred under an atmosphere of hydrogen (1 bar) for 16 h. The catalyst was removed using a pad of Celite (10 g) and rinsed with EtOAc. The combined organics were concentrated in vacuo to give ethyl 2-(4,4-difluorocyclohexyl)acetate (<b>12b</b>) (10.94 g, 98%) as a colorless oil which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.16 (q, <i>J</i> = 7.1 Hz, 2H), 2.26 (d, <i>J</i> = 7.1 Hz, 2H), 2.16–2.04 (m, 2H), 1.97–1.66 (m, 5H), 1.42–1.31 (m, 2H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 2-(1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)acetaldehyde (<b>13a</b>)</h3><div class="NLM_p last">Step 1: A solution of ethyl 2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)acetate (<b>12a</b>) (6.42 g, 29.1 mmol) in THF (150 mL) under nitrogen at −30 °C was treated with LiAlH<sub>4</sub> (1 M in Et<sub>2</sub>O, 29.1 mL, 29.1 mmol), and the resulting mixture was stirred for 1.5 h between −30 °C and −20 °C. The mixture was then carefully treated with water (0.960 mL, 1 mL/g LiAlH<sub>4</sub>), then with 15% w/w NaOH in water (0.960 mL), and then with 3 times 0.960 mL of water. The corresponding mixture was stirred at room temperature for 10 min. The white precipitate formed was filtered through a 2.5 g pad of Celite and rinsed with THF. The combined organics were concentrated in vacuo to give 4-(2-hydroxyethyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (5.51 g, 106%) as a colorless oil which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.80–3.67 (m, 2H), 3.13–2.93 (m, 4H), 2.15 (m, 2H), 1.97–1.50 (m, 6H). Step 2: A solution of oxalyl chloride (2.27 mL, 26.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (100 mL) at −78 °C under nitrogen was treated with DMSO (2.83 mL, 39.9 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) dropwise, and the resulting mixture was stirred at this temperature for 15 min before being treated with 4-(2-hydroxyethyl)tetrahydro-2<i>H</i>-thiopyran 1,1-dioxide (3.56 g, 20.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (15 mL plus 2 mL rinse). The resulting mixture was stirred for 30 min at this temperature before being treated with NEt<sub>3</sub> (8.35 mL, 59.9 mmol). The resulting mixture was stirred at this temperature for 30 min then was warmed to room temperature and stirred for 30 min before being diluted with CH<sub>2</sub>Cl<sub>2.</sub> The organic phase was washed with water, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organics were dried using a phase separator and concentrated in vacuo to give 2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)acetaldehyde (<b>13a</b>) (3.52 g, 100%) as a white solid which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.77 (br s, 1H), 3.07–2.99 (m, 4H), 2.52 (dd, <i>J</i> = 6.6, 1.0 Hz, 2H), 2.28–2.16 (m, 1H), 2.16–2.05 (m, 2H), 1.99–1.81 (m, 2H).</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2-(4,4-Difluorocyclohexyl)acetaldehyde (<b>13b</b>)</h3><div class="NLM_p last">Step 1: A solution of ethyl 2-(4,4-difluorocyclohexyl)acetate (<b>12b</b>) (10.5 g, 50.9 mmol) in THF (300 mL) under nitrogen at −30 °C was treated with LiAlH<sub>4</sub> (2 M in Et<sub>2</sub>O, 25.5 mL, 51.0 mmol), and the resulting mixture was stirred for 1.5 h between −30 and −20 °C then was carefully treated with water (1.9 mL, 1 mL/g LaH) then 1.9 mL of a 15% w/w NaOH in water then with 3 × 1.9 mL of water. The corresponding mixture was stirred at room temperature for 10 min, then the white precipitate formed was filtered through Celite (2.5 g pad) and rinsed with THF. The combined organics were concentrated in vacuo to give 2-(4,4-difluorocyclohexyl)ethanol (7.34 g, 88%) as a colorless oil which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.77–3.63 (m, 2H), 2.15–2.01 (m, 2H), 1.89–1.62 (m, 5H), 1.59–1.46 (m, 3H), 1.39–1.20 (m, 2H). Step 2: A solution of oxalyl chloride (5.87 mL, 67.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (250 mL) at −78 °C under nitrogen was treated with DMSO (7.93 mL, 112 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) dropwise, and the resulting mixture was stirred at this temperature for 20 min before being treated with 2-(4,4-difluorocyclohexyl)ethanol (7.34 g, 44.7 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL + 2 mL rinse). The resulting mixture was stirred for 30 min at this temperature before being treated with triethylamine (19.9 mL, 143 mmol). The resulting mixture was stirred at this temperature for 1 h, then was warmed to room temperature, stirred 20 min, and diluted with CH<sub>2</sub>Cl<sub>2</sub>. The organic phase was washed with water, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organics were washed with water/brine 1:1, dried using a phase separator, and concentrated in vacuo to give 2-(4,4-difluorocyclohexyl)acetaldehyde (<b>13b</b>) (7.5 g, assumed quantitative) as a yellow solid which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.80 (t, <i>J</i> = 1.6 Hz, 1H), 2.43 (dd, <i>J</i> = 6.8, 1.6, Hz, 2H), 2.17–1.95 (m, 2H), 2.04–1.95 (m, 1H), 1.87–1.78 (m, 3H), 1.78–1.66 (m, 1H), 1.43–1.29 (m, 2H).</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 2-(Tetrahydro-2<i>H</i>-pyran-4-yl)acetaldehyde (<b>13c</b>)</h3><div class="NLM_p last">Step 1: A solution of 2-(tetrahydro-2<i>H</i>-pyran-4-yl)acetic acid (<b>12c</b>) (5.00 g, 34.7 mmol) in THF (100 mL) at 0 °C under nitrogen was slowly treated with LiAlH<sub>4</sub> (1 M in THF, 34.7 mL, 34.7 mmol). The mixture was then allowed to warm to room temperature and was stirred for 24 h. The mixture was then cooled to 0 °C and carefully treated with water (1.3 mL, 1 mL/g LiAlH<sub>4</sub>) then with 1.3 mL of 15% w/w NaOH in water then with 3 × 1.3 mL of water. The resulting mixture was stirred at room temperature for 10 min, then the white precipitate formed was filtered through a pad of Celite (2.5 g) and rinsed with EtOAc. The combined organic phases were concentrated in vacuo. The residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub>, and the organic phase was dried using a phase separator and concentrated in vacuo to give 2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanol (4.28 g, 95%) as a very pale-yellow oil which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.96 (dd, <i>J</i> = 11.1 4.0 Hz, 2H), 3.78–3.65 (m, 2H), 3.40 (t, <i>J</i> = 11.7 Hz, 2H), 1.79–1.24 (m, 8H). Step 2: A solution of oxalyl chloride (4.29 mL, 49.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (100 mL) under nitrogen was cooled to −78 °C then treated with DMSO (5.8 mL, 82 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (15 mL) dropwise. The resulting mixture was stirred for 15 min at this temperature before being treated with 2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanol (4.25 g, 32.6 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The resulting mixture was stirred at this temperature for 30 min before being treated with NEt<sub>3</sub> (14.6 mL, 104 mmol) . The resulting mixture was stirred at this temperature for 1 h then was warmed to room temperature and diluted with CH<sub>2</sub>Cl<sub>2</sub> after 20 min. The organic phase was washed with water (which was then extracted with CH<sub>2</sub>Cl<sub>2</sub>), and the combined organics were washed with water again, dried using a phase separator, and concentrated in vacuo to give crude 2-(tetrahydro-2<i>H</i>-pyran-4-yl)acetaldehyde (<b>13c</b>) (5 g, assumed quantitative) as a pale-yellow oil which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.79 (t, <i>J</i> = 1.7 Hz, 1H), 3.99–3.92 (m, 2H), 3.44 (dt, <i>J</i> = 11.8, 2.1 Hz, 2H), 2.40 (dd, <i>J</i> = 6.7, 1.8, Hz, 2H), 2.24–2.08 (m, 1H), 1.69–1.59 (m, 2H), 1.44–1.30 (m, 2H).</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> (<i>E</i>)-<i>tert</i>-Butyl 2-(2-(1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15a</b>)</h3><div class="NLM_p last">Step 1: A solution of diisopropylamine (3.98 mL, 28.0 mmol) in THF (100 mL) at −78 °C under nitrogen was treated with <i>n</i>-butyllithium (1.6 M in hexanes, 16.2 mL, 26.0 mmol) dropwise. After 5 min, the mixture was warmed to 0 °C using an ice bath, stirred 30 min at this temperature, and then cooled again at −78 °C. The mixture was then treated with <i>tert</i>-butyl 3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>14</b>) (5.40 g, 24.0 mmol) in THF (25 mL plus 5 mL rinse). The resulting mixture was stirred for 1 h at this temperature before being treated with 2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)acetaldehyde (<b>13a</b>) (3.52 g, 20.0 mmol) in THF (10 mL plus 2 mL rinse). The resulting mixture was stirred for 2 h at −78 °C then was treated with a saturated NH<sub>4</sub>Cl aqueous solution. The mixture was warmed to room temperature and diluted with EtOAc and water. The layers were separated. The aqueous phase was extracted with EtOAc, and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo to give crude <i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-1-hydroxyethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (9.0 g, 112%) as a pale-yellow solid which was used in the next step without further purification. Step 2: A solution of crude <i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-1-hydroxyethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (9.0 g, 20 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (100 mL) at 0 °C under nitrogen was treated with NEt<sub>3</sub> (5.57 mL, 39.9 mmol) then methanesulfonic anhydride (3.83 g, 22.0 mmol) and the resulting mixture was stirred at this temperature for 40 min and then was washed with water. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> and the combined organics were dried using a phase separator and concentrated in vacuo to give crude <i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-1-((methylsulfonyl)oxy)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (9.58 g, 100%) as a white foam which was used in the next step without further purification. Step 3: A solution of crude <i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)-1-((methylsulfonyl)oxy)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (9.58 g, 20.0 mmol) in THF (100 mL) at room temperature was treated with DBU (6.02 mL, 39.9 mmol), and the resulting mixture was stirred at this temperature for 30 min then most of the solvent was evaporated in vacuo. The residue was partitioned between EtOAc and water, and the layers were separated. The aqueous phase was extracted twice with EtOAc, and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo to give a yellow solid. Trituration of this residue with methanol gave (<i>E</i>)-<i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15a</b>) (4.11 g, 54%) as a white solid. The mother liquors were concentrated in vacuo. Purification of the residue obtained by flash chromatography on silica gel (100 g column, 50% GLOBAL gradient (AcOEt in hexanes)) gave (<i>E</i>)-<i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15a</b>) (1.18 g, 15%) as a white solid after trituration with MeOH. Overall yield of 69% over 4 steps. The stereochemistry of the alkene was confirmed by NMR. LCMS (method high pH): Retention time 0.93 min, [M + H]<sup>+</sup> = 384.3. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 6.36 (t, <i>J</i> = 7.9 Hz, 1H), 5.01 (d, <i>J</i> = 6.0 Hz, 1H), 4.33 (dd, <i>J</i> = 6.0, 4.5 Hz, 1H), 3.15–3.07 (m, 2H), 3.04–2.97 (m, 2H), 2.63 (dd, <i>J</i> = 18.0, 4.5 Hz, 1H), 2.38 (d, <i>J</i> = 18.3 Hz, 1H), 2.25–2.18 (m, 2H), 2.18–2.09 (m, 2H), 1.96 (d, <i>J</i> = 13.9 Hz, 2H), 1.85–1.76 (m, 1H), 1.71–1.61 (m, 3H), 1.61–1.53 (m, 1H), 1.38 (s, 9H).</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> (<i>E</i>)-<i>tert</i>-Butyl 2-(2-(4,4-Difluorocyclohexyl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15b</b>)</h3><div class="NLM_p last">Step 1: A solution of diisopropylamine (8.92 mL, 62.6 mmol) in THF (200 mL) at −78 °C under nitrogen was treated with <i>n</i>-butyllithium (1.6 M in hexanes, 36.3 mL, 58.1 mmol) dropwise, and after 5 min the mixture was warmed to 0 °C using an ice bath, stirred 30 min at this temperature, then cooled again at −78 °C. The solution was then treated with <i>tert</i>-butyl 3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>14</b>) (12.1 g, 53.6 mmol) in THF (25 mL plus 3 mL rinse), and the resulting mixture was stirred for 1 h at this temperature before being treated with 2-(4,4-difluorocyclohexyl)acetaldehyde (<b>13b</b>) (7.25 g, 44.7 mmol) in THF (25 mL plus 4 mL rinse). The resulting mixture was stirred for 2 h at this temperature then was treated with a saturated NH<sub>4</sub>Cl aqueous solution and warmed to room temperature. The mixture was partitioned between EtOAc and water, and the layers were separated. The aqueous phase was extracted with EtOAc, and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo to give crude <i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)-1-hydroxyethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (20 g, assumed quantitative) as a pale-yellow solid which was used in the next step without further purification. Step 2: A solution of crude <i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)-1-hydroxyethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (20 g, 44.7 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (200 mL) at 0 °C under nitrogen was treated with NEt<sub>3</sub> (12.5 mL, 89.0 mmol) then methanesulfonic anhydride (8.57 g, 49.2 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL), and the resulting mixture was stirred at this temperature for 40 min. The organic phase was then washed with water. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub>, and the combined organics were washed with brine/water 1:1, dried using a phase separator, and concentrated in vacuo to give crude <i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)-1-((methylsulfonyl)oxy)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (23 g, assumed quantitative) as a brown oil which was used in the next step without further purification. Step 3: A solution of crude <i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)-1-((methylsulfonyl)oxy)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (23 g, 44.7 mmol) in THF (100 mL) at room temperature was treated with DBU (13.5 mL, 89.0 mmol), and the resulting mixture was stirred at this temperature for 30 min then most of the solvent was removed in vacuo. The residue was partitioned between EtOAc and water and the layers were separated. The aqueous phase was extracted with EtOAc and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue obtained by flash chromatography on silica gel (330 g column, 10% GLOBAL gradient (AcOEt in hexanes)) gave (<i>E</i>)-<i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (17.94 g, 108%) as a pale-yellow oil contaminated with <i>tert</i>-butyl 3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate. This material was used in the next step without further purification. LCMS (method high pH): Retention time 1.29 min, [M + H]<sup>+</sup> = 370.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.56 (s, 1H), 5.17–4.94 (m, 1H), 4.50 (br s, 3H), 2.90–2.56 (m, 3H), 2.47–2.31 (m, 3H), 2.30–1.99 (m, 3H), 1.88–1.58 (m, 4H), 1.45 (s, 9H), 1.41–1.24 (m, 2H).</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> (<i>E</i>)-<i>tert</i>-Butyl-3-oxo-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethylidene)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15c</b>)</h3><div class="NLM_p last">Step 1: A solution of diisopropylamine (6.50 mL, 45.6 mmol) in THF (150 mL) at −78 °C under nitrogen was treated with <i>n</i>-butyllithium (1.6 M in hexanes, 26.5 mL, 42.4 mmol), and after 5 min, the resulting mixture was warmed to 0 °C using an ice bath and stirred at this temperature for 30 min before being cooled again to −78 °C. The mixture was then treated with <i>tert</i>-butyl 3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>14</b>) (8.81 g, 39.1 mmol) (in 20 mL of THF) dropwise, and the mixture was stirred 90 min before being treated with crude 2-(tetrahydro-2<i>H</i>-pyran-4-yl)acetaldehyde (<b>13c</b>) (4.18 g, 32.6 mmol) in 20 mL of THF. The resulting yellow solution was stirred for 2 h at this temperature before a saturated NH<sub>4</sub>Cl aqueous solution (100 mL) was added at −78 °C. The mixture was warmed to room temperature and diluted with EtOAc and water. The layers were separated, and the aqueous phase was extracted twice with EtOAc. The combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo to give 14 g of crude <i>tert</i>-butyl 2-(1-hydroxy-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate as a brown oil which was used in the next step without further purification. Assumed quantitative. Step 2: A solution of crude <i>tert</i>-butyl 2-(1-hydroxy-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (14.0 g, 32,6 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (200 mL) at 0 °C under nitrogen was treated with NEt<sub>3</sub> (9.09 mL, 65.2 mmol) then with methanesulfonic anhydride (5.96 g, 34.2 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The resulting pale-yellow mixture was stirred at this temperature for 1 h then was diluted with CH<sub>2</sub>Cl<sub>2</sub> and washed with water. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2.</sub> The combined organics were dried using a phase separator and concentrated in vacuo to give crude <i>tert</i>-butyl 2-(1-((methylsulfonyl)oxy)-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (15.0 g, considered quantitative) as a light-brown oil which was used in the next step without further purification. Step 3: A solution of crude <i>tert</i>-butyl 2-(1-((methylsulfonyl)oxy)-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (15.0 g, 32.6 mmol) in THF (200 mL) at room temperature was treated with DBU (9.83 mL, 65.2 mmol), and the resulting yellow-brown mixture was stirred at this temperature for 45 min then most of the THF was removed in vacuo. The residue was partitioned between EtOAc and water, and the layers were separated. The aqueous phase was extracted with EtOAc, and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (330 G column, 40% GLOBAL gradient (AcOEt in hexanes)) gave (<i>E</i>)-<i>tert</i>-butyl 3-oxo-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethylidene)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15c</b>) (7.6 g, 70% yield over 4 steps) as a colorless oil. LCMS (method high pH): Retention time 1.09 min, [M + H]<sup>+</sup> = 336.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.63–6.51 (m, 1H), 5.18–4.96 (m, 1H), 4.58–4.40 (m, 1H), 3.97 (dt, <i>J</i> = 11.5, 2.0 Hz, 2H), 3.43–3.30 (m, 2H), 2.91–2.62 (m, 1H), 2.41 (dd, <i>J</i> = 18.1, 1.0 Hz, 1H), 2.31–2.04 (m, 4H), 1.77–1.58 (m, 5H), 1.47 (s, 9H), 1.42–1.26 (m, 2H).</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-Butyl 3-Amino-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16a</b>)</h3><div class="NLM_p last">Step 1: (<i>E</i>)-<i>tert</i>-Butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15a</b>) (5.5 g, 14.34 mmol) was suspended in warm MeOH (150 mL), and THF (50 mL) was added to facilitate dissolution. The suspension was then treated with palladium on carbon (10% w/w, 50% wet, 1 g), and the resulting mixture was stirred at room temperature under an atmosphere of hydrogen (1 bar) for 4 h. The catalyst was removed using a pad of Celite (2.5 g) and rinsed with AcOEt and the combined organics were concentrated in vacuo to give a white solid. Purification of this residue by flash chromatography on silica gel (25 g column, 50% Global gradient (AcOEt in hexanes)) gave (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (4.1 g, 74%) as a white solid. The stereochemistry of this derivative was confirmed by NMR experiments. <sup>1</sup>H NMR (400 MHz DMSO-<i>d</i><sub>6</sub>): δ 4.46–4.32 (m, 1H), 4.26 (dd, <i>J</i> = 7.5, 4.0 Hz, 1H), 3.08–2.89 (m, 4H), 2.64 (dd, <i>J</i> = 15.1, 4.5 Hz, 1H), 2.49–2.43 (m, 1H), 2.20 (dd, <i>J</i> = 15.0, 1.9 Hz, 1H), 2.03 (s, 2H), 2.01–1.91 (m, 1H), 1.92–1.82 (m, 1H), 1.90–1.81 (m, 1H), 1.74–1.62 (m, 2H), 1.67–1.55 (m, 1H), 1.57–1.51 (m, 1H), 1.61–1.50 (m, 1H), 1.49 (s, 9H), 1.40–1.29 (m, 2H),1.08 (ddt, <i>J</i> = 14.0, 9.0, 7.0 Hz, 1H). Step 2: A solution of (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (0.450 g, 1.17 mmol) in EtOH (20 mL) at room temperature was treated with hydroxylamine hydrochloride (0.162 g, 2.33 mmol) and pyridine (0.189 mL, 2.33 mmol), and the resulting mixture was stirred at reflux for 1 h then was cooled to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a phase separator and concentrated in vacuo to give (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-3-(hydroxyimino)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.50 g, 107%, mixture of <i>Z</i> and <i>E</i> isomers) as a colorless oil which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.60–7.39 (m, 1H), 4.42–4.07 (m, 2H), 3.24–3.14 (m, 1H), 3.05 (br s, 2H), 2.96 (br s, 2H), 2.61–2.33 (m, 1H), 2.11 (br s, 2H), 2.10–1.65 (m, 8H), 1.60–1.36 (m, 12H), 1.25–1.11 (m, 1H). Step 3: A solution of (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-3-(hydroxyimino)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.30 g, 0.75 mmol) in EtOH (20 mL) at reflux under nitrogen was treated with sodium (1 g) portionwise (around 50 mg each) over 1 h, allowing effervescence to cease before adding the next piece. The mixture was then cooled to room temperature and concentrated in vacuo. The residue was partitioned between water and EtOAc, and the layers were separated. The organic phase was dried over MgSO<sub>4</sub> and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-amino-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16a</b>) (0.22 g, 76%) as a pale-yellow gum. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 4.33–3.94 (m, 1H), 3.05 (br s, 1H), 2.97 (d, <i>J</i> = 13.6 Hz, 1H), 2.71–2.62 (m, 1H), 2.14 (br s, 2H), 2.03–1.95 (m, 1H), 1.89–1.72 (m, 7H), 1.70–1.19 (m, 19H), 1.12–1.02 (m, 1H).</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-Butyl 3-Amino-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16b</b>)</h3><div class="NLM_p last">Step 1: A solution of (<i>E</i>)-<i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)ethylidene)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15b</b>) (14.5 g, 39.2 mmol) in MeOH (250 mL) was treated with palladium on carbon (10% w/w, 50% wet, 2 g), and the resulting mixture was stirred under an atmosphere of hydrogen for 16 h. The catalyst was filtered off using a pad of Celite (10 g) and rinsed with EtOAc and the combined organics were concentrated in vacuo. Purification of this residue by flash chromatography on silica gel (340 g column, 20% Global gradient (AcOEt in hexanes)) gave (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (7.07 g, 48%) as a white solid. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i>6) δ 4.34–4.29 (m, 1H), 4.24–4.11 (m, 1H), 2.68–2.52 (m, 1H), 2.46–2.38 (m, 1H), 2.17 (d, <i>J</i> = 14.7 Hz, 1H), 2.03–1.93 (m, 2H), 1.96–1.89 (m, 1H), 1.91–1.80 (m, 1H), 1.85–1.78 (m, 1H), 1.80–1.69 (m, 4H), 1.53–1.45 (m, 2H), 1.44 (s, 9H), 1.38–1.29 (m, 1H), 1.26–1.20 (m, 2H),1.19–1.07 (m, 2H), 1.06–0.98 (m, 1H). Step 2: A solution of (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)ethyl)-3-oxo-8-azabicyclo[3.2.1]octane-8-carboxylate (3.5 g, 9.4 mmol) in EtOH (20 mL) at room temperature was treated with hydroxylamine hydrochloride (1.31 g, 18.8 mmol) and pyridine (1.52 mL, 18.8 mmol), and the resulting mixture was stirred at reflux for 1 h then was cooled to room temperature and diluted with water (40 mL). The mixture was stirred for 30 min, then the precipitate formed was filtered off and dried under vacuum at 40 °C to give (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)ethyl)-3-(hydroxyimino)-8-azabicyclo[3.2.1]octane-8-carboxylate (3.35 g, 92%, mixture of <i>Z</i> and <i>E</i> isomers) which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.65–7.42 (m, 1H), 4.44–4.08 (m, 2H), 3.24–3.09 (m, 1H), 2.65–2.36 (m, 1H), 2.06 (br s, 4H), 1.80 (d, <i>J</i> = 12.5 Hz, 4H), 1.72–1.57 (m, 7H), 1.50 (s, 9H), 1.42–1.15 (m, 3H). Step 3: A solution of (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 2-(2-(4,4-difluorocyclohexyl)ethyl)-3-(hydroxyimino)-8-azabicyclo[3.2.1]octane-8-carboxylate (1.0 g, 2.6 mmol) in EtOH (60 mL) at reflux under nitrogen was treated with sodium (1 g) in small portions (around 50 mg each) over 1 h, allowing effervescence to cease before adding the next piece. The mixture was then cooled to room temperature and concentrated in vacuo. The residue was partitioned between water and EtOAc, and the layers were separated. The organic phase was dried over MgSO<sub>4</sub> and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–10% (2N NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-amino-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16b</b>) (0.45 g, 47%) as a pale-yellow gum. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 4.05–3.98 (m, 1H), 3.96–3.84 (m, 1H), 2.55–2.43 (m, 4H), 2.00 (d, <i>J</i> = 7.1 Hz, 2H), 1.89–1.58 (m, 8H), 1.56–1.23 (m, 4H), 1.45 (s, 9H), 1.16 (d, <i>J</i> = 10.6 Hz, 4H), 0.97–0.84 (m, 1H).</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-Butyl 3-Amino-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16c</b>)</h3><div class="NLM_p last">Step 1: A solution of (E)-<i>tert</i>-butyl 3-oxo-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethylidene)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>15c</b>) (7.56 g, 22.5 mmol) in MeOH (150 mL) was treated with palladium on carbon (10% w/w, 50% wet, 1.5 g), and the resulting mixture was stirred under hydrogen (1 bar) at room temperature for 3 h. The catalyst was filtered off through a pad of Celite (2.5 g) and rinsed with MeOH. The combined organics were concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (100 g column, 30% GLOBAL gradient (AcOEt in hexanes)) gave (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-oxo-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (5.97 g, 79%) as a white foam. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 4.34–4.28 (m, 1H), 4.24–4.13 (m, 1H), 3.85–3.78 (m, 2H), 3.28–3.22 (m, 2H), 2.68–2.54 (m, 1H), 2.46–2.37 (m, 1H), 2.17 (d, <i>J</i> = 15.0 Hz, 1H), 2.00–1.89 (m, 1H), 1.87–1.75 (m, 2H), 1.56 (d, <i>J</i> = 12.8 Hz, 2H), 1.51–1.44 (m, 3H), 1.45 (s, 9H), 1.25–1.19 (m, 2H), 1.18–1.07 (m, 2H), 1.06–0.98 (m, 1H). Step 2: A solution of (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-oxo-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.45 g, 1.3 mmol) in EtOH (20 mL) at room temperature was treated with hydroxylamine hydrochloride (0.185 g, 2.67 mmol) and pyridine (0.216 mL, 2.67 mmol). The resulting mixture was stirred at reflux for 1 h then was cooled to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a phase separator and concentrated in vacuo to give (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-(hydroxyimino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.42 g, 89%) as a colorless oil (mixture of <i>Z</i> and <i>E</i> isomers) which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.97–8.69 (m, 1H), 4.44–4.09 (m, 2H), 4.02–3.84 (m, 2H), 3.37 (t, <i>J</i> = 11.7 Hz, 2H), 3.24–3.08 (m, 1H), 2.66–2.34 (m, 1H), 1.95 (br s, 5H), 1.61 (d, <i>J</i> = 13.0 Hz, 3H), 1.49 (s, 9H), 1.12–1.51 (m, 6H). Step 3: Sodium (1.0 g, 43 mmol) was cut into small pieces and placed in a beaker containing cyclohexane, then was added piece by piece over 30 min to a solution of (1<i>R</i>*,2<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-(hydroxyimino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.40 g, 1.1 mmol) in EtOH (30 mL) at reflux. The mixture was stirred for a further 30 min, then was cooled to room temperature and diluted with water (100 mL). The mixture was extracted with EtOAc (2 × 50 mL), and the combined organics were dried over MgSO<sub>4</sub> and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (gradient: 0–10% (2 M NH<sub>3</sub> in MeOH) in CH<sub>2</sub>Cl<sub>2</sub>) gave (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-amino-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16</b>) (0.22 g, 57%) as a pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.37–4.05 (m, 2H), 3.99 (dd, <i>J</i> = 11.0, 3.0 Hz, 2H), 3.41 (d, <i>J</i> = 2.0 Hz, 2H), 2.80–2.66 (m, 1H), 2.20–1.93 (m, 3H), 1.83 (d, <i>J</i> = 3.3 Hz, 3H), 1.75–1.55 (m, 5H), 1.51 (s, 9H), 1.48–1.22 (m, 5H), 1.14–1.01 (m, 2H).</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (3<i>R</i>,4<i>r</i>,5<i>S</i>)-<i>tert</i>-Butyl 4-Amino-3,5-dimethoxypiperidine-1-carboxylate (<b>18a</b>)</h3><div class="NLM_p last">Step 1: A solution of (3<i>R</i>*,4<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 4-azido-3-hydroxy-5-methoxypiperidine-1-carboxylate (see ref <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>, 560 mg, 2.06 mmol) in DMF (8 mL) at 0 °C under nitrogen was treated with sodium hydride (60% w/w in mineral oil, 99 mg, 2.5 mmol) and then after 5 min with methyl iodide (0.257 mL, 4.11 mmol). The resulting mixture was stirred for 30 min at this temperature then was diluted with water. The aqueous phase was extracted three times with Et<sub>2</sub>O. The combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, 20% Global gradient (AcOEt in hexanes)) gave (3<i>R</i>,4<i>r</i>,5<i>S</i>)-<i>tert</i>-butyl 4-azido-3,5-dimethoxypiperidine-1-carboxylate (530 mg, 90%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.55–4.17 (m, 2H), 3.51 (s, 6H), 3.31–3.22 (m, 1H), 3.11–2.94 (m, 2H), 2.56–2.37 (m, 2H), 1.49 (s, 9H). Step 2: A solution of (3<i>R</i>,4<i>r</i>,5<i>S</i>)-<i>tert</i>-butyl 4-azido-3,5-dimethoxypiperidine-1-carboxylate (500 mg, 1.75 mmol) in MeOH (30 mL) at room temperature was treated with palladium on carbon (10% w/w, 50% wet, 100 mg), and the resulting mixture was stirred at room temperature for 16 h under hydrogen (1 bar). The catalyst was filtered off using a pad of Celite (2.5 g) and rinsed with MeOH. The combined organics were concentrated in vacuo to give (3<i>R</i>,4<i>r</i>,5<i>S</i>)-<i>tert</i>-butyl 4-amino-3,5-dimethoxypiperidine-1-carboxylate (<b>18a</b>) (410 mg, 90%) as a black solid which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.55–4.14 (m, 2H), 3.46 (s, 6H), 3.02–2.86 (m, 2H), 2.75–2.67 (m, 1H), 2.51–2.32 (m, 2H), 1.81 (br s, 2H), 1.48 (s, 9H).</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-Butyl 4-amino-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (<b>18b</b>)</h3><div class="NLM_p last">Step 1: A solution of (tetrahydro-2<i>H</i>-pyran-4-yl)methanol (1.50 g, 12.9 mmol) and pyridine (1.04 mL, 12.9 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (30 mL) under nitrogen was cooled to 0 °C using an ice bath, then was treated with Tf<sub>2</sub>O (2.18 mL, 12.9 mmol) dropwise. The resulting mixture was stirred for 1 h at 0 °C, then was washed with water (2 × 30 mL), dried using a phase separator, and concentrated in vacuo to give (tetrahydro-2<i>H</i>-pyran-4-yl)methyl trifluoromethanesulfonate (3.02 g, 94%) as a colorless oil which was used immediately in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.38 (d, <i>J</i> = 6.6 Hz, 2H), 4.08–3.98 (m, 2H), 3.42 (td, <i>J</i> = 11.9, 2.0 Hz, 2H), 2.19–2.06 (m, 1H), 1.74–1.64 (m, 2H), 1.50–1.35 (m, 2H). Step 2: A solution of (3<i>R</i>*,4<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 4-azido-3-hydroxy-5-methoxypiperidine-1-carboxylate (0.82 g, 3.0 mmol) in THF (20 mL) under nitrogen was cooled to 0 °C using an ice bath, then was treated with potassium <i>tert</i>-butoxide (0.439 g, 3.91 mmol). The resulting mixture was stirred at this temperature for 20 min then was treated with the dropwise addition of (tetrahydro-2<i>H</i>-pyran-4-yl)methyl trifluoromethanesulfonate (1.12 g, 4.52 mmol). The resulting mixture was stirred for 1 h at 0 °C, then allowed to warm to room temperature over 1 h before being diluted with water (60 mL). The aqueous phase was extracted with EtOAc (2 × 50 mL), and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (50 g column, gradient: 0–50% EtOAc in cyclohexane) gave (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-azido-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.74 g, 66%) as a pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.53–4.13 (m, 2H), 3.99 (dt, <i>J</i> = 11.5, 2.1 Hz, 2H), 3.51 (s, 3H), 3.46–3.34 (m, 3H), 3.31–3.22 (m, 1H), 3.16–3.06 (m, 1H), 2.91–3.04 (m, 1H), 2.58–2.37 (m, 2H), 1.94–1.78 (m, 1H), 1.75–1.59 (m, 3H), 1.48 (s, 9H), 1.42–1.26 (m, 2H). Step 3: A solution of (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-azido-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.800 g, 2.16 mmol) in MeOH (60 mL) was hydrogenated in the H Cube over a Pd/C catalyst cartridge at atmospheric pressure. The eluant was evaporated in vacuo to give (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-amino-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (<b>18b</b>) (0.72 g, 97%) as a colorless oil, which was used in the next step without purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.53–4.19 (br s, 2H), 3.58–3.44 (m, 2H), 3,51 (s, 3H), 3.33–3.21 (m, 1H), 3.16–3.06 (m, 3H), 3.05–2.93 (m, 3H), 2.59–2.39 (m, 2H), 2.28–2.15 (m, 2H), 2.00–1.80 (m, 3H), 1.51–1.42 (m, 2H), 1.48 (s, 9H).</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 8-(((3<i>S</i>,4<i>R</i>,5<i>R</i>)-3-Methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>22</b>)</h3><div class="NLM_p">Step 1: A mixture of (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-amino-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (<b>18b</b>) (0.200 g, 0.579 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>19a</b>) (0.15 g, 0.53 mmol), sodium <i>tert</i>-butoxide (0.152 g, 1.58 mmol), BrettPhos (0.028 g, 0.053 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.024 g, 0.026 mmol) in THF (2 mL) was stirred at room temperature for 1 h, then at 60 °C for 18 h before being cooled to room temperature and diluted with EtOAc (20 mL). The organic phase was washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–100% EtOAc in cyclohexane) gave (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.12 g, 38%) as a pale-yellow gum. LCMS (method high pH): Retention time 1.42 min, [M + H]<sup>+</sup> = 593. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (d, <i>J</i> = 5.9 Hz, 1H), 7.68 (s, 1H), 7.55–7.47 (m, 2H), 7.45–7.38 (m, 2H), 7.35 (d, <i>J</i> = 7.1 Hz, 1H), 6.73 (d, <i>J</i> = 5.6 Hz, 1H), 6.25 (d, <i>J</i> = 9.0 Hz, 1H), 5.52 (s, 2H), 4.58–4.24 (m, 2H), 3.69–3.31 (m, 9H), 3.08–2.86 (m, 3H), 2.83–2.63 (m, 2H), 2.41 (s, 3H), 1.54 (s, 9H), 1.45–1.34 (m, 1H), 1.29–1.16 (m, 1H), 1.14–1.03 (m, 1H), 1.01–0.87 (m, 1H), 0.80–0.64 (m, 1H). Step 2: A solution of (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (130 mg, 0.219 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was cooled using an ice bath then was treated with NBS (39.0 mg, 0.219 mmol), and the resulting mixture was stirred at this temperature for 1 h. The mixture was then treated with a saturated aqueous sodium metabisulphite solution, and after 10 min the layers were separated. The organic phase was dried using a phase separator and concentrated in vacuo to give (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.15 g, 102%) which was used in the next step without further purification.</div><div class="NLM_p last">LCMS (method high pH): Retention time 1.56 min, [M + H]<sup>+</sup> = 673.6 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.05 (s, 1H), 7.99 (d, <i>J</i> = 1.0 Hz, 1H), 7.50 (d, <i>J</i> = 6.8 Hz, 2H), 7.39–7.44 (m, 2H), 7.33–7.38 (m, 1H), 6.26 (d, <i>J</i> = 9.3 Hz, 1H), 5.52 (s, 2H), 4.17–4.57 (m, 2H), 4.05–4.16 (m, 1H), 3.59–3.73 (m, 2H), 3.43–3.59 (m, 2H), 3.38 (s, 3H), 3.32–3.37 (m, 1H), 2.94–3.11 (m, 3H), 2.60–2.78 (m, 2H), 2.47 (d, <i>J</i> = 1.0 Hz, 3H), 1.54 (s, 9H), 1.39–1.50 (m, 1H), 1.21–1.30 (m, 1H), 1.08–1.17 (m, 1H), 0.98 (qd, <i>J</i> = 12.3, 4.6 Hz, 1H), 0.68–0.82 (m, 1H). Step 3: A 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (0.046 g, 0.335 mmol), (3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 4-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidine-1-carboxylate (0.150 g, 0.223 mmol), K<sub>2</sub>CO<sub>3</sub> (0.093 g, 0.67 mmol), Pd(OAc)<sub>2</sub> (5.0 mg, 0.022 mmol), di((3<i>S</i>,5<i>S</i>,7<i>S</i>)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (8.0 mg, 0.022 mmol), and then was filled with 1,4-dioxane (4 mL), and water (2 mL). The reaction mixture was stirred and degassed for 20 min with nitrogen. The reaction vessel was sealed and the mixture was stirred under microwave irradiations at 100 °C for 30 min then was cooled to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried through a hydrophobic frit and concentrated in vacuo to give a brown residue. This was dissolved in TFA (2 mL), reluxed for 4 h, then cooled to room temperature and concentrated in vacuo. Purification of the residue by MDAP (method high pH) gave 8-(((3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (58 mg, 53%) as a white foam. LCMS (method high pH): Retention time 0.69 min, [M + H]<sup>+</sup> = 494.5. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 12.93–13.92 (m, 1H), 8.52 (d, <i>J</i> = 1.5 Hz, 1H), 8.47 (d, <i>J</i> = 2.0 Hz, 1H), 7.96 (s, 1H), 7.75 (d, <i>J</i> = 1.0 Hz, 1H), 7.52 (d, <i>J</i> = 1.0 Hz, 1H), 6.60–6.81 (m, 1H), 4.54–4.73 (m, 1H), 3.77 (br dd, <i>J</i> = 11.2, 2.4 Hz, 1H), 3.48 (br d, <i>J</i> = 3.9 Hz, 3H), 3.45 (s, 4H), 3.34–3.40 (m, 1H), 3.33–3.42 (m, 1H), 3.25 (dd, <i>J</i> = 8.8, 7.3 Hz, 1H), 3.17–3.22 (m, 1H), 3.11–3.16 (m, 1H), 2.59–2.69 (m, 2H), 2.46 (s, 3H), 2.36 (d, <i>J</i> = 1.0 Hz, 3H), 1.67 (br s, 3H), 1.37 (br dd, <i>J</i> = 13.0, 1.7 Hz, 1H), 1.24–1.32 (m, 1H), 1.11 (qd, <i>J</i> = 12.4, 4.4 Hz, 1H). The two enantiomers of 8-(((3<i>S</i>*,4<i>R</i>*,5<i>R</i>*)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one were separated by chiral chromatograhy. Analytical method: Approximately 0.5 mg of racemate were dissolved in 50% EtOH in heptane (1 mL) and 20 μL were injected onto the column, eluting with 30% EtOH (+0.2%isopropylamine) in heptane, <i>f</i> = 1.0 mL/min, wavelength 215 nm. Column 4.6 mm i.d. × 25 cm Chiralcel OD-H. Preparative method: approximately 44 mg of material were dissolved in 3 mL of EtOH (+0.2% isopropylamine). Injections (3 in total): 1 mL of the solution was injected onto the column eluting with 30% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 25 mL/min, wavelength 215 nm. Column 30 mm × 25 cm Chiralcel OD-H. The appropriate fractions were combined and concentrated in vacuo to give 8-(((3<i>S</i>,4<i>R</i>,5<i>R</i>)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (13 mg, 59%) as the fastest running enantiomer and 8-(((3<i>R</i>,4<i>S</i>,5<i>S</i>)-3-methoxy-5-((tetrahydro-2<i>H</i>-pyran-4-yl)methoxy)piperidin-4-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (11 mg, 50%) as the slowest running enantiomer.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 3-Methyl-5-(5-methylpyridin-3-yl)-8-(((1<i>S</i>,2<i>R</i>,3<i>R</i>,5<i>R</i>)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>23</b>)</h3><div class="NLM_p last">Step 1: A mixture of (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-amino-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16c</b>) (0.18 g, 0.53 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>19a</b>) (0.15 g, 0.53 mmol), sodium <i>tert</i>-butoxide (0.152 g, 1.58 mmol), BrettPhos (0.028 g, 0.053 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.024 g, 0.026 mmol) in THF (2 mL) was stirred at room temperature for 1 h, then at 60 °C for 18 h. The mixture was then cooled to room temperature and diluted with EtOAc (20 mL). The organic phases was washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient 0–80% EtOAc in cyclohexane) gave (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.245 g, 79%). LCMS (method high pH): Retention time 1.08 min, [M + H]<sup>+</sup> = 587.6. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.86 (d, <i>J</i> = 5.6 Hz, 1H), 7.65 (d, <i>J</i> = 1.0 Hz, 1H), 7.52–7.46 (m, 2H), 7.45–7.39 (m, 2H), 7.37–7.30 (m, 1H), 6.70 (d, <i>J</i> = 5.9 Hz, 1H), 5.92 (d, <i>J</i> = 9.0 Hz, 1H), 5.49 (s, 2H), 4.30 (br s, 3H), 3.87 (dd, <i>J</i> = 11.0, 2.7 Hz, 2H), 3.34–3.21 (m, 2H), 2.79 (s, 1H), 2.45 (d, <i>J</i> = 1.0 Hz, 3H), 2.15–1.99 (m, 2H), 1.99–1.80 (m, 3H), 1.79–1.43 (m, 14H), 1.37 (d, <i>J</i> = 10.5 Hz, 2H), 1.29–1.08 (m, 3H). Step 2: A solution of (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.18 g, 0.31 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) at 0 °C was treated with NBS (0.055 g, 0.31 mmol), and the resulting mixture was stirred at this temperature for 1 h then was treated with a 10% w/w sodium metabisulphite aqueous solution. The biphasic mixture was stirred for 20 min, and then the layers were separated. The organic layer was dried using an hydrophobic frit and concentrated in vacuo to give (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.21 g, 103%) as an orange foam which was used in the next step without further purification. LCMS (method high pH): Retention time 1.71 min, [M + H]<sup>+</sup> = 667.5 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (s, 1H), 7.96 (d, <i>J</i> = 1.2 Hz, 1H), 7.50-.46 (m, 2H) 7.45–7.39 (m, 2H), 7.38–7.32 (m, 1H), 5.92 (d, <i>J</i> = 9.0 Hz, 1H), 5.49 (s, 2H), 4.30 (br s, 3H), 3.87 (dd, <i>J</i> = 11.0, 2.7 Hz, 2H), 3.34–3.21 (m, 2H), 2.45 (d, <i>J</i> = 1.0 Hz, 3H), 2.15–1.99 (m, 2H), 1.99–1.79 (m, 3H), 1.78–1.30 (m, 8H), 1.55 (s, 9H), 1.29–1.08 (m, 3H). Step 3: A 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (0.062 g, 0.45 mmol), (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.20 g, 0.30 mmol), K<sub>2</sub>CO<sub>3</sub> (0.125 g, 0.901 mmol), Pd(OAc)<sub>2</sub> (6.75 mg, 0.030 mmol), cataCXium A (10.8 mg, 0.0300 mmol), then was filled with 1,4-dioxane (4 mL) and water (2 mL). The resulting mixture was stirred and degassed for 20 min with nitrogen. The mixture was then stirred at 100 °C for 30 min under microwave irradiations then was cooled to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic layer was dried using an hydrophobic frit and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient 0–100% EtOAc in cyclohexane) gave (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (145 mg, 71%) as a pale-yellow gum. This product was dissolved in TFA (2 mL), and the resulting solution was stirred at reflux for 3 h then was cooled to room temperature and concentrated in vacuo. The residue was dissolved in MeOH and loaded onto a 5 g SCX2 cartridge, which was washed with MeOH (10 mL) then eluted with a 2N NH<sub>3</sub> solution in MeOH. The eluant was evaporated in vacuo to give 3-methyl-5-(5-methylpyridin-3-yl)-8-(((1<i>S</i>*,2<i>R</i>*,3<i>R</i>*,5<i>R</i>*)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (92 mg, 63%) as a pale-yellow foam. LCMS (method high pH): Retention time 0.82 min, [M + H]<sup>+</sup> = 488.6. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.44 (d, <i>J</i> = 1.5 Hz, 1H), 8.37 (d, <i>J</i> = 1.8 Hz, 1H), 7.70 (s, 1H), 7.59–7.56 (m, 1H), 7.48 (d, <i>J</i> = 1.3 Hz, 1H), 6.55 (d, <i>J</i> = 8.1 Hz, 1H), 4.27 (br s, 1H), 3.80–3.71 (m, 2H), 3.64–3.57 (m, 1H), 3.53 (d, <i>J</i> = 3.8 Hz, 1H), 3.28–3.19 (m, 2H), 2.85 (br s 2H), 2.40 (s, 3H), 2.11 (d, <i>J</i> = 1.3 Hz, 3H), 2.04–1.96 (m, 1H), 1.91–1.65 (m, 6H), 1.64–1.44 (m, 4H), 1.44–1.28 (m, 2H), 1.28–1.17 (m, 1H), 1.16–1.00 (m, 2H). Step 4: The two enantiomers of 3-methyl-5-(5-methylpyridin-3-yl)-8-(((1<i>S</i>*,2<i>R</i>*,3<i>R</i>*,5<i>R</i>*)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one were separated by chiral chromatography. Analytical method: approximately 0.5 mg of racemate were dissolved in 1:1 EtOH/heptane (1 mL). Then 20 μL were injected on column. Eluant: 50% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 1.0 mL/min, wavelength 215 nm. Column 4.6 mm i.d. × 25 cm Chiralpak IC. Preparative method: 35 mg of racemate were dissolved in EtOH (1 mL) then were injected onto the column. Eluant: 40% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 15 mL/min, wavelength = 215 nm. Column 2 cm × 25 cm Chiralpak IC. Total number of injection: 1. The fractions were collected to give 3-methyl-5-(5-methylpyridin-3-yl)-8-(((1<i>S</i>,2<i>R</i>,3<i>R</i>,5<i>R</i>)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>22</b>) (13 mg, 74%) as the fastest running enantiomer and 3-methyl-5-(5-methylpyridin-3-yl)-8-(((1<i>R</i>,2<i>S</i>,3<i>S</i>,5<i>S</i>)-2-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-1,7-naphthyridin-2(1<i>H</i>)-one (11 mg, 63%) as slowest running enantiomer.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 8-(((1<i>S</i>,2<i>R</i>,3<i>R</i>,5<i>R</i>)-2-(2-(1,1-Dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>24</b>)</h3><div class="NLM_p last">Step 1: A mixture of (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-amino-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16a</b>) (0.22 g, 0.57 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>19a</b>) (0.15 g, 0.53 mmol), sodium <i>tert</i>-butoxide (0.152 g, 1.58 mmol), BrettPhos (0.028 g, 0.053 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.024 g, 0.026 mmol) in THF (2 mL) was stirred at room temperature for 1 h then at 60 °C for 18 h. The mixture was cooled to room temperature and diluted with EtOAc (20 mL). The organic phase was washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–80% EtOAc in cyclohexane) gave (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (135 mg, 40%) as a pale-yellow foam. LCMS (method TFA): Retention time 1.06 min, [M + H]<sup>+</sup> = 635. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.84 (d, <i>J</i> = 5.6 Hz, 1H), 7.66 (d, <i>J</i> = 1.0 Hz, 1H), 7.51–7.46 (m, 2H), 7.45–7.38 (m, 2H), 7.35 (d, <i>J</i> = 7.3 Hz, 1H), 6.70 (d, <i>J</i> = 5.9 Hz, 1H), 5.90 (d, <i>J</i> = 9.8 Hz, 1H), 5.54–5.42 (m, 2H), 4.46–4.18 (m, 3H), 2.94–2.70 (m, 4H), 2.40 (d, <i>J</i> = 1.0 Hz, 3H), 2.06 (s, 6H), 1.98–1.80 (m, 6H), 1.79–1.57 (m, 2H), 1.54 (s, 9H), 1.27 (m, <i>J</i> = 7.2 Hz, 2H). Step 2: A solution of (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.13 g, 0.21 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was cooled using an ice bath then was treated with NBS (0.036 g, 0.205 mmol) in two portions, of 22 mg and 14 mg, with 2 min stirring interval between additions. The mixture was then treated with a 10% w/w sodium metabisulphite aqueous solution, and the layers were separated. The organic phase was dried using a phase separator and concentrated in vacuo to give (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.15 g, 103%) as an orange foam which was used in the next step without further purification. LCMS (method high pH): Retention time 1.62 min. [M + H]<sup>+</sup> = 715.5 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.01–7.94 (m, 1H), 7.50–7.40 (m, 2H), 7.36 (d, <i>J</i> = 7.1 Hz, 1H), 5.89 (d, <i>J</i> = 9.5 Hz, 1H), 5.49 (d, <i>J</i> = 5.1 Hz, 2H), 4.45–4.08 (m, 3H), 3.02–2.80 (m, 4H), 2.46 (d, <i>J</i> = 1.0 Hz, 3H), 2.13–1.80 (m, 8H), 1.79–1.61 (m, 5H), 1.57–1.52 (m, 10H), 1.45 (s, 5H). Step 3: A 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (0.043 g, 0.32 mmol), (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.15 g, 0.21 mmol), K<sub>2</sub>CO<sub>3</sub> (0.087 g, 0.63 mmol), Pd(OAc)<sub>2</sub> (4.7 mg, 0.021 mmol), and di((3<i>S</i>,5<i>S</i>,7<i>S</i>)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (7.5 mg, 0.021 mmol) then was filled with 1,4-dioxane (4 mL), and water (2 mL). The resulting mixture was stirred and degassed for 20 min with nitrogen then was stirred at 100 °C for 30 min under microwave irradiations before being cooled to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using an hydrophobic frit and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–100% EtOAc in cyclohexane) gave (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.122 g, 80%) as a yellow gum. LCMS (method high pH): Retention time 1.46 min, [M + H]<sup>+</sup> = 726. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.51–8.47 (m, 1H), 7.80 (s, 1H), 7.74–7.68 (m, 1H), 7.56–7. 47 (m, 3H), 7.46–7.40 (m, 3H), 7.36 (d, <i>J</i> = 7.3 Hz, 1H), 6.02 (d, <i>J</i> = 9.8 Hz, 1H), 5.56–5.46 (m, 2H), 4.52–4.38 (m, 2H), 4.37–4.12 (m, 2H), 2.99–2.74 (m, 6H), 2.44 (s, 3H), 2.37 (d, <i>J</i> = 1.0 Hz, 3H), 2.15–1.82 (m, 8H), 1.80–1.60 (m, 5H), 1.56 (s, 6H), 1.49–1.15 (m, 3H). Step 4: A solution of (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.12 g, 0.16 mmol) in TFA (1 mL) was refluxed for 4 h then was allowed to stand at room temperature for 48 h before being concentrated in vacuo. The brown residue obtained was purified by MDAP (method high pH). The residue obtained was further purified by MDAP (method formic) to give 8-(((1<i>S</i>*,2<i>R</i>*,3<i>R</i>*,5<i>R</i>*)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (60 mg, 68%) as a pale-yellow solid. LCMS (method high pH): Retention time 0.74 min, [M + H]<sup>+</sup> = 536. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.52 (br s, 1H), 8.46 (br s, 1H), 7.85–7.79 (m, 1H), 7.78 (s, 1H), 7.59 (d, <i>J</i> = 3.7 Hz, 2H), 4.61–4.45 (m, 1H), 4.18 (d, <i>J</i> = 4.9 Hz, 1H), 4.11 (br s, 1H), 3.12–2.88 (m, 4H), 2.48 (s, 3H), 2.33 (d, <i>J</i> = 12.0 Hz, 3H), 2.19 (s, 6H), 2.04 (d, <i>J</i> = 11.0 Hz, 3H), 1.78 (br s, 4H), 1.40 (br s, 2H), 1.34–1.19 (m, 1H). Step 5: The two enantiomers of 8-(((1<i>S</i>*,2<i>R</i>*,3<i>R</i>*,5<i>R</i>*)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one were separated by chiral chromatography. Analytical method: Approximately 0.5 mg of material were dissolved in 50% EtOH in heptane (1 mL) and 20 μL were injected onto the column. Eluant: 70% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 1.0 mL/min, wavelength = 215 nm. Column 4.6 mm i.d. × 25 m Chiralpak IC. Preparative method: Approximately 50 mg of material were dissolved in EtOH (1 mL) and isopropylamine (100 μL). The solution was injected onto the column. Eluant: 70% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 25 mL/min, wavelength 215 nm. Column 30 mm × 25 cm Chiralpak IC. The appropriate fractions were collected and concentrated in vacuo to give 8-(((1<i>S</i>,2<i>R</i>,3<i>R</i>,5<i>R</i>)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>24</b>) (13 mg, 52%) as the fastest running enantiomer and 8-(((1<i>R</i>,2<i>S</i>,3<i>S</i>,5<i>S</i>)-2-(2-(1,1-dioxidotetrahydro-2<i>H</i>-thiopyran-4-yl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (13 mg, 52%) as the slowest running enantiomer. Each product was obtained as pale-yellow solid.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 8-(((1<i>S</i>,2<i>S</i>,3<i>S</i>,5<i>R</i>)-2-(2-(4,4-Difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>26</b>)</h3><div class="NLM_p last">Step 1: A mixture of (1<i>R</i>*,2<i>R</i>*,3<i>S</i>*,5<i>S</i>*)-<i>tert</i>-butyl 3-amino-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>16b</b>), (0.36 g, 0.97 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>19a</b>) (0.25 g, 0.88 mmol), sodium <i>tert</i>-butoxide (0.253 g, 2.63 mmol), BrettPhos (0.094 g, 0.18 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.080 g, 0.088 mmol) in THF (2 mL) was stirred at room temperature for 1 h then at 60 °C for 18 h. The mixture was then cooled to room temperature and diluted with EtOAc (20 mL). The mixture was washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–80% EtOAc in cyclohexane) gave (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.51 g, 94%). LCMS (method high pH): Retention time 1.27 min, [M + H]<sup>+</sup> = 621. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.86 (d, <i>J</i> = 5.6 Hz, 1H), 7.65 (d, <i>J</i> = 1.0 Hz, 1H), 7.52–7.46 (m, 2H), 7.46–7.39 (m, 2H), 7.35 (d, <i>J</i> = 7.3 Hz, 1H), 6.70 (d, <i>J</i> = 5.6 Hz, 1H), 5.92 (d, <i>J</i> = 9.3 Hz, 1H), 5.49 (d, <i>J</i> = 2.9 Hz, 2H), 4.45–4.12 (m, 3H), 2.40 (d, <i>J</i> = 1.0 Hz, 3H), 2.20–1.83 (m, 8H), 1.80–1.58 (m, 7H), 1.55 (s, 9H), 1.42–1.06 (m, 5H). Step 2: A solution of (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.30 g, 0.48 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was cooled using an ice bath then was treated with the portionwise addition of NBS (0.086 g, 0.48 mmol) with a 10 min stirring interval between additions. The resulting mixture was stirred for 20 min then was treated with a 10% w/w sodium metabisulphite aqueous solution. The layers were separated, and the organic phase was dried over MgSO<sub>4</sub> and concentrated in vacuo to give (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.35 g, 104%) as a pale-yellow foam which was used in the next step without further purification. LCMS (method high pH): Retention time 1.84 min, [M + H]<sup>+</sup> = 699/701 (1 Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.98 (s, 1H), 7.96 (d, <i>J</i> = 1.0 Hz, 1H), 7.50–7.46 (m, 2H), 7.45–7.39 (m, 2H), 7.35 (d, <i>J</i> = 7.1 Hz, 1H), 5.91 (d, <i>J</i> = 9.5 Hz, 1H), 5.49 (d, <i>J</i> = 2.4 Hz, 2H), 4.31 (m, 3H), 2.45 (d, <i>J</i> = 1.0 Hz, 3H), 2.13–1.79 (m, 8H), 1.75–1.46 (m, 14H), 1.43- 1.06 (m, 7H). Step 3: A 20 mL microwave vial was charged with (5-methylpyridin-3-yl)boronic acid (0.103 g, 0.750 mmol), (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-5-bromo-3-methyl-1,7-naphthyridin-8-yl)amino)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.35 g, 0.50 mmol), K<sub>2</sub>CO<sub>3</sub> (0.207 g, 1.50 mmol), Pd(OAc)<sub>2</sub> (0.011 g, 0.050 mmol), and di((3<i>S</i>,5<i>S</i>,7<i>S</i>)-adamantan-1-yl)(butyl)phosphine (cataCXium A) (0.018 g, 0.050 mmol) then was filled with 1,4-dioxane (4 mL), and water (2 mL). The resulting mixture was stirred and degassed for 20 min with nitrogen, was stirred at 100 °C for 30 min under microwave irradiations, and then was cooled to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using an hydrophobic frit and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–100% EtOAc in cyclohexane) gave (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.33 g, 93%) as a yellow gum. LCMS (method high pH): Retention time 1.68 min, [M + H]<sup>+</sup> = 712. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.49 (d, <i>J</i> = 2.0 Hz, 1H), 8.48 (d, <i>J</i> = 1.5 Hz, 1H), 7.81 (s, 1H), 7.72 (d, <i>J</i> = 1.0 Hz, 1H), 7.54 (s, 1H), 7.51–7.47 (m, 2H), 7.45–7.40 (m, 2H), 7.37–7.31 (m, 1H), 6.05 (d, <i>J</i> = 9.5 Hz, 1H), 5.51 (d, <i>J</i> = 2.4 Hz, 2H), 4.30 (br s, 3H), 2.44 (s, 3H), 2.37 (d, <i>J</i> = 0.7 Hz, 3H), 2.04–1.83 (m, 7H), 1.76–1.59 (m, 6H), 1.56 (s, 10H), 1.33–1.06 (m, 6H). Step 4: A solution of (1<i>S</i>*,2<i>S</i>*,3<i>R</i>*,5<i>R</i>*)-<i>tert</i>-butyl 3-((2-(benzyloxy)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-8-yl)amino)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octane-8-carboxylate (0.33 g, 0.46 mmol) in TFA (3 mL) was refluxed for 5 h then was cooled to room temperature and concentrated in vacuo. The residue was loaded onto a 10 g SCX2 cartridge, which was washed with MeOH (20 mL) and eluted with a 2N NH<sub>3</sub> solution in MeOH (20 mL). The eluant was concentrated in vacuo to give 8-(((1<i>S</i>*,2<i>R</i>*,3<i>R</i>*,5<i>R</i>*)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (0.18 g, 74%) as a white foam. LCMS (method high pH): Retention time 1.01 min, [M + H]<sup>+</sup> = 522. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.44 (d, <i>J</i> = 1.5 Hz, 1H), 8.37 (d, <i>J</i> = 2.0 Hz, 1H),7.69 (s, 1H), 7.61 (s, 1H), 7.51 (d, <i>J</i> = 1.2 Hz, 1H), 6.72 (d, <i>J</i> = 8.1 Hz, 1H), 4.22–4.07 (m, 1H), 3.49–3.18 (m, 5H), 2.38 (s, 3H), 2.08 (d, <i>J</i> = 1.0 Hz, 3H), 1.98–1.84 (m, 3H), 1.81–1.46 (m, 10H), 1.38–1.12 (m, 4H), 1.03–0.94 (m, 2H). Step 5: The two enantiomers of 8-(((1<i>S</i>*,2<i>R</i>*,3<i>R</i>*,5<i>R</i>*)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one were separated by chiral chromatography. Analytical method:: Approximately 0.5 mg of racemate were dissolved in 50% EtOH in heptane (1 mL) and 20 μLwere injected onto the column. Eluant: 10% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 1.0 mL/min, wavelength = 215 nm. Column 4.6 mm i.d. × 25 cm Chiralcel OD-H. Preparative method: Approximately 160 mg were dissolved in EtOH (3 mL) and isopropylamine (1 mL). Then 1 mL of the solution was injected onto the column. Eluant: 10% EtOH (+0.2% isopropylamine) in heptane, <i>f</i> = 30 mL/min, wavelength 215 nm. Column 30 mm × 25 cm Chiralcel OD-H. Total number of injections: 5. The fractions were collected and concentrated in vacuo to give 8-(((1<i>R</i>,2<i>R</i>,3<i>R</i>,5<i>S</i>)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (76 mg, 95%) as the fastest running enantiomer and 8-(((1<i>S</i>,2<i>S</i>,3<i>S</i>,5<i>R</i>)-2-(2-(4,4-difluorocyclohexyl)ethyl)-8-azabicyclo[3.2.1]octan-3-yl)amino)-3-methyl-5-(5-methylpyridin-3-yl)-1,7-naphthyridin-2(1<i>H</i>)-one (<b>25</b>) (71 mg, 89%) as the slowest running enantiomer.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 8-(((3<i>S</i>,4<i>r</i>,5<i>R</i>)-3,5-Dimethoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>30</b>)</h3><div class="NLM_p last">Step 1: A mixture of (3<i>R</i>,4<i>r</i>,5<i>S</i>)-<i>tert</i>-butyl 4-amino-3,5-dimethoxypiperidine-1-carboxylate (<b>18a</b>) (0.20 g, 0.77 mmol), 2-(benzyloxy)-8-chloro-3-methyl-1,7-naphthyridine (<b>19a</b>) (0.14 g, 0.49 mmol), sodium <i>tert</i>-butoxide (0.142 g, 1.48 mmol), BrettPhos (0.026 g, 0.049 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (0.023 g, 0.025 mmol) in THF (5 mL) was stirred at room temperature for 1 h, then at 60 °C for 2 h before being cooled to room temperature and diluted with EtOAc (20 mL). The organic phase was washed with water, dried over MgSO<sub>4</sub>, and concentrated in vacuo to give (3<i>S</i>,4<i>r</i>,5<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3,5-dimethoxypiperidine-1-carboxylate, a brown residue (100 mg, 40%) as a brown oil which was used in the next step without purification. LCMS (method high pH): Retention time 1.41 min, [M + H]<sup>+</sup> = 509. Step 2: A solution of (3<i>S</i>,4<i>r</i>,5<i>R</i>)-<i>tert</i>-butyl 4-((2-(benzyloxy)-3-methyl-1,7-naphthyridin-8-yl)amino)-3,5-dimethoxypiperidine-1-carboxylate (80 mg, 0.16 mmol) in TFA (3 mL) was refluxed for 3 h then was cooled to room temperature and concentrated in vacuo. The residue was dissolved in MeOH (3 mL) and loaded onto a 5 g SCX cartridge. This was washed with MeOH (20 mL) and then eluted with a 2N NH<sub>3</sub> solution in MeOH (20 mL). The eluant was concentrated in vacuo to give 8-(((3<i>S</i>,4<i>r</i>,5<i>R</i>)-3,5-dimethoxypiperidin-4-yl)amino)-3-methyl-1,7-naphthyridin-2(1<i>H</i>)-one (<b>27</b>) (48 mg, 96%) as a beige solid. LCMS (method high pH): Retention time 0.60 min, [M + H]<sup>+</sup> = 319. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 13.3–13.0 (br s, 1H), 8.01 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (d, <i>J</i> = 1.0 Hz, 1H), 6.72 (d, <i>J</i> = 5.4 Hz, 1H), 6.45–6.28 (m, 1H), 4.59–4.43 (m, 1H), 3.48–3.35 (m, 11H), 2.74–2.61 (m, 2H), 2.39 (d, <i>J</i> = 1.0 Hz, 3H).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00862" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00862" class="ext-link">10.1021/acs.jmedchem.8b00862</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full ATAD2 and BRD4 BD1 assay and ChromLogD data for exemplified compounds; full assay data for BRD4 BD1 and BD2; assay data for a set of ATAD2 inhibitors at BD1 for BRD2, BRD3, BRD4, and BRDT; comparison of activities of O- and C-linked piperidine derivatives; data collection and refinement statistics for ATAD2 and BRD4 BD1 X-ray structures; OMIT maps for ATAD2 and BRD4 BD1 X-ray structures; side chain torsion angles of Trp81 in the complex with <b>26</b> compared to other structures of BRD4 BD1 from the Protein Data Bank; supplementary methods; LCMS traces (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_001.docx" class="ext-link">DOCX</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_001.docx">jm8b00862_si_001.docx (1.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_002.csv">jm8b00862_si_002.csv (1.38 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB IDs of new crystal (X-ray) structures: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDN">6HDN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDO">6HDO</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ">6HDQ</a>. Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00862" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53400" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53400" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery Research, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#db8bbaaeb7f59af599bab6b9b4a9b4aebcb39bbca8b0f5b8b4b6"><span class="__cf_email__" data-cfemail="35655440591b741b775458575a475a40525d7552465e1b565a58">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H. Demont</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Discovery Performance Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#43062e2e222d36262f6d0b6d07262e2c2d37032430286d202c2e"><span class="__cf_email__" data-cfemail="d297bfbfb3bca7b7befc9afc96b7bfbdbca692b5a1b9fcb1bdbf">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery Research, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebecca C. Furze</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Discovery Performance Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Michon</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Discovery Performance Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Discovery Performance Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heather Barnett</span> - <span class="hlFld-Affiliation affiliation">Flexible Discovery
Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Discovery Performance Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christina Rau</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Sheppard</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Discovery Performance Unit, , and , GlaxoSmithKline Medicines Research
Centre, Gunnels Wood
Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span>Present Address:
                        For R.J.S.: Oncology Innovative Medicines and Early Development, AstraZeneca, Cambridge Science Park, Milton Road, Cambridge CB4 OWG, U.K</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thilo Werner</span> - <span class="hlFld-Affiliation affiliation">Molecular
Discovery Research, Cellzome GmbH, GlaxoSmithKline, Meyerhofstrasse 1, 69117 Heidelberg, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are current or previous employees of GSK.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4996-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Thanks to Dr. Richard Upton for NMR support and Phylicia Dassardo Joseph and Abigail Lucas for crystallization support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ANCCA</td><td class="NLM_def"><p class="first last">AAA nuclear coregulator cancer-associated protein</p></td></tr><tr><td class="NLM_term">ATAD2</td><td class="NLM_def"><p class="first last">ATPase family AAA domain containing 2</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra terminal domain</p></td></tr><tr><td class="NLM_term">BRD4/7/9</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4/7/9</p></td></tr><tr><td class="NLM_term">BRPF1</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger containing 1</p></td></tr><tr><td class="NLM_term">CBP</td><td class="NLM_def"><p class="first last">CREB binding protein</p></td></tr><tr><td class="NLM_term">CREB</td><td class="NLM_def"><p class="first last">cAMP response element-binding protein</p></td></tr><tr><td class="NLM_term">EP300</td><td class="NLM_def"><p class="first last">histone acetyltransferase p300</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetyl lysine</p></td></tr><tr><td class="NLM_term">pIC<sub>50</sub></td><td class="NLM_def"><p class="first last">–log<sub>10</sub> IC<sub>50</sub></p></td></tr><tr><td class="NLM_term">SMARCA2/4</td><td class="NLM_def"><p class="first last">SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily a, member 2/4</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 40 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petosa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dieuleveult, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitte, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabatel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosset, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoeyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emadali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guardiola, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gérard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span>. <i>J. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjv060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1093%2Fjmcb%2Fmjv060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26459632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=349-362&author=Y.+Morozumiauthor=F.+Boussouarauthor=M.+Tanauthor=A.+Chaikuadauthor=M.+Jamshidikiaauthor=G.+Colakauthor=H.+Heauthor=L.+Nieauthor=C.+Petosaauthor=M.+de+Dieuleveultauthor=S.+Curtetauthor=A.-L.+Vitteauthor=C.+Rabatelauthor=A.+Debernardiauthor=F.-L.+Cossetauthor=E.+Verhoeyenauthor=A.+Emadaliauthor=N.+Schweiferauthor=D.+Gianniauthor=M.+Gutauthor=P.+Guardiolaauthor=S.+Rousseauxauthor=M.+G%C3%A9rardauthor=S.+Knappauthor=Y.+Zhaoauthor=S.+Khochbin&title=Atad2+is+a+generalist+facilitator+of+chromatin+dynamics+in+embryonic+stem+cells&doi=10.1093%2Fjmcb%2Fmjv060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span></div><div class="casAuthors">Morozumi, Yuichi; Boussouar, Faycal; Tan, Minjia; Chaikuad, Apirat; Jamshidikia, Mahya; Colak, Gozde; He, Huang; Nie, Litong; Petosa, Carlo; de Dieuleveult, Maud; Curtet, Sandrine; Vitte, Anne-Laure; Rabatel, Clothilde; Debernardi, Alexandra; Cosset, Francois-Loic; Verhoeyen, Els; Emadali, Anouk; Schweifer, Norbert; Gianni, Davide; Gut, Marta; Guardiola, Philippe; Rousseaux, Sophie; Gerard, Matthieu; Knapp, Stefan; Zhao, Yingming; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-362</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although the conserved AAA ATPase and bromodomain factor, ATAD2, has been described as a transcriptional co-activator upregulated in many cancers, its function remains poorly understood.  Here, using a combination of ChIP-seq, ChIP-proteomics, and RNA-seq expts. in embryonic stem cells where Atad2 is normally highly expressed, we found that Atad2 is an abundant nucleosome-bound protein present on active genes, assocd. with chromatin remodelling, DNA replication, and DNA repair factors.  A structural anal. of its bromodomain and subsequent investigations demonstrate that histone acetylation guides ATAD2 to chromatin, resulting in an overall increase of chromatin accessibility and histone dynamics, which is required for the proper activity of the highly expressed gene fraction of the genome.  While in exponentially growing cells Atad2 appears dispensable for cell growth, in differentiating ES cells Atad2 becomes crit. in sustaining specific gene expression programs, controlling proliferation and differentiation.  Altogether, this work defines Atad2 as a facilitator of general chromatin-templated activities such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMr0U-BpiFIbVg90H21EOLACvtfcHk0lh5enHUeqY0Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D&md5=5a84cdc73ed9b7abeb5595425d07d3eb</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjv060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjv060%26sid%3Dliteratum%253Aachs%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DColak%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DNie%26aufirst%3DL.%26aulast%3DPetosa%26aufirst%3DC.%26aulast%3Dde%2BDieuleveult%26aufirst%3DM.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVitte%26aufirst%3DA.-L.%26aulast%3DRabatel%26aufirst%3DC.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DCosset%26aufirst%3DF.-L.%26aulast%3DVerhoeyen%26aufirst%3DE.%26aulast%3DEmadali%26aufirst%3DA.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DGut%26aufirst%3DM.%26aulast%3DGuardiola%26aufirst%3DP.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DG%25C3%25A9rard%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DAtad2%2520is%2520a%2520generalist%2520facilitator%2520of%2520chromatin%2520dynamics%2520in%2520embryonic%2520stem%2520cells%26jtitle%3DJ.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D8%26spage%3D349%26epage%3D362%26doi%3D10.1093%2Fjmcb%2Fmjv060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERalpha, is required for coregulator occupancy and chromatin modification</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">18067</span>– <span class="NLM_lpage">18072</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705814104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1073%2Fpnas.0705814104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=17998543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18067-18072&author=J.+X.+Zouauthor=A.+S.+Revenkoauthor=L.+B.+Liauthor=A.+T.+Gemoauthor=H.+W.+Chen&title=ANCCA%2C+an+estrogen-regulated+AAA%2B+ATPase+coactivator+for+ERalpha%2C+is+required+for+coregulator+occupancy+and+chromatin+modification&doi=10.1073%2Fpnas.0705814104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERα, is required for coregulator occupancy and chromatin modification</span></div><div class="casAuthors">Zou, June X.; Revenko, Alexey S.; Li, Li B.; Gemo, Abigael T.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18067-18072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AAA+ proteins play crucial roles in diverse biol. processes via their ATPase-driven remodeling of macromol. complexes.  Here the authors report the authors' identification of an evolutionarily conserved AAA+ protein, ANCCA/Pro 2000, endowed with a bromodomain that is strongly induced by estrogen in human breast cancer cells and is a direct target of protooncogene ACTR/AlB1/SRC-3.  The authors found that ANCCA assocs. directly with estrogen-bound estrogen receptor (ER) α and ACTR.  It is selectively recruited, upon estrogen stimulation, to a subset of ERα target genes including cyclin D1, c-myc, and E2F1 and is required for their estrogen-induced expression as well as breast cancer cell proliferation.  Further studies indicate that ANCCA binds and hydrolyzes ATP and is crit. for recruitment of coregulator CBP and histone hyperacetylation at the ER target chromatin.  Moreover, mutations at the ATP binding motifs rendered ANCCA defective as a coactivator in mediating estrogen induction of gene expression.  Together, the authors' findings reveal an unexpected layer of regulatory mechanism in hormone signaling mediated by ANCCA and suggest that hormone-induced assembly of transcriptional coregulator complexes at chromatin is a process facilitated by AAA+ ATPase proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NFjjiBVoRbVg90H21EOLACvtfcHk0lh5enHUeqY0Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO&md5=618fcc431bf4b0a7589213fea55732ca</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705814104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705814104%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DLi%26aufirst%3DL.%2BB.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252C%2520an%2520estrogen-regulated%2520AAA%252B%2520ATPase%2520coactivator%2520for%2520ERalpha%252C%2520is%2520required%2520for%2520coregulator%2520occupancy%2520and%2520chromatin%2520modification%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D18067%26epage%3D18072%26doi%3D10.1073%2Fpnas.0705814104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardiff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA/ATAD2 overexpression identifies breast cancer patients with poor prognosis, acting to drive proliferation and survival of triple-negative cells through control of B-Myb and EZH2</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">9402</span>– <span class="NLM_lpage">9412</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-1199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F0008-5472.CAN-10-1199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20864510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=9402-9412&author=E.+V.+Kalashnikovaauthor=A.+S.+Revenkoauthor=A.+T.+Gemoauthor=N.+P.+Andrewsauthor=C.+G.+Tepperauthor=J.+X.+Zouauthor=R.+D.+Cardiffauthor=A.+D.+Borowskyauthor=H.+W.+Chen&title=ANCCA%2FATAD2+overexpression+identifies+breast+cancer+patients+with+poor+prognosis%2C+acting+to+drive+proliferation+and+survival+of+triple-negative+cells+through+control+of+B-Myb+and+EZH2&doi=10.1158%2F0008-5472.CAN-10-1199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA/ATAD2 Overexpression Identifies Breast Cancer Patients with Poor Prognosis, Acting to Drive Proliferation and Survival of Triple-Negative Cells through Control of B-Myb and EZH2</span></div><div class="casAuthors">Kalashnikova, Ekaterina V.; Revenko, Alexey S.; Gemo, Abigael T.; Andrews, Nicolas P.; Tepper, Clifford G.; Zou, June X.; Cardiff, Robert D.; Borowsky, Alexander D.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9402-9412</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Chromatin coregulators are important factors in tumorigenesis and cancer progression.  ANCCA is an AAA+ ATPase and a bromodomain-contg. nuclear coactivator for the estrogen and androgen receptors that is crucial for assembly of chromatin-modifying complexes and proliferation of hormone-responsive cancer cells.  In this study, we show that ANCCA is overexpressed in >70% of breast tumors and that its high protein level correlates well with tumor histol. grades (P < 0.0001), highlighting ANCCA as a prognostic factor for poor overall survival and disease recurrence.  Strikingly, high-level ANCCA correlated with triple-neg. tumors that represent highly aggressive disease.  Anal. of ANCCA transcript levels in multiple expression profiles of breast cancer identified ANCCA as a common signature gene, indicating that elevated transcripts also strongly correlate with tumor metastasis and poor survival.  Biol. and mechanistic investigations revealed that ANCCA is crucial for proliferation and survival of triple-neg./basal-like cancer cells and that it controls the expression of B-Myb, histone methyltransferase EZH2, and an Rb-E2F core program for proliferation, along with a subset of key mitotic kinesins and cell survival genes (IRS2, VEGF, and Akt1).  In particular, ANCCA overexpression correlated strongly with EZH2 in tumors.  Our results suggest that ANCCA may integrate multiple oncogenic programs in breast cancer, serving in particular as a prognostic marker and a therapeutic target for triple-neg. cancers.  Cancer Res; 70(22); 9402-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-adY-FJac6rVg90H21EOLACvtfcHk0lirwz7_D_9QyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP&md5=d7f74575c9c8cf4ebebdcfa5e3cc45fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-1199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-1199%26sid%3Dliteratum%253Aachs%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DAndrews%26aufirst%3DN.%2BP.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DCardiff%26aufirst%3DR.%2BD.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252FATAD2%2520overexpression%2520identifies%2520breast%2520cancer%2520patients%2520with%2520poor%2520prognosis%252C%2520acting%2520to%2520drive%2520proliferation%2520and%2520survival%2520of%2520triple-negative%2520cells%2520through%2520control%2520of%2520B-Myb%2520and%2520EZH2%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D9402%26epage%3D9412%26doi%3D10.1158%2F0008-5472.CAN-10-1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lestrat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoffier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virolle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">5171</span>– <span class="NLM_lpage">5181</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+characterization+of+ATAD2+as+a+new+cancer%2Ftestis+factor+and+a+predictor+of+poor+prognosis+in+breast+and+lung+cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0lirwz7_D_9QyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520characterization%2520of%2520ATAD2%2520as%2520a%2520new%2520cancer%252Ftestis%2520factor%2520and%2520a%2520predictor%2520of%2520poor%2520prognosis%2520in%2520breast%2520and%2520lung%2520cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fouret, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffaire, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beau-Faller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazieres, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Validire, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camilleri-Broet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaylet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroy-Ladurie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouret, P.</span></span> <span> </span><span class="NLM_article-title">A comparative and integrative approach identifies ATPase family, AAA domain containing 2 as a likely driver of cell proliferation in lung adenocarcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5606</span>– <span class="NLM_lpage">5616</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-0505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1078-0432.CCR-12-0505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=22914773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFahu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5606-5616&author=R.+Fouretauthor=J.+Laffaireauthor=P.+Hofmanauthor=M.+Beau-Fallerauthor=J.+Mazieresauthor=P.+Validireauthor=P.+Girardauthor=S.+Camilleri-Broetauthor=F.+Vayletauthor=F.+Leroy-Ladurieauthor=J.+C.+Soriaauthor=P.+Fouret&title=A+comparative+and+integrative+approach+identifies+ATPase+family%2C+AAA+domain+containing+2+as+a+likely+driver+of+cell+proliferation+in+lung+adenocarcinoma&doi=10.1158%2F1078-0432.CCR-12-0505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative and Integrative Approach Identifies ATPase Family, AAA Domain Containing 2 as a Likely Driver of Cell Proliferation in Lung Adenocarcinoma</span></div><div class="casAuthors">Fouret, Robert; Laffaire, Julien; Hofman, Paul; Beau-Faller, Michele; Mazieres, Julien; Validire, Pierre; Girard, Philippe; Camilleri-Broeet, Sophie; Vaylet, Fabien; Leroy-Ladurie, Francois; Soria, Jean-Charles; Fouret, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5606-5616</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The aim was to identify genetic changes that could drive cancer pathogenesis in never and ever smokers with lung adenocarcinoma.  We analyzed the copy no. and gene expression profiles of lung adenocarcinomas in 165 patients and related the alterations to smoking status.  Having found differences in the tumor profiles, we integrated copy no. and gene expression data from 80 paired samples.  Amplifications at 8q24.12 overlapping MYC and ATAD2 were more frequent in ever smokers.  Unsupervised anal. of gene expression revealed two groups: in the group with mainly never smokers, the tumors expressed genes common to normal lung; in the group with more ever smokers, the tumors expressed "proliferative" and "invasive" gene clusters.  Integration of copy no. and gene expression data identified one module enriched in mitotic genes and MYC targets.  Its main assocd. modulator was ATAD2, a cofactor of MYC.  A strong dose-response relationship between ATAD2 and proliferation-related gene expression was noted in both never and ever smokers, which was verified in two independent cohorts.  Both ATAD2 and MYC expression correlated with 8q24.12 amplification and were higher in ever smokers.  However, only ATAD2, and not MYC, overexpression explained the behavior of proliferation-related genes and predicted a worse prognosis independently of disease stage in a large validation cohort.  The likely driving force behind MYC contribution to uncontrolled cell proliferation in lung adenocarcinoma is ATAD2.  Deregulation of ATAD2 is mainly related to gene amplification and is more frequent in ever smokers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7I6bbL4KEqLVg90H21EOLACvtfcHk0lirwz7_D_9QyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFahu7rO&md5=939438d8b41c4e0b8f609d228dd6cc9f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-0505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-0505%26sid%3Dliteratum%253Aachs%26aulast%3DFouret%26aufirst%3DR.%26aulast%3DLaffaire%26aufirst%3DJ.%26aulast%3DHofman%26aufirst%3DP.%26aulast%3DBeau-Faller%26aufirst%3DM.%26aulast%3DMazieres%26aufirst%3DJ.%26aulast%3DValidire%26aufirst%3DP.%26aulast%3DGirard%26aufirst%3DP.%26aulast%3DCamilleri-Broet%26aufirst%3DS.%26aulast%3DVaylet%26aufirst%3DF.%26aulast%3DLeroy-Ladurie%26aufirst%3DF.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DFouret%26aufirst%3DP.%26atitle%3DA%2520comparative%2520and%2520integrative%2520approach%2520identifies%2520ATPase%2520family%252C%2520AAA%2520domain%2520containing%25202%2520as%2520a%2520likely%2520driver%2520of%2520cell%2520proliferation%2520in%2520lung%2520adenocarcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5606%26epage%3D5616%26doi%3D10.1158%2F1078-0432.CCR-12-0505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">miR-372 down-regulates the oncogene ATAD2 to influence hepatocellular carcinoma proliferation and metastasis</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1186/1471-2407-14-107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1186%2F1471-2407-14-107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=107-118&author=G.+Wuauthor=H.+Liuauthor=H.+Heauthor=Y.+Wangauthor=X.+Luauthor=Y.+Yuauthor=S.+Xiaauthor=X.+Mengauthor=Y.+Liu&title=miR-372+down-regulates+the+oncogene+ATAD2+to+influence+hepatocellular+carcinoma+proliferation+and+metastasis&doi=10.1186%2F1471-2407-14-107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-107%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DmiR-372%2520down-regulates%2520the%2520oncogene%2520ATAD2%2520to%2520influence%2520hepatocellular%2520carcinoma%2520proliferation%2520and%2520metastasis%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D107%26epage%3D118%26doi%3D10.1186%2F1471-2407-14-107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Epigenetic high regulation of ATAD2 regulates the Hh pathway in human hepatocellular carcinoma</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.3892/ijo.2014.2416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.3892%2Fijo.2014.2416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24805933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2014&pages=351-361&author=G.+Wuauthor=X.+Luauthor=Y.+Wangauthor=H.+Heauthor=X.+Mengauthor=S.+Xiaauthor=K.+Zhenauthor=Y.+Liu&title=Epigenetic+high+regulation+of+ATAD2+regulates+the+Hh+pathway+in+human+hepatocellular+carcinoma&doi=10.3892%2Fijo.2014.2416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic high regulation of ATAD2 regulates the Hh pathway in human hepatocellular carcinoma</span></div><div class="casAuthors">Wu, Gang; Lu, Xiaojun; Wang, Yawei; He, Hui; Meng, Xiangyu; Xia, Shuguan; Zhen, Kunming; Liu, Yongfeng</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">351-361</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">ATAD2 is assocd. with many cellular progresses such as cell growth, differentiation and apoptosis.  Some studies suggest ATAD2 is highly expressed in cancer cells.  In our previous studies, we found that ATAD2 is highly expressed in HCC tissues, compared with adjacent normal tissues, and patients with high expression of ATAD2 had a poorer prognosis.  Moreover, we found mir-372 can regulate the expression of ATAD2 in HCC cell lines.  We also detected a relationship between the mRNA expression of ATAD2 and Ptch1 by gene microarray.  Here, we completed the function studies of ATAD2 in vivo and in vitro, and tested whether ATAD2 could regulate the Hh pathway.  ATAD2 and Hh pathway protein expressions in 80 HCC specimens were examd. by immunohistochem. (IHC).  The mRNA expression of ATAD2 and Hh pathway members in paired-HCC tissues and cell lines were, resp., analyzed using quant. PCR.  ATAD2-RNAi was transduced into HCCLM3 and Huh7 cells, using a lentiviral vector.  The effect of ATAD2 in HCC cell lines on cell cycle and apoptosis were evaluated by flow cytometry.  Tumorigenicity expts. in nude mice were performed to test the function of ATAD2 in vivo.  Pharmacol. regulation of Hh signaling was performed to test the relation between the ATAD2 and Hh pathways and C-myc.  We found that ATAD2 and Ptch1 were both highly expressed in HCC tissues, compared with paired normal hepatic tissues.  In addn., we found that ATAD2 could affect the expression of the Hh pathway by PCR and western blot anal. in HCC cell lines, by observing the outcome before and after transfection.  We speculate that ATAD2 cooperates with the MYC gene to regulate the expression of SMO and Gli, activating the Hh pathway and inducing an active feedback of the Hh pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooC9ebryKIgrVg90H21EOLACvtfcHk0lj5Oh0uZpj5_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtL%252FK&md5=5b5a12bb3456343d386c5de7d51d49ff</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3892%2Fijo.2014.2416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2014.2416%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DXia%26aufirst%3DS.%26aulast%3DZhen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DEpigenetic%2520high%2520regulation%2520of%2520ATAD2%2520regulates%2520the%2520Hh%2520pathway%2520in%2520human%2520hepatocellular%2520carcinoma%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2014%26volume%3D45%26spage%3D351%26epage%3D361%26doi%3D10.3892%2Fijo.2014.2416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span> <span> </span><span class="NLM_article-title">ATAD2 overexpression is associated with progression and prognosis in colorectal cancer</span>. <i>Jpn. J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1093/jjco/hyv195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1093%2Fjjco%2Fhyv195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26819280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A280%3ADC%252BC28nkvVeltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=222-227&author=M.+Houauthor=R.+Huangauthor=Y.+Songauthor=D.+Fengauthor=Y.+Jiangauthor=M.+Liu&title=ATAD2+overexpression+is+associated+with+progression+and+prognosis+in+colorectal+cancer&doi=10.1093%2Fjjco%2Fhyv195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 overexpression is associated with progression and prognosis in colorectal cancer</span></div><div class="casAuthors">Hou Mingming; Huang Rui; Song Yanni; Feng Di; Jiang Yang; Liu Ming</div><div class="citationInfo"><span class="NLM_cas:title">Japanese journal of clinical oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">222-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  ATPase family AAA domain-containing 2 plays an important role in tumor progression including cell cycle, proliferation, apoptosis and chemoresistance.  However, the expression of ATPase family AAA domain-containing 2 in colorectal cancer and its significance are still unclear.  The aim of this study was to examine the expression of ATPase family AAA domain-containing 2 in colorectal cancer.  METHODS:  Immunohistochemistry was used to determine the expression of ATPase family AAA domain-containing 2 in 155 colorectal cancer and 30 matched adjacent noncancerous tissues.  The correlation of ATPase family AAA domain-containing 2 expression with clinicopathological variables was assessed using chi-square test.  Patient survival was analyzed using the Kaplan-Meier and log-rank tests.  Cox regression was performed for the multivariate analysis of prognostic factors.  RESULTS:  High expression of ATPase family AAA domain-containing 2 was detected in 58.1% of the colorectal cancers and was significantly associated with advanced tumor-node-metastasis stage (P = 0.044), poor differentiation (P = 0.028), deep infiltration (P < 0.001), lymphovascular invasion (P = 0.006), lymph node metastasis (P = 0.024) and recurrence (P = 0.022).  Patients with high ATPase family AAA domain-containing 2 expression had significantly poorer overall survival and disease-free survival (both P < 0.001) when compared with patients with low expression of ATPase family AAA domain-containing 2.  The multivariate analysis showed that ATPase family AAA domain-containing 2 was an independent factor for both overall survival (P = 0.003; hazard ratio (HR): 2.356; 95% confidence interval (CI): 1.335-4.158) and disease-free survival (P = 0.001; HR: 2.643; 95% CI: 1.489-4.693).  CONCLUSIONS:  These results showed that ATPase family AAA domain-containing 2 overexpression was associated with progression and prognosis of colorectal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyV3TCK4srS_9pSBfyDvp7fW6udTcc2eZzwNAydhg0Srntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nkvVeltA%253D%253D&md5=9d2996cb944237ca53217979f6d88a41</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fjjco%2Fhyv195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjjco%252Fhyv195%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26atitle%3DATAD2%2520overexpression%2520is%2520associated%2520with%2520progression%2520and%2520prognosis%2520in%2520colorectal%2520cancer%26jtitle%3DJpn.%2520J.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D46%26spage%3D222%26epage%3D227%26doi%3D10.1093%2Fjjco%2Fhyv195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wrzeszczynski, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrnes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamalakaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimitrova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucito, R.</span></span> <span> </span><span class="NLM_article-title">Identification of tumor suppressors and oncogenes from genomic and epigenetic features in ovarian cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e28503</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0028503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1371%2Fjournal.pone.0028503" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e28503&author=K.+O.+Wrzeszczynskiauthor=V.+Varadanauthor=J.+Byrnesauthor=E.+Lumauthor=S.+Kamalakaranauthor=D.+A.+Levineauthor=N.+Dimitrovaauthor=M.+Q.+Zhangauthor=R.+Lucito&title=Identification+of+tumor+suppressors+and+oncogenes+from+genomic+and+epigenetic+features+in+ovarian+cancer&doi=10.1371%2Fjournal.pone.0028503"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0028503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0028503%26sid%3Dliteratum%253Aachs%26aulast%3DWrzeszczynski%26aufirst%3DK.%2BO.%26aulast%3DVaradan%26aufirst%3DV.%26aulast%3DByrnes%26aufirst%3DJ.%26aulast%3DLum%26aufirst%3DE.%26aulast%3DKamalakaran%26aufirst%3DS.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26aulast%3DDimitrova%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DM.%2BQ.%26aulast%3DLucito%26aufirst%3DR.%26atitle%3DIdentification%2520of%2520tumor%2520suppressors%2520and%2520oncogenes%2520from%2520genomic%2520and%2520epigenetic%2520features%2520in%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De28503%26doi%3D10.1371%2Fjournal.pone.0028503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raeder, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkeland, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trovik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krakstad, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shehata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zack, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krohn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wik, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefansson, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalland, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akslen, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroukhim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, H. B.</span></span> <span> </span><span class="NLM_article-title">Integrated genomic analysis of the 8q24 amplification in endometrial cancers identifies ATAD2 as essential to MYC-dependent cancers</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e54873</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0054873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1371%2Fjournal.pone.0054873" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e54873&author=M.+B.+Raederauthor=E.+Birkelandauthor=J.+Trovikauthor=C.+Krakstadauthor=S.+Shehataauthor=S.+Schumacherauthor=T.+I.+Zackauthor=A.+Krohnauthor=H.+M.+Wernerauthor=S.+E.+Moodyauthor=E.+Wikauthor=I.+M.+Stefanssonauthor=F.+Holstauthor=A.+M.+Oyanauthor=P.+Tamayoauthor=J.+P.+Mesirovauthor=K.+H.+Kallandauthor=L.+A.+Akslenauthor=R.+Simonauthor=R.+Beroukhimauthor=H.+B.+Salvesen&title=Integrated+genomic+analysis+of+the+8q24+amplification+in+endometrial+cancers+identifies+ATAD2+as+essential+to+MYC-dependent+cancers&doi=10.1371%2Fjournal.pone.0054873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0054873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0054873%26sid%3Dliteratum%253Aachs%26aulast%3DRaeder%26aufirst%3DM.%2BB.%26aulast%3DBirkeland%26aufirst%3DE.%26aulast%3DTrovik%26aufirst%3DJ.%26aulast%3DKrakstad%26aufirst%3DC.%26aulast%3DShehata%26aufirst%3DS.%26aulast%3DSchumacher%26aufirst%3DS.%26aulast%3DZack%26aufirst%3DT.%2BI.%26aulast%3DKrohn%26aufirst%3DA.%26aulast%3DWerner%26aufirst%3DH.%2BM.%26aulast%3DMoody%26aufirst%3DS.%2BE.%26aulast%3DWik%26aufirst%3DE.%26aulast%3DStefansson%26aufirst%3DI.%2BM.%26aulast%3DHolst%26aufirst%3DF.%26aulast%3DOyan%26aufirst%3DA.%2BM.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DKalland%26aufirst%3DK.%2BH.%26aulast%3DAkslen%26aufirst%3DL.%2BA.%26aulast%3DSimon%26aufirst%3DR.%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DSalvesen%26aufirst%3DH.%2BB.%26atitle%3DIntegrated%2520genomic%2520analysis%2520of%2520the%25208q24%2520amplification%2520in%2520endometrial%2520cancers%2520identifies%2520ATAD2%2520as%2520essential%2520to%2520MYC-dependent%2520cancers%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De54873%26doi%3D10.1371%2Fjournal.pone.0054873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fellenberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernd, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delling, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahlten-Hinguranage, A.</span></span> <span> </span><span class="NLM_article-title">Prognostic significance of drug-regulated genes in highgrade osteosarcoma</span>. <i>Mod. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1085</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi"> DOI: 10.1038/modpathol.3800937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fmodpathol.3800937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=17660802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCku7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2007&pages=1085-1094&author=J.+Fellenbergauthor=L.+Berndauthor=G.+Dellingauthor=D.+Witteauthor=A.+Zahlten-Hinguranage&title=Prognostic+significance+of+drug-regulated+genes+in+highgrade+osteosarcoma&doi=10.1038%2Fmodpathol.3800937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic significance of drug-regulated genes in high-grade osteosarcoma</span></div><div class="casAuthors">Fellenberg, Joerg; Bernd, Ludger; Delling, Guenter; Witte, Daniela; Zahlten-Hinguranage, Anita</div><div class="citationInfo"><span class="NLM_cas:title">Modern Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1085-1094</span>CODEN:
                <span class="NLM_cas:coden">MODPEO</span>;
        ISSN:<span class="NLM_cas:issn">0893-3952</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">About 25-45% of patients with high-grade osteosarcoma poorly respond to chemotherapy with an increased risk of relapse and the development of metastasis.  Therefore, the aim of this study was the evaluation of the prognostic value of eight previously identified drug-regulated candidate genes on osteosarcoma therapy outcome.  Gene expression of 8 candidate genes was analyzed in 35 formalin-fixed, paraffin-embedded, laser-microdissected osteosarcoma biopsies.  The prognostic value of these genes was evaluated by the correlation of gene expression with therapy outcome, overall survival and event-free survival in univariate and multivariate anal.  Upon univariate anal., the expression of MALAT-1, IMPDH2, FTL and RHOA significantly correlated with response to chemotherapy.  Expression of all four genes was increased in the poor responder group.  Upon multivariate anal., IMPDH2 maintained its independent prognostic value (P=0.025).  Concerning the overall survival of the patients, we obsd. a significant assocn. with the expression of FTL, PHB, ATAD2, ACTN1 and RRM2 as well as lactate dehydrogenase serum levels.  In the subgroups of patients with high expression of these genes and those with elevated lactate dehydrogenase levels, the mean overall survival was decreased 1.7-, 1.9-, 2.2-, 2.4-, 1.5- and 4.5-fold, resp.  Except RRM2, all genes and lactate dehydrogenase serum levels remained significant in the multivariate anal.  In addn., the event-free survival was significantly decreased in the subgroups of patients with high FTL, ATAD2 and IMPDH2 expression (1.8-, 6.3- and 2.4-fold, resp.).  These data demonstrate that among the identified genes are valuable markers for the prediction of osteosarcoma therapy outcome.  Esp. IMPDH2 and FTL are promising candidates for the stratification of osteosarcoma patients into low- and high-risk groups.  Owing to their involvement in drug action these genes may further be potential targets for the modulation of drug sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppkmWTNEifO7Vg90H21EOLACvtfcHk0likyw0koDB4ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCku7%252FO&md5=05061311e1ee1b7b4007e2873a7dcbed</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fmodpathol.3800937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmodpathol.3800937%26sid%3Dliteratum%253Aachs%26aulast%3DFellenberg%26aufirst%3DJ.%26aulast%3DBernd%26aufirst%3DL.%26aulast%3DDelling%26aufirst%3DG.%26aulast%3DWitte%26aufirst%3DD.%26aulast%3DZahlten-Hinguranage%26aufirst%3DA.%26atitle%3DPrognostic%2520significance%2520of%2520drug-regulated%2520genes%2520in%2520highgrade%2520osteosarcoma%26jtitle%3DMod.%2520Pathol.%26date%3D2007%26volume%3D20%26spage%3D1085%26epage%3D1094%26doi%3D10.1038%2Fmodpathol.3800937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is overexpressed in gastric cancer and serves as an independent poor prognostic biomarker</span>. <i>Clin. Transl. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1007/s12094-015-1430-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1007%2Fs12094-015-1430-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26527032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVegt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=776-781&author=M.+Zhangauthor=C.+Zhangauthor=W.+Duauthor=X.+Yangauthor=Z.+Chen&title=ATAD2+is+overexpressed+in+gastric+cancer+and+serves+as+an+independent+poor+prognostic+biomarker&doi=10.1007%2Fs12094-015-1430-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 is overexpressed in gastric cancer and serves as an independent poor prognostic biomarker</span></div><div class="casAuthors">Zhang, M.-J.; Zhang, C.-Z.; Du, W.-J.; Yang, X.-Z.; Chen, Z.-P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Translational Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">776-781</span>CODEN:
                <span class="NLM_cas:coden">CTOLAM</span>;
        ISSN:<span class="NLM_cas:issn">1699-048X</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag Italia S.r.l.</span>)
        </div><div class="casAbstract">Objective: ATPase family, AAA domain contg. 2 (ATAD2) has been found overexpressed in various cancer types and correlated with malignant status and poor prognosis.  However, little is known about the clin. significance of ATAD2 in gastric cancer patients.  The aim of this study was to explore the clin. and prognostic significance of ATAD2 in gastric cancer.  Methods: The mRNA and protein levels expression of ATAD2 were detected in clin. tissue samples by qRT-PCR and immunohistochem., resp.  We examd. the ATAD2 protein expression by immunohistochem.  Furthermore, we analyzed the assocn. between ATAD2 expression and clinicopathol. features including prognosis in 166 gastric cancer samples.  Results: In our results, ATAD2 mRNA and protein were highly expressed in gastric cancer samples.  ATAD2 overexpression was correlated with advanced clin. stage, tumor depth, lymph node metastasis, and distant metastasis.  According to the survival anal., ATAD2 protein overexpression was a poor independent prognostic factor for gastric cancer patients.  Conclusions: In summary, ATAD2 could serve as a prognostic biomarker for gastric cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTZFFLPifZirVg90H21EOLACvtfcHk0likyw0koDB4ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVegt7vJ&md5=7ee3c65a9eb75519202fa614d8698f79</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs12094-015-1430-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12094-015-1430-8%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DATAD2%2520is%2520overexpressed%2520in%2520gastric%2520cancer%2520and%2520serves%2520as%2520an%2520independent%2520poor%2520prognostic%2520biomarker%26jtitle%3DClin.%2520Transl.%2520Oncol.%26date%3D2016%26volume%3D18%26spage%3D776%26epage%3D781%26doi%3D10.1007%2Fs12094-015-1430-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokolov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Kinesin family deregulation coordinated by bromodomain protein ANCCA and histone methyltransferase MLL for breast cancer cell growth, survival, and tamoxifen resistance</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1541-7786.MCR-13-0459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24391143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=539-549&author=J.+X.+Zouauthor=Z.+Duanauthor=J.+Wangauthor=A.+Sokolovauthor=J.+Xuauthor=C.+Z.+Chenauthor=J.+J.+Liauthor=H.+W.+Chen&title=Kinesin+family+deregulation+coordinated+by+bromodomain+protein+ANCCA+and+histone+methyltransferase+MLL+for+breast+cancer+cell+growth%2C+survival%2C+and+tamoxifen+resistance&doi=10.1158%2F1541-7786.MCR-13-0459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Kinesin Family Deregulation Coordinated by Bromodomain Protein ANCCA and Histone Methyltransferase MLL for Breast Cancer Cell Growth, Survival, and Tamoxifen Resistance</span></div><div class="casAuthors">Zou, June X.; Duan, Zhijian; Wang, Junjian; Sokolov, Alex; Xu, Jianzhen; Chen, Christopher Z.; Li, Jian Jian; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">539-549</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kinesins are a superfamily of motor proteins and often deregulated in different cancers.  However, the mechanism of their deregulation has been poorly understood.  Through examg. kinesin gene family expression in estrogen receptor (ER)-pos. breast cancer cells, we found that estrogen stimulation of cancer cell proliferation involves a concerted regulation of specific kinesins.  Estrogen strongly induces expression of 19 kinesin genes such as Kif4A/4B, Kif5A/5B, Kif10, Kif11, Kif15, Kif18A/18B, Kif20A/20B, Kif21, Kif23, Kif24, Kif25, and KifC1, whereas suppresses the expression of seven others, including Kif1A, Kif1C, Kif7, and KifC3.  Interestingly, the bromodomain protein ANCCA/ATAD2, previously shown to be an estrogen-induced chromatin regulator, plays a crucial role in the up- and downregulation of kinesins by estrogen.  Its overexpression drives estrogen-independent upregulation of specific kinesins.  Mechanistically, ANCCA (AAA nuclear coregulator cancer assocd.) mediates E2-dependent recruitment of E2F and MLL1 histone methyltransferase at kinesin gene promoters for gene activation-assocd. H3K4me3 methylation.  Importantly, elevated levels of Kif4A, Kif15, Kif20A, and Kif23 correlate with that of ANCCA in the tumors and with poor relapse-free survival of patients with ER-pos. breast cancer.  Their knockdown strongly impeded proliferation and induced apoptosis of both tamoxifen-sensitive and resistant cancer cells.  Together, the study reveals ANCCA as a key mediator of kinesin family deregulation in breast cancer and the crucial role of multiple kinesins in growth and survival of the tumor cells.  Implications: These findings support the development of novel inhibitors of cancer-assocd. kinesins and their regulator ANCCA for effective treatment of cancers including tamoxifen-resistant breast cancers.  Mol Cancer Res; 12(4); 539-49. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaol_2gF7Tc7Vg90H21EOLACvtfcHk0lh29iXZGQ_yHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D&md5=aa5cda3daca4e8e08dd64827d72a313c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0459%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSokolov%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DKinesin%2520family%2520deregulation%2520coordinated%2520by%2520bromodomain%2520protein%2520ANCCA%2520and%2520histone%2520methyltransferase%2520MLL%2520for%2520breast%2520cancer%2520cell%2520growth%252C%2520survival%252C%2520and%2520tamoxifen%2520resistance%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2014%26volume%3D12%26spage%3D539%26epage%3D549%26doi%3D10.1158%2F1541-7786.MCR-13-0459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Ahsen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toedling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">70323</span>– <span class="NLM_lpage">70335</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.18632%2Foncotarget.11855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=27612420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=70323-70335&author=S.+J.+Kooauthor=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=V.+Badockauthor=C.+J.+Ottauthor=S.+J.+Holtonauthor=O.+von+Ahsenauthor=J.+Toedlingauthor=S.+Vittoriauthor=J.+E.+Bradnerauthor=M.+Gorj%C3%A1n%C3%A1cz&title=ATAD2+is+an+epigenetic+reader+of+newly+synthesized+histone+marks+during+DNA+replication&doi=10.18632%2Foncotarget.11855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span></div><div class="casAuthors">Koo Seong Joo; Fernandez-Montalvan Amaury E; Badock Volker; Holton Simon J; von Ahsen Oliver; Toedling Joern; Gorjanacz Matyas; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Bradner James E; Bradner James E</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">70323-70335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ATAD2 (ATPase family AAA domain-containing protein 2) is a chromatin regulator harboring an AAA+ ATPase domain and a bromodomain, previously proposed to function as an oncogenic transcription co-factor.  Here we suggest that ATAD2 is also required for DNA replication.  ATAD2 is co-expressed with genes involved in DNA replication in various cancer types and predominantly expressed in S phase cells where it localized on nascent chromatin (replication sites).  Our extensive biochemical and cellular analyses revealed that ATAD2 is recruited to replication sites through a direct interaction with di-acetylated histone H4 at K5 and K12, indicative of newly synthesized histones during replication-coupled chromatin reassembly.  Similar to ATAD2-depletion, ectopic expression of ATAD2 mutants that are deficient in binding to these di-acetylation marks resulted in reduced DNA replication and impaired loading of PCNA onto chromatin, suggesting relevance of ATAD2 in DNA replication.  Taken together, our data show a novel function of ATAD2 in cancer and for the first time identify a reader of newly synthesized histone di-acetylation-marks during replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAi97EmAYvEFQKX3ooHxWjfW6udTcc2eY7kGB8tZFmvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D&md5=38a41ce654ca0579c4c517d9537d6c0c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11855%26sid%3Dliteratum%253Aachs%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3Dvon%2BAhsen%26aufirst%3DO.%26aulast%3DToedling%26aufirst%3DJ.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DATAD2%2520is%2520an%2520epigenetic%2520reader%2520of%2520newly%2520synthesized%2520histone%2520marks%2520during%2520DNA%2520replication%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D70323%26epage%3D70335%26doi%3D10.18632%2Foncotarget.11855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">5260</span>– <span class="NLM_lpage">5272</span>, <span class="refDoi"> DOI: 10.1128/MCB.00484-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1128%2FMCB.00484-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20855524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=5260-5272&author=A.+S.+Revenkoauthor=E.+V.+Kalashnikovaauthor=A.+T.+Gemoauthor=J.+X.+Zouauthor=H.+W.+Chen&title=Chromatin+loading+of+E2F-MLL+complex+by+cancer-associated+coregulator+ANCCA+via+reading+a+specific+histone+mark&doi=10.1128%2FMCB.00484-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span></div><div class="casAuthors">Revenko, Alexey S.; Kalashnikova, Ekaterina V.; Gemo, Abigael T.; Zou, June X.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5260-5272</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Histone modifications are regarded as the carrier of epigenetic memory through cell divisions.  How the marks facilitate cell cycle-dependent gene expression is poorly understood.  The evolutionarily conserved AAA ATPase ANCCA (AAA nuclear coregulator cancer-assocd. protein)/ATAD2 was identified as a direct target of oncogene AIB1/ACTR/SRC-3 and a transcriptional coregulator for estrogen and androgen receptors and is strongly implicated in tumorigenesis.  We report here that ANCCA directly interacts with E2F1 to E2F3 and that its N terminus interacts with both the N and C termini of E2F1.  ANCCA preferentially assocs. via its bromodomain with H3 acetylated at lysine 14 (H3K14ac) and is required for key cell cycle gene expression and cancer cell proliferation.  ANCCA assocs. with chromosomes at late mitosis, and its occupancy at E2F targets peaks at the G1-to-S transition.  Strikingly, ANCCA is required for recruitment of specific E2Fs to their targets and chromatin assembly of the host cell factor 1 (HCF-1)-MLL histone methyltransferase complex.  ANCCA depletion results in a marked decrease of the gene activation-linked H3K4me3 mark.  Bromodomain mutations disable ANCCA function as an E2F coactivator and its ability to promote cancer cell proliferation, while ANCCA overexpression in tumors correlates with tumor growth.  Together, these results suggest that ANCCA acts as a pioneer factor in E2F-dependent gene activation and that a novel mechanism involving ANCCA bromodomain may contribute to cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9jiEXxtPP3rVg90H21EOLACvtfcHk0lh29iXZGQ_yHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D&md5=958e54242a3a8c16ac4bf5b51acd80b8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FMCB.00484-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00484-10%26sid%3Dliteratum%253Aachs%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DChromatin%2520loading%2520of%2520E2F-MLL%2520complex%2520by%2520cancer-associated%2520coregulator%2520ANCCA%2520via%2520reading%2520a%2520specific%2520histone%2520mark%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D5260%26epage%3D5272%26doi%3D10.1128%2FMCB.00484-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Androgen-induced coactivator ANCCA mediates specific androgen receptor signalling in prostate cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">3339</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-3440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F0008-5472.CAN-08-3440" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=3339-3346&author=J.+X.+Zouauthor=L.+Guoauthor=A.+S.+Revenkoauthor=C.+G.+Tepperauthor=A.+T.+Gemoauthor=H.+J.+Kungauthor=H.+W.+Chen&title=Androgen-induced+coactivator+ANCCA+mediates+specific+androgen+receptor+signalling+in+prostate+cancer&doi=10.1158%2F0008-5472.CAN-08-3440"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3440%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DAndrogen-induced%2520coactivator%2520ANCCA%2520mediates%2520specific%2520androgen%2520receptor%2520signalling%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D3339%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-08-3440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsia, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Deregulated E2F and the AAA+ coregulator ANCCA drive proto-oncogene ACTR/AIB1 overexpression in breast cancer</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-09-0095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1541-7786.MCR-09-0095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20124470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVOmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=183-193&author=E.+Y.+Hsiaauthor=E.+V.+Kalashnikovaauthor=A.+S.+Revenkoauthor=J.+X.+Zouauthor=A.+D.+Borowskyauthor=H.+W.+Chen&title=Deregulated+E2F+and+the+AAA%2B+coregulator+ANCCA+drive+proto-oncogene+ACTR%2FAIB1+overexpression+in+breast+cancer&doi=10.1158%2F1541-7786.MCR-09-0095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulated E2F and the AAA+ Coregulator ANCCA Drive Proto-Oncogene ACTR/AIB1 Overexpression in Breast Cancer</span></div><div class="casAuthors">Hsia, Elaine Y. C.; Kalashnikova, Ekaterina V.; Revenko, Alexey S.; Zou, June X.; Borowsky, Alexander D.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-193</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The proto-oncogene ACTR/AIB1, a coactivator for transcription factors such as the nuclear receptors and E2Fs, is frequently overexpressed in various cancers including breast cancers.  However, the underlying mechanism is poorly understood.  Here, we identified several functional, noncanonical E2F binding sites in the ACTR first exon and intron that are crit. for ACTR gene activation.  We also found that the newly identified AAA+ coregulator AAA+ nuclear coregulator cancer assocd. (ANCCA) is recruited to the ACTR promoter and directly controls ACTR expression in breast cancer cells.  Importantly, immunohistochem. anal. indicated that ACTR overexpression is highly correlated with the expression of E2F1 and ANCCA in a cohort of human primary and lymph node-metastasized breast cancer specimens.  Along with previous findings from us and others that ACTR is involved in its own gene regulation, these results suggest that one major mechanism of ACTR overexpression in cancer is the concerted, aberrant function of the nuclear coregulators such as ANCCA and ACTR, and they point to therapeutic strategies that target the Rb-E2F axis and/or the coregulator ANCCA for ACTR-overexpressing cancers.  Mol Cancer Res. 8(2); 183-93.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBYcHe3haHbLVg90H21EOLACvtfcHk0lg4nWnKw1Csjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVOmtro%253D&md5=b2a7dac37ab9fa36fefe489bc64c6534</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-09-0095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-09-0095%26sid%3Dliteratum%253Aachs%26aulast%3DHsia%26aufirst%3DE.%2BY.%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DDeregulated%2520E2F%2520and%2520the%2520AAA%252B%2520coregulator%2520ANCCA%2520drive%2520proto-oncogene%2520ACTR%252FAIB1%2520overexpression%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2010%26volume%3D8%26spage%3D183%26epage%3D193%26doi%3D10.1158%2F1541-7786.MCR-09-0095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Developmental and androgenic regulation of chromatin regulators EZH2 and ANCCA/ATAD2 in the prostate via MLL histone methylase complex</span>. <i>Prostate</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1002/pros.22587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1002%2Fpros.22587" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=455-466&author=Z.+Duanauthor=J.+X.+Zouauthor=P.+Yangauthor=Y.+Wangauthor=A.+D.+Borowskyauthor=A.+C.+Gaoauthor=H.+W.+Chen&title=Developmental+and+androgenic+regulation+of+chromatin+regulators+EZH2+and+ANCCA%2FATAD2+in+the+prostate+via+MLL+histone+methylase+complex&doi=10.1002%2Fpros.22587"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fpros.22587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.22587%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DGao%26aufirst%3DA.%2BC.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DDevelopmental%2520and%2520androgenic%2520regulation%2520of%2520chromatin%2520regulators%2520EZH2%2520and%2520ANCCA%252FATAD2%2520in%2520the%2520prostate%2520via%2520MLL%2520histone%2520methylase%2520complex%26jtitle%3DProstate%26date%3D2013%26volume%3D73%26spage%3D455%26epage%3D466%26doi%3D10.1002%2Fpros.22587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altintas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goutte-Gattat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimitrov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarut, J.</span></span> <span> </span><span class="NLM_article-title">Direct cooperation between androgen receptor and E2F1 reveals a common regulation mechanism for androgen-responsive genes in prostate cells</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1531</span>– <span class="NLM_lpage">1541</span>, <span class="refDoi"> DOI: 10.1210/me.2012-1016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1210%2Fme.2012-1016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=22771493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlentLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1531-1541&author=D.+M.+Altintasauthor=M.+S.+Shuklaauthor=D.+Goutte-Gattatauthor=D.+Angelovauthor=J.+P.+Rouaultauthor=S.+Dimitrovauthor=J.+Samarut&title=Direct+cooperation+between+androgen+receptor+and+E2F1+reveals+a+common+regulation+mechanism+for+androgen-responsive+genes+in+prostate+cells&doi=10.1210%2Fme.2012-1016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Direct cooperation between androgen receptor and E2F1 reveals a common regulation mechanism for androgen-responsive genes in prostate cells</span></div><div class="casAuthors">Altintas, D. M.; Shukla, M. S.; Goutte-Gattat, D.; Angelov, D.; Rouault, J. P.; Dimitrov, S.; Samarut, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1531-1541</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">We have studied the regulation of ATAD2 gene expression by androgens in prostate cells.  ATAD2 is a coactivator of the androgen receptor (AR) and the MYC protein.  We showed that ATAD2 expression is directly regulated by AR via an AR binding sequence (ARBS) located in the distal enhancer of its regulatory region.  The gene is also regulated by the E2F1 transcription factor.  Using knockdown and chromatin immunopptn. technique approaches, we could demonstrate that AR and E2F1 functionally collaborate and phys. interact between each other.  From the anal. of chromatin conformation, we conclude that this cooperation results from a chromatin looping over the ATAD2 promoter region between the ARBS and E2F1 binding site in an androgen-dependent manner.  Furthermore, we could show that several genes overexpressed in prostate cancer and potentially involved in several aspects of tumor development have an ARBS and an E2F1 binding site in their regulatory regions and exhibit the same mechanism of regulation by both transcription factors as ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDhmKxaoJhMbVg90H21EOLACvtfcHk0lg4nWnKw1Csjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlentLnO&md5=d5d0654f559baf25eb63232334383b2e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1210%2Fme.2012-1016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2012-1016%26sid%3Dliteratum%253Aachs%26aulast%3DAltintas%26aufirst%3DD.%2BM.%26aulast%3DShukla%26aufirst%3DM.%2BS.%26aulast%3DGoutte-Gattat%26aufirst%3DD.%26aulast%3DAngelov%26aufirst%3DD.%26aulast%3DRouault%26aufirst%3DJ.%2BP.%26aulast%3DDimitrov%26aufirst%3DS.%26aulast%3DSamarut%26aufirst%3DJ.%26atitle%3DDirect%2520cooperation%2520between%2520androgen%2520receptor%2520and%2520E2F1%2520reveals%2520a%2520common%2520regulation%2520mechanism%2520for%2520androgen-responsive%2520genes%2520in%2520prostate%2520cells%26jtitle%3DMol.%2520Endocrinol.%26date%3D2012%26volume%3D26%26spage%3D1531%26epage%3D1541%26doi%3D10.1210%2Fme.2012-1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosperini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quarto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazini, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walfridsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBlane, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucifero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacchiana, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is a novel cofactor for MYC, overexpressed and amplified in aggressive tumors</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">8491</span>– <span class="NLM_lpage">8498</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-2131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F0008-5472.CAN-09-2131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=19843847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlantrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=8491-8498&author=M.+Ciroauthor=E.+Prosperiniauthor=M.+Quartoauthor=U.+Graziniauthor=J.+Walfridssonauthor=F.+McBlaneauthor=P.+Nuciferoauthor=G.+Pacchianaauthor=M.+Capraauthor=J.+Christensenauthor=K.+Helin&title=ATAD2+is+a+novel+cofactor+for+MYC%2C+overexpressed+and+amplified+in+aggressive+tumors&doi=10.1158%2F0008-5472.CAN-09-2131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 Is a Novel Cofactor for MYC, Overexpressed and Amplified in Aggressive Tumors</span></div><div class="casAuthors">Ciro, Marco; Prosperini, Elena; Quarto, Micaela; Grazini, Ursula; Walfridsson, Julian; McBlane, Fraser; Nucifero, Paolo; Pacchiana, Giovanni; Capra, Maria; Christensen, Jesper; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8491-8498</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The E2F and MYC transcription factors are crit. regulators of cell proliferation and contribute to the development of human cancers.  Here, we report on the identification of a novel E2F target gene, ATAD2, the predicted protein product of which contains both a bromodomain and an ATPase domain.  The pRB-E2F pathway regulates ATAD2 expression, which is limiting for the entry into the S phase of the cell cycle.  We show that ATAD2 binds the MYC oncogene and stimulates its transcriptional activity.  ATAD2 maps to chromosome 8q24, 4.3 Mb distal to MYC, in a region that is frequently found amplified in cancer.  Consistent with this, we show that ATAD2 expression is high in several human tumors and that the expression levels correlate with clin. outcome of breast cancer patients.  We suggest that ATAD2 links the E2F and MYC pathways and contributes to the development of aggressive cancer through the enhancement of MYC-dependent transcription. [Cancer Res 2009;69(21):8491-8].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-O94KXJv2i7Vg90H21EOLACvtfcHk0lgkocmUOkcUkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlantrnI&md5=85b462fb8a0c9505d98414ab8d90520b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-2131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-2131%26sid%3Dliteratum%253Aachs%26aulast%3DCiro%26aufirst%3DM.%26aulast%3DProsperini%26aufirst%3DE.%26aulast%3DQuarto%26aufirst%3DM.%26aulast%3DGrazini%26aufirst%3DU.%26aulast%3DWalfridsson%26aufirst%3DJ.%26aulast%3DMcBlane%26aufirst%3DF.%26aulast%3DNucifero%26aufirst%3DP.%26aulast%3DPacchiana%26aufirst%3DG.%26aulast%3DCapra%26aufirst%3DM.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DATAD2%2520is%2520a%2520novel%2520cofactor%2520for%2520MYC%252C%2520overexpressed%2520and%2520amplified%2520in%2520aggressive%2520tumors%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D8491%26epage%3D8498%26doi%3D10.1158%2F0008-5472.CAN-09-2131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mjelle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegre, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aas, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drablos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slupphaug, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saetrom, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krokan, H. E.</span></span> <span> </span><span class="NLM_article-title">Cell cycle regulation of human DNA repair and chromatin remodeling genes</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1016%2Fj.dnarep.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=25881042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFWlsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2015&pages=53-67&author=R.+Mjelleauthor=S.+A.+Hegreauthor=P.+A.+Aasauthor=F.+Drablosauthor=G.+Slupphaugauthor=P.+Saetromauthor=H.+E.+Krokan&title=Cell+cycle+regulation+of+human+DNA+repair+and+chromatin+remodeling+genes&doi=10.1016%2Fj.dnarep.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle regulation of human DNA repair and chromatin remodeling genes</span></div><div class="casAuthors">Mjelle, Robin; Hegre, Siv Anita; Aas, Per Arne; Slupphaug, Geir; Drabloes, Finn; Saetrom, Paal; Krokan, Hans E.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-67</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Maintenance of a genome requires DNA repair integrated with chromatin remodeling.  We have analyzed six transcriptome data sets and one data set on translational regulation of known DNA repair and remodeling genes in synchronized human cells.  These data are available through our new database: www.dnarepairgenes.com.  Genes that have similar transcription profiles in at least two of our data sets generally agree well with known protein profiles.  In brief, long patch base excision repair (BER) is enriched for S phase genes, whereas short patch BER uses genes essentially equally expressed in all cell cycle phases.  Furthermore, most genes related to DNA mismatch repair, Fanconi anemia and homologous recombination have their highest expression in the S phase.  In contrast, genes specific for direct repair, nucleotide excision repair, as well as non-homologous end joining do not show cell cycle-related expression.  Cell cycle regulated chromatin remodeling genes were most frequently confined to G1/S and S.  These include e.g. genes for chromatin assembly factor 1 (CAF-1) major subunits CHAF1A and CHAF1B; the putative helicases HELLS and ATAD2 that both co-activate E2F transcription factors central in G1/S-transition and recruit DNA repair and chromatin-modifying proteins and DNA double strand break repair proteins; and RAD54L and RAD54B involved in double strand break repair.  TOP2A was consistently most highly expressed in G2, but also expressed in late S phase, supporting a role in regulating entry into mitosis.  Translational regulation complements transcriptional regulation and appears to be a relatively common cell cycle regulatory mechanism for DNA repair genes.  Our results identify cell cycle phases in which different pathways have highest activity, and demonstrate that periodically expressed genes in a pathway are frequently co-expressed.  Furthermore, the data suggest that S phase expression and over-expression of some multifunctional chromatin remodeling proteins may set up feedback loops driving cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRILpuYb39LVg90H21EOLACvtfcHk0lgkocmUOkcUkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFWlsr4%253D&md5=15f76faf18a4b198a119b61c302f6f4b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DMjelle%26aufirst%3DR.%26aulast%3DHegre%26aufirst%3DS.%2BA.%26aulast%3DAas%26aufirst%3DP.%2BA.%26aulast%3DDrablos%26aufirst%3DF.%26aulast%3DSlupphaug%26aufirst%3DG.%26aulast%3DSaetrom%26aufirst%3DP.%26aulast%3DKrokan%26aufirst%3DH.%2BE.%26atitle%3DCell%2520cycle%2520regulation%2520of%2520human%2520DNA%2520repair%2520and%2520chromatin%2520remodeling%2520genes%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D30%26spage%3D53%26epage%3D67%26doi%3D10.1016%2Fj.dnarep.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Ahsen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toedling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">70323</span>– <span class="NLM_lpage">70335</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.18632%2Foncotarget.11855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=27612420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=70323-70335&author=S.+J.+Kooauthor=A.+E.+Fernandez-Montalvanauthor=V.+Badockauthor=C.+J.+Ottauthor=S.+J.+Holtonauthor=O.+von+Ahsenauthor=J.+Toedlingauthor=S.+Vittoriauthor=J.+E.+Bradnerauthor=M.+Gorjanacz&title=ATAD2+is+an+epigenetic+reader+of+newly+synthesized+histone+marks+during+DNA+replication&doi=10.18632%2Foncotarget.11855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span></div><div class="casAuthors">Koo Seong Joo; Fernandez-Montalvan Amaury E; Badock Volker; Holton Simon J; von Ahsen Oliver; Toedling Joern; Gorjanacz Matyas; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Bradner James E; Bradner James E</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">70323-70335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ATAD2 (ATPase family AAA domain-containing protein 2) is a chromatin regulator harboring an AAA+ ATPase domain and a bromodomain, previously proposed to function as an oncogenic transcription co-factor.  Here we suggest that ATAD2 is also required for DNA replication.  ATAD2 is co-expressed with genes involved in DNA replication in various cancer types and predominantly expressed in S phase cells where it localized on nascent chromatin (replication sites).  Our extensive biochemical and cellular analyses revealed that ATAD2 is recruited to replication sites through a direct interaction with di-acetylated histone H4 at K5 and K12, indicative of newly synthesized histones during replication-coupled chromatin reassembly.  Similar to ATAD2-depletion, ectopic expression of ATAD2 mutants that are deficient in binding to these di-acetylation marks resulted in reduced DNA replication and impaired loading of PCNA onto chromatin, suggesting relevance of ATAD2 in DNA replication.  Taken together, our data show a novel function of ATAD2 in cancer and for the first time identify a reader of newly synthesized histone di-acetylation-marks during replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAi97EmAYvEFQKX3ooHxWjfW6udTcc2eaJnlkFRvCtSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D&md5=38a41ce654ca0579c4c517d9537d6c0c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11855%26sid%3Dliteratum%253Aachs%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3Dvon%2BAhsen%26aufirst%3DO.%26aulast%3DToedling%26aufirst%3DJ.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DATAD2%2520is%2520an%2520epigenetic%2520reader%2520of%2520newly%2520synthesized%2520histone%2520marks%2520during%2520DNA%2520replication%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D70323%26epage%3D70335%26doi%3D10.18632%2Foncotarget.11855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokolov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Kinesin family deregulation coordinated by bromodomain protein ANCCA and histone methyltransferase MLL for breast cancer cell growth, survival, and tamoxifen resistance</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1541-7786.MCR-13-0459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24391143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=539-549&author=J.+X.+Zouauthor=Z.+Duanauthor=J.+Wangauthor=A.+Sokolovauthor=J.+Xuauthor=C.+Z.+Chenauthor=J.+J.+Liauthor=H.+W.+Chen&title=Kinesin+family+deregulation+coordinated+by+bromodomain+protein+ANCCA+and+histone+methyltransferase+MLL+for+breast+cancer+cell+growth%2C+survival%2C+and+tamoxifen+resistance&doi=10.1158%2F1541-7786.MCR-13-0459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Kinesin Family Deregulation Coordinated by Bromodomain Protein ANCCA and Histone Methyltransferase MLL for Breast Cancer Cell Growth, Survival, and Tamoxifen Resistance</span></div><div class="casAuthors">Zou, June X.; Duan, Zhijian; Wang, Junjian; Sokolov, Alex; Xu, Jianzhen; Chen, Christopher Z.; Li, Jian Jian; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">539-549</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kinesins are a superfamily of motor proteins and often deregulated in different cancers.  However, the mechanism of their deregulation has been poorly understood.  Through examg. kinesin gene family expression in estrogen receptor (ER)-pos. breast cancer cells, we found that estrogen stimulation of cancer cell proliferation involves a concerted regulation of specific kinesins.  Estrogen strongly induces expression of 19 kinesin genes such as Kif4A/4B, Kif5A/5B, Kif10, Kif11, Kif15, Kif18A/18B, Kif20A/20B, Kif21, Kif23, Kif24, Kif25, and KifC1, whereas suppresses the expression of seven others, including Kif1A, Kif1C, Kif7, and KifC3.  Interestingly, the bromodomain protein ANCCA/ATAD2, previously shown to be an estrogen-induced chromatin regulator, plays a crucial role in the up- and downregulation of kinesins by estrogen.  Its overexpression drives estrogen-independent upregulation of specific kinesins.  Mechanistically, ANCCA (AAA nuclear coregulator cancer assocd.) mediates E2-dependent recruitment of E2F and MLL1 histone methyltransferase at kinesin gene promoters for gene activation-assocd. H3K4me3 methylation.  Importantly, elevated levels of Kif4A, Kif15, Kif20A, and Kif23 correlate with that of ANCCA in the tumors and with poor relapse-free survival of patients with ER-pos. breast cancer.  Their knockdown strongly impeded proliferation and induced apoptosis of both tamoxifen-sensitive and resistant cancer cells.  Together, the study reveals ANCCA as a key mediator of kinesin family deregulation in breast cancer and the crucial role of multiple kinesins in growth and survival of the tumor cells.  Implications: These findings support the development of novel inhibitors of cancer-assocd. kinesins and their regulator ANCCA for effective treatment of cancers including tamoxifen-resistant breast cancers.  Mol Cancer Res; 12(4); 539-49. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaol_2gF7Tc7Vg90H21EOLACvtfcHk0lgY3JFYLBGr0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D&md5=aa5cda3daca4e8e08dd64827d72a313c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0459%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSokolov%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DKinesin%2520family%2520deregulation%2520coordinated%2520by%2520bromodomain%2520protein%2520ANCCA%2520and%2520histone%2520methyltransferase%2520MLL%2520for%2520breast%2520cancer%2520cell%2520growth%252C%2520survival%252C%2520and%2520tamoxifen%2520resistance%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2014%26volume%3D12%26spage%3D539%26epage%3D549%26doi%3D10.1158%2F1541-7786.MCR-13-0459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Malignant genome reprogramming by ATAD2</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1829</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2013.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1016%2Fj.bbagrm.2013.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=23831842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=1010-1014&author=F.+Boussouarauthor=M.+Jamshidikiaauthor=Y.+Morozumiauthor=S.+Rousseauxauthor=S.+Khochbin&title=Malignant+genome+reprogramming+by+ATAD2&doi=10.1016%2Fj.bbagrm.2013.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Malignant genome reprogramming by ATAD2</span></div><div class="casAuthors">Boussouar, Faycal; Jamshidikia, Mahya; Morozumi, Yuichi; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1829</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Unscheduled expression of crit. cellular regulators could be central to malignant genome reprogramming and tumor establishment.  One such factor appears to be ATAD2, a remarkably conserved protein normally predominantly expressed in germ cells but almost systematically over-expressed in a variety of unrelated cancers.  The presence of a bromodomain adjacent to an AAA type ATPase domain, points to ATAD2 as a factor preliminarily acting on chromatin structure and function.  Accordingly, ATAD2 has been shown to cooperate with a series of transcription factors and chromatin modifiers to regulate specific set of genes.  Here we discuss our knowledge on ATAD2 to evaluate its role as a cancer driver and its value as a new anti-cancer target.  Upon its activation, ATAD2 through its interaction with defined transcription factors, initiates a loop of transcriptional stimulation of target genes, including ATAD2 itself, leading to enhanced cell proliferation and resistance to apoptosis in an ATAD2-dependent manner.  Approaches aiming at neutralizing ATAD2 activity in cancer, including the use of small mol. inhibitors of its two "druggable" domains, AAA ATPase and bromodomain, could become part of a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp824JMo6Dp4rVg90H21EOLACvtfcHk0lgY3JFYLBGr0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF&md5=cba5991a2dc5f96f522e1104ae4e4889</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2013.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2013.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DMalignant%2520genome%2520reprogramming%2520by%2520ATAD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2013%26volume%3D1829%26spage%3D1010%26epage%3D1014%26doi%3D10.1016%2Fj.bbagrm.2013.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, H. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ATAD2 in cancer: a pharmacologically challenging but tractable target</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1080%2F14728222.2018.1406921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=29148850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=85-96&author=M.+Hussainauthor=Y.+Zhouauthor=Y.+Songauthor=H.+M.+A.+Hameedauthor=H.+Jiangauthor=Y.+Tuauthor=J.+Zhang&title=ATAD2+in+cancer%3A+a+pharmacologically+challenging+but+tractable+target&doi=10.1080%2F14728222.2018.1406921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 in cancer: a pharmacologically challenging but tractable target</span></div><div class="casAuthors">Hussain, Muzammal; Zhou, Yang; Song, Yu; Hameed, H. M. Adnan; Jiang, Hao; Tu, Yaoquan; Zhang, Jiancun</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-96</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  ATAD2 protein is an emerging oncogene that has strongly been linked to the etiol. of multiple advanced human cancers.  Therapeutically, despite the fact that genetic suppression/knockdown studies have validated it as a compelling drug target for future therapeutic development, recent druggability assessment data suggest that direct targeting of ATAD2's bromodomain (BRD) may be a very challenging task.  ATAD2's BRD has been predicted as a 'difficult to drug' or 'least druggable' target due to the concern that its binding pocket, and the areas around it, seem to be unfeasible for ligand binding.: In this review, after shedding light on the multifaceted roles of ATAD2 in normal physiol. as well as in cancer-etiol., we discuss tech. challenges rendered by ATAD2's BRD active site and the recent drug discovery efforts to find small mol. inhibitors against it.: The identification of a novel low-nanomolar semi-permeable chem. probe against ATAD2's BRD by recent drug discovery campaign has demonstrated it to be a pharmacol. tractable target.  Nevertheless, the development of high quality bioavailable inhibitors against ATAD2 is still a pending task.  Moreover, ATAD2 may also potentially be utilized as a promising target for future development of RNAi-based therapy to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bdigfdvMjbVg90H21EOLACvtfcHk0ljHEcL6YyHSKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN&md5=5d0ff7c936bb9eff125efeeaa4d275d1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DHameed%26aufirst%3DH.%2BM.%2BA.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DATAD2%2520in%2520cancer%253A%2520a%2520pharmacologically%2520challenging%2520but%2520tractable%2520target%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D85%26epage%3D96%26doi%3D10.1080%2F14728222.2018.1406921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljHEcL6YyHSKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0ljgPtPPM8iMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0ljgPtPPM8iMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.05.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1016%2Fj.cbpa.2016.05.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=27295577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=58-66&author=N.+H.+Theodoulouauthor=N.+C.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Clinical+progress+and+pharmacology+of+small+molecule+bromodomain+inhibitors&doi=10.1016%2Fj.cbpa.2016.05.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains have emerged as an exciting target class for drug discovery over the past decade.  Research has primarily focused on the bromodomain and extra terminal (BET) family of bromodomains, which has led to the development of multiple small mol. inhibitors and an increasing no. of clin. assets.  The excitement centered on the clin. potential of BET inhibition has stimulated intense interest in the broader family and the growing no. of non-BET bromodomain chem. probes has facilitated phenotypic investigations, implicating these targets in a variety of disease pathways including cancer, inflammation, embryonic development and neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEE0jCnUuyk7Vg90H21EOLACvtfcHk0ljgPtPPM8iMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D&md5=4bcd42755b2dd3cebd7fae417e69e83f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DClinical%2520progress%2520and%2520pharmacology%2520of%2520small%2520molecule%2520bromodomain%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.cbpa.2016.05.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Progress in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1002%2Fcmdc.201500540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26749027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=477-487&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Progress+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201500540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-487</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain and extra terminal (BET) family of bromodomains have been the focus of extensive research, leading to the development of many potent, selective chem. probes and recent clin. assets.  The profound biol. assocd. with BET bromodomain inhibition has provided a convincing rationale for targeting bromodomains for the treatment of disease.  However, the BET family represents just eight of the at least 56 human bromodomains identified to date.  Until recently, there has been significantly less interest in non-BET bromodomains, leaving a vast area of research and the majority of this new target class yet to be thoroughly investigated.  It has been widely reported that several non-BET bromodomain contg. proteins are assocd. with various diseases including cancer and HIV.  Therefore, the development of chem. probes for non-BET bromodomains will facilitate elucidation of their precise biol. roles and potentially lead to the development of new medicines.  This review summarises the progress made towards the development of non-BET bromodomain chem. probes to date.  In addn., we highlight the potential for future work in this new and exciting area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYLj_VyHOS5bVg90H21EOLACvtfcHk0ljgPtPPM8iMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D&md5=1e1339bf09a10bde8621c18344172fa6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DProgress%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D477%26epage%3D487%26doi%3D10.1002%2Fcmdc.201500540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical probes and inhibitors of bromodomains outside the BET family</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/C6MD00373G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1039%2FC6MD00373G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+probes+and+inhibitors+of+bromodomains+outside+the+BET+family&doi=10.1039%2FC6MD00373G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC6MD00373G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00373G%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520probes%2520and%2520inhibitors%2520of%2520bromodomains%2520outside%2520the%2520BET%2520family%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2FC6MD00373G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lisine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+Coteauthor=S.+Magnuson&title=Disrupting+acetyl-lisine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lisine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of BET bromodomain inhibitors and their role in target validation</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1039/C3MD00291H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1039%2FC3MD00291H" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=288-296&author=S.+Mullerauthor=S.+Knapp&title=Discovery+of+BET+bromodomain+inhibitors+and+their+role+in+target+validation&doi=10.1039%2FC3MD00291H"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FC3MD00291H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00291H%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520BET%2520bromodomain%2520inhibitors%2520and%2520their%2520role%2520in%2520target%2520validation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D288%26epage%3D296%26doi%3D10.1039%2FC3MD00291H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouzon, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of low-micromolar ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5649</span>– <span class="NLM_lpage">5673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649-5673&author=E.+H.+Demontauthor=C.-W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+J.+Emmonsauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-based+discovery+of+low-micromolar+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0lgstjLJr7J1uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520low-micromolar%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5649%26epage%3D5673%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of naphthyridones into potent ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.-W+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-based+optimization+of+naphthyridones+into+potent+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lixYiNEvwUkvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-based%2520optimization%2520of%2520naphthyridones%2520into%2520potent%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heptinstall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molyneux, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedge, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1850</span>, <span class="refDoi"> DOI: 10.1039/C8OB00099A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1039%2FC8OB00099A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=29469144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=1843-1850&author=D.+C.+Millerauthor=M.+P.+Martinauthor=S.+Adhikariauthor=A.+Brennanauthor=J.+A.+Endicottauthor=B.+T.+Goldingauthor=I.+R.+Hardcastleauthor=A.+Heptinstallauthor=S.+Hobsonauthor=C.+Jenningsauthor=L.+Molyneuxauthor=Y.+Ngauthor=S.+R.+Wedgeauthor=M.+E.+M.+Nobleauthor=C.+Cano&title=Identification+of+a+novel+ligand+for+the+ATAD2+bromodomain+with+selectivity+over+BRD4+through+a+fragment+growing+approach&doi=10.1039%2FC8OB00099A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span></div><div class="casAuthors">Miller, Duncan C.; Martin, Mathew P.; Adhikari, Santosh; Brennan, Alfie; Endicott, Jane A.; Golding, Bernard T.; Hardcastle, Ian R.; Heptinstall, Amy; Hobson, Stephen; Jennings, Claire; Molyneux, Lauren; Ng, Yvonne; Wedge, Stephen R.; Noble, Martin E. M.; Cano, Celine</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1843-1850</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">ATAD2 is an ATPase that is overexpressed in a variety of cancers and assocd. with a poor patient prognosis.  This protein has been suggested to function as a cofactor for a range of transcription factors, including proto-oncogene MYC and the androgen receptor.  ATAD2 comprises an ATPase domain, implicated in chromatin remodeling, and a bromodomain which allows it to interact with acetylated histone tails.  Dissection of the functional roles of these 2 domains would benefit from the availability of selective, cell-permeable pharmacol. probes.  An in silico evaluation of the 3D structures of various bromodomains suggested that developing small mol. ligands for the bromodomain of ATAD2 is likely to be challenging, although recent reports have shown that ATAD2 bromodomain ligands can be identified.  Here, we report a structure-guided fragment-based approach to identify lead compds. for ATAD2 bromodomain inhibitor development.  Our findings indicated that the ATAD2 bromodomain could accommodate fragment hits (mol. wt., <200) that yield productive structure-activity relations, and structure-guided design enabled the introduction of selectivity over BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKPvd0l4_jgLVg90H21EOLACvtfcHk0lixYiNEvwUkvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D&md5=606eacb758cb091e4c35bf3da13d8298</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC8OB00099A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB00099A%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DA.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DHeptinstall%26aufirst%3DA.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DJennings%26aufirst%3DC.%26aulast%3DMolyneux%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DY.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DCano%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520a%2520novel%2520ligand%2520for%2520the%2520ATAD2%2520bromodomain%2520with%2520selectivity%2520over%2520BRD4%2520through%2520a%2520fragment%2520growing%2520approach%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D1843%26epage%3D1850%26doi%3D10.1039%2FC8OB00099A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-selective ATAD2 chemical probe with novel chemical structure and unusual mode of action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&issue=12&author=A.+E.+Fernandez-Montalvanauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+St%C3%B6ckigtauthor=A.+ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+Diaz-S%C3%A1ezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M+Huberauthor=J.+Hubnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Isoform-selective+ATAD2+chemical+probe+with+novel+chemical+structure+and+unusual+mode+of+action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DSt%25C3%25B6ckigt%26aufirst%3DD.%26aulast%3Dter%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DDiaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DHubner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DIsoform-selective%2520ATAD2%2520chemical%2520probe%2520with%2520novel%2520chemical%2520structure%2520and%2520unusual%2520mode%2520of%2520action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauder, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Fragment-based screening of the bromodomain of ATAD2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9687</span>– <span class="NLM_lpage">9692</span>, <span class="refDoi"> DOI: 10.1021/jm501035j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501035j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9687-9692&author=M.+J.+Harnerauthor=B.+A.+Chauderauthor=J.+Phanauthor=S.+W.+Fesik&title=Fragment-based+screening+of+the+bromodomain+of+ATAD2&doi=10.1021%2Fjm501035j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Screening of the Bromodomain of ATAD2</span></div><div class="casAuthors">Harner, Mary J.; Chauder, Brian A.; Phan, Jason; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9687-9692</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cellular and genetic evidence suggest that inhibition of ATAD2 (AAA domain-contg. protein 2) could be a useful strategy to treat several types of cancer.  To discover small-mol. inhibitors of the bromodomain of ATAD2, we used a fragment-based approach.  Fragment hits were identified using NMR spectroscopy, and ATAD2 was crystd. with three of the hits identified in the fragment screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3781FiAn_x7Vg90H21EOLACvtfcHk0lgadZmjZJ23Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE&md5=b3ba9d32e92c8d92b37132214940e9d2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm501035j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501035j%26sid%3Dliteratum%253Aachs%26aulast%3DHarner%26aufirst%3DM.%2BJ.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DFragment-based%2520screening%2520of%2520the%2520bromodomain%2520of%2520ATAD2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9687%26epage%3D9692%26doi%3D10.1021%2Fjm501035j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A chemical probe for the ATAD2 bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11386</span>, <span class="refDoi"> DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&author=P.+Bamboroughauthor=C.-W+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+chemical+probe+for+the+ATAD2+bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lgadZmjZJ23Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520chemical%2520probe%2520for%2520the%2520ATAD2%2520bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span>GSK8814
is available via the Structural Genomic Consortium: <a href="http://www.thesgc.org/" class="extLink">http://www.thesgc.org/</a> (accessed
June 28, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GSK8814%0Ais+available+via+the+Structural+Genomic+Consortium%3A+http%3A%2F%2Fwww.thesgc.org%2F+%28accessed%0AJune+28%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Aymeric Dolbois, Laurent Batiste, Lars Wiedmer, Jing Dong, Manuela Brütsch, Danzhi Huang, Nicholas M. Deerain, Dimitrios Spiliotopoulos, Iván Cheng-Sánchez, Eleen Laul, Cristina Nevado, Paweł Śledź, <span class="NLM_string-name hlFld-ContribAuthor">Amedeo Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Hitting a Moving Target: Simulation and Crystallography Study of ATAD2 Bromodomain Blockers. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1573-1580. <a href="https://doi.org/10.1021/acsmedchemlett.0c00080" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00080</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00080%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHitting%252Ba%252BMoving%252BTarget%25253A%252BSimulation%252Band%252BCrystallography%252BStudy%252Bof%252BATAD2%252BBromodomain%252BBlockers%26aulast%3DDolbois%26aufirst%3DAymeric%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29042020%26date%3D30062020%26date%3D06072020%26date%3D30062020%26volume%3D11%26issue%3D8%26spage%3D1573%26epage%3D1580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul Bamborough, Chun-wa Chung, Emmanuel H. Demont, Angela M. Bridges, Peter D. Craggs, David P. Dixon, Peter Francis, Rebecca C. Furze, Paola Grandi, Emma J. Jones, Bhumika Karamshi, Kelly Locke, Simon C. C. Lucas, Anne-Marie Michon, Darren J. Mitchell, Peter Pogány, Rab K. Prinjha, Christina Rau, Ana Maria Roa, Andrew D. Roberts, Robert J. Sheppard, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Watson</span>. </span><span class="cited-content_cbyCitation_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (16)
                                     , 7506-7525. <a href="https://doi.org/10.1021/acs.jmedchem.9b00673" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00673</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BQualified%252BSuccess%25253A%252BDiscovery%252Bof%252Ba%252BNew%252BSeries%252Bof%252BATAD2%252BBromodomain%252BInhibitors%252Bwith%252Ba%252BNovel%252BBinding%252BMode%252BUsing%252BHigh-Throughput%252BScreening%252Band%252BHit%252BQualification%26aulast%3DBamborough%26aufirst%3DPaul%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23042019%26date%3D09082019%26volume%3D62%26issue%3D16%26spage%3D7506%26epage%3D7525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Faraz  Shaikh</span>, <span class="hlFld-ContribAuthor ">Hio Kuan  Tai</span>, <span class="hlFld-ContribAuthor ">Nirali  Desai</span>, <span class="hlFld-ContribAuthor ">Shirley W. I.  Siu</span>. </span><span class="cited-content_cbyCitation_article-title">LigTMap: ligand and structure-based target identification and activity prediction for small molecular compounds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-021-00523-1" title="DOI URL">https://doi.org/10.1186/s13321-021-00523-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-021-00523-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-021-00523-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DLigTMap%25253A%252Bligand%252Band%252Bstructure-based%252Btarget%252Bidentification%252Band%252Bactivity%252Bprediction%252Bfor%252Bsmall%252Bmolecular%252Bcompounds%26aulast%3DShaikh%26aufirst%3DFaraz%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditi  Nayak</span>, <span class="hlFld-ContribAuthor ">Madhuri  Dutta</span>, <span class="hlFld-ContribAuthor ">Anasuya  Roychowdhury</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging oncogene ATAD2: Signaling cascades and therapeutic initiatives. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2021,</strong> <em>276 </em>, 119322. <a href="https://doi.org/10.1016/j.lfs.2021.119322" title="DOI URL">https://doi.org/10.1016/j.lfs.2021.119322</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2021.119322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2021.119322%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DEmerging%252Boncogene%252BATAD2%25253A%252BSignaling%252Bcascades%252Band%252Btherapeutic%252Binitiatives%26aulast%3DNayak%26aufirst%3DAditi%26date%3D2021%26volume%3D276%26spage%3D119322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Zhang</span>, <span class="hlFld-ContribAuthor ">Shan  Li</span>, <span class="hlFld-ContribAuthor ">Kai-Wen  Cheng</span>, <span class="hlFld-ContribAuthor ">Tsui-Fen  Chou</span>. </span><span class="cited-content_cbyCitation_article-title">AAA ATPases as therapeutic targets: Structure, functions, and small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>219 </em>, 113446. <a href="https://doi.org/10.1016/j.ejmech.2021.113446" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113446%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAAA%252BATPases%252Bas%252Btherapeutic%252Btargets%25253A%252BStructure%25252C%252Bfunctions%25252C%252Band%252Bsmall-molecule%252Binhibitors%26aulast%3DZhang%26aufirst%3DGang%26date%3D2021%26volume%3D219%26spage%3D113446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dahong  Yao</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinhui  Wang</span>, <span class="hlFld-ContribAuthor ">Dabo  Pan</span>, <span class="hlFld-ContribAuthor ">Zhendan  He</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel ATAD2 bromodomain inhibitors that trigger apoptosis and autophagy in breast cells by structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2020,</strong> <em>35 </em>
                                    (1)
                                     , 713-725. <a href="https://doi.org/10.1080/14756366.2020.1740924" title="DOI URL">https://doi.org/10.1080/14756366.2020.1740924</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2020.1740924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2020.1740924%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252BATAD2%252Bbromodomain%252Binhibitors%252Bthat%252Btrigger%252Bapoptosis%252Band%252Bautophagy%252Bin%252Bbreast%252Bcells%252Bby%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DYao%26aufirst%3DDahong%26date%3D2020%26date%3D2020%26volume%3D35%26issue%3D1%26spage%3D713%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemische Epigenetik: der Einfluss chemischer und chemo‐biologischer Techniken auf die Zielstruktur‐Validierung von Bromodomänen. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2019,</strong> <em>131 </em>
                                    (50)
                                     , 18096-18120. <a href="https://doi.org/10.1002/ange.201812164" title="DOI URL">https://doi.org/10.1002/ange.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DChemische%252BEpigenetik%25253A%252Bder%252BEinfluss%252Bchemischer%252Bund%252Bchemo%2525E2%252580%252590biologischer%252BTechniken%252Bauf%252Bdie%252BZielstruktur%2525E2%252580%252590Validierung%252Bvon%252BBromodom%2525C3%2525A4nen%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26date%3D2019%26volume%3D131%26issue%3D50%26spage%3D18096%26epage%3D18120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2019,</strong> <em>58 </em>
                                    (50)
                                     , 17930-17952. <a href="https://doi.org/10.1002/anie.201812164" title="DOI URL">https://doi.org/10.1002/anie.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DChemical%252BEpigenetics%25253A%252BThe%252BImpact%252Bof%252BChemical%252Band%252BChemical%252BBiology%252BTechniques%252Bon%252BBromodomain%252BTarget%252BValidation%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26volume%3D58%26issue%3D50%26spage%3D17930%26epage%3D17952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashley  Jarvis</span>, <span class="hlFld-ContribAuthor ">Gilles  Ouvry</span>. </span><span class="cited-content_cbyCitation_article-title">Essential ingredients for rational drug design. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (20)
                                     , 126674. <a href="https://doi.org/10.1016/j.bmcl.2019.126674" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.126674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.126674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.126674%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DEssential%252Bingredients%252Bfor%252Brational%252Bdrug%252Bdesign%26aulast%3DJarvis%26aufirst%3DAshley%26date%3D2019%26volume%3D29%26issue%3D20%26spage%3D126674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael A.  Clegg</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Advancements in the Development of non‐BET Bromodomain Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (4)
                                     , 362-385. <a href="https://doi.org/10.1002/cmdc.201800738" title="DOI URL">https://doi.org/10.1002/cmdc.201800738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DAdvancements%252Bin%252Bthe%252BDevelopment%252Bof%252Bnon%2525E2%252580%252590BET%252BBromodomain%252BChemical%252BProbes%26aulast%3DClegg%26aufirst%3DMichael%2BA.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D4%26spage%3D362%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>6</b> bound to ATAD2 and BRD4 (same orientation) illustrating the conformational basis for selectivity. In both proteins, the residues of the RVF/WPF motif are shown in pale yellow and the gatekeeper residue in salmon. In ATAD2 (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0">5LJ0</a>), the piperidine ring is triequatorial, positioning the C3′-substituent upon the RVF shelf where its CF<sub>2</sub> group interacts with R1077. In BRD4 (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ1">5LJ1</a>), the triaxial piperidine geometry moves the C3′ substituent away from the WPF shelf into solvent.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ATAD2 crystal structures of the NH piperidine GSK8814 (<b>6</b>, white, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0">5LJ0</a>) and the simplified tropane <b>10</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDN">6HDN</a>). Compound <b>6</b> uses its piperidine C3′-vector to make its CF<sub>2</sub> interaction with R1077, and its naphthyridone 5-position to make its pyridyl interaction with D1014. The analogous vectors are available to the tropane template <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route to Functionalized Tropanes and Piperidines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (Ph)<sub>3</sub>P = CHCOOEt, toluene, 100 °C, 75–99%; (b) X = S, NiCl<sub>2</sub>, NaBH<sub>4</sub>, EtOH, 0 °C, 70%; (c) X = S, <i>m</i>-CPBA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 98%; (d) X = CF<sub>2</sub>, H<sub>2</sub>, Pd/C, MeOH, room temperature, 98%; (e) LiAlH<sub>4</sub>, THF/Et<sub>2</sub>O, −30 °C to room temperature, 88–100%; (f) (COCl)<sub>2</sub>, DMSO, CH<sub>2</sub>Cl<sub>2</sub>, −78 °C then NEt<sub>3</sub>; (g) LDA, THF, −78 °C then <b>13a</b>, <b>b</b>, or <b>c</b>; (h) (CH<sub>3</sub>SO<sub>2</sub>)<sub>2</sub>O, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C; (i) DBU, THF, room temperature, 69–70% over 4 steps; (j) H<sub>2</sub>, Pd/C, MeOH/THF, room temperature, 74–79%; (k) NH<sub>2</sub>OH.HCl, pyridine, EtOH, reflux, 89–100%; Na, EtOH, reflux, 47–76%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of ATAD2 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>t</i>BuONa, BrettPhos, Pd<sub>2</sub>(dba)<sub>3</sub>, THF, 60 °C, 25–94%; (b) TFA, reflux, 63–96%; (c) NBS, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C; (d) ArB(OH)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, Pd(OAc)<sub>2</sub>, di(1-adamantyl)-<i>n</i>-butylphosphine, 1,4-dioxane/water, 100 °C, 71–93% (2 steps); (e) chiral chromatography.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Superimposed crystal structures of the ATAD2-bound racemate of compound <b>27</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HD0">6HD0</a>) and its direct piperidine analogue <b>28</b> (white, PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A83">5A83</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystallographic binding mode of compound <b>26</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ">6HDQ</a>) in BRD4 BD1, showing alternative conformers of Trp81 (orange). This is superimposed on the BRD4 structure of the racemate of <b>1</b> (white, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A85">5A85</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/medium/jm-2018-00862q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray structures of ATAD2 bound to <b>6</b> (white, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0">5LJ0</a>) and BRD4 BD1 bound to <b>26</b> (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ">6HDQ</a>). The magenta dashed line shows the closest contact between <b>26</b> and BRD4 Leu94. Molecular surfaces of the closest side chains to the C3′-position (BRD4 Leu94 and ATAD2 Val1018) highlight the relatively greater space that is available for C3′ substituents in ATAD2 compared to BRD4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-18/acs.jmedchem.8b00862/20180920/images/large/jm-2018-00862q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00862&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67751" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67751" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 40 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petosa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dieuleveult, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitte, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabatel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosset, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoeyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emadali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guardiola, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gérard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span>. <i>J. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjv060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1093%2Fjmcb%2Fmjv060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26459632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=349-362&author=Y.+Morozumiauthor=F.+Boussouarauthor=M.+Tanauthor=A.+Chaikuadauthor=M.+Jamshidikiaauthor=G.+Colakauthor=H.+Heauthor=L.+Nieauthor=C.+Petosaauthor=M.+de+Dieuleveultauthor=S.+Curtetauthor=A.-L.+Vitteauthor=C.+Rabatelauthor=A.+Debernardiauthor=F.-L.+Cossetauthor=E.+Verhoeyenauthor=A.+Emadaliauthor=N.+Schweiferauthor=D.+Gianniauthor=M.+Gutauthor=P.+Guardiolaauthor=S.+Rousseauxauthor=M.+G%C3%A9rardauthor=S.+Knappauthor=Y.+Zhaoauthor=S.+Khochbin&title=Atad2+is+a+generalist+facilitator+of+chromatin+dynamics+in+embryonic+stem+cells&doi=10.1093%2Fjmcb%2Fmjv060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span></div><div class="casAuthors">Morozumi, Yuichi; Boussouar, Faycal; Tan, Minjia; Chaikuad, Apirat; Jamshidikia, Mahya; Colak, Gozde; He, Huang; Nie, Litong; Petosa, Carlo; de Dieuleveult, Maud; Curtet, Sandrine; Vitte, Anne-Laure; Rabatel, Clothilde; Debernardi, Alexandra; Cosset, Francois-Loic; Verhoeyen, Els; Emadali, Anouk; Schweifer, Norbert; Gianni, Davide; Gut, Marta; Guardiola, Philippe; Rousseaux, Sophie; Gerard, Matthieu; Knapp, Stefan; Zhao, Yingming; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-362</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although the conserved AAA ATPase and bromodomain factor, ATAD2, has been described as a transcriptional co-activator upregulated in many cancers, its function remains poorly understood.  Here, using a combination of ChIP-seq, ChIP-proteomics, and RNA-seq expts. in embryonic stem cells where Atad2 is normally highly expressed, we found that Atad2 is an abundant nucleosome-bound protein present on active genes, assocd. with chromatin remodelling, DNA replication, and DNA repair factors.  A structural anal. of its bromodomain and subsequent investigations demonstrate that histone acetylation guides ATAD2 to chromatin, resulting in an overall increase of chromatin accessibility and histone dynamics, which is required for the proper activity of the highly expressed gene fraction of the genome.  While in exponentially growing cells Atad2 appears dispensable for cell growth, in differentiating ES cells Atad2 becomes crit. in sustaining specific gene expression programs, controlling proliferation and differentiation.  Altogether, this work defines Atad2 as a facilitator of general chromatin-templated activities such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMr0U-BpiFIbVg90H21EOLACvtfcHk0ljizkL_-V3IGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D&md5=5a84cdc73ed9b7abeb5595425d07d3eb</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjv060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjv060%26sid%3Dliteratum%253Aachs%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DColak%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DNie%26aufirst%3DL.%26aulast%3DPetosa%26aufirst%3DC.%26aulast%3Dde%2BDieuleveult%26aufirst%3DM.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVitte%26aufirst%3DA.-L.%26aulast%3DRabatel%26aufirst%3DC.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DCosset%26aufirst%3DF.-L.%26aulast%3DVerhoeyen%26aufirst%3DE.%26aulast%3DEmadali%26aufirst%3DA.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DGut%26aufirst%3DM.%26aulast%3DGuardiola%26aufirst%3DP.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DG%25C3%25A9rard%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DAtad2%2520is%2520a%2520generalist%2520facilitator%2520of%2520chromatin%2520dynamics%2520in%2520embryonic%2520stem%2520cells%26jtitle%3DJ.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D8%26spage%3D349%26epage%3D362%26doi%3D10.1093%2Fjmcb%2Fmjv060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERalpha, is required for coregulator occupancy and chromatin modification</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">18067</span>– <span class="NLM_lpage">18072</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705814104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1073%2Fpnas.0705814104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=17998543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18067-18072&author=J.+X.+Zouauthor=A.+S.+Revenkoauthor=L.+B.+Liauthor=A.+T.+Gemoauthor=H.+W.+Chen&title=ANCCA%2C+an+estrogen-regulated+AAA%2B+ATPase+coactivator+for+ERalpha%2C+is+required+for+coregulator+occupancy+and+chromatin+modification&doi=10.1073%2Fpnas.0705814104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERα, is required for coregulator occupancy and chromatin modification</span></div><div class="casAuthors">Zou, June X.; Revenko, Alexey S.; Li, Li B.; Gemo, Abigael T.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18067-18072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AAA+ proteins play crucial roles in diverse biol. processes via their ATPase-driven remodeling of macromol. complexes.  Here the authors report the authors' identification of an evolutionarily conserved AAA+ protein, ANCCA/Pro 2000, endowed with a bromodomain that is strongly induced by estrogen in human breast cancer cells and is a direct target of protooncogene ACTR/AlB1/SRC-3.  The authors found that ANCCA assocs. directly with estrogen-bound estrogen receptor (ER) α and ACTR.  It is selectively recruited, upon estrogen stimulation, to a subset of ERα target genes including cyclin D1, c-myc, and E2F1 and is required for their estrogen-induced expression as well as breast cancer cell proliferation.  Further studies indicate that ANCCA binds and hydrolyzes ATP and is crit. for recruitment of coregulator CBP and histone hyperacetylation at the ER target chromatin.  Moreover, mutations at the ATP binding motifs rendered ANCCA defective as a coactivator in mediating estrogen induction of gene expression.  Together, the authors' findings reveal an unexpected layer of regulatory mechanism in hormone signaling mediated by ANCCA and suggest that hormone-induced assembly of transcriptional coregulator complexes at chromatin is a process facilitated by AAA+ ATPase proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NFjjiBVoRbVg90H21EOLACvtfcHk0ljizkL_-V3IGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO&md5=618fcc431bf4b0a7589213fea55732ca</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705814104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705814104%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DLi%26aufirst%3DL.%2BB.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252C%2520an%2520estrogen-regulated%2520AAA%252B%2520ATPase%2520coactivator%2520for%2520ERalpha%252C%2520is%2520required%2520for%2520coregulator%2520occupancy%2520and%2520chromatin%2520modification%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D18067%26epage%3D18072%26doi%3D10.1073%2Fpnas.0705814104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardiff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA/ATAD2 overexpression identifies breast cancer patients with poor prognosis, acting to drive proliferation and survival of triple-negative cells through control of B-Myb and EZH2</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">9402</span>– <span class="NLM_lpage">9412</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-1199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F0008-5472.CAN-10-1199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20864510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=9402-9412&author=E.+V.+Kalashnikovaauthor=A.+S.+Revenkoauthor=A.+T.+Gemoauthor=N.+P.+Andrewsauthor=C.+G.+Tepperauthor=J.+X.+Zouauthor=R.+D.+Cardiffauthor=A.+D.+Borowskyauthor=H.+W.+Chen&title=ANCCA%2FATAD2+overexpression+identifies+breast+cancer+patients+with+poor+prognosis%2C+acting+to+drive+proliferation+and+survival+of+triple-negative+cells+through+control+of+B-Myb+and+EZH2&doi=10.1158%2F0008-5472.CAN-10-1199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA/ATAD2 Overexpression Identifies Breast Cancer Patients with Poor Prognosis, Acting to Drive Proliferation and Survival of Triple-Negative Cells through Control of B-Myb and EZH2</span></div><div class="casAuthors">Kalashnikova, Ekaterina V.; Revenko, Alexey S.; Gemo, Abigael T.; Andrews, Nicolas P.; Tepper, Clifford G.; Zou, June X.; Cardiff, Robert D.; Borowsky, Alexander D.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9402-9412</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Chromatin coregulators are important factors in tumorigenesis and cancer progression.  ANCCA is an AAA+ ATPase and a bromodomain-contg. nuclear coactivator for the estrogen and androgen receptors that is crucial for assembly of chromatin-modifying complexes and proliferation of hormone-responsive cancer cells.  In this study, we show that ANCCA is overexpressed in >70% of breast tumors and that its high protein level correlates well with tumor histol. grades (P < 0.0001), highlighting ANCCA as a prognostic factor for poor overall survival and disease recurrence.  Strikingly, high-level ANCCA correlated with triple-neg. tumors that represent highly aggressive disease.  Anal. of ANCCA transcript levels in multiple expression profiles of breast cancer identified ANCCA as a common signature gene, indicating that elevated transcripts also strongly correlate with tumor metastasis and poor survival.  Biol. and mechanistic investigations revealed that ANCCA is crucial for proliferation and survival of triple-neg./basal-like cancer cells and that it controls the expression of B-Myb, histone methyltransferase EZH2, and an Rb-E2F core program for proliferation, along with a subset of key mitotic kinesins and cell survival genes (IRS2, VEGF, and Akt1).  In particular, ANCCA overexpression correlated strongly with EZH2 in tumors.  Our results suggest that ANCCA may integrate multiple oncogenic programs in breast cancer, serving in particular as a prognostic marker and a therapeutic target for triple-neg. cancers.  Cancer Res; 70(22); 9402-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-adY-FJac6rVg90H21EOLACvtfcHk0ljizkL_-V3IGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP&md5=d7f74575c9c8cf4ebebdcfa5e3cc45fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-1199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-1199%26sid%3Dliteratum%253Aachs%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DAndrews%26aufirst%3DN.%2BP.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DCardiff%26aufirst%3DR.%2BD.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252FATAD2%2520overexpression%2520identifies%2520breast%2520cancer%2520patients%2520with%2520poor%2520prognosis%252C%2520acting%2520to%2520drive%2520proliferation%2520and%2520survival%2520of%2520triple-negative%2520cells%2520through%2520control%2520of%2520B-Myb%2520and%2520EZH2%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D9402%26epage%3D9412%26doi%3D10.1158%2F0008-5472.CAN-10-1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lestrat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoffier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virolle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">5171</span>– <span class="NLM_lpage">5181</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+characterization+of+ATAD2+as+a+new+cancer%2Ftestis+factor+and+a+predictor+of+poor+prognosis+in+breast+and+lung+cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0liOKQEmKvvZyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520characterization%2520of%2520ATAD2%2520as%2520a%2520new%2520cancer%252Ftestis%2520factor%2520and%2520a%2520predictor%2520of%2520poor%2520prognosis%2520in%2520breast%2520and%2520lung%2520cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fouret, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffaire, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beau-Faller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazieres, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Validire, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camilleri-Broet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaylet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroy-Ladurie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouret, P.</span></span> <span> </span><span class="NLM_article-title">A comparative and integrative approach identifies ATPase family, AAA domain containing 2 as a likely driver of cell proliferation in lung adenocarcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5606</span>– <span class="NLM_lpage">5616</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-0505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1078-0432.CCR-12-0505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=22914773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFahu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5606-5616&author=R.+Fouretauthor=J.+Laffaireauthor=P.+Hofmanauthor=M.+Beau-Fallerauthor=J.+Mazieresauthor=P.+Validireauthor=P.+Girardauthor=S.+Camilleri-Broetauthor=F.+Vayletauthor=F.+Leroy-Ladurieauthor=J.+C.+Soriaauthor=P.+Fouret&title=A+comparative+and+integrative+approach+identifies+ATPase+family%2C+AAA+domain+containing+2+as+a+likely+driver+of+cell+proliferation+in+lung+adenocarcinoma&doi=10.1158%2F1078-0432.CCR-12-0505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparative and Integrative Approach Identifies ATPase Family, AAA Domain Containing 2 as a Likely Driver of Cell Proliferation in Lung Adenocarcinoma</span></div><div class="casAuthors">Fouret, Robert; Laffaire, Julien; Hofman, Paul; Beau-Faller, Michele; Mazieres, Julien; Validire, Pierre; Girard, Philippe; Camilleri-Broeet, Sophie; Vaylet, Fabien; Leroy-Ladurie, Francois; Soria, Jean-Charles; Fouret, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5606-5616</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The aim was to identify genetic changes that could drive cancer pathogenesis in never and ever smokers with lung adenocarcinoma.  We analyzed the copy no. and gene expression profiles of lung adenocarcinomas in 165 patients and related the alterations to smoking status.  Having found differences in the tumor profiles, we integrated copy no. and gene expression data from 80 paired samples.  Amplifications at 8q24.12 overlapping MYC and ATAD2 were more frequent in ever smokers.  Unsupervised anal. of gene expression revealed two groups: in the group with mainly never smokers, the tumors expressed genes common to normal lung; in the group with more ever smokers, the tumors expressed "proliferative" and "invasive" gene clusters.  Integration of copy no. and gene expression data identified one module enriched in mitotic genes and MYC targets.  Its main assocd. modulator was ATAD2, a cofactor of MYC.  A strong dose-response relationship between ATAD2 and proliferation-related gene expression was noted in both never and ever smokers, which was verified in two independent cohorts.  Both ATAD2 and MYC expression correlated with 8q24.12 amplification and were higher in ever smokers.  However, only ATAD2, and not MYC, overexpression explained the behavior of proliferation-related genes and predicted a worse prognosis independently of disease stage in a large validation cohort.  The likely driving force behind MYC contribution to uncontrolled cell proliferation in lung adenocarcinoma is ATAD2.  Deregulation of ATAD2 is mainly related to gene amplification and is more frequent in ever smokers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7I6bbL4KEqLVg90H21EOLACvtfcHk0liOKQEmKvvZyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFahu7rO&md5=939438d8b41c4e0b8f609d228dd6cc9f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-0505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-0505%26sid%3Dliteratum%253Aachs%26aulast%3DFouret%26aufirst%3DR.%26aulast%3DLaffaire%26aufirst%3DJ.%26aulast%3DHofman%26aufirst%3DP.%26aulast%3DBeau-Faller%26aufirst%3DM.%26aulast%3DMazieres%26aufirst%3DJ.%26aulast%3DValidire%26aufirst%3DP.%26aulast%3DGirard%26aufirst%3DP.%26aulast%3DCamilleri-Broet%26aufirst%3DS.%26aulast%3DVaylet%26aufirst%3DF.%26aulast%3DLeroy-Ladurie%26aufirst%3DF.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DFouret%26aufirst%3DP.%26atitle%3DA%2520comparative%2520and%2520integrative%2520approach%2520identifies%2520ATPase%2520family%252C%2520AAA%2520domain%2520containing%25202%2520as%2520a%2520likely%2520driver%2520of%2520cell%2520proliferation%2520in%2520lung%2520adenocarcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5606%26epage%3D5616%26doi%3D10.1158%2F1078-0432.CCR-12-0505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">miR-372 down-regulates the oncogene ATAD2 to influence hepatocellular carcinoma proliferation and metastasis</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">118</span>, <span class="refDoi"> DOI: 10.1186/1471-2407-14-107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1186%2F1471-2407-14-107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=107-118&author=G.+Wuauthor=H.+Liuauthor=H.+Heauthor=Y.+Wangauthor=X.+Luauthor=Y.+Yuauthor=S.+Xiaauthor=X.+Mengauthor=Y.+Liu&title=miR-372+down-regulates+the+oncogene+ATAD2+to+influence+hepatocellular+carcinoma+proliferation+and+metastasis&doi=10.1186%2F1471-2407-14-107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-107%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DmiR-372%2520down-regulates%2520the%2520oncogene%2520ATAD2%2520to%2520influence%2520hepatocellular%2520carcinoma%2520proliferation%2520and%2520metastasis%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D107%26epage%3D118%26doi%3D10.1186%2F1471-2407-14-107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Epigenetic high regulation of ATAD2 regulates the Hh pathway in human hepatocellular carcinoma</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.3892/ijo.2014.2416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.3892%2Fijo.2014.2416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24805933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2014&pages=351-361&author=G.+Wuauthor=X.+Luauthor=Y.+Wangauthor=H.+Heauthor=X.+Mengauthor=S.+Xiaauthor=K.+Zhenauthor=Y.+Liu&title=Epigenetic+high+regulation+of+ATAD2+regulates+the+Hh+pathway+in+human+hepatocellular+carcinoma&doi=10.3892%2Fijo.2014.2416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic high regulation of ATAD2 regulates the Hh pathway in human hepatocellular carcinoma</span></div><div class="casAuthors">Wu, Gang; Lu, Xiaojun; Wang, Yawei; He, Hui; Meng, Xiangyu; Xia, Shuguan; Zhen, Kunming; Liu, Yongfeng</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">351-361</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">ATAD2 is assocd. with many cellular progresses such as cell growth, differentiation and apoptosis.  Some studies suggest ATAD2 is highly expressed in cancer cells.  In our previous studies, we found that ATAD2 is highly expressed in HCC tissues, compared with adjacent normal tissues, and patients with high expression of ATAD2 had a poorer prognosis.  Moreover, we found mir-372 can regulate the expression of ATAD2 in HCC cell lines.  We also detected a relationship between the mRNA expression of ATAD2 and Ptch1 by gene microarray.  Here, we completed the function studies of ATAD2 in vivo and in vitro, and tested whether ATAD2 could regulate the Hh pathway.  ATAD2 and Hh pathway protein expressions in 80 HCC specimens were examd. by immunohistochem. (IHC).  The mRNA expression of ATAD2 and Hh pathway members in paired-HCC tissues and cell lines were, resp., analyzed using quant. PCR.  ATAD2-RNAi was transduced into HCCLM3 and Huh7 cells, using a lentiviral vector.  The effect of ATAD2 in HCC cell lines on cell cycle and apoptosis were evaluated by flow cytometry.  Tumorigenicity expts. in nude mice were performed to test the function of ATAD2 in vivo.  Pharmacol. regulation of Hh signaling was performed to test the relation between the ATAD2 and Hh pathways and C-myc.  We found that ATAD2 and Ptch1 were both highly expressed in HCC tissues, compared with paired normal hepatic tissues.  In addn., we found that ATAD2 could affect the expression of the Hh pathway by PCR and western blot anal. in HCC cell lines, by observing the outcome before and after transfection.  We speculate that ATAD2 cooperates with the MYC gene to regulate the expression of SMO and Gli, activating the Hh pathway and inducing an active feedback of the Hh pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooC9ebryKIgrVg90H21EOLACvtfcHk0liR10qa2OEobA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtL%252FK&md5=5b5a12bb3456343d386c5de7d51d49ff</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3892%2Fijo.2014.2416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2014.2416%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DXia%26aufirst%3DS.%26aulast%3DZhen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DEpigenetic%2520high%2520regulation%2520of%2520ATAD2%2520regulates%2520the%2520Hh%2520pathway%2520in%2520human%2520hepatocellular%2520carcinoma%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2014%26volume%3D45%26spage%3D351%26epage%3D361%26doi%3D10.3892%2Fijo.2014.2416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span> <span> </span><span class="NLM_article-title">ATAD2 overexpression is associated with progression and prognosis in colorectal cancer</span>. <i>Jpn. J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1093/jjco/hyv195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1093%2Fjjco%2Fhyv195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26819280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A280%3ADC%252BC28nkvVeltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=222-227&author=M.+Houauthor=R.+Huangauthor=Y.+Songauthor=D.+Fengauthor=Y.+Jiangauthor=M.+Liu&title=ATAD2+overexpression+is+associated+with+progression+and+prognosis+in+colorectal+cancer&doi=10.1093%2Fjjco%2Fhyv195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 overexpression is associated with progression and prognosis in colorectal cancer</span></div><div class="casAuthors">Hou Mingming; Huang Rui; Song Yanni; Feng Di; Jiang Yang; Liu Ming</div><div class="citationInfo"><span class="NLM_cas:title">Japanese journal of clinical oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">222-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  ATPase family AAA domain-containing 2 plays an important role in tumor progression including cell cycle, proliferation, apoptosis and chemoresistance.  However, the expression of ATPase family AAA domain-containing 2 in colorectal cancer and its significance are still unclear.  The aim of this study was to examine the expression of ATPase family AAA domain-containing 2 in colorectal cancer.  METHODS:  Immunohistochemistry was used to determine the expression of ATPase family AAA domain-containing 2 in 155 colorectal cancer and 30 matched adjacent noncancerous tissues.  The correlation of ATPase family AAA domain-containing 2 expression with clinicopathological variables was assessed using chi-square test.  Patient survival was analyzed using the Kaplan-Meier and log-rank tests.  Cox regression was performed for the multivariate analysis of prognostic factors.  RESULTS:  High expression of ATPase family AAA domain-containing 2 was detected in 58.1% of the colorectal cancers and was significantly associated with advanced tumor-node-metastasis stage (P = 0.044), poor differentiation (P = 0.028), deep infiltration (P < 0.001), lymphovascular invasion (P = 0.006), lymph node metastasis (P = 0.024) and recurrence (P = 0.022).  Patients with high ATPase family AAA domain-containing 2 expression had significantly poorer overall survival and disease-free survival (both P < 0.001) when compared with patients with low expression of ATPase family AAA domain-containing 2.  The multivariate analysis showed that ATPase family AAA domain-containing 2 was an independent factor for both overall survival (P = 0.003; hazard ratio (HR): 2.356; 95% confidence interval (CI): 1.335-4.158) and disease-free survival (P = 0.001; HR: 2.643; 95% CI: 1.489-4.693).  CONCLUSIONS:  These results showed that ATPase family AAA domain-containing 2 overexpression was associated with progression and prognosis of colorectal cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyV3TCK4srS_9pSBfyDvp7fW6udTcc2eZu_ux5KSLMzLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nkvVeltA%253D%253D&md5=9d2996cb944237ca53217979f6d88a41</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fjjco%2Fhyv195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjjco%252Fhyv195%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26atitle%3DATAD2%2520overexpression%2520is%2520associated%2520with%2520progression%2520and%2520prognosis%2520in%2520colorectal%2520cancer%26jtitle%3DJpn.%2520J.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D46%26spage%3D222%26epage%3D227%26doi%3D10.1093%2Fjjco%2Fhyv195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wrzeszczynski, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrnes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamalakaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimitrova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucito, R.</span></span> <span> </span><span class="NLM_article-title">Identification of tumor suppressors and oncogenes from genomic and epigenetic features in ovarian cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e28503</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0028503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1371%2Fjournal.pone.0028503" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e28503&author=K.+O.+Wrzeszczynskiauthor=V.+Varadanauthor=J.+Byrnesauthor=E.+Lumauthor=S.+Kamalakaranauthor=D.+A.+Levineauthor=N.+Dimitrovaauthor=M.+Q.+Zhangauthor=R.+Lucito&title=Identification+of+tumor+suppressors+and+oncogenes+from+genomic+and+epigenetic+features+in+ovarian+cancer&doi=10.1371%2Fjournal.pone.0028503"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0028503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0028503%26sid%3Dliteratum%253Aachs%26aulast%3DWrzeszczynski%26aufirst%3DK.%2BO.%26aulast%3DVaradan%26aufirst%3DV.%26aulast%3DByrnes%26aufirst%3DJ.%26aulast%3DLum%26aufirst%3DE.%26aulast%3DKamalakaran%26aufirst%3DS.%26aulast%3DLevine%26aufirst%3DD.%2BA.%26aulast%3DDimitrova%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DM.%2BQ.%26aulast%3DLucito%26aufirst%3DR.%26atitle%3DIdentification%2520of%2520tumor%2520suppressors%2520and%2520oncogenes%2520from%2520genomic%2520and%2520epigenetic%2520features%2520in%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De28503%26doi%3D10.1371%2Fjournal.pone.0028503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raeder, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkeland, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trovik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krakstad, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shehata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zack, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krohn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wik, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefansson, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holst, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalland, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akslen, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroukhim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, H. B.</span></span> <span> </span><span class="NLM_article-title">Integrated genomic analysis of the 8q24 amplification in endometrial cancers identifies ATAD2 as essential to MYC-dependent cancers</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e54873</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0054873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1371%2Fjournal.pone.0054873" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e54873&author=M.+B.+Raederauthor=E.+Birkelandauthor=J.+Trovikauthor=C.+Krakstadauthor=S.+Shehataauthor=S.+Schumacherauthor=T.+I.+Zackauthor=A.+Krohnauthor=H.+M.+Wernerauthor=S.+E.+Moodyauthor=E.+Wikauthor=I.+M.+Stefanssonauthor=F.+Holstauthor=A.+M.+Oyanauthor=P.+Tamayoauthor=J.+P.+Mesirovauthor=K.+H.+Kallandauthor=L.+A.+Akslenauthor=R.+Simonauthor=R.+Beroukhimauthor=H.+B.+Salvesen&title=Integrated+genomic+analysis+of+the+8q24+amplification+in+endometrial+cancers+identifies+ATAD2+as+essential+to+MYC-dependent+cancers&doi=10.1371%2Fjournal.pone.0054873"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0054873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0054873%26sid%3Dliteratum%253Aachs%26aulast%3DRaeder%26aufirst%3DM.%2BB.%26aulast%3DBirkeland%26aufirst%3DE.%26aulast%3DTrovik%26aufirst%3DJ.%26aulast%3DKrakstad%26aufirst%3DC.%26aulast%3DShehata%26aufirst%3DS.%26aulast%3DSchumacher%26aufirst%3DS.%26aulast%3DZack%26aufirst%3DT.%2BI.%26aulast%3DKrohn%26aufirst%3DA.%26aulast%3DWerner%26aufirst%3DH.%2BM.%26aulast%3DMoody%26aufirst%3DS.%2BE.%26aulast%3DWik%26aufirst%3DE.%26aulast%3DStefansson%26aufirst%3DI.%2BM.%26aulast%3DHolst%26aufirst%3DF.%26aulast%3DOyan%26aufirst%3DA.%2BM.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DKalland%26aufirst%3DK.%2BH.%26aulast%3DAkslen%26aufirst%3DL.%2BA.%26aulast%3DSimon%26aufirst%3DR.%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DSalvesen%26aufirst%3DH.%2BB.%26atitle%3DIntegrated%2520genomic%2520analysis%2520of%2520the%25208q24%2520amplification%2520in%2520endometrial%2520cancers%2520identifies%2520ATAD2%2520as%2520essential%2520to%2520MYC-dependent%2520cancers%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De54873%26doi%3D10.1371%2Fjournal.pone.0054873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fellenberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernd, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delling, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahlten-Hinguranage, A.</span></span> <span> </span><span class="NLM_article-title">Prognostic significance of drug-regulated genes in highgrade osteosarcoma</span>. <i>Mod. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1085</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi"> DOI: 10.1038/modpathol.3800937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fmodpathol.3800937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=17660802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCku7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2007&pages=1085-1094&author=J.+Fellenbergauthor=L.+Berndauthor=G.+Dellingauthor=D.+Witteauthor=A.+Zahlten-Hinguranage&title=Prognostic+significance+of+drug-regulated+genes+in+highgrade+osteosarcoma&doi=10.1038%2Fmodpathol.3800937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic significance of drug-regulated genes in high-grade osteosarcoma</span></div><div class="casAuthors">Fellenberg, Joerg; Bernd, Ludger; Delling, Guenter; Witte, Daniela; Zahlten-Hinguranage, Anita</div><div class="citationInfo"><span class="NLM_cas:title">Modern Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1085-1094</span>CODEN:
                <span class="NLM_cas:coden">MODPEO</span>;
        ISSN:<span class="NLM_cas:issn">0893-3952</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">About 25-45% of patients with high-grade osteosarcoma poorly respond to chemotherapy with an increased risk of relapse and the development of metastasis.  Therefore, the aim of this study was the evaluation of the prognostic value of eight previously identified drug-regulated candidate genes on osteosarcoma therapy outcome.  Gene expression of 8 candidate genes was analyzed in 35 formalin-fixed, paraffin-embedded, laser-microdissected osteosarcoma biopsies.  The prognostic value of these genes was evaluated by the correlation of gene expression with therapy outcome, overall survival and event-free survival in univariate and multivariate anal.  Upon univariate anal., the expression of MALAT-1, IMPDH2, FTL and RHOA significantly correlated with response to chemotherapy.  Expression of all four genes was increased in the poor responder group.  Upon multivariate anal., IMPDH2 maintained its independent prognostic value (P=0.025).  Concerning the overall survival of the patients, we obsd. a significant assocn. with the expression of FTL, PHB, ATAD2, ACTN1 and RRM2 as well as lactate dehydrogenase serum levels.  In the subgroups of patients with high expression of these genes and those with elevated lactate dehydrogenase levels, the mean overall survival was decreased 1.7-, 1.9-, 2.2-, 2.4-, 1.5- and 4.5-fold, resp.  Except RRM2, all genes and lactate dehydrogenase serum levels remained significant in the multivariate anal.  In addn., the event-free survival was significantly decreased in the subgroups of patients with high FTL, ATAD2 and IMPDH2 expression (1.8-, 6.3- and 2.4-fold, resp.).  These data demonstrate that among the identified genes are valuable markers for the prediction of osteosarcoma therapy outcome.  Esp. IMPDH2 and FTL are promising candidates for the stratification of osteosarcoma patients into low- and high-risk groups.  Owing to their involvement in drug action these genes may further be potential targets for the modulation of drug sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppkmWTNEifO7Vg90H21EOLACvtfcHk0lj6Y59gfbuCJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCku7%252FO&md5=05061311e1ee1b7b4007e2873a7dcbed</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fmodpathol.3800937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmodpathol.3800937%26sid%3Dliteratum%253Aachs%26aulast%3DFellenberg%26aufirst%3DJ.%26aulast%3DBernd%26aufirst%3DL.%26aulast%3DDelling%26aufirst%3DG.%26aulast%3DWitte%26aufirst%3DD.%26aulast%3DZahlten-Hinguranage%26aufirst%3DA.%26atitle%3DPrognostic%2520significance%2520of%2520drug-regulated%2520genes%2520in%2520highgrade%2520osteosarcoma%26jtitle%3DMod.%2520Pathol.%26date%3D2007%26volume%3D20%26spage%3D1085%26epage%3D1094%26doi%3D10.1038%2Fmodpathol.3800937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is overexpressed in gastric cancer and serves as an independent poor prognostic biomarker</span>. <i>Clin. Transl. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1007/s12094-015-1430-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1007%2Fs12094-015-1430-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26527032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVegt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=776-781&author=M.+Zhangauthor=C.+Zhangauthor=W.+Duauthor=X.+Yangauthor=Z.+Chen&title=ATAD2+is+overexpressed+in+gastric+cancer+and+serves+as+an+independent+poor+prognostic+biomarker&doi=10.1007%2Fs12094-015-1430-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 is overexpressed in gastric cancer and serves as an independent poor prognostic biomarker</span></div><div class="casAuthors">Zhang, M.-J.; Zhang, C.-Z.; Du, W.-J.; Yang, X.-Z.; Chen, Z.-P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Translational Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">776-781</span>CODEN:
                <span class="NLM_cas:coden">CTOLAM</span>;
        ISSN:<span class="NLM_cas:issn">1699-048X</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag Italia S.r.l.</span>)
        </div><div class="casAbstract">Objective: ATPase family, AAA domain contg. 2 (ATAD2) has been found overexpressed in various cancer types and correlated with malignant status and poor prognosis.  However, little is known about the clin. significance of ATAD2 in gastric cancer patients.  The aim of this study was to explore the clin. and prognostic significance of ATAD2 in gastric cancer.  Methods: The mRNA and protein levels expression of ATAD2 were detected in clin. tissue samples by qRT-PCR and immunohistochem., resp.  We examd. the ATAD2 protein expression by immunohistochem.  Furthermore, we analyzed the assocn. between ATAD2 expression and clinicopathol. features including prognosis in 166 gastric cancer samples.  Results: In our results, ATAD2 mRNA and protein were highly expressed in gastric cancer samples.  ATAD2 overexpression was correlated with advanced clin. stage, tumor depth, lymph node metastasis, and distant metastasis.  According to the survival anal., ATAD2 protein overexpression was a poor independent prognostic factor for gastric cancer patients.  Conclusions: In summary, ATAD2 could serve as a prognostic biomarker for gastric cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTZFFLPifZirVg90H21EOLACvtfcHk0lj6Y59gfbuCJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVegt7vJ&md5=7ee3c65a9eb75519202fa614d8698f79</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs12094-015-1430-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12094-015-1430-8%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26atitle%3DATAD2%2520is%2520overexpressed%2520in%2520gastric%2520cancer%2520and%2520serves%2520as%2520an%2520independent%2520poor%2520prognostic%2520biomarker%26jtitle%3DClin.%2520Transl.%2520Oncol.%26date%3D2016%26volume%3D18%26spage%3D776%26epage%3D781%26doi%3D10.1007%2Fs12094-015-1430-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokolov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Kinesin family deregulation coordinated by bromodomain protein ANCCA and histone methyltransferase MLL for breast cancer cell growth, survival, and tamoxifen resistance</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1541-7786.MCR-13-0459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24391143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=539-549&author=J.+X.+Zouauthor=Z.+Duanauthor=J.+Wangauthor=A.+Sokolovauthor=J.+Xuauthor=C.+Z.+Chenauthor=J.+J.+Liauthor=H.+W.+Chen&title=Kinesin+family+deregulation+coordinated+by+bromodomain+protein+ANCCA+and+histone+methyltransferase+MLL+for+breast+cancer+cell+growth%2C+survival%2C+and+tamoxifen+resistance&doi=10.1158%2F1541-7786.MCR-13-0459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Kinesin Family Deregulation Coordinated by Bromodomain Protein ANCCA and Histone Methyltransferase MLL for Breast Cancer Cell Growth, Survival, and Tamoxifen Resistance</span></div><div class="casAuthors">Zou, June X.; Duan, Zhijian; Wang, Junjian; Sokolov, Alex; Xu, Jianzhen; Chen, Christopher Z.; Li, Jian Jian; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">539-549</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kinesins are a superfamily of motor proteins and often deregulated in different cancers.  However, the mechanism of their deregulation has been poorly understood.  Through examg. kinesin gene family expression in estrogen receptor (ER)-pos. breast cancer cells, we found that estrogen stimulation of cancer cell proliferation involves a concerted regulation of specific kinesins.  Estrogen strongly induces expression of 19 kinesin genes such as Kif4A/4B, Kif5A/5B, Kif10, Kif11, Kif15, Kif18A/18B, Kif20A/20B, Kif21, Kif23, Kif24, Kif25, and KifC1, whereas suppresses the expression of seven others, including Kif1A, Kif1C, Kif7, and KifC3.  Interestingly, the bromodomain protein ANCCA/ATAD2, previously shown to be an estrogen-induced chromatin regulator, plays a crucial role in the up- and downregulation of kinesins by estrogen.  Its overexpression drives estrogen-independent upregulation of specific kinesins.  Mechanistically, ANCCA (AAA nuclear coregulator cancer assocd.) mediates E2-dependent recruitment of E2F and MLL1 histone methyltransferase at kinesin gene promoters for gene activation-assocd. H3K4me3 methylation.  Importantly, elevated levels of Kif4A, Kif15, Kif20A, and Kif23 correlate with that of ANCCA in the tumors and with poor relapse-free survival of patients with ER-pos. breast cancer.  Their knockdown strongly impeded proliferation and induced apoptosis of both tamoxifen-sensitive and resistant cancer cells.  Together, the study reveals ANCCA as a key mediator of kinesin family deregulation in breast cancer and the crucial role of multiple kinesins in growth and survival of the tumor cells.  Implications: These findings support the development of novel inhibitors of cancer-assocd. kinesins and their regulator ANCCA for effective treatment of cancers including tamoxifen-resistant breast cancers.  Mol Cancer Res; 12(4); 539-49. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaol_2gF7Tc7Vg90H21EOLACvtfcHk0lj6Y59gfbuCJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D&md5=aa5cda3daca4e8e08dd64827d72a313c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0459%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSokolov%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DKinesin%2520family%2520deregulation%2520coordinated%2520by%2520bromodomain%2520protein%2520ANCCA%2520and%2520histone%2520methyltransferase%2520MLL%2520for%2520breast%2520cancer%2520cell%2520growth%252C%2520survival%252C%2520and%2520tamoxifen%2520resistance%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2014%26volume%3D12%26spage%3D539%26epage%3D549%26doi%3D10.1158%2F1541-7786.MCR-13-0459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Ahsen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toedling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">70323</span>– <span class="NLM_lpage">70335</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.18632%2Foncotarget.11855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=27612420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=70323-70335&author=S.+J.+Kooauthor=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=V.+Badockauthor=C.+J.+Ottauthor=S.+J.+Holtonauthor=O.+von+Ahsenauthor=J.+Toedlingauthor=S.+Vittoriauthor=J.+E.+Bradnerauthor=M.+Gorj%C3%A1n%C3%A1cz&title=ATAD2+is+an+epigenetic+reader+of+newly+synthesized+histone+marks+during+DNA+replication&doi=10.18632%2Foncotarget.11855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span></div><div class="casAuthors">Koo Seong Joo; Fernandez-Montalvan Amaury E; Badock Volker; Holton Simon J; von Ahsen Oliver; Toedling Joern; Gorjanacz Matyas; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Bradner James E; Bradner James E</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">70323-70335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ATAD2 (ATPase family AAA domain-containing protein 2) is a chromatin regulator harboring an AAA+ ATPase domain and a bromodomain, previously proposed to function as an oncogenic transcription co-factor.  Here we suggest that ATAD2 is also required for DNA replication.  ATAD2 is co-expressed with genes involved in DNA replication in various cancer types and predominantly expressed in S phase cells where it localized on nascent chromatin (replication sites).  Our extensive biochemical and cellular analyses revealed that ATAD2 is recruited to replication sites through a direct interaction with di-acetylated histone H4 at K5 and K12, indicative of newly synthesized histones during replication-coupled chromatin reassembly.  Similar to ATAD2-depletion, ectopic expression of ATAD2 mutants that are deficient in binding to these di-acetylation marks resulted in reduced DNA replication and impaired loading of PCNA onto chromatin, suggesting relevance of ATAD2 in DNA replication.  Taken together, our data show a novel function of ATAD2 in cancer and for the first time identify a reader of newly synthesized histone di-acetylation-marks during replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAi97EmAYvEFQKX3ooHxWjfW6udTcc2eY5hjwnpbOqFbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D&md5=38a41ce654ca0579c4c517d9537d6c0c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11855%26sid%3Dliteratum%253Aachs%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3Dvon%2BAhsen%26aufirst%3DO.%26aulast%3DToedling%26aufirst%3DJ.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DATAD2%2520is%2520an%2520epigenetic%2520reader%2520of%2520newly%2520synthesized%2520histone%2520marks%2520during%2520DNA%2520replication%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D70323%26epage%3D70335%26doi%3D10.18632%2Foncotarget.11855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">5260</span>– <span class="NLM_lpage">5272</span>, <span class="refDoi"> DOI: 10.1128/MCB.00484-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1128%2FMCB.00484-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20855524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=5260-5272&author=A.+S.+Revenkoauthor=E.+V.+Kalashnikovaauthor=A.+T.+Gemoauthor=J.+X.+Zouauthor=H.+W.+Chen&title=Chromatin+loading+of+E2F-MLL+complex+by+cancer-associated+coregulator+ANCCA+via+reading+a+specific+histone+mark&doi=10.1128%2FMCB.00484-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span></div><div class="casAuthors">Revenko, Alexey S.; Kalashnikova, Ekaterina V.; Gemo, Abigael T.; Zou, June X.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5260-5272</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Histone modifications are regarded as the carrier of epigenetic memory through cell divisions.  How the marks facilitate cell cycle-dependent gene expression is poorly understood.  The evolutionarily conserved AAA ATPase ANCCA (AAA nuclear coregulator cancer-assocd. protein)/ATAD2 was identified as a direct target of oncogene AIB1/ACTR/SRC-3 and a transcriptional coregulator for estrogen and androgen receptors and is strongly implicated in tumorigenesis.  We report here that ANCCA directly interacts with E2F1 to E2F3 and that its N terminus interacts with both the N and C termini of E2F1.  ANCCA preferentially assocs. via its bromodomain with H3 acetylated at lysine 14 (H3K14ac) and is required for key cell cycle gene expression and cancer cell proliferation.  ANCCA assocs. with chromosomes at late mitosis, and its occupancy at E2F targets peaks at the G1-to-S transition.  Strikingly, ANCCA is required for recruitment of specific E2Fs to their targets and chromatin assembly of the host cell factor 1 (HCF-1)-MLL histone methyltransferase complex.  ANCCA depletion results in a marked decrease of the gene activation-linked H3K4me3 mark.  Bromodomain mutations disable ANCCA function as an E2F coactivator and its ability to promote cancer cell proliferation, while ANCCA overexpression in tumors correlates with tumor growth.  Together, these results suggest that ANCCA acts as a pioneer factor in E2F-dependent gene activation and that a novel mechanism involving ANCCA bromodomain may contribute to cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9jiEXxtPP3rVg90H21EOLACvtfcHk0lilsguokrH2fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D&md5=958e54242a3a8c16ac4bf5b51acd80b8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FMCB.00484-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00484-10%26sid%3Dliteratum%253Aachs%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DChromatin%2520loading%2520of%2520E2F-MLL%2520complex%2520by%2520cancer-associated%2520coregulator%2520ANCCA%2520via%2520reading%2520a%2520specific%2520histone%2520mark%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D5260%26epage%3D5272%26doi%3D10.1128%2FMCB.00484-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Androgen-induced coactivator ANCCA mediates specific androgen receptor signalling in prostate cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">3339</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-3440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F0008-5472.CAN-08-3440" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=3339-3346&author=J.+X.+Zouauthor=L.+Guoauthor=A.+S.+Revenkoauthor=C.+G.+Tepperauthor=A.+T.+Gemoauthor=H.+J.+Kungauthor=H.+W.+Chen&title=Androgen-induced+coactivator+ANCCA+mediates+specific+androgen+receptor+signalling+in+prostate+cancer&doi=10.1158%2F0008-5472.CAN-08-3440"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3440%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DAndrogen-induced%2520coactivator%2520ANCCA%2520mediates%2520specific%2520androgen%2520receptor%2520signalling%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D3339%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-08-3440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsia, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Deregulated E2F and the AAA+ coregulator ANCCA drive proto-oncogene ACTR/AIB1 overexpression in breast cancer</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-09-0095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1541-7786.MCR-09-0095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20124470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVOmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=183-193&author=E.+Y.+Hsiaauthor=E.+V.+Kalashnikovaauthor=A.+S.+Revenkoauthor=J.+X.+Zouauthor=A.+D.+Borowskyauthor=H.+W.+Chen&title=Deregulated+E2F+and+the+AAA%2B+coregulator+ANCCA+drive+proto-oncogene+ACTR%2FAIB1+overexpression+in+breast+cancer&doi=10.1158%2F1541-7786.MCR-09-0095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulated E2F and the AAA+ Coregulator ANCCA Drive Proto-Oncogene ACTR/AIB1 Overexpression in Breast Cancer</span></div><div class="casAuthors">Hsia, Elaine Y. C.; Kalashnikova, Ekaterina V.; Revenko, Alexey S.; Zou, June X.; Borowsky, Alexander D.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-193</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The proto-oncogene ACTR/AIB1, a coactivator for transcription factors such as the nuclear receptors and E2Fs, is frequently overexpressed in various cancers including breast cancers.  However, the underlying mechanism is poorly understood.  Here, we identified several functional, noncanonical E2F binding sites in the ACTR first exon and intron that are crit. for ACTR gene activation.  We also found that the newly identified AAA+ coregulator AAA+ nuclear coregulator cancer assocd. (ANCCA) is recruited to the ACTR promoter and directly controls ACTR expression in breast cancer cells.  Importantly, immunohistochem. anal. indicated that ACTR overexpression is highly correlated with the expression of E2F1 and ANCCA in a cohort of human primary and lymph node-metastasized breast cancer specimens.  Along with previous findings from us and others that ACTR is involved in its own gene regulation, these results suggest that one major mechanism of ACTR overexpression in cancer is the concerted, aberrant function of the nuclear coregulators such as ANCCA and ACTR, and they point to therapeutic strategies that target the Rb-E2F axis and/or the coregulator ANCCA for ACTR-overexpressing cancers.  Mol Cancer Res. 8(2); 183-93.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBYcHe3haHbLVg90H21EOLACvtfcHk0lilsguokrH2fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVOmtro%253D&md5=b2a7dac37ab9fa36fefe489bc64c6534</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-09-0095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-09-0095%26sid%3Dliteratum%253Aachs%26aulast%3DHsia%26aufirst%3DE.%2BY.%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DDeregulated%2520E2F%2520and%2520the%2520AAA%252B%2520coregulator%2520ANCCA%2520drive%2520proto-oncogene%2520ACTR%252FAIB1%2520overexpression%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2010%26volume%3D8%26spage%3D183%26epage%3D193%26doi%3D10.1158%2F1541-7786.MCR-09-0095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Developmental and androgenic regulation of chromatin regulators EZH2 and ANCCA/ATAD2 in the prostate via MLL histone methylase complex</span>. <i>Prostate</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1002/pros.22587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1002%2Fpros.22587" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=455-466&author=Z.+Duanauthor=J.+X.+Zouauthor=P.+Yangauthor=Y.+Wangauthor=A.+D.+Borowskyauthor=A.+C.+Gaoauthor=H.+W.+Chen&title=Developmental+and+androgenic+regulation+of+chromatin+regulators+EZH2+and+ANCCA%2FATAD2+in+the+prostate+via+MLL+histone+methylase+complex&doi=10.1002%2Fpros.22587"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fpros.22587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.22587%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DGao%26aufirst%3DA.%2BC.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DDevelopmental%2520and%2520androgenic%2520regulation%2520of%2520chromatin%2520regulators%2520EZH2%2520and%2520ANCCA%252FATAD2%2520in%2520the%2520prostate%2520via%2520MLL%2520histone%2520methylase%2520complex%26jtitle%3DProstate%26date%3D2013%26volume%3D73%26spage%3D455%26epage%3D466%26doi%3D10.1002%2Fpros.22587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altintas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goutte-Gattat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimitrov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarut, J.</span></span> <span> </span><span class="NLM_article-title">Direct cooperation between androgen receptor and E2F1 reveals a common regulation mechanism for androgen-responsive genes in prostate cells</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1531</span>– <span class="NLM_lpage">1541</span>, <span class="refDoi"> DOI: 10.1210/me.2012-1016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1210%2Fme.2012-1016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=22771493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlentLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1531-1541&author=D.+M.+Altintasauthor=M.+S.+Shuklaauthor=D.+Goutte-Gattatauthor=D.+Angelovauthor=J.+P.+Rouaultauthor=S.+Dimitrovauthor=J.+Samarut&title=Direct+cooperation+between+androgen+receptor+and+E2F1+reveals+a+common+regulation+mechanism+for+androgen-responsive+genes+in+prostate+cells&doi=10.1210%2Fme.2012-1016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Direct cooperation between androgen receptor and E2F1 reveals a common regulation mechanism for androgen-responsive genes in prostate cells</span></div><div class="casAuthors">Altintas, D. M.; Shukla, M. S.; Goutte-Gattat, D.; Angelov, D.; Rouault, J. P.; Dimitrov, S.; Samarut, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1531-1541</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">We have studied the regulation of ATAD2 gene expression by androgens in prostate cells.  ATAD2 is a coactivator of the androgen receptor (AR) and the MYC protein.  We showed that ATAD2 expression is directly regulated by AR via an AR binding sequence (ARBS) located in the distal enhancer of its regulatory region.  The gene is also regulated by the E2F1 transcription factor.  Using knockdown and chromatin immunopptn. technique approaches, we could demonstrate that AR and E2F1 functionally collaborate and phys. interact between each other.  From the anal. of chromatin conformation, we conclude that this cooperation results from a chromatin looping over the ATAD2 promoter region between the ARBS and E2F1 binding site in an androgen-dependent manner.  Furthermore, we could show that several genes overexpressed in prostate cancer and potentially involved in several aspects of tumor development have an ARBS and an E2F1 binding site in their regulatory regions and exhibit the same mechanism of regulation by both transcription factors as ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDhmKxaoJhMbVg90H21EOLACvtfcHk0lhfmTNFW11PqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlentLnO&md5=d5d0654f559baf25eb63232334383b2e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1210%2Fme.2012-1016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2012-1016%26sid%3Dliteratum%253Aachs%26aulast%3DAltintas%26aufirst%3DD.%2BM.%26aulast%3DShukla%26aufirst%3DM.%2BS.%26aulast%3DGoutte-Gattat%26aufirst%3DD.%26aulast%3DAngelov%26aufirst%3DD.%26aulast%3DRouault%26aufirst%3DJ.%2BP.%26aulast%3DDimitrov%26aufirst%3DS.%26aulast%3DSamarut%26aufirst%3DJ.%26atitle%3DDirect%2520cooperation%2520between%2520androgen%2520receptor%2520and%2520E2F1%2520reveals%2520a%2520common%2520regulation%2520mechanism%2520for%2520androgen-responsive%2520genes%2520in%2520prostate%2520cells%26jtitle%3DMol.%2520Endocrinol.%26date%3D2012%26volume%3D26%26spage%3D1531%26epage%3D1541%26doi%3D10.1210%2Fme.2012-1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosperini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quarto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazini, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walfridsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBlane, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucifero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacchiana, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is a novel cofactor for MYC, overexpressed and amplified in aggressive tumors</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">8491</span>– <span class="NLM_lpage">8498</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-2131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F0008-5472.CAN-09-2131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=19843847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlantrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=8491-8498&author=M.+Ciroauthor=E.+Prosperiniauthor=M.+Quartoauthor=U.+Graziniauthor=J.+Walfridssonauthor=F.+McBlaneauthor=P.+Nuciferoauthor=G.+Pacchianaauthor=M.+Capraauthor=J.+Christensenauthor=K.+Helin&title=ATAD2+is+a+novel+cofactor+for+MYC%2C+overexpressed+and+amplified+in+aggressive+tumors&doi=10.1158%2F0008-5472.CAN-09-2131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 Is a Novel Cofactor for MYC, Overexpressed and Amplified in Aggressive Tumors</span></div><div class="casAuthors">Ciro, Marco; Prosperini, Elena; Quarto, Micaela; Grazini, Ursula; Walfridsson, Julian; McBlane, Fraser; Nucifero, Paolo; Pacchiana, Giovanni; Capra, Maria; Christensen, Jesper; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8491-8498</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The E2F and MYC transcription factors are crit. regulators of cell proliferation and contribute to the development of human cancers.  Here, we report on the identification of a novel E2F target gene, ATAD2, the predicted protein product of which contains both a bromodomain and an ATPase domain.  The pRB-E2F pathway regulates ATAD2 expression, which is limiting for the entry into the S phase of the cell cycle.  We show that ATAD2 binds the MYC oncogene and stimulates its transcriptional activity.  ATAD2 maps to chromosome 8q24, 4.3 Mb distal to MYC, in a region that is frequently found amplified in cancer.  Consistent with this, we show that ATAD2 expression is high in several human tumors and that the expression levels correlate with clin. outcome of breast cancer patients.  We suggest that ATAD2 links the E2F and MYC pathways and contributes to the development of aggressive cancer through the enhancement of MYC-dependent transcription. [Cancer Res 2009;69(21):8491-8].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-O94KXJv2i7Vg90H21EOLACvtfcHk0lhfmTNFW11PqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlantrnI&md5=85b462fb8a0c9505d98414ab8d90520b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-2131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-2131%26sid%3Dliteratum%253Aachs%26aulast%3DCiro%26aufirst%3DM.%26aulast%3DProsperini%26aufirst%3DE.%26aulast%3DQuarto%26aufirst%3DM.%26aulast%3DGrazini%26aufirst%3DU.%26aulast%3DWalfridsson%26aufirst%3DJ.%26aulast%3DMcBlane%26aufirst%3DF.%26aulast%3DNucifero%26aufirst%3DP.%26aulast%3DPacchiana%26aufirst%3DG.%26aulast%3DCapra%26aufirst%3DM.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DATAD2%2520is%2520a%2520novel%2520cofactor%2520for%2520MYC%252C%2520overexpressed%2520and%2520amplified%2520in%2520aggressive%2520tumors%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D8491%26epage%3D8498%26doi%3D10.1158%2F0008-5472.CAN-09-2131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mjelle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegre, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aas, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drablos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slupphaug, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saetrom, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krokan, H. E.</span></span> <span> </span><span class="NLM_article-title">Cell cycle regulation of human DNA repair and chromatin remodeling genes</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1016%2Fj.dnarep.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=25881042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFWlsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2015&pages=53-67&author=R.+Mjelleauthor=S.+A.+Hegreauthor=P.+A.+Aasauthor=F.+Drablosauthor=G.+Slupphaugauthor=P.+Saetromauthor=H.+E.+Krokan&title=Cell+cycle+regulation+of+human+DNA+repair+and+chromatin+remodeling+genes&doi=10.1016%2Fj.dnarep.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle regulation of human DNA repair and chromatin remodeling genes</span></div><div class="casAuthors">Mjelle, Robin; Hegre, Siv Anita; Aas, Per Arne; Slupphaug, Geir; Drabloes, Finn; Saetrom, Paal; Krokan, Hans E.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53-67</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Maintenance of a genome requires DNA repair integrated with chromatin remodeling.  We have analyzed six transcriptome data sets and one data set on translational regulation of known DNA repair and remodeling genes in synchronized human cells.  These data are available through our new database: www.dnarepairgenes.com.  Genes that have similar transcription profiles in at least two of our data sets generally agree well with known protein profiles.  In brief, long patch base excision repair (BER) is enriched for S phase genes, whereas short patch BER uses genes essentially equally expressed in all cell cycle phases.  Furthermore, most genes related to DNA mismatch repair, Fanconi anemia and homologous recombination have their highest expression in the S phase.  In contrast, genes specific for direct repair, nucleotide excision repair, as well as non-homologous end joining do not show cell cycle-related expression.  Cell cycle regulated chromatin remodeling genes were most frequently confined to G1/S and S.  These include e.g. genes for chromatin assembly factor 1 (CAF-1) major subunits CHAF1A and CHAF1B; the putative helicases HELLS and ATAD2 that both co-activate E2F transcription factors central in G1/S-transition and recruit DNA repair and chromatin-modifying proteins and DNA double strand break repair proteins; and RAD54L and RAD54B involved in double strand break repair.  TOP2A was consistently most highly expressed in G2, but also expressed in late S phase, supporting a role in regulating entry into mitosis.  Translational regulation complements transcriptional regulation and appears to be a relatively common cell cycle regulatory mechanism for DNA repair genes.  Our results identify cell cycle phases in which different pathways have highest activity, and demonstrate that periodically expressed genes in a pathway are frequently co-expressed.  Furthermore, the data suggest that S phase expression and over-expression of some multifunctional chromatin remodeling proteins may set up feedback loops driving cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRILpuYb39LVg90H21EOLACvtfcHk0lhT2LldcLOUeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFWlsr4%253D&md5=15f76faf18a4b198a119b61c302f6f4b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DMjelle%26aufirst%3DR.%26aulast%3DHegre%26aufirst%3DS.%2BA.%26aulast%3DAas%26aufirst%3DP.%2BA.%26aulast%3DDrablos%26aufirst%3DF.%26aulast%3DSlupphaug%26aufirst%3DG.%26aulast%3DSaetrom%26aufirst%3DP.%26aulast%3DKrokan%26aufirst%3DH.%2BE.%26atitle%3DCell%2520cycle%2520regulation%2520of%2520human%2520DNA%2520repair%2520and%2520chromatin%2520remodeling%2520genes%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D30%26spage%3D53%26epage%3D67%26doi%3D10.1016%2Fj.dnarep.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Ahsen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toedling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">70323</span>– <span class="NLM_lpage">70335</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.18632%2Foncotarget.11855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=27612420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=70323-70335&author=S.+J.+Kooauthor=A.+E.+Fernandez-Montalvanauthor=V.+Badockauthor=C.+J.+Ottauthor=S.+J.+Holtonauthor=O.+von+Ahsenauthor=J.+Toedlingauthor=S.+Vittoriauthor=J.+E.+Bradnerauthor=M.+Gorjanacz&title=ATAD2+is+an+epigenetic+reader+of+newly+synthesized+histone+marks+during+DNA+replication&doi=10.18632%2Foncotarget.11855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 is an epigenetic reader of newly synthesized histone marks during DNA replication</span></div><div class="casAuthors">Koo Seong Joo; Fernandez-Montalvan Amaury E; Badock Volker; Holton Simon J; von Ahsen Oliver; Toedling Joern; Gorjanacz Matyas; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Vittori Sarah; Bradner James E; Ott Christopher J; Bradner James E; Bradner James E</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">70323-70335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ATAD2 (ATPase family AAA domain-containing protein 2) is a chromatin regulator harboring an AAA+ ATPase domain and a bromodomain, previously proposed to function as an oncogenic transcription co-factor.  Here we suggest that ATAD2 is also required for DNA replication.  ATAD2 is co-expressed with genes involved in DNA replication in various cancer types and predominantly expressed in S phase cells where it localized on nascent chromatin (replication sites).  Our extensive biochemical and cellular analyses revealed that ATAD2 is recruited to replication sites through a direct interaction with di-acetylated histone H4 at K5 and K12, indicative of newly synthesized histones during replication-coupled chromatin reassembly.  Similar to ATAD2-depletion, ectopic expression of ATAD2 mutants that are deficient in binding to these di-acetylation marks resulted in reduced DNA replication and impaired loading of PCNA onto chromatin, suggesting relevance of ATAD2 in DNA replication.  Taken together, our data show a novel function of ATAD2 in cancer and for the first time identify a reader of newly synthesized histone di-acetylation-marks during replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAi97EmAYvEFQKX3ooHxWjfW6udTcc2eYEjaYmE0Kh3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptFWhuw%253D%253D&md5=38a41ce654ca0579c4c517d9537d6c0c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11855%26sid%3Dliteratum%253Aachs%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3Dvon%2BAhsen%26aufirst%3DO.%26aulast%3DToedling%26aufirst%3DJ.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DATAD2%2520is%2520an%2520epigenetic%2520reader%2520of%2520newly%2520synthesized%2520histone%2520marks%2520during%2520DNA%2520replication%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D70323%26epage%3D70335%26doi%3D10.18632%2Foncotarget.11855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokolov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Kinesin family deregulation coordinated by bromodomain protein ANCCA and histone methyltransferase MLL for breast cancer cell growth, survival, and tamoxifen resistance</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1158%2F1541-7786.MCR-13-0459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24391143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=539-549&author=J.+X.+Zouauthor=Z.+Duanauthor=J.+Wangauthor=A.+Sokolovauthor=J.+Xuauthor=C.+Z.+Chenauthor=J.+J.+Liauthor=H.+W.+Chen&title=Kinesin+family+deregulation+coordinated+by+bromodomain+protein+ANCCA+and+histone+methyltransferase+MLL+for+breast+cancer+cell+growth%2C+survival%2C+and+tamoxifen+resistance&doi=10.1158%2F1541-7786.MCR-13-0459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Kinesin Family Deregulation Coordinated by Bromodomain Protein ANCCA and Histone Methyltransferase MLL for Breast Cancer Cell Growth, Survival, and Tamoxifen Resistance</span></div><div class="casAuthors">Zou, June X.; Duan, Zhijian; Wang, Junjian; Sokolov, Alex; Xu, Jianzhen; Chen, Christopher Z.; Li, Jian Jian; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">539-549</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kinesins are a superfamily of motor proteins and often deregulated in different cancers.  However, the mechanism of their deregulation has been poorly understood.  Through examg. kinesin gene family expression in estrogen receptor (ER)-pos. breast cancer cells, we found that estrogen stimulation of cancer cell proliferation involves a concerted regulation of specific kinesins.  Estrogen strongly induces expression of 19 kinesin genes such as Kif4A/4B, Kif5A/5B, Kif10, Kif11, Kif15, Kif18A/18B, Kif20A/20B, Kif21, Kif23, Kif24, Kif25, and KifC1, whereas suppresses the expression of seven others, including Kif1A, Kif1C, Kif7, and KifC3.  Interestingly, the bromodomain protein ANCCA/ATAD2, previously shown to be an estrogen-induced chromatin regulator, plays a crucial role in the up- and downregulation of kinesins by estrogen.  Its overexpression drives estrogen-independent upregulation of specific kinesins.  Mechanistically, ANCCA (AAA nuclear coregulator cancer assocd.) mediates E2-dependent recruitment of E2F and MLL1 histone methyltransferase at kinesin gene promoters for gene activation-assocd. H3K4me3 methylation.  Importantly, elevated levels of Kif4A, Kif15, Kif20A, and Kif23 correlate with that of ANCCA in the tumors and with poor relapse-free survival of patients with ER-pos. breast cancer.  Their knockdown strongly impeded proliferation and induced apoptosis of both tamoxifen-sensitive and resistant cancer cells.  Together, the study reveals ANCCA as a key mediator of kinesin family deregulation in breast cancer and the crucial role of multiple kinesins in growth and survival of the tumor cells.  Implications: These findings support the development of novel inhibitors of cancer-assocd. kinesins and their regulator ANCCA for effective treatment of cancers including tamoxifen-resistant breast cancers.  Mol Cancer Res; 12(4); 539-49. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaol_2gF7Tc7Vg90H21EOLACvtfcHk0li9ZaF6HQ9qlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D&md5=aa5cda3daca4e8e08dd64827d72a313c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0459%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSokolov%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DKinesin%2520family%2520deregulation%2520coordinated%2520by%2520bromodomain%2520protein%2520ANCCA%2520and%2520histone%2520methyltransferase%2520MLL%2520for%2520breast%2520cancer%2520cell%2520growth%252C%2520survival%252C%2520and%2520tamoxifen%2520resistance%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2014%26volume%3D12%26spage%3D539%26epage%3D549%26doi%3D10.1158%2F1541-7786.MCR-13-0459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Malignant genome reprogramming by ATAD2</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1829</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2013.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1016%2Fj.bbagrm.2013.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=23831842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=1010-1014&author=F.+Boussouarauthor=M.+Jamshidikiaauthor=Y.+Morozumiauthor=S.+Rousseauxauthor=S.+Khochbin&title=Malignant+genome+reprogramming+by+ATAD2&doi=10.1016%2Fj.bbagrm.2013.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Malignant genome reprogramming by ATAD2</span></div><div class="casAuthors">Boussouar, Faycal; Jamshidikia, Mahya; Morozumi, Yuichi; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1829</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Unscheduled expression of crit. cellular regulators could be central to malignant genome reprogramming and tumor establishment.  One such factor appears to be ATAD2, a remarkably conserved protein normally predominantly expressed in germ cells but almost systematically over-expressed in a variety of unrelated cancers.  The presence of a bromodomain adjacent to an AAA type ATPase domain, points to ATAD2 as a factor preliminarily acting on chromatin structure and function.  Accordingly, ATAD2 has been shown to cooperate with a series of transcription factors and chromatin modifiers to regulate specific set of genes.  Here we discuss our knowledge on ATAD2 to evaluate its role as a cancer driver and its value as a new anti-cancer target.  Upon its activation, ATAD2 through its interaction with defined transcription factors, initiates a loop of transcriptional stimulation of target genes, including ATAD2 itself, leading to enhanced cell proliferation and resistance to apoptosis in an ATAD2-dependent manner.  Approaches aiming at neutralizing ATAD2 activity in cancer, including the use of small mol. inhibitors of its two "druggable" domains, AAA ATPase and bromodomain, could become part of a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp824JMo6Dp4rVg90H21EOLACvtfcHk0li9ZaF6HQ9qlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF&md5=cba5991a2dc5f96f522e1104ae4e4889</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2013.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2013.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DMalignant%2520genome%2520reprogramming%2520by%2520ATAD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2013%26volume%3D1829%26spage%3D1010%26epage%3D1014%26doi%3D10.1016%2Fj.bbagrm.2013.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, H. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ATAD2 in cancer: a pharmacologically challenging but tractable target</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1080%2F14728222.2018.1406921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=29148850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=85-96&author=M.+Hussainauthor=Y.+Zhouauthor=Y.+Songauthor=H.+M.+A.+Hameedauthor=H.+Jiangauthor=Y.+Tuauthor=J.+Zhang&title=ATAD2+in+cancer%3A+a+pharmacologically+challenging+but+tractable+target&doi=10.1080%2F14728222.2018.1406921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 in cancer: a pharmacologically challenging but tractable target</span></div><div class="casAuthors">Hussain, Muzammal; Zhou, Yang; Song, Yu; Hameed, H. M. Adnan; Jiang, Hao; Tu, Yaoquan; Zhang, Jiancun</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-96</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  ATAD2 protein is an emerging oncogene that has strongly been linked to the etiol. of multiple advanced human cancers.  Therapeutically, despite the fact that genetic suppression/knockdown studies have validated it as a compelling drug target for future therapeutic development, recent druggability assessment data suggest that direct targeting of ATAD2's bromodomain (BRD) may be a very challenging task.  ATAD2's BRD has been predicted as a 'difficult to drug' or 'least druggable' target due to the concern that its binding pocket, and the areas around it, seem to be unfeasible for ligand binding.: In this review, after shedding light on the multifaceted roles of ATAD2 in normal physiol. as well as in cancer-etiol., we discuss tech. challenges rendered by ATAD2's BRD active site and the recent drug discovery efforts to find small mol. inhibitors against it.: The identification of a novel low-nanomolar semi-permeable chem. probe against ATAD2's BRD by recent drug discovery campaign has demonstrated it to be a pharmacol. tractable target.  Nevertheless, the development of high quality bioavailable inhibitors against ATAD2 is still a pending task.  Moreover, ATAD2 may also potentially be utilized as a promising target for future development of RNAi-based therapy to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bdigfdvMjbVg90H21EOLACvtfcHk0li9ZaF6HQ9qlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN&md5=5d0ff7c936bb9eff125efeeaa4d275d1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DHameed%26aufirst%3DH.%2BM.%2BA.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DATAD2%2520in%2520cancer%253A%2520a%2520pharmacologically%2520challenging%2520but%2520tractable%2520target%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D85%26epage%3D96%26doi%3D10.1080%2F14728222.2018.1406921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lj-MGZxp3Ww5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lj-MGZxp3Ww5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lj-MGZxp3Ww5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.05.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1016%2Fj.cbpa.2016.05.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=27295577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=58-66&author=N.+H.+Theodoulouauthor=N.+C.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Clinical+progress+and+pharmacology+of+small+molecule+bromodomain+inhibitors&doi=10.1016%2Fj.cbpa.2016.05.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains have emerged as an exciting target class for drug discovery over the past decade.  Research has primarily focused on the bromodomain and extra terminal (BET) family of bromodomains, which has led to the development of multiple small mol. inhibitors and an increasing no. of clin. assets.  The excitement centered on the clin. potential of BET inhibition has stimulated intense interest in the broader family and the growing no. of non-BET bromodomain chem. probes has facilitated phenotypic investigations, implicating these targets in a variety of disease pathways including cancer, inflammation, embryonic development and neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEE0jCnUuyk7Vg90H21EOLACvtfcHk0liqdoIOtFTy6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D&md5=4bcd42755b2dd3cebd7fae417e69e83f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DClinical%2520progress%2520and%2520pharmacology%2520of%2520small%2520molecule%2520bromodomain%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.cbpa.2016.05.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Progress in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1002%2Fcmdc.201500540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=26749027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=477-487&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Progress+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201500540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-487</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain and extra terminal (BET) family of bromodomains have been the focus of extensive research, leading to the development of many potent, selective chem. probes and recent clin. assets.  The profound biol. assocd. with BET bromodomain inhibition has provided a convincing rationale for targeting bromodomains for the treatment of disease.  However, the BET family represents just eight of the at least 56 human bromodomains identified to date.  Until recently, there has been significantly less interest in non-BET bromodomains, leaving a vast area of research and the majority of this new target class yet to be thoroughly investigated.  It has been widely reported that several non-BET bromodomain contg. proteins are assocd. with various diseases including cancer and HIV.  Therefore, the development of chem. probes for non-BET bromodomains will facilitate elucidation of their precise biol. roles and potentially lead to the development of new medicines.  This review summarises the progress made towards the development of non-BET bromodomain chem. probes to date.  In addn., we highlight the potential for future work in this new and exciting area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYLj_VyHOS5bVg90H21EOLACvtfcHk0liqdoIOtFTy6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D&md5=1e1339bf09a10bde8621c18344172fa6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DProgress%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D477%26epage%3D487%26doi%3D10.1002%2Fcmdc.201500540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical probes and inhibitors of bromodomains outside the BET family</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/C6MD00373G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1039%2FC6MD00373G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+probes+and+inhibitors+of+bromodomains+outside+the+BET+family&doi=10.1039%2FC6MD00373G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC6MD00373G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00373G%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520probes%2520and%2520inhibitors%2520of%2520bromodomains%2520outside%2520the%2520BET%2520family%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2FC6MD00373G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lisine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+Coteauthor=S.+Magnuson&title=Disrupting+acetyl-lisine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lisine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of BET bromodomain inhibitors and their role in target validation</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1039/C3MD00291H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1039%2FC3MD00291H" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=288-296&author=S.+Mullerauthor=S.+Knapp&title=Discovery+of+BET+bromodomain+inhibitors+and+their+role+in+target+validation&doi=10.1039%2FC3MD00291H"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FC3MD00291H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00291H%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520BET%2520bromodomain%2520inhibitors%2520and%2520their%2520role%2520in%2520target%2520validation%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D288%26epage%3D296%26doi%3D10.1039%2FC3MD00291H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouzon, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of low-micromolar ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5649</span>– <span class="NLM_lpage">5673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649-5673&author=E.+H.+Demontauthor=C.-W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+J.+Emmonsauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-based+discovery+of+low-micromolar+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0liRfegRRbx15Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520low-micromolar%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5649%26epage%3D5673%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of naphthyridones into potent ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.-W+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-based+optimization+of+naphthyridones+into+potent+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0liRfegRRbx15Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-based%2520optimization%2520of%2520naphthyridones%2520into%2520potent%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heptinstall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molyneux, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedge, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1850</span>, <span class="refDoi"> DOI: 10.1039/C8OB00099A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1039%2FC8OB00099A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=29469144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=1843-1850&author=D.+C.+Millerauthor=M.+P.+Martinauthor=S.+Adhikariauthor=A.+Brennanauthor=J.+A.+Endicottauthor=B.+T.+Goldingauthor=I.+R.+Hardcastleauthor=A.+Heptinstallauthor=S.+Hobsonauthor=C.+Jenningsauthor=L.+Molyneuxauthor=Y.+Ngauthor=S.+R.+Wedgeauthor=M.+E.+M.+Nobleauthor=C.+Cano&title=Identification+of+a+novel+ligand+for+the+ATAD2+bromodomain+with+selectivity+over+BRD4+through+a+fragment+growing+approach&doi=10.1039%2FC8OB00099A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span></div><div class="casAuthors">Miller, Duncan C.; Martin, Mathew P.; Adhikari, Santosh; Brennan, Alfie; Endicott, Jane A.; Golding, Bernard T.; Hardcastle, Ian R.; Heptinstall, Amy; Hobson, Stephen; Jennings, Claire; Molyneux, Lauren; Ng, Yvonne; Wedge, Stephen R.; Noble, Martin E. M.; Cano, Celine</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1843-1850</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">ATAD2 is an ATPase that is overexpressed in a variety of cancers and assocd. with a poor patient prognosis.  This protein has been suggested to function as a cofactor for a range of transcription factors, including proto-oncogene MYC and the androgen receptor.  ATAD2 comprises an ATPase domain, implicated in chromatin remodeling, and a bromodomain which allows it to interact with acetylated histone tails.  Dissection of the functional roles of these 2 domains would benefit from the availability of selective, cell-permeable pharmacol. probes.  An in silico evaluation of the 3D structures of various bromodomains suggested that developing small mol. ligands for the bromodomain of ATAD2 is likely to be challenging, although recent reports have shown that ATAD2 bromodomain ligands can be identified.  Here, we report a structure-guided fragment-based approach to identify lead compds. for ATAD2 bromodomain inhibitor development.  Our findings indicated that the ATAD2 bromodomain could accommodate fragment hits (mol. wt., <200) that yield productive structure-activity relations, and structure-guided design enabled the introduction of selectivity over BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKPvd0l4_jgLVg90H21EOLACvtfcHk0liHuKR3k-pCBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D&md5=606eacb758cb091e4c35bf3da13d8298</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC8OB00099A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB00099A%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DA.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DHeptinstall%26aufirst%3DA.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DJennings%26aufirst%3DC.%26aulast%3DMolyneux%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DY.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DCano%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520a%2520novel%2520ligand%2520for%2520the%2520ATAD2%2520bromodomain%2520with%2520selectivity%2520over%2520BRD4%2520through%2520a%2520fragment%2520growing%2520approach%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D1843%26epage%3D1850%26doi%3D10.1039%2FC8OB00099A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-selective ATAD2 chemical probe with novel chemical structure and unusual mode of action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&issue=12&author=A.+E.+Fernandez-Montalvanauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+St%C3%B6ckigtauthor=A.+ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+Diaz-S%C3%A1ezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M+Huberauthor=J.+Hubnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Isoform-selective+ATAD2+chemical+probe+with+novel+chemical+structure+and+unusual+mode+of+action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFernandez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DSt%25C3%25B6ckigt%26aufirst%3DD.%26aulast%3Dter%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DDiaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DHubner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DIsoform-selective%2520ATAD2%2520chemical%2520probe%2520with%2520novel%2520chemical%2520structure%2520and%2520unusual%2520mode%2520of%2520action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauder, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Fragment-based screening of the bromodomain of ATAD2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9687</span>– <span class="NLM_lpage">9692</span>, <span class="refDoi"> DOI: 10.1021/jm501035j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501035j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9687-9692&author=M.+J.+Harnerauthor=B.+A.+Chauderauthor=J.+Phanauthor=S.+W.+Fesik&title=Fragment-based+screening+of+the+bromodomain+of+ATAD2&doi=10.1021%2Fjm501035j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Screening of the Bromodomain of ATAD2</span></div><div class="casAuthors">Harner, Mary J.; Chauder, Brian A.; Phan, Jason; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9687-9692</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cellular and genetic evidence suggest that inhibition of ATAD2 (AAA domain-contg. protein 2) could be a useful strategy to treat several types of cancer.  To discover small-mol. inhibitors of the bromodomain of ATAD2, we used a fragment-based approach.  Fragment hits were identified using NMR spectroscopy, and ATAD2 was crystd. with three of the hits identified in the fragment screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3781FiAn_x7Vg90H21EOLACvtfcHk0liHuKR3k-pCBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE&md5=b3ba9d32e92c8d92b37132214940e9d2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm501035j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501035j%26sid%3Dliteratum%253Aachs%26aulast%3DHarner%26aufirst%3DM.%2BJ.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DFragment-based%2520screening%2520of%2520the%2520bromodomain%2520of%2520ATAD2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9687%26epage%3D9692%26doi%3D10.1021%2Fjm501035j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A chemical probe for the ATAD2 bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11386</span>, <span class="refDoi"> DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&author=P.+Bamboroughauthor=C.-W+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+chemical+probe+for+the+ATAD2+bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lgHR7GvhWad8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520chemical%2520probe%2520for%2520the%2520ATAD2%2520bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span>GSK8814
is available via the Structural Genomic Consortium: <a href="http://www.thesgc.org/" class="extLink">http://www.thesgc.org/</a> (accessed
June 28, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GSK8814%0Ais+available+via+the+Structural+Genomic+Consortium%3A+http%3A%2F%2Fwww.thesgc.org%2F+%28accessed%0AJune+28%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ0','PDB','5LJ0'); return false;">PDB: 5LJ0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LJ1','PDB','5LJ1'); return false;">PDB: 5LJ1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDN','PDB','6HDN'); return false;">PDB: 6HDN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HD0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HD0','PDB','6HD0'); return false;">PDB: 6HD0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A83" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A83','PDB','5A83'); return false;">PDB: 5A83</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A85" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A85','PDB','5A85'); return false;">PDB: 5A85</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ','PDB','6HDQ'); return false;">PDB: 6HDQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDO','PDB','6HDO'); return false;">PDB: 6HDO</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i37"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00862">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_68663"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00862">10.1021/acs.jmedchem.8b00862</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full ATAD2 and BRD4 BD1 assay and ChromLogD data for exemplified compounds; full assay data for BRD4 BD1 and BD2; assay data for a set of ATAD2 inhibitors at BD1 for BRD2, BRD3, BRD4, and BRDT; comparison of activities of O- and C-linked piperidine derivatives; data collection and refinement statistics for ATAD2 and BRD4 BD1 X-ray structures; OMIT maps for ATAD2 and BRD4 BD1 X-ray structures; side chain torsion angles of Trp81 in the complex with <b>26</b> compared to other structures of BRD4 BD1 from the Protein Data Bank; supplementary methods; LCMS traces (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_001.docx">DOCX</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_001.docx">jm8b00862_si_001.docx (1.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00862/suppl_file/jm8b00862_si_002.csv">jm8b00862_si_002.csv (1.38 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB IDs of new crystal (X-ray) structures: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDN">6HDN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDO">6HDO</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HDQ">6HDQ</a>. Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00862%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-18%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00862" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b39f1e891248","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
